{
 "cells": [
  {
   "cell_type": "code",
   "id": "initial_id",
   "metadata": {
    "collapsed": true,
    "ExecuteTime": {
     "end_time": "2025-08-29T20:16:55.295664Z",
     "start_time": "2025-08-29T20:16:42.724175Z"
    }
   },
   "source": [
    "import os\n",
    "from data_gatherer.data_gatherer import DataGatherer\n",
    "import pandas as pd"
   ],
   "outputs": [],
   "execution_count": 1
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-08-29T20:16:55.317378Z",
     "start_time": "2025-08-29T20:16:55.297728Z"
    }
   },
   "cell_type": "code",
   "source": [
    "df = pd.read_csv(\"exp_input/SAGE_groundtruth.csv\")\n",
    "input_file = \"exp_input/SAGE_input.txt\""
   ],
   "id": "78d2b2bc4106d2ef",
   "outputs": [],
   "execution_count": 2
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-08-29T20:16:55.321895Z",
     "start_time": "2025-08-29T20:16:55.318878Z"
    }
   },
   "cell_type": "code",
   "source": [
    "model_name = \"gpt-4o-mini\"  # \"gemini-2.0-flash\" or \"gpt-4o-mini\"\n",
    "prompt = \"GPT_FDR_FewShot_Syn\"  # \"GPT_from_full_input_Examples\" or \"retrieve_datasets_simple_JSON\"\n",
    "FDR = True\n",
    "semantic_retrieval = False\n",
    "section_filter= None"
   ],
   "id": "cca05a29b7c100d1",
   "outputs": [],
   "execution_count": 3
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-08-29T20:16:55.328492Z",
     "start_time": "2025-08-29T20:16:55.324127Z"
    }
   },
   "cell_type": "code",
   "source": [
    "# write list to a text file\n",
    "with open(input_file, 'r') as f:\n",
    "    pmcids = f.read().splitlines()\n",
    "\n",
    "print(\"Number of PMCIDs:\", len(pmcids))"
   ],
   "id": "a4ae25cbd54d4fe1",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of PMCIDs: 100\n"
     ]
    }
   ],
   "execution_count": 4
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-08-29T20:16:55.344243Z",
     "start_time": "2025-08-29T20:16:55.331660Z"
    }
   },
   "cell_type": "code",
   "source": "dg = DataGatherer(llm_name=model_name, log_level='INFO', process_entire_document=FDR) #, driver_path='../../Firefox/geckodriver', save_dynamic_prompts=True",
   "id": "b79cbd7265b497e1",
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001B[97mdata_gatherer.py - line 266 - INFO - Setting up data fetcher...\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 296 - INFO - Data fetcher setup completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 107 - INFO - DataGatherer orchestrator initialized. Extraction Model: gpt-4o-mini\u001B[0m\n"
     ]
    }
   ],
   "execution_count": 5
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-08-29T20:16:55.348823Z",
     "start_time": "2025-08-29T20:16:55.345427Z"
    }
   },
   "cell_type": "code",
   "source": [
    "# These are the problematic urls for information extraction for GEMINI\n",
    "#urls = [\"https://pmc.ncbi.nlm.nih.gov/articles/PMC9314356\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527/\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC11929800/\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436/\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452/\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8565335/\",\"https://pmc.ncbi.nlm.nih.gov/articles/PMC7778917/\"]\n",
    "# v1 = \"https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360/\"\n",
    "# v2 = \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8006362\"\n",
    "#urls = [\"https://pmc.ncbi.nlm.nih.gov/articles/PMC9710693/\"]\n",
    "\n",
    "# These are the problematic urls for information extraction for GPT - (2025 Aug 29) GPT gets stuck for some reason after 50 calls or so...\n",
    "#urls = [\"https://pmc.ncbi.nlm.nih.gov/articles/PMC11240079\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC5752539\",\"https://pmc.ncbi.nlm.nih.gov/articles/PMC7032692\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10547713\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8102856\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8055881\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8226229\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10769298\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC11659981\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8082263\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8565335\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC9094742\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC6289083\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC7658217\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC9780309\",\"https://pmc.ncbi.nlm.nih.gov/articles/PMC3788619\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10680627\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8131595\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC11420198\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC9915613\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10238095\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452\" , \"https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10836119\",  \"https://pmc.ncbi.nlm.nih.gov/articles/PMC9280291\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC11661334\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC5161470\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277\"]\n",
    "\n",
    "#out = dg.process_articles(urls, semantic_retrieval=semantic_retrieval, section_filter=section_filter, prompt_name=prompt)\n",
    "\n",
    "#out\n",
    "#out['https://pmc.ncbi.nlm.nih.gov/articles/PMC9710693']"
   ],
   "id": "5e0e0500474a6c34",
   "outputs": [],
   "execution_count": 6
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-08-29T20:44:39.387952Z",
     "start_time": "2025-08-29T20:16:55.350156Z"
    }
   },
   "cell_type": "code",
   "source": [
    "input_file = \"exp_input/SAGE_input.txt\"\n",
    "combined_df = dg.run(input_file=input_file, semantic_retrieval=semantic_retrieval, section_filter=section_filter, prompt_name=prompt)"
   ],
   "id": "bfd0572331e32fe2",
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001B[97mdata_gatherer.py - line 639 - INFO - Loaded 100 URLs from file.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 0th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC11792374&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 461 - WARNING - No links found in the section.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 406 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 430 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 443 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 264 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 454 - INFO - Found 10 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1aff46900>, <Element supplementary-material at 0x1aff58940>, <Element supplementary-material at 0x1aff58d80>, <Element supplementary-material at 0x1aff58d00>, <Element supplementary-material at 0x1aff58dc0>, <Element supplementary-material at 0x1aff58d40>, <Element supplementary-material at 0x1aff58f80>, <Element supplementary-material at 0x1aff58f00>, <Element supplementary-material at 0x1aff58fc0>, <Element supplementary-material at 0x1aff59000>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 462 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1aff46900>, <Element supplementary-material at 0x1aff58940>, <Element supplementary-material at 0x1aff58d80>, <Element supplementary-material at 0x1aff58d00>, <Element supplementary-material at 0x1aff58dc0>, <Element supplementary-material at 0x1aff58d40>, <Element supplementary-material at 0x1aff58f80>, <Element supplementary-material at 0x1aff58f00>, <Element supplementary-material at 0x1aff58fc0>, <Element supplementary-material at 0x1aff59000>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 548 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM1: ABCA1 expression also correlated with SASP scores in the same human brain samples (Supplementary Fig. 1 [this file] A). Sn-RNA sequencing analysis also indicated that mTOR activation scores were positively associated with ABCA1 expression in astrocytes, excitatory and inhibitory neurons, microglia, oligodendrocytes, and oligodendrocyte progenitor cells (Supplementary Fig. 1 [this file] B). AD characteristics, such as Braak stage, NFT burden, and neuritic plaque burden, were also positively correlated with mTOR activation in excitatory neurons (Supplementary Fig. 1 [this file] B), where APOE4 was associated with increased mTOR expression in excitatory neurons, oligodendrocytes, and pericytes (Supplementary Fig. 1 [this file] B). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM2: Using SenTraGor (STG, a reagent for detection of lipofuscin) [ 50 [rid=CR50] ] staining in postmortem human brain slices (mid-frontal lobe region) from ROS, we observed higher STG intensity in GFAP-positive astrocytes in AD participants compared to NCI participants with the APOE3/4 genotype (Fig. 2 [rid=Fig2] A, the STG/GFAP staining test is shown in Supplementary Fig. 2 [this file] A), corroborating the increased astrocyte senescence markers in AD observed in the transcriptomic data. \n",
      "Given that ABCA1 is predominantly localized in cell membranes [ 51 [rid=CR51] ], cytosolic proteins were removed (Fig. 2 [rid=Fig2] B, Supplementary Fig. 2 [this file] B). Total membrane ABCA1 levels were higher in APOE3/3 and APOE3/4 carriers with AD dementia than in APOE3/3 and APOE3/4 carriers in the NCI group (Fig. 2 [rid=Fig2] B; full blots are shown in Supplementary Fig. 2 [this file] C). \n",
      "In human postmortem middle frontal lobe tissues, caveolin-1 protein levels were higher in both APOE3/3 and APOE3/4 carriers with AD dementia than in the NCI group, although there was no difference between the APOE3/3 and APOE3/4 genotypes in either group (Fig. 3 [rid=Fig3] M; full blots are shown in Supplementary Fig. 2 [this file] C). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM3: ABCA1 was more abundant in lysosome-enriched fractions in the AD group than in the NCI group within the APOE3/4 genotype but not in the APOE3/3 genotype (Fig. 2 [rid=Fig2] C, full blots are shown in Supplementary Fig. 3 [this file] ), supporting that more ABCA1 was trapped in lysosomes in APOE4 AD. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM4: To explore the underlying mechanism, we treated ABCA1-GFP overexpressing HeLa cells with recombinant ApoE3 (rE3) or ApoE4 (rE4). rE4 treatment resulted in lower total ABCA1 protein levels than rE3 treatment, indicating greater ABCA1 degradation (Supplementary Fig. 4 [this file] A). ABCA1-binding complexes were enriched for proteomic analysis (Fig. 3 [rid=Fig3] A and Supplementary Fig. 4 [this file] B). Proteomic analysis revealed 768 shared proteins between rE3 and rE4 treatments, with 314 proteins unique to rE4 (Supplementary Fig. 4 [this file] C). Nine of these proteins (Supplementary Table 4) showed higher expression in the rE4 treatment group than in the rE3 group (Supplementary Fig. 4 [this file] D). Reducing caveolin-1, but not, AP2B1 expression by siRNA increased plasma membrane ABCA1 levels in mouse primary astrocytes (Fig. 3 [rid=Fig3] F, Supplementary Fig. 4 [this file] E), supporting that caveolae-mediated endocytosis regulated ABCA1 degradation. Surprisingly, reducing caveolin-1 expression significantly decreased ABCA1-mediated cholesterol efflux (Fig. 3 [rid=Fig3] G) despite an increase in total ABCA1 protein levels (Supplementary Fig. 4 [this file] F). Furthermore, overexpression of caveolin-1 in astrocytes enhanced cholesterol efflux (Supplementary Fig. 4 [this file] G). \n",
      "Cholesterol loading, by adding LDL, led to increased caveolin-1 expression (Fig. 3 [rid=Fig3] H and I) and showed a dose-dependent increase in baby hamster kidney (BHK) cells (Supplementary Fig. 4 [this file] H). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM5: Mouse cortices were fractionated using TBSX and guanidine HCl to isolate soluble membrane and insoluble aggregated proteins (Supplementary Fig. 5 [this file] A). \n",
      "To identify the main cell type in which ABCA1 and caveolin-1 interact, we isolated microglia, astrocytes and axon-enriched fraction from the brains of adult mice (Supplementary Fig. 5 [this file] B). Astrocytes were enriched in the CD11b fraction (Supplementary Fig. 5 [this file] C). ABCA1 was mainly expressed in astrocytes and axon-enriched fractions, and lower ABCA1 expression was detected in microglial cells (Supplementary Fig. 5 [this file] D). Caveolin-1 was mainly expressed in astrocytes and microglia but not in axon-enriched fractions (Supplementary Fig. 5 [this file] E). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM6: In agreement, ABCA1 brain knockout mice showed higher p21 intensity in the brain than ABCA1 WT mice (Fig. 4 [rid=Fig4] G, p21 staining test is shown in Supplementary Fig. 6 [this file] A). To indirectly assess lysosomal ABCA1 localization in APOE4-TR brains, tissue slides were double stained with LAMP1 and ABCA1 (Supplementary Fig. 6 [this file] B). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM7: Body weight was not affected by HPCD treatment (Supplementary Fig. 7 [this file] A). Total cholesterol levels in the brain (Fig. 5 [rid=Fig5] B) or plasma (Supplementary Fig. 7 [this file] B) were also no different between HPCD and PBS groups. Lysosomal ABCA1 levels were lower in the HPCD treatment group, as verified by lysosome isolation and immunostaining (Fig. 5 [rid=Fig5] G and H; validation of lysosome isolation is shown in Supplementary Fig. 7 [this file] C). Treatment with HPCD was associated with more efficient cholesterol efflux, as evidenced by increased trends of small HDL particle formation in cerebrospinal fluid (CSF) and plasma, although these changes did not reach statistical significance due to the small sample size (Supplementary Fig. 7 [this file] D-F). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM8: HPCD also diminished the activated astrocytes and microglia (Fig. 6 [rid=Fig6] E), and several inflammatory cytokines (Supplementary Fig. 8 [this file] A), but increased synaptophysin, PSD95, and vGLuT1 levels (Supplementary Fig. 8 [this file] B and 8 [this file] C). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM9: To validate these findings in human cells, we cultured human induced pluripotent stem cell (hiPSC)-derived astrocytes with APOE4/4 genotype (Supplementary Fig. 9 [this file] A). When these human iPSc-derived astrocytes were treated with methyl-β-cyclodextrin (MβCD) after LDL loading, cellular cholesterol accumulation (Supplementary Fig. 9 [this file] B) and IL-1β and CCL2 mRNA levels were decreased (Supplementary Fig. 9 [this file] C). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM10: Clinical and pathological characteristics of the samples are detailed in Supplementarys Table ( 1 [this file] ) and Table ( 2 [this file] ). \n",
      "Detailed information regarding all samples is provided in Supplementary Table 1 [this file] . \n",
      "Detailed information about the samples is provided in Supplementary Table 2 [this file] . \n",
      "Detailed information on these samples is presented in Supplementary Table 3 [this file] . \n",
      "The antibodies used are listed in Supplementary Table 5 [this file] . \n",
      "The primers were listed in Supplementary Table 6 [this file] . \n",
      "Cellular senescence pathway enrichment scores for each sample were calculated using single-sample Gene Set Enrichment Analysis (ssGSEA) (the cellular senescence genes are listed in Supplementary Table 7 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 243 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 79161\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 311300\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 79883 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-b796f997a46e1f3f7b578274d833fcf76597e65ba8cb60314a0bab4333b94439\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 79883 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\"datasets\":[{\"dataset_identifier\":\"syn8456637\",\"repository_reference\":\"https://www.synapse.org/#!Synapse:syn8456637\"},{\"dataset_identifier\":\"syn2580853\",\"repository_reference\":\"https://www.synapse.org/#!Synapse:syn2580853\"},{\"dataset_identifier\":\"n/a\",\"repository_reference\":\"n/a\"}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'syn8456637', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn8456637'}, {'dataset_identifier': 'syn2580853', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn2580853'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 3 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn8456637\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn2580853\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'syn8456637', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn8456637', 'dataset_id': 'syn8456637'}, {'dataset_identifier': 'syn2580853', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn2580853', 'dataset_id': 'syn2580853'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn8456637', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn8456637', 'dataset_id': 'syn8456637'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn8456637\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn8456637, synapse.org, https://www.synapse.org/#!Synapse:syn8456637\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn8456637\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn8456637', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn2580853', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn2580853', 'dataset_id': 'syn2580853'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn2580853\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn2580853, synapse.org, https://www.synapse.org/#!Synapse:syn2580853\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn2580853\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn2580853', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 677 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 605 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 256 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn8456637', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn2580853', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 2 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn8456637\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn8456637\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn8456637 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn8456637 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn8456637 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn8456637 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn8456637 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn8456637\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn8456637\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn2580853\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn2580853\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn2580853 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn2580853 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn2580853 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn2580853 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn2580853 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn2580853\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn2580853\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 2\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 265 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 12, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 477 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 574 - INFO - \n",
      "Progress: 1/100 (1.00%) | Elapsed: 00:00:06 | ETA: 00:10:19\n",
      "\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 1th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC4318527&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 394 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1ab7439d0>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 445 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 547 - INFO - Extracted 23 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 557 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 5 <a> elements with href='#Tab1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 5 <a> elements with href='#Tab1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 5 <a> elements with href='#Tab1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 5 <a> elements with href='#Tab1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 5 <a> elements with href='#Tab1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 5 <a> elements with href='#Tab1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 5 <a> elements with href='#Tab1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 5 <a> elements with href='#Tab1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 5 <a> elements with href='#Tab1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 5 <a> elements with href='#Tab1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 5 <a> elements with href='#Tab1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 5 <a> elements with href='#Tab1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 5 <a> elements with href='#Tab1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 5 <a> elements with href='#Tab1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 5 <a> elements with href='#Tab1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 5 <a> elements with href='#Tab1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#Bib1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for Bib1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#Bib1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for Bib1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#Bib1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for Bib1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#Bib1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for Bib1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 39828\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 138300\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 40550 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-5a92660a4b138d90811961053a409a8f306dd60aaf60cb4d5f9dc63c1c323ce7\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 40550 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\n",
      "  \"datasets\": [\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"http://assemblathon.org/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"http://biofunctionprediction.org/node/8\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"http://genomeinterpretation.org/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"http://www.ebi.ac.uk/msd-srv/capri/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"http://predictioncenter.org/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"http://www.chalearn.org/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"http://www.childrenshospital.org/research-and-innovation/research-initiatives/clarity-challenge\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"https://www.synapse.org/#!Challenges:DREAM\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"http://flowcap.flowsite.org/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"https://www.synapse.org/#!Synapse:syn312572\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"https://sbvimprover.com/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"http://www.innocentive.com/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"http://www.kaggle.com/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"http://www.gencodegenes.org/rgasp/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"http://sequencesqueeze.org/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"http://www.xprize.org/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"https://www.synapse.org/#!Synapse:syn1720047\"\n",
      "    }\n",
      "  ]\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'http://assemblathon.org/'}, {'dataset_identifier': 'n/a', 'repository_reference': 'http://biofunctionprediction.org/node/8'}, {'dataset_identifier': 'n/a', 'repository_reference': 'http://genomeinterpretation.org/'}, {'dataset_identifier': 'n/a', 'repository_reference': 'http://www.ebi.ac.uk/msd-srv/capri/'}, {'dataset_identifier': 'n/a', 'repository_reference': 'http://predictioncenter.org/'}, {'dataset_identifier': 'n/a', 'repository_reference': 'http://www.chalearn.org/'}, {'dataset_identifier': 'n/a', 'repository_reference': 'http://www.childrenshospital.org/research-and-innovation/research-initiatives/clarity-challenge'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.synapse.org/#!Challenges:DREAM'}, {'dataset_identifier': 'n/a', 'repository_reference': 'http://flowcap.flowsite.org/'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn312572'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://sbvimprover.com/'}, {'dataset_identifier': 'n/a', 'repository_reference': 'http://www.innocentive.com/'}, {'dataset_identifier': 'n/a', 'repository_reference': 'http://www.kaggle.com/'}, {'dataset_identifier': 'n/a', 'repository_reference': 'http://www.gencodegenes.org/rgasp/'}, {'dataset_identifier': 'n/a', 'repository_reference': 'http://sequencesqueeze.org/'}, {'dataset_identifier': 'n/a', 'repository_reference': 'http://www.xprize.org/'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn1720047'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 18 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'http://assemblathon.org/'}, {'dataset_identifier': 'n/a', 'repository_reference': 'http://biofunctionprediction.org/node/8'}, {'dataset_identifier': 'n/a', 'repository_reference': 'http://genomeinterpretation.org/'}, {'dataset_identifier': 'n/a', 'repository_reference': 'http://www.ebi.ac.uk/msd-srv/capri/'}, {'dataset_identifier': 'n/a', 'repository_reference': 'http://predictioncenter.org/'}, {'dataset_identifier': 'n/a', 'repository_reference': 'http://www.chalearn.org/'}, {'dataset_identifier': 'n/a', 'repository_reference': 'http://www.childrenshospital.org/research-and-innovation/research-initiatives/clarity-challenge'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.synapse.org/#!Challenges:DREAM'}, {'dataset_identifier': 'n/a', 'repository_reference': 'http://flowcap.flowsite.org/'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn312572'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://sbvimprover.com/'}, {'dataset_identifier': 'n/a', 'repository_reference': 'http://www.innocentive.com/'}, {'dataset_identifier': 'n/a', 'repository_reference': 'http://www.kaggle.com/'}, {'dataset_identifier': 'n/a', 'repository_reference': 'http://www.gencodegenes.org/rgasp/'}, {'dataset_identifier': 'n/a', 'repository_reference': 'http://sequencesqueeze.org/'}, {'dataset_identifier': 'n/a', 'repository_reference': 'http://www.xprize.org/'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn1720047'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 677 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 605 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'http://assemblathon.org/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: http://assemblathon.org/, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: http://assemblathon.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: http://assemblathon.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'assemblathon.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'http://biofunctionprediction.org/node/8'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: http://biofunctionprediction.org/node/8, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: http://biofunctionprediction.org/node/8\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: http://biofunctionprediction.org/node/8\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'biofunctionprediction.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'http://genomeinterpretation.org/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: http://genomeinterpretation.org/, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: http://genomeinterpretation.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: http://genomeinterpretation.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'genomeinterpretation.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'http://www.ebi.ac.uk/msd-srv/capri/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: http://www.ebi.ac.uk/msd-srv/capri/, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: http://www.ebi.ac.uk/msd-srv/capri/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: http://www.ebi.ac.uk/msd-srv/capri/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 673 - INFO - Dataset ID is 'n/a' and repository name from prompt\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 605 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'n/a', 'repository_reference': 'http://www.ebi.ac.uk/msd-srv/capri/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'http://predictioncenter.org/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: http://predictioncenter.org/, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: http://predictioncenter.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: http://predictioncenter.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'predictioncenter.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'http://www.chalearn.org/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: http://www.chalearn.org/, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: http://www.chalearn.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: http://www.chalearn.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'www.chalearn.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'http://www.childrenshospital.org/research-and-innovation/research-initiatives/clarity-challenge'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: http://www.childrenshospital.org/research-and-innovation/research-initiatives/clarity-challenge, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: http://www.childrenshospital.org/research-and-innovation/research-initiatives/clarity-challenge\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: http://www.childrenshospital.org/research-and-innovation/research-initiatives/clarity-challenge\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'www.childrenshospital.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.synapse.org/#!Challenges:DREAM'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Challenges:DREAM, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Challenges:DREAM\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Found partial match for https://www.synapse.org/#!Challenges:DREAM in Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 673 - INFO - Dataset ID is 'n/a' and repository name from prompt\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 605 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.synapse.org/#!Challenges:DREAM'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'http://flowcap.flowsite.org/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: http://flowcap.flowsite.org/, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: http://flowcap.flowsite.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: http://flowcap.flowsite.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'flowcap.flowsite.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn312572'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn312572, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn312572\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn312572 in Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 673 - INFO - Dataset ID is 'n/a' and repository name from prompt\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 605 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn312572'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://sbvimprover.com/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://sbvimprover.com/, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://sbvimprover.com/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://sbvimprover.com/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'sbvimprover.com', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'http://www.innocentive.com/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: http://www.innocentive.com/, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: http://www.innocentive.com/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: http://www.innocentive.com/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'www.innocentive.com', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'http://www.kaggle.com/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: http://www.kaggle.com/, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: http://www.kaggle.com/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: http://www.kaggle.com/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'www.kaggle.com', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'http://www.gencodegenes.org/rgasp/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: http://www.gencodegenes.org/rgasp/, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: http://www.gencodegenes.org/rgasp/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: http://www.gencodegenes.org/rgasp/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'www.gencodegenes.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'http://sequencesqueeze.org/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: http://sequencesqueeze.org/, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: http://sequencesqueeze.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: http://sequencesqueeze.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'sequencesqueeze.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'http://www.xprize.org/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: http://www.xprize.org/, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: http://www.xprize.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: http://www.xprize.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'www.xprize.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn1720047'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn1720047\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn1720047, None, https://www.synapse.org/#!Synapse:syn1720047\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn1720047\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn1720047, identifier: syn1720047\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn1720047\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn1720047 for https://www.synapse.org/#!Synapse:syn1720047\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn1720047 for https://www.synapse.org/#!Synapse:syn1720047\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn1720047 for https://www.synapse.org/#!Synapse:syn1720047\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn1720047 for https://www.synapse.org/#!Synapse:syn1720047\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn1720047 in Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn1720047', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'n/a', 'data_repository': 'assemblathon.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'biofunctionprediction.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'genomeinterpretation.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'predictioncenter.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'www.chalearn.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'www.childrenshospital.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'flowcap.flowsite.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'sbvimprover.com', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'www.innocentive.com', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'www.kaggle.com', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'www.gencodegenes.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'sequencesqueeze.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'www.xprize.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn1720047', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 14 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Skipping dataset 1: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Skipping dataset 2: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Skipping dataset 3: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Skipping dataset 4: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Skipping dataset 5: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Skipping dataset 6: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Skipping dataset 7: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Skipping dataset 8: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Skipping dataset 9: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 10 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Skipping dataset 10: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 11 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Skipping dataset 11: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 12 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Skipping dataset 12: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 13 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Skipping dataset 13: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 14 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn1720047\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn1720047\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn1720047 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn1720047 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn1720047 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn1720047 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn1720047 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 14 with repo: synapse.org and accession_id: syn1720047\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn1720047\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 14: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 14\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 37, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 640 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - Parsed data extraction completed. Elements collected: 3\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 481 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 2th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC4339277&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 394 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b00da950>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 445 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 547 - INFO - Extracted 7 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 557 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#S9'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for S9: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#S9'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for S9: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#S12'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for S12: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 18 <a> elements with href='#SD1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 4802) for SD1: Moreover, samples in the paired-end group had many more mappable reads than those in the single-end group ( Table 1 [href=#T1] , Supplementary Fig. 1 [href=this file] ). Indeed, the two groups showed distinct global patterns of pseudogene expression ( Supplementary Fig. 2 [href=this file] ). For each cancer type, we observed generally weak correlations between the expression level of pseudogenes and their WT genes, which is consistent with the previous study ( Supplementary Fig. 3 [href=this file] ). As observed in Kalyana-Sundaram et al. (2012) , we detected some tumor-lineage-specific pseudogenes (296 from the paired-end group and 41 from the single-end group, Supplementary Fig. 4 [href=this file] ). In addition, for three cancer types with available RNA-seq data from nontumor tissue samples, we identified differentially expressed pseudogenes between tumor and nontumor samples (54 in BRCA, 110 in KIRC and 138 in LUSC, Supplementary Fig. 5 [href=this file] ).\n",
      "For several cancer types with established tumor subtypes, we performed the supervised analysis and revealed substantial numbers of pseudogenes with significant differential expression: 48 in UCEC (endometrioid vs. serous) , 138 in LUSC (basal, classical, primitive and secretory) , 71 in GBM (classical, mesenchymal, neural and proneural) and 547 in BRCA (PAM50 subtypes: luminal A, luminal B, basal-like, Her2-enriched and normal-like) (Methods, Fig. 2a [href=#F2] , Supplementary Data 1 [href=this file] ).\n",
      "The predictive power of pseudogene expression is comparable to those achieved by the mRNA expression profiles, suggesting that both pseudogene and mRNA expression can classify the UCEC subtypes independently ( Supplementary Fig. 6 [href=this file] ).\n",
      "Based on the NMF consensus clustering, 837 BRCA samples can be classified into three distinct subtypes: subtype 1 ( = 144), subtype 2 ( = 390), and subtype 3 ( = 303) ( Fig. 4b [href=#F4] , Supplementary Data 2 [href=this file] ).\n",
      "Based on the 500 pseudogenes with the most variable expression, we were able to classify 446 KIRC samples into two distinct subtypes ( Fig. 5a [href=#F5] , Supplementary Data 3 [href=this file] ). We observed an enrichment of pseudogenes (115 out of 500) with a statistically significant P-value ( < 0.05) ( Fig. 5c [href=#F5] , Supplementary Data 4 [href=this file] ).\n",
      "To further assess the clinical utility of the observed pseudogene-expression subtypes, we classified the KIRC samples into four risk quartiles based on the risk scores (in terms of overall survival) calculated from the multivariate Cox model, employing only clinical variables: low risk group (Q1, = 110), low-medium risk group (Q2, = 111), medium-high risk group (Q3, = 112), and high risk group (Q4, = 112) (Methods, Supplementary Data 3 [href=this file] ).\n",
      "To gain some mechanistic insight into how expressed pseudogenes contribute to the observed KIRC pseudogene-expression subtypes, we performed a systematic analysis ( Supplementary Fig. 9a and Supplementary Data 5 [href=this file] ). Among the WT cognate genes with strong positive correlations with their pseudogenes, we noticed that the survival correlations of individual WT genes with prognostic value match the survival pattern of the pseudogene-expression subtypes: WT genes with better prognosis (potentially tumor suppressors, hazard ratio < 1) show higher expression levels in subtype 1 (the better survival group) and the genes with worse prognosis (potentially oncogenes, hazard ratio > 1) show higher expression levels in subtype 2 (the worse survival group, Supplementary Fig. 8b [href=this file] ). Finally, we examined the classic miRNA decoy model as proposed in Poliseno et al. (2010) and identified 38 such candidates ( Methods and Supplementary Data 5 [href=this file] ). One candidate of interest is the potential regulation of a putative tumor suppressor α-catenin (CTNNA1) by the pseudogene PGOHUM00000257111 through competition for up to 9 shared miRNA regulators ( Supplementary Data 5 [href=this file] ). Indeed, the expression levels of PGOHUM00000257111 were significantly higher in cluster 1 (t-test P = 1.48×10-7), which may lead to the elevated levels of CTNNA1 in subtype 1 ( Supplementary Fig. 8b [href=this file] ) and therefore better survival.\n",
      "Finally, we performed feature selection ( Supplementary Data 6 [href=this file] ) and built the classifier from the whole training set using the best-performing algorithm (with the highest AUC) identified through the cross-validation, and applied it to the test set in order to independently validate the predictive power.\n",
      "We gratefully acknowledge contributions from the TCGA Research Network and its TCGA Pan-Cancer Analysis Working Group (contributing consortium members are listed in Supplementary Note 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#SD2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for SD2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 36186\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 122627\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 36908 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-93619c79f0ce9b49104b4e4083a843c427169eb7cf1dcc823d7305e616394f00\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 36908 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\n",
      "  \"datasets\": [\n",
      "    {\n",
      "      \"dataset_identifier\": \"syn1732077\",\n",
      "      \"repository_reference\": \"https://www.synapse.org/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"PMC4339277\",\n",
      "      \"repository_reference\": \"https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    }\n",
      "  ]\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'syn1732077', 'repository_reference': 'https://www.synapse.org/'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'PMC4339277', 'repository_reference': 'https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277/'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 10 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn1732077\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: PMC4339277\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'syn1732077', 'repository_reference': 'https://www.synapse.org/', 'dataset_id': 'syn1732077'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}, {'dataset_identifier': 'PMC4339277', 'repository_reference': 'https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277/', 'dataset_id': 'PMC4339277'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn1732077', 'repository_reference': 'https://www.synapse.org/', 'dataset_id': 'syn1732077'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn1732077, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn1732077\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn1732077', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 677 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 605 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'PMC4339277', 'repository_reference': 'https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277/', 'dataset_id': 'PMC4339277'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: PMC4339277\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID PMC4339277 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277/, identifier: PMC4339277\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier PMC4339277 for https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier PMC4339277 for https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier PMC4339277 for https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier PMC4339277 for https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier PMC4339277 for https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'PMC4339277', 'data_repository': 'pmc.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn1732077', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'PMC4339277', 'data_repository': 'pmc.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 2 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn1732077\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn1732077\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn1732077 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn1732077 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn1732077 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn1732077 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn1732077 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn1732077\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn1732077\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: PMC4339277\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: pmc.ncbi.nlm.nih.gov, identifier: PMC4339277\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier PMC4339277 for pmc.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier PMC4339277 for pmc.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier PMC4339277 for pmc.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier PMC4339277 for pmc.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier PMC4339277 for pmc.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: pmc.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 976 - WARNING - URL pmc.ncbi.nlm.nih.gov may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for PMC4339277 in pmc.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 2 with repo: pmc.ncbi.nlm.nih.gov and accession_id: PMC4339277\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1254 - WARNING - Repository pmc.ncbi.nlm.nih.gov unknown in Ontology. Skipping dataset page 2.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 2\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 9, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 640 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - Parsed data extraction completed. Elements collected: 5\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 481 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 3th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC8452114)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC8452114\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC8452114&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 394 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC8452114.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b0499710>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC8452114 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC8452114, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 445 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC8452114)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 547 - INFO - Extracted 8 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 557 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 2 <a> elements with href='#SD1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 343) for SD1: We obtained a total of six coexpression modules ( Figure S1 [href=this file] ) in amygdala with the number of genes in each module ranging from 204 (red module) to 3,049 (turquoise). After eigengene extraction and differential analysis, no modules were differentially expressed between COVID-19+ and CTL in AMY ( Figure S1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 2 <a> elements with href='#SD2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 224) for SD2: Relationships between these modules are reported in Figure S2 [href=this file] . Interestingly, the two synaptic modules were not related to each other ( Figure S2 [href=this file] ) despite a similar functional enrichment.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#SD3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 75) for SD3: The top genes for AMY and OBT are reported in Figure S3 [href=this file] .\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 10 <a> elements with href='#SD4'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 1320) for SD4: The top 9 genes for the two regions are reported in Figure 2A [href=#F2] and 2B [href=#F2] , whereas the complete list of DEGs is reported in Table S1 [href=this file] and S2 [href=this file] . Across downregulated genes we observed an enrichment of synaptic and neuronal pathways ( Figure 3 [href=#F3] ; Table S3 [href=this file] ). Finally, synaptic pathways were enriched within downregulated genes ( Figure 3 [href=#F3] ; Table S4 [href=this file] ).\n",
      "Other modules were enriched for development/angiogenesis (green, upregulated, and enriched for End and Per), immune system (tan, upregulated), and synaptic signaling (green-yellow and turquoise, both downregulated and enriched for neuronal cell genes) ( Table S5 [href=this file] ).\n",
      "The only differential pathways we detected were related to platelet activation processes/hemostasis, which was only detected in the olfactory bulb ( Figure 7 [href=#F7] ; Table S6 [href=this file] and S7 [href=this file] ).\n",
      "We detected a total of 4,275 genes only associated with pseudotime in amygdala and 3,102 only associated with pseudotime in olfactory bulb ( Table S8 [href=this file] and Table S9 [href=this file] , respectively).\n",
      "It is important to stress that the two modules are not related (see Table S2 [href=this file] ) representing two distinct biological processes.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#S12'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 34) for S12: ACKNOWLEDGEMENTS [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#S12'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 34) for S12: ACKNOWLEDGEMENTS [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 84005\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 262949\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 84727 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-cbfc57b35b1370da68125d33fe1add5babb15b3bb872c24ae108d7652c02ba9c\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 84727 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\n",
      "  \"datasets\": [\n",
      "    {\n",
      "      \"dataset_identifier\": \"10.1101/2021.09.12.21263291\",\n",
      "      \"repository_reference\": \"https://doi.org/10.1101/2021.09.12.21263291\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    }\n",
      "  ]\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': '10.1101/2021.09.12.21263291', 'repository_reference': 'https://doi.org/10.1101/2021.09.12.21263291'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 10 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: 10.1101/2021.09.12.21263291\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': '10.1101/2021.09.12.21263291', 'repository_reference': 'https://doi.org/10.1101/2021.09.12.21263291', 'dataset_id': '10.1101/2021.09.12.21263291'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': '10.1101/2021.09.12.21263291', 'repository_reference': 'https://doi.org/10.1101/2021.09.12.21263291', 'dataset_id': '10.1101/2021.09.12.21263291'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(10\\.\\d{4,9}/).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: 10.1101/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: 10.1101/2021.09.12.21263291, doi.org, https://doi.org/10.1101/2021.09.12.21263291\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: 10.1101/2021.09.12.21263291\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': '10.1101/2021.09.12.21263291', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 677 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 605 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': '10.1101/2021.09.12.21263291', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 1 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: 10.1101/2021.09.12.21263291\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: doi.org, identifier: 10.1101/2021.09.12.21263291\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.1101/2021.09.12.21263291 for doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for 10.1101/2021.09.12.21263291 in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 1 with repo: doi.org and accession_id: 10.1101/2021.09.12.21263291\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://doi.org/10.1101/2021.09.12.21263291\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 1\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 9, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage', 'link',\n",
      "       'title', 'file_info', 'description', 'source_section', 'section_class',\n",
      "       'download_link', 'file_extension', 'a_attr_href', 'a_attr_class',\n",
      "       'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 640 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - Parsed data extraction completed. Elements collected: 6\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 481 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 4th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC9641692)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC9641692\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC9641692&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 406 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC9641692.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC9641692 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 430 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC9641692, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 443 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 264 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 454 - INFO - Found 3 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b6b39600>, <Element supplementary-material at 0x1b6a4be00>, <Element supplementary-material at 0x1b6a4ab00>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 462 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b6b39600>, <Element supplementary-material at 0x1b6a4be00>, <Element supplementary-material at 0x1b6a4ab00>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 548 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM1: When evaluated on cell-type level, TAPE has the best performance of MAE, and comparable CCC with relatively small variance on real datasets (Supplementary Table 8 [this file] ). \n",
      "Meanwhile, we find that deep learning methods are robust for the cross-protocol prediction, while the performance of some statistical methods will drop if the reference and pseudo-bulk data type (UMI to counts or counts to UMI) are exchanged (Supplementary Figs. 1 [this file] and 2 [this file] ). \n",
      "Detailed comparison results are available in Supplementary Table 8 [this file] . For the “similar” scenario, we investigated TAPE’s performance on distinguishing similar cell types (CD4 T cell and CD8 T cell) in all the three PBMC datasets (Supplementary Fig. 3 [this file] ). With 13 defined cell subtypes (Supplementary Table 1 [this file] ), all the methods can not achieve satisfying results (Supplementary Fig. 4 [this file] , median CCC < 0.1), which clearly shows the common limitation of current methods. \n",
      "In Supplementary Fig. 6 [this file] , the two colors stand for the different methods, and the two coordinates of each dot represent overall MAE and CCC for all cell types respectively. \n",
      "From the results (Supplementary Fig. 10 [this file] ), we can have four conclusions: (1) TAPE’s predictive power is better than CIBERSORTx when the randomly selected DEGs are less than 1000; (2) both methods can achieve good performance when the DEGs are signature genes and there are not any similar cell types; (3) both methods can not distinguish DEGs from CD8 T cell rather than CD4 T cell if the DEGs are randomly selected; (4) CIBERSORTx is better than TAPE if the DEGs are signature genes and there exist similar cell types. \n",
      "The results show that TAPE only predicts and as DEGs in NK cell and no DEGs for other cell types (Supplementary Fig. 8 [this file] ). \n",
      "Firstly, when we study the correlation at the gene level using “overall” mode (Supplementary Fig. 7 [this file] ), about 30% of the predicted genes have the negative correlation. However, this phenomenon can also be observed in CIBERSORTx (Supplementary Fig. 8 [this file] ), which indicates the information loss between bulk samples and inferred cell-type-specific GEP is hard to be reconstructed. Thirdly, we notice that both CIBERSORTx and our method can not distinguish DEGs from similar cell subtypes correctly if the DEGs are not signature genes (Supplementary Fig. 9 [this file] ) which means that their resolution is still limited. But CIBERSORTx has displayed its advantages in distinguishing signature DEGs from similar cell types because of the incorporation of the signature matrix (Supplementary Fig. 10 [this file] ). \n",
      "When we used it to test whether deconvolution methods can achieve good performance with immune cell subtypes, we merged all 30 cell types into 13 cell subtypes (Supplementary Table 1 [this file] ). \n",
      "This step is very important, because TAPE will fail in predicting test bulk data proportions properly without proper filtering (Supplementary Table 2 [this file] and Supplementary Fig. 11 [this file] ) . If the less variable genes are not filtered out, TAPE can not predict a good result because of the noises (Supplementary Table 2 [this file] and Supplementary Fig. 11 [this file] ). \n",
      "Hyperparameter tuning file is available in Supplementary Tables 3 [this file] – 7 [this file] . \n",
      "All other relevant data supporting the key findings of this study are available within the article and its Supplementary Information [this file] files or from the corresponding author upon reasonable request. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM2: Peer reviewer reports [this file] are available. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM3: Further information on research design is available in the Nature Portfolio Reporting Summary [this file] linked to this article. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 243 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 53976\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 198358\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 54698 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-86cb598c908b569d175b1c6665718faa482f3361f477824bea2102482d7d0817\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 54698 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\n",
      "  \"datasets\": [\n",
      "    {\n",
      "      \"dataset_identifier\": \"PMC9641692\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9641692/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE107011\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE107011\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE65133\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65133\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"syn3219045\",\n",
      "      \"repository_reference\": \"https://www.synapse.org/#!Synapse:syn3219045\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE84133\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84133\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE87544\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87544\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE67835\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE67835\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE119117\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE119117\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"https://sctape.readthedocs.io/datasets/\"\n",
      "    }\n",
      "  ]\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'PMC9641692', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9641692/'}, {'dataset_identifier': 'GSE107011', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE107011'}, {'dataset_identifier': 'GSE65133', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65133'}, {'dataset_identifier': 'syn3219045', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn3219045'}, {'dataset_identifier': 'GSE84133', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84133'}, {'dataset_identifier': 'GSE87544', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87544'}, {'dataset_identifier': 'GSE67835', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE67835'}, {'dataset_identifier': 'GSE119117', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE119117'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://sctape.readthedocs.io/datasets/'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 9 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: PMC9641692\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE107011\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE65133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn3219045\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE84133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE87544\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE67835\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE119117\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'PMC9641692', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9641692/', 'dataset_id': 'PMC9641692'}, {'dataset_identifier': 'GSE107011', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE107011', 'dataset_id': 'GSE107011'}, {'dataset_identifier': 'GSE65133', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65133', 'dataset_id': 'GSE65133'}, {'dataset_identifier': 'syn3219045', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn3219045', 'dataset_id': 'syn3219045'}, {'dataset_identifier': 'GSE84133', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84133', 'dataset_id': 'GSE84133'}, {'dataset_identifier': 'GSE87544', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87544', 'dataset_id': 'GSE87544'}, {'dataset_identifier': 'GSE67835', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE67835', 'dataset_id': 'GSE67835'}, {'dataset_identifier': 'GSE119117', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE119117', 'dataset_id': 'GSE119117'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://sctape.readthedocs.io/datasets/', 'dataset_id': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'PMC9641692', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9641692/', 'dataset_id': 'PMC9641692'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: PMC9641692\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID PMC9641692 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9641692/, identifier: PMC9641692\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9641692/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier PMC9641692 for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9641692/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier PMC9641692 for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9641692/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier PMC9641692 for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9641692/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier PMC9641692 for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9641692/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier PMC9641692 for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9641692/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9641692/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'PMC9641692', 'data_repository': 'www.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE107011', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE107011', 'dataset_id': 'GSE107011'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE107011\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE107011, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE107011\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE107011\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE107011', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE65133', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65133', 'dataset_id': 'GSE65133'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE65133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE65133, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE65133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE65133', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn3219045', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn3219045', 'dataset_id': 'syn3219045'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn3219045\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn3219045, synapse.org, https://www.synapse.org/#!Synapse:syn3219045\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn3219045\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn3219045', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE84133', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84133', 'dataset_id': 'GSE84133'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE84133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE84133, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE84133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE84133', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE87544', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87544', 'dataset_id': 'GSE87544'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE87544\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE87544, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87544\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE87544\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE87544', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE67835', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE67835', 'dataset_id': 'GSE67835'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE67835\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE67835, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE67835\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE67835\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE67835', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE119117', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE119117', 'dataset_id': 'GSE119117'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE119117\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE119117, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE119117\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE119117\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE119117', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://sctape.readthedocs.io/datasets/', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://sctape.readthedocs.io/datasets/, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://sctape.readthedocs.io/datasets/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://sctape.readthedocs.io/datasets/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'sctape.readthedocs.io', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 256 - INFO - Augmented dataset links: [{'dataset_identifier': 'PMC9641692', 'data_repository': 'www.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE107011', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE65133', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn3219045', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE84133', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE87544', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE67835', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE119117', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'sctape.readthedocs.io', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 9 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: PMC9641692\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: www.ncbi.nlm.nih.gov, identifier: PMC9641692\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier PMC9641692 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier PMC9641692 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier PMC9641692 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier PMC9641692 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier PMC9641692 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 976 - WARNING - URL www.ncbi.nlm.nih.gov may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for PMC9641692 in www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 1 with repo: www.ncbi.nlm.nih.gov and accession_id: PMC9641692\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1254 - WARNING - Repository www.ncbi.nlm.nih.gov unknown in Ontology. Skipping dataset page 1.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE107011\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE107011\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE107011 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE107011 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE107011 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 2 with repo: geo and accession_id: GSE107011\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE107011\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 2: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE65133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE65133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE65133 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE65133 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE65133 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 3 with repo: geo and accession_id: GSE65133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 3: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn3219045\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn3219045\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3219045 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn3219045 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn3219045 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3219045 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn3219045 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 4 with repo: synapse.org and accession_id: syn3219045\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn3219045\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 4: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE84133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE84133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE84133 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE84133 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE84133 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 5 with repo: geo and accession_id: GSE84133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 5: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE87544\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE87544\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE87544 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE87544 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE87544 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 6 with repo: geo and accession_id: GSE87544\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87544\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 6: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE67835\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE67835\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE67835 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE67835 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE67835 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 7 with repo: geo and accession_id: GSE67835\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE67835\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 7: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE119117\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE119117\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE119117 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE119117 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE119117 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 8 with repo: geo and accession_id: GSE119117\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE119117\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 8: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Skipping dataset 9: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 9\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 265 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 12, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 477 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 5th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC5161470)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC5161470\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC5161470&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 394 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC5161470.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x11173bcd0>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC5161470 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC5161470, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 445 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC5161470)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 547 - INFO - Extracted 19 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 557 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#pone.0167994.s001'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 95) for pone.0167994.s001: In both results the tagging SNPs are spread across the chromosome ( S1 Fig [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#pone.0167994.s002'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 207) for pone.0167994.s002: To compare the minimum numbers of the tag SNPs that were identified by the two methods to reach >95% coverage, we tested the difference of the tag SNP numbers using the t-test ( S1 Table [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#pone.0167994.s003'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 87) for pone.0167994.s003: The comparison of the discovered tag SNP sets are given in S2 Table [href=this file] .\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 3 <a> elements with href='#pone.0167994.s004'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 284) for pone.0167994.s004: An example output corresponding to ENr113 genotypes results is given in S2 [href=this file] – S4 [href=#pone.0167994.s006] Figs and in the Supplemental Data ( S4 Table [href=#pone.0167994.s008] ) as well.\n",
      "Data corresponding to S2 [href=this file] – S4 [href=#pone.0167994.s006] Figs.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#pone.0167994.s005'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for pone.0167994.s005: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 3 <a> elements with href='#pone.0167994.s006'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 284) for pone.0167994.s006: An example output corresponding to ENr113 genotypes results is given in S2 [href=#pone.0167994.s004] – S4 [href=this file] Figs and in the Supplemental Data ( S4 Table [href=#pone.0167994.s008] ) as well.\n",
      "Data corresponding to S2 [href=#pone.0167994.s004] – S4 [href=this file] Figs.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#pone.0167994.s007'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 178) for pone.0167994.s007: The association of these three SNPs and the phenotype are significant ( S3 Table [href=this file] ), which suggests that the metaSNP method correctly tagged the designated SNPs.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#pone.0167994.s008'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 205) for pone.0167994.s008: An example output corresponding to ENr113 genotypes results is given in S2 [href=#pone.0167994.s004] – S4 [href=#pone.0167994.s006] Figs and in the Supplemental Data ( S4 Table [href=this file] ) as well.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 2 <a> elements with href='#pone.0167994.s009'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 49) for pone.0167994.s009: Details are given in S5 Table [href=this file] .\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#pone.0167994.s010'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 92) for pone.0167994.s010: The details of the mutual information estimation is given in the S1 File [href=this file] .\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 47729\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 158274\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 48451 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-fdbe21a436ab859eff5f87ca543c433426f99503bf392b9a0394503e333faa59\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 48451 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\n",
      "  \"datasets\": [\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    }\n",
      "  ]\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 21 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 677 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 605 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: []\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 0 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 0\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 19, with cols: Index(['link', 'title', 'file_info', 'description', 'source_section',\n",
      "       'section_class', 'download_link', 'file_extension', 'a_attr_href',\n",
      "       'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 640 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - Parsed data extraction completed. Elements collected: 11\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 481 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 6th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC9925742)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC9925742\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC9925742&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 406 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC9925742.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC9925742 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 430 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC9925742, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 443 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 264 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 454 - INFO - Found 6 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b69fee00>, <Element supplementary-material at 0x1b69fdd80>, <Element supplementary-material at 0x1b69fea80>, <Element supplementary-material at 0x1b69fde40>, <Element supplementary-material at 0x1b69fe300>, <Element supplementary-material at 0x1b69fcd80>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 462 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b69fee00>, <Element supplementary-material at 0x1b69fdd80>, <Element supplementary-material at 0x1b69fea80>, <Element supplementary-material at 0x1b69fde40>, <Element supplementary-material at 0x1b69fe300>, <Element supplementary-material at 0x1b69fcd80>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 548 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM1: After clustering (Supplementary Fig. 1a–e [this file] ), a total of 37,943 cells were annotated into 23 clusters using a predefined set of marker genes (Supplementary Fig. 1c [this file] ). \n",
      "To reveal heterogeneity within oligodendrocytes, we grouped the aforementioned oligodendrocytes into six classes described in a published database , as follows (Fig. 1a [rid=Fig1] and Supplementary Fig. 2a [this file] –d and 3a [this file] ): (1) Differentiation-committed oligodendrocyte precursors (COPs) and newly formed oligodendrocytes (NFOL) that express Sox6, Pdgfra, and Cspg4. (2) Myelin-forming oligodendrocytes (MFOL) that express Ctps, Ablims, and Thbs3. (3) Mature oligodendrocytes 1, 2 (MOL 1, 2) that express Il33, Ptgds, and Opalin. (4) Mature oligodendrocytes 3, 4 (MOL 3, 4) that express Klk6, S100b, Anxa5, Cd59a and Mgst3. Moreover, we discovered an additional oligodendrocyte cluster, which gradually increased in the 3-, 6-month-old App KI mice (Fig. 1b–d [rid=Fig1] and Supplementary Fig. 2e [this file] ). This cluster was present in the UMAP plot even with down-sampled data (400, 1,200, and 2,400 cells) (Supplementary Fig. 2f [this file] ). Remarkably, we found that this cluster shared some signatures with MOL2, while Alzheimer’s disease risk factors such as C4b, Apoe and Cd74 are strongly expressed (Supplementary Fig. 2c [this file] , 3a [this file] , 3b [this file] , and Source data file). \n",
      "We further confirmed the lineage transitions of DAOs and homeostatic MOLs from COP and NFOL in control and AD brains (Fig. 1e–g [rid=Fig1] and Supplementary Fig. 4 [this file] ), but density from the DAO cluster exclusively appeared in 3 and 6-month-old AD brains (Fig. 1 [rid=Fig1] g). \n",
      "We reanalyzed previously published single-cell transcriptome datasets from postmortem AD brains (Supplementary Fig. 5a–c [this file] ) . \n",
      "Notably, consistent with scRNA-seq results, we observed a significant increase in Mbp + , Cd74+ oligodendrocytes in 3- and 6-month-old App KI and 6-month-old 5xFAD mice, whereas the number of NeuroD1+ post-mitotic neurons significantly decreased (Fig. 2 [rid=Fig2] a, b [rid=Fig2] , Supplementary Fig. 6a–c [this file] , and Supplementary Fig. 7a [this file] , b [this file] ). We further confirmed an increase in Cd74+ and Oligo2+ AD-associated oligodendrocytes in App KI mice using flow cytometry (Fig. 2 [rid=Fig2] c, d [rid=Fig2] , and Supplementary Fig. 6d [this file] ). \n",
      "Transmission electron microscopy (TEM) analysis revealed that 6-month-old App KI mice exhibited a thinner myelin layer relative to the axon diameter (Fig. 2e [rid=Fig2] and supplementary fig. 8 [this file] ). \n",
      "Consistently, we found a significant increase in Mbp+ and Cd74+ DAOs in the hippocampus of premature progeria (6-month-old) and 13-month-old mice (Supplementary Fig. 9a–d [this file] ), whereas the number of NeuroD1-positive post-mitotic neurons decreased in the hippocampus of 13-month-old mice (Supplementary Fig. 9e and f [this file] ). We further confirmed that the genes enriched in DAOs were highly correlated with gene signatures shown in the aged human brain (Supplementary Fig. 10a, b [this file] ), indicating that Cd74-positive DAOs increased in the aged brains and had transcriptional profiles more similar to the aged human brain. \n",
      "In particular, we found significant alternations in the transcriptional profiles between DAOs and MOL 3/4 clusters and GO-term of up-regulated genes in DAO versus MOL3/4 showed the interleukin & chemokine production and ERK signaling (Fig. 3 [rid=Fig3] a and c [rid=Fig3] ), which was also confirmed during the disease progression in AD mice brain (Supplementary Fig. 11a, b [this file] ). \n",
      "We first identified microglia with a high DAM module score based on previously published DAM signatures (Supplementary Fig. 1e [this file] , 12a, b [this file] ), that gradually increased during AD progression (Fig. 3 [rid=Fig3] e, f [rid=Fig3] , and Supplementary Fig. 12c [this file] ). \n",
      "Supplementary Fig. 13a–c [this file] shows efficient inhibition of phosphor 44/42 MAPK signaling in primary oligodendrocytes upon treatment with 200 ng/ml SCH772984, a specific Erk1 and Erk2 inhibitor. Remarkably, oligodendrocytes treated with SCH772984 exhibited a significant increase in Mbp+ branching compared with the controls, whereas the number of Cd74+ cells significantly decreased (Supplementary Fig. 13d [this file] , 13e [this file] , and Fig. 4a–c [rid=Fig4] ). \n",
      "First, in order to examine the potential interactions between DAOs and other neuronal cell types, AD DAOs were co-cultured with AD neurons, astrocytes, and microglia treated with SCH772984 (Supplementary Fig. 14a [this file] ). We found that AD DAOs with other neuronal cell types did not increase the number of Mbp+ branches in the presence and absence of SCH772984 treatment (Supplementary Fig. 14b–f [this file] ), suggesting that inhibition of ERK/MAPK signaling in other neuronal cell types did not contribute to the phenotypic changes in DAOs (Fig. 4d–f [rid=Fig4] ). Moreover, to investigate whether other neural cell types can contribute to the AD-associated phenotypes via ERK signaling, each of the neurons, microglia, astrocytes and oligodendrocytes isolated from AD mice were subjected to SCH772984 treatment, and AD pathology was determined after co-culture with the remaining untreated AD neuronal cell types (Supplementary Fig. 15a [this file] ). Remarkably, we found that ERK1/2-inhibited oligodendrocytes significantly diminished the Aβ42/40 ratio in the AD primary neuronal cultures (Supplementary Fig. 15b [this file] ). Also, we observed a decrease in the amyloidogenic processing marker C99 in the AD neuronal cultures with SCH772984 treatment (Supplementary Fig. 15c–e [this file] ), indicating that AD oligodendrocytes predominantly contribute to the pathophysiology of AD via ERK signaling. \n",
      "Consistent with previous results, systemic administration of SCH772984 significantly reduced the occurrence of DAOs in the hippocampus of App KI mice (Fig. 5b, c [rid=Fig5] , and Supplementary Fig. 16a–d [this file] ) but not Cd74 + Iba1+ microglia (Supplementary Fig. 16c, d [this file] ). \n",
      "Moreover, we confirmed a decrease in the number of Aβ plaques (Fig. 5m, n [rid=Fig5] ), coupled with an increase in the number of NeuroD1+ cells in the SCH772984-treated AD mouse hippocampus (Supplementary Fig. 16e, f [this file] ). \n",
      "Figure 5a, b, c [this file] obtained from the AD Knowledge Portal ( https://adknowledgeportal.org (https://adknowledgeportal.org) ) under study 557 snRNAseqAD_TREM2(syn21126450). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM2: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM3: To identify transcriptional patterns of this cluster, we conducted differential expression gene analysis by using the FindAllMarkers function in the Seurat package (Supplementary Data 1 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM4: We next explored the unique molecular signatures of DAOs during AD development (Supplementary Data 2 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM5: Hippocampus tissues from 1-, 3-, and 6-month-old wild-type and App-KI mice were then dissected (Supplementary Data 3 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM6: Further information on research design is available in the Nature Portfolio Reporting Summary [this file] linked to this article. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 243 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 36434\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 140680\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 37156 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-62dbdd60735f55b700de9f129827a2be3f1852af67af3f97c6c0ca345e444364\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 37156 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\n",
      "  \"datasets\": [\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE224398\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE224398\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"syn21126450\",\n",
      "      \"repository_reference\": \"https://adknowledgeportal.org\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    }\n",
      "  ]\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'GSE224398', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE224398'}, {'dataset_identifier': 'syn21126450', 'repository_reference': 'https://adknowledgeportal.org'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 5 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE224398\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn21126450\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'GSE224398', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE224398', 'dataset_id': 'GSE224398'}, {'dataset_identifier': 'syn21126450', 'repository_reference': 'https://adknowledgeportal.org', 'dataset_id': 'syn21126450'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE224398', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE224398', 'dataset_id': 'GSE224398'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE224398\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE224398, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE224398\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE224398\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE224398', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn21126450', 'repository_reference': 'https://adknowledgeportal.org', 'dataset_id': 'syn21126450'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn21126450, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn21126450\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn21126450', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 677 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 605 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 256 - INFO - Augmented dataset links: [{'dataset_identifier': 'GSE224398', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn21126450', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 2 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE224398\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE224398\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE224398 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE224398 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE224398 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE224398\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE224398\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn21126450\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn21126450\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn21126450 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn21126450 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn21126450 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn21126450 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn21126450 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn21126450\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn21126450\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 2\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 265 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 8, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 477 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 7th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC9235070)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC9235070\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC9235070&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 394 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC9235070.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b62b4650>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC9235070 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC9235070, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 445 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC9235070)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 547 - INFO - Extracted 13 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 557 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#s2a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for s2a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#s2b'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for s2b: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#s2b'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for s2b: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#s2c'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for s2c: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#s2d'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for s2d: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#s2d'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for s2d: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#s2d'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for s2d: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#s4a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for s4a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#s4a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for s4a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#s4a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for s4a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#SD1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for SD1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 64325\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 215213\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 65047 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-b7fbf8e0c9f2230da7902779a8d10bf11e1eebc0835a4541c0db672f6837eae8\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 65047 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\n",
      "  \"datasets\": [\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE81861\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE81861\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE75688\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE75688\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"E-MTAB-6149\",\n",
      "      \"repository_reference\": \"https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-6149/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"syn30243143\",\n",
      "      \"repository_reference\": \"https://www.synapse.org/#!Synapse:syn30243143\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    }\n",
      "  ]\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'GSE81861', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE81861'}, {'dataset_identifier': 'GSE75688', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE75688'}, {'dataset_identifier': 'E-MTAB-6149', 'repository_reference': 'https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-6149/'}, {'dataset_identifier': 'syn30243143', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn30243143'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 5 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE81861\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE75688\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: E-MTAB-6149\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn30243143\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'GSE81861', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE81861', 'dataset_id': 'GSE81861'}, {'dataset_identifier': 'GSE75688', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE75688', 'dataset_id': 'GSE75688'}, {'dataset_identifier': 'E-MTAB-6149', 'repository_reference': 'https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-6149/', 'dataset_id': 'E-MTAB-6149'}, {'dataset_identifier': 'syn30243143', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn30243143', 'dataset_id': 'syn30243143'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE81861', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE81861', 'dataset_id': 'GSE81861'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE81861\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE81861, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE81861\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE81861\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE81861', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE75688', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE75688', 'dataset_id': 'GSE75688'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE75688\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE75688, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE75688\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE75688\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE75688', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'E-MTAB-6149', 'repository_reference': 'https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-6149/', 'dataset_id': 'E-MTAB-6149'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: E-MTAB-6149\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID E-MTAB-6149 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-6149/, identifier: E-MTAB-6149\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-6149/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-MTAB-6149 for https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-6149/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-MTAB-6149 for https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-6149/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-MTAB-6149 for https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-6149/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-MTAB-6149 for https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-6149/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Found partial match for https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-6149/ in ArrayExpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'E-MTAB-6149', 'data_repository': 'arrayexpress', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn30243143', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn30243143', 'dataset_id': 'syn30243143'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn30243143\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn30243143, synapse.org, https://www.synapse.org/#!Synapse:syn30243143\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn30243143\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn30243143', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 677 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 605 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'GSE81861', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE75688', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'E-MTAB-6149', 'data_repository': 'arrayexpress', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn30243143', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 4 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE81861\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE81861\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE81861 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE81861 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE81861 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE81861\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE81861\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE75688\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE75688\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE75688 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE75688 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE75688 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 2 with repo: geo and accession_id: GSE75688\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE75688\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 2: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: E-MTAB-6149\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: arrayexpress, identifier: E-MTAB-6149\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-MTAB-6149 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-MTAB-6149 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-MTAB-6149 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-MTAB-6149 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for E-MTAB-6149 in arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 3 with repo: arrayexpress and accession_id: E-MTAB-6149\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ebi.ac.uk/biostudies/ArrayExpress/studies/E-MTAB-6149\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 3: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn30243143\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn30243143\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn30243143 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn30243143 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn30243143 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn30243143 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn30243143 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 4 with repo: synapse.org and accession_id: syn30243143\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn30243143\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 4: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 4\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 17, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 640 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - Parsed data extraction completed. Elements collected: 6\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 481 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 8th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11661334)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11661334\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC11661334&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 406 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC11661334.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11661334 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 430 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11661334, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 443 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 264 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 454 - INFO - Found 2 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b006a5c0>, <Element supplementary-material at 0x1b67ef480>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 462 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b006a5c0>, <Element supplementary-material at 0x1b67ef480>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 548 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref SD1: We also observed a clear linear trend between the measured correlation of protein expression within well-imputed pairs and their imputed co-expression, further supporting the robustness of COWAS imputation models ( Supplementary Figure 1 [this file] ). \n",
      "For LDL cholesterol, which had the best-powered GWAS of the three traits we considered, we identified 38 protein pairs with a significant co-expression effect ( Supplementary Figure 2 [this file] ). \n",
      "By visualizing significant COWAS pairs as a network, we identified complexes of mutually interacting proteins that modulate LDL cholesterol levels ( Supplementary Figure 3 [this file] ). For example, we found that APOE and PLTP both decrease LDL cholesterol levels, while their co-expression is associated with increased LDL cholesterol levels ( Supplementary Figure 4 [this file] ). \n",
      "Effect sizes, standard errors, and values for the pairs shown in Figure 3 [rid=F3] are also summarized in Supplementary Table 1 [this file] . \n",
      "Results for protein pairs identified as significant using at least one of these three tests with IGAP GWAS data are shown in Supplementary Figure 5 [this file] and Supplementary Table 2 [this file] . \n",
      "Finally, we considered an alternative, product-based formulation of COWAS (see Supplementary Note 1 [this file] for details). The mean out-of-sample for product-based co-expression imputation models was slightly higher than for any of the residual-based co-expression imputation models, and over 99% of well-imputed pairs again had a nominally significant correlation between imputed and observed expression ( Supplementary Figure 6 [this file] and Supplementary Data 1 [rid=SD2] ). \n",
      "We also discovered 22 pairs with an overall effect on AD using the product-based COWAS global test and EADB summary statistics ( Supplementary Figure 7 [this file] and Supplementary Table 3 [this file] ). Full results for AD using the product-based version of COWAS are provided in Supplementary Figure 7 [this file] , Supplementary Table 3 [this file] , and Supplementary Data 18 [rid=SD2] – 19 [rid=SD2] . \n",
      "Effect sizes, standard errors, and values for the pairs shown in Figure 4 [rid=F4] are also summarized in Supplementary Table 4 [this file] . \n",
      "In Supplementary Note 1 [this file] we define an alternative, product-based version of COWAS in which is replaced with . \n",
      "In Supplementary Note 2 [this file] we derive a summary-level version of the second stage following the approach introduced in the multivariate imaging-wide association study (MV-IWAS) method [ 14 [rid=R14] , 16 [rid=R16] ], which is related to other methods in the field of Mendelian randomization [ 87 [rid=R87] , 88 [rid=R88] ]. We also provide full derivations for both individual-level and summary-level versions of standard TWAS/PWAS in Supplementary Note 3 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref SD2: Across all of the model types we considered, the median out-of-sample correlation between estimated and imputed co-expression was always lower than the median out-of-sample correlation between measured and imputed single-protein expression ( Figure 2a [rid=F2] , Figure 2c [rid=F2] , and Supplementary Data 1 [this file] ). \n",
      "Among these well-imputed pairs, lasso regression with -pQTLs pre-screened by their effect sizes achieved the highest mean out-of-sample for predicting co-expression (mean , Supplementary Data 1 [this file] ). On the other hand, ridge regression with both -pQTLs and -pQTLs pre-screened by their values yielded the greatest number of well-imputed protein pairs ( Figure 2d [rid=F2] and Supplementary Data 1 [this file] ). \n",
      "Model performance metrics for every combination of protein pair and model type are provided in Supplementary Data 2 [this file] – 13 [this file] . \n",
      "Full imputation performance metrics for all analyzed proteins and outcome traits are provided in Supplementary Data 14 [this file] – 20 [this file] . \n",
      "Full results for LDL cholesterol, including the estimated effect sizes and standard errors for each pair, are provided in Supplementary Data 14 [this file] . \n",
      "Full results from our validation study for LDL cholesterol are provided in Supplementary Data 15 [this file] . \n",
      "Full results for AD, including the estimated effect sizes and standard errors for each pair, are provided in Supplementary Data 16 [this file] . \n",
      "Full results for all protein pairs passing quality control are provided in Supplementary Data 17 [this file] . \n",
      "The mean out-of-sample for product-based co-expression imputation models was slightly higher than for any of the residual-based co-expression imputation models, and over 99% of well-imputed pairs again had a nominally significant correlation between imputed and observed expression ( Supplementary Figure 6 [rid=SD1] and Supplementary Data 1 [this file] ). \n",
      "Full results for AD using the product-based version of COWAS are provided in Supplementary Figure 7 [rid=SD1] , Supplementary Table 3 [rid=SD1] , and Supplementary Data 18 [this file] – 19 [this file] . \n",
      "Full results for PD, including the estimated effect sizes and standard errors for each pair, are provided in Supplementary Data 20 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 243 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 77300\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 248046\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 78022 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-1e4f17e4150b8e6e9e1286e996eb3949f4b541032670a1bdb71120eaddda0bd2\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 78022 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\"datasets\":[{\"dataset_identifier\":\"PMC11661334\",\"repository_reference\":\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11661334/\"},{\"dataset_identifier\":\"n/a\",\"repository_reference\":\"n/a\"},{\"dataset_identifier\":\"n/a\",\"repository_reference\":\"n/a\"},{\"dataset_identifier\":\"n/a\",\"repository_reference\":\"n/a\"},{\"dataset_identifier\":\"n/a\",\"repository_reference\":\"n/a\"},{\"dataset_identifier\":\"n/a\",\"repository_reference\":\"n/a\"},{\"dataset_identifier\":\"n/a\",\"repository_reference\":\"n/a\"},{\"dataset_identifier\":\"n/a\",\"repository_reference\":\"n/a\"},{\"dataset_identifier\":\"n/a\",\"repository_reference\":\"n/a\"},{\"dataset_identifier\":\"n/a\",\"repository_reference\":\"n/a\"},{\"dataset_identifier\":\"n/a\",\"repository_reference\":\"n/a\"},{\"dataset_identifier\":\"n/a\",\"repository_reference\":\"n/a\"},{\"dataset_identifier\":\"n/a\",\"repository_reference\":\"n/a\"},{\"dataset_identifier\":\"n/a\",\"repository_reference\":\"n/a\"},{\"dataset_identifier\":\"n/a\",\"repository_reference\":\"n/a\"},{\"dataset_identifier\":\"n/a\",\"repository_reference\":\"n/a\"},{\"dataset_identifier\":\"n/a\",\"repository_reference\":\"n/a\"},{\"dataset_identifier\":\"n/a\",\"repository_reference\":\"n/a\"},{\"dataset_identifier\":\"n/a\",\"repository_reference\":\"n/a\"},{\"dataset_identifier\":\"n/a\",\"repository_reference\":\"n/a\"},{\"dataset_identifier\":\"n/a\",\"repository_reference\":\"n/a\"}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'PMC11661334', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11661334/'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 21 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: PMC11661334\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'PMC11661334', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11661334/', 'dataset_id': 'PMC11661334'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'PMC11661334', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11661334/', 'dataset_id': 'PMC11661334'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: PMC11661334\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID PMC11661334 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11661334/, identifier: PMC11661334\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11661334/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier PMC11661334 for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11661334/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier PMC11661334 for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11661334/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier PMC11661334 for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11661334/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier PMC11661334 for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11661334/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier PMC11661334 for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11661334/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11661334/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'PMC11661334', 'data_repository': 'www.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 677 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 605 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 256 - INFO - Augmented dataset links: [{'dataset_identifier': 'PMC11661334', 'data_repository': 'www.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 1 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: PMC11661334\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: www.ncbi.nlm.nih.gov, identifier: PMC11661334\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier PMC11661334 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier PMC11661334 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier PMC11661334 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier PMC11661334 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier PMC11661334 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 976 - WARNING - URL www.ncbi.nlm.nih.gov may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for PMC11661334 in www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 1 with repo: www.ncbi.nlm.nih.gov and accession_id: PMC11661334\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1254 - WARNING - Repository www.ncbi.nlm.nih.gov unknown in Ontology. Skipping dataset page 1.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 1\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 265 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 3, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage', 'link',\n",
      "       'source_url', 'download_link', 'title', 'content_type', 'id', 'caption',\n",
      "       'description', 'source_section', 'retrieval_pattern',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 477 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 9th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11929800)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11929800\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC11929800&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 406 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC11929800.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11929800 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 430 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11929800, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 443 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 264 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 454 - INFO - Found 40 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b04d6100>, <Element supplementary-material at 0x1b60be400>, <Element supplementary-material at 0x1b60bf000>, <Element supplementary-material at 0x1b60bf0c0>, <Element supplementary-material at 0x1b60bf180>, <Element supplementary-material at 0x1b60bf240>, <Element supplementary-material at 0x1b60bfbc0>, <Element supplementary-material at 0x1b60bf7c0>, <Element supplementary-material at 0x1b60bfb00>, <Element supplementary-material at 0x1b60bfb40>, <Element supplementary-material at 0x1b06640c0>, <Element supplementary-material at 0x1b60bfa00>, <Element supplementary-material at 0x1b60bfc40>, <Element supplementary-material at 0x1b60bf6c0>, <Element supplementary-material at 0x1b60bf980>, <Element supplementary-material at 0x1b60bfe40>, <Element supplementary-material at 0x1b60bfc80>, <Element supplementary-material at 0x1b0544780>, <Element supplementary-material at 0x1b60ccf80>, <Element supplementary-material at 0x1b60cfdc0>, <Element supplementary-material at 0x1b60cff40>, <Element supplementary-material at 0x1b0068900>, <Element supplementary-material at 0x1b00688c0>, <Element supplementary-material at 0x1b0069600>, <Element supplementary-material at 0x1b006b380>, <Element supplementary-material at 0x1b006a640>, <Element supplementary-material at 0x1b006ad00>, <Element supplementary-material at 0x1b006a200>, <Element supplementary-material at 0x1b00694c0>, <Element supplementary-material at 0x1b0069240>, <Element supplementary-material at 0x1b006a0c0>, <Element supplementary-material at 0x1b006a580>, <Element supplementary-material at 0x1b00698c0>, <Element supplementary-material at 0x1b5f714c0>, <Element supplementary-material at 0x1b0175a00>, <Element supplementary-material at 0x1b664a7c0>, <Element supplementary-material at 0x1b664ae40>, <Element supplementary-material at 0x1b05c1100>, <Element supplementary-material at 0x1b0682240>, <Element supplementary-material at 0x1affd1e80>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 462 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b04d6100>, <Element supplementary-material at 0x1b60be400>, <Element supplementary-material at 0x1b60bf000>, <Element supplementary-material at 0x1b60bf0c0>, <Element supplementary-material at 0x1b60bf180>, <Element supplementary-material at 0x1b60bf240>, <Element supplementary-material at 0x1b60bfbc0>, <Element supplementary-material at 0x1b60bf7c0>, <Element supplementary-material at 0x1b60bfb00>, <Element supplementary-material at 0x1b60bfb40>, <Element supplementary-material at 0x1b06640c0>, <Element supplementary-material at 0x1b60bfa00>, <Element supplementary-material at 0x1b60bfc40>, <Element supplementary-material at 0x1b60bf6c0>, <Element supplementary-material at 0x1b60bf980>, <Element supplementary-material at 0x1b60bfe40>, <Element supplementary-material at 0x1b60bfc80>, <Element supplementary-material at 0x1b0544780>, <Element supplementary-material at 0x1b60ccf80>, <Element supplementary-material at 0x1b60cfdc0>, <Element supplementary-material at 0x1b60cff40>, <Element supplementary-material at 0x1b0068900>, <Element supplementary-material at 0x1b00688c0>, <Element supplementary-material at 0x1b0069600>, <Element supplementary-material at 0x1b006b380>, <Element supplementary-material at 0x1b006a640>, <Element supplementary-material at 0x1b006ad00>, <Element supplementary-material at 0x1b006a200>, <Element supplementary-material at 0x1b00694c0>, <Element supplementary-material at 0x1b0069240>, <Element supplementary-material at 0x1b006a0c0>, <Element supplementary-material at 0x1b006a580>, <Element supplementary-material at 0x1b00698c0>, <Element supplementary-material at 0x1b5f714c0>, <Element supplementary-material at 0x1b0175a00>, <Element supplementary-material at 0x1b664a7c0>, <Element supplementary-material at 0x1b664ae40>, <Element supplementary-material at 0x1b05c1100>, <Element supplementary-material at 0x1b0682240>, <Element supplementary-material at 0x1affd1e80>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 548 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM1: The procedures for sample selection are shown in Supplementary Fig. 1 [this file] . More details for the distribution of brain 5hmC data can be found in Supplementary Fig. 2 [this file] – 3 [this file] . \n",
      "Of the 2821 DhMRs (606 associated with AD diagnosis, 1542 associated with Aβ load, 1399 associated with PHFtau tangles, annotated to 2642 distinct genes) identified in stage II (replication), about 69% reside in exons, ~15% in intergenic regions, and ~17% in introns (Supplementary Fig. 4a [this file] ). Compared to non-AD associated 5hmC regions (total 194,944), AD-associated regions (total 2821) were significantly more likely to coincide with regions related to transcription start site (TSS), transcription states, and enhancer states, but less likely to co-position with regions corresponding to heterochromatin and quiescent states (Supplementary Fig. 4b [this file] , Supplementary Data 20 [rid=MOESM22] ). \n",
      "Of these, 108 AD-associated, 581 Aβ-associated and 135 PHFtau-associated 5hmC regions were positively correlated with gene expression, whereas 65 AD-associated, 240 Aβ-associated and 53 PHFtau-associated 5hmC regions were negatively correlated with gene expression (Supplementary Fig. 5 [this file] , Supplementary Data 22 [rid=MOESM24] ). These genes were clustered into multiple protein-protein interaction (PPI) networks (Supplementary Fig. 6 [this file] ). The most significant biological processes and pathways of these functional clusters were associated with mTOR signaling, mediator complex and transcriptional regulation of RNA polymerase transcripts (II and III), SWI/SNF and npBAF complex, γ-secretase complex, DNA replication and repair, etc. (Supplementary Fig. 6 [this file] , Supplementary Data 24 [rid=MOESM26] ). \n",
      "At GWAS threshold of 1 × 10 , genetic variants residing in regions harboring AD-associated 5hmC alterations were significantly enriched in DNase I hypersensitive sites (DHSs), transcribed histone modifications, TSS, and transcribed chromatin states, but depleted in repressed chromatin states and H3K27me3 in various tissues such as blood, blood vessel, embryonic stem cells (ESCs), colon, liver, and fetal intestine(Supplementary Figs. 7 [this file] – 9 [this file] ; Supplementary Data 30 [rid=MOESM32] – 32 [rid=MOESM34] ). \n",
      "We also performed colocalization analysis using summary statistics from a cell-type specific -eQTL analysis of eight human brain cell types and found that one of the colocalized genes, , is expressed in both astrocytes (Supplementary Fig. 10 [this file] ) and microglia (Supplementary Fig. 11 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM2: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM3: After data pre-processing and quality control, our dataset included 197,765 5hmC regions (Supplementary Data 1 [this file] ) in 1050 older individuals (mean age: 89.5, age range: 65.9–108.3, Table 1 [rid=Tab1] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM4: This analysis resulted in 2939 DhMRs associated with AD diagnosis (Supplementary Data 2 [this file] ), 1,770 DhMRs associated with Aβ load (Supplementary Data 3 [rid=MOESM5] ), and 3110 DhMRs associated with PHFtau tangles (Supplementary Data 4 [rid=MOESM6] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM5: This analysis resulted in 2939 DhMRs associated with AD diagnosis (Supplementary Data 2 [rid=MOESM4] ), 1,770 DhMRs associated with Aβ load (Supplementary Data 3 [this file] ), and 3110 DhMRs associated with PHFtau tangles (Supplementary Data 4 [rid=MOESM6] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM6: This analysis resulted in 2939 DhMRs associated with AD diagnosis (Supplementary Data 2 [rid=MOESM4] ), 1,770 DhMRs associated with Aβ load (Supplementary Data 3 [rid=MOESM5] ), and 3110 DhMRs associated with PHFtau tangles (Supplementary Data 4 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM7: At the Benjamin-Hochberg (BH) corrected false discovery rate (FDR) < 0.05, 2705 regions (92% of EN-selected) were associated with AD diagnosis (Supplementary Data 5 [this file] ), 1750 regions (98.9% of EN-selected) were associated with Aβ load (Supplementary Data 6 [rid=MOESM8] ) and 2626 regions (84.4% of EN-selected) were associated with PHFtau tangles (Supplementary Data 7 [rid=MOESM9] ) in the regression analysis. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM8: At the Benjamin-Hochberg (BH) corrected false discovery rate (FDR) < 0.05, 2705 regions (92% of EN-selected) were associated with AD diagnosis (Supplementary Data 5 [rid=MOESM7] ), 1750 regions (98.9% of EN-selected) were associated with Aβ load (Supplementary Data 6 [this file] ) and 2626 regions (84.4% of EN-selected) were associated with PHFtau tangles (Supplementary Data 7 [rid=MOESM9] ) in the regression analysis. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM9: At the Benjamin-Hochberg (BH) corrected false discovery rate (FDR) < 0.05, 2705 regions (92% of EN-selected) were associated with AD diagnosis (Supplementary Data 5 [rid=MOESM7] ), 1750 regions (98.9% of EN-selected) were associated with Aβ load (Supplementary Data 6 [rid=MOESM8] ) and 2626 regions (84.4% of EN-selected) were associated with PHFtau tangles (Supplementary Data 7 [this file] ) in the regression analysis. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM10: At FDR < 0.05, a total of 2821 DhMRs, including 606 (22.4% of those identified in the discovery sample) associated with AD diagnosis, 1542 (88.1% of those identified in the discovery sample) associated with Aβ load, and 1399 (53.3% of those identified in the discovery sample) associated with PHFtau tangles, remained statistically significant with the same direction of association (Supplementary Data 8 [this file] – 10 [rid=MOESM12] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM11: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM12: At FDR < 0.05, a total of 2821 DhMRs, including 606 (22.4% of those identified in the discovery sample) associated with AD diagnosis, 1542 (88.1% of those identified in the discovery sample) associated with Aβ load, and 1399 (53.3% of those identified in the discovery sample) associated with PHFtau tangles, remained statistically significant with the same direction of association (Supplementary Data 8 [rid=MOESM10] – 10 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM13: Of the 2821 DhMRs replicated in ROS, 93 regions were significantly associated with all three neuropathological phenotypes: AD diagnosis, Aβ,and PHFtau (Supplementary Data 11 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM14: Notably, 43 DhMRs (out of 2821) also harbor known AD variants (Supplementary Data 12 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM15: However, the association between 5hmC and AD pathology appeared not to be mediated by genetic variants because additionally adjusting for the genotype of lead SNPs in the candidate regions did not substantially attenuate the observed associations (Supplementary Data 13 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM16: It showed that the observed associations on chromosome 19 remained significant after further controlling for genotype (Supplementary Data 14 [this file] ). \n",
      "The observed associations of 5hmC regions on chromosome 19 with AD pathology are unlikely to be driven by because additional adjustment for genotype did not change the results (Supplementary Data 14 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM17: The effect of an individual 5hmC region on AD pathology was in general small to moderate (Supplementary Data 15 [this file] ). \n",
      "Results for the variance explained in Aβ load and PHFtau tangles are shown in Supplementary Data 15 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM18: At FDR < 0.05, 5hmC alterations in 2065, 1770, and 2972 regions were significantly associated with AD diagnosis (Supplementary Data 16 [this file] ), Aβ (Supplementary Data 17 [rid=MOESM19] ), and PHFtau (Supplementary Data 18 [rid=MOESM20] ), respectively. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM19: At FDR < 0.05, 5hmC alterations in 2065, 1770, and 2972 regions were significantly associated with AD diagnosis (Supplementary Data 16 [rid=MOESM18] ), Aβ (Supplementary Data 17 [this file] ), and PHFtau (Supplementary Data 18 [rid=MOESM20] ), respectively. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM20: At FDR < 0.05, 5hmC alterations in 2065, 1770, and 2972 regions were significantly associated with AD diagnosis (Supplementary Data 16 [rid=MOESM18] ), Aβ (Supplementary Data 17 [rid=MOESM19] ), and PHFtau (Supplementary Data 18 [this file] ), respectively. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM21: Of these, 118 regions were associated with all three phenotypes (Supplementary Data 19 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM22: Compared to non-AD associated 5hmC regions (total 194,944), AD-associated regions (total 2821) were significantly more likely to coincide with regions related to transcription start site (TSS), transcription states, and enhancer states, but less likely to co-position with regions corresponding to heterochromatin and quiescent states (Supplementary Fig. 4b [rid=MOESM1] , Supplementary Data 20 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM23: Focusing on the 2821 DhMRs associated with AD pathologies (606 regions associated with AD diagnosis, 1542 regions associated with Aβ, 1399 regions associated with PHFtau, annotated to 2642 distinct genes), we evaluated whether altered DNA hydroxymethylation affects the expression of genes located in the vicinity ( ± 50 kb) of each DhMR using brain RNA-seq data (DLPFC) from the same participants (Supplementary Data 21 [this file] ). \n",
      "To further functionally annotate the AD-associated 5hmC regions, we calculated the correlation between the level of 5hmC and the level of H3K9ac generated in the dorsolateral prefrontal cortex from the same individuals (Supplementary Data 21 [this file] ) in ROSMAP . \n",
      "In a subset of individuals (Supplementary Data 21 [this file] ) with both 5hmC and brain proteomics data (DLPFC) , we calculated the correlation of AD-associated 5hmC with protein abundances (Methods). \n",
      "A total of 731 unique individuals with both 5hmC and proteomic data were included in this analysis (Supplementary Data 21 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM24: At FDR < 0.05, the levels of 886 DhMRs (annotated to 818 distinct genes) were correlated with gene expression (Supplementary Data 22 [this file] ). Of these, 108 AD-associated, 581 Aβ-associated and 135 PHFtau-associated 5hmC regions were positively correlated with gene expression, whereas 65 AD-associated, 240 Aβ-associated and 53 PHFtau-associated 5hmC regions were negatively correlated with gene expression (Supplementary Fig. 5 [rid=MOESM1] , Supplementary Data 22 [this file] ). \n",
      "A total of 819 samples have complete data for both 5hmC and RNA-seq data (Supplementary Data 22 [this file] ). \n",
      "A total of 513 ROSMAP participants with both 5hmC and H3K9ac data were included in this analysis (Supplementary Data 22 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM25: Moreover, the expression level for most of these genes (83%) was also significantly associated with AD diagnosis, Aβ or PHFtau (Supplementary Data 23 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM26: The most significant biological processes and pathways of these functional clusters were associated with mTOR signaling, mediator complex and transcriptional regulation of RNA polymerase transcripts (II and III), SWI/SNF and npBAF complex, γ-secretase complex, DNA replication and repair, etc. (Supplementary Fig. 6 [rid=MOESM1] , Supplementary Data 24 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM27: At FDR < 0.05, none of the 5hmC regions associated with AD diagnosis (out of 606) was correlated with H3K9ac, despite that eight regions did show nominal correlation with H3K9ac at FDR < 0.1 (Supplementary Data 25 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM28: In comparison, the levels of 90 Aβ-associated (out of 1542) and 180 PHFtau-associated (out of 1399) 5hmC regions were correlated with the level of H3K9ac at FDR < 0.05 (Supplementary Data 26 [this file] – 27 [rid=MOESM29] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM29: In comparison, the levels of 90 Aβ-associated (out of 1542) and 180 PHFtau-associated (out of 1399) 5hmC regions were correlated with the level of H3K9ac at FDR < 0.05 (Supplementary Data 26 [rid=MOESM28] – 27 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM30: At FDR < 0.05, the expression levels of 91 proteins were significantly correlated with the abundance of 5hmC regions that were significantly associated with AD pathologies (Supplementary Data 28 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM31: These proteins are significantly enriched in biological processes related to amyloid precursor protein catabolic process, regulation of Aβ formation, regulation of signal transduction, neurogenesis, phosphorylation, synapse, protein binding, proteolysis, neuronal ion channel clustering, regulation of protein modification process, etc. (Supplementary Data 29 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM32: At GWAS threshold of 1 × 10 , genetic variants residing in regions harboring AD-associated 5hmC alterations were significantly enriched in DNase I hypersensitive sites (DHSs), transcribed histone modifications, TSS, and transcribed chromatin states, but depleted in repressed chromatin states and H3K27me3 in various tissues such as blood, blood vessel, embryonic stem cells (ESCs), colon, liver, and fetal intestine(Supplementary Figs. 7 [rid=MOESM1] – 9 [rid=MOESM1] ; Supplementary Data 30 [this file] – 32 [rid=MOESM34] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM33: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM34: At GWAS threshold of 1 × 10 , genetic variants residing in regions harboring AD-associated 5hmC alterations were significantly enriched in DNase I hypersensitive sites (DHSs), transcribed histone modifications, TSS, and transcribed chromatin states, but depleted in repressed chromatin states and H3K27me3 in various tissues such as blood, blood vessel, embryonic stem cells (ESCs), colon, liver, and fetal intestine(Supplementary Figs. 7 [rid=MOESM1] – 9 [rid=MOESM1] ; Supplementary Data 30 [rid=MOESM32] – 32 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM35: At PP4 > 0.8 (i.e., a high posterior probability that a single shared variant is responsible for two signals), three 5hmC regions, including one associated with Aβ (C12P044736) and two associated with PHFtau (C12P044736, C14P057817), colocalize with known AD genes ( and ) near the variants rs12582221 and rs12590654 (Supplementary Data 33 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM36: Finally, the observed associations of 5hmC with Aβ or PHFtau slightly attenuated but remained significant after controlling for each other (Supplementary Data 34 [this file] – 35 [rid=MOESM37] ), suggesting that 5hmC alterations may have an independent effect on these two hallmarks of AD pathologies. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM37: Finally, the observed associations of 5hmC with Aβ or PHFtau slightly attenuated but remained significant after controlling for each other (Supplementary Data 34 [rid=MOESM36] – 35 [this file] ), suggesting that 5hmC alterations may have an independent effect on these two hallmarks of AD pathologies. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM38: Further information on research design is available in the Nature Portfolio Reporting Summary [this file] linked to this article. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM39: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM40: Source data are provided as a Source Data [this file] file. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 243 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 68810\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 255203\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 69532 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-febbc8c83a1e92accb10f3f0a8d6022fb0c706e32ca74db6f2ffc266e0917dd8\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 69532 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\n",
      "  \"datasets\": [\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE291402\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE291402\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"syn11724057\",\n",
      "      \"repository_reference\": \"https://www.synapse.org/Synapse:syn11724057\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"syn3388564\",\n",
      "      \"repository_reference\": \"https://www.synapse.org/#!Synapse:syn3388564\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"syn17015098\",\n",
      "      \"repository_reference\": \"https://www.synapse.org/Synapse:syn17015098\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"syn4896408\",\n",
      "      \"repository_reference\": \"https://www.synapse.org/Synapse:syn4896408\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    }\n",
      "  ]\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'GSE291402', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE291402'}, {'dataset_identifier': 'syn11724057', 'repository_reference': 'https://www.synapse.org/Synapse:syn11724057'}, {'dataset_identifier': 'syn3388564', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn3388564'}, {'dataset_identifier': 'syn17015098', 'repository_reference': 'https://www.synapse.org/Synapse:syn17015098'}, {'dataset_identifier': 'syn4896408', 'repository_reference': 'https://www.synapse.org/Synapse:syn4896408'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 6 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE291402\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn11724057\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn3388564\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn17015098\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn4896408\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'GSE291402', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE291402', 'dataset_id': 'GSE291402'}, {'dataset_identifier': 'syn11724057', 'repository_reference': 'https://www.synapse.org/Synapse:syn11724057', 'dataset_id': 'syn11724057'}, {'dataset_identifier': 'syn3388564', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn3388564', 'dataset_id': 'syn3388564'}, {'dataset_identifier': 'syn17015098', 'repository_reference': 'https://www.synapse.org/Synapse:syn17015098', 'dataset_id': 'syn17015098'}, {'dataset_identifier': 'syn4896408', 'repository_reference': 'https://www.synapse.org/Synapse:syn4896408', 'dataset_id': 'syn4896408'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE291402', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE291402', 'dataset_id': 'GSE291402'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE291402\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE291402, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE291402\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE291402\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE291402', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn11724057', 'repository_reference': 'https://www.synapse.org/Synapse:syn11724057', 'dataset_id': 'syn11724057'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn11724057\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn11724057, synapse.org, https://www.synapse.org/Synapse:syn11724057\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn11724057\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn11724057', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn3388564', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn3388564', 'dataset_id': 'syn3388564'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn3388564\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn3388564, synapse.org, https://www.synapse.org/#!Synapse:syn3388564\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn3388564\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn3388564', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn17015098', 'repository_reference': 'https://www.synapse.org/Synapse:syn17015098', 'dataset_id': 'syn17015098'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn17015098\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn17015098, synapse.org, https://www.synapse.org/Synapse:syn17015098\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn17015098\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn17015098', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn4896408', 'repository_reference': 'https://www.synapse.org/Synapse:syn4896408', 'dataset_id': 'syn4896408'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn4896408\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn4896408, synapse.org, https://www.synapse.org/Synapse:syn4896408\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn4896408\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn4896408', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 677 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 605 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 256 - INFO - Augmented dataset links: [{'dataset_identifier': 'GSE291402', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn11724057', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn3388564', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn17015098', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn4896408', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 5 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE291402\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE291402\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE291402 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE291402 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE291402 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE291402\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE291402\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn11724057\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn11724057\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn11724057 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn11724057 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn11724057 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn11724057 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn11724057 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn11724057\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn11724057\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn3388564\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn3388564\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3388564 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn3388564 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn3388564 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3388564 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn3388564 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 3 with repo: synapse.org and accession_id: syn3388564\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn3388564\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 3: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn17015098\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn17015098\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn17015098 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn17015098 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn17015098 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn17015098 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn17015098 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 4 with repo: synapse.org and accession_id: syn17015098\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn17015098\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 4: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn4896408\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn4896408\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn4896408 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn4896408 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn4896408 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn4896408 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn4896408 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 5 with repo: synapse.org and accession_id: syn4896408\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn4896408\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 5: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 5\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 265 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 45, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 477 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 10th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC7113080&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 394 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x13dbcc690>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 445 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 547 - INFO - Extracted 25 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 557 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#sec1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for sec1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#s1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 30) for s1: INTRODUCTION [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#s3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for s3: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#s7'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for s7: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#s7'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for s7: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#s8'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for s8: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#s8'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for s8: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#s8'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for s8: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#s8'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for s8: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#s8'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for s8: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#s8'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for s8: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#s8'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for s8: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#s8'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for s8: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#s8'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for s8: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#s9'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for s9: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#s13'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 28) for s13: DISCUSSION [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#fn-group1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 27) for fn-group1: Footnotes [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#sec20'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 73) for sec20: AUTHORS' DISCLOSURES OF POTENTIAL CONFLICTS OF INTEREST [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#sec20'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 73) for sec20: AUTHORS' DISCLOSURES OF POTENTIAL CONFLICTS OF INTEREST [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#sec20'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 73) for sec20: AUTHORS' DISCLOSURES OF POTENTIAL CONFLICTS OF INTEREST [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 23636\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 83686\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 24358 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-0693d6fdf57ac063bca17298ed29a4ba2e7d8e6b7719c641b18defe3c3a09b0c\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 24358 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\n",
      "  \"datasets\": [\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"https://doi.org/10.1200/CCI.19.00127\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"syn17394041\",\n",
      "      \"repository_reference\": \"https://www.synapse.org/#!Synapse:syn17394041\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"PMC7113080\",\n",
      "      \"repository_reference\": \"https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080/\"\n",
      "    }\n",
      "  ]\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://doi.org/10.1200/CCI.19.00127'}, {'dataset_identifier': 'syn17394041', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn17394041'}, {'dataset_identifier': 'PMC7113080', 'repository_reference': 'https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080/'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 4 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn17394041\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: PMC7113080\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://doi.org/10.1200/CCI.19.00127'}, {'dataset_identifier': 'syn17394041', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn17394041', 'dataset_id': 'syn17394041'}, {'dataset_identifier': 'PMC7113080', 'repository_reference': 'https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080/', 'dataset_id': 'PMC7113080'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 677 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 605 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://doi.org/10.1200/CCI.19.00127'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(10\\.\\d{4,9}/).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: 10.1200/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: 10.1200/, None, https://doi.org/10.1200/CCI.19.00127\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: 10.1200/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://doi.org/10.1200/CCI.19.00127, identifier: 10.1200/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://doi.org/10.1200/CCI.19.00127\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.1200/ for https://doi.org/10.1200/CCI.19.00127\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 10.1200/ for https://doi.org/10.1200/CCI.19.00127\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1153 - INFO - Found id_pattern match for https://doi.org/10.1200/CCI.19.00127 in 10\\.\\d{4,9}/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': '10.1200/', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn17394041', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn17394041', 'dataset_id': 'syn17394041'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn17394041\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn17394041, synapse.org, https://www.synapse.org/#!Synapse:syn17394041\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn17394041\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn17394041', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'PMC7113080', 'repository_reference': 'https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080/', 'dataset_id': 'PMC7113080'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: PMC7113080\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID PMC7113080 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080/, identifier: PMC7113080\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier PMC7113080 for https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier PMC7113080 for https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier PMC7113080 for https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier PMC7113080 for https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier PMC7113080 for https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'PMC7113080', 'data_repository': 'pmc.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': '10.1200/', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn17394041', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'PMC7113080', 'data_repository': 'pmc.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 3 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: 10.1200/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: doi.org, identifier: 10.1200/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.1200/ for doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for 10.1200/ in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 1 with repo: doi.org and accession_id: 10.1200/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://doi.org/10.1200/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn17394041\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn17394041\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn17394041 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn17394041 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn17394041 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn17394041 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn17394041 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn17394041\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn17394041\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: PMC7113080\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: pmc.ncbi.nlm.nih.gov, identifier: PMC7113080\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier PMC7113080 for pmc.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier PMC7113080 for pmc.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier PMC7113080 for pmc.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier PMC7113080 for pmc.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier PMC7113080 for pmc.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: pmc.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 976 - WARNING - URL pmc.ncbi.nlm.nih.gov may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for PMC7113080 in pmc.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 3 with repo: pmc.ncbi.nlm.nih.gov and accession_id: PMC7113080\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1254 - WARNING - Repository pmc.ncbi.nlm.nih.gov unknown in Ontology. Skipping dataset page 3.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 3\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 28, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 640 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - Parsed data extraction completed. Elements collected: 4\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 481 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 574 - INFO - \n",
      "Progress: 11/100 (11.00%) | Elapsed: 00:02:52 | ETA: 00:23:18\n",
      "\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 11th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC8847344)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC8847344\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC8847344&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 406 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC8847344.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC8847344 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 430 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC8847344, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 443 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 264 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 454 - INFO - Found 7 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b0068500>, <Element supplementary-material at 0x1b6063740>, <Element supplementary-material at 0x1b6060f00>, <Element supplementary-material at 0x1b6061b40>, <Element supplementary-material at 0x1b6061a40>, <Element supplementary-material at 0x1b6060240>, <Element supplementary-material at 0x1b60625c0>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 462 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b0068500>, <Element supplementary-material at 0x1b6063740>, <Element supplementary-material at 0x1b6060f00>, <Element supplementary-material at 0x1b6061b40>, <Element supplementary-material at 0x1b6061a40>, <Element supplementary-material at 0x1b6060240>, <Element supplementary-material at 0x1b60625c0>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 548 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM1: The data set is released in its entirety as Supplementary Materials [this file] to this paper, via an interactive website ( https://cancertrials.io/ (https://cancertrials.io/) ), and through the online Synapse repository (ID: syn25813713) . \n",
      "For trials reporting event-free survival data (e.g., PFS; Supplementary Fig. 1 [this file] ) median = 0.950 (Q1 = 0.909, mean: 0.929, Q3 = 0.970) as compared to a theoretical maximal = 0.996, which corresponds to 5% of observed variance not explained by the Weibull model. This is illustrated in Supplementary Fig. 2a, b [this file] for Weibull fits to OS and PFS data for panitumumab in combination with FOLFIRI and for FOLFIRI alone in wild-type and mutant metastatic colorectal cancer (trial 20050181 ; average = 0.99 for OS curves and 0.92 for PFS curves). \n",
      "Across the entire data set, some of the worst fits for two-parameter Weibull forms were observed for trials with relatively few events, for example, the Chronicle trial ( NCT00427713 (NCT00427713) ) with only eight deaths in the treatment arm and 16 progression events in the observation arm (Fig. 2c [rid=Fig2] ; Supplementary Fig. 1 [this file] ). \n",
      "For a small number of trials in which the asymptote of the fitted survival curve was greater than zero (i.e., patients were expected to be alive at the end of the longest follow-up), a three-parameter Weibull distribution consisting of the traditional two-parameter distribution with an additional “cure rate” term had an improved fit (Supplementary Fig. 3 [this file] ). \n",
      "This is potentially consistent with a two-population hypothesis (Fig. 3a [rid=Fig3] ; Supplementary Fig. 4 [this file] ). Moreover, a mixture model exhibited no meaningful improvement in fit as compared to a single-distribution for OS data from ICI or control arms in these trials (Fig. 3b [rid=Fig3] ; Supplementary Fig. 4 [this file] ). This generated the steep decline in PFS values observable in the control and experimental arms of actual ICI trials, and improved fit to PFS distributions, raising mean from 0.93 to 0.98 (Fig. 3c [rid=Fig3] ; Supplementary Fig. 4 [this file] ; “Methods”). \n",
      "A challenge in the analysis of such data is that, when too few events have occurred, nonparametric numeric confidence estimates return non-informative values (usually reported as a value “not reached” or “indeterminate” as illustrated for two different scenarios in Supplementary Fig. 5a [this file] ). We found that 12-month survival estimates were non-informative for 20–40% of OS trial arms, and 23–61% of event-free survival trial arms, with higher failure rates occurring when the sample size was smaller (Supplementary Fig. 5b, c [this file] ). \n",
      "For event-free survival (for which 99 event-free survival curves could be compared), the precision of a Weibull-based approach was also greater than a nonparametric approach across all sample sizes, while accuracy was equivalent for small sample sizes (fewer than 40 patients) (Supplementary Fig. 6a [this file] ). \n",
      "Thus, parametric fitting of data from 15 patients made a comparably precise and accurate estimate of median PFS as nonparametric analysis of 60 patients (Supplementary Fig. 6b [this file] ). \n",
      "We observed that for a representative simulation, a two-parameter Weibull form accurately described the synthetic trial data ( = 0.98; Supplementary Fig. 7 [this file] ). \n",
      "Parameter values for trials reporting event-free survival followed a similar pattern to OS values (Supplementary Fig. 8 [this file] and Supplementary Data 2 [rid=MOESM5] ). \n",
      "A similar pattern was observed for event-free survival data, with control and experimental values differing by 1.0% for unsuccessful trials and 36% for successful trials (Supplementary Fig. 9 [this file] ). \n",
      "For event-free survival data, 155 comparisons of experimental and control arms from 146 trial figures revealed a range of Δ values from +0.55 to −0.85 (median |Δ | = 0.08) (Supplementary Fig. 9 [this file] ). \n",
      "All data generated or analyzed during this study are included in this published article (and its supplementary information [this file] files). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM2: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM3: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM4: The accuracy of IPD imputation is discussed further in Supplementary Data 1 [this file] and “Methods”. \n",
      "Trials with a hazard ratio difference greater than 0.1, or with perceptible visual differences, were removed from the final data set and not analyzed further (Supplementary Data 1 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM5: Best-fit Weibull parameters were compared across cancer types and metastatic status using an ANOVA test with a Bonferroni correction for multiple hypothesis testing at a two-tailed significance level of 0.05 (see Supplementary Data 2 [this file] ). Parameter values for trials reporting event-free survival followed a similar pattern to OS values (Supplementary Fig. 8 [rid=MOESM1] and Supplementary Data 2 [this file] ). \n",
      "All code used in this study is included in Supplementary Data 2 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM6: Finally, when we examined PFS data from an additional 25 ICI trials, we found that the drop in survival at early event times (which we identified as the time corresponding to the greatest change in the slope of the survival curve) was strongly correlated with the time of the first radiological scan (as reported in trials’ methods sections; Pearson correlation 0.982, < 10 ; Supplementary Data 3 [this file] ). \n",
      "All extracted values can be found in Supplementary Data 3 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM7: Further information on research design is available in the Nature Research Reporting Summary [this file] linked to this article. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 243 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 38131\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 157076\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 38853 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-e9aacaf3106d5000f1e534b298af48500335d588a6cee0d191c952db0a00d191\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 38853 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\n",
      "  \"datasets\": [\n",
      "    {\n",
      "      \"dataset_identifier\": \"syn25813713\",\n",
      "      \"repository_reference\": \"https://www.synapse.org/#!Synapse:syn25813713\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"https://cancertrials.io/\"\n",
      "    }\n",
      "  ]\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'syn25813713', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn25813713'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://cancertrials.io/'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 2 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn25813713\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'syn25813713', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn25813713', 'dataset_id': 'syn25813713'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://cancertrials.io/', 'dataset_id': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn25813713', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn25813713', 'dataset_id': 'syn25813713'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn25813713\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn25813713, synapse.org, https://www.synapse.org/#!Synapse:syn25813713\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn25813713\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn25813713', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://cancertrials.io/', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://cancertrials.io/, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://cancertrials.io/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://cancertrials.io/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'cancertrials.io', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 256 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn25813713', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'cancertrials.io', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 2 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn25813713\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn25813713\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn25813713 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn25813713 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn25813713 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn25813713 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn25813713 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn25813713\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn25813713\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Skipping dataset 2: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 2\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 265 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 9, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 477 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 12th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC9280291)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC9280291\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC9280291&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 406 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC9280291.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC9280291 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 430 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC9280291, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 443 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 264 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 454 - INFO - Found 1 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b006a5c0>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 462 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b006a5c0>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 548 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref DS1: The IPP gene set contains 26 immune-related genes and 3 genes used for normalization, and we used those the total of 29 as input in the IPP models ( Supplementary Table 1 [this file] ). \n",
      "In the 7 studies ( 43 [rid=B43] , 45 [rid=B45] , 46 [rid=B46] , 51 [rid=B51] – 53 [rid=B53] , 58 [rid=B58] ) with GE data collected at time points 3 to 7, there were 270 arrays sampled on 173 patients, of whom 134 were alive and 39 (22.5%) deceased at day 30; 122 were used for training and 51 for testing models ( Supplementary Tables 2 [this file] , 3 [this file] ). \n",
      "We ran the COCONUT algorithm on the 17 studies selected for analysis and assessed the effect of co-normalization: (1) on patient-level GE data across studies ( Figure 1 [rid=F1] and Supplementary Figure 1 [this file] ); (2) at the gene level in controls and cases ( Supplementary Figure 2 [this file] presenting data for CD3D); (3) for 2 genes in controls and cases, here with CLDN8 (a housekeeping gene, with minimal difference in mean GE between controls and cases and minimal overall GE variance) and CEACAM1, up-regulated during sepsis ( Supplementary Figure 3 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 243 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 47355\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 177152\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 48077 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-7ca77868ad56e362b6388a4d23e1f59f8cf781b17ba53019ef6c29957ec5e4aa\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 48077 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\n",
      "  \"datasets\": [\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE27131\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE27131\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE32707\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE32707\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE40586\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE40586\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE66099\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE66099\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE21802\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE21802\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE54514\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE54514\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE20346\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE20346\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE40012\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE40012\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE57065\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE57065\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE60244\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60244\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE65682\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65682\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE95233\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE95233\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"E-MEXP-3589\",\n",
      "      \"repository_reference\": \"https://www.ebi.ac.uk/arrayexpress/experiments/E-MEXP-3589\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"E-MTAB-1548\",\n",
      "      \"repository_reference\": \"https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-1548\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"E-MTAB-5273/5274\",\n",
      "      \"repository_reference\": \"https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-5273\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE13015\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE13015\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE25504\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE25504\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    }\n",
      "  ]\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'GSE27131', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE27131'}, {'dataset_identifier': 'GSE32707', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE32707'}, {'dataset_identifier': 'GSE40586', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE40586'}, {'dataset_identifier': 'GSE66099', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE66099'}, {'dataset_identifier': 'GSE21802', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE21802'}, {'dataset_identifier': 'GSE54514', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE54514'}, {'dataset_identifier': 'GSE20346', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE20346'}, {'dataset_identifier': 'GSE40012', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE40012'}, {'dataset_identifier': 'GSE57065', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE57065'}, {'dataset_identifier': 'GSE60244', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60244'}, {'dataset_identifier': 'GSE65682', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65682'}, {'dataset_identifier': 'GSE95233', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE95233'}, {'dataset_identifier': 'E-MEXP-3589', 'repository_reference': 'https://www.ebi.ac.uk/arrayexpress/experiments/E-MEXP-3589'}, {'dataset_identifier': 'E-MTAB-1548', 'repository_reference': 'https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-1548'}, {'dataset_identifier': 'E-MTAB-5273/5274', 'repository_reference': 'https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-5273'}, {'dataset_identifier': 'GSE13015', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE13015'}, {'dataset_identifier': 'GSE25504', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE25504'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 18 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE27131\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE32707\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE40586\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE66099\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE21802\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE54514\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE20346\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE40012\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE57065\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE60244\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE65682\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE95233\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: E-MEXP-3589\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: E-MTAB-1548\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: E-MTAB-5273/5274\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE13015\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE25504\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'GSE27131', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE27131', 'dataset_id': 'GSE27131'}, {'dataset_identifier': 'GSE32707', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE32707', 'dataset_id': 'GSE32707'}, {'dataset_identifier': 'GSE40586', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE40586', 'dataset_id': 'GSE40586'}, {'dataset_identifier': 'GSE66099', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE66099', 'dataset_id': 'GSE66099'}, {'dataset_identifier': 'GSE21802', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE21802', 'dataset_id': 'GSE21802'}, {'dataset_identifier': 'GSE54514', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE54514', 'dataset_id': 'GSE54514'}, {'dataset_identifier': 'GSE20346', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE20346', 'dataset_id': 'GSE20346'}, {'dataset_identifier': 'GSE40012', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE40012', 'dataset_id': 'GSE40012'}, {'dataset_identifier': 'GSE57065', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE57065', 'dataset_id': 'GSE57065'}, {'dataset_identifier': 'GSE60244', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60244', 'dataset_id': 'GSE60244'}, {'dataset_identifier': 'GSE65682', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65682', 'dataset_id': 'GSE65682'}, {'dataset_identifier': 'GSE95233', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE95233', 'dataset_id': 'GSE95233'}, {'dataset_identifier': 'E-MEXP-3589', 'repository_reference': 'https://www.ebi.ac.uk/arrayexpress/experiments/E-MEXP-3589', 'dataset_id': 'E-MEXP-3589'}, {'dataset_identifier': 'E-MTAB-1548', 'repository_reference': 'https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-1548', 'dataset_id': 'E-MTAB-1548'}, {'dataset_identifier': 'E-MTAB-5273/5274', 'repository_reference': 'https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-5273', 'dataset_id': 'E-MTAB-5273/5274'}, {'dataset_identifier': 'GSE13015', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE13015', 'dataset_id': 'GSE13015'}, {'dataset_identifier': 'GSE25504', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE25504', 'dataset_id': 'GSE25504'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE27131', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE27131', 'dataset_id': 'GSE27131'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE27131\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE27131, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE27131\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE27131\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE27131', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE32707', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE32707', 'dataset_id': 'GSE32707'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE32707\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE32707, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE32707\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE32707\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE32707', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE40586', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE40586', 'dataset_id': 'GSE40586'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE40586\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE40586, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE40586\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE40586\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE40586', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE66099', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE66099', 'dataset_id': 'GSE66099'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE66099\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE66099, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE66099\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE66099\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE66099', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE21802', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE21802', 'dataset_id': 'GSE21802'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE21802\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE21802, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE21802\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE21802\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE21802', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE54514', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE54514', 'dataset_id': 'GSE54514'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE54514\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE54514, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE54514\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE54514\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE54514', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE20346', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE20346', 'dataset_id': 'GSE20346'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE20346\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE20346, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE20346\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE20346\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE20346', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE40012', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE40012', 'dataset_id': 'GSE40012'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE40012\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE40012, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE40012\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE40012\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE40012', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE57065', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE57065', 'dataset_id': 'GSE57065'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE57065\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE57065, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE57065\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE57065\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE57065', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE60244', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60244', 'dataset_id': 'GSE60244'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE60244\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE60244, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60244\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE60244\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE60244', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE65682', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65682', 'dataset_id': 'GSE65682'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE65682\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE65682, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65682\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE65682\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE65682', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE95233', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE95233', 'dataset_id': 'GSE95233'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE95233\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE95233, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE95233\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE95233\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE95233', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'E-MEXP-3589', 'repository_reference': 'https://www.ebi.ac.uk/arrayexpress/experiments/E-MEXP-3589', 'dataset_id': 'E-MEXP-3589'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: E-MEXP-3589\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID E-MEXP-3589 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.ebi.ac.uk/arrayexpress/experiments/E-MEXP-3589, identifier: E-MEXP-3589\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.ebi.ac.uk/arrayexpress/experiments/E-MEXP-3589\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-MEXP-3589 for https://www.ebi.ac.uk/arrayexpress/experiments/E-MEXP-3589\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-MEXP-3589 for https://www.ebi.ac.uk/arrayexpress/experiments/E-MEXP-3589\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-MEXP-3589 for https://www.ebi.ac.uk/arrayexpress/experiments/E-MEXP-3589\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-MEXP-3589 for https://www.ebi.ac.uk/arrayexpress/experiments/E-MEXP-3589\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Found partial match for https://www.ebi.ac.uk/arrayexpress/experiments/E-MEXP-3589 in ArrayExpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'E-MEXP-3589', 'data_repository': 'arrayexpress', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'E-MTAB-1548', 'repository_reference': 'https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-1548', 'dataset_id': 'E-MTAB-1548'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: E-MTAB-1548\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID E-MTAB-1548 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-1548, identifier: E-MTAB-1548\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-1548\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-MTAB-1548 for https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-1548\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-MTAB-1548 for https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-1548\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-MTAB-1548 for https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-1548\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-MTAB-1548 for https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-1548\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Found partial match for https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-1548 in ArrayExpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'E-MTAB-1548', 'data_repository': 'arrayexpress', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'E-MTAB-5273/5274', 'repository_reference': 'https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-5273', 'dataset_id': 'E-MTAB-5273/5274'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: E-MTAB-5273/5274\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID E-MTAB-5273/5274 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-5273, identifier: E-MTAB-5273/5274\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-5273\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-MTAB-5273/5274 for https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-5273\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-MTAB-5273/5274 for https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-5273\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-MTAB-5273/5274 for https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-5273\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-MTAB-5273/5274 for https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-5273\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Found partial match for https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-5273 in ArrayExpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'E-MTAB-5273/5274', 'data_repository': 'arrayexpress', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE13015', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE13015', 'dataset_id': 'GSE13015'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE13015\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE13015, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE13015\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE13015\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE13015', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE25504', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE25504', 'dataset_id': 'GSE25504'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE25504\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE25504, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE25504\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE25504\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE25504', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 677 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 605 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 256 - INFO - Augmented dataset links: [{'dataset_identifier': 'GSE27131', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE32707', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE40586', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE66099', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE21802', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE54514', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE20346', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE40012', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE57065', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE60244', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE65682', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE95233', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'E-MEXP-3589', 'data_repository': 'arrayexpress', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'E-MTAB-1548', 'data_repository': 'arrayexpress', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'E-MTAB-5273/5274', 'data_repository': 'arrayexpress', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE13015', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE25504', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 17 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE27131\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE27131\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE27131 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE27131 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE27131 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE27131\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE27131\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE32707\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE32707\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE32707 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE32707 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE32707 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 2 with repo: geo and accession_id: GSE32707\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE32707\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 2: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE40586\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE40586\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE40586 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE40586 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE40586 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 3 with repo: geo and accession_id: GSE40586\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE40586\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 3: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE66099\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE66099\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE66099 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE66099 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE66099 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 4 with repo: geo and accession_id: GSE66099\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE66099\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 4: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE21802\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE21802\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE21802 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE21802 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE21802 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 5 with repo: geo and accession_id: GSE21802\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE21802\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 5: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE54514\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE54514\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE54514 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE54514 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE54514 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 6 with repo: geo and accession_id: GSE54514\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE54514\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 6: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE20346\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE20346\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE20346 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE20346 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE20346 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 7 with repo: geo and accession_id: GSE20346\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE20346\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 7: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE40012\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE40012\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE40012 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE40012 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE40012 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 8 with repo: geo and accession_id: GSE40012\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE40012\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 8: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE57065\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE57065\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE57065 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE57065 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE57065 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 9 with repo: geo and accession_id: GSE57065\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE57065\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 9: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 10 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE60244\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE60244\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE60244 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE60244 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE60244 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 10 with repo: geo and accession_id: GSE60244\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60244\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 10: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 11 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE65682\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE65682\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE65682 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE65682 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE65682 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 11 with repo: geo and accession_id: GSE65682\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65682\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 11: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 12 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE95233\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE95233\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE95233 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE95233 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE95233 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 12 with repo: geo and accession_id: GSE95233\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE95233\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 12: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 13 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: E-MEXP-3589\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: arrayexpress, identifier: E-MEXP-3589\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-MEXP-3589 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-MEXP-3589 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-MEXP-3589 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-MEXP-3589 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for E-MEXP-3589 in arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 13 with repo: arrayexpress and accession_id: E-MEXP-3589\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ebi.ac.uk/biostudies/ArrayExpress/studies/E-MEXP-3589\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 13: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 14 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: E-MTAB-1548\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: arrayexpress, identifier: E-MTAB-1548\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-MTAB-1548 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-MTAB-1548 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-MTAB-1548 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-MTAB-1548 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for E-MTAB-1548 in arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 14 with repo: arrayexpress and accession_id: E-MTAB-1548\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ebi.ac.uk/biostudies/ArrayExpress/studies/E-MTAB-1548\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 14: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 15 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: E-MTAB-5273/5274\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: arrayexpress, identifier: E-MTAB-5273/5274\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-MTAB-5273/5274 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-MTAB-5273/5274 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-MTAB-5273/5274 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-MTAB-5273/5274 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for E-MTAB-5273/5274 in arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 15 with repo: arrayexpress and accession_id: E-MTAB-5273/5274\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ebi.ac.uk/biostudies/ArrayExpress/studies/E-MTAB-5273/5274\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 15: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 16 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE13015\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE13015\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE13015 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE13015 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE13015 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 16 with repo: geo and accession_id: GSE13015\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE13015\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 16: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 17 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE25504\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE25504\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE25504 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE25504 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE25504 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 17 with repo: geo and accession_id: GSE25504\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE25504\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 17: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 17\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 265 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 18, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 477 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 13th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10029021)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10029021\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC10029021&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 406 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC10029021.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC10029021 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 430 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC10029021, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 443 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 264 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 454 - INFO - Found 1 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b08c5400>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 462 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b08c5400>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 548 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref SD1: S1 [this file] ) comprising 672 distinct donors. S2 [this file] ; Methods [this file] ), we retained 654 samples with matched genotype and cortical RNA-seq data spanning 4-39 PCW, the majority falling within the first two trimesters (Tri1, n=216; Tri2, n=433; Table S1 [this file] ). S3 [this file] ; Table S1 [this file] ; Methods [this file] ). \n",
      "Compared with adult brain eQTL datasets from PsychENCODE ( 13 [rid=R13] , 14 [rid=R14] ) and GTEx ( 16 [rid=R16] ), we identified 2,488 unique “fetal-specific” eGenes in this dataset ( Fig. 1B [rid=F1] ; Table S1 [this file] ). S3 [this file] ). S3 [this file] ), suggesting that these context-specific annotations may be more relevant for interpretation of neuropsychiatric GWAS signals, which are known to be under negative selection. S3 [this file] ; Table S1 [this file] ). \n",
      "S4 [this file] ; Fig. S5 [this file] ; Methods [this file] ), identifying 11,845 and 7,490 genes with a significant isoQTL or sQTL (termed “isoGenes” and “sGenes”; Table S1 [this file] ), respectively. As expected, -xQTL are strongly clustered near their target genes’ transcription start site (TSS) ( Fig. 1E [rid=F1] ) and are correspondingly enriched in functional regions of the genome ( Fig. 1F [rid=F1] ; Table S1 [this file] ). To investigate the variant-level overlap among -e/iso/sQTLs, we calculated Storey’s pi1 statistic ( 40 [rid=R40] ) and found that genetic variants directly associated with RNA splicing (especially isoQTL) capture independent and additional signals from eQTLs ( Fig. 1H [rid=F1] ; Methods [this file] ). \n",
      "S3 [this file] ; Table S1 [this file] ; Methods [this file] ), we identified multiple independent regulatory signals for 3,570 eGenes in the fetal brain, with some exhibiting up to 10 groups of conditionally independent -eQTL signals. Each CS prioritized a median of 5 SNPs, with 2,423 containing exactly one SNP, which are strong candidates for functional validation ( Data S1 [this file] ). \n",
      "S6 [this file] ; Table S1 [this file] ; Methods [this file] ). \n",
      "S2 [this file] ; Methods [this file] ), we inferred genetic ancestries in the fetal brain samples in reference to the 1000 Genomes ( 44 [rid=R44] ). S1 [this file] ; Table S1 [this file] ). Following the multi-ancestry QTL mapping pipeline, we independently mapped xQTLs in the three largest ancestry groupings: EUR, AMR, and AFR. 986 eGenes were shared between the multi-ancestry dataset and the three ancestry groupings ( Fig. 2B [rid=F2] ; Table S2 [this file] ). We then performed statistical fine-mapping separately in the ancestry groupings ( 42 [rid=R42] ), and show that AFR, despite of the smallest sample size, has on average the smallest credible sets (CS) of causal variants ( Fig. 2D [rid=F2] ; Data S2 [this file] , S3 [this file] , S4 [this file] ). \n",
      "S7 [this file] ; Table S3 [this file] ). Mirroring eQTL results, we found that heritability drops significantly from Tri1 to Tri2 ( Fig. 3C [rid=F3] ; Wilcoxon p < 2.2e-16; Table S3 [this file] ; Data S5 [this file] , S6 [this file] ). \n",
      "S7 [this file] ), gene expression levels, or phenotypic variance across Tri1 and Tri2 that could be driving these results ( Fig. S8 [this file] ). S7 [this file] ; Table S3 [this file] ; Data S7 [this file] , S8 [this file] ). \n",
      "We generated continuous annotations for genetic variants based on fine-mapping posterior probabilities ( 51 [rid=R51] ) and found SCZ GWAS signals to be highly enriched among fetal brain regulatory elements -- more so than in the adult brain ( Fig. 4B [rid=F4] ; Table S4 [this file] ; Methods [this file] ). S9 [this file] ; Table S4 [this file] ; Methods [this file] ). S9 [this file] ; Table S4 [this file] ). \n",
      "Across these traits, we consistently observed greater heritability mediated by isoform regulation (isoform h /h 6.45±1.92%) than that of gene expression (3.15±1.25%) or splicing (2.13±1.74%), although the overall extent of mediation remains small ( Fig. 4C [rid=F4] ; Table S4 [this file] ; Methods [this file] ). Extending these analyses across trimester-specific annotations, although less well powered, pointed to isoform-regulation in Tri2 as being particularly important, especially for BIP ( Fig. 4D [rid=F4] ; Table S4 [this file] ). \n",
      "S10 [this file] ; Table S5 [this file] ; Methods [this file] ). \n",
      "S10 [this file] ; Table S5 [this file] ). S10 [this file] ) ( 10 [rid=R10] ). S10 [this file] ). \n",
      "Other novel disease mechanisms prioritized by these analyses include: downregulation of the GABA -receptor subunit in SCZ (eQTL: rs514733, CLPP=0.32; Fig. 5B [rid=F5] ; Table S5 [this file] ), providing genetic evidence for GABAergic dysfunction hypothesized to contribute to disease pathophysiology ( 60 [rid=R60] ); splicing dysregulation of in BIP (top colocalization sQTL: rs11750832, CLPP=0.97; Fig. S10 [this file] ; Table S5 [this file] ) which encodes adenylate cyclase, a cell membrane-bound enzyme regulating cAMP signaling and a target of the mood stabilizer lithium ( 61 [rid=R61] ); splicing dysregulation in BIP of (sQTL: rs17183814, CLPP=0.70; Fig. S10 [this file] ; Table S5 [this file] ), a voltage gated sodium channel subunit with rare variant associations in ASD and epilepsy ( 62 [rid=R62] , 63 [rid=R63] ); and in MDD, the dysregulated splicing of (CLPP=0.43; Fig. S10 [this file] ; Table S5 [this file] ), a highly constrained E3 ubiquitin-protein ligase and regulator of N6-methyladenosine RNA modification involved in neural-immune activation ( 64 [rid=R64] ). \n",
      "We next conducted an isoform-centric transcriptome-wide association study (isoTWAS) ( 65 [rid=R65] ), to identify genes and their isoforms whose regulated expression is associated with SCZ risk. isoTWAS identified 536 isoforms across 271 unique genes with significant isoTWAS associations (Bonferroni-corrected P < 0.05 across genes, FWER-corrected P < 0.05 across transcripts of the same gene, permutation P < 0.05; Methods [this file] ). To leverage local LD and to account for SNP weight correlations, we performed fine-mapping on isoTWAS associations that passed permutation testing ( Fig. 5D [rid=F5] ; Table S5 [this file] ). \n",
      "Among 14 TWAS-identified genes with >3 conditionally independent eQTLs, the MRLocus framework provided additional support for 7 genes with FDR<0.2 ( Table S5 [this file] ; Methods [this file] ). S11 [this file] ). \n",
      "In total, we identified 124 gene and isoform co-expression modules demonstrating enrichment for all major developmental cell types ( 36 [rid=R36] ) and recapitulating early biochemical processes and developmental pathways ( Fig. 6A [rid=F6] ; Methods [this file] ). \n",
      "Overlaying enrichment for common and rare variation within the hierarchical structure defined by co-expression pinpointed groups of modules that harbor most of neurodevelopmental disease risk ( Fig. 6B [rid=F6] ; Table S6 [this file] ; Methods [this file] ). S12 [this file] ). \n",
      "S12 [this file] ), which exhibits strong enrichment for cross-disorder common variation and rare variation risk for ASD, epilepsy, and SCZ. S12 [this file] ), which strongly enriched for synaptic gene ontologies and excitatory neuron marker genes. S12 [this file] ). \n",
      "S12 [this file] ) displayed robust BIP, education attainment (EA), and SCZ enrichment while harboring high risk hub genes like and – similar to the prenatally-enriched M37 module described in Li et al ( 18 [rid=R18] ). S12 [this file] ). \n",
      "CellWalker mapped 21.7% of eQTL-containing genes to a specific cellular context in the developing human brain, corresponding to 3,739 of the bulk-derived eQTLs ( Fig. 7A [rid=F7] ; Table S7 [this file] ; Methods [this file] ). To identify such ieQTLs, in which SNP-gene associations are modulated by the levels of a given module, we tested the interaction effect between genotype and module eigengenes on gene and isoform expression ( Fig. 7B [rid=F7] ; Methods [this file] ). Across all gene and isoform modules, 8,008 ieQTLs were identified after the permutation test (BH-corrected FDR<0.05; Methods [this file] ), in which 3,960 ieQTLs were specific to a single module ( Fig. 7C [rid=F7] ; Data S9 [this file] ). Finally, integrating disease GWAS, we identified significant colocalizations (CLPP>0.01) between ieQTLs and 22 SCZ GWAS loci, 13 of which are not found with bulk cA-eQTLs ( Table S5 [this file] ). \n",
      "See Methods [this file] for details. ( ) A schematic of module interaction ieQTL mapping and validation in cultured neurons and progenitors. ( ) Results from module ieQTL mapping. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 243 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 161558\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 161016\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 160559\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 160024\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 159435\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 158865\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 158338\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 157747\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 157156\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 156574\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 155990\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 155414\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 154822\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 154238\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 153663\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 153091\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 152522\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 151958\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 151397\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 150832\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 150267\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 149707\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 149140\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 148573\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 148006\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 147430\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 146852\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 146259\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 145684\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 145089\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 144508\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 143934\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 143350\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 142778\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 142208\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 141629\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 141052\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 140478\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 139897\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 139322\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 138747\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 138148\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 137542\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 136973\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 136402\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 135825\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 135216\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 134634\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 134064\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 133473\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 132882\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 132293\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 131709\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 131115\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 130521\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 129934\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 129336\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 128759\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 128152\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 127572\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 127001\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 445584\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 127723 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-b1a9e844d89a668f1de5ab506e0759c3341377b0c450dd6ba48a04b7215d44d4\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 127723 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\"datasets\":[{\"dataset_identifier\":\"PMC10029021\",\"repository_reference\":\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10029021/\"},{\"dataset_identifier\":\"syn50897018\",\"repository_reference\":\"https://www.synapse.org/#!Synapse:syn50897018.3/datasets/\"}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'PMC10029021', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10029021/'}, {'dataset_identifier': 'syn50897018', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn50897018.3/datasets/'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 2 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: PMC10029021\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn50897018\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'PMC10029021', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10029021/', 'dataset_id': 'PMC10029021'}, {'dataset_identifier': 'syn50897018', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn50897018.3/datasets/', 'dataset_id': 'syn50897018'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'PMC10029021', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10029021/', 'dataset_id': 'PMC10029021'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: PMC10029021\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID PMC10029021 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10029021/, identifier: PMC10029021\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10029021/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier PMC10029021 for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10029021/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier PMC10029021 for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10029021/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier PMC10029021 for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10029021/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier PMC10029021 for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10029021/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier PMC10029021 for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10029021/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10029021/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'PMC10029021', 'data_repository': 'www.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn50897018', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn50897018.3/datasets/', 'dataset_id': 'syn50897018'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn50897018\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn50897018, synapse.org, https://www.synapse.org/#!Synapse:syn50897018.3/datasets/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn50897018\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn50897018', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 256 - INFO - Augmented dataset links: [{'dataset_identifier': 'PMC10029021', 'data_repository': 'www.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn50897018', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 2 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: PMC10029021\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: www.ncbi.nlm.nih.gov, identifier: PMC10029021\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier PMC10029021 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier PMC10029021 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier PMC10029021 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier PMC10029021 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier PMC10029021 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 976 - WARNING - URL www.ncbi.nlm.nih.gov may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for PMC10029021 in www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 1 with repo: www.ncbi.nlm.nih.gov and accession_id: PMC10029021\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1254 - WARNING - Repository www.ncbi.nlm.nih.gov unknown in Ontology. Skipping dataset page 1.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn50897018\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn50897018\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn50897018 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn50897018 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn50897018 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn50897018 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn50897018 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn50897018\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn50897018\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 2\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 265 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 3, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 477 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 14th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11525589)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11525589\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC11525589&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 461 - WARNING - No links found in the section.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 406 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC11525589.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11525589 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 430 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11525589, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 443 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 264 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 454 - INFO - Found 5 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b6739180>, <Element supplementary-material at 0x1b0175ec0>, <Element supplementary-material at 0x1b69bc200>, <Element supplementary-material at 0x1b006bfc0>, <Element supplementary-material at 0x1b7281d00>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 462 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b6739180>, <Element supplementary-material at 0x1b0175ec0>, <Element supplementary-material at 0x1b69bc200>, <Element supplementary-material at 0x1b006bfc0>, <Element supplementary-material at 0x1b7281d00>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 548 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM1: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM2: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM3: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM4: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM5: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 243 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 24926\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 98364\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 25648 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-3ad2fe8fdc5d482ea4045b75a6a21945d4c464ac240bd90ea538fe02c0d35ded\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 25648 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\"datasets\":[{\"dataset_identifier\":\"BraTS-MEN\",\"repository_reference\":\"https://doi.org/10.7303/syn51514106\"}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'BraTS-MEN', 'repository_reference': 'https://doi.org/10.7303/syn51514106'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 1 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: BraTS-MEN\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'BraTS-MEN', 'repository_reference': 'https://doi.org/10.7303/syn51514106', 'dataset_id': 'BraTS-MEN'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'BraTS-MEN', 'repository_reference': 'https://doi.org/10.7303/syn51514106', 'dataset_id': 'BraTS-MEN'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(10\\.\\d{4,9}/).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: 10.7303/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn51514106\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn51514106, None, https://doi.org/10.7303/syn51514106\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn51514106\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://doi.org/10.7303/syn51514106, identifier: syn51514106\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://doi.org/10.7303/syn51514106\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn51514106 for https://doi.org/10.7303/syn51514106\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn51514106 for https://doi.org/10.7303/syn51514106\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn51514106 for https://doi.org/10.7303/syn51514106\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn51514106 for https://doi.org/10.7303/syn51514106\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier syn51514106 for https://doi.org/10.7303/syn51514106\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1153 - INFO - Found id_pattern match for https://doi.org/10.7303/syn51514106 in syn\\d{7,12}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn51514106', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 256 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn51514106', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 1 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn51514106\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn51514106\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn51514106 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn51514106 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn51514106 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn51514106 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn51514106 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn51514106\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn51514106\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 1\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 265 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 6, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 477 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 15th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10836119)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10836119\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC10836119&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 406 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC10836119.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC10836119 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 430 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC10836119, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 443 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 264 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 454 - INFO - Found 2 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b7260d40>, <Element supplementary-material at 0x1b7261280>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 462 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b7260d40>, <Element supplementary-material at 0x1b7261280>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 548 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref SD1: Study characteristics and acylcarnitine descriptions can be found in Supplementary Tables 1 [this file] and 2 [this file] , respectively. \n",
      "The analysis split ADNI participants into two clusters, 2 and 3, at the top of the tree, with significantly different levels of p-tau in cerebrospinal fluid (CSF), glucose uptake measured by [ F] fluorodeoxyglucose-positron emission tomography (FDG-PET), and the 13-item AD assessment scale-cognitive subscale (ADAS-Cog. 13; Fig. 2b [rid=F2] - d [rid=F2] , Supplementary Table 3 [this file] ). All associations remained significant ( Supplementary Table 4 [this file] ). \n",
      "Specifically, we examined acylcarnitine-related SNPs reported by Shin et al. as non-modifiable factors (SNP list is provided in Supplementary Table 5 [this file] , replication in ADNI in Supplementary Table 6 [this file] ) and modifiable acylcarnitine levels (i.e., residuals adjusted for genetic and other non-modifiable factors) to better understand their impact on subgroup separation. \n",
      "Genetic influences only accounted for a small proportion of variance ( Supplementary Table 7 [this file] - 9 [this file] ), except for one epistatic interaction effect of two SNPs: The interplay of rs17806888 (mapped to ) and rs924135 (mapped to ), both of which have been reported to affect succinylcarnitine levels , explained 32% of the initial partitioning of the data into cluster 2 and cluster 3 at the top of the tree ( Fig. 2j [rid=F2] and Supplementary Fig. 2 [rid=SD2] ). When we examined relations to AD, we observed that both single acylcarnitine-related SNPs and epistatic models were not strongly linked to AD and its biomarkers, although some associations were independently confirmed in the MayoLOAD and ROS/MAP studies ( Supplementary Fig. 3 [rid=SD2] and Supplementary Table 10 [this file] ). \n",
      "After cross-normalizing the different cohort datasets for better comparability ( Methods [rid=S6] and Supplementary Fig. 4 [rid=SD2] ), we fitted a multivariable linear model that regresses age on the fasting serum levels of acylcarnitines in the KORA cohort ( Supplementary Fig. 5 [rid=SD2] and Supplementary Table 11 [this file] ). When applied to the ADNI cohort, correlation dropped to 28% (cognitively normal participants only), which was very similar to what we observed when replicating the bioenergetic age computation process in AGES-RS (r = 29%, Supplementary Table 12 [this file] ). \n",
      "Bioenergetic age highlighted interesting relationships between subgroups and AD pathology throughout the tree ( Supplementary Table 13 [this file] ). This was further confirmed by significant associations of bioenergetic age with AD biomarkers across the A-T-(N)-(C) spectrum ( Supplementary Table 14 [this file] ), and consistent findings for cognitive function and grey matter volume in AGES-RS ( Supplementary Table 15 [this file] ). \n",
      "Specifically, bioenergetically younger individuals demonstrated a significantly slower decline over five years ( Supplementary Table 16 [this file] ). This result was replicated using longitudinal diagnosis data from the AGES-RS study ( Fig. 4d [rid=F4] , Supplementary Table 15 [this file] ), providing further evidence that younger bioenergetic age is protective against cognitive decline. \n",
      "We selected the grouping that had the most significant associations across three cognitive measures: ADAS-Cog 13, memory, and executive function ( Supplementary Table 17 [this file] ). \n",
      "The final dataset contained 140 metabolite measurements, covering 23 acylcarnitine species ( Supplementary Table 2 [this file] ), for 1,531 individuals. \n",
      "Data for 22 out of 23 acylcarnitines investigated in ADNI ( Supplementary Table 2 [this file] ) were available here, with the exception being levels of C4:1, which was not reliably measured in KORA. \n",
      "We obtained a list of 32 acylcarnitine-associated SNPs ( Supplementary Table 5 [this file] ) from a large GWAS study performed on a total of 7,824 individuals from the KORA and TwinsUK studies . \n",
      "To investigate robustness of the model, we performed 10-fold, 3 times repeated cross-validation ( Supplementary Table 11 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref SD2: Notably, the most substantial contributions were observed for medium- and long-chain acylcarnitines, highlighting a likely modifiable role for beta-oxidation function ( Supplementary Fig. 1 [this file] ). \n",
      "Genetic influences only accounted for a small proportion of variance ( Supplementary Table 7 [rid=SD1] - 9 [rid=SD1] ), except for one epistatic interaction effect of two SNPs: The interplay of rs17806888 (mapped to ) and rs924135 (mapped to ), both of which have been reported to affect succinylcarnitine levels , explained 32% of the initial partitioning of the data into cluster 2 and cluster 3 at the top of the tree ( Fig. 2j [rid=F2] and Supplementary Fig. 2 [this file] ). When we examined relations to AD, we observed that both single acylcarnitine-related SNPs and epistatic models were not strongly linked to AD and its biomarkers, although some associations were independently confirmed in the MayoLOAD and ROS/MAP studies ( Supplementary Fig. 3 [this file] and Supplementary Table 10 [rid=SD1] ). \n",
      "After cross-normalizing the different cohort datasets for better comparability ( Methods [rid=S6] and Supplementary Fig. 4 [this file] ), we fitted a multivariable linear model that regresses age on the fasting serum levels of acylcarnitines in the KORA cohort ( Supplementary Fig. 5 [this file] and Supplementary Table 11 [rid=SD1] ). \n",
      "Furthermore, this analysis revealed a strong correlation between bioenergetic age and the first principal component of acylcarnitine levels, indicating that the primary axis of variation in the data and the clustering is associated with bioenergetic age ( Supplementary Fig. 6 [this file] ). \n",
      "Rather, we found that individuals within cluster 7 consistently exhibit the same chronological but significantly younger bioenergetic age than other participants irrespective of their diagnostic group ( Supplementary Fig. 7 [this file] ). \n",
      "We attempted to replicate this grouping, which had similar genotype frequencies as in ADNI (59% vs. 58% in ROS/MAP; Supplementary Fig. 8 [this file] ), using trajectories of global cognition over 13 years in the ROS/MAP cohort. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 243 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 37558\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 146240\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 38280 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-1e898efd17a6a1bb95fb90da30144df14beb36c75e0fe5c6f5fa602bf5f9d8e8\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 38280 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\"datasets\":[{\"dataset_identifier\":\"ADNI\",\"repository_reference\":\"https://adni.loni.usc.edu/\"},{\"dataset_identifier\":\"KORA\",\"repository_reference\":\"https://helmholtz-muenchen.managed-otrs.com/external/\"},{\"dataset_identifier\":\"AGES-RS\",\"repository_reference\":\"https://hjarta.is/en/research/ages-phase-1/\"},{\"dataset_identifier\":\"ROS/MAP\",\"repository_reference\":\"https://adknowledgeportal.synapse.org/\"},{\"dataset_identifier\":\"MayoLOAD\",\"repository_reference\":\"https://adknowledgeportal.synapse.org/\"}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'ADNI', 'repository_reference': 'https://adni.loni.usc.edu/'}, {'dataset_identifier': 'KORA', 'repository_reference': 'https://helmholtz-muenchen.managed-otrs.com/external/'}, {'dataset_identifier': 'AGES-RS', 'repository_reference': 'https://hjarta.is/en/research/ages-phase-1/'}, {'dataset_identifier': 'ROS/MAP', 'repository_reference': 'https://adknowledgeportal.synapse.org/'}, {'dataset_identifier': 'MayoLOAD', 'repository_reference': 'https://adknowledgeportal.synapse.org/'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 5 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: KORA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: AGES-RS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: MayoLOAD\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'ADNI', 'repository_reference': 'https://adni.loni.usc.edu/', 'dataset_id': 'ADNI'}, {'dataset_identifier': 'KORA', 'repository_reference': 'https://helmholtz-muenchen.managed-otrs.com/external/', 'dataset_id': 'KORA'}, {'dataset_identifier': 'AGES-RS', 'repository_reference': 'https://hjarta.is/en/research/ages-phase-1/', 'dataset_id': 'AGES-RS'}, {'dataset_identifier': 'ROS/MAP', 'repository_reference': 'https://adknowledgeportal.synapse.org/', 'dataset_id': 'ROS/MAP'}, {'dataset_identifier': 'MayoLOAD', 'repository_reference': 'https://adknowledgeportal.synapse.org/', 'dataset_id': 'MayoLOAD'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'ADNI', 'repository_reference': 'https://adni.loni.usc.edu/', 'dataset_id': 'ADNI'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID ADNI is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://adni.loni.usc.edu/, identifier: ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://adni.loni.usc.edu/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier ADNI for https://adni.loni.usc.edu/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier ADNI for https://adni.loni.usc.edu/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier ADNI for https://adni.loni.usc.edu/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier ADNI for https://adni.loni.usc.edu/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier ADNI for https://adni.loni.usc.edu/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://adni.loni.usc.edu/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'ADNI', 'data_repository': 'adni.loni.usc.edu', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'KORA', 'repository_reference': 'https://helmholtz-muenchen.managed-otrs.com/external/', 'dataset_id': 'KORA'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: KORA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID KORA is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://helmholtz-muenchen.managed-otrs.com/external/, identifier: KORA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://helmholtz-muenchen.managed-otrs.com/external/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier KORA for https://helmholtz-muenchen.managed-otrs.com/external/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier KORA for https://helmholtz-muenchen.managed-otrs.com/external/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier KORA for https://helmholtz-muenchen.managed-otrs.com/external/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier KORA for https://helmholtz-muenchen.managed-otrs.com/external/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier KORA for https://helmholtz-muenchen.managed-otrs.com/external/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://helmholtz-muenchen.managed-otrs.com/external/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'KORA', 'data_repository': 'helmholtz-muenchen.managed-otrs.com', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'AGES-RS', 'repository_reference': 'https://hjarta.is/en/research/ages-phase-1/', 'dataset_id': 'AGES-RS'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: AGES-RS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID AGES-RS is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://hjarta.is/en/research/ages-phase-1/, identifier: AGES-RS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://hjarta.is/en/research/ages-phase-1/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier AGES-RS for https://hjarta.is/en/research/ages-phase-1/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier AGES-RS for https://hjarta.is/en/research/ages-phase-1/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier AGES-RS for https://hjarta.is/en/research/ages-phase-1/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier AGES-RS for https://hjarta.is/en/research/ages-phase-1/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier AGES-RS for https://hjarta.is/en/research/ages-phase-1/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://hjarta.is/en/research/ages-phase-1/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'AGES-RS', 'data_repository': 'hjarta.is', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'ROS/MAP', 'repository_reference': 'https://adknowledgeportal.synapse.org/', 'dataset_id': 'ROS/MAP'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID ROS/MAP is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://adknowledgeportal.synapse.org/, identifier: ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://adknowledgeportal.synapse.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier ROS/MAP for https://adknowledgeportal.synapse.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier ROS/MAP for https://adknowledgeportal.synapse.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier ROS/MAP for https://adknowledgeportal.synapse.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier ROS/MAP for https://adknowledgeportal.synapse.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Found partial match for https://adknowledgeportal.synapse.org/ in Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'ROS/MAP', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'MayoLOAD', 'repository_reference': 'https://adknowledgeportal.synapse.org/', 'dataset_id': 'MayoLOAD'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: MayoLOAD\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID MayoLOAD is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://adknowledgeportal.synapse.org/, identifier: MayoLOAD\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://adknowledgeportal.synapse.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier MayoLOAD for https://adknowledgeportal.synapse.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier MayoLOAD for https://adknowledgeportal.synapse.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier MayoLOAD for https://adknowledgeportal.synapse.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier MayoLOAD for https://adknowledgeportal.synapse.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Found partial match for https://adknowledgeportal.synapse.org/ in Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'MayoLOAD', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 256 - INFO - Augmented dataset links: [{'dataset_identifier': 'ADNI', 'data_repository': 'adni.loni.usc.edu', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'KORA', 'data_repository': 'helmholtz-muenchen.managed-otrs.com', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'AGES-RS', 'data_repository': 'hjarta.is', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'ROS/MAP', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'MayoLOAD', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 5 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: adni.loni.usc.edu, identifier: ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier ADNI for adni.loni.usc.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier ADNI for adni.loni.usc.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier ADNI for adni.loni.usc.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier ADNI for adni.loni.usc.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier ADNI for adni.loni.usc.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: adni.loni.usc.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 976 - WARNING - URL adni.loni.usc.edu may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for ADNI in adni.loni.usc.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 1 with repo: adni.loni.usc.edu and accession_id: ADNI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1254 - WARNING - Repository adni.loni.usc.edu unknown in Ontology. Skipping dataset page 1.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: KORA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: helmholtz-muenchen.managed-otrs.com, identifier: KORA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier KORA for helmholtz-muenchen.managed-otrs.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier KORA for helmholtz-muenchen.managed-otrs.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier KORA for helmholtz-muenchen.managed-otrs.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier KORA for helmholtz-muenchen.managed-otrs.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier KORA for helmholtz-muenchen.managed-otrs.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: helmholtz-muenchen.managed-otrs.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 976 - WARNING - URL helmholtz-muenchen.managed-otrs.com may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for KORA in helmholtz-muenchen.managed-otrs.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 2 with repo: helmholtz-muenchen.managed-otrs.com and accession_id: KORA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1254 - WARNING - Repository helmholtz-muenchen.managed-otrs.com unknown in Ontology. Skipping dataset page 2.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: AGES-RS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: hjarta.is, identifier: AGES-RS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier AGES-RS for hjarta.is\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier AGES-RS for hjarta.is\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier AGES-RS for hjarta.is\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier AGES-RS for hjarta.is\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier AGES-RS for hjarta.is\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: hjarta.is\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 976 - WARNING - URL hjarta.is may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for AGES-RS in hjarta.is\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 3 with repo: hjarta.is and accession_id: AGES-RS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1254 - WARNING - Repository hjarta.is unknown in Ontology. Skipping dataset page 3.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier ROS/MAP for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier ROS/MAP for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier ROS/MAP for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier ROS/MAP for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for ROS/MAP in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1090 - WARNING - Identifier ROS/MAP does not match pattern for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 4 with repo: synapse.org and accession_id: ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:ROS/MAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 4: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: MayoLOAD\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: MayoLOAD\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier MayoLOAD for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier MayoLOAD for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier MayoLOAD for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier MayoLOAD for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for MayoLOAD in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1090 - WARNING - Identifier MayoLOAD does not match pattern for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 5 with repo: synapse.org and accession_id: MayoLOAD\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:MayoLOAD\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 5: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 5\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 265 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 7, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 477 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 16th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC6646046)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC6646046\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC6646046&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 461 - WARNING - No links found in the section.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 406 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC6646046.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC6646046 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 430 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC6646046, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 443 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 264 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 454 - INFO - Found 4 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b7298fc0>, <Element supplementary-material at 0x1b7298f80>, <Element supplementary-material at 0x1b6060b40>, <Element supplementary-material at 0x1b60616c0>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 462 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b7298fc0>, <Element supplementary-material at 0x1b7298f80>, <Element supplementary-material at 0x1b6060b40>, <Element supplementary-material at 0x1b60616c0>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 548 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref SD1: We found that NRGs are indeed connected to more DREs in Hi-C and FANTOM5 data ( Fig. 2b [rid=F2] , Supplementary Fig. 1 [this file] , Methods [rid=S15] , and Supplementary Note [rid=SD3] ). \n",
      "As expected, no difference was observed for WBGs between prenatal and postnatal stages ( = 0.27, Supplementary Fig. 2 [this file] ). \n",
      "Interestingly, in GTEx data, we observed that nearest non-HRG genes are highly expressed in a majority of brain tissues while non-nearest HRGs are not ( Supplementary Fig. 3 [this file] ). \n",
      "Specifically, and both encoding voltage-gated calcium channel subunits, are involved in CCS and contribute to risk for SCZ . is differentially expressed in SCZ patients ( = 0.03), and both and capture multiple Hi-C links in brain Hi-C data ( Supplementary Fig. 4 [this file] ), contributing to the high PP of both genes in iRIGS. We also predicted two DNA binding proteins, SOX2 and SATB2 ( Supplementary Fig. 4 [this file] ), which have important roles in neurogenesis and have been widely implicated in SCZ. Several of our predicted HRGs are miR-137 target genes, including the aforementioned , and three other genes, and . , a glutamate-gated ion channel protein and a key mediator of synaptic plasticity , has a pdDNM and multiple regulatory connections ( Supplementary Fig. 4 [this file] ). , encoding Transcription Factor 4, participates in the initiation of neuronal differentiation by regulating the intrinsic excitability of prefrontal cortical neurons , and knockdown of alters the expression of genes important for developing prefrontal neocortex . is also linked with numerous DREs ( Supplementary Fig. 4 [this file] ) in iRIGS. \n",
      "We took a closer look at this region to gain more insights. captures 55 Hi-C links in brain Hi-C data ( Supplementary Fig. 5 [this file] ), while there is only one for . In GTEx data, is highly expressed in basal ganglia caudate, hypothalamus, basal ganglia nucleus accumbens, basal ganglia putamen and substantia nigra, but the expression in cortex and frontal cortex is rather low ( Supplementary Fig. 6 [this file] ). is uniformly and highly expressed in all brain tissues ( Supplementary Fig. 6 [this file] ). In BrainSpan data, shows constitutively high expression across all stages, and particularly higher expression at prenatal stages with a trajectory that peaks at the early-mid fetal stage, while shows lower expression across all developmental stages and no obvious pattern of transition between prenatal and postnatal stages ( Supplementary Fig. 7 [this file] ). \n",
      "One pdDNM and a high number of regulatory links were observed for ( Supplementary Fig. 8 [this file] ), promoting it as the top gene predicted by iRIGS. \n",
      "We curated a list of 2263 confirmed druggable targets from multiple sources ( Methods [rid=S15] ), and found that 28 (27%) HRGs are targets of 198 FDA-approved, clinically investigational, or preclinical drugs ( Supplementary Fig. 9 [this file] and Supplementary Table 7 [rid=SD2] ). In particular, we found that 5 HRGs ( and ) are targets of nervous system drugs ( Supplementary Fig. 9 [this file] ), corresponding to a statistically significant enrichment ( = 0.01, OR = 3.78). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref SD2: Specifically, we collected 278 gene sets relevant to CNS and behavior/neurological phenotypes from Mouse Genome Informatics (MGI) Mammalian Phenotype Ontology (MPO) database ( Methods [rid=S15] ) and observed significant enrichment of HRGs in 33 gene sets after Bonferroni correction ( Supplementary Table 4 [this file] ). \n",
      "In addition, we also observed that the 65 non-nearest HRGs are significantly enriched in 19 MPO gene sets ( Supplementary Table 5 [this file] ), while no significantly enriched gene sets were observed for the 65 nearest non-HRG genes ( Supplementary Table 6 [this file] ), providing strong and orthogonal support for iRIGS identified risk genes that are beyond the proximity to the GWAS index SNPs. \n",
      "We curated a list of 2263 confirmed druggable targets from multiple sources ( Methods [rid=S15] ), and found that 28 (27%) HRGs are targets of 198 FDA-approved, clinically investigational, or preclinical drugs ( Supplementary Fig. 9 [rid=SD1] and Supplementary Table 7 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref SD3: Details of iRIGS can be found in Methods [rid=S15] and Supplementary Note [this file] . \n",
      "We performed iRIGS on these loci to identify risk genes of SCZ ( Supplementary Table 1 [this file] ). \n",
      "We also tried other thresholds to define LBGs, and the enrichment patterns are similar ( Supplementary Table 2 [this file] ). \n",
      "We found that NRGs are indeed connected to more DREs in Hi-C and FANTOM5 data ( Fig. 2b [rid=F2] , Supplementary Fig. 1 [rid=SD1] , Methods [rid=S15] , and Supplementary Note [this file] ). \n",
      "We therefore integrated them into iRIGS ( Methods [rid=S15] ) and predicted 104 high-confident risk genes (HRGs) in total ( Supplementary Table 1 [this file] ). \n",
      "We found that non-nearest HRGs are more significantly enriched in the gene sets compared to nearest non-HRG genes ( Supplementary Table 3 [this file] ), suggesting that the 65 non-nearest HRGs identified by iRIGS are more likely to be true risk genes than their nearest counterparts. \n",
      "We highlighted these genes with extended supporting evidence and description in Supplementary Table 1 [this file] . \n",
      "The distance of or to in the network is calculated using the random walk with restart (RWR) algorithm ( Supplemental Note [this file] ). We employed the Mahalanobis transformation to de-correlate the integrated multi-dimensional data so that any supportive genomic features can be properly incorporated ( Supplementary Note [this file] ). \n",
      "We adopted the Jensen-Shannon divergence to measure the tissue-specificity of each gene in each tissue ( Supplementary Note [this file] ). \n",
      "All the data used in this study are from public resources that were specified in Methods [rid=S15] and Supplementary Note [this file] . \n",
      "See main text and Supplementary Note [this file] for details. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref SD4: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 243 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 47711\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 164921\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 48433 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-42539cb8407c37c8a861d5a02b05132e7092690e72fcf6e139e698a3503040bd\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 48433 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\n",
      "  \"datasets\": [\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"https://www.synapse.org/#!Synapse:syn5609493\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"https://www.gtexportal.org/home/datasets\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"http://help.brain-map.org/display/devhumanbrain/Documentation\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"https://www.vumc.org/cgg\"\n",
      "    }\n",
      "  ]\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn5609493'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.gtexportal.org/home/datasets'}, {'dataset_identifier': 'n/a', 'repository_reference': 'http://help.brain-map.org/display/devhumanbrain/Documentation'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.vumc.org/cgg'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 5 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn5609493'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.gtexportal.org/home/datasets'}, {'dataset_identifier': 'n/a', 'repository_reference': 'http://help.brain-map.org/display/devhumanbrain/Documentation'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.vumc.org/cgg'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 677 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 605 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn5609493'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn5609493\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn5609493, None, https://www.synapse.org/#!Synapse:syn5609493\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn5609493\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn5609493, identifier: syn5609493\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn5609493\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn5609493 for https://www.synapse.org/#!Synapse:syn5609493\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn5609493 for https://www.synapse.org/#!Synapse:syn5609493\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn5609493 for https://www.synapse.org/#!Synapse:syn5609493\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn5609493 for https://www.synapse.org/#!Synapse:syn5609493\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn5609493 in Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn5609493', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.gtexportal.org/home/datasets'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.gtexportal.org/home/datasets, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.gtexportal.org/home/datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://www.gtexportal.org/home/datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'www.gtexportal.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'http://help.brain-map.org/display/devhumanbrain/Documentation'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: http://help.brain-map.org/display/devhumanbrain/Documentation, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: http://help.brain-map.org/display/devhumanbrain/Documentation\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: http://help.brain-map.org/display/devhumanbrain/Documentation\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'help.brain-map.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.vumc.org/cgg'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.vumc.org/cgg, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.vumc.org/cgg\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://www.vumc.org/cgg\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'www.vumc.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 256 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn5609493', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'www.gtexportal.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'help.brain-map.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'www.vumc.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 4 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn5609493\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn5609493\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn5609493 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn5609493 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn5609493 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn5609493 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn5609493 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn5609493\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn5609493\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Skipping dataset 2: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Skipping dataset 3: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Skipping dataset 4: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 4\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 265 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 8, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 477 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 17th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10387117)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10387117\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC10387117&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 406 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC10387117.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC10387117 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 430 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC10387117, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 443 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 264 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 454 - INFO - Found 9 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b090d840>, <Element supplementary-material at 0x1b72cc740>, <Element supplementary-material at 0x1b72cf540>, <Element supplementary-material at 0x1b72cd800>, <Element supplementary-material at 0x1b72cca00>, <Element supplementary-material at 0x1b60ad580>, <Element supplementary-material at 0x1b04ac900>, <Element supplementary-material at 0x1b7287500>, <Element supplementary-material at 0x1b72846c0>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 462 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b090d840>, <Element supplementary-material at 0x1b72cc740>, <Element supplementary-material at 0x1b72cf540>, <Element supplementary-material at 0x1b72cd800>, <Element supplementary-material at 0x1b72cca00>, <Element supplementary-material at 0x1b60ad580>, <Element supplementary-material at 0x1b04ac900>, <Element supplementary-material at 0x1b7287500>, <Element supplementary-material at 0x1b72846c0>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 548 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM1: MNN outperforms RPCA and Harmony on both the UMAP inspection and cluster stability metrics measurement, we therefore set the mutual nearest neighbor (MNN) matching method in Monocle 3 as default, and Seurat’s reciprocal principal component analysis (RPCA) based integration and Harmony as alternatives (Supplementary Fig. 1 [this file] ). \n",
      "Cells with consistent cell type annotations were combined with the CellRef Seed to form the LungMAP Human Lung CellRef (347,970 cells) (Fig. 2B [rid=Fig2] , Supplementary Figs. 1 [this file] – 3 [this file] , “Methods”). \n",
      "Using the same approach, we constructed a cell type dictionary based on the LungMAP CellCards to define cell types in mouse lung during perinatal development, identified seed cells for each cell type (termed LungMAP Mouse Lung Development CellRef Seed”), and constructed a CellRef for mouse lung development (denoted as LungMAP Mouse Lung Development CellRef) using Drop-seq of mouse lungs ( = 95,658, 8 time points, 17 samples) from embryonic day 16.5 to postnatal day 28 (Fig. 3 [rid=Fig3] , Supplementary Fig. 4 [this file] , Supplementary Data 3 [rid=MOESM5] , 4 [rid=MOESM6] ). \n",
      "These tools provide highly interactive search, analyzing, and visualization functionalities for users to explore and reanalyze cell type and gene expression patterns provided by the LungMAP CellRefs (Fig. 4 [rid=Fig4] , Supplementary Fig. 6 [this file] ). Users can also select and highlight cells of interest (Supplementary Fig. 6 [this file] ). \n",
      "Evaluation of the other three test data samples were shown in Supplementary Fig. 7 [this file] . \n",
      "As shown in Fig. 5D–I [rid=Fig5] and Supplementary Fig. 7 [this file] , predicted cell types were well separated and formed clusters. Cell-type-specific marker genes from CellCards were selectively expressed in each predicted cell type, supporting the concordance of the cell identities (Fig. 5E, F [rid=Fig5] , Supplementary Fig. 7 [this file] ). The median values of all metrics are greater than 0.92 (Supplementary Fig. 8 [this file] ), supporting the high consistency of the automated CellRef cell type prediction with the original cell annotations. Both the LungMAP human and mouse lung CellRefs achieved high cross validation performance with ~0.92 median F1 and MCC scores for the human lung CellRef and 0.98 median F1 and MCC scores for the mouse lung CellRef, respectively (Supplementary Fig. 9 [this file] ). \n",
      "We summarized the one-to-one mapped cell types and the unique cell types/states in the Supplementary Fig. 10 [this file] . \n",
      "Similarly, when applying these stability metrics to CellRef and HLCA using their own data collections, CellRef produced slightly higher scores on SCCAF and TF-IDF; as well as a significantly higher score on cell re-assignment ( = 0.011) as compared to HLCA (Supplementary Fig. 11 [this file] ). scTriangulate leverages the computed stability scores to further assess the marginal importance of each annotation (CellRef and HLCA) using a cooperative game theory framework. scTriangulate predicted that 50% ( = 24/48) of the CellRef cell types to be of higher confidence (AEC, AF1, AF2, AM, ASMC, CD8 T, cDC1, cDC2, Ciliated, Goblet, Ionocyte, LEC, Platelet, pMON, Suprabasal, SVEC, VSMC), compared to 46% ( = 27/58) of the HCLA cell types (AT2, aCAP, Pericyte, Mesothelium, Peribronchial fibroblasts and Adventitial fibroblasts, etc.) (Fig. 8D [rid=Fig8] ). \n",
      "The overall distributions of iLISI scores in both CellRef and HLCA are not ideal, on average about 15% for CellRef, and 5% for HLCA, respectively (Supplementary Fig. 11 [this file] ), likely due to that both atlases were constructed using large collections of heterogeneous data from different biological regions and conditions rather than a well-designed balanced cohort. Nevertheless, CellRef consistently outperforms HLCA on the measurements of iLISI, and has the cLISI scores close to 1, indicating a well separation of the CellRef cell types (Supplementary Fig. 11 [this file] ). \n",
      "Results supported the high accuracy, cluster stability, and marker specificity of our mouse lung reference cell types (Supplementary Fig. 12 [this file] ; mean AUC 95.8, mean SCCAF score: 0.970, mean Reassign score: 0.881, mean TFID10 score: 0.892). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM2: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM3: This collection contains data from similar numbers of female and male donors ( = 48 and 55, respectively; 1 unannotated) (Fig. 1A [rid=Fig1] ; Supplementary Data 1 [this file] ). In total, 505,256 lung cells from 148 sc/snRNA-seq of normal human lung samples from 104 donors were used for LungMAP human lung CellRef construction (Supplementary Data 1 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM4: Using this guided approach and a cell type dictionary derived from LungMAP CellCards (Supplementary Data 2 [this file] ), we identified 8,080 seed cells representing 48 normal human lung cell types, termed LungMAP Human Lung CellRef Seed (Fig. 2A [rid=Fig2] ). \n",
      "Evaluation of cell type annotations using CellRef cell type markers from Supplementary Data 2 [this file] . Percentages of cell type markers (Supplementary Data 2 [this file] ) that are differentially expressed in their corresponding cell type predictions ( = 34 cell types) in ( ). \n",
      "The human lung CellRef cell type selective markers were generated by including dictionary marker genes (Supplementary Data 2 [this file] ) and top selectively expressed markers for each cell type. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM5: Using the same approach, we constructed a cell type dictionary based on the LungMAP CellCards to define cell types in mouse lung during perinatal development, identified seed cells for each cell type (termed LungMAP Mouse Lung Development CellRef Seed”), and constructed a CellRef for mouse lung development (denoted as LungMAP Mouse Lung Development CellRef) using Drop-seq of mouse lungs ( = 95,658, 8 time points, 17 samples) from embryonic day 16.5 to postnatal day 28 (Fig. 3 [rid=Fig3] , Supplementary Fig. 4 [rid=MOESM1] , Supplementary Data 3 [this file] , 4 [rid=MOESM6] ). \n",
      "In total, gene expression in 17 Drop-seq samples from eight time points (Supplementary Data 3 [this file] ) of mouse lung development were generated. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM6: Using the same approach, we constructed a cell type dictionary based on the LungMAP CellCards to define cell types in mouse lung during perinatal development, identified seed cells for each cell type (termed LungMAP Mouse Lung Development CellRef Seed”), and constructed a CellRef for mouse lung development (denoted as LungMAP Mouse Lung Development CellRef) using Drop-seq of mouse lungs ( = 95,658, 8 time points, 17 samples) from embryonic day 16.5 to postnatal day 28 (Fig. 3 [rid=Fig3] , Supplementary Fig. 4 [rid=MOESM1] , Supplementary Data 3 [rid=MOESM5] , 4 [this file] ). In total, 40 mouse lung cell types have been identified with the guidance of the mouse lung cell type dictionary (Fig. 3 [rid=Fig3] , Supplementary Data 4 [this file] ). \n",
      "We constructed a cell type dictionary for mouse lung (Supplementary Data 4 [this file] ) based on our constructed dictionary derived from the LungMAP CellCards. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM7: AUCs values for each of the mapped cell types ( = 42) in the CellRef (orange) and HLCA (blue) calculated using the cell type selective marker genes identified by CellRef (Supplementary Data 5 [this file] ). \n",
      "Next, we performed the same analysis using the cell type marker genes identified by CellRef (Supplementary Data 5 [this file] ). \n",
      "Top-ranked cell type specific DEGs (ranked by fold change in average expression) were combined with the known markers genes to form the CellRef cell type selective marker gene list (Supplementary Data 5 [this file] , up to 10 genes for each of the 48 CellRef human lung cell types). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM8: Last, we evaluated the LungMAP mouse lung CellRef using the three stability metrics calculated by scTriangulate and the marker-based AUC analysis (“Methods” and Supplementary Data 6 [this file] ). \n",
      "Using the same approach, the mouse lung CellRef cell type selective marker genes were selected (Supplementary Data 6 [this file] , up to 10 marker genes per cell type) and used for the AUC based assessment of mouse lung CellRef. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM9: Further information on research design is available in the Nature Portfolio Reporting Summary [this file] linked to this article. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 243 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 70052\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 247377\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 70774 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-a25caeb9425ad6209f0a26c73b867ba557fca278292761417b0aaac762e85262\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 70774 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\"datasets\":[{\"dataset_identifier\":\"GSE135893\",\"repository_reference\":\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE135893\"},{\"dataset_identifier\":\"GSE136831\",\"repository_reference\":\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE136831\"},{\"dataset_identifier\":\"GSE122960\",\"repository_reference\":\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122960\"},{\"dataset_identifier\":\"GSE134174\",\"repository_reference\":\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE134174\"},{\"dataset_identifier\":\"GSE161382\",\"repository_reference\":\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE161382\"},{\"dataset_identifier\":\"GSE171524\",\"repository_reference\":\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE171524\"},{\"dataset_identifier\":\"EGAS00001004082\",\"repository_reference\":\"https://ega-archive.org/studies/EGAS00001004082\"},{\"dataset_identifier\":\"syn21041850\",\"repository_reference\":\"https://www.synapse.org/#!Synapse:syn21041850\"},{\"dataset_identifier\":\"LMEX0000004396\",\"repository_reference\":\"https://lungmap.net/breath-omics-experiment-page/?experiment_id=LMEX0000004396\"},{\"dataset_identifier\":\"LMEX0000004397\",\"repository_reference\":\"https://lungmap.net/breath-omics-experiment-page/?experiment_id=LMEX0000004397\"},{\"dataset_identifier\":\"GSE122332\",\"repository_reference\":\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122332\"},{\"dataset_identifier\":\"GSE178362\",\"repository_reference\":\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178362\"}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'GSE135893', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE135893'}, {'dataset_identifier': 'GSE136831', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE136831'}, {'dataset_identifier': 'GSE122960', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122960'}, {'dataset_identifier': 'GSE134174', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE134174'}, {'dataset_identifier': 'GSE161382', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE161382'}, {'dataset_identifier': 'GSE171524', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE171524'}, {'dataset_identifier': 'EGAS00001004082', 'repository_reference': 'https://ega-archive.org/studies/EGAS00001004082'}, {'dataset_identifier': 'syn21041850', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn21041850'}, {'dataset_identifier': 'LMEX0000004396', 'repository_reference': 'https://lungmap.net/breath-omics-experiment-page/?experiment_id=LMEX0000004396'}, {'dataset_identifier': 'LMEX0000004397', 'repository_reference': 'https://lungmap.net/breath-omics-experiment-page/?experiment_id=LMEX0000004397'}, {'dataset_identifier': 'GSE122332', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122332'}, {'dataset_identifier': 'GSE178362', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178362'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 12 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE135893\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE136831\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE122960\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE134174\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE161382\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE171524\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: EGAS00001004082\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn21041850\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: LMEX0000004396\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: LMEX0000004397\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE122332\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE178362\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'GSE135893', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE135893', 'dataset_id': 'GSE135893'}, {'dataset_identifier': 'GSE136831', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE136831', 'dataset_id': 'GSE136831'}, {'dataset_identifier': 'GSE122960', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122960', 'dataset_id': 'GSE122960'}, {'dataset_identifier': 'GSE134174', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE134174', 'dataset_id': 'GSE134174'}, {'dataset_identifier': 'GSE161382', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE161382', 'dataset_id': 'GSE161382'}, {'dataset_identifier': 'GSE171524', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE171524', 'dataset_id': 'GSE171524'}, {'dataset_identifier': 'EGAS00001004082', 'repository_reference': 'https://ega-archive.org/studies/EGAS00001004082', 'dataset_id': 'EGAS00001004082'}, {'dataset_identifier': 'syn21041850', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn21041850', 'dataset_id': 'syn21041850'}, {'dataset_identifier': 'LMEX0000004396', 'repository_reference': 'https://lungmap.net/breath-omics-experiment-page/?experiment_id=LMEX0000004396', 'dataset_id': 'LMEX0000004396'}, {'dataset_identifier': 'LMEX0000004397', 'repository_reference': 'https://lungmap.net/breath-omics-experiment-page/?experiment_id=LMEX0000004397', 'dataset_id': 'LMEX0000004397'}, {'dataset_identifier': 'GSE122332', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122332', 'dataset_id': 'GSE122332'}, {'dataset_identifier': 'GSE178362', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178362', 'dataset_id': 'GSE178362'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE135893', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE135893', 'dataset_id': 'GSE135893'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE135893\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE135893, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE135893\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE135893\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE135893', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE136831', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE136831', 'dataset_id': 'GSE136831'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE136831\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE136831, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE136831\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE136831\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE136831', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE122960', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122960', 'dataset_id': 'GSE122960'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE122960\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE122960, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122960\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE122960\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE122960', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE134174', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE134174', 'dataset_id': 'GSE134174'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE134174\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE134174, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE134174\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE134174\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE134174', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE161382', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE161382', 'dataset_id': 'GSE161382'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE161382\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE161382, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE161382\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE161382\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE161382', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE171524', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE171524', 'dataset_id': 'GSE171524'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE171524\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE171524, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE171524\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE171524\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE171524', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'EGAS00001004082', 'repository_reference': 'https://ega-archive.org/studies/EGAS00001004082', 'dataset_id': 'EGAS00001004082'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: EGAS00001004082\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID EGAS00001004082 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://ega-archive.org/studies/EGAS00001004082, identifier: EGAS00001004082\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://ega-archive.org/studies/EGAS00001004082\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier EGAS00001004082 for https://ega-archive.org/studies/EGAS00001004082\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier EGAS00001004082 for https://ega-archive.org/studies/EGAS00001004082\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier EGAS00001004082 for https://ega-archive.org/studies/EGAS00001004082\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier EGAS00001004082 for https://ega-archive.org/studies/EGAS00001004082\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier EGAS00001004082 for https://ega-archive.org/studies/EGAS00001004082\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://ega-archive.org/studies/EGAS00001004082\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'EGAS00001004082', 'data_repository': 'ega-archive.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn21041850', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn21041850', 'dataset_id': 'syn21041850'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn21041850\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn21041850, synapse.org, https://www.synapse.org/#!Synapse:syn21041850\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn21041850\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn21041850', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'LMEX0000004396', 'repository_reference': 'https://lungmap.net/breath-omics-experiment-page/?experiment_id=LMEX0000004396', 'dataset_id': 'LMEX0000004396'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: LMEX0000004396\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID LMEX0000004396 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://lungmap.net/breath-omics-experiment-page/?experiment_id=LMEX0000004396, identifier: LMEX0000004396\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://lungmap.net/breath-omics-experiment-page/?experiment_id=LMEX0000004396\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier LMEX0000004396 for https://lungmap.net/breath-omics-experiment-page/?experiment_id=LMEX0000004396\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier LMEX0000004396 for https://lungmap.net/breath-omics-experiment-page/?experiment_id=LMEX0000004396\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier LMEX0000004396 for https://lungmap.net/breath-omics-experiment-page/?experiment_id=LMEX0000004396\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier LMEX0000004396 for https://lungmap.net/breath-omics-experiment-page/?experiment_id=LMEX0000004396\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier LMEX0000004396 for https://lungmap.net/breath-omics-experiment-page/?experiment_id=LMEX0000004396\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://lungmap.net/breath-omics-experiment-page/?experiment_id=LMEX0000004396\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'LMEX0000004396', 'data_repository': 'lungmap.net', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'LMEX0000004397', 'repository_reference': 'https://lungmap.net/breath-omics-experiment-page/?experiment_id=LMEX0000004397', 'dataset_id': 'LMEX0000004397'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: LMEX0000004397\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID LMEX0000004397 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://lungmap.net/breath-omics-experiment-page/?experiment_id=LMEX0000004397, identifier: LMEX0000004397\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://lungmap.net/breath-omics-experiment-page/?experiment_id=LMEX0000004397\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier LMEX0000004397 for https://lungmap.net/breath-omics-experiment-page/?experiment_id=LMEX0000004397\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier LMEX0000004397 for https://lungmap.net/breath-omics-experiment-page/?experiment_id=LMEX0000004397\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier LMEX0000004397 for https://lungmap.net/breath-omics-experiment-page/?experiment_id=LMEX0000004397\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier LMEX0000004397 for https://lungmap.net/breath-omics-experiment-page/?experiment_id=LMEX0000004397\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier LMEX0000004397 for https://lungmap.net/breath-omics-experiment-page/?experiment_id=LMEX0000004397\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://lungmap.net/breath-omics-experiment-page/?experiment_id=LMEX0000004397\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'LMEX0000004397', 'data_repository': 'lungmap.net', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE122332', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122332', 'dataset_id': 'GSE122332'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE122332\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE122332, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122332\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE122332\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE122332', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE178362', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178362', 'dataset_id': 'GSE178362'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE178362\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE178362, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178362\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE178362\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE178362', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 256 - INFO - Augmented dataset links: [{'dataset_identifier': 'GSE135893', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE136831', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE122960', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE134174', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE161382', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE171524', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'EGAS00001004082', 'data_repository': 'ega-archive.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn21041850', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'LMEX0000004396', 'data_repository': 'lungmap.net', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'LMEX0000004397', 'data_repository': 'lungmap.net', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE122332', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE178362', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 12 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE135893\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE135893\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE135893 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE135893 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE135893 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE135893\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE135893\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE136831\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE136831\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE136831 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE136831 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE136831 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 2 with repo: geo and accession_id: GSE136831\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE136831\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 2: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE122960\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE122960\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE122960 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE122960 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE122960 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 3 with repo: geo and accession_id: GSE122960\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122960\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 3: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE134174\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE134174\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE134174 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE134174 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE134174 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 4 with repo: geo and accession_id: GSE134174\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE134174\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 4: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE161382\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE161382\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE161382 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE161382 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE161382 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 5 with repo: geo and accession_id: GSE161382\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE161382\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 5: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE171524\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE171524\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE171524 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE171524 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE171524 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 6 with repo: geo and accession_id: GSE171524\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE171524\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 6: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: EGAS00001004082\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: ega-archive.org, identifier: EGAS00001004082\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: ega-archive.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for EGAS00001004082 in ega-archive.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 7 with repo: ega-archive.org and accession_id: EGAS00001004082\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://ega-archive.org/studies/EGAS00001004082\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 7: Contact to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn21041850\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn21041850\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn21041850 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn21041850 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn21041850 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn21041850 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn21041850 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 8 with repo: synapse.org and accession_id: syn21041850\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn21041850\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 8: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: LMEX0000004396\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: lungmap.net, identifier: LMEX0000004396\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier LMEX0000004396 for lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier LMEX0000004396 for lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier LMEX0000004396 for lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier LMEX0000004396 for lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier LMEX0000004396 for lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 976 - WARNING - URL lungmap.net may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for LMEX0000004396 in lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 9 with repo: lungmap.net and accession_id: LMEX0000004396\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1254 - WARNING - Repository lungmap.net unknown in Ontology. Skipping dataset page 9.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 10 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: LMEX0000004397\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: lungmap.net, identifier: LMEX0000004397\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier LMEX0000004397 for lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier LMEX0000004397 for lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier LMEX0000004397 for lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier LMEX0000004397 for lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier LMEX0000004397 for lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 976 - WARNING - URL lungmap.net may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for LMEX0000004397 in lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 10 with repo: lungmap.net and accession_id: LMEX0000004397\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1254 - WARNING - Repository lungmap.net unknown in Ontology. Skipping dataset page 10.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 11 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE122332\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE122332\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE122332 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE122332 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE122332 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 11 with repo: geo and accession_id: GSE122332\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122332\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 11: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 12 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE178362\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE178362\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE178362 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE178362 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE178362 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 12 with repo: geo and accession_id: GSE178362\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178362\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 12: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 12\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 265 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 21, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 477 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 18th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC4444046)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC4444046\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC4444046&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 394 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC4444046.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b00552d0>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC4444046 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC4444046, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 445 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC4444046)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 547 - INFO - Extracted 11 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 557 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#S7'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for S7: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 51 <a> elements with href='#SD1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 9833) for SD1: We assembled a TCGA Pan-Cancer dataset of exome sequencing, array copy number, and RNA-seq data from 3,281 samples from 12 cancer types, analyzing single nucleotide variants (SNVs), small indels, and copy number aberrations (CNAs) in 19,424 transcripts ( Figure 1a [href=#F1] and Supplementary Figure 1 [href=this file] ). The resulting “star subnetworks” centered on the hot node ( Supplementary Figure 2 [href=this file] ; Online Methods) contain many neighboring genes that are not mutated at appreciable frequency and are of limited biological interest.\n",
      "We introduce the HotNet2 (HotNet iffusion riented ubnetworks) algorithm to address the problem of finding significantly mutated subnetworks on large, broad mutation frequency spectrum datasets like Pan-Cancer ( Figure 1c [href=#F1] and Supplementary Figure 3 [href=this file] ). HotNet2 uses a modified diffusion process and considers the source, or directionality, of heat flow in the identification of subnetworks ( Supplementary Figure 4 [href=this file] ).\n",
      "We combined the resulting subnetworks into 14 consensus subnetworks that were found across different gene scores and networks ( < 0.004, Supplementary Table 3 [href=#SD2] ), plus the condensin complex and CLASP/CLIP proteins ( Supplementary Figure 5 [href=this file] ) that were significant in individual interaction networks ( Supplementary Tables 6,7 [href=#SD2] ). We developed an online interactive viewer (see URLs and Supplementary Figure 6 [href=this file] ) for Pan-Cancer HotNet2 subnetworks.\n",
      "The subnetworks and linker genes ( Figure 2a [href=#F2] ) include: portions of well-known cancer pathways such as TP53, PI3K, NOTCH, and receptor tyrosine kinases (RTKs; Supplementary Figure 7 [href=this file] ), as well as pathways and complexes that have more recently been observed to be important in cancer such as SWI/SNF complex, BAP1 complex, NFE2L2-KEAP1 ( Supplementary Figures 8,9 [href=this file] ), and RUNX1-CBFB core binding complex ( Supplementary Figure 10 [href=this file] ). The fifth most mutated subnetwork (16.9% of samples) consists of MLL2 and MLL3 and the putative interacting protein KDM6A ( Supplementary Figure 11 [href=this file] ), and was highly mutated (28.9% of samples) in TCGA Pan-Cancer squamous integrated subtype . The MHC Class I subnetwork ( Supplementary Figure 12 [href=this file] ) is an example of the ability of HotNet2 ability to identify rarely mutated cancer genes; all of the genes in the subnetwork are mutated in fewer than 35 samples (1.1%), yet four of the five genes have recently been proposed as novel cancer genes . Additional analyses are in the Supplementary Note [href=this file] .\n",
      "To obtain additional support for these genes we examined whether they had either an excess of inactivating mutations or a cluster of missense mutations in protein sequence (using NMC ) or in protein structure (using iPAC ; Supplementary Figures 13,14 [href=this file] and Supplementary Tables 21,22 [href=#SD2] ). We find that genes in HotNet2 consensus subnetworks are enriched for inactivating mutations ( < 0.0001) or mutation clusters ( < 0.0001) compared to genes not in subnetworks ( Supplementary Table 6–18 [href=#SD2] and Supplementary Note Section 5.1 [href=this file] ). Finally, we evaluated a subset of the mutations in these genes using RNA-Seq and whole-genome sequencing (WGS) data from the same samples, and found RNA-Seq and/or WGS reads that validated 39 mutations in these novel genes ( Supplementary Note Section 6 [href=this file] and Supplementary Table 23 [href=#SD2] ).\n",
      "This exclusivity was largely due to LUAD samples with mutually exclusive and mutations ( Supplementary Figure 15 [href=this file] ).\n",
      "The three largest subnetworks – including a subnetwork, a subnetwork, and a subnetwork – contain many well-known cancer genes ( Supplementary Tables 8–10 [href=#SD2] and Supplementary Figures 16,17 [href=this file] ).\n",
      "The subnetwork also includes , which has a significant ( < 8×10 ) cluster of missense mutations ( Supplementary Figure 18 [href=this file] ).\n",
      "We find that mutations in the SWI/SNF and BAP1 complexes show even greater mutual exclusivity ( =9.4×10 ) in KIRC because of mutations in additional genes in these complexes besides and , respectively ( Supplementary Note Section 5.8.1 [href=this file] ).\n",
      "Finally, RNA-seq and whole-genome sequencing data from the same samples provide further validation of the somatic mutations in , , , , , and and show that some of these mutations are expressed ( Supplementary Note Section 6 [href=this file] and Supplementary Table 39 [href=#SD2] ).\n",
      "Our analysis complements other recent Pan-Cancer analyses including studies that analyze only one type of aberration or restrict attention to recurrent aberrations ( Supplementary Note Section 8.3 [href=this file] and Supplementary Table 27 [href=#SD2] ).\n",
      "We selected the threshold of 400 aberrations per sample as the derivative of the number of mutations per sample starts increasing rapidly beyond this value ( Supplementary Figure 19 [href=this file] ). Finally, we used only those genes that had at least 3 reads from RNA-seq data in at least 70% of samples of at least one of the cancer types, as described in The resulting dataset contained aberrations in 11,565 genes and 3110 samples ( Supplementary Figure 1 [href=this file] ). Additional details of the datasets are in the Supplementary Note [href=this file] .\n",
      "While interaction networks have proven useful in analyzing various types of genomic data , statistically robust identification of significantly mutated subnetworks is a difficult problem with several major challenges ( Supplemental Note Section 1.1 [href=this file] ). HotNet2 addresses these challenges and identifies significantly mutated subnetworks of a genome-scale interaction network, using an insulated heat diffusion process that considers both the scores on individual genes/proteins as well as the topology of interactions between genes/proteins ( Supplementary Figure 3 [href=this file] ).\n",
      "The insulated heat model can also described in terms of a random walk with restart ( Supplementary Note Section 1.2 [href=this file] ).\n",
      "Since the diffusion matrix is not symmetric and in general ( ) ≠ ( ), the similarity ( , ) is also not symmetric ( Supplementary Note Section 1.2.1 [href=this file] ).\n",
      "The statistical test is the same as the two-stage statistical test introduced in the original HotNet algorithm ( Supplementary Note Section 1.3 [href=this file] , Supplementary Figures 20–23 [href=this file] and Supplementary Table 28 [href=#SD2] ).\n",
      "HotNet2 has two parameters β and δ, and selects values for both of these parameters using procedures. β is selected from the protein-protein interaction network, of any gene scores ( Supplementary Note Section 1.4.1 [href=this file] , Supplementary Figure 24 [href=this file] , and Supplementary Table 29 [href=#SD2] ). The value of δ is chosen such that large connected components are not found using the observed gene score distribution on random networks with the same degree distribution as the observed network ( Supplementary Note Section 1.4.2 [href=this file] , Supplementary Figure 25 [href=this file] , and Supplementary Table 30 [href=#SD2] ).\n",
      "See Supplementary Figure 2 [href=this file] .\n",
      "Expanding on this third point, when undirected diffusion algorithms like HotNet or related network propagation algorithms are run on large datasets containing a wide range of gene scores (e.g. the Pan-Cancer dataset), many of the resulting subnetworks are “hot” determined by a single high-scoring node and the immediate neighbors of this node ( Supplementary Figure 2 [href=this file] ).\n",
      "The HotNet2 subnetworks also have a higher fraction of interactions with proteins other than a hot central node ( Supplementary Note Section 7.1 [href=this file] ). In short, the results show that HotNet2 achieves higher sensitivity and specificity than HotNet ( Supplementary Note Section 7.2 [href=this file] and Supplementary Figure 26 [href=this file] ).\n",
      "We find that HotNet2 provides both new insights and a simpler summary of groups of interacting genes, and is a useful complement (or arguably a replacement for) other pathway tests ( Supplementary Note Section 8.1 [href=this file] ). We also show that HotNet2 has much higher specificity than HotNet, DAVID, and GSEA in identifying genes satisfying the 20/20 rule ( Supplementary Note Section 8.1.4 [href=this file] , Supplementary Figure 27 [href=this file] , and Supplementary Tables 34–36 [href=#SD2] ). Finally, we find that HotNet2 was more stable than HotNet in identifying 20/20 genes using cross-validation ( Supplementary Note Section 7.3 [href=this file] and Supplementary Figure 28 [href=this file] ).\n",
      "We found that MEMo was unable to run on the Pan-Cancer mutation frequency dataset, consistent with the authors’ recommendation that MEMo should be run only on a small number of significant mutations (details in Supplementary Note Section 8.2 [href=this file] ).\n",
      "We ran HotNet2 on each combination of gene scores (mutation frequency and MutSigCV -values; see Supplementary Note Section 2.2 [href=this file] ) and interaction networks (HINT+HI2012 , iRefIndex , and Multinet ; Supplementary Note Section 2.5 [href=this file] and Supplementary Figure 29 [href=this file] ). This procedure is described in Supplementary Note Section 1.5 [href=this file] .\n",
      "Additional details of the statistical procedure are in Supplementary Note Section 1.3 [href=this file] .\n",
      "Notable among the additional subnetworks identified when we remove the requirement for RNA-Seq expression is a subnetwork ( Supplementary Table 25 [href=#SD2] ) containing members of the telomerase complex (including and ) that has a well-studied role in cancer ( Supplementary Figure 30 [href=this file] and Supplementary Table 38 [href=#SD2] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 34 <a> elements with href='#SD2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 7301) for SD2: HotNet2 identified a significant number of subnetworks ( < 0.01 Supplementary Tables 1–2 [href=this file] ) for each of the two gene scores and three networks. We combined the resulting subnetworks into 14 consensus subnetworks that were found across different gene scores and networks ( < 0.004, Supplementary Table 3 [href=this file] ), plus the condensin complex and CLASP/CLIP proteins ( Supplementary Figure 5 [href=#SD1] ) that were significant in individual interaction networks ( Supplementary Tables 6,7 [href=this file] ).\n",
      "Many of the subnetworks exhibit a significant enrichment for mutations in a subset of cancer types, including many previously unreported associations ( Supplementary Tables 6–18 [href=this file] ). In addition, the HotNet2 Pan-Cancer analysis provides a clearer and more robust summary of subnetworks and novel genes than HotNet2 analysis of individual cancer types ( Supplementary Table 19 [href=this file] ).\n",
      "These subnetworks and linkers include a total of 147 genes, including many well-known cancer genes and pathways, but also including genes with mutations that are too rare to be significant by the single-gene tests ( Supplementary Table 20 [href=this file] ). Table 1 [href=#T1] lists a subset of promising candidates, with the full list and associated references in Supplementary Table 20 [href=this file] .\n",
      "More information on these genes – as well as other candidate driver genes – is in Supplementary Table 20 [href=this file] .\n",
      "To obtain additional support for these genes we examined whether they had either an excess of inactivating mutations or a cluster of missense mutations in protein sequence (using NMC ) or in protein structure (using iPAC ; Supplementary Figures 13,14 [href=#SD1] and Supplementary Tables 21,22 [href=this file] ). We find that genes in HotNet2 consensus subnetworks are enriched for inactivating mutations ( < 0.0001) or mutation clusters ( < 0.0001) compared to genes not in subnetworks ( Supplementary Table 6–18 [href=this file] and Supplementary Note Section 5.1 [href=#SD1] ). Finally, we evaluated a subset of the mutations in these genes using RNA-Seq and whole-genome sequencing (WGS) data from the same samples, and found RNA-Seq and/or WGS reads that validated 39 mutations in these novel genes ( Supplementary Note Section 6 [href=#SD1] and Supplementary Table 23 [href=this file] ).\n",
      "Multiple pairs of genes within these subnetworks show co-occurring mutations ( Supplementary Table 24 [href=this file] ). We observe significant mutual exclusivity within 4 of our subnetworks ( Supplementary Table 25 [href=this file] ).\n",
      "The three largest subnetworks – including a subnetwork, a subnetwork, and a subnetwork – contain many well-known cancer genes ( Supplementary Tables 8–10 [href=this file] and Supplementary Figures 16,17 [href=#SD1] ).\n",
      "HotNet2 Pan-Cancer analyses also revealed a number of novel genes ( Supplementary Table 20 [href=this file] ) within these three subnetworks.\n",
      "The sixth most mutated HotNet2 Pan-Cancer subnetwork (16.8% of samples) includes multiple members of the SWI/SNF chromatin-remodeling complex ( Figure 3a [href=#F3] and Supplementary Table 12 [href=this file] ).\n",
      "Another HotNet2 Pan-Cancer subnetwork (mutated in 7.1% of samples) overlaps the BAP1 complex ( Figure 3b [href=#F3] and Supplementary Table 13 [href=this file] ).\n",
      "HotNet2 Pan-Cancer analysis identifies 4/5 members of the cohesin complex as a significantly mutated subnetwork (7.3% of samples, Figure 4a [href=#F4] and Supplementary Table 15 [href=this file] ).\n",
      "The combined subnetwork is mutated in 4.2% of samples ( Figure 4b [href=#F4] and Supplementary Table 6 [href=this file] ). Finally, RNA-seq and whole-genome sequencing data from the same samples provide further validation of the somatic mutations in , , , , , and and show that some of these mutations are expressed ( Supplementary Note Section 6 [href=#SD1] and Supplementary Table 39 [href=this file] ).\n",
      "Some of these rarely mutated genes are inevitably false positive predictions of the analysis, but many (including , , , , , and others) exhibit either mutational clustering or protein interactions that support a role for the observed somatic aberrations ( Supplementary Tables 6–18 [href=this file] ).\n",
      "Our analysis complements other recent Pan-Cancer analyses including studies that analyze only one type of aberration or restrict attention to recurrent aberrations ( Supplementary Note Section 8.3 [href=#SD1] and Supplementary Table 27 [href=this file] ).\n",
      "The statistical test is the same as the two-stage statistical test introduced in the original HotNet algorithm ( Supplementary Note Section 1.3 [href=#SD1] , Supplementary Figures 20–23 [href=#SD1] and Supplementary Table 28 [href=this file] ).\n",
      "HotNet2 has two parameters β and δ, and selects values for both of these parameters using procedures. β is selected from the protein-protein interaction network, of any gene scores ( Supplementary Note Section 1.4.1 [href=#SD1] , Supplementary Figure 24 [href=#SD1] , and Supplementary Table 29 [href=this file] ). We evaluated the sensitivity of the HotNet2 results to the value of β and found that varying β ±10% has only a minor effect on the results, with at most 7 genes (3.8% of total) added/removed from the subnetworks ( Supplementary Table 28 [href=this file] ). The value of δ is chosen such that large connected components are not found using the observed gene score distribution on random networks with the same degree distribution as the observed network ( Supplementary Note Section 1.4.2 [href=#SD1] , Supplementary Figure 25 [href=#SD1] , and Supplementary Table 30 [href=this file] ). We evaluated the sensitivity of the HotNet2 results to the value of δ, and found that varying δ ±5% changed at most 35 genes (12.3% of total) in the subnetworks ( Supplementary Table 29 [href=this file] ).\n",
      "We found that HotNet2 returns >80% fewer hot stars/spiders than HotNet on the Pan-Cancer datasets ( Supplementary Table 31 [href=this file] ). This is a major difference between the algorithms and is one of the reasons why HotNet fails to find statistically significant results ( ≤ 0.01 for any subnetwork size ) on three of six runs ( Supplementary Table 32,33 [href=this file] ), while HotNet2 finds statistically significant results on all six runs.\n",
      "We also show that HotNet2 has much higher specificity than HotNet, DAVID, and GSEA in identifying genes satisfying the 20/20 rule ( Supplementary Note Section 8.1.4 [href=#SD1] , Supplementary Figure 27 [href=#SD1] , and Supplementary Tables 34–36 [href=this file] ).\n",
      "Most of the subnetworks (12/14) identified by HotNet2 were also found when we remove the requirement for RNA-Seq expression ( Supplementary Table 37 [href=this file] ). Notable among the additional subnetworks identified when we remove the requirement for RNA-Seq expression is a subnetwork ( Supplementary Table 25 [href=this file] ) containing members of the telomerase complex (including and ) that has a well-studied role in cancer ( Supplementary Figure 30 [href=#SD1] and Supplementary Table 38 [href=this file] ). We found only minor differences in the HotNet2 subnetworks ( Supplementary Table 39 [href=this file] ), demonstrating that our reported subnetworks are altered by somatic aberrations in these samples.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#SD3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for SD3: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#fn-group1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 27) for fn-group1: Footnotes [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#fn-group1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 27) for fn-group1: Footnotes [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#fn-group1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 27) for fn-group1: Footnotes [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#fn-group1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 27) for fn-group1: Footnotes [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#fn-group1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 27) for fn-group1: Footnotes [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 56720\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 186016\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 57442 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-1bf51a191ee8695be5f1708943608acf1e8ea17ac528f6b4d308e286b7f8e2b9\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 57442 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\n",
      "  \"datasets\": [\n",
      "    {\n",
      "      \"dataset_identifier\": \"syn1710680\",\n",
      "      \"repository_reference\": \"https://www.synapse.org/#!Synapse:syn1710680\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"syn1729383\",\n",
      "      \"repository_reference\": \"https://www.synapse.org/#!Synapse:syn1729383\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    }\n",
      "  ]\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'syn1710680', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn1710680'}, {'dataset_identifier': 'syn1729383', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn1729383'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 3 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn1710680\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn1729383\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'syn1710680', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn1710680', 'dataset_id': 'syn1710680'}, {'dataset_identifier': 'syn1729383', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn1729383', 'dataset_id': 'syn1729383'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn1710680', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn1710680', 'dataset_id': 'syn1710680'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn1710680\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn1710680, synapse.org, https://www.synapse.org/#!Synapse:syn1710680\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn1710680\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn1710680', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn1729383', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn1729383', 'dataset_id': 'syn1729383'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn1729383\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn1729383, synapse.org, https://www.synapse.org/#!Synapse:syn1729383\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn1729383\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn1729383', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 677 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 605 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn1710680', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn1729383', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 2 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn1710680\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn1710680\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn1710680 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn1710680 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn1710680 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn1710680 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn1710680 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn1710680\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn1710680\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn1729383\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn1729383\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn1729383 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn1729383 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn1729383 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn1729383 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn1729383 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn1729383\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn1729383\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 2\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 13, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 640 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - Parsed data extraction completed. Elements collected: 6\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 481 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 19th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC10329279&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 394 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b5f945d0>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 445 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 547 - INFO - Extracted 1 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 557 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 51472\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 162786\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 52194 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-0e33a86e488e1f963880ff370f8c7aa483a8715b6a45846095ad216b6e394f00\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 52194 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\n",
      "  \"datasets\": [\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    }\n",
      "  ]\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 1 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 677 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 605 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: []\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 0 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 0\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 1, with cols: Index(['link', 'title', 'file_info', 'description', 'source_section',\n",
      "       'section_class', 'download_link', 'file_extension', 'a_attr_href',\n",
      "       'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 640 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - Parsed data extraction completed. Elements collected: 1\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 481 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 20th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11145390)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11145390\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC11145390&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 406 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC11145390.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11145390 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 430 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11145390, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 443 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 264 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 454 - INFO - Found 1 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b7de8cc0>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 462 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b7de8cc0>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 548 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref mmc1: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 243 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 39291\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 149110\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 40013 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-cd52e3990e52563813e7c02a2249d6cf0a4d688af152a1a87753213aff27a83f\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 40013 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\n",
      "  \"datasets\": [\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE147807\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE147807\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"syn21788425\",\n",
      "      \"repository_reference\": \"https://www.synapse.org/#!Synapse:syn21788425/files/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    }\n",
      "  ]\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'GSE147807', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE147807'}, {'dataset_identifier': 'syn21788425', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn21788425/files/'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 3 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE147807\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn21788425\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'GSE147807', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE147807', 'dataset_id': 'GSE147807'}, {'dataset_identifier': 'syn21788425', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn21788425/files/', 'dataset_id': 'syn21788425'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE147807', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE147807', 'dataset_id': 'GSE147807'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE147807\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE147807, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE147807\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE147807\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE147807', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn21788425', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn21788425/files/', 'dataset_id': 'syn21788425'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn21788425\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn21788425, synapse.org, https://www.synapse.org/#!Synapse:syn21788425/files/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn21788425\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn21788425', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 677 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 605 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 256 - INFO - Augmented dataset links: [{'dataset_identifier': 'GSE147807', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn21788425', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 2 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE147807\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE147807\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE147807 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE147807 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE147807 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE147807\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE147807\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn21788425\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn21788425\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn21788425 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn21788425 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn21788425 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn21788425 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn21788425 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn21788425\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn21788425\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 2\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 265 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 3, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 477 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 574 - INFO - \n",
      "Progress: 21/100 (21.00%) | Elapsed: 00:05:20 | ETA: 00:20:05\n",
      "\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 21th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC11032436&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 461 - WARNING - No links found in the section.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 406 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 430 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 443 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 264 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 454 - INFO - Found 1 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b090b300>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 462 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b090b300>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 548 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref No ID: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 243 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 37576\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 137085\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 38298 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-2d46c812fbf0727c36df0345d592b4d91f2bdce72b5b771469a51db0df19ac30\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 38298 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\n",
      "  \"datasets\": [\n",
      "    {\n",
      "      \"dataset_identifier\": \"PMC11032436\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11032436/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"10.1073/pnas.2322924121\",\n",
      "      \"repository_reference\": \"https://doi.org/10.1073/pnas.2322924121\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    }\n",
      "  ]\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'PMC11032436', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11032436/'}, {'dataset_identifier': '10.1073/pnas.2322924121', 'repository_reference': 'https://doi.org/10.1073/pnas.2322924121'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 3 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: PMC11032436\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: 10.1073/pnas.2322924121\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'PMC11032436', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11032436/', 'dataset_id': 'PMC11032436'}, {'dataset_identifier': '10.1073/pnas.2322924121', 'repository_reference': 'https://doi.org/10.1073/pnas.2322924121', 'dataset_id': '10.1073/pnas.2322924121'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'PMC11032436', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11032436/', 'dataset_id': 'PMC11032436'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: PMC11032436\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID PMC11032436 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11032436/, identifier: PMC11032436\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11032436/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier PMC11032436 for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11032436/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier PMC11032436 for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11032436/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier PMC11032436 for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11032436/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier PMC11032436 for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11032436/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier PMC11032436 for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11032436/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11032436/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'PMC11032436', 'data_repository': 'www.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': '10.1073/pnas.2322924121', 'repository_reference': 'https://doi.org/10.1073/pnas.2322924121', 'dataset_id': '10.1073/pnas.2322924121'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(10\\.\\d{4,9}/).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: 10.1073/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: 10.1073/pnas.2322924121, doi.org, https://doi.org/10.1073/pnas.2322924121\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: 10.1073/pnas.2322924121\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': '10.1073/pnas.2322924121', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 677 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 605 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 256 - INFO - Augmented dataset links: [{'dataset_identifier': 'PMC11032436', 'data_repository': 'www.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}, {'dataset_identifier': '10.1073/pnas.2322924121', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 2 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: PMC11032436\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: www.ncbi.nlm.nih.gov, identifier: PMC11032436\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier PMC11032436 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier PMC11032436 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier PMC11032436 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier PMC11032436 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier PMC11032436 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 976 - WARNING - URL www.ncbi.nlm.nih.gov may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for PMC11032436 in www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 1 with repo: www.ncbi.nlm.nih.gov and accession_id: PMC11032436\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1254 - WARNING - Repository www.ncbi.nlm.nih.gov unknown in Ontology. Skipping dataset page 1.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: 10.1073/pnas.2322924121\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: doi.org, identifier: 10.1073/pnas.2322924121\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.1073/pnas.2322924121 for doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for 10.1073/pnas.2322924121 in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 2 with repo: doi.org and accession_id: 10.1073/pnas.2322924121\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://doi.org/10.1073/pnas.2322924121\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 2\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 265 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 3, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage', 'link',\n",
      "       'source_url', 'download_link', 'title', 'content_type', 'id', 'caption',\n",
      "       'description', 'source_section', 'retrieval_pattern',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 477 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 22th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC9984184)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC9984184\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC9984184&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 394 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC9984184.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b7298bd0>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC9984184 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC9984184, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 445 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC9984184)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 547 - INFO - Extracted 17 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 557 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#sec19'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for sec19: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#sec24'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for sec24: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#sec25'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for sec25: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#sec25'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for sec25: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#sec26'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for sec26: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#sec26'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for sec26: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#sec33'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for sec33: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#sec37'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for sec37: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#sec37'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for sec37: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#sec37'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for sec37: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#sec37'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for sec37: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#sec37'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for sec37: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#sec37'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for sec37: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#supplementary-material1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for supplementary-material1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#sec40'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for sec40: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 71229\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 231605\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 71951 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-e9be3456f23554ace0bcf1a852ba8d510e691ab7df77336ad9c050e4220f94d6\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 71951 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\"datasets\":[{\"dataset_identifier\":\"GSE147205\",\"repository_reference\":\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE147205\"},{\"dataset_identifier\":\"GSE169185\",\"repository_reference\":\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE169185\"},{\"dataset_identifier\":\"syn25576878\",\"repository_reference\":\"http://www.synapse.org\"},{\"dataset_identifier\":\"syn39456106\",\"repository_reference\":\"http://www.synapse.org\"},{\"dataset_identifier\":\"n/a\",\"repository_reference\":\"n/a\"}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'GSE147205', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE147205'}, {'dataset_identifier': 'GSE169185', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE169185'}, {'dataset_identifier': 'syn25576878', 'repository_reference': 'http://www.synapse.org'}, {'dataset_identifier': 'syn39456106', 'repository_reference': 'http://www.synapse.org'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 5 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE147205\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE169185\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn25576878\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn39456106\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'GSE147205', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE147205', 'dataset_id': 'GSE147205'}, {'dataset_identifier': 'GSE169185', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE169185', 'dataset_id': 'GSE169185'}, {'dataset_identifier': 'syn25576878', 'repository_reference': 'http://www.synapse.org', 'dataset_id': 'syn25576878'}, {'dataset_identifier': 'syn39456106', 'repository_reference': 'http://www.synapse.org', 'dataset_id': 'syn39456106'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE147205', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE147205', 'dataset_id': 'GSE147205'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE147205\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE147205, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE147205\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE147205\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE147205', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE169185', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE169185', 'dataset_id': 'GSE169185'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE169185\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE169185, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE169185\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE169185\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE169185', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn25576878', 'repository_reference': 'http://www.synapse.org', 'dataset_id': 'syn25576878'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn25576878, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn25576878\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn25576878', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn39456106', 'repository_reference': 'http://www.synapse.org', 'dataset_id': 'syn39456106'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn39456106, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn39456106\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn39456106', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 677 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 605 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'GSE147205', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE169185', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn25576878', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn39456106', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 4 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE147205\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE147205\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE147205 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE147205 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE147205 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE147205\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE147205\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE169185\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE169185\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE169185 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE169185 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE169185 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 2 with repo: geo and accession_id: GSE169185\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE169185\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 2: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn25576878\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn25576878\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn25576878 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn25576878 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn25576878 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn25576878 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn25576878 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 3 with repo: synapse.org and accession_id: syn25576878\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn25576878\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 3: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn39456106\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn39456106\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn39456106 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn39456106 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn39456106 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn39456106 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn39456106 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 4 with repo: synapse.org and accession_id: syn39456106\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn39456106\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 4: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 4\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 21, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 640 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - Parsed data extraction completed. Elements collected: 6\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 481 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 23th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC7347884)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC7347884\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC7347884&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 406 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC7347884.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC7347884 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 430 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC7347884, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 443 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 264 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 454 - INFO - Found 5 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1affd0dc0>, <Element supplementary-material at 0x1b68daf40>, <Element supplementary-material at 0x1b68db040>, <Element supplementary-material at 0x1b68d9140>, <Element supplementary-material at 0x1b68d8300>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 462 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1affd0dc0>, <Element supplementary-material at 0x1b68daf40>, <Element supplementary-material at 0x1b68db040>, <Element supplementary-material at 0x1b68d9140>, <Element supplementary-material at 0x1b68d8300>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 548 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM1: As gene expression analyses do not allow to accurately discriminate mesenchymal cancer cells from normal stromal cells, we used the allele-specific copy number analysis of tumors (ASCAT) copy number-based tumor purity estimation method to assess tumor cell fraction (Supplementary Fig. 1a [this file] ). To avoid any substantial bias due to non-tumor cell contamination, a stringent purity threshold was determined by applying Wilcoxon test, for which the degree of contamination of claudin-low tumors was not statistically different from that of non-claudin-low tumors, thus enabling a comparable distribution of tumors (Supplementary Fig. 1b [this file] ). Demonstrating an unexpected degree of diversity, only 35.6% of these tumors were classified as TNBCs (Supplementary Fig. 1d [this file] ). This result was not due to the purity selection, as the percentage of TNBCs within all claudin-low tumors was similar before applying the purity threshold (Supplementary Fig. 1c [this file] ). \n",
      "Only 17.8% were found in cluster 10, whereas 48.8% and 17.8% were found in integrative clusters 4 and 3, respectively, regardless of tumor purity when compared with non-claudin-low tumors (Supplementary Fig. 2 [this file] ). After having previously removed germline copy number variations (CNVs), we analyzed B allele frequency (BAF) and log R ratio (LRR) profiles of single-nucleotide polymorphisms (SNPs) localized in genomic regions of deletion (LRR segment mean cutoff value < −0.4) (Fig. 1b [rid=Fig1] and Supplementary Fig. 3a, c [this file] ). Consistent with these data, the analysis of somatic mutations highlighted the presence of several point mutations with a variant allele frequency higher than 0.4 (Fig. 1c [rid=Fig1] and Supplementary Fig. 3b [this file] ). \n",
      "Of note, although these analyses were performed on tumors selected for their high tumor purity, similar data were obtained when studying the whole cohort of claudin-low tumors (Supplementary Fig. 4 [this file] ), further strengthening the characterization of the three claudin-low subgroups. \n",
      "Of note, global gene expression signatures demonstrated that the CL2 and CL3 subgroups showed lower luminal- and basal-related signature scores, respectively, compared with luminal- and basal-like breast tumors, while exhibiting a significant enrichment in invasiveness-related signatures (Supplementary Fig. 5a, b [this file] ). \n",
      "A claudin-low gene list of 137 genes discriminating the 3 claudin-low subgroups was thus created (Supplementary Fig. 6a, b [this file] ). This expression-based classifier showed an accuracy of 91% when compared with the FGA-based classification using Gaussian finite mixture models (Supplementary Fig. 5c, d [this file] ). Validating the accuracy of our classifier, CL1, CL2, and CL3 tumors and cell lines showed low, moderate, and elevated FGA levels, respectively (Supplementary Fig. 6e, f [this file] ). Of note, we performed a pathway analysis of CL1 discriminant genes that revealed an enrichment in stem cell and pediatric cancer markers (Supplementary Fig. 6g [this file] ). \n",
      "Differentially methylated gene analysis was next performed in claudin-low subgroups followed by pathway-enrichment analysis, using over 15,000 pathways available in MSigDB (HALLMARKS, C2, and C5 gene sets) (Supplementary Fig. 7 [this file] ). The CL1 subgroup had the highest score for stemness- ( , ) and EMT- (mesenchymal markers: , , ; epithelial markers: , ) related genes, whereas CL2 and CL3 displayed an intermediate score compared with CL1 tumors and with their relative luminal/basal counterparts (Supplementary Fig. 8 [this file] ). \n",
      "CL1 tumors mainly display hypomethylated genes that are related to a stemness phenotype, CL2 tumors are overall hypermethylated for genes associated with basal-like features, and CL3 tumors show more heterogeneous methylation profiles with both hypermethylated genes (linked to luminal phenotype) and hypomethylated genes (relative to basal-like characteristics) (see Supplementary Fig. 5 [this file] for methodology). \n",
      "Additional gene expression analyses performed on METABRIC and TCGA cohorts using two different cellular signature deconvolution algorithms confirmed that these tumors were highly infiltrated by immune cells (Supplementary Fig. 9a–d [this file] ). Nevertheless, unsupervised clustering of microenvironment signatures (immune and non-immune cell subsets) did not allow to discriminate the different claudin-low subgroups (Supplementary Fig. 9e–h [this file] ). \n",
      "Supporting our previous results, CL2 and CL3 showed, among the three claudin-low subgroups, the highest score for luminal- (estrogen-related response) and basal-related (proliferation and DDR) signatures, respectively (Supplementary Fig. 10 [this file] ). Furthermore, both CL2 and CL3 showed an activation of the EMT process (Fig. 6a, b, g [rid=Fig6] ), coherent with the downregulation of hormone (in CL2) and proliferation/DDR (in CL3) signatures, compared with luminal and basal tumors, respectively (Supplementary Fig. 10 [this file] and Fig. 6g [rid=Fig6] ). Moreover, CL1 displayed a low proliferation activity and a low DDR (Supplementary Fig. 10a, b, e, f [this file] ), as well as a strong TP53 signaling pathway signature associated with a low frequency of mutations (Fig. 6c, d, g [rid=Fig6] ). \n",
      "When stratified by CL subgroups, variations in survival outcome were found, consistent with their presumed origin (Supplementary Fig. 11 [this file] ). \n",
      "These traits include (i) a high expression of the ZEB1 EMT-inducing transcription factor and of its target, the methionine sulfoxide reductase MSRB3; (ii) a frequent activation of the RAS-MAPK signaling pathway; (iii) a low activation of the DDR; and (iv) a low frequency of mutations (Supplementary Figs. 8 [this file] and 10 [this file] , and Fig. 6 [rid=Fig6] ). \n",
      "The use of a high-purity threshold for tumor selection allowed to circumvent the contamination bias related to the tumor-related centroid (Supplementary Fig. 1 [this file] ). \n",
      "Finally, the nearest shrunken centroid classifier encompassed 137 genes whose expression discriminate the three FGA-related claudin-low subgroups in METABRIC cohort (Supplementary Fig. 6 [this file] ). \n",
      "As previously described for genome-wide expression–methylation quantitative trait loci analysis , genes for which methylation -values were negatively correlated with expression values (Spearman’s < 0 and FDR -value < 0.05) were selected for differential methylation analysis (Supplementary Fig. 7 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM2: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM3: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM4: Gene expression signatures for MaSCs, LPs, and mL cells were generated based on the transcription profiles of subpopulations of normal human mammary epithelial cells isolated from reduction mammoplasty tissues, as shown in Morel et al. (Supplementary Data 1 [this file] ). \n",
      "The top 500 false discovery rate (FDR) -value ranked genes differentially expressed between one subpopulation (MaSC or LP, or mL cells) vs. the two others (FDR -value < 0.05) were used as transcriptomic signature for each of the three cell subpopulations (Supplementary Data 1 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM5: Further information on research design is available in the Nature Research Reporting Summary [this file] linked to this article. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 243 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 38549\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 149516\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 39271 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-98b8a414306a6caebc0c08897098bfc2d9729c34bf5986dcde59683519356c1e\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 39271 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\n",
      "  \"datasets\": [\n",
      "    {\n",
      "      \"dataset_identifier\": \"EGAS00000000083\",\n",
      "      \"repository_reference\": \"http://www.ebi.ac.uk/ega/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE56031\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE56031\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"syn1757053\",\n",
      "      \"repository_reference\": \"https://www.synapse.org/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"https://portal.gdc.cancer.gov/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"https://portals.broadinstitute.org/ccle\"\n",
      "    }\n",
      "  ]\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'EGAS00000000083', 'repository_reference': 'http://www.ebi.ac.uk/ega/'}, {'dataset_identifier': 'GSE56031', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE56031'}, {'dataset_identifier': 'syn1757053', 'repository_reference': 'https://www.synapse.org/'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://portal.gdc.cancer.gov/'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://portals.broadinstitute.org/ccle'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 6 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: EGAS00000000083\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE56031\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn1757053\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'EGAS00000000083', 'repository_reference': 'http://www.ebi.ac.uk/ega/', 'dataset_id': 'EGAS00000000083'}, {'dataset_identifier': 'GSE56031', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE56031', 'dataset_id': 'GSE56031'}, {'dataset_identifier': 'syn1757053', 'repository_reference': 'https://www.synapse.org/', 'dataset_id': 'syn1757053'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://portal.gdc.cancer.gov/'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://portals.broadinstitute.org/ccle'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'EGAS00000000083', 'repository_reference': 'http://www.ebi.ac.uk/ega/', 'dataset_id': 'EGAS00000000083'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: EGAS00000000083\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID EGAS00000000083 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: http://www.ebi.ac.uk/ega/, identifier: EGAS00000000083\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: http://www.ebi.ac.uk/ega/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier EGAS00000000083 for http://www.ebi.ac.uk/ega/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier EGAS00000000083 for http://www.ebi.ac.uk/ega/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier EGAS00000000083 for http://www.ebi.ac.uk/ega/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier EGAS00000000083 for http://www.ebi.ac.uk/ega/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier EGAS00000000083 for http://www.ebi.ac.uk/ega/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: http://www.ebi.ac.uk/ega/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'EGAS00000000083', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE56031', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE56031', 'dataset_id': 'GSE56031'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE56031\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE56031, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE56031\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE56031\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE56031', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn1757053', 'repository_reference': 'https://www.synapse.org/', 'dataset_id': 'syn1757053'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn1757053, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn1757053\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn1757053', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 677 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 605 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://portal.gdc.cancer.gov/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://portal.gdc.cancer.gov/, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://portal.gdc.cancer.gov/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://portal.gdc.cancer.gov/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'portal.gdc.cancer.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://portals.broadinstitute.org/ccle'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://portals.broadinstitute.org/ccle, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://portals.broadinstitute.org/ccle\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://portals.broadinstitute.org/ccle\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'portals.broadinstitute.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 256 - INFO - Augmented dataset links: [{'dataset_identifier': 'EGAS00000000083', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE56031', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn1757053', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'portal.gdc.cancer.gov', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'portals.broadinstitute.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 5 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: EGAS00000000083\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: www.ebi.ac.uk, identifier: EGAS00000000083\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for EGAS00000000083 in www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 1 with repo: www.ebi.ac.uk and accession_id: EGAS00000000083\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ebi.ac.uk/pride/archive/projects/EGAS00000000083\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE56031\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE56031\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE56031 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE56031 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE56031 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 2 with repo: geo and accession_id: GSE56031\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE56031\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 2: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn1757053\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn1757053\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn1757053 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn1757053 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn1757053 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn1757053 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn1757053 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 3 with repo: synapse.org and accession_id: syn1757053\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn1757053\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 3: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Skipping dataset 4: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Skipping dataset 5: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 5\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 265 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 10, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 477 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 24th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC10802452&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 461 - WARNING - No links found in the section.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 406 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 430 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 443 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 264 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 454 - INFO - Found 5 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b0962a40>, <Element supplementary-material at 0x1b0962880>, <Element supplementary-material at 0x1b0962040>, <Element supplementary-material at 0x1b0962480>, <Element supplementary-material at 0x1b0962140>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 462 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b0962a40>, <Element supplementary-material at 0x1b0962880>, <Element supplementary-material at 0x1b0962040>, <Element supplementary-material at 0x1b0962480>, <Element supplementary-material at 0x1b0962140>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 548 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref SD1: This cohort (n=104) comprised neurotypical (healthy) controls (HC, n = 38), and individuals with SCZ (ISCZ, n=38), BD (n = 20) and MDD (n = 8) ( Supplementary Table 1 [this file] ) with a mean age of onset in the late 20s adolescence (mean 28.5/27.1/29.1 SCZ/BD/MDD). \n",
      "From the latter, we also included a cohort of Amish individuals suffering from BD/MDD and respective controls, see Supplementary Table 1 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref SD2: Importantly, 55.4% of the identified eQTLs overlapped with eQTLs identified in adult human tissue from the prefrontal cortex (PFC) , supporting their relevance ( Fig. 1f [rid=F1] and Supplementary Table 2 [this file] ). Nevertheless, 21.7% of the identified open chromatin peaks associated with caQTLs overlapped with eQTLs detected in bulk PFC. ( Fig. 1g [rid=F1] and Supplementary Table 2 [this file] ). \n",
      "Interestingly, the overlap of QTLs across molecular features in iNs was limited, with 24.9% of eQTL overlapping with caQTLs ( Fig. 1l [rid=F1] and Supplementary Table 2 [this file] ). Moreover, 28 genes, 78 peaks and no microRNA with e/ca/miQTLs overlapped with GWAS risk loci for SCZ ( Fig. 1f [rid=F1] - h [rid=F1] and Supplementary Table 2 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref SD3: Following a detailed power evaluation ( Extended Data Fig. 3b [rid=F9] - d [rid=F9] ), we performed differential expression analysis and identified 467 transcripts (365 protein coding and 102 non-coding transcripts) (FDR≤0.01, ∣log FC∣≥0.4; Fig. 2a [rid=F2] and Extended Data Fig. 3e [rid=F9] , and Supplementary Table 3 [this file] ), henceforth referred to as DEGs as well as 67 micro RNAs (FDR≤0.05; Fig. 2b [rid=F2] and Supplementary Table 3 [this file] ). \n",
      "This analysis revealed widespread differences in APA, identifying 628 differential APA (dAPA) events ( Fig. 3c [rid=F3] and Supplementary Table 3 [this file] ) that resulted from an increased usage of either the distal or the proximal PAS to similar degree (14.29% of all genes with more than 1 polyadenylation site). \n",
      "Consistent with the observations in iNs, this analysis confirmed widespread dAPA in postmortem PFC from ISCZ ( Fig. 3e [rid=F3] and Supplementary Table 3 [this file] ) as well as substantial overlap with dAPA in iNs ( Fig. 3f [rid=F3] ). \n",
      "Strikingly, genes subject to dAPA in ISCZ were strongly enriched for synapse related processes such as glutamatergic neurotransmission, synaptic membrane components, synaptic plasticity or vesicle biology in both iNs and human PFC ( Fig. 4a [rid=F4] , b [rid=F4] , and Supplementary Table 3 [this file] ). \n",
      "RBP motif analysis in the UTRs of transcripts subject to dAPA identified numerous expressed RBPs with critical roles for neuronal biology such as QKI, ELAVL and RBM proteins ( Fig. 5c [rid=F5] and Supplementary Table 3 [this file] ). \n",
      "Similarly, we also corrected the R values for gender, batch and TechnicalPC1 using beta regression and used the residuals to compute the average difference in R values between the groups (ΔAPA, Supplementary Table 3 [this file] ). \n",
      "Resulting enrichment scores and p-values for the set of transcripts with increased usage of the distal PAS are depicted in Fig. 5c [rid=F5] with p-values corrected for multiple testing using the q-value method and listed in Supplementary Table 3 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref SD4: MAP2, RTF2 or Distal/Proximal UTR, Supplementary Table 4 [this file] ) were used to calculate target specific relative expression as 2^(-(CT -CT )) giving rise to normalized transcript levels. \n",
      "Statistical testing of NRIs ( Supplementary Table 4 [this file] ) was conducted using by linear mixed models as implemented in the lme4 R package modeling diagnosis or treatment as fixed effect and batch as well as donor (for PSD95 analysis) as random effect. \n",
      "All well level density values used can be found in Supplementary Table 4 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref SD5: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 243 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 71025\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 267302\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 71747 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-65f6f1d8030fadd812bc390b6509a28084c90f8695448f2063272345cc6793f4\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 71747 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\n",
      "  \"datasets\": [\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    }\n",
      "  ]\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 1 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 677 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 605 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 256 - INFO - Augmented dataset links: []\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 0 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 0\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 265 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 5, with cols: Index(['link', 'source_url', 'download_link', 'title', 'content_type', 'id',\n",
      "       'caption', 'description', 'source_section', 'retrieval_pattern',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 477 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 25th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC8907344)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC8907344\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC8907344&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 394 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC8907344.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b5f70810>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC8907344 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC8907344, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 445 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC8907344)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 547 - INFO - Extracted 12 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 557 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#Sec17'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for Sec17: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#Sec18'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for Sec18: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#Sec18'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for Sec18: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#Sec25'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for Sec25: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 31 <a> elements with href='#MOESM1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 2279) for MOESM1: S1 [href=this file] a,b).\n",
      "S2 [href=this file] shows the GO classifications of the acetylated proteins enriched in changed lists after SIRT2 knockdown with Supplementary Fig. S2 [href=this file] a indicating proteins with a decrease in acetylation and Supplementary Fig. S2 [href=this file] b listing those with an increase in acetylation. S3 [href=this file] shows the GO classification of the acetylated proteins after SIRT2 overexpression, with Supplementary Fig. S3 [href=this file] a indicating proteins with a decrease in acetylation and Supplementary Fig. S3 [href=this file] b listing those with an increase in acetylation.\n",
      "S4 [href=this file] a shows the correlation between up-regulated acetyl peptides in SIRT2 knockdown and down-regulated acetyl peptides in SIRT2 overexpression with identical corresponding peptides. S4 [href=this file] b shows the overlap between down-regulated acetyl peptides in SIRT2 knockdown and up-regulated acetyl peptides in SIRT2 overexpression. S4 [href=this file] a), suggesting that these acetylation sites are direct SIRT2 deacetylation targets. S4 [href=this file] b), suggesting that their acetylation may be indirectly dependent on regulation downstream of SIRT2. S2 [href=this file] d), providing validation for BCL9 as a novel SIRT2 substrate.\n",
      "S4 [href=this file] a shown in Supplementary Fig. S5 [href=this file] a. S5 [href=this file] a and Fig. 3 [href=#Fig3] a, red bars), with both comprising terms for actomyosin, actin filament bundles, histone acetyltransferase complex, and protein-DNA complex. S5 [href=this file] a and Fig. 3 [href=#Fig3] a, green bars). S5 [href=this file] a, blue bars). S5 [href=this file] b).\n",
      "S6 [href=this file] ).\n",
      "S7 [href=this file] ).\n",
      "S8 [href=this file] a). S8 [href=this file] b).\n",
      "S8 [href=this file] a). S9 [href=this file] a). S9 [href=this file] b). S8 [href=this file] a), suggesting that proteins important in tumorigenesis are key SIRT2 targets. S5 [href=this file] b), which discovered a significant enrichment of tumor-related GO terms. S9 [href=this file] c, an interaction network with these proteins revealed ubiquitin and p300 as two central hubs, with PARP1 and CBP being secondary hubs.\n",
      "S10 [href=this file] a), and protein-level FPR (Supplementary Fig. S10 [href=this file] b).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#MOESM2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 199) for MOESM2: Using a false discovery rate (FDR) of < 1%, we identified a total of 2846 unique lysine acetylation sites from 1414 protein isoforms of 1404 gene products (Supplementary Table S1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#MOESM3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for MOESM3: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#MOESM4'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for MOESM4: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 2 <a> elements with href='#MOESM5'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 471) for MOESM5: To identify how many of the direct SIRT2 targets are validated known SIRT2 substrates, we conducted an extensive literature review to curate experimentally validated SIRT2 substrates (Supplementary Table S4 [href=this file] ). More importantly, we found that 13 proteins are among our high-confidence direct SIRT2 target list and the known SIRT2 substrates, accounting for 23% of the total known list (Fig. 2 [href=#Fig2] c and Supplementary Table S4 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#MOESM6'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 506) for MOESM6: To determine the overall specificity of the enrichment PSMs for acetylated lysine-containing peptides/total peptide PSMs for each of the four samples that underwent Ac-Lys enrichment with the Cell Signaling antibody prior to LC–MS/MS, we also conducted enrichment calculations from the alternate Proteome Discoverer (Thermo Scientific) software (PD) search for up to 3 Acetyl [K] per peptide, fully tryptic and 1% FDR enforced (high confidence peptide PSMs) (see Supplementary Table S5 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 44559\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 148722\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 45281 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-a01b7394109f0fdfc285ad5355c03edb08476dce37a813bf3031bf06edfb2874\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 45281 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\n",
      "  \"datasets\": [\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"https://doi.org/10.1038/s41598-022-06793-5\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8907344/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"https://www.synapse.org/#!Synapse:syn26134616\"\n",
      "    }\n",
      "  ]\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://doi.org/10.1038/s41598-022-06793-5'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8907344/'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn26134616'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 4 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://doi.org/10.1038/s41598-022-06793-5'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8907344/'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn26134616'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 677 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 605 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://doi.org/10.1038/s41598-022-06793-5'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(10\\.\\d{4,9}/).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: 10.1038/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: 10.1038/, None, https://doi.org/10.1038/s41598-022-06793-5\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: 10.1038/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://doi.org/10.1038/s41598-022-06793-5, identifier: 10.1038/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://doi.org/10.1038/s41598-022-06793-5\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.1038/ for https://doi.org/10.1038/s41598-022-06793-5\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 10.1038/ for https://doi.org/10.1038/s41598-022-06793-5\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1153 - INFO - Found id_pattern match for https://doi.org/10.1038/s41598-022-06793-5 in 10\\.\\d{4,9}/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': '10.1038/', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8907344/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8907344/, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8907344/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8907344/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'www.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn26134616'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn26134616\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn26134616, None, https://www.synapse.org/#!Synapse:syn26134616\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn26134616\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn26134616, identifier: syn26134616\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn26134616\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn26134616 for https://www.synapse.org/#!Synapse:syn26134616\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn26134616 for https://www.synapse.org/#!Synapse:syn26134616\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn26134616 for https://www.synapse.org/#!Synapse:syn26134616\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn26134616 for https://www.synapse.org/#!Synapse:syn26134616\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn26134616 in Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn26134616', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': '10.1038/', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'www.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn26134616', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 3 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: 10.1038/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: doi.org, identifier: 10.1038/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.1038/ for doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for 10.1038/ in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 1 with repo: doi.org and accession_id: 10.1038/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://doi.org/10.1038/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Skipping dataset 2: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn26134616\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn26134616\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn26134616 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn26134616 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn26134616 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn26134616 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn26134616 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 3 with repo: synapse.org and accession_id: syn26134616\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn26134616\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 3: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 3\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 15, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 640 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - Parsed data extraction completed. Elements collected: 10\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 481 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 26th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10238095)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10238095\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC10238095&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 406 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC10238095.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC10238095 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 430 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC10238095, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 443 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 264 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 454 - INFO - Found 4 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1aff7fd80>, <Element supplementary-material at 0x1b036a380>, <Element supplementary-material at 0x1b0368cc0>, <Element supplementary-material at 0x1b0368b40>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 462 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1aff7fd80>, <Element supplementary-material at 0x1b036a380>, <Element supplementary-material at 0x1b0368cc0>, <Element supplementary-material at 0x1b0368b40>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 548 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref supp1: According to the above biological framework, a total of 61 TME-related signatures were collected to form the TME panel ( Supplementary file 1a [this file] , Supplementary file 1b [this file] ), which ultimately established a holistic approach to thoroughly dissect the TME of CRC. \n",
      "Principal component analysis (PCA) indicated that after using the ‘ComBat’ function to remove batch effects, there was no significant batch effect in the merged cohorts of eight microarray datasets ( Figure 1—figure supplement 1A, B [rid=fig1s1] ) and two RNA sequencing datasets ( Figure 1—figure supplement 1C, D [rid=fig1s1] , Supplementary file 1c [this file] ). \n",
      "The stromal and tumor components significantly were correlated with decreased survival, particularly in the case of mesenchymal cells, endothelial cells, metastasis, differentiation, and EMT signatures ( Figure 1—figure supplement 1L [rid=fig1s1] , Supplementary file 1d [this file] ). \n",
      "Differences in the TME components between the CCCRC subtypes were also observed in the analysis of previously reported immune and stromal signatures obtained by the microenvironment cell populations (MCP)-counter, CIBERSORT, and ESTIMATE algorithm, and 10 classical oncogenic pathway activities and 86 metabolic pathway enrichment scores calculated by GSVA ( Figure 1—figure supplement 3A–E [rid=fig1s3] , Supplementary file 1e [this file] ). \n",
      "We analyzed the 86 metabolic pathways obtained from the Kyoto Encyclopedia of Genes and Genomes (KEGG) database ( Supplementary file 1e [this file] ) and observed that the number of upregulated metabolic pathways of the C3 subtype was the lowest. \n",
      "We further analyzed the association of CCCRC subtypes with clinicopathological characteristics ( Supplementary file 1f [this file] , Supplementary file 1g [this file] ). Multivariate Cox proportional hazard regression analysis showed that the C4 subtype was an independent predictor of the best OS and DFS, whereas the C3 subtype was an independent predictor of the worst OS and DFS after adjustment for age, gender, tumor site, TNM stage, grade, adjuvant chemotherapy or not, MSI status, BRAF and KRAS mutations, and the CMS classification system in the combined cohort (the CRC-AFFY and CRC-RNAseq cohorts) ( Supplementary file 1h [this file] ). \n",
      "The 254 CRC samples in the TCGA-CRC dataset were categorized as these three phenotypes based on the abundance of lymphocytes and their spatial location with malignant epithelial cells ( Supplementary file 1i [this file] ). \n",
      "A total of 2196 samples were obtained from ten publicly available datasets ( Supplementary file 1c [this file] ). Detailed information on the sample size and the corresponding clinicopathological data of the CRC-AFFY and CRC-RNAseq cohorts are summarized in Supplementary file 1c [this file] and Supplementary file 1f [this file] . \n",
      "The 4525 genes from each of the 61 TME-related signatures are listed in Supplementary file 1a [this file] , as well as the source of each signature. \n",
      "A total of 254 WSIs were included after removing the WSIs with poor quality and without views of the IM ( Supplementary file 1i [this file] ). \n",
      "The markers for the six cell types are listed in the Supplementary file 1a [this file] and Supplementary file 3a [rid=supp3] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref supp2: Among the frequently mutated genes (>5%), the mutation frequencies of (85.8%), (64.9%), and (46.7%) were the highest in the C1 subtype compared to the other subtypes (all p-value <0.05), followed by the C3, C2, and C4 subtypes, which are closely associated with the occurrence of CRC ( Figure 3A [rid=fig3] , Supplementary file 2a [this file] ). In terms of the differences in SCNA, the C1 subtype with the highest CIN level harbored significantly more amplified chromosomal regions (20q12, 20q13.12, 20q11.21, and 20q13.32) and deleted chromosomal regions (18q21.2, 18q22.1, and 18q12.3) (all p-value <0.05) ( Figure 3A, B [rid=fig3] , Supplementary file 2b [this file] ). The C4 subtype had the highest frequency of mutations in the cell cycle, HIPPO, MYC, NOTCH, PI3K, TGFB, and RTK–RAS pathways (all p-value <0.05) ( Figure 3C [rid=fig3] , Supplementary file 2c [this file] ). \n",
      "The lists of differentially expressed genes (DEGs) between CCCRC subtypes were compiled in Supplementary file 2d [this file] . In addition, we also analyzed the correlations between all genes and the exclusion and dysfunction scores obtained from the Tumor Immune Dysfunction and Exclusion (TIDE) website, as well as the MSI expression signature, which provided a useful reference for subsequent research in the discovery of new therapeutic targets ( Supplementary file 2e [this file] ). \n",
      "Furthermore, we intersected all the positives for increasing gene expression from C1 to C4, C2, and C3 subtypes in the CRC-AFFY and CRC-RNAseq cohorts and obtained 20 CCCRC genes ( Figure 4E [rid=fig4] , Supplementary file 2f [this file] ), which were associated with TGF-beta signaling and infiltrating nerves. \n",
      "We obtained the GEP for 7766T cells from 12 patients with CRC, including four patients with the C1 subtype, one patient with the C2 subtype, two patients with the C3 subtype, and four patients with the C4 subtype ( Supplementary file 2g [this file] ). The markers of the T cell clusters were summarized in Supplementary file 2h [this file] . \n",
      "For instance, KDM1A and RAD21 are specific genes for C1 subtype, and it has been reported that interfering with them could activate interferon signaling and induce tumor intrinsic immunogenicity ( Deng et al., 2022 [rid=bib26] ; Nguyen et al., 2022 [rid=bib65] ; Supplementary file 2d [this file] ). In this study, the expression levels of SPP1 and SERPINB9 were found to be significantly positively correlated with the exclusion and dysfunction scores (TIDE score), respectively ( Supplementary file 2e [this file] ). \n",
      "A total of 12 CRC samples with bulk RNAseq and scRNAseq data were obtained from the GSE108989 (https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE108989) dataset ( Zhang et al., 2018 [rid=bib96] ; Supplementary file 2g [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref supp3: To further explore the treatment strategies of the CCCRC subtypes, we trained a pre-clinical model based on a filtered gene set comprised of 335 CCCRC subtype-specific and cancer cell-intrinsic gene markers ( Supplementary file 3a [this file] ). The 71 human CRC cell lines were classified into four CCCRC subtypes ( Supplementary file 3b [this file] ). \n",
      "We established the CCCRC classifier on the training set by utilizing multiple machine learning algorithms based on the GEP of 80 upregulated subtype-specific genes (Materials and methods, Supplementary file 3c [this file] ). Notably, the CCCRC classifier based on the xgboost algorithm displayed robust performance across gene expression platforms, Affymetrix and RNA-sequencing platforms, exhibiting a balanced accuracy of >80% for all subtypes ( Supplementary file 3d [this file] ). \n",
      "The 1660 genes assigned to 86 human metabolism-related pathways are listed in Supplementary file 3e [this file] . Ten oncogenic signatures containing 331 genes and the terminally exhausted T cell signature were retrieved from a previously published study ( Beltra et al., 2020 [rid=bib11] ; Sanchez-Vega et al., 2018 [rid=bib73] ; Supplementary file 3e [this file] ). \n",
      "The R package ‘RTN’ was used to reconstruct the transcriptional regulatory networks of regulons ( Robertson et al., 2017 [rid=bib69] ), including 31 transcription factors and 82 chromatin remodeling genes, that were associated with CRC ( Vymetalkova et al., 2019 [rid=bib85] ; Xu et al., 2021 [rid=bib91] ; Supplementary file 3f [this file] ). \n",
      "To classify CRC samples into different CRC subtypes according to the previously published gene classifier, gene lists for the four classifiers were extracted from relevant publications and summarized ( Supplementary file 3g [this file] ). \n",
      "Detailed information on the sample size and the corresponding treatment data of the therapy-associated datasets are summarized in Supplementary file 3h [this file] . \n",
      "The markers for the six cell types are listed in the Supplementary file 1a [rid=supp1] and Supplementary file 3a [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref mdar: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 243 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 131220\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 130513\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 129857\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 129185\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 128690\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 128251\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 127780\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 127322\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 126833\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 537018\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 127555 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-52f77ef9331644fc282f61d75f5381daefae7ada0057b8543beccc20b7ebf377\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 127555 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\"datasets\":[{\"dataset_identifier\":\"GSE39582\",\"repository_reference\":\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE39582\"},{\"dataset_identifier\":\"GSE13067\",\"repository_reference\":\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE13067\"},{\"dataset_identifier\":\"GSE13294\",\"repository_reference\":\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE13294\"},{\"dataset_identifier\":\"GSE14333\",\"repository_reference\":\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE14333\"},{\"dataset_identifier\":\"GSE17536\",\"repository_reference\":\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE17536\"},{\"dataset_identifier\":\"GSE33113\",\"repository_reference\":\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE33113\"},{\"dataset_identifier\":\"GSE37892\",\"repository_reference\":\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE37892\"},{\"dataset_identifier\":\"GSE38832\",\"repository_reference\":\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38832\"},{\"dataset_identifier\":\"PRJEB23709\",\"repository_reference\":\"https://www.ncbi.nlm.nih.gov/bioproject/PRJEB23709\"},{\"dataset_identifier\":\"GSE103479\",\"repository_reference\":\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE103479\"},{\"dataset_identifier\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 759 - WARNING - Initial JSON parsing failed. Attempting json_repair...\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'GSE39582', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE39582'}, {'dataset_identifier': 'GSE13067', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE13067'}, {'dataset_identifier': 'GSE13294', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE13294'}, {'dataset_identifier': 'GSE14333', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE14333'}, {'dataset_identifier': 'GSE17536', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE17536'}, {'dataset_identifier': 'GSE33113', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE33113'}, {'dataset_identifier': 'GSE37892', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE37892'}, {'dataset_identifier': 'GSE38832', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38832'}, {'dataset_identifier': 'PRJEB23709', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/bioproject/PRJEB23709'}, {'dataset_identifier': 'GSE103479', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE103479'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 10 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE39582\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE13067\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE13294\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE14333\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE17536\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE33113\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE37892\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE38832\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: PRJEB23709\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE103479\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'GSE39582', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE39582', 'dataset_id': 'GSE39582'}, {'dataset_identifier': 'GSE13067', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE13067', 'dataset_id': 'GSE13067'}, {'dataset_identifier': 'GSE13294', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE13294', 'dataset_id': 'GSE13294'}, {'dataset_identifier': 'GSE14333', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE14333', 'dataset_id': 'GSE14333'}, {'dataset_identifier': 'GSE17536', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE17536', 'dataset_id': 'GSE17536'}, {'dataset_identifier': 'GSE33113', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE33113', 'dataset_id': 'GSE33113'}, {'dataset_identifier': 'GSE37892', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE37892', 'dataset_id': 'GSE37892'}, {'dataset_identifier': 'GSE38832', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38832', 'dataset_id': 'GSE38832'}, {'dataset_identifier': 'PRJEB23709', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/bioproject/PRJEB23709', 'dataset_id': 'PRJEB23709'}, {'dataset_identifier': 'GSE103479', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE103479', 'dataset_id': 'GSE103479'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE39582', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE39582', 'dataset_id': 'GSE39582'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE39582\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE39582, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE39582\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE39582\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE39582', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE13067', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE13067', 'dataset_id': 'GSE13067'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE13067\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE13067, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE13067\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE13067\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE13067', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE13294', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE13294', 'dataset_id': 'GSE13294'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE13294\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE13294, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE13294\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE13294\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE13294', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE14333', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE14333', 'dataset_id': 'GSE14333'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE14333\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE14333, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE14333\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE14333\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE14333', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE17536', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE17536', 'dataset_id': 'GSE17536'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE17536\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE17536, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE17536\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE17536\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE17536', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE33113', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE33113', 'dataset_id': 'GSE33113'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE33113\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE33113, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE33113\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE33113\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE33113', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE37892', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE37892', 'dataset_id': 'GSE37892'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE37892\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE37892, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE37892\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE37892\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE37892', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE38832', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38832', 'dataset_id': 'GSE38832'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE38832\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE38832, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38832\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE38832\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE38832', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'PRJEB23709', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/bioproject/PRJEB23709', 'dataset_id': 'PRJEB23709'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: PRJEB23709\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID PRJEB23709 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.ncbi.nlm.nih.gov/bioproject/PRJEB23709, identifier: PRJEB23709\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.ncbi.nlm.nih.gov/bioproject/PRJEB23709\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier PRJEB23709 for https://www.ncbi.nlm.nih.gov/bioproject/PRJEB23709\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier PRJEB23709 for https://www.ncbi.nlm.nih.gov/bioproject/PRJEB23709\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier PRJEB23709 for https://www.ncbi.nlm.nih.gov/bioproject/PRJEB23709\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier PRJEB23709 for https://www.ncbi.nlm.nih.gov/bioproject/PRJEB23709\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier PRJEB23709 for https://www.ncbi.nlm.nih.gov/bioproject/PRJEB23709\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://www.ncbi.nlm.nih.gov/bioproject/PRJEB23709\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'PRJEB23709', 'data_repository': 'www.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE103479', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE103479', 'dataset_id': 'GSE103479'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE103479\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE103479, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE103479\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE103479\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE103479', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 256 - INFO - Augmented dataset links: [{'dataset_identifier': 'GSE39582', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE13067', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE13294', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE14333', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE17536', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE33113', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE37892', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE38832', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'PRJEB23709', 'data_repository': 'www.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE103479', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 10 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE39582\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE39582\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE39582 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE39582 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE39582 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE39582\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE39582\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE13067\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE13067\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE13067 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE13067 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE13067 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 2 with repo: geo and accession_id: GSE13067\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE13067\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 2: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE13294\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE13294\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE13294 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE13294 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE13294 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 3 with repo: geo and accession_id: GSE13294\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE13294\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 3: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE14333\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE14333\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE14333 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE14333 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE14333 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 4 with repo: geo and accession_id: GSE14333\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE14333\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 4: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE17536\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE17536\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE17536 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE17536 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE17536 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 5 with repo: geo and accession_id: GSE17536\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE17536\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 5: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE33113\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE33113\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE33113 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE33113 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE33113 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 6 with repo: geo and accession_id: GSE33113\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE33113\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 6: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE37892\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE37892\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE37892 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE37892 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE37892 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 7 with repo: geo and accession_id: GSE37892\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE37892\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 7: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE38832\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE38832\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE38832 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE38832 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE38832 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 8 with repo: geo and accession_id: GSE38832\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38832\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 8: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: PRJEB23709\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: www.ncbi.nlm.nih.gov, identifier: PRJEB23709\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier PRJEB23709 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier PRJEB23709 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier PRJEB23709 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier PRJEB23709 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier PRJEB23709 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 976 - WARNING - URL www.ncbi.nlm.nih.gov may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for PRJEB23709 in www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 9 with repo: www.ncbi.nlm.nih.gov and accession_id: PRJEB23709\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1254 - WARNING - Repository www.ncbi.nlm.nih.gov unknown in Ontology. Skipping dataset page 9.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 10 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE103479\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE103479\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE103479 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE103479 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE103479 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 10 with repo: geo and accession_id: GSE103479\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE103479\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 10: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 10\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 265 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 14, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 477 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 27th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC9710693)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC9710693\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC9710693&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: supplementary_data_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 394 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC9710693.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b6217dd0>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC9710693 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC9710693, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 445 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC9710693)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 547 - INFO - Extracted 9 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 557 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#s3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for s3: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#ack1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 34) for ack1: Acknowledgements [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#sec18'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 35) for sec18: Data availability [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#sec18'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 35) for sec18: Data availability [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#sec18'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 35) for sec18: Data availability [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#sec18'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 35) for sec18: Data availability [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#ref1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for ref1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 39333\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 131750\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 40055 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-2de3645643baf37c7622769e4fd22b7048bef4244356f54d82b11bf3cea47953\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 40055 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\n",
      "  \"datasets\": [\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE75748\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE75748\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE72056\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE72056\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"E-MTAB-5061\",\n",
      "      \"repository_reference\": \"https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-5061/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"E-MTAB-5060\",\n",
      "      \"repository_reference\": \"https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-5060/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"syn23554292\",\n",
      "      \"repository_reference\": \"https://www.synapse.org/#!Synapse:syn23554292\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"syn3505720\",\n",
      "      \"repository_reference\": \"https://www.synapse.org/#!Synapse:syn3505720\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"syn3191087\",\n",
      "      \"repository_reference\": \"https://www.synapse.org/#!Synapse:syn3191087\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"https://cibersortx.stanford.edu/download.php\"\n",
      "    }\n",
      "  ]\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'GSE75748', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE75748'}, {'dataset_identifier': 'GSE72056', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE72056'}, {'dataset_identifier': 'E-MTAB-5061', 'repository_reference': 'https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-5061/'}, {'dataset_identifier': 'E-MTAB-5060', 'repository_reference': 'https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-5060/'}, {'dataset_identifier': 'syn23554292', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn23554292'}, {'dataset_identifier': 'syn3505720', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn3505720'}, {'dataset_identifier': 'syn3191087', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn3191087'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://cibersortx.stanford.edu/download.php'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 8 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE75748\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE72056\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: E-MTAB-5061\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: E-MTAB-5060\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn23554292\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn3505720\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn3191087\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'GSE75748', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE75748', 'dataset_id': 'GSE75748'}, {'dataset_identifier': 'GSE72056', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE72056', 'dataset_id': 'GSE72056'}, {'dataset_identifier': 'E-MTAB-5061', 'repository_reference': 'https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-5061/', 'dataset_id': 'E-MTAB-5061'}, {'dataset_identifier': 'E-MTAB-5060', 'repository_reference': 'https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-5060/', 'dataset_id': 'E-MTAB-5060'}, {'dataset_identifier': 'syn23554292', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn23554292', 'dataset_id': 'syn23554292'}, {'dataset_identifier': 'syn3505720', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn3505720', 'dataset_id': 'syn3505720'}, {'dataset_identifier': 'syn3191087', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn3191087', 'dataset_id': 'syn3191087'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://cibersortx.stanford.edu/download.php', 'dataset_id': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE75748', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE75748', 'dataset_id': 'GSE75748'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE75748\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE75748, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE75748\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE75748\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE75748', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE72056', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE72056', 'dataset_id': 'GSE72056'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE72056\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE72056, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE72056\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE72056\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE72056', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'E-MTAB-5061', 'repository_reference': 'https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-5061/', 'dataset_id': 'E-MTAB-5061'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: E-MTAB-5061\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID E-MTAB-5061 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-5061/, identifier: E-MTAB-5061\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-5061/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-MTAB-5061 for https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-5061/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-MTAB-5061 for https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-5061/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-MTAB-5061 for https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-5061/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-MTAB-5061 for https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-5061/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Found partial match for https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-5061/ in ArrayExpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'E-MTAB-5061', 'data_repository': 'arrayexpress', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'E-MTAB-5060', 'repository_reference': 'https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-5060/', 'dataset_id': 'E-MTAB-5060'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: E-MTAB-5060\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID E-MTAB-5060 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-5060/, identifier: E-MTAB-5060\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-5060/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-MTAB-5060 for https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-5060/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-MTAB-5060 for https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-5060/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-MTAB-5060 for https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-5060/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-MTAB-5060 for https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-5060/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Found partial match for https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-5060/ in ArrayExpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'E-MTAB-5060', 'data_repository': 'arrayexpress', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn23554292', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn23554292', 'dataset_id': 'syn23554292'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn23554292\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn23554292, synapse.org, https://www.synapse.org/#!Synapse:syn23554292\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn23554292\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn23554292', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn3505720', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn3505720', 'dataset_id': 'syn3505720'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn3505720\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn3505720, synapse.org, https://www.synapse.org/#!Synapse:syn3505720\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn3505720\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn3505720', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn3191087', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn3191087', 'dataset_id': 'syn3191087'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn3191087\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn3191087, synapse.org, https://www.synapse.org/#!Synapse:syn3191087\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn3191087\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn3191087', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://cibersortx.stanford.edu/download.php', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://cibersortx.stanford.edu/download.php, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://cibersortx.stanford.edu/download.php\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://cibersortx.stanford.edu/download.php\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'cibersortx.stanford.edu', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'GSE75748', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE72056', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'E-MTAB-5061', 'data_repository': 'arrayexpress', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'E-MTAB-5060', 'data_repository': 'arrayexpress', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn23554292', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn3505720', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn3191087', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'cibersortx.stanford.edu', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 8 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE75748\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE75748\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE75748 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE75748 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE75748 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE75748\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE75748\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE72056\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE72056\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE72056 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE72056 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE72056 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 2 with repo: geo and accession_id: GSE72056\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE72056\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 2: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: E-MTAB-5061\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: arrayexpress, identifier: E-MTAB-5061\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-MTAB-5061 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-MTAB-5061 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-MTAB-5061 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-MTAB-5061 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for E-MTAB-5061 in arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 3 with repo: arrayexpress and accession_id: E-MTAB-5061\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ebi.ac.uk/biostudies/ArrayExpress/studies/E-MTAB-5061\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 3: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: E-MTAB-5060\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: arrayexpress, identifier: E-MTAB-5060\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-MTAB-5060 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-MTAB-5060 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-MTAB-5060 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-MTAB-5060 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for E-MTAB-5060 in arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 4 with repo: arrayexpress and accession_id: E-MTAB-5060\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ebi.ac.uk/biostudies/ArrayExpress/studies/E-MTAB-5060\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 4: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn23554292\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn23554292\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn23554292 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn23554292 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn23554292 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn23554292 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn23554292 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 5 with repo: synapse.org and accession_id: syn23554292\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn23554292\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 5: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn3505720\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn3505720\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3505720 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn3505720 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn3505720 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3505720 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn3505720 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 6 with repo: synapse.org and accession_id: syn3505720\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn3505720\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 6: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn3191087\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn3191087\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3191087 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn3191087 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn3191087 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3191087 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn3191087 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 7 with repo: synapse.org and accession_id: syn3191087\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn3191087\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 7: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Skipping dataset 8: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 8\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 17, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 640 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - Parsed data extraction completed. Elements collected: 9\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 481 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 28th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC9915613)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC9915613\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC9915613&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 406 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC9915613.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC9915613 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 430 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC9915613, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 443 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 264 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 454 - INFO - Found 3 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1aff36600>, <Element supplementary-material at 0x13da87040>, <Element supplementary-material at 0x1b0866700>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 462 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1aff36600>, <Element supplementary-material at 0x13da87040>, <Element supplementary-material at 0x1b0866700>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 548 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref SD1: S1A [this file] ), suggesting that there was not a genomic location bias associated with the biological variation. \n",
      "S1B [this file] ) (i.e., annotating genes from upstream and downstream of a certain distance from each peak). \n",
      "S2A [this file] ). S2B [this file] ), indicating that the vast majority of genes do not bear platform-based bias. S2C [this file] ). S2D [this file] ). S2E [this file] ). S2F [this file] ). S2G [this file] ). \n",
      "S3A [this file] ). S3B [this file] ). S3C [this file] ). S3D [this file] ). \n",
      "S3E [this file] ). \n",
      "S4A [this file] ). S5B [this file] and Supplementary Table S6 [rid=SD3] ). \n",
      "S4C [this file] ), suggesting the unique gene induction program was due to context-dependent transcriptional regulomes rather than epigenetic changes. \n",
      "S5A [this file] ). S5B [this file] ). \n",
      "S5C [this file] ), likely due to a large number of genes which were not uniformly regulated. \n",
      "S5D [this file] ). \n",
      "S5E [this file] ) and we further extracted and re-clustered cancer cells by corresponding epithelial markers ( Supplementary Fig. S5F [this file] , G [this file] and Fig. 7H [rid=F7] ). \n",
      "S5H [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref SD2: To extend the database, we are further crowdsourcing datasets via social media and a google form ( https://docs.google.com/spreadsheets/d/1PFMGB_-COSrUujMKl_M-Ogkmq4cAyuIEgoLwSLFuYTs/edit#gid=0 (https://docs.google.com/spreadsheets/d/1PFMGB_-COSrUujMKl_M-Ogkmq4cAyuIEgoLwSLFuYTs/edit#gid=0) ) ( Fig. 1A [rid=F1] and Supplementary Table S1 [this file] ). \n",
      "Data-set level documentations such as source publications, experimental conditions and sequencing parameters are presented in Supplementary Data Table S1 [this file] . \n",
      "All the relevant information is summarized in Supplementary Table S1 [this file] . \n",
      "Details of all the curated 136 data sets are summarized in Supplementary Table S1 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref SD3: This was to be expected as some minor technical variations in cell culture, hormone deprivation and sequencing platform were noted ( Supplementary Table S2 [this file] ). \n",
      "The intersection of up- and down-regulated DEGs from four data sets (|log2FC|>1, padj<0.05) showed that greater than 70% of DEGs were unique to one experiment, whereas only ~3% genes were regulated in all four data sets ( Fig. 3C [rid=F3] and 3D [rid=F3] , Supplementary Table S3 [this file] ). We classified the DEGs into four classes based upon the number of data sets they shared and compared the fold change regulation ( Supplementary Table S3 [this file] ). \n",
      "Importantly, the same ER antibody (Santa Cruz sc543) was utilized in all experiments to pull down ER ( Supplementary Table S4 [this file] ). We also examined the quality parameters (e.g., percentage of reads within peaks and percentage of reads within Blacklist regions) of derived peaks to examine technical variation ( Supplementary Table S5 [this file] ). The number of peaks in the vehicle groups showed a large variation ranging from 332 (C42) to 23,189 (C42) ( Supplementary Table S5 [this file] ). As expected, E2 stimulation resulted in a 1.6-to-27.5-fold gain of ER binding ( Supplementary Table S5 [this file] ). \n",
      "We identified 165, 59 and 136 genes representing early, mid, and late estrogen response signatures respectively ( Fig. 5B [rid=F5] and Supplementary Table S6 [this file] ). \n",
      "Regulatory factor prediction by Lisa( 38 [rid=R38] ) confirmed the ER/FOXA1/GATA3 as the central axis of gene regulation regardless of treatment duration ( Fig. 5D [rid=F5] and Supplementary Table S7 [this file] ). \n",
      "S5B [rid=SD1] and Supplementary Table S6 [this file] ). \n",
      "Mechanistically, MCF7 and T47D unique response genes were enriched for contextual regulatory factors ( Fig. 6D [rid=F6] and Supplementary Table S7 [this file] ) but showed equivalent levels of chromatin accessibility at each other’s open chromatin regions ( Supplementary Fig. \n",
      "We also identified a subset of bidirectionally regulated genes (n=101) that are present in the top 10% of both up- and down-regulated targets and in at least 10% of comparisons ( Fig. 7A [rid=F7] and Supplementary Table S6 [this file] ), such as , and ( Supplementary Fig. \n",
      "For the inter-study concordance analysis, detailed information of four RNA-seq and four ER ChIP-seq data sets are summarized in Supplementary Table S2 [this file] and S4 [this file] respectively.RNA-seq data and clinical information from TCGA and METABRIC were obtained from the GSE62944 (GSE62944) and Synapse software platform under accession number syn1688369, respectively. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 243 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 54012\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 200173\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 54734 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-e87258f8c58d7c8fd708ef56c9c33b13dc4b0a3964cb892d5dddd607daa3432d\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 54734 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\n",
      "  \"datasets\": [\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE69091\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE69091\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"EGAS00001000925\",\n",
      "      \"repository_reference\": \"https://ega-archive.org/datasets/EGAS00001000925\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"PXD049309\",\n",
      "      \"repository_reference\": \"https://www.ebi.ac.uk/pride/archive/projects/PXD049309\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"PXD0402123\",\n",
      "      \"repository_reference\": \"ProteomeXchange\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"MSV000092944\",\n",
      "      \"repository_reference\": \"https://massive.ucsd.edu/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"syn1827001\",\n",
      "      \"repository_reference\": \"https://www.synapse.org/[#!]*Synapse:syn28580053\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"https://data.broadinstitute.org/ccle_legacy_data/mRNA_expression/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"https://estrogene.org/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"https://docs.google.com/spreadsheets/d/1PFMGB_-COSrUujMKl_M-Ogkmq4cAyuIEgoLwSLFuYTs/edit#gid=0\"\n",
      "    }\n",
      "  ]\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'GSE69091', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE69091'}, {'dataset_identifier': 'EGAS00001000925', 'repository_reference': 'https://ega-archive.org/datasets/EGAS00001000925'}, {'dataset_identifier': 'PXD049309', 'repository_reference': 'https://www.ebi.ac.uk/pride/archive/projects/PXD049309'}, {'dataset_identifier': 'PXD0402123', 'repository_reference': 'ProteomeXchange'}, {'dataset_identifier': 'MSV000092944', 'repository_reference': 'https://massive.ucsd.edu/'}, {'dataset_identifier': 'syn1827001', 'repository_reference': 'https://www.synapse.org/[#!]*Synapse:syn28580053'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://data.broadinstitute.org/ccle_legacy_data/mRNA_expression/'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://estrogene.org/'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://docs.google.com/spreadsheets/d/1PFMGB_-COSrUujMKl_M-Ogkmq4cAyuIEgoLwSLFuYTs/edit#gid=0'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 9 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE69091\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: EGAS00001000925\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: PXD049309\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: PXD0402123\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: MSV000092944\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn1827001\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'GSE69091', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE69091', 'dataset_id': 'GSE69091'}, {'dataset_identifier': 'EGAS00001000925', 'repository_reference': 'https://ega-archive.org/datasets/EGAS00001000925', 'dataset_id': 'EGAS00001000925'}, {'dataset_identifier': 'PXD049309', 'repository_reference': 'https://www.ebi.ac.uk/pride/archive/projects/PXD049309', 'dataset_id': 'PXD049309'}, {'dataset_identifier': 'PXD0402123', 'repository_reference': 'ProteomeXchange', 'dataset_id': 'PXD0402123'}, {'dataset_identifier': 'MSV000092944', 'repository_reference': 'https://massive.ucsd.edu/', 'dataset_id': 'MSV000092944'}, {'dataset_identifier': 'syn1827001', 'repository_reference': 'https://www.synapse.org/[#!]*Synapse:syn28580053', 'dataset_id': 'syn1827001'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://data.broadinstitute.org/ccle_legacy_data/mRNA_expression/', 'dataset_id': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://estrogene.org/'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://docs.google.com/spreadsheets/d/1PFMGB_-COSrUujMKl_M-Ogkmq4cAyuIEgoLwSLFuYTs/edit#gid=0'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE69091', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE69091', 'dataset_id': 'GSE69091'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE69091\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE69091, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE69091\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE69091\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE69091', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'EGAS00001000925', 'repository_reference': 'https://ega-archive.org/datasets/EGAS00001000925', 'dataset_id': 'EGAS00001000925'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: EGAS00001000925\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID EGAS00001000925 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://ega-archive.org/datasets/EGAS00001000925, identifier: EGAS00001000925\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://ega-archive.org/datasets/EGAS00001000925\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier EGAS00001000925 for https://ega-archive.org/datasets/EGAS00001000925\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier EGAS00001000925 for https://ega-archive.org/datasets/EGAS00001000925\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier EGAS00001000925 for https://ega-archive.org/datasets/EGAS00001000925\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier EGAS00001000925 for https://ega-archive.org/datasets/EGAS00001000925\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier EGAS00001000925 for https://ega-archive.org/datasets/EGAS00001000925\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://ega-archive.org/datasets/EGAS00001000925\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'EGAS00001000925', 'data_repository': 'ega-archive.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'PXD049309', 'repository_reference': 'https://www.ebi.ac.uk/pride/archive/projects/PXD049309', 'dataset_id': 'PXD049309'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: PXD049309\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID PXD049309 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.ebi.ac.uk/pride/archive/projects/PXD049309, identifier: PXD049309\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.ebi.ac.uk/pride/archive/projects/PXD049309\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Found partial match for https://www.ebi.ac.uk/pride/archive/projects/PXD049309 in PRIDE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'PXD049309', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'PXD0402123', 'repository_reference': 'ProteomeXchange', 'dataset_id': 'PXD0402123'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: PXD0402123\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID PXD0402123 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: ProteomeXchange, identifier: PXD0402123\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1139 - INFO - Found repo_name match for ProteomeXchange\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1164 - INFO - Resolved data repository: proteomecentral.proteomexchange.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'PXD0402123', 'data_repository': 'proteomecentral.proteomexchange.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'MSV000092944', 'repository_reference': 'https://massive.ucsd.edu/', 'dataset_id': 'MSV000092944'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: MSV000092944\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID MSV000092944 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://massive.ucsd.edu/, identifier: MSV000092944\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://massive.ucsd.edu/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Found partial match for https://massive.ucsd.edu/ in MassIVE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'MSV000092944', 'data_repository': 'massive.ucsd.edu', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn1827001', 'repository_reference': 'https://www.synapse.org/[#!]*Synapse:syn28580053', 'dataset_id': 'syn1827001'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn28580053\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn1827001, synapse.org, https://www.synapse.org/[#!]*Synapse:syn28580053\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn1827001\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn1827001', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://data.broadinstitute.org/ccle_legacy_data/mRNA_expression/', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://data.broadinstitute.org/ccle_legacy_data/mRNA_expression/, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://data.broadinstitute.org/ccle_legacy_data/mRNA_expression/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://data.broadinstitute.org/ccle_legacy_data/mRNA_expression/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'data.broadinstitute.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://estrogene.org/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://estrogene.org/, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://estrogene.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://estrogene.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'estrogene.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://docs.google.com/spreadsheets/d/1PFMGB_-COSrUujMKl_M-Ogkmq4cAyuIEgoLwSLFuYTs/edit#gid=0'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://docs.google.com/spreadsheets/d/1PFMGB_-COSrUujMKl_M-Ogkmq4cAyuIEgoLwSLFuYTs/edit#gid=0, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://docs.google.com/spreadsheets/d/1PFMGB_-COSrUujMKl_M-Ogkmq4cAyuIEgoLwSLFuYTs/edit#gid=0\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://docs.google.com/spreadsheets/d/1PFMGB_-COSrUujMKl_M-Ogkmq4cAyuIEgoLwSLFuYTs/edit#gid=0\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'docs.google.com', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 256 - INFO - Augmented dataset links: [{'dataset_identifier': 'GSE69091', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'EGAS00001000925', 'data_repository': 'ega-archive.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'PXD049309', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'PXD0402123', 'data_repository': 'proteomecentral.proteomexchange.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'MSV000092944', 'data_repository': 'massive.ucsd.edu', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn1827001', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'data.broadinstitute.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'estrogene.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'docs.google.com', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 9 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE69091\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE69091\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE69091 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE69091 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE69091 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE69091\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE69091\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: EGAS00001000925\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: ega-archive.org, identifier: EGAS00001000925\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: ega-archive.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for EGAS00001000925 in ega-archive.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 2 with repo: ega-archive.org and accession_id: EGAS00001000925\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://ega-archive.org/studies/EGAS00001000925\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 2: Contact to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: PXD049309\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: www.ebi.ac.uk, identifier: PXD049309\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for PXD049309 in www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 3 with repo: www.ebi.ac.uk and accession_id: PXD049309\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ebi.ac.uk/pride/archive/projects/PXD049309\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 3: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: PXD0402123\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: proteomecentral.proteomexchange.org, identifier: PXD0402123\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: proteomecentral.proteomexchange.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for PXD0402123 in proteomecentral.proteomexchange.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 4 with repo: proteomecentral.proteomexchange.org and accession_id: PXD0402123\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD0402123\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 4: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: MSV000092944\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: massive.ucsd.edu, identifier: MSV000092944\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: massive.ucsd.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for MSV000092944 in massive.ucsd.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 5 with repo: massive.ucsd.edu and accession_id: MSV000092944\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://massive.ucsd.edu/ProteoSAFe/dataset.jsp?accession=MSV000092944\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 5: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn1827001\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn1827001\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn1827001 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn1827001 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn1827001 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn1827001 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn1827001 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 6 with repo: synapse.org and accession_id: syn1827001\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn1827001\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 6: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Skipping dataset 7: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Skipping dataset 8: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Skipping dataset 9: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 9\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 265 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 12, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 477 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 29th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11420198)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11420198\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC11420198&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 406 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC11420198.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11420198 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 430 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11420198, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 443 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 264 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 454 - INFO - Found 14 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b7278cc0>, <Element supplementary-material at 0x1b716a100>, <Element supplementary-material at 0x1b71c2340>, <Element supplementary-material at 0x1b0969780>, <Element supplementary-material at 0x1b0271c40>, <Element supplementary-material at 0x1b0271d80>, <Element supplementary-material at 0x1b0272180>, <Element supplementary-material at 0x1b0271dc0>, <Element supplementary-material at 0x1b0271bc0>, <Element supplementary-material at 0x1b0271900>, <Element supplementary-material at 0x1b0132b80>, <Element supplementary-material at 0x1b02717c0>, <Element supplementary-material at 0x1b0271700>, <Element supplementary-material at 0x1b0271840>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 462 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b7278cc0>, <Element supplementary-material at 0x1b716a100>, <Element supplementary-material at 0x1b71c2340>, <Element supplementary-material at 0x1b0969780>, <Element supplementary-material at 0x1b0271c40>, <Element supplementary-material at 0x1b0271d80>, <Element supplementary-material at 0x1b0272180>, <Element supplementary-material at 0x1b0271dc0>, <Element supplementary-material at 0x1b0271bc0>, <Element supplementary-material at 0x1b0271900>, <Element supplementary-material at 0x1b0132b80>, <Element supplementary-material at 0x1b02717c0>, <Element supplementary-material at 0x1b0271700>, <Element supplementary-material at 0x1b0271840>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 548 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref No ID: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref No ID: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref No ID: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref No ID: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref No ID: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref No ID: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref No ID: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref No ID: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref No ID: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref No ID: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref No ID: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref No ID: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref No ID: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref No ID: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 243 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 40289\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 148881\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 41011 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-ef2bf1db8e25367e7f4d267f1277426ed577af405d81ea6ebf5e003b13db591f\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 41011 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\n",
      "  \"datasets\": [\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE249687\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE249687\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE84176\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84176\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE59119\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59119\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE96107\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE96107\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE126957\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE126957\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE142881\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE142881\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"syn4921369\",\n",
      "      \"repository_reference\": \"https://www.synapse.org/Synapse:syn4921369\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    }\n",
      "  ]\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'GSE249687', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE249687'}, {'dataset_identifier': 'GSE84176', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84176'}, {'dataset_identifier': 'GSE59119', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59119'}, {'dataset_identifier': 'GSE96107', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE96107'}, {'dataset_identifier': 'GSE126957', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE126957'}, {'dataset_identifier': 'GSE142881', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE142881'}, {'dataset_identifier': 'syn4921369', 'repository_reference': 'https://www.synapse.org/Synapse:syn4921369'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 8 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE249687\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE84176\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE59119\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE96107\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE126957\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE142881\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn4921369\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'GSE249687', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE249687', 'dataset_id': 'GSE249687'}, {'dataset_identifier': 'GSE84176', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84176', 'dataset_id': 'GSE84176'}, {'dataset_identifier': 'GSE59119', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59119', 'dataset_id': 'GSE59119'}, {'dataset_identifier': 'GSE96107', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE96107', 'dataset_id': 'GSE96107'}, {'dataset_identifier': 'GSE126957', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE126957', 'dataset_id': 'GSE126957'}, {'dataset_identifier': 'GSE142881', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE142881', 'dataset_id': 'GSE142881'}, {'dataset_identifier': 'syn4921369', 'repository_reference': 'https://www.synapse.org/Synapse:syn4921369', 'dataset_id': 'syn4921369'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE249687', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE249687', 'dataset_id': 'GSE249687'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE249687\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE249687, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE249687\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE249687\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE249687', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE84176', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84176', 'dataset_id': 'GSE84176'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE84176\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE84176, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84176\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE84176\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE84176', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE59119', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59119', 'dataset_id': 'GSE59119'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE59119\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE59119, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59119\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE59119\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE59119', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE96107', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE96107', 'dataset_id': 'GSE96107'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE96107\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE96107, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE96107\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE96107\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE96107', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE126957', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE126957', 'dataset_id': 'GSE126957'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE126957\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE126957, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE126957\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE126957\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE126957', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE142881', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE142881', 'dataset_id': 'GSE142881'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE142881\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE142881, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE142881\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE142881\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE142881', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn4921369', 'repository_reference': 'https://www.synapse.org/Synapse:syn4921369', 'dataset_id': 'syn4921369'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn4921369\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn4921369, synapse.org, https://www.synapse.org/Synapse:syn4921369\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn4921369\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn4921369', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 677 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 605 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 256 - INFO - Augmented dataset links: [{'dataset_identifier': 'GSE249687', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE84176', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE59119', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE96107', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE126957', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE142881', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn4921369', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 7 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE249687\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE249687\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE249687 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE249687 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE249687 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE249687\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE249687\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE84176\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE84176\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE84176 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE84176 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE84176 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 2 with repo: geo and accession_id: GSE84176\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84176\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 2: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE59119\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE59119\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE59119 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE59119 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE59119 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 3 with repo: geo and accession_id: GSE59119\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59119\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 3: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE96107\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE96107\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE96107 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE96107 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE96107 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 4 with repo: geo and accession_id: GSE96107\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE96107\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 4: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE126957\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE126957\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE126957 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE126957 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE126957 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 5 with repo: geo and accession_id: GSE126957\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE126957\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 5: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE142881\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE142881\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE142881 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE142881 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE142881 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 6 with repo: geo and accession_id: GSE142881\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE142881\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 6: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn4921369\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn4921369\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn4921369 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn4921369 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn4921369 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn4921369 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn4921369 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 7 with repo: synapse.org and accession_id: syn4921369\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn4921369\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 7: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 7\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 265 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 21, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 477 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 30th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC7865022)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC7865022\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC7865022&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 461 - WARNING - No links found in the section.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: supplementary_data_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 394 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC7865022.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b0a40fd0>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC7865022 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC7865022, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 445 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC7865022)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 547 - INFO - Extracted 6 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 557 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#Sec13'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 29) for Sec13: Usage Notes [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#Sec14'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for Sec14: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#Sec14'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for Sec14: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 50975\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 162080\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 51697 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-4ceed94635ad2d69d12b63a0fcb8a72bea24b743b6b1137cd956e43965d02722\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 51697 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\n",
      "  \"datasets\": [\n",
      "    {\n",
      "      \"dataset_identifier\": \"10.7303/syn20681023\",\n",
      "      \"repository_reference\": \"https://www.synapse.org/#!Synapse:syn20681023/wiki\"\n",
      "    }\n",
      "  ]\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': '10.7303/syn20681023', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn20681023/wiki'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 1 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: 10.7303/syn20681023\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': '10.7303/syn20681023', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn20681023/wiki', 'dataset_id': '10.7303/syn20681023'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': '10.7303/syn20681023', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn20681023/wiki', 'dataset_id': '10.7303/syn20681023'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn20681023\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: 10.7303/syn20681023, synapse.org, https://www.synapse.org/#!Synapse:syn20681023/wiki\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: 10.7303/syn20681023\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': '10.7303/syn20681023', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': '10.7303/syn20681023', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 1 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: 10.7303/syn20681023\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: 10.7303/syn20681023\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.7303/syn20681023 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 10.7303/syn20681023 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1153 - INFO - Found id_pattern match for synapse.org in 10\\.\\d{4,9}/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for 10.7303/syn20681023 in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 1 with repo: doi.org and accession_id: 10.7303/syn20681023\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://doi.org/10.7303/syn20681023\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 1\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 7, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage', 'link',\n",
      "       'title', 'file_info', 'description', 'source_section', 'section_class',\n",
      "       'download_link', 'file_extension', 'a_attr_href', 'a_attr_class',\n",
      "       'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 640 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - Parsed data extraction completed. Elements collected: 2\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 481 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 574 - INFO - \n",
      "Progress: 31/100 (31.00%) | Elapsed: 00:08:27 | ETA: 00:18:48\n",
      "\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 31th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC8131595)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC8131595\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC8131595&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 406 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC8131595.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC8131595 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 430 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC8131595, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 443 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 264 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 454 - INFO - Found 3 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b6f2b580>, <Element supplementary-material at 0x1b731bd40>, <Element supplementary-material at 0x1b731bd00>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 462 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b6f2b580>, <Element supplementary-material at 0x1b731bd40>, <Element supplementary-material at 0x1b731bd00>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 548 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM1: We sourced frozen post-mortem prefrontal cortex from four CdLS patients aged 19–48 years and six age-matched controls (Supplementary Fig. 1a [this file] ). Three of the patients had heterozygous mutations in , the gene most frequently mutated in CdLS (ref. , Fig. 1a [rid=Fig1] , Supplementary Fig. 1a, b [this file] ). \n",
      "To characterise gene expression in CdLS patient neurons we isolated nuclei from the prefrontal cortex, stained with the neuronal marker NeuN (ref. ), and sorted NeuN positive and NeuN negative nuclei by flow cytometry (Fig. 1b [rid=Fig1] , Supplementary Fig. 1c [this file] ). However, CAGE analysis of promoters showed separation of CdLS patient and control samples along the major principle component PC1, and revealed significant deregulation of 766 gene promoters in CdLS neurons (adj < 0.05; 358 upregulated, 408 downregulated; Supplementary Fig. 2a–c [this file] , Supplementary Data 1 [this file] ). Gene promoters that were downregulated in CdLS neurons were enriched for the gene ontology (GO) terms synaptic signalling and organization, ion and neurotransmitter transport, axon development, behavior and cognition, while upregulated promoters showed no striking functional enrichment (Supplementary Fig. 2d [this file] , Supplementary Data 2 [this file] ). RNA-seq identified 617 differentially expressed genes (adj < 0.05) in CdLS neurons, 310 genes were up- and 307 were downregulated (Fig. 1c [rid=Fig1] , Supplementary Fig. 3a [this file] ). In close agreement with CAGE, genes that were downregulated by RNA-seq were related to synaptic transmission, signalling processes, learning and behaviour, and sphingolipid metabolism, while upregulated genes showed no striking functional enrichment (Fig. 1d [rid=Fig1] , Supplementary Data 4 [this file] ). \n",
      "The direction of gene deregulation by RNA-seq and CAGE was highly correlated ( = 0.68, = < 2.2e-16 for genes found deregulated by RNA-seq or CAGE, and = 0.95, = < 2.2e-16 for genes found deregulated by both RNA-seq and CAGE; Supplementary Fig. 2e [this file] ). \n",
      "RNA-seq of NeuN-negative nuclei showed upregulation of immune response genes that were not seen in NeuN-positive nuclei, including interferon response, JAK-STAT signalling, and the complement pathway (Supplementary Fig. 4 [this file] , Supplementary Data 5 [this file] , 6 [this file] ). There was significant overlap between changes in gene expression in human CdLS with published gene expression in heterozygous mouse brain at embryonic day 13.5 (e13.5; whole brain versus human CdLS NeuN positive and NeuN negative nuclei combined; = 1.32e-14, Odds ratio = 2.05) and with gene expression in cortical neurons in explant culture ( cortical neurons versus human CdLS NeuN positive neurons; Supplementary Data 7 [this file] ; = 8.38e-06, Odds ratio = 3.03). \n",
      "We established explant cultures from cortices at e14.5 under conditions that enrich for post-mitotic neurons and promote neuronal maturation (Fig. 2a [rid=Fig2] , Supplementary Fig. 5a, b [this file] ). \n",
      "Constructs were tagged by V5 and t2a-fused GFP (ERt2-TEV) for identification of ERt2-TEV-expressing cells (Fig. 2b [rid=Fig2] , Supplementary Fig. 5c [this file] ). ERt2-TEV remained cytoplasmic until nuclear translocation of ERt2-TEV was triggered, which occurred within minutes of 4-hydroxy tamoxifen (4-OHT) addition (Fig. 2b [rid=Fig2] , Supplementary Fig. 5c [this file] ). Addition of doxycycline leads to the expression of TEV protease with an exogeneous nuclear localization sequence (NLS-TEV, Supplementary Fig. 6a, b [this file] ). Induction of NLS-TEV depleted ~70% of RAD21-TEV within 24 h of doxycycline addition (Supplementary Fig. 6c [this file] ). \n",
      "RNA-seq 24 h after 4-OHT-induced nuclear translocation of ERt2-TEV identified 750 deregulated (adj < 0.05) genes, of which 463 were down- and 287 were upregulated (Fig. 2d [rid=Fig2] , Supplementary Data 8 [this file] ). As observed in CdLS neurons, no striking enrichment was observed for upregulated genes (Fig. 2e [rid=Fig2] , Supplementary Data 9 [this file] ). \n",
      "As observed for the nuclear translocation of ERt2-TEV, downregulated genes were enriched for adhesion, signalling, synaptic function and development (Supplementary Fig. 5d, e [this file] , Supplementary Data 10 [this file] , 11 [this file] ). There was extensive overlap between genes deregulated by acute cohesin depletion with genes deregulated in neurons (this study) and embryonic brain , but less so with genes deregulated in other mouse models of neuronal dysfunction (Supplementary Fig. 7 [this file] ). \n",
      "Reduced mRNA expression translated into reduced levels of protein in RAD21-TEV neurons, as illustrated for NLGN1, encoded by (Supplementary Fig. 8 [this file] , Supplementary Data 12 [this file] ). \n",
      "These shared ASD genes mediate important neuronal functions in cell adhesion ( ) signalling ( ), ion channels ( ), synaptic transmission ( ), transcription ( ) and sphingolipid metabolism ( ) (Fig. 3b [rid=Fig3] , Supplementary Table 1 [this file] ). \n",
      "Gene expression changes in Rett syndrome, but not other human neurological diseases , showed significant overlap with neurons (Odds ratio = 2.06, = 4.64e-10; Supplementary Fig. 9 [this file] ). \n",
      "Remarkably, comparison of RNA-seq during RAD21-TEV depletion (24 h) and after RAD21-TEV rescue (day 7) showed that the vast majority of genes that were found deregulated after 24 h (695 of 720 or 96.5%) were fully rescued upon restoration of cohesin levels (Fig. 4c, e [rid=Fig4] ; Supplementary Data 13 [this file] ). These experiments confirmed that rescue of gene expression required the restoration of RAD21 expression and was not due to the adaptation of neurons to reduced cohesin expression (Supplementary Fig 10 [this file] ; Supplementary Data 14 [this file] ). \n",
      "These included 14 genes implicated in ASD and shared between CdLS and NLS-TEV (Supplementary Table 1 [this file] ). \n",
      "Sequence data was analysed using the 4 Cseqpipe software suite with the setting of “-stat_type mean -trend_resolution 40,000”. 4C-seq primers are listed in Supplementary Data 15 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM2: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM3: Further information on research design is available in the Nature Research Reporting Summary [this file] linked to this article. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 243 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 51069\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 195469\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 51791 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-ec7d94cdb820682e9cd60273f31b6a65cef32851c5934d07f448b3a0ab9d8d78\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 51791 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\n",
      "  \"datasets\": [\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE150130\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE150130\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"E-MTAB-9045\",\n",
      "      \"repository_reference\": \"https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-9045/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"syn4587609\",\n",
      "      \"repository_reference\": \"https://www.synapse.org/#!Synapse:syn4587609\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE96107\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE96107\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"SRX3381923\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/sra/SRX3381923\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"ENCSR677HXC\",\n",
      "      \"repository_reference\": \"https://www.encodeproject.org/experiments/ENCSR677HXC/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE60192\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60192\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"ENCSR198ZYJ\",\n",
      "      \"repository_reference\": \"https://www.encodeproject.org/experiments/ENCSR198ZYJ/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"ENCSR822CEA\",\n",
      "      \"repository_reference\": \"https://www.encodeproject.org/experiments/ENCSR822CEA/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"phs001373.v2.p2\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001373.v2.p2\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE84245\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84245\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE113673\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE113673\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE81912\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE81912\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE65776\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65776\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE59318\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59318\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE59630\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59630\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    }\n",
      "  ]\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'GSE150130', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE150130'}, {'dataset_identifier': 'E-MTAB-9045', 'repository_reference': 'https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-9045/'}, {'dataset_identifier': 'syn4587609', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn4587609'}, {'dataset_identifier': 'GSE96107', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE96107'}, {'dataset_identifier': 'SRX3381923', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/sra/SRX3381923'}, {'dataset_identifier': 'ENCSR677HXC', 'repository_reference': 'https://www.encodeproject.org/experiments/ENCSR677HXC/'}, {'dataset_identifier': 'GSE60192', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60192'}, {'dataset_identifier': 'ENCSR198ZYJ', 'repository_reference': 'https://www.encodeproject.org/experiments/ENCSR198ZYJ/'}, {'dataset_identifier': 'ENCSR822CEA', 'repository_reference': 'https://www.encodeproject.org/experiments/ENCSR822CEA/'}, {'dataset_identifier': 'phs001373.v2.p2', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001373.v2.p2'}, {'dataset_identifier': 'GSE84245', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84245'}, {'dataset_identifier': 'GSE113673', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE113673'}, {'dataset_identifier': 'GSE81912', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE81912'}, {'dataset_identifier': 'GSE65776', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65776'}, {'dataset_identifier': 'GSE59318', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59318'}, {'dataset_identifier': 'GSE59630', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59630'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 17 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE150130\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: E-MTAB-9045\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE96107\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: SRX3381923\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: ENCSR677HXC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE60192\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: ENCSR198ZYJ\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: ENCSR822CEA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: phs001373.v2.p2\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE84245\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE113673\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE81912\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE65776\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE59318\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE59630\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'GSE150130', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE150130', 'dataset_id': 'GSE150130'}, {'dataset_identifier': 'E-MTAB-9045', 'repository_reference': 'https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-9045/', 'dataset_id': 'E-MTAB-9045'}, {'dataset_identifier': 'syn4587609', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn4587609', 'dataset_id': 'syn4587609'}, {'dataset_identifier': 'GSE96107', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE96107', 'dataset_id': 'GSE96107'}, {'dataset_identifier': 'SRX3381923', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/sra/SRX3381923', 'dataset_id': 'SRX3381923'}, {'dataset_identifier': 'ENCSR677HXC', 'repository_reference': 'https://www.encodeproject.org/experiments/ENCSR677HXC/', 'dataset_id': 'ENCSR677HXC'}, {'dataset_identifier': 'GSE60192', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60192', 'dataset_id': 'GSE60192'}, {'dataset_identifier': 'ENCSR198ZYJ', 'repository_reference': 'https://www.encodeproject.org/experiments/ENCSR198ZYJ/', 'dataset_id': 'ENCSR198ZYJ'}, {'dataset_identifier': 'ENCSR822CEA', 'repository_reference': 'https://www.encodeproject.org/experiments/ENCSR822CEA/', 'dataset_id': 'ENCSR822CEA'}, {'dataset_identifier': 'phs001373.v2.p2', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001373.v2.p2', 'dataset_id': 'phs001373.v2.p2'}, {'dataset_identifier': 'GSE84245', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84245', 'dataset_id': 'GSE84245'}, {'dataset_identifier': 'GSE113673', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE113673', 'dataset_id': 'GSE113673'}, {'dataset_identifier': 'GSE81912', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE81912', 'dataset_id': 'GSE81912'}, {'dataset_identifier': 'GSE65776', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65776', 'dataset_id': 'GSE65776'}, {'dataset_identifier': 'GSE59318', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59318', 'dataset_id': 'GSE59318'}, {'dataset_identifier': 'GSE59630', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59630', 'dataset_id': 'GSE59630'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE150130', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE150130', 'dataset_id': 'GSE150130'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE150130\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE150130, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE150130\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE150130\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE150130', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'E-MTAB-9045', 'repository_reference': 'https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-9045/', 'dataset_id': 'E-MTAB-9045'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: E-MTAB-9045\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID E-MTAB-9045 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-9045/, identifier: E-MTAB-9045\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-9045/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-MTAB-9045 for https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-9045/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-MTAB-9045 for https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-9045/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-MTAB-9045 for https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-9045/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-MTAB-9045 for https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-9045/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Found partial match for https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-9045/ in ArrayExpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'E-MTAB-9045', 'data_repository': 'arrayexpress', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn4587609', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn4587609', 'dataset_id': 'syn4587609'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn4587609, synapse.org, https://www.synapse.org/#!Synapse:syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn4587609', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE96107', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE96107', 'dataset_id': 'GSE96107'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE96107\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE96107, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE96107\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE96107\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE96107', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'SRX3381923', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/sra/SRX3381923', 'dataset_id': 'SRX3381923'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: SRX3381923\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID SRX3381923 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.ncbi.nlm.nih.gov/sra/SRX3381923, identifier: SRX3381923\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.ncbi.nlm.nih.gov/sra/SRX3381923\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier SRX3381923 for https://www.ncbi.nlm.nih.gov/sra/SRX3381923\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier SRX3381923 for https://www.ncbi.nlm.nih.gov/sra/SRX3381923\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier SRX3381923 for https://www.ncbi.nlm.nih.gov/sra/SRX3381923\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier SRX3381923 for https://www.ncbi.nlm.nih.gov/sra/SRX3381923\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier SRX3381923 for https://www.ncbi.nlm.nih.gov/sra/SRX3381923\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://www.ncbi.nlm.nih.gov/sra/SRX3381923\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'SRX3381923', 'data_repository': 'www.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'ENCSR677HXC', 'repository_reference': 'https://www.encodeproject.org/experiments/ENCSR677HXC/', 'dataset_id': 'ENCSR677HXC'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: ENCSR677HXC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID ENCSR677HXC is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.encodeproject.org/experiments/ENCSR677HXC/, identifier: ENCSR677HXC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.encodeproject.org/experiments/ENCSR677HXC/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier ENCSR677HXC for https://www.encodeproject.org/experiments/ENCSR677HXC/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier ENCSR677HXC for https://www.encodeproject.org/experiments/ENCSR677HXC/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier ENCSR677HXC for https://www.encodeproject.org/experiments/ENCSR677HXC/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier ENCSR677HXC for https://www.encodeproject.org/experiments/ENCSR677HXC/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier ENCSR677HXC for https://www.encodeproject.org/experiments/ENCSR677HXC/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://www.encodeproject.org/experiments/ENCSR677HXC/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'ENCSR677HXC', 'data_repository': 'www.encodeproject.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE60192', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60192', 'dataset_id': 'GSE60192'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE60192\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE60192, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60192\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE60192\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE60192', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'ENCSR198ZYJ', 'repository_reference': 'https://www.encodeproject.org/experiments/ENCSR198ZYJ/', 'dataset_id': 'ENCSR198ZYJ'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: ENCSR198ZYJ\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID ENCSR198ZYJ is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.encodeproject.org/experiments/ENCSR198ZYJ/, identifier: ENCSR198ZYJ\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.encodeproject.org/experiments/ENCSR198ZYJ/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier ENCSR198ZYJ for https://www.encodeproject.org/experiments/ENCSR198ZYJ/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier ENCSR198ZYJ for https://www.encodeproject.org/experiments/ENCSR198ZYJ/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier ENCSR198ZYJ for https://www.encodeproject.org/experiments/ENCSR198ZYJ/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier ENCSR198ZYJ for https://www.encodeproject.org/experiments/ENCSR198ZYJ/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier ENCSR198ZYJ for https://www.encodeproject.org/experiments/ENCSR198ZYJ/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://www.encodeproject.org/experiments/ENCSR198ZYJ/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'ENCSR198ZYJ', 'data_repository': 'www.encodeproject.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'ENCSR822CEA', 'repository_reference': 'https://www.encodeproject.org/experiments/ENCSR822CEA/', 'dataset_id': 'ENCSR822CEA'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: ENCSR822CEA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID ENCSR822CEA is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.encodeproject.org/experiments/ENCSR822CEA/, identifier: ENCSR822CEA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.encodeproject.org/experiments/ENCSR822CEA/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier ENCSR822CEA for https://www.encodeproject.org/experiments/ENCSR822CEA/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier ENCSR822CEA for https://www.encodeproject.org/experiments/ENCSR822CEA/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier ENCSR822CEA for https://www.encodeproject.org/experiments/ENCSR822CEA/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier ENCSR822CEA for https://www.encodeproject.org/experiments/ENCSR822CEA/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier ENCSR822CEA for https://www.encodeproject.org/experiments/ENCSR822CEA/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://www.encodeproject.org/experiments/ENCSR822CEA/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'ENCSR822CEA', 'data_repository': 'www.encodeproject.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'phs001373.v2.p2', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001373.v2.p2', 'dataset_id': 'phs001373.v2.p2'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: phs001373.v2.p2\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: phs001373.v2.p2, dbgap, https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001373.v2.p2\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: phs001373.v2.p2\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: dbgap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'phs001373.v2.p2', 'data_repository': 'dbgap', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE84245', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84245', 'dataset_id': 'GSE84245'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE84245\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE84245, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84245\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE84245\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE84245', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE113673', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE113673', 'dataset_id': 'GSE113673'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE113673\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE113673, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE113673\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE113673\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE113673', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE81912', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE81912', 'dataset_id': 'GSE81912'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE81912\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE81912, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE81912\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE81912\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE81912', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE65776', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65776', 'dataset_id': 'GSE65776'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE65776\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE65776, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65776\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE65776\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE65776', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE59318', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59318', 'dataset_id': 'GSE59318'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE59318\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE59318, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59318\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE59318\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE59318', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE59630', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59630', 'dataset_id': 'GSE59630'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE59630\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE59630, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59630\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE59630\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE59630', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 677 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 605 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 256 - INFO - Augmented dataset links: [{'dataset_identifier': 'GSE150130', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'E-MTAB-9045', 'data_repository': 'arrayexpress', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn4587609', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE96107', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'SRX3381923', 'data_repository': 'www.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'ENCSR677HXC', 'data_repository': 'www.encodeproject.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE60192', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'ENCSR198ZYJ', 'data_repository': 'www.encodeproject.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'ENCSR822CEA', 'data_repository': 'www.encodeproject.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'phs001373.v2.p2', 'data_repository': 'dbgap', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE84245', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE113673', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE81912', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE65776', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE59318', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE59630', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 16 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE150130\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE150130\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE150130 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE150130 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE150130 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE150130\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE150130\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: E-MTAB-9045\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: arrayexpress, identifier: E-MTAB-9045\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-MTAB-9045 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-MTAB-9045 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-MTAB-9045 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-MTAB-9045 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for E-MTAB-9045 in arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 2 with repo: arrayexpress and accession_id: E-MTAB-9045\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ebi.ac.uk/biostudies/ArrayExpress/studies/E-MTAB-9045\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 2: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn4587609 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn4587609 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn4587609 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn4587609 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn4587609 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 3 with repo: synapse.org and accession_id: syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 3: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE96107\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE96107\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE96107 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE96107 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE96107 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 4 with repo: geo and accession_id: GSE96107\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE96107\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 4: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: SRX3381923\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: www.ncbi.nlm.nih.gov, identifier: SRX3381923\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier SRX3381923 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier SRX3381923 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier SRX3381923 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier SRX3381923 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier SRX3381923 for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 976 - WARNING - URL www.ncbi.nlm.nih.gov may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for SRX3381923 in www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 5 with repo: www.ncbi.nlm.nih.gov and accession_id: SRX3381923\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1254 - WARNING - Repository www.ncbi.nlm.nih.gov unknown in Ontology. Skipping dataset page 5.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: ENCSR677HXC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: www.encodeproject.org, identifier: ENCSR677HXC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier ENCSR677HXC for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier ENCSR677HXC for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier ENCSR677HXC for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier ENCSR677HXC for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier ENCSR677HXC for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 976 - WARNING - URL www.encodeproject.org may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for ENCSR677HXC in www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 6 with repo: www.encodeproject.org and accession_id: ENCSR677HXC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1254 - WARNING - Repository www.encodeproject.org unknown in Ontology. Skipping dataset page 6.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE60192\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE60192\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE60192 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE60192 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE60192 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 7 with repo: geo and accession_id: GSE60192\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60192\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 7: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: ENCSR198ZYJ\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: www.encodeproject.org, identifier: ENCSR198ZYJ\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier ENCSR198ZYJ for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier ENCSR198ZYJ for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier ENCSR198ZYJ for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier ENCSR198ZYJ for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier ENCSR198ZYJ for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 976 - WARNING - URL www.encodeproject.org may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for ENCSR198ZYJ in www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 8 with repo: www.encodeproject.org and accession_id: ENCSR198ZYJ\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1254 - WARNING - Repository www.encodeproject.org unknown in Ontology. Skipping dataset page 8.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: ENCSR822CEA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: www.encodeproject.org, identifier: ENCSR822CEA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier ENCSR822CEA for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier ENCSR822CEA for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier ENCSR822CEA for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier ENCSR822CEA for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier ENCSR822CEA for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 976 - WARNING - URL www.encodeproject.org may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for ENCSR822CEA in www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 9 with repo: www.encodeproject.org and accession_id: ENCSR822CEA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1254 - WARNING - Repository www.encodeproject.org unknown in Ontology. Skipping dataset page 9.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 10 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: phs001373.v2.p2\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: dbgap, identifier: phs001373.v2.p2\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier phs001373.v2.p2 for dbgap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier phs001373.v2.p2 for dbgap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier phs001373.v2.p2 for dbgap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: dbgap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for phs001373.v2.p2 in dbgap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 10 with repo: dbgap and accession_id: phs001373.v2.p2.v1.p1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001373.v2.p2.v1.p1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 10: Application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 11 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE84245\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE84245\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE84245 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE84245 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE84245 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 11 with repo: geo and accession_id: GSE84245\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84245\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 11: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 12 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE113673\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE113673\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE113673 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE113673 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE113673 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 12 with repo: geo and accession_id: GSE113673\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE113673\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 12: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 13 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE81912\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE81912\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE81912 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE81912 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE81912 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 13 with repo: geo and accession_id: GSE81912\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE81912\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 13: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 14 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE65776\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE65776\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE65776 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE65776 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE65776 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 14 with repo: geo and accession_id: GSE65776\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65776\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 14: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 15 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE59318\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE59318\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE59318 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE59318 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE59318 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 15 with repo: geo and accession_id: GSE59318\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59318\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 15: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 16 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE59630\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE59630\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE59630 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE59630 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE59630 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 16 with repo: geo and accession_id: GSE59630\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59630\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 16: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 16\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 265 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 19, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 477 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 32th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10828232)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10828232\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC10828232&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 461 - WARNING - No links found in the section.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 406 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC10828232.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC10828232 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 430 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC10828232, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 443 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 264 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 454 - INFO - Found 4 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b64ee500>, <Element supplementary-material at 0x1b6437ec0>, <Element supplementary-material at 0x1b6437c80>, <Element supplementary-material at 0x1b6435c00>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 462 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b64ee500>, <Element supplementary-material at 0x1b6437ec0>, <Element supplementary-material at 0x1b6437c80>, <Element supplementary-material at 0x1b6435c00>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 548 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM1: The sequence of the specific primers used is provided in Supplemental Table 1 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM2: RNA-sequencing analysis of i-astrocyte cultures 72 h after transduction identified a total of 19,778 unique transcripts (Supplemental Table 2 [this file] ). Statistical analysis identified 500 transcripts with at least a 2-fold change in expression (adjusted ≤ 0.05) (Supplemental Table 2 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM3: Representative images obtained from sections where the primary antibodies were omitted are shown in Supplementary Figure 1 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM4: Differentiation and maturation of i-astrocyte cultures were monitored by real-time PCR and immunostaining for selected markers (Supplementary Fig. 2 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 243 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 49076\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 200008\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 49798 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-925a6296497a7add5048d236c6f1e883d8e2fa43a7fc61691a4a43f4b4baa222\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 49798 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\n",
      "  \"datasets\": [\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE214628\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE214628\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"syn13363290-v49\",\n",
      "      \"repository_reference\": \"https://agora.adknowledgeportal.org/genes/ENSG00000164434/evidence/protein\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    }\n",
      "  ]\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'GSE214628', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE214628'}, {'dataset_identifier': 'syn13363290-v49', 'repository_reference': 'https://agora.adknowledgeportal.org/genes/ENSG00000164434/evidence/protein'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 3 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE214628\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn13363290-v49\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'GSE214628', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE214628', 'dataset_id': 'GSE214628'}, {'dataset_identifier': 'syn13363290-v49', 'repository_reference': 'https://agora.adknowledgeportal.org/genes/ENSG00000164434/evidence/protein', 'dataset_id': 'syn13363290-v49'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE214628', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE214628', 'dataset_id': 'GSE214628'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE214628\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE214628, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE214628\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE214628\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE214628', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn13363290-v49', 'repository_reference': 'https://agora.adknowledgeportal.org/genes/ENSG00000164434/evidence/protein', 'dataset_id': 'syn13363290-v49'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn13363290-v49, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn13363290-v49\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn13363290-v49', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 677 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 605 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 256 - INFO - Augmented dataset links: [{'dataset_identifier': 'GSE214628', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn13363290-v49', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 2 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE214628\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE214628\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE214628 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE214628 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE214628 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE214628\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE214628\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn13363290-v49\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn13363290-v49\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn13363290-v49 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn13363290-v49 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn13363290-v49 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn13363290-v49 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn13363290-v49 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn13363290-v49\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn13363290-v49\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 2\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 265 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 6, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 477 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 33th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11852220)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11852220\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC11852220&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 461 - WARNING - No links found in the section.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <supplementary-material xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.or...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 406 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC11852220.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11852220 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 430 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11852220, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 443 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 264 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 454 - INFO - Found 1 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b0276280>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 462 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b0276280>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 548 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref bioengineering-12-00126-s001: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 243 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 42458\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 179116\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 43180 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-f2c73478c615391b4d2d4d507b980d77b5583b2006f5d9d2255414bf9b671341\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 43180 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\n",
      "  \"datasets\": [\n",
      "    {\n",
      "      \"dataset_identifier\": \"UKB-PPP\",\n",
      "      \"repository_reference\": \"https://www.synapse.org/#!Synapse:syn51365301\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"FinnGen\",\n",
      "      \"repository_reference\": \"https://r9.finngen.fi\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"DsigDB\",\n",
      "      \"repository_reference\": \"http://dsigdb.tanlab.org/DSigDBv1.0/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"PubChem\",\n",
      "      \"repository_reference\": \"https://pubchem.ncbi.nlm.nih.gov/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"RCSB\",\n",
      "      \"repository_reference\": \"https://www.rcsb.org/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    }\n",
      "  ]\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'UKB-PPP', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn51365301'}, {'dataset_identifier': 'FinnGen', 'repository_reference': 'https://r9.finngen.fi'}, {'dataset_identifier': 'DsigDB', 'repository_reference': 'http://dsigdb.tanlab.org/DSigDBv1.0/'}, {'dataset_identifier': 'PubChem', 'repository_reference': 'https://pubchem.ncbi.nlm.nih.gov/'}, {'dataset_identifier': 'RCSB', 'repository_reference': 'https://www.rcsb.org/'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 6 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: UKB-PPP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: FinnGen\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: DsigDB\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: PubChem\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: RCSB\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'UKB-PPP', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn51365301', 'dataset_id': 'UKB-PPP'}, {'dataset_identifier': 'FinnGen', 'repository_reference': 'https://r9.finngen.fi', 'dataset_id': 'FinnGen'}, {'dataset_identifier': 'DsigDB', 'repository_reference': 'http://dsigdb.tanlab.org/DSigDBv1.0/', 'dataset_id': 'DsigDB'}, {'dataset_identifier': 'PubChem', 'repository_reference': 'https://pubchem.ncbi.nlm.nih.gov/', 'dataset_id': 'PubChem'}, {'dataset_identifier': 'RCSB', 'repository_reference': 'https://www.rcsb.org/', 'dataset_id': 'RCSB'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'UKB-PPP', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn51365301', 'dataset_id': 'UKB-PPP'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn51365301\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn51365301, None, https://www.synapse.org/#!Synapse:syn51365301\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn51365301\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn51365301, identifier: syn51365301\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn51365301\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn51365301 for https://www.synapse.org/#!Synapse:syn51365301\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn51365301 for https://www.synapse.org/#!Synapse:syn51365301\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn51365301 for https://www.synapse.org/#!Synapse:syn51365301\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn51365301 for https://www.synapse.org/#!Synapse:syn51365301\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn51365301 in Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn51365301', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'FinnGen', 'repository_reference': 'https://r9.finngen.fi', 'dataset_id': 'FinnGen'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: FinnGen\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID FinnGen is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://r9.finngen.fi, identifier: FinnGen\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://r9.finngen.fi\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier FinnGen for https://r9.finngen.fi\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier FinnGen for https://r9.finngen.fi\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier FinnGen for https://r9.finngen.fi\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier FinnGen for https://r9.finngen.fi\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier FinnGen for https://r9.finngen.fi\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://r9.finngen.fi\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'FinnGen', 'data_repository': 'r9.finngen.fi', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'DsigDB', 'repository_reference': 'http://dsigdb.tanlab.org/DSigDBv1.0/', 'dataset_id': 'DsigDB'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: DsigDB\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID DsigDB is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: http://dsigdb.tanlab.org/DSigDBv1.0/, identifier: DsigDB\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: http://dsigdb.tanlab.org/DSigDBv1.0/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier DsigDB for http://dsigdb.tanlab.org/DSigDBv1.0/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier DsigDB for http://dsigdb.tanlab.org/DSigDBv1.0/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier DsigDB for http://dsigdb.tanlab.org/DSigDBv1.0/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier DsigDB for http://dsigdb.tanlab.org/DSigDBv1.0/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier DsigDB for http://dsigdb.tanlab.org/DSigDBv1.0/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: http://dsigdb.tanlab.org/DSigDBv1.0/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'DsigDB', 'data_repository': 'dsigdb.tanlab.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'PubChem', 'repository_reference': 'https://pubchem.ncbi.nlm.nih.gov/', 'dataset_id': 'PubChem'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: PubChem\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID PubChem is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://pubchem.ncbi.nlm.nih.gov/, identifier: PubChem\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://pubchem.ncbi.nlm.nih.gov/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier PubChem for https://pubchem.ncbi.nlm.nih.gov/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier PubChem for https://pubchem.ncbi.nlm.nih.gov/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier PubChem for https://pubchem.ncbi.nlm.nih.gov/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier PubChem for https://pubchem.ncbi.nlm.nih.gov/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier PubChem for https://pubchem.ncbi.nlm.nih.gov/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://pubchem.ncbi.nlm.nih.gov/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'PubChem', 'data_repository': 'pubchem.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'RCSB', 'repository_reference': 'https://www.rcsb.org/', 'dataset_id': 'RCSB'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: RCSB\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID RCSB is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.rcsb.org/, identifier: RCSB\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.rcsb.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier RCSB for https://www.rcsb.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier RCSB for https://www.rcsb.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier RCSB for https://www.rcsb.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier RCSB for https://www.rcsb.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier RCSB for https://www.rcsb.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://www.rcsb.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'RCSB', 'data_repository': 'www.rcsb.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 677 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 605 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 256 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn51365301', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'FinnGen', 'data_repository': 'r9.finngen.fi', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'DsigDB', 'data_repository': 'dsigdb.tanlab.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'PubChem', 'data_repository': 'pubchem.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'RCSB', 'data_repository': 'www.rcsb.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 5 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn51365301\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn51365301\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn51365301 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn51365301 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn51365301 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn51365301 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn51365301 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn51365301\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn51365301\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: FinnGen\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: r9.finngen.fi, identifier: FinnGen\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier FinnGen for r9.finngen.fi\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier FinnGen for r9.finngen.fi\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier FinnGen for r9.finngen.fi\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier FinnGen for r9.finngen.fi\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier FinnGen for r9.finngen.fi\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: r9.finngen.fi\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 976 - WARNING - URL r9.finngen.fi may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for FinnGen in r9.finngen.fi\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 2 with repo: r9.finngen.fi and accession_id: FinnGen\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1254 - WARNING - Repository r9.finngen.fi unknown in Ontology. Skipping dataset page 2.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: DsigDB\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: dsigdb.tanlab.org, identifier: DsigDB\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier DsigDB for dsigdb.tanlab.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier DsigDB for dsigdb.tanlab.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier DsigDB for dsigdb.tanlab.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier DsigDB for dsigdb.tanlab.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier DsigDB for dsigdb.tanlab.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: dsigdb.tanlab.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 976 - WARNING - URL dsigdb.tanlab.org may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for DsigDB in dsigdb.tanlab.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 3 with repo: dsigdb.tanlab.org and accession_id: DsigDB\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1254 - WARNING - Repository dsigdb.tanlab.org unknown in Ontology. Skipping dataset page 3.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: PubChem\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: pubchem.ncbi.nlm.nih.gov, identifier: PubChem\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier PubChem for pubchem.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier PubChem for pubchem.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier PubChem for pubchem.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier PubChem for pubchem.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier PubChem for pubchem.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: pubchem.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 976 - WARNING - URL pubchem.ncbi.nlm.nih.gov may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for PubChem in pubchem.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 4 with repo: pubchem.ncbi.nlm.nih.gov and accession_id: PubChem\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1254 - WARNING - Repository pubchem.ncbi.nlm.nih.gov unknown in Ontology. Skipping dataset page 4.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: RCSB\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: www.rcsb.org, identifier: RCSB\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier RCSB for www.rcsb.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier RCSB for www.rcsb.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier RCSB for www.rcsb.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier RCSB for www.rcsb.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier RCSB for www.rcsb.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: www.rcsb.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 976 - WARNING - URL www.rcsb.org may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for RCSB in www.rcsb.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 5 with repo: www.rcsb.org and accession_id: RCSB\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1254 - WARNING - Repository www.rcsb.org unknown in Ontology. Skipping dataset page 5.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 5\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 265 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 6, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 477 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 34th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC9314356)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC9314356\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC9314356&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 406 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC9314356.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC9314356 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 430 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC9314356, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 443 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 264 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 454 - INFO - Found 1 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b6409200>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 462 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b6409200>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 548 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM1: The parameters of each analysis step are summarized in Supplementary Table 3 [this file] . \n",
      "Summary statistics of age-at-onset (AAO) for LOAD cases and age-at-last-examination (AAE) for normal controls are presented in Supplementary Table 1 [this file] . The full name and accessibility of each study can be found in Supplementary Table 2 [this file] . \n",
      "The variance in AD explained by PRS maximizes at a -value threshold of 0.5 in participants from GERAD (Genetic and Environmental Risk for Alzheimer’s disease) [ 23 [rid=CR23] ] and 22 locally available ADGC (Alzheimer’s Disease Genetics Consortium) studies (Supplementary Fig. 4 [this file] ). \n",
      "Because our analytic approaches used only subsets of all available LOAD case–control GWAS data, we neither had nor anticipated having sufficient power to detect individual SNPs with genome-wide significant association with resilience (Supplementary Fig. 3 [this file] ). \n",
      "In the full samples from three locally downloaded replication studies (Alzheimer Disease Centers Wave 7 [ADC7], AddNeuroMed, and Alzheimer’s Disease Neuroimaging Initiative stage GO/2/3 [ADNI-GO/2/3]; normal controls, = 1321; LOAD cases, = 943) (Table 1 [rid=Tab1] and Supplementary Table 1 [this file] ), we tested for correlations between PRSs and polygenic resilience scores. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 243 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 61342\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 245965\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 62064 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-a844e209689cd5321f41bc0304507ab85f299f523eddd37e12dfadffb4be6c61\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 62064 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\n",
      "  \"datasets\": [\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"https://www.niagads.org\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"http://adni.loni.usc.edu\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"syn4907804\",\n",
      "      \"repository_reference\": \"https://www.synapse.org/#!Synapse:syn4907804\"\n",
      "    }\n",
      "  ]\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'n/a', 'repository_reference': 'https://www.niagads.org'}, {'dataset_identifier': 'n/a', 'repository_reference': 'http://adni.loni.usc.edu'}, {'dataset_identifier': 'syn4907804', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn4907804'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 3 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn4907804\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'n/a', 'repository_reference': 'https://www.niagads.org', 'dataset_id': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'http://adni.loni.usc.edu'}, {'dataset_identifier': 'syn4907804', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn4907804', 'dataset_id': 'syn4907804'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.niagads.org', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.niagads.org, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.niagads.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://www.niagads.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'www.niagads.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'http://adni.loni.usc.edu'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: http://adni.loni.usc.edu, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: http://adni.loni.usc.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: http://adni.loni.usc.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'adni.loni.usc.edu', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn4907804', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn4907804', 'dataset_id': 'syn4907804'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn4907804\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn4907804, synapse.org, https://www.synapse.org/#!Synapse:syn4907804\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn4907804\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn4907804', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 256 - INFO - Augmented dataset links: [{'dataset_identifier': 'n/a', 'data_repository': 'www.niagads.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'adni.loni.usc.edu', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn4907804', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 3 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Skipping dataset 1: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Skipping dataset 2: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn4907804\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn4907804\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn4907804 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn4907804 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn4907804 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn4907804 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn4907804 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 3 with repo: synapse.org and accession_id: syn4907804\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn4907804\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 3: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 3\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 265 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 4, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 477 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 35th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11439699)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11439699\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC11439699&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 394 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC11439699.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b0281bd0>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11439699 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11439699, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 445 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC11439699)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 547 - INFO - Extracted 16 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 557 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#s1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 32) for s1: 1 Introduction [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#s1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 32) for s1: 1 Introduction [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#s1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 32) for s1: 1 Introduction [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#s2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 28) for s2: 2 Genomics [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#s2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 28) for s2: 2 Genomics [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#s2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 28) for s2: 2 Genomics [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#s3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 35) for s3: 3 Transcriptomics [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#s3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 35) for s3: 3 Transcriptomics [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#s4'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 30) for s4: 4 Proteomics [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#s6'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 32) for s6: 6 Metabolomics [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#sec13'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for sec13: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#sec13'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for sec13: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#s12'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 40) for s12: Supplementary material [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#SM1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 190) for SM1: A brief meta-analysis of the topics studied concerning Alzheimer's disease using ROSMAP data, as well as outside of the cohort, can be found in the Supplementary material [href=this file] .\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 90277\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 296276\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 90999 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-aa9e2bfc0bdde99423351a3cf785b4d306b2f0ad2a1986f6ee1588b027d923e4\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 90999 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\"datasets\":[{\"dataset_identifier\":\"syn2580853\",\"repository_reference\":\"https://www.synapse.org/Synapse:syn2580853\"},{\"dataset_identifier\":\"syn22264775\",\"repository_reference\":\"https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn22264775\"},{\"dataset_identifier\":\"syn9702085\",\"repository_reference\":\"https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn9702085\"},{\"dataset_identifier\":\"syn338856\",\"repository_reference\":\"https://www.synapse.org/#!Synapse:syn338856\"},{\"dataset_identifier\":\"syn26720676\",\"repository_reference\":\"https://www.synapse.org/Synapse:syn26720676\"},{\"dataset_identifier\":\"syn3157325\",\"repository_reference\":\"https://doi.org/10.7303/syn3157325\"},{\"dataset_identifier\":\"syn17008935\",\"repository_reference\":\"https://www.synapse.org/Synapse:syn17008935\"},{\"dataset_identifier\":\"syn3157275\",\"repository_reference\":\"https://www.synapse.org/Synapse:syn3157275\"},{\"dataset_identifier\":\"n/a\",\"repository_reference\":\"https://www.radc.rush.edu\"},{\"dataset_identifier\":\"n/a\",\"repository_reference\":\"https://www.niagads.org\"}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'syn2580853', 'repository_reference': 'https://www.synapse.org/Synapse:syn2580853'}, {'dataset_identifier': 'syn22264775', 'repository_reference': 'https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn22264775'}, {'dataset_identifier': 'syn9702085', 'repository_reference': 'https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn9702085'}, {'dataset_identifier': 'syn338856', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn338856'}, {'dataset_identifier': 'syn26720676', 'repository_reference': 'https://www.synapse.org/Synapse:syn26720676'}, {'dataset_identifier': 'syn3157325', 'repository_reference': 'https://doi.org/10.7303/syn3157325'}, {'dataset_identifier': 'syn17008935', 'repository_reference': 'https://www.synapse.org/Synapse:syn17008935'}, {'dataset_identifier': 'syn3157275', 'repository_reference': 'https://www.synapse.org/Synapse:syn3157275'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.radc.rush.edu'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.niagads.org'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 10 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn2580853\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn22264775\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn9702085\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn338856\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn26720676\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn3157325\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn17008935\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn3157275\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'syn2580853', 'repository_reference': 'https://www.synapse.org/Synapse:syn2580853', 'dataset_id': 'syn2580853'}, {'dataset_identifier': 'syn22264775', 'repository_reference': 'https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn22264775', 'dataset_id': 'syn22264775'}, {'dataset_identifier': 'syn9702085', 'repository_reference': 'https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn9702085', 'dataset_id': 'syn9702085'}, {'dataset_identifier': 'syn338856', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn338856', 'dataset_id': 'syn338856'}, {'dataset_identifier': 'syn26720676', 'repository_reference': 'https://www.synapse.org/Synapse:syn26720676', 'dataset_id': 'syn26720676'}, {'dataset_identifier': 'syn3157325', 'repository_reference': 'https://doi.org/10.7303/syn3157325', 'dataset_id': 'syn3157325'}, {'dataset_identifier': 'syn17008935', 'repository_reference': 'https://www.synapse.org/Synapse:syn17008935', 'dataset_id': 'syn17008935'}, {'dataset_identifier': 'syn3157275', 'repository_reference': 'https://www.synapse.org/Synapse:syn3157275', 'dataset_id': 'syn3157275'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.radc.rush.edu', 'dataset_id': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.niagads.org'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn2580853', 'repository_reference': 'https://www.synapse.org/Synapse:syn2580853', 'dataset_id': 'syn2580853'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn2580853\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn2580853, synapse.org, https://www.synapse.org/Synapse:syn2580853\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn2580853\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn2580853', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn22264775', 'repository_reference': 'https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn22264775', 'dataset_id': 'syn22264775'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn22264775\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn22264775, synapse.org, https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn22264775\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn22264775\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn22264775', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn9702085', 'repository_reference': 'https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn9702085', 'dataset_id': 'syn9702085'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn9702085\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn9702085, synapse.org, https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn9702085\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn9702085\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn9702085', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn338856', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn338856', 'dataset_id': 'syn338856'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: syn338856\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID syn338856 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn338856, identifier: syn338856\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn338856\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn338856 for https://www.synapse.org/#!Synapse:syn338856\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn338856 for https://www.synapse.org/#!Synapse:syn338856\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn338856 for https://www.synapse.org/#!Synapse:syn338856\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn338856 for https://www.synapse.org/#!Synapse:syn338856\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn338856 in Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn338856', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn26720676', 'repository_reference': 'https://www.synapse.org/Synapse:syn26720676', 'dataset_id': 'syn26720676'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn26720676\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn26720676, synapse.org, https://www.synapse.org/Synapse:syn26720676\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn26720676\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn26720676', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn3157325', 'repository_reference': 'https://doi.org/10.7303/syn3157325', 'dataset_id': 'syn3157325'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(10\\.\\d{4,9}/).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: 10.7303/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn3157325\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn3157325, synapse.org, https://doi.org/10.7303/syn3157325\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn3157325\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn3157325', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn17008935', 'repository_reference': 'https://www.synapse.org/Synapse:syn17008935', 'dataset_id': 'syn17008935'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn17008935\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn17008935, synapse.org, https://www.synapse.org/Synapse:syn17008935\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn17008935\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn17008935', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn3157275', 'repository_reference': 'https://www.synapse.org/Synapse:syn3157275', 'dataset_id': 'syn3157275'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn3157275\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn3157275, synapse.org, https://www.synapse.org/Synapse:syn3157275\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn3157275\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn3157275', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.radc.rush.edu', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.radc.rush.edu, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.radc.rush.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://www.radc.rush.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'www.radc.rush.edu', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.niagads.org'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.niagads.org, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.niagads.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://www.niagads.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'www.niagads.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn2580853', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn22264775', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn9702085', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn338856', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn26720676', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn3157325', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn17008935', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn3157275', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'www.radc.rush.edu', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'www.niagads.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 10 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn2580853\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn2580853\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn2580853 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn2580853 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn2580853 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn2580853 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn2580853 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn2580853\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn2580853\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn22264775\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn22264775\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn22264775 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn22264775 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn22264775 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn22264775 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn22264775 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn22264775\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn22264775\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn9702085\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn9702085\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn9702085 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn9702085 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn9702085 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn9702085 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn9702085 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 3 with repo: synapse.org and accession_id: syn9702085\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn9702085\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 3: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn338856\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn338856\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn338856 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn338856 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn338856 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn338856 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn338856 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1090 - WARNING - Identifier syn338856 does not match pattern for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 4 with repo: synapse.org and accession_id: syn338856\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn338856\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 4: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn26720676\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn26720676\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn26720676 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn26720676 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn26720676 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn26720676 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn26720676 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 5 with repo: synapse.org and accession_id: syn26720676\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn26720676\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 5: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn3157325\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn3157325\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3157325 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn3157325 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn3157325 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3157325 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn3157325 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 6 with repo: synapse.org and accession_id: syn3157325\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn3157325\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 6: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn17008935\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn17008935\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn17008935 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn17008935 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn17008935 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn17008935 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn17008935 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 7 with repo: synapse.org and accession_id: syn17008935\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn17008935\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 7: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn3157275\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn3157275\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3157275 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn3157275 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn3157275 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3157275 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn3157275 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 8 with repo: synapse.org and accession_id: syn3157275\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn3157275\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 8: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Skipping dataset 9: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 10 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Skipping dataset 10: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 10\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 26, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 640 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - Parsed data extraction completed. Elements collected: 11\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 481 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 36th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC6323985&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 394 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b710fed0>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 445 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 547 - INFO - Extracted 23 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 557 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#abstract1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 26) for abstract1: Abstract [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#SEC1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 30) for SEC1: INTRODUCTION [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#SEC1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 30) for SEC1: INTRODUCTION [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#SEC1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 30) for SEC1: INTRODUCTION [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#SEC1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 30) for SEC1: INTRODUCTION [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#SEC1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 30) for SEC1: INTRODUCTION [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#SEC1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 30) for SEC1: INTRODUCTION [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#SEC1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 30) for SEC1: INTRODUCTION [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#SEC2-1-1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for SEC2-1-1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#SEC2-1-1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for SEC2-1-1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#SEC2-1-1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for SEC2-1-1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#SEC2-1-1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for SEC2-1-1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#SEC2-1-1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for SEC2-1-1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#SEC2-1-3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for SEC2-1-3: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#SEC2-1-3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for SEC2-1-3: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#SEC2-1-4'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for SEC2-1-4: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#SEC2-2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for SEC2-2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#SEC2-2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for SEC2-2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#SEC2-4'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for SEC2-4: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 2 <a> elements with href='#sup1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 206) for sup1: The details of the extended description and metric advantages are given in the supplementary materials [href=this file] .\n",
      "Supplementary Table S1 [href=this file] summarizes the GTRD content and statistics.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#SEC4'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 36) for SEC4: SUPPLEMENTARY DATA [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 32820\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 107422\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 33542 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-088a1f6c9949218780168bb5798a84b4af9074e63f6e7a92c68e9c7cc59fb573\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 33542 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\n",
      "  \"datasets\": [\n",
      "    {\n",
      "      \"dataset_identifier\": \"GTRD\",\n",
      "      \"repository_reference\": \"http://gtrd.biouml.org\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"SRA\",\n",
      "      \"repository_reference\": \"http://www.ncbi.nlm.nih.gov/sra\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"ENCODE\",\n",
      "      \"repository_reference\": \"https://www.encodeproject.org\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GEO\",\n",
      "      \"repository_reference\": \"http://www.ncbi.nlm.nih.gov/geo/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"PMC6323985\",\n",
      "      \"repository_reference\": \"https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"10.1093/nar/gky1128\",\n",
      "      \"repository_reference\": \"https://doi.org/10.1093/nar/gky1128\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    }\n",
      "  ]\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'GTRD', 'repository_reference': 'http://gtrd.biouml.org'}, {'dataset_identifier': 'SRA', 'repository_reference': 'http://www.ncbi.nlm.nih.gov/sra'}, {'dataset_identifier': 'ENCODE', 'repository_reference': 'https://www.encodeproject.org'}, {'dataset_identifier': 'GEO', 'repository_reference': 'http://www.ncbi.nlm.nih.gov/geo/'}, {'dataset_identifier': 'PMC6323985', 'repository_reference': 'https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985/'}, {'dataset_identifier': '10.1093/nar/gky1128', 'repository_reference': 'https://doi.org/10.1093/nar/gky1128'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 7 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GTRD\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: ENCODE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: PMC6323985\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: 10.1093/nar/gky1128\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'GTRD', 'repository_reference': 'http://gtrd.biouml.org', 'dataset_id': 'GTRD'}, {'dataset_identifier': 'SRA', 'repository_reference': 'http://www.ncbi.nlm.nih.gov/sra', 'dataset_id': 'SRA'}, {'dataset_identifier': 'ENCODE', 'repository_reference': 'https://www.encodeproject.org', 'dataset_id': 'ENCODE'}, {'dataset_identifier': 'GEO', 'repository_reference': 'http://www.ncbi.nlm.nih.gov/geo/', 'dataset_id': 'GEO'}, {'dataset_identifier': 'PMC6323985', 'repository_reference': 'https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985/', 'dataset_id': 'PMC6323985'}, {'dataset_identifier': '10.1093/nar/gky1128', 'repository_reference': 'https://doi.org/10.1093/nar/gky1128', 'dataset_id': '10.1093/nar/gky1128'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GTRD', 'repository_reference': 'http://gtrd.biouml.org', 'dataset_id': 'GTRD'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: GTRD\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID GTRD is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: http://gtrd.biouml.org, identifier: GTRD\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: http://gtrd.biouml.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GTRD for http://gtrd.biouml.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GTRD for http://gtrd.biouml.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier GTRD for http://gtrd.biouml.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier GTRD for http://gtrd.biouml.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier GTRD for http://gtrd.biouml.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: http://gtrd.biouml.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GTRD', 'data_repository': 'gtrd.biouml.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'SRA', 'repository_reference': 'http://www.ncbi.nlm.nih.gov/sra', 'dataset_id': 'SRA'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID SRA is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: http://www.ncbi.nlm.nih.gov/sra, identifier: SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: http://www.ncbi.nlm.nih.gov/sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier SRA for http://www.ncbi.nlm.nih.gov/sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier SRA for http://www.ncbi.nlm.nih.gov/sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier SRA for http://www.ncbi.nlm.nih.gov/sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier SRA for http://www.ncbi.nlm.nih.gov/sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier SRA for http://www.ncbi.nlm.nih.gov/sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: http://www.ncbi.nlm.nih.gov/sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'SRA', 'data_repository': 'www.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'ENCODE', 'repository_reference': 'https://www.encodeproject.org', 'dataset_id': 'ENCODE'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: ENCODE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID ENCODE is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.encodeproject.org, identifier: ENCODE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier ENCODE for https://www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier ENCODE for https://www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier ENCODE for https://www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier ENCODE for https://www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier ENCODE for https://www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'ENCODE', 'data_repository': 'www.encodeproject.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GEO', 'repository_reference': 'http://www.ncbi.nlm.nih.gov/geo/', 'dataset_id': 'GEO'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID GEO is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: http://www.ncbi.nlm.nih.gov/geo/, identifier: GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: http://www.ncbi.nlm.nih.gov/geo/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GEO for http://www.ncbi.nlm.nih.gov/geo/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GEO for http://www.ncbi.nlm.nih.gov/geo/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier GEO for http://www.ncbi.nlm.nih.gov/geo/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier GEO for http://www.ncbi.nlm.nih.gov/geo/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier GEO for http://www.ncbi.nlm.nih.gov/geo/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: http://www.ncbi.nlm.nih.gov/geo/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GEO', 'data_repository': 'www.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'PMC6323985', 'repository_reference': 'https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985/', 'dataset_id': 'PMC6323985'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: PMC6323985\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID PMC6323985 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985/, identifier: PMC6323985\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier PMC6323985 for https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier PMC6323985 for https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier PMC6323985 for https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier PMC6323985 for https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier PMC6323985 for https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'PMC6323985', 'data_repository': 'pmc.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': '10.1093/nar/gky1128', 'repository_reference': 'https://doi.org/10.1093/nar/gky1128', 'dataset_id': '10.1093/nar/gky1128'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(10\\.\\d{4,9}/).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: 10.1093/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: 10.1093/nar/gky1128, doi.org, https://doi.org/10.1093/nar/gky1128\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: 10.1093/nar/gky1128\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': '10.1093/nar/gky1128', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 677 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 605 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'GTRD', 'data_repository': 'gtrd.biouml.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'SRA', 'data_repository': 'www.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'ENCODE', 'data_repository': 'www.encodeproject.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GEO', 'data_repository': 'www.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'PMC6323985', 'data_repository': 'pmc.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}, {'dataset_identifier': '10.1093/nar/gky1128', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 6 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GTRD\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: gtrd.biouml.org, identifier: GTRD\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GTRD for gtrd.biouml.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GTRD for gtrd.biouml.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier GTRD for gtrd.biouml.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier GTRD for gtrd.biouml.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier GTRD for gtrd.biouml.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: gtrd.biouml.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 976 - WARNING - URL gtrd.biouml.org may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GTRD in gtrd.biouml.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 1 with repo: gtrd.biouml.org and accession_id: GTRD\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1254 - WARNING - Repository gtrd.biouml.org unknown in Ontology. Skipping dataset page 1.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: www.ncbi.nlm.nih.gov, identifier: SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier SRA for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier SRA for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier SRA for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier SRA for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier SRA for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 976 - WARNING - URL www.ncbi.nlm.nih.gov may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for SRA in www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 2 with repo: www.ncbi.nlm.nih.gov and accession_id: SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1254 - WARNING - Repository www.ncbi.nlm.nih.gov unknown in Ontology. Skipping dataset page 2.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: ENCODE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: www.encodeproject.org, identifier: ENCODE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier ENCODE for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier ENCODE for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier ENCODE for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier ENCODE for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier ENCODE for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 976 - WARNING - URL www.encodeproject.org may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for ENCODE in www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 3 with repo: www.encodeproject.org and accession_id: ENCODE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1254 - WARNING - Repository www.encodeproject.org unknown in Ontology. Skipping dataset page 3.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: www.ncbi.nlm.nih.gov, identifier: GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GEO for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GEO for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier GEO for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier GEO for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier GEO for www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 976 - WARNING - URL www.ncbi.nlm.nih.gov may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GEO in www.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 4 with repo: www.ncbi.nlm.nih.gov and accession_id: GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1254 - WARNING - Repository www.ncbi.nlm.nih.gov unknown in Ontology. Skipping dataset page 4.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: PMC6323985\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: pmc.ncbi.nlm.nih.gov, identifier: PMC6323985\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier PMC6323985 for pmc.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier PMC6323985 for pmc.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier PMC6323985 for pmc.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier PMC6323985 for pmc.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier PMC6323985 for pmc.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: pmc.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 976 - WARNING - URL pmc.ncbi.nlm.nih.gov may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for PMC6323985 in pmc.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 5 with repo: pmc.ncbi.nlm.nih.gov and accession_id: PMC6323985\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1254 - WARNING - Repository pmc.ncbi.nlm.nih.gov unknown in Ontology. Skipping dataset page 5.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: 10.1093/nar/gky1128\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: doi.org, identifier: 10.1093/nar/gky1128\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.1093/nar/gky1128 for doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for 10.1093/nar/gky1128 in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 6 with repo: doi.org and accession_id: 10.1093/nar/gky1128\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://doi.org/10.1093/nar/gky1128\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 6\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 29, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage', 'link',\n",
      "       'title', 'file_info', 'description', 'source_section', 'section_class',\n",
      "       'download_link', 'file_extension', 'a_attr_href', 'a_attr_class',\n",
      "       'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 640 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - Parsed data extraction completed. Elements collected: 8\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 481 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 37th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10680627)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10680627\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC10680627&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 406 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC10680627.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC10680627 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 430 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC10680627, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 443 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 264 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 454 - INFO - Found 21 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b72036c0>, <Element supplementary-material at 0x1b7200400>, <Element supplementary-material at 0x1b72027c0>, <Element supplementary-material at 0x1b7203480>, <Element supplementary-material at 0x1b7202d80>, <Element supplementary-material at 0x1b7202cc0>, <Element supplementary-material at 0x1b7202b80>, <Element supplementary-material at 0x1b7202f40>, <Element supplementary-material at 0x1b72033c0>, <Element supplementary-material at 0x1b7203040>, <Element supplementary-material at 0x1b714f000>, <Element supplementary-material at 0x1b7202e00>, <Element supplementary-material at 0x1b72030c0>, <Element supplementary-material at 0x1b7202c40>, <Element supplementary-material at 0x1b7202f00>, <Element supplementary-material at 0x1b7202f80>, <Element supplementary-material at 0x1b7203340>, <Element supplementary-material at 0x1b7203240>, <Element supplementary-material at 0x1b7203280>, <Element supplementary-material at 0x1b7203100>, <Element supplementary-material at 0x1b7203f80>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 462 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b72036c0>, <Element supplementary-material at 0x1b7200400>, <Element supplementary-material at 0x1b72027c0>, <Element supplementary-material at 0x1b7203480>, <Element supplementary-material at 0x1b7202d80>, <Element supplementary-material at 0x1b7202cc0>, <Element supplementary-material at 0x1b7202b80>, <Element supplementary-material at 0x1b7202f40>, <Element supplementary-material at 0x1b72033c0>, <Element supplementary-material at 0x1b7203040>, <Element supplementary-material at 0x1b714f000>, <Element supplementary-material at 0x1b7202e00>, <Element supplementary-material at 0x1b72030c0>, <Element supplementary-material at 0x1b7202c40>, <Element supplementary-material at 0x1b7202f00>, <Element supplementary-material at 0x1b7202f80>, <Element supplementary-material at 0x1b7203340>, <Element supplementary-material at 0x1b7203240>, <Element supplementary-material at 0x1b7203280>, <Element supplementary-material at 0x1b7203100>, <Element supplementary-material at 0x1b7203f80>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 548 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref SD1: We reanalyzed and harmonized data from previous studies that used genetic perturbations (mostly CRISPR interference (CRISPRi)) to inhibit candidate enhancers and measure effects on gene expression (see Note S1 [this file] ). S1.1f [this file] ) and 9,938 “negative” element-gene pairs where no significant reduction in expression was observed despite the experiment having good power to detect >15–25% effects on gene expression ( Note S1 [this file] ). \n",
      "S10 [this file] ). \n",
      "S8c [this file] ), and the correlation between the predictive scores and the measured quantitative effect on expression ( Extended Data Fig. 2f [this file] , Fig. S9 [this file] ), using hold-one-chromosome-out cross-validation for supervised models. S8c [this file] , Extended Data Fig. 2f [this file] ), including at different distance thresholds ( Fig. 2c [rid=F2] ). S8c [this file] , Extended Data Fig. 2f [this file] ). We describe detailed evaluations of these other predictors and baseline methods in Note S2 [this file] , Note S3 [this file] , Note S5 [this file] , and Note S7 [this file] . \n",
      "ENCODE-rE2G and ENCODE-rE2G perform comparably to other wellperforming models (both with enrichment = 28 at a recall of 15%) ( Fig. 2d [rid=F2] , Table S15 [this file7] ). We also examined fine-mapped eQTL variant-gene links from 11 other GTEx tissues that were represented in ENCODE biosamples ( Table S14 [this file6] ). Using the score threshold corresponding to 70% in the K562 CRISPR dataset, ENCODE-rE2G achieved an average enrichment of 6.5 across tissues (range: 3.5–11.4), similar than other predictive and baseline models, while achieving a stronger recall, identifying an average of 16% (range: 10–22%) of fine-mapped eQTL variants and linking them to the correct eGene ( Extended Data Fig. 3c [this file] , d [this file] ). \n",
      "These conclusions were supported by additional benchmarking analyses ( Extended Data Fig. 4 [this file] ). \n",
      "S2.1a [this file] ). \n",
      "Many ENCODE-rE2G enhancers show elevated levels of active chromatin marks, such as H3K27ac and DNase-seq (mean DNase-seq signal for candidate elements across biosamples = 3.27; mean for enhancers linked to at least one gene = 7.52) — but importantly, many do not (15.6% of enhancers have DNase-seq scores below the global median) ( Extended Data Fig 5h [this file] ). S2.1d [this file] ). In part for this reason, annotations of enhancers based on 1D chromatin state alone miss 33% (chromHMM) or 61% (cCRE Catalog) of ENCODE-rE2G enhancers, as well as 30% or 12%, respectively, of the validated enhancergene pairs from the Nasser ., 2021 unbiased CRISPR DHS tiling dataset ( Extended Data Fig. 6 [this file] ). \n",
      "As expected, many element-gene regulatory interactions were highly cell-type specific: 26.4% were present in only one biosample (mean = 34.49, median = 7 biosamples) ( Fig. 3e [rid=F3] ), and predicted regulatory interactions were much more specific than the activity of either the promoters or enhancers alone ( Extended Data Fig 5g [this file] , h [this file] ). \n",
      "As previously observed, fine-mapped variants underlying enhancers were often predicted to regulate multiple genes (average=3.4, max=56), including 62% that are linked to different genes in different cell types; this appears to be a fundamental property of enhancer-promoter regulation , and motivates combining locus-specific variant-to-gene information with orthogonal information about gene function ( Note S7 [this file] , Fig S7.1 [this file] ). \n",
      "This combination of ENCODE-rE2G and PoPS predicted a target gene for 3,528 out of 13,012 non-coding credible sets with a PIP > 0.1 fine-mapped variant spanning all traits; this included 5,851 out of 29.2K fine-mapped GWAS variants ( Table S13 [this file5] ), representing a 3.7-fold excess overlap compared to control variants (all distal noncoding variants detected in the 1000 Genomes project, =9.2 million) ( Fig. 4b [rid=F4] ). Other interesting examples at the and loci are reported in Note S7 [this file] ( Fig S7.2 [this file] ). \n",
      "We provide further analysis and guidelines regarding the value of adding new cell types to the ENCODE resource, assessing tissue-specificity of the ENCODE-rE2G biosample predictions, evaluating other combinations of ENCODE-rE2G and PoPS, and restricting our ENCODE-rE2G analyses to top disease-enriched biosamples for identifying possible cell types and target genes for GWAS loci in Note S7 [this file] ( Fig S7.1 [this file] . \n",
      "S4.1 [this file] - 2 [this file] , Note S4 [this file] , Note S5 [this file] ). \n",
      "These results are consistent with the importance of enhancer activity and 3D contact in enhancer regulation and highlight the utility of combining them as in the ABC model ( Fig. 2b [rid=F2] , c [rid=F2] , Fig S4.3 [this file] ). \n",
      "S11 [this file] ). S11a [this file] ). \n",
      "Out of 6 methods for estimating 3D enhancer-promoter contact frequency in the ABC model, we found that the best dataset was cell-type-specific ENCODE Hi-C (precision at 70% recall = 53.8%; Hi-C depth = 2 billion reads, 5-kb resolution), which outperformed other datasets including Hi-C (1 billion reads) or an inverse function of distance (precision at 70% recall = 50.4% or 31.8%, respectively) ( Fig. 5g [rid=F5] , Note S2 [this file] ). S2.1f [this file] , g [this file] ). \n",
      "Key additional features were related to promoter class and enhancerenhancer interactions ( Fig. 5b [rid=F5] , h [rid=F5] , Fig S4.4 [this file] ). \n",
      "S6.1b [this file] ) to have a fine-mapped eQTL variant in a distal accessible site ( Fig. S6.1a [this file] ). S6.1c [this file] - f [this file] ). Together, these observations support a model in which sequence-intrinsic features of the promoters of ubiquitously expressed genes make them less sensitive to distal enhancers (see Note S6 [this file] ). \n",
      "In each dataset, perturbations to individual enhancers on average reduced H3K27ac signal at other nearby enhancers, and the magnitude of the effect decreased as enhancer-enhancer distance increased ( Fig. 6g [rid=F6] , h [rid=F6] , Extended Data Fig. 8 [this file] ). \n",
      "S6.1b [this file] ), possibly because they contain built-in motifs for activating transcription factors that buffer them against distal enhancers . \n",
      "The CRISPR datasets still have a relatively small number of positive examples; were designed with certain biases in enhancer and gene selection; do not distinguish from effects; are conducted largely in a single cell type (K562); and are underpowered to detect regulatory interactions with small effect sizes ( ., <25%) (see Note S1 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref SD2: For 197 non-coding credible sets corresponding to 11 blood-related traits that are linked to exactly one “putatively causal” gene with a coding fine-mapped variant within 2Mb on either side of the lead variant, we compute the precision and recall in linking it to this gene for each of the element-gene predictions in blood biosamples (see Methods [this file] ). \n",
      "See Methods [this file] , ‘CRISPRi benchmark’ section for description of statistical test. \n",
      "See Methods [this file] , ‘Data visualization’ section for definition of box plot elements. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref SD3: We constructed two logistic regression classifiers to predict CRISPR-validated element-gene pairs using different combinations of molecular features ( Tables S1 [this file] , S3 [rid=SD5] ). \n",
      "We used this pipeline to evaluate ENCODE-rE2G and 574 other models ( Table S1 [this file] ), including (i) the ABC model, which we re-computed here natively in hg38 using various combinations of ENCODE assays to estimate enhancer activity and enhancer-promoter 3D contact; (ii) previous machine learning models that infer enhancer-gene regulation, including EpiMap , GraphReg , CIA , Enformer , and EPIraction ; and (iii) simple baselines such as linking elements to the closest expressed transcription start site (TSS) or gene, linking elements to genes solely on the basis of loops or domains from 3D contact measurements, or correlating distal element and promoter accessibility across cell types. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref SD4: ENCODE file accessions are listed in Table S2 [this file] and Table S10 [rid=SD12] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref SD5: We constructed two logistic regression classifiers to predict CRISPR-validated element-gene pairs using different combinations of molecular features ( Tables S1 [rid=SD3] , S3 [this file] ). Specifically, we included features related to (i) chromatin state (quantitative DNase-seq signals at the element and promoter), (ii) 3D contact frequency (averaged from ENCODE Hi-C in 35 diploid biosamples to capture cell-type-invariant features of genome organization); (iii) the Activity-by-Contact (ABC) model , which multiplies enhancer activity and 3D contact frequency; (iv) genomic position ( ., distance from element to promoter); (v) promoter class ( ., whether the gene is uniformly and ubiquitously expressed across cell types ( ., a “housekeeping” gene)); and (vi) information about nearby enhancers ( ., the activity of all other elements within 5 kb of the perturbed element) ( Table S3 [this file] ). In the ENCODE-rE2G model, we sought to evaluate the utility of additional assays available in Tier 1 ENCODE cell types, and expanded to include a total of 47 features derived from DNase-seq, histone ChIP-seq, cell-type specific ENCODE Hi-C, and ChIA-PET experiments, as well as features from additional predictive models such as EpiMap and GraphReg that jointly analyze many datasets ( Table S3 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref SD6: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref SD7: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref SD8: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref SD9: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref SD10: ENCODE portal accession ids and public links for GraphReg and EPIraction predictions are available in Table S8 [this file] and Table S9 [rid=SD11] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref SD11: ENCODE portal accession ids and public links for GraphReg and EPIraction predictions are available in Table S8 [rid=SD10] and Table S9 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref SD12: ENCODE file accessions are listed in Table S2 [rid=SD4] and Table S10 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref SD13: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref SD14: We next applied ENCODE-rE2G across 352 biosamples with DNase-seq data generated by the ENCODE Project, and identified a total of 13,455,443 element-gene regulatory interactions using the threshold at 70% recall in the CRISPR benchmark ( Fig. 3a [rid=F3] , Table S12 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref SD15: This combination of ENCODE-rE2G and PoPS predicted a target gene for 3,528 out of 13,012 non-coding credible sets with a PIP > 0.1 fine-mapped variant spanning all traits; this included 5,851 out of 29.2K fine-mapped GWAS variants ( Table S13 [this file] ), representing a 3.7-fold excess overlap compared to control variants (all distal noncoding variants detected in the 1000 Genomes project, =9.2 million) ( Fig. 4b [rid=F4] ). \n",
      "Numerical results are reported in Table S13 [this file] . \n",
      "Numerical results for each of the Figure panels are reported in Table S13 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref SD16: We also examined fine-mapped eQTL variant-gene links from 11 other GTEx tissues that were represented in ENCODE biosamples ( Table S14 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref SD17: ENCODE-rE2G and ENCODE-rE2G perform comparably to other wellperforming models (both with enrichment = 28 at a recall of 15%) ( Fig. 2d [rid=F2] , Table S15 [this file] ). \n",
      "Numerical results are reported in Table S15 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref SD18: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref SD19: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref SD20: Summary statistics available in Table S18 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref SD21: Summary statistics available in Table S19 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 243 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 43696\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 161219\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 44418 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-3e318b393dae8e4a817d0f8209d08a1b926f1e44a690871fd64be8ff69ef183b\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 44418 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\n",
      "  \"datasets\": [\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"https://github.com/karbalayghareh/ENCODE-rE2G\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"https://github.com/broadinstitute/ABC-Enhancer-Gene-Prediction\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"https://github.com/karbalayghareh/GraphReg\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"https://github.com/EngreitzLab/CRISPR_comparison\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"https://github.com/EngreitzLab/eQTLEnrichment\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"https://github.com/EngreitzLab/ABC-Max-pipeline\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"https://github.com/guigolab/EPIraction\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"https://github.com/wangxi001/CIA\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"https://github.com/KellisLab/EpiMap_GRCh38_linking\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"https://github.com/argschwind/ENCODE_CRISPR_data\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"ENCSR998YDI\",\n",
      "      \"repository_reference\": \"https://www.encodeproject.org/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE225157\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE225157\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"syn52264240\",\n",
      "      \"repository_reference\": \"https://www.synapse.org/#!Synapse:syn52264240\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"https://www.finucanelab.org/data/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"syn52234396\",\n",
      "      \"repository_reference\": \"https://www.synapse.org/#!Synapse:syn52234396\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"ENCSR443VTK\",\n",
      "      \"repository_reference\": \"https://www.encodeproject.org/\"\n",
      "    }\n",
      "  ]\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://github.com/karbalayghareh/ENCODE-rE2G'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://github.com/broadinstitute/ABC-Enhancer-Gene-Prediction'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://github.com/karbalayghareh/GraphReg'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://github.com/EngreitzLab/CRISPR_comparison'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://github.com/EngreitzLab/eQTLEnrichment'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://github.com/EngreitzLab/ABC-Max-pipeline'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://github.com/guigolab/EPIraction'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://github.com/wangxi001/CIA'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://github.com/KellisLab/EpiMap_GRCh38_linking'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://github.com/argschwind/ENCODE_CRISPR_data'}, {'dataset_identifier': 'ENCSR998YDI', 'repository_reference': 'https://www.encodeproject.org/'}, {'dataset_identifier': 'GSE225157', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE225157'}, {'dataset_identifier': 'syn52264240', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn52264240'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.finucanelab.org/data/'}, {'dataset_identifier': 'syn52234396', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn52234396'}, {'dataset_identifier': 'ENCSR443VTK', 'repository_reference': 'https://www.encodeproject.org/'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 17 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: ENCSR998YDI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE225157\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn52264240\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn52234396\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: ENCSR443VTK\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://github.com/karbalayghareh/ENCODE-rE2G'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://github.com/broadinstitute/ABC-Enhancer-Gene-Prediction'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://github.com/karbalayghareh/GraphReg'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://github.com/EngreitzLab/CRISPR_comparison'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://github.com/EngreitzLab/eQTLEnrichment'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://github.com/EngreitzLab/ABC-Max-pipeline'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://github.com/guigolab/EPIraction'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://github.com/wangxi001/CIA'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://github.com/KellisLab/EpiMap_GRCh38_linking'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://github.com/argschwind/ENCODE_CRISPR_data'}, {'dataset_identifier': 'ENCSR998YDI', 'repository_reference': 'https://www.encodeproject.org/', 'dataset_id': 'ENCSR998YDI'}, {'dataset_identifier': 'GSE225157', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE225157', 'dataset_id': 'GSE225157'}, {'dataset_identifier': 'syn52264240', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn52264240', 'dataset_id': 'syn52264240'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.finucanelab.org/data/'}, {'dataset_identifier': 'syn52234396', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn52234396', 'dataset_id': 'syn52234396'}, {'dataset_identifier': 'ENCSR443VTK', 'repository_reference': 'https://www.encodeproject.org/', 'dataset_id': 'ENCSR443VTK'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 677 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 605 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://github.com/karbalayghareh/ENCODE-rE2G'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://github.com/karbalayghareh/ENCODE-rE2G, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://github.com/karbalayghareh/ENCODE-rE2G\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://github.com/karbalayghareh/ENCODE-rE2G\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://github.com/broadinstitute/ABC-Enhancer-Gene-Prediction'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://github.com/broadinstitute/ABC-Enhancer-Gene-Prediction, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://github.com/broadinstitute/ABC-Enhancer-Gene-Prediction\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://github.com/broadinstitute/ABC-Enhancer-Gene-Prediction\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://github.com/karbalayghareh/GraphReg'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://github.com/karbalayghareh/GraphReg, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://github.com/karbalayghareh/GraphReg\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://github.com/karbalayghareh/GraphReg\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://github.com/EngreitzLab/CRISPR_comparison'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://github.com/EngreitzLab/CRISPR_comparison, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://github.com/EngreitzLab/CRISPR_comparison\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://github.com/EngreitzLab/CRISPR_comparison\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://github.com/EngreitzLab/eQTLEnrichment'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://github.com/EngreitzLab/eQTLEnrichment, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://github.com/EngreitzLab/eQTLEnrichment\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://github.com/EngreitzLab/eQTLEnrichment\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://github.com/EngreitzLab/ABC-Max-pipeline'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://github.com/EngreitzLab/ABC-Max-pipeline, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://github.com/EngreitzLab/ABC-Max-pipeline\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://github.com/EngreitzLab/ABC-Max-pipeline\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://github.com/guigolab/EPIraction'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://github.com/guigolab/EPIraction, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://github.com/guigolab/EPIraction\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://github.com/guigolab/EPIraction\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://github.com/wangxi001/CIA'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://github.com/wangxi001/CIA, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://github.com/wangxi001/CIA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://github.com/wangxi001/CIA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://github.com/KellisLab/EpiMap_GRCh38_linking'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://github.com/KellisLab/EpiMap_GRCh38_linking, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://github.com/KellisLab/EpiMap_GRCh38_linking\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://github.com/KellisLab/EpiMap_GRCh38_linking\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://github.com/argschwind/ENCODE_CRISPR_data'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://github.com/argschwind/ENCODE_CRISPR_data, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://github.com/argschwind/ENCODE_CRISPR_data\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://github.com/argschwind/ENCODE_CRISPR_data\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'ENCSR998YDI', 'repository_reference': 'https://www.encodeproject.org/', 'dataset_id': 'ENCSR998YDI'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: ENCSR998YDI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID ENCSR998YDI is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.encodeproject.org/, identifier: ENCSR998YDI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.encodeproject.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier ENCSR998YDI for https://www.encodeproject.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier ENCSR998YDI for https://www.encodeproject.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier ENCSR998YDI for https://www.encodeproject.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier ENCSR998YDI for https://www.encodeproject.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier ENCSR998YDI for https://www.encodeproject.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://www.encodeproject.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'ENCSR998YDI', 'data_repository': 'www.encodeproject.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE225157', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE225157', 'dataset_id': 'GSE225157'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE225157\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE225157, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE225157\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE225157\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE225157', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn52264240', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn52264240', 'dataset_id': 'syn52264240'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn52264240\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn52264240, synapse.org, https://www.synapse.org/#!Synapse:syn52264240\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn52264240\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn52264240', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.finucanelab.org/data/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.finucanelab.org/data/, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.finucanelab.org/data/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://www.finucanelab.org/data/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'www.finucanelab.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn52234396', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn52234396', 'dataset_id': 'syn52234396'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn52234396\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn52234396, synapse.org, https://www.synapse.org/#!Synapse:syn52234396\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn52234396\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn52234396', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'ENCSR443VTK', 'repository_reference': 'https://www.encodeproject.org/', 'dataset_id': 'ENCSR443VTK'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: ENCSR443VTK\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID ENCSR443VTK is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.encodeproject.org/, identifier: ENCSR443VTK\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.encodeproject.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier ENCSR443VTK for https://www.encodeproject.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier ENCSR443VTK for https://www.encodeproject.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier ENCSR443VTK for https://www.encodeproject.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier ENCSR443VTK for https://www.encodeproject.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier ENCSR443VTK for https://www.encodeproject.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://www.encodeproject.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'ENCSR443VTK', 'data_repository': 'www.encodeproject.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 256 - INFO - Augmented dataset links: [{'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'ENCSR998YDI', 'data_repository': 'www.encodeproject.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE225157', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn52264240', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'www.finucanelab.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn52234396', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'ENCSR443VTK', 'data_repository': 'www.encodeproject.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 16 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Skipping dataset 1: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Skipping dataset 2: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Skipping dataset 3: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Skipping dataset 4: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Skipping dataset 5: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Skipping dataset 6: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Skipping dataset 7: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Skipping dataset 8: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Skipping dataset 9: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 10 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Skipping dataset 10: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 11 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: ENCSR998YDI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: www.encodeproject.org, identifier: ENCSR998YDI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier ENCSR998YDI for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier ENCSR998YDI for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier ENCSR998YDI for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier ENCSR998YDI for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier ENCSR998YDI for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 976 - WARNING - URL www.encodeproject.org may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for ENCSR998YDI in www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 11 with repo: www.encodeproject.org and accession_id: ENCSR998YDI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1254 - WARNING - Repository www.encodeproject.org unknown in Ontology. Skipping dataset page 11.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 12 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE225157\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE225157\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE225157 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE225157 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE225157 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 12 with repo: geo and accession_id: GSE225157\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE225157\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 12: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 13 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn52264240\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn52264240\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn52264240 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn52264240 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn52264240 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn52264240 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn52264240 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 13 with repo: synapse.org and accession_id: syn52264240\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn52264240\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 13: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 14 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Skipping dataset 14: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 15 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn52234396\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn52234396\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn52234396 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn52234396 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn52234396 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn52234396 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn52234396 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 15 with repo: synapse.org and accession_id: syn52234396\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn52234396\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 15: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 16 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: ENCSR443VTK\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: www.encodeproject.org, identifier: ENCSR443VTK\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier ENCSR443VTK for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier ENCSR443VTK for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier ENCSR443VTK for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier ENCSR443VTK for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier ENCSR443VTK for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 976 - WARNING - URL www.encodeproject.org may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for ENCSR443VTK in www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 16 with repo: www.encodeproject.org and accession_id: ENCSR443VTK\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1254 - WARNING - Repository www.encodeproject.org unknown in Ontology. Skipping dataset page 16.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 16\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 265 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 37, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 477 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 38th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC5192600)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC5192600\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC5192600&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 394 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC5192600.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1422b3d50>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC5192600 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC5192600, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 445 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC5192600)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 547 - INFO - Extracted 7 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 557 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#Sec11'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for Sec11: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#FPar2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for FPar2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#FPar2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for FPar2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#Sec17'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for Sec17: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 40092\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 128541\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 40814 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-c46a5be53e43d21b5f2be439ca30e39da2d403448bfc84d4c380d67ad879e3c4\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 40814 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\n",
      "  \"datasets\": [\n",
      "    {\n",
      "      \"dataset_identifier\": \"DREAM4\",\n",
      "      \"repository_reference\": \"http://www.synapse.org/#!Synapse:syn3049712/files/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"DREAM5\",\n",
      "      \"repository_reference\": \"http://www.synapse.org/#!Synapse:syn2787209/files/\"\n",
      "    }\n",
      "  ]\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'DREAM4', 'repository_reference': 'http://www.synapse.org/#!Synapse:syn3049712/files/'}, {'dataset_identifier': 'DREAM5', 'repository_reference': 'http://www.synapse.org/#!Synapse:syn2787209/files/'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 2 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: DREAM4\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: DREAM5\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'DREAM4', 'repository_reference': 'http://www.synapse.org/#!Synapse:syn3049712/files/', 'dataset_id': 'DREAM4'}, {'dataset_identifier': 'DREAM5', 'repository_reference': 'http://www.synapse.org/#!Synapse:syn2787209/files/', 'dataset_id': 'DREAM5'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'DREAM4', 'repository_reference': 'http://www.synapse.org/#!Synapse:syn3049712/files/', 'dataset_id': 'DREAM4'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn3049712\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn3049712, None, http://www.synapse.org/#!Synapse:syn3049712/files/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn3049712\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: http://www.synapse.org/#!Synapse:syn3049712/files/, identifier: syn3049712\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: http://www.synapse.org/#!Synapse:syn3049712/files/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3049712 for http://www.synapse.org/#!Synapse:syn3049712/files/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn3049712 for http://www.synapse.org/#!Synapse:syn3049712/files/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn3049712 for http://www.synapse.org/#!Synapse:syn3049712/files/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3049712 for http://www.synapse.org/#!Synapse:syn3049712/files/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Found partial match for http://www.synapse.org/#!Synapse:syn3049712/files/ in Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn3049712', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'DREAM5', 'repository_reference': 'http://www.synapse.org/#!Synapse:syn2787209/files/', 'dataset_id': 'DREAM5'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn2787209\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn2787209, None, http://www.synapse.org/#!Synapse:syn2787209/files/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn2787209\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: http://www.synapse.org/#!Synapse:syn2787209/files/, identifier: syn2787209\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: http://www.synapse.org/#!Synapse:syn2787209/files/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn2787209 for http://www.synapse.org/#!Synapse:syn2787209/files/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn2787209 for http://www.synapse.org/#!Synapse:syn2787209/files/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn2787209 for http://www.synapse.org/#!Synapse:syn2787209/files/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn2787209 for http://www.synapse.org/#!Synapse:syn2787209/files/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Found partial match for http://www.synapse.org/#!Synapse:syn2787209/files/ in Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn2787209', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn3049712', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn2787209', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 2 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn3049712\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn3049712\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3049712 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn3049712 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn3049712 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3049712 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn3049712 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn3049712\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn3049712\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn2787209\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn2787209\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn2787209 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn2787209 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn2787209 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn2787209 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn2787209 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn2787209\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn2787209\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 2\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 9, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 640 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - Parsed data extraction completed. Elements collected: 4\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 481 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 39th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC7752919)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC7752919\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC7752919&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 406 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC7752919.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC7752919 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 430 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC7752919, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 443 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 264 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 454 - INFO - Found 7 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1affdd8c0>, <Element supplementary-material at 0x1b0402080>, <Element supplementary-material at 0x1b0400fc0>, <Element supplementary-material at 0x1b0400600>, <Element supplementary-material at 0x1b0402f80>, <Element supplementary-material at 0x1b0400280>, <Element supplementary-material at 0x1b0402e00>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 462 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1affdd8c0>, <Element supplementary-material at 0x1b0402080>, <Element supplementary-material at 0x1b0400fc0>, <Element supplementary-material at 0x1b0400600>, <Element supplementary-material at 0x1b0402f80>, <Element supplementary-material at 0x1b0400280>, <Element supplementary-material at 0x1b0402e00>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 548 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM1: Colon carcinomas showed substantial MLL1 expression at all tumor stages (T0–T4), scored from weak to moderate and strong (Fig. 1a [rid=Fig1] and Supplementary Fig. 1a [this file] ). These analyses showed no significant changes in expression across tumor stages (Supplementary Fig. 1b [this file] ). Patients of stages I-III and stage IV were stratified in three groups based on the level of expression (MLL1 low, medium, and high, Supplementary Fig. 1c [this file] ). High levels were associated with shorter survival and increased risk of disease relapse (Supplementary Fig. 1c, d [this file] ). Immuno-staining for β-catenin revealed high levels of nuclear β-catenin, indicative of high Wnt signaling, in a major portion of the analyzed tumors (Supplementary Fig. 1e [this file] ). Across all tumor stages, carcinomas with high levels of nuclear β-catenin showed strong expression of MLL1 (Fig. 1b [rid=Fig1] and Supplementary Fig. 1f, g [this file] ). \n",
      "Mll1 is gradually lost towards the differentiated compartment (Fig. 1d [rid=Fig1] and Supplementary Fig. 1h [this file] ). A crypt–villus gradient was also observed for H3K4 tri-methylation (H3K4me3) (Supplementary Fig. 1i [this file] ). We observed a similar distribution of Mll1, Ki67, and H3K4me3 in the mouse colon (Supplementary Fig. 1j–l [this file] ). \n",
      "The dysplastic tissue in the β-cat ; Mll1 intestines showed an expansion of Lgr5-GFP stem cells (Fig. 2c [rid=Fig2] , upper picture, Supplementary Fig. 2a [this file] ) and an increase in the stem cell genes , , and (Supplementary Fig. 2b [this file] ). The stem cell expansion in β-cat tumorous intestines was associated with an increase in Mll1 protein levels (Supplementary Fig. 2c [this file] , the mutant β-catenin allele is marked by an arrow, quantification is shown below). Specifically, tumor cells with high levels of β-catenin expressed high nuclear levels of Mll1 (Supplementary Fig. 2d [this file] ; green arrows mark β-catenin cells, white arrowheads indicate β-catenin cells). Equal recombination was observed at day 5 in 25% of crypts (Supplementary Fig. 2e [this file] , upper row, quantification in f). Subsequently, the mutant cells in β-cat ; Mll1 mice expanded at day 10 and formed LacZ dysplasia at day 30 (Supplementary Fig. 2e [this file] , middle and lower left pictures, quantification in f). In contrast, β-cat ; Mll1 mice showed a gradual reduction of LacZ cells down to 10% over time (Supplementary Fig. 2e [this file] , lower right picture, quantification in f). At 30 days, the β-cat ; Mll1 mice exhibited crypts with LacZ long-lived Paneth cells interspersed with non-mutant (LacZ ) stem cells (Supplementary Fig. 2g [this file] , marked by an arrow). Furthermore, we observed crypts that were normally shaped and depleted of Mll1 but with nuclear β-catenin (β-cat ) and positive for Lgr5-GFP; they exhibited a slight reduction in proliferation, as seen by a decrease in the incorporation of BrdU (Fig. 2c [rid=Fig2] , lower picture, Supplementary Fig. 2h, i [this file] ). Apoptosis was not detected in β-cat ; Mll1 crypts (Supplementary Fig. 2j [this file] ). \n",
      "All β-cat organoids grew independently of R-spondin1 (Supplementary Fig. 2k [this file] ). β-cat induced a strong expansion of Lgr5-GFP stem cells (Fig. 2d, e [rid=Fig2] , second panel). Single β-cat ; Mll1 cells re-formed spheroid-shaped organoids over many passages with high efficiency (Supplementary Fig. 2l [this file] ), demonstrating the proliferative potential and self-renewal capacity of Mll1-deficient β-cat cells. From the seventh passage on, β-cat ; Mll1 cells formed less organoids, which were smaller and acquired a thick-walled shape (Supplementary Fig. 2m [this file] ), indicating differentiation and progressive exhaustion of stemness upon loss of Mll1. \n",
      "Under non-oncogenic conditions without genetic stabilization of β-catenin, mutant Mll1 stem cells produced progeny up to 20 days after the induction of mutagenesis, as seen by LacZ lineage tracing in Mll1 mice (Supplementary Fig. 3a [this file] , quantification on the right). From 20 days onwards, Mll1 crypts were progressively lost, but a substantial number of Mll1 crypts persisted beyond 50 days after induction (Supplementary Fig. 3b [this file] , quantification on the right). \n",
      "Genes specific for enteroendocrine and tuft cells did not undergo major changes (Supplementary Fig. 3c [this file] ). β-cat ; Mll1 stem cells exhibited a strongly increased expression of , a transcription factor essential for secretory lineage determination , and its target gene , which instructs secretory cell differentiation in Paneth and goblet cells (Fig. 3b [rid=Fig3] ). \n",
      "To investigate the essential role of Mll1 in the stemness of human colon tumors, we established stable, doxycycline-inducible shMLL1 single-cell clones of the human colon adenocarcinoma cell lines Ls174T and DLD1, which exhibited a strong reduction of MLL1 (Supplementary Fig. 4a–d [this file] ). The ablation of MLL1 did not affect the expression of other MLL family members (Supplementary Fig. 4e [this file] ). A significant growth reduction was detected only after four passages and a total of 10–14 days of culture (Supplementary Fig. 4f, g [this file] ). We found that MLL1 protein was strongly increased in 3D (non-adherent, self-renewing) sphere cultures of human Ls174T cells compared to adherent cultures, while the expression of the family member SETD1A was not changed (Supplementary Fig. 4h [this file] , quantification on the right). In contrast to the 2D culture, the ablation of MLL1 in 3D sphere cultures had a rapid effect on the growth and self-renewal of the tumor cells: secondary sphere formation of shMLL1 cells was reduced by 80% in Ls174T and by 60% in DLD1 cells, compared to control cells (Fig. 4a [rid=Fig4] and Supplementary Fig. 4i [this file] , quantifications on the right). Residual shMLL1 spheres exhibited minimal MLL1 expression and reduced proliferation, as shown by staining for Ki67 (Supplementary Fig. 4j [this file] ). shMLL1 spheres were negative for the apoptosis marker cleaved Caspase-3 (Supplementary Fig. 4k [this file] , western blot on the right). \n",
      "The residual xenografts showed a strong reduction of MLL1 (Supplementary Fig. 4l [this file] , quantification below) and H3K4me3 (Supplementary Fig. 4m [this file] , upper panel), and significantly fewer proliferating cells (Supplementary Fig. 4m [this file] , middle panel, quantifications on the right). The apoptosis marker cleaved Caspase-3 did not increase in shMLL1 Ls174T tumors (Supplementary Fig. 4m [this file] , lower panel, western blot on the right). The shMLL1 human Ls174T xenografts and sphere cells showed enhanced differentiation with an increase in the expression of the Paneth cell-specific genes and and the goblet cell markers and (Supplementary Fig. 4n [this file] ), as was also observed in the β-cat mouse tumor model. An increase in expression was also observed in DLD1 sphere cells (Supplementary Fig. 4o [this file] ). The expression of the differentiation-associated cell cycle inhibitor was upregulated in shMLL1 Ls174T xenografts and spheres (Supplementary Fig. 4p, q [this file] , quantifications on the right). \n",
      "Comparison of the differentially expressed genes with a signature of Mll1-ablated fibroblasts revealed deregulation of MLL1 target genes such as and (Supplementary Fig. 4r [this file] , Supplementary Data 2 [rid=MOESM5] ). A volcano plot of the deregulated genes overlaid with this signature showed a predominant downregulation of stem cell genes upon knockdown of MLL1; these included , , , and the Wnt-regulated gene (Supplementary Data 3 [rid=MOESM6] and Fig. 4d [this file] ). The selective regulation of Wnt-dependent stem cell genes was also observed in DLD1 human colon cancer cells (Supplementary Fig. 4s [this file] ). \n",
      "Chromatin immunoprecipitation (ChIP) revealed high H3K4me3 and low H3K27me3 levels at the transcription start sites (TSS) of the MLL1-regulated stem cell genes , , and (Fig. 5a [rid=Fig5] and Supplementary Fig. 5a [this file] ). Ablation of MLL1 shifted the epigenetic state of the stem cell genes from transcriptionally permissive (H3K4me3) to repressed (H3K27me3) (Fig. 5a [rid=Fig5] and Supplementary Fig. 5a [this file] ). Global levels of H3K4 and H3K27 methylation were not changed (Supplementary Fig. 5b [this file] ). We identified MLL1 binding to the promoters of the intestinal stem cell genes , , and (Fig. 5b [rid=Fig5] and Supplementary Fig. 5c, d [this file] ). Strongly reduced binding of MLL1 was observed in shMLL1 Ls174T and DLD1 cells (Fig. 5b [rid=Fig5] and Supplementary Fig. 5c–e [this file] ). \n",
      "Primer sequences used for ChIP-qPCR are given in Supplementary Information (Supplementary Table 1) [this file] . \n",
      "Primer sequences used for qRT-PCR are listed in Supplementary Information (Supplementary Table 2) [this file] . \n",
      "Transcriptome analyses in human CRC were carried out on 2207 samples that were available in two public repositories (Supplementary Table 3 [this file] ): (i) RNA seq (version 2) data from The Cancer Genome Atlas (TCGA) project ( http://cancergenome.nih.gov/ (http://cancergenome.nih.gov/) ) were downloaded from the legacy version of the NCI Genomic Data Commons (GDC) repository . \n",
      "The RNA sequencing data from FACS-isolated Lgr5-GFP intestinal stem cells (Fig. 3b, d [rid=Fig3] and Supplementary Fig. 3c [this file] , Supplementary Data 1 [rid=MOESM4] ) have been deposited in the Gene expression omnibus under accession number GSE148394 (https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE148394) . The RNA sequencing data from shMLL1 Ls174T colon cancer cells (Fig. 4c, d [rid=Fig4] and Supplementary Fig. 4r [this file] , Supplementary Data 2 [rid=MOESM5] ) are available in the ArrayExpress database under accession number E-MTAB-8152 (https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-8152/) . The human colon cancer data referenced during the study (Supplementary Table 3 [this file] ) are available in public repositories from the TCGA website ( TCGA-COAD (https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga) , TCGA-READ (https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga) ) and in the Gene Expression Omnibus (GEO) at NCBI ( GSE14333 (https://www.ncbi.nlm.nih.gov/gds/?term=gse14333) , GSE33113 (https://www.ncbi.nlm.nih.gov/gds/?term=gse33113) , GSE39582 (https://www.ncbi.nlm.nih.gov/gds/?term=gse39582) , GSE38832 (https://www.ncbi.nlm.nih.gov/gds/?term=gse38832) , GSE44076 (https://www.ncbi.nlm.nih.gov/gds/?term=gse44076) , GSE13294 (https://www.ncbi.nlm.nih.gov/gds/?term=gse13294) , GSE18088 (https://www.ncbi.nlm.nih.gov/gds/?term=gse18088) and GSE2109 (https://www.ncbi.nlm.nih.gov/gds/?term=gse2109) ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM2: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM3: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM4: A volcano plot of the differentially expressed genes in β cat ; Mll1 versus β-cat ; Mll1 stem cells showed both up- and downregulation of genes (Fig. 3b [rid=Fig3] and Supplementary Data 1 [this file] ). There were also no decreases in other Notch target genes such as and (Supplementary Data 1 [this file] ), which means that the upregulation of occurs independently of a global decrease in Notch signaling. \n",
      "RNA seq differential expression data can be found in Supplementary Data 1 [this file] . \n",
      "The RNA sequencing data from FACS-isolated Lgr5-GFP intestinal stem cells (Fig. 3b, d [rid=Fig3] and Supplementary Fig. 3c [rid=MOESM1] , Supplementary Data 1 [this file] ) have been deposited in the Gene expression omnibus under accession number GSE148394 (https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE148394) . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM5: Comparison of the differentially expressed genes with a signature of Mll1-ablated fibroblasts revealed deregulation of MLL1 target genes such as and (Supplementary Fig. 4r [rid=MOESM1] , Supplementary Data 2 [this file] ). \n",
      "RNA seq differential expression data can be found in Supplementary Data 2 [this file] . \n",
      "The RNA sequencing data from shMLL1 Ls174T colon cancer cells (Fig. 4c, d [rid=Fig4] and Supplementary Fig. 4r [rid=MOESM1] , Supplementary Data 2 [this file] ) are available in the ArrayExpress database under accession number E-MTAB-8152 (https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-8152/) . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM6: Volcano plot of differentially expressed genes in shMLL1 human Ls174T cells (blue dots), overlaid with MLL1-regulated colon cancer stem cell signature (Supplementary Data 3 [this file] ), marked as green dots and indicated by name. mRNA expression of intestinal stem cell genes and classical Wnt targets in control and 6d doxycycline-induced shMLL1 human Ls174T spheres, = 4 samples derived from three biologically independent cell clones (  = 3), two-tailed unpaired -test, *** = 0.001, ** = 0.0018, * = 0.015, * = 0.03. \n",
      "We compiled 132 deregulated stem cell genes to establish an MLL1-regulated colon cancer stem cell signature (Supplementary Data 3 [this file] ). A volcano plot of the deregulated genes overlaid with this signature showed a predominant downregulation of stem cell genes upon knockdown of MLL1; these included , , , and the Wnt-regulated gene (Supplementary Data 3 [this file] and Fig. 4d [rid=MOESM1] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref MOESM7: Further information on research design is available in the Nature Research Reporting Summary [this file] linked to this article. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 243 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 52104\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 193485\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 52826 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-a23284aff812c471692a8b488474cc363b4f2b3a41488ff59536f7d0b02c75f9\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 52826 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\n",
      "  \"datasets\": [\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE148394\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE148394\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"E-MTAB-8152\",\n",
      "      \"repository_reference\": \"https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-8152/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"TCGA-COAD\",\n",
      "      \"repository_reference\": \"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"TCGA-READ\",\n",
      "      \"repository_reference\": \"https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE14333\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/gds/?term=gse14333\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE33113\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/gds/?term=gse33113\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE39582\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/gds/?term=gse39582\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE38832\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/gds/?term=gse38832\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE44076\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/gds/?term=gse44076\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE13294\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/gds/?term=gse13294\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE18088\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/gds/?term=gse18088\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE2109\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/gds/?term=gse2109\"\n",
      "    }\n",
      "  ]\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'GSE148394', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE148394'}, {'dataset_identifier': 'E-MTAB-8152', 'repository_reference': 'https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-8152/'}, {'dataset_identifier': 'TCGA-COAD', 'repository_reference': 'https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga'}, {'dataset_identifier': 'TCGA-READ', 'repository_reference': 'https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga'}, {'dataset_identifier': 'GSE14333', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/gds/?term=gse14333'}, {'dataset_identifier': 'GSE33113', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/gds/?term=gse33113'}, {'dataset_identifier': 'GSE39582', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/gds/?term=gse39582'}, {'dataset_identifier': 'GSE38832', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/gds/?term=gse38832'}, {'dataset_identifier': 'GSE44076', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/gds/?term=gse44076'}, {'dataset_identifier': 'GSE13294', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/gds/?term=gse13294'}, {'dataset_identifier': 'GSE18088', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/gds/?term=gse18088'}, {'dataset_identifier': 'GSE2109', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/gds/?term=gse2109'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 12 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE148394\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: E-MTAB-8152\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: TCGA-COAD\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: TCGA-READ\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE14333\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE33113\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE39582\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE38832\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE44076\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE13294\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE18088\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE2109\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'GSE148394', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE148394', 'dataset_id': 'GSE148394'}, {'dataset_identifier': 'E-MTAB-8152', 'repository_reference': 'https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-8152/', 'dataset_id': 'E-MTAB-8152'}, {'dataset_identifier': 'TCGA-COAD', 'repository_reference': 'https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga', 'dataset_id': 'TCGA-COAD'}, {'dataset_identifier': 'TCGA-READ', 'repository_reference': 'https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga', 'dataset_id': 'TCGA-READ'}, {'dataset_identifier': 'GSE14333', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/gds/?term=gse14333', 'dataset_id': 'GSE14333'}, {'dataset_identifier': 'GSE33113', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/gds/?term=gse33113', 'dataset_id': 'GSE33113'}, {'dataset_identifier': 'GSE39582', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/gds/?term=gse39582', 'dataset_id': 'GSE39582'}, {'dataset_identifier': 'GSE38832', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/gds/?term=gse38832', 'dataset_id': 'GSE38832'}, {'dataset_identifier': 'GSE44076', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/gds/?term=gse44076', 'dataset_id': 'GSE44076'}, {'dataset_identifier': 'GSE13294', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/gds/?term=gse13294', 'dataset_id': 'GSE13294'}, {'dataset_identifier': 'GSE18088', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/gds/?term=gse18088', 'dataset_id': 'GSE18088'}, {'dataset_identifier': 'GSE2109', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/gds/?term=gse2109', 'dataset_id': 'GSE2109'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE148394', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE148394', 'dataset_id': 'GSE148394'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE148394\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE148394, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE148394\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE148394\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE148394', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'E-MTAB-8152', 'repository_reference': 'https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-8152/', 'dataset_id': 'E-MTAB-8152'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: E-MTAB-8152\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID E-MTAB-8152 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-8152/, identifier: E-MTAB-8152\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-8152/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-MTAB-8152 for https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-8152/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-MTAB-8152 for https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-8152/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-MTAB-8152 for https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-8152/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-MTAB-8152 for https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-8152/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Found partial match for https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-8152/ in ArrayExpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'E-MTAB-8152', 'data_repository': 'arrayexpress', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'TCGA-COAD', 'repository_reference': 'https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga', 'dataset_id': 'TCGA-COAD'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: TCGA-COAD\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID TCGA-COAD is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga, identifier: TCGA-COAD\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier TCGA-COAD for https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier TCGA-COAD for https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier TCGA-COAD for https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier TCGA-COAD for https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier TCGA-COAD for https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'TCGA-COAD', 'data_repository': 'www.cancer.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'TCGA-READ', 'repository_reference': 'https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga', 'dataset_id': 'TCGA-READ'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: TCGA-READ\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID TCGA-READ is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga, identifier: TCGA-READ\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier TCGA-READ for https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier TCGA-READ for https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier TCGA-READ for https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier TCGA-READ for https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier TCGA-READ for https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'TCGA-READ', 'data_repository': 'www.cancer.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE14333', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/gds/?term=gse14333', 'dataset_id': 'GSE14333'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: gse14333\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE14333, geo, https://www.ncbi.nlm.nih.gov/gds/?term=gse14333\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE14333\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE14333', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE33113', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/gds/?term=gse33113', 'dataset_id': 'GSE33113'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: gse33113\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE33113, geo, https://www.ncbi.nlm.nih.gov/gds/?term=gse33113\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE33113\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE33113', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE39582', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/gds/?term=gse39582', 'dataset_id': 'GSE39582'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: gse39582\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE39582, geo, https://www.ncbi.nlm.nih.gov/gds/?term=gse39582\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE39582\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE39582', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE38832', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/gds/?term=gse38832', 'dataset_id': 'GSE38832'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: gse38832\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE38832, geo, https://www.ncbi.nlm.nih.gov/gds/?term=gse38832\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE38832\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE38832', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE44076', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/gds/?term=gse44076', 'dataset_id': 'GSE44076'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: gse44076\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE44076, geo, https://www.ncbi.nlm.nih.gov/gds/?term=gse44076\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE44076\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE44076', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE13294', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/gds/?term=gse13294', 'dataset_id': 'GSE13294'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: gse13294\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE13294, geo, https://www.ncbi.nlm.nih.gov/gds/?term=gse13294\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE13294\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE13294', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE18088', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/gds/?term=gse18088', 'dataset_id': 'GSE18088'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: gse18088\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE18088, geo, https://www.ncbi.nlm.nih.gov/gds/?term=gse18088\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE18088\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE18088', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE2109', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/gds/?term=gse2109', 'dataset_id': 'GSE2109'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: gse2109\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE2109, geo, https://www.ncbi.nlm.nih.gov/gds/?term=gse2109\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE2109\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE2109', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 256 - INFO - Augmented dataset links: [{'dataset_identifier': 'GSE148394', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'E-MTAB-8152', 'data_repository': 'arrayexpress', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'TCGA-COAD', 'data_repository': 'www.cancer.gov', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'TCGA-READ', 'data_repository': 'www.cancer.gov', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE14333', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE33113', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE39582', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE38832', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE44076', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE13294', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE18088', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE2109', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 12 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE148394\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE148394\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE148394 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE148394 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE148394 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE148394\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE148394\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: E-MTAB-8152\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: arrayexpress, identifier: E-MTAB-8152\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-MTAB-8152 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-MTAB-8152 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-MTAB-8152 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-MTAB-8152 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for E-MTAB-8152 in arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 2 with repo: arrayexpress and accession_id: E-MTAB-8152\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ebi.ac.uk/biostudies/ArrayExpress/studies/E-MTAB-8152\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 2: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: TCGA-COAD\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: www.cancer.gov, identifier: TCGA-COAD\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier TCGA-COAD for www.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier TCGA-COAD for www.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier TCGA-COAD for www.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier TCGA-COAD for www.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier TCGA-COAD for www.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: www.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 976 - WARNING - URL www.cancer.gov may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for TCGA-COAD in www.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 3 with repo: www.cancer.gov and accession_id: TCGA-COAD\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1254 - WARNING - Repository www.cancer.gov unknown in Ontology. Skipping dataset page 3.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: TCGA-READ\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: www.cancer.gov, identifier: TCGA-READ\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier TCGA-READ for www.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier TCGA-READ for www.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier TCGA-READ for www.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier TCGA-READ for www.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier TCGA-READ for www.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: www.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 976 - WARNING - URL www.cancer.gov may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for TCGA-READ in www.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 4 with repo: www.cancer.gov and accession_id: TCGA-READ\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1254 - WARNING - Repository www.cancer.gov unknown in Ontology. Skipping dataset page 4.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE14333\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE14333\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE14333 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE14333 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE14333 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 5 with repo: geo and accession_id: GSE14333\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE14333\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 5: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE33113\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE33113\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE33113 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE33113 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE33113 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 6 with repo: geo and accession_id: GSE33113\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE33113\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 6: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE39582\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE39582\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE39582 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE39582 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE39582 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 7 with repo: geo and accession_id: GSE39582\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE39582\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 7: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE38832\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE38832\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE38832 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE38832 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE38832 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 8 with repo: geo and accession_id: GSE38832\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38832\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 8: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE44076\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE44076\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE44076 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE44076 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE44076 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 9 with repo: geo and accession_id: GSE44076\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE44076\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 9: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 10 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE13294\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE13294\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE13294 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE13294 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE13294 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 10 with repo: geo and accession_id: GSE13294\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE13294\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 10: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 11 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE18088\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE18088\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE18088 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE18088 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE18088 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 11 with repo: geo and accession_id: GSE18088\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE18088\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 11: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 12 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE2109\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE2109\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE2109 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE2109 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE2109 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 12 with repo: geo and accession_id: GSE2109\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE2109\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 12: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 12\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 265 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 19, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 477 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 40th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC8859891&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 394 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b5f8ab90>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 445 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 547 - INFO - Extracted 6 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 557 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#Bib1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for Bib1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#Bib1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for Bib1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#Bib1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for Bib1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 47961\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 161432\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 48683 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-bdb45fab758c03483fcfffbc604274dc2135e33db825ad13c19e4d8d2686b959\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 48683 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\"datasets\":[{\"dataset_identifier\":\"n/a\",\"repository_reference\":\"n/a\"},{\"dataset_identifier\":\"PMC8859891\",\"repository_reference\":\"https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891/\"},{\"dataset_identifier\":\"10.1186/s13058-022-01509-z\",\"repository_reference\":\"https://doi.org/10.1186/s13058-022-01509-z\"}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'PMC8859891', 'repository_reference': 'https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891/'}, {'dataset_identifier': '10.1186/s13058-022-01509-z', 'repository_reference': 'https://doi.org/10.1186/s13058-022-01509-z'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 3 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: PMC8859891\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: 10.1186/s13058-022-01509-z\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}, {'dataset_identifier': 'PMC8859891', 'repository_reference': 'https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891/', 'dataset_id': 'PMC8859891'}, {'dataset_identifier': '10.1186/s13058-022-01509-z', 'repository_reference': 'https://doi.org/10.1186/s13058-022-01509-z', 'dataset_id': '10.1186/s13058-022-01509-z'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 677 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 605 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'PMC8859891', 'repository_reference': 'https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891/', 'dataset_id': 'PMC8859891'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: PMC8859891\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID PMC8859891 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891/, identifier: PMC8859891\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier PMC8859891 for https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier PMC8859891 for https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier PMC8859891 for https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier PMC8859891 for https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier PMC8859891 for https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'PMC8859891', 'data_repository': 'pmc.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': '10.1186/s13058-022-01509-z', 'repository_reference': 'https://doi.org/10.1186/s13058-022-01509-z', 'dataset_id': '10.1186/s13058-022-01509-z'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(10\\.\\d{4,9}/).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: 10.1186/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: 10.1186/s13058-022-01509-z, doi.org, https://doi.org/10.1186/s13058-022-01509-z\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: 10.1186/s13058-022-01509-z\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': '10.1186/s13058-022-01509-z', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'PMC8859891', 'data_repository': 'pmc.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}, {'dataset_identifier': '10.1186/s13058-022-01509-z', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 2 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: PMC8859891\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: pmc.ncbi.nlm.nih.gov, identifier: PMC8859891\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier PMC8859891 for pmc.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier PMC8859891 for pmc.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier PMC8859891 for pmc.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier PMC8859891 for pmc.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier PMC8859891 for pmc.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: pmc.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 976 - WARNING - URL pmc.ncbi.nlm.nih.gov may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for PMC8859891 in pmc.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 1 with repo: pmc.ncbi.nlm.nih.gov and accession_id: PMC8859891\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1254 - WARNING - Repository pmc.ncbi.nlm.nih.gov unknown in Ontology. Skipping dataset page 1.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: 10.1186/s13058-022-01509-z\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: doi.org, identifier: 10.1186/s13058-022-01509-z\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.1186/s13058-022-01509-z for doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for 10.1186/s13058-022-01509-z in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 2 with repo: doi.org and accession_id: 10.1186/s13058-022-01509-z\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://doi.org/10.1186/s13058-022-01509-z\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 2\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 8, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage', 'link',\n",
      "       'title', 'file_info', 'description', 'source_section', 'section_class',\n",
      "       'download_link', 'file_extension', 'a_attr_href', 'a_attr_class',\n",
      "       'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 640 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - Parsed data extraction completed. Elements collected: 3\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 481 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 574 - INFO - \n",
      "Progress: 41/100 (41.00%) | Elapsed: 00:11:10 | ETA: 00:16:04\n",
      "\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 41th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC9853518)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC9853518\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC9853518&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: supplementary_data_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 394 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC9853518.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b711a050>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC9853518 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC9853518, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 445 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC9853518)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 547 - INFO - Extracted 3 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 557 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#s6'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 45) for s6: Data availability statement [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#fn-group1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 27) for fn-group1: Footnotes [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 41970\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 140004\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 42692 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-05e2a1400df697d2dcacf35463bca0575bf5234419f8601cf7274732b8712d4a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 42692 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\n",
      "  \"datasets\": [\n",
      "    {\n",
      "      \"dataset_identifier\": \"Covid20\",\n",
      "      \"repository_reference\": \"https://zenodo.org/record/3757476#.Y1NGmy1h1B1\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"Placenta\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"Synapse\",\n",
      "      \"repository_reference\": \"https://www.synapse.org#!Synapse:syn3193805/wiki/217789\"\n",
      "    }\n",
      "  ]\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'Covid20', 'repository_reference': 'https://zenodo.org/record/3757476#.Y1NGmy1h1B1'}, {'dataset_identifier': 'Placenta', 'repository_reference': 'n/a'}, {'dataset_identifier': 'Synapse', 'repository_reference': 'https://www.synapse.org#!Synapse:syn3193805/wiki/217789'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 3 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: Covid20\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: Placenta\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'Covid20', 'repository_reference': 'https://zenodo.org/record/3757476#.Y1NGmy1h1B1', 'dataset_id': 'Covid20'}, {'dataset_identifier': 'Placenta', 'repository_reference': 'n/a', 'dataset_id': 'Placenta'}, {'dataset_identifier': 'Synapse', 'repository_reference': 'https://www.synapse.org#!Synapse:syn3193805/wiki/217789', 'dataset_id': 'Synapse'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'Covid20', 'repository_reference': 'https://zenodo.org/record/3757476#.Y1NGmy1h1B1', 'dataset_id': 'Covid20'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: Covid20\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID Covid20 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://zenodo.org/record/3757476#.Y1NGmy1h1B1, identifier: Covid20\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://zenodo.org/record/3757476#.Y1NGmy1h1B1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Covid20 for https://zenodo.org/record/3757476#.Y1NGmy1h1B1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier Covid20 for https://zenodo.org/record/3757476#.Y1NGmy1h1B1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier Covid20 for https://zenodo.org/record/3757476#.Y1NGmy1h1B1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Covid20 for https://zenodo.org/record/3757476#.Y1NGmy1h1B1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Covid20 for https://zenodo.org/record/3757476#.Y1NGmy1h1B1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://zenodo.org/record/3757476#.Y1NGmy1h1B1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'Covid20', 'data_repository': 'zenodo.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'Placenta', 'repository_reference': 'n/a', 'dataset_id': 'Placenta'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: Placenta\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID Placenta is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: n/a, identifier: Placenta\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Placenta for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier Placenta for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier Placenta for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Placenta for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Placenta for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 677 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 605 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'Placenta', 'repository_reference': 'n/a', 'dataset_id': 'Placenta'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'Synapse', 'repository_reference': 'https://www.synapse.org#!Synapse:syn3193805/wiki/217789', 'dataset_id': 'Synapse'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn3193805\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn3193805, None, https://www.synapse.org#!Synapse:syn3193805/wiki/217789\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn3193805\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.synapse.org#!Synapse:syn3193805/wiki/217789, identifier: syn3193805\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org#!Synapse:syn3193805/wiki/217789\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3193805 for https://www.synapse.org#!Synapse:syn3193805/wiki/217789\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn3193805 for https://www.synapse.org#!Synapse:syn3193805/wiki/217789\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn3193805 for https://www.synapse.org#!Synapse:syn3193805/wiki/217789\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3193805 for https://www.synapse.org#!Synapse:syn3193805/wiki/217789\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Found partial match for https://www.synapse.org#!Synapse:syn3193805/wiki/217789 in Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn3193805', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'Covid20', 'data_repository': 'zenodo.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn3193805', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 2 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: Covid20\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: zenodo.org, identifier: Covid20\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Covid20 for zenodo.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier Covid20 for zenodo.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier Covid20 for zenodo.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Covid20 for zenodo.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Covid20 for zenodo.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: zenodo.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 976 - WARNING - URL zenodo.org may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for Covid20 in zenodo.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 1 with repo: zenodo.org and accession_id: Covid20\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1254 - WARNING - Repository zenodo.org unknown in Ontology. Skipping dataset page 1.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn3193805\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn3193805\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3193805 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn3193805 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn3193805 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3193805 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn3193805 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn3193805\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn3193805\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 2\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 5, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 640 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - Parsed data extraction completed. Elements collected: 3\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 481 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 42th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11852190)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11852190\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC11852190&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 461 - WARNING - No links found in the section.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: supplementary_data_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 394 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC11852190.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b7063ed0>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11852190 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11852190, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 445 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC11852190)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 547 - INFO - Extracted 3 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 557 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#notes4'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 45) for notes4: Data Availability Statement [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 35899\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 132773\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 36621 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-61e28e35fc5c7a8f942103ff13ea2edd9cafeb809705a957562174ebdc908600\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 36621 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\"datasets\":[{\"dataset_identifier\":\"n/a\",\"repository_reference\":\"n/a\"},{\"dataset_identifier\":\"syn3193805\",\"repository_reference\":\"https://www.synapse.org/#!Synapse:syn3193805/wiki/217789\"}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'syn3193805', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn3193805/wiki/217789'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 2 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn3193805\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}, {'dataset_identifier': 'syn3193805', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn3193805/wiki/217789', 'dataset_id': 'syn3193805'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 677 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 605 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn3193805', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn3193805/wiki/217789', 'dataset_id': 'syn3193805'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn3193805\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn3193805, synapse.org, https://www.synapse.org/#!Synapse:syn3193805/wiki/217789\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn3193805\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn3193805', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn3193805', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 1 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn3193805\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn3193805\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3193805 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn3193805 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn3193805 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3193805 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn3193805 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn3193805\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn3193805\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 1\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 4, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 640 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - Parsed data extraction completed. Elements collected: 2\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 481 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 43th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10652415)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10652415\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC10652415&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 461 - WARNING - No links found in the section.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 406 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC10652415.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC10652415 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 430 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC10652415, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 443 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 264 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 454 - INFO - Found 2 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b7611a40>, <Element supplementary-material at 0x1b7610640>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 462 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b7611a40>, <Element supplementary-material at 0x1b7610640>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 548 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref SM1: We constructed a linear regression model to investigate the correlation between CNVs and the age of death (AOD) of AD cases based on copy number profiles and meta-information of 1,127 North American white individuals from the ROSMAP AD cohort ( Chibnik et al., 2018 [rid=ref12] ; De Jager et al., 2018 [rid=ref15] ; Mostafavi et al., 2018 [rid=ref42] ; Lee et al., 2023 [rid=ref32] ) ( Supplementary Table S1 [this file] and Supplementary Figure S1 [rid=SM2] ), while sex and disease status were treated as covariates (Methods 4.5, Figure 2 [rid=fig2] ). \n",
      "We found all 10 AOD-correlated CNVs with low frequency in both AD and healthy control groups, and they showed no significant frequency difference in both groups ( Supplementary Table S2 [this file] ). They also showed rare frequency in human populations ( Supplementary Table S2 [this file] ). \n",
      "Based on the linear additive model with sex and disease status as covariates, there were 110 genes identified with their expression level significantly correlated with copy number dosage of the 10 AOD-correlated CNVs under the FDR threshold at 5%, forming 125 CNV-Gene pairs ( Supplementary Table S3 [this file] ). \n",
      "There were 2,056, 1,294, and 2,774 AOD-correlated genes identified based on cogdx, braaksc, and ceradsc respectively, under the genome-wide FDR threshold of 0.05 ( Supplementary Table S4 [this file] ). \n",
      "We further constructed correlation network for above CNV-gene-AOD pairs. ( Figure 4 [rid=fig4] ; Supplementary Table S5 [this file] ; Methods 4.9). \n",
      "Differential expression analysis showed and had no difference between the AD and normal groups indicating that and ’s contribution to longevity was independent of AD status ( Supplementary Table S8 [this file] ). \n",
      "The original contributions presented in the study are included in the article/ Supplementary material [this file] , further inquiries can be directed to the corresponding author. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref SM2: We constructed a linear regression model to investigate the correlation between CNVs and the age of death (AOD) of AD cases based on copy number profiles and meta-information of 1,127 North American white individuals from the ROSMAP AD cohort ( Chibnik et al., 2018 [rid=ref12] ; De Jager et al., 2018 [rid=ref15] ; Mostafavi et al., 2018 [rid=ref42] ; Lee et al., 2023 [rid=ref32] ) ( Supplementary Table S1 [rid=SM1] and Supplementary Figure S1 [this file] ), while sex and disease status were treated as covariates (Methods 4.5, Figure 2 [rid=fig2] ). These findings each maintained a false discovery rate (FDR) of less than 0.05 ( Supplementary Figure S2 [this file] ). \n",
      "Copy number dosage had a significant influence on AOD and that was consistent with our correlation result between these 10 CNVs and AOD, except mCNV62126 ( Supplementary Figure S3 [this file] ). \n",
      "The transcriptomic profile was further adjusted for AD pathological traits using limma R package ( Ritchie et al., 2015 [rid=ref46] ) to exclude bias from disease status ( Supplementary Figure S4 [this file] , Methods 4.8). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 243 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 38229\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 141409\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 38951 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-949a3a093eee0e3289c4270d75ef8d22d284828ff1ab2edde7c83e8c2467caa8\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 38951 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\n",
      "  \"datasets\": [\n",
      "    {\n",
      "      \"dataset_identifier\": \"syn3388564\",\n",
      "      \"repository_reference\": \"https://www.synapse.org/#!Synapse:syn3388564\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"syn3157322\",\n",
      "      \"repository_reference\": \"https://www.synapse.org/#!Synapse:syn3157322\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"http://doi.org/10.7303/syn26254632\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"https://adknowledgeportal.org\"\n",
      "    }\n",
      "  ]\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'syn3388564', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn3388564'}, {'dataset_identifier': 'syn3157322', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn3157322'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'http://doi.org/10.7303/syn26254632'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://adknowledgeportal.org'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 5 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn3388564\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn3157322\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'syn3388564', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn3388564', 'dataset_id': 'syn3388564'}, {'dataset_identifier': 'syn3157322', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn3157322', 'dataset_id': 'syn3157322'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'http://doi.org/10.7303/syn26254632'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://adknowledgeportal.org'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn3388564', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn3388564', 'dataset_id': 'syn3388564'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn3388564\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn3388564, synapse.org, https://www.synapse.org/#!Synapse:syn3388564\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn3388564\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn3388564', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn3157322', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn3157322', 'dataset_id': 'syn3157322'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn3157322\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn3157322, synapse.org, https://www.synapse.org/#!Synapse:syn3157322\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn3157322\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn3157322', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 677 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 605 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'http://doi.org/10.7303/syn26254632'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(10\\.\\d{4,9}/).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: 10.7303/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn26254632\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn26254632, None, http://doi.org/10.7303/syn26254632\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn26254632\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: http://doi.org/10.7303/syn26254632, identifier: syn26254632\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: http://doi.org/10.7303/syn26254632\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn26254632 for http://doi.org/10.7303/syn26254632\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn26254632 for http://doi.org/10.7303/syn26254632\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn26254632 for http://doi.org/10.7303/syn26254632\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn26254632 for http://doi.org/10.7303/syn26254632\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier syn26254632 for http://doi.org/10.7303/syn26254632\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1153 - INFO - Found id_pattern match for http://doi.org/10.7303/syn26254632 in syn\\d{7,12}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn26254632', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://adknowledgeportal.org'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://adknowledgeportal.org, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://adknowledgeportal.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://adknowledgeportal.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'adknowledgeportal.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 256 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn3388564', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn3157322', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn26254632', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'adknowledgeportal.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 4 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn3388564\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn3388564\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3388564 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn3388564 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn3388564 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3388564 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn3388564 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn3388564\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn3388564\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn3157322\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn3157322\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3157322 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn3157322 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn3157322 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3157322 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn3157322 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn3157322\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn3157322\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn26254632\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn26254632\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn26254632 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn26254632 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn26254632 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn26254632 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn26254632 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 3 with repo: synapse.org and accession_id: syn26254632\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn26254632\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 3: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Skipping dataset 4: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 4\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 265 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 6, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 477 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 44th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC7842189)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC7842189\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC7842189&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 406 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC7842189.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC7842189 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 430 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC7842189, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 443 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 264 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 454 - INFO - Found 5 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b725d880>, <Element supplementary-material at 0x1b038e040>, <Element supplementary-material at 0x1b038e080>, <Element supplementary-material at 0x1b038ca40>, <Element supplementary-material at 0x1b038db80>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 462 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b725d880>, <Element supplementary-material at 0x1b038e040>, <Element supplementary-material at 0x1b038e080>, <Element supplementary-material at 0x1b038ca40>, <Element supplementary-material at 0x1b038db80>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 548 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref SD1: We identified 13 distinct, robust co-expression modules, including 7 modules enriched for microglial genes, but varying in their relationship to disease, cell type, and temporal trajectories; we call these microglial neurodegeneration-associated modules, MNMs ( Figures 1 [rid=F1] , S1 [this file] , and S2 [this file] ; Table S1 [rid=SD2] ). We compared them with six independent microglia-specific gene expression datasets from transgenic mouse models and humans with AD pathology to identify overlapping patterns of gene expression ( Figures 1B [rid=F1] , S1C [this file] – S1E [this file] , and S2F [this file] ; Table S2 [rid=SD3] ), supporting their generalizability. \n",
      "We confirmed that the MNMs include expected traits indicative of neuroimmune activity, including: (1) known neuroregulatory pathways, such as complement activation ( Schafer et al., 2013 [rid=R74] ), Toll-like receptor (TLR), and interleukin-1β (IL-1β) ( Heneka et al., 2013 [rid=R32] ; Figure 1E [rid=F1] ); (2) strong association with tau pathology, a neuronal driver of neurodegeneration ( Figures 1F [rid=F1] and S2A [this file] ); and (3) high overlap with the conserved neuroinflammatory module we identified in our prior study (NAI; Swarup et al., 2019 [rid=R85] ; Figures S2B [this file] and S2C [this file] ). \n",
      "In addition, all MNMs are preserved in three different transgenic mouse models expressing human mutations ( Swarup et al., 2019 [rid=R85] ; Figure S2D [this file] ) and in microglial-specific datasets from mouse models expressing ( Srinivasan et al., 2016 [rid=R80] ) and ( Wang et al., 2015 [rid=R90] ) mutations, with the exception of M_UP3, which is only weakly preserved in one of the two datasets from mice expressing early AD mutations ( Figure S2E [this file] ). A few modules display variability in their differential expression in different transgenic models, suggesting that they may be conditional on mutational interactions with disease-stage and disease-specific pathology, or possibly by other technical factors distinguishing these mouse models ( Figure S2F [this file] ). \n",
      "Finally, we observed an increased expression of genes involved in viral response and type II IFN response represented by M_UP3 ( Figures 1E [rid=F1] , 2A [rid=F2] , 2B [rid=F2] , and S2G [this file] ). \n",
      "We identified the earliest interconnected MNM genes present in pre-symptomatic disease tissue and called them early MNM sub-modules, reasoning that casual disease pathways should enrich among the earliest MNM components in disease ( Figure S3 [this file] ; STAR Methods [rid=S10] ). We confirmed significant associations using exome array data ( Chen et al., 2015 [rid=R9] ), replicating significant associations between AD and M_UP3 (β= 0.19, p < 0.001), FTD and early_UP1 (β= 0.25, p < 0.001), and FTD and early_DOWN1 (β= 0.15 p < 0.001) ( Figure S3G [this file] ). \n",
      "Consistent with these causal relationships defined by genetic enrichment, we were able to demonstrate that the expression of genes involved with viral defense, including the type I IFN response and dsRNA binding, were highly associated with the level of pathological tau phosphorylation in the brains of TPR50 mice ( Figures S3H [this file] – S3J [this file] ). \n",
      "We found that gene overexpression has inverse effects on modules A and B ( Figure 4D [rid=F4] ; 2,161 genes; STAR Methods [rid=S10] ), which we also confirmed at the levels of gene connectivity ( Figure S4A [this file] ) and PPI ( Figure S4B [this file] ). These patterns were confirmed in 12 independent transcriptome datasets ( Figures S4D [this file] – S4I [this file] ), indicating that these two modules represent robust, early microglia responses mediating opposing signaling pathways. \n",
      "Module A includes protein complexes and pathways of microglial innate neuroimmune activation, including the NLRP3 inflammasome ( , , , , and ) ( Ahmed et al., 2017 [rid=R2] ; Ising et al., 2019 [rid=R38] ; Lučiūnaitė et al., 2019 [rid=R54] ), TLRs ( , , , and ), and nuclear factor κB (NF-κB) targets ( Figures 4E [rid=F4] , 4G [rid=F4] , and S4A [this file] ). Supporting this interpretation, we found that purified pathological Aβ, an extracellular inducer of pro-inflammatory microglial activity ( Lučiūnaitė et al., 2019 [rid=R54] ), stimulates NAct in microglia ( Figures 4F [rid=F4] and S4C [this file] ). In contrast, module B includes viral and stress response genes, drivers of immune suppression ( [PD-L1], , , , and ), and the senescence marker, ( Figures 4H [rid=F4] and S4A [this file] ). \n",
      "Several features of NSupp suggest that in the context of tauopathy it may also represent an immunosuppressive type of IFN-β signaling in disease-associated microglia that are expressing inhibitory immune checkpoint genes (e.g., [PDL-1], , and ) ( Snell et al., 2017 [rid=R78] ; Wykes and Lewin, 2018 [rid=R94] ) and inhibitors of immune hyperactivity (e.g., ; Basters et al., 2018 [rid=R4] ; Taylor et al., 2018 [rid=R87] ; Figures 4G [rid=F4] , 4H [rid=F4] , and S4A [this file] ). \n",
      "NAct is activated by extracellular stimuli working through TLR and related receptors ( Figures 4E [rid=F4] , 4F [rid=F4] , S4A [this file] , and S4C [this file] ). \n",
      "Then, as tau pathology leads to chromatin and nucleotide dysregulation before neuronal loss, nucleotide damage signals activate dsRNA receptors to trigger a microglial type I IFN response that drives the expression of NSupp pathways involved in stress response, exosome, and viral entry ( Figure 4H [rid=F4] ), while suppressing the specific microglial immune attack pathways represented by NAct, including the NLRP3 inflammasome and TLR1 and −2 signaling ( Figures 4E [rid=F4] and S4A [this file] ). As disease progresses to include frank neurodegeneration, microglia undergo additional dynamic states, including increased type II IFN signaling, exogenous antigen presentation, T cell regulation, and cell death ( Figures 1E [rid=F1] , 2A [rid=F2] , 2B [rid=F2] , and S2G [this file] ). \n",
      "We pre-treated human-induced pluripotent stem cell (iPSC)-derived microglia ( Haenseler et al., 2017 [rid=R29] ; STAR Methods [rid=S10] ) with IFN-β and tested the ability of fibrillar Aβ, a NAct activator ( Figures 4F [rid=F4] and S4C [this file] ), to induce the secretion of IL-1β, a prominent member of the NAct pathway in these human cells ( Figures 4E [rid=F4] and 4G [rid=F4] ). \n",
      "Another prediction based on the mouse microglia module trajectory analysis ( Figure 2 [rid=F2] ) is a delayed, biphasic type II IFN response with early suppression and late increase (M_DOWN1, M_UP3; Figures 1C [rid=F1] , 1E [rid=F1] , 2A [rid=F2] , 2B [rid=F2] , and S2G [this file] ). Remarkably, we observed concordance between disease-associated signaling pathways represented in M_UP3 and microglia from patients with FTD-Pick’s; in particular in type II IFN, a specific activator of M_UP3 ( Figures 2A [rid=F2] , 2B [rid=F2] , 6C [rid=F6] , and S2G [this file] ). Significantly upregulated in bvFTD microglia are genes that can stimulate IFN-γ production ( , , , and ) ( Renkl et al., 2005 [rid=R66] ) and genes that mediate IFN-γ response ( , , , and ; Figures 6C [rid=F6] , 6D [rid=F6] , and S5E [this file] ). \n",
      "We find that both dsRNA detection and IFN pathways are highly correlated with pathological tau burden in brain ( Figures S3H [this file] – S3J [this file] ) and both the cytosolic dsRNA receptor (MDA5) and RIG1 pathway are hubs of the NSupp module ( Figure 4H [rid=F4] ). \n",
      "The significance of intramodular connectivity was assessed for each module using a permutation test (10,000 permutations), and all modules were confirmed to have permuted p value < 0.001. “Early submodules,” described in Figures 3 [rid=F3] and S3 [this file] , were derived by re-clustering M_UP1 and M_UP2 genes to generate “earlyUP” modules, or M_DOWN1, M_DOWN2 and M_DOWN3 genes to generate “earlyDOWN” modules, using the 3 month of age frontal cortex TPR50 gene expression data ( Swarup et al., 2019 [rid=R85] ). “NSupp and NAct,” described in Figures 4 [rid=F4] and 5 [rid=F5] , were derived from re-clustering the M_UP1 and M_UP2 genes, keeping only genes with annotated PPI in either Bio-grid ( Stark et al., 2006 [rid=R82] ) or Inweb databases ( Rossin et al., 2011 [rid=R69] ), using the same gene expression data used for the consensus WGCNA analysis (purified microglia from the Tg4510 model and frontal cortex TPR50 dataset (6 months of age)). \n",
      "Each cluster was then annotated as a major cell type using mean expression of groups of cell type marker genes, including the microglial markers CD74, CSF1R, C3 ( Mathys et al., 2019 [rid=R58] ), PTPRC, and DOCK8 ( Kelley et al., 2018 [rid=R42] ) ( Figure S5A [this file] ). \n",
      "In addition, to check that our findings were representative of group effects rather than driven by a few individuals, we compared all disease and control samples for average gene expression and total number of microglia contributed to the analysis ( Figure S5E [this file] ). \n",
      "For all boxplots, the center line is at the median and the upper and lower lines are at the 75 and 25 percentiles, respectively ( Figures 4I [rid=F4] , 5C [rid=F5] , 5F [rid=F5] , 5K [rid=F5] , S3B [this file] , S3C [this file] , S4G [this file] – S4I [this file] , and S5C [this file] ). Results of tissue culture experiments show each sample data point as a separate dot, the center line at the group mean, and error bars showing s.e.m. ( Figures 5D [rid=F5] , 5G [rid=F5] , and 5I [rid=F5] ) The barplot in Figure S2G [this file] has the center line at the group mean and error bars as s.e.m. In Figure S5D [this file] , the center line of the boxplot is the sample mean, the upper and lower lines are the 75 and 25 percentiles, respectively, and each dot is the average gene expression among the microglia from one subject, with the size of the dot proportional to the number of microglia contributing to the final analysis from that subject. \n",
      "See also Figure S1 [this file] . \n",
      "See also Figure S2 [this file] . \n",
      "See also Figure S3 [this file] . \n",
      "See also Figure S4 [this file] . \n",
      "See also Figure S5 [this file] and Table S3 [rid=SD4] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref SD2: We identified 13 distinct, robust co-expression modules, including 7 modules enriched for microglial genes, but varying in their relationship to disease, cell type, and temporal trajectories; we call these microglial neurodegeneration-associated modules, MNMs ( Figures 1 [rid=F1] , S1 [rid=SD1] , and S2 [rid=SD1] ; Table S1 [this file] ). \n",
      "To further explore the early microglial response in disease, we re-clustered only the genes from M_UP1 and M_UP2 to identify additional co-expression relationships among them ( Figure 4A [rid=F4] , Module A and Module B; Table S1 [this file] ). \n",
      "We next used the CMAP database to find candidate drivers of NSupp and NAct ( STAR Methods [rid=S10] ), identifying genes that regulate the viral activation of type I IFN among the top drivers of NSupp and suppressors of NAct ( ; Marazzi et al., 2012 [rid=R56] ; Table S1 [this file] ). \n",
      "For our analysis, we used the mean “connectivity scores” which is calculated from the combining data generated independently in 9 cell lines (see Table S1 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref SD3: We compared them with six independent microglia-specific gene expression datasets from transgenic mouse models and humans with AD pathology to identify overlapping patterns of gene expression ( Figures 1B [rid=F1] , S1C [rid=SD1] – S1E [rid=SD1] , and S2F [rid=SD1] ; Table S2 [this file] ), supporting their generalizability. \n",
      "To explore their preservation across neurodegenerative dementias, we tested module preservation in multiple mouse and human disease datasets (total n = 525 samples from 17 datasets; STAR Methods [rid=S10] ; Table S2 [this file] ), finding significant preservation in transcriptome data from post-mortem human brain tissue from AD ( Allen et al., 2016 [rid=R3] ), FTD ( Chen-Plotkin et al., 2008 [rid=R10] ; Swarup et al., 2019 [rid=R85] ), and PSP ( Allen et al., 2016 [rid=R3] ) patients ( Figure 1G [rid=F1] ). \n",
      "Samples were balanced age (mean: 68 years old -Pick’s, 66 years old -control), PMI (mean: 15.98 hours – Pick’s, 12.78 hours - control) and tissue RNA integrity number (RIN; mean: 6.1 -Pick’s, 5.6 -controls) ( Table S2 [this file] ). \n",
      "Additional publicly available datasets were used throughout the study for validation or comparison ( Table S2 [this file] ; and Key Resources Table [rid=T1] ). \n",
      "One outlier sample was identified based on abnormal frequencies of major cell types and divergent gene expression patterns, and was removed from the analysis ( Table S2 [this file] ). \n",
      "While RIN, PMI and age were not significantly different between case and control subjects (Mann Whitney U test; Table S2 [this file] ); sex and brank bank were not evenly split across groups (UCSF brain bank –bvFTD = 70%, control = 12%; male sex –bvFTD = 62%, controls = 50% ( Table S2 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref SD4: We used the top 250 genes associated with bvFTD (at significance thresholds of beta > 0.1 and p-adjusted < 5.0 E-04) for further analysis ( Table S3 [this file] ). We report the effect size output from the linear models as beta; which is approximate to the log2-fold change between groups ( Table S3 [this file] ). \n",
      "We identified all genes significantly associated with either UCSF brain bank or sex using the thresholds applied to diagnosis (beta 0.1, p value < 1.0 E-05, p-adjusted < 5.0 E-04) ( Table S3 [this file] ) and excluded them from further analysis. No genes were significantly associated with PMI or number of genes or UMI detected; and genes found to be significantly associated with RIN ( Table S3 [this file] ) were excluded from further analysis (beta >0.1, p value < 1.0 E-05, p-adjusted < 5.0 E-04). \n",
      "See also Figure S5 [rid=SD1] and Table S3 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref SD5: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 243 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 79068\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 278849\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 79790 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-5c2919bbe5c82fbd02b8e923953b65117b1a11f215d752b32700804117f861b2\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 79790 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\n",
      "  \"datasets\": [\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE146866\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE146866\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"phs002197.v1.p1\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs002197.v1.p1\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE90696\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE90696\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE75431\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE75431\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE61499\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE61499\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE65067\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65067\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE13162\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE13162\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE138852\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE138852\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE121654\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE121654\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE98401\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98401\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE146866\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE146866\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE98969\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98969\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/assembly/GCF_000001635.20\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"https://www.gencodegenes.org/mouse/release_M11.html\"\n",
      "    }\n",
      "  ]\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'GSE146866', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE146866'}, {'dataset_identifier': 'phs002197.v1.p1', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs002197.v1.p1'}, {'dataset_identifier': 'GSE90696', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE90696'}, {'dataset_identifier': 'GSE75431', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE75431'}, {'dataset_identifier': 'GSE61499', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE61499'}, {'dataset_identifier': 'GSE65067', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65067'}, {'dataset_identifier': 'GSE13162', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE13162'}, {'dataset_identifier': 'GSE138852', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE138852'}, {'dataset_identifier': 'GSE121654', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE121654'}, {'dataset_identifier': 'GSE98401', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98401'}, {'dataset_identifier': 'GSE146866', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE146866'}, {'dataset_identifier': 'GSE98969', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98969'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/assembly/GCF_000001635.20'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.gencodegenes.org/mouse/release_M11.html'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 14 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE146866\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: phs002197.v1.p1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE90696\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE75431\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE61499\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE65067\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE13162\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE138852\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE121654\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE98401\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 719 - INFO - Duplicate found and skipped: GSE146866\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE98969\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'GSE146866', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE146866', 'dataset_id': 'GSE146866'}, {'dataset_identifier': 'phs002197.v1.p1', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs002197.v1.p1', 'dataset_id': 'phs002197.v1.p1'}, {'dataset_identifier': 'GSE90696', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE90696', 'dataset_id': 'GSE90696'}, {'dataset_identifier': 'GSE75431', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE75431', 'dataset_id': 'GSE75431'}, {'dataset_identifier': 'GSE61499', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE61499', 'dataset_id': 'GSE61499'}, {'dataset_identifier': 'GSE65067', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65067', 'dataset_id': 'GSE65067'}, {'dataset_identifier': 'GSE13162', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE13162', 'dataset_id': 'GSE13162'}, {'dataset_identifier': 'GSE138852', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE138852', 'dataset_id': 'GSE138852'}, {'dataset_identifier': 'GSE121654', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE121654', 'dataset_id': 'GSE121654'}, {'dataset_identifier': 'GSE98401', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98401', 'dataset_id': 'GSE98401'}, {'dataset_identifier': 'GSE98969', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98969', 'dataset_id': 'GSE98969'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/assembly/GCF_000001635.20', 'dataset_id': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.gencodegenes.org/mouse/release_M11.html'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE146866', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE146866', 'dataset_id': 'GSE146866'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE146866\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE146866, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE146866\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE146866\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE146866', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'phs002197.v1.p1', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs002197.v1.p1', 'dataset_id': 'phs002197.v1.p1'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: phs002197.v1.p1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: phs002197.v1.p1, dbgap, https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs002197.v1.p1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: phs002197.v1.p1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: dbgap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'phs002197.v1.p1', 'data_repository': 'dbgap', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE90696', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE90696', 'dataset_id': 'GSE90696'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE90696\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE90696, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE90696\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE90696\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE90696', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE75431', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE75431', 'dataset_id': 'GSE75431'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE75431\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE75431, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE75431\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE75431\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE75431', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE61499', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE61499', 'dataset_id': 'GSE61499'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE61499\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE61499, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE61499\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE61499\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE61499', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE65067', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65067', 'dataset_id': 'GSE65067'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE65067\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE65067, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65067\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE65067\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE65067', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE13162', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE13162', 'dataset_id': 'GSE13162'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE13162\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE13162, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE13162\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE13162\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE13162', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE138852', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE138852', 'dataset_id': 'GSE138852'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE138852\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE138852, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE138852\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE138852\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE138852', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE121654', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE121654', 'dataset_id': 'GSE121654'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE121654\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE121654, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE121654\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE121654\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE121654', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE98401', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98401', 'dataset_id': 'GSE98401'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE98401\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE98401, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98401\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE98401\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE98401', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE98969', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98969', 'dataset_id': 'GSE98969'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE98969\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE98969, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98969\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE98969\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE98969', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/assembly/GCF_000001635.20', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.ncbi.nlm.nih.gov/assembly/GCF_000001635.20, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.ncbi.nlm.nih.gov/assembly/GCF_000001635.20\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://www.ncbi.nlm.nih.gov/assembly/GCF_000001635.20\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'www.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.gencodegenes.org/mouse/release_M11.html'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.gencodegenes.org/mouse/release_M11.html, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.gencodegenes.org/mouse/release_M11.html\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://www.gencodegenes.org/mouse/release_M11.html\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'www.gencodegenes.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 256 - INFO - Augmented dataset links: [{'dataset_identifier': 'GSE146866', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'phs002197.v1.p1', 'data_repository': 'dbgap', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE90696', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE75431', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE61499', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE65067', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE13162', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE138852', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE121654', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE98401', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE98969', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'www.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'www.gencodegenes.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 13 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE146866\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE146866\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE146866 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE146866 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE146866 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE146866\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE146866\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: phs002197.v1.p1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: dbgap, identifier: phs002197.v1.p1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier phs002197.v1.p1 for dbgap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier phs002197.v1.p1 for dbgap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier phs002197.v1.p1 for dbgap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: dbgap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for phs002197.v1.p1 in dbgap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 2 with repo: dbgap and accession_id: phs002197.v1.p1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs002197.v1.p1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 2: Application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE90696\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE90696\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE90696 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE90696 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE90696 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 3 with repo: geo and accession_id: GSE90696\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE90696\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 3: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE75431\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE75431\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE75431 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE75431 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE75431 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 4 with repo: geo and accession_id: GSE75431\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE75431\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 4: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE61499\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE61499\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE61499 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE61499 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE61499 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 5 with repo: geo and accession_id: GSE61499\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE61499\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 5: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE65067\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE65067\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE65067 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE65067 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE65067 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 6 with repo: geo and accession_id: GSE65067\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65067\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 6: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE13162\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE13162\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE13162 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE13162 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE13162 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 7 with repo: geo and accession_id: GSE13162\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE13162\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 7: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE138852\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE138852\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE138852 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE138852 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE138852 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 8 with repo: geo and accession_id: GSE138852\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE138852\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 8: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE121654\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE121654\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE121654 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE121654 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE121654 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 9 with repo: geo and accession_id: GSE121654\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE121654\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 9: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 10 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE98401\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE98401\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE98401 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE98401 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE98401 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 10 with repo: geo and accession_id: GSE98401\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98401\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 10: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 11 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE98969\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE98969\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE98969 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE98969 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE98969 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 11 with repo: geo and accession_id: GSE98969\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98969\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 11: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 12 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Skipping dataset 12: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 13 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Skipping dataset 13: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 13\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 265 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 18, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 477 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 45th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC3788619)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC3788619\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC3788619&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 394 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC3788619.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b808e510>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC3788619 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC3788619, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 445 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC3788619)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 547 - INFO - Extracted 7 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 557 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#sec3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for sec3: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#sec9'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for sec9: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#d33e24'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for d33e24: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#d33e30'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for d33e30: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#d33e36'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for d33e36: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 41860\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 140130\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 42582 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-c14e7c9a3b80bb86b42c6d411b21155fa07450f3cf29ba6317cd707232fe7ebc\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 42582 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\n",
      "  \"datasets\": [\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"https://doi.org/10.1038/srep02651\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"http://www.synapse.org\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Rep%22%5Bjour%5D\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Rep%22%5Bjour%5D\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Rep%22%5BTitle%20Abbreviation%5D\"\n",
      "    }\n",
      "  ]\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://doi.org/10.1038/srep02651'}, {'dataset_identifier': 'n/a', 'repository_reference': 'http://www.synapse.org'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Rep%22%5Bjour%5D'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Rep%22%5Bjour%5D'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Rep%22%5BTitle%20Abbreviation%5D'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 6 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://doi.org/10.1038/srep02651'}, {'dataset_identifier': 'n/a', 'repository_reference': 'http://www.synapse.org'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Rep%22%5Bjour%5D'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Rep%22%5Bjour%5D'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Rep%22%5BTitle%20Abbreviation%5D'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 677 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 605 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://doi.org/10.1038/srep02651'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(10\\.\\d{4,9}/).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: 10.1038/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: 10.1038/, None, https://doi.org/10.1038/srep02651\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: 10.1038/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://doi.org/10.1038/srep02651, identifier: 10.1038/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://doi.org/10.1038/srep02651\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.1038/ for https://doi.org/10.1038/srep02651\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 10.1038/ for https://doi.org/10.1038/srep02651\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1153 - INFO - Found id_pattern match for https://doi.org/10.1038/srep02651 in 10\\.\\d{4,9}/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': '10.1038/', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'http://www.synapse.org'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: http://www.synapse.org, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: http://www.synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Found partial match for http://www.synapse.org in Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 673 - INFO - Dataset ID is 'n/a' and repository name from prompt\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 605 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'n/a', 'repository_reference': 'http://www.synapse.org'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Rep%22%5Bjour%5D'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Rep%22%5Bjour%5D, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Rep%22%5Bjour%5D\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Rep%22%5Bjour%5D\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'www.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Rep%22%5Bjour%5D'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Rep%22%5Bjour%5D, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Rep%22%5Bjour%5D\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Rep%22%5Bjour%5D\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'pubmed.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Rep%22%5BTitle%20Abbreviation%5D'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Rep%22%5BTitle%20Abbreviation%5D, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Rep%22%5BTitle%20Abbreviation%5D\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Rep%22%5BTitle%20Abbreviation%5D\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'www.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': '10.1038/', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'www.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'pubmed.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'www.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 4 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: 10.1038/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: doi.org, identifier: 10.1038/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.1038/ for doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for 10.1038/ in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 1 with repo: doi.org and accession_id: 10.1038/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://doi.org/10.1038/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Skipping dataset 2: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Skipping dataset 3: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Skipping dataset 4: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 4\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 11, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage', 'link',\n",
      "       'title', 'file_info', 'description', 'source_section', 'section_class',\n",
      "       'download_link', 'file_extension', 'a_attr_href', 'a_attr_class',\n",
      "       'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 640 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - Parsed data extraction completed. Elements collected: 6\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 481 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 46th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC6750256)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC6750256\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC6750256&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 394 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC6750256.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b0864b50>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC6750256 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC6750256, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 445 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC6750256)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 547 - INFO - Extracted 17 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 557 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#S15'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for S15: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#S15'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for S15: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#S17'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for S17: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#S19'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for S19: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#S19'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for S19: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#S20'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for S20: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#S22'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for S22: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#S31'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for S31: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 45 <a> elements with href='#SD1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 7597) for SD1: We compiled ~40 million possible variants based on the union of SNPs called from RNA-seq data ( Supplementary Fig. 1a [href=this file] ) and SNPs already identified from microarray probe sets and the 1000 Genomes (Methods). To collect transcriptome wide genotyping data from our brain samples, we performed imputation from genotyping array and RNA-seq based genotyping (Methods; Supplementary Fig. 1b [href=this file] and 1c [href=this file] ), and ancestry was identified for all samples (Methods; Supplementary Fig. 1d [href=this file] ).\n",
      "The values of excluded covariates showed balanced distributions between control and ASD ( Supplementary Fig. 1e [href=this file] ; Methods).\n",
      "As expected, SNPs within the same haplotype blocks show similar allelic expression patterns ( Supplementary Fig. 1f [href=this file] and 1g [href=this file] ).\n",
      "Although this database contains data from mostly non-neural tissues, genes manifesting ASE in our study showed significant overlap with those in dbMAE ( Supplementary Fig. 2a [href=this file] ). For example, the overlap between genes showing ASE in the two cortical regions, BA9 and BA41 is extremely high (odds ratio (OR)=221.00 and p-value<2.2e-16; Supplementary Fig. 2b [href=this file] ), whereas the degree of overlap between two more developmentally and evolutionarily distinct brain regions, cortex and cerebellum, is substantial, but lower than in the two more similar cortical regions (between BA9 and vermis: OR=49.00 and p-value<2.2e-16; between BA41 and vermis: OR=72.54 and p-value<2.2e-16).\n",
      "For further external validation, we applied the specific methods and pipeline used in the primary analysis of GTEx ( Supplementary Fig. 2c [href=this file] ).\n",
      "The distribution of expression of imbalanced alleles was significantly shifted lower than balanced alleles (Kolmogorov-Smirnov (KS) test, p-value<2.0e-16, the means of log (RPSM): 0.37 vs 0.71; Supplementary Fig. 2d [href=this file] ). Genes showing evidence for ASE were also preferentially enriched for neuronal and astrocyte markers ( Fig. 2b [href=#F2] and Supplementary Fig. 2e [href=this file] ). Remarkably, across all three brain regions, ASE genes are enriched for genes previously shown to be down-regulated in postmortem brain from patients with ASD ( Supplementary Fig. 2f [href=this file] ).\n",
      "Similar to controls, idiopathic ASD cases showed tissue specific ASE across the three brain regions ( Fig. 3a [href=#F3] , Supplementary Fig. 3a [href=this file] , and Supplementary Fig. 3b [href=this file] ).\n",
      "Genes exhibiting control-specific ASE are significantly enriched for pathways related to growth cone and synapse ( Supplementary Fig. 3c [href=this file] and Supplementary Table 4 [href=#SD6] ). Though there are fewer idiopathic ASD-specific ASE genes than control-specific ASE genes, those showing idiopathic ASD-specific ASD are significantly enriched for pathways involved in transport ( Supplementary Fig. 3d [href=this file] and Supplementary Table 4 [href=#SD6] ). Genes exhibiting ASE in both idiopathic ASD and controls show significant enrichment in genes harboring ASD risk variants ( Supplementary Fig. 3e [href=this file] ; OR=1.68, p-value=5e-04 for ASD1; OR=3.83, p-value=0.001 for ASD2). However, control-specific and idiopathic ASD-specific ASE genes do not show any enrichment with either the rare ASD risk variants or common variants from GWAS data ( Supplementary Fig. 3e [href=this file] ).\n",
      "Controls had a higher density of balanced alleles than ASD cases while idiopathic ASD cases showed more imbalanced alleles ( Supplementary Fig. 4a [href=this file] and Fig. 4a [href=#F4] ). However, controls showed significantly more major allele MAE for low frequency alleles (minor allele frequency (MAF)<0.05) compared to idiopathic ASD ( Supplementary Fig. 4b [href=this file] ). These patterns were recapitulated in the pseudoautosomal region (PAR) of the chromosome X ( Supplementary Fig. 4c [href=this file] ).\n",
      "We next investigated whether the differential ASE pattern was related to the MAF genome-wide ( Fig. 4b [href=#F4] and Supplementary Fig. 4b [href=this file] ).\n",
      "Although MAE SNPs were observed across the genome ( Supplementary Fig. 5 [href=this file] ), there was an approximately 2-fold enrichment on chromosome 15 harboring a number of known imprinted loci ( Supplementary Table 5 [href=#SD2] ). This genome-wide enrichment pattern persisted when SNPs were analyzed at individual genes ( Supplementary Fig. 6a [href=this file] and 6b [href=this file] ).\n",
      "For example, genes showing minor allele MAE showed no overlap with astrocyte markers , PSD , or ASD-specific down-regulated genes ( Supplementary Fig. 6c [href=this file] ). Like genes that manifest ASE ( Fig. 2c [href=#F2] ), major allele MAE genes show significant enrichment with genes associated with the rare ASD risk variants ( Supplementary Fig. 6d [href=this file] ). However, MAE genes in control and ASD showed some enrichment with common variants from GWAS data of BPD and SCZ ( Supplementary Fig. 6d [href=this file] ).\n",
      "This analysis identities only two genomic regions, 14q32 (chr14:101302,638–101,544,745) and 15q11 (chr15:25,223,730–25,582,395), which are known imprinted loci that have many small splice junctions ( Fig. 6 [href=#F6] ) and show continuous high expression ( Supplementary Fig. 7 [href=this file] ) as single transcript units .\n",
      "The snoRNA targeting sites were located significantly proximal to alternatively spliced junctions of their target genes in ASD ( Supplementary Fig. 8a [href=this file] ), and their splicing changes show significant correlations with the expression changes of their specific snoRNAs ( Supplementary Fig. 8b [href=this file] and 8c [href=this file] ). When one snoRNA has two different targets, their splicing changes also show strong correlation ( Supplementary Fig. 8d [href=this file] ).\n",
      "Compared with the previously published RNA-seq based genotyping methods by Quinn et al. , these genotyping parameters show much higher accuracy with sensitivity exceeding 95% ( Supplementary Fig. 1b [href=this file] ) and specificity in the range of 90–95% ( Supplementary Fig. 1c [href=this file] ; see Quinn at el. for accuracy tests).\n",
      "We categorized our samples into European, Mexican, Asian, African, and ambiguous groups ( Supplementary Fig. 1d [href=this file] ).\n",
      "In BA9 samples, the distributions of the covariate values were balanced between control and ASD ( Supplementary Fig. 1e [href=this file] ).\n",
      "We calculated deviations of log (fold change) values (the beta values of allele covariate from the group based ASE identifications) per haplotype block in cortex and compared their distributions with normal distributions ( Supplementary Fig. 1f [href=this file] and 1g [href=this file] ). For further validation, we compared ASE genes identified in our study with those present in dbMAE ( https://mae.hms.harvard.edu/ [href=https://mae.hms.harvard.edu/] ) using Fisher’s exact test ( Supplementary Fig. 2a [href=this file] ). Like GTEx data, we calculated ASE rates per tissue ( Supplementary Fig. 2c [href=this file] ).\n",
      "We compared their log (RPSM) values using KS tests ( Supplementary Fig. 2d [href=this file] ).\n",
      "We visualized their distributions using ggplot2 R package ( http://ggplot2.org/ [href=http://ggplot2.org/] ) and compared them between control and idiopathic ASD at autosome ( Fig. 4a [href=#F4] and Supplementary Fig. 4b [href=this file] ) and chromosome X ( Supplementary Fig. 4c [href=#SD7] ).\n",
      "For major and minor allele MAE SNPs, we visualized them on the different lines ( Supplementary Fig. 5 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 2 <a> elements with href='#SD2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 532) for SD2: Although MAE SNPs were observed across the genome ( Supplementary Fig. 5 [href=#SD1] ), there was an approximately 2-fold enrichment on chromosome 15 harboring a number of known imprinted loci ( Supplementary Table 5 [href=this file] ).\n",
      "Compared to chromosome 14, chromosome 15 exhibited a substantially higher minor allele MAE fraction in both ASD and dup15q than controls (3.9- and 1.6-fold, respectively; chi-square test p-values<2.2e-16 and 0.0184, respectively; Fig. 5c [href=#F5] and Supplementary Table 5 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 7 <a> elements with href='#SD3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 1249) for SD3: For investigation of group-level ASE patterns across cases and brain regions, 1,163,249 heterozygous SNPs in 21,929 genes were used following quality control (Methods), and their significance was quantified using a linear mixed model to account for sample variation and covariates ( Supplementary Table 1 [href=this file] ; Methods).\n",
      "Though most sample information was already published , we provided it again including further information ( Supplementary Table 1 [href=this file] ).\n",
      "To increase their SNP calling accuracy, we optimized bam files following the recommended GATK pipeline ( Supplementary Fig. 1a [href=this file] ).\n",
      "To ensure if groups were balanced with respect to these covariates, we removed samples based on age (≦7), brain banks (The UK Brain Bank for Autism and Related Developmental Research and the MRC London Neurodegenerative Diseases Brain Banks), and SNP annotation (mysterious) ( Supplementary Table 1 [href=this file] ) . GC content was tested using AT and GC dropout values from Picard ( https://broadinstitute.github.io/picard/ [href=https://broadinstitute.github.io/picard/] ; Supplementary Table 1 [href=this file] ).\n",
      "The detailed description of brain samples is provided in Supplementary Table 1 [href=this file] .\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 3 <a> elements with href='#SD4'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 341) for SD4: Furthermore, although assessed separately in each brain region, there was strong concordance in common ASE genes between regions in our data ( Fig. 2a [href=#F2] and Supplementary Table 2 [href=this file] ).\n",
      "For each tissue, group-based ASE identification results are available in Supplementary Tables 2 [href=this file] and 3 [href=#SD5] .\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 3 <a> elements with href='#SD5'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 361) for SD5: The majority of genes exhibiting ASE (70.41%) are shared across the case and control groups providing further confidence in the reproducibility of these results ( Fig. 3c [href=#F3] and Supplementary Table 3 [href=this file] ).\n",
      "For each tissue, group-based ASE identification results are available in Supplementary Tables 2 [href=#SD4] and 3 [href=this file] .\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 4 <a> elements with href='#SD6'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 769) for SD6: The p-values and other results of the GO analysis are at Supplementary Table 4 [href=this file] .\n",
      "Genes exhibiting ASE in both idiopathic ASD and control are enriched in phosphoprotein, RNA splicing, and neuron projection pathways ( Fig. 3e [href=#F3] and Supplementary Table 4 [href=this file] ). Genes exhibiting control-specific ASE are significantly enriched for pathways related to growth cone and synapse ( Supplementary Fig. 3c [href=#SD1] and Supplementary Table 4 [href=this file] ). Though there are fewer idiopathic ASD-specific ASE genes than control-specific ASE genes, those showing idiopathic ASD-specific ASD are significantly enriched for pathways involved in transport ( Supplementary Fig. 3d [href=#SD1] and Supplementary Table 4 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 9 <a> elements with href='#SD7'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 2164) for SD7: For known ASD risk gene list (SFARI gene), we selected genes from category 1, 2, and 3 corresponding to gene scoring criteria ( https://gene.sfari.org/autdb/GS_Home.do [href=https://gene.sfari.org/autdb/GS_Home.do] ); Supplementary Table 6 [href=this file] ) The gene list was previously used at the previous publication .\n",
      "For this gene set enrichment studies, we also used known imprinted genes ( www.geneimprint.com [href=http://www.geneimprint.com/] ), PSD , FMRP target , HuR target , and RBFOX1 target genes , cell marker genes (neuron, astrocyte, oligodendrocyte, microglia, and endothelial), expression up- and down-regulated genes in ASD cortex ( Supplementary Table 6 [href=this file] ). HuR target gene list was collected from photoactivatable ribonucleoside crosslinking and immunoprecipitation (PAR-CLIP) data instead of RNA binding proteins immunoprecipitation complementary DNA array (RIP-chip) data since PAR-CLIP data identified RNA binding proteins binding sites more precisely than RIP-chip ( Supplementary Table 6 [href=this file] ).\n",
      "For the variant data, we used the risk genes containing rare likely gene disrupting mutations in SCZ, ID, and ASD from the dataset of Iossifov et al. ( Supplementary Table 6 [href=this file] ). For the risk gene list, we analyzed gene set enrichment with different FDR cutoffs (FDR≦0.01, 0.01<FDR≦0.05, and 0.05<FDR≦0.1) and found the highest enrichments using the most stringent FDR cutoff. of FDR≦0.01, which we report in our study ( Supplementary Table 6 [href=this file] ).\n",
      "For both studies, brain expressed genes were used as background genes that have more than 10 allele counts ( Supplementary Table 6 [href=this file] ).\n",
      "We visualized their distributions using ggplot2 R package ( http://ggplot2.org/ [href=http://ggplot2.org/] ) and compared them between control and idiopathic ASD at autosome ( Fig. 4a [href=#F4] and Supplementary Fig. 4b [href=#SD1] ) and chromosome X ( Supplementary Fig. 4c [href=this file] ).\n",
      "We also include gene lists that we used for gene set enrichment and GO analysis that includes brain expressed gene that we used for our study ( Supplementary Table 6 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 59944\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 199861\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 60666 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-befbf19fa92d9f38d8d322410aca36f1af412f06179a62240c554aa496825a07\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 60666 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\n",
      "  \"datasets\": [\n",
      "    {\n",
      "      \"dataset_identifier\": \"PMC6750256\",\n",
      "      \"repository_reference\": \"https://pmc.ncbi.nlm.nih.gov/articles/PMC6750256/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"NIHMS1534152\",\n",
      "      \"repository_reference\": \"http://dx.doi.org/10.7303/syn4587609\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    }\n",
      "  ]\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'PMC6750256', 'repository_reference': 'https://pmc.ncbi.nlm.nih.gov/articles/PMC6750256/'}, {'dataset_identifier': 'NIHMS1534152', 'repository_reference': 'http://dx.doi.org/10.7303/syn4587609'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 3 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: PMC6750256\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: NIHMS1534152\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'PMC6750256', 'repository_reference': 'https://pmc.ncbi.nlm.nih.gov/articles/PMC6750256/', 'dataset_id': 'PMC6750256'}, {'dataset_identifier': 'NIHMS1534152', 'repository_reference': 'http://dx.doi.org/10.7303/syn4587609', 'dataset_id': 'NIHMS1534152'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'PMC6750256', 'repository_reference': 'https://pmc.ncbi.nlm.nih.gov/articles/PMC6750256/', 'dataset_id': 'PMC6750256'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: PMC6750256\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID PMC6750256 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://pmc.ncbi.nlm.nih.gov/articles/PMC6750256/, identifier: PMC6750256\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://pmc.ncbi.nlm.nih.gov/articles/PMC6750256/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier PMC6750256 for https://pmc.ncbi.nlm.nih.gov/articles/PMC6750256/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier PMC6750256 for https://pmc.ncbi.nlm.nih.gov/articles/PMC6750256/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier PMC6750256 for https://pmc.ncbi.nlm.nih.gov/articles/PMC6750256/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier PMC6750256 for https://pmc.ncbi.nlm.nih.gov/articles/PMC6750256/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier PMC6750256 for https://pmc.ncbi.nlm.nih.gov/articles/PMC6750256/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC6750256/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'PMC6750256', 'data_repository': 'pmc.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'NIHMS1534152', 'repository_reference': 'http://dx.doi.org/10.7303/syn4587609', 'dataset_id': 'NIHMS1534152'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(10\\.\\d{4,9}/).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: 10.7303/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn4587609, None, http://dx.doi.org/10.7303/syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: http://dx.doi.org/10.7303/syn4587609, identifier: syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: http://dx.doi.org/10.7303/syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn4587609 for http://dx.doi.org/10.7303/syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn4587609 for http://dx.doi.org/10.7303/syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn4587609 for http://dx.doi.org/10.7303/syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn4587609 for http://dx.doi.org/10.7303/syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier syn4587609 for http://dx.doi.org/10.7303/syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1153 - INFO - Found id_pattern match for http://dx.doi.org/10.7303/syn4587609 in syn\\d{7,12}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn4587609', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 677 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 605 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'PMC6750256', 'data_repository': 'pmc.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn4587609', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 2 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: PMC6750256\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: pmc.ncbi.nlm.nih.gov, identifier: PMC6750256\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier PMC6750256 for pmc.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier PMC6750256 for pmc.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier PMC6750256 for pmc.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier PMC6750256 for pmc.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier PMC6750256 for pmc.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: pmc.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 976 - WARNING - URL pmc.ncbi.nlm.nih.gov may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for PMC6750256 in pmc.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 1 with repo: pmc.ncbi.nlm.nih.gov and accession_id: PMC6750256\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1254 - WARNING - Repository pmc.ncbi.nlm.nih.gov unknown in Ontology. Skipping dataset page 1.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn4587609 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn4587609 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn4587609 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn4587609 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn4587609 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn4587609\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 2\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 19, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 640 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - Parsed data extraction completed. Elements collected: 10\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 481 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 47th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11370573)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11370573\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC11370573&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 461 - WARNING - No links found in the section.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 406 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC11370573.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11370573 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 430 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11370573, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 443 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 264 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 454 - INFO - Found 2 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b6512540>, <Element supplementary-material at 0x1b61b4540>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 462 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b6512540>, <Element supplementary-material at 0x1b61b4540>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 548 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref SD1: Clones with their genomic features of interest can be found in supplementary table 5 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref SD2: All 19 patients had cfDNA collected at or close to the time of first debulking surgery or laparoscopic biopsy (baseline), and a subset (n=10) had radiographically confirmed disease recurrence along with cfDNA collections post-recurrence and during therapy ( Supplementary Figure 1 [this file] ). See Supplementary Figure 1 [this file] for clinical details, treatment history and sample collections for all 19 patients. \n",
      "From the 19 patients included in this study we generated scWGS data from 19,454 cells (range 200-2015 cells per patient, Supplementary Figure 2 [this file] ) with mean coverage of 0.089X (range 0.002-0.392X per cell, Supplementary Figure 2). We inferred the clonal composition at the time of diagnosis based on copy number data ( Methods [rid=S11] ), with the aim of following these clones over time as patients received chemotherapy, maintenance therapies and experienced disease recurrences using cfDNA ( Supplementary Figure 3 [this file] ). \n",
      "We then merged cells from each clone and re-computed copy number at 10kb resolution using a new Hidden Markov Model (HMM) based copy number caller, ( Methods [rid=S11] ). improves the resolution of pseudobulk clone copy number profiles and enables more precise matching between copy number and SVs ( Supplementary Figure 4 [this file] ). \n",
      "For each patient, we constructed a panel of mutations comprising a mix of clonal and subclonal somatic SNVs/SVs (≥100 SVs per patient, Supplementary Figure 2 [this file] ) and a small number of germline single nucleotide polymorphisms (SNPs) for QC purposes. \n",
      "Clone-specific SVs inferred by scWGS analysis were present in all patients with at least 200 cells (18 patients, average n=144, range 29-361 Supplementary Figure 2 [this file] ). \n",
      "See Supplementary Figure 5 [this file] for the scWGS data for these 10 patients. For all patients, ctDNA VAF of truncal SVs decreased during initial chemotherapy, as patients responded to therapy with decreased burden and decreased serum CA-125 levels ( Fig. 4 [rid=F4] & Supplementary Figure 6 [this file] ). All patients were positive for ctDNA at the time point closest to first recurrence (defined as average VAF across the panel exceeding LOD ( Fig. 4 [rid=F4] & Supplementary Figure 6 [this file] )). In 6 patients with sufficient plasma samples, ctDNA was detected prior to clinically confirmed disease recurrence but subsequent to completion of initial chemotherapy: 002 (76 days), 004 (26 days), 009 (184 days), 045 (109 days), 075 (233 days), 081 (314 days) ( Fig. 4 [rid=F4] & Supplementary Figure 6 [this file] ). \n",
      "Copy number profiles from these data were consistent with predictions derived from scWGS derived copy number profiles and ctDNA clone frequencies ( Supplementary Figure 7a [this file] – c [this file] ). In patient 045, the locus (chr19q) was enriched at baseline, and (chr1q) at recurrence ( Supplementary Figure 7d [this file] ), while in patient 107, and a region on chr20q were enriched at recurrence as expected ( Supplementary Figure 7e [this file] ). Clone frequencies across time for patient 045 showed highly similar patterns for SVs ( Supplementary Figure 7f [this file] ) and SNVs ( Supplementary Figure 7g [this file] ) and were consistent using these two distinct sets of genomic features for all patients ( Supplementary Figure 7h [this file] , R=0.93, p<10 , Pearson correlation). \n",
      "In 6/10 patients (002, 006, 014, 045, 081, 107; see Fig. 4 [rid=F4] & Supplementary Figure 6 [this file] ), multiple clones were detected in post-recurrence plasma samples, highlighting that chemo-resistance may be polyclonal in many patients. In 4 of these cases (009, 045, 081 and 107), clones that were not detected in cfDNA or were detected at minor frequencies at baseline became dominant at first recurrence, suggesting that although multiple clones may become chemo-resistant, some have relative fitness advantages in the context of treatment ( Fig. 4 [rid=F4] & Supplementary Figure 6 [this file] ). Notably, while the presence of multiple resistant clones was a common observation, the overall diversity, as quantified by Shannon entropy, decreased in the final timepoint relative to baseline in 8/9 cases (p=0.027, t-test, Supplementary Figure 6h [this file] ). The number of clones detected also decreased at the final time point relative to baseline (p=0.086, t-test, Supplementary Figure 6i [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 243 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 37253\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 138696\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 37975 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-265d64754c2aa5b9d2170b0e9706dae2bc58db76aa14e16ff4ae69d2e0e7afce\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 37975 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\"datasets\":[{\"dataset_identifier\":\"syn25569736\",\"repository_reference\":\"https://www.synapse.org/#!Synapse:syn25569736\"},{\"dataset_identifier\":\"dbGAP\",\"repository_reference\":\"n/a\"}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'syn25569736', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn25569736'}, {'dataset_identifier': 'dbGAP', 'repository_reference': 'n/a'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 2 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn25569736\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: dbGAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'syn25569736', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn25569736', 'dataset_id': 'syn25569736'}, {'dataset_identifier': 'dbGAP', 'repository_reference': 'n/a', 'dataset_id': 'dbGAP'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn25569736', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn25569736', 'dataset_id': 'syn25569736'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn25569736\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn25569736, synapse.org, https://www.synapse.org/#!Synapse:syn25569736\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn25569736\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn25569736', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'dbGAP', 'repository_reference': 'n/a', 'dataset_id': 'dbGAP'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: dbGAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID dbGAP is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: n/a, identifier: dbGAP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier dbGAP for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier dbGAP for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier dbGAP for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier dbGAP for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier dbGAP for n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 677 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 605 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'dbGAP', 'repository_reference': 'n/a', 'dataset_id': 'dbGAP'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 256 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn25569736', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 1 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn25569736\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn25569736\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn25569736 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn25569736 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn25569736 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn25569736 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn25569736 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn25569736\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn25569736\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 1\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 265 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 3, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 477 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 48th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC4344205)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC4344205\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC4344205&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 394 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC4344205.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b741a210>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC4344205 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC4344205, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 445 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC4344205)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 547 - INFO - Extracted 13 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 557 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#sec006'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 25) for sec006: Results [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#pone.0117658.s001'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 80) for pone.0117658.s001: S1 [href=this file] – S4 [href=#pone.0117658.s004] Tables show the raw results.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#pone.0117658.s002'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 148) for pone.0117658.s002: Yet, as we can see from S2 Table [href=this file] , we can get fairly good prediction (concordance index equal to 0.654) using the top 50 features.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#pone.0117658.s003'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for pone.0117658.s003: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#pone.0117658.s004'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 80) for pone.0117658.s004: S1 [href=#pone.0117658.s001] – S4 [href=this file] Tables show the raw results.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#pone.0117658.s005'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 175) for pone.0117658.s005: S5 Table [href=this file] shows the top 150 features extracted by ReliefF from the entire data set for Method 1 and Method 2, and for 5 year, 10 year, and 15 year prediction.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 36756\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 122128\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 37478 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-ba55280708feb6684050d98a4dd27ac50180cbcf8acb19bf4e0f81a1cd943079\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 37478 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\n",
      "  \"datasets\": [\n",
      "    {\n",
      "      \"dataset_identifier\": \"METABRIC\",\n",
      "      \"repository_reference\": \"https://www.synapse.org/#!Synapse:syn1688369\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    }\n",
      "  ]\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'METABRIC', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn1688369'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 2 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: METABRIC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'METABRIC', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn1688369', 'dataset_id': 'METABRIC'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'METABRIC', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn1688369', 'dataset_id': 'METABRIC'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn1688369\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn1688369, None, https://www.synapse.org/#!Synapse:syn1688369\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn1688369\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn1688369, identifier: syn1688369\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn1688369\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn1688369 for https://www.synapse.org/#!Synapse:syn1688369\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn1688369 for https://www.synapse.org/#!Synapse:syn1688369\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn1688369 for https://www.synapse.org/#!Synapse:syn1688369\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn1688369 for https://www.synapse.org/#!Synapse:syn1688369\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn1688369 in Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn1688369', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 677 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 605 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn1688369', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 1 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn1688369\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn1688369\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn1688369 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn1688369 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn1688369 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn1688369 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn1688369 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn1688369\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn1688369\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 1\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 14, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 640 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - Parsed data extraction completed. Elements collected: 7\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 481 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 49th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC9630780)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC9630780\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC9630780&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: supplementary_data_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 394 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC9630780.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b0862090>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC9630780 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC9630780, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 445 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC9630780)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 547 - INFO - Extracted 20 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 557 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#fn1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for fn1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#sec1.1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for sec1.1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#sec1.2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for sec1.2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#sec1.2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for sec1.2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#sec1.3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for sec1.3: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#sec1.4'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for sec1.4: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#sec1.4'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for sec1.4: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for undtbl1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for undtbl1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for undtbl1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for undtbl1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#sec2.1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for sec2.1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#sec2.2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for sec2.2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#sec2.3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 179) for sec2.3: Refer to section 2c of running CAROM-ML in Python [href=this file] for details on how to incorporate either oversampling, undersampling, or class weights into the model training.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#sec2.3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 179) for sec2.3: Refer to section 2c of running CAROM-ML in Python [href=this file] for details on how to incorporate either oversampling, undersampling, or class weights into the model training.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#sec2.3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 179) for sec2.3: Refer to section 2c of running CAROM-ML in Python [href=this file] for details on how to incorporate either oversampling, undersampling, or class weights into the model training.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#sec5.2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for sec5.2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 2 <a> elements with href='#sec6.1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 147) for sec6.1: Any additional information required to reanalyze the data reported in this paper is available from the lead contact [href=this file] upon request.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#da00101'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 44) for da00101: Data and code availability [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 43314\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 140781\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 44036 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-482dd7ccd0ad80cb0f7d47113d8a41007c0c1fa5a8b294eda75f3cef8fcfe6ea\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 44036 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\n",
      "  \"datasets\": [\n",
      "    {\n",
      "      \"dataset_identifier\": \"syn34542449\",\n",
      "      \"repository_reference\": \"https://www.synapse.org/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"syn34542450\",\n",
      "      \"repository_reference\": \"https://www.synapse.org/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"syn34539904\",\n",
      "      \"repository_reference\": \"https://www.synapse.org/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"syn34542574\",\n",
      "      \"repository_reference\": \"https://www.synapse.org/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"syn25792906\",\n",
      "      \"repository_reference\": \"https://www.synapse.org/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"syn25886762\",\n",
      "      \"repository_reference\": \"https://www.synapse.org/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"syn25876213\",\n",
      "      \"repository_reference\": \"https://www.synapse.org/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"https://www.synapse.org/CAROM\"\n",
      "    }\n",
      "  ]\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'syn34542449', 'repository_reference': 'https://www.synapse.org/'}, {'dataset_identifier': 'syn34542450', 'repository_reference': 'https://www.synapse.org/'}, {'dataset_identifier': 'syn34539904', 'repository_reference': 'https://www.synapse.org/'}, {'dataset_identifier': 'syn34542574', 'repository_reference': 'https://www.synapse.org/'}, {'dataset_identifier': 'syn25792906', 'repository_reference': 'https://www.synapse.org/'}, {'dataset_identifier': 'syn25886762', 'repository_reference': 'https://www.synapse.org/'}, {'dataset_identifier': 'syn25876213', 'repository_reference': 'https://www.synapse.org/'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.synapse.org/CAROM'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 8 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn34542449\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn34542450\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn34539904\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn34542574\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn25792906\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn25886762\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn25876213\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'syn34542449', 'repository_reference': 'https://www.synapse.org/', 'dataset_id': 'syn34542449'}, {'dataset_identifier': 'syn34542450', 'repository_reference': 'https://www.synapse.org/', 'dataset_id': 'syn34542450'}, {'dataset_identifier': 'syn34539904', 'repository_reference': 'https://www.synapse.org/', 'dataset_id': 'syn34539904'}, {'dataset_identifier': 'syn34542574', 'repository_reference': 'https://www.synapse.org/', 'dataset_id': 'syn34542574'}, {'dataset_identifier': 'syn25792906', 'repository_reference': 'https://www.synapse.org/', 'dataset_id': 'syn25792906'}, {'dataset_identifier': 'syn25886762', 'repository_reference': 'https://www.synapse.org/', 'dataset_id': 'syn25886762'}, {'dataset_identifier': 'syn25876213', 'repository_reference': 'https://www.synapse.org/', 'dataset_id': 'syn25876213'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.synapse.org/CAROM', 'dataset_id': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn34542449', 'repository_reference': 'https://www.synapse.org/', 'dataset_id': 'syn34542449'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn34542449, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn34542449\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn34542449', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn34542450', 'repository_reference': 'https://www.synapse.org/', 'dataset_id': 'syn34542450'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn34542450, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn34542450\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn34542450', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn34539904', 'repository_reference': 'https://www.synapse.org/', 'dataset_id': 'syn34539904'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn34539904, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn34539904\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn34539904', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn34542574', 'repository_reference': 'https://www.synapse.org/', 'dataset_id': 'syn34542574'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn34542574, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn34542574\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn34542574', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn25792906', 'repository_reference': 'https://www.synapse.org/', 'dataset_id': 'syn25792906'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn25792906, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn25792906\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn25792906', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn25886762', 'repository_reference': 'https://www.synapse.org/', 'dataset_id': 'syn25886762'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn25886762, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn25886762\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn25886762', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn25876213', 'repository_reference': 'https://www.synapse.org/', 'dataset_id': 'syn25876213'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn25876213, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn25876213\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn25876213', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.synapse.org/CAROM', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.synapse.org/CAROM, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/CAROM\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Found partial match for https://www.synapse.org/CAROM in Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 673 - INFO - Dataset ID is 'n/a' and repository name from prompt\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 605 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.synapse.org/CAROM', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn34542449', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn34542450', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn34539904', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn34542574', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn25792906', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn25886762', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn25876213', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 7 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn34542449\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn34542449\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn34542449 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn34542449 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn34542449 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn34542449 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn34542449 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn34542449\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn34542449\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn34542450\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn34542450\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn34542450 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn34542450 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn34542450 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn34542450 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn34542450 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn34542450\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn34542450\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn34539904\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn34539904\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn34539904 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn34539904 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn34539904 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn34539904 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn34539904 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 3 with repo: synapse.org and accession_id: syn34539904\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn34539904\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 3: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn34542574\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn34542574\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn34542574 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn34542574 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn34542574 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn34542574 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn34542574 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 4 with repo: synapse.org and accession_id: syn34542574\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn34542574\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 4: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn25792906\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn25792906\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn25792906 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn25792906 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn25792906 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn25792906 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn25792906 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 5 with repo: synapse.org and accession_id: syn25792906\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn25792906\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 5: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn25886762\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn25886762\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn25886762 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn25886762 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn25886762 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn25886762 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn25886762 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 6 with repo: synapse.org and accession_id: syn25886762\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn25886762\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 6: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn25876213\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn25876213\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn25876213 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn25876213 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn25876213 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn25876213 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn25876213 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 7 with repo: synapse.org and accession_id: syn25876213\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn25876213\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 7: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 7\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 27, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 640 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - Parsed data extraction completed. Elements collected: 8\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 481 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 50th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10813059)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10813059\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC10813059&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 461 - WARNING - No links found in the section.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <supplementary-material xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.or...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 406 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC10813059.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC10813059 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 430 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC10813059, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 443 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 264 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 454 - INFO - Found 1 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b7e32340>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 462 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b7e32340>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 548 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref biomolecules-14-00066-s001: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 243 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 51800\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 206446\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 52522 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-f9b70dc9789386f8562fca02d2a6f1bf6697ad93bce5316eba15db2f6f6f0496\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 52522 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\n",
      "  \"datasets\": [\n",
      "    {\n",
      "      \"dataset_identifier\": \"syn52138250\",\n",
      "      \"repository_reference\": \"https://adknowledgeportal.synapse.org\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"syn31952297\",\n",
      "      \"repository_reference\": \"https://adknowledgeportal.synapse.org\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"syn31947506\",\n",
      "      \"repository_reference\": \"https://adknowledgeportal.synapse.org\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    }\n",
      "  ]\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'syn52138250', 'repository_reference': 'https://adknowledgeportal.synapse.org'}, {'dataset_identifier': 'syn31952297', 'repository_reference': 'https://adknowledgeportal.synapse.org'}, {'dataset_identifier': 'syn31947506', 'repository_reference': 'https://adknowledgeportal.synapse.org'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 4 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn52138250\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn31952297\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn31947506\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'syn52138250', 'repository_reference': 'https://adknowledgeportal.synapse.org', 'dataset_id': 'syn52138250'}, {'dataset_identifier': 'syn31952297', 'repository_reference': 'https://adknowledgeportal.synapse.org', 'dataset_id': 'syn31952297'}, {'dataset_identifier': 'syn31947506', 'repository_reference': 'https://adknowledgeportal.synapse.org', 'dataset_id': 'syn31947506'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn52138250', 'repository_reference': 'https://adknowledgeportal.synapse.org', 'dataset_id': 'syn52138250'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn52138250, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn52138250\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn52138250', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn31952297', 'repository_reference': 'https://adknowledgeportal.synapse.org', 'dataset_id': 'syn31952297'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn31952297, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn31952297\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn31952297', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn31947506', 'repository_reference': 'https://adknowledgeportal.synapse.org', 'dataset_id': 'syn31947506'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn31947506, synapse.org, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn31947506\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn31947506', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 677 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 605 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 256 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn52138250', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn31952297', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn31947506', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 3 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn52138250\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn52138250\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn52138250 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn52138250 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn52138250 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn52138250 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn52138250 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn52138250\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn52138250\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn31952297\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn31952297\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn31952297 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn31952297 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn31952297 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn31952297 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn31952297 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn31952297\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn31952297\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn31947506\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn31947506\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn31947506 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn31947506 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn31947506 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn31947506 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn31947506 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 3 with repo: synapse.org and accession_id: syn31947506\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn31947506\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 3: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 3\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 265 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 4, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 477 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 574 - INFO - \n",
      "Progress: 51/100 (51.00%) | Elapsed: 00:14:04 | ETA: 00:13:30\n",
      "\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 51th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC6089972)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC6089972\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC6089972&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 394 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC6089972.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b6abde10>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC6089972 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC6089972, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 445 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC6089972)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 547 - INFO - Extracted 28 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 557 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#Sec13'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for Sec13: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#Sec13'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for Sec13: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#Sec13'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for Sec13: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#Sec15'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for Sec15: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#Sec15'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for Sec15: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#Sec19'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for Sec19: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#Sec20'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for Sec20: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#Sec21'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for Sec21: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#Sec21'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for Sec21: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#Sec26'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for Sec26: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#Sec25'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for Sec25: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#Sec25'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for Sec25: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#Sec25'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for Sec25: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#Sec25'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for Sec25: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 7 <a> elements with href='#MOESM1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 2417) for MOESM1: Analysis of residuals (method estimate—ABSOLUTE estimate) suggested close concordance with ABSOLUTE estimates for MethylCIBERSORT, with larger deviations only seen when samples were of very high purity (>80%), while other methods tended to overestimate tumour cell content in samples of low purity (Supplementary Figure 1a [href=this file] ), resulting in statistically significant differences in distributions (FDR < 2.2e−16).\n",
      "We also compared the mRNA expression of a panel of cellular lineage markers with MethylCIBERSORT estimates and found significant associations for multiple cell types (Supplementary Figure 1b [href=this file] ) even though they are derived from different samplings of the same tumour.\n",
      "For nine of these we were able to compare our predictions of tumour purity with ABSOLUTE estimates and observed strong correlations and significantly lower error margins compared to LUMP and ESTIMATE (Supplementary Figure 2a, b [href=this file] ). Further, we observed a robust preservation of positive correlations between MethylCIBERSORT and marker expression pan-cancer (Supplementary Figure 2c [href=this file] ), again with the caveat that the samples were taken from different aliquots of the tumour. We therefore compared estimates pertaining to fresh frozen and matched FFPE samples ( = 21 from three tumours) and recorded very high correlations, indicating our method is applicable also to the archival material (Supplementary Figure 2d [href=this file] ).\n",
      "Among the genomic alterations we identify (for a full list of predicted driver events see Supplementary Data 8 [href=this file0] ), it is likely that some establish, while others are selected for, in different immune microenvironments. The positive correlation between EGFR levels (which are themselves correlated with EGFR phosphorylation (activation), Supplementary Figure 3 [href=this file] ) and the glycolytic signature is maintained across TCGA when matched RPPA profiles and RNA-seq data are compared (Fig. 5f [href=#Fig5] ).\n",
      "Signature features were derived from 450k profiles using the aforementioned heuristic (delta-Beta and FDR cutoffs) with a maximum of 100 features per cell type for a wide range of tumour types, using cell lines allocated to the corresponding tissue in GSE68379 [href=https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE68379] (Table S10 [href=this file] ) and the aforementioned infiltrating cell types.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#MOESM2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for MOESM2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 3 <a> elements with href='#MOESM3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 825) for MOESM3: Using limma-trend analysis, we identified 458 differentially expressed genes (DEGs) between the hot and cold clusters at a fold-change of greater than 2 (FDR < 0.01, Supplementary Data 1 [href=this file] , genes highlighted in bold). In extended analyses of all genes at FDR < 0.01 (Supplementary Data 1 [href=this file] ), multiple other genes, including the Class 1 MHC gene (FC = 1.39), PD-1 ligand (FC = 1.62) and , which encodes SMA (FC = 0.68), are also differentially expressed between the two clusters, the latter validating fibroblast estimates from MethylCIBERSORT (Fig. 2a [href=#Fig2] ).\n",
      "Fas-ligand ( ), an upstream activator of pro-apoptotic signalling through Caspase 8 is also upregulated in the immune hot tumours, further highlighting the importance of this pathway (Supplementary Data 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#MOESM4'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 332) for MOESM4: Ingenuity Pathway Analysis further confirmed observations made using MethylCIBERSORT estimates, identifying differential regulation of multiple canonical pathways associated with immune function and inflammatory conditions (Supplementary Data 2 [href=this file] ), consistent with differential lymphocyte infiltration and activity.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#MOESM5'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 191) for MOESM5: Diseases and functions ontology (Supplementary Data 3 [href=this file] ) indicated that the top few pathways activated in hot tumours were associated with leukocyte and lymphocyte migration.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#MOESM6'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 209) for MOESM6: Upstream regulatory analysis implicated increased activation of STAT1 and IRF1, and inhibition of Interferon-stimulated transcription mediated by IRF4, in hot tumours (Supplementary Data 4 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#MOESM7'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 152) for MOESM7: Finally, analysis of RPPA data identified 10 differentially abundant (FDR < 0.1) proteins or phospho-proteins (Supplementary Data 5 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#MOESM8'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 250) for MOESM8: We identified 365 genes at FDR < 0.01, FC > 2 and in pathway analysis, the top pathways were significantly associated with T-helper 1 (Th1) versus T-helper 2 (Th2) lymphocyte responses (Fig. 3e [href=#Fig3] , Supplementary Data 6 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#MOESM9'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 409) for MOESM9: Next, we examined if the genomic features associated with immune cluster were also reproducible across cancer types, performing adjusted binomial regressions to estimate cluster association after controlling for tumour type for genes previously implicated as pan-cancer drivers based on signatures of positive selection and recorded 56 hits at FDR < 0.05 (Supplementary Data 7 and Fig. 5c [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#MOESM10'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 235) for MOESM10: Among the genomic alterations we identify (for a full list of predicted driver events see Supplementary Data 8 [href=this file] ), it is likely that some establish, while others are selected for, in different immune microenvironments.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#MOESM11'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for MOESM11: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#ack1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 34) for ack1: Acknowledgements [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 59418\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 199452\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 60140 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-9fb54301610f831b072f8801d4c9fa5ade13814bcb0f8515c54b3b741ee29fec\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 60140 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\n",
      "  \"datasets\": [\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE35069\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE35069\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE74877\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE74877\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE38270\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38270\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"GSE68379\",\n",
      "      \"repository_reference\": \"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE68379\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"PRJEB11844\",\n",
      "      \"repository_reference\": \"https://www.ebi.ac.uk/ena/data/view/PRJEB11844\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"PRJNA258216\",\n",
      "      \"repository_reference\": \"https://www.ebi.ac.uk/ena/data/view/PRJNA258216\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"PRJEB5468\",\n",
      "      \"repository_reference\": \"https://www.ebi.ac.uk/ena/data/view/PRJEB5468\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"syn7214402\",\n",
      "      \"repository_reference\": \"https://www.synapse.org/#!Synapse:syn7214402\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"syn7343873\",\n",
      "      \"repository_reference\": \"https://www.synapse.org/#!Synapse:syn7343873\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"syn2812961\",\n",
      "      \"repository_reference\": \"https://www.synapse.org/#!Synapse:syn2812961\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"PMC6089972\",\n",
      "      \"repository_reference\": \"https://pmc.ncbi.nlm.nih.gov/articles/PMC6089972/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"30104673\",\n",
      "      \"repository_reference\": \"https://pubmed.ncbi.nlm.nih.gov/30104673/\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    }\n",
      "  ]\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'GSE35069', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE35069'}, {'dataset_identifier': 'GSE74877', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE74877'}, {'dataset_identifier': 'GSE38270', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38270'}, {'dataset_identifier': 'GSE68379', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE68379'}, {'dataset_identifier': 'PRJEB11844', 'repository_reference': 'https://www.ebi.ac.uk/ena/data/view/PRJEB11844'}, {'dataset_identifier': 'PRJNA258216', 'repository_reference': 'https://www.ebi.ac.uk/ena/data/view/PRJNA258216'}, {'dataset_identifier': 'PRJEB5468', 'repository_reference': 'https://www.ebi.ac.uk/ena/data/view/PRJEB5468'}, {'dataset_identifier': 'syn7214402', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn7214402'}, {'dataset_identifier': 'syn7343873', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn7343873'}, {'dataset_identifier': 'syn2812961', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn2812961'}, {'dataset_identifier': 'PMC6089972', 'repository_reference': 'https://pmc.ncbi.nlm.nih.gov/articles/PMC6089972/'}, {'dataset_identifier': '30104673', 'repository_reference': 'https://pubmed.ncbi.nlm.nih.gov/30104673/'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 13 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE35069\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE74877\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE38270\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: GSE68379\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: PRJEB11844\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: PRJNA258216\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: PRJEB5468\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn7214402\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn7343873\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: syn2812961\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: PMC6089972\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: 30104673\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'GSE35069', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE35069', 'dataset_id': 'GSE35069'}, {'dataset_identifier': 'GSE74877', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE74877', 'dataset_id': 'GSE74877'}, {'dataset_identifier': 'GSE38270', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38270', 'dataset_id': 'GSE38270'}, {'dataset_identifier': 'GSE68379', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE68379', 'dataset_id': 'GSE68379'}, {'dataset_identifier': 'PRJEB11844', 'repository_reference': 'https://www.ebi.ac.uk/ena/data/view/PRJEB11844', 'dataset_id': 'PRJEB11844'}, {'dataset_identifier': 'PRJNA258216', 'repository_reference': 'https://www.ebi.ac.uk/ena/data/view/PRJNA258216', 'dataset_id': 'PRJNA258216'}, {'dataset_identifier': 'PRJEB5468', 'repository_reference': 'https://www.ebi.ac.uk/ena/data/view/PRJEB5468', 'dataset_id': 'PRJEB5468'}, {'dataset_identifier': 'syn7214402', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn7214402', 'dataset_id': 'syn7214402'}, {'dataset_identifier': 'syn7343873', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn7343873', 'dataset_id': 'syn7343873'}, {'dataset_identifier': 'syn2812961', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn2812961', 'dataset_id': 'syn2812961'}, {'dataset_identifier': 'PMC6089972', 'repository_reference': 'https://pmc.ncbi.nlm.nih.gov/articles/PMC6089972/', 'dataset_id': 'PMC6089972'}, {'dataset_identifier': '30104673', 'repository_reference': 'https://pubmed.ncbi.nlm.nih.gov/30104673/', 'dataset_id': '30104673'}, {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE35069', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE35069', 'dataset_id': 'GSE35069'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE35069\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE35069, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE35069\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE35069\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE35069', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE74877', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE74877', 'dataset_id': 'GSE74877'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE74877\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE74877, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE74877\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE74877\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE74877', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE38270', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38270', 'dataset_id': 'GSE38270'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE38270\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE38270, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38270\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE38270\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE38270', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'GSE68379', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE68379', 'dataset_id': 'GSE68379'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(\\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: GSE68379\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: GSE68379, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE68379\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: GSE68379\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'GSE68379', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'PRJEB11844', 'repository_reference': 'https://www.ebi.ac.uk/ena/data/view/PRJEB11844', 'dataset_id': 'PRJEB11844'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: PRJEB11844\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID PRJEB11844 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.ebi.ac.uk/ena/data/view/PRJEB11844, identifier: PRJEB11844\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.ebi.ac.uk/ena/data/view/PRJEB11844\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier PRJEB11844 for https://www.ebi.ac.uk/ena/data/view/PRJEB11844\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier PRJEB11844 for https://www.ebi.ac.uk/ena/data/view/PRJEB11844\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier PRJEB11844 for https://www.ebi.ac.uk/ena/data/view/PRJEB11844\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier PRJEB11844 for https://www.ebi.ac.uk/ena/data/view/PRJEB11844\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier PRJEB11844 for https://www.ebi.ac.uk/ena/data/view/PRJEB11844\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://www.ebi.ac.uk/ena/data/view/PRJEB11844\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'PRJEB11844', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'PRJNA258216', 'repository_reference': 'https://www.ebi.ac.uk/ena/data/view/PRJNA258216', 'dataset_id': 'PRJNA258216'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: PRJNA258216\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID PRJNA258216 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.ebi.ac.uk/ena/data/view/PRJNA258216, identifier: PRJNA258216\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.ebi.ac.uk/ena/data/view/PRJNA258216\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier PRJNA258216 for https://www.ebi.ac.uk/ena/data/view/PRJNA258216\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier PRJNA258216 for https://www.ebi.ac.uk/ena/data/view/PRJNA258216\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier PRJNA258216 for https://www.ebi.ac.uk/ena/data/view/PRJNA258216\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier PRJNA258216 for https://www.ebi.ac.uk/ena/data/view/PRJNA258216\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier PRJNA258216 for https://www.ebi.ac.uk/ena/data/view/PRJNA258216\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://www.ebi.ac.uk/ena/data/view/PRJNA258216\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'PRJNA258216', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'PRJEB5468', 'repository_reference': 'https://www.ebi.ac.uk/ena/data/view/PRJEB5468', 'dataset_id': 'PRJEB5468'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: PRJEB5468\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID PRJEB5468 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.ebi.ac.uk/ena/data/view/PRJEB5468, identifier: PRJEB5468\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.ebi.ac.uk/ena/data/view/PRJEB5468\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier PRJEB5468 for https://www.ebi.ac.uk/ena/data/view/PRJEB5468\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier PRJEB5468 for https://www.ebi.ac.uk/ena/data/view/PRJEB5468\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier PRJEB5468 for https://www.ebi.ac.uk/ena/data/view/PRJEB5468\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier PRJEB5468 for https://www.ebi.ac.uk/ena/data/view/PRJEB5468\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier PRJEB5468 for https://www.ebi.ac.uk/ena/data/view/PRJEB5468\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://www.ebi.ac.uk/ena/data/view/PRJEB5468\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'PRJEB5468', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn7214402', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn7214402', 'dataset_id': 'syn7214402'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn7214402\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn7214402, synapse.org, https://www.synapse.org/#!Synapse:syn7214402\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn7214402\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn7214402', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn7343873', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn7343873', 'dataset_id': 'syn7343873'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn7343873\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn7343873, synapse.org, https://www.synapse.org/#!Synapse:syn7343873\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn7343873\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn7343873', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'syn2812961', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn2812961', 'dataset_id': 'syn2812961'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn2812961\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn2812961, synapse.org, https://www.synapse.org/#!Synapse:syn2812961\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn2812961\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 662 - INFO - Data repository found via pattern matching: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn2812961', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'PMC6089972', 'repository_reference': 'https://pmc.ncbi.nlm.nih.gov/articles/PMC6089972/', 'dataset_id': 'PMC6089972'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: PMC6089972\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID PMC6089972 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://pmc.ncbi.nlm.nih.gov/articles/PMC6089972/, identifier: PMC6089972\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://pmc.ncbi.nlm.nih.gov/articles/PMC6089972/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier PMC6089972 for https://pmc.ncbi.nlm.nih.gov/articles/PMC6089972/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier PMC6089972 for https://pmc.ncbi.nlm.nih.gov/articles/PMC6089972/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier PMC6089972 for https://pmc.ncbi.nlm.nih.gov/articles/PMC6089972/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier PMC6089972 for https://pmc.ncbi.nlm.nih.gov/articles/PMC6089972/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier PMC6089972 for https://pmc.ncbi.nlm.nih.gov/articles/PMC6089972/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC6089972/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'PMC6089972', 'data_repository': 'pmc.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': '30104673', 'repository_reference': 'https://pubmed.ncbi.nlm.nih.gov/30104673/', 'dataset_id': '30104673'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: 30104673\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1028 - INFO - Accession ID 30104673 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://pubmed.ncbi.nlm.nih.gov/30104673/, identifier: 30104673\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://pubmed.ncbi.nlm.nih.gov/30104673/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 30104673 for https://pubmed.ncbi.nlm.nih.gov/30104673/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 30104673 for https://pubmed.ncbi.nlm.nih.gov/30104673/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier 30104673 for https://pubmed.ncbi.nlm.nih.gov/30104673/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier 30104673 for https://pubmed.ncbi.nlm.nih.gov/30104673/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier 30104673 for https://pubmed.ncbi.nlm.nih.gov/30104673/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: https://pubmed.ncbi.nlm.nih.gov/30104673/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': '30104673', 'data_repository': 'pubmed.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 677 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 605 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'GSE35069', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE74877', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE38270', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GSE68379', 'data_repository': 'geo', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'PRJEB11844', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'PRJNA258216', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'PRJEB5468', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn7214402', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn7343873', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn2812961', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'PMC6089972', 'data_repository': 'pmc.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}, {'dataset_identifier': '30104673', 'data_repository': 'pubmed.ncbi.nlm.nih.gov', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 12 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE35069\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE35069\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE35069 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE35069 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE35069 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE35069\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE35069\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE74877\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE74877\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE74877 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE74877 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE74877 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 2 with repo: geo and accession_id: GSE74877\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE74877\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 2: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE38270\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE38270\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE38270 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE38270 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE38270 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 3 with repo: geo and accession_id: GSE38270\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38270\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 3: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: GSE68379\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: geo, identifier: GSE68379\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE68379 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE68379 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for GSE68379 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 4 with repo: geo and accession_id: GSE68379\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE68379\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 4: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: PRJEB11844\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: www.ebi.ac.uk, identifier: PRJEB11844\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for PRJEB11844 in www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 5 with repo: www.ebi.ac.uk and accession_id: PRJEB11844\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ebi.ac.uk/pride/archive/projects/PRJEB11844\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 5: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: PRJNA258216\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: www.ebi.ac.uk, identifier: PRJNA258216\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for PRJNA258216 in www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 6 with repo: www.ebi.ac.uk and accession_id: PRJNA258216\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ebi.ac.uk/pride/archive/projects/PRJNA258216\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 6: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: PRJEB5468\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: www.ebi.ac.uk, identifier: PRJEB5468\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for PRJEB5468 in www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 7 with repo: www.ebi.ac.uk and accession_id: PRJEB5468\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.ebi.ac.uk/pride/archive/projects/PRJEB5468\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 7: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn7214402\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn7214402\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn7214402 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn7214402 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn7214402 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn7214402 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn7214402 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 8 with repo: synapse.org and accession_id: syn7214402\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn7214402\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 8: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn7343873\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn7343873\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn7343873 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn7343873 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn7343873 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn7343873 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn7343873 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 9 with repo: synapse.org and accession_id: syn7343873\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn7343873\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 9: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 10 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn2812961\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn2812961\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn2812961 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn2812961 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn2812961 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn2812961 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn2812961 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 10 with repo: synapse.org and accession_id: syn2812961\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn2812961\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 10: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 11 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: PMC6089972\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: pmc.ncbi.nlm.nih.gov, identifier: PMC6089972\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier PMC6089972 for pmc.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier PMC6089972 for pmc.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier PMC6089972 for pmc.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier PMC6089972 for pmc.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier PMC6089972 for pmc.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: pmc.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 976 - WARNING - URL pmc.ncbi.nlm.nih.gov may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for PMC6089972 in pmc.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 11 with repo: pmc.ncbi.nlm.nih.gov and accession_id: PMC6089972\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1254 - WARNING - Repository pmc.ncbi.nlm.nih.gov unknown in Ontology. Skipping dataset page 11.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 12 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: 30104673\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: pubmed.ncbi.nlm.nih.gov, identifier: 30104673\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 30104673 for pubmed.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 30104673 for pubmed.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier 30104673 for pubmed.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier 30104673 for pubmed.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern syn\\d{7,12} match with identifier 30104673 for pubmed.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: pubmed.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 976 - WARNING - URL pubmed.ncbi.nlm.nih.gov may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for 30104673 in pubmed.ncbi.nlm.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 12 with repo: pubmed.ncbi.nlm.nih.gov and accession_id: 30104673\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1254 - WARNING - Repository pubmed.ncbi.nlm.nih.gov unknown in Ontology. Skipping dataset page 12.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 12\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 40, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 640 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - Parsed data extraction completed. Elements collected: 24\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 481 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 52th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC8130821)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC8130821\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC8130821&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 394 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC8130821.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b833ddd0>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 376 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 311 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC8130821 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 453 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC8130821, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 445 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC8130821)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 547 - INFO - Extracted 9 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 557 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 0 <a> elements with href='#SD1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 0) for SD1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 24 <a> elements with href='#SD2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 4412) for SD2: In the discovery phase, we performed a PWAS by integrating AD GWAS results (N=455,258) with 376 human brain proteomes profiled from the dorsolateral prefrontal cortex (dPFC; Supplementary Table 1a [href=this file] ) using the FUSION pipeline . The PWAS identified 13 genes whose -regulated brain protein levels were associated with AD at false discovery rate (FDR) p<0.05 ( Figure 1 [href=#F1] , Table 1 [href=#T1] , Extended Data Figure 1a [href=#F2] , Supplementary Table 2 [href=this file] ).\n",
      "A confirmatory PWAS was performed using the same AD GWAS and an independent set of 152 human brain proteomes profiled from the dPFC ( Supplementary Table 1b [href=this file] ) . Correlation between the protein weights in the discovery and confirmatory datasets was high (median 0.85, interquartile range 0.21; Supplementary table 3 [href=this file] ). Ten of these 13 proteins could be tested and all 10 proteins replicated in the confirmatory PWAS ( Table 1 [href=#T1] ; Extended Data Figure 1b [href=#F2] ; Supplementary table 4 [href=this file] ).\n",
      "We found 9 of 13 genes consistent with being causal ( Table 2 [href=#T2] ; Supplementary Table 5 [href=this file] ). SMR results suggest that the -regulated protein abundance mediates the association between genetic variants and AD for all these 13 genes, but HEIDI results argue against a causal role for 4 genes due to linkage disequilibrium ( Table 2 [href=#T2] ; Supplementary Table 6 [href=this file] ).\n",
      "Full results for TWAS is in Supplementary tables 17 [href=this file] - 18 [href=this file] . “ ” in “TWAS significant” column refers to genes with 0.05 <TWAS nominal p <0.1. refers to genes not within 1Mb window of SNPs with p < 5E-08 identified in Jansen et. al.\n",
      "That analysis found the 13 original PWAS-significant genes and 6 additional significant genes at FDR p<0.05 ( , , ; Supplementary table 7 [href=this file] ). Moreover, results from COLOC and SMR/HEIDI tests found the same evidence of causality as the original findings except that was now consistent with causality by both COLOC and SMR/HEIDI compared to mixed findings before ( Supplementary tables 8 [href=this file] - 9 [href=this file] ). The 6 additional genes were not consistent with being causal by COLOC ( Supplementary Table 8 [href=this file] ).\n",
      "The PWAS of clinical AD identified 4 genes, ALS 7 genes, PD 17 genes, neuroticism 72 genes, height 662 genes, BMI 395 genes, and WHRadjBMI 244 genes ( Supplementary Tables 10 [href=this file] - 16 [href=this file] ).\n",
      "The 888 transcriptomes were mainly from the frontal cortex donated by participants of European descent ( Supplementary table 1c [href=this file] ), and quality control was analogous to that of the proteomes to remove technical and clinical characteristics before estimating the effect of genetic variants on mRNA expression. The AD TWAS identified 40 genes whose genetically regulated mRNA expression levels were associated with AD at FDR p<0.05 ( Extended Data Figure 3 [href=#F4] ; Supplementary table 17 [href=this file] ). Among the 11 potentially causal genes identified at the protein-level, five genes, , , , , and showed at least nominal significance with similar directions of association with AD as seen at the protein-level ( Table 3 [href=#T3] ; Supplementary table 18a [href=this file] ).\n",
      "For the 5 genes with evidence at both the transcript and protein levels, results from SMR test for two molecular traits suggested their protein abundance is mediated by mRNA expression ( Supplementary table 18a [href=this file] , b [href=this file] ). For the three genes with suggestive evidence for -regulated mRNA’s association with AD ( , , and ), only had evidence to suggest protein expression is mediated by mRNA expression level ( Supplementary table 18a [href=this file] , b [href=this file] ).\n",
      "Using human single-cell RNA-sequencing data profiled from the dPFC we found cell-type specific enrichment for expression of 6 of the 11 causal genes at FDR p-value < 0.05 (adjusted for 17,775 genes). , , , and were enriched in excitatory neurons, whereas showed enrichment in oligodendrocytes and in astrocytes and microglia ( Extended Data Figure 4 [href=#F5] ; Supplementary table 19 [href=this file] ).\n",
      "Principal component analysis was performed to compare genetic ancestry of these individuals to CEU from 1000 Genomes Project ( Extended Data Figure 5 [href=#F6] ; Supplementary Table 20 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#fn-group1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 27) for fn-group1: Footnotes [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#fn-group1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 27) for fn-group1: Footnotes [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#fn-group1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 27) for fn-group1: Footnotes [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#fn-group1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 27) for fn-group1: Footnotes [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 568 - INFO - Found 1 <a> elements with href='#fn-group1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 576 - INFO - Extracted context description (len: 27) for fn-group1: Footnotes [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 367 - INFO - Extracting links from full HTML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 46287\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 148659\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 47009 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-eb9123ea03f5e2b68eb101e66353f46b2135cab8c95085e1bc3ff0d42c924dbc\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 47009 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 480 - INFO - GPT response: {\n",
      "  \"datasets\": [\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"n/a\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"https://www.synapse.org/#!Synapse:syn17015098\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"https://www.synapse.org/#!Synapse:syn9884314\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"https://www.synapse.org/#!Synapse:syn23191787\"\n",
      "    },\n",
      "    {\n",
      "      \"dataset_identifier\": \"n/a\",\n",
      "      \"repository_reference\": \"https://www.synapse.org/#!Synapse:syn20803583\"\n",
      "    }\n",
      "  ]\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 484 - INFO - Response is <class 'dict'>: {'datasets': [{'dataset_identifier': 'n/a', 'repository_reference': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn17015098'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn9884314'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn23191787'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn20803583'}]}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 693 - INFO - Deduplicating response with 5 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 711 - INFO - Adding unique item: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 487 - INFO - Response is <class 'list'>: [{'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn17015098'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn9884314'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn23191787'}, {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn20803583'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: None, None, None\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1011 - INFO - Validating accession ID: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1016 - INFO - Accession ID n/a is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: n/a, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Extracting repo domain from URL: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1160 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 677 - INFO - Data repository is 'n/a', skipping dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 605 - INFO - Skipping dataset due to missing ID or repository: {'dataset_identifier': 'n/a', 'repository_reference': 'n/a', 'dataset_id': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn17015098'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn17015098\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn17015098, None, https://www.synapse.org/#!Synapse:syn17015098\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn17015098\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn17015098, identifier: syn17015098\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn17015098\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn17015098 for https://www.synapse.org/#!Synapse:syn17015098\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn17015098 for https://www.synapse.org/#!Synapse:syn17015098\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn17015098 for https://www.synapse.org/#!Synapse:syn17015098\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn17015098 for https://www.synapse.org/#!Synapse:syn17015098\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn17015098 in Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn17015098', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn9884314'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn9884314\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn9884314, None, https://www.synapse.org/#!Synapse:syn9884314\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn9884314\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn9884314, identifier: syn9884314\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn9884314\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn9884314 for https://www.synapse.org/#!Synapse:syn9884314\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn9884314 for https://www.synapse.org/#!Synapse:syn9884314\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn9884314 for https://www.synapse.org/#!Synapse:syn9884314\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn9884314 for https://www.synapse.org/#!Synapse:syn9884314\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn9884314 in Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn9884314', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn23191787'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn23191787\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn23191787, None, https://www.synapse.org/#!Synapse:syn23191787\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn23191787\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn23191787, identifier: syn23191787\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn23191787\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn23191787 for https://www.synapse.org/#!Synapse:syn23191787\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn23191787 for https://www.synapse.org/#!Synapse:syn23191787\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn23191787 for https://www.synapse.org/#!Synapse:syn23191787\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn23191787 for https://www.synapse.org/#!Synapse:syn23191787\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn23191787 in Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn23191787', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 585 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn20803583'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 600 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 641 - INFO - Using id_pattern to extract ID: .*(syn\\d{7,12}).*\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted ID: syn20803583\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Schema validation vals: syn20803583, None, https://www.synapse.org/#!Synapse:syn20803583\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 654 - INFO - Dataset ID found via pattern matching: syn20803583\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn20803583, identifier: syn20803583\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 886 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn20803583\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn20803583 for https://www.synapse.org/#!Synapse:syn20803583\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn20803583 for https://www.synapse.org/#!Synapse:syn20803583\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn20803583 for https://www.synapse.org/#!Synapse:syn20803583\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn20803583 for https://www.synapse.org/#!Synapse:syn20803583\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn20803583 in Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 620 - INFO - Extracted dataset: {'dataset_identifier': 'syn20803583', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 376 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn17015098', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn9884314', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn23191787', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn20803583', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1181 - INFO - Fetching metadata for 4 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn17015098\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn17015098\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn17015098 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn17015098 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn17015098 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn17015098 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn17015098 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn17015098\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn17015098\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn9884314\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn9884314\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn9884314 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn9884314 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn9884314 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn9884314 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn9884314 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn9884314\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn9884314\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn23191787\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn23191787\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn23191787 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn23191787 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn23191787 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn23191787 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn23191787 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 3 with repo: synapse.org and accession_id: syn23191787\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn23191787\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 3: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1189 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1200 - INFO - Raw accession ID: syn20803583\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1103 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn20803583\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn20803583 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn20803583 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn20803583 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1151 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn20803583 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1132 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1085 - INFO - Resolving accession ID for syn20803583 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1214 - INFO - Processing dataset 4 with repo: synapse.org and accession_id: syn20803583\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1240 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn20803583\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1247 - INFO - Adding access mode for dataset 4: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1258 - INFO - Updated datasets len: 4\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 13, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 640 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 465 - INFO - Parsed data extraction completed. Elements collected: 7\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 471 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 481 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 554 - INFO - 53th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11224609)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 358 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11224609\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 365 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 534 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC11224609&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 406 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC11224609.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 410 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11224609 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 424 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 430 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 177 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11224609, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 443 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 264 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 454 - INFO - Found 2 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b689e7c0>, <Element supplementary-material at 0x1b03ddd80>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 462 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b689e7c0>, <Element supplementary-material at 0x1b03ddd80>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 548 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref sup1: To illustrate the overall workflow, we present a toy example showing a simple schematic view and the order of major computation steps in CWAS-Plus ( Supplementary Fig. 1 [this file] ). \n",
      "Additionally, we examined the memory usage of CWAS-Plus with an increasing number of samples and found an approximately linear increase ( Supplementary Fig. 2 [this file] ). \n",
      "Our observations revealed that variants from most clusters exhibited a higher percentage of variants in higher score bins than non-RNVs ( Supplementary Fig. 3 [this file] ), indicating that CWAS-Plus prioritized more pathogenic variants within the broader pool of noncoding variants. \n",
      "CWAS-Plus demonstrated significant case enrichment (OR = 1.3; = 5.7 10 , 95% CI: 1.1–1.6), while Sei, Enformer, and STAARpipeline showed no significant case enrichment ( Supplementary Fig. 4 [this file] , Supplementary Table 6 [rid=sup2] ). \n",
      "CWAS analyses for variants showed highly consistent estimates between the variant-level and sample-level tests ( Supplementary Fig. 5 [this file] , Supplementary Table 7 [rid=sup2] ). \n",
      "Our extended analysis comparing the ability to identify risk variants revealed that only CWAS-Plus demonstrated significant case enrichment in risk variant carriers, whereas Sei, Enformer, and STAARpipeline did not ( Supplementary Fig. 4 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 564 - INFO - Extracted context_descr for xref sup2: CWAS-Plus was 50 times faster than CWAS, highlighting its superior computational speed ( Fig. 2A [rid=f2] , Supplementary Table 1 [this file] ). \n",
      "The scalability of the CWAS-Plus package was tested for run with an increasing number of samples ( Fig. 2B [rid=f2] , Supplementary Table 2 [this file] ) and datasets ( Fig. 2C [rid=f2] , Supplementary Table 3 [this file] ). \n",
      "The results showed that noncoding, including intergenic ( = 0.20%, = 1.3 10 ; Supplementary Tables 4 [this file] and 5 [this file] ) and 5’ UTR ( = 0.15%, = 1.1 10 ), categories were significantly enriched ( Fig. 3B [rid=f3] ). \n",
      "CWAS-Plus demonstrated significant case enrichment (OR = 1.3; = 5.7 10 , 95% CI: 1.1–1.6), while Sei, Enformer, and STAARpipeline showed no significant case enrichment ( Supplementary Fig. 4 [rid=sup1] , Supplementary Table 6 [this file] ). \n",
      "CWAS analyses for variants showed highly consistent estimates between the variant-level and sample-level tests ( Supplementary Fig. 5 [rid=sup1] , Supplementary Table 7 [this file] ). \n",
      "Since no single category fully explained AD risk associated with rare noncoding variants, we conducted a risk score analysis to identify predictors among diverse categories ( Fig. 4A [rid=f4] , Supplementary Tables 8 [this file] and 9 [this file] ). \n",
      "Set 3 showed the highest performance, outperforming set 1, followed by set 2 ( = 0.08%, = 5.5 10 for set 1; = 0.17%, = 4.3 10 for set 2; = 0.33%, = 3.7 10 for set 3; Fig. 5B [rid=f5] , Supplementary Table 10 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 243 - INFO - Extracting links from full XML content.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1282 - INFO - Number of tokens: 45367\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 417 - INFO - Content length: 174591\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 428 - INFO - Prompt messages total length: 46089 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 434 - INFO - Prompt ID: gpt-4o-mini-0-39f91cfbe3a26b93dedee4e532f44b11bd59a98a1f65d66df2d238adb8e6c673\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1299 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 453 - INFO - Requesting datasets from content using model: gpt-4o-mini, temperature: 0, messages length: 46089 tokens\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 1068 - INFO - Closing the EntrezFetcher.\u001B[0m\n"
     ]
    },
    {
     "ename": "KeyboardInterrupt",
     "evalue": "",
     "output_type": "error",
     "traceback": [
      "\u001B[31m---------------------------------------------------------------------------\u001B[39m",
      "\u001B[31mKeyboardInterrupt\u001B[39m                         Traceback (most recent call last)",
      "\u001B[36mCell\u001B[39m\u001B[36m \u001B[39m\u001B[32mIn[7]\u001B[39m\u001B[32m, line 2\u001B[39m\n\u001B[32m      1\u001B[39m input_file = \u001B[33m\"\u001B[39m\u001B[33mexp_input/SAGE_input.txt\u001B[39m\u001B[33m\"\u001B[39m\n\u001B[32m----> \u001B[39m\u001B[32m2\u001B[39m combined_df = dg.run(input_file=input_file, semantic_retrieval=semantic_retrieval, section_filter=section_filter, prompt_name=prompt)\n",
      "\u001B[36mFile \u001B[39m\u001B[32m~/Desktop/VIDA-NYU/data-gatherer/data_gatherer/data_gatherer.py:1028\u001B[39m, in \u001B[36mDataGatherer.run\u001B[39m\u001B[34m(self, input_file, semantic_retrieval, section_filter, prompt_name)\u001B[39m\n\u001B[32m   1025\u001B[39m urls = \u001B[38;5;28mself\u001B[39m.load_urls_from_input(input_file)\n\u001B[32m   1027\u001B[39m \u001B[38;5;66;03m# Process each URL and return results as a dictionary like source_url: DataFrame_of_data_links\u001B[39;00m\n\u001B[32m-> \u001B[39m\u001B[32m1028\u001B[39m results = \u001B[38;5;28mself\u001B[39m.process_articles(urls, semantic_retrieval=semantic_retrieval, section_filter=section_filter,\n\u001B[32m   1029\u001B[39m                                 prompt_name=prompt_name, driver_path=\u001B[38;5;28mself\u001B[39m.fetcher_driver_path)\n\u001B[32m   1031\u001B[39m \u001B[38;5;66;03m# return the union of all the results\u001B[39;00m\n\u001B[32m   1032\u001B[39m combined_df = pd.DataFrame()\n",
      "\u001B[36mFile \u001B[39m\u001B[32m~/Desktop/VIDA-NYU/data-gatherer/data_gatherer/data_gatherer.py:556\u001B[39m, in \u001B[36mDataGatherer.process_articles\u001B[39m\u001B[34m(self, url_list, log_modulo, save_staging_table, article_file_dir, driver_path, browser, headless, use_portkey_for_gemini, prompt_name, semantic_retrieval, section_filter)\u001B[39m\n\u001B[32m    553\u001B[39m url = \u001B[38;5;28mself\u001B[39m.preprocess_url(url)\n\u001B[32m    554\u001B[39m \u001B[38;5;28mself\u001B[39m.logger.info(\u001B[33mf\u001B[39m\u001B[33m\"\u001B[39m\u001B[38;5;132;01m{\u001B[39;00miteration\u001B[38;5;132;01m}\u001B[39;00m\u001B[33mth function call: self.process_url(\u001B[39m\u001B[38;5;132;01m{\u001B[39;00murl\u001B[38;5;132;01m}\u001B[39;00m\u001B[33m)\u001B[39m\u001B[33m\"\u001B[39m)\n\u001B[32m--> \u001B[39m\u001B[32m556\u001B[39m results[url] = \u001B[38;5;28mself\u001B[39m.process_url(\n\u001B[32m    557\u001B[39m     url,\n\u001B[32m    558\u001B[39m     save_staging_table=save_staging_table,\n\u001B[32m    559\u001B[39m     article_file_dir=article_file_dir,\n\u001B[32m    560\u001B[39m     driver_path=driver_path,\n\u001B[32m    561\u001B[39m     browser=browser,\n\u001B[32m    562\u001B[39m     headless=headless,\n\u001B[32m    563\u001B[39m     use_portkey_for_gemini=use_portkey_for_gemini,\n\u001B[32m    564\u001B[39m     prompt_name=prompt_name,\n\u001B[32m    565\u001B[39m     semantic_retrieval=semantic_retrieval,\n\u001B[32m    566\u001B[39m     section_filter=section_filter\n\u001B[32m    567\u001B[39m )\n\u001B[32m    569\u001B[39m \u001B[38;5;28;01mif\u001B[39;00m iteration % log_modulo == \u001B[32m0\u001B[39m:\n\u001B[32m    570\u001B[39m     elapsed = time.time() - start_time  \u001B[38;5;66;03m# Time elapsed since start\u001B[39;00m\n",
      "\u001B[36mFile \u001B[39m\u001B[32m~/Desktop/VIDA-NYU/data-gatherer/data_gatherer/data_gatherer.py:438\u001B[39m, in \u001B[36mDataGatherer.process_url\u001B[39m\u001B[34m(self, url, save_staging_table, article_file_dir, use_portkey_for_gemini, driver_path, browser, headless, prompt_name, semantic_retrieval, section_filter)\u001B[39m\n\u001B[32m    431\u001B[39m \u001B[38;5;28mself\u001B[39m.parser = XMLParser(\u001B[38;5;28mself\u001B[39m.open_data_repos_ontology, \u001B[38;5;28mself\u001B[39m.logger,\n\u001B[32m    432\u001B[39m                         llm_name=\u001B[38;5;28mself\u001B[39m.llm,\n\u001B[32m    433\u001B[39m                         full_document_read=\u001B[38;5;28mself\u001B[39m.full_document_read,\n\u001B[32m    434\u001B[39m                         use_portkey_for_gemini=use_portkey_for_gemini,\n\u001B[32m    435\u001B[39m                         save_dynamic_prompts=\u001B[38;5;28mself\u001B[39m.save_dynamic_prompts)\n\u001B[32m    437\u001B[39m \u001B[38;5;28;01mif\u001B[39;00m additional_data \u001B[38;5;129;01mis\u001B[39;00m \u001B[38;5;28;01mNone\u001B[39;00m:\n\u001B[32m--> \u001B[39m\u001B[32m438\u001B[39m     parsed_data = \u001B[38;5;28mself\u001B[39m.parser.parse_data(raw_data, \u001B[38;5;28mself\u001B[39m.publisher, \u001B[38;5;28mself\u001B[39m.current_url,\n\u001B[32m    439\u001B[39m                                          prompt_name=prompt_name, semantic_retrieval=semantic_retrieval,\n\u001B[32m    440\u001B[39m                                          section_filter=section_filter)\n\u001B[32m    442\u001B[39m \u001B[38;5;28;01melse\u001B[39;00m:\n\u001B[32m    443\u001B[39m     \u001B[38;5;28mself\u001B[39m.logger.info(\u001B[33mf\u001B[39m\u001B[33m\"\u001B[39m\u001B[33mProcessing additional data. # of items: \u001B[39m\u001B[38;5;132;01m{\u001B[39;00m\u001B[38;5;28mlen\u001B[39m(additional_data)\u001B[38;5;132;01m}\u001B[39;00m\u001B[33m\"\u001B[39m)\n",
      "\u001B[36mFile \u001B[39m\u001B[32m~/Desktop/VIDA-NYU/data-gatherer/data_gatherer/parser/xml_parser.py:250\u001B[39m, in \u001B[36mXMLParser.parse_data\u001B[39m\u001B[34m(self, api_data, publisher, current_url_address, additional_data, raw_data_format, article_file_dir, process_DAS_links_separately, section_filter, prompt_name, use_portkey_for_gemini, semantic_retrieval)\u001B[39m\n\u001B[32m    247\u001B[39m \u001B[38;5;28mself\u001B[39m.logger.debug(\u001B[33mf\u001B[39m\u001B[33m\"\u001B[39m\u001B[33mPreprocessed data: \u001B[39m\u001B[38;5;132;01m{\u001B[39;00mpreprocessed_data\u001B[38;5;132;01m}\u001B[39;00m\u001B[33m\"\u001B[39m)\n\u001B[32m    249\u001B[39m \u001B[38;5;66;03m# Extract dataset links from the entire text\u001B[39;00m\n\u001B[32m--> \u001B[39m\u001B[32m250\u001B[39m augmented_dataset_links = \u001B[38;5;28mself\u001B[39m.extract_datasets_info_from_content(preprocessed_data,\n\u001B[32m    251\u001B[39m                                                               \u001B[38;5;28mself\u001B[39m.open_data_repos_ontology[\u001B[33m'\u001B[39m\u001B[33mrepos\u001B[39m\u001B[33m'\u001B[39m],\n\u001B[32m    252\u001B[39m                                                               model=\u001B[38;5;28mself\u001B[39m.llm_name,\n\u001B[32m    253\u001B[39m                                                               temperature=\u001B[32m0\u001B[39m,\n\u001B[32m    254\u001B[39m                                                               prompt_name=prompt_name)\n\u001B[32m    256\u001B[39m \u001B[38;5;28mself\u001B[39m.logger.info(\u001B[33mf\u001B[39m\u001B[33m\"\u001B[39m\u001B[33mAugmented dataset links: \u001B[39m\u001B[38;5;132;01m{\u001B[39;00maugmented_dataset_links\u001B[38;5;132;01m}\u001B[39;00m\u001B[33m\"\u001B[39m)\n\u001B[32m    258\u001B[39m dataset_links_w_target_pages = \u001B[38;5;28mself\u001B[39m.get_dataset_page(augmented_dataset_links)\n",
      "\u001B[36mFile \u001B[39m\u001B[32m~/Desktop/VIDA-NYU/data-gatherer/data_gatherer/parser/base_parser.py:470\u001B[39m, in \u001B[36mLLMParser.extract_datasets_info_from_content\u001B[39m\u001B[34m(self, content, repos, model, temperature, prompt_name, full_document_read)\u001B[39m\n\u001B[32m    468\u001B[39m response = \u001B[38;5;28;01mNone\u001B[39;00m\n\u001B[32m    469\u001B[39m \u001B[38;5;28;01mif\u001B[39;00m \u001B[38;5;28mself\u001B[39m.full_document_read:\n\u001B[32m--> \u001B[39m\u001B[32m470\u001B[39m     response = \u001B[38;5;28mself\u001B[39m.client.chat.completions.create(\n\u001B[32m    471\u001B[39m         model=model,\n\u001B[32m    472\u001B[39m         messages=messages,\n\u001B[32m    473\u001B[39m         temperature=temperature,\n\u001B[32m    474\u001B[39m         response_format=dataset_response_schema_gpt\n\u001B[32m    475\u001B[39m     )\n\u001B[32m    476\u001B[39m \u001B[38;5;28;01melse\u001B[39;00m:\n\u001B[32m    477\u001B[39m     response = \u001B[38;5;28mself\u001B[39m.client.chat.completions.create(model=model, messages=messages,\n\u001B[32m    478\u001B[39m                                                    temperature=temperature)\n",
      "\u001B[36mFile \u001B[39m\u001B[32m~/opt/miniconda3/envs/Summer24/lib/python3.11/site-packages/openai/_utils/_utils.py:279\u001B[39m, in \u001B[36mrequired_args.<locals>.inner.<locals>.wrapper\u001B[39m\u001B[34m(*args, **kwargs)\u001B[39m\n\u001B[32m    277\u001B[39m             msg = \u001B[33mf\u001B[39m\u001B[33m\"\u001B[39m\u001B[33mMissing required argument: \u001B[39m\u001B[38;5;132;01m{\u001B[39;00mquote(missing[\u001B[32m0\u001B[39m])\u001B[38;5;132;01m}\u001B[39;00m\u001B[33m\"\u001B[39m\n\u001B[32m    278\u001B[39m     \u001B[38;5;28;01mraise\u001B[39;00m \u001B[38;5;167;01mTypeError\u001B[39;00m(msg)\n\u001B[32m--> \u001B[39m\u001B[32m279\u001B[39m \u001B[38;5;28;01mreturn\u001B[39;00m func(*args, **kwargs)\n",
      "\u001B[36mFile \u001B[39m\u001B[32m~/opt/miniconda3/envs/Summer24/lib/python3.11/site-packages/openai/resources/chat/completions/completions.py:914\u001B[39m, in \u001B[36mCompletions.create\u001B[39m\u001B[34m(self, messages, model, audio, frequency_penalty, function_call, functions, logit_bias, logprobs, max_completion_tokens, max_tokens, metadata, modalities, n, parallel_tool_calls, prediction, presence_penalty, reasoning_effort, response_format, seed, service_tier, stop, store, stream, stream_options, temperature, tool_choice, tools, top_logprobs, top_p, user, web_search_options, extra_headers, extra_query, extra_body, timeout)\u001B[39m\n\u001B[32m    871\u001B[39m \u001B[38;5;129m@required_args\u001B[39m([\u001B[33m\"\u001B[39m\u001B[33mmessages\u001B[39m\u001B[33m\"\u001B[39m, \u001B[33m\"\u001B[39m\u001B[33mmodel\u001B[39m\u001B[33m\"\u001B[39m], [\u001B[33m\"\u001B[39m\u001B[33mmessages\u001B[39m\u001B[33m\"\u001B[39m, \u001B[33m\"\u001B[39m\u001B[33mmodel\u001B[39m\u001B[33m\"\u001B[39m, \u001B[33m\"\u001B[39m\u001B[33mstream\u001B[39m\u001B[33m\"\u001B[39m])\n\u001B[32m    872\u001B[39m \u001B[38;5;28;01mdef\u001B[39;00m\u001B[38;5;250m \u001B[39m\u001B[34mcreate\u001B[39m(\n\u001B[32m    873\u001B[39m     \u001B[38;5;28mself\u001B[39m,\n\u001B[32m   (...)\u001B[39m\u001B[32m    911\u001B[39m     timeout: \u001B[38;5;28mfloat\u001B[39m | httpx.Timeout | \u001B[38;5;28;01mNone\u001B[39;00m | NotGiven = NOT_GIVEN,\n\u001B[32m    912\u001B[39m ) -> ChatCompletion | Stream[ChatCompletionChunk]:\n\u001B[32m    913\u001B[39m     validate_response_format(response_format)\n\u001B[32m--> \u001B[39m\u001B[32m914\u001B[39m     \u001B[38;5;28;01mreturn\u001B[39;00m \u001B[38;5;28mself\u001B[39m._post(\n\u001B[32m    915\u001B[39m         \u001B[33m\"\u001B[39m\u001B[33m/chat/completions\u001B[39m\u001B[33m\"\u001B[39m,\n\u001B[32m    916\u001B[39m         body=maybe_transform(\n\u001B[32m    917\u001B[39m             {\n\u001B[32m    918\u001B[39m                 \u001B[33m\"\u001B[39m\u001B[33mmessages\u001B[39m\u001B[33m\"\u001B[39m: messages,\n\u001B[32m    919\u001B[39m                 \u001B[33m\"\u001B[39m\u001B[33mmodel\u001B[39m\u001B[33m\"\u001B[39m: model,\n\u001B[32m    920\u001B[39m                 \u001B[33m\"\u001B[39m\u001B[33maudio\u001B[39m\u001B[33m\"\u001B[39m: audio,\n\u001B[32m    921\u001B[39m                 \u001B[33m\"\u001B[39m\u001B[33mfrequency_penalty\u001B[39m\u001B[33m\"\u001B[39m: frequency_penalty,\n\u001B[32m    922\u001B[39m                 \u001B[33m\"\u001B[39m\u001B[33mfunction_call\u001B[39m\u001B[33m\"\u001B[39m: function_call,\n\u001B[32m    923\u001B[39m                 \u001B[33m\"\u001B[39m\u001B[33mfunctions\u001B[39m\u001B[33m\"\u001B[39m: functions,\n\u001B[32m    924\u001B[39m                 \u001B[33m\"\u001B[39m\u001B[33mlogit_bias\u001B[39m\u001B[33m\"\u001B[39m: logit_bias,\n\u001B[32m    925\u001B[39m                 \u001B[33m\"\u001B[39m\u001B[33mlogprobs\u001B[39m\u001B[33m\"\u001B[39m: logprobs,\n\u001B[32m    926\u001B[39m                 \u001B[33m\"\u001B[39m\u001B[33mmax_completion_tokens\u001B[39m\u001B[33m\"\u001B[39m: max_completion_tokens,\n\u001B[32m    927\u001B[39m                 \u001B[33m\"\u001B[39m\u001B[33mmax_tokens\u001B[39m\u001B[33m\"\u001B[39m: max_tokens,\n\u001B[32m    928\u001B[39m                 \u001B[33m\"\u001B[39m\u001B[33mmetadata\u001B[39m\u001B[33m\"\u001B[39m: metadata,\n\u001B[32m    929\u001B[39m                 \u001B[33m\"\u001B[39m\u001B[33mmodalities\u001B[39m\u001B[33m\"\u001B[39m: modalities,\n\u001B[32m    930\u001B[39m                 \u001B[33m\"\u001B[39m\u001B[33mn\u001B[39m\u001B[33m\"\u001B[39m: n,\n\u001B[32m    931\u001B[39m                 \u001B[33m\"\u001B[39m\u001B[33mparallel_tool_calls\u001B[39m\u001B[33m\"\u001B[39m: parallel_tool_calls,\n\u001B[32m    932\u001B[39m                 \u001B[33m\"\u001B[39m\u001B[33mprediction\u001B[39m\u001B[33m\"\u001B[39m: prediction,\n\u001B[32m    933\u001B[39m                 \u001B[33m\"\u001B[39m\u001B[33mpresence_penalty\u001B[39m\u001B[33m\"\u001B[39m: presence_penalty,\n\u001B[32m    934\u001B[39m                 \u001B[33m\"\u001B[39m\u001B[33mreasoning_effort\u001B[39m\u001B[33m\"\u001B[39m: reasoning_effort,\n\u001B[32m    935\u001B[39m                 \u001B[33m\"\u001B[39m\u001B[33mresponse_format\u001B[39m\u001B[33m\"\u001B[39m: response_format,\n\u001B[32m    936\u001B[39m                 \u001B[33m\"\u001B[39m\u001B[33mseed\u001B[39m\u001B[33m\"\u001B[39m: seed,\n\u001B[32m    937\u001B[39m                 \u001B[33m\"\u001B[39m\u001B[33mservice_tier\u001B[39m\u001B[33m\"\u001B[39m: service_tier,\n\u001B[32m    938\u001B[39m                 \u001B[33m\"\u001B[39m\u001B[33mstop\u001B[39m\u001B[33m\"\u001B[39m: stop,\n\u001B[32m    939\u001B[39m                 \u001B[33m\"\u001B[39m\u001B[33mstore\u001B[39m\u001B[33m\"\u001B[39m: store,\n\u001B[32m    940\u001B[39m                 \u001B[33m\"\u001B[39m\u001B[33mstream\u001B[39m\u001B[33m\"\u001B[39m: stream,\n\u001B[32m    941\u001B[39m                 \u001B[33m\"\u001B[39m\u001B[33mstream_options\u001B[39m\u001B[33m\"\u001B[39m: stream_options,\n\u001B[32m    942\u001B[39m                 \u001B[33m\"\u001B[39m\u001B[33mtemperature\u001B[39m\u001B[33m\"\u001B[39m: temperature,\n\u001B[32m    943\u001B[39m                 \u001B[33m\"\u001B[39m\u001B[33mtool_choice\u001B[39m\u001B[33m\"\u001B[39m: tool_choice,\n\u001B[32m    944\u001B[39m                 \u001B[33m\"\u001B[39m\u001B[33mtools\u001B[39m\u001B[33m\"\u001B[39m: tools,\n\u001B[32m    945\u001B[39m                 \u001B[33m\"\u001B[39m\u001B[33mtop_logprobs\u001B[39m\u001B[33m\"\u001B[39m: top_logprobs,\n\u001B[32m    946\u001B[39m                 \u001B[33m\"\u001B[39m\u001B[33mtop_p\u001B[39m\u001B[33m\"\u001B[39m: top_p,\n\u001B[32m    947\u001B[39m                 \u001B[33m\"\u001B[39m\u001B[33muser\u001B[39m\u001B[33m\"\u001B[39m: user,\n\u001B[32m    948\u001B[39m                 \u001B[33m\"\u001B[39m\u001B[33mweb_search_options\u001B[39m\u001B[33m\"\u001B[39m: web_search_options,\n\u001B[32m    949\u001B[39m             },\n\u001B[32m    950\u001B[39m             completion_create_params.CompletionCreateParams,\n\u001B[32m    951\u001B[39m         ),\n\u001B[32m    952\u001B[39m         options=make_request_options(\n\u001B[32m    953\u001B[39m             extra_headers=extra_headers, extra_query=extra_query, extra_body=extra_body, timeout=timeout\n\u001B[32m    954\u001B[39m         ),\n\u001B[32m    955\u001B[39m         cast_to=ChatCompletion,\n\u001B[32m    956\u001B[39m         stream=stream \u001B[38;5;129;01mor\u001B[39;00m \u001B[38;5;28;01mFalse\u001B[39;00m,\n\u001B[32m    957\u001B[39m         stream_cls=Stream[ChatCompletionChunk],\n\u001B[32m    958\u001B[39m     )\n",
      "\u001B[36mFile \u001B[39m\u001B[32m~/opt/miniconda3/envs/Summer24/lib/python3.11/site-packages/openai/_base_client.py:1242\u001B[39m, in \u001B[36mSyncAPIClient.post\u001B[39m\u001B[34m(self, path, cast_to, body, options, files, stream, stream_cls)\u001B[39m\n\u001B[32m   1228\u001B[39m \u001B[38;5;28;01mdef\u001B[39;00m\u001B[38;5;250m \u001B[39m\u001B[34mpost\u001B[39m(\n\u001B[32m   1229\u001B[39m     \u001B[38;5;28mself\u001B[39m,\n\u001B[32m   1230\u001B[39m     path: \u001B[38;5;28mstr\u001B[39m,\n\u001B[32m   (...)\u001B[39m\u001B[32m   1237\u001B[39m     stream_cls: \u001B[38;5;28mtype\u001B[39m[_StreamT] | \u001B[38;5;28;01mNone\u001B[39;00m = \u001B[38;5;28;01mNone\u001B[39;00m,\n\u001B[32m   1238\u001B[39m ) -> ResponseT | _StreamT:\n\u001B[32m   1239\u001B[39m     opts = FinalRequestOptions.construct(\n\u001B[32m   1240\u001B[39m         method=\u001B[33m\"\u001B[39m\u001B[33mpost\u001B[39m\u001B[33m\"\u001B[39m, url=path, json_data=body, files=to_httpx_files(files), **options\n\u001B[32m   1241\u001B[39m     )\n\u001B[32m-> \u001B[39m\u001B[32m1242\u001B[39m     \u001B[38;5;28;01mreturn\u001B[39;00m cast(ResponseT, \u001B[38;5;28mself\u001B[39m.request(cast_to, opts, stream=stream, stream_cls=stream_cls))\n",
      "\u001B[36mFile \u001B[39m\u001B[32m~/opt/miniconda3/envs/Summer24/lib/python3.11/site-packages/openai/_base_client.py:919\u001B[39m, in \u001B[36mSyncAPIClient.request\u001B[39m\u001B[34m(self, cast_to, options, remaining_retries, stream, stream_cls)\u001B[39m\n\u001B[32m    916\u001B[39m \u001B[38;5;28;01melse\u001B[39;00m:\n\u001B[32m    917\u001B[39m     retries_taken = \u001B[32m0\u001B[39m\n\u001B[32m--> \u001B[39m\u001B[32m919\u001B[39m \u001B[38;5;28;01mreturn\u001B[39;00m \u001B[38;5;28mself\u001B[39m._request(\n\u001B[32m    920\u001B[39m     cast_to=cast_to,\n\u001B[32m    921\u001B[39m     options=options,\n\u001B[32m    922\u001B[39m     stream=stream,\n\u001B[32m    923\u001B[39m     stream_cls=stream_cls,\n\u001B[32m    924\u001B[39m     retries_taken=retries_taken,\n\u001B[32m    925\u001B[39m )\n",
      "\u001B[36mFile \u001B[39m\u001B[32m~/opt/miniconda3/envs/Summer24/lib/python3.11/site-packages/openai/_base_client.py:964\u001B[39m, in \u001B[36mSyncAPIClient._request\u001B[39m\u001B[34m(self, cast_to, options, retries_taken, stream, stream_cls)\u001B[39m\n\u001B[32m    961\u001B[39m log.debug(\u001B[33m\"\u001B[39m\u001B[33mEncountered httpx.TimeoutException\u001B[39m\u001B[33m\"\u001B[39m, exc_info=\u001B[38;5;28;01mTrue\u001B[39;00m)\n\u001B[32m    963\u001B[39m \u001B[38;5;28;01mif\u001B[39;00m remaining_retries > \u001B[32m0\u001B[39m:\n\u001B[32m--> \u001B[39m\u001B[32m964\u001B[39m     \u001B[38;5;28;01mreturn\u001B[39;00m \u001B[38;5;28mself\u001B[39m._retry_request(\n\u001B[32m    965\u001B[39m         input_options,\n\u001B[32m    966\u001B[39m         cast_to,\n\u001B[32m    967\u001B[39m         retries_taken=retries_taken,\n\u001B[32m    968\u001B[39m         stream=stream,\n\u001B[32m    969\u001B[39m         stream_cls=stream_cls,\n\u001B[32m    970\u001B[39m         response_headers=\u001B[38;5;28;01mNone\u001B[39;00m,\n\u001B[32m    971\u001B[39m     )\n\u001B[32m    973\u001B[39m log.debug(\u001B[33m\"\u001B[39m\u001B[33mRaising timeout error\u001B[39m\u001B[33m\"\u001B[39m)\n\u001B[32m    974\u001B[39m \u001B[38;5;28;01mraise\u001B[39;00m APITimeoutError(request=request) \u001B[38;5;28;01mfrom\u001B[39;00m\u001B[38;5;250m \u001B[39m\u001B[34;01merr\u001B[39;00m\n",
      "\u001B[36mFile \u001B[39m\u001B[32m~/opt/miniconda3/envs/Summer24/lib/python3.11/site-packages/openai/_base_client.py:1057\u001B[39m, in \u001B[36mSyncAPIClient._retry_request\u001B[39m\u001B[34m(self, options, cast_to, retries_taken, response_headers, stream, stream_cls)\u001B[39m\n\u001B[32m   1053\u001B[39m \u001B[38;5;66;03m# In a synchronous context we are blocking the entire thread. Up to the library user to run the client in a\u001B[39;00m\n\u001B[32m   1054\u001B[39m \u001B[38;5;66;03m# different thread if necessary.\u001B[39;00m\n\u001B[32m   1055\u001B[39m time.sleep(timeout)\n\u001B[32m-> \u001B[39m\u001B[32m1057\u001B[39m \u001B[38;5;28;01mreturn\u001B[39;00m \u001B[38;5;28mself\u001B[39m._request(\n\u001B[32m   1058\u001B[39m     options=options,\n\u001B[32m   1059\u001B[39m     cast_to=cast_to,\n\u001B[32m   1060\u001B[39m     retries_taken=retries_taken + \u001B[32m1\u001B[39m,\n\u001B[32m   1061\u001B[39m     stream=stream,\n\u001B[32m   1062\u001B[39m     stream_cls=stream_cls,\n\u001B[32m   1063\u001B[39m )\n",
      "\u001B[36mFile \u001B[39m\u001B[32m~/opt/miniconda3/envs/Summer24/lib/python3.11/site-packages/openai/_base_client.py:955\u001B[39m, in \u001B[36mSyncAPIClient._request\u001B[39m\u001B[34m(self, cast_to, options, retries_taken, stream, stream_cls)\u001B[39m\n\u001B[32m    952\u001B[39m log.debug(\u001B[33m\"\u001B[39m\u001B[33mSending HTTP Request: \u001B[39m\u001B[38;5;132;01m%s\u001B[39;00m\u001B[33m \u001B[39m\u001B[38;5;132;01m%s\u001B[39;00m\u001B[33m\"\u001B[39m, request.method, request.url)\n\u001B[32m    954\u001B[39m \u001B[38;5;28;01mtry\u001B[39;00m:\n\u001B[32m--> \u001B[39m\u001B[32m955\u001B[39m     response = \u001B[38;5;28mself\u001B[39m._client.send(\n\u001B[32m    956\u001B[39m         request,\n\u001B[32m    957\u001B[39m         stream=stream \u001B[38;5;129;01mor\u001B[39;00m \u001B[38;5;28mself\u001B[39m._should_stream_response_body(request=request),\n\u001B[32m    958\u001B[39m         **kwargs,\n\u001B[32m    959\u001B[39m     )\n\u001B[32m    960\u001B[39m \u001B[38;5;28;01mexcept\u001B[39;00m httpx.TimeoutException \u001B[38;5;28;01mas\u001B[39;00m err:\n\u001B[32m    961\u001B[39m     log.debug(\u001B[33m\"\u001B[39m\u001B[33mEncountered httpx.TimeoutException\u001B[39m\u001B[33m\"\u001B[39m, exc_info=\u001B[38;5;28;01mTrue\u001B[39;00m)\n",
      "\u001B[36mFile \u001B[39m\u001B[32m~/opt/miniconda3/envs/Summer24/lib/python3.11/site-packages/httpx/_client.py:914\u001B[39m, in \u001B[36mClient.send\u001B[39m\u001B[34m(self, request, stream, auth, follow_redirects)\u001B[39m\n\u001B[32m    906\u001B[39m follow_redirects = (\n\u001B[32m    907\u001B[39m     \u001B[38;5;28mself\u001B[39m.follow_redirects\n\u001B[32m    908\u001B[39m     \u001B[38;5;28;01mif\u001B[39;00m \u001B[38;5;28misinstance\u001B[39m(follow_redirects, UseClientDefault)\n\u001B[32m    909\u001B[39m     \u001B[38;5;28;01melse\u001B[39;00m follow_redirects\n\u001B[32m    910\u001B[39m )\n\u001B[32m    912\u001B[39m auth = \u001B[38;5;28mself\u001B[39m._build_request_auth(request, auth)\n\u001B[32m--> \u001B[39m\u001B[32m914\u001B[39m response = \u001B[38;5;28mself\u001B[39m._send_handling_auth(\n\u001B[32m    915\u001B[39m     request,\n\u001B[32m    916\u001B[39m     auth=auth,\n\u001B[32m    917\u001B[39m     follow_redirects=follow_redirects,\n\u001B[32m    918\u001B[39m     history=[],\n\u001B[32m    919\u001B[39m )\n\u001B[32m    920\u001B[39m \u001B[38;5;28;01mtry\u001B[39;00m:\n\u001B[32m    921\u001B[39m     \u001B[38;5;28;01mif\u001B[39;00m \u001B[38;5;129;01mnot\u001B[39;00m stream:\n",
      "\u001B[36mFile \u001B[39m\u001B[32m~/opt/miniconda3/envs/Summer24/lib/python3.11/site-packages/httpx/_client.py:942\u001B[39m, in \u001B[36mClient._send_handling_auth\u001B[39m\u001B[34m(self, request, auth, follow_redirects, history)\u001B[39m\n\u001B[32m    939\u001B[39m request = \u001B[38;5;28mnext\u001B[39m(auth_flow)\n\u001B[32m    941\u001B[39m \u001B[38;5;28;01mwhile\u001B[39;00m \u001B[38;5;28;01mTrue\u001B[39;00m:\n\u001B[32m--> \u001B[39m\u001B[32m942\u001B[39m     response = \u001B[38;5;28mself\u001B[39m._send_handling_redirects(\n\u001B[32m    943\u001B[39m         request,\n\u001B[32m    944\u001B[39m         follow_redirects=follow_redirects,\n\u001B[32m    945\u001B[39m         history=history,\n\u001B[32m    946\u001B[39m     )\n\u001B[32m    947\u001B[39m     \u001B[38;5;28;01mtry\u001B[39;00m:\n\u001B[32m    948\u001B[39m         \u001B[38;5;28;01mtry\u001B[39;00m:\n",
      "\u001B[36mFile \u001B[39m\u001B[32m~/opt/miniconda3/envs/Summer24/lib/python3.11/site-packages/httpx/_client.py:979\u001B[39m, in \u001B[36mClient._send_handling_redirects\u001B[39m\u001B[34m(self, request, follow_redirects, history)\u001B[39m\n\u001B[32m    976\u001B[39m \u001B[38;5;28;01mfor\u001B[39;00m hook \u001B[38;5;129;01min\u001B[39;00m \u001B[38;5;28mself\u001B[39m._event_hooks[\u001B[33m\"\u001B[39m\u001B[33mrequest\u001B[39m\u001B[33m\"\u001B[39m]:\n\u001B[32m    977\u001B[39m     hook(request)\n\u001B[32m--> \u001B[39m\u001B[32m979\u001B[39m response = \u001B[38;5;28mself\u001B[39m._send_single_request(request)\n\u001B[32m    980\u001B[39m \u001B[38;5;28;01mtry\u001B[39;00m:\n\u001B[32m    981\u001B[39m     \u001B[38;5;28;01mfor\u001B[39;00m hook \u001B[38;5;129;01min\u001B[39;00m \u001B[38;5;28mself\u001B[39m._event_hooks[\u001B[33m\"\u001B[39m\u001B[33mresponse\u001B[39m\u001B[33m\"\u001B[39m]:\n",
      "\u001B[36mFile \u001B[39m\u001B[32m~/opt/miniconda3/envs/Summer24/lib/python3.11/site-packages/httpx/_client.py:1015\u001B[39m, in \u001B[36mClient._send_single_request\u001B[39m\u001B[34m(self, request)\u001B[39m\n\u001B[32m   1010\u001B[39m     \u001B[38;5;28;01mraise\u001B[39;00m \u001B[38;5;167;01mRuntimeError\u001B[39;00m(\n\u001B[32m   1011\u001B[39m         \u001B[33m\"\u001B[39m\u001B[33mAttempted to send an async request with a sync Client instance.\u001B[39m\u001B[33m\"\u001B[39m\n\u001B[32m   1012\u001B[39m     )\n\u001B[32m   1014\u001B[39m \u001B[38;5;28;01mwith\u001B[39;00m request_context(request=request):\n\u001B[32m-> \u001B[39m\u001B[32m1015\u001B[39m     response = transport.handle_request(request)\n\u001B[32m   1017\u001B[39m \u001B[38;5;28;01massert\u001B[39;00m \u001B[38;5;28misinstance\u001B[39m(response.stream, SyncByteStream)\n\u001B[32m   1019\u001B[39m response.request = request\n",
      "\u001B[36mFile \u001B[39m\u001B[32m~/opt/miniconda3/envs/Summer24/lib/python3.11/site-packages/httpx/_transports/default.py:233\u001B[39m, in \u001B[36mHTTPTransport.handle_request\u001B[39m\u001B[34m(self, request)\u001B[39m\n\u001B[32m    220\u001B[39m req = httpcore.Request(\n\u001B[32m    221\u001B[39m     method=request.method,\n\u001B[32m    222\u001B[39m     url=httpcore.URL(\n\u001B[32m   (...)\u001B[39m\u001B[32m    230\u001B[39m     extensions=request.extensions,\n\u001B[32m    231\u001B[39m )\n\u001B[32m    232\u001B[39m \u001B[38;5;28;01mwith\u001B[39;00m map_httpcore_exceptions():\n\u001B[32m--> \u001B[39m\u001B[32m233\u001B[39m     resp = \u001B[38;5;28mself\u001B[39m._pool.handle_request(req)\n\u001B[32m    235\u001B[39m \u001B[38;5;28;01massert\u001B[39;00m \u001B[38;5;28misinstance\u001B[39m(resp.stream, typing.Iterable)\n\u001B[32m    237\u001B[39m \u001B[38;5;28;01mreturn\u001B[39;00m Response(\n\u001B[32m    238\u001B[39m     status_code=resp.status,\n\u001B[32m    239\u001B[39m     headers=resp.headers,\n\u001B[32m    240\u001B[39m     stream=ResponseStream(resp.stream),\n\u001B[32m    241\u001B[39m     extensions=resp.extensions,\n\u001B[32m    242\u001B[39m )\n",
      "\u001B[36mFile \u001B[39m\u001B[32m~/opt/miniconda3/envs/Summer24/lib/python3.11/site-packages/httpcore/_sync/connection_pool.py:216\u001B[39m, in \u001B[36mConnectionPool.handle_request\u001B[39m\u001B[34m(self, request)\u001B[39m\n\u001B[32m    213\u001B[39m         closing = \u001B[38;5;28mself\u001B[39m._assign_requests_to_connections()\n\u001B[32m    215\u001B[39m     \u001B[38;5;28mself\u001B[39m._close_connections(closing)\n\u001B[32m--> \u001B[39m\u001B[32m216\u001B[39m     \u001B[38;5;28;01mraise\u001B[39;00m exc \u001B[38;5;28;01mfrom\u001B[39;00m\u001B[38;5;250m \u001B[39m\u001B[38;5;28;01mNone\u001B[39;00m\n\u001B[32m    218\u001B[39m \u001B[38;5;66;03m# Return the response. Note that in this case we still have to manage\u001B[39;00m\n\u001B[32m    219\u001B[39m \u001B[38;5;66;03m# the point at which the response is closed.\u001B[39;00m\n\u001B[32m    220\u001B[39m \u001B[38;5;28;01massert\u001B[39;00m \u001B[38;5;28misinstance\u001B[39m(response.stream, Iterable)\n",
      "\u001B[36mFile \u001B[39m\u001B[32m~/opt/miniconda3/envs/Summer24/lib/python3.11/site-packages/httpcore/_sync/connection_pool.py:196\u001B[39m, in \u001B[36mConnectionPool.handle_request\u001B[39m\u001B[34m(self, request)\u001B[39m\n\u001B[32m    192\u001B[39m connection = pool_request.wait_for_connection(timeout=timeout)\n\u001B[32m    194\u001B[39m \u001B[38;5;28;01mtry\u001B[39;00m:\n\u001B[32m    195\u001B[39m     \u001B[38;5;66;03m# Send the request on the assigned connection.\u001B[39;00m\n\u001B[32m--> \u001B[39m\u001B[32m196\u001B[39m     response = connection.handle_request(\n\u001B[32m    197\u001B[39m         pool_request.request\n\u001B[32m    198\u001B[39m     )\n\u001B[32m    199\u001B[39m \u001B[38;5;28;01mexcept\u001B[39;00m ConnectionNotAvailable:\n\u001B[32m    200\u001B[39m     \u001B[38;5;66;03m# In some cases a connection may initially be available to\u001B[39;00m\n\u001B[32m    201\u001B[39m     \u001B[38;5;66;03m# handle a request, but then become unavailable.\u001B[39;00m\n\u001B[32m    202\u001B[39m     \u001B[38;5;66;03m#\u001B[39;00m\n\u001B[32m    203\u001B[39m     \u001B[38;5;66;03m# In this case we clear the connection and try again.\u001B[39;00m\n\u001B[32m    204\u001B[39m     pool_request.clear_connection()\n",
      "\u001B[36mFile \u001B[39m\u001B[32m~/opt/miniconda3/envs/Summer24/lib/python3.11/site-packages/httpcore/_sync/connection.py:101\u001B[39m, in \u001B[36mHTTPConnection.handle_request\u001B[39m\u001B[34m(self, request)\u001B[39m\n\u001B[32m     98\u001B[39m     \u001B[38;5;28mself\u001B[39m._connect_failed = \u001B[38;5;28;01mTrue\u001B[39;00m\n\u001B[32m     99\u001B[39m     \u001B[38;5;28;01mraise\u001B[39;00m exc\n\u001B[32m--> \u001B[39m\u001B[32m101\u001B[39m \u001B[38;5;28;01mreturn\u001B[39;00m \u001B[38;5;28mself\u001B[39m._connection.handle_request(request)\n",
      "\u001B[36mFile \u001B[39m\u001B[32m~/opt/miniconda3/envs/Summer24/lib/python3.11/site-packages/httpcore/_sync/http11.py:143\u001B[39m, in \u001B[36mHTTP11Connection.handle_request\u001B[39m\u001B[34m(self, request)\u001B[39m\n\u001B[32m    141\u001B[39m     \u001B[38;5;28;01mwith\u001B[39;00m Trace(\u001B[33m\"\u001B[39m\u001B[33mresponse_closed\u001B[39m\u001B[33m\"\u001B[39m, logger, request) \u001B[38;5;28;01mas\u001B[39;00m trace:\n\u001B[32m    142\u001B[39m         \u001B[38;5;28mself\u001B[39m._response_closed()\n\u001B[32m--> \u001B[39m\u001B[32m143\u001B[39m \u001B[38;5;28;01mraise\u001B[39;00m exc\n",
      "\u001B[36mFile \u001B[39m\u001B[32m~/opt/miniconda3/envs/Summer24/lib/python3.11/site-packages/httpcore/_sync/http11.py:113\u001B[39m, in \u001B[36mHTTP11Connection.handle_request\u001B[39m\u001B[34m(self, request)\u001B[39m\n\u001B[32m    102\u001B[39m     \u001B[38;5;28;01mpass\u001B[39;00m\n\u001B[32m    104\u001B[39m \u001B[38;5;28;01mwith\u001B[39;00m Trace(\n\u001B[32m    105\u001B[39m     \u001B[33m\"\u001B[39m\u001B[33mreceive_response_headers\u001B[39m\u001B[33m\"\u001B[39m, logger, request, kwargs\n\u001B[32m    106\u001B[39m ) \u001B[38;5;28;01mas\u001B[39;00m trace:\n\u001B[32m    107\u001B[39m     (\n\u001B[32m    108\u001B[39m         http_version,\n\u001B[32m    109\u001B[39m         status,\n\u001B[32m    110\u001B[39m         reason_phrase,\n\u001B[32m    111\u001B[39m         headers,\n\u001B[32m    112\u001B[39m         trailing_data,\n\u001B[32m--> \u001B[39m\u001B[32m113\u001B[39m     ) = \u001B[38;5;28mself\u001B[39m._receive_response_headers(**kwargs)\n\u001B[32m    114\u001B[39m     trace.return_value = (\n\u001B[32m    115\u001B[39m         http_version,\n\u001B[32m    116\u001B[39m         status,\n\u001B[32m    117\u001B[39m         reason_phrase,\n\u001B[32m    118\u001B[39m         headers,\n\u001B[32m    119\u001B[39m     )\n\u001B[32m    121\u001B[39m network_stream = \u001B[38;5;28mself\u001B[39m._network_stream\n",
      "\u001B[36mFile \u001B[39m\u001B[32m~/opt/miniconda3/envs/Summer24/lib/python3.11/site-packages/httpcore/_sync/http11.py:186\u001B[39m, in \u001B[36mHTTP11Connection._receive_response_headers\u001B[39m\u001B[34m(self, request)\u001B[39m\n\u001B[32m    183\u001B[39m timeout = timeouts.get(\u001B[33m\"\u001B[39m\u001B[33mread\u001B[39m\u001B[33m\"\u001B[39m, \u001B[38;5;28;01mNone\u001B[39;00m)\n\u001B[32m    185\u001B[39m \u001B[38;5;28;01mwhile\u001B[39;00m \u001B[38;5;28;01mTrue\u001B[39;00m:\n\u001B[32m--> \u001B[39m\u001B[32m186\u001B[39m     event = \u001B[38;5;28mself\u001B[39m._receive_event(timeout=timeout)\n\u001B[32m    187\u001B[39m     \u001B[38;5;28;01mif\u001B[39;00m \u001B[38;5;28misinstance\u001B[39m(event, h11.Response):\n\u001B[32m    188\u001B[39m         \u001B[38;5;28;01mbreak\u001B[39;00m\n",
      "\u001B[36mFile \u001B[39m\u001B[32m~/opt/miniconda3/envs/Summer24/lib/python3.11/site-packages/httpcore/_sync/http11.py:224\u001B[39m, in \u001B[36mHTTP11Connection._receive_event\u001B[39m\u001B[34m(self, timeout)\u001B[39m\n\u001B[32m    221\u001B[39m     event = \u001B[38;5;28mself\u001B[39m._h11_state.next_event()\n\u001B[32m    223\u001B[39m \u001B[38;5;28;01mif\u001B[39;00m event \u001B[38;5;129;01mis\u001B[39;00m h11.NEED_DATA:\n\u001B[32m--> \u001B[39m\u001B[32m224\u001B[39m     data = \u001B[38;5;28mself\u001B[39m._network_stream.read(\n\u001B[32m    225\u001B[39m         \u001B[38;5;28mself\u001B[39m.READ_NUM_BYTES, timeout=timeout\n\u001B[32m    226\u001B[39m     )\n\u001B[32m    228\u001B[39m     \u001B[38;5;66;03m# If we feed this case through h11 we'll raise an exception like:\u001B[39;00m\n\u001B[32m    229\u001B[39m     \u001B[38;5;66;03m#\u001B[39;00m\n\u001B[32m    230\u001B[39m     \u001B[38;5;66;03m#     httpcore.RemoteProtocolError: can't handle event type\u001B[39;00m\n\u001B[32m   (...)\u001B[39m\u001B[32m    234\u001B[39m     \u001B[38;5;66;03m# perspective. Instead we handle this case distinctly and treat\u001B[39;00m\n\u001B[32m    235\u001B[39m     \u001B[38;5;66;03m# it as a ConnectError.\u001B[39;00m\n\u001B[32m    236\u001B[39m     \u001B[38;5;28;01mif\u001B[39;00m data == \u001B[33mb\u001B[39m\u001B[33m\"\u001B[39m\u001B[33m\"\u001B[39m \u001B[38;5;129;01mand\u001B[39;00m \u001B[38;5;28mself\u001B[39m._h11_state.their_state == h11.SEND_RESPONSE:\n",
      "\u001B[36mFile \u001B[39m\u001B[32m~/opt/miniconda3/envs/Summer24/lib/python3.11/site-packages/httpcore/_backends/sync.py:126\u001B[39m, in \u001B[36mSyncStream.read\u001B[39m\u001B[34m(self, max_bytes, timeout)\u001B[39m\n\u001B[32m    124\u001B[39m \u001B[38;5;28;01mwith\u001B[39;00m map_exceptions(exc_map):\n\u001B[32m    125\u001B[39m     \u001B[38;5;28mself\u001B[39m._sock.settimeout(timeout)\n\u001B[32m--> \u001B[39m\u001B[32m126\u001B[39m     \u001B[38;5;28;01mreturn\u001B[39;00m \u001B[38;5;28mself\u001B[39m._sock.recv(max_bytes)\n",
      "\u001B[36mFile \u001B[39m\u001B[32m~/opt/miniconda3/envs/Summer24/lib/python3.11/ssl.py:1295\u001B[39m, in \u001B[36mSSLSocket.recv\u001B[39m\u001B[34m(self, buflen, flags)\u001B[39m\n\u001B[32m   1291\u001B[39m     \u001B[38;5;28;01mif\u001B[39;00m flags != \u001B[32m0\u001B[39m:\n\u001B[32m   1292\u001B[39m         \u001B[38;5;28;01mraise\u001B[39;00m \u001B[38;5;167;01mValueError\u001B[39;00m(\n\u001B[32m   1293\u001B[39m             \u001B[33m\"\u001B[39m\u001B[33mnon-zero flags not allowed in calls to recv() on \u001B[39m\u001B[38;5;132;01m%s\u001B[39;00m\u001B[33m\"\u001B[39m %\n\u001B[32m   1294\u001B[39m             \u001B[38;5;28mself\u001B[39m.\u001B[34m__class__\u001B[39m)\n\u001B[32m-> \u001B[39m\u001B[32m1295\u001B[39m     \u001B[38;5;28;01mreturn\u001B[39;00m \u001B[38;5;28mself\u001B[39m.read(buflen)\n\u001B[32m   1296\u001B[39m \u001B[38;5;28;01melse\u001B[39;00m:\n\u001B[32m   1297\u001B[39m     \u001B[38;5;28;01mreturn\u001B[39;00m \u001B[38;5;28msuper\u001B[39m().recv(buflen, flags)\n",
      "\u001B[36mFile \u001B[39m\u001B[32m~/opt/miniconda3/envs/Summer24/lib/python3.11/ssl.py:1168\u001B[39m, in \u001B[36mSSLSocket.read\u001B[39m\u001B[34m(self, len, buffer)\u001B[39m\n\u001B[32m   1166\u001B[39m         \u001B[38;5;28;01mreturn\u001B[39;00m \u001B[38;5;28mself\u001B[39m._sslobj.read(\u001B[38;5;28mlen\u001B[39m, buffer)\n\u001B[32m   1167\u001B[39m     \u001B[38;5;28;01melse\u001B[39;00m:\n\u001B[32m-> \u001B[39m\u001B[32m1168\u001B[39m         \u001B[38;5;28;01mreturn\u001B[39;00m \u001B[38;5;28mself\u001B[39m._sslobj.read(\u001B[38;5;28mlen\u001B[39m)\n\u001B[32m   1169\u001B[39m \u001B[38;5;28;01mexcept\u001B[39;00m SSLError \u001B[38;5;28;01mas\u001B[39;00m x:\n\u001B[32m   1170\u001B[39m     \u001B[38;5;28;01mif\u001B[39;00m x.args[\u001B[32m0\u001B[39m] == SSL_ERROR_EOF \u001B[38;5;129;01mand\u001B[39;00m \u001B[38;5;28mself\u001B[39m.suppress_ragged_eofs:\n",
      "\u001B[31mKeyboardInterrupt\u001B[39m: "
     ]
    }
   ],
   "execution_count": 7
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-08-29T20:44:48.332835Z",
     "start_time": "2025-08-29T20:44:48.287605Z"
    }
   },
   "cell_type": "code",
   "source": "combined_df",
   "id": "9fd1be22241ced6b",
   "outputs": [
    {
     "ename": "NameError",
     "evalue": "name 'combined_df' is not defined",
     "output_type": "error",
     "traceback": [
      "\u001B[31m---------------------------------------------------------------------------\u001B[39m",
      "\u001B[31mNameError\u001B[39m                                 Traceback (most recent call last)",
      "\u001B[36mCell\u001B[39m\u001B[36m \u001B[39m\u001B[32mIn[8]\u001B[39m\u001B[32m, line 1\u001B[39m\n\u001B[32m----> \u001B[39m\u001B[32m1\u001B[39m combined_df\n",
      "\u001B[31mNameError\u001B[39m: name 'combined_df' is not defined"
     ]
    }
   ],
   "execution_count": 8
  },
  {
   "metadata": {},
   "cell_type": "code",
   "source": "ground_truth = pd.read_parquet(\"exp_input/dataset_citation_records_Table.parquet\")",
   "id": "305d7d5f299551c1",
   "outputs": [],
   "execution_count": null
  },
  {
   "metadata": {},
   "cell_type": "code",
   "source": "ground_truth.head()",
   "id": "dadaf8654a9d103d",
   "outputs": [],
   "execution_count": null
  },
  {
   "metadata": {},
   "cell_type": "code",
   "source": [
    "def evaluate_performance(predict_df, ground_truth, orchestrator, false_positives_file):\n",
    "    \"\"\" Evaluates dataset extraction performance using precision, recall, and F1-score. \"\"\"\n",
    "\n",
    "    recall_list, false_positives_output = [], []\n",
    "    total_precision, total_recall, num_sources = 0, 0, 0\n",
    "\n",
    "    for source_page in predict_df['source_url'].unique():\n",
    "        pub_id = source_page.split('/')[-1].lower()\n",
    "        \n",
    "        orchestrator.logger.info(f\"Evaluating pub_id: {pub_id}\")\n",
    "        gt_data = ground_truth[ground_truth['pmcid'].str.lower() == pub_id.lower()]  # extract ground truth\n",
    "\n",
    "        gt_datasets = set()\n",
    "        for dataset_string in gt_data['identifier'].dropna().str.lower():\n",
    "            gt_datasets.update(dataset_string.split(','))  # Convert CSV string into set of IDs\n",
    "\n",
    "        orchestrator.logger.info(f\"# of elements in gt_data: {len(gt_data)}. Element IDs: {gt_datasets}\")\n",
    "\n",
    "        num_sources += 1\n",
    "\n",
    "        # Extract evaluation datasets for this source page\n",
    "        eval_data = predict_df[predict_df['source_url'] == source_page]\n",
    "        eval_datasets = set(eval_data['dataset_identifier'].dropna().str.lower())\n",
    "        # Remove invalid entries\n",
    "        eval_datasets.discard('n/a')\n",
    "        eval_datasets.discard('')\n",
    "\n",
    "        orchestrator.logger.info(f\"Evaluation datasets: {eval_datasets}\")\n",
    "\n",
    "        # Handle cases where both ground truth and evaluation are empty\n",
    "        if not gt_datasets and not eval_datasets:\n",
    "            orchestrator.logger.info(\"No datasets in both ground truth and evaluation. Perfect precision and recall.\")\n",
    "            total_precision += 1\n",
    "            total_recall += 1\n",
    "            continue\n",
    "\n",
    "        # Match Extraction Logic\n",
    "        matched_gt, matched_eval = set(), set()\n",
    "\n",
    "        # Exact Matches\n",
    "        exact_matches = gt_datasets & eval_datasets  # Intersection of ground truth and extracted datasets\n",
    "        matched_gt.update(exact_matches)\n",
    "        matched_eval.update(exact_matches)\n",
    "\n",
    "        # Partial Matches (Aliased Identifiers)\n",
    "        for eval_id in eval_datasets - matched_eval:\n",
    "            for gt_id in gt_datasets - matched_gt:\n",
    "                if eval_id in gt_id or gt_id in eval_id:  # Partial match or alias\n",
    "                    orchestrator.logger.info(f\"Partial or alias match found: eval_id={eval_id}, gt_id={gt_id}\")\n",
    "                    matched_gt.add(gt_id)\n",
    "                    matched_eval.add(eval_id)\n",
    "                    break  # Stop once matched\n",
    "\n",
    "        # **False Positives (Unmatched extracted datasets)**\n",
    "        FP = eval_datasets - matched_eval\n",
    "        false_positives_output.extend(FP)\n",
    "\n",
    "        # **False Negatives (Unmatched ground truth datasets)**\n",
    "        FN = gt_datasets - matched_gt\n",
    "\n",
    "        # **Precision and Recall Calculation**\n",
    "        true_positives = len(matched_gt)\n",
    "        false_positives = len(FP)\n",
    "        false_negatives = len(FN)\n",
    "\n",
    "        precision = true_positives / (true_positives + false_positives) if (true_positives + false_positives) > 0 else 0\n",
    "        recall = true_positives / (true_positives + false_negatives) if (true_positives + false_negatives) > 0 else 0\n",
    "\n",
    "        orchestrator.logger.info(f\"Precision for {source_page}: {precision}\")\n",
    "        orchestrator.logger.info(f\"Recall for {source_page}: {recall}\")\n",
    "\n",
    "        if recall == 0:\n",
    "            recall_list.append(source_page)\n",
    "\n",
    "        # Accumulate totals\n",
    "        total_precision += precision\n",
    "        total_recall += recall\n",
    "\n",
    "    # **Compute Overall Metrics**\n",
    "    average_precision = total_precision / num_sources if num_sources > 0 else 0\n",
    "    average_recall = total_recall / num_sources if num_sources > 0 else 0\n",
    "    f1_score = (\n",
    "        2 * (average_precision * average_recall) / (average_precision + average_recall)\n",
    "        if (average_precision + average_recall) > 0\n",
    "        else 0\n",
    "    )\n",
    "\n",
    "    orchestrator.logger.info(f\"\\nPerformance evaluation completed for {num_sources} source pages.\")\n",
    "\n",
    "    # **Save false positives**\n",
    "    with open(false_positives_file, 'w') as f:\n",
    "        for item in false_positives_output:\n",
    "            f.write(\"%s\\n\" % item)\n",
    "\n",
    "    return {\n",
    "        \"average_precision\": average_precision,\n",
    "        \"average_recall\": average_recall,\n",
    "        \"f1_score\": f1_score\n",
    "    }"
   ],
   "id": "583f880224783dc7",
   "outputs": [],
   "execution_count": null
  },
  {
   "metadata": {},
   "cell_type": "code",
   "source": "#results = evaluate_performance(out, ground_truth, dg, \"output/false_positives.txt\")",
   "id": "d5bd9405ee5dcd60",
   "outputs": [],
   "execution_count": null
  },
  {
   "metadata": {},
   "cell_type": "code",
   "source": "results = evaluate_performance(combined_df, ground_truth, dg, \"output/false_positives.txt\")",
   "id": "3ae367545bb1b42b",
   "outputs": [],
   "execution_count": null
  },
  {
   "metadata": {},
   "cell_type": "code",
   "source": "results",
   "id": "fff3dc3075cb13b5",
   "outputs": [],
   "execution_count": null
  },
  {
   "metadata": {},
   "cell_type": "code",
   "source": "\"https://pmc.ncbi.nlm.nih.gov/articles/PMC11240079\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC5752539\",\"https://pmc.ncbi.nlm.nih.gov/articles/PMC7032692\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10547713\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8102856\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8055881\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8226229\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10769298\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC11659981\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8082263\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8565335\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC9094742\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC6289083\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC7658217\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC9780309\",\"https://pmc.ncbi.nlm.nih.gov/articles/PMC3788619\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10680627\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8131595\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC11420198\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC9915613\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10238095\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452\" , \"https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10836119\",  \"https://pmc.ncbi.nlm.nih.gov/articles/PMC9280291\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC11661334\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC5161470\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277\"",
   "id": "1869e71b9d253e7d",
   "outputs": [],
   "execution_count": null
  },
  {
   "metadata": {},
   "cell_type": "code",
   "source": "combined_df.to_csv(\"output/SAGE_GPT_FDR_FewShot_Synapse_output.csv\", index=False)",
   "id": "88837a62c359df60",
   "outputs": [],
   "execution_count": null
  },
  {
   "metadata": {},
   "cell_type": "code",
   "source": "",
   "id": "54baf5d5dcfee95e",
   "outputs": [],
   "execution_count": null
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 2
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython2",
   "version": "2.7.6"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
