{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "initial_id",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-09-04T16:30:53.289617Z",
     "start_time": "2025-09-04T16:30:29.500758Z"
    },
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "import os\n",
    "from data_gatherer.data_gatherer import DataGatherer\n",
    "import pandas as pd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "78d2b2bc4106d2ef",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-09-04T16:31:15.898169Z",
     "start_time": "2025-09-04T16:31:15.880961Z"
    }
   },
   "outputs": [],
   "source": [
    "df = pd.read_csv(\"scripts/exp_input/SAGE_groundtruth.csv\")\n",
    "input_file = \"scripts/exp_input/SAGE_input_2.txt\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "cca05a29b7c100d1",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-09-04T16:31:17.295747Z",
     "start_time": "2025-09-04T16:31:17.291744Z"
    }
   },
   "outputs": [],
   "source": [
    "model_name = \"gemini-2.0-flash\"  # \"gemini-2.0-flash\" or \"gpt-4o-mini\"\n",
    "prompt = \"GPT_FDR_FewShot_Syn\"  # \"GPT_from_full_input_Examples\" or \"retrieve_datasets_simple_JSON\"\n",
    "FDR = True\n",
    "semantic_retrieval = False\n",
    "section_filter= None"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "a4ae25cbd54d4fe1",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-09-04T16:31:19.865917Z",
     "start_time": "2025-09-04T16:31:19.860295Z"
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of PMCIDs: 1\n"
     ]
    }
   ],
   "source": [
    "# write list to a text file\n",
    "with open(input_file, 'r') as f:\n",
    "    pmcids = f.read().splitlines()\n",
    "\n",
    "print(\"Number of PMCIDs:\", len(pmcids))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "b79cbd7265b497e1",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-09-04T16:31:22.492692Z",
     "start_time": "2025-09-04T16:31:22.479800Z"
    }
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97mdata_gatherer.py - line 277 - INFO - Setting up data fetcher...\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 26 - DEBUG - DataFetcher initialized.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 277 - DEBUG - WebScraper initialized.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 307 - INFO - Data fetcher setup completed.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 675 - DEBUG - DataCompletenessChecker initialized.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 109 - INFO - DataGatherer orchestrator initialized. Extraction Model: gemini-2.0-flash\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "dg = DataGatherer(\n",
    "    llm_name=model_name, \n",
    "    log_level='DEBUG', \n",
    "    process_entire_document=FDR, \n",
    "    driver_path=None, \n",
    "    save_to_cache=False, \n",
    "    load_from_cache=False,\n",
    "    full_output_file=\"scripts/output/result.csv\"\n",
    ") #, save_dynamic_prompts=True"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "5e0e0500474a6c34",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-09-04T16:31:25.329784Z",
     "start_time": "2025-09-04T16:31:25.325226Z"
    }
   },
   "outputs": [],
   "source": [
    "# These are the problematic urls for information extraction for GEMINI\n",
    "#urls = [\"https://pmc.ncbi.nlm.nih.gov/articles/PMC9314356\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527/\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC11929800/\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436/\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452/\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8565335/\",\"https://pmc.ncbi.nlm.nih.gov/articles/PMC7778917/\"]\n",
    "# v1 = \"https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360/\"\n",
    "# v2 = \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8006362\"\n",
    "#urls = [\"https://pmc.ncbi.nlm.nih.gov/articles/PMC9710693/\"]\n",
    "\n",
    "# These are the problematic urls for information extraction for GPT - (2025 Aug 29) GPT gets stuck for some reason after 50 calls or so...\n",
    "#urls = [\"https://pmc.ncbi.nlm.nih.gov/articles/PMC11240079\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC5752539\",\"https://pmc.ncbi.nlm.nih.gov/articles/PMC7032692\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10547713\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8102856\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8055881\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8226229\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10769298\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC11659981\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8082263\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8565335\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC9094742\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC6289083\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC7658217\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC9780309\",\"https://pmc.ncbi.nlm.nih.gov/articles/PMC3788619\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10680627\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8131595\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC11420198\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC9915613\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10238095\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452\" , \"https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10836119\",  \"https://pmc.ncbi.nlm.nih.gov/articles/PMC9280291\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC11661334\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC5161470\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277\"]\n",
    "\n",
    "#out = dg.process_articles(urls, semantic_retrieval=semantic_retrieval, section_filter=section_filter, prompt_name=prompt)\n",
    "\n",
    "#out\n",
    "#out['https://pmc.ncbi.nlm.nih.gov/articles/PMC9710693']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "bfd0572331e32fe2",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-09-04T17:36:22.343172Z",
     "start_time": "2025-09-04T16:31:29.415907Z"
    }
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97mdata_gatherer.py - line 1027 - DEBUG - DataGatherer run started.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 635 - DEBUG - Loading URLs from file: scripts/exp_input/SAGE_input_2.txt\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 644 - INFO - Loaded 1 URLs from file.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 552 - DEBUG - Starting to process URL list...\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 559 - INFO - 0th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 362 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 50 - DEBUG - URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 158 - DEBUG - update_DataFetcher_settings for current URL\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 217 - DEBUG - Checking PMC with pattern https://www.ncbi.nlm.nih.gov/pmc/articles/\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 217 - DEBUG - Checking PMC with pattern https://pmc.ncbi.nlm.nih.gov/\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 220 - DEBUG - URL detected as PMC.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 26 - DEBUG - DataFetcher initialized.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 521 - DEBUG - EntrezFetcher initialized.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 369 - INFO - Type of data_fetcher EntrezFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 380 - DEBUG - Fetching Raw content...\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 538 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC4318527&retmode=xml\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 546 - DEBUG - Successfully fetched data for PMC4318527\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 55 - DEBUG - Checking XML completeness for https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 444 - DEBUG - type of raw_data: <class 'lxml.etree._Element'>, raw_data: <Element pmc-articleset at 0x15714fa80>\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 69 - INFO - Loading target sections for section name: data_availability_sections\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 448 - DEBUG - Section patterns: [\".//sec[@sec-type='data-availability']\", \".//notes[@notes-type='data-availability']\", \".//notes[@notes-type='data-availability']\"]\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 450 - DEBUG - Namespaces: {'xlink': 'http://www.w3.org/1999/xlink', 'ali': 'http://www.niso.org/schemas/ali/1.0/'}\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 453 - DEBUG - Checking pattern: .//sec[@sec-type='data-availability']\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 453 - DEBUG - Checking pattern: .//notes[@notes-type='data-availability']\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 453 - DEBUG - Checking pattern: .//notes[@notes-type='data-availability']\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 398 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 158 - DEBUG - update_DataFetcher_settings for current URL\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 232 - DEBUG - Checking if URL is a PDF: https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver None, Firefox, True.\u001b[0m\n",
      "\u001b[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\u001b[0m\n",
      "\u001b[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001b[0m\n",
      "\u001b[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x155733e30>.\u001b[0m\n",
      "\u001b[97mselenium_setup.py - line 46 - INFO - Firefox WebDriver created successfully.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 26 - DEBUG - DataFetcher initialized.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 277 - DEBUG - WebScraper initialized.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 294 - DEBUG - Fetching data with function call: self.scraper_tool.get(url)\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 296 - DEBUG - http get complete, now waiting 2 seconds for page to load\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 370 - DEBUG - Extracting publication title from page source\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 372 - DEBUG - Publication name pointer: <selenium.webdriver.remote.webelement.WebElement (session=\"21089d36-caf0-49c4-96a7-05419468b4d5\", element=\"c7f6c469-4536-4791-8ad8-3ba8e3565250\")>\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 380 - WARNING - Publication name not found in the page title.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 313 - INFO - No cookie pattern 1 found in HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 414 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 428 - INFO - Skipping raw HTML/XML saving.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 459 - INFO - WebScraper driver quit.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 458 - INFO - Using HTMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 52 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 985 - DEBUG - All repository names: ['BIGD', 'European Genome-Phenome Archive', 'MassIVE', 'PRIDE', 'iProX', 'ProteomeCentral', 'ProteomeXchange', 'Proteomic Data Commons', 'Genomic Data Commons', 'The Cancer Imaging Archive', 'Imaging Data Commons', 'JPOST', 'Gene Expression Omnibus', 'database of Genotypes and Phenotypes', 'Sequence Read Archive', 'ArrayExpress', 'Nucleotide', 'Mendeley', 'Synapse', 'doi.org', 'Protein Data Bank']\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 998 - DEBUG - Repo mapping: {'bigd': 'bigd.big.ac.cn', 'european genome-phenome archive': 'ega-archive.org', 'massive': 'massive.ucsd.edu', 'pride': 'www.ebi.ac.uk', 'iprox': 'www.iprox.cn', 'proteomecentral': 'proteomecentral.proteomexchange.org', 'proteomexchange': 'www.proteomexchange.org', 'proteomic data commons': 'pdc.cancer.gov', 'genomic data commons': 'gdc.cancer.gov', 'the cancer imaging archive': 'cancerimagingarchive.net', 'imaging data commons': 'imaging.datacommons.cancer.gov', 'jpost': 'repository.jpostdb.org', 'gene expression omnibus': 'geo', 'database of genotypes and phenotypes': 'dbgap', 'sequence read archive': 'sra', 'arrayexpress': 'arrayexpress', 'nucleotide': 'nuccore', 'mendeley': 'data.mendeley.com', 'synapse': 'synapse.org', 'doi.org': 'doi.org', 'protein data bank': 'pdb'}\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 82 - INFO - Initializing htmlRetriever\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 350 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527, additional_data, HTML)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 448 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 456 - DEBUG - Found 126 anchors with data-ga-action='click_feat_suppl'.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 165 - DEBUG - Function_call: reconstruct_download_link(https://doi.org/10.1186/s13059-014-0462-7, pmc-layout__citation font-secondary font-xs, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 168 - DEBUG - Inputs to reconstruct_download_link: https://doi.org/10.1186/s13059-014-0462-7, pmc-layout__citation font-secondary font-xs, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527, 4318527\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 165 - DEBUG - Function_call: reconstruct_download_link(http://creativecommons.org/licenses/by/4.0, d-panel p, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 168 - DEBUG - Inputs to reconstruct_download_link: http://creativecommons.org/licenses/by/4.0, d-panel p, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527, 4318527\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 165 - DEBUG - Function_call: reconstruct_download_link(http://creativecommons.org/publicdomain/zero/1.0/, d-panel p, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 168 - DEBUG - Inputs to reconstruct_download_link: http://creativecommons.org/publicdomain/zero/1.0/, d-panel p, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527, 4318527\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 165 - DEBUG - Function_call: reconstruct_download_link(http://assemblathon.org/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 168 - DEBUG - Inputs to reconstruct_download_link: http://assemblathon.org/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527, 4318527\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 165 - DEBUG - Function_call: reconstruct_download_link(http://biofunctionprediction.org/node/8, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 168 - DEBUG - Inputs to reconstruct_download_link: http://biofunctionprediction.org/node/8, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527, 4318527\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 165 - DEBUG - Function_call: reconstruct_download_link(http://genomeinterpretation.org/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 168 - DEBUG - Inputs to reconstruct_download_link: http://genomeinterpretation.org/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527, 4318527\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 165 - DEBUG - Function_call: reconstruct_download_link(http://www.ebi.ac.uk/msd-srv/capri/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 168 - DEBUG - Inputs to reconstruct_download_link: http://www.ebi.ac.uk/msd-srv/capri/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527, 4318527\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 165 - DEBUG - Function_call: reconstruct_download_link(http://predictioncenter.org/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 168 - DEBUG - Inputs to reconstruct_download_link: http://predictioncenter.org/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527, 4318527\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 165 - DEBUG - Function_call: reconstruct_download_link(http://www.chalearn.org/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 168 - DEBUG - Inputs to reconstruct_download_link: http://www.chalearn.org/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527, 4318527\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 165 - DEBUG - Function_call: reconstruct_download_link(http://www.childrenshospital.org/research-and-innovation/research-initiatives/clarity-challenge, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 168 - DEBUG - Inputs to reconstruct_download_link: http://www.childrenshospital.org/research-and-innovation/research-initiatives/clarity-challenge, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527, 4318527\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 165 - DEBUG - Function_call: reconstruct_download_link(https://www.synapse.org/#!Challenges:DREAM, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 168 - DEBUG - Inputs to reconstruct_download_link: https://www.synapse.org/#!Challenges:DREAM, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527, 4318527\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 165 - DEBUG - Function_call: reconstruct_download_link(http://flowcap.flowsite.org/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 168 - DEBUG - Inputs to reconstruct_download_link: http://flowcap.flowsite.org/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527, 4318527\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 165 - DEBUG - Function_call: reconstruct_download_link(https://www.synapse.org/#!Synapse:syn312572, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 168 - DEBUG - Inputs to reconstruct_download_link: https://www.synapse.org/#!Synapse:syn312572, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527, 4318527\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 165 - DEBUG - Function_call: reconstruct_download_link(https://sbvimprover.com/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 168 - DEBUG - Inputs to reconstruct_download_link: https://sbvimprover.com/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527, 4318527\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 165 - DEBUG - Function_call: reconstruct_download_link(http://www.innocentive.com/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 168 - DEBUG - Inputs to reconstruct_download_link: http://www.innocentive.com/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527, 4318527\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 165 - DEBUG - Function_call: reconstruct_download_link(http://www.kaggle.com/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 168 - DEBUG - Inputs to reconstruct_download_link: http://www.kaggle.com/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527, 4318527\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 165 - DEBUG - Function_call: reconstruct_download_link(http://www.gencodegenes.org/rgasp/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 168 - DEBUG - Inputs to reconstruct_download_link: http://www.gencodegenes.org/rgasp/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527, 4318527\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 165 - DEBUG - Function_call: reconstruct_download_link(http://sequencesqueeze.org/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 168 - DEBUG - Inputs to reconstruct_download_link: http://sequencesqueeze.org/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527, 4318527\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 165 - DEBUG - Function_call: reconstruct_download_link(http://www.xprize.org/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 168 - DEBUG - Inputs to reconstruct_download_link: http://www.xprize.org/, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527, 4318527\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 165 - DEBUG - Function_call: reconstruct_download_link(http://www.sagebase.org/synapse, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 168 - DEBUG - Inputs to reconstruct_download_link: http://www.sagebase.org/synapse, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527, 4318527\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 165 - DEBUG - Function_call: reconstruct_download_link(http://genomicsandhealth.org, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 168 - DEBUG - Inputs to reconstruct_download_link: http://genomicsandhealth.org, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527, 4318527\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 165 - DEBUG - Function_call: reconstruct_download_link(https://www.synapse.org/#!Synapse:syn2455683, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 168 - DEBUG - Inputs to reconstruct_download_link: https://www.synapse.org/#!Synapse:syn2455683, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527, 4318527\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 165 - DEBUG - Function_call: reconstruct_download_link(https://www.synapse.org/#!Synapse:syn1720047, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 168 - DEBUG - Inputs to reconstruct_download_link: https://www.synapse.org/#!Synapse:syn1720047, n/a, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527, 4318527\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 527 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 550 - INFO - Extracted 23 unique supplementary material links from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 560 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#clp_a'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for clp_a: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#clp_a'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for clp_a: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 5 <a> elements with href='#Tab1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 5 <a> elements with href='#Tab1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 5 <a> elements with href='#Tab1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 5 <a> elements with href='#Tab1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 5 <a> elements with href='#Tab1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 5 <a> elements with href='#Tab1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 5 <a> elements with href='#Tab1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 5 <a> elements with href='#Tab1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 5 <a> elements with href='#Tab1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 5 <a> elements with href='#Tab1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 5 <a> elements with href='#Tab1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 5 <a> elements with href='#Tab1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 5 <a> elements with href='#Tab1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 5 <a> elements with href='#Tab1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 5 <a> elements with href='#Tab1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 5 <a> elements with href='#Tab1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#Bib1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for Bib1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#Bib1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for Bib1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#Bib1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for Bib1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#Bib1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for Bib1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 365 - DEBUG - Preprocessed data: <html class=\"\" lang=\"en\"> <head> <link href=\"/static/assets/style-8258669d.css\" rel=\"stylesheet\"/> <link href=\"/static/assets/style-ef962842.css\" rel=\"stylesheet\"/> <link href=\"/static/assets/style-3ade8b5c.css\" rel=\"stylesheet\"/> <link href=\"/static/img/favicons/apple-touch-icon.png\" rel=\"apple-touch-icon\" sizes=\"180x180\"/> <link href=\"/static/img/favicons/favicon-48x48.png\" rel=\"icon\" sizes=\"48x48\" type=\"image/png\"/> <link href=\"/static/img/favicons/favicon-32x32.png\" rel=\"icon\" sizes=\"32x32\" type=\"image/png\"/> <link href=\"/static/img/favicons/favicon-16x16.png\" rel=\"icon\" sizes=\"16x16\" type=\"image/png\"/> <link href=\"/static/img/favicons/site.webmanifest\" rel=\"manifest\"/> <link color=\"#0071bc\" href=\"/static/img/favicons/safari-pinned-tab.svg\" rel=\"mask-icon\"/> <title> Toward better benchmarking: challenge-based methods assessment in cancer genomics - PMC </title> <link href=\"https://www.google-analytics.com\" rel=\"preconnect\"/> <link href=\"https://cdn.ncbi.nlm.nih.gov\" rel=\"preconnect\"/> <link href=\"https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527/\" rel=\"canonical\"/> </head> <body> <a class=\"usa-skipnav\" href=\"#main-content\"> Skip to main content </a> <section aria-label=\"Official website of the United States government\" class=\"usa-banner\"> <div class=\"usa-accordion\"> <header class=\"usa-banner__header\"> <div class=\"usa-banner__inner\"> <div class=\"grid-col-auto\"> </div> <div aria-hidden=\"true\" class=\"grid-col-fill tablet:grid-col-auto\"> <p class=\"usa-banner__header-text\"> An official website of the United States government </p> <span class=\"usa-banner__header-action\"> Here's how you know </span> </div> </div> </header> <div class=\"usa-banner__content usa-accordion__content\" hidden=\"\" id=\"gov-banner-default\"> <div class=\"grid-row grid-gap-lg\"> <div class=\"usa-banner__guidance tablet:grid-col-6\"> <div class=\"usa-media-block__body\"> <p> <strong> Official websites use .gov </strong> <br/> A <strong> .gov </strong> website belongs to an official government organization in the United States. </p> </div> </div> <div class=\"usa-banner__guidance tablet:grid-col-6\"> <div class=\"usa-media-block__body\"> <p> <strong> Secure .gov websites use HTTPS </strong> <br/> A <strong> lock </strong> ( <span class=\"icon-lock\"> </span> ) or <strong> https:// </strong> means you've safely connected to the .gov website. Share sensitive information only on official, secure websites. </p> </div> </div> </div> </div> </div> </section> <div class=\"usa-overlay\"> </div> <header class=\"usa-header usa-header--extended usa-header--wide\"> <div class=\"ncbi-header\"> <div class=\"ncbi-header__container\"> <a class=\"ncbi-header__logo-container\" href=\"https://www.ncbi.nlm.nih.gov/\"> </a> <div class=\"ncbi-header__desktop-buttons\"> <div class=\"ncbi-header__login-dropdown\"> <ul class=\"usa-nav__submenu ncbi-header__login-dropdown-menu\" hidden=\"\" id=\"login-dropdown-menu\"> <li class=\"usa-nav__submenu-item\"> <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/myncbi/\"> Dashboard </a> </li> <li class=\"usa-nav__submenu-item\"> <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/\"> Publications </a> </li> <li class=\"usa-nav__submenu-item\"> <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/account/settings/\"> Account settings </a> </li> <li class=\"usa-nav__submenu-item\"> </li> </ul> </div> </div> </div> </div> <div class=\"ncbi-search-panel ncbi--show-only-at-desktop\" hidden=\"\"> <div class=\"ncbi-search-panel__container\"> </div> </div> <nav aria-label=\"Primary navigation\" class=\"usa-nav\"> <p class=\"usa-sr-only\" id=\"primary-navigation-sr-only-title\"> Primary site navigation </p> <div class=\"ncbi-nav__mobile-login-menu ncbi--hide-at-desktop\" hidden=\"\"> <p class=\"ncbi-nav__mobile-login-menu-status\"> Logged in as: <strong class=\"ncbi-nav__mobile-login-menu-email\"> </strong> </p> <ul class=\"usa-nav__primary usa-accordion\"> <li class=\"usa-nav__primary-item\"> <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/myncbi/\"> Dashboard </a> </li> <li class=\"usa-nav__primary-item\"> <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/\"> Publications </a> </li> <li class=\"usa-nav__primary-item\"> <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/account/settings/\"> Account settings </a> </li> </ul> </div> </nav> </header> <section aria-label=\"PMC Header with search box\" class=\"pmc-header pmc-header--basic\"> <div class=\"pmc-nav-container\"> <div class=\"pmc-header__bar\"> <div class=\"pmc-header__logo\"> <a aria-label=\"PMC Home\" href=\"/\" title=\"Home\"> </a> </div> </div> <div class=\"pmc-header__search\"> <div class=\"display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44\"> <ul class=\"pmc-header__search__menu\"> <li> <a class=\"usa-link\" href=\"/journals/\"> Journal List </a> </li> <li> <a class=\"usa-link\" href=\"/about/userguide/\"> User Guide </a> </li> </ul> </div> </div> </div> </section> <div class=\"usa-section padding-top-0 desktop:padding-top-6 pmc-article-section\"> <div aria-label=\"Actions bar\" class=\"grid-container pmc-actions-bar is-not-intersecting\" role=\"complementary\"> <div class=\"grid-row\"> <div class=\"grid-col-fill display-flex\"> <div class=\"display-flex\"> <ul class=\"usa-list usa-list--unstyled usa-list--horizontal\"> <li class=\"margin-right-2 mobile-lg:margin-right-4 display-flex mob\"> </li> <li class=\"margin-right-2 mobile-lg:margin-right-4 display-flex mob\"> <span class=\"usa-tooltip\"> <a aria-label=\"View on publisher site\" class=\"usa-link display-flex usa-tooltip__trigger\" href=\"https://doi.org/10.1186/s13059-014-0462-7\" rel=\"noreferrer noopener\" role=\"button\" tabindex=\"0\" target=\"_blank\"> </a> <span aria-hidden=\"true\" class=\"usa-tooltip__body\" role=\"tooltip\"> View on publisher site </span> </span> </li> <li class=\"margin-right-2 mobile-lg:margin-right-4 display-flex\"> <span class=\"usa-tooltip\"> <a aria-label=\"Download PDF\" class=\"usa-link display-flex usa-tooltip__trigger\" href=\"pdf/13059_2014_Article_462.pdf\" role=\"button\" tabindex=\"0\"> </a> <span aria-hidden=\"true\" class=\"usa-tooltip__body\" role=\"tooltip\"> Download PDF </span> </span> </li> <li class=\"margin-right-2 mobile-lg:margin-right-4 display-flex\"> <span class=\"usa-tooltip\"> <span aria-hidden=\"true\" class=\"usa-tooltip__body\" role=\"tooltip\"> Add to Collections </span> </span> </li> <li class=\"margin-right-2 mobile-lg:margin-right-4 display-flex\"> <span class=\"usa-tooltip\"> <span aria-hidden=\"true\" class=\"usa-tooltip__body\" role=\"tooltip\"> Cite </span> </span> </li> <li class=\"pmc-permalink display-flex\"> <span class=\"usa-tooltip\"> <span aria-hidden=\"true\" class=\"usa-tooltip__body\" role=\"tooltip\"> Permalink </span> </span> <div class=\"pmc-permalink__dropdown\" hidden=\"\"> <div class=\"pmc-permalink__dropdown__container\"> <h2 class=\"usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs\"> PERMALINK </h2> <div class=\"pmc-permalink__dropdown__content\"> </div> </div> </div> </li> </ul> </div> </div> </div> </div> <div class=\"grid-container desktop:padding-left-6\"> <div class=\"grid-row grid-gap\" id=\"article-container\"> <div class=\"grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content\"> <div class=\"grid-container padding-left-0 padding-right-0\"> <div class=\"grid-row desktop:margin-left-neg-6\"> <div class=\"grid-col-12\"> <div aria-label=\"Disclaimer note\" class=\"pmc-layout__disclaimer\" role=\"complementary\"> As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with, the contents by NLM or the National Institutes of Health. <br/> Learn more: <a class=\"usa-link\" href=\"/about/disclaimer/\"> PMC Disclaimer </a> | <a class=\"usa-link\" href=\"/about/copyright/\"> PMC Copyright Notice </a> </div> </div> </div> <div class=\"grid-row pmc-wm desktop:margin-left-neg-6\"> <main class=\"usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0\" id=\"main-content\"> <section aria-label=\"Journal banner\" class=\"pmc-journal-banner text-center line-height-none\"> <map name=\"pmc-banner-imagemap\"> <area alt=\"Link to Genome Biology\" href=\"https://genomebiology.biomedcentral.com/\" rel=\"noopener noreferrer\" shape=\"default\" target=\"_blank\" title=\"Link to Genome Biology\"/> </map> </section> <article lang=\"en\"> <section aria-label=\"Article citation and metadata\"> <section class=\"pmc-layout__citation font-secondary font-xs\"> <div> <div class=\"display-inline-block\"> </div> . 2014 Sep 17;15(9):462. doi: <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/s13059-014-0462-7\" rel=\"noopener noreferrer\" target=\"_blank\"> 10.1186/s13059-014-0462-7 </a> </div> <nav hidden=\"hidden\" id=\"journal_context_menu\"> <ul class=\"menu-list font-family-ui\" role=\"menu\"> <li role=\"presentation\"> <a class=\"usa-link\" href=\"https://pmc.ncbi.nlm.nih.gov/search/?term=%22Genome%20Biol%22%5Bjour%5D\" role=\"menuitem\"> Search in PMC </a> </li> <li role=\"presentation\"> <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Genome%20Biol%22%5Bjour%5D\" lang=\"en\" role=\"menuitem\"> Search in PubMed </a> </li> <li role=\"presentation\"> <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Genome%20Biol%22%5BTitle%20Abbreviation%5D\" role=\"menuitem\"> View in NLM Catalog </a> </li> <li role=\"presentation\"> <a class=\"usa-link\" href=\"?term=%22Genome%20Biol%22%5Bjour%5D\" role=\"menuitem\"> Add to search </a> </li> </ul> </nav> </section> <section class=\"front-matter\"> <div class=\"ameta p font-secondary font-xs\"> <hgroup> <h1> Toward better benchmarking: challenge-based methods assessment in cancer genomics </h1> </hgroup> <div class=\"cg p\"> <a aria-describedby=\"id1\" aria-expanded=\"false\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Boutros%20PC%22%5BAuthor%5D\"> <span class=\"name western\"> Paul C Boutros </span> </a> <div hidden=\"hidden\" id=\"id1\"> <h3> <span class=\"name western\"> Paul C Boutros </span> </h3> <div class=\"p\"> <sup> 1 </sup> Informatics &amp; Biocomputing Program, Ontario Institute for Cancer Research, University Avenue, Toronto, ON M5G 0A3 Canada </div> <div class=\"p\"> <sup> 2 </sup> Department of Medical Biophysics, University of Toronto, College Street, Toronto, ON M5G 1L7 Canada </div> <div class=\"p\"> <sup> 3 </sup> Department of Pharmacology &amp; Toxicology, University of Toronto, King’s College Circle, Toronto, ON M5S 1A8 Canada </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Boutros%20PC%22%5BAuthor%5D\"> <span class=\"name western\"> Paul C Boutros </span> </a> </div> </div> <sup> 1, </sup> <sup> 2, </sup> <sup> 3 </sup> , <a aria-describedby=\"id2\" aria-expanded=\"false\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Margolin%20AA%22%5BAuthor%5D\"> <span class=\"name western\"> Adam A Margolin </span> </a> <div hidden=\"hidden\" id=\"id2\"> <h3> <span class=\"name western\"> Adam A Margolin </span> </h3> <div class=\"p\"> <sup> 4 </sup> Sage Bionetworks, Fairview Ave North, Seattle, WA 98109 USA </div> <div class=\"p\"> <sup> 5 </sup> Computational Biology Program, Oregon Health &amp; Science University, SW Sam Jackson Park Road, Portland, OR 97239-3098 USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Margolin%20AA%22%5BAuthor%5D\"> <span class=\"name western\"> Adam A Margolin </span> </a> </div> </div> <sup> 4, </sup> <sup> 5 </sup> , <a aria-describedby=\"id3\" aria-expanded=\"false\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Stuart%20JM%22%5BAuthor%5D\"> <span class=\"name western\"> Joshua M Stuart </span> </a> <div hidden=\"hidden\" id=\"id3\"> <h3> <span class=\"name western\"> Joshua M Stuart </span> </h3> <div class=\"p\"> <sup> 6 </sup> Department of Biomolecular Engineering, University of California, Santa Cruz, High Street, Santa Cruz, CA 95064 USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Stuart%20JM%22%5BAuthor%5D\"> <span class=\"name western\"> Joshua M Stuart </span> </a> </div> </div> <sup> 6 </sup> , <a aria-describedby=\"id4\" aria-expanded=\"false\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Califano%20A%22%5BAuthor%5D\"> <span class=\"name western\"> Andrea Califano </span> </a> <div hidden=\"hidden\" id=\"id4\"> <h3> <span class=\"name western\"> Andrea Califano </span> </h3> <div class=\"p\"> <sup> 7 </sup> Department of Systems Biology, Biochemistry &amp; Molecular Biophysics, Herbert Irving Comprehensive Cancer Center, Columbia University, St. Nicholas Avenue, New York, NY 10032 USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Califano%20A%22%5BAuthor%5D\"> <span class=\"name western\"> Andrea Califano </span> </a> </div> </div> <sup> 7 </sup> , <a aria-describedby=\"id5\" aria-expanded=\"false\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Stolovitzky%20G%22%5BAuthor%5D\"> <span class=\"name western\"> Gustavo Stolovitzky </span> </a> <div hidden=\"hidden\" id=\"id5\"> <h3> <span class=\"name western\"> Gustavo Stolovitzky </span> </h3> <div class=\"p\"> <sup> 8 </sup> IBM Computational Biology Center, TJ Watson Research Center, Kitchawan Road, Yorktown Heights, NY 10598 USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Stolovitzky%20G%22%5BAuthor%5D\"> <span class=\"name western\"> Gustavo Stolovitzky </span> </a> </div> </div> <sup> 8, </sup> <sup> ✉ </sup> </div> <ul class=\"d-buttons inline-list\"> <li> </li> <li> </li> <li> </li> </ul> <div class=\"d-panels font-secondary-light\"> <div class=\"d-panel p\" id=\"aip_a\"> <div class=\"p\" id=\"Aff1\"> <sup> 1 </sup> Informatics &amp; Biocomputing Program, Ontario Institute for Cancer Research, University Avenue, Toronto, ON M5G 0A3 Canada </div> <div id=\"Aff2\"> <sup> 2 </sup> Department of Medical Biophysics, University of Toronto, College Street, Toronto, ON M5G 1L7 Canada </div> <div id=\"Aff3\"> <sup> 3 </sup> Department of Pharmacology &amp; Toxicology, University of Toronto, King’s College Circle, Toronto, ON M5S 1A8 Canada </div> <div id=\"Aff4\"> <sup> 4 </sup> Sage Bionetworks, Fairview Ave North, Seattle, WA 98109 USA </div> <div id=\"Aff5\"> <sup> 5 </sup> Computational Biology Program, Oregon Health &amp; Science University, SW Sam Jackson Park Road, Portland, OR 97239-3098 USA </div> <div id=\"Aff6\"> <sup> 6 </sup> Department of Biomolecular Engineering, University of California, Santa Cruz, High Street, Santa Cruz, CA 95064 USA </div> <div id=\"Aff7\"> <sup> 7 </sup> Department of Systems Biology, Biochemistry &amp; Molecular Biophysics, Herbert Irving Comprehensive Cancer Center, Columbia University, St. Nicholas Avenue, New York, NY 10032 USA </div> <div id=\"Aff8\"> <sup> 8 </sup> IBM Computational Biology Center, TJ Watson Research Center, Kitchawan Road, Yorktown Heights, NY 10598 USA </div> <div class=\"author-notes p\"> <div class=\"fn\" id=\"_fncrsp93pmc__\"> <sup> ✉ </sup> <p class=\"display-inline\"> Corresponding author. </p> </div> </div> </div> <div class=\"d-panel p\" id=\"anp_a\"> <div class=\"notes p\"> <section class=\"history\" id=\"historyarticle-meta1\"> <p> Issue date 2014. </p> </section> </div> </div> <div class=\"d-panel p\" id=\"clp_a\"> <div> © Boutros et al.; licensee BioMed Central Ltd. 2014 </div> <p> The licensee has exclusive rights to distribute this article, in any medium, for 12 months following its publication. After this time, the article is available under the terms of the Creative Commons Attribution License ( <a class=\"usa-link usa-link--external\" href=\"http://creativecommons.org/licenses/by/4.0\" rel=\"noopener noreferrer\" target=\"_blank\"> http://creativecommons.org/licenses/by/4.0 </a> ), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly credited. The Creative Commons Public Domain Dedication waiver ( <a class=\"usa-link usa-link--external\" href=\"http://creativecommons.org/publicdomain/zero/1.0/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://creativecommons.org/publicdomain/zero/1.0/ </a> ) applies to the data made available in this article, unless otherwise stated. </p> <div class=\"p\"> <a class=\"usa-link\" href=\"/about/copyright/\"> PMC Copyright notice </a> </div> </div> </div> <div> PMCID: PMC4318527 PMID: <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/25314947/\"> 25314947 </a> </div> </div> </section> </section> <section aria-label=\"Article content\"> <section class=\"body main-article-body\"> <section class=\"abstract\" id=\"Abs1\"> <h2> <a class=\"usa-anchor\" id=\"Abs1-anchor\"> </a> Abstract </h2> <p> Rapid technological development has created an urgent need for improved evaluation of algorithms for the analysis of cancer genomics data. We outline how challenge-based assessment may help fill this gap by leveraging crowd-sourcing to distribute effort and reduce bias. </p> </section> <hr class=\"headless\"/> <p> Computational biology comprises three inter-connected activities: algorithm development, validation through benchmarking, and application. In the biomedical sciences, benchmarking occupies a central and indispensable role as it maps algorithms from the space of theoretical possibilities to the realm of practical value. Critically, this process attributes specific probabilities to an algorithm’s discovery of biologically relevant knowledge (measured by the sensitivity of the algorithm) while not overwhelming the researcher with incorrect predictions (quantified by the algorithm specificity). Benchmarking is, however, a complex task, requiring the creation of comprehensive gold standards and the design of sophisticated validation strategies that may require additional experimental data. Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table <a class=\"usa-link\" href=\"#Tab1\"> 1 </a> ). In particular, the rapidly increasing size of whole-genome molecular profile datasets from large sample repositories underscores the importance of benchmarking; it has become virtually impossible to validate algorithmic predictions that are based on such large datasets systematically. </p> <section class=\"tw xbox font-sm\" id=\"Tab1\"> <h2 class=\"obj_head\"> Table 1. </h2> <div class=\"caption p\"> <p> <strong> Non-comprehensive list of important and current challenge efforts and platforms </strong> </p> </div> <div class=\"tbl-box p\" tabindex=\"0\"> <table class=\"content\" frame=\"hsides\" rules=\"groups\"> <thead> <tr valign=\"top\"> <th colspan=\"1\" rowspan=\"1\"> <strong> Challenge </strong> </th> <th colspan=\"1\" rowspan=\"1\"> <strong> Scope </strong> </th> <th colspan=\"1\" rowspan=\"1\"> <strong> Assessment type </strong> </th> <th colspan=\"1\" rowspan=\"1\"> <strong> Organizers </strong> </th> <th colspan=\"1\" rowspan=\"1\"> <strong> Website </strong> </th> </tr> </thead> <tbody> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> Assemblathon1&amp;2 </td> <td colspan=\"1\" rowspan=\"1\"> Sequence assembly </td> <td colspan=\"1\" rowspan=\"1\"> Objective scoring </td> <td colspan=\"1\" rowspan=\"1\"> UC Davis Genome Center </td> <td colspan=\"1\" rowspan=\"1\"> <a class=\"usa-link usa-link--external\" href=\"http://assemblathon.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://assemblathon.org/ </a> </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> CAFA </td> <td colspan=\"1\" rowspan=\"1\"> Protein function prediction </td> <td colspan=\"1\" rowspan=\"1\"> Objective scoring </td> <td colspan=\"1\" rowspan=\"1\"> Community collaboration </td> <td colspan=\"1\" rowspan=\"1\"> <a class=\"usa-link usa-link--external\" href=\"http://biofunctionprediction.org/node/8\" rel=\"noopener noreferrer\" target=\"_blank\"> http://biofunctionprediction.org/node/8 </a> </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> CAGI </td> <td colspan=\"1\" rowspan=\"1\"> Systems biology </td> <td colspan=\"1\" rowspan=\"1\"> Objective scoring </td> <td colspan=\"1\" rowspan=\"1\"> UC Berkley/University of Maryland </td> <td colspan=\"1\" rowspan=\"1\"> <a class=\"usa-link usa-link--external\" href=\"http://genomeinterpretation.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://genomeinterpretation.org/ </a> </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> CAPRI </td> <td colspan=\"1\" rowspan=\"1\"> Protein docking </td> <td colspan=\"1\" rowspan=\"1\"> Objective scoring </td> <td colspan=\"1\" rowspan=\"1\"> Community collaboration </td> <td colspan=\"1\" rowspan=\"1\"> <a class=\"usa-link usa-link--external\" href=\"http://www.ebi.ac.uk/msd-srv/capri/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.ebi.ac.uk/msd-srv/capri/ </a> </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> CASP </td> <td colspan=\"1\" rowspan=\"1\"> Structure prediction </td> <td colspan=\"1\" rowspan=\"1\"> Objective scoring </td> <td colspan=\"1\" rowspan=\"1\"> Community collaboration </td> <td colspan=\"1\" rowspan=\"1\"> <a class=\"usa-link usa-link--external\" href=\"http://predictioncenter.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://predictioncenter.org/ </a> </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> ChaLearn </td> <td colspan=\"1\" rowspan=\"1\"> Machine learning </td> <td colspan=\"1\" rowspan=\"1\"> Objective scoring </td> <td colspan=\"1\" rowspan=\"1\"> ChaLearn Organization (non-for profit) </td> <td colspan=\"1\" rowspan=\"1\"> <a class=\"usa-link usa-link--external\" href=\"http://www.chalearn.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.chalearn.org/ </a> </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> CLARITY </td> <td colspan=\"1\" rowspan=\"1\"> Clinical genome interpretation </td> <td colspan=\"1\" rowspan=\"1\"> Objective scoring and evaluation by judges </td> <td colspan=\"1\" rowspan=\"1\"> Boston Children’s Hospital </td> <td colspan=\"1\" rowspan=\"1\"> <a class=\"usa-link usa-link--external\" href=\"http://www.childrenshospital.org/research-and-innovation/research-initiatives/clarity-challenge\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.childrenshospital.org/research-and-innovation/research-initiatives/clarity-challenge </a> </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> DREAM </td> <td colspan=\"1\" rowspan=\"1\"> Network inference and systems biology </td> <td colspan=\"1\" rowspan=\"1\"> Objective scoring </td> <td colspan=\"1\" rowspan=\"1\"> Community collaboration &amp; Sage Bionetworks </td> <td colspan=\"1\" rowspan=\"1\"> <a class=\"usa-link usa-link--external\" href=\"https://www.synapse.org/#!Challenges:DREAM\" rel=\"noopener noreferrer\" target=\"_blank\"> https://www.synapse.org/#!Challenges:DREAM </a> </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> FlowCAP </td> <td colspan=\"1\" rowspan=\"1\"> Flow cytometry analysis </td> <td colspan=\"1\" rowspan=\"1\"> Objective scoring </td> <td colspan=\"1\" rowspan=\"1\"> Community collaboration </td> <td colspan=\"1\" rowspan=\"1\"> <a class=\"usa-link usa-link--external\" href=\"http://flowcap.flowsite.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://flowcap.flowsite.org/ </a> </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> IGCG-TCGA DREAM Somatic Mutation Calling </td> <td colspan=\"1\" rowspan=\"1\"> Sequence analysis </td> <td colspan=\"1\" rowspan=\"1\"> Objective evaluation </td> <td colspan=\"1\" rowspan=\"1\"> Community collaboration &amp; Sage Bionetworks </td> <td colspan=\"1\" rowspan=\"1\"> <a class=\"usa-link usa-link--external\" href=\"https://www.synapse.org/#!Synapse:syn312572\" rel=\"noopener noreferrer\" target=\"_blank\"> https://www.synapse.org/#!Synapse:syn312572 </a> </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> IMPROVER </td> <td colspan=\"1\" rowspan=\"1\"> Systems biology </td> <td colspan=\"1\" rowspan=\"1\"> Objective evaluation and crowd-verification </td> <td colspan=\"1\" rowspan=\"1\"> Phillip Morris International </td> <td colspan=\"1\" rowspan=\"1\"> <a class=\"usa-link usa-link--external\" href=\"https://sbvimprover.com/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://sbvimprover.com/ </a> </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> Innocentive </td> <td colspan=\"1\" rowspan=\"1\"> Topics in various industries </td> <td colspan=\"1\" rowspan=\"1\"> Objective scoring and evaluation by judges </td> <td colspan=\"1\" rowspan=\"1\"> Commercial platform </td> <td colspan=\"1\" rowspan=\"1\"> <a class=\"usa-link usa-link--external\" href=\"http://www.innocentive.com/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.innocentive.com/ </a> </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> Kaggle </td> <td colspan=\"1\" rowspan=\"1\"> Topics in various industries </td> <td colspan=\"1\" rowspan=\"1\"> Objective scoring and evaluation by judges </td> <td colspan=\"1\" rowspan=\"1\"> Commercial platform </td> <td colspan=\"1\" rowspan=\"1\"> <a class=\"usa-link usa-link--external\" href=\"http://www.kaggle.com/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.kaggle.com/ </a> </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> RGASP </td> <td colspan=\"1\" rowspan=\"1\"> RNA-seq analyses </td> <td colspan=\"1\" rowspan=\"1\"> Objective scoring </td> <td colspan=\"1\" rowspan=\"1\"> European Bioinformatics Institute </td> <td colspan=\"1\" rowspan=\"1\"> <a class=\"usa-link usa-link--external\" href=\"http://www.gencodegenes.org/rgasp/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.gencodegenes.org/rgasp/ </a> </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> Sequence Squeeze </td> <td colspan=\"1\" rowspan=\"1\"> Sequence compression </td> <td colspan=\"1\" rowspan=\"1\"> Objective scoring and evaluation by judges </td> <td colspan=\"1\" rowspan=\"1\"> Pistoia Alliance </td> <td colspan=\"1\" rowspan=\"1\"> <a class=\"usa-link usa-link--external\" href=\"http://sequencesqueeze.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://sequencesqueeze.org/ </a> </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> X-Prize </td> <td colspan=\"1\" rowspan=\"1\"> Technology </td> <td colspan=\"1\" rowspan=\"1\"> Evaluation by judges </td> <td colspan=\"1\" rowspan=\"1\"> X-Prize Organization (non-for-profit) </td> <td colspan=\"1\" rowspan=\"1\"> <a class=\"usa-link usa-link--external\" href=\"http://www.xprize.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.xprize.org/ </a> </td> </tr> </tbody> </table> </div> <div class=\"p text-right font-secondary\"> <a class=\"usa-link\" href=\"table/Tab1/\" rel=\"noopener noreferrer\" target=\"_blank\"> Open in a new tab </a> </div> <div class=\"tw-foot p\"> <div class=\"fn\" id=\"_fn_p4\"> <p> The challenges were chosen based on relevance to cancer genomics or the representativeness of a type of challenge. Different challenges specialize in specific areas of research (see ‘Scope’), and may use different assessment types such as objective scoring against a gold standard, evaluation by judges, or community consensus (‘crowd-verification’). Organizers can be researchers from specific institutions (such as universities or hospitals), a group of diverse researchers from academia and industry collaborating in the challenge organization (community collaboration), not-for-profit associations, or commercial platforms that run challenges as their business model (such as Innocentive and Kaggle). Initiatives such as CAFA, CAGI, CAPRI, CASP, ChaLearn, DREAM, FlowCAP and IMPROVER organize several challenges each year, and only the generic project is listed in this table, with the exception of DREAM, for which we also show the IGCG-TCGA DREAM Somatic Mutation Calling Challenge because of its relevance to this paper. More information about these efforts can be found on the listed websites. </p> </div> </div> </section> <p> Benchmarking is not a matter of simply running a few algorithms on a few datasets and comparing the results. Drawing generalizable conclusions from the exercise requires significant care in design and execution. The maturity of bioinformatics as a discipline has been greatly advanced by the adoption of key principles that guide robust method evaluation, including evaluator objectiveness (lack of bias), clearly defined scoring metrics that align with real-world goals, and the public release of gold-standard datasets and of the results and code of prediction algorithms. Challenge-based (also known as ‘competition-based’) method assessment is an increasingly popular mechanism for benchmarking [ <a aria-describedby=\"CR1\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR1\"> 1 </a> , <a aria-describedby=\"CR2\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR2\"> 2 </a> ]. In this type of study an impartial group of scientists organizes a ‘challenge’ that is based on a carefully curated dataset. This dataset is typically split into a training dataset, a validation dataset (which might be used in real-time leaderboards, typically implemented as a table that reports the comparative performance of the methods under development), and a gold standard (or test) dataset that is withheld from challenge participants and used for final evaluation (Figure <a class=\"usa-link\" href=\"#Fig1\"> 1 </a> ). Following algorithm development on the training dataset and real-time feedback to participants based on the validation dataset and reported in the leaderboard, the challenge organizers can objectively evaluate the quality of final submitted predictions using a gold-standard dataset. Such a design closely reflects the actual difficulties faced by real-world users trying to determine whether an algorithm generalizes to unseen cases. </p> <figure class=\"fig xbox font-sm\" id=\"Fig1\"> <h2 class=\"obj_head\"> Figure 1. </h2> <p class=\"img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center\"> </p> <div class=\"p text-right font-secondary\"> <a class=\"usa-link\" href=\"figure/Fig1/\" rel=\"noopener noreferrer\" target=\"_blank\"> Open in a new tab </a> </div> <figcaption> <p> <strong> Typical design of a crowd-sourced challenge. </strong> A dataset is split into a training set, a validation (or leaderboard set) and the test set (or gold standard). Participants have access to the challenge input data and the known answers for just the training set. For the validation and test sets only, the challenge input data are provided but the answers to the challenge questions are withheld. In the challenge open phase, participants optimize their algorithms by making repeated submissions to predict the validation set answers. These submissions are scored and returned to the participants who can use the information to improve their methods. In the final evaluation phase, the optimized algorithms are submitted and evaluated against the final test set (the gold standard), and the resulting scores are used to compute the statistical significance and the ranking of the participating algorithms. </p> </figcaption> </figure> <p> When flawed, benchmarking can lead to the emergence of suboptimal standards that may be applied to many large datasets, imposing an immense cost to the community and creating misleading results. Conversely, the acceptance of knowledge without robust benchmarking can lead to the adoption of inaccurate conventions. For example, during the 1990s, it was generally accepted that the number of loci coding for proteins in the human genome was 100,000, a number that was based on unverified hypotheses [ <a aria-describedby=\"CR3\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR3\"> 3 </a> ]. When the human genome was finally sequenced in 2000, the total number of coding loci was found to be a factor of 5 lower. Similarly, a design error in the early implementation of the GC Robust Multi-Array (GCRMA) algorithm, which was revealed by systematic benchmarking of network reconstruction analyses, may have led to the publication of thousands of papers that contain incorrect mRNA abundance profiles before the error was detected and corrected [ <a aria-describedby=\"CR4\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR4\"> 4 </a> ]. As a third example, in 2006, a group of Duke University researchers published a pair of high-impact papers claiming accurate prediction of the prognosis of lung cancer patients and of chemotherapy-sensitivity in lung, breast and ovarian cancers. Attempts to reproduce those claims ensued almost immediately, with most of the results falling short of replication because of a combination of programming and data-entry errors, and possible fraud [ <a aria-describedby=\"CR5\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR5\"> 5 </a> ]. Proper objective benchmarking by a neutral third-party on private validation data helps to resolve quickly or to detect many of the issues associated with these kinds of studies. </p> <p> One concern in algorithm benchmarking and validation is that computational biology algorithms are often developed and evaluated by the same researchers. This creates an inherent conflict of interest, where objective assessment of accuracy is polluted by the fact that the developers become simultaneously judge, jury and executioner of the validity of their own work. This can result in biases in study design and over-optimistic performance estimates, whether intentional or unintentional [ <a aria-describedby=\"CR6\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR6\"> 6 </a> ]. For instance, the use of non-blinded data in the evaluation by methods developers of their own protein structure prediction methods led, in the early’ 80s, to the false belief that protein structure prediction was essentially a solved problem. Not until 1994, when double-blinded data were used in the first Workshop on the Critical Assessment of Protein Structure Prediction (CASP), was a very different picture revealed [ <a aria-describedby=\"CR7\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR7\"> 7 </a> ]. </p> <p> Challenge-based benchmarking efforts, such as CASP [ <a aria-describedby=\"CR8\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR8\"> 8 </a> – <a aria-describedby=\"CR10\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR10\"> 10 </a> ], CAFA [ <a aria-describedby=\"CR11\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR11\"> 11 </a> ] and DREAM [ <a aria-describedby=\"CR12\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR12\"> 12 </a> , <a aria-describedby=\"CR13\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR13\"> 13 </a> ], among others (Table <a class=\"usa-link\" href=\"#Tab1\"> 1 </a> ), offer a robust framework for algorithm evaluation. These efforts have proven the value of engaging both active challenge leaders and motivated algorithm developers to improve their work in a forum that has high visibility and rapid feedback. </p> <p> We believe that challenge-based methods assessment will play an increasingly important role in standardizing and optimizing the analysis of cancer genomics data, and its broader adoption will drive progress in both algorithm development and biological discovery. Conversely, failing to exploit challenge-benchmarking as a fundamental validation methodology for cancer genomics algorithms may result in lost opportunities to translate results derived from best-in-class methods into patient care. </p> <p> Here, we provide our perspective on the growing use of challenge-based methods to benchmark algorithms in cancer genomics. We outline the different types of problems faced and some of the key considerations that need to be explored to determine whether a challenge might be successful, and to provide suggestions for challenge organization and execution. Finally, we look to the future and consider how challenge-based assessment may change in the coming decade. </p> <section id=\"Sec1\"> <h2 class=\"pmc_sec_title\"> <a class=\"usa-anchor\" id=\"Sec1-anchor\"> </a> Challenge design and dynamics </h2> <p> Over the past few years, an established challenge-based design paradigm has emerged in which portions of a private (that is, not globally released) dataset are made publicly available according to a predefined schedule. Such a dataset provides increased user engagement based on continuous feedback; an opportunity for participants to refine and improve their methods on the basis of results obtained throughout the challenge; and multiple independent rounds of validation, which can be used to assess the consistency and robustness of results. After the initial training dataset is made publicly available, a real-time leaderboard can be generated in which the performance of different algorithms is evaluated against a withheld private portion of the data (Figure <a class=\"usa-link\" href=\"#Fig1\"> 1 </a> ). Previous research has shown that the provision of real-time feedback is among the most important factors in ensuring user engagement in crowd-sourcing projects [ <a aria-describedby=\"CR14\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR14\"> 14 </a> ]. (Here, we use the term crowd-sourcing in the sense that a community of tens to hundreds of researchers are engaged in working on the same problem. In other contexts, crowd-sourcing activities may engage different numbers of participants.) After a period of time in which several iterations of the leaderboard can be posted, one of the participating groups is declared the best performer in this initial phase of the challenge, either on the basis of their position on the leaderboard or because they were the first to reach some pre-specified performance level. The challenge may include multiple rounds of assessment based on different portions of the private data. A final round is typically invoked in which methods are rated against a withheld evaluation dataset to determine the overall challenge winner (Figure <a class=\"usa-link\" href=\"#Fig1\"> 1 </a> ). The most robust validation set is often reserved for this final evaluation - often with larger sample size, newly generated data or prospective validation designed on the basis of challenge results. Each participating team submits a small number (for example, one to five) of independent predictions made by their algorithm(s), which are scored and ranked to determine a winner. Finally, the public release of all of the data kept private throughout the challenge, along with the predictions and ideally source code from each group, provides a long-term resource to spur further development of new and improved methods. </p> <p> The collection of algorithm source code allows developers to share insights that promote future improvements. If required as part of the final submission, this code can also be used to ensure objective scoring and verification of reproducibility. In the 2012 Sage Bionetworks-DREAM Breast Cancer Prognosis Challenge, participants were required to submit their models as open source R-code [ <a aria-describedby=\"CR15\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR15\"> 15 </a> ] that was visible to all participants and executed by an automated system to generate the results reported on the leaderboard. This was enabled by Synapse [ <a aria-describedby=\"CR16\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR16\"> 16 </a> ], a software platform that supports scientific challenges as well as large distributed collaborations, such as those in the TCGA Pan-Cancer consortium [ <a aria-describedby=\"CR17\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR17\"> 17 </a> ]. Planned challenges, such as the RNA-seq follow-up to the ICGC-TCGA DREAM Somatic Mutation Calling (SMC) Challenge, are considering the use of cloud-computing solutions to provide a central computing facility and a harness upon which contestant code is directly run. This will inherently force the deposition of complete analysis workflows, which can be run routinely on new datasets. Further, this approach would help to standardize application programming interfaces and file formats, such that multiple algorithms use similar inputs and produce easily comparable outputs. This vision of interoperability is shared by many practitioners in the field and has most recently been championed by the Global Alliance for Genomics And Health [ <a aria-describedby=\"CR18\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR18\"> 18 </a> ]. </p> <p> Several criteria should be used to help participants limit over-fitting to the training data. Over-fitting is a known peril in statistics, occurring when a predictive model has enough flexibility in its parameters that optimization effectively leads to ‘memorization’ of the training data and an inability to generalize to unseen cases. The most common way to help participants avoid over-fitting, while enabling the testing of their models, is to provide leaderboard scoring that is based on a subset of the private dataset, optimally a subset that is not used in the final evaluation. The latter condition is sometimes not feasible (for example, when the number of patients available to predict clinical outcomes is limited), in which case the leaderboard will be based on data that are also used for the final scoring. If this is the case, limiting the number of submissions can reduce over-fitting. </p> <p> While most challenges share some common design principles, each research area has its own unique characteristics that require customized experimental designs and consideration of risks and benefits. Indeed, the utility of organizing a challenge to help advance a particular research area depends on a balance between challenge-based benchmarking advantages and limitations, as well as consideration of the potential barriers for participation (Table <a class=\"usa-link\" href=\"#Tab2\"> 2 </a> ). In the sections below, we highlight three research areas in which rapid development of new algorithms has led to a concomitant need for benchmarking: accurate identification of tumor-specific genomic alterations, association of clinical data with genomic profiles (that is, biomarkers) and identifying network-biology features that underlie cancer phenotypes. </p> <section class=\"tw xbox font-sm\" id=\"Tab2\"> <h3 class=\"obj_head\"> Table 2. </h3> <div class=\"caption p\"> <p> <strong> Some advantages and limitations of challenge-based methods assessment, along with barriers to participation in them </strong> </p> </div> <div class=\"tbl-box p\" tabindex=\"0\"> <table class=\"content\" frame=\"hsides\" rules=\"groups\"> <thead> <tr valign=\"top\"> <th colspan=\"1\" rowspan=\"1\"> <strong> Advantages </strong> </th> <th colspan=\"1\" rowspan=\"1\"> <strong> Limitations </strong> </th> <th colspan=\"1\" rowspan=\"1\"> <strong> Participation barriers </strong> </th> </tr> </thead> <tbody> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> Reduction of over-fitting </td> <td colspan=\"1\" rowspan=\"1\"> Narrower scope compared to traditional open-ended research </td> <td colspan=\"1\" rowspan=\"1\"> Incentives not strong enough to promote participation </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> Benchmarking individual methods </td> <td colspan=\"1\" rowspan=\"1\"> Ground truth needed for objective scoring </td> <td colspan=\"1\" rowspan=\"1\"> No funding available to support time spent participating in challenges </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> Impartial comparison across methods using same datasets </td> <td colspan=\"1\" rowspan=\"1\"> Mostly limited to computational approaches </td> <td colspan=\"1\" rowspan=\"1\"> Fatigue resulting from many ongoing challenges </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> Fostering collaborative work, including code sharing </td> <td colspan=\"1\" rowspan=\"1\"> Requires data producers to share their data before publication </td> <td colspan=\"1\" rowspan=\"1\"> Time assigned by organizers to solve a difficult challenge question may be too short </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> Acceleration of research </td> <td colspan=\"1\" rowspan=\"1\"> Sufficient amount of high-quality data needed for meaningful results </td> <td colspan=\"1\" rowspan=\"1\"> Lack of computing capabilities </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> Enhancing data access and impact </td> <td colspan=\"1\" rowspan=\"1\"> Large number of participants not always available </td> <td colspan=\"1\" rowspan=\"1\"> New data modality or datasets that are too complex or too big poses entry barrier </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> Determination of problem solvability </td> <td colspan=\"1\" rowspan=\"1\"> Challenge questions may not be solvable with data at hand </td> <td colspan=\"1\" rowspan=\"1\"> Challenge questions not interesting or impactful enough </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> Tapping the ‘Wisdom of Crowds’ </td> <td colspan=\"1\" rowspan=\"1\"> Traditional grant mechanisms not adequate to fund challenge efforts </td> <td colspan=\"1\" rowspan=\"1\"> Cumbersome approvals to acquire sensitive datasets </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> Objective assessment </td> <td colspan=\"1\" rowspan=\"1\"> Difficulties to distribute datasets with sensitive information </td> <td colspan=\"1\" rowspan=\"1\"> </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> Standardizes experimental design </td> <td colspan=\"1\" rowspan=\"1\"> </td> <td colspan=\"1\" rowspan=\"1\"> </td> </tr> </tbody> </table> </div> <div class=\"p text-right font-secondary\"> <a class=\"usa-link\" href=\"table/Tab2/\" rel=\"noopener noreferrer\" target=\"_blank\"> Open in a new tab </a> </div> </section> </section> <section id=\"Sec2\"> <h2 class=\"pmc_sec_title\"> <a class=\"usa-anchor\" id=\"Sec2-anchor\"> </a> Analyzing genome assembly and structural variants </h2> <p> Technologies for identifying cancer-related somatic alterations from genomic or transcriptomic data are advancing extremely rapidly. In only 6 years, next-generation sequencing (NGS) has rapidly progressed from the measurement of millions of short sequences (of around 25 bp) to that of hundreds of millions of longer segments (of around 100 bp). This creates an urgent need for on-going benchmarking studies as old algorithms become rapidly out-dated and new algorithmic approaches are required to handle new technologies and new scales of data. Small-scale studies have resulted in dramatic discordance when different researchers apply their algorithms to the same genomic data (Figure <a class=\"usa-link\" href=\"#Fig2\"> 2 </a> ) [ <a aria-describedby=\"CR19\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR19\"> 19 </a> – <a aria-describedby=\"CR21\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR21\"> 21 </a> ]. These studies have shown that accuracy and generalizability vary dramatically across samples and regions of the genome. The constantly shifting landscape presented by rapidly evolving technologies and tools fuels the urgency in the need to identify the best-performing methods objectively and to re-evaluate them frequently, and to identify particularly error-prone aspects of existing tumor genome analysis methods [ <a aria-describedby=\"CR22\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR22\"> 22 </a> ]. Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of <em> de novo </em> sequence assembly algorithms [ <a aria-describedby=\"CR23\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR23\"> 23 </a> ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ <a aria-describedby=\"CR24\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR24\"> 24 </a> ] (Table <a class=\"usa-link\" href=\"#Tab1\"> 1 </a> ). </p> <figure class=\"fig xbox font-sm\" id=\"Fig2\"> <h3 class=\"obj_head\"> Figure 2. </h3> <p class=\"img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center\"> </p> <div class=\"p text-right font-secondary\"> <a class=\"usa-link\" href=\"figure/Fig2/\" rel=\"noopener noreferrer\" target=\"_blank\"> Open in a new tab </a> </div> <figcaption> <p> <strong> Different researchers studying the same data may arrive at discordant conclusions. </strong> Benchmarking becomes essential as a way to separate true findings from spurious ones. (Illustration by Natasha Stolovitzky-Brunner© inspired by the parable of the six blind men and the elephant). </p> </figcaption> </figure> <p> Challenge-based benchmarking of methods for somatic variant detection in cancer faces several unique hurdles. First, genomic sequence is inherently identifiable [ <a aria-describedby=\"CR25\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR25\"> 25 </a> ], and is thus considered personal health information (PHI) in many countries. This places a burden on challenge contestants to acquire ethics approval from the appropriate authorities, such as dbGaP in the USA or ICGC in Canada. Second, because of the inherent complexity of both the data and file formats, it may be difficult for researchers from other fields to acquire sufficient domain knowledge to compete effectively against domain experts. This point may be ameliorated by gamifying the problem, that is, using game tools that require puzzle solving or geometric thinking to engage users in genomics problems [ <a aria-describedby=\"CR26\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR26\"> 26 </a> , <a aria-describedby=\"CR27\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR27\"> 27 </a> ]. Gamification may not be possible or appropriate, however, because it may require sacrificing domain-specific prior knowledge that is essential to the correct solution. Third, the size of the raw genomic data necessary to perform these challenges creates a ‘big-data’ problem. For example, the ICGC-TCGA DREAM SMC Challenge [ <a aria-describedby=\"CR28\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR28\"> 28 </a> ] (Table <a class=\"usa-link\" href=\"#Tab1\"> 1 </a> ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs. Two different solutions to this problem are to provide access to high-speed, cloud-based download technologies (such as GeneTorrent or Aspera) or to provide co-location of computers and data in a hosted environment [ <a aria-describedby=\"CR29\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR29\"> 29 </a> ]. The latter solution has the advantage of providing implementations of the best-performing algorithms in a form that is more readily redistributed to the community, as well as allowing more ‘democratized’ participation for groups that do not have large in-house computing resources. Nevertheless, this solution also has disadvantages: cloud-computing may require additional overhead expenditure for groups that are familiar with developing methods within their local computing environments; many researchers have access to in-house computing options subsidized by their institution and have limited incentive to transfer their analysis to the cloud; and access permissions for some datasets can hinder redistribution through cloud platforms. Furthermore, the assessment of predictions is challenging because the ground-truth for genetic alterations is unknown. The SMC Challenge employs two strategies for evaluation. The first involves an <em> in silico </em> method for simulating cancer genomes called BAMSurgeon, which was developed to allow the comparison of methods predictions against a synthetic ground-truth (work by Ewing and colleagues). In the second strategy, targeted deep-sequencing allows prospective validation of a large number of predicted mutations, chosen by an algorithm that most accurately computes false-positive and false-negative rates across submissions. It is unclear how important it is for prospective validation data to be orthogonal to that used by the original challenge participants. Verification in TCGA projects typically relies on deep sequencing using the same technology, but on selected targets and with the construction of new sequencing libraries. This approach assumes that most errors are randomly distributed and/or associated with only a small fraction of reads. The more orthogonal the validation technology, the more this assumption is relaxed. Nevertheless, the error profile of the final evaluation dataset is crucial, and there are currently no error-free approaches to generating this gold-standard data for NGS. </p> </section> <section id=\"Sec3\"> <h2 class=\"pmc_sec_title\"> <a class=\"usa-anchor\" id=\"Sec3-anchor\"> </a> Finding genomic biomarkers that are associated with phenotype </h2> <p> Once a set of somatic variants have been identified from genomic interrogation of patient-derived samples, one of the most common analyses is to attempt to develop biomarkers that can predict patient survival, response to therapy or other outcomes [ <a aria-describedby=\"CR30\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR30\"> 30 </a> – <a aria-describedby=\"CR33\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR33\"> 33 </a> ]. The development of genomic-based personalized medicine has immense clinical potential, but the optimal approach to predicting such biomarkers <em> de novo </em> remains poorly understood and controversial. Indeed, it is widely known that inferred biomarkers are highly sensitive to factors such as choice of algorithm and data pre-processing methods [ <a aria-describedby=\"CR34\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR34\"> 34 </a> – <a aria-describedby=\"CR37\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR37\"> 37 </a> ]. </p> <p> Nevertheless, developing challenges to benchmark biomarker discovery problems is relatively straightforward. Participants are given training data in which features (for example, genome-wide mRNA transcript abundance) are paired with outcome (for example, patient survival) data. Participants are given only the features for the test set and asked to predict the outcome data using a model inferred from the training data. Alternatively, participants may submit trained models as executable code to be run on the test data, thus allowing the test feature data to be hidden from participants [ <a aria-describedby=\"CR15\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR15\"> 15 </a> ]. Model results are scored on the basis of the correspondence between predicted and measured outcome data from the test set. </p> <p> Prediction challenges have been employed in many domains outside of biomedical research [ <a aria-describedby=\"CR38\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR38\"> 38 </a> ]. Because biomarker-based challenges fit the setup of the classic supervised machine-learning paradigm, they attract new ideas and participation from the broader machine-learning community. Benchmarking in biomarker discovery is crucial, however, as outlined by the case of the retracted Duke study on chemotherapy selection noted above. </p> <p> Two key difficulties exist in the creation of benchmarking challenges for biomarker discovery. First, the ideal datasets for biomarker-discovery challenges are uniquely defined, especially when data were collected from large cohorts requiring long-term follow-up or expensive standardized treatment protocols (such as clinical trials). These datasets can potentially lead to high-impact publications or concerns over the intellectual property of the data-generating groups. Second, the potential size of patient cohorts is currently limiting for many biomarker-development questions. If the amount of data available is inadequate, they may not generate enough statistical power to distinguish the performance of the top-ranked groups accurately. These factors also complicate the ability to obtain independent datasets for final method assessment. Despite these problems, several successful challenges pertaining to diagnostics, prognostics and treatment outcomes have been conducted, including the MAQC-II study [ <a aria-describedby=\"CR39\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR39\"> 39 </a> ], the IMPROVER Challenge on Diagnostic Signatures [ <a aria-describedby=\"CR40\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR40\"> 40 </a> ], the Sage Bionetworks DREAM Breast Cancer Prognostics Challenge [ <a aria-describedby=\"CR15\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR15\"> 15 </a> ], and the DREAM AML Treatment Outcome Challenge [ <a aria-describedby=\"CR41\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR41\"> 41 </a> ]. </p> </section> <section id=\"Sec4\"> <h2 class=\"pmc_sec_title\"> <a class=\"usa-anchor\" id=\"Sec4-anchor\"> </a> Inferring biological networks underlying cancer phenotypes </h2> <p> Identifying the relationships between biological (transcriptional and signaling) networks and cancer onset and progression is another potential area for challenge benchmarking. Network analysis involves several aspects, including the coherent modeling of different types of alteration and dysregulation events and their integration into a unified network-based model [ <a aria-describedby=\"CR42\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR42\"> 42 </a> – <a aria-describedby=\"CR44\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR44\"> 44 </a> ]. One of the major problems with organizing challenges in this area is that the underlying cellular regulatory networks are mostly unknown, especially in complex systems such mammalian tumor cells. So how can a challenge be organized when a pre-known gold-standard network cannot be defined? Several strategies employed by the DREAM project include using synthetic biology networks [ <a aria-describedby=\"CR13\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR13\"> 13 </a> ], <em> in silico </em> networks [ <a aria-describedby=\"CR45\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR45\"> 45 </a> ], and experimentally assessed bacterial networks [ <a aria-describedby=\"CR46\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR46\"> 46 </a> ]. An alternative strategy is to evaluate methods on the basis of their ability to predict the response of a system to a set of perturbations, such as drugs or receptor ligands, as surrogates for predicting the underlying network connectivity [ <a aria-describedby=\"CR47\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR47\"> 47 </a> ]. The introduction of ingenious surrogates to the gold standard has enabled the formulation of other network reverse-engineering challenges, such as the 2013 HPN-DREAM Breast Cancer Network Inference Challenge [ <a aria-describedby=\"CR48\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR48\"> 48 </a> ]. In this challenge, participants were asked to submit predicted signaling networks that were activated by a set of stimuli in four breast cancer cell lines. These networks were scored on the basis of their ability to identify the set of proteins that are downstream of a given phosphoprotein. The predicted protein set was compared to an experimentally determined set of proteins (the surrogate gold standard), defined as those proteins whose phosphorylation levels were affected by inhibiting that phosphoprotein. Further research on benchmarking network-inference algorithms would be highly beneficial to help advance the field of network biology, whose role in unraveling biological mechanisms in cancer is hard to overestimate. </p> </section> <section id=\"Sec5\"> <h2 class=\"pmc_sec_title\"> <a class=\"usa-anchor\" id=\"Sec5-anchor\"> </a> The truth is hard to find </h2> <p> From the previous discussion, it is clear that the single most crucial aspect in benchmarking is the definition and assembly of gold standards. A gold standard fundamentally defines the problem under study, and it provides the limiting resolution of error for the overall endeavor. As outlined in this article, gold standards can be defined in several ways. First, a single experiment can be performed with portions of the resulting data used for training and evaluation. This approach avoids experimental inconsistencies, but requires that a large selection of true results is generated prior to the challenge. Simulated datasets are ideal for this strategy but have been criticized as only partially representing a biological system [ <a aria-describedby=\"CR49\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR49\"> 49 </a> ]. While validation of simulated data is straight forward, because the ground-truth is completely known, in most cases the value of benchmarking is perceived to be in the ability to assess best-performing methods when applied to real biological data as opposed to simulated data. An important caveat is that the synthetic data may fail to reflect some of the underlying assumptions of the system they attempt to emulate. Indeed, the most common question about simulations is how well they reflect experimental samples [ <a aria-describedby=\"CR49\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR49\"> 49 </a> ]. </p> <p> Second, for systems that are difficult to benchmark directly, such as the structure of a biological network, characteristics of the systems can be evaluated instead. These might include the effects of the systems’ perturbation or other phenomena, such as the identification of the networks that best predict patient outcomes. </p> <p> Third, the results of a study can be validated after the challenge is completed by additional experimental work, either on the same sample or on others. This has the advantage of directly addressing the predictions made by challenge participants, but has the disadvantage of introducing a time lag between challenge completion and the availability of full results. In addition, the effort and cost of follow-up validation may be prohibitive given the resources available to the challenge organizers. </p> <p> For genomic studies, wet-lab validation can be both time-consuming and expensive. For example, the MAQC study considered approximately 20,000 genes on microarray platforms, but only validated approximately 1,000 (5%) by real-time PCR as a gold standard [ <a aria-describedby=\"CR50\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR50\"> 50 </a> ]. Because of this cost, both in terms of time and money, it is critical that a good validation be sufficiently representative, providing similar levels of statistical power for assessing the accuracy of each group. In the context of somatic mutation calling, this means selecting calls that are unique to individual predictors as well as those common to multiple predictors. Indeed, the validation techniques will often be experimentally limited to a subset of results, leaving a bias in the distribution of what is tested. There is thus a clear need for research into the optimal selection of validation candidates in many biological settings. Further, validating a small subset (&lt;10%) of results comes with the possibility, however small, of producing an incorrect relative ordering of different algorithms. In practice, a combination of synthetic and real-world validation is best, and finding the right balance is challenge-dependent. </p> <p> Finally, some very important elements of cancer genomics are difficult to validate. For example, almost all NGS analyses rely on sequence alignment as a first step. It is, however, very difficult to benchmark the accuracy of an alignment algorithm on real tumor data, because there is no obvious way to create a ground-truth dataset. Thus, rather than benchmarking the aligners, challenges benchmark the results of entire pipelines such as those for detecting somatic variants [ <a aria-describedby=\"CR28\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR28\"> 28 </a> ], which may incorporate different aligners and different data pre-processing and statistical approaches. Similarly, it is of great interest to infer cancer-driver genes. Unfortunately, the definition of a ‘driver gene’ (beyond simple statistical recurrence) is unclear, and does not yet allow unambiguous, high-throughput experimental validation. Most experimental techniques in this area probe only one aspect of a driver gene (such as its influence on proliferation or metastasis), while many subtle phenotypes (such as angiogenesis or local spread) are challenging to probe. Also, these designs ignore the potentially polygenic nature of tumor initiation and progression. In designing a new challenge, one of the first questions must be whether or not suitable gold-standard test datasets can be generated. </p> </section> <section id=\"Sec6\"> <h2 class=\"pmc_sec_title\"> <a class=\"usa-anchor\" id=\"Sec6-anchor\"> </a> Closing considerations </h2> <p> Benchmarking is a fundamental part of computational biology and is increasingly being appreciated by the biomedical community as a whole. Recent benchmarking studies both within [ <a aria-describedby=\"CR19\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR19\"> 19 </a> , <a aria-describedby=\"CR51\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR51\"> 51 </a> ] and outside of cancer genomics [ <a aria-describedby=\"CR39\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR39\"> 39 </a> , <a aria-describedby=\"CR52\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR52\"> 52 </a> – <a aria-describedby=\"CR54\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR54\"> 54 </a> ] have helped highlight new ways to analyze data and have prompted reconsideration of the error profiles of datasets. Challenge-based assessments have also recently surged in other fields [ <a aria-describedby=\"CR55\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR55\"> 55 </a> ] in which the use of incentives (including prizes and prestige) have stimulated increased attention and algorithm development [ <a aria-describedby=\"CR56\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR56\"> 56 </a> ]. </p> <p> As the profile of the results of benchmarking studies increases, it is becoming increasingly clear that benchmarking itself is a serious scientific endeavor. The design of a challenge is non-trivial and in some ways is easy ‘to get wrong’ - there needs to be a careful integration between experts in challenge-based benchmarking and domain experts in the challenge topic. At the outset, there is a fundamental requirement for the benchmarking team to foster a community that supports and promotes the exercise. Indeed, some topic areas may be unsuitable to challenge-based benchmarking because a sufficiently big community of interested algorithm developers has not yet emerged (although in these cases, appropriate incentives may be useful in helping to focus attention on a potential challenge topic). Further, the challenge organizing team must be able to assure the broader community of its neutrality and objectivity. There is a clear advantage to building groups of ‘challenge-based benchmarking experts’ who can bring their expertise to diverse topics within cancer genomics, or any other field. Such groups may be well-placed to develop and optimize the statistical methods needed to improve challenge-based benchmarks. Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table <a class=\"usa-link\" href=\"#Tab1\"> 1 </a> ). </p> <p> Cancer genomics is characterized by rapid technological development, and this trend is likely to persist for many years. As a result, benchmarking cannot be a static endeavor. Rather, each new technology will have its own specific error profiles and distinct algorithms that are used for data analysis. In a world of continual technological and algorithmic innovation, it may be impossible to have definitive, permanent benchmarks, because any effort will be based on a snapshot of technology and will rapidly become out-dated. Instead, a long-running series of ‘living benchmarks’ may allow the co-evolution of benchmarks with technology. In this mutualistic scenario, regular releases of new datasets capturing the current state of experimental methodologies will allow users at any point in time to identify the best tool for their dataset, and algorithm developers to have a dataset suitable for developing and optimizing methods on the latest data. </p> </section> <section class=\"ack\" id=\"ack1\"> <h2 class=\"pmc_sec_title\"> <a class=\"usa-anchor\" id=\"ack1-anchor\"> </a> Acknowledgements </h2> <p> We are indebted to the DREAM community for teaching us how to run challenges, and to Sage Bionetworks for their contributions in organizing DREAM challenges. This study was conducted with the support of the Ontario Institute for Cancer Research through funding provided by the Government of Ontario (to PCB). This work was supported by Prostate Cancer Canada and is proudly funded by the Movember Foundation (Grant #RS2014-01). Dr Boutros was supported by a Terry Fox Research Institute New Investigator Award and a CIHR New Investigator Award. This project was supported by Genome Canada through a Large-Scale Applied Project contract. This work was also supported by the Discovery Frontiers: Advancing Big Data Science in Genomics Research program, which is jointly funded by the Natural Sciences and Engineering Research Council (NSERC) of Canada, the Canadian Institutes of Health Research (CIHR), Genome Canada, and the Canada Foundation for Innovation (CFI). NIH grants R01 CA180778 (JMS) and U24-CA143858 (JMS) supported this work. </p> </section> <section class=\"glossary\" id=\"glossary1\"> <h2 class=\"pmc_sec_title\"> <a class=\"usa-anchor\" id=\"glossary1-anchor\"> </a> Abbreviations </h2> <dl class=\"def-list\"> </dl> <dl class=\"def-list\"> <dt> CASP </dt> <dd> <p> Critical Assessment of Protein Structure Prediction </p> </dd> <dt> GCRMA </dt> <dd> <p> GC Robust Multi-Array </p> </dd> <dt> PHI </dt> <dd> <p> Personal health information </p> </dd> <dt> NGS </dt> <dd> <p> Next-generation sequencing </p> </dd> <dt> SMC </dt> <dd> <p> Somatic Mutation Calling </p> </dd> </dl> </section> <section class=\"fn-group\" id=\"fn-group1\"> <h2 class=\"pmc_sec_title\"> <a class=\"usa-anchor\" id=\"fn-group1-anchor\"> </a> Footnotes </h2> <div class=\"fn-group p font-secondary-light font-sm\"> <div class=\"fn p\" id=\"fn1\"> <p> <strong> Competing interests </strong> </p> <p> The authors declare that they have no competing interests. </p> </div> </div> </section> <section class=\"contrib-info\" id=\"_ci93_\" lang=\"en\"> <h2 class=\"pmc_sec_title\"> <a class=\"usa-anchor\" id=\"_ci93_-anchor\"> </a> Contributor Information </h2> <p> Paul C Boutros, Email: Paul.Boutros@oicr.on.ca. </p> <p> Adam A Margolin, Email: margolin@ohsu.edu. </p> <p> Joshua M Stuart, Email: jstuart@ucsc.edu. </p> <p> Andrea Califano, Email: ac2248@c2b2.columbia.edu. </p> <p> Gustavo Stolovitzky, Email: gustavo@us.ibm.com. </p> </section> <section class=\"ref-list\" id=\"Bib1\"> <h2 class=\"pmc_sec_title\"> <a class=\"usa-anchor\" id=\"Bib1-anchor\"> </a> References </h2> <section id=\"Bib1_sec2\"> <ul class=\"ref-list font-sm\"> <li id=\"CR1\"> <span class=\"label\"> 1. </span> <cite> Costello JC, Stolovitzky G. Seeking the wisdom of crowds through challenge-based competitions in biomedical research. Clin Pharmacol Ther. 2013;93:396–398. doi: 10.1038/clpt.2013.36. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/clpt.2013.36\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23549146/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&amp;title=Seeking%20the%20wisdom%20of%20crowds%20through%20challenge-based%20competitions%20in%20biomedical%20research&amp;author=JC%20Costello&amp;author=G%20Stolovitzky&amp;volume=93&amp;publication_year=2013&amp;pages=396-398&amp;pmid=23549146&amp;doi=10.1038/clpt.2013.36&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR2\"> <span class=\"label\"> 2. </span> <cite> Meyer P, Alexopoulos LG, Bonk T, Califano A, Cho CR, de la Fuente A, de Graaf D, Hartemink AJ, Hoeng J, Ivanov NV, Koeppl H, Linding R, Marbach D, Norel R, Peitsch MC, Rice JJ, Royyuru A, Schacherer F, Sprengel J, Stolle K, Vitkup D, Stolovitzky G. Verification of systems biology research in the age of collaborative competition. Nat Biotechnol. 2011;29:811–815. doi: 10.1038/nbt.1968. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nbt.1968\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/21904331/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&amp;title=Verification%20of%20systems%20biology%20research%20in%20the%20age%20of%20collaborative%20competition&amp;author=P%20Meyer&amp;author=LG%20Alexopoulos&amp;author=T%20Bonk&amp;author=A%20Califano&amp;author=CR%20Cho&amp;volume=29&amp;publication_year=2011&amp;pages=811-815&amp;pmid=21904331&amp;doi=10.1038/nbt.1968&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR3\"> <span class=\"label\"> 3. </span> <cite> Pertea M, Salzberg SL. Between a chicken and a grape: estimating the number of human genes. Genome Biol. 2010;11:206. doi: 10.1186/gb-2010-11-5-206. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/gb-2010-11-5-206\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2898077/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/20441615/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=Between%20a%20chicken%20and%20a%20grape:%20estimating%20the%20number%20of%20human%20genes&amp;author=M%20Pertea&amp;author=SL%20Salzberg&amp;volume=11&amp;publication_year=2010&amp;pages=206&amp;pmid=20441615&amp;doi=10.1186/gb-2010-11-5-206&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR4\"> <span class=\"label\"> 4. </span> <cite> Lim WK, Wang K, Lefebvre C, Califano A. Comparative analysis of microarray normalization procedures: effects on reverse engineering gene networks. Bioinformatics. 2007;23:i282–i288. doi: 10.1093/bioinformatics/btm201. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/bioinformatics/btm201\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/17646307/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Comparative%20analysis%20of%20microarray%20normalization%20procedures:%20effects%20on%20reverse%20engineering%20gene%20networks&amp;author=WK%20Lim&amp;author=K%20Wang&amp;author=C%20Lefebvre&amp;author=A%20Califano&amp;volume=23&amp;publication_year=2007&amp;pages=i282-i288&amp;pmid=17646307&amp;doi=10.1093/bioinformatics/btm201&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR5\"> <span class=\"label\"> 5. </span> <cite> Baggerly KA, Coombes KR. Deriving chemosensitivity from cell lines: forensic bioinformatics and reproducible research in high-throughput biology. Ann Appl Stat. 2009;3:1309–1334. doi: 10.1214/09-AOAS291. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1214/09-AOAS291\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Ann%20Appl%20Stat&amp;title=Deriving%20chemosensitivity%20from%20cell%20lines:%20forensic%20bioinformatics%20and%20reproducible%20research%20in%20high-throughput%20biology&amp;author=KA%20Baggerly&amp;author=KR%20Coombes&amp;volume=3&amp;publication_year=2009&amp;pages=1309-1334&amp;doi=10.1214/09-AOAS291&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR6\"> <span class=\"label\"> 6. </span> <cite> Norel R, Rice JJ, Stolovitzky G. The self-assessment trap: can we all be better than average? Mol Syst Biol. 2011;7:537. doi: 10.1038/msb.2011.70. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/msb.2011.70\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3261704/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/21988833/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Mol%20Syst%20Biol&amp;title=The%20self-assessment%20trap:%20can%20we%20all%20be%20better%20than%20average?&amp;author=R%20Norel&amp;author=JJ%20Rice&amp;author=G%20Stolovitzky&amp;volume=7&amp;publication_year=2011&amp;pages=537&amp;pmid=21988833&amp;doi=10.1038/msb.2011.70&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR7\"> <span class=\"label\"> 7. </span> <cite> Moult J, Pedersen JT, Judson R, Fidelis K. A large-scale experiment to assess protein structure prediction methods. Proteins. 1995;23:ii–v. doi: 10.1002/prot.340230303. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1002/prot.340230303\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/8710822/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Proteins&amp;title=A%20large-scale%20experiment%20to%20assess%20protein%20structure%20prediction%20methods&amp;author=J%20Moult&amp;author=JT%20Pedersen&amp;author=R%20Judson&amp;author=K%20Fidelis&amp;volume=23&amp;publication_year=1995&amp;pages=ii-v&amp;pmid=8710822&amp;doi=10.1002/prot.340230303&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR8\"> <span class=\"label\"> 8. </span> <cite> Cozzetto D, Kryshtafovych A, Tramontano A. Evaluation of CASP8 model quality predictions. Proteins. 2009;77:157–166. doi: 10.1002/prot.22534. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1002/prot.22534\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19714774/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Proteins&amp;title=Evaluation%20of%20CASP8%20model%20quality%20predictions&amp;author=D%20Cozzetto&amp;author=A%20Kryshtafovych&amp;author=A%20Tramontano&amp;volume=77&amp;publication_year=2009&amp;pages=157-166&amp;pmid=19714774&amp;doi=10.1002/prot.22534&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR9\"> <span class=\"label\"> 9. </span> <cite> Shi S, Pei J, Sadreyev RI, Kinch LN, Majumdar I, Tong J, Cheng H, Kim BH, Grishin NV. Analysis of CASP8 targets, predictions and assessment methods. Database (Oxford) 2009;2009:bap003. doi: 10.1093/database/bap003. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/database/bap003\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2794793/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/20157476/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Database%20(Oxford)&amp;title=Analysis%20of%20CASP8%20targets,%20predictions%20and%20assessment%20methods&amp;author=S%20Shi&amp;author=J%20Pei&amp;author=RI%20Sadreyev&amp;author=LN%20Kinch&amp;author=I%20Majumdar&amp;volume=2009&amp;publication_year=2009&amp;pages=bap003&amp;pmid=20157476&amp;doi=10.1093/database/bap003&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR10\"> <span class=\"label\"> 10. </span> <cite> Tramontano A, Morea V. Assessment of homology-based predictions in CASP5. Proteins. 2004;55:782. doi: 10.1002/prot.20187. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1002/prot.20187\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/14579324/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Proteins&amp;title=Assessment%20of%20homology-based%20predictions%20in%20CASP5&amp;author=A%20Tramontano&amp;author=V%20Morea&amp;volume=55&amp;publication_year=2004&amp;pages=782&amp;pmid=14579324&amp;doi=10.1002/prot.20187&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR11\"> <span class=\"label\"> 11. </span> <cite> Radivojac P, Clark WT, Oron TR, Schnoes AM, Wittkop T, Sokolov A, Graim K, Funk C, Verspoor K, Ben-Hur A, Pandey G, Yunes JM, Talwalkar AS, Repo S, Souza ML, Piovesan D, Casadio R, Wang Z, Cheng J, Fang H, Gough J, Koskinen P, Törönen P, Nokso-Koivisto J, Holm L, Cozzetto D, Buchan DW, Bryson K, Jones DT, Limaye B, et al. A large-scale evaluation of computational protein function prediction. Nat Methods. 2013;10:221–227. doi: 10.1038/nmeth.2340. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nmeth.2340\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3584181/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23353650/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&amp;title=A%20large-scale%20evaluation%20of%20computational%20protein%20function%20prediction&amp;author=P%20Radivojac&amp;author=WT%20Clark&amp;author=TR%20Oron&amp;author=AM%20Schnoes&amp;author=T%20Wittkop&amp;volume=10&amp;publication_year=2013&amp;pages=221-227&amp;pmid=23353650&amp;doi=10.1038/nmeth.2340&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR12\"> <span class=\"label\"> 12. </span> <cite> Prill RJ, Marbach D, Saez-Rodriguez J, Sorger PK, Alexopoulos LG, Xue X, Clarke ND, Altan-Bonnet G, Stolovitzky G. Towards a rigorous assessment of systems biology models: the DREAM3 challenges. PLoS One. 2010;5:e9202. doi: 10.1371/journal.pone.0009202. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1371/journal.pone.0009202\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2826397/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/20186320/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Towards%20a%20rigorous%20assessment%20of%20systems%20biology%20models:%20the%20DREAM3%20challenges&amp;author=RJ%20Prill&amp;author=D%20Marbach&amp;author=J%20Saez-Rodriguez&amp;author=PK%20Sorger&amp;author=LG%20Alexopoulos&amp;volume=5&amp;publication_year=2010&amp;pages=e9202&amp;pmid=20186320&amp;doi=10.1371/journal.pone.0009202&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR13\"> <span class=\"label\"> 13. </span> <cite> Stolovitzky G, Prill RJ, Califano A. Lessons from the DREAM2 challenges. Ann N Y Acad Sci. 2009;1158:159–195. doi: 10.1111/j.1749-6632.2009.04497.x. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1111/j.1749-6632.2009.04497.x\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19348640/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Ann%20N%20Y%20Acad%20Sci&amp;title=Lessons%20from%20the%20DREAM2%20challenges&amp;author=G%20Stolovitzky&amp;author=RJ%20Prill&amp;author=A%20Califano&amp;volume=1158&amp;publication_year=2009&amp;pages=159-195&amp;pmid=19348640&amp;doi=10.1111/j.1749-6632.2009.04497.x&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR14\"> <span class=\"label\"> 14. </span> <cite> Athanasopoulos G, Hyndman RJ. The value of feedback in forecasting competitions. Int J Forecast. 2011;27:845–849. doi: 10.1016/j.ijforecast.2011.03.002. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.ijforecast.2011.03.002\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Int%20J%20Forecast&amp;title=The%20value%20of%20feedback%20in%20forecasting%20competitions&amp;author=G%20Athanasopoulos&amp;author=RJ%20Hyndman&amp;volume=27&amp;publication_year=2011&amp;pages=845-849&amp;doi=10.1016/j.ijforecast.2011.03.002&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR15\"> <span class=\"label\"> 15. </span> <cite> Margolin AA, Bilal E, Huang E, Norman TC, Ottestad L, Mecham BH, Sauerwine B, Kellen MR, Mangravite LM, Furia MD, Vollan HK, Rueda OM, Guinney J, Deflaux NA, Hoff B, Schildwachter X, Russnes HG, Park D, Vang VO, Pirtle T, Youseff L, Citro C, Curtis C, Kristensen VN, Hellerstein J, Friend SH, Stolovitzky G, Aparicio S, Caldas C, Børresen-Dale AL. Systematic analysis of challenge-driven improvements in molecular prognostic models for breast cancer. Sci Transl Med. 2013;5:181re181. doi: 10.1126/scitranslmed.3006112. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1126/scitranslmed.3006112\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3897241/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23596205/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Sci%20Transl%20Med&amp;title=Systematic%20analysis%20of%20challenge-driven%20improvements%20in%20molecular%20prognostic%20models%20for%20breast%20cancer&amp;author=AA%20Margolin&amp;author=E%20Bilal&amp;author=E%20Huang&amp;author=TC%20Norman&amp;author=L%20Ottestad&amp;volume=5&amp;publication_year=2013&amp;pages=181re181&amp;pmid=23596205&amp;doi=10.1126/scitranslmed.3006112&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR16\"> <span class=\"label\"> 16. </span> <cite> <em> Synapse </em> ; [ <a class=\"usa-link usa-link--external\" href=\"http://www.sagebase.org/synapse\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.sagebase.org/synapse </a> ] </cite> </li> <li id=\"CR17\"> <span class=\"label\"> 17. </span> <cite> Omberg L, Ellrott K, Yuan Y, Kandoth C, Wong C, Kellen MR, Friend SH, Stuart J, Liang H, Margolin AA. Enabling transparent and collaborative computational analysis of 12 tumor types within The Cancer Genome Atlas. Nat Genet. 2013;45:1121–1126. doi: 10.1038/ng.2761. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/ng.2761\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3950337/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24071850/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&amp;title=Enabling%20transparent%20and%20collaborative%20computational%20analysis%20of%2012%20tumor%20types%20within%20The%20Cancer%20Genome%20Atlas&amp;author=L%20Omberg&amp;author=K%20Ellrott&amp;author=Y%20Yuan&amp;author=C%20Kandoth&amp;author=C%20Wong&amp;volume=45&amp;publication_year=2013&amp;pages=1121-1126&amp;pmid=24071850&amp;doi=10.1038/ng.2761&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR18\"> <span class=\"label\"> 18. </span> <cite> Global Alliance for Genomics and Health ; [ <a class=\"usa-link usa-link--external\" href=\"http://genomicsandhealth.org\" rel=\"noopener noreferrer\" target=\"_blank\"> http://genomicsandhealth.org </a> ] </cite> </li> <li id=\"CR19\"> <span class=\"label\"> 19. </span> <cite> Kim SY, Speed TP. Comparing somatic mutation-callers: beyond Venn diagrams. BMC Bioinformatics. 2013;14:189. doi: 10.1186/1471-2105-14-189. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/1471-2105-14-189\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3702398/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23758877/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&amp;title=Comparing%20somatic%20mutation-callers:%20beyond%20Venn%20diagrams&amp;author=SY%20Kim&amp;author=TP%20Speed&amp;volume=14&amp;publication_year=2013&amp;pages=189&amp;pmid=23758877&amp;doi=10.1186/1471-2105-14-189&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR20\"> <span class=\"label\"> 20. </span> <cite> O’Rawe J, Jiang T, Sun G, Wu Y, Wang W, Hu J, Bodily P, Tian L, Hakonarson H, Johnson WE, Wei Z, Wang K, Lyon GJ. Low concordance of multiple variant-calling pipelines: practical implications for exome and genome sequencing. Genome Med. 2013;5:28. doi: 10.1186/gm432. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/gm432\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3706896/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23537139/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Genome%20Med&amp;title=Low%20concordance%20of%20multiple%20variant-calling%20pipelines:%20practical%20implications%20for%20exome%20and%20genome%20sequencing&amp;author=J%20O%E2%80%99Rawe&amp;author=T%20Jiang&amp;author=G%20Sun&amp;author=Y%20Wu&amp;author=W%20Wang&amp;volume=5&amp;publication_year=2013&amp;pages=28&amp;pmid=23537139&amp;doi=10.1186/gm432&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR21\"> <span class=\"label\"> 21. </span> <cite> Alkan C, Coe BP, Eichler EE. Genome structural variation discovery and genotyping. Nat Rev Genet. 2011;12:363–376. doi: 10.1038/nrg2958. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nrg2958\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4108431/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/21358748/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Genet&amp;title=Genome%20structural%20variation%20discovery%20and%20genotyping&amp;author=C%20Alkan&amp;author=BP%20Coe&amp;author=EE%20Eichler&amp;volume=12&amp;publication_year=2011&amp;pages=363-376&amp;pmid=21358748&amp;doi=10.1038/nrg2958&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR22\"> <span class=\"label\"> 22. </span> <cite> <strong> Taking pan-cancer analysis global. </strong> <em> Nat Genet </em> 2013, <strong> 45: </strong> 1263. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/ng.2825\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24165723/\"> PubMed </a> ] </li> <li id=\"CR23\"> <span class=\"label\"> 23. </span> <cite> Bradnam KR, Fass JN, Alexandrov A, Baranay P, Bechner M, Birol I, Boisvert S, Chapman JA, Chapuis G, Chikhi R, Chitsaz H, Chou WC, Corbeil J, Del Fabbro C, Docking TR, Durbin R, Earl D, Emrich S, Fedotov P, Fonseca NA, Ganapathy G, Gibbs RA, Gnerre S, Godzaridis E, Goldstein S, Haimel M, Hall G, Haussler D, Hiatt JB, Ho IY, et al. Assemblathon 2: evaluating de novo methods of genome assembly in three vertebrate species. Gigascience. 2013;2:10. doi: 10.1186/2047-217X-2-10. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/2047-217X-2-10\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3844414/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23870653/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Gigascience&amp;title=Assemblathon%202:%20evaluating%20de%20novo%20methods%20of%20genome%20assembly%20in%20three%20vertebrate%20species&amp;author=KR%20Bradnam&amp;author=JN%20Fass&amp;author=A%20Alexandrov&amp;author=P%20Baranay&amp;author=M%20Bechner&amp;volume=2&amp;publication_year=2013&amp;pages=10&amp;pmid=23870653&amp;doi=10.1186/2047-217X-2-10&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR24\"> <span class=\"label\"> 24. </span> <cite> Brownstein CA, Beggs AH, Homer N, Merriman B, Yu TW, Flannery KC, Dechene ET, Towne MC, Savage SK, Price EN, Holm IA, Luquette LJ, Lyon E, Majzoub J, Neupert P, McCallie D, Jr, Szolovits P, Willard HF, Mendelsohn NJ, Temme R, Finkel RS, Yum SW, Medne L, Sunyaev SR, Adzhubey I, Cassa CA, de Bakker PI, Duzkale H, Dworzy Ski P, Fairbrother W, et al. An international effort towards developing standards for best practices in analysis, interpretation and reporting of clinical genome sequencing results in the CLARITY challenge. Genome Biol. 2014;15:R53. doi: 10.1186/gb-2014-15-3-r53. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/gb-2014-15-3-r53\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4073084/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24667040/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=An%20international%20effort%20towards%20developing%20standards%20for%20best%20practices%20in%20analysis,%20interpretation%20and%20reporting%20of%20clinical%20genome%20sequencing%20results%20in%20the%20CLARITY%20challenge&amp;author=CA%20Brownstein&amp;author=AH%20Beggs&amp;author=N%20Homer&amp;author=B%20Merriman&amp;author=TW%20Yu&amp;volume=15&amp;publication_year=2014&amp;pages=R53&amp;pmid=24667040&amp;doi=10.1186/gb-2014-15-3-r53&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR25\"> <span class=\"label\"> 25. </span> <cite> Gymrek M, McGuire AL, Golan D, Halperin E, Erlich Y. Identifying personal genomes by surname inference. Science. 2013;339:321–324. doi: 10.1126/science.1229566. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1126/science.1229566\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23329047/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Identifying%20personal%20genomes%20by%20surname%20inference&amp;author=M%20Gymrek&amp;author=AL%20McGuire&amp;author=D%20Golan&amp;author=E%20Halperin&amp;author=Y%20Erlich&amp;volume=339&amp;publication_year=2013&amp;pages=321-324&amp;pmid=23329047&amp;doi=10.1126/science.1229566&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR26\"> <span class=\"label\"> 26. </span> <cite> Good BM, Su AI. Games with a scientific purpose. Genome Biol. 2011;12:135. doi: 10.1186/gb-2011-12-12-135. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/gb-2011-12-12-135\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3334605/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/22204700/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=Games%20with%20a%20scientific%20purpose&amp;author=BM%20Good&amp;author=AI%20Su&amp;volume=12&amp;publication_year=2011&amp;pages=135&amp;pmid=22204700&amp;doi=10.1186/gb-2011-12-12-135&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR27\"> <span class=\"label\"> 27. </span> <cite> Lee J, Kladwang W, Lee M, Cantu D, Azizyan M, Kim H, Limpaecher A, Yoon S, Treuille A, Das R, Ete RNAP. RNA design rules from a massive open laboratory. Proc Natl Acad Sci U S A. 2014;111:2122–2127. doi: 10.1073/pnas.1313039111. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1073/pnas.1313039111\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3926058/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24469816/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=RNA%20design%20rules%20from%20a%20massive%20open%20laboratory&amp;author=J%20Lee&amp;author=W%20Kladwang&amp;author=M%20Lee&amp;author=D%20Cantu&amp;author=M%20Azizyan&amp;volume=111&amp;publication_year=2014&amp;pages=2122-2127&amp;pmid=24469816&amp;doi=10.1073/pnas.1313039111&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR28\"> <span class=\"label\"> 28. </span> <cite> Boutros PC, Ewing AD, Ellrott K, Norman TC, Dang KK, Hu Y, Kellen MR, Suver C, Bare JC, Stein LD, Spellman PT, Stolovitzky G, Friend SH, Margolin AA, Stuart JM. Global optimization of somatic variant identification in cancer genomes with a global community challenge. Nat Genet. 2014;46:318–319. doi: 10.1038/ng.2932. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/ng.2932\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4035501/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24675517/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&amp;title=Global%20optimization%20of%20somatic%20variant%20identification%20in%20cancer%20genomes%20with%20a%20global%20community%20challenge&amp;author=PC%20Boutros&amp;author=AD%20Ewing&amp;author=K%20Ellrott&amp;author=TC%20Norman&amp;author=KK%20Dang&amp;volume=46&amp;publication_year=2014&amp;pages=318-319&amp;pmid=24675517&amp;doi=10.1038/ng.2932&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR29\"> <span class=\"label\"> 29. </span> <cite> Dudley JT, Butte AJ. In silico research in the era of cloud computing. Nat Biotechnol. 2010;28:1181–1185. doi: 10.1038/nbt1110-1181. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nbt1110-1181\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3755123/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/21057489/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&amp;title=In%20silico%20research%20in%20the%20era%20of%20cloud%20computing&amp;author=JT%20Dudley&amp;author=AJ%20Butte&amp;volume=28&amp;publication_year=2010&amp;pages=1181-1185&amp;pmid=21057489&amp;doi=10.1038/nbt1110-1181&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR30\"> <span class=\"label\"> 30. </span> <cite> Lambin P, van Stiphout RG, Starmans MH, Rios-Velazquez E, Nalbantov G, Aerts HJ, Roelofs E, van Elmpt W, Boutros PC, Granone P, Valentini V, Begg AC, De Ruysscher D, Dekker A. Predicting outcomes in radiation oncology - multifactorial decision support systems. Nat Rev Clin Oncol. 2013;10:27–40. doi: 10.1038/nrclinonc.2012.196. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nrclinonc.2012.196\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4555846/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23165123/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Clin%20Oncol&amp;title=Predicting%20outcomes%20in%20radiation%20oncology%20-%20multifactorial%20decision%20support%20systems&amp;author=P%20Lambin&amp;author=RG%20van%20Stiphout&amp;author=MH%20Starmans&amp;author=E%20Rios-Velazquez&amp;author=G%20Nalbantov&amp;volume=10&amp;publication_year=2013&amp;pages=27-40&amp;pmid=23165123&amp;doi=10.1038/nrclinonc.2012.196&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR31\"> <span class=\"label\"> 31. </span> <cite> Chin L, Gray JW. Translating insights from the cancer genome into clinical practice. Nature. 2008;452:553–563. doi: 10.1038/nature06914. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nature06914\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2730524/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/18385729/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Translating%20insights%20from%20the%20cancer%20genome%20into%20clinical%20practice&amp;author=L%20Chin&amp;author=JW%20Gray&amp;volume=452&amp;publication_year=2008&amp;pages=553-563&amp;pmid=18385729&amp;doi=10.1038/nature06914&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR32\"> <span class=\"label\"> 32. </span> <cite> Khleif SN, Doroshow JH, Hait WN. AACR-FDA-NCI Cancer Biomarkers Collaborative consensus report: advancing the use of biomarkers in cancer drug development. Clin Cancer Res. 2010;16:3299–3318. doi: 10.1158/1078-0432.CCR-10-0880. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1158/1078-0432.CCR-10-0880\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/20501613/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&amp;title=AACR-FDA-NCI%20Cancer%20Biomarkers%20Collaborative%20consensus%20report:%20advancing%20the%20use%20of%20biomarkers%20in%20cancer%20drug%20development&amp;author=SN%20Khleif&amp;author=JH%20Doroshow&amp;author=WN%20Hait&amp;volume=16&amp;publication_year=2010&amp;pages=3299-3318&amp;pmid=20501613&amp;doi=10.1158/1078-0432.CCR-10-0880&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR33\"> <span class=\"label\"> 33. </span> <cite> van’t Veer LJ, Bernards R. Enabling personalized cancer medicine through analysis of gene-expression patterns. Nature. 2008;452:564–570. doi: 10.1038/nature06915. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nature06915\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/18385730/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Enabling%20personalized%20cancer%20medicine%20through%20analysis%20of%20gene-expression%20patterns&amp;author=LJ%20van%E2%80%99t%20Veer&amp;author=R%20Bernards&amp;volume=452&amp;publication_year=2008&amp;pages=564-570&amp;pmid=18385730&amp;doi=10.1038/nature06915&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR34\"> <span class=\"label\"> 34. </span> <cite> Starmans MH, Pintilie M, John T, Der SD, Shepherd FA, Jurisica I, Lambin P, Tsao MS, Boutros PC. Exploiting the noise: improving biomarkers with ensembles of data analysis methodologies. Genome Med. 2012;4:84. doi: 10.1186/gm385. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/gm385\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3580418/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23146350/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Genome%20Med&amp;title=Exploiting%20the%20noise:%20improving%20biomarkers%20with%20ensembles%20of%20data%20analysis%20methodologies&amp;author=MH%20Starmans&amp;author=M%20Pintilie&amp;author=T%20John&amp;author=SD%20Der&amp;author=FA%20Shepherd&amp;volume=4&amp;publication_year=2012&amp;pages=84&amp;pmid=23146350&amp;doi=10.1186/gm385&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR35\"> <span class=\"label\"> 35. </span> <cite> Starmans MH, Fung G, Steck H, Wouters BG, Lambin P. A simple but highly effective approach to evaluate the prognostic performance of gene expression signatures. PLoS One. 2011;6:e28320. doi: 10.1371/journal.pone.0028320. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1371/journal.pone.0028320\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3233554/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/22163293/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=A%20simple%20but%20highly%20effective%20approach%20to%20evaluate%20the%20prognostic%20performance%20of%20gene%20expression%20signatures&amp;author=MH%20Starmans&amp;author=G%20Fung&amp;author=H%20Steck&amp;author=BG%20Wouters&amp;author=P%20Lambin&amp;volume=6&amp;publication_year=2011&amp;pages=e28320&amp;pmid=22163293&amp;doi=10.1371/journal.pone.0028320&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR36\"> <span class=\"label\"> 36. </span> <cite> Venet D, Dumont JE, Detours V. Most random gene expression signatures are significantly associated with breast cancer outcome. PLoS Comput Biol. 2011;7:e1002240. doi: 10.1371/journal.pcbi.1002240. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1371/journal.pcbi.1002240\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3197658/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/22028643/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=PLoS%20Comput%20Biol&amp;title=Most%20random%20gene%20expression%20signatures%20are%20significantly%20associated%20with%20breast%20cancer%20outcome&amp;author=D%20Venet&amp;author=JE%20Dumont&amp;author=V%20Detours&amp;volume=7&amp;publication_year=2011&amp;pages=e1002240&amp;pmid=22028643&amp;doi=10.1371/journal.pcbi.1002240&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR37\"> <span class=\"label\"> 37. </span> <cite> Boutros PC, Lau SK, Pintilie M, Liu N, Shepherd FA, Der SD, Tsao MS, Penn LZ, Jurisica I. Prognostic gene signatures for non-small-cell lung cancer. Proc Natl Acad Sci U S A. 2009;106:2824–2828. doi: 10.1073/pnas.0809444106. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1073/pnas.0809444106\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2636731/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19196983/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=Prognostic%20gene%20signatures%20for%20non-small-cell%20lung%20cancer&amp;author=PC%20Boutros&amp;author=SK%20Lau&amp;author=M%20Pintilie&amp;author=N%20Liu&amp;author=FA%20Shepherd&amp;volume=106&amp;publication_year=2009&amp;pages=2824-2828&amp;pmid=19196983&amp;doi=10.1073/pnas.0809444106&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR38\"> <span class=\"label\"> 38. </span> <cite> Bentzien J, Muegge I, Hamner B, Thompson DC. Crowd computing: using competitive dynamics to develop and refine highly predictive models. Drug Discov Today. 2013;18:472–478. doi: 10.1016/j.drudis.2013.01.002. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.drudis.2013.01.002\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23337388/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Drug%20Discov%20Today&amp;title=Crowd%20computing:%20using%20competitive%20dynamics%20to%20develop%20and%20refine%20highly%20predictive%20models&amp;author=J%20Bentzien&amp;author=I%20Muegge&amp;author=B%20Hamner&amp;author=DC%20Thompson&amp;volume=18&amp;publication_year=2013&amp;pages=472-478&amp;pmid=23337388&amp;doi=10.1016/j.drudis.2013.01.002&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR39\"> <span class=\"label\"> 39. </span> <cite> Shi L, Campbell G, Jones WD, Campagne F, Wen Z, Walker SJ, Su Z, Chu TM, Goodsaid FM, Pusztai L, Shaughnessy JD, Jr, Oberthuer A, Thomas RS, Paules RS, Fielden M, Barlogie B, Chen W, Du P, Fischer M, Furlanello C, Gallas BD, Ge X, Megherbi DB, Symmans WF, Wang MD, Zhang J, Bitter H, Brors B, Bushel PR, Bylesjo M, et al. The MicroArray Quality Control (MAQC)-II study of common practices for the development and validation of microarray-based predictive models. Nat Biotechnol. 2010;28:827–838. doi: 10.1038/nbt.1665. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nbt.1665\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3315840/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/20676074/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&amp;title=The%20MicroArray%20Quality%20Control%20(MAQC)-II%20study%20of%20common%20practices%20for%20the%20development%20and%20validation%20of%20microarray-based%20predictive%20models&amp;author=L%20Shi&amp;author=G%20Campbell&amp;author=WD%20Jones&amp;author=F%20Campagne&amp;author=Z%20Wen&amp;volume=28&amp;publication_year=2010&amp;pages=827-838&amp;pmid=20676074&amp;doi=10.1038/nbt.1665&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR40\"> <span class=\"label\"> 40. </span> <cite> Tarca AL, Lauria M, Unger M, Bilal E, Boue S, Kumar Dey K, Hoeng J, Koeppl H, Martin F, Meyer P, Nandy P, Norel R, Peitsch M, Rice JJ, Romero R, Stolovitzky G, Talikka M, Xiang Y, Zechner C, IMPROVER DSC Collaborators Strengths and limitations of microarray-based phenotype prediction: lessons learned from the IMPROVER Diagnostic Signature Challenge. Bioinformatics. 2013;29:2892–2899. doi: 10.1093/bioinformatics/btt492. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/bioinformatics/btt492\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3810846/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23966112/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Strengths%20and%20limitations%20of%20microarray-based%20phenotype%20prediction:%20lessons%20learned%20from%20the%20IMPROVER%20Diagnostic%20Signature%20Challenge&amp;author=AL%20Tarca&amp;author=M%20Lauria&amp;author=M%20Unger&amp;author=E%20Bilal&amp;author=S%20Boue&amp;volume=29&amp;publication_year=2013&amp;pages=2892-2899&amp;pmid=23966112&amp;doi=10.1093/bioinformatics/btt492&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR41\"> <span class=\"label\"> 41. </span> <cite> <em> Acute Myeloid Leukemia Outcome Prediction Challenge </em> ; [ <a class=\"usa-link usa-link--external\" href=\"https://www.synapse.org/#!Synapse:syn2455683\" rel=\"noopener noreferrer\" target=\"_blank\"> https://www.synapse.org/#!Synapse:syn2455683 </a> ] </cite> </li> <li id=\"CR42\"> <span class=\"label\"> 42. </span> <cite> Pujana MA, Han JD, Starita LM, Stevens KN, Tewari M, Ahn JS, Rennert G, Moreno V, Kirchhoff T, Gold B, Assmann V, Elshamy WM, Rual JF, Levine D, Rozek LS, Gelman RS, Gunsalus KC, Greenberg RA, Sobhian B, Bertin N, Venkatesan K, Ayivi-Guedehoussou N, Solé X, Hernández P, Lázaro C, Nathanson KL, Weber BL, Cusick ME, Hill DE, Offit K, et al. Network modeling links breast cancer susceptibility and centrosome dysfunction. Nat Genet. 2007;39:1338–1349. doi: 10.1038/ng.2007.2. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/ng.2007.2\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/17922014/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&amp;title=Network%20modeling%20links%20breast%20cancer%20susceptibility%20and%20centrosome%20dysfunction&amp;author=MA%20Pujana&amp;author=JD%20Han&amp;author=LM%20Starita&amp;author=KN%20Stevens&amp;author=M%20Tewari&amp;volume=39&amp;publication_year=2007&amp;pages=1338-1349&amp;pmid=17922014&amp;doi=10.1038/ng.2007.2&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR43\"> <span class=\"label\"> 43. </span> <cite> Taylor IW, Linding R, Warde-Farley D, Liu Y, Pesquita C, Faria D, Bull S, Pawson T, Morris Q, Wrana JL. Dynamic modularity in protein interaction networks predicts breast cancer outcome. Nat Biotechnol. 2009;27:199–204. doi: 10.1038/nbt.1522. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nbt.1522\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19182785/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&amp;title=Dynamic%20modularity%20in%20protein%20interaction%20networks%20predicts%20breast%20cancer%20outcome&amp;author=IW%20Taylor&amp;author=R%20Linding&amp;author=D%20Warde-Farley&amp;author=Y%20Liu&amp;author=C%20Pesquita&amp;volume=27&amp;publication_year=2009&amp;pages=199-204&amp;pmid=19182785&amp;doi=10.1038/nbt.1522&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR44\"> <span class=\"label\"> 44. </span> <cite> Vaske CJ, Benz SC, Sanborn JZ, Earl D, Szeto C, Zhu J, Haussler D, Stuart JM. Inference of patient-specific pathway activities from multi-dimensional cancer genomics data using PARADIGM. Bioinformatics. 2010;26:i237–i245. doi: 10.1093/bioinformatics/btq182. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/bioinformatics/btq182\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2881367/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/20529912/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Inference%20of%20patient-specific%20pathway%20activities%20from%20multi-dimensional%20cancer%20genomics%20data%20using%20PARADIGM&amp;author=CJ%20Vaske&amp;author=SC%20Benz&amp;author=JZ%20Sanborn&amp;author=D%20Earl&amp;author=C%20Szeto&amp;volume=26&amp;publication_year=2010&amp;pages=i237-i245&amp;pmid=20529912&amp;doi=10.1093/bioinformatics/btq182&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR45\"> <span class=\"label\"> 45. </span> <cite> Marbach D, Prill RJ, Schaffter T, Mattiussi C, Floreano D, Stolovitzky G. Revealing strengths and weaknesses of methods for gene network inference. Proc Natl Acad Sci U S A. 2010;107:6286–6291. doi: 10.1073/pnas.0913357107. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1073/pnas.0913357107\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2851985/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/20308593/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=Revealing%20strengths%20and%20weaknesses%20of%20methods%20for%20gene%20network%20inference&amp;author=D%20Marbach&amp;author=RJ%20Prill&amp;author=T%20Schaffter&amp;author=C%20Mattiussi&amp;author=D%20Floreano&amp;volume=107&amp;publication_year=2010&amp;pages=6286-6291&amp;pmid=20308593&amp;doi=10.1073/pnas.0913357107&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR46\"> <span class=\"label\"> 46. </span> <cite> Marbach D, Costello JC, Kuffner R, Vega NM, Prill RJ, Camacho DM, Allison KR, Consortium D, Kellis M, Collins JJ, Stolovitzky G. Wisdom of crowds for robust gene network inference. Nat Methods. 2012;9:796–804. doi: 10.1038/nmeth.2016. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nmeth.2016\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3512113/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/22796662/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&amp;title=Wisdom%20of%20crowds%20for%20robust%20gene%20network%20inference&amp;author=D%20Marbach&amp;author=JC%20Costello&amp;author=R%20Kuffner&amp;author=NM%20Vega&amp;author=RJ%20Prill&amp;volume=9&amp;publication_year=2012&amp;pages=796-804&amp;pmid=22796662&amp;doi=10.1038/nmeth.2016&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR47\"> <span class=\"label\"> 47. </span> <cite> Prill RJ, Saez-Rodriguez J, Alexopoulos LG, Sorger PK, Stolovitzky G. Crowdsourcing network inference: the DREAM predictive signaling network challenge. Sci Signal. 2011;4:mr7. doi: 10.1126/scisignal.2002212. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1126/scisignal.2002212\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3465072/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/21900204/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Sci%20Signal&amp;title=Crowdsourcing%20network%20inference:%20the%20DREAM%20predictive%20signaling%20network%20challenge&amp;author=RJ%20Prill&amp;author=J%20Saez-Rodriguez&amp;author=LG%20Alexopoulos&amp;author=PK%20Sorger&amp;author=G%20Stolovitzky&amp;volume=4&amp;publication_year=2011&amp;pages=mr7&amp;pmid=21900204&amp;doi=10.1126/scisignal.2002212&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR48\"> <span class=\"label\"> 48. </span> <cite> <em> HPN-DREAM breast cancer network inference challenge </em> ; [ <a class=\"usa-link usa-link--external\" href=\"https://www.synapse.org/#!Synapse:syn1720047\" rel=\"noopener noreferrer\" target=\"_blank\"> https://www.synapse.org/#!Synapse:syn1720047 </a> ] </cite> </li> <li id=\"CR49\"> <span class=\"label\"> 49. </span> <cite> Maier R, Zimmer R, Kuffner R. A Turing test for artificial expression data. Bioinformatics. 2013;29:2603–2609. doi: 10.1093/bioinformatics/btt438. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/bioinformatics/btt438\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23956305/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=A%20Turing%20test%20for%20artificial%20expression%20data&amp;author=R%20Maier&amp;author=R%20Zimmer&amp;author=R%20Kuffner&amp;volume=29&amp;publication_year=2013&amp;pages=2603-2609&amp;pmid=23956305&amp;doi=10.1093/bioinformatics/btt438&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR50\"> <span class=\"label\"> 50. </span> <cite> Canales RD, Luo Y, Willey JC, Austermiller B, Barbacioru CC, Boysen C, Hunkapiller K, Jensen RV, Knight CR, Lee KY, Ma Y, Maqsodi B, Papallo A, Peters EH, Poulter K, Ruppel PL, Samaha RR, Shi L, Yang W, Zhang L, Goodsaid FM. Evaluation of DNA microarray results with quantitative gene expression platforms. Nat Biotechnol. 2006;24:1115–1122. doi: 10.1038/nbt1236. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nbt1236\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/16964225/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&amp;title=Evaluation%20of%20DNA%20microarray%20results%20with%20quantitative%20gene%20expression%20platforms&amp;author=RD%20Canales&amp;author=Y%20Luo&amp;author=JC%20Willey&amp;author=B%20Austermiller&amp;author=CC%20Barbacioru&amp;volume=24&amp;publication_year=2006&amp;pages=1115-1122&amp;pmid=16964225&amp;doi=10.1038/nbt1236&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR51\"> <span class=\"label\"> 51. </span> <cite> Roberts ND, Kortschak RD, Parker WT, Schreiber AW, Branford S, Scott HS, Glonek G, Adelson DL. A comparative analysis of algorithms for somatic SNV detection in cancer. Bioinformatics. 2013;29:2223–2230. doi: 10.1093/bioinformatics/btt375. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/bioinformatics/btt375\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3753564/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23842810/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=A%20comparative%20analysis%20of%20algorithms%20for%20somatic%20SNV%20detection%20in%20cancer&amp;author=ND%20Roberts&amp;author=RD%20Kortschak&amp;author=WT%20Parker&amp;author=AW%20Schreiber&amp;author=S%20Branford&amp;volume=29&amp;publication_year=2013&amp;pages=2223-2230&amp;pmid=23842810&amp;doi=10.1093/bioinformatics/btt375&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR52\"> <span class=\"label\"> 52. </span> <cite> Bell AW, Deutsch EW, Au CE, Kearney RE, Beavis R, Sechi S, Nilsson T, Bergeron JJ, Group HTSW. A HUPO test sample study reveals common problems in mass spectrometry-based proteomics. Nat Methods. 2009;6:423–430. doi: 10.1038/nmeth.1333. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nmeth.1333\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2785450/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19448641/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&amp;title=A%20HUPO%20test%20sample%20study%20reveals%20common%20problems%20in%20mass%20spectrometry-based%20proteomics&amp;author=AW%20Bell&amp;author=EW%20Deutsch&amp;author=CE%20Au&amp;author=RE%20Kearney&amp;author=R%20Beavis&amp;volume=6&amp;publication_year=2009&amp;pages=423-430&amp;pmid=19448641&amp;doi=10.1038/nmeth.1333&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR53\"> <span class=\"label\"> 53. </span> <cite> ‘t Hoen PA, Friedländer MR, Almlöf J, Sammeth M, Pulyakhina I, Anvar SY, Laros JF, Buermans HP, Karlberg O, Brännvall M, GEUVADIS Consortium. den Dunnen JT, van Ommen GJ, Gut IG, Guigó R, Estivill X, Syvänen AC, Dermitzakis ET, Lappalainen T. Reproducibility of high-throughput mRNA and small RNA sequencing across laboratories. Nat Biotechnol. 2013;31:1015–1022. doi: 10.1038/nbt.2702. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nbt.2702\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24037425/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&amp;title=Reproducibility%20of%20high-throughput%20mRNA%20and%20small%20RNA%20sequencing%20across%20laboratories&amp;author=PA%20%E2%80%98t%20Hoen&amp;author=MR%20Friedl%C3%A4nder&amp;author=J%20Alml%C3%B6f&amp;author=M%20Sammeth&amp;author=I%20Pulyakhina&amp;volume=31&amp;publication_year=2013&amp;pages=1015-1022&amp;pmid=24037425&amp;doi=10.1038/nbt.2702&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR54\"> <span class=\"label\"> 54. </span> <cite> Steijger T, Abril JF, Engstrom PG, Kokocinski F, Consortium R, Akerman M, Alioto T, Ambrosini G, Antonarakis SE, Behr J, Bertone P. Assessment of transcript reconstruction methods for RNA-seq. Nat Methods. 2013;10:1177–1184. doi: 10.1038/nmeth.2714. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nmeth.2714\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3851240/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24185837/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&amp;title=Assessment%20of%20transcript%20reconstruction%20methods%20for%20RNA-seq&amp;author=T%20Steijger&amp;author=JF%20Abril&amp;author=PG%20Engstrom&amp;author=F%20Kokocinski&amp;author=R%20Consortium&amp;volume=10&amp;publication_year=2013&amp;pages=1177-1184&amp;pmid=24185837&amp;doi=10.1038/nmeth.2714&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR55\"> <span class=\"label\"> 55. </span> <cite> Ransohoff DF. Proteomics research to discover markers: what can we learn from Netflix? Clin Chem. 2010;56:172–176. doi: 10.1373/clinchem.2009.126698. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1373/clinchem.2009.126698\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/20040622/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Clin%20Chem&amp;title=Proteomics%20research%20to%20discover%20markers:%20what%20can%20we%20learn%20from%20Netflix?&amp;author=DF%20Ransohoff&amp;volume=56&amp;publication_year=2010&amp;pages=172-176&amp;pmid=20040622&amp;doi=10.1373/clinchem.2009.126698&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR56\"> <span class=\"label\"> 56. </span> <cite> Waters H. New $10 million X Prize launched for tricorder-style medical device. Nat Med. 2011;17:754. doi: 10.1038/nm0711-754a. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nm0711-754a\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/21738131/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat%20Med&amp;title=New%20%2410%20million%20X%20Prize%20launched%20for%20tricorder-style%20medical%20device&amp;author=H%20Waters&amp;volume=17&amp;publication_year=2011&amp;pages=754&amp;pmid=21738131&amp;doi=10.1038/nm0711-754a&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> </ul> </section> </section> </section> <footer class=\"p courtesy-note font-secondary font-sm text-center\"> <hr class=\"headless\"/> <p> Articles from Genome Biology are provided here courtesy of <strong> BMC </strong> </p> </footer> </section> </article> </main> </div> </div> </div> <div class=\"pmc-sidenav desktop:grid-col-4 display-flex\"> <section aria-label=\"Article resources and navigation\" class=\"pmc-sidenav__container\"> <div class=\"display-none desktop:display-block\"> <section class=\"margin-top-4 desktop:margin-top-0\"> <h2 class=\"margin-top-0\"> ACTIONS </h2> <ul class=\"usa-list usa-list--unstyled usa-list--actions\"> <li> <a class=\"usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1\" href=\"https://doi.org/10.1186/s13059-014-0462-7\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"display-inline-flex flex-justify-center flex-1 padding-right-2\"> View on publisher site </span> </a> </li> <li> <a class=\"usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1\" href=\"pdf/13059_2014_Article_462.pdf\"> <span class=\"display-inline-flex flex-justify-center flex-1\"> PDF (542.9 KB) </span> </a> </li> <li> </li> <li> </li> <li class=\"pmc-permalink\"> <div class=\"pmc-permalink__dropdown\" hidden=\"\"> <div class=\"pmc-permalink__dropdown__container\"> <h2 class=\"usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs\"> PERMALINK </h2> <div class=\"pmc-permalink__dropdown__content\"> </div> </div> </div> </li> </ul> </section> </div> <section class=\"pmc-resources margin-top-6 desktop:margin-top-4\"> <h2 class=\"margin-top-0\"> RESOURCES </h2> <div class=\"usa-accordion usa-accordion--multiselectable\"> <h3 class=\"usa-accordion__heading\"> </h3> <div class=\"usa-accordion__content usa-prose\" hidden=\"\" id=\"resources-similar-articles\"> </div> <h3 class=\"usa-accordion__heading\"> </h3> <div class=\"usa-accordion__content usa-prose\" hidden=\"\" id=\"resources-cited-by-other-articles\"> </div> <h3 class=\"usa-accordion__heading\"> </h3> <div class=\"usa-accordion__content usa-prose\" hidden=\"\" id=\"resources-links-to-ncbi-databases\"> </div> </div> </section> <section class=\"usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4\"> <nav aria-label=\"On this page\" class=\"usa-in-page-nav__nav\"> <h2 class=\"usa-in-page-nav__heading\" tabindex=\"0\"> On this page </h2> <ul class=\"usa-in-page-nav__list\"> <li class=\"usa-in-page-nav__item\"> <a class=\"usa-in-page-nav__link\" href=\"#Abs1\"> Abstract </a> </li> <li class=\"usa-in-page-nav__item\"> <a class=\"usa-in-page-nav__link\" href=\"#Sec1\"> Challenge design and dynamics </a> </li> <li class=\"usa-in-page-nav__item\"> <a class=\"usa-in-page-nav__link\" href=\"#Sec2\"> Analyzing genome assembly and structural variants </a> </li> <li class=\"usa-in-page-nav__item\"> <a class=\"usa-in-page-nav__link\" href=\"#Sec3\"> Finding genomic biomarkers that are associated with phenotype </a> </li> <li class=\"usa-in-page-nav__item\"> <a class=\"usa-in-page-nav__link\" href=\"#Sec4\"> Inferring biological networks underlying cancer phenotypes </a> </li> <li class=\"usa-in-page-nav__item\"> <a class=\"usa-in-page-nav__link\" href=\"#Sec5\"> The truth is hard to find </a> </li> <li class=\"usa-in-page-nav__item\"> <a class=\"usa-in-page-nav__link\" href=\"#Sec6\"> Closing considerations </a> </li> <li class=\"usa-in-page-nav__item\"> <a class=\"usa-in-page-nav__link\" href=\"#ack1\"> Acknowledgements </a> </li> <li class=\"usa-in-page-nav__item\"> <a class=\"usa-in-page-nav__link\" href=\"#glossary1\"> Abbreviations </a> </li> <li class=\"usa-in-page-nav__item\"> <a class=\"usa-in-page-nav__link\" href=\"#fn-group1\"> Footnotes </a> </li> <li class=\"usa-in-page-nav__item\"> <a class=\"usa-in-page-nav__link\" href=\"#_ci93_\"> Contributor Information </a> </li> <li class=\"usa-in-page-nav__item\"> <a class=\"usa-in-page-nav__link\" href=\"#Bib1\"> References </a> </li> </ul> </nav> </section> </section> </div> <div aria-label=\"Citation Dialog\" class=\"overlay citation-dialog-overlay\" role=\"dialog\"> <div aria-hidden=\"true\" class=\"dialog citation-dialog\" role=\"document\"> <div class=\"display-inline-flex flex-align-center flex-justify width-full margin-bottom-2\"> <h2 class=\"usa-modal__heading margin-0\"> Cite </h2> </div> <div class=\"citation-text-block\"> <div class=\"citation-text margin-bottom-2\"> </div> <ul class=\"usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center\"> <li> </li> <li> <a class=\"usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button\" href=\"#\" role=\"button\" tabindex=\"3\" title=\"Download a file for external citation management software\"> <span class=\"display-none mobile-lg:display-inline\"> Download .nbib </span> <span class=\"display-inline mobile-lg:display-none\"> .nbib </span> </a> </li> <li> <div class=\"display-inline-flex flex-align-center\"> <label class=\"usa-label margin-top-0\"> Format: </label> <select aria-label=\"Format\" class=\"usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4\" tabindex=\"4\"> <option value=\"AMA\"> AMA </option> <option value=\"APA\"> APA </option> <option value=\"MLA\"> MLA </option> <option selected=\"selected\" value=\"NLM\"> NLM </option> </select> </div> </li> </ul> <div class=\"dots-loading-indicator citation-loading-indicator\"> <div class=\"dot dot-1\"> </div> <div class=\"dot dot-2\"> </div> <div class=\"dot dot-3\"> </div> </div> </div> </div> </div> <div class=\"overlay collections-dialog-overlay\" role=\"dialog\"> <div aria-hidden=\"true\" class=\"dialog collections-dialog\" id=\"collections-action-dialog\" role=\"document\"> <div class=\"display-inline-flex flex-align-center flex-justify width-full margin-bottom-2\"> <h2 class=\"usa-modal__heading margin-0\"> Add to Collections </h2> </div> <div class=\"collections-action-panel action-panel\"> </div> </div> </div> </div> </div> </div> <footer class=\"ncbi-footer ncbi-dark-background\"> <div class=\"ncbi-footer__icon-section\"> <div class=\"ncbi-footer__social-header\"> Follow NCBI </div> <div class=\"grid-container ncbi-footer__ncbi-social-icons-container\"> <a class=\"ncbi-footer__social-icon ncbi-footer__social-icon--gray\" href=\"https://twitter.com/ncbi\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"usa-sr-only\"> NCBI on X (formerly known as Twitter) </span> </a> <a class=\"ncbi-footer__social-icon ncbi-footer__social-icon--gray\" href=\"https://www.facebook.com/ncbi.nlm\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"usa-sr-only\"> NCBI on Facebook </span> </a> <a class=\"ncbi-footer__social-icon ncbi-footer__social-icon--gray\" href=\"https://www.linkedin.com/company/ncbinlm\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"usa-sr-only\"> NCBI on LinkedIn </span> </a> <a class=\"ncbi-footer__social-icon ncbi-footer__social-icon--gray\" href=\"https://github.com/ncbi\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"usa-sr-only\"> NCBI on GitHub </span> </a> <a class=\"ncbi-footer__social-icon ncbi-footer__social-icon--gray\" href=\"https://ncbiinsights.ncbi.nlm.nih.gov/\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"usa-sr-only\"> NCBI RSS feed </span> </a> </div> </div> <div class=\"grid-container ncbi-footer__container\"> <div class=\"grid-row ncbi-footer__main-content-container\"> <div class=\"ncbi-footer__column\"> <p class=\"ncbi-footer__circled-icons-heading\"> Connect with NLM </p> <div class=\"ncbi-footer__circled-icons-list\"> <a class=\"ncbi-footer__social-icon ncbi-footer__social-icon--circled\" href=\"https://twitter.com/nlm_nih\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"usa-sr-only\"> NLM on X (formerly known as Twitter) </span> </a> <a class=\"ncbi-footer__social-icon ncbi-footer__social-icon--circled\" href=\"https://www.facebook.com/nationallibraryofmedicine\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"usa-sr-only\"> NLM on Facebook </span> </a> <a class=\"ncbi-footer__social-icon ncbi-footer__social-icon--circled\" href=\"https://www.youtube.com/user/NLMNIH\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"usa-sr-only\"> NLM on YouTube </span> </a> </div> </div> <address class=\"ncbi-footer__address ncbi-footer__column\"> <p> <a class=\"usa-link usa-link--external\" href=\"https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508, -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508! 4d-77.0988323\" rel=\"noopener noreferrer\" target=\"_blank\"> National Library of Medicine <br/> 8600 Rockville Pike <br/> Bethesda, MD 20894 </a> </p> </address> <ul class=\"usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column\"> <li class=\"ncbi-footer__vertical-list-item\"> <a class=\"usa-link usa-link--alt ncbi-footer__link\" href=\"https://www.nlm.nih.gov/web_policies.html\"> Web Policies </a> </li> <li class=\"ncbi-footer__vertical-list-item\"> <a class=\"usa-link usa-link--alt ncbi-footer__link\" href=\"https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office\"> FOIA </a> </li> <li class=\"ncbi-footer__vertical-list-item\"> <a class=\"usa-link usa-link--external usa-link--alt ncbi-footer__link\" href=\"https://www.hhs.gov/vulnerability-disclosure-policy/index.html\" rel=\"noreferrer noopener\" target=\"_blank\"> HHS Vulnerability Disclosure </a> </li> </ul> <ul class=\"usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column\"> <li class=\"ncbi-footer__vertical-list-item\"> <a class=\"usa-link usa-link--alt ncbi-footer__link\" href=\"https://support.nlm.nih.gov/?pagename=cloudpmc-viewer%3Apmc%3Aarticle%3Ajournal\"> Help </a> </li> <li class=\"ncbi-footer__vertical-list-item\"> <a class=\"usa-link usa-link--alt ncbi-footer__link\" href=\"https://www.nlm.nih.gov/accessibility.html\"> Accessibility </a> </li> <li class=\"ncbi-footer__vertical-list-item\"> <a class=\"usa-link usa-link--alt ncbi-footer__link\" href=\"https://www.nlm.nih.gov/careers/careers.html\"> Careers </a> </li> </ul> </div> <div class=\"grid-row grid-col-12\"> <ul class=\"ncbi-footer__bottom-links-list\"> <li class=\"ncbi-footer__bottom-list-item\"> <a class=\"usa-link usa-link--alt ncbi-footer__link\" href=\"https://www.nlm.nih.gov/\"> NLM </a> </li> <li class=\"ncbi-footer__bottom-list-item\"> <a class=\"usa-link usa-link--alt ncbi-footer__link\" href=\"https://www.nih.gov/\"> NIH </a> </li> <li class=\"ncbi-footer__bottom-list-item\"> <a class=\"usa-link usa-link--external usa-link--alt ncbi-footer__link\" href=\"https://www.hhs.gov/\" rel=\"noreferrer noopener\" target=\"_blank\"> HHS </a> </li> <li class=\"ncbi-footer__bottom-list-item\"> <a class=\"usa-link usa-link--external usa-link--alt ncbi-footer__link\" href=\"https://www.usa.gov/\" rel=\"noreferrer noopener\" target=\"_blank\"> USA.gov </a> </li> </ul> </div> </div> </footer> <div id=\"ZN_dikYWqsjiUWN0Q5\"> </div> <div class=\"fake-body-scroll\"> </div> <div class=\"fake-body-scroll\"> </div> <div class=\"QSIFeedbackButton\"> </div> </body> </html>\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 368 - INFO - Extracting links from full HTML content.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 301 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 302 - DEBUG - Loading prompt: GPT_FDR_FewShot_Syn for model gemini-2.0-flash\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1329 - DEBUG - Counting tokens for model: gemini-2.0-flash, prompt length: 2691 char\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1338 - DEBUG - Rough estimate of token count for Gemini model 'gemini-2.0-flash': 672\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 309 - INFO - Content length: 137786\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 317 - DEBUG - static_prompt: [{'role': 'developer', 'content': 'I am a large language model trained to be informative and comprehensive. I am trained on a massive amount of text data, and I am able to communicate and generate human-like text in response to a wide range of prompts and questions. For this task, I will act as a specialized assistant that can identify datasets mentioned in a publication and create a summary suitable for non-specialists.\\n\\nThe output should be a JSON array of objects, where each object has the following keys:\\n- \"dataset_identifier\": This is any alphanumeric string (maybe including punctuation marks) that uniquely identifies or provides access to a dataset, most frequently an accession code.\\n- \"repository_reference\": This is the URL or reference to the data repository where the dataset can be found.\\n\\nHere are some examples for reference:\\n\\n[\\'dataset_identifier\\'=> \\'EGAS00001000925\\', \\'repository_reference\\'=> \\'https://ega-archive.org/datasets/EGAS00001000925\\',\\n \\'dataset_identifier\\'=> \\'GSE69091\\', \\'repository_reference\\'=> \\'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE69091\\',\\n \\'dataset_identifier\\'=> \\'PRJNA306801\\', \\'repository_reference\\'=> \\'https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA306801\\',\\n \\'dataset_identifier\\'=> \\'phs003416.v1.p1\\', \\'repository_reference\\'=> \\'dbGaP\\',\\n \\'dataset_identifier\\'=> \\'PXD049309\\', \\'repository_reference\\'=> \\'https://www.ebi.ac.uk/pride/archive/projects/PXD049309\\',\\n \\'dataset_identifier\\'=> \\'PXD0402123\\', \\'repository_reference\\'=> \\'ProteomeXchange\\',\\n \\'dataset_identifier\\'=> \\'IPX0004230000\\', \\'repository_reference\\'=> \\'http://www.iprox.org\\',\\n \\'dataset_identifier\\'=> \\'MSV000092944\\', \\'repository_reference\\'=> \\'https://massive.ucsd.edu/\\',\\n \\'dataset_identifier\\'=> \\'syn1827001\\', \\'repository_reference\\'=> \\'https://www.synapse.org/[#!]*Synapse:syn28580053\\',\\n \\'dataset_identifier\\'=> \\'n/a\\', \\'repository_reference\\'=> \\'https://data.broadinstitute.org/ccle_legacy_data/mRNA_expression/\\'\\n]'}, {'role': 'user', 'content': 'Given the information that I am going to share:\\n 1) A sample of already known data repositories.\\n2) The webpage in HTML or XML format from which you have to extract datasets information.\\n\\nPlease return a JSON array of objects where each object has the following structure:\\n- `dataset_identifier`: The dataset identifier (a code). If not found, set it to \"n/a\".\\n- `repository_reference`: The URL or reference to the data repository. If not found, set it to \"n/a\".\\n\\nPlease follow these strict instructions:\\n- The output must be a valid JSON array of objects.\\n- Each object must contain the keys `dataset_identifier` and `repository_reference`.\\n- Any other output format will be considered invalid.\\n\\nBelow is the input data that you will use to generate the output:\\n1) repos => {repos}\\n2) html => {content}.'}]\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 38 - DEBUG - Rendering prompt with static_prompt:[{'role': 'developer', 'content': 'I am a large language model trained to be informative and comprehensive. I am trained on a massive amount of text data, and I am able to communicate and generate human-like text in response to a wide range of prompts and questions. For this task, I will act as a specialized assistant that can identify datasets mentioned in a publication and create a summary suitable for non-specialists.\\n\\nThe output should be a JSON array of objects, where each object has the following keys:\\n- \"dataset_identifier\": This is any alphanumeric string (maybe including punctuation marks) that uniquely identifies or provides access to a dataset, most frequently an accession code.\\n- \"repository_reference\": This is the URL or reference to the data repository where the dataset can be found.\\n\\nHere are some examples for reference:\\n\\n[\\'dataset_identifier\\'=> \\'EGAS00001000925\\', \\'repository_reference\\'=> \\'https://ega-archive.org/datasets/EGAS00001000925\\',\\n \\'dataset_identifier\\'=> \\'GSE69091\\', \\'repository_reference\\'=> \\'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE69091\\',\\n \\'dataset_identifier\\'=> \\'PRJNA306801\\', \\'repository_reference\\'=> \\'https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA306801\\',\\n \\'dataset_identifier\\'=> \\'phs003416.v1.p1\\', \\'repository_reference\\'=> \\'dbGaP\\',\\n \\'dataset_identifier\\'=> \\'PXD049309\\', \\'repository_reference\\'=> \\'https://www.ebi.ac.uk/pride/archive/projects/PXD049309\\',\\n \\'dataset_identifier\\'=> \\'PXD0402123\\', \\'repository_reference\\'=> \\'ProteomeXchange\\',\\n \\'dataset_identifier\\'=> \\'IPX0004230000\\', \\'repository_reference\\'=> \\'http://www.iprox.org\\',\\n \\'dataset_identifier\\'=> \\'MSV000092944\\', \\'repository_reference\\'=> \\'https://massive.ucsd.edu/\\',\\n \\'dataset_identifier\\'=> \\'syn1827001\\', \\'repository_reference\\'=> \\'https://www.synapse.org/[#!]*Synapse:syn28580053\\',\\n \\'dataset_identifier\\'=> \\'n/a\\', \\'repository_reference\\'=> \\'https://data.broadinstitute.org/ccle_legacy_data/mRNA_expression/\\'\\n]'}, {'role': 'user', 'content': 'Given the information that I am going to share:\\n 1) A sample of already known data repositories.\\n2) The webpage in HTML or XML format from which you have to extract datasets information.\\n\\nPlease return a JSON array of objects where each object has the following structure:\\n- `dataset_identifier`: The dataset identifier (a code). If not found, set it to \"n/a\".\\n- `repository_reference`: The URL or reference to the data repository. If not found, set it to \"n/a\".\\n\\nPlease follow these strict instructions:\\n- The output must be a valid JSON array of objects.\\n- Each object must contain the keys `dataset_identifier` and `repository_reference`.\\n- Any other output format will be considered invalid.\\n\\nBelow is the input data that you will use to generate the output:\\n1) repos => {repos}\\n2) html => {content}.'}], entire_doc: True, dynamic parts(<class 'dict'>): {'content': '<html class=\"\" lang=\"en\"> <head> <link href=\"/static/assets/style-8258669d.css\" rel=\"stylesheet\"/> <link href=\"/static/assets/style-ef962842.css\" rel=\"stylesheet\"/> <link href=\"/static/assets/style-3ade8b5c.css\" rel=\"stylesheet\"/> <link href=\"/static/img/favicons/apple-touch-icon.png\" rel=\"apple-touch-icon\" sizes=\"180x180\"/> <link href=\"/static/img/favicons/favicon-48x48.png\" rel=\"icon\" sizes=\"48x48\" type=\"image/png\"/> <link href=\"/static/img/favicons/favicon-32x32.png\" rel=\"icon\" sizes=\"32x32\" type=\"image/png\"/> <link href=\"/static/img/favicons/favicon-16x16.png\" rel=\"icon\" sizes=\"16x16\" type=\"image/png\"/> <link href=\"/static/img/favicons/site.webmanifest\" rel=\"manifest\"/> <link color=\"#0071bc\" href=\"/static/img/favicons/safari-pinned-tab.svg\" rel=\"mask-icon\"/> <title> Toward better benchmarking: challenge-based methods assessment in cancer genomics - PMC </title> <link href=\"https://www.google-analytics.com\" rel=\"preconnect\"/> <link href=\"https://cdn.ncbi.nlm.nih.gov\" rel=\"preconnect\"/> <link href=\"https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527/\" rel=\"canonical\"/> </head> <body> <a class=\"usa-skipnav\" href=\"#main-content\"> Skip to main content </a> <section aria-label=\"Official website of the United States government\" class=\"usa-banner\"> <div class=\"usa-accordion\"> <header class=\"usa-banner__header\"> <div class=\"usa-banner__inner\"> <div class=\"grid-col-auto\"> </div> <div aria-hidden=\"true\" class=\"grid-col-fill tablet:grid-col-auto\"> <p class=\"usa-banner__header-text\"> An official website of the United States government </p> <span class=\"usa-banner__header-action\"> Here\\'s how you know </span> </div> </div> </header> <div class=\"usa-banner__content usa-accordion__content\" hidden=\"\" id=\"gov-banner-default\"> <div class=\"grid-row grid-gap-lg\"> <div class=\"usa-banner__guidance tablet:grid-col-6\"> <div class=\"usa-media-block__body\"> <p> <strong> Official websites use .gov </strong> <br/> A <strong> .gov </strong> website belongs to an official government organization in the United States. </p> </div> </div> <div class=\"usa-banner__guidance tablet:grid-col-6\"> <div class=\"usa-media-block__body\"> <p> <strong> Secure .gov websites use HTTPS </strong> <br/> A <strong> lock </strong> ( <span class=\"icon-lock\"> </span> ) or <strong> https:// </strong> means you\\'ve safely connected to the .gov website. Share sensitive information only on official, secure websites. </p> </div> </div> </div> </div> </div> </section> <div class=\"usa-overlay\"> </div> <header class=\"usa-header usa-header--extended usa-header--wide\"> <div class=\"ncbi-header\"> <div class=\"ncbi-header__container\"> <a class=\"ncbi-header__logo-container\" href=\"https://www.ncbi.nlm.nih.gov/\"> </a> <div class=\"ncbi-header__desktop-buttons\"> <div class=\"ncbi-header__login-dropdown\"> <ul class=\"usa-nav__submenu ncbi-header__login-dropdown-menu\" hidden=\"\" id=\"login-dropdown-menu\"> <li class=\"usa-nav__submenu-item\"> <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/myncbi/\"> Dashboard </a> </li> <li class=\"usa-nav__submenu-item\"> <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/\"> Publications </a> </li> <li class=\"usa-nav__submenu-item\"> <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/account/settings/\"> Account settings </a> </li> <li class=\"usa-nav__submenu-item\"> </li> </ul> </div> </div> </div> </div> <div class=\"ncbi-search-panel ncbi--show-only-at-desktop\" hidden=\"\"> <div class=\"ncbi-search-panel__container\"> </div> </div> <nav aria-label=\"Primary navigation\" class=\"usa-nav\"> <p class=\"usa-sr-only\" id=\"primary-navigation-sr-only-title\"> Primary site navigation </p> <div class=\"ncbi-nav__mobile-login-menu ncbi--hide-at-desktop\" hidden=\"\"> <p class=\"ncbi-nav__mobile-login-menu-status\"> Logged in as: <strong class=\"ncbi-nav__mobile-login-menu-email\"> </strong> </p> <ul class=\"usa-nav__primary usa-accordion\"> <li class=\"usa-nav__primary-item\"> <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/myncbi/\"> Dashboard </a> </li> <li class=\"usa-nav__primary-item\"> <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/\"> Publications </a> </li> <li class=\"usa-nav__primary-item\"> <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/account/settings/\"> Account settings </a> </li> </ul> </div> </nav> </header> <section aria-label=\"PMC Header with search box\" class=\"pmc-header pmc-header--basic\"> <div class=\"pmc-nav-container\"> <div class=\"pmc-header__bar\"> <div class=\"pmc-header__logo\"> <a aria-label=\"PMC Home\" href=\"/\" title=\"Home\"> </a> </div> </div> <div class=\"pmc-header__search\"> <div class=\"display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44\"> <ul class=\"pmc-header__search__menu\"> <li> <a class=\"usa-link\" href=\"/journals/\"> Journal List </a> </li> <li> <a class=\"usa-link\" href=\"/about/userguide/\"> User Guide </a> </li> </ul> </div> </div> </div> </section> <div class=\"usa-section padding-top-0 desktop:padding-top-6 pmc-article-section\"> <div aria-label=\"Actions bar\" class=\"grid-container pmc-actions-bar is-not-intersecting\" role=\"complementary\"> <div class=\"grid-row\"> <div class=\"grid-col-fill display-flex\"> <div class=\"display-flex\"> <ul class=\"usa-list usa-list--unstyled usa-list--horizontal\"> <li class=\"margin-right-2 mobile-lg:margin-right-4 display-flex mob\"> </li> <li class=\"margin-right-2 mobile-lg:margin-right-4 display-flex mob\"> <span class=\"usa-tooltip\"> <a aria-label=\"View on publisher site\" class=\"usa-link display-flex usa-tooltip__trigger\" href=\"https://doi.org/10.1186/s13059-014-0462-7\" rel=\"noreferrer noopener\" role=\"button\" tabindex=\"0\" target=\"_blank\"> </a> <span aria-hidden=\"true\" class=\"usa-tooltip__body\" role=\"tooltip\"> View on publisher site </span> </span> </li> <li class=\"margin-right-2 mobile-lg:margin-right-4 display-flex\"> <span class=\"usa-tooltip\"> <a aria-label=\"Download PDF\" class=\"usa-link display-flex usa-tooltip__trigger\" href=\"pdf/13059_2014_Article_462.pdf\" role=\"button\" tabindex=\"0\"> </a> <span aria-hidden=\"true\" class=\"usa-tooltip__body\" role=\"tooltip\"> Download PDF </span> </span> </li> <li class=\"margin-right-2 mobile-lg:margin-right-4 display-flex\"> <span class=\"usa-tooltip\"> <span aria-hidden=\"true\" class=\"usa-tooltip__body\" role=\"tooltip\"> Add to Collections </span> </span> </li> <li class=\"margin-right-2 mobile-lg:margin-right-4 display-flex\"> <span class=\"usa-tooltip\"> <span aria-hidden=\"true\" class=\"usa-tooltip__body\" role=\"tooltip\"> Cite </span> </span> </li> <li class=\"pmc-permalink display-flex\"> <span class=\"usa-tooltip\"> <span aria-hidden=\"true\" class=\"usa-tooltip__body\" role=\"tooltip\"> Permalink </span> </span> <div class=\"pmc-permalink__dropdown\" hidden=\"\"> <div class=\"pmc-permalink__dropdown__container\"> <h2 class=\"usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs\"> PERMALINK </h2> <div class=\"pmc-permalink__dropdown__content\"> </div> </div> </div> </li> </ul> </div> </div> </div> </div> <div class=\"grid-container desktop:padding-left-6\"> <div class=\"grid-row grid-gap\" id=\"article-container\"> <div class=\"grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content\"> <div class=\"grid-container padding-left-0 padding-right-0\"> <div class=\"grid-row desktop:margin-left-neg-6\"> <div class=\"grid-col-12\"> <div aria-label=\"Disclaimer note\" class=\"pmc-layout__disclaimer\" role=\"complementary\"> As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with, the contents by NLM or the National Institutes of Health. <br/> Learn more: <a class=\"usa-link\" href=\"/about/disclaimer/\"> PMC Disclaimer </a> | <a class=\"usa-link\" href=\"/about/copyright/\"> PMC Copyright Notice </a> </div> </div> </div> <div class=\"grid-row pmc-wm desktop:margin-left-neg-6\"> <main class=\"usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0\" id=\"main-content\"> <section aria-label=\"Journal banner\" class=\"pmc-journal-banner text-center line-height-none\"> <map name=\"pmc-banner-imagemap\"> <area alt=\"Link to Genome Biology\" href=\"https://genomebiology.biomedcentral.com/\" rel=\"noopener noreferrer\" shape=\"default\" target=\"_blank\" title=\"Link to Genome Biology\"/> </map> </section> <article lang=\"en\"> <section aria-label=\"Article citation and metadata\"> <section class=\"pmc-layout__citation font-secondary font-xs\"> <div> <div class=\"display-inline-block\"> </div> . 2014 Sep 17;15(9):462. doi: <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/s13059-014-0462-7\" rel=\"noopener noreferrer\" target=\"_blank\"> 10.1186/s13059-014-0462-7 </a> </div> <nav hidden=\"hidden\" id=\"journal_context_menu\"> <ul class=\"menu-list font-family-ui\" role=\"menu\"> <li role=\"presentation\"> <a class=\"usa-link\" href=\"https://pmc.ncbi.nlm.nih.gov/search/?term=%22Genome%20Biol%22%5Bjour%5D\" role=\"menuitem\"> Search in PMC </a> </li> <li role=\"presentation\"> <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Genome%20Biol%22%5Bjour%5D\" lang=\"en\" role=\"menuitem\"> Search in PubMed </a> </li> <li role=\"presentation\"> <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Genome%20Biol%22%5BTitle%20Abbreviation%5D\" role=\"menuitem\"> View in NLM Catalog </a> </li> <li role=\"presentation\"> <a class=\"usa-link\" href=\"?term=%22Genome%20Biol%22%5Bjour%5D\" role=\"menuitem\"> Add to search </a> </li> </ul> </nav> </section> <section class=\"front-matter\"> <div class=\"ameta p font-secondary font-xs\"> <hgroup> <h1> Toward better benchmarking: challenge-based methods assessment in cancer genomics </h1> </hgroup> <div class=\"cg p\"> <a aria-describedby=\"id1\" aria-expanded=\"false\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Boutros%20PC%22%5BAuthor%5D\"> <span class=\"name western\"> Paul C Boutros </span> </a> <div hidden=\"hidden\" id=\"id1\"> <h3> <span class=\"name western\"> Paul C Boutros </span> </h3> <div class=\"p\"> <sup> 1 </sup> Informatics &amp; Biocomputing Program, Ontario Institute for Cancer Research, University Avenue, Toronto, ON M5G 0A3 Canada </div> <div class=\"p\"> <sup> 2 </sup> Department of Medical Biophysics, University of Toronto, College Street, Toronto, ON M5G 1L7 Canada </div> <div class=\"p\"> <sup> 3 </sup> Department of Pharmacology &amp; Toxicology, University of Toronto, King’s College Circle, Toronto, ON M5S 1A8 Canada </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Boutros%20PC%22%5BAuthor%5D\"> <span class=\"name western\"> Paul C Boutros </span> </a> </div> </div> <sup> 1, </sup> <sup> 2, </sup> <sup> 3 </sup> , <a aria-describedby=\"id2\" aria-expanded=\"false\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Margolin%20AA%22%5BAuthor%5D\"> <span class=\"name western\"> Adam A Margolin </span> </a> <div hidden=\"hidden\" id=\"id2\"> <h3> <span class=\"name western\"> Adam A Margolin </span> </h3> <div class=\"p\"> <sup> 4 </sup> Sage Bionetworks, Fairview Ave North, Seattle, WA 98109 USA </div> <div class=\"p\"> <sup> 5 </sup> Computational Biology Program, Oregon Health &amp; Science University, SW Sam Jackson Park Road, Portland, OR 97239-3098 USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Margolin%20AA%22%5BAuthor%5D\"> <span class=\"name western\"> Adam A Margolin </span> </a> </div> </div> <sup> 4, </sup> <sup> 5 </sup> , <a aria-describedby=\"id3\" aria-expanded=\"false\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Stuart%20JM%22%5BAuthor%5D\"> <span class=\"name western\"> Joshua M Stuart </span> </a> <div hidden=\"hidden\" id=\"id3\"> <h3> <span class=\"name western\"> Joshua M Stuart </span> </h3> <div class=\"p\"> <sup> 6 </sup> Department of Biomolecular Engineering, University of California, Santa Cruz, High Street, Santa Cruz, CA 95064 USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Stuart%20JM%22%5BAuthor%5D\"> <span class=\"name western\"> Joshua M Stuart </span> </a> </div> </div> <sup> 6 </sup> , <a aria-describedby=\"id4\" aria-expanded=\"false\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Califano%20A%22%5BAuthor%5D\"> <span class=\"name western\"> Andrea Califano </span> </a> <div hidden=\"hidden\" id=\"id4\"> <h3> <span class=\"name western\"> Andrea Califano </span> </h3> <div class=\"p\"> <sup> 7 </sup> Department of Systems Biology, Biochemistry &amp; Molecular Biophysics, Herbert Irving Comprehensive Cancer Center, Columbia University, St. Nicholas Avenue, New York, NY 10032 USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Califano%20A%22%5BAuthor%5D\"> <span class=\"name western\"> Andrea Califano </span> </a> </div> </div> <sup> 7 </sup> , <a aria-describedby=\"id5\" aria-expanded=\"false\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Stolovitzky%20G%22%5BAuthor%5D\"> <span class=\"name western\"> Gustavo Stolovitzky </span> </a> <div hidden=\"hidden\" id=\"id5\"> <h3> <span class=\"name western\"> Gustavo Stolovitzky </span> </h3> <div class=\"p\"> <sup> 8 </sup> IBM Computational Biology Center, TJ Watson Research Center, Kitchawan Road, Yorktown Heights, NY 10598 USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Stolovitzky%20G%22%5BAuthor%5D\"> <span class=\"name western\"> Gustavo Stolovitzky </span> </a> </div> </div> <sup> 8, </sup> <sup> ✉ </sup> </div> <ul class=\"d-buttons inline-list\"> <li> </li> <li> </li> <li> </li> </ul> <div class=\"d-panels font-secondary-light\"> <div class=\"d-panel p\" id=\"aip_a\"> <div class=\"p\" id=\"Aff1\"> <sup> 1 </sup> Informatics &amp; Biocomputing Program, Ontario Institute for Cancer Research, University Avenue, Toronto, ON M5G 0A3 Canada </div> <div id=\"Aff2\"> <sup> 2 </sup> Department of Medical Biophysics, University of Toronto, College Street, Toronto, ON M5G 1L7 Canada </div> <div id=\"Aff3\"> <sup> 3 </sup> Department of Pharmacology &amp; Toxicology, University of Toronto, King’s College Circle, Toronto, ON M5S 1A8 Canada </div> <div id=\"Aff4\"> <sup> 4 </sup> Sage Bionetworks, Fairview Ave North, Seattle, WA 98109 USA </div> <div id=\"Aff5\"> <sup> 5 </sup> Computational Biology Program, Oregon Health &amp; Science University, SW Sam Jackson Park Road, Portland, OR 97239-3098 USA </div> <div id=\"Aff6\"> <sup> 6 </sup> Department of Biomolecular Engineering, University of California, Santa Cruz, High Street, Santa Cruz, CA 95064 USA </div> <div id=\"Aff7\"> <sup> 7 </sup> Department of Systems Biology, Biochemistry &amp; Molecular Biophysics, Herbert Irving Comprehensive Cancer Center, Columbia University, St. Nicholas Avenue, New York, NY 10032 USA </div> <div id=\"Aff8\"> <sup> 8 </sup> IBM Computational Biology Center, TJ Watson Research Center, Kitchawan Road, Yorktown Heights, NY 10598 USA </div> <div class=\"author-notes p\"> <div class=\"fn\" id=\"_fncrsp93pmc__\"> <sup> ✉ </sup> <p class=\"display-inline\"> Corresponding author. </p> </div> </div> </div> <div class=\"d-panel p\" id=\"anp_a\"> <div class=\"notes p\"> <section class=\"history\" id=\"historyarticle-meta1\"> <p> Issue date 2014. </p> </section> </div> </div> <div class=\"d-panel p\" id=\"clp_a\"> <div> © Boutros et al.; licensee BioMed Central Ltd. 2014 </div> <p> The licensee has exclusive rights to distribute this article, in any medium, for 12 months following its publication. After this time, the article is available under the terms of the Creative Commons Attribution License ( <a class=\"usa-link usa-link--external\" href=\"http://creativecommons.org/licenses/by/4.0\" rel=\"noopener noreferrer\" target=\"_blank\"> http://creativecommons.org/licenses/by/4.0 </a> ), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly credited. The Creative Commons Public Domain Dedication waiver ( <a class=\"usa-link usa-link--external\" href=\"http://creativecommons.org/publicdomain/zero/1.0/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://creativecommons.org/publicdomain/zero/1.0/ </a> ) applies to the data made available in this article, unless otherwise stated. </p> <div class=\"p\"> <a class=\"usa-link\" href=\"/about/copyright/\"> PMC Copyright notice </a> </div> </div> </div> <div> PMCID: PMC4318527 PMID: <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/25314947/\"> 25314947 </a> </div> </div> </section> </section> <section aria-label=\"Article content\"> <section class=\"body main-article-body\"> <section class=\"abstract\" id=\"Abs1\"> <h2> <a class=\"usa-anchor\" id=\"Abs1-anchor\"> </a> Abstract </h2> <p> Rapid technological development has created an urgent need for improved evaluation of algorithms for the analysis of cancer genomics data. We outline how challenge-based assessment may help fill this gap by leveraging crowd-sourcing to distribute effort and reduce bias. </p> </section> <hr class=\"headless\"/> <p> Computational biology comprises three inter-connected activities: algorithm development, validation through benchmarking, and application. In the biomedical sciences, benchmarking occupies a central and indispensable role as it maps algorithms from the space of theoretical possibilities to the realm of practical value. Critically, this process attributes specific probabilities to an algorithm’s discovery of biologically relevant knowledge (measured by the sensitivity of the algorithm) while not overwhelming the researcher with incorrect predictions (quantified by the algorithm specificity). Benchmarking is, however, a complex task, requiring the creation of comprehensive gold standards and the design of sophisticated validation strategies that may require additional experimental data. Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table <a class=\"usa-link\" href=\"#Tab1\"> 1 </a> ). In particular, the rapidly increasing size of whole-genome molecular profile datasets from large sample repositories underscores the importance of benchmarking; it has become virtually impossible to validate algorithmic predictions that are based on such large datasets systematically. </p> <section class=\"tw xbox font-sm\" id=\"Tab1\"> <h2 class=\"obj_head\"> Table 1. </h2> <div class=\"caption p\"> <p> <strong> Non-comprehensive list of important and current challenge efforts and platforms </strong> </p> </div> <div class=\"tbl-box p\" tabindex=\"0\"> <table class=\"content\" frame=\"hsides\" rules=\"groups\"> <thead> <tr valign=\"top\"> <th colspan=\"1\" rowspan=\"1\"> <strong> Challenge </strong> </th> <th colspan=\"1\" rowspan=\"1\"> <strong> Scope </strong> </th> <th colspan=\"1\" rowspan=\"1\"> <strong> Assessment type </strong> </th> <th colspan=\"1\" rowspan=\"1\"> <strong> Organizers </strong> </th> <th colspan=\"1\" rowspan=\"1\"> <strong> Website </strong> </th> </tr> </thead> <tbody> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> Assemblathon1&amp;2 </td> <td colspan=\"1\" rowspan=\"1\"> Sequence assembly </td> <td colspan=\"1\" rowspan=\"1\"> Objective scoring </td> <td colspan=\"1\" rowspan=\"1\"> UC Davis Genome Center </td> <td colspan=\"1\" rowspan=\"1\"> <a class=\"usa-link usa-link--external\" href=\"http://assemblathon.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://assemblathon.org/ </a> </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> CAFA </td> <td colspan=\"1\" rowspan=\"1\"> Protein function prediction </td> <td colspan=\"1\" rowspan=\"1\"> Objective scoring </td> <td colspan=\"1\" rowspan=\"1\"> Community collaboration </td> <td colspan=\"1\" rowspan=\"1\"> <a class=\"usa-link usa-link--external\" href=\"http://biofunctionprediction.org/node/8\" rel=\"noopener noreferrer\" target=\"_blank\"> http://biofunctionprediction.org/node/8 </a> </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> CAGI </td> <td colspan=\"1\" rowspan=\"1\"> Systems biology </td> <td colspan=\"1\" rowspan=\"1\"> Objective scoring </td> <td colspan=\"1\" rowspan=\"1\"> UC Berkley/University of Maryland </td> <td colspan=\"1\" rowspan=\"1\"> <a class=\"usa-link usa-link--external\" href=\"http://genomeinterpretation.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://genomeinterpretation.org/ </a> </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> CAPRI </td> <td colspan=\"1\" rowspan=\"1\"> Protein docking </td> <td colspan=\"1\" rowspan=\"1\"> Objective scoring </td> <td colspan=\"1\" rowspan=\"1\"> Community collaboration </td> <td colspan=\"1\" rowspan=\"1\"> <a class=\"usa-link usa-link--external\" href=\"http://www.ebi.ac.uk/msd-srv/capri/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.ebi.ac.uk/msd-srv/capri/ </a> </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> CASP </td> <td colspan=\"1\" rowspan=\"1\"> Structure prediction </td> <td colspan=\"1\" rowspan=\"1\"> Objective scoring </td> <td colspan=\"1\" rowspan=\"1\"> Community collaboration </td> <td colspan=\"1\" rowspan=\"1\"> <a class=\"usa-link usa-link--external\" href=\"http://predictioncenter.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://predictioncenter.org/ </a> </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> ChaLearn </td> <td colspan=\"1\" rowspan=\"1\"> Machine learning </td> <td colspan=\"1\" rowspan=\"1\"> Objective scoring </td> <td colspan=\"1\" rowspan=\"1\"> ChaLearn Organization (non-for profit) </td> <td colspan=\"1\" rowspan=\"1\"> <a class=\"usa-link usa-link--external\" href=\"http://www.chalearn.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.chalearn.org/ </a> </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> CLARITY </td> <td colspan=\"1\" rowspan=\"1\"> Clinical genome interpretation </td> <td colspan=\"1\" rowspan=\"1\"> Objective scoring and evaluation by judges </td> <td colspan=\"1\" rowspan=\"1\"> Boston Children’s Hospital </td> <td colspan=\"1\" rowspan=\"1\"> <a class=\"usa-link usa-link--external\" href=\"http://www.childrenshospital.org/research-and-innovation/research-initiatives/clarity-challenge\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.childrenshospital.org/research-and-innovation/research-initiatives/clarity-challenge </a> </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> DREAM </td> <td colspan=\"1\" rowspan=\"1\"> Network inference and systems biology </td> <td colspan=\"1\" rowspan=\"1\"> Objective scoring </td> <td colspan=\"1\" rowspan=\"1\"> Community collaboration &amp; Sage Bionetworks </td> <td colspan=\"1\" rowspan=\"1\"> <a class=\"usa-link usa-link--external\" href=\"https://www.synapse.org/#!Challenges:DREAM\" rel=\"noopener noreferrer\" target=\"_blank\"> https://www.synapse.org/#!Challenges:DREAM </a> </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> FlowCAP </td> <td colspan=\"1\" rowspan=\"1\"> Flow cytometry analysis </td> <td colspan=\"1\" rowspan=\"1\"> Objective scoring </td> <td colspan=\"1\" rowspan=\"1\"> Community collaboration </td> <td colspan=\"1\" rowspan=\"1\"> <a class=\"usa-link usa-link--external\" href=\"http://flowcap.flowsite.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://flowcap.flowsite.org/ </a> </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> IGCG-TCGA DREAM Somatic Mutation Calling </td> <td colspan=\"1\" rowspan=\"1\"> Sequence analysis </td> <td colspan=\"1\" rowspan=\"1\"> Objective evaluation </td> <td colspan=\"1\" rowspan=\"1\"> Community collaboration &amp; Sage Bionetworks </td> <td colspan=\"1\" rowspan=\"1\"> <a class=\"usa-link usa-link--external\" href=\"https://www.synapse.org/#!Synapse:syn312572\" rel=\"noopener noreferrer\" target=\"_blank\"> https://www.synapse.org/#!Synapse:syn312572 </a> </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> IMPROVER </td> <td colspan=\"1\" rowspan=\"1\"> Systems biology </td> <td colspan=\"1\" rowspan=\"1\"> Objective evaluation and crowd-verification </td> <td colspan=\"1\" rowspan=\"1\"> Phillip Morris International </td> <td colspan=\"1\" rowspan=\"1\"> <a class=\"usa-link usa-link--external\" href=\"https://sbvimprover.com/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://sbvimprover.com/ </a> </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> Innocentive </td> <td colspan=\"1\" rowspan=\"1\"> Topics in various industries </td> <td colspan=\"1\" rowspan=\"1\"> Objective scoring and evaluation by judges </td> <td colspan=\"1\" rowspan=\"1\"> Commercial platform </td> <td colspan=\"1\" rowspan=\"1\"> <a class=\"usa-link usa-link--external\" href=\"http://www.innocentive.com/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.innocentive.com/ </a> </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> Kaggle </td> <td colspan=\"1\" rowspan=\"1\"> Topics in various industries </td> <td colspan=\"1\" rowspan=\"1\"> Objective scoring and evaluation by judges </td> <td colspan=\"1\" rowspan=\"1\"> Commercial platform </td> <td colspan=\"1\" rowspan=\"1\"> <a class=\"usa-link usa-link--external\" href=\"http://www.kaggle.com/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.kaggle.com/ </a> </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> RGASP </td> <td colspan=\"1\" rowspan=\"1\"> RNA-seq analyses </td> <td colspan=\"1\" rowspan=\"1\"> Objective scoring </td> <td colspan=\"1\" rowspan=\"1\"> European Bioinformatics Institute </td> <td colspan=\"1\" rowspan=\"1\"> <a class=\"usa-link usa-link--external\" href=\"http://www.gencodegenes.org/rgasp/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.gencodegenes.org/rgasp/ </a> </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> Sequence Squeeze </td> <td colspan=\"1\" rowspan=\"1\"> Sequence compression </td> <td colspan=\"1\" rowspan=\"1\"> Objective scoring and evaluation by judges </td> <td colspan=\"1\" rowspan=\"1\"> Pistoia Alliance </td> <td colspan=\"1\" rowspan=\"1\"> <a class=\"usa-link usa-link--external\" href=\"http://sequencesqueeze.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://sequencesqueeze.org/ </a> </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> X-Prize </td> <td colspan=\"1\" rowspan=\"1\"> Technology </td> <td colspan=\"1\" rowspan=\"1\"> Evaluation by judges </td> <td colspan=\"1\" rowspan=\"1\"> X-Prize Organization (non-for-profit) </td> <td colspan=\"1\" rowspan=\"1\"> <a class=\"usa-link usa-link--external\" href=\"http://www.xprize.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.xprize.org/ </a> </td> </tr> </tbody> </table> </div> <div class=\"p text-right font-secondary\"> <a class=\"usa-link\" href=\"table/Tab1/\" rel=\"noopener noreferrer\" target=\"_blank\"> Open in a new tab </a> </div> <div class=\"tw-foot p\"> <div class=\"fn\" id=\"_fn_p4\"> <p> The challenges were chosen based on relevance to cancer genomics or the representativeness of a type of challenge. Different challenges specialize in specific areas of research (see ‘Scope’), and may use different assessment types such as objective scoring against a gold standard, evaluation by judges, or community consensus (‘crowd-verification’). Organizers can be researchers from specific institutions (such as universities or hospitals), a group of diverse researchers from academia and industry collaborating in the challenge organization (community collaboration), not-for-profit associations, or commercial platforms that run challenges as their business model (such as Innocentive and Kaggle). Initiatives such as CAFA, CAGI, CAPRI, CASP, ChaLearn, DREAM, FlowCAP and IMPROVER organize several challenges each year, and only the generic project is listed in this table, with the exception of DREAM, for which we also show the IGCG-TCGA DREAM Somatic Mutation Calling Challenge because of its relevance to this paper. More information about these efforts can be found on the listed websites. </p> </div> </div> </section> <p> Benchmarking is not a matter of simply running a few algorithms on a few datasets and comparing the results. Drawing generalizable conclusions from the exercise requires significant care in design and execution. The maturity of bioinformatics as a discipline has been greatly advanced by the adoption of key principles that guide robust method evaluation, including evaluator objectiveness (lack of bias), clearly defined scoring metrics that align with real-world goals, and the public release of gold-standard datasets and of the results and code of prediction algorithms. Challenge-based (also known as ‘competition-based’) method assessment is an increasingly popular mechanism for benchmarking [ <a aria-describedby=\"CR1\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR1\"> 1 </a> , <a aria-describedby=\"CR2\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR2\"> 2 </a> ]. In this type of study an impartial group of scientists organizes a ‘challenge’ that is based on a carefully curated dataset. This dataset is typically split into a training dataset, a validation dataset (which might be used in real-time leaderboards, typically implemented as a table that reports the comparative performance of the methods under development), and a gold standard (or test) dataset that is withheld from challenge participants and used for final evaluation (Figure <a class=\"usa-link\" href=\"#Fig1\"> 1 </a> ). Following algorithm development on the training dataset and real-time feedback to participants based on the validation dataset and reported in the leaderboard, the challenge organizers can objectively evaluate the quality of final submitted predictions using a gold-standard dataset. Such a design closely reflects the actual difficulties faced by real-world users trying to determine whether an algorithm generalizes to unseen cases. </p> <figure class=\"fig xbox font-sm\" id=\"Fig1\"> <h2 class=\"obj_head\"> Figure 1. </h2> <p class=\"img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center\"> </p> <div class=\"p text-right font-secondary\"> <a class=\"usa-link\" href=\"figure/Fig1/\" rel=\"noopener noreferrer\" target=\"_blank\"> Open in a new tab </a> </div> <figcaption> <p> <strong> Typical design of a crowd-sourced challenge. </strong> A dataset is split into a training set, a validation (or leaderboard set) and the test set (or gold standard). Participants have access to the challenge input data and the known answers for just the training set. For the validation and test sets only, the challenge input data are provided but the answers to the challenge questions are withheld. In the challenge open phase, participants optimize their algorithms by making repeated submissions to predict the validation set answers. These submissions are scored and returned to the participants who can use the information to improve their methods. In the final evaluation phase, the optimized algorithms are submitted and evaluated against the final test set (the gold standard), and the resulting scores are used to compute the statistical significance and the ranking of the participating algorithms. </p> </figcaption> </figure> <p> When flawed, benchmarking can lead to the emergence of suboptimal standards that may be applied to many large datasets, imposing an immense cost to the community and creating misleading results. Conversely, the acceptance of knowledge without robust benchmarking can lead to the adoption of inaccurate conventions. For example, during the 1990s, it was generally accepted that the number of loci coding for proteins in the human genome was 100,000, a number that was based on unverified hypotheses [ <a aria-describedby=\"CR3\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR3\"> 3 </a> ]. When the human genome was finally sequenced in 2000, the total number of coding loci was found to be a factor of 5 lower. Similarly, a design error in the early implementation of the GC Robust Multi-Array (GCRMA) algorithm, which was revealed by systematic benchmarking of network reconstruction analyses, may have led to the publication of thousands of papers that contain incorrect mRNA abundance profiles before the error was detected and corrected [ <a aria-describedby=\"CR4\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR4\"> 4 </a> ]. As a third example, in 2006, a group of Duke University researchers published a pair of high-impact papers claiming accurate prediction of the prognosis of lung cancer patients and of chemotherapy-sensitivity in lung, breast and ovarian cancers. Attempts to reproduce those claims ensued almost immediately, with most of the results falling short of replication because of a combination of programming and data-entry errors, and possible fraud [ <a aria-describedby=\"CR5\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR5\"> 5 </a> ]. Proper objective benchmarking by a neutral third-party on private validation data helps to resolve quickly or to detect many of the issues associated with these kinds of studies. </p> <p> One concern in algorithm benchmarking and validation is that computational biology algorithms are often developed and evaluated by the same researchers. This creates an inherent conflict of interest, where objective assessment of accuracy is polluted by the fact that the developers become simultaneously judge, jury and executioner of the validity of their own work. This can result in biases in study design and over-optimistic performance estimates, whether intentional or unintentional [ <a aria-describedby=\"CR6\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR6\"> 6 </a> ]. For instance, the use of non-blinded data in the evaluation by methods developers of their own protein structure prediction methods led, in the early’ 80s, to the false belief that protein structure prediction was essentially a solved problem. Not until 1994, when double-blinded data were used in the first Workshop on the Critical Assessment of Protein Structure Prediction (CASP), was a very different picture revealed [ <a aria-describedby=\"CR7\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR7\"> 7 </a> ]. </p> <p> Challenge-based benchmarking efforts, such as CASP [ <a aria-describedby=\"CR8\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR8\"> 8 </a> – <a aria-describedby=\"CR10\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR10\"> 10 </a> ], CAFA [ <a aria-describedby=\"CR11\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR11\"> 11 </a> ] and DREAM [ <a aria-describedby=\"CR12\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR12\"> 12 </a> , <a aria-describedby=\"CR13\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR13\"> 13 </a> ], among others (Table <a class=\"usa-link\" href=\"#Tab1\"> 1 </a> ), offer a robust framework for algorithm evaluation. These efforts have proven the value of engaging both active challenge leaders and motivated algorithm developers to improve their work in a forum that has high visibility and rapid feedback. </p> <p> We believe that challenge-based methods assessment will play an increasingly important role in standardizing and optimizing the analysis of cancer genomics data, and its broader adoption will drive progress in both algorithm development and biological discovery. Conversely, failing to exploit challenge-benchmarking as a fundamental validation methodology for cancer genomics algorithms may result in lost opportunities to translate results derived from best-in-class methods into patient care. </p> <p> Here, we provide our perspective on the growing use of challenge-based methods to benchmark algorithms in cancer genomics. We outline the different types of problems faced and some of the key considerations that need to be explored to determine whether a challenge might be successful, and to provide suggestions for challenge organization and execution. Finally, we look to the future and consider how challenge-based assessment may change in the coming decade. </p> <section id=\"Sec1\"> <h2 class=\"pmc_sec_title\"> <a class=\"usa-anchor\" id=\"Sec1-anchor\"> </a> Challenge design and dynamics </h2> <p> Over the past few years, an established challenge-based design paradigm has emerged in which portions of a private (that is, not globally released) dataset are made publicly available according to a predefined schedule. Such a dataset provides increased user engagement based on continuous feedback; an opportunity for participants to refine and improve their methods on the basis of results obtained throughout the challenge; and multiple independent rounds of validation, which can be used to assess the consistency and robustness of results. After the initial training dataset is made publicly available, a real-time leaderboard can be generated in which the performance of different algorithms is evaluated against a withheld private portion of the data (Figure <a class=\"usa-link\" href=\"#Fig1\"> 1 </a> ). Previous research has shown that the provision of real-time feedback is among the most important factors in ensuring user engagement in crowd-sourcing projects [ <a aria-describedby=\"CR14\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR14\"> 14 </a> ]. (Here, we use the term crowd-sourcing in the sense that a community of tens to hundreds of researchers are engaged in working on the same problem. In other contexts, crowd-sourcing activities may engage different numbers of participants.) After a period of time in which several iterations of the leaderboard can be posted, one of the participating groups is declared the best performer in this initial phase of the challenge, either on the basis of their position on the leaderboard or because they were the first to reach some pre-specified performance level. The challenge may include multiple rounds of assessment based on different portions of the private data. A final round is typically invoked in which methods are rated against a withheld evaluation dataset to determine the overall challenge winner (Figure <a class=\"usa-link\" href=\"#Fig1\"> 1 </a> ). The most robust validation set is often reserved for this final evaluation - often with larger sample size, newly generated data or prospective validation designed on the basis of challenge results. Each participating team submits a small number (for example, one to five) of independent predictions made by their algorithm(s), which are scored and ranked to determine a winner. Finally, the public release of all of the data kept private throughout the challenge, along with the predictions and ideally source code from each group, provides a long-term resource to spur further development of new and improved methods. </p> <p> The collection of algorithm source code allows developers to share insights that promote future improvements. If required as part of the final submission, this code can also be used to ensure objective scoring and verification of reproducibility. In the 2012 Sage Bionetworks-DREAM Breast Cancer Prognosis Challenge, participants were required to submit their models as open source R-code [ <a aria-describedby=\"CR15\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR15\"> 15 </a> ] that was visible to all participants and executed by an automated system to generate the results reported on the leaderboard. This was enabled by Synapse [ <a aria-describedby=\"CR16\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR16\"> 16 </a> ], a software platform that supports scientific challenges as well as large distributed collaborations, such as those in the TCGA Pan-Cancer consortium [ <a aria-describedby=\"CR17\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR17\"> 17 </a> ]. Planned challenges, such as the RNA-seq follow-up to the ICGC-TCGA DREAM Somatic Mutation Calling (SMC) Challenge, are considering the use of cloud-computing solutions to provide a central computing facility and a harness upon which contestant code is directly run. This will inherently force the deposition of complete analysis workflows, which can be run routinely on new datasets. Further, this approach would help to standardize application programming interfaces and file formats, such that multiple algorithms use similar inputs and produce easily comparable outputs. This vision of interoperability is shared by many practitioners in the field and has most recently been championed by the Global Alliance for Genomics And Health [ <a aria-describedby=\"CR18\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR18\"> 18 </a> ]. </p> <p> Several criteria should be used to help participants limit over-fitting to the training data. Over-fitting is a known peril in statistics, occurring when a predictive model has enough flexibility in its parameters that optimization effectively leads to ‘memorization’ of the training data and an inability to generalize to unseen cases. The most common way to help participants avoid over-fitting, while enabling the testing of their models, is to provide leaderboard scoring that is based on a subset of the private dataset, optimally a subset that is not used in the final evaluation. The latter condition is sometimes not feasible (for example, when the number of patients available to predict clinical outcomes is limited), in which case the leaderboard will be based on data that are also used for the final scoring. If this is the case, limiting the number of submissions can reduce over-fitting. </p> <p> While most challenges share some common design principles, each research area has its own unique characteristics that require customized experimental designs and consideration of risks and benefits. Indeed, the utility of organizing a challenge to help advance a particular research area depends on a balance between challenge-based benchmarking advantages and limitations, as well as consideration of the potential barriers for participation (Table <a class=\"usa-link\" href=\"#Tab2\"> 2 </a> ). In the sections below, we highlight three research areas in which rapid development of new algorithms has led to a concomitant need for benchmarking: accurate identification of tumor-specific genomic alterations, association of clinical data with genomic profiles (that is, biomarkers) and identifying network-biology features that underlie cancer phenotypes. </p> <section class=\"tw xbox font-sm\" id=\"Tab2\"> <h3 class=\"obj_head\"> Table 2. </h3> <div class=\"caption p\"> <p> <strong> Some advantages and limitations of challenge-based methods assessment, along with barriers to participation in them </strong> </p> </div> <div class=\"tbl-box p\" tabindex=\"0\"> <table class=\"content\" frame=\"hsides\" rules=\"groups\"> <thead> <tr valign=\"top\"> <th colspan=\"1\" rowspan=\"1\"> <strong> Advantages </strong> </th> <th colspan=\"1\" rowspan=\"1\"> <strong> Limitations </strong> </th> <th colspan=\"1\" rowspan=\"1\"> <strong> Participation barriers </strong> </th> </tr> </thead> <tbody> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> Reduction of over-fitting </td> <td colspan=\"1\" rowspan=\"1\"> Narrower scope compared to traditional open-ended research </td> <td colspan=\"1\" rowspan=\"1\"> Incentives not strong enough to promote participation </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> Benchmarking individual methods </td> <td colspan=\"1\" rowspan=\"1\"> Ground truth needed for objective scoring </td> <td colspan=\"1\" rowspan=\"1\"> No funding available to support time spent participating in challenges </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> Impartial comparison across methods using same datasets </td> <td colspan=\"1\" rowspan=\"1\"> Mostly limited to computational approaches </td> <td colspan=\"1\" rowspan=\"1\"> Fatigue resulting from many ongoing challenges </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> Fostering collaborative work, including code sharing </td> <td colspan=\"1\" rowspan=\"1\"> Requires data producers to share their data before publication </td> <td colspan=\"1\" rowspan=\"1\"> Time assigned by organizers to solve a difficult challenge question may be too short </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> Acceleration of research </td> <td colspan=\"1\" rowspan=\"1\"> Sufficient amount of high-quality data needed for meaningful results </td> <td colspan=\"1\" rowspan=\"1\"> Lack of computing capabilities </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> Enhancing data access and impact </td> <td colspan=\"1\" rowspan=\"1\"> Large number of participants not always available </td> <td colspan=\"1\" rowspan=\"1\"> New data modality or datasets that are too complex or too big poses entry barrier </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> Determination of problem solvability </td> <td colspan=\"1\" rowspan=\"1\"> Challenge questions may not be solvable with data at hand </td> <td colspan=\"1\" rowspan=\"1\"> Challenge questions not interesting or impactful enough </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> Tapping the ‘Wisdom of Crowds’ </td> <td colspan=\"1\" rowspan=\"1\"> Traditional grant mechanisms not adequate to fund challenge efforts </td> <td colspan=\"1\" rowspan=\"1\"> Cumbersome approvals to acquire sensitive datasets </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> Objective assessment </td> <td colspan=\"1\" rowspan=\"1\"> Difficulties to distribute datasets with sensitive information </td> <td colspan=\"1\" rowspan=\"1\"> </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> Standardizes experimental design </td> <td colspan=\"1\" rowspan=\"1\"> </td> <td colspan=\"1\" rowspan=\"1\"> </td> </tr> </tbody> </table> </div> <div class=\"p text-right font-secondary\"> <a class=\"usa-link\" href=\"table/Tab2/\" rel=\"noopener noreferrer\" target=\"_blank\"> Open in a new tab </a> </div> </section> </section> <section id=\"Sec2\"> <h2 class=\"pmc_sec_title\"> <a class=\"usa-anchor\" id=\"Sec2-anchor\"> </a> Analyzing genome assembly and structural variants </h2> <p> Technologies for identifying cancer-related somatic alterations from genomic or transcriptomic data are advancing extremely rapidly. In only 6 years, next-generation sequencing (NGS) has rapidly progressed from the measurement of millions of short sequences (of around 25 bp) to that of hundreds of millions of longer segments (of around 100 bp). This creates an urgent need for on-going benchmarking studies as old algorithms become rapidly out-dated and new algorithmic approaches are required to handle new technologies and new scales of data. Small-scale studies have resulted in dramatic discordance when different researchers apply their algorithms to the same genomic data (Figure <a class=\"usa-link\" href=\"#Fig2\"> 2 </a> ) [ <a aria-describedby=\"CR19\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR19\"> 19 </a> – <a aria-describedby=\"CR21\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR21\"> 21 </a> ]. These studies have shown that accuracy and generalizability vary dramatically across samples and regions of the genome. The constantly shifting landscape presented by rapidly evolving technologies and tools fuels the urgency in the need to identify the best-performing methods objectively and to re-evaluate them frequently, and to identify particularly error-prone aspects of existing tumor genome analysis methods [ <a aria-describedby=\"CR22\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR22\"> 22 </a> ]. Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of <em> de novo </em> sequence assembly algorithms [ <a aria-describedby=\"CR23\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR23\"> 23 </a> ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ <a aria-describedby=\"CR24\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR24\"> 24 </a> ] (Table <a class=\"usa-link\" href=\"#Tab1\"> 1 </a> ). </p> <figure class=\"fig xbox font-sm\" id=\"Fig2\"> <h3 class=\"obj_head\"> Figure 2. </h3> <p class=\"img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center\"> </p> <div class=\"p text-right font-secondary\"> <a class=\"usa-link\" href=\"figure/Fig2/\" rel=\"noopener noreferrer\" target=\"_blank\"> Open in a new tab </a> </div> <figcaption> <p> <strong> Different researchers studying the same data may arrive at discordant conclusions. </strong> Benchmarking becomes essential as a way to separate true findings from spurious ones. (Illustration by Natasha Stolovitzky-Brunner© inspired by the parable of the six blind men and the elephant). </p> </figcaption> </figure> <p> Challenge-based benchmarking of methods for somatic variant detection in cancer faces several unique hurdles. First, genomic sequence is inherently identifiable [ <a aria-describedby=\"CR25\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR25\"> 25 </a> ], and is thus considered personal health information (PHI) in many countries. This places a burden on challenge contestants to acquire ethics approval from the appropriate authorities, such as dbGaP in the USA or ICGC in Canada. Second, because of the inherent complexity of both the data and file formats, it may be difficult for researchers from other fields to acquire sufficient domain knowledge to compete effectively against domain experts. This point may be ameliorated by gamifying the problem, that is, using game tools that require puzzle solving or geometric thinking to engage users in genomics problems [ <a aria-describedby=\"CR26\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR26\"> 26 </a> , <a aria-describedby=\"CR27\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR27\"> 27 </a> ]. Gamification may not be possible or appropriate, however, because it may require sacrificing domain-specific prior knowledge that is essential to the correct solution. Third, the size of the raw genomic data necessary to perform these challenges creates a ‘big-data’ problem. For example, the ICGC-TCGA DREAM SMC Challenge [ <a aria-describedby=\"CR28\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR28\"> 28 </a> ] (Table <a class=\"usa-link\" href=\"#Tab1\"> 1 </a> ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs. Two different solutions to this problem are to provide access to high-speed, cloud-based download technologies (such as GeneTorrent or Aspera) or to provide co-location of computers and data in a hosted environment [ <a aria-describedby=\"CR29\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR29\"> 29 </a> ]. The latter solution has the advantage of providing implementations of the best-performing algorithms in a form that is more readily redistributed to the community, as well as allowing more ‘democratized’ participation for groups that do not have large in-house computing resources. Nevertheless, this solution also has disadvantages: cloud-computing may require additional overhead expenditure for groups that are familiar with developing methods within their local computing environments; many researchers have access to in-house computing options subsidized by their institution and have limited incentive to transfer their analysis to the cloud; and access permissions for some datasets can hinder redistribution through cloud platforms. Furthermore, the assessment of predictions is challenging because the ground-truth for genetic alterations is unknown. The SMC Challenge employs two strategies for evaluation. The first involves an <em> in silico </em> method for simulating cancer genomes called BAMSurgeon, which was developed to allow the comparison of methods predictions against a synthetic ground-truth (work by Ewing and colleagues). In the second strategy, targeted deep-sequencing allows prospective validation of a large number of predicted mutations, chosen by an algorithm that most accurately computes false-positive and false-negative rates across submissions. It is unclear how important it is for prospective validation data to be orthogonal to that used by the original challenge participants. Verification in TCGA projects typically relies on deep sequencing using the same technology, but on selected targets and with the construction of new sequencing libraries. This approach assumes that most errors are randomly distributed and/or associated with only a small fraction of reads. The more orthogonal the validation technology, the more this assumption is relaxed. Nevertheless, the error profile of the final evaluation dataset is crucial, and there are currently no error-free approaches to generating this gold-standard data for NGS. </p> </section> <section id=\"Sec3\"> <h2 class=\"pmc_sec_title\"> <a class=\"usa-anchor\" id=\"Sec3-anchor\"> </a> Finding genomic biomarkers that are associated with phenotype </h2> <p> Once a set of somatic variants have been identified from genomic interrogation of patient-derived samples, one of the most common analyses is to attempt to develop biomarkers that can predict patient survival, response to therapy or other outcomes [ <a aria-describedby=\"CR30\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR30\"> 30 </a> – <a aria-describedby=\"CR33\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR33\"> 33 </a> ]. The development of genomic-based personalized medicine has immense clinical potential, but the optimal approach to predicting such biomarkers <em> de novo </em> remains poorly understood and controversial. Indeed, it is widely known that inferred biomarkers are highly sensitive to factors such as choice of algorithm and data pre-processing methods [ <a aria-describedby=\"CR34\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR34\"> 34 </a> – <a aria-describedby=\"CR37\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR37\"> 37 </a> ]. </p> <p> Nevertheless, developing challenges to benchmark biomarker discovery problems is relatively straightforward. Participants are given training data in which features (for example, genome-wide mRNA transcript abundance) are paired with outcome (for example, patient survival) data. Participants are given only the features for the test set and asked to predict the outcome data using a model inferred from the training data. Alternatively, participants may submit trained models as executable code to be run on the test data, thus allowing the test feature data to be hidden from participants [ <a aria-describedby=\"CR15\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR15\"> 15 </a> ]. Model results are scored on the basis of the correspondence between predicted and measured outcome data from the test set. </p> <p> Prediction challenges have been employed in many domains outside of biomedical research [ <a aria-describedby=\"CR38\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR38\"> 38 </a> ]. Because biomarker-based challenges fit the setup of the classic supervised machine-learning paradigm, they attract new ideas and participation from the broader machine-learning community. Benchmarking in biomarker discovery is crucial, however, as outlined by the case of the retracted Duke study on chemotherapy selection noted above. </p> <p> Two key difficulties exist in the creation of benchmarking challenges for biomarker discovery. First, the ideal datasets for biomarker-discovery challenges are uniquely defined, especially when data were collected from large cohorts requiring long-term follow-up or expensive standardized treatment protocols (such as clinical trials). These datasets can potentially lead to high-impact publications or concerns over the intellectual property of the data-generating groups. Second, the potential size of patient cohorts is currently limiting for many biomarker-development questions. If the amount of data available is inadequate, they may not generate enough statistical power to distinguish the performance of the top-ranked groups accurately. These factors also complicate the ability to obtain independent datasets for final method assessment. Despite these problems, several successful challenges pertaining to diagnostics, prognostics and treatment outcomes have been conducted, including the MAQC-II study [ <a aria-describedby=\"CR39\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR39\"> 39 </a> ], the IMPROVER Challenge on Diagnostic Signatures [ <a aria-describedby=\"CR40\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR40\"> 40 </a> ], the Sage Bionetworks DREAM Breast Cancer Prognostics Challenge [ <a aria-describedby=\"CR15\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR15\"> 15 </a> ], and the DREAM AML Treatment Outcome Challenge [ <a aria-describedby=\"CR41\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR41\"> 41 </a> ]. </p> </section> <section id=\"Sec4\"> <h2 class=\"pmc_sec_title\"> <a class=\"usa-anchor\" id=\"Sec4-anchor\"> </a> Inferring biological networks underlying cancer phenotypes </h2> <p> Identifying the relationships between biological (transcriptional and signaling) networks and cancer onset and progression is another potential area for challenge benchmarking. Network analysis involves several aspects, including the coherent modeling of different types of alteration and dysregulation events and their integration into a unified network-based model [ <a aria-describedby=\"CR42\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR42\"> 42 </a> – <a aria-describedby=\"CR44\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR44\"> 44 </a> ]. One of the major problems with organizing challenges in this area is that the underlying cellular regulatory networks are mostly unknown, especially in complex systems such mammalian tumor cells. So how can a challenge be organized when a pre-known gold-standard network cannot be defined? Several strategies employed by the DREAM project include using synthetic biology networks [ <a aria-describedby=\"CR13\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR13\"> 13 </a> ], <em> in silico </em> networks [ <a aria-describedby=\"CR45\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR45\"> 45 </a> ], and experimentally assessed bacterial networks [ <a aria-describedby=\"CR46\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR46\"> 46 </a> ]. An alternative strategy is to evaluate methods on the basis of their ability to predict the response of a system to a set of perturbations, such as drugs or receptor ligands, as surrogates for predicting the underlying network connectivity [ <a aria-describedby=\"CR47\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR47\"> 47 </a> ]. The introduction of ingenious surrogates to the gold standard has enabled the formulation of other network reverse-engineering challenges, such as the 2013 HPN-DREAM Breast Cancer Network Inference Challenge [ <a aria-describedby=\"CR48\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR48\"> 48 </a> ]. In this challenge, participants were asked to submit predicted signaling networks that were activated by a set of stimuli in four breast cancer cell lines. These networks were scored on the basis of their ability to identify the set of proteins that are downstream of a given phosphoprotein. The predicted protein set was compared to an experimentally determined set of proteins (the surrogate gold standard), defined as those proteins whose phosphorylation levels were affected by inhibiting that phosphoprotein. Further research on benchmarking network-inference algorithms would be highly beneficial to help advance the field of network biology, whose role in unraveling biological mechanisms in cancer is hard to overestimate. </p> </section> <section id=\"Sec5\"> <h2 class=\"pmc_sec_title\"> <a class=\"usa-anchor\" id=\"Sec5-anchor\"> </a> The truth is hard to find </h2> <p> From the previous discussion, it is clear that the single most crucial aspect in benchmarking is the definition and assembly of gold standards. A gold standard fundamentally defines the problem under study, and it provides the limiting resolution of error for the overall endeavor. As outlined in this article, gold standards can be defined in several ways. First, a single experiment can be performed with portions of the resulting data used for training and evaluation. This approach avoids experimental inconsistencies, but requires that a large selection of true results is generated prior to the challenge. Simulated datasets are ideal for this strategy but have been criticized as only partially representing a biological system [ <a aria-describedby=\"CR49\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR49\"> 49 </a> ]. While validation of simulated data is straight forward, because the ground-truth is completely known, in most cases the value of benchmarking is perceived to be in the ability to assess best-performing methods when applied to real biological data as opposed to simulated data. An important caveat is that the synthetic data may fail to reflect some of the underlying assumptions of the system they attempt to emulate. Indeed, the most common question about simulations is how well they reflect experimental samples [ <a aria-describedby=\"CR49\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR49\"> 49 </a> ]. </p> <p> Second, for systems that are difficult to benchmark directly, such as the structure of a biological network, characteristics of the systems can be evaluated instead. These might include the effects of the systems’ perturbation or other phenomena, such as the identification of the networks that best predict patient outcomes. </p> <p> Third, the results of a study can be validated after the challenge is completed by additional experimental work, either on the same sample or on others. This has the advantage of directly addressing the predictions made by challenge participants, but has the disadvantage of introducing a time lag between challenge completion and the availability of full results. In addition, the effort and cost of follow-up validation may be prohibitive given the resources available to the challenge organizers. </p> <p> For genomic studies, wet-lab validation can be both time-consuming and expensive. For example, the MAQC study considered approximately 20,000 genes on microarray platforms, but only validated approximately 1,000 (5%) by real-time PCR as a gold standard [ <a aria-describedby=\"CR50\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR50\"> 50 </a> ]. Because of this cost, both in terms of time and money, it is critical that a good validation be sufficiently representative, providing similar levels of statistical power for assessing the accuracy of each group. In the context of somatic mutation calling, this means selecting calls that are unique to individual predictors as well as those common to multiple predictors. Indeed, the validation techniques will often be experimentally limited to a subset of results, leaving a bias in the distribution of what is tested. There is thus a clear need for research into the optimal selection of validation candidates in many biological settings. Further, validating a small subset (&lt;10%) of results comes with the possibility, however small, of producing an incorrect relative ordering of different algorithms. In practice, a combination of synthetic and real-world validation is best, and finding the right balance is challenge-dependent. </p> <p> Finally, some very important elements of cancer genomics are difficult to validate. For example, almost all NGS analyses rely on sequence alignment as a first step. It is, however, very difficult to benchmark the accuracy of an alignment algorithm on real tumor data, because there is no obvious way to create a ground-truth dataset. Thus, rather than benchmarking the aligners, challenges benchmark the results of entire pipelines such as those for detecting somatic variants [ <a aria-describedby=\"CR28\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR28\"> 28 </a> ], which may incorporate different aligners and different data pre-processing and statistical approaches. Similarly, it is of great interest to infer cancer-driver genes. Unfortunately, the definition of a ‘driver gene’ (beyond simple statistical recurrence) is unclear, and does not yet allow unambiguous, high-throughput experimental validation. Most experimental techniques in this area probe only one aspect of a driver gene (such as its influence on proliferation or metastasis), while many subtle phenotypes (such as angiogenesis or local spread) are challenging to probe. Also, these designs ignore the potentially polygenic nature of tumor initiation and progression. In designing a new challenge, one of the first questions must be whether or not suitable gold-standard test datasets can be generated. </p> </section> <section id=\"Sec6\"> <h2 class=\"pmc_sec_title\"> <a class=\"usa-anchor\" id=\"Sec6-anchor\"> </a> Closing considerations </h2> <p> Benchmarking is a fundamental part of computational biology and is increasingly being appreciated by the biomedical community as a whole. Recent benchmarking studies both within [ <a aria-describedby=\"CR19\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR19\"> 19 </a> , <a aria-describedby=\"CR51\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR51\"> 51 </a> ] and outside of cancer genomics [ <a aria-describedby=\"CR39\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR39\"> 39 </a> , <a aria-describedby=\"CR52\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR52\"> 52 </a> – <a aria-describedby=\"CR54\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR54\"> 54 </a> ] have helped highlight new ways to analyze data and have prompted reconsideration of the error profiles of datasets. Challenge-based assessments have also recently surged in other fields [ <a aria-describedby=\"CR55\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR55\"> 55 </a> ] in which the use of incentives (including prizes and prestige) have stimulated increased attention and algorithm development [ <a aria-describedby=\"CR56\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR56\"> 56 </a> ]. </p> <p> As the profile of the results of benchmarking studies increases, it is becoming increasingly clear that benchmarking itself is a serious scientific endeavor. The design of a challenge is non-trivial and in some ways is easy ‘to get wrong’ - there needs to be a careful integration between experts in challenge-based benchmarking and domain experts in the challenge topic. At the outset, there is a fundamental requirement for the benchmarking team to foster a community that supports and promotes the exercise. Indeed, some topic areas may be unsuitable to challenge-based benchmarking because a sufficiently big community of interested algorithm developers has not yet emerged (although in these cases, appropriate incentives may be useful in helping to focus attention on a potential challenge topic). Further, the challenge organizing team must be able to assure the broader community of its neutrality and objectivity. There is a clear advantage to building groups of ‘challenge-based benchmarking experts’ who can bring their expertise to diverse topics within cancer genomics, or any other field. Such groups may be well-placed to develop and optimize the statistical methods needed to improve challenge-based benchmarks. Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table <a class=\"usa-link\" href=\"#Tab1\"> 1 </a> ). </p> <p> Cancer genomics is characterized by rapid technological development, and this trend is likely to persist for many years. As a result, benchmarking cannot be a static endeavor. Rather, each new technology will have its own specific error profiles and distinct algorithms that are used for data analysis. In a world of continual technological and algorithmic innovation, it may be impossible to have definitive, permanent benchmarks, because any effort will be based on a snapshot of technology and will rapidly become out-dated. Instead, a long-running series of ‘living benchmarks’ may allow the co-evolution of benchmarks with technology. In this mutualistic scenario, regular releases of new datasets capturing the current state of experimental methodologies will allow users at any point in time to identify the best tool for their dataset, and algorithm developers to have a dataset suitable for developing and optimizing methods on the latest data. </p> </section> <section class=\"ack\" id=\"ack1\"> <h2 class=\"pmc_sec_title\"> <a class=\"usa-anchor\" id=\"ack1-anchor\"> </a> Acknowledgements </h2> <p> We are indebted to the DREAM community for teaching us how to run challenges, and to Sage Bionetworks for their contributions in organizing DREAM challenges. This study was conducted with the support of the Ontario Institute for Cancer Research through funding provided by the Government of Ontario (to PCB). This work was supported by Prostate Cancer Canada and is proudly funded by the Movember Foundation (Grant #RS2014-01). Dr Boutros was supported by a Terry Fox Research Institute New Investigator Award and a CIHR New Investigator Award. This project was supported by Genome Canada through a Large-Scale Applied Project contract. This work was also supported by the Discovery Frontiers: Advancing Big Data Science in Genomics Research program, which is jointly funded by the Natural Sciences and Engineering Research Council (NSERC) of Canada, the Canadian Institutes of Health Research (CIHR), Genome Canada, and the Canada Foundation for Innovation (CFI). NIH grants R01 CA180778 (JMS) and U24-CA143858 (JMS) supported this work. </p> </section> <section class=\"glossary\" id=\"glossary1\"> <h2 class=\"pmc_sec_title\"> <a class=\"usa-anchor\" id=\"glossary1-anchor\"> </a> Abbreviations </h2> <dl class=\"def-list\"> </dl> <dl class=\"def-list\"> <dt> CASP </dt> <dd> <p> Critical Assessment of Protein Structure Prediction </p> </dd> <dt> GCRMA </dt> <dd> <p> GC Robust Multi-Array </p> </dd> <dt> PHI </dt> <dd> <p> Personal health information </p> </dd> <dt> NGS </dt> <dd> <p> Next-generation sequencing </p> </dd> <dt> SMC </dt> <dd> <p> Somatic Mutation Calling </p> </dd> </dl> </section> <section class=\"fn-group\" id=\"fn-group1\"> <h2 class=\"pmc_sec_title\"> <a class=\"usa-anchor\" id=\"fn-group1-anchor\"> </a> Footnotes </h2> <div class=\"fn-group p font-secondary-light font-sm\"> <div class=\"fn p\" id=\"fn1\"> <p> <strong> Competing interests </strong> </p> <p> The authors declare that they have no competing interests. </p> </div> </div> </section> <section class=\"contrib-info\" id=\"_ci93_\" lang=\"en\"> <h2 class=\"pmc_sec_title\"> <a class=\"usa-anchor\" id=\"_ci93_-anchor\"> </a> Contributor Information </h2> <p> Paul C Boutros, Email: Paul.Boutros@oicr.on.ca. </p> <p> Adam A Margolin, Email: margolin@ohsu.edu. </p> <p> Joshua M Stuart, Email: jstuart@ucsc.edu. </p> <p> Andrea Califano, Email: ac2248@c2b2.columbia.edu. </p> <p> Gustavo Stolovitzky, Email: gustavo@us.ibm.com. </p> </section> <section class=\"ref-list\" id=\"Bib1\"> <h2 class=\"pmc_sec_title\"> <a class=\"usa-anchor\" id=\"Bib1-anchor\"> </a> References </h2> <section id=\"Bib1_sec2\"> <ul class=\"ref-list font-sm\"> <li id=\"CR1\"> <span class=\"label\"> 1. </span> <cite> Costello JC, Stolovitzky G. Seeking the wisdom of crowds through challenge-based competitions in biomedical research. Clin Pharmacol Ther. 2013;93:396–398. doi: 10.1038/clpt.2013.36. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/clpt.2013.36\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23549146/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&amp;title=Seeking%20the%20wisdom%20of%20crowds%20through%20challenge-based%20competitions%20in%20biomedical%20research&amp;author=JC%20Costello&amp;author=G%20Stolovitzky&amp;volume=93&amp;publication_year=2013&amp;pages=396-398&amp;pmid=23549146&amp;doi=10.1038/clpt.2013.36&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR2\"> <span class=\"label\"> 2. </span> <cite> Meyer P, Alexopoulos LG, Bonk T, Califano A, Cho CR, de la Fuente A, de Graaf D, Hartemink AJ, Hoeng J, Ivanov NV, Koeppl H, Linding R, Marbach D, Norel R, Peitsch MC, Rice JJ, Royyuru A, Schacherer F, Sprengel J, Stolle K, Vitkup D, Stolovitzky G. Verification of systems biology research in the age of collaborative competition. Nat Biotechnol. 2011;29:811–815. doi: 10.1038/nbt.1968. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nbt.1968\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/21904331/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&amp;title=Verification%20of%20systems%20biology%20research%20in%20the%20age%20of%20collaborative%20competition&amp;author=P%20Meyer&amp;author=LG%20Alexopoulos&amp;author=T%20Bonk&amp;author=A%20Califano&amp;author=CR%20Cho&amp;volume=29&amp;publication_year=2011&amp;pages=811-815&amp;pmid=21904331&amp;doi=10.1038/nbt.1968&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR3\"> <span class=\"label\"> 3. </span> <cite> Pertea M, Salzberg SL. Between a chicken and a grape: estimating the number of human genes. Genome Biol. 2010;11:206. doi: 10.1186/gb-2010-11-5-206. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/gb-2010-11-5-206\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2898077/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/20441615/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=Between%20a%20chicken%20and%20a%20grape:%20estimating%20the%20number%20of%20human%20genes&amp;author=M%20Pertea&amp;author=SL%20Salzberg&amp;volume=11&amp;publication_year=2010&amp;pages=206&amp;pmid=20441615&amp;doi=10.1186/gb-2010-11-5-206&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR4\"> <span class=\"label\"> 4. </span> <cite> Lim WK, Wang K, Lefebvre C, Califano A. Comparative analysis of microarray normalization procedures: effects on reverse engineering gene networks. Bioinformatics. 2007;23:i282–i288. doi: 10.1093/bioinformatics/btm201. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/bioinformatics/btm201\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/17646307/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Comparative%20analysis%20of%20microarray%20normalization%20procedures:%20effects%20on%20reverse%20engineering%20gene%20networks&amp;author=WK%20Lim&amp;author=K%20Wang&amp;author=C%20Lefebvre&amp;author=A%20Califano&amp;volume=23&amp;publication_year=2007&amp;pages=i282-i288&amp;pmid=17646307&amp;doi=10.1093/bioinformatics/btm201&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR5\"> <span class=\"label\"> 5. </span> <cite> Baggerly KA, Coombes KR. Deriving chemosensitivity from cell lines: forensic bioinformatics and reproducible research in high-throughput biology. Ann Appl Stat. 2009;3:1309–1334. doi: 10.1214/09-AOAS291. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1214/09-AOAS291\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Ann%20Appl%20Stat&amp;title=Deriving%20chemosensitivity%20from%20cell%20lines:%20forensic%20bioinformatics%20and%20reproducible%20research%20in%20high-throughput%20biology&amp;author=KA%20Baggerly&amp;author=KR%20Coombes&amp;volume=3&amp;publication_year=2009&amp;pages=1309-1334&amp;doi=10.1214/09-AOAS291&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR6\"> <span class=\"label\"> 6. </span> <cite> Norel R, Rice JJ, Stolovitzky G. The self-assessment trap: can we all be better than average? Mol Syst Biol. 2011;7:537. doi: 10.1038/msb.2011.70. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/msb.2011.70\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3261704/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/21988833/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Mol%20Syst%20Biol&amp;title=The%20self-assessment%20trap:%20can%20we%20all%20be%20better%20than%20average?&amp;author=R%20Norel&amp;author=JJ%20Rice&amp;author=G%20Stolovitzky&amp;volume=7&amp;publication_year=2011&amp;pages=537&amp;pmid=21988833&amp;doi=10.1038/msb.2011.70&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR7\"> <span class=\"label\"> 7. </span> <cite> Moult J, Pedersen JT, Judson R, Fidelis K. A large-scale experiment to assess protein structure prediction methods. Proteins. 1995;23:ii–v. doi: 10.1002/prot.340230303. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1002/prot.340230303\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/8710822/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Proteins&amp;title=A%20large-scale%20experiment%20to%20assess%20protein%20structure%20prediction%20methods&amp;author=J%20Moult&amp;author=JT%20Pedersen&amp;author=R%20Judson&amp;author=K%20Fidelis&amp;volume=23&amp;publication_year=1995&amp;pages=ii-v&amp;pmid=8710822&amp;doi=10.1002/prot.340230303&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR8\"> <span class=\"label\"> 8. </span> <cite> Cozzetto D, Kryshtafovych A, Tramontano A. Evaluation of CASP8 model quality predictions. Proteins. 2009;77:157–166. doi: 10.1002/prot.22534. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1002/prot.22534\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19714774/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Proteins&amp;title=Evaluation%20of%20CASP8%20model%20quality%20predictions&amp;author=D%20Cozzetto&amp;author=A%20Kryshtafovych&amp;author=A%20Tramontano&amp;volume=77&amp;publication_year=2009&amp;pages=157-166&amp;pmid=19714774&amp;doi=10.1002/prot.22534&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR9\"> <span class=\"label\"> 9. </span> <cite> Shi S, Pei J, Sadreyev RI, Kinch LN, Majumdar I, Tong J, Cheng H, Kim BH, Grishin NV. Analysis of CASP8 targets, predictions and assessment methods. Database (Oxford) 2009;2009:bap003. doi: 10.1093/database/bap003. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/database/bap003\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2794793/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/20157476/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Database%20(Oxford)&amp;title=Analysis%20of%20CASP8%20targets,%20predictions%20and%20assessment%20methods&amp;author=S%20Shi&amp;author=J%20Pei&amp;author=RI%20Sadreyev&amp;author=LN%20Kinch&amp;author=I%20Majumdar&amp;volume=2009&amp;publication_year=2009&amp;pages=bap003&amp;pmid=20157476&amp;doi=10.1093/database/bap003&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR10\"> <span class=\"label\"> 10. </span> <cite> Tramontano A, Morea V. Assessment of homology-based predictions in CASP5. Proteins. 2004;55:782. doi: 10.1002/prot.20187. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1002/prot.20187\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/14579324/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Proteins&amp;title=Assessment%20of%20homology-based%20predictions%20in%20CASP5&amp;author=A%20Tramontano&amp;author=V%20Morea&amp;volume=55&amp;publication_year=2004&amp;pages=782&amp;pmid=14579324&amp;doi=10.1002/prot.20187&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR11\"> <span class=\"label\"> 11. </span> <cite> Radivojac P, Clark WT, Oron TR, Schnoes AM, Wittkop T, Sokolov A, Graim K, Funk C, Verspoor K, Ben-Hur A, Pandey G, Yunes JM, Talwalkar AS, Repo S, Souza ML, Piovesan D, Casadio R, Wang Z, Cheng J, Fang H, Gough J, Koskinen P, Törönen P, Nokso-Koivisto J, Holm L, Cozzetto D, Buchan DW, Bryson K, Jones DT, Limaye B, et al. A large-scale evaluation of computational protein function prediction. Nat Methods. 2013;10:221–227. doi: 10.1038/nmeth.2340. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nmeth.2340\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3584181/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23353650/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&amp;title=A%20large-scale%20evaluation%20of%20computational%20protein%20function%20prediction&amp;author=P%20Radivojac&amp;author=WT%20Clark&amp;author=TR%20Oron&amp;author=AM%20Schnoes&amp;author=T%20Wittkop&amp;volume=10&amp;publication_year=2013&amp;pages=221-227&amp;pmid=23353650&amp;doi=10.1038/nmeth.2340&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR12\"> <span class=\"label\"> 12. </span> <cite> Prill RJ, Marbach D, Saez-Rodriguez J, Sorger PK, Alexopoulos LG, Xue X, Clarke ND, Altan-Bonnet G, Stolovitzky G. Towards a rigorous assessment of systems biology models: the DREAM3 challenges. PLoS One. 2010;5:e9202. doi: 10.1371/journal.pone.0009202. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1371/journal.pone.0009202\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2826397/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/20186320/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Towards%20a%20rigorous%20assessment%20of%20systems%20biology%20models:%20the%20DREAM3%20challenges&amp;author=RJ%20Prill&amp;author=D%20Marbach&amp;author=J%20Saez-Rodriguez&amp;author=PK%20Sorger&amp;author=LG%20Alexopoulos&amp;volume=5&amp;publication_year=2010&amp;pages=e9202&amp;pmid=20186320&amp;doi=10.1371/journal.pone.0009202&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR13\"> <span class=\"label\"> 13. </span> <cite> Stolovitzky G, Prill RJ, Califano A. Lessons from the DREAM2 challenges. Ann N Y Acad Sci. 2009;1158:159–195. doi: 10.1111/j.1749-6632.2009.04497.x. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1111/j.1749-6632.2009.04497.x\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19348640/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Ann%20N%20Y%20Acad%20Sci&amp;title=Lessons%20from%20the%20DREAM2%20challenges&amp;author=G%20Stolovitzky&amp;author=RJ%20Prill&amp;author=A%20Califano&amp;volume=1158&amp;publication_year=2009&amp;pages=159-195&amp;pmid=19348640&amp;doi=10.1111/j.1749-6632.2009.04497.x&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR14\"> <span class=\"label\"> 14. </span> <cite> Athanasopoulos G, Hyndman RJ. The value of feedback in forecasting competitions. Int J Forecast. 2011;27:845–849. doi: 10.1016/j.ijforecast.2011.03.002. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.ijforecast.2011.03.002\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Int%20J%20Forecast&amp;title=The%20value%20of%20feedback%20in%20forecasting%20competitions&amp;author=G%20Athanasopoulos&amp;author=RJ%20Hyndman&amp;volume=27&amp;publication_year=2011&amp;pages=845-849&amp;doi=10.1016/j.ijforecast.2011.03.002&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR15\"> <span class=\"label\"> 15. </span> <cite> Margolin AA, Bilal E, Huang E, Norman TC, Ottestad L, Mecham BH, Sauerwine B, Kellen MR, Mangravite LM, Furia MD, Vollan HK, Rueda OM, Guinney J, Deflaux NA, Hoff B, Schildwachter X, Russnes HG, Park D, Vang VO, Pirtle T, Youseff L, Citro C, Curtis C, Kristensen VN, Hellerstein J, Friend SH, Stolovitzky G, Aparicio S, Caldas C, Børresen-Dale AL. Systematic analysis of challenge-driven improvements in molecular prognostic models for breast cancer. Sci Transl Med. 2013;5:181re181. doi: 10.1126/scitranslmed.3006112. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1126/scitranslmed.3006112\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3897241/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23596205/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Sci%20Transl%20Med&amp;title=Systematic%20analysis%20of%20challenge-driven%20improvements%20in%20molecular%20prognostic%20models%20for%20breast%20cancer&amp;author=AA%20Margolin&amp;author=E%20Bilal&amp;author=E%20Huang&amp;author=TC%20Norman&amp;author=L%20Ottestad&amp;volume=5&amp;publication_year=2013&amp;pages=181re181&amp;pmid=23596205&amp;doi=10.1126/scitranslmed.3006112&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR16\"> <span class=\"label\"> 16. </span> <cite> <em> Synapse </em> ; [ <a class=\"usa-link usa-link--external\" href=\"http://www.sagebase.org/synapse\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.sagebase.org/synapse </a> ] </cite> </li> <li id=\"CR17\"> <span class=\"label\"> 17. </span> <cite> Omberg L, Ellrott K, Yuan Y, Kandoth C, Wong C, Kellen MR, Friend SH, Stuart J, Liang H, Margolin AA. Enabling transparent and collaborative computational analysis of 12 tumor types within The Cancer Genome Atlas. Nat Genet. 2013;45:1121–1126. doi: 10.1038/ng.2761. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/ng.2761\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3950337/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24071850/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&amp;title=Enabling%20transparent%20and%20collaborative%20computational%20analysis%20of%2012%20tumor%20types%20within%20The%20Cancer%20Genome%20Atlas&amp;author=L%20Omberg&amp;author=K%20Ellrott&amp;author=Y%20Yuan&amp;author=C%20Kandoth&amp;author=C%20Wong&amp;volume=45&amp;publication_year=2013&amp;pages=1121-1126&amp;pmid=24071850&amp;doi=10.1038/ng.2761&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR18\"> <span class=\"label\"> 18. </span> <cite> Global Alliance for Genomics and Health ; [ <a class=\"usa-link usa-link--external\" href=\"http://genomicsandhealth.org\" rel=\"noopener noreferrer\" target=\"_blank\"> http://genomicsandhealth.org </a> ] </cite> </li> <li id=\"CR19\"> <span class=\"label\"> 19. </span> <cite> Kim SY, Speed TP. Comparing somatic mutation-callers: beyond Venn diagrams. BMC Bioinformatics. 2013;14:189. doi: 10.1186/1471-2105-14-189. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/1471-2105-14-189\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3702398/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23758877/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&amp;title=Comparing%20somatic%20mutation-callers:%20beyond%20Venn%20diagrams&amp;author=SY%20Kim&amp;author=TP%20Speed&amp;volume=14&amp;publication_year=2013&amp;pages=189&amp;pmid=23758877&amp;doi=10.1186/1471-2105-14-189&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR20\"> <span class=\"label\"> 20. </span> <cite> O’Rawe J, Jiang T, Sun G, Wu Y, Wang W, Hu J, Bodily P, Tian L, Hakonarson H, Johnson WE, Wei Z, Wang K, Lyon GJ. Low concordance of multiple variant-calling pipelines: practical implications for exome and genome sequencing. Genome Med. 2013;5:28. doi: 10.1186/gm432. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/gm432\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3706896/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23537139/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Genome%20Med&amp;title=Low%20concordance%20of%20multiple%20variant-calling%20pipelines:%20practical%20implications%20for%20exome%20and%20genome%20sequencing&amp;author=J%20O%E2%80%99Rawe&amp;author=T%20Jiang&amp;author=G%20Sun&amp;author=Y%20Wu&amp;author=W%20Wang&amp;volume=5&amp;publication_year=2013&amp;pages=28&amp;pmid=23537139&amp;doi=10.1186/gm432&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR21\"> <span class=\"label\"> 21. </span> <cite> Alkan C, Coe BP, Eichler EE. Genome structural variation discovery and genotyping. Nat Rev Genet. 2011;12:363–376. doi: 10.1038/nrg2958. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nrg2958\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4108431/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/21358748/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Genet&amp;title=Genome%20structural%20variation%20discovery%20and%20genotyping&amp;author=C%20Alkan&amp;author=BP%20Coe&amp;author=EE%20Eichler&amp;volume=12&amp;publication_year=2011&amp;pages=363-376&amp;pmid=21358748&amp;doi=10.1038/nrg2958&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR22\"> <span class=\"label\"> 22. </span> <cite> <strong> Taking pan-cancer analysis global. </strong> <em> Nat Genet </em> 2013, <strong> 45: </strong> 1263. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/ng.2825\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24165723/\"> PubMed </a> ] </li> <li id=\"CR23\"> <span class=\"label\"> 23. </span> <cite> Bradnam KR, Fass JN, Alexandrov A, Baranay P, Bechner M, Birol I, Boisvert S, Chapman JA, Chapuis G, Chikhi R, Chitsaz H, Chou WC, Corbeil J, Del Fabbro C, Docking TR, Durbin R, Earl D, Emrich S, Fedotov P, Fonseca NA, Ganapathy G, Gibbs RA, Gnerre S, Godzaridis E, Goldstein S, Haimel M, Hall G, Haussler D, Hiatt JB, Ho IY, et al. Assemblathon 2: evaluating de novo methods of genome assembly in three vertebrate species. Gigascience. 2013;2:10. doi: 10.1186/2047-217X-2-10. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/2047-217X-2-10\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3844414/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23870653/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Gigascience&amp;title=Assemblathon%202:%20evaluating%20de%20novo%20methods%20of%20genome%20assembly%20in%20three%20vertebrate%20species&amp;author=KR%20Bradnam&amp;author=JN%20Fass&amp;author=A%20Alexandrov&amp;author=P%20Baranay&amp;author=M%20Bechner&amp;volume=2&amp;publication_year=2013&amp;pages=10&amp;pmid=23870653&amp;doi=10.1186/2047-217X-2-10&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR24\"> <span class=\"label\"> 24. </span> <cite> Brownstein CA, Beggs AH, Homer N, Merriman B, Yu TW, Flannery KC, Dechene ET, Towne MC, Savage SK, Price EN, Holm IA, Luquette LJ, Lyon E, Majzoub J, Neupert P, McCallie D, Jr, Szolovits P, Willard HF, Mendelsohn NJ, Temme R, Finkel RS, Yum SW, Medne L, Sunyaev SR, Adzhubey I, Cassa CA, de Bakker PI, Duzkale H, Dworzy Ski P, Fairbrother W, et al. An international effort towards developing standards for best practices in analysis, interpretation and reporting of clinical genome sequencing results in the CLARITY challenge. Genome Biol. 2014;15:R53. doi: 10.1186/gb-2014-15-3-r53. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/gb-2014-15-3-r53\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4073084/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24667040/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=An%20international%20effort%20towards%20developing%20standards%20for%20best%20practices%20in%20analysis,%20interpretation%20and%20reporting%20of%20clinical%20genome%20sequencing%20results%20in%20the%20CLARITY%20challenge&amp;author=CA%20Brownstein&amp;author=AH%20Beggs&amp;author=N%20Homer&amp;author=B%20Merriman&amp;author=TW%20Yu&amp;volume=15&amp;publication_year=2014&amp;pages=R53&amp;pmid=24667040&amp;doi=10.1186/gb-2014-15-3-r53&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR25\"> <span class=\"label\"> 25. </span> <cite> Gymrek M, McGuire AL, Golan D, Halperin E, Erlich Y. Identifying personal genomes by surname inference. Science. 2013;339:321–324. doi: 10.1126/science.1229566. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1126/science.1229566\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23329047/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Identifying%20personal%20genomes%20by%20surname%20inference&amp;author=M%20Gymrek&amp;author=AL%20McGuire&amp;author=D%20Golan&amp;author=E%20Halperin&amp;author=Y%20Erlich&amp;volume=339&amp;publication_year=2013&amp;pages=321-324&amp;pmid=23329047&amp;doi=10.1126/science.1229566&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR26\"> <span class=\"label\"> 26. </span> <cite> Good BM, Su AI. Games with a scientific purpose. Genome Biol. 2011;12:135. doi: 10.1186/gb-2011-12-12-135. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/gb-2011-12-12-135\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3334605/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/22204700/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=Games%20with%20a%20scientific%20purpose&amp;author=BM%20Good&amp;author=AI%20Su&amp;volume=12&amp;publication_year=2011&amp;pages=135&amp;pmid=22204700&amp;doi=10.1186/gb-2011-12-12-135&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR27\"> <span class=\"label\"> 27. </span> <cite> Lee J, Kladwang W, Lee M, Cantu D, Azizyan M, Kim H, Limpaecher A, Yoon S, Treuille A, Das R, Ete RNAP. RNA design rules from a massive open laboratory. Proc Natl Acad Sci U S A. 2014;111:2122–2127. doi: 10.1073/pnas.1313039111. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1073/pnas.1313039111\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3926058/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24469816/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=RNA%20design%20rules%20from%20a%20massive%20open%20laboratory&amp;author=J%20Lee&amp;author=W%20Kladwang&amp;author=M%20Lee&amp;author=D%20Cantu&amp;author=M%20Azizyan&amp;volume=111&amp;publication_year=2014&amp;pages=2122-2127&amp;pmid=24469816&amp;doi=10.1073/pnas.1313039111&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR28\"> <span class=\"label\"> 28. </span> <cite> Boutros PC, Ewing AD, Ellrott K, Norman TC, Dang KK, Hu Y, Kellen MR, Suver C, Bare JC, Stein LD, Spellman PT, Stolovitzky G, Friend SH, Margolin AA, Stuart JM. Global optimization of somatic variant identification in cancer genomes with a global community challenge. Nat Genet. 2014;46:318–319. doi: 10.1038/ng.2932. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/ng.2932\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4035501/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24675517/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&amp;title=Global%20optimization%20of%20somatic%20variant%20identification%20in%20cancer%20genomes%20with%20a%20global%20community%20challenge&amp;author=PC%20Boutros&amp;author=AD%20Ewing&amp;author=K%20Ellrott&amp;author=TC%20Norman&amp;author=KK%20Dang&amp;volume=46&amp;publication_year=2014&amp;pages=318-319&amp;pmid=24675517&amp;doi=10.1038/ng.2932&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR29\"> <span class=\"label\"> 29. </span> <cite> Dudley JT, Butte AJ. In silico research in the era of cloud computing. Nat Biotechnol. 2010;28:1181–1185. doi: 10.1038/nbt1110-1181. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nbt1110-1181\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3755123/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/21057489/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&amp;title=In%20silico%20research%20in%20the%20era%20of%20cloud%20computing&amp;author=JT%20Dudley&amp;author=AJ%20Butte&amp;volume=28&amp;publication_year=2010&amp;pages=1181-1185&amp;pmid=21057489&amp;doi=10.1038/nbt1110-1181&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR30\"> <span class=\"label\"> 30. </span> <cite> Lambin P, van Stiphout RG, Starmans MH, Rios-Velazquez E, Nalbantov G, Aerts HJ, Roelofs E, van Elmpt W, Boutros PC, Granone P, Valentini V, Begg AC, De Ruysscher D, Dekker A. Predicting outcomes in radiation oncology - multifactorial decision support systems. Nat Rev Clin Oncol. 2013;10:27–40. doi: 10.1038/nrclinonc.2012.196. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nrclinonc.2012.196\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4555846/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23165123/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Clin%20Oncol&amp;title=Predicting%20outcomes%20in%20radiation%20oncology%20-%20multifactorial%20decision%20support%20systems&amp;author=P%20Lambin&amp;author=RG%20van%20Stiphout&amp;author=MH%20Starmans&amp;author=E%20Rios-Velazquez&amp;author=G%20Nalbantov&amp;volume=10&amp;publication_year=2013&amp;pages=27-40&amp;pmid=23165123&amp;doi=10.1038/nrclinonc.2012.196&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR31\"> <span class=\"label\"> 31. </span> <cite> Chin L, Gray JW. Translating insights from the cancer genome into clinical practice. Nature. 2008;452:553–563. doi: 10.1038/nature06914. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nature06914\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2730524/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/18385729/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Translating%20insights%20from%20the%20cancer%20genome%20into%20clinical%20practice&amp;author=L%20Chin&amp;author=JW%20Gray&amp;volume=452&amp;publication_year=2008&amp;pages=553-563&amp;pmid=18385729&amp;doi=10.1038/nature06914&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR32\"> <span class=\"label\"> 32. </span> <cite> Khleif SN, Doroshow JH, Hait WN. AACR-FDA-NCI Cancer Biomarkers Collaborative consensus report: advancing the use of biomarkers in cancer drug development. Clin Cancer Res. 2010;16:3299–3318. doi: 10.1158/1078-0432.CCR-10-0880. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1158/1078-0432.CCR-10-0880\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/20501613/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&amp;title=AACR-FDA-NCI%20Cancer%20Biomarkers%20Collaborative%20consensus%20report:%20advancing%20the%20use%20of%20biomarkers%20in%20cancer%20drug%20development&amp;author=SN%20Khleif&amp;author=JH%20Doroshow&amp;author=WN%20Hait&amp;volume=16&amp;publication_year=2010&amp;pages=3299-3318&amp;pmid=20501613&amp;doi=10.1158/1078-0432.CCR-10-0880&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR33\"> <span class=\"label\"> 33. </span> <cite> van’t Veer LJ, Bernards R. Enabling personalized cancer medicine through analysis of gene-expression patterns. Nature. 2008;452:564–570. doi: 10.1038/nature06915. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nature06915\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/18385730/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Enabling%20personalized%20cancer%20medicine%20through%20analysis%20of%20gene-expression%20patterns&amp;author=LJ%20van%E2%80%99t%20Veer&amp;author=R%20Bernards&amp;volume=452&amp;publication_year=2008&amp;pages=564-570&amp;pmid=18385730&amp;doi=10.1038/nature06915&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR34\"> <span class=\"label\"> 34. </span> <cite> Starmans MH, Pintilie M, John T, Der SD, Shepherd FA, Jurisica I, Lambin P, Tsao MS, Boutros PC. Exploiting the noise: improving biomarkers with ensembles of data analysis methodologies. Genome Med. 2012;4:84. doi: 10.1186/gm385. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/gm385\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3580418/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23146350/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Genome%20Med&amp;title=Exploiting%20the%20noise:%20improving%20biomarkers%20with%20ensembles%20of%20data%20analysis%20methodologies&amp;author=MH%20Starmans&amp;author=M%20Pintilie&amp;author=T%20John&amp;author=SD%20Der&amp;author=FA%20Shepherd&amp;volume=4&amp;publication_year=2012&amp;pages=84&amp;pmid=23146350&amp;doi=10.1186/gm385&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR35\"> <span class=\"label\"> 35. </span> <cite> Starmans MH, Fung G, Steck H, Wouters BG, Lambin P. A simple but highly effective approach to evaluate the prognostic performance of gene expression signatures. PLoS One. 2011;6:e28320. doi: 10.1371/journal.pone.0028320. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1371/journal.pone.0028320\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3233554/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/22163293/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=A%20simple%20but%20highly%20effective%20approach%20to%20evaluate%20the%20prognostic%20performance%20of%20gene%20expression%20signatures&amp;author=MH%20Starmans&amp;author=G%20Fung&amp;author=H%20Steck&amp;author=BG%20Wouters&amp;author=P%20Lambin&amp;volume=6&amp;publication_year=2011&amp;pages=e28320&amp;pmid=22163293&amp;doi=10.1371/journal.pone.0028320&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR36\"> <span class=\"label\"> 36. </span> <cite> Venet D, Dumont JE, Detours V. Most random gene expression signatures are significantly associated with breast cancer outcome. PLoS Comput Biol. 2011;7:e1002240. doi: 10.1371/journal.pcbi.1002240. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1371/journal.pcbi.1002240\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3197658/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/22028643/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=PLoS%20Comput%20Biol&amp;title=Most%20random%20gene%20expression%20signatures%20are%20significantly%20associated%20with%20breast%20cancer%20outcome&amp;author=D%20Venet&amp;author=JE%20Dumont&amp;author=V%20Detours&amp;volume=7&amp;publication_year=2011&amp;pages=e1002240&amp;pmid=22028643&amp;doi=10.1371/journal.pcbi.1002240&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR37\"> <span class=\"label\"> 37. </span> <cite> Boutros PC, Lau SK, Pintilie M, Liu N, Shepherd FA, Der SD, Tsao MS, Penn LZ, Jurisica I. Prognostic gene signatures for non-small-cell lung cancer. Proc Natl Acad Sci U S A. 2009;106:2824–2828. doi: 10.1073/pnas.0809444106. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1073/pnas.0809444106\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2636731/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19196983/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=Prognostic%20gene%20signatures%20for%20non-small-cell%20lung%20cancer&amp;author=PC%20Boutros&amp;author=SK%20Lau&amp;author=M%20Pintilie&amp;author=N%20Liu&amp;author=FA%20Shepherd&amp;volume=106&amp;publication_year=2009&amp;pages=2824-2828&amp;pmid=19196983&amp;doi=10.1073/pnas.0809444106&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR38\"> <span class=\"label\"> 38. </span> <cite> Bentzien J, Muegge I, Hamner B, Thompson DC. Crowd computing: using competitive dynamics to develop and refine highly predictive models. Drug Discov Today. 2013;18:472–478. doi: 10.1016/j.drudis.2013.01.002. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.drudis.2013.01.002\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23337388/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Drug%20Discov%20Today&amp;title=Crowd%20computing:%20using%20competitive%20dynamics%20to%20develop%20and%20refine%20highly%20predictive%20models&amp;author=J%20Bentzien&amp;author=I%20Muegge&amp;author=B%20Hamner&amp;author=DC%20Thompson&amp;volume=18&amp;publication_year=2013&amp;pages=472-478&amp;pmid=23337388&amp;doi=10.1016/j.drudis.2013.01.002&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR39\"> <span class=\"label\"> 39. </span> <cite> Shi L, Campbell G, Jones WD, Campagne F, Wen Z, Walker SJ, Su Z, Chu TM, Goodsaid FM, Pusztai L, Shaughnessy JD, Jr, Oberthuer A, Thomas RS, Paules RS, Fielden M, Barlogie B, Chen W, Du P, Fischer M, Furlanello C, Gallas BD, Ge X, Megherbi DB, Symmans WF, Wang MD, Zhang J, Bitter H, Brors B, Bushel PR, Bylesjo M, et al. The MicroArray Quality Control (MAQC)-II study of common practices for the development and validation of microarray-based predictive models. Nat Biotechnol. 2010;28:827–838. doi: 10.1038/nbt.1665. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nbt.1665\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3315840/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/20676074/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&amp;title=The%20MicroArray%20Quality%20Control%20(MAQC)-II%20study%20of%20common%20practices%20for%20the%20development%20and%20validation%20of%20microarray-based%20predictive%20models&amp;author=L%20Shi&amp;author=G%20Campbell&amp;author=WD%20Jones&amp;author=F%20Campagne&amp;author=Z%20Wen&amp;volume=28&amp;publication_year=2010&amp;pages=827-838&amp;pmid=20676074&amp;doi=10.1038/nbt.1665&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR40\"> <span class=\"label\"> 40. </span> <cite> Tarca AL, Lauria M, Unger M, Bilal E, Boue S, Kumar Dey K, Hoeng J, Koeppl H, Martin F, Meyer P, Nandy P, Norel R, Peitsch M, Rice JJ, Romero R, Stolovitzky G, Talikka M, Xiang Y, Zechner C, IMPROVER DSC Collaborators Strengths and limitations of microarray-based phenotype prediction: lessons learned from the IMPROVER Diagnostic Signature Challenge. Bioinformatics. 2013;29:2892–2899. doi: 10.1093/bioinformatics/btt492. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/bioinformatics/btt492\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3810846/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23966112/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Strengths%20and%20limitations%20of%20microarray-based%20phenotype%20prediction:%20lessons%20learned%20from%20the%20IMPROVER%20Diagnostic%20Signature%20Challenge&amp;author=AL%20Tarca&amp;author=M%20Lauria&amp;author=M%20Unger&amp;author=E%20Bilal&amp;author=S%20Boue&amp;volume=29&amp;publication_year=2013&amp;pages=2892-2899&amp;pmid=23966112&amp;doi=10.1093/bioinformatics/btt492&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR41\"> <span class=\"label\"> 41. </span> <cite> <em> Acute Myeloid Leukemia Outcome Prediction Challenge </em> ; [ <a class=\"usa-link usa-link--external\" href=\"https://www.synapse.org/#!Synapse:syn2455683\" rel=\"noopener noreferrer\" target=\"_blank\"> https://www.synapse.org/#!Synapse:syn2455683 </a> ] </cite> </li> <li id=\"CR42\"> <span class=\"label\"> 42. </span> <cite> Pujana MA, Han JD, Starita LM, Stevens KN, Tewari M, Ahn JS, Rennert G, Moreno V, Kirchhoff T, Gold B, Assmann V, Elshamy WM, Rual JF, Levine D, Rozek LS, Gelman RS, Gunsalus KC, Greenberg RA, Sobhian B, Bertin N, Venkatesan K, Ayivi-Guedehoussou N, Solé X, Hernández P, Lázaro C, Nathanson KL, Weber BL, Cusick ME, Hill DE, Offit K, et al. Network modeling links breast cancer susceptibility and centrosome dysfunction. Nat Genet. 2007;39:1338–1349. doi: 10.1038/ng.2007.2. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/ng.2007.2\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/17922014/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&amp;title=Network%20modeling%20links%20breast%20cancer%20susceptibility%20and%20centrosome%20dysfunction&amp;author=MA%20Pujana&amp;author=JD%20Han&amp;author=LM%20Starita&amp;author=KN%20Stevens&amp;author=M%20Tewari&amp;volume=39&amp;publication_year=2007&amp;pages=1338-1349&amp;pmid=17922014&amp;doi=10.1038/ng.2007.2&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR43\"> <span class=\"label\"> 43. </span> <cite> Taylor IW, Linding R, Warde-Farley D, Liu Y, Pesquita C, Faria D, Bull S, Pawson T, Morris Q, Wrana JL. Dynamic modularity in protein interaction networks predicts breast cancer outcome. Nat Biotechnol. 2009;27:199–204. doi: 10.1038/nbt.1522. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nbt.1522\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19182785/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&amp;title=Dynamic%20modularity%20in%20protein%20interaction%20networks%20predicts%20breast%20cancer%20outcome&amp;author=IW%20Taylor&amp;author=R%20Linding&amp;author=D%20Warde-Farley&amp;author=Y%20Liu&amp;author=C%20Pesquita&amp;volume=27&amp;publication_year=2009&amp;pages=199-204&amp;pmid=19182785&amp;doi=10.1038/nbt.1522&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR44\"> <span class=\"label\"> 44. </span> <cite> Vaske CJ, Benz SC, Sanborn JZ, Earl D, Szeto C, Zhu J, Haussler D, Stuart JM. Inference of patient-specific pathway activities from multi-dimensional cancer genomics data using PARADIGM. Bioinformatics. 2010;26:i237–i245. doi: 10.1093/bioinformatics/btq182. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/bioinformatics/btq182\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2881367/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/20529912/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Inference%20of%20patient-specific%20pathway%20activities%20from%20multi-dimensional%20cancer%20genomics%20data%20using%20PARADIGM&amp;author=CJ%20Vaske&amp;author=SC%20Benz&amp;author=JZ%20Sanborn&amp;author=D%20Earl&amp;author=C%20Szeto&amp;volume=26&amp;publication_year=2010&amp;pages=i237-i245&amp;pmid=20529912&amp;doi=10.1093/bioinformatics/btq182&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR45\"> <span class=\"label\"> 45. </span> <cite> Marbach D, Prill RJ, Schaffter T, Mattiussi C, Floreano D, Stolovitzky G. Revealing strengths and weaknesses of methods for gene network inference. Proc Natl Acad Sci U S A. 2010;107:6286–6291. doi: 10.1073/pnas.0913357107. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1073/pnas.0913357107\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2851985/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/20308593/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=Revealing%20strengths%20and%20weaknesses%20of%20methods%20for%20gene%20network%20inference&amp;author=D%20Marbach&amp;author=RJ%20Prill&amp;author=T%20Schaffter&amp;author=C%20Mattiussi&amp;author=D%20Floreano&amp;volume=107&amp;publication_year=2010&amp;pages=6286-6291&amp;pmid=20308593&amp;doi=10.1073/pnas.0913357107&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR46\"> <span class=\"label\"> 46. </span> <cite> Marbach D, Costello JC, Kuffner R, Vega NM, Prill RJ, Camacho DM, Allison KR, Consortium D, Kellis M, Collins JJ, Stolovitzky G. Wisdom of crowds for robust gene network inference. Nat Methods. 2012;9:796–804. doi: 10.1038/nmeth.2016. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nmeth.2016\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3512113/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/22796662/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&amp;title=Wisdom%20of%20crowds%20for%20robust%20gene%20network%20inference&amp;author=D%20Marbach&amp;author=JC%20Costello&amp;author=R%20Kuffner&amp;author=NM%20Vega&amp;author=RJ%20Prill&amp;volume=9&amp;publication_year=2012&amp;pages=796-804&amp;pmid=22796662&amp;doi=10.1038/nmeth.2016&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR47\"> <span class=\"label\"> 47. </span> <cite> Prill RJ, Saez-Rodriguez J, Alexopoulos LG, Sorger PK, Stolovitzky G. Crowdsourcing network inference: the DREAM predictive signaling network challenge. Sci Signal. 2011;4:mr7. doi: 10.1126/scisignal.2002212. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1126/scisignal.2002212\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3465072/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/21900204/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Sci%20Signal&amp;title=Crowdsourcing%20network%20inference:%20the%20DREAM%20predictive%20signaling%20network%20challenge&amp;author=RJ%20Prill&amp;author=J%20Saez-Rodriguez&amp;author=LG%20Alexopoulos&amp;author=PK%20Sorger&amp;author=G%20Stolovitzky&amp;volume=4&amp;publication_year=2011&amp;pages=mr7&amp;pmid=21900204&amp;doi=10.1126/scisignal.2002212&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR48\"> <span class=\"label\"> 48. </span> <cite> <em> HPN-DREAM breast cancer network inference challenge </em> ; [ <a class=\"usa-link usa-link--external\" href=\"https://www.synapse.org/#!Synapse:syn1720047\" rel=\"noopener noreferrer\" target=\"_blank\"> https://www.synapse.org/#!Synapse:syn1720047 </a> ] </cite> </li> <li id=\"CR49\"> <span class=\"label\"> 49. </span> <cite> Maier R, Zimmer R, Kuffner R. A Turing test for artificial expression data. Bioinformatics. 2013;29:2603–2609. doi: 10.1093/bioinformatics/btt438. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/bioinformatics/btt438\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23956305/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=A%20Turing%20test%20for%20artificial%20expression%20data&amp;author=R%20Maier&amp;author=R%20Zimmer&amp;author=R%20Kuffner&amp;volume=29&amp;publication_year=2013&amp;pages=2603-2609&amp;pmid=23956305&amp;doi=10.1093/bioinformatics/btt438&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR50\"> <span class=\"label\"> 50. </span> <cite> Canales RD, Luo Y, Willey JC, Austermiller B, Barbacioru CC, Boysen C, Hunkapiller K, Jensen RV, Knight CR, Lee KY, Ma Y, Maqsodi B, Papallo A, Peters EH, Poulter K, Ruppel PL, Samaha RR, Shi L, Yang W, Zhang L, Goodsaid FM. Evaluation of DNA microarray results with quantitative gene expression platforms. Nat Biotechnol. 2006;24:1115–1122. doi: 10.1038/nbt1236. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nbt1236\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/16964225/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&amp;title=Evaluation%20of%20DNA%20microarray%20results%20with%20quantitative%20gene%20expression%20platforms&amp;author=RD%20Canales&amp;author=Y%20Luo&amp;author=JC%20Willey&amp;author=B%20Austermiller&amp;author=CC%20Barbacioru&amp;volume=24&amp;publication_year=2006&amp;pages=1115-1122&amp;pmid=16964225&amp;doi=10.1038/nbt1236&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR51\"> <span class=\"label\"> 51. </span> <cite> Roberts ND, Kortschak RD, Parker WT, Schreiber AW, Branford S, Scott HS, Glonek G, Adelson DL. A comparative analysis of algorithms for somatic SNV detection in cancer. Bioinformatics. 2013;29:2223–2230. doi: 10.1093/bioinformatics/btt375. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/bioinformatics/btt375\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3753564/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23842810/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=A%20comparative%20analysis%20of%20algorithms%20for%20somatic%20SNV%20detection%20in%20cancer&amp;author=ND%20Roberts&amp;author=RD%20Kortschak&amp;author=WT%20Parker&amp;author=AW%20Schreiber&amp;author=S%20Branford&amp;volume=29&amp;publication_year=2013&amp;pages=2223-2230&amp;pmid=23842810&amp;doi=10.1093/bioinformatics/btt375&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR52\"> <span class=\"label\"> 52. </span> <cite> Bell AW, Deutsch EW, Au CE, Kearney RE, Beavis R, Sechi S, Nilsson T, Bergeron JJ, Group HTSW. A HUPO test sample study reveals common problems in mass spectrometry-based proteomics. Nat Methods. 2009;6:423–430. doi: 10.1038/nmeth.1333. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nmeth.1333\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2785450/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19448641/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&amp;title=A%20HUPO%20test%20sample%20study%20reveals%20common%20problems%20in%20mass%20spectrometry-based%20proteomics&amp;author=AW%20Bell&amp;author=EW%20Deutsch&amp;author=CE%20Au&amp;author=RE%20Kearney&amp;author=R%20Beavis&amp;volume=6&amp;publication_year=2009&amp;pages=423-430&amp;pmid=19448641&amp;doi=10.1038/nmeth.1333&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR53\"> <span class=\"label\"> 53. </span> <cite> ‘t Hoen PA, Friedländer MR, Almlöf J, Sammeth M, Pulyakhina I, Anvar SY, Laros JF, Buermans HP, Karlberg O, Brännvall M, GEUVADIS Consortium. den Dunnen JT, van Ommen GJ, Gut IG, Guigó R, Estivill X, Syvänen AC, Dermitzakis ET, Lappalainen T. Reproducibility of high-throughput mRNA and small RNA sequencing across laboratories. Nat Biotechnol. 2013;31:1015–1022. doi: 10.1038/nbt.2702. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nbt.2702\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24037425/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&amp;title=Reproducibility%20of%20high-throughput%20mRNA%20and%20small%20RNA%20sequencing%20across%20laboratories&amp;author=PA%20%E2%80%98t%20Hoen&amp;author=MR%20Friedl%C3%A4nder&amp;author=J%20Alml%C3%B6f&amp;author=M%20Sammeth&amp;author=I%20Pulyakhina&amp;volume=31&amp;publication_year=2013&amp;pages=1015-1022&amp;pmid=24037425&amp;doi=10.1038/nbt.2702&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR54\"> <span class=\"label\"> 54. </span> <cite> Steijger T, Abril JF, Engstrom PG, Kokocinski F, Consortium R, Akerman M, Alioto T, Ambrosini G, Antonarakis SE, Behr J, Bertone P. Assessment of transcript reconstruction methods for RNA-seq. Nat Methods. 2013;10:1177–1184. doi: 10.1038/nmeth.2714. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nmeth.2714\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3851240/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24185837/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&amp;title=Assessment%20of%20transcript%20reconstruction%20methods%20for%20RNA-seq&amp;author=T%20Steijger&amp;author=JF%20Abril&amp;author=PG%20Engstrom&amp;author=F%20Kokocinski&amp;author=R%20Consortium&amp;volume=10&amp;publication_year=2013&amp;pages=1177-1184&amp;pmid=24185837&amp;doi=10.1038/nmeth.2714&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR55\"> <span class=\"label\"> 55. </span> <cite> Ransohoff DF. Proteomics research to discover markers: what can we learn from Netflix? Clin Chem. 2010;56:172–176. doi: 10.1373/clinchem.2009.126698. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1373/clinchem.2009.126698\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/20040622/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Clin%20Chem&amp;title=Proteomics%20research%20to%20discover%20markers:%20what%20can%20we%20learn%20from%20Netflix?&amp;author=DF%20Ransohoff&amp;volume=56&amp;publication_year=2010&amp;pages=172-176&amp;pmid=20040622&amp;doi=10.1373/clinchem.2009.126698&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR56\"> <span class=\"label\"> 56. </span> <cite> Waters H. New $10 million X Prize launched for tricorder-style medical device. Nat Med. 2011;17:754. doi: 10.1038/nm0711-754a. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nm0711-754a\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/21738131/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat%20Med&amp;title=New%20%2410%20million%20X%20Prize%20launched%20for%20tricorder-style%20medical%20device&amp;author=H%20Waters&amp;volume=17&amp;publication_year=2011&amp;pages=754&amp;pmid=21738131&amp;doi=10.1038/nm0711-754a&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> </ul> </section> </section> </section> <footer class=\"p courtesy-note font-secondary font-sm text-center\"> <hr class=\"headless\"/> <p> Articles from Genome Biology are provided here courtesy of <strong> BMC </strong> </p> </footer> </section> </article> </main> </div> </div> </div> <div class=\"pmc-sidenav desktop:grid-col-4 display-flex\"> <section aria-label=\"Article resources and navigation\" class=\"pmc-sidenav__container\"> <div class=\"display-none desktop:display-block\"> <section class=\"margin-top-4 desktop:margin-top-0\"> <h2 class=\"margin-top-0\"> ACTIONS </h2> <ul class=\"usa-list usa-list--unstyled usa-list--actions\"> <li> <a class=\"usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1\" href=\"https://doi.org/10.1186/s13059-014-0462-7\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"display-inline-flex flex-justify-center flex-1 padding-right-2\"> View on publisher site </span> </a> </li> <li> <a class=\"usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1\" href=\"pdf/13059_2014_Article_462.pdf\"> <span class=\"display-inline-flex flex-justify-center flex-1\"> PDF (542.9 KB) </span> </a> </li> <li> </li> <li> </li> <li class=\"pmc-permalink\"> <div class=\"pmc-permalink__dropdown\" hidden=\"\"> <div class=\"pmc-permalink__dropdown__container\"> <h2 class=\"usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs\"> PERMALINK </h2> <div class=\"pmc-permalink__dropdown__content\"> </div> </div> </div> </li> </ul> </section> </div> <section class=\"pmc-resources margin-top-6 desktop:margin-top-4\"> <h2 class=\"margin-top-0\"> RESOURCES </h2> <div class=\"usa-accordion usa-accordion--multiselectable\"> <h3 class=\"usa-accordion__heading\"> </h3> <div class=\"usa-accordion__content usa-prose\" hidden=\"\" id=\"resources-similar-articles\"> </div> <h3 class=\"usa-accordion__heading\"> </h3> <div class=\"usa-accordion__content usa-prose\" hidden=\"\" id=\"resources-cited-by-other-articles\"> </div> <h3 class=\"usa-accordion__heading\"> </h3> <div class=\"usa-accordion__content usa-prose\" hidden=\"\" id=\"resources-links-to-ncbi-databases\"> </div> </div> </section> <section class=\"usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4\"> <nav aria-label=\"On this page\" class=\"usa-in-page-nav__nav\"> <h2 class=\"usa-in-page-nav__heading\" tabindex=\"0\"> On this page </h2> <ul class=\"usa-in-page-nav__list\"> <li class=\"usa-in-page-nav__item\"> <a class=\"usa-in-page-nav__link\" href=\"#Abs1\"> Abstract </a> </li> <li class=\"usa-in-page-nav__item\"> <a class=\"usa-in-page-nav__link\" href=\"#Sec1\"> Challenge design and dynamics </a> </li> <li class=\"usa-in-page-nav__item\"> <a class=\"usa-in-page-nav__link\" href=\"#Sec2\"> Analyzing genome assembly and structural variants </a> </li> <li class=\"usa-in-page-nav__item\"> <a class=\"usa-in-page-nav__link\" href=\"#Sec3\"> Finding genomic biomarkers that are associated with phenotype </a> </li> <li class=\"usa-in-page-nav__item\"> <a class=\"usa-in-page-nav__link\" href=\"#Sec4\"> Inferring biological networks underlying cancer phenotypes </a> </li> <li class=\"usa-in-page-nav__item\"> <a class=\"usa-in-page-nav__link\" href=\"#Sec5\"> The truth is hard to find </a> </li> <li class=\"usa-in-page-nav__item\"> <a class=\"usa-in-page-nav__link\" href=\"#Sec6\"> Closing considerations </a> </li> <li class=\"usa-in-page-nav__item\"> <a class=\"usa-in-page-nav__link\" href=\"#ack1\"> Acknowledgements </a> </li> <li class=\"usa-in-page-nav__item\"> <a class=\"usa-in-page-nav__link\" href=\"#glossary1\"> Abbreviations </a> </li> <li class=\"usa-in-page-nav__item\"> <a class=\"usa-in-page-nav__link\" href=\"#fn-group1\"> Footnotes </a> </li> <li class=\"usa-in-page-nav__item\"> <a class=\"usa-in-page-nav__link\" href=\"#_ci93_\"> Contributor Information </a> </li> <li class=\"usa-in-page-nav__item\"> <a class=\"usa-in-page-nav__link\" href=\"#Bib1\"> References </a> </li> </ul> </nav> </section> </section> </div> <div aria-label=\"Citation Dialog\" class=\"overlay citation-dialog-overlay\" role=\"dialog\"> <div aria-hidden=\"true\" class=\"dialog citation-dialog\" role=\"document\"> <div class=\"display-inline-flex flex-align-center flex-justify width-full margin-bottom-2\"> <h2 class=\"usa-modal__heading margin-0\"> Cite </h2> </div> <div class=\"citation-text-block\"> <div class=\"citation-text margin-bottom-2\"> </div> <ul class=\"usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center\"> <li> </li> <li> <a class=\"usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button\" href=\"#\" role=\"button\" tabindex=\"3\" title=\"Download a file for external citation management software\"> <span class=\"display-none mobile-lg:display-inline\"> Download .nbib </span> <span class=\"display-inline mobile-lg:display-none\"> .nbib </span> </a> </li> <li> <div class=\"display-inline-flex flex-align-center\"> <label class=\"usa-label margin-top-0\"> Format: </label> <select aria-label=\"Format\" class=\"usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4\" tabindex=\"4\"> <option value=\"AMA\"> AMA </option> <option value=\"APA\"> APA </option> <option value=\"MLA\"> MLA </option> <option selected=\"selected\" value=\"NLM\"> NLM </option> </select> </div> </li> </ul> <div class=\"dots-loading-indicator citation-loading-indicator\"> <div class=\"dot dot-1\"> </div> <div class=\"dot dot-2\"> </div> <div class=\"dot dot-3\"> </div> </div> </div> </div> </div> <div class=\"overlay collections-dialog-overlay\" role=\"dialog\"> <div aria-hidden=\"true\" class=\"dialog collections-dialog\" id=\"collections-action-dialog\" role=\"document\"> <div class=\"display-inline-flex flex-align-center flex-justify width-full margin-bottom-2\"> <h2 class=\"usa-modal__heading margin-0\"> Add to Collections </h2> </div> <div class=\"collections-action-panel action-panel\"> </div> </div> </div> </div> </div> </div> <footer class=\"ncbi-footer ncbi-dark-background\"> <div class=\"ncbi-footer__icon-section\"> <div class=\"ncbi-footer__social-header\"> Follow NCBI </div> <div class=\"grid-container ncbi-footer__ncbi-social-icons-container\"> <a class=\"ncbi-footer__social-icon ncbi-footer__social-icon--gray\" href=\"https://twitter.com/ncbi\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"usa-sr-only\"> NCBI on X (formerly known as Twitter) </span> </a> <a class=\"ncbi-footer__social-icon ncbi-footer__social-icon--gray\" href=\"https://www.facebook.com/ncbi.nlm\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"usa-sr-only\"> NCBI on Facebook </span> </a> <a class=\"ncbi-footer__social-icon ncbi-footer__social-icon--gray\" href=\"https://www.linkedin.com/company/ncbinlm\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"usa-sr-only\"> NCBI on LinkedIn </span> </a> <a class=\"ncbi-footer__social-icon ncbi-footer__social-icon--gray\" href=\"https://github.com/ncbi\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"usa-sr-only\"> NCBI on GitHub </span> </a> <a class=\"ncbi-footer__social-icon ncbi-footer__social-icon--gray\" href=\"https://ncbiinsights.ncbi.nlm.nih.gov/\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"usa-sr-only\"> NCBI RSS feed </span> </a> </div> </div> <div class=\"grid-container ncbi-footer__container\"> <div class=\"grid-row ncbi-footer__main-content-container\"> <div class=\"ncbi-footer__column\"> <p class=\"ncbi-footer__circled-icons-heading\"> Connect with NLM </p> <div class=\"ncbi-footer__circled-icons-list\"> <a class=\"ncbi-footer__social-icon ncbi-footer__social-icon--circled\" href=\"https://twitter.com/nlm_nih\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"usa-sr-only\"> NLM on X (formerly known as Twitter) </span> </a> <a class=\"ncbi-footer__social-icon ncbi-footer__social-icon--circled\" href=\"https://www.facebook.com/nationallibraryofmedicine\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"usa-sr-only\"> NLM on Facebook </span> </a> <a class=\"ncbi-footer__social-icon ncbi-footer__social-icon--circled\" href=\"https://www.youtube.com/user/NLMNIH\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"usa-sr-only\"> NLM on YouTube </span> </a> </div> </div> <address class=\"ncbi-footer__address ncbi-footer__column\"> <p> <a class=\"usa-link usa-link--external\" href=\"https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508, -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508! 4d-77.0988323\" rel=\"noopener noreferrer\" target=\"_blank\"> National Library of Medicine <br/> 8600 Rockville Pike <br/> Bethesda, MD 20894 </a> </p> </address> <ul class=\"usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column\"> <li class=\"ncbi-footer__vertical-list-item\"> <a class=\"usa-link usa-link--alt ncbi-footer__link\" href=\"https://www.nlm.nih.gov/web_policies.html\"> Web Policies </a> </li> <li class=\"ncbi-footer__vertical-list-item\"> <a class=\"usa-link usa-link--alt ncbi-footer__link\" href=\"https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office\"> FOIA </a> </li> <li class=\"ncbi-footer__vertical-list-item\"> <a class=\"usa-link usa-link--external usa-link--alt ncbi-footer__link\" href=\"https://www.hhs.gov/vulnerability-disclosure-policy/index.html\" rel=\"noreferrer noopener\" target=\"_blank\"> HHS Vulnerability Disclosure </a> </li> </ul> <ul class=\"usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column\"> <li class=\"ncbi-footer__vertical-list-item\"> <a class=\"usa-link usa-link--alt ncbi-footer__link\" href=\"https://support.nlm.nih.gov/?pagename=cloudpmc-viewer%3Apmc%3Aarticle%3Ajournal\"> Help </a> </li> <li class=\"ncbi-footer__vertical-list-item\"> <a class=\"usa-link usa-link--alt ncbi-footer__link\" href=\"https://www.nlm.nih.gov/accessibility.html\"> Accessibility </a> </li> <li class=\"ncbi-footer__vertical-list-item\"> <a class=\"usa-link usa-link--alt ncbi-footer__link\" href=\"https://www.nlm.nih.gov/careers/careers.html\"> Careers </a> </li> </ul> </div> <div class=\"grid-row grid-col-12\"> <ul class=\"ncbi-footer__bottom-links-list\"> <li class=\"ncbi-footer__bottom-list-item\"> <a class=\"usa-link usa-link--alt ncbi-footer__link\" href=\"https://www.nlm.nih.gov/\"> NLM </a> </li> <li class=\"ncbi-footer__bottom-list-item\"> <a class=\"usa-link usa-link--alt ncbi-footer__link\" href=\"https://www.nih.gov/\"> NIH </a> </li> <li class=\"ncbi-footer__bottom-list-item\"> <a class=\"usa-link usa-link--external usa-link--alt ncbi-footer__link\" href=\"https://www.hhs.gov/\" rel=\"noreferrer noopener\" target=\"_blank\"> HHS </a> </li> <li class=\"ncbi-footer__bottom-list-item\"> <a class=\"usa-link usa-link--external usa-link--alt ncbi-footer__link\" href=\"https://www.usa.gov/\" rel=\"noreferrer noopener\" target=\"_blank\"> USA.gov </a> </li> </ul> </div> </div> </footer> <div id=\"ZN_dikYWqsjiUWN0Q5\"> </div> <div class=\"fake-body-scroll\"> </div> <div class=\"fake-body-scroll\"> </div> <div class=\"QSIFeedbackButton\"> </div> </body> </html>', 'repos': 'bigd.big.ac.cn, ega-archive.org, massive.ucsd.edu, www.ebi.ac.uk, www.iprox.cn, proteomecentral.proteomexchange.org, www.proteomexchange.org, pdc.cancer.gov, gdc.cancer.gov, cancerimagingarchive.net, imaging.datacommons.cancer.gov, repository.jpostdb.org, geo, dbgap, sra, arrayexpress, nuccore, data.mendeley.com, synapse.org, doi.org, pdb'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1329 - DEBUG - Counting tokens for model: gemini-2.0-flash, prompt length: 140801 char\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1338 - DEBUG - Rough estimate of token count for Gemini model 'gemini-2.0-flash': 35200\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 326 - INFO - Prompt messages total length: 35200 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 327 - DEBUG - Prompt messages: [{'role': 'developer', 'content': 'I am a large language model trained to be informative and comprehensive. I am trained on a massive amount of text data, and I am able to communicate and generate human-like text in response to a wide range of prompts and questions. For this task, I will act as a specialized assistant that can identify datasets mentioned in a publication and create a summary suitable for non-specialists.\\n\\nThe output should be a JSON array of objects, where each object has the following keys:\\n- \"dataset_identifier\": This is any alphanumeric string (maybe including punctuation marks) that uniquely identifies or provides access to a dataset, most frequently an accession code.\\n- \"repository_reference\": This is the URL or reference to the data repository where the dataset can be found.\\n\\nHere are some examples for reference:\\n\\n[\\'dataset_identifier\\'=> \\'EGAS00001000925\\', \\'repository_reference\\'=> \\'https://ega-archive.org/datasets/EGAS00001000925\\',\\n \\'dataset_identifier\\'=> \\'GSE69091\\', \\'repository_reference\\'=> \\'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE69091\\',\\n \\'dataset_identifier\\'=> \\'PRJNA306801\\', \\'repository_reference\\'=> \\'https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA306801\\',\\n \\'dataset_identifier\\'=> \\'phs003416.v1.p1\\', \\'repository_reference\\'=> \\'dbGaP\\',\\n \\'dataset_identifier\\'=> \\'PXD049309\\', \\'repository_reference\\'=> \\'https://www.ebi.ac.uk/pride/archive/projects/PXD049309\\',\\n \\'dataset_identifier\\'=> \\'PXD0402123\\', \\'repository_reference\\'=> \\'ProteomeXchange\\',\\n \\'dataset_identifier\\'=> \\'IPX0004230000\\', \\'repository_reference\\'=> \\'http://www.iprox.org\\',\\n \\'dataset_identifier\\'=> \\'MSV000092944\\', \\'repository_reference\\'=> \\'https://massive.ucsd.edu/\\',\\n \\'dataset_identifier\\'=> \\'syn1827001\\', \\'repository_reference\\'=> \\'https://www.synapse.org/[#!]*Synapse:syn28580053\\',\\n \\'dataset_identifier\\'=> \\'n/a\\', \\'repository_reference\\'=> \\'https://data.broadinstitute.org/ccle_legacy_data/mRNA_expression/\\'\\n]'}, {'role': 'user', 'content': 'Given the information that I am going to share:\\n 1) A sample of already known data repositories.\\n2) The webpage in HTML or XML format from which you have to extract datasets information.\\n\\nPlease return a JSON array of objects where each object has the following structure:\\n- `dataset_identifier`: The dataset identifier (a code). If not found, set it to \"n/a\".\\n- `repository_reference`: The URL or reference to the data repository. If not found, set it to \"n/a\".\\n\\nPlease follow these strict instructions:\\n- The output must be a valid JSON array of objects.\\n- Each object must contain the keys `dataset_identifier` and `repository_reference`.\\n- Any other output format will be considered invalid.\\n\\nBelow is the input data that you will use to generate the output:\\n1) repos => bigd.big.ac.cn, ega-archive.org, massive.ucsd.edu, www.ebi.ac.uk, www.iprox.cn, proteomecentral.proteomexchange.org, www.proteomexchange.org, pdc.cancer.gov, gdc.cancer.gov, cancerimagingarchive.net, imaging.datacommons.cancer.gov, repository.jpostdb.org, geo, dbgap, sra, arrayexpress, nuccore, data.mendeley.com, synapse.org, doi.org, pdb\\n2) html => <html class=\"\" lang=\"en\"> <head> <link href=\"/static/assets/style-8258669d.css\" rel=\"stylesheet\"/> <link href=\"/static/assets/style-ef962842.css\" rel=\"stylesheet\"/> <link href=\"/static/assets/style-3ade8b5c.css\" rel=\"stylesheet\"/> <link href=\"/static/img/favicons/apple-touch-icon.png\" rel=\"apple-touch-icon\" sizes=\"180x180\"/> <link href=\"/static/img/favicons/favicon-48x48.png\" rel=\"icon\" sizes=\"48x48\" type=\"image/png\"/> <link href=\"/static/img/favicons/favicon-32x32.png\" rel=\"icon\" sizes=\"32x32\" type=\"image/png\"/> <link href=\"/static/img/favicons/favicon-16x16.png\" rel=\"icon\" sizes=\"16x16\" type=\"image/png\"/> <link href=\"/static/img/favicons/site.webmanifest\" rel=\"manifest\"/> <link color=\"#0071bc\" href=\"/static/img/favicons/safari-pinned-tab.svg\" rel=\"mask-icon\"/> <title> Toward better benchmarking: challenge-based methods assessment in cancer genomics - PMC </title> <link href=\"https://www.google-analytics.com\" rel=\"preconnect\"/> <link href=\"https://cdn.ncbi.nlm.nih.gov\" rel=\"preconnect\"/> <link href=\"https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527/\" rel=\"canonical\"/> </head> <body> <a class=\"usa-skipnav\" href=\"#main-content\"> Skip to main content </a> <section aria-label=\"Official website of the United States government\" class=\"usa-banner\"> <div class=\"usa-accordion\"> <header class=\"usa-banner__header\"> <div class=\"usa-banner__inner\"> <div class=\"grid-col-auto\"> </div> <div aria-hidden=\"true\" class=\"grid-col-fill tablet:grid-col-auto\"> <p class=\"usa-banner__header-text\"> An official website of the United States government </p> <span class=\"usa-banner__header-action\"> Here\\'s how you know </span> </div> </div> </header> <div class=\"usa-banner__content usa-accordion__content\" hidden=\"\" id=\"gov-banner-default\"> <div class=\"grid-row grid-gap-lg\"> <div class=\"usa-banner__guidance tablet:grid-col-6\"> <div class=\"usa-media-block__body\"> <p> <strong> Official websites use .gov </strong> <br/> A <strong> .gov </strong> website belongs to an official government organization in the United States. </p> </div> </div> <div class=\"usa-banner__guidance tablet:grid-col-6\"> <div class=\"usa-media-block__body\"> <p> <strong> Secure .gov websites use HTTPS </strong> <br/> A <strong> lock </strong> ( <span class=\"icon-lock\"> </span> ) or <strong> https:// </strong> means you\\'ve safely connected to the .gov website. Share sensitive information only on official, secure websites. </p> </div> </div> </div> </div> </div> </section> <div class=\"usa-overlay\"> </div> <header class=\"usa-header usa-header--extended usa-header--wide\"> <div class=\"ncbi-header\"> <div class=\"ncbi-header__container\"> <a class=\"ncbi-header__logo-container\" href=\"https://www.ncbi.nlm.nih.gov/\"> </a> <div class=\"ncbi-header__desktop-buttons\"> <div class=\"ncbi-header__login-dropdown\"> <ul class=\"usa-nav__submenu ncbi-header__login-dropdown-menu\" hidden=\"\" id=\"login-dropdown-menu\"> <li class=\"usa-nav__submenu-item\"> <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/myncbi/\"> Dashboard </a> </li> <li class=\"usa-nav__submenu-item\"> <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/\"> Publications </a> </li> <li class=\"usa-nav__submenu-item\"> <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/account/settings/\"> Account settings </a> </li> <li class=\"usa-nav__submenu-item\"> </li> </ul> </div> </div> </div> </div> <div class=\"ncbi-search-panel ncbi--show-only-at-desktop\" hidden=\"\"> <div class=\"ncbi-search-panel__container\"> </div> </div> <nav aria-label=\"Primary navigation\" class=\"usa-nav\"> <p class=\"usa-sr-only\" id=\"primary-navigation-sr-only-title\"> Primary site navigation </p> <div class=\"ncbi-nav__mobile-login-menu ncbi--hide-at-desktop\" hidden=\"\"> <p class=\"ncbi-nav__mobile-login-menu-status\"> Logged in as: <strong class=\"ncbi-nav__mobile-login-menu-email\"> </strong> </p> <ul class=\"usa-nav__primary usa-accordion\"> <li class=\"usa-nav__primary-item\"> <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/myncbi/\"> Dashboard </a> </li> <li class=\"usa-nav__primary-item\"> <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/\"> Publications </a> </li> <li class=\"usa-nav__primary-item\"> <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/account/settings/\"> Account settings </a> </li> </ul> </div> </nav> </header> <section aria-label=\"PMC Header with search box\" class=\"pmc-header pmc-header--basic\"> <div class=\"pmc-nav-container\"> <div class=\"pmc-header__bar\"> <div class=\"pmc-header__logo\"> <a aria-label=\"PMC Home\" href=\"/\" title=\"Home\"> </a> </div> </div> <div class=\"pmc-header__search\"> <div class=\"display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44\"> <ul class=\"pmc-header__search__menu\"> <li> <a class=\"usa-link\" href=\"/journals/\"> Journal List </a> </li> <li> <a class=\"usa-link\" href=\"/about/userguide/\"> User Guide </a> </li> </ul> </div> </div> </div> </section> <div class=\"usa-section padding-top-0 desktop:padding-top-6 pmc-article-section\"> <div aria-label=\"Actions bar\" class=\"grid-container pmc-actions-bar is-not-intersecting\" role=\"complementary\"> <div class=\"grid-row\"> <div class=\"grid-col-fill display-flex\"> <div class=\"display-flex\"> <ul class=\"usa-list usa-list--unstyled usa-list--horizontal\"> <li class=\"margin-right-2 mobile-lg:margin-right-4 display-flex mob\"> </li> <li class=\"margin-right-2 mobile-lg:margin-right-4 display-flex mob\"> <span class=\"usa-tooltip\"> <a aria-label=\"View on publisher site\" class=\"usa-link display-flex usa-tooltip__trigger\" href=\"https://doi.org/10.1186/s13059-014-0462-7\" rel=\"noreferrer noopener\" role=\"button\" tabindex=\"0\" target=\"_blank\"> </a> <span aria-hidden=\"true\" class=\"usa-tooltip__body\" role=\"tooltip\"> View on publisher site </span> </span> </li> <li class=\"margin-right-2 mobile-lg:margin-right-4 display-flex\"> <span class=\"usa-tooltip\"> <a aria-label=\"Download PDF\" class=\"usa-link display-flex usa-tooltip__trigger\" href=\"pdf/13059_2014_Article_462.pdf\" role=\"button\" tabindex=\"0\"> </a> <span aria-hidden=\"true\" class=\"usa-tooltip__body\" role=\"tooltip\"> Download PDF </span> </span> </li> <li class=\"margin-right-2 mobile-lg:margin-right-4 display-flex\"> <span class=\"usa-tooltip\"> <span aria-hidden=\"true\" class=\"usa-tooltip__body\" role=\"tooltip\"> Add to Collections </span> </span> </li> <li class=\"margin-right-2 mobile-lg:margin-right-4 display-flex\"> <span class=\"usa-tooltip\"> <span aria-hidden=\"true\" class=\"usa-tooltip__body\" role=\"tooltip\"> Cite </span> </span> </li> <li class=\"pmc-permalink display-flex\"> <span class=\"usa-tooltip\"> <span aria-hidden=\"true\" class=\"usa-tooltip__body\" role=\"tooltip\"> Permalink </span> </span> <div class=\"pmc-permalink__dropdown\" hidden=\"\"> <div class=\"pmc-permalink__dropdown__container\"> <h2 class=\"usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs\"> PERMALINK </h2> <div class=\"pmc-permalink__dropdown__content\"> </div> </div> </div> </li> </ul> </div> </div> </div> </div> <div class=\"grid-container desktop:padding-left-6\"> <div class=\"grid-row grid-gap\" id=\"article-container\"> <div class=\"grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content\"> <div class=\"grid-container padding-left-0 padding-right-0\"> <div class=\"grid-row desktop:margin-left-neg-6\"> <div class=\"grid-col-12\"> <div aria-label=\"Disclaimer note\" class=\"pmc-layout__disclaimer\" role=\"complementary\"> As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with, the contents by NLM or the National Institutes of Health. <br/> Learn more: <a class=\"usa-link\" href=\"/about/disclaimer/\"> PMC Disclaimer </a> | <a class=\"usa-link\" href=\"/about/copyright/\"> PMC Copyright Notice </a> </div> </div> </div> <div class=\"grid-row pmc-wm desktop:margin-left-neg-6\"> <main class=\"usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0\" id=\"main-content\"> <section aria-label=\"Journal banner\" class=\"pmc-journal-banner text-center line-height-none\"> <map name=\"pmc-banner-imagemap\"> <area alt=\"Link to Genome Biology\" href=\"https://genomebiology.biomedcentral.com/\" rel=\"noopener noreferrer\" shape=\"default\" target=\"_blank\" title=\"Link to Genome Biology\"/> </map> </section> <article lang=\"en\"> <section aria-label=\"Article citation and metadata\"> <section class=\"pmc-layout__citation font-secondary font-xs\"> <div> <div class=\"display-inline-block\"> </div> . 2014 Sep 17;15(9):462. doi: <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/s13059-014-0462-7\" rel=\"noopener noreferrer\" target=\"_blank\"> 10.1186/s13059-014-0462-7 </a> </div> <nav hidden=\"hidden\" id=\"journal_context_menu\"> <ul class=\"menu-list font-family-ui\" role=\"menu\"> <li role=\"presentation\"> <a class=\"usa-link\" href=\"https://pmc.ncbi.nlm.nih.gov/search/?term=%22Genome%20Biol%22%5Bjour%5D\" role=\"menuitem\"> Search in PMC </a> </li> <li role=\"presentation\"> <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Genome%20Biol%22%5Bjour%5D\" lang=\"en\" role=\"menuitem\"> Search in PubMed </a> </li> <li role=\"presentation\"> <a class=\"usa-link\" href=\"https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Genome%20Biol%22%5BTitle%20Abbreviation%5D\" role=\"menuitem\"> View in NLM Catalog </a> </li> <li role=\"presentation\"> <a class=\"usa-link\" href=\"?term=%22Genome%20Biol%22%5Bjour%5D\" role=\"menuitem\"> Add to search </a> </li> </ul> </nav> </section> <section class=\"front-matter\"> <div class=\"ameta p font-secondary font-xs\"> <hgroup> <h1> Toward better benchmarking: challenge-based methods assessment in cancer genomics </h1> </hgroup> <div class=\"cg p\"> <a aria-describedby=\"id1\" aria-expanded=\"false\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Boutros%20PC%22%5BAuthor%5D\"> <span class=\"name western\"> Paul C Boutros </span> </a> <div hidden=\"hidden\" id=\"id1\"> <h3> <span class=\"name western\"> Paul C Boutros </span> </h3> <div class=\"p\"> <sup> 1 </sup> Informatics &amp; Biocomputing Program, Ontario Institute for Cancer Research, University Avenue, Toronto, ON M5G 0A3 Canada </div> <div class=\"p\"> <sup> 2 </sup> Department of Medical Biophysics, University of Toronto, College Street, Toronto, ON M5G 1L7 Canada </div> <div class=\"p\"> <sup> 3 </sup> Department of Pharmacology &amp; Toxicology, University of Toronto, King’s College Circle, Toronto, ON M5S 1A8 Canada </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Boutros%20PC%22%5BAuthor%5D\"> <span class=\"name western\"> Paul C Boutros </span> </a> </div> </div> <sup> 1, </sup> <sup> 2, </sup> <sup> 3 </sup> , <a aria-describedby=\"id2\" aria-expanded=\"false\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Margolin%20AA%22%5BAuthor%5D\"> <span class=\"name western\"> Adam A Margolin </span> </a> <div hidden=\"hidden\" id=\"id2\"> <h3> <span class=\"name western\"> Adam A Margolin </span> </h3> <div class=\"p\"> <sup> 4 </sup> Sage Bionetworks, Fairview Ave North, Seattle, WA 98109 USA </div> <div class=\"p\"> <sup> 5 </sup> Computational Biology Program, Oregon Health &amp; Science University, SW Sam Jackson Park Road, Portland, OR 97239-3098 USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Margolin%20AA%22%5BAuthor%5D\"> <span class=\"name western\"> Adam A Margolin </span> </a> </div> </div> <sup> 4, </sup> <sup> 5 </sup> , <a aria-describedby=\"id3\" aria-expanded=\"false\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Stuart%20JM%22%5BAuthor%5D\"> <span class=\"name western\"> Joshua M Stuart </span> </a> <div hidden=\"hidden\" id=\"id3\"> <h3> <span class=\"name western\"> Joshua M Stuart </span> </h3> <div class=\"p\"> <sup> 6 </sup> Department of Biomolecular Engineering, University of California, Santa Cruz, High Street, Santa Cruz, CA 95064 USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Stuart%20JM%22%5BAuthor%5D\"> <span class=\"name western\"> Joshua M Stuart </span> </a> </div> </div> <sup> 6 </sup> , <a aria-describedby=\"id4\" aria-expanded=\"false\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Califano%20A%22%5BAuthor%5D\"> <span class=\"name western\"> Andrea Califano </span> </a> <div hidden=\"hidden\" id=\"id4\"> <h3> <span class=\"name western\"> Andrea Califano </span> </h3> <div class=\"p\"> <sup> 7 </sup> Department of Systems Biology, Biochemistry &amp; Molecular Biophysics, Herbert Irving Comprehensive Cancer Center, Columbia University, St. Nicholas Avenue, New York, NY 10032 USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Califano%20A%22%5BAuthor%5D\"> <span class=\"name western\"> Andrea Califano </span> </a> </div> </div> <sup> 7 </sup> , <a aria-describedby=\"id5\" aria-expanded=\"false\" class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Stolovitzky%20G%22%5BAuthor%5D\"> <span class=\"name western\"> Gustavo Stolovitzky </span> </a> <div hidden=\"hidden\" id=\"id5\"> <h3> <span class=\"name western\"> Gustavo Stolovitzky </span> </h3> <div class=\"p\"> <sup> 8 </sup> IBM Computational Biology Center, TJ Watson Research Center, Kitchawan Road, Yorktown Heights, NY 10598 USA </div> <div class=\"p\"> Find articles by <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/?term=%22Stolovitzky%20G%22%5BAuthor%5D\"> <span class=\"name western\"> Gustavo Stolovitzky </span> </a> </div> </div> <sup> 8, </sup> <sup> ✉ </sup> </div> <ul class=\"d-buttons inline-list\"> <li> </li> <li> </li> <li> </li> </ul> <div class=\"d-panels font-secondary-light\"> <div class=\"d-panel p\" id=\"aip_a\"> <div class=\"p\" id=\"Aff1\"> <sup> 1 </sup> Informatics &amp; Biocomputing Program, Ontario Institute for Cancer Research, University Avenue, Toronto, ON M5G 0A3 Canada </div> <div id=\"Aff2\"> <sup> 2 </sup> Department of Medical Biophysics, University of Toronto, College Street, Toronto, ON M5G 1L7 Canada </div> <div id=\"Aff3\"> <sup> 3 </sup> Department of Pharmacology &amp; Toxicology, University of Toronto, King’s College Circle, Toronto, ON M5S 1A8 Canada </div> <div id=\"Aff4\"> <sup> 4 </sup> Sage Bionetworks, Fairview Ave North, Seattle, WA 98109 USA </div> <div id=\"Aff5\"> <sup> 5 </sup> Computational Biology Program, Oregon Health &amp; Science University, SW Sam Jackson Park Road, Portland, OR 97239-3098 USA </div> <div id=\"Aff6\"> <sup> 6 </sup> Department of Biomolecular Engineering, University of California, Santa Cruz, High Street, Santa Cruz, CA 95064 USA </div> <div id=\"Aff7\"> <sup> 7 </sup> Department of Systems Biology, Biochemistry &amp; Molecular Biophysics, Herbert Irving Comprehensive Cancer Center, Columbia University, St. Nicholas Avenue, New York, NY 10032 USA </div> <div id=\"Aff8\"> <sup> 8 </sup> IBM Computational Biology Center, TJ Watson Research Center, Kitchawan Road, Yorktown Heights, NY 10598 USA </div> <div class=\"author-notes p\"> <div class=\"fn\" id=\"_fncrsp93pmc__\"> <sup> ✉ </sup> <p class=\"display-inline\"> Corresponding author. </p> </div> </div> </div> <div class=\"d-panel p\" id=\"anp_a\"> <div class=\"notes p\"> <section class=\"history\" id=\"historyarticle-meta1\"> <p> Issue date 2014. </p> </section> </div> </div> <div class=\"d-panel p\" id=\"clp_a\"> <div> © Boutros et al.; licensee BioMed Central Ltd. 2014 </div> <p> The licensee has exclusive rights to distribute this article, in any medium, for 12 months following its publication. After this time, the article is available under the terms of the Creative Commons Attribution License ( <a class=\"usa-link usa-link--external\" href=\"http://creativecommons.org/licenses/by/4.0\" rel=\"noopener noreferrer\" target=\"_blank\"> http://creativecommons.org/licenses/by/4.0 </a> ), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly credited. The Creative Commons Public Domain Dedication waiver ( <a class=\"usa-link usa-link--external\" href=\"http://creativecommons.org/publicdomain/zero/1.0/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://creativecommons.org/publicdomain/zero/1.0/ </a> ) applies to the data made available in this article, unless otherwise stated. </p> <div class=\"p\"> <a class=\"usa-link\" href=\"/about/copyright/\"> PMC Copyright notice </a> </div> </div> </div> <div> PMCID: PMC4318527 PMID: <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/25314947/\"> 25314947 </a> </div> </div> </section> </section> <section aria-label=\"Article content\"> <section class=\"body main-article-body\"> <section class=\"abstract\" id=\"Abs1\"> <h2> <a class=\"usa-anchor\" id=\"Abs1-anchor\"> </a> Abstract </h2> <p> Rapid technological development has created an urgent need for improved evaluation of algorithms for the analysis of cancer genomics data. We outline how challenge-based assessment may help fill this gap by leveraging crowd-sourcing to distribute effort and reduce bias. </p> </section> <hr class=\"headless\"/> <p> Computational biology comprises three inter-connected activities: algorithm development, validation through benchmarking, and application. In the biomedical sciences, benchmarking occupies a central and indispensable role as it maps algorithms from the space of theoretical possibilities to the realm of practical value. Critically, this process attributes specific probabilities to an algorithm’s discovery of biologically relevant knowledge (measured by the sensitivity of the algorithm) while not overwhelming the researcher with incorrect predictions (quantified by the algorithm specificity). Benchmarking is, however, a complex task, requiring the creation of comprehensive gold standards and the design of sophisticated validation strategies that may require additional experimental data. Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table <a class=\"usa-link\" href=\"#Tab1\"> 1 </a> ). In particular, the rapidly increasing size of whole-genome molecular profile datasets from large sample repositories underscores the importance of benchmarking; it has become virtually impossible to validate algorithmic predictions that are based on such large datasets systematically. </p> <section class=\"tw xbox font-sm\" id=\"Tab1\"> <h2 class=\"obj_head\"> Table 1. </h2> <div class=\"caption p\"> <p> <strong> Non-comprehensive list of important and current challenge efforts and platforms </strong> </p> </div> <div class=\"tbl-box p\" tabindex=\"0\"> <table class=\"content\" frame=\"hsides\" rules=\"groups\"> <thead> <tr valign=\"top\"> <th colspan=\"1\" rowspan=\"1\"> <strong> Challenge </strong> </th> <th colspan=\"1\" rowspan=\"1\"> <strong> Scope </strong> </th> <th colspan=\"1\" rowspan=\"1\"> <strong> Assessment type </strong> </th> <th colspan=\"1\" rowspan=\"1\"> <strong> Organizers </strong> </th> <th colspan=\"1\" rowspan=\"1\"> <strong> Website </strong> </th> </tr> </thead> <tbody> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> Assemblathon1&amp;2 </td> <td colspan=\"1\" rowspan=\"1\"> Sequence assembly </td> <td colspan=\"1\" rowspan=\"1\"> Objective scoring </td> <td colspan=\"1\" rowspan=\"1\"> UC Davis Genome Center </td> <td colspan=\"1\" rowspan=\"1\"> <a class=\"usa-link usa-link--external\" href=\"http://assemblathon.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://assemblathon.org/ </a> </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> CAFA </td> <td colspan=\"1\" rowspan=\"1\"> Protein function prediction </td> <td colspan=\"1\" rowspan=\"1\"> Objective scoring </td> <td colspan=\"1\" rowspan=\"1\"> Community collaboration </td> <td colspan=\"1\" rowspan=\"1\"> <a class=\"usa-link usa-link--external\" href=\"http://biofunctionprediction.org/node/8\" rel=\"noopener noreferrer\" target=\"_blank\"> http://biofunctionprediction.org/node/8 </a> </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> CAGI </td> <td colspan=\"1\" rowspan=\"1\"> Systems biology </td> <td colspan=\"1\" rowspan=\"1\"> Objective scoring </td> <td colspan=\"1\" rowspan=\"1\"> UC Berkley/University of Maryland </td> <td colspan=\"1\" rowspan=\"1\"> <a class=\"usa-link usa-link--external\" href=\"http://genomeinterpretation.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://genomeinterpretation.org/ </a> </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> CAPRI </td> <td colspan=\"1\" rowspan=\"1\"> Protein docking </td> <td colspan=\"1\" rowspan=\"1\"> Objective scoring </td> <td colspan=\"1\" rowspan=\"1\"> Community collaboration </td> <td colspan=\"1\" rowspan=\"1\"> <a class=\"usa-link usa-link--external\" href=\"http://www.ebi.ac.uk/msd-srv/capri/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.ebi.ac.uk/msd-srv/capri/ </a> </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> CASP </td> <td colspan=\"1\" rowspan=\"1\"> Structure prediction </td> <td colspan=\"1\" rowspan=\"1\"> Objective scoring </td> <td colspan=\"1\" rowspan=\"1\"> Community collaboration </td> <td colspan=\"1\" rowspan=\"1\"> <a class=\"usa-link usa-link--external\" href=\"http://predictioncenter.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://predictioncenter.org/ </a> </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> ChaLearn </td> <td colspan=\"1\" rowspan=\"1\"> Machine learning </td> <td colspan=\"1\" rowspan=\"1\"> Objective scoring </td> <td colspan=\"1\" rowspan=\"1\"> ChaLearn Organization (non-for profit) </td> <td colspan=\"1\" rowspan=\"1\"> <a class=\"usa-link usa-link--external\" href=\"http://www.chalearn.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.chalearn.org/ </a> </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> CLARITY </td> <td colspan=\"1\" rowspan=\"1\"> Clinical genome interpretation </td> <td colspan=\"1\" rowspan=\"1\"> Objective scoring and evaluation by judges </td> <td colspan=\"1\" rowspan=\"1\"> Boston Children’s Hospital </td> <td colspan=\"1\" rowspan=\"1\"> <a class=\"usa-link usa-link--external\" href=\"http://www.childrenshospital.org/research-and-innovation/research-initiatives/clarity-challenge\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.childrenshospital.org/research-and-innovation/research-initiatives/clarity-challenge </a> </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> DREAM </td> <td colspan=\"1\" rowspan=\"1\"> Network inference and systems biology </td> <td colspan=\"1\" rowspan=\"1\"> Objective scoring </td> <td colspan=\"1\" rowspan=\"1\"> Community collaboration &amp; Sage Bionetworks </td> <td colspan=\"1\" rowspan=\"1\"> <a class=\"usa-link usa-link--external\" href=\"https://www.synapse.org/#!Challenges:DREAM\" rel=\"noopener noreferrer\" target=\"_blank\"> https://www.synapse.org/#!Challenges:DREAM </a> </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> FlowCAP </td> <td colspan=\"1\" rowspan=\"1\"> Flow cytometry analysis </td> <td colspan=\"1\" rowspan=\"1\"> Objective scoring </td> <td colspan=\"1\" rowspan=\"1\"> Community collaboration </td> <td colspan=\"1\" rowspan=\"1\"> <a class=\"usa-link usa-link--external\" href=\"http://flowcap.flowsite.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://flowcap.flowsite.org/ </a> </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> IGCG-TCGA DREAM Somatic Mutation Calling </td> <td colspan=\"1\" rowspan=\"1\"> Sequence analysis </td> <td colspan=\"1\" rowspan=\"1\"> Objective evaluation </td> <td colspan=\"1\" rowspan=\"1\"> Community collaboration &amp; Sage Bionetworks </td> <td colspan=\"1\" rowspan=\"1\"> <a class=\"usa-link usa-link--external\" href=\"https://www.synapse.org/#!Synapse:syn312572\" rel=\"noopener noreferrer\" target=\"_blank\"> https://www.synapse.org/#!Synapse:syn312572 </a> </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> IMPROVER </td> <td colspan=\"1\" rowspan=\"1\"> Systems biology </td> <td colspan=\"1\" rowspan=\"1\"> Objective evaluation and crowd-verification </td> <td colspan=\"1\" rowspan=\"1\"> Phillip Morris International </td> <td colspan=\"1\" rowspan=\"1\"> <a class=\"usa-link usa-link--external\" href=\"https://sbvimprover.com/\" rel=\"noopener noreferrer\" target=\"_blank\"> https://sbvimprover.com/ </a> </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> Innocentive </td> <td colspan=\"1\" rowspan=\"1\"> Topics in various industries </td> <td colspan=\"1\" rowspan=\"1\"> Objective scoring and evaluation by judges </td> <td colspan=\"1\" rowspan=\"1\"> Commercial platform </td> <td colspan=\"1\" rowspan=\"1\"> <a class=\"usa-link usa-link--external\" href=\"http://www.innocentive.com/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.innocentive.com/ </a> </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> Kaggle </td> <td colspan=\"1\" rowspan=\"1\"> Topics in various industries </td> <td colspan=\"1\" rowspan=\"1\"> Objective scoring and evaluation by judges </td> <td colspan=\"1\" rowspan=\"1\"> Commercial platform </td> <td colspan=\"1\" rowspan=\"1\"> <a class=\"usa-link usa-link--external\" href=\"http://www.kaggle.com/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.kaggle.com/ </a> </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> RGASP </td> <td colspan=\"1\" rowspan=\"1\"> RNA-seq analyses </td> <td colspan=\"1\" rowspan=\"1\"> Objective scoring </td> <td colspan=\"1\" rowspan=\"1\"> European Bioinformatics Institute </td> <td colspan=\"1\" rowspan=\"1\"> <a class=\"usa-link usa-link--external\" href=\"http://www.gencodegenes.org/rgasp/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.gencodegenes.org/rgasp/ </a> </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> Sequence Squeeze </td> <td colspan=\"1\" rowspan=\"1\"> Sequence compression </td> <td colspan=\"1\" rowspan=\"1\"> Objective scoring and evaluation by judges </td> <td colspan=\"1\" rowspan=\"1\"> Pistoia Alliance </td> <td colspan=\"1\" rowspan=\"1\"> <a class=\"usa-link usa-link--external\" href=\"http://sequencesqueeze.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://sequencesqueeze.org/ </a> </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> X-Prize </td> <td colspan=\"1\" rowspan=\"1\"> Technology </td> <td colspan=\"1\" rowspan=\"1\"> Evaluation by judges </td> <td colspan=\"1\" rowspan=\"1\"> X-Prize Organization (non-for-profit) </td> <td colspan=\"1\" rowspan=\"1\"> <a class=\"usa-link usa-link--external\" href=\"http://www.xprize.org/\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.xprize.org/ </a> </td> </tr> </tbody> </table> </div> <div class=\"p text-right font-secondary\"> <a class=\"usa-link\" href=\"table/Tab1/\" rel=\"noopener noreferrer\" target=\"_blank\"> Open in a new tab </a> </div> <div class=\"tw-foot p\"> <div class=\"fn\" id=\"_fn_p4\"> <p> The challenges were chosen based on relevance to cancer genomics or the representativeness of a type of challenge. Different challenges specialize in specific areas of research (see ‘Scope’), and may use different assessment types such as objective scoring against a gold standard, evaluation by judges, or community consensus (‘crowd-verification’). Organizers can be researchers from specific institutions (such as universities or hospitals), a group of diverse researchers from academia and industry collaborating in the challenge organization (community collaboration), not-for-profit associations, or commercial platforms that run challenges as their business model (such as Innocentive and Kaggle). Initiatives such as CAFA, CAGI, CAPRI, CASP, ChaLearn, DREAM, FlowCAP and IMPROVER organize several challenges each year, and only the generic project is listed in this table, with the exception of DREAM, for which we also show the IGCG-TCGA DREAM Somatic Mutation Calling Challenge because of its relevance to this paper. More information about these efforts can be found on the listed websites. </p> </div> </div> </section> <p> Benchmarking is not a matter of simply running a few algorithms on a few datasets and comparing the results. Drawing generalizable conclusions from the exercise requires significant care in design and execution. The maturity of bioinformatics as a discipline has been greatly advanced by the adoption of key principles that guide robust method evaluation, including evaluator objectiveness (lack of bias), clearly defined scoring metrics that align with real-world goals, and the public release of gold-standard datasets and of the results and code of prediction algorithms. Challenge-based (also known as ‘competition-based’) method assessment is an increasingly popular mechanism for benchmarking [ <a aria-describedby=\"CR1\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR1\"> 1 </a> , <a aria-describedby=\"CR2\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR2\"> 2 </a> ]. In this type of study an impartial group of scientists organizes a ‘challenge’ that is based on a carefully curated dataset. This dataset is typically split into a training dataset, a validation dataset (which might be used in real-time leaderboards, typically implemented as a table that reports the comparative performance of the methods under development), and a gold standard (or test) dataset that is withheld from challenge participants and used for final evaluation (Figure <a class=\"usa-link\" href=\"#Fig1\"> 1 </a> ). Following algorithm development on the training dataset and real-time feedback to participants based on the validation dataset and reported in the leaderboard, the challenge organizers can objectively evaluate the quality of final submitted predictions using a gold-standard dataset. Such a design closely reflects the actual difficulties faced by real-world users trying to determine whether an algorithm generalizes to unseen cases. </p> <figure class=\"fig xbox font-sm\" id=\"Fig1\"> <h2 class=\"obj_head\"> Figure 1. </h2> <p class=\"img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center\"> </p> <div class=\"p text-right font-secondary\"> <a class=\"usa-link\" href=\"figure/Fig1/\" rel=\"noopener noreferrer\" target=\"_blank\"> Open in a new tab </a> </div> <figcaption> <p> <strong> Typical design of a crowd-sourced challenge. </strong> A dataset is split into a training set, a validation (or leaderboard set) and the test set (or gold standard). Participants have access to the challenge input data and the known answers for just the training set. For the validation and test sets only, the challenge input data are provided but the answers to the challenge questions are withheld. In the challenge open phase, participants optimize their algorithms by making repeated submissions to predict the validation set answers. These submissions are scored and returned to the participants who can use the information to improve their methods. In the final evaluation phase, the optimized algorithms are submitted and evaluated against the final test set (the gold standard), and the resulting scores are used to compute the statistical significance and the ranking of the participating algorithms. </p> </figcaption> </figure> <p> When flawed, benchmarking can lead to the emergence of suboptimal standards that may be applied to many large datasets, imposing an immense cost to the community and creating misleading results. Conversely, the acceptance of knowledge without robust benchmarking can lead to the adoption of inaccurate conventions. For example, during the 1990s, it was generally accepted that the number of loci coding for proteins in the human genome was 100,000, a number that was based on unverified hypotheses [ <a aria-describedby=\"CR3\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR3\"> 3 </a> ]. When the human genome was finally sequenced in 2000, the total number of coding loci was found to be a factor of 5 lower. Similarly, a design error in the early implementation of the GC Robust Multi-Array (GCRMA) algorithm, which was revealed by systematic benchmarking of network reconstruction analyses, may have led to the publication of thousands of papers that contain incorrect mRNA abundance profiles before the error was detected and corrected [ <a aria-describedby=\"CR4\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR4\"> 4 </a> ]. As a third example, in 2006, a group of Duke University researchers published a pair of high-impact papers claiming accurate prediction of the prognosis of lung cancer patients and of chemotherapy-sensitivity in lung, breast and ovarian cancers. Attempts to reproduce those claims ensued almost immediately, with most of the results falling short of replication because of a combination of programming and data-entry errors, and possible fraud [ <a aria-describedby=\"CR5\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR5\"> 5 </a> ]. Proper objective benchmarking by a neutral third-party on private validation data helps to resolve quickly or to detect many of the issues associated with these kinds of studies. </p> <p> One concern in algorithm benchmarking and validation is that computational biology algorithms are often developed and evaluated by the same researchers. This creates an inherent conflict of interest, where objective assessment of accuracy is polluted by the fact that the developers become simultaneously judge, jury and executioner of the validity of their own work. This can result in biases in study design and over-optimistic performance estimates, whether intentional or unintentional [ <a aria-describedby=\"CR6\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR6\"> 6 </a> ]. For instance, the use of non-blinded data in the evaluation by methods developers of their own protein structure prediction methods led, in the early’ 80s, to the false belief that protein structure prediction was essentially a solved problem. Not until 1994, when double-blinded data were used in the first Workshop on the Critical Assessment of Protein Structure Prediction (CASP), was a very different picture revealed [ <a aria-describedby=\"CR7\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR7\"> 7 </a> ]. </p> <p> Challenge-based benchmarking efforts, such as CASP [ <a aria-describedby=\"CR8\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR8\"> 8 </a> – <a aria-describedby=\"CR10\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR10\"> 10 </a> ], CAFA [ <a aria-describedby=\"CR11\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR11\"> 11 </a> ] and DREAM [ <a aria-describedby=\"CR12\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR12\"> 12 </a> , <a aria-describedby=\"CR13\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR13\"> 13 </a> ], among others (Table <a class=\"usa-link\" href=\"#Tab1\"> 1 </a> ), offer a robust framework for algorithm evaluation. These efforts have proven the value of engaging both active challenge leaders and motivated algorithm developers to improve their work in a forum that has high visibility and rapid feedback. </p> <p> We believe that challenge-based methods assessment will play an increasingly important role in standardizing and optimizing the analysis of cancer genomics data, and its broader adoption will drive progress in both algorithm development and biological discovery. Conversely, failing to exploit challenge-benchmarking as a fundamental validation methodology for cancer genomics algorithms may result in lost opportunities to translate results derived from best-in-class methods into patient care. </p> <p> Here, we provide our perspective on the growing use of challenge-based methods to benchmark algorithms in cancer genomics. We outline the different types of problems faced and some of the key considerations that need to be explored to determine whether a challenge might be successful, and to provide suggestions for challenge organization and execution. Finally, we look to the future and consider how challenge-based assessment may change in the coming decade. </p> <section id=\"Sec1\"> <h2 class=\"pmc_sec_title\"> <a class=\"usa-anchor\" id=\"Sec1-anchor\"> </a> Challenge design and dynamics </h2> <p> Over the past few years, an established challenge-based design paradigm has emerged in which portions of a private (that is, not globally released) dataset are made publicly available according to a predefined schedule. Such a dataset provides increased user engagement based on continuous feedback; an opportunity for participants to refine and improve their methods on the basis of results obtained throughout the challenge; and multiple independent rounds of validation, which can be used to assess the consistency and robustness of results. After the initial training dataset is made publicly available, a real-time leaderboard can be generated in which the performance of different algorithms is evaluated against a withheld private portion of the data (Figure <a class=\"usa-link\" href=\"#Fig1\"> 1 </a> ). Previous research has shown that the provision of real-time feedback is among the most important factors in ensuring user engagement in crowd-sourcing projects [ <a aria-describedby=\"CR14\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR14\"> 14 </a> ]. (Here, we use the term crowd-sourcing in the sense that a community of tens to hundreds of researchers are engaged in working on the same problem. In other contexts, crowd-sourcing activities may engage different numbers of participants.) After a period of time in which several iterations of the leaderboard can be posted, one of the participating groups is declared the best performer in this initial phase of the challenge, either on the basis of their position on the leaderboard or because they were the first to reach some pre-specified performance level. The challenge may include multiple rounds of assessment based on different portions of the private data. A final round is typically invoked in which methods are rated against a withheld evaluation dataset to determine the overall challenge winner (Figure <a class=\"usa-link\" href=\"#Fig1\"> 1 </a> ). The most robust validation set is often reserved for this final evaluation - often with larger sample size, newly generated data or prospective validation designed on the basis of challenge results. Each participating team submits a small number (for example, one to five) of independent predictions made by their algorithm(s), which are scored and ranked to determine a winner. Finally, the public release of all of the data kept private throughout the challenge, along with the predictions and ideally source code from each group, provides a long-term resource to spur further development of new and improved methods. </p> <p> The collection of algorithm source code allows developers to share insights that promote future improvements. If required as part of the final submission, this code can also be used to ensure objective scoring and verification of reproducibility. In the 2012 Sage Bionetworks-DREAM Breast Cancer Prognosis Challenge, participants were required to submit their models as open source R-code [ <a aria-describedby=\"CR15\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR15\"> 15 </a> ] that was visible to all participants and executed by an automated system to generate the results reported on the leaderboard. This was enabled by Synapse [ <a aria-describedby=\"CR16\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR16\"> 16 </a> ], a software platform that supports scientific challenges as well as large distributed collaborations, such as those in the TCGA Pan-Cancer consortium [ <a aria-describedby=\"CR17\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR17\"> 17 </a> ]. Planned challenges, such as the RNA-seq follow-up to the ICGC-TCGA DREAM Somatic Mutation Calling (SMC) Challenge, are considering the use of cloud-computing solutions to provide a central computing facility and a harness upon which contestant code is directly run. This will inherently force the deposition of complete analysis workflows, which can be run routinely on new datasets. Further, this approach would help to standardize application programming interfaces and file formats, such that multiple algorithms use similar inputs and produce easily comparable outputs. This vision of interoperability is shared by many practitioners in the field and has most recently been championed by the Global Alliance for Genomics And Health [ <a aria-describedby=\"CR18\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR18\"> 18 </a> ]. </p> <p> Several criteria should be used to help participants limit over-fitting to the training data. Over-fitting is a known peril in statistics, occurring when a predictive model has enough flexibility in its parameters that optimization effectively leads to ‘memorization’ of the training data and an inability to generalize to unseen cases. The most common way to help participants avoid over-fitting, while enabling the testing of their models, is to provide leaderboard scoring that is based on a subset of the private dataset, optimally a subset that is not used in the final evaluation. The latter condition is sometimes not feasible (for example, when the number of patients available to predict clinical outcomes is limited), in which case the leaderboard will be based on data that are also used for the final scoring. If this is the case, limiting the number of submissions can reduce over-fitting. </p> <p> While most challenges share some common design principles, each research area has its own unique characteristics that require customized experimental designs and consideration of risks and benefits. Indeed, the utility of organizing a challenge to help advance a particular research area depends on a balance between challenge-based benchmarking advantages and limitations, as well as consideration of the potential barriers for participation (Table <a class=\"usa-link\" href=\"#Tab2\"> 2 </a> ). In the sections below, we highlight three research areas in which rapid development of new algorithms has led to a concomitant need for benchmarking: accurate identification of tumor-specific genomic alterations, association of clinical data with genomic profiles (that is, biomarkers) and identifying network-biology features that underlie cancer phenotypes. </p> <section class=\"tw xbox font-sm\" id=\"Tab2\"> <h3 class=\"obj_head\"> Table 2. </h3> <div class=\"caption p\"> <p> <strong> Some advantages and limitations of challenge-based methods assessment, along with barriers to participation in them </strong> </p> </div> <div class=\"tbl-box p\" tabindex=\"0\"> <table class=\"content\" frame=\"hsides\" rules=\"groups\"> <thead> <tr valign=\"top\"> <th colspan=\"1\" rowspan=\"1\"> <strong> Advantages </strong> </th> <th colspan=\"1\" rowspan=\"1\"> <strong> Limitations </strong> </th> <th colspan=\"1\" rowspan=\"1\"> <strong> Participation barriers </strong> </th> </tr> </thead> <tbody> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> Reduction of over-fitting </td> <td colspan=\"1\" rowspan=\"1\"> Narrower scope compared to traditional open-ended research </td> <td colspan=\"1\" rowspan=\"1\"> Incentives not strong enough to promote participation </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> Benchmarking individual methods </td> <td colspan=\"1\" rowspan=\"1\"> Ground truth needed for objective scoring </td> <td colspan=\"1\" rowspan=\"1\"> No funding available to support time spent participating in challenges </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> Impartial comparison across methods using same datasets </td> <td colspan=\"1\" rowspan=\"1\"> Mostly limited to computational approaches </td> <td colspan=\"1\" rowspan=\"1\"> Fatigue resulting from many ongoing challenges </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> Fostering collaborative work, including code sharing </td> <td colspan=\"1\" rowspan=\"1\"> Requires data producers to share their data before publication </td> <td colspan=\"1\" rowspan=\"1\"> Time assigned by organizers to solve a difficult challenge question may be too short </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> Acceleration of research </td> <td colspan=\"1\" rowspan=\"1\"> Sufficient amount of high-quality data needed for meaningful results </td> <td colspan=\"1\" rowspan=\"1\"> Lack of computing capabilities </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> Enhancing data access and impact </td> <td colspan=\"1\" rowspan=\"1\"> Large number of participants not always available </td> <td colspan=\"1\" rowspan=\"1\"> New data modality or datasets that are too complex or too big poses entry barrier </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> Determination of problem solvability </td> <td colspan=\"1\" rowspan=\"1\"> Challenge questions may not be solvable with data at hand </td> <td colspan=\"1\" rowspan=\"1\"> Challenge questions not interesting or impactful enough </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> Tapping the ‘Wisdom of Crowds’ </td> <td colspan=\"1\" rowspan=\"1\"> Traditional grant mechanisms not adequate to fund challenge efforts </td> <td colspan=\"1\" rowspan=\"1\"> Cumbersome approvals to acquire sensitive datasets </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> Objective assessment </td> <td colspan=\"1\" rowspan=\"1\"> Difficulties to distribute datasets with sensitive information </td> <td colspan=\"1\" rowspan=\"1\"> </td> </tr> <tr valign=\"top\"> <td colspan=\"1\" rowspan=\"1\"> Standardizes experimental design </td> <td colspan=\"1\" rowspan=\"1\"> </td> <td colspan=\"1\" rowspan=\"1\"> </td> </tr> </tbody> </table> </div> <div class=\"p text-right font-secondary\"> <a class=\"usa-link\" href=\"table/Tab2/\" rel=\"noopener noreferrer\" target=\"_blank\"> Open in a new tab </a> </div> </section> </section> <section id=\"Sec2\"> <h2 class=\"pmc_sec_title\"> <a class=\"usa-anchor\" id=\"Sec2-anchor\"> </a> Analyzing genome assembly and structural variants </h2> <p> Technologies for identifying cancer-related somatic alterations from genomic or transcriptomic data are advancing extremely rapidly. In only 6 years, next-generation sequencing (NGS) has rapidly progressed from the measurement of millions of short sequences (of around 25 bp) to that of hundreds of millions of longer segments (of around 100 bp). This creates an urgent need for on-going benchmarking studies as old algorithms become rapidly out-dated and new algorithmic approaches are required to handle new technologies and new scales of data. Small-scale studies have resulted in dramatic discordance when different researchers apply their algorithms to the same genomic data (Figure <a class=\"usa-link\" href=\"#Fig2\"> 2 </a> ) [ <a aria-describedby=\"CR19\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR19\"> 19 </a> – <a aria-describedby=\"CR21\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR21\"> 21 </a> ]. These studies have shown that accuracy and generalizability vary dramatically across samples and regions of the genome. The constantly shifting landscape presented by rapidly evolving technologies and tools fuels the urgency in the need to identify the best-performing methods objectively and to re-evaluate them frequently, and to identify particularly error-prone aspects of existing tumor genome analysis methods [ <a aria-describedby=\"CR22\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR22\"> 22 </a> ]. Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of <em> de novo </em> sequence assembly algorithms [ <a aria-describedby=\"CR23\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR23\"> 23 </a> ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ <a aria-describedby=\"CR24\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR24\"> 24 </a> ] (Table <a class=\"usa-link\" href=\"#Tab1\"> 1 </a> ). </p> <figure class=\"fig xbox font-sm\" id=\"Fig2\"> <h3 class=\"obj_head\"> Figure 2. </h3> <p class=\"img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center\"> </p> <div class=\"p text-right font-secondary\"> <a class=\"usa-link\" href=\"figure/Fig2/\" rel=\"noopener noreferrer\" target=\"_blank\"> Open in a new tab </a> </div> <figcaption> <p> <strong> Different researchers studying the same data may arrive at discordant conclusions. </strong> Benchmarking becomes essential as a way to separate true findings from spurious ones. (Illustration by Natasha Stolovitzky-Brunner© inspired by the parable of the six blind men and the elephant). </p> </figcaption> </figure> <p> Challenge-based benchmarking of methods for somatic variant detection in cancer faces several unique hurdles. First, genomic sequence is inherently identifiable [ <a aria-describedby=\"CR25\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR25\"> 25 </a> ], and is thus considered personal health information (PHI) in many countries. This places a burden on challenge contestants to acquire ethics approval from the appropriate authorities, such as dbGaP in the USA or ICGC in Canada. Second, because of the inherent complexity of both the data and file formats, it may be difficult for researchers from other fields to acquire sufficient domain knowledge to compete effectively against domain experts. This point may be ameliorated by gamifying the problem, that is, using game tools that require puzzle solving or geometric thinking to engage users in genomics problems [ <a aria-describedby=\"CR26\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR26\"> 26 </a> , <a aria-describedby=\"CR27\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR27\"> 27 </a> ]. Gamification may not be possible or appropriate, however, because it may require sacrificing domain-specific prior knowledge that is essential to the correct solution. Third, the size of the raw genomic data necessary to perform these challenges creates a ‘big-data’ problem. For example, the ICGC-TCGA DREAM SMC Challenge [ <a aria-describedby=\"CR28\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR28\"> 28 </a> ] (Table <a class=\"usa-link\" href=\"#Tab1\"> 1 </a> ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs. Two different solutions to this problem are to provide access to high-speed, cloud-based download technologies (such as GeneTorrent or Aspera) or to provide co-location of computers and data in a hosted environment [ <a aria-describedby=\"CR29\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR29\"> 29 </a> ]. The latter solution has the advantage of providing implementations of the best-performing algorithms in a form that is more readily redistributed to the community, as well as allowing more ‘democratized’ participation for groups that do not have large in-house computing resources. Nevertheless, this solution also has disadvantages: cloud-computing may require additional overhead expenditure for groups that are familiar with developing methods within their local computing environments; many researchers have access to in-house computing options subsidized by their institution and have limited incentive to transfer their analysis to the cloud; and access permissions for some datasets can hinder redistribution through cloud platforms. Furthermore, the assessment of predictions is challenging because the ground-truth for genetic alterations is unknown. The SMC Challenge employs two strategies for evaluation. The first involves an <em> in silico </em> method for simulating cancer genomes called BAMSurgeon, which was developed to allow the comparison of methods predictions against a synthetic ground-truth (work by Ewing and colleagues). In the second strategy, targeted deep-sequencing allows prospective validation of a large number of predicted mutations, chosen by an algorithm that most accurately computes false-positive and false-negative rates across submissions. It is unclear how important it is for prospective validation data to be orthogonal to that used by the original challenge participants. Verification in TCGA projects typically relies on deep sequencing using the same technology, but on selected targets and with the construction of new sequencing libraries. This approach assumes that most errors are randomly distributed and/or associated with only a small fraction of reads. The more orthogonal the validation technology, the more this assumption is relaxed. Nevertheless, the error profile of the final evaluation dataset is crucial, and there are currently no error-free approaches to generating this gold-standard data for NGS. </p> </section> <section id=\"Sec3\"> <h2 class=\"pmc_sec_title\"> <a class=\"usa-anchor\" id=\"Sec3-anchor\"> </a> Finding genomic biomarkers that are associated with phenotype </h2> <p> Once a set of somatic variants have been identified from genomic interrogation of patient-derived samples, one of the most common analyses is to attempt to develop biomarkers that can predict patient survival, response to therapy or other outcomes [ <a aria-describedby=\"CR30\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR30\"> 30 </a> – <a aria-describedby=\"CR33\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR33\"> 33 </a> ]. The development of genomic-based personalized medicine has immense clinical potential, but the optimal approach to predicting such biomarkers <em> de novo </em> remains poorly understood and controversial. Indeed, it is widely known that inferred biomarkers are highly sensitive to factors such as choice of algorithm and data pre-processing methods [ <a aria-describedby=\"CR34\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR34\"> 34 </a> – <a aria-describedby=\"CR37\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR37\"> 37 </a> ]. </p> <p> Nevertheless, developing challenges to benchmark biomarker discovery problems is relatively straightforward. Participants are given training data in which features (for example, genome-wide mRNA transcript abundance) are paired with outcome (for example, patient survival) data. Participants are given only the features for the test set and asked to predict the outcome data using a model inferred from the training data. Alternatively, participants may submit trained models as executable code to be run on the test data, thus allowing the test feature data to be hidden from participants [ <a aria-describedby=\"CR15\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR15\"> 15 </a> ]. Model results are scored on the basis of the correspondence between predicted and measured outcome data from the test set. </p> <p> Prediction challenges have been employed in many domains outside of biomedical research [ <a aria-describedby=\"CR38\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR38\"> 38 </a> ]. Because biomarker-based challenges fit the setup of the classic supervised machine-learning paradigm, they attract new ideas and participation from the broader machine-learning community. Benchmarking in biomarker discovery is crucial, however, as outlined by the case of the retracted Duke study on chemotherapy selection noted above. </p> <p> Two key difficulties exist in the creation of benchmarking challenges for biomarker discovery. First, the ideal datasets for biomarker-discovery challenges are uniquely defined, especially when data were collected from large cohorts requiring long-term follow-up or expensive standardized treatment protocols (such as clinical trials). These datasets can potentially lead to high-impact publications or concerns over the intellectual property of the data-generating groups. Second, the potential size of patient cohorts is currently limiting for many biomarker-development questions. If the amount of data available is inadequate, they may not generate enough statistical power to distinguish the performance of the top-ranked groups accurately. These factors also complicate the ability to obtain independent datasets for final method assessment. Despite these problems, several successful challenges pertaining to diagnostics, prognostics and treatment outcomes have been conducted, including the MAQC-II study [ <a aria-describedby=\"CR39\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR39\"> 39 </a> ], the IMPROVER Challenge on Diagnostic Signatures [ <a aria-describedby=\"CR40\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR40\"> 40 </a> ], the Sage Bionetworks DREAM Breast Cancer Prognostics Challenge [ <a aria-describedby=\"CR15\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR15\"> 15 </a> ], and the DREAM AML Treatment Outcome Challenge [ <a aria-describedby=\"CR41\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR41\"> 41 </a> ]. </p> </section> <section id=\"Sec4\"> <h2 class=\"pmc_sec_title\"> <a class=\"usa-anchor\" id=\"Sec4-anchor\"> </a> Inferring biological networks underlying cancer phenotypes </h2> <p> Identifying the relationships between biological (transcriptional and signaling) networks and cancer onset and progression is another potential area for challenge benchmarking. Network analysis involves several aspects, including the coherent modeling of different types of alteration and dysregulation events and their integration into a unified network-based model [ <a aria-describedby=\"CR42\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR42\"> 42 </a> – <a aria-describedby=\"CR44\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR44\"> 44 </a> ]. One of the major problems with organizing challenges in this area is that the underlying cellular regulatory networks are mostly unknown, especially in complex systems such mammalian tumor cells. So how can a challenge be organized when a pre-known gold-standard network cannot be defined? Several strategies employed by the DREAM project include using synthetic biology networks [ <a aria-describedby=\"CR13\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR13\"> 13 </a> ], <em> in silico </em> networks [ <a aria-describedby=\"CR45\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR45\"> 45 </a> ], and experimentally assessed bacterial networks [ <a aria-describedby=\"CR46\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR46\"> 46 </a> ]. An alternative strategy is to evaluate methods on the basis of their ability to predict the response of a system to a set of perturbations, such as drugs or receptor ligands, as surrogates for predicting the underlying network connectivity [ <a aria-describedby=\"CR47\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR47\"> 47 </a> ]. The introduction of ingenious surrogates to the gold standard has enabled the formulation of other network reverse-engineering challenges, such as the 2013 HPN-DREAM Breast Cancer Network Inference Challenge [ <a aria-describedby=\"CR48\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR48\"> 48 </a> ]. In this challenge, participants were asked to submit predicted signaling networks that were activated by a set of stimuli in four breast cancer cell lines. These networks were scored on the basis of their ability to identify the set of proteins that are downstream of a given phosphoprotein. The predicted protein set was compared to an experimentally determined set of proteins (the surrogate gold standard), defined as those proteins whose phosphorylation levels were affected by inhibiting that phosphoprotein. Further research on benchmarking network-inference algorithms would be highly beneficial to help advance the field of network biology, whose role in unraveling biological mechanisms in cancer is hard to overestimate. </p> </section> <section id=\"Sec5\"> <h2 class=\"pmc_sec_title\"> <a class=\"usa-anchor\" id=\"Sec5-anchor\"> </a> The truth is hard to find </h2> <p> From the previous discussion, it is clear that the single most crucial aspect in benchmarking is the definition and assembly of gold standards. A gold standard fundamentally defines the problem under study, and it provides the limiting resolution of error for the overall endeavor. As outlined in this article, gold standards can be defined in several ways. First, a single experiment can be performed with portions of the resulting data used for training and evaluation. This approach avoids experimental inconsistencies, but requires that a large selection of true results is generated prior to the challenge. Simulated datasets are ideal for this strategy but have been criticized as only partially representing a biological system [ <a aria-describedby=\"CR49\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR49\"> 49 </a> ]. While validation of simulated data is straight forward, because the ground-truth is completely known, in most cases the value of benchmarking is perceived to be in the ability to assess best-performing methods when applied to real biological data as opposed to simulated data. An important caveat is that the synthetic data may fail to reflect some of the underlying assumptions of the system they attempt to emulate. Indeed, the most common question about simulations is how well they reflect experimental samples [ <a aria-describedby=\"CR49\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR49\"> 49 </a> ]. </p> <p> Second, for systems that are difficult to benchmark directly, such as the structure of a biological network, characteristics of the systems can be evaluated instead. These might include the effects of the systems’ perturbation or other phenomena, such as the identification of the networks that best predict patient outcomes. </p> <p> Third, the results of a study can be validated after the challenge is completed by additional experimental work, either on the same sample or on others. This has the advantage of directly addressing the predictions made by challenge participants, but has the disadvantage of introducing a time lag between challenge completion and the availability of full results. In addition, the effort and cost of follow-up validation may be prohibitive given the resources available to the challenge organizers. </p> <p> For genomic studies, wet-lab validation can be both time-consuming and expensive. For example, the MAQC study considered approximately 20,000 genes on microarray platforms, but only validated approximately 1,000 (5%) by real-time PCR as a gold standard [ <a aria-describedby=\"CR50\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR50\"> 50 </a> ]. Because of this cost, both in terms of time and money, it is critical that a good validation be sufficiently representative, providing similar levels of statistical power for assessing the accuracy of each group. In the context of somatic mutation calling, this means selecting calls that are unique to individual predictors as well as those common to multiple predictors. Indeed, the validation techniques will often be experimentally limited to a subset of results, leaving a bias in the distribution of what is tested. There is thus a clear need for research into the optimal selection of validation candidates in many biological settings. Further, validating a small subset (&lt;10%) of results comes with the possibility, however small, of producing an incorrect relative ordering of different algorithms. In practice, a combination of synthetic and real-world validation is best, and finding the right balance is challenge-dependent. </p> <p> Finally, some very important elements of cancer genomics are difficult to validate. For example, almost all NGS analyses rely on sequence alignment as a first step. It is, however, very difficult to benchmark the accuracy of an alignment algorithm on real tumor data, because there is no obvious way to create a ground-truth dataset. Thus, rather than benchmarking the aligners, challenges benchmark the results of entire pipelines such as those for detecting somatic variants [ <a aria-describedby=\"CR28\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR28\"> 28 </a> ], which may incorporate different aligners and different data pre-processing and statistical approaches. Similarly, it is of great interest to infer cancer-driver genes. Unfortunately, the definition of a ‘driver gene’ (beyond simple statistical recurrence) is unclear, and does not yet allow unambiguous, high-throughput experimental validation. Most experimental techniques in this area probe only one aspect of a driver gene (such as its influence on proliferation or metastasis), while many subtle phenotypes (such as angiogenesis or local spread) are challenging to probe. Also, these designs ignore the potentially polygenic nature of tumor initiation and progression. In designing a new challenge, one of the first questions must be whether or not suitable gold-standard test datasets can be generated. </p> </section> <section id=\"Sec6\"> <h2 class=\"pmc_sec_title\"> <a class=\"usa-anchor\" id=\"Sec6-anchor\"> </a> Closing considerations </h2> <p> Benchmarking is a fundamental part of computational biology and is increasingly being appreciated by the biomedical community as a whole. Recent benchmarking studies both within [ <a aria-describedby=\"CR19\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR19\"> 19 </a> , <a aria-describedby=\"CR51\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR51\"> 51 </a> ] and outside of cancer genomics [ <a aria-describedby=\"CR39\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR39\"> 39 </a> , <a aria-describedby=\"CR52\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR52\"> 52 </a> – <a aria-describedby=\"CR54\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR54\"> 54 </a> ] have helped highlight new ways to analyze data and have prompted reconsideration of the error profiles of datasets. Challenge-based assessments have also recently surged in other fields [ <a aria-describedby=\"CR55\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR55\"> 55 </a> ] in which the use of incentives (including prizes and prestige) have stimulated increased attention and algorithm development [ <a aria-describedby=\"CR56\" aria-expanded=\"false\" class=\"usa-link\" href=\"#CR56\"> 56 </a> ]. </p> <p> As the profile of the results of benchmarking studies increases, it is becoming increasingly clear that benchmarking itself is a serious scientific endeavor. The design of a challenge is non-trivial and in some ways is easy ‘to get wrong’ - there needs to be a careful integration between experts in challenge-based benchmarking and domain experts in the challenge topic. At the outset, there is a fundamental requirement for the benchmarking team to foster a community that supports and promotes the exercise. Indeed, some topic areas may be unsuitable to challenge-based benchmarking because a sufficiently big community of interested algorithm developers has not yet emerged (although in these cases, appropriate incentives may be useful in helping to focus attention on a potential challenge topic). Further, the challenge organizing team must be able to assure the broader community of its neutrality and objectivity. There is a clear advantage to building groups of ‘challenge-based benchmarking experts’ who can bring their expertise to diverse topics within cancer genomics, or any other field. Such groups may be well-placed to develop and optimize the statistical methods needed to improve challenge-based benchmarks. Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table <a class=\"usa-link\" href=\"#Tab1\"> 1 </a> ). </p> <p> Cancer genomics is characterized by rapid technological development, and this trend is likely to persist for many years. As a result, benchmarking cannot be a static endeavor. Rather, each new technology will have its own specific error profiles and distinct algorithms that are used for data analysis. In a world of continual technological and algorithmic innovation, it may be impossible to have definitive, permanent benchmarks, because any effort will be based on a snapshot of technology and will rapidly become out-dated. Instead, a long-running series of ‘living benchmarks’ may allow the co-evolution of benchmarks with technology. In this mutualistic scenario, regular releases of new datasets capturing the current state of experimental methodologies will allow users at any point in time to identify the best tool for their dataset, and algorithm developers to have a dataset suitable for developing and optimizing methods on the latest data. </p> </section> <section class=\"ack\" id=\"ack1\"> <h2 class=\"pmc_sec_title\"> <a class=\"usa-anchor\" id=\"ack1-anchor\"> </a> Acknowledgements </h2> <p> We are indebted to the DREAM community for teaching us how to run challenges, and to Sage Bionetworks for their contributions in organizing DREAM challenges. This study was conducted with the support of the Ontario Institute for Cancer Research through funding provided by the Government of Ontario (to PCB). This work was supported by Prostate Cancer Canada and is proudly funded by the Movember Foundation (Grant #RS2014-01). Dr Boutros was supported by a Terry Fox Research Institute New Investigator Award and a CIHR New Investigator Award. This project was supported by Genome Canada through a Large-Scale Applied Project contract. This work was also supported by the Discovery Frontiers: Advancing Big Data Science in Genomics Research program, which is jointly funded by the Natural Sciences and Engineering Research Council (NSERC) of Canada, the Canadian Institutes of Health Research (CIHR), Genome Canada, and the Canada Foundation for Innovation (CFI). NIH grants R01 CA180778 (JMS) and U24-CA143858 (JMS) supported this work. </p> </section> <section class=\"glossary\" id=\"glossary1\"> <h2 class=\"pmc_sec_title\"> <a class=\"usa-anchor\" id=\"glossary1-anchor\"> </a> Abbreviations </h2> <dl class=\"def-list\"> </dl> <dl class=\"def-list\"> <dt> CASP </dt> <dd> <p> Critical Assessment of Protein Structure Prediction </p> </dd> <dt> GCRMA </dt> <dd> <p> GC Robust Multi-Array </p> </dd> <dt> PHI </dt> <dd> <p> Personal health information </p> </dd> <dt> NGS </dt> <dd> <p> Next-generation sequencing </p> </dd> <dt> SMC </dt> <dd> <p> Somatic Mutation Calling </p> </dd> </dl> </section> <section class=\"fn-group\" id=\"fn-group1\"> <h2 class=\"pmc_sec_title\"> <a class=\"usa-anchor\" id=\"fn-group1-anchor\"> </a> Footnotes </h2> <div class=\"fn-group p font-secondary-light font-sm\"> <div class=\"fn p\" id=\"fn1\"> <p> <strong> Competing interests </strong> </p> <p> The authors declare that they have no competing interests. </p> </div> </div> </section> <section class=\"contrib-info\" id=\"_ci93_\" lang=\"en\"> <h2 class=\"pmc_sec_title\"> <a class=\"usa-anchor\" id=\"_ci93_-anchor\"> </a> Contributor Information </h2> <p> Paul C Boutros, Email: Paul.Boutros@oicr.on.ca. </p> <p> Adam A Margolin, Email: margolin@ohsu.edu. </p> <p> Joshua M Stuart, Email: jstuart@ucsc.edu. </p> <p> Andrea Califano, Email: ac2248@c2b2.columbia.edu. </p> <p> Gustavo Stolovitzky, Email: gustavo@us.ibm.com. </p> </section> <section class=\"ref-list\" id=\"Bib1\"> <h2 class=\"pmc_sec_title\"> <a class=\"usa-anchor\" id=\"Bib1-anchor\"> </a> References </h2> <section id=\"Bib1_sec2\"> <ul class=\"ref-list font-sm\"> <li id=\"CR1\"> <span class=\"label\"> 1. </span> <cite> Costello JC, Stolovitzky G. Seeking the wisdom of crowds through challenge-based competitions in biomedical research. Clin Pharmacol Ther. 2013;93:396–398. doi: 10.1038/clpt.2013.36. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/clpt.2013.36\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23549146/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Clin%20Pharmacol%20Ther&amp;title=Seeking%20the%20wisdom%20of%20crowds%20through%20challenge-based%20competitions%20in%20biomedical%20research&amp;author=JC%20Costello&amp;author=G%20Stolovitzky&amp;volume=93&amp;publication_year=2013&amp;pages=396-398&amp;pmid=23549146&amp;doi=10.1038/clpt.2013.36&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR2\"> <span class=\"label\"> 2. </span> <cite> Meyer P, Alexopoulos LG, Bonk T, Califano A, Cho CR, de la Fuente A, de Graaf D, Hartemink AJ, Hoeng J, Ivanov NV, Koeppl H, Linding R, Marbach D, Norel R, Peitsch MC, Rice JJ, Royyuru A, Schacherer F, Sprengel J, Stolle K, Vitkup D, Stolovitzky G. Verification of systems biology research in the age of collaborative competition. Nat Biotechnol. 2011;29:811–815. doi: 10.1038/nbt.1968. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nbt.1968\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/21904331/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&amp;title=Verification%20of%20systems%20biology%20research%20in%20the%20age%20of%20collaborative%20competition&amp;author=P%20Meyer&amp;author=LG%20Alexopoulos&amp;author=T%20Bonk&amp;author=A%20Califano&amp;author=CR%20Cho&amp;volume=29&amp;publication_year=2011&amp;pages=811-815&amp;pmid=21904331&amp;doi=10.1038/nbt.1968&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR3\"> <span class=\"label\"> 3. </span> <cite> Pertea M, Salzberg SL. Between a chicken and a grape: estimating the number of human genes. Genome Biol. 2010;11:206. doi: 10.1186/gb-2010-11-5-206. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/gb-2010-11-5-206\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2898077/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/20441615/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=Between%20a%20chicken%20and%20a%20grape:%20estimating%20the%20number%20of%20human%20genes&amp;author=M%20Pertea&amp;author=SL%20Salzberg&amp;volume=11&amp;publication_year=2010&amp;pages=206&amp;pmid=20441615&amp;doi=10.1186/gb-2010-11-5-206&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR4\"> <span class=\"label\"> 4. </span> <cite> Lim WK, Wang K, Lefebvre C, Califano A. Comparative analysis of microarray normalization procedures: effects on reverse engineering gene networks. Bioinformatics. 2007;23:i282–i288. doi: 10.1093/bioinformatics/btm201. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/bioinformatics/btm201\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/17646307/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Comparative%20analysis%20of%20microarray%20normalization%20procedures:%20effects%20on%20reverse%20engineering%20gene%20networks&amp;author=WK%20Lim&amp;author=K%20Wang&amp;author=C%20Lefebvre&amp;author=A%20Califano&amp;volume=23&amp;publication_year=2007&amp;pages=i282-i288&amp;pmid=17646307&amp;doi=10.1093/bioinformatics/btm201&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR5\"> <span class=\"label\"> 5. </span> <cite> Baggerly KA, Coombes KR. Deriving chemosensitivity from cell lines: forensic bioinformatics and reproducible research in high-throughput biology. Ann Appl Stat. 2009;3:1309–1334. doi: 10.1214/09-AOAS291. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1214/09-AOAS291\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Ann%20Appl%20Stat&amp;title=Deriving%20chemosensitivity%20from%20cell%20lines:%20forensic%20bioinformatics%20and%20reproducible%20research%20in%20high-throughput%20biology&amp;author=KA%20Baggerly&amp;author=KR%20Coombes&amp;volume=3&amp;publication_year=2009&amp;pages=1309-1334&amp;doi=10.1214/09-AOAS291&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR6\"> <span class=\"label\"> 6. </span> <cite> Norel R, Rice JJ, Stolovitzky G. The self-assessment trap: can we all be better than average? Mol Syst Biol. 2011;7:537. doi: 10.1038/msb.2011.70. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/msb.2011.70\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3261704/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/21988833/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Mol%20Syst%20Biol&amp;title=The%20self-assessment%20trap:%20can%20we%20all%20be%20better%20than%20average?&amp;author=R%20Norel&amp;author=JJ%20Rice&amp;author=G%20Stolovitzky&amp;volume=7&amp;publication_year=2011&amp;pages=537&amp;pmid=21988833&amp;doi=10.1038/msb.2011.70&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR7\"> <span class=\"label\"> 7. </span> <cite> Moult J, Pedersen JT, Judson R, Fidelis K. A large-scale experiment to assess protein structure prediction methods. Proteins. 1995;23:ii–v. doi: 10.1002/prot.340230303. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1002/prot.340230303\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/8710822/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Proteins&amp;title=A%20large-scale%20experiment%20to%20assess%20protein%20structure%20prediction%20methods&amp;author=J%20Moult&amp;author=JT%20Pedersen&amp;author=R%20Judson&amp;author=K%20Fidelis&amp;volume=23&amp;publication_year=1995&amp;pages=ii-v&amp;pmid=8710822&amp;doi=10.1002/prot.340230303&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR8\"> <span class=\"label\"> 8. </span> <cite> Cozzetto D, Kryshtafovych A, Tramontano A. Evaluation of CASP8 model quality predictions. Proteins. 2009;77:157–166. doi: 10.1002/prot.22534. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1002/prot.22534\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19714774/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Proteins&amp;title=Evaluation%20of%20CASP8%20model%20quality%20predictions&amp;author=D%20Cozzetto&amp;author=A%20Kryshtafovych&amp;author=A%20Tramontano&amp;volume=77&amp;publication_year=2009&amp;pages=157-166&amp;pmid=19714774&amp;doi=10.1002/prot.22534&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR9\"> <span class=\"label\"> 9. </span> <cite> Shi S, Pei J, Sadreyev RI, Kinch LN, Majumdar I, Tong J, Cheng H, Kim BH, Grishin NV. Analysis of CASP8 targets, predictions and assessment methods. Database (Oxford) 2009;2009:bap003. doi: 10.1093/database/bap003. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/database/bap003\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2794793/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/20157476/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Database%20(Oxford)&amp;title=Analysis%20of%20CASP8%20targets,%20predictions%20and%20assessment%20methods&amp;author=S%20Shi&amp;author=J%20Pei&amp;author=RI%20Sadreyev&amp;author=LN%20Kinch&amp;author=I%20Majumdar&amp;volume=2009&amp;publication_year=2009&amp;pages=bap003&amp;pmid=20157476&amp;doi=10.1093/database/bap003&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR10\"> <span class=\"label\"> 10. </span> <cite> Tramontano A, Morea V. Assessment of homology-based predictions in CASP5. Proteins. 2004;55:782. doi: 10.1002/prot.20187. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1002/prot.20187\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/14579324/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Proteins&amp;title=Assessment%20of%20homology-based%20predictions%20in%20CASP5&amp;author=A%20Tramontano&amp;author=V%20Morea&amp;volume=55&amp;publication_year=2004&amp;pages=782&amp;pmid=14579324&amp;doi=10.1002/prot.20187&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR11\"> <span class=\"label\"> 11. </span> <cite> Radivojac P, Clark WT, Oron TR, Schnoes AM, Wittkop T, Sokolov A, Graim K, Funk C, Verspoor K, Ben-Hur A, Pandey G, Yunes JM, Talwalkar AS, Repo S, Souza ML, Piovesan D, Casadio R, Wang Z, Cheng J, Fang H, Gough J, Koskinen P, Törönen P, Nokso-Koivisto J, Holm L, Cozzetto D, Buchan DW, Bryson K, Jones DT, Limaye B, et al. A large-scale evaluation of computational protein function prediction. Nat Methods. 2013;10:221–227. doi: 10.1038/nmeth.2340. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nmeth.2340\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3584181/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23353650/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&amp;title=A%20large-scale%20evaluation%20of%20computational%20protein%20function%20prediction&amp;author=P%20Radivojac&amp;author=WT%20Clark&amp;author=TR%20Oron&amp;author=AM%20Schnoes&amp;author=T%20Wittkop&amp;volume=10&amp;publication_year=2013&amp;pages=221-227&amp;pmid=23353650&amp;doi=10.1038/nmeth.2340&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR12\"> <span class=\"label\"> 12. </span> <cite> Prill RJ, Marbach D, Saez-Rodriguez J, Sorger PK, Alexopoulos LG, Xue X, Clarke ND, Altan-Bonnet G, Stolovitzky G. Towards a rigorous assessment of systems biology models: the DREAM3 challenges. PLoS One. 2010;5:e9202. doi: 10.1371/journal.pone.0009202. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1371/journal.pone.0009202\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2826397/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/20186320/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Towards%20a%20rigorous%20assessment%20of%20systems%20biology%20models:%20the%20DREAM3%20challenges&amp;author=RJ%20Prill&amp;author=D%20Marbach&amp;author=J%20Saez-Rodriguez&amp;author=PK%20Sorger&amp;author=LG%20Alexopoulos&amp;volume=5&amp;publication_year=2010&amp;pages=e9202&amp;pmid=20186320&amp;doi=10.1371/journal.pone.0009202&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR13\"> <span class=\"label\"> 13. </span> <cite> Stolovitzky G, Prill RJ, Califano A. Lessons from the DREAM2 challenges. Ann N Y Acad Sci. 2009;1158:159–195. doi: 10.1111/j.1749-6632.2009.04497.x. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1111/j.1749-6632.2009.04497.x\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19348640/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Ann%20N%20Y%20Acad%20Sci&amp;title=Lessons%20from%20the%20DREAM2%20challenges&amp;author=G%20Stolovitzky&amp;author=RJ%20Prill&amp;author=A%20Califano&amp;volume=1158&amp;publication_year=2009&amp;pages=159-195&amp;pmid=19348640&amp;doi=10.1111/j.1749-6632.2009.04497.x&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR14\"> <span class=\"label\"> 14. </span> <cite> Athanasopoulos G, Hyndman RJ. The value of feedback in forecasting competitions. Int J Forecast. 2011;27:845–849. doi: 10.1016/j.ijforecast.2011.03.002. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.ijforecast.2011.03.002\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Int%20J%20Forecast&amp;title=The%20value%20of%20feedback%20in%20forecasting%20competitions&amp;author=G%20Athanasopoulos&amp;author=RJ%20Hyndman&amp;volume=27&amp;publication_year=2011&amp;pages=845-849&amp;doi=10.1016/j.ijforecast.2011.03.002&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR15\"> <span class=\"label\"> 15. </span> <cite> Margolin AA, Bilal E, Huang E, Norman TC, Ottestad L, Mecham BH, Sauerwine B, Kellen MR, Mangravite LM, Furia MD, Vollan HK, Rueda OM, Guinney J, Deflaux NA, Hoff B, Schildwachter X, Russnes HG, Park D, Vang VO, Pirtle T, Youseff L, Citro C, Curtis C, Kristensen VN, Hellerstein J, Friend SH, Stolovitzky G, Aparicio S, Caldas C, Børresen-Dale AL. Systematic analysis of challenge-driven improvements in molecular prognostic models for breast cancer. Sci Transl Med. 2013;5:181re181. doi: 10.1126/scitranslmed.3006112. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1126/scitranslmed.3006112\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3897241/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23596205/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Sci%20Transl%20Med&amp;title=Systematic%20analysis%20of%20challenge-driven%20improvements%20in%20molecular%20prognostic%20models%20for%20breast%20cancer&amp;author=AA%20Margolin&amp;author=E%20Bilal&amp;author=E%20Huang&amp;author=TC%20Norman&amp;author=L%20Ottestad&amp;volume=5&amp;publication_year=2013&amp;pages=181re181&amp;pmid=23596205&amp;doi=10.1126/scitranslmed.3006112&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR16\"> <span class=\"label\"> 16. </span> <cite> <em> Synapse </em> ; [ <a class=\"usa-link usa-link--external\" href=\"http://www.sagebase.org/synapse\" rel=\"noopener noreferrer\" target=\"_blank\"> http://www.sagebase.org/synapse </a> ] </cite> </li> <li id=\"CR17\"> <span class=\"label\"> 17. </span> <cite> Omberg L, Ellrott K, Yuan Y, Kandoth C, Wong C, Kellen MR, Friend SH, Stuart J, Liang H, Margolin AA. Enabling transparent and collaborative computational analysis of 12 tumor types within The Cancer Genome Atlas. Nat Genet. 2013;45:1121–1126. doi: 10.1038/ng.2761. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/ng.2761\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3950337/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24071850/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&amp;title=Enabling%20transparent%20and%20collaborative%20computational%20analysis%20of%2012%20tumor%20types%20within%20The%20Cancer%20Genome%20Atlas&amp;author=L%20Omberg&amp;author=K%20Ellrott&amp;author=Y%20Yuan&amp;author=C%20Kandoth&amp;author=C%20Wong&amp;volume=45&amp;publication_year=2013&amp;pages=1121-1126&amp;pmid=24071850&amp;doi=10.1038/ng.2761&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR18\"> <span class=\"label\"> 18. </span> <cite> Global Alliance for Genomics and Health ; [ <a class=\"usa-link usa-link--external\" href=\"http://genomicsandhealth.org\" rel=\"noopener noreferrer\" target=\"_blank\"> http://genomicsandhealth.org </a> ] </cite> </li> <li id=\"CR19\"> <span class=\"label\"> 19. </span> <cite> Kim SY, Speed TP. Comparing somatic mutation-callers: beyond Venn diagrams. BMC Bioinformatics. 2013;14:189. doi: 10.1186/1471-2105-14-189. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/1471-2105-14-189\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3702398/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23758877/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&amp;title=Comparing%20somatic%20mutation-callers:%20beyond%20Venn%20diagrams&amp;author=SY%20Kim&amp;author=TP%20Speed&amp;volume=14&amp;publication_year=2013&amp;pages=189&amp;pmid=23758877&amp;doi=10.1186/1471-2105-14-189&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR20\"> <span class=\"label\"> 20. </span> <cite> O’Rawe J, Jiang T, Sun G, Wu Y, Wang W, Hu J, Bodily P, Tian L, Hakonarson H, Johnson WE, Wei Z, Wang K, Lyon GJ. Low concordance of multiple variant-calling pipelines: practical implications for exome and genome sequencing. Genome Med. 2013;5:28. doi: 10.1186/gm432. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/gm432\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3706896/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23537139/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Genome%20Med&amp;title=Low%20concordance%20of%20multiple%20variant-calling%20pipelines:%20practical%20implications%20for%20exome%20and%20genome%20sequencing&amp;author=J%20O%E2%80%99Rawe&amp;author=T%20Jiang&amp;author=G%20Sun&amp;author=Y%20Wu&amp;author=W%20Wang&amp;volume=5&amp;publication_year=2013&amp;pages=28&amp;pmid=23537139&amp;doi=10.1186/gm432&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR21\"> <span class=\"label\"> 21. </span> <cite> Alkan C, Coe BP, Eichler EE. Genome structural variation discovery and genotyping. Nat Rev Genet. 2011;12:363–376. doi: 10.1038/nrg2958. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nrg2958\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4108431/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/21358748/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Genet&amp;title=Genome%20structural%20variation%20discovery%20and%20genotyping&amp;author=C%20Alkan&amp;author=BP%20Coe&amp;author=EE%20Eichler&amp;volume=12&amp;publication_year=2011&amp;pages=363-376&amp;pmid=21358748&amp;doi=10.1038/nrg2958&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR22\"> <span class=\"label\"> 22. </span> <cite> <strong> Taking pan-cancer analysis global. </strong> <em> Nat Genet </em> 2013, <strong> 45: </strong> 1263. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/ng.2825\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24165723/\"> PubMed </a> ] </li> <li id=\"CR23\"> <span class=\"label\"> 23. </span> <cite> Bradnam KR, Fass JN, Alexandrov A, Baranay P, Bechner M, Birol I, Boisvert S, Chapman JA, Chapuis G, Chikhi R, Chitsaz H, Chou WC, Corbeil J, Del Fabbro C, Docking TR, Durbin R, Earl D, Emrich S, Fedotov P, Fonseca NA, Ganapathy G, Gibbs RA, Gnerre S, Godzaridis E, Goldstein S, Haimel M, Hall G, Haussler D, Hiatt JB, Ho IY, et al. Assemblathon 2: evaluating de novo methods of genome assembly in three vertebrate species. Gigascience. 2013;2:10. doi: 10.1186/2047-217X-2-10. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/2047-217X-2-10\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3844414/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23870653/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Gigascience&amp;title=Assemblathon%202:%20evaluating%20de%20novo%20methods%20of%20genome%20assembly%20in%20three%20vertebrate%20species&amp;author=KR%20Bradnam&amp;author=JN%20Fass&amp;author=A%20Alexandrov&amp;author=P%20Baranay&amp;author=M%20Bechner&amp;volume=2&amp;publication_year=2013&amp;pages=10&amp;pmid=23870653&amp;doi=10.1186/2047-217X-2-10&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR24\"> <span class=\"label\"> 24. </span> <cite> Brownstein CA, Beggs AH, Homer N, Merriman B, Yu TW, Flannery KC, Dechene ET, Towne MC, Savage SK, Price EN, Holm IA, Luquette LJ, Lyon E, Majzoub J, Neupert P, McCallie D, Jr, Szolovits P, Willard HF, Mendelsohn NJ, Temme R, Finkel RS, Yum SW, Medne L, Sunyaev SR, Adzhubey I, Cassa CA, de Bakker PI, Duzkale H, Dworzy Ski P, Fairbrother W, et al. An international effort towards developing standards for best practices in analysis, interpretation and reporting of clinical genome sequencing results in the CLARITY challenge. Genome Biol. 2014;15:R53. doi: 10.1186/gb-2014-15-3-r53. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/gb-2014-15-3-r53\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4073084/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24667040/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=An%20international%20effort%20towards%20developing%20standards%20for%20best%20practices%20in%20analysis,%20interpretation%20and%20reporting%20of%20clinical%20genome%20sequencing%20results%20in%20the%20CLARITY%20challenge&amp;author=CA%20Brownstein&amp;author=AH%20Beggs&amp;author=N%20Homer&amp;author=B%20Merriman&amp;author=TW%20Yu&amp;volume=15&amp;publication_year=2014&amp;pages=R53&amp;pmid=24667040&amp;doi=10.1186/gb-2014-15-3-r53&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR25\"> <span class=\"label\"> 25. </span> <cite> Gymrek M, McGuire AL, Golan D, Halperin E, Erlich Y. Identifying personal genomes by surname inference. Science. 2013;339:321–324. doi: 10.1126/science.1229566. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1126/science.1229566\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23329047/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Identifying%20personal%20genomes%20by%20surname%20inference&amp;author=M%20Gymrek&amp;author=AL%20McGuire&amp;author=D%20Golan&amp;author=E%20Halperin&amp;author=Y%20Erlich&amp;volume=339&amp;publication_year=2013&amp;pages=321-324&amp;pmid=23329047&amp;doi=10.1126/science.1229566&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR26\"> <span class=\"label\"> 26. </span> <cite> Good BM, Su AI. Games with a scientific purpose. Genome Biol. 2011;12:135. doi: 10.1186/gb-2011-12-12-135. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/gb-2011-12-12-135\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3334605/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/22204700/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=Games%20with%20a%20scientific%20purpose&amp;author=BM%20Good&amp;author=AI%20Su&amp;volume=12&amp;publication_year=2011&amp;pages=135&amp;pmid=22204700&amp;doi=10.1186/gb-2011-12-12-135&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR27\"> <span class=\"label\"> 27. </span> <cite> Lee J, Kladwang W, Lee M, Cantu D, Azizyan M, Kim H, Limpaecher A, Yoon S, Treuille A, Das R, Ete RNAP. RNA design rules from a massive open laboratory. Proc Natl Acad Sci U S A. 2014;111:2122–2127. doi: 10.1073/pnas.1313039111. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1073/pnas.1313039111\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3926058/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24469816/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=RNA%20design%20rules%20from%20a%20massive%20open%20laboratory&amp;author=J%20Lee&amp;author=W%20Kladwang&amp;author=M%20Lee&amp;author=D%20Cantu&amp;author=M%20Azizyan&amp;volume=111&amp;publication_year=2014&amp;pages=2122-2127&amp;pmid=24469816&amp;doi=10.1073/pnas.1313039111&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR28\"> <span class=\"label\"> 28. </span> <cite> Boutros PC, Ewing AD, Ellrott K, Norman TC, Dang KK, Hu Y, Kellen MR, Suver C, Bare JC, Stein LD, Spellman PT, Stolovitzky G, Friend SH, Margolin AA, Stuart JM. Global optimization of somatic variant identification in cancer genomes with a global community challenge. Nat Genet. 2014;46:318–319. doi: 10.1038/ng.2932. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/ng.2932\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4035501/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24675517/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&amp;title=Global%20optimization%20of%20somatic%20variant%20identification%20in%20cancer%20genomes%20with%20a%20global%20community%20challenge&amp;author=PC%20Boutros&amp;author=AD%20Ewing&amp;author=K%20Ellrott&amp;author=TC%20Norman&amp;author=KK%20Dang&amp;volume=46&amp;publication_year=2014&amp;pages=318-319&amp;pmid=24675517&amp;doi=10.1038/ng.2932&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR29\"> <span class=\"label\"> 29. </span> <cite> Dudley JT, Butte AJ. In silico research in the era of cloud computing. Nat Biotechnol. 2010;28:1181–1185. doi: 10.1038/nbt1110-1181. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nbt1110-1181\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3755123/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/21057489/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&amp;title=In%20silico%20research%20in%20the%20era%20of%20cloud%20computing&amp;author=JT%20Dudley&amp;author=AJ%20Butte&amp;volume=28&amp;publication_year=2010&amp;pages=1181-1185&amp;pmid=21057489&amp;doi=10.1038/nbt1110-1181&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR30\"> <span class=\"label\"> 30. </span> <cite> Lambin P, van Stiphout RG, Starmans MH, Rios-Velazquez E, Nalbantov G, Aerts HJ, Roelofs E, van Elmpt W, Boutros PC, Granone P, Valentini V, Begg AC, De Ruysscher D, Dekker A. Predicting outcomes in radiation oncology - multifactorial decision support systems. Nat Rev Clin Oncol. 2013;10:27–40. doi: 10.1038/nrclinonc.2012.196. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nrclinonc.2012.196\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC4555846/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23165123/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Clin%20Oncol&amp;title=Predicting%20outcomes%20in%20radiation%20oncology%20-%20multifactorial%20decision%20support%20systems&amp;author=P%20Lambin&amp;author=RG%20van%20Stiphout&amp;author=MH%20Starmans&amp;author=E%20Rios-Velazquez&amp;author=G%20Nalbantov&amp;volume=10&amp;publication_year=2013&amp;pages=27-40&amp;pmid=23165123&amp;doi=10.1038/nrclinonc.2012.196&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR31\"> <span class=\"label\"> 31. </span> <cite> Chin L, Gray JW. Translating insights from the cancer genome into clinical practice. Nature. 2008;452:553–563. doi: 10.1038/nature06914. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nature06914\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2730524/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/18385729/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Translating%20insights%20from%20the%20cancer%20genome%20into%20clinical%20practice&amp;author=L%20Chin&amp;author=JW%20Gray&amp;volume=452&amp;publication_year=2008&amp;pages=553-563&amp;pmid=18385729&amp;doi=10.1038/nature06914&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR32\"> <span class=\"label\"> 32. </span> <cite> Khleif SN, Doroshow JH, Hait WN. AACR-FDA-NCI Cancer Biomarkers Collaborative consensus report: advancing the use of biomarkers in cancer drug development. Clin Cancer Res. 2010;16:3299–3318. doi: 10.1158/1078-0432.CCR-10-0880. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1158/1078-0432.CCR-10-0880\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/20501613/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&amp;title=AACR-FDA-NCI%20Cancer%20Biomarkers%20Collaborative%20consensus%20report:%20advancing%20the%20use%20of%20biomarkers%20in%20cancer%20drug%20development&amp;author=SN%20Khleif&amp;author=JH%20Doroshow&amp;author=WN%20Hait&amp;volume=16&amp;publication_year=2010&amp;pages=3299-3318&amp;pmid=20501613&amp;doi=10.1158/1078-0432.CCR-10-0880&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR33\"> <span class=\"label\"> 33. </span> <cite> van’t Veer LJ, Bernards R. Enabling personalized cancer medicine through analysis of gene-expression patterns. Nature. 2008;452:564–570. doi: 10.1038/nature06915. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nature06915\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/18385730/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Enabling%20personalized%20cancer%20medicine%20through%20analysis%20of%20gene-expression%20patterns&amp;author=LJ%20van%E2%80%99t%20Veer&amp;author=R%20Bernards&amp;volume=452&amp;publication_year=2008&amp;pages=564-570&amp;pmid=18385730&amp;doi=10.1038/nature06915&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR34\"> <span class=\"label\"> 34. </span> <cite> Starmans MH, Pintilie M, John T, Der SD, Shepherd FA, Jurisica I, Lambin P, Tsao MS, Boutros PC. Exploiting the noise: improving biomarkers with ensembles of data analysis methodologies. Genome Med. 2012;4:84. doi: 10.1186/gm385. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1186/gm385\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3580418/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23146350/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Genome%20Med&amp;title=Exploiting%20the%20noise:%20improving%20biomarkers%20with%20ensembles%20of%20data%20analysis%20methodologies&amp;author=MH%20Starmans&amp;author=M%20Pintilie&amp;author=T%20John&amp;author=SD%20Der&amp;author=FA%20Shepherd&amp;volume=4&amp;publication_year=2012&amp;pages=84&amp;pmid=23146350&amp;doi=10.1186/gm385&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR35\"> <span class=\"label\"> 35. </span> <cite> Starmans MH, Fung G, Steck H, Wouters BG, Lambin P. A simple but highly effective approach to evaluate the prognostic performance of gene expression signatures. PLoS One. 2011;6:e28320. doi: 10.1371/journal.pone.0028320. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1371/journal.pone.0028320\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3233554/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/22163293/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=A%20simple%20but%20highly%20effective%20approach%20to%20evaluate%20the%20prognostic%20performance%20of%20gene%20expression%20signatures&amp;author=MH%20Starmans&amp;author=G%20Fung&amp;author=H%20Steck&amp;author=BG%20Wouters&amp;author=P%20Lambin&amp;volume=6&amp;publication_year=2011&amp;pages=e28320&amp;pmid=22163293&amp;doi=10.1371/journal.pone.0028320&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR36\"> <span class=\"label\"> 36. </span> <cite> Venet D, Dumont JE, Detours V. Most random gene expression signatures are significantly associated with breast cancer outcome. PLoS Comput Biol. 2011;7:e1002240. doi: 10.1371/journal.pcbi.1002240. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1371/journal.pcbi.1002240\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3197658/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/22028643/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=PLoS%20Comput%20Biol&amp;title=Most%20random%20gene%20expression%20signatures%20are%20significantly%20associated%20with%20breast%20cancer%20outcome&amp;author=D%20Venet&amp;author=JE%20Dumont&amp;author=V%20Detours&amp;volume=7&amp;publication_year=2011&amp;pages=e1002240&amp;pmid=22028643&amp;doi=10.1371/journal.pcbi.1002240&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR37\"> <span class=\"label\"> 37. </span> <cite> Boutros PC, Lau SK, Pintilie M, Liu N, Shepherd FA, Der SD, Tsao MS, Penn LZ, Jurisica I. Prognostic gene signatures for non-small-cell lung cancer. Proc Natl Acad Sci U S A. 2009;106:2824–2828. doi: 10.1073/pnas.0809444106. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1073/pnas.0809444106\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2636731/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19196983/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=Prognostic%20gene%20signatures%20for%20non-small-cell%20lung%20cancer&amp;author=PC%20Boutros&amp;author=SK%20Lau&amp;author=M%20Pintilie&amp;author=N%20Liu&amp;author=FA%20Shepherd&amp;volume=106&amp;publication_year=2009&amp;pages=2824-2828&amp;pmid=19196983&amp;doi=10.1073/pnas.0809444106&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR38\"> <span class=\"label\"> 38. </span> <cite> Bentzien J, Muegge I, Hamner B, Thompson DC. Crowd computing: using competitive dynamics to develop and refine highly predictive models. Drug Discov Today. 2013;18:472–478. doi: 10.1016/j.drudis.2013.01.002. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1016/j.drudis.2013.01.002\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23337388/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Drug%20Discov%20Today&amp;title=Crowd%20computing:%20using%20competitive%20dynamics%20to%20develop%20and%20refine%20highly%20predictive%20models&amp;author=J%20Bentzien&amp;author=I%20Muegge&amp;author=B%20Hamner&amp;author=DC%20Thompson&amp;volume=18&amp;publication_year=2013&amp;pages=472-478&amp;pmid=23337388&amp;doi=10.1016/j.drudis.2013.01.002&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR39\"> <span class=\"label\"> 39. </span> <cite> Shi L, Campbell G, Jones WD, Campagne F, Wen Z, Walker SJ, Su Z, Chu TM, Goodsaid FM, Pusztai L, Shaughnessy JD, Jr, Oberthuer A, Thomas RS, Paules RS, Fielden M, Barlogie B, Chen W, Du P, Fischer M, Furlanello C, Gallas BD, Ge X, Megherbi DB, Symmans WF, Wang MD, Zhang J, Bitter H, Brors B, Bushel PR, Bylesjo M, et al. The MicroArray Quality Control (MAQC)-II study of common practices for the development and validation of microarray-based predictive models. Nat Biotechnol. 2010;28:827–838. doi: 10.1038/nbt.1665. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nbt.1665\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3315840/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/20676074/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&amp;title=The%20MicroArray%20Quality%20Control%20(MAQC)-II%20study%20of%20common%20practices%20for%20the%20development%20and%20validation%20of%20microarray-based%20predictive%20models&amp;author=L%20Shi&amp;author=G%20Campbell&amp;author=WD%20Jones&amp;author=F%20Campagne&amp;author=Z%20Wen&amp;volume=28&amp;publication_year=2010&amp;pages=827-838&amp;pmid=20676074&amp;doi=10.1038/nbt.1665&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR40\"> <span class=\"label\"> 40. </span> <cite> Tarca AL, Lauria M, Unger M, Bilal E, Boue S, Kumar Dey K, Hoeng J, Koeppl H, Martin F, Meyer P, Nandy P, Norel R, Peitsch M, Rice JJ, Romero R, Stolovitzky G, Talikka M, Xiang Y, Zechner C, IMPROVER DSC Collaborators Strengths and limitations of microarray-based phenotype prediction: lessons learned from the IMPROVER Diagnostic Signature Challenge. Bioinformatics. 2013;29:2892–2899. doi: 10.1093/bioinformatics/btt492. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/bioinformatics/btt492\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3810846/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23966112/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Strengths%20and%20limitations%20of%20microarray-based%20phenotype%20prediction:%20lessons%20learned%20from%20the%20IMPROVER%20Diagnostic%20Signature%20Challenge&amp;author=AL%20Tarca&amp;author=M%20Lauria&amp;author=M%20Unger&amp;author=E%20Bilal&amp;author=S%20Boue&amp;volume=29&amp;publication_year=2013&amp;pages=2892-2899&amp;pmid=23966112&amp;doi=10.1093/bioinformatics/btt492&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR41\"> <span class=\"label\"> 41. </span> <cite> <em> Acute Myeloid Leukemia Outcome Prediction Challenge </em> ; [ <a class=\"usa-link usa-link--external\" href=\"https://www.synapse.org/#!Synapse:syn2455683\" rel=\"noopener noreferrer\" target=\"_blank\"> https://www.synapse.org/#!Synapse:syn2455683 </a> ] </cite> </li> <li id=\"CR42\"> <span class=\"label\"> 42. </span> <cite> Pujana MA, Han JD, Starita LM, Stevens KN, Tewari M, Ahn JS, Rennert G, Moreno V, Kirchhoff T, Gold B, Assmann V, Elshamy WM, Rual JF, Levine D, Rozek LS, Gelman RS, Gunsalus KC, Greenberg RA, Sobhian B, Bertin N, Venkatesan K, Ayivi-Guedehoussou N, Solé X, Hernández P, Lázaro C, Nathanson KL, Weber BL, Cusick ME, Hill DE, Offit K, et al. Network modeling links breast cancer susceptibility and centrosome dysfunction. Nat Genet. 2007;39:1338–1349. doi: 10.1038/ng.2007.2. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/ng.2007.2\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/17922014/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&amp;title=Network%20modeling%20links%20breast%20cancer%20susceptibility%20and%20centrosome%20dysfunction&amp;author=MA%20Pujana&amp;author=JD%20Han&amp;author=LM%20Starita&amp;author=KN%20Stevens&amp;author=M%20Tewari&amp;volume=39&amp;publication_year=2007&amp;pages=1338-1349&amp;pmid=17922014&amp;doi=10.1038/ng.2007.2&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR43\"> <span class=\"label\"> 43. </span> <cite> Taylor IW, Linding R, Warde-Farley D, Liu Y, Pesquita C, Faria D, Bull S, Pawson T, Morris Q, Wrana JL. Dynamic modularity in protein interaction networks predicts breast cancer outcome. Nat Biotechnol. 2009;27:199–204. doi: 10.1038/nbt.1522. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nbt.1522\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19182785/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&amp;title=Dynamic%20modularity%20in%20protein%20interaction%20networks%20predicts%20breast%20cancer%20outcome&amp;author=IW%20Taylor&amp;author=R%20Linding&amp;author=D%20Warde-Farley&amp;author=Y%20Liu&amp;author=C%20Pesquita&amp;volume=27&amp;publication_year=2009&amp;pages=199-204&amp;pmid=19182785&amp;doi=10.1038/nbt.1522&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR44\"> <span class=\"label\"> 44. </span> <cite> Vaske CJ, Benz SC, Sanborn JZ, Earl D, Szeto C, Zhu J, Haussler D, Stuart JM. Inference of patient-specific pathway activities from multi-dimensional cancer genomics data using PARADIGM. Bioinformatics. 2010;26:i237–i245. doi: 10.1093/bioinformatics/btq182. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/bioinformatics/btq182\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2881367/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/20529912/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Inference%20of%20patient-specific%20pathway%20activities%20from%20multi-dimensional%20cancer%20genomics%20data%20using%20PARADIGM&amp;author=CJ%20Vaske&amp;author=SC%20Benz&amp;author=JZ%20Sanborn&amp;author=D%20Earl&amp;author=C%20Szeto&amp;volume=26&amp;publication_year=2010&amp;pages=i237-i245&amp;pmid=20529912&amp;doi=10.1093/bioinformatics/btq182&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR45\"> <span class=\"label\"> 45. </span> <cite> Marbach D, Prill RJ, Schaffter T, Mattiussi C, Floreano D, Stolovitzky G. Revealing strengths and weaknesses of methods for gene network inference. Proc Natl Acad Sci U S A. 2010;107:6286–6291. doi: 10.1073/pnas.0913357107. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1073/pnas.0913357107\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2851985/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/20308593/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=Revealing%20strengths%20and%20weaknesses%20of%20methods%20for%20gene%20network%20inference&amp;author=D%20Marbach&amp;author=RJ%20Prill&amp;author=T%20Schaffter&amp;author=C%20Mattiussi&amp;author=D%20Floreano&amp;volume=107&amp;publication_year=2010&amp;pages=6286-6291&amp;pmid=20308593&amp;doi=10.1073/pnas.0913357107&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR46\"> <span class=\"label\"> 46. </span> <cite> Marbach D, Costello JC, Kuffner R, Vega NM, Prill RJ, Camacho DM, Allison KR, Consortium D, Kellis M, Collins JJ, Stolovitzky G. Wisdom of crowds for robust gene network inference. Nat Methods. 2012;9:796–804. doi: 10.1038/nmeth.2016. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nmeth.2016\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3512113/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/22796662/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&amp;title=Wisdom%20of%20crowds%20for%20robust%20gene%20network%20inference&amp;author=D%20Marbach&amp;author=JC%20Costello&amp;author=R%20Kuffner&amp;author=NM%20Vega&amp;author=RJ%20Prill&amp;volume=9&amp;publication_year=2012&amp;pages=796-804&amp;pmid=22796662&amp;doi=10.1038/nmeth.2016&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR47\"> <span class=\"label\"> 47. </span> <cite> Prill RJ, Saez-Rodriguez J, Alexopoulos LG, Sorger PK, Stolovitzky G. Crowdsourcing network inference: the DREAM predictive signaling network challenge. Sci Signal. 2011;4:mr7. doi: 10.1126/scisignal.2002212. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1126/scisignal.2002212\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3465072/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/21900204/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Sci%20Signal&amp;title=Crowdsourcing%20network%20inference:%20the%20DREAM%20predictive%20signaling%20network%20challenge&amp;author=RJ%20Prill&amp;author=J%20Saez-Rodriguez&amp;author=LG%20Alexopoulos&amp;author=PK%20Sorger&amp;author=G%20Stolovitzky&amp;volume=4&amp;publication_year=2011&amp;pages=mr7&amp;pmid=21900204&amp;doi=10.1126/scisignal.2002212&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR48\"> <span class=\"label\"> 48. </span> <cite> <em> HPN-DREAM breast cancer network inference challenge </em> ; [ <a class=\"usa-link usa-link--external\" href=\"https://www.synapse.org/#!Synapse:syn1720047\" rel=\"noopener noreferrer\" target=\"_blank\"> https://www.synapse.org/#!Synapse:syn1720047 </a> ] </cite> </li> <li id=\"CR49\"> <span class=\"label\"> 49. </span> <cite> Maier R, Zimmer R, Kuffner R. A Turing test for artificial expression data. Bioinformatics. 2013;29:2603–2609. doi: 10.1093/bioinformatics/btt438. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/bioinformatics/btt438\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23956305/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=A%20Turing%20test%20for%20artificial%20expression%20data&amp;author=R%20Maier&amp;author=R%20Zimmer&amp;author=R%20Kuffner&amp;volume=29&amp;publication_year=2013&amp;pages=2603-2609&amp;pmid=23956305&amp;doi=10.1093/bioinformatics/btt438&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR50\"> <span class=\"label\"> 50. </span> <cite> Canales RD, Luo Y, Willey JC, Austermiller B, Barbacioru CC, Boysen C, Hunkapiller K, Jensen RV, Knight CR, Lee KY, Ma Y, Maqsodi B, Papallo A, Peters EH, Poulter K, Ruppel PL, Samaha RR, Shi L, Yang W, Zhang L, Goodsaid FM. Evaluation of DNA microarray results with quantitative gene expression platforms. Nat Biotechnol. 2006;24:1115–1122. doi: 10.1038/nbt1236. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nbt1236\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/16964225/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&amp;title=Evaluation%20of%20DNA%20microarray%20results%20with%20quantitative%20gene%20expression%20platforms&amp;author=RD%20Canales&amp;author=Y%20Luo&amp;author=JC%20Willey&amp;author=B%20Austermiller&amp;author=CC%20Barbacioru&amp;volume=24&amp;publication_year=2006&amp;pages=1115-1122&amp;pmid=16964225&amp;doi=10.1038/nbt1236&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR51\"> <span class=\"label\"> 51. </span> <cite> Roberts ND, Kortschak RD, Parker WT, Schreiber AW, Branford S, Scott HS, Glonek G, Adelson DL. A comparative analysis of algorithms for somatic SNV detection in cancer. Bioinformatics. 2013;29:2223–2230. doi: 10.1093/bioinformatics/btt375. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1093/bioinformatics/btt375\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3753564/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/23842810/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=A%20comparative%20analysis%20of%20algorithms%20for%20somatic%20SNV%20detection%20in%20cancer&amp;author=ND%20Roberts&amp;author=RD%20Kortschak&amp;author=WT%20Parker&amp;author=AW%20Schreiber&amp;author=S%20Branford&amp;volume=29&amp;publication_year=2013&amp;pages=2223-2230&amp;pmid=23842810&amp;doi=10.1093/bioinformatics/btt375&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR52\"> <span class=\"label\"> 52. </span> <cite> Bell AW, Deutsch EW, Au CE, Kearney RE, Beavis R, Sechi S, Nilsson T, Bergeron JJ, Group HTSW. A HUPO test sample study reveals common problems in mass spectrometry-based proteomics. Nat Methods. 2009;6:423–430. doi: 10.1038/nmeth.1333. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nmeth.1333\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC2785450/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/19448641/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&amp;title=A%20HUPO%20test%20sample%20study%20reveals%20common%20problems%20in%20mass%20spectrometry-based%20proteomics&amp;author=AW%20Bell&amp;author=EW%20Deutsch&amp;author=CE%20Au&amp;author=RE%20Kearney&amp;author=R%20Beavis&amp;volume=6&amp;publication_year=2009&amp;pages=423-430&amp;pmid=19448641&amp;doi=10.1038/nmeth.1333&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR53\"> <span class=\"label\"> 53. </span> <cite> ‘t Hoen PA, Friedländer MR, Almlöf J, Sammeth M, Pulyakhina I, Anvar SY, Laros JF, Buermans HP, Karlberg O, Brännvall M, GEUVADIS Consortium. den Dunnen JT, van Ommen GJ, Gut IG, Guigó R, Estivill X, Syvänen AC, Dermitzakis ET, Lappalainen T. Reproducibility of high-throughput mRNA and small RNA sequencing across laboratories. Nat Biotechnol. 2013;31:1015–1022. doi: 10.1038/nbt.2702. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nbt.2702\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24037425/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&amp;title=Reproducibility%20of%20high-throughput%20mRNA%20and%20small%20RNA%20sequencing%20across%20laboratories&amp;author=PA%20%E2%80%98t%20Hoen&amp;author=MR%20Friedl%C3%A4nder&amp;author=J%20Alml%C3%B6f&amp;author=M%20Sammeth&amp;author=I%20Pulyakhina&amp;volume=31&amp;publication_year=2013&amp;pages=1015-1022&amp;pmid=24037425&amp;doi=10.1038/nbt.2702&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR54\"> <span class=\"label\"> 54. </span> <cite> Steijger T, Abril JF, Engstrom PG, Kokocinski F, Consortium R, Akerman M, Alioto T, Ambrosini G, Antonarakis SE, Behr J, Bertone P. Assessment of transcript reconstruction methods for RNA-seq. Nat Methods. 2013;10:1177–1184. doi: 10.1038/nmeth.2714. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nmeth.2714\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"/articles/PMC3851240/\"> PMC free article </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/24185837/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&amp;title=Assessment%20of%20transcript%20reconstruction%20methods%20for%20RNA-seq&amp;author=T%20Steijger&amp;author=JF%20Abril&amp;author=PG%20Engstrom&amp;author=F%20Kokocinski&amp;author=R%20Consortium&amp;volume=10&amp;publication_year=2013&amp;pages=1177-1184&amp;pmid=24185837&amp;doi=10.1038/nmeth.2714&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR55\"> <span class=\"label\"> 55. </span> <cite> Ransohoff DF. Proteomics research to discover markers: what can we learn from Netflix? Clin Chem. 2010;56:172–176. doi: 10.1373/clinchem.2009.126698. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1373/clinchem.2009.126698\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/20040622/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Clin%20Chem&amp;title=Proteomics%20research%20to%20discover%20markers:%20what%20can%20we%20learn%20from%20Netflix?&amp;author=DF%20Ransohoff&amp;volume=56&amp;publication_year=2010&amp;pages=172-176&amp;pmid=20040622&amp;doi=10.1373/clinchem.2009.126698&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> <li id=\"CR56\"> <span class=\"label\"> 56. </span> <cite> Waters H. New $10 million X Prize launched for tricorder-style medical device. Nat Med. 2011;17:754. doi: 10.1038/nm0711-754a. </cite> [ <a class=\"usa-link usa-link--external\" href=\"https://doi.org/10.1038/nm0711-754a\" rel=\"noopener noreferrer\" target=\"_blank\"> DOI </a> ] [ <a class=\"usa-link\" href=\"https://pubmed.ncbi.nlm.nih.gov/21738131/\"> PubMed </a> ] [ <a class=\"usa-link usa-link--external\" href=\"https://scholar.google.com/scholar_lookup?journal=Nat%20Med&amp;title=New%20%2410%20million%20X%20Prize%20launched%20for%20tricorder-style%20medical%20device&amp;author=H%20Waters&amp;volume=17&amp;publication_year=2011&amp;pages=754&amp;pmid=21738131&amp;doi=10.1038/nm0711-754a&amp;\" rel=\"noopener noreferrer\" target=\"_blank\"> Google Scholar </a> ] </li> </ul> </section> </section> </section> <footer class=\"p courtesy-note font-secondary font-sm text-center\"> <hr class=\"headless\"/> <p> Articles from Genome Biology are provided here courtesy of <strong> BMC </strong> </p> </footer> </section> </article> </main> </div> </div> </div> <div class=\"pmc-sidenav desktop:grid-col-4 display-flex\"> <section aria-label=\"Article resources and navigation\" class=\"pmc-sidenav__container\"> <div class=\"display-none desktop:display-block\"> <section class=\"margin-top-4 desktop:margin-top-0\"> <h2 class=\"margin-top-0\"> ACTIONS </h2> <ul class=\"usa-list usa-list--unstyled usa-list--actions\"> <li> <a class=\"usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1\" href=\"https://doi.org/10.1186/s13059-014-0462-7\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"display-inline-flex flex-justify-center flex-1 padding-right-2\"> View on publisher site </span> </a> </li> <li> <a class=\"usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1\" href=\"pdf/13059_2014_Article_462.pdf\"> <span class=\"display-inline-flex flex-justify-center flex-1\"> PDF (542.9 KB) </span> </a> </li> <li> </li> <li> </li> <li class=\"pmc-permalink\"> <div class=\"pmc-permalink__dropdown\" hidden=\"\"> <div class=\"pmc-permalink__dropdown__container\"> <h2 class=\"usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs\"> PERMALINK </h2> <div class=\"pmc-permalink__dropdown__content\"> </div> </div> </div> </li> </ul> </section> </div> <section class=\"pmc-resources margin-top-6 desktop:margin-top-4\"> <h2 class=\"margin-top-0\"> RESOURCES </h2> <div class=\"usa-accordion usa-accordion--multiselectable\"> <h3 class=\"usa-accordion__heading\"> </h3> <div class=\"usa-accordion__content usa-prose\" hidden=\"\" id=\"resources-similar-articles\"> </div> <h3 class=\"usa-accordion__heading\"> </h3> <div class=\"usa-accordion__content usa-prose\" hidden=\"\" id=\"resources-cited-by-other-articles\"> </div> <h3 class=\"usa-accordion__heading\"> </h3> <div class=\"usa-accordion__content usa-prose\" hidden=\"\" id=\"resources-links-to-ncbi-databases\"> </div> </div> </section> <section class=\"usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4\"> <nav aria-label=\"On this page\" class=\"usa-in-page-nav__nav\"> <h2 class=\"usa-in-page-nav__heading\" tabindex=\"0\"> On this page </h2> <ul class=\"usa-in-page-nav__list\"> <li class=\"usa-in-page-nav__item\"> <a class=\"usa-in-page-nav__link\" href=\"#Abs1\"> Abstract </a> </li> <li class=\"usa-in-page-nav__item\"> <a class=\"usa-in-page-nav__link\" href=\"#Sec1\"> Challenge design and dynamics </a> </li> <li class=\"usa-in-page-nav__item\"> <a class=\"usa-in-page-nav__link\" href=\"#Sec2\"> Analyzing genome assembly and structural variants </a> </li> <li class=\"usa-in-page-nav__item\"> <a class=\"usa-in-page-nav__link\" href=\"#Sec3\"> Finding genomic biomarkers that are associated with phenotype </a> </li> <li class=\"usa-in-page-nav__item\"> <a class=\"usa-in-page-nav__link\" href=\"#Sec4\"> Inferring biological networks underlying cancer phenotypes </a> </li> <li class=\"usa-in-page-nav__item\"> <a class=\"usa-in-page-nav__link\" href=\"#Sec5\"> The truth is hard to find </a> </li> <li class=\"usa-in-page-nav__item\"> <a class=\"usa-in-page-nav__link\" href=\"#Sec6\"> Closing considerations </a> </li> <li class=\"usa-in-page-nav__item\"> <a class=\"usa-in-page-nav__link\" href=\"#ack1\"> Acknowledgements </a> </li> <li class=\"usa-in-page-nav__item\"> <a class=\"usa-in-page-nav__link\" href=\"#glossary1\"> Abbreviations </a> </li> <li class=\"usa-in-page-nav__item\"> <a class=\"usa-in-page-nav__link\" href=\"#fn-group1\"> Footnotes </a> </li> <li class=\"usa-in-page-nav__item\"> <a class=\"usa-in-page-nav__link\" href=\"#_ci93_\"> Contributor Information </a> </li> <li class=\"usa-in-page-nav__item\"> <a class=\"usa-in-page-nav__link\" href=\"#Bib1\"> References </a> </li> </ul> </nav> </section> </section> </div> <div aria-label=\"Citation Dialog\" class=\"overlay citation-dialog-overlay\" role=\"dialog\"> <div aria-hidden=\"true\" class=\"dialog citation-dialog\" role=\"document\"> <div class=\"display-inline-flex flex-align-center flex-justify width-full margin-bottom-2\"> <h2 class=\"usa-modal__heading margin-0\"> Cite </h2> </div> <div class=\"citation-text-block\"> <div class=\"citation-text margin-bottom-2\"> </div> <ul class=\"usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center\"> <li> </li> <li> <a class=\"usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button\" href=\"#\" role=\"button\" tabindex=\"3\" title=\"Download a file for external citation management software\"> <span class=\"display-none mobile-lg:display-inline\"> Download .nbib </span> <span class=\"display-inline mobile-lg:display-none\"> .nbib </span> </a> </li> <li> <div class=\"display-inline-flex flex-align-center\"> <label class=\"usa-label margin-top-0\"> Format: </label> <select aria-label=\"Format\" class=\"usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4\" tabindex=\"4\"> <option value=\"AMA\"> AMA </option> <option value=\"APA\"> APA </option> <option value=\"MLA\"> MLA </option> <option selected=\"selected\" value=\"NLM\"> NLM </option> </select> </div> </li> </ul> <div class=\"dots-loading-indicator citation-loading-indicator\"> <div class=\"dot dot-1\"> </div> <div class=\"dot dot-2\"> </div> <div class=\"dot dot-3\"> </div> </div> </div> </div> </div> <div class=\"overlay collections-dialog-overlay\" role=\"dialog\"> <div aria-hidden=\"true\" class=\"dialog collections-dialog\" id=\"collections-action-dialog\" role=\"document\"> <div class=\"display-inline-flex flex-align-center flex-justify width-full margin-bottom-2\"> <h2 class=\"usa-modal__heading margin-0\"> Add to Collections </h2> </div> <div class=\"collections-action-panel action-panel\"> </div> </div> </div> </div> </div> </div> <footer class=\"ncbi-footer ncbi-dark-background\"> <div class=\"ncbi-footer__icon-section\"> <div class=\"ncbi-footer__social-header\"> Follow NCBI </div> <div class=\"grid-container ncbi-footer__ncbi-social-icons-container\"> <a class=\"ncbi-footer__social-icon ncbi-footer__social-icon--gray\" href=\"https://twitter.com/ncbi\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"usa-sr-only\"> NCBI on X (formerly known as Twitter) </span> </a> <a class=\"ncbi-footer__social-icon ncbi-footer__social-icon--gray\" href=\"https://www.facebook.com/ncbi.nlm\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"usa-sr-only\"> NCBI on Facebook </span> </a> <a class=\"ncbi-footer__social-icon ncbi-footer__social-icon--gray\" href=\"https://www.linkedin.com/company/ncbinlm\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"usa-sr-only\"> NCBI on LinkedIn </span> </a> <a class=\"ncbi-footer__social-icon ncbi-footer__social-icon--gray\" href=\"https://github.com/ncbi\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"usa-sr-only\"> NCBI on GitHub </span> </a> <a class=\"ncbi-footer__social-icon ncbi-footer__social-icon--gray\" href=\"https://ncbiinsights.ncbi.nlm.nih.gov/\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"usa-sr-only\"> NCBI RSS feed </span> </a> </div> </div> <div class=\"grid-container ncbi-footer__container\"> <div class=\"grid-row ncbi-footer__main-content-container\"> <div class=\"ncbi-footer__column\"> <p class=\"ncbi-footer__circled-icons-heading\"> Connect with NLM </p> <div class=\"ncbi-footer__circled-icons-list\"> <a class=\"ncbi-footer__social-icon ncbi-footer__social-icon--circled\" href=\"https://twitter.com/nlm_nih\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"usa-sr-only\"> NLM on X (formerly known as Twitter) </span> </a> <a class=\"ncbi-footer__social-icon ncbi-footer__social-icon--circled\" href=\"https://www.facebook.com/nationallibraryofmedicine\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"usa-sr-only\"> NLM on Facebook </span> </a> <a class=\"ncbi-footer__social-icon ncbi-footer__social-icon--circled\" href=\"https://www.youtube.com/user/NLMNIH\" rel=\"noreferrer noopener\" target=\"_blank\"> <span class=\"usa-sr-only\"> NLM on YouTube </span> </a> </div> </div> <address class=\"ncbi-footer__address ncbi-footer__column\"> <p> <a class=\"usa-link usa-link--external\" href=\"https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508, -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508! 4d-77.0988323\" rel=\"noopener noreferrer\" target=\"_blank\"> National Library of Medicine <br/> 8600 Rockville Pike <br/> Bethesda, MD 20894 </a> </p> </address> <ul class=\"usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column\"> <li class=\"ncbi-footer__vertical-list-item\"> <a class=\"usa-link usa-link--alt ncbi-footer__link\" href=\"https://www.nlm.nih.gov/web_policies.html\"> Web Policies </a> </li> <li class=\"ncbi-footer__vertical-list-item\"> <a class=\"usa-link usa-link--alt ncbi-footer__link\" href=\"https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office\"> FOIA </a> </li> <li class=\"ncbi-footer__vertical-list-item\"> <a class=\"usa-link usa-link--external usa-link--alt ncbi-footer__link\" href=\"https://www.hhs.gov/vulnerability-disclosure-policy/index.html\" rel=\"noreferrer noopener\" target=\"_blank\"> HHS Vulnerability Disclosure </a> </li> </ul> <ul class=\"usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column\"> <li class=\"ncbi-footer__vertical-list-item\"> <a class=\"usa-link usa-link--alt ncbi-footer__link\" href=\"https://support.nlm.nih.gov/?pagename=cloudpmc-viewer%3Apmc%3Aarticle%3Ajournal\"> Help </a> </li> <li class=\"ncbi-footer__vertical-list-item\"> <a class=\"usa-link usa-link--alt ncbi-footer__link\" href=\"https://www.nlm.nih.gov/accessibility.html\"> Accessibility </a> </li> <li class=\"ncbi-footer__vertical-list-item\"> <a class=\"usa-link usa-link--alt ncbi-footer__link\" href=\"https://www.nlm.nih.gov/careers/careers.html\"> Careers </a> </li> </ul> </div> <div class=\"grid-row grid-col-12\"> <ul class=\"ncbi-footer__bottom-links-list\"> <li class=\"ncbi-footer__bottom-list-item\"> <a class=\"usa-link usa-link--alt ncbi-footer__link\" href=\"https://www.nlm.nih.gov/\"> NLM </a> </li> <li class=\"ncbi-footer__bottom-list-item\"> <a class=\"usa-link usa-link--alt ncbi-footer__link\" href=\"https://www.nih.gov/\"> NIH </a> </li> <li class=\"ncbi-footer__bottom-list-item\"> <a class=\"usa-link usa-link--external usa-link--alt ncbi-footer__link\" href=\"https://www.hhs.gov/\" rel=\"noreferrer noopener\" target=\"_blank\"> HHS </a> </li> <li class=\"ncbi-footer__bottom-list-item\"> <a class=\"usa-link usa-link--external usa-link--alt ncbi-footer__link\" href=\"https://www.usa.gov/\" rel=\"noreferrer noopener\" target=\"_blank\"> USA.gov </a> </li> </ul> </div> </div> </footer> <div id=\"ZN_dikYWqsjiUWN0Q5\"> </div> <div class=\"fake-body-scroll\"> </div> <div class=\"fake-body-scroll\"> </div> <div class=\"QSIFeedbackButton\"> </div> </body> </html>.'}]\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 132 - DEBUG - Calculating checksum for content: prompt\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 332 - INFO - Prompt ID: gemini-2.0-flash-0-b3957ef191c09db8f0f90cc8e120239756fe272bbc0f66d39447d7b1be8951d9\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1329 - DEBUG - Counting tokens for model: gemini-2.0-flash, prompt length: 140801 char\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1338 - DEBUG - Rough estimate of token count for Gemini model 'gemini-2.0-flash': 35200\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 351 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 35200 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-8889dbca-d809-4add-a817-d723986c7584\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"Assemblathon1&2\\\",\\n    \\\"repository_reference\\\": \\\"http://assemblathon.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"CAFA\\\",\\n    \\\"repository_reference\\\": \\\"http://biofunctionprediction.org/node/8\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"CAGI\\\",\\n    \\\"repository_reference\\\": \\\"http://genomeinterpretation.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"CAPRI\\\",\\n    \\\"repository_reference\\\": \\\"http://www.ebi.ac.uk/msd-srv/capri/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"CASP\\\",\\n    \\\"repository_reference\\\": \\\"http://predictioncenter.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"ChaLearn\\\",\\n    \\\"repository_reference\\\": \\\"http://www.chalearn.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"CLARITY\\\",\\n    \\\"repository_reference\\\": \\\"http://www.childrenshospital.org/research-and-innovation/research-initiatives/clarity-challenge\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"DREAM\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Challenges:DREAM\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"FlowCAP\\\",\\n    \\\"repository_reference\\\": \\\"http://flowcap.flowsite.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"ICGC-TCGA DREAM Somatic Mutation Calling\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn312572\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"IMPROVER\\\",\\n    \\\"repository_reference\\\": \\\"https://sbvimprover.com/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Innocentive\\\",\\n    \\\"repository_reference\\\": \\\"http://www.innocentive.com/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Kaggle\\\",\\n    \\\"repository_reference\\\": \\\"http://www.kaggle.com/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"RGASP\\\",\\n    \\\"repository_reference\\\": \\\"http://www.gencodegenes.org/rgasp/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Sequence Squeeze\\\",\\n    \\\"repository_reference\\\": \\\"http://sequencesqueeze.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"X-Prize\\\",\\n    \\\"repository_reference\\\": \\\"http://www.xprize.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Synapse\\\",\\n    \\\"repository_reference\\\": \\\"http://www.sagebase.org/synapse\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Global Alliance for Genomics and Health\\\",\\n    \\\"repository_reference\\\": \\\"http://genomicsandhealth.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Acute Myeloid Leukemia Outcome Prediction Challenge\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn2455683\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"HPN-DREAM breast cancer network inference challenge\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn1720047\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"Assemblathon1&2\\\",\\n    \\\"repository_reference\\\": \\\"http://assemblathon.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"CAFA\\\",\\n    \\\"repository_reference\\\": \\\"http://biofunctionprediction.org/node/8\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"CAGI\\\",\\n    \\\"repository_reference\\\": \\\"http://genomeinterpretation.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"CAPRI\\\",\\n    \\\"repository_reference\\\": \\\"http://www.ebi.ac.uk/msd-srv/capri/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"CASP\\\",\\n    \\\"repository_reference\\\": \\\"http://predictioncenter.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"ChaLearn\\\",\\n    \\\"repository_reference\\\": \\\"http://www.chalearn.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"CLARITY\\\",\\n    \\\"repository_reference\\\": \\\"http://www.childrenshospital.org/research-and-innovation/research-initiatives/clarity-challenge\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"DREAM\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Challenges:DREAM\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"FlowCAP\\\",\\n    \\\"repository_reference\\\": \\\"http://flowcap.flowsite.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"ICGC-TCGA DREAM Somatic Mutation Calling\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn312572\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"IMPROVER\\\",\\n    \\\"repository_reference\\\": \\\"https://sbvimprover.com/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Innocentive\\\",\\n    \\\"repository_reference\\\": \\\"http://www.innocentive.com/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Kaggle\\\",\\n    \\\"repository_reference\\\": \\\"http://www.kaggle.com/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"RGASP\\\",\\n    \\\"repository_reference\\\": \\\"http://www.gencodegenes.org/rgasp/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Sequence Squeeze\\\",\\n    \\\"repository_reference\\\": \\\"http://sequencesqueeze.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"X-Prize\\\",\\n    \\\"repository_reference\\\": \\\"http://www.xprize.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Synapse\\\",\\n    \\\"repository_reference\\\": \\\"http://www.sagebase.org/synapse\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Global Alliance for Genomics and Health\\\",\\n    \\\"repository_reference\\\": \\\"http://genomicsandhealth.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Acute Myeloid Leukemia Outcome Prediction Challenge\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn2455683\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"HPN-DREAM breast cancer network inference challenge\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn1720047\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1757718417,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 45734,\n",
      "        \"completion_tokens\": 792,\n",
      "        \"total_tokens\": 46526,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 664 - INFO - Deduplicating response with 20 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 672 - DEBUG - Processing item: {'dataset_identifier': 'Assemblathon1&2', 'repository_reference': 'http://assemblathon.org/'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: Assemblathon1&2\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 672 - DEBUG - Processing item: {'dataset_identifier': 'CAFA', 'repository_reference': 'http://biofunctionprediction.org/node/8'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: CAFA\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 672 - DEBUG - Processing item: {'dataset_identifier': 'CAGI', 'repository_reference': 'http://genomeinterpretation.org/'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: CAGI\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 672 - DEBUG - Processing item: {'dataset_identifier': 'CAPRI', 'repository_reference': 'http://www.ebi.ac.uk/msd-srv/capri/'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: CAPRI\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 672 - DEBUG - Processing item: {'dataset_identifier': 'CASP', 'repository_reference': 'http://predictioncenter.org/'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: CASP\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 672 - DEBUG - Processing item: {'dataset_identifier': 'ChaLearn', 'repository_reference': 'http://www.chalearn.org/'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: ChaLearn\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 672 - DEBUG - Processing item: {'dataset_identifier': 'CLARITY', 'repository_reference': 'http://www.childrenshospital.org/research-and-innovation/research-initiatives/clarity-challenge'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: CLARITY\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 672 - DEBUG - Processing item: {'dataset_identifier': 'DREAM', 'repository_reference': 'https://www.synapse.org/#!Challenges:DREAM'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: DREAM\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 672 - DEBUG - Processing item: {'dataset_identifier': 'FlowCAP', 'repository_reference': 'http://flowcap.flowsite.org/'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: FlowCAP\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 672 - DEBUG - Processing item: {'dataset_identifier': 'ICGC-TCGA DREAM Somatic Mutation Calling', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn312572'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: ICGC-TCGA DREAM Somatic Mutation Calling\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 672 - DEBUG - Processing item: {'dataset_identifier': 'IMPROVER', 'repository_reference': 'https://sbvimprover.com/'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: IMPROVER\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 672 - DEBUG - Processing item: {'dataset_identifier': 'Innocentive', 'repository_reference': 'http://www.innocentive.com/'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: Innocentive\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 672 - DEBUG - Processing item: {'dataset_identifier': 'Kaggle', 'repository_reference': 'http://www.kaggle.com/'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: Kaggle\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 672 - DEBUG - Processing item: {'dataset_identifier': 'RGASP', 'repository_reference': 'http://www.gencodegenes.org/rgasp/'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: RGASP\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 672 - DEBUG - Processing item: {'dataset_identifier': 'Sequence Squeeze', 'repository_reference': 'http://sequencesqueeze.org/'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: Sequence Squeeze\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 672 - DEBUG - Processing item: {'dataset_identifier': 'X-Prize', 'repository_reference': 'http://www.xprize.org/'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: X-Prize\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 672 - DEBUG - Processing item: {'dataset_identifier': 'Synapse', 'repository_reference': 'http://www.sagebase.org/synapse'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 672 - DEBUG - Processing item: {'dataset_identifier': 'Global Alliance for Genomics and Health', 'repository_reference': 'http://genomicsandhealth.org'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: Global Alliance for Genomics and Health\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 672 - DEBUG - Processing item: {'dataset_identifier': 'Acute Myeloid Leukemia Outcome Prediction Challenge', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn2455683'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: Acute Myeloid Leukemia Outcome Prediction Challenge\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 672 - DEBUG - Processing item: {'dataset_identifier': 'HPN-DREAM breast cancer network inference challenge', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn1720047'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: HPN-DREAM breast cancer network inference challenge\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'Assemblathon1&2', 'repository_reference': 'http://assemblathon.org/', 'dataset_id': 'Assemblathon1&2'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: Assemblathon1&2\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 985 - DEBUG - All repository names: ['bigd', 'european genome-phenome archive', 'massive', 'pride', 'iprox', 'proteomecentral', 'proteomexchange', 'proteomic data commons', 'genomic data commons', 'the cancer imaging archive', 'imaging data commons', 'jpost', 'gene expression omnibus', 'database of genotypes and phenotypes', 'sequence read archive', 'arrayexpress', 'nucleotide', 'mendeley', 'synapse', 'doi.org', 'protein data bank']\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID Assemblathon1&2 is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: http://assemblathon.org/, identifier: Assemblathon1&2, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: http://assemblathon.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://assemblathon.org/ == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://assemblathon.org/ == ega-archive.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://assemblathon.org/ == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://assemblathon.org/ == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://assemblathon.org/ == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://assemblathon.org/ == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://assemblathon.org/ == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://assemblathon.org/ == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Assemblathon1&2 for http://assemblathon.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://assemblathon.org/ == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://assemblathon.org/ == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://assemblathon.org/ == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://assemblathon.org/ == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://assemblathon.org/ == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier Assemblathon1&2 for http://assemblathon.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://assemblathon.org/ == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Assemblathon1&2 for http://assemblathon.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://assemblathon.org/ == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://assemblathon.org/ == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://assemblathon.org/ == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://assemblathon.org/ == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://assemblathon.org/ == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Assemblathon1&2 for http://assemblathon.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://assemblathon.org/ == doi.org. Repo vals: dict_keys(['dataset_webpage_url_ptr', 'id_pattern', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier Assemblathon1&2 for http://assemblathon.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://assemblathon.org/ == pdb. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier Assemblathon1&2 for http://assemblathon.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: http://assemblathon.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 962 - DEBUG - Repo Domain: assemblathon.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'Assemblathon1&2', 'data_repository': 'assemblathon.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'CAFA', 'repository_reference': 'http://biofunctionprediction.org/node/8', 'dataset_id': 'CAFA'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: CAFA\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 985 - DEBUG - All repository names: ['bigd', 'european genome-phenome archive', 'massive', 'pride', 'iprox', 'proteomecentral', 'proteomexchange', 'proteomic data commons', 'genomic data commons', 'the cancer imaging archive', 'imaging data commons', 'jpost', 'gene expression omnibus', 'database of genotypes and phenotypes', 'sequence read archive', 'arrayexpress', 'nucleotide', 'mendeley', 'synapse', 'doi.org', 'protein data bank']\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID CAFA is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: http://biofunctionprediction.org/node/8, identifier: CAFA, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: http://biofunctionprediction.org/node/8\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://biofunctionprediction.org/node/8 == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://biofunctionprediction.org/node/8 == ega-archive.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://biofunctionprediction.org/node/8 == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://biofunctionprediction.org/node/8 == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://biofunctionprediction.org/node/8 == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://biofunctionprediction.org/node/8 == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://biofunctionprediction.org/node/8 == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://biofunctionprediction.org/node/8 == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier CAFA for http://biofunctionprediction.org/node/8\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://biofunctionprediction.org/node/8 == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://biofunctionprediction.org/node/8 == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://biofunctionprediction.org/node/8 == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://biofunctionprediction.org/node/8 == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://biofunctionprediction.org/node/8 == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier CAFA for http://biofunctionprediction.org/node/8\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://biofunctionprediction.org/node/8 == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier CAFA for http://biofunctionprediction.org/node/8\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://biofunctionprediction.org/node/8 == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://biofunctionprediction.org/node/8 == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://biofunctionprediction.org/node/8 == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://biofunctionprediction.org/node/8 == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://biofunctionprediction.org/node/8 == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier CAFA for http://biofunctionprediction.org/node/8\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://biofunctionprediction.org/node/8 == doi.org. Repo vals: dict_keys(['dataset_webpage_url_ptr', 'id_pattern', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier CAFA for http://biofunctionprediction.org/node/8\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://biofunctionprediction.org/node/8 == pdb. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier CAFA for http://biofunctionprediction.org/node/8\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: http://biofunctionprediction.org/node/8\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 962 - DEBUG - Repo Domain: biofunctionprediction.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'CAFA', 'data_repository': 'biofunctionprediction.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'CAGI', 'repository_reference': 'http://genomeinterpretation.org/', 'dataset_id': 'CAGI'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: CAGI\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 985 - DEBUG - All repository names: ['bigd', 'european genome-phenome archive', 'massive', 'pride', 'iprox', 'proteomecentral', 'proteomexchange', 'proteomic data commons', 'genomic data commons', 'the cancer imaging archive', 'imaging data commons', 'jpost', 'gene expression omnibus', 'database of genotypes and phenotypes', 'sequence read archive', 'arrayexpress', 'nucleotide', 'mendeley', 'synapse', 'doi.org', 'protein data bank']\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID CAGI is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: http://genomeinterpretation.org/, identifier: CAGI, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: http://genomeinterpretation.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://genomeinterpretation.org/ == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://genomeinterpretation.org/ == ega-archive.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://genomeinterpretation.org/ == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://genomeinterpretation.org/ == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://genomeinterpretation.org/ == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://genomeinterpretation.org/ == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://genomeinterpretation.org/ == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://genomeinterpretation.org/ == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier CAGI for http://genomeinterpretation.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://genomeinterpretation.org/ == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://genomeinterpretation.org/ == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://genomeinterpretation.org/ == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://genomeinterpretation.org/ == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://genomeinterpretation.org/ == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier CAGI for http://genomeinterpretation.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://genomeinterpretation.org/ == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier CAGI for http://genomeinterpretation.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://genomeinterpretation.org/ == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://genomeinterpretation.org/ == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://genomeinterpretation.org/ == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://genomeinterpretation.org/ == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://genomeinterpretation.org/ == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier CAGI for http://genomeinterpretation.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://genomeinterpretation.org/ == doi.org. Repo vals: dict_keys(['dataset_webpage_url_ptr', 'id_pattern', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier CAGI for http://genomeinterpretation.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://genomeinterpretation.org/ == pdb. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier CAGI for http://genomeinterpretation.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: http://genomeinterpretation.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 962 - DEBUG - Repo Domain: genomeinterpretation.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'CAGI', 'data_repository': 'genomeinterpretation.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'CAPRI', 'repository_reference': 'http://www.ebi.ac.uk/msd-srv/capri/', 'dataset_id': 'CAPRI'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: CAPRI\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 985 - DEBUG - All repository names: ['bigd', 'european genome-phenome archive', 'massive', 'pride', 'iprox', 'proteomecentral', 'proteomexchange', 'proteomic data commons', 'genomic data commons', 'the cancer imaging archive', 'imaging data commons', 'jpost', 'gene expression omnibus', 'database of genotypes and phenotypes', 'sequence read archive', 'arrayexpress', 'nucleotide', 'mendeley', 'synapse', 'doi.org', 'protein data bank']\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID CAPRI is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: http://www.ebi.ac.uk/msd-srv/capri/, identifier: CAPRI, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: http://www.ebi.ac.uk/msd-srv/capri/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.ebi.ac.uk/msd-srv/capri/ == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.ebi.ac.uk/msd-srv/capri/ == ega-archive.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.ebi.ac.uk/msd-srv/capri/ == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.ebi.ac.uk/msd-srv/capri/ == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.ebi.ac.uk/msd-srv/capri/ == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.ebi.ac.uk/msd-srv/capri/ == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.ebi.ac.uk/msd-srv/capri/ == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.ebi.ac.uk/msd-srv/capri/ == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier CAPRI for http://www.ebi.ac.uk/msd-srv/capri/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.ebi.ac.uk/msd-srv/capri/ == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.ebi.ac.uk/msd-srv/capri/ == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.ebi.ac.uk/msd-srv/capri/ == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.ebi.ac.uk/msd-srv/capri/ == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.ebi.ac.uk/msd-srv/capri/ == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier CAPRI for http://www.ebi.ac.uk/msd-srv/capri/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.ebi.ac.uk/msd-srv/capri/ == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier CAPRI for http://www.ebi.ac.uk/msd-srv/capri/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.ebi.ac.uk/msd-srv/capri/ == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.ebi.ac.uk/msd-srv/capri/ == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.ebi.ac.uk/msd-srv/capri/ == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.ebi.ac.uk/msd-srv/capri/ == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.ebi.ac.uk/msd-srv/capri/ == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier CAPRI for http://www.ebi.ac.uk/msd-srv/capri/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.ebi.ac.uk/msd-srv/capri/ == doi.org. Repo vals: dict_keys(['dataset_webpage_url_ptr', 'id_pattern', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier CAPRI for http://www.ebi.ac.uk/msd-srv/capri/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.ebi.ac.uk/msd-srv/capri/ == pdb. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier CAPRI for http://www.ebi.ac.uk/msd-srv/capri/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: http://www.ebi.ac.uk/msd-srv/capri/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 962 - DEBUG - Repo Domain: www.ebi.ac.uk\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'CAPRI', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'CASP', 'repository_reference': 'http://predictioncenter.org/', 'dataset_id': 'CASP'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: CASP\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 985 - DEBUG - All repository names: ['bigd', 'european genome-phenome archive', 'massive', 'pride', 'iprox', 'proteomecentral', 'proteomexchange', 'proteomic data commons', 'genomic data commons', 'the cancer imaging archive', 'imaging data commons', 'jpost', 'gene expression omnibus', 'database of genotypes and phenotypes', 'sequence read archive', 'arrayexpress', 'nucleotide', 'mendeley', 'synapse', 'doi.org', 'protein data bank']\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID CASP is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: http://predictioncenter.org/, identifier: CASP, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: http://predictioncenter.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://predictioncenter.org/ == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://predictioncenter.org/ == ega-archive.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://predictioncenter.org/ == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://predictioncenter.org/ == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://predictioncenter.org/ == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://predictioncenter.org/ == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://predictioncenter.org/ == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://predictioncenter.org/ == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier CASP for http://predictioncenter.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://predictioncenter.org/ == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://predictioncenter.org/ == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://predictioncenter.org/ == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://predictioncenter.org/ == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://predictioncenter.org/ == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier CASP for http://predictioncenter.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://predictioncenter.org/ == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier CASP for http://predictioncenter.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://predictioncenter.org/ == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://predictioncenter.org/ == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://predictioncenter.org/ == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://predictioncenter.org/ == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://predictioncenter.org/ == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier CASP for http://predictioncenter.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://predictioncenter.org/ == doi.org. Repo vals: dict_keys(['dataset_webpage_url_ptr', 'id_pattern', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier CASP for http://predictioncenter.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://predictioncenter.org/ == pdb. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier CASP for http://predictioncenter.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: http://predictioncenter.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 962 - DEBUG - Repo Domain: predictioncenter.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'CASP', 'data_repository': 'predictioncenter.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'ChaLearn', 'repository_reference': 'http://www.chalearn.org/', 'dataset_id': 'ChaLearn'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: ChaLearn\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 985 - DEBUG - All repository names: ['bigd', 'european genome-phenome archive', 'massive', 'pride', 'iprox', 'proteomecentral', 'proteomexchange', 'proteomic data commons', 'genomic data commons', 'the cancer imaging archive', 'imaging data commons', 'jpost', 'gene expression omnibus', 'database of genotypes and phenotypes', 'sequence read archive', 'arrayexpress', 'nucleotide', 'mendeley', 'synapse', 'doi.org', 'protein data bank']\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID ChaLearn is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: http://www.chalearn.org/, identifier: ChaLearn, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: http://www.chalearn.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.chalearn.org/ == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.chalearn.org/ == ega-archive.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.chalearn.org/ == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.chalearn.org/ == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.chalearn.org/ == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.chalearn.org/ == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.chalearn.org/ == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.chalearn.org/ == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier ChaLearn for http://www.chalearn.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.chalearn.org/ == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.chalearn.org/ == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.chalearn.org/ == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.chalearn.org/ == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.chalearn.org/ == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier ChaLearn for http://www.chalearn.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.chalearn.org/ == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier ChaLearn for http://www.chalearn.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.chalearn.org/ == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.chalearn.org/ == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.chalearn.org/ == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.chalearn.org/ == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.chalearn.org/ == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier ChaLearn for http://www.chalearn.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.chalearn.org/ == doi.org. Repo vals: dict_keys(['dataset_webpage_url_ptr', 'id_pattern', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier ChaLearn for http://www.chalearn.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.chalearn.org/ == pdb. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier ChaLearn for http://www.chalearn.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: http://www.chalearn.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 962 - DEBUG - Repo Domain: www.chalearn.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'ChaLearn', 'data_repository': 'www.chalearn.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'CLARITY', 'repository_reference': 'http://www.childrenshospital.org/research-and-innovation/research-initiatives/clarity-challenge', 'dataset_id': 'CLARITY'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: CLARITY\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 985 - DEBUG - All repository names: ['bigd', 'european genome-phenome archive', 'massive', 'pride', 'iprox', 'proteomecentral', 'proteomexchange', 'proteomic data commons', 'genomic data commons', 'the cancer imaging archive', 'imaging data commons', 'jpost', 'gene expression omnibus', 'database of genotypes and phenotypes', 'sequence read archive', 'arrayexpress', 'nucleotide', 'mendeley', 'synapse', 'doi.org', 'protein data bank']\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID CLARITY is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: http://www.childrenshospital.org/research-and-innovation/research-initiatives/clarity-challenge, identifier: CLARITY, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: http://www.childrenshospital.org/research-and-innovation/research-initiatives/clarity-challenge\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.childrenshospital.org/research-and-innovation/research-initiatives/clarity-challenge == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.childrenshospital.org/research-and-innovation/research-initiatives/clarity-challenge == ega-archive.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.childrenshospital.org/research-and-innovation/research-initiatives/clarity-challenge == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.childrenshospital.org/research-and-innovation/research-initiatives/clarity-challenge == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.childrenshospital.org/research-and-innovation/research-initiatives/clarity-challenge == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.childrenshospital.org/research-and-innovation/research-initiatives/clarity-challenge == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.childrenshospital.org/research-and-innovation/research-initiatives/clarity-challenge == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.childrenshospital.org/research-and-innovation/research-initiatives/clarity-challenge == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier CLARITY for http://www.childrenshospital.org/research-and-innovation/research-initiatives/clarity-challenge\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.childrenshospital.org/research-and-innovation/research-initiatives/clarity-challenge == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.childrenshospital.org/research-and-innovation/research-initiatives/clarity-challenge == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.childrenshospital.org/research-and-innovation/research-initiatives/clarity-challenge == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.childrenshospital.org/research-and-innovation/research-initiatives/clarity-challenge == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.childrenshospital.org/research-and-innovation/research-initiatives/clarity-challenge == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier CLARITY for http://www.childrenshospital.org/research-and-innovation/research-initiatives/clarity-challenge\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.childrenshospital.org/research-and-innovation/research-initiatives/clarity-challenge == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier CLARITY for http://www.childrenshospital.org/research-and-innovation/research-initiatives/clarity-challenge\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.childrenshospital.org/research-and-innovation/research-initiatives/clarity-challenge == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.childrenshospital.org/research-and-innovation/research-initiatives/clarity-challenge == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.childrenshospital.org/research-and-innovation/research-initiatives/clarity-challenge == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.childrenshospital.org/research-and-innovation/research-initiatives/clarity-challenge == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.childrenshospital.org/research-and-innovation/research-initiatives/clarity-challenge == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier CLARITY for http://www.childrenshospital.org/research-and-innovation/research-initiatives/clarity-challenge\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.childrenshospital.org/research-and-innovation/research-initiatives/clarity-challenge == doi.org. Repo vals: dict_keys(['dataset_webpage_url_ptr', 'id_pattern', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier CLARITY for http://www.childrenshospital.org/research-and-innovation/research-initiatives/clarity-challenge\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.childrenshospital.org/research-and-innovation/research-initiatives/clarity-challenge == pdb. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier CLARITY for http://www.childrenshospital.org/research-and-innovation/research-initiatives/clarity-challenge\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: http://www.childrenshospital.org/research-and-innovation/research-initiatives/clarity-challenge\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 962 - DEBUG - Repo Domain: www.childrenshospital.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'CLARITY', 'data_repository': 'www.childrenshospital.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'DREAM', 'repository_reference': 'https://www.synapse.org/#!Challenges:DREAM', 'dataset_id': 'DREAM'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: DREAM\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 985 - DEBUG - All repository names: ['bigd', 'european genome-phenome archive', 'massive', 'pride', 'iprox', 'proteomecentral', 'proteomexchange', 'proteomic data commons', 'genomic data commons', 'the cancer imaging archive', 'imaging data commons', 'jpost', 'gene expression omnibus', 'database of genotypes and phenotypes', 'sequence read archive', 'arrayexpress', 'nucleotide', 'mendeley', 'synapse', 'doi.org', 'protein data bank']\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID DREAM is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Challenges:DREAM, identifier: DREAM, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Challenges:DREAM\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Challenges:DREAM == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Challenges:DREAM == ega-archive.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Challenges:DREAM == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Challenges:DREAM == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Challenges:DREAM == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Challenges:DREAM == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Challenges:DREAM == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Challenges:DREAM == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier DREAM for https://www.synapse.org/#!Challenges:DREAM\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Challenges:DREAM == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Challenges:DREAM == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Challenges:DREAM == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Challenges:DREAM == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Challenges:DREAM == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier DREAM for https://www.synapse.org/#!Challenges:DREAM\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Challenges:DREAM == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier DREAM for https://www.synapse.org/#!Challenges:DREAM\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Challenges:DREAM == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Challenges:DREAM == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Challenges:DREAM == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Challenges:DREAM == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Challenges:DREAM == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Challenges:DREAM in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'DREAM', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'FlowCAP', 'repository_reference': 'http://flowcap.flowsite.org/', 'dataset_id': 'FlowCAP'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: FlowCAP\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 985 - DEBUG - All repository names: ['bigd', 'european genome-phenome archive', 'massive', 'pride', 'iprox', 'proteomecentral', 'proteomexchange', 'proteomic data commons', 'genomic data commons', 'the cancer imaging archive', 'imaging data commons', 'jpost', 'gene expression omnibus', 'database of genotypes and phenotypes', 'sequence read archive', 'arrayexpress', 'nucleotide', 'mendeley', 'synapse', 'doi.org', 'protein data bank']\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID FlowCAP is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: http://flowcap.flowsite.org/, identifier: FlowCAP, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: http://flowcap.flowsite.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://flowcap.flowsite.org/ == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://flowcap.flowsite.org/ == ega-archive.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://flowcap.flowsite.org/ == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://flowcap.flowsite.org/ == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://flowcap.flowsite.org/ == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://flowcap.flowsite.org/ == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://flowcap.flowsite.org/ == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://flowcap.flowsite.org/ == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier FlowCAP for http://flowcap.flowsite.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://flowcap.flowsite.org/ == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://flowcap.flowsite.org/ == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://flowcap.flowsite.org/ == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://flowcap.flowsite.org/ == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://flowcap.flowsite.org/ == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier FlowCAP for http://flowcap.flowsite.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://flowcap.flowsite.org/ == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier FlowCAP for http://flowcap.flowsite.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://flowcap.flowsite.org/ == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://flowcap.flowsite.org/ == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://flowcap.flowsite.org/ == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://flowcap.flowsite.org/ == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://flowcap.flowsite.org/ == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier FlowCAP for http://flowcap.flowsite.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://flowcap.flowsite.org/ == doi.org. Repo vals: dict_keys(['dataset_webpage_url_ptr', 'id_pattern', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier FlowCAP for http://flowcap.flowsite.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://flowcap.flowsite.org/ == pdb. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier FlowCAP for http://flowcap.flowsite.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: http://flowcap.flowsite.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 962 - DEBUG - Repo Domain: flowcap.flowsite.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'FlowCAP', 'data_repository': 'flowcap.flowsite.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'ICGC-TCGA DREAM Somatic Mutation Calling', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn312572', 'dataset_id': 'ICGC-TCGA DREAM Somatic Mutation Calling'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn312572\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn312572\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 583 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn312572, identifier: ICGC-TCGA DREAM Somatic Mutation Calling, dataset_page: https://www.synapse.org/#!Synapse:syn312572\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn312572\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn312572 == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn312572 == ega-archive.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn312572 == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn312572 == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn312572 == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn312572 == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn312572 == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn312572 == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier ICGC-TCGA DREAM Somatic Mutation Calling for https://www.synapse.org/#!Synapse:syn312572\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn312572 == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn312572 == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn312572 == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn312572 == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn312572 == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier ICGC-TCGA DREAM Somatic Mutation Calling for https://www.synapse.org/#!Synapse:syn312572\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn312572 == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier ICGC-TCGA DREAM Somatic Mutation Calling for https://www.synapse.org/#!Synapse:syn312572\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn312572 == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn312572 == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn312572 == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn312572 == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn312572 == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn312572 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn312572\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, synapse.org, https://www.synapse.org/#!Synapse:syn312572\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: ICGC-TCGA DREAM Somatic Mutation Calling\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 985 - DEBUG - All repository names: ['bigd', 'european genome-phenome archive', 'massive', 'pride', 'iprox', 'proteomecentral', 'proteomexchange', 'proteomic data commons', 'genomic data commons', 'the cancer imaging archive', 'imaging data commons', 'jpost', 'gene expression omnibus', 'database of genotypes and phenotypes', 'sequence read archive', 'arrayexpress', 'nucleotide', 'mendeley', 'synapse', 'doi.org', 'protein data bank']\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID ICGC-TCGA DREAM Somatic Mutation Calling is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn312572\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 644 - INFO - Dataset ID is 'n/a' and repository name from prompt\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 541 - INFO - Skipping dataset due to missing ID, repository, dataset page: {'dataset_identifier': 'ICGC-TCGA DREAM Somatic Mutation Calling', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn312572', 'dataset_id': 'ICGC-TCGA DREAM Somatic Mutation Calling'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'IMPROVER', 'repository_reference': 'https://sbvimprover.com/', 'dataset_id': 'IMPROVER'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: IMPROVER\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 985 - DEBUG - All repository names: ['bigd', 'european genome-phenome archive', 'massive', 'pride', 'iprox', 'proteomecentral', 'proteomexchange', 'proteomic data commons', 'genomic data commons', 'the cancer imaging archive', 'imaging data commons', 'jpost', 'gene expression omnibus', 'database of genotypes and phenotypes', 'sequence read archive', 'arrayexpress', 'nucleotide', 'mendeley', 'synapse', 'doi.org', 'protein data bank']\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID IMPROVER is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://sbvimprover.com/, identifier: IMPROVER, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://sbvimprover.com/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://sbvimprover.com/ == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://sbvimprover.com/ == ega-archive.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://sbvimprover.com/ == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://sbvimprover.com/ == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://sbvimprover.com/ == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://sbvimprover.com/ == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://sbvimprover.com/ == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://sbvimprover.com/ == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier IMPROVER for https://sbvimprover.com/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://sbvimprover.com/ == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://sbvimprover.com/ == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://sbvimprover.com/ == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://sbvimprover.com/ == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://sbvimprover.com/ == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier IMPROVER for https://sbvimprover.com/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://sbvimprover.com/ == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier IMPROVER for https://sbvimprover.com/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://sbvimprover.com/ == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://sbvimprover.com/ == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://sbvimprover.com/ == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://sbvimprover.com/ == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://sbvimprover.com/ == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier IMPROVER for https://sbvimprover.com/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://sbvimprover.com/ == doi.org. Repo vals: dict_keys(['dataset_webpage_url_ptr', 'id_pattern', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier IMPROVER for https://sbvimprover.com/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://sbvimprover.com/ == pdb. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier IMPROVER for https://sbvimprover.com/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://sbvimprover.com/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 962 - DEBUG - Repo Domain: sbvimprover.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'IMPROVER', 'data_repository': 'sbvimprover.com', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'Innocentive', 'repository_reference': 'http://www.innocentive.com/', 'dataset_id': 'Innocentive'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: Innocentive\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 985 - DEBUG - All repository names: ['bigd', 'european genome-phenome archive', 'massive', 'pride', 'iprox', 'proteomecentral', 'proteomexchange', 'proteomic data commons', 'genomic data commons', 'the cancer imaging archive', 'imaging data commons', 'jpost', 'gene expression omnibus', 'database of genotypes and phenotypes', 'sequence read archive', 'arrayexpress', 'nucleotide', 'mendeley', 'synapse', 'doi.org', 'protein data bank']\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID Innocentive is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: http://www.innocentive.com/, identifier: Innocentive, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: http://www.innocentive.com/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.innocentive.com/ == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.innocentive.com/ == ega-archive.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.innocentive.com/ == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.innocentive.com/ == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.innocentive.com/ == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.innocentive.com/ == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.innocentive.com/ == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.innocentive.com/ == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Innocentive for http://www.innocentive.com/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.innocentive.com/ == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.innocentive.com/ == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.innocentive.com/ == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.innocentive.com/ == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.innocentive.com/ == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier Innocentive for http://www.innocentive.com/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.innocentive.com/ == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Innocentive for http://www.innocentive.com/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.innocentive.com/ == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.innocentive.com/ == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.innocentive.com/ == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.innocentive.com/ == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.innocentive.com/ == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Innocentive for http://www.innocentive.com/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.innocentive.com/ == doi.org. Repo vals: dict_keys(['dataset_webpage_url_ptr', 'id_pattern', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier Innocentive for http://www.innocentive.com/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.innocentive.com/ == pdb. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier Innocentive for http://www.innocentive.com/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: http://www.innocentive.com/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 962 - DEBUG - Repo Domain: www.innocentive.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'Innocentive', 'data_repository': 'www.innocentive.com', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'Kaggle', 'repository_reference': 'http://www.kaggle.com/', 'dataset_id': 'Kaggle'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: Kaggle\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 985 - DEBUG - All repository names: ['bigd', 'european genome-phenome archive', 'massive', 'pride', 'iprox', 'proteomecentral', 'proteomexchange', 'proteomic data commons', 'genomic data commons', 'the cancer imaging archive', 'imaging data commons', 'jpost', 'gene expression omnibus', 'database of genotypes and phenotypes', 'sequence read archive', 'arrayexpress', 'nucleotide', 'mendeley', 'synapse', 'doi.org', 'protein data bank']\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID Kaggle is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: http://www.kaggle.com/, identifier: Kaggle, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: http://www.kaggle.com/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.kaggle.com/ == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.kaggle.com/ == ega-archive.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.kaggle.com/ == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.kaggle.com/ == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.kaggle.com/ == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.kaggle.com/ == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.kaggle.com/ == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.kaggle.com/ == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Kaggle for http://www.kaggle.com/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.kaggle.com/ == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.kaggle.com/ == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.kaggle.com/ == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.kaggle.com/ == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.kaggle.com/ == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier Kaggle for http://www.kaggle.com/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.kaggle.com/ == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Kaggle for http://www.kaggle.com/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.kaggle.com/ == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.kaggle.com/ == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.kaggle.com/ == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.kaggle.com/ == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.kaggle.com/ == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Kaggle for http://www.kaggle.com/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.kaggle.com/ == doi.org. Repo vals: dict_keys(['dataset_webpage_url_ptr', 'id_pattern', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier Kaggle for http://www.kaggle.com/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.kaggle.com/ == pdb. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier Kaggle for http://www.kaggle.com/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: http://www.kaggle.com/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 962 - DEBUG - Repo Domain: www.kaggle.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'Kaggle', 'data_repository': 'www.kaggle.com', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'RGASP', 'repository_reference': 'http://www.gencodegenes.org/rgasp/', 'dataset_id': 'RGASP'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: RGASP\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 985 - DEBUG - All repository names: ['bigd', 'european genome-phenome archive', 'massive', 'pride', 'iprox', 'proteomecentral', 'proteomexchange', 'proteomic data commons', 'genomic data commons', 'the cancer imaging archive', 'imaging data commons', 'jpost', 'gene expression omnibus', 'database of genotypes and phenotypes', 'sequence read archive', 'arrayexpress', 'nucleotide', 'mendeley', 'synapse', 'doi.org', 'protein data bank']\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID RGASP is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: http://www.gencodegenes.org/rgasp/, identifier: RGASP, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: http://www.gencodegenes.org/rgasp/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.gencodegenes.org/rgasp/ == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.gencodegenes.org/rgasp/ == ega-archive.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.gencodegenes.org/rgasp/ == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.gencodegenes.org/rgasp/ == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.gencodegenes.org/rgasp/ == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.gencodegenes.org/rgasp/ == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.gencodegenes.org/rgasp/ == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.gencodegenes.org/rgasp/ == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier RGASP for http://www.gencodegenes.org/rgasp/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.gencodegenes.org/rgasp/ == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.gencodegenes.org/rgasp/ == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.gencodegenes.org/rgasp/ == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.gencodegenes.org/rgasp/ == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.gencodegenes.org/rgasp/ == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier RGASP for http://www.gencodegenes.org/rgasp/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.gencodegenes.org/rgasp/ == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier RGASP for http://www.gencodegenes.org/rgasp/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.gencodegenes.org/rgasp/ == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.gencodegenes.org/rgasp/ == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.gencodegenes.org/rgasp/ == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.gencodegenes.org/rgasp/ == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.gencodegenes.org/rgasp/ == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier RGASP for http://www.gencodegenes.org/rgasp/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.gencodegenes.org/rgasp/ == doi.org. Repo vals: dict_keys(['dataset_webpage_url_ptr', 'id_pattern', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier RGASP for http://www.gencodegenes.org/rgasp/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.gencodegenes.org/rgasp/ == pdb. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier RGASP for http://www.gencodegenes.org/rgasp/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: http://www.gencodegenes.org/rgasp/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 962 - DEBUG - Repo Domain: www.gencodegenes.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'RGASP', 'data_repository': 'www.gencodegenes.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'Sequence Squeeze', 'repository_reference': 'http://sequencesqueeze.org/', 'dataset_id': 'Sequence Squeeze'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: Sequence Squeeze\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 985 - DEBUG - All repository names: ['bigd', 'european genome-phenome archive', 'massive', 'pride', 'iprox', 'proteomecentral', 'proteomexchange', 'proteomic data commons', 'genomic data commons', 'the cancer imaging archive', 'imaging data commons', 'jpost', 'gene expression omnibus', 'database of genotypes and phenotypes', 'sequence read archive', 'arrayexpress', 'nucleotide', 'mendeley', 'synapse', 'doi.org', 'protein data bank']\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID Sequence Squeeze is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: http://sequencesqueeze.org/, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: http://sequencesqueeze.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://sequencesqueeze.org/ == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://sequencesqueeze.org/ == ega-archive.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://sequencesqueeze.org/ == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://sequencesqueeze.org/ == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://sequencesqueeze.org/ == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://sequencesqueeze.org/ == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://sequencesqueeze.org/ == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://sequencesqueeze.org/ == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://sequencesqueeze.org/ == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://sequencesqueeze.org/ == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://sequencesqueeze.org/ == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://sequencesqueeze.org/ == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://sequencesqueeze.org/ == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://sequencesqueeze.org/ == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://sequencesqueeze.org/ == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://sequencesqueeze.org/ == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://sequencesqueeze.org/ == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://sequencesqueeze.org/ == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://sequencesqueeze.org/ == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://sequencesqueeze.org/ == doi.org. Repo vals: dict_keys(['dataset_webpage_url_ptr', 'id_pattern', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://sequencesqueeze.org/ == pdb. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: http://sequencesqueeze.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 962 - DEBUG - Repo Domain: sequencesqueeze.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'sequencesqueeze.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'X-Prize', 'repository_reference': 'http://www.xprize.org/', 'dataset_id': 'X-Prize'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: X-Prize\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 985 - DEBUG - All repository names: ['bigd', 'european genome-phenome archive', 'massive', 'pride', 'iprox', 'proteomecentral', 'proteomexchange', 'proteomic data commons', 'genomic data commons', 'the cancer imaging archive', 'imaging data commons', 'jpost', 'gene expression omnibus', 'database of genotypes and phenotypes', 'sequence read archive', 'arrayexpress', 'nucleotide', 'mendeley', 'synapse', 'doi.org', 'protein data bank']\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID X-Prize is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: http://www.xprize.org/, identifier: X-Prize, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: http://www.xprize.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.xprize.org/ == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.xprize.org/ == ega-archive.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.xprize.org/ == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.xprize.org/ == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.xprize.org/ == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.xprize.org/ == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.xprize.org/ == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.xprize.org/ == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier X-Prize for http://www.xprize.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.xprize.org/ == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.xprize.org/ == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.xprize.org/ == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.xprize.org/ == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.xprize.org/ == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier X-Prize for http://www.xprize.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.xprize.org/ == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier X-Prize for http://www.xprize.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.xprize.org/ == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.xprize.org/ == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.xprize.org/ == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.xprize.org/ == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.xprize.org/ == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier X-Prize for http://www.xprize.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.xprize.org/ == doi.org. Repo vals: dict_keys(['dataset_webpage_url_ptr', 'id_pattern', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier X-Prize for http://www.xprize.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.xprize.org/ == pdb. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier X-Prize for http://www.xprize.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: http://www.xprize.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 962 - DEBUG - Repo Domain: www.xprize.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'X-Prize', 'data_repository': 'www.xprize.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'Synapse', 'repository_reference': 'http://www.sagebase.org/synapse', 'dataset_id': 'Synapse'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 985 - DEBUG - All repository names: ['bigd', 'european genome-phenome archive', 'massive', 'pride', 'iprox', 'proteomecentral', 'proteomexchange', 'proteomic data commons', 'genomic data commons', 'the cancer imaging archive', 'imaging data commons', 'jpost', 'gene expression omnibus', 'database of genotypes and phenotypes', 'sequence read archive', 'arrayexpress', 'nucleotide', 'mendeley', 'synapse', 'doi.org', 'protein data bank']\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1007 - INFO - Accession ID Synapse is a known repository --> invalid, returning 'n/a'\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: http://www.sagebase.org/synapse, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: http://www.sagebase.org/synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.sagebase.org/synapse == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.sagebase.org/synapse == ega-archive.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.sagebase.org/synapse == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.sagebase.org/synapse == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.sagebase.org/synapse == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.sagebase.org/synapse == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.sagebase.org/synapse == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.sagebase.org/synapse == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.sagebase.org/synapse == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.sagebase.org/synapse == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.sagebase.org/synapse == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.sagebase.org/synapse == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.sagebase.org/synapse == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.sagebase.org/synapse == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.sagebase.org/synapse == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.sagebase.org/synapse == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.sagebase.org/synapse == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.sagebase.org/synapse == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://www.sagebase.org/synapse == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for http://www.sagebase.org/synapse in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 644 - INFO - Dataset ID is 'n/a' and repository name from prompt\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 541 - INFO - Skipping dataset due to missing ID, repository, dataset page: {'dataset_identifier': 'Synapse', 'repository_reference': 'http://www.sagebase.org/synapse', 'dataset_id': 'Synapse'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'Global Alliance for Genomics and Health', 'repository_reference': 'http://genomicsandhealth.org', 'dataset_id': 'Global Alliance for Genomics and Health'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: Global Alliance for Genomics and Health\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 985 - DEBUG - All repository names: ['bigd', 'european genome-phenome archive', 'massive', 'pride', 'iprox', 'proteomecentral', 'proteomexchange', 'proteomic data commons', 'genomic data commons', 'the cancer imaging archive', 'imaging data commons', 'jpost', 'gene expression omnibus', 'database of genotypes and phenotypes', 'sequence read archive', 'arrayexpress', 'nucleotide', 'mendeley', 'synapse', 'doi.org', 'protein data bank']\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID Global Alliance for Genomics and Health is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: http://genomicsandhealth.org, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: http://genomicsandhealth.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://genomicsandhealth.org == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://genomicsandhealth.org == ega-archive.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://genomicsandhealth.org == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://genomicsandhealth.org == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://genomicsandhealth.org == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://genomicsandhealth.org == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://genomicsandhealth.org == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://genomicsandhealth.org == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://genomicsandhealth.org == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://genomicsandhealth.org == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://genomicsandhealth.org == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://genomicsandhealth.org == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://genomicsandhealth.org == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://genomicsandhealth.org == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://genomicsandhealth.org == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://genomicsandhealth.org == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://genomicsandhealth.org == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://genomicsandhealth.org == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://genomicsandhealth.org == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://genomicsandhealth.org == doi.org. Repo vals: dict_keys(['dataset_webpage_url_ptr', 'id_pattern', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if http://genomicsandhealth.org == pdb. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: http://genomicsandhealth.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 962 - DEBUG - Repo Domain: genomicsandhealth.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'genomicsandhealth.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'Acute Myeloid Leukemia Outcome Prediction Challenge', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn2455683', 'dataset_id': 'Acute Myeloid Leukemia Outcome Prediction Challenge'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn2455683\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn2455683\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 583 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn2455683, identifier: Acute Myeloid Leukemia Outcome Prediction Challenge, dataset_page: https://www.synapse.org/#!Synapse:syn2455683\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn2455683\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn2455683 == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn2455683 == ega-archive.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn2455683 == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn2455683 == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn2455683 == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn2455683 == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn2455683 == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn2455683 == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Acute Myeloid Leukemia Outcome Prediction Challenge for https://www.synapse.org/#!Synapse:syn2455683\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn2455683 == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn2455683 == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn2455683 == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn2455683 == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn2455683 == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier Acute Myeloid Leukemia Outcome Prediction Challenge for https://www.synapse.org/#!Synapse:syn2455683\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn2455683 == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Acute Myeloid Leukemia Outcome Prediction Challenge for https://www.synapse.org/#!Synapse:syn2455683\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn2455683 == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn2455683 == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn2455683 == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn2455683 == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn2455683 == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn2455683 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn2455683\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn2455683\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, synapse.org, https://www.synapse.org/#!Synapse:syn2455683\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: Acute Myeloid Leukemia Outcome Prediction Challenge\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 985 - DEBUG - All repository names: ['bigd', 'european genome-phenome archive', 'massive', 'pride', 'iprox', 'proteomecentral', 'proteomexchange', 'proteomic data commons', 'genomic data commons', 'the cancer imaging archive', 'imaging data commons', 'jpost', 'gene expression omnibus', 'database of genotypes and phenotypes', 'sequence read archive', 'arrayexpress', 'nucleotide', 'mendeley', 'synapse', 'doi.org', 'protein data bank']\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID Acute Myeloid Leukemia Outcome Prediction Challenge is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn2455683\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 644 - INFO - Dataset ID is 'n/a' and repository name from prompt\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 541 - INFO - Skipping dataset due to missing ID, repository, dataset page: {'dataset_identifier': 'Acute Myeloid Leukemia Outcome Prediction Challenge', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn2455683', 'dataset_id': 'Acute Myeloid Leukemia Outcome Prediction Challenge'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'HPN-DREAM breast cancer network inference challenge', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn1720047', 'dataset_id': 'HPN-DREAM breast cancer network inference challenge'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn1720047\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn1720047\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 583 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn1720047, identifier: HPN-DREAM breast cancer network inference challenge, dataset_page: https://www.synapse.org/#!Synapse:syn1720047\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn1720047\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn1720047 == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn1720047 == ega-archive.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn1720047 == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn1720047 == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn1720047 == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn1720047 == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn1720047 == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn1720047 == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier HPN-DREAM breast cancer network inference challenge for https://www.synapse.org/#!Synapse:syn1720047\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn1720047 == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn1720047 == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn1720047 == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn1720047 == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn1720047 == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier HPN-DREAM breast cancer network inference challenge for https://www.synapse.org/#!Synapse:syn1720047\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn1720047 == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier HPN-DREAM breast cancer network inference challenge for https://www.synapse.org/#!Synapse:syn1720047\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn1720047 == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn1720047 == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn1720047 == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn1720047 == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if https://www.synapse.org/#!Synapse:syn1720047 == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn1720047 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn1720047\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn1720047\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, synapse.org, https://www.synapse.org/#!Synapse:syn1720047\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: HPN-DREAM breast cancer network inference challenge\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 985 - DEBUG - All repository names: ['bigd', 'european genome-phenome archive', 'massive', 'pride', 'iprox', 'proteomecentral', 'proteomexchange', 'proteomic data commons', 'genomic data commons', 'the cancer imaging archive', 'imaging data commons', 'jpost', 'gene expression omnibus', 'database of genotypes and phenotypes', 'sequence read archive', 'arrayexpress', 'nucleotide', 'mendeley', 'synapse', 'doi.org', 'protein data bank']\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID HPN-DREAM breast cancer network inference challenge is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn1720047\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 644 - INFO - Dataset ID is 'n/a' and repository name from prompt\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 541 - INFO - Skipping dataset due to missing ID, repository, dataset page: {'dataset_identifier': 'HPN-DREAM breast cancer network inference challenge', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn1720047', 'dataset_id': 'HPN-DREAM breast cancer network inference challenge'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 559 - DEBUG - Final result: [{'dataset_identifier': 'Assemblathon1&2', 'data_repository': 'assemblathon.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'CAFA', 'data_repository': 'biofunctionprediction.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'CAGI', 'data_repository': 'genomeinterpretation.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'CAPRI', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'CASP', 'data_repository': 'predictioncenter.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'ChaLearn', 'data_repository': 'www.chalearn.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'CLARITY', 'data_repository': 'www.childrenshospital.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'DREAM', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'FlowCAP', 'data_repository': 'flowcap.flowsite.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'IMPROVER', 'data_repository': 'sbvimprover.com', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'Innocentive', 'data_repository': 'www.innocentive.com', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'Kaggle', 'data_repository': 'www.kaggle.com', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'RGASP', 'data_repository': 'www.gencodegenes.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'sequencesqueeze.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'X-Prize', 'data_repository': 'www.xprize.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'genomicsandhealth.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}]\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 378 - INFO - Augmented dataset links: [{'dataset_identifier': 'Assemblathon1&2', 'data_repository': 'assemblathon.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'CAFA', 'data_repository': 'biofunctionprediction.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'CAGI', 'data_repository': 'genomeinterpretation.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'CAPRI', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'CASP', 'data_repository': 'predictioncenter.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'ChaLearn', 'data_repository': 'www.chalearn.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'CLARITY', 'data_repository': 'www.childrenshospital.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'DREAM', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'FlowCAP', 'data_repository': 'flowcap.flowsite.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'IMPROVER', 'data_repository': 'sbvimprover.com', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'Innocentive', 'data_repository': 'www.innocentive.com', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'Kaggle', 'data_repository': 'www.kaggle.com', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'RGASP', 'data_repository': 'www.gencodegenes.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'sequencesqueeze.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'X-Prize', 'data_repository': 'www.xprize.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'genomicsandhealth.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1191 - INFO - Fetching metadata for 16 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: Assemblathon1&2\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: assemblathon.org, identifier: Assemblathon1&2, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if assemblathon.org == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if assemblathon.org == ega-archive.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if assemblathon.org == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if assemblathon.org == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if assemblathon.org == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if assemblathon.org == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if assemblathon.org == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if assemblathon.org == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Assemblathon1&2 for assemblathon.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if assemblathon.org == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if assemblathon.org == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if assemblathon.org == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if assemblathon.org == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if assemblathon.org == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier Assemblathon1&2 for assemblathon.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if assemblathon.org == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Assemblathon1&2 for assemblathon.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if assemblathon.org == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if assemblathon.org == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if assemblathon.org == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if assemblathon.org == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if assemblathon.org == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Assemblathon1&2 for assemblathon.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if assemblathon.org == doi.org. Repo vals: dict_keys(['dataset_webpage_url_ptr', 'id_pattern', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier Assemblathon1&2 for assemblathon.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if assemblathon.org == pdb. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier Assemblathon1&2 for assemblathon.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: assemblathon.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL assemblathon.org may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for Assemblathon1&2 in assemblathon.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 1 with repo: assemblathon.org and accession_id: Assemblathon1&2\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1233 - DEBUG - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository assemblathon.org unknown in Ontology. Skipping dataset page 1.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: CAFA\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: biofunctionprediction.org, identifier: CAFA, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if biofunctionprediction.org == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if biofunctionprediction.org == ega-archive.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if biofunctionprediction.org == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if biofunctionprediction.org == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if biofunctionprediction.org == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if biofunctionprediction.org == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if biofunctionprediction.org == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if biofunctionprediction.org == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier CAFA for biofunctionprediction.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if biofunctionprediction.org == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if biofunctionprediction.org == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if biofunctionprediction.org == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if biofunctionprediction.org == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if biofunctionprediction.org == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier CAFA for biofunctionprediction.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if biofunctionprediction.org == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier CAFA for biofunctionprediction.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if biofunctionprediction.org == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if biofunctionprediction.org == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if biofunctionprediction.org == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if biofunctionprediction.org == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if biofunctionprediction.org == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier CAFA for biofunctionprediction.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if biofunctionprediction.org == doi.org. Repo vals: dict_keys(['dataset_webpage_url_ptr', 'id_pattern', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier CAFA for biofunctionprediction.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if biofunctionprediction.org == pdb. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier CAFA for biofunctionprediction.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: biofunctionprediction.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL biofunctionprediction.org may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for CAFA in biofunctionprediction.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 2 with repo: biofunctionprediction.org and accession_id: CAFA\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1233 - DEBUG - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository biofunctionprediction.org unknown in Ontology. Skipping dataset page 2.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: CAGI\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: genomeinterpretation.org, identifier: CAGI, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if genomeinterpretation.org == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if genomeinterpretation.org == ega-archive.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if genomeinterpretation.org == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if genomeinterpretation.org == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if genomeinterpretation.org == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if genomeinterpretation.org == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if genomeinterpretation.org == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if genomeinterpretation.org == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier CAGI for genomeinterpretation.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if genomeinterpretation.org == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if genomeinterpretation.org == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if genomeinterpretation.org == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if genomeinterpretation.org == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if genomeinterpretation.org == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier CAGI for genomeinterpretation.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if genomeinterpretation.org == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier CAGI for genomeinterpretation.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if genomeinterpretation.org == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if genomeinterpretation.org == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if genomeinterpretation.org == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if genomeinterpretation.org == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if genomeinterpretation.org == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier CAGI for genomeinterpretation.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if genomeinterpretation.org == doi.org. Repo vals: dict_keys(['dataset_webpage_url_ptr', 'id_pattern', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier CAGI for genomeinterpretation.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if genomeinterpretation.org == pdb. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier CAGI for genomeinterpretation.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: genomeinterpretation.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL genomeinterpretation.org may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for CAGI in genomeinterpretation.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 3 with repo: genomeinterpretation.org and accession_id: CAGI\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1233 - DEBUG - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository genomeinterpretation.org unknown in Ontology. Skipping dataset page 3.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: CAPRI\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: www.ebi.ac.uk, identifier: CAPRI, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.ebi.ac.uk == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.ebi.ac.uk == ega-archive.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.ebi.ac.uk == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.ebi.ac.uk == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: www.ebi.ac.uk\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for CAPRI in www.ebi.ac.uk\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 4 with repo: www.ebi.ac.uk and accession_id: CAPRI\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1233 - DEBUG - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.ebi.ac.uk/pride/archive/projects/CAPRI\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 4: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: CASP\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: predictioncenter.org, identifier: CASP, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if predictioncenter.org == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if predictioncenter.org == ega-archive.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if predictioncenter.org == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if predictioncenter.org == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if predictioncenter.org == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if predictioncenter.org == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if predictioncenter.org == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if predictioncenter.org == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier CASP for predictioncenter.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if predictioncenter.org == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if predictioncenter.org == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if predictioncenter.org == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if predictioncenter.org == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if predictioncenter.org == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier CASP for predictioncenter.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if predictioncenter.org == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier CASP for predictioncenter.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if predictioncenter.org == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if predictioncenter.org == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if predictioncenter.org == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if predictioncenter.org == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if predictioncenter.org == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier CASP for predictioncenter.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if predictioncenter.org == doi.org. Repo vals: dict_keys(['dataset_webpage_url_ptr', 'id_pattern', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier CASP for predictioncenter.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if predictioncenter.org == pdb. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier CASP for predictioncenter.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: predictioncenter.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL predictioncenter.org may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for CASP in predictioncenter.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 5 with repo: predictioncenter.org and accession_id: CASP\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1233 - DEBUG - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository predictioncenter.org unknown in Ontology. Skipping dataset page 5.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: ChaLearn\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: www.chalearn.org, identifier: ChaLearn, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.chalearn.org == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.chalearn.org == ega-archive.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.chalearn.org == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.chalearn.org == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.chalearn.org == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.chalearn.org == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.chalearn.org == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.chalearn.org == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier ChaLearn for www.chalearn.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.chalearn.org == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.chalearn.org == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.chalearn.org == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.chalearn.org == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.chalearn.org == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier ChaLearn for www.chalearn.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.chalearn.org == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier ChaLearn for www.chalearn.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.chalearn.org == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.chalearn.org == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.chalearn.org == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.chalearn.org == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.chalearn.org == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier ChaLearn for www.chalearn.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.chalearn.org == doi.org. Repo vals: dict_keys(['dataset_webpage_url_ptr', 'id_pattern', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier ChaLearn for www.chalearn.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.chalearn.org == pdb. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier ChaLearn for www.chalearn.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: www.chalearn.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL www.chalearn.org may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for ChaLearn in www.chalearn.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 6 with repo: www.chalearn.org and accession_id: ChaLearn\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1233 - DEBUG - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository www.chalearn.org unknown in Ontology. Skipping dataset page 6.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: CLARITY\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: www.childrenshospital.org, identifier: CLARITY, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.childrenshospital.org == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.childrenshospital.org == ega-archive.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.childrenshospital.org == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.childrenshospital.org == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.childrenshospital.org == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.childrenshospital.org == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.childrenshospital.org == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.childrenshospital.org == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier CLARITY for www.childrenshospital.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.childrenshospital.org == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.childrenshospital.org == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.childrenshospital.org == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.childrenshospital.org == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.childrenshospital.org == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier CLARITY for www.childrenshospital.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.childrenshospital.org == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier CLARITY for www.childrenshospital.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.childrenshospital.org == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.childrenshospital.org == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.childrenshospital.org == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.childrenshospital.org == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.childrenshospital.org == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier CLARITY for www.childrenshospital.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.childrenshospital.org == doi.org. Repo vals: dict_keys(['dataset_webpage_url_ptr', 'id_pattern', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier CLARITY for www.childrenshospital.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.childrenshospital.org == pdb. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier CLARITY for www.childrenshospital.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: www.childrenshospital.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL www.childrenshospital.org may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for CLARITY in www.childrenshospital.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 7 with repo: www.childrenshospital.org and accession_id: CLARITY\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1233 - DEBUG - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository www.childrenshospital.org unknown in Ontology. Skipping dataset page 7.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: DREAM\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: DREAM, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if synapse.org == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if synapse.org == ega-archive.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if synapse.org == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if synapse.org == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if synapse.org == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if synapse.org == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if synapse.org == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if synapse.org == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier DREAM for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if synapse.org == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if synapse.org == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if synapse.org == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if synapse.org == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if synapse.org == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier DREAM for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if synapse.org == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier DREAM for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if synapse.org == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if synapse.org == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if synapse.org == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if synapse.org == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if synapse.org == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for DREAM in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1085 - WARNING - Identifier DREAM does not match pattern for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 8 with repo: synapse.org and accession_id: DREAM\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1233 - DEBUG - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:DREAM\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 8: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: FlowCAP\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: flowcap.flowsite.org, identifier: FlowCAP, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if flowcap.flowsite.org == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if flowcap.flowsite.org == ega-archive.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if flowcap.flowsite.org == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if flowcap.flowsite.org == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if flowcap.flowsite.org == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if flowcap.flowsite.org == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if flowcap.flowsite.org == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if flowcap.flowsite.org == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier FlowCAP for flowcap.flowsite.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if flowcap.flowsite.org == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if flowcap.flowsite.org == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if flowcap.flowsite.org == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if flowcap.flowsite.org == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if flowcap.flowsite.org == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier FlowCAP for flowcap.flowsite.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if flowcap.flowsite.org == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier FlowCAP for flowcap.flowsite.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if flowcap.flowsite.org == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if flowcap.flowsite.org == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if flowcap.flowsite.org == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if flowcap.flowsite.org == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if flowcap.flowsite.org == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier FlowCAP for flowcap.flowsite.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if flowcap.flowsite.org == doi.org. Repo vals: dict_keys(['dataset_webpage_url_ptr', 'id_pattern', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier FlowCAP for flowcap.flowsite.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if flowcap.flowsite.org == pdb. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier FlowCAP for flowcap.flowsite.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: flowcap.flowsite.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL flowcap.flowsite.org may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for FlowCAP in flowcap.flowsite.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 9 with repo: flowcap.flowsite.org and accession_id: FlowCAP\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1233 - DEBUG - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository flowcap.flowsite.org unknown in Ontology. Skipping dataset page 9.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 10 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: IMPROVER\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: sbvimprover.com, identifier: IMPROVER, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if sbvimprover.com == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if sbvimprover.com == ega-archive.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if sbvimprover.com == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if sbvimprover.com == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if sbvimprover.com == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if sbvimprover.com == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if sbvimprover.com == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if sbvimprover.com == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier IMPROVER for sbvimprover.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if sbvimprover.com == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if sbvimprover.com == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if sbvimprover.com == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if sbvimprover.com == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if sbvimprover.com == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier IMPROVER for sbvimprover.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if sbvimprover.com == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier IMPROVER for sbvimprover.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if sbvimprover.com == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if sbvimprover.com == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if sbvimprover.com == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if sbvimprover.com == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if sbvimprover.com == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier IMPROVER for sbvimprover.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if sbvimprover.com == doi.org. Repo vals: dict_keys(['dataset_webpage_url_ptr', 'id_pattern', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier IMPROVER for sbvimprover.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if sbvimprover.com == pdb. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier IMPROVER for sbvimprover.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: sbvimprover.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL sbvimprover.com may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for IMPROVER in sbvimprover.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 10 with repo: sbvimprover.com and accession_id: IMPROVER\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1233 - DEBUG - Processing dataset 10 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository sbvimprover.com unknown in Ontology. Skipping dataset page 10.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 11 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: Innocentive\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: www.innocentive.com, identifier: Innocentive, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.innocentive.com == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.innocentive.com == ega-archive.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.innocentive.com == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.innocentive.com == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.innocentive.com == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.innocentive.com == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.innocentive.com == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.innocentive.com == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Innocentive for www.innocentive.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.innocentive.com == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.innocentive.com == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.innocentive.com == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.innocentive.com == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.innocentive.com == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier Innocentive for www.innocentive.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.innocentive.com == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Innocentive for www.innocentive.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.innocentive.com == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.innocentive.com == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.innocentive.com == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.innocentive.com == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.innocentive.com == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Innocentive for www.innocentive.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.innocentive.com == doi.org. Repo vals: dict_keys(['dataset_webpage_url_ptr', 'id_pattern', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier Innocentive for www.innocentive.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.innocentive.com == pdb. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier Innocentive for www.innocentive.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: www.innocentive.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL www.innocentive.com may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for Innocentive in www.innocentive.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 11 with repo: www.innocentive.com and accession_id: Innocentive\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1233 - DEBUG - Processing dataset 11 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository www.innocentive.com unknown in Ontology. Skipping dataset page 11.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 12 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: Kaggle\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: www.kaggle.com, identifier: Kaggle, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.kaggle.com == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.kaggle.com == ega-archive.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.kaggle.com == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.kaggle.com == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.kaggle.com == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.kaggle.com == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.kaggle.com == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.kaggle.com == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Kaggle for www.kaggle.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.kaggle.com == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.kaggle.com == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.kaggle.com == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.kaggle.com == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.kaggle.com == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier Kaggle for www.kaggle.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.kaggle.com == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Kaggle for www.kaggle.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.kaggle.com == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.kaggle.com == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.kaggle.com == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.kaggle.com == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.kaggle.com == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Kaggle for www.kaggle.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.kaggle.com == doi.org. Repo vals: dict_keys(['dataset_webpage_url_ptr', 'id_pattern', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier Kaggle for www.kaggle.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.kaggle.com == pdb. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier Kaggle for www.kaggle.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: www.kaggle.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL www.kaggle.com may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for Kaggle in www.kaggle.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 12 with repo: www.kaggle.com and accession_id: Kaggle\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1233 - DEBUG - Processing dataset 12 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository www.kaggle.com unknown in Ontology. Skipping dataset page 12.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 13 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: RGASP\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: www.gencodegenes.org, identifier: RGASP, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.gencodegenes.org == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.gencodegenes.org == ega-archive.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.gencodegenes.org == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.gencodegenes.org == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.gencodegenes.org == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.gencodegenes.org == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.gencodegenes.org == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.gencodegenes.org == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier RGASP for www.gencodegenes.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.gencodegenes.org == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.gencodegenes.org == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.gencodegenes.org == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.gencodegenes.org == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.gencodegenes.org == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier RGASP for www.gencodegenes.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.gencodegenes.org == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier RGASP for www.gencodegenes.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.gencodegenes.org == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.gencodegenes.org == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.gencodegenes.org == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.gencodegenes.org == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.gencodegenes.org == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier RGASP for www.gencodegenes.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.gencodegenes.org == doi.org. Repo vals: dict_keys(['dataset_webpage_url_ptr', 'id_pattern', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier RGASP for www.gencodegenes.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.gencodegenes.org == pdb. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier RGASP for www.gencodegenes.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: www.gencodegenes.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL www.gencodegenes.org may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for RGASP in www.gencodegenes.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 13 with repo: www.gencodegenes.org and accession_id: RGASP\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1233 - DEBUG - Processing dataset 13 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository www.gencodegenes.org unknown in Ontology. Skipping dataset page 13.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 14 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 14: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 15 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: X-Prize\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: www.xprize.org, identifier: X-Prize, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.xprize.org == bigd.big.ac.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.xprize.org == ega-archive.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.xprize.org == massive.ucsd.edu. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.xprize.org == www.ebi.ac.uk. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.xprize.org == www.iprox.cn. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.xprize.org == proteomecentral.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.xprize.org == www.proteomexchange.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'repo_mapping', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.xprize.org == pdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'javascript_load_required', 'url_concat_string', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier X-Prize for www.xprize.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.xprize.org == gdc.cancer.gov. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.xprize.org == cancerimagingarchive.net. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.xprize.org == imaging.datacommons.cancer.gov. Repo vals: dict_keys(['repo_name', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.xprize.org == repository.jpostdb.org. Repo vals: dict_keys(['repo_name', 'url_concat_string', 'download_root', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.xprize.org == geo. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier X-Prize for www.xprize.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.xprize.org == dbgap. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'id_pattern', 'default_id_suffix', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier X-Prize for www.xprize.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.xprize.org == sra. Repo vals: dict_keys(['repo_name', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required', 'informative_html_metadata_tags'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.xprize.org == arrayexpress. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'access_mode', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.xprize.org == nuccore. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.xprize.org == data.mendeley.com. Repo vals: dict_keys(['repo_name'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.xprize.org == synapse.org. Repo vals: dict_keys(['repo_name', 'repo_root', 'dataset_webpage_url_ptr', 'id_pattern', 'access_mode'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier X-Prize for www.xprize.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.xprize.org == doi.org. Repo vals: dict_keys(['dataset_webpage_url_ptr', 'id_pattern', 'javascript_load_required'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier X-Prize for www.xprize.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1131 - DEBUG - Checking if www.xprize.org == pdb. Repo vals: dict_keys(['repo_name', 'id_pattern', 'dataset_webpage_url_ptr'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier X-Prize for www.xprize.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: www.xprize.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL www.xprize.org may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for X-Prize in www.xprize.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 15 with repo: www.xprize.org and accession_id: X-Prize\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1233 - DEBUG - Processing dataset 15 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository www.xprize.org unknown in Ontology. Skipping dataset page 15.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 16 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 16: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1277 - INFO - Updated datasets len: 16\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 414 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 39, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'title', 'file_info',\n",
      "       'description', 'source_section', 'section_class', 'download_link',\n",
      "       'file_extension', 'a_attr_href', 'a_attr_class',\n",
      "       'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 135 - DEBUG - Function_call: extract_file_extension(nan)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 135 - DEBUG - Function_call: extract_file_extension(nan)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 135 - DEBUG - Function_call: extract_file_extension(nan)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 135 - DEBUG - Function_call: extract_file_extension(nan)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 135 - DEBUG - Function_call: extract_file_extension(nan)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 135 - DEBUG - Function_call: extract_file_extension(nan)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 135 - DEBUG - Function_call: extract_file_extension(nan)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 135 - DEBUG - Function_call: extract_file_extension(nan)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 135 - DEBUG - Function_call: extract_file_extension(nan)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 135 - DEBUG - Function_call: extract_file_extension(nan)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 135 - DEBUG - Function_call: extract_file_extension(nan)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 135 - DEBUG - Function_call: extract_file_extension(nan)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 135 - DEBUG - Function_call: extract_file_extension(nan)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 135 - DEBUG - Function_call: extract_file_extension(nan)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 135 - DEBUG - Function_call: extract_file_extension(nan)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 135 - DEBUG - Function_call: extract_file_extension(nan)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 135 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 135 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 135 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 135 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 135 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 135 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 135 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 135 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 135 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 135 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 135 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 135 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 135 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 135 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 135 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 135 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 135 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 135 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 135 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 135 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 135 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 135 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 135 - DEBUG - Function_call: extract_file_extension(None)\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 643 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 470 - INFO - Parsed data extraction completed. Elements collected: 16\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 476 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 486 - INFO - HTML element classification not supported. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 579 - INFO - \n",
      "Progress: 1/1 (100.00%) | Elapsed: 00:00:12 | ETA: 00:00:00\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 584 - DEBUG - Completed processing all URLs.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1055 - INFO - Output written to file: scripts/output/result.csv\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1057 - INFO - File Download Schedule: []\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1059 - DEBUG - DataGatherer run completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1070 - INFO - Quitting the WebDriver.\u001b[0m\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x157867500>: Failed to establish a new connection: [Errno 61] Connection refused')': /session/21089d36-caf0-49c4-96a7-05419468b4d5\n",
      "WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x157843c50>: Failed to establish a new connection: [Errno 61] Connection refused')': /session/21089d36-caf0-49c4-96a7-05419468b4d5\n",
      "WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x157c8ea20>: Failed to establish a new connection: [Errno 61] Connection refused')': /session/21089d36-caf0-49c4-96a7-05419468b4d5\n"
     ]
    }
   ],
   "source": [
    "combined_df = dg.run(input_file=input_file, semantic_retrieval=semantic_retrieval, section_filter=section_filter, prompt_name=prompt)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "9fd1be22241ced6b",
   "metadata": {
    "jupyter": {
     "is_executing": true
    }
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>dataset_identifier</th>\n",
       "      <th>data_repository</th>\n",
       "      <th>dataset_webpage</th>\n",
       "      <th>citation_type</th>\n",
       "      <th>access_mode</th>\n",
       "      <th>link</th>\n",
       "      <th>title</th>\n",
       "      <th>file_info</th>\n",
       "      <th>description</th>\n",
       "      <th>source_section</th>\n",
       "      <th>...</th>\n",
       "      <th>file_extension</th>\n",
       "      <th>a_attr_href</th>\n",
       "      <th>a_attr_class</th>\n",
       "      <th>a_attr_data-ga-action</th>\n",
       "      <th>a_attr_target</th>\n",
       "      <th>a_attr_rel</th>\n",
       "      <th>context_description</th>\n",
       "      <th>source_url</th>\n",
       "      <th>pub_title</th>\n",
       "      <th>raw_data_format</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Assemblathon1&amp;2</td>\n",
       "      <td>assemblathon.org</td>\n",
       "      <td>n/a</td>\n",
       "      <td>n/a</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527</td>\n",
       "      <td>Toward better benchmarking: challenge-based me...</td>\n",
       "      <td>HTML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>CAFA</td>\n",
       "      <td>biofunctionprediction.org</td>\n",
       "      <td>n/a</td>\n",
       "      <td>n/a</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527</td>\n",
       "      <td>Toward better benchmarking: challenge-based me...</td>\n",
       "      <td>HTML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>CAGI</td>\n",
       "      <td>genomeinterpretation.org</td>\n",
       "      <td>n/a</td>\n",
       "      <td>n/a</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527</td>\n",
       "      <td>Toward better benchmarking: challenge-based me...</td>\n",
       "      <td>HTML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>CAPRI</td>\n",
       "      <td>www.ebi.ac.uk</td>\n",
       "      <td>https://www.ebi.ac.uk/pride/archive/projects/C...</td>\n",
       "      <td>n/a</td>\n",
       "      <td>Complex download</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527</td>\n",
       "      <td>Toward better benchmarking: challenge-based me...</td>\n",
       "      <td>HTML</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>CASP</td>\n",
       "      <td>predictioncenter.org</td>\n",
       "      <td>n/a</td>\n",
       "      <td>n/a</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527</td>\n",
       "      <td>Toward better benchmarking: challenge-based me...</td>\n",
       "      <td>HTML</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5 rows × 22 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "  dataset_identifier            data_repository  \\\n",
       "0    Assemblathon1&2           assemblathon.org   \n",
       "1               CAFA  biofunctionprediction.org   \n",
       "2               CAGI   genomeinterpretation.org   \n",
       "3              CAPRI              www.ebi.ac.uk   \n",
       "4               CASP       predictioncenter.org   \n",
       "\n",
       "                                     dataset_webpage citation_type  \\\n",
       "0                                                n/a           n/a   \n",
       "1                                                n/a           n/a   \n",
       "2                                                n/a           n/a   \n",
       "3  https://www.ebi.ac.uk/pride/archive/projects/C...           n/a   \n",
       "4                                                n/a           n/a   \n",
       "\n",
       "        access_mode link title file_info description source_section  ...  \\\n",
       "0               NaN  NaN   NaN       NaN         NaN            NaN  ...   \n",
       "1               NaN  NaN   NaN       NaN         NaN            NaN  ...   \n",
       "2               NaN  NaN   NaN       NaN         NaN            NaN  ...   \n",
       "3  Complex download  NaN   NaN       NaN         NaN            NaN  ...   \n",
       "4               NaN  NaN   NaN       NaN         NaN            NaN  ...   \n",
       "\n",
       "  file_extension a_attr_href a_attr_class a_attr_data-ga-action a_attr_target  \\\n",
       "0           None         NaN          NaN                   NaN           NaN   \n",
       "1           None         NaN          NaN                   NaN           NaN   \n",
       "2           None         NaN          NaN                   NaN           NaN   \n",
       "3           None         NaN          NaN                   NaN           NaN   \n",
       "4           None         NaN          NaN                   NaN           NaN   \n",
       "\n",
       "  a_attr_rel context_description  \\\n",
       "0        NaN                 NaN   \n",
       "1        NaN                 NaN   \n",
       "2        NaN                 NaN   \n",
       "3        NaN                 NaN   \n",
       "4        NaN                 NaN   \n",
       "\n",
       "                                         source_url  \\\n",
       "0  https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527   \n",
       "1  https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527   \n",
       "2  https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527   \n",
       "3  https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527   \n",
       "4  https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527   \n",
       "\n",
       "                                           pub_title raw_data_format  \n",
       "0  Toward better benchmarking: challenge-based me...            HTML  \n",
       "1  Toward better benchmarking: challenge-based me...            HTML  \n",
       "2  Toward better benchmarking: challenge-based me...            HTML  \n",
       "3  Toward better benchmarking: challenge-based me...            HTML  \n",
       "4  Toward better benchmarking: challenge-based me...            HTML  \n",
       "\n",
       "[5 rows x 22 columns]"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "combined_df.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "305d7d5f299551c1",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-09-04T21:20:38.136808Z",
     "start_time": "2025-09-04T21:20:38.134052Z"
    }
   },
   "outputs": [],
   "source": [
    "#ground_truth = pd.read_parquet(\"scripts/exp_input/dataset_citation_records_Table.parquet\")\n",
    "ground_truth = df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "dadaf8654a9d103d",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-09-04T21:20:38.857028Z",
     "start_time": "2025-09-04T21:20:38.851074Z"
    }
   },
   "outputs": [],
   "source": [
    "ground_truth.head()\n",
    "# rename colum synid to 'identifier'\n",
    "ground_truth = ground_truth.rename(columns={'synid': 'identifier'})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "583f880224783dc7",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-09-04T21:26:52.754959Z",
     "start_time": "2025-09-04T21:26:52.722141Z"
    }
   },
   "outputs": [],
   "source": [
    "def evaluate_performance(predict_df, ground_truth, orchestrator, false_positives_file, false_negatives_file=None):\n",
    "    \"\"\" Evaluates dataset extraction performance using precision, recall, and F1-score. \"\"\"\n",
    "\n",
    "    recall_list, false_positives_output, false_negatives_output = [], [], []\n",
    "    total_precision, total_recall, num_sources = 0, 0, 0\n",
    "\n",
    "    for source_page in predict_df['source_url'].unique():\n",
    "        pub_id = source_page.split('/')[-1].lower()\n",
    "        \n",
    "        orchestrator.logger.info(f\"Evaluating pub_id: {pub_id}\")\n",
    "        gt_data = ground_truth[ground_truth['pmcid'].str.lower() == pub_id.lower()]  # extract ground truth\n",
    "\n",
    "        gt_datasets = set()\n",
    "        for dataset_string in gt_data['identifier'].dropna().str.lower():\n",
    "            gt_datasets.update(dataset_string.split(','))  # Convert CSV string into set of IDs\n",
    "\n",
    "        orchestrator.logger.info(f\"# of elements in gt_data: {len(gt_data)}. Element IDs: {gt_datasets}\")\n",
    "\n",
    "        num_sources += 1\n",
    "\n",
    "        # Extract evaluation datasets for this source page\n",
    "        eval_data = predict_df[predict_df['source_url'] == source_page]\n",
    "        eval_datasets = set(eval_data['dataset_identifier'].dropna().str.lower())\n",
    "        # Remove invalid entries\n",
    "        eval_datasets.discard('n/a')\n",
    "        eval_datasets.discard('')\n",
    "\n",
    "        orchestrator.logger.info(f\"Evaluation datasets: {eval_datasets}\")\n",
    "\n",
    "        # Handle cases where both ground truth and evaluation are empty\n",
    "        if not gt_datasets and not eval_datasets:\n",
    "            orchestrator.logger.info(\"No datasets in both ground truth and evaluation. Perfect precision and recall.\")\n",
    "            total_precision += 1\n",
    "            total_recall += 1\n",
    "            continue\n",
    "\n",
    "        # Match Extraction Logic\n",
    "        matched_gt, matched_eval = set(), set()\n",
    "\n",
    "        # Exact Matches\n",
    "        exact_matches = gt_datasets & eval_datasets  # Intersection of ground truth and extracted datasets\n",
    "        matched_gt.update(exact_matches)\n",
    "        matched_eval.update(exact_matches)\n",
    "\n",
    "        # Partial Matches (Aliased Identifiers)\n",
    "        for eval_id in eval_datasets - matched_eval:\n",
    "            for gt_id in gt_datasets - matched_gt:\n",
    "                if eval_id in gt_id or gt_id in eval_id:  # Partial match or alias\n",
    "                    orchestrator.logger.info(f\"Partial or alias match found: eval_id={eval_id}, gt_id={gt_id}\")\n",
    "                    matched_gt.add(gt_id)\n",
    "                    matched_eval.add(eval_id)\n",
    "                    break  # Stop once matched\n",
    "\n",
    "        # **False Positives (Unmatched extracted datasets)**\n",
    "        FP = eval_datasets - matched_eval\n",
    "        false_positives_output.extend([false_p, pub_id] for false_p in FP)\n",
    "\n",
    "        # **False Negatives (Unmatched ground truth datasets)**\n",
    "        FN = gt_datasets - matched_gt\n",
    "        false_negatives_output.extend((FN, pub_id)) if len(FN) > 0 else None\n",
    "\n",
    "        # **Precision and Recall Calculation**\n",
    "        true_positives = len(matched_gt)\n",
    "        false_positives = len(FP)\n",
    "        false_negatives = len(FN)\n",
    "\n",
    "        precision = true_positives / (true_positives + false_positives) if (true_positives + false_positives) > 0 else 0\n",
    "        recall = true_positives / (true_positives + false_negatives) if (true_positives + false_negatives) > 0 else 0\n",
    "\n",
    "        orchestrator.logger.info(f\"Precision for {source_page}: {precision}\")\n",
    "        orchestrator.logger.info(f\"Recall for {source_page}: {recall}\")\n",
    "\n",
    "        if recall == 0:\n",
    "            recall_list.append(source_page)\n",
    "\n",
    "        # Accumulate totals\n",
    "        total_precision += precision\n",
    "        total_recall += recall\n",
    "\n",
    "    # **Compute Overall Metrics**\n",
    "    average_precision = total_precision / num_sources if num_sources > 0 else 0\n",
    "    average_recall = total_recall / num_sources if num_sources > 0 else 0\n",
    "    f1_score = (\n",
    "        2 * (average_precision * average_recall) / (average_precision + average_recall)\n",
    "        if (average_precision + average_recall) > 0\n",
    "        else 0\n",
    "    )\n",
    "\n",
    "    orchestrator.logger.info(f\"\\nPerformance evaluation completed for {num_sources} source pages.\")\n",
    "\n",
    "    # **Save false positives**\n",
    "    with open(false_positives_file, 'w') as f:\n",
    "        for item in false_positives_output:\n",
    "            f.write(\"%s\\n\" % item)\n",
    "\n",
    "    if false_negatives_file:\n",
    "        with open(false_negatives_file, 'w') as f:\n",
    "            for item in false_negatives_output:\n",
    "                f.write(\"%s\\n\" % item)\n",
    "\n",
    "    return {\n",
    "        \"average_precision\": average_precision,\n",
    "        \"average_recall\": average_recall,\n",
    "        \"f1_score\": f1_score\n",
    "    }"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "d5bd9405ee5dcd60",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-09-04T21:26:53.669418Z",
     "start_time": "2025-09-04T21:26:53.666829Z"
    }
   },
   "outputs": [],
   "source": [
    "#results = evaluate_performance(out, ground_truth, dg, \"output/false_positives.txt\")\n",
    "# PMC4318527, "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "id": "3ae367545bb1b42b",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-09-04T21:26:55.530950Z",
     "start_time": "2025-09-04T21:26:54.622805Z"
    }
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4318527\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn1720047', 'syn2455683'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'x-prize', 'rgasp', 'kaggle', 'capri', 'innocentive', 'chalearn', 'casp', 'dream', 'cafa', 'clarity', 'cagi', 'improver', 'flowcap', 'assemblathon1&2'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 89 - INFO - \n",
      "Performance evaluation completed for 1 source pages.\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "results = evaluate_performance(combined_df, ground_truth, dg, \"scripts/output/false_positives.txt\", \"scripts/output/false_negatives.txt\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "fff3dc3075cb13b5",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-09-04T21:25:01.310798Z",
     "start_time": "2025-09-04T21:25:01.304645Z"
    }
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'average_precision': 0.0, 'average_recall': 0.0, 'f1_score': 0}"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "results"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "88837a62c359df60",
   "metadata": {
    "jupyter": {
     "is_executing": true
    }
   },
   "outputs": [],
   "source": [
    "#combined_df.to_csv(\"output/SAGE_GPT_FDR_FewShot_Synapse_output.csv\", index=False)\n",
    "#\"https://pmc.ncbi.nlm.nih.gov/articles/PMC11240079\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC5752539\",\"https://pmc.ncbi.nlm.nih.gov/articles/PMC7032692\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10547713\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8102856\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8055881\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8226229\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10769298\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC11659981\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8082263\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8565335\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC9094742\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC6289083\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC7658217\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC9780309\",\"https://pmc.ncbi.nlm.nih.gov/articles/PMC3788619\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10680627\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8131595\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC11420198\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC9915613\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10238095\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452\" , \"https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10836119\",  \"https://pmc.ncbi.nlm.nih.gov/articles/PMC9280291\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC11661334\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC5161470\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "54baf5d5dcfee95e",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-09-04T20:50:38.300941Z",
     "start_time": "2025-09-04T20:50:25.797543Z"
    }
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11792374\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 4. Element IDs: {'syn9702085', 'syn2580853', 'syn8456637', 'syn52293417'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn9702085', 'syn2580853', 'syn8456637', 'syn52293417'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4318527\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn1720047', 'syn2455683'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn1720047', 'syn2455683', 'syn312572'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527: 0.6666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4339277\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 4. Element IDs: {'syn1701558', 'syn1732077', 'syn1682511', 'syn1688309'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn1701558', 'syn1732077', 'syn1682511', 'syn1688309'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8452114\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn18485175'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn18485175'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8452114: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8452114: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9641692\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn3219045'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse107011', 'gse119117', 'gse84133', 'gse65133', 'gse87544', 'gse115449', 'syn3219045', 'gse67835'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9641692: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9641692: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc5161470\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1446295'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn1446295'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC5161470: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC5161470: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9925742\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21126450'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'srr21079269', 'gse224398', 'srr21079270', 'syn21126450', 'srr21079267', 'srr21079265', 'srr21079266', 'srr21079268'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9925742: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9925742: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9235070\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn30243143'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn213309', 'gse75688', 'gse81861', 'https://gdc-portal.ncbi.nih.gov', 'e-mtab-6149', 'syn30243143', 'gse70630', 'gse76312', 'gse81383', 'gse103322'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9235070: 0.1\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9235070: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11661334\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn51364943'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gcst90027158', 'hippie', 'gcst007511', 'gcst009325', 'syn51364943'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11661334: 0.2\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11661334: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11929800\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 5. Element IDs: {'syn3388564', 'syn11724057', 'syn17015098', 'syn4896408', 'syn18485175'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn3388564', 'syn11724057', 'gcst90027158', 'syn17015098', 'syn4896408', 'gse291402'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11929800: 0.6666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11929800: 0.8\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7113080\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn17394041'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn17394041'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8847344\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn25813713'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn25813713'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8847344: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8847344: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9280291\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn5612563'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse27131', 'gse95233', 'gse40586', 'e-mexp-3589', 'gse60244', 'gse25504', '10.7303/syn5612563', 'gse21802', 'gse13015', 'gse54514', 'gse40012', 'gse32707', 'gse65682', 'e-mtab-5273/5274', 'gse66099', 'gse57065', 'e-mtab-1548', 'gse20346'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn5612563, gt_id=syn5612563\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9280291: 0.05555555555555555\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9280291: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10029021\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn50897018'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn50897018.3'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=syn50897018.3, gt_id=syn50897018\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10029021: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10029021: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11525589\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn51514106'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'brats-men', '10.7303/syn51514106'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn51514106, gt_id=syn51514106\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11525589: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11525589: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10836119\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 6. Element IDs: {'syn3157238', 'syn9705278', 'syn3205821', 'syn5592519', 'syn3157329', 'syn3157322'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'r01ag36042', 'r01ag17917', 'rc2ag036547', 'r01ag48015', 'u01ag32984', 'r01ag15819', 'syn3205821.6', 'rf1ag57473', 'u01ag46152', 'r01ag30146', 'syn3157329', 'syn3157322', 'r01ag046171', 'u01ag61356', 'u01ag46161', 'syn9705278', 'r01ag36836', '3u01ag024904-09s4', 'syn3157238', 'n01-ag-12100', 'hhsn271201200022c', 'rf1ag051550', 'syn5592519', 'w81xwh-12-2-0012', 'p30ag10161'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=syn3205821.6, gt_id=syn3205821\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10836119: 0.24\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10836119: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6646046\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn5609493'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gm12878', 'chembl'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6646046: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6646046: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10387117\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21041850'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'lmex0000004397', 'gse149563', 'gse122332', 'gsm4504966', 'gse135851', 'gse122960', 'gse178362', 'gsm4035472', 'gse134174', 'egas00001004082', 'gsm5388412', 'dataset-427f1eee6dd44f50bae1ab13f0f3c6a9', 'gsm5388413', 'gse136832', 'gse171524', 'syn21041850', 'gse161382', 'gsm5388411', 'lmex0000004396', 'gse135893', 'gsm4504967'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10387117: 0.047619047619047616\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10387117: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4444046\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn1729383', 'syn1710680', 'syn1734155'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn1729383', 'syn1710680', 'pb002581', 'syn1734155'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4444046: 0.75\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4444046: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10329279\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21478998'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: set()\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279: 0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11145390\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21788425'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'entropy_functions.r', 'clean_060720.rdata', 'gse147807', 'clean_nodatasets_060720.rdata'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11145390: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11145390: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11032436\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2759792'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn2759792', 'nds00143'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9984184\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn39456106', 'syn25576878'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse147205', 'syn39456106', 'gse169185', 'syn25576878'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9984184: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9984184: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7347884\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1757053'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn1757053', 'e-mtab-4145', 'egas00000000083', 'gse56031'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7347884: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7347884: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10802452\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2759792'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'nimh', 'tba', 'geo'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8907344\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn26134616'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn26134616', '10.7303/syn26134616'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8907344: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8907344: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10238095\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2623706'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse104645', 'gse37892', 'https://www.synapse.org/#!synapse:syn2623706', 'gse135222', 'gse39582', 'gse13067', 'gse103479', 'gse17536', 'gse33113', 'gse13294', 'gse108989', 'prjeb23709', 'gse78220', 'gse38832', 'gse14333'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=https://www.synapse.org/#!synapse:syn2623706, gt_id=syn2623706\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10238095: 0.06666666666666667\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10238095: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9710693\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 4. Element IDs: {'syn23554292', 'syn3505720', 'syn3219045', 'syn3191087'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse75748', 'e-mtab-5061', 'syn23554292', 'syn3505720', 'syn3191087', '10.7303/syn3219045', 'gse72056', 'e-mtab-5060'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn3219045, gt_id=syn3219045\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9710693: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9710693: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9915613\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1688369'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse56066', 'syn1688369', 'gse99542', 'gse102441', 'gse62944', 'gse154873', 'gse96517', 'gse78913', 'gse25710', 'gse84515', 'gse73663', 'gse78167', 'gse57436', 'gse51403'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9915613: 0.07142857142857142\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9915613: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11420198\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn26164834', 'syn4921369'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse59119', 'syn4921369', 'gse126957', 'gse249687', 'gse84176', 'gse142881', 'gse96107'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11420198: 0.14285714285714285\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11420198: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7865022\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20681023'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.7303/syn20681023', '10.6084/m9.figshare.13342055'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn20681023, gt_id=syn20681023\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7865022: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7865022: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8131595\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn4587609', 'syn18485175'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse84245', 'phs001373.v2.p2', 'gse59318', 'gse150130', 'gse60192', 'gse113673', 'gse65776', 'encsr677hxc', 'e-mtab-9045', 'encsr822cea', 'srx3381923', 'syn4587609', 'gse59630', 'encsr198zyj', 'gse96107', 'syn18485175'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8131595: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8131595: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10828232\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn13363290'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse214628', 'syn13363290'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10828232: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10828232: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11852220\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn51365301'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn51365301', 'r9'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11852220: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11852220: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9314356\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4907804'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'adni.loni.usc.edu', 'https://www.niagads.org', 'http://adni.loni.usc.edu', 'https://www.synapse.org/#!synapse:syn4907804'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=https://www.synapse.org/#!synapse:syn4907804, gt_id=syn4907804\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9314356: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9314356: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11439699\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 10. Element IDs: {'syn22264775', 'syn10235592', 'syn2580853', 'syn3157325', 'syn9702085', 'syn4896408', 'syn20068543', 'syn3157275', 'syn26720676', 'syn17008935'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn22264775', 'syn10235592', 'syn2580853', '10.7303/syn3157325', 'https://adknowledgeportal.synapse.org/explore/studies/detailspage/studydetails?study=syn22264775', 'https://adknowledgeportal.synapse.org/explore/studies/detailspage/studydetails?study=syn9702085', 'syn338856', 'syn4896408', 'syn20068543', 'syn3157275', 'syn26720676', 'syn17008935'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn3157325, gt_id=syn3157325\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=https://adknowledgeportal.synapse.org/explore/studies/detailspage/studydetails?study=syn9702085, gt_id=syn9702085\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11439699: 0.8333333333333334\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11439699: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6323985\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn6131484'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'hotspot2', 'encode', 'sra', 'uberon', 'hocomoco', 'geo', 'trim-adapters-illumina', 'cellosaurus', 'cpics'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10680627\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn52264240', 'syn52234396'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse225157', 'encsr443vtk', 'syn52234396', 'syn52264240', 'encsr998ydi'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10680627: 0.4\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10680627: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc5192600\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn3049712', 'syn2787209'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn3049712', 'syn2787209'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC5192600: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC5192600: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7752919\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2623706'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn2623706', 'gse18088', 'gse44076', 'tcga-read', 'gse39582', 'e-mtab-8152', 'gse33113', 'tcga-coad', 'gse13294', 'gse2109', 'gse148394', 'gse38832', 'gse14333'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7752919: 0.07692307692307693\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7752919: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8859891\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4224222'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn4224222'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9853518\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn3193805'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn3193805', '3757476'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9853518: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9853518: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11852190\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn3193805'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn3193805'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11852190: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11852190: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10652415\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn3388564', 'syn3157322', 'syn26254632'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn3388564', 'syn3157322'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10652415: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10652415: 0.6666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7842189\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2580853'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse1432', 'https://adknowledgeportal.synapse.org/explore/studies/detailspage?study=syn3157182', 'gse75431', 'phs002197.v1.p1', 'ng00040', 'gse146866', 'gse13162', 'gse121654', '10.7303/syn2580853', 'gse57181', 'gse90696', 'https://www.synapse.org/#!synapse:syn18485175', 'https://repo-prod.prod.sagebase.org/repo/v1/doi/locate?id=syn3163039&type=entity', 'gse98401', 'gse98969', 'gse55627', 'gse61499', 'gse61500', 'gse138852', 'gse65067', 'gse103334', 'https://www.synapse.org/#!synapse:syn9884351', 'https://www.synapse.org/#!synapse:syn7818788'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn2580853, gt_id=syn2580853\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7842189: 0.043478260869565216\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7842189: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc3788619\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn1729383', 'syn2237931'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn1729383', 'syn2237931'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC3788619: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC3788619: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6750256\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4587609'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'http://dx.doi.org/10.7303/syn4587609', 'syn4587609'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6750256: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6750256: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11370573\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn25569736'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn25569736'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11370573: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11370573: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4344205\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1688369'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'doi:10.7303/syn1688369'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=doi:10.7303/syn1688369, gt_id=syn1688369\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4344205: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4344205: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9630780\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 8. Element IDs: {'syn34542450', 'syn25792906', 'syn25876213', 'syn20843407', 'syn34542574', 'syn25886762', 'syn34539904', 'syn34542449'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn34542450', 'syn25792906', 'syn25876213', 'ijo1366', 'syn20843407', 'syn34542574', 'syn25886762', 'syn34539904', 'syn34542449', '10.7303/syn20843407'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9630780: 0.8\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9630780: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10813059\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn31947506', 'syn31952297', 'syn52138250'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn31947506', 'syn31952297', 'syn52138250'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10813059: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10813059: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6089972\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn2812961', 'syn7343873', 'syn7214402'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse35069', 'syn7343873', 'prjeb5468', 'gse38266', 'gse68379', '10.5281/zenodo.1304766', 'gse38270', 'syn7214402', '10.5281/zenodo.1298968', 'prjeb11844', 'prjna258216', 'gse74877', 'syn2812961'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6089972: 0.23076923076923078\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6089972: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8130821\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 4. Element IDs: {'syn17015098', 'syn9884314', 'syn20803583', 'syn23191787'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn17015098', 'syn9884314', 'syn20803583', 'syn23191787'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8130821: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8130821: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11224609\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn10901595'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.5281/zenodo.11240547', 'syn10901595', 'https://github.com/joonan-lab/cwas', '10.5281/zenodo.10795678'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11224609: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11224609: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7577712\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20821165'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'pxd009589', 'syn20821165', 'gse33000'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7577712: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7577712: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7257173\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21541022'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn21541022'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7257173: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7257173: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9780309\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn26560886'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'26', '10.7303/syn26560886'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=26, gt_id=syn26560886\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9780309: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9780309: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7658217\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21041850'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse132355', 'c4077b3c-5c98-4d26-a614-246d12c2e5d7', 'gse122960', 'syn21041850', '10.1101/742320', '10.2139/ssrn.3582067'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7658217: 0.16666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7658217: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6289083\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn11967914'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'egas00001003068'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6289083: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6289083: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8807579\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20808171'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse36980', 'gse158233', 'syn20808171', 'prjeb24858', 'gse149149', 'gse164466', 'gse151999', 'gse156167'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8807579: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8807579: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6700281\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2759792'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'phs000424.v7.p1', 'https://datadryad.org/resource/doi:10.5061/dryad.5d4s2r8', 'https://www.synapse.org/#!synapse:syn2759792/wiki/69613'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=https://www.synapse.org/#!synapse:syn2759792/wiki/69613, gt_id=syn2759792\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6700281: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6700281: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7029360\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn8717496'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn8717496'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8219045\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn3163039'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'e-mtab-5061', 'gse64810', 'syn3163039', 'gse97942'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8219045: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8219045: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10543041\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn50996324'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn50996324'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10543041: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10543041: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6546784\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4921369'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse77565', 'phs001190.v1.p1'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6546784: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6546784: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10696419\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn22255433'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn22255433'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10696419: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10696419: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc5387919\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 10. Element IDs: {'syn5522788', 'syn5004874', 'syn4984604', 'syn5556216', 'syn5522790', 'syn6022474', 'syn5492805', 'syn6022465', 'syn6035832', 'syn5493036'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn5522788', 'syn5004874', 'syn4984604', 'syn5556216', 'syn5522790'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC5387919: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC5387919: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8094983\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn23532550'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'epi_isl_461058', 'epi_isl_461002', 'epi_isl_461059', 'epi_isl_461053', 'epi_isl_415464', 'epi_isl_461054', 'epi_isl_461056', 'epi_isl_461055', 'epi_isl_461060', 'epi_isl_461052', 'syn23532550', 'epi_isl_461051', 'epi_isl_461003', 'epi_isl_461057', 'epi_isl_461001'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8094983: 0.06666666666666667\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8094983: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7962955\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20640275'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn20640275', 'gse102470', 'e-mtab-9905', 'gse57763', 'nc_045512.2', 'mn996532.1', 'gse137834', 'gse33584'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7962955: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7962955: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11601401\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn20933797', 'syn51150434', 'syn62110225'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.7303/syn62110225', 'syn20933797', 'syn51150434', '10.58085/hr6s-2991'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn62110225, gt_id=syn62110225\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11601401: 0.75\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11601401: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9712627\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn12514624'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse183068', 'gse148822', 'gse178265', 'syn12514624', 'gse174367', 'gse174332', 'gse167494', 'gse137444', 'gse160936'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9712627: 0.1111111111111111\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9712627: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4697768\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2813581'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'https://www.synapse.org/#!synapse:syn2813581'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=https://www.synapse.org/#!synapse:syn2813581, gt_id=syn2813581\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4697768: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4697768: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9094742\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21125841'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse775330', 'gse129788', 'gse60361', 'gse116470', 'pxd029727', 'human-archived-brain', 'pxd020007', 'gse97930', 'gse118257', 'gse140511', 'gse130119', 'srp135960', 'syn21125841', 'gse113973', 'gse138852'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9094742: 0.06666666666666667\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9094742: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8565335\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn6131484'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse115046'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8565335: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8565335: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10985232\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn10003873'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'https://github.com/star-fusion', 'https://github.com/suhrig/arriba', 'https://github.com/shenlab-genomics/anchored-fusion', 'https://github.com/fusioninspector/fusioninspector', '10.5281/zenodo.10677267', 'sra:srp158590', 'sra:srp186687', 'synapse:syn10003873', 'https://github.com/xidslab/scfusion', 'sra:srp291312', 'https://github.com/ndaniel/fusioncatcher'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=synapse:syn10003873, gt_id=syn10003873\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10985232: 0.09090909090909091\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10985232: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8784774\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20968331'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn20968331', 'gse186341', 'https://www.doi.org/10.7303/syn20968331'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8784774: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8784774: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8082263\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn11801497'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'https://api.gdc.cancer.gov/data/1a7d7be8-675d-4e60-a105-19d4121bdebf', 'https://gdc.cancer.gov/about-data/publications/ccg-aim-2020', 'https://gdc.cancer.gov/about-data/publications/pancanatlas', 'https://www.synapse.org/#!synapse:syn11801497', 'https://api.gdc.cancer.gov/data/1b5f413e-a8d1-4d10-92eb-7c4ae739ed81', 'http://api.gdc.cancer.gov/data/7d4c0344-f018-4ab0-949a-09815f483480', 'https://api.gdc.cancer.gov/data/d773a53e-b78d-4a20-848c-866616352328', 'http://api.gdc.cancer.gov/data/4f277128-f793-4354-a13d-30cc7fe9f6b5'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=https://www.synapse.org/#!synapse:syn11801497, gt_id=syn11801497\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8082263: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8082263: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11659981\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21041850'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse115469', 'e-mtab-6653', 'tcga-luad', 'e-mtab-6149', 'gse134432', 'scp498', 'syn21041850', 'gse162170'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11659981: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11659981: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9356164\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn12169715', 'syn12169709'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'tcga', 'syn12169715', 'syn12169709'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9356164: 0.6666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9356164: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11271596\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn53009474'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.7303/syn53009474'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn53009474, gt_id=syn53009474\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11271596: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11271596: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7520714\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn18344812', 'syn17009517'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.6084/m9.figshare.8159525.v1', '8159525.v1', 'syn17009517'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7520714: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7520714: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10769298\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn11724057'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'rosmap'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10769298: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10769298: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8226229\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2701943'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'https://portal.gdc.cancer.gov', 'syn2701943', 'https://figshare.com/articles/dataset/njhncc-rnaseq-stemsig_txt/14540610', 'gse30784'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8226229: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8226229: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11716409\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4231880'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse16476', 'gse89413', 'gpl18573', '10.13026/c28g6p', '10.7303/syn4231880', 'gse79209', '10.13026/c2f305', 'e-mtab-8248'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn4231880, gt_id=syn4231880\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11716409: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11716409: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8055881\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn7824274'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'https://cptac-data-portal.georgetown.edu/study-summary/s053', 'https://cptac-data-portal.georgetown.edu/study-summary/s037', 'https://cptac-data-portal.georgetown.edu/study-summary/s050', 'https://github.com/ding-lab/hotpho_analysis', 'https://cptac-data-portal.georgetown.edu/study-summary/s038', 'https://github.com/ding-lab/hotspot3d', 'syn7824274', 'https://cptac-data-portal.georgetown.edu/study-summary/s039'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8055881: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8055881: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8102856\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn7214402'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'tcga', 'syn7214402', 'cnp0001438'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8102856: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8102856: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9085578\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn8612097'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn8612097'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9085578: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9085578: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9840277\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn23569889', 'syn14237651', 'syn11914606'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn23569889', 'syn14237651', 'syn11914606'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9840277: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9840277: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10547713\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn26707219', 'syn52137895'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.7303/syn26707219'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn26707219, gt_id=syn26707219\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10547713: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10547713: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10571258\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn30821562'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'https://github.com/davidrconnell/igraph-se2', 'https://www.synapse.org/#!synapse:syn30821562', '10.5281/zenodo.8322854', 'https://github.com/aghasemian/communityfitnet', 'https://github.com/mkrzak/benchmarking_clustering_methods_scrnaseq'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=https://www.synapse.org/#!synapse:syn30821562, gt_id=syn30821562\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10571258: 0.2\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10571258: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6829597\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn18796452'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn18796452', 'https://www.ncbi.nlm.nih.gov/nuccore/r25005', 'https://www.ncbi.nlm.nih.gov/protein/q32852'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6829597: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6829597: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11240079\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn52566791'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn52566791'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11240079: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11240079: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7805663\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn11737848'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn11737848'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7805663: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7805663: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10600818\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn23627957'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn23627957'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10600818: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10600818: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc5752539\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1741407'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse63459', 'gse63805', 'gse29623'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC5752539: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC5752539: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6602064\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn7818788', 'syn9884357'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse90696', 'syn7818788', 'syn9884357'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6602064: 0.6666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6602064: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7032692\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1710250'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn1710250', 'egas00000000083'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7032692: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7032692: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10061294\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn2468297', 'syn1715755'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.7303/syn2468297', 'syn1715755'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn2468297, gt_id=syn2468297\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10061294: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10061294: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8006362\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn22024464'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.15154/1504248', 'syn22024464', '792'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8006362: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8006362: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7778917\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn18360482'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'pharos'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7778917: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7778917: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 89 - INFO - \n",
      "Performance evaluation completed for 100 source pages.\u001b[0m\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "{'average_precision': 0.48885651088042414,\n",
       " 'average_recall': 0.8646666666666667,\n",
       " 'f1_score': 0.6245891267370887}"
      ]
     },
     "execution_count": 16,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "check_df = pd.read_csv(\"scripts/output/syn_data_results/result_first try.csv\")\n",
    "ret = evaluate_performance(check_df, ground_truth, dg, \"scripts/output/false_positives.txt\")\n",
    "ret"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "49f712fb5db57045",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-09-04T20:51:39.708798Z",
     "start_time": "2025-09-04T20:51:29.136798Z"
    }
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11792374\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 4. Element IDs: {'syn9702085', 'syn2580853', 'syn8456637', 'syn52293417'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'m27188', 'syn2580853', 'm27187', 'syn52293417', 'syn9702085', 'syn8456637'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374: 0.6666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4318527\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn1720047', 'syn2455683'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn2455683', 'x-prize', 'syn1720047', 'rgasp', 'kaggle', 'capri', 'innocentive', 'chalearn', 'casp', 'dream', 'cafa', 'clarity', 'cagi', 'improver', 'flowcap', 'assemblathon1&2'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4339277\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 4. Element IDs: {'syn1701558', 'syn1732077', 'syn1682511', 'syn1688309'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn1732077', 'syn300013', 'syn1701558', 'syn1682511', 'syn1688309'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277: 0.8\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8452114\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn18485175'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn18485175'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8452114: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8452114: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9641692\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn3219045'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse107011', 'gse119117', 'gse84133', 'gse65133', 'gse87544', 'gse115449', 'syn3219045', 'gse67835'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9641692: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9641692: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc5161470\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1446295'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn1446295'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC5161470: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC5161470: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9925742\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21126450'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'srr21079269', 'gse224398', 'srr21079270', 'syn21126450', 'srr21079267', 'srr21079265', 'srr21079266', 'srr21079268'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9925742: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9925742: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9235070\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn30243143'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn213309', 'gse75688', 'gse81861', 'syn30243143', 'gse70630', 'gse76312', 'gse81383', 'gse103322'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9235070: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9235070: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11661334\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn51364943'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gcst90027158', 'cowas', 'gcst007511', 'gcst009325', 'syn51364943'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11661334: 0.2\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11661334: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11929800\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 5. Element IDs: {'syn3388564', 'syn11724057', 'syn17015098', 'syn4896408', 'syn18485175'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn3388564', 'syn11724057', 'gcst90027158', 'syn17015098', 'syn4896408', 'gse291402'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11929800: 0.6666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11929800: 0.8\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7113080\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn17394041'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn17394041'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8847344\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn25813713'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn25813713'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8847344: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8847344: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9280291\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn5612563'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse27131', 'gse95233', 'gse40586', 'e-mexp-3589', 'gse60244', 'gse25504', 'syn5612563', 'gse21802', 'gse13015', 'gse54514', 'gse40012', 'gse32707', 'gse65682', 'e-mtab-5273/5274', 'gse66099', 'gse57065', 'e-mtab-1548', 'gse20346'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9280291: 0.05555555555555555\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9280291: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10029021\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn50897018'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn50897018.3'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=syn50897018.3, gt_id=syn50897018\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10029021: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10029021: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11525589\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn51514106'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'brats-men', '10.7303/syn51514106'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn51514106, gt_id=syn51514106\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11525589: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11525589: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10836119\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 6. Element IDs: {'syn3157238', 'syn9705278', 'syn3205821', 'syn5592519', 'syn3157329', 'syn3157322'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'r01ag046171', 'u01ag61356', '10.7303/syn9705278', 'r01ag17917', 'r01ag15819', 'n01-ag-12100', 'hhsn271201200022c', 'rf1ag051550', '3u01ag024904-09s4', '10.7303/syn3157322', 'u01ag46152', '10.7303/syn3205821.6', '10.7303/syn5592519', 'w81xwh-12-2-0012', '10.7303/syn3157329', '10.7303/syn3157238', 'p30ag10161'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn3205821.6, gt_id=syn3205821\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn5592519, gt_id=syn5592519\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn3157329, gt_id=syn3157329\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn3157238, gt_id=syn3157238\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn9705278, gt_id=syn9705278\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn3157322, gt_id=syn3157322\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10836119: 0.35294117647058826\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10836119: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6646046\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn5609493'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn5609493', 'e-mtab-2323'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6646046: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6646046: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10387117\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21041850'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'dataset-427f1eee6dd44f50bae1ab13f0f3c6a9', 'gse161382', 'lmex0000004397', 'gse149563', 'lmex0000004396', 'gse135851', 'gse122960', 'gse136832', 'gse171524', 'gse122332', 'gse178362', 'syn21041850', 'gse135893', 'gse134174', 'egas00001004082'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10387117: 0.06666666666666667\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10387117: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4444046\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn1729383', 'syn1710680', 'syn1734155'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn1729383', 'syn1710680', 'pb002581', 'syn1734155'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4444046: 0.75\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4444046: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10329279\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21478998'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'pmc7805732', 'pmc6921374', 'pmc6422329', 'pmc6166867', 'pmc7786238', 'pmc6939339', 'pmc6720753', 'pmc5789045', 'pmc3135413', 'pmc3766928'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11145390\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21788425'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn21788425', '10.5281/zenodo.10971926', 'gse147807'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11145390: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11145390: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11032436\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2759792'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn2759792'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9984184\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn39456106', 'syn25576878'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse147205', 'syn39456106', 'gse169185', 'syn25576878'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9984184: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9984184: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7347884\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1757053'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn1757053', 'e-mtab-4145', 'egas00000000083', 'gse56031'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7347884: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7347884: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10802452\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2759792'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'tba'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8907344\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn26134616'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn26134616', '10.7303/syn26134616'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8907344: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8907344: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10238095\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2623706'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse104645', 'gse37892', 'tcga', 'gse135222', 'gse39582', 'gse13067', 'gse103479', 'gse17536', 'gse33113', 'gse13294', 'gse108989', 'prjeb23709', 'gse35144', 'gse78220', 'gse38832', 'gse14333'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10238095: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10238095: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9710693\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 4. Element IDs: {'syn23554292', 'syn3505720', 'syn3219045', 'syn3191087'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse75748', 'e-mtab-5061', 'syn23554292', 'syn3505720', 'syn3191087', '10.7303/syn3219045', 'gse72056', 'e-mtab-5060'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn3219045, gt_id=syn3219045\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9710693: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9710693: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9915613\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1688369'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse56066', 'syn1688369', 'gse99542', 'gse102441', 'gse62944', 'gse154873', 'gse96517', 'gse78913', 'gse25710', 'gse84515', 'gse73663', 'gse78167', 'gse57436', 'gse51403'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9915613: 0.07142857142857142\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9915613: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11420198\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn26164834', 'syn4921369'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse59119', 'syn4921369', 'gse126957', 'gse249687', 'gse84176', '10.1073/', 'gse142881', 'gse96107'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11420198: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11420198: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7865022\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20681023'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.7303/syn20681023', '10.6084/m9.figshare.13342055'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn20681023, gt_id=syn20681023\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7865022: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7865022: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8131595\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn4587609', 'syn18485175'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse84245', 'phs001373.v2.p2', 'gse59318', 'gse150130', 'gse60192', 'gse113673', 'gse65776', 'encsr677hxc', 'e-mtab-9045', 'encsr822cea', 'srx3381923', 'syn4587609', 'gse59630', 'encsr198zyj', 'gse96107', 'syn18485175'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8131595: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8131595: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10828232\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn13363290'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse214628', 'syn13363290'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10828232: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10828232: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11852220\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn51365301'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn51365301'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11852220: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11852220: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9314356\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4907804'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'adni.loni.usc.edu', 'syn4907804', 'www.niagads.org'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9314356: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9314356: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11439699\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 10. Element IDs: {'syn22264775', 'syn10235592', 'syn2580853', 'syn3157325', 'syn9702085', 'syn4896408', 'syn20068543', 'syn3157275', 'syn26720676', 'syn17008935'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn22264775', 'syn10235592', 'syn2580853', '10.7303/syn3157325', 'syn9702085', 'syn338856', 'syn4896408', 'syn20068543', 'syn3157275', 'syn26720676', 'syn17008935'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn3157325, gt_id=syn3157325\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11439699: 0.9090909090909091\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11439699: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6323985\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn6131484'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn6131484', '10.1093/'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10680627\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn52264240', 'syn52234396'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse225157', 'encsr443vtk', 'syn52234396', 'syn52264240', 'encsr998ydi'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10680627: 0.4\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10680627: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc5192600\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn3049712', 'syn2787209'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn3049712', 'syn2787209'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC5192600: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC5192600: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7752919\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2623706'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn2623706', 'gse18088', 'gse44076', 'tcga-read', 'gse39582', 'e-mtab-8152', 'gse33113', 'tcga-coad', 'gse13294', 'gse2109', 'gse148394', 'gse38832', 'gse14333'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7752919: 0.07692307692307693\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7752919: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8859891\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4224222'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn4224222'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9853518\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn3193805'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn3193805', '3757476'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9853518: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9853518: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11852190\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn3193805'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn3193805'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11852190: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11852190: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10652415\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn3388564', 'syn3157322', 'syn26254632'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn3388564', 'syn3157322', 'syn26254632'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10652415: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10652415: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7842189\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2580853'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse1432', 'syn2580853', 'syn3163039', 'gse75431', 'phs002197.v1.p1', 'syn3157182', 'gse146866', 'syn7818788', 'gse13162', 'gse57181', 'gse90696', 'gse98401', 'gse98969', 'gse55627', 'gse61499', 'gse61500', 'syn18485175', 'gse138852', 'gse65067', 'gse103334', 'syn9884351'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7842189: 0.047619047619047616\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7842189: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc3788619\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn1729383', 'syn2237931'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn1729383', 'syn2237931'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC3788619: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC3788619: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6750256\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4587609'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn4587609'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6750256: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6750256: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11370573\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn25569736'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn25569736'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11370573: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11370573: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4344205\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1688369'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'doi:10.7303/syn1688369'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=doi:10.7303/syn1688369, gt_id=syn1688369\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4344205: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4344205: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9630780\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 8. Element IDs: {'syn34542450', 'syn25792906', 'syn25876213', 'syn20843407', 'syn34542574', 'syn25886762', 'syn34539904', 'syn34542449'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn34542450', 'syn25792906', 'syn25876213', 'syn20843407', 'syn34542574', 'syn25886762', 'syn34539904', 'syn34542449', '10.7303/syn20843407'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9630780: 0.8888888888888888\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9630780: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10813059\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn31947506', 'syn31952297', 'syn52138250'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn31947506', 'syn31952297', 'syn52138250'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10813059: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10813059: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6089972\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn2812961', 'syn7343873', 'syn7214402'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse35069', 'syn7343873', 'prjeb5468', 'gse38266', 'gse68379', '10.5281/zenodo.1304766', 'gse38270', 'syn7214402', '10.5281/zenodo.1298968', 'prjeb11844', 'prjna258216', 'gse74877', 'syn2812961'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6089972: 0.23076923076923078\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6089972: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8130821\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 4. Element IDs: {'syn17015098', 'syn9884314', 'syn20803583', 'syn23191787'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn17015098', 'syn9884314', 'syn20803583', 'syn23191787'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8130821: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8130821: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11224609\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn10901595'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.5281/', 'syn10901595'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11224609: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11224609: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7577712\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20821165'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'pxd009589', 'syn20821165', 'gse33000'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7577712: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7577712: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7257173\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21541022'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn21541022'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7257173: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7257173: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9780309\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn26560886'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn26560886'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9780309: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9780309: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7658217\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21041850'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.1101/', 'gse132355', 'gse122960', 'syn21041850', '10.2139/'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7658217: 0.2\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7658217: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6289083\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn11967914'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'egas00001003068'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6289083: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6289083: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8807579\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20808171'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse36980', 'gse158233', 'syn20808171', 'prjeb24858', 'gse149149', 'gse164466', 'gse151999', 'gse156167'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8807579: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8807579: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6700281\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2759792'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'phs000424.v7.p1', 'syn2759792', 'doi:10.5061/dryad.5d4s2r8'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6700281: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6700281: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7029360\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn8717496'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: set()\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360: 0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8219045\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn3163039'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'e-mtab-5061', 'gse64810', 'syn3163039', 'gse97942'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8219045: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8219045: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10543041\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn50996324'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.13020/', 'syn50996324'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10543041: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10543041: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6546784\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4921369'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse77565', 'syn4921369', 'phs001190.v1.p1'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6546784: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6546784: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10696419\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn22255433'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn22255433'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10696419: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10696419: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc5387919\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 10. Element IDs: {'syn5522788', 'syn5004874', 'syn4984604', 'syn5556216', 'syn5522790', 'syn6022474', 'syn5492805', 'syn6022465', 'syn6035832', 'syn5493036'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn5522788', 'syn5004874', 'syn4984604', 'syn5556216', 'syn5522790', 'syn6022474', 'syn5492805', 'syn6022465', 'syn6035832', 'syn5493036'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC5387919: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC5387919: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8094983\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn23532550'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'epi_isl_461058', 'epi_isl_461002', 'epi_isl_461059', 'epi_isl_461053', 'epi_isl_415464', 'epi_isl_461054', 'epi_isl_461056', 'epi_isl_461055', 'epi_isl_461060', 'epi_isl_461052', 'syn23532550', 'epi_isl_461051', 'epi_isl_461003', 'epi_isl_461057', 'epi_isl_461001'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8094983: 0.06666666666666667\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8094983: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7962955\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20640275'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn20640275', 'gse102470', 'e-mtab-9905', 'gse57763', 'nc_045512.2', 'mn996532.1', 'gse137834', 'gse33584'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7962955: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7962955: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11601401\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn20933797', 'syn51150434', 'syn62110225'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.7303/syn62110225', 'syn20933797', 'syn51150434', '10.58085/hr6s-2991'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn62110225, gt_id=syn62110225\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11601401: 0.75\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11601401: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9712627\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn12514624'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse183068', 'gse148822', 'gse178265', 'gse174367', 'gse174332', 'gse167494', 'gse137444', 'gse160936'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9712627: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9712627: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4697768\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2813581'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn2813581'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4697768: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4697768: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9094742\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21125841'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse775330', 'gse129788', 'gse60361', 'gse116470', 'pxd029727', 'human-archived-brain', 'pxd020007', 'gse97930', 'gse118257', 'gse140511', 'gse130119', 'srp135960', 'syn21125841', 'gse113973', 'gse138852'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9094742: 0.06666666666666667\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9094742: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8565335\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn6131484'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'encsr000e', 'sra', 'encsr483rkn', 'gse115046', 'encsr000ekp', 'encsr220asc'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8565335: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8565335: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10985232\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn10003873'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn10003873', 'nm_001306068.3', 'srp291312', 'srp158590', 'nm_133330.3', 'srp186687', '10.5281/'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10985232: 0.14285714285714285\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10985232: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8784774\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20968331'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn20968331', 'gse186341'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8784774: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8784774: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8082263\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn11801497'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn11801497'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8082263: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8082263: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11659981\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21041850'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse115469', 'e-mtab-6653', 'e-mtab-6149', 'gse134432', 'scp498', 'syn21041850', 'gse162170'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11659981: 0.14285714285714285\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11659981: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9356164\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn12169715', 'syn12169709'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'3586c0da-64d0-4b74-a449-5ff4d9136611', 'syn12169715', 'syn12169709'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9356164: 0.6666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9356164: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11271596\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn53009474'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'voc-als.git', '10.7303/syn53009474'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn53009474, gt_id=syn53009474\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11271596: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11271596: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7520714\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn18344812', 'syn17009517'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'8159525.v1', 'syn17009517'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7520714: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7520714: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10769298\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn11724057'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'38187779', 'pmc10769298'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10769298: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10769298: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8226229\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2701943'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn2701943', 'gse30784'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8226229: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8226229: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11716409\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4231880'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse16476', 'gse89413', 'medquad', 'mit-bih', 'mimic-iii', 'gse79209', 'e-mtab-8248'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11716409: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11716409: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8055881\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn7824274'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn7824274'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8055881: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8055881: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8102856\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn7214402'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'tcga', 'syn7214402', 'cnp0001438'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8102856: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8102856: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9085578\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn8612097'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn8612097'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9085578: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9085578: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9840277\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn23569889', 'syn14237651', 'syn11914606'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn23569889', 'syn14237651', 'syn11914606'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9840277: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9840277: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10547713\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn26707219', 'syn52137895'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.7303/syn26707219'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn26707219, gt_id=syn26707219\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10547713: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10547713: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10571258\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn30821562'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.5281/', 'syn30821562', 'github.com/cogdishion/se2', '10.5281/zenodo.8338532'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10571258: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10571258: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6829597\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn18796452'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn18796452', '10.1084/', 'r25005'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6829597: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6829597: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11240079\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn52566791'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn52566791'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11240079: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11240079: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7805663\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn11737848'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn11737848'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7805663: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7805663: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10600818\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn23627957'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn23627957'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10600818: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10600818: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc5752539\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1741407'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse63459', 'gse63805', 'gse29623'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC5752539: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC5752539: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6602064\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn7818788', 'syn9884357'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse90696', 'syn7818788', 'syn9884357'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6602064: 0.6666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6602064: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7032692\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1710250'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn1710250', 'egas00000000083', 'egac00001000484'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7032692: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7032692: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10061294\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn2468297', 'syn1715755'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.7303/syn2468297', 'syn1715755'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn2468297, gt_id=syn2468297\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10061294: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10061294: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8006362\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn22024464'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.5281/zenodo.4321679', '10.15154/1504248', 'syn22024464', '792'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8006362: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8006362: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7778917\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn18360482'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn18360482', 'pharos'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7778917: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7778917: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 89 - INFO - \n",
      "Performance evaluation completed for 100 source pages.\u001b[0m\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "{'average_precision': 0.5255226407579351,\n",
       " 'average_recall': 0.883,\n",
       " 'f1_score': 0.6588981651577225}"
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "check_df = pd.read_csv(\"scripts/output/syn_data_results/result_schema_validation.csv\")\n",
    "ret = evaluate_performance(check_df, ground_truth, dg, \"scripts/output/false_positives.txt\")\n",
    "ret"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "id": "5768c91b8f925301",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-09-04T20:51:49.942643Z",
     "start_time": "2025-09-04T20:51:39.748779Z"
    }
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11792374\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 4. Element IDs: {'syn9702085', 'syn2580853', 'syn8456637', 'syn52293417'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'m27188', 'syn2580853', 'm27187', 'syn52293417', 'syn9702085', 'syn8456637'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374: 0.6666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4318527\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn1720047', 'syn2455683'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn2455683', 'x-prize', 'syn1720047', 'rgasp', 'kaggle', 'capri', 'innocentive', 'chalearn', 'casp', 'dream', 'cafa', 'clarity', 'cagi', 'improver', 'flowcap', 'assemblathon1&2'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4339277\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 4. Element IDs: {'syn1701558', 'syn1732077', 'syn1682511', 'syn1688309'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn1732077'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8452114\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn18485175'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn18485175'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8452114: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8452114: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9641692\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn3219045'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse107011', 'gse119117', 'gse84133', 'gse65133', 'gse87544', 'gse115449', 'syn3219045', 'gse67835'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9641692: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9641692: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc5161470\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1446295'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'enm014', 'enr112', 'chr22', '1kgp', 'syn1446295', 'enr113', 'enm013'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC5161470: 0.14285714285714285\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC5161470: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9925742\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21126450'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'srr21079269', 'gse224398', 'srr21079270', 'syn21126450', 'srr21079267', 'srr21079265', 'srr21079266', 'srr21079268'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9925742: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9925742: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9235070\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn30243143'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse75688', 'gse81861', 'e-mtab-6149', 'syn30243143', 'gse70630', 'gse76312', 'gse81383', 'gse103322'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9235070: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9235070: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11661334\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn51364943'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gcst90027158', 'gcst009325', 'syn51364943', 'gcst007511'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11661334: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11661334: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11929800\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 5. Element IDs: {'syn3388564', 'syn11724057', 'syn17015098', 'syn4896408', 'syn18485175'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn3388564', 'syn11724057', 'gcst90027158', 'syn17015098', 'syn4896408', 'gse291402', 'syn18485175'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11929800: 0.7142857142857143\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11929800: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7113080\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn17394041'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn17394041'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8847344\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn25813713'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn25813713'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8847344: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8847344: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9280291\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn5612563'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse27131', 'gse95233', 'gse40586', 'e-mexp-3589', 'gse60244', 'gse25504', '10.7303/syn5612563', 'gse21802', 'gse13015', 'gse54514', 'gse40012', 'gse32707', 'gse65682', 'e-mtab-5273/5274', 'gse66099', 'gse57065', 'e-mtab-1548', 'gse20346'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn5612563, gt_id=syn5612563\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9280291: 0.05555555555555555\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9280291: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10029021\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn50897018'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn50897018.3'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=syn50897018.3, gt_id=syn50897018\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10029021: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10029021: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11525589\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn51514106'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn51514106', '10.7303/syn51514106'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11525589: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11525589: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10836119\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 6. Element IDs: {'syn3157238', 'syn9705278', 'syn3205821', 'syn5592519', 'syn3157329', 'syn3157322'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'r01ag36042', 'r01ag17917', '10.7303/syn3205821.6', '10.7303/syn3157329', 'rc2ag036547', 'r01ag48015', 'u01ag32984', 'r01ag15819', 'rf1ag57473', 'u01ag46152', 'r01ag30146', 'r01ag046171', 'u01ag61356', 'u01ag46161', 'r01ag36836', '3u01ag024904-09s4', '10.7303/syn5592519', '10.7303/syn3157238', '10.7303/syn9705278', 'n01-ag-12100', 'hhsn271201200022c', 'rf1ag051550', '10.7303/syn3157322', 'w81xwh-12-2-0012', 'p30ag10161'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn3205821.6, gt_id=syn3205821\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn3157329, gt_id=syn3157329\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn5592519, gt_id=syn5592519\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn3157238, gt_id=syn3157238\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn9705278, gt_id=syn9705278\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn3157322, gt_id=syn3157322\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10836119: 0.24\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10836119: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6646046\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn5609493'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn5609493'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6646046: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6646046: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10387117\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21041850'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'cellref', 'lmex0000004397', 'gse149563', 'gse122332', 'gsm4504966', 'gse135851', 'gse122960', 'gsm4035472', 'gse178360', 'gse134174', 'egas00001004082', 'gsm5388412', 'dataset-427f1eee6dd44f50bae1ab13f0f3c6a9', 'gsm5388413', 'gse136832', 'gse171524', 'syn21041850', 'gse161382', 'gsm5388411', 'lmex0000004396', 'gse135893', 'gsm4504967'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10387117: 0.045454545454545456\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10387117: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4444046\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn1729383', 'syn1710680', 'syn1734155'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn1729383', 'syn1710680', 'syn1734155'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4444046: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4444046: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10329279\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21478998'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: set()\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279: 0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11145390\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21788425'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn21788425', '10.5281/zenodo.10971926', 'gse147807'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11145390: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11145390: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11032436\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2759792'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'crl-3216', 'syn2759792', 'nds00143'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9984184\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn39456106', 'syn25576878'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse147205', 'gse95587', 'gse169185', 'syn39456106', 'syn25576878', 'gse15222'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9984184: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9984184: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7347884\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1757053'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn1757053', 'e-mtab-4145', 'egas00000000083', 'gse56031'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7347884: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7347884: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10802452\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2759792'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn28580053'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8907344\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn26134616'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'doi.org/10.7303/syn26134616', 'syn26134616'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8907344: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8907344: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10238095\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2623706'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn2623706', 'gse37892', 'gse104645', 'gse135222', 'gse39582', 'gse13067', 'gse103479', 'gse17536', 'gse33113', 'gse13294', 'gse108989', 'prjeb23709', 'gse35144', 'gse78220', 'gse38832', 'gse14333'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10238095: 0.0625\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10238095: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9710693\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 4. Element IDs: {'syn23554292', 'syn3505720', 'syn3219045', 'syn3191087'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse75748', 'e-mtab-5061', 'syn23554292', 'syn3505720', 'syn3191087', 'syn3219045', 'gse72056', 'e-mtab-5060'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9710693: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9710693: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9915613\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1688369'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse56066', 'syn1688369', 'gse99542', 'gse102441', 'gse73663', 'gse62944', 'gse154873', 'gse96517', 'gse25710', 'gse84515', 'gse78913', 'gse78167', 'gse57436', 'gse51403'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9915613: 0.07142857142857142\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9915613: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11420198\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn26164834', 'syn4921369'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse59119', 'syn4921369', 'gse126957', 'gse249687', 'gse84176', '10.1073/', 'gse142881', 'gse96107'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11420198: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11420198: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7865022\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20681023'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.7303/syn20681023', '10.6084/m9.figshare.13342055'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn20681023, gt_id=syn20681023\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7865022: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7865022: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8131595\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn4587609', 'syn18485175'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse84245', 'phs001373.v2.p2', 'gse59318', 'gse150130', 'gse60192', 'gse113673', 'gse65776', 'encsr677hxc', 'e-mtab-9045', 'encsr822cea', 'gse81912', 'srx3381923', 'syn4587609', '10.1371/', 'gse59630', 'encsr198zyj', 'gse96107', 'syn18485175'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8131595: 0.1111111111111111\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8131595: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10828232\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn13363290'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'nm_021272.3', 'gse214628', 'syn13363290-v49'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=syn13363290-v49, gt_id=syn13363290\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10828232: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10828232: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11852220\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn51365301'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn51365301'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11852220: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11852220: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9314356\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4907804'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'adni', 'niagads', 'syn4907804'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9314356: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9314356: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11439699\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 10. Element IDs: {'syn22264775', 'syn10235592', 'syn2580853', 'syn3157325', 'syn9702085', 'syn4896408', 'syn20068543', 'syn3157275', 'syn26720676', 'syn17008935'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn22264775', 'syn2580853', '10.7303/syn3157325', 'syn9702085', 'syn338856', 'syn20068543', 'syn26720676', 'syn17008935'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn3157325, gt_id=syn3157325\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11439699: 0.875\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11439699: 0.7\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6323985\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn6131484'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'sra', 'encode', 'geo'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10680627\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn52264240', 'syn52234396'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse225157', 'encsr443vtk', 'syn52234396', 'syn52264240', 'encsr998ydi'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10680627: 0.4\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10680627: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc5192600\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn3049712', 'syn2787209'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn3049712', 'syn2787209'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC5192600: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC5192600: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7752919\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2623706'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn2623706', 'gse18088', 'gse44076', 'tcga-read', 'gse39582', 'e-mtab-8152', 'gse33113', 'tcga-coad', 'gse13294', 'gse2109', 'gse148394', 'gse38832', 'gse14333'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7752919: 0.07692307692307693\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7752919: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8859891\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4224222'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn4224222'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9853518\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn3193805'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn3193805'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9853518: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9853518: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11852190\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn3193805'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn3193805', 'brats'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11852190: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11852190: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10652415\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn3388564', 'syn3157322', 'syn26254632'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn3388564', 'syn3157322', 'syn26254632'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10652415: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10652415: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7842189\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2580853'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse1432', 'syn2580853', 'syn3163039', 'gse75431', 'phs002197.v1.p1', 'ng00040', 'syn3157182', 'gse146866', 'syn7818788', 'gse13162', 'gse121654', 'gse57181', 'gse90696', 'gse98401', 'gse98969', 'gse55627', 'gse61499', 'gse61500', 'syn18485175', 'gse138852', 'gse65067', 'gse103334', 'syn9884351'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7842189: 0.043478260869565216\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7842189: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc3788619\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn1729383', 'syn2237931'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn1729383', 'syn2237931'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC3788619: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC3788619: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6750256\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4587609'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn4587609'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6750256: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6750256: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11370573\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn25569736'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn25569736'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11370573: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11370573: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4344205\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1688369'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'doi:10.7303/syn1688369'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=doi:10.7303/syn1688369, gt_id=syn1688369\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4344205: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4344205: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9630780\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 8. Element IDs: {'syn34542450', 'syn25792906', 'syn25876213', 'syn20843407', 'syn34542574', 'syn25886762', 'syn34539904', 'syn34542449'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn34542450', 'syn25792906', 'syn25876213', 'ijo1366', 'syn20843407', 'syn34542574', 'syn25886762', 'syn34539904', 'syn34542449', '10.7303/syn20843407'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9630780: 0.8\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9630780: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10813059\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn31947506', 'syn31952297', 'syn52138250'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn31947506', 'syn31952297', 'syn52138250'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10813059: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10813059: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6089972\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn2812961', 'syn7343873', 'syn7214402'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse35069', 'syn7343873', 'prjeb5468', 'doi.org/10.5281/zenodo.1304766', 'gse68379', 'gse38270', 'syn7214402', 'prjeb11844', 'prjna258216', 'gse74877', 'doi.org/10.5281/zenodo.1298968', 'syn2812961'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6089972: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6089972: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8130821\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 4. Element IDs: {'syn17015098', 'syn9884314', 'syn20803583', 'syn23191787'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn17015098', 'syn9884314', 'syn20803583', 'syn23191787'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8130821: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8130821: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11224609\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn10901595'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.5281/zenodo.11240547', 'syn10901595', '10.5281/zenodo.10795678'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11224609: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11224609: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7577712\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20821165'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'pxd009589', 'syn20821165', 'gse33000'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7577712: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7577712: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7257173\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21541022'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn21541022'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7257173: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7257173: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9780309\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn26560886'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn26560886'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9780309: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9780309: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7658217\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21041850'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.1101/', 'gse132355', 'c4077b3c-5c98-4d26-a614-246d12c2e5d7', 'gse122960', 'syn21041850', '10.2139/'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7658217: 0.16666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7658217: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6289083\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn11967914'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'egas00001003068'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6289083: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6289083: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8807579\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20808171'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse36980', 'gse158233', 'syn20808171', 'prjeb24858', 'gse149149', 'gse164466', 'gse151999', 'gse156167'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8807579: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8807579: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6700281\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2759792'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'phs000424.v7.p1', '10.5061/', 'syn2759792'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6700281: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6700281: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7029360\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn8717496'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'pmc7029360', 'syn8717496'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8219045\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn3163039'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'e-mtab-5061', 'gse64810', 'syn3163039', 'gse97942'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8219045: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8219045: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10543041\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn50996324'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.13020/', 'syn50996324'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10543041: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10543041: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6546784\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4921369'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse77565', 'syn4921369', 'phs001190.v1.p1'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6546784: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6546784: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10696419\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn22255433'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn22255433'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10696419: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10696419: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc5387919\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 10. Element IDs: {'syn5522788', 'syn5004874', 'syn4984604', 'syn5556216', 'syn5522790', 'syn6022474', 'syn5492805', 'syn6022465', 'syn6035832', 'syn5493036'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn5522788', 'syn5004874', 'syn4984604', 'syn5556216', 'syn5522790', 'syn6022474', 'syn5492805', 'syn6022465', 'syn6035832', 'syn5493036'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC5387919: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC5387919: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8094983\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn23532550'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'epi_isl_461058', 'epi_isl_461002', 'epi_isl_461059', 'epi_isl_461053', 'epi_isl_415464', 'epi_isl_461054', 'epi_isl_461056', 'epi_isl_461055', 'epi_isl_461060', 'epi_isl_461052', 'syn23532550', 'epi_isl_461051', 'epi_isl_461003', 'epi_isl_461057', 'epi_isl_461001'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8094983: 0.06666666666666667\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8094983: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7962955\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20640275'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn20640275', 'gse102470', 'e-mtab-9905', 'gse57763', 'nc_045512.2', 'mn996532.1', 'gse137834', 'gse33584'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7962955: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7962955: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11601401\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn20933797', 'syn51150434', 'syn62110225'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.7303/syn62110225', 'syn20933797', 'syn51150434', '10.58085/hr6s-2991'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn62110225, gt_id=syn62110225\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11601401: 0.75\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11601401: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9712627\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn12514624'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse183068', 'gse148822', 'gse178265', 'syn12514624', 'gse174367', 'gse174332', 'gse167494', 'gse137444', 'gse160936'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9712627: 0.1111111111111111\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9712627: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4697768\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2813581'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn2813581'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4697768: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4697768: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9094742\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21125841'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse775330', 'gse129788', 'gse60361', 'gse116470', 'pxd029727', 'human-archived-brain', 'pxd020007', 'gse97930', 'gse118257', 'gse140511', 'gse130119', 'srp135960', 'syn21125841', 'gse113973', 'gse138852'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9094742: 0.06666666666666667\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9094742: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8565335\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn6131484'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse115046'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8565335: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8565335: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10985232\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn10003873'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'sra:srp158590', 'sra:srp186687', 'synapse:syn10003873', '10.5281/', 'sra:srp291312'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=synapse:syn10003873, gt_id=syn10003873\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10985232: 0.2\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10985232: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8784774\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20968331'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn20968331', 'gse186341'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8784774: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8784774: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8082263\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn11801497'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn11801497'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8082263: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8082263: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11659981\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21041850'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'e-mtab-6653', '10.5281/zenodo.12092871', 'e-mtab-6149', 'gse134432', '10.5524/102624', 'syn21041850', 'gse162170'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11659981: 0.14285714285714285\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11659981: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9356164\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn12169715', 'syn12169709'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'tcga', 'syn12169715', 'syn12169709'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9356164: 0.6666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9356164: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11271596\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn53009474'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn53009474', '10.7303/syn53009474'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11271596: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11271596: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7520714\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn18344812', 'syn17009517'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.6084/m9.figshare.8159525.v1', '10.7303/syn18344812', 'syn17009517'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn18344812, gt_id=syn18344812\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7520714: 0.6666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7520714: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10769298\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn11724057'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'38187779', 'syn11724057', 'pmc10769298'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10769298: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10769298: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8226229\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2701943'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'tcga', 'syn2701943', 'gse30784'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8226229: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8226229: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11716409\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4231880'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse16476', 'gse89413', 'gpl18573', 'gse79209', 'e-mtab-8248'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11716409: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11716409: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8055881\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn7824274'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn7824274'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8055881: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8055881: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8102856\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn7214402'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'mc3', 'syn7214402', 'cnp0001438'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8102856: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8102856: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9085578\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn8612097'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn8612097'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9085578: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9085578: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9840277\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn23569889', 'syn14237651', 'syn11914606'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn23569889', 'syn14237651', 'syn11914606'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9840277: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9840277: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10547713\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn26707219', 'syn52137895'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.7303/syn26707219'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn26707219, gt_id=syn26707219\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10547713: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10547713: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10571258\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn30821562'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'se2', '10.5281/', 'syn30821562', 'igraph-se2'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10571258: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10571258: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6829597\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn18796452'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn18796452'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6829597: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6829597: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11240079\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn52566791'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn52566791'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11240079: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11240079: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7805663\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn11737848'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn11737848'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7805663: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7805663: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10600818\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn23627957'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn23627957'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10600818: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10600818: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc5752539\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1741407'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse63459', 'gse63805', 'gse29623'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC5752539: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC5752539: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6602064\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn7818788', 'syn9884357'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse90696', 'syn7818788', 'syn9884357'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6602064: 0.6666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6602064: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7032692\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1710250'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn1710250', 'egas00000000083', 'egac00001000484'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7032692: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7032692: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10061294\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn2468297', 'syn1715755'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.7303/syn2468297', 'syn1715755'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn2468297, gt_id=syn2468297\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10061294: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10061294: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8006362\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn22024464'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.5281/zenodo.4321679', '10.15154/1504248', 'syn22024464', 'ftp://ftp-trace.ncbi.nlm.nih.gov/1000genomes/ftp/1000g_2504_high_coverage'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8006362: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8006362: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7778917\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn18360482'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn18360482'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7778917: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7778917: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 89 - INFO - \n",
      "Performance evaluation completed for 100 source pages.\u001b[0m\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "{'average_precision': 0.5301756223245355,\n",
       " 'average_recall': 0.9095,\n",
       " 'f1_score': 0.6698657961931753}"
      ]
     },
     "execution_count": 18,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "check_df = pd.read_csv(\"scripts/output/syn_data_results/result_Prompt_enhancement.csv\")\n",
    "ret = evaluate_performance(check_df, ground_truth, dg, \"scripts/output/false_positives.txt\")\n",
    "ret"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "id": "3c757d7d13496047",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11792374\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 4. Element IDs: {'syn9702085', 'syn2580853', 'syn8456637', 'syn52293417'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'m27188', 'syn2580853', 'm27187', 'syn52293417', 'syn9702085', 'syn8456637'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374: 0.6666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4318527\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn1720047', 'syn2455683'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn2455683', 'syn312572'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4339277\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 4. Element IDs: {'syn1701558', 'syn1732077', 'syn1682511', 'syn1688309'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn1701558', 'syn1732077', 'syn1682511', 'syn1688309'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8452114\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn18485175'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn18485175'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8452114: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8452114: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9641692\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn3219045'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse107011', 'gse119117', 'gse84133', 'gse65133', 'gse87544', 'gse115449', 'syn3219045', 'gse67835'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9641692: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9641692: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc5161470\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1446295'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'chr22', '1000gp_phase3', 'pone.0167994.s003.xls', 'syn1446295', 'pone.0167994.s009.xls', 'enr113', 'enm013', 'pone.0167994.s002.xls', 'enr112', 'pone.0167994.s008.xls', 'pone.0167994.s001.pdf', 'pone.0167994.s004.pdf', 'pone.0167994.s007.xls', 'pone.0167994.s010.pdf', 'tagsnp', 'pone.0167994.s005.pdf', 'enm014', 'pone.0167994.s006.pdf', '20130502'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC5161470: 0.05263157894736842\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC5161470: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9925742\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21126450'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'srr21079269', 'gse224398', 'srr21079270', 'syn21126450', 'srr21079267', 'srr21079265', 'srr21079266', 'srr21079268'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9925742: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9925742: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9235070\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn30243143'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn213309', 'gse75688', 'gse81861', 'e-mtab-6149', 'syn30243143', 'gse70630', 'gse76312', 'gse81383', 'gse103322'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9235070: 0.1111111111111111\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9235070: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11661334\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn51364943'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gcst90027158', '35107', 'gcst007511', 'gcst009325', 'syn51364943'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11661334: 0.2\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11661334: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11929800\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 5. Element IDs: {'syn3388564', 'syn11724057', 'syn17015098', 'syn4896408', 'syn18485175'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn3388564', 'syn11724057', 'gcst90027158', 'syn17015098', '10.5281/zenodo.14804357', 'syn4896408', 'gse291402'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11929800: 0.5714285714285714\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11929800: 0.8\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7113080\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn17394041'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn17394041'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8847344\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn25813713'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'nct00982111', 'nct00427713', 'nct01673867', 'nct01584830', 'nct01057810', 'syn25813713', 'nct01377376', 'nct00973609', 'nct00601900', 'nct02130466', 'nct01229813', 'nct00110162'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8847344: 0.08333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8847344: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9280291\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn5612563'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse40586', 'e-mtab-5273', 'gse13015', 'gse54514', 'gse40012', 'gse65682', 'gse27131', 'e-mexp-3589', 'syn5612563', 'e-mtab-5274', 'gse25504', 'gse66099', 'gse20346', 'gse32707', 'gse95233', 'gse60244', 'gse21802', 'gse57065', 'e-mtab-1548'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9280291: 0.05263157894736842\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9280291: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10029021\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn50897018'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn50897018.3'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=syn50897018.3, gt_id=syn50897018\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10029021: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10029021: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11525589\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn51514106'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn51514106'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11525589: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11525589: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10836119\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 6. Element IDs: {'syn3157238', 'syn9705278', 'syn3205821', 'syn5592519', 'syn3157329', 'syn3157322'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn3157238', 'syn9705278', 'syn3205821.6', 'syn5592519', 'syn3157329', 'syn3157322'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=syn3205821.6, gt_id=syn3205821\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10836119: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10836119: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6646046\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn5609493'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'e-mtab-2323', 'syn5609493'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6646046: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6646046: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10387117\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21041850'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'lmex0000004397', 'gse149563', 'gse122332', 'gsm4504966', 'gse135851', 'gse122960', 'gse178362', 'gse136831', 'gsm4035472', 'gse134174', 'egas00001004082', 'gsm5388412', 'dataset-427f1eee6dd44f50bae1ab13f0f3c6a9', 'gsm5388413', 'gse171524', 'syn21041850', 'gse161382', 'gsm5388411', 'lmex0000004396', 'gse135893', 'gsm4504967', '10.5281/zenodo.8111545'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10387117: 0.045454545454545456\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10387117: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4444046\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn1729383', 'syn1710680', 'syn1734155'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn1729383', 'syn1710680', 'pb002581', 'syn1734155'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4444046: 0.75\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4444046: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10329279\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21478998'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn21478998'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11145390\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21788425'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn21788425', '10.5281/zenodo.10971926', 'gse147807'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11145390: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11145390: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11032436\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2759792'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn2759792', 'phs000424', 'phs000648', 'nds00143'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9984184\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn39456106', 'syn25576878'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse147205', 'e-geod-44770', 'gse95587', 'gouveiaroque_chipmera', 'gse169185', 'syn39456106', 'syn25576878', 'gse15222'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9984184: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9984184: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7347884\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1757053'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn1757053', 'e-mtab-4145', 'egas00000000083', 'gse56031'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7347884: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7347884: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10802452\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2759792'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: set()\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452: 0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8907344\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn26134616'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn26134616'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8907344: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8907344: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10238095\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2623706'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'swh:1:rev:5adf0c170407f30df33e15faf19c1091af9f752f', 'swh:1:rev:c8b87c46eab6c4fc2a528c136ce62e51b9ad71af', 'gse14333', 'qupath-0.3.2', 'nct-crc-he-100k', 'gse78220', 'gse38832', 'gse135222', '10.1038/nature25501', '6917', 'pre_clinical_model', 'human-colorectal-cancer-11-mm-capture-area-ffpe-2-standard', 'swh:1:rev:8de317a40f03e7272fd6031e12663595d595fa5f', 'gse39582', 'nct-crc-he-7k', 'gse17536', 'gse13294', 'cccrc_classifier', 'tcga-crc', 'syn2623706', 'ccle_broad_2019', 'gse104645', 'gse37892', 'cinmetrics', 'gse13067', 'human-intestine-cancer-1-standard', 'pamr_classifier', 'gse33113', 'gse108989', 'prjeb23709', 'gse103479'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10238095: 0.03225806451612903\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10238095: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9710693\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 4. Element IDs: {'syn23554292', 'syn3505720', 'syn3219045', 'syn3191087'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse75748', 'e-mtab-5061', 'syn23554292', 'syn3505720', 'syn3191087', '10.7303/syn3219045', 'gse72056', 'e-mtab-5060'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn3219045, gt_id=syn3219045\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9710693: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9710693: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9915613\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1688369'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse56066', 'gse78284', 'gse99542', 'gse102441', 'gse62944', 'syn1688366', 'gse154873', 'gse96517', 'gse78913', 'gse25710', 'gse84515', 'gse73663', 'gse78167', 'gse57436', 'gse51403'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9915613: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9915613: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11420198\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn26164834', 'syn4921369'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse59119', 'syn4921369', 'gse126957', 'gse249687', 'gse84176', 'gse142881', 'gse96107'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11420198: 0.14285714285714285\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11420198: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7865022\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20681023'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn20681023', '10.6084/m9.figshare.13342055'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7865022: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7865022: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8131595\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn4587609', 'syn18485175'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.1371/journal.pone.0143563', 'gse150130', 'gse113673', 'gse81912', 'gse84245', 'encsr822cea', 'srx3381923', 'syn4587609', 'encsr198zyj', 'phs001373.v2.p2', 'gse59318', 'gse60192', 'encsr677hxc', 'syn18485175', '10.1371/journal.pgen.1000650', 'gse65776', 'e-mtab-9045', 'gse59630', 'gse96107'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8131595: 0.10526315789473684\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8131595: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10828232\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn13363290'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse214628', 'syn13363290-v49'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=syn13363290-v49, gt_id=syn13363290\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10828232: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10828232: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11852220\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn51365301'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'2lll', '10.3390/', 'syn51365301', '8caa', '7lwh'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11852220: 0.2\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11852220: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9314356\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4907804'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'adni', 'syn4907804'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9314356: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9314356: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11439699\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 10. Element IDs: {'syn22264775', 'syn10235592', 'syn2580853', 'syn3157325', 'syn9702085', 'syn4896408', 'syn20068543', 'syn3157275', 'syn26720676', 'syn17008935'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn22264775', 'syn10235592', 'syn2580853', '10.7303/syn3157325', 'syn9702085', 'syn338856', 'syn4896408', 'syn20068543', 'syn3157275', 'syn26720676', 'syn17008935'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn3157325, gt_id=syn3157325\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11439699: 0.9090909090909091\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11439699: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6323985\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn6131484'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn6131484'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10680627\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn52264240', 'syn52234396'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse225157', 'encsr443vtk', 'syn52234396', 'syn52264240', 'epcrisprbenchmark_ensemble_data_grch38.tsv.gz', 'encsr998ydi'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10680627: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10680627: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc5192600\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn3049712', 'syn2787209'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn3049712', 'syn2787209'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC5192600: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC5192600: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7752919\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2623706'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn2623706', 'gse18088', 'gse44076', 'tcga-read', 'gse39582', 'e-mtab-8152', 'gse33113', 'tcga-coad', 'gse13294', 'gse2109', 'gse148394', 'gse38832', 'gse14333'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7752919: 0.07692307692307693\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7752919: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8859891\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4224222'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn4224222'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9853518\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn3193805'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn3193805', '3757476'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9853518: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9853518: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11852190\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn3193805'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn3193805'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11852190: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11852190: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10652415\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn3388564', 'syn3157322', 'syn26254632'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn3388564', 'syn26254632', 'syn3157322'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10652415: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10652415: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7842189\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2580853'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse1432', 'syn3163039', 'gse75431', 'phs002197.v1.p1', 'ng00040', 'syn3157182', 'gse146866', 'syn7818788', 'gse13162', 'gse121654', 'gse57181', 'gse90696', 'gse98401', 'gse55627', 'gse61499', 'gse61500', 'gse138852', 'syn18485175', 'gse65067', 'gse103334', 'syn9884351'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7842189: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7842189: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc3788619\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn1729383', 'syn2237931'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'srep02651-s2.zip', 'syn1729383', 'syn2237931'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC3788619: 0.6666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC3788619: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6750256\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4587609'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'nihms1534152-supplement-2.pdf', 'nihms1534152-supplement-3.xlsx', 'nihms1534152-supplement-6.xlsx', 'nihms1534152-supplement-1.docx', 'syn4587609', 'nihms1534152-supplement-5.xlsx', 'nihms1534152-supplement-4.xlsx', 'nihms1534152-supplement-7.xlsx'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6750256: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6750256: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11370573\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn25569736'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn25569736'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11370573: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11370573: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4344205\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1688369'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn1688369'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4344205: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4344205: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9630780\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 8. Element IDs: {'syn34542450', 'syn25792906', 'syn25876213', 'syn20843407', 'syn34542574', 'syn25886762', 'syn34539904', 'syn34542449'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn34542450', 'syn25792906', 'syn25876213', 'ijo1366', 'syn20843407', 'syn34542574', 'syn25886762', 'syn34539904', 'syn34542449'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9630780: 0.8888888888888888\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9630780: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10813059\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn31947506', 'syn31952297', 'syn52138250'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.3390/', 'syn52138250', 'syn31947506', 'p12023', 'syn31952297'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10813059: 0.6\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10813059: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6089972\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn2812961', 'syn7343873', 'syn7214402'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse35069', 'syn7343873', 'prjeb5468', 'gse38266', 'gse68379', '10.5281/zenodo.1304766', 'gse38270', 'syn7214402', '10.5281/zenodo.1298968', 'prjeb11844', 'prjna258216', 'gse74877', 'syn2812961'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6089972: 0.23076923076923078\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6089972: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8130821\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 4. Element IDs: {'syn17015098', 'syn9884314', 'syn20803583', 'syn23191787'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn17015098', 'syn9884314', 'syn20803583', 'syn23191787'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8130821: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8130821: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11224609\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn10901595'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn10901595', '10.5281/zenodo.11240547', '10.5281/zenodo.10795678'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11224609: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11224609: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7577712\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20821165'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'pxd009589', 'syn20821165', 'gse33000'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7577712: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7577712: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7257173\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21541022'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'pxd013342', 'syn18485233', 'pxd013349', '10.1101/806752', 'pxd013352', 'syn21541022', 'pxd013350', 'syn18485244', 'pxd013347', 'pxd013351'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7257173: 0.1\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7257173: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9780309\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn26560886'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn26560886'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9780309: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9780309: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7658217\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21041850'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse132355', 'c4077b3c-5c98-4d26-a614-246d12c2e5d7', 'gse122960', 'syn21041850', '10.1101/742320'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7658217: 0.2\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7658217: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6289083\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn11967914'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'egas00001003068', 'syn11967914'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6289083: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6289083: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8807579\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20808171'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: set()\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8807579: 0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8807579: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6700281\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2759792'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'phs000424.v7.p1', 'syn2759792', 'doi:10.5061/dryad.5d4s2r8', '47eb8891-86da-11e7-a10d-0ac135e8bacf'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6700281: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6700281: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7029360\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn8717496'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn8717496'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8219045\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn3163039'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'e-mtab-5061', 'gse64810', 'syn3163039', 'gse97942'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8219045: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8219045: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10543041\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn50996324'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.13020/', 'syn50996324'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10543041: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10543041: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6546784\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4921369'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.1038/nature19847', 'gse77565', 'syn4921369', 'phs001190.v1.p1'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6546784: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6546784: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10696419\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn22255433'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn22255433'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10696419: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10696419: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc5387919\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 10. Element IDs: {'syn5522788', 'syn5004874', 'syn4984604', 'syn5556216', 'syn5522790', 'syn6022474', 'syn5492805', 'syn6022465', 'syn6035832', 'syn5493036'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn5522788', 'syn5004874', 'syn4984604', 'syn5556216', 'syn5522790', 'syn6022474', 'syn5492805', 'syn6022465', 'syn6035832', 'syn5493036', 'cutaneousnf'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC5387919: 0.9090909090909091\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC5387919: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8094983\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn23532550'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'epi_isl_461058', 'epi_isl_461002', 'epi_isl_461059', 'epi_isl_461053', 'epi_isl_415464', 'epi_isl_461054', 'epi_isl_461056', 'epi_isl_461055', 'epi_isl_461060', 'epi_isl_461052', 'syn23532550', 'epi_isl_461051', 'epi_isl_461003', 'epi_isl_461057', 'epi_isl_461001'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8094983: 0.06666666666666667\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8094983: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7962955\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20640275'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn20640275', 'gse102470', 'e-mtab-9905', 'gse57763', 'nc_045512.2', 'mn996532.1', 'gse137834', 'gse33584'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7962955: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7962955: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11601401\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn20933797', 'syn51150434', 'syn62110225'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.7303/syn62110225', 'syn20933797', 'syn51150434', '10.58085/hr6s-2991'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn62110225, gt_id=syn62110225\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11601401: 0.75\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11601401: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9712627\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn12514624'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse183068', 'supplemental_table_s1', 'supplemental_table_s3', 'gse174367', 'gse174332', 'gse160936', 'supplemental_table_s8', 'gse148822', 'supplemental_table_s2', 'cg000376', 'gse178265', 'syn12514624', 'supplemental_data_s2', 'supplemental_table_s7', 'supplemental_table_s4', 'supplemental_table_s5', 'supplemental_data_s1', 'gse167494', 'supplemental_table_s6', 'gse137444'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9712627: 0.05\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9712627: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4697768\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2813581'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.1038/455001a', '10.4137/cmo.s38279', '10.1038/nature07634', 'syn2813581', '10.1093/nar/gku1075', '10.1038/527s108a', '10.3322/caac.21329'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4697768: 0.14285714285714285\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4697768: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9094742\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21125841'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse75330', 'gse129788', 'gse116470', 'gse60361', 'pxd029727', 'gse118257', 'gse97930', 'pxd020007', 'gse140511', 'gse130119', 'scp90', 'srp135960', 'syn21125841', 'gse113973'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9094742: 0.07142857142857142\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9094742: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8565335\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn6131484'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'encsr000e', 'encsr483rkn', 'syn6131484', 'hg38-blacklist.v2.bed.gz', 'gse115046', 'encsr000ekp', 'encsr220asc'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8565335: 0.14285714285714285\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8565335: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10985232\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn10003873'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn10003873', 'srp291312', '10.5281/zenodo.10677267', 'srp158590', 'srp186687'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10985232: 0.2\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10985232: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8784774\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20968331'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn20968331', 'gse186341', '4257', 'gse70138', 'gse92742'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8784774: 0.2\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8784774: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8082263\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn11801497'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn11801497', 's0092867418303593-mmc1.xlsx', 'cytoband.txt.gz', 'hsa.gff3.txt', '4f277128-f793-4354-a13d-30cc7fe9f6b5', 'd773a53e-b78d-4a20-848c-866616352328', 's1535610818301119-mmc2.xlsx', 'pmd', 'mature.fa', 'infiniumannotation', '1b5f413e-a8d1-4d10-92eb-7c4ae739ed81', 'mirna_information.xlsx', '7d4c0344-f018-4ab0-949a-09815f483480', 'tcga_ancestry_assignments', '1a7d7be8-675d-4e60-a105-19d4121bdebf', '10.1016/', 's0092867417310565-mmc1.xlsx'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8082263: 0.058823529411764705\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8082263: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11659981\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21041850'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse115469', 'e-mtab-6653', 'tcga-luad', '10.5281/zenodo.12092871', 'v.4.4.210', 'e-mtab-6149', 'gse134432', '10.5524/102624', 'v14', 'scp498', 'syn21041850', 'gse162170', 'wouters_human_melanoma'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11659981: 0.07692307692307693\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11659981: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9356164\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn12169715', 'syn12169709'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'3586c0da-64d0-4b74-a449-5ff4d9136611', 'syn12169715', '10.1016/', '1a7d7be8-675d-4e60-a105-19d4121bdebf', 'syn12169709'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9356164: 0.4\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9356164: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11271596\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn53009474'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.7303/syn53009474'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn53009474, gt_id=syn53009474\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11271596: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11271596: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7520714\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn18344812', 'syn17009517'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.6084/m9.figshare.8159525.v1', '10.7303/syn18344812', 'syn17009517'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn18344812, gt_id=syn18344812\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7520714: 0.6666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7520714: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10769298\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn11724057'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn11724057', '10.1101/2023.12.20.572491'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10769298: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10769298: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8226229\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2701943'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn2701943', '14540610', 'gse30784'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8226229: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8226229: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11716409\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4231880'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'medquad', 'syn4231880', '10.24432/c5bs32', 'paquid', 'gse79209', 'gse16476', '10.13026/c28g6p', 'quadeer15sh/lfw-facial-recognition', 'nantonio/a-hotels-customers-dataset', 'breast-cancer-screening-dbt', 'gse89413', '7036235', '351', 'mimiciii/1.4', 'ronanpickell/1980s-album-covers', '10.13026/c2f305', 'mimiciv/2.0', 'panfordofori/fitbitdata', 'e-mtab-8248'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11716409: 0.05263157894736842\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11716409: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8055881\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn7824274'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'s050', 's053', 's039', 'mc3', 's038', 's037', 'syn7824274'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8055881: 0.14285714285714285\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8055881: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8102856\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn7214402'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'mc3', 'syn7214402', 'cnp0001438'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8102856: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8102856: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9085578\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn8612097'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.1038/sdata.2018.185', 'phs000673.v2.p1', '10.1038/sdata.2016.89', '10.3389/', 'syn8612097'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9085578: 0.2\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9085578: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9840277\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn23569889', 'syn14237651', 'syn11914606'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn23569889', 'syn14237651', 'syn11914606'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9840277: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9840277: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10547713\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn26707219', 'syn52137895'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.1055/', 'syn26707219', 'syn52137895', '10.1038/'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10547713: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10547713: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10571258\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn30821562'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.5281/zenodo.8322854', 'syn30821562', '10.5281/zenodo.8338532'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10571258: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10571258: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6829597\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn18796452'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn18796452', 'q32852', 'r25005', 'l34975'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6829597: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6829597: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11240079\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn52566791'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn52566791'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11240079: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11240079: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7805663\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn11737848'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn11737848'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7805663: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7805663: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10600818\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn23627957'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn23627957', '10.1017/'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10600818: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10600818: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc5752539\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1741407'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'tcga', 'syn1741407', 'gse63459', 'mirnatarget', 'gse63805', 'mirdb', 'mir2disease', 'hprd', 'gse29623'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC5752539: 0.1111111111111111\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC5752539: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6602064\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn7818788', 'syn9884357'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'phs000767.v1.p1', 'gse44772', 'pxd005703', 'syn7818788', 'gse65057', 'phs000728.v1.p1', 'gse74193', 'gse66504', 'gse53683', 'gse67140', 'gse75510', 'gse45480', 'gse90696', 'gse45479', 'syn9884357', 'gse77242', 'gse45481', 'gse47572'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6602064: 0.1111111111111111\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6602064: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7032692\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1710250'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'tcga', 'syn1710250', 'egas00000000083'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7032692: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7032692: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10061294\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn2468297', 'syn1715755'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.7303/syn2468297', 'syn1715755'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn2468297, gt_id=syn2468297\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10061294: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10061294: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8006362\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn22024464'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.5281/zenodo.4321679', '10.15154/1504248', 'syn22024464'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8006362: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8006362: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7778917\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn18360482'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.1093/nar/gkaa853', 'syn18360482'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7778917: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7778917: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 89 - INFO - \n",
      "Performance evaluation completed for 100 source pages.\u001b[0m\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "{'average_precision': 0.4678899917342445,\n",
       " 'average_recall': 0.948,\n",
       " 'f1_score': 0.6265454445663144}"
      ]
     },
     "execution_count": 19,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "check_df = pd.read_csv(\"scripts/output/syn_data_results/result_gemini-2.5-flash.csv\")\n",
    "ret = evaluate_performance(check_df, ground_truth, dg, \"scripts/output/false_positives.txt\")\n",
    "ret"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "id": "35d77225",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11792374\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 4. Element IDs: {'syn9702085', 'syn2580853', 'syn8456637', 'syn52293417'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn9702085', 'syn2580853', 'syn8456637', 'syn52293417'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4318527\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn1720047', 'syn2455683'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn1720047', 'syn2455683', 'syn312572'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527: 0.6666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4339277\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 4. Element IDs: {'syn1701558', 'syn1732077', 'syn1682511', 'syn1688309'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn1732077', '10.1038/ncomms4963', 'nihms590622-supplement-supp_info_2.rar', 'syn300013', 'pmc4339277', 'syn1701558', '24999802', 'syn1682511', 'nihms590622-supplement-supp_info_1.pdf', 'syn1688309'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277: 0.4\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8452114\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn18485175'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'media-1.pdf', 'media-4.xlsx', 'media-3.pdf', 'media-2.pdf', 'syn18485175'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8452114: 0.2\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8452114: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9641692\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn3219045'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse107011', 'gse119117', 'gse84133', 'gse65133', 'gse87544', 'gse115449', 'sdy67', '8k_pbmc', '6k_pbmc', 'syn3219045', '10.5281/zenodo.7183923', 'gse67835', '10k_pbmc'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9641692: 0.07692307692307693\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9641692: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc5161470\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1446295'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'s1_table', 'hapmap_encode_non-redundant', 's3_table', 'hapmap_phase3_chr22_consensus', 'syn1446295', '1000genomes_release_20130502', 's2_table', '1000genomes_tagsnp', 's5_table', 's4_table'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC5161470: 0.1\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC5161470: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9925742\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21126450'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'srr21079269', 'gse224398', 'srr21079270', 'syn21126450', 'srr21079267', 'srr21079265', 'srr21079266', 'srr21079268'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9925742: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9925742: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9235070\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn30243143'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'tcga', 'syn213309', 'gse75688', 'gse81861', 'e-mtab-6149', 'syn30243143', 'gse70630', '10.1242/bio.059256', 'gse76312', 'gse81383', 'gse103322'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9235070: 0.09090909090909091\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9235070: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11661334\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn51364943'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gcst90027158', 'cowas', '35107', 'gcst007511', 'gcst009325', 'syn51364943', 'mykmal/cowas', 'glgc_lipids2021', 'hippie_v2.3'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11661334: 0.1111111111111111\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11661334: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11929800\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 5. Element IDs: {'syn3388564', 'syn11724057', 'syn17015098', 'syn4896408', 'syn18485175'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn3388564', 'syn11724057', 'gcst90027158', '7276971', 'zhao-research/5hmc', 'syn17015098', 'e073_15_coremarks_hg38lift_stateno.bed', '10.5281/zenodo.14804357', 'syn4896408', 'gse291402', 'jansen_summary_statistics'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11929800: 0.36363636363636365\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11929800: 0.8\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7113080\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn17394041'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'783134v1', '3572974', '366856', '32191543', 'pmc7113080', 'syn17394041'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080: 0.16666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8847344\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn25813713'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'moesm2', 'moesm5', 'moesm4', 'syn25813713', 'moesm1', 'moesm7', 'moesm3', '10.7303/syn25813713', 'moesm6'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8847344: 0.1111111111111111\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8847344: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9280291\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn5612563'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse40586', 'e-mtab-5273', 'gse13015', 'gse54514', 'gse40012', 'gse65682', 'gse27131', 'e-mexp-3589', 'syn5612563', 'e-mtab-5274', 'gse25504', 'gse66099', 'gse20346', 'gse32707', 'gse95233', 'gse60244', 'gse21802', 'gse57065', 'e-mtab-1548'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9280291: 0.05263157894736842\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9280291: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10029021\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn50897018'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn50897018.3'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=syn50897018.3, gt_id=syn50897018\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10029021: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10029021: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11525589\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn51514106'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn51514106'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11525589: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11525589: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10836119\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 6. Element IDs: {'syn3157238', 'syn9705278', 'syn3205821', 'syn5592519', 'syn3157329', 'syn3157322'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn3157238', 'syn9705278', 'syn3205821.6', 'syn5592519', 'syn3157329', 'syn3157322'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=syn3205821.6, gt_id=syn3205821\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10836119: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10836119: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6646046\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn5609493'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'e-mtab-2323', 'syn5609493'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6646046: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6646046: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10387117\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21041850'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'lmex0000004397', 'gse149563', 'gse122332', 'gsm4504966', 'gse135851', 'gse122960', 'gse178362', 'gse136831', 'gsm4035472', 'gse134174', 'egas00001004082', 'github.com/xu-lab/cellref', 'gsm5388412', 'dataset-427f1eee6dd44f50bae1ab13f0f3c6a9', 'gsm5388413', 'gse136832', 'gse171524', 'syn21041850', 'gse161382', 'gsm5388411', 'lmex0000004396', 'gse135893', 'gsm4504967'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10387117: 0.043478260869565216\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10387117: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4444046\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn1729383', 'syn1710680', 'syn1734155'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn1729383', 'pmc4444046', 'pb002581', '10.1038/ng.3168', 'nihms644198-supplement-2.xlsx', 'nihms644198-supplement-1.pdf', 'nihms644198-supplement-3.xlsx', 'syn1710680', 'syn1734155'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4444046: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4444046: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10329279\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21478998'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn21478998'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11145390\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21788425'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.5281/zenodo.10971926', 'skannan4/cm-entropy-score', 'syn21788425', 'elainezhlchen/entropy_star_protocol', 'gse147807'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11145390: 0.2\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11145390: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11032436\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2759792'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'crl-3216', 'syn2759792', 'nd50034', 'nds00143'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9984184\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn39456106', 'syn25576878'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse147205', 'gouveiaroque_chipmera', 'gse169185', 'syn39456106', 'syn25576878'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9984184: 0.4\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9984184: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7347884\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1757053'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn1757053', 'e-mtab-4145', 'egas00000000083', 'gse56031'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7347884: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7347884: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10802452\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2759792'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'media-2.xlsx', 'media-4.xlsx', 'nihpp2024.01.09.574815v2-supplement-5.pdf', 'media-1.xlsx', 'syn2759792', 'media-3.xlsx', 'all_sumstats'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452: 0.14285714285714285\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8907344\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn26134616'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn26134616', '10.7303/syn26134616'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8907344: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8907344: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10238095\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2623706'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gdsc', 'imvigor210', 'gse14333', 'gse78220', 'gse38832', 'gse103479', 'pre_clinical_model', 'human-colorectal-cancer-11-mm-capture-area-ffpe-2-standard', 'gse39582', 'gse17536', 'gse13294', 'cccrc_classifier', 'syn2623706', 'ccle_broad_2019', 'gse104645', 'gse37892', 'tcga', 'gse13067', 'human-intestine-cancer-1-standard', 'pamr_classifier', 'gse33113', 'gse108989', 'prjeb23709', 'gse35144', 'gse135222'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10238095: 0.04\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10238095: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9710693\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 4. Element IDs: {'syn23554292', 'syn3505720', 'syn3219045', 'syn3191087'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse75748', 'e-mtab-5061', 'syn23554292', 'syn3505720', 'syn3191087', 'syn3219045', 'gse72056', 'e-mtab-5060'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9710693: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9710693: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9915613\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1688369'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse56066', 'syn1688369', 'gse99542', 'gse78284', 'gse102441', 'gse62944', 'gse154873', 'gse96517', 'gse78913', 'gse25710', 'gse84515', 'gse73663', 'gse78167', 'gse57436', 'gse51403'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9915613: 0.06666666666666667\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9915613: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11420198\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn26164834', 'syn4921369'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse59119', 'syn4921369', 'pnas.2402518121.sd12.xlsx', 'syn26164834', 'pnas.2402518121.sd06.xlsx', 'gse249687', 'rad21-2024', 'pnas.2402518121.sd04.xlsx', 'gse126957', 'gse84176', 'pnas.2402518121.sd05.xlsx', 'gse142881', 'pnas.2402518121.sd01.xlsx', 'pnas.2402518121.sd10.xlsx', 'pnas.2402518121.sd07.xlsx', 'pnas.2402518121.sd09.xlsx', 'pnas.2402518121.sd11.xlsx', 'pnas.2402518121.sd13.xlsx', 'pnas.2402518121.sd03.xlsx', 'pnas.2402518121.sapp.pdf', 'pnas.2402518121.sd02.xlsx', 'pnas.2402518121.sd08.xlsx', 'gse96107'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11420198: 0.08695652173913043\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11420198: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7865022\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20681023'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn20681023', '10.6084/m9.figshare.13342055'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7865022: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7865022: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8131595\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn4587609', 'syn18485175'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse84245', 'phs001373.v2.p2', 'gse59318', 'gse150130', 'gse60192', 'gse113673', 'gse65776', 'encsr677hxc', 'e-mtab-9045', 'encsr822cea', 'gse81912', 'srx3381923', 'syn4587609', 'gse59630', 'encsr198zyj', 'gse96107', 'syn18485175'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8131595: 0.11764705882352941\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8131595: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10828232\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn13363290'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'nm_021272.3', 'gse214628', 'syn13363290-v49'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=syn13363290-v49, gt_id=syn13363290\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10828232: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10828232: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11852220\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn51365301'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'2lll', 'bioengineering-12-00126-s001', 'syn51365301', '8caa', '7lwh'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11852220: 0.2\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11852220: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9314356\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4907804'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn4907804'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9314356: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9314356: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11439699\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 10. Element IDs: {'syn22264775', 'syn10235592', 'syn2580853', 'syn3157325', 'syn9702085', 'syn4896408', 'syn20068543', 'syn3157275', 'syn26720676', 'syn17008935'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn22264775', 'syn10235592', 'syn2580853', 'syn3157325', 'syn9702085', 'syn338856', 'syn4896408', 'syn20068543', 'syn3157275', 'syn26720676', 'syn17008935'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11439699: 0.9090909090909091\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11439699: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6323985\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn6131484'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'pmc6323985', 'syn6131484', '10.1093/nar/gky1128'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10680627\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn52264240', 'syn52234396'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse225157', 'encsr443vtk', 'syn52234396', 'syn52264240', 'encsr998ydi', 'epcrispr'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10680627: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10680627: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc5192600\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn3049712', 'syn2787209'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn3049712', 'syn2787209'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC5192600: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC5192600: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7752919\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2623706'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse18088', 'gse44076', 'tcga-read', 'gse39582', 'e-mtab-8152', 'gse33113', 'tcga-coad', 'gse13294', 'gse2109', 'gse148394', 'syn2623706', 'gse38832', 'gse14333'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7752919: 0.07692307692307693\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7752919: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8859891\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4224222'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.1186/s13058-022-01509-z', 'syn4224222', 'pmc8859891'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9853518\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn3193805'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'zenodo:3757476', 'syn3193805'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9853518: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9853518: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11852190\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn3193805'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn3193805', 'brats'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11852190: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11852190: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10652415\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn3388564', 'syn3157322', 'syn26254632'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn3388564', 'syn3157322', 'syn26254632'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10652415: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10652415: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7842189\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2580853'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse1432', 'syn2580853', 'syn3163039', 'gse75431', 'phs002197.v1.p1', 'ng00040', 'syn3157182', 'gse146866', 'syn7818788', 'gse13162', 'gse121654', 'gse57181', 'gse90696', 'gse98401', 'gse98969', 'gse55627', 'gse61499', 'gse61500', 'syn18485175', 'gse138852', 'gse65067', 'gse103334', 'syn9884351'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7842189: 0.043478260869565216\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7842189: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc3788619\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn1729383', 'syn2237931'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn1729383', 'srep02651-s1.pdf', 'syn2237931', 'srep02651-s3.pdf', 'srep02651-s2.zip', '10.1038/srep02651'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC3788619: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC3788619: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6750256\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4587609'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'nihms1534152-supplement-5', 'nihms1534152-supplement-2', 'nihms1534152-supplement-1', 'nihms1534152-supplement-3', 'nihms1534152-supplement-6', 'nihms1534152-supplement-7', 'syn4587609', 'nihms1534152-supplement-4'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6750256: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6750256: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11370573\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn25569736'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn25569736'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11370573: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11370573: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4344205\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1688369'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn1688369'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4344205: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4344205: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9630780\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 8. Element IDs: {'syn34542450', 'syn25792906', 'syn25876213', 'syn20843407', 'syn34542574', 'syn25886762', 'syn34539904', 'syn34542449'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn34542450', 'syn25792906', 'syn25876213', 'ijo1366', 'syn20843407', 'syn34542574', 'syn25886762', 'syn34539904', 'syn34542449', 'carom'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9630780: 0.8\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9630780: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10813059\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn31947506', 'syn31952297', 'syn52138250'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn31947506', 'biomolecules-14-00066-s001.zip', 'syn31952297', 'syn52138250'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10813059: 0.75\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10813059: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6089972\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn2812961', 'syn7343873', 'syn7214402'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse35069', 'syn7343873', 'prjeb5468', 'gse68379', '10.5281/zenodo.1304766', 'gse38270', 'syn7214402', '10.5281/zenodo.1298968', 'prjeb11844', 'prjna258216', 'gse38266', 'gse74877', 'syn2812961'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6089972: 0.23076923076923078\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6089972: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8130821\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 4. Element IDs: {'syn17015098', 'syn9884314', 'syn20803583', 'syn23191787'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'nihms1656683-supplement-1.pdf', 'syn9884314', 'syn20803583', 'syn17015098', 'nihms1656683-supplement-2.xlsx', 'syn23191787'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8130821: 0.6666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8130821: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11224609\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn10901595'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.5281/zenodo.8409105', 'syn10901595', '10.5281/zenodo.11240547', '10.5281/zenodo.10795678'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11224609: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11224609: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7577712\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20821165'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'pxd009589', 'syn20821165', 'gse33000'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7577712: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7577712: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7257173\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21541022'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn21541022'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7257173: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7257173: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9780309\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn26560886'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn26560886'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9780309: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9780309: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7658217\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21041850'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse132355', 'kwl0kho7pbzf', 'c4077b3c-5c98-4d26-a614-246d12c2e5d7', 'gse122960', 'syn21041850', '10.1101/742320'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7658217: 0.16666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7658217: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6289083\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn11967914'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'egas00001003068', 'syn11967914'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6289083: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6289083: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8807579\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20808171'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse36980', 'gse158233', 'syn20808171', 'prjeb24858', 'gse149149', 'gse164466', 'gse151999', 'gse156167'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8807579: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8807579: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6700281\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2759792'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'phs000424.v7.p1', 'syn2759792', 'doi:10.5061/dryad.5d4s2r8'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6700281: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6700281: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7029360\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn8717496'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn8717496'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8219045\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn3163039'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'e-mtab-5061', 'gse64810', 'syn3163039', 'gse97942', 'lqab056_supplemental_files.zip'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8219045: 0.2\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8219045: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10543041\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn50996324'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'media-1.pdf', 'media-2.xlsx', 'media-4.xlsx', 'media-5.xlsx', 'media-7.xlsx', 'syn50996324', '10.1101/2023.09.18.23295431', 'media-8.xlsx', 'media-3.xlsx', 'media-6.xlsx', 'media-10.xlsx', 'media-9.xlsx', 'pmc10543041', '10.13020/3b1n-ff32'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10543041: 0.07142857142857142\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10543041: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6546784\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4921369'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse77565', 'moesm4', 'syn4921369', 'moesm8', 'moesm1', 'phs001190.v1.p1'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6546784: 0.16666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6546784: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10696419\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn22255433'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn22255433', 'ih_2300094_supplemental_1.pdf'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10696419: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10696419: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc5387919\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 10. Element IDs: {'syn5522788', 'syn5004874', 'syn4984604', 'syn5556216', 'syn5522790', 'syn6022474', 'syn5492805', 'syn6022465', 'syn6035832', 'syn5493036'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn5522788', 'syn5004874', 'syn4984604', 'syn5556216', 'syn5522790', 'syn6022474', 'syn5492805', 'syn6022465', 'syn6035832', 'syn5493036', 'cutaneousnf'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC5387919: 0.9090909090909091\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC5387919: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8094983\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn23532550'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'epi_isl_461058', 'epi_isl_461002', 'epi_isl_461059', 'epi_isl_461053', 'epi_isl_415464', 'epi_isl_461054', 'epi_isl_461056', 'epi_isl_461055', 'epi_isl_461060', 'epi_isl_461052', 'syn23532550', 'epi_isl_461051', 'epi_isl_461003', 'epi_isl_461057', 'epi_isl_461001'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8094983: 0.06666666666666667\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8094983: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7962955\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20640275'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn20640275', 'gse102470', 'e-mtab-9905', 'gse57763', 'nc_045512.2', 'mn996532.1', 'gse137834', 'gse33584'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7962955: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7962955: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11601401\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn20933797', 'syn51150434', 'syn62110225'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn20933797', 'syn51150434', 'syn62110225', '10.58085/hr6s-2991'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11601401: 0.75\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11601401: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9712627\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn12514624'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse183068', 'gse148822', 'fresh-frozen-lymph-node-with-b-cell-lymphoma-14-k-sorted-nuclei-1-standard-2-0-0', 'gse178265', 'syn12514624', 'gse174367', 'gse174332', 'supp_gr.276409.121_supplemental_data_s1.zip', 'gse167494', 'gse137444', 'gse160936', 'supp_gr.276409.121_supplemental_data_s2.zip', 'microglia-loss-of-y'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9712627: 0.07692307692307693\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9712627: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4697768\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2813581'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.4137/cmo.s38279', 'syn2813581'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4697768: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4697768: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9094742\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21125841'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse775330', 'gse75330', 'gse129788', 'gse60361', 'gse116470', 'pxd029727', 'gse97930', 'pxd020007', 'gse118257', 'human-archived-brain', 'gse140511', 'gse130119', 'srp135960', 'syn21125841', 'gse113973', 'gse138852'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9094742: 0.0625\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9094742: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8565335\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn6131484'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'encsr000e', 'encsr483rkn', 'gkab676_supplemental_files.zip', 'gse115046', 'encsr000ekp', 'encsr220asc'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8565335: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8565335: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10985232\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn10003873'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn10003873', 'srp291312', 'fusioninspector/fusioninspector', 'star-fusion', '10.5281/zenodo.10677267', 'srp158590', 'xidslab/scfusion', 'suhrig/arriba', 'srp186687', 'shenlab-genomics/anchored-fusion', 'nm_133330.3', 'nm_001306068.3', 'ndaniel/fusioncatcher'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10985232: 0.07692307692307693\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10985232: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8784774\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20968331'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn20968331', 'proteomicsdb_project_4257', 'gse186341', 'gse70138', 'gse92742'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8784774: 0.2\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8784774: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8082263\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn11801497'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn11801497', 'mi0000466', '10.1016/j.xpro.2021.100483', 'mmc2', 'ccg-aim-2020', 'mmc1', '10.1016/j.ccell.2020.04.012', '4f277128-f793-4354-a13d-30cc7fe9f6b5', 'd773a53e-b78d-4a20-848c-866616352328', '1b5f413e-a8d1-4d10-92eb-7c4ae739ed81', '7d4c0344-f018-4ab0-949a-09815f483480', '1a7d7be8-675d-4e60-a105-19d4121bdebf'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8082263: 0.08333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8082263: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11659981\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21041850'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'pathwaycommons_v14', 'gse115469', 'e-mtab-6653', 'tcga-luad', '10.5281/zenodo.12092871', 'e-mtab-6149', 'gse134432', '10.5524/102624', 'scp498', 'syn21041850', 'gse162170', 'scgraph2vec', 'biogrid_v4.4.210', '10x_pbmc_3k', 'trrust_v2'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11659981: 0.06666666666666667\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11659981: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9356164\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn12169715', 'syn12169709'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'legacy-archive', 'genomewidesnp_6', '3586c0da-64d0-4b74-a449-5ff4d9136611', 'syn12169715', '1-s2.0-s1535610820302117-mmc5.xlsx', 'sayaman_tables1_mmc2.xlsx', 'ccg-aim-2020', 'tcga_pancancer_genotyping_imputation', '1-s2.0-s1535610820302117-mmc2.xlsx', 'pancanqtl', '10.5281/zenodo.6658317', '1a7d7be8-675d-4e60-a105-19d4121bdebf', 'syn12169709'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9356164: 0.15384615384615385\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9356164: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11271596\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn53009474'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn53009474', '10.7303/syn53009474'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11271596: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11271596: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7520714\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn18344812', 'syn17009517'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.6084/m9.figshare.8159525.v1', '10.7303/syn18344812', 'syn17009517'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn18344812, gt_id=syn18344812\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7520714: 0.6666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7520714: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10769298\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn11724057'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn11724057', 'nihpp2023.12.20.572491v1-supplement-4.pdf', '10.1101/2023.12.20.572491', 'media-1.xlsx', 'nihpp2023.12.20.572491v1-supplement-2.pdf', 'pmc10769298', 'nihpp2023.12.20.572491v1-supplement-3.pdf', 'pa01-pa22'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10769298: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10769298: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8226229\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2701943'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn2701943', 'gse30784', 'njhncc-rnaseq-stemsig [14540610]'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8226229: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8226229: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11716409\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4231880'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse16476', 'gse89413', 'gpl18573', '10.13026/c28g6p', 'syn4231880', 'medquad', '10.24432/c5bs32', 'mimic-iii', 'gse79209', 'breast-cancer-screening-dbt', '10.13026/c2f305', 'e-mtab-8248'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11716409: 0.08333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11716409: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8055881\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn7824274'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'s050', 'phosphositeplus_snapshot_2018-02-14', 'depmap', 's053', 's039', 'mc3', 'ptmcosmos', 's038', 'rcsb_pdb_structures_2017-05-22', 's037', 'uniprotkb_v2018.01', 'syn7824274', 'cptac_data_portal'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8055881: 0.07692307692307693\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8055881: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8102856\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn7214402'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'mc3', 'syn7214402', 'cnp0001438'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8102856: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8102856: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9085578\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn8612097'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn8612097'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9085578: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9085578: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9840277\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn23569889', 'syn14237651', 'syn11914606'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn23569889', 'syn14237651', 'syn11914606'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9840277: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9840277: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10547713\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn26707219', 'syn52137895'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.7303/syn26707219', 'syn26707219', 'syn52137895'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10547713: 0.6666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10547713: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10571258\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn30821562'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'github.com/davidrconnell/igraph-se2', 'syn30821562', '10.5281/zenodo.8338532', '10.5281/zenodo.8322854', 'github.com/cogdishion/se2'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10571258: 0.2\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10571258: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6829597\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn18796452'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'jem_20190430_datas1.xlsx', 'jem_20190430_sm.pdf', 'syn18796452'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6829597: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6829597: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11240079\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn52566791'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn52566791'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11240079: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11240079: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7805663\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn11737848'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn11737848'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7805663: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7805663: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10600818\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn23627957'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'s0033291723000235a', 'focus.db', '10.1017/s0033291723000235', 's0033291723000235sup001', 'syn23627957'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10600818: 0.2\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10600818: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc5752539\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1741407'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn1741407', 'gse63459', 'mirnatar', 'gse63805', 'oncotarget-08-109522-s002.xls', 'pmc5752539', '10.18632/oncotarget.22724', 'oncotarget-08-109522-s001.pdf', 'gse29623'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC5752539: 0.1111111111111111\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC5752539: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6602064\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn7818788', 'syn9884357'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse90696', 'syn7818788', 'syn9884357'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6602064: 0.6666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6602064: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7032692\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1710250'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'tcga', 'syn1710250', 'metabric', 'egas00000000083', 'egac00001000484'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7032692: 0.2\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7032692: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10061294\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn2468297', 'syn1715755'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.7303/syn2468297', 'syn1715755'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn2468297, gt_id=syn2468297\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10061294: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10061294: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8006362\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn22024464'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'1000g_2504_high_coverage', '10.5281/zenodo.4321679', '10.15154/1504248', 'bsmn-aws-setup', 'syn22024464', 'bsmn-pipeline', 'bsmn.synapse.org'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8006362: 0.14285714285714285\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8006362: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7778917\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn18360482'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'pmid:33079988', 'idg-kinase/darkkinasekb', 'pmc7778917', '10.1093/nar/gkaa853', 'syn18360482', 'idg-kinase/kinase_expression'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7778917: 0.16666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7778917: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 89 - INFO - \n",
      "Performance evaluation completed for 100 source pages.\u001b[0m\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "{'average_precision': 0.41387459247016745,\n",
       " 'average_recall': 0.988,\n",
       " 'f1_score': 0.5833732911016106}"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "check_df = pd.read_csv(\"scripts/output/syn_data_results/result_gpt-5-mini.csv\")\n",
    "ret = evaluate_performance(check_df, ground_truth, dg, \"scripts/output/false_positives.txt\", \"scripts/output/false_negatives.txt\")\n",
    "ret"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "473650bb",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "2025_arm",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.12.11"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
