{
 "cells": [
  {
   "cell_type": "code",
   "id": "initial_id",
   "metadata": {
    "collapsed": true,
    "ExecuteTime": {
     "end_time": "2025-08-18T22:55:15.715735Z",
     "start_time": "2025-08-18T22:55:05.271848Z"
    }
   },
   "source": [
    "from data_gatherer.data_gatherer import DataGatherer\n",
    "from experiment_utils import evaluate_performance\n",
    "import pandas as pd"
   ],
   "outputs": [],
   "execution_count": 1
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-08-18T22:55:15.731386Z",
     "start_time": "2025-08-18T22:55:15.717920Z"
    }
   },
   "cell_type": "code",
   "source": [
    "df = pd.read_csv(\"exp_input/SAGE_groundtruth.csv\")\n",
    "input_file = \"exp_input/SAGE_input.txt\""
   ],
   "id": "78d2b2bc4106d2ef",
   "outputs": [],
   "execution_count": 2
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-08-18T22:55:15.736656Z",
     "start_time": "2025-08-18T22:55:15.732624Z"
    }
   },
   "cell_type": "code",
   "source": [
    "model_name = \"gemini-2.0-flash\"  # \"gemini-2.0-flash\" or \"gpt-4o-mini\"\n",
    "prompt = \"retrieve_datasets_simple_JSON\"  # \"GPT_from_full_input_Examples\" or \"retrieve_datasets_simple_JSON\"\n",
    "FDR = False\n",
    "semantic_retrieval = True\n",
    "section_filter= None"
   ],
   "id": "cca05a29b7c100d1",
   "outputs": [],
   "execution_count": 3
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-08-18T22:55:15.742655Z",
     "start_time": "2025-08-18T22:55:15.738238Z"
    }
   },
   "cell_type": "code",
   "source": [
    "# write list to a text file\n",
    "with open(input_file, 'r') as f:\n",
    "    pmcids = f.read().splitlines()\n",
    "\n",
    "print(\"Number of PMCIDs:\", len(pmcids))"
   ],
   "id": "a4ae25cbd54d4fe1",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of PMCIDs: 25\n"
     ]
    }
   ],
   "execution_count": 4
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-08-18T22:55:15.753683Z",
     "start_time": "2025-08-18T22:55:15.745247Z"
    }
   },
   "cell_type": "code",
   "source": "dg = DataGatherer(llm_name=model_name, log_level='INFO', process_entire_document=FDR)",
   "id": "b79cbd7265b497e1",
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001B[97mdata_gatherer.py - line 255 - INFO - Setting up data fetcher...\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 285 - INFO - Data fetcher setup completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 103 - INFO - DataGatherer orchestrator initialized. Extraction Model: gemini-2.0-flash\u001B[0m\n"
     ]
    }
   ],
   "execution_count": 5
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-08-18T23:00:13.688910Z",
     "start_time": "2025-08-18T22:55:15.755142Z"
    }
   },
   "cell_type": "code",
   "source": [
    "input_file = \"exp_input/SAGE_input.txt\"\n",
    "combined_df = dg.run(input_file=input_file, semantic_retrieval=semantic_retrieval, section_filter=section_filter, prompt_name=prompt)"
   ],
   "id": "bfd0572331e32fe2",
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001B[97mdata_gatherer.py - line 624 - INFO - Loaded 25 URLs from file.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 539 - INFO - 0th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 347 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 354 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 533 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC11792374&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 461 - WARNING - No links found in the section.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 395 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 399 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 413 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 417 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 10 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b0044e80>, <Element supplementary-material at 0x1b00549c0>, <Element supplementary-material at 0x1b0054d00>, <Element supplementary-material at 0x1b0054980>, <Element supplementary-material at 0x1b0054dc0>, <Element supplementary-material at 0x1b0054d80>, <Element supplementary-material at 0x1b0055080>, <Element supplementary-material at 0x1b0055200>, <Element supplementary-material at 0x1b0054f40>, <Element supplementary-material at 0x1b0055240>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b0044e80>, <Element supplementary-material at 0x1b00549c0>, <Element supplementary-material at 0x1b0054d00>, <Element supplementary-material at 0x1b0054980>, <Element supplementary-material at 0x1b0054dc0>, <Element supplementary-material at 0x1b0054d80>, <Element supplementary-material at 0x1b0055080>, <Element supplementary-material at 0x1b0055200>, <Element supplementary-material at 0x1b0054f40>, <Element supplementary-material at 0x1b0055240>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM1: ABCA1 expression also correlated with SASP scores in the same human brain samples (Supplementary Fig. 1 [this file] A). Sn-RNA sequencing analysis also indicated that mTOR activation scores were positively associated with ABCA1 expression in astrocytes, excitatory and inhibitory neurons, microglia, oligodendrocytes, and oligodendrocyte progenitor cells (Supplementary Fig. 1 [this file] B). AD characteristics, such as Braak stage, NFT burden, and neuritic plaque burden, were also positively correlated with mTOR activation in excitatory neurons (Supplementary Fig. 1 [this file] B), where APOE4 was associated with increased mTOR expression in excitatory neurons, oligodendrocytes, and pericytes (Supplementary Fig. 1 [this file] B). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM2: Using SenTraGor (STG, a reagent for detection of lipofuscin) [ 50 [rid=CR50] ] staining in postmortem human brain slices (mid-frontal lobe region) from ROS, we observed higher STG intensity in GFAP-positive astrocytes in AD participants compared to NCI participants with the APOE3/4 genotype (Fig. 2 [rid=Fig2] A, the STG/GFAP staining test is shown in Supplementary Fig. 2 [this file] A), corroborating the increased astrocyte senescence markers in AD observed in the transcriptomic data. \n",
      "Given that ABCA1 is predominantly localized in cell membranes [ 51 [rid=CR51] ], cytosolic proteins were removed (Fig. 2 [rid=Fig2] B, Supplementary Fig. 2 [this file] B). Total membrane ABCA1 levels were higher in APOE3/3 and APOE3/4 carriers with AD dementia than in APOE3/3 and APOE3/4 carriers in the NCI group (Fig. 2 [rid=Fig2] B; full blots are shown in Supplementary Fig. 2 [this file] C). \n",
      "In human postmortem middle frontal lobe tissues, caveolin-1 protein levels were higher in both APOE3/3 and APOE3/4 carriers with AD dementia than in the NCI group, although there was no difference between the APOE3/3 and APOE3/4 genotypes in either group (Fig. 3 [rid=Fig3] M; full blots are shown in Supplementary Fig. 2 [this file] C). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM3: ABCA1 was more abundant in lysosome-enriched fractions in the AD group than in the NCI group within the APOE3/4 genotype but not in the APOE3/3 genotype (Fig. 2 [rid=Fig2] C, full blots are shown in Supplementary Fig. 3 [this file] ), supporting that more ABCA1 was trapped in lysosomes in APOE4 AD. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM4: To explore the underlying mechanism, we treated ABCA1-GFP overexpressing HeLa cells with recombinant ApoE3 (rE3) or ApoE4 (rE4). rE4 treatment resulted in lower total ABCA1 protein levels than rE3 treatment, indicating greater ABCA1 degradation (Supplementary Fig. 4 [this file] A). ABCA1-binding complexes were enriched for proteomic analysis (Fig. 3 [rid=Fig3] A and Supplementary Fig. 4 [this file] B). Proteomic analysis revealed 768 shared proteins between rE3 and rE4 treatments, with 314 proteins unique to rE4 (Supplementary Fig. 4 [this file] C). Nine of these proteins (Supplementary Table 4) showed higher expression in the rE4 treatment group than in the rE3 group (Supplementary Fig. 4 [this file] D). Reducing caveolin-1, but not, AP2B1 expression by siRNA increased plasma membrane ABCA1 levels in mouse primary astrocytes (Fig. 3 [rid=Fig3] F, Supplementary Fig. 4 [this file] E), supporting that caveolae-mediated endocytosis regulated ABCA1 degradation. Surprisingly, reducing caveolin-1 expression significantly decreased ABCA1-mediated cholesterol efflux (Fig. 3 [rid=Fig3] G) despite an increase in total ABCA1 protein levels (Supplementary Fig. 4 [this file] F). Furthermore, overexpression of caveolin-1 in astrocytes enhanced cholesterol efflux (Supplementary Fig. 4 [this file] G). \n",
      "Cholesterol loading, by adding LDL, led to increased caveolin-1 expression (Fig. 3 [rid=Fig3] H and I) and showed a dose-dependent increase in baby hamster kidney (BHK) cells (Supplementary Fig. 4 [this file] H). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM5: Mouse cortices were fractionated using TBSX and guanidine HCl to isolate soluble membrane and insoluble aggregated proteins (Supplementary Fig. 5 [this file] A). \n",
      "To identify the main cell type in which ABCA1 and caveolin-1 interact, we isolated microglia, astrocytes and axon-enriched fraction from the brains of adult mice (Supplementary Fig. 5 [this file] B). Astrocytes were enriched in the CD11b fraction (Supplementary Fig. 5 [this file] C). ABCA1 was mainly expressed in astrocytes and axon-enriched fractions, and lower ABCA1 expression was detected in microglial cells (Supplementary Fig. 5 [this file] D). Caveolin-1 was mainly expressed in astrocytes and microglia but not in axon-enriched fractions (Supplementary Fig. 5 [this file] E). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM6: In agreement, ABCA1 brain knockout mice showed higher p21 intensity in the brain than ABCA1 WT mice (Fig. 4 [rid=Fig4] G, p21 staining test is shown in Supplementary Fig. 6 [this file] A). To indirectly assess lysosomal ABCA1 localization in APOE4-TR brains, tissue slides were double stained with LAMP1 and ABCA1 (Supplementary Fig. 6 [this file] B). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM7: Body weight was not affected by HPCD treatment (Supplementary Fig. 7 [this file] A). Total cholesterol levels in the brain (Fig. 5 [rid=Fig5] B) or plasma (Supplementary Fig. 7 [this file] B) were also no different between HPCD and PBS groups. Lysosomal ABCA1 levels were lower in the HPCD treatment group, as verified by lysosome isolation and immunostaining (Fig. 5 [rid=Fig5] G and H; validation of lysosome isolation is shown in Supplementary Fig. 7 [this file] C). Treatment with HPCD was associated with more efficient cholesterol efflux, as evidenced by increased trends of small HDL particle formation in cerebrospinal fluid (CSF) and plasma, although these changes did not reach statistical significance due to the small sample size (Supplementary Fig. 7 [this file] D-F). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM8: HPCD also diminished the activated astrocytes and microglia (Fig. 6 [rid=Fig6] E), and several inflammatory cytokines (Supplementary Fig. 8 [this file] A), but increased synaptophysin, PSD95, and vGLuT1 levels (Supplementary Fig. 8 [this file] B and 8 [this file] C). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM9: To validate these findings in human cells, we cultured human induced pluripotent stem cell (hiPSC)-derived astrocytes with APOE4/4 genotype (Supplementary Fig. 9 [this file] A). When these human iPSc-derived astrocytes were treated with methyl-β-cyclodextrin (MβCD) after LDL loading, cellular cholesterol accumulation (Supplementary Fig. 9 [this file] B) and IL-1β and CCL2 mRNA levels were decreased (Supplementary Fig. 9 [this file] C). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM10: Clinical and pathological characteristics of the samples are detailed in Supplementarys Table ( 1 [this file] ) and Table ( 2 [this file] ). \n",
      "Detailed information regarding all samples is provided in Supplementary Table 1 [this file] . \n",
      "Detailed information about the samples is provided in Supplementary Table 2 [this file] . \n",
      "Detailed information on these samples is presented in Supplementary Table 3 [this file] . \n",
      "The antibodies used are listed in Supplementary Table 5 [this file] . \n",
      "The primers were listed in Supplementary Table 6 [this file] . \n",
      "Cellular senescence pathway enrichment scores for each sample were calculated using single-sample Gene Set Enrichment Analysis (ssGSEA) (the cellular senescence genes are listed in Supplementary Table 7 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 123 - INFO - Loading patterns for section 'data_availability_sections' for publisher 'PMC'.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 131 - INFO - Section pattern keys for publisher PMC: dict_keys(['data_availability_sections', 'supplementary_material_sections', 'supplementary_data_sections', 'key_resources_table', 'methods_sections'])\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 133 - INFO - Section name: data_availability_sections\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 135 - INFO - Found section 'data_availability_sections' in patterns for publisher 'PMC'.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 137 - INFO - Loaded patterns for section 'data_availability_sections': [\".//sec[@sec-type='data-availability']\", \".//notes[@notes-type='data-availability']\", \".//notes[@notes-type='data-availability']\"]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_data_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 706 - INFO - Found 11 supplementary data sections\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(key_resources_table)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "Batches:   0%|          | 0/2 [00:00<?, ?it/s]"
      ],
      "application/vnd.jupyter.widget-view+json": {
       "version_major": 2,
       "version_minor": 0,
       "model_id": "23c9c3a43b284b94afc0dc40d0094e23"
      }
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001B[97membeddings_retriever.py - line 51 - INFO - Searching for top-2 passages similar to the query by embeddings.\u001B[0m\n",
      "\u001B[97membeddings_retriever.py - line 33 - INFO - Computing L2 distances using numpy.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 364 - INFO - Processing DAS_content: 3 elements\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 221\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 398 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-7395ba4a1be1cbccc86e246e52b6405bb697b22d0997e6bd9c0c4f99d1b22c2f\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 398 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-c05dc7df-4360-4220-ac13-1a073c49d190\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755540874,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 482,\n",
      "        \"completion_tokens\": 6,\n",
      "        \"total_tokens\": 488,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 791\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 541 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-dbb7ce156f0502589129b8c5a1b84f5e23b7555b343486571541e789e89f4a15\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 541 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-9329cf9b-2f36-4a0b-9ef7-45f3cdb7128d\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"syn2580853\\\",\\n        \\\"data_repository\\\": \\\"Synapse AD Knowledge Portal\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.synapse.org/#!Synapse:syn2580853\\\"\\n    }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"syn2580853\\\",\\n        \\\"data_repository\\\": \\\"Synapse AD Knowledge Portal\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.synapse.org/#!Synapse:syn2580853\\\"\\n    }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755540875,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 712,\n",
      "        \"completion_tokens\": 76,\n",
      "        \"total_tokens\": 788,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn2580853', 'data_repository': 'Synapse AD Knowledge Portal', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn2580853', 'dataset_id': 'syn2580853'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: syn2580853\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID syn2580853 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: Synapse AD Knowledge Portal, identifier: syn2580853\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn2580853 for Synapse AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn2580853 for Synapse AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn2580853 for Synapse AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn2580853 for Synapse AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1043 - INFO - Found partial match for Synapse AD Knowledge Portal in Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.synapse.org/#!Synapse:syn2580853, repo synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.synapse.org/#!Synapse:syn2580853\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Link does not match the pattern https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'syn2580853', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 791\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 541 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-dbb7ce156f0502589129b8c5a1b84f5e23b7555b343486571541e789e89f4a15\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 541 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-9329cf9b-2f36-4a0b-9ef7-45f3cdb7128d\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"syn2580853\\\",\\n        \\\"data_repository\\\": \\\"Synapse AD Knowledge Portal\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.synapse.org/#!Synapse:syn2580853\\\"\\n    }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"syn2580853\\\",\\n        \\\"data_repository\\\": \\\"Synapse AD Knowledge Portal\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.synapse.org/#!Synapse:syn2580853\\\"\\n    }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755540875,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 712,\n",
      "        \"completion_tokens\": 76,\n",
      "        \"total_tokens\": 788,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn2580853', 'data_repository': 'Synapse AD Knowledge Portal', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn2580853', 'dataset_id': 'syn2580853'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: syn2580853\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID syn2580853 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: Synapse AD Knowledge Portal, identifier: syn2580853\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn2580853 for Synapse AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn2580853 for Synapse AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn2580853 for Synapse AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn2580853 for Synapse AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1043 - INFO - Found partial match for Synapse AD Knowledge Portal in Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.synapse.org/#!Synapse:syn2580853, repo synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.synapse.org/#!Synapse:syn2580853\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Link does not match the pattern https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'syn2580853', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 377 - INFO - datasets (<class 'list'>): [{'dataset_identifier': 'syn2580853', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn2580853', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'syn2580853', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'syn2580853', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 385 - INFO - Final ret data availability: 2 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 386 - INFO - Final ret data availability: [{'dataset_identifier': 'syn2580853', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'syn2580853', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1079 - INFO - Fetching metadata for 2 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: syn2580853\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn2580853\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn2580853 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn2580853 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn2580853 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn2580853 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for syn2580853 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn2580853\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.synapse.org/Synapse:syn2580853\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: syn2580853\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn2580853\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn2580853 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn2580853 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn2580853 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn2580853 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for syn2580853 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn2580853\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.synapse.org/Synapse:syn2580853\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1156 - INFO - Updated datasets len: 2\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 221 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 12, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'source_section', 'retrieval_pattern', 'access_mode', 'link',\n",
      "       'source_url', 'download_link', 'title', 'content_type', 'id', 'caption',\n",
      "       'description', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 456 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 462 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 559 - INFO - \n",
      "Progress: 1/25 (4.00%) | Elapsed: 00:00:07 | ETA: 00:02:57\n",
      "\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 539 - INFO - 1th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 347 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 354 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 533 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC4318527&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 383 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b02f2a10>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 375 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 310 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 399 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 413 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 439 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 23 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 5 <a> elements with href='#Tab1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 5 <a> elements with href='#Tab1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 5 <a> elements with href='#Tab1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 5 <a> elements with href='#Tab1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 5 <a> elements with href='#Tab1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 5 <a> elements with href='#Tab1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 5 <a> elements with href='#Tab1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 5 <a> elements with href='#Tab1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 5 <a> elements with href='#Tab1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 5 <a> elements with href='#Tab1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 5 <a> elements with href='#Tab1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 5 <a> elements with href='#Tab1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 5 <a> elements with href='#Tab1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 5 <a> elements with href='#Tab1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 5 <a> elements with href='#Tab1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 5 <a> elements with href='#Tab1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1173) for Tab1: Indeed, as the use of computational methods in biomedical research becomes widespread, the need for appropriate benchmarking projects, especially those involving community participation, is substantially growing (Table 1 [href=this file] ).\n",
      "Challenge-based benchmarking efforts, such as CASP [ 8 [href=#CR8] – 10 [href=#CR10] ], CAFA [ 11 [href=#CR11] ] and DREAM [ 12 [href=#CR12] , 13 [href=#CR13] ], among others (Table 1 [href=this file] ), offer a robust framework for algorithm evaluation.\n",
      "Several non-cancer-focused challenge-based benchmarking efforts are on-going, including the Assemblathon benchmarking of sequence assembly algorithms [ 23 [href=#CR23] ] and the CLARITY Challenge for standardizing clinical genome sequencing analysis and reporting [ 24 [href=#CR24] ] (Table 1 [href=this file] ).\n",
      "For example, the ICGC-TCGA DREAM SMC Challenge [ 28 [href=#CR28] ] (Table 1 [href=this file] ) involved transmitting over 10 TB of data to every contestant, so that each had a copy of the 15 tumor-normal whole-genome pairs.\n",
      "Several groups are developing the expertise to facilitate this process, including CASP, DREAM, CAFA and others (Table 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#Bib1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for Bib1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#Bib1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for Bib1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#Bib1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for Bib1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#Bib1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for Bib1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 384 - INFO - Chunking the HTML content for the parsing step.\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 172 - INFO - Extracting data availability elements from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: section.data-availability-statement\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: section.associated-data\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Data Availability')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: h3.pmc_sec_title:-soup-contains('Data Availability')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Associated Data')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Footnotes')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 221 - INFO - Found 1 data availability elements from HTML.\u001B[0m\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "Batches:   0%|          | 0/1 [00:00<?, ?it/s]"
      ],
      "application/vnd.jupyter.widget-view+json": {
       "version_major": 2,
       "version_minor": 0,
       "model_id": "e2cdd5d30a40491a8cc8b85310372824"
      }
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001B[97membeddings_retriever.py - line 51 - INFO - Searching for top-5 passages similar to the query by embeddings.\u001B[0m\n",
      "\u001B[97membeddings_retriever.py - line 33 - INFO - Computing L2 distances using numpy.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 11190\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 3140 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-678603b6c19899f8b8d8ef99f88185656fbc435a0ec6964302e3e5f1cd2d83e4\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 3140 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-35cfc3e9-1545-4a62-9579-2487282f84eb\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"ICGC-TCGA DREAM SMC Challenge\\\",\\n        \\\"data_repository\\\": \\\"ICGC\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.synapse.org/#!Synapse:syn2324344\\\"\\n    }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"ICGC-TCGA DREAM SMC Challenge\\\",\\n        \\\"data_repository\\\": \\\"ICGC\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.synapse.org/#!Synapse:syn2324344\\\"\\n    }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755540890,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 2629,\n",
      "        \"completion_tokens\": 73,\n",
      "        \"total_tokens\": 2702,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'ICGC-TCGA DREAM SMC Challenge', 'data_repository': 'ICGC', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn2324344', 'dataset_id': 'ICGC-TCGA DREAM SMC Challenge'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: ICGC-TCGA DREAM SMC Challenge\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 940 - INFO - Accession ID ICGC-TCGA DREAM SMC Challenge is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: ICGC, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier n/a for ICGC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier n/a for ICGC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier n/a for ICGC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier n/a for ICGC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier n/a for ICGC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: https://www.synapse.org/#!Synapse:syn2324344\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.synapse.org/#!Synapse:syn2324344, repo www.synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.synapse.org/#!Synapse:syn2324344\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 972 - INFO - Repository www.synapse.org not found in ontology\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'www.synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1079 - INFO - Fetching metadata for 1 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1095 - INFO - Skipping dataset 1: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1156 - INFO - Updated datasets len: 1\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 24, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage', 'link',\n",
      "       'title', 'file_info', 'description', 'source_section', 'section_class',\n",
      "       'download_link', 'file_extension', 'a_attr_href', 'a_attr_class',\n",
      "       'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 450 - INFO - Parsed data extraction completed. Elements collected: 2\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 456 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 466 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 539 - INFO - 2th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 347 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 354 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 533 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC4339277&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 383 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b03dbb90>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 375 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 310 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 399 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 413 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 439 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 7 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#S9'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for S9: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#S9'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for S9: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#S12'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for S12: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 18 <a> elements with href='#SD1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 4802) for SD1: Moreover, samples in the paired-end group had many more mappable reads than those in the single-end group ( Table 1 [href=#T1] , Supplementary Fig. 1 [href=this file] ). Indeed, the two groups showed distinct global patterns of pseudogene expression ( Supplementary Fig. 2 [href=this file] ). For each cancer type, we observed generally weak correlations between the expression level of pseudogenes and their WT genes, which is consistent with the previous study ( Supplementary Fig. 3 [href=this file] ). As observed in Kalyana-Sundaram et al. (2012) , we detected some tumor-lineage-specific pseudogenes (296 from the paired-end group and 41 from the single-end group, Supplementary Fig. 4 [href=this file] ). In addition, for three cancer types with available RNA-seq data from nontumor tissue samples, we identified differentially expressed pseudogenes between tumor and nontumor samples (54 in BRCA, 110 in KIRC and 138 in LUSC, Supplementary Fig. 5 [href=this file] ).\n",
      "For several cancer types with established tumor subtypes, we performed the supervised analysis and revealed substantial numbers of pseudogenes with significant differential expression: 48 in UCEC (endometrioid vs. serous) , 138 in LUSC (basal, classical, primitive and secretory) , 71 in GBM (classical, mesenchymal, neural and proneural) and 547 in BRCA (PAM50 subtypes: luminal A, luminal B, basal-like, Her2-enriched and normal-like) (Methods, Fig. 2a [href=#F2] , Supplementary Data 1 [href=this file] ).\n",
      "The predictive power of pseudogene expression is comparable to those achieved by the mRNA expression profiles, suggesting that both pseudogene and mRNA expression can classify the UCEC subtypes independently ( Supplementary Fig. 6 [href=this file] ).\n",
      "Based on the NMF consensus clustering, 837 BRCA samples can be classified into three distinct subtypes: subtype 1 ( = 144), subtype 2 ( = 390), and subtype 3 ( = 303) ( Fig. 4b [href=#F4] , Supplementary Data 2 [href=this file] ).\n",
      "Based on the 500 pseudogenes with the most variable expression, we were able to classify 446 KIRC samples into two distinct subtypes ( Fig. 5a [href=#F5] , Supplementary Data 3 [href=this file] ). We observed an enrichment of pseudogenes (115 out of 500) with a statistically significant P-value ( < 0.05) ( Fig. 5c [href=#F5] , Supplementary Data 4 [href=this file] ).\n",
      "To further assess the clinical utility of the observed pseudogene-expression subtypes, we classified the KIRC samples into four risk quartiles based on the risk scores (in terms of overall survival) calculated from the multivariate Cox model, employing only clinical variables: low risk group (Q1, = 110), low-medium risk group (Q2, = 111), medium-high risk group (Q3, = 112), and high risk group (Q4, = 112) (Methods, Supplementary Data 3 [href=this file] ).\n",
      "To gain some mechanistic insight into how expressed pseudogenes contribute to the observed KIRC pseudogene-expression subtypes, we performed a systematic analysis ( Supplementary Fig. 9a and Supplementary Data 5 [href=this file] ). Among the WT cognate genes with strong positive correlations with their pseudogenes, we noticed that the survival correlations of individual WT genes with prognostic value match the survival pattern of the pseudogene-expression subtypes: WT genes with better prognosis (potentially tumor suppressors, hazard ratio < 1) show higher expression levels in subtype 1 (the better survival group) and the genes with worse prognosis (potentially oncogenes, hazard ratio > 1) show higher expression levels in subtype 2 (the worse survival group, Supplementary Fig. 8b [href=this file] ). Finally, we examined the classic miRNA decoy model as proposed in Poliseno et al. (2010) and identified 38 such candidates ( Methods and Supplementary Data 5 [href=this file] ). One candidate of interest is the potential regulation of a putative tumor suppressor α-catenin (CTNNA1) by the pseudogene PGOHUM00000257111 through competition for up to 9 shared miRNA regulators ( Supplementary Data 5 [href=this file] ). Indeed, the expression levels of PGOHUM00000257111 were significantly higher in cluster 1 (t-test P = 1.48×10-7), which may lead to the elevated levels of CTNNA1 in subtype 1 ( Supplementary Fig. 8b [href=this file] ) and therefore better survival.\n",
      "Finally, we performed feature selection ( Supplementary Data 6 [href=this file] ) and built the classifier from the whole training set using the best-performing algorithm (with the highest AUC) identified through the cross-validation, and applied it to the test set in order to independently validate the predictive power.\n",
      "We gratefully acknowledge contributions from the TCGA Research Network and its TCGA Pan-Cancer Analysis Working Group (contributing consortium members are listed in Supplementary Note 1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#SD2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for SD2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 384 - INFO - Chunking the HTML content for the parsing step.\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 172 - INFO - Extracting data availability elements from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: section.data-availability-statement\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: section.associated-data\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Data Availability')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: h3.pmc_sec_title:-soup-contains('Data Availability')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Associated Data')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Footnotes')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 221 - INFO - Found 3 data availability elements from HTML.\u001B[0m\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "Batches:   0%|          | 0/1 [00:00<?, ?it/s]"
      ],
      "application/vnd.jupyter.widget-view+json": {
       "version_major": 2,
       "version_minor": 0,
       "model_id": "1943a8c493a449a9adf4774e7ef8b09f"
      }
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001B[97membeddings_retriever.py - line 51 - INFO - Searching for top-5 passages similar to the query by embeddings.\u001B[0m\n",
      "\u001B[97membeddings_retriever.py - line 33 - INFO - Computing L2 distances using numpy.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 7567\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 2235 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-3f379af441fd859374d95ba41e23e2d5704948dc9c41671b1fedfbfc68d80e6d\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 2235 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-76875d85-d75a-4096-955a-01b06b8190cc\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"TCGA RNA-seq\\\",\\n        \\\"data_repository\\\": \\\"TCGA\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.cancer.gov/tcga\\\"\\n    ]\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"TCGA RNA-seq\\\",\\n        \\\"data_repository\\\": \\\"TCGA\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.cancer.gov/tcga\\\"\\n    ]\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755540903,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 2895,\n",
      "        \"completion_tokens\": 59,\n",
      "        \"total_tokens\": 2954,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'TCGA RNA-seq', 'data_repository': 'TCGA', 'dataset_webpage': 'https://www.cancer.gov/tcga', 'dataset_id': 'TCGA RNA-seq'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: TCGA RNA-seq\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 940 - INFO - Accession ID TCGA RNA-seq is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: TCGA, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier n/a for TCGA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier n/a for TCGA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier n/a for TCGA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier n/a for TCGA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier n/a for TCGA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: https://www.cancer.gov/tcga\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.cancer.gov/tcga, repo www.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.cancer.gov/ccg/research/genome-sequencing/tcga\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 972 - INFO - Repository www.cancer.gov not found in ontology\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'www.cancer.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1079 - INFO - Fetching metadata for 1 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1095 - INFO - Skipping dataset 1: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1156 - INFO - Updated datasets len: 1\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 8, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage', 'link',\n",
      "       'title', 'file_info', 'description', 'source_section', 'section_class',\n",
      "       'download_link', 'file_extension', 'a_attr_href', 'a_attr_class',\n",
      "       'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 450 - INFO - Parsed data extraction completed. Elements collected: 4\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 456 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 466 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 539 - INFO - 3th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC8452114)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 347 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC8452114\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 354 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 533 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC8452114&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 383 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC8452114.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b5996c50>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 375 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 310 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 399 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC8452114 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 413 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 439 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC8452114, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC8452114)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 8 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 2 <a> elements with href='#SD1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 343) for SD1: We obtained a total of six coexpression modules ( Figure S1 [href=this file] ) in amygdala with the number of genes in each module ranging from 204 (red module) to 3,049 (turquoise). After eigengene extraction and differential analysis, no modules were differentially expressed between COVID-19+ and CTL in AMY ( Figure S1 [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 2 <a> elements with href='#SD2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 224) for SD2: Relationships between these modules are reported in Figure S2 [href=this file] . Interestingly, the two synaptic modules were not related to each other ( Figure S2 [href=this file] ) despite a similar functional enrichment.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#SD3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 75) for SD3: The top genes for AMY and OBT are reported in Figure S3 [href=this file] .\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 10 <a> elements with href='#SD4'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 1320) for SD4: The top 9 genes for the two regions are reported in Figure 2A [href=#F2] and 2B [href=#F2] , whereas the complete list of DEGs is reported in Table S1 [href=this file] and S2 [href=this file] . Across downregulated genes we observed an enrichment of synaptic and neuronal pathways ( Figure 3 [href=#F3] ; Table S3 [href=this file] ). Finally, synaptic pathways were enriched within downregulated genes ( Figure 3 [href=#F3] ; Table S4 [href=this file] ).\n",
      "Other modules were enriched for development/angiogenesis (green, upregulated, and enriched for End and Per), immune system (tan, upregulated), and synaptic signaling (green-yellow and turquoise, both downregulated and enriched for neuronal cell genes) ( Table S5 [href=this file] ).\n",
      "The only differential pathways we detected were related to platelet activation processes/hemostasis, which was only detected in the olfactory bulb ( Figure 7 [href=#F7] ; Table S6 [href=this file] and S7 [href=this file] ).\n",
      "We detected a total of 4,275 genes only associated with pseudotime in amygdala and 3,102 only associated with pseudotime in olfactory bulb ( Table S8 [href=this file] and Table S9 [href=this file] , respectively).\n",
      "It is important to stress that the two modules are not related (see Table S2 [href=this file] ) representing two distinct biological processes.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#S12'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 34) for S12: ACKNOWLEDGEMENTS [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#S12'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 34) for S12: ACKNOWLEDGEMENTS [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 384 - INFO - Chunking the HTML content for the parsing step.\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 172 - INFO - Extracting data availability elements from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: section.data-availability-statement\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: section.associated-data\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Data Availability')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: h3.pmc_sec_title:-soup-contains('Data Availability')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Associated Data')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Footnotes')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 221 - INFO - Found 2 data availability elements from HTML.\u001B[0m\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "Batches:   0%|          | 0/1 [00:00<?, ?it/s]"
      ],
      "application/vnd.jupyter.widget-view+json": {
       "version_major": 2,
       "version_minor": 0,
       "model_id": "47a57b8ca1804f9480913616cc49f3aa"
      }
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001B[97membeddings_retriever.py - line 51 - INFO - Searching for top-5 passages similar to the query by embeddings.\u001B[0m\n",
      "\u001B[97membeddings_retriever.py - line 33 - INFO - Computing L2 distances using numpy.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 15761\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 4283 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-0d1809bd3637f979764db2d4bc8a11d9cc3505c63384e8592cc98c8a4acd71ea\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 4283 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-d6020166-4ef2-4534-8027-dbd2a85f5e0e\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"[\\n    {\\n        \\\"dataset_identifier\\\": \\\"syn18485175\\\",\\n        \\\"data_repository\\\": \\\"Accelerated Medicine Partnership-AD portal (AMP-AD)\\\",\\n        \\\"dataset_webpage\\\": \\\"https://adknowledgeportal.org/explore/synapse/syn18485175\\\"\\n    }\\n]\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"[\\n    {\\n        \\\"dataset_identifier\\\": \\\"syn18485175\\\",\\n        \\\"data_repository\\\": \\\"Accelerated Medicine Partnership-AD portal (AMP-AD)\\\",\\n        \\\"dataset_webpage\\\": \\\"https://adknowledgeportal.org/explore/synapse/syn18485175\\\"\\n    }\\n]\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755540918,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 5818,\n",
      "        \"completion_tokens\": 79,\n",
      "        \"total_tokens\": 5897,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn18485175', 'data_repository': 'Accelerated Medicine Partnership-AD portal (AMP-AD)', 'dataset_webpage': 'https://adknowledgeportal.org/explore/synapse/syn18485175', 'dataset_id': 'syn18485175'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: syn18485175\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID syn18485175 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: Accelerated Medicine Partnership-AD portal (AMP-AD), identifier: syn18485175\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn18485175 for Accelerated Medicine Partnership-AD portal  AMP-AD \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn18485175 for Accelerated Medicine Partnership-AD portal  AMP-AD \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn18485175 for Accelerated Medicine Partnership-AD portal  AMP-AD \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn18485175 for Accelerated Medicine Partnership-AD portal  AMP-AD \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier syn18485175 for Accelerated Medicine Partnership-AD portal  AMP-AD \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1051 - INFO - Found id_pattern match for Accelerated Medicine Partnership-AD portal  AMP-AD  in (?<!\\d)syn\\d{7,12}(?!\\d)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://adknowledgeportal.org/explore/synapse/syn18485175, repo synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://adknowledgeportal.synapse.org/explore/synapse/syn18485175\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Link does not match the pattern https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'syn18485175', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1079 - INFO - Fetching metadata for 1 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: syn18485175\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn18485175\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn18485175 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn18485175 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn18485175 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn18485175 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for syn18485175 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn18485175\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.synapse.org/Synapse:syn18485175\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1156 - INFO - Updated datasets len: 1\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 9, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 450 - INFO - Parsed data extraction completed. Elements collected: 6\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 456 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 466 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 539 - INFO - 4th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC9641692)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 347 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC9641692\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 354 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 533 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC9641692&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 395 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC9641692.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 399 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC9641692 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 413 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 417 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC9641692, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 3 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b0fcc900>, <Element supplementary-material at 0x1b10b7d80>, <Element supplementary-material at 0x1b10b5800>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b0fcc900>, <Element supplementary-material at 0x1b10b7d80>, <Element supplementary-material at 0x1b10b5800>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM1: When evaluated on cell-type level, TAPE has the best performance of MAE, and comparable CCC with relatively small variance on real datasets (Supplementary Table 8 [this file] ). \n",
      "Meanwhile, we find that deep learning methods are robust for the cross-protocol prediction, while the performance of some statistical methods will drop if the reference and pseudo-bulk data type (UMI to counts or counts to UMI) are exchanged (Supplementary Figs. 1 [this file] and 2 [this file] ). \n",
      "Detailed comparison results are available in Supplementary Table 8 [this file] . For the “similar” scenario, we investigated TAPE’s performance on distinguishing similar cell types (CD4 T cell and CD8 T cell) in all the three PBMC datasets (Supplementary Fig. 3 [this file] ). With 13 defined cell subtypes (Supplementary Table 1 [this file] ), all the methods can not achieve satisfying results (Supplementary Fig. 4 [this file] , median CCC < 0.1), which clearly shows the common limitation of current methods. \n",
      "In Supplementary Fig. 6 [this file] , the two colors stand for the different methods, and the two coordinates of each dot represent overall MAE and CCC for all cell types respectively. \n",
      "From the results (Supplementary Fig. 10 [this file] ), we can have four conclusions: (1) TAPE’s predictive power is better than CIBERSORTx when the randomly selected DEGs are less than 1000; (2) both methods can achieve good performance when the DEGs are signature genes and there are not any similar cell types; (3) both methods can not distinguish DEGs from CD8 T cell rather than CD4 T cell if the DEGs are randomly selected; (4) CIBERSORTx is better than TAPE if the DEGs are signature genes and there exist similar cell types. \n",
      "The results show that TAPE only predicts and as DEGs in NK cell and no DEGs for other cell types (Supplementary Fig. 8 [this file] ). \n",
      "Firstly, when we study the correlation at the gene level using “overall” mode (Supplementary Fig. 7 [this file] ), about 30% of the predicted genes have the negative correlation. However, this phenomenon can also be observed in CIBERSORTx (Supplementary Fig. 8 [this file] ), which indicates the information loss between bulk samples and inferred cell-type-specific GEP is hard to be reconstructed. Thirdly, we notice that both CIBERSORTx and our method can not distinguish DEGs from similar cell subtypes correctly if the DEGs are not signature genes (Supplementary Fig. 9 [this file] ) which means that their resolution is still limited. But CIBERSORTx has displayed its advantages in distinguishing signature DEGs from similar cell types because of the incorporation of the signature matrix (Supplementary Fig. 10 [this file] ). \n",
      "When we used it to test whether deconvolution methods can achieve good performance with immune cell subtypes, we merged all 30 cell types into 13 cell subtypes (Supplementary Table 1 [this file] ). \n",
      "This step is very important, because TAPE will fail in predicting test bulk data proportions properly without proper filtering (Supplementary Table 2 [this file] and Supplementary Fig. 11 [this file] ) . If the less variable genes are not filtered out, TAPE can not predict a good result because of the noises (Supplementary Table 2 [this file] and Supplementary Fig. 11 [this file] ). \n",
      "Hyperparameter tuning file is available in Supplementary Tables 3 [this file] – 7 [this file] . \n",
      "All other relevant data supporting the key findings of this study are available within the article and its Supplementary Information [this file] files or from the corresponding author upon reasonable request. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM2: Peer reviewer reports [this file] are available. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM3: Further information on research design is available in the Nature Portfolio Reporting Summary [this file] linked to this article. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 123 - INFO - Loading patterns for section 'data_availability_sections' for publisher 'PMC'.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 131 - INFO - Section pattern keys for publisher PMC: dict_keys(['data_availability_sections', 'supplementary_material_sections', 'supplementary_data_sections', 'key_resources_table', 'methods_sections'])\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 133 - INFO - Section name: data_availability_sections\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 135 - INFO - Found section 'data_availability_sections' in patterns for publisher 'PMC'.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 137 - INFO - Loaded patterns for section 'data_availability_sections': [\".//sec[@sec-type='data-availability']\", \".//notes[@notes-type='data-availability']\", \".//notes[@notes-type='data-availability']\"]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_data_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 706 - INFO - Found 4 supplementary data sections\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(key_resources_table)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "Batches:   0%|          | 0/1 [00:00<?, ?it/s]"
      ],
      "application/vnd.jupyter.widget-view+json": {
       "version_major": 2,
       "version_minor": 0,
       "model_id": "0926dc2c254649ec9d775c1e1626ba30"
      }
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001B[97membeddings_retriever.py - line 51 - INFO - Searching for top-2 passages similar to the query by embeddings.\u001B[0m\n",
      "\u001B[97membeddings_retriever.py - line 33 - INFO - Computing L2 distances using numpy.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 364 - INFO - Processing DAS_content: 4 elements\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 636\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 502 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-a1daa66051aa6c2238327c5ee43770a68d03a341de59eb9dab6e587b586a8e68\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 502 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-249127ac-3d73-4156-b2c2-bcd80026b755\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"syn3219045\\\",\\n        \\\"data_repository\\\": \\\"Synapse\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.synapse.org/#!Synapse:syn3219045\\\"\\n    ]\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"syn3219045\\\",\\n        \\\"data_repository\\\": \\\"Synapse\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.synapse.org/#!Synapse:syn3219045\\\"\\n    ]\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755540924,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 585,\n",
      "        \"completion_tokens\": 73,\n",
      "        \"total_tokens\": 658,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn3219045', 'data_repository': 'Synapse', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn3219045', 'dataset_id': 'syn3219045'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: syn3219045\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID syn3219045 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: Synapse, identifier: syn3219045\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3219045 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn3219045 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn3219045 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3219045 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1037 - INFO - Found repo_name match for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.synapse.org/#!Synapse:syn3219045, repo synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.synapse.org/#!Synapse:syn3219045\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Link does not match the pattern https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'syn3219045', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 383\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 439 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-83fbae0733fc1b4416ef197c6270240a777dae87282763bd215aef4a4613ca94\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 439 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-3f054229-5e3e-486e-b12d-7cc3dac377c3\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"TAPE\\\",\\n        \\\"data_repository\\\": \\\"GitHub\\\",\\n        \\\"dataset_webpage\\\": \\\"https://github.com/poseidonchan/TAPE\\\"\\n    ]\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"TAPE\\\",\\n        \\\"data_repository\\\": \\\"GitHub\\\",\\n        \\\"dataset_webpage\\\": \\\"https://github.com/poseidonchan/TAPE\\\"\\n    ]\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755540925,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 542,\n",
      "        \"completion_tokens\": 57,\n",
      "        \"total_tokens\": 599,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'TAPE', 'data_repository': 'GitHub', 'dataset_webpage': 'https://github.com/poseidonchan/TAPE', 'dataset_id': 'TAPE'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: TAPE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID TAPE is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: GitHub, identifier: TAPE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier TAPE for GitHub\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier TAPE for GitHub\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier TAPE for GitHub\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier TAPE for GitHub\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier TAPE for GitHub\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: https://github.com/poseidonchan/TAPE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://github.com/poseidonchan/TAPE, repo github.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://github.com/poseidonchan/TAPE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 970 - INFO - Dataset ID TAPE found in resolved dataset page https://github.com/poseidonchan/TAPE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'TAPE', 'data_repository': 'github.com', 'dataset_webpage': 'https://github.com/poseidonchan/TAPE'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 62752\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 16031 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-f73369a7a1b2a03de551c6cc1454e142d09ec6ab8172a270327cb0be3cc233c1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 16031 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-56d214f7-8a1e-4223-bd9d-446fa0e067b0\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE107011\\\",\\n    \\\"data_repository\\\": \\\"GEO\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE107011\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE65133\\\",\\n    \\\"data_repository\\\": \\\"GEO\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65133\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE87544\\\",\\n    \\\"data_repository\\\": \\\"GEO\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87544\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE67835\\\",\\n    \\\"data_repository\\\": \\\"GEO\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE67835\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE84133\\\",\\n    \\\"data_repository\\\": \\\"GEO\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84133\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE115449\\\",\\n    \\\"data_repository\\\": \\\"GEO\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE115449\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE119117\\\",\\n    \\\"data_repository\\\": \\\"GEO\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE119117\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE107011\\\",\\n    \\\"data_repository\\\": \\\"GEO\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE107011\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE65133\\\",\\n    \\\"data_repository\\\": \\\"GEO\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65133\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE87544\\\",\\n    \\\"data_repository\\\": \\\"GEO\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87544\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE67835\\\",\\n    \\\"data_repository\\\": \\\"GEO\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE67835\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE84133\\\",\\n    \\\"data_repository\\\": \\\"GEO\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84133\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE115449\\\",\\n    \\\"data_repository\\\": \\\"GEO\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE115449\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE119117\\\",\\n    \\\"data_repository\\\": \\\"GEO\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE119117\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755540929,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 20251,\n",
      "        \"completion_tokens\": 511,\n",
      "        \"total_tokens\": 20762,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE107011', 'data_repository': 'GEO', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE107011', 'dataset_id': 'GSE107011'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: GSE107011\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID GSE107011 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: GEO, identifier: GSE107011\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE107011 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE107011 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE107011, repo geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE107011\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 961 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'GSE107011', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE107011'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE65133', 'data_repository': 'GEO', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65133', 'dataset_id': 'GSE65133'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: GSE65133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID GSE65133 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: GEO, identifier: GSE65133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE65133 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE65133 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65133, repo geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 961 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'GSE65133', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65133'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE87544', 'data_repository': 'GEO', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87544', 'dataset_id': 'GSE87544'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: GSE87544\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID GSE87544 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: GEO, identifier: GSE87544\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE87544 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE87544 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87544, repo geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87544\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 961 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'GSE87544', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87544'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE67835', 'data_repository': 'GEO', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE67835', 'dataset_id': 'GSE67835'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: GSE67835\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID GSE67835 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: GEO, identifier: GSE67835\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE67835 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE67835 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE67835, repo geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE67835\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 961 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'GSE67835', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE67835'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE84133', 'data_repository': 'GEO', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84133', 'dataset_id': 'GSE84133'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: GSE84133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID GSE84133 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: GEO, identifier: GSE84133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE84133 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE84133 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84133, repo geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 961 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'GSE84133', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84133'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE115449', 'data_repository': 'GEO', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE115449', 'dataset_id': 'GSE115449'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: GSE115449\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID GSE115449 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: GEO, identifier: GSE115449\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE115449 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE115449 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE115449, repo geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE115449\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 961 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'GSE115449', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE115449'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE119117', 'data_repository': 'GEO', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE119117', 'dataset_id': 'GSE119117'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: GSE119117\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID GSE119117 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: GEO, identifier: GSE119117\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE119117 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE119117 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE119117, repo geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE119117\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 961 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'GSE119117', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE119117'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 7408\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 2195 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-bc5e76f7dcb541c53f40b7e9a7900b75d3d53404eda0812f861cf84d134ef03f\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 2195 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-de9c0d53-d399-4ff2-9b5f-076d88d1bedc\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE107011\\\",\\n        \\\"data_repository\\\": \\\"GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE107011\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE65133\\\",\\n        \\\"data_repository\\\": \\\"GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65133\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE87544\\\",\\n        \\\"data_repository\\\": \\\"GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87544\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE67835\\\",\\n        \\\"data_repository\\\": \\\"GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE67835\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE84133\\\",\\n        \\\"data_repository\\\": \\\"GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84133\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE115449\\\",\\n        \\\"data_repository\\\": \\\"GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE115449\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE119117\\\",\\n        \\\"data_repository\\\": \\\"GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE119117\\\"\\n    }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE107011\\\",\\n        \\\"data_repository\\\": \\\"GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE107011\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE65133\\\",\\n        \\\"data_repository\\\": \\\"GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65133\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE87544\\\",\\n        \\\"data_repository\\\": \\\"GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87544\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE67835\\\",\\n        \\\"data_repository\\\": \\\"GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE67835\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE84133\\\",\\n        \\\"data_repository\\\": \\\"GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84133\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE115449\\\",\\n        \\\"data_repository\\\": \\\"GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE115449\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE119117\\\",\\n        \\\"data_repository\\\": \\\"GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE119117\\\"\\n    }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755540938,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 2579,\n",
      "        \"completion_tokens\": 511,\n",
      "        \"total_tokens\": 3090,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE107011', 'data_repository': 'GEO', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE107011', 'dataset_id': 'GSE107011'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: GSE107011\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID GSE107011 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: GEO, identifier: GSE107011\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE107011 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE107011 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE107011, repo geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE107011\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 961 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'GSE107011', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE107011'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE65133', 'data_repository': 'GEO', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65133', 'dataset_id': 'GSE65133'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: GSE65133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID GSE65133 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: GEO, identifier: GSE65133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE65133 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE65133 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65133, repo geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 961 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'GSE65133', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65133'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE87544', 'data_repository': 'GEO', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87544', 'dataset_id': 'GSE87544'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: GSE87544\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID GSE87544 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: GEO, identifier: GSE87544\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE87544 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE87544 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87544, repo geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87544\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 961 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'GSE87544', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87544'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE67835', 'data_repository': 'GEO', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE67835', 'dataset_id': 'GSE67835'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: GSE67835\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID GSE67835 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: GEO, identifier: GSE67835\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE67835 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE67835 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE67835, repo geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE67835\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 961 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'GSE67835', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE67835'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE84133', 'data_repository': 'GEO', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84133', 'dataset_id': 'GSE84133'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: GSE84133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID GSE84133 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: GEO, identifier: GSE84133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE84133 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE84133 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84133, repo geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 961 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'GSE84133', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84133'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE115449', 'data_repository': 'GEO', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE115449', 'dataset_id': 'GSE115449'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: GSE115449\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID GSE115449 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: GEO, identifier: GSE115449\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE115449 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE115449 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE115449, repo geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE115449\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 961 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'GSE115449', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE115449'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE119117', 'data_repository': 'GEO', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE119117', 'dataset_id': 'GSE119117'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: GSE119117\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID GSE119117 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: GEO, identifier: GSE119117\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE119117 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE119117 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE119117, repo geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE119117\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 961 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'GSE119117', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE119117'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 377 - INFO - datasets (<class 'list'>): [{'dataset_identifier': 'syn3219045', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'TAPE', 'data_repository': 'github.com', 'dataset_webpage': 'https://github.com/poseidonchan/TAPE'}, {'dataset_identifier': 'GSE107011', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE107011'}, {'dataset_identifier': 'GSE65133', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65133'}, {'dataset_identifier': 'GSE87544', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87544'}, {'dataset_identifier': 'GSE67835', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE67835'}, {'dataset_identifier': 'GSE84133', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84133'}, {'dataset_identifier': 'GSE115449', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE115449'}, {'dataset_identifier': 'GSE119117', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE119117'}, {'dataset_identifier': 'GSE107011', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE107011'}, {'dataset_identifier': 'GSE65133', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65133'}, {'dataset_identifier': 'GSE87544', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87544'}, {'dataset_identifier': 'GSE67835', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE67835'}, {'dataset_identifier': 'GSE84133', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84133'}, {'dataset_identifier': 'GSE115449', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE115449'}, {'dataset_identifier': 'GSE119117', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE119117'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'syn3219045', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'TAPE', 'data_repository': 'github.com', 'dataset_webpage': 'https://github.com/poseidonchan/TAPE'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'GSE107011', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE107011'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'GSE65133', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65133'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'GSE87544', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87544'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'GSE67835', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE67835'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'GSE84133', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84133'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'GSE115449', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE115449'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'GSE119117', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE119117'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'GSE107011', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE107011'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'GSE65133', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65133'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'GSE87544', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87544'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'GSE67835', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE67835'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'GSE84133', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84133'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'GSE115449', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE115449'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'GSE119117', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE119117'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 385 - INFO - Final ret data availability: 16 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 386 - INFO - Final ret data availability: [{'dataset_identifier': 'syn3219045', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'TAPE', 'data_repository': 'github.com', 'dataset_webpage': 'https://github.com/poseidonchan/TAPE', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'GSE107011', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE107011', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'GSE65133', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65133', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'GSE87544', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87544', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'GSE67835', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE67835', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'GSE84133', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84133', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'GSE115449', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE115449', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'GSE119117', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE119117', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'GSE107011', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE107011', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'GSE65133', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65133', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'GSE87544', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87544', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'GSE67835', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE67835', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'GSE84133', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84133', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'GSE115449', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE115449', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'GSE119117', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE119117', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1079 - INFO - Fetching metadata for 16 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: syn3219045\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn3219045\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3219045 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn3219045 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn3219045 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3219045 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for syn3219045 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn3219045\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.synapse.org/Synapse:syn3219045\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: TAPE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: github.com, identifier: TAPE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier TAPE for github.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier TAPE for github.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier TAPE for github.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier TAPE for github.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier TAPE for github.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: github.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 900 - WARNING - URL github.com may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for TAPE in github.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 2 with repo: github.com and accession_id: TAPE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1152 - WARNING - Repository github.com unknown in Ontology. Skipping dataset page 2.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: GSE107011\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: geo, identifier: GSE107011\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE107011 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE107011 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for GSE107011 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 3 with repo: geo and accession_id: GSE107011\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE107011\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 3: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: GSE65133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: geo, identifier: GSE65133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE65133 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE65133 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for GSE65133 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 4 with repo: geo and accession_id: GSE65133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 4: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: GSE87544\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: geo, identifier: GSE87544\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE87544 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE87544 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for GSE87544 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 5 with repo: geo and accession_id: GSE87544\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87544\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 5: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: GSE67835\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: geo, identifier: GSE67835\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE67835 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE67835 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for GSE67835 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 6 with repo: geo and accession_id: GSE67835\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE67835\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 6: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: GSE84133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: geo, identifier: GSE84133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE84133 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE84133 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for GSE84133 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 7 with repo: geo and accession_id: GSE84133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 7: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: GSE115449\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: geo, identifier: GSE115449\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE115449 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE115449 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for GSE115449 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 8 with repo: geo and accession_id: GSE115449\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE115449\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 8: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: GSE119117\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: geo, identifier: GSE119117\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE119117 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE119117 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for GSE119117 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 9 with repo: geo and accession_id: GSE119117\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE119117\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 9: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 10 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: GSE107011\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: geo, identifier: GSE107011\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE107011 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE107011 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for GSE107011 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 10 with repo: geo and accession_id: GSE107011\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE107011\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 10: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 11 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: GSE65133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: geo, identifier: GSE65133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE65133 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE65133 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for GSE65133 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 11 with repo: geo and accession_id: GSE65133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 11: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 12 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: GSE87544\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: geo, identifier: GSE87544\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE87544 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE87544 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for GSE87544 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 12 with repo: geo and accession_id: GSE87544\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87544\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 12: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 13 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: GSE67835\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: geo, identifier: GSE67835\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE67835 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE67835 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for GSE67835 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 13 with repo: geo and accession_id: GSE67835\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE67835\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 13: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 14 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: GSE84133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: geo, identifier: GSE84133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE84133 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE84133 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for GSE84133 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 14 with repo: geo and accession_id: GSE84133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84133\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 14: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 15 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: GSE115449\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: geo, identifier: GSE115449\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE115449 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE115449 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for GSE115449 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 15 with repo: geo and accession_id: GSE115449\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE115449\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 15: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 16 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: GSE119117\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: geo, identifier: GSE119117\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE119117 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE119117 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for GSE119117 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 16 with repo: geo and accession_id: GSE119117\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE119117\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 16: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1156 - INFO - Updated datasets len: 16\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 221 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 19, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'source_section', 'retrieval_pattern', 'access_mode', 'link',\n",
      "       'source_url', 'download_link', 'title', 'content_type', 'id', 'caption',\n",
      "       'description', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 456 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 462 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 539 - INFO - 5th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC5161470)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 347 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC5161470\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 354 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 533 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC5161470&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 383 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC5161470.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b5997310>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 375 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 310 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 399 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC5161470 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 413 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 439 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC5161470, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC5161470)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 19 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#pone.0167994.s001'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 95) for pone.0167994.s001: In both results the tagging SNPs are spread across the chromosome ( S1 Fig [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#pone.0167994.s002'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 207) for pone.0167994.s002: To compare the minimum numbers of the tag SNPs that were identified by the two methods to reach >95% coverage, we tested the difference of the tag SNP numbers using the t-test ( S1 Table [href=this file] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#pone.0167994.s003'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 87) for pone.0167994.s003: The comparison of the discovered tag SNP sets are given in S2 Table [href=this file] .\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 3 <a> elements with href='#pone.0167994.s004'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 284) for pone.0167994.s004: An example output corresponding to ENr113 genotypes results is given in S2 [href=this file] – S4 [href=#pone.0167994.s006] Figs and in the Supplemental Data ( S4 Table [href=#pone.0167994.s008] ) as well.\n",
      "Data corresponding to S2 [href=this file] – S4 [href=#pone.0167994.s006] Figs.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#pone.0167994.s005'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for pone.0167994.s005: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 3 <a> elements with href='#pone.0167994.s006'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 284) for pone.0167994.s006: An example output corresponding to ENr113 genotypes results is given in S2 [href=#pone.0167994.s004] – S4 [href=this file] Figs and in the Supplemental Data ( S4 Table [href=#pone.0167994.s008] ) as well.\n",
      "Data corresponding to S2 [href=#pone.0167994.s004] – S4 [href=this file] Figs.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#pone.0167994.s007'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 178) for pone.0167994.s007: The association of these three SNPs and the phenotype are significant ( S3 Table [href=this file] ), which suggests that the metaSNP method correctly tagged the designated SNPs.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#pone.0167994.s008'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 205) for pone.0167994.s008: An example output corresponding to ENr113 genotypes results is given in S2 [href=#pone.0167994.s004] – S4 [href=#pone.0167994.s006] Figs and in the Supplemental Data ( S4 Table [href=this file] ) as well.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 2 <a> elements with href='#pone.0167994.s009'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 49) for pone.0167994.s009: Details are given in S5 Table [href=this file] .\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#pone.0167994.s010'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 92) for pone.0167994.s010: The details of the mutual information estimation is given in the S1 File [href=this file] .\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 384 - INFO - Chunking the HTML content for the parsing step.\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 172 - INFO - Extracting data availability elements from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: section.data-availability-statement\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: section.associated-data\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Data Availability')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: h3.pmc_sec_title:-soup-contains('Data Availability')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Associated Data')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Footnotes')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 221 - INFO - Found 5 data availability elements from HTML.\u001B[0m\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "Batches:   0%|          | 0/2 [00:00<?, ?it/s]"
      ],
      "application/vnd.jupyter.widget-view+json": {
       "version_major": 2,
       "version_minor": 0,
       "model_id": "10d6eedb731f47f8ba3a5413de4a174f"
      }
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001B[97membeddings_retriever.py - line 51 - INFO - Searching for top-5 passages similar to the query by embeddings.\u001B[0m\n",
      "\u001B[97membeddings_retriever.py - line 33 - INFO - Computing L2 distances using numpy.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 27780\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 7288 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-4cc7998f1dd70bb145a4ddba422ed2dda5a21087e2dd65e76f7149c67ea48ebf\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 7288 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-32be28ea-4c3e-475c-8bbc-f3627595c8ff\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"1KGP\\\",\\n        \\\"data_repository\\\": \\\"1000 Genomes Project\\\",\\n        \\\"dataset_webpage\\\": \\\"http://www.internationalgenome.org/\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"HapMap\\\",\\n        \\\"data_repository\\\": \\\"HapMap\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.genome.gov/Funded-Projects/HapMap\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"ENCODE\\\",\\n        \\\"data_repository\\\": \\\"ENCODE\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.encodeproject.org/\\\"\\n    }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"1KGP\\\",\\n        \\\"data_repository\\\": \\\"1000 Genomes Project\\\",\\n        \\\"dataset_webpage\\\": \\\"http://www.internationalgenome.org/\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"HapMap\\\",\\n        \\\"data_repository\\\": \\\"HapMap\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.genome.gov/Funded-Projects/HapMap\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"ENCODE\\\",\\n        \\\"data_repository\\\": \\\"ENCODE\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.encodeproject.org/\\\"\\n    }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755540961,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 10340,\n",
      "        \"completion_tokens\": 156,\n",
      "        \"total_tokens\": 10496,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': '1KGP', 'data_repository': '1000 Genomes Project', 'dataset_webpage': 'http://www.internationalgenome.org/', 'dataset_id': '1KGP'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: 1KGP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID 1KGP is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: 1000 Genomes Project, identifier: 1KGP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 1KGP for 1000 Genomes Project\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 1KGP for 1000 Genomes Project\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier 1KGP for 1000 Genomes Project\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier 1KGP for 1000 Genomes Project\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier 1KGP for 1000 Genomes Project\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: http://www.internationalgenome.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: http://www.internationalgenome.org/, repo www.internationalgenome.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 981 - WARNING - Error resolving URL http://www.internationalgenome.org/: HTTPSConnectionPool(host='www.internationalgenome.org', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1006)')))\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 972 - INFO - Repository www.internationalgenome.org not found in ontology\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': '1KGP', 'data_repository': 'www.internationalgenome.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'HapMap', 'data_repository': 'HapMap', 'dataset_webpage': 'https://www.genome.gov/Funded-Projects/HapMap', 'dataset_id': 'HapMap'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: HapMap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID HapMap is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: HapMap, identifier: HapMap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier HapMap for HapMap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier HapMap for HapMap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier HapMap for HapMap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier HapMap for HapMap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier HapMap for HapMap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: https://www.genome.gov/Funded-Projects/HapMap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.genome.gov/Funded-Projects/HapMap, repo www.genome.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.genome.gov/Funded-Projects/HapMap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 970 - INFO - Dataset ID HapMap found in resolved dataset page https://www.genome.gov/Funded-Projects/HapMap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'HapMap', 'data_repository': 'www.genome.gov', 'dataset_webpage': 'https://www.genome.gov/Funded-Projects/HapMap'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'ENCODE', 'data_repository': 'ENCODE', 'dataset_webpage': 'https://www.encodeproject.org/', 'dataset_id': 'ENCODE'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: ENCODE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID ENCODE is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: ENCODE, identifier: ENCODE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier ENCODE for ENCODE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier ENCODE for ENCODE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier ENCODE for ENCODE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier ENCODE for ENCODE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier ENCODE for ENCODE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: https://www.encodeproject.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.encodeproject.org/, repo www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.encodeproject.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 972 - INFO - Repository www.encodeproject.org not found in ontology\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'ENCODE', 'data_repository': 'www.encodeproject.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1079 - INFO - Fetching metadata for 3 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: 1KGP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: www.internationalgenome.org, identifier: 1KGP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 1KGP for www.internationalgenome.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 1KGP for www.internationalgenome.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier 1KGP for www.internationalgenome.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier 1KGP for www.internationalgenome.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier 1KGP for www.internationalgenome.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: www.internationalgenome.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 900 - WARNING - URL www.internationalgenome.org may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for 1KGP in www.internationalgenome.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 1 with repo: www.internationalgenome.org and accession_id: 1KGP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1152 - WARNING - Repository www.internationalgenome.org unknown in Ontology. Skipping dataset page 1.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: HapMap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: www.genome.gov, identifier: HapMap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier HapMap for www.genome.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier HapMap for www.genome.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier HapMap for www.genome.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier HapMap for www.genome.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier HapMap for www.genome.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: www.genome.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 900 - WARNING - URL www.genome.gov may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for HapMap in www.genome.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 2 with repo: www.genome.gov and accession_id: HapMap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1152 - WARNING - Repository www.genome.gov unknown in Ontology. Skipping dataset page 2.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: ENCODE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: www.encodeproject.org, identifier: ENCODE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier ENCODE for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier ENCODE for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier ENCODE for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier ENCODE for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier ENCODE for www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 900 - WARNING - URL www.encodeproject.org may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for ENCODE in www.encodeproject.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 3 with repo: www.encodeproject.org and accession_id: ENCODE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1152 - WARNING - Repository www.encodeproject.org unknown in Ontology. Skipping dataset page 3.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1156 - INFO - Updated datasets len: 3\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 22, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage', 'link',\n",
      "       'title', 'file_info', 'description', 'source_section', 'section_class',\n",
      "       'download_link', 'file_extension', 'a_attr_href', 'a_attr_class',\n",
      "       'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 450 - INFO - Parsed data extraction completed. Elements collected: 14\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 456 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 466 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 539 - INFO - 6th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC9925742)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 347 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC9925742\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 354 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 533 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC9925742&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 395 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC9925742.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 399 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC9925742 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 413 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 417 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC9925742, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 6 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b0fcff40>, <Element supplementary-material at 0x1b12c5380>, <Element supplementary-material at 0x1b12dc440>, <Element supplementary-material at 0x1b12dff40>, <Element supplementary-material at 0x1b12ebdc0>, <Element supplementary-material at 0x1b12e9f00>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b0fcff40>, <Element supplementary-material at 0x1b12c5380>, <Element supplementary-material at 0x1b12dc440>, <Element supplementary-material at 0x1b12dff40>, <Element supplementary-material at 0x1b12ebdc0>, <Element supplementary-material at 0x1b12e9f00>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM1: After clustering (Supplementary Fig. 1a–e [this file] ), a total of 37,943 cells were annotated into 23 clusters using a predefined set of marker genes (Supplementary Fig. 1c [this file] ). \n",
      "To reveal heterogeneity within oligodendrocytes, we grouped the aforementioned oligodendrocytes into six classes described in a published database , as follows (Fig. 1a [rid=Fig1] and Supplementary Fig. 2a [this file] –d and 3a [this file] ): (1) Differentiation-committed oligodendrocyte precursors (COPs) and newly formed oligodendrocytes (NFOL) that express Sox6, Pdgfra, and Cspg4. (2) Myelin-forming oligodendrocytes (MFOL) that express Ctps, Ablims, and Thbs3. (3) Mature oligodendrocytes 1, 2 (MOL 1, 2) that express Il33, Ptgds, and Opalin. (4) Mature oligodendrocytes 3, 4 (MOL 3, 4) that express Klk6, S100b, Anxa5, Cd59a and Mgst3. Moreover, we discovered an additional oligodendrocyte cluster, which gradually increased in the 3-, 6-month-old App KI mice (Fig. 1b–d [rid=Fig1] and Supplementary Fig. 2e [this file] ). This cluster was present in the UMAP plot even with down-sampled data (400, 1,200, and 2,400 cells) (Supplementary Fig. 2f [this file] ). Remarkably, we found that this cluster shared some signatures with MOL2, while Alzheimer’s disease risk factors such as C4b, Apoe and Cd74 are strongly expressed (Supplementary Fig. 2c [this file] , 3a [this file] , 3b [this file] , and Source data file). \n",
      "We further confirmed the lineage transitions of DAOs and homeostatic MOLs from COP and NFOL in control and AD brains (Fig. 1e–g [rid=Fig1] and Supplementary Fig. 4 [this file] ), but density from the DAO cluster exclusively appeared in 3 and 6-month-old AD brains (Fig. 1 [rid=Fig1] g). \n",
      "We reanalyzed previously published single-cell transcriptome datasets from postmortem AD brains (Supplementary Fig. 5a–c [this file] ) . \n",
      "Notably, consistent with scRNA-seq results, we observed a significant increase in Mbp + , Cd74+ oligodendrocytes in 3- and 6-month-old App KI and 6-month-old 5xFAD mice, whereas the number of NeuroD1+ post-mitotic neurons significantly decreased (Fig. 2 [rid=Fig2] a, b [rid=Fig2] , Supplementary Fig. 6a–c [this file] , and Supplementary Fig. 7a [this file] , b [this file] ). We further confirmed an increase in Cd74+ and Oligo2+ AD-associated oligodendrocytes in App KI mice using flow cytometry (Fig. 2 [rid=Fig2] c, d [rid=Fig2] , and Supplementary Fig. 6d [this file] ). \n",
      "Transmission electron microscopy (TEM) analysis revealed that 6-month-old App KI mice exhibited a thinner myelin layer relative to the axon diameter (Fig. 2e [rid=Fig2] and supplementary fig. 8 [this file] ). \n",
      "Consistently, we found a significant increase in Mbp+ and Cd74+ DAOs in the hippocampus of premature progeria (6-month-old) and 13-month-old mice (Supplementary Fig. 9a–d [this file] ), whereas the number of NeuroD1-positive post-mitotic neurons decreased in the hippocampus of 13-month-old mice (Supplementary Fig. 9e and f [this file] ). We further confirmed that the genes enriched in DAOs were highly correlated with gene signatures shown in the aged human brain (Supplementary Fig. 10a, b [this file] ), indicating that Cd74-positive DAOs increased in the aged brains and had transcriptional profiles more similar to the aged human brain. \n",
      "In particular, we found significant alternations in the transcriptional profiles between DAOs and MOL 3/4 clusters and GO-term of up-regulated genes in DAO versus MOL3/4 showed the interleukin & chemokine production and ERK signaling (Fig. 3 [rid=Fig3] a and c [rid=Fig3] ), which was also confirmed during the disease progression in AD mice brain (Supplementary Fig. 11a, b [this file] ). \n",
      "We first identified microglia with a high DAM module score based on previously published DAM signatures (Supplementary Fig. 1e [this file] , 12a, b [this file] ), that gradually increased during AD progression (Fig. 3 [rid=Fig3] e, f [rid=Fig3] , and Supplementary Fig. 12c [this file] ). \n",
      "Supplementary Fig. 13a–c [this file] shows efficient inhibition of phosphor 44/42 MAPK signaling in primary oligodendrocytes upon treatment with 200 ng/ml SCH772984, a specific Erk1 and Erk2 inhibitor. Remarkably, oligodendrocytes treated with SCH772984 exhibited a significant increase in Mbp+ branching compared with the controls, whereas the number of Cd74+ cells significantly decreased (Supplementary Fig. 13d [this file] , 13e [this file] , and Fig. 4a–c [rid=Fig4] ). \n",
      "First, in order to examine the potential interactions between DAOs and other neuronal cell types, AD DAOs were co-cultured with AD neurons, astrocytes, and microglia treated with SCH772984 (Supplementary Fig. 14a [this file] ). We found that AD DAOs with other neuronal cell types did not increase the number of Mbp+ branches in the presence and absence of SCH772984 treatment (Supplementary Fig. 14b–f [this file] ), suggesting that inhibition of ERK/MAPK signaling in other neuronal cell types did not contribute to the phenotypic changes in DAOs (Fig. 4d–f [rid=Fig4] ). Moreover, to investigate whether other neural cell types can contribute to the AD-associated phenotypes via ERK signaling, each of the neurons, microglia, astrocytes and oligodendrocytes isolated from AD mice were subjected to SCH772984 treatment, and AD pathology was determined after co-culture with the remaining untreated AD neuronal cell types (Supplementary Fig. 15a [this file] ). Remarkably, we found that ERK1/2-inhibited oligodendrocytes significantly diminished the Aβ42/40 ratio in the AD primary neuronal cultures (Supplementary Fig. 15b [this file] ). Also, we observed a decrease in the amyloidogenic processing marker C99 in the AD neuronal cultures with SCH772984 treatment (Supplementary Fig. 15c–e [this file] ), indicating that AD oligodendrocytes predominantly contribute to the pathophysiology of AD via ERK signaling. \n",
      "Consistent with previous results, systemic administration of SCH772984 significantly reduced the occurrence of DAOs in the hippocampus of App KI mice (Fig. 5b, c [rid=Fig5] , and Supplementary Fig. 16a–d [this file] ) but not Cd74 + Iba1+ microglia (Supplementary Fig. 16c, d [this file] ). \n",
      "Moreover, we confirmed a decrease in the number of Aβ plaques (Fig. 5m, n [rid=Fig5] ), coupled with an increase in the number of NeuroD1+ cells in the SCH772984-treated AD mouse hippocampus (Supplementary Fig. 16e, f [this file] ). \n",
      "Figure 5a, b, c [this file] obtained from the AD Knowledge Portal ( https://adknowledgeportal.org (https://adknowledgeportal.org) ) under study 557 snRNAseqAD_TREM2(syn21126450). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM2: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM3: To identify transcriptional patterns of this cluster, we conducted differential expression gene analysis by using the FindAllMarkers function in the Seurat package (Supplementary Data 1 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM4: We next explored the unique molecular signatures of DAOs during AD development (Supplementary Data 2 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM5: Hippocampus tissues from 1-, 3-, and 6-month-old wild-type and App-KI mice were then dissected (Supplementary Data 3 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM6: Further information on research design is available in the Nature Portfolio Reporting Summary [this file] linked to this article. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 123 - INFO - Loading patterns for section 'data_availability_sections' for publisher 'PMC'.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 131 - INFO - Section pattern keys for publisher PMC: dict_keys(['data_availability_sections', 'supplementary_material_sections', 'supplementary_data_sections', 'key_resources_table', 'methods_sections'])\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 133 - INFO - Section name: data_availability_sections\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 135 - INFO - Found section 'data_availability_sections' in patterns for publisher 'PMC'.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 137 - INFO - Loaded patterns for section 'data_availability_sections': [\".//sec[@sec-type='data-availability']\", \".//notes[@notes-type='data-availability']\", \".//notes[@notes-type='data-availability']\"]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_data_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 706 - INFO - Found 7 supplementary data sections\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(key_resources_table)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "Batches:   0%|          | 0/1 [00:00<?, ?it/s]"
      ],
      "application/vnd.jupyter.widget-view+json": {
       "version_major": 2,
       "version_minor": 0,
       "model_id": "cb0b85ed029a4c2fabc4372b321bbcc9"
      }
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001B[97membeddings_retriever.py - line 51 - INFO - Searching for top-2 passages similar to the query by embeddings.\u001B[0m\n",
      "\u001B[97membeddings_retriever.py - line 33 - INFO - Computing L2 distances using numpy.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 364 - INFO - Processing DAS_content: 4 elements\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 823\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 549 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-3be805a3aa4bec0340f8c5a339694eb068aa501c8fa618edf3a1e6b4b04a2036\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 549 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-6cf6b700-b53a-44ad-8faa-db2ed9c52589\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE224398\\\",\\n    \\\"data_repository\\\": \\\"Gene Expression Omnibus (GEO)\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE224398\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"SRR21079269\\\",\\n    \\\"data_repository\\\": \\\"SRA\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/sra/?term=SRR21079269\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"SRR21079266\\\",\\n    \\\"data_repository\\\": \\\"SRA\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/sra/?term=SRR21079266\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"SRR21079265\\\",\\n    \\\"data_repository\\\": \\\"SRA\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/sra/?term=SRR21079265\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"SRR21079270\\\",\\n    \\\"data_repository\\\": \\\"SRA\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/sra/?term=SRR21079270\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"SRR21079268\\\",\\n    \\\"data_repository\\\": \\\"SRA\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/sra/?term=SRR21079268\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"SRR21079267\\\",\\n    \\\"data_repository\\\": \\\"SRA\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/sra/?term=SRR21079267\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn21126450\\\",\\n    \\\"data_repository\\\": \\\"AD Knowledge Portal\\\",\\n    \\\"dataset_webpage\\\": \\\"https://adknowledgeportal.org/study/filter?study=syn21126450\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE224398\\\",\\n    \\\"data_repository\\\": \\\"Gene Expression Omnibus (GEO)\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE224398\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"SRR21079269\\\",\\n    \\\"data_repository\\\": \\\"SRA\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/sra/?term=SRR21079269\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"SRR21079266\\\",\\n    \\\"data_repository\\\": \\\"SRA\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/sra/?term=SRR21079266\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"SRR21079265\\\",\\n    \\\"data_repository\\\": \\\"SRA\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/sra/?term=SRR21079265\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"SRR21079270\\\",\\n    \\\"data_repository\\\": \\\"SRA\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/sra/?term=SRR21079270\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"SRR21079268\\\",\\n    \\\"data_repository\\\": \\\"SRA\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/sra/?term=SRR21079268\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"SRR21079267\\\",\\n    \\\"data_repository\\\": \\\"SRA\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/sra/?term=SRR21079267\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn21126450\\\",\\n    \\\"data_repository\\\": \\\"AD Knowledge Portal\\\",\\n    \\\"dataset_webpage\\\": \\\"https://adknowledgeportal.org/study/filter?study=syn21126450\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755540972,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 718,\n",
      "        \"completion_tokens\": 580,\n",
      "        \"total_tokens\": 1298,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE224398', 'data_repository': 'Gene Expression Omnibus (GEO)', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE224398', 'dataset_id': 'GSE224398'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: GSE224398\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID GSE224398 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: Gene Expression Omnibus (GEO), identifier: GSE224398\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE224398 for Gene Expression Omnibus  GEO \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE224398 for Gene Expression Omnibus  GEO \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1043 - INFO - Found partial match for Gene Expression Omnibus  GEO  in Gene Expression Omnibus\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE224398, repo geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE224398\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 961 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'GSE224398', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE224398'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'SRR21079269', 'data_repository': 'SRA', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/sra/?term=SRR21079269', 'dataset_id': 'SRR21079269'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: SRR21079269\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID SRR21079269 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: SRA, identifier: SRR21079269\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier SRR21079269 for SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier SRR21079269 for SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier SRR21079269 for SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier SRR21079269 for SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/sra/?term=SRR21079269, repo sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/sra/?term=SRR21079269\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/bioproject/?term=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Link does not match the pattern https://www.ncbi.nlm.nih.gov/bioproject/?term=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'SRR21079269', 'data_repository': 'sra', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'SRR21079266', 'data_repository': 'SRA', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/sra/?term=SRR21079266', 'dataset_id': 'SRR21079266'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: SRR21079266\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID SRR21079266 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: SRA, identifier: SRR21079266\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier SRR21079266 for SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier SRR21079266 for SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier SRR21079266 for SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier SRR21079266 for SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/sra/?term=SRR21079266, repo sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/sra/?term=SRR21079266\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/bioproject/?term=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Link does not match the pattern https://www.ncbi.nlm.nih.gov/bioproject/?term=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'SRR21079266', 'data_repository': 'sra', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'SRR21079265', 'data_repository': 'SRA', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/sra/?term=SRR21079265', 'dataset_id': 'SRR21079265'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: SRR21079265\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID SRR21079265 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: SRA, identifier: SRR21079265\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier SRR21079265 for SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier SRR21079265 for SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier SRR21079265 for SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier SRR21079265 for SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/sra/?term=SRR21079265, repo sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/sra/?term=SRR21079265\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/bioproject/?term=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Link does not match the pattern https://www.ncbi.nlm.nih.gov/bioproject/?term=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'SRR21079265', 'data_repository': 'sra', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'SRR21079270', 'data_repository': 'SRA', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/sra/?term=SRR21079270', 'dataset_id': 'SRR21079270'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: SRR21079270\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID SRR21079270 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: SRA, identifier: SRR21079270\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier SRR21079270 for SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier SRR21079270 for SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier SRR21079270 for SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier SRR21079270 for SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/sra/?term=SRR21079270, repo sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/sra/?term=SRR21079270\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/bioproject/?term=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Link does not match the pattern https://www.ncbi.nlm.nih.gov/bioproject/?term=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'SRR21079270', 'data_repository': 'sra', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'SRR21079268', 'data_repository': 'SRA', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/sra/?term=SRR21079268', 'dataset_id': 'SRR21079268'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: SRR21079268\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID SRR21079268 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: SRA, identifier: SRR21079268\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier SRR21079268 for SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier SRR21079268 for SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier SRR21079268 for SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier SRR21079268 for SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/sra/?term=SRR21079268, repo sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/sra/?term=SRR21079268\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/bioproject/?term=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Link does not match the pattern https://www.ncbi.nlm.nih.gov/bioproject/?term=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'SRR21079268', 'data_repository': 'sra', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'SRR21079267', 'data_repository': 'SRA', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/sra/?term=SRR21079267', 'dataset_id': 'SRR21079267'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: SRR21079267\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID SRR21079267 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: SRA, identifier: SRR21079267\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier SRR21079267 for SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier SRR21079267 for SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier SRR21079267 for SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier SRR21079267 for SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: SRA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/sra/?term=SRR21079267, repo sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/sra/?term=SRR21079267\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/bioproject/?term=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Link does not match the pattern https://www.ncbi.nlm.nih.gov/bioproject/?term=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'SRR21079267', 'data_repository': 'sra', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn21126450', 'data_repository': 'AD Knowledge Portal', 'dataset_webpage': 'https://adknowledgeportal.org/study/filter?study=syn21126450', 'dataset_id': 'syn21126450'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: syn21126450\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID syn21126450 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: AD Knowledge Portal, identifier: syn21126450\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn21126450 for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn21126450 for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn21126450 for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn21126450 for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier syn21126450 for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1051 - INFO - Found id_pattern match for AD Knowledge Portal in (?<!\\d)syn\\d{7,12}(?!\\d)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://adknowledgeportal.org/study/filter?study=syn21126450, repo synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://adknowledgeportal.synapse.org/study/filter\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Link does not match the pattern https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'syn21126450', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 254\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 406 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-3e7b241a66a18c5a64b48d70c8f3d1fdd2b9397b048d308a5187cd07e02786e0\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 406 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-5bdc9dbb-c836-4841-88b1-76da4ff0a49b\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"APPKI_oligo_scRNAseq\\\",\\n        \\\"data_repository\\\": \\\"GitHub\\\",\\n        \\\"dataset_webpage\\\": \\\"https://github.com/Jobang9286/APPKI_oligo_scRNAseq\\\"\\n    ]\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"APPKI_oligo_scRNAseq\\\",\\n        \\\"data_repository\\\": \\\"GitHub\\\",\\n        \\\"dataset_webpage\\\": \\\"https://github.com/Jobang9286/APPKI_oligo_scRNAseq\\\"\\n    ]\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755540978,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 518,\n",
      "        \"completion_tokens\": 74,\n",
      "        \"total_tokens\": 592,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'APPKI_oligo_scRNAseq', 'data_repository': 'GitHub', 'dataset_webpage': 'https://github.com/Jobang9286/APPKI_oligo_scRNAseq', 'dataset_id': 'APPKI_oligo_scRNAseq'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: APPKI_oligo_scRNAseq\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID APPKI_oligo_scRNAseq is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: GitHub, identifier: APPKI_oligo_scRNAseq\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier APPKI_oligo_scRNAseq for GitHub\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier APPKI_oligo_scRNAseq for GitHub\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier APPKI_oligo_scRNAseq for GitHub\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier APPKI_oligo_scRNAseq for GitHub\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier APPKI_oligo_scRNAseq for GitHub\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: https://github.com/Jobang9286/APPKI_oligo_scRNAseq\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://github.com/Jobang9286/APPKI_oligo_scRNAseq, repo github.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://github.com/Jobang9286/APPKI_oligo_scRNAseq\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 970 - INFO - Dataset ID APPKI_oligo_scRNAseq found in resolved dataset page https://github.com/Jobang9286/APPKI_oligo_scRNAseq\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'APPKI_oligo_scRNAseq', 'data_repository': 'github.com', 'dataset_webpage': 'https://github.com/Jobang9286/APPKI_oligo_scRNAseq'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 5947\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 1830 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-1aa8210e057db36f7d6a8aa5a66d0f5ca4c8921d6d80b50e0bf409227fc1ebbb\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 1830 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-87a4b5a3-5c27-41b5-9bb4-9e4dcfedd86a\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755540979,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 1925,\n",
      "        \"completion_tokens\": 6,\n",
      "        \"total_tokens\": 1931,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 196\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 392 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-a6757b70463cc081fdf63cbd8e81916f2a21643b5c4d6563a0cc51a508d51ca4\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 392 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-dd45d9d2-d45d-4266-a16f-407b1e0ee210\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755540979,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 529,\n",
      "        \"completion_tokens\": 6,\n",
      "        \"total_tokens\": 535,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 377 - INFO - datasets (<class 'list'>): [{'dataset_identifier': 'GSE224398', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE224398'}, {'dataset_identifier': 'SRR21079269', 'data_repository': 'sra', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'SRR21079266', 'data_repository': 'sra', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'SRR21079265', 'data_repository': 'sra', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'SRR21079270', 'data_repository': 'sra', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'SRR21079268', 'data_repository': 'sra', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'SRR21079267', 'data_repository': 'sra', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn21126450', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'APPKI_oligo_scRNAseq', 'data_repository': 'github.com', 'dataset_webpage': 'https://github.com/Jobang9286/APPKI_oligo_scRNAseq'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'GSE224398', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE224398'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'SRR21079269', 'data_repository': 'sra', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'SRR21079266', 'data_repository': 'sra', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'SRR21079265', 'data_repository': 'sra', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'SRR21079270', 'data_repository': 'sra', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'SRR21079268', 'data_repository': 'sra', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'SRR21079267', 'data_repository': 'sra', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'syn21126450', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'APPKI_oligo_scRNAseq', 'data_repository': 'github.com', 'dataset_webpage': 'https://github.com/Jobang9286/APPKI_oligo_scRNAseq'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 385 - INFO - Final ret data availability: 9 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 386 - INFO - Final ret data availability: [{'dataset_identifier': 'GSE224398', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE224398', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'SRR21079269', 'data_repository': 'sra', 'dataset_webpage': 'n/a', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'SRR21079266', 'data_repository': 'sra', 'dataset_webpage': 'n/a', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'SRR21079265', 'data_repository': 'sra', 'dataset_webpage': 'n/a', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'SRR21079270', 'data_repository': 'sra', 'dataset_webpage': 'n/a', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'SRR21079268', 'data_repository': 'sra', 'dataset_webpage': 'n/a', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'SRR21079267', 'data_repository': 'sra', 'dataset_webpage': 'n/a', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'syn21126450', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'APPKI_oligo_scRNAseq', 'data_repository': 'github.com', 'dataset_webpage': 'https://github.com/Jobang9286/APPKI_oligo_scRNAseq', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1079 - INFO - Fetching metadata for 9 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: GSE224398\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: geo, identifier: GSE224398\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE224398 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE224398 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for GSE224398 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE224398\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE224398\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: SRR21079269\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: sra, identifier: SRR21079269\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier SRR21079269 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier SRR21079269 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier SRR21079269 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier SRR21079269 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for SRR21079269 in sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 2 with repo: sra and accession_id: SRR21079269\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/bioproject/?term=SRR21079269\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 2: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: SRR21079266\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: sra, identifier: SRR21079266\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier SRR21079266 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier SRR21079266 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier SRR21079266 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier SRR21079266 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for SRR21079266 in sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 3 with repo: sra and accession_id: SRR21079266\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/bioproject/?term=SRR21079266\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 3: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: SRR21079265\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: sra, identifier: SRR21079265\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier SRR21079265 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier SRR21079265 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier SRR21079265 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier SRR21079265 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for SRR21079265 in sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 4 with repo: sra and accession_id: SRR21079265\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/bioproject/?term=SRR21079265\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 4: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: SRR21079270\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: sra, identifier: SRR21079270\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier SRR21079270 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier SRR21079270 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier SRR21079270 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier SRR21079270 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for SRR21079270 in sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 5 with repo: sra and accession_id: SRR21079270\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/bioproject/?term=SRR21079270\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 5: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: SRR21079268\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: sra, identifier: SRR21079268\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier SRR21079268 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier SRR21079268 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier SRR21079268 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier SRR21079268 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for SRR21079268 in sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 6 with repo: sra and accession_id: SRR21079268\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/bioproject/?term=SRR21079268\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 6: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: SRR21079267\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: sra, identifier: SRR21079267\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier SRR21079267 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier SRR21079267 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier SRR21079267 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier SRR21079267 for sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for SRR21079267 in sra\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 7 with repo: sra and accession_id: SRR21079267\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/bioproject/?term=SRR21079267\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 7: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: syn21126450\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn21126450\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn21126450 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn21126450 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn21126450 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn21126450 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for syn21126450 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 8 with repo: synapse.org and accession_id: syn21126450\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.synapse.org/Synapse:syn21126450\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 8: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: APPKI_oligo_scRNAseq\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: github.com, identifier: APPKI_oligo_scRNAseq\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier APPKI_oligo_scRNAseq for github.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier APPKI_oligo_scRNAseq for github.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier APPKI_oligo_scRNAseq for github.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier APPKI_oligo_scRNAseq for github.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier APPKI_oligo_scRNAseq for github.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: github.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 900 - WARNING - URL github.com may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for APPKI_oligo_scRNAseq in github.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 9 with repo: github.com and accession_id: APPKI_oligo_scRNAseq\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1152 - WARNING - Repository github.com unknown in Ontology. Skipping dataset page 9.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1156 - INFO - Updated datasets len: 9\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 221 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 15, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'source_section', 'retrieval_pattern', 'access_mode', 'link',\n",
      "       'source_url', 'download_link', 'title', 'content_type', 'id', 'caption',\n",
      "       'description', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 456 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 462 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 539 - INFO - 7th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC9235070)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 347 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC9235070\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 354 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 533 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC9235070&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 383 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC9235070.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1ab1bbd10>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 375 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 310 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 399 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC9235070 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 413 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 439 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC9235070, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC9235070)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 13 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s2a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s2a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s2b'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s2b: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s2b'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s2b: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s2c'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s2c: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s2d'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s2d: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s2d'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s2d: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s2d'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s2d: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s4a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s4a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s4a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s4a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s4a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s4a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#SD1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for SD1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 384 - INFO - Chunking the HTML content for the parsing step.\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 172 - INFO - Extracting data availability elements from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: section.data-availability-statement\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: section.associated-data\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Data Availability')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: h3.pmc_sec_title:-soup-contains('Data Availability')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Associated Data')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Footnotes')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 221 - INFO - Found 3 data availability elements from HTML.\u001B[0m\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "Batches:   0%|          | 0/2 [00:00<?, ?it/s]"
      ],
      "application/vnd.jupyter.widget-view+json": {
       "version_major": 2,
       "version_minor": 0,
       "model_id": "4cb6cafc3180408eb81430ed119ab18e"
      }
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001B[97membeddings_retriever.py - line 51 - INFO - Searching for top-5 passages similar to the query by embeddings.\u001B[0m\n",
      "\u001B[97membeddings_retriever.py - line 33 - INFO - Computing L2 distances using numpy.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 21277\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 5662 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-4198e1589c1aff668c5e861ccda33ee49f9aaf671814fceac5db03d7a2df7cda\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 5662 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-2165f894-bb2d-4184-ab73-cc23f51b30a1\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"E-MTAB-6149\\\",\\n        \\\"data_repository\\\": \\\"EBI ArrayExpress\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-6149\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"METABRIC\\\",\\n        \\\"data_repository\\\": \\\"Synapse\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.synapse.org/#!Synapse:syn213309\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"TCGA\\\",\\n        \\\"data_repository\\\": \\\"TCGA data portal\\\",\\n        \\\"dataset_webpage\\\": \\\"https://gdc-portal.ncbi.nih.gov\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"MSigDB\\\",\\n        \\\"data_repository\\\": \\\"Molecular Signatures Database\\\",\\n        \\\"dataset_webpage\\\": \\\"http://software.broadinstitute.org/gsea/msigdb\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE76312\\\",\\n        \\\"data_repository\\\": \\\"GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE76312\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"syn30243143\\\",\\n        \\\"data_repository\\\": \\\"Synapse\\\",\\n        \\\"dataset_webpage\\\": \\\"https://synapse.org/#Synapse:syn30243143\\\"\\n    }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"E-MTAB-6149\\\",\\n        \\\"data_repository\\\": \\\"EBI ArrayExpress\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-6149\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"METABRIC\\\",\\n        \\\"data_repository\\\": \\\"Synapse\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.synapse.org/#!Synapse:syn213309\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"TCGA\\\",\\n        \\\"data_repository\\\": \\\"TCGA data portal\\\",\\n        \\\"dataset_webpage\\\": \\\"https://gdc-portal.ncbi.nih.gov\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"MSigDB\\\",\\n        \\\"data_repository\\\": \\\"Molecular Signatures Database\\\",\\n        \\\"dataset_webpage\\\": \\\"http://software.broadinstitute.org/gsea/msigdb\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE76312\\\",\\n        \\\"data_repository\\\": \\\"GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE76312\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"syn30243143\\\",\\n        \\\"data_repository\\\": \\\"Synapse\\\",\\n        \\\"dataset_webpage\\\": \\\"https://synapse.org/#Synapse:syn30243143\\\"\\n    }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755540999,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 6185,\n",
      "        \"completion_tokens\": 380,\n",
      "        \"total_tokens\": 6565,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'E-MTAB-6149', 'data_repository': 'EBI ArrayExpress', 'dataset_webpage': 'https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-6149', 'dataset_id': 'E-MTAB-6149'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: E-MTAB-6149\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID E-MTAB-6149 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: EBI ArrayExpress, identifier: E-MTAB-6149\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-MTAB-6149 for EBI ArrayExpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-MTAB-6149 for EBI ArrayExpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-MTAB-6149 for EBI ArrayExpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-MTAB-6149 for EBI ArrayExpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1043 - INFO - Found partial match for EBI ArrayExpress in ArrayExpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-6149, repo arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ebi.ac.uk/biostudies/arrayexpress/studies/E-MTAB-6149\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.ebi.ac.uk/biostudies/ArrayExpress/studies/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 961 - INFO - Link matches the pattern https://www.ebi.ac.uk/biostudies/ArrayExpress/studies/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'E-MTAB-6149', 'data_repository': 'arrayexpress', 'dataset_webpage': 'https://www.ebi.ac.uk/biostudies/arrayexpress/studies/E-MTAB-6149'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'METABRIC', 'data_repository': 'Synapse', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn213309', 'dataset_id': 'METABRIC'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: METABRIC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID METABRIC is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: Synapse, identifier: METABRIC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier METABRIC for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier METABRIC for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier METABRIC for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier METABRIC for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1037 - INFO - Found repo_name match for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.synapse.org/#!Synapse:syn213309, repo synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.synapse.org/#!Synapse:syn213309\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Link does not match the pattern https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'METABRIC', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'TCGA', 'data_repository': 'TCGA data portal', 'dataset_webpage': 'https://gdc-portal.ncbi.nih.gov', 'dataset_id': 'TCGA'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: TCGA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID TCGA is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: TCGA data portal, identifier: TCGA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier TCGA for TCGA data portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier TCGA for TCGA data portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier TCGA for TCGA data portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier TCGA for TCGA data portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier TCGA for TCGA data portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: https://gdc-portal.ncbi.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://gdc-portal.ncbi.nih.gov, repo gdc-portal.ncbi.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 981 - WARNING - Error resolving URL https://gdc-portal.ncbi.nih.gov: HTTPSConnectionPool(host='gdc-portal.ncbi.nih.gov', port=443): Max retries exceeded with url: / (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x1b0af0250>: Failed to resolve 'gdc-portal.ncbi.nih.gov' ([Errno 8] nodename nor servname provided, or not known)\"))\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 972 - INFO - Repository gdc-portal.ncbi.nih.gov not found in ontology\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'TCGA', 'data_repository': 'gdc-portal.ncbi.nih.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'MSigDB', 'data_repository': 'Molecular Signatures Database', 'dataset_webpage': 'http://software.broadinstitute.org/gsea/msigdb', 'dataset_id': 'MSigDB'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: MSigDB\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID MSigDB is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: Molecular Signatures Database, identifier: MSigDB\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier MSigDB for Molecular Signatures Database\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier MSigDB for Molecular Signatures Database\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier MSigDB for Molecular Signatures Database\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier MSigDB for Molecular Signatures Database\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier MSigDB for Molecular Signatures Database\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: http://software.broadinstitute.org/gsea/msigdb\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: http://software.broadinstitute.org/gsea/msigdb, repo software.broadinstitute.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.gsea-msigdb.org/gsea/msigdb\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 972 - INFO - Repository software.broadinstitute.org not found in ontology\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'MSigDB', 'data_repository': 'software.broadinstitute.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE76312', 'data_repository': 'GEO', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE76312', 'dataset_id': 'GSE76312'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: GSE76312\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID GSE76312 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: GEO, identifier: GSE76312\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE76312 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE76312 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE76312, repo geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE76312\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 961 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'GSE76312', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE76312'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn30243143', 'data_repository': 'Synapse', 'dataset_webpage': 'https://synapse.org/#Synapse:syn30243143', 'dataset_id': 'syn30243143'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: syn30243143\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID syn30243143 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: Synapse, identifier: syn30243143\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn30243143 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn30243143 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn30243143 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn30243143 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1037 - INFO - Found repo_name match for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://synapse.org/#Synapse:syn30243143, repo synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.synapse.org/#Synapse:syn30243143\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Link does not match the pattern https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'syn30243143', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1079 - INFO - Fetching metadata for 6 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: E-MTAB-6149\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: arrayexpress, identifier: E-MTAB-6149\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-MTAB-6149 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-MTAB-6149 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-MTAB-6149 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-MTAB-6149 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for E-MTAB-6149 in arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 1 with repo: arrayexpress and accession_id: E-MTAB-6149\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ebi.ac.uk/biostudies/ArrayExpress/studies/E-MTAB-6149\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: METABRIC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: synapse.org, identifier: METABRIC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier METABRIC for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier METABRIC for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier METABRIC for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier METABRIC for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for METABRIC in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 994 - WARNING - Identifier METABRIC does not match pattern for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: METABRIC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.synapse.org/Synapse:METABRIC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: TCGA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: gdc-portal.ncbi.nih.gov, identifier: TCGA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier TCGA for gdc-portal.ncbi.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier TCGA for gdc-portal.ncbi.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier TCGA for gdc-portal.ncbi.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier TCGA for gdc-portal.ncbi.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier TCGA for gdc-portal.ncbi.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: gdc-portal.ncbi.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 900 - WARNING - URL gdc-portal.ncbi.nih.gov may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for TCGA in gdc-portal.ncbi.nih.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 3 with repo: gdc-portal.ncbi.nih.gov and accession_id: TCGA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1152 - WARNING - Repository gdc-portal.ncbi.nih.gov unknown in Ontology. Skipping dataset page 3.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: MSigDB\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: software.broadinstitute.org, identifier: MSigDB\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier MSigDB for software.broadinstitute.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier MSigDB for software.broadinstitute.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier MSigDB for software.broadinstitute.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier MSigDB for software.broadinstitute.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier MSigDB for software.broadinstitute.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: software.broadinstitute.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 900 - WARNING - URL software.broadinstitute.org may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for MSigDB in software.broadinstitute.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 4 with repo: software.broadinstitute.org and accession_id: MSigDB\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1152 - WARNING - Repository software.broadinstitute.org unknown in Ontology. Skipping dataset page 4.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: GSE76312\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: geo, identifier: GSE76312\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE76312 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE76312 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for GSE76312 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 5 with repo: geo and accession_id: GSE76312\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE76312\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 5: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: syn30243143\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn30243143\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn30243143 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn30243143 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn30243143 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn30243143 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for syn30243143 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 6 with repo: synapse.org and accession_id: syn30243143\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.synapse.org/Synapse:syn30243143\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 6: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1156 - INFO - Updated datasets len: 6\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 19, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 450 - INFO - Parsed data extraction completed. Elements collected: 8\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 456 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 466 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 539 - INFO - 8th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11661334)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 347 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11661334\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 354 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 533 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC11661334&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 395 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC11661334.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 399 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11661334 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 413 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 417 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11661334, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 2 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b031d800>, <Element supplementary-material at 0x1b601fec0>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b031d800>, <Element supplementary-material at 0x1b601fec0>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD1: We also observed a clear linear trend between the measured correlation of protein expression within well-imputed pairs and their imputed co-expression, further supporting the robustness of COWAS imputation models ( Supplementary Figure 1 [this file] ). \n",
      "For LDL cholesterol, which had the best-powered GWAS of the three traits we considered, we identified 38 protein pairs with a significant co-expression effect ( Supplementary Figure 2 [this file] ). \n",
      "By visualizing significant COWAS pairs as a network, we identified complexes of mutually interacting proteins that modulate LDL cholesterol levels ( Supplementary Figure 3 [this file] ). For example, we found that APOE and PLTP both decrease LDL cholesterol levels, while their co-expression is associated with increased LDL cholesterol levels ( Supplementary Figure 4 [this file] ). \n",
      "Effect sizes, standard errors, and values for the pairs shown in Figure 3 [rid=F3] are also summarized in Supplementary Table 1 [this file] . \n",
      "Results for protein pairs identified as significant using at least one of these three tests with IGAP GWAS data are shown in Supplementary Figure 5 [this file] and Supplementary Table 2 [this file] . \n",
      "Finally, we considered an alternative, product-based formulation of COWAS (see Supplementary Note 1 [this file] for details). The mean out-of-sample for product-based co-expression imputation models was slightly higher than for any of the residual-based co-expression imputation models, and over 99% of well-imputed pairs again had a nominally significant correlation between imputed and observed expression ( Supplementary Figure 6 [this file] and Supplementary Data 1 [rid=SD2] ). \n",
      "We also discovered 22 pairs with an overall effect on AD using the product-based COWAS global test and EADB summary statistics ( Supplementary Figure 7 [this file] and Supplementary Table 3 [this file] ). Full results for AD using the product-based version of COWAS are provided in Supplementary Figure 7 [this file] , Supplementary Table 3 [this file] , and Supplementary Data 18 [rid=SD2] – 19 [rid=SD2] . \n",
      "Effect sizes, standard errors, and values for the pairs shown in Figure 4 [rid=F4] are also summarized in Supplementary Table 4 [this file] . \n",
      "In Supplementary Note 1 [this file] we define an alternative, product-based version of COWAS in which is replaced with . \n",
      "In Supplementary Note 2 [this file] we derive a summary-level version of the second stage following the approach introduced in the multivariate imaging-wide association study (MV-IWAS) method [ 14 [rid=R14] , 16 [rid=R16] ], which is related to other methods in the field of Mendelian randomization [ 87 [rid=R87] , 88 [rid=R88] ]. We also provide full derivations for both individual-level and summary-level versions of standard TWAS/PWAS in Supplementary Note 3 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD2: Across all of the model types we considered, the median out-of-sample correlation between estimated and imputed co-expression was always lower than the median out-of-sample correlation between measured and imputed single-protein expression ( Figure 2a [rid=F2] , Figure 2c [rid=F2] , and Supplementary Data 1 [this file] ). \n",
      "Among these well-imputed pairs, lasso regression with -pQTLs pre-screened by their effect sizes achieved the highest mean out-of-sample for predicting co-expression (mean , Supplementary Data 1 [this file] ). On the other hand, ridge regression with both -pQTLs and -pQTLs pre-screened by their values yielded the greatest number of well-imputed protein pairs ( Figure 2d [rid=F2] and Supplementary Data 1 [this file] ). \n",
      "Model performance metrics for every combination of protein pair and model type are provided in Supplementary Data 2 [this file] – 13 [this file] . \n",
      "Full imputation performance metrics for all analyzed proteins and outcome traits are provided in Supplementary Data 14 [this file] – 20 [this file] . \n",
      "Full results for LDL cholesterol, including the estimated effect sizes and standard errors for each pair, are provided in Supplementary Data 14 [this file] . \n",
      "Full results from our validation study for LDL cholesterol are provided in Supplementary Data 15 [this file] . \n",
      "Full results for AD, including the estimated effect sizes and standard errors for each pair, are provided in Supplementary Data 16 [this file] . \n",
      "Full results for all protein pairs passing quality control are provided in Supplementary Data 17 [this file] . \n",
      "The mean out-of-sample for product-based co-expression imputation models was slightly higher than for any of the residual-based co-expression imputation models, and over 99% of well-imputed pairs again had a nominally significant correlation between imputed and observed expression ( Supplementary Figure 6 [rid=SD1] and Supplementary Data 1 [this file] ). \n",
      "Full results for AD using the product-based version of COWAS are provided in Supplementary Figure 7 [rid=SD1] , Supplementary Table 3 [rid=SD1] , and Supplementary Data 18 [this file] – 19 [this file] . \n",
      "Full results for PD, including the estimated effect sizes and standard errors for each pair, are provided in Supplementary Data 20 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 123 - INFO - Loading patterns for section 'data_availability_sections' for publisher 'PMC'.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 131 - INFO - Section pattern keys for publisher PMC: dict_keys(['data_availability_sections', 'supplementary_material_sections', 'supplementary_data_sections', 'key_resources_table', 'methods_sections'])\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 133 - INFO - Section name: data_availability_sections\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 135 - INFO - Found section 'data_availability_sections' in patterns for publisher 'PMC'.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 137 - INFO - Loaded patterns for section 'data_availability_sections': [\".//sec[@sec-type='data-availability']\", \".//notes[@notes-type='data-availability']\", \".//notes[@notes-type='data-availability']\"]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_data_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 706 - INFO - Found 3 supplementary data sections\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(key_resources_table)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "Batches:   0%|          | 0/1 [00:00<?, ?it/s]"
      ],
      "application/vnd.jupyter.widget-view+json": {
       "version_major": 2,
       "version_minor": 0,
       "model_id": "bf3bad4047ad4440905e1d52051f5eff"
      }
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001B[97membeddings_retriever.py - line 51 - INFO - Searching for top-2 passages similar to the query by embeddings.\u001B[0m\n",
      "\u001B[97membeddings_retriever.py - line 33 - INFO - Computing L2 distances using numpy.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 364 - INFO - Processing DAS_content: 3 elements\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 1253\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 656 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-c3ac27a0def00719fadb83770c97d79afb062a22769221755148374f274cdb3f\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 656 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-24a3c7f9-20cd-42de-93d0-3230fdecdd82\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  [\\n    \\\"dataset_identifier\\\": \\\"35107\\\",\\n    \\\"data_repository\\\": \\\"UK Biobank\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.ukbiobank.ac.uk/enable-your-research\\\"\\n  ],\\n  [\\n    \\\"dataset_identifier\\\": \\\"syn51364943\\\",\\n    \\\"data_repository\\\": \\\"Synapse\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.synapse.org/Synapse:syn51364943\\\"\\n  ],\\n  [\\n    \\\"dataset_identifier\\\": \\\"GCST90027158\\\",\\n    \\\"data_repository\\\": \\\"NHGRI-EBI GWAS Catalog\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/gwas\\\"\\n  ],\\n  [\\n    \\\"dataset_identifier\\\": \\\"GCST007511\\\",\\n    \\\"data_repository\\\": \\\"NHGRI-EBI GWAS Catalog\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/gwas\\\"\\n  ],\\n  [\\n    \\\"dataset_identifier\\\": \\\"GCST009325\\\",\\n    \\\"data_repository\\\": \\\"NHGRI-EBI GWAS Catalog\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/gwas\\\"\\n  ]\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  [\\n    \\\"dataset_identifier\\\": \\\"35107\\\",\\n    \\\"data_repository\\\": \\\"UK Biobank\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.ukbiobank.ac.uk/enable-your-research\\\"\\n  ],\\n  [\\n    \\\"dataset_identifier\\\": \\\"syn51364943\\\",\\n    \\\"data_repository\\\": \\\"Synapse\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.synapse.org/Synapse:syn51364943\\\"\\n  ],\\n  [\\n    \\\"dataset_identifier\\\": \\\"GCST90027158\\\",\\n    \\\"data_repository\\\": \\\"NHGRI-EBI GWAS Catalog\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/gwas\\\"\\n  ],\\n  [\\n    \\\"dataset_identifier\\\": \\\"GCST007511\\\",\\n    \\\"data_repository\\\": \\\"NHGRI-EBI GWAS Catalog\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/gwas\\\"\\n  ],\\n  [\\n    \\\"dataset_identifier\\\": \\\"GCST009325\\\",\\n    \\\"data_repository\\\": \\\"NHGRI-EBI GWAS Catalog\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/gwas\\\"\\n  ]\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755541010,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 833,\n",
      "        \"completion_tokens\": 321,\n",
      "        \"total_tokens\": 1154,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': '35107', 'data_repository': 'UK Biobank', 'dataset_webpage': 'https://www.ukbiobank.ac.uk/enable-your-research', 'dataset_id': '35107'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: 35107\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID 35107 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: UK Biobank, identifier: 35107\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 35107 for UK Biobank\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 35107 for UK Biobank\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier 35107 for UK Biobank\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier 35107 for UK Biobank\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier 35107 for UK Biobank\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: https://www.ukbiobank.ac.uk/enable-your-research\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ukbiobank.ac.uk/enable-your-research, repo www.ukbiobank.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ukbiobank.ac.uk/enable-your-research\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 972 - INFO - Repository www.ukbiobank.ac.uk not found in ontology\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': '35107', 'data_repository': 'www.ukbiobank.ac.uk', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn51364943', 'data_repository': 'Synapse', 'dataset_webpage': 'https://www.synapse.org/Synapse:syn51364943', 'dataset_id': 'syn51364943'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: syn51364943\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID syn51364943 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: Synapse, identifier: syn51364943\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn51364943 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn51364943 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn51364943 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn51364943 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1037 - INFO - Found repo_name match for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.synapse.org/Synapse:syn51364943, repo synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.synapse.org/Synapse:syn51364943\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 961 - INFO - Link matches the pattern https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'syn51364943', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/Synapse:syn51364943'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GCST90027158', 'data_repository': 'NHGRI-EBI GWAS Catalog', 'dataset_webpage': 'https://www.ebi.ac.uk/gwas', 'dataset_id': 'GCST90027158'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: GCST90027158\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID GCST90027158 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: NHGRI-EBI GWAS Catalog, identifier: GCST90027158\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GCST90027158 for NHGRI-EBI GWAS Catalog\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GCST90027158 for NHGRI-EBI GWAS Catalog\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier GCST90027158 for NHGRI-EBI GWAS Catalog\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier GCST90027158 for NHGRI-EBI GWAS Catalog\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier GCST90027158 for NHGRI-EBI GWAS Catalog\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: https://www.ebi.ac.uk/gwas\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ebi.ac.uk/gwas, repo www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ebi.ac.uk/gwas/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 967 - INFO - No dataset_webpage_url_ptr found for www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'GCST90027158', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'https://www.ebi.ac.uk/gwas/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GCST007511', 'data_repository': 'NHGRI-EBI GWAS Catalog', 'dataset_webpage': 'https://www.ebi.ac.uk/gwas', 'dataset_id': 'GCST007511'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: GCST007511\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID GCST007511 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: NHGRI-EBI GWAS Catalog, identifier: GCST007511\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GCST007511 for NHGRI-EBI GWAS Catalog\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GCST007511 for NHGRI-EBI GWAS Catalog\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier GCST007511 for NHGRI-EBI GWAS Catalog\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier GCST007511 for NHGRI-EBI GWAS Catalog\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier GCST007511 for NHGRI-EBI GWAS Catalog\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: https://www.ebi.ac.uk/gwas\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ebi.ac.uk/gwas, repo www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ebi.ac.uk/gwas/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 967 - INFO - No dataset_webpage_url_ptr found for www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'GCST007511', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'https://www.ebi.ac.uk/gwas/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GCST009325', 'data_repository': 'NHGRI-EBI GWAS Catalog', 'dataset_webpage': 'https://www.ebi.ac.uk/gwas', 'dataset_id': 'GCST009325'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: GCST009325\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID GCST009325 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: NHGRI-EBI GWAS Catalog, identifier: GCST009325\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GCST009325 for NHGRI-EBI GWAS Catalog\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GCST009325 for NHGRI-EBI GWAS Catalog\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier GCST009325 for NHGRI-EBI GWAS Catalog\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier GCST009325 for NHGRI-EBI GWAS Catalog\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier GCST009325 for NHGRI-EBI GWAS Catalog\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: https://www.ebi.ac.uk/gwas\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ebi.ac.uk/gwas, repo www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ebi.ac.uk/gwas/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 967 - INFO - No dataset_webpage_url_ptr found for www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'GCST009325', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'https://www.ebi.ac.uk/gwas/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 1806\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 794 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-cad7a47724c52a80d7d6aeefe03048aa1e12feab382cd6d510d8053ccdb8e0e7\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 794 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-6d6c9b8c-0b4f-4643-8cbb-bf5d58ba2655\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"Application #35107\\\",\\n        \\\"data_repository\\\": \\\"UK Biobank\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ukbiobank.ac.uk/enable-your-research\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"syn51364943\\\",\\n        \\\"data_repository\\\": \\\"Synapse\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.synapse.org/Synapse:syn51364943\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GCST90027158, GCST007511, and GCST009325\\\",\\n        \\\"data_repository\\\": \\\"NHGRI-EBI GWAS Catalog\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/gwas\\\"\\n    }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"Application #35107\\\",\\n        \\\"data_repository\\\": \\\"UK Biobank\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ukbiobank.ac.uk/enable-your-research\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"syn51364943\\\",\\n        \\\"data_repository\\\": \\\"Synapse\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.synapse.org/Synapse:syn51364943\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GCST90027158, GCST007511, and GCST009325\\\",\\n        \\\"data_repository\\\": \\\"NHGRI-EBI GWAS Catalog\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/gwas\\\"\\n    }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755541014,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 1084,\n",
      "        \"completion_tokens\": 218,\n",
      "        \"total_tokens\": 1302,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'Application #35107', 'data_repository': 'UK Biobank', 'dataset_webpage': 'https://www.ukbiobank.ac.uk/enable-your-research', 'dataset_id': 'Application #35107'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: Application #35107\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 940 - INFO - Accession ID Application #35107 is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: UK Biobank, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier n/a for UK Biobank\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier n/a for UK Biobank\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier n/a for UK Biobank\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier n/a for UK Biobank\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier n/a for UK Biobank\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: https://www.ukbiobank.ac.uk/enable-your-research\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ukbiobank.ac.uk/enable-your-research, repo www.ukbiobank.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ukbiobank.ac.uk/enable-your-research\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 972 - INFO - Repository www.ukbiobank.ac.uk not found in ontology\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'www.ukbiobank.ac.uk', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn51364943', 'data_repository': 'Synapse', 'dataset_webpage': 'https://www.synapse.org/Synapse:syn51364943', 'dataset_id': 'syn51364943'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: syn51364943\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID syn51364943 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: Synapse, identifier: syn51364943\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn51364943 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn51364943 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn51364943 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn51364943 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1037 - INFO - Found repo_name match for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.synapse.org/Synapse:syn51364943, repo synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.synapse.org/Synapse:syn51364943\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 961 - INFO - Link matches the pattern https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'syn51364943', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/Synapse:syn51364943'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GCST90027158, GCST007511, and GCST009325', 'data_repository': 'NHGRI-EBI GWAS Catalog', 'dataset_webpage': 'https://www.ebi.ac.uk/gwas', 'dataset_id': 'GCST90027158, GCST007511, and GCST009325'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: GCST90027158, GCST007511, and GCST009325\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 940 - INFO - Accession ID GCST90027158, GCST007511, and GCST009325 is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: NHGRI-EBI GWAS Catalog, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier n/a for NHGRI-EBI GWAS Catalog\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier n/a for NHGRI-EBI GWAS Catalog\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier n/a for NHGRI-EBI GWAS Catalog\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier n/a for NHGRI-EBI GWAS Catalog\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier n/a for NHGRI-EBI GWAS Catalog\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: https://www.ebi.ac.uk/gwas\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ebi.ac.uk/gwas, repo www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ebi.ac.uk/gwas/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 967 - INFO - No dataset_webpage_url_ptr found for www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 626 - INFO - Dataset ID is 'n/a' and repository name from prompt\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 3722\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 1273 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-d00eb82168ae14e44ef7ab2ac1c3f1acc727d4399a8549edc50fa477d2c5af9a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 1273 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-43924317-4efa-4ad3-b92a-9107b328d48c\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"[\\n    {\\n        \\\"dataset_identifier\\\": \\\"HIPPIE database version 2.3\\\",\\n        \\\"data_repository\\\": \\\"Human Integrated Protein\\u2013Protein Interaction rEference (HIPPIE)\\\",\\n        \\\"dataset_webpage\\\": \\\"http://cbdm.uni-mainz.de/hippie/\\\"\\n    }\\n]\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"[\\n    {\\n        \\\"dataset_identifier\\\": \\\"HIPPIE database version 2.3\\\",\\n        \\\"data_repository\\\": \\\"Human Integrated Protein\\u2013Protein Interaction rEference (HIPPIE)\\\",\\n        \\\"dataset_webpage\\\": \\\"http://cbdm.uni-mainz.de/hippie/\\\"\\n    }\\n]\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755541016,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 1420,\n",
      "        \"completion_tokens\": 73,\n",
      "        \"total_tokens\": 1493,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'HIPPIE database version 2.3', 'data_repository': 'Human Integrated Protein–Protein Interaction rEference (HIPPIE)', 'dataset_webpage': 'http://cbdm.uni-mainz.de/hippie/', 'dataset_id': 'HIPPIE database version 2.3'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: HIPPIE database version 2.3\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 940 - INFO - Accession ID HIPPIE database version 2.3 is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: Human Integrated Protein–Protein Interaction rEference (HIPPIE), identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier n/a for Human Integrated Protein–Protein Interaction rEference  HIPPIE \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier n/a for Human Integrated Protein–Protein Interaction rEference  HIPPIE \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier n/a for Human Integrated Protein–Protein Interaction rEference  HIPPIE \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier n/a for Human Integrated Protein–Protein Interaction rEference  HIPPIE \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier n/a for Human Integrated Protein–Protein Interaction rEference  HIPPIE \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: http://cbdm.uni-mainz.de/hippie/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: http://cbdm.uni-mainz.de/hippie/, repo cbdm.uni-mainz.de\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://cbdm-01.zdv.uni-mainz.de/~mschaefer/hippie/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 972 - INFO - Repository cbdm.uni-mainz.de not found in ontology\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'cbdm.uni-mainz.de', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 377 - INFO - datasets (<class 'list'>): [{'dataset_identifier': '35107', 'data_repository': 'www.ukbiobank.ac.uk', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn51364943', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/Synapse:syn51364943'}, {'dataset_identifier': 'GCST90027158', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'https://www.ebi.ac.uk/gwas/'}, {'dataset_identifier': 'GCST007511', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'https://www.ebi.ac.uk/gwas/'}, {'dataset_identifier': 'GCST009325', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'https://www.ebi.ac.uk/gwas/'}, {'dataset_identifier': 'n/a', 'data_repository': 'www.ukbiobank.ac.uk', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn51364943', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/Synapse:syn51364943'}, {'dataset_identifier': 'n/a', 'data_repository': 'cbdm.uni-mainz.de', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': '35107', 'data_repository': 'www.ukbiobank.ac.uk', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'syn51364943', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/Synapse:syn51364943'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'GCST90027158', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'https://www.ebi.ac.uk/gwas/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'GCST007511', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'https://www.ebi.ac.uk/gwas/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'GCST009325', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'https://www.ebi.ac.uk/gwas/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'n/a', 'data_repository': 'www.ukbiobank.ac.uk', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'syn51364943', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/Synapse:syn51364943'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'n/a', 'data_repository': 'cbdm.uni-mainz.de', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 385 - INFO - Final ret data availability: 8 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 386 - INFO - Final ret data availability: [{'dataset_identifier': '35107', 'data_repository': 'www.ukbiobank.ac.uk', 'dataset_webpage': 'n/a', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'syn51364943', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/Synapse:syn51364943', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'GCST90027158', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'https://www.ebi.ac.uk/gwas/', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'GCST007511', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'https://www.ebi.ac.uk/gwas/', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'GCST009325', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'https://www.ebi.ac.uk/gwas/', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'n/a', 'data_repository': 'www.ukbiobank.ac.uk', 'dataset_webpage': 'n/a', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'syn51364943', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/Synapse:syn51364943', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'n/a', 'data_repository': 'cbdm.uni-mainz.de', 'dataset_webpage': 'n/a', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1079 - INFO - Fetching metadata for 8 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: 35107\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: www.ukbiobank.ac.uk, identifier: 35107\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 35107 for www.ukbiobank.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 35107 for www.ukbiobank.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier 35107 for www.ukbiobank.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier 35107 for www.ukbiobank.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier 35107 for www.ukbiobank.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: www.ukbiobank.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 900 - WARNING - URL www.ukbiobank.ac.uk may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for 35107 in www.ukbiobank.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 1 with repo: www.ukbiobank.ac.uk and accession_id: 35107\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1152 - WARNING - Repository www.ukbiobank.ac.uk unknown in Ontology. Skipping dataset page 1.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: syn51364943\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn51364943\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn51364943 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn51364943 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn51364943 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn51364943 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for syn51364943 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn51364943\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.synapse.org/Synapse:syn51364943\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: GCST90027158\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: www.ebi.ac.uk, identifier: GCST90027158\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for GCST90027158 in www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 3 with repo: www.ebi.ac.uk and accession_id: GCST90027158\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ebi.ac.uk/pride/archive/projects/GCST90027158\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 3: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: GCST007511\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: www.ebi.ac.uk, identifier: GCST007511\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for GCST007511 in www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 4 with repo: www.ebi.ac.uk and accession_id: GCST007511\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ebi.ac.uk/pride/archive/projects/GCST007511\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 4: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: GCST009325\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: www.ebi.ac.uk, identifier: GCST009325\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for GCST009325 in www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 5 with repo: www.ebi.ac.uk and accession_id: GCST009325\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ebi.ac.uk/pride/archive/projects/GCST009325\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 5: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1095 - INFO - Skipping dataset 6: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: syn51364943\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn51364943\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn51364943 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn51364943 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn51364943 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn51364943 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for syn51364943 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 7 with repo: synapse.org and accession_id: syn51364943\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.synapse.org/Synapse:syn51364943\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 7: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1095 - INFO - Skipping dataset 8: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1156 - INFO - Updated datasets len: 8\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 221 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 10, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'source_section', 'retrieval_pattern', 'access_mode', 'link',\n",
      "       'source_url', 'download_link', 'title', 'content_type', 'id', 'caption',\n",
      "       'description', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 456 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 462 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 539 - INFO - 9th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11929800)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 347 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11929800\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 354 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 533 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC11929800&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 395 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC11929800.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 399 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11929800 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 413 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 417 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11929800, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 40 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b7219880>, <Element supplementary-material at 0x1b7218d00>, <Element supplementary-material at 0x1b7218d40>, <Element supplementary-material at 0x1b7218e80>, <Element supplementary-material at 0x1b7218f00>, <Element supplementary-material at 0x1b6159840>, <Element supplementary-material at 0x1b07bab40>, <Element supplementary-material at 0x1b07b9100>, <Element supplementary-material at 0x1b07b99c0>, <Element supplementary-material at 0x1b07b9380>, <Element supplementary-material at 0x1b0d87f40>, <Element supplementary-material at 0x1b07b9000>, <Element supplementary-material at 0x1b07ba900>, <Element supplementary-material at 0x1b07b9300>, <Element supplementary-material at 0x1b07bae00>, <Element supplementary-material at 0x1b07bbb40>, <Element supplementary-material at 0x1b07bb3c0>, <Element supplementary-material at 0x1b07b8b40>, <Element supplementary-material at 0x1b61266c0>, <Element supplementary-material at 0x1b5d69d80>, <Element supplementary-material at 0x1b6513a40>, <Element supplementary-material at 0x1b0903a00>, <Element supplementary-material at 0x1b116fe40>, <Element supplementary-material at 0x1b71fc7c0>, <Element supplementary-material at 0x1b71fde40>, <Element supplementary-material at 0x1b5ea4ec0>, <Element supplementary-material at 0x1b7147680>, <Element supplementary-material at 0x1b616bc80>, <Element supplementary-material at 0x1b6169980>, <Element supplementary-material at 0x1b6169200>, <Element supplementary-material at 0x1b0af3a80>, <Element supplementary-material at 0x1b0af3780>, <Element supplementary-material at 0x1b0af2c00>, <Element supplementary-material at 0x1b1128c40>, <Element supplementary-material at 0x1b112a2c0>, <Element supplementary-material at 0x1b1129740>, <Element supplementary-material at 0x1b112bd00>, <Element supplementary-material at 0x1b1129880>, <Element supplementary-material at 0x1b112ba00>, <Element supplementary-material at 0x1b1128a00>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b7219880>, <Element supplementary-material at 0x1b7218d00>, <Element supplementary-material at 0x1b7218d40>, <Element supplementary-material at 0x1b7218e80>, <Element supplementary-material at 0x1b7218f00>, <Element supplementary-material at 0x1b6159840>, <Element supplementary-material at 0x1b07bab40>, <Element supplementary-material at 0x1b07b9100>, <Element supplementary-material at 0x1b07b99c0>, <Element supplementary-material at 0x1b07b9380>, <Element supplementary-material at 0x1b0d87f40>, <Element supplementary-material at 0x1b07b9000>, <Element supplementary-material at 0x1b07ba900>, <Element supplementary-material at 0x1b07b9300>, <Element supplementary-material at 0x1b07bae00>, <Element supplementary-material at 0x1b07bbb40>, <Element supplementary-material at 0x1b07bb3c0>, <Element supplementary-material at 0x1b07b8b40>, <Element supplementary-material at 0x1b61266c0>, <Element supplementary-material at 0x1b5d69d80>, <Element supplementary-material at 0x1b6513a40>, <Element supplementary-material at 0x1b0903a00>, <Element supplementary-material at 0x1b116fe40>, <Element supplementary-material at 0x1b71fc7c0>, <Element supplementary-material at 0x1b71fde40>, <Element supplementary-material at 0x1b5ea4ec0>, <Element supplementary-material at 0x1b7147680>, <Element supplementary-material at 0x1b616bc80>, <Element supplementary-material at 0x1b6169980>, <Element supplementary-material at 0x1b6169200>, <Element supplementary-material at 0x1b0af3a80>, <Element supplementary-material at 0x1b0af3780>, <Element supplementary-material at 0x1b0af2c00>, <Element supplementary-material at 0x1b1128c40>, <Element supplementary-material at 0x1b112a2c0>, <Element supplementary-material at 0x1b1129740>, <Element supplementary-material at 0x1b112bd00>, <Element supplementary-material at 0x1b1129880>, <Element supplementary-material at 0x1b112ba00>, <Element supplementary-material at 0x1b1128a00>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM1: The procedures for sample selection are shown in Supplementary Fig. 1 [this file] . More details for the distribution of brain 5hmC data can be found in Supplementary Fig. 2 [this file] – 3 [this file] . \n",
      "Of the 2821 DhMRs (606 associated with AD diagnosis, 1542 associated with Aβ load, 1399 associated with PHFtau tangles, annotated to 2642 distinct genes) identified in stage II (replication), about 69% reside in exons, ~15% in intergenic regions, and ~17% in introns (Supplementary Fig. 4a [this file] ). Compared to non-AD associated 5hmC regions (total 194,944), AD-associated regions (total 2821) were significantly more likely to coincide with regions related to transcription start site (TSS), transcription states, and enhancer states, but less likely to co-position with regions corresponding to heterochromatin and quiescent states (Supplementary Fig. 4b [this file] , Supplementary Data 20 [rid=MOESM22] ). \n",
      "Of these, 108 AD-associated, 581 Aβ-associated and 135 PHFtau-associated 5hmC regions were positively correlated with gene expression, whereas 65 AD-associated, 240 Aβ-associated and 53 PHFtau-associated 5hmC regions were negatively correlated with gene expression (Supplementary Fig. 5 [this file] , Supplementary Data 22 [rid=MOESM24] ). These genes were clustered into multiple protein-protein interaction (PPI) networks (Supplementary Fig. 6 [this file] ). The most significant biological processes and pathways of these functional clusters were associated with mTOR signaling, mediator complex and transcriptional regulation of RNA polymerase transcripts (II and III), SWI/SNF and npBAF complex, γ-secretase complex, DNA replication and repair, etc. (Supplementary Fig. 6 [this file] , Supplementary Data 24 [rid=MOESM26] ). \n",
      "At GWAS threshold of 1 × 10 , genetic variants residing in regions harboring AD-associated 5hmC alterations were significantly enriched in DNase I hypersensitive sites (DHSs), transcribed histone modifications, TSS, and transcribed chromatin states, but depleted in repressed chromatin states and H3K27me3 in various tissues such as blood, blood vessel, embryonic stem cells (ESCs), colon, liver, and fetal intestine(Supplementary Figs. 7 [this file] – 9 [this file] ; Supplementary Data 30 [rid=MOESM32] – 32 [rid=MOESM34] ). \n",
      "We also performed colocalization analysis using summary statistics from a cell-type specific -eQTL analysis of eight human brain cell types and found that one of the colocalized genes, , is expressed in both astrocytes (Supplementary Fig. 10 [this file] ) and microglia (Supplementary Fig. 11 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM2: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM3: After data pre-processing and quality control, our dataset included 197,765 5hmC regions (Supplementary Data 1 [this file] ) in 1050 older individuals (mean age: 89.5, age range: 65.9–108.3, Table 1 [rid=Tab1] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM4: This analysis resulted in 2939 DhMRs associated with AD diagnosis (Supplementary Data 2 [this file] ), 1,770 DhMRs associated with Aβ load (Supplementary Data 3 [rid=MOESM5] ), and 3110 DhMRs associated with PHFtau tangles (Supplementary Data 4 [rid=MOESM6] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM5: This analysis resulted in 2939 DhMRs associated with AD diagnosis (Supplementary Data 2 [rid=MOESM4] ), 1,770 DhMRs associated with Aβ load (Supplementary Data 3 [this file] ), and 3110 DhMRs associated with PHFtau tangles (Supplementary Data 4 [rid=MOESM6] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM6: This analysis resulted in 2939 DhMRs associated with AD diagnosis (Supplementary Data 2 [rid=MOESM4] ), 1,770 DhMRs associated with Aβ load (Supplementary Data 3 [rid=MOESM5] ), and 3110 DhMRs associated with PHFtau tangles (Supplementary Data 4 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM7: At the Benjamin-Hochberg (BH) corrected false discovery rate (FDR) < 0.05, 2705 regions (92% of EN-selected) were associated with AD diagnosis (Supplementary Data 5 [this file] ), 1750 regions (98.9% of EN-selected) were associated with Aβ load (Supplementary Data 6 [rid=MOESM8] ) and 2626 regions (84.4% of EN-selected) were associated with PHFtau tangles (Supplementary Data 7 [rid=MOESM9] ) in the regression analysis. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM8: At the Benjamin-Hochberg (BH) corrected false discovery rate (FDR) < 0.05, 2705 regions (92% of EN-selected) were associated with AD diagnosis (Supplementary Data 5 [rid=MOESM7] ), 1750 regions (98.9% of EN-selected) were associated with Aβ load (Supplementary Data 6 [this file] ) and 2626 regions (84.4% of EN-selected) were associated with PHFtau tangles (Supplementary Data 7 [rid=MOESM9] ) in the regression analysis. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM9: At the Benjamin-Hochberg (BH) corrected false discovery rate (FDR) < 0.05, 2705 regions (92% of EN-selected) were associated with AD diagnosis (Supplementary Data 5 [rid=MOESM7] ), 1750 regions (98.9% of EN-selected) were associated with Aβ load (Supplementary Data 6 [rid=MOESM8] ) and 2626 regions (84.4% of EN-selected) were associated with PHFtau tangles (Supplementary Data 7 [this file] ) in the regression analysis. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM10: At FDR < 0.05, a total of 2821 DhMRs, including 606 (22.4% of those identified in the discovery sample) associated with AD diagnosis, 1542 (88.1% of those identified in the discovery sample) associated with Aβ load, and 1399 (53.3% of those identified in the discovery sample) associated with PHFtau tangles, remained statistically significant with the same direction of association (Supplementary Data 8 [this file] – 10 [rid=MOESM12] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM11: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM12: At FDR < 0.05, a total of 2821 DhMRs, including 606 (22.4% of those identified in the discovery sample) associated with AD diagnosis, 1542 (88.1% of those identified in the discovery sample) associated with Aβ load, and 1399 (53.3% of those identified in the discovery sample) associated with PHFtau tangles, remained statistically significant with the same direction of association (Supplementary Data 8 [rid=MOESM10] – 10 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM13: Of the 2821 DhMRs replicated in ROS, 93 regions were significantly associated with all three neuropathological phenotypes: AD diagnosis, Aβ,and PHFtau (Supplementary Data 11 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM14: Notably, 43 DhMRs (out of 2821) also harbor known AD variants (Supplementary Data 12 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM15: However, the association between 5hmC and AD pathology appeared not to be mediated by genetic variants because additionally adjusting for the genotype of lead SNPs in the candidate regions did not substantially attenuate the observed associations (Supplementary Data 13 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM16: It showed that the observed associations on chromosome 19 remained significant after further controlling for genotype (Supplementary Data 14 [this file] ). \n",
      "The observed associations of 5hmC regions on chromosome 19 with AD pathology are unlikely to be driven by because additional adjustment for genotype did not change the results (Supplementary Data 14 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM17: The effect of an individual 5hmC region on AD pathology was in general small to moderate (Supplementary Data 15 [this file] ). \n",
      "Results for the variance explained in Aβ load and PHFtau tangles are shown in Supplementary Data 15 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM18: At FDR < 0.05, 5hmC alterations in 2065, 1770, and 2972 regions were significantly associated with AD diagnosis (Supplementary Data 16 [this file] ), Aβ (Supplementary Data 17 [rid=MOESM19] ), and PHFtau (Supplementary Data 18 [rid=MOESM20] ), respectively. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM19: At FDR < 0.05, 5hmC alterations in 2065, 1770, and 2972 regions were significantly associated with AD diagnosis (Supplementary Data 16 [rid=MOESM18] ), Aβ (Supplementary Data 17 [this file] ), and PHFtau (Supplementary Data 18 [rid=MOESM20] ), respectively. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM20: At FDR < 0.05, 5hmC alterations in 2065, 1770, and 2972 regions were significantly associated with AD diagnosis (Supplementary Data 16 [rid=MOESM18] ), Aβ (Supplementary Data 17 [rid=MOESM19] ), and PHFtau (Supplementary Data 18 [this file] ), respectively. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM21: Of these, 118 regions were associated with all three phenotypes (Supplementary Data 19 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM22: Compared to non-AD associated 5hmC regions (total 194,944), AD-associated regions (total 2821) were significantly more likely to coincide with regions related to transcription start site (TSS), transcription states, and enhancer states, but less likely to co-position with regions corresponding to heterochromatin and quiescent states (Supplementary Fig. 4b [rid=MOESM1] , Supplementary Data 20 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM23: Focusing on the 2821 DhMRs associated with AD pathologies (606 regions associated with AD diagnosis, 1542 regions associated with Aβ, 1399 regions associated with PHFtau, annotated to 2642 distinct genes), we evaluated whether altered DNA hydroxymethylation affects the expression of genes located in the vicinity ( ± 50 kb) of each DhMR using brain RNA-seq data (DLPFC) from the same participants (Supplementary Data 21 [this file] ). \n",
      "To further functionally annotate the AD-associated 5hmC regions, we calculated the correlation between the level of 5hmC and the level of H3K9ac generated in the dorsolateral prefrontal cortex from the same individuals (Supplementary Data 21 [this file] ) in ROSMAP . \n",
      "In a subset of individuals (Supplementary Data 21 [this file] ) with both 5hmC and brain proteomics data (DLPFC) , we calculated the correlation of AD-associated 5hmC with protein abundances (Methods). \n",
      "A total of 731 unique individuals with both 5hmC and proteomic data were included in this analysis (Supplementary Data 21 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM24: At FDR < 0.05, the levels of 886 DhMRs (annotated to 818 distinct genes) were correlated with gene expression (Supplementary Data 22 [this file] ). Of these, 108 AD-associated, 581 Aβ-associated and 135 PHFtau-associated 5hmC regions were positively correlated with gene expression, whereas 65 AD-associated, 240 Aβ-associated and 53 PHFtau-associated 5hmC regions were negatively correlated with gene expression (Supplementary Fig. 5 [rid=MOESM1] , Supplementary Data 22 [this file] ). \n",
      "A total of 819 samples have complete data for both 5hmC and RNA-seq data (Supplementary Data 22 [this file] ). \n",
      "A total of 513 ROSMAP participants with both 5hmC and H3K9ac data were included in this analysis (Supplementary Data 22 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM25: Moreover, the expression level for most of these genes (83%) was also significantly associated with AD diagnosis, Aβ or PHFtau (Supplementary Data 23 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM26: The most significant biological processes and pathways of these functional clusters were associated with mTOR signaling, mediator complex and transcriptional regulation of RNA polymerase transcripts (II and III), SWI/SNF and npBAF complex, γ-secretase complex, DNA replication and repair, etc. (Supplementary Fig. 6 [rid=MOESM1] , Supplementary Data 24 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM27: At FDR < 0.05, none of the 5hmC regions associated with AD diagnosis (out of 606) was correlated with H3K9ac, despite that eight regions did show nominal correlation with H3K9ac at FDR < 0.1 (Supplementary Data 25 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM28: In comparison, the levels of 90 Aβ-associated (out of 1542) and 180 PHFtau-associated (out of 1399) 5hmC regions were correlated with the level of H3K9ac at FDR < 0.05 (Supplementary Data 26 [this file] – 27 [rid=MOESM29] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM29: In comparison, the levels of 90 Aβ-associated (out of 1542) and 180 PHFtau-associated (out of 1399) 5hmC regions were correlated with the level of H3K9ac at FDR < 0.05 (Supplementary Data 26 [rid=MOESM28] – 27 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM30: At FDR < 0.05, the expression levels of 91 proteins were significantly correlated with the abundance of 5hmC regions that were significantly associated with AD pathologies (Supplementary Data 28 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM31: These proteins are significantly enriched in biological processes related to amyloid precursor protein catabolic process, regulation of Aβ formation, regulation of signal transduction, neurogenesis, phosphorylation, synapse, protein binding, proteolysis, neuronal ion channel clustering, regulation of protein modification process, etc. (Supplementary Data 29 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM32: At GWAS threshold of 1 × 10 , genetic variants residing in regions harboring AD-associated 5hmC alterations were significantly enriched in DNase I hypersensitive sites (DHSs), transcribed histone modifications, TSS, and transcribed chromatin states, but depleted in repressed chromatin states and H3K27me3 in various tissues such as blood, blood vessel, embryonic stem cells (ESCs), colon, liver, and fetal intestine(Supplementary Figs. 7 [rid=MOESM1] – 9 [rid=MOESM1] ; Supplementary Data 30 [this file] – 32 [rid=MOESM34] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM33: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM34: At GWAS threshold of 1 × 10 , genetic variants residing in regions harboring AD-associated 5hmC alterations were significantly enriched in DNase I hypersensitive sites (DHSs), transcribed histone modifications, TSS, and transcribed chromatin states, but depleted in repressed chromatin states and H3K27me3 in various tissues such as blood, blood vessel, embryonic stem cells (ESCs), colon, liver, and fetal intestine(Supplementary Figs. 7 [rid=MOESM1] – 9 [rid=MOESM1] ; Supplementary Data 30 [rid=MOESM32] – 32 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM35: At PP4 > 0.8 (i.e., a high posterior probability that a single shared variant is responsible for two signals), three 5hmC regions, including one associated with Aβ (C12P044736) and two associated with PHFtau (C12P044736, C14P057817), colocalize with known AD genes ( and ) near the variants rs12582221 and rs12590654 (Supplementary Data 33 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM36: Finally, the observed associations of 5hmC with Aβ or PHFtau slightly attenuated but remained significant after controlling for each other (Supplementary Data 34 [this file] – 35 [rid=MOESM37] ), suggesting that 5hmC alterations may have an independent effect on these two hallmarks of AD pathologies. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM37: Finally, the observed associations of 5hmC with Aβ or PHFtau slightly attenuated but remained significant after controlling for each other (Supplementary Data 34 [rid=MOESM36] – 35 [this file] ), suggesting that 5hmC alterations may have an independent effect on these two hallmarks of AD pathologies. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM38: Further information on research design is available in the Nature Portfolio Reporting Summary [this file] linked to this article. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM39: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM40: Source data are provided as a Source Data [this file] file. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 123 - INFO - Loading patterns for section 'data_availability_sections' for publisher 'PMC'.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 131 - INFO - Section pattern keys for publisher PMC: dict_keys(['data_availability_sections', 'supplementary_material_sections', 'supplementary_data_sections', 'key_resources_table', 'methods_sections'])\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 133 - INFO - Section name: data_availability_sections\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 135 - INFO - Found section 'data_availability_sections' in patterns for publisher 'PMC'.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 137 - INFO - Loaded patterns for section 'data_availability_sections': [\".//sec[@sec-type='data-availability']\", \".//notes[@notes-type='data-availability']\", \".//notes[@notes-type='data-availability']\"]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_data_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 706 - INFO - Found 42 supplementary data sections\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(key_resources_table)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "Batches:   0%|          | 0/2 [00:00<?, ?it/s]"
      ],
      "application/vnd.jupyter.widget-view+json": {
       "version_major": 2,
       "version_minor": 0,
       "model_id": "99643a860a2b4b30980c7f399a3de09f"
      }
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001B[97membeddings_retriever.py - line 51 - INFO - Searching for top-2 passages similar to the query by embeddings.\u001B[0m\n",
      "\u001B[97membeddings_retriever.py - line 33 - INFO - Computing L2 distances using numpy.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 364 - INFO - Processing DAS_content: 4 elements\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 1662\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 758 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-446a6045cc40ec2d005b6de0d7da1ae3ac203365bac5f7ba681784c57dfd3551\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 758 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-72f5560b-3ef6-49ae-9190-e61db77297a4\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE291402\\\",\\n    \\\"data_repository\\\": \\\"Gene Expression Omnibus (GEO)\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE291402\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn11724057\\\",\\n    \\\"data_repository\\\": \\\"Synapse\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.synapse.org/Synapse:syn11724057\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn3388564\\\",\\n    \\\"data_repository\\\": \\\"Synapse\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.synapse.org/#!Synapse:syn3388564\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn17015098\\\",\\n    \\\"data_repository\\\": \\\"Synapse\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.synapse.org/Synapse:syn17015098\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn4896408\\\",\\n    \\\"data_repository\\\": \\\"Synapse\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.synapse.org/Synapse:syn4896408\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GCST90027158\\\",\\n    \\\"data_repository\\\": \\\"National Human Genome Research Institute-European Bioinformatics Institute GWAS catalog\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/gwas/studies/GCST90027158\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE291402\\\",\\n    \\\"data_repository\\\": \\\"Gene Expression Omnibus (GEO)\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE291402\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn11724057\\\",\\n    \\\"data_repository\\\": \\\"Synapse\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.synapse.org/Synapse:syn11724057\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn3388564\\\",\\n    \\\"data_repository\\\": \\\"Synapse\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.synapse.org/#!Synapse:syn3388564\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn17015098\\\",\\n    \\\"data_repository\\\": \\\"Synapse\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.synapse.org/Synapse:syn17015098\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn4896408\\\",\\n    \\\"data_repository\\\": \\\"Synapse\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.synapse.org/Synapse:syn4896408\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GCST90027158\\\",\\n    \\\"data_repository\\\": \\\"National Human Genome Research Institute-European Bioinformatics Institute GWAS catalog\\\",\\n    \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/gwas/studies/GCST90027158\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755541027,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 948,\n",
      "        \"completion_tokens\": 427,\n",
      "        \"total_tokens\": 1375,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE291402', 'data_repository': 'Gene Expression Omnibus (GEO)', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE291402', 'dataset_id': 'GSE291402'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: GSE291402\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID GSE291402 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: Gene Expression Omnibus (GEO), identifier: GSE291402\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE291402 for Gene Expression Omnibus  GEO \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE291402 for Gene Expression Omnibus  GEO \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1043 - INFO - Found partial match for Gene Expression Omnibus  GEO  in Gene Expression Omnibus\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE291402, repo geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE291402\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 961 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'GSE291402', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE291402'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn11724057', 'data_repository': 'Synapse', 'dataset_webpage': 'https://www.synapse.org/Synapse:syn11724057', 'dataset_id': 'syn11724057'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: syn11724057\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID syn11724057 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: Synapse, identifier: syn11724057\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn11724057 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn11724057 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn11724057 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn11724057 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1037 - INFO - Found repo_name match for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.synapse.org/Synapse:syn11724057, repo synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.synapse.org/Synapse:syn11724057\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 961 - INFO - Link matches the pattern https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'syn11724057', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/Synapse:syn11724057'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn3388564', 'data_repository': 'Synapse', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn3388564', 'dataset_id': 'syn3388564'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: syn3388564\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID syn3388564 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: Synapse, identifier: syn3388564\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3388564 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn3388564 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn3388564 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3388564 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1037 - INFO - Found repo_name match for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.synapse.org/#!Synapse:syn3388564, repo synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.synapse.org/#!Synapse:syn3388564\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Link does not match the pattern https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'syn3388564', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn17015098', 'data_repository': 'Synapse', 'dataset_webpage': 'https://www.synapse.org/Synapse:syn17015098', 'dataset_id': 'syn17015098'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: syn17015098\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID syn17015098 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: Synapse, identifier: syn17015098\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn17015098 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn17015098 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn17015098 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn17015098 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1037 - INFO - Found repo_name match for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.synapse.org/Synapse:syn17015098, repo synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.synapse.org/Synapse:syn17015098\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 961 - INFO - Link matches the pattern https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'syn17015098', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/Synapse:syn17015098'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn4896408', 'data_repository': 'Synapse', 'dataset_webpage': 'https://www.synapse.org/Synapse:syn4896408', 'dataset_id': 'syn4896408'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: syn4896408\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID syn4896408 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: Synapse, identifier: syn4896408\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn4896408 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn4896408 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn4896408 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn4896408 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1037 - INFO - Found repo_name match for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.synapse.org/Synapse:syn4896408, repo synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.synapse.org/Synapse:syn4896408\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 961 - INFO - Link matches the pattern https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'syn4896408', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/Synapse:syn4896408'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GCST90027158', 'data_repository': 'National Human Genome Research Institute-European Bioinformatics Institute GWAS catalog', 'dataset_webpage': 'https://www.ebi.ac.uk/gwas/studies/GCST90027158', 'dataset_id': 'GCST90027158'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: GCST90027158\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID GCST90027158 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: National Human Genome Research Institute-European Bioinformatics Institute GWAS catalog, identifier: GCST90027158\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GCST90027158 for National Human Genome Research Institute-European Bioinformatics Institute GWAS catalog\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GCST90027158 for National Human Genome Research Institute-European Bioinformatics Institute GWAS catalog\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier GCST90027158 for National Human Genome Research Institute-European Bioinformatics Institute GWAS catalog\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier GCST90027158 for National Human Genome Research Institute-European Bioinformatics Institute GWAS catalog\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier GCST90027158 for National Human Genome Research Institute-European Bioinformatics Institute GWAS catalog\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: https://www.ebi.ac.uk/gwas/studies/GCST90027158\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ebi.ac.uk/gwas/studies/GCST90027158, repo www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ebi.ac.uk/gwas/studies/GCST90027158\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 967 - INFO - No dataset_webpage_url_ptr found for www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'GCST90027158', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'https://www.ebi.ac.uk/gwas/studies/GCST90027158'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 211\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 396 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-f252e82f4332e60f68f418eacb801c3963f14570385f59fccb87c857d45f8540\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 396 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-a6eca27f-0433-4a47-8134-c9c2d97c2b44\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755541030,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 503,\n",
      "        \"completion_tokens\": 6,\n",
      "        \"total_tokens\": 509,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 935\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 577 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-e341342fd39be3b85dd64336bd7e1cb1e482e957a7baec8a7e8ab7d1f59ebcfd\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 577 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-4ac74d01-9b65-4739-ac90-9fb6d6692031\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"E073\\\",\\n        \\\"data_repository\\\": \\\"NIH Roadmap Epigenomics Project\\\",\\n        \\\"dataset_webpage\\\": \\\"http://www.roadmapepigenomics.org/\\\"\\n    ]\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"E073\\\",\\n        \\\"data_repository\\\": \\\"NIH Roadmap Epigenomics Project\\\",\\n        \\\"dataset_webpage\\\": \\\"http://www.roadmapepigenomics.org/\\\"\\n    ]\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755541031,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 773,\n",
      "        \"completion_tokens\": 63,\n",
      "        \"total_tokens\": 836,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'E073', 'data_repository': 'NIH Roadmap Epigenomics Project', 'dataset_webpage': 'http://www.roadmapepigenomics.org/', 'dataset_id': 'E073'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: E073\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID E073 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: NIH Roadmap Epigenomics Project, identifier: E073\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E073 for NIH Roadmap Epigenomics Project\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E073 for NIH Roadmap Epigenomics Project\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E073 for NIH Roadmap Epigenomics Project\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E073 for NIH Roadmap Epigenomics Project\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier E073 for NIH Roadmap Epigenomics Project\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: http://www.roadmapepigenomics.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: http://www.roadmapepigenomics.org/, repo www.roadmapepigenomics.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: http://www.roadmapepigenomics.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 972 - INFO - Repository www.roadmapepigenomics.org not found in ontology\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'E073', 'data_repository': 'www.roadmapepigenomics.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 3131\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 1126 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-f3159b7d70776e7dbf0ac1e12801340ac727a861d0f86db1588b86f5d7e69216\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 1126 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-81d24d02-b088-4251-adce-1b3631ffaef6\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"syn3388564\\\",\\n        \\\"data_repository\\\": \\\"Synapse\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.synapse.org/#!Synapse:syn3388564\\\"\\n    ]\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"syn3388564\\\",\\n        \\\"data_repository\\\": \\\"Synapse\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.synapse.org/#!Synapse:syn3388564\\\"\\n    ]\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755541032,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 1349,\n",
      "        \"completion_tokens\": 73,\n",
      "        \"total_tokens\": 1422,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn3388564', 'data_repository': 'Synapse', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn3388564', 'dataset_id': 'syn3388564'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: syn3388564\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID syn3388564 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: Synapse, identifier: syn3388564\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3388564 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn3388564 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn3388564 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3388564 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1037 - INFO - Found repo_name match for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.synapse.org/#!Synapse:syn3388564, repo synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.synapse.org/#!Synapse:syn3388564\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Link does not match the pattern https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'syn3388564', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 377 - INFO - datasets (<class 'list'>): [{'dataset_identifier': 'GSE291402', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE291402'}, {'dataset_identifier': 'syn11724057', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/Synapse:syn11724057'}, {'dataset_identifier': 'syn3388564', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn17015098', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/Synapse:syn17015098'}, {'dataset_identifier': 'syn4896408', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/Synapse:syn4896408'}, {'dataset_identifier': 'GCST90027158', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'https://www.ebi.ac.uk/gwas/studies/GCST90027158'}, {'dataset_identifier': 'E073', 'data_repository': 'www.roadmapepigenomics.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn3388564', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'GSE291402', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE291402'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'syn11724057', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/Synapse:syn11724057'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'syn3388564', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'syn17015098', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/Synapse:syn17015098'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'syn4896408', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/Synapse:syn4896408'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'GCST90027158', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'https://www.ebi.ac.uk/gwas/studies/GCST90027158'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'E073', 'data_repository': 'www.roadmapepigenomics.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'syn3388564', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 385 - INFO - Final ret data availability: 8 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 386 - INFO - Final ret data availability: [{'dataset_identifier': 'GSE291402', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE291402', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'syn11724057', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/Synapse:syn11724057', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'syn3388564', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'syn17015098', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/Synapse:syn17015098', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'syn4896408', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/Synapse:syn4896408', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'GCST90027158', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'https://www.ebi.ac.uk/gwas/studies/GCST90027158', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'E073', 'data_repository': 'www.roadmapepigenomics.org', 'dataset_webpage': 'n/a', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'syn3388564', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1079 - INFO - Fetching metadata for 8 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: GSE291402\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: geo, identifier: GSE291402\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE291402 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE291402 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for GSE291402 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE291402\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE291402\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: syn11724057\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn11724057\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn11724057 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn11724057 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn11724057 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn11724057 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for syn11724057 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn11724057\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.synapse.org/Synapse:syn11724057\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: syn3388564\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn3388564\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3388564 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn3388564 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn3388564 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3388564 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for syn3388564 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 3 with repo: synapse.org and accession_id: syn3388564\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.synapse.org/Synapse:syn3388564\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 3: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: syn17015098\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn17015098\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn17015098 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn17015098 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn17015098 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn17015098 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for syn17015098 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 4 with repo: synapse.org and accession_id: syn17015098\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.synapse.org/Synapse:syn17015098\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 4: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: syn4896408\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn4896408\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn4896408 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn4896408 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn4896408 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn4896408 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for syn4896408 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 5 with repo: synapse.org and accession_id: syn4896408\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.synapse.org/Synapse:syn4896408\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 5: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: GCST90027158\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: www.ebi.ac.uk, identifier: GCST90027158\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for GCST90027158 in www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 6 with repo: www.ebi.ac.uk and accession_id: GCST90027158\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ebi.ac.uk/pride/archive/projects/GCST90027158\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 6: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: E073\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: www.roadmapepigenomics.org, identifier: E073\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E073 for www.roadmapepigenomics.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E073 for www.roadmapepigenomics.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E073 for www.roadmapepigenomics.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E073 for www.roadmapepigenomics.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier E073 for www.roadmapepigenomics.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: www.roadmapepigenomics.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 900 - WARNING - URL www.roadmapepigenomics.org may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for E073 in www.roadmapepigenomics.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 7 with repo: www.roadmapepigenomics.org and accession_id: E073\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1152 - WARNING - Repository www.roadmapepigenomics.org unknown in Ontology. Skipping dataset page 7.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: syn3388564\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn3388564\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3388564 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn3388564 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn3388564 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3388564 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for syn3388564 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 8 with repo: synapse.org and accession_id: syn3388564\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.synapse.org/Synapse:syn3388564\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 8: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1156 - INFO - Updated datasets len: 8\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 221 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 48, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'source_section', 'retrieval_pattern', 'access_mode', 'link',\n",
      "       'source_url', 'download_link', 'title', 'content_type', 'id', 'caption',\n",
      "       'description', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 456 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 462 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 539 - INFO - 10th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 347 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 354 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 533 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC7113080&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 383 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b5f91bd0>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 375 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 310 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 399 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 413 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 439 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 25 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#s1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 30) for s1: INTRODUCTION [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s3: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s7'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s7: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s7'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s7: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s8'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s8: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s8'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s8: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s8'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s8: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s8'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s8: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s8'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s8: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s8'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s8: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s8'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s8: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s8'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s8: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s8'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s8: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#s9'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for s9: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#s13'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 28) for s13: DISCUSSION [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#fn-group1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 27) for fn-group1: Footnotes [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#sec20'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 73) for sec20: AUTHORS' DISCLOSURES OF POTENTIAL CONFLICTS OF INTEREST [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#sec20'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 73) for sec20: AUTHORS' DISCLOSURES OF POTENTIAL CONFLICTS OF INTEREST [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#sec20'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 73) for sec20: AUTHORS' DISCLOSURES OF POTENTIAL CONFLICTS OF INTEREST [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#ref-list1_sec2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for ref-list1_sec2: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 384 - INFO - Chunking the HTML content for the parsing step.\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 172 - INFO - Extracting data availability elements from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: section.data-availability-statement\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: section.associated-data\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Data Availability')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: h3.pmc_sec_title:-soup-contains('Data Availability')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Associated Data')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Footnotes')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 221 - INFO - Found 1 data availability elements from HTML.\u001B[0m\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "Batches:   0%|          | 0/1 [00:00<?, ?it/s]"
      ],
      "application/vnd.jupyter.widget-view+json": {
       "version_major": 2,
       "version_minor": 0,
       "model_id": "7d546618778a4a03b571cd62028c6e9e"
      }
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001B[97membeddings_retriever.py - line 51 - INFO - Searching for top-5 passages similar to the query by embeddings.\u001B[0m\n",
      "\u001B[97membeddings_retriever.py - line 33 - INFO - Computing L2 distances using numpy.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 8402\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 2443 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-244ccf9ae80c16f807f8cd8d25df9b132f26dceee29372a41840c383c6388f57\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 2443 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-9d9950ac-ed5a-4ea4-ac78-4080d9626f6c\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"syn17394041\\\",\\n        \\\"data_repository\\\": \\\"Project GENIE\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.synapse.org/#!Synapse:syn17394041\\\"\\n    ]\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"syn17394041\\\",\\n        \\\"data_repository\\\": \\\"Project GENIE\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.synapse.org/#!Synapse:syn17394041\\\"\\n    ]\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755541048,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 2673,\n",
      "        \"completion_tokens\": 76,\n",
      "        \"total_tokens\": 2749,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn17394041', 'data_repository': 'Project GENIE', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn17394041', 'dataset_id': 'syn17394041'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: syn17394041\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID syn17394041 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: Project GENIE, identifier: syn17394041\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn17394041 for Project GENIE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn17394041 for Project GENIE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn17394041 for Project GENIE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn17394041 for Project GENIE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier syn17394041 for Project GENIE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1051 - INFO - Found id_pattern match for Project GENIE in (?<!\\d)syn\\d{7,12}(?!\\d)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.synapse.org/#!Synapse:syn17394041, repo synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.synapse.org/#!Synapse:syn17394041\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Link does not match the pattern https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'syn17394041', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1079 - INFO - Fetching metadata for 1 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: syn17394041\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn17394041\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn17394041 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn17394041 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn17394041 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn17394041 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for syn17394041 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn17394041\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.synapse.org/Synapse:syn17394041\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1156 - INFO - Updated datasets len: 1\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 26, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 450 - INFO - Parsed data extraction completed. Elements collected: 2\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 456 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 466 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 559 - INFO - \n",
      "Progress: 11/25 (44.00%) | Elapsed: 00:02:17 | ETA: 00:02:55\n",
      "\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 539 - INFO - 11th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC8847344)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 347 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC8847344\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 354 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 533 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC8847344&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 395 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC8847344.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 399 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC8847344 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 413 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 417 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC8847344, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 7 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b5f13340>, <Element supplementary-material at 0x1b5f91bc0>, <Element supplementary-material at 0x1b101ba40>, <Element supplementary-material at 0x1b0fa1600>, <Element supplementary-material at 0x1b0fa2b80>, <Element supplementary-material at 0x1b0fa0180>, <Element supplementary-material at 0x1b1072440>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b5f13340>, <Element supplementary-material at 0x1b5f91bc0>, <Element supplementary-material at 0x1b101ba40>, <Element supplementary-material at 0x1b0fa1600>, <Element supplementary-material at 0x1b0fa2b80>, <Element supplementary-material at 0x1b0fa0180>, <Element supplementary-material at 0x1b1072440>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM1: The data set is released in its entirety as Supplementary Materials [this file] to this paper, via an interactive website ( https://cancertrials.io/ (https://cancertrials.io/) ), and through the online Synapse repository (ID: syn25813713) . \n",
      "For trials reporting event-free survival data (e.g., PFS; Supplementary Fig. 1 [this file] ) median = 0.950 (Q1 = 0.909, mean: 0.929, Q3 = 0.970) as compared to a theoretical maximal = 0.996, which corresponds to 5% of observed variance not explained by the Weibull model. This is illustrated in Supplementary Fig. 2a, b [this file] for Weibull fits to OS and PFS data for panitumumab in combination with FOLFIRI and for FOLFIRI alone in wild-type and mutant metastatic colorectal cancer (trial 20050181 ; average = 0.99 for OS curves and 0.92 for PFS curves). \n",
      "Across the entire data set, some of the worst fits for two-parameter Weibull forms were observed for trials with relatively few events, for example, the Chronicle trial ( NCT00427713 (NCT00427713) ) with only eight deaths in the treatment arm and 16 progression events in the observation arm (Fig. 2c [rid=Fig2] ; Supplementary Fig. 1 [this file] ). \n",
      "For a small number of trials in which the asymptote of the fitted survival curve was greater than zero (i.e., patients were expected to be alive at the end of the longest follow-up), a three-parameter Weibull distribution consisting of the traditional two-parameter distribution with an additional “cure rate” term had an improved fit (Supplementary Fig. 3 [this file] ). \n",
      "This is potentially consistent with a two-population hypothesis (Fig. 3a [rid=Fig3] ; Supplementary Fig. 4 [this file] ). Moreover, a mixture model exhibited no meaningful improvement in fit as compared to a single-distribution for OS data from ICI or control arms in these trials (Fig. 3b [rid=Fig3] ; Supplementary Fig. 4 [this file] ). This generated the steep decline in PFS values observable in the control and experimental arms of actual ICI trials, and improved fit to PFS distributions, raising mean from 0.93 to 0.98 (Fig. 3c [rid=Fig3] ; Supplementary Fig. 4 [this file] ; “Methods”). \n",
      "A challenge in the analysis of such data is that, when too few events have occurred, nonparametric numeric confidence estimates return non-informative values (usually reported as a value “not reached” or “indeterminate” as illustrated for two different scenarios in Supplementary Fig. 5a [this file] ). We found that 12-month survival estimates were non-informative for 20–40% of OS trial arms, and 23–61% of event-free survival trial arms, with higher failure rates occurring when the sample size was smaller (Supplementary Fig. 5b, c [this file] ). \n",
      "For event-free survival (for which 99 event-free survival curves could be compared), the precision of a Weibull-based approach was also greater than a nonparametric approach across all sample sizes, while accuracy was equivalent for small sample sizes (fewer than 40 patients) (Supplementary Fig. 6a [this file] ). \n",
      "Thus, parametric fitting of data from 15 patients made a comparably precise and accurate estimate of median PFS as nonparametric analysis of 60 patients (Supplementary Fig. 6b [this file] ). \n",
      "We observed that for a representative simulation, a two-parameter Weibull form accurately described the synthetic trial data ( = 0.98; Supplementary Fig. 7 [this file] ). \n",
      "Parameter values for trials reporting event-free survival followed a similar pattern to OS values (Supplementary Fig. 8 [this file] and Supplementary Data 2 [rid=MOESM5] ). \n",
      "A similar pattern was observed for event-free survival data, with control and experimental values differing by 1.0% for unsuccessful trials and 36% for successful trials (Supplementary Fig. 9 [this file] ). \n",
      "For event-free survival data, 155 comparisons of experimental and control arms from 146 trial figures revealed a range of Δ values from +0.55 to −0.85 (median |Δ | = 0.08) (Supplementary Fig. 9 [this file] ). \n",
      "All data generated or analyzed during this study are included in this published article (and its supplementary information [this file] files). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM2: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM3: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM4: The accuracy of IPD imputation is discussed further in Supplementary Data 1 [this file] and “Methods”. \n",
      "Trials with a hazard ratio difference greater than 0.1, or with perceptible visual differences, were removed from the final data set and not analyzed further (Supplementary Data 1 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM5: Best-fit Weibull parameters were compared across cancer types and metastatic status using an ANOVA test with a Bonferroni correction for multiple hypothesis testing at a two-tailed significance level of 0.05 (see Supplementary Data 2 [this file] ). Parameter values for trials reporting event-free survival followed a similar pattern to OS values (Supplementary Fig. 8 [rid=MOESM1] and Supplementary Data 2 [this file] ). \n",
      "All code used in this study is included in Supplementary Data 2 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM6: Finally, when we examined PFS data from an additional 25 ICI trials, we found that the drop in survival at early event times (which we identified as the time corresponding to the greatest change in the slope of the survival curve) was strongly correlated with the time of the first radiological scan (as reported in trials’ methods sections; Pearson correlation 0.982, < 10 ; Supplementary Data 3 [this file] ). \n",
      "All extracted values can be found in Supplementary Data 3 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM7: Further information on research design is available in the Nature Research Reporting Summary [this file] linked to this article. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 123 - INFO - Loading patterns for section 'data_availability_sections' for publisher 'PMC'.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 131 - INFO - Section pattern keys for publisher PMC: dict_keys(['data_availability_sections', 'supplementary_material_sections', 'supplementary_data_sections', 'key_resources_table', 'methods_sections'])\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 133 - INFO - Section name: data_availability_sections\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 135 - INFO - Found section 'data_availability_sections' in patterns for publisher 'PMC'.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 137 - INFO - Loaded patterns for section 'data_availability_sections': [\".//sec[@sec-type='data-availability']\", \".//notes[@notes-type='data-availability']\", \".//notes[@notes-type='data-availability']\"]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_data_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 706 - INFO - Found 8 supplementary data sections\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(key_resources_table)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "Batches:   0%|          | 0/1 [00:00<?, ?it/s]"
      ],
      "application/vnd.jupyter.widget-view+json": {
       "version_major": 2,
       "version_minor": 0,
       "model_id": "21a205e4130c436e94c0d0ba19fb8bd0"
      }
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001B[97membeddings_retriever.py - line 51 - INFO - Searching for top-2 passages similar to the query by embeddings.\u001B[0m\n",
      "\u001B[97membeddings_retriever.py - line 33 - INFO - Computing L2 distances using numpy.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 364 - INFO - Processing DAS_content: 4 elements\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 312\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 421 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-c2a801129052b5b691873c6c5d7d82865c295e18a354d0e0f558aa9030f3448f\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 421 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-d8daad7d-f4d5-43ca-8659-c2ef8a18bea1\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"[\\n    [\\n        \\\"dataset_identifier\\\": \\\"syn25813713\\\",\\n        \\\"data_repository\\\": \\\"Synapse\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.synapse.org/#!Synapse:syn25813713\\\"\\n    ]\\n]\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"[\\n    [\\n        \\\"dataset_identifier\\\": \\\"syn25813713\\\",\\n        \\\"data_repository\\\": \\\"Synapse\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.synapse.org/#!Synapse:syn25813713\\\"\\n    ]\\n]\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755541054,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 516,\n",
      "        \"completion_tokens\": 70,\n",
      "        \"total_tokens\": 586,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn25813713', 'data_repository': 'Synapse', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn25813713', 'dataset_id': 'syn25813713'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: syn25813713\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID syn25813713 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: Synapse, identifier: syn25813713\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn25813713 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn25813713 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn25813713 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn25813713 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1037 - INFO - Found repo_name match for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.synapse.org/#!Synapse:syn25813713, repo synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.synapse.org/#!Synapse:syn25813713\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Link does not match the pattern https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'syn25813713', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 696\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 517 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-fcc0bf34e5b9538da133e8d57e8fd769911e815a29cd595e3fa9a89dbec834f1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 517 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-19a130dc-a5ba-441f-86c5-a3265c2ab46c\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"syn25813713\\\",\\n        \\\"data_repository\\\": \\\"Synapse\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.synapse.org/#!Synapse:syn25813713\\\"\\n    ]\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"syn25813713\\\",\\n        \\\"data_repository\\\": \\\"Synapse\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.synapse.org/#!Synapse:syn25813713\\\"\\n    ]\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755541055,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 586,\n",
      "        \"completion_tokens\": 75,\n",
      "        \"total_tokens\": 661,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn25813713', 'data_repository': 'Synapse', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn25813713', 'dataset_id': 'syn25813713'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: syn25813713\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID syn25813713 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: Synapse, identifier: syn25813713\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn25813713 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn25813713 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn25813713 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn25813713 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1037 - INFO - Found repo_name match for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.synapse.org/#!Synapse:syn25813713, repo synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.synapse.org/#!Synapse:syn25813713\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Link does not match the pattern https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'syn25813713', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 951\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 581 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-b585a9d5797e01552d9702abc9d3b7db9cafddb09d7d6f71de209324e175a164\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 581 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-d9371963-c348-4cac-806d-9d6317387fbd\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755541056,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 693,\n",
      "        \"completion_tokens\": 6,\n",
      "        \"total_tokens\": 699,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 8632\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 2501 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-1631f1ae4ed89a61bf1b4648867a7a0c50138fd25c6a3a83da5744a5adfd253b\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 2501 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-9b5b0ce9-625c-4fff-a2cb-006aff7ff9ed\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755541056,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 2617,\n",
      "        \"completion_tokens\": 6,\n",
      "        \"total_tokens\": 2623,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 377 - INFO - datasets (<class 'list'>): [{'dataset_identifier': 'syn25813713', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn25813713', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'syn25813713', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'syn25813713', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 385 - INFO - Final ret data availability: 2 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 386 - INFO - Final ret data availability: [{'dataset_identifier': 'syn25813713', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'syn25813713', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1079 - INFO - Fetching metadata for 2 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: syn25813713\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn25813713\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn25813713 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn25813713 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn25813713 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn25813713 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for syn25813713 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn25813713\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.synapse.org/Synapse:syn25813713\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: syn25813713\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn25813713\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn25813713 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn25813713 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn25813713 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn25813713 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for syn25813713 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn25813713\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.synapse.org/Synapse:syn25813713\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1156 - INFO - Updated datasets len: 2\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 221 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 9, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'source_section', 'retrieval_pattern', 'access_mode', 'link',\n",
      "       'source_url', 'download_link', 'title', 'content_type', 'id', 'caption',\n",
      "       'description', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 456 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 462 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 539 - INFO - 12th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC9280291)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 347 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC9280291\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 354 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 533 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC9280291&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 395 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC9280291.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 399 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC9280291 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 413 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 417 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC9280291, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 1 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b0917f40>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b0917f40>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref DS1: The IPP gene set contains 26 immune-related genes and 3 genes used for normalization, and we used those the total of 29 as input in the IPP models ( Supplementary Table 1 [this file] ). \n",
      "In the 7 studies ( 43 [rid=B43] , 45 [rid=B45] , 46 [rid=B46] , 51 [rid=B51] – 53 [rid=B53] , 58 [rid=B58] ) with GE data collected at time points 3 to 7, there were 270 arrays sampled on 173 patients, of whom 134 were alive and 39 (22.5%) deceased at day 30; 122 were used for training and 51 for testing models ( Supplementary Tables 2 [this file] , 3 [this file] ). \n",
      "We ran the COCONUT algorithm on the 17 studies selected for analysis and assessed the effect of co-normalization: (1) on patient-level GE data across studies ( Figure 1 [rid=F1] and Supplementary Figure 1 [this file] ); (2) at the gene level in controls and cases ( Supplementary Figure 2 [this file] presenting data for CD3D); (3) for 2 genes in controls and cases, here with CLDN8 (a housekeeping gene, with minimal difference in mean GE between controls and cases and minimal overall GE variance) and CEACAM1, up-regulated during sepsis ( Supplementary Figure 3 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 123 - INFO - Loading patterns for section 'data_availability_sections' for publisher 'PMC'.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 131 - INFO - Section pattern keys for publisher PMC: dict_keys(['data_availability_sections', 'supplementary_material_sections', 'supplementary_data_sections', 'key_resources_table', 'methods_sections'])\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 133 - INFO - Section name: data_availability_sections\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 135 - INFO - Found section 'data_availability_sections' in patterns for publisher 'PMC'.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 137 - INFO - Loaded patterns for section 'data_availability_sections': [\".//sec[@sec-type='data-availability']\", \".//notes[@notes-type='data-availability']\", \".//notes[@notes-type='data-availability']\"]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_data_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 706 - INFO - Found 2 supplementary data sections\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(key_resources_table)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "Batches:   0%|          | 0/1 [00:00<?, ?it/s]"
      ],
      "application/vnd.jupyter.widget-view+json": {
       "version_major": 2,
       "version_minor": 0,
       "model_id": "8dccf22ecc3748abb84e6243cb56fc0b"
      }
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001B[97membeddings_retriever.py - line 51 - INFO - Searching for top-2 passages similar to the query by embeddings.\u001B[0m\n",
      "\u001B[97membeddings_retriever.py - line 33 - INFO - Computing L2 distances using numpy.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 364 - INFO - Processing DAS_content: 3 elements\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 568\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 485 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-50d05ba033f5b06df7b84e7c50cb386571983a0b63221a8de9f4aa0eac756154\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 485 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-b8af2fb1-c33b-4f73-ae14-af5a50e36197\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"syn5612563\\\",\\n        \\\"data_repository\\\": \\\"Synapse\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.synapse.org/#!Synapse:syn5612563\\\"\\n    ]\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"syn5612563\\\",\\n        \\\"data_repository\\\": \\\"Synapse\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.synapse.org/#!Synapse:syn5612563\\\"\\n    ]\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755541063,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 601,\n",
      "        \"completion_tokens\": 73,\n",
      "        \"total_tokens\": 674,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn5612563', 'data_repository': 'Synapse', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn5612563', 'dataset_id': 'syn5612563'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: syn5612563\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID syn5612563 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: Synapse, identifier: syn5612563\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn5612563 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn5612563 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn5612563 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn5612563 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1037 - INFO - Found repo_name match for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.synapse.org/#!Synapse:syn5612563, repo synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.synapse.org/#!Synapse:syn5612563\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Link does not match the pattern https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'syn5612563', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 826\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 549 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-e730686726a2f9b874e98769413c1042cc346de0832c2946dff2c4a9e6747c2b\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 549 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-77d877d1-5fbf-436d-b067-3bcc7a2172cb\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"syn5612563\\\",\\n        \\\"data_repository\\\": \\\"Synapse\\\",\\n        \\\"dataset_webpage\\\": \\\"https://doi.org/10.7303/syn5612563\\\"\\n    }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"syn5612563\\\",\\n        \\\"data_repository\\\": \\\"Synapse\\\",\\n        \\\"dataset_webpage\\\": \\\"https://doi.org/10.7303/syn5612563\\\"\\n    }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755541064,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 726,\n",
      "        \"completion_tokens\": 74,\n",
      "        \"total_tokens\": 800,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn5612563', 'data_repository': 'Synapse', 'dataset_webpage': 'https://doi.org/10.7303/syn5612563', 'dataset_id': 'syn5612563'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: syn5612563\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID syn5612563 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: Synapse, identifier: syn5612563\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn5612563 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn5612563 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn5612563 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn5612563 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1037 - INFO - Found repo_name match for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://doi.org/10.7303/syn5612563, repo synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.synapse.org/#!Synapse:syn5612563\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Link does not match the pattern https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'syn5612563', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 1806\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 794 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-c1c4a9f1789f2ab009b42f61f95955ad926803a35299b684f8eaee2686d7cdb1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 794 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-f500eee0-ecf4-414d-9f34-ab5a67be26e1\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755541065,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 912,\n",
      "        \"completion_tokens\": 6,\n",
      "        \"total_tokens\": 918,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 377 - INFO - datasets (<class 'list'>): [{'dataset_identifier': 'syn5612563', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn5612563', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'syn5612563', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'syn5612563', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 385 - INFO - Final ret data availability: 2 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 386 - INFO - Final ret data availability: [{'dataset_identifier': 'syn5612563', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'syn5612563', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1079 - INFO - Fetching metadata for 2 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: syn5612563\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn5612563\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn5612563 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn5612563 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn5612563 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn5612563 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for syn5612563 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn5612563\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.synapse.org/Synapse:syn5612563\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: syn5612563\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn5612563\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn5612563 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn5612563 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn5612563 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn5612563 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for syn5612563 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn5612563\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.synapse.org/Synapse:syn5612563\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1156 - INFO - Updated datasets len: 2\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 221 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 3, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'source_section', 'retrieval_pattern', 'access_mode', 'link',\n",
      "       'source_url', 'download_link', 'title', 'content_type', 'id', 'caption',\n",
      "       'description', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 456 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 462 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 539 - INFO - 13th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10029021)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 347 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10029021\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 354 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 533 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC10029021&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 395 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC10029021.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 399 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC10029021 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 413 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 417 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC10029021, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 1 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b5c3cd80>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b5c3cd80>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD1: S1 [this file] ) comprising 672 distinct donors. S2 [this file] ; Methods [this file] ), we retained 654 samples with matched genotype and cortical RNA-seq data spanning 4-39 PCW, the majority falling within the first two trimesters (Tri1, n=216; Tri2, n=433; Table S1 [this file] ). S3 [this file] ; Table S1 [this file] ; Methods [this file] ). \n",
      "Compared with adult brain eQTL datasets from PsychENCODE ( 13 [rid=R13] , 14 [rid=R14] ) and GTEx ( 16 [rid=R16] ), we identified 2,488 unique “fetal-specific” eGenes in this dataset ( Fig. 1B [rid=F1] ; Table S1 [this file] ). S3 [this file] ). S3 [this file] ), suggesting that these context-specific annotations may be more relevant for interpretation of neuropsychiatric GWAS signals, which are known to be under negative selection. S3 [this file] ; Table S1 [this file] ). \n",
      "S4 [this file] ; Fig. S5 [this file] ; Methods [this file] ), identifying 11,845 and 7,490 genes with a significant isoQTL or sQTL (termed “isoGenes” and “sGenes”; Table S1 [this file] ), respectively. As expected, -xQTL are strongly clustered near their target genes’ transcription start site (TSS) ( Fig. 1E [rid=F1] ) and are correspondingly enriched in functional regions of the genome ( Fig. 1F [rid=F1] ; Table S1 [this file] ). To investigate the variant-level overlap among -e/iso/sQTLs, we calculated Storey’s pi1 statistic ( 40 [rid=R40] ) and found that genetic variants directly associated with RNA splicing (especially isoQTL) capture independent and additional signals from eQTLs ( Fig. 1H [rid=F1] ; Methods [this file] ). \n",
      "S3 [this file] ; Table S1 [this file] ; Methods [this file] ), we identified multiple independent regulatory signals for 3,570 eGenes in the fetal brain, with some exhibiting up to 10 groups of conditionally independent -eQTL signals. Each CS prioritized a median of 5 SNPs, with 2,423 containing exactly one SNP, which are strong candidates for functional validation ( Data S1 [this file] ). \n",
      "S6 [this file] ; Table S1 [this file] ; Methods [this file] ). \n",
      "S2 [this file] ; Methods [this file] ), we inferred genetic ancestries in the fetal brain samples in reference to the 1000 Genomes ( 44 [rid=R44] ). S1 [this file] ; Table S1 [this file] ). Following the multi-ancestry QTL mapping pipeline, we independently mapped xQTLs in the three largest ancestry groupings: EUR, AMR, and AFR. 986 eGenes were shared between the multi-ancestry dataset and the three ancestry groupings ( Fig. 2B [rid=F2] ; Table S2 [this file] ). We then performed statistical fine-mapping separately in the ancestry groupings ( 42 [rid=R42] ), and show that AFR, despite of the smallest sample size, has on average the smallest credible sets (CS) of causal variants ( Fig. 2D [rid=F2] ; Data S2 [this file] , S3 [this file] , S4 [this file] ). \n",
      "S7 [this file] ; Table S3 [this file] ). Mirroring eQTL results, we found that heritability drops significantly from Tri1 to Tri2 ( Fig. 3C [rid=F3] ; Wilcoxon p < 2.2e-16; Table S3 [this file] ; Data S5 [this file] , S6 [this file] ). \n",
      "S7 [this file] ), gene expression levels, or phenotypic variance across Tri1 and Tri2 that could be driving these results ( Fig. S8 [this file] ). S7 [this file] ; Table S3 [this file] ; Data S7 [this file] , S8 [this file] ). \n",
      "We generated continuous annotations for genetic variants based on fine-mapping posterior probabilities ( 51 [rid=R51] ) and found SCZ GWAS signals to be highly enriched among fetal brain regulatory elements -- more so than in the adult brain ( Fig. 4B [rid=F4] ; Table S4 [this file] ; Methods [this file] ). S9 [this file] ; Table S4 [this file] ; Methods [this file] ). S9 [this file] ; Table S4 [this file] ). \n",
      "Across these traits, we consistently observed greater heritability mediated by isoform regulation (isoform h /h 6.45±1.92%) than that of gene expression (3.15±1.25%) or splicing (2.13±1.74%), although the overall extent of mediation remains small ( Fig. 4C [rid=F4] ; Table S4 [this file] ; Methods [this file] ). Extending these analyses across trimester-specific annotations, although less well powered, pointed to isoform-regulation in Tri2 as being particularly important, especially for BIP ( Fig. 4D [rid=F4] ; Table S4 [this file] ). \n",
      "S10 [this file] ; Table S5 [this file] ; Methods [this file] ). \n",
      "S10 [this file] ; Table S5 [this file] ). S10 [this file] ) ( 10 [rid=R10] ). S10 [this file] ). \n",
      "Other novel disease mechanisms prioritized by these analyses include: downregulation of the GABA -receptor subunit in SCZ (eQTL: rs514733, CLPP=0.32; Fig. 5B [rid=F5] ; Table S5 [this file] ), providing genetic evidence for GABAergic dysfunction hypothesized to contribute to disease pathophysiology ( 60 [rid=R60] ); splicing dysregulation of in BIP (top colocalization sQTL: rs11750832, CLPP=0.97; Fig. S10 [this file] ; Table S5 [this file] ) which encodes adenylate cyclase, a cell membrane-bound enzyme regulating cAMP signaling and a target of the mood stabilizer lithium ( 61 [rid=R61] ); splicing dysregulation in BIP of (sQTL: rs17183814, CLPP=0.70; Fig. S10 [this file] ; Table S5 [this file] ), a voltage gated sodium channel subunit with rare variant associations in ASD and epilepsy ( 62 [rid=R62] , 63 [rid=R63] ); and in MDD, the dysregulated splicing of (CLPP=0.43; Fig. S10 [this file] ; Table S5 [this file] ), a highly constrained E3 ubiquitin-protein ligase and regulator of N6-methyladenosine RNA modification involved in neural-immune activation ( 64 [rid=R64] ). \n",
      "We next conducted an isoform-centric transcriptome-wide association study (isoTWAS) ( 65 [rid=R65] ), to identify genes and their isoforms whose regulated expression is associated with SCZ risk. isoTWAS identified 536 isoforms across 271 unique genes with significant isoTWAS associations (Bonferroni-corrected P < 0.05 across genes, FWER-corrected P < 0.05 across transcripts of the same gene, permutation P < 0.05; Methods [this file] ). To leverage local LD and to account for SNP weight correlations, we performed fine-mapping on isoTWAS associations that passed permutation testing ( Fig. 5D [rid=F5] ; Table S5 [this file] ). \n",
      "Among 14 TWAS-identified genes with >3 conditionally independent eQTLs, the MRLocus framework provided additional support for 7 genes with FDR<0.2 ( Table S5 [this file] ; Methods [this file] ). S11 [this file] ). \n",
      "In total, we identified 124 gene and isoform co-expression modules demonstrating enrichment for all major developmental cell types ( 36 [rid=R36] ) and recapitulating early biochemical processes and developmental pathways ( Fig. 6A [rid=F6] ; Methods [this file] ). \n",
      "Overlaying enrichment for common and rare variation within the hierarchical structure defined by co-expression pinpointed groups of modules that harbor most of neurodevelopmental disease risk ( Fig. 6B [rid=F6] ; Table S6 [this file] ; Methods [this file] ). S12 [this file] ). \n",
      "S12 [this file] ), which exhibits strong enrichment for cross-disorder common variation and rare variation risk for ASD, epilepsy, and SCZ. S12 [this file] ), which strongly enriched for synaptic gene ontologies and excitatory neuron marker genes. S12 [this file] ). \n",
      "S12 [this file] ) displayed robust BIP, education attainment (EA), and SCZ enrichment while harboring high risk hub genes like and – similar to the prenatally-enriched M37 module described in Li et al ( 18 [rid=R18] ). S12 [this file] ). \n",
      "CellWalker mapped 21.7% of eQTL-containing genes to a specific cellular context in the developing human brain, corresponding to 3,739 of the bulk-derived eQTLs ( Fig. 7A [rid=F7] ; Table S7 [this file] ; Methods [this file] ). To identify such ieQTLs, in which SNP-gene associations are modulated by the levels of a given module, we tested the interaction effect between genotype and module eigengenes on gene and isoform expression ( Fig. 7B [rid=F7] ; Methods [this file] ). Across all gene and isoform modules, 8,008 ieQTLs were identified after the permutation test (BH-corrected FDR<0.05; Methods [this file] ), in which 3,960 ieQTLs were specific to a single module ( Fig. 7C [rid=F7] ; Data S9 [this file] ). Finally, integrating disease GWAS, we identified significant colocalizations (CLPP>0.01) between ieQTLs and 22 SCZ GWAS loci, 13 of which are not found with bulk cA-eQTLs ( Table S5 [this file] ). \n",
      "See Methods [this file] for details. ( ) A schematic of module interaction ieQTL mapping and validation in cultured neurons and progenitors. ( ) Results from module ieQTL mapping. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 123 - INFO - Loading patterns for section 'data_availability_sections' for publisher 'PMC'.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 131 - INFO - Section pattern keys for publisher PMC: dict_keys(['data_availability_sections', 'supplementary_material_sections', 'supplementary_data_sections', 'key_resources_table', 'methods_sections'])\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 133 - INFO - Section name: data_availability_sections\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 135 - INFO - Found section 'data_availability_sections' in patterns for publisher 'PMC'.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 137 - INFO - Loaded patterns for section 'data_availability_sections': [\".//sec[@sec-type='data-availability']\", \".//notes[@notes-type='data-availability']\", \".//notes[@notes-type='data-availability']\"]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_data_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 706 - INFO - Found 2 supplementary data sections\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(key_resources_table)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "Batches:   0%|          | 0/1 [00:00<?, ?it/s]"
      ],
      "application/vnd.jupyter.widget-view+json": {
       "version_major": 2,
       "version_minor": 0,
       "model_id": "a1d5fcfb208c4908ba36f221ec065141"
      }
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001B[97membeddings_retriever.py - line 51 - INFO - Searching for top-2 passages similar to the query by embeddings.\u001B[0m\n",
      "\u001B[97membeddings_retriever.py - line 33 - INFO - Computing L2 distances using numpy.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 364 - INFO - Processing DAS_content: 3 elements\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 185\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 389 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-4fe0f8151e50a301f713526b1d43bb97017cfab110b16f66b520e8f3de5c4568\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 389 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-78c6c448-e833-45c5-993e-80ae47d8818e\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"syn50897018.3\\\",\\n        \\\"data_repository\\\": \\\"Synapse\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.synapse.org/#!Synapse:syn50897018.3/datasets/\\\"\\n    ]\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"syn50897018.3\\\",\\n        \\\"data_repository\\\": \\\"Synapse\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.synapse.org/#!Synapse:syn50897018.3/datasets/\\\"\\n    ]\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755541071,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 513,\n",
      "        \"completion_tokens\": 81,\n",
      "        \"total_tokens\": 594,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn50897018.3', 'data_repository': 'Synapse', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn50897018.3/datasets/', 'dataset_id': 'syn50897018.3'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: syn50897018.3\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID syn50897018.3 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: Synapse, identifier: syn50897018.3\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn50897018.3 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn50897018.3 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn50897018.3 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn50897018.3 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1037 - INFO - Found repo_name match for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.synapse.org/#!Synapse:syn50897018.3/datasets/, repo synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.synapse.org/#!Synapse:syn50897018.3/datasets/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Link does not match the pattern https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'syn50897018.3', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 371\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 436 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-f3b4cfc8f8af788cb050129f9488d50aeda3a064aeeed8566c5ec92e32faca34\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 436 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-362b2a62-0c65-4d0c-88bb-b4d098c9de25\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"[\\n    [\\n        \\\"dataset_identifier\\\": \\\"syn50897018.3\\\",\\n        \\\"data_repository\\\": \\\"Synapse\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.synapse.org/#!Synapse:syn50897018.3/datasets/\\\"\\n    ]\\n]\\n\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"[\\n    [\\n        \\\"dataset_identifier\\\": \\\"syn50897018.3\\\",\\n        \\\"data_repository\\\": \\\"Synapse\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.synapse.org/#!Synapse:syn50897018.3/datasets/\\\"\\n    ]\\n]\\n\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755541072,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 604,\n",
      "        \"completion_tokens\": 77,\n",
      "        \"total_tokens\": 681,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn50897018.3', 'data_repository': 'Synapse', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn50897018.3/datasets/', 'dataset_id': 'syn50897018.3'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: syn50897018.3\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID syn50897018.3 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: Synapse, identifier: syn50897018.3\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn50897018.3 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn50897018.3 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn50897018.3 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn50897018.3 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1037 - INFO - Found repo_name match for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.synapse.org/#!Synapse:syn50897018.3/datasets/, repo synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.synapse.org/#!Synapse:syn50897018.3/datasets/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Link does not match the pattern https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'syn50897018.3', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 9226\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 2649 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-72275783ca1294b08c4a0ea496d5da316c11e8da8f1811207dc89766fd333ae5\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 2649 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-13538f0e-7811-43a6-9644-c09dac51893e\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"GTEx\\\",\\n        \\\"data_repository\\\": \\\"GTEx\\\",\\n        \\\"dataset_webpage\\\": \\\"https://gtexportal.org/home/\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"PsychENCODE\\\",\\n        \\\"data_repository\\\": \\\"PsychENCODE\\\",\\n        \\\"dataset_webpage\\\": \\\"http://resource.psychencode.org/\\\"\\n    }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"GTEx\\\",\\n        \\\"data_repository\\\": \\\"GTEx\\\",\\n        \\\"dataset_webpage\\\": \\\"https://gtexportal.org/home/\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"PsychENCODE\\\",\\n        \\\"data_repository\\\": \\\"PsychENCODE\\\",\\n        \\\"dataset_webpage\\\": \\\"http://resource.psychencode.org/\\\"\\n    }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755541073,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 3174,\n",
      "        \"completion_tokens\": 102,\n",
      "        \"total_tokens\": 3276,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GTEx', 'data_repository': 'GTEx', 'dataset_webpage': 'https://gtexportal.org/home/', 'dataset_id': 'GTEx'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: GTEx\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID GTEx is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: GTEx, identifier: GTEx\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GTEx for GTEx\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GTEx for GTEx\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier GTEx for GTEx\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier GTEx for GTEx\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier GTEx for GTEx\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: https://gtexportal.org/home/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://gtexportal.org/home/, repo gtexportal.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://gtexportal.org/home/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 972 - INFO - Repository gtexportal.org not found in ontology\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'GTEx', 'data_repository': 'gtexportal.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'PsychENCODE', 'data_repository': 'PsychENCODE', 'dataset_webpage': 'http://resource.psychencode.org/', 'dataset_id': 'PsychENCODE'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: PsychENCODE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID PsychENCODE is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: PsychENCODE, identifier: PsychENCODE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier PsychENCODE for PsychENCODE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier PsychENCODE for PsychENCODE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier PsychENCODE for PsychENCODE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier PsychENCODE for PsychENCODE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier PsychENCODE for PsychENCODE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: http://resource.psychencode.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: http://resource.psychencode.org/, repo resource.psychencode.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: http://resource.psychencode.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 972 - INFO - Repository resource.psychencode.org not found in ontology\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'PsychENCODE', 'data_repository': 'resource.psychencode.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 377 - INFO - datasets (<class 'list'>): [{'dataset_identifier': 'syn50897018.3', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn50897018.3', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'GTEx', 'data_repository': 'gtexportal.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'PsychENCODE', 'data_repository': 'resource.psychencode.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'syn50897018.3', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'syn50897018.3', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'GTEx', 'data_repository': 'gtexportal.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'PsychENCODE', 'data_repository': 'resource.psychencode.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 385 - INFO - Final ret data availability: 4 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 386 - INFO - Final ret data availability: [{'dataset_identifier': 'syn50897018.3', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'syn50897018.3', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'GTEx', 'data_repository': 'gtexportal.org', 'dataset_webpage': 'n/a', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'PsychENCODE', 'data_repository': 'resource.psychencode.org', 'dataset_webpage': 'n/a', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1079 - INFO - Fetching metadata for 4 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: syn50897018.3\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn50897018.3\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn50897018.3 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn50897018.3 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn50897018.3 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn50897018.3 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for syn50897018.3 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn50897018.3\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.synapse.org/Synapse:syn50897018.3\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: syn50897018.3\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn50897018.3\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn50897018.3 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn50897018.3 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn50897018.3 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn50897018.3 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for syn50897018.3 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn50897018.3\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.synapse.org/Synapse:syn50897018.3\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: GTEx\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: gtexportal.org, identifier: GTEx\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GTEx for gtexportal.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GTEx for gtexportal.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier GTEx for gtexportal.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier GTEx for gtexportal.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier GTEx for gtexportal.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: gtexportal.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 900 - WARNING - URL gtexportal.org may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for GTEx in gtexportal.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 3 with repo: gtexportal.org and accession_id: GTEx\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1152 - WARNING - Repository gtexportal.org unknown in Ontology. Skipping dataset page 3.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: PsychENCODE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: resource.psychencode.org, identifier: PsychENCODE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier PsychENCODE for resource.psychencode.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier PsychENCODE for resource.psychencode.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier PsychENCODE for resource.psychencode.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier PsychENCODE for resource.psychencode.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier PsychENCODE for resource.psychencode.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: resource.psychencode.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 900 - WARNING - URL resource.psychencode.org may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for PsychENCODE in resource.psychencode.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 4 with repo: resource.psychencode.org and accession_id: PsychENCODE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1152 - WARNING - Repository resource.psychencode.org unknown in Ontology. Skipping dataset page 4.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1156 - INFO - Updated datasets len: 4\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 221 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 5, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'source_section', 'retrieval_pattern', 'access_mode', 'link',\n",
      "       'source_url', 'download_link', 'title', 'content_type', 'id', 'caption',\n",
      "       'description', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 456 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 462 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 539 - INFO - 14th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11525589)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 347 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11525589\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 354 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 533 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC11525589&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 461 - WARNING - No links found in the section.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 395 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC11525589.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 399 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11525589 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 413 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 417 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11525589, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 5 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b74851c0>, <Element supplementary-material at 0x1b5c3de40>, <Element supplementary-material at 0x1b5c3c280>, <Element supplementary-material at 0x1b789c4c0>, <Element supplementary-material at 0x1b789d900>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b74851c0>, <Element supplementary-material at 0x1b5c3de40>, <Element supplementary-material at 0x1b5c3c280>, <Element supplementary-material at 0x1b789c4c0>, <Element supplementary-material at 0x1b789d900>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM1: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM2: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM3: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM4: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM5: \u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 123 - INFO - Loading patterns for section 'data_availability_sections' for publisher 'PMC'.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 131 - INFO - Section pattern keys for publisher PMC: dict_keys(['data_availability_sections', 'supplementary_material_sections', 'supplementary_data_sections', 'key_resources_table', 'methods_sections'])\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 133 - INFO - Section name: data_availability_sections\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 135 - INFO - Found section 'data_availability_sections' in patterns for publisher 'PMC'.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 137 - INFO - Loaded patterns for section 'data_availability_sections': [\".//sec[@sec-type='data-availability']\", \".//notes[@notes-type='data-availability']\", \".//notes[@notes-type='data-availability']\"]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_data_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 706 - INFO - Found 6 supplementary data sections\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(key_resources_table)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "Batches:   0%|          | 0/1 [00:00<?, ?it/s]"
      ],
      "application/vnd.jupyter.widget-view+json": {
       "version_major": 2,
       "version_minor": 0,
       "model_id": "29bfa945518645f882b318a504c14bbc"
      }
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001B[97membeddings_retriever.py - line 51 - INFO - Searching for top-2 passages similar to the query by embeddings.\u001B[0m\n",
      "\u001B[97membeddings_retriever.py - line 33 - INFO - Computing L2 distances using numpy.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 364 - INFO - Processing DAS_content: 3 elements\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 174\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 386 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-4beefa52ea0cdd502f6c475edfbb0f8065b2ff5be120194973080b8e5cf53617\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 386 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-668caa25-be62-42a1-9cd3-80013739fc13\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"syn51514106\\\",\\n        \\\"data_repository\\\": \\\"Synapse\\\",\\n        \\\"dataset_webpage\\\": \\\"https://doi.org/10.7303/syn51514106\\\"\\n    }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"syn51514106\\\",\\n        \\\"data_repository\\\": \\\"Synapse\\\",\\n        \\\"dataset_webpage\\\": \\\"https://doi.org/10.7303/syn51514106\\\"\\n    }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755541078,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 504,\n",
      "        \"completion_tokens\": 76,\n",
      "        \"total_tokens\": 580,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn51514106', 'data_repository': 'Synapse', 'dataset_webpage': 'https://doi.org/10.7303/syn51514106', 'dataset_id': 'syn51514106'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: syn51514106\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID syn51514106 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: Synapse, identifier: syn51514106\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn51514106 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn51514106 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn51514106 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn51514106 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1037 - INFO - Found repo_name match for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://doi.org/10.7303/syn51514106, repo synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.synapse.org/#!Synapse:syn51514106\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Link does not match the pattern https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'syn51514106', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 542\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 478 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-fb5af0ec5c531d5a8eeef77dd1696af620540cea66382eb9eea97943be380270\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 478 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-ed8800fe-6bff-4dc0-b308-9fc83d665ab8\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755541079,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 575,\n",
      "        \"completion_tokens\": 6,\n",
      "        \"total_tokens\": 581,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 10968\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 3085 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-1178b047cc879f3d9ea9cdc9a4d01ba488649af98e5cc46490ecb78dde92f2af\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 3085 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-64cb8458-e3cf-4d87-a4c0-2d9ab594d13e\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"BraTS-MEN\\\",\\n        \\\"data_repository\\\": \\\"multi-site, heterogeneous, multi-parametric MRI (mpMRI) meningioma dataset\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.synapse.org/#!Synapse:syn20875330/wiki/\\\"\\n    }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"BraTS-MEN\\\",\\n        \\\"data_repository\\\": \\\"multi-site, heterogeneous, multi-parametric MRI (mpMRI) meningioma dataset\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.synapse.org/#!Synapse:syn20875330/wiki/\\\"\\n    }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755541080,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 3227,\n",
      "        \"completion_tokens\": 87,\n",
      "        \"total_tokens\": 3314,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'BraTS-MEN', 'data_repository': 'multi-site, heterogeneous, multi-parametric MRI (mpMRI) meningioma dataset', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn20875330/wiki/', 'dataset_id': 'BraTS-MEN'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: BraTS-MEN\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID BraTS-MEN is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: multi-site, heterogeneous, multi-parametric MRI (mpMRI) meningioma dataset, identifier: BraTS-MEN\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier BraTS-MEN for multi-site, heterogeneous, multi-parametric MRI  mpMRI  meningioma dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier BraTS-MEN for multi-site, heterogeneous, multi-parametric MRI  mpMRI  meningioma dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier BraTS-MEN for multi-site, heterogeneous, multi-parametric MRI  mpMRI  meningioma dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier BraTS-MEN for multi-site, heterogeneous, multi-parametric MRI  mpMRI  meningioma dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier BraTS-MEN for multi-site, heterogeneous, multi-parametric MRI  mpMRI  meningioma dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: https://www.synapse.org/#!Synapse:syn20875330/wiki/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.synapse.org/#!Synapse:syn20875330/wiki/, repo www.synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.synapse.org/#!Synapse:syn20875330/wiki/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 972 - INFO - Repository www.synapse.org not found in ontology\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'BraTS-MEN', 'data_repository': 'www.synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 377 - INFO - datasets (<class 'list'>): [{'dataset_identifier': 'syn51514106', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'BraTS-MEN', 'data_repository': 'www.synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'syn51514106', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'BraTS-MEN', 'data_repository': 'www.synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 385 - INFO - Final ret data availability: 2 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 386 - INFO - Final ret data availability: [{'dataset_identifier': 'syn51514106', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'BraTS-MEN', 'data_repository': 'www.synapse.org', 'dataset_webpage': 'n/a', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1079 - INFO - Fetching metadata for 2 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: syn51514106\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn51514106\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn51514106 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn51514106 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn51514106 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn51514106 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for syn51514106 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn51514106\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.synapse.org/Synapse:syn51514106\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: BraTS-MEN\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: www.synapse.org, identifier: BraTS-MEN\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier BraTS-MEN for www.synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier BraTS-MEN for www.synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier BraTS-MEN for www.synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier BraTS-MEN for www.synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1043 - INFO - Found partial match for www.synapse.org in Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for BraTS-MEN in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 994 - WARNING - Identifier BraTS-MEN does not match pattern for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: BraTS-MEN\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.synapse.org/Synapse:BraTS-MEN\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1156 - INFO - Updated datasets len: 2\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 221 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 7, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'source_section', 'retrieval_pattern', 'access_mode', 'link',\n",
      "       'source_url', 'download_link', 'title', 'content_type', 'id', 'caption',\n",
      "       'description', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 456 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 462 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 539 - INFO - 15th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10836119)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 347 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10836119\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 354 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 533 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC10836119&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 395 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC10836119.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 399 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC10836119 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 413 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 417 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC10836119, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 2 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b71477c0>, <Element supplementary-material at 0x1b09170c0>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b71477c0>, <Element supplementary-material at 0x1b09170c0>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD1: Study characteristics and acylcarnitine descriptions can be found in Supplementary Tables 1 [this file] and 2 [this file] , respectively. \n",
      "The analysis split ADNI participants into two clusters, 2 and 3, at the top of the tree, with significantly different levels of p-tau in cerebrospinal fluid (CSF), glucose uptake measured by [ F] fluorodeoxyglucose-positron emission tomography (FDG-PET), and the 13-item AD assessment scale-cognitive subscale (ADAS-Cog. 13; Fig. 2b [rid=F2] - d [rid=F2] , Supplementary Table 3 [this file] ). All associations remained significant ( Supplementary Table 4 [this file] ). \n",
      "Specifically, we examined acylcarnitine-related SNPs reported by Shin et al. as non-modifiable factors (SNP list is provided in Supplementary Table 5 [this file] , replication in ADNI in Supplementary Table 6 [this file] ) and modifiable acylcarnitine levels (i.e., residuals adjusted for genetic and other non-modifiable factors) to better understand their impact on subgroup separation. \n",
      "Genetic influences only accounted for a small proportion of variance ( Supplementary Table 7 [this file] - 9 [this file] ), except for one epistatic interaction effect of two SNPs: The interplay of rs17806888 (mapped to ) and rs924135 (mapped to ), both of which have been reported to affect succinylcarnitine levels , explained 32% of the initial partitioning of the data into cluster 2 and cluster 3 at the top of the tree ( Fig. 2j [rid=F2] and Supplementary Fig. 2 [rid=SD2] ). When we examined relations to AD, we observed that both single acylcarnitine-related SNPs and epistatic models were not strongly linked to AD and its biomarkers, although some associations were independently confirmed in the MayoLOAD and ROS/MAP studies ( Supplementary Fig. 3 [rid=SD2] and Supplementary Table 10 [this file] ). \n",
      "After cross-normalizing the different cohort datasets for better comparability ( Methods [rid=S6] and Supplementary Fig. 4 [rid=SD2] ), we fitted a multivariable linear model that regresses age on the fasting serum levels of acylcarnitines in the KORA cohort ( Supplementary Fig. 5 [rid=SD2] and Supplementary Table 11 [this file] ). When applied to the ADNI cohort, correlation dropped to 28% (cognitively normal participants only), which was very similar to what we observed when replicating the bioenergetic age computation process in AGES-RS (r = 29%, Supplementary Table 12 [this file] ). \n",
      "Bioenergetic age highlighted interesting relationships between subgroups and AD pathology throughout the tree ( Supplementary Table 13 [this file] ). This was further confirmed by significant associations of bioenergetic age with AD biomarkers across the A-T-(N)-(C) spectrum ( Supplementary Table 14 [this file] ), and consistent findings for cognitive function and grey matter volume in AGES-RS ( Supplementary Table 15 [this file] ). \n",
      "Specifically, bioenergetically younger individuals demonstrated a significantly slower decline over five years ( Supplementary Table 16 [this file] ). This result was replicated using longitudinal diagnosis data from the AGES-RS study ( Fig. 4d [rid=F4] , Supplementary Table 15 [this file] ), providing further evidence that younger bioenergetic age is protective against cognitive decline. \n",
      "We selected the grouping that had the most significant associations across three cognitive measures: ADAS-Cog 13, memory, and executive function ( Supplementary Table 17 [this file] ). \n",
      "The final dataset contained 140 metabolite measurements, covering 23 acylcarnitine species ( Supplementary Table 2 [this file] ), for 1,531 individuals. \n",
      "Data for 22 out of 23 acylcarnitines investigated in ADNI ( Supplementary Table 2 [this file] ) were available here, with the exception being levels of C4:1, which was not reliably measured in KORA. \n",
      "We obtained a list of 32 acylcarnitine-associated SNPs ( Supplementary Table 5 [this file] ) from a large GWAS study performed on a total of 7,824 individuals from the KORA and TwinsUK studies . \n",
      "To investigate robustness of the model, we performed 10-fold, 3 times repeated cross-validation ( Supplementary Table 11 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD2: Notably, the most substantial contributions were observed for medium- and long-chain acylcarnitines, highlighting a likely modifiable role for beta-oxidation function ( Supplementary Fig. 1 [this file] ). \n",
      "Genetic influences only accounted for a small proportion of variance ( Supplementary Table 7 [rid=SD1] - 9 [rid=SD1] ), except for one epistatic interaction effect of two SNPs: The interplay of rs17806888 (mapped to ) and rs924135 (mapped to ), both of which have been reported to affect succinylcarnitine levels , explained 32% of the initial partitioning of the data into cluster 2 and cluster 3 at the top of the tree ( Fig. 2j [rid=F2] and Supplementary Fig. 2 [this file] ). When we examined relations to AD, we observed that both single acylcarnitine-related SNPs and epistatic models were not strongly linked to AD and its biomarkers, although some associations were independently confirmed in the MayoLOAD and ROS/MAP studies ( Supplementary Fig. 3 [this file] and Supplementary Table 10 [rid=SD1] ). \n",
      "After cross-normalizing the different cohort datasets for better comparability ( Methods [rid=S6] and Supplementary Fig. 4 [this file] ), we fitted a multivariable linear model that regresses age on the fasting serum levels of acylcarnitines in the KORA cohort ( Supplementary Fig. 5 [this file] and Supplementary Table 11 [rid=SD1] ). \n",
      "Furthermore, this analysis revealed a strong correlation between bioenergetic age and the first principal component of acylcarnitine levels, indicating that the primary axis of variation in the data and the clustering is associated with bioenergetic age ( Supplementary Fig. 6 [this file] ). \n",
      "Rather, we found that individuals within cluster 7 consistently exhibit the same chronological but significantly younger bioenergetic age than other participants irrespective of their diagnostic group ( Supplementary Fig. 7 [this file] ). \n",
      "We attempted to replicate this grouping, which had similar genotype frequencies as in ADNI (59% vs. 58% in ROS/MAP; Supplementary Fig. 8 [this file] ), using trajectories of global cognition over 13 years in the ROS/MAP cohort. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 123 - INFO - Loading patterns for section 'data_availability_sections' for publisher 'PMC'.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 131 - INFO - Section pattern keys for publisher PMC: dict_keys(['data_availability_sections', 'supplementary_material_sections', 'supplementary_data_sections', 'key_resources_table', 'methods_sections'])\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 133 - INFO - Section name: data_availability_sections\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 135 - INFO - Found section 'data_availability_sections' in patterns for publisher 'PMC'.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 137 - INFO - Loaded patterns for section 'data_availability_sections': [\".//sec[@sec-type='data-availability']\", \".//notes[@notes-type='data-availability']\", \".//notes[@notes-type='data-availability']\"]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_data_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 706 - INFO - Found 3 supplementary data sections\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(key_resources_table)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "Batches:   0%|          | 0/1 [00:00<?, ?it/s]"
      ],
      "application/vnd.jupyter.widget-view+json": {
       "version_major": 2,
       "version_minor": 0,
       "model_id": "be3d6f48f5b84af1a3a7a43cd57439b1"
      }
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001B[97membeddings_retriever.py - line 51 - INFO - Searching for top-2 passages similar to the query by embeddings.\u001B[0m\n",
      "\u001B[97membeddings_retriever.py - line 33 - INFO - Computing L2 distances using numpy.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 364 - INFO - Processing DAS_content: 3 elements\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 2424\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 949 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-fbcf64f0eb4714de026bdbb4421a69feb825dcd3e30a76af367c5e8206d120de\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 949 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-626cd009-bd9b-4d5b-9384-3ffba775769d\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"syn5592519\\\",\\n        \\\"data_repository\\\": \\\"AD Knowledge Portal\\\",\\n        \\\"dataset_webpage\\\": \\\"https://doi.org/10.7303/syn5592519\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"syn9705278\\\",\\n        \\\"data_repository\\\": \\\"AD Knowledge Portal\\\",\\n        \\\"dataset_webpage\\\": \\\"https://doi.org/10.7303/syn9705278\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"syn3157329\\\",\\n        \\\"data_repository\\\": \\\"AD Knowledge Portal\\\",\\n        \\\"dataset_webpage\\\": \\\"https://doi.org/10.7303/syn3157329\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"syn3157322\\\",\\n        \\\"data_repository\\\": \\\"AD Knowledge Portal\\\",\\n        \\\"dataset_webpage\\\": \\\"https://doi.org/10.7303/syn3157322\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"syn3157238\\\",\\n        \\\"data_repository\\\": \\\"AD Knowledge Portal\\\",\\n        \\\"dataset_webpage\\\": \\\"https://doi.org/10.7303/syn3157238\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"syn3205821\\\",\\n        \\\"data_repository\\\": \\\"AD Knowledge Portal\\\",\\n        \\\"dataset_webpage\\\": \\\"https://doi.org/10.7303/syn3205821.6\\\"\\n    }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"syn5592519\\\",\\n        \\\"data_repository\\\": \\\"AD Knowledge Portal\\\",\\n        \\\"dataset_webpage\\\": \\\"https://doi.org/10.7303/syn5592519\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"syn9705278\\\",\\n        \\\"data_repository\\\": \\\"AD Knowledge Portal\\\",\\n        \\\"dataset_webpage\\\": \\\"https://doi.org/10.7303/syn9705278\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"syn3157329\\\",\\n        \\\"data_repository\\\": \\\"AD Knowledge Portal\\\",\\n        \\\"dataset_webpage\\\": \\\"https://doi.org/10.7303/syn3157329\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"syn3157322\\\",\\n        \\\"data_repository\\\": \\\"AD Knowledge Portal\\\",\\n        \\\"dataset_webpage\\\": \\\"https://doi.org/10.7303/syn3157322\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"syn3157238\\\",\\n        \\\"data_repository\\\": \\\"AD Knowledge Portal\\\",\\n        \\\"dataset_webpage\\\": \\\"https://doi.org/10.7303/syn3157238\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"syn3205821\\\",\\n        \\\"data_repository\\\": \\\"AD Knowledge Portal\\\",\\n        \\\"dataset_webpage\\\": \\\"https://doi.org/10.7303/syn3205821.6\\\"\\n    }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755541087,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 1186,\n",
      "        \"completion_tokens\": 412,\n",
      "        \"total_tokens\": 1598,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn5592519', 'data_repository': 'AD Knowledge Portal', 'dataset_webpage': 'https://doi.org/10.7303/syn5592519', 'dataset_id': 'syn5592519'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: syn5592519\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID syn5592519 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: AD Knowledge Portal, identifier: syn5592519\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn5592519 for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn5592519 for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn5592519 for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn5592519 for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier syn5592519 for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1051 - INFO - Found id_pattern match for AD Knowledge Portal in (?<!\\d)syn\\d{7,12}(?!\\d)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://doi.org/10.7303/syn5592519, repo synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.synapse.org/#!Synapse:syn5592519\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Link does not match the pattern https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'syn5592519', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn9705278', 'data_repository': 'AD Knowledge Portal', 'dataset_webpage': 'https://doi.org/10.7303/syn9705278', 'dataset_id': 'syn9705278'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: syn9705278\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID syn9705278 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: AD Knowledge Portal, identifier: syn9705278\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn9705278 for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn9705278 for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn9705278 for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn9705278 for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier syn9705278 for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1051 - INFO - Found id_pattern match for AD Knowledge Portal in (?<!\\d)syn\\d{7,12}(?!\\d)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://doi.org/10.7303/syn9705278, repo synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.synapse.org/#!Synapse:syn9705278\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Link does not match the pattern https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'syn9705278', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn3157329', 'data_repository': 'AD Knowledge Portal', 'dataset_webpage': 'https://doi.org/10.7303/syn3157329', 'dataset_id': 'syn3157329'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: syn3157329\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID syn3157329 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: AD Knowledge Portal, identifier: syn3157329\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3157329 for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn3157329 for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn3157329 for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3157329 for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier syn3157329 for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1051 - INFO - Found id_pattern match for AD Knowledge Portal in (?<!\\d)syn\\d{7,12}(?!\\d)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://doi.org/10.7303/syn3157329, repo synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.synapse.org/#!Synapse:syn3157329\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Link does not match the pattern https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'syn3157329', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn3157322', 'data_repository': 'AD Knowledge Portal', 'dataset_webpage': 'https://doi.org/10.7303/syn3157322', 'dataset_id': 'syn3157322'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: syn3157322\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID syn3157322 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: AD Knowledge Portal, identifier: syn3157322\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3157322 for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn3157322 for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn3157322 for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3157322 for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier syn3157322 for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1051 - INFO - Found id_pattern match for AD Knowledge Portal in (?<!\\d)syn\\d{7,12}(?!\\d)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://doi.org/10.7303/syn3157322, repo synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.synapse.org/#!Synapse:syn3157322\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Link does not match the pattern https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'syn3157322', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn3157238', 'data_repository': 'AD Knowledge Portal', 'dataset_webpage': 'https://doi.org/10.7303/syn3157238', 'dataset_id': 'syn3157238'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: syn3157238\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID syn3157238 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: AD Knowledge Portal, identifier: syn3157238\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3157238 for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn3157238 for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn3157238 for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3157238 for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier syn3157238 for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1051 - INFO - Found id_pattern match for AD Knowledge Portal in (?<!\\d)syn\\d{7,12}(?!\\d)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://doi.org/10.7303/syn3157238, repo synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.synapse.org/#!Synapse:syn3157238\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Link does not match the pattern https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'syn3157238', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn3205821', 'data_repository': 'AD Knowledge Portal', 'dataset_webpage': 'https://doi.org/10.7303/syn3205821.6', 'dataset_id': 'syn3205821'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: syn3205821\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID syn3205821 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: AD Knowledge Portal, identifier: syn3205821\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3205821 for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn3205821 for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn3205821 for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3205821 for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier syn3205821 for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1051 - INFO - Found id_pattern match for AD Knowledge Portal in (?<!\\d)syn\\d{7,12}(?!\\d)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://doi.org/10.7303/syn3205821.6, repo synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.synapse.org/#!Synapse:syn3205821/version/6\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Link does not match the pattern https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'syn3205821', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 2016\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 847 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-9b9dfb0065c4021931e1bc7283ce60c3eeb7b1ba620a8c1b5fbe26ff0cf6d064\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 847 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-c44fe1ee-106b-413a-b0af-35e98696f2f3\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755541090,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 922,\n",
      "        \"completion_tokens\": 6,\n",
      "        \"total_tokens\": 928,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 4338\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 1427 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-51fd1bdccdaa3f86883f4468966b0ac5e01127796c20d4e74bd6f35fc9648837\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 1427 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-fd49a707-b758-40e7-ae07-525793c856dd\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"KORA\\\",\\n        \\\"data_repository\\\": \\\"Helmholtz Zentrum M\\u00fcnchen\\\",\\n        \\\"dataset_webpage\\\": \\\"https://helmholtz-muenchen.managed-otrs.com/external/\\\"\\n    ],\\n    [\\n        \\\"dataset_identifier\\\": \\\"AGES-RS\\\",\\n        \\\"data_repository\\\": \\\"hjarta.is\\\",\\n        \\\"dataset_webpage\\\": \\\"https://hjarta.is/en/research/ages-phase-1/\\\"\\n    ],\\n    [\\n        \\\"dataset_identifier\\\": \\\"ADNI metabolomics data from the AbsoluteIDQ-p180 kit (ADNI-1)\\\",\\n        \\\"data_repository\\\": \\\"AD Knowledge Portal\\\",\\n        \\\"dataset_webpage\\\": \\\"https://doi.org/10.7303/syn5592519\\\"\\n    ],\\n    [\\n        \\\"dataset_identifier\\\": \\\"ADNI metabolomics data from the AbsoluteIDQ-p180 kit (ADNI-GO/\\u22122)\\\",\\n        \\\"data_repository\\\": \\\"AD Knowledge Portal\\\",\\n        \\\"dataset_webpage\\\": \\\"https://doi.org/10.7303/syn9705278\\\"\\n    ],\\n    [\\n        \\\"dataset_identifier\\\": \\\"ADNI clinical and demographic data\\\",\\n        \\\"data_repository\\\": \\\"LONI\\\",\\n        \\\"dataset_webpage\\\": \\\"http://adni.loni.usc.edu/data-samples/access-data/\\\"\\n    ],\\n    [\\n        \\\"dataset_identifier\\\": \\\"ROS/MAP imputed genotype data\\\",\\n        \\\"data_repository\\\": \\\"AD Knowledge Portal\\\",\\n        \\\"dataset_webpage\\\": \\\"https://doi.org/10.7303/syn3157329\\\"\\n    ],\\n    [\\n        \\\"dataset_identifier\\\": \\\"ROS/MAP study meta-data, basic covariates and clinical variables\\\",\\n        \\\"data_repository\\\": \\\"AD Knowledge Portal\\\",\\n        \\\"dataset_webpage\\\": \\\"https://doi.org/10.7303/syn3157322\\\"\\n    ],\\n    [\\n        \\\"dataset_identifier\\\": \\\"ROS/MAP clinical and demographic data\\\",\\n        \\\"data_repository\\\": \\\"RADC\\\",\\n        \\\"dataset_webpage\\\": \\\"http://www.radc.rush.edu/\\\"\\n    ],\\n    [\\n        \\\"dataset_identifier\\\": \\\"MayoLOAD genotyping data\\\",\\n        \\\"data_repository\\\": \\\"AD Knowledge Portal\\\",\\n        \\\"dataset_webpage\\\": \\\"https://doi.org/10.7303/syn3157238\\\"\\n    ],\\n    [\\n        \\\"dataset_identifier\\\": \\\"MayoLOAD study meta-data and covariates\\\",\\n        \\\"data_repository\\\": \\\"AD Knowledge Portal\\\",\\n        \\\"dataset_webpage\\\": \\\"https://doi.org/10.7303/syn3205821.6\\\"\\n    ]\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"KORA\\\",\\n        \\\"data_repository\\\": \\\"Helmholtz Zentrum M\\u00fcnchen\\\",\\n        \\\"dataset_webpage\\\": \\\"https://helmholtz-muenchen.managed-otrs.com/external/\\\"\\n    ],\\n    [\\n        \\\"dataset_identifier\\\": \\\"AGES-RS\\\",\\n        \\\"data_repository\\\": \\\"hjarta.is\\\",\\n        \\\"dataset_webpage\\\": \\\"https://hjarta.is/en/research/ages-phase-1/\\\"\\n    ],\\n    [\\n        \\\"dataset_identifier\\\": \\\"ADNI metabolomics data from the AbsoluteIDQ-p180 kit (ADNI-1)\\\",\\n        \\\"data_repository\\\": \\\"AD Knowledge Portal\\\",\\n        \\\"dataset_webpage\\\": \\\"https://doi.org/10.7303/syn5592519\\\"\\n    ],\\n    [\\n        \\\"dataset_identifier\\\": \\\"ADNI metabolomics data from the AbsoluteIDQ-p180 kit (ADNI-GO/\\u22122)\\\",\\n        \\\"data_repository\\\": \\\"AD Knowledge Portal\\\",\\n        \\\"dataset_webpage\\\": \\\"https://doi.org/10.7303/syn9705278\\\"\\n    ],\\n    [\\n        \\\"dataset_identifier\\\": \\\"ADNI clinical and demographic data\\\",\\n        \\\"data_repository\\\": \\\"LONI\\\",\\n        \\\"dataset_webpage\\\": \\\"http://adni.loni.usc.edu/data-samples/access-data/\\\"\\n    ],\\n    [\\n        \\\"dataset_identifier\\\": \\\"ROS/MAP imputed genotype data\\\",\\n        \\\"data_repository\\\": \\\"AD Knowledge Portal\\\",\\n        \\\"dataset_webpage\\\": \\\"https://doi.org/10.7303/syn3157329\\\"\\n    ],\\n    [\\n        \\\"dataset_identifier\\\": \\\"ROS/MAP study meta-data, basic covariates and clinical variables\\\",\\n        \\\"data_repository\\\": \\\"AD Knowledge Portal\\\",\\n        \\\"dataset_webpage\\\": \\\"https://doi.org/10.7303/syn3157322\\\"\\n    ],\\n    [\\n        \\\"dataset_identifier\\\": \\\"ROS/MAP clinical and demographic data\\\",\\n        \\\"data_repository\\\": \\\"RADC\\\",\\n        \\\"dataset_webpage\\\": \\\"http://www.radc.rush.edu/\\\"\\n    ],\\n    [\\n        \\\"dataset_identifier\\\": \\\"MayoLOAD genotyping data\\\",\\n        \\\"data_repository\\\": \\\"AD Knowledge Portal\\\",\\n        \\\"dataset_webpage\\\": \\\"https://doi.org/10.7303/syn3157238\\\"\\n    ],\\n    [\\n        \\\"dataset_identifier\\\": \\\"MayoLOAD study meta-data and covariates\\\",\\n        \\\"data_repository\\\": \\\"AD Knowledge Portal\\\",\\n        \\\"dataset_webpage\\\": \\\"https://doi.org/10.7303/syn3205821.6\\\"\\n    ]\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755541094,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 2046,\n",
      "        \"completion_tokens\": 668,\n",
      "        \"total_tokens\": 2714,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'KORA', 'data_repository': 'Helmholtz Zentrum München', 'dataset_webpage': 'https://helmholtz-muenchen.managed-otrs.com/external/', 'dataset_id': 'KORA'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: KORA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID KORA is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: Helmholtz Zentrum München, identifier: KORA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier KORA for Helmholtz Zentrum München\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier KORA for Helmholtz Zentrum München\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier KORA for Helmholtz Zentrum München\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier KORA for Helmholtz Zentrum München\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier KORA for Helmholtz Zentrum München\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: https://helmholtz-muenchen.managed-otrs.com/external/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://helmholtz-muenchen.managed-otrs.com/external/, repo helmholtz-muenchen.managed-otrs.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://helmholtz-muenchen.managed-otrs.com/external/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 972 - INFO - Repository helmholtz-muenchen.managed-otrs.com not found in ontology\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'KORA', 'data_repository': 'helmholtz-muenchen.managed-otrs.com', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'AGES-RS', 'data_repository': 'hjarta.is', 'dataset_webpage': 'https://hjarta.is/en/research/ages-phase-1/', 'dataset_id': 'AGES-RS'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: AGES-RS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID AGES-RS is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: hjarta.is, identifier: AGES-RS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier AGES-RS for hjarta.is\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier AGES-RS for hjarta.is\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier AGES-RS for hjarta.is\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier AGES-RS for hjarta.is\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier AGES-RS for hjarta.is\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: https://hjarta.is/en/research/ages-phase-1/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://hjarta.is/en/research/ages-phase-1/, repo hjarta.is\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://hjarta.is/en/research/ages-phase-1/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 972 - INFO - Repository hjarta.is not found in ontology\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'AGES-RS', 'data_repository': 'hjarta.is', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'ADNI metabolomics data from the AbsoluteIDQ-p180 kit (ADNI-1)', 'data_repository': 'AD Knowledge Portal', 'dataset_webpage': 'https://doi.org/10.7303/syn5592519', 'dataset_id': 'ADNI metabolomics data from the AbsoluteIDQ-p180 kit (ADNI-1)'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: ADNI metabolomics data from the AbsoluteIDQ-p180 kit (ADNI-1)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 940 - INFO - Accession ID ADNI metabolomics data from the AbsoluteIDQ-p180 kit (ADNI-1) is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: AD Knowledge Portal, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier n/a for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier n/a for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier n/a for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier n/a for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier n/a for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: https://doi.org/10.7303/syn5592519\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://doi.org/10.7303/syn5592519, repo doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.synapse.org/#!Synapse:syn5592519\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://doi.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Link does not match the pattern https://doi.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 626 - INFO - Dataset ID is 'n/a' and repository name from prompt\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'ADNI metabolomics data from the AbsoluteIDQ-p180 kit (ADNI-GO/−2)', 'data_repository': 'AD Knowledge Portal', 'dataset_webpage': 'https://doi.org/10.7303/syn9705278', 'dataset_id': 'ADNI metabolomics data from the AbsoluteIDQ-p180 kit (ADNI-GO/−2)'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: ADNI metabolomics data from the AbsoluteIDQ-p180 kit (ADNI-GO/−2)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 940 - INFO - Accession ID ADNI metabolomics data from the AbsoluteIDQ-p180 kit (ADNI-GO/−2) is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: AD Knowledge Portal, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier n/a for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier n/a for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier n/a for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier n/a for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier n/a for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: https://doi.org/10.7303/syn9705278\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://doi.org/10.7303/syn9705278, repo doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.synapse.org/#!Synapse:syn9705278\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://doi.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Link does not match the pattern https://doi.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 626 - INFO - Dataset ID is 'n/a' and repository name from prompt\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'ADNI clinical and demographic data', 'data_repository': 'LONI', 'dataset_webpage': 'http://adni.loni.usc.edu/data-samples/access-data/', 'dataset_id': 'ADNI clinical and demographic data'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: ADNI clinical and demographic data\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 940 - INFO - Accession ID ADNI clinical and demographic data is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: LONI, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier n/a for LONI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier n/a for LONI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier n/a for LONI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier n/a for LONI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier n/a for LONI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: http://adni.loni.usc.edu/data-samples/access-data/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: http://adni.loni.usc.edu/data-samples/access-data/, repo adni.loni.usc.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://adni.loni.usc.edu/data-samples/access-data/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 972 - INFO - Repository adni.loni.usc.edu not found in ontology\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'adni.loni.usc.edu', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'ROS/MAP imputed genotype data', 'data_repository': 'AD Knowledge Portal', 'dataset_webpage': 'https://doi.org/10.7303/syn3157329', 'dataset_id': 'ROS/MAP imputed genotype data'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: ROS/MAP imputed genotype data\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 940 - INFO - Accession ID ROS/MAP imputed genotype data is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: AD Knowledge Portal, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier n/a for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier n/a for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier n/a for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier n/a for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier n/a for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: https://doi.org/10.7303/syn3157329\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://doi.org/10.7303/syn3157329, repo doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.synapse.org/#!Synapse:syn3157329\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://doi.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Link does not match the pattern https://doi.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 626 - INFO - Dataset ID is 'n/a' and repository name from prompt\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'ROS/MAP study meta-data, basic covariates and clinical variables', 'data_repository': 'AD Knowledge Portal', 'dataset_webpage': 'https://doi.org/10.7303/syn3157322', 'dataset_id': 'ROS/MAP study meta-data, basic covariates and clinical variables'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: ROS/MAP study meta-data, basic covariates and clinical variables\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 940 - INFO - Accession ID ROS/MAP study meta-data, basic covariates and clinical variables is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: AD Knowledge Portal, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier n/a for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier n/a for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier n/a for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier n/a for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier n/a for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: https://doi.org/10.7303/syn3157322\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://doi.org/10.7303/syn3157322, repo doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.synapse.org/#!Synapse:syn3157322\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://doi.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Link does not match the pattern https://doi.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 626 - INFO - Dataset ID is 'n/a' and repository name from prompt\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'ROS/MAP clinical and demographic data', 'data_repository': 'RADC', 'dataset_webpage': 'http://www.radc.rush.edu/', 'dataset_id': 'ROS/MAP clinical and demographic data'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: ROS/MAP clinical and demographic data\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 940 - INFO - Accession ID ROS/MAP clinical and demographic data is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: RADC, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier n/a for RADC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier n/a for RADC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier n/a for RADC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier n/a for RADC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier n/a for RADC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: http://www.radc.rush.edu/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: http://www.radc.rush.edu/, repo www.radc.rush.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 981 - WARNING - Error resolving URL http://www.radc.rush.edu/: HTTPSConnectionPool(host='www.radc.rush.edu', port=8443): Max retries exceeded with url: / (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x1b0c66a10>, 'Connection to www.radc.rush.edu timed out. (connect timeout=5)'))\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 972 - INFO - Repository www.radc.rush.edu not found in ontology\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'www.radc.rush.edu', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'MayoLOAD genotyping data', 'data_repository': 'AD Knowledge Portal', 'dataset_webpage': 'https://doi.org/10.7303/syn3157238', 'dataset_id': 'MayoLOAD genotyping data'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: MayoLOAD genotyping data\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 940 - INFO - Accession ID MayoLOAD genotyping data is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: AD Knowledge Portal, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier n/a for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier n/a for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier n/a for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier n/a for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier n/a for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: https://doi.org/10.7303/syn3157238\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://doi.org/10.7303/syn3157238, repo doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.synapse.org/#!Synapse:syn3157238\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://doi.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Link does not match the pattern https://doi.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 626 - INFO - Dataset ID is 'n/a' and repository name from prompt\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'MayoLOAD study meta-data and covariates', 'data_repository': 'AD Knowledge Portal', 'dataset_webpage': 'https://doi.org/10.7303/syn3205821.6', 'dataset_id': 'MayoLOAD study meta-data and covariates'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: MayoLOAD study meta-data and covariates\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 940 - INFO - Accession ID MayoLOAD study meta-data and covariates is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: AD Knowledge Portal, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier n/a for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier n/a for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier n/a for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier n/a for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier n/a for AD Knowledge Portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: https://doi.org/10.7303/syn3205821.6\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://doi.org/10.7303/syn3205821.6, repo doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.synapse.org/#!Synapse:syn3205821/version/6\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://doi.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Link does not match the pattern https://doi.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 626 - INFO - Dataset ID is 'n/a' and repository name from prompt\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 377 - INFO - datasets (<class 'list'>): [{'dataset_identifier': 'syn5592519', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn9705278', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn3157329', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn3157322', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn3157238', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'syn3205821', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'KORA', 'data_repository': 'helmholtz-muenchen.managed-otrs.com', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'AGES-RS', 'data_repository': 'hjarta.is', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'adni.loni.usc.edu', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'www.radc.rush.edu', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'syn5592519', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'syn9705278', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'syn3157329', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'syn3157322', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'syn3157238', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'syn3205821', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'KORA', 'data_repository': 'helmholtz-muenchen.managed-otrs.com', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'AGES-RS', 'data_repository': 'hjarta.is', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'n/a', 'data_repository': 'adni.loni.usc.edu', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'n/a', 'data_repository': 'www.radc.rush.edu', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 385 - INFO - Final ret data availability: 10 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 386 - INFO - Final ret data availability: [{'dataset_identifier': 'syn5592519', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'syn9705278', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'syn3157329', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'syn3157322', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'syn3157238', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'syn3205821', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'KORA', 'data_repository': 'helmholtz-muenchen.managed-otrs.com', 'dataset_webpage': 'n/a', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'AGES-RS', 'data_repository': 'hjarta.is', 'dataset_webpage': 'n/a', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'n/a', 'data_repository': 'adni.loni.usc.edu', 'dataset_webpage': 'n/a', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'n/a', 'data_repository': 'www.radc.rush.edu', 'dataset_webpage': 'n/a', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1079 - INFO - Fetching metadata for 10 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: syn5592519\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn5592519\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn5592519 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn5592519 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn5592519 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn5592519 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for syn5592519 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn5592519\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.synapse.org/Synapse:syn5592519\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: syn9705278\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn9705278\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn9705278 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn9705278 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn9705278 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn9705278 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for syn9705278 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn9705278\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.synapse.org/Synapse:syn9705278\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: syn3157329\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn3157329\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3157329 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn3157329 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn3157329 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3157329 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for syn3157329 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 3 with repo: synapse.org and accession_id: syn3157329\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.synapse.org/Synapse:syn3157329\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 3: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: syn3157322\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn3157322\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3157322 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn3157322 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn3157322 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3157322 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for syn3157322 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 4 with repo: synapse.org and accession_id: syn3157322\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.synapse.org/Synapse:syn3157322\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 4: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: syn3157238\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn3157238\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3157238 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn3157238 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn3157238 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3157238 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for syn3157238 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 5 with repo: synapse.org and accession_id: syn3157238\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.synapse.org/Synapse:syn3157238\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 5: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: syn3205821\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn3205821\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3205821 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn3205821 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn3205821 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3205821 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for syn3205821 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 6 with repo: synapse.org and accession_id: syn3205821\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.synapse.org/Synapse:syn3205821\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 6: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: KORA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: helmholtz-muenchen.managed-otrs.com, identifier: KORA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier KORA for helmholtz-muenchen.managed-otrs.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier KORA for helmholtz-muenchen.managed-otrs.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier KORA for helmholtz-muenchen.managed-otrs.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier KORA for helmholtz-muenchen.managed-otrs.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier KORA for helmholtz-muenchen.managed-otrs.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: helmholtz-muenchen.managed-otrs.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 900 - WARNING - URL helmholtz-muenchen.managed-otrs.com may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for KORA in helmholtz-muenchen.managed-otrs.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 7 with repo: helmholtz-muenchen.managed-otrs.com and accession_id: KORA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1152 - WARNING - Repository helmholtz-muenchen.managed-otrs.com unknown in Ontology. Skipping dataset page 7.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: AGES-RS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: hjarta.is, identifier: AGES-RS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier AGES-RS for hjarta.is\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier AGES-RS for hjarta.is\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier AGES-RS for hjarta.is\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier AGES-RS for hjarta.is\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier AGES-RS for hjarta.is\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: hjarta.is\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 900 - WARNING - URL hjarta.is may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for AGES-RS in hjarta.is\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 8 with repo: hjarta.is and accession_id: AGES-RS\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1152 - WARNING - Repository hjarta.is unknown in Ontology. Skipping dataset page 8.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1095 - INFO - Skipping dataset 9: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 10 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1095 - INFO - Skipping dataset 10: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1156 - INFO - Updated datasets len: 10\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 221 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 12, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'source_section', 'retrieval_pattern', 'access_mode', 'link',\n",
      "       'source_url', 'download_link', 'title', 'content_type', 'id', 'caption',\n",
      "       'description', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 456 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 462 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 539 - INFO - 16th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC6646046)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 347 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC6646046\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 354 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 533 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC6646046&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 461 - WARNING - No links found in the section.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 395 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC6646046.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 399 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC6646046 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 413 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 417 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC6646046, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 4 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b6cedb40>, <Element supplementary-material at 0x1b6cc0e40>, <Element supplementary-material at 0x1b6cc1680>, <Element supplementary-material at 0x1b6cc23c0>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b6cedb40>, <Element supplementary-material at 0x1b6cc0e40>, <Element supplementary-material at 0x1b6cc1680>, <Element supplementary-material at 0x1b6cc23c0>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD1: We found that NRGs are indeed connected to more DREs in Hi-C and FANTOM5 data ( Fig. 2b [rid=F2] , Supplementary Fig. 1 [this file] , Methods [rid=S15] , and Supplementary Note [rid=SD3] ). \n",
      "As expected, no difference was observed for WBGs between prenatal and postnatal stages ( = 0.27, Supplementary Fig. 2 [this file] ). \n",
      "Interestingly, in GTEx data, we observed that nearest non-HRG genes are highly expressed in a majority of brain tissues while non-nearest HRGs are not ( Supplementary Fig. 3 [this file] ). \n",
      "Specifically, and both encoding voltage-gated calcium channel subunits, are involved in CCS and contribute to risk for SCZ . is differentially expressed in SCZ patients ( = 0.03), and both and capture multiple Hi-C links in brain Hi-C data ( Supplementary Fig. 4 [this file] ), contributing to the high PP of both genes in iRIGS. We also predicted two DNA binding proteins, SOX2 and SATB2 ( Supplementary Fig. 4 [this file] ), which have important roles in neurogenesis and have been widely implicated in SCZ. Several of our predicted HRGs are miR-137 target genes, including the aforementioned , and three other genes, and . , a glutamate-gated ion channel protein and a key mediator of synaptic plasticity , has a pdDNM and multiple regulatory connections ( Supplementary Fig. 4 [this file] ). , encoding Transcription Factor 4, participates in the initiation of neuronal differentiation by regulating the intrinsic excitability of prefrontal cortical neurons , and knockdown of alters the expression of genes important for developing prefrontal neocortex . is also linked with numerous DREs ( Supplementary Fig. 4 [this file] ) in iRIGS. \n",
      "We took a closer look at this region to gain more insights. captures 55 Hi-C links in brain Hi-C data ( Supplementary Fig. 5 [this file] ), while there is only one for . In GTEx data, is highly expressed in basal ganglia caudate, hypothalamus, basal ganglia nucleus accumbens, basal ganglia putamen and substantia nigra, but the expression in cortex and frontal cortex is rather low ( Supplementary Fig. 6 [this file] ). is uniformly and highly expressed in all brain tissues ( Supplementary Fig. 6 [this file] ). In BrainSpan data, shows constitutively high expression across all stages, and particularly higher expression at prenatal stages with a trajectory that peaks at the early-mid fetal stage, while shows lower expression across all developmental stages and no obvious pattern of transition between prenatal and postnatal stages ( Supplementary Fig. 7 [this file] ). \n",
      "One pdDNM and a high number of regulatory links were observed for ( Supplementary Fig. 8 [this file] ), promoting it as the top gene predicted by iRIGS. \n",
      "We curated a list of 2263 confirmed druggable targets from multiple sources ( Methods [rid=S15] ), and found that 28 (27%) HRGs are targets of 198 FDA-approved, clinically investigational, or preclinical drugs ( Supplementary Fig. 9 [this file] and Supplementary Table 7 [rid=SD2] ). In particular, we found that 5 HRGs ( and ) are targets of nervous system drugs ( Supplementary Fig. 9 [this file] ), corresponding to a statistically significant enrichment ( = 0.01, OR = 3.78). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD2: Specifically, we collected 278 gene sets relevant to CNS and behavior/neurological phenotypes from Mouse Genome Informatics (MGI) Mammalian Phenotype Ontology (MPO) database ( Methods [rid=S15] ) and observed significant enrichment of HRGs in 33 gene sets after Bonferroni correction ( Supplementary Table 4 [this file] ). \n",
      "In addition, we also observed that the 65 non-nearest HRGs are significantly enriched in 19 MPO gene sets ( Supplementary Table 5 [this file] ), while no significantly enriched gene sets were observed for the 65 nearest non-HRG genes ( Supplementary Table 6 [this file] ), providing strong and orthogonal support for iRIGS identified risk genes that are beyond the proximity to the GWAS index SNPs. \n",
      "We curated a list of 2263 confirmed druggable targets from multiple sources ( Methods [rid=S15] ), and found that 28 (27%) HRGs are targets of 198 FDA-approved, clinically investigational, or preclinical drugs ( Supplementary Fig. 9 [rid=SD1] and Supplementary Table 7 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD3: Details of iRIGS can be found in Methods [rid=S15] and Supplementary Note [this file] . \n",
      "We performed iRIGS on these loci to identify risk genes of SCZ ( Supplementary Table 1 [this file] ). \n",
      "We also tried other thresholds to define LBGs, and the enrichment patterns are similar ( Supplementary Table 2 [this file] ). \n",
      "We found that NRGs are indeed connected to more DREs in Hi-C and FANTOM5 data ( Fig. 2b [rid=F2] , Supplementary Fig. 1 [rid=SD1] , Methods [rid=S15] , and Supplementary Note [this file] ). \n",
      "We therefore integrated them into iRIGS ( Methods [rid=S15] ) and predicted 104 high-confident risk genes (HRGs) in total ( Supplementary Table 1 [this file] ). \n",
      "We found that non-nearest HRGs are more significantly enriched in the gene sets compared to nearest non-HRG genes ( Supplementary Table 3 [this file] ), suggesting that the 65 non-nearest HRGs identified by iRIGS are more likely to be true risk genes than their nearest counterparts. \n",
      "We highlighted these genes with extended supporting evidence and description in Supplementary Table 1 [this file] . \n",
      "The distance of or to in the network is calculated using the random walk with restart (RWR) algorithm ( Supplemental Note [this file] ). We employed the Mahalanobis transformation to de-correlate the integrated multi-dimensional data so that any supportive genomic features can be properly incorporated ( Supplementary Note [this file] ). \n",
      "We adopted the Jensen-Shannon divergence to measure the tissue-specificity of each gene in each tissue ( Supplementary Note [this file] ). \n",
      "All the data used in this study are from public resources that were specified in Methods [rid=S15] and Supplementary Note [this file] . \n",
      "See main text and Supplementary Note [this file] for details. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD4: \u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 123 - INFO - Loading patterns for section 'data_availability_sections' for publisher 'PMC'.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 131 - INFO - Section pattern keys for publisher PMC: dict_keys(['data_availability_sections', 'supplementary_material_sections', 'supplementary_data_sections', 'key_resources_table', 'methods_sections'])\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 133 - INFO - Section name: data_availability_sections\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 135 - INFO - Found section 'data_availability_sections' in patterns for publisher 'PMC'.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 137 - INFO - Loaded patterns for section 'data_availability_sections': [\".//sec[@sec-type='data-availability']\", \".//notes[@notes-type='data-availability']\", \".//notes[@notes-type='data-availability']\"]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_data_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 706 - INFO - Found 5 supplementary data sections\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(key_resources_table)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "Batches:   0%|          | 0/1 [00:00<?, ?it/s]"
      ],
      "application/vnd.jupyter.widget-view+json": {
       "version_major": 2,
       "version_minor": 0,
       "model_id": "fee6200f14b447698657109c6a4b359d"
      }
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001B[97membeddings_retriever.py - line 51 - INFO - Searching for top-2 passages similar to the query by embeddings.\u001B[0m\n",
      "\u001B[97membeddings_retriever.py - line 33 - INFO - Computing L2 distances using numpy.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 364 - INFO - Processing DAS_content: 3 elements\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 170\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 385 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-0e8fd5d386717ea80863f6306701e10896de06c17e04c3231b31cfc7eb1f304a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 385 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-6f362e9f-8d79-4981-b68d-45ce6f525d86\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755541110,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 476,\n",
      "        \"completion_tokens\": 6,\n",
      "        \"total_tokens\": 482,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 350\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 430 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-f70425fec75a53c6667f308b13b4ced0b47236091832eabcc515cea027c643ea\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 430 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-79714745-d71a-4236-889f-9b64c148870e\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"genomics datasets\\\",\\n        \\\"data_repository\\\": \\\"Vanderbilt University Medical Center, Center for Genetic Epidemiology (CGG)\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.vumc.org/cgg\\\"\\n    ]\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"genomics datasets\\\",\\n        \\\"data_repository\\\": \\\"Vanderbilt University Medical Center, Center for Genetic Epidemiology (CGG)\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.vumc.org/cgg\\\"\\n    ]\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755541111,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 581,\n",
      "        \"completion_tokens\": 69,\n",
      "        \"total_tokens\": 650,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'genomics datasets', 'data_repository': 'Vanderbilt University Medical Center, Center for Genetic Epidemiology (CGG)', 'dataset_webpage': 'https://www.vumc.org/cgg', 'dataset_id': 'genomics datasets'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: genomics datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 940 - INFO - Accession ID genomics datasets is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: Vanderbilt University Medical Center, Center for Genetic Epidemiology (CGG), identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier n/a for Vanderbilt University Medical Center, Center for Genetic Epidemiology  CGG \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier n/a for Vanderbilt University Medical Center, Center for Genetic Epidemiology  CGG \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier n/a for Vanderbilt University Medical Center, Center for Genetic Epidemiology  CGG \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier n/a for Vanderbilt University Medical Center, Center for Genetic Epidemiology  CGG \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier n/a for Vanderbilt University Medical Center, Center for Genetic Epidemiology  CGG \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: https://www.vumc.org/cgg\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.vumc.org/cgg, repo www.vumc.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.vumc.org/cgg\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 972 - INFO - Repository www.vumc.org not found in ontology\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'www.vumc.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 1933\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 826 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-ada02810704301e3f10bcac6ed4dc1a28a9f17d0de09ce5d50a19fb48c73ace3\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 826 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-fd48eac0-4be6-4495-b674-4e13e2b751d5\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"4.3\\\",\\n        \\\"data_repository\\\": \\\"DrugBank\\\",\\n        \\\"dataset_webpage\\\": \\\"https://go.drugbank.com/\\\"\\n    ],\\n    [\\n        \\\"dataset_identifier\\\": \\\"version 21\\\",\\n        \\\"data_repository\\\": \\\"ChEMBL\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/chembl/\\\"\\n    ]\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"4.3\\\",\\n        \\\"data_repository\\\": \\\"DrugBank\\\",\\n        \\\"dataset_webpage\\\": \\\"https://go.drugbank.com/\\\"\\n    ],\\n    [\\n        \\\"dataset_identifier\\\": \\\"version 21\\\",\\n        \\\"data_repository\\\": \\\"ChEMBL\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/chembl/\\\"\\n    ]\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755541113,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 1015,\n",
      "        \"completion_tokens\": 109,\n",
      "        \"total_tokens\": 1124,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': '4.3', 'data_repository': 'DrugBank', 'dataset_webpage': 'https://go.drugbank.com/', 'dataset_id': '4.3'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: 4.3\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID 4.3 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: DrugBank, identifier: 4.3\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 4.3 for DrugBank\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 4.3 for DrugBank\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier 4.3 for DrugBank\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier 4.3 for DrugBank\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier 4.3 for DrugBank\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: https://go.drugbank.com/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://go.drugbank.com/, repo go.drugbank.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://go.drugbank.com/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 972 - INFO - Repository go.drugbank.com not found in ontology\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': '4.3', 'data_repository': 'go.drugbank.com', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'version 21', 'data_repository': 'ChEMBL', 'dataset_webpage': 'https://www.ebi.ac.uk/chembl/', 'dataset_id': 'version 21'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: version 21\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 940 - INFO - Accession ID version 21 is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: ChEMBL, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier n/a for ChEMBL\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier n/a for ChEMBL\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier n/a for ChEMBL\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier n/a for ChEMBL\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier n/a for ChEMBL\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: https://www.ebi.ac.uk/chembl/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ebi.ac.uk/chembl/, repo www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ebi.ac.uk/chembl/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 967 - INFO - No dataset_webpage_url_ptr found for www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 626 - INFO - Dataset ID is 'n/a' and repository name from prompt\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 377 - INFO - datasets (<class 'list'>): [{'dataset_identifier': 'n/a', 'data_repository': 'www.vumc.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': '4.3', 'data_repository': 'go.drugbank.com', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'n/a', 'data_repository': 'www.vumc.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': '4.3', 'data_repository': 'go.drugbank.com', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 385 - INFO - Final ret data availability: 2 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 386 - INFO - Final ret data availability: [{'dataset_identifier': 'n/a', 'data_repository': 'www.vumc.org', 'dataset_webpage': 'n/a', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': '4.3', 'data_repository': 'go.drugbank.com', 'dataset_webpage': 'n/a', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1079 - INFO - Fetching metadata for 2 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1095 - INFO - Skipping dataset 1: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: 4.3\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: go.drugbank.com, identifier: 4.3\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 4.3 for go.drugbank.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier 4.3 for go.drugbank.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier 4.3 for go.drugbank.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier 4.3 for go.drugbank.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier 4.3 for go.drugbank.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: go.drugbank.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 900 - WARNING - URL go.drugbank.com may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for 4.3 in go.drugbank.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 2 with repo: go.drugbank.com and accession_id: 4.3\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1152 - WARNING - Repository go.drugbank.com unknown in Ontology. Skipping dataset page 2.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1156 - INFO - Updated datasets len: 2\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 221 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 6, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'source_section', 'retrieval_pattern', 'link', 'source_url',\n",
      "       'download_link', 'title', 'content_type', 'id', 'caption',\n",
      "       'description', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 456 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 462 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 539 - INFO - 17th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10387117)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 347 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10387117\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 354 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 533 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC10387117&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 395 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC10387117.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 399 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC10387117 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 413 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 417 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC10387117, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 9 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b0943080>, <Element supplementary-material at 0x1b0c09940>, <Element supplementary-material at 0x1b0c08f80>, <Element supplementary-material at 0x1b0c09c40>, <Element supplementary-material at 0x1b0c0bd00>, <Element supplementary-material at 0x1b0c0b200>, <Element supplementary-material at 0x1b0c08a00>, <Element supplementary-material at 0x1b0c09480>, <Element supplementary-material at 0x1b0c0b440>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b0943080>, <Element supplementary-material at 0x1b0c09940>, <Element supplementary-material at 0x1b0c08f80>, <Element supplementary-material at 0x1b0c09c40>, <Element supplementary-material at 0x1b0c0bd00>, <Element supplementary-material at 0x1b0c0b200>, <Element supplementary-material at 0x1b0c08a00>, <Element supplementary-material at 0x1b0c09480>, <Element supplementary-material at 0x1b0c0b440>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM1: MNN outperforms RPCA and Harmony on both the UMAP inspection and cluster stability metrics measurement, we therefore set the mutual nearest neighbor (MNN) matching method in Monocle 3 as default, and Seurat’s reciprocal principal component analysis (RPCA) based integration and Harmony as alternatives (Supplementary Fig. 1 [this file] ). \n",
      "Cells with consistent cell type annotations were combined with the CellRef Seed to form the LungMAP Human Lung CellRef (347,970 cells) (Fig. 2B [rid=Fig2] , Supplementary Figs. 1 [this file] – 3 [this file] , “Methods”). \n",
      "Using the same approach, we constructed a cell type dictionary based on the LungMAP CellCards to define cell types in mouse lung during perinatal development, identified seed cells for each cell type (termed LungMAP Mouse Lung Development CellRef Seed”), and constructed a CellRef for mouse lung development (denoted as LungMAP Mouse Lung Development CellRef) using Drop-seq of mouse lungs ( = 95,658, 8 time points, 17 samples) from embryonic day 16.5 to postnatal day 28 (Fig. 3 [rid=Fig3] , Supplementary Fig. 4 [this file] , Supplementary Data 3 [rid=MOESM5] , 4 [rid=MOESM6] ). \n",
      "These tools provide highly interactive search, analyzing, and visualization functionalities for users to explore and reanalyze cell type and gene expression patterns provided by the LungMAP CellRefs (Fig. 4 [rid=Fig4] , Supplementary Fig. 6 [this file] ). Users can also select and highlight cells of interest (Supplementary Fig. 6 [this file] ). \n",
      "Evaluation of the other three test data samples were shown in Supplementary Fig. 7 [this file] . \n",
      "As shown in Fig. 5D–I [rid=Fig5] and Supplementary Fig. 7 [this file] , predicted cell types were well separated and formed clusters. Cell-type-specific marker genes from CellCards were selectively expressed in each predicted cell type, supporting the concordance of the cell identities (Fig. 5E, F [rid=Fig5] , Supplementary Fig. 7 [this file] ). The median values of all metrics are greater than 0.92 (Supplementary Fig. 8 [this file] ), supporting the high consistency of the automated CellRef cell type prediction with the original cell annotations. Both the LungMAP human and mouse lung CellRefs achieved high cross validation performance with ~0.92 median F1 and MCC scores for the human lung CellRef and 0.98 median F1 and MCC scores for the mouse lung CellRef, respectively (Supplementary Fig. 9 [this file] ). \n",
      "We summarized the one-to-one mapped cell types and the unique cell types/states in the Supplementary Fig. 10 [this file] . \n",
      "Similarly, when applying these stability metrics to CellRef and HLCA using their own data collections, CellRef produced slightly higher scores on SCCAF and TF-IDF; as well as a significantly higher score on cell re-assignment ( = 0.011) as compared to HLCA (Supplementary Fig. 11 [this file] ). scTriangulate leverages the computed stability scores to further assess the marginal importance of each annotation (CellRef and HLCA) using a cooperative game theory framework. scTriangulate predicted that 50% ( = 24/48) of the CellRef cell types to be of higher confidence (AEC, AF1, AF2, AM, ASMC, CD8 T, cDC1, cDC2, Ciliated, Goblet, Ionocyte, LEC, Platelet, pMON, Suprabasal, SVEC, VSMC), compared to 46% ( = 27/58) of the HCLA cell types (AT2, aCAP, Pericyte, Mesothelium, Peribronchial fibroblasts and Adventitial fibroblasts, etc.) (Fig. 8D [rid=Fig8] ). \n",
      "The overall distributions of iLISI scores in both CellRef and HLCA are not ideal, on average about 15% for CellRef, and 5% for HLCA, respectively (Supplementary Fig. 11 [this file] ), likely due to that both atlases were constructed using large collections of heterogeneous data from different biological regions and conditions rather than a well-designed balanced cohort. Nevertheless, CellRef consistently outperforms HLCA on the measurements of iLISI, and has the cLISI scores close to 1, indicating a well separation of the CellRef cell types (Supplementary Fig. 11 [this file] ). \n",
      "Results supported the high accuracy, cluster stability, and marker specificity of our mouse lung reference cell types (Supplementary Fig. 12 [this file] ; mean AUC 95.8, mean SCCAF score: 0.970, mean Reassign score: 0.881, mean TFID10 score: 0.892). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM2: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM3: This collection contains data from similar numbers of female and male donors ( = 48 and 55, respectively; 1 unannotated) (Fig. 1A [rid=Fig1] ; Supplementary Data 1 [this file] ). In total, 505,256 lung cells from 148 sc/snRNA-seq of normal human lung samples from 104 donors were used for LungMAP human lung CellRef construction (Supplementary Data 1 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM4: Using this guided approach and a cell type dictionary derived from LungMAP CellCards (Supplementary Data 2 [this file] ), we identified 8,080 seed cells representing 48 normal human lung cell types, termed LungMAP Human Lung CellRef Seed (Fig. 2A [rid=Fig2] ). \n",
      "Evaluation of cell type annotations using CellRef cell type markers from Supplementary Data 2 [this file] . Percentages of cell type markers (Supplementary Data 2 [this file] ) that are differentially expressed in their corresponding cell type predictions ( = 34 cell types) in ( ). \n",
      "The human lung CellRef cell type selective markers were generated by including dictionary marker genes (Supplementary Data 2 [this file] ) and top selectively expressed markers for each cell type. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM5: Using the same approach, we constructed a cell type dictionary based on the LungMAP CellCards to define cell types in mouse lung during perinatal development, identified seed cells for each cell type (termed LungMAP Mouse Lung Development CellRef Seed”), and constructed a CellRef for mouse lung development (denoted as LungMAP Mouse Lung Development CellRef) using Drop-seq of mouse lungs ( = 95,658, 8 time points, 17 samples) from embryonic day 16.5 to postnatal day 28 (Fig. 3 [rid=Fig3] , Supplementary Fig. 4 [rid=MOESM1] , Supplementary Data 3 [this file] , 4 [rid=MOESM6] ). \n",
      "In total, gene expression in 17 Drop-seq samples from eight time points (Supplementary Data 3 [this file] ) of mouse lung development were generated. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM6: Using the same approach, we constructed a cell type dictionary based on the LungMAP CellCards to define cell types in mouse lung during perinatal development, identified seed cells for each cell type (termed LungMAP Mouse Lung Development CellRef Seed”), and constructed a CellRef for mouse lung development (denoted as LungMAP Mouse Lung Development CellRef) using Drop-seq of mouse lungs ( = 95,658, 8 time points, 17 samples) from embryonic day 16.5 to postnatal day 28 (Fig. 3 [rid=Fig3] , Supplementary Fig. 4 [rid=MOESM1] , Supplementary Data 3 [rid=MOESM5] , 4 [this file] ). In total, 40 mouse lung cell types have been identified with the guidance of the mouse lung cell type dictionary (Fig. 3 [rid=Fig3] , Supplementary Data 4 [this file] ). \n",
      "We constructed a cell type dictionary for mouse lung (Supplementary Data 4 [this file] ) based on our constructed dictionary derived from the LungMAP CellCards. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM7: AUCs values for each of the mapped cell types ( = 42) in the CellRef (orange) and HLCA (blue) calculated using the cell type selective marker genes identified by CellRef (Supplementary Data 5 [this file] ). \n",
      "Next, we performed the same analysis using the cell type marker genes identified by CellRef (Supplementary Data 5 [this file] ). \n",
      "Top-ranked cell type specific DEGs (ranked by fold change in average expression) were combined with the known markers genes to form the CellRef cell type selective marker gene list (Supplementary Data 5 [this file] , up to 10 genes for each of the 48 CellRef human lung cell types). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM8: Last, we evaluated the LungMAP mouse lung CellRef using the three stability metrics calculated by scTriangulate and the marker-based AUC analysis (“Methods” and Supplementary Data 6 [this file] ). \n",
      "Using the same approach, the mouse lung CellRef cell type selective marker genes were selected (Supplementary Data 6 [this file] , up to 10 marker genes per cell type) and used for the AUC based assessment of mouse lung CellRef. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM9: Further information on research design is available in the Nature Portfolio Reporting Summary [this file] linked to this article. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 123 - INFO - Loading patterns for section 'data_availability_sections' for publisher 'PMC'.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 131 - INFO - Section pattern keys for publisher PMC: dict_keys(['data_availability_sections', 'supplementary_material_sections', 'supplementary_data_sections', 'key_resources_table', 'methods_sections'])\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 133 - INFO - Section name: data_availability_sections\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 135 - INFO - Found section 'data_availability_sections' in patterns for publisher 'PMC'.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 137 - INFO - Loaded patterns for section 'data_availability_sections': [\".//sec[@sec-type='data-availability']\", \".//notes[@notes-type='data-availability']\", \".//notes[@notes-type='data-availability']\"]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_data_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 706 - INFO - Found 10 supplementary data sections\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(key_resources_table)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "Batches:   0%|          | 0/2 [00:00<?, ?it/s]"
      ],
      "application/vnd.jupyter.widget-view+json": {
       "version_major": 2,
       "version_minor": 0,
       "model_id": "528abb0fb1c448098c031995f1e489ac"
      }
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001B[97membeddings_retriever.py - line 51 - INFO - Searching for top-2 passages similar to the query by embeddings.\u001B[0m\n",
      "\u001B[97membeddings_retriever.py - line 33 - INFO - Computing L2 distances using numpy.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 364 - INFO - Processing DAS_content: 4 elements\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 3030\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 1100 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-32b957ffe90a808d2e41d9f2ea03dd4938afd22c5b61b15af098b1f37bb0eb1b\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 1100 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-002c267c-2dae-433b-9c5c-1e888803e82f\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE135893\\\",\\n        \\\"data_repository\\\": \\\"Gene Expression Omnibus\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE135893\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE136831\\\",\\n        \\\"data_repository\\\": \\\"Gene Expression Omnibus\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE136831\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE122960\\\",\\n        \\\"data_repository\\\": \\\"Gene Expression Omnibus\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122960\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE134174\\\",\\n        \\\"data_repository\\\": \\\"Gene Expression Omnibus\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE134174\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE161382\\\",\\n        \\\"data_repository\\\": \\\"Gene Expression Omnibus\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE161382\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE171524\\\",\\n        \\\"data_repository\\\": \\\"Gene Expression Omnibus\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE171524\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"EGAS00001004082\\\",\\n        \\\"data_repository\\\": \\\"European Genome-phenome Archive\\\",\\n        \\\"dataset_webpage\\\": \\\"https://ega-archive.org/studies/EGAS00001004082\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"syn21041850\\\",\\n        \\\"data_repository\\\": \\\"Synapse.org\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.synapse.org/#!Synapse:syn21041850\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"LMEX0000004396\\\",\\n        \\\"data_repository\\\": \\\"LungMAP.net\\\",\\n        \\\"dataset_webpage\\\": \\\"https://lungmap.net/breath-omics-experiment-page/?experiment_id=LMEX0000004396\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE122332\\\",\\n        \\\"data_repository\\\": \\\"Gene Expression Omnibus\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122332\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"LMEX0000004397\\\",\\n        \\\"data_repository\\\": \\\"LungMAP.net\\\",\\n        \\\"dataset_webpage\\\": \\\"https://lungmap.net/breath-omics-experiment-page/?experiment_id=LMEX0000004397\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE178362\\\",\\n        \\\"data_repository\\\": \\\"Gene Expression Omnibus\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178362\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE135851\\\",\\n        \\\"data_repository\\\": \\\"Gene Expression Omnibus\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE135851\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE149563\\\",\\n        \\\"data_repository\\\": \\\"Gene Expression Omnibus\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE149563\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"dataset-427f1eee6dd44f50bae1ab13f0f3c6a9\\\",\\n        \\\"data_repository\\\": \\\"FASTGenomics\\\",\\n        \\\"dataset_webpage\\\": \\\"https://beta.fastgenomics.org/datasets/detail-dataset-427f1eee6dd44f50bae1ab13f0f3c6a9\\\"\\n    }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE135893\\\",\\n        \\\"data_repository\\\": \\\"Gene Expression Omnibus\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE135893\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE136831\\\",\\n        \\\"data_repository\\\": \\\"Gene Expression Omnibus\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE136831\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE122960\\\",\\n        \\\"data_repository\\\": \\\"Gene Expression Omnibus\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122960\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE134174\\\",\\n        \\\"data_repository\\\": \\\"Gene Expression Omnibus\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE134174\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE161382\\\",\\n        \\\"data_repository\\\": \\\"Gene Expression Omnibus\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE161382\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE171524\\\",\\n        \\\"data_repository\\\": \\\"Gene Expression Omnibus\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE171524\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"EGAS00001004082\\\",\\n        \\\"data_repository\\\": \\\"European Genome-phenome Archive\\\",\\n        \\\"dataset_webpage\\\": \\\"https://ega-archive.org/studies/EGAS00001004082\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"syn21041850\\\",\\n        \\\"data_repository\\\": \\\"Synapse.org\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.synapse.org/#!Synapse:syn21041850\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"LMEX0000004396\\\",\\n        \\\"data_repository\\\": \\\"LungMAP.net\\\",\\n        \\\"dataset_webpage\\\": \\\"https://lungmap.net/breath-omics-experiment-page/?experiment_id=LMEX0000004396\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE122332\\\",\\n        \\\"data_repository\\\": \\\"Gene Expression Omnibus\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122332\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"LMEX0000004397\\\",\\n        \\\"data_repository\\\": \\\"LungMAP.net\\\",\\n        \\\"dataset_webpage\\\": \\\"https://lungmap.net/breath-omics-experiment-page/?experiment_id=LMEX0000004397\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE178362\\\",\\n        \\\"data_repository\\\": \\\"Gene Expression Omnibus\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178362\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE135851\\\",\\n        \\\"data_repository\\\": \\\"Gene Expression Omnibus\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE135851\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE149563\\\",\\n        \\\"data_repository\\\": \\\"Gene Expression Omnibus\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE149563\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"dataset-427f1eee6dd44f50bae1ab13f0f3c6a9\\\",\\n        \\\"data_repository\\\": \\\"FASTGenomics\\\",\\n        \\\"dataset_webpage\\\": \\\"https://beta.fastgenomics.org/datasets/detail-dataset-427f1eee6dd44f50bae1ab13f0f3c6a9\\\"\\n    }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755541126,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 1600,\n",
      "        \"completion_tokens\": 1173,\n",
      "        \"total_tokens\": 2773,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE135893', 'data_repository': 'Gene Expression Omnibus', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE135893', 'dataset_id': 'GSE135893'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: GSE135893\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID GSE135893 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: Gene Expression Omnibus, identifier: GSE135893\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE135893 for Gene Expression Omnibus\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE135893 for Gene Expression Omnibus\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1037 - INFO - Found repo_name match for Gene Expression Omnibus\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE135893, repo geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE135893\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 961 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'GSE135893', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE135893'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE136831', 'data_repository': 'Gene Expression Omnibus', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE136831', 'dataset_id': 'GSE136831'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: GSE136831\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID GSE136831 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: Gene Expression Omnibus, identifier: GSE136831\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE136831 for Gene Expression Omnibus\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE136831 for Gene Expression Omnibus\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1037 - INFO - Found repo_name match for Gene Expression Omnibus\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE136831, repo geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE136831\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 961 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'GSE136831', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE136831'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE122960', 'data_repository': 'Gene Expression Omnibus', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122960', 'dataset_id': 'GSE122960'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: GSE122960\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID GSE122960 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: Gene Expression Omnibus, identifier: GSE122960\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE122960 for Gene Expression Omnibus\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE122960 for Gene Expression Omnibus\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1037 - INFO - Found repo_name match for Gene Expression Omnibus\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122960, repo geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122960\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 961 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'GSE122960', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122960'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE134174', 'data_repository': 'Gene Expression Omnibus', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE134174', 'dataset_id': 'GSE134174'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: GSE134174\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID GSE134174 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: Gene Expression Omnibus, identifier: GSE134174\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE134174 for Gene Expression Omnibus\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE134174 for Gene Expression Omnibus\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1037 - INFO - Found repo_name match for Gene Expression Omnibus\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE134174, repo geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE134174\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 961 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'GSE134174', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE134174'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE161382', 'data_repository': 'Gene Expression Omnibus', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE161382', 'dataset_id': 'GSE161382'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: GSE161382\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID GSE161382 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: Gene Expression Omnibus, identifier: GSE161382\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE161382 for Gene Expression Omnibus\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE161382 for Gene Expression Omnibus\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1037 - INFO - Found repo_name match for Gene Expression Omnibus\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE161382, repo geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE161382\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 961 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'GSE161382', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE161382'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE171524', 'data_repository': 'Gene Expression Omnibus', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE171524', 'dataset_id': 'GSE171524'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: GSE171524\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID GSE171524 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: Gene Expression Omnibus, identifier: GSE171524\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE171524 for Gene Expression Omnibus\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE171524 for Gene Expression Omnibus\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1037 - INFO - Found repo_name match for Gene Expression Omnibus\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE171524, repo geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE171524\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 961 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'GSE171524', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE171524'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'EGAS00001004082', 'data_repository': 'European Genome-phenome Archive', 'dataset_webpage': 'https://ega-archive.org/studies/EGAS00001004082', 'dataset_id': 'EGAS00001004082'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: EGAS00001004082\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID EGAS00001004082 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: European Genome-phenome Archive, identifier: EGAS00001004082\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1037 - INFO - Found repo_name match for European Genome-phenome Archive\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://ega-archive.org/studies/EGAS00001004082, repo ega-archive.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://ega-archive.org/studies/EGAS00001004082\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 967 - INFO - No dataset_webpage_url_ptr found for ega-archive.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'EGAS00001004082', 'data_repository': 'ega-archive.org', 'dataset_webpage': 'https://ega-archive.org/studies/EGAS00001004082'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn21041850', 'data_repository': 'Synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn21041850', 'dataset_id': 'syn21041850'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: syn21041850\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID syn21041850 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: Synapse.org, identifier: syn21041850\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn21041850 for Synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn21041850 for Synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn21041850 for Synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn21041850 for Synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: Synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.synapse.org/#!Synapse:syn21041850, repo synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.synapse.org/#!Synapse:syn21041850\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Link does not match the pattern https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'syn21041850', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'LMEX0000004396', 'data_repository': 'LungMAP.net', 'dataset_webpage': 'https://lungmap.net/breath-omics-experiment-page/?experiment_id=LMEX0000004396', 'dataset_id': 'LMEX0000004396'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: LMEX0000004396\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID LMEX0000004396 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: LungMAP.net, identifier: LMEX0000004396\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier LMEX0000004396 for LungMAP.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier LMEX0000004396 for LungMAP.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier LMEX0000004396 for LungMAP.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier LMEX0000004396 for LungMAP.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier LMEX0000004396 for LungMAP.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: https://lungmap.net/breath-omics-experiment-page/?experiment_id=LMEX0000004396\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://lungmap.net/breath-omics-experiment-page/?experiment_id=LMEX0000004396, repo lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.lungmap.net/dataset?experiment_id=LMEX0000004396\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 970 - INFO - Dataset ID LMEX0000004396 found in resolved dataset page https://www.lungmap.net/dataset?experiment_id=LMEX0000004396\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'LMEX0000004396', 'data_repository': 'lungmap.net', 'dataset_webpage': 'https://www.lungmap.net/dataset?experiment_id=LMEX0000004396'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE122332', 'data_repository': 'Gene Expression Omnibus', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122332', 'dataset_id': 'GSE122332'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: GSE122332\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID GSE122332 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: Gene Expression Omnibus, identifier: GSE122332\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE122332 for Gene Expression Omnibus\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE122332 for Gene Expression Omnibus\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1037 - INFO - Found repo_name match for Gene Expression Omnibus\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122332, repo geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122332\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 961 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'GSE122332', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122332'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'LMEX0000004397', 'data_repository': 'LungMAP.net', 'dataset_webpage': 'https://lungmap.net/breath-omics-experiment-page/?experiment_id=LMEX0000004397', 'dataset_id': 'LMEX0000004397'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: LMEX0000004397\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID LMEX0000004397 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: LungMAP.net, identifier: LMEX0000004397\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier LMEX0000004397 for LungMAP.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier LMEX0000004397 for LungMAP.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier LMEX0000004397 for LungMAP.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier LMEX0000004397 for LungMAP.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier LMEX0000004397 for LungMAP.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: https://lungmap.net/breath-omics-experiment-page/?experiment_id=LMEX0000004397\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://lungmap.net/breath-omics-experiment-page/?experiment_id=LMEX0000004397, repo lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.lungmap.net/dataset?experiment_id=LMEX0000004397\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 970 - INFO - Dataset ID LMEX0000004397 found in resolved dataset page https://www.lungmap.net/dataset?experiment_id=LMEX0000004397\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'LMEX0000004397', 'data_repository': 'lungmap.net', 'dataset_webpage': 'https://www.lungmap.net/dataset?experiment_id=LMEX0000004397'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE178362', 'data_repository': 'Gene Expression Omnibus', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178362', 'dataset_id': 'GSE178362'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: GSE178362\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID GSE178362 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: Gene Expression Omnibus, identifier: GSE178362\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE178362 for Gene Expression Omnibus\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE178362 for Gene Expression Omnibus\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1037 - INFO - Found repo_name match for Gene Expression Omnibus\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178362, repo geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178362\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 961 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'GSE178362', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178362'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE135851', 'data_repository': 'Gene Expression Omnibus', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE135851', 'dataset_id': 'GSE135851'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: GSE135851\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID GSE135851 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: Gene Expression Omnibus, identifier: GSE135851\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE135851 for Gene Expression Omnibus\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE135851 for Gene Expression Omnibus\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1037 - INFO - Found repo_name match for Gene Expression Omnibus\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE135851, repo geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 981 - WARNING - Error resolving URL https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE135851: HTTPSConnectionPool(host='www.ncbi.nlm.nih.gov', port=443): Read timed out. (read timeout=5)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 961 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'GSE135851', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE135851'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE149563', 'data_repository': 'Gene Expression Omnibus', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE149563', 'dataset_id': 'GSE149563'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: GSE149563\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID GSE149563 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: Gene Expression Omnibus, identifier: GSE149563\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE149563 for Gene Expression Omnibus\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE149563 for Gene Expression Omnibus\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1037 - INFO - Found repo_name match for Gene Expression Omnibus\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE149563, repo geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE149563\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 961 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'GSE149563', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE149563'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'dataset-427f1eee6dd44f50bae1ab13f0f3c6a9', 'data_repository': 'FASTGenomics', 'dataset_webpage': 'https://beta.fastgenomics.org/datasets/detail-dataset-427f1eee6dd44f50bae1ab13f0f3c6a9', 'dataset_id': 'dataset-427f1eee6dd44f50bae1ab13f0f3c6a9'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: dataset-427f1eee6dd44f50bae1ab13f0f3c6a9\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID dataset-427f1eee6dd44f50bae1ab13f0f3c6a9 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: FASTGenomics, identifier: dataset-427f1eee6dd44f50bae1ab13f0f3c6a9\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier dataset-427f1eee6dd44f50bae1ab13f0f3c6a9 for FASTGenomics\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier dataset-427f1eee6dd44f50bae1ab13f0f3c6a9 for FASTGenomics\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier dataset-427f1eee6dd44f50bae1ab13f0f3c6a9 for FASTGenomics\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier dataset-427f1eee6dd44f50bae1ab13f0f3c6a9 for FASTGenomics\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier dataset-427f1eee6dd44f50bae1ab13f0f3c6a9 for FASTGenomics\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: https://beta.fastgenomics.org/datasets/detail-dataset-427f1eee6dd44f50bae1ab13f0f3c6a9\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://beta.fastgenomics.org/datasets/detail-dataset-427f1eee6dd44f50bae1ab13f0f3c6a9, repo beta.fastgenomics.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.fastgenomics.org/datasets/detail-dataset-427f1eee6dd44f50bae1ab13f0f3c6a9\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 970 - INFO - Dataset ID dataset-427f1eee6dd44f50bae1ab13f0f3c6a9 found in resolved dataset page https://www.fastgenomics.org/datasets/detail-dataset-427f1eee6dd44f50bae1ab13f0f3c6a9\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'dataset-427f1eee6dd44f50bae1ab13f0f3c6a9', 'data_repository': 'beta.fastgenomics.org', 'dataset_webpage': 'https://www.fastgenomics.org/datasets/detail-dataset-427f1eee6dd44f50bae1ab13f0f3c6a9'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 216\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 397 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-3c58f83b349a979f7e363eb32d29d05ca9bbd1f27ec46c074a9fa635b2040359\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 397 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-1225d78d-6a61-4ad7-9c3c-60a08b90ac6c\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755541139,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 502,\n",
      "        \"completion_tokens\": 6,\n",
      "        \"total_tokens\": 508,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 5134\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 1626 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-f1248a7f55a43518ceebca19bd71db92403eccb20defd3b0a5b9b2e2605910c7\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 1626 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-2e2f97fe-8575-4055-b4f9-9b505430ce9f\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE122960\\\",\\n        \\\"data_repository\\\": \\\"GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122960\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE135893\\\",\\n        \\\"data_repository\\\": \\\"GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE135893\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE134174\\\",\\n        \\\"data_repository\\\": \\\"GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE134174\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE136832\\\",\\n        \\\"data_repository\\\": \\\"GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE136832\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE161382\\\",\\n        \\\"data_repository\\\": \\\"GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE161382\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"EGAS00001004082\\\",\\n        \\\"data_repository\\\": \\\"EGA\\\",\\n        \\\"dataset_webpage\\\": \\\"https://ega-archive.org/studies/EGAS00001004082\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE171524\\\",\\n        \\\"data_repository\\\": \\\"GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE171524\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"syn21041850\\\",\\n        \\\"data_repository\\\": \\\"Synapse\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.synapse.org/#!Synapse:syn21041850\\\"\\n    }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE122960\\\",\\n        \\\"data_repository\\\": \\\"GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122960\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE135893\\\",\\n        \\\"data_repository\\\": \\\"GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE135893\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE134174\\\",\\n        \\\"data_repository\\\": \\\"GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE134174\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE136832\\\",\\n        \\\"data_repository\\\": \\\"GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE136832\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE161382\\\",\\n        \\\"data_repository\\\": \\\"GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE161382\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"EGAS00001004082\\\",\\n        \\\"data_repository\\\": \\\"EGA\\\",\\n        \\\"dataset_webpage\\\": \\\"https://ega-archive.org/studies/EGAS00001004082\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE171524\\\",\\n        \\\"data_repository\\\": \\\"GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE171524\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"syn21041850\\\",\\n        \\\"data_repository\\\": \\\"Synapse\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.synapse.org/#!Synapse:syn21041850\\\"\\n    }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755541143,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 2153,\n",
      "        \"completion_tokens\": 586,\n",
      "        \"total_tokens\": 2739,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE122960', 'data_repository': 'GEO', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122960', 'dataset_id': 'GSE122960'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: GSE122960\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID GSE122960 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: GEO, identifier: GSE122960\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE122960 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE122960 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122960, repo geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122960\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 961 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'GSE122960', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122960'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE135893', 'data_repository': 'GEO', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE135893', 'dataset_id': 'GSE135893'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: GSE135893\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID GSE135893 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: GEO, identifier: GSE135893\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE135893 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE135893 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE135893, repo geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE135893\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 961 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'GSE135893', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE135893'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE134174', 'data_repository': 'GEO', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE134174', 'dataset_id': 'GSE134174'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: GSE134174\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID GSE134174 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: GEO, identifier: GSE134174\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE134174 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE134174 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE134174, repo geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE134174\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 961 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'GSE134174', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE134174'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE136832', 'data_repository': 'GEO', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE136832', 'dataset_id': 'GSE136832'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: GSE136832\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID GSE136832 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: GEO, identifier: GSE136832\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE136832 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE136832 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE136832, repo geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE136832\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 961 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'GSE136832', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE136832'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE161382', 'data_repository': 'GEO', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE161382', 'dataset_id': 'GSE161382'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: GSE161382\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID GSE161382 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: GEO, identifier: GSE161382\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE161382 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE161382 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE161382, repo geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE161382\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 961 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'GSE161382', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE161382'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'EGAS00001004082', 'data_repository': 'EGA', 'dataset_webpage': 'https://ega-archive.org/studies/EGAS00001004082', 'dataset_id': 'EGAS00001004082'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: EGAS00001004082\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID EGAS00001004082 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: EGA, identifier: EGAS00001004082\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier EGAS00001004082 for EGA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier EGAS00001004082 for EGA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier EGAS00001004082 for EGA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier EGAS00001004082 for EGA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier EGAS00001004082 for EGA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: https://ega-archive.org/studies/EGAS00001004082\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://ega-archive.org/studies/EGAS00001004082, repo ega-archive.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://ega-archive.org/studies/EGAS00001004082\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 967 - INFO - No dataset_webpage_url_ptr found for ega-archive.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'EGAS00001004082', 'data_repository': 'ega-archive.org', 'dataset_webpage': 'https://ega-archive.org/studies/EGAS00001004082'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE171524', 'data_repository': 'GEO', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE171524', 'dataset_id': 'GSE171524'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: GSE171524\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID GSE171524 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: GEO, identifier: GSE171524\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE171524 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE171524 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE171524, repo geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE171524\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 961 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'GSE171524', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE171524'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn21041850', 'data_repository': 'Synapse', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn21041850', 'dataset_id': 'syn21041850'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: syn21041850\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID syn21041850 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: Synapse, identifier: syn21041850\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn21041850 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn21041850 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn21041850 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn21041850 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1037 - INFO - Found repo_name match for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.synapse.org/#!Synapse:syn21041850, repo synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.synapse.org/#!Synapse:syn21041850\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Link does not match the pattern https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'syn21041850', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 1157\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 632 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-ec036946abb2657490536e67f8c61f865bc4f0f97fb4be11db8273817125e91c\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 632 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-9f85b85f-3207-461c-b9e7-27f2cc220ddb\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755541150,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 778,\n",
      "        \"completion_tokens\": 6,\n",
      "        \"total_tokens\": 784,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 377 - INFO - datasets (<class 'list'>): [{'dataset_identifier': 'GSE135893', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE135893'}, {'dataset_identifier': 'GSE136831', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE136831'}, {'dataset_identifier': 'GSE122960', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122960'}, {'dataset_identifier': 'GSE134174', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE134174'}, {'dataset_identifier': 'GSE161382', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE161382'}, {'dataset_identifier': 'GSE171524', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE171524'}, {'dataset_identifier': 'EGAS00001004082', 'data_repository': 'ega-archive.org', 'dataset_webpage': 'https://ega-archive.org/studies/EGAS00001004082'}, {'dataset_identifier': 'syn21041850', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'LMEX0000004396', 'data_repository': 'lungmap.net', 'dataset_webpage': 'https://www.lungmap.net/dataset?experiment_id=LMEX0000004396'}, {'dataset_identifier': 'GSE122332', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122332'}, {'dataset_identifier': 'LMEX0000004397', 'data_repository': 'lungmap.net', 'dataset_webpage': 'https://www.lungmap.net/dataset?experiment_id=LMEX0000004397'}, {'dataset_identifier': 'GSE178362', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178362'}, {'dataset_identifier': 'GSE135851', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE135851'}, {'dataset_identifier': 'GSE149563', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE149563'}, {'dataset_identifier': 'dataset-427f1eee6dd44f50bae1ab13f0f3c6a9', 'data_repository': 'beta.fastgenomics.org', 'dataset_webpage': 'https://www.fastgenomics.org/datasets/detail-dataset-427f1eee6dd44f50bae1ab13f0f3c6a9'}, {'dataset_identifier': 'GSE122960', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122960'}, {'dataset_identifier': 'GSE135893', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE135893'}, {'dataset_identifier': 'GSE134174', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE134174'}, {'dataset_identifier': 'GSE136832', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE136832'}, {'dataset_identifier': 'GSE161382', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE161382'}, {'dataset_identifier': 'EGAS00001004082', 'data_repository': 'ega-archive.org', 'dataset_webpage': 'https://ega-archive.org/studies/EGAS00001004082'}, {'dataset_identifier': 'GSE171524', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE171524'}, {'dataset_identifier': 'syn21041850', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'GSE135893', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE135893'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'GSE136831', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE136831'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'GSE122960', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122960'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'GSE134174', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE134174'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'GSE161382', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE161382'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'GSE171524', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE171524'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'EGAS00001004082', 'data_repository': 'ega-archive.org', 'dataset_webpage': 'https://ega-archive.org/studies/EGAS00001004082'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'syn21041850', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'LMEX0000004396', 'data_repository': 'lungmap.net', 'dataset_webpage': 'https://www.lungmap.net/dataset?experiment_id=LMEX0000004396'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'GSE122332', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122332'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'LMEX0000004397', 'data_repository': 'lungmap.net', 'dataset_webpage': 'https://www.lungmap.net/dataset?experiment_id=LMEX0000004397'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'GSE178362', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178362'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'GSE135851', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE135851'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'GSE149563', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE149563'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'dataset-427f1eee6dd44f50bae1ab13f0f3c6a9', 'data_repository': 'beta.fastgenomics.org', 'dataset_webpage': 'https://www.fastgenomics.org/datasets/detail-dataset-427f1eee6dd44f50bae1ab13f0f3c6a9'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'GSE122960', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122960'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'GSE135893', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE135893'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'GSE134174', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE134174'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'GSE136832', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE136832'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'GSE161382', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE161382'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'EGAS00001004082', 'data_repository': 'ega-archive.org', 'dataset_webpage': 'https://ega-archive.org/studies/EGAS00001004082'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'GSE171524', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE171524'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'syn21041850', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 385 - INFO - Final ret data availability: 23 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 386 - INFO - Final ret data availability: [{'dataset_identifier': 'GSE135893', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE135893', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'GSE136831', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE136831', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'GSE122960', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122960', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'GSE134174', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE134174', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'GSE161382', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE161382', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'GSE171524', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE171524', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'EGAS00001004082', 'data_repository': 'ega-archive.org', 'dataset_webpage': 'https://ega-archive.org/studies/EGAS00001004082', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'syn21041850', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'LMEX0000004396', 'data_repository': 'lungmap.net', 'dataset_webpage': 'https://www.lungmap.net/dataset?experiment_id=LMEX0000004396', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'GSE122332', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122332', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'LMEX0000004397', 'data_repository': 'lungmap.net', 'dataset_webpage': 'https://www.lungmap.net/dataset?experiment_id=LMEX0000004397', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'GSE178362', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178362', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'GSE135851', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE135851', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'GSE149563', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE149563', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'dataset-427f1eee6dd44f50bae1ab13f0f3c6a9', 'data_repository': 'beta.fastgenomics.org', 'dataset_webpage': 'https://www.fastgenomics.org/datasets/detail-dataset-427f1eee6dd44f50bae1ab13f0f3c6a9', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'GSE122960', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122960', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'GSE135893', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE135893', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'GSE134174', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE134174', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'GSE136832', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE136832', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'GSE161382', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE161382', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'EGAS00001004082', 'data_repository': 'ega-archive.org', 'dataset_webpage': 'https://ega-archive.org/studies/EGAS00001004082', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'GSE171524', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE171524', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'syn21041850', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1079 - INFO - Fetching metadata for 23 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: GSE135893\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: geo, identifier: GSE135893\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE135893 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE135893 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for GSE135893 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE135893\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE135893\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: GSE136831\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: geo, identifier: GSE136831\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE136831 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE136831 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for GSE136831 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 2 with repo: geo and accession_id: GSE136831\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE136831\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 2: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: GSE122960\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: geo, identifier: GSE122960\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE122960 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE122960 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for GSE122960 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 3 with repo: geo and accession_id: GSE122960\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122960\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 3: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: GSE134174\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: geo, identifier: GSE134174\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE134174 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE134174 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for GSE134174 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 4 with repo: geo and accession_id: GSE134174\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE134174\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 4: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: GSE161382\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: geo, identifier: GSE161382\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE161382 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE161382 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for GSE161382 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 5 with repo: geo and accession_id: GSE161382\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE161382\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 5: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: GSE171524\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: geo, identifier: GSE171524\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE171524 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE171524 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for GSE171524 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 6 with repo: geo and accession_id: GSE171524\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE171524\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 6: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: EGAS00001004082\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: ega-archive.org, identifier: EGAS00001004082\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: ega-archive.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for EGAS00001004082 in ega-archive.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 7 with repo: ega-archive.org and accession_id: EGAS00001004082\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://ega-archive.org/studies/EGAS00001004082\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 7: Contact to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: syn21041850\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn21041850\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn21041850 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn21041850 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn21041850 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn21041850 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for syn21041850 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 8 with repo: synapse.org and accession_id: syn21041850\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.synapse.org/Synapse:syn21041850\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 8: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: LMEX0000004396\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: lungmap.net, identifier: LMEX0000004396\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier LMEX0000004396 for lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier LMEX0000004396 for lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier LMEX0000004396 for lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier LMEX0000004396 for lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier LMEX0000004396 for lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 900 - WARNING - URL lungmap.net may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for LMEX0000004396 in lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 9 with repo: lungmap.net and accession_id: LMEX0000004396\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1152 - WARNING - Repository lungmap.net unknown in Ontology. Skipping dataset page 9.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 10 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: GSE122332\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: geo, identifier: GSE122332\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE122332 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE122332 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for GSE122332 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 10 with repo: geo and accession_id: GSE122332\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122332\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 10: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 11 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: LMEX0000004397\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: lungmap.net, identifier: LMEX0000004397\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier LMEX0000004397 for lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier LMEX0000004397 for lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier LMEX0000004397 for lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier LMEX0000004397 for lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier LMEX0000004397 for lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 900 - WARNING - URL lungmap.net may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for LMEX0000004397 in lungmap.net\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 11 with repo: lungmap.net and accession_id: LMEX0000004397\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1152 - WARNING - Repository lungmap.net unknown in Ontology. Skipping dataset page 11.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 12 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: GSE178362\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: geo, identifier: GSE178362\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE178362 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE178362 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for GSE178362 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 12 with repo: geo and accession_id: GSE178362\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178362\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 12: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 13 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: GSE135851\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: geo, identifier: GSE135851\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE135851 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE135851 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for GSE135851 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 13 with repo: geo and accession_id: GSE135851\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE135851\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 13: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 14 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: GSE149563\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: geo, identifier: GSE149563\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE149563 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE149563 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for GSE149563 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 14 with repo: geo and accession_id: GSE149563\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE149563\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 14: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 15 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: dataset-427f1eee6dd44f50bae1ab13f0f3c6a9\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: beta.fastgenomics.org, identifier: dataset-427f1eee6dd44f50bae1ab13f0f3c6a9\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier dataset-427f1eee6dd44f50bae1ab13f0f3c6a9 for beta.fastgenomics.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier dataset-427f1eee6dd44f50bae1ab13f0f3c6a9 for beta.fastgenomics.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier dataset-427f1eee6dd44f50bae1ab13f0f3c6a9 for beta.fastgenomics.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier dataset-427f1eee6dd44f50bae1ab13f0f3c6a9 for beta.fastgenomics.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier dataset-427f1eee6dd44f50bae1ab13f0f3c6a9 for beta.fastgenomics.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: beta.fastgenomics.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 900 - WARNING - URL beta.fastgenomics.org may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for dataset-427f1eee6dd44f50bae1ab13f0f3c6a9 in beta.fastgenomics.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 15 with repo: beta.fastgenomics.org and accession_id: dataset-427f1eee6dd44f50bae1ab13f0f3c6a9\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1152 - WARNING - Repository beta.fastgenomics.org unknown in Ontology. Skipping dataset page 15.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 16 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: GSE122960\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: geo, identifier: GSE122960\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE122960 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE122960 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for GSE122960 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 16 with repo: geo and accession_id: GSE122960\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122960\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 16: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 17 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: GSE135893\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: geo, identifier: GSE135893\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE135893 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE135893 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for GSE135893 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 17 with repo: geo and accession_id: GSE135893\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE135893\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 17: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 18 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: GSE134174\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: geo, identifier: GSE134174\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE134174 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE134174 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for GSE134174 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 18 with repo: geo and accession_id: GSE134174\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE134174\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 18: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 19 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: GSE136832\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: geo, identifier: GSE136832\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE136832 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE136832 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for GSE136832 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 19 with repo: geo and accession_id: GSE136832\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE136832\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 19: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 20 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: GSE161382\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: geo, identifier: GSE161382\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE161382 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE161382 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for GSE161382 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 20 with repo: geo and accession_id: GSE161382\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE161382\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 20: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 21 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: EGAS00001004082\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: ega-archive.org, identifier: EGAS00001004082\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: ega-archive.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for EGAS00001004082 in ega-archive.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 21 with repo: ega-archive.org and accession_id: EGAS00001004082\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://ega-archive.org/studies/EGAS00001004082\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 21: Contact to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 22 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: GSE171524\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: geo, identifier: GSE171524\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE171524 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE171524 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for GSE171524 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 22 with repo: geo and accession_id: GSE171524\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE171524\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 22: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 23 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: syn21041850\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn21041850\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn21041850 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn21041850 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn21041850 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn21041850 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for syn21041850 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 23 with repo: synapse.org and accession_id: syn21041850\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.synapse.org/Synapse:syn21041850\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 23: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1156 - INFO - Updated datasets len: 23\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 221 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 32, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'source_section', 'retrieval_pattern', 'access_mode', 'link',\n",
      "       'source_url', 'download_link', 'title', 'content_type', 'id', 'caption',\n",
      "       'description', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 456 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 462 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 539 - INFO - 18th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC4444046)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 347 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC4444046\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 354 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 533 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC4444046&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 383 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC4444046.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b616bc50>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 375 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 310 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 399 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC4444046 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 413 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 439 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC4444046, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC4444046)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 11 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#S7'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for S7: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 51 <a> elements with href='#SD1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 9833) for SD1: We assembled a TCGA Pan-Cancer dataset of exome sequencing, array copy number, and RNA-seq data from 3,281 samples from 12 cancer types, analyzing single nucleotide variants (SNVs), small indels, and copy number aberrations (CNAs) in 19,424 transcripts ( Figure 1a [href=#F1] and Supplementary Figure 1 [href=this file] ). The resulting “star subnetworks” centered on the hot node ( Supplementary Figure 2 [href=this file] ; Online Methods) contain many neighboring genes that are not mutated at appreciable frequency and are of limited biological interest.\n",
      "We introduce the HotNet2 (HotNet iffusion riented ubnetworks) algorithm to address the problem of finding significantly mutated subnetworks on large, broad mutation frequency spectrum datasets like Pan-Cancer ( Figure 1c [href=#F1] and Supplementary Figure 3 [href=this file] ). HotNet2 uses a modified diffusion process and considers the source, or directionality, of heat flow in the identification of subnetworks ( Supplementary Figure 4 [href=this file] ).\n",
      "We combined the resulting subnetworks into 14 consensus subnetworks that were found across different gene scores and networks ( < 0.004, Supplementary Table 3 [href=#SD2] ), plus the condensin complex and CLASP/CLIP proteins ( Supplementary Figure 5 [href=this file] ) that were significant in individual interaction networks ( Supplementary Tables 6,7 [href=#SD2] ). We developed an online interactive viewer (see URLs and Supplementary Figure 6 [href=this file] ) for Pan-Cancer HotNet2 subnetworks.\n",
      "The subnetworks and linker genes ( Figure 2a [href=#F2] ) include: portions of well-known cancer pathways such as TP53, PI3K, NOTCH, and receptor tyrosine kinases (RTKs; Supplementary Figure 7 [href=this file] ), as well as pathways and complexes that have more recently been observed to be important in cancer such as SWI/SNF complex, BAP1 complex, NFE2L2-KEAP1 ( Supplementary Figures 8,9 [href=this file] ), and RUNX1-CBFB core binding complex ( Supplementary Figure 10 [href=this file] ). The fifth most mutated subnetwork (16.9% of samples) consists of MLL2 and MLL3 and the putative interacting protein KDM6A ( Supplementary Figure 11 [href=this file] ), and was highly mutated (28.9% of samples) in TCGA Pan-Cancer squamous integrated subtype . The MHC Class I subnetwork ( Supplementary Figure 12 [href=this file] ) is an example of the ability of HotNet2 ability to identify rarely mutated cancer genes; all of the genes in the subnetwork are mutated in fewer than 35 samples (1.1%), yet four of the five genes have recently been proposed as novel cancer genes . Additional analyses are in the Supplementary Note [href=this file] .\n",
      "To obtain additional support for these genes we examined whether they had either an excess of inactivating mutations or a cluster of missense mutations in protein sequence (using NMC ) or in protein structure (using iPAC ; Supplementary Figures 13,14 [href=this file] and Supplementary Tables 21,22 [href=#SD2] ). We find that genes in HotNet2 consensus subnetworks are enriched for inactivating mutations ( < 0.0001) or mutation clusters ( < 0.0001) compared to genes not in subnetworks ( Supplementary Table 6–18 [href=#SD2] and Supplementary Note Section 5.1 [href=this file] ). Finally, we evaluated a subset of the mutations in these genes using RNA-Seq and whole-genome sequencing (WGS) data from the same samples, and found RNA-Seq and/or WGS reads that validated 39 mutations in these novel genes ( Supplementary Note Section 6 [href=this file] and Supplementary Table 23 [href=#SD2] ).\n",
      "This exclusivity was largely due to LUAD samples with mutually exclusive and mutations ( Supplementary Figure 15 [href=this file] ).\n",
      "The three largest subnetworks – including a subnetwork, a subnetwork, and a subnetwork – contain many well-known cancer genes ( Supplementary Tables 8–10 [href=#SD2] and Supplementary Figures 16,17 [href=this file] ).\n",
      "The subnetwork also includes , which has a significant ( < 8×10 ) cluster of missense mutations ( Supplementary Figure 18 [href=this file] ).\n",
      "We find that mutations in the SWI/SNF and BAP1 complexes show even greater mutual exclusivity ( =9.4×10 ) in KIRC because of mutations in additional genes in these complexes besides and , respectively ( Supplementary Note Section 5.8.1 [href=this file] ).\n",
      "Finally, RNA-seq and whole-genome sequencing data from the same samples provide further validation of the somatic mutations in , , , , , and and show that some of these mutations are expressed ( Supplementary Note Section 6 [href=this file] and Supplementary Table 39 [href=#SD2] ).\n",
      "Our analysis complements other recent Pan-Cancer analyses including studies that analyze only one type of aberration or restrict attention to recurrent aberrations ( Supplementary Note Section 8.3 [href=this file] and Supplementary Table 27 [href=#SD2] ).\n",
      "We selected the threshold of 400 aberrations per sample as the derivative of the number of mutations per sample starts increasing rapidly beyond this value ( Supplementary Figure 19 [href=this file] ). Finally, we used only those genes that had at least 3 reads from RNA-seq data in at least 70% of samples of at least one of the cancer types, as described in The resulting dataset contained aberrations in 11,565 genes and 3110 samples ( Supplementary Figure 1 [href=this file] ). Additional details of the datasets are in the Supplementary Note [href=this file] .\n",
      "While interaction networks have proven useful in analyzing various types of genomic data , statistically robust identification of significantly mutated subnetworks is a difficult problem with several major challenges ( Supplemental Note Section 1.1 [href=this file] ). HotNet2 addresses these challenges and identifies significantly mutated subnetworks of a genome-scale interaction network, using an insulated heat diffusion process that considers both the scores on individual genes/proteins as well as the topology of interactions between genes/proteins ( Supplementary Figure 3 [href=this file] ).\n",
      "The insulated heat model can also described in terms of a random walk with restart ( Supplementary Note Section 1.2 [href=this file] ).\n",
      "Since the diffusion matrix is not symmetric and in general ( ) ≠ ( ), the similarity ( , ) is also not symmetric ( Supplementary Note Section 1.2.1 [href=this file] ).\n",
      "The statistical test is the same as the two-stage statistical test introduced in the original HotNet algorithm ( Supplementary Note Section 1.3 [href=this file] , Supplementary Figures 20–23 [href=this file] and Supplementary Table 28 [href=#SD2] ).\n",
      "HotNet2 has two parameters β and δ, and selects values for both of these parameters using procedures. β is selected from the protein-protein interaction network, of any gene scores ( Supplementary Note Section 1.4.1 [href=this file] , Supplementary Figure 24 [href=this file] , and Supplementary Table 29 [href=#SD2] ). The value of δ is chosen such that large connected components are not found using the observed gene score distribution on random networks with the same degree distribution as the observed network ( Supplementary Note Section 1.4.2 [href=this file] , Supplementary Figure 25 [href=this file] , and Supplementary Table 30 [href=#SD2] ).\n",
      "See Supplementary Figure 2 [href=this file] .\n",
      "Expanding on this third point, when undirected diffusion algorithms like HotNet or related network propagation algorithms are run on large datasets containing a wide range of gene scores (e.g. the Pan-Cancer dataset), many of the resulting subnetworks are “hot” determined by a single high-scoring node and the immediate neighbors of this node ( Supplementary Figure 2 [href=this file] ).\n",
      "The HotNet2 subnetworks also have a higher fraction of interactions with proteins other than a hot central node ( Supplementary Note Section 7.1 [href=this file] ). In short, the results show that HotNet2 achieves higher sensitivity and specificity than HotNet ( Supplementary Note Section 7.2 [href=this file] and Supplementary Figure 26 [href=this file] ).\n",
      "We find that HotNet2 provides both new insights and a simpler summary of groups of interacting genes, and is a useful complement (or arguably a replacement for) other pathway tests ( Supplementary Note Section 8.1 [href=this file] ). We also show that HotNet2 has much higher specificity than HotNet, DAVID, and GSEA in identifying genes satisfying the 20/20 rule ( Supplementary Note Section 8.1.4 [href=this file] , Supplementary Figure 27 [href=this file] , and Supplementary Tables 34–36 [href=#SD2] ). Finally, we find that HotNet2 was more stable than HotNet in identifying 20/20 genes using cross-validation ( Supplementary Note Section 7.3 [href=this file] and Supplementary Figure 28 [href=this file] ).\n",
      "We found that MEMo was unable to run on the Pan-Cancer mutation frequency dataset, consistent with the authors’ recommendation that MEMo should be run only on a small number of significant mutations (details in Supplementary Note Section 8.2 [href=this file] ).\n",
      "We ran HotNet2 on each combination of gene scores (mutation frequency and MutSigCV -values; see Supplementary Note Section 2.2 [href=this file] ) and interaction networks (HINT+HI2012 , iRefIndex , and Multinet ; Supplementary Note Section 2.5 [href=this file] and Supplementary Figure 29 [href=this file] ). This procedure is described in Supplementary Note Section 1.5 [href=this file] .\n",
      "Additional details of the statistical procedure are in Supplementary Note Section 1.3 [href=this file] .\n",
      "Notable among the additional subnetworks identified when we remove the requirement for RNA-Seq expression is a subnetwork ( Supplementary Table 25 [href=#SD2] ) containing members of the telomerase complex (including and ) that has a well-studied role in cancer ( Supplementary Figure 30 [href=this file] and Supplementary Table 38 [href=#SD2] ).\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 34 <a> elements with href='#SD2'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 7301) for SD2: HotNet2 identified a significant number of subnetworks ( < 0.01 Supplementary Tables 1–2 [href=this file] ) for each of the two gene scores and three networks. We combined the resulting subnetworks into 14 consensus subnetworks that were found across different gene scores and networks ( < 0.004, Supplementary Table 3 [href=this file] ), plus the condensin complex and CLASP/CLIP proteins ( Supplementary Figure 5 [href=#SD1] ) that were significant in individual interaction networks ( Supplementary Tables 6,7 [href=this file] ).\n",
      "Many of the subnetworks exhibit a significant enrichment for mutations in a subset of cancer types, including many previously unreported associations ( Supplementary Tables 6–18 [href=this file] ). In addition, the HotNet2 Pan-Cancer analysis provides a clearer and more robust summary of subnetworks and novel genes than HotNet2 analysis of individual cancer types ( Supplementary Table 19 [href=this file] ).\n",
      "These subnetworks and linkers include a total of 147 genes, including many well-known cancer genes and pathways, but also including genes with mutations that are too rare to be significant by the single-gene tests ( Supplementary Table 20 [href=this file] ). Table 1 [href=#T1] lists a subset of promising candidates, with the full list and associated references in Supplementary Table 20 [href=this file] .\n",
      "More information on these genes – as well as other candidate driver genes – is in Supplementary Table 20 [href=this file] .\n",
      "To obtain additional support for these genes we examined whether they had either an excess of inactivating mutations or a cluster of missense mutations in protein sequence (using NMC ) or in protein structure (using iPAC ; Supplementary Figures 13,14 [href=#SD1] and Supplementary Tables 21,22 [href=this file] ). We find that genes in HotNet2 consensus subnetworks are enriched for inactivating mutations ( < 0.0001) or mutation clusters ( < 0.0001) compared to genes not in subnetworks ( Supplementary Table 6–18 [href=this file] and Supplementary Note Section 5.1 [href=#SD1] ). Finally, we evaluated a subset of the mutations in these genes using RNA-Seq and whole-genome sequencing (WGS) data from the same samples, and found RNA-Seq and/or WGS reads that validated 39 mutations in these novel genes ( Supplementary Note Section 6 [href=#SD1] and Supplementary Table 23 [href=this file] ).\n",
      "Multiple pairs of genes within these subnetworks show co-occurring mutations ( Supplementary Table 24 [href=this file] ). We observe significant mutual exclusivity within 4 of our subnetworks ( Supplementary Table 25 [href=this file] ).\n",
      "The three largest subnetworks – including a subnetwork, a subnetwork, and a subnetwork – contain many well-known cancer genes ( Supplementary Tables 8–10 [href=this file] and Supplementary Figures 16,17 [href=#SD1] ).\n",
      "HotNet2 Pan-Cancer analyses also revealed a number of novel genes ( Supplementary Table 20 [href=this file] ) within these three subnetworks.\n",
      "The sixth most mutated HotNet2 Pan-Cancer subnetwork (16.8% of samples) includes multiple members of the SWI/SNF chromatin-remodeling complex ( Figure 3a [href=#F3] and Supplementary Table 12 [href=this file] ).\n",
      "Another HotNet2 Pan-Cancer subnetwork (mutated in 7.1% of samples) overlaps the BAP1 complex ( Figure 3b [href=#F3] and Supplementary Table 13 [href=this file] ).\n",
      "HotNet2 Pan-Cancer analysis identifies 4/5 members of the cohesin complex as a significantly mutated subnetwork (7.3% of samples, Figure 4a [href=#F4] and Supplementary Table 15 [href=this file] ).\n",
      "The combined subnetwork is mutated in 4.2% of samples ( Figure 4b [href=#F4] and Supplementary Table 6 [href=this file] ). Finally, RNA-seq and whole-genome sequencing data from the same samples provide further validation of the somatic mutations in , , , , , and and show that some of these mutations are expressed ( Supplementary Note Section 6 [href=#SD1] and Supplementary Table 39 [href=this file] ).\n",
      "Some of these rarely mutated genes are inevitably false positive predictions of the analysis, but many (including , , , , , and others) exhibit either mutational clustering or protein interactions that support a role for the observed somatic aberrations ( Supplementary Tables 6–18 [href=this file] ).\n",
      "Our analysis complements other recent Pan-Cancer analyses including studies that analyze only one type of aberration or restrict attention to recurrent aberrations ( Supplementary Note Section 8.3 [href=#SD1] and Supplementary Table 27 [href=this file] ).\n",
      "The statistical test is the same as the two-stage statistical test introduced in the original HotNet algorithm ( Supplementary Note Section 1.3 [href=#SD1] , Supplementary Figures 20–23 [href=#SD1] and Supplementary Table 28 [href=this file] ).\n",
      "HotNet2 has two parameters β and δ, and selects values for both of these parameters using procedures. β is selected from the protein-protein interaction network, of any gene scores ( Supplementary Note Section 1.4.1 [href=#SD1] , Supplementary Figure 24 [href=#SD1] , and Supplementary Table 29 [href=this file] ). We evaluated the sensitivity of the HotNet2 results to the value of β and found that varying β ±10% has only a minor effect on the results, with at most 7 genes (3.8% of total) added/removed from the subnetworks ( Supplementary Table 28 [href=this file] ). The value of δ is chosen such that large connected components are not found using the observed gene score distribution on random networks with the same degree distribution as the observed network ( Supplementary Note Section 1.4.2 [href=#SD1] , Supplementary Figure 25 [href=#SD1] , and Supplementary Table 30 [href=this file] ). We evaluated the sensitivity of the HotNet2 results to the value of δ, and found that varying δ ±5% changed at most 35 genes (12.3% of total) in the subnetworks ( Supplementary Table 29 [href=this file] ).\n",
      "We found that HotNet2 returns >80% fewer hot stars/spiders than HotNet on the Pan-Cancer datasets ( Supplementary Table 31 [href=this file] ). This is a major difference between the algorithms and is one of the reasons why HotNet fails to find statistically significant results ( ≤ 0.01 for any subnetwork size ) on three of six runs ( Supplementary Table 32,33 [href=this file] ), while HotNet2 finds statistically significant results on all six runs.\n",
      "We also show that HotNet2 has much higher specificity than HotNet, DAVID, and GSEA in identifying genes satisfying the 20/20 rule ( Supplementary Note Section 8.1.4 [href=#SD1] , Supplementary Figure 27 [href=#SD1] , and Supplementary Tables 34–36 [href=this file] ).\n",
      "Most of the subnetworks (12/14) identified by HotNet2 were also found when we remove the requirement for RNA-Seq expression ( Supplementary Table 37 [href=this file] ). Notable among the additional subnetworks identified when we remove the requirement for RNA-Seq expression is a subnetwork ( Supplementary Table 25 [href=this file] ) containing members of the telomerase complex (including and ) that has a well-studied role in cancer ( Supplementary Figure 30 [href=#SD1] and Supplementary Table 38 [href=this file] ). We found only minor differences in the HotNet2 subnetworks ( Supplementary Table 39 [href=this file] ), demonstrating that our reported subnetworks are altered by somatic aberrations in these samples.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#SD3'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for SD3: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#fn-group1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 27) for fn-group1: Footnotes [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#fn-group1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 27) for fn-group1: Footnotes [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#fn-group1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 27) for fn-group1: Footnotes [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#fn-group1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 27) for fn-group1: Footnotes [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 1 <a> elements with href='#fn-group1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 27) for fn-group1: Footnotes [href=this file]\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 384 - INFO - Chunking the HTML content for the parsing step.\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 172 - INFO - Extracting data availability elements from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: section.data-availability-statement\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: section.associated-data\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Data Availability')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: h3.pmc_sec_title:-soup-contains('Data Availability')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Associated Data')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Footnotes')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 221 - INFO - Found 3 data availability elements from HTML.\u001B[0m\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "Batches:   0%|          | 0/1 [00:00<?, ?it/s]"
      ],
      "application/vnd.jupyter.widget-view+json": {
       "version_major": 2,
       "version_minor": 0,
       "model_id": "7e81e5aa7c7e4d3587befbf5ea44147c"
      }
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001B[97membeddings_retriever.py - line 51 - INFO - Searching for top-5 passages similar to the query by embeddings.\u001B[0m\n",
      "\u001B[97membeddings_retriever.py - line 33 - INFO - Computing L2 distances using numpy.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 23241\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 6153 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-66492bb9c0d77a23159cba34b298855de5443cf0367e2a82595110e9675d1fe9\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 6153 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-87840bd7-1884-4a37-b03e-fc3a6da820e9\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"syn1710680\\\",\\n        \\\"data_repository\\\": \\\"Synapse\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.synapse.org/#!Synapse:syn1710680\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"syn1729383\\\",\\n        \\\"data_repository\\\": \\\"Synapse\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.synapse.org/#!Synapse:syn1729383\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"syn1734155\\\",\\n        \\\"data_repository\\\": \\\"TCGA Pan-Cancer dataset\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.synapse.org/#!Synapse:syn1734155\\\"\\n    }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"syn1710680\\\",\\n        \\\"data_repository\\\": \\\"Synapse\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.synapse.org/#!Synapse:syn1710680\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"syn1729383\\\",\\n        \\\"data_repository\\\": \\\"Synapse\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.synapse.org/#!Synapse:syn1729383\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"syn1734155\\\",\\n        \\\"data_repository\\\": \\\"TCGA Pan-Cancer dataset\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.synapse.org/#!Synapse:syn1734155\\\"\\n    }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755541166,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 7766,\n",
      "        \"completion_tokens\": 207,\n",
      "        \"total_tokens\": 7973,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn1710680', 'data_repository': 'Synapse', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn1710680', 'dataset_id': 'syn1710680'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: syn1710680\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID syn1710680 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: Synapse, identifier: syn1710680\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn1710680 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn1710680 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn1710680 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn1710680 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1037 - INFO - Found repo_name match for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.synapse.org/#!Synapse:syn1710680, repo synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.synapse.org/#!Synapse:syn1710680\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Link does not match the pattern https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'syn1710680', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn1729383', 'data_repository': 'Synapse', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn1729383', 'dataset_id': 'syn1729383'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: syn1729383\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID syn1729383 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: Synapse, identifier: syn1729383\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn1729383 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn1729383 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn1729383 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn1729383 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1037 - INFO - Found repo_name match for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.synapse.org/#!Synapse:syn1729383, repo synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.synapse.org/#!Synapse:syn1729383\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Link does not match the pattern https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'syn1729383', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn1734155', 'data_repository': 'TCGA Pan-Cancer dataset', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn1734155', 'dataset_id': 'syn1734155'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: syn1734155\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID syn1734155 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: TCGA Pan-Cancer dataset, identifier: syn1734155\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn1734155 for TCGA Pan-Cancer dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn1734155 for TCGA Pan-Cancer dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn1734155 for TCGA Pan-Cancer dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn1734155 for TCGA Pan-Cancer dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier syn1734155 for TCGA Pan-Cancer dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1051 - INFO - Found id_pattern match for TCGA Pan-Cancer dataset in (?<!\\d)syn\\d{7,12}(?!\\d)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.synapse.org/#!Synapse:syn1734155, repo synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.synapse.org/#!Synapse:syn1734155\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Link does not match the pattern https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'syn1734155', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1079 - INFO - Fetching metadata for 3 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: syn1710680\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn1710680\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn1710680 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn1710680 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn1710680 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn1710680 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for syn1710680 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn1710680\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.synapse.org/Synapse:syn1710680\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: syn1729383\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn1729383\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn1729383 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn1729383 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn1729383 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn1729383 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for syn1729383 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn1729383\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.synapse.org/Synapse:syn1729383\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: syn1734155\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn1734155\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn1734155 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn1734155 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn1734155 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn1734155 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for syn1734155 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 3 with repo: synapse.org and accession_id: syn1734155\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.synapse.org/Synapse:syn1734155\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 3: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1156 - INFO - Updated datasets len: 3\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 14, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 450 - INFO - Parsed data extraction completed. Elements collected: 7\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 456 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 466 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 539 - INFO - 19th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 347 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 354 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 533 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC10329279&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 383 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b5a01710>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 375 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 310 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 399 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 413 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 439 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 1 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 384 - INFO - Chunking the HTML content for the parsing step.\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 172 - INFO - Extracting data availability elements from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: section.data-availability-statement\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: section.associated-data\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Data Availability')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: h3.pmc_sec_title:-soup-contains('Data Availability')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Associated Data')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Footnotes')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 221 - INFO - Found 1 data availability elements from HTML.\u001B[0m\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "Batches:   0%|          | 0/1 [00:00<?, ?it/s]"
      ],
      "application/vnd.jupyter.widget-view+json": {
       "version_major": 2,
       "version_minor": 0,
       "model_id": "c57f548aa51947c98810dd6fa3f34911"
      }
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001B[97membeddings_retriever.py - line 51 - INFO - Searching for top-5 passages similar to the query by embeddings.\u001B[0m\n",
      "\u001B[97membeddings_retriever.py - line 33 - INFO - Computing L2 distances using numpy.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 3554\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 1231 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-82fbc0ccb79febe095564aae32c986efd3a7795a97662b9cd1dce693f648a6c5\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 1231 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-e1bbfb38-d323-4144-8bab-03a25210ee6e\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755541181,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 1689,\n",
      "        \"completion_tokens\": 6,\n",
      "        \"total_tokens\": 1695,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1079 - INFO - Fetching metadata for 0 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1156 - INFO - Updated datasets len: 0\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 1, with cols: Index(['link', 'title', 'file_info', 'description', 'source_section',\n",
      "       'section_class', 'download_link', 'file_extension', 'a_attr_href',\n",
      "       'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 450 - INFO - Parsed data extraction completed. Elements collected: 1\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 456 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 466 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 539 - INFO - 20th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11145390)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 347 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11145390\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 354 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 533 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC11145390&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 395 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC11145390.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 399 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11145390 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 413 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 417 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11145390, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 1 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b612a2c0>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b612a2c0>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref mmc1: \u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 123 - INFO - Loading patterns for section 'data_availability_sections' for publisher 'PMC'.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 131 - INFO - Section pattern keys for publisher PMC: dict_keys(['data_availability_sections', 'supplementary_material_sections', 'supplementary_data_sections', 'key_resources_table', 'methods_sections'])\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 133 - INFO - Section name: data_availability_sections\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 135 - INFO - Found section 'data_availability_sections' in patterns for publisher 'PMC'.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 137 - INFO - Loaded patterns for section 'data_availability_sections': [\".//sec[@sec-type='data-availability']\", \".//notes[@notes-type='data-availability']\", \".//notes[@notes-type='data-availability']\"]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_data_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 706 - INFO - Found 2 supplementary data sections\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(key_resources_table)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 725 - INFO - Found key resources table: <Element table-wrap at 0x1b0ac5800>.\u001B[0m\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "Batches:   0%|          | 0/1 [00:00<?, ?it/s]"
      ],
      "application/vnd.jupyter.widget-view+json": {
       "version_major": 2,
       "version_minor": 0,
       "model_id": "a0e380eabb92410eaea81f7524dbf042"
      }
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001B[97membeddings_retriever.py - line 51 - INFO - Searching for top-2 passages similar to the query by embeddings.\u001B[0m\n",
      "\u001B[97membeddings_retriever.py - line 33 - INFO - Computing L2 distances using numpy.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 364 - INFO - Processing DAS_content: 4 elements\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 900\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 568 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-a1ddc1c4620c7e543bb7783e2f5c18e002da23ac707981fe800969bb0b49b42c\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 568 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-960dd02e-0d8d-4382-940f-f89d92840147\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"GSE147807\\\",\\n        \\\"data_repository\\\": \\\"GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE147807\\\"\\n    ]\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"GSE147807\\\",\\n        \\\"data_repository\\\": \\\"GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE147807\\\"\\n    ]\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755541188,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 738,\n",
      "        \"completion_tokens\": 81,\n",
      "        \"total_tokens\": 819,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE147807', 'data_repository': 'GEO', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE147807', 'dataset_id': 'GSE147807'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: GSE147807\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID GSE147807 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: GEO, identifier: GSE147807\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE147807 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE147807 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE147807, repo geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE147807\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 961 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'GSE147807', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE147807'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 1174\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 636 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-da2f8a0dcbbc0bb57e4fa1091491b462aa2599ed24e480ad2480c7bbcb31dbfb\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 636 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-797486d6-f937-402a-9fb5-44bc3611d119\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"GSE147807\\\",\\n        \\\"data_repository\\\": \\\"GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE147807\\\"\\n    ]\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"GSE147807\\\",\\n        \\\"data_repository\\\": \\\"GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE147807\\\"\\n    ]\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755541192,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 930,\n",
      "        \"completion_tokens\": 81,\n",
      "        \"total_tokens\": 1011,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE147807', 'data_repository': 'GEO', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE147807', 'dataset_id': 'GSE147807'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: GSE147807\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID GSE147807 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: GEO, identifier: GSE147807\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE147807 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE147807 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE147807, repo geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE147807\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 961 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'GSE147807', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE147807'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 13475\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 3712 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-55894a9d38ad0c6d7e5bd88a1c1a291b10f9ed59e729671a683726db28c64bac\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 3712 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-3f3868e9-5822-46e7-b5e1-07ec1d1058f5\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"Murphy_data\\\",\\n        \\\"data_repository\\\": \\\"Included in clean_ 060720.RData\\\",\\n        \\\"dataset_webpage\\\": \\\"Unavailable\\\"\\n    ]\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"Murphy_data\\\",\\n        \\\"data_repository\\\": \\\"Included in clean_ 060720.RData\\\",\\n        \\\"dataset_webpage\\\": \\\"Unavailable\\\"\\n    ]\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755541193,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 5144,\n",
      "        \"completion_tokens\": 60,\n",
      "        \"total_tokens\": 5204,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'Murphy_data', 'data_repository': 'Included in clean_ 060720.RData', 'dataset_webpage': 'Unavailable', 'dataset_id': 'Murphy_data'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: Murphy_data\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID Murphy_data is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: Included in clean_ 060720.RData, identifier: Murphy_data\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Murphy_data for Included in clean_ 060720.RData\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier Murphy_data for Included in clean_ 060720.RData\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier Murphy_data for Included in clean_ 060720.RData\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Murphy_data for Included in clean_ 060720.RData\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier Murphy_data for Included in clean_ 060720.RData\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: Unavailable\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: Unavailable, repo Unavailable\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 981 - WARNING - Error resolving URL Unavailable: Invalid URL 'Unavailable': No scheme supplied. Perhaps you meant https://Unavailable?\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 972 - INFO - Repository Unavailable not found in ontology\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'Murphy_data', 'data_repository': 'Unavailable', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 397\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 442 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-95b26a782fee3420a79d27fe9d1853d64689f3d3df369d30845b256fb92b518e\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 442 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-d1b25350-d26c-4dc1-8f9a-33a599d760c4\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755541193,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 605,\n",
      "        \"completion_tokens\": 6,\n",
      "        \"total_tokens\": 611,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 377 - INFO - datasets (<class 'list'>): [{'dataset_identifier': 'GSE147807', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE147807'}, {'dataset_identifier': 'GSE147807', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE147807'}, {'dataset_identifier': 'Murphy_data', 'data_repository': 'Unavailable', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'GSE147807', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE147807'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'GSE147807', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE147807'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'Murphy_data', 'data_repository': 'Unavailable', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 385 - INFO - Final ret data availability: 3 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 386 - INFO - Final ret data availability: [{'dataset_identifier': 'GSE147807', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE147807', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'GSE147807', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE147807', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'Murphy_data', 'data_repository': 'Unavailable', 'dataset_webpage': 'n/a', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1079 - INFO - Fetching metadata for 3 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: GSE147807\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: geo, identifier: GSE147807\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE147807 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE147807 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for GSE147807 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE147807\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE147807\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: GSE147807\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: geo, identifier: GSE147807\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE147807 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE147807 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for GSE147807 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 2 with repo: geo and accession_id: GSE147807\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE147807\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 2: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: Murphy_data\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: Unavailable, identifier: Murphy_data\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Murphy_data for Unavailable\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier Murphy_data for Unavailable\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier Murphy_data for Unavailable\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Murphy_data for Unavailable\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier Murphy_data for Unavailable\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: Unavailable\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for Murphy_data in unavailable\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 3 with repo: unavailable and accession_id: Murphy_data\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1152 - WARNING - Repository unavailable unknown in Ontology. Skipping dataset page 3.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1156 - INFO - Updated datasets len: 3\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 221 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 4, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'source_section', 'retrieval_pattern', 'access_mode', 'link',\n",
      "       'source_url', 'download_link', 'title', 'content_type', 'id', 'caption',\n",
      "       'description', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 456 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 462 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 559 - INFO - \n",
      "Progress: 21/25 (84.00%) | Elapsed: 00:04:08 | ETA: 00:00:47\n",
      "\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 539 - INFO - 21th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 347 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 354 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 533 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC11032436&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 461 - WARNING - No links found in the section.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 395 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 399 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 413 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 417 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 1 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b0ac7fc0>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b0ac7fc0>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref No ID: \u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 123 - INFO - Loading patterns for section 'data_availability_sections' for publisher 'PMC'.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 131 - INFO - Section pattern keys for publisher PMC: dict_keys(['data_availability_sections', 'supplementary_material_sections', 'supplementary_data_sections', 'key_resources_table', 'methods_sections'])\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 133 - INFO - Section name: data_availability_sections\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 135 - INFO - Found section 'data_availability_sections' in patterns for publisher 'PMC'.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 137 - INFO - Loaded patterns for section 'data_availability_sections': [\".//sec[@sec-type='data-availability']\", \".//notes[@notes-type='data-availability']\", \".//notes[@notes-type='data-availability']\"]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_data_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 706 - INFO - Found 3 supplementary data sections\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(key_resources_table)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "Batches:   0%|          | 0/1 [00:00<?, ?it/s]"
      ],
      "application/vnd.jupyter.widget-view+json": {
       "version_major": 2,
       "version_minor": 0,
       "model_id": "0d428997df434587aef24326f111dc17"
      }
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001B[97membeddings_retriever.py - line 51 - INFO - Searching for top-2 passages similar to the query by embeddings.\u001B[0m\n",
      "\u001B[97membeddings_retriever.py - line 33 - INFO - Computing L2 distances using numpy.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 364 - INFO - Processing DAS_content: 3 elements\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 109\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 370 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-dc3ffd9d2c3924353a4470bed2714c11bba1309b01014f46bd13521324f216ee\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 370 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-251193dc-6eec-45ed-80ce-9aeb96c6d659\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755541198,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 469,\n",
      "        \"completion_tokens\": 6,\n",
      "        \"total_tokens\": 475,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 1470\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 710 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-f8d2ae42c35567d83112012a9502e456bcf442c8184211d1dab5e243c14e40ad\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 710 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-7ba269ae-505b-4b9d-ab7e-d143ec4ccc92\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"syn2759792\\\",\\n        \\\"data_repository\\\": \\\"CMC\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.synapse.org//#!Synapse:syn2759792/wiki/69613\\\"\\n    ]\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"syn2759792\\\",\\n        \\\"data_repository\\\": \\\"CMC\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.synapse.org//#!Synapse:syn2759792/wiki/69613\\\"\\n    ]\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755541199,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 837,\n",
      "        \"completion_tokens\": 80,\n",
      "        \"total_tokens\": 917,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn2759792', 'data_repository': 'CMC', 'dataset_webpage': 'https://www.synapse.org//#!Synapse:syn2759792/wiki/69613', 'dataset_id': 'syn2759792'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: syn2759792\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID syn2759792 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: CMC, identifier: syn2759792\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn2759792 for CMC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn2759792 for CMC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn2759792 for CMC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn2759792 for CMC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier syn2759792 for CMC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1051 - INFO - Found id_pattern match for CMC in (?<!\\d)syn\\d{7,12}(?!\\d)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.synapse.org//#!Synapse:syn2759792/wiki/69613, repo synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.synapse.org//#!Synapse:syn2759792/wiki/69613\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Link does not match the pattern https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'syn2759792', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 224\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 399 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-af0b2c827963effbf4fc7121e5686dbd1cda8a4377d58d481120145a2b5eb1f3\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 399 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-f650c715-5dd8-4ece-8fb3-2fedc965dd28\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755541200,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 529,\n",
      "        \"completion_tokens\": 8,\n",
      "        \"total_tokens\": 537,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 377 - INFO - datasets (<class 'list'>): [{'dataset_identifier': 'syn2759792', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'syn2759792', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 385 - INFO - Final ret data availability: 1 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 386 - INFO - Final ret data availability: [{'dataset_identifier': 'syn2759792', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1079 - INFO - Fetching metadata for 1 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: syn2759792\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn2759792\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn2759792 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn2759792 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn2759792 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn2759792 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for syn2759792 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn2759792\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.synapse.org/Synapse:syn2759792\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1156 - INFO - Updated datasets len: 1\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 221 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 2, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'source_section', 'retrieval_pattern', 'access_mode', 'link',\n",
      "       'source_url', 'download_link', 'title', 'content_type', 'id', 'caption',\n",
      "       'description', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 456 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 462 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 539 - INFO - 22th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC9984184)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 347 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC9984184\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 354 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 533 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC9984184&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 383 - INFO - Fallback to HTML data fetcher for https://pmc.ncbi.nlm.nih.gov/articles/PMC9984184.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 185 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 186 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 187 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 189 - INFO - Initializing new selenium driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1b0c09d10>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 375 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 310 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 399 - INFO - Raw HTML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC9984184 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 413 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 439 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 81 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 349 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC9984184, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 444 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC9984184)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 546 - INFO - Extracted 17 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 556 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#clp_a'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for clp_a: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec19'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec19: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec24'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec24: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec25'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec25: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec25'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec25: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec26'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec26: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec26'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec26: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec33'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec33: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec37'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec37: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec37'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec37: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec37'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec37: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec37'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec37: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec37'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec37: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec37'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec37: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#supplementary-material1'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for supplementary-material1: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 567 - INFO - Found 0 <a> elements with href='#sec40'.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 575 - INFO - Extracted context description (len: 0) for sec40: \u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 384 - INFO - Chunking the HTML content for the parsing step.\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 172 - INFO - Extracting data availability elements from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: section.data-availability-statement\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: section.associated-data\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Data Availability')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: h3.pmc_sec_title:-soup-contains('Data Availability')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Associated Data')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 183 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Footnotes')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 221 - INFO - Found 2 data availability elements from HTML.\u001B[0m\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "Batches:   0%|          | 0/2 [00:00<?, ?it/s]"
      ],
      "application/vnd.jupyter.widget-view+json": {
       "version_major": 2,
       "version_minor": 0,
       "model_id": "e43ce5d5b7864613afa3d854d7baa137"
      }
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001B[97membeddings_retriever.py - line 51 - INFO - Searching for top-5 passages similar to the query by embeddings.\u001B[0m\n",
      "\u001B[97membeddings_retriever.py - line 33 - INFO - Computing L2 distances using numpy.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 13960\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 3833 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-e8284514bd4ccfe69e46cdeec81d673ee7a5be65b718e4ba01a4a82605582a6c\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 3833 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-85948ccf-d6b0-41b0-be9a-4ce3eb361efe\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"E-GEOD-44770\\\",\\n        \\\"data_repository\\\": \\\"ArrayExpress\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/arrayexpress/experiments/E-GEOD-44770\\\"\\n    ],\\n    [\\n        \\\"dataset_identifier\\\": \\\"GSE95587\\\",\\n        \\\"data_repository\\\": \\\"GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE95587\\\"\\n    ],\\n    [\\n        \\\"dataset_identifier\\\": \\\"GSE15222\\\",\\n        \\\"data_repository\\\": \\\"GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE15222\\\"\\n    ]\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"E-GEOD-44770\\\",\\n        \\\"data_repository\\\": \\\"ArrayExpress\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/arrayexpress/experiments/E-GEOD-44770\\\"\\n    ],\\n    [\\n        \\\"dataset_identifier\\\": \\\"GSE95587\\\",\\n        \\\"data_repository\\\": \\\"GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE95587\\\"\\n    ],\\n    [\\n        \\\"dataset_identifier\\\": \\\"GSE15222\\\",\\n        \\\"data_repository\\\": \\\"GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE15222\\\"\\n    ]\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755541216,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 4373,\n",
      "        \"completion_tokens\": 221,\n",
      "        \"total_tokens\": 4594,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'E-GEOD-44770', 'data_repository': 'ArrayExpress', 'dataset_webpage': 'https://www.ebi.ac.uk/arrayexpress/experiments/E-GEOD-44770', 'dataset_id': 'E-GEOD-44770'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: E-GEOD-44770\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID E-GEOD-44770 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: ArrayExpress, identifier: E-GEOD-44770\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-GEOD-44770 for ArrayExpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-GEOD-44770 for ArrayExpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-GEOD-44770 for ArrayExpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-GEOD-44770 for ArrayExpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: ArrayExpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ebi.ac.uk/arrayexpress/experiments/E-GEOD-44770, repo arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ebi.ac.uk/biostudies/arrayexpress/studies/E-GEOD-44770\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.ebi.ac.uk/biostudies/ArrayExpress/studies/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 961 - INFO - Link matches the pattern https://www.ebi.ac.uk/biostudies/ArrayExpress/studies/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'E-GEOD-44770', 'data_repository': 'arrayexpress', 'dataset_webpage': 'https://www.ebi.ac.uk/biostudies/arrayexpress/studies/E-GEOD-44770'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE95587', 'data_repository': 'GEO', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE95587', 'dataset_id': 'GSE95587'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: GSE95587\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID GSE95587 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: GEO, identifier: GSE95587\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE95587 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE95587 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE95587, repo geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE95587\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 961 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'GSE95587', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE95587'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE15222', 'data_repository': 'GEO', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE15222', 'dataset_id': 'GSE15222'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: GSE15222\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID GSE15222 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: GEO, identifier: GSE15222\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE15222 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE15222 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE15222, repo geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE15222\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 961 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'GSE15222', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE15222'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1079 - INFO - Fetching metadata for 3 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: E-GEOD-44770\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: arrayexpress, identifier: E-GEOD-44770\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-GEOD-44770 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-GEOD-44770 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-GEOD-44770 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-GEOD-44770 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for E-GEOD-44770 in arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 1 with repo: arrayexpress and accession_id: E-GEOD-44770\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ebi.ac.uk/biostudies/ArrayExpress/studies/E-GEOD-44770\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: GSE95587\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: geo, identifier: GSE95587\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE95587 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE95587 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for GSE95587 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 2 with repo: geo and accession_id: GSE95587\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE95587\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 2: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: GSE15222\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: geo, identifier: GSE15222\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE15222 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE15222 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for GSE15222 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 3 with repo: geo and accession_id: GSE15222\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE15222\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 3: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1156 - INFO - Updated datasets len: 3\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 411 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 20, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 230 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 240 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 639 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 450 - INFO - Parsed data extraction completed. Elements collected: 5\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 456 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 466 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 539 - INFO - 23th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC7347884)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 347 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC7347884\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 354 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 533 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC7347884&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <notes xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 395 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC7347884.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 399 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC7347884 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 413 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 417 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC7347884, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 5 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b6062f40>, <Element supplementary-material at 0x1b6d35fc0>, <Element supplementary-material at 0x1b60820c0>, <Element supplementary-material at 0x1b608d600>, <Element supplementary-material at 0x1b608e800>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b6062f40>, <Element supplementary-material at 0x1b6d35fc0>, <Element supplementary-material at 0x1b60820c0>, <Element supplementary-material at 0x1b608d600>, <Element supplementary-material at 0x1b608e800>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM1: As gene expression analyses do not allow to accurately discriminate mesenchymal cancer cells from normal stromal cells, we used the allele-specific copy number analysis of tumors (ASCAT) copy number-based tumor purity estimation method to assess tumor cell fraction (Supplementary Fig. 1a [this file] ). To avoid any substantial bias due to non-tumor cell contamination, a stringent purity threshold was determined by applying Wilcoxon test, for which the degree of contamination of claudin-low tumors was not statistically different from that of non-claudin-low tumors, thus enabling a comparable distribution of tumors (Supplementary Fig. 1b [this file] ). Demonstrating an unexpected degree of diversity, only 35.6% of these tumors were classified as TNBCs (Supplementary Fig. 1d [this file] ). This result was not due to the purity selection, as the percentage of TNBCs within all claudin-low tumors was similar before applying the purity threshold (Supplementary Fig. 1c [this file] ). \n",
      "Only 17.8% were found in cluster 10, whereas 48.8% and 17.8% were found in integrative clusters 4 and 3, respectively, regardless of tumor purity when compared with non-claudin-low tumors (Supplementary Fig. 2 [this file] ). After having previously removed germline copy number variations (CNVs), we analyzed B allele frequency (BAF) and log R ratio (LRR) profiles of single-nucleotide polymorphisms (SNPs) localized in genomic regions of deletion (LRR segment mean cutoff value < −0.4) (Fig. 1b [rid=Fig1] and Supplementary Fig. 3a, c [this file] ). Consistent with these data, the analysis of somatic mutations highlighted the presence of several point mutations with a variant allele frequency higher than 0.4 (Fig. 1c [rid=Fig1] and Supplementary Fig. 3b [this file] ). \n",
      "Of note, although these analyses were performed on tumors selected for their high tumor purity, similar data were obtained when studying the whole cohort of claudin-low tumors (Supplementary Fig. 4 [this file] ), further strengthening the characterization of the three claudin-low subgroups. \n",
      "Of note, global gene expression signatures demonstrated that the CL2 and CL3 subgroups showed lower luminal- and basal-related signature scores, respectively, compared with luminal- and basal-like breast tumors, while exhibiting a significant enrichment in invasiveness-related signatures (Supplementary Fig. 5a, b [this file] ). \n",
      "A claudin-low gene list of 137 genes discriminating the 3 claudin-low subgroups was thus created (Supplementary Fig. 6a, b [this file] ). This expression-based classifier showed an accuracy of 91% when compared with the FGA-based classification using Gaussian finite mixture models (Supplementary Fig. 5c, d [this file] ). Validating the accuracy of our classifier, CL1, CL2, and CL3 tumors and cell lines showed low, moderate, and elevated FGA levels, respectively (Supplementary Fig. 6e, f [this file] ). Of note, we performed a pathway analysis of CL1 discriminant genes that revealed an enrichment in stem cell and pediatric cancer markers (Supplementary Fig. 6g [this file] ). \n",
      "Differentially methylated gene analysis was next performed in claudin-low subgroups followed by pathway-enrichment analysis, using over 15,000 pathways available in MSigDB (HALLMARKS, C2, and C5 gene sets) (Supplementary Fig. 7 [this file] ). The CL1 subgroup had the highest score for stemness- ( , ) and EMT- (mesenchymal markers: , , ; epithelial markers: , ) related genes, whereas CL2 and CL3 displayed an intermediate score compared with CL1 tumors and with their relative luminal/basal counterparts (Supplementary Fig. 8 [this file] ). \n",
      "CL1 tumors mainly display hypomethylated genes that are related to a stemness phenotype, CL2 tumors are overall hypermethylated for genes associated with basal-like features, and CL3 tumors show more heterogeneous methylation profiles with both hypermethylated genes (linked to luminal phenotype) and hypomethylated genes (relative to basal-like characteristics) (see Supplementary Fig. 5 [this file] for methodology). \n",
      "Additional gene expression analyses performed on METABRIC and TCGA cohorts using two different cellular signature deconvolution algorithms confirmed that these tumors were highly infiltrated by immune cells (Supplementary Fig. 9a–d [this file] ). Nevertheless, unsupervised clustering of microenvironment signatures (immune and non-immune cell subsets) did not allow to discriminate the different claudin-low subgroups (Supplementary Fig. 9e–h [this file] ). \n",
      "Supporting our previous results, CL2 and CL3 showed, among the three claudin-low subgroups, the highest score for luminal- (estrogen-related response) and basal-related (proliferation and DDR) signatures, respectively (Supplementary Fig. 10 [this file] ). Furthermore, both CL2 and CL3 showed an activation of the EMT process (Fig. 6a, b, g [rid=Fig6] ), coherent with the downregulation of hormone (in CL2) and proliferation/DDR (in CL3) signatures, compared with luminal and basal tumors, respectively (Supplementary Fig. 10 [this file] and Fig. 6g [rid=Fig6] ). Moreover, CL1 displayed a low proliferation activity and a low DDR (Supplementary Fig. 10a, b, e, f [this file] ), as well as a strong TP53 signaling pathway signature associated with a low frequency of mutations (Fig. 6c, d, g [rid=Fig6] ). \n",
      "When stratified by CL subgroups, variations in survival outcome were found, consistent with their presumed origin (Supplementary Fig. 11 [this file] ). \n",
      "These traits include (i) a high expression of the ZEB1 EMT-inducing transcription factor and of its target, the methionine sulfoxide reductase MSRB3; (ii) a frequent activation of the RAS-MAPK signaling pathway; (iii) a low activation of the DDR; and (iv) a low frequency of mutations (Supplementary Figs. 8 [this file] and 10 [this file] , and Fig. 6 [rid=Fig6] ). \n",
      "The use of a high-purity threshold for tumor selection allowed to circumvent the contamination bias related to the tumor-related centroid (Supplementary Fig. 1 [this file] ). \n",
      "Finally, the nearest shrunken centroid classifier encompassed 137 genes whose expression discriminate the three FGA-related claudin-low subgroups in METABRIC cohort (Supplementary Fig. 6 [this file] ). \n",
      "As previously described for genome-wide expression–methylation quantitative trait loci analysis , genes for which methylation -values were negatively correlated with expression values (Spearman’s < 0 and FDR -value < 0.05) were selected for differential methylation analysis (Supplementary Fig. 7 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM2: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM3: \u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM4: Gene expression signatures for MaSCs, LPs, and mL cells were generated based on the transcription profiles of subpopulations of normal human mammary epithelial cells isolated from reduction mammoplasty tissues, as shown in Morel et al. (Supplementary Data 1 [this file] ). \n",
      "The top 500 false discovery rate (FDR) -value ranked genes differentially expressed between one subpopulation (MaSC or LP, or mL cells) vs. the two others (FDR -value < 0.05) were used as transcriptomic signature for each of the three cell subpopulations (Supplementary Data 1 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref MOESM5: Further information on research design is available in the Nature Research Reporting Summary [this file] linked to this article. \n",
      "\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 123 - INFO - Loading patterns for section 'data_availability_sections' for publisher 'PMC'.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 131 - INFO - Section pattern keys for publisher PMC: dict_keys(['data_availability_sections', 'supplementary_material_sections', 'supplementary_data_sections', 'key_resources_table', 'methods_sections'])\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 133 - INFO - Section name: data_availability_sections\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 135 - INFO - Found section 'data_availability_sections' in patterns for publisher 'PMC'.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 137 - INFO - Loaded patterns for section 'data_availability_sections': [\".//sec[@sec-type='data-availability']\", \".//notes[@notes-type='data-availability']\", \".//notes[@notes-type='data-availability']\"]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_data_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 706 - INFO - Found 6 supplementary data sections\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(key_resources_table)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "Batches:   0%|          | 0/1 [00:00<?, ?it/s]"
      ],
      "application/vnd.jupyter.widget-view+json": {
       "version_major": 2,
       "version_minor": 0,
       "model_id": "685d8dfeebdd4f929bdd3995c54e5020"
      }
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001B[97membeddings_retriever.py - line 51 - INFO - Searching for top-2 passages similar to the query by embeddings.\u001B[0m\n",
      "\u001B[97membeddings_retriever.py - line 33 - INFO - Computing L2 distances using numpy.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 364 - INFO - Processing DAS_content: 3 elements\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 2940\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 1078 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-7433dbbc8eaa34526c29e8d5dc3f78d8907093a881934a3f845b5620d5cb9372\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 1078 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-a78bdc20-9e69-4b2d-b383-a295cbcf2626\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE56031\\\",\\n        \\\"data_repository\\\": \\\"GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE56031\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"E-MTAB-4145\\\",\\n        \\\"data_repository\\\": \\\"ArrayExpress\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-4145\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"EGAS00000000083\\\",\\n        \\\"data_repository\\\": \\\"EMBL\\u2013EBI\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/ega/studies/EGAS00000000083\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"syn1757053\\\",\\n        \\\"data_repository\\\": \\\"Synapse\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.synapse.org/#!Synapse:syn1757053\\\"\\n    }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE56031\\\",\\n        \\\"data_repository\\\": \\\"GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE56031\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"E-MTAB-4145\\\",\\n        \\\"data_repository\\\": \\\"ArrayExpress\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-4145\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"EGAS00000000083\\\",\\n        \\\"data_repository\\\": \\\"EMBL\\u2013EBI\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/ega/studies/EGAS00000000083\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"syn1757053\\\",\\n        \\\"data_repository\\\": \\\"Synapse\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.synapse.org/#!Synapse:syn1757053\\\"\\n    }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755541228,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 1323,\n",
      "        \"completion_tokens\": 292,\n",
      "        \"total_tokens\": 1615,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'GSE56031', 'data_repository': 'GEO', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE56031', 'dataset_id': 'GSE56031'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: GSE56031\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID GSE56031 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: GEO, identifier: GSE56031\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE56031 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE56031 for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE56031, repo geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE56031\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 961 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'GSE56031', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE56031'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'E-MTAB-4145', 'data_repository': 'ArrayExpress', 'dataset_webpage': 'https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-4145', 'dataset_id': 'E-MTAB-4145'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: E-MTAB-4145\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID E-MTAB-4145 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: ArrayExpress, identifier: E-MTAB-4145\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-MTAB-4145 for ArrayExpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-MTAB-4145 for ArrayExpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-MTAB-4145 for ArrayExpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-MTAB-4145 for ArrayExpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: ArrayExpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-4145, repo arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ebi.ac.uk/biostudies/arrayexpress/studies/E-MTAB-4145\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.ebi.ac.uk/biostudies/ArrayExpress/studies/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 961 - INFO - Link matches the pattern https://www.ebi.ac.uk/biostudies/ArrayExpress/studies/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'E-MTAB-4145', 'data_repository': 'arrayexpress', 'dataset_webpage': 'https://www.ebi.ac.uk/biostudies/arrayexpress/studies/E-MTAB-4145'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'EGAS00000000083', 'data_repository': 'EMBL–EBI', 'dataset_webpage': 'https://www.ebi.ac.uk/ega/studies/EGAS00000000083', 'dataset_id': 'EGAS00000000083'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: EGAS00000000083\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID EGAS00000000083 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: EMBL–EBI, identifier: EGAS00000000083\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier EGAS00000000083 for EMBL–EBI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier EGAS00000000083 for EMBL–EBI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier EGAS00000000083 for EMBL–EBI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier EGAS00000000083 for EMBL–EBI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier EGAS00000000083 for EMBL–EBI\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: https://www.ebi.ac.uk/ega/studies/EGAS00000000083\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ebi.ac.uk/ega/studies/EGAS00000000083, repo www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://ega-archive.org/studies/EGAS00000000083\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 967 - INFO - No dataset_webpage_url_ptr found for www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'EGAS00000000083', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'https://ega-archive.org/studies/EGAS00000000083'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'syn1757053', 'data_repository': 'Synapse', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn1757053', 'dataset_id': 'syn1757053'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: syn1757053\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID syn1757053 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: Synapse, identifier: syn1757053\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn1757053 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn1757053 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn1757053 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn1757053 for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1037 - INFO - Found repo_name match for Synapse\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.synapse.org/#!Synapse:syn1757053, repo synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.synapse.org/#!Synapse:syn1757053\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 964 - INFO - Link does not match the pattern https://www.synapse.org/Synapse:\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'syn1757053', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 4814\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 1546 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-fa8db10e67db6ea46f1156bc50b554ca52a4d0c65eb2d5456dc946ccbefa9287\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 1546 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-d80c1f18-e71c-4ae1-8b33-a06b123ac264\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"EGAS00000000083\\\",\\n        \\\"data_repository\\\": \\\"EGA\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/ega/studies/EGAS00000000083\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"TCGA BRCA\\\",\\n        \\\"data_repository\\\": \\\"GDC data portal\\\",\\n        \\\"dataset_webpage\\\": \\\"https://portal.gdc.cancer.gov/\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"ASCAT\\\",\\n        \\\"data_repository\\\": \\\"COSMIC\\\",\\n        \\\"dataset_webpage\\\": \\\"https://cancer.sanger.ac.uk/cosmic/\\\"\\n    }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"EGAS00000000083\\\",\\n        \\\"data_repository\\\": \\\"EGA\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/ega/studies/EGAS00000000083\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"TCGA BRCA\\\",\\n        \\\"data_repository\\\": \\\"GDC data portal\\\",\\n        \\\"dataset_webpage\\\": \\\"https://portal.gdc.cancer.gov/\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"ASCAT\\\",\\n        \\\"data_repository\\\": \\\"COSMIC\\\",\\n        \\\"dataset_webpage\\\": \\\"https://cancer.sanger.ac.uk/cosmic/\\\"\\n    }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755541231,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 2035,\n",
      "        \"completion_tokens\": 187,\n",
      "        \"total_tokens\": 2222,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'EGAS00000000083', 'data_repository': 'EGA', 'dataset_webpage': 'https://www.ebi.ac.uk/ega/studies/EGAS00000000083', 'dataset_id': 'EGAS00000000083'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: EGAS00000000083\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID EGAS00000000083 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: EGA, identifier: EGAS00000000083\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier EGAS00000000083 for EGA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier EGAS00000000083 for EGA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier EGAS00000000083 for EGA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier EGAS00000000083 for EGA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier EGAS00000000083 for EGA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: https://www.ebi.ac.uk/ega/studies/EGAS00000000083\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ebi.ac.uk/ega/studies/EGAS00000000083, repo www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://ega-archive.org/studies/EGAS00000000083\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 967 - INFO - No dataset_webpage_url_ptr found for www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'EGAS00000000083', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'https://ega-archive.org/studies/EGAS00000000083'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'TCGA BRCA', 'data_repository': 'GDC data portal', 'dataset_webpage': 'https://portal.gdc.cancer.gov/', 'dataset_id': 'TCGA BRCA'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: TCGA BRCA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 940 - INFO - Accession ID TCGA BRCA is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: GDC data portal, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier n/a for GDC data portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier n/a for GDC data portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier n/a for GDC data portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier n/a for GDC data portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier n/a for GDC data portal\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: https://portal.gdc.cancer.gov/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://portal.gdc.cancer.gov/, repo portal.gdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://portal.gdc.cancer.gov/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 972 - INFO - Repository portal.gdc.cancer.gov not found in ontology\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'portal.gdc.cancer.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'ASCAT', 'data_repository': 'COSMIC', 'dataset_webpage': 'https://cancer.sanger.ac.uk/cosmic/', 'dataset_id': 'ASCAT'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: ASCAT\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID ASCAT is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: COSMIC, identifier: ASCAT\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier ASCAT for COSMIC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier ASCAT for COSMIC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier ASCAT for COSMIC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier ASCAT for COSMIC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier ASCAT for COSMIC\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: https://cancer.sanger.ac.uk/cosmic/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://cancer.sanger.ac.uk/cosmic/, repo cancer.sanger.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://cancer.sanger.ac.uk/cosmic/login\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 972 - INFO - Repository cancer.sanger.ac.uk not found in ontology\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'ASCAT', 'data_repository': 'cancer.sanger.ac.uk', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 470\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 460 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-d59ce63304362e792b0a578bce304ce5ac442a2ba6b1c5d56991b5d642bfb071\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 460 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-82e113ac-e33d-46de-ae68-23ff936db7df\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"[\\n    {\\n        \\\"dataset_identifier\\\": \\\"IC50 data for MEKi\\\",\\n        \\\"data_repository\\\": \\\"Genomics of Drug Sensitivity in Cancer (GDSC)\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.cancerrxgene.org/\\\"\\n    }\\n]\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"[\\n    {\\n        \\\"dataset_identifier\\\": \\\"IC50 data for MEKi\\\",\\n        \\\"data_repository\\\": \\\"Genomics of Drug Sensitivity in Cancer (GDSC)\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.cancerrxgene.org/\\\"\\n    }\\n]\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755541234,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 623,\n",
      "        \"completion_tokens\": 64,\n",
      "        \"total_tokens\": 687,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'IC50 data for MEKi', 'data_repository': 'Genomics of Drug Sensitivity in Cancer (GDSC)', 'dataset_webpage': 'https://www.cancerrxgene.org/', 'dataset_id': 'IC50 data for MEKi'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: IC50 data for MEKi\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 940 - INFO - Accession ID IC50 data for MEKi is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: Genomics of Drug Sensitivity in Cancer (GDSC), identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier n/a for Genomics of Drug Sensitivity in Cancer  GDSC \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier n/a for Genomics of Drug Sensitivity in Cancer  GDSC \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier n/a for Genomics of Drug Sensitivity in Cancer  GDSC \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier n/a for Genomics of Drug Sensitivity in Cancer  GDSC \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier n/a for Genomics of Drug Sensitivity in Cancer  GDSC \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: https://www.cancerrxgene.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.cancerrxgene.org/, repo www.cancerrxgene.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.cancerrxgene.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 972 - INFO - Repository www.cancerrxgene.org not found in ontology\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'www.cancerrxgene.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 377 - INFO - datasets (<class 'list'>): [{'dataset_identifier': 'GSE56031', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE56031'}, {'dataset_identifier': 'E-MTAB-4145', 'data_repository': 'arrayexpress', 'dataset_webpage': 'https://www.ebi.ac.uk/biostudies/arrayexpress/studies/E-MTAB-4145'}, {'dataset_identifier': 'EGAS00000000083', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'https://ega-archive.org/studies/EGAS00000000083'}, {'dataset_identifier': 'syn1757053', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'EGAS00000000083', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'https://ega-archive.org/studies/EGAS00000000083'}, {'dataset_identifier': 'n/a', 'data_repository': 'portal.gdc.cancer.gov', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'ASCAT', 'data_repository': 'cancer.sanger.ac.uk', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'www.cancerrxgene.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'GSE56031', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE56031'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'E-MTAB-4145', 'data_repository': 'arrayexpress', 'dataset_webpage': 'https://www.ebi.ac.uk/biostudies/arrayexpress/studies/E-MTAB-4145'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'EGAS00000000083', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'https://ega-archive.org/studies/EGAS00000000083'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'syn1757053', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'EGAS00000000083', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'https://ega-archive.org/studies/EGAS00000000083'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'n/a', 'data_repository': 'portal.gdc.cancer.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'ASCAT', 'data_repository': 'cancer.sanger.ac.uk', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'n/a', 'data_repository': 'www.cancerrxgene.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 385 - INFO - Final ret data availability: 8 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 386 - INFO - Final ret data availability: [{'dataset_identifier': 'GSE56031', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE56031', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'E-MTAB-4145', 'data_repository': 'arrayexpress', 'dataset_webpage': 'https://www.ebi.ac.uk/biostudies/arrayexpress/studies/E-MTAB-4145', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'EGAS00000000083', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'https://ega-archive.org/studies/EGAS00000000083', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'syn1757053', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'EGAS00000000083', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'https://ega-archive.org/studies/EGAS00000000083', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'n/a', 'data_repository': 'portal.gdc.cancer.gov', 'dataset_webpage': 'n/a', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'ASCAT', 'data_repository': 'cancer.sanger.ac.uk', 'dataset_webpage': 'n/a', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'n/a', 'data_repository': 'www.cancerrxgene.org', 'dataset_webpage': 'n/a', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1079 - INFO - Fetching metadata for 8 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: GSE56031\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: geo, identifier: GSE56031\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE56031 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier GSE56031 for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for GSE56031 in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE56031\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE56031\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: E-MTAB-4145\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: arrayexpress, identifier: E-MTAB-4145\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier E-MTAB-4145 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier E-MTAB-4145 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier E-MTAB-4145 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier E-MTAB-4145 for arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for E-MTAB-4145 in arrayexpress\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 2 with repo: arrayexpress and accession_id: E-MTAB-4145\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ebi.ac.uk/biostudies/ArrayExpress/studies/E-MTAB-4145\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 2: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: EGAS00000000083\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: www.ebi.ac.uk, identifier: EGAS00000000083\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for EGAS00000000083 in www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 3 with repo: www.ebi.ac.uk and accession_id: EGAS00000000083\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ebi.ac.uk/pride/archive/projects/EGAS00000000083\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 3: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: syn1757053\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn1757053\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn1757053 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier syn1757053 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier syn1757053 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn1757053 for synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for syn1757053 in synapse.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 4 with repo: synapse.org and accession_id: syn1757053\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.synapse.org/Synapse:syn1757053\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 4: Complex download or application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: EGAS00000000083\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: www.ebi.ac.uk, identifier: EGAS00000000083\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for EGAS00000000083 in www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 5 with repo: www.ebi.ac.uk and accession_id: EGAS00000000083\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ebi.ac.uk/pride/archive/projects/EGAS00000000083\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 5: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1095 - INFO - Skipping dataset 6: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: ASCAT\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: cancer.sanger.ac.uk, identifier: ASCAT\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier ASCAT for cancer.sanger.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier ASCAT for cancer.sanger.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier ASCAT for cancer.sanger.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier ASCAT for cancer.sanger.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier ASCAT for cancer.sanger.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: cancer.sanger.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 900 - WARNING - URL cancer.sanger.ac.uk may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for ASCAT in cancer.sanger.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 7 with repo: cancer.sanger.ac.uk and accession_id: ASCAT\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1152 - WARNING - Repository cancer.sanger.ac.uk unknown in Ontology. Skipping dataset page 7.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1095 - INFO - Skipping dataset 8: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1156 - INFO - Updated datasets len: 8\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 221 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 13, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'source_section', 'retrieval_pattern', 'access_mode', 'link',\n",
      "       'source_url', 'download_link', 'title', 'content_type', 'id', 'caption',\n",
      "       'description', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 456 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 462 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 539 - INFO - 24th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452)\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 347 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 166 - INFO - raw_HTML_data_filepath: None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 173 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 354 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 533 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC10802452&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 461 - WARNING - No links found in the section.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\"...\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 395 - INFO - XML data is complete for https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 399 - INFO - Raw XML data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452 is ready for parsing.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 413 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 417 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 176 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 441 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 452 - INFO - Found 5 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1b75e2a00>, <Element supplementary-material at 0x1b042f600>, <Element supplementary-material at 0x1b659f7c0>, <Element supplementary-material at 0x1b659f5c0>, <Element supplementary-material at 0x1b659f6c0>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 460 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1b75e2a00>, <Element supplementary-material at 0x1b042f600>, <Element supplementary-material at 0x1b659f7c0>, <Element supplementary-material at 0x1b659f5c0>, <Element supplementary-material at 0x1b659f6c0>]}\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 546 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD1: This cohort (n=104) comprised neurotypical (healthy) controls (HC, n = 38), and individuals with SCZ (ISCZ, n=38), BD (n = 20) and MDD (n = 8) ( Supplementary Table 1 [this file] ) with a mean age of onset in the late 20s adolescence (mean 28.5/27.1/29.1 SCZ/BD/MDD). \n",
      "From the latter, we also included a cohort of Amish individuals suffering from BD/MDD and respective controls, see Supplementary Table 1 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD2: Importantly, 55.4% of the identified eQTLs overlapped with eQTLs identified in adult human tissue from the prefrontal cortex (PFC) , supporting their relevance ( Fig. 1f [rid=F1] and Supplementary Table 2 [this file] ). Nevertheless, 21.7% of the identified open chromatin peaks associated with caQTLs overlapped with eQTLs detected in bulk PFC. ( Fig. 1g [rid=F1] and Supplementary Table 2 [this file] ). \n",
      "Interestingly, the overlap of QTLs across molecular features in iNs was limited, with 24.9% of eQTL overlapping with caQTLs ( Fig. 1l [rid=F1] and Supplementary Table 2 [this file] ). Moreover, 28 genes, 78 peaks and no microRNA with e/ca/miQTLs overlapped with GWAS risk loci for SCZ ( Fig. 1f [rid=F1] - h [rid=F1] and Supplementary Table 2 [this file] ). \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD3: Following a detailed power evaluation ( Extended Data Fig. 3b [rid=F9] - d [rid=F9] ), we performed differential expression analysis and identified 467 transcripts (365 protein coding and 102 non-coding transcripts) (FDR≤0.01, ∣log FC∣≥0.4; Fig. 2a [rid=F2] and Extended Data Fig. 3e [rid=F9] , and Supplementary Table 3 [this file] ), henceforth referred to as DEGs as well as 67 micro RNAs (FDR≤0.05; Fig. 2b [rid=F2] and Supplementary Table 3 [this file] ). \n",
      "This analysis revealed widespread differences in APA, identifying 628 differential APA (dAPA) events ( Fig. 3c [rid=F3] and Supplementary Table 3 [this file] ) that resulted from an increased usage of either the distal or the proximal PAS to similar degree (14.29% of all genes with more than 1 polyadenylation site). \n",
      "Consistent with the observations in iNs, this analysis confirmed widespread dAPA in postmortem PFC from ISCZ ( Fig. 3e [rid=F3] and Supplementary Table 3 [this file] ) as well as substantial overlap with dAPA in iNs ( Fig. 3f [rid=F3] ). \n",
      "Strikingly, genes subject to dAPA in ISCZ were strongly enriched for synapse related processes such as glutamatergic neurotransmission, synaptic membrane components, synaptic plasticity or vesicle biology in both iNs and human PFC ( Fig. 4a [rid=F4] , b [rid=F4] , and Supplementary Table 3 [this file] ). \n",
      "RBP motif analysis in the UTRs of transcripts subject to dAPA identified numerous expressed RBPs with critical roles for neuronal biology such as QKI, ELAVL and RBM proteins ( Fig. 5c [rid=F5] and Supplementary Table 3 [this file] ). \n",
      "Similarly, we also corrected the R values for gender, batch and TechnicalPC1 using beta regression and used the residuals to compute the average difference in R values between the groups (ΔAPA, Supplementary Table 3 [this file] ). \n",
      "Resulting enrichment scores and p-values for the set of transcripts with increased usage of the distal PAS are depicted in Fig. 5c [rid=F5] with p-values corrected for multiple testing using the q-value method and listed in Supplementary Table 3 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD4: MAP2, RTF2 or Distal/Proximal UTR, Supplementary Table 4 [this file] ) were used to calculate target specific relative expression as 2^(-(CT -CT )) giving rise to normalized transcript levels. \n",
      "Statistical testing of NRIs ( Supplementary Table 4 [this file] ) was conducted using by linear mixed models as implemented in the lme4 R package modeling diagnosis or treatment as fixed effect and batch as well as donor (for PSD95 analysis) as random effect. \n",
      "All well level density values used can be found in Supplementary Table 4 [this file] . \n",
      "\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 562 - INFO - Extracted context_descr for xref SD5: \u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 123 - INFO - Loading patterns for section 'data_availability_sections' for publisher 'PMC'.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 131 - INFO - Section pattern keys for publisher PMC: dict_keys(['data_availability_sections', 'supplementary_material_sections', 'supplementary_data_sections', 'key_resources_table', 'methods_sections'])\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 133 - INFO - Section name: data_availability_sections\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 135 - INFO - Found section 'data_availability_sections' in patterns for publisher 'PMC'.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 137 - INFO - Loaded patterns for section 'data_availability_sections': [\".//sec[@sec-type='data-availability']\", \".//notes[@notes-type='data-availability']\", \".//notes[@notes-type='data-availability']\"]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_data_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 706 - INFO - Found 7 supplementary data sections\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 262 - INFO - Function_call: load_patterns_for_tgt_section(key_resources_table)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 263 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "Batches:   0%|          | 0/3 [00:00<?, ?it/s]"
      ],
      "application/vnd.jupyter.widget-view+json": {
       "version_major": 2,
       "version_minor": 0,
       "model_id": "afd0ef6b58594e20b8526ad7cb272137"
      }
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001B[97membeddings_retriever.py - line 51 - INFO - Searching for top-2 passages similar to the query by embeddings.\u001B[0m\n",
      "\u001B[97membeddings_retriever.py - line 33 - INFO - Computing L2 distances using numpy.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 364 - INFO - Processing DAS_content: 3 elements\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 186\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 389 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-774f43211cb21e6068b9dc6bf5fca4b91569bbb67b6b32df8f93b5a7f7d35a93\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 389 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-a1e7bb21-c990-42f5-8fbe-70898a8b22ee\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"TBA\\\",\\n        \\\"data_repository\\\": \\\"GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=TBA\\\"\\n    ],\\n    [\\n        \\\"dataset_identifier\\\": \\\"TBA\\\",\\n        \\\"data_repository\\\": \\\"European genome phenome archive\\\",\\n        \\\"dataset_webpage\\\": \\\"https://ega-archive.org/studies/TBA\\\"\\n    ]\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"TBA\\\",\\n        \\\"data_repository\\\": \\\"GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=TBA\\\"\\n    ],\\n    [\\n        \\\"dataset_identifier\\\": \\\"TBA\\\",\\n        \\\"data_repository\\\": \\\"European genome phenome archive\\\",\\n        \\\"dataset_webpage\\\": \\\"https://ega-archive.org/studies/TBA\\\"\\n    ]\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755541249,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 480,\n",
      "        \"completion_tokens\": 119,\n",
      "        \"total_tokens\": 599,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'TBA', 'data_repository': 'GEO', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=TBA', 'dataset_id': 'TBA'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: TBA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID TBA is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: GEO, identifier: TBA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier TBA for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier TBA for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=TBA, repo geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=TBA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 961 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'TBA', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=TBA'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 280\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 413 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-3007dea07792a6dbf0041a8e384e64046ddf06ad716d3855d6cb1969d79680ef\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 413 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-db79c396-b740-4c1d-a594-90998ce0bf52\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"[TBA]\\\",\\n        \\\"data_repository\\\": \\\"GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=[TBA]\\\"\\n    ],\\n    [\\n        \\\"dataset_identifier\\\": \\\"[TBA]\\\",\\n        \\\"data_repository\\\": \\\"European genome phenome archive\\\",\\n        \\\"dataset_webpage\\\": \\\"https://ega-archive.org/studies/[TBA]\\\"\\n    ]\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"[TBA]\\\",\\n        \\\"data_repository\\\": \\\"GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=[TBA]\\\"\\n    ],\\n    [\\n        \\\"dataset_identifier\\\": \\\"[TBA]\\\",\\n        \\\"data_repository\\\": \\\"European genome phenome archive\\\",\\n        \\\"dataset_webpage\\\": \\\"https://ega-archive.org/studies/[TBA]\\\"\\n    ]\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755541250,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 533,\n",
      "        \"completion_tokens\": 119,\n",
      "        \"total_tokens\": 652,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': '{TBA}', 'data_repository': 'GEO', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc={TBA}', 'dataset_id': '{TBA}'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: {TBA}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID {TBA} is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: GEO, identifier: {TBA}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier {TBA} for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier {TBA} for GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: GEO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc={TBA}, repo geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=%7BTBA%7D\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Pattern: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 961 - INFO - Link matches the pattern https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': '{TBA}', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=%7BTBA%7D'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 408 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 33 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 416 - INFO - Content length: 397\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 427 - INFO - Prompt messages total length: 442 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 433 - INFO - Prompt ID: gemini-2.0-flash-0-2a4d3ebd18b4fb8efb6f5c1976287b23c7f6ae828c366feb408f6e86c76f1a51\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1197 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 452 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages length: 442 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 517 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-11797d63-82f3-4694-bcc7-82fc171001cc\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"SYNGO\\\",\\n        \\\"data_repository\\\": \\\"SYNGO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.syngoportal.org/\\\"\\n    ]\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"SYNGO\\\",\\n        \\\"data_repository\\\": \\\"SYNGO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.syngoportal.org/\\\"\\n    ]\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1755541251,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 581,\n",
      "        \"completion_tokens\": 55,\n",
      "        \"total_tokens\": 636,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 584 - INFO - Processing dataset: {'dataset_identifier': 'SYNGO', 'data_repository': 'SYNGO', 'dataset_webpage': 'https://www.syngoportal.org/', 'dataset_id': 'SYNGO'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 599 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 935 - INFO - Validating accession ID: SYNGO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 946 - INFO - Accession ID SYNGO is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: SYNGO, identifier: SYNGO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier SYNGO for SYNGO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier SYNGO for SYNGO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier SYNGO for SYNGO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier SYNGO for SYNGO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier SYNGO for SYNGO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: https://www.syngoportal.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 953 - INFO - Validating Dataset Page: https://www.syngoportal.org/, repo www.syngoportal.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 978 - INFO - Resolved URL: https://www.syngoportal.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 972 - INFO - Repository www.syngoportal.org not found in ontology\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 645 - INFO - Extracted dataset: {'dataset_identifier': 'SYNGO', 'data_repository': 'www.syngoportal.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 377 - INFO - datasets (<class 'list'>): [{'dataset_identifier': 'TBA', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=TBA'}, {'dataset_identifier': '{TBA}', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=%7BTBA%7D'}, {'dataset_identifier': 'SYNGO', 'data_repository': 'www.syngoportal.org', 'dataset_webpage': 'n/a'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'TBA', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=TBA'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': '{TBA}', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=%7BTBA%7D'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 379 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'SYNGO', 'data_repository': 'www.syngoportal.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 381 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 385 - INFO - Final ret data availability: 3 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 386 - INFO - Final ret data availability: [{'dataset_identifier': 'TBA', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=TBA', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': '{TBA}', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=%7BTBA%7D', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'SYNGO', 'data_repository': 'www.syngoportal.org', 'dataset_webpage': 'n/a', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1079 - INFO - Fetching metadata for 3 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: TBA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: geo, identifier: TBA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier TBA for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier TBA for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for TBA in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 994 - WARNING - Identifier TBA does not match pattern for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 1 with repo: geo and accession_id: TBA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=TBA\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: {TBA}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: geo, identifier: {TBA}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier {TBA} for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier {TBA} for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1030 - INFO - Exact - case insensitive - match found for repo: geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for {TBA} in geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 994 - WARNING - Identifier {TBA} does not match pattern for geo\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 2 with repo: geo and accession_id: {TBA}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1138 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc={TBA}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1145 - INFO - Adding access mode for dataset 2: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1087 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1098 - INFO - Raw accession ID: SYNGO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1007 - INFO - Resolving data repository for candidate: www.syngoportal.org, identifier: SYNGO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern PDC[0-9]+ match with identifier SYNGO for www.syngoportal.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern 10\\.\\d{4,9}/ match with identifier SYNGO for www.syngoportal.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern \\bGSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\\b match with identifier SYNGO for www.syngoportal.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier SYNGO for www.syngoportal.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1049 - INFO - Checking id_pattern (?<!\\d)syn\\d{7,12}(?!\\d) match with identifier SYNGO for www.syngoportal.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 888 - INFO - Extracting repo domain from URL: www.syngoportal.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 900 - WARNING - URL www.syngoportal.org may be a domain already.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1058 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 989 - INFO - Resolving accession ID for SYNGO in www.syngoportal.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1112 - INFO - Processing dataset 3 with repo: www.syngoportal.org and accession_id: SYNGO\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1152 - WARNING - Repository www.syngoportal.org unknown in Ontology. Skipping dataset page 3.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1156 - INFO - Updated datasets len: 3\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 221 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 8, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'source_section', 'retrieval_pattern', 'access_mode', 'link',\n",
      "       'source_url', 'download_link', 'title', 'content_type', 'id', 'caption',\n",
      "       'description', 'context_description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 456 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 462 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 1034 - INFO - Output written to file: output/result.csv\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 1036 - INFO - File Download Schedule: []\u001B[0m\n",
      "\u001B[97mdata_gatherer.py - line 1053 - INFO - Closing the EntrezFetcher.\u001B[0m\n"
     ]
    }
   ],
   "execution_count": 6
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-08-18T23:00:13.714678Z",
     "start_time": "2025-08-18T23:00:13.690711Z"
    }
   },
   "cell_type": "code",
   "source": "combined_df",
   "id": "9fd1be22241ced6b",
   "outputs": [
    {
     "data": {
      "text/plain": [
       "    dataset_identifier data_repository  \\\n",
       "0           syn2580853     synapse.org   \n",
       "1                  NaN             NaN   \n",
       "2                  NaN             NaN   \n",
       "3                  NaN             NaN   \n",
       "4                  NaN             NaN   \n",
       "..                 ...             ...   \n",
       "224                NaN             NaN   \n",
       "225                NaN             NaN   \n",
       "226                NaN             NaN   \n",
       "227                NaN             NaN   \n",
       "228                NaN             NaN   \n",
       "\n",
       "                                dataset_webpage          source_section  \\\n",
       "0    https://www.synapse.org/Synapse:syn2580853       data_availability   \n",
       "1                                           NaN  supplementary material   \n",
       "2                                           NaN  supplementary material   \n",
       "3                                           NaN  supplementary material   \n",
       "4                                           NaN  supplementary material   \n",
       "..                                          ...                     ...   \n",
       "224                                         NaN  supplementary material   \n",
       "225                                         NaN  supplementary material   \n",
       "226                                         NaN  supplementary material   \n",
       "227                                         NaN  supplementary material   \n",
       "228                                         NaN  supplementary material   \n",
       "\n",
       "             retrieval_pattern                                access_mode  \\\n",
       "0            data availability  Complex download or application to access   \n",
       "1    .//supplementary-material                                        NaN   \n",
       "2    .//supplementary-material                                        NaN   \n",
       "3    .//supplementary-material                                        NaN   \n",
       "4    .//supplementary-material                                        NaN   \n",
       "..                         ...                                        ...   \n",
       "224  .//supplementary-material                                        NaN   \n",
       "225  .//supplementary-material                                        NaN   \n",
       "226  .//supplementary-material                                        NaN   \n",
       "227  .//supplementary-material                                        NaN   \n",
       "228  .//supplementary-material                                        NaN   \n",
       "\n",
       "                                          link  \\\n",
       "0                                          NaN   \n",
       "1                13024_2025_802_MOESM1_ESM.tif   \n",
       "2                13024_2025_802_MOESM2_ESM.tif   \n",
       "3                13024_2025_802_MOESM3_ESM.tif   \n",
       "4                13024_2025_802_MOESM4_ESM.tif   \n",
       "..                                         ...   \n",
       "224                               media-1.xlsx   \n",
       "225                               media-2.xlsx   \n",
       "226                               media-3.xlsx   \n",
       "227                               media-4.xlsx   \n",
       "228  NIHPP2024.01.09.574815v2-supplement-5.pdf   \n",
       "\n",
       "                                            source_url  \\\n",
       "0    https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374   \n",
       "1    https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374   \n",
       "2    https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374   \n",
       "3    https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374   \n",
       "4    https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374   \n",
       "..                                                 ...   \n",
       "224  https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452   \n",
       "225  https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452   \n",
       "226  https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452   \n",
       "227  https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452   \n",
       "228  https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452   \n",
       "\n",
       "                                         download_link     title  ...  \\\n",
       "0                                                  NaN       NaN  ...   \n",
       "1    https://pmc.ncbi.nlm.nih.gov/articles/instance...  No Title  ...   \n",
       "2    https://pmc.ncbi.nlm.nih.gov/articles/instance...  No Title  ...   \n",
       "3    https://pmc.ncbi.nlm.nih.gov/articles/instance...  No Title  ...   \n",
       "4    https://pmc.ncbi.nlm.nih.gov/articles/instance...  No Title  ...   \n",
       "..                                                 ...       ...  ...   \n",
       "224  https://pmc.ncbi.nlm.nih.gov/articles/instance...  No Title  ...   \n",
       "225  https://pmc.ncbi.nlm.nih.gov/articles/instance...  No Title  ...   \n",
       "226  https://pmc.ncbi.nlm.nih.gov/articles/instance...  No Title  ...   \n",
       "227  https://pmc.ncbi.nlm.nih.gov/articles/instance...  No Title  ...   \n",
       "228  https://pmc.ncbi.nlm.nih.gov/articles/instance...  No Title  ...   \n",
       "\n",
       "                                   context_description file_extension  \\\n",
       "0                                                  NaN           None   \n",
       "1    ABCA1 expression also correlated with SASP sco...            tif   \n",
       "2    Using SenTraGor (STG, a reagent for detection ...            tif   \n",
       "3    ABCA1 was more abundant in lysosome-enriched f...            tif   \n",
       "4    To explore the underlying mechanism, we treate...            tif   \n",
       "..                                                 ...            ...   \n",
       "224  This cohort (n=104) comprised neurotypical (he...           xlsx   \n",
       "225  Importantly, 55.4% of the identified eQTLs ove...           xlsx   \n",
       "226  Following a detailed power evaluation ( Extend...           xlsx   \n",
       "227  MAP2, RTF2 or Distal/Proximal UTR, Supplementa...           xlsx   \n",
       "228                                                               pdf   \n",
       "\n",
       "                                             pub_title file_info  \\\n",
       "0    Cellular senescence induced by cholesterol acc...       NaN   \n",
       "1    Cellular senescence induced by cholesterol acc...       NaN   \n",
       "2    Cellular senescence induced by cholesterol acc...       NaN   \n",
       "3    Cellular senescence induced by cholesterol acc...       NaN   \n",
       "4    Cellular senescence induced by cholesterol acc...       NaN   \n",
       "..                                                 ...       ...   \n",
       "224  Polygenic risk for schizophrenia converges on ...       NaN   \n",
       "225  Polygenic risk for schizophrenia converges on ...       NaN   \n",
       "226  Polygenic risk for schizophrenia converges on ...       NaN   \n",
       "227  Polygenic risk for schizophrenia converges on ...       NaN   \n",
       "228  Polygenic risk for schizophrenia converges on ...       NaN   \n",
       "\n",
       "    section_class a_attr_href a_attr_class a_attr_data-ga-action  \\\n",
       "0             NaN         NaN          NaN                   NaN   \n",
       "1             NaN         NaN          NaN                   NaN   \n",
       "2             NaN         NaN          NaN                   NaN   \n",
       "3             NaN         NaN          NaN                   NaN   \n",
       "4             NaN         NaN          NaN                   NaN   \n",
       "..            ...         ...          ...                   ...   \n",
       "224           NaN         NaN          NaN                   NaN   \n",
       "225           NaN         NaN          NaN                   NaN   \n",
       "226           NaN         NaN          NaN                   NaN   \n",
       "227           NaN         NaN          NaN                   NaN   \n",
       "228           NaN         NaN          NaN                   NaN   \n",
       "\n",
       "    a_attr_target a_attr_rel  \n",
       "0             NaN        NaN  \n",
       "1             NaN        NaN  \n",
       "2             NaN        NaN  \n",
       "3             NaN        NaN  \n",
       "4             NaN        NaN  \n",
       "..            ...        ...  \n",
       "224           NaN        NaN  \n",
       "225           NaN        NaN  \n",
       "226           NaN        NaN  \n",
       "227           NaN        NaN  \n",
       "228           NaN        NaN  \n",
       "\n",
       "[229 rows x 24 columns]"
      ],
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>dataset_identifier</th>\n",
       "      <th>data_repository</th>\n",
       "      <th>dataset_webpage</th>\n",
       "      <th>source_section</th>\n",
       "      <th>retrieval_pattern</th>\n",
       "      <th>access_mode</th>\n",
       "      <th>link</th>\n",
       "      <th>source_url</th>\n",
       "      <th>download_link</th>\n",
       "      <th>title</th>\n",
       "      <th>...</th>\n",
       "      <th>context_description</th>\n",
       "      <th>file_extension</th>\n",
       "      <th>pub_title</th>\n",
       "      <th>file_info</th>\n",
       "      <th>section_class</th>\n",
       "      <th>a_attr_href</th>\n",
       "      <th>a_attr_class</th>\n",
       "      <th>a_attr_data-ga-action</th>\n",
       "      <th>a_attr_target</th>\n",
       "      <th>a_attr_rel</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>syn2580853</td>\n",
       "      <td>synapse.org</td>\n",
       "      <td>https://www.synapse.org/Synapse:syn2580853</td>\n",
       "      <td>data_availability</td>\n",
       "      <td>data availability</td>\n",
       "      <td>Complex download or application to access</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "      <td>Cellular senescence induced by cholesterol acc...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>supplementary material</td>\n",
       "      <td>.//supplementary-material</td>\n",
       "      <td>NaN</td>\n",
       "      <td>13024_2025_802_MOESM1_ESM.tif</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/instance...</td>\n",
       "      <td>No Title</td>\n",
       "      <td>...</td>\n",
       "      <td>ABCA1 expression also correlated with SASP sco...</td>\n",
       "      <td>tif</td>\n",
       "      <td>Cellular senescence induced by cholesterol acc...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>supplementary material</td>\n",
       "      <td>.//supplementary-material</td>\n",
       "      <td>NaN</td>\n",
       "      <td>13024_2025_802_MOESM2_ESM.tif</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/instance...</td>\n",
       "      <td>No Title</td>\n",
       "      <td>...</td>\n",
       "      <td>Using SenTraGor (STG, a reagent for detection ...</td>\n",
       "      <td>tif</td>\n",
       "      <td>Cellular senescence induced by cholesterol acc...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>supplementary material</td>\n",
       "      <td>.//supplementary-material</td>\n",
       "      <td>NaN</td>\n",
       "      <td>13024_2025_802_MOESM3_ESM.tif</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/instance...</td>\n",
       "      <td>No Title</td>\n",
       "      <td>...</td>\n",
       "      <td>ABCA1 was more abundant in lysosome-enriched f...</td>\n",
       "      <td>tif</td>\n",
       "      <td>Cellular senescence induced by cholesterol acc...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>supplementary material</td>\n",
       "      <td>.//supplementary-material</td>\n",
       "      <td>NaN</td>\n",
       "      <td>13024_2025_802_MOESM4_ESM.tif</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/instance...</td>\n",
       "      <td>No Title</td>\n",
       "      <td>...</td>\n",
       "      <td>To explore the underlying mechanism, we treate...</td>\n",
       "      <td>tif</td>\n",
       "      <td>Cellular senescence induced by cholesterol acc...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>224</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>supplementary material</td>\n",
       "      <td>.//supplementary-material</td>\n",
       "      <td>NaN</td>\n",
       "      <td>media-1.xlsx</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/instance...</td>\n",
       "      <td>No Title</td>\n",
       "      <td>...</td>\n",
       "      <td>This cohort (n=104) comprised neurotypical (he...</td>\n",
       "      <td>xlsx</td>\n",
       "      <td>Polygenic risk for schizophrenia converges on ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>225</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>supplementary material</td>\n",
       "      <td>.//supplementary-material</td>\n",
       "      <td>NaN</td>\n",
       "      <td>media-2.xlsx</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/instance...</td>\n",
       "      <td>No Title</td>\n",
       "      <td>...</td>\n",
       "      <td>Importantly, 55.4% of the identified eQTLs ove...</td>\n",
       "      <td>xlsx</td>\n",
       "      <td>Polygenic risk for schizophrenia converges on ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>226</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>supplementary material</td>\n",
       "      <td>.//supplementary-material</td>\n",
       "      <td>NaN</td>\n",
       "      <td>media-3.xlsx</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/instance...</td>\n",
       "      <td>No Title</td>\n",
       "      <td>...</td>\n",
       "      <td>Following a detailed power evaluation ( Extend...</td>\n",
       "      <td>xlsx</td>\n",
       "      <td>Polygenic risk for schizophrenia converges on ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>227</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>supplementary material</td>\n",
       "      <td>.//supplementary-material</td>\n",
       "      <td>NaN</td>\n",
       "      <td>media-4.xlsx</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/instance...</td>\n",
       "      <td>No Title</td>\n",
       "      <td>...</td>\n",
       "      <td>MAP2, RTF2 or Distal/Proximal UTR, Supplementa...</td>\n",
       "      <td>xlsx</td>\n",
       "      <td>Polygenic risk for schizophrenia converges on ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>228</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>supplementary material</td>\n",
       "      <td>.//supplementary-material</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NIHPP2024.01.09.574815v2-supplement-5.pdf</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/instance...</td>\n",
       "      <td>No Title</td>\n",
       "      <td>...</td>\n",
       "      <td></td>\n",
       "      <td>pdf</td>\n",
       "      <td>Polygenic risk for schizophrenia converges on ...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>229 rows × 24 columns</p>\n",
       "</div>"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "execution_count": 7
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-08-18T23:00:14.585157Z",
     "start_time": "2025-08-18T23:00:13.715808Z"
    }
   },
   "cell_type": "code",
   "source": "ground_truth = pd.read_parquet(\"exp_input/dataset_citation_records_Table.parquet\")",
   "id": "305d7d5f299551c1",
   "outputs": [],
   "execution_count": 8
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-08-18T23:00:14.605506Z",
     "start_time": "2025-08-18T23:00:14.587876Z"
    }
   },
   "cell_type": "code",
   "source": "ground_truth.head()",
   "id": "dadaf8654a9d103d",
   "outputs": [
    {
     "data": {
      "text/plain": [
       "  identifier repository                            citing_publication_link  \\\n",
       "0  PXD059466      PRIDE      https://dx.doi.org/10.1038/S41467-025-56720-1   \n",
       "1  PXD051312      PRIDE               https://dx.doi.org/10.6019/PXD051312   \n",
       "2  PXD051312      PRIDE          https://dx.doi.org/10.1002/prca.202400095   \n",
       "3  PXD051312      PRIDE  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...   \n",
       "4  PXD054431      PRIDE        https://dx.doi.org/10.17159/SAJS.2025/18571   \n",
       "\n",
       "       citation_record_source  citation_record_from_doi  \\\n",
       "0  proteomexchange_search.tsv                         1   \n",
       "1  proteomexchange_search.tsv                         1   \n",
       "2  proteomexchange_search.tsv                         1   \n",
       "3  proteomexchange_search.tsv                         0   \n",
       "4  proteomexchange_search.tsv                         1   \n",
       "\n",
       "                          doi        pmcid  \n",
       "0  10.1038/S41467-025-56720-1               \n",
       "1           10.6019/PXD051312               \n",
       "2      10.1002/prca.202400095               \n",
       "3                              PMC11895760  \n",
       "4    10.17159/SAJS.2025/18571               "
      ],
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>identifier</th>\n",
       "      <th>repository</th>\n",
       "      <th>citing_publication_link</th>\n",
       "      <th>citation_record_source</th>\n",
       "      <th>citation_record_from_doi</th>\n",
       "      <th>doi</th>\n",
       "      <th>pmcid</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>PXD059466</td>\n",
       "      <td>PRIDE</td>\n",
       "      <td>https://dx.doi.org/10.1038/S41467-025-56720-1</td>\n",
       "      <td>proteomexchange_search.tsv</td>\n",
       "      <td>1</td>\n",
       "      <td>10.1038/S41467-025-56720-1</td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>PXD051312</td>\n",
       "      <td>PRIDE</td>\n",
       "      <td>https://dx.doi.org/10.6019/PXD051312</td>\n",
       "      <td>proteomexchange_search.tsv</td>\n",
       "      <td>1</td>\n",
       "      <td>10.6019/PXD051312</td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>PXD051312</td>\n",
       "      <td>PRIDE</td>\n",
       "      <td>https://dx.doi.org/10.1002/prca.202400095</td>\n",
       "      <td>proteomexchange_search.tsv</td>\n",
       "      <td>1</td>\n",
       "      <td>10.1002/prca.202400095</td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>PXD051312</td>\n",
       "      <td>PRIDE</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...</td>\n",
       "      <td>proteomexchange_search.tsv</td>\n",
       "      <td>0</td>\n",
       "      <td></td>\n",
       "      <td>PMC11895760</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>PXD054431</td>\n",
       "      <td>PRIDE</td>\n",
       "      <td>https://dx.doi.org/10.17159/SAJS.2025/18571</td>\n",
       "      <td>proteomexchange_search.tsv</td>\n",
       "      <td>1</td>\n",
       "      <td>10.17159/SAJS.2025/18571</td>\n",
       "      <td></td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "execution_count": 9
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-08-18T23:00:14.637797Z",
     "start_time": "2025-08-18T23:00:14.609198Z"
    }
   },
   "cell_type": "code",
   "source": [
    "def evaluate_performance(predict_df, ground_truth, orchestrator, false_positives_file):\n",
    "    \"\"\" Evaluates dataset extraction performance using precision, recall, and F1-score. \"\"\"\n",
    "\n",
    "    recall_list, false_positives_output = [], []\n",
    "    total_precision, total_recall, num_sources = 0, 0, 0\n",
    "\n",
    "    for source_page in predict_df['source_url'].unique():\n",
    "        pub_id = source_page.split('/')[-1].lower()\n",
    "        \n",
    "        orchestrator.logger.info(f\"Evaluating pub_id: {pub_id}\")\n",
    "        gt_data = ground_truth[ground_truth['pmcid'].str.lower() == pub_id.lower()]  # extract ground truth\n",
    "\n",
    "        gt_datasets = set()\n",
    "        for dataset_string in gt_data['identifier'].dropna().str.lower():\n",
    "            gt_datasets.update(dataset_string.split(','))  # Convert CSV string into set of IDs\n",
    "\n",
    "        orchestrator.logger.info(f\"# of elements in gt_data: {len(gt_data)}. Element IDs: {gt_datasets}\")\n",
    "\n",
    "        num_sources += 1\n",
    "\n",
    "        # Extract evaluation datasets for this source page\n",
    "        eval_data = predict_df[predict_df['source_url'] == source_page]\n",
    "        eval_datasets = set(eval_data['dataset_identifier'].dropna().str.lower())\n",
    "        # Remove invalid entries\n",
    "        eval_datasets.discard('n/a')\n",
    "        eval_datasets.discard('')\n",
    "\n",
    "        orchestrator.logger.info(f\"Evaluation datasets: {eval_datasets}\")\n",
    "\n",
    "        # Handle cases where both ground truth and evaluation are empty\n",
    "        if not gt_datasets and not eval_datasets:\n",
    "            orchestrator.logger.info(\"No datasets in both ground truth and evaluation. Perfect precision and recall.\")\n",
    "            total_precision += 1\n",
    "            total_recall += 1\n",
    "            continue\n",
    "\n",
    "        # Match Extraction Logic\n",
    "        matched_gt, matched_eval = set(), set()\n",
    "\n",
    "        # Exact Matches\n",
    "        exact_matches = gt_datasets & eval_datasets  # Intersection of ground truth and extracted datasets\n",
    "        matched_gt.update(exact_matches)\n",
    "        matched_eval.update(exact_matches)\n",
    "\n",
    "        # Partial Matches (Aliased Identifiers)\n",
    "        for eval_id in eval_datasets - matched_eval:\n",
    "            for gt_id in gt_datasets - matched_gt:\n",
    "                if eval_id in gt_id or gt_id in eval_id:  # Partial match or alias\n",
    "                    orchestrator.logger.info(f\"Partial or alias match found: eval_id={eval_id}, gt_id={gt_id}\")\n",
    "                    matched_gt.add(gt_id)\n",
    "                    matched_eval.add(eval_id)\n",
    "                    break  # Stop once matched\n",
    "\n",
    "        # **False Positives (Unmatched extracted datasets)**\n",
    "        FP = eval_datasets - matched_eval\n",
    "        false_positives_output.extend(FP)\n",
    "\n",
    "        # **False Negatives (Unmatched ground truth datasets)**\n",
    "        FN = gt_datasets - matched_gt\n",
    "\n",
    "        # **Precision and Recall Calculation**\n",
    "        true_positives = len(matched_gt)\n",
    "        false_positives = len(FP)\n",
    "        false_negatives = len(FN)\n",
    "\n",
    "        precision = true_positives / (true_positives + false_positives) if (true_positives + false_positives) > 0 else 0\n",
    "        recall = true_positives / (true_positives + false_negatives) if (true_positives + false_negatives) > 0 else 0\n",
    "\n",
    "        orchestrator.logger.info(f\"Precision for {source_page}: {precision}\")\n",
    "        orchestrator.logger.info(f\"Recall for {source_page}: {recall}\")\n",
    "\n",
    "        if recall == 0:\n",
    "            recall_list.append(source_page)\n",
    "\n",
    "        # Accumulate totals\n",
    "        total_precision += precision\n",
    "        total_recall += recall\n",
    "\n",
    "    # **Compute Overall Metrics**\n",
    "    average_precision = total_precision / num_sources if num_sources > 0 else 0\n",
    "    average_recall = total_recall / num_sources if num_sources > 0 else 0\n",
    "    f1_score = (\n",
    "        2 * (average_precision * average_recall) / (average_precision + average_recall)\n",
    "        if (average_precision + average_recall) > 0\n",
    "        else 0\n",
    "    )\n",
    "\n",
    "    orchestrator.logger.info(f\"\\nPerformance evaluation completed for {num_sources} source pages.\")\n",
    "\n",
    "    # **Save false positives**\n",
    "    with open(false_positives_file, 'w') as f:\n",
    "        for item in false_positives_output:\n",
    "            f.write(\"%s\\n\" % item)\n",
    "\n",
    "    return {\n",
    "        \"average_precision\": average_precision,\n",
    "        \"average_recall\": average_recall,\n",
    "        \"f1_score\": f1_score\n",
    "    }"
   ],
   "id": "583f880224783dc7",
   "outputs": [],
   "execution_count": 10
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-08-18T23:00:17.659966Z",
     "start_time": "2025-08-18T23:00:14.640441Z"
    }
   },
   "cell_type": "code",
   "source": "results = evaluate_performance(combined_df, ground_truth, dg, \"output/false_positives.txt\")",
   "id": "3ae367545bb1b42b",
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc11792374\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 4. Element IDs: {'syn52293417', 'syn8456637', 'syn9702085', 'syn2580853'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn2580853'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374: 0.25\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc4318527\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn1720047', 'syn2455683'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: set()\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527: 0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527: 0.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc4339277\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 4. Element IDs: {'syn1732077', 'syn1688309', 'syn1682511', 'syn1701558'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: set()\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277: 0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277: 0.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc8452114\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn18485175'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn18485175'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8452114: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8452114: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc9641692\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn3219045'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'gse119117', 'gse84133', 'syn3219045', 'gse87544', 'tape', 'gse107011', 'gse65133', 'gse67835', 'gse115449'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9641692: 0.1111111111111111\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9641692: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc5161470\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1446295'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'encode', '1kgp', 'hapmap'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC5161470: 0.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC5161470: 0.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc9925742\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'gse224398', 'syn21126450'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'srr21079269', 'srr21079268', 'srr21079267', 'srr21079265', 'appki_oligo_scrnaseq', 'syn21126450', 'srr21079270', 'gse224398', 'srr21079266'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9925742: 0.2222222222222222\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9925742: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc9235070\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn30243143'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'metabric', 'tcga', 'msigdb', 'gse76312', 'syn30243143', 'e-mtab-6149'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9235070: 0.16666666666666666\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9235070: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc11661334\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn51364943'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn51364943', '35107', 'gcst90027158', 'gcst007511', 'gcst009325'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11661334: 0.2\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11661334: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc11929800\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 5. Element IDs: {'syn18485175', 'syn11724057', 'syn4896408', 'syn3388564', 'syn17015098'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn3388564', 'syn4896408', 'gcst90027158', 'e073', 'gse291402', 'syn11724057', 'syn17015098'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11929800: 0.5714285714285714\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11929800: 0.8\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc7113080\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn17394041'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn17394041'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc8847344\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn25813713'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn25813713'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8847344: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8847344: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc9280291\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn5612563'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn5612563'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9280291: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9280291: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc10029021\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn50897018'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'gtex', 'psychencode', 'syn50897018.3'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 49 - INFO - Partial or alias match found: eval_id=syn50897018.3, gt_id=syn50897018\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10029021: 0.3333333333333333\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10029021: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc11525589\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn51514106'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'brats-men', 'syn51514106'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11525589: 0.5\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11525589: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc10836119\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 6. Element IDs: {'syn3205821', 'syn3157329', 'syn3157322', 'syn5592519', 'syn3157238', 'syn9705278'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn3205821', 'syn3157329', 'syn3157322', 'ages-rs', 'kora', 'syn5592519', 'syn3157238', 'syn9705278'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10836119: 0.75\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10836119: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc6646046\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn5609493'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'4.3'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6646046: 0.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6646046: 0.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc10387117\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21041850'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'gse134174', 'gse161382', 'gse136831', 'gse135851', 'gse122332', 'gse135893', 'dataset-427f1eee6dd44f50bae1ab13f0f3c6a9', 'gse171524', 'gse122960', 'gse178362', 'gse136832', 'lmex0000004396', 'gse149563', 'lmex0000004397', 'syn21041850', 'egas00001004082'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10387117: 0.0625\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10387117: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc4444046\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn1710680', 'syn1729383', 'syn1734155'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn1710680', 'syn1729383', 'syn1734155'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4444046: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4444046: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc10329279\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21478998'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: set()\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279: 0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279: 0.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc11145390\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn21788425', 'gse147807'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'murphy_data', 'gse147807'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11145390: 0.5\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11145390: 0.5\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc11032436\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2759792'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn2759792'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc9984184\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn39456106', 'syn25576878'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'gse95587', 'gse15222', 'e-geod-44770'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9984184: 0.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9984184: 0.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc7347884\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1757053'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'syn1757053', 'egas00000000083', 'gse56031', 'ascat', 'e-mtab-4145'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7347884: 0.2\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7347884: 1.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 10 - INFO - Evaluating pub_id: pmc10802452\u001B[0m\n",
      "\u001B[97m1292017431.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2759792'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 28 - INFO - Evaluation datasets: {'tba', '{tba}', 'syngo'}\u001B[0m\n",
      "\u001B[97m1292017431.py - line 69 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452: 0.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 70 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452: 0.0\u001B[0m\n",
      "\u001B[97m1292017431.py - line 88 - INFO - \n",
      "Performance evaluation completed for 25 source pages.\u001B[0m\n"
     ]
    }
   ],
   "execution_count": 11
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-08-18T23:00:17.665313Z",
     "start_time": "2025-08-18T23:00:17.661193Z"
    }
   },
   "cell_type": "code",
   "source": "results",
   "id": "fff3dc3075cb13b5",
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'average_precision': 0.4246904761904762,\n",
       " 'average_recall': 0.662,\n",
       " 'f1_score': 0.5174336232773163}"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "execution_count": 12
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 2
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython2",
   "version": "2.7.6"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
