{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "initial_id",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-09-04T16:30:53.289617Z",
     "start_time": "2025-09-04T16:30:29.500758Z"
    },
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "import os\n",
    "from data_gatherer.data_gatherer import DataGatherer\n",
    "import pandas as pd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "78d2b2bc4106d2ef",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-09-04T16:31:15.898169Z",
     "start_time": "2025-09-04T16:31:15.880961Z"
    }
   },
   "outputs": [],
   "source": [
    "df = pd.read_csv(\"scripts/exp_input/SAGE_groundtruth.csv\")\n",
    "input_file = \"scripts/exp_input/europe_pmc_synapse_id_mining.txt\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "cca05a29b7c100d1",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-09-04T16:31:17.295747Z",
     "start_time": "2025-09-04T16:31:17.291744Z"
    }
   },
   "outputs": [],
   "source": [
    "model_name = \"gemini-2.5-flash\"  # \"gemini-2.0-flash\" or \"gpt-4o-mini\"\n",
    "prompt = \"GPT_FDR_FewShot_Syn\"  # \"GPT_from_full_input_Examples\" or \"retrieve_datasets_simple_JSON\"\n",
    "FDR = True\n",
    "semantic_retrieval = False\n",
    "section_filter= None"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "a4ae25cbd54d4fe1",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-09-04T16:31:19.865917Z",
     "start_time": "2025-09-04T16:31:19.860295Z"
    }
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of PMCIDs: 2190\n"
     ]
    }
   ],
   "source": [
    "# write list to a text file\n",
    "with open(input_file, 'r') as f:\n",
    "    pmcids = f.read().splitlines()\n",
    "\n",
    "print(\"Number of PMCIDs:\", len(pmcids))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "b79cbd7265b497e1",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-09-04T16:31:22.492692Z",
     "start_time": "2025-09-04T16:31:22.479800Z"
    }
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97mdata_gatherer.py - line 301 - INFO - Setting up data fetcher...\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 33 - INFO - Loaded 2190 publications from local DataFrame.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 331 - INFO - Data fetcher setup completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 103 - INFO - DataGatherer orchestrator initialized. Extraction Model: gemini-2.5-flash\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "dg = DataGatherer(\n",
    "    llm_name=model_name, \n",
    "    log_level='INFO', \n",
    "    process_entire_document=FDR, \n",
    "    driver_path=None, \n",
    "    save_to_cache=True, \n",
    "    load_from_cache=True,\n",
    "    full_output_file=\"scripts/output/result.csv\"\n",
    ") #, save_dynamic_prompts=True"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "5e0e0500474a6c34",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-09-04T16:31:25.329784Z",
     "start_time": "2025-09-04T16:31:25.325226Z"
    }
   },
   "outputs": [],
   "source": [
    "# These are the problematic urls for information extraction for GEMINI\n",
    "#urls = [\"https://pmc.ncbi.nlm.nih.gov/articles/PMC9314356\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527/\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC11929800/\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436/\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452/\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8565335/\",\"https://pmc.ncbi.nlm.nih.gov/articles/PMC7778917/\"]\n",
    "# v1 = \"https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360/\"\n",
    "# v2 = \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8006362\"\n",
    "#urls = [\"https://pmc.ncbi.nlm.nih.gov/articles/PMC9710693/\"]\n",
    "\n",
    "# These are the problematic urls for information extraction for GPT - (2025 Aug 29) GPT gets stuck for some reason after 50 calls or so...\n",
    "#urls = [\"https://pmc.ncbi.nlm.nih.gov/articles/PMC11240079\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC5752539\",\"https://pmc.ncbi.nlm.nih.gov/articles/PMC7032692\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10547713\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8102856\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8055881\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8226229\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10769298\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC11659981\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8082263\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8565335\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC9094742\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC6289083\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC7658217\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC9780309\",\"https://pmc.ncbi.nlm.nih.gov/articles/PMC3788619\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10680627\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8131595\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC11420198\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC9915613\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10238095\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452\" , \"https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10836119\",  \"https://pmc.ncbi.nlm.nih.gov/articles/PMC9280291\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC11661334\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC5161470\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277\"]\n",
    "\n",
    "#out = dg.process_articles(urls, semantic_retrieval=semantic_retrieval, section_filter=section_filter, prompt_name=prompt)\n",
    "\n",
    "#out\n",
    "#out['https://pmc.ncbi.nlm.nih.gov/articles/PMC9710693']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "bfd0572331e32fe2",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-09-04T17:36:22.343172Z",
     "start_time": "2025-09-04T16:31:29.415907Z"
    }
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97mdata_gatherer.py - line 668 - INFO - Loaded 2190 URLs from file.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 0th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10001072)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10001072\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10001072 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 33 - INFO - Loaded 2190 publications from local DataFrame.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10001072\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 1/2190 (0.05%) | Elapsed: 00:00:05 | ETA: 03:16:33\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 1th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10002464)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10002464\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10002464 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10002464\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 2th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10005937)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10005937\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10005937 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10005937\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 3th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10011140)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10011140\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10011140 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10011140\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 4th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10013873)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10013873\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10013873 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10013873\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 5th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10014304)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10014304\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10014304 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10014304\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 6th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10025452)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10025452\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10025452 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10025452\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 7th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10025681)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10025681\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10025681 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10025681\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 8th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10028786)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10028786\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10028786 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10028786\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 9th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10028977)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10028977\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10028977 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10028977\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 10th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10029021)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10029021\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10029021 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10029021\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 11/2190 (0.50%) | Elapsed: 00:00:06 | ETA: 00:21:22\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 11th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10031009)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10031009\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10031009 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10031009\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 12th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10039068)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10039068\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10039068 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10039068\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 13th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10039465)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10039465\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10039465 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10039465\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 14th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10041447)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10041447\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10041447 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10041447\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 15th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10041734)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10041734\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10041734 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10041734\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 16th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10043753)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10043753\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10043753 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10043753\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 17th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10044680)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10044680\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10044680 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10044680\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 18th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10046116)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10046116\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10046116 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10046116\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 19th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10050155)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10050155\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10050155 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10050155\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 20th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10054947)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10054947\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10054947 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10054947\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 21/2190 (0.96%) | Elapsed: 00:00:07 | ETA: 00:13:19\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 21th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10055057)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10055057\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10055057 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10055057\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 22th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10055234)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10055234\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10055234 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10055234\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 23th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10055242)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10055242\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10055242 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10055242\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 24th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10055503)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10055503\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10055503 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10055503\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 25th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10055654)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10055654\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10055654 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10055654\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 26th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10060837)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10060837\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10060837 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10060837\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 27th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10061294)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10061294\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10061294 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10061294\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 28th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10066241)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10066241\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10066241 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10066241\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 29th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10068328)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10068328\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10068328 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10068328\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 30th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10074678)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10074678\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10074678 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10074678\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 31/2190 (1.42%) | Elapsed: 00:00:08 | ETA: 00:10:13\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 31th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10078980)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10078980\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10078980 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10078980\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 32th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10080956)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10080956\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10080956 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10080956\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 33th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10081415)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10081415\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10081415 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10081415\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 34th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10081846)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10081846\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10081846 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10081846\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 35th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10092953)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10092953\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10092953 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10092953\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 36th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10094615)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10094615\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10094615 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10094615\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 37th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10099222)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10099222\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10099222 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10099222\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 38th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10100131)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10100131\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10100131 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10100131\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 39th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10102849)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10102849\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10102849 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10102849\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 40th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10104061)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10104061\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10104061 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10104061\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 41/2190 (1.87%) | Elapsed: 00:00:09 | ETA: 00:08:41\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 41th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10106526)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10106526\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10106526 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10106526\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 42th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10109523)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10109523\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10109523 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10109523\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 43th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10110975)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10110975\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10110975 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10110975\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 44th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10111747)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10111747\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10111747 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10111747\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 45th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10114348)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10114348\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10114348 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10114348\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 46th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10115253)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10115253\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10115253 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10115253\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 47th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10117220)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10117220\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10117220 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10117220\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 48th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10120121)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10120121\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10120121 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10120121\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 49th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10120699)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10120699\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10120699 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10120699\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 50th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10121712)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10121712\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10121712 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10121712\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 51/2190 (2.33%) | Elapsed: 00:00:11 | ETA: 00:07:42\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 51th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10126952)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10126952\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10126952 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10126952\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 52th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10131083)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10131083\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10131083 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10131083\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 53th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10131459)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10131459\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10131459 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10131459\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 54th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10131710)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10131710\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10131710 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10131710\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 55th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10136684)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10136684\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10136684 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10136684\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 56th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10137632)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10137632\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10137632 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10137632\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 57th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10153468)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10153468\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10153468 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10153468\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 58th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10156891)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10156891\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10156891 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10156891\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 59th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10157587)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10157587\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10157587 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10157587\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 60th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10159362)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10159362\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10159362 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10159362\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 61/2190 (2.79%) | Elapsed: 00:00:12 | ETA: 00:07:02\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 61th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10162998)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10162998\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10162998 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10162998\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 62th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10165836)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10165836\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10165836 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10165836\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 63th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10168239)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10168239\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10168239 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10168239\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 64th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10168282)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10168282\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10168282 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10168282\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 65th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10170100)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10170100\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10170100 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10170100\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 66th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10174396)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10174396\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10174396 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10174396\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 67th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10175346)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10175346\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10175346 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10175346\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 68th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10175463)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10175463\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10175463 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10175463\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 69th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10178328)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10178328\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10178328 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10178328\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 70th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10182086)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10182086\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10182086 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10182086\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 71/2190 (3.24%) | Elapsed: 00:00:13 | ETA: 00:06:32\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 71th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10183026)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10183026\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10183026 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10183026\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 72th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10185548)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10185548\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10185548 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10185548\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 73th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10185676)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10185676\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10185676 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10185676\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 74th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10186610)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10186610\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10186610 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10186610\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 75th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10186603)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10186603\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10186603 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10186603\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 76th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10190536)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10190536\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10190536 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10190536\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 77th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10196687)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10196687\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10196687 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10196687\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 78th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10197798)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10197798\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10197798 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10197798\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 79th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10202958)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10202958\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10202958 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10202958\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 80th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10213211)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10213211\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10213211 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10213211\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 81/2190 (3.70%) | Elapsed: 00:00:14 | ETA: 00:06:10\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 81th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10226446)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10226446\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10226446 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10226446\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 82th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10229139)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10229139\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10229139 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10229139\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 83th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10229758)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10229758\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10229758 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10229758\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 84th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10231934)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10231934\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10231934 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10231934\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 85th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10233303)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10233303\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10233303 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10233303\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 86th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10236395)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10236395\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10236395 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10236395\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 87th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10238095)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10238095\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10238095 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10238095\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 88th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10239491)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10239491\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10239491 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10239491\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 89th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10242325)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10242325\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10242325 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10242325\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 90th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10245417)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10245417\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10245417 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10245417\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 91/2190 (4.16%) | Elapsed: 00:00:15 | ETA: 00:05:52\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 91th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10246028)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10246028\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10246028 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10246028\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 92th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10246227)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10246227\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10246227 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10246227\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 93th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10246587)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10246587\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10246587 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10246587\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 94th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10257663)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10257663\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10257663 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10257663\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 95th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10258469)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10258469\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10258469 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10258469\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 96th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10259480)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10259480\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10259480 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10259480\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 97th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10267126)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10267126\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10267126 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10267126\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 98th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10272561)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10272561\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10272561 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10272561\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 99th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10274725)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10274725\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10274725 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10274725\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 100th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10274943)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10274943\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10274943 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10274943\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 101/2190 (4.61%) | Elapsed: 00:00:17 | ETA: 00:05:52\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 101th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10278156)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10278156\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10278156 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10278156\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 102th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10280644)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10280644\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10280644 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10280644\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 103th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10280841)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10280841\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10280841 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10280841\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 104th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10287567)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10287567\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10287567 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10287567\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 105th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10288858)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10288858\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10288858 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10288858\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 106th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10291472)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10291472\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10291472 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10291472\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 107th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10294276)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10294276\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10294276 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10294276\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 108th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10296721)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10296721\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10296721 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10296721\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 109th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10297489)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10297489\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10297489 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10297489\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 110th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10300032)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10300032\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10300032 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10300032\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 111/2190 (5.07%) | Elapsed: 00:00:18 | ETA: 00:05:39\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 111th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10300551)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10300551\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10300551 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10300551\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 112th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10306300)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10306300\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10306300 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10306300\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 113th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10308376)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10308376\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10308376 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10308376\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 114th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10311304)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10311304\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10311304 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10311304\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 115th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10315586)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10315586\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10315586 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10315586\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 116th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10317018)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10317018\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10317018 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10317018\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 117th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10326379)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10326379\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10326379 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10326379\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 118th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10327246)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10327246\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10327246 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10327246\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 119th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10329279\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 120th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10352249)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10352249\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10352249 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10352249\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 121/2190 (5.53%) | Elapsed: 00:00:19 | ETA: 00:05:28\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 121th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10352362)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10352362\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10352362 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10352362\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 122th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10353192)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10353192\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10353192 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10353192\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 123th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10353942)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10353942\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10353942 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10353942\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 124th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10360115)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10360115\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10360115 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10360115\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 125th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10360316)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10360316\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10360316 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10360316\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 126th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10386887)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10386887\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10386887 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10386887\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 127th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10387117)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10387117\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10387117 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10387117\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 128th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10391684)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10391684\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10391684 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10391684\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 129th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10391960)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10391960\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10391960 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10391960\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 130th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10394903)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10394903\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10394903 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10394903\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 131/2190 (5.98%) | Elapsed: 00:00:20 | ETA: 00:05:18\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 131th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10399592)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10399592\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10399592 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10399592\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 132th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10401032)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10401032\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10401032 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10401032\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 133th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10401760)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10401760\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10401760 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10401760\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 134th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10401923)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10401923\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10401923 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10401923\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 135th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10402031)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10402031\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10402031 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10402031\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 136th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10402094)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10402094\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10402094 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10402094\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 137th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10402180)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10402180\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10402180 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10402180\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 138th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10402214)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10402214\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10402214 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10402214\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 139th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10403526)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10403526\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10403526 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10403526\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 140th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10409573)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10409573\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10409573 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10409573\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 141/2190 (6.44%) | Elapsed: 00:00:21 | ETA: 00:05:10\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 141th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10412409)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10412409\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10412409 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10412409\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 142th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10412627)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10412627\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10412627 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10412627\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 143th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10412668)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10412668\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10412668 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10412668\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 144th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10415210)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10415210\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10415210 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10415210\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 145th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10415334)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10415334\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10415334 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10415334\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 146th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10415381)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10415381\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10415381 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10415381\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 147th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10418231)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10418231\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10418231 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10418231\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 148th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10432546)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10432546\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10432546 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10432546\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 149th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10433568)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10433568\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10433568 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10433568\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 150th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10436389)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10436389\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10436389 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10436389\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 151/2190 (6.89%) | Elapsed: 00:00:22 | ETA: 00:05:05\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 151th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10436507)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10436507\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10436507 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10436507\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 152th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10440785)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10440785\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10440785 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10440785\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 153th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10441303)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10441303\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10441303 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10441303\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 154th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10441762)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10441762\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10441762 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10441762\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 155th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10445840)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10445840\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10445840 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10445840\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 156th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10447420)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10447420\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10447420 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10447420\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 157th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10447532)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10447532\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10447532 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10447532\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 158th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10449643)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10449643\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10449643 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10449643\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 159th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10461976)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10461976\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10461976 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10461976\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 160th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10462057)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10462057\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10462057 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10462057\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 161/2190 (7.35%) | Elapsed: 00:00:23 | ETA: 00:04:58\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 161th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10462195)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10462195\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10462195 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10462195\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 162th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10462231)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10462231\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10462231 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10462231\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 163th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10462763)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10462763\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10462763 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10462763\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 164th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10465358)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10465358\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10465358 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10465358\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 165th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10467431)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10467431\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10467431 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10467431\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 166th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10469413)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10469413\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10469413 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10469413\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 167th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10471569)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10471569\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10471569 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10471569\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 168th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10472103)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10472103\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10472103 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10472103\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 169th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10474289)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10474289\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10474289 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10474289\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 170th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10481165)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10481165\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10481165 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10481165\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 171/2190 (7.81%) | Elapsed: 00:00:24 | ETA: 00:04:52\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 171th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10491382)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10491382\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10491382 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10491382\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 172th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10491684)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10491684\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10491684 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10491684\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 173th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10491694)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10491694\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10491694 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10491694\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 174th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10492838)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10492838\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10492838 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10492838\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 175th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10495446)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10495446\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10495446 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10495446\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 176th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10497992)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10497992\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10497992 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10497992\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 177th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10498615)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10498615\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10498615 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10498615\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 178th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10499619)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10499619\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10499619 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10499619\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 179th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10499828)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10499828\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10499828 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10499828\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 180th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10499989)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10499989\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10499989 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10499989\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 181/2190 (8.26%) | Elapsed: 00:00:25 | ETA: 00:04:46\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 181th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10500750)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10500750\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10500750 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10500750\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 182th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10504083)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10504083\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10504083 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10504083\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 183th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10507917)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10507917\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10507917 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10507917\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 184th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10510131)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10510131\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10510131 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10510131\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 185th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10511325)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10511325\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10511325 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10511325\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 186th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10511328)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10511328\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10511328 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10511328\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 187th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10511327)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10511327\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10511327 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10511327\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 188th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10512627)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10512627\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10512627 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10512627\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 189th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10514332)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10514332\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10514332 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10514332\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 190th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10516495)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10516495\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10516495 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10516495\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 191/2190 (8.72%) | Elapsed: 00:00:27 | ETA: 00:04:43\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 191th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10518080)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10518080\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10518080 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10518080\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 192th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10519377)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10519377\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10519377 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10519377\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 193th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10535074)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10535074\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10535074 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10535074\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 194th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10539500)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10539500\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10539500 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10539500\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 195th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10543041)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10543041\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10543041 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10543041\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 196th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10543129)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10543129\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10543129 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10543129\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 197th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10543369)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10543369\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10543369 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10543369\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 198th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10545814)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10545814\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10545814 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10545814\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 199th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10545906)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10545906\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10545906 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10545906\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 200th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10546724)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10546724\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10546724 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10546724\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 201/2190 (9.18%) | Elapsed: 00:00:28 | ETA: 00:04:38\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 201th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10547713)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10547713\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10547713 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10547713\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 202th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10547771)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10547771\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10547771 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10547771\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 203th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10548686)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10548686\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10548686 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10548686\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 204th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10556090)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10556090\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10556090 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10556090\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 205th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10558209)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10558209\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10558209 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10558209\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 206th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10558435)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10558435\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10558435 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10558435\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 207th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10560417)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10560417\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10560417 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10560417\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 208th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10563262)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10563262\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10563262 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10563262\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 209th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10568487)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10568487\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10568487 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10568487\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 210th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10570048)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10570048\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10570048 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10570048\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 211/2190 (9.63%) | Elapsed: 00:00:29 | ETA: 00:04:34\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 211th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10571258)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10571258\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10571258 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10571258\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 212th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10571470)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10571470\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10571470 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10571470\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 213th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10581603)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10581603\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10581603 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10581603\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 214th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10592607)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10592607\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10592607 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10592607\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 215th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10592666)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10592666\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10592666 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10592666\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 216th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10592962)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10592962\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10592962 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10592962\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 217th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10593028)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10593028\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10593028 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10593028\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 218th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10593042)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10593042\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10593042 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10593042\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 219th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10600242)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10600242\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10600242 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10600242\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 220th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10600818)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10600818\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10600818 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10600818\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 221/2190 (10.09%) | Elapsed: 00:00:30 | ETA: 00:04:32\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 221th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10602153)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10602153\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10602153 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10602153\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 222th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10602154)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10602154\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10602154 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10602154\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 223th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10602176)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10602176\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10602176 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10602176\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 224th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10602695)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10602695\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10602695 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10602695\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 225th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10605063)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10605063\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10605063 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10605063\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 226th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10605344)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10605344\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10605344 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10605344\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 227th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10611347)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10611347\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10611347 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10611347\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 228th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10612362)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10612362\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10612362 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10612362\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 229th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10614458)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10614458\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10614458 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10614458\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 230th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10615768)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10615768\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10615768 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10615768\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 231/2190 (10.55%) | Elapsed: 00:00:31 | ETA: 00:04:28\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 231th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10617196)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10617196\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10617196 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10617196\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 232th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10619509)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10619509\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10619509 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10619509\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 233th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10622416)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10622416\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10622416 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10622416\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 234th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10623726)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10623726\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10623726 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10623726\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 235th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10623894)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10623894\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10623894 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10623894\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 236th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10624900)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10624900\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10624900 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10624900\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 237th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10625629)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10625629\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10625629 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10625629\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 238th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10628938)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10628938\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10628938 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10628938\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 239th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10629206)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10629206\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10629206 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10629206\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 240th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10629934)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10629934\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10629934 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10629934\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 241/2190 (11.00%) | Elapsed: 00:00:32 | ETA: 00:04:25\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 241th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10634790)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10634790\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10634790 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10634790\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 242th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10634977)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10634977\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10634977 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10634977\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 243th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10635399)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10635399\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10635399 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10635399\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 244th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10636288)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10636288\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10636288 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10636288\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 245th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10639075)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10639075\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10639075 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10639075\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 246th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10643363)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10643363\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10643363 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10643363\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 247th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10644972)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10644972\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10644972 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10644972\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 248th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10649553)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10649553\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10649553 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10649553\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 249th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10651487)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10651487\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10651487 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10651487\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 250th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10651789)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10651789\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10651789 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10651789\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 251/2190 (11.46%) | Elapsed: 00:00:34 | ETA: 00:04:23\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 251th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10652011)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10652011\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10652011 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10652011\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 252th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10652298)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10652298\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10652298 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10652298\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 253th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10652311)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10652311\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10652311 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10652311\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 254th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10652415)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10652415\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10652415 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10652415\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 255th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10659492)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10659492\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10659492 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10659492\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 256th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10660488)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10660488\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10660488 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10660488\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 257th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10661827)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10661827\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10661827 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10661827\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 258th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10662149)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10662149\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10662149 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10662149\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 259th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10663468)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10663468\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10663468 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10663468\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 260th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10667307)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10667307\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10667307 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10667307\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 261/2190 (11.92%) | Elapsed: 00:00:35 | ETA: 00:04:20\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 261th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10667362)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10667362\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10667362 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10667362\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 262th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10669538)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10669538\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10669538 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10669538\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 263th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10672174)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10672174\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10672174 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10672174\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 264th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10679640)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10679640\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10679640 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10679640\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 265th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10679867)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10679867\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10679867 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10679867\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 266th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10680627)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10680627\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10680627 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10680627\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 267th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10680636)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10680636\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10680636 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10680636\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 268th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10681996)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10681996\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10681996 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10681996\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 269th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10684891)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10684891\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10684891 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10684891\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 270th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10685699)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10685699\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10685699 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10685699\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 271/2190 (12.37%) | Elapsed: 00:00:36 | ETA: 00:04:19\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 271th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10687892)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10687892\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10687892 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10687892\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 272th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10689816)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10689816\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10689816 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10689816\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 273th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10690099)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10690099\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10690099 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10690099\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 274th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10690143)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10690143\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10690143 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10690143\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 275th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10691533)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10691533\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10691533 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10691533\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 276th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10695556)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10695556\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10695556 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10695556\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 277th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10696419)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10696419\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10696419 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10696419\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 278th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10703708)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10703708\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10703708 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10703708\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 279th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10704700)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10704700\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10704700 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10704700\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 280th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10705658)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10705658\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10705658 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10705658\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 281/2190 (12.83%) | Elapsed: 00:00:37 | ETA: 00:04:17\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 281th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10706888)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10706888\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10706888 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10706888\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 282th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10708613)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10708613\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10708613 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10708613\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 283th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10708859)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10708859\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10708859 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10708859\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 284th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10709411)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10709411\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10709411 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10709411\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 285th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10710425)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10710425\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10710425 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10710425\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 286th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10715669)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10715669\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10715669 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10715669\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 287th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10716771)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10716771\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10716771 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10716771\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 288th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10724249)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10724249\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10724249 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10724249\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 289th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10725467)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10725467\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10725467 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10725467\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 290th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10726135)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10726135\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10726135 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10726135\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 291/2190 (13.29%) | Elapsed: 00:00:39 | ETA: 00:04:14\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 291th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10729864)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10729864\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10729864 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10729864\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 292th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10733301)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10733301\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10733301 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10733301\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 293th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10733336)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10733336\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10733336 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10733336\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 294th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10742026)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10742026\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10742026 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10742026\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 295th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10745139)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10745139\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10745139 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10745139\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 296th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10749789)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10749789\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10749789 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10749789\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 297th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10750427)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10750427\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10750427 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10750427\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 298th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10757721)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10757721\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10757721 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10757721\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 299th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10760108)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10760108\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10760108 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10760108\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 300th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10762111)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10762111\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10762111 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10762111\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 301/2190 (13.74%) | Elapsed: 00:00:40 | ETA: 00:04:13\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 301th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10762261)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10762261\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10762261 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10762261\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 302th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10764846)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10764846\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10764846 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10764846\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 303th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10765928)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10765928\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10765928 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10765928\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 304th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10766724)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10766724\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10766724 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10766724\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 305th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10767859)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10767859\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10767859 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10767859\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 306th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10768370)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10768370\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10768370 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10768370\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 307th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10769232)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10769232\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10769232 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10769232\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 308th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10769298)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10769298\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10769298 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10769298\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 309th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10769366)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10769366\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10769366 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10769366\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 310th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10769393)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10769393\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10769393 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10769393\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 311/2190 (14.20%) | Elapsed: 00:00:41 | ETA: 00:04:10\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 311th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10769817)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10769817\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10769817 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10769817\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 312th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10773825)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10773825\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10773825 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10773825\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 313th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10775662)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10775662\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10775662 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10775662\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 314th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10778372)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10778372\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10778372 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10778372\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 315th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10782919)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10782919\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10782919 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10782919\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 316th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10783859)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10783859\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10783859 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10783859\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 317th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10784705)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10784705\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10784705 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10784705\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 318th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10786718)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10786718\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10786718 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10786718\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 319th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10787788)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10787788\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10787788 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10787788\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 320th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10791605)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10791605\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10791605 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10791605\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 321/2190 (14.66%) | Elapsed: 00:00:42 | ETA: 00:04:08\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 321th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10793400)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10793400\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10793400 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10793400\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 322th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10793412)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10793412\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10793412 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10793412\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 323th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10796008)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10796008\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10796008 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10796008\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 324th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10796047)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10796047\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10796047 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10796047\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 325th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10802033)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10802033\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10802033 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10802033\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 326th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10802066)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10802066\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10802066 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10802066\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 327th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10802452\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 328th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10805698)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10805698\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10805698 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10805698\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 329th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10805784)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10805784\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10805784 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10805784\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 330th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10805888)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10805888\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10805888 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10805888\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 331/2190 (15.11%) | Elapsed: 00:00:43 | ETA: 00:04:05\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 331th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10808534)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10808534\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10808534 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10808534\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 332th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10809448)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10809448\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10809448 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10809448\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 333th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10809680)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10809680\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10809680 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10809680\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 334th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10810793)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10810793\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10810793 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10810793\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 335th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10813059)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10813059\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10813059 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10813059\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 336th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10824791)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10824791\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10824791 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10824791\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 337th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10828232)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10828232\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10828232 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10828232\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 338th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10829797)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10829797\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10829797 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10829797\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 339th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10829875)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10829875\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10829875 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10829875\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 340th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10830800)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10830800\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10830800 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10830800\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 341/2190 (15.57%) | Elapsed: 00:00:44 | ETA: 00:04:02\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 341th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10836119)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10836119\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10836119 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10836119\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 342th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10836573)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10836573\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10836573 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10836573\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 343th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10847229)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10847229\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10847229 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10847229\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 344th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10847567)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10847567\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10847567 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10847567\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 345th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10848526)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10848526\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10848526 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10848526\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 346th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10849701)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10849701\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10849701 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10849701\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 347th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10853183)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10853183\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10853183 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10853183\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 348th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10854306)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10854306\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10854306 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10854306\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 349th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10861196)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10861196\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10861196 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10861196\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 350th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10865408)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10865408\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10865408 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10865408\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 351/2190 (16.03%) | Elapsed: 00:00:46 | ETA: 00:04:01\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 351th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10866412)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10866412\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10866412 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10866412\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 352th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10866903)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10866903\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10866903 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10866903\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 353th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10867641)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10867641\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10867641 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10867641\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 354th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10868801)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10868801\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10868801 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10868801\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 355th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10871236)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10871236\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10871236 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10871236\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 356th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10872365)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10872365\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10872365 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10872365\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 357th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10873159)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10873159\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10873159 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10873159\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 358th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10873509)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10873509\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10873509 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10873509\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 359th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10876516)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10876516\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10876516 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10876516\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 360th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10877777)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10877777\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10877777 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10877777\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 361/2190 (16.48%) | Elapsed: 00:00:47 | ETA: 00:04:00\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 361th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10880244)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10880244\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10880244 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10880244\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 362th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10881585)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10881585\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10881585 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10881585\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 363th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10882847)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10882847\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10882847 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10882847\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 364th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10888823)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10888823\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10888823 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10888823\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 365th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10888977)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10888977\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10888977 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10888977\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 366th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10888995)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10888995\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10888995 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10888995\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 367th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10890730)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10890730\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10890730 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10890730\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 368th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10894962)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10894962\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10894962 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10894962\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 369th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10896738)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10896738\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10896738 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10896738\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 370th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10899643)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10899643\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10899643 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10899643\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 371/2190 (16.94%) | Elapsed: 00:00:48 | ETA: 00:03:59\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 371th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10906224)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10906224\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10906224 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10906224\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 372th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10913284)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10913284\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10913284 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10913284\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 373th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10914599)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10914599\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10914599 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10914599\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 374th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10916590)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10916590\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10916590 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10916590\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 375th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10918213)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10918213\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10918213 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10918213\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 376th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10919475)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10919475\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10919475 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10919475\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 377th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10925271)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10925271\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10925271 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10925271\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 378th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10925398)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10925398\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10925398 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10925398\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 379th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10926682)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10926682\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10926682 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10926682\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 380th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10927722)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10927722\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10927722 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10927722\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 381/2190 (17.40%) | Elapsed: 00:00:50 | ETA: 00:03:57\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 381th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10928088)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10928088\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10928088 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10928088\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 382th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10944527)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10944527\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10944527 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10944527\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 383th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10945611)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10945611\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10945611 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10945611\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 384th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10950923)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10950923\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10950923 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10950923\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 385th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10958877)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10958877\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10958877 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10958877\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 386th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10959667)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10959667\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10959667 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10959667\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 387th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10971725)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10971725\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10971725 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10971725\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 388th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10973452)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10973452\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10973452 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10973452\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 389th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10976681)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10976681\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10976681 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10976681\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 390th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10979919)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10979919\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10979919 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10979919\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 391/2190 (17.85%) | Elapsed: 00:00:51 | ETA: 00:03:55\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 391th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10980062)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10980062\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10980062 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10980062\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 392th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10980014)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10980014\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10980014 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10980014\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 393th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10982113)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10982113\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10982113 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10982113\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 394th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10983041)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10983041\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10983041 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10983041\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 395th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10983874)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10983874\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10983874 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10983874\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 396th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10983955)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10983955\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10983955 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10983955\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 397th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10984903)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10984903\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10984903 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10984903\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 398th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10985119)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10985119\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10985119 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10985119\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 399th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10985162)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10985162\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10985162 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10985162\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 400th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10985232)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10985232\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10985232 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10985232\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 401/2190 (18.31%) | Elapsed: 00:00:52 | ETA: 00:03:54\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 401th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10994238)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10994238\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10994238 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10994238\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 402th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10996630)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10996630\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10996630 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10996630\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 403th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10996751)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10996751\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10996751 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10996751\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 404th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10996749)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10996749\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10996749 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10996749\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 405th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10998380)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10998380\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10998380 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10998380\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 406th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC10999606)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC10999606\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC10999606 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC10999606\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 407th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11003869)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11003869\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11003869 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11003869\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 408th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11005142)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11005142\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11005142 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11005142\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 409th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11009252)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11009252\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11009252 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11009252\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 410th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11009983)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11009983\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11009983 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11009983\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 411/2190 (18.77%) | Elapsed: 00:00:53 | ETA: 00:03:52\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 411th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11021587)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11021587\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11021587 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11021587\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 412th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11023937)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11023937\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11023937 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11023937\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 413th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11024353)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11024353\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11024353 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11024353\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 414th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11030506)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11030506\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11030506 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11030506\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 415th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11032436\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 416th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11034945)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11034945\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11034945 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11034945\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 417th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11035624)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11035624\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11035624 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11035624\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 418th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11036294)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11036294\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11036294 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11036294\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 419th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11037540)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11037540\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11037540 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11037540\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 420th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11042309)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11042309\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11042309 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11042309\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 421/2190 (19.22%) | Elapsed: 00:00:54 | ETA: 00:03:50\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 421th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11043077)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11043077\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11043077 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11043077\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 422th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11044291)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11044291\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11044291 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11044291\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 423th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11045878)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11045878\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11045878 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11045878\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 424th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11049136)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11049136\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11049136 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11049136\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 425th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11049304)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11049304\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11049304 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11049304\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 426th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11052574)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11052574\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11052574 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11052574\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 427th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11058328)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11058328\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11058328 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11058328\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 428th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11059185)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11059185\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11059185 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11059185\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 429th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11059271)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11059271\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11059271 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11059271\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 430th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11061925)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11061925\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11061925 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11061925\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 431/2190 (19.68%) | Elapsed: 00:00:56 | ETA: 00:03:49\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 431th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11065001)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11065001\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11065001 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11065001\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 432th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11065022)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11065022\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11065022 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11065022\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 433th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11065257)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11065257\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11065257 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11065257\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 434th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11065703)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11065703\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11065703 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11065703\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 435th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11068064)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11068064\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11068064 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11068064\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 436th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11070398)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11070398\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11070398 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11070398\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 437th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11071432)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11071432\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11071432 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11071432\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 438th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11071554)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11071554\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11071554 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11071554\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 439th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11071577)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11071577\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11071577 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11071577\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 440th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11074339)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11074339\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11074339 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11074339\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 441/2190 (20.14%) | Elapsed: 00:00:57 | ETA: 00:03:47\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 441th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11076546)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11076546\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11076546 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11076546\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 442th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11076581)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11076581\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11076581 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11076581\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 443th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11076587)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11076587\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11076587 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11076587\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 444th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11077411)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11077411\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11077411 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11077411\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 445th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11081546)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11081546\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11081546 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11081546\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 446th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11082197)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11082197\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11082197 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11082197\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 447th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11086901)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11086901\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11086901 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11086901\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 448th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11087195)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11087195\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11087195 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11087195\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 449th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11087501)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11087501\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11087501 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11087501\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 450th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11089003)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11089003\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11089003 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11089003\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 451/2190 (20.59%) | Elapsed: 00:00:58 | ETA: 00:03:47\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 451th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11089696)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11089696\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11089696 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11089696\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 452th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11089773)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11089773\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11089773 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11089773\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 453th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11092597)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11092597\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11092597 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11092597\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 454th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11094420)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11094420\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11094420 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11094420\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 455th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11094870)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11094870\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11094870 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11094870\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 456th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11096318)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11096318\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11096318 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11096318\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 457th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11096330)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11096330\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11096330 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11096330\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 458th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11098751)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11098751\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11098751 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11098751\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 459th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11098881)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11098881\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11098881 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11098881\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 460th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11099140)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11099140\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11099140 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11099140\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 461/2190 (21.05%) | Elapsed: 00:01:00 | ETA: 00:03:45\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 461th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11100843)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11100843\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11100843 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11100843\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 462th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11100897)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11100897\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11100897 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11100897\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 463th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11101435)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11101435\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11101435 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11101435\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 464th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11101463)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11101463\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11101463 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11101463\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 465th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11102269)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11102269\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11102269 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11102269\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 466th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11105346)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11105346\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11105346 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11105346\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 467th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11106394)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11106394\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11106394 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11106394\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 468th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11108776)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11108776\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11108776 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11108776\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 469th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11109119)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11109119\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11109119 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11109119\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 470th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11111684)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11111684\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11111684 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11111684\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 471/2190 (21.51%) | Elapsed: 00:01:01 | ETA: 00:03:43\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 471th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11111906)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11111906\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11111906 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11111906\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 472th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11112958)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11112958\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11112958 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11112958\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 473th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11114225)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11114225\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11114225 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11114225\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 474th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11114236)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11114236\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11114236 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11114236\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 475th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11118306)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11118306\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11118306 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11118306\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 476th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11118681)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11118681\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11118681 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11118681\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 477th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11127995)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11127995\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11127995 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11127995\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 478th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11133839)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11133839\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11133839 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11133839\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 479th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11142156)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11142156\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11142156 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11142156\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 480th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11143375)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11143375\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11143375 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11143375\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 481/2190 (21.96%) | Elapsed: 00:01:02 | ETA: 00:03:43\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 481th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11143965)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11143965\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11143965 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11143965\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 482th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11144322)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11144322\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11144322 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11144322\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 483th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11144614)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11144614\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11144614 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11144614\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 484th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11144676)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11144676\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11144676 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11144676\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 485th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11145170)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11145170\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11145170 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11145170\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 486th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11145390)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11145390\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11145390 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11145390\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 487th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11150309)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11150309\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11150309 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11150309\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 488th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11178567)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11178567\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11178567 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11178567\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 489th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11180851)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11180851\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11180851 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11180851\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 490th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11180854)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11180854\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11180854 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11180854\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 491/2190 (22.42%) | Elapsed: 00:01:04 | ETA: 00:03:41\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 491th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11186631)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11186631\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11186631 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11186631\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 492th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11187912)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11187912\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11187912 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11187912\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 493th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11190273)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11190273\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11190273 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11190273\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 494th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11192077)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11192077\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11192077 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11192077\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 495th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11192082)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11192082\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11192082 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11192082\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 496th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11192733)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11192733\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11192733 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11192733\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 497th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11194087)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11194087\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11194087 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11194087\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 498th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11195283)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11195283\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11195283 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11195283\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 499th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11199635)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11199635\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11199635 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11199635\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 500th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11204300)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11204300\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11204300 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11204300\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 501/2190 (22.88%) | Elapsed: 00:01:05 | ETA: 00:03:40\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 501th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11208446)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11208446\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11208446 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11208446\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 502th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11208617)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11208617\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11208617 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11208617\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 503th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11210445)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11210445\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11210445 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11210445\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 504th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11213071)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11213071\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11213071 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11213071\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 505th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11215552)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11215552\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11215552 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11215552\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 506th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11217418)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11217418\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11217418 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11217418\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 507th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11217612)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11217612\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11217612 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11217612\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 508th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11218415)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11218415\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11218415 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11218415\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 509th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11219798)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11219798\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11219798 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11219798\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 510th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11221658)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11221658\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11221658 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11221658\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 511/2190 (23.33%) | Elapsed: 00:01:06 | ETA: 00:03:38\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 511th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11224332)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11224332\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11224332 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11224332\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 512th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11224609)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11224609\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11224609 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11224609\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 513th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11227559)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11227559\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11227559 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11227559\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 514th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11228020)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11228020\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11228020 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11228020\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 515th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11228499)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11228499\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11228499 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11228499\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 516th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11228799)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11228799\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11228799 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11228799\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 517th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11229404)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11229404\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11229404 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11229404\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 518th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11230164)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11230164\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11230164 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11230164\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 519th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11231994)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11231994\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11231994 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11231994\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 520th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11233968)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11233968\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11233968 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11233968\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 521/2190 (23.79%) | Elapsed: 00:01:07 | ETA: 00:03:37\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 521th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11237088)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11237088\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11237088 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11237088\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 522th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11239576)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11239576\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11239576 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11239576\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 523th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11240079)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11240079\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11240079 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11240079\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 524th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11245855)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11245855\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11245855 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11245855\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 525th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11246468)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11246468\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11246468 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11246468\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 526th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11246497)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11246497\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11246497 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11246497\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 527th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11247674)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11247674\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11247674 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11247674\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 528th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11247676)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11247676\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11247676 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11247676\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 529th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11247716)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11247716\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11247716 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11247716\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 530th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11250264)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11250264\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11250264 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11250264\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 531/2190 (24.25%) | Elapsed: 00:01:09 | ETA: 00:03:35\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 531th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11252419)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11252419\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11252419 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11252419\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 532th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11254464)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11254464\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11254464 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11254464\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 533th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11254630)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11254630\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11254630 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11254630\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 534th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11255727)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11255727\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11255727 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11255727\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 535th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11257892)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11257892\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11257892 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11257892\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 536th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11258126)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11258126\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11258126 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11258126\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 537th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11258338)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11258338\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11258338 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11258338\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 538th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11258485)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11258485\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11258485 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11258485\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 539th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11262832)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11262832\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11262832 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11262832\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 540th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11265146)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11265146\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11265146 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11265146\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 541/2190 (24.70%) | Elapsed: 00:01:10 | ETA: 00:03:34\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 541th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11267505)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11267505\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11267505 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11267505\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 542th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11271286)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11271286\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11271286 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11271286\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 543th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11271532)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11271532\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11271532 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11271532\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 544th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11271596)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11271596\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11271596 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11271596\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 545th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11271822)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11271822\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11271822 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11271822\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 546th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11275870)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11275870\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11275870 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11275870\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 547th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11278172)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11278172\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11278172 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11278172\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 548th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11280240)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11280240\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11280240 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11280240\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 549th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11280964)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11280964\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11280964 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11280964\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 550th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11282107)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11282107\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11282107 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11282107\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 551/2190 (25.16%) | Elapsed: 00:01:12 | ETA: 00:03:34\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 551th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11291244)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11291244\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11291244 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11291244\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 552th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11291675)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11291675\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11291675 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11291675\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 553th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11294446)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11294446\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11294446 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11294446\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 554th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11297156)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11297156\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11297156 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11297156\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 555th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11297319)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11297319\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11297319 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11297319\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 556th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11300466)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11300466\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11300466 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11300466\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 557th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11300572)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11300572\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11300572 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11300572\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 558th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11302177)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11302177\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11302177 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11302177\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 559th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11303399)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11303399\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11303399 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11303399\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 560th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11306111)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11306111\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11306111 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11306111\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 561/2190 (25.62%) | Elapsed: 00:01:13 | ETA: 00:03:32\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 561th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11309927)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11309927\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11309927 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11309927\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 562th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11312442)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11312442\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11312442 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11312442\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 563th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11313277)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11313277\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11313277 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11313277\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 564th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11319208)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11319208\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11319208 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11319208\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 565th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11320747)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11320747\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11320747 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11320747\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 566th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11321170)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11321170\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11321170 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11321170\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 567th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11323136)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11323136\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11323136 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11323136\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 568th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11325151)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11325151\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11325151 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11325151\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 569th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11326155)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11326155\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11326155 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11326155\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 570th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11326359)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11326359\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11326359 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11326359\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 571/2190 (26.07%) | Elapsed: 00:01:14 | ETA: 00:03:31\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 571th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11326360)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11326360\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11326360 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11326360\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 572th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11326371)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11326371\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11326371 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11326371\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 573th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11328419)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11328419\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11328419 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11328419\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 574th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11330236)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11330236\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11330236 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11330236\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 575th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11333280)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11333280\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11333280 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11333280\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 576th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11338827)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11338827\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11338827 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11338827\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 577th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11338834)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11338834\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11338834 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11338834\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 578th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11339421)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11339421\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11339421 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11339421\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 579th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11339621)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11339621\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11339621 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11339621\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 580th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11342913)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11342913\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11342913 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11342913\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 581/2190 (26.53%) | Elapsed: 00:01:16 | ETA: 00:03:30\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 581th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11347063)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11347063\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11347063 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11347063\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 582th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11347662)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11347662\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11347662 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11347662\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 583th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11349943)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11349943\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11349943 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11349943\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 584th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11350008)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11350008\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11350008 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11350008\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 585th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11350143)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11350143\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11350143 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11350143\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 586th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11353479)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11353479\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11353479 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11353479\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 587th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11358483)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11358483\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11358483 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11358483\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 588th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11361112)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11361112\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11361112 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11361112\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 589th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11364851)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11364851\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11364851 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11364851\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 590th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11365968)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11365968\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11365968 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11365968\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 591/2190 (26.99%) | Elapsed: 00:01:17 | ETA: 00:03:29\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 591th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11369196)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11369196\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11369196 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11369196\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 592th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11370495)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11370495\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11370495 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11370495\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 593th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11370573)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11370573\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11370573 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11370573\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 594th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11370593)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11370593\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11370593 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11370593\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 595th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11372374)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11372374\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11372374 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11372374\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 596th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11375158)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11375158\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11375158 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11375158\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 597th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11377155)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11377155\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11377155 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11377155\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 598th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11377741)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11377741\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11377741 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11377741\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 599th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11380310)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11380310\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11380310 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11380310\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 600th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11383014)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11383014\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11383014 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11383014\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 601/2190 (27.44%) | Elapsed: 00:01:18 | ETA: 00:03:28\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 601th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11383059)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11383059\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11383059 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11383059\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 602th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11383294)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11383294\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11383294 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11383294\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 603th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11384960)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11384960\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11384960 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11384960\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 604th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11385183)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11385183\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11385183 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11385183\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 605th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11385559)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11385559\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11385559 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11385559\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 606th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11388281)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11388281\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11388281 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11388281\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 607th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11393344)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11393344\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11393344 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11393344\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 608th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11393413)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11393413\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11393413 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11393413\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 609th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11394787)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11394787\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11394787 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11394787\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 610th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11395524)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11395524\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11395524 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11395524\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 611/2190 (27.90%) | Elapsed: 00:01:20 | ETA: 00:03:27\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 611th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11398495)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11398495\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11398495 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11398495\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 612th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11399249)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11399249\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11399249 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11399249\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 613th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11399277)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11399277\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11399277 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11399277\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 614th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11401852)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11401852\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11401852 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11401852\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 615th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11402992)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11402992\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11402992 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11402992\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 616th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11415437)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11415437\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11415437 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11415437\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 617th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11415491)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11415491\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11415491 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11415491\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 618th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11418138)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11418138\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11418138 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11418138\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 619th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11419020)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11419020\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11419020 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11419020\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 620th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11420198)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11420198\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11420198 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11420198\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 621/2190 (28.36%) | Elapsed: 00:01:21 | ETA: 00:03:25\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 621th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11421795)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11421795\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11421795 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11421795\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 622th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11429458)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11429458\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11429458 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11429458\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 623th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11429704)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11429704\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11429704 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11429704\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 624th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11434165)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11434165\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11434165 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11434165\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 625th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11436371)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11436371\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11436371 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11436371\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 626th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11438217)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11438217\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11438217 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11438217\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 627th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11439059)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11439059\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11439059 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11439059\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 628th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11439699)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11439699\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11439699 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11439699\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 629th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11440320)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11440320\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11440320 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11440320\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 630th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11440915)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11440915\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11440915 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 400 - INFO - Loading results from cache for process ID: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11440915\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 631/2190 (28.81%) | Elapsed: 00:01:22 | ETA: 00:03:24\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 631th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11442147)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11442147\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11442147 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 634 - INFO - Fetching data for pmc11442147\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 637 - INFO - Fetching data from scripts/exp_input/Local_fetched_data_1.parquet\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 438 - INFO - Raw xml data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11442147 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 452 - INFO - Skipping raw HTML/XML saving.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 458 - INFO - Using XMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 52 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 24 - INFO - Initializing xmlRetriever\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 466 - INFO - No additional data provided. Parsing raw data only.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 147 - INFO - Function call: parse_data(api_data(<class 'str'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11442147, additional_data, XML)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 162 - INFO - api_data converted to lxml element\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 466 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 153 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 154 - INFO - Consider migrating this function to the BaseRetriever class.\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 69 - INFO - Loading target sections for section name: supplementary_material_sections\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 477 - INFO - Found 4 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x15d6c60c0>, <Element supplementary-material at 0x156afa740>, <Element supplementary-material at 0x15d435400>, <Element supplementary-material at 0x15d434900>]\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 485 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x15d6c60c0>, <Element supplementary-material at 0x156afa740>, <Element supplementary-material at 0x15d435400>, <Element supplementary-material at 0x15d434900>]}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 574 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref sup1: To demonstrate scPanel's scalability to large scRNA-seq datasets with hundreds of patients, we applied the method to a dSSc skin dataset containing 35,417 cells from 91 donors (41 controls, 50 SSc patients; Fig. 2a [rid=f2] , Supplementary Fig. 1a [this file] ). \n",
      "All five classifiers trained with selected genes and the Fibro_LGR5 cells consistently showed good performance ( Supplementary Fig. 1d [this file] ). Overall, the patient-level classification accuracy is 0.86 (12 out of 14 patients) in the testing set ( Fig. 2f [rid=f2] , Supplementary Fig. 1c [this file] ). \n",
      "scPanel identified known and potentially new markers of dSSc within the LGR5-expressing fibroblasts ( Fig. 2e [rid=f2] , Supplementary Fig. 1b [this file] ). \n",
      "To determine whether scPanel can classify patients with very heterogeneous diseases such as cancer, we applied the method to a CRC dataset comprising 365,913 cells from 7 distinct anatomical regions of the colon from 38 iCMS2 and 25 iCMS3 patients [ 13 [rid=ref13] ] ( Fig. 3a [rid=f3] , Supplementary Fig. 2a [this file] ). These genes showed significant gene expression differences (adjusted -value <.05) between iCMS2 and iCMS3 subtype patients across both training and testing set ( Supplementary Fig. 2b [this file] , Fig. 3e [rid=f3] ). Twelve out of 13 patients (92.3% accuracy) were predicted confidently with AUC scores significantly different from 0.5 (random guess; Supplementary Fig. 2c [this file] and Fig. 3f [rid=f3] ). All five classifiers consistently showed the same good performance ( Supplementary Fig. 2d [this file] ). \n",
      "To investigate the wider applicability of scPanel, we applied the method to a severe COVID-19 dataset with 39,196 PBMCs from 13 patients (6 controls, 7 COVID-19 patients) [ 14 [rid=ref14] ] ( Fig. 4a [rid=f4] , Supplementary Fig. 3a [this file] ). These six genes ( , , , , , and ) showed consistent gene expression patterns between training and testing set ( Supplementary Fig. 3b [this file] , Fig. 4e [rid=f4] ). All patients were predicted confidently with AUC scores significantly different from 0.5 (random guess; Supplementary Fig. 3c [this file] ). All five classifiers consistently showed the same good performance ( Supplementary Fig. 3d [this file] ). \n",
      "To demonstrate the generalizability of scPanel across different datasets, we applied the severe COVID-19 classifiers to an independent su2020covid dataset ( Supplementary Fig. 4a and b [this file] ). \n",
      "Interestingly, the GAT classifier showed an inherent robustness to batch effects, consistently maintaining 100% accuracy, irrespective of batch removal integration ( Supplementary Fig. 4c and d [this file] ). \n",
      "To assess the specificity of our severe COVID-19 classifiers (LR, RF, SVM, kNN, and GAT), we incorporated a PBMC dataset, referred to as lee2020cov_flu, from patients with various COVID-19 severities and including influenza cases ( Supplementary Fig. 5a–c [this file] ). After integration, all five classifiers demonstrated a 100% accuracy in predicting healthy individuals and severe COVID-19 patients within the lee2020cov_flu dataset ( Supplementary Fig. 5d and e [this file] ). However, LR, RF, SVM, and kNN classifiers demonstrated a lack of specificity for severe COVID-19, incorrectly classifying all severe influenza cases as severe COVID-19 with 100% prediction probabilities ( Supplementary Fig. 5e [this file] ). \n",
      "To complete our benchmark, we performed a comparative analysis of computational cost (runtime) of scPanel and the other SOTA gene selection methods, COMET and ActiveSVM, across the three datasets ( Supplementary Fig. 6 [this file] ). \n",
      "In the severe COVID-19 dataset (wilk2020covid), cell-level analysis consistently identified CD14+ monocytes as the most responsive cell type across all subsets ( Supplementary Fig. 7a [this file] ), with some variation in the identified genes ( Supplementary Fig. 7b [this file] ) but a smaller set of stably selected genes ( Supplementary Fig. 7c [this file] ). Despite this, predictions for all patients in the test set remained accurate across all subsets (lower bound AUC > 0.85), albeit with a marginal reduction in AUC scores for subsets with size smaller than 70% ( Supplementary Fig. 7d [this file] ). Further cell-level power analysis within the dSSc dataset (gur2022ssc) identified LGR5+ fibroblasts as the most responsive cell type when more than 13,141 cells (60% of the total) were used ( Supplementary Fig. 8a [this file] ). Above this number of cells, there were no significant differences in the number of selected genes with similar sets of genes being identified consistently ( = .18, Kruskal–Wallis test) and the patient-level predictions remain the same for LGR5+ fibroblasts ( Supplementary Fig. 8b–d [this file] ). \n",
      "It was observed that the cell type responsiveness score (AUC) for LGR5+ fibroblasts began to plateau at 40% (33 samples) of the full sample size ( Supplementary Fig. 9a [this file] ). Correspondingly, the gene selection started to exhibit large variations when the sample sizes were smaller than 40% ( Supplementary Fig. 9b and c [this file] ). Despite the variability in gene selection, the top 5 most important genes ( , , , , and ) were selected with frequency at least 65% and the patient-level prediction remains stable across all subsets with size ≥40% ( Supplementary Fig. 9d [this file] ). Computationally, both peak memory usage and processing time displayed an almost linear increase with respect to different sizes of the input cells and samples ( Supplementary Figs 7e and f [this file] , 8e and f, and\u0001[this file] 9e and f [this file] ). \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref sup2: We developed a machine learning based tool named scPanel to identify a sparse gene panel for patient-level classification of perturbations such as disease severity and treatment responses from scRNA-seq data ( Fig. 1a [rid=f1] , see Supplementary methods [this file] for technical details). scPanel comprises three key steps: (i) identifying the most responsive cell type to perturbation; (ii) selecting a minimal number of informative genes from this cell type using a wrapper-based feature selection method [ 17 [rid=ref17] ]; (iii) training multiple machine learning models with the selected cell types and genes for consensus patient-level classification. \n",
      "Inspired by the findPC tool [ 19 [rid=ref19] ], the minimal number of genes is automatically determined using the perpendicular line method, which corresponds to the gene count just before a rapid decline in classification accuracy (see Supplementary methods [this file] ). \n",
      "Patient-level classification is determined by aggregating cell prediction probabilities using the AUC score, with a corresponding -value to quantify significance (see Supplementary methods [this file] ). \n",
      "Each cell's gene expression is normalized by the total Unique Molecular Identifier (UMI) counts using standard pipelines (see Supplementary methods [this file] ). \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref sup3: We applied scPanel on five scRNA-seq datasets: a dSSc skin dataset (gur2022ssc), a CRC dataset (joanito2022crc), two severe COVID-19 PBMC datasets (wilk2020covid, su2020covid), and an infection PBMC datasets (lee2020cov_flu; Supplementary Table 1 [this file] ). \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref sup4: Details on the parameters used for each model are provided in Supplementary Table 2 [this file] . \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 263 - INFO - Extracting links from full XML content.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 301 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 309 - INFO - Content length: 130902\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 326 - INFO - Prompt messages total length: 33479 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 332 - INFO - Prompt ID: gemini-2.5-flash-0-12403f1635c4241cc16a25c6cd787cc5e1d999d92107ad5aaf9b50f9cd9dde64\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 351 - INFO - Requesting datasets from content using model: gemini-2.5-flash, temperature: 0, messages length: 33479 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-e4060d93-1c53-4e34-a49a-2e115afe3589\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"wilk2020covid\\\",\\n    \\\"repository_reference\\\": \\\"https://atlas.fredhutch.org/fredhutch/covid\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"su2020covid\\\",\\n    \\\"repository_reference\\\": \\\"https://atlas.fredhutch.org/fredhutch/covid\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"lee2020cov_flu\\\",\\n    \\\"repository_reference\\\": \\\"https://atlas.fredhutch.org/fredhutch/covid\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE195452\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE195452\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn26844071\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn26844071/\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"wilk2020covid\\\",\\n    \\\"repository_reference\\\": \\\"https://atlas.fredhutch.org/fredhutch/covid\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"su2020covid\\\",\\n    \\\"repository_reference\\\": \\\"https://atlas.fredhutch.org/fredhutch/covid\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"lee2020cov_flu\\\",\\n    \\\"repository_reference\\\": \\\"https://atlas.fredhutch.org/fredhutch/covid\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE195452\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE195452\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn26844071\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn26844071/\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1758209976,\n",
      "    \"model\": \"gemini-2.5-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 38970,\n",
      "        \"completion_tokens\": 263,\n",
      "        \"total_tokens\": 41041,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 1808,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 664 - INFO - Deduplicating response with 5 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: wilk2020covid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: su2020covid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: lee2020cov_flu\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: GSE195452\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn26844071\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'wilk2020covid', 'repository_reference': 'https://atlas.fredhutch.org/fredhutch/covid', 'dataset_id': 'wilk2020covid'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: wilk2020covid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: wilk2020covid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://atlas.fredhutch.org/fredhutch/covid, identifier: wilk2020covid, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://atlas.fredhutch.org/fredhutch/covid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier wilk2020covid for https://atlas.fredhutch.org/fredhutch/covid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier wilk2020covid for https://atlas.fredhutch.org/fredhutch/covid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier wilk2020covid for https://atlas.fredhutch.org/fredhutch/covid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier wilk2020covid for https://atlas.fredhutch.org/fredhutch/covid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier wilk2020covid for https://atlas.fredhutch.org/fredhutch/covid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier wilk2020covid for https://atlas.fredhutch.org/fredhutch/covid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://atlas.fredhutch.org/fredhutch/covid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: wilk2020covid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: wilk2020covid, atlas.fredhutch.org, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: wilk2020covid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: atlas.fredhutch.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'wilk2020covid', 'data_repository': 'atlas.fredhutch.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'su2020covid', 'repository_reference': 'https://atlas.fredhutch.org/fredhutch/covid', 'dataset_id': 'su2020covid'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: su2020covid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: su2020covid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://atlas.fredhutch.org/fredhutch/covid, identifier: su2020covid, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://atlas.fredhutch.org/fredhutch/covid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier su2020covid for https://atlas.fredhutch.org/fredhutch/covid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier su2020covid for https://atlas.fredhutch.org/fredhutch/covid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier su2020covid for https://atlas.fredhutch.org/fredhutch/covid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier su2020covid for https://atlas.fredhutch.org/fredhutch/covid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier su2020covid for https://atlas.fredhutch.org/fredhutch/covid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier su2020covid for https://atlas.fredhutch.org/fredhutch/covid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://atlas.fredhutch.org/fredhutch/covid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: su2020covid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: su2020covid, atlas.fredhutch.org, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: su2020covid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: atlas.fredhutch.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'su2020covid', 'data_repository': 'atlas.fredhutch.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'lee2020cov_flu', 'repository_reference': 'https://atlas.fredhutch.org/fredhutch/covid', 'dataset_id': 'lee2020cov_flu'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: lee2020cov_flu\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: lee2020cov_flu\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://atlas.fredhutch.org/fredhutch/covid, identifier: lee2020cov_flu, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://atlas.fredhutch.org/fredhutch/covid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier lee2020cov_flu for https://atlas.fredhutch.org/fredhutch/covid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier lee2020cov_flu for https://atlas.fredhutch.org/fredhutch/covid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier lee2020cov_flu for https://atlas.fredhutch.org/fredhutch/covid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier lee2020cov_flu for https://atlas.fredhutch.org/fredhutch/covid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier lee2020cov_flu for https://atlas.fredhutch.org/fredhutch/covid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier lee2020cov_flu for https://atlas.fredhutch.org/fredhutch/covid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://atlas.fredhutch.org/fredhutch/covid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: lee2020cov_flu\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: lee2020cov_flu, atlas.fredhutch.org, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: lee2020cov_flu\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: atlas.fredhutch.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'lee2020cov_flu', 'data_repository': 'atlas.fredhutch.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'GSE195452', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE195452', 'dataset_id': 'GSE195452'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE195452\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: GSE195452\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE195452, identifier: GSE195452, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE195452\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE195452 for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE195452\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier GSE195452 for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE195452\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1157 - INFO - Found id_pattern match for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE195452 in GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE195452\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE195452\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: GSE195452\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE195452\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE195452\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE195452\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: GSE195452\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE195452\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: GSE195452, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE195452\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: GSE195452\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE195452\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'GSE195452', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE195452', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn26844071', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn26844071/', 'dataset_id': 'syn26844071'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn26844071\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn26844071\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn26844071/, identifier: syn26844071, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn26844071/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn26844071 for https://www.synapse.org/#!Synapse:syn26844071/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn26844071 for https://www.synapse.org/#!Synapse:syn26844071/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn26844071 for https://www.synapse.org/#!Synapse:syn26844071/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn26844071/ in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn26844071/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn26844071/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn26844071/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn26844071\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn26844071\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn26844071\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn26844071/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn26844071\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn26844071\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn26844071, synapse.org, https://www.synapse.org/#!Synapse:syn26844071/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn26844071\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn26844071/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn26844071', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn26844071/', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 278 - INFO - Augmented dataset links: [{'dataset_identifier': 'wilk2020covid', 'data_repository': 'atlas.fredhutch.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'su2020covid', 'data_repository': 'atlas.fredhutch.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'lee2020cov_flu', 'data_repository': 'atlas.fredhutch.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'GSE195452', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE195452', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn26844071', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn26844071/', 'citation_type': 'n/a'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1191 - INFO - Fetching metadata for 5 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: wilk2020covid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: atlas.fredhutch.org, identifier: wilk2020covid, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier wilk2020covid for atlas.fredhutch.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier wilk2020covid for atlas.fredhutch.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier wilk2020covid for atlas.fredhutch.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier wilk2020covid for atlas.fredhutch.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier wilk2020covid for atlas.fredhutch.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier wilk2020covid for atlas.fredhutch.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: atlas.fredhutch.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL atlas.fredhutch.org may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for wilk2020covid in atlas.fredhutch.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 1 with repo: atlas.fredhutch.org and accession_id: wilk2020covid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository atlas.fredhutch.org unknown in Ontology. Skipping dataset page 1.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: su2020covid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: atlas.fredhutch.org, identifier: su2020covid, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier su2020covid for atlas.fredhutch.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier su2020covid for atlas.fredhutch.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier su2020covid for atlas.fredhutch.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier su2020covid for atlas.fredhutch.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier su2020covid for atlas.fredhutch.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier su2020covid for atlas.fredhutch.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: atlas.fredhutch.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL atlas.fredhutch.org may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for su2020covid in atlas.fredhutch.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 2 with repo: atlas.fredhutch.org and accession_id: su2020covid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository atlas.fredhutch.org unknown in Ontology. Skipping dataset page 2.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: lee2020cov_flu\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: atlas.fredhutch.org, identifier: lee2020cov_flu, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier lee2020cov_flu for atlas.fredhutch.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier lee2020cov_flu for atlas.fredhutch.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier lee2020cov_flu for atlas.fredhutch.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier lee2020cov_flu for atlas.fredhutch.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier lee2020cov_flu for atlas.fredhutch.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier lee2020cov_flu for atlas.fredhutch.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: atlas.fredhutch.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL atlas.fredhutch.org may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for lee2020cov_flu in atlas.fredhutch.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 3 with repo: atlas.fredhutch.org and accession_id: lee2020cov_flu\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository atlas.fredhutch.org unknown in Ontology. Skipping dataset page 3.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: GSE195452\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: geo, identifier: GSE195452, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE195452 for geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for GSE195452 in geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 4 with repo: geo and accession_id: GSE195452\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE195452\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 4: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn26844071\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn26844071, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn26844071 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn26844071 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn26844071 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn26844071 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 5 with repo: synapse.org and accession_id: syn26844071\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn26844071\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 5: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1277 - INFO - Updated datasets len: 5\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 287 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 9, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'source_url', 'download_link',\n",
      "       'title', 'content_type', 'id', 'caption', 'description',\n",
      "       'source_section', 'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 500 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 506 - INFO - XML element classification not needed. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1012 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11442147\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1020 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11442147\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 632th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11442384)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11442384\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11442384 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 634 - INFO - Fetching data for pmc11442384\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 637 - INFO - Fetching data from scripts/exp_input/Local_fetched_data_1.parquet\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 438 - INFO - Raw xml data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11442384 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 452 - INFO - Skipping raw HTML/XML saving.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 458 - INFO - Using XMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 52 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 24 - INFO - Initializing xmlRetriever\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 466 - INFO - No additional data provided. Parsing raw data only.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 147 - INFO - Function call: parse_data(api_data(<class 'str'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11442384, additional_data, XML)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 162 - INFO - api_data converted to lxml element\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 466 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 153 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 154 - INFO - Consider migrating this function to the BaseRetriever class.\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 69 - INFO - Loading target sections for section name: supplementary_material_sections\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 477 - INFO - Found 2 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x15d739c80>, <Element supplementary-material at 0x15d391680>]\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 485 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x15d739c80>, <Element supplementary-material at 0x15d391680>]}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 574 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref SM1: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref SM2: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 263 - INFO - Extracting links from full XML content.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 301 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 309 - INFO - Content length: 186431\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 326 - INFO - Prompt messages total length: 47361 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 332 - INFO - Prompt ID: gemini-2.5-flash-0-b97ef697201fe106ef339ce8918ba1ff1e0fe746015a015d0622dc83e25f0a9a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 351 - INFO - Requesting datasets from content using model: gemini-2.5-flash, temperature: 0, messages length: 47361 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-6ffa8e9e-6da7-42d0-ac6c-ac6e4bc745ac\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn22277191\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn22277191\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn26051803\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn26051803\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn24983526\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn24983526\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn5201007\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn5201007\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn4650265\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn4650265\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn3505720\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn3505720\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://twc-stanford.shinyapps.io/aging_plasma_proteome_v2/\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn22277191\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn22277191\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn26051803\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn26051803\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn24983526\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn24983526\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn5201007\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn5201007\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn4650265\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn4650265\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn3505720\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn3505720\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://twc-stanford.shinyapps.io/aging_plasma_proteome_v2/\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1758210003,\n",
      "    \"model\": \"gemini-2.5-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 58477,\n",
      "        \"completion_tokens\": 380,\n",
      "        \"total_tokens\": 63315,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 4458,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 664 - INFO - Deduplicating response with 7 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn22277191\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn26051803\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn24983526\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn5201007\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn4650265\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn3505720\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn22277191', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn22277191', 'dataset_id': 'syn22277191'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn22277191\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn22277191\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn22277191, identifier: syn22277191, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn22277191\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn22277191 for https://www.synapse.org/#!Synapse:syn22277191\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn22277191 for https://www.synapse.org/#!Synapse:syn22277191\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn22277191 for https://www.synapse.org/#!Synapse:syn22277191\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn22277191 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn22277191\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn22277191\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn22277191\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn22277191\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn22277191\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn22277191\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn22277191\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn22277191\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn22277191\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn22277191, synapse.org, https://www.synapse.org/#!Synapse:syn22277191\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn22277191\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn22277191\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn22277191', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn22277191', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn26051803', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn26051803', 'dataset_id': 'syn26051803'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn26051803\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn26051803\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn26051803, identifier: syn26051803, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn26051803\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn26051803 for https://www.synapse.org/#!Synapse:syn26051803\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn26051803 for https://www.synapse.org/#!Synapse:syn26051803\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn26051803 for https://www.synapse.org/#!Synapse:syn26051803\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn26051803 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn26051803\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn26051803\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn26051803\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn26051803\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn26051803\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn26051803\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn26051803\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn26051803\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn26051803\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn26051803, synapse.org, https://www.synapse.org/#!Synapse:syn26051803\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn26051803\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn26051803\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn26051803', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn26051803', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn24983526', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn24983526', 'dataset_id': 'syn24983526'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn24983526\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn24983526\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn24983526, identifier: syn24983526, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn24983526\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn24983526 for https://www.synapse.org/#!Synapse:syn24983526\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn24983526 for https://www.synapse.org/#!Synapse:syn24983526\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn24983526 for https://www.synapse.org/#!Synapse:syn24983526\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn24983526 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn24983526\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn24983526\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn24983526\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn24983526\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn24983526\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn24983526\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn24983526\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn24983526\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn24983526\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn24983526, synapse.org, https://www.synapse.org/#!Synapse:syn24983526\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn24983526\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn24983526\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn24983526', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn24983526', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn5201007', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn5201007', 'dataset_id': 'syn5201007'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn5201007\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn5201007\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn5201007, identifier: syn5201007, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn5201007\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn5201007 for https://www.synapse.org/#!Synapse:syn5201007\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn5201007 for https://www.synapse.org/#!Synapse:syn5201007\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn5201007 for https://www.synapse.org/#!Synapse:syn5201007\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn5201007 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn5201007\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn5201007\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn5201007\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn5201007\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn5201007\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn5201007\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn5201007\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn5201007\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn5201007\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn5201007, synapse.org, https://www.synapse.org/#!Synapse:syn5201007\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn5201007\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn5201007\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn5201007', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn5201007', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn4650265', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn4650265', 'dataset_id': 'syn4650265'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn4650265\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn4650265\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn4650265, identifier: syn4650265, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn4650265\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn4650265 for https://www.synapse.org/#!Synapse:syn4650265\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn4650265 for https://www.synapse.org/#!Synapse:syn4650265\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn4650265 for https://www.synapse.org/#!Synapse:syn4650265\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn4650265 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn4650265\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn4650265\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn4650265\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn4650265\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn4650265\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn4650265\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn4650265\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn4650265\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn4650265\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn4650265, synapse.org, https://www.synapse.org/#!Synapse:syn4650265\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn4650265\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn4650265\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn4650265', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn4650265', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn3505720', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn3505720', 'dataset_id': 'syn3505720'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn3505720\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn3505720\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn3505720, identifier: syn3505720, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn3505720\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3505720 for https://www.synapse.org/#!Synapse:syn3505720\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn3505720 for https://www.synapse.org/#!Synapse:syn3505720\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3505720 for https://www.synapse.org/#!Synapse:syn3505720\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn3505720 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn3505720\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn3505720\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn3505720\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn3505720\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn3505720\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn3505720\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn3505720\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn3505720\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn3505720\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn3505720, synapse.org, https://www.synapse.org/#!Synapse:syn3505720\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn3505720\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn3505720\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn3505720', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn3505720', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://twc-stanford.shinyapps.io/aging_plasma_proteome_v2/', 'dataset_id': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://twc-stanford.shinyapps.io/aging_plasma_proteome_v2/, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://twc-stanford.shinyapps.io/aging_plasma_proteome_v2/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://twc-stanford.shinyapps.io/aging_plasma_proteome_v2/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'twc-stanford.shinyapps.io', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 278 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn22277191', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn22277191', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn26051803', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn26051803', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn24983526', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn24983526', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn5201007', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn5201007', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn4650265', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn4650265', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn3505720', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn3505720', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'twc-stanford.shinyapps.io', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1191 - INFO - Fetching metadata for 7 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn22277191\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn22277191, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn22277191 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn22277191 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn22277191 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn22277191 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn22277191\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn22277191\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn26051803\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn26051803, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn26051803 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn26051803 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn26051803 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn26051803 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn26051803\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn26051803\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn24983526\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn24983526, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn24983526 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn24983526 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn24983526 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn24983526 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 3 with repo: synapse.org and accession_id: syn24983526\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn24983526\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 3: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn5201007\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn5201007, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn5201007 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn5201007 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn5201007 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn5201007 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 4 with repo: synapse.org and accession_id: syn5201007\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn5201007\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 4: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn4650265\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn4650265, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn4650265 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn4650265 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn4650265 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn4650265 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 5 with repo: synapse.org and accession_id: syn4650265\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn4650265\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 5: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn3505720\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn3505720, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3505720 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn3505720 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3505720 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn3505720 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 6 with repo: synapse.org and accession_id: syn3505720\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn3505720\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 6: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 7: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1277 - INFO - Updated datasets len: 7\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 287 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 9, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'source_url', 'download_link',\n",
      "       'title', 'content_type', 'id', 'caption', 'description',\n",
      "       'source_section', 'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 500 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 506 - INFO - XML element classification not needed. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1012 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11442384\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1020 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11442384\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 633th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11443687)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11443687\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11443687 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 634 - INFO - Fetching data for pmc11443687\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 637 - INFO - Fetching data from scripts/exp_input/Local_fetched_data_1.parquet\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 438 - INFO - Raw xml data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11443687 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 452 - INFO - Skipping raw HTML/XML saving.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 458 - INFO - Using XMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 52 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 24 - INFO - Initializing xmlRetriever\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 466 - INFO - No additional data provided. Parsing raw data only.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 147 - INFO - Function call: parse_data(api_data(<class 'str'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11443687, additional_data, XML)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 162 - INFO - api_data converted to lxml element\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 466 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 153 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 154 - INFO - Consider migrating this function to the BaseRetriever class.\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 69 - INFO - Loading target sections for section name: supplementary_material_sections\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 477 - INFO - Found 2 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x15d686740>, <Element supplementary-material at 0x15d684440>]\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 485 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x15d686740>, <Element supplementary-material at 0x15d684440>]}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 574 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM1: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM2: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 263 - INFO - Extracting links from full XML content.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 301 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 309 - INFO - Content length: 110079\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 326 - INFO - Prompt messages total length: 28273 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 332 - INFO - Prompt ID: gemini-2.5-flash-0-549cb360821980b6b3955c15a0a38fb745e62c88ca110f084c686850e2f1bb3e\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 351 - INFO - Requesting datasets from content using model: gemini-2.5-flash, temperature: 0, messages length: 28273 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-cd5f83d1-6556-4420-ad66-dc33bd126c62\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn56849053\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn56849053\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.7303/syn56849053\\\",\\n    \\\"repository_reference\\\": \\\"https://doi.org/10.7303/syn56849053\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn56849053\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn56849053\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.7303/syn56849053\\\",\\n    \\\"repository_reference\\\": \\\"https://doi.org/10.7303/syn56849053\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1758210018,\n",
      "    \"model\": \"gemini-2.5-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 37190,\n",
      "        \"completion_tokens\": 127,\n",
      "        \"total_tokens\": 39159,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 1842,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 664 - INFO - Deduplicating response with 2 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: 10.7303/syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn56849053', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn56849053', 'dataset_id': 'syn56849053'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn56849053, identifier: syn56849053, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn56849053 for https://www.synapse.org/#!Synapse:syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn56849053 for https://www.synapse.org/#!Synapse:syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn56849053 for https://www.synapse.org/#!Synapse:syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn56849053 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn56849053, synapse.org, https://www.synapse.org/#!Synapse:syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn56849053', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn56849053', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': '10.7303/syn56849053', 'repository_reference': 'https://doi.org/10.7303/syn56849053', 'dataset_id': '10.7303/syn56849053'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: 10.7303/syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 607 - INFO - Setting dataset_webpage to DOI URL value: https://doi.org/10.7303/syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: 10.7303/syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://doi.org/10.7303/syn56849053, identifier: 10.7303/syn56849053, dataset_page: https://doi.org/10.7303/syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://doi.org/10.7303/syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.7303/syn56849053 for https://doi.org/10.7303/syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier 10.7303/syn56849053 for https://doi.org/10.7303/syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier 10.7303/syn56849053 for https://doi.org/10.7303/syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier 10.7303/syn56849053 for https://doi.org/10.7303/syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier 10.7303/syn56849053 for https://doi.org/10.7303/syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1157 - INFO - Found id_pattern match for https://doi.org/10.7303/syn56849053 in 10\\.\\d{4,9}/[^\\s,]+\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://doi.org/10.7303/syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: 10.7303/syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: 10.7303/syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: 10\\.\\d{4,9}/[^\\s,]+ to value: 10.7303/syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://doi.org/__ID__ to value: https://doi.org/10.7303/syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: 10\\.\\d{4,9}/[^\\s,]+ to value: https://doi.org/10.7303/syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(10\\.\\d{4,9}/[^\\s,]+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(10\\.\\d{4,9}/[^\\s,]+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: 10.7303/syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: 10\\.\\d{4,9}/[^\\s,]+ to value: 10.7303/syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: 10.7303/syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://doi.org/10.7303/syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: 10.7303/syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: 10.7303/syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: 10.7303/syn56849053, doi.org, https://doi.org/10.7303/syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: 10.7303/syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: doi.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://doi.org/10.7303/syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': '10.7303/syn56849053', 'data_repository': 'doi.org', 'dataset_webpage': 'https://doi.org/10.7303/syn56849053', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 278 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn56849053', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn56849053', 'citation_type': 'n/a'}, {'dataset_identifier': '10.7303/syn56849053', 'data_repository': 'doi.org', 'dataset_webpage': 'https://doi.org/10.7303/syn56849053', 'citation_type': 'n/a'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1191 - INFO - Fetching metadata for 2 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn56849053, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn56849053 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn56849053 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn56849053 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn56849053 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: 10.7303/syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: doi.org, identifier: 10.7303/syn56849053, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.7303/syn56849053 for doi.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier 10.7303/syn56849053 for doi.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier 10.7303/syn56849053 for doi.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier 10.7303/syn56849053 for doi.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: doi.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for 10.7303/syn56849053 in doi.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 2 with repo: doi.org and accession_id: 10.7303/syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://doi.org/10.7303/syn56849053\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1277 - INFO - Updated datasets len: 2\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 287 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 4, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'source_url', 'download_link',\n",
      "       'title', 'content_type', 'id', 'caption', 'description',\n",
      "       'source_section', 'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 500 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 506 - INFO - XML element classification not needed. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1012 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11443687\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1020 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11443687\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 634th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11444172)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11444172\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11444172 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 634 - INFO - Fetching data for pmc11444172\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 637 - INFO - Fetching data from scripts/exp_input/Local_fetched_data_1.parquet\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 438 - INFO - Raw html data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11444172 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 452 - INFO - Skipping raw HTML/XML saving.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 482 - INFO - Using HTMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 52 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 82 - INFO - Initializing htmlRetriever\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 350 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11444172, additional_data, html)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 448 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC11444172)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 550 - INFO - Extracted 26 unique supplementary material links from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 560 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#clp_a'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for clp_a: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#clp_a'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for clp_a: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#clp_a'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for clp_a: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#s3b'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for s3b: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#s3i'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for s3i: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#s3k'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for s3k: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#s3k'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for s3k: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#s3m'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for s3m: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#s4'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for s4: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#s4'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for s4: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 4 <a> elements with href='#SD1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 316) for SD1: S1A [href=this file] ). We observed an average increase in total read mapping of 1.24% when using personalized graph genomes compared with the GRCh38 linear reference via Bowtie 2 ( Supplemental Table S2 [href=this file1] ; Supplemental Fig. S1B [href=this file] ). S1C [href=this file] ).\n",
      "S1D,E [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 4 <a> elements with href='#SD2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 117) for SD2: S2A [href=this file] ; Loupe et al. 2024 [href=#GR278601ANDC35] ). S2A,B [href=this file] ). S2C [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 5 <a> elements with href='#SD3'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 253) for SD3: S4A [href=this file] ). S4B [href=this file] ).\n",
      "S4C [href=this file] ). S4D [href=this file] ).\n",
      "S4E [href=this file] ), suggesting that lower read-depth at cell type–specific binding sites does not explain the depletion of ASB in neuron-specific peaks.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 2 <a> elements with href='#SD4'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 143) for SD4: S5A [href=this file] ), supporting the hypothesis that larger changes in binding lead to larger changes in expression. S5B [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 2 <a> elements with href='#SD5'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 292) for SD5: The config file is provided in the Supplemental_Code.zip [href=this file] file as vep108.ini.\n",
      "All code used for these analyses is available via GitHub ( https://github.com/aanderson54/BrainTF_ASB [href=https://github.com/aanderson54/BrainTF_ASB] ) and as Supplemental Code [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 2 <a> elements with href='#SD6'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 496) for SD6: We found that at a nominal -value cutoff of 0.05, concordance is 79.8%, and at a -value cutoff of 0.001, concordance increases to 94.2% ( Fig. 1 [href=#GR278601ANDF1] D; Supplemental Table S3 [href=this file] ); these correspond to empirical FDRs of 40% and 12%, respectively. Reproducibility between donors was lower than that within-donor but still substantial, being 70.6% reproducible at < 0.05 and 92.3% at < 0.001 ( Fig. 1 [href=#GR278601ANDF1] E; Supplemental Table S3 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 1 <a> elements with href='#SD7'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 263) for SD7: To study the impact of genetic variation on TF binding, we first performed linked-read sequencing (10x Genomics) to generate phased genomes and call variants for two donors (see Methods) ( Fig. 1 [href=#GR278601ANDF1] A; Supplemental Table S1 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 2 <a> elements with href='#SD8'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 304) for SD8: In these data, we observe significant ( < 0.05) allele-specific expression of across brain regions, whereas expression remains unaltered ( Fig. 5 [href=#GR278601ANDF5] B; Supplemental Table S8 [href=this file] ).\n",
      "ASoC data were downloaded from Supplemental Tables S4 [href=#SD13] – S8 [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 2 <a> elements with href='#SD9'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 268) for SD9: However, ASB variants showed a higher degree of enrichment for all three diseases ( Fig. 3 [href=#GR278601ANDF3] E; Supplemental Tables S6 [href=#SD10] , S7 [href=this file] ).\n",
      "Links to GWAS summary statistics are available in Supplemental Table S7 [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 1 <a> elements with href='#SD10'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 176) for SD10: However, ASB variants showed a higher degree of enrichment for all three diseases ( Fig. 3 [href=#GR278601ANDF3] E; Supplemental Tables S6 [href=this file] , S7 [href=#SD9] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 1 <a> elements with href='#SD11'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 217) for SD11: We observed an average increase in total read mapping of 1.24% when using personalized graph genomes compared with the GRCh38 linear reference via Bowtie 2 ( Supplemental Table S2 [href=this file] ; Supplemental Fig.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 1 <a> elements with href='#SD12'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 283) for SD12: To assess potential mechanisms driving ASB, we utilized motifbreakR and JASPAR 2022 human motifs to identify motifs affected by ASB variants ( Supplemental Table S5 [href=this file] ; Coetzee et al. 2015 [href=#GR278601ANDC13] ; Castro-Mondragon et al. 2022 [href=#GR278601ANDC9] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 4 <a> elements with href='#SD13'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 782) for SD13: We also generated Benjamini–Hochberg (BH)-adjusted values based on the total numbers of heterozygous variants with at least seven reads in each experiment ( Supplemental Table S4 [href=this file] ); the BH FDR estimates are higher and likely relatively conservative; and 2837 variants meet a BH-corrected -value of 0.05.\n",
      "CTCF, RAD21, and SMC3 all exhibit a strong preference for the alternate allele across all brain regions, with >90% of the reads aligning to it ( Fig. 5 [href=#GR278601ANDF5] C; Supplemental Table S4 [href=this file] ).\n",
      "Last, BH -value corrections yielded larger FDR estimates (these are provided alongside the nominal -values in Supplemental Table S4 [href=this file] ).\n",
      "ASoC data were downloaded from Supplemental Tables S4 [href=this file] – S8 [href=#SD8] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 7 <a> elements with href='#SD14'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 144) for SD14: S3A [href=this file] ). S3B [href=this file] ).\n",
      "S3C [href=this file] ). S3D [href=this file] ).\n",
      "S3E [href=this file] ).\n",
      "S3F [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#fn-group1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for fn-group1: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 368 - INFO - Extracting links from full HTML content.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 301 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 309 - INFO - Content length: 189178\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 326 - INFO - Prompt messages total length: 48048 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 332 - INFO - Prompt ID: gemini-2.5-flash-0-fd49a6d448eabd37ad61c27f0a6728769e9cb8346cd723b2a035aa7cb5535988\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 351 - INFO - Requesting datasets from content using model: gemini-2.5-flash, temperature: 0, messages length: 48048 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-cdc9cd4d-4acc-4585-8ce5-257ec4f29622\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"BrainTF_ASB\\\",\\n    \\\"repository_reference\\\": \\\"https://github.com/aanderson54/BrainTF_ASB\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn51942384\\\",\\n    \\\"repository_reference\\\": \\\"https://doi.org/10.7303/syn4921369\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Supplemental_Code.zip\\\",\\n    \\\"repository_reference\\\": \\\"https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Code.zip\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Supplemental_Fig_1.pdf\\\",\\n    \\\"repository_reference\\\": \\\"https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_1.pdf\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Supplemental_Fig_2.pdf\\\",\\n    \\\"repository_reference\\\": \\\"https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_2.pdf\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Supplemental_Fig_4.pdf\\\",\\n    \\\"repository_reference\\\": \\\"https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_4.pdf\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Supplemental_Fig_5.pdf\\\",\\n    \\\"repository_reference\\\": \\\"https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_5.pdf\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Supplemental_Table_3.xlsx\\\",\\n    \\\"repository_reference\\\": \\\"https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_3.xlsx\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Supplemental_Table_1.xlsx\\\",\\n    \\\"repository_reference\\\": \\\"https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_1.xlsx\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Supplemental_Table_8.xlsx\\\",\\n    \\\"repository_reference\\\": \\\"https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_8.xlsx\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Supplemental_Table_7.xlsx\\\",\\n    \\\"repository_reference\\\": \\\"https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_7.xlsx\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Supplemental_Table_6.xlsx\\\",\\n    \\\"repository_reference\\\": \\\"https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_6.xlsx\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Supplemental_Table_2.xlsx\\\",\\n    \\\"repository_reference\\\": \\\"https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_2.xlsx\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Supplemental_Table_5.xlsx\\\",\\n    \\\"repository_reference\\\": \\\"https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_5.xlsx\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Supplemental_Table_4.xlsx\\\",\\n    \\\"repository_reference\\\": \\\"https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_4.xlsx\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Supplemental_Fig_3.pdf\\\",\\n    \\\"repository_reference\\\": \\\"https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_3.pdf\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GTEx_Analysis_v8_eQTL.tar\\\",\\n    \\\"repository_reference\\\": \\\"https://storage.googleapis.com/gtex_analysis_v8/single_tissue_qtl_data/GTEx_Analysis_v8_eQTL.tar\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"V3 cCRE human data set\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"ADASTRA v5.1.3\\\",\\n    \\\"repository_reference\\\": \\\"https://adastra.autosome.org/bill-cipher/downloads\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"BrainTF_ASB\\\",\\n    \\\"repository_reference\\\": \\\"https://github.com/aanderson54/BrainTF_ASB\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn51942384\\\",\\n    \\\"repository_reference\\\": \\\"https://doi.org/10.7303/syn4921369\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Supplemental_Code.zip\\\",\\n    \\\"repository_reference\\\": \\\"https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Code.zip\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Supplemental_Fig_1.pdf\\\",\\n    \\\"repository_reference\\\": \\\"https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_1.pdf\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Supplemental_Fig_2.pdf\\\",\\n    \\\"repository_reference\\\": \\\"https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_2.pdf\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Supplemental_Fig_4.pdf\\\",\\n    \\\"repository_reference\\\": \\\"https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_4.pdf\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Supplemental_Fig_5.pdf\\\",\\n    \\\"repository_reference\\\": \\\"https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_5.pdf\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Supplemental_Table_3.xlsx\\\",\\n    \\\"repository_reference\\\": \\\"https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_3.xlsx\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Supplemental_Table_1.xlsx\\\",\\n    \\\"repository_reference\\\": \\\"https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_1.xlsx\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Supplemental_Table_8.xlsx\\\",\\n    \\\"repository_reference\\\": \\\"https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_8.xlsx\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Supplemental_Table_7.xlsx\\\",\\n    \\\"repository_reference\\\": \\\"https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_7.xlsx\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Supplemental_Table_6.xlsx\\\",\\n    \\\"repository_reference\\\": \\\"https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_6.xlsx\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Supplemental_Table_2.xlsx\\\",\\n    \\\"repository_reference\\\": \\\"https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_2.xlsx\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Supplemental_Table_5.xlsx\\\",\\n    \\\"repository_reference\\\": \\\"https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_5.xlsx\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Supplemental_Table_4.xlsx\\\",\\n    \\\"repository_reference\\\": \\\"https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_4.xlsx\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Supplemental_Fig_3.pdf\\\",\\n    \\\"repository_reference\\\": \\\"https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_3.pdf\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GTEx_Analysis_v8_eQTL.tar\\\",\\n    \\\"repository_reference\\\": \\\"https://storage.googleapis.com/gtex_analysis_v8/single_tissue_qtl_data/GTEx_Analysis_v8_eQTL.tar\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"V3 cCRE human data set\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"ADASTRA v5.1.3\\\",\\n    \\\"repository_reference\\\": \\\"https://adastra.autosome.org/bill-cipher/downloads\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1758210065,\n",
      "    \"model\": \"gemini-2.5-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 68804,\n",
      "        \"completion_tokens\": 1171,\n",
      "        \"total_tokens\": 77964,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 7989,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 664 - INFO - Deduplicating response with 19 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: BrainTF_ASB\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn51942384\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: Supplemental_Code.zip\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: Supplemental_Fig_1.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: Supplemental_Fig_2.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: Supplemental_Fig_4.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: Supplemental_Fig_5.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: Supplemental_Table_3.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: Supplemental_Table_1.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: Supplemental_Table_8.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: Supplemental_Table_7.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: Supplemental_Table_6.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: Supplemental_Table_2.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: Supplemental_Table_5.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: Supplemental_Table_4.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: Supplemental_Fig_3.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: GTEx_Analysis_v8_eQTL.tar\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: V3 cCRE human data set\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: ADASTRA v5.1.3\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'BrainTF_ASB', 'repository_reference': 'https://github.com/aanderson54/BrainTF_ASB', 'dataset_id': 'BrainTF_ASB'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: BrainTF_ASB\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID BrainTF_ASB is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://github.com/aanderson54/BrainTF_ASB, identifier: BrainTF_ASB, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://github.com/aanderson54/BrainTF_ASB\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier BrainTF_ASB for https://github.com/aanderson54/BrainTF_ASB\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier BrainTF_ASB for https://github.com/aanderson54/BrainTF_ASB\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier BrainTF_ASB for https://github.com/aanderson54/BrainTF_ASB\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier BrainTF_ASB for https://github.com/aanderson54/BrainTF_ASB\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier BrainTF_ASB for https://github.com/aanderson54/BrainTF_ASB\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier BrainTF_ASB for https://github.com/aanderson54/BrainTF_ASB\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://github.com/aanderson54/BrainTF_ASB\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'BrainTF_ASB', 'data_repository': 'github.com', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn51942384', 'repository_reference': 'https://doi.org/10.7303/syn4921369', 'dataset_id': 'syn51942384'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn51942384\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn51942384\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://doi.org/10.7303/syn4921369, identifier: syn51942384, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://doi.org/10.7303/syn4921369\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn51942384 for https://doi.org/10.7303/syn4921369\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn51942384 for https://doi.org/10.7303/syn4921369\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn51942384 for https://doi.org/10.7303/syn4921369\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier syn51942384 for https://doi.org/10.7303/syn4921369\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1157 - INFO - Found id_pattern match for https://doi.org/10.7303/syn4921369 in syn\\d{7,12}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://doi.org/10.7303/syn4921369\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://doi.org/10.7303/syn4921369\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn51942384\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn51942384\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://doi.org/__ID__ to value: https://doi.org/10.7303/syn4921369\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: 10\\.\\d{4,9}/[^\\s,]+ to value: https://doi.org/10.7303/syn4921369\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(10\\.\\d{4,9}/[^\\s,]+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(10\\.\\d{4,9}/[^\\s,]+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn51942384\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn51942384\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://doi.org/10.7303/syn4921369\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn51942384\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn51942384\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn51942384, synapse.org, https://doi.org/10.7303/syn4921369\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn51942384\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://doi.org/10.7303/syn4921369\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn51942384', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://doi.org/10.7303/syn4921369', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'Supplemental_Code.zip', 'repository_reference': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Code.zip', 'dataset_id': 'Supplemental_Code.zip'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Code.zip\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Code.zip\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Code.zip\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: Supplemental_Code.zip\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID Supplemental_Code.zip is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Code.zip, identifier: Supplemental_Code.zip, dataset_page: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Code.zip\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Code.zip\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Supplemental_Code.zip for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Code.zip\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier Supplemental_Code.zip for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Code.zip\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Supplemental_Code.zip for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Code.zip\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Supplemental_Code.zip for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Code.zip\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier Supplemental_Code.zip for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Code.zip\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier Supplemental_Code.zip for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Code.zip\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Code.zip\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Code.zip\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'Supplemental_Code.zip', 'data_repository': 'pmc.ncbi.nlm.nih.gov', 'dataset_webpage': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Code.zip', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'Supplemental_Fig_1.pdf', 'repository_reference': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_1.pdf', 'dataset_id': 'Supplemental_Fig_1.pdf'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_1.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_1.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_1.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: Supplemental_Fig_1.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID Supplemental_Fig_1.pdf is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_1.pdf, identifier: Supplemental_Fig_1.pdf, dataset_page: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_1.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_1.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Supplemental_Fig_1.pdf for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_1.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier Supplemental_Fig_1.pdf for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_1.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Supplemental_Fig_1.pdf for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_1.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Supplemental_Fig_1.pdf for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_1.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier Supplemental_Fig_1.pdf for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_1.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier Supplemental_Fig_1.pdf for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_1.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_1.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_1.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'Supplemental_Fig_1.pdf', 'data_repository': 'pmc.ncbi.nlm.nih.gov', 'dataset_webpage': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_1.pdf', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'Supplemental_Fig_2.pdf', 'repository_reference': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_2.pdf', 'dataset_id': 'Supplemental_Fig_2.pdf'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_2.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_2.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_2.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: Supplemental_Fig_2.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID Supplemental_Fig_2.pdf is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_2.pdf, identifier: Supplemental_Fig_2.pdf, dataset_page: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_2.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_2.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Supplemental_Fig_2.pdf for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_2.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier Supplemental_Fig_2.pdf for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_2.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Supplemental_Fig_2.pdf for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_2.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Supplemental_Fig_2.pdf for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_2.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier Supplemental_Fig_2.pdf for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_2.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier Supplemental_Fig_2.pdf for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_2.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_2.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_2.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'Supplemental_Fig_2.pdf', 'data_repository': 'pmc.ncbi.nlm.nih.gov', 'dataset_webpage': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_2.pdf', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'Supplemental_Fig_4.pdf', 'repository_reference': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_4.pdf', 'dataset_id': 'Supplemental_Fig_4.pdf'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_4.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_4.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_4.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: Supplemental_Fig_4.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID Supplemental_Fig_4.pdf is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_4.pdf, identifier: Supplemental_Fig_4.pdf, dataset_page: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_4.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_4.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Supplemental_Fig_4.pdf for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_4.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier Supplemental_Fig_4.pdf for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_4.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Supplemental_Fig_4.pdf for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_4.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Supplemental_Fig_4.pdf for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_4.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier Supplemental_Fig_4.pdf for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_4.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier Supplemental_Fig_4.pdf for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_4.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_4.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_4.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'Supplemental_Fig_4.pdf', 'data_repository': 'pmc.ncbi.nlm.nih.gov', 'dataset_webpage': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_4.pdf', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'Supplemental_Fig_5.pdf', 'repository_reference': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_5.pdf', 'dataset_id': 'Supplemental_Fig_5.pdf'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_5.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_5.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_5.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: Supplemental_Fig_5.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID Supplemental_Fig_5.pdf is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_5.pdf, identifier: Supplemental_Fig_5.pdf, dataset_page: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_5.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_5.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Supplemental_Fig_5.pdf for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_5.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier Supplemental_Fig_5.pdf for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_5.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Supplemental_Fig_5.pdf for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_5.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Supplemental_Fig_5.pdf for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_5.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier Supplemental_Fig_5.pdf for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_5.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier Supplemental_Fig_5.pdf for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_5.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_5.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_5.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'Supplemental_Fig_5.pdf', 'data_repository': 'pmc.ncbi.nlm.nih.gov', 'dataset_webpage': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_5.pdf', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'Supplemental_Table_3.xlsx', 'repository_reference': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_3.xlsx', 'dataset_id': 'Supplemental_Table_3.xlsx'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_3.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_3.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_3.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: Supplemental_Table_3.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID Supplemental_Table_3.xlsx is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_3.xlsx, identifier: Supplemental_Table_3.xlsx, dataset_page: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_3.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_3.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Supplemental_Table_3.xlsx for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_3.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier Supplemental_Table_3.xlsx for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_3.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Supplemental_Table_3.xlsx for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_3.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Supplemental_Table_3.xlsx for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_3.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier Supplemental_Table_3.xlsx for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_3.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier Supplemental_Table_3.xlsx for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_3.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_3.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_3.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'Supplemental_Table_3.xlsx', 'data_repository': 'pmc.ncbi.nlm.nih.gov', 'dataset_webpage': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_3.xlsx', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'Supplemental_Table_1.xlsx', 'repository_reference': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_1.xlsx', 'dataset_id': 'Supplemental_Table_1.xlsx'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_1.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_1.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_1.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: Supplemental_Table_1.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID Supplemental_Table_1.xlsx is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_1.xlsx, identifier: Supplemental_Table_1.xlsx, dataset_page: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_1.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_1.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Supplemental_Table_1.xlsx for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_1.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier Supplemental_Table_1.xlsx for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_1.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Supplemental_Table_1.xlsx for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_1.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Supplemental_Table_1.xlsx for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_1.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier Supplemental_Table_1.xlsx for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_1.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier Supplemental_Table_1.xlsx for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_1.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_1.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_1.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'Supplemental_Table_1.xlsx', 'data_repository': 'pmc.ncbi.nlm.nih.gov', 'dataset_webpage': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_1.xlsx', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'Supplemental_Table_8.xlsx', 'repository_reference': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_8.xlsx', 'dataset_id': 'Supplemental_Table_8.xlsx'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_8.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_8.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_8.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: Supplemental_Table_8.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID Supplemental_Table_8.xlsx is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_8.xlsx, identifier: Supplemental_Table_8.xlsx, dataset_page: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_8.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_8.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Supplemental_Table_8.xlsx for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_8.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier Supplemental_Table_8.xlsx for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_8.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Supplemental_Table_8.xlsx for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_8.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Supplemental_Table_8.xlsx for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_8.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier Supplemental_Table_8.xlsx for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_8.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier Supplemental_Table_8.xlsx for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_8.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_8.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_8.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'Supplemental_Table_8.xlsx', 'data_repository': 'pmc.ncbi.nlm.nih.gov', 'dataset_webpage': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_8.xlsx', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'Supplemental_Table_7.xlsx', 'repository_reference': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_7.xlsx', 'dataset_id': 'Supplemental_Table_7.xlsx'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_7.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_7.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_7.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: Supplemental_Table_7.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID Supplemental_Table_7.xlsx is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_7.xlsx, identifier: Supplemental_Table_7.xlsx, dataset_page: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_7.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_7.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Supplemental_Table_7.xlsx for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_7.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier Supplemental_Table_7.xlsx for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_7.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Supplemental_Table_7.xlsx for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_7.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Supplemental_Table_7.xlsx for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_7.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier Supplemental_Table_7.xlsx for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_7.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier Supplemental_Table_7.xlsx for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_7.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_7.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_7.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'Supplemental_Table_7.xlsx', 'data_repository': 'pmc.ncbi.nlm.nih.gov', 'dataset_webpage': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_7.xlsx', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'Supplemental_Table_6.xlsx', 'repository_reference': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_6.xlsx', 'dataset_id': 'Supplemental_Table_6.xlsx'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_6.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_6.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_6.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: Supplemental_Table_6.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID Supplemental_Table_6.xlsx is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_6.xlsx, identifier: Supplemental_Table_6.xlsx, dataset_page: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_6.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_6.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Supplemental_Table_6.xlsx for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_6.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier Supplemental_Table_6.xlsx for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_6.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Supplemental_Table_6.xlsx for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_6.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Supplemental_Table_6.xlsx for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_6.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier Supplemental_Table_6.xlsx for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_6.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier Supplemental_Table_6.xlsx for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_6.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_6.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_6.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'Supplemental_Table_6.xlsx', 'data_repository': 'pmc.ncbi.nlm.nih.gov', 'dataset_webpage': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_6.xlsx', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'Supplemental_Table_2.xlsx', 'repository_reference': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_2.xlsx', 'dataset_id': 'Supplemental_Table_2.xlsx'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_2.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_2.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_2.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: Supplemental_Table_2.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID Supplemental_Table_2.xlsx is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_2.xlsx, identifier: Supplemental_Table_2.xlsx, dataset_page: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_2.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_2.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Supplemental_Table_2.xlsx for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_2.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier Supplemental_Table_2.xlsx for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_2.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Supplemental_Table_2.xlsx for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_2.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Supplemental_Table_2.xlsx for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_2.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier Supplemental_Table_2.xlsx for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_2.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier Supplemental_Table_2.xlsx for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_2.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_2.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_2.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'Supplemental_Table_2.xlsx', 'data_repository': 'pmc.ncbi.nlm.nih.gov', 'dataset_webpage': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_2.xlsx', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'Supplemental_Table_5.xlsx', 'repository_reference': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_5.xlsx', 'dataset_id': 'Supplemental_Table_5.xlsx'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_5.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_5.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_5.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: Supplemental_Table_5.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID Supplemental_Table_5.xlsx is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_5.xlsx, identifier: Supplemental_Table_5.xlsx, dataset_page: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_5.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_5.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Supplemental_Table_5.xlsx for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_5.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier Supplemental_Table_5.xlsx for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_5.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Supplemental_Table_5.xlsx for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_5.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Supplemental_Table_5.xlsx for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_5.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier Supplemental_Table_5.xlsx for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_5.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier Supplemental_Table_5.xlsx for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_5.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_5.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_5.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'Supplemental_Table_5.xlsx', 'data_repository': 'pmc.ncbi.nlm.nih.gov', 'dataset_webpage': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_5.xlsx', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'Supplemental_Table_4.xlsx', 'repository_reference': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_4.xlsx', 'dataset_id': 'Supplemental_Table_4.xlsx'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_4.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_4.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_4.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: Supplemental_Table_4.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID Supplemental_Table_4.xlsx is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_4.xlsx, identifier: Supplemental_Table_4.xlsx, dataset_page: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_4.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_4.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Supplemental_Table_4.xlsx for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_4.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier Supplemental_Table_4.xlsx for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_4.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Supplemental_Table_4.xlsx for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_4.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Supplemental_Table_4.xlsx for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_4.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier Supplemental_Table_4.xlsx for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_4.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier Supplemental_Table_4.xlsx for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_4.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_4.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_4.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'Supplemental_Table_4.xlsx', 'data_repository': 'pmc.ncbi.nlm.nih.gov', 'dataset_webpage': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_4.xlsx', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'Supplemental_Fig_3.pdf', 'repository_reference': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_3.pdf', 'dataset_id': 'Supplemental_Fig_3.pdf'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_3.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_3.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_3.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: Supplemental_Fig_3.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID Supplemental_Fig_3.pdf is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_3.pdf, identifier: Supplemental_Fig_3.pdf, dataset_page: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_3.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_3.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Supplemental_Fig_3.pdf for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_3.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier Supplemental_Fig_3.pdf for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_3.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Supplemental_Fig_3.pdf for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_3.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Supplemental_Fig_3.pdf for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_3.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier Supplemental_Fig_3.pdf for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_3.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier Supplemental_Fig_3.pdf for https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_3.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_3.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_3.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'Supplemental_Fig_3.pdf', 'data_repository': 'pmc.ncbi.nlm.nih.gov', 'dataset_webpage': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_3.pdf', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'GTEx_Analysis_v8_eQTL.tar', 'repository_reference': 'https://storage.googleapis.com/gtex_analysis_v8/single_tissue_qtl_data/GTEx_Analysis_v8_eQTL.tar', 'dataset_id': 'GTEx_Analysis_v8_eQTL.tar'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: GTEx_Analysis_v8_eQTL.tar\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID GTEx_Analysis_v8_eQTL.tar is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://storage.googleapis.com/gtex_analysis_v8/single_tissue_qtl_data/GTEx_Analysis_v8_eQTL.tar, identifier: GTEx_Analysis_v8_eQTL.tar, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://storage.googleapis.com/gtex_analysis_v8/single_tissue_qtl_data/GTEx_Analysis_v8_eQTL.tar\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GTEx_Analysis_v8_eQTL.tar for https://storage.googleapis.com/gtex_analysis_v8/single_tissue_qtl_data/GTEx_Analysis_v8_eQTL.tar\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier GTEx_Analysis_v8_eQTL.tar for https://storage.googleapis.com/gtex_analysis_v8/single_tissue_qtl_data/GTEx_Analysis_v8_eQTL.tar\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier GTEx_Analysis_v8_eQTL.tar for https://storage.googleapis.com/gtex_analysis_v8/single_tissue_qtl_data/GTEx_Analysis_v8_eQTL.tar\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier GTEx_Analysis_v8_eQTL.tar for https://storage.googleapis.com/gtex_analysis_v8/single_tissue_qtl_data/GTEx_Analysis_v8_eQTL.tar\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier GTEx_Analysis_v8_eQTL.tar for https://storage.googleapis.com/gtex_analysis_v8/single_tissue_qtl_data/GTEx_Analysis_v8_eQTL.tar\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier GTEx_Analysis_v8_eQTL.tar for https://storage.googleapis.com/gtex_analysis_v8/single_tissue_qtl_data/GTEx_Analysis_v8_eQTL.tar\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://storage.googleapis.com/gtex_analysis_v8/single_tissue_qtl_data/GTEx_Analysis_v8_eQTL.tar\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'GTEx_Analysis_v8_eQTL.tar', 'data_repository': 'storage.googleapis.com', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'V3 cCRE human data set', 'repository_reference': 'n/a', 'dataset_id': 'V3 cCRE human data set'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: V3 cCRE human data set\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID V3 cCRE human data set is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: n/a, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'n/a', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'ADASTRA v5.1.3', 'repository_reference': 'https://adastra.autosome.org/bill-cipher/downloads', 'dataset_id': 'ADASTRA v5.1.3'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: ADASTRA v5.1.3\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID ADASTRA v5.1.3 is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://adastra.autosome.org/bill-cipher/downloads, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://adastra.autosome.org/bill-cipher/downloads\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://adastra.autosome.org/bill-cipher/downloads\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'adastra.autosome.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 378 - INFO - Augmented dataset links: [{'dataset_identifier': 'BrainTF_ASB', 'data_repository': 'github.com', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn51942384', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://doi.org/10.7303/syn4921369', 'citation_type': 'n/a'}, {'dataset_identifier': 'Supplemental_Code.zip', 'data_repository': 'pmc.ncbi.nlm.nih.gov', 'dataset_webpage': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Code.zip', 'citation_type': 'n/a'}, {'dataset_identifier': 'Supplemental_Fig_1.pdf', 'data_repository': 'pmc.ncbi.nlm.nih.gov', 'dataset_webpage': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_1.pdf', 'citation_type': 'n/a'}, {'dataset_identifier': 'Supplemental_Fig_2.pdf', 'data_repository': 'pmc.ncbi.nlm.nih.gov', 'dataset_webpage': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_2.pdf', 'citation_type': 'n/a'}, {'dataset_identifier': 'Supplemental_Fig_4.pdf', 'data_repository': 'pmc.ncbi.nlm.nih.gov', 'dataset_webpage': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_4.pdf', 'citation_type': 'n/a'}, {'dataset_identifier': 'Supplemental_Fig_5.pdf', 'data_repository': 'pmc.ncbi.nlm.nih.gov', 'dataset_webpage': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_5.pdf', 'citation_type': 'n/a'}, {'dataset_identifier': 'Supplemental_Table_3.xlsx', 'data_repository': 'pmc.ncbi.nlm.nih.gov', 'dataset_webpage': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_3.xlsx', 'citation_type': 'n/a'}, {'dataset_identifier': 'Supplemental_Table_1.xlsx', 'data_repository': 'pmc.ncbi.nlm.nih.gov', 'dataset_webpage': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_1.xlsx', 'citation_type': 'n/a'}, {'dataset_identifier': 'Supplemental_Table_8.xlsx', 'data_repository': 'pmc.ncbi.nlm.nih.gov', 'dataset_webpage': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_8.xlsx', 'citation_type': 'n/a'}, {'dataset_identifier': 'Supplemental_Table_7.xlsx', 'data_repository': 'pmc.ncbi.nlm.nih.gov', 'dataset_webpage': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_7.xlsx', 'citation_type': 'n/a'}, {'dataset_identifier': 'Supplemental_Table_6.xlsx', 'data_repository': 'pmc.ncbi.nlm.nih.gov', 'dataset_webpage': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_6.xlsx', 'citation_type': 'n/a'}, {'dataset_identifier': 'Supplemental_Table_2.xlsx', 'data_repository': 'pmc.ncbi.nlm.nih.gov', 'dataset_webpage': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_2.xlsx', 'citation_type': 'n/a'}, {'dataset_identifier': 'Supplemental_Table_5.xlsx', 'data_repository': 'pmc.ncbi.nlm.nih.gov', 'dataset_webpage': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_5.xlsx', 'citation_type': 'n/a'}, {'dataset_identifier': 'Supplemental_Table_4.xlsx', 'data_repository': 'pmc.ncbi.nlm.nih.gov', 'dataset_webpage': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Table_4.xlsx', 'citation_type': 'n/a'}, {'dataset_identifier': 'Supplemental_Fig_3.pdf', 'data_repository': 'pmc.ncbi.nlm.nih.gov', 'dataset_webpage': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/11444172/bin/Supplemental_Fig_3.pdf', 'citation_type': 'n/a'}, {'dataset_identifier': 'GTEx_Analysis_v8_eQTL.tar', 'data_repository': 'storage.googleapis.com', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'n/a', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'adastra.autosome.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1191 - INFO - Fetching metadata for 19 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: BrainTF_ASB\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: github.com, identifier: BrainTF_ASB, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier BrainTF_ASB for github.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier BrainTF_ASB for github.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier BrainTF_ASB for github.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier BrainTF_ASB for github.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier BrainTF_ASB for github.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier BrainTF_ASB for github.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: github.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL github.com may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for BrainTF_ASB in github.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 1 with repo: github.com and accession_id: BrainTF_ASB\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository github.com unknown in Ontology. Skipping dataset page 1.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn51942384\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn51942384, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn51942384 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn51942384 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn51942384 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn51942384 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn51942384\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn51942384\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: Supplemental_Code.zip\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: pmc.ncbi.nlm.nih.gov, identifier: Supplemental_Code.zip, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Supplemental_Code.zip for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier Supplemental_Code.zip for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Supplemental_Code.zip for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Supplemental_Code.zip for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier Supplemental_Code.zip for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier Supplemental_Code.zip for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL pmc.ncbi.nlm.nih.gov may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for Supplemental_Code.zip in pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 3 with repo: pmc.ncbi.nlm.nih.gov and accession_id: Supplemental_Code.zip\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository pmc.ncbi.nlm.nih.gov unknown in Ontology. Skipping dataset page 3.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: Supplemental_Fig_1.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: pmc.ncbi.nlm.nih.gov, identifier: Supplemental_Fig_1.pdf, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Supplemental_Fig_1.pdf for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier Supplemental_Fig_1.pdf for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Supplemental_Fig_1.pdf for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Supplemental_Fig_1.pdf for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier Supplemental_Fig_1.pdf for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier Supplemental_Fig_1.pdf for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL pmc.ncbi.nlm.nih.gov may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for Supplemental_Fig_1.pdf in pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 4 with repo: pmc.ncbi.nlm.nih.gov and accession_id: Supplemental_Fig_1.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository pmc.ncbi.nlm.nih.gov unknown in Ontology. Skipping dataset page 4.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: Supplemental_Fig_2.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: pmc.ncbi.nlm.nih.gov, identifier: Supplemental_Fig_2.pdf, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Supplemental_Fig_2.pdf for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier Supplemental_Fig_2.pdf for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Supplemental_Fig_2.pdf for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Supplemental_Fig_2.pdf for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier Supplemental_Fig_2.pdf for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier Supplemental_Fig_2.pdf for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL pmc.ncbi.nlm.nih.gov may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for Supplemental_Fig_2.pdf in pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 5 with repo: pmc.ncbi.nlm.nih.gov and accession_id: Supplemental_Fig_2.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository pmc.ncbi.nlm.nih.gov unknown in Ontology. Skipping dataset page 5.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: Supplemental_Fig_4.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: pmc.ncbi.nlm.nih.gov, identifier: Supplemental_Fig_4.pdf, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Supplemental_Fig_4.pdf for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier Supplemental_Fig_4.pdf for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Supplemental_Fig_4.pdf for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Supplemental_Fig_4.pdf for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier Supplemental_Fig_4.pdf for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier Supplemental_Fig_4.pdf for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL pmc.ncbi.nlm.nih.gov may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for Supplemental_Fig_4.pdf in pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 6 with repo: pmc.ncbi.nlm.nih.gov and accession_id: Supplemental_Fig_4.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository pmc.ncbi.nlm.nih.gov unknown in Ontology. Skipping dataset page 6.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: Supplemental_Fig_5.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: pmc.ncbi.nlm.nih.gov, identifier: Supplemental_Fig_5.pdf, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Supplemental_Fig_5.pdf for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier Supplemental_Fig_5.pdf for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Supplemental_Fig_5.pdf for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Supplemental_Fig_5.pdf for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier Supplemental_Fig_5.pdf for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier Supplemental_Fig_5.pdf for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL pmc.ncbi.nlm.nih.gov may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for Supplemental_Fig_5.pdf in pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 7 with repo: pmc.ncbi.nlm.nih.gov and accession_id: Supplemental_Fig_5.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository pmc.ncbi.nlm.nih.gov unknown in Ontology. Skipping dataset page 7.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: Supplemental_Table_3.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: pmc.ncbi.nlm.nih.gov, identifier: Supplemental_Table_3.xlsx, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Supplemental_Table_3.xlsx for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier Supplemental_Table_3.xlsx for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Supplemental_Table_3.xlsx for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Supplemental_Table_3.xlsx for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier Supplemental_Table_3.xlsx for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier Supplemental_Table_3.xlsx for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL pmc.ncbi.nlm.nih.gov may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for Supplemental_Table_3.xlsx in pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 8 with repo: pmc.ncbi.nlm.nih.gov and accession_id: Supplemental_Table_3.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository pmc.ncbi.nlm.nih.gov unknown in Ontology. Skipping dataset page 8.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: Supplemental_Table_1.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: pmc.ncbi.nlm.nih.gov, identifier: Supplemental_Table_1.xlsx, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Supplemental_Table_1.xlsx for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier Supplemental_Table_1.xlsx for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Supplemental_Table_1.xlsx for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Supplemental_Table_1.xlsx for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier Supplemental_Table_1.xlsx for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier Supplemental_Table_1.xlsx for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL pmc.ncbi.nlm.nih.gov may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for Supplemental_Table_1.xlsx in pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 9 with repo: pmc.ncbi.nlm.nih.gov and accession_id: Supplemental_Table_1.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository pmc.ncbi.nlm.nih.gov unknown in Ontology. Skipping dataset page 9.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 10 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: Supplemental_Table_8.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: pmc.ncbi.nlm.nih.gov, identifier: Supplemental_Table_8.xlsx, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Supplemental_Table_8.xlsx for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier Supplemental_Table_8.xlsx for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Supplemental_Table_8.xlsx for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Supplemental_Table_8.xlsx for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier Supplemental_Table_8.xlsx for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier Supplemental_Table_8.xlsx for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL pmc.ncbi.nlm.nih.gov may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for Supplemental_Table_8.xlsx in pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 10 with repo: pmc.ncbi.nlm.nih.gov and accession_id: Supplemental_Table_8.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository pmc.ncbi.nlm.nih.gov unknown in Ontology. Skipping dataset page 10.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 11 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: Supplemental_Table_7.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: pmc.ncbi.nlm.nih.gov, identifier: Supplemental_Table_7.xlsx, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Supplemental_Table_7.xlsx for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier Supplemental_Table_7.xlsx for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Supplemental_Table_7.xlsx for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Supplemental_Table_7.xlsx for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier Supplemental_Table_7.xlsx for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier Supplemental_Table_7.xlsx for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL pmc.ncbi.nlm.nih.gov may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for Supplemental_Table_7.xlsx in pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 11 with repo: pmc.ncbi.nlm.nih.gov and accession_id: Supplemental_Table_7.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository pmc.ncbi.nlm.nih.gov unknown in Ontology. Skipping dataset page 11.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 12 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: Supplemental_Table_6.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: pmc.ncbi.nlm.nih.gov, identifier: Supplemental_Table_6.xlsx, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Supplemental_Table_6.xlsx for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier Supplemental_Table_6.xlsx for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Supplemental_Table_6.xlsx for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Supplemental_Table_6.xlsx for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier Supplemental_Table_6.xlsx for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier Supplemental_Table_6.xlsx for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL pmc.ncbi.nlm.nih.gov may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for Supplemental_Table_6.xlsx in pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 12 with repo: pmc.ncbi.nlm.nih.gov and accession_id: Supplemental_Table_6.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository pmc.ncbi.nlm.nih.gov unknown in Ontology. Skipping dataset page 12.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 13 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: Supplemental_Table_2.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: pmc.ncbi.nlm.nih.gov, identifier: Supplemental_Table_2.xlsx, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Supplemental_Table_2.xlsx for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier Supplemental_Table_2.xlsx for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Supplemental_Table_2.xlsx for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Supplemental_Table_2.xlsx for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier Supplemental_Table_2.xlsx for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier Supplemental_Table_2.xlsx for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL pmc.ncbi.nlm.nih.gov may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for Supplemental_Table_2.xlsx in pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 13 with repo: pmc.ncbi.nlm.nih.gov and accession_id: Supplemental_Table_2.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository pmc.ncbi.nlm.nih.gov unknown in Ontology. Skipping dataset page 13.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 14 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: Supplemental_Table_5.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: pmc.ncbi.nlm.nih.gov, identifier: Supplemental_Table_5.xlsx, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Supplemental_Table_5.xlsx for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier Supplemental_Table_5.xlsx for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Supplemental_Table_5.xlsx for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Supplemental_Table_5.xlsx for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier Supplemental_Table_5.xlsx for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier Supplemental_Table_5.xlsx for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL pmc.ncbi.nlm.nih.gov may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for Supplemental_Table_5.xlsx in pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 14 with repo: pmc.ncbi.nlm.nih.gov and accession_id: Supplemental_Table_5.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository pmc.ncbi.nlm.nih.gov unknown in Ontology. Skipping dataset page 14.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 15 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: Supplemental_Table_4.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: pmc.ncbi.nlm.nih.gov, identifier: Supplemental_Table_4.xlsx, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Supplemental_Table_4.xlsx for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier Supplemental_Table_4.xlsx for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Supplemental_Table_4.xlsx for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Supplemental_Table_4.xlsx for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier Supplemental_Table_4.xlsx for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier Supplemental_Table_4.xlsx for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL pmc.ncbi.nlm.nih.gov may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for Supplemental_Table_4.xlsx in pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 15 with repo: pmc.ncbi.nlm.nih.gov and accession_id: Supplemental_Table_4.xlsx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository pmc.ncbi.nlm.nih.gov unknown in Ontology. Skipping dataset page 15.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 16 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: Supplemental_Fig_3.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: pmc.ncbi.nlm.nih.gov, identifier: Supplemental_Fig_3.pdf, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Supplemental_Fig_3.pdf for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier Supplemental_Fig_3.pdf for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Supplemental_Fig_3.pdf for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Supplemental_Fig_3.pdf for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier Supplemental_Fig_3.pdf for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier Supplemental_Fig_3.pdf for pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL pmc.ncbi.nlm.nih.gov may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for Supplemental_Fig_3.pdf in pmc.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 16 with repo: pmc.ncbi.nlm.nih.gov and accession_id: Supplemental_Fig_3.pdf\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository pmc.ncbi.nlm.nih.gov unknown in Ontology. Skipping dataset page 16.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 17 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: GTEx_Analysis_v8_eQTL.tar\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: storage.googleapis.com, identifier: GTEx_Analysis_v8_eQTL.tar, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GTEx_Analysis_v8_eQTL.tar for storage.googleapis.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier GTEx_Analysis_v8_eQTL.tar for storage.googleapis.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier GTEx_Analysis_v8_eQTL.tar for storage.googleapis.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier GTEx_Analysis_v8_eQTL.tar for storage.googleapis.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier GTEx_Analysis_v8_eQTL.tar for storage.googleapis.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier GTEx_Analysis_v8_eQTL.tar for storage.googleapis.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: storage.googleapis.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL storage.googleapis.com may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for GTEx_Analysis_v8_eQTL.tar in storage.googleapis.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 17 with repo: storage.googleapis.com and accession_id: GTEx_Analysis_v8_eQTL.tar\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository storage.googleapis.com unknown in Ontology. Skipping dataset page 17.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 18 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 18: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 19 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 19: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1277 - INFO - Updated datasets len: 19\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 414 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 45, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'title', 'file_info',\n",
      "       'description', 'source_section', 'section_class', 'download_link',\n",
      "       'file_extension', 'a_attr_href', 'a_attr_class',\n",
      "       'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 173 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 183 - WARNING - No publication title found in the HTML data.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 643 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 494 - INFO - Parsed data extraction completed. Elements collected: 33\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 500 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 510 - INFO - HTML element classification not supported. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1012 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11444172\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1020 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11444172\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 635th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11445576)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11445576\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11445576 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 634 - INFO - Fetching data for pmc11445576\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 637 - INFO - Fetching data from scripts/exp_input/Local_fetched_data_1.parquet\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 438 - INFO - Raw xml data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11445576 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 452 - INFO - Skipping raw HTML/XML saving.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 458 - INFO - Using XMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 52 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 24 - INFO - Initializing xmlRetriever\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 466 - INFO - No additional data provided. Parsing raw data only.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 147 - INFO - Function call: parse_data(api_data(<class 'str'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11445576, additional_data, XML)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 162 - INFO - api_data converted to lxml element\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 466 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 153 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 154 - INFO - Consider migrating this function to the BaseRetriever class.\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 69 - INFO - Loading target sections for section name: supplementary_material_sections\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 477 - INFO - Found 6 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x15d6d2380>, <Element supplementary-material at 0x15d5dcf80>, <Element supplementary-material at 0x15d2f6980>, <Element supplementary-material at 0x15d2f5400>, <Element supplementary-material at 0x15de40100>, <Element supplementary-material at 0x15de40200>]\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 485 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x15d6d2380>, <Element supplementary-material at 0x15d5dcf80>, <Element supplementary-material at 0x15d2f6980>, <Element supplementary-material at 0x15d2f5400>, <Element supplementary-material at 0x15de40100>, <Element supplementary-material at 0x15de40200>]}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 574 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM1: Observed read depth ( ) is modeled as dependent on both latent CN ( ) and replication ( ) states across all genomic bins ( cells\u0001× loci) where the replication state depends on each cell’s time within S-phase ( ), each locus’s average RT ( ), and a replication stochasticity parameter ( ) (Fig. 1 [rid=Fig1] a–f, S1 [this file] ). All terms in the graphical model as well as additional mathematical, inference, and implementation details can be found in Fig. S1 [this file] and the Methods. \n",
      "The performance gap between PERT and Kronos was significant ( < 10 ) for all parameter combinations and increased as a function of cell CNA rate, number of clones, and noise (Fig. 1 [rid=Fig1] g, S2 [this file] a–j). PERT’s replication and CN accuracies decayed at high cell CNA rate and overdispersion ( ) values but remained robust to replication stochasticity ( ), GC bias ( ), number of clones, and number of cells (Fig. S2 [this file] a–t). Comparing the two PERT CN priors, we found higher accuracies when the composite prior was used, particularly at high cell CNA rates, and thus only used the composite CN prior throughout the rest of our analysis (Fig. S2 [this file] a–t). Accordingly, PERT significantly outperformed ( < 10 ) the Laks et al. classifier in phase accuracy for all simulated datasets and was critically robust to fluctuations in GC bias ( ), indicating that PERT’s performance will not fluctuate between DLP+ , 10X Chromium scDNA , and other scWGS modalities with unique GC bias relationships (Fig. S2 [this file] u–ad). Additional benchmarking information can be found in Supplementary Notes 1 [this file] – 2 [this file] . \n",
      "We combined diploid lymphoblastoid GM18507 cells (657 G1 cells, 585 S cells, 337 G2 cells, 1 clone,\u0001~ 0.01-0.1x coverage depth) and aneuploid breast cancer T47D cells (703 G1 cells, 623 S cells, 522 G2 cells, 5 clones,\u0001~ 0.01-0.1x coverage depth) into one merged sample for analysis with PERT (Fig. S3 [this file] a). Both GM18507 and T47D samples were enriched for mid-S-phase cells which was expected based on the sorting protocol described in Laks et al. (Fig. S3 [this file] b). \n",
      "Encouragingly, we found that both cell lines had RT Pearson correlations of ≥0.99 between their merged and split PERT runs (Fig. 2 [rid=Fig2] e, S3 [this file] c). Again, we found true and inferred clone RT profiles all had Pearson correlations ≥0.99 (Fig. S3 [this file] d, e). \n",
      "We found that\u0001>90% of all mislabeled cells were accurately recovered (predicted G1/2) across all permutation datasets without compromising identification of non-mislabeled S-phase cells and cell line-specific RT profiles (Fig. 2 [rid=Fig2] f, g, S3 [this file] f). Orthogonal per-cell features for these cells were concordant with late S-phase (Fig. 2 [rid=Fig2] f, S3 [this file] g), suggesting they were actually in late S-phase but mistakenly sorted into G2-phase by FACS. \n",
      "We first compared the 500kb resolution RT profiles from these unsorted scWGS libraries to ENCODE RepliSeq data, confirming that PERT RT profiles were of high-quality before proceeding with our analysis (Fig. 3 [rid=Fig3] b, S4 [this file] , Supplementary Note 3 [this file] ). \n",
      "To test this, we computed a reference RT profile from the ancestral hTERT WT ( and WT) cells with no CNAs (SA039 clone A) and then counted the number of gains, losses, and CNA breakpoints in CN profiles that descended from this ancestral WT population (Fig. 3 [rid=Fig3] c,d, S5 [this file] a–c). We found that gains preferentially arose from early RT loci, losses from late RT loci, and CNA breakpoints from late RT loci ( < 10 ) (Fig. 3 [rid=Fig3] e,f, S5 [this file] d,e). \n",
      "Two representative examples of isogenic CNAs are the 19p gain and 19q loss which are both specific to clone G in sample SA906b (Fig. S6 [this file] a). Across all isogenic events, we found that losses significantly shifted to earlier RT ( = 3.85 − 37 per bin, = 1.32 − 5 per event) while gains shifted to later RT ( = 5.97 − 6 per bin, = 3.32 − 1 per event) (Fig. 3 [rid=Fig3] g, S6 [this file] b-d). \n",
      "Investigating these RT profiles inferred by either RepliSeq or PERT at chromosome-level resolution, we found high agreement across cell types on most autosomes with chromosome X having the highest variability in mean RT (Fig. 4 [rid=Fig4] f, S7 [this file] ). \n",
      "This correspondence was further supported by chrX having a lower BAF in S-phase cells than G1/2-phase cells, indicating that the B-allele was replicating later than the A-allele in all hTERT samples (Fig. S8 [this file] a–c). \n",
      "As expected, all 19 tumors had earlier chrX RT than the allelically balanced (BAF=0.5) hTERT samples, with a negative correlation between chrX RT and DNA BAF, confirming that many of these tumors had more copies of Xa than Xi (Fig. 5 [rid=Fig5] d, e, S8 [this file] d). \n",
      "Autosomes maintained a 1:1 relationship between RNA and DNA BAF, indicating that each allelic copy is equally transcribed, and samples with Xi alleles present had lower RNA BAF than DNA BAF, in line with less per-copy transcription from the Xi allele (Fig. S8 [this file] e). \n",
      "We identified clonal loss of heterozygosity (LOH, BAF=0) on Xq but not Xp in 4 of 19 tumors (2 HGSOC, 2 TNBC, Fig. S8 [this file] f). The clonal nature of these events indicate that they all happened early in tumor evolution (Fig. S8 [this file] f). \n",
      "Encouragingly, high PERT G1/2-phase fractions correlated with high doubling times and high scRNA G1-phase fractions (Fig. S9 [this file] a–c, Supplementary Data 2 [rid=MOESM5] ). Interestingly, the scWGS-defined cell cycle fractions had higher correlation with doubling time ( = 1.00, = 3.3 − 3) than the scRNA-defined cell cycle fractions ( = 0.88, = 3.1 − 1), suggesting that direct measurement of DNA replication in scWGS might be a better proxy measurement for the proliferation rate than expression of cell cycle marker genes in scRNA (Fig. S9 [this file] a–c). In the hTERT WT data, we found that SPE scores had a positive trend with expansion ( = 0.51, = 2.5 − 1, Fig. S9 [this file] d, e), indicating that more S-phase cells corresponded to faster proliferation in normal epithelial cells. \n",
      "To determine whether SPE corresponded to high fitness of genomically unstable cells, we analyzed time-series scWGS data generated from −/− hTERT cell lines and serially propagated TNBC patient derived xenografts (PDXs) (Fig. 6 [rid=Fig6] e, S9 [this file] d). We found no consistent correlation between SPE scores and observed fitness with intra-sample correlations ranging from = 0.20 to = − 0.43 (Fig. 6 [rid=Fig6] f, S9 [this file] f,g, S10 [this file] a–d). \n",
      "Indeed, we found this correlation to be significantly more negative in the on-treatment than the untreated context (Wald = 5.12 − 2, Fig. 6 [rid=Fig6] f, S10 [this file] e–h). \n",
      "Patient OV-081 presented with a primary tumor in the left adnexa consisting of exclusively NGD tumor cells and a metastasis in the omentum consisting of mostly WGD tumor cells (Fig. 6 [rid=Fig6] g, S11 [this file] a). Running PERT on the tumor and contaminating normal scWGS cells from both sites, we found that the NGD tumor clones (B-D) had the highest SPE scores, followed by WGD tumor clone (A) and the normal clone (E) (Fig. 6 [rid=Fig6] h, S11 [this file] c). The discrepancy in clone SPE was validated with site-specific scRNA data showing that 53% of tumor cells are cycling in the NGD-dominant left adnexa (22% S-phase, 31% G2/M-phase) but only 33% of tumor cells are cycling in the WGD-dominant omentum (16% S-phase, 17% G2/M-phase, Fig. S11 [this file] b). \n",
      "Here, we found that both WGD clones were depleted for S-phase cells (rank 7/13 and 13/13 in overall clone SPE scores; 7 of 8 clone\u0001× timepoint combinations with\u0001>10 G1/2-phase cells have negative SPE) despite having the highest overall fitness (rank 1/13 and 2/13 in fitness scores) (Fig. S12 [this file] ). \n",
      "The input for PERT is scWGS read depth ( ) and called CN states for all bins ( cells\u0001× loci) (Fig. S1 [this file] a). \n",
      "The initialized sets of cells are passed into a probabilistic model which infers somatic CN ( ) and replication states ( ) through three distinct learning steps (Fig. S1 [this file] c–e). In Step 1, PERT learns parameters associated with library-level GC bias ( , ) and sequencing overdispersion ( ) by training on high-confidence G1/2 cells (Fig. S1 [this file] c). Once , and have been learned in Step 1, we can condition on them in Step 2 (Fig. S1 [this file] d). CN prior concentrations are set for each cell by using the consensus CN profile of the most similar G1/2 clone or a composite scoring of the most similar G1/2 clone and cell CN profiles (Fig. S1 [this file] f) A full list of model parameters, domains, and distributions can be found in Fig. S1 [this file] b. Step 3 is an optional final step which learns CN and replication states for high-confidence G1/2 cells (Fig. S1 [this file] e). \n",
      "A comparison of the two methods can be seen when benchmarking PERT on simulated data (Supplementary Notes 1 [this file] - 2 [this file] ). \n",
      "The Laks classifier was retrained and benchmarked with said features removed prior to deployment on simulated data (Fig. S13 [this file] ). \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM2: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM3: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM4: Interestingly, many of these allelic imbalances were fully clonal events and arose from either loss of Xi (4/19 clonal full chrX LOH, 4/19 clonal Xq LOH, 2/19 clonal partial LOH, 1/19 subclonal LOH) or gain of Xa (1/19 clonal full chrX, 4/19 subclonal and/or partial) (Supplementary Data 1 [this file] ), suggesting that Xa > Xi imbalances emerge early in tumor evolution and that both Xa gain and Xi loss may be independently favorable events. \n",
      "SIGNALS output for all scDNA samples is shown in Supplementary Data 1 [this file] . \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM5: Encouragingly, high PERT G1/2-phase fractions correlated with high doubling times and high scRNA G1-phase fractions (Fig. S9 [rid=MOESM1] a–c, Supplementary Data 2 [this file] ). \n",
      "PERT output for these sames is shown in Supplementary Data 2 [this file] . \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM6: Further information on research design is available in the Nature Portfolio Reporting Summary [this file] linked to this article. \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 263 - INFO - Extracting links from full XML content.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 301 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 309 - INFO - Content length: 302937\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 326 - INFO - Prompt messages total length: 76488 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 332 - INFO - Prompt ID: gemini-2.5-flash-0-07f194b05e43d406315d6ca5f87a6bba785e2db7a3290a0599b414e38566ad1d\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 351 - INFO - Requesting datasets from content using model: gemini-2.5-flash, temperature: 0, messages length: 76488 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-b5716c36-df42-47da-8652-ed6109f7cbb1\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"12786373\\\",\\n    \\\"repository_reference\\\": \\\"https://doi.org/10.5281/zenodo.12786373\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"6362733\\\",\\n    \\\"repository_reference\\\": \\\"https://zenodo.org/doi/10.5281/zenodo.6362733\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"EGAS00001004448\\\",\\n    \\\"repository_reference\\\": \\\"https://ega-archive.org/search/EGAS00001004448\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"EGAS00001003190\\\",\\n    \\\"repository_reference\\\": \\\"https://ega-archive.org/search/EGAS00001003190\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn25569736\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn25569736\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"PRJNA498809\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA498809\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"12786373\\\",\\n    \\\"repository_reference\\\": \\\"https://doi.org/10.5281/zenodo.12786373\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"6362733\\\",\\n    \\\"repository_reference\\\": \\\"https://zenodo.org/doi/10.5281/zenodo.6362733\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"EGAS00001004448\\\",\\n    \\\"repository_reference\\\": \\\"https://ega-archive.org/search/EGAS00001004448\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"EGAS00001003190\\\",\\n    \\\"repository_reference\\\": \\\"https://ega-archive.org/search/EGAS00001003190\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn25569736\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn25569736\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"PRJNA498809\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA498809\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1758210097,\n",
      "    \"model\": \"gemini-2.5-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 100932,\n",
      "        \"completion_tokens\": 359,\n",
      "        \"total_tokens\": 105336,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 4045,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 664 - INFO - Deduplicating response with 6 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: 12786373\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: 6362733\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: EGAS00001004448\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: EGAS00001003190\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn25569736\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: PRJNA498809\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': '12786373', 'repository_reference': 'https://doi.org/10.5281/zenodo.12786373', 'dataset_id': '12786373'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://doi.org/__ID__ to value: https://doi.org/10.5281/zenodo.12786373\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://doi.org/10.5281/zenodo.12786373\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 583 - INFO - Candidate data_repository: doi.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://doi.org/10.5281/zenodo.12786373, identifier: 12786373, dataset_page: https://doi.org/10.5281/zenodo.12786373\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://doi.org/10.5281/zenodo.12786373\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 12786373 for https://doi.org/10.5281/zenodo.12786373\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier 12786373 for https://doi.org/10.5281/zenodo.12786373\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier 12786373 for https://doi.org/10.5281/zenodo.12786373\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier 12786373 for https://doi.org/10.5281/zenodo.12786373\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier 12786373 for https://doi.org/10.5281/zenodo.12786373\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier 12786373 for https://doi.org/10.5281/zenodo.12786373\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1157 - INFO - Found id_pattern match for https://doi.org/10.5281/zenodo.12786373 in [1-9][a-zA-Z0-9]{3}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: 10\\.\\d{4,9}/[^\\s,]+ to value: https://doi.org/10.5281/zenodo.12786373\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(10\\.\\d{4,9}/[^\\s,]+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(10\\.\\d{4,9}/[^\\s,]+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: 12786373\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: 12786373\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://doi.org/10.5281/zenodo.12786373\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: 12786373\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: 12786373\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: 12786373, pdb, https://doi.org/10.5281/zenodo.12786373\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: 12786373\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://doi.org/10.5281/zenodo.12786373\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': '12786373', 'data_repository': 'pdb', 'dataset_webpage': 'https://doi.org/10.5281/zenodo.12786373', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': '6362733', 'repository_reference': 'https://zenodo.org/doi/10.5281/zenodo.6362733', 'dataset_id': '6362733'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: 10\\.\\d{4,9}/[^\\s,]+ to value: https://zenodo.org/doi/10.5281/zenodo.6362733\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://zenodo.org/doi/10.5281/zenodo.6362733\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(10\\.\\d{4,9}/[^\\s,]+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(10\\.\\d{4,9}/[^\\s,]+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: 6362733\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: 6362733\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://zenodo.org/doi/10.5281/zenodo.6362733, identifier: 6362733, dataset_page: https://zenodo.org/doi/10.5281/zenodo.6362733\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://zenodo.org/doi/10.5281/zenodo.6362733\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 6362733 for https://zenodo.org/doi/10.5281/zenodo.6362733\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier 6362733 for https://zenodo.org/doi/10.5281/zenodo.6362733\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier 6362733 for https://zenodo.org/doi/10.5281/zenodo.6362733\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier 6362733 for https://zenodo.org/doi/10.5281/zenodo.6362733\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier 6362733 for https://zenodo.org/doi/10.5281/zenodo.6362733\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier 6362733 for https://zenodo.org/doi/10.5281/zenodo.6362733\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1157 - INFO - Found id_pattern match for https://zenodo.org/doi/10.5281/zenodo.6362733 in [1-9][a-zA-Z0-9]{3}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://zenodo.org/doi/10.5281/zenodo.6362733\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: 6362733\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: 6362733\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: 6362733, pdb, https://zenodo.org/doi/10.5281/zenodo.6362733\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: 6362733\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://zenodo.org/doi/10.5281/zenodo.6362733\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': '6362733', 'data_repository': 'pdb', 'dataset_webpage': 'https://zenodo.org/doi/10.5281/zenodo.6362733', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'EGAS00001004448', 'repository_reference': 'https://ega-archive.org/search/EGAS00001004448', 'dataset_id': 'EGAS00001004448'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: EGAS00001004448\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: EGAS00001004448\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://ega-archive.org/search/EGAS00001004448, identifier: EGAS00001004448, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://ega-archive.org/search/EGAS00001004448\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier EGAS00001004448 for https://ega-archive.org/search/EGAS00001004448\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier EGAS00001004448 for https://ega-archive.org/search/EGAS00001004448\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier EGAS00001004448 for https://ega-archive.org/search/EGAS00001004448\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier EGAS00001004448 for https://ega-archive.org/search/EGAS00001004448\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier EGAS00001004448 for https://ega-archive.org/search/EGAS00001004448\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier EGAS00001004448 for https://ega-archive.org/search/EGAS00001004448\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://ega-archive.org/search/EGAS00001004448\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://ega-archive.org/search/EGAS00001004448\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://ega-archive.org/search/EGAS00001004448\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: EGAS00001004448\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: EGAS00001004448\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: EGAS00001004448, ega-archive.org, https://ega-archive.org/search/EGAS00001004448\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: EGAS00001004448\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: ega-archive.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://ega-archive.org/search/EGAS00001004448\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'EGAS00001004448', 'data_repository': 'ega-archive.org', 'dataset_webpage': 'https://ega-archive.org/search/EGAS00001004448', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'EGAS00001003190', 'repository_reference': 'https://ega-archive.org/search/EGAS00001003190', 'dataset_id': 'EGAS00001003190'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: EGAS00001003190\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: EGAS00001003190\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://ega-archive.org/search/EGAS00001003190, identifier: EGAS00001003190, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://ega-archive.org/search/EGAS00001003190\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier EGAS00001003190 for https://ega-archive.org/search/EGAS00001003190\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier EGAS00001003190 for https://ega-archive.org/search/EGAS00001003190\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier EGAS00001003190 for https://ega-archive.org/search/EGAS00001003190\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier EGAS00001003190 for https://ega-archive.org/search/EGAS00001003190\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier EGAS00001003190 for https://ega-archive.org/search/EGAS00001003190\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier EGAS00001003190 for https://ega-archive.org/search/EGAS00001003190\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://ega-archive.org/search/EGAS00001003190\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://ega-archive.org/search/EGAS00001003190\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://ega-archive.org/search/EGAS00001003190\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: EGAS00001003190\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: EGAS00001003190\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: EGAS00001003190, ega-archive.org, https://ega-archive.org/search/EGAS00001003190\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: EGAS00001003190\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: ega-archive.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://ega-archive.org/search/EGAS00001003190\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'EGAS00001003190', 'data_repository': 'ega-archive.org', 'dataset_webpage': 'https://ega-archive.org/search/EGAS00001003190', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn25569736', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn25569736', 'dataset_id': 'syn25569736'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn25569736\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn25569736\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn25569736, identifier: syn25569736, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn25569736\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn25569736 for https://www.synapse.org/#!Synapse:syn25569736\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn25569736 for https://www.synapse.org/#!Synapse:syn25569736\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn25569736 for https://www.synapse.org/#!Synapse:syn25569736\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn25569736 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn25569736\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn25569736\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn25569736\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn25569736\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn25569736\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn25569736\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn25569736\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn25569736\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn25569736\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn25569736, synapse.org, https://www.synapse.org/#!Synapse:syn25569736\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn25569736\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn25569736\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn25569736', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn25569736', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'PRJNA498809', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA498809', 'dataset_id': 'PRJNA498809'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: PRJNA498809\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: PRJNA498809\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA498809, identifier: PRJNA498809, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA498809\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier PRJNA498809 for https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA498809\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier PRJNA498809 for https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA498809\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier PRJNA498809 for https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA498809\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier PRJNA498809 for https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA498809\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier PRJNA498809 for https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA498809\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier PRJNA498809 for https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA498809\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA498809\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA498809\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA498809\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: PRJNA498809\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: PRJNA498809\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: PRJNA498809, www.ncbi.nlm.nih.gov, https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA498809\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: PRJNA498809\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: www.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA498809\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'PRJNA498809', 'data_repository': 'www.ncbi.nlm.nih.gov', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA498809', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 278 - INFO - Augmented dataset links: [{'dataset_identifier': '12786373', 'data_repository': 'pdb', 'dataset_webpage': 'https://doi.org/10.5281/zenodo.12786373', 'citation_type': 'n/a'}, {'dataset_identifier': '6362733', 'data_repository': 'pdb', 'dataset_webpage': 'https://zenodo.org/doi/10.5281/zenodo.6362733', 'citation_type': 'n/a'}, {'dataset_identifier': 'EGAS00001004448', 'data_repository': 'ega-archive.org', 'dataset_webpage': 'https://ega-archive.org/search/EGAS00001004448', 'citation_type': 'n/a'}, {'dataset_identifier': 'EGAS00001003190', 'data_repository': 'ega-archive.org', 'dataset_webpage': 'https://ega-archive.org/search/EGAS00001003190', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn25569736', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn25569736', 'citation_type': 'n/a'}, {'dataset_identifier': 'PRJNA498809', 'data_repository': 'www.ncbi.nlm.nih.gov', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/sra/?term=PRJNA498809', 'citation_type': 'n/a'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1191 - INFO - Fetching metadata for 6 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: 12786373\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: pdb, identifier: 12786373, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 12786373 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier 12786373 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier 12786373 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier 12786373 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier 12786373 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for 12786373 in pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 1 with repo: pdb and accession_id: 12786373\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.rcsb.org/structure/12786373\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: 6362733\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: pdb, identifier: 6362733, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 6362733 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier 6362733 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier 6362733 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier 6362733 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier 6362733 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for 6362733 in pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 2 with repo: pdb and accession_id: 6362733\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.rcsb.org/structure/6362733\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: EGAS00001004448\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: ega-archive.org, identifier: EGAS00001004448, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: ega-archive.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for EGAS00001004448 in ega-archive.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 3 with repo: ega-archive.org and accession_id: EGAS00001004448\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://ega-archive.org/studies/EGAS00001004448\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 3: Contact to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: EGAS00001003190\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: ega-archive.org, identifier: EGAS00001003190, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: ega-archive.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for EGAS00001003190 in ega-archive.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 4 with repo: ega-archive.org and accession_id: EGAS00001003190\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://ega-archive.org/studies/EGAS00001003190\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 4: Contact to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn25569736\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn25569736, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn25569736 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn25569736 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn25569736 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn25569736 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 5 with repo: synapse.org and accession_id: syn25569736\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn25569736\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 5: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: PRJNA498809\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: www.ncbi.nlm.nih.gov, identifier: PRJNA498809, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier PRJNA498809 for www.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier PRJNA498809 for www.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier PRJNA498809 for www.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier PRJNA498809 for www.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier PRJNA498809 for www.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier PRJNA498809 for www.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: www.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL www.ncbi.nlm.nih.gov may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for PRJNA498809 in www.ncbi.nlm.nih.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 6 with repo: www.ncbi.nlm.nih.gov and accession_id: PRJNA498809\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository www.ncbi.nlm.nih.gov unknown in Ontology. Skipping dataset page 6.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1277 - INFO - Updated datasets len: 6\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 287 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 12, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'source_url', 'download_link',\n",
      "       'title', 'content_type', 'id', 'caption', 'description',\n",
      "       'source_section', 'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 500 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 506 - INFO - XML element classification not needed. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1012 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11445576\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1020 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11445576\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 636th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11452344)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11452344\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11452344 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 634 - INFO - Fetching data for pmc11452344\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 637 - INFO - Fetching data from scripts/exp_input/Local_fetched_data_1.parquet\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 438 - INFO - Raw xml data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11452344 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 452 - INFO - Skipping raw HTML/XML saving.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 458 - INFO - Using XMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 52 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 24 - INFO - Initializing xmlRetriever\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 466 - INFO - No additional data provided. Parsing raw data only.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 147 - INFO - Function call: parse_data(api_data(<class 'str'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11452344, additional_data, XML)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 162 - INFO - api_data converted to lxml element\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 466 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 153 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 154 - INFO - Consider migrating this function to the BaseRetriever class.\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 69 - INFO - Loading target sections for section name: supplementary_material_sections\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 477 - INFO - Found 3 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x15d172900>, <Element supplementary-material at 0x15d4b6ec0>, <Element supplementary-material at 0x15d3cc500>]\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 485 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x15d172900>, <Element supplementary-material at 0x15d4b6ec0>, <Element supplementary-material at 0x15d3cc500>]}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 574 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM1: In the Supplementary Results [this file] , we detail analyses based on all Olink proteins. \n",
      "We represented some of the key DAPs pertaining to the different group comparisons in Fig. 5e [rid=Fig5] , with more description of these proteins in Supplementary Results [this file] . Higher levels of DDC in the non-AD group was particularly attributable to the high proportion of patients with a Parkinsonian disorder (Supplementary Fig. 3 [this file] ), in line with recent work . \n",
      "To provide a granular and sequential measure of advancing AD pathology, we used trajectory inference methods to compare key DAPs on a common continuous scale, representing increasing Aβ and fibrillar tau pathology (Fig. 5f [rid=Fig5] and Supplementary Fig. 4 [this file] ). \n",
      "Lastly, half of our module assignment showed consistency with previous CSF proteomic modules, particularly those related to lysosomes, the complement cascade and axonal development (details in Supplementary Fig. 5 [this file] ). \n",
      "We noted a clear interaction between 4 genotype and apoprotein E (ApoE) levels in Olink and provided all analyses to that effect in Supplementary Results [this file] and Supplementary Figs. 1 [this file] and 2 [this file] . Given that we cannot yet explain this effect, we have removed ApoE from the main analyses and discuss it in the Supplementary Results [this file] section ‘Olink CSF ApoE results’. \n",
      "All individual significant GO terms are provided in Supplementary Tables 6–8 [this file] . \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM2: Further information on research design is available in the Nature Portfolio Reporting Summary [this file] linked to this article. \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM3: All summary statistics from the differential expression analyses are reported in Supplementary Table 1 [this file] . \n",
      "In this validation cohort, we compared participants based on their Aβ status (213 A versus 418 A ) because tau-PET was unavailable (Fig. 2d [rid=Fig2] and see Supplementary Table 2 [this file] for all statistical results). We analyzed SomaLogic proteins that matched the ones analyzed in BioFINDER-2, yielding an overall 90% consistency in significant DAPs between cohorts (Fig. 1e [rid=Fig1] and Supplementary Table 3 [this file] ). \n",
      "In complementary analyses, we investigated whether the regional gene expression pattern of DAPs correlated with the pattern of Aβ or tau aggregates in the brain using imaging transcriptomics (Extended Data Fig. 1 [rid=Fig7] , Supplementary Table 4 [this file] and Supplementary Results [rid=Sec29] ). \n",
      "See Supplementary Table 5 [this file] for all statistical results across the 127 DAPs and Extended Data Fig. 2 [rid=Fig8] for results when Aβ- and tau-PET are investigated individually (that is, separate regression models for each pathology). \n",
      "Functional enrichment analysis allowed further understanding of the biological roles of the DAPs (summary results in Fig. 3c [rid=Fig3] ; see Supplementary Table 6 [this file] for all significant gene ontology (GO) terms). \n",
      "We compared the non-AD group with the A T group (Fig. 5a [rid=Fig5] ), as well as the A T group (Fig. 5b [rid=Fig5] , see statistical results in Supplementary Table 1 [this file] ), to assess which DAPs in AD were also differentially abundant in other neurodegenerative contexts. Enrichment analyses on the set of significantly less abundant proteins common in the non-AD and the A T groups highlighted biological processes related to axonogenesis, axon development and neuronal morphogenesis (Supplementary Table 7 [this file] ), supporting the notion that these processes are probably impaired in the later stage of neurodegeneration. \n",
      "Six principal modules were identified (Fig. 6a [rid=Fig6] and see Supplementary Table 1 [this file] for protein assignment to each module). \n",
      "To better characterize the different modules, cell-type and functional enrichment analyses were also performed (summary in Fig. 6d [rid=Fig6] ; all significant terms in Supplementary Table 8 [this file] ). \n",
      "Distribution of the proteins meeting the LOD criteria across the different Olink panels is provided in Supplementary Table 9 [this file] . \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 263 - INFO - Extracting links from full XML content.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 301 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 309 - INFO - Content length: 200720\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 326 - INFO - Prompt messages total length: 50933 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 332 - INFO - Prompt ID: gemini-2.5-flash-0-a2945b5ca3ebc8d90368519c3e34389cad627354d12519e5da9ee775d759a2f0\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 351 - INFO - Requesting datasets from content using model: gemini-2.5-flash, temperature: 0, messages length: 50933 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-d15486fa-e9da-4d03-a9c0-89b1960f3b10\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"adni.loni.usc.edu\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn52293433\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn52293433\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://portal.brain-map.org/atlases-and-data/rnaseq\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"adni.loni.usc.edu\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn52293433\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn52293433\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://portal.brain-map.org/atlases-and-data/rnaseq\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1758210127,\n",
      "    \"model\": \"gemini-2.5-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 61850,\n",
      "        \"completion_tokens\": 144,\n",
      "        \"total_tokens\": 66781,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 4787,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 664 - INFO - Deduplicating response with 3 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn52293433\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'adni.loni.usc.edu', 'dataset_id': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: adni.loni.usc.edu, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: adni.loni.usc.edu\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL adni.loni.usc.edu may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'adni.loni.usc.edu', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn52293433', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn52293433', 'dataset_id': 'syn52293433'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn52293433\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn52293433\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn52293433, identifier: syn52293433, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn52293433\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn52293433 for https://www.synapse.org/#!Synapse:syn52293433\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn52293433 for https://www.synapse.org/#!Synapse:syn52293433\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn52293433 for https://www.synapse.org/#!Synapse:syn52293433\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn52293433 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn52293433\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn52293433\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn52293433\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn52293433\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn52293433\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn52293433\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn52293433\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn52293433\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn52293433\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn52293433, synapse.org, https://www.synapse.org/#!Synapse:syn52293433\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn52293433\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn52293433\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn52293433', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn52293433', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://portal.brain-map.org/atlases-and-data/rnaseq'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://portal.brain-map.org/atlases-and-data/rnaseq, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://portal.brain-map.org/atlases-and-data/rnaseq\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://portal.brain-map.org/atlases-and-data/rnaseq\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'portal.brain-map.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 278 - INFO - Augmented dataset links: [{'dataset_identifier': 'n/a', 'data_repository': 'adni.loni.usc.edu', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn52293433', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn52293433', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'portal.brain-map.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1191 - INFO - Fetching metadata for 3 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 1: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn52293433\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn52293433, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn52293433 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn52293433 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn52293433 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn52293433 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn52293433\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn52293433\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 3: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1277 - INFO - Updated datasets len: 3\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 287 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 6, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'source_url', 'download_link',\n",
      "       'title', 'content_type', 'id', 'caption', 'description',\n",
      "       'source_section', 'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 500 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 506 - INFO - XML element classification not needed. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1012 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11452344\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1020 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11452344\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 637th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11452345)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11452345\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11452345 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 634 - INFO - Fetching data for pmc11452345\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 637 - INFO - Fetching data from scripts/exp_input/Local_fetched_data_1.parquet\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 438 - INFO - Raw xml data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11452345 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 452 - INFO - Skipping raw HTML/XML saving.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 458 - INFO - Using XMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 52 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 24 - INFO - Initializing xmlRetriever\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 466 - INFO - No additional data provided. Parsing raw data only.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 147 - INFO - Function call: parse_data(api_data(<class 'str'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11452345, additional_data, XML)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 162 - INFO - api_data converted to lxml element\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 466 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 153 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 154 - INFO - Consider migrating this function to the BaseRetriever class.\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 69 - INFO - Loading target sections for section name: supplementary_material_sections\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 477 - INFO - Found 2 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x15d407bc0>, <Element supplementary-material at 0x15d42c400>]\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 485 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x15d407bc0>, <Element supplementary-material at 0x15d42c400>]}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 574 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM1: Further information on research design is available in the Nature Portfolio Reporting Summary [this file] linked to this article. \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM2: We generated snRNA-seq data from a total of 87 individuals (mean age = 54.85 years; range, 26–84 years; 32 women and 55 men; 54 individuals with a psychiatric disorder and 33 neurotypical individuals; Supplementary Tables 1 [this file] and 2 [this file] and Extended Data Fig. 1a [rid=Fig8] ), totaling around 800,000 nuclei. There was no difference in mean number of nuclei per cell type between individuals with a psychiatric disorder and neurotypical individuals (Supplementary Table 3 [this file] ). \n",
      "Most cell types did not change in abundance, only the proportion of OPCs significantly decreased with age (false discovery rate (FDR)-adjusted\u00010.002), going along with a trend-line increase in oligodendrocytes (FDR-adjusted\u00010.05) and decrease in VIP inhibitory neurons (In_VIP, FDR-adjusted\u00010.05; Extended Data Fig. 4a [rid=Fig11] and Supplementary Table 4 [this file] ). \n",
      "Changes in gene expression were detected in all identified cell types, with the largest number of DE genes in upper-layer excitatory neurons (Exc_L2–L3) neurons (Supplementary Tables 5 [this file] and 6 [this file] ). This analysis showed that In_LAMP5_2 neurons, an inhibitory neuron subtype characterized by the coexpression of and (Extended Data Fig. 3b [rid=Fig10] ), showed by far the most relative DE genes with age, followed by the deep-layer neuron cluster Exc_L4–L6_2 (Fig. 2b [rid=Fig2] and Supplementary Table 7 [this file] ), a finding supported by variance partitioning analysis (Extended Data Fig. 4c [rid=Fig11] and Supplementary Table 8 [this file] ). \n",
      "For clarity, only the value for the comparison between In_LAMP5_2 neurons and all other cell types is shown (*** < 0.001); exact values are shown in Supplementary Table 7 [this file] . \n",
      "The vast majority of DE genes were unique to a single cell type, followed by shared DE genes between groups of two to three cell types (Extended Data Fig. 6a,b [rid=Fig13] ). was the only gene shared across all major cell types at this cutoff (Fig. 2c [rid=Fig2] and Supplementary Table 9 [this file] ), whereas no common DE genes across all 21 cell types were identified (Extended Data Fig. 6a,b [rid=Fig13] ). \n",
      "The identified DE genes (Supplementary Table 10 [this file] ) showed significant overlap with those previously reported in (pre)frontal cortex bulk data (Supplementary Table 11 [this file] ), emphasizing the validity of our analysis. For all investigated cell types except In_PVALB_Ch neurons, Fisher’s exact test revealed a significant overlap in upregulated age-associated genes with the highest odds ratio in microglia and Exc_L4–L6_1 neurons (Fig. 5a [rid=Fig5] and Supplementary Table 12 [this file] ). Downregulated age-associated genes significantly overlapped across all cell types, with the highest odds ratio in astrocytes, In_LAMP5_2 neurons and microglia (Fig. 5a [rid=Fig5] and Supplementary Table 12 [this file] ). Moreover, the directionality of expression changes (log FC) was highly congruent, with high correlations of the effect sizes between the overlapping DE genes (Spearman correlation coefficient ( ) ranging from 0.58 in In_PVALB_Ch neurons to 0.92 in microglia) (Fig. 5a,b [rid=Fig5] and Supplementary Table 12 [this file] ). \n",
      "Exact values are shown in Supplementary Table 12 [this file] . \n",
      "Genes downregulated in excitatory and inhibitory neurons and astrocytes in individuals with AD also showed significant overlap with the age-downregulated DE genes in the corresponding cell types in our dataset (Fig. 6a,b [rid=Fig6] and Supplementary Table 13 [this file] ). \n",
      "Exact values are shown in Supplementary Table 13 [this file] . \n",
      "However, we did not observe accelerated epigenetic aging in individuals with psychiatric diseases (Supplementary Table 14 [this file] ). \n",
      "Multiple linear regression confirmed a significant transcriptomic age acceleration in individuals with psychiatric disease compared to neurotypical individuals ( = 0.02; Supplementary Table 15 [this file] ). \n",
      "We performed differential gene expression analysis within our datasets to identify disease-associated genes (Supplementary Table 16 [this file] ). Disease-associated genes were identified in four excitatory neuron cell types, of which three cell types showed a significant overlap with age-regulated genes (Fisher’s exact test; FDR-adjusted < 0.05; Exc_L2–L3, Exc_L4–L6_1 and Exc_L4–L6_3; Supplementary Table 17 [this file] ). Across 16 cell types, we could show that age- and SCZ-associated genes significantly overlap, and more than 80% of overlapping genes are regulated in a concordant direction (Fig. 7b [rid=Fig7] and Supplementary Table 18 [this file] ). \n",
      "To understand whether the convergent effects of aging and psychopathology are in part driven by genetic liability, we first calculated polygenic risk scores (PRSs) for SCZ and cross-disorder psychiatric disease (Supplementary Table 19 [this file] ) in our cohort. \n",
      "Supplementary Table 1 [this file] provides a summary of sample characteristics, and Supplementary Table 2 [this file] provides detailed information on all donor characteristics. \n",
      "The number of DE genes (at FDR-adj. p-value < 0.05; N) per cell type is shown in Supplementary Table 5 [this file] . Exact p-values and N (that is genes) per cell type are shown in Supplementary Table 8 [this file] . \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 263 - INFO - Extracting links from full XML content.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 301 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 309 - INFO - Content length: 263342\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 326 - INFO - Prompt messages total length: 66589 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 332 - INFO - Prompt ID: gemini-2.5-flash-0-3981ba2a8b2f3165b76015f0506d52a1345f80e712eec092c49ed1e8bfd5e893\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 351 - INFO - Requesting datasets from content using model: gemini-2.5-flash, temperature: 0, messages length: 66589 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-4fe419fb-95c3-467d-8528-ca1f3e0fa625\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE254293\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE254293\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE254569\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE254569\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn33235943\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/Synapse:syn33235943\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn39718968\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/Synapse:syn39718968\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"19426775\\\",\\n    \\\"repository_reference\\\": \\\"https://figshare.com/articles/dataset/scz2022/19426775\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"14672034\\\",\\n    \\\"repository_reference\\\": \\\"https://figshare.com/articles/dataset/cdg2019/14672034\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"l7aiRr1UEgdoJfZ\\\",\\n    \\\"repository_reference\\\": \\\"https://vu.data.surfsara.nl/index.php/s/l7aiRr1UEgdoJfZ\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"14102594\\\",\\n    \\\"repository_reference\\\": \\\"https://figshare.com/articles/dataset/PGC3_bipolar_disorder_GWAS_summary_statistics/14102594\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"3203\\\",\\n    \\\"repository_reference\\\": \\\"https://datashare.ed.ac.uk/handle/10283/3203\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"I9_HYPERTENSION.gwas.imputed_v3.both_sexes.tsv.bgz\\\",\\n    \\\"repository_reference\\\": \\\"https://broad-ukb-sumstats-us-east-1.s3.amazonaws.com/round2/additive-tsvs/I9_HYPERTENSION.gwas.imputed_v3.both_sexes.tsv.bgz\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://github.com/thewonlab/H-MAGMA\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE254293\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE254293\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE254569\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE254569\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn33235943\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/Synapse:syn33235943\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn39718968\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/Synapse:syn39718968\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"19426775\\\",\\n    \\\"repository_reference\\\": \\\"https://figshare.com/articles/dataset/scz2022/19426775\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"14672034\\\",\\n    \\\"repository_reference\\\": \\\"https://figshare.com/articles/dataset/cdg2019/14672034\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"l7aiRr1UEgdoJfZ\\\",\\n    \\\"repository_reference\\\": \\\"https://vu.data.surfsara.nl/index.php/s/l7aiRr1UEgdoJfZ\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"14102594\\\",\\n    \\\"repository_reference\\\": \\\"https://figshare.com/articles/dataset/PGC3_bipolar_disorder_GWAS_summary_statistics/14102594\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"3203\\\",\\n    \\\"repository_reference\\\": \\\"https://datashare.ed.ac.uk/handle/10283/3203\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"I9_HYPERTENSION.gwas.imputed_v3.both_sexes.tsv.bgz\\\",\\n    \\\"repository_reference\\\": \\\"https://broad-ukb-sumstats-us-east-1.s3.amazonaws.com/round2/additive-tsvs/I9_HYPERTENSION.gwas.imputed_v3.both_sexes.tsv.bgz\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://github.com/thewonlab/H-MAGMA\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1758210161,\n",
      "    \"model\": \"gemini-2.5-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 84773,\n",
      "        \"completion_tokens\": 684,\n",
      "        \"total_tokens\": 90823,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 5366,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 664 - INFO - Deduplicating response with 11 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: GSE254293\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: GSE254569\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn33235943\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn39718968\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: 19426775\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: 14672034\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: l7aiRr1UEgdoJfZ\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: 14102594\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: 3203\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: I9_HYPERTENSION.gwas.imputed_v3.both_sexes.tsv.bgz\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'GSE254293', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE254293', 'dataset_id': 'GSE254293'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE254293\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: GSE254293\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE254293, identifier: GSE254293, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE254293\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE254293 for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE254293\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier GSE254293 for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE254293\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1157 - INFO - Found id_pattern match for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE254293 in GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE254293\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE254293\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: GSE254293\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE254293\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE254293\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE254293\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: GSE254293\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE254293\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: GSE254293, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE254293\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: GSE254293\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE254293\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'GSE254293', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE254293', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'GSE254569', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE254569', 'dataset_id': 'GSE254569'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE254569\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: GSE254569\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE254569, identifier: GSE254569, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE254569\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE254569 for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE254569\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier GSE254569 for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE254569\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1157 - INFO - Found id_pattern match for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE254569 in GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE254569\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE254569\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: GSE254569\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE254569\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE254569\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE254569\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: GSE254569\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE254569\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: GSE254569, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE254569\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: GSE254569\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE254569\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'GSE254569', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE254569', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn33235943', 'repository_reference': 'https://www.synapse.org/Synapse:syn33235943', 'dataset_id': 'syn33235943'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn33235943\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn33235943\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/Synapse:syn33235943, identifier: syn33235943, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/Synapse:syn33235943\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn33235943 for https://www.synapse.org/Synapse:syn33235943\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn33235943 for https://www.synapse.org/Synapse:syn33235943\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn33235943 for https://www.synapse.org/Synapse:syn33235943\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/Synapse:syn33235943 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/Synapse:syn33235943\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/Synapse:syn33235943\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/Synapse:syn33235943\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn33235943\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn33235943\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn33235943\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/Synapse:syn33235943\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn33235943\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn33235943\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn33235943, synapse.org, https://www.synapse.org/Synapse:syn33235943\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn33235943\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/Synapse:syn33235943\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn33235943', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/Synapse:syn33235943', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn39718968', 'repository_reference': 'https://www.synapse.org/Synapse:syn39718968', 'dataset_id': 'syn39718968'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn39718968\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn39718968\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/Synapse:syn39718968, identifier: syn39718968, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/Synapse:syn39718968\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn39718968 for https://www.synapse.org/Synapse:syn39718968\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn39718968 for https://www.synapse.org/Synapse:syn39718968\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn39718968 for https://www.synapse.org/Synapse:syn39718968\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/Synapse:syn39718968 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/Synapse:syn39718968\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/Synapse:syn39718968\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/Synapse:syn39718968\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn39718968\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn39718968\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn39718968\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/Synapse:syn39718968\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn39718968\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn39718968\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn39718968, synapse.org, https://www.synapse.org/Synapse:syn39718968\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn39718968\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/Synapse:syn39718968\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn39718968', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/Synapse:syn39718968', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': '19426775', 'repository_reference': 'https://figshare.com/articles/dataset/scz2022/19426775', 'dataset_id': '19426775'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: 19426775\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: 19426775\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://figshare.com/articles/dataset/scz2022/19426775, identifier: 19426775, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://figshare.com/articles/dataset/scz2022/19426775\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 19426775 for https://figshare.com/articles/dataset/scz2022/19426775\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier 19426775 for https://figshare.com/articles/dataset/scz2022/19426775\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier 19426775 for https://figshare.com/articles/dataset/scz2022/19426775\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier 19426775 for https://figshare.com/articles/dataset/scz2022/19426775\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier 19426775 for https://figshare.com/articles/dataset/scz2022/19426775\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier 19426775 for https://figshare.com/articles/dataset/scz2022/19426775\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1157 - INFO - Found id_pattern match for https://figshare.com/articles/dataset/scz2022/19426775 in [1-9][a-zA-Z0-9]{3}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://figshare.com/articles/dataset/scz2022/19426775\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://figshare.com/articles/dataset/scz2022/19426775\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: 19426775\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: 19426775\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: 19426775, pdb, https://figshare.com/articles/dataset/scz2022/19426775\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: 19426775\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://figshare.com/articles/dataset/scz2022/19426775\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': '19426775', 'data_repository': 'pdb', 'dataset_webpage': 'https://figshare.com/articles/dataset/scz2022/19426775', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': '14672034', 'repository_reference': 'https://figshare.com/articles/dataset/cdg2019/14672034', 'dataset_id': '14672034'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: 14672034\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: 14672034\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://figshare.com/articles/dataset/cdg2019/14672034, identifier: 14672034, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://figshare.com/articles/dataset/cdg2019/14672034\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 14672034 for https://figshare.com/articles/dataset/cdg2019/14672034\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier 14672034 for https://figshare.com/articles/dataset/cdg2019/14672034\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier 14672034 for https://figshare.com/articles/dataset/cdg2019/14672034\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier 14672034 for https://figshare.com/articles/dataset/cdg2019/14672034\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier 14672034 for https://figshare.com/articles/dataset/cdg2019/14672034\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier 14672034 for https://figshare.com/articles/dataset/cdg2019/14672034\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1157 - INFO - Found id_pattern match for https://figshare.com/articles/dataset/cdg2019/14672034 in [1-9][a-zA-Z0-9]{3}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://figshare.com/articles/dataset/cdg2019/14672034\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://figshare.com/articles/dataset/cdg2019/14672034\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: 14672034\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: 14672034\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: 14672034, pdb, https://figshare.com/articles/dataset/cdg2019/14672034\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: 14672034\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://figshare.com/articles/dataset/cdg2019/14672034\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': '14672034', 'data_repository': 'pdb', 'dataset_webpage': 'https://figshare.com/articles/dataset/cdg2019/14672034', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'l7aiRr1UEgdoJfZ', 'repository_reference': 'https://vu.data.surfsara.nl/index.php/s/l7aiRr1UEgdoJfZ', 'dataset_id': 'l7aiRr1UEgdoJfZ'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: l7aiRr1UEgdoJfZ\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: l7aiRr1UEgdoJfZ\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://vu.data.surfsara.nl/index.php/s/l7aiRr1UEgdoJfZ, identifier: l7aiRr1UEgdoJfZ, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://vu.data.surfsara.nl/index.php/s/l7aiRr1UEgdoJfZ\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier l7aiRr1UEgdoJfZ for https://vu.data.surfsara.nl/index.php/s/l7aiRr1UEgdoJfZ\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier l7aiRr1UEgdoJfZ for https://vu.data.surfsara.nl/index.php/s/l7aiRr1UEgdoJfZ\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier l7aiRr1UEgdoJfZ for https://vu.data.surfsara.nl/index.php/s/l7aiRr1UEgdoJfZ\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier l7aiRr1UEgdoJfZ for https://vu.data.surfsara.nl/index.php/s/l7aiRr1UEgdoJfZ\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier l7aiRr1UEgdoJfZ for https://vu.data.surfsara.nl/index.php/s/l7aiRr1UEgdoJfZ\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier l7aiRr1UEgdoJfZ for https://vu.data.surfsara.nl/index.php/s/l7aiRr1UEgdoJfZ\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://vu.data.surfsara.nl/index.php/s/l7aiRr1UEgdoJfZ\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://vu.data.surfsara.nl/index.php/s/l7aiRr1UEgdoJfZ\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://vu.data.surfsara.nl/index.php/s/l7aiRr1UEgdoJfZ\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: l7aiRr1UEgdoJfZ\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: l7aiRr1UEgdoJfZ\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: l7aiRr1UEgdoJfZ, vu.data.surfsara.nl, https://vu.data.surfsara.nl/index.php/s/l7aiRr1UEgdoJfZ\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: l7aiRr1UEgdoJfZ\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: vu.data.surfsara.nl\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://vu.data.surfsara.nl/index.php/s/l7aiRr1UEgdoJfZ\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'l7aiRr1UEgdoJfZ', 'data_repository': 'vu.data.surfsara.nl', 'dataset_webpage': 'https://vu.data.surfsara.nl/index.php/s/l7aiRr1UEgdoJfZ', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': '14102594', 'repository_reference': 'https://figshare.com/articles/dataset/PGC3_bipolar_disorder_GWAS_summary_statistics/14102594', 'dataset_id': '14102594'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: 14102594\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: 14102594\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://figshare.com/articles/dataset/PGC3_bipolar_disorder_GWAS_summary_statistics/14102594, identifier: 14102594, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://figshare.com/articles/dataset/PGC3_bipolar_disorder_GWAS_summary_statistics/14102594\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 14102594 for https://figshare.com/articles/dataset/PGC3_bipolar_disorder_GWAS_summary_statistics/14102594\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier 14102594 for https://figshare.com/articles/dataset/PGC3_bipolar_disorder_GWAS_summary_statistics/14102594\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier 14102594 for https://figshare.com/articles/dataset/PGC3_bipolar_disorder_GWAS_summary_statistics/14102594\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier 14102594 for https://figshare.com/articles/dataset/PGC3_bipolar_disorder_GWAS_summary_statistics/14102594\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier 14102594 for https://figshare.com/articles/dataset/PGC3_bipolar_disorder_GWAS_summary_statistics/14102594\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier 14102594 for https://figshare.com/articles/dataset/PGC3_bipolar_disorder_GWAS_summary_statistics/14102594\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1157 - INFO - Found id_pattern match for https://figshare.com/articles/dataset/PGC3_bipolar_disorder_GWAS_summary_statistics/14102594 in [1-9][a-zA-Z0-9]{3}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://figshare.com/articles/dataset/PGC3_bipolar_disorder_GWAS_summary_statistics/14102594\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://figshare.com/articles/dataset/PGC3_bipolar_disorder_GWAS_summary_statistics/14102594\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: 14102594\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: 14102594\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: 14102594, pdb, https://figshare.com/articles/dataset/PGC3_bipolar_disorder_GWAS_summary_statistics/14102594\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: 14102594\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://figshare.com/articles/dataset/PGC3_bipolar_disorder_GWAS_summary_statistics/14102594\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': '14102594', 'data_repository': 'pdb', 'dataset_webpage': 'https://figshare.com/articles/dataset/PGC3_bipolar_disorder_GWAS_summary_statistics/14102594', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': '3203', 'repository_reference': 'https://datashare.ed.ac.uk/handle/10283/3203', 'dataset_id': '3203'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: 3203\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: 3203\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://datashare.ed.ac.uk/handle/10283/3203, identifier: 3203, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://datashare.ed.ac.uk/handle/10283/3203\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 3203 for https://datashare.ed.ac.uk/handle/10283/3203\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier 3203 for https://datashare.ed.ac.uk/handle/10283/3203\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier 3203 for https://datashare.ed.ac.uk/handle/10283/3203\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier 3203 for https://datashare.ed.ac.uk/handle/10283/3203\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier 3203 for https://datashare.ed.ac.uk/handle/10283/3203\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier 3203 for https://datashare.ed.ac.uk/handle/10283/3203\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1157 - INFO - Found id_pattern match for https://datashare.ed.ac.uk/handle/10283/3203 in [1-9][a-zA-Z0-9]{3}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://datashare.ed.ac.uk/handle/10283/3203\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://datashare.ed.ac.uk/handle/10283/3203\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: 3203\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: 3203\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: 3203, pdb, https://datashare.ed.ac.uk/handle/10283/3203\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: 3203\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://datashare.ed.ac.uk/handle/10283/3203\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': '3203', 'data_repository': 'pdb', 'dataset_webpage': 'https://datashare.ed.ac.uk/handle/10283/3203', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'I9_HYPERTENSION.gwas.imputed_v3.both_sexes.tsv.bgz', 'repository_reference': 'https://broad-ukb-sumstats-us-east-1.s3.amazonaws.com/round2/additive-tsvs/I9_HYPERTENSION.gwas.imputed_v3.both_sexes.tsv.bgz', 'dataset_id': 'I9_HYPERTENSION.gwas.imputed_v3.both_sexes.tsv.bgz'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: I9_HYPERTENSION.gwas.imputed_v3.both_sexes.tsv.bgz\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID I9_HYPERTENSION.gwas.imputed_v3.both_sexes.tsv.bgz is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://broad-ukb-sumstats-us-east-1.s3.amazonaws.com/round2/additive-tsvs/I9_HYPERTENSION.gwas.imputed_v3.both_sexes.tsv.bgz, identifier: I9_HYPERTENSION.gwas.imputed_v3.both_sexes.tsv.bgz, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://broad-ukb-sumstats-us-east-1.s3.amazonaws.com/round2/additive-tsvs/I9_HYPERTENSION.gwas.imputed_v3.both_sexes.tsv.bgz\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier I9_HYPERTENSION.gwas.imputed_v3.both_sexes.tsv.bgz for https://broad-ukb-sumstats-us-east-1.s3.amazonaws.com/round2/additive-tsvs/I9_HYPERTENSION.gwas.imputed_v3.both_sexes.tsv.bgz\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier I9_HYPERTENSION.gwas.imputed_v3.both_sexes.tsv.bgz for https://broad-ukb-sumstats-us-east-1.s3.amazonaws.com/round2/additive-tsvs/I9_HYPERTENSION.gwas.imputed_v3.both_sexes.tsv.bgz\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier I9_HYPERTENSION.gwas.imputed_v3.both_sexes.tsv.bgz for https://broad-ukb-sumstats-us-east-1.s3.amazonaws.com/round2/additive-tsvs/I9_HYPERTENSION.gwas.imputed_v3.both_sexes.tsv.bgz\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier I9_HYPERTENSION.gwas.imputed_v3.both_sexes.tsv.bgz for https://broad-ukb-sumstats-us-east-1.s3.amazonaws.com/round2/additive-tsvs/I9_HYPERTENSION.gwas.imputed_v3.both_sexes.tsv.bgz\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier I9_HYPERTENSION.gwas.imputed_v3.both_sexes.tsv.bgz for https://broad-ukb-sumstats-us-east-1.s3.amazonaws.com/round2/additive-tsvs/I9_HYPERTENSION.gwas.imputed_v3.both_sexes.tsv.bgz\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier I9_HYPERTENSION.gwas.imputed_v3.both_sexes.tsv.bgz for https://broad-ukb-sumstats-us-east-1.s3.amazonaws.com/round2/additive-tsvs/I9_HYPERTENSION.gwas.imputed_v3.both_sexes.tsv.bgz\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://broad-ukb-sumstats-us-east-1.s3.amazonaws.com/round2/additive-tsvs/I9_HYPERTENSION.gwas.imputed_v3.both_sexes.tsv.bgz\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'I9_HYPERTENSION.gwas.imputed_v3.both_sexes.tsv.bgz', 'data_repository': 'broad-ukb-sumstats-us-east-1.s3.amazonaws.com', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://github.com/thewonlab/H-MAGMA', 'dataset_id': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://github.com/thewonlab/H-MAGMA, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://github.com/thewonlab/H-MAGMA\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://github.com/thewonlab/H-MAGMA\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 278 - INFO - Augmented dataset links: [{'dataset_identifier': 'GSE254293', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE254293', 'citation_type': 'n/a'}, {'dataset_identifier': 'GSE254569', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE254569', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn33235943', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/Synapse:syn33235943', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn39718968', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/Synapse:syn39718968', 'citation_type': 'n/a'}, {'dataset_identifier': '19426775', 'data_repository': 'pdb', 'dataset_webpage': 'https://figshare.com/articles/dataset/scz2022/19426775', 'citation_type': 'n/a'}, {'dataset_identifier': '14672034', 'data_repository': 'pdb', 'dataset_webpage': 'https://figshare.com/articles/dataset/cdg2019/14672034', 'citation_type': 'n/a'}, {'dataset_identifier': 'l7aiRr1UEgdoJfZ', 'data_repository': 'vu.data.surfsara.nl', 'dataset_webpage': 'https://vu.data.surfsara.nl/index.php/s/l7aiRr1UEgdoJfZ', 'citation_type': 'n/a'}, {'dataset_identifier': '14102594', 'data_repository': 'pdb', 'dataset_webpage': 'https://figshare.com/articles/dataset/PGC3_bipolar_disorder_GWAS_summary_statistics/14102594', 'citation_type': 'n/a'}, {'dataset_identifier': '3203', 'data_repository': 'pdb', 'dataset_webpage': 'https://datashare.ed.ac.uk/handle/10283/3203', 'citation_type': 'n/a'}, {'dataset_identifier': 'I9_HYPERTENSION.gwas.imputed_v3.both_sexes.tsv.bgz', 'data_repository': 'broad-ukb-sumstats-us-east-1.s3.amazonaws.com', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1191 - INFO - Fetching metadata for 11 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: GSE254293\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: geo, identifier: GSE254293, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE254293 for geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for GSE254293 in geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE254293\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE254293\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 1: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: GSE254569\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: geo, identifier: GSE254569, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE254569 for geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for GSE254569 in geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 2 with repo: geo and accession_id: GSE254569\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE254569\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 2: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn33235943\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn33235943, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn33235943 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn33235943 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn33235943 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn33235943 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 3 with repo: synapse.org and accession_id: syn33235943\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn33235943\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 3: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn39718968\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn39718968, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn39718968 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn39718968 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn39718968 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn39718968 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 4 with repo: synapse.org and accession_id: syn39718968\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn39718968\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 4: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: 19426775\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: pdb, identifier: 19426775, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 19426775 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier 19426775 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier 19426775 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier 19426775 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier 19426775 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for 19426775 in pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 5 with repo: pdb and accession_id: 19426775\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.rcsb.org/structure/19426775\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: 14672034\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: pdb, identifier: 14672034, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 14672034 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier 14672034 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier 14672034 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier 14672034 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier 14672034 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for 14672034 in pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 6 with repo: pdb and accession_id: 14672034\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.rcsb.org/structure/14672034\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: l7aiRr1UEgdoJfZ\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: vu.data.surfsara.nl, identifier: l7aiRr1UEgdoJfZ, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier l7aiRr1UEgdoJfZ for vu.data.surfsara.nl\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier l7aiRr1UEgdoJfZ for vu.data.surfsara.nl\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier l7aiRr1UEgdoJfZ for vu.data.surfsara.nl\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier l7aiRr1UEgdoJfZ for vu.data.surfsara.nl\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier l7aiRr1UEgdoJfZ for vu.data.surfsara.nl\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier l7aiRr1UEgdoJfZ for vu.data.surfsara.nl\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: vu.data.surfsara.nl\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL vu.data.surfsara.nl may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for l7aiRr1UEgdoJfZ in vu.data.surfsara.nl\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 7 with repo: vu.data.surfsara.nl and accession_id: l7aiRr1UEgdoJfZ\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository vu.data.surfsara.nl unknown in Ontology. Skipping dataset page 7.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: 14102594\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: pdb, identifier: 14102594, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 14102594 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier 14102594 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier 14102594 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier 14102594 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier 14102594 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for 14102594 in pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 8 with repo: pdb and accession_id: 14102594\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.rcsb.org/structure/14102594\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: 3203\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: pdb, identifier: 3203, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 3203 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier 3203 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier 3203 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier 3203 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier 3203 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for 3203 in pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 9 with repo: pdb and accession_id: 3203\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.rcsb.org/structure/3203\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 10 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: I9_HYPERTENSION.gwas.imputed_v3.both_sexes.tsv.bgz\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: broad-ukb-sumstats-us-east-1.s3.amazonaws.com, identifier: I9_HYPERTENSION.gwas.imputed_v3.both_sexes.tsv.bgz, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier I9_HYPERTENSION.gwas.imputed_v3.both_sexes.tsv.bgz for broad-ukb-sumstats-us-east-1.s3.amazonaws.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier I9_HYPERTENSION.gwas.imputed_v3.both_sexes.tsv.bgz for broad-ukb-sumstats-us-east-1.s3.amazonaws.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier I9_HYPERTENSION.gwas.imputed_v3.both_sexes.tsv.bgz for broad-ukb-sumstats-us-east-1.s3.amazonaws.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier I9_HYPERTENSION.gwas.imputed_v3.both_sexes.tsv.bgz for broad-ukb-sumstats-us-east-1.s3.amazonaws.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier I9_HYPERTENSION.gwas.imputed_v3.both_sexes.tsv.bgz for broad-ukb-sumstats-us-east-1.s3.amazonaws.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier I9_HYPERTENSION.gwas.imputed_v3.both_sexes.tsv.bgz for broad-ukb-sumstats-us-east-1.s3.amazonaws.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: broad-ukb-sumstats-us-east-1.s3.amazonaws.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL broad-ukb-sumstats-us-east-1.s3.amazonaws.com may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for I9_HYPERTENSION.gwas.imputed_v3.both_sexes.tsv.bgz in broad-ukb-sumstats-us-east-1.s3.amazonaws.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 10 with repo: broad-ukb-sumstats-us-east-1.s3.amazonaws.com and accession_id: I9_HYPERTENSION.gwas.imputed_v3.both_sexes.tsv.bgz\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository broad-ukb-sumstats-us-east-1.s3.amazonaws.com unknown in Ontology. Skipping dataset page 10.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 11 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 11: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1277 - INFO - Updated datasets len: 11\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 287 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 13, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'source_url', 'download_link',\n",
      "       'title', 'content_type', 'id', 'caption', 'description',\n",
      "       'source_section', 'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 500 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 506 - INFO - XML element classification not needed. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1012 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11452345\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1020 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11452345\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 638th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11457430)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11457430\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11457430 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 634 - INFO - Fetching data for pmc11457430\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 637 - INFO - Fetching data from scripts/exp_input/Local_fetched_data_1.parquet\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 438 - INFO - Raw xml data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11457430 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 452 - INFO - Skipping raw HTML/XML saving.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 458 - INFO - Using XMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 52 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 24 - INFO - Initializing xmlRetriever\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 466 - INFO - No additional data provided. Parsing raw data only.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 147 - INFO - Function call: parse_data(api_data(<class 'str'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11457430, additional_data, XML)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 162 - INFO - api_data converted to lxml element\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 466 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 153 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 154 - INFO - Consider migrating this function to the BaseRetriever class.\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 69 - INFO - Loading target sections for section name: supplementary_material_sections\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 477 - INFO - Found 2 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x15d68b580>, <Element supplementary-material at 0x15d001740>]\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 485 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x15d68b580>, <Element supplementary-material at 0x15d001740>]}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 574 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM1: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM2: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 263 - INFO - Extracting links from full XML content.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 301 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 309 - INFO - Content length: 134302\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 326 - INFO - Prompt messages total length: 34329 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 332 - INFO - Prompt ID: gemini-2.5-flash-0-953aa406a0aef7fa9b0ae11825788cb0de833e3adb6e626571473fc721f260fe\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 351 - INFO - Requesting datasets from content using model: gemini-2.5-flash, temperature: 0, messages length: 34329 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-fc72e0d6-1b90-4e28-bccd-360947fb6509\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"PRIMEG\\\",\\n    \\\"repository_reference\\\": \\\"http://www.szflab.site/PRIMEG/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://brainimmuneatlas.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.6084/m9.figshare.22434341\\\",\\n    \\\"repository_reference\\\": \\\"https://doi.org/10.6084/m9.figshare.22434341\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://glioblastoma.alleninstitute.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"CGGA\\\",\\n    \\\"repository_reference\\\": \\\"http://gliovis.bioinfo.cnio.es/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"TCGA\\\",\\n    \\\"repository_reference\\\": \\\"http://gliovis.bioinfo.cnio.es/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Rembrandt\\\",\\n    \\\"repository_reference\\\": \\\"http://gliovis.bioinfo.cnio.es/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn17038081\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn17038081\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE79671\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE79671\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn61807589\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn61807589\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"PRIMEG\\\",\\n    \\\"repository_reference\\\": \\\"http://www.szflab.site/PRIMEG/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://brainimmuneatlas.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.6084/m9.figshare.22434341\\\",\\n    \\\"repository_reference\\\": \\\"https://doi.org/10.6084/m9.figshare.22434341\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://glioblastoma.alleninstitute.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"CGGA\\\",\\n    \\\"repository_reference\\\": \\\"http://gliovis.bioinfo.cnio.es/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"TCGA\\\",\\n    \\\"repository_reference\\\": \\\"http://gliovis.bioinfo.cnio.es/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Rembrandt\\\",\\n    \\\"repository_reference\\\": \\\"http://gliovis.bioinfo.cnio.es/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn17038081\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn17038081\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE79671\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE79671\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn61807589\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn61807589\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1758210195,\n",
      "    \"model\": \"gemini-2.5-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 42982,\n",
      "        \"completion_tokens\": 474,\n",
      "        \"total_tokens\": 48580,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 5124,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 664 - INFO - Deduplicating response with 10 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: PRIMEG\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: 10.6084/m9.figshare.22434341\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: CGGA\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: TCGA\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: Rembrandt\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn17038081\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: GSE79671\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn61807589\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'PRIMEG', 'repository_reference': 'http://www.szflab.site/PRIMEG/', 'dataset_id': 'PRIMEG'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: PRIMEG\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID PRIMEG is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: http://www.szflab.site/PRIMEG/, identifier: PRIMEG, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: http://www.szflab.site/PRIMEG/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier PRIMEG for http://www.szflab.site/PRIMEG/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier PRIMEG for http://www.szflab.site/PRIMEG/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier PRIMEG for http://www.szflab.site/PRIMEG/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier PRIMEG for http://www.szflab.site/PRIMEG/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier PRIMEG for http://www.szflab.site/PRIMEG/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier PRIMEG for http://www.szflab.site/PRIMEG/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: http://www.szflab.site/PRIMEG/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'PRIMEG', 'data_repository': 'www.szflab.site', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://brainimmuneatlas.org/', 'dataset_id': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://brainimmuneatlas.org/, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://brainimmuneatlas.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://brainimmuneatlas.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'brainimmuneatlas.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': '10.6084/m9.figshare.22434341', 'repository_reference': 'https://doi.org/10.6084/m9.figshare.22434341', 'dataset_id': '10.6084/m9.figshare.22434341'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: 10\\.\\d{4,9}/[^\\s,]+ to value: 10.6084/m9.figshare.22434341\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 607 - INFO - Setting dataset_webpage to DOI URL value: https://doi.org/10.6084/m9.figshare.22434341\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: 10.6084/m9.figshare.22434341\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: doi.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://doi.org/10.6084/m9.figshare.22434341, identifier: 10.6084/m9.figshare.22434341, dataset_page: https://doi.org/10.6084/m9.figshare.22434341\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://doi.org/10.6084/m9.figshare.22434341\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.6084/m9.figshare.22434341 for https://doi.org/10.6084/m9.figshare.22434341\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier 10.6084/m9.figshare.22434341 for https://doi.org/10.6084/m9.figshare.22434341\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier 10.6084/m9.figshare.22434341 for https://doi.org/10.6084/m9.figshare.22434341\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier 10.6084/m9.figshare.22434341 for https://doi.org/10.6084/m9.figshare.22434341\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier 10.6084/m9.figshare.22434341 for https://doi.org/10.6084/m9.figshare.22434341\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1157 - INFO - Found id_pattern match for https://doi.org/10.6084/m9.figshare.22434341 in 10\\.\\d{4,9}/[^\\s,]+\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://doi.org/__ID__ to value: https://doi.org/10.6084/m9.figshare.22434341\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: 10\\.\\d{4,9}/[^\\s,]+ to value: https://doi.org/10.6084/m9.figshare.22434341\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(10\\.\\d{4,9}/[^\\s,]+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(10\\.\\d{4,9}/[^\\s,]+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: 10.6084/m9.figshare.22434341\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: 10\\.\\d{4,9}/[^\\s,]+ to value: 10.6084/m9.figshare.22434341\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: 10.6084/m9.figshare.22434341\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://doi.org/10.6084/m9.figshare.22434341\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: 10.6084/m9.figshare.22434341\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: 10.6084/m9.figshare.22434341\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: 10.6084/m9.figshare.22434341, doi.org, https://doi.org/10.6084/m9.figshare.22434341\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: 10.6084/m9.figshare.22434341\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: doi.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://doi.org/10.6084/m9.figshare.22434341\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': '10.6084/m9.figshare.22434341', 'data_repository': 'doi.org', 'dataset_webpage': 'https://doi.org/10.6084/m9.figshare.22434341', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://glioblastoma.alleninstitute.org/'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://glioblastoma.alleninstitute.org/, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://glioblastoma.alleninstitute.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://glioblastoma.alleninstitute.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'glioblastoma.alleninstitute.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'CGGA', 'repository_reference': 'http://gliovis.bioinfo.cnio.es/', 'dataset_id': 'CGGA'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: CGGA\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID CGGA is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: http://gliovis.bioinfo.cnio.es/, identifier: CGGA, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: http://gliovis.bioinfo.cnio.es/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier CGGA for http://gliovis.bioinfo.cnio.es/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier CGGA for http://gliovis.bioinfo.cnio.es/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier CGGA for http://gliovis.bioinfo.cnio.es/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier CGGA for http://gliovis.bioinfo.cnio.es/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier CGGA for http://gliovis.bioinfo.cnio.es/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier CGGA for http://gliovis.bioinfo.cnio.es/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: http://gliovis.bioinfo.cnio.es/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'CGGA', 'data_repository': 'gliovis.bioinfo.cnio.es', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'TCGA', 'repository_reference': 'http://gliovis.bioinfo.cnio.es/', 'dataset_id': 'TCGA'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: TCGA\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID TCGA is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: http://gliovis.bioinfo.cnio.es/, identifier: TCGA, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: http://gliovis.bioinfo.cnio.es/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier TCGA for http://gliovis.bioinfo.cnio.es/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier TCGA for http://gliovis.bioinfo.cnio.es/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier TCGA for http://gliovis.bioinfo.cnio.es/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier TCGA for http://gliovis.bioinfo.cnio.es/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier TCGA for http://gliovis.bioinfo.cnio.es/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier TCGA for http://gliovis.bioinfo.cnio.es/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: http://gliovis.bioinfo.cnio.es/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'TCGA', 'data_repository': 'gliovis.bioinfo.cnio.es', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'Rembrandt', 'repository_reference': 'http://gliovis.bioinfo.cnio.es/', 'dataset_id': 'Rembrandt'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: Rembrandt\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID Rembrandt is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: http://gliovis.bioinfo.cnio.es/, identifier: Rembrandt, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: http://gliovis.bioinfo.cnio.es/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Rembrandt for http://gliovis.bioinfo.cnio.es/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier Rembrandt for http://gliovis.bioinfo.cnio.es/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Rembrandt for http://gliovis.bioinfo.cnio.es/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Rembrandt for http://gliovis.bioinfo.cnio.es/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier Rembrandt for http://gliovis.bioinfo.cnio.es/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier Rembrandt for http://gliovis.bioinfo.cnio.es/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: http://gliovis.bioinfo.cnio.es/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'Rembrandt', 'data_repository': 'gliovis.bioinfo.cnio.es', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn17038081', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn17038081', 'dataset_id': 'syn17038081'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn17038081\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn17038081\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn17038081, identifier: syn17038081, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn17038081\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn17038081 for https://www.synapse.org/#!Synapse:syn17038081\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn17038081 for https://www.synapse.org/#!Synapse:syn17038081\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn17038081 for https://www.synapse.org/#!Synapse:syn17038081\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn17038081 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn17038081\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn17038081\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn17038081\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn17038081\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn17038081\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn17038081\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn17038081\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn17038081\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn17038081\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn17038081, synapse.org, https://www.synapse.org/#!Synapse:syn17038081\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn17038081\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn17038081\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn17038081', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn17038081', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'GSE79671', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE79671', 'dataset_id': 'GSE79671'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE79671\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: GSE79671\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE79671, identifier: GSE79671, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE79671\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE79671 for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE79671\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier GSE79671 for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE79671\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1157 - INFO - Found id_pattern match for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE79671 in GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE79671\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE79671\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: GSE79671\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE79671\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE79671\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE79671\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: GSE79671\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE79671\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: GSE79671, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE79671\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: GSE79671\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE79671\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'GSE79671', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE79671', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn61807589', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn61807589', 'dataset_id': 'syn61807589'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn61807589\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn61807589\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn61807589, identifier: syn61807589, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn61807589\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn61807589 for https://www.synapse.org/#!Synapse:syn61807589\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn61807589 for https://www.synapse.org/#!Synapse:syn61807589\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn61807589 for https://www.synapse.org/#!Synapse:syn61807589\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn61807589 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn61807589\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn61807589\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn61807589\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn61807589\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn61807589\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn61807589\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn61807589\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn61807589\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn61807589\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn61807589, synapse.org, https://www.synapse.org/#!Synapse:syn61807589\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn61807589\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn61807589\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn61807589', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn61807589', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 278 - INFO - Augmented dataset links: [{'dataset_identifier': 'PRIMEG', 'data_repository': 'www.szflab.site', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'brainimmuneatlas.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': '10.6084/m9.figshare.22434341', 'data_repository': 'doi.org', 'dataset_webpage': 'https://doi.org/10.6084/m9.figshare.22434341', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'glioblastoma.alleninstitute.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'CGGA', 'data_repository': 'gliovis.bioinfo.cnio.es', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'TCGA', 'data_repository': 'gliovis.bioinfo.cnio.es', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'Rembrandt', 'data_repository': 'gliovis.bioinfo.cnio.es', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn17038081', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn17038081', 'citation_type': 'n/a'}, {'dataset_identifier': 'GSE79671', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE79671', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn61807589', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn61807589', 'citation_type': 'n/a'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1191 - INFO - Fetching metadata for 10 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: PRIMEG\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: www.szflab.site, identifier: PRIMEG, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier PRIMEG for www.szflab.site\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier PRIMEG for www.szflab.site\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier PRIMEG for www.szflab.site\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier PRIMEG for www.szflab.site\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier PRIMEG for www.szflab.site\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier PRIMEG for www.szflab.site\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: www.szflab.site\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL www.szflab.site may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for PRIMEG in www.szflab.site\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 1 with repo: www.szflab.site and accession_id: PRIMEG\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository www.szflab.site unknown in Ontology. Skipping dataset page 1.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 2: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: 10.6084/m9.figshare.22434341\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: doi.org, identifier: 10.6084/m9.figshare.22434341, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.6084/m9.figshare.22434341 for doi.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier 10.6084/m9.figshare.22434341 for doi.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier 10.6084/m9.figshare.22434341 for doi.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier 10.6084/m9.figshare.22434341 for doi.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: doi.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for 10.6084/m9.figshare.22434341 in doi.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 3 with repo: doi.org and accession_id: 10.6084/m9.figshare.22434341\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://doi.org/10.6084/m9.figshare.22434341\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 4: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: CGGA\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: gliovis.bioinfo.cnio.es, identifier: CGGA, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier CGGA for gliovis.bioinfo.cnio.es\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier CGGA for gliovis.bioinfo.cnio.es\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier CGGA for gliovis.bioinfo.cnio.es\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier CGGA for gliovis.bioinfo.cnio.es\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier CGGA for gliovis.bioinfo.cnio.es\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier CGGA for gliovis.bioinfo.cnio.es\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: gliovis.bioinfo.cnio.es\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL gliovis.bioinfo.cnio.es may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for CGGA in gliovis.bioinfo.cnio.es\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 5 with repo: gliovis.bioinfo.cnio.es and accession_id: CGGA\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository gliovis.bioinfo.cnio.es unknown in Ontology. Skipping dataset page 5.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: TCGA\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: gliovis.bioinfo.cnio.es, identifier: TCGA, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier TCGA for gliovis.bioinfo.cnio.es\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier TCGA for gliovis.bioinfo.cnio.es\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier TCGA for gliovis.bioinfo.cnio.es\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier TCGA for gliovis.bioinfo.cnio.es\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier TCGA for gliovis.bioinfo.cnio.es\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier TCGA for gliovis.bioinfo.cnio.es\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: gliovis.bioinfo.cnio.es\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL gliovis.bioinfo.cnio.es may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for TCGA in gliovis.bioinfo.cnio.es\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 6 with repo: gliovis.bioinfo.cnio.es and accession_id: TCGA\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository gliovis.bioinfo.cnio.es unknown in Ontology. Skipping dataset page 6.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: Rembrandt\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: gliovis.bioinfo.cnio.es, identifier: Rembrandt, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Rembrandt for gliovis.bioinfo.cnio.es\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier Rembrandt for gliovis.bioinfo.cnio.es\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Rembrandt for gliovis.bioinfo.cnio.es\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Rembrandt for gliovis.bioinfo.cnio.es\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier Rembrandt for gliovis.bioinfo.cnio.es\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier Rembrandt for gliovis.bioinfo.cnio.es\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: gliovis.bioinfo.cnio.es\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL gliovis.bioinfo.cnio.es may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for Rembrandt in gliovis.bioinfo.cnio.es\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 7 with repo: gliovis.bioinfo.cnio.es and accession_id: Rembrandt\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository gliovis.bioinfo.cnio.es unknown in Ontology. Skipping dataset page 7.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn17038081\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn17038081, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn17038081 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn17038081 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn17038081 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn17038081 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 8 with repo: synapse.org and accession_id: syn17038081\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn17038081\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 8: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: GSE79671\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: geo, identifier: GSE79671, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE79671 for geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for GSE79671 in geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 9 with repo: geo and accession_id: GSE79671\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE79671\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 9: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 10 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn61807589\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn61807589, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn61807589 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn61807589 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn61807589 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn61807589 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 10 with repo: synapse.org and accession_id: syn61807589\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn61807589\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 10: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1277 - INFO - Updated datasets len: 10\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 287 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 12, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'source_url', 'download_link',\n",
      "       'title', 'content_type', 'id', 'caption', 'description',\n",
      "       'source_section', 'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 500 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 506 - INFO - XML element classification not needed. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1012 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11457430\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1020 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11457430\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 639th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11460197)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11460197\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11460197 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 634 - INFO - Fetching data for pmc11460197\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 637 - INFO - Fetching data from scripts/exp_input/Local_fetched_data_1.parquet\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 438 - INFO - Raw xml data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11460197 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 452 - INFO - Skipping raw HTML/XML saving.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 458 - INFO - Using XMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 52 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 24 - INFO - Initializing xmlRetriever\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 466 - INFO - No additional data provided. Parsing raw data only.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 147 - INFO - Function call: parse_data(api_data(<class 'str'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11460197, additional_data, XML)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 162 - INFO - api_data converted to lxml element\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 466 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 153 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 154 - INFO - Consider migrating this function to the BaseRetriever class.\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 69 - INFO - Loading target sections for section name: supplementary_material_sections\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 477 - INFO - Found 9 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x15d3ea900>, <Element supplementary-material at 0x15dd6bfc0>, <Element supplementary-material at 0x15dd6b380>, <Element supplementary-material at 0x15dd68600>, <Element supplementary-material at 0x15dd687c0>, <Element supplementary-material at 0x15dd6adc0>, <Element supplementary-material at 0x15dd6b040>, <Element supplementary-material at 0x15dd6a540>, <Element supplementary-material at 0x15dd6b480>]\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 485 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x15d3ea900>, <Element supplementary-material at 0x15dd6bfc0>, <Element supplementary-material at 0x15dd6b380>, <Element supplementary-material at 0x15dd68600>, <Element supplementary-material at 0x15dd687c0>, <Element supplementary-material at 0x15dd6adc0>, <Element supplementary-material at 0x15dd6b040>, <Element supplementary-material at 0x15dd6a540>, <Element supplementary-material at 0x15dd6b480>]}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 574 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM1: In total, 518 proteins exhibited significant changes in their abundance levels across the two fractions (Supplemental Fig. 1 [this file] A and Supplemental Table 3 [rid=MOESM9] ), with 338 increased and 180 decreased. Interestingly, APOE4 showed the most enrichment in Hp-enriched fractions (15.5-fold, = 0.0003), followed by APOE3 (14.7-fold, = 0.0004), which was inferred due to a lack of variant specific peptides, and APOE2 (7.4-fold, = 0.006) (Supplemental Fig. 1 [this file] B). \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM2: Prior to subjecting the samples to TMT-MS, we confirmed the heparin enrichment of APOE and thrombin via immunoblotting of the input, FT, and Hp-enriched fractions (Supplemental Fig. 2 [this file] ). \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM3: To assess any biases that heparin enrichment may have caused, and to compare the differences in proteome coverage to traditional immunodepletion methods [ 59 [rid=CR59] , 60 [rid=CR60] ], we benchmarked the Hp-enriched proteome against the immunodepleted proteome from the same 36 samples in Set 1 (Supplemental Fig. 3 [this file] A). Among these, 432 proteins were increased in AD and 383 were decreased (Supplemental Fig. 3 [this file] B and Supplemental Table 8 [rid=MOESM9] ). We found that 804 out of the 1,129 proteins (71%) in the immunodepletion dataset overlapped with the Hp-enriched dataset, with 325 proteins not detected by heparin enrichment (Supplemental Fig. 3 [this file] C and Supplemental Table 9 [rid=MOESM9] ). These include many members of the M42 matrisome module, such as SMOC1, SMOC2, OLFML3, SPON1, FRZB, and MDK (Supplemental Fig. 3 [this file] D). Additionally, significant protein levels (AD vs. Control) measured following either heparin enrichment or immunodepletion exhibited a strong correlation regarding the directional changes observed in AD plasma (Supplemental Fig. 3 [this file] E). \n",
      "Given that the significant protein levels ( -value < 0.05) measured following either heparin enrichment or immunodepletion exhibited strong correlation regarding the directional changes observed in AD plasma (Supplemental Fig. 3 [this file] E), this discrepancy may be attributed to the methodology. \n",
      "This number is more than twofold greater than the coverage generated with immunodepletion method followed by TMT-MS (Supplemental Fig. 3 [this file] C), supporting the depth and sensitivity of the heparin enrichment approach when coupled to TMT-MS. \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM4: However, consistent with our previous results [ 30 [rid=CR30] ], we observed a notable bias towards decreased protein levels in AD plasma on the SomaScan and Olink platform even following median-centered normalization (Supplemental Fig. 4 [this file] A-B and Supplemental Tables 13–14 [rid=MOESM9] ). This bias was more pronounced in SomaScan, even when restricted to only significant proteins ( -value < 0.05) (Supplemental Fig. 4 [this file] C-D). \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM5: To enhance the specificity of our analysis, we excluded cases from the initial 121 samples (including 13 overlapping controls as described above) that did not meet strict CSF biomarker criteria (tTau/Aβ ratio) or MoCA cutoffs [ 49 [rid=CR49] ] as described in methods (Supplemental Fig. 5 [this file] A). Boxplots for each measurement across all unique cases ( = 96) were generated to show their separation based on disease status (Supplemental Fig. 5 [this file] B). \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM6: To ensure that the observed changes in these proteins in AD were not influenced by age, sex, or race, we performed regression analyses for these traits within Set 1 and Set 2, followed by the same -analysis described earlier (Supplemental Fig. 6 [this file] and Supplemental Table 26 [rid=MOESM9] ). \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM7: To ensure the consistency of protein overlap analysis, we employed the same Uniprot database and utilized the FragPipe search algorithm we used for Heparin-MS plasma analysis to re-analyze 456 dorsolateral prefrontal cortex (DLPFC) tissues from control, asymptomatic AD (AsymAD) and AD brains from the ROSMAP and the Banner cohorts (Supplemental Fig. 7 [this file] A and Supplemental Table 28 [rid=MOESM9] ) [ 11 [rid=CR11] ]. When compared to a previous brain proteome dataset generated using Proteome Discoverer (PD) and an older 2015 Uniprot protein database [ 11 [rid=CR11] ], it was found that 7,476 of these unique gene symbols overlapped between the two outputs, while 1,428 were exclusive to the FragPipe, representing an approximately 18% increase in the number of identifications (Supplemental Fig. 7 [this file] B and Supplemental Table 30 [rid=MOESM9] ). Furthermore, to ensure the consistency of the direction of change in the AD brain proteome compared to controls, we correlated the differentially abundant proteins across these two sets of results (Supplemental Table 31–32 [rid=MOESM9] ), revealing a correlation coefficient of 0.9 between AD and control groups and a correlation coefficient of 0.84 between AsymAD and control groups (Supplemental Fig. 7 [this file] C). \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM8: Finally, modules that showed significant overlap with the Hp-enriched plasma proteome, yet exhibited moderate to low correlations with AD clinicopathological traits are detailed in Supplemental Fig. 8 [this file] . \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM9: Further details about demographics, cognitive scores, CSF biomarkers, and pTau181 plasma biomarker levels for individuals in both sets of samples are available in Table 1 [rid=Tab1] and Supplemental Tables 4 [this file] and 15 [this file] . \n",
      "Supplemental Table 4 [this file] and 15 [this file] provides the sample-to-batch arrangement for each of these samples. \n",
      "All proteins along with corresponding -value and fold-change were listed in Supplemental Table 21 [this file] . Same correlations performed for SomaScan ( = 35 samples) and Olink ( = 35 samples) were listed in Supplemental Table 24–25 [this file] . \n",
      "Additional ROC curve characteristics including sensitivity, specificity, and accuracy were calculated with the reportROC R package and listed in Supplemental Table 27 [this file] . \n",
      "To assess the consistency of the Hp-enriched MS proteomic approach, measurements of proteins with no missing values across all three replicates within each fraction were listed in Supplemental Table 1 [this file] along with their coefficient of variation (CV). To evaluate the depth of protein identification across the fractions, we assessed the overlap of proteins that were detected in at least two out of three replicates of each fraction (Fig. 1 [rid=Fig1] E and Supplemental Table 2 [this file] ). \n",
      "In total, 518 proteins exhibited significant changes in their abundance levels across the two fractions (Supplemental Fig. 1 [rid=MOESM1] A and Supplemental Table 3 [this file] ), with 338 increased and 180 decreased. \n",
      "After successfully demonstrating enrichment of HBPs from human plasma, we aimed to both enhance the depth and determine the differences in the plasma proteome between AD ( = 18) and control ( = 18) individuals using discovery Set 1 (Supplemental Table 4 [this file] ). Following heparin-affinity enrichment, we employed tandem mass tag mass spectrometry (TMT-MS) in conjunction with high-pH off-line fractionation to identify a total of 3284 proteins (Fig. 2 [rid=Fig2] A and Supplemental Table 5 [this file] ). To enhance data completeness, we considered only those proteins quantified in at least 50% of the samples for subsequent analyses, culminating in the final quantification of 2077 proteins (Supplemental Table 6 [this file] ). The complete list of differentially abundant proteins is provided in Supplementary Table 7 [this file] . \n",
      "Among these, 432 proteins were increased in AD and 383 were decreased (Supplemental Fig. 3 [rid=MOESM3] B and Supplemental Table 8 [this file] ). We found that 804 out of the 1,129 proteins (71%) in the immunodepletion dataset overlapped with the Hp-enriched dataset, with 325 proteins not detected by heparin enrichment (Supplemental Fig. 3 [rid=MOESM3] C and Supplemental Table 9 [this file] ). \n",
      "The protein measures were obtained from these two platforms following cross-platform median-centered normalization as previously described [ 30 [rid=CR30] ] and are provided in Supplemental Table 10 [this file] and 11 [this file] . We used gene symbols from each output to calculate the number of measurements and their overlap across the various platforms (Fig. 3 [rid=Fig3] A and Supplemental Table 12 [this file] ). However, consistent with our previous results [ 30 [rid=CR30] ], we observed a notable bias towards decreased protein levels in AD plasma on the SomaScan and Olink platform even following median-centered normalization (Supplemental Fig. 4 [rid=MOESM4] A-B and Supplemental Tables 13–14 [this file] ). \n",
      "To demonstrate the reproducibility of the Heparin-MS approach in plasma, we analyzed a separate set of samples (Set 2) involving 49 controls and 36 AD individuals (Supplemental Table 15 [this file] ), which included 13 overlapping controls with the discovery Set 1. TMT-MS was employed for the analysis of the Hp-enriched fraction, carried out across five batches, resulting in the identification of 2618 proteins measured in 50% or more of the samples. (Supplemental Table 16 [this file] ). Impressively, this encompassed proteins across 10 orders of magnitude in concentration within the plasma, even extending to those with the lowest concentration (1.47 pg/ml), as evidenced by inclusion of proteins such as LAG3 and RNF213 as well as many M42 matrisome members based on The Human Protein Atlas Database (Fig. 4 [rid=Fig4] C and Supplemental Table 17 [this file] ). In total, 1829 out of 2866 (64%) proteins overlapped between the two independent TMT-MS datasets (Fig. 4 [rid=Fig4] D and Supplemental Table 18 [this file] ). Differential expression analyses were performed on all 2618 proteins within Set 2, resulting in 826 proteins being increased and 795 proteins decreased in AD (Supplemental Table 19 [this file] ). \n",
      "As a result of this selection process, 12 non-overlapping samples were excluded from Set 2, resulting in a total of 109 samples (Set 1 = 36 samples, Set 2 = 73 samples) for further analysis (Supplemental Table 20 [this file] ). This -analysis revealed a substantial increase in differentially abundant proteins in the Hp-enriched AD plasma, with 945 proteins showing an increase and 923 proteins showing a decrease in AD, as compared to analyzing either dataset alone (Fig. 5 [rid=Fig5] A and Supplemental Table 21 [this file] ). To assess the relationship between Hp-enriched plasma proteins and AD biomarkers and cognitive measures, we conducted correlation analyses between Z-transformed protein abundance (Supplemental Table 22 [this file] ) and MoCA scores, as well as immunoassay values of CSF (Aβ , tTau, pTau181, tTau/Aβ ) and plasma pTau181 (Supplemental Table 23 [this file] ). Correlation between SomaScan or Olink protein abundance and MoCA scores, as well as immunoassay values of CSF (Aβ , tTau, pTau181, tTau/Aβ ) and plasma pTau181 were also provided in Supplemental Table 24–25 [this file] . To ensure that the observed changes in these proteins in AD were not influenced by age, sex, or race, we performed regression analyses for these traits within Set 1 and Set 2, followed by the same -analysis described earlier (Supplemental Fig. 6 [rid=MOESM6] and Supplemental Table 26 [this file] ). \n",
      "ROC curve statistics for highly significant proteins, including AUC, -value, 95% DeLong confidence interval, accuracy, specificity, and sensitivity for AD vs CTL, are provided in Supplemental Table 27 [this file] . \n",
      "This analysis used plasma pTau181 measurements and only included plasma proteins with no missing values and those that were highly significant ( -value < 0.0001, = 536 proteins) across 34 control and 61 AD individuals (Supplemental Table 27 [this file] ). \n",
      "To ensure the consistency of protein overlap analysis, we employed the same Uniprot database and utilized the FragPipe search algorithm we used for Heparin-MS plasma analysis to re-analyze 456 dorsolateral prefrontal cortex (DLPFC) tissues from control, asymptomatic AD (AsymAD) and AD brains from the ROSMAP and the Banner cohorts (Supplemental Fig. 7 [rid=MOESM7] A and Supplemental Table 28 [this file] ) [ 11 [rid=CR11] ]. By applying a data normalization approach similar to the one used for the Hp-enriched plasma samples, we successfully identified 8,956 proteins detected in 50% or more of the samples from brain tissues corresponding to 8,904 unique gene symbols (Supplemental Table 29 [this file] ). When compared to a previous brain proteome dataset generated using Proteome Discoverer (PD) and an older 2015 Uniprot protein database [ 11 [rid=CR11] ], it was found that 7,476 of these unique gene symbols overlapped between the two outputs, while 1,428 were exclusive to the FragPipe, representing an approximately 18% increase in the number of identifications (Supplemental Fig. 7 [rid=MOESM7] B and Supplemental Table 30 [this file] ). Furthermore, to ensure the consistency of the direction of change in the AD brain proteome compared to controls, we correlated the differentially abundant proteins across these two sets of results (Supplemental Table 31–32 [this file] ), revealing a correlation coefficient of 0.9 between AD and control groups and a correlation coefficient of 0.84 between AsymAD and control groups (Supplemental Fig. 7 [rid=MOESM7] C). This mapping was achieved by recalculating the kME (bicor correlation to module eigenprotein) for each protein and assigning it to the module with the highest correlation (Supplemental Table 33 [this file] ). When comparing the proteins identified in plasma with those in the brain proteome, we discovered that a substantial portion, approximately 77% (2211 out of 2685) of the plasma-identified proteins were also quantified in the brain (Fig. 7 [rid=Fig7] B and Supplemental Table 34 [this file] ). This represents a significant improvement compared to the immunodepletion approach, which only detected 5 out of 35 members (APOE4, APP, DAG1, GPNMB, and LRP1), reflecting a mere 14% of the module (Supplemental Tables 8 [this file] and 33 [this file] ). \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 263 - INFO - Extracting links from full XML content.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 301 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 309 - INFO - Content length: 302368\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 326 - INFO - Prompt messages total length: 76345 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 332 - INFO - Prompt ID: gemini-2.5-flash-0-ac0e94fbe43e65015d4fa251fa9fd315d30b279d02d397a2c7996697193bed54\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 351 - INFO - Requesting datasets from content using model: gemini-2.5-flash, temperature: 0, messages length: 76345 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-c2401f5b-dac9-4ddd-9217-bd7c024da06a\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn52525880\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn52525880/files/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/HeparinPlasma\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn52525880\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn52525880/files/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/HeparinPlasma\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1758210225,\n",
      "    \"model\": \"gemini-2.5-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 93779,\n",
      "        \"completion_tokens\": 104,\n",
      "        \"total_tokens\": 99182,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 5299,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 664 - INFO - Deduplicating response with 2 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn52525880\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn52525880', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn52525880/files/', 'dataset_id': 'syn52525880'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn52525880\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn52525880\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn52525880/files/, identifier: syn52525880, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn52525880/files/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn52525880 for https://www.synapse.org/#!Synapse:syn52525880/files/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn52525880 for https://www.synapse.org/#!Synapse:syn52525880/files/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn52525880 for https://www.synapse.org/#!Synapse:syn52525880/files/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn52525880/files/ in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn52525880/files/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn52525880/files/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn52525880/files/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn52525880\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn52525880\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn52525880\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn52525880/files/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn52525880\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn52525880\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn52525880, synapse.org, https://www.synapse.org/#!Synapse:syn52525880/files/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn52525880\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn52525880/files/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn52525880', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn52525880/files/', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.synapse.org/HeparinPlasma', 'dataset_id': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/HeparinPlasma, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/HeparinPlasma\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/HeparinPlasma in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 644 - INFO - Dataset ID is 'n/a' and repository name from prompt\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 541 - INFO - Skipping dataset due to missing ID, repository, dataset page: {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.synapse.org/HeparinPlasma', 'dataset_id': 'n/a'}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 278 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn52525880', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn52525880/files/', 'citation_type': 'n/a'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1191 - INFO - Fetching metadata for 1 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn52525880\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn52525880, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn52525880 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn52525880 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn52525880 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn52525880 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn52525880\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn52525880\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1277 - INFO - Updated datasets len: 1\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 287 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 10, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'source_url', 'download_link',\n",
      "       'title', 'content_type', 'id', 'caption', 'description',\n",
      "       'source_section', 'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 500 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 506 - INFO - XML element classification not needed. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1012 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11460197\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1020 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11460197\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 640th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11462282)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11462282\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11462282 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 634 - INFO - Fetching data for pmc11462282\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 637 - INFO - Fetching data from scripts/exp_input/Local_fetched_data_1.parquet\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 438 - INFO - Raw html data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11462282 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 452 - INFO - Skipping raw HTML/XML saving.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 482 - INFO - Using HTMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 52 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 82 - INFO - Initializing htmlRetriever\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 350 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11462282, additional_data, html)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 448 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC11462282)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 550 - INFO - Extracted 14 unique supplementary material links from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 560 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#ab0010'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for ab0010: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#sec0015'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for sec0015: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#sec0015'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for sec0015: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#sec0050'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for sec0050: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#sec0050'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for sec0050: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#ec0005'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for ec0005: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#ec0010'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for ec0010: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#ec0015'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for ec0015: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#ec0020'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for ec0020: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#ec0025'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for ec0025: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#ec0030'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for ec0030: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#ec0035'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for ec0035: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#ec0040'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for ec0040: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 368 - INFO - Extracting links from full HTML content.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 301 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 309 - INFO - Content length: 162008\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 326 - INFO - Prompt messages total length: 41255 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 332 - INFO - Prompt ID: gemini-2.5-flash-0-d6e252a8646c084105372504e4466056c1650773b7ab590ca43e10205b3bfda2\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 351 - INFO - Requesting datasets from content using model: gemini-2.5-flash, temperature: 0, messages length: 41255 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-bba60555-9ee2-4c10-96ef-ad775ea21318\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn2374375\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://portal.gdc.cancer.gov/\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn2374375\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://portal.gdc.cancer.gov/\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1758210248,\n",
      "    \"model\": \"gemini-2.5-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 56867,\n",
      "        \"completion_tokens\": 85,\n",
      "        \"total_tokens\": 60648,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 3696,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 664 - INFO - Deduplicating response with 2 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn2374375\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn2374375', 'repository_reference': 'https://www.synapse.org/', 'dataset_id': 'syn2374375'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn2374375\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn2374375\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/, identifier: syn2374375, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn2374375 for https://www.synapse.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn2374375 for https://www.synapse.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn2374375 for https://www.synapse.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/ in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn2374375\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn2374375\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn2374375\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn2374375, synapse.org, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn2374375\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn2374375', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://portal.gdc.cancer.gov/', 'dataset_id': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://portal.gdc.cancer.gov/, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://portal.gdc.cancer.gov/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://portal.gdc.cancer.gov/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'portal.gdc.cancer.gov', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 378 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn2374375', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'portal.gdc.cancer.gov', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1191 - INFO - Fetching metadata for 2 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn2374375\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn2374375, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn2374375 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn2374375 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn2374375 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn2374375 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn2374375\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn2374375\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 2: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1277 - INFO - Updated datasets len: 2\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 414 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 16, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'title', 'file_info',\n",
      "       'description', 'source_section', 'section_class', 'download_link',\n",
      "       'file_extension', 'a_attr_href', 'a_attr_class',\n",
      "       'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 173 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 183 - WARNING - No publication title found in the HTML data.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 643 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 494 - INFO - Parsed data extraction completed. Elements collected: 11\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 500 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 510 - INFO - HTML element classification not supported. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1012 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11462282\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1020 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11462282\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 641/2190 (29.27%) | Elapsed: 00:06:08 | ETA: 00:14:50\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 641th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11462655)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11462655\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11462655 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 634 - INFO - Fetching data for pmc11462655\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 637 - INFO - Fetching data from scripts/exp_input/Local_fetched_data_1.parquet\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 438 - INFO - Raw html data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11462655 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 452 - INFO - Skipping raw HTML/XML saving.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 482 - INFO - Using HTMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 52 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 82 - INFO - Initializing htmlRetriever\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 350 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11462655, additional_data, html)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 448 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC11462655)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 550 - INFO - Extracted 5 unique supplementary material links from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 560 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#clp_a'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for clp_a: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#Bib1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for Bib1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#Bib1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for Bib1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#Bib1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for Bib1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 368 - INFO - Extracting links from full HTML content.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 301 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 309 - INFO - Content length: 69508\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 326 - INFO - Prompt messages total length: 18130 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 332 - INFO - Prompt ID: gemini-2.5-flash-0-88da25f3c9851249662501f2859b70cb37fd4c70693ccb2e85738eef63feb2f0\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 351 - INFO - Requesting datasets from content using model: gemini-2.5-flash, temperature: 0, messages length: 18130 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-1d25465f-e8fb-4b0c-92a7-1173cb863df6\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn26454542\\\",\\n    \\\"repository_reference\\\": \\\"https://doi.org/10.7303/syn26454542\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://github.com/cefet-rj-dal-members/ccimm\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn25148356\\\",\\n    \\\"repository_reference\\\": \\\"https://doi.org/10.7303/syn25148356\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn25049520\\\",\\n    \\\"repository_reference\\\": \\\"https://doi.org/10.7303/syn25049520\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn26454542\\\",\\n    \\\"repository_reference\\\": \\\"https://doi.org/10.7303/syn26454542\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://github.com/cefet-rj-dal-members/ccimm\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn25148356\\\",\\n    \\\"repository_reference\\\": \\\"https://doi.org/10.7303/syn25148356\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn25049520\\\",\\n    \\\"repository_reference\\\": \\\"https://doi.org/10.7303/syn25049520\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1758210277,\n",
      "    \"model\": \"gemini-2.5-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 23818,\n",
      "        \"completion_tokens\": 220,\n",
      "        \"total_tokens\": 28750,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 4712,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 664 - INFO - Deduplicating response with 4 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn26454542\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn25148356\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn25049520\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn26454542', 'repository_reference': 'https://doi.org/10.7303/syn26454542', 'dataset_id': 'syn26454542'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn26454542\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn26454542\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://doi.org/10.7303/syn26454542, identifier: syn26454542, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://doi.org/10.7303/syn26454542\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn26454542 for https://doi.org/10.7303/syn26454542\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn26454542 for https://doi.org/10.7303/syn26454542\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn26454542 for https://doi.org/10.7303/syn26454542\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier syn26454542 for https://doi.org/10.7303/syn26454542\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1157 - INFO - Found id_pattern match for https://doi.org/10.7303/syn26454542 in syn\\d{7,12}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://doi.org/10.7303/syn26454542\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://doi.org/10.7303/syn26454542\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn26454542\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn26454542\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://doi.org/__ID__ to value: https://doi.org/10.7303/syn26454542\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: 10\\.\\d{4,9}/[^\\s,]+ to value: https://doi.org/10.7303/syn26454542\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(10\\.\\d{4,9}/[^\\s,]+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(10\\.\\d{4,9}/[^\\s,]+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn26454542\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn26454542\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://doi.org/10.7303/syn26454542\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn26454542\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn26454542\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn26454542, synapse.org, https://doi.org/10.7303/syn26454542\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn26454542\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://doi.org/10.7303/syn26454542\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn26454542', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://doi.org/10.7303/syn26454542', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://github.com/cefet-rj-dal-members/ccimm', 'dataset_id': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://github.com/cefet-rj-dal-members/ccimm, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://github.com/cefet-rj-dal-members/ccimm\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://github.com/cefet-rj-dal-members/ccimm\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn25148356', 'repository_reference': 'https://doi.org/10.7303/syn25148356', 'dataset_id': 'syn25148356'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn25148356\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn25148356\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://doi.org/10.7303/syn25148356, identifier: syn25148356, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://doi.org/10.7303/syn25148356\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn25148356 for https://doi.org/10.7303/syn25148356\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn25148356 for https://doi.org/10.7303/syn25148356\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn25148356 for https://doi.org/10.7303/syn25148356\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier syn25148356 for https://doi.org/10.7303/syn25148356\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1157 - INFO - Found id_pattern match for https://doi.org/10.7303/syn25148356 in syn\\d{7,12}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://doi.org/10.7303/syn25148356\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://doi.org/10.7303/syn25148356\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn25148356\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn25148356\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://doi.org/__ID__ to value: https://doi.org/10.7303/syn25148356\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: 10\\.\\d{4,9}/[^\\s,]+ to value: https://doi.org/10.7303/syn25148356\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(10\\.\\d{4,9}/[^\\s,]+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(10\\.\\d{4,9}/[^\\s,]+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn25148356\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn25148356\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://doi.org/10.7303/syn25148356\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn25148356\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn25148356\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn25148356, synapse.org, https://doi.org/10.7303/syn25148356\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn25148356\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://doi.org/10.7303/syn25148356\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn25148356', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://doi.org/10.7303/syn25148356', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn25049520', 'repository_reference': 'https://doi.org/10.7303/syn25049520', 'dataset_id': 'syn25049520'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn25049520\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn25049520\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://doi.org/10.7303/syn25049520, identifier: syn25049520, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://doi.org/10.7303/syn25049520\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn25049520 for https://doi.org/10.7303/syn25049520\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn25049520 for https://doi.org/10.7303/syn25049520\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn25049520 for https://doi.org/10.7303/syn25049520\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier syn25049520 for https://doi.org/10.7303/syn25049520\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1157 - INFO - Found id_pattern match for https://doi.org/10.7303/syn25049520 in syn\\d{7,12}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://doi.org/10.7303/syn25049520\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://doi.org/10.7303/syn25049520\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn25049520\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn25049520\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://doi.org/__ID__ to value: https://doi.org/10.7303/syn25049520\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: 10\\.\\d{4,9}/[^\\s,]+ to value: https://doi.org/10.7303/syn25049520\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(10\\.\\d{4,9}/[^\\s,]+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(10\\.\\d{4,9}/[^\\s,]+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn25049520\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn25049520\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://doi.org/10.7303/syn25049520\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn25049520\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn25049520\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn25049520, synapse.org, https://doi.org/10.7303/syn25049520\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn25049520\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://doi.org/10.7303/syn25049520\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn25049520', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://doi.org/10.7303/syn25049520', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 378 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn26454542', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://doi.org/10.7303/syn26454542', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'github.com', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn25148356', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://doi.org/10.7303/syn25148356', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn25049520', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://doi.org/10.7303/syn25049520', 'citation_type': 'n/a'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1191 - INFO - Fetching metadata for 4 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn26454542\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn26454542, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn26454542 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn26454542 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn26454542 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn26454542 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn26454542\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn26454542\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 2: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn25148356\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn25148356, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn25148356 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn25148356 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn25148356 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn25148356 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 3 with repo: synapse.org and accession_id: syn25148356\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn25148356\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 3: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn25049520\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn25049520, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn25049520 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn25049520 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn25049520 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn25049520 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 4 with repo: synapse.org and accession_id: syn25049520\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn25049520\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 4: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1277 - INFO - Updated datasets len: 4\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 414 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 9, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'title', 'file_info',\n",
      "       'description', 'source_section', 'section_class', 'download_link',\n",
      "       'file_extension', 'a_attr_href', 'a_attr_class',\n",
      "       'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 173 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 183 - WARNING - No publication title found in the HTML data.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 643 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 494 - INFO - Parsed data extraction completed. Elements collected: 5\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 500 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 510 - INFO - HTML element classification not supported. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1012 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11462655\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1020 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11462655\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 642th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11463607)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11463607\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11463607 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 634 - INFO - Fetching data for pmc11463607\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 637 - INFO - Fetching data from scripts/exp_input/Local_fetched_data_1.parquet\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 438 - INFO - Raw html data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11463607 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 452 - INFO - Skipping raw HTML/XML saving.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 482 - INFO - Using HTMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 52 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 82 - INFO - Initializing htmlRetriever\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 350 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11463607, additional_data, html)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 448 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC11463607)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 550 - INFO - Extracted 6 unique supplementary material links from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 560 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#clp_a'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for clp_a: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#S6'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for S6: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#S19'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for S19: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#S20'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for S20: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#S23'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for S23: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 368 - INFO - Extracting links from full HTML content.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 301 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 309 - INFO - Content length: 112451\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 326 - INFO - Prompt messages total length: 28866 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 332 - INFO - Prompt ID: gemini-2.5-flash-0-7826f226d3e6f26c6c6ed321496c8c82ca53b380b2193766763c68504da96b66\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 351 - INFO - Requesting datasets from content using model: gemini-2.5-flash, temperature: 0, messages length: 28866 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-300da0ed-cc5f-4149-8d46-89360b3c6a3a\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE178317\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178317\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn2580853\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn11724057\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://cellxgene.cziscience.com/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE178317\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178317\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn2580853\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn11724057\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://cellxgene.cziscience.com/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1758210308,\n",
      "    \"model\": \"gemini-2.5-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 37604,\n",
      "        \"completion_tokens\": 219,\n",
      "        \"total_tokens\": 43280,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 5457,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 664 - INFO - Deduplicating response with 5 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: GSE178317\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn2580853\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn11724057\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 693 - INFO - Duplicate found and skipped: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'GSE178317', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178317', 'dataset_id': 'GSE178317'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE178317\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: GSE178317\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178317, identifier: GSE178317, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178317\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE178317 for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178317\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier GSE178317 for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178317\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1157 - INFO - Found id_pattern match for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178317 in GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178317\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178317\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: GSE178317\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE178317\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE178317\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178317\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: GSE178317\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE178317\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: GSE178317, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178317\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: GSE178317\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178317\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'GSE178317', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178317', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn2580853', 'repository_reference': 'https://www.synapse.org/', 'dataset_id': 'syn2580853'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn2580853\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn2580853\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/, identifier: syn2580853, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn2580853 for https://www.synapse.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn2580853 for https://www.synapse.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn2580853 for https://www.synapse.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/ in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn2580853\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn2580853\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn2580853\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn2580853, synapse.org, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn2580853\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn2580853', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn11724057', 'repository_reference': 'https://www.synapse.org/', 'dataset_id': 'syn11724057'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn11724057\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn11724057\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/, identifier: syn11724057, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn11724057 for https://www.synapse.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn11724057 for https://www.synapse.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn11724057 for https://www.synapse.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/ in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn11724057\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn11724057\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn11724057\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn11724057, synapse.org, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn11724057\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn11724057', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://cellxgene.cziscience.com/', 'dataset_id': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://cellxgene.cziscience.com/, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://cellxgene.cziscience.com/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://cellxgene.cziscience.com/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'cellxgene.cziscience.com', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 378 - INFO - Augmented dataset links: [{'dataset_identifier': 'GSE178317', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178317', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn2580853', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn11724057', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'cellxgene.cziscience.com', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1191 - INFO - Fetching metadata for 4 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: GSE178317\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: geo, identifier: GSE178317, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE178317 for geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for GSE178317 in geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE178317\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178317\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 1: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn2580853\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn2580853, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn2580853 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn2580853 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn2580853 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn2580853 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn2580853\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn2580853\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn11724057\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn11724057, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn11724057 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn11724057 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn11724057 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn11724057 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 3 with repo: synapse.org and accession_id: syn11724057\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn11724057\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 3: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 4: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1277 - INFO - Updated datasets len: 4\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 414 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 10, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'title', 'file_info',\n",
      "       'description', 'source_section', 'section_class', 'download_link',\n",
      "       'file_extension', 'a_attr_href', 'a_attr_class',\n",
      "       'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 173 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 183 - WARNING - No publication title found in the HTML data.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 643 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 494 - INFO - Parsed data extraction completed. Elements collected: 5\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 500 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 510 - INFO - HTML element classification not supported. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1012 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11463607\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1020 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11463607\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 643th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11463662)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11463662\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11463662 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 634 - INFO - Fetching data for pmc11463662\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 637 - INFO - Fetching data from scripts/exp_input/Local_fetched_data_1.parquet\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 438 - INFO - Raw html data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11463662 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 452 - INFO - Skipping raw HTML/XML saving.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 482 - INFO - Using HTMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 52 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 82 - INFO - Initializing htmlRetriever\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 350 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11463662, additional_data, html)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 448 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC11463662)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 550 - INFO - Extracted 4 unique supplementary material links from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 560 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#clp_a'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for clp_a: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#ABS1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for ABS1: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#S17'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for S17: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 368 - INFO - Extracting links from full HTML content.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 301 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 309 - INFO - Content length: 141117\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 326 - INFO - Prompt messages total length: 36033 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 332 - INFO - Prompt ID: gemini-2.5-flash-0-05e4b9c03d74c4a7db8d320193256b128245cf44eb2510114c60a0b966811e8c\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 351 - INFO - Requesting datasets from content using model: gemini-2.5-flash, temperature: 0, messages length: 36033 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-74230fe2-5101-4f86-9f69-60cd61a50a73\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn61804540\\\",\\n    \\\"repository_reference\\\": \\\"https://doi.org/10.7303/syn61804540\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn61804540\\\",\\n    \\\"repository_reference\\\": \\\"https://doi.org/10.7303/syn61804540\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1758210440,\n",
      "    \"model\": \"gemini-2.5-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 50539,\n",
      "        \"completion_tokens\": 64,\n",
      "        \"total_tokens\": 52226,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 1623,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 664 - INFO - Deduplicating response with 1 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn61804540\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn61804540', 'repository_reference': 'https://doi.org/10.7303/syn61804540', 'dataset_id': 'syn61804540'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn61804540\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn61804540\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://doi.org/10.7303/syn61804540, identifier: syn61804540, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://doi.org/10.7303/syn61804540\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn61804540 for https://doi.org/10.7303/syn61804540\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn61804540 for https://doi.org/10.7303/syn61804540\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn61804540 for https://doi.org/10.7303/syn61804540\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier syn61804540 for https://doi.org/10.7303/syn61804540\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1157 - INFO - Found id_pattern match for https://doi.org/10.7303/syn61804540 in syn\\d{7,12}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://doi.org/10.7303/syn61804540\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://doi.org/10.7303/syn61804540\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn61804540\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn61804540\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://doi.org/__ID__ to value: https://doi.org/10.7303/syn61804540\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: 10\\.\\d{4,9}/[^\\s,]+ to value: https://doi.org/10.7303/syn61804540\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(10\\.\\d{4,9}/[^\\s,]+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(10\\.\\d{4,9}/[^\\s,]+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn61804540\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn61804540\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://doi.org/10.7303/syn61804540\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn61804540\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn61804540\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn61804540, synapse.org, https://doi.org/10.7303/syn61804540\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn61804540\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://doi.org/10.7303/syn61804540\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn61804540', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://doi.org/10.7303/syn61804540', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 378 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn61804540', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://doi.org/10.7303/syn61804540', 'citation_type': 'n/a'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1191 - INFO - Fetching metadata for 1 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn61804540\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn61804540, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn61804540 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn61804540 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn61804540 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn61804540 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn61804540\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn61804540\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1277 - INFO - Updated datasets len: 1\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 414 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 5, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'title', 'file_info',\n",
      "       'description', 'source_section', 'section_class', 'download_link',\n",
      "       'file_extension', 'a_attr_href', 'a_attr_class',\n",
      "       'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 173 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 183 - WARNING - No publication title found in the HTML data.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 643 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 494 - INFO - Parsed data extraction completed. Elements collected: 2\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 500 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 510 - INFO - HTML element classification not supported. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1012 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11463662\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1020 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11463662\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 644th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11465928)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11465928\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11465928 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 634 - INFO - Fetching data for pmc11465928\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 637 - INFO - Fetching data from scripts/exp_input/Local_fetched_data_1.parquet\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 438 - INFO - Raw html data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11465928 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 452 - INFO - Skipping raw HTML/XML saving.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 482 - INFO - Using HTMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 52 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 82 - INFO - Initializing htmlRetriever\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 350 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11465928, additional_data, html)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 448 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC11465928)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 550 - INFO - Extracted 11 unique supplementary material links from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 560 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#clp_a'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for clp_a: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#notes3'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for notes3: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#notes3'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for notes3: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#notes3'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for notes3: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#fn-group1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for fn-group1: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#Bib1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for Bib1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#Bib1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for Bib1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#Bib1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for Bib1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#Bib1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for Bib1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#Bib1_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for Bib1_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 368 - INFO - Extracting links from full HTML content.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 301 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 309 - INFO - Content length: 197350\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 326 - INFO - Prompt messages total length: 50091 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 332 - INFO - Prompt ID: gemini-2.5-flash-0-35e6d2ef2d467979ec0e2626f92442079fc0a76360f5d544f6ae53373fce005e\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 351 - INFO - Requesting datasets from content using model: gemini-2.5-flash, temperature: 0, messages length: 50091 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-5d58967e-a6e9-4994-b3e4-f33747233802\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"17094\\\",\\n    \\\"repository_reference\\\": \\\"https://competitions.codalab.org/competitions/17094\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn3193805\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn3193805/wiki/217789\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Chaos\\\",\\n    \\\"repository_reference\\\": \\\"https://chaos.grand-challenge.org/Download/\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"17094\\\",\\n    \\\"repository_reference\\\": \\\"https://competitions.codalab.org/competitions/17094\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn3193805\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn3193805/wiki/217789\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Chaos\\\",\\n    \\\"repository_reference\\\": \\\"https://chaos.grand-challenge.org/Download/\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1758210463,\n",
      "    \"model\": \"gemini-2.5-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 67249,\n",
      "        \"completion_tokens\": 154,\n",
      "        \"total_tokens\": 70651,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 3248,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 664 - INFO - Deduplicating response with 3 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: 17094\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn3193805\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: Chaos\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': '17094', 'repository_reference': 'https://competitions.codalab.org/competitions/17094', 'dataset_id': '17094'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: 17094\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: 17094\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://competitions.codalab.org/competitions/17094, identifier: 17094, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://competitions.codalab.org/competitions/17094\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 17094 for https://competitions.codalab.org/competitions/17094\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier 17094 for https://competitions.codalab.org/competitions/17094\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier 17094 for https://competitions.codalab.org/competitions/17094\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier 17094 for https://competitions.codalab.org/competitions/17094\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier 17094 for https://competitions.codalab.org/competitions/17094\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier 17094 for https://competitions.codalab.org/competitions/17094\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1157 - INFO - Found id_pattern match for https://competitions.codalab.org/competitions/17094 in [1-9][a-zA-Z0-9]{3}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://competitions.codalab.org/competitions/17094\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://competitions.codalab.org/competitions/17094\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: 17094\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: 17094\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: 17094, pdb, https://competitions.codalab.org/competitions/17094\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: 17094\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://competitions.codalab.org/competitions/17094\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': '17094', 'data_repository': 'pdb', 'dataset_webpage': 'https://competitions.codalab.org/competitions/17094', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn3193805', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn3193805/wiki/217789', 'dataset_id': 'syn3193805'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn3193805\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn3193805\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn3193805/wiki/217789, identifier: syn3193805, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn3193805/wiki/217789\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3193805 for https://www.synapse.org/#!Synapse:syn3193805/wiki/217789\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn3193805 for https://www.synapse.org/#!Synapse:syn3193805/wiki/217789\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3193805 for https://www.synapse.org/#!Synapse:syn3193805/wiki/217789\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn3193805/wiki/217789 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn3193805/wiki/217789\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn3193805/wiki/217789\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn3193805/wiki/217789\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn3193805\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn3193805\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn3193805\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn3193805/wiki/217789\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn3193805\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn3193805\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn3193805, synapse.org, https://www.synapse.org/#!Synapse:syn3193805/wiki/217789\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn3193805\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn3193805/wiki/217789\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn3193805', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn3193805/wiki/217789', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'Chaos', 'repository_reference': 'https://chaos.grand-challenge.org/Download/', 'dataset_id': 'Chaos'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: Chaos\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID Chaos is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://chaos.grand-challenge.org/Download/, identifier: Chaos, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://chaos.grand-challenge.org/Download/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Chaos for https://chaos.grand-challenge.org/Download/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier Chaos for https://chaos.grand-challenge.org/Download/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Chaos for https://chaos.grand-challenge.org/Download/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Chaos for https://chaos.grand-challenge.org/Download/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier Chaos for https://chaos.grand-challenge.org/Download/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier Chaos for https://chaos.grand-challenge.org/Download/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://chaos.grand-challenge.org/Download/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'Chaos', 'data_repository': 'chaos.grand-challenge.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 378 - INFO - Augmented dataset links: [{'dataset_identifier': '17094', 'data_repository': 'pdb', 'dataset_webpage': 'https://competitions.codalab.org/competitions/17094', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn3193805', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn3193805/wiki/217789', 'citation_type': 'n/a'}, {'dataset_identifier': 'Chaos', 'data_repository': 'chaos.grand-challenge.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1191 - INFO - Fetching metadata for 3 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: 17094\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: pdb, identifier: 17094, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 17094 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier 17094 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier 17094 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier 17094 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier 17094 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for 17094 in pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 1 with repo: pdb and accession_id: 17094\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.rcsb.org/structure/17094\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn3193805\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn3193805, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3193805 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn3193805 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3193805 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn3193805 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn3193805\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn3193805\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: Chaos\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: chaos.grand-challenge.org, identifier: Chaos, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Chaos for chaos.grand-challenge.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier Chaos for chaos.grand-challenge.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Chaos for chaos.grand-challenge.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Chaos for chaos.grand-challenge.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier Chaos for chaos.grand-challenge.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier Chaos for chaos.grand-challenge.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: chaos.grand-challenge.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL chaos.grand-challenge.org may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for Chaos in chaos.grand-challenge.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 3 with repo: chaos.grand-challenge.org and accession_id: Chaos\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository chaos.grand-challenge.org unknown in Ontology. Skipping dataset page 3.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1277 - INFO - Updated datasets len: 3\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 414 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 14, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'title', 'file_info',\n",
      "       'description', 'source_section', 'section_class', 'download_link',\n",
      "       'file_extension', 'a_attr_href', 'a_attr_class',\n",
      "       'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 173 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 183 - WARNING - No publication title found in the HTML data.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 643 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 494 - INFO - Parsed data extraction completed. Elements collected: 4\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 500 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 510 - INFO - HTML element classification not supported. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1012 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11465928\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1020 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11465928\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 645th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11466430)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11466430\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11466430 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 634 - INFO - Fetching data for pmc11466430\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 637 - INFO - Fetching data from scripts/exp_input/Local_fetched_data_1.parquet\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 438 - INFO - Raw html data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11466430 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 452 - INFO - Skipping raw HTML/XML saving.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 482 - INFO - Using HTMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 52 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 82 - INFO - Initializing htmlRetriever\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 350 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11466430, additional_data, html)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 448 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC11466430)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 550 - INFO - Extracted 40 unique supplementary material links from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 560 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#clp_a'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for clp_a: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 2 <a> elements with href='#pgen.1011412.s001'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 24) for pgen.1011412.s001: S1 Fig [href=this file]\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 3 <a> elements with href='#pgen.1011412.s002'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 74) for pgen.1011412.s002: S2 Fig [href=this file]\n",
      "S2B Fig [href=this file]\n",
      "S2A Fig [href=this file]\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 3 <a> elements with href='#pgen.1011412.s003'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 74) for pgen.1011412.s003: S3 Fig [href=this file]\n",
      "S3A Fig [href=this file]\n",
      "S3B Fig [href=this file]\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 1 <a> elements with href='#pgen.1011412.s004'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 24) for pgen.1011412.s004: S4 Fig [href=this file]\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 1 <a> elements with href='#pgen.1011412.s005'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 24) for pgen.1011412.s005: S5 Fig [href=this file]\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 1 <a> elements with href='#pgen.1011412.s006'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 58) for pgen.1011412.s006: S1 [href=this file] – S5 [href=#pgen.1011412.s010] Tables\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#pgen.1011412.s007'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for pgen.1011412.s007: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#pgen.1011412.s008'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for pgen.1011412.s008: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 2 <a> elements with href='#pgen.1011412.s009'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 26) for pgen.1011412.s009: S4 Table [href=this file]\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 2 <a> elements with href='#pgen.1011412.s010'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 84) for pgen.1011412.s010: S1 [href=#pgen.1011412.s006] – S5 [href=this file] Tables\n",
      "S5 Table [href=this file]\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#notes1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for notes1: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#notes1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for notes1: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#notes1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for notes1: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#notes1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for notes1: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#notes1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for notes1: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#notes1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for notes1: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#notes1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for notes1: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#notes1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for notes1: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#pgen.1011412.s011'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for pgen.1011412.s011: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#pgen.1011412.s012'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for pgen.1011412.s012: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 368 - INFO - Extracting links from full HTML content.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 301 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 309 - INFO - Content length: 181325\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 326 - INFO - Prompt messages total length: 46085 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 332 - INFO - Prompt ID: gemini-2.5-flash-0-39c93292fcbcef6f3faa3566fc334531c4000bee14f8465cceb39b5dd85d90b4\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 351 - INFO - Requesting datasets from content using model: gemini-2.5-flash, temperature: 0, messages length: 46085 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-be5ec50d-3bf9-4de4-a7e2-434daf4352a9\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn22264775\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn22264775\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn21241740\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn21241740\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"gnomAD v2.1.1\\\",\\n    \\\"repository_reference\\\": \\\"https://gnomad.broadinstitute.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GTEx project version 8\\\",\\n    \\\"repository_reference\\\": \\\"https://gtexportal.org/home/downloads/adult-gtex/bulk_tissue_expression\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn22264775\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn22264775\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn21241740\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn21241740\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"gnomAD v2.1.1\\\",\\n    \\\"repository_reference\\\": \\\"https://gnomad.broadinstitute.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GTEx project version 8\\\",\\n    \\\"repository_reference\\\": \\\"https://gtexportal.org/home/downloads/adult-gtex/bulk_tissue_expression\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1758210485,\n",
      "    \"model\": \"gemini-2.5-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 62769,\n",
      "        \"completion_tokens\": 212,\n",
      "        \"total_tokens\": 66252,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 3271,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 664 - INFO - Deduplicating response with 4 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn22264775\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn21241740\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: gnomAD v2.1.1\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: GTEx project version 8\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn22264775', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn22264775', 'dataset_id': 'syn22264775'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn22264775\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn22264775\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn22264775, identifier: syn22264775, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn22264775\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn22264775 for https://www.synapse.org/#!Synapse:syn22264775\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn22264775 for https://www.synapse.org/#!Synapse:syn22264775\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn22264775 for https://www.synapse.org/#!Synapse:syn22264775\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn22264775 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn22264775\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn22264775\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn22264775\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn22264775\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn22264775\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn22264775\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn22264775\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn22264775\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn22264775\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn22264775, synapse.org, https://www.synapse.org/#!Synapse:syn22264775\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn22264775\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn22264775\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn22264775', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn22264775', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn21241740', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn21241740', 'dataset_id': 'syn21241740'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn21241740\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn21241740\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn21241740, identifier: syn21241740, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn21241740\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn21241740 for https://www.synapse.org/#!Synapse:syn21241740\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn21241740 for https://www.synapse.org/#!Synapse:syn21241740\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn21241740 for https://www.synapse.org/#!Synapse:syn21241740\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn21241740 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn21241740\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn21241740\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn21241740\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn21241740\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn21241740\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn21241740\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn21241740\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn21241740\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn21241740\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn21241740, synapse.org, https://www.synapse.org/#!Synapse:syn21241740\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn21241740\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn21241740\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn21241740', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn21241740', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'gnomAD v2.1.1', 'repository_reference': 'https://gnomad.broadinstitute.org/', 'dataset_id': 'gnomAD v2.1.1'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: gnomAD v2.1.1\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID gnomAD v2.1.1 is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://gnomad.broadinstitute.org/, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://gnomad.broadinstitute.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://gnomad.broadinstitute.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'gnomad.broadinstitute.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'GTEx project version 8', 'repository_reference': 'https://gtexportal.org/home/downloads/adult-gtex/bulk_tissue_expression', 'dataset_id': 'GTEx project version 8'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: GTEx project version 8\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID GTEx project version 8 is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://gtexportal.org/home/downloads/adult-gtex/bulk_tissue_expression, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://gtexportal.org/home/downloads/adult-gtex/bulk_tissue_expression\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://gtexportal.org/home/downloads/adult-gtex/bulk_tissue_expression\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'gtexportal.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 378 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn22264775', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn22264775', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn21241740', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn21241740', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'gnomad.broadinstitute.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'gtexportal.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1191 - INFO - Fetching metadata for 4 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn22264775\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn22264775, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn22264775 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn22264775 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn22264775 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn22264775 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn22264775\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn22264775\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn21241740\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn21241740, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn21241740 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn21241740 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn21241740 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn21241740 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn21241740\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn21241740\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 3: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 4: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1277 - INFO - Updated datasets len: 4\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 414 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 44, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'title', 'file_info',\n",
      "       'description', 'source_section', 'section_class', 'download_link',\n",
      "       'file_extension', 'a_attr_href', 'a_attr_class',\n",
      "       'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 173 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 183 - WARNING - No publication title found in the HTML data.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 643 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 494 - INFO - Parsed data extraction completed. Elements collected: 16\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 500 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 510 - INFO - HTML element classification not supported. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1012 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11466430\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1020 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11466430\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 646th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11469718)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11469718\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11469718 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 634 - INFO - Fetching data for pmc11469718\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 637 - INFO - Fetching data from scripts/exp_input/Local_fetched_data_1.parquet\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 438 - INFO - Raw xml data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11469718 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 452 - INFO - Skipping raw HTML/XML saving.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 458 - INFO - Using XMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 52 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 24 - INFO - Initializing xmlRetriever\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 466 - INFO - No additional data provided. Parsing raw data only.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 147 - INFO - Function call: parse_data(api_data(<class 'str'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11469718, additional_data, XML)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 162 - INFO - api_data converted to lxml element\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 466 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 153 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 154 - INFO - Consider migrating this function to the BaseRetriever class.\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 69 - INFO - Loading target sections for section name: supplementary_material_sections\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 477 - INFO - Found 2 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x15e9e1180>, <Element supplementary-material at 0x15fa2b5c0>]\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 485 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x15e9e1180>, <Element supplementary-material at 0x15fa2b5c0>]}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 574 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref SD1: A description of the ARIC participants included in the genetic proteomic analyses is presented in Supplementary Table 1 [this file] . \n",
      "After conditional analysis, we identified 1,746 significant aptamer-region pairs associations (37% cis) among 807 regions and 1,408 proteins in AA ( Supplementary Tables 2 [this file] and 3 [rid=SD2] , Figure 2 [rid=F2] ) and 4,315 significant associations (24% cis) among 1,490 regions and 2,565 proteins in EA ( Supplementary Tables 2 [this file] and 4 [rid=SD2] , Figure 2 [rid=F2] ). The breakdown of cis versus trans associations for the sentinel variant to protein associations or genomic regions or proteins is provided in Supplementary Table 2 [this file] . \n",
      "Of 1,746 aptamer-region pairs pQTLs (involving 1,053 proteins) identified in AA, 1,113 (64% overall, 67% cis, 62% trans) were shared by EA, and 597 (involving 578 proteins, 34% overall, 31% cis, 36% trans) were distinct in AA (see Methods [rid=S13] for definition of shared pQTLs), ( Supplementary Tables 3 [rid=SD2] and 9 [this file] ). Of 4,315 aptamer-region pairs pQTLs identified in EA, in addition to the 1,113 shared pQTLs with AA (26% overall, 41% cis, 21% trans), and 2,481 (involving 2,168 proteins, 57% overall, 45% cis, 62% trans) were distinct in EA (see Methods [rid=S13] ), ( Supplementary Tables 4 [rid=SD2] and 10 [this file] ). \n",
      "Among the conditionally independent AA pQTLs (aptamer-variant pairs), 1,290 (60% overall, 58% cis, 60% trans) were identified in EA at the level of P<0.05/2,160=2.31×10 with the same variant or a variant at strong LD (R2>0.6) with a directionally concordant effect ( Supplementary Tables 5 [rid=SD2] and 11 [this file] ). Among the conditionally independent EA pQTLs (aptamer-variant pairs), 2,306 (39% overall, 44% cis, 36% trans) were identified in AA at the level of P<0.05/5,901=8.47×10 with the same variant or a variant at strong LD (R2>0.6) with a directionally concordant effect ( Supplementary Tables 6 [rid=SD2] and 11 [this file] ). \n",
      "Among 31,639 associated aptamer-region pairs derived from previously reported associations, 4,936 (15% overall, 40% cis, 13% trans) were replicated (P<1.58×10 ) in EA with the same variant or a variant at strong LD (R2>0.6); 1,493 (5% overall, 20% cis, 3% trans) were replicated (P<1.58×10 ) in AA; and 1,337 (4% overall, 17% cis, 3% trans) were replicated in both EA and AA ( Supplementary Table 12 [this file] ). \n",
      "The identified pQTLs were strongly enriched for variants in the coding regions (OR>2.5) and for locations in 3’ untranslated regions (UTR) (OR>2) ( Supplementary Fig. 1 [this file] ). We also found relatively strong (OR>1.5, P<8.5×10 ) enrichment of pQTLs at features indicative of transcriptional activation in blood cells and at hepatocyte (chief functional cell of the liver) regulatory elements, as expected based on the role of the liver in protein synthesis and secretion ( Supplementary Fig. 1 [this file] ). The identified pQTLs in EA were also strongly enriched for 5’ UTR (OR>2.5), in addition to variants in the coding regions (OR>4) and for locations in 3’ UTR ( Supplementary Fig. 1 [this file] ). \n",
      "An example of horizontal pleiotropy was rs3733402, a missense variant in the gene kallikrein B1 ( ) on chromosome 4 (4q35.2), which showed significant association at the level of P<1.025×10 with 50 proteins in AA and 265 proteins in EA ( Supplementary Fig. 2 [this file] ). The SNP rs3733402 has been reported associated with various traits by GWASs including vascular endothelial growth factor D levels,( 22 [rid=R22] ) venous thromboembolism,( 23 [rid=R23] ) B-type natriuretic peptide levels,( 25 [rid=R25] ) serum metabolite levels,( 19 [rid=R19] ) and obesity-related traits.( 26 [rid=R26] ) An example of vertical pleiotropy in AA was a 5’ UTR variant (rs5471) located in gene that also mapped to the gene on chromosome 16 (16q22.2) was significantly associated with six proteins ( Supplementary Fig. 2 [this file] ) and five out of these six proteins were related to the GO term “receptor-mediated endocytosis”. \n",
      "Supplementary Table 17 [this file] lists the 14 pairs of pQTLs and eQTLs found to highly likely colocalize in both EA and AA. \n",
      "A total of 371 signals highly likely to colocalize were identified (8% in cis), involving 191 proteins and 65 causal variants, among which 138 (31%) were shared across EA and AA ( Supplementary Tables 18 [rid=SD2] and 19 [this file] ). For HF, we identified 124 colocalized proteins in 10 meta-regions on eight chromosomes and there were 36 candidate causal variants ( Supplementary Fig. 3 [this file] ). \n",
      "At a Bonferroni-corrected threshold of P<3.62×10 , we identified 565 protein-phenotype associations with MR evidence ( Supplementary Table 20 [rid=SD2] , Supplementary Fig. 4 [this file] , involving 252 aptamers (representing 245 targeted proteins) and 63 phenotypes (18 diseases and 48 intermediate traits). \n",
      "We identified five AA-specific trans-pQTLs ( Supplementary Table 22 [this file] ) that did not overlap with the region of EA pQTLs with the same protein. \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref SD2: After conditional analysis, we identified 1,746 significant aptamer-region pairs associations (37% cis) among 807 regions and 1,408 proteins in AA ( Supplementary Tables 2 [rid=SD1] and 3 [this file] , Figure 2 [rid=F2] ) and 4,315 significant associations (24% cis) among 1,490 regions and 2,565 proteins in EA ( Supplementary Tables 2 [rid=SD1] and 4 [this file] , Figure 2 [rid=F2] ). Conditional analyses yielded 2,160 pairs of conditionally independent aptamer-variants associations in AA and 5,901 in EA ( Supplementary Tables 5 [this file] and 6 [this file] ). \n",
      "In both EA and AA, the P-values of the cis-acting associations increased with distance of the sentinel variant to the TSS of the coding gene ( Figure 3 [rid=F3] ), mirroring previous pQTLs and eQTLs findings. ( 2 [rid=R2] , 15 [rid=R15] ) The majority of significantly associated proteins (AA: 80%, n=1,114 and EA: 56%, n=1,414) had a single pQTL, while 18% in AA (27% in EA) had two, and 2% in AA (17% in EA) had 3–6 respectively ( Figure 3 [rid=F3] ).We calculated the proportions of variance explained by the conditionally independent associated genetic variants for each of the 1,441 (AA) or 2,648 (EA) aptamers using GCTA ( Methods [rid=S13] , Supplementary Tables 7 [this file] and 8 [this file] , Figure 3 [rid=F3] ). \n",
      "Of 1,746 aptamer-region pairs pQTLs (involving 1,053 proteins) identified in AA, 1,113 (64% overall, 67% cis, 62% trans) were shared by EA, and 597 (involving 578 proteins, 34% overall, 31% cis, 36% trans) were distinct in AA (see Methods [rid=S13] for definition of shared pQTLs), ( Supplementary Tables 3 [this file] and 9 [rid=SD1] ). Of 4,315 aptamer-region pairs pQTLs identified in EA, in addition to the 1,113 shared pQTLs with AA (26% overall, 41% cis, 21% trans), and 2,481 (involving 2,168 proteins, 57% overall, 45% cis, 62% trans) were distinct in EA (see Methods [rid=S13] ), ( Supplementary Tables 4 [this file] and 10 [rid=SD1] ). \n",
      "Among the conditionally independent AA pQTLs (aptamer-variant pairs), 1,290 (60% overall, 58% cis, 60% trans) were identified in EA at the level of P<0.05/2,160=2.31×10 with the same variant or a variant at strong LD (R2>0.6) with a directionally concordant effect ( Supplementary Tables 5 [this file] and 11 [rid=SD1] ). Among the conditionally independent EA pQTLs (aptamer-variant pairs), 2,306 (39% overall, 44% cis, 36% trans) were identified in AA at the level of P<0.05/5,901=8.47×10 with the same variant or a variant at strong LD (R2>0.6) with a directionally concordant effect ( Supplementary Tables 6 [this file] and 11 [rid=SD1] ). \n",
      "Of 1,746 aptamer-region pairs identified in AA, 808 (46% overall, 52% cis, 43% trans) were previously identified ( Supplementary Table 3 [this file] ). For EA, of 4,315 aptamer-region pairs, 2,782 (64% overall, 75% cis, 61% trans) were previously reported ( Supplementary Table 4 [this file] ). \n",
      "In AA, among these 1,005 annotation features, 529 (53%) were found significantly (P<8.5×10 ) enriched (OR>1) and 55 (5%) depleted (OR<1) ( Supplementary Table 13 [this file] ), with the most enriched feature being genic annotation: exon. Similar findings were observed in EA, but with much more significantly enriched annotation features: 922 (99%) out of the 1,005 annotation features were found significantly (P<9.5×10 ) enriched (OR>1) and seven (<1%) depleted (OR<1) ( Supplementary Table 14 [this file] ), with the most enriched feature being also genic annotation: exon. \n",
      "Among the conditionally independent variants identified in AA (EA), 816 (1,576) were specific for a single protein, 106 (394) showed evidence of vertical pleiotropy, and 368 (1,266) showed evidence of horizontal pleiotropy ( Supplementary Tables 15 [this file] and 16 [this file] ). \n",
      "Of the 111 overlapped pairs in EA, 64 (58%) had a highly (PP>0.5) or most highly likely (PP>0.8) colocalization, and thus shared a common causal variant ( Supplementary Table 4 [this file] ). For AA, 25 out of the 60 (42%) overlapped pairs of pQTL and eQTL, colocalization was highly likely or most highly likely ( Supplementary Tables 3 [this file] ). \n",
      "A total of 371 signals highly likely to colocalize were identified (8% in cis), involving 191 proteins and 65 causal variants, among which 138 (31%) were shared across EA and AA ( Supplementary Tables 18 [this file] and 19 [rid=SD1] ). \n",
      "In total, there were 16 unique protein-trait of CVD type (HF or AF) pairs for which we identified colocalization signals for both EA and AA with the same candidate causal variants ( Supplementary Table 18 [this file] ). We also observed shared colocalization signals with sleep between EA and AA with the same candidate causal variants for 18 proteins ( Supplementary Table 18 [this file] ). \n",
      "At a Bonferroni-corrected threshold of P<3.62×10 , we identified 565 protein-phenotype associations with MR evidence ( Supplementary Table 20 [this file] , Supplementary Fig. 4 [rid=SD1] , involving 252 aptamers (representing 245 targeted proteins) and 63 phenotypes (18 diseases and 48 intermediate traits). In the rest of the 507 new associations, 19 represented a candidate causal effect of 14 proteins on 14 diseases ( Supplementary Table 21 [this file] ). Among these 19 MR-based associations, we found that seven have been reported by disease-specific proteomic MR study such as MOTI vs. type 2 diabetes, ( 43 [rid=R43] ) TFF3 vs. rheumatoid arthritis, ( 44 [rid=R44] ) or PYY vs colorectal cancer ( 45 [rid=R45] ) ( Supplementary Table 21 [this file] ). \n",
      "We identified 640 EA-specific pQTLs ( Supplementary Table 23 [this file] ) that did not overlap with the region of AA pQTLs with the same protein or any AA-pQTLs reported in Katz et al. At least for some of ancestry-specific pQTLs, the difference in the association of the protein level and the genomic locus between ancestries may reflect a complex interplay between a pQTL and other genetic variants, which has been previously observed in the gene expression analysis,( 47 [rid=R47] ) and/or environmental factors that could be investigated in future studies. \n",
      "As most (if not all) of these summary statistics were drawn from studies for populations of European ancestry, we selected exposures and instrumental variables (IVs) for MR analysis from the list of conditionally independent associated pairs of aptamer and variants identified in EA ( Supplementary Table 7 [this file] ). \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 263 - INFO - Extracting links from full XML content.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 301 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 309 - INFO - Content length: 174114\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 326 - INFO - Prompt messages total length: 44282 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 332 - INFO - Prompt ID: gemini-2.5-flash-0-28ed35605005e0dcb252f86ce79d424ee106ceed03c64deabe6036db97f085d1\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 351 - INFO - Requesting datasets from content using model: gemini-2.5-flash, temperature: 0, messages length: 44282 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-ea2ebec8-4a4a-4453-aa54-38bacc8c555f\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"phs000280.v8.p2\\\",\\n    \\\"repository_reference\\\": \\\"dbGaP\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://biolincc.nhlbi.nih.gov/studies/aric/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn63141185\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/Synapse:syn63141185\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"phs000280.v8.p2\\\",\\n    \\\"repository_reference\\\": \\\"dbGaP\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://biolincc.nhlbi.nih.gov/studies/aric/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn63141185\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/Synapse:syn63141185\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1758210500,\n",
      "    \"model\": \"gemini-2.5-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 56615,\n",
      "        \"completion_tokens\": 146,\n",
      "        \"total_tokens\": 58585,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 1824,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 664 - INFO - Deduplicating response with 3 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: phs000280.v8.p2\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn63141185\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'phs000280.v8.p2', 'repository_reference': 'dbGaP', 'dataset_id': 'phs000280.v8.p2'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ to value: phs000280.v8.p2\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: phs000280.v8.p2\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: dbgap\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: dbGaP, identifier: phs000280.v8.p2, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier phs000280.v8.p2 for dbGaP\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier phs000280.v8.p2 for dbGaP\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: dbGaP\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ to value: phs000280.v8.p2\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: phs000280.v8.p2, dbgap, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: phs000280.v8.p2\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: dbgap\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'phs000280.v8.p2', 'data_repository': 'dbgap', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://biolincc.nhlbi.nih.gov/studies/aric/', 'dataset_id': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://biolincc.nhlbi.nih.gov/studies/aric/, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://biolincc.nhlbi.nih.gov/studies/aric/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://biolincc.nhlbi.nih.gov/studies/aric/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'biolincc.nhlbi.nih.gov', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn63141185', 'repository_reference': 'https://www.synapse.org/Synapse:syn63141185', 'dataset_id': 'syn63141185'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn63141185\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn63141185\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/Synapse:syn63141185, identifier: syn63141185, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/Synapse:syn63141185\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn63141185 for https://www.synapse.org/Synapse:syn63141185\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn63141185 for https://www.synapse.org/Synapse:syn63141185\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn63141185 for https://www.synapse.org/Synapse:syn63141185\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/Synapse:syn63141185 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/Synapse:syn63141185\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/Synapse:syn63141185\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/Synapse:syn63141185\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn63141185\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn63141185\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn63141185\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/Synapse:syn63141185\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn63141185\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn63141185\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn63141185, synapse.org, https://www.synapse.org/Synapse:syn63141185\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn63141185\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/Synapse:syn63141185\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn63141185', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/Synapse:syn63141185', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 278 - INFO - Augmented dataset links: [{'dataset_identifier': 'phs000280.v8.p2', 'data_repository': 'dbgap', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'biolincc.nhlbi.nih.gov', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn63141185', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/Synapse:syn63141185', 'citation_type': 'n/a'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1191 - INFO - Fetching metadata for 3 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: phs000280.v8.p2\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: dbgap, identifier: phs000280.v8.p2, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier phs000280.v8.p2 for dbgap\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier phs000280.v8.p2 for dbgap\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: dbgap\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for phs000280.v8.p2 in dbgap\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 1 with repo: dbgap and accession_id: phs000280.v8.p2.v1.p1\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000280.v8.p2.v1.p1\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 1: Application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 2: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn63141185\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn63141185, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn63141185 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn63141185 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn63141185 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn63141185 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 3 with repo: synapse.org and accession_id: syn63141185\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn63141185\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 3: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1277 - INFO - Updated datasets len: 3\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 287 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 5, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'source_url', 'download_link',\n",
      "       'title', 'content_type', 'id', 'caption', 'description',\n",
      "       'source_section', 'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 500 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 506 - INFO - XML element classification not needed. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1012 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11469718\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1020 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11469718\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 647th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11472622)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11472622\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11472622 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 634 - INFO - Fetching data for pmc11472622\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 637 - INFO - Fetching data from scripts/exp_input/Local_fetched_data_1.parquet\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 438 - INFO - Raw html data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11472622 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 452 - INFO - Skipping raw HTML/XML saving.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 482 - INFO - Using HTMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 52 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 82 - INFO - Initializing htmlRetriever\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 350 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11472622, additional_data, html)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 448 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC11472622)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 550 - INFO - Extracted 18 unique supplementary material links from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 560 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#sec1.2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for sec1.2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for undtbl1: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for undtbl1: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for undtbl1: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for undtbl1: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for undtbl1: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for undtbl1: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for undtbl1: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for undtbl1: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for undtbl1: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for undtbl1: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for undtbl1: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for undtbl1: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for undtbl1: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for undtbl1: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#undtbl1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for undtbl1: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#cebib0010_sec2'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for cebib0010_sec2: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 368 - INFO - Extracting links from full HTML content.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 301 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 309 - INFO - Content length: 137289\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 326 - INFO - Prompt messages total length: 35076 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 332 - INFO - Prompt ID: gemini-2.5-flash-0-3926ccbde37f6b8ca1ebab55b1d04473b552fac40e2f4b87cd39e11fe82272d9\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 351 - INFO - Requesting datasets from content using model: gemini-2.5-flash, temperature: 0, messages length: 35076 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-7eed81e4-82ff-4067-82af-63a6de333724\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn18485175\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn18485175\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn18485175\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn18485175\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1758210515,\n",
      "    \"model\": \"gemini-2.5-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 48626,\n",
      "        \"completion_tokens\": 63,\n",
      "        \"total_tokens\": 50822,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 2133,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 664 - INFO - Deduplicating response with 1 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn18485175\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn18485175', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn18485175', 'dataset_id': 'syn18485175'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn18485175\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn18485175\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn18485175, identifier: syn18485175, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn18485175\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn18485175 for https://www.synapse.org/#!Synapse:syn18485175\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn18485175 for https://www.synapse.org/#!Synapse:syn18485175\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn18485175 for https://www.synapse.org/#!Synapse:syn18485175\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn18485175 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn18485175\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn18485175\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn18485175\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn18485175\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn18485175\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn18485175\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn18485175\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn18485175\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn18485175\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn18485175, synapse.org, https://www.synapse.org/#!Synapse:syn18485175\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn18485175\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn18485175\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn18485175', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn18485175', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 378 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn18485175', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn18485175', 'citation_type': 'n/a'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1191 - INFO - Fetching metadata for 1 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn18485175\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn18485175, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn18485175 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn18485175 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn18485175 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn18485175 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn18485175\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn18485175\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1277 - INFO - Updated datasets len: 1\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 414 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 19, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'title', 'file_info',\n",
      "       'description', 'source_section', 'section_class', 'download_link',\n",
      "       'file_extension', 'a_attr_href', 'a_attr_class',\n",
      "       'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 173 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 183 - WARNING - No publication title found in the HTML data.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 643 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 494 - INFO - Parsed data extraction completed. Elements collected: 2\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 500 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 510 - INFO - HTML element classification not supported. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1012 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11472622\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1020 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11472622\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 648th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11476461)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11476461\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11476461 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 634 - INFO - Fetching data for pmc11476461\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 637 - INFO - Fetching data from scripts/exp_input/Local_fetched_data_1.parquet\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 438 - INFO - Raw xml data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11476461 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 452 - INFO - Skipping raw HTML/XML saving.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 458 - INFO - Using XMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 52 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 24 - INFO - Initializing xmlRetriever\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 466 - INFO - No additional data provided. Parsing raw data only.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 147 - INFO - Function call: parse_data(api_data(<class 'str'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11476461, additional_data, XML)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 162 - INFO - api_data converted to lxml element\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 466 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 153 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 154 - INFO - Consider migrating this function to the BaseRetriever class.\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 69 - INFO - Loading target sections for section name: supplementary_material_sections\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 477 - INFO - Found 1 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x15e4bc880>]\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 485 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x15e4bc880>]}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 574 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref ijms-25-10822-s001: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 263 - INFO - Extracting links from full XML content.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 301 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 309 - INFO - Content length: 147732\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 326 - INFO - Prompt messages total length: 37686 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 332 - INFO - Prompt ID: gemini-2.5-flash-0-46dfcfe301a30a6fcf8f6ae345745bdc581cde424d72b2657d2e10250f88fe70\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 351 - INFO - Requesting datasets from content using model: gemini-2.5-flash, temperature: 0, messages length: 37686 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-1e14c006-cca3-4a80-be42-1d81538f97d9\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn63432900\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn63432900\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"TCGA-SARC\\\",\\n    \\\"repository_reference\\\": \\\"https://portal.gdc.cancer.gov/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"TCGA-LUAD\\\",\\n    \\\"repository_reference\\\": \\\"https://portal.gdc.cancer.gov/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"TCGA-LUSC\\\",\\n    \\\"repository_reference\\\": \\\"https://portal.gdc.cancer.gov/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"TCGA-SKCM\\\",\\n    \\\"repository_reference\\\": \\\"https://portal.gdc.cancer.gov/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.cbioportal.org/\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn63432900\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn63432900\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"TCGA-SARC\\\",\\n    \\\"repository_reference\\\": \\\"https://portal.gdc.cancer.gov/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"TCGA-LUAD\\\",\\n    \\\"repository_reference\\\": \\\"https://portal.gdc.cancer.gov/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"TCGA-LUSC\\\",\\n    \\\"repository_reference\\\": \\\"https://portal.gdc.cancer.gov/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"TCGA-SKCM\\\",\\n    \\\"repository_reference\\\": \\\"https://portal.gdc.cancer.gov/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.cbioportal.org/\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1758210531,\n",
      "    \"model\": \"gemini-2.5-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 47448,\n",
      "        \"completion_tokens\": 255,\n",
      "        \"total_tokens\": 50046,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 2343,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 664 - INFO - Deduplicating response with 6 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn63432900\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: TCGA-SARC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: TCGA-LUAD\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: TCGA-LUSC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: TCGA-SKCM\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn63432900', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn63432900', 'dataset_id': 'syn63432900'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn63432900\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn63432900\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn63432900, identifier: syn63432900, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn63432900\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn63432900 for https://www.synapse.org/#!Synapse:syn63432900\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn63432900 for https://www.synapse.org/#!Synapse:syn63432900\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn63432900 for https://www.synapse.org/#!Synapse:syn63432900\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn63432900 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn63432900\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn63432900\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn63432900\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn63432900\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn63432900\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn63432900\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn63432900\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn63432900\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn63432900\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn63432900, synapse.org, https://www.synapse.org/#!Synapse:syn63432900\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn63432900\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn63432900\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn63432900', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn63432900', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'TCGA-SARC', 'repository_reference': 'https://portal.gdc.cancer.gov/', 'dataset_id': 'TCGA-SARC'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: TCGA-SARC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID TCGA-SARC is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://portal.gdc.cancer.gov/, identifier: TCGA-SARC, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://portal.gdc.cancer.gov/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier TCGA-SARC for https://portal.gdc.cancer.gov/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier TCGA-SARC for https://portal.gdc.cancer.gov/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier TCGA-SARC for https://portal.gdc.cancer.gov/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier TCGA-SARC for https://portal.gdc.cancer.gov/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier TCGA-SARC for https://portal.gdc.cancer.gov/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier TCGA-SARC for https://portal.gdc.cancer.gov/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://portal.gdc.cancer.gov/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'TCGA-SARC', 'data_repository': 'portal.gdc.cancer.gov', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'TCGA-LUAD', 'repository_reference': 'https://portal.gdc.cancer.gov/', 'dataset_id': 'TCGA-LUAD'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: TCGA-LUAD\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID TCGA-LUAD is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://portal.gdc.cancer.gov/, identifier: TCGA-LUAD, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://portal.gdc.cancer.gov/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier TCGA-LUAD for https://portal.gdc.cancer.gov/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier TCGA-LUAD for https://portal.gdc.cancer.gov/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier TCGA-LUAD for https://portal.gdc.cancer.gov/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier TCGA-LUAD for https://portal.gdc.cancer.gov/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier TCGA-LUAD for https://portal.gdc.cancer.gov/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier TCGA-LUAD for https://portal.gdc.cancer.gov/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://portal.gdc.cancer.gov/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'TCGA-LUAD', 'data_repository': 'portal.gdc.cancer.gov', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'TCGA-LUSC', 'repository_reference': 'https://portal.gdc.cancer.gov/', 'dataset_id': 'TCGA-LUSC'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: TCGA-LUSC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID TCGA-LUSC is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://portal.gdc.cancer.gov/, identifier: TCGA-LUSC, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://portal.gdc.cancer.gov/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier TCGA-LUSC for https://portal.gdc.cancer.gov/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier TCGA-LUSC for https://portal.gdc.cancer.gov/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier TCGA-LUSC for https://portal.gdc.cancer.gov/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier TCGA-LUSC for https://portal.gdc.cancer.gov/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier TCGA-LUSC for https://portal.gdc.cancer.gov/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier TCGA-LUSC for https://portal.gdc.cancer.gov/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://portal.gdc.cancer.gov/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'TCGA-LUSC', 'data_repository': 'portal.gdc.cancer.gov', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'TCGA-SKCM', 'repository_reference': 'https://portal.gdc.cancer.gov/', 'dataset_id': 'TCGA-SKCM'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: TCGA-SKCM\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID TCGA-SKCM is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://portal.gdc.cancer.gov/, identifier: TCGA-SKCM, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://portal.gdc.cancer.gov/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier TCGA-SKCM for https://portal.gdc.cancer.gov/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier TCGA-SKCM for https://portal.gdc.cancer.gov/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier TCGA-SKCM for https://portal.gdc.cancer.gov/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier TCGA-SKCM for https://portal.gdc.cancer.gov/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier TCGA-SKCM for https://portal.gdc.cancer.gov/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier TCGA-SKCM for https://portal.gdc.cancer.gov/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://portal.gdc.cancer.gov/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'TCGA-SKCM', 'data_repository': 'portal.gdc.cancer.gov', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.cbioportal.org/', 'dataset_id': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.cbioportal.org/, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.cbioportal.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://www.cbioportal.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'www.cbioportal.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 278 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn63432900', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn63432900', 'citation_type': 'n/a'}, {'dataset_identifier': 'TCGA-SARC', 'data_repository': 'portal.gdc.cancer.gov', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'TCGA-LUAD', 'data_repository': 'portal.gdc.cancer.gov', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'TCGA-LUSC', 'data_repository': 'portal.gdc.cancer.gov', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'TCGA-SKCM', 'data_repository': 'portal.gdc.cancer.gov', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'www.cbioportal.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1191 - INFO - Fetching metadata for 6 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn63432900\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn63432900, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn63432900 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn63432900 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn63432900 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn63432900 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn63432900\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn63432900\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: TCGA-SARC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: portal.gdc.cancer.gov, identifier: TCGA-SARC, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier TCGA-SARC for portal.gdc.cancer.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier TCGA-SARC for portal.gdc.cancer.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier TCGA-SARC for portal.gdc.cancer.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier TCGA-SARC for portal.gdc.cancer.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier TCGA-SARC for portal.gdc.cancer.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier TCGA-SARC for portal.gdc.cancer.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: portal.gdc.cancer.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL portal.gdc.cancer.gov may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for TCGA-SARC in portal.gdc.cancer.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 2 with repo: portal.gdc.cancer.gov and accession_id: TCGA-SARC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository portal.gdc.cancer.gov unknown in Ontology. Skipping dataset page 2.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: TCGA-LUAD\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: portal.gdc.cancer.gov, identifier: TCGA-LUAD, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier TCGA-LUAD for portal.gdc.cancer.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier TCGA-LUAD for portal.gdc.cancer.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier TCGA-LUAD for portal.gdc.cancer.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier TCGA-LUAD for portal.gdc.cancer.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier TCGA-LUAD for portal.gdc.cancer.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier TCGA-LUAD for portal.gdc.cancer.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: portal.gdc.cancer.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL portal.gdc.cancer.gov may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for TCGA-LUAD in portal.gdc.cancer.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 3 with repo: portal.gdc.cancer.gov and accession_id: TCGA-LUAD\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository portal.gdc.cancer.gov unknown in Ontology. Skipping dataset page 3.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: TCGA-LUSC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: portal.gdc.cancer.gov, identifier: TCGA-LUSC, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier TCGA-LUSC for portal.gdc.cancer.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier TCGA-LUSC for portal.gdc.cancer.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier TCGA-LUSC for portal.gdc.cancer.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier TCGA-LUSC for portal.gdc.cancer.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier TCGA-LUSC for portal.gdc.cancer.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier TCGA-LUSC for portal.gdc.cancer.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: portal.gdc.cancer.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL portal.gdc.cancer.gov may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for TCGA-LUSC in portal.gdc.cancer.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 4 with repo: portal.gdc.cancer.gov and accession_id: TCGA-LUSC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository portal.gdc.cancer.gov unknown in Ontology. Skipping dataset page 4.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: TCGA-SKCM\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: portal.gdc.cancer.gov, identifier: TCGA-SKCM, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier TCGA-SKCM for portal.gdc.cancer.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier TCGA-SKCM for portal.gdc.cancer.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier TCGA-SKCM for portal.gdc.cancer.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier TCGA-SKCM for portal.gdc.cancer.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier TCGA-SKCM for portal.gdc.cancer.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier TCGA-SKCM for portal.gdc.cancer.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: portal.gdc.cancer.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL portal.gdc.cancer.gov may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for TCGA-SKCM in portal.gdc.cancer.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 5 with repo: portal.gdc.cancer.gov and accession_id: TCGA-SKCM\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository portal.gdc.cancer.gov unknown in Ontology. Skipping dataset page 5.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 6: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1277 - INFO - Updated datasets len: 6\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 287 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 7, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'source_url', 'download_link',\n",
      "       'title', 'content_type', 'id', 'caption', 'description',\n",
      "       'source_section', 'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 500 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 506 - INFO - XML element classification not needed. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1012 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11476461\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1020 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11476461\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 649th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11476968)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11476968\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11476968 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 634 - INFO - Fetching data for pmc11476968\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 637 - INFO - Fetching data from scripts/exp_input/Local_fetched_data_1.parquet\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 438 - INFO - Raw xml data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11476968 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 452 - INFO - Skipping raw HTML/XML saving.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 458 - INFO - Using XMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 52 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 24 - INFO - Initializing xmlRetriever\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 466 - INFO - No additional data provided. Parsing raw data only.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 147 - INFO - Function call: parse_data(api_data(<class 'str'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11476968, additional_data, XML)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 162 - INFO - api_data converted to lxml element\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 466 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 153 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 154 - INFO - Consider migrating this function to the BaseRetriever class.\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 69 - INFO - Loading target sections for section name: supplementary_material_sections\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 477 - INFO - Found 8 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x15f7b08c0>, <Element supplementary-material at 0x15f891c40>, <Element supplementary-material at 0x15f8916c0>, <Element supplementary-material at 0x15f891780>, <Element supplementary-material at 0x15f891a40>, <Element supplementary-material at 0x15f891e00>, <Element supplementary-material at 0x15f891e40>, <Element supplementary-material at 0x15f891cc0>]\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 485 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x15f7b08c0>, <Element supplementary-material at 0x15f891c40>, <Element supplementary-material at 0x15f8916c0>, <Element supplementary-material at 0x15f891780>, <Element supplementary-material at 0x15f891a40>, <Element supplementary-material at 0x15f891e00>, <Element supplementary-material at 0x15f891e40>, <Element supplementary-material at 0x15f891cc0>]}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 574 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM1: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM2: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM3: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM4: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM5: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM6: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM7: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM8: The review history is available as Additional file 8 [this file] . \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 263 - INFO - Extracting links from full XML content.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 301 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 309 - INFO - Content length: 347514\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 326 - INFO - Prompt messages total length: 87632 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 332 - INFO - Prompt ID: gemini-2.5-flash-0-31099a5b57447ecc7a0f38e244ae64a02afcf0fa6dea30581cb6e4036ea36193\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 351 - INFO - Requesting datasets from content using model: gemini-2.5-flash, temperature: 0, messages length: 87632 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-a9eedc11-4f00-42a7-912e-e6f5b59b06c7\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"phs001503\\\",\\n    \\\"repository_reference\\\": \\\"dbGaP\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.5281/zenodo.3629742\\\",\\n    \\\"repository_reference\\\": \\\"https://doi.org/10.5281/zenodo.3629742\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn23446022\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn23446022\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.wtccc.org.uk\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"phs001503\\\",\\n    \\\"repository_reference\\\": \\\"dbGaP\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.5281/zenodo.3629742\\\",\\n    \\\"repository_reference\\\": \\\"https://doi.org/10.5281/zenodo.3629742\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn23446022\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn23446022\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.wtccc.org.uk\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1758210557,\n",
      "    \"model\": \"gemini-2.5-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 121083,\n",
      "        \"completion_tokens\": 200,\n",
      "        \"total_tokens\": 124633,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 3350,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 664 - INFO - Deduplicating response with 4 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: phs001503\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: 10.5281/zenodo.3629742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn23446022\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'phs001503', 'repository_reference': 'dbGaP', 'dataset_id': 'phs001503'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: phs001503\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: phs001503\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: dbGaP, identifier: phs001503, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier phs001503 for dbGaP\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier phs001503 for dbGaP\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: dbGaP\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: phs001503\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: phs001503, dbgap, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: phs001503\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: dbgap\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'phs001503', 'data_repository': 'dbgap', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': '10.5281/zenodo.3629742', 'repository_reference': 'https://doi.org/10.5281/zenodo.3629742', 'dataset_id': '10.5281/zenodo.3629742'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: 10\\.\\d{4,9}/[^\\s,]+ to value: 10.5281/zenodo.3629742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 607 - INFO - Setting dataset_webpage to DOI URL value: https://doi.org/10.5281/zenodo.3629742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: 10.5281/zenodo.3629742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: doi.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://doi.org/10.5281/zenodo.3629742, identifier: 10.5281/zenodo.3629742, dataset_page: https://doi.org/10.5281/zenodo.3629742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://doi.org/10.5281/zenodo.3629742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.5281/zenodo.3629742 for https://doi.org/10.5281/zenodo.3629742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier 10.5281/zenodo.3629742 for https://doi.org/10.5281/zenodo.3629742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier 10.5281/zenodo.3629742 for https://doi.org/10.5281/zenodo.3629742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier 10.5281/zenodo.3629742 for https://doi.org/10.5281/zenodo.3629742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier 10.5281/zenodo.3629742 for https://doi.org/10.5281/zenodo.3629742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1157 - INFO - Found id_pattern match for https://doi.org/10.5281/zenodo.3629742 in 10\\.\\d{4,9}/[^\\s,]+\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://doi.org/__ID__ to value: https://doi.org/10.5281/zenodo.3629742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: 10\\.\\d{4,9}/[^\\s,]+ to value: https://doi.org/10.5281/zenodo.3629742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(10\\.\\d{4,9}/[^\\s,]+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(10\\.\\d{4,9}/[^\\s,]+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: 10.5281/zenodo.3629742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: 10\\.\\d{4,9}/[^\\s,]+ to value: 10.5281/zenodo.3629742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: 10.5281/zenodo.3629742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://doi.org/10.5281/zenodo.3629742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: 10.5281/zenodo.3629742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: 10.5281/zenodo.3629742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: 10.5281/zenodo.3629742, doi.org, https://doi.org/10.5281/zenodo.3629742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: 10.5281/zenodo.3629742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: doi.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://doi.org/10.5281/zenodo.3629742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': '10.5281/zenodo.3629742', 'data_repository': 'doi.org', 'dataset_webpage': 'https://doi.org/10.5281/zenodo.3629742', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn23446022', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn23446022', 'dataset_id': 'syn23446022'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn23446022\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn23446022\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn23446022, identifier: syn23446022, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn23446022\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn23446022 for https://www.synapse.org/#!Synapse:syn23446022\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn23446022 for https://www.synapse.org/#!Synapse:syn23446022\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn23446022 for https://www.synapse.org/#!Synapse:syn23446022\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn23446022 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn23446022\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn23446022\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn23446022\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn23446022\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn23446022\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn23446022\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn23446022\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn23446022\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn23446022\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn23446022, synapse.org, https://www.synapse.org/#!Synapse:syn23446022\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn23446022\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn23446022\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn23446022', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn23446022', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.wtccc.org.uk', 'dataset_id': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.wtccc.org.uk, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.wtccc.org.uk\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://www.wtccc.org.uk\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'www.wtccc.org.uk', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 278 - INFO - Augmented dataset links: [{'dataset_identifier': 'phs001503', 'data_repository': 'dbgap', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': '10.5281/zenodo.3629742', 'data_repository': 'doi.org', 'dataset_webpage': 'https://doi.org/10.5281/zenodo.3629742', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn23446022', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn23446022', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'www.wtccc.org.uk', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1191 - INFO - Fetching metadata for 4 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: phs001503\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: dbgap, identifier: phs001503, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier phs001503 for dbgap\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier phs001503 for dbgap\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: dbgap\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for phs001503 in dbgap\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1085 - WARNING - Identifier phs001503 does not match pattern for dbgap\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 1 with repo: dbgap and accession_id: phs001503.v1.p1\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001503.v1.p1\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 1: Application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: 10.5281/zenodo.3629742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: doi.org, identifier: 10.5281/zenodo.3629742, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.5281/zenodo.3629742 for doi.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier 10.5281/zenodo.3629742 for doi.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier 10.5281/zenodo.3629742 for doi.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier 10.5281/zenodo.3629742 for doi.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: doi.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for 10.5281/zenodo.3629742 in doi.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 2 with repo: doi.org and accession_id: 10.5281/zenodo.3629742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://doi.org/10.5281/zenodo.3629742\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn23446022\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn23446022, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn23446022 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn23446022 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn23446022 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn23446022 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 3 with repo: synapse.org and accession_id: syn23446022\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn23446022\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 3: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 4: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1277 - INFO - Updated datasets len: 4\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 287 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 12, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'source_url', 'download_link',\n",
      "       'title', 'content_type', 'id', 'caption', 'description',\n",
      "       'source_section', 'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 500 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 506 - INFO - XML element classification not needed. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1012 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11476968\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1020 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11476968\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 650th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11476976)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11476976\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11476976 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 634 - INFO - Fetching data for pmc11476976\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 637 - INFO - Fetching data from scripts/exp_input/Local_fetched_data_1.parquet\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 438 - INFO - Raw xml data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11476976 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 452 - INFO - Skipping raw HTML/XML saving.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 458 - INFO - Using XMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 52 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 24 - INFO - Initializing xmlRetriever\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 466 - INFO - No additional data provided. Parsing raw data only.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 147 - INFO - Function call: parse_data(api_data(<class 'str'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11476976, additional_data, XML)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 162 - INFO - api_data converted to lxml element\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 466 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 153 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 154 - INFO - Consider migrating this function to the BaseRetriever class.\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 69 - INFO - Loading target sections for section name: supplementary_material_sections\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 477 - INFO - Found 1 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x15d497e00>]\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 485 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x15d497e00>]}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 574 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref ijms-25-10236-s001: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 263 - INFO - Extracting links from full XML content.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 301 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 309 - INFO - Content length: 169873\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 326 - INFO - Prompt messages total length: 43222 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 332 - INFO - Prompt ID: gemini-2.5-flash-0-cb3d2f48dd76161b138c1d93a54ea4a0c59c66038ed30fc20ccd93a1a13b1083\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 351 - INFO - Requesting datasets from content using model: gemini-2.5-flash, temperature: 0, messages length: 43222 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-f1ba0737-ab11-420f-adca-91d15b398e10\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"ukb-b-6353\\\",\\n    \\\"repository_reference\\\": \\\"https://gwas.mrcieu.ac.uk/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.finngen.fi/en/access_results\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"http://nilanjanchatterjeelab.org/pwas/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn23191787\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn23191787\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"ukb-b-6353\\\",\\n    \\\"repository_reference\\\": \\\"https://gwas.mrcieu.ac.uk/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.finngen.fi/en/access_results\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"http://nilanjanchatterjeelab.org/pwas/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn23191787\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn23191787\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1758210576,\n",
      "    \"model\": \"gemini-2.5-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 55069,\n",
      "        \"completion_tokens\": 189,\n",
      "        \"total_tokens\": 57980,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 2722,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 664 - INFO - Deduplicating response with 4 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: ukb-b-6353\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn23191787\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'ukb-b-6353', 'repository_reference': 'https://gwas.mrcieu.ac.uk/', 'dataset_id': 'ukb-b-6353'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: ukb-b-6353\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: ukb-b-6353\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://gwas.mrcieu.ac.uk/, identifier: ukb-b-6353, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://gwas.mrcieu.ac.uk/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier ukb-b-6353 for https://gwas.mrcieu.ac.uk/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier ukb-b-6353 for https://gwas.mrcieu.ac.uk/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier ukb-b-6353 for https://gwas.mrcieu.ac.uk/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier ukb-b-6353 for https://gwas.mrcieu.ac.uk/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier ukb-b-6353 for https://gwas.mrcieu.ac.uk/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier ukb-b-6353 for https://gwas.mrcieu.ac.uk/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://gwas.mrcieu.ac.uk/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: ukb-b-6353\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: ukb-b-6353, gwas.mrcieu.ac.uk, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: ukb-b-6353\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: gwas.mrcieu.ac.uk\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'ukb-b-6353', 'data_repository': 'gwas.mrcieu.ac.uk', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.finngen.fi/en/access_results', 'dataset_id': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.finngen.fi/en/access_results, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.finngen.fi/en/access_results\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://www.finngen.fi/en/access_results\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'www.finngen.fi', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'http://nilanjanchatterjeelab.org/pwas/'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: http://nilanjanchatterjeelab.org/pwas/, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: http://nilanjanchatterjeelab.org/pwas/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: http://nilanjanchatterjeelab.org/pwas/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'nilanjanchatterjeelab.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn23191787', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn23191787', 'dataset_id': 'syn23191787'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn23191787\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn23191787\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn23191787, identifier: syn23191787, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn23191787\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn23191787 for https://www.synapse.org/#!Synapse:syn23191787\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn23191787 for https://www.synapse.org/#!Synapse:syn23191787\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn23191787 for https://www.synapse.org/#!Synapse:syn23191787\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn23191787 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn23191787\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn23191787\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn23191787\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn23191787\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn23191787\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn23191787\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn23191787\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn23191787\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn23191787\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn23191787, synapse.org, https://www.synapse.org/#!Synapse:syn23191787\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn23191787\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn23191787\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn23191787', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn23191787', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 278 - INFO - Augmented dataset links: [{'dataset_identifier': 'ukb-b-6353', 'data_repository': 'gwas.mrcieu.ac.uk', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'www.finngen.fi', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'nilanjanchatterjeelab.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn23191787', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn23191787', 'citation_type': 'n/a'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1191 - INFO - Fetching metadata for 4 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: ukb-b-6353\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: gwas.mrcieu.ac.uk, identifier: ukb-b-6353, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier ukb-b-6353 for gwas.mrcieu.ac.uk\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier ukb-b-6353 for gwas.mrcieu.ac.uk\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier ukb-b-6353 for gwas.mrcieu.ac.uk\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier ukb-b-6353 for gwas.mrcieu.ac.uk\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier ukb-b-6353 for gwas.mrcieu.ac.uk\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier ukb-b-6353 for gwas.mrcieu.ac.uk\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: gwas.mrcieu.ac.uk\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL gwas.mrcieu.ac.uk may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for ukb-b-6353 in gwas.mrcieu.ac.uk\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 1 with repo: gwas.mrcieu.ac.uk and accession_id: ukb-b-6353\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository gwas.mrcieu.ac.uk unknown in Ontology. Skipping dataset page 1.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 2: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 3: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn23191787\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn23191787, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn23191787 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn23191787 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn23191787 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn23191787 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 4 with repo: synapse.org and accession_id: syn23191787\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn23191787\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 4: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1277 - INFO - Updated datasets len: 4\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 287 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 5, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'source_url', 'download_link',\n",
      "       'title', 'content_type', 'id', 'caption', 'description',\n",
      "       'source_section', 'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 500 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 506 - INFO - XML element classification not needed. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1012 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11476976\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1020 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11476976\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 651/2190 (29.73%) | Elapsed: 00:11:37 | ETA: 00:27:27\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 651th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11479675)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11479675\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11479675 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 634 - INFO - Fetching data for pmc11479675\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 637 - INFO - Fetching data from scripts/exp_input/Local_fetched_data_1.parquet\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 438 - INFO - Raw xml data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11479675 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 452 - INFO - Skipping raw HTML/XML saving.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 458 - INFO - Using XMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 52 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 24 - INFO - Initializing xmlRetriever\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 466 - INFO - No additional data provided. Parsing raw data only.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 147 - INFO - Function call: parse_data(api_data(<class 'str'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11479675, additional_data, XML)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 162 - INFO - api_data converted to lxml element\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 466 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 153 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 154 - INFO - Consider migrating this function to the BaseRetriever class.\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 69 - INFO - Loading target sections for section name: supplementary_material_sections\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 477 - INFO - Found 6 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x15df32580>, <Element supplementary-material at 0x15d72f240>, <Element supplementary-material at 0x15d72ee00>, <Element supplementary-material at 0x15d72ce40>, <Element supplementary-material at 0x15d72d980>, <Element supplementary-material at 0x15d72ff00>]\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 485 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x15df32580>, <Element supplementary-material at 0x15d72f240>, <Element supplementary-material at 0x15d72ee00>, <Element supplementary-material at 0x15d72ce40>, <Element supplementary-material at 0x15d72d980>, <Element supplementary-material at 0x15d72ff00>]}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 574 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref SD1: Gene set enrichment analysis (GSEA) of whole-blood transcriptome data from the HTP reveals that individuals with T21 not only show enhanced IFN responses and other inflammatory signatures, but they also display elevated expression of genes involved in heme metabolism ( Figures 1A [rid=F1] and S1A [this file] ). Genes in the heme metabolism and HIF1A signatures upregulated in T21 display minimal overlap ( Figures 1D [rid=F1] – 1G [rid=F1] and S1B [this file] ). The heme metabolism and HIF1A signatures also show little overlap with chr21-encoded genes or IFN-stimulated genes (ISGs) from the IFN gamma/alpha response signatures ( Figures 1D [rid=F1] and S1C [this file] ). \n",
      "It also includes critical erythrocyte glycoproteins (e.g., and ), key factors for erythropoiesis and erythrocyte differentiation (e.g., and ), as well as genes involved in heme biosynthesis (e.g., and ) ( Figures 1H [rid=F1] and S1D [this file] ; Table S1 [rid=SD2] ). \n",
      "This signature also encompasses genes that regulate cell survival and apoptosis (e.g., and ), modulate fibrinolysis and clotting (e.g., and ), and contribute to cell adhesion and vascular function (e.g., and ) ( Figures 1I [rid=F1] and S1E [this file] ; Table S1 [rid=SD2] ). \n",
      "Interestingly, whereas the Heme score is slightly higher in euploid females compared to males, this difference is not observed in individuals with T21 ( Figure S1F [this file] ), nor are there sex disparities in HIF1A scores for either karyotype group ( Figure S1G [this file] ). Finally, both scores are significantly associated with age and body mass index (BMI) in both euploid controls and individuals with T21, with the Heme score revealing a particularly pronounced association with BMI ( Figures S1H [this file] – S1K [this file] ). \n",
      "In contrast, HIF1A genes showed more similar expression distributions across both specimen types ( Figures 1L [rid=F1] and S1L [this file] ). \n",
      "Both scores also positively correlate with EPO, general inflammatory markers (e.g., CRP and SAA1), and complement subunits (e.g., CFB and C9) in addition to several other factors ( Figures 2E [rid=F2] , S2A [this file] , and S2B [this file] ; Table S2 [rid=SD3] ). \n",
      "Individuals with T21 consistently exhibit higher EPO levels compared to euploid controls across the lifespan, with no significant associations with age in either group ( Figure S2C [this file] ). \n",
      "In line with these previous reports, analysis of complete blood count (CBC) data revealed significant differences in multiple blood parameters in research participants with DS, including elevated MCV, MCH, Hgb, and Hct alongside decreased RBC, WBC, and lymphocyte counts ( Figures 2J [rid=F2] , 2K [rid=F2] , and S2D [this file] ). Plotting levels of these parameters across all ages reveals dysregulation in individuals with T21 across the lifespan, with clear age trajectories ( Figure S2E [this file] ). \n",
      "Although no associations were found with the Heme score, likely due to the fact that these genes are mostly expressed in non-WBCs ( Figures 1L [rid=F1] and 1M [rid=F1] ), we found many interesting associations with the HIF1A score ( Figures 3A [rid=F3] and S3A [this file] – S3D [this file] ; Table S3 [rid=SD4] ). \n",
      "However, the striking elevation of basophils observed in DS is not associated with increased HIF1A signaling ( Figures 3A [rid=F3] and S3A [this file] ). \n",
      "Notably, whereas the Dp16 model demonstrates hallmarks of macrocytosis (increased MCV and MCH and decreased RBCs), this is not the case for Dp10 or Dp17 mice ( Figures 4A [rid=F4] , 4B [rid=F4] , S4A [this file] , and S4B [this file] ). GSEA of these data reveals that hallmark heme metabolism and hypoxia signatures are enriched across multiple tissues in the Dp16 mice ( Figures 4C [rid=F4] , 4D [rid=F4] , and S4C [this file] ). \n",
      "Heme metabolism shows the most prominent enrichment in the spleen, as evidenced by the overexpression of genes such as , , , , , , , and ( Figure 4F [rid=F4] ), all of which are elevated in the bloodstream of individuals with DS ( Figures 1 [rid=F1] and S1 [this file] ; Table S1 [rid=SD2] ). \n",
      "Conversely, the hypoxia signature is most prominent in embryonic (E12.5 and E18.5) and adult hearts, followed by embryonic facial mesenchyme, brain cortex, liver, and kidney ( Figures 4D [rid=F4] and S4C [this file] ). Notably, both and are upregulated in the heart of Dp16 mice across all time points analyzed ( Figures 4G [rid=F4] and S4D [this file] ). Additional overexpressed hypoxia-related genes in the heart include and in adults, and at E12.5, and and at E18.5 ( Figures 4G [rid=F4] and S4D [this file] ). \n",
      "Overlap analyses demonstrate statistically significant commonalities between the dysregulated signatures in human whole blood and heme signatures in the murine spleen, hypoxia signatures in the embryonic and adult murine heart, and HIF1A signatures across multiple tissues ( Figures 4J [rid=F4] , S4E [this file] , and S4F [this file] ), underscoring that the Dp16 strain is a suitable model for investigating dysregulation of heme metabolism, hypoxic signaling, and stress erythropoiesis in DS. \n",
      "This revealed numerous positive and inverse correlations between both scores and chr21 genes ( Figures 5A [rid=F5] , 5B [rid=F5] , S5A [this file] , and S5B [this file] ; Table S4 [rid=SD5] ), with overall similar trends ( Figure 5C [rid=F5] ). Beyond , both scores correlate with several other notable chr21 genes, including , , , and ( Figures 5A [rid=F5] – 5F [rid=F5] , S5A [this file] , and S5B [this file] ). For example, the Heme, but not HIF1A, score correlates with expression of and ( Figures S5A [this file] and S5B [this file] ). Conversely, the HIF1A, but not Heme, score correlates with expression of the IFN receptors and ( Figures S5A [this file] and S5B [this file] ). \n",
      "When analyzing CBCs from this participant, we find that while WBCs increase with JAK inhibition, there are no changes in markers of macrocytosis or dyserythropoiesis ( Figures S5C [this file] and S5D [this file] ). \n",
      "Whereas the genetic correction does indeed reverse overexpression ( Figure S5E [this file] ), it does not attenuate overexpression of genes in the heme metabolism, hypoxia, or HIF1A signatures ( Figure 5H [rid=F5] ), as illustrated by and in the spleen or and in the adult heart ( Figures 5I [rid=F5] and 5J [rid=F5] ). \n",
      "Lastly, we analyzed the relative expression of chr21 genes in the whole blood versus WBC transcriptome ( Figure 5K [rid=F5] ), which revealed genes more highly expressed in the whole-blood dataset, such as and , versus genes preferentially expressed in the WBC dataset, such as and ( Figures S5F [this file] and S5G [this file] ). \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref SD2: Ingenuity Pathway Analysis (IPA) of upstream regulator networks indicates that, beyond factors driving inflammatory signaling (e.g., IRFs [interferon regulatory factors]), HIF1A ranks 5 among upstream regulators predicted to drive gene expression changes in T21 ( Figure 1B [rid=F1] ; Table S1 [this file] ). Also predicted to be activated are GATA1 (ranked 13 ), a master regulator of erythropoiesis and RBC maturation, and its downstream target NFE2 (ranked 18 ), which regulates expression of myriad erythroid-specific genes ( Table S1 [this file] ). \n",
      "The heme metabolism signature encompasses genes that regulate hemoglobin oxygen affinity (e.g., and ) and those essential for hemoglobin assembly and stabilization (e.g., and ) ( Figure 1H [rid=F1] ; Table S1 [this file] ). It also includes critical erythrocyte glycoproteins (e.g., and ), key factors for erythropoiesis and erythrocyte differentiation (e.g., and ), as well as genes involved in heme biosynthesis (e.g., and ) ( Figures 1H [rid=F1] and S1D [rid=SD1] ; Table S1 [this file] ). \n",
      "In contrast, the HIF1A signature includes genes involved in extracellular matrix (ECM) remodeling and tissue repair (e.g., and ) and those that contribute to inflammatory and immune responses (e.g., and ) ( Figure 1I [rid=F1] ; Table S1 [this file] ). This signature also encompasses genes that regulate cell survival and apoptosis (e.g., and ), modulate fibrinolysis and clotting (e.g., and ), and contribute to cell adhesion and vascular function (e.g., and ) ( Figures 1I [rid=F1] and S1E [rid=SD1] ; Table S1 [this file] ). \n",
      "Heme metabolism shows the most prominent enrichment in the spleen, as evidenced by the overexpression of genes such as , , , , , , , and ( Figure 4F [rid=F4] ), all of which are elevated in the bloodstream of individuals with DS ( Figures 1 [rid=F1] and S1 [rid=SD1] ; Table S1 [this file] ). \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref SD3: To define the interplay between these proteomic changes and elevated heme metabolism and HIF1A signaling, we used Spearman correlation analysis to identify associations between the Heme and HIF1A scores and ~4,600 proteins in the matching HTP SomaScan plasma proteomics dataset ( Figures 2A [rid=F2] and 2B [rid=F2] ; Table S2 [this file] ). Both scores also positively correlate with EPO, general inflammatory markers (e.g., CRP and SAA1), and complement subunits (e.g., CFB and C9) in addition to several other factors ( Figures 2E [rid=F2] , S2A [rid=SD1] , and S2B [rid=SD1] ; Table S2 [this file] ). \n",
      "When defining Spearman correlations against all other plasma proteins ( Figure 2F [rid=F2] ; Table S2 [this file] ), TFRC emerges as the protein most strongly correlated with increased EPO production in individuals with T21 ( Figures 2F [rid=F2] and 2G [rid=F2] ; Table S2 [this file] ). \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref SD4: Although no associations were found with the Heme score, likely due to the fact that these genes are mostly expressed in non-WBCs ( Figures 1L [rid=F1] and 1M [rid=F1] ), we found many interesting associations with the HIF1A score ( Figures 3A [rid=F3] and S3A [rid=SD1] – S3D [rid=SD1] ; Table S3 [this file] ). \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref SD5: This revealed numerous positive and inverse correlations between both scores and chr21 genes ( Figures 5A [rid=F5] , 5B [rid=F5] , S5A [rid=SD1] , and S5B [rid=SD1] ; Table S4 [this file] ), with overall similar trends ( Figure 5C [rid=F5] ). \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref SD6: See key resources table [rid=T1] and Table S5 [this file] for antibody details. \n",
      "Arcsinh transformation was applied to marker expression data with cofactor values ranging from ~0.2 to ~15 to give optimal separation of positive and negative populations for each marker, using the function from the flowVS R package and based on visual inspection of marker histograms (see Table S5 [this file] ). \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 263 - INFO - Extracting links from full XML content.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 301 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 309 - INFO - Content length: 227757\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 326 - INFO - Prompt messages total length: 57693 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 332 - INFO - Prompt ID: gemini-2.5-flash-0-9d3ebdef6ef80bfe8359d63e045efcac47a0518f8054339346905101aed786ca\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 351 - INFO - Requesting datasets from content using model: gemini-2.5-flash, temperature: 0, messages length: 57693 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-32d75371-a198-4df6-a580-8272f583068d\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn31488784\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn31488784\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn31488780\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn31488780\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE190125\\\",\\n    \\\"repository_reference\\\": \\\"GEO\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE251967\\\",\\n    \\\"repository_reference\\\": \\\"GEO\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn31488781\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn31488781\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn31488783\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn31488783\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"FR-FCM-Z5GE\\\",\\n    \\\"repository_reference\\\": \\\"Flow Repository\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE218883\\\",\\n    \\\"repository_reference\\\": \\\"GEO\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE218885\\\",\\n    \\\"repository_reference\\\": \\\"GEO\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE272567\\\",\\n    \\\"repository_reference\\\": \\\"GEO\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE272568\\\",\\n    \\\"repository_reference\\\": \\\"GEO\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE272569\\\",\\n    \\\"repository_reference\\\": \\\"GEO\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE272570\\\",\\n    \\\"repository_reference\\\": \\\"GEO\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE272572\\\",\\n    \\\"repository_reference\\\": \\\"GEO\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE218887\\\",\\n    \\\"repository_reference\\\": \\\"GEO\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE218888\\\",\\n    \\\"repository_reference\\\": \\\"GEO\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE218889\\\",\\n    \\\"repository_reference\\\": \\\"GEO\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE218890\\\",\\n    \\\"repository_reference\\\": \\\"GEO\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE229762\\\",\\n    \\\"repository_reference\\\": \\\"GEO\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn31488784\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn31488784\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn31488780\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn31488780\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE190125\\\",\\n    \\\"repository_reference\\\": \\\"GEO\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE251967\\\",\\n    \\\"repository_reference\\\": \\\"GEO\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn31488781\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn31488781\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn31488783\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn31488783\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"FR-FCM-Z5GE\\\",\\n    \\\"repository_reference\\\": \\\"Flow Repository\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE218883\\\",\\n    \\\"repository_reference\\\": \\\"GEO\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE218885\\\",\\n    \\\"repository_reference\\\": \\\"GEO\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE272567\\\",\\n    \\\"repository_reference\\\": \\\"GEO\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE272568\\\",\\n    \\\"repository_reference\\\": \\\"GEO\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE272569\\\",\\n    \\\"repository_reference\\\": \\\"GEO\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE272570\\\",\\n    \\\"repository_reference\\\": \\\"GEO\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE272572\\\",\\n    \\\"repository_reference\\\": \\\"GEO\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE218887\\\",\\n    \\\"repository_reference\\\": \\\"GEO\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE218888\\\",\\n    \\\"repository_reference\\\": \\\"GEO\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE218889\\\",\\n    \\\"repository_reference\\\": \\\"GEO\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE218890\\\",\\n    \\\"repository_reference\\\": \\\"GEO\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE229762\\\",\\n    \\\"repository_reference\\\": \\\"GEO\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1758210596,\n",
      "    \"model\": \"gemini-2.5-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 73488,\n",
      "        \"completion_tokens\": 723,\n",
      "        \"total_tokens\": 77059,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 2848,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 664 - INFO - Deduplicating response with 19 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn31488784\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn31488780\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: GSE190125\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: GSE251967\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn31488781\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn31488783\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: FR-FCM-Z5GE\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: GSE218883\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: GSE218885\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: GSE272567\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: GSE272568\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: GSE272569\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: GSE272570\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: GSE272572\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: GSE218887\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: GSE218888\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: GSE218889\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: GSE218890\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: GSE229762\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn31488784', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn31488784', 'dataset_id': 'syn31488784'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn31488784\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn31488784\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn31488784, identifier: syn31488784, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn31488784\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn31488784 for https://www.synapse.org/#!Synapse:syn31488784\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn31488784 for https://www.synapse.org/#!Synapse:syn31488784\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn31488784 for https://www.synapse.org/#!Synapse:syn31488784\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn31488784 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn31488784\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn31488784\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn31488784\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn31488784\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn31488784\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn31488784\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn31488784\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn31488784\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn31488784\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn31488784, synapse.org, https://www.synapse.org/#!Synapse:syn31488784\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn31488784\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn31488784\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn31488784', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn31488784', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn31488780', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn31488780', 'dataset_id': 'syn31488780'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn31488780\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn31488780\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn31488780, identifier: syn31488780, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn31488780\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn31488780 for https://www.synapse.org/#!Synapse:syn31488780\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn31488780 for https://www.synapse.org/#!Synapse:syn31488780\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn31488780 for https://www.synapse.org/#!Synapse:syn31488780\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn31488780 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn31488780\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn31488780\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn31488780\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn31488780\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn31488780\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn31488780\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn31488780\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn31488780\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn31488780\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn31488780, synapse.org, https://www.synapse.org/#!Synapse:syn31488780\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn31488780\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn31488780\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn31488780', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn31488780', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'GSE190125', 'repository_reference': 'GEO', 'dataset_id': 'GSE190125'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE190125\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: GSE190125\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: GEO, identifier: GSE190125, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE190125 for GEO\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: GEO\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE190125\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE190125\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE190125\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: GSE190125, geo, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: GSE190125\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'GSE190125', 'data_repository': 'geo', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'GSE251967', 'repository_reference': 'GEO', 'dataset_id': 'GSE251967'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE251967\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: GSE251967\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: GEO, identifier: GSE251967, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE251967 for GEO\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: GEO\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE251967\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE251967\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE251967\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: GSE251967, geo, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: GSE251967\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'GSE251967', 'data_repository': 'geo', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn31488781', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn31488781', 'dataset_id': 'syn31488781'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn31488781\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn31488781\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn31488781, identifier: syn31488781, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn31488781\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn31488781 for https://www.synapse.org/#!Synapse:syn31488781\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn31488781 for https://www.synapse.org/#!Synapse:syn31488781\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn31488781 for https://www.synapse.org/#!Synapse:syn31488781\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn31488781 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn31488781\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn31488781\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn31488781\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn31488781\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn31488781\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn31488781\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn31488781\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn31488781\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn31488781\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn31488781, synapse.org, https://www.synapse.org/#!Synapse:syn31488781\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn31488781\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn31488781\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn31488781', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn31488781', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn31488783', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn31488783', 'dataset_id': 'syn31488783'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn31488783\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn31488783\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn31488783, identifier: syn31488783, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn31488783\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn31488783 for https://www.synapse.org/#!Synapse:syn31488783\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn31488783 for https://www.synapse.org/#!Synapse:syn31488783\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn31488783 for https://www.synapse.org/#!Synapse:syn31488783\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn31488783 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn31488783\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn31488783\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn31488783\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn31488783\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn31488783\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn31488783\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn31488783\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn31488783\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn31488783\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn31488783, synapse.org, https://www.synapse.org/#!Synapse:syn31488783\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn31488783\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn31488783\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn31488783', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn31488783', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'FR-FCM-Z5GE', 'repository_reference': 'Flow Repository', 'dataset_id': 'FR-FCM-Z5GE'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: FR-FCM-Z5GE\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID FR-FCM-Z5GE is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: Flow Repository, identifier: FR-FCM-Z5GE, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier FR-FCM-Z5GE for Flow Repository\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier FR-FCM-Z5GE for Flow Repository\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier FR-FCM-Z5GE for Flow Repository\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier FR-FCM-Z5GE for Flow Repository\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier FR-FCM-Z5GE for Flow Repository\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier FR-FCM-Z5GE for Flow Repository\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: Flow Repository\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'FR-FCM-Z5GE', 'data_repository': 'flow repository', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'GSE218883', 'repository_reference': 'GEO', 'dataset_id': 'GSE218883'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE218883\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: GSE218883\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: GEO, identifier: GSE218883, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE218883 for GEO\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: GEO\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE218883\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE218883\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE218883\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: GSE218883, geo, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: GSE218883\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'GSE218883', 'data_repository': 'geo', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'GSE218885', 'repository_reference': 'GEO', 'dataset_id': 'GSE218885'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE218885\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: GSE218885\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: GEO, identifier: GSE218885, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE218885 for GEO\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: GEO\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE218885\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE218885\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE218885\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: GSE218885, geo, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: GSE218885\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'GSE218885', 'data_repository': 'geo', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'GSE272567', 'repository_reference': 'GEO', 'dataset_id': 'GSE272567'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE272567\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: GSE272567\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: GEO, identifier: GSE272567, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE272567 for GEO\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: GEO\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE272567\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE272567\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE272567\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: GSE272567, geo, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: GSE272567\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'GSE272567', 'data_repository': 'geo', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'GSE272568', 'repository_reference': 'GEO', 'dataset_id': 'GSE272568'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE272568\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: GSE272568\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: GEO, identifier: GSE272568, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE272568 for GEO\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: GEO\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE272568\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE272568\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE272568\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: GSE272568, geo, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: GSE272568\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'GSE272568', 'data_repository': 'geo', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'GSE272569', 'repository_reference': 'GEO', 'dataset_id': 'GSE272569'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE272569\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: GSE272569\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: GEO, identifier: GSE272569, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE272569 for GEO\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: GEO\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE272569\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE272569\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE272569\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: GSE272569, geo, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: GSE272569\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'GSE272569', 'data_repository': 'geo', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'GSE272570', 'repository_reference': 'GEO', 'dataset_id': 'GSE272570'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE272570\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: GSE272570\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: GEO, identifier: GSE272570, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE272570 for GEO\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: GEO\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE272570\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE272570\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE272570\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: GSE272570, geo, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: GSE272570\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'GSE272570', 'data_repository': 'geo', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'GSE272572', 'repository_reference': 'GEO', 'dataset_id': 'GSE272572'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE272572\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: GSE272572\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: GEO, identifier: GSE272572, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE272572 for GEO\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: GEO\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE272572\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE272572\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE272572\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: GSE272572, geo, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: GSE272572\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'GSE272572', 'data_repository': 'geo', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'GSE218887', 'repository_reference': 'GEO', 'dataset_id': 'GSE218887'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE218887\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: GSE218887\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: GEO, identifier: GSE218887, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE218887 for GEO\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: GEO\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE218887\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE218887\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE218887\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: GSE218887, geo, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: GSE218887\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'GSE218887', 'data_repository': 'geo', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'GSE218888', 'repository_reference': 'GEO', 'dataset_id': 'GSE218888'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE218888\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: GSE218888\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: GEO, identifier: GSE218888, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE218888 for GEO\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: GEO\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE218888\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE218888\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE218888\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: GSE218888, geo, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: GSE218888\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'GSE218888', 'data_repository': 'geo', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'GSE218889', 'repository_reference': 'GEO', 'dataset_id': 'GSE218889'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE218889\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: GSE218889\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: GEO, identifier: GSE218889, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE218889 for GEO\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: GEO\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE218889\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE218889\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE218889\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: GSE218889, geo, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: GSE218889\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'GSE218889', 'data_repository': 'geo', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'GSE218890', 'repository_reference': 'GEO', 'dataset_id': 'GSE218890'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE218890\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: GSE218890\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: GEO, identifier: GSE218890, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE218890 for GEO\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: GEO\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE218890\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE218890\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE218890\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: GSE218890, geo, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: GSE218890\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'GSE218890', 'data_repository': 'geo', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'GSE229762', 'repository_reference': 'GEO', 'dataset_id': 'GSE229762'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE229762\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: GSE229762\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: GEO, identifier: GSE229762, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE229762 for GEO\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: GEO\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE229762\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE229762\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE229762\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: GSE229762, geo, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: GSE229762\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'GSE229762', 'data_repository': 'geo', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 278 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn31488784', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn31488784', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn31488780', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn31488780', 'citation_type': 'n/a'}, {'dataset_identifier': 'GSE190125', 'data_repository': 'geo', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'GSE251967', 'data_repository': 'geo', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn31488781', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn31488781', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn31488783', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn31488783', 'citation_type': 'n/a'}, {'dataset_identifier': 'FR-FCM-Z5GE', 'data_repository': 'flow repository', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'GSE218883', 'data_repository': 'geo', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'GSE218885', 'data_repository': 'geo', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'GSE272567', 'data_repository': 'geo', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'GSE272568', 'data_repository': 'geo', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'GSE272569', 'data_repository': 'geo', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'GSE272570', 'data_repository': 'geo', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'GSE272572', 'data_repository': 'geo', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'GSE218887', 'data_repository': 'geo', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'GSE218888', 'data_repository': 'geo', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'GSE218889', 'data_repository': 'geo', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'GSE218890', 'data_repository': 'geo', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'GSE229762', 'data_repository': 'geo', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1191 - INFO - Fetching metadata for 19 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn31488784\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn31488784, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn31488784 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn31488784 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn31488784 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn31488784 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn31488784\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn31488784\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn31488780\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn31488780, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn31488780 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn31488780 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn31488780 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn31488780 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn31488780\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn31488780\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: GSE190125\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: geo, identifier: GSE190125, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE190125 for geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for GSE190125 in geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 3 with repo: geo and accession_id: GSE190125\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE190125\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 3: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: GSE251967\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: geo, identifier: GSE251967, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE251967 for geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for GSE251967 in geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 4 with repo: geo and accession_id: GSE251967\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE251967\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 4: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn31488781\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn31488781, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn31488781 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn31488781 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn31488781 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn31488781 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 5 with repo: synapse.org and accession_id: syn31488781\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn31488781\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 5: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn31488783\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn31488783, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn31488783 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn31488783 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn31488783 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn31488783 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 6 with repo: synapse.org and accession_id: syn31488783\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn31488783\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 6: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: FR-FCM-Z5GE\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: flow repository, identifier: FR-FCM-Z5GE, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier FR-FCM-Z5GE for flow repository\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier FR-FCM-Z5GE for flow repository\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier FR-FCM-Z5GE for flow repository\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier FR-FCM-Z5GE for flow repository\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier FR-FCM-Z5GE for flow repository\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier FR-FCM-Z5GE for flow repository\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: flow repository\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for FR-FCM-Z5GE in flow repository\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 7 with repo: flow repository and accession_id: FR-FCM-Z5GE\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository flow repository unknown in Ontology. Skipping dataset page 7.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: GSE218883\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: geo, identifier: GSE218883, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE218883 for geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for GSE218883 in geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 8 with repo: geo and accession_id: GSE218883\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE218883\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 8: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: GSE218885\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: geo, identifier: GSE218885, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE218885 for geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for GSE218885 in geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 9 with repo: geo and accession_id: GSE218885\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE218885\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 9: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 10 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: GSE272567\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: geo, identifier: GSE272567, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE272567 for geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for GSE272567 in geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 10 with repo: geo and accession_id: GSE272567\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE272567\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 10: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 11 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: GSE272568\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: geo, identifier: GSE272568, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE272568 for geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for GSE272568 in geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 11 with repo: geo and accession_id: GSE272568\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE272568\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 11: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 12 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: GSE272569\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: geo, identifier: GSE272569, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE272569 for geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for GSE272569 in geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 12 with repo: geo and accession_id: GSE272569\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE272569\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 12: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 13 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: GSE272570\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: geo, identifier: GSE272570, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE272570 for geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for GSE272570 in geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 13 with repo: geo and accession_id: GSE272570\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE272570\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 13: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 14 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: GSE272572\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: geo, identifier: GSE272572, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE272572 for geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for GSE272572 in geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 14 with repo: geo and accession_id: GSE272572\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE272572\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 14: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 15 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: GSE218887\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: geo, identifier: GSE218887, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE218887 for geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for GSE218887 in geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 15 with repo: geo and accession_id: GSE218887\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE218887\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 15: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 16 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: GSE218888\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: geo, identifier: GSE218888, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE218888 for geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for GSE218888 in geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 16 with repo: geo and accession_id: GSE218888\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE218888\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 16: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 17 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: GSE218889\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: geo, identifier: GSE218889, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE218889 for geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for GSE218889 in geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 17 with repo: geo and accession_id: GSE218889\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE218889\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 17: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 18 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: GSE218890\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: geo, identifier: GSE218890, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE218890 for geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for GSE218890 in geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 18 with repo: geo and accession_id: GSE218890\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE218890\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 18: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 19 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: GSE229762\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: geo, identifier: GSE229762, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE229762 for geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for GSE229762 in geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 19 with repo: geo and accession_id: GSE229762\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE229762\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 19: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1277 - INFO - Updated datasets len: 19\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 287 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 25, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'source_url', 'download_link',\n",
      "       'title', 'content_type', 'id', 'caption', 'description',\n",
      "       'source_section', 'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 500 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 506 - INFO - XML element classification not needed. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1012 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11479675\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1020 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11479675\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 652th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11481439)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11481439\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11481439 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 634 - INFO - Fetching data for pmc11481439\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 637 - INFO - Fetching data from scripts/exp_input/Local_fetched_data_1.parquet\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 438 - INFO - Raw xml data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11481439 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 452 - INFO - Skipping raw HTML/XML saving.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 458 - INFO - Using XMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 52 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 24 - INFO - Initializing xmlRetriever\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 466 - INFO - No additional data provided. Parsing raw data only.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 147 - INFO - Function call: parse_data(api_data(<class 'str'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11481439, additional_data, XML)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 162 - INFO - api_data converted to lxml element\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 466 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 153 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 154 - INFO - Consider migrating this function to the BaseRetriever class.\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 69 - INFO - Loading target sections for section name: supplementary_material_sections\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 477 - INFO - Found 1 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x15e9e1140>]\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 485 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x15e9e1140>]}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 574 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM1: Subsequently, independent confirmatory PWAS using a different reference brain proteome (ROSMAP) replicated 9 genes ( ≤ 0.05/1761 = 2.84E-05), accounting for a high proportion of 64.3% (Table 1 [rid=Tab1] ; Fig. 2 [rid=Fig2] A; Additional file1 Table S1 [this file] ). \n",
      "Subsequently, independent confirmatory PWAS replicated 9 genes ( ≤ 0.05/1761 = 2.84E-05), accounting for a high proportion of 64.3% (Table 1 [rid=Tab1] ; Fig. 2 [rid=Fig2] B; Additional file1 Table S2 [this file] ). \n",
      "Subsequently, independent confirmatory PWAS replicated 6 genes ( ≤ 0.05/1761 = 2.84E-05), accounting for a high proportion of 66.7% (Table 1 [rid=Tab1] ; Fig. 2 [rid=Fig2] C; Additional file1 Table S3 [this file] ). \n",
      "Finally, TWAS provided secondary validation for the “vital protein” SULT1A1 and FDPS, both strongly associated with IBD and CD, while other genes did not yield significant findings (Table 4 [rid=Tab4] ; Fig. 4 [rid=Fig4] , Additional file1 Table S4 [this file] ). \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 263 - INFO - Extracting links from full XML content.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 301 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 309 - INFO - Content length: 155810\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 326 - INFO - Prompt messages total length: 39706 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 332 - INFO - Prompt ID: gemini-2.5-flash-0-bdfeef981a444b51300b6798685259e55f3524747cdd0431ab56de8141eff85f\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 351 - INFO - Requesting datasets from content using model: gemini-2.5-flash, temperature: 0, messages length: 39706 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-2db64c01-3943-436f-a3ab-963f9e73861e\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn23627957\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn23627957\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"IIBDGC\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ibdgc.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://cnsgenomics.com/software/smr/#eQTLsummarydata\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"CMC\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/CMC\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn23627957\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn23627957\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"IIBDGC\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ibdgc.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://cnsgenomics.com/software/smr/#eQTLsummarydata\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"CMC\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/CMC\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1758210626,\n",
      "    \"model\": \"gemini-2.5-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 52298,\n",
      "        \"completion_tokens\": 180,\n",
      "        \"total_tokens\": 56608,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 4130,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 664 - INFO - Deduplicating response with 4 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn23627957\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: IIBDGC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: CMC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn23627957', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn23627957', 'dataset_id': 'syn23627957'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn23627957\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn23627957\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn23627957, identifier: syn23627957, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn23627957\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn23627957 for https://www.synapse.org/#!Synapse:syn23627957\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn23627957 for https://www.synapse.org/#!Synapse:syn23627957\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn23627957 for https://www.synapse.org/#!Synapse:syn23627957\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn23627957 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn23627957\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn23627957\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn23627957\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn23627957\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn23627957\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn23627957\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn23627957\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn23627957\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn23627957\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn23627957, synapse.org, https://www.synapse.org/#!Synapse:syn23627957\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn23627957\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn23627957\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn23627957', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn23627957', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'IIBDGC', 'repository_reference': 'https://www.ibdgc.org/', 'dataset_id': 'IIBDGC'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: IIBDGC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID IIBDGC is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.ibdgc.org/, identifier: IIBDGC, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.ibdgc.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier IIBDGC for https://www.ibdgc.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier IIBDGC for https://www.ibdgc.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier IIBDGC for https://www.ibdgc.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier IIBDGC for https://www.ibdgc.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier IIBDGC for https://www.ibdgc.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier IIBDGC for https://www.ibdgc.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://www.ibdgc.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'IIBDGC', 'data_repository': 'www.ibdgc.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://cnsgenomics.com/software/smr/#eQTLsummarydata', 'dataset_id': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://cnsgenomics.com/software/smr/#eQTLsummarydata, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://cnsgenomics.com/software/smr/#eQTLsummarydata\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://cnsgenomics.com/software/smr/#eQTLsummarydata\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'cnsgenomics.com', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'CMC', 'repository_reference': 'https://www.synapse.org/CMC', 'dataset_id': 'CMC'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: CMC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID CMC is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/CMC, identifier: CMC, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/CMC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier CMC for https://www.synapse.org/CMC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier CMC for https://www.synapse.org/CMC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier CMC for https://www.synapse.org/CMC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/CMC in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'CMC', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 278 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn23627957', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn23627957', 'citation_type': 'n/a'}, {'dataset_identifier': 'IIBDGC', 'data_repository': 'www.ibdgc.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'cnsgenomics.com', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'CMC', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1191 - INFO - Fetching metadata for 4 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn23627957\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn23627957, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn23627957 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn23627957 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn23627957 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn23627957 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn23627957\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn23627957\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: IIBDGC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: www.ibdgc.org, identifier: IIBDGC, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier IIBDGC for www.ibdgc.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier IIBDGC for www.ibdgc.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier IIBDGC for www.ibdgc.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier IIBDGC for www.ibdgc.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier IIBDGC for www.ibdgc.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier IIBDGC for www.ibdgc.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: www.ibdgc.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL www.ibdgc.org may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for IIBDGC in www.ibdgc.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 2 with repo: www.ibdgc.org and accession_id: IIBDGC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository www.ibdgc.org unknown in Ontology. Skipping dataset page 2.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 3: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: CMC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: CMC, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier CMC for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier CMC for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier CMC for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for CMC in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1085 - WARNING - Identifier CMC does not match pattern for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 4 with repo: synapse.org and accession_id: CMC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:CMC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 4: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1277 - INFO - Updated datasets len: 4\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 287 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 5, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'source_url', 'download_link',\n",
      "       'title', 'content_type', 'id', 'caption', 'description',\n",
      "       'source_section', 'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 500 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 506 - INFO - XML element classification not needed. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1012 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11481439\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1020 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11481439\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 653th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11481771)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11481771\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11481771 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 634 - INFO - Fetching data for pmc11481771\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 637 - INFO - Fetching data from scripts/exp_input/Local_fetched_data_1.parquet\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 438 - INFO - Raw xml data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11481771 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 452 - INFO - Skipping raw HTML/XML saving.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 458 - INFO - Using XMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 52 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 24 - INFO - Initializing xmlRetriever\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 466 - INFO - No additional data provided. Parsing raw data only.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 147 - INFO - Function call: parse_data(api_data(<class 'str'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11481771, additional_data, XML)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 162 - INFO - api_data converted to lxml element\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 466 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 153 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 154 - INFO - Consider migrating this function to the BaseRetriever class.\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 69 - INFO - Loading target sections for section name: supplementary_material_sections\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 477 - INFO - Found 3 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x15d4d5140>, <Element supplementary-material at 0x15d63bb80>, <Element supplementary-material at 0x15d638c40>]\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 485 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x15d4d5140>, <Element supplementary-material at 0x15d63bb80>, <Element supplementary-material at 0x15d638c40>]}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 574 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM1: These correlations were much more significant than those between random gene pairs (Supplementary Information 3 A, Fig. S1 [this file] , S2 [this file] ). These results indicate that the microglia network is robust and reliable (Supplementary Information 3B, Table S1 [this file] ). \n",
      "The average expression of amyloid beta plaques between the clusters correlated with Braak staging, showing that the clusters appropriately represent clinical phenotypes. (Supplementary Information 3E, Fig. S5 [this file] ) \n",
      "This indicates that the pivotal risk factors play an important role in subject classification (Supplementary Information 3G, Fig. S6 [this file] ). \n",
      "A literature search on the identified targets verified their potential as therapeutic targets (Table S6 [this file] ). \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM2: By using known experimental results, perturbation simulations on nodes were performed, and changes in the model and experimental results were compared to ensure the appropriateness and fidelity of our network model (Supplementary Information 3B, Table S2 [this file] ). \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM3: We employed 1,000,000 initial states for 500 steps of synchronous updates, with the average node expression at time steps 401 to 500 selected as a suitable measure, signifying sufficient convergence of network expression levels (Supplementary Information 3D, Fig. S3 [this file] ). \n",
      "Hence, for precision medicine, it becomes imperative to perform subject classification that incorporates the genetic backgrounds of individuals, which is possible using network simulations (Fig. S4 [this file] ). \n",
      "Based on these average node activities within the network, we successfully identified nine distinct subtypes among the subjects with different clinical phenotypes (Fig. 4 [rid=Fig4] A and B, Table S3 [this file] ). \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 263 - INFO - Extracting links from full XML content.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 301 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 309 - INFO - Content length: 186338\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 326 - INFO - Prompt messages total length: 47338 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 332 - INFO - Prompt ID: gemini-2.5-flash-0-dd5732354ca89086931368efae2dd8903f4f3feba2daae7daf9a50a9342bdb1e\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 351 - INFO - Requesting datasets from content using model: gemini-2.5-flash, temperature: 0, messages length: 47338 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-10d237f4-176a-4262-9d3e-752bcec622fe\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn21438358\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn21438358\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn11724057\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn11724057\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn11723899\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn11723899\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn21438358\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn21438358\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn11724057\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn11724057\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn11723899\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn11723899\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1758210647,\n",
      "    \"model\": \"gemini-2.5-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 58999,\n",
      "        \"completion_tokens\": 173,\n",
      "        \"total_tokens\": 62145,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 2973,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 664 - INFO - Deduplicating response with 3 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn21438358\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn11724057\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn11723899\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn21438358', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn21438358', 'dataset_id': 'syn21438358'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn21438358\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn21438358\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn21438358, identifier: syn21438358, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn21438358\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn21438358 for https://www.synapse.org/#!Synapse:syn21438358\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn21438358 for https://www.synapse.org/#!Synapse:syn21438358\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn21438358 for https://www.synapse.org/#!Synapse:syn21438358\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn21438358 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn21438358\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn21438358\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn21438358\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn21438358\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn21438358\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn21438358\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn21438358\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn21438358\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn21438358\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn21438358, synapse.org, https://www.synapse.org/#!Synapse:syn21438358\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn21438358\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn21438358\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn21438358', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn21438358', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn11724057', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn11724057', 'dataset_id': 'syn11724057'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn11724057\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn11724057\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn11724057, identifier: syn11724057, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn11724057\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn11724057 for https://www.synapse.org/#!Synapse:syn11724057\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn11724057 for https://www.synapse.org/#!Synapse:syn11724057\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn11724057 for https://www.synapse.org/#!Synapse:syn11724057\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn11724057 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn11724057\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn11724057\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn11724057\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn11724057\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn11724057\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn11724057\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn11724057\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn11724057\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn11724057\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn11724057, synapse.org, https://www.synapse.org/#!Synapse:syn11724057\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn11724057\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn11724057\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn11724057', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn11724057', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn11723899', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn11723899', 'dataset_id': 'syn11723899'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn11723899\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn11723899\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn11723899, identifier: syn11723899, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn11723899\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn11723899 for https://www.synapse.org/#!Synapse:syn11723899\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn11723899 for https://www.synapse.org/#!Synapse:syn11723899\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn11723899 for https://www.synapse.org/#!Synapse:syn11723899\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn11723899 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn11723899\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn11723899\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn11723899\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn11723899\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn11723899\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn11723899\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn11723899\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn11723899\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn11723899\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn11723899, synapse.org, https://www.synapse.org/#!Synapse:syn11723899\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn11723899\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn11723899\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn11723899', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn11723899', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 278 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn21438358', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn21438358', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn11724057', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn11724057', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn11723899', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn11723899', 'citation_type': 'n/a'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1191 - INFO - Fetching metadata for 3 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn21438358\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn21438358, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn21438358 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn21438358 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn21438358 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn21438358 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn21438358\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn21438358\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn11724057\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn11724057, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn11724057 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn11724057 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn11724057 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn11724057 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn11724057\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn11724057\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn11723899\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn11723899, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn11723899 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn11723899 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn11723899 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn11723899 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 3 with repo: synapse.org and accession_id: syn11723899\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn11723899\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 3: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1277 - INFO - Updated datasets len: 3\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 287 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 6, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'source_url', 'download_link',\n",
      "       'title', 'content_type', 'id', 'caption', 'description',\n",
      "       'source_section', 'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 500 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 506 - INFO - XML element classification not needed. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1012 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11481771\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1020 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11481771\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 654th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11485001)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11485001\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11485001 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 634 - INFO - Fetching data for pmc11485001\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 637 - INFO - Fetching data from scripts/exp_input/Local_fetched_data_1.parquet\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 438 - INFO - Raw xml data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11485001 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 452 - INFO - Skipping raw HTML/XML saving.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 458 - INFO - Using XMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 52 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 24 - INFO - Initializing xmlRetriever\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 466 - INFO - No additional data provided. Parsing raw data only.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 147 - INFO - Function call: parse_data(api_data(<class 'str'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11485001, additional_data, XML)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 162 - INFO - api_data converted to lxml element\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 466 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 153 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 154 - INFO - Consider migrating this function to the BaseRetriever class.\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 69 - INFO - Loading target sections for section name: supplementary_material_sections\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 477 - INFO - Found 1 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x15d9a6f00>]\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 485 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x15d9a6f00>]}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 574 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM1: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 263 - INFO - Extracting links from full XML content.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 301 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 309 - INFO - Content length: 103330\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 326 - INFO - Prompt messages total length: 26586 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 332 - INFO - Prompt ID: gemini-2.5-flash-0-f0b362f93b35eaed5fbe20c4a933814375f3dc262fb75360a4764627cb2d9f2d\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 351 - INFO - Requesting datasets from content using model: gemini-2.5-flash, temperature: 0, messages length: 26586 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-faef81da-41a8-4420-8eb2-2b4c47de0a8e\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"http://xena.ucsc.edu/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://portal.gdc.cancer.gov/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn7824274\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn7824274\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://tcpaportal.org/tcpa/index.html\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://guolab.wchscu.cn/GSCA/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://platform.opentargets.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"http://timer.cistrome.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"http://gepia2.cancer-pku.cn/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"http://biocc.hrbmu.edu.cn/CancerSEA/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"http://tismo.cistrome.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://clue.io/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"http://www.rcsb.org/pdb/home/home.do\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"http://xena.ucsc.edu/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://portal.gdc.cancer.gov/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn7824274\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn7824274\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://tcpaportal.org/tcpa/index.html\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://guolab.wchscu.cn/GSCA/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://platform.opentargets.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"http://timer.cistrome.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"http://gepia2.cancer-pku.cn/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"http://biocc.hrbmu.edu.cn/CancerSEA/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"http://tismo.cistrome.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://clue.io/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"http://www.rcsb.org/pdb/home/home.do\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1758210672,\n",
      "    \"model\": \"gemini-2.5-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 31574,\n",
      "        \"completion_tokens\": 479,\n",
      "        \"total_tokens\": 36068,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 4015,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 664 - INFO - Deduplicating response with 12 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn7824274\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'http://xena.ucsc.edu/', 'dataset_id': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: http://xena.ucsc.edu/, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: http://xena.ucsc.edu/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: http://xena.ucsc.edu/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'xena.ucsc.edu', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://portal.gdc.cancer.gov/'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://portal.gdc.cancer.gov/, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://portal.gdc.cancer.gov/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://portal.gdc.cancer.gov/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'portal.gdc.cancer.gov', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn7824274', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn7824274', 'dataset_id': 'syn7824274'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn7824274\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn7824274\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn7824274, identifier: syn7824274, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn7824274\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn7824274 for https://www.synapse.org/#!Synapse:syn7824274\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn7824274 for https://www.synapse.org/#!Synapse:syn7824274\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn7824274 for https://www.synapse.org/#!Synapse:syn7824274\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn7824274 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn7824274\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn7824274\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn7824274\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn7824274\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn7824274\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn7824274\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn7824274\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn7824274\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn7824274\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn7824274, synapse.org, https://www.synapse.org/#!Synapse:syn7824274\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn7824274\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn7824274\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn7824274', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn7824274', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://tcpaportal.org/tcpa/index.html'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://tcpaportal.org/tcpa/index.html, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://tcpaportal.org/tcpa/index.html\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://tcpaportal.org/tcpa/index.html\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'tcpaportal.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://guolab.wchscu.cn/GSCA/'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://guolab.wchscu.cn/GSCA/, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://guolab.wchscu.cn/GSCA/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://guolab.wchscu.cn/GSCA/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'guolab.wchscu.cn', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://platform.opentargets.org/'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://platform.opentargets.org/, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://platform.opentargets.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://platform.opentargets.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'platform.opentargets.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'http://timer.cistrome.org/'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: http://timer.cistrome.org/, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: http://timer.cistrome.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: http://timer.cistrome.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'timer.cistrome.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'http://gepia2.cancer-pku.cn/'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: http://gepia2.cancer-pku.cn/, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: http://gepia2.cancer-pku.cn/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: http://gepia2.cancer-pku.cn/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'gepia2.cancer-pku.cn', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'http://biocc.hrbmu.edu.cn/CancerSEA/'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: http://biocc.hrbmu.edu.cn/CancerSEA/, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: http://biocc.hrbmu.edu.cn/CancerSEA/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: http://biocc.hrbmu.edu.cn/CancerSEA/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'biocc.hrbmu.edu.cn', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'http://tismo.cistrome.org/'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: http://tismo.cistrome.org/, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: http://tismo.cistrome.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: http://tismo.cistrome.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'tismo.cistrome.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://clue.io/'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://clue.io/, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://clue.io/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://clue.io/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'clue.io', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'http://www.rcsb.org/pdb/home/home.do'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: http://www.rcsb.org/pdb/home/home.do, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: http://www.rcsb.org/pdb/home/home.do\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: http://www.rcsb.org/pdb/home/home.do\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'www.rcsb.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 278 - INFO - Augmented dataset links: [{'dataset_identifier': 'n/a', 'data_repository': 'xena.ucsc.edu', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'portal.gdc.cancer.gov', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn7824274', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn7824274', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'tcpaportal.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'guolab.wchscu.cn', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'platform.opentargets.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'timer.cistrome.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'gepia2.cancer-pku.cn', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'biocc.hrbmu.edu.cn', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'tismo.cistrome.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'clue.io', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'www.rcsb.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1191 - INFO - Fetching metadata for 12 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 1: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 2: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn7824274\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn7824274, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn7824274 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn7824274 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn7824274 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn7824274 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 3 with repo: synapse.org and accession_id: syn7824274\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn7824274\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 3: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 4: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 5: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 6: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 7: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 8: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 9: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 10 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 10: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 11 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 11: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 12 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 12: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1277 - INFO - Updated datasets len: 12\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 287 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 13, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'source_url', 'download_link',\n",
      "       'title', 'content_type', 'id', 'caption', 'description',\n",
      "       'source_section', 'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 500 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 506 - INFO - XML element classification not needed. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1012 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11485001\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1020 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11485001\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 655th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11485084)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11485084\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11485084 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 634 - INFO - Fetching data for pmc11485084\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 637 - INFO - Fetching data from scripts/exp_input/Local_fetched_data_1.parquet\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 438 - INFO - Raw xml data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11485084 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 452 - INFO - Skipping raw HTML/XML saving.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 458 - INFO - Using XMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 52 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 24 - INFO - Initializing xmlRetriever\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 466 - INFO - No additional data provided. Parsing raw data only.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 147 - INFO - Function call: parse_data(api_data(<class 'str'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11485084, additional_data, XML)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 162 - INFO - api_data converted to lxml element\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 466 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 153 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 154 - INFO - Consider migrating this function to the BaseRetriever class.\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 69 - INFO - Loading target sections for section name: supplementary_material_sections\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 477 - INFO - Found 4 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x15d4d6e00>, <Element supplementary-material at 0x15d73a300>, <Element supplementary-material at 0x15d63adc0>, <Element supplementary-material at 0x15d638600>]\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 485 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x15d4d6e00>, <Element supplementary-material at 0x15d73a300>, <Element supplementary-material at 0x15d63adc0>, <Element supplementary-material at 0x15d638600>]}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 574 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref alz14208-supitem-0001: The following summary metrics are summarized by tissue contribution site and sequencing site in Figure S1 [this file] in supporting information. \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref alz14208-supitem-0002: After conditional quantile normalization (CQN) to identify expressed genes, principal component analysis (PCA) was performed to evaluate stratification among samples (Figure S2 [this file] in supporting information). When PCs were generated by tissue (one set of PCs each of DLPFC and STG) and plotted together, there was no separation by tissue contribution site (Figure S2A [this file] ), although there was some separation by sequencing site (Figure S2B [this file] ), and indeed, sequencing site was the largest source of technical variation. Sinai, Mayo, and Rush) and plotted together, there was no separation by sequencing site but only by tissue (Figure S2C [this file] ). \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref alz14208-supitem-0003: Each of the following institutions (Mayo Clinic, Rush University, Mount Sinai, Columbia University, and Emory University) coordinated the collection of these brain samples from their own networks of affiliated brain banks, cohort studies, and Alzheimer's Disease Research Centers (Table 1 [rid=alz14208-tbl-0001] , with additional details of racial category and ethnicity per cohort in Table S1 [this file] in supporting information). In addition to new donors from the studies and cohorts described below, 307 predominantly NHW (96%) individuals previously characterized in the AMP‐AD 1.0 initiative are described in Table S2 [this file] in supporting information. \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref alz14208-supitem-0004: Author disclosures are available in the supporting information [this file] . \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 263 - INFO - Extracting links from full XML content.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 301 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 309 - INFO - Content length: 225819\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 326 - INFO - Prompt messages total length: 57208 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 332 - INFO - Prompt ID: gemini-2.5-flash-0-88ea43b99f80c08dff74f8382c2b9cb54b328ce1fdb68bad81cabef0917778a3\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 351 - INFO - Requesting datasets from content using model: gemini-2.5-flash, temperature: 0, messages length: 57208 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-3fec6a46-8b54-441d-bc86-6fecd92e5b2c\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn53420676\\\",\\n    \\\"repository_reference\\\": \\\"https://doi.org/10.7303/syn53420676\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn53420674\\\",\\n    \\\"repository_reference\\\": \\\"https://doi.org/10.7303/syn53420674\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn53420673\\\",\\n    \\\"repository_reference\\\": \\\"https://doi.org/10.7303/syn53420673\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn53420677\\\",\\n    \\\"repository_reference\\\": \\\"https://doi.org/10.7303/syn53420677\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn53420672\\\",\\n    \\\"repository_reference\\\": \\\"https://doi.org/10.7303/syn53420672\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn51732482\\\",\\n    \\\"repository_reference\\\": \\\"https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn51732482\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn53352733\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn53352733\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn53420676\\\",\\n    \\\"repository_reference\\\": \\\"https://doi.org/10.7303/syn53420676\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn53420674\\\",\\n    \\\"repository_reference\\\": \\\"https://doi.org/10.7303/syn53420674\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn53420673\\\",\\n    \\\"repository_reference\\\": \\\"https://doi.org/10.7303/syn53420673\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn53420677\\\",\\n    \\\"repository_reference\\\": \\\"https://doi.org/10.7303/syn53420677\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn53420672\\\",\\n    \\\"repository_reference\\\": \\\"https://doi.org/10.7303/syn53420672\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn51732482\\\",\\n    \\\"repository_reference\\\": \\\"https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn51732482\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn53352733\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn53352733\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1758210711,\n",
      "    \"model\": \"gemini-2.5-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 75701,\n",
      "        \"completion_tokens\": 408,\n",
      "        \"total_tokens\": 82426,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 6317,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 664 - INFO - Deduplicating response with 7 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn53420676\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn53420674\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn53420673\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn53420677\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn53420672\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn51732482\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn53352733\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn53420676', 'repository_reference': 'https://doi.org/10.7303/syn53420676', 'dataset_id': 'syn53420676'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn53420676\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn53420676\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://doi.org/10.7303/syn53420676, identifier: syn53420676, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://doi.org/10.7303/syn53420676\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn53420676 for https://doi.org/10.7303/syn53420676\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn53420676 for https://doi.org/10.7303/syn53420676\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn53420676 for https://doi.org/10.7303/syn53420676\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier syn53420676 for https://doi.org/10.7303/syn53420676\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1157 - INFO - Found id_pattern match for https://doi.org/10.7303/syn53420676 in syn\\d{7,12}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://doi.org/10.7303/syn53420676\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://doi.org/10.7303/syn53420676\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn53420676\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn53420676\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://doi.org/__ID__ to value: https://doi.org/10.7303/syn53420676\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: 10\\.\\d{4,9}/[^\\s,]+ to value: https://doi.org/10.7303/syn53420676\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(10\\.\\d{4,9}/[^\\s,]+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(10\\.\\d{4,9}/[^\\s,]+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn53420676\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn53420676\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://doi.org/10.7303/syn53420676\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn53420676\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn53420676\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn53420676, synapse.org, https://doi.org/10.7303/syn53420676\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn53420676\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://doi.org/10.7303/syn53420676\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn53420676', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://doi.org/10.7303/syn53420676', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn53420674', 'repository_reference': 'https://doi.org/10.7303/syn53420674', 'dataset_id': 'syn53420674'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn53420674\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn53420674\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://doi.org/10.7303/syn53420674, identifier: syn53420674, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://doi.org/10.7303/syn53420674\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn53420674 for https://doi.org/10.7303/syn53420674\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn53420674 for https://doi.org/10.7303/syn53420674\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn53420674 for https://doi.org/10.7303/syn53420674\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier syn53420674 for https://doi.org/10.7303/syn53420674\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1157 - INFO - Found id_pattern match for https://doi.org/10.7303/syn53420674 in syn\\d{7,12}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://doi.org/10.7303/syn53420674\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://doi.org/10.7303/syn53420674\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn53420674\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn53420674\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://doi.org/__ID__ to value: https://doi.org/10.7303/syn53420674\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: 10\\.\\d{4,9}/[^\\s,]+ to value: https://doi.org/10.7303/syn53420674\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(10\\.\\d{4,9}/[^\\s,]+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(10\\.\\d{4,9}/[^\\s,]+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn53420674\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn53420674\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://doi.org/10.7303/syn53420674\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn53420674\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn53420674\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn53420674, synapse.org, https://doi.org/10.7303/syn53420674\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn53420674\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://doi.org/10.7303/syn53420674\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn53420674', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://doi.org/10.7303/syn53420674', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn53420673', 'repository_reference': 'https://doi.org/10.7303/syn53420673', 'dataset_id': 'syn53420673'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn53420673\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn53420673\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://doi.org/10.7303/syn53420673, identifier: syn53420673, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://doi.org/10.7303/syn53420673\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn53420673 for https://doi.org/10.7303/syn53420673\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn53420673 for https://doi.org/10.7303/syn53420673\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn53420673 for https://doi.org/10.7303/syn53420673\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier syn53420673 for https://doi.org/10.7303/syn53420673\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1157 - INFO - Found id_pattern match for https://doi.org/10.7303/syn53420673 in syn\\d{7,12}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://doi.org/10.7303/syn53420673\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://doi.org/10.7303/syn53420673\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn53420673\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn53420673\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://doi.org/__ID__ to value: https://doi.org/10.7303/syn53420673\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: 10\\.\\d{4,9}/[^\\s,]+ to value: https://doi.org/10.7303/syn53420673\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(10\\.\\d{4,9}/[^\\s,]+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(10\\.\\d{4,9}/[^\\s,]+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn53420673\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn53420673\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://doi.org/10.7303/syn53420673\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn53420673\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn53420673\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn53420673, synapse.org, https://doi.org/10.7303/syn53420673\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn53420673\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://doi.org/10.7303/syn53420673\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn53420673', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://doi.org/10.7303/syn53420673', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn53420677', 'repository_reference': 'https://doi.org/10.7303/syn53420677', 'dataset_id': 'syn53420677'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn53420677\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn53420677\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://doi.org/10.7303/syn53420677, identifier: syn53420677, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://doi.org/10.7303/syn53420677\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn53420677 for https://doi.org/10.7303/syn53420677\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn53420677 for https://doi.org/10.7303/syn53420677\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn53420677 for https://doi.org/10.7303/syn53420677\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier syn53420677 for https://doi.org/10.7303/syn53420677\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1157 - INFO - Found id_pattern match for https://doi.org/10.7303/syn53420677 in syn\\d{7,12}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://doi.org/10.7303/syn53420677\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://doi.org/10.7303/syn53420677\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn53420677\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn53420677\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://doi.org/__ID__ to value: https://doi.org/10.7303/syn53420677\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: 10\\.\\d{4,9}/[^\\s,]+ to value: https://doi.org/10.7303/syn53420677\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(10\\.\\d{4,9}/[^\\s,]+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(10\\.\\d{4,9}/[^\\s,]+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn53420677\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn53420677\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://doi.org/10.7303/syn53420677\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn53420677\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn53420677\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn53420677, synapse.org, https://doi.org/10.7303/syn53420677\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn53420677\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://doi.org/10.7303/syn53420677\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn53420677', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://doi.org/10.7303/syn53420677', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn53420672', 'repository_reference': 'https://doi.org/10.7303/syn53420672', 'dataset_id': 'syn53420672'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn53420672\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn53420672\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://doi.org/10.7303/syn53420672, identifier: syn53420672, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://doi.org/10.7303/syn53420672\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn53420672 for https://doi.org/10.7303/syn53420672\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn53420672 for https://doi.org/10.7303/syn53420672\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn53420672 for https://doi.org/10.7303/syn53420672\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier syn53420672 for https://doi.org/10.7303/syn53420672\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1157 - INFO - Found id_pattern match for https://doi.org/10.7303/syn53420672 in syn\\d{7,12}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://doi.org/10.7303/syn53420672\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://doi.org/10.7303/syn53420672\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn53420672\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn53420672\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://doi.org/__ID__ to value: https://doi.org/10.7303/syn53420672\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: 10\\.\\d{4,9}/[^\\s,]+ to value: https://doi.org/10.7303/syn53420672\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(10\\.\\d{4,9}/[^\\s,]+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(10\\.\\d{4,9}/[^\\s,]+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn53420672\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn53420672\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://doi.org/10.7303/syn53420672\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn53420672\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn53420672\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn53420672, synapse.org, https://doi.org/10.7303/syn53420672\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn53420672\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://doi.org/10.7303/syn53420672\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn53420672', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://doi.org/10.7303/syn53420672', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn51732482', 'repository_reference': 'https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn51732482', 'dataset_id': 'syn51732482'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn51732482\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn51732482\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn51732482, identifier: syn51732482, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn51732482\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn51732482 for https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn51732482\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn51732482 for https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn51732482\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn51732482 for https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn51732482\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn51732482 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn51732482\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn51732482\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn51732482\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn51732482\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn51732482\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn51732482\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn51732482\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn51732482\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn51732482, synapse.org, https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn51732482\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn51732482\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn51732482\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn51732482', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn51732482', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn53352733', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn53352733', 'dataset_id': 'syn53352733'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn53352733\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn53352733\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn53352733, identifier: syn53352733, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn53352733\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn53352733 for https://www.synapse.org/#!Synapse:syn53352733\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn53352733 for https://www.synapse.org/#!Synapse:syn53352733\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn53352733 for https://www.synapse.org/#!Synapse:syn53352733\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn53352733 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn53352733\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn53352733\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn53352733\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn53352733\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn53352733\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn53352733\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn53352733\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn53352733\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn53352733\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn53352733, synapse.org, https://www.synapse.org/#!Synapse:syn53352733\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn53352733\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn53352733\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn53352733', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn53352733', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 278 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn53420676', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://doi.org/10.7303/syn53420676', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn53420674', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://doi.org/10.7303/syn53420674', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn53420673', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://doi.org/10.7303/syn53420673', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn53420677', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://doi.org/10.7303/syn53420677', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn53420672', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://doi.org/10.7303/syn53420672', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn51732482', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://adknowledgeportal.synapse.org/Explore/Studies/DetailsPage/StudyDetails?Study=syn51732482', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn53352733', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn53352733', 'citation_type': 'n/a'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1191 - INFO - Fetching metadata for 7 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn53420676\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn53420676, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn53420676 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn53420676 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn53420676 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn53420676 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn53420676\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn53420676\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn53420674\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn53420674, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn53420674 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn53420674 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn53420674 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn53420674 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn53420674\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn53420674\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn53420673\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn53420673, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn53420673 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn53420673 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn53420673 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn53420673 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 3 with repo: synapse.org and accession_id: syn53420673\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn53420673\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 3: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn53420677\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn53420677, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn53420677 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn53420677 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn53420677 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn53420677 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 4 with repo: synapse.org and accession_id: syn53420677\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn53420677\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 4: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn53420672\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn53420672, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn53420672 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn53420672 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn53420672 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn53420672 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 5 with repo: synapse.org and accession_id: syn53420672\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn53420672\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 5: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn51732482\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn51732482, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn51732482 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn51732482 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn51732482 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn51732482 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 6 with repo: synapse.org and accession_id: syn51732482\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn51732482\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 6: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn53352733\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn53352733, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn53352733 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn53352733 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn53352733 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn53352733 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 7 with repo: synapse.org and accession_id: syn53352733\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn53352733\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 7: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1277 - INFO - Updated datasets len: 7\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 287 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 11, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'source_url', 'download_link',\n",
      "       'title', 'content_type', 'id', 'caption', 'description',\n",
      "       'source_section', 'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 500 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 506 - INFO - XML element classification not needed. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1012 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11485084\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1020 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11485084\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 656th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11487119)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11487119\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11487119 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 634 - INFO - Fetching data for pmc11487119\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 637 - INFO - Fetching data from scripts/exp_input/Local_fetched_data_1.parquet\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 438 - INFO - Raw xml data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11487119 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 452 - INFO - Skipping raw HTML/XML saving.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 458 - INFO - Using XMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 52 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 24 - INFO - Initializing xmlRetriever\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 466 - INFO - No additional data provided. Parsing raw data only.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 147 - INFO - Function call: parse_data(api_data(<class 'str'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11487119, additional_data, XML)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 162 - INFO - api_data converted to lxml element\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 466 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 153 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 154 - INFO - Consider migrating this function to the BaseRetriever class.\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 69 - INFO - Loading target sections for section name: supplementary_material_sections\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 477 - INFO - Found 3 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x15df8a840>, <Element supplementary-material at 0x15d9cdac0>, <Element supplementary-material at 0x15d9cfb80>]\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 485 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x15df8a840>, <Element supplementary-material at 0x15d9cdac0>, <Element supplementary-material at 0x15d9cfb80>]}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 574 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref SM1: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref SM2: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref SM3: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 263 - INFO - Extracting links from full XML content.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 301 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 309 - INFO - Content length: 142162\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 326 - INFO - Prompt messages total length: 36294 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 332 - INFO - Prompt ID: gemini-2.5-flash-0-151e45ef1ba3bf43daa46d4a73f25ac6d566047f0f16de83fe2e8e64925131ba\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 351 - INFO - Requesting datasets from content using model: gemini-2.5-flash, temperature: 0, messages length: 36294 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-da098f1a-7061-434d-a155-03bd1ff75d42\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"http://adsn.ddnetbio.com/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn22252881\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn22252881\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"http://adni.loni.usc.edu\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"PDBP\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn7170616\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn7170616\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn3606086\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn3606086\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn3218563\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn3218563\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"http://adsn.ddnetbio.com/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn22252881\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn22252881\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"http://adni.loni.usc.edu\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"PDBP\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn7170616\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn7170616\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn3606086\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn3606086\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn3218563\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn3218563\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1758210742,\n",
      "    \"model\": \"gemini-2.5-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 44934,\n",
      "        \"completion_tokens\": 325,\n",
      "        \"total_tokens\": 50072,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 4813,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 664 - INFO - Deduplicating response with 7 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn22252881\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn7170616\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn3606086\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn3218563\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'http://adsn.ddnetbio.com/', 'dataset_id': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: http://adsn.ddnetbio.com/, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: http://adsn.ddnetbio.com/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: http://adsn.ddnetbio.com/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'adsn.ddnetbio.com', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn22252881', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn22252881', 'dataset_id': 'syn22252881'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn22252881\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn22252881\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn22252881, identifier: syn22252881, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn22252881\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn22252881 for https://www.synapse.org/#!Synapse:syn22252881\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn22252881 for https://www.synapse.org/#!Synapse:syn22252881\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn22252881 for https://www.synapse.org/#!Synapse:syn22252881\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn22252881 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn22252881\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn22252881\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn22252881\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn22252881\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn22252881\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn22252881\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn22252881\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn22252881\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn22252881\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn22252881, synapse.org, https://www.synapse.org/#!Synapse:syn22252881\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn22252881\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn22252881\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn22252881', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn22252881', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'http://adni.loni.usc.edu'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: http://adni.loni.usc.edu, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: http://adni.loni.usc.edu\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: http://adni.loni.usc.edu\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'adni.loni.usc.edu', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'PDBP'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: PDBP, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: PDBP\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'pdbp', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn7170616', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn7170616', 'dataset_id': 'syn7170616'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn7170616\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn7170616\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn7170616, identifier: syn7170616, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn7170616\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn7170616 for https://www.synapse.org/#!Synapse:syn7170616\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn7170616 for https://www.synapse.org/#!Synapse:syn7170616\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn7170616 for https://www.synapse.org/#!Synapse:syn7170616\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn7170616 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn7170616\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn7170616\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn7170616\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn7170616\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn7170616\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn7170616\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn7170616\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn7170616\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn7170616\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn7170616, synapse.org, https://www.synapse.org/#!Synapse:syn7170616\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn7170616\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn7170616\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn7170616', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn7170616', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn3606086', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn3606086', 'dataset_id': 'syn3606086'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn3606086\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn3606086\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn3606086, identifier: syn3606086, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn3606086\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3606086 for https://www.synapse.org/#!Synapse:syn3606086\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn3606086 for https://www.synapse.org/#!Synapse:syn3606086\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3606086 for https://www.synapse.org/#!Synapse:syn3606086\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn3606086 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn3606086\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn3606086\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn3606086\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn3606086\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn3606086\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn3606086\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn3606086\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn3606086\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn3606086\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn3606086, synapse.org, https://www.synapse.org/#!Synapse:syn3606086\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn3606086\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn3606086\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn3606086', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn3606086', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn3218563', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn3218563', 'dataset_id': 'syn3218563'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn3218563\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn3218563\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn3218563, identifier: syn3218563, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn3218563\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3218563 for https://www.synapse.org/#!Synapse:syn3218563\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn3218563 for https://www.synapse.org/#!Synapse:syn3218563\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3218563 for https://www.synapse.org/#!Synapse:syn3218563\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn3218563 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn3218563\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn3218563\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn3218563\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn3218563\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn3218563\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn3218563\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn3218563\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn3218563\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn3218563\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn3218563, synapse.org, https://www.synapse.org/#!Synapse:syn3218563\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn3218563\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn3218563\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn3218563', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn3218563', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 278 - INFO - Augmented dataset links: [{'dataset_identifier': 'n/a', 'data_repository': 'adsn.ddnetbio.com', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn22252881', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn22252881', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'adni.loni.usc.edu', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'pdbp', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn7170616', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn7170616', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn3606086', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn3606086', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn3218563', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn3218563', 'citation_type': 'n/a'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1191 - INFO - Fetching metadata for 7 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 1: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn22252881\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn22252881, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn22252881 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn22252881 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn22252881 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn22252881 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn22252881\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn22252881\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 3: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 4: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn7170616\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn7170616, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn7170616 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn7170616 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn7170616 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn7170616 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 5 with repo: synapse.org and accession_id: syn7170616\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn7170616\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 5: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn3606086\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn3606086, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3606086 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn3606086 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3606086 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn3606086 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 6 with repo: synapse.org and accession_id: syn3606086\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn3606086\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 6: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn3218563\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn3218563, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3218563 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn3218563 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3218563 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn3218563 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 7 with repo: synapse.org and accession_id: syn3218563\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn3218563\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 7: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1277 - INFO - Updated datasets len: 7\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 287 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 10, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'source_url', 'download_link',\n",
      "       'title', 'content_type', 'id', 'caption', 'description',\n",
      "       'source_section', 'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 500 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 506 - INFO - XML element classification not needed. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1012 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11487119\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1020 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11487119\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 657th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11489794)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11489794\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11489794 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 634 - INFO - Fetching data for pmc11489794\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 637 - INFO - Fetching data from scripts/exp_input/Local_fetched_data_1.parquet\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 438 - INFO - Raw xml data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11489794 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 452 - INFO - Skipping raw HTML/XML saving.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 458 - INFO - Using XMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 52 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 24 - INFO - Initializing xmlRetriever\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 466 - INFO - No additional data provided. Parsing raw data only.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 147 - INFO - Function call: parse_data(api_data(<class 'str'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11489794, additional_data, XML)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 162 - INFO - api_data converted to lxml element\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 466 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 153 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 154 - INFO - Consider migrating this function to the BaseRetriever class.\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 69 - INFO - Loading target sections for section name: supplementary_material_sections\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 477 - INFO - Found 1 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x15de2c980>]\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 485 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x15de2c980>]}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 574 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref app1: A model evaluation schematic is shown in Figure S1 in Multimedia Appendix 1 [this file] . \n",
      "Full inclusion and exclusion criteria are shown in Table S1 in Multimedia Appendix 1 [this file] . \n",
      "Of these 840 participants, 639 (76.1%) were influenza negative and 201 (23.9%) were influenza positive (Figure S1 in Multimedia Appendix 1 [this file] ). The maximum self-reported symptom severities and wearable sensor data during ILI days –4 to +1 are shown in Table 2 [rid=table2] and Table S2 in Multimedia Appendix 1 [this file] , respectively. \n",
      "ROC curves and stratified k-fold cross-validation analyses for all models are presented in Figure 1 [rid=figure1] , with confusion matrices shown in Figure S2 in Multimedia Appendix 1 [this file] . \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 263 - INFO - Extracting links from full XML content.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 301 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 309 - INFO - Content length: 140374\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 326 - INFO - Prompt messages total length: 35847 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 332 - INFO - Prompt ID: gemini-2.5-flash-0-9c6d613e8dcb79d5e2ccbaa64847e10d37b3b47fba9aed66127a0583b204e833\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 351 - INFO - Requesting datasets from content using model: gemini-2.5-flash, temperature: 0, messages length: 35847 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-e2764fa3-c67f-49e1-9132-0f830f04d2df\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"NCT04245800\\\",\\n    \\\"repository_reference\\\": \\\"ClinicalTrials.gov\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn22803188\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn22803188/wiki/606343\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"NCT04245800\\\",\\n    \\\"repository_reference\\\": \\\"ClinicalTrials.gov\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn22803188\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn22803188/wiki/606343\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1758210760,\n",
      "    \"model\": \"gemini-2.5-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 44771,\n",
      "        \"completion_tokens\": 109,\n",
      "        \"total_tokens\": 47148,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 2268,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 664 - INFO - Deduplicating response with 2 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: NCT04245800\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn22803188\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'NCT04245800', 'repository_reference': 'ClinicalTrials.gov', 'dataset_id': 'NCT04245800'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: NCT04245800\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: NCT04245800\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: ClinicalTrials.gov, identifier: NCT04245800, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier NCT04245800 for ClinicalTrials.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier NCT04245800 for ClinicalTrials.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier NCT04245800 for ClinicalTrials.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier NCT04245800 for ClinicalTrials.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier NCT04245800 for ClinicalTrials.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier NCT04245800 for ClinicalTrials.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: ClinicalTrials.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL ClinicalTrials.gov may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: NCT04245800\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: NCT04245800, clinicaltrials.gov, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: NCT04245800\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: clinicaltrials.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'NCT04245800', 'data_repository': 'clinicaltrials.gov', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn22803188', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn22803188/wiki/606343', 'dataset_id': 'syn22803188'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn22803188\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn22803188\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn22803188/wiki/606343, identifier: syn22803188, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn22803188/wiki/606343\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn22803188 for https://www.synapse.org/#!Synapse:syn22803188/wiki/606343\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn22803188 for https://www.synapse.org/#!Synapse:syn22803188/wiki/606343\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn22803188 for https://www.synapse.org/#!Synapse:syn22803188/wiki/606343\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn22803188/wiki/606343 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn22803188/wiki/606343\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn22803188/wiki/606343\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn22803188/wiki/606343\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn22803188\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn22803188\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn22803188\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn22803188/wiki/606343\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn22803188\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn22803188\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn22803188, synapse.org, https://www.synapse.org/#!Synapse:syn22803188/wiki/606343\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn22803188\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn22803188/wiki/606343\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn22803188', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn22803188/wiki/606343', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 278 - INFO - Augmented dataset links: [{'dataset_identifier': 'NCT04245800', 'data_repository': 'clinicaltrials.gov', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn22803188', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn22803188/wiki/606343', 'citation_type': 'n/a'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1191 - INFO - Fetching metadata for 2 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: NCT04245800\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: clinicaltrials.gov, identifier: NCT04245800, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier NCT04245800 for clinicaltrials.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier NCT04245800 for clinicaltrials.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier NCT04245800 for clinicaltrials.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier NCT04245800 for clinicaltrials.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier NCT04245800 for clinicaltrials.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier NCT04245800 for clinicaltrials.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: clinicaltrials.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL clinicaltrials.gov may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for NCT04245800 in clinicaltrials.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 1 with repo: clinicaltrials.gov and accession_id: NCT04245800\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository clinicaltrials.gov unknown in Ontology. Skipping dataset page 1.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn22803188\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn22803188, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn22803188 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn22803188 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn22803188 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn22803188 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn22803188\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn22803188\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1277 - INFO - Updated datasets len: 2\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 287 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 3, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'source_url', 'download_link',\n",
      "       'title', 'content_type', 'id', 'caption', 'description',\n",
      "       'source_section', 'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 500 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 506 - INFO - XML element classification not needed. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1012 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11489794\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1020 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11489794\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 658th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11489852)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11489852\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11489852 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 634 - INFO - Fetching data for pmc11489852\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 637 - INFO - Fetching data from scripts/exp_input/Local_fetched_data_1.parquet\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 438 - INFO - Raw xml data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11489852 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 452 - INFO - Skipping raw HTML/XML saving.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 458 - INFO - Using XMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 52 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 24 - INFO - Initializing xmlRetriever\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 466 - INFO - No additional data provided. Parsing raw data only.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 147 - INFO - Function call: parse_data(api_data(<class 'str'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11489852, additional_data, XML)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 162 - INFO - api_data converted to lxml element\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 466 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 153 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 154 - INFO - Consider migrating this function to the BaseRetriever class.\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 69 - INFO - Loading target sections for section name: supplementary_material_sections\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 477 - INFO - Found 1 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x15f8f0cc0>]\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 485 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x15f8f0cc0>]}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 574 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM1: A summary of participant demographics and country distribution are available in Supplementary table 1 [this file] . \n",
      "A summary of the available variables is shown in Supplementary table 2 [this file] . \n",
      "Specific phone models used in the different participating sites are presented in Supplementary Materials 1 [this file] . \n",
      "Sage Bionetworks independently verified the variable balance between the training and validation sets as demonstrated in Supplementary table 1 [this file] . \n",
      "Terms of Use are summarized in Supplementary Materials 2 [this file] . \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 263 - INFO - Extracting links from full XML content.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 301 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 309 - INFO - Content length: 62175\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 326 - INFO - Prompt messages total length: 16297 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 332 - INFO - Prompt ID: gemini-2.5-flash-0-b23e614b5d4d49f068b11e354fa6a578f37a303a78eb7c6e472c3ab4df5ac6c8\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 351 - INFO - Requesting datasets from content using model: gemini-2.5-flash, temperature: 0, messages length: 16297 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-119f2c61-2759-40f7-b9a3-de240fe19dfd\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn31472953\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn31472953\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn31472953\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn31472953\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1758210770,\n",
      "    \"model\": \"gemini-2.5-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 19529,\n",
      "        \"completion_tokens\": 63,\n",
      "        \"total_tokens\": 20806,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 1214,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 664 - INFO - Deduplicating response with 1 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn31472953\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn31472953', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn31472953', 'dataset_id': 'syn31472953'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn31472953\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn31472953\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn31472953, identifier: syn31472953, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn31472953\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn31472953 for https://www.synapse.org/#!Synapse:syn31472953\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn31472953 for https://www.synapse.org/#!Synapse:syn31472953\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn31472953 for https://www.synapse.org/#!Synapse:syn31472953\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn31472953 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn31472953\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn31472953\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn31472953\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn31472953\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn31472953\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn31472953\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn31472953\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn31472953\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn31472953\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn31472953, synapse.org, https://www.synapse.org/#!Synapse:syn31472953\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn31472953\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn31472953\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn31472953', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn31472953', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 278 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn31472953', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn31472953', 'citation_type': 'n/a'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1191 - INFO - Fetching metadata for 1 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn31472953\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn31472953, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn31472953 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn31472953 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn31472953 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn31472953 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn31472953\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn31472953\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1277 - INFO - Updated datasets len: 1\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 287 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 2, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'source_url', 'download_link',\n",
      "       'title', 'content_type', 'id', 'caption', 'description',\n",
      "       'source_section', 'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 500 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 506 - INFO - XML element classification not needed. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1012 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11489852\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1020 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11489852\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 659th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11490488)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11490488\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11490488 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 634 - INFO - Fetching data for pmc11490488\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 637 - INFO - Fetching data from scripts/exp_input/Local_fetched_data_1.parquet\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 438 - INFO - Raw xml data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11490488 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 452 - INFO - Skipping raw HTML/XML saving.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 458 - INFO - Using XMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 52 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 24 - INFO - Initializing xmlRetriever\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 466 - INFO - No additional data provided. Parsing raw data only.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 147 - INFO - Function call: parse_data(api_data(<class 'str'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11490488, additional_data, XML)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 162 - INFO - api_data converted to lxml element\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 466 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 153 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 154 - INFO - Consider migrating this function to the BaseRetriever class.\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 69 - INFO - Loading target sections for section name: supplementary_material_sections\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 477 - INFO - Found 7 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x15deae9c0>, <Element supplementary-material at 0x15f41efc0>, <Element supplementary-material at 0x15f41f400>, <Element supplementary-material at 0x15f41f480>, <Element supplementary-material at 0x15f41ea00>, <Element supplementary-material at 0x15f41ea80>, <Element supplementary-material at 0x15f41eac0>]\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 485 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x15deae9c0>, <Element supplementary-material at 0x15f41efc0>, <Element supplementary-material at 0x15f41f400>, <Element supplementary-material at 0x15f41f480>, <Element supplementary-material at 0x15f41ea00>, <Element supplementary-material at 0x15f41ea80>, <Element supplementary-material at 0x15f41eac0>]}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 574 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM1: Consequently, we identified organ-specific circRNA clusters for brain (2436 clusters), heart (751 clusters), liver (980 clusters), lung (569 clusters), ovary (554 clusters), and spleen (701 clusters) (Supplementary Fig. 1 [this file] ). We also used other brain regions to represent the brain organ and demonstrated the expression of organ-specific circRNA clusters (Supplementary Fig. 2 [this file] ). \n",
      "Importantly, more than 55% of circRNAs formed circRNA clusters in all four cortex regions (Supplementary Fig. 3 [this file] ). \n",
      "Similarly, circRNA cluster ARHGAP26-10077-A5BS was also significantly downregulated in AD, resulting in the release of from its common sequence, which also harbors the binding site (Supplementary Fig. 4A, B [this file] ). Importantly, by comparing the expression change (log FC) of targets versus random non-target genes, we confirmed that the activity of was truly enhanced in both BM44 and BM22 cortex regions (Fig. 7D [rid=Fig7] , p-value = 1.22 x 10 , Supplementary Fig. 4C [this file] , -value = 1.08 × 10− ). \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM2: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM3: Ribosomal RNA (rRNA)-depleted RNA-seq datasets and matched miRNA-seq datasets were collected the across six brain regions including cerebellum, diencephalon, frontal cortex, occipital lobe, parietal lobe, and temporal lobe, as well as six different organs including spinal cord, heart, liver, lung, ovary, and spleen, each with two biological replicates from ENCODE (Supplementary Data 1 [this file] ). \n",
      "Within the brain, we further evaluated six distinct brain regions including the frontal cortex, cerebellum, diencephalon, occipital lobe, parietal lobe, and temporal lobe, each with two biological replicates (Supplementary Data 1 [this file] ). \n",
      "All Raw data used in this study is publicly available at ENCODE database (Experiment IDs available in supplementary Data 1 [this file] ) and Synapse via SynID: syn3159438. \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM4: Interestingly, the common sequences of both circRNA clusters are predicted to bind to 27 miRNAs (Supplementary Data 2 [this file] ). \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM5: To investigate the roles of circRNA clusters in different AD stages, we first sub-grouped AD samples into normal healthy donors (NP.1 = 1), possible AD (NP.1 = 3 or 4), and definite AD (NP.1 = 2) groups based on neuropathology categories as measured by the Consortium to Establish a Registry for Alzheimer’s Disease (CERAD) and identified specific circRNA clusters associated with AD progression based on their continuous accumulation or depletion along the severity scale (Fig. 8A [rid=Fig8] and Supplementary Data 3 [this file] ) . \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM6: CircRNA clusters differential analysis results for ENCODE and MSBB data is available in Supplementary Data 4 [this file] . \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM7: Further information on research design is available in the Nature Portfolio Reporting Summary [this file] linked to this article. \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 263 - INFO - Extracting links from full XML content.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 301 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 309 - INFO - Content length: 169190\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 326 - INFO - Prompt messages total length: 43051 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 332 - INFO - Prompt ID: gemini-2.5-flash-0-e42ca611aa8db2f0b7e2d95f443c6738af2a612fa21dd16a1ee219cefeeb5495\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 351 - INFO - Requesting datasets from content using model: gemini-2.5-flash, temperature: 0, messages length: 43051 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-9caa8749-bea1-4094-9fe9-c68e331eccd9\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"ENCODE database\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn3159438\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn3159438\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"ENCODE database\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn3159438\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn3159438\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1758210784,\n",
      "    \"model\": \"gemini-2.5-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 55223,\n",
      "        \"completion_tokens\": 90,\n",
      "        \"total_tokens\": 57050,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 1737,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 664 - INFO - Deduplicating response with 2 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn3159438\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'ENCODE database', 'dataset_id': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: ENCODE database, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: ENCODE database\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'encode database', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn3159438', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn3159438', 'dataset_id': 'syn3159438'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn3159438\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn3159438\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn3159438, identifier: syn3159438, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn3159438\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3159438 for https://www.synapse.org/#!Synapse:syn3159438\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn3159438 for https://www.synapse.org/#!Synapse:syn3159438\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3159438 for https://www.synapse.org/#!Synapse:syn3159438\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn3159438 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn3159438\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn3159438\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn3159438\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn3159438\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn3159438\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn3159438\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn3159438\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn3159438\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn3159438\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn3159438, synapse.org, https://www.synapse.org/#!Synapse:syn3159438\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn3159438\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn3159438\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn3159438', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn3159438', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 278 - INFO - Augmented dataset links: [{'dataset_identifier': 'n/a', 'data_repository': 'encode database', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn3159438', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn3159438', 'citation_type': 'n/a'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1191 - INFO - Fetching metadata for 2 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 1: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn3159438\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn3159438, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3159438 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn3159438 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3159438 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn3159438 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn3159438\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn3159438\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1277 - INFO - Updated datasets len: 2\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 287 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 9, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'source_url', 'download_link',\n",
      "       'title', 'content_type', 'id', 'caption', 'description',\n",
      "       'source_section', 'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 500 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 506 - INFO - XML element classification not needed. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1012 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11490488\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1020 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11490488\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 660th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11496171)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11496171\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11496171 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 634 - INFO - Fetching data for pmc11496171\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 637 - INFO - Fetching data from scripts/exp_input/Local_fetched_data_1.parquet\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 438 - INFO - Raw xml data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11496171 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 452 - INFO - Skipping raw HTML/XML saving.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 458 - INFO - Using XMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 52 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 24 - INFO - Initializing xmlRetriever\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 466 - INFO - No additional data provided. Parsing raw data only.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 147 - INFO - Function call: parse_data(api_data(<class 'str'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11496171, additional_data, XML)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 162 - INFO - api_data converted to lxml element\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 466 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 153 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 154 - INFO - Consider migrating this function to the BaseRetriever class.\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 69 - INFO - Loading target sections for section name: supplementary_material_sections\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 477 - INFO - Found 3 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x15f8f2880>, <Element supplementary-material at 0x15f80e1c0>, <Element supplementary-material at 0x15f80d500>]\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 485 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x15f8f2880>, <Element supplementary-material at 0x15f80e1c0>, <Element supplementary-material at 0x15f80d500>]}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 574 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref SM1: Due to residual batch effects, we reapplied the TAMPOR method ( Supplementary Figures S1A,B [this file] ). Finally, we used non-parametric bootstrap regression to adjust for covariate effects [age, sex, and post-mortem interval (PMI)] on protein abundance, which had minimal impact on the final proteomic profiles ( Supplementary Figure S1C [this file] ). \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref SM2: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref SM3: In the proteomic analysis of our cohort, ERBB2IP and LSP1 were found to be significantly upregulated in the EOAD group compared to the control group ( Supplementary Figure S3 [this file] ). \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 263 - INFO - Extracting links from full XML content.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 301 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 309 - INFO - Content length: 94641\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 326 - INFO - Prompt messages total length: 24414 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 332 - INFO - Prompt ID: gemini-2.5-flash-0-ad4018ab14de6944825763a50231ca2fbde0d98e0385f917e1348794c20b42a8\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 351 - INFO - Requesting datasets from content using model: gemini-2.5-flash, temperature: 0, messages length: 24414 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-369c6d6e-1a3c-45f9-a5fb-1112b0cb7655\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn20821165\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"SMOC1|Q9H4F8\\u20132\\\",\\n    \\\"repository_reference\\\": \\\"pdb\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"NTN1|O95631\\\",\\n    \\\"repository_reference\\\": \\\"pdb\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"COL25A1|Q9BXS0\\\",\\n    \\\"repository_reference\\\": \\\"pdb\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GMPR|P36959\\\",\\n    \\\"repository_reference\\\": \\\"pdb\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"ERBB2IP|Q96RT1\\u20138\\\",\\n    \\\"repository_reference\\\": \\\"pdb\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"AHNAK|Q09666\\\",\\n    \\\"repository_reference\\\": \\\"pdb\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"SNTA1|Q13424\\\",\\n    \\\"repository_reference\\\": \\\"pdb\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"CSK|P41240\\\",\\n    \\\"repository_reference\\\": \\\"pdb\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"ILK|A0A0A0MTH3\\\",\\n    \\\"repository_reference\\\": \\\"pdb\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"LSP1|P33241-3\\\",\\n    \\\"repository_reference\\\": \\\"pdb\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"RENBP|P51606\\\",\\n    \\\"repository_reference\\\": \\\"pdb\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn20821165\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"SMOC1|Q9H4F8\\u20132\\\",\\n    \\\"repository_reference\\\": \\\"pdb\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"NTN1|O95631\\\",\\n    \\\"repository_reference\\\": \\\"pdb\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"COL25A1|Q9BXS0\\\",\\n    \\\"repository_reference\\\": \\\"pdb\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GMPR|P36959\\\",\\n    \\\"repository_reference\\\": \\\"pdb\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"ERBB2IP|Q96RT1\\u20138\\\",\\n    \\\"repository_reference\\\": \\\"pdb\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"AHNAK|Q09666\\\",\\n    \\\"repository_reference\\\": \\\"pdb\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"SNTA1|Q13424\\\",\\n    \\\"repository_reference\\\": \\\"pdb\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"CSK|P41240\\\",\\n    \\\"repository_reference\\\": \\\"pdb\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"ILK|A0A0A0MTH3\\\",\\n    \\\"repository_reference\\\": \\\"pdb\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"LSP1|P33241-3\\\",\\n    \\\"repository_reference\\\": \\\"pdb\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"RENBP|P51606\\\",\\n    \\\"repository_reference\\\": \\\"pdb\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1758210809,\n",
      "    \"model\": \"gemini-2.5-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 30389,\n",
      "        \"completion_tokens\": 439,\n",
      "        \"total_tokens\": 34626,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 3798,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 664 - INFO - Deduplicating response with 12 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn20821165\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: SMOC1|Q9H4F8–2\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: NTN1|O95631\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: COL25A1|Q9BXS0\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: GMPR|P36959\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: ERBB2IP|Q96RT1–8\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: AHNAK|Q09666\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: SNTA1|Q13424\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: CSK|P41240\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: ILK|A0A0A0MTH3\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: LSP1|P33241-3\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: RENBP|P51606\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn20821165', 'repository_reference': 'https://www.synapse.org/', 'dataset_id': 'syn20821165'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn20821165\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn20821165\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/, identifier: syn20821165, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn20821165 for https://www.synapse.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn20821165 for https://www.synapse.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn20821165 for https://www.synapse.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/ in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn20821165\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn20821165\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn20821165\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn20821165, synapse.org, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn20821165\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn20821165', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'SMOC1|Q9H4F8–2', 'repository_reference': 'pdb', 'dataset_id': 'SMOC1|Q9H4F8–2'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: SMOC1|Q9H4F8–2\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: SMOC1|Q9H4F8–2\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: pdb, identifier: SMOC1|Q9H4F8–2, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier SMOC1|Q9H4F8–2 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier SMOC1|Q9H4F8–2 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier SMOC1|Q9H4F8–2 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier SMOC1|Q9H4F8–2 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier SMOC1|Q9H4F8–2 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: SMOC1|Q9H4F8–2\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: SMOC1|Q9H4F8–2, pdb, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: SMOC1|Q9H4F8–2\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'SMOC1|Q9H4F8–2', 'data_repository': 'pdb', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'NTN1|O95631', 'repository_reference': 'pdb', 'dataset_id': 'NTN1|O95631'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: NTN1|O95631\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: NTN1|O95631\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: pdb, identifier: NTN1|O95631, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier NTN1|O95631 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier NTN1|O95631 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier NTN1|O95631 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier NTN1|O95631 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier NTN1|O95631 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: NTN1|O95631\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: NTN1|O95631, pdb, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: NTN1|O95631\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'NTN1|O95631', 'data_repository': 'pdb', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'COL25A1|Q9BXS0', 'repository_reference': 'pdb', 'dataset_id': 'COL25A1|Q9BXS0'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: COL25A1|Q9BXS0\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: COL25A1|Q9BXS0\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: pdb, identifier: COL25A1|Q9BXS0, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier COL25A1|Q9BXS0 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier COL25A1|Q9BXS0 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier COL25A1|Q9BXS0 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier COL25A1|Q9BXS0 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier COL25A1|Q9BXS0 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: COL25A1|Q9BXS0\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: COL25A1|Q9BXS0, pdb, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: COL25A1|Q9BXS0\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'COL25A1|Q9BXS0', 'data_repository': 'pdb', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'GMPR|P36959', 'repository_reference': 'pdb', 'dataset_id': 'GMPR|P36959'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GMPR|P36959\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: GMPR|P36959\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: pdb, identifier: GMPR|P36959, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GMPR|P36959 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier GMPR|P36959 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier GMPR|P36959 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier GMPR|P36959 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier GMPR|P36959 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GMPR|P36959\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: GMPR|P36959, pdb, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: GMPR|P36959\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'GMPR|P36959', 'data_repository': 'pdb', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'ERBB2IP|Q96RT1–8', 'repository_reference': 'pdb', 'dataset_id': 'ERBB2IP|Q96RT1–8'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: ERBB2IP|Q96RT1–8\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: ERBB2IP|Q96RT1–8\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: pdb, identifier: ERBB2IP|Q96RT1–8, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier ERBB2IP|Q96RT1–8 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier ERBB2IP|Q96RT1–8 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier ERBB2IP|Q96RT1–8 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier ERBB2IP|Q96RT1–8 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier ERBB2IP|Q96RT1–8 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: ERBB2IP|Q96RT1–8\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: ERBB2IP|Q96RT1–8, pdb, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: ERBB2IP|Q96RT1–8\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'ERBB2IP|Q96RT1–8', 'data_repository': 'pdb', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'AHNAK|Q09666', 'repository_reference': 'pdb', 'dataset_id': 'AHNAK|Q09666'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: AHNAK|Q09666\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: AHNAK|Q09666\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: pdb, identifier: AHNAK|Q09666, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier AHNAK|Q09666 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier AHNAK|Q09666 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier AHNAK|Q09666 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier AHNAK|Q09666 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier AHNAK|Q09666 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: AHNAK|Q09666\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: AHNAK|Q09666, pdb, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: AHNAK|Q09666\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'AHNAK|Q09666', 'data_repository': 'pdb', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'SNTA1|Q13424', 'repository_reference': 'pdb', 'dataset_id': 'SNTA1|Q13424'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: SNTA1|Q13424\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: SNTA1|Q13424\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: pdb, identifier: SNTA1|Q13424, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier SNTA1|Q13424 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier SNTA1|Q13424 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier SNTA1|Q13424 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier SNTA1|Q13424 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier SNTA1|Q13424 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: SNTA1|Q13424\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: SNTA1|Q13424, pdb, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: SNTA1|Q13424\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'SNTA1|Q13424', 'data_repository': 'pdb', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'CSK|P41240', 'repository_reference': 'pdb', 'dataset_id': 'CSK|P41240'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: CSK|P41240\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: CSK|P41240\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: pdb, identifier: CSK|P41240, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier CSK|P41240 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier CSK|P41240 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier CSK|P41240 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier CSK|P41240 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier CSK|P41240 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: CSK|P41240\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: CSK|P41240, pdb, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: CSK|P41240\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'CSK|P41240', 'data_repository': 'pdb', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'ILK|A0A0A0MTH3', 'repository_reference': 'pdb', 'dataset_id': 'ILK|A0A0A0MTH3'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: ILK|A0A0A0MTH3\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID ILK|A0A0A0MTH3 is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: pdb, identifier: ILK|A0A0A0MTH3, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier ILK|A0A0A0MTH3 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier ILK|A0A0A0MTH3 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier ILK|A0A0A0MTH3 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier ILK|A0A0A0MTH3 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier ILK|A0A0A0MTH3 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'ILK|A0A0A0MTH3', 'data_repository': 'pdb', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'LSP1|P33241-3', 'repository_reference': 'pdb', 'dataset_id': 'LSP1|P33241-3'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: LSP1|P33241-3\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: LSP1|P33241-3\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: pdb, identifier: LSP1|P33241-3, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier LSP1|P33241-3 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier LSP1|P33241-3 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier LSP1|P33241-3 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier LSP1|P33241-3 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier LSP1|P33241-3 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: LSP1|P33241-3\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: LSP1|P33241-3, pdb, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: LSP1|P33241-3\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'LSP1|P33241-3', 'data_repository': 'pdb', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'RENBP|P51606', 'repository_reference': 'pdb', 'dataset_id': 'RENBP|P51606'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: RENBP|P51606\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: RENBP|P51606\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: pdb, identifier: RENBP|P51606, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier RENBP|P51606 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier RENBP|P51606 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier RENBP|P51606 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier RENBP|P51606 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier RENBP|P51606 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: RENBP|P51606\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: RENBP|P51606, pdb, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: RENBP|P51606\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'RENBP|P51606', 'data_repository': 'pdb', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 278 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn20821165', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'SMOC1|Q9H4F8–2', 'data_repository': 'pdb', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'NTN1|O95631', 'data_repository': 'pdb', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'COL25A1|Q9BXS0', 'data_repository': 'pdb', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'GMPR|P36959', 'data_repository': 'pdb', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'ERBB2IP|Q96RT1–8', 'data_repository': 'pdb', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'AHNAK|Q09666', 'data_repository': 'pdb', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'SNTA1|Q13424', 'data_repository': 'pdb', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'CSK|P41240', 'data_repository': 'pdb', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'ILK|A0A0A0MTH3', 'data_repository': 'pdb', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'LSP1|P33241-3', 'data_repository': 'pdb', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'RENBP|P51606', 'data_repository': 'pdb', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1191 - INFO - Fetching metadata for 12 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn20821165\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn20821165, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn20821165 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn20821165 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn20821165 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn20821165 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn20821165\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn20821165\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: SMOC1|Q9H4F8–2\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: pdb, identifier: SMOC1|Q9H4F8–2, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier SMOC1|Q9H4F8–2 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier SMOC1|Q9H4F8–2 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier SMOC1|Q9H4F8–2 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier SMOC1|Q9H4F8–2 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier SMOC1|Q9H4F8–2 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for SMOC1|Q9H4F8–2 in pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1085 - WARNING - Identifier SMOC1|Q9H4F8–2 does not match pattern for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 2 with repo: pdb and accession_id: SMOC1|Q9H4F8–2\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.rcsb.org/structure/SMOC1|Q9H4F8–2\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: NTN1|O95631\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: pdb, identifier: NTN1|O95631, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier NTN1|O95631 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier NTN1|O95631 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier NTN1|O95631 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier NTN1|O95631 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier NTN1|O95631 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for NTN1|O95631 in pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1085 - WARNING - Identifier NTN1|O95631 does not match pattern for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 3 with repo: pdb and accession_id: NTN1|O95631\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.rcsb.org/structure/NTN1|O95631\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: COL25A1|Q9BXS0\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: pdb, identifier: COL25A1|Q9BXS0, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier COL25A1|Q9BXS0 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier COL25A1|Q9BXS0 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier COL25A1|Q9BXS0 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier COL25A1|Q9BXS0 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier COL25A1|Q9BXS0 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for COL25A1|Q9BXS0 in pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1085 - WARNING - Identifier COL25A1|Q9BXS0 does not match pattern for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 4 with repo: pdb and accession_id: COL25A1|Q9BXS0\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.rcsb.org/structure/COL25A1|Q9BXS0\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: GMPR|P36959\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: pdb, identifier: GMPR|P36959, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GMPR|P36959 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier GMPR|P36959 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier GMPR|P36959 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier GMPR|P36959 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier GMPR|P36959 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for GMPR|P36959 in pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1085 - WARNING - Identifier GMPR|P36959 does not match pattern for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 5 with repo: pdb and accession_id: GMPR|P36959\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.rcsb.org/structure/GMPR|P36959\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: ERBB2IP|Q96RT1–8\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: pdb, identifier: ERBB2IP|Q96RT1–8, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier ERBB2IP|Q96RT1–8 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier ERBB2IP|Q96RT1–8 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier ERBB2IP|Q96RT1–8 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier ERBB2IP|Q96RT1–8 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier ERBB2IP|Q96RT1–8 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for ERBB2IP|Q96RT1–8 in pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1085 - WARNING - Identifier ERBB2IP|Q96RT1–8 does not match pattern for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 6 with repo: pdb and accession_id: ERBB2IP|Q96RT1–8\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.rcsb.org/structure/ERBB2IP|Q96RT1–8\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: AHNAK|Q09666\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: pdb, identifier: AHNAK|Q09666, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier AHNAK|Q09666 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier AHNAK|Q09666 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier AHNAK|Q09666 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier AHNAK|Q09666 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier AHNAK|Q09666 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for AHNAK|Q09666 in pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1085 - WARNING - Identifier AHNAK|Q09666 does not match pattern for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 7 with repo: pdb and accession_id: AHNAK|Q09666\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.rcsb.org/structure/AHNAK|Q09666\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: SNTA1|Q13424\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: pdb, identifier: SNTA1|Q13424, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier SNTA1|Q13424 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier SNTA1|Q13424 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier SNTA1|Q13424 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier SNTA1|Q13424 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier SNTA1|Q13424 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for SNTA1|Q13424 in pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1085 - WARNING - Identifier SNTA1|Q13424 does not match pattern for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 8 with repo: pdb and accession_id: SNTA1|Q13424\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.rcsb.org/structure/SNTA1|Q13424\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: CSK|P41240\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: pdb, identifier: CSK|P41240, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier CSK|P41240 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier CSK|P41240 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier CSK|P41240 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier CSK|P41240 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier CSK|P41240 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for CSK|P41240 in pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1085 - WARNING - Identifier CSK|P41240 does not match pattern for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 9 with repo: pdb and accession_id: CSK|P41240\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.rcsb.org/structure/CSK|P41240\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 10 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: ILK|A0A0A0MTH3\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: pdb, identifier: ILK|A0A0A0MTH3, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier ILK|A0A0A0MTH3 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier ILK|A0A0A0MTH3 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier ILK|A0A0A0MTH3 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier ILK|A0A0A0MTH3 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier ILK|A0A0A0MTH3 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for ILK|A0A0A0MTH3 in pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1085 - WARNING - Identifier ILK|A0A0A0MTH3 does not match pattern for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 10 with repo: pdb and accession_id: ILK|A0A0A0MTH3\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.rcsb.org/structure/ILK|A0A0A0MTH3\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 11 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: LSP1|P33241-3\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: pdb, identifier: LSP1|P33241-3, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier LSP1|P33241-3 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier LSP1|P33241-3 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier LSP1|P33241-3 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier LSP1|P33241-3 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier LSP1|P33241-3 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for LSP1|P33241-3 in pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1085 - WARNING - Identifier LSP1|P33241-3 does not match pattern for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 11 with repo: pdb and accession_id: LSP1|P33241-3\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.rcsb.org/structure/LSP1|P33241-3\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 12 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: RENBP|P51606\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: pdb, identifier: RENBP|P51606, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier RENBP|P51606 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier RENBP|P51606 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier RENBP|P51606 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier RENBP|P51606 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier RENBP|P51606 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for RENBP|P51606 in pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1085 - WARNING - Identifier RENBP|P51606 does not match pattern for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 12 with repo: pdb and accession_id: RENBP|P51606\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.rcsb.org/structure/RENBP|P51606\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1277 - INFO - Updated datasets len: 12\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 287 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 15, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'source_url', 'download_link',\n",
      "       'title', 'content_type', 'id', 'caption', 'description',\n",
      "       'source_section', 'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 500 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 506 - INFO - XML element classification not needed. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1012 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11496171\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1020 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11496171\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 661/2190 (30.18%) | Elapsed: 00:15:31 | ETA: 00:35:55\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 661th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11496294)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11496294\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11496294 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 634 - INFO - Fetching data for pmc11496294\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 637 - INFO - Fetching data from scripts/exp_input/Local_fetched_data_1.parquet\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 438 - INFO - Raw xml data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11496294 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 452 - INFO - Skipping raw HTML/XML saving.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 458 - INFO - Using XMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 52 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 24 - INFO - Initializing xmlRetriever\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 466 - INFO - No additional data provided. Parsing raw data only.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 147 - INFO - Function call: parse_data(api_data(<class 'str'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11496294, additional_data, XML)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 162 - INFO - api_data converted to lxml element\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 466 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 153 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 154 - INFO - Consider migrating this function to the BaseRetriever class.\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 69 - INFO - Loading target sections for section name: supplementary_material_sections\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 477 - INFO - Found 1 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x15f80eb80>]\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 485 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x15f80eb80>]}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 574 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref SD1: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 263 - INFO - Extracting links from full XML content.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 301 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 309 - INFO - Content length: 102536\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 326 - INFO - Prompt messages total length: 26387 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 332 - INFO - Prompt ID: gemini-2.5-flash-0-9cfd263855dc15f51f6b90cc3c0af72c77de81c2e4a5b25aaf82a33a780811b3\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 351 - INFO - Requesting datasets from content using model: gemini-2.5-flash, temperature: 0, messages length: 26387 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-1d027f27-b7c4-48f7-ad81-9be7ac3120da\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn4993293\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn4993293\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn4993293\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn4993293\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1758210836,\n",
      "    \"model\": \"gemini-2.5-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 30804,\n",
      "        \"completion_tokens\": 61,\n",
      "        \"total_tokens\": 34607,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 3742,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 664 - INFO - Deduplicating response with 1 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn4993293\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn4993293', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn4993293', 'dataset_id': 'syn4993293'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn4993293\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn4993293\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn4993293, identifier: syn4993293, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn4993293\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn4993293 for https://www.synapse.org/#!Synapse:syn4993293\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn4993293 for https://www.synapse.org/#!Synapse:syn4993293\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn4993293 for https://www.synapse.org/#!Synapse:syn4993293\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn4993293 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn4993293\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn4993293\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn4993293\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn4993293\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn4993293\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn4993293\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn4993293\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn4993293\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn4993293\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn4993293, synapse.org, https://www.synapse.org/#!Synapse:syn4993293\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn4993293\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn4993293\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn4993293', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn4993293', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 278 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn4993293', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn4993293', 'citation_type': 'n/a'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1191 - INFO - Fetching metadata for 1 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn4993293\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn4993293, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn4993293 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn4993293 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn4993293 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn4993293 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn4993293\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn4993293\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1277 - INFO - Updated datasets len: 1\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 287 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 2, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'source_url', 'download_link',\n",
      "       'title', 'content_type', 'id', 'caption', 'description',\n",
      "       'source_section', 'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 500 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 506 - INFO - XML element classification not needed. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1012 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11496294\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1020 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11496294\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 662th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11497642)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11497642\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11497642 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 634 - INFO - Fetching data for pmc11497642\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 637 - INFO - Fetching data from scripts/exp_input/Local_fetched_data_1.parquet\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 438 - INFO - Raw html data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11497642 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 452 - INFO - Skipping raw HTML/XML saving.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 482 - INFO - Using HTMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 52 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 82 - INFO - Initializing htmlRetriever\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 350 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11497642, additional_data, html)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 448 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC11497642)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 550 - INFO - Extracted 6 unique supplementary material links from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 560 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#clp_a'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for clp_a: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 12 <a> elements with href='#alz14121-supitem-0001'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 3443) for alz14121-supitem-0001: We also systematically examined the shared and unique common variants and genes between EOAD and LOAD at different significance levels ( Supplementary Methods [href=this file] ).\n",
      "We conducted enrichment analyses of cell types and pathways using all candidate AD genes to gain functional insights ( Supplementary Methods [href=this file] ).\n",
      "To this end, we inquired into two databases, the Alzdata and Agora. 31 [href=#alz14121-bib-0031] , 32 [href=#alz14121-bib-0032] The detailed descriptions of data and analytical models can be found in Supplementary Methods [href=this file] . We assigned two or three points to each gene within each category, depending on their supporting evidence ( Supplementary Methods [href=this file] ).\n",
      "The gene‐level analysis showed a similar trend, with the shared elements continuously decreasing as the threshold was relaxed (Figure S1 [href=this file] and Table S4 [href=#alz14121-supitem-0002] ).\n",
      "These three signals showed statistical significance in both Chinese cohorts (discovery < 1 × 10 and replication < 0.05), and no gender bias was found in the stratified analysis (Figure S2 [href=this file] ).\n",
      "Furthermore, we performed gene‐based association tests, identifying 10 significant genes after Bonferroni correction (Table S10 [href=#alz14121-supitem-0002] and Figure S1 [href=this file] ).\n",
      "The expression levels of seven candidate genes also showed significant associations with AD hallmarks in human post‐mortem samples, including distribution and severity of neurofibrillary tangles ( , and ), neuritic plaque density ( , and ), and consensus cognitive diagnosis at time of death ( and ), although these genes except have not been linked to CSF diagnostic biomarkers in vivo. 42 [href=#alz14121-bib-0042] , 43 [href=#alz14121-bib-0043] Genes associated with AD hallmarks could be grouped into four clusters, characterized by their involvement in lipid metabolism, ribosomal ribonucleic acid, epidermal growth factor receptor (EGFR) binding, and mitogen‐activated protein kinase signaling (Figure S3 [href=this file] ).\n",
      "Furthermore, is expressed predominantly in astrocytes (Figure S4 [href=this file] ), and the gene was found to be overexpressed in multiple brain regions affected by AD (Figure 4 [href=#alz14121-fig-0004] ). Furthermore, could interact with , , and (Figure S5 [href=this file] ) and had favorable therapeutic potential and safety profiles, indicating that it is a promising drug target for AD. , although mapping to the region, outscored in our gene prioritization because was significant in brain eQTL mapping, and its expression level was correlated significantly with AD neuropathology, such as the distribution and severity of neurofibrillary tangles. has also been implicated in lipid metabolism traits (Table S15 [href=#alz14121-supitem-0002] ).\n",
      "In addition, the eQTL in the dorsolateral prefrontal cortex, rs5997690, is in linkage disequilibrium (LD, = 0.59 in our sample) with rs75680863, and it is also nominally associated with AD in our sample (OR = 0.73, = 4.48 × 10 ). may be involved in vascular mechanisms as it is expressed in endothelial cells (Figure S4 [href=this file] ), and its variants have been reported to be associated with multiple cerebrovascular disorders (Table S16 [href=#alz14121-supitem-0002] ).\n",
      "Moreover, the results remained stable in the leave‐one‐out analysis and were not biased by instruments within the region (Figure S6 [href=this file] ).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 22 <a> elements with href='#alz14121-supitem-0002'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 5293) for alz14121-supitem-0002: We conducted separate GWASs for Chinese EOAD and LOAD using data from the Stage 1 discovery sample (Tables S1, S2 [href=this file] ).\n",
      "Specifically, the shared proportion of variants was 100% at < 1 × 10 , 92% at < 5 × 10 , 73% at < 1 × 10 , 38% at < 5 × 10 , and 26% at < 1 × 10 (Table S3 [href=this file] ). The gene‐level analysis showed a similar trend, with the shared elements continuously decreasing as the threshold was relaxed (Figure S1 [href=#alz14121-supitem-0001] and Table S4 [href=this file] ). To gain mechanistic insights, we further conducted gene‐set enrichment analysis and found the enriched ontologies of shared genes between EOAD and LOAD mainly involved in lipid metabolism (Table S5 [href=this file] ).\n",
      "A sensitivity analysis combining EOAD and LOAD individuals yielded results similar to those of the meta‐analysis (Table S6 [href=this file] ).\n",
      "It was found that 31% of all suggestive variants ( < 1 × 10 ) identified in Stage 1 could be validated in an independent Chinese sample after Bonferroni correction (Stage 2A, Table S7 [href=this file] ). In contrast to Stage 2A, the replication in European samples (Stage 2B) failed to identify any Bonferroni‐corrected significant associations, with the exception of variants in the region (Table S8 [href=this file] ).\n",
      ": This table displays the results of Stage 1 + 2A meta‐analysis, and the full results are available in Table S7 [href=this file] . In contrast with Stage 1 + 2A, Stage 1 + 2B meta‐analysis did not yield any new significant locus other than , and full results can be found in Table S8 [href=this file] .\n",
      "These three methods identified 79 unique candidate AD genes in total (Table S9 [href=this file] ). Furthermore, we performed gene‐based association tests, identifying 10 significant genes after Bonferroni correction (Table S10 [href=this file] and Figure S1 [href=#alz14121-supitem-0001] ). Of interest, these ontologies were consistent with those enriched from the shared genes between EOAD and LOAD (Table S5 [href=this file] ).\n",
      "Only genes with a priority score ≥5 are shown in this figure, and the full results can be found in Table S13 [href=this file] . †Gene expression in human post‐mortem dorsolateral prefrontal cortex is correlated with the distribution and severity of neurofibrillary tangles (BRAAK), neuritic plaque density (CERAD), or final consensus cognitive diagnosis (COGDX) in the Agora database. ‡Gene expression is correlated with AD pathology in Aβ/tau line AD mouse models according to the AlzData database.\n",
      "We observed that 23 genes had meaningful differential expression between AD and healthy controls after multiple testing adjustments (Table S11 [href=this file] ). In addition, the expression levels of 19 genes were correlated with amyloid beta (Aβ) or tau pathology in mouse models (Table S12 [href=this file] ).\n",
      "This score ranged from 0 to 10 points (Table S13 [href=this file] ).\n",
      "encodes a receptor for epidermal growth factor, which participates in early brain development and brain atrophy. 44 [href=#alz14121-bib-0044] Variants within have been reported to be associated with AD and related dementias, 29 [href=#alz14121-bib-0029] as well as deaths due to amyloidosis and brain diseases in European studies (Table S14 [href=this file] ), although none of them were related to the lead variant in our Chinese AD GWAS, rs1815157. Furthermore, could interact with , , and (Figure S5 [href=#alz14121-supitem-0001] ) and had favorable therapeutic potential and safety profiles, indicating that it is a promising drug target for AD. , although mapping to the region, outscored in our gene prioritization because was significant in brain eQTL mapping, and its expression level was correlated significantly with AD neuropathology, such as the distribution and severity of neurofibrillary tangles. has also been implicated in lipid metabolism traits (Table S15 [href=this file] ).\n",
      "In addition, the eQTL in the dorsolateral prefrontal cortex, rs5997690, is in linkage disequilibrium (LD, = 0.59 in our sample) with rs75680863, and it is also nominally associated with AD in our sample (OR = 0.73, = 4.48 × 10 ). may be involved in vascular mechanisms as it is expressed in endothelial cells (Figure S4 [href=#alz14121-supitem-0001] ), and its variants have been reported to be associated with multiple cerebrovascular disorders (Table S16 [href=this file] ).\n",
      "Finally, in order to provide causal insights into AD etiology in Asian populations, we performed phenome‐wide MR (or phe‐MR) using exposures from the BioBank Japan Project (Tables S17–S19 [href=this file] ). The effect direction was consistent in all sensitivity analyses, and no heterogeneity or pleiotropy was observed (Table S17 [href=this file] ).\n",
      "Variations in allele frequencies among different populations could account for the observed discrepancies to some extent, as genetic analyses tend to identify significant variants that have higher allele frequencies within the specific population. 46 [href=#alz14121-bib-0046] Our results support this hypothesis because 12% of lead variants of the established loci identified from European studies were rare in Chinese (Table S1 [href=this file] ) and most of the novel loci identified in this study also exhibit different MAFs than those in other populations.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 1 <a> elements with href='#alz14121-supitem-0003'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 82) for alz14121-supitem-0003: Author disclosures are available in the supporting information [href=this file] .\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#alz14121-sec-0240'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for alz14121-sec-0240: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 368 - INFO - Extracting links from full HTML content.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 301 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 309 - INFO - Content length: 186977\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 326 - INFO - Prompt messages total length: 47498 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 332 - INFO - Prompt ID: gemini-2.5-flash-0-109fd53819cc951da0a6ae0371829f136c1017a51b5322750b90db4ea9525abc\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 351 - INFO - Requesting datasets from content using model: gemini-2.5-flash, temperature: 0, messages length: 47498 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-90f1bd0b-f14b-4a74-9cfa-5e720c0a7a75\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.57718/agora-adknowledgeportal\\\",\\n    \\\"repository_reference\\\": \\\"https://doi.org/10.57718/agora-adknowledgeportal\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn13363443\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn13363443\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.57718/agora-adknowledgeportal\\\",\\n    \\\"repository_reference\\\": \\\"https://doi.org/10.57718/agora-adknowledgeportal\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn13363443\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn13363443\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1758210978,\n",
      "    \"model\": \"gemini-2.5-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 67405,\n",
      "        \"completion_tokens\": 121,\n",
      "        \"total_tokens\": 70524,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 2998,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 664 - INFO - Deduplicating response with 2 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: 10.57718/agora-adknowledgeportal\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn13363443\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': '10.57718/agora-adknowledgeportal', 'repository_reference': 'https://doi.org/10.57718/agora-adknowledgeportal', 'dataset_id': '10.57718/agora-adknowledgeportal'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: 10\\.\\d{4,9}/[^\\s,]+ to value: 10.57718/agora-adknowledgeportal\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 607 - INFO - Setting dataset_webpage to DOI URL value: https://doi.org/10.57718/agora-adknowledgeportal\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: 10.57718/agora-adknowledgeportal\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: doi.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://doi.org/10.57718/agora-adknowledgeportal, identifier: 10.57718/agora-adknowledgeportal, dataset_page: https://doi.org/10.57718/agora-adknowledgeportal\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://doi.org/10.57718/agora-adknowledgeportal\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.57718/agora-adknowledgeportal for https://doi.org/10.57718/agora-adknowledgeportal\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier 10.57718/agora-adknowledgeportal for https://doi.org/10.57718/agora-adknowledgeportal\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier 10.57718/agora-adknowledgeportal for https://doi.org/10.57718/agora-adknowledgeportal\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier 10.57718/agora-adknowledgeportal for https://doi.org/10.57718/agora-adknowledgeportal\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier 10.57718/agora-adknowledgeportal for https://doi.org/10.57718/agora-adknowledgeportal\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1157 - INFO - Found id_pattern match for https://doi.org/10.57718/agora-adknowledgeportal in 10\\.\\d{4,9}/[^\\s,]+\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://doi.org/__ID__ to value: https://doi.org/10.57718/agora-adknowledgeportal\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: 10\\.\\d{4,9}/[^\\s,]+ to value: https://doi.org/10.57718/agora-adknowledgeportal\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(10\\.\\d{4,9}/[^\\s,]+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(10\\.\\d{4,9}/[^\\s,]+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: 10.57718/agora-adknowledgeportal\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: 10\\.\\d{4,9}/[^\\s,]+ to value: 10.57718/agora-adknowledgeportal\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: 10.57718/agora-adknowledgeportal\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://doi.org/10.57718/agora-adknowledgeportal\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: 10.57718/agora-adknowledgeportal\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: 10.57718/agora-adknowledgeportal\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: 10.57718/agora-adknowledgeportal, doi.org, https://doi.org/10.57718/agora-adknowledgeportal\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: 10.57718/agora-adknowledgeportal\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: doi.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://doi.org/10.57718/agora-adknowledgeportal\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': '10.57718/agora-adknowledgeportal', 'data_repository': 'doi.org', 'dataset_webpage': 'https://doi.org/10.57718/agora-adknowledgeportal', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn13363443', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn13363443', 'dataset_id': 'syn13363443'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn13363443\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn13363443\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn13363443, identifier: syn13363443, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn13363443\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn13363443 for https://www.synapse.org/#!Synapse:syn13363443\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn13363443 for https://www.synapse.org/#!Synapse:syn13363443\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn13363443 for https://www.synapse.org/#!Synapse:syn13363443\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn13363443 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn13363443\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn13363443\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn13363443\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn13363443\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn13363443\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn13363443\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn13363443\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn13363443\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn13363443\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn13363443, synapse.org, https://www.synapse.org/#!Synapse:syn13363443\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn13363443\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn13363443\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn13363443', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn13363443', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 378 - INFO - Augmented dataset links: [{'dataset_identifier': '10.57718/agora-adknowledgeportal', 'data_repository': 'doi.org', 'dataset_webpage': 'https://doi.org/10.57718/agora-adknowledgeportal', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn13363443', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn13363443', 'citation_type': 'n/a'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1191 - INFO - Fetching metadata for 2 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: 10.57718/agora-adknowledgeportal\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: doi.org, identifier: 10.57718/agora-adknowledgeportal, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.57718/agora-adknowledgeportal for doi.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier 10.57718/agora-adknowledgeportal for doi.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier 10.57718/agora-adknowledgeportal for doi.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier 10.57718/agora-adknowledgeportal for doi.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: doi.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for 10.57718/agora-adknowledgeportal in doi.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 1 with repo: doi.org and accession_id: 10.57718/agora-adknowledgeportal\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://doi.org/10.57718/agora-adknowledgeportal\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn13363443\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn13363443, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn13363443 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn13363443 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn13363443 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn13363443 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn13363443\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn13363443\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1277 - INFO - Updated datasets len: 2\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 414 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 8, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'title', 'file_info',\n",
      "       'description', 'source_section', 'section_class', 'download_link',\n",
      "       'file_extension', 'a_attr_href', 'a_attr_class',\n",
      "       'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 173 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 183 - WARNING - No publication title found in the HTML data.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 643 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 494 - INFO - Parsed data extraction completed. Elements collected: 6\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 500 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 510 - INFO - HTML element classification not supported. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1012 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11497642\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1020 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11497642\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 663th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11502818)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11502818\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11502818 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 634 - INFO - Fetching data for pmc11502818\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 637 - INFO - Fetching data from scripts/exp_input/Local_fetched_data_1.parquet\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 438 - INFO - Raw xml data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11502818 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 452 - INFO - Skipping raw HTML/XML saving.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 458 - INFO - Using XMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 52 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 24 - INFO - Initializing xmlRetriever\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 466 - INFO - No additional data provided. Parsing raw data only.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 147 - INFO - Function call: parse_data(api_data(<class 'str'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11502818, additional_data, XML)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 162 - INFO - api_data converted to lxml element\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 466 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 153 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 154 - INFO - Consider migrating this function to the BaseRetriever class.\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 69 - INFO - Loading target sections for section name: supplementary_material_sections\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 477 - INFO - Found 3 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x15ec21900>, <Element supplementary-material at 0x15f8cc8c0>, <Element supplementary-material at 0x15f8cfb00>]\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 485 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x15ec21900>, <Element supplementary-material at 0x15f8cc8c0>, <Element supplementary-material at 0x15f8cfb00>]}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 574 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM1: In this study, the effects of five RNA-seq normalization methods (RLE, TMM, GeTMM, TPM and FPKM) and covariate adjustment on reconstructing context-specific personalized GEMs were systematically investigated using two most popular mapping algorithms for human metabolism, iMAT and INIT (Supplementary Table 1 [this file] ). \n",
      "Although the number of active reactions in the personalized metabolic models reconstructed by INIT is low compared to the iMAT-based models, the clustering behavior of between-sample and within-sample normalization methods was found to be similar (Supplementary Fig. 1 [this file] ). \n",
      "Within-sample normalization methods have low accuracy based on the metabolite-based predictions for both mapping algorithms (Supplementary Fig. 3 [this file] ). \n",
      "We further assessed the five normalization methods and their covariate-adjusted versions by comparing the list of genes in the significantly affected reactions and the list of genes known to be associated with AD (Supplementary Table 2 [this file] ) or LUAD phenotypes (Supplementary Table 3 [this file] ). \n",
      "We uncovered a considerably higher number of AD-related terms with the covariate-adjusted version of TMM through the analysis of the INIT models even if the use of this dataset with the iMAT algorithm only allowed the significant enrichment of “Alzheimer’s Disease” (Supplementary Fig. 4a [this file] ). \n",
      "On the other hand, the analysis of the covariate-adjusted datasets by reconstructing the AD-specific INIT models demonstrated the separate clustering of between- and within-sample normalization methods (Supplementary Fig. 4b [this file] ). \n",
      "For the AD dataset, FPKM and TPM normalization methods allowed the prediction of more AD-related experimentally confirmed genes (Supplementary Tables 4 [this file] , 5 [this file] ). Consistent results were obtained through the INIT model-based predictions (Supplementary Fig. 4c, d [this file] ). On the contrary, the number of true positives was found to be close to each other regardless of the selected normalization methods for the LUAD dataset due to the presence of fewer disease-associated metabolic genes (Supplementary Table 6 [this file] ) in the validation set (Fig. 4c [rid=Fig4] ). \n",
      "Metabolite-based predictions (Supplementary Fig. 3 [this file] ) also showed that FPKM and TPM had low accuracy compared to the between-sample normalization methods. \n",
      "While the GEMs reconstructed by iMAT and INIT are completely different in terms of binary active reaction matrix similarity, general inference about the effect of RNA-seq normalization methods on metabolic models is the same (Supplementary Fig. 2 [this file] ). \n",
      "Moreover, disease-related gene-based analyses demonstrated in Figs. 3 [rid=Fig3] – 4 [rid=Fig4] , and a supportive analysis based on the comparison of ROSMAP metabolome data analysis and metabolites included in the significantly affected reactions (Supplementary Fig. 3 [this file] ) show that between-sample normalization methods are preferable to within-sample normalization methods. \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM2: Significantly affected reactions and overrepresented pathways are listed in Supplementary Data 1 [this file] for ROSMAP, and Supplementary Data 2 [rid=MOESM3] for LUAD for each normalization method. \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM3: Significantly affected reactions and overrepresented pathways are listed in Supplementary Data 1 [rid=MOESM2] for ROSMAP, and Supplementary Data 2 [this file] for LUAD for each normalization method. \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 263 - INFO - Extracting links from full XML content.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 301 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 309 - INFO - Content length: 156646\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 326 - INFO - Prompt messages total length: 39915 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 332 - INFO - Prompt ID: gemini-2.5-flash-0-4a2a2dba59959ea43f3457ff06ce1e4fb516f7568d11b18d15e3cf4acf72782b\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 351 - INFO - Requesting datasets from content using model: gemini-2.5-flash, temperature: 0, messages length: 39915 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-5defcf76-45b4-4e7a-80df-576ca00c82b6\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn17024112\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/ampad\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"gdc.cancer.gov\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn17024112\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/ampad\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"gdc.cancer.gov\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1758210987,\n",
      "    \"model\": \"gemini-2.5-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 49050,\n",
      "        \"completion_tokens\": 85,\n",
      "        \"total_tokens\": 50047,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 912,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 664 - INFO - Deduplicating response with 2 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn17024112\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn17024112', 'repository_reference': 'https://www.synapse.org/ampad', 'dataset_id': 'syn17024112'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn17024112\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn17024112\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/ampad, identifier: syn17024112, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/ampad\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn17024112 for https://www.synapse.org/ampad\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn17024112 for https://www.synapse.org/ampad\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn17024112 for https://www.synapse.org/ampad\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/ampad in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn17024112\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn17024112\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn17024112\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn17024112, synapse.org, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn17024112\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn17024112', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'gdc.cancer.gov', 'dataset_id': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: gdc.cancer.gov, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: gdc.cancer.gov\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 644 - INFO - Dataset ID is 'n/a' and repository name from prompt\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 541 - INFO - Skipping dataset due to missing ID, repository, dataset page: {'dataset_identifier': 'n/a', 'repository_reference': 'gdc.cancer.gov', 'dataset_id': 'n/a'}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 278 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn17024112', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1191 - INFO - Fetching metadata for 1 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn17024112\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn17024112, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn17024112 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn17024112 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn17024112 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn17024112 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn17024112\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn17024112\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1277 - INFO - Updated datasets len: 1\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 287 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 4, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'source_url', 'download_link',\n",
      "       'title', 'content_type', 'id', 'caption', 'description',\n",
      "       'source_section', 'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 500 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 506 - INFO - XML element classification not needed. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1012 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11502818\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1020 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11502818\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 664th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11507780)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11507780\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11507780 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 634 - INFO - Fetching data for pmc11507780\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 637 - INFO - Fetching data from scripts/exp_input/Local_fetched_data_1.parquet\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 438 - INFO - Raw html data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11507780 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 452 - INFO - Skipping raw HTML/XML saving.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 482 - INFO - Using HTMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 52 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 82 - INFO - Initializing htmlRetriever\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 350 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11507780, additional_data, html)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 448 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC11507780)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 550 - INFO - Extracted 5 unique supplementary material links from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 560 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#clp_a'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for clp_a: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#S3'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for S3: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#S3'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for S3: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 2 <a> elements with href='#SD1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 517) for SD1: We then determined how many genes positively correlate with RIN in all four datasets and found 2,512 genes that commonly overlap (see Supplemental Data [href=this file] for genes that commonly correlate with RIN and PMI in the four datasets). To understand whether there are common biological processes that characterize the sets of genes that correlate with RIN and PMI, we performed ontology analysis on the genes in common ( Table 3 [href=#T5] ; See Supplemental Data [href=this file] for full ontology analysis).\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 368 - INFO - Extracting links from full HTML content.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 301 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 309 - INFO - Content length: 106424\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 326 - INFO - Prompt messages total length: 27359 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 332 - INFO - Prompt ID: gemini-2.5-flash-0-ba8f2909ef8be79bfd4e9cea7445518782ef31316af0c99d7b9277b1e8b8745e\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 351 - INFO - Requesting datasets from content using model: gemini-2.5-flash, temperature: 0, messages length: 27359 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-6abd099e-31e6-40eb-be0d-62d6c0e1cf62\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn3219045\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn3219045\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.1101/2024.10.14.618253\\\",\\n    \\\"repository_reference\\\": \\\"https://doi.org/10.1101/2024.10.14.618253\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn3219045\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn3219045\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.1101/2024.10.14.618253\\\",\\n    \\\"repository_reference\\\": \\\"https://doi.org/10.1101/2024.10.14.618253\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1758211002,\n",
      "    \"model\": \"gemini-2.5-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 36170,\n",
      "        \"completion_tokens\": 141,\n",
      "        \"total_tokens\": 38518,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 2207,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 664 - INFO - Deduplicating response with 2 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn3219045\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: 10.1101/2024.10.14.618253\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn3219045', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn3219045', 'dataset_id': 'syn3219045'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn3219045\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn3219045\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn3219045, identifier: syn3219045, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn3219045\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3219045 for https://www.synapse.org/#!Synapse:syn3219045\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn3219045 for https://www.synapse.org/#!Synapse:syn3219045\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3219045 for https://www.synapse.org/#!Synapse:syn3219045\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn3219045 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn3219045\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn3219045\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn3219045\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn3219045\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn3219045\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn3219045\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn3219045\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn3219045\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn3219045\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn3219045, synapse.org, https://www.synapse.org/#!Synapse:syn3219045\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn3219045\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn3219045\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn3219045', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn3219045', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': '10.1101/2024.10.14.618253', 'repository_reference': 'https://doi.org/10.1101/2024.10.14.618253', 'dataset_id': '10.1101/2024.10.14.618253'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: 10\\.\\d{4,9}/[^\\s,]+ to value: 10.1101/2024.10.14.618253\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 607 - INFO - Setting dataset_webpage to DOI URL value: https://doi.org/10.1101/2024.10.14.618253\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: 10.1101/2024.10.14.618253\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: doi.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://doi.org/10.1101/2024.10.14.618253, identifier: 10.1101/2024.10.14.618253, dataset_page: https://doi.org/10.1101/2024.10.14.618253\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://doi.org/10.1101/2024.10.14.618253\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.1101/2024.10.14.618253 for https://doi.org/10.1101/2024.10.14.618253\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier 10.1101/2024.10.14.618253 for https://doi.org/10.1101/2024.10.14.618253\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier 10.1101/2024.10.14.618253 for https://doi.org/10.1101/2024.10.14.618253\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier 10.1101/2024.10.14.618253 for https://doi.org/10.1101/2024.10.14.618253\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier 10.1101/2024.10.14.618253 for https://doi.org/10.1101/2024.10.14.618253\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1157 - INFO - Found id_pattern match for https://doi.org/10.1101/2024.10.14.618253 in 10\\.\\d{4,9}/[^\\s,]+\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://doi.org/__ID__ to value: https://doi.org/10.1101/2024.10.14.618253\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: 10\\.\\d{4,9}/[^\\s,]+ to value: https://doi.org/10.1101/2024.10.14.618253\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(10\\.\\d{4,9}/[^\\s,]+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(10\\.\\d{4,9}/[^\\s,]+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: 10.1101/2024.10.14.618253\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: 10\\.\\d{4,9}/[^\\s,]+ to value: 10.1101/2024.10.14.618253\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: 10.1101/2024.10.14.618253\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://doi.org/10.1101/2024.10.14.618253\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: 10.1101/2024.10.14.618253\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: 10.1101/2024.10.14.618253\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: 10.1101/2024.10.14.618253, doi.org, https://doi.org/10.1101/2024.10.14.618253\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: 10.1101/2024.10.14.618253\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: doi.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://doi.org/10.1101/2024.10.14.618253\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': '10.1101/2024.10.14.618253', 'data_repository': 'doi.org', 'dataset_webpage': 'https://doi.org/10.1101/2024.10.14.618253', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 378 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn3219045', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn3219045', 'citation_type': 'n/a'}, {'dataset_identifier': '10.1101/2024.10.14.618253', 'data_repository': 'doi.org', 'dataset_webpage': 'https://doi.org/10.1101/2024.10.14.618253', 'citation_type': 'n/a'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1191 - INFO - Fetching metadata for 2 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn3219045\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn3219045, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3219045 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn3219045 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3219045 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn3219045 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn3219045\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn3219045\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: 10.1101/2024.10.14.618253\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: doi.org, identifier: 10.1101/2024.10.14.618253, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.1101/2024.10.14.618253 for doi.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier 10.1101/2024.10.14.618253 for doi.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier 10.1101/2024.10.14.618253 for doi.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier 10.1101/2024.10.14.618253 for doi.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: doi.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for 10.1101/2024.10.14.618253 in doi.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 2 with repo: doi.org and accession_id: 10.1101/2024.10.14.618253\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://doi.org/10.1101/2024.10.14.618253\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1277 - INFO - Updated datasets len: 2\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 414 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 7, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'title', 'file_info',\n",
      "       'description', 'source_section', 'section_class', 'download_link',\n",
      "       'file_extension', 'a_attr_href', 'a_attr_class',\n",
      "       'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 173 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 183 - WARNING - No publication title found in the HTML data.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 643 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 494 - INFO - Parsed data extraction completed. Elements collected: 4\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 500 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 510 - INFO - HTML element classification not supported. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1012 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11507780\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1020 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11507780\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 665th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11514281)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11514281\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11514281 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 634 - INFO - Fetching data for pmc11514281\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 637 - INFO - Fetching data from scripts/exp_input/Local_fetched_data_1.parquet\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 438 - INFO - Raw xml data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11514281 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 452 - INFO - Skipping raw HTML/XML saving.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 458 - INFO - Using XMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 52 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 24 - INFO - Initializing xmlRetriever\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 466 - INFO - No additional data provided. Parsing raw data only.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 147 - INFO - Function call: parse_data(api_data(<class 'str'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11514281, additional_data, XML)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 162 - INFO - api_data converted to lxml element\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 466 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 153 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 154 - INFO - Consider migrating this function to the BaseRetriever class.\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 69 - INFO - Loading target sections for section name: supplementary_material_sections\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 477 - INFO - Found 1 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x15f087d00>]\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 485 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x15f087d00>]}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 574 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM1: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 263 - INFO - Extracting links from full XML content.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 301 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 309 - INFO - Content length: 227328\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 326 - INFO - Prompt messages total length: 57585 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 332 - INFO - Prompt ID: gemini-2.5-flash-0-d1e84dbfd93d9bc3a6660753ad2cd503f197cfc4270a0677d9bf111e64d66ed8\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 351 - INFO - Requesting datasets from content using model: gemini-2.5-flash, temperature: 0, messages length: 57585 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-51ab67b2-8a9b-40da-87c8-50c188f022c7\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"ACDC\\\",\\n    \\\"repository_reference\\\": \\\"https://www.creatis.insa-lyon.fr/Challenge/acdc/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn3193805\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn3193805\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Kvasir-SEG\\\",\\n    \\\"repository_reference\\\": \\\"https://kvasir.no/datasets/kvasir-seg/\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"ACDC\\\",\\n    \\\"repository_reference\\\": \\\"https://www.creatis.insa-lyon.fr/Challenge/acdc/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn3193805\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn3193805\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Kvasir-SEG\\\",\\n    \\\"repository_reference\\\": \\\"https://kvasir.no/datasets/kvasir-seg/\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1758211022,\n",
      "    \"model\": \"gemini-2.5-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 72051,\n",
      "        \"completion_tokens\": 148,\n",
      "        \"total_tokens\": 74810,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 2611,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 664 - INFO - Deduplicating response with 3 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: ACDC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn3193805\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: Kvasir-SEG\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'ACDC', 'repository_reference': 'https://www.creatis.insa-lyon.fr/Challenge/acdc/', 'dataset_id': 'ACDC'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: ACDC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID ACDC is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.creatis.insa-lyon.fr/Challenge/acdc/, identifier: ACDC, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.creatis.insa-lyon.fr/Challenge/acdc/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier ACDC for https://www.creatis.insa-lyon.fr/Challenge/acdc/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier ACDC for https://www.creatis.insa-lyon.fr/Challenge/acdc/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier ACDC for https://www.creatis.insa-lyon.fr/Challenge/acdc/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier ACDC for https://www.creatis.insa-lyon.fr/Challenge/acdc/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier ACDC for https://www.creatis.insa-lyon.fr/Challenge/acdc/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier ACDC for https://www.creatis.insa-lyon.fr/Challenge/acdc/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://www.creatis.insa-lyon.fr/Challenge/acdc/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'ACDC', 'data_repository': 'www.creatis.insa-lyon.fr', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn3193805', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn3193805', 'dataset_id': 'syn3193805'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn3193805\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn3193805\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn3193805, identifier: syn3193805, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn3193805\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3193805 for https://www.synapse.org/#!Synapse:syn3193805\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn3193805 for https://www.synapse.org/#!Synapse:syn3193805\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3193805 for https://www.synapse.org/#!Synapse:syn3193805\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn3193805 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn3193805\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn3193805\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn3193805\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn3193805\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn3193805\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn3193805\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn3193805\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn3193805\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn3193805\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn3193805, synapse.org, https://www.synapse.org/#!Synapse:syn3193805\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn3193805\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn3193805\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn3193805', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn3193805', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'Kvasir-SEG', 'repository_reference': 'https://kvasir.no/datasets/kvasir-seg/', 'dataset_id': 'Kvasir-SEG'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: Kvasir-SEG\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID Kvasir-SEG is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://kvasir.no/datasets/kvasir-seg/, identifier: Kvasir-SEG, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://kvasir.no/datasets/kvasir-seg/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Kvasir-SEG for https://kvasir.no/datasets/kvasir-seg/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier Kvasir-SEG for https://kvasir.no/datasets/kvasir-seg/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Kvasir-SEG for https://kvasir.no/datasets/kvasir-seg/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Kvasir-SEG for https://kvasir.no/datasets/kvasir-seg/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier Kvasir-SEG for https://kvasir.no/datasets/kvasir-seg/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier Kvasir-SEG for https://kvasir.no/datasets/kvasir-seg/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://kvasir.no/datasets/kvasir-seg/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'Kvasir-SEG', 'data_repository': 'kvasir.no', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 278 - INFO - Augmented dataset links: [{'dataset_identifier': 'ACDC', 'data_repository': 'www.creatis.insa-lyon.fr', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn3193805', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn3193805', 'citation_type': 'n/a'}, {'dataset_identifier': 'Kvasir-SEG', 'data_repository': 'kvasir.no', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1191 - INFO - Fetching metadata for 3 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: ACDC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: www.creatis.insa-lyon.fr, identifier: ACDC, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier ACDC for www.creatis.insa-lyon.fr\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier ACDC for www.creatis.insa-lyon.fr\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier ACDC for www.creatis.insa-lyon.fr\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier ACDC for www.creatis.insa-lyon.fr\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier ACDC for www.creatis.insa-lyon.fr\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier ACDC for www.creatis.insa-lyon.fr\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: www.creatis.insa-lyon.fr\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL www.creatis.insa-lyon.fr may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for ACDC in www.creatis.insa-lyon.fr\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 1 with repo: www.creatis.insa-lyon.fr and accession_id: ACDC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository www.creatis.insa-lyon.fr unknown in Ontology. Skipping dataset page 1.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn3193805\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn3193805, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3193805 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn3193805 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3193805 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn3193805 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn3193805\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn3193805\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: Kvasir-SEG\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: kvasir.no, identifier: Kvasir-SEG, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier Kvasir-SEG for kvasir.no\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier Kvasir-SEG for kvasir.no\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier Kvasir-SEG for kvasir.no\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier Kvasir-SEG for kvasir.no\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier Kvasir-SEG for kvasir.no\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier Kvasir-SEG for kvasir.no\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: kvasir.no\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL kvasir.no may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for Kvasir-SEG in kvasir.no\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 3 with repo: kvasir.no and accession_id: Kvasir-SEG\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository kvasir.no unknown in Ontology. Skipping dataset page 3.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1277 - INFO - Updated datasets len: 3\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 287 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 4, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'source_url', 'download_link',\n",
      "       'title', 'content_type', 'id', 'caption', 'description',\n",
      "       'source_section', 'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 500 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 506 - INFO - XML element classification not needed. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1012 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11514281\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1020 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11514281\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 666th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11514822)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11514822\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11514822 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 634 - INFO - Fetching data for pmc11514822\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 637 - INFO - Fetching data from scripts/exp_input/Local_fetched_data_1.parquet\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 438 - INFO - Raw xml data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11514822 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 452 - INFO - Skipping raw HTML/XML saving.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 458 - INFO - Using XMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 52 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 24 - INFO - Initializing xmlRetriever\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 466 - INFO - No additional data provided. Parsing raw data only.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 147 - INFO - Function call: parse_data(api_data(<class 'str'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11514822, additional_data, XML)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 162 - INFO - api_data converted to lxml element\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 466 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 153 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 154 - INFO - Consider migrating this function to the BaseRetriever class.\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 69 - INFO - Loading target sections for section name: supplementary_material_sections\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 477 - INFO - Found 12 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x15f830fc0>, <Element supplementary-material at 0x15dff8100>, <Element supplementary-material at 0x15dffa880>, <Element supplementary-material at 0x15dff9dc0>, <Element supplementary-material at 0x15dff9d00>, <Element supplementary-material at 0x15dff9d40>, <Element supplementary-material at 0x15dff9bc0>, <Element supplementary-material at 0x15dff9a00>, <Element supplementary-material at 0x15dff9c40>, <Element supplementary-material at 0x15dff9040>, <Element supplementary-material at 0x15eb969c0>, <Element supplementary-material at 0x15dff9c00>]\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 485 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x15f830fc0>, <Element supplementary-material at 0x15dff8100>, <Element supplementary-material at 0x15dffa880>, <Element supplementary-material at 0x15dff9dc0>, <Element supplementary-material at 0x15dff9d00>, <Element supplementary-material at 0x15dff9d40>, <Element supplementary-material at 0x15dff9bc0>, <Element supplementary-material at 0x15dff9a00>, <Element supplementary-material at 0x15dff9c40>, <Element supplementary-material at 0x15dff9040>, <Element supplementary-material at 0x15eb969c0>, <Element supplementary-material at 0x15dff9c00>]}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 574 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM1: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM2: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM3: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM4: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM5: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM6: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM7: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM8: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM9: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM10: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM11: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM12: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 263 - INFO - Extracting links from full XML content.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 301 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 309 - INFO - Content length: 198332\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 326 - INFO - Prompt messages total length: 50336 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 332 - INFO - Prompt ID: gemini-2.5-flash-0-1a44837d7791eaa79c048d5040f7762f6a203958fac7c05e7e4fbbb66a63d822\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 351 - INFO - Requesting datasets from content using model: gemini-2.5-flash, temperature: 0, messages length: 50336 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-77ef213b-cac3-40fb-8b63-d43b66828027\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"eQTL Catalogue\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ebi.ac.uk/eqtl/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"PsychENCODE\\\",\\n    \\\"repository_reference\\\": \\\"http://resource.psychencode.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"xQTL\\\",\\n    \\\"repository_reference\\\": \\\"http://mostafavilab.stat.ubc.ca/xqtl/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn5585484\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org//#!Synapse:syn5585484\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"BRAINEAC\\\",\\n    \\\"repository_reference\\\": \\\"http://www.braineac.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GTEx ver. 8\\\",\\n    \\\"repository_reference\\\": \\\"http://www.gtexportal.org/home/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"FANTOM5\\\",\\n    \\\"repository_reference\\\": \\\"http://fantom.gsc.riken.jp/5/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"E053 to E082 (Roadmap Consortium)\\\",\\n    \\\"repository_reference\\\": \\\"http://egg2.wustl.edu/roadmap/web_portal/DNase_reg.html\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"FINNGEN (R10)\\\",\\n    \\\"repository_reference\\\": \\\"https://www.finngen.fi/en\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn2580853\\\",\\n    \\\"repository_reference\\\": \\\"Synapse\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Human Protein Atlas\\\",\\n    \\\"repository_reference\\\": \\\"https://www.proteinatlas.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Uniprot Knowledge Base\\\",\\n    \\\"repository_reference\\\": \\\"https://www.uniprot.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"BioGRID\\\",\\n    \\\"repository_reference\\\": \\\"https://thebiogrid.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"STRING\\\",\\n    \\\"repository_reference\\\": \\\"https://string-db.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"IntAct\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ebi.ac.uk/intact/home\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"eQTL Catalogue\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ebi.ac.uk/eqtl/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"PsychENCODE\\\",\\n    \\\"repository_reference\\\": \\\"http://resource.psychencode.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"xQTL\\\",\\n    \\\"repository_reference\\\": \\\"http://mostafavilab.stat.ubc.ca/xqtl/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn5585484\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org//#!Synapse:syn5585484\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"BRAINEAC\\\",\\n    \\\"repository_reference\\\": \\\"http://www.braineac.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GTEx ver. 8\\\",\\n    \\\"repository_reference\\\": \\\"http://www.gtexportal.org/home/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"FANTOM5\\\",\\n    \\\"repository_reference\\\": \\\"http://fantom.gsc.riken.jp/5/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"E053 to E082 (Roadmap Consortium)\\\",\\n    \\\"repository_reference\\\": \\\"http://egg2.wustl.edu/roadmap/web_portal/DNase_reg.html\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"FINNGEN (R10)\\\",\\n    \\\"repository_reference\\\": \\\"https://www.finngen.fi/en\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn2580853\\\",\\n    \\\"repository_reference\\\": \\\"Synapse\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Human Protein Atlas\\\",\\n    \\\"repository_reference\\\": \\\"https://www.proteinatlas.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"Uniprot Knowledge Base\\\",\\n    \\\"repository_reference\\\": \\\"https://www.uniprot.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"BioGRID\\\",\\n    \\\"repository_reference\\\": \\\"https://thebiogrid.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"STRING\\\",\\n    \\\"repository_reference\\\": \\\"https://string-db.org/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"IntAct\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ebi.ac.uk/intact/home\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1758211072,\n",
      "    \"model\": \"gemini-2.5-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 65710,\n",
      "        \"completion_tokens\": 609,\n",
      "        \"total_tokens\": 75642,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 9323,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 664 - INFO - Deduplicating response with 15 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: eQTL Catalogue\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: PsychENCODE\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: xQTL\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn5585484\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: BRAINEAC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: GTEx ver. 8\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: FANTOM5\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: E053 to E082 (Roadmap Consortium)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: FINNGEN (R10)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn2580853\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: Human Protein Atlas\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: Uniprot Knowledge Base\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: BioGRID\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: STRING\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: IntAct\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'eQTL Catalogue', 'repository_reference': 'https://www.ebi.ac.uk/eqtl/', 'dataset_id': 'eQTL Catalogue'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: eQTL Catalogue\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID eQTL Catalogue is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.ebi.ac.uk/eqtl/, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.ebi.ac.uk/eqtl/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://www.ebi.ac.uk/eqtl/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 644 - INFO - Dataset ID is 'n/a' and repository name from prompt\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 541 - INFO - Skipping dataset due to missing ID, repository, dataset page: {'dataset_identifier': 'eQTL Catalogue', 'repository_reference': 'https://www.ebi.ac.uk/eqtl/', 'dataset_id': 'eQTL Catalogue'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'PsychENCODE', 'repository_reference': 'http://resource.psychencode.org/', 'dataset_id': 'PsychENCODE'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: PsychENCODE\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID PsychENCODE is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: http://resource.psychencode.org/, identifier: PsychENCODE, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: http://resource.psychencode.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier PsychENCODE for http://resource.psychencode.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier PsychENCODE for http://resource.psychencode.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier PsychENCODE for http://resource.psychencode.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier PsychENCODE for http://resource.psychencode.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier PsychENCODE for http://resource.psychencode.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier PsychENCODE for http://resource.psychencode.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: http://resource.psychencode.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'PsychENCODE', 'data_repository': 'resource.psychencode.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'xQTL', 'repository_reference': 'http://mostafavilab.stat.ubc.ca/xqtl/', 'dataset_id': 'xQTL'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: xQTL\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID xQTL is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: http://mostafavilab.stat.ubc.ca/xqtl/, identifier: xQTL, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: http://mostafavilab.stat.ubc.ca/xqtl/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier xQTL for http://mostafavilab.stat.ubc.ca/xqtl/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier xQTL for http://mostafavilab.stat.ubc.ca/xqtl/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier xQTL for http://mostafavilab.stat.ubc.ca/xqtl/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier xQTL for http://mostafavilab.stat.ubc.ca/xqtl/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier xQTL for http://mostafavilab.stat.ubc.ca/xqtl/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier xQTL for http://mostafavilab.stat.ubc.ca/xqtl/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: http://mostafavilab.stat.ubc.ca/xqtl/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'xQTL', 'data_repository': 'mostafavilab.stat.ubc.ca', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn5585484', 'repository_reference': 'https://www.synapse.org//#!Synapse:syn5585484', 'dataset_id': 'syn5585484'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn5585484\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn5585484\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org//#!Synapse:syn5585484, identifier: syn5585484, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org//#!Synapse:syn5585484\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn5585484 for https://www.synapse.org//#!Synapse:syn5585484\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn5585484 for https://www.synapse.org//#!Synapse:syn5585484\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn5585484 for https://www.synapse.org//#!Synapse:syn5585484\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org//#!Synapse:syn5585484 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org//#!Synapse:syn5585484\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://www.synapse.org//#!Synapse:syn5585484\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn5585484\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn5585484\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn5585484\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org//#!Synapse:syn5585484\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn5585484\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn5585484\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn5585484, synapse.org, https://www.synapse.org//#!Synapse:syn5585484\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn5585484\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org//#!Synapse:syn5585484\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn5585484', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org//#!Synapse:syn5585484', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'BRAINEAC', 'repository_reference': 'http://www.braineac.org/', 'dataset_id': 'BRAINEAC'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: BRAINEAC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID BRAINEAC is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: http://www.braineac.org/, identifier: BRAINEAC, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: http://www.braineac.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier BRAINEAC for http://www.braineac.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier BRAINEAC for http://www.braineac.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier BRAINEAC for http://www.braineac.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier BRAINEAC for http://www.braineac.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier BRAINEAC for http://www.braineac.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier BRAINEAC for http://www.braineac.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: http://www.braineac.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'BRAINEAC', 'data_repository': 'www.braineac.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'GTEx ver. 8', 'repository_reference': 'http://www.gtexportal.org/home/', 'dataset_id': 'GTEx ver. 8'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: GTEx ver. 8\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID GTEx ver. 8 is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: http://www.gtexportal.org/home/, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: http://www.gtexportal.org/home/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: http://www.gtexportal.org/home/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'www.gtexportal.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'FANTOM5', 'repository_reference': 'http://fantom.gsc.riken.jp/5/', 'dataset_id': 'FANTOM5'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: FANTOM5\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID FANTOM5 is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: http://fantom.gsc.riken.jp/5/, identifier: FANTOM5, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: http://fantom.gsc.riken.jp/5/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier FANTOM5 for http://fantom.gsc.riken.jp/5/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier FANTOM5 for http://fantom.gsc.riken.jp/5/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier FANTOM5 for http://fantom.gsc.riken.jp/5/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier FANTOM5 for http://fantom.gsc.riken.jp/5/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier FANTOM5 for http://fantom.gsc.riken.jp/5/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier FANTOM5 for http://fantom.gsc.riken.jp/5/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: http://fantom.gsc.riken.jp/5/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'FANTOM5', 'data_repository': 'fantom.gsc.riken.jp', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'E053 to E082 (Roadmap Consortium)', 'repository_reference': 'http://egg2.wustl.edu/roadmap/web_portal/DNase_reg.html', 'dataset_id': 'E053 to E082 (Roadmap Consortium)'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: E053 to E082 (Roadmap Consortium)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID E053 to E082 (Roadmap Consortium) is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: http://egg2.wustl.edu/roadmap/web_portal/DNase_reg.html, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: http://egg2.wustl.edu/roadmap/web_portal/DNase_reg.html\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: http://egg2.wustl.edu/roadmap/web_portal/DNase_reg.html\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'egg2.wustl.edu', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'FINNGEN (R10)', 'repository_reference': 'https://www.finngen.fi/en', 'dataset_id': 'FINNGEN (R10)'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: FINNGEN (R10)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID FINNGEN (R10) is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.finngen.fi/en, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.finngen.fi/en\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://www.finngen.fi/en\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'www.finngen.fi', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn2580853', 'repository_reference': 'Synapse', 'dataset_id': 'syn2580853'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn2580853\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn2580853\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: Synapse, identifier: syn2580853, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn2580853 for Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn2580853 for Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn2580853 for Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1143 - INFO - Found repo_name match for Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn2580853\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn2580853\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn2580853\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn2580853, synapse.org, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn2580853\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn2580853', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'Human Protein Atlas', 'repository_reference': 'https://www.proteinatlas.org/', 'dataset_id': 'Human Protein Atlas'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: Human Protein Atlas\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID Human Protein Atlas is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.proteinatlas.org/, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.proteinatlas.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://www.proteinatlas.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'www.proteinatlas.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'Uniprot Knowledge Base', 'repository_reference': 'https://www.uniprot.org/', 'dataset_id': 'Uniprot Knowledge Base'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: Uniprot Knowledge Base\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID Uniprot Knowledge Base is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.uniprot.org/, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.uniprot.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://www.uniprot.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'www.uniprot.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'BioGRID', 'repository_reference': 'https://thebiogrid.org/', 'dataset_id': 'BioGRID'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: BioGRID\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID BioGRID is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://thebiogrid.org/, identifier: BioGRID, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://thebiogrid.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier BioGRID for https://thebiogrid.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier BioGRID for https://thebiogrid.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier BioGRID for https://thebiogrid.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier BioGRID for https://thebiogrid.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier BioGRID for https://thebiogrid.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier BioGRID for https://thebiogrid.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://thebiogrid.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'BioGRID', 'data_repository': 'thebiogrid.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'STRING', 'repository_reference': 'https://string-db.org/', 'dataset_id': 'STRING'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: STRING\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID STRING is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://string-db.org/, identifier: STRING, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://string-db.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier STRING for https://string-db.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier STRING for https://string-db.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier STRING for https://string-db.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier STRING for https://string-db.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier STRING for https://string-db.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier STRING for https://string-db.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://string-db.org/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'STRING', 'data_repository': 'string-db.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'IntAct', 'repository_reference': 'https://www.ebi.ac.uk/intact/home', 'dataset_id': 'IntAct'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: IntAct\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID IntAct is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.ebi.ac.uk/intact/home, identifier: IntAct, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.ebi.ac.uk/intact/home\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier IntAct for https://www.ebi.ac.uk/intact/home\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier IntAct for https://www.ebi.ac.uk/intact/home\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier IntAct for https://www.ebi.ac.uk/intact/home\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier IntAct for https://www.ebi.ac.uk/intact/home\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier IntAct for https://www.ebi.ac.uk/intact/home\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier IntAct for https://www.ebi.ac.uk/intact/home\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://www.ebi.ac.uk/intact/home\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'IntAct', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 278 - INFO - Augmented dataset links: [{'dataset_identifier': 'PsychENCODE', 'data_repository': 'resource.psychencode.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'xQTL', 'data_repository': 'mostafavilab.stat.ubc.ca', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn5585484', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org//#!Synapse:syn5585484', 'citation_type': 'n/a'}, {'dataset_identifier': 'BRAINEAC', 'data_repository': 'www.braineac.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'www.gtexportal.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'FANTOM5', 'data_repository': 'fantom.gsc.riken.jp', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'egg2.wustl.edu', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'www.finngen.fi', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn2580853', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'www.proteinatlas.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'www.uniprot.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'BioGRID', 'data_repository': 'thebiogrid.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'STRING', 'data_repository': 'string-db.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'IntAct', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1191 - INFO - Fetching metadata for 14 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: PsychENCODE\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: resource.psychencode.org, identifier: PsychENCODE, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier PsychENCODE for resource.psychencode.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier PsychENCODE for resource.psychencode.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier PsychENCODE for resource.psychencode.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier PsychENCODE for resource.psychencode.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier PsychENCODE for resource.psychencode.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier PsychENCODE for resource.psychencode.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: resource.psychencode.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL resource.psychencode.org may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for PsychENCODE in resource.psychencode.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 1 with repo: resource.psychencode.org and accession_id: PsychENCODE\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository resource.psychencode.org unknown in Ontology. Skipping dataset page 1.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: xQTL\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: mostafavilab.stat.ubc.ca, identifier: xQTL, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier xQTL for mostafavilab.stat.ubc.ca\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier xQTL for mostafavilab.stat.ubc.ca\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier xQTL for mostafavilab.stat.ubc.ca\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier xQTL for mostafavilab.stat.ubc.ca\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier xQTL for mostafavilab.stat.ubc.ca\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier xQTL for mostafavilab.stat.ubc.ca\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: mostafavilab.stat.ubc.ca\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL mostafavilab.stat.ubc.ca may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for xQTL in mostafavilab.stat.ubc.ca\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 2 with repo: mostafavilab.stat.ubc.ca and accession_id: xQTL\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository mostafavilab.stat.ubc.ca unknown in Ontology. Skipping dataset page 2.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn5585484\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn5585484, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn5585484 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn5585484 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn5585484 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn5585484 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 3 with repo: synapse.org and accession_id: syn5585484\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn5585484\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 3: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: BRAINEAC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: www.braineac.org, identifier: BRAINEAC, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier BRAINEAC for www.braineac.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier BRAINEAC for www.braineac.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier BRAINEAC for www.braineac.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier BRAINEAC for www.braineac.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier BRAINEAC for www.braineac.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier BRAINEAC for www.braineac.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: www.braineac.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL www.braineac.org may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for BRAINEAC in www.braineac.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 4 with repo: www.braineac.org and accession_id: BRAINEAC\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository www.braineac.org unknown in Ontology. Skipping dataset page 4.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 5: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: FANTOM5\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: fantom.gsc.riken.jp, identifier: FANTOM5, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier FANTOM5 for fantom.gsc.riken.jp\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier FANTOM5 for fantom.gsc.riken.jp\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier FANTOM5 for fantom.gsc.riken.jp\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier FANTOM5 for fantom.gsc.riken.jp\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier FANTOM5 for fantom.gsc.riken.jp\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier FANTOM5 for fantom.gsc.riken.jp\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: fantom.gsc.riken.jp\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL fantom.gsc.riken.jp may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for FANTOM5 in fantom.gsc.riken.jp\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 6 with repo: fantom.gsc.riken.jp and accession_id: FANTOM5\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository fantom.gsc.riken.jp unknown in Ontology. Skipping dataset page 6.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 7: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 8: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn2580853\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn2580853, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn2580853 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn2580853 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn2580853 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn2580853 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 9 with repo: synapse.org and accession_id: syn2580853\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn2580853\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 9: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 10 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 10: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 11 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 11: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 12 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: BioGRID\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: thebiogrid.org, identifier: BioGRID, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier BioGRID for thebiogrid.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier BioGRID for thebiogrid.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier BioGRID for thebiogrid.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier BioGRID for thebiogrid.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier BioGRID for thebiogrid.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier BioGRID for thebiogrid.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: thebiogrid.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL thebiogrid.org may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for BioGRID in thebiogrid.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 12 with repo: thebiogrid.org and accession_id: BioGRID\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository thebiogrid.org unknown in Ontology. Skipping dataset page 12.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 13 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: STRING\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: string-db.org, identifier: STRING, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier STRING for string-db.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier STRING for string-db.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier STRING for string-db.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier STRING for string-db.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier STRING for string-db.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier STRING for string-db.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: string-db.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL string-db.org may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for STRING in string-db.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 13 with repo: string-db.org and accession_id: STRING\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository string-db.org unknown in Ontology. Skipping dataset page 13.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 14 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: IntAct\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: www.ebi.ac.uk, identifier: IntAct, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: www.ebi.ac.uk\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for IntAct in www.ebi.ac.uk\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 14 with repo: www.ebi.ac.uk and accession_id: IntAct\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.ebi.ac.uk/pride/archive/projects/IntAct\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 14: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1277 - INFO - Updated datasets len: 14\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 287 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 26, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'source_url', 'download_link',\n",
      "       'title', 'content_type', 'id', 'caption', 'description',\n",
      "       'source_section', 'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 500 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 506 - INFO - XML element classification not needed. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1012 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11514822\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1020 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11514822\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 667th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11514917)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11514917\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11514917 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 634 - INFO - Fetching data for pmc11514917\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 637 - INFO - Fetching data from scripts/exp_input/Local_fetched_data_1.parquet\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 438 - INFO - Raw xml data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11514917 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 452 - INFO - Skipping raw HTML/XML saving.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 458 - INFO - Using XMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 52 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 24 - INFO - Initializing xmlRetriever\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 466 - INFO - No additional data provided. Parsing raw data only.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 147 - INFO - Function call: parse_data(api_data(<class 'str'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11514917, additional_data, XML)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 162 - INFO - api_data converted to lxml element\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 466 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 153 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 154 - INFO - Consider migrating this function to the BaseRetriever class.\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 69 - INFO - Loading target sections for section name: supplementary_material_sections\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 477 - INFO - Found 1 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x15f41f2c0>]\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 485 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x15f41f2c0>]}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 574 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref acn352175-supitem-0001: Association analysis of the 44 candidate miRNAs identified miR‐133b as significantly associated with NPTX2 protein levels after multiple comparison correction (β [standard error (SE)], 0.21 [0.056]; FDR‐corrected = 0.014) (Fig. 1A [rid=acn352175-fig-0001] and Table S1 [this file] ). \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 263 - INFO - Extracting links from full XML content.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 301 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 309 - INFO - Content length: 54947\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 326 - INFO - Prompt messages total length: 14490 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 332 - INFO - Prompt ID: gemini-2.5-flash-0-9563eec2a0d2960dd9b44fbeffc4c761c617058e42f3790268ca14fc658bce14\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 351 - INFO - Requesting datasets from content using model: gemini-2.5-flash, temperature: 0, messages length: 14490 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-dc4326d2-a454-4c29-a14b-fdea706a363a\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn3387325\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn8456638\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn17015098\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn3387325\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn8456638\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn17015098\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1758211083,\n",
      "    \"model\": \"gemini-2.5-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 19036,\n",
      "        \"completion_tokens\": 129,\n",
      "        \"total_tokens\": 20047,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 882,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 664 - INFO - Deduplicating response with 3 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn3387325\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn8456638\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn17015098\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn3387325', 'repository_reference': 'https://www.synapse.org', 'dataset_id': 'syn3387325'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn3387325\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn3387325\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org, identifier: syn3387325, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3387325 for https://www.synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn3387325 for https://www.synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3387325 for https://www.synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn3387325\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn3387325\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn3387325\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn3387325, synapse.org, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn3387325\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn3387325', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn8456638', 'repository_reference': 'https://www.synapse.org', 'dataset_id': 'syn8456638'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn8456638\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn8456638\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org, identifier: syn8456638, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn8456638 for https://www.synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn8456638 for https://www.synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn8456638 for https://www.synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn8456638\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn8456638\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn8456638\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn8456638, synapse.org, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn8456638\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn8456638', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn17015098', 'repository_reference': 'https://www.synapse.org', 'dataset_id': 'syn17015098'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn17015098\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn17015098\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org, identifier: syn17015098, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn17015098 for https://www.synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn17015098 for https://www.synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn17015098 for https://www.synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn17015098\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn17015098\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn17015098\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn17015098, synapse.org, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn17015098\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn17015098', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 278 - INFO - Augmented dataset links: [{'dataset_identifier': 'syn3387325', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn8456638', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn17015098', 'data_repository': 'synapse.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1191 - INFO - Fetching metadata for 3 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn3387325\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn3387325, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn3387325 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn3387325 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn3387325 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn3387325 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 1 with repo: synapse.org and accession_id: syn3387325\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn3387325\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 1: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn8456638\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn8456638, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn8456638 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn8456638 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn8456638 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn8456638 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn8456638\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn8456638\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn17015098\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn17015098, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn17015098 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn17015098 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn17015098 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn17015098 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 3 with repo: synapse.org and accession_id: syn17015098\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn17015098\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 3: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1277 - INFO - Updated datasets len: 3\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 287 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 4, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'source_url', 'download_link',\n",
      "       'title', 'content_type', 'id', 'caption', 'description',\n",
      "       'source_section', 'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 500 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 506 - INFO - XML element classification not needed. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1012 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11514917\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1020 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11514917\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 668th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11520681)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11520681\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11520681 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 634 - INFO - Fetching data for pmc11520681\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 637 - INFO - Fetching data from scripts/exp_input/Local_fetched_data_1.parquet\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 438 - INFO - Raw xml data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11520681 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 452 - INFO - Skipping raw HTML/XML saving.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 458 - INFO - Using XMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 52 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 24 - INFO - Initializing xmlRetriever\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 466 - INFO - No additional data provided. Parsing raw data only.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 147 - INFO - Function call: parse_data(api_data(<class 'str'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11520681, additional_data, XML)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 162 - INFO - api_data converted to lxml element\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 466 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 153 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 154 - INFO - Consider migrating this function to the BaseRetriever class.\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 69 - INFO - Loading target sections for section name: supplementary_material_sections\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 477 - INFO - Found 2 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x15eeba300>, <Element supplementary-material at 0x15eeb8240>]\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 485 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x15eeba300>, <Element supplementary-material at 0x15eeb8240>]}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 574 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM1: RNA-seq data were collected from multiple resources, including 510 human samples covering four developmental periods (fetal period, childhood, adolescence, and adulthood) and 16 brain regions from the psychENCODE website ( http://development.psychencode.org (http://development.psychencode.org) ) [ 16 [rid=CR16] ], 176 macaque samples covering two developmental periods (5 years old, 10 years old) and 14 brain regions from the GEO website (accession GSE128537 (GSE128537) ) [ 17 [rid=CR17] ], and samples and controls from patients with brain diseases from public databases (Supplementary Table 1 [this file] ). \n",
      "The number of period- and region-specific sex-biased genes that are shared across regions and regions is limited (Fig. 1 [rid=Fig1] BC; Supplementary Fig. 1 [this file] ), which is consistent with previous findings that few sex-biased genes are shared across organs [ 8 [rid=CR8] ]. \n",
      "To verify this conclusion, we examined AR and ER expression in the brain and found that their expression varies spatiotemporally (Fig. 2 [rid=Fig2] A; Supplementary Fig. 2 [this file] ). In accordance with the GTEx data (Supplementary Fig. 3 [this file] ), the ratio, while varying spatiotemporally, is > 1.0 in both sexes in most regions. Additionally, the enrichment of multiple sex hormone-related GO terms revealed period-specific differences between males and females (Supplementary Fig. 4 [this file] ). \n",
      "Sex-biased gene expression shows spatiotemporal variations in the macaque brain (Supplementary Fig. 5 [this file] ), but the number of sex-biased genes shared between humans and macaques is limited in specific periods and regions (Fig. 4 [rid=Fig4] A; Supplementary Fig. 6 [this file] ). \n",
      "Gene set enrichment analysis via the program for the intersection between the one2zero genes and sex-biased genes in adulthood revealed that these genes are enriched for immune-related GO terms (Benjamini‒Hochberg FDR < 0.05) (Fig. 4 [rid=Fig4] D; Supplementary Fig. 7 [this file] ). \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM2: The 16 human brain regions included the neocortex (A1C, DFC, IPC, ITC, M1C, MFC, OFC, S1C, STC, V1C, and VFC), amygdala (AMY), cerebellar cortex (CBC), hippocampus (HIP), thalamus (MD), and striatum (STR) (Supplementary Table 2 [this file] ), and the 14 macaque brain regions are their equivalent. \n",
      "The union of the 64 gene sets showing sex-biased expression in at least one region or period contained 7647 genes (Supplementary Table 3 [this file] ). \n",
      "This model yielded four sets of genes (Supplementary Table 4 [this file] ). \n",
      "This model yielded 16 sets of genes (Supplementary Table 5 [this file] ). \n",
      "Spatiotemporal-specific sex-biased genes were identified with thresholds of |log2FC|>1.0 and LFSR < 0.005 (Supplementary Table 11 [this file] ). \n",
      "The likely regulatory relationship was defined as greater than 60 (corresponding to DBS length > = 90 bp) (Supplementary Table 9 [this file] ). \n",
      "We also investigated whether sex-biased lncRNA genes are more likely to colocalize with sex-biased protein-coding genes than with non-sex-biased lncRNA genes and found that for most sex-biased genes, the likelihood is high (Fisher’s exact test, FDR < < 0.05) (Supplementary Table 10 [this file] ). \n",
      "Enriched GO terms with a negative normalized enrichment score (NES) indicated enrichment of male-biased genes, and enriched GO terms with a positive NES indicated enrichment of female-biased genes (Supplementary Table 6 [this file] ). \n",
      "Second, to reveal the functions of the consensus coexpression module of period-specific sex-biased genes and region-specific sex-biased genes (gene lists without weights), we performed ORA using the function and the GO database (Supplementary Tables 7 [this file] , 8 [this file] ). \n",
      "We used it to identify the enriched GO terms of human-specific (in humans but not in macaques) sex-biased protein-coding genes (Fig. 4 [rid=Fig4] D), enriched GO terms of consensus coexpression modules of sex-biased genes in macaques (Supplementary Table 12 [this file] ), and enriched GO terms of sex-specific differentially expressed genes in the four brain diseases (Supplementary Table 14 [this file] ). \n",
      "We also obtained male- and female-specific DEGs by comparing gene expression differences between males and females (instead of between patients and controls) (Supplementary Table 13 [this file] ). \n",
      "Most of the results agree with the Fisher exact test results, with large OA values in many cases (Figs. 3 [rid=Fig3] A and 5 [rid=Fig5] E; Supplementary Table 10 [this file] ; Supplementary Table 15 [this file] ). \n",
      "The data include 510 human brain samples covering four developmental periods and 16 brain regions from the psychENCODE website (Fig. 1 [rid=Fig1] A) [ 16 [rid=CR16] ] and 176 macaque brain samples covering two developmental periods (5 years old and 10 years old, corresponding to adolescence and adulthood in humans) and 14 orthologous brain regions (Supplementary Table 1 [this file] ) [ 17 [rid=CR17] ]. These brain regions have distinct functions (Supplementary Table 2 [this file] ). \n",
      "A total of 7647 genes were identified as “spatiotemporal-specific sex-biased genes” (|log2FC|>1.0 and local false sign rate (LFSR) < 0.001) that presented sex-biased expression across regions and periods (Fig. 1 [rid=Fig1] A; Supplementary Table 3 [this file] ). By treating “period” as a fixed effect and “region” as a confounding factor and treating “region” as a fixed effect and “period” as a confounding factor, we also used LMM models to identify “period-specific sex-biased genes” (|log2FC|>1.0 and FDR < 0.03) that show period-specific sex-biased expression and “region-specific sex-biased genes” (|log2FC|>1.0 and LFSR < 0.001) that show region-specific sex-biased expression (Supplementary Tables 4 [this file] , 5 [this file] ). \n",
      "Second, although lncRNAs can regulate transcription and , we found that sex-biased lncRNA genes are significantly more likely to be located near sex-biased protein-coding genes than lncRNA genes not identified as sex biased (Fisher’s exact test, FDR < 0.05) (Supplementary Table 10 [this file] ). \n",
      "Seventeen modules were identified from the four periods (Fig. 3 [rid=Fig3] A; Supplementary Table 7 [this file] ). \n",
      "Notably, was also enriched for immune response-, mitochondrial function-, and neurogenesis-related GO terms (FDR < 0.05) (Supplementary Table 8 [this file] ) and shared 740 genes with . \n",
      "In the module, 18, 29, 34, and 59 lncRNAs have a DBS in 73, 87, 182, and 251 genes in the four periods, respectively (Supplementary Table 9 [this file] ), and these high ratios of the targeting relationship support that lncRNAs regulate sex bias by regulating sex-biased genes. \n",
      "First, using the methods described above, we identified spatiotemporal-specific sex-biased genes (|log2FC|>1.0 and LFSR < 0.005) (Supplementary Table 11 [this file] ) (also period- and region-specific sex-biased genes) in the macaque brain. \n",
      "Among the four modules identified from the two macaque developmental periods, the largest brown module is enriched for neural development-related genes (Supplementary Table 12 [this file] ). \n",
      "The 185 one2zero genes share many overlaps with sex-biased genes in adulthood (but not in other periods; see the “hg38-rheMac10 one2zero” column in Supplementary Table 3 [this file] ). \n",
      "To address this question, we first identified sex-specific differentially expressed genes (DEGs) upon gene expression in disease and normal samples from the same sex in four brain diseases, including glioblastoma multiform (GBM), lower grade glioma (LGG), schizophrenia (SCZ), and autism spectrum disorder (ASD) (Supplementary Table 13 [this file] ). These sex-specific DEGs are enriched for metabolic processes, immune responses, and neurogenesis (Fig. 5 [rid=Fig5] D; Supplementary Table 12 [this file] ), which is consistent with the above-described finding that sex-biased genes are enriched in the immune response in humans (Fig. 4 [rid=Fig4] B). The sex-specific DEGs in the four brain disease samples from DFC and HIP were significantly more enriched with sex-biased genes than non-sex-biased genes in the two regions (i.e., DFC and HIP, Fisher’s exact test, < 0.05; OA > 1.2 for 16/20 sex-specific DEG sets) (Fig. 5 [rid=Fig5] E; Supplementary Table 15 [this file] ). \n",
      "Fourth, caution is needed when interpreting the proximity of sex-biased lncRNAs to their targets (Supplementary Table 10 [this file] ) because little is known about whether a lncRNA prefers functioning or . \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 263 - INFO - Extracting links from full XML content.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 301 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 309 - INFO - Content length: 162007\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 326 - INFO - Prompt messages total length: 41255 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 332 - INFO - Prompt ID: gemini-2.5-flash-0-984944b7d32c2d7d8d724a466af599aa1c9ce56707f799d353860b22433562cd\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 351 - INFO - Requesting datasets from content using model: gemini-2.5-flash, temperature: 0, messages length: 41255 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-b3c8cfc9-0b0d-4ee2-bd15-b7ea2d040265\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE128537\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE128537\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn4587609\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn4587609\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"http://development.psychencode.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://xena.ucsc.edu\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"TCGA\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"GTEx\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"http://eqtl.brainseq.org/phase2/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://mart.ensembl.org/index.html\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://genome.ucsc.edu\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"LongMan\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://genome.senckenberg.de/\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE128537\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE128537\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn4587609\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn4587609\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"http://development.psychencode.org\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://xena.ucsc.edu\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"TCGA\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"GTEx\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"http://eqtl.brainseq.org/phase2/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://mart.ensembl.org/index.html\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://genome.ucsc.edu\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"LongMan\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://genome.senckenberg.de/\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1758211115,\n",
      "    \"model\": \"gemini-2.5-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 52538,\n",
      "        \"completion_tokens\": 432,\n",
      "        \"total_tokens\": 57816,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 4846,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 664 - INFO - Deduplicating response with 11 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: GSE128537\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn4587609\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'GSE128537', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE128537', 'dataset_id': 'GSE128537'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE128537\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: GSE128537\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE128537, identifier: GSE128537, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE128537\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE128537 for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE128537\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier GSE128537 for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE128537\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1157 - INFO - Found id_pattern match for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE128537 in GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE128537\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE128537\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: GSE128537\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE128537\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE128537\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE128537\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: GSE128537\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE128537\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: GSE128537, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE128537\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: GSE128537\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE128537\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'GSE128537', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE128537', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn4587609', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn4587609', 'dataset_id': 'syn4587609'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn4587609\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn4587609\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn4587609, identifier: syn4587609, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn4587609\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn4587609 for https://www.synapse.org/#!Synapse:syn4587609\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn4587609 for https://www.synapse.org/#!Synapse:syn4587609\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn4587609 for https://www.synapse.org/#!Synapse:syn4587609\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn4587609 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn4587609\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn4587609\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn4587609\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn4587609\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn4587609\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn4587609\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn4587609\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn4587609\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn4587609\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn4587609, synapse.org, https://www.synapse.org/#!Synapse:syn4587609\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn4587609\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn4587609\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn4587609', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn4587609', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'http://development.psychencode.org', 'dataset_id': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: http://development.psychencode.org, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: http://development.psychencode.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: http://development.psychencode.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'development.psychencode.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://xena.ucsc.edu'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://xena.ucsc.edu, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://xena.ucsc.edu\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://xena.ucsc.edu\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'xena.ucsc.edu', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'TCGA'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: TCGA, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: TCGA\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'tcga', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'GTEx'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: GTEx, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: GTEx\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'gtex', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'http://eqtl.brainseq.org/phase2/'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: http://eqtl.brainseq.org/phase2/, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: http://eqtl.brainseq.org/phase2/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: http://eqtl.brainseq.org/phase2/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'eqtl.brainseq.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://mart.ensembl.org/index.html'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://mart.ensembl.org/index.html, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://mart.ensembl.org/index.html\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://mart.ensembl.org/index.html\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'mart.ensembl.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://genome.ucsc.edu'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://genome.ucsc.edu, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://genome.ucsc.edu\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://genome.ucsc.edu\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'genome.ucsc.edu', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'LongMan'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: LongMan, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: LongMan\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'longman', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://genome.senckenberg.de/'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://genome.senckenberg.de/, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://genome.senckenberg.de/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://genome.senckenberg.de/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'genome.senckenberg.de', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 278 - INFO - Augmented dataset links: [{'dataset_identifier': 'GSE128537', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE128537', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn4587609', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn4587609', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'development.psychencode.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'xena.ucsc.edu', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'tcga', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'gtex', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'eqtl.brainseq.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'mart.ensembl.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'genome.ucsc.edu', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'longman', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'genome.senckenberg.de', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1191 - INFO - Fetching metadata for 11 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: GSE128537\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: geo, identifier: GSE128537, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE128537 for geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for GSE128537 in geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE128537\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE128537\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 1: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn4587609\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn4587609, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn4587609 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn4587609 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn4587609 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn4587609 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn4587609\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn4587609\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 3: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 4: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 5: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 6: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 7: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 8: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 9: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 10 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 10: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 11 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 11: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1277 - INFO - Updated datasets len: 11\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 287 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 13, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'source_url', 'download_link',\n",
      "       'title', 'content_type', 'id', 'caption', 'description',\n",
      "       'source_section', 'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 500 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 506 - INFO - XML element classification not needed. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1012 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11520681\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1020 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11520681\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 669th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11525184)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11525184\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11525184 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 634 - INFO - Fetching data for pmc11525184\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 637 - INFO - Fetching data from scripts/exp_input/Local_fetched_data_1.parquet\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 438 - INFO - Raw xml data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11525184 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 452 - INFO - Skipping raw HTML/XML saving.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 458 - INFO - Using XMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 52 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 24 - INFO - Initializing xmlRetriever\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 466 - INFO - No additional data provided. Parsing raw data only.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 147 - INFO - Function call: parse_data(api_data(<class 'str'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11525184, additional_data, XML)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 162 - INFO - api_data converted to lxml element\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 466 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 153 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 154 - INFO - Consider migrating this function to the BaseRetriever class.\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 69 - INFO - Loading target sections for section name: supplementary_material_sections\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 477 - INFO - Found 3 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x15eb96840>, <Element supplementary-material at 0x15eb94a40>, <Element supplementary-material at 0x15eb97540>]\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 485 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x15eb96840>, <Element supplementary-material at 0x15eb94a40>, <Element supplementary-material at 0x15eb97540>]}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 574 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM1: The power gain compared to the previous version of our fine-mapping call from 465 samples is represented by a 2.96-fold increase in the number of putative causal eQTLs (from 1,169 to 3,464; Extended Data Fig. 2a [rid=Fig9] ); the largely maintained quality of fine-mapping was validated by functional enrichment and consistency with external datasets (Extended Data Fig. 2b [rid=Fig9] , and Supplementary Figs. 1 [this file] and 2 [this file] ). The list of splice QTLs (sQTLs) was also updated (Supplementary Fig. 3 [this file] ). \n",
      "Using the expression of 2,932 plasma proteins measured with the Olink Explore 3072 assay for 1,384 samples, we performed genome-wide -pQTL calling and fine-mapping (Supplementary Fig. 4 [this file] and Methods [rid=Sec15] ). \n",
      "The fraction of the lead variants in the UKB PPP dataset also increased along with the PIP (as an approximation of causality in the absence of fine-mapping data in the UKB PPP; Supplementary Fig. 5 [this file] ). Lower consistency was observed when comparing with other datasets based on different technological platforms (Fig. 1c [rid=Fig1] ), as discussed in detail in Supplementary Figs. 6 [this file] and 7 [this file] and in Supplementary Note [this file] . \n",
      "These observations suggest that missense mutations could show pQTL effects by affecting protein structure and stability, especially in an extracellular environment, as discussed in ref. , who reported an enrichment of secreted protein-related features in genes harboring pQTLs (Supplementary Fig. 7 [this file] ). \n",
      "Missense variations were the most enriched functional annotation, even when focusing on possibly causal pQTLs replicated in the ARIC study (PIP > 0.1 in both studies; Fig. 2c [rid=Fig2] and Supplementary Fig. 8 [this file] ). \n",
      "To characterize such pQTL variants as colocalizing with eQTL variants, we compared our fine-mapping results with the eQTL fine-mapping results from the Genotype-Tissue Expression (GTEx) project v.8 (ref. ) in 49 major tissues (Supplementary Fig. 9a,b [this file] and Methods [rid=Sec15] ). \n",
      "Further, we created mRNA-adjusted protein expression and protein-adjusted mRNA expression profiles, and performed adjusted mRNA or protein expression QTL calling followed by statistical fine-mapping (Supplementary Fig. 10 [this file] ). For coding annotations, missense variants were strongly enriched for protein-specific QTLs as expected (8.95-fold compared to mRNA-specific QTLs, = 1.1 × 10 ; Fig. 3c [rid=Fig3] and Supplementary Fig. 11a [this file] ). \n",
      "We then examined their mRNA expression level across tissues in the GTEx, as well as their constraint from the Genome Aggregation Database (gnomAD) (LoF observed/expected upper bound fraction (LOEUF) ; Fig. 3e [rid=Fig3] and Supplementary Fig. 11b [this file] ). \n",
      "Using an alternative method (Mendelian randomization; Supplementary Fig. 12 [this file] and Methods [rid=Sec15] ) also suggested limited overlap; we explored possible reasons for such a low colocalization level. \n",
      "Even with a lenient assumption that a significant value in eQTLgen is a sign of colocalization (in reality, many are noncausal, tagged variants), the colocalization level remained low (only less than 25% were resolved; Supplementary Fig. 13a–d [this file] ). While more colocalization could be declared depending on the threshold setting, the overall fraction of genes with colocalizing evidence remained low (<15% at a PP.H4 = 0.1 threshold; Supplementary Fig. 13e,f [this file] ). \n",
      "Considering additional biological properties, such as active secretion of proteins into the bloodstream (Supplementary Fig. 13g,h [this file] ) or testing colocalization with fine-mapped variants in the GTEx across 49 tissues also increased the colocalization signal (from approximately 5% to over 10%; Supplementary Figs. 9c [this file] and 13i [this file] ). \n",
      "As another example of a gene with limited colocalization evidence, we highlight the unresolved locus in Supplementary Note [this file] , Supplementary Fig. 14 [this file] and Supplementary Table 1 [this file] . \n",
      "We used large-scale fine-mapping results for 79 traits from the BBJ to quantify the level of trait-causal variant colocalization (Fig. 4a [rid=Fig4] , Supplementary Fig. 15 [this file] and Supplementary Data 3 [rid=MOESM3] ). Enrichment is not likely to be driven by simple measurement traits such as blood cell count or lipid levels as enrichment persisted in several major disease and complex traits, although not universal to all traits as the opposite enrichment pattern was observed for body mass index (Supplementary Fig. 15a [this file] ). Stratification according to the percentage of mRNA expression of the genes suggested that the power gain of protein-specific QTLs is mainly from genes that are relatively lowly mRNA-expressed in the blood (Fig. 4c [rid=Fig4] and Supplementary Fig. 15d [this file] ), highlighting the value of the multi-tissue origin of plasma proteins. Comparison with nonspecific QTLs is more nuanced, as described in detail in the Supplementary Note [this file] . \n",
      "As specific examples, we highlighted three scenarios where plasma pQTL fine-mapping is beneficial in addition to eQTL fine-mapping in whole blood for complex trait-causal variant interpretation: (1) a 3′ UTR variant possibly involved in posttranscriptional regulation (rs884205 on the gene; Fig. 4d [rid=Fig4] ); (2) a missense variant probably affecting protein stability (rs429358, the well-known constituent of the APOE-ε4 allele ; Fig. 4e [rid=Fig4] and Supplementary Fig. 16 [this file] ); and (3) mRNA expression regulation taking place in non-blood tissue (rs13395911 on and rs28372783 on ; Supplementary Fig. 15h,i [this file] ), with detailed interpretation in Supplementary Note [this file] . \n",
      "In this study, we used two different approaches to maximize the prior and to control for the multiple test burden ( Supplementary Note [this file] ). \n",
      "First, we observed an increase in the correlation between mRNA and protein expression in patients with severe COVID-19 (Fig. 6a [rid=Fig6] and Supplementary Fig. 17 [this file] ), presumably because of the natural immune response triggering an increase in relevant genes at both the blood mRNA and plasma protein level. In addition, the shared biological entities of blood mRNA and plasma protein dynamics were visible at the module level (Supplementary Fig. 18 [this file] ). \n",
      "Specifically, we performed COVID-19 severity interaction QTL (iQTL) analysis as described previously in and compared the interaction landscape at the mRNA and protein levels (Supplementary Figs. 19 [this file] and 20 [this file] and Supplementary Data 5 [rid=MOESM3] ). We attribute this to buffering from high and stable expression of mRNAs in other tissues, such as lymphocytes (Fig. 6c [rid=Fig6] ), the dynamics of solid components missing in plasma and the limitation of detection (LOD) specific to the quantification of affinity-based protein expression (Supplementary Fig. 21 [this file] ), with further details described in Supplementary Note [this file] . \n",
      "Consistent with previous studies , the correlation between mRNA and protein expression was limited in our dataset (Fig. 3f [rid=Fig3] and Supplementary Fig. 17a,b [this file] ; mean Pearson = 0.047); a large fraction of genes had a negative correlation between mRNA and protein expression (866 of 2,211 (39.1%)). The gene has partially overlapping eQTL and pQTL peaks with a different effect direction; statistical fine-mapping identified five distinct putative -causal variants (three separate eQTLs plus two pQTLs, hereafter called e1, e2, e3, p1 and p2, with further details shown in Supplementary Table 2 [this file] ). \n",
      "We expected potential bias because the disease-ascertained nature of the WGS data in the reference panel was relatively low, as we observed an improvement in the eQTL fine-mapping results when compared to using the previous reference panel (Supplementary Fig. 1 [this file] ), suggesting a major benefit of a larger sample size and population specificity ( Supplementary Note [this file] ). \n",
      "The loose threshold setting was based on recent simulation-based observations that fine-mapping benefits more from including low-frequency variants even with a relatively limited quality, and was validated with functional enrichment analyses (although warning us that low-frequency variants could be slightly enriched for false positives; Supplementary Fig. 2i–l [this file] and Supplementary Note [this file] ). \n",
      "The following criteria were applied for sample QC: 0.5 × 10 < number of mapped reads < 3 × 10 , mapping rate > 0.97, intergenic rate <0.05, rRNA rate <0.05, base mismatch rate <0.005 and intersample correlation deviation measure > −15 (the threshold was slightly different from the GTEx pipeline or our previous pipeline to fit the observed data distribution; Supplementary Fig. 22 [this file] ). mRNA expression was then trimmed mean of M component (TMM)-normalized while low-expression data were filtered out as in the GTEx pipeline. Potential variations originating from technical factors, such as RNA integrity number, were thought to be captured by the PEER factors (Supplementary Fig. 23 [this file] ). Performing eQTL mapping for each disease severity stratum and combining the test statistics using a fixed effects model yielded generally consistent results (Supplementary Fig. 24 [this file] ). \n",
      "The distribution of COVID-19 severity was similar between batches (Supplementary Fig. 4 [this file] ). We did not apply additional sample QC after confirming that there were no outlier and major batch effects in the first two principal component spaces (Supplementary Fig. 4 [this file] ) after bridge normalization. We instead evaluated the potential effect of samples below the LOD post hoc and found no major bias introduced by the inclusion of samples below the LOD (Supplementary Fig. 21 [this file] and Supplementary Note [this file] ). \n",
      "In all such cases, the differences between measurements were minimal, limiting the bias introduced by this step (Supplementary Fig. 4 [this file] ). \n",
      "The biological properties of these 2,932 genes compared to all coding genes; for example, enrichment in inflammatory functions, as noted on their website ( https://olink.com/products/olink-explore-3072-384 (https://olink.com/products/olink-explore-3072-384) ) are summarized in Supplementary Fig. 25 [this file] . \n",
      "Results were largely consistent when excluding age from the covariates (Supplementary Fig. 20 [this file] ); they were far from identical when testing for the interaction with age instead of COVID-19 phenotype, suggesting that the inclusion of age was not introducing bias and that the COVID-19 interaction effects were not simply driven by shifts in age distribution ( Supplementary Note [this file] ). \n",
      "MPRAflow was used to associate the barcodes to individual 200-bp sequences and to count the number of DNA and RNA reads (Supplementary Fig. 26 [this file] ). As described in detail in the Supplementary Note [this file] , our analysis was based on a simplified model where RNA count follows a Poisson distribution with , where the translation rate is a function of the 200-bp sequence alone. \n",
      "We took the minimum PIPs from SuSiE and FINEMAP, resulting in a slightly more conservative CLPP estimation than using one of the two (Supplementary Fig. 13i [this file] ). \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM2: Further information on research design is available in the Nature Portfolio Reporting Summary [this file] linked to this article. \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM3: The fraction of MPRA hits (Supplementary Data 1 [this file] and Methods [rid=Sec15] ) increased along with the PIPs. \n",
      "Turning to a gene-centric view, we classified 557 genes carrying variants belonging to one (or, rarely, more than one) of the three QTL classes defined above (Supplementary Data 2 [this file] and Fig. 3d [rid=Fig3] ). \n",
      "We used large-scale fine-mapping results for 79 traits from the BBJ to quantify the level of trait-causal variant colocalization (Fig. 4a [rid=Fig4] , Supplementary Fig. 15 [rid=MOESM1] and Supplementary Data 3 [this file] ). \n",
      "This identified significant associations ( < 1 × 10 ) for 42 genes (Supplementary Data 4 [this file] ), including those between class 1 HLA alleles (especially, ) and killer immunoglobulin-like receptor ( ) genes ( at the mRNA level and KIRDL3 at the protein level; = 7.2 × 10 and = 1.2 × 10 ), providing a genetic basis for the well-known molecular interaction between KIR and class 1 HLA at the protein–protein level . \n",
      "Specifically, we performed COVID-19 severity interaction QTL (iQTL) analysis as described previously in and compared the interaction landscape at the mRNA and protein levels (Supplementary Figs. 19 [rid=MOESM1] and 20 [rid=MOESM1] and Supplementary Data 5 [this file] ). \n",
      "For the mass spectrometry-derived pQTL data, we downloaded the supplementary data from ref. (Supplementary Data 3 [this file] , summary of identified pQTLs) and matched them on the variant ID (lifted to hg38). \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 263 - INFO - Extracting links from full XML content.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 301 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 309 - INFO - Content length: 217411\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 326 - INFO - Prompt messages total length: 55106 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 332 - INFO - Prompt ID: gemini-2.5-flash-0-e2717c18a4c58151cd3a36f8b0cd075ce9631fcab5f00e4860629844dc09552a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 351 - INFO - Requesting datasets from content using model: gemini-2.5-flash, temperature: 0, messages length: 55106 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-81119bca-bd27-430a-8b5c-0b4602f9f8e0\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"hum0343\\\",\\n    \\\"repository_reference\\\": \\\"https://humandbs.biosciencedbc.jp/en/hum0343\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://japan-omics.jp/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"EGAS00001006284\\\",\\n    \\\"repository_reference\\\": \\\"https://ega-archive.org/studies/EGAS00001006284\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"hum0197\\\",\\n    \\\"repository_reference\\\": \\\"https://humandbs.biosciencedbc.jp/en/hum0197\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.finucanelab.org/data\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://gtexportal.org/home/datasets\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://hgdownload.soe.ucsc.edu/goldenPath/hg38/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://genome.ucsc.edu/cgi-bin/hgTables\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"http://nilanjanchatterjeelab.org/pwas/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn51365301\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn51365301\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"hum0343\\\",\\n    \\\"repository_reference\\\": \\\"https://humandbs.biosciencedbc.jp/en/hum0343\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://japan-omics.jp/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"EGAS00001006284\\\",\\n    \\\"repository_reference\\\": \\\"https://ega-archive.org/studies/EGAS00001006284\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"hum0197\\\",\\n    \\\"repository_reference\\\": \\\"https://humandbs.biosciencedbc.jp/en/hum0197\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.finucanelab.org/data\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://gtexportal.org/home/datasets\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://hgdownload.soe.ucsc.edu/goldenPath/hg38/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://genome.ucsc.edu/cgi-bin/hgTables\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"http://nilanjanchatterjeelab.org/pwas/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn51365301\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn51365301\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1758211138,\n",
      "    \"model\": \"gemini-2.5-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 71351,\n",
      "        \"completion_tokens\": 461,\n",
      "        \"total_tokens\": 74388,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 2576,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 664 - INFO - Deduplicating response with 10 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: hum0343\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: EGAS00001006284\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: hum0197\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn51365301\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'hum0343', 'repository_reference': 'https://humandbs.biosciencedbc.jp/en/hum0343', 'dataset_id': 'hum0343'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: hum0343\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID hum0343 is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://humandbs.biosciencedbc.jp/en/hum0343, identifier: hum0343, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://humandbs.biosciencedbc.jp/en/hum0343\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier hum0343 for https://humandbs.biosciencedbc.jp/en/hum0343\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier hum0343 for https://humandbs.biosciencedbc.jp/en/hum0343\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier hum0343 for https://humandbs.biosciencedbc.jp/en/hum0343\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier hum0343 for https://humandbs.biosciencedbc.jp/en/hum0343\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier hum0343 for https://humandbs.biosciencedbc.jp/en/hum0343\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier hum0343 for https://humandbs.biosciencedbc.jp/en/hum0343\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://humandbs.biosciencedbc.jp/en/hum0343\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'hum0343', 'data_repository': 'humandbs.biosciencedbc.jp', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://japan-omics.jp/', 'dataset_id': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://japan-omics.jp/, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://japan-omics.jp/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://japan-omics.jp/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'japan-omics.jp', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'EGAS00001006284', 'repository_reference': 'https://ega-archive.org/studies/EGAS00001006284', 'dataset_id': 'EGAS00001006284'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: EGAS00001006284\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: EGAS00001006284\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://ega-archive.org/studies/EGAS00001006284, identifier: EGAS00001006284, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://ega-archive.org/studies/EGAS00001006284\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier EGAS00001006284 for https://ega-archive.org/studies/EGAS00001006284\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier EGAS00001006284 for https://ega-archive.org/studies/EGAS00001006284\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier EGAS00001006284 for https://ega-archive.org/studies/EGAS00001006284\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier EGAS00001006284 for https://ega-archive.org/studies/EGAS00001006284\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier EGAS00001006284 for https://ega-archive.org/studies/EGAS00001006284\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier EGAS00001006284 for https://ega-archive.org/studies/EGAS00001006284\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://ega-archive.org/studies/EGAS00001006284\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://ega-archive.org/studies/EGAS00001006284\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://ega-archive.org/studies/EGAS00001006284\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: EGAS00001006284\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: EGAS00001006284\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: EGAS00001006284, ega-archive.org, https://ega-archive.org/studies/EGAS00001006284\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: EGAS00001006284\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: ega-archive.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://ega-archive.org/studies/EGAS00001006284\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'EGAS00001006284', 'data_repository': 'ega-archive.org', 'dataset_webpage': 'https://ega-archive.org/studies/EGAS00001006284', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'hum0197', 'repository_reference': 'https://humandbs.biosciencedbc.jp/en/hum0197', 'dataset_id': 'hum0197'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: hum0197\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID hum0197 is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://humandbs.biosciencedbc.jp/en/hum0197, identifier: hum0197, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://humandbs.biosciencedbc.jp/en/hum0197\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier hum0197 for https://humandbs.biosciencedbc.jp/en/hum0197\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier hum0197 for https://humandbs.biosciencedbc.jp/en/hum0197\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier hum0197 for https://humandbs.biosciencedbc.jp/en/hum0197\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier hum0197 for https://humandbs.biosciencedbc.jp/en/hum0197\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier hum0197 for https://humandbs.biosciencedbc.jp/en/hum0197\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier hum0197 for https://humandbs.biosciencedbc.jp/en/hum0197\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://humandbs.biosciencedbc.jp/en/hum0197\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'hum0197', 'data_repository': 'humandbs.biosciencedbc.jp', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.finucanelab.org/data'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.finucanelab.org/data, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.finucanelab.org/data\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://www.finucanelab.org/data\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'www.finucanelab.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://gtexportal.org/home/datasets'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://gtexportal.org/home/datasets, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://gtexportal.org/home/datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://gtexportal.org/home/datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'gtexportal.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://hgdownload.soe.ucsc.edu/goldenPath/hg38/'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://hgdownload.soe.ucsc.edu/goldenPath/hg38/, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://hgdownload.soe.ucsc.edu/goldenPath/hg38/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://hgdownload.soe.ucsc.edu/goldenPath/hg38/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'hgdownload.soe.ucsc.edu', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://genome.ucsc.edu/cgi-bin/hgTables'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://genome.ucsc.edu/cgi-bin/hgTables, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://genome.ucsc.edu/cgi-bin/hgTables\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://genome.ucsc.edu/cgi-bin/hgTables\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'genome.ucsc.edu', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'http://nilanjanchatterjeelab.org/pwas/'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: http://nilanjanchatterjeelab.org/pwas/, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: http://nilanjanchatterjeelab.org/pwas/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: http://nilanjanchatterjeelab.org/pwas/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'nilanjanchatterjeelab.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn51365301', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn51365301', 'dataset_id': 'syn51365301'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn51365301\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn51365301\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn51365301, identifier: syn51365301, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn51365301\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn51365301 for https://www.synapse.org/#!Synapse:syn51365301\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn51365301 for https://www.synapse.org/#!Synapse:syn51365301\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn51365301 for https://www.synapse.org/#!Synapse:syn51365301\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn51365301 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn51365301\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn51365301\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn51365301\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn51365301\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn51365301\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn51365301\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn51365301\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn51365301\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn51365301\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn51365301, synapse.org, https://www.synapse.org/#!Synapse:syn51365301\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn51365301\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn51365301\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn51365301', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn51365301', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 278 - INFO - Augmented dataset links: [{'dataset_identifier': 'hum0343', 'data_repository': 'humandbs.biosciencedbc.jp', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'japan-omics.jp', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'EGAS00001006284', 'data_repository': 'ega-archive.org', 'dataset_webpage': 'https://ega-archive.org/studies/EGAS00001006284', 'citation_type': 'n/a'}, {'dataset_identifier': 'hum0197', 'data_repository': 'humandbs.biosciencedbc.jp', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'www.finucanelab.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'gtexportal.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'hgdownload.soe.ucsc.edu', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'genome.ucsc.edu', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'nilanjanchatterjeelab.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn51365301', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn51365301', 'citation_type': 'n/a'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1191 - INFO - Fetching metadata for 10 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: hum0343\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: humandbs.biosciencedbc.jp, identifier: hum0343, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier hum0343 for humandbs.biosciencedbc.jp\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier hum0343 for humandbs.biosciencedbc.jp\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier hum0343 for humandbs.biosciencedbc.jp\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier hum0343 for humandbs.biosciencedbc.jp\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier hum0343 for humandbs.biosciencedbc.jp\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier hum0343 for humandbs.biosciencedbc.jp\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: humandbs.biosciencedbc.jp\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL humandbs.biosciencedbc.jp may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for hum0343 in humandbs.biosciencedbc.jp\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 1 with repo: humandbs.biosciencedbc.jp and accession_id: hum0343\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository humandbs.biosciencedbc.jp unknown in Ontology. Skipping dataset page 1.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 2: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: EGAS00001006284\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: ega-archive.org, identifier: EGAS00001006284, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: ega-archive.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for EGAS00001006284 in ega-archive.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 3 with repo: ega-archive.org and accession_id: EGAS00001006284\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://ega-archive.org/studies/EGAS00001006284\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 3: Contact to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: hum0197\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: humandbs.biosciencedbc.jp, identifier: hum0197, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier hum0197 for humandbs.biosciencedbc.jp\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier hum0197 for humandbs.biosciencedbc.jp\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier hum0197 for humandbs.biosciencedbc.jp\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier hum0197 for humandbs.biosciencedbc.jp\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier hum0197 for humandbs.biosciencedbc.jp\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier hum0197 for humandbs.biosciencedbc.jp\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: humandbs.biosciencedbc.jp\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL humandbs.biosciencedbc.jp may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for hum0197 in humandbs.biosciencedbc.jp\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 4 with repo: humandbs.biosciencedbc.jp and accession_id: hum0197\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository humandbs.biosciencedbc.jp unknown in Ontology. Skipping dataset page 4.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 5: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 6: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 7: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 8: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 9: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 10 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn51365301\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn51365301, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn51365301 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn51365301 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn51365301 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn51365301 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 10 with repo: synapse.org and accession_id: syn51365301\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn51365301\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 10: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1277 - INFO - Updated datasets len: 10\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 287 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 13, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'source_url', 'download_link',\n",
      "       'title', 'content_type', 'id', 'caption', 'description',\n",
      "       'source_section', 'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 500 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 506 - INFO - XML element classification not needed. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1012 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11525184\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1020 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11525184\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 670th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11525270)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11525270\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11525270 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 634 - INFO - Fetching data for pmc11525270\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 637 - INFO - Fetching data from scripts/exp_input/Local_fetched_data_1.parquet\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 438 - INFO - Raw xml data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11525270 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 452 - INFO - Skipping raw HTML/XML saving.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 458 - INFO - Using XMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 52 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 24 - INFO - Initializing xmlRetriever\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 466 - INFO - No additional data provided. Parsing raw data only.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 147 - INFO - Function call: parse_data(api_data(<class 'str'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11525270, additional_data, XML)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 162 - INFO - api_data converted to lxml element\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 466 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 153 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 154 - INFO - Consider migrating this function to the BaseRetriever class.\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 69 - INFO - Loading target sections for section name: supplementary_material_sections\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 477 - INFO - Found 2 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x15fa193c0>, <Element supplementary-material at 0x15fa1a240>]\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 485 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x15fa193c0>, <Element supplementary-material at 0x15fa1a240>]}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 574 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM1: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM2: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 263 - INFO - Extracting links from full XML content.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 301 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 309 - INFO - Content length: 147250\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 326 - INFO - Prompt messages total length: 37566 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 332 - INFO - Prompt ID: gemini-2.5-flash-0-cfc6389db27e143ea06aa090d0591721d4a81dd7e2c8ad6d3f04a79fdf59912e\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 351 - INFO - Requesting datasets from content using model: gemini-2.5-flash, temperature: 0, messages length: 37566 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-8082b966-267e-4d71-94b8-9bf306a00cc8\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.radc.rush.edu\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn51247298\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/Synapse:syn51247298\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://www.radc.rush.edu\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn51247298\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/Synapse:syn51247298\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1758211161,\n",
      "    \"model\": \"gemini-2.5-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 45311,\n",
      "        \"completion_tokens\": 99,\n",
      "        \"total_tokens\": 48318,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 2908,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 664 - INFO - Deduplicating response with 2 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn51247298\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://www.radc.rush.edu', 'dataset_id': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.radc.rush.edu, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.radc.rush.edu\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://www.radc.rush.edu\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'www.radc.rush.edu', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn51247298', 'repository_reference': 'https://www.synapse.org/Synapse:syn51247298', 'dataset_id': 'syn51247298'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn51247298\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn51247298\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/Synapse:syn51247298, identifier: syn51247298, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/Synapse:syn51247298\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn51247298 for https://www.synapse.org/Synapse:syn51247298\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn51247298 for https://www.synapse.org/Synapse:syn51247298\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn51247298 for https://www.synapse.org/Synapse:syn51247298\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/Synapse:syn51247298 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/Synapse:syn51247298\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/Synapse:syn51247298\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/Synapse:syn51247298\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn51247298\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn51247298\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn51247298\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/Synapse:syn51247298\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn51247298\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn51247298\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn51247298, synapse.org, https://www.synapse.org/Synapse:syn51247298\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn51247298\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/Synapse:syn51247298\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn51247298', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/Synapse:syn51247298', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 278 - INFO - Augmented dataset links: [{'dataset_identifier': 'n/a', 'data_repository': 'www.radc.rush.edu', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn51247298', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/Synapse:syn51247298', 'citation_type': 'n/a'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1191 - INFO - Fetching metadata for 2 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 1: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn51247298\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn51247298, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn51247298 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn51247298 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn51247298 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn51247298 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 2 with repo: synapse.org and accession_id: syn51247298\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn51247298\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 2: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1277 - INFO - Updated datasets len: 2\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 287 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 4, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'source_url', 'download_link',\n",
      "       'title', 'content_type', 'id', 'caption', 'description',\n",
      "       'source_section', 'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 500 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 506 - INFO - XML element classification not needed. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1012 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11525270\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1020 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11525270\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 603 - INFO - \n",
      "Progress: 671/2190 (30.64%) | Elapsed: 00:21:21 | ETA: 00:48:20\n",
      "\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 671th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11525589)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11525589\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11525589 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 634 - INFO - Fetching data for pmc11525589\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 637 - INFO - Fetching data from scripts/exp_input/Local_fetched_data_1.parquet\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 438 - INFO - Raw xml data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11525589 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 452 - INFO - Skipping raw HTML/XML saving.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 458 - INFO - Using XMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 52 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 24 - INFO - Initializing xmlRetriever\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 466 - INFO - No additional data provided. Parsing raw data only.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 147 - INFO - Function call: parse_data(api_data(<class 'str'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11525589, additional_data, XML)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 162 - INFO - api_data converted to lxml element\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 466 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 153 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 154 - INFO - Consider migrating this function to the BaseRetriever class.\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 69 - INFO - Loading target sections for section name: supplementary_material_sections\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 477 - INFO - Found 5 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x15ebe3000>, <Element supplementary-material at 0x15ebe2600>, <Element supplementary-material at 0x15e2c4500>, <Element supplementary-material at 0x15e2c4dc0>, <Element supplementary-material at 0x15e2c6300>]\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 485 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x15ebe3000>, <Element supplementary-material at 0x15ebe2600>, <Element supplementary-material at 0x15e2c4500>, <Element supplementary-material at 0x15e2c4dc0>, <Element supplementary-material at 0x15e2c6300>]}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 574 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM1: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM2: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM3: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM4: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM5: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 263 - INFO - Extracting links from full XML content.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 301 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 309 - INFO - Content length: 84382\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 326 - INFO - Prompt messages total length: 21849 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 332 - INFO - Prompt ID: gemini-2.5-flash-0-f165759c7ec0834373fa35615629c345de71f9321425dc90ca51f7d1bde14bea\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 351 - INFO - Requesting datasets from content using model: gemini-2.5-flash, temperature: 0, messages length: 21849 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-0f98f90b-1045-4d09-acf1-010328a3f639\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.7303/syn51514106\\\",\\n    \\\"repository_reference\\\": \\\"https://doi.org/10.7303/syn51514106\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"10.7303/syn51514106\\\",\\n    \\\"repository_reference\\\": \\\"https://doi.org/10.7303/syn51514106\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1758211178,\n",
      "    \"model\": \"gemini-2.5-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 27099,\n",
      "        \"completion_tokens\": 72,\n",
      "        \"total_tokens\": 29685,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 2514,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 664 - INFO - Deduplicating response with 1 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: 10.7303/syn51514106\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': '10.7303/syn51514106', 'repository_reference': 'https://doi.org/10.7303/syn51514106', 'dataset_id': '10.7303/syn51514106'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: 10.7303/syn51514106\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 607 - INFO - Setting dataset_webpage to DOI URL value: https://doi.org/10.7303/syn51514106\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: 10.7303/syn51514106\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://doi.org/10.7303/syn51514106, identifier: 10.7303/syn51514106, dataset_page: https://doi.org/10.7303/syn51514106\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://doi.org/10.7303/syn51514106\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.7303/syn51514106 for https://doi.org/10.7303/syn51514106\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier 10.7303/syn51514106 for https://doi.org/10.7303/syn51514106\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier 10.7303/syn51514106 for https://doi.org/10.7303/syn51514106\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier 10.7303/syn51514106 for https://doi.org/10.7303/syn51514106\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier 10.7303/syn51514106 for https://doi.org/10.7303/syn51514106\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1157 - INFO - Found id_pattern match for https://doi.org/10.7303/syn51514106 in 10\\.\\d{4,9}/[^\\s,]+\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://doi.org/10.7303/syn51514106\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: 10.7303/syn51514106\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: 10.7303/syn51514106\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: 10\\.\\d{4,9}/[^\\s,]+ to value: 10.7303/syn51514106\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://doi.org/__ID__ to value: https://doi.org/10.7303/syn51514106\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: 10\\.\\d{4,9}/[^\\s,]+ to value: https://doi.org/10.7303/syn51514106\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(10\\.\\d{4,9}/[^\\s,]+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(10\\.\\d{4,9}/[^\\s,]+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: 10.7303/syn51514106\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: 10\\.\\d{4,9}/[^\\s,]+ to value: 10.7303/syn51514106\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: 10.7303/syn51514106\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://doi.org/10.7303/syn51514106\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: 10.7303/syn51514106\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: 10.7303/syn51514106\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: 10.7303/syn51514106, doi.org, https://doi.org/10.7303/syn51514106\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: 10.7303/syn51514106\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: doi.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://doi.org/10.7303/syn51514106\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': '10.7303/syn51514106', 'data_repository': 'doi.org', 'dataset_webpage': 'https://doi.org/10.7303/syn51514106', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 278 - INFO - Augmented dataset links: [{'dataset_identifier': '10.7303/syn51514106', 'data_repository': 'doi.org', 'dataset_webpage': 'https://doi.org/10.7303/syn51514106', 'citation_type': 'n/a'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1191 - INFO - Fetching metadata for 1 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: 10.7303/syn51514106\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: doi.org, identifier: 10.7303/syn51514106, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 10.7303/syn51514106 for doi.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier 10.7303/syn51514106 for doi.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier 10.7303/syn51514106 for doi.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier 10.7303/syn51514106 for doi.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: doi.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for 10.7303/syn51514106 in doi.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 1 with repo: doi.org and accession_id: 10.7303/syn51514106\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://doi.org/10.7303/syn51514106\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1277 - INFO - Updated datasets len: 1\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 287 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 6, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'link', 'source_url', 'download_link', 'title',\n",
      "       'content_type', 'id', 'caption', 'description', 'source_section',\n",
      "       'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 500 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 506 - INFO - XML element classification not needed. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1012 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11525589\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1020 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11525589\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 672th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11525848)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11525848\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11525848 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 634 - INFO - Fetching data for pmc11525848\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 637 - INFO - Fetching data from scripts/exp_input/Local_fetched_data_1.parquet\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 438 - INFO - Raw xml data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11525848 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 452 - INFO - Skipping raw HTML/XML saving.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 458 - INFO - Using XMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 52 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 24 - INFO - Initializing xmlRetriever\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 466 - INFO - No additional data provided. Parsing raw data only.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 147 - INFO - Function call: parse_data(api_data(<class 'str'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11525848, additional_data, XML)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 162 - INFO - api_data converted to lxml element\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 466 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 153 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 154 - INFO - Consider migrating this function to the BaseRetriever class.\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 69 - INFO - Loading target sections for section name: supplementary_material_sections\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 477 - INFO - Found 1 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x15f6e3600>]\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 485 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x15f6e3600>]}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 574 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM1: Additionally, we employed SPIRAL to analyze bulk RNA-seq samples (see Supplementary Table 1 [this file] for the list of datasets). \n",
      "The clustering made in the first step to create 100 repcells in this dataset appears in Supplementary Fig. 1 [this file] . \n",
      "SPIRAL results indicate that genes involved in translation activity are highly expressed at the beginning of differentiation, moderately expressed at the middle and less expressed at the end of differentiation process (Supplementary Fig. 2 [this file] , Structure A). Expression of genes involved in anatomical structure development starts low (days 0-10), increases in the middle of the process (11-17) and then decreases again towards the end of the process (Supplementary Fig. 2 [this file] , Structure B). Finally, genes responsible for regulating cell differentiation, cell migration, and T-cell activation displayed an initial low expression, followed by a consistent increase throughout the differentiation process, reaching their peak expression levels towards the end (Supplementary Fig. 2 [this file] , Structure C). SPIRAL also detected batch effects between replicates, manifested as one replicate in each color (Supplementary Fig. 2 [this file] , Structure D). \n",
      "SPIRAL identified increased regulation of B cell activation at minutes post stimulation (Supplementary Fig. 3 [this file] , Structure C), increased regulation of cell morphogenesis and cell differentiation at minutes post stimulation (Supplementary Fig. 3 [this file] , Structure B), increased lymphocyte and leukocyte differentiation at minutes (Supplementary Fig. 3 [this file] , Structure D), and increased translational activity at minutes (Supplementary Fig. 3 [this file] , Structure A). SPIRAL also detected batch effects (Supplementary Fig. 3 [this file] , structures E and F). \n",
      "We discuss our work in this perspective in Supplementary Section 1 [this file] . \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 263 - INFO - Extracting links from full XML content.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 301 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 309 - INFO - Content length: 274307\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 326 - INFO - Prompt messages total length: 69330 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 332 - INFO - Prompt ID: gemini-2.5-flash-0-add5807c3ef469adce74fec17a31b041c2532873382c1099fa8db907a09413b8\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 351 - INFO - Requesting datasets from content using model: gemini-2.5-flash, temperature: 0, messages length: 69330 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-f542a0fa-6384-42a1-8d23-20234c93910d\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE111912\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE111912\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE112294\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE112294\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"mouse-brain-serial-section-2-sagittal-posterior-1-standard-1-1-0\\\",\\n    \\\"repository_reference\\\": \\\"https://www.10xgenomics.com/resources/datasets/mouse-brain-serial-section-2-sagittal-posterior-1-standard-1-1-0\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"normal-human-prostate-ffpe-1-standard-1-3-0\\\",\\n    \\\"repository_reference\\\": \\\"https://www.10xgenomics.com/resources/datasets/normal-human-prostate-ffpe-1-standard-1-3-0\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn22153884\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn22153884\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE129536\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE129536\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"8009633\\\",\\n    \\\"repository_reference\\\": \\\"https://zenodo.org/record/8009633\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE111912\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE111912\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE112294\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE112294\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"mouse-brain-serial-section-2-sagittal-posterior-1-standard-1-1-0\\\",\\n    \\\"repository_reference\\\": \\\"https://www.10xgenomics.com/resources/datasets/mouse-brain-serial-section-2-sagittal-posterior-1-standard-1-1-0\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"normal-human-prostate-ffpe-1-standard-1-3-0\\\",\\n    \\\"repository_reference\\\": \\\"https://www.10xgenomics.com/resources/datasets/normal-human-prostate-ffpe-1-standard-1-3-0\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn22153884\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn22153884\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE129536\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE129536\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"8009633\\\",\\n    \\\"repository_reference\\\": \\\"https://zenodo.org/record/8009633\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1758211199,\n",
      "    \"model\": \"gemini-2.5-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 93925,\n",
      "        \"completion_tokens\": 464,\n",
      "        \"total_tokens\": 98051,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 3662,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 664 - INFO - Deduplicating response with 7 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: GSE111912\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: GSE112294\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: mouse-brain-serial-section-2-sagittal-posterior-1-standard-1-1-0\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: normal-human-prostate-ffpe-1-standard-1-3-0\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn22153884\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: GSE129536\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: 8009633\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'GSE111912', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE111912', 'dataset_id': 'GSE111912'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE111912\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: GSE111912\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE111912, identifier: GSE111912, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE111912\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE111912 for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE111912\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier GSE111912 for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE111912\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1157 - INFO - Found id_pattern match for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE111912 in GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE111912\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE111912\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: GSE111912\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE111912\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE111912\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE111912\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: GSE111912\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE111912\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: GSE111912, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE111912\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: GSE111912\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE111912\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'GSE111912', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE111912', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'GSE112294', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE112294', 'dataset_id': 'GSE112294'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE112294\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: GSE112294\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE112294, identifier: GSE112294, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE112294\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE112294 for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE112294\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier GSE112294 for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE112294\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1157 - INFO - Found id_pattern match for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE112294 in GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE112294\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE112294\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: GSE112294\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE112294\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE112294\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE112294\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: GSE112294\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE112294\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: GSE112294, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE112294\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: GSE112294\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE112294\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'GSE112294', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE112294', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'mouse-brain-serial-section-2-sagittal-posterior-1-standard-1-1-0', 'repository_reference': 'https://www.10xgenomics.com/resources/datasets/mouse-brain-serial-section-2-sagittal-posterior-1-standard-1-1-0', 'dataset_id': 'mouse-brain-serial-section-2-sagittal-posterior-1-standard-1-1-0'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.10xgenomics.com/resources/datasets/mouse-brain-serial-section-2-sagittal-posterior-1-standard-1-1-0\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://www.10xgenomics.com/resources/datasets/mouse-brain-serial-section-2-sagittal-posterior-1-standard-1-1-0\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, https://www.10xgenomics.com/resources/datasets/mouse-brain-serial-section-2-sagittal-posterior-1-standard-1-1-0\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: mouse-brain-serial-section-2-sagittal-posterior-1-standard-1-1-0\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID mouse-brain-serial-section-2-sagittal-posterior-1-standard-1-1-0 is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.10xgenomics.com/resources/datasets/mouse-brain-serial-section-2-sagittal-posterior-1-standard-1-1-0, identifier: mouse-brain-serial-section-2-sagittal-posterior-1-standard-1-1-0, dataset_page: https://www.10xgenomics.com/resources/datasets/mouse-brain-serial-section-2-sagittal-posterior-1-standard-1-1-0\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.10xgenomics.com/resources/datasets/mouse-brain-serial-section-2-sagittal-posterior-1-standard-1-1-0\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier mouse-brain-serial-section-2-sagittal-posterior-1-standard-1-1-0 for https://www.10xgenomics.com/resources/datasets/mouse-brain-serial-section-2-sagittal-posterior-1-standard-1-1-0\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier mouse-brain-serial-section-2-sagittal-posterior-1-standard-1-1-0 for https://www.10xgenomics.com/resources/datasets/mouse-brain-serial-section-2-sagittal-posterior-1-standard-1-1-0\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier mouse-brain-serial-section-2-sagittal-posterior-1-standard-1-1-0 for https://www.10xgenomics.com/resources/datasets/mouse-brain-serial-section-2-sagittal-posterior-1-standard-1-1-0\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier mouse-brain-serial-section-2-sagittal-posterior-1-standard-1-1-0 for https://www.10xgenomics.com/resources/datasets/mouse-brain-serial-section-2-sagittal-posterior-1-standard-1-1-0\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier mouse-brain-serial-section-2-sagittal-posterior-1-standard-1-1-0 for https://www.10xgenomics.com/resources/datasets/mouse-brain-serial-section-2-sagittal-posterior-1-standard-1-1-0\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier mouse-brain-serial-section-2-sagittal-posterior-1-standard-1-1-0 for https://www.10xgenomics.com/resources/datasets/mouse-brain-serial-section-2-sagittal-posterior-1-standard-1-1-0\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://www.10xgenomics.com/resources/datasets/mouse-brain-serial-section-2-sagittal-posterior-1-standard-1-1-0\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.10xgenomics.com/resources/datasets/mouse-brain-serial-section-2-sagittal-posterior-1-standard-1-1-0\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'mouse-brain-serial-section-2-sagittal-posterior-1-standard-1-1-0', 'data_repository': 'www.10xgenomics.com', 'dataset_webpage': 'https://www.10xgenomics.com/resources/datasets/mouse-brain-serial-section-2-sagittal-posterior-1-standard-1-1-0', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'normal-human-prostate-ffpe-1-standard-1-3-0', 'repository_reference': 'https://www.10xgenomics.com/resources/datasets/normal-human-prostate-ffpe-1-standard-1-3-0', 'dataset_id': 'normal-human-prostate-ffpe-1-standard-1-3-0'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.10xgenomics.com/resources/datasets/normal-human-prostate-ffpe-1-standard-1-3-0\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://www.10xgenomics.com/resources/datasets/normal-human-prostate-ffpe-1-standard-1-3-0\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, https://www.10xgenomics.com/resources/datasets/normal-human-prostate-ffpe-1-standard-1-3-0\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: normal-human-prostate-ffpe-1-standard-1-3-0\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID normal-human-prostate-ffpe-1-standard-1-3-0 is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.10xgenomics.com/resources/datasets/normal-human-prostate-ffpe-1-standard-1-3-0, identifier: normal-human-prostate-ffpe-1-standard-1-3-0, dataset_page: https://www.10xgenomics.com/resources/datasets/normal-human-prostate-ffpe-1-standard-1-3-0\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.10xgenomics.com/resources/datasets/normal-human-prostate-ffpe-1-standard-1-3-0\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier normal-human-prostate-ffpe-1-standard-1-3-0 for https://www.10xgenomics.com/resources/datasets/normal-human-prostate-ffpe-1-standard-1-3-0\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier normal-human-prostate-ffpe-1-standard-1-3-0 for https://www.10xgenomics.com/resources/datasets/normal-human-prostate-ffpe-1-standard-1-3-0\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier normal-human-prostate-ffpe-1-standard-1-3-0 for https://www.10xgenomics.com/resources/datasets/normal-human-prostate-ffpe-1-standard-1-3-0\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier normal-human-prostate-ffpe-1-standard-1-3-0 for https://www.10xgenomics.com/resources/datasets/normal-human-prostate-ffpe-1-standard-1-3-0\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier normal-human-prostate-ffpe-1-standard-1-3-0 for https://www.10xgenomics.com/resources/datasets/normal-human-prostate-ffpe-1-standard-1-3-0\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier normal-human-prostate-ffpe-1-standard-1-3-0 for https://www.10xgenomics.com/resources/datasets/normal-human-prostate-ffpe-1-standard-1-3-0\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://www.10xgenomics.com/resources/datasets/normal-human-prostate-ffpe-1-standard-1-3-0\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.10xgenomics.com/resources/datasets/normal-human-prostate-ffpe-1-standard-1-3-0\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'normal-human-prostate-ffpe-1-standard-1-3-0', 'data_repository': 'www.10xgenomics.com', 'dataset_webpage': 'https://www.10xgenomics.com/resources/datasets/normal-human-prostate-ffpe-1-standard-1-3-0', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn22153884', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn22153884', 'dataset_id': 'syn22153884'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn22153884\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn22153884\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn22153884, identifier: syn22153884, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn22153884\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn22153884 for https://www.synapse.org/#!Synapse:syn22153884\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn22153884 for https://www.synapse.org/#!Synapse:syn22153884\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn22153884 for https://www.synapse.org/#!Synapse:syn22153884\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn22153884 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn22153884\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn22153884\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn22153884\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn22153884\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn22153884\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn22153884\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn22153884\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn22153884\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn22153884\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn22153884, synapse.org, https://www.synapse.org/#!Synapse:syn22153884\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn22153884\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn22153884\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn22153884', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn22153884', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'GSE129536', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE129536', 'dataset_id': 'GSE129536'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE129536\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: GSE129536\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE129536, identifier: GSE129536, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE129536\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE129536 for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE129536\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier GSE129536 for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE129536\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1157 - INFO - Found id_pattern match for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE129536 in GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE129536\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE129536\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: GSE129536\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE129536\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE129536\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE129536\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: GSE129536\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE129536\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: GSE129536, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE129536\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: GSE129536\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE129536\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'GSE129536', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE129536', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': '8009633', 'repository_reference': 'https://zenodo.org/record/8009633', 'dataset_id': '8009633'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: 8009633\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: 8009633\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://zenodo.org/record/8009633, identifier: 8009633, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://zenodo.org/record/8009633\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 8009633 for https://zenodo.org/record/8009633\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier 8009633 for https://zenodo.org/record/8009633\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier 8009633 for https://zenodo.org/record/8009633\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier 8009633 for https://zenodo.org/record/8009633\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier 8009633 for https://zenodo.org/record/8009633\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier 8009633 for https://zenodo.org/record/8009633\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1157 - INFO - Found id_pattern match for https://zenodo.org/record/8009633 in [1-9][a-zA-Z0-9]{3}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://zenodo.org/record/8009633\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://zenodo.org/record/8009633\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: 8009633\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: 8009633\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: 8009633, pdb, https://zenodo.org/record/8009633\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: 8009633\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://zenodo.org/record/8009633\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': '8009633', 'data_repository': 'pdb', 'dataset_webpage': 'https://zenodo.org/record/8009633', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 278 - INFO - Augmented dataset links: [{'dataset_identifier': 'GSE111912', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE111912', 'citation_type': 'n/a'}, {'dataset_identifier': 'GSE112294', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE112294', 'citation_type': 'n/a'}, {'dataset_identifier': 'mouse-brain-serial-section-2-sagittal-posterior-1-standard-1-1-0', 'data_repository': 'www.10xgenomics.com', 'dataset_webpage': 'https://www.10xgenomics.com/resources/datasets/mouse-brain-serial-section-2-sagittal-posterior-1-standard-1-1-0', 'citation_type': 'n/a'}, {'dataset_identifier': 'normal-human-prostate-ffpe-1-standard-1-3-0', 'data_repository': 'www.10xgenomics.com', 'dataset_webpage': 'https://www.10xgenomics.com/resources/datasets/normal-human-prostate-ffpe-1-standard-1-3-0', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn22153884', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn22153884', 'citation_type': 'n/a'}, {'dataset_identifier': 'GSE129536', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE129536', 'citation_type': 'n/a'}, {'dataset_identifier': '8009633', 'data_repository': 'pdb', 'dataset_webpage': 'https://zenodo.org/record/8009633', 'citation_type': 'n/a'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1191 - INFO - Fetching metadata for 7 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: GSE111912\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: geo, identifier: GSE111912, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE111912 for geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for GSE111912 in geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE111912\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE111912\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 1: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: GSE112294\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: geo, identifier: GSE112294, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE112294 for geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for GSE112294 in geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 2 with repo: geo and accession_id: GSE112294\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE112294\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 2: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: mouse-brain-serial-section-2-sagittal-posterior-1-standard-1-1-0\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: www.10xgenomics.com, identifier: mouse-brain-serial-section-2-sagittal-posterior-1-standard-1-1-0, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier mouse-brain-serial-section-2-sagittal-posterior-1-standard-1-1-0 for www.10xgenomics.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier mouse-brain-serial-section-2-sagittal-posterior-1-standard-1-1-0 for www.10xgenomics.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier mouse-brain-serial-section-2-sagittal-posterior-1-standard-1-1-0 for www.10xgenomics.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier mouse-brain-serial-section-2-sagittal-posterior-1-standard-1-1-0 for www.10xgenomics.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier mouse-brain-serial-section-2-sagittal-posterior-1-standard-1-1-0 for www.10xgenomics.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier mouse-brain-serial-section-2-sagittal-posterior-1-standard-1-1-0 for www.10xgenomics.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: www.10xgenomics.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL www.10xgenomics.com may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for mouse-brain-serial-section-2-sagittal-posterior-1-standard-1-1-0 in www.10xgenomics.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 3 with repo: www.10xgenomics.com and accession_id: mouse-brain-serial-section-2-sagittal-posterior-1-standard-1-1-0\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository www.10xgenomics.com unknown in Ontology. Skipping dataset page 3.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: normal-human-prostate-ffpe-1-standard-1-3-0\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: www.10xgenomics.com, identifier: normal-human-prostate-ffpe-1-standard-1-3-0, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier normal-human-prostate-ffpe-1-standard-1-3-0 for www.10xgenomics.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier normal-human-prostate-ffpe-1-standard-1-3-0 for www.10xgenomics.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier normal-human-prostate-ffpe-1-standard-1-3-0 for www.10xgenomics.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier normal-human-prostate-ffpe-1-standard-1-3-0 for www.10xgenomics.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier normal-human-prostate-ffpe-1-standard-1-3-0 for www.10xgenomics.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier normal-human-prostate-ffpe-1-standard-1-3-0 for www.10xgenomics.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: www.10xgenomics.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL www.10xgenomics.com may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for normal-human-prostate-ffpe-1-standard-1-3-0 in www.10xgenomics.com\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 4 with repo: www.10xgenomics.com and accession_id: normal-human-prostate-ffpe-1-standard-1-3-0\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository www.10xgenomics.com unknown in Ontology. Skipping dataset page 4.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn22153884\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn22153884, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn22153884 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn22153884 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn22153884 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn22153884 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 5 with repo: synapse.org and accession_id: syn22153884\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn22153884\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 5: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: GSE129536\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: geo, identifier: GSE129536, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE129536 for geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for GSE129536 in geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 6 with repo: geo and accession_id: GSE129536\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE129536\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 6: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: 8009633\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: pdb, identifier: 8009633, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier 8009633 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier 8009633 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier 8009633 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier 8009633 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier 8009633 for pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for 8009633 in pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 7 with repo: pdb and accession_id: 8009633\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.rcsb.org/structure/8009633\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1277 - INFO - Updated datasets len: 7\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 287 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 8, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'source_url', 'download_link',\n",
      "       'title', 'content_type', 'id', 'caption', 'description',\n",
      "       'source_section', 'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 500 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 506 - INFO - XML element classification not needed. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1012 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11525848\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1020 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11525848\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 673th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11526681)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11526681\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11526681 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 634 - INFO - Fetching data for pmc11526681\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 637 - INFO - Fetching data from scripts/exp_input/Local_fetched_data_1.parquet\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 438 - INFO - Raw xml data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11526681 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 452 - INFO - Skipping raw HTML/XML saving.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 458 - INFO - Using XMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 52 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 24 - INFO - Initializing xmlRetriever\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 466 - INFO - No additional data provided. Parsing raw data only.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 147 - INFO - Function call: parse_data(api_data(<class 'str'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11526681, additional_data, XML)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 162 - INFO - api_data converted to lxml element\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 466 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 153 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 154 - INFO - Consider migrating this function to the BaseRetriever class.\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 69 - INFO - Loading target sections for section name: supplementary_material_sections\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 477 - INFO - Found 10 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x15e4e7840>, <Element supplementary-material at 0x15e4e4880>, <Element supplementary-material at 0x15e4e6180>, <Element supplementary-material at 0x15e4e4240>, <Element supplementary-material at 0x15e4e5240>, <Element supplementary-material at 0x15e4e4c40>, <Element supplementary-material at 0x15e4e4340>, <Element supplementary-material at 0x15e4e5bc0>, <Element supplementary-material at 0x15e4e5400>, <Element supplementary-material at 0x15e4e5940>]\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 485 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x15e4e7840>, <Element supplementary-material at 0x15e4e4880>, <Element supplementary-material at 0x15e4e6180>, <Element supplementary-material at 0x15e4e4240>, <Element supplementary-material at 0x15e4e5240>, <Element supplementary-material at 0x15e4e4c40>, <Element supplementary-material at 0x15e4e4340>, <Element supplementary-material at 0x15e4e5bc0>, <Element supplementary-material at 0x15e4e5400>, <Element supplementary-material at 0x15e4e5940>]}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 574 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM1: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM2: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM3: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM4: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM5: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM6: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM7: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM8: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM9: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM10: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 263 - INFO - Extracting links from full XML content.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 301 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 309 - INFO - Content length: 147365\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 326 - INFO - Prompt messages total length: 37595 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 332 - INFO - Prompt ID: gemini-2.5-flash-0-07471668fef0a64507c9699b4b822060bc49880eed087d55617215450c0e4c1d\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 351 - INFO - Requesting datasets from content using model: gemini-2.5-flash, temperature: 0, messages length: 37595 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-c51c8e99-8ec2-4604-b3a5-a45ff1897fa5\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE202203\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE202203\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE62944\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE62944\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn1688369\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn1688369\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE73526\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE73526\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE202203\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE202203\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE62944\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE62944\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn1688369\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn1688369\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE73526\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE73526\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1758211212,\n",
      "    \"model\": \"gemini-2.5-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 49842,\n",
      "        \"completion_tokens\": 243,\n",
      "        \"total_tokens\": 51452,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 1367,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 664 - INFO - Deduplicating response with 4 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: GSE202203\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: GSE62944\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn1688369\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: GSE73526\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'GSE202203', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE202203', 'dataset_id': 'GSE202203'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE202203\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: GSE202203\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE202203, identifier: GSE202203, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE202203\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE202203 for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE202203\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier GSE202203 for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE202203\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1157 - INFO - Found id_pattern match for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE202203 in GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE202203\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE202203\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: GSE202203\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE202203\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE202203\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE202203\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: GSE202203\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE202203\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: GSE202203, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE202203\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: GSE202203\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE202203\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'GSE202203', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE202203', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'GSE62944', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE62944', 'dataset_id': 'GSE62944'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE62944\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: GSE62944\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE62944, identifier: GSE62944, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE62944\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE62944 for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE62944\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier GSE62944 for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE62944\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1157 - INFO - Found id_pattern match for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE62944 in GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE62944\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE62944\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: GSE62944\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE62944\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE62944\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE62944\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: GSE62944\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE62944\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: GSE62944, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE62944\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: GSE62944\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE62944\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'GSE62944', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE62944', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn1688369', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn1688369', 'dataset_id': 'syn1688369'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn1688369\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn1688369\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn1688369, identifier: syn1688369, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn1688369\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn1688369 for https://www.synapse.org/#!Synapse:syn1688369\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn1688369 for https://www.synapse.org/#!Synapse:syn1688369\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn1688369 for https://www.synapse.org/#!Synapse:syn1688369\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn1688369 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn1688369\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn1688369\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn1688369\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn1688369\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn1688369\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn1688369\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn1688369\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn1688369\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn1688369\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn1688369, synapse.org, https://www.synapse.org/#!Synapse:syn1688369\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn1688369\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn1688369\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn1688369', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn1688369', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'GSE73526', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE73526', 'dataset_id': 'GSE73526'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE73526\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: GSE73526\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE73526, identifier: GSE73526, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE73526\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE73526 for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE73526\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier GSE73526 for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE73526\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1157 - INFO - Found id_pattern match for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE73526 in GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE73526\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE73526\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: GSE73526\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE73526\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE73526\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE73526\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: GSE73526\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE73526\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: GSE73526, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE73526\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: GSE73526\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE73526\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'GSE73526', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE73526', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 278 - INFO - Augmented dataset links: [{'dataset_identifier': 'GSE202203', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE202203', 'citation_type': 'n/a'}, {'dataset_identifier': 'GSE62944', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE62944', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn1688369', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn1688369', 'citation_type': 'n/a'}, {'dataset_identifier': 'GSE73526', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE73526', 'citation_type': 'n/a'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1191 - INFO - Fetching metadata for 4 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: GSE202203\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: geo, identifier: GSE202203, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE202203 for geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for GSE202203 in geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE202203\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE202203\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 1: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: GSE62944\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: geo, identifier: GSE62944, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE62944 for geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for GSE62944 in geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 2 with repo: geo and accession_id: GSE62944\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE62944\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 2: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn1688369\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn1688369, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn1688369 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn1688369 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn1688369 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn1688369 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 3 with repo: synapse.org and accession_id: syn1688369\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn1688369\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 3: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: GSE73526\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: geo, identifier: GSE73526, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE73526 for geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for GSE73526 in geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 4 with repo: geo and accession_id: GSE73526\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE73526\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 4: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1277 - INFO - Updated datasets len: 4\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 287 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 14, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'source_url', 'download_link',\n",
      "       'title', 'content_type', 'id', 'caption', 'description',\n",
      "       'source_section', 'retrieval_pattern', 'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 500 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 506 - INFO - XML element classification not needed. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1012 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11526681\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1020 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11526681\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 674th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11527220)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11527220\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11527220 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 634 - INFO - Fetching data for pmc11527220\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 637 - INFO - Fetching data from scripts/exp_input/Local_fetched_data_1.parquet\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 438 - INFO - Raw html data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11527220 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 452 - INFO - Skipping raw HTML/XML saving.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 482 - INFO - Using HTMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 52 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 82 - INFO - Initializing htmlRetriever\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 350 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11527220, additional_data, html)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 448 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC11527220)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 550 - INFO - Extracted 11 unique supplementary material links from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 560 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#clp_a'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for clp_a: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#S1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for S1: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#S5'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for S5: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#S5'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for S5: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#S7'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for S7: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#S8'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for S8: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#S33'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for S33: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#S33'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for S33: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#S33'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for S33: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#S33'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for S33: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 368 - INFO - Extracting links from full HTML content.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 301 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 309 - INFO - Content length: 223117\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 326 - INFO - Prompt messages total length: 56533 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 332 - INFO - Prompt ID: gemini-2.5-flash-0-eb8b0425a122638c354da33ed01056158c2801b6800482718f16f39c22a3cf65\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 351 - INFO - Requesting datasets from content using model: gemini-2.5-flash, temperature: 0, messages length: 56533 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-e3b84811-d924-47f8-b0e1-2f1862e0c90d\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GCST009325\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ebi.ac.uk/gwas/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GCST007780\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ebi.ac.uk/gwas/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn25398075\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn25398075\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn24172458\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn24172458\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn23627957\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn23627957\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://yanglab.westlake.edu.cn/data/brainmeta/cis_sqtl/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"http://mostafavilab.stat.ubc.ca/xQTLServe\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE178265\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178265\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"GCST009325\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ebi.ac.uk/gwas/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GCST007780\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ebi.ac.uk/gwas/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn25398075\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn25398075\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn24172458\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn24172458\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn23627957\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn23627957\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://yanglab.westlake.edu.cn/data/brainmeta/cis_sqtl/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"http://mostafavilab.stat.ubc.ca/xQTLServe\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE178265\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178265\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1758211239,\n",
      "    \"model\": \"gemini-2.5-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 81195,\n",
      "        \"completion_tokens\": 417,\n",
      "        \"total_tokens\": 85804,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 4192,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 664 - INFO - Deduplicating response with 8 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: GCST009325\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: GCST007780\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn25398075\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn24172458\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn23627957\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: GSE178265\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'GCST009325', 'repository_reference': 'https://www.ebi.ac.uk/gwas/', 'dataset_id': 'GCST009325'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GCST009325\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: GCST009325\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.ebi.ac.uk/gwas/, identifier: GCST009325, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.ebi.ac.uk/gwas/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GCST009325 for https://www.ebi.ac.uk/gwas/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier GCST009325 for https://www.ebi.ac.uk/gwas/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier GCST009325 for https://www.ebi.ac.uk/gwas/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier GCST009325 for https://www.ebi.ac.uk/gwas/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier GCST009325 for https://www.ebi.ac.uk/gwas/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier GCST009325 for https://www.ebi.ac.uk/gwas/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://www.ebi.ac.uk/gwas/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GCST009325\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: GCST009325, www.ebi.ac.uk, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: GCST009325\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: www.ebi.ac.uk\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'GCST009325', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'GCST007780', 'repository_reference': 'https://www.ebi.ac.uk/gwas/', 'dataset_id': 'GCST007780'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GCST007780\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: GCST007780\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: pdb\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.ebi.ac.uk/gwas/, identifier: GCST007780, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.ebi.ac.uk/gwas/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GCST007780 for https://www.ebi.ac.uk/gwas/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier GCST007780 for https://www.ebi.ac.uk/gwas/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier GCST007780 for https://www.ebi.ac.uk/gwas/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier GCST007780 for https://www.ebi.ac.uk/gwas/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier GCST007780 for https://www.ebi.ac.uk/gwas/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier GCST007780 for https://www.ebi.ac.uk/gwas/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://www.ebi.ac.uk/gwas/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GCST007780\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: GCST007780, www.ebi.ac.uk, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: GCST007780\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: www.ebi.ac.uk\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'GCST007780', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn25398075', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn25398075', 'dataset_id': 'syn25398075'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn25398075\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn25398075\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn25398075, identifier: syn25398075, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn25398075\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn25398075 for https://www.synapse.org/#!Synapse:syn25398075\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn25398075 for https://www.synapse.org/#!Synapse:syn25398075\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn25398075 for https://www.synapse.org/#!Synapse:syn25398075\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn25398075 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn25398075\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn25398075\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn25398075\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn25398075\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn25398075\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn25398075\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn25398075\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn25398075\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn25398075\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn25398075, synapse.org, https://www.synapse.org/#!Synapse:syn25398075\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn25398075\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn25398075\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn25398075', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn25398075', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn24172458', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn24172458', 'dataset_id': 'syn24172458'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn24172458\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn24172458\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn24172458, identifier: syn24172458, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn24172458\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn24172458 for https://www.synapse.org/#!Synapse:syn24172458\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn24172458 for https://www.synapse.org/#!Synapse:syn24172458\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn24172458 for https://www.synapse.org/#!Synapse:syn24172458\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn24172458 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn24172458\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn24172458\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn24172458\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn24172458\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn24172458\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn24172458\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn24172458\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn24172458\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn24172458\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn24172458, synapse.org, https://www.synapse.org/#!Synapse:syn24172458\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn24172458\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn24172458\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn24172458', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn24172458', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn23627957', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn23627957', 'dataset_id': 'syn23627957'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn23627957\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn23627957\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn23627957, identifier: syn23627957, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn23627957\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn23627957 for https://www.synapse.org/#!Synapse:syn23627957\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn23627957 for https://www.synapse.org/#!Synapse:syn23627957\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn23627957 for https://www.synapse.org/#!Synapse:syn23627957\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn23627957 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn23627957\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn23627957\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn23627957\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn23627957\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn23627957\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn23627957\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn23627957\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn23627957\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn23627957\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn23627957, synapse.org, https://www.synapse.org/#!Synapse:syn23627957\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn23627957\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn23627957\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn23627957', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn23627957', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://yanglab.westlake.edu.cn/data/brainmeta/cis_sqtl/', 'dataset_id': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://yanglab.westlake.edu.cn/data/brainmeta/cis_sqtl/, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://yanglab.westlake.edu.cn/data/brainmeta/cis_sqtl/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://yanglab.westlake.edu.cn/data/brainmeta/cis_sqtl/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'yanglab.westlake.edu.cn', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'http://mostafavilab.stat.ubc.ca/xQTLServe'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: http://mostafavilab.stat.ubc.ca/xQTLServe, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: http://mostafavilab.stat.ubc.ca/xQTLServe\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: http://mostafavilab.stat.ubc.ca/xQTLServe\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'mostafavilab.stat.ubc.ca', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'GSE178265', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178265', 'dataset_id': 'GSE178265'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE178265\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: GSE178265\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178265, identifier: GSE178265, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178265\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE178265 for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178265\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier GSE178265 for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178265\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1157 - INFO - Found id_pattern match for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178265 in GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178265\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178265\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: GSE178265\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE178265\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE178265\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178265\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: GSE178265\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE178265\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: GSE178265, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178265\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: GSE178265\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178265\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'GSE178265', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178265', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 378 - INFO - Augmented dataset links: [{'dataset_identifier': 'GCST009325', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'GCST007780', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn25398075', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn25398075', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn24172458', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn24172458', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn23627957', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn23627957', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'yanglab.westlake.edu.cn', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'mostafavilab.stat.ubc.ca', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'GSE178265', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178265', 'citation_type': 'n/a'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1191 - INFO - Fetching metadata for 8 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: GCST009325\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: www.ebi.ac.uk, identifier: GCST009325, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: www.ebi.ac.uk\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for GCST009325 in www.ebi.ac.uk\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 1 with repo: www.ebi.ac.uk and accession_id: GCST009325\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.ebi.ac.uk/pride/archive/projects/GCST009325\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 1: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: GCST007780\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: www.ebi.ac.uk, identifier: GCST007780, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: www.ebi.ac.uk\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for GCST007780 in www.ebi.ac.uk\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 2 with repo: www.ebi.ac.uk and accession_id: GCST007780\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.ebi.ac.uk/pride/archive/projects/GCST007780\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 2: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn25398075\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn25398075, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn25398075 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn25398075 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn25398075 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn25398075 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 3 with repo: synapse.org and accession_id: syn25398075\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn25398075\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 3: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn24172458\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn24172458, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn24172458 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn24172458 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn24172458 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn24172458 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 4 with repo: synapse.org and accession_id: syn24172458\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn24172458\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 4: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn23627957\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn23627957, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn23627957 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn23627957 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn23627957 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn23627957 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 5 with repo: synapse.org and accession_id: syn23627957\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn23627957\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 5: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 6: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 7: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: GSE178265\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: geo, identifier: GSE178265, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE178265 for geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for GSE178265 in geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 8 with repo: geo and accession_id: GSE178265\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178265\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 8: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1277 - INFO - Updated datasets len: 8\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 414 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 19, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'title', 'file_info',\n",
      "       'description', 'source_section', 'section_class', 'download_link',\n",
      "       'file_extension', 'a_attr_href', 'a_attr_class',\n",
      "       'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 173 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 183 - WARNING - No publication title found in the HTML data.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 643 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 494 - INFO - Parsed data extraction completed. Elements collected: 8\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 500 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 510 - INFO - HTML element classification not supported. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1012 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11527220\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1020 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11527220\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 675th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11527241)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11527241\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11527241 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 634 - INFO - Fetching data for pmc11527241\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 637 - INFO - Fetching data from scripts/exp_input/Local_fetched_data_1.parquet\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 438 - INFO - Raw html data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11527241 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 452 - INFO - Skipping raw HTML/XML saving.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 482 - INFO - Using HTMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 52 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 82 - INFO - Initializing htmlRetriever\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 350 - INFO - Function call: parse_data(html_str, PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11527241, additional_data, html)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 448 - INFO - Function_call: extract_href_from_supplementary_material(tree, https://pmc.ncbi.nlm.nih.gov/articles/PMC11527241)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 550 - INFO - Extracted 8 unique supplementary material links from HTML.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 560 - INFO - Function_call: extract_supplementary_material_refs(raw_html, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#clp_a'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for clp_a: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#S25'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for S25: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#fn-group1'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for fn-group1: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#S26'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for S26: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#S26'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for S26: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#S26'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for S26: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 571 - INFO - Found 0 <a> elements with href='#S26'.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 579 - INFO - Extracted context description (len: 0) for S26: \u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 368 - INFO - Extracting links from full HTML content.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 301 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 309 - INFO - Content length: 132354\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 326 - INFO - Prompt messages total length: 33842 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 332 - INFO - Prompt ID: gemini-2.5-flash-0-d3cc619b1fdc7cc27432cec7471685a3389c03e1e781f4669cd609e6af32446c\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 351 - INFO - Requesting datasets from content using model: gemini-2.5-flash, temperature: 0, messages length: 33842 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 452 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-a4f725cd-f7ef-49ff-bd3d-969ab5c78c87\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"NG00108\\\",\\n    \\\"repository_reference\\\": \\\"https://www.niagads.org/datasets/ng00108\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://dian.wustl.edu/our-research/for-investigators/dian-observational-study-investigator-resources/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn52293417\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn52293417\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE178317\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178317\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n  {\\n    \\\"dataset_identifier\\\": \\\"NG00108\\\",\\n    \\\"repository_reference\\\": \\\"https://www.niagads.org/datasets/ng00108\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"https://dian.wustl.edu/our-research/for-investigators/dian-observational-study-investigator-resources/\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"syn52293417\\\",\\n    \\\"repository_reference\\\": \\\"https://www.synapse.org/#!Synapse:syn52293417\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"GSE178317\\\",\\n    \\\"repository_reference\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178317\\\"\\n  },\\n  {\\n    \\\"dataset_identifier\\\": \\\"n/a\\\",\\n    \\\"repository_reference\\\": \\\"n/a\\\"\\n  }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1758211265,\n",
      "    \"model\": \"gemini-2.5-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 44467,\n",
      "        \"completion_tokens\": 260,\n",
      "        \"total_tokens\": 48559,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 3832,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 664 - INFO - Deduplicating response with 5 items\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: NG00108\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: syn52293417\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 685 - INFO - Adding unique item: GSE178317\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 693 - INFO - Duplicate found and skipped: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'NG00108', 'repository_reference': 'https://www.niagads.org/datasets/ng00108', 'dataset_id': 'NG00108'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: NG00108\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1023 - INFO - Accession ID NG00108 is valid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.niagads.org/datasets/ng00108, identifier: NG00108, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.niagads.org/datasets/ng00108\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier NG00108 for https://www.niagads.org/datasets/ng00108\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier NG00108 for https://www.niagads.org/datasets/ng00108\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier NG00108 for https://www.niagads.org/datasets/ng00108\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier NG00108 for https://www.niagads.org/datasets/ng00108\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier NG00108 for https://www.niagads.org/datasets/ng00108\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier NG00108 for https://www.niagads.org/datasets/ng00108\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://www.niagads.org/datasets/ng00108\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'NG00108', 'data_repository': 'www.niagads.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'n/a', 'repository_reference': 'https://dian.wustl.edu/our-research/for-investigators/dian-observational-study-investigator-resources/', 'dataset_id': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: None, None, None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1005 - INFO - Validating accession ID: n/a\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1010 - INFO - Accession ID n/a is invalid\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://dian.wustl.edu/our-research/for-investigators/dian-observational-study-investigator-resources/, identifier: n/a, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://dian.wustl.edu/our-research/for-investigators/dian-observational-study-investigator-resources/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: https://dian.wustl.edu/our-research/for-investigators/dian-observational-study-investigator-resources/\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 639 - INFO - Dataset webpage not extracted\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'dian.wustl.edu', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'syn52293417', 'repository_reference': 'https://www.synapse.org/#!Synapse:syn52293417', 'dataset_id': 'syn52293417'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn52293417\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: syn52293417\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.synapse.org/#!Synapse:syn52293417, identifier: syn52293417, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.synapse.org/#!Synapse:syn52293417\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn52293417 for https://www.synapse.org/#!Synapse:syn52293417\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn52293417 for https://www.synapse.org/#!Synapse:syn52293417\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn52293417 for https://www.synapse.org/#!Synapse:syn52293417\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1149 - INFO - Found partial match for https://www.synapse.org/#!Synapse:syn52293417 in Synapse\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 578 - INFO - Matched dataset_webpage_url_ptr: https://www.synapse.org/[#!]*Synapse:__ID__ to value: https://www.synapse.org/#!Synapse:syn52293417\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 580 - INFO - Candidate dataset_webpage: https://www.synapse.org/#!Synapse:syn52293417\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: https://www.synapse.org/#!Synapse:syn52293417\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(syn\\d{7,12})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn52293417\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: syn\\d{7,12} to value: syn52293417\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn52293417\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.synapse.org/#!Synapse:syn52293417\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: syn52293417\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: syn52293417\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: syn52293417, synapse.org, https://www.synapse.org/#!Synapse:syn52293417\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: syn52293417\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.synapse.org/#!Synapse:syn52293417\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'syn52293417', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn52293417', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 521 - INFO - Processing dataset: {'dataset_identifier': 'GSE178317', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178317', 'dataset_id': 'GSE178317'}\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 536 - INFO - Dataset is a dictionary\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE178317\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 610 - INFO - Setting dataset_id to value: GSE178317\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 613 - INFO - Candidate data_repository: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178317, identifier: GSE178317, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 880 - INFO - Brute-force checking if link points to dataset webpage: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178317\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE178317 for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178317\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier GSE178317 for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178317\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1157 - INFO - Found id_pattern match for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178317 in GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178317\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 595 - INFO - Setting dataset_webpage to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178317\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:(GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:(GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: GSE178317\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ to value: GSE178317\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE178317\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178317\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 598 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 599 - INFO - Using id_pattern to extract ID:([1-9][a-zA-Z0-9]{3})\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 604 - INFO - Extracted ID: GSE178317\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 592 - INFO - Matched id_pattern: [1-9][a-zA-Z0-9]{3} to value: GSE178317\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 620 - INFO - Schema validation vals: GSE178317, geo, https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178317\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 625 - INFO - Dataset ID found via pattern matching: GSE178317\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 633 - INFO - Data repository found via pattern matching: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 641 - INFO - Dataset webpage found via pattern matching: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178317\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 557 - INFO - Extracted dataset: {'dataset_identifier': 'GSE178317', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178317', 'citation_type': 'n/a'}\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 378 - INFO - Augmented dataset links: [{'dataset_identifier': 'NG00108', 'data_repository': 'www.niagads.org', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'n/a', 'data_repository': 'dian.wustl.edu', 'dataset_webpage': 'n/a', 'citation_type': 'n/a'}, {'dataset_identifier': 'syn52293417', 'data_repository': 'synapse.org', 'dataset_webpage': 'https://www.synapse.org/#!Synapse:syn52293417', 'citation_type': 'n/a'}, {'dataset_identifier': 'GSE178317', 'data_repository': 'geo', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178317', 'citation_type': 'n/a'}]\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1191 - INFO - Fetching metadata for 4 datasets\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: NG00108\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: www.niagads.org, identifier: NG00108, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier NG00108 for www.niagads.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier NG00108 for www.niagads.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier NG00108 for www.niagads.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern syn\\d{7,12} match with identifier NG00108 for www.niagads.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern 10\\.\\d{4,9}/[^\\s,]+ match with identifier NG00108 for www.niagads.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern [1-9][a-zA-Z0-9]{3} match with identifier NG00108 for www.niagads.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 958 - INFO - Extracting repo domain from URL: www.niagads.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 970 - WARNING - URL www.niagads.org may be a domain already.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1164 - INFO - data_repository not resolved, using domain\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for NG00108 in www.niagads.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 1 with repo: www.niagads.org and accession_id: NG00108\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1273 - WARNING - Repository www.niagads.org unknown in Ontology. Skipping dataset page 1.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1207 - INFO - Skipping dataset 2: no dataset_identifier for item\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: syn52293417\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: synapse.org, identifier: syn52293417, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier syn52293417 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern GSE\\d+|GSM\\d+|GPL\\d+|GDS\\d+ match with identifier syn52293417 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern phs[0-9]+\\.\\w+\\d+\\.\\w+\\d+ match with identifier syn52293417 for synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for syn52293417 in synapse.org\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 3 with repo: synapse.org and accession_id: syn52293417\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.synapse.org/[#!]*Synapse:syn52293417\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 3: Complex download or application to access\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1199 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'citation_type'])\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1210 - INFO - Raw accession ID: GSE178317\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1098 - INFO - Resolving data repository for candidate: geo, identifier: GSE178317, dataset_page: None\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1155 - INFO - Checking id_pattern PDC[0-9]+ match with identifier GSE178317 for geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1136 - INFO - Exact - case insensitive - match found for repo: geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1080 - INFO - Resolving accession ID for GSE178317 in geo\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1232 - INFO - Processing dataset 4 with repo: geo and accession_id: GSE178317\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1259 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178317\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1266 - INFO - Adding access mode for dataset 4: Complex download\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1277 - INFO - Updated datasets len: 4\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 414 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 12, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'citation_type', 'access_mode', 'link', 'title', 'file_info',\n",
      "       'description', 'source_section', 'section_class', 'download_link',\n",
      "       'file_extension', 'a_attr_href', 'a_attr_class',\n",
      "       'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel',\n",
      "       'context_description'],\n",
      "      dtype='object')\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 173 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mhtml_retriever.py - line 183 - WARNING - No publication title found in the HTML data.\u001b[0m\n",
      "\u001b[97mhtml_parser.py - line 643 - INFO - Extracting publication title from HTML\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 494 - INFO - Parsed data extraction completed. Elements collected: 5\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 500 - INFO - Raw Data parsing completed.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 510 - INFO - HTML element classification not supported. Using parsed_data.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1012 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11527241\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 1020 - INFO - Saving results to cache with process_id: gemini-2.5-flash-FDR-pmc_ncbi_nlm_nih_gov_articles_PMC11527241\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 583 - INFO - 676th function call: self.process_url(https://pmc.ncbi.nlm.nih.gov/articles/PMC11535397)\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 386 - INFO - Processing URL: https://pmc.ncbi.nlm.nih.gov/articles/PMC11535397\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 170 - INFO - URL https://pmc.ncbi.nlm.nih.gov/articles/PMC11535397 found in DataFrame. Using DatabaseFetcher.\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 172 - INFO - Reusing existing DatabaseFetcher instance.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 393 - INFO - Type of data_fetcher DatabaseFetcher\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 634 - INFO - Fetching data for pmc11535397\u001b[0m\n",
      "\u001b[97mdata_fetcher.py - line 637 - INFO - Fetching data from scripts/exp_input/Local_fetched_data_1.parquet\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 438 - INFO - Raw xml data fetched from https://pmc.ncbi.nlm.nih.gov/articles/PMC11535397 is ready for parsing.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 452 - INFO - Skipping raw HTML/XML saving.\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 458 - INFO - Using XMLParser to parse data.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 52 - INFO - LLMParser initialized.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 24 - INFO - Initializing xmlRetriever\u001b[0m\n",
      "\u001b[97mdata_gatherer.py - line 466 - INFO - No additional data provided. Parsing raw data only.\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 147 - INFO - Function call: parse_data(api_data(<class 'str'>), PMC, https://pmc.ncbi.nlm.nih.gov/articles/PMC11535397, additional_data, XML)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 162 - INFO - api_data converted to lxml element\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 466 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 153 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 154 - INFO - Consider migrating this function to the BaseRetriever class.\u001b[0m\n",
      "\u001b[97mxml_retriever.py - line 69 - INFO - Loading target sections for section name: supplementary_material_sections\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 477 - INFO - Found 11 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x15dfeb9c0>, <Element supplementary-material at 0x15f71dbc0>, <Element supplementary-material at 0x15f71ec00>, <Element supplementary-material at 0x15e50d9c0>, <Element supplementary-material at 0x15e50cb40>, <Element supplementary-material at 0x15e7fe980>, <Element supplementary-material at 0x15e7fea00>, <Element supplementary-material at 0x15e7fea80>, <Element supplementary-material at 0x15e7fea40>, <Element supplementary-material at 0x15e7fe780>, <Element supplementary-material at 0x15d66d400>]\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 485 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x15dfeb9c0>, <Element supplementary-material at 0x15f71dbc0>, <Element supplementary-material at 0x15f71ec00>, <Element supplementary-material at 0x15e50d9c0>, <Element supplementary-material at 0x15e50cb40>, <Element supplementary-material at 0x15e7fe980>, <Element supplementary-material at 0x15e7fea00>, <Element supplementary-material at 0x15e7fea80>, <Element supplementary-material at 0x15e7fea40>, <Element supplementary-material at 0x15e7fe780>, <Element supplementary-material at 0x15d66d400>]}\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 574 - INFO - Function_call: extract_supplementary_material_refs(api_xml, supplementary_material_links)\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM1: We applied PaxDb and PTR to the meta-assembly and accounted for method-specific biases by shift-scaling the outputs globally to maximize the ∆CDF for the training data of Pinferna (“Methods”; Appendix Fig. S1 [this file] ). Although modestly positive, the ∆CDF value for Pinferna was >50% of the practical maximum for this metric (Appendix Text S1 [this file] ). \n",
      "Separately, we took the NCI–CPTAC transcriptomics and found that log fold-change estimates derived from Pinferna were significantly closer to the measured TMT datasets than the original RNA-seq (Appendix Fig. S2 [this file] ). \n",
      "For consistency, our analysis focused on the 4366 transcripts compatible with protein inference (Fig. 1A [rid=Fig1] ), but results were unchanged when using the entire available transcriptome (Appendix Fig. S3A–D [this file] ). Consensus clustering of mRNA profiles identified five ordered and stable groups, which were statistically enriched in PAM50-assigned cases of (1) Normal-like ( = 3.8e-2), (2) HER2+ ( = 2.7e-3), (3) Luminal A ( = 3.4e-39), (4) Basal-like ( = 1.1e-42), and (5) Luminal B breast cancer ( = 1.6e-23; Fig. 5A [rid=Fig5] ; Appendix Fig. S3A,B [this file] ). When the analysis was repeated with Pinferna estimates after standardization, the smallest number of stable and significant consensus clusters was again five (Appendix Fig. S3E,F [this file] ). \n",
      "Interestingly, Luminal A-reassigned patients had a ~twofold worse overall survival than those that were not reassigned (Appendix Fig. S4A,B [this file] ). Luminal A/B cases are often intermingled in clusters defined by transcriptomics (Cancer Genome Atlas, 2012 [rid=CR10] ), prompting us to look more deeply at their reassignment with Pinferna (Appendix Fig. S5A,B [this file] ). We surveyed for HL + LASSO genes whose Z-score standardized values changed the most from mRNA to inferred protein and looked within these influential genes for features (other genes) that were STRING interactors (“Methods”; Fig. 2H [rid=Fig2] ; Appendix Fig. S5C [this file] ). \n",
      "Unfortunately, NUP37 knockdown was not effective in MCF7 cells or another luminal breast cancer line (T47D; Appendix Fig. S6 [this file] ), preventing further interpretations about CDK4. For DNM1L, we pursued similar gain- and loss-of-function approaches but were repeatedly stymied by dosage compensation from the induced DNM1L allele (Appendix Fig. S7 [this file] ). \n",
      "For ZR751 cells (Appendix Fig. S7A [this file] ), a tertiary detection scheme was also used to detect V5 (Invitrogen, R960-25, 1:1000), with unlabeled goat anti-mouse (Jackson ImmunoResearch, 115-005-146, 1:1000) for 1 h followed by IRDye 800CW-conjugated donkey anti-goat (LI-COR, 926-32214, 1:20,000). Before normalizing to control conditions, cell types were normalized to the following loading controls: vinculin–tubulin (B2B1, BT20, CAMA1 (Fig. 2E [rid=Fig2] ), HCC38, HCC1395, K562, T47D (Fig. 2E [rid=Fig2] ), and ZR751 (Fig. 2E [rid=Fig2] ; Appendix Fig. 7A,B [this file] )), vinculin–p38 (CAMA1 (Appendix Fig. 7C,D [this file] ) and T47D (Appendix Fig. 7C,E [this file] )), vinculin–GRB2 (AC16 (Figs. 4I,K [rid=Fig4] and EV4J, L [rid=Fig10] )), vinculin–tubulin–p38–GRB2 (T47D (Fig. 6G [rid=Fig6] ) and ZR751 (Fig. 6H [rid=Fig6] )), and no loading normalization (AC16 (Figs. 4H,J [rid=Fig4] and EV4I,K [rid=Fig10] ) and MCF7). \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM2: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM3: We discerned mathematical relationships that best captured abundance relationships between mRNA and protein by merging SWATH-scaled proteomics with paired Cancer Cell Line Encyclopedia (CCLE) transcriptomics (Ghandi et al, 2019 [rid=CR27] ) and building gene-specific regressions of different classes (Fig. 1A [rid=Fig1] , Step 2; Datasets EV1 [this file] – EV3 [rid=MOESM5] ). \n",
      "Data for regressions are available in Datasets EV1 [this file] and EV2 [rid=MOESM4] . \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM4: Data for regressions are available in Datasets EV1 [rid=MOESM3] and EV2 [this file] . \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM5: We discerned mathematical relationships that best captured abundance relationships between mRNA and protein by merging SWATH-scaled proteomics with paired Cancer Cell Line Encyclopedia (CCLE) transcriptomics (Ghandi et al, 2019 [rid=CR27] ) and building gene-specific regressions of different classes (Fig. 1A [rid=Fig1] , Step 2; Datasets EV1 [rid=MOESM3] – EV3 [this file] ). \n",
      "The final harmonized table of ten identifiers (nine for the 83 genes not in UniProt) for 19,062 genes is available in Dataset EV3 [this file] . \n",
      "After removing proteins annotated as “Fragments”, gene symbols were matched to UniProt accession codes by using the harmonized identifier table (Dataset EV3 [this file] ). The RNA-seq dataset (Ghandi et al, 2019 [rid=CR27] ) was obtained from the Depmap portal, back-transformed from log to TPM, and renamed with the harmonized identifier table (Dataset EV3 [this file] ). \n",
      "Residuals from the HL fit were regressed against all other coding genes in the MANE-harmonized transcriptome (Dataset EV3 [this file] ) by LASSO with the glmnet package (version 4.1-6) in R. \n",
      "HL + LASSO genes were mapped with the string_db$map function, and up to 1000 medium-confidence interactions were retrieved with the rba_string_interaction_partners function and harmonized with the gene identifier table (Dataset EV3 [this file] ). \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM6: The best model among the three for each gene was distinguished by the Bayesian Information Criterion (BIC; Figs. 1A [rid=Fig1] , Step 3 and EV1C–E [rid=Fig7] ) to arrive at preferred M, HL, or HL + LASSO relationships for 4366 genes (Dataset EV4 [this file] ). \n",
      "Among 73 features for CDK4, NUP37 was the ninth most negative dependency and DNM1L the fifth most positive (Dataset EV4 [this file] ). \n",
      "Genes in each class (Dataset EV4 [this file] ) were separately tested on the GO Enrichment Analysis web page ( https://geneontology.org/ (https://geneontology.org/) ) and searched for “Biological Process” at a false-discovery rate of 5%. \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM7: Consistent with previous findings (Mun et al, 2019 [rid=CR58] ; Zhang et al, 2014a [rid=CR106] ), M genes with no clear transcript dependence showed gene ontology (GO) enrichments for translation and mitochondrial electron transport (Fig. 2A [rid=Fig2] ; Dataset EV5 [this file] ). \n",
      "The complete list of GO enrichments for each relationship class is available in Dataset EV5 [this file] . ( ) HL outperforms competing mRNA-to-protein relationships. \n",
      "The complete list of enrichments is available in Dataset EV5 [this file] . \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM8: A cell line-specific PaxDb estimate was only available for U251 cells. ( ) Pinferna reconstitutes proteome-wide distributions of proportional copy numbers in normal prostate tissue (left; case L1N), low-grade prostate cancer (middle; case M7U), and high-grade prostate cancer (right; case H9T) ( PRJNA579899 (https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA579899) ; PXD004589; Dataset EV6 [this file] ). ( ) ProteoEstimator, the best-performing model of the NCI–CPTAC DREAM proteogenomic challenge (Yang et al, 2020 [rid=CR103] ), does not accurately predict protein copy numbers when mapped to a proportional scale (“Methods”). \n",
      "We organized transcriptomic and SWATH data for 39 normal human prostate samples and 40 primary prostate cancers (Charmpi et al, 2020 [rid=CR12] ) (Dataset EV6 [this file] ). \n",
      "The final patient annotations and cross-referencing key is available in Dataset EV6 [this file] . \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM9: The realignment confirmed no biases in expression based on data source for (Fig. EV4A [rid=Fig10] ; Dataset EV7 [this file] ). expression increases in cases of cardiomyopathy (Fechner et al, 2003 [rid=CR20] ; Kaur et al, 2012 [rid=CR40] ), and we reproduced this result by stratifying cases from the three sources ( = 6.2e–245; Fig. EV4B [rid=Fig10] ). \n",
      "( ) Distribution of abundance (Dataset EV7 [this file] ) separated by source data: EGA (brown; EGAS00001002454 (https://ega-archive.org/studies/EGAS00001002454) ), GTEx (gray; phs000424.v9.p2 (https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000424.v9.p2) ), MAGNet (blue; GSE141910 (https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE141910) ). ( ) is upregulated in failing hearts. abundance (Dataset EV7 [this file] ) was stratified by heart health and colored by source data as in ( ). ( , ) Calibration plots (orange ( ); purple ( )) and Pinferna predictions for CD55 ( ) and CXADR ( ) in human heart samples (Heart RNA-seq; red). CD55 deviations from the smoothed best fit are caused by cardiac-specific features in the HL + LASSO regressions. ( , ) CD55–CXADR coregulation is increased at the protein level ( ) compared to the mRNA level ( ). ( ) Replotted histogram of Fig. 4C [rid=Fig4] separated by heart health (Dataset EV7 [this file] ). ( ) Predicted prevalence of susceptibility groups based on randomized measurements. \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM10: Encouraged by the robustness of predictions to read depth and alignment details, we provide Pinferna as an open resource ( http://janeslab.shinyapps.io/Pinferna (http://janeslab.shinyapps.io/Pinferna) ; Dataset EV8 [this file] ). \n",
      "\u001b[0m\n",
      "\u001b[97mxml_parser.py - line 590 - INFO - Extracted context_descr for xref MOESM11: \u001b[0m\n",
      "\u001b[97mxml_parser.py - line 263 - INFO - Extracting links from full XML content.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 301 - INFO - Function_call: extract_datasets_info_from_content(...)\u001b[0m\n",
      "\u001b[97mprompt_manager.py - line 33 - INFO - Loading prompt: GPT_FDR_FewShot_Syn from user_prompt_dir: None, subdir: \u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 309 - INFO - Content length: 355093\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 326 - INFO - Prompt messages total length: 89527 tokens\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 332 - INFO - Prompt ID: gemini-2.5-flash-0-21bf1547a55ab66e5a96f81a3b3c39f87889c3e5455ad9c0024e8a097b390a99\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 1322 - INFO - Expected string but got list. Converting list to string.\u001b[0m\n",
      "\u001b[97mbase_parser.py - line 351 - INFO - Requesting datasets from content using model: gemini-2.5-flash, temperature: 0, messages length: 89527 tokens, schema: {'type': 'json_schema', 'name': 'GPT_responses_schema', 'schema': {'type': 'object', 'properties': {'datasets': {'type': 'array', 'items': {'type': 'object', 'properties': {'dataset_identifier': {'type': 'string', 'description': 'A unique identifier or accession code for the dataset.'}, 'repository_reference': {'type': 'string', 'description': 'A valid URI or string referring to the repository.'}}, 'additionalProperties': False, 'required': ['dataset_identifier', 'repository_reference']}, 'minItems': 1, 'additionalProperties': False}}, 'additionalProperties': False, 'required': ['datasets']}}\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "combined_df = dg.run(input_file=input_file, semantic_retrieval=semantic_retrieval, section_filter=section_filter, prompt_name=prompt)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "9fd1be22241ced6b",
   "metadata": {
    "jupyter": {
     "is_executing": true
    }
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>dataset_identifier</th>\n",
       "      <th>data_repository</th>\n",
       "      <th>dataset_webpage</th>\n",
       "      <th>citation_type</th>\n",
       "      <th>access_mode</th>\n",
       "      <th>link</th>\n",
       "      <th>source_url</th>\n",
       "      <th>download_link</th>\n",
       "      <th>title</th>\n",
       "      <th>content_type</th>\n",
       "      <th>...</th>\n",
       "      <th>file_extension</th>\n",
       "      <th>pub_title</th>\n",
       "      <th>raw_data_format</th>\n",
       "      <th>file_info</th>\n",
       "      <th>section_class</th>\n",
       "      <th>a_attr_href</th>\n",
       "      <th>a_attr_class</th>\n",
       "      <th>a_attr_data-ga-action</th>\n",
       "      <th>a_attr_target</th>\n",
       "      <th>a_attr_rel</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>syn47354875</td>\n",
       "      <td>synapse.org</td>\n",
       "      <td>https://www.synapse.org/[#!]*Synapse:syn47354875</td>\n",
       "      <td>n/a</td>\n",
       "      <td>Complex download or application to access</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC10002464</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>Whole-Exome Sequencing and cfDNA Analysis Unco...</td>\n",
       "      <td>XML</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>ijms-24-04302-s001</td>\n",
       "      <td>www.mdpi.com</td>\n",
       "      <td>n/a</td>\n",
       "      <td>n/a</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC10002464</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>Whole-Exome Sequencing and cfDNA Analysis Unco...</td>\n",
       "      <td>XML</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>ijms-24-04302-s001.zip</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC10002464</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/instance...</td>\n",
       "      <td>No Title</td>\n",
       "      <td>local-data</td>\n",
       "      <td>...</td>\n",
       "      <td>zip</td>\n",
       "      <td>Whole-Exome Sequencing and cfDNA Analysis Unco...</td>\n",
       "      <td>XML</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>syn3163039</td>\n",
       "      <td>synapse.org</td>\n",
       "      <td>https://www.synapse.org/[#!]*Synapse:syn3163039</td>\n",
       "      <td>n/a</td>\n",
       "      <td>Complex download or application to access</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC10185548</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>Predictive network analysis identifies</td>\n",
       "      <td>XML</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>syn8650953</td>\n",
       "      <td>synapse.org</td>\n",
       "      <td>https://www.synapse.org/[#!]*Synapse:syn8650953</td>\n",
       "      <td>n/a</td>\n",
       "      <td>Complex download or application to access</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/PMC10185548</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>Predictive network analysis identifies</td>\n",
       "      <td>XML</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>5 rows × 26 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "   dataset_identifier data_repository  \\\n",
       "0         syn47354875     synapse.org   \n",
       "1  ijms-24-04302-s001    www.mdpi.com   \n",
       "2                 NaN             NaN   \n",
       "3          syn3163039     synapse.org   \n",
       "4          syn8650953     synapse.org   \n",
       "\n",
       "                                    dataset_webpage citation_type  \\\n",
       "0  https://www.synapse.org/[#!]*Synapse:syn47354875           n/a   \n",
       "1                                               n/a           n/a   \n",
       "2                                               NaN           NaN   \n",
       "3   https://www.synapse.org/[#!]*Synapse:syn3163039           n/a   \n",
       "4   https://www.synapse.org/[#!]*Synapse:syn8650953           n/a   \n",
       "\n",
       "                                 access_mode                    link  \\\n",
       "0  Complex download or application to access                     NaN   \n",
       "1                                        NaN                     NaN   \n",
       "2                                        NaN  ijms-24-04302-s001.zip   \n",
       "3  Complex download or application to access                     NaN   \n",
       "4  Complex download or application to access                     NaN   \n",
       "\n",
       "                                          source_url  \\\n",
       "0  https://pmc.ncbi.nlm.nih.gov/articles/PMC10002464   \n",
       "1  https://pmc.ncbi.nlm.nih.gov/articles/PMC10002464   \n",
       "2  https://pmc.ncbi.nlm.nih.gov/articles/PMC10002464   \n",
       "3  https://pmc.ncbi.nlm.nih.gov/articles/PMC10185548   \n",
       "4  https://pmc.ncbi.nlm.nih.gov/articles/PMC10185548   \n",
       "\n",
       "                                       download_link     title content_type  \\\n",
       "0                                                NaN       NaN          NaN   \n",
       "1                                                NaN       NaN          NaN   \n",
       "2  https://pmc.ncbi.nlm.nih.gov/articles/instance...  No Title   local-data   \n",
       "3                                                NaN       NaN          NaN   \n",
       "4                                                NaN       NaN          NaN   \n",
       "\n",
       "   ... file_extension                                          pub_title  \\\n",
       "0  ...           None  Whole-Exome Sequencing and cfDNA Analysis Unco...   \n",
       "1  ...           None  Whole-Exome Sequencing and cfDNA Analysis Unco...   \n",
       "2  ...            zip  Whole-Exome Sequencing and cfDNA Analysis Unco...   \n",
       "3  ...           None             Predictive network analysis identifies   \n",
       "4  ...           None             Predictive network analysis identifies   \n",
       "\n",
       "  raw_data_format file_info section_class a_attr_href a_attr_class  \\\n",
       "0             XML       NaN           NaN         NaN          NaN   \n",
       "1             XML       NaN           NaN         NaN          NaN   \n",
       "2             XML       NaN           NaN         NaN          NaN   \n",
       "3             XML       NaN           NaN         NaN          NaN   \n",
       "4             XML       NaN           NaN         NaN          NaN   \n",
       "\n",
       "  a_attr_data-ga-action a_attr_target a_attr_rel  \n",
       "0                   NaN           NaN        NaN  \n",
       "1                   NaN           NaN        NaN  \n",
       "2                   NaN           NaN        NaN  \n",
       "3                   NaN           NaN        NaN  \n",
       "4                   NaN           NaN        NaN  \n",
       "\n",
       "[5 rows x 26 columns]"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "combined_df.head()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "305d7d5f299551c1",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-09-04T21:20:38.136808Z",
     "start_time": "2025-09-04T21:20:38.134052Z"
    }
   },
   "outputs": [],
   "source": [
    "#ground_truth = pd.read_parquet(\"scripts/exp_input/dataset_citation_records_Table.parquet\")\n",
    "ground_truth = df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "dadaf8654a9d103d",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-09-04T21:20:38.857028Z",
     "start_time": "2025-09-04T21:20:38.851074Z"
    }
   },
   "outputs": [],
   "source": [
    "ground_truth.head()\n",
    "# rename colum synid to 'identifier'\n",
    "ground_truth = ground_truth.rename(columns={'synid': 'identifier'})"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "583f880224783dc7",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-09-04T21:26:52.754959Z",
     "start_time": "2025-09-04T21:26:52.722141Z"
    }
   },
   "outputs": [],
   "source": [
    "def evaluate_performance(predict_df, ground_truth, orchestrator, false_positives_file, false_negatives_file=None):\n",
    "    \"\"\" Evaluates dataset extraction performance using precision, recall, and F1-score. \"\"\"\n",
    "\n",
    "    recall_list, false_positives_output, false_negatives_output = [], [], []\n",
    "    total_precision, total_recall, num_sources = 0, 0, 0\n",
    "\n",
    "    for source_page in predict_df['source_url'].unique():\n",
    "        pub_id = source_page.split('/')[-1].lower()\n",
    "        \n",
    "        orchestrator.logger.info(f\"Evaluating pub_id: {pub_id}\")\n",
    "        gt_data = ground_truth[ground_truth['pmcid'].str.lower() == pub_id.lower()]  # extract ground truth\n",
    "\n",
    "        gt_datasets = set()\n",
    "        for dataset_string in gt_data['identifier'].dropna().str.lower():\n",
    "            gt_datasets.update(dataset_string.split(','))  # Convert CSV string into set of IDs\n",
    "\n",
    "        orchestrator.logger.info(f\"# of elements in gt_data: {len(gt_data)}. Element IDs: {gt_datasets}\")\n",
    "\n",
    "        num_sources += 1\n",
    "\n",
    "        # Extract evaluation datasets for this source page\n",
    "        eval_data = predict_df[predict_df['source_url'] == source_page]\n",
    "        eval_datasets = set(eval_data['dataset_identifier'].dropna().str.lower())\n",
    "        # Remove invalid entries\n",
    "        eval_datasets.discard('n/a')\n",
    "        eval_datasets.discard('')\n",
    "\n",
    "        orchestrator.logger.info(f\"Evaluation datasets: {eval_datasets}\")\n",
    "\n",
    "        # Handle cases where both ground truth and evaluation are empty\n",
    "        if not gt_datasets and not eval_datasets:\n",
    "            orchestrator.logger.info(\"No datasets in both ground truth and evaluation. Perfect precision and recall.\")\n",
    "            total_precision += 1\n",
    "            total_recall += 1\n",
    "            continue\n",
    "\n",
    "        # Match Extraction Logic\n",
    "        matched_gt, matched_eval = set(), set()\n",
    "\n",
    "        # Exact Matches\n",
    "        exact_matches = gt_datasets & eval_datasets  # Intersection of ground truth and extracted datasets\n",
    "        matched_gt.update(exact_matches)\n",
    "        matched_eval.update(exact_matches)\n",
    "\n",
    "        # Partial Matches (Aliased Identifiers)\n",
    "        for eval_id in eval_datasets - matched_eval:\n",
    "            for gt_id in gt_datasets - matched_gt:\n",
    "                if eval_id in gt_id or gt_id in eval_id:  # Partial match or alias\n",
    "                    orchestrator.logger.info(f\"Partial or alias match found: eval_id={eval_id}, gt_id={gt_id}\")\n",
    "                    matched_gt.add(gt_id)\n",
    "                    matched_eval.add(eval_id)\n",
    "                    break  # Stop once matched\n",
    "\n",
    "        # **False Positives (Unmatched extracted datasets)**\n",
    "        FP = eval_datasets - matched_eval\n",
    "        false_positives_output.extend([false_p, pub_id] for false_p in FP)\n",
    "\n",
    "        # **False Negatives (Unmatched ground truth datasets)**\n",
    "        FN = gt_datasets - matched_gt\n",
    "        false_negatives_output.extend((FN, pub_id)) if len(FN) > 0 else None\n",
    "\n",
    "        # **Precision and Recall Calculation**\n",
    "        true_positives = len(matched_gt)\n",
    "        false_positives = len(FP)\n",
    "        false_negatives = len(FN)\n",
    "\n",
    "        precision = true_positives / (true_positives + false_positives) if (true_positives + false_positives) > 0 else 0\n",
    "        recall = true_positives / (true_positives + false_negatives) if (true_positives + false_negatives) > 0 else 0\n",
    "\n",
    "        orchestrator.logger.info(f\"Precision for {source_page}: {precision}\")\n",
    "        orchestrator.logger.info(f\"Recall for {source_page}: {recall}\")\n",
    "\n",
    "        if recall == 0:\n",
    "            recall_list.append(source_page)\n",
    "\n",
    "        # Accumulate totals\n",
    "        total_precision += precision\n",
    "        total_recall += recall\n",
    "\n",
    "    # **Compute Overall Metrics**\n",
    "    average_precision = total_precision / num_sources if num_sources > 0 else 0\n",
    "    average_recall = total_recall / num_sources if num_sources > 0 else 0\n",
    "    f1_score = (\n",
    "        2 * (average_precision * average_recall) / (average_precision + average_recall)\n",
    "        if (average_precision + average_recall) > 0\n",
    "        else 0\n",
    "    )\n",
    "\n",
    "    orchestrator.logger.info(f\"\\nPerformance evaluation completed for {num_sources} source pages.\")\n",
    "\n",
    "    # **Save false positives**\n",
    "    with open(false_positives_file, 'w') as f:\n",
    "        for item in false_positives_output:\n",
    "            f.write(\"%s\\n\" % item)\n",
    "\n",
    "    if false_negatives_file:\n",
    "        with open(false_negatives_file, 'w') as f:\n",
    "            for item in false_negatives_output:\n",
    "                f.write(\"%s\\n\" % item)\n",
    "\n",
    "    return {\n",
    "        \"average_precision\": average_precision,\n",
    "        \"average_recall\": average_recall,\n",
    "        \"f1_score\": f1_score\n",
    "    }"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "d5bd9405ee5dcd60",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-09-04T21:26:53.669418Z",
     "start_time": "2025-09-04T21:26:53.666829Z"
    }
   },
   "outputs": [],
   "source": [
    "#results = evaluate_performance(out, ground_truth, dg, \"output/false_positives.txt\")\n",
    "# PMC4318527, "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "3ae367545bb1b42b",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-09-04T21:26:55.530950Z",
     "start_time": "2025-09-04T21:26:54.622805Z"
    }
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10002464\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn47354875'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'ijms-24-04302-s001', 'syn47354875'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10002464: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10002464: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10185548\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 6. Element IDs: {'syn4164376', 'syn3157325', 'syn8650953', 'syn2580853', 'syn3163039', 'syn3191087'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn4164376', 'syn3157325', 'syn8650953', 'gse228156', 'syn3191087', 'syn3163039', 'syn2580853'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10185548: 0.8571428571428571\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10185548: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10353192\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 10. Element IDs: {'syn47194124', 'syn47194116', 'syn4719361', 'syn47193617', 'syn47028485', 'syn47199024', 'syn47199836', 'syn25882127', 'syn26343262', 'syn47028493'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn47194124', 'syn47194116', 'syn47193617', 'syn47028485', 'syn47199024', 'syn47199836', 'syn26343262', 'syn47028493'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10353192: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10353192: 0.8\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9970873\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn11726602'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'egas00001006642', 'syn11726602', 'egas00001006610', 'egas00001006034'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9970873: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9970873: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7520714\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn18344812', 'syn17009517'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.6084/m9.figshare.8159525.v1', 'syn18344812', 'syn17009517', 'atlas'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7520714: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7520714: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 89 - INFO - \n",
      "Performance evaluation completed for 5 source pages.\u001b[0m\n"
     ]
    }
   ],
   "source": [
    "results = evaluate_performance(combined_df, ground_truth, dg, \"scripts/output/false_positives.txt\", \"scripts/output/false_negatives.txt\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "fff3dc3075cb13b5",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-09-04T21:25:01.310798Z",
     "start_time": "2025-09-04T21:25:01.304645Z"
    }
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'average_precision': 0.6214285714285714,\n",
       " 'average_recall': 0.96,\n",
       " 'f1_score': 0.7544715447154471}"
      ]
     },
     "execution_count": 15,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "results"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "88837a62c359df60",
   "metadata": {
    "jupyter": {
     "is_executing": true
    }
   },
   "outputs": [],
   "source": [
    "#combined_df.to_csv(\"output/SAGE_GPT_FDR_FewShot_Synapse_output.csv\", index=False)\n",
    "#\"https://pmc.ncbi.nlm.nih.gov/articles/PMC11240079\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC5752539\",\"https://pmc.ncbi.nlm.nih.gov/articles/PMC7032692\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10547713\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8102856\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8055881\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8226229\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10769298\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC11659981\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8082263\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8565335\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC9094742\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC6289083\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC7658217\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC9780309\",\"https://pmc.ncbi.nlm.nih.gov/articles/PMC3788619\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10680627\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC8131595\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC11420198\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC9915613\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10238095\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452\" , \"https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC10836119\",  \"https://pmc.ncbi.nlm.nih.gov/articles/PMC9280291\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC11661334\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC5161470\", \"https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "54baf5d5dcfee95e",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-09-04T20:50:38.300941Z",
     "start_time": "2025-09-04T20:50:25.797543Z"
    }
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11792374\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 4. Element IDs: {'syn52293417', 'syn8456637', 'syn2580853', 'syn9702085'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn52293417', 'syn8456637', 'syn2580853', 'syn9702085'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4318527\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn2455683', 'syn1720047'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn2455683', 'syn312572', 'syn1720047'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527: 0.6666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4339277\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 4. Element IDs: {'syn1701558', 'syn1682511', 'syn1688309', 'syn1732077'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn1701558', 'syn1682511', 'syn1688309', 'syn1732077'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8452114\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn18485175'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn18485175'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8452114: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8452114: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9641692\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn3219045'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse119117', 'gse67835', 'gse65133', 'syn3219045', 'gse107011', 'gse115449', 'gse87544', 'gse84133'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9641692: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9641692: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc5161470\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1446295'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn1446295'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC5161470: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC5161470: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9925742\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21126450'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'srr21079269', 'syn21126450', 'gse224398', 'srr21079265', 'srr21079270', 'srr21079266', 'srr21079268', 'srr21079267'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9925742: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9925742: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9235070\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn30243143'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse76312', 'gse75688', 'gse103322', 'e-mtab-6149', 'https://gdc-portal.ncbi.nih.gov', 'gse81383', 'syn213309', 'gse70630', 'gse81861', 'syn30243143'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9235070: 0.1\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9235070: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11661334\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn51364943'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn51364943', 'gcst009325', 'hippie', 'gcst007511', 'gcst90027158'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11661334: 0.2\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11661334: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11929800\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 5. Element IDs: {'syn3388564', 'syn11724057', 'syn4896408', 'syn18485175', 'syn17015098'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn3388564', 'syn11724057', 'syn4896408', 'gse291402', 'syn17015098', 'gcst90027158'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11929800: 0.6666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11929800: 0.8\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7113080\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn17394041'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn17394041'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8847344\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn25813713'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn25813713'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8847344: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8847344: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9280291\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn5612563'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.7303/syn5612563', 'gse21802', 'gse40586', 'gse57065', 'e-mtab-1548', 'gse54514', 'gse66099', 'e-mtab-5273/5274', 'gse40012', 'e-mexp-3589', 'gse13015', 'gse20346', 'gse27131', 'gse65682', 'gse60244', 'gse32707', 'gse25504', 'gse95233'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn5612563, gt_id=syn5612563\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9280291: 0.05555555555555555\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9280291: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10029021\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn50897018'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn50897018.3'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=syn50897018.3, gt_id=syn50897018\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10029021: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10029021: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11525589\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn51514106'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.7303/syn51514106', 'brats-men'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn51514106, gt_id=syn51514106\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11525589: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11525589: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10836119\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 6. Element IDs: {'syn5592519', 'syn9705278', 'syn3157322', 'syn3205821', 'syn3157238', 'syn3157329'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'rf1ag57473', 'hhsn271201200022c', 'syn9705278', 'n01-ag-12100', 'r01ag15819', 'u01ag32984', 'rc2ag036547', 'syn3157238', 'rf1ag051550', 'w81xwh-12-2-0012', 'syn3205821.6', 'syn3157322', 'r01ag30146', 'r01ag17917', 'syn3157329', 'r01ag36836', 'r01ag48015', 'p30ag10161', 'u01ag46152', 'syn5592519', 'r01ag046171', 'u01ag46161', 'u01ag61356', 'r01ag36042', '3u01ag024904-09s4'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=syn3205821.6, gt_id=syn3205821\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10836119: 0.24\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10836119: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6646046\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn5609493'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gm12878', 'chembl'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6646046: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6646046: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10387117\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21041850'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse161382', 'gse149563', 'gsm5388412', 'gse122332', 'gsm4504966', 'dataset-427f1eee6dd44f50bae1ab13f0f3c6a9', 'gsm5388411', 'gsm4035472', 'gse135893', 'gse136832', 'gsm5388413', 'egas00001004082', 'lmex0000004397', 'gse178362', 'gsm4504967', 'lmex0000004396', 'gse171524', 'gse122960', 'gse134174', 'syn21041850', 'gse135851'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10387117: 0.047619047619047616\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10387117: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4444046\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn1734155', 'syn1729383', 'syn1710680'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn1734155', 'syn1729383', 'syn1710680', 'pb002581'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4444046: 0.75\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4444046: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10329279\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21478998'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: set()\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279: 0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11145390\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21788425'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse147807', 'clean_nodatasets_060720.rdata', 'clean_060720.rdata', 'entropy_functions.r'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11145390: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11145390: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11032436\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2759792'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'nds00143', 'syn2759792'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9984184\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn39456106', 'syn25576878'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse169185', 'syn39456106', 'gse147205', 'syn25576878'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9984184: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9984184: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7347884\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1757053'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'e-mtab-4145', 'gse56031', 'syn1757053', 'egas00000000083'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7347884: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7347884: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10802452\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2759792'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'nimh', 'tba', 'geo'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8907344\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn26134616'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.7303/syn26134616', 'syn26134616'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8907344: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8907344: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10238095\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2623706'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse17536', 'gse13067', 'gse38832', 'gse14333', 'gse39582', 'gse33113', 'gse103479', 'gse104645', 'gse78220', 'https://www.synapse.org/#!synapse:syn2623706', 'gse13294', 'gse37892', 'gse135222', 'prjeb23709', 'gse108989'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=https://www.synapse.org/#!synapse:syn2623706, gt_id=syn2623706\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10238095: 0.06666666666666667\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10238095: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9710693\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 4. Element IDs: {'syn3505720', 'syn3191087', 'syn23554292', 'syn3219045'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse72056', '10.7303/syn3219045', 'e-mtab-5060', 'syn23554292', 'e-mtab-5061', 'syn3191087', 'syn3505720', 'gse75748'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn3219045, gt_id=syn3219045\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9710693: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9710693: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9915613\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1688369'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse102441', 'gse99542', 'gse84515', 'gse78167', 'gse25710', 'gse78913', 'syn1688369', 'gse154873', 'gse73663', 'gse96517', 'gse62944', 'gse57436', 'gse51403', 'gse56066'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9915613: 0.07142857142857142\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9915613: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11420198\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn4921369', 'syn26164834'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn4921369', 'gse126957', 'gse96107', 'gse59119', 'gse142881', 'gse249687', 'gse84176'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11420198: 0.14285714285714285\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11420198: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7865022\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20681023'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.6084/m9.figshare.13342055', '10.7303/syn20681023'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn20681023, gt_id=syn20681023\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7865022: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7865022: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8131595\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn18485175', 'syn4587609'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse60192', 'gse113673', 'gse96107', 'gse65776', 'e-mtab-9045', 'gse84245', 'gse59318', 'phs001373.v2.p2', 'gse150130', 'syn18485175', 'encsr198zyj', 'encsr677hxc', 'encsr822cea', 'srx3381923', 'gse59630', 'syn4587609'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8131595: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8131595: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10828232\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn13363290'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn13363290', 'gse214628'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10828232: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10828232: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11852220\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn51365301'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn51365301', 'r9'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11852220: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11852220: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9314356\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4907804'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'https://www.niagads.org', 'https://www.synapse.org/#!synapse:syn4907804', 'http://adni.loni.usc.edu', 'adni.loni.usc.edu'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=https://www.synapse.org/#!synapse:syn4907804, gt_id=syn4907804\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9314356: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9314356: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11439699\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 10. Element IDs: {'syn3157325', 'syn10235592', 'syn26720676', 'syn3157275', 'syn4896408', 'syn2580853', 'syn17008935', 'syn20068543', 'syn22264775', 'syn9702085'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn10235592', '10.7303/syn3157325', 'syn338856', 'syn26720676', 'syn3157275', 'syn4896408', 'syn2580853', 'syn17008935', 'https://adknowledgeportal.synapse.org/explore/studies/detailspage/studydetails?study=syn9702085', 'https://adknowledgeportal.synapse.org/explore/studies/detailspage/studydetails?study=syn22264775', 'syn20068543', 'syn22264775'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn3157325, gt_id=syn3157325\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=https://adknowledgeportal.synapse.org/explore/studies/detailspage/studydetails?study=syn9702085, gt_id=syn9702085\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11439699: 0.8333333333333334\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11439699: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6323985\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn6131484'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'hotspot2', 'uberon', 'cpics', 'hocomoco', 'cellosaurus', 'trim-adapters-illumina', 'sra', 'encode', 'geo'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10680627\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn52234396', 'syn52264240'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'encsr998ydi', 'gse225157', 'syn52234396', 'syn52264240', 'encsr443vtk'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10680627: 0.4\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10680627: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc5192600\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn2787209', 'syn3049712'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn2787209', 'syn3049712'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC5192600: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC5192600: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7752919\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2623706'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse2109', 'gse38832', 'gse14333', 'gse39582', 'gse33113', 'tcga-read', 'gse44076', 'gse18088', 'gse13294', 'gse148394', 'e-mtab-8152', 'syn2623706', 'tcga-coad'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7752919: 0.07692307692307693\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7752919: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8859891\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4224222'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn4224222'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9853518\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn3193805'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'3757476', 'syn3193805'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9853518: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9853518: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11852190\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn3193805'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn3193805'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11852190: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11852190: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10652415\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn26254632', 'syn3157322', 'syn3388564'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn3157322', 'syn3388564'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10652415: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10652415: 0.6666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7842189\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2580853'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'https://www.synapse.org/#!synapse:syn7818788', 'gse57181', 'gse1432', 'https://www.synapse.org/#!synapse:syn18485175', 'https://www.synapse.org/#!synapse:syn9884351', 'gse75431', 'gse61500', 'ng00040', 'phs002197.v1.p1', 'gse13162', 'gse65067', 'gse121654', 'https://repo-prod.prod.sagebase.org/repo/v1/doi/locate?id=syn3163039&type=entity', 'gse146866', 'gse98969', 'gse55627', 'https://adknowledgeportal.synapse.org/explore/studies/detailspage?study=syn3157182', 'gse103334', '10.7303/syn2580853', 'gse138852', 'gse98401', 'gse61499', 'gse90696'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn2580853, gt_id=syn2580853\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7842189: 0.043478260869565216\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7842189: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc3788619\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn1729383', 'syn2237931'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn1729383', 'syn2237931'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC3788619: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC3788619: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6750256\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4587609'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'http://dx.doi.org/10.7303/syn4587609', 'syn4587609'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6750256: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6750256: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11370573\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn25569736'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn25569736'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11370573: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11370573: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4344205\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1688369'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'doi:10.7303/syn1688369'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=doi:10.7303/syn1688369, gt_id=syn1688369\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4344205: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4344205: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9630780\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 8. Element IDs: {'syn34542574', 'syn25876213', 'syn25886762', 'syn34539904', 'syn34542449', 'syn20843407', 'syn25792906', 'syn34542450'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn34542574', 'syn25876213', 'syn25886762', 'syn34539904', 'ijo1366', 'syn34542449', 'syn20843407', '10.7303/syn20843407', 'syn25792906', 'syn34542450'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9630780: 0.8\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9630780: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10813059\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn52138250', 'syn31952297', 'syn31947506'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn52138250', 'syn31952297', 'syn31947506'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10813059: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10813059: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6089972\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn7343873', 'syn2812961', 'syn7214402'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'prjna258216', 'gse68379', 'gse74877', '10.5281/zenodo.1304766', 'syn7214402', 'prjeb5468', 'syn7343873', 'gse38266', 'syn2812961', 'gse38270', 'prjeb11844', '10.5281/zenodo.1298968', 'gse35069'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6089972: 0.23076923076923078\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6089972: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8130821\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 4. Element IDs: {'syn23191787', 'syn9884314', 'syn17015098', 'syn20803583'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn23191787', 'syn9884314', 'syn17015098', 'syn20803583'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8130821: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8130821: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11224609\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn10901595'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.5281/zenodo.10795678', 'syn10901595', 'https://github.com/joonan-lab/cwas', '10.5281/zenodo.11240547'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11224609: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11224609: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7577712\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20821165'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse33000', 'pxd009589', 'syn20821165'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7577712: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7577712: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7257173\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21541022'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn21541022'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7257173: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7257173: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9780309\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn26560886'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'26', '10.7303/syn26560886'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=26, gt_id=syn26560886\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9780309: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9780309: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7658217\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21041850'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'c4077b3c-5c98-4d26-a614-246d12c2e5d7', '10.2139/ssrn.3582067', 'gse122960', '10.1101/742320', 'gse132355', 'syn21041850'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7658217: 0.16666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7658217: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6289083\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn11967914'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'egas00001003068'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6289083: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6289083: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8807579\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20808171'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse164466', 'gse36980', 'gse158233', 'syn20808171', 'prjeb24858', 'gse151999', 'gse149149', 'gse156167'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8807579: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8807579: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6700281\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2759792'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'https://www.synapse.org/#!synapse:syn2759792/wiki/69613', 'https://datadryad.org/resource/doi:10.5061/dryad.5d4s2r8', 'phs000424.v7.p1'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=https://www.synapse.org/#!synapse:syn2759792/wiki/69613, gt_id=syn2759792\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6700281: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6700281: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7029360\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn8717496'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn8717496'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8219045\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn3163039'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse97942', 'e-mtab-5061', 'gse64810', 'syn3163039'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8219045: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8219045: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10543041\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn50996324'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn50996324'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10543041: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10543041: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6546784\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4921369'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse77565', 'phs001190.v1.p1'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6546784: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6546784: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10696419\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn22255433'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn22255433'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10696419: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10696419: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc5387919\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 10. Element IDs: {'syn4984604', 'syn6035832', 'syn6022474', 'syn6022465', 'syn5493036', 'syn5492805', 'syn5522790', 'syn5556216', 'syn5522788', 'syn5004874'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn4984604', 'syn5522790', 'syn5556216', 'syn5522788', 'syn5004874'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC5387919: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC5387919: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8094983\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn23532550'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'epi_isl_461052', 'epi_isl_461054', 'epi_isl_461001', 'epi_isl_461056', 'epi_isl_461003', 'epi_isl_461002', 'syn23532550', 'epi_isl_461051', 'epi_isl_461055', 'epi_isl_461057', 'epi_isl_461059', 'epi_isl_461060', 'epi_isl_461058', 'epi_isl_461053', 'epi_isl_415464'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8094983: 0.06666666666666667\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8094983: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7962955\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20640275'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse57763', 'gse33584', 'nc_045512.2', 'syn20640275', 'gse137834', 'mn996532.1', 'gse102470', 'e-mtab-9905'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7962955: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7962955: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11601401\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn51150434', 'syn62110225', 'syn20933797'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn51150434', '10.58085/hr6s-2991', 'syn20933797', '10.7303/syn62110225'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn62110225, gt_id=syn62110225\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11601401: 0.75\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11601401: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9712627\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn12514624'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse174367', 'gse160936', 'gse174332', 'gse167494', 'gse183068', 'gse137444', 'gse178265', 'syn12514624', 'gse148822'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9712627: 0.1111111111111111\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9712627: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4697768\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2813581'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'https://www.synapse.org/#!synapse:syn2813581'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=https://www.synapse.org/#!synapse:syn2813581, gt_id=syn2813581\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4697768: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4697768: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9094742\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21125841'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse116470', 'gse97930', 'gse775330', 'human-archived-brain', 'pxd020007', 'syn21125841', 'gse113973', 'gse129788', 'gse118257', 'srp135960', 'gse60361', 'gse138852', 'gse130119', 'gse140511', 'pxd029727'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9094742: 0.06666666666666667\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9094742: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8565335\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn6131484'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse115046'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8565335: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8565335: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10985232\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn10003873'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'https://github.com/xidslab/scfusion', 'https://github.com/ndaniel/fusioncatcher', 'https://github.com/fusioninspector/fusioninspector', 'https://github.com/shenlab-genomics/anchored-fusion', 'sra:srp291312', 'sra:srp186687', 'synapse:syn10003873', 'https://github.com/star-fusion', 'https://github.com/suhrig/arriba', 'sra:srp158590', '10.5281/zenodo.10677267'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=synapse:syn10003873, gt_id=syn10003873\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10985232: 0.09090909090909091\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10985232: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8784774\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20968331'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse186341', 'https://www.doi.org/10.7303/syn20968331', 'syn20968331'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8784774: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8784774: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8082263\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn11801497'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'https://api.gdc.cancer.gov/data/1b5f413e-a8d1-4d10-92eb-7c4ae739ed81', 'http://api.gdc.cancer.gov/data/7d4c0344-f018-4ab0-949a-09815f483480', 'https://api.gdc.cancer.gov/data/1a7d7be8-675d-4e60-a105-19d4121bdebf', 'https://www.synapse.org/#!synapse:syn11801497', 'http://api.gdc.cancer.gov/data/4f277128-f793-4354-a13d-30cc7fe9f6b5', 'https://gdc.cancer.gov/about-data/publications/ccg-aim-2020', 'https://gdc.cancer.gov/about-data/publications/pancanatlas', 'https://api.gdc.cancer.gov/data/d773a53e-b78d-4a20-848c-866616352328'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=https://www.synapse.org/#!synapse:syn11801497, gt_id=syn11801497\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8082263: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8082263: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11659981\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21041850'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse162170', 'gse134432', 'e-mtab-6149', 'tcga-luad', 'e-mtab-6653', 'scp498', 'syn21041850', 'gse115469'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11659981: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11659981: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9356164\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn12169715', 'syn12169709'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'tcga', 'syn12169715', 'syn12169709'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9356164: 0.6666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9356164: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11271596\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn53009474'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.7303/syn53009474'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn53009474, gt_id=syn53009474\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11271596: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11271596: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7520714\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn18344812', 'syn17009517'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.6084/m9.figshare.8159525.v1', '8159525.v1', 'syn17009517'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7520714: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7520714: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10769298\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn11724057'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'rosmap'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10769298: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10769298: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8226229\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2701943'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'https://portal.gdc.cancer.gov', 'syn2701943', 'gse30784', 'https://figshare.com/articles/dataset/njhncc-rnaseq-stemsig_txt/14540610'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8226229: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8226229: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11716409\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4231880'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gpl18573', '10.7303/syn4231880', '10.13026/c2f305', 'gse16476', 'gse89413', 'e-mtab-8248', 'gse79209', '10.13026/c28g6p'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn4231880, gt_id=syn4231880\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11716409: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11716409: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8055881\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn7824274'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'https://cptac-data-portal.georgetown.edu/study-summary/s039', 'https://cptac-data-portal.georgetown.edu/study-summary/s038', 'https://cptac-data-portal.georgetown.edu/study-summary/s037', 'syn7824274', 'https://cptac-data-portal.georgetown.edu/study-summary/s053', 'https://github.com/ding-lab/hotpho_analysis', 'https://github.com/ding-lab/hotspot3d', 'https://cptac-data-portal.georgetown.edu/study-summary/s050'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8055881: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8055881: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8102856\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn7214402'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'tcga', 'syn7214402', 'cnp0001438'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8102856: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8102856: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9085578\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn8612097'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn8612097'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9085578: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9085578: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9840277\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn11914606', 'syn23569889', 'syn14237651'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn11914606', 'syn23569889', 'syn14237651'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9840277: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9840277: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10547713\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn26707219', 'syn52137895'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.7303/syn26707219'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn26707219, gt_id=syn26707219\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10547713: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10547713: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10571258\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn30821562'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.5281/zenodo.8322854', 'https://www.synapse.org/#!synapse:syn30821562', 'https://github.com/aghasemian/communityfitnet', 'https://github.com/davidrconnell/igraph-se2', 'https://github.com/mkrzak/benchmarking_clustering_methods_scrnaseq'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=https://www.synapse.org/#!synapse:syn30821562, gt_id=syn30821562\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10571258: 0.2\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10571258: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6829597\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn18796452'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'https://www.ncbi.nlm.nih.gov/protein/q32852', 'syn18796452', 'https://www.ncbi.nlm.nih.gov/nuccore/r25005'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6829597: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6829597: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11240079\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn52566791'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn52566791'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11240079: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11240079: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7805663\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn11737848'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn11737848'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7805663: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7805663: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10600818\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn23627957'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn23627957'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10600818: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10600818: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc5752539\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1741407'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse29623', 'gse63459', 'gse63805'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC5752539: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC5752539: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6602064\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn9884357', 'syn7818788'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn9884357', 'gse90696', 'syn7818788'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6602064: 0.6666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6602064: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7032692\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1710250'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn1710250', 'egas00000000083'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7032692: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7032692: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10061294\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn1715755', 'syn2468297'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.7303/syn2468297', 'syn1715755'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn2468297, gt_id=syn2468297\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10061294: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10061294: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8006362\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn22024464'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'792', 'syn22024464', '10.15154/1504248'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8006362: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8006362: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7778917\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn18360482'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'pharos'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7778917: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7778917: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 89 - INFO - \n",
      "Performance evaluation completed for 100 source pages.\u001b[0m\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "{'average_precision': 0.48885651088042414,\n",
       " 'average_recall': 0.8646666666666667,\n",
       " 'f1_score': 0.6245891267370887}"
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "check_df = pd.read_csv(\"scripts/output/syn_data_results/result_first try.csv\")\n",
    "ret = evaluate_performance(check_df, ground_truth, dg, \"scripts/output/false_positives.txt\")\n",
    "ret"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "49f712fb5db57045",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-09-04T20:51:39.708798Z",
     "start_time": "2025-09-04T20:51:29.136798Z"
    }
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11792374\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 4. Element IDs: {'syn52293417', 'syn8456637', 'syn2580853', 'syn9702085'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn8456637', 'm27188', 'syn2580853', 'm27187', 'syn52293417', 'syn9702085'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374: 0.6666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4318527\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn2455683', 'syn1720047'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'assemblathon1&2', 'dream', 'cafa', 'clarity', 'improver', 'rgasp', 'syn1720047', 'cagi', 'innocentive', 'flowcap', 'x-prize', 'syn2455683', 'casp', 'chalearn', 'kaggle', 'capri'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4339277\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 4. Element IDs: {'syn1701558', 'syn1682511', 'syn1688309', 'syn1732077'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn300013', 'syn1688309', 'syn1682511', 'syn1732077', 'syn1701558'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277: 0.8\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8452114\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn18485175'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn18485175'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8452114: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8452114: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9641692\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn3219045'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse119117', 'gse67835', 'gse65133', 'syn3219045', 'gse107011', 'gse115449', 'gse87544', 'gse84133'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9641692: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9641692: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc5161470\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1446295'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn1446295'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC5161470: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC5161470: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9925742\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21126450'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'srr21079269', 'syn21126450', 'gse224398', 'srr21079265', 'srr21079270', 'srr21079266', 'srr21079268', 'srr21079267'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9925742: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9925742: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9235070\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn30243143'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse76312', 'gse75688', 'gse103322', 'gse81383', 'syn213309', 'gse70630', 'gse81861', 'syn30243143'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9235070: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9235070: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11661334\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn51364943'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn51364943', 'gcst009325', 'cowas', 'gcst007511', 'gcst90027158'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11661334: 0.2\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11661334: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11929800\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 5. Element IDs: {'syn3388564', 'syn11724057', 'syn4896408', 'syn18485175', 'syn17015098'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn3388564', 'syn11724057', 'syn4896408', 'gse291402', 'syn17015098', 'gcst90027158'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11929800: 0.6666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11929800: 0.8\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7113080\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn17394041'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn17394041'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8847344\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn25813713'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn25813713'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8847344: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8847344: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9280291\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn5612563'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn5612563', 'gse21802', 'gse40586', 'gse57065', 'e-mtab-1548', 'gse54514', 'gse66099', 'e-mtab-5273/5274', 'gse40012', 'e-mexp-3589', 'gse13015', 'gse20346', 'gse27131', 'gse65682', 'gse60244', 'gse32707', 'gse25504', 'gse95233'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9280291: 0.05555555555555555\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9280291: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10029021\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn50897018'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn50897018.3'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=syn50897018.3, gt_id=syn50897018\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10029021: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10029021: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11525589\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn51514106'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.7303/syn51514106', 'brats-men'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn51514106, gt_id=syn51514106\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11525589: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11525589: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10836119\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 6. Element IDs: {'syn5592519', 'syn9705278', 'syn3157322', 'syn3205821', 'syn3157238', 'syn3157329'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'rf1ag051550', 'u01ag46152', 'hhsn271201200022c', 'n01-ag-12100', 'r01ag046171', 'w81xwh-12-2-0012', '10.7303/syn3205821.6', '10.7303/syn3157329', 'r01ag15819', '10.7303/syn3157322', '10.7303/syn5592519', 'p30ag10161', 'r01ag17917', 'u01ag61356', '10.7303/syn3157238', '10.7303/syn9705278', '3u01ag024904-09s4'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn3205821.6, gt_id=syn3205821\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn5592519, gt_id=syn5592519\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn3157238, gt_id=syn3157238\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn9705278, gt_id=syn9705278\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn3157329, gt_id=syn3157329\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn3157322, gt_id=syn3157322\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10836119: 0.35294117647058826\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10836119: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6646046\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn5609493'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'e-mtab-2323', 'syn5609493'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6646046: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6646046: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10387117\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21041850'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'lmex0000004396', 'gse161382', 'gse171524', 'gse149563', 'gse135893', 'gse122960', 'egas00001004082', 'gse134174', 'gse122332', 'gse135851', 'syn21041850', 'gse136832', 'lmex0000004397', 'dataset-427f1eee6dd44f50bae1ab13f0f3c6a9', 'gse178362'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10387117: 0.06666666666666667\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10387117: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4444046\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn1734155', 'syn1729383', 'syn1710680'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn1734155', 'syn1729383', 'syn1710680', 'pb002581'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4444046: 0.75\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4444046: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10329279\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21478998'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'pmc5789045', 'pmc7786238', 'pmc6720753', 'pmc7805732', 'pmc6422329', 'pmc6166867', 'pmc6921374', 'pmc3766928', 'pmc3135413', 'pmc6939339'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11145390\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21788425'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn21788425', 'gse147807', '10.5281/zenodo.10971926'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11145390: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11145390: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11032436\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2759792'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn2759792'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9984184\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn39456106', 'syn25576878'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse169185', 'syn39456106', 'gse147205', 'syn25576878'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9984184: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9984184: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7347884\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1757053'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'e-mtab-4145', 'gse56031', 'syn1757053', 'egas00000000083'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7347884: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7347884: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10802452\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2759792'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'tba'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8907344\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn26134616'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.7303/syn26134616', 'syn26134616'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8907344: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8907344: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10238095\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2623706'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse17536', 'gse13067', 'gse38832', 'gse14333', 'gse39582', 'gse33113', 'gse103479', 'tcga', 'gse104645', 'gse78220', 'gse13294', 'gse37892', 'gse35144', 'gse135222', 'prjeb23709', 'gse108989'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10238095: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10238095: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9710693\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 4. Element IDs: {'syn3505720', 'syn3191087', 'syn23554292', 'syn3219045'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse72056', '10.7303/syn3219045', 'e-mtab-5060', 'syn23554292', 'e-mtab-5061', 'syn3191087', 'syn3505720', 'gse75748'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn3219045, gt_id=syn3219045\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9710693: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9710693: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9915613\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1688369'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse102441', 'gse99542', 'gse84515', 'gse78167', 'gse25710', 'gse78913', 'syn1688369', 'gse154873', 'gse73663', 'gse96517', 'gse62944', 'gse57436', 'gse51403', 'gse56066'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9915613: 0.07142857142857142\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9915613: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11420198\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn4921369', 'syn26164834'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn4921369', 'gse126957', 'gse96107', '10.1073/', 'gse59119', 'gse142881', 'gse249687', 'gse84176'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11420198: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11420198: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7865022\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20681023'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.6084/m9.figshare.13342055', '10.7303/syn20681023'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn20681023, gt_id=syn20681023\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7865022: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7865022: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8131595\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn18485175', 'syn4587609'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse60192', 'gse113673', 'gse96107', 'gse65776', 'e-mtab-9045', 'gse84245', 'gse59318', 'phs001373.v2.p2', 'gse150130', 'syn18485175', 'encsr198zyj', 'encsr677hxc', 'encsr822cea', 'srx3381923', 'gse59630', 'syn4587609'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8131595: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8131595: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10828232\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn13363290'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn13363290', 'gse214628'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10828232: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10828232: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11852220\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn51365301'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn51365301'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11852220: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11852220: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9314356\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4907804'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn4907804', 'www.niagads.org', 'adni.loni.usc.edu'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9314356: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9314356: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11439699\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 10. Element IDs: {'syn3157325', 'syn10235592', 'syn26720676', 'syn3157275', 'syn4896408', 'syn2580853', 'syn17008935', 'syn20068543', 'syn22264775', 'syn9702085'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn10235592', '10.7303/syn3157325', 'syn338856', 'syn26720676', 'syn3157275', 'syn4896408', 'syn2580853', 'syn17008935', 'syn20068543', 'syn22264775', 'syn9702085'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn3157325, gt_id=syn3157325\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11439699: 0.9090909090909091\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11439699: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6323985\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn6131484'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn6131484', '10.1093/'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10680627\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn52234396', 'syn52264240'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'encsr998ydi', 'gse225157', 'syn52234396', 'syn52264240', 'encsr443vtk'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10680627: 0.4\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10680627: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc5192600\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn2787209', 'syn3049712'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn2787209', 'syn3049712'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC5192600: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC5192600: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7752919\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2623706'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse2109', 'gse38832', 'gse14333', 'gse39582', 'gse33113', 'tcga-read', 'gse44076', 'gse18088', 'gse13294', 'gse148394', 'e-mtab-8152', 'syn2623706', 'tcga-coad'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7752919: 0.07692307692307693\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7752919: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8859891\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4224222'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn4224222'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9853518\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn3193805'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'3757476', 'syn3193805'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9853518: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9853518: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11852190\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn3193805'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn3193805'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11852190: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11852190: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10652415\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn26254632', 'syn3157322', 'syn3388564'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn26254632', 'syn3157322', 'syn3388564'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10652415: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10652415: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7842189\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2580853'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse57181', 'gse1432', 'syn18485175', 'gse75431', 'gse61500', 'syn7818788', 'phs002197.v1.p1', 'gse13162', 'syn9884351', 'gse65067', 'gse146866', 'gse98969', 'gse55627', 'gse103334', 'syn2580853', 'syn3163039', 'gse138852', 'gse98401', 'syn3157182', 'gse61499', 'gse90696'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7842189: 0.047619047619047616\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7842189: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc3788619\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn1729383', 'syn2237931'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn1729383', 'syn2237931'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC3788619: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC3788619: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6750256\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4587609'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn4587609'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6750256: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6750256: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11370573\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn25569736'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn25569736'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11370573: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11370573: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4344205\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1688369'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'doi:10.7303/syn1688369'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=doi:10.7303/syn1688369, gt_id=syn1688369\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4344205: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4344205: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9630780\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 8. Element IDs: {'syn34542574', 'syn25876213', 'syn25886762', 'syn34539904', 'syn34542449', 'syn20843407', 'syn25792906', 'syn34542450'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn34542574', 'syn25876213', 'syn25886762', 'syn34539904', 'syn34542449', 'syn20843407', '10.7303/syn20843407', 'syn25792906', 'syn34542450'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9630780: 0.8888888888888888\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9630780: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10813059\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn52138250', 'syn31952297', 'syn31947506'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn52138250', 'syn31952297', 'syn31947506'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10813059: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10813059: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6089972\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn7343873', 'syn2812961', 'syn7214402'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'prjna258216', 'gse68379', 'gse74877', '10.5281/zenodo.1304766', 'syn7214402', 'prjeb5468', 'syn7343873', 'gse38266', 'syn2812961', 'gse38270', 'prjeb11844', '10.5281/zenodo.1298968', 'gse35069'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6089972: 0.23076923076923078\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6089972: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8130821\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 4. Element IDs: {'syn23191787', 'syn9884314', 'syn17015098', 'syn20803583'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn23191787', 'syn9884314', 'syn17015098', 'syn20803583'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8130821: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8130821: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11224609\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn10901595'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn10901595', '10.5281/'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11224609: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11224609: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7577712\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20821165'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse33000', 'pxd009589', 'syn20821165'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7577712: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7577712: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7257173\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21541022'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn21541022'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7257173: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7257173: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9780309\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn26560886'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn26560886'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9780309: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9780309: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7658217\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21041850'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.1101/', 'gse122960', '10.2139/', 'gse132355', 'syn21041850'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7658217: 0.2\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7658217: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6289083\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn11967914'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'egas00001003068'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6289083: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6289083: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8807579\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20808171'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse164466', 'gse36980', 'gse158233', 'syn20808171', 'prjeb24858', 'gse151999', 'gse149149', 'gse156167'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8807579: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8807579: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6700281\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2759792'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'doi:10.5061/dryad.5d4s2r8', 'syn2759792', 'phs000424.v7.p1'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6700281: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6700281: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7029360\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn8717496'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: set()\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360: 0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8219045\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn3163039'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse97942', 'e-mtab-5061', 'gse64810', 'syn3163039'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8219045: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8219045: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10543041\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn50996324'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn50996324', '10.13020/'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10543041: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10543041: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6546784\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4921369'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn4921369', 'gse77565', 'phs001190.v1.p1'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6546784: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6546784: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10696419\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn22255433'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn22255433'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10696419: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10696419: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc5387919\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 10. Element IDs: {'syn4984604', 'syn6035832', 'syn6022474', 'syn6022465', 'syn5493036', 'syn5492805', 'syn5522790', 'syn5556216', 'syn5522788', 'syn5004874'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn4984604', 'syn6035832', 'syn6022474', 'syn6022465', 'syn5493036', 'syn5492805', 'syn5522790', 'syn5556216', 'syn5522788', 'syn5004874'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC5387919: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC5387919: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8094983\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn23532550'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'epi_isl_461052', 'epi_isl_461054', 'epi_isl_461001', 'epi_isl_461056', 'epi_isl_461003', 'epi_isl_461002', 'syn23532550', 'epi_isl_461051', 'epi_isl_461055', 'epi_isl_461057', 'epi_isl_461059', 'epi_isl_461060', 'epi_isl_461058', 'epi_isl_461053', 'epi_isl_415464'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8094983: 0.06666666666666667\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8094983: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7962955\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20640275'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse57763', 'gse33584', 'nc_045512.2', 'syn20640275', 'gse137834', 'mn996532.1', 'gse102470', 'e-mtab-9905'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7962955: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7962955: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11601401\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn51150434', 'syn62110225', 'syn20933797'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn51150434', '10.58085/hr6s-2991', 'syn20933797', '10.7303/syn62110225'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn62110225, gt_id=syn62110225\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11601401: 0.75\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11601401: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9712627\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn12514624'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse174367', 'gse160936', 'gse174332', 'gse167494', 'gse183068', 'gse137444', 'gse178265', 'gse148822'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9712627: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9712627: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4697768\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2813581'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn2813581'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4697768: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4697768: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9094742\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21125841'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse116470', 'gse97930', 'gse775330', 'human-archived-brain', 'pxd020007', 'syn21125841', 'gse113973', 'gse129788', 'gse118257', 'srp135960', 'gse60361', 'gse138852', 'gse130119', 'gse140511', 'pxd029727'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9094742: 0.06666666666666667\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9094742: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8565335\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn6131484'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'encsr000ekp', 'encsr483rkn', 'gse115046', 'encsr220asc', 'encsr000e', 'sra'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8565335: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8565335: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10985232\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn10003873'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn10003873', 'srp186687', 'nm_001306068.3', 'srp158590', 'srp291312', 'nm_133330.3', '10.5281/'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10985232: 0.14285714285714285\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10985232: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8784774\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20968331'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse186341', 'syn20968331'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8784774: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8784774: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8082263\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn11801497'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn11801497'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8082263: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8082263: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11659981\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21041850'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse162170', 'gse134432', 'e-mtab-6149', 'e-mtab-6653', 'scp498', 'syn21041850', 'gse115469'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11659981: 0.14285714285714285\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11659981: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9356164\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn12169715', 'syn12169709'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn12169715', 'syn12169709', '3586c0da-64d0-4b74-a449-5ff4d9136611'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9356164: 0.6666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9356164: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11271596\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn53009474'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.7303/syn53009474', 'voc-als.git'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn53009474, gt_id=syn53009474\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11271596: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11271596: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7520714\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn18344812', 'syn17009517'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'8159525.v1', 'syn17009517'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7520714: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7520714: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10769298\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn11724057'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'pmc10769298', '38187779'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10769298: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10769298: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8226229\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2701943'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn2701943', 'gse30784'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8226229: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8226229: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11716409\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4231880'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'medquad', 'mit-bih', 'mimic-iii', 'gse16476', 'gse89413', 'e-mtab-8248', 'gse79209'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11716409: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11716409: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8055881\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn7824274'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn7824274'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8055881: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8055881: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8102856\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn7214402'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'tcga', 'syn7214402', 'cnp0001438'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8102856: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8102856: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9085578\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn8612097'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn8612097'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9085578: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9085578: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9840277\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn11914606', 'syn23569889', 'syn14237651'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn11914606', 'syn23569889', 'syn14237651'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9840277: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9840277: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10547713\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn26707219', 'syn52137895'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.7303/syn26707219'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn26707219, gt_id=syn26707219\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10547713: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10547713: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10571258\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn30821562'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'github.com/cogdishion/se2', 'syn30821562', '10.5281/zenodo.8338532', '10.5281/'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10571258: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10571258: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6829597\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn18796452'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.1084/', 'syn18796452', 'r25005'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6829597: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6829597: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11240079\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn52566791'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn52566791'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11240079: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11240079: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7805663\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn11737848'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn11737848'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7805663: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7805663: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10600818\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn23627957'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn23627957'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10600818: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10600818: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc5752539\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1741407'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse29623', 'gse63459', 'gse63805'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC5752539: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC5752539: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6602064\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn9884357', 'syn7818788'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn9884357', 'gse90696', 'syn7818788'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6602064: 0.6666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6602064: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7032692\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1710250'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'egac00001000484', 'syn1710250', 'egas00000000083'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7032692: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7032692: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10061294\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn1715755', 'syn2468297'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.7303/syn2468297', 'syn1715755'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn2468297, gt_id=syn2468297\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10061294: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10061294: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8006362\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn22024464'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'792', '10.5281/zenodo.4321679', 'syn22024464', '10.15154/1504248'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8006362: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8006362: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7778917\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn18360482'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'pharos', 'syn18360482'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7778917: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7778917: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 89 - INFO - \n",
      "Performance evaluation completed for 100 source pages.\u001b[0m\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "{'average_precision': 0.5255226407579351,\n",
       " 'average_recall': 0.883,\n",
       " 'f1_score': 0.6588981651577225}"
      ]
     },
     "execution_count": 18,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "check_df = pd.read_csv(\"scripts/output/syn_data_results/result_schema_validation.csv\")\n",
    "ret = evaluate_performance(check_df, ground_truth, dg, \"scripts/output/false_positives.txt\")\n",
    "ret"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "5768c91b8f925301",
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-09-04T20:51:49.942643Z",
     "start_time": "2025-09-04T20:51:39.748779Z"
    }
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11792374\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 4. Element IDs: {'syn52293417', 'syn8456637', 'syn2580853', 'syn9702085'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn8456637', 'm27188', 'syn2580853', 'm27187', 'syn52293417', 'syn9702085'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374: 0.6666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4318527\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn2455683', 'syn1720047'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'assemblathon1&2', 'dream', 'cafa', 'clarity', 'improver', 'rgasp', 'syn1720047', 'cagi', 'innocentive', 'flowcap', 'x-prize', 'syn2455683', 'casp', 'chalearn', 'kaggle', 'capri'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4339277\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 4. Element IDs: {'syn1701558', 'syn1682511', 'syn1688309', 'syn1732077'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn1732077'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8452114\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn18485175'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn18485175'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8452114: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8452114: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9641692\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn3219045'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse119117', 'gse67835', 'gse65133', 'syn3219045', 'gse107011', 'gse115449', 'gse87544', 'gse84133'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9641692: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9641692: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc5161470\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1446295'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'1kgp', 'syn1446295', 'enr113', 'enm014', 'enr112', 'enm013', 'chr22'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC5161470: 0.14285714285714285\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC5161470: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9925742\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21126450'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'srr21079269', 'syn21126450', 'gse224398', 'srr21079265', 'srr21079270', 'srr21079266', 'srr21079268', 'srr21079267'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9925742: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9925742: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9235070\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn30243143'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse76312', 'gse75688', 'gse103322', 'e-mtab-6149', 'gse81383', 'gse70630', 'gse81861', 'syn30243143'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9235070: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9235070: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11661334\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn51364943'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn51364943', 'gcst007511', 'gcst009325', 'gcst90027158'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11661334: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11661334: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11929800\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 5. Element IDs: {'syn3388564', 'syn11724057', 'syn4896408', 'syn18485175', 'syn17015098'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn3388564', 'syn11724057', 'syn4896408', 'gse291402', 'syn18485175', 'syn17015098', 'gcst90027158'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11929800: 0.7142857142857143\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11929800: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7113080\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn17394041'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn17394041'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8847344\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn25813713'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn25813713'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8847344: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8847344: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9280291\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn5612563'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.7303/syn5612563', 'gse21802', 'gse40586', 'gse57065', 'e-mtab-1548', 'gse54514', 'gse66099', 'e-mtab-5273/5274', 'gse40012', 'e-mexp-3589', 'gse13015', 'gse20346', 'gse27131', 'gse65682', 'gse60244', 'gse32707', 'gse25504', 'gse95233'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn5612563, gt_id=syn5612563\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9280291: 0.05555555555555555\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9280291: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10029021\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn50897018'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn50897018.3'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=syn50897018.3, gt_id=syn50897018\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10029021: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10029021: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11525589\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn51514106'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn51514106', '10.7303/syn51514106'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11525589: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11525589: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10836119\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 6. Element IDs: {'syn5592519', 'syn9705278', 'syn3157322', 'syn3205821', 'syn3157238', 'syn3157329'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'rf1ag57473', 'hhsn271201200022c', 'n01-ag-12100', 'r01ag15819', 'u01ag32984', 'rc2ag036547', 'rf1ag051550', 'w81xwh-12-2-0012', '10.7303/syn3157329', 'r01ag30146', 'r01ag17917', 'r01ag36836', 'r01ag48015', '10.7303/syn3205821.6', '10.7303/syn5592519', 'p30ag10161', '10.7303/syn3157238', '10.7303/syn9705278', 'u01ag46152', 'r01ag046171', 'u01ag46161', '10.7303/syn3157322', 'u01ag61356', 'r01ag36042', '3u01ag024904-09s4'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn3205821.6, gt_id=syn3205821\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn5592519, gt_id=syn5592519\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn3157238, gt_id=syn3157238\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn9705278, gt_id=syn9705278\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn3157329, gt_id=syn3157329\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn3157322, gt_id=syn3157322\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10836119: 0.24\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10836119: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6646046\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn5609493'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn5609493'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6646046: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6646046: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10387117\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21041850'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse161382', 'gse149563', 'gsm5388412', 'gse122332', 'gsm4504966', 'dataset-427f1eee6dd44f50bae1ab13f0f3c6a9', 'gsm5388411', 'gsm4035472', 'cellref', 'gse135893', 'gse136832', 'gsm5388413', 'egas00001004082', 'lmex0000004397', 'gsm4504967', 'lmex0000004396', 'gse171524', 'gse122960', 'gse134174', 'syn21041850', 'gse135851', 'gse178360'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10387117: 0.045454545454545456\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10387117: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4444046\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn1734155', 'syn1729383', 'syn1710680'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn1734155', 'syn1729383', 'syn1710680'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4444046: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4444046: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10329279\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21478998'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: set()\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279: 0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11145390\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21788425'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn21788425', 'gse147807', '10.5281/zenodo.10971926'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11145390: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11145390: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11032436\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2759792'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'crl-3216', 'nds00143', 'syn2759792'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9984184\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn39456106', 'syn25576878'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse169185', 'gse147205', 'syn39456106', 'gse15222', 'gse95587', 'syn25576878'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9984184: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9984184: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7347884\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1757053'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'e-mtab-4145', 'gse56031', 'syn1757053', 'egas00000000083'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7347884: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7347884: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10802452\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2759792'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn28580053'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8907344\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn26134616'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'doi.org/10.7303/syn26134616', 'syn26134616'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8907344: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8907344: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10238095\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2623706'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse17536', 'gse13067', 'gse38832', 'gse14333', 'gse39582', 'gse33113', 'gse103479', 'gse104645', 'gse78220', 'gse13294', 'gse37892', 'gse35144', 'gse135222', 'prjeb23709', 'gse108989', 'syn2623706'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10238095: 0.0625\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10238095: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9710693\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 4. Element IDs: {'syn3505720', 'syn3191087', 'syn23554292', 'syn3219045'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse72056', 'e-mtab-5060', 'syn23554292', 'e-mtab-5061', 'syn3191087', 'syn3505720', 'gse75748', 'syn3219045'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9710693: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9710693: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9915613\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1688369'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse102441', 'gse99542', 'gse84515', 'gse78167', 'gse25710', 'gse78913', 'gse56066', 'syn1688369', 'gse154873', 'gse73663', 'gse62944', 'gse57436', 'gse51403', 'gse96517'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9915613: 0.07142857142857142\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9915613: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11420198\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn4921369', 'syn26164834'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn4921369', 'gse126957', 'gse96107', '10.1073/', 'gse59119', 'gse142881', 'gse249687', 'gse84176'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11420198: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11420198: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7865022\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20681023'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.6084/m9.figshare.13342055', '10.7303/syn20681023'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn20681023, gt_id=syn20681023\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7865022: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7865022: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8131595\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn18485175', 'syn4587609'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse60192', 'gse113673', 'gse96107', 'gse65776', 'e-mtab-9045', 'gse84245', 'gse81912', 'gse59318', 'phs001373.v2.p2', 'gse150130', 'syn18485175', 'encsr198zyj', 'encsr677hxc', 'encsr822cea', '10.1371/', 'srx3381923', 'gse59630', 'syn4587609'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8131595: 0.1111111111111111\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8131595: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10828232\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn13363290'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse214628', 'nm_021272.3', 'syn13363290-v49'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=syn13363290-v49, gt_id=syn13363290\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10828232: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10828232: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11852220\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn51365301'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn51365301'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11852220: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11852220: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9314356\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4907804'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'adni', 'syn4907804', 'niagads'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9314356: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9314356: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11439699\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 10. Element IDs: {'syn3157325', 'syn10235592', 'syn26720676', 'syn3157275', 'syn4896408', 'syn2580853', 'syn17008935', 'syn20068543', 'syn22264775', 'syn9702085'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.7303/syn3157325', 'syn338856', 'syn26720676', 'syn2580853', 'syn17008935', 'syn20068543', 'syn22264775', 'syn9702085'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn3157325, gt_id=syn3157325\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11439699: 0.875\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11439699: 0.7\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6323985\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn6131484'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'sra', 'encode', 'geo'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10680627\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn52234396', 'syn52264240'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'encsr998ydi', 'gse225157', 'syn52234396', 'syn52264240', 'encsr443vtk'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10680627: 0.4\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10680627: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc5192600\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn2787209', 'syn3049712'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn2787209', 'syn3049712'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC5192600: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC5192600: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7752919\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2623706'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse2109', 'gse38832', 'gse14333', 'gse39582', 'gse33113', 'tcga-read', 'gse44076', 'gse18088', 'gse13294', 'gse148394', 'e-mtab-8152', 'syn2623706', 'tcga-coad'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7752919: 0.07692307692307693\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7752919: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8859891\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4224222'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn4224222'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9853518\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn3193805'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn3193805'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9853518: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9853518: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11852190\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn3193805'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'brats', 'syn3193805'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11852190: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11852190: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10652415\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn26254632', 'syn3157322', 'syn3388564'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn26254632', 'syn3157322', 'syn3388564'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10652415: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10652415: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7842189\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2580853'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse57181', 'gse1432', 'syn18485175', 'gse75431', 'gse61500', 'syn7818788', 'ng00040', 'phs002197.v1.p1', 'gse13162', 'syn9884351', 'gse65067', 'gse121654', 'gse146866', 'gse98969', 'gse55627', 'gse103334', 'syn2580853', 'syn3163039', 'gse138852', 'gse98401', 'syn3157182', 'gse61499', 'gse90696'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7842189: 0.043478260869565216\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7842189: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc3788619\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn1729383', 'syn2237931'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn1729383', 'syn2237931'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC3788619: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC3788619: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6750256\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4587609'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn4587609'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6750256: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6750256: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11370573\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn25569736'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn25569736'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11370573: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11370573: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4344205\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1688369'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'doi:10.7303/syn1688369'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=doi:10.7303/syn1688369, gt_id=syn1688369\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4344205: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4344205: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9630780\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 8. Element IDs: {'syn34542574', 'syn25876213', 'syn25886762', 'syn34539904', 'syn34542449', 'syn20843407', 'syn25792906', 'syn34542450'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn34542574', 'syn25876213', 'syn25886762', 'syn34539904', 'ijo1366', 'syn34542449', 'syn20843407', '10.7303/syn20843407', 'syn25792906', 'syn34542450'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9630780: 0.8\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9630780: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10813059\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn52138250', 'syn31952297', 'syn31947506'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn52138250', 'syn31952297', 'syn31947506'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10813059: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10813059: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6089972\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn7343873', 'syn2812961', 'syn7214402'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'prjna258216', 'gse68379', 'gse74877', 'doi.org/10.5281/zenodo.1304766', 'doi.org/10.5281/zenodo.1298968', 'gse35069', 'syn7214402', 'prjeb5468', 'syn7343873', 'syn2812961', 'prjeb11844', 'gse38270'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6089972: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6089972: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8130821\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 4. Element IDs: {'syn23191787', 'syn9884314', 'syn17015098', 'syn20803583'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn23191787', 'syn9884314', 'syn17015098', 'syn20803583'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8130821: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8130821: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11224609\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn10901595'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn10901595', '10.5281/zenodo.11240547', '10.5281/zenodo.10795678'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11224609: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11224609: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7577712\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20821165'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse33000', 'syn20821165', 'pxd009589'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7577712: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7577712: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7257173\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21541022'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn21541022'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7257173: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7257173: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9780309\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn26560886'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn26560886'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9780309: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9780309: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7658217\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21041850'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.1101/', 'c4077b3c-5c98-4d26-a614-246d12c2e5d7', 'gse122960', '10.2139/', 'gse132355', 'syn21041850'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7658217: 0.16666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7658217: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6289083\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn11967914'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'egas00001003068'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6289083: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6289083: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8807579\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20808171'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse164466', 'gse36980', 'gse158233', 'syn20808171', 'prjeb24858', 'gse151999', 'gse149149', 'gse156167'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8807579: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8807579: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6700281\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2759792'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.5061/', 'syn2759792', 'phs000424.v7.p1'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6700281: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6700281: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7029360\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn8717496'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn8717496', 'pmc7029360'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8219045\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn3163039'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse97942', 'e-mtab-5061', 'gse64810', 'syn3163039'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8219045: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8219045: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10543041\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn50996324'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn50996324', '10.13020/'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10543041: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10543041: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6546784\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4921369'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn4921369', 'gse77565', 'phs001190.v1.p1'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6546784: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6546784: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10696419\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn22255433'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn22255433'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10696419: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10696419: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc5387919\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 10. Element IDs: {'syn4984604', 'syn6035832', 'syn6022474', 'syn6022465', 'syn5493036', 'syn5492805', 'syn5522790', 'syn5556216', 'syn5522788', 'syn5004874'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn4984604', 'syn6035832', 'syn6022474', 'syn6022465', 'syn5493036', 'syn5492805', 'syn5522790', 'syn5556216', 'syn5522788', 'syn5004874'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC5387919: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC5387919: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8094983\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn23532550'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'epi_isl_461052', 'epi_isl_461054', 'epi_isl_461001', 'epi_isl_461056', 'epi_isl_461003', 'epi_isl_461002', 'syn23532550', 'epi_isl_461051', 'epi_isl_461055', 'epi_isl_461057', 'epi_isl_461059', 'epi_isl_461060', 'epi_isl_461058', 'epi_isl_461053', 'epi_isl_415464'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8094983: 0.06666666666666667\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8094983: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7962955\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20640275'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse57763', 'gse33584', 'nc_045512.2', 'syn20640275', 'gse137834', 'mn996532.1', 'gse102470', 'e-mtab-9905'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7962955: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7962955: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11601401\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn51150434', 'syn62110225', 'syn20933797'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn51150434', '10.58085/hr6s-2991', 'syn20933797', '10.7303/syn62110225'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn62110225, gt_id=syn62110225\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11601401: 0.75\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11601401: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9712627\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn12514624'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse174367', 'gse160936', 'gse174332', 'gse167494', 'gse183068', 'gse137444', 'gse178265', 'syn12514624', 'gse148822'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9712627: 0.1111111111111111\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9712627: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4697768\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2813581'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn2813581'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4697768: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4697768: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9094742\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21125841'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse116470', 'gse97930', 'gse775330', 'human-archived-brain', 'pxd020007', 'syn21125841', 'gse113973', 'gse129788', 'gse118257', 'srp135960', 'gse60361', 'gse138852', 'gse130119', 'gse140511', 'pxd029727'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9094742: 0.06666666666666667\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9094742: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8565335\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn6131484'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse115046'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8565335: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8565335: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10985232\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn10003873'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'sra:srp291312', 'sra:srp186687', 'synapse:syn10003873', 'sra:srp158590', '10.5281/'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=synapse:syn10003873, gt_id=syn10003873\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10985232: 0.2\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10985232: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8784774\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20968331'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse186341', 'syn20968331'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8784774: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8784774: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8082263\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn11801497'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn11801497'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8082263: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8082263: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11659981\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21041850'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse162170', 'gse134432', 'e-mtab-6149', '10.5524/102624', '10.5281/zenodo.12092871', 'e-mtab-6653', 'syn21041850'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11659981: 0.14285714285714285\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11659981: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9356164\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn12169715', 'syn12169709'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'tcga', 'syn12169715', 'syn12169709'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9356164: 0.6666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9356164: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11271596\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn53009474'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn53009474', '10.7303/syn53009474'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11271596: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11271596: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7520714\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn18344812', 'syn17009517'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.6084/m9.figshare.8159525.v1', 'syn17009517', '10.7303/syn18344812'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn18344812, gt_id=syn18344812\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7520714: 0.6666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7520714: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10769298\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn11724057'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'pmc10769298', 'syn11724057', '38187779'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10769298: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10769298: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8226229\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2701943'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn2701943', 'tcga', 'gse30784'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8226229: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8226229: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11716409\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4231880'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gpl18573', 'gse16476', 'gse89413', 'e-mtab-8248', 'gse79209'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11716409: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11716409: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8055881\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn7824274'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn7824274'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8055881: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8055881: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8102856\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn7214402'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'mc3', 'syn7214402', 'cnp0001438'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8102856: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8102856: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9085578\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn8612097'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn8612097'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9085578: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9085578: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9840277\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn11914606', 'syn23569889', 'syn14237651'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn11914606', 'syn23569889', 'syn14237651'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9840277: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9840277: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10547713\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn26707219', 'syn52137895'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.7303/syn26707219'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn26707219, gt_id=syn26707219\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10547713: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10547713: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10571258\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn30821562'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'igraph-se2', 'syn30821562', 'se2', '10.5281/'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10571258: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10571258: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6829597\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn18796452'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn18796452'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6829597: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6829597: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11240079\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn52566791'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn52566791'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11240079: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11240079: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7805663\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn11737848'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn11737848'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7805663: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7805663: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10600818\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn23627957'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn23627957'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10600818: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10600818: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc5752539\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1741407'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse29623', 'gse63459', 'gse63805'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC5752539: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC5752539: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6602064\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn9884357', 'syn7818788'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn9884357', 'gse90696', 'syn7818788'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6602064: 0.6666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6602064: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7032692\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1710250'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'egac00001000484', 'egas00000000083', 'syn1710250'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7032692: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7032692: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10061294\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn1715755', 'syn2468297'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.7303/syn2468297', 'syn1715755'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn2468297, gt_id=syn2468297\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10061294: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10061294: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8006362\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn22024464'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.5281/zenodo.4321679', 'syn22024464', 'ftp://ftp-trace.ncbi.nlm.nih.gov/1000genomes/ftp/1000g_2504_high_coverage', '10.15154/1504248'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8006362: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8006362: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7778917\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn18360482'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn18360482'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7778917: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7778917: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 89 - INFO - \n",
      "Performance evaluation completed for 100 source pages.\u001b[0m\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "{'average_precision': 0.5301756223245355,\n",
       " 'average_recall': 0.9095,\n",
       " 'f1_score': 0.6698657961931753}"
      ]
     },
     "execution_count": 19,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "check_df = pd.read_csv(\"scripts/output/syn_data_results/result_Prompt_enhancement.csv\")\n",
    "ret = evaluate_performance(check_df, ground_truth, dg, \"scripts/output/false_positives.txt\")\n",
    "ret"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "3c757d7d13496047",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11792374\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 4. Element IDs: {'syn52293417', 'syn8456637', 'syn2580853', 'syn9702085'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn8456637', 'm27188', 'syn2580853', 'm27187', 'syn52293417', 'syn9702085'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374: 0.6666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4318527\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn2455683', 'syn1720047'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn2455683', 'syn312572'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4339277\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 4. Element IDs: {'syn1701558', 'syn1682511', 'syn1688309', 'syn1732077'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn1701558', 'syn1682511', 'syn1688309', 'syn1732077'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8452114\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn18485175'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn18485175'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8452114: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8452114: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9641692\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn3219045'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse119117', 'gse67835', 'gse65133', 'gse107011', 'gse115449', 'syn3219045', 'gse87544', 'gse84133'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9641692: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9641692: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc5161470\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1446295'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'1000gp_phase3', '20130502', 'enr113', 'pone.0167994.s006.pdf', 'enm014', 'pone.0167994.s004.pdf', 'enr112', 'pone.0167994.s001.pdf', 'pone.0167994.s010.pdf', 'pone.0167994.s009.xls', 'pone.0167994.s007.xls', 'tagsnp', 'enm013', 'pone.0167994.s003.xls', 'pone.0167994.s002.xls', 'pone.0167994.s008.xls', 'syn1446295', 'chr22', 'pone.0167994.s005.pdf'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC5161470: 0.05263157894736842\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC5161470: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9925742\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21126450'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'srr21079269', 'syn21126450', 'gse224398', 'srr21079265', 'srr21079270', 'srr21079266', 'srr21079268', 'srr21079267'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9925742: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9925742: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9235070\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn30243143'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse76312', 'gse75688', 'gse103322', 'e-mtab-6149', 'gse81383', 'syn213309', 'gse70630', 'gse81861', 'syn30243143'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9235070: 0.1111111111111111\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9235070: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11661334\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn51364943'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'35107', 'syn51364943', 'gcst009325', 'gcst007511', 'gcst90027158'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11661334: 0.2\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11661334: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11929800\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 5. Element IDs: {'syn3388564', 'syn11724057', 'syn4896408', 'syn18485175', 'syn17015098'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.5281/zenodo.14804357', 'syn3388564', 'syn11724057', 'syn4896408', 'gse291402', 'syn17015098', 'gcst90027158'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11929800: 0.5714285714285714\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11929800: 0.8\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7113080\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn17394041'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn17394041'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8847344\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn25813713'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn25813713', 'nct01057810', 'nct00110162', 'nct00601900', 'nct00427713', 'nct00973609', 'nct01584830', 'nct01377376', 'nct00982111', 'nct02130466', 'nct01673867', 'nct01229813'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8847344: 0.08333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8847344: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9280291\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn5612563'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'e-mtab-5273', 'gse66099', 'e-mexp-3589', 'gse20346', 'gse65682', 'gse60244', 'gse54514', 'gse25504', 'gse40586', 'gse32707', 'e-mtab-5274', 'gse21802', 'gse57065', 'e-mtab-1548', 'syn5612563', 'gse40012', 'gse13015', 'gse27131', 'gse95233'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9280291: 0.05263157894736842\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9280291: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10029021\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn50897018'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn50897018.3'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=syn50897018.3, gt_id=syn50897018\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10029021: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10029021: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11525589\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn51514106'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn51514106'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11525589: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11525589: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10836119\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 6. Element IDs: {'syn5592519', 'syn9705278', 'syn3157322', 'syn3205821', 'syn3157238', 'syn3157329'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn5592519', 'syn9705278', 'syn3205821.6', 'syn3157322', 'syn3157238', 'syn3157329'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=syn3205821.6, gt_id=syn3205821\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10836119: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10836119: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6646046\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn5609493'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'e-mtab-2323', 'syn5609493'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6646046: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6646046: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10387117\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21041850'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse161382', 'gse149563', '10.5281/zenodo.8111545', 'gsm5388412', 'gse122332', 'gsm4504966', 'dataset-427f1eee6dd44f50bae1ab13f0f3c6a9', 'gsm5388411', 'gsm4035472', 'gse135893', 'gsm5388413', 'egas00001004082', 'lmex0000004397', 'gse178362', 'gsm4504967', 'lmex0000004396', 'gse171524', 'gse122960', 'gse134174', 'syn21041850', 'gse135851', 'gse136831'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10387117: 0.045454545454545456\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10387117: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4444046\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn1734155', 'syn1729383', 'syn1710680'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn1734155', 'syn1729383', 'syn1710680', 'pb002581'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4444046: 0.75\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4444046: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10329279\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21478998'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn21478998'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11145390\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21788425'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn21788425', 'gse147807', '10.5281/zenodo.10971926'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11145390: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11145390: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11032436\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2759792'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'nds00143', 'phs000648', 'phs000424', 'syn2759792'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9984184\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn39456106', 'syn25576878'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse169185', 'e-geod-44770', 'gse147205', 'syn39456106', 'gse15222', 'gse95587', 'gouveiaroque_chipmera', 'syn25576878'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9984184: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9984184: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7347884\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1757053'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'e-mtab-4145', 'gse56031', 'syn1757053', 'egas00000000083'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7347884: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7347884: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10802452\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2759792'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: set()\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452: 0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8907344\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn26134616'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn26134616'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8907344: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8907344: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10238095\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2623706'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'pre_clinical_model', 'qupath-0.3.2', '6917', 'gse14333', 'pamr_classifier', 'cinmetrics', 'prjeb23709', 'gse108989', 'tcga-crc', 'gse13067', 'gse38832', 'swh:1:rev:c8b87c46eab6c4fc2a528c136ce62e51b9ad71af', 'gse78220', 'swh:1:rev:8de317a40f03e7272fd6031e12663595d595fa5f', 'syn2623706', 'ccle_broad_2019', 'gse17536', 'gse39582', 'human-intestine-cancer-1-standard', 'gse13294', 'nct-crc-he-7k', 'human-colorectal-cancer-11-mm-capture-area-ffpe-2-standard', 'gse33113', 'gse103479', 'gse104645', '10.1038/nature25501', 'gse37892', 'cccrc_classifier', 'nct-crc-he-100k', 'gse135222', 'swh:1:rev:5adf0c170407f30df33e15faf19c1091af9f752f'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10238095: 0.03225806451612903\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10238095: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9710693\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 4. Element IDs: {'syn3505720', 'syn3191087', 'syn23554292', 'syn3219045'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse72056', '10.7303/syn3219045', 'e-mtab-5060', 'syn23554292', 'e-mtab-5061', 'syn3191087', 'syn3505720', 'gse75748'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn3219045, gt_id=syn3219045\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9710693: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9710693: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9915613\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1688369'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse78284', 'gse102441', 'gse99542', 'gse84515', 'syn1688366', 'gse78167', 'gse25710', 'gse78913', 'gse154873', 'gse73663', 'gse96517', 'gse62944', 'gse57436', 'gse51403', 'gse56066'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9915613: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9915613: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11420198\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn4921369', 'syn26164834'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn4921369', 'gse126957', 'gse96107', 'gse59119', 'gse142881', 'gse249687', 'gse84176'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11420198: 0.14285714285714285\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11420198: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7865022\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20681023'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.6084/m9.figshare.13342055', 'syn20681023'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7865022: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7865022: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8131595\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn18485175', 'syn4587609'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse65776', 'gse84245', 'phs001373.v2.p2', 'encsr198zyj', 'srx3381923', 'gse59630', 'gse96107', 'gse81912', 'gse150130', 'syn18485175', 'encsr822cea', '10.1371/journal.pone.0143563', 'gse60192', 'gse113673', 'e-mtab-9045', 'encsr677hxc', '10.1371/journal.pgen.1000650', 'syn4587609', 'gse59318'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8131595: 0.10526315789473684\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8131595: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10828232\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn13363290'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse214628', 'syn13363290-v49'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=syn13363290-v49, gt_id=syn13363290\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10828232: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10828232: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11852220\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn51365301'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'7lwh', '8caa', '10.3390/', '2lll', 'syn51365301'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11852220: 0.2\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11852220: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9314356\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4907804'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'adni', 'syn4907804'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9314356: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9314356: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11439699\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 10. Element IDs: {'syn3157325', 'syn10235592', 'syn26720676', 'syn3157275', 'syn4896408', 'syn2580853', 'syn17008935', 'syn20068543', 'syn22264775', 'syn9702085'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn10235592', '10.7303/syn3157325', 'syn338856', 'syn26720676', 'syn3157275', 'syn4896408', 'syn2580853', 'syn17008935', 'syn20068543', 'syn22264775', 'syn9702085'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn3157325, gt_id=syn3157325\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11439699: 0.9090909090909091\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11439699: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6323985\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn6131484'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn6131484'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10680627\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn52234396', 'syn52264240'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'encsr998ydi', 'gse225157', 'epcrisprbenchmark_ensemble_data_grch38.tsv.gz', 'syn52234396', 'syn52264240', 'encsr443vtk'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10680627: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10680627: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc5192600\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn2787209', 'syn3049712'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn2787209', 'syn3049712'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC5192600: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC5192600: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7752919\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2623706'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse2109', 'gse38832', 'gse14333', 'gse39582', 'gse33113', 'tcga-read', 'gse44076', 'gse18088', 'gse13294', 'gse148394', 'e-mtab-8152', 'syn2623706', 'tcga-coad'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7752919: 0.07692307692307693\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7752919: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8859891\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4224222'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn4224222'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9853518\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn3193805'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'3757476', 'syn3193805'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9853518: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9853518: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11852190\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn3193805'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn3193805'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11852190: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11852190: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10652415\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn26254632', 'syn3157322', 'syn3388564'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn3157322', 'syn3388564', 'syn26254632'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10652415: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10652415: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7842189\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2580853'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse57181', 'gse1432', 'syn18485175', 'gse75431', 'gse61500', 'syn7818788', 'ng00040', 'phs002197.v1.p1', 'gse13162', 'syn9884351', 'gse65067', 'gse121654', 'gse146866', 'gse55627', 'gse103334', 'syn3163039', 'gse138852', 'gse98401', 'syn3157182', 'gse61499', 'gse90696'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7842189: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7842189: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc3788619\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn1729383', 'syn2237931'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn1729383', 'syn2237931', 'srep02651-s2.zip'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC3788619: 0.6666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC3788619: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6750256\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4587609'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'nihms1534152-supplement-6.xlsx', 'nihms1534152-supplement-4.xlsx', 'nihms1534152-supplement-3.xlsx', 'nihms1534152-supplement-2.pdf', 'nihms1534152-supplement-5.xlsx', 'nihms1534152-supplement-1.docx', 'nihms1534152-supplement-7.xlsx', 'syn4587609'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6750256: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6750256: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11370573\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn25569736'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn25569736'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11370573: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11370573: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4344205\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1688369'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn1688369'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4344205: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4344205: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9630780\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 8. Element IDs: {'syn34542574', 'syn25876213', 'syn25886762', 'syn34539904', 'syn34542449', 'syn20843407', 'syn25792906', 'syn34542450'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn34542574', 'syn25876213', 'syn25886762', 'syn34539904', 'ijo1366', 'syn20843407', 'syn34542449', 'syn25792906', 'syn34542450'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9630780: 0.8888888888888888\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9630780: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10813059\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn52138250', 'syn31952297', 'syn31947506'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn52138250', '10.3390/', 'p12023', 'syn31952297', 'syn31947506'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10813059: 0.6\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10813059: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6089972\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn7343873', 'syn2812961', 'syn7214402'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'prjna258216', 'gse68379', 'gse74877', '10.5281/zenodo.1304766', 'gse35069', 'syn7214402', 'prjeb5468', 'syn7343873', 'gse38266', 'syn2812961', 'prjeb11844', '10.5281/zenodo.1298968', 'gse38270'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6089972: 0.23076923076923078\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6089972: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8130821\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 4. Element IDs: {'syn23191787', 'syn9884314', 'syn17015098', 'syn20803583'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn23191787', 'syn9884314', 'syn17015098', 'syn20803583'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8130821: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8130821: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11224609\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn10901595'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.5281/zenodo.10795678', 'syn10901595', '10.5281/zenodo.11240547'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11224609: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11224609: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7577712\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20821165'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'pxd009589', 'syn20821165', 'gse33000'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7577712: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7577712: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7257173\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21541022'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn21541022', 'pxd013352', '10.1101/806752', 'syn18485233', 'pxd013350', 'pxd013347', 'pxd013342', 'pxd013349', 'pxd013351', 'syn18485244'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7257173: 0.1\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7257173: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9780309\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn26560886'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn26560886'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9780309: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9780309: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7658217\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21041850'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'c4077b3c-5c98-4d26-a614-246d12c2e5d7', 'gse122960', '10.1101/742320', 'gse132355', 'syn21041850'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7658217: 0.2\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7658217: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6289083\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn11967914'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn11967914', 'egas00001003068'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6289083: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6289083: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8807579\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20808171'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: set()\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8807579: 0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8807579: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6700281\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2759792'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'47eb8891-86da-11e7-a10d-0ac135e8bacf', 'doi:10.5061/dryad.5d4s2r8', 'syn2759792', 'phs000424.v7.p1'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6700281: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6700281: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7029360\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn8717496'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn8717496'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8219045\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn3163039'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse97942', 'e-mtab-5061', 'gse64810', 'syn3163039'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8219045: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8219045: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10543041\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn50996324'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn50996324', '10.13020/'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10543041: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10543041: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6546784\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4921369'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn4921369', '10.1038/nature19847', 'gse77565', 'phs001190.v1.p1'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6546784: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6546784: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10696419\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn22255433'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn22255433'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10696419: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10696419: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc5387919\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 10. Element IDs: {'syn4984604', 'syn6035832', 'syn6022474', 'syn6022465', 'syn5493036', 'syn5492805', 'syn5522790', 'syn5556216', 'syn5522788', 'syn5004874'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn4984604', 'cutaneousnf', 'syn6035832', 'syn6022474', 'syn6022465', 'syn5493036', 'syn5492805', 'syn5522790', 'syn5556216', 'syn5522788', 'syn5004874'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC5387919: 0.9090909090909091\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC5387919: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8094983\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn23532550'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'epi_isl_461052', 'epi_isl_461054', 'epi_isl_461001', 'epi_isl_461056', 'epi_isl_461003', 'epi_isl_461002', 'syn23532550', 'epi_isl_461051', 'epi_isl_461055', 'epi_isl_461057', 'epi_isl_461059', 'epi_isl_461060', 'epi_isl_461058', 'epi_isl_461053', 'epi_isl_415464'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8094983: 0.06666666666666667\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8094983: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7962955\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20640275'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse57763', 'gse33584', 'nc_045512.2', 'syn20640275', 'gse137834', 'mn996532.1', 'gse102470', 'e-mtab-9905'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7962955: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7962955: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11601401\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn51150434', 'syn62110225', 'syn20933797'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn51150434', '10.58085/hr6s-2991', 'syn20933797', '10.7303/syn62110225'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn62110225, gt_id=syn62110225\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11601401: 0.75\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11601401: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9712627\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn12514624'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'supplemental_data_s1', 'gse174367', 'supplemental_table_s4', 'gse167494', 'supplemental_table_s2', 'syn12514624', 'gse148822', 'supplemental_table_s5', 'supplemental_table_s3', 'gse178265', 'gse160936', 'gse183068', 'supplemental_table_s6', 'supplemental_table_s7', 'supplemental_table_s8', 'gse174332', 'gse137444', 'supplemental_table_s1', 'supplemental_data_s2', 'cg000376'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9712627: 0.05\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9712627: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4697768\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2813581'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.3322/caac.21329', 'syn2813581', '10.4137/cmo.s38279', '10.1038/455001a', '10.1093/nar/gku1075', '10.1038/527s108a', '10.1038/nature07634'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4697768: 0.14285714285714285\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4697768: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9094742\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21125841'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse116470', 'gse97930', 'pxd020007', 'syn21125841', 'gse113973', 'gse129788', 'gse118257', 'scp90', 'srp135960', 'gse60361', 'gse130119', 'gse75330', 'gse140511', 'pxd029727'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9094742: 0.07142857142857142\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9094742: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8565335\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn6131484'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn6131484', 'encsr000ekp', 'encsr483rkn', 'gse115046', 'hg38-blacklist.v2.bed.gz', 'encsr220asc', 'encsr000e'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8565335: 0.14285714285714285\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8565335: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10985232\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn10003873'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn10003873', 'srp186687', 'srp158590', 'srp291312', '10.5281/zenodo.10677267'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10985232: 0.2\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10985232: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8784774\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20968331'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse186341', '4257', 'syn20968331', 'gse92742', 'gse70138'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8784774: 0.2\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8784774: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8082263\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn11801497'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'s1535610818301119-mmc2.xlsx', 'pmd', 's0092867418303593-mmc1.xlsx', 'tcga_ancestry_assignments', '4f277128-f793-4354-a13d-30cc7fe9f6b5', 'mature.fa', '1b5f413e-a8d1-4d10-92eb-7c4ae739ed81', 'syn11801497', 'mirna_information.xlsx', '7d4c0344-f018-4ab0-949a-09815f483480', 'infiniumannotation', 'hsa.gff3.txt', '10.1016/', '1a7d7be8-675d-4e60-a105-19d4121bdebf', 'd773a53e-b78d-4a20-848c-866616352328', 'cytoband.txt.gz', 's0092867417310565-mmc1.xlsx'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8082263: 0.058823529411764705\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8082263: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11659981\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21041850'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse162170', 'wouters_human_melanoma', 'gse134432', 'v.4.4.210', 'e-mtab-6149', '10.5524/102624', 'tcga-luad', '10.5281/zenodo.12092871', 'e-mtab-6653', 'scp498', 'v14', 'syn21041850', 'gse115469'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11659981: 0.07692307692307693\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11659981: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9356164\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn12169715', 'syn12169709'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'3586c0da-64d0-4b74-a449-5ff4d9136611', '10.1016/', '1a7d7be8-675d-4e60-a105-19d4121bdebf', 'syn12169715', 'syn12169709'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9356164: 0.4\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9356164: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11271596\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn53009474'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.7303/syn53009474'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn53009474, gt_id=syn53009474\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11271596: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11271596: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7520714\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn18344812', 'syn17009517'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.6084/m9.figshare.8159525.v1', 'syn17009517', '10.7303/syn18344812'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn18344812, gt_id=syn18344812\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7520714: 0.6666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7520714: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10769298\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn11724057'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn11724057', '10.1101/2023.12.20.572491'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10769298: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10769298: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8226229\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2701943'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'14540610', 'syn2701943', 'gse30784'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8226229: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8226229: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11716409\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4231880'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'paquid', 'ronanpickell/1980s-album-covers', '10.13026/c2f305', 'gse16476', '7036235', 'breast-cancer-screening-dbt', 'mimiciii/1.4', '351', 'syn4231880', 'medquad', 'mimiciv/2.0', '10.13026/c28g6p', 'gse79209', '10.24432/c5bs32', 'quadeer15sh/lfw-facial-recognition', 'gse89413', 'e-mtab-8248', 'panfordofori/fitbitdata', 'nantonio/a-hotels-customers-dataset'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11716409: 0.05263157894736842\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11716409: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8055881\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn7824274'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'s050', 's039', 's053', 'mc3', 'syn7824274', 's037', 's038'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8055881: 0.14285714285714285\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8055881: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8102856\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn7214402'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'mc3', 'syn7214402', 'cnp0001438'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8102856: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8102856: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9085578\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn8612097'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.1038/sdata.2018.185', '10.3389/', 'phs000673.v2.p1', '10.1038/sdata.2016.89', 'syn8612097'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9085578: 0.2\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9085578: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9840277\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn11914606', 'syn23569889', 'syn14237651'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn11914606', 'syn23569889', 'syn14237651'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9840277: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9840277: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10547713\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn26707219', 'syn52137895'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.1038/', 'syn26707219', '10.1055/', 'syn52137895'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10547713: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10547713: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10571258\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn30821562'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.5281/zenodo.8322854', 'syn30821562', '10.5281/zenodo.8338532'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10571258: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10571258: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6829597\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn18796452'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn18796452', 'q32852', 'l34975', 'r25005'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6829597: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6829597: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11240079\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn52566791'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn52566791'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11240079: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11240079: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7805663\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn11737848'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn11737848'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7805663: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7805663: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10600818\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn23627957'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.1017/', 'syn23627957'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10600818: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10600818: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc5752539\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1741407'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse29623', 'hprd', 'mir2disease', 'mirnatarget', 'tcga', 'syn1741407', 'gse63805', 'gse63459', 'mirdb'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC5752539: 0.1111111111111111\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC5752539: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6602064\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn9884357', 'syn7818788'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse65057', 'phs000728.v1.p1', 'gse75510', 'pxd005703', 'gse45480', 'gse74193', 'phs000767.v1.p1', 'gse47572', 'gse77242', 'syn7818788', 'gse66504', 'gse53683', 'syn9884357', 'gse45479', 'gse45481', 'gse44772', 'gse90696', 'gse67140'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6602064: 0.1111111111111111\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6602064: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7032692\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1710250'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'tcga', 'egas00000000083', 'syn1710250'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7032692: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7032692: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10061294\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn1715755', 'syn2468297'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.7303/syn2468297', 'syn1715755'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn2468297, gt_id=syn2468297\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10061294: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10061294: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8006362\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn22024464'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.5281/zenodo.4321679', 'syn22024464', '10.15154/1504248'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8006362: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8006362: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7778917\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn18360482'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.1093/nar/gkaa853', 'syn18360482'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7778917: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7778917: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 89 - INFO - \n",
      "Performance evaluation completed for 100 source pages.\u001b[0m\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "{'average_precision': 0.4678899917342445,\n",
       " 'average_recall': 0.948,\n",
       " 'f1_score': 0.6265454445663144}"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "check_df = pd.read_csv(\"scripts/output/syn_data_results/result_gemini-2.5-flash.csv\")\n",
    "ret = evaluate_performance(check_df, ground_truth, dg, \"scripts/output/false_positives.txt\")\n",
    "ret"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "35d77225",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11792374\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 4. Element IDs: {'syn52293417', 'syn8456637', 'syn2580853', 'syn9702085'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn52293417', 'syn8456637', 'syn2580853', 'syn9702085'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11792374: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4318527\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn2455683', 'syn1720047'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn2455683', 'syn312572', 'syn1720047'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527: 0.6666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4318527: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4339277\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 4. Element IDs: {'syn1701558', 'syn1682511', 'syn1688309', 'syn1732077'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn300013', 'pmc4339277', 'nihms590622-supplement-supp_info_2.rar', 'syn1688309', 'syn1682511', '10.1038/ncomms4963', 'syn1732077', '24999802', 'nihms590622-supplement-supp_info_1.pdf', 'syn1701558'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277: 0.4\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4339277: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8452114\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn18485175'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'media-3.pdf', 'syn18485175', 'media-2.pdf', 'media-4.xlsx', 'media-1.pdf'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8452114: 0.2\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8452114: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9641692\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn3219045'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse119117', 'gse67835', '6k_pbmc', 'gse65133', 'syn3219045', 'gse107011', '10k_pbmc', 'sdy67', 'gse115449', 'gse87544', '10.5281/zenodo.7183923', 'gse84133', '8k_pbmc'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9641692: 0.07692307692307693\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9641692: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc5161470\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1446295'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'hapmap_phase3_chr22_consensus', 'syn1446295', 's4_table', 'hapmap_encode_non-redundant', 's2_table', 's5_table', '1000genomes_tagsnp', 's3_table', '1000genomes_release_20130502', 's1_table'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC5161470: 0.1\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC5161470: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9925742\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21126450'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'srr21079269', 'syn21126450', 'gse224398', 'srr21079265', 'srr21079270', 'srr21079266', 'srr21079268', 'srr21079267'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9925742: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9925742: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9235070\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn30243143'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse76312', 'gse75688', 'gse103322', 'e-mtab-6149', 'tcga', 'gse81383', 'syn213309', 'gse70630', 'gse81861', '10.1242/bio.059256', 'syn30243143'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9235070: 0.09090909090909091\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9235070: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11661334\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn51364943'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn51364943', '35107', 'glgc_lipids2021', 'gcst009325', 'cowas', 'mykmal/cowas', 'gcst007511', 'hippie_v2.3', 'gcst90027158'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11661334: 0.1111111111111111\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11661334: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11929800\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 5. Element IDs: {'syn3388564', 'syn11724057', 'syn4896408', 'syn18485175', 'syn17015098'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'zhao-research/5hmc', '10.5281/zenodo.14804357', 'jansen_summary_statistics', 'syn3388564', 'syn11724057', 'syn4896408', 'gse291402', '7276971', 'syn17015098', 'e073_15_coremarks_hg38lift_stateno.bed', 'gcst90027158'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11929800: 0.36363636363636365\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11929800: 0.8\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7113080\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn17394041'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'366856', 'syn17394041', 'pmc7113080', '783134v1', '32191543', '3572974'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080: 0.16666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7113080: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8847344\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn25813713'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.7303/syn25813713', 'moesm5', 'syn25813713', 'moesm4', 'moesm3', 'moesm6', 'moesm2', 'moesm1', 'moesm7'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8847344: 0.1111111111111111\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8847344: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9280291\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn5612563'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'e-mtab-5273', 'gse66099', 'e-mexp-3589', 'gse20346', 'gse65682', 'gse60244', 'gse54514', 'gse25504', 'gse40586', 'gse32707', 'e-mtab-5274', 'gse21802', 'gse57065', 'e-mtab-1548', 'syn5612563', 'gse40012', 'gse13015', 'gse27131', 'gse95233'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9280291: 0.05263157894736842\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9280291: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10029021\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn50897018'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn50897018.3'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=syn50897018.3, gt_id=syn50897018\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10029021: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10029021: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11525589\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn51514106'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn51514106'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11525589: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11525589: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10836119\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 6. Element IDs: {'syn5592519', 'syn9705278', 'syn3157322', 'syn3205821', 'syn3157238', 'syn3157329'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn5592519', 'syn9705278', 'syn3205821.6', 'syn3157322', 'syn3157238', 'syn3157329'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=syn3205821.6, gt_id=syn3205821\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10836119: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10836119: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6646046\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn5609493'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'e-mtab-2323', 'syn5609493'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6646046: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6646046: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10387117\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21041850'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse161382', 'gse149563', 'gsm5388412', 'gse122332', 'gsm4504966', 'dataset-427f1eee6dd44f50bae1ab13f0f3c6a9', 'gsm5388411', 'gsm4035472', 'gse135893', 'gse136832', 'gsm5388413', 'egas00001004082', 'github.com/xu-lab/cellref', 'lmex0000004397', 'gse178362', 'gsm4504967', 'lmex0000004396', 'gse171524', 'gse122960', 'gse134174', 'syn21041850', 'gse135851', 'gse136831'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10387117: 0.043478260869565216\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10387117: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4444046\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn1734155', 'syn1729383', 'syn1710680'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn1729383', '10.1038/ng.3168', 'pb002581', 'pmc4444046', 'syn1734155', 'nihms644198-supplement-3.xlsx', 'syn1710680', 'nihms644198-supplement-2.xlsx', 'nihms644198-supplement-1.pdf'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4444046: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4444046: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10329279\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21478998'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn21478998'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10329279: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11145390\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21788425'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn21788425', '10.5281/zenodo.10971926', 'skannan4/cm-entropy-score', 'gse147807', 'elainezhlchen/entropy_star_protocol'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11145390: 0.2\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11145390: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11032436\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2759792'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'nds00143', 'nd50034', 'syn2759792', 'crl-3216'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11032436: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9984184\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn39456106', 'syn25576878'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse169185', 'gse147205', 'syn39456106', 'gouveiaroque_chipmera', 'syn25576878'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9984184: 0.4\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9984184: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7347884\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1757053'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'e-mtab-4145', 'gse56031', 'syn1757053', 'egas00000000083'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7347884: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7347884: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10802452\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2759792'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'nihpp2024.01.09.574815v2-supplement-5.pdf', 'syn2759792', 'media-3.xlsx', 'media-1.xlsx', 'all_sumstats', 'media-2.xlsx', 'media-4.xlsx'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452: 0.14285714285714285\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10802452: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8907344\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn26134616'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.7303/syn26134616', 'syn26134616'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8907344: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8907344: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10238095\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2623706'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'pre_clinical_model', 'gse14333', 'tcga', 'pamr_classifier', 'prjeb23709', 'gse108989', 'gdsc', 'gse13067', 'gse38832', 'gse78220', 'syn2623706', 'gse17536', 'ccle_broad_2019', 'gse39582', 'human-intestine-cancer-1-standard', 'gse13294', 'gse35144', 'human-colorectal-cancer-11-mm-capture-area-ffpe-2-standard', 'gse33113', 'gse103479', 'gse104645', 'gse37892', 'cccrc_classifier', 'gse135222', 'imvigor210'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10238095: 0.04\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10238095: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9710693\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 4. Element IDs: {'syn3505720', 'syn3191087', 'syn23554292', 'syn3219045'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse72056', 'e-mtab-5060', 'syn23554292', 'e-mtab-5061', 'syn3191087', 'syn3505720', 'gse75748', 'syn3219045'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9710693: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9710693: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9915613\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1688369'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse78284', 'gse102441', 'gse99542', 'gse84515', 'gse78167', 'gse25710', 'gse78913', 'syn1688369', 'gse154873', 'gse73663', 'gse96517', 'gse62944', 'gse57436', 'gse51403', 'gse56066'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9915613: 0.06666666666666667\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9915613: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11420198\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn4921369', 'syn26164834'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn4921369', 'pnas.2402518121.sd02.xlsx', 'gse126957', 'pnas.2402518121.sd12.xlsx', 'syn26164834', 'gse84176', 'pnas.2402518121.sd13.xlsx', 'pnas.2402518121.sapp.pdf', 'gse96107', 'pnas.2402518121.sd04.xlsx', 'pnas.2402518121.sd08.xlsx', 'pnas.2402518121.sd10.xlsx', 'pnas.2402518121.sd01.xlsx', 'pnas.2402518121.sd06.xlsx', 'pnas.2402518121.sd07.xlsx', 'gse59119', 'gse249687', 'pnas.2402518121.sd05.xlsx', 'pnas.2402518121.sd03.xlsx', 'rad21-2024', 'pnas.2402518121.sd09.xlsx', 'gse142881', 'pnas.2402518121.sd11.xlsx'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11420198: 0.08695652173913043\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11420198: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7865022\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20681023'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.6084/m9.figshare.13342055', 'syn20681023'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7865022: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7865022: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8131595\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn18485175', 'syn4587609'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse60192', 'gse113673', 'gse96107', 'gse65776', 'e-mtab-9045', 'gse84245', 'gse81912', 'gse59318', 'phs001373.v2.p2', 'gse150130', 'syn18485175', 'encsr198zyj', 'encsr677hxc', 'encsr822cea', 'srx3381923', 'gse59630', 'syn4587609'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8131595: 0.11764705882352941\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8131595: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10828232\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn13363290'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse214628', 'nm_021272.3', 'syn13363290-v49'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=syn13363290-v49, gt_id=syn13363290\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10828232: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10828232: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11852220\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn51365301'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'7lwh', '8caa', '2lll', 'syn51365301', 'bioengineering-12-00126-s001'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11852220: 0.2\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11852220: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9314356\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4907804'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn4907804'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9314356: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9314356: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11439699\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 10. Element IDs: {'syn3157325', 'syn10235592', 'syn26720676', 'syn3157275', 'syn4896408', 'syn2580853', 'syn17008935', 'syn20068543', 'syn22264775', 'syn9702085'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn3157325', 'syn10235592', 'syn338856', 'syn26720676', 'syn3157275', 'syn4896408', 'syn2580853', 'syn17008935', 'syn20068543', 'syn22264775', 'syn9702085'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11439699: 0.9090909090909091\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11439699: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6323985\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn6131484'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn6131484', 'pmc6323985', '10.1093/nar/gky1128'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6323985: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10680627\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn52234396', 'syn52264240'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'encsr998ydi', 'gse225157', 'syn52234396', 'syn52264240', 'encsr443vtk', 'epcrispr'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10680627: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10680627: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc5192600\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn2787209', 'syn3049712'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn2787209', 'syn3049712'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC5192600: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC5192600: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7752919\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2623706'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse2109', 'gse38832', 'gse14333', 'gse39582', 'gse33113', 'tcga-read', 'gse44076', 'gse18088', 'gse13294', 'gse148394', 'e-mtab-8152', 'syn2623706', 'tcga-coad'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7752919: 0.07692307692307693\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7752919: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8859891\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4224222'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'pmc8859891', 'syn4224222', '10.1186/s13058-022-01509-z'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8859891: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9853518\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn3193805'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn3193805', 'zenodo:3757476'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9853518: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9853518: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11852190\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn3193805'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'brats', 'syn3193805'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11852190: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11852190: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10652415\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn26254632', 'syn3157322', 'syn3388564'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn26254632', 'syn3157322', 'syn3388564'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10652415: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10652415: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7842189\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2580853'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse57181', 'gse1432', 'syn18485175', 'gse75431', 'gse61500', 'syn7818788', 'ng00040', 'phs002197.v1.p1', 'gse13162', 'syn9884351', 'gse65067', 'gse121654', 'gse146866', 'gse98969', 'gse55627', 'gse103334', 'syn2580853', 'syn3163039', 'gse138852', 'gse98401', 'syn3157182', 'gse61499', 'gse90696'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7842189: 0.043478260869565216\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7842189: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc3788619\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn1729383', 'syn2237931'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn1729383', 'syn2237931', '10.1038/srep02651', 'srep02651-s2.zip', 'srep02651-s3.pdf', 'srep02651-s1.pdf'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC3788619: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC3788619: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6750256\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4587609'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'nihms1534152-supplement-6', 'nihms1534152-supplement-5', 'nihms1534152-supplement-4', 'nihms1534152-supplement-7', 'nihms1534152-supplement-3', 'nihms1534152-supplement-1', 'nihms1534152-supplement-2', 'syn4587609'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6750256: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6750256: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11370573\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn25569736'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn25569736'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11370573: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11370573: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4344205\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1688369'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn1688369'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4344205: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4344205: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9630780\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 8. Element IDs: {'syn34542574', 'syn25876213', 'syn25886762', 'syn34539904', 'syn34542449', 'syn20843407', 'syn25792906', 'syn34542450'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn34542574', 'syn25876213', 'syn25886762', 'carom', 'syn34539904', 'ijo1366', 'syn34542449', 'syn20843407', 'syn25792906', 'syn34542450'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9630780: 0.8\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9630780: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10813059\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn52138250', 'syn31952297', 'syn31947506'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn52138250', 'biomolecules-14-00066-s001.zip', 'syn31952297', 'syn31947506'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10813059: 0.75\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10813059: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6089972\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn7343873', 'syn2812961', 'syn7214402'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'prjna258216', 'gse68379', 'gse74877', '10.5281/zenodo.1304766', 'gse35069', 'syn7214402', 'prjeb5468', 'syn7343873', 'gse38266', 'syn2812961', 'prjeb11844', '10.5281/zenodo.1298968', 'gse38270'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6089972: 0.23076923076923078\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6089972: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8130821\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 4. Element IDs: {'syn23191787', 'syn9884314', 'syn17015098', 'syn20803583'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'nihms1656683-supplement-2.xlsx', 'syn9884314', 'syn20803583', 'nihms1656683-supplement-1.pdf', 'syn17015098', 'syn23191787'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8130821: 0.6666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8130821: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11224609\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn10901595'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.5281/zenodo.10795678', '10.5281/zenodo.8409105', '10.5281/zenodo.11240547', 'syn10901595'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11224609: 0.25\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11224609: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7577712\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20821165'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'pxd009589', 'syn20821165', 'gse33000'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7577712: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7577712: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7257173\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21541022'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn21541022'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7257173: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7257173: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9780309\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn26560886'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn26560886'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9780309: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9780309: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7658217\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21041850'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'c4077b3c-5c98-4d26-a614-246d12c2e5d7', 'kwl0kho7pbzf', 'gse122960', '10.1101/742320', 'gse132355', 'syn21041850'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7658217: 0.16666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7658217: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6289083\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn11967914'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn11967914', 'egas00001003068'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6289083: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6289083: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8807579\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20808171'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse164466', 'gse36980', 'gse158233', 'syn20808171', 'prjeb24858', 'gse151999', 'gse149149', 'gse156167'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8807579: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8807579: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6700281\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2759792'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'doi:10.5061/dryad.5d4s2r8', 'syn2759792', 'phs000424.v7.p1'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6700281: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6700281: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7029360\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn8717496'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn8717496'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7029360: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8219045\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn3163039'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse64810', 'e-mtab-5061', 'syn3163039', 'gse97942', 'lqab056_supplemental_files.zip'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8219045: 0.2\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8219045: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10543041\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn50996324'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn50996324', 'media-5.xlsx', 'media-3.xlsx', 'media-7.xlsx', 'media-10.xlsx', '10.1101/2023.09.18.23295431', 'media-2.xlsx', 'media-9.xlsx', '10.13020/3b1n-ff32', 'pmc10543041', 'media-4.xlsx', 'media-1.pdf', 'media-6.xlsx', 'media-8.xlsx'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10543041: 0.07142857142857142\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10543041: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6546784\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4921369'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn4921369', 'phs001190.v1.p1', 'moesm4', 'moesm8', 'moesm1', 'gse77565'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6546784: 0.16666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6546784: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10696419\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn22255433'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn22255433', 'ih_2300094_supplemental_1.pdf'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10696419: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10696419: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc5387919\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 10. Element IDs: {'syn4984604', 'syn6035832', 'syn6022474', 'syn6022465', 'syn5493036', 'syn5492805', 'syn5522790', 'syn5556216', 'syn5522788', 'syn5004874'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn4984604', 'cutaneousnf', 'syn6035832', 'syn6022474', 'syn6022465', 'syn5493036', 'syn5492805', 'syn5522790', 'syn5556216', 'syn5522788', 'syn5004874'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC5387919: 0.9090909090909091\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC5387919: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8094983\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn23532550'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'epi_isl_461052', 'epi_isl_461054', 'epi_isl_461001', 'epi_isl_461056', 'epi_isl_461003', 'epi_isl_461002', 'syn23532550', 'epi_isl_461051', 'epi_isl_461055', 'epi_isl_461057', 'epi_isl_461059', 'epi_isl_461060', 'epi_isl_461058', 'epi_isl_461053', 'epi_isl_415464'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8094983: 0.06666666666666667\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8094983: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7962955\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20640275'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse57763', 'gse33584', 'nc_045512.2', 'syn20640275', 'gse137834', 'mn996532.1', 'gse102470', 'e-mtab-9905'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7962955: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7962955: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11601401\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn51150434', 'syn62110225', 'syn20933797'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn51150434', '10.58085/hr6s-2991', 'syn62110225', 'syn20933797'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11601401: 0.75\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11601401: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9712627\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn12514624'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse174367', 'gse160936', 'gse174332', 'gse167494', 'fresh-frozen-lymph-node-with-b-cell-lymphoma-14-k-sorted-nuclei-1-standard-2-0-0', 'gse183068', 'supp_gr.276409.121_supplemental_data_s2.zip', 'gse137444', 'microglia-loss-of-y', 'supp_gr.276409.121_supplemental_data_s1.zip', 'gse178265', 'syn12514624', 'gse148822'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9712627: 0.07692307692307693\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9712627: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc4697768\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2813581'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.4137/cmo.s38279', 'syn2813581'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC4697768: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC4697768: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9094742\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21125841'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse116470', 'gse97930', 'gse775330', 'human-archived-brain', 'pxd020007', 'syn21125841', 'gse113973', 'gse129788', 'gse118257', 'srp135960', 'gse60361', 'gse138852', 'gse130119', 'gse75330', 'gse140511', 'pxd029727'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9094742: 0.0625\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9094742: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8565335\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn6131484'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'encsr000ekp', 'encsr483rkn', 'gse115046', 'encsr220asc', 'encsr000e', 'gkab676_supplemental_files.zip'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8565335: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8565335: 0.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10985232\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn10003873'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn10003873', 'srp186687', 'suhrig/arriba', 'star-fusion', 'ndaniel/fusioncatcher', 'nm_001306068.3', 'fusioninspector/fusioninspector', 'srp158590', 'shenlab-genomics/anchored-fusion', 'srp291312', 'xidslab/scfusion', 'nm_133330.3', '10.5281/zenodo.10677267'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10985232: 0.07692307692307693\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10985232: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8784774\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn20968331'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse186341', 'syn20968331', 'proteomicsdb_project_4257', 'gse92742', 'gse70138'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8784774: 0.2\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8784774: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8082263\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn11801497'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'mi0000466', '4f277128-f793-4354-a13d-30cc7fe9f6b5', '1b5f413e-a8d1-4d10-92eb-7c4ae739ed81', 'syn11801497', '10.1016/j.ccell.2020.04.012', '7d4c0344-f018-4ab0-949a-09815f483480', 'ccg-aim-2020', 'mmc2', '10.1016/j.xpro.2021.100483', '1a7d7be8-675d-4e60-a105-19d4121bdebf', 'd773a53e-b78d-4a20-848c-866616352328', 'mmc1'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8082263: 0.08333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8082263: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11659981\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn21041850'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse162170', 'scgraph2vec', 'trrust_v2', 'pathwaycommons_v14', 'gse134432', '10x_pbmc_3k', 'e-mtab-6149', '10.5524/102624', 'tcga-luad', '10.5281/zenodo.12092871', 'biogrid_v4.4.210', 'e-mtab-6653', 'scp498', 'syn21041850', 'gse115469'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11659981: 0.06666666666666667\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11659981: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9356164\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn12169715', 'syn12169709'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'1a7d7be8-675d-4e60-a105-19d4121bdebf', '10.5281/zenodo.6658317', 'ccg-aim-2020', 'legacy-archive', 'pancanqtl', '1-s2.0-s1535610820302117-mmc2.xlsx', 'tcga_pancancer_genotyping_imputation', '3586c0da-64d0-4b74-a449-5ff4d9136611', 'genomewidesnp_6', 'sayaman_tables1_mmc2.xlsx', '1-s2.0-s1535610820302117-mmc5.xlsx', 'syn12169715', 'syn12169709'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9356164: 0.15384615384615385\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9356164: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11271596\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn53009474'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn53009474', '10.7303/syn53009474'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11271596: 0.5\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11271596: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7520714\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn18344812', 'syn17009517'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.6084/m9.figshare.8159525.v1', 'syn17009517', '10.7303/syn18344812'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn18344812, gt_id=syn18344812\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7520714: 0.6666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7520714: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10769298\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn11724057'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'pa01-pa22', 'pmc10769298', 'media-1.xlsx', 'syn11724057', '10.1101/2023.12.20.572491', 'nihpp2023.12.20.572491v1-supplement-4.pdf', 'nihpp2023.12.20.572491v1-supplement-2.pdf', 'nihpp2023.12.20.572491v1-supplement-3.pdf'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10769298: 0.125\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10769298: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8226229\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn2701943'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn2701943', 'gse30784', 'njhncc-rnaseq-stemsig [14540610]'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8226229: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8226229: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11716409\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn4231880'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gpl18573', 'syn4231880', 'medquad', 'mimic-iii', '10.13026/c2f305', 'gse16476', 'gse89413', 'e-mtab-8248', 'gse79209', '10.13026/c28g6p', '10.24432/c5bs32', 'breast-cancer-screening-dbt'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11716409: 0.08333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11716409: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8055881\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn7824274'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'s050', 's039', 'rcsb_pdb_structures_2017-05-22', 's053', 'ptmcosmos', 'phosphositeplus_snapshot_2018-02-14', 'cptac_data_portal', 'mc3', 'syn7824274', 'depmap', 's037', 's038', 'uniprotkb_v2018.01'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8055881: 0.07692307692307693\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8055881: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8102856\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn7214402'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'mc3', 'syn7214402', 'cnp0001438'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8102856: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8102856: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9085578\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn8612097'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn8612097'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9085578: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9085578: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc9840277\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 3. Element IDs: {'syn11914606', 'syn23569889', 'syn14237651'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn11914606', 'syn23569889', 'syn14237651'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC9840277: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC9840277: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10547713\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn26707219', 'syn52137895'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn26707219', '10.7303/syn26707219', 'syn52137895'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10547713: 0.6666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10547713: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10571258\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn30821562'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.5281/zenodo.8322854', 'syn30821562', 'github.com/cogdishion/se2', '10.5281/zenodo.8338532', 'github.com/davidrconnell/igraph-se2'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10571258: 0.2\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10571258: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6829597\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn18796452'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'jem_20190430_datas1.xlsx', 'syn18796452', 'jem_20190430_sm.pdf'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6829597: 0.3333333333333333\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6829597: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc11240079\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn52566791'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn52566791'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC11240079: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC11240079: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7805663\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn11737848'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn11737848'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7805663: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7805663: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10600818\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn23627957'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'s0033291723000235a', 'syn23627957', '10.1017/s0033291723000235', 's0033291723000235sup001', 'focus.db'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10600818: 0.2\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10600818: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc5752539\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1741407'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'gse29623', 'oncotarget-08-109522-s001.pdf', 'syn1741407', '10.18632/oncotarget.22724', 'mirnatar', 'gse63805', 'oncotarget-08-109522-s002.xls', 'gse63459', 'pmc5752539'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC5752539: 0.1111111111111111\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC5752539: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc6602064\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn9884357', 'syn7818788'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn9884357', 'gse90696', 'syn7818788'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC6602064: 0.6666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC6602064: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7032692\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn1710250'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'syn1710250', 'tcga', 'metabric', 'egas00000000083', 'egac00001000484'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7032692: 0.2\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7032692: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc10061294\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 2. Element IDs: {'syn1715755', 'syn2468297'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.7303/syn2468297', 'syn1715755'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 49 - INFO - Partial or alias match found: eval_id=10.7303/syn2468297, gt_id=syn2468297\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC10061294: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC10061294: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc8006362\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn22024464'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'10.5281/zenodo.4321679', 'bsmn.synapse.org', 'bsmn-aws-setup', 'syn22024464', 'bsmn-pipeline', '1000g_2504_high_coverage', '10.15154/1504248'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC8006362: 0.14285714285714285\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC8006362: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 10 - INFO - Evaluating pub_id: pmc7778917\u001b[0m\n",
      "\u001b[97m3801855323.py - line 17 - INFO - # of elements in gt_data: 1. Element IDs: {'syn18360482'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 28 - INFO - Evaluation datasets: {'pmc7778917', 'idg-kinase/kinase_expression', 'idg-kinase/darkkinasekb', '10.1093/nar/gkaa853', 'syn18360482', 'pmid:33079988'}\u001b[0m\n",
      "\u001b[97m3801855323.py - line 70 - INFO - Precision for https://pmc.ncbi.nlm.nih.gov/articles/PMC7778917: 0.16666666666666666\u001b[0m\n",
      "\u001b[97m3801855323.py - line 71 - INFO - Recall for https://pmc.ncbi.nlm.nih.gov/articles/PMC7778917: 1.0\u001b[0m\n",
      "\u001b[97m3801855323.py - line 89 - INFO - \n",
      "Performance evaluation completed for 100 source pages.\u001b[0m\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "{'average_precision': 0.41387459247016745,\n",
       " 'average_recall': 0.988,\n",
       " 'f1_score': 0.5833732911016106}"
      ]
     },
     "execution_count": 21,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "check_df = pd.read_csv(\"scripts/output/syn_data_results/result_gpt-5-mini.csv\")\n",
    "ret = evaluate_performance(check_df, ground_truth, dg, \"scripts/output/false_positives.txt\", \"scripts/output/false_negatives.txt\")\n",
    "ret"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "473650bb",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "2025_arm",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.12.11"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
