{
 "cells": [
  {
   "cell_type": "code",
   "id": "initial_id",
   "metadata": {
    "collapsed": true,
    "ExecuteTime": {
     "end_time": "2025-03-15T01:17:05.298236Z",
     "start_time": "2025-03-15T01:16:57.077030Z"
    }
   },
   "source": [
    "import numpy as np\n",
    "import pandas as pd\n",
    "import re\n",
    "import json\n",
    "import requests\n",
    "from xml.etree import ElementTree\n",
    "import time\n",
    "from dotenv import load_dotenv\n",
    "import ast  # To safely evaluate string representation of a list\n",
    "import bdikit as bdi\n",
    "import os\n",
    "from experiment_utils import *"
   ],
   "outputs": [
    {
     "data": {
      "text/html": [
       "<script type=\"esms-options\">{\"shimMode\": true}</script><style>*[data-root-id],\n",
       "*[data-root-id] > * {\n",
       "  box-sizing: border-box;\n",
       "  font-family: var(--jp-ui-font-family);\n",
       "  font-size: var(--jp-ui-font-size1);\n",
       "  color: var(--vscode-editor-foreground, var(--jp-ui-font-color1));\n",
       "}\n",
       "\n",
       "/* Override VSCode background color */\n",
       ".cell-output-ipywidget-background:has(\n",
       "    > .cell-output-ipywidget-background > .lm-Widget > *[data-root-id]\n",
       "  ),\n",
       ".cell-output-ipywidget-background:has(> .lm-Widget > *[data-root-id]) {\n",
       "  background-color: transparent !important;\n",
       "}\n",
       "</style>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "application/javascript": "(function(root) {\n  function now() {\n    return new Date();\n  }\n\n  const force = true;\n  const py_version = '3.6.3'.replace('rc', '-rc.').replace('.dev', '-dev.');\n  const reloading = false;\n  const Bokeh = root.Bokeh;\n\n  // Set a timeout for this load but only if we are not already initializing\n  if (typeof (root._bokeh_timeout) === \"undefined\" || (force || !root._bokeh_is_initializing)) {\n    root._bokeh_timeout = Date.now() + 5000;\n    root._bokeh_failed_load = false;\n  }\n\n  function run_callbacks() {\n    try {\n      root._bokeh_onload_callbacks.forEach(function(callback) {\n        if (callback != null)\n          callback();\n      });\n    } finally {\n      delete root._bokeh_onload_callbacks;\n    }\n    console.debug(\"Bokeh: all callbacks have finished\");\n  }\n\n  function load_libs(css_urls, js_urls, js_modules, js_exports, callback) {\n    if (css_urls == null) css_urls = [];\n    if (js_urls == null) js_urls = [];\n    if (js_modules == null) js_modules = [];\n    if (js_exports == null) js_exports = {};\n\n    root._bokeh_onload_callbacks.push(callback);\n\n    if (root._bokeh_is_loading > 0) {\n      // Don't load bokeh if it is still initializing\n      console.debug(\"Bokeh: BokehJS is being loaded, scheduling callback at\", now());\n      return null;\n    } else if (js_urls.length === 0 && js_modules.length === 0 && Object.keys(js_exports).length === 0) {\n      // There is nothing to load\n      run_callbacks();\n      return null;\n    }\n\n    function on_load() {\n      root._bokeh_is_loading--;\n      if (root._bokeh_is_loading === 0) {\n        console.debug(\"Bokeh: all BokehJS libraries/stylesheets loaded\");\n        run_callbacks()\n      }\n    }\n    window._bokeh_on_load = on_load\n\n    function on_error(e) {\n      const src_el = e.srcElement\n      console.error(\"failed to load \" + (src_el.href || src_el.src));\n    }\n\n    const skip = [];\n    if (window.requirejs) {\n      window.requirejs.config({'packages': {}, 'paths': {'tabulator': 'https://cdn.jsdelivr.net/npm/tabulator-tables@6.3.1/dist/js/tabulator.min', 'moment': 'https://cdn.jsdelivr.net/npm/luxon/build/global/luxon.min'}, 'shim': {}});\n      require([\"tabulator\"], function(Tabulator) {\n        window.Tabulator = Tabulator\n        on_load()\n      })\n      require([\"moment\"], function(moment) {\n        window.moment = moment\n        on_load()\n      })\n      root._bokeh_is_loading = css_urls.length + 2;\n    } else {\n      root._bokeh_is_loading = css_urls.length + js_urls.length + js_modules.length + Object.keys(js_exports).length;\n    }\n\n    const existing_stylesheets = []\n    const links = document.getElementsByTagName('link')\n    for (let i = 0; i < links.length; i++) {\n      const link = links[i]\n      if (link.href != null) {\n        existing_stylesheets.push(link.href)\n      }\n    }\n    for (let i = 0; i < css_urls.length; i++) {\n      const url = css_urls[i];\n      const escaped = encodeURI(url)\n      if (existing_stylesheets.indexOf(escaped) !== -1) {\n        on_load()\n        continue;\n      }\n      const element = document.createElement(\"link\");\n      element.onload = on_load;\n      element.onerror = on_error;\n      element.rel = \"stylesheet\";\n      element.type = \"text/css\";\n      element.href = url;\n      console.debug(\"Bokeh: injecting link tag for BokehJS stylesheet: \", url);\n      document.body.appendChild(element);\n    }    if (((window.Tabulator !== undefined) && (!(window.Tabulator instanceof HTMLElement))) || window.requirejs) {\n      var urls = ['https://cdn.holoviz.org/panel/1.6.1/dist/bundled/datatabulator/tabulator-tables@6.3.1/dist/js/tabulator.min.js'];\n      for (var i = 0; i < urls.length; i++) {\n        skip.push(encodeURI(urls[i]))\n      }\n    }    if (((window.moment !== undefined) && (!(window.moment instanceof HTMLElement))) || window.requirejs) {\n      var urls = ['https://cdn.holoviz.org/panel/1.6.1/dist/bundled/datatabulator/luxon/build/global/luxon.min.js'];\n      for (var i = 0; i < urls.length; i++) {\n        skip.push(encodeURI(urls[i]))\n      }\n    }    var existing_scripts = []\n    const scripts = document.getElementsByTagName('script')\n    for (let i = 0; i < scripts.length; i++) {\n      var script = scripts[i]\n      if (script.src != null) {\n        existing_scripts.push(script.src)\n      }\n    }\n    for (let i = 0; i < js_urls.length; i++) {\n      const url = js_urls[i];\n      const escaped = encodeURI(url)\n      if (skip.indexOf(escaped) !== -1 || existing_scripts.indexOf(escaped) !== -1) {\n        if (!window.requirejs) {\n          on_load();\n        }\n        continue;\n      }\n      const element = document.createElement('script');\n      element.onload = on_load;\n      element.onerror = on_error;\n      element.async = false;\n      element.src = url;\n      console.debug(\"Bokeh: injecting script tag for BokehJS library: \", url);\n      document.head.appendChild(element);\n    }\n    for (let i = 0; i < js_modules.length; i++) {\n      const url = js_modules[i];\n      const escaped = encodeURI(url)\n      if (skip.indexOf(escaped) !== -1 || existing_scripts.indexOf(escaped) !== -1) {\n        if (!window.requirejs) {\n          on_load();\n        }\n        continue;\n      }\n      var element = document.createElement('script');\n      element.onload = on_load;\n      element.onerror = on_error;\n      element.async = false;\n      element.src = url;\n      element.type = \"module\";\n      console.debug(\"Bokeh: injecting script tag for BokehJS library: \", url);\n      document.head.appendChild(element);\n    }\n    for (const name in js_exports) {\n      const url = js_exports[name];\n      const escaped = encodeURI(url)\n      if (skip.indexOf(escaped) >= 0 || root[name] != null) {\n        if (!window.requirejs) {\n          on_load();\n        }\n        continue;\n      }\n      var element = document.createElement('script');\n      element.onerror = on_error;\n      element.async = false;\n      element.type = \"module\";\n      console.debug(\"Bokeh: injecting script tag for BokehJS library: \", url);\n      element.textContent = `\n      import ${name} from \"${url}\"\n      window.${name} = ${name}\n      window._bokeh_on_load()\n      `\n      document.head.appendChild(element);\n    }\n    if (!js_urls.length && !js_modules.length) {\n      on_load()\n    }\n  };\n\n  function inject_raw_css(css) {\n    const element = document.createElement(\"style\");\n    element.appendChild(document.createTextNode(css));\n    document.body.appendChild(element);\n  }\n\n  const js_urls = [\"https://cdn.holoviz.org/panel/1.6.1/dist/bundled/reactiveesm/es-module-shims@^1.10.0/dist/es-module-shims.min.js\", \"https://cdn.holoviz.org/panel/1.6.1/dist/bundled/datatabulator/tabulator-tables@6.3.1/dist/js/tabulator.min.js\", \"https://cdn.holoviz.org/panel/1.6.1/dist/bundled/datatabulator/luxon/build/global/luxon.min.js\", \"https://cdn.bokeh.org/bokeh/release/bokeh-3.6.3.min.js\", \"https://cdn.bokeh.org/bokeh/release/bokeh-gl-3.6.3.min.js\", \"https://cdn.bokeh.org/bokeh/release/bokeh-widgets-3.6.3.min.js\", \"https://cdn.bokeh.org/bokeh/release/bokeh-tables-3.6.3.min.js\", \"https://cdn.holoviz.org/panel/1.6.1/dist/panel.min.js\"];\n  const js_modules = [];\n  const js_exports = {};\n  const css_urls = [\"https://cdn.holoviz.org/panel/1.6.1/dist/bundled/datatabulator/tabulator-tables@6.3.1/dist/css/tabulator_simple.min.css?v=1.6.1\"];\n  const inline_js = [    function(Bokeh) {\n      Bokeh.set_log_level(\"info\");\n    },\nfunction(Bokeh) {} // ensure no trailing comma for IE\n  ];\n\n  function run_inline_js() {\n    if ((root.Bokeh !== undefined) || (force === true)) {\n      for (let i = 0; i < inline_js.length; i++) {\n        try {\n          inline_js[i].call(root, root.Bokeh);\n        } catch(e) {\n          if (!reloading) {\n            throw e;\n          }\n        }\n      }\n      // Cache old bokeh versions\n      if (Bokeh != undefined && !reloading) {\n        var NewBokeh = root.Bokeh;\n        if (Bokeh.versions === undefined) {\n          Bokeh.versions = new Map();\n        }\n        if (NewBokeh.version !== Bokeh.version) {\n          Bokeh.versions.set(NewBokeh.version, NewBokeh)\n        }\n        root.Bokeh = Bokeh;\n      }\n    } else if (Date.now() < root._bokeh_timeout) {\n      setTimeout(run_inline_js, 100);\n    } else if (!root._bokeh_failed_load) {\n      console.log(\"Bokeh: BokehJS failed to load within specified timeout.\");\n      root._bokeh_failed_load = true;\n    }\n    root._bokeh_is_initializing = false\n  }\n\n  function load_or_wait() {\n    // Implement a backoff loop that tries to ensure we do not load multiple\n    // versions of Bokeh and its dependencies at the same time.\n    // In recent versions we use the root._bokeh_is_initializing flag\n    // to determine whether there is an ongoing attempt to initialize\n    // bokeh, however for backward compatibility we also try to ensure\n    // that we do not start loading a newer (Panel>=1.0 and Bokeh>3) version\n    // before older versions are fully initialized.\n    if (root._bokeh_is_initializing && Date.now() > root._bokeh_timeout) {\n      // If the timeout and bokeh was not successfully loaded we reset\n      // everything and try loading again\n      root._bokeh_timeout = Date.now() + 5000;\n      root._bokeh_is_initializing = false;\n      root._bokeh_onload_callbacks = undefined;\n      root._bokeh_is_loading = 0\n      console.log(\"Bokeh: BokehJS was loaded multiple times but one version failed to initialize.\");\n      load_or_wait();\n    } else if (root._bokeh_is_initializing || (typeof root._bokeh_is_initializing === \"undefined\" && root._bokeh_onload_callbacks !== undefined)) {\n      setTimeout(load_or_wait, 100);\n    } else {\n      root._bokeh_is_initializing = true\n      root._bokeh_onload_callbacks = []\n      const bokeh_loaded = root.Bokeh != null && (root.Bokeh.version === py_version || (root.Bokeh.versions !== undefined && root.Bokeh.versions.has(py_version)));\n      if (!reloading && !bokeh_loaded) {\n        if (root.Bokeh) {\n          root.Bokeh = undefined;\n        }\n        console.debug(\"Bokeh: BokehJS not loaded, scheduling load and callback at\", now());\n      }\n      load_libs(css_urls, js_urls, js_modules, js_exports, function() {\n        console.debug(\"Bokeh: BokehJS plotting callback run at\", now());\n        run_inline_js();\n      });\n    }\n  }\n  // Give older versions of the autoload script a head-start to ensure\n  // they initialize before we start loading newer version.\n  setTimeout(load_or_wait, 100)\n}(window));",
      "application/vnd.holoviews_load.v0+json": "(function(root) {\n  function now() {\n    return new Date();\n  }\n\n  const force = true;\n  const py_version = '3.6.3'.replace('rc', '-rc.').replace('.dev', '-dev.');\n  const reloading = false;\n  const Bokeh = root.Bokeh;\n\n  // Set a timeout for this load but only if we are not already initializing\n  if (typeof (root._bokeh_timeout) === \"undefined\" || (force || !root._bokeh_is_initializing)) {\n    root._bokeh_timeout = Date.now() + 5000;\n    root._bokeh_failed_load = false;\n  }\n\n  function run_callbacks() {\n    try {\n      root._bokeh_onload_callbacks.forEach(function(callback) {\n        if (callback != null)\n          callback();\n      });\n    } finally {\n      delete root._bokeh_onload_callbacks;\n    }\n    console.debug(\"Bokeh: all callbacks have finished\");\n  }\n\n  function load_libs(css_urls, js_urls, js_modules, js_exports, callback) {\n    if (css_urls == null) css_urls = [];\n    if (js_urls == null) js_urls = [];\n    if (js_modules == null) js_modules = [];\n    if (js_exports == null) js_exports = {};\n\n    root._bokeh_onload_callbacks.push(callback);\n\n    if (root._bokeh_is_loading > 0) {\n      // Don't load bokeh if it is still initializing\n      console.debug(\"Bokeh: BokehJS is being loaded, scheduling callback at\", now());\n      return null;\n    } else if (js_urls.length === 0 && js_modules.length === 0 && Object.keys(js_exports).length === 0) {\n      // There is nothing to load\n      run_callbacks();\n      return null;\n    }\n\n    function on_load() {\n      root._bokeh_is_loading--;\n      if (root._bokeh_is_loading === 0) {\n        console.debug(\"Bokeh: all BokehJS libraries/stylesheets loaded\");\n        run_callbacks()\n      }\n    }\n    window._bokeh_on_load = on_load\n\n    function on_error(e) {\n      const src_el = e.srcElement\n      console.error(\"failed to load \" + (src_el.href || src_el.src));\n    }\n\n    const skip = [];\n    if (window.requirejs) {\n      window.requirejs.config({'packages': {}, 'paths': {'tabulator': 'https://cdn.jsdelivr.net/npm/tabulator-tables@6.3.1/dist/js/tabulator.min', 'moment': 'https://cdn.jsdelivr.net/npm/luxon/build/global/luxon.min'}, 'shim': {}});\n      require([\"tabulator\"], function(Tabulator) {\n        window.Tabulator = Tabulator\n        on_load()\n      })\n      require([\"moment\"], function(moment) {\n        window.moment = moment\n        on_load()\n      })\n      root._bokeh_is_loading = css_urls.length + 2;\n    } else {\n      root._bokeh_is_loading = css_urls.length + js_urls.length + js_modules.length + Object.keys(js_exports).length;\n    }\n\n    const existing_stylesheets = []\n    const links = document.getElementsByTagName('link')\n    for (let i = 0; i < links.length; i++) {\n      const link = links[i]\n      if (link.href != null) {\n        existing_stylesheets.push(link.href)\n      }\n    }\n    for (let i = 0; i < css_urls.length; i++) {\n      const url = css_urls[i];\n      const escaped = encodeURI(url)\n      if (existing_stylesheets.indexOf(escaped) !== -1) {\n        on_load()\n        continue;\n      }\n      const element = document.createElement(\"link\");\n      element.onload = on_load;\n      element.onerror = on_error;\n      element.rel = \"stylesheet\";\n      element.type = \"text/css\";\n      element.href = url;\n      console.debug(\"Bokeh: injecting link tag for BokehJS stylesheet: \", url);\n      document.body.appendChild(element);\n    }    if (((window.Tabulator !== undefined) && (!(window.Tabulator instanceof HTMLElement))) || window.requirejs) {\n      var urls = ['https://cdn.holoviz.org/panel/1.6.1/dist/bundled/datatabulator/tabulator-tables@6.3.1/dist/js/tabulator.min.js'];\n      for (var i = 0; i < urls.length; i++) {\n        skip.push(encodeURI(urls[i]))\n      }\n    }    if (((window.moment !== undefined) && (!(window.moment instanceof HTMLElement))) || window.requirejs) {\n      var urls = ['https://cdn.holoviz.org/panel/1.6.1/dist/bundled/datatabulator/luxon/build/global/luxon.min.js'];\n      for (var i = 0; i < urls.length; i++) {\n        skip.push(encodeURI(urls[i]))\n      }\n    }    var existing_scripts = []\n    const scripts = document.getElementsByTagName('script')\n    for (let i = 0; i < scripts.length; i++) {\n      var script = scripts[i]\n      if (script.src != null) {\n        existing_scripts.push(script.src)\n      }\n    }\n    for (let i = 0; i < js_urls.length; i++) {\n      const url = js_urls[i];\n      const escaped = encodeURI(url)\n      if (skip.indexOf(escaped) !== -1 || existing_scripts.indexOf(escaped) !== -1) {\n        if (!window.requirejs) {\n          on_load();\n        }\n        continue;\n      }\n      const element = document.createElement('script');\n      element.onload = on_load;\n      element.onerror = on_error;\n      element.async = false;\n      element.src = url;\n      console.debug(\"Bokeh: injecting script tag for BokehJS library: \", url);\n      document.head.appendChild(element);\n    }\n    for (let i = 0; i < js_modules.length; i++) {\n      const url = js_modules[i];\n      const escaped = encodeURI(url)\n      if (skip.indexOf(escaped) !== -1 || existing_scripts.indexOf(escaped) !== -1) {\n        if (!window.requirejs) {\n          on_load();\n        }\n        continue;\n      }\n      var element = document.createElement('script');\n      element.onload = on_load;\n      element.onerror = on_error;\n      element.async = false;\n      element.src = url;\n      element.type = \"module\";\n      console.debug(\"Bokeh: injecting script tag for BokehJS library: \", url);\n      document.head.appendChild(element);\n    }\n    for (const name in js_exports) {\n      const url = js_exports[name];\n      const escaped = encodeURI(url)\n      if (skip.indexOf(escaped) >= 0 || root[name] != null) {\n        if (!window.requirejs) {\n          on_load();\n        }\n        continue;\n      }\n      var element = document.createElement('script');\n      element.onerror = on_error;\n      element.async = false;\n      element.type = \"module\";\n      console.debug(\"Bokeh: injecting script tag for BokehJS library: \", url);\n      element.textContent = `\n      import ${name} from \"${url}\"\n      window.${name} = ${name}\n      window._bokeh_on_load()\n      `\n      document.head.appendChild(element);\n    }\n    if (!js_urls.length && !js_modules.length) {\n      on_load()\n    }\n  };\n\n  function inject_raw_css(css) {\n    const element = document.createElement(\"style\");\n    element.appendChild(document.createTextNode(css));\n    document.body.appendChild(element);\n  }\n\n  const js_urls = [\"https://cdn.holoviz.org/panel/1.6.1/dist/bundled/reactiveesm/es-module-shims@^1.10.0/dist/es-module-shims.min.js\", \"https://cdn.holoviz.org/panel/1.6.1/dist/bundled/datatabulator/tabulator-tables@6.3.1/dist/js/tabulator.min.js\", \"https://cdn.holoviz.org/panel/1.6.1/dist/bundled/datatabulator/luxon/build/global/luxon.min.js\", \"https://cdn.bokeh.org/bokeh/release/bokeh-3.6.3.min.js\", \"https://cdn.bokeh.org/bokeh/release/bokeh-gl-3.6.3.min.js\", \"https://cdn.bokeh.org/bokeh/release/bokeh-widgets-3.6.3.min.js\", \"https://cdn.bokeh.org/bokeh/release/bokeh-tables-3.6.3.min.js\", \"https://cdn.holoviz.org/panel/1.6.1/dist/panel.min.js\"];\n  const js_modules = [];\n  const js_exports = {};\n  const css_urls = [\"https://cdn.holoviz.org/panel/1.6.1/dist/bundled/datatabulator/tabulator-tables@6.3.1/dist/css/tabulator_simple.min.css?v=1.6.1\"];\n  const inline_js = [    function(Bokeh) {\n      Bokeh.set_log_level(\"info\");\n    },\nfunction(Bokeh) {} // ensure no trailing comma for IE\n  ];\n\n  function run_inline_js() {\n    if ((root.Bokeh !== undefined) || (force === true)) {\n      for (let i = 0; i < inline_js.length; i++) {\n        try {\n          inline_js[i].call(root, root.Bokeh);\n        } catch(e) {\n          if (!reloading) {\n            throw e;\n          }\n        }\n      }\n      // Cache old bokeh versions\n      if (Bokeh != undefined && !reloading) {\n        var NewBokeh = root.Bokeh;\n        if (Bokeh.versions === undefined) {\n          Bokeh.versions = new Map();\n        }\n        if (NewBokeh.version !== Bokeh.version) {\n          Bokeh.versions.set(NewBokeh.version, NewBokeh)\n        }\n        root.Bokeh = Bokeh;\n      }\n    } else if (Date.now() < root._bokeh_timeout) {\n      setTimeout(run_inline_js, 100);\n    } else if (!root._bokeh_failed_load) {\n      console.log(\"Bokeh: BokehJS failed to load within specified timeout.\");\n      root._bokeh_failed_load = true;\n    }\n    root._bokeh_is_initializing = false\n  }\n\n  function load_or_wait() {\n    // Implement a backoff loop that tries to ensure we do not load multiple\n    // versions of Bokeh and its dependencies at the same time.\n    // In recent versions we use the root._bokeh_is_initializing flag\n    // to determine whether there is an ongoing attempt to initialize\n    // bokeh, however for backward compatibility we also try to ensure\n    // that we do not start loading a newer (Panel>=1.0 and Bokeh>3) version\n    // before older versions are fully initialized.\n    if (root._bokeh_is_initializing && Date.now() > root._bokeh_timeout) {\n      // If the timeout and bokeh was not successfully loaded we reset\n      // everything and try loading again\n      root._bokeh_timeout = Date.now() + 5000;\n      root._bokeh_is_initializing = false;\n      root._bokeh_onload_callbacks = undefined;\n      root._bokeh_is_loading = 0\n      console.log(\"Bokeh: BokehJS was loaded multiple times but one version failed to initialize.\");\n      load_or_wait();\n    } else if (root._bokeh_is_initializing || (typeof root._bokeh_is_initializing === \"undefined\" && root._bokeh_onload_callbacks !== undefined)) {\n      setTimeout(load_or_wait, 100);\n    } else {\n      root._bokeh_is_initializing = true\n      root._bokeh_onload_callbacks = []\n      const bokeh_loaded = root.Bokeh != null && (root.Bokeh.version === py_version || (root.Bokeh.versions !== undefined && root.Bokeh.versions.has(py_version)));\n      if (!reloading && !bokeh_loaded) {\n        if (root.Bokeh) {\n          root.Bokeh = undefined;\n        }\n        console.debug(\"Bokeh: BokehJS not loaded, scheduling load and callback at\", now());\n      }\n      load_libs(css_urls, js_urls, js_modules, js_exports, function() {\n        console.debug(\"Bokeh: BokehJS plotting callback run at\", now());\n        run_inline_js();\n      });\n    }\n  }\n  // Give older versions of the autoload script a head-start to ensure\n  // they initialize before we start loading newer version.\n  setTimeout(load_or_wait, 100)\n}(window));"
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "application/vnd.holoviews_load.v0+json": "\nif ((window.PyViz === undefined) || (window.PyViz instanceof HTMLElement)) {\n  window.PyViz = {comms: {}, comm_status:{}, kernels:{}, receivers: {}, plot_index: []}\n}\n\n\n    function JupyterCommManager() {\n    }\n\n    JupyterCommManager.prototype.register_target = function(plot_id, comm_id, msg_handler) {\n      if (window.comm_manager || ((window.Jupyter !== undefined) && (Jupyter.notebook.kernel != null))) {\n        var comm_manager = window.comm_manager || Jupyter.notebook.kernel.comm_manager;\n        comm_manager.register_target(comm_id, function(comm) {\n          comm.on_msg(msg_handler);\n        });\n      } else if ((plot_id in window.PyViz.kernels) && (window.PyViz.kernels[plot_id])) {\n        window.PyViz.kernels[plot_id].registerCommTarget(comm_id, function(comm) {\n          comm.onMsg = msg_handler;\n        });\n      } else if (typeof google != 'undefined' && google.colab.kernel != null) {\n        google.colab.kernel.comms.registerTarget(comm_id, (comm) => {\n          var messages = comm.messages[Symbol.asyncIterator]();\n          function processIteratorResult(result) {\n            var message = result.value;\n            console.log(message)\n            var content = {data: message.data, comm_id};\n            var buffers = []\n            for (var buffer of message.buffers || []) {\n              buffers.push(new DataView(buffer))\n            }\n            var metadata = message.metadata || {};\n            var msg = {content, buffers, metadata}\n            msg_handler(msg);\n            return messages.next().then(processIteratorResult);\n          }\n          return messages.next().then(processIteratorResult);\n        })\n      }\n    }\n\n    JupyterCommManager.prototype.get_client_comm = function(plot_id, comm_id, msg_handler) {\n      if (comm_id in window.PyViz.comms) {\n        return window.PyViz.comms[comm_id];\n      } else if (window.comm_manager || ((window.Jupyter !== undefined) && (Jupyter.notebook.kernel != null))) {\n        var comm_manager = window.comm_manager || Jupyter.notebook.kernel.comm_manager;\n        var comm = comm_manager.new_comm(comm_id, {}, {}, {}, comm_id);\n        if (msg_handler) {\n          comm.on_msg(msg_handler);\n        }\n      } else if ((plot_id in window.PyViz.kernels) && (window.PyViz.kernels[plot_id])) {\n        var comm = window.PyViz.kernels[plot_id].connectToComm(comm_id);\n        comm.open();\n        if (msg_handler) {\n          comm.onMsg = msg_handler;\n        }\n      } else if (typeof google != 'undefined' && google.colab.kernel != null) {\n        var comm_promise = google.colab.kernel.comms.open(comm_id)\n        comm_promise.then((comm) => {\n          window.PyViz.comms[comm_id] = comm;\n          if (msg_handler) {\n            var messages = comm.messages[Symbol.asyncIterator]();\n            function processIteratorResult(result) {\n              var message = result.value;\n              var content = {data: message.data};\n              var metadata = message.metadata || {comm_id};\n              var msg = {content, metadata}\n              msg_handler(msg);\n              return messages.next().then(processIteratorResult);\n            }\n            return messages.next().then(processIteratorResult);\n          }\n        })\n        var sendClosure = (data, metadata, buffers, disposeOnDone) => {\n          return comm_promise.then((comm) => {\n            comm.send(data, metadata, buffers, disposeOnDone);\n          });\n        };\n        var comm = {\n          send: sendClosure\n        };\n      }\n      window.PyViz.comms[comm_id] = comm;\n      return comm;\n    }\n    window.PyViz.comm_manager = new JupyterCommManager();\n    \n\n\nvar JS_MIME_TYPE = 'application/javascript';\nvar HTML_MIME_TYPE = 'text/html';\nvar EXEC_MIME_TYPE = 'application/vnd.holoviews_exec.v0+json';\nvar CLASS_NAME = 'output';\n\n/**\n * Render data to the DOM node\n */\nfunction render(props, node) {\n  var div = document.createElement(\"div\");\n  var script = document.createElement(\"script\");\n  node.appendChild(div);\n  node.appendChild(script);\n}\n\n/**\n * Handle when a new output is added\n */\nfunction handle_add_output(event, handle) {\n  var output_area = handle.output_area;\n  var output = handle.output;\n  if ((output.data == undefined) || (!output.data.hasOwnProperty(EXEC_MIME_TYPE))) {\n    return\n  }\n  var id = output.metadata[EXEC_MIME_TYPE][\"id\"];\n  var toinsert = output_area.element.find(\".\" + CLASS_NAME.split(' ')[0]);\n  if (id !== undefined) {\n    var nchildren = toinsert.length;\n    var html_node = toinsert[nchildren-1].children[0];\n    html_node.innerHTML = output.data[HTML_MIME_TYPE];\n    var scripts = [];\n    var nodelist = html_node.querySelectorAll(\"script\");\n    for (var i in nodelist) {\n      if (nodelist.hasOwnProperty(i)) {\n        scripts.push(nodelist[i])\n      }\n    }\n\n    scripts.forEach( function (oldScript) {\n      var newScript = document.createElement(\"script\");\n      var attrs = [];\n      var nodemap = oldScript.attributes;\n      for (var j in nodemap) {\n        if (nodemap.hasOwnProperty(j)) {\n          attrs.push(nodemap[j])\n        }\n      }\n      attrs.forEach(function(attr) { newScript.setAttribute(attr.name, attr.value) });\n      newScript.appendChild(document.createTextNode(oldScript.innerHTML));\n      oldScript.parentNode.replaceChild(newScript, oldScript);\n    });\n    if (JS_MIME_TYPE in output.data) {\n      toinsert[nchildren-1].children[1].textContent = output.data[JS_MIME_TYPE];\n    }\n    output_area._hv_plot_id = id;\n    if ((window.Bokeh !== undefined) && (id in Bokeh.index)) {\n      window.PyViz.plot_index[id] = Bokeh.index[id];\n    } else {\n      window.PyViz.plot_index[id] = null;\n    }\n  } else if (output.metadata[EXEC_MIME_TYPE][\"server_id\"] !== undefined) {\n    var bk_div = document.createElement(\"div\");\n    bk_div.innerHTML = output.data[HTML_MIME_TYPE];\n    var script_attrs = bk_div.children[0].attributes;\n    for (var i = 0; i < script_attrs.length; i++) {\n      toinsert[toinsert.length - 1].childNodes[1].setAttribute(script_attrs[i].name, script_attrs[i].value);\n    }\n    // store reference to server id on output_area\n    output_area._bokeh_server_id = output.metadata[EXEC_MIME_TYPE][\"server_id\"];\n  }\n}\n\n/**\n * Handle when an output is cleared or removed\n */\nfunction handle_clear_output(event, handle) {\n  var id = handle.cell.output_area._hv_plot_id;\n  var server_id = handle.cell.output_area._bokeh_server_id;\n  if (((id === undefined) || !(id in PyViz.plot_index)) && (server_id !== undefined)) { return; }\n  var comm = window.PyViz.comm_manager.get_client_comm(\"hv-extension-comm\", \"hv-extension-comm\", function () {});\n  if (server_id !== null) {\n    comm.send({event_type: 'server_delete', 'id': server_id});\n    return;\n  } else if (comm !== null) {\n    comm.send({event_type: 'delete', 'id': id});\n  }\n  delete PyViz.plot_index[id];\n  if ((window.Bokeh !== undefined) & (id in window.Bokeh.index)) {\n    var doc = window.Bokeh.index[id].model.document\n    doc.clear();\n    const i = window.Bokeh.documents.indexOf(doc);\n    if (i > -1) {\n      window.Bokeh.documents.splice(i, 1);\n    }\n  }\n}\n\n/**\n * Handle kernel restart event\n */\nfunction handle_kernel_cleanup(event, handle) {\n  delete PyViz.comms[\"hv-extension-comm\"];\n  window.PyViz.plot_index = {}\n}\n\n/**\n * Handle update_display_data messages\n */\nfunction handle_update_output(event, handle) {\n  handle_clear_output(event, {cell: {output_area: handle.output_area}})\n  handle_add_output(event, handle)\n}\n\nfunction register_renderer(events, OutputArea) {\n  function append_mime(data, metadata, element) {\n    // create a DOM node to render to\n    var toinsert = this.create_output_subarea(\n    metadata,\n    CLASS_NAME,\n    EXEC_MIME_TYPE\n    );\n    this.keyboard_manager.register_events(toinsert);\n    // Render to node\n    var props = {data: data, metadata: metadata[EXEC_MIME_TYPE]};\n    render(props, toinsert[0]);\n    element.append(toinsert);\n    return toinsert\n  }\n\n  events.on('output_added.OutputArea', handle_add_output);\n  events.on('output_updated.OutputArea', handle_update_output);\n  events.on('clear_output.CodeCell', handle_clear_output);\n  events.on('delete.Cell', handle_clear_output);\n  events.on('kernel_ready.Kernel', handle_kernel_cleanup);\n\n  OutputArea.prototype.register_mime_type(EXEC_MIME_TYPE, append_mime, {\n    safe: true,\n    index: 0\n  });\n}\n\nif (window.Jupyter !== undefined) {\n  try {\n    var events = require('base/js/events');\n    var OutputArea = require('notebook/js/outputarea').OutputArea;\n    if (OutputArea.prototype.mime_types().indexOf(EXEC_MIME_TYPE) == -1) {\n      register_renderer(events, OutputArea);\n    }\n  } catch(err) {\n  }\n}\n",
      "application/javascript": "\nif ((window.PyViz === undefined) || (window.PyViz instanceof HTMLElement)) {\n  window.PyViz = {comms: {}, comm_status:{}, kernels:{}, receivers: {}, plot_index: []}\n}\n\n\n    function JupyterCommManager() {\n    }\n\n    JupyterCommManager.prototype.register_target = function(plot_id, comm_id, msg_handler) {\n      if (window.comm_manager || ((window.Jupyter !== undefined) && (Jupyter.notebook.kernel != null))) {\n        var comm_manager = window.comm_manager || Jupyter.notebook.kernel.comm_manager;\n        comm_manager.register_target(comm_id, function(comm) {\n          comm.on_msg(msg_handler);\n        });\n      } else if ((plot_id in window.PyViz.kernels) && (window.PyViz.kernels[plot_id])) {\n        window.PyViz.kernels[plot_id].registerCommTarget(comm_id, function(comm) {\n          comm.onMsg = msg_handler;\n        });\n      } else if (typeof google != 'undefined' && google.colab.kernel != null) {\n        google.colab.kernel.comms.registerTarget(comm_id, (comm) => {\n          var messages = comm.messages[Symbol.asyncIterator]();\n          function processIteratorResult(result) {\n            var message = result.value;\n            console.log(message)\n            var content = {data: message.data, comm_id};\n            var buffers = []\n            for (var buffer of message.buffers || []) {\n              buffers.push(new DataView(buffer))\n            }\n            var metadata = message.metadata || {};\n            var msg = {content, buffers, metadata}\n            msg_handler(msg);\n            return messages.next().then(processIteratorResult);\n          }\n          return messages.next().then(processIteratorResult);\n        })\n      }\n    }\n\n    JupyterCommManager.prototype.get_client_comm = function(plot_id, comm_id, msg_handler) {\n      if (comm_id in window.PyViz.comms) {\n        return window.PyViz.comms[comm_id];\n      } else if (window.comm_manager || ((window.Jupyter !== undefined) && (Jupyter.notebook.kernel != null))) {\n        var comm_manager = window.comm_manager || Jupyter.notebook.kernel.comm_manager;\n        var comm = comm_manager.new_comm(comm_id, {}, {}, {}, comm_id);\n        if (msg_handler) {\n          comm.on_msg(msg_handler);\n        }\n      } else if ((plot_id in window.PyViz.kernels) && (window.PyViz.kernels[plot_id])) {\n        var comm = window.PyViz.kernels[plot_id].connectToComm(comm_id);\n        comm.open();\n        if (msg_handler) {\n          comm.onMsg = msg_handler;\n        }\n      } else if (typeof google != 'undefined' && google.colab.kernel != null) {\n        var comm_promise = google.colab.kernel.comms.open(comm_id)\n        comm_promise.then((comm) => {\n          window.PyViz.comms[comm_id] = comm;\n          if (msg_handler) {\n            var messages = comm.messages[Symbol.asyncIterator]();\n            function processIteratorResult(result) {\n              var message = result.value;\n              var content = {data: message.data};\n              var metadata = message.metadata || {comm_id};\n              var msg = {content, metadata}\n              msg_handler(msg);\n              return messages.next().then(processIteratorResult);\n            }\n            return messages.next().then(processIteratorResult);\n          }\n        })\n        var sendClosure = (data, metadata, buffers, disposeOnDone) => {\n          return comm_promise.then((comm) => {\n            comm.send(data, metadata, buffers, disposeOnDone);\n          });\n        };\n        var comm = {\n          send: sendClosure\n        };\n      }\n      window.PyViz.comms[comm_id] = comm;\n      return comm;\n    }\n    window.PyViz.comm_manager = new JupyterCommManager();\n    \n\n\nvar JS_MIME_TYPE = 'application/javascript';\nvar HTML_MIME_TYPE = 'text/html';\nvar EXEC_MIME_TYPE = 'application/vnd.holoviews_exec.v0+json';\nvar CLASS_NAME = 'output';\n\n/**\n * Render data to the DOM node\n */\nfunction render(props, node) {\n  var div = document.createElement(\"div\");\n  var script = document.createElement(\"script\");\n  node.appendChild(div);\n  node.appendChild(script);\n}\n\n/**\n * Handle when a new output is added\n */\nfunction handle_add_output(event, handle) {\n  var output_area = handle.output_area;\n  var output = handle.output;\n  if ((output.data == undefined) || (!output.data.hasOwnProperty(EXEC_MIME_TYPE))) {\n    return\n  }\n  var id = output.metadata[EXEC_MIME_TYPE][\"id\"];\n  var toinsert = output_area.element.find(\".\" + CLASS_NAME.split(' ')[0]);\n  if (id !== undefined) {\n    var nchildren = toinsert.length;\n    var html_node = toinsert[nchildren-1].children[0];\n    html_node.innerHTML = output.data[HTML_MIME_TYPE];\n    var scripts = [];\n    var nodelist = html_node.querySelectorAll(\"script\");\n    for (var i in nodelist) {\n      if (nodelist.hasOwnProperty(i)) {\n        scripts.push(nodelist[i])\n      }\n    }\n\n    scripts.forEach( function (oldScript) {\n      var newScript = document.createElement(\"script\");\n      var attrs = [];\n      var nodemap = oldScript.attributes;\n      for (var j in nodemap) {\n        if (nodemap.hasOwnProperty(j)) {\n          attrs.push(nodemap[j])\n        }\n      }\n      attrs.forEach(function(attr) { newScript.setAttribute(attr.name, attr.value) });\n      newScript.appendChild(document.createTextNode(oldScript.innerHTML));\n      oldScript.parentNode.replaceChild(newScript, oldScript);\n    });\n    if (JS_MIME_TYPE in output.data) {\n      toinsert[nchildren-1].children[1].textContent = output.data[JS_MIME_TYPE];\n    }\n    output_area._hv_plot_id = id;\n    if ((window.Bokeh !== undefined) && (id in Bokeh.index)) {\n      window.PyViz.plot_index[id] = Bokeh.index[id];\n    } else {\n      window.PyViz.plot_index[id] = null;\n    }\n  } else if (output.metadata[EXEC_MIME_TYPE][\"server_id\"] !== undefined) {\n    var bk_div = document.createElement(\"div\");\n    bk_div.innerHTML = output.data[HTML_MIME_TYPE];\n    var script_attrs = bk_div.children[0].attributes;\n    for (var i = 0; i < script_attrs.length; i++) {\n      toinsert[toinsert.length - 1].childNodes[1].setAttribute(script_attrs[i].name, script_attrs[i].value);\n    }\n    // store reference to server id on output_area\n    output_area._bokeh_server_id = output.metadata[EXEC_MIME_TYPE][\"server_id\"];\n  }\n}\n\n/**\n * Handle when an output is cleared or removed\n */\nfunction handle_clear_output(event, handle) {\n  var id = handle.cell.output_area._hv_plot_id;\n  var server_id = handle.cell.output_area._bokeh_server_id;\n  if (((id === undefined) || !(id in PyViz.plot_index)) && (server_id !== undefined)) { return; }\n  var comm = window.PyViz.comm_manager.get_client_comm(\"hv-extension-comm\", \"hv-extension-comm\", function () {});\n  if (server_id !== null) {\n    comm.send({event_type: 'server_delete', 'id': server_id});\n    return;\n  } else if (comm !== null) {\n    comm.send({event_type: 'delete', 'id': id});\n  }\n  delete PyViz.plot_index[id];\n  if ((window.Bokeh !== undefined) & (id in window.Bokeh.index)) {\n    var doc = window.Bokeh.index[id].model.document\n    doc.clear();\n    const i = window.Bokeh.documents.indexOf(doc);\n    if (i > -1) {\n      window.Bokeh.documents.splice(i, 1);\n    }\n  }\n}\n\n/**\n * Handle kernel restart event\n */\nfunction handle_kernel_cleanup(event, handle) {\n  delete PyViz.comms[\"hv-extension-comm\"];\n  window.PyViz.plot_index = {}\n}\n\n/**\n * Handle update_display_data messages\n */\nfunction handle_update_output(event, handle) {\n  handle_clear_output(event, {cell: {output_area: handle.output_area}})\n  handle_add_output(event, handle)\n}\n\nfunction register_renderer(events, OutputArea) {\n  function append_mime(data, metadata, element) {\n    // create a DOM node to render to\n    var toinsert = this.create_output_subarea(\n    metadata,\n    CLASS_NAME,\n    EXEC_MIME_TYPE\n    );\n    this.keyboard_manager.register_events(toinsert);\n    // Render to node\n    var props = {data: data, metadata: metadata[EXEC_MIME_TYPE]};\n    render(props, toinsert[0]);\n    element.append(toinsert);\n    return toinsert\n  }\n\n  events.on('output_added.OutputArea', handle_add_output);\n  events.on('output_updated.OutputArea', handle_update_output);\n  events.on('clear_output.CodeCell', handle_clear_output);\n  events.on('delete.Cell', handle_clear_output);\n  events.on('kernel_ready.Kernel', handle_kernel_cleanup);\n\n  OutputArea.prototype.register_mime_type(EXEC_MIME_TYPE, append_mime, {\n    safe: true,\n    index: 0\n  });\n}\n\nif (window.Jupyter !== undefined) {\n  try {\n    var events = require('base/js/events');\n    var OutputArea = require('notebook/js/outputarea').OutputArea;\n    if (OutputArea.prototype.mime_types().indexOf(EXEC_MIME_TYPE) == -1) {\n      register_renderer(events, OutputArea);\n    }\n  } catch(err) {\n  }\n}\n"
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div id='c043216a-5de7-4857-9553-55eaba14212d'>\n",
       "  <div id=\"b4119065-a15e-4953-8240-4822abeb957e\" data-root-id=\"c043216a-5de7-4857-9553-55eaba14212d\" style=\"display: contents;\"></div>\n",
       "</div>\n",
       "<script type=\"application/javascript\">(function(root) {\n",
       "  var docs_json = {\"dc83f493-0a31-4add-916c-e1a9b3d83c1d\":{\"version\":\"3.6.3\",\"title\":\"Bokeh Application\",\"roots\":[{\"type\":\"object\",\"name\":\"panel.models.browser.BrowserInfo\",\"id\":\"c043216a-5de7-4857-9553-55eaba14212d\"},{\"type\":\"object\",\"name\":\"panel.models.comm_manager.CommManager\",\"id\":\"c0119b9e-57e7-4102-b7e5-155fdaa975ca\",\"attributes\":{\"plot_id\":\"c043216a-5de7-4857-9553-55eaba14212d\",\"comm_id\":\"7b8aa2dbd0bf4f9e8967ee84bfbb4dfe\",\"client_comm_id\":\"5b2330710b3c4aac860e14ce0c939f27\"}}],\"defs\":[{\"type\":\"model\",\"name\":\"ReactiveHTML1\"},{\"type\":\"model\",\"name\":\"FlexBox1\",\"properties\":[{\"name\":\"align_content\",\"kind\":\"Any\",\"default\":\"flex-start\"},{\"name\":\"align_items\",\"kind\":\"Any\",\"default\":\"flex-start\"},{\"name\":\"flex_direction\",\"kind\":\"Any\",\"default\":\"row\"},{\"name\":\"flex_wrap\",\"kind\":\"Any\",\"default\":\"wrap\"},{\"name\":\"gap\",\"kind\":\"Any\",\"default\":\"\"},{\"name\":\"justify_content\",\"kind\":\"Any\",\"default\":\"flex-start\"}]},{\"type\":\"model\",\"name\":\"FloatPanel1\",\"properties\":[{\"name\":\"config\",\"kind\":\"Any\",\"default\":{\"type\":\"map\"}},{\"name\":\"contained\",\"kind\":\"Any\",\"default\":true},{\"name\":\"position\",\"kind\":\"Any\",\"default\":\"right-top\"},{\"name\":\"offsetx\",\"kind\":\"Any\",\"default\":null},{\"name\":\"offsety\",\"kind\":\"Any\",\"default\":null},{\"name\":\"theme\",\"kind\":\"Any\",\"default\":\"primary\"},{\"name\":\"status\",\"kind\":\"Any\",\"default\":\"normalized\"}]},{\"type\":\"model\",\"name\":\"GridStack1\",\"properties\":[{\"name\":\"mode\",\"kind\":\"Any\",\"default\":\"warn\"},{\"name\":\"ncols\",\"kind\":\"Any\",\"default\":null},{\"name\":\"nrows\",\"kind\":\"Any\",\"default\":null},{\"name\":\"allow_resize\",\"kind\":\"Any\",\"default\":true},{\"name\":\"allow_drag\",\"kind\":\"Any\",\"default\":true},{\"name\":\"state\",\"kind\":\"Any\",\"default\":[]}]},{\"type\":\"model\",\"name\":\"drag1\",\"properties\":[{\"name\":\"slider_width\",\"kind\":\"Any\",\"default\":5},{\"name\":\"slider_color\",\"kind\":\"Any\",\"default\":\"black\"},{\"name\":\"value\",\"kind\":\"Any\",\"default\":50}]},{\"type\":\"model\",\"name\":\"click1\",\"properties\":[{\"name\":\"terminal_output\",\"kind\":\"Any\",\"default\":\"\"},{\"name\":\"debug_name\",\"kind\":\"Any\",\"default\":\"\"},{\"name\":\"clears\",\"kind\":\"Any\",\"default\":0}]},{\"type\":\"model\",\"name\":\"FastWrapper1\",\"properties\":[{\"name\":\"object\",\"kind\":\"Any\",\"default\":null},{\"name\":\"style\",\"kind\":\"Any\",\"default\":null}]},{\"type\":\"model\",\"name\":\"NotificationAreaBase1\",\"properties\":[{\"name\":\"js_events\",\"kind\":\"Any\",\"default\":{\"type\":\"map\"}},{\"name\":\"position\",\"kind\":\"Any\",\"default\":\"bottom-right\"},{\"name\":\"_clear\",\"kind\":\"Any\",\"default\":0}]},{\"type\":\"model\",\"name\":\"NotificationArea1\",\"properties\":[{\"name\":\"js_events\",\"kind\":\"Any\",\"default\":{\"type\":\"map\"}},{\"name\":\"notifications\",\"kind\":\"Any\",\"default\":[]},{\"name\":\"position\",\"kind\":\"Any\",\"default\":\"bottom-right\"},{\"name\":\"_clear\",\"kind\":\"Any\",\"default\":0},{\"name\":\"types\",\"kind\":\"Any\",\"default\":[{\"type\":\"map\",\"entries\":[[\"type\",\"warning\"],[\"background\",\"#ffc107\"],[\"icon\",{\"type\":\"map\",\"entries\":[[\"className\",\"fas fa-exclamation-triangle\"],[\"tagName\",\"i\"],[\"color\",\"white\"]]}]]},{\"type\":\"map\",\"entries\":[[\"type\",\"info\"],[\"background\",\"#007bff\"],[\"icon\",{\"type\":\"map\",\"entries\":[[\"className\",\"fas fa-info-circle\"],[\"tagName\",\"i\"],[\"color\",\"white\"]]}]]}]}]},{\"type\":\"model\",\"name\":\"Notification\",\"properties\":[{\"name\":\"background\",\"kind\":\"Any\",\"default\":null},{\"name\":\"duration\",\"kind\":\"Any\",\"default\":3000},{\"name\":\"icon\",\"kind\":\"Any\",\"default\":null},{\"name\":\"message\",\"kind\":\"Any\",\"default\":\"\"},{\"name\":\"notification_type\",\"kind\":\"Any\",\"default\":null},{\"name\":\"_rendered\",\"kind\":\"Any\",\"default\":false},{\"name\":\"_destroyed\",\"kind\":\"Any\",\"default\":false}]},{\"type\":\"model\",\"name\":\"TemplateActions1\",\"properties\":[{\"name\":\"open_modal\",\"kind\":\"Any\",\"default\":0},{\"name\":\"close_modal\",\"kind\":\"Any\",\"default\":0}]},{\"type\":\"model\",\"name\":\"BootstrapTemplateActions1\",\"properties\":[{\"name\":\"open_modal\",\"kind\":\"Any\",\"default\":0},{\"name\":\"close_modal\",\"kind\":\"Any\",\"default\":0}]},{\"type\":\"model\",\"name\":\"TemplateEditor1\",\"properties\":[{\"name\":\"layout\",\"kind\":\"Any\",\"default\":[]}]},{\"type\":\"model\",\"name\":\"MaterialTemplateActions1\",\"properties\":[{\"name\":\"open_modal\",\"kind\":\"Any\",\"default\":0},{\"name\":\"close_modal\",\"kind\":\"Any\",\"default\":0}]},{\"type\":\"model\",\"name\":\"ReactiveESM1\",\"properties\":[{\"name\":\"esm_constants\",\"kind\":\"Any\",\"default\":{\"type\":\"map\"}}]},{\"type\":\"model\",\"name\":\"JSComponent1\",\"properties\":[{\"name\":\"esm_constants\",\"kind\":\"Any\",\"default\":{\"type\":\"map\"}}]},{\"type\":\"model\",\"name\":\"ReactComponent1\",\"properties\":[{\"name\":\"esm_constants\",\"kind\":\"Any\",\"default\":{\"type\":\"map\"}}]},{\"type\":\"model\",\"name\":\"AnyWidgetComponent1\",\"properties\":[{\"name\":\"esm_constants\",\"kind\":\"Any\",\"default\":{\"type\":\"map\"}}]},{\"type\":\"model\",\"name\":\"request_value1\",\"properties\":[{\"name\":\"fill\",\"kind\":\"Any\",\"default\":\"none\"},{\"name\":\"_synced\",\"kind\":\"Any\",\"default\":null},{\"name\":\"_request_sync\",\"kind\":\"Any\",\"default\":0}]}]}};\n",
       "  var render_items = [{\"docid\":\"dc83f493-0a31-4add-916c-e1a9b3d83c1d\",\"roots\":{\"c043216a-5de7-4857-9553-55eaba14212d\":\"b4119065-a15e-4953-8240-4822abeb957e\"},\"root_ids\":[\"c043216a-5de7-4857-9553-55eaba14212d\"]}];\n",
       "  var docs = Object.values(docs_json)\n",
       "  if (!docs) {\n",
       "    return\n",
       "  }\n",
       "  const py_version = docs[0].version.replace('rc', '-rc.').replace('.dev', '-dev.')\n",
       "  async function embed_document(root) {\n",
       "    var Bokeh = get_bokeh(root)\n",
       "    await Bokeh.embed.embed_items_notebook(docs_json, render_items);\n",
       "    for (const render_item of render_items) {\n",
       "      for (const root_id of render_item.root_ids) {\n",
       "\tconst id_el = document.getElementById(root_id)\n",
       "\tif (id_el.children.length && id_el.children[0].hasAttribute('data-root-id')) {\n",
       "\t  const root_el = id_el.children[0]\n",
       "\t  root_el.id = root_el.id + '-rendered'\n",
       "\t  for (const child of root_el.children) {\n",
       "            // Ensure JupyterLab does not capture keyboard shortcuts\n",
       "            // see: https://jupyterlab.readthedocs.io/en/4.1.x/extension/notebook.html#keyboard-interaction-model\n",
       "\t    child.setAttribute('data-lm-suppress-shortcuts', 'true')\n",
       "\t  }\n",
       "\t}\n",
       "      }\n",
       "    }\n",
       "  }\n",
       "  function get_bokeh(root) {\n",
       "    if (root.Bokeh === undefined) {\n",
       "      return null\n",
       "    } else if (root.Bokeh.version !== py_version) {\n",
       "      if (root.Bokeh.versions === undefined || !root.Bokeh.versions.has(py_version)) {\n",
       "\treturn null\n",
       "      }\n",
       "      return root.Bokeh.versions.get(py_version);\n",
       "    } else if (root.Bokeh.version === py_version) {\n",
       "      return root.Bokeh\n",
       "    }\n",
       "    return null\n",
       "  }\n",
       "  function is_loaded(root) {\n",
       "    var Bokeh = get_bokeh(root)\n",
       "    return (Bokeh != null && Bokeh.Panel !== undefined && ( root.Tabulator !== undefined) && ( root.Tabulator !== undefined))\n",
       "  }\n",
       "  if (is_loaded(root)) {\n",
       "    embed_document(root);\n",
       "  } else {\n",
       "    var attempts = 0;\n",
       "    var timer = setInterval(function(root) {\n",
       "      if (is_loaded(root)) {\n",
       "        clearInterval(timer);\n",
       "        embed_document(root);\n",
       "      } else if (document.readyState == \"complete\") {\n",
       "        attempts++;\n",
       "        if (attempts > 200) {\n",
       "          clearInterval(timer);\n",
       "\t  var Bokeh = get_bokeh(root)\n",
       "\t  if (Bokeh == null || Bokeh.Panel == null) {\n",
       "            console.warn(\"Panel: ERROR: Unable to run Panel code because Bokeh or Panel library is missing\");\n",
       "\t  } else {\n",
       "\t    console.warn(\"Panel: WARNING: Attempting to render but not all required libraries could be resolved.\")\n",
       "\t    embed_document(root)\n",
       "\t  }\n",
       "        }\n",
       "      }\n",
       "    }, 25, root)\n",
       "  }\n",
       "})(window);</script>"
      ],
      "application/vnd.holoviews_exec.v0+json": ""
     },
     "metadata": {
      "application/vnd.holoviews_exec.v0+json": {
       "id": "c043216a-5de7-4857-9553-55eaba14212d"
      }
     },
     "output_type": "display_data"
    }
   ],
   "execution_count": 1
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-15T01:17:05.360931Z",
     "start_time": "2025-03-15T01:17:05.299851Z"
    }
   },
   "cell_type": "code",
   "source": [
    "pmid_doi_mapping_file = \"exp_output/pmid_doi_mapping.json\"\n",
    "if os.path.exists(pmid_doi_mapping_file): \n",
    "    with open(pmid_doi_mapping_file, \"r\") as f:\n",
    "        pmid_doi_mapping = json.load(f)  # read json file\n",
    "else:\n",
    "    pmid_doi_mapping = {}"
   ],
   "id": "50756180b4c24f93",
   "outputs": [],
   "execution_count": 2
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-15T01:17:05.417626Z",
     "start_time": "2025-03-15T01:17:05.361846Z"
    }
   },
   "cell_type": "code",
   "source": [
    "pmid_pmcid_mapping_file = \"exp_output/pmid_pmcid_mapping.json\"\n",
    "if os.path.exists(pmid_pmcid_mapping_file): \n",
    "    with open(pmid_pmcid_mapping_file, \"r\") as f:\n",
    "        pmid_pmcid_mapping = json.load(f)  # read json file\n",
    "else:\n",
    "    pmid_pmcid_mapping = {}"
   ],
   "id": "29fd4f17725c6c15",
   "outputs": [],
   "execution_count": 3
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-15T01:17:05.479424Z",
     "start_time": "2025-03-15T01:17:05.421445Z"
    }
   },
   "cell_type": "code",
   "source": [
    "doi_pmid_mapping_file = \"exp_output/doi_pmid_mapping.json\"\n",
    "if os.path.exists(doi_pmid_mapping_file): \n",
    "    with open(doi_pmid_mapping_file, \"r\") as f:\n",
    "        doi_pmid_mapping = json.load(f)  # read json file\n",
    "else:\n",
    "    doi_pmid_mapping = {}"
   ],
   "id": "18408bf774092f3",
   "outputs": [],
   "execution_count": 4
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-15T01:17:05.539016Z",
     "start_time": "2025-03-15T01:17:05.480583Z"
    }
   },
   "cell_type": "code",
   "source": [
    "doi_to_pmcid_mapping_file = \"exp_output/doi_pmcid_mapping.json\"\n",
    "if os.path.exists(doi_to_pmcid_mapping_file):\n",
    "    with open(doi_to_pmcid_mapping_file, \"r\") as f:\n",
    "        doi_pmcid_mapping = json.load(f)  # read json file\n",
    "else:\n",
    "    doi_pmcid_mapping = {}"
   ],
   "id": "3325f0d07d692efb",
   "outputs": [],
   "execution_count": 5
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-15T01:17:05.551776Z",
     "start_time": "2025-03-15T01:17:05.540517Z"
    }
   },
   "cell_type": "code",
   "source": "len(pmid_doi_mapping), len(pmid_pmcid_mapping), len(doi_pmid_mapping), len(doi_pmcid_mapping)",
   "id": "fb2fdc261ff79ecb",
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(144993, 144993, 125273, 125011)"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "execution_count": 6
  },
  {
   "metadata": {},
   "cell_type": "markdown",
   "source": [
    "## Extract the datasets data from ProteomeCentral and GEO\n",
    "\n",
    "Start from the export file from ProteomeCentral: https://proteomecentral.proteomexchange.org/"
   ],
   "id": "bb128744ad141f19"
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-15T01:17:05.762821Z",
     "start_time": "2025-03-15T01:17:05.552863Z"
    }
   },
   "cell_type": "code",
   "source": [
    "export_df = pd.read_csv('exp_input/proteomexchange_search.tsv', sep='\\t')\n",
    "print(f\"number of dataset ids in export dataframe from ProteomeCentral: {len(export_df)}\")"
   ],
   "id": "dbca6a1609d3f90f",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "number of dataset ids in export dataframe from ProteomeCentral: 40137\n"
     ]
    }
   ],
   "execution_count": 7
  },
  {
   "metadata": {},
   "cell_type": "markdown",
   "source": "to get an idea of the most frequent values in df publication, for all the dataset ids from the source file",
   "id": "216c36ad08a8ccc8"
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-15T01:17:05.780923Z",
     "start_time": "2025-03-15T01:17:05.764203Z"
    }
   },
   "cell_type": "code",
   "source": "export_df['publication'].value_counts().head()",
   "id": "3ea4110aabcad351",
   "outputs": [
    {
     "data": {
      "text/plain": [
       "publication\n",
       "Dataset with its publication pending                                                                 11719\n",
       "no publication                                                                                        2550\n",
       "<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/35084980\" target=\"_blank\">Melani et al. (2022)</a>          56\n",
       "<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/28267743\" target=\"_blank\">Matsumoto et al. (2017)</a>       28\n",
       "<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/28071820\" target=\"_blank\">Kreutz et al. (2017)</a>          18\n",
       "Name: count, dtype: int64"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "execution_count": 8
  },
  {
   "metadata": {},
   "cell_type": "markdown",
   "source": "Clean the `publication` column by filtering out unwanted values like `Dataset with its publication pending`, `no publication`. Remove rows with unwanted values",
   "id": "6c9abdec1b840be7"
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-15T01:17:05.791024Z",
     "start_time": "2025-03-15T01:17:05.781872Z"
    }
   },
   "cell_type": "code",
   "source": [
    "filtered_df = export_df[~export_df['publication'].isin([\"Dataset with its publication pending\", \"no publication\"])]\n",
    "print(len(filtered_df),'\\n',filtered_df.columns)"
   ],
   "id": "d2d0fb59a01d4418",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "25868 \n",
      " Index(['identifier', 'title', 'repository', 'species', 'instrument',\n",
      "       'publication', 'lab_head', 'announce_date', 'keywords'],\n",
      "      dtype='object')\n"
     ]
    }
   ],
   "execution_count": 9
  },
  {
   "metadata": {},
   "cell_type": "markdown",
   "source": "Normalize the `publication` column by extracting the links from the html tags",
   "id": "d0913bd823b2b838"
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-15T01:17:06.825328Z",
     "start_time": "2025-03-15T01:17:05.794619Z"
    }
   },
   "cell_type": "code",
   "source": [
    "filtered_df = filtered_df.copy()\n",
    "\n",
    "filtered_df.loc[:, 'citing_publications_links'] = None\n",
    "\n",
    "for i, row in filtered_df.iterrows():\n",
    "    pub = str(row['publication'])  # Ensure string type\n",
    "    if \"href\" in pub:\n",
    "        match = re.findall(r'href=[\\'\"]([^\\'\"]+)[\\'\"]', pub)  # Extract href links\n",
    "        filtered_df.at[i, 'citing_publications_links'] = match if match else None\n",
    "    else:\n",
    "        filtered_df.at[i, 'citing_publications_links'] = None\n",
    "\n",
    "# Drop rows with missing links safely\n",
    "filtered_df = filtered_df.dropna(subset=['citing_publications_links']).reset_index(drop=True)\n",
    "filtered_df.drop(columns=['publication'], inplace=True)  # Drop original column\n",
    "\n",
    "print(f\"# n datasets with publications: {len(filtered_df)}\") \n",
    "print(filtered_df.columns)"
   ],
   "id": "2329fcd5dd45e72a",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# n datasets with publications: 25696\n",
      "Index(['identifier', 'title', 'repository', 'species', 'instrument',\n",
      "       'lab_head', 'announce_date', 'keywords', 'citing_publications_links'],\n",
      "      dtype='object')\n"
     ]
    }
   ],
   "execution_count": 10
  },
  {
   "metadata": {},
   "cell_type": "markdown",
   "source": [
    "Normalize links:\n",
    "\n",
    "\n",
    "- if pubmed link, find the pmcid and the doi mapped to the pmid\n",
    "\n",
    "\n",
    "- if doi link, find the doi and the pmcid mapped to the doi"
   ],
   "id": "92c5f052811859e8"
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-15T01:17:09.172267Z",
     "start_time": "2025-03-15T01:17:06.826260Z"
    }
   },
   "cell_type": "code",
   "source": [
    "filtered_df = filtered_df.explode('citing_publications_links', ignore_index=True)  # Split lists into rows\n",
    "print(f\"Length: {len(filtered_df)}\")\n",
    "\n",
    "filtered_df = extract_publication_ids_from_PX_export(filtered_df, pmid_doi_mapping, pmid_pmcid_mapping, doi_pmid_mapping, doi_pmcid_mapping)\n",
    "print(f\"Length should be the same: {len(filtered_df)}\")\n",
    "\n",
    "filtered_df = add_citing_publication_link_columns(filtered_df)\n",
    "print(f\"Length should be greater: {len(filtered_df)}\")\n",
    "\n",
    "filtered_df[\"citing_publications_links\"] = filtered_df[\"citing_publications_links\"].astype(str)\n",
    "filtered_df[\"DOI\"] = filtered_df[\"DOI\"].astype(str)\n",
    "filtered_df[\"PMCID\"] = filtered_df[\"PMCID\"].astype(str)\n",
    "\n",
    "# Now drop duplicates\n",
    "filtered_df.drop_duplicates(subset=['citing_publications_links', 'DOI', 'PMCID'], inplace=True)\n",
    "print(f\"Length could be a bit smaller: {len(filtered_df)}\")\n",
    "\n",
    "filtered_df[['citing_publications_links','DOI','PMID','PMCID']].sample(10)"
   ],
   "id": "1659f3f351bbbfcd",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Length: 39347\n",
      "Length should be the same: 39347\n",
      "Before explode sample:                           DOI        PMCID  \\\n",
      "0  10.1038/S41467-025-56720-1         None   \n",
      "1           10.6019/PXD051312         None   \n",
      "2      10.1002/prca.202400095  PMC11895760   \n",
      "3      10.1002/prca.202400095  PMC11895760   \n",
      "4    10.17159/SAJS.2025/18571         None   \n",
      "5   10.1101/2024.04.03.587901         None   \n",
      "6   10.1101/2025.02.05.636703         None   \n",
      "7  10.1186/S13100-024-00339-4         None   \n",
      "8  10.1186/s12964-025-02046-w  PMC11773904   \n",
      "9  10.1186/s12964-025-02046-w  PMC11773904   \n",
      "\n",
      "                           citing_publications_links  \n",
      "0    [https://dx.doi.org/10.1038/S41467-025-56720-1]  \n",
      "1             [https://dx.doi.org/10.6019/PXD051312]  \n",
      "2  [https://dx.doi.org/10.1002/prca.202400095, ht...  \n",
      "3  [https://dx.doi.org/10.1002/prca.202400095, ht...  \n",
      "4      [https://dx.doi.org/10.17159/SAJS.2025/18571]  \n",
      "5     [https://dx.doi.org/10.1101/2024.04.03.587901]  \n",
      "6     [https://dx.doi.org/10.1101/2025.02.05.636703]  \n",
      "7    [https://dx.doi.org/10.1186/S13100-024-00339-4]  \n",
      "8  [https://dx.doi.org/10.1186/s12964-025-02046-w...  \n",
      "9  [https://dx.doi.org/10.1186/s12964-025-02046-w...  \n",
      "Final dataframe length: 59833\n",
      "Length should be greater: 59833\n",
      "Length could be a bit smaller: 38688\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "                               citing_publications_links  \\\n",
       "23039      https://dx.doi.org/10.1038/S41467-024-44988-8   \n",
       "20273      https://dx.doi.org/10.1038/s41423-024-01150-0   \n",
       "41660           https://dx.doi.org/10.3390/cells10061306   \n",
       "43111  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8...   \n",
       "1351                https://dx.doi.org/10.6019/PXD046271   \n",
       "28362  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9...   \n",
       "54036      https://dx.doi.org/10.1038/s41467-018-07321-8   \n",
       "34061  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8...   \n",
       "54198               https://dx.doi.org/10.6019/PXD009649   \n",
       "13141  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...   \n",
       "\n",
       "                              DOI      PMID        PMCID  \n",
       "23039  10.1038/S41467-024-44988-8      None         None  \n",
       "20273  10.1038/s41423-024-01150-0  38689020  PMC11143349  \n",
       "41660       10.3390/cells10061306  34074003   PMC8225046  \n",
       "43111  10.1161/ATVBAHA.120.315007  33356393   PMC8105275  \n",
       "1351            10.6019/PXD046271      None         None  \n",
       "28362  10.1038/s41586-023-05780-8  36859550   PMC9995272  \n",
       "54036  10.1038/s41467-018-07321-8  30451861   PMC6242847  \n",
       "34061  10.1038/s42003-022-03259-2  35393494   PMC8991201  \n",
       "54198           10.6019/PXD009649      None         None  \n",
       "13141    10.3389/fcimb.2019.00104  31069174   PMC6491454  "
      ],
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>citing_publications_links</th>\n",
       "      <th>DOI</th>\n",
       "      <th>PMID</th>\n",
       "      <th>PMCID</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>23039</th>\n",
       "      <td>https://dx.doi.org/10.1038/S41467-024-44988-8</td>\n",
       "      <td>10.1038/S41467-024-44988-8</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20273</th>\n",
       "      <td>https://dx.doi.org/10.1038/s41423-024-01150-0</td>\n",
       "      <td>10.1038/s41423-024-01150-0</td>\n",
       "      <td>38689020</td>\n",
       "      <td>PMC11143349</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41660</th>\n",
       "      <td>https://dx.doi.org/10.3390/cells10061306</td>\n",
       "      <td>10.3390/cells10061306</td>\n",
       "      <td>34074003</td>\n",
       "      <td>PMC8225046</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43111</th>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8...</td>\n",
       "      <td>10.1161/ATVBAHA.120.315007</td>\n",
       "      <td>33356393</td>\n",
       "      <td>PMC8105275</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1351</th>\n",
       "      <td>https://dx.doi.org/10.6019/PXD046271</td>\n",
       "      <td>10.6019/PXD046271</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28362</th>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9...</td>\n",
       "      <td>10.1038/s41586-023-05780-8</td>\n",
       "      <td>36859550</td>\n",
       "      <td>PMC9995272</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>54036</th>\n",
       "      <td>https://dx.doi.org/10.1038/s41467-018-07321-8</td>\n",
       "      <td>10.1038/s41467-018-07321-8</td>\n",
       "      <td>30451861</td>\n",
       "      <td>PMC6242847</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34061</th>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8...</td>\n",
       "      <td>10.1038/s42003-022-03259-2</td>\n",
       "      <td>35393494</td>\n",
       "      <td>PMC8991201</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>54198</th>\n",
       "      <td>https://dx.doi.org/10.6019/PXD009649</td>\n",
       "      <td>10.6019/PXD009649</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13141</th>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...</td>\n",
       "      <td>10.3389/fcimb.2019.00104</td>\n",
       "      <td>31069174</td>\n",
       "      <td>PMC6491454</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "execution_count": 11
  },
  {
   "metadata": {},
   "cell_type": "markdown",
   "source": "Extraction of data from GEO",
   "id": "55bbf6947515dd93"
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-15T01:17:19.111299Z",
     "start_time": "2025-03-15T01:17:09.173200Z"
    }
   },
   "cell_type": "code",
   "source": [
    "GEO_extract_file = \"exp_input/GEO_data.csv\"\n",
    "GEO_df = pd.read_csv(GEO_extract_file, low_memory=False)\n",
    "print(f\"number of dataset ids in GEO dataframe: {len(GEO_df)}\")"
   ],
   "id": "295128eefbe94e80",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "number of dataset ids in GEO dataframe: 248748\n"
     ]
    }
   ],
   "execution_count": 12
  },
  {
   "metadata": {},
   "cell_type": "markdown",
   "source": "Extracting all the datasets with eutils api and get all the uids of GEO series datasets",
   "id": "546fbf3d16cf4bee"
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-15T01:17:20.087677Z",
     "start_time": "2025-03-15T01:17:19.112619Z"
    }
   },
   "cell_type": "code",
   "source": [
    "root = ElementTree.fromstring(requests.get(\"https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi\", \n",
    "                                           params={\"db\": \"gds\", \"term\": '\"gse\"[Entry Type]', \"retmax\": \"1000000\",  \"retmode\": \"xml\"}\n",
    "                                           ).content)\n",
    "\n",
    "gse_ids = [id_elem.text for id_elem in root.findall(\".//Id\")]\n",
    "missing_ids = set(gse_ids)-set([str(x) for x in GEO_df.uid])\n",
    "\n",
    "print(f\"Total datasets found: {len(gse_ids)}\")\n",
    "print(\"Missing GSE IDs:\", len(missing_ids))  # Show difference with data we already got"
   ],
   "id": "4617db8aaafb7e89",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Total datasets found: 249060\n",
      "Missing GSE IDs: 326\n"
     ]
    }
   ],
   "execution_count": 13
  },
  {
   "metadata": {},
   "cell_type": "markdown",
   "source": "Add new datasets from GEO to the GEO dataframe",
   "id": "31296faf5c71f288"
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-15T01:17:58.491896Z",
     "start_time": "2025-03-15T01:17:20.088658Z"
    }
   },
   "cell_type": "code",
   "source": [
    "new_GEO_series_data = {}\n",
    "i, mxm = 0, 1\n",
    "while True:\n",
    "    print(f\"Progress: {min(i / len(missing_ids), 1) * 100} %\")\n",
    "    if i > len(missing_ids):\n",
    "        break\n",
    "    try:\n",
    "        new_data = fetch_GEO_data(list(missing_ids),\"https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esummary.fcgi\",i,mxm)\n",
    "        for uid, details in new_data[\"result\"].items():\n",
    "            if uid == \"uids\":  # Ignore metadata key\n",
    "                continue\n",
    "            new_GEO_series_data[uid] = details\n",
    "    except Exception as e:\n",
    "        print(f\"Error at {i}, {mxm} with error: {e}\")\n",
    "            \n",
    "    i += 1\n",
    "    mxm += 1\n",
    "    time.sleep(0.005) \n",
    "\n",
    "add_new_data = False # add new datasets to the GEO_df\n",
    "if add_new_data:\n",
    "    GEO_df = pd.concat([GEO_df, pd.DataFrame(new_GEO_series_data).T], axis=0)"
   ],
   "id": "71bbda7af9fb1a78",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Progress: 0.0 %\n",
      "Progress: 0.3067484662576687 %\n",
      "Progress: 0.6134969325153374 %\n",
      "Progress: 0.9202453987730062 %\n",
      "Error at 3, 4 with error: 'result'\n",
      "Progress: 1.2269938650306749 %\n",
      "Error at 4, 5 with error: 'result'\n",
      "Progress: 1.5337423312883436 %\n",
      "Error at 5, 6 with error: 'result'\n",
      "Progress: 1.8404907975460123 %\n",
      "Error at 6, 7 with error: 'result'\n",
      "Progress: 2.147239263803681 %\n",
      "Error at 7, 8 with error: 'result'\n",
      "Progress: 2.4539877300613497 %\n",
      "Progress: 2.7607361963190185 %\n",
      "Progress: 3.067484662576687 %\n",
      "Error at 10, 11 with error: 'result'\n",
      "Progress: 3.374233128834356 %\n",
      "Error at 11, 12 with error: 'result'\n",
      "Progress: 3.6809815950920246 %\n",
      "Error at 12, 13 with error: 'result'\n",
      "Progress: 3.9877300613496933 %\n",
      "Error at 13, 14 with error: 'result'\n",
      "Progress: 4.294478527607362 %\n",
      "Error at 14, 15 with error: 'result'\n",
      "Progress: 4.601226993865031 %\n",
      "Error at 15, 16 with error: 'result'\n",
      "Progress: 4.9079754601226995 %\n",
      "Progress: 5.214723926380368 %\n",
      "Error at 17, 18 with error: 'result'\n",
      "Progress: 5.521472392638037 %\n",
      "Error at 18, 19 with error: 'result'\n",
      "Progress: 5.828220858895705 %\n",
      "Error at 19, 20 with error: 'result'\n",
      "Progress: 6.134969325153374 %\n",
      "Error at 20, 21 with error: 'result'\n",
      "Progress: 6.441717791411043 %\n",
      "Error at 21, 22 with error: 'result'\n",
      "Progress: 6.748466257668712 %\n",
      "Error at 22, 23 with error: 'result'\n",
      "Progress: 7.05521472392638 %\n",
      "Progress: 7.361963190184049 %\n",
      "Progress: 7.668711656441718 %\n",
      "Error at 25, 26 with error: 'result'\n",
      "Progress: 7.975460122699387 %\n",
      "Progress: 8.282208588957054 %\n",
      "Progress: 8.588957055214724 %\n",
      "Progress: 8.895705521472392 %\n",
      "Error at 29, 30 with error: 'result'\n",
      "Progress: 9.202453987730062 %\n",
      "Error at 30, 31 with error: 'result'\n",
      "Progress: 9.509202453987731 %\n",
      "Error at 31, 32 with error: 'result'\n",
      "Progress: 9.815950920245399 %\n",
      "Error at 32, 33 with error: 'result'\n",
      "Progress: 10.122699386503067 %\n",
      "Error at 33, 34 with error: 'result'\n",
      "Progress: 10.429447852760736 %\n",
      "Error at 34, 35 with error: 'result'\n",
      "Progress: 10.736196319018406 %\n",
      "Progress: 11.042944785276074 %\n",
      "Error at 36, 37 with error: 'result'\n",
      "Progress: 11.349693251533742 %\n",
      "Error at 37, 38 with error: 'result'\n",
      "Progress: 11.65644171779141 %\n",
      "Error at 38, 39 with error: 'result'\n",
      "Progress: 11.96319018404908 %\n",
      "Error at 39, 40 with error: 'result'\n",
      "Progress: 12.269938650306749 %\n",
      "Progress: 12.576687116564417 %\n",
      "Progress: 12.883435582822086 %\n",
      "Error at 42, 43 with error: 'result'\n",
      "Progress: 13.190184049079754 %\n",
      "Error at 43, 44 with error: 'result'\n",
      "Progress: 13.496932515337424 %\n",
      "Error at 44, 45 with error: 'result'\n",
      "Progress: 13.803680981595093 %\n",
      "Error at 45, 46 with error: 'result'\n",
      "Progress: 14.11042944785276 %\n",
      "Error at 46, 47 with error: 'result'\n",
      "Progress: 14.417177914110429 %\n",
      "Error at 47, 48 with error: 'result'\n",
      "Progress: 14.723926380368098 %\n",
      "Error at 48, 49 with error: 'result'\n",
      "Progress: 15.030674846625766 %\n",
      "Progress: 15.337423312883436 %\n",
      "Error at 50, 51 with error: 'result'\n",
      "Progress: 15.644171779141105 %\n",
      "Error at 51, 52 with error: 'result'\n",
      "Progress: 15.950920245398773 %\n",
      "Error at 52, 53 with error: 'result'\n",
      "Progress: 16.257668711656443 %\n",
      "Error at 53, 54 with error: 'result'\n",
      "Progress: 16.56441717791411 %\n",
      "Progress: 16.87116564417178 %\n",
      "Progress: 17.177914110429448 %\n",
      "Error at 56, 57 with error: 'result'\n",
      "Progress: 17.484662576687114 %\n",
      "Error at 57, 58 with error: 'result'\n",
      "Progress: 17.791411042944784 %\n",
      "Error at 58, 59 with error: 'result'\n",
      "Progress: 18.098159509202453 %\n",
      "Error at 59, 60 with error: 'result'\n",
      "Progress: 18.404907975460123 %\n",
      "Error at 60, 61 with error: 'result'\n",
      "Progress: 18.711656441717793 %\n",
      "Error at 61, 62 with error: 'result'\n",
      "Progress: 19.018404907975462 %\n",
      "Error at 62, 63 with error: 'result'\n",
      "Progress: 19.32515337423313 %\n",
      "Progress: 19.631901840490798 %\n",
      "Error at 64, 65 with error: 'result'\n",
      "Progress: 19.938650306748464 %\n",
      "Error at 65, 66 with error: 'result'\n",
      "Progress: 20.245398773006134 %\n",
      "Error at 66, 67 with error: 'result'\n",
      "Progress: 20.552147239263803 %\n",
      "Error at 67, 68 with error: 'result'\n",
      "Progress: 20.858895705521473 %\n",
      "Progress: 21.165644171779142 %\n",
      "Progress: 21.472392638036812 %\n",
      "Error at 70, 71 with error: 'result'\n",
      "Progress: 21.779141104294478 %\n",
      "Error at 71, 72 with error: 'result'\n",
      "Progress: 22.085889570552148 %\n",
      "Error at 72, 73 with error: 'result'\n",
      "Progress: 22.392638036809817 %\n",
      "Error at 73, 74 with error: 'result'\n",
      "Progress: 22.699386503067483 %\n",
      "Error at 74, 75 with error: 'result'\n",
      "Progress: 23.006134969325153 %\n",
      "Progress: 23.31288343558282 %\n",
      "Error at 76, 77 with error: 'result'\n",
      "Progress: 23.619631901840492 %\n",
      "Error at 77, 78 with error: 'result'\n",
      "Progress: 23.92638036809816 %\n",
      "Error at 78, 79 with error: 'result'\n",
      "Progress: 24.233128834355828 %\n",
      "Error at 79, 80 with error: 'result'\n",
      "Progress: 24.539877300613497 %\n",
      "Error at 80, 81 with error: 'result'\n",
      "Progress: 24.846625766871167 %\n",
      "Progress: 25.153374233128833 %\n",
      "Progress: 25.4601226993865 %\n",
      "Error at 83, 84 with error: 'result'\n",
      "Progress: 25.766871165644172 %\n",
      "Error at 84, 85 with error: 'result'\n",
      "Progress: 26.07361963190184 %\n",
      "Progress: 26.380368098159508 %\n",
      "Error at 86, 87 with error: 'result'\n",
      "Progress: 26.68711656441718 %\n",
      "Error at 87, 88 with error: 'result'\n",
      "Progress: 26.993865030674847 %\n",
      "Progress: 27.300613496932513 %\n",
      "Progress: 27.607361963190186 %\n",
      "Progress: 27.914110429447852 %\n",
      "Error at 91, 92 with error: 'result'\n",
      "Progress: 28.22085889570552 %\n",
      "Error at 92, 93 with error: 'result'\n",
      "Progress: 28.52760736196319 %\n",
      "Error at 93, 94 with error: 'result'\n",
      "Progress: 28.834355828220858 %\n",
      "Error at 94, 95 with error: 'result'\n",
      "Progress: 29.141104294478527 %\n",
      "Error at 95, 96 with error: 'result'\n",
      "Progress: 29.447852760736197 %\n",
      "Progress: 29.754601226993866 %\n",
      "Progress: 30.061349693251532 %\n",
      "Error at 98, 99 with error: 'result'\n",
      "Progress: 30.368098159509206 %\n",
      "Error at 99, 100 with error: 'result'\n",
      "Progress: 30.67484662576687 %\n",
      "Error at 100, 101 with error: 'result'\n",
      "Progress: 30.981595092024538 %\n",
      "Progress: 31.28834355828221 %\n",
      "Error at 102, 103 with error: 'result'\n",
      "Progress: 31.595092024539877 %\n",
      "Error at 103, 104 with error: 'result'\n",
      "Progress: 31.901840490797547 %\n",
      "Progress: 32.20858895705521 %\n",
      "Error at 105, 106 with error: 'result'\n",
      "Progress: 32.515337423312886 %\n",
      "Progress: 32.82208588957055 %\n",
      "Error at 107, 108 with error: 'result'\n",
      "Progress: 33.12883435582822 %\n",
      "Error at 108, 109 with error: 'result'\n",
      "Progress: 33.43558282208589 %\n",
      "Progress: 33.74233128834356 %\n",
      "Error at 110, 111 with error: 'result'\n",
      "Progress: 34.04907975460122 %\n",
      "Error at 111, 112 with error: 'result'\n",
      "Progress: 34.355828220858896 %\n",
      "Error at 112, 113 with error: 'result'\n",
      "Progress: 34.66257668711656 %\n",
      "Progress: 34.96932515337423 %\n",
      "Error at 114, 115 with error: 'result'\n",
      "Progress: 35.2760736196319 %\n",
      "Progress: 35.58282208588957 %\n",
      "Error at 116, 117 with error: 'result'\n",
      "Progress: 35.88957055214724 %\n",
      "Error at 117, 118 with error: 'result'\n",
      "Progress: 36.19631901840491 %\n",
      "Progress: 36.50306748466258 %\n",
      "Progress: 36.809815950920246 %\n",
      "Error at 120, 121 with error: 'result'\n",
      "Progress: 37.11656441717792 %\n",
      "Progress: 37.423312883435585 %\n",
      "Progress: 37.73006134969325 %\n",
      "Error at 123, 124 with error: 'result'\n",
      "Progress: 38.036809815950924 %\n",
      "Error at 124, 125 with error: 'result'\n",
      "Progress: 38.34355828220859 %\n",
      "Progress: 38.65030674846626 %\n",
      "Progress: 38.95705521472393 %\n",
      "Progress: 39.263803680981596 %\n",
      "Progress: 39.57055214723926 %\n",
      "Error at 129, 130 with error: 'result'\n",
      "Progress: 39.87730061349693 %\n",
      "Error at 130, 131 with error: 'result'\n",
      "Progress: 40.1840490797546 %\n",
      "Error at 131, 132 with error: 'result'\n",
      "Progress: 40.49079754601227 %\n",
      "Error at 132, 133 with error: 'result'\n",
      "Progress: 40.79754601226993 %\n",
      "Progress: 41.104294478527606 %\n",
      "Progress: 41.41104294478527 %\n",
      "Progress: 41.717791411042946 %\n",
      "Error at 136, 137 with error: 'result'\n",
      "Progress: 42.02453987730061 %\n",
      "Error at 137, 138 with error: 'result'\n",
      "Progress: 42.331288343558285 %\n",
      "Error at 138, 139 with error: 'result'\n",
      "Progress: 42.63803680981595 %\n",
      "Progress: 42.944785276073624 %\n",
      "Progress: 43.25153374233129 %\n",
      "Progress: 43.558282208588956 %\n",
      "Progress: 43.86503067484663 %\n",
      "Progress: 44.171779141104295 %\n",
      "Error at 144, 145 with error: 'result'\n",
      "Progress: 44.47852760736196 %\n",
      "Error at 145, 146 with error: 'result'\n",
      "Progress: 44.785276073619634 %\n",
      "Error at 146, 147 with error: 'result'\n",
      "Progress: 45.0920245398773 %\n",
      "Error at 147, 148 with error: 'result'\n",
      "Progress: 45.39877300613497 %\n",
      "Progress: 45.70552147239264 %\n",
      "Progress: 46.012269938650306 %\n",
      "Progress: 46.31901840490797 %\n",
      "Error at 151, 152 with error: 'result'\n",
      "Progress: 46.62576687116564 %\n",
      "Error at 152, 153 with error: 'result'\n",
      "Progress: 46.93251533742331 %\n",
      "Error at 153, 154 with error: 'result'\n",
      "Progress: 47.239263803680984 %\n",
      "Error at 154, 155 with error: 'result'\n",
      "Progress: 47.54601226993865 %\n",
      "Error at 155, 156 with error: 'result'\n",
      "Progress: 47.85276073619632 %\n",
      "Error at 156, 157 with error: 'result'\n",
      "Progress: 48.15950920245399 %\n",
      "Progress: 48.466257668711656 %\n",
      "Progress: 48.77300613496933 %\n",
      "Progress: 49.079754601226995 %\n",
      "Error at 160, 161 with error: 'result'\n",
      "Progress: 49.38650306748466 %\n",
      "Error at 161, 162 with error: 'result'\n",
      "Progress: 49.693251533742334 %\n",
      "Error at 162, 163 with error: 'result'\n",
      "Progress: 50.0 %\n",
      "Error at 163, 164 with error: 'result'\n",
      "Progress: 50.306748466257666 %\n",
      "Error at 164, 165 with error: 'result'\n",
      "Progress: 50.61349693251533 %\n",
      "Error at 165, 166 with error: 'result'\n",
      "Progress: 50.920245398773 %\n",
      "Progress: 51.22699386503068 %\n",
      "Progress: 51.533742331288344 %\n",
      "Progress: 51.84049079754601 %\n",
      "Error at 169, 170 with error: 'result'\n",
      "Progress: 52.14723926380368 %\n",
      "Error at 170, 171 with error: 'result'\n",
      "Progress: 52.45398773006135 %\n",
      "Error at 171, 172 with error: 'result'\n",
      "Progress: 52.760736196319016 %\n",
      "Error at 172, 173 with error: 'result'\n",
      "Progress: 53.06748466257669 %\n",
      "Error at 173, 174 with error: 'result'\n",
      "Progress: 53.37423312883436 %\n",
      "Progress: 53.68098159509203 %\n",
      "Error at 175, 176 with error: 'result'\n",
      "Progress: 53.987730061349694 %\n",
      "Progress: 54.29447852760736 %\n",
      "Progress: 54.601226993865026 %\n",
      "Error at 178, 179 with error: 'result'\n",
      "Progress: 54.90797546012271 %\n",
      "Error at 179, 180 with error: 'result'\n",
      "Progress: 55.21472392638037 %\n",
      "Error at 180, 181 with error: 'result'\n",
      "Progress: 55.52147239263804 %\n",
      "Error at 181, 182 with error: 'result'\n",
      "Progress: 55.828220858895705 %\n",
      "Error at 182, 183 with error: 'result'\n",
      "Progress: 56.13496932515337 %\n",
      "Error at 183, 184 with error: 'result'\n",
      "Progress: 56.44171779141104 %\n",
      "Progress: 56.74846625766872 %\n",
      "Progress: 57.05521472392638 %\n",
      "Progress: 57.36196319018405 %\n",
      "Error at 187, 188 with error: 'result'\n",
      "Progress: 57.668711656441715 %\n",
      "Error at 188, 189 with error: 'result'\n",
      "Progress: 57.97546012269939 %\n",
      "Error at 189, 190 with error: 'result'\n",
      "Progress: 58.282208588957054 %\n",
      "Error at 190, 191 with error: 'result'\n",
      "Progress: 58.58895705521472 %\n",
      "Progress: 58.895705521472394 %\n",
      "Progress: 59.20245398773007 %\n",
      "Progress: 59.50920245398773 %\n",
      "Error at 194, 195 with error: 'result'\n",
      "Progress: 59.8159509202454 %\n",
      "Error at 195, 196 with error: 'result'\n",
      "Progress: 60.122699386503065 %\n",
      "Error at 196, 197 with error: 'result'\n",
      "Progress: 60.42944785276073 %\n",
      "Progress: 60.73619631901841 %\n",
      "Progress: 61.04294478527608 %\n",
      "Progress: 61.34969325153374 %\n",
      "Error at 200, 201 with error: 'result'\n",
      "Progress: 61.65644171779141 %\n",
      "Error at 201, 202 with error: 'result'\n",
      "Progress: 61.963190184049076 %\n",
      "Progress: 62.26993865030674 %\n",
      "Error at 203, 204 with error: 'result'\n",
      "Progress: 62.57668711656442 %\n",
      "Error at 204, 205 with error: 'result'\n",
      "Progress: 62.88343558282209 %\n",
      "Error at 205, 206 with error: 'result'\n",
      "Progress: 63.190184049079754 %\n",
      "Error at 206, 207 with error: 'result'\n",
      "Progress: 63.49693251533742 %\n",
      "Progress: 63.80368098159509 %\n",
      "Progress: 64.11042944785275 %\n",
      "Error at 209, 210 with error: 'result'\n",
      "Progress: 64.41717791411043 %\n",
      "Error at 210, 211 with error: 'result'\n",
      "Progress: 64.7239263803681 %\n",
      "Error at 211, 212 with error: 'result'\n",
      "Progress: 65.03067484662577 %\n",
      "Error at 212, 213 with error: 'result'\n",
      "Progress: 65.33742331288343 %\n",
      "Progress: 65.6441717791411 %\n",
      "Error at 214, 215 with error: 'result'\n",
      "Progress: 65.95092024539878 %\n",
      "Error at 215, 216 with error: 'result'\n",
      "Progress: 66.25766871165644 %\n",
      "Error at 216, 217 with error: 'result'\n",
      "Progress: 66.56441717791411 %\n",
      "Error at 217, 218 with error: 'result'\n",
      "Progress: 66.87116564417178 %\n",
      "Error at 218, 219 with error: 'result'\n",
      "Progress: 67.17791411042946 %\n",
      "Progress: 67.48466257668711 %\n",
      "Progress: 67.79141104294479 %\n",
      "Progress: 68.09815950920245 %\n",
      "Progress: 68.40490797546013 %\n",
      "Error at 223, 224 with error: 'result'\n",
      "Progress: 68.71165644171779 %\n",
      "Error at 224, 225 with error: 'result'\n",
      "Progress: 69.01840490797547 %\n",
      "Error at 225, 226 with error: 'result'\n",
      "Progress: 69.32515337423312 %\n",
      "Error at 226, 227 with error: 'result'\n",
      "Progress: 69.6319018404908 %\n",
      "Error at 227, 228 with error: 'result'\n",
      "Progress: 69.93865030674846 %\n",
      "Error at 228, 229 with error: 'result'\n",
      "Progress: 70.24539877300614 %\n",
      "Error at 229, 230 with error: 'result'\n",
      "Progress: 70.5521472392638 %\n",
      "Error at 230, 231 with error: 'result'\n",
      "Progress: 70.85889570552148 %\n",
      "Error at 231, 232 with error: 'result'\n",
      "Progress: 71.16564417177914 %\n",
      "Error at 232, 233 with error: 'result'\n",
      "Progress: 71.47239263803681 %\n",
      "Error at 233, 234 with error: 'result'\n",
      "Progress: 71.77914110429448 %\n",
      "Progress: 72.08588957055214 %\n",
      "Progress: 72.39263803680981 %\n",
      "Progress: 72.69938650306749 %\n",
      "Error at 237, 238 with error: 'result'\n",
      "Progress: 73.00613496932516 %\n",
      "Error at 238, 239 with error: 'result'\n",
      "Progress: 73.31288343558282 %\n",
      "Error at 239, 240 with error: 'result'\n",
      "Progress: 73.61963190184049 %\n",
      "Error at 240, 241 with error: 'result'\n",
      "Progress: 73.92638036809815 %\n",
      "Error at 241, 242 with error: 'result'\n",
      "Progress: 74.23312883435584 %\n",
      "Error at 242, 243 with error: 'result'\n",
      "Progress: 74.5398773006135 %\n",
      "Error at 243, 244 with error: 'result'\n",
      "Progress: 74.84662576687117 %\n",
      "Error at 244, 245 with error: 'result'\n",
      "Progress: 75.15337423312883 %\n",
      "Progress: 75.4601226993865 %\n",
      "Progress: 75.76687116564416 %\n",
      "Progress: 76.07361963190185 %\n",
      "Error at 248, 249 with error: 'result'\n",
      "Progress: 76.38036809815951 %\n",
      "Error at 249, 250 with error: 'result'\n",
      "Progress: 76.68711656441718 %\n",
      "Error at 250, 251 with error: 'result'\n",
      "Progress: 76.99386503067484 %\n",
      "Error at 251, 252 with error: 'result'\n",
      "Progress: 77.30061349693251 %\n",
      "Error at 252, 253 with error: 'result'\n",
      "Progress: 77.60736196319019 %\n",
      "Error at 253, 254 with error: 'result'\n",
      "Progress: 77.91411042944786 %\n",
      "Error at 254, 255 with error: 'result'\n",
      "Progress: 78.22085889570552 %\n",
      "Error at 255, 256 with error: 'result'\n",
      "Progress: 78.52760736196319 %\n",
      "Progress: 78.83435582822086 %\n",
      "Progress: 79.14110429447852 %\n",
      "Progress: 79.4478527607362 %\n",
      "Error at 259, 260 with error: 'result'\n",
      "Progress: 79.75460122699386 %\n",
      "Error at 260, 261 with error: 'result'\n",
      "Progress: 80.06134969325154 %\n",
      "Error at 261, 262 with error: 'result'\n",
      "Progress: 80.3680981595092 %\n",
      "Error at 262, 263 with error: 'result'\n",
      "Progress: 80.67484662576688 %\n",
      "Error at 263, 264 with error: 'result'\n",
      "Progress: 80.98159509202453 %\n",
      "Error at 264, 265 with error: 'result'\n",
      "Progress: 81.28834355828221 %\n",
      "Error at 265, 266 with error: 'result'\n",
      "Progress: 81.59509202453987 %\n",
      "Error at 266, 267 with error: 'result'\n",
      "Progress: 81.90184049079755 %\n",
      "Error at 267, 268 with error: 'result'\n",
      "Progress: 82.20858895705521 %\n",
      "Progress: 82.51533742331289 %\n",
      "Error at 269, 270 with error: 'result'\n",
      "Progress: 82.82208588957054 %\n",
      "Error at 270, 271 with error: 'result'\n",
      "Progress: 83.12883435582822 %\n",
      "Progress: 83.43558282208589 %\n",
      "Error at 272, 273 with error: 'result'\n",
      "Progress: 83.74233128834356 %\n",
      "Progress: 84.04907975460122 %\n",
      "Error at 274, 275 with error: 'result'\n",
      "Progress: 84.3558282208589 %\n",
      "Error at 275, 276 with error: 'result'\n",
      "Progress: 84.66257668711657 %\n",
      "Error at 276, 277 with error: 'result'\n",
      "Progress: 84.96932515337423 %\n",
      "Error at 277, 278 with error: 'result'\n",
      "Progress: 85.2760736196319 %\n",
      "Error at 278, 279 with error: 'result'\n",
      "Progress: 85.58282208588957 %\n",
      "Error at 279, 280 with error: 'result'\n",
      "Progress: 85.88957055214725 %\n",
      "Error at 280, 281 with error: 'result'\n",
      "Progress: 86.1963190184049 %\n",
      "Error at 281, 282 with error: 'result'\n",
      "Progress: 86.50306748466258 %\n",
      "Error at 282, 283 with error: 'result'\n",
      "Progress: 86.80981595092024 %\n",
      "Progress: 87.11656441717791 %\n",
      "Error at 284, 285 with error: 'result'\n",
      "Progress: 87.42331288343557 %\n",
      "Progress: 87.73006134969326 %\n",
      "Error at 286, 287 with error: 'result'\n",
      "Progress: 88.03680981595092 %\n",
      "Progress: 88.34355828220859 %\n",
      "Error at 288, 289 with error: 'result'\n",
      "Progress: 88.65030674846625 %\n",
      "Error at 289, 290 with error: 'result'\n",
      "Progress: 88.95705521472392 %\n",
      "Error at 290, 291 with error: 'result'\n",
      "Progress: 89.2638036809816 %\n",
      "Error at 291, 292 with error: 'result'\n",
      "Progress: 89.57055214723927 %\n",
      "Error at 292, 293 with error: 'result'\n",
      "Progress: 89.87730061349694 %\n",
      "Error at 293, 294 with error: 'result'\n",
      "Progress: 90.1840490797546 %\n",
      "Error at 294, 295 with error: 'result'\n",
      "Progress: 90.49079754601227 %\n",
      "Progress: 90.79754601226993 %\n",
      "Progress: 91.1042944785276 %\n",
      "Progress: 91.41104294478528 %\n",
      "Progress: 91.71779141104295 %\n",
      "Progress: 92.02453987730061 %\n",
      "Error at 300, 301 with error: 'result'\n",
      "Progress: 92.33128834355828 %\n",
      "Error at 301, 302 with error: 'result'\n",
      "Progress: 92.63803680981594 %\n",
      "Error at 302, 303 with error: 'result'\n",
      "Progress: 92.94478527607362 %\n",
      "Error at 303, 304 with error: 'result'\n",
      "Progress: 93.25153374233128 %\n",
      "Error at 304, 305 with error: 'result'\n",
      "Progress: 93.55828220858896 %\n",
      "Error at 305, 306 with error: 'result'\n",
      "Progress: 93.86503067484662 %\n",
      "Error at 306, 307 with error: 'result'\n",
      "Progress: 94.1717791411043 %\n",
      "Error at 307, 308 with error: 'result'\n",
      "Progress: 94.47852760736197 %\n",
      "Error at 308, 309 with error: 'result'\n",
      "Progress: 94.78527607361963 %\n",
      "Error at 309, 310 with error: 'result'\n",
      "Progress: 95.0920245398773 %\n",
      "Error at 310, 311 with error: 'result'\n",
      "Progress: 95.39877300613497 %\n",
      "Progress: 95.70552147239265 %\n",
      "Progress: 96.0122699386503 %\n",
      "Progress: 96.31901840490798 %\n",
      "Error at 314, 315 with error: 'result'\n",
      "Progress: 96.62576687116564 %\n",
      "Error at 315, 316 with error: 'result'\n",
      "Progress: 96.93251533742331 %\n",
      "Error at 316, 317 with error: 'result'\n",
      "Progress: 97.23926380368098 %\n",
      "Error at 317, 318 with error: 'result'\n",
      "Progress: 97.54601226993866 %\n",
      "Error at 318, 319 with error: 'result'\n",
      "Progress: 97.85276073619632 %\n",
      "Error at 319, 320 with error: 'result'\n",
      "Progress: 98.15950920245399 %\n",
      "Error at 320, 321 with error: 'result'\n",
      "Progress: 98.46625766871165 %\n",
      "Error at 321, 322 with error: 'result'\n",
      "Progress: 98.77300613496932 %\n",
      "Error at 322, 323 with error: 'result'\n",
      "Progress: 99.079754601227 %\n",
      "Error at 323, 324 with error: 'result'\n",
      "Progress: 99.38650306748467 %\n",
      "Error at 324, 325 with error: 'result'\n",
      "Progress: 99.69325153374233 %\n",
      "Error at 325, 326 with error: 'result'\n",
      "Progress: 100.0 %\n",
      "Error at 326, 327 with error: 'result'\n",
      "Progress: 100 %\n"
     ]
    }
   ],
   "execution_count": 14
  },
  {
   "metadata": {},
   "cell_type": "markdown",
   "source": "We will update the filtered_df with the new datasets from GEO, after matching the schema",
   "id": "9e4dbff7754cb951"
  },
  {
   "metadata": {},
   "cell_type": "markdown",
   "source": "Add the url to the GEO datasets by reconstruction from pubmedids",
   "id": "ca6166a0e0741439"
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-15T01:18:00.249104Z",
     "start_time": "2025-03-15T01:17:58.493390Z"
    }
   },
   "cell_type": "code",
   "source": [
    "# eval string to list\n",
    "GEO_df['pubmedids'] = GEO_df['pubmedids'].apply(lambda x: ast.literal_eval(x) if isinstance(x, str) else x)\n",
    "print(len(GEO_df))\n",
    "GEO_df = GEO_df[GEO_df['pubmedids'].astype(bool)]  # Drop rows with empty lists\n",
    "print(len(GEO_df))\n",
    "GEO_df = GEO_df.explode('pubmedids', ignore_index=True)  # Split lists into rows\n",
    "print(len(GEO_df))"
   ],
   "id": "ffc6a180b21f49ba",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "248748\n",
      "194093\n",
      "208151\n"
     ]
    }
   ],
   "execution_count": 15
  },
  {
   "metadata": {},
   "cell_type": "markdown",
   "source": [
    "- if pubmed link, find the pmcid and the doi mapped to the pmid\n",
    "\n",
    "\n",
    "- Create Links for the publications"
   ],
   "id": "e7ce599657d4d14f"
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-15T01:18:00.406691Z",
     "start_time": "2025-03-15T01:18:00.250020Z"
    }
   },
   "cell_type": "code",
   "source": [
    "GEO_df['PMCID'] = GEO_df['pubmedids'].map(\n",
    "    lambda x: pmid_pmcid_mapping.get(x, None) if isinstance(x, str) else None )\n",
    "GEO_df['DOI'] = GEO_df['pubmedids'].map(\n",
    "    lambda x: pmid_doi_mapping.get(x, None) if isinstance(x, str) else None)\n",
    "GEO_df[['pubmedids','PMCID','DOI']].sample(10)"
   ],
   "id": "e693614cf6b9c589",
   "outputs": [
    {
     "data": {
      "text/plain": [
       "       pubmedids        PMCID                           DOI\n",
       "160949  25204429   PMC4177250      10.1186/1476-4598-13-209\n",
       "72254   34017133         None                          None\n",
       "75702   22955616   PMC3439153           10.1038/nature11247\n",
       "10356   38922876         None                          None\n",
       "203094  17910761   PMC2078596       10.1186/1471-2164-8-349\n",
       "41741   36428978   PMC9688917         10.3390/cells11223549\n",
       "32969   37108567  PMC10138523          10.3390/ijms24087403\n",
       "54127   34010639   PMC9678241  10.1016/j.celrep.2021.109136\n",
       "146641  27058788   PMC4826471  10.1016/j.molcel.2016.03.001\n",
       "168889  24317252   PMC4018771            10.1038/nmeth.2762"
      ],
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>pubmedids</th>\n",
       "      <th>PMCID</th>\n",
       "      <th>DOI</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>160949</th>\n",
       "      <td>25204429</td>\n",
       "      <td>PMC4177250</td>\n",
       "      <td>10.1186/1476-4598-13-209</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>72254</th>\n",
       "      <td>34017133</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>75702</th>\n",
       "      <td>22955616</td>\n",
       "      <td>PMC3439153</td>\n",
       "      <td>10.1038/nature11247</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10356</th>\n",
       "      <td>38922876</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>203094</th>\n",
       "      <td>17910761</td>\n",
       "      <td>PMC2078596</td>\n",
       "      <td>10.1186/1471-2164-8-349</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41741</th>\n",
       "      <td>36428978</td>\n",
       "      <td>PMC9688917</td>\n",
       "      <td>10.3390/cells11223549</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32969</th>\n",
       "      <td>37108567</td>\n",
       "      <td>PMC10138523</td>\n",
       "      <td>10.3390/ijms24087403</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>54127</th>\n",
       "      <td>34010639</td>\n",
       "      <td>PMC9678241</td>\n",
       "      <td>10.1016/j.celrep.2021.109136</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>146641</th>\n",
       "      <td>27058788</td>\n",
       "      <td>PMC4826471</td>\n",
       "      <td>10.1016/j.molcel.2016.03.001</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>168889</th>\n",
       "      <td>24317252</td>\n",
       "      <td>PMC4018771</td>\n",
       "      <td>10.1038/nmeth.2762</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ]
     },
     "execution_count": 16,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "execution_count": 16
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-15T01:18:02.989956Z",
     "start_time": "2025-03-15T01:18:00.407740Z"
    }
   },
   "cell_type": "code",
   "source": [
    "# Generate citing_publications_links as a list of links\n",
    "GEO_df = add_citing_publication_link_columns(GEO_df)\n",
    "\n",
    "GEO_df[['pubmedids','PMCID','DOI','citing_publications_links']].sample(10)"
   ],
   "id": "5fbf937a11e6591",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Before explode sample:                        DOI        PMCID  \\\n",
      "0                     None         None   \n",
      "1   10.1093/immhor/vlae005  PMC11841973   \n",
      "2      10.1093/nar/gkaf109  PMC11840560   \n",
      "3                     None         None   \n",
      "4                     None         None   \n",
      "5                     None         None   \n",
      "6                     None         None   \n",
      "7                     None         None   \n",
      "8                     None         None   \n",
      "9  10.1073/pnas.2416384122  PMC11892594   \n",
      "\n",
      "                           citing_publications_links  \n",
      "0                                                 []  \n",
      "1  [https://dx.doi.org/10.1093/immhor/vlae005, ht...  \n",
      "2  [https://dx.doi.org/10.1093/nar/gkaf109, https...  \n",
      "3                                                 []  \n",
      "4                                                 []  \n",
      "5                                                 []  \n",
      "6                                                 []  \n",
      "7                                                 []  \n",
      "8                                                 []  \n",
      "9  [https://dx.doi.org/10.1073/pnas.2416384122, h...  \n",
      "Final dataframe length: 352982\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "       pubmedids        PMCID                                DOI  \\\n",
       "112075  35184570   PMC8923958  10.1161/CIRCULATIONAHA.121.057613   \n",
       "171466  33420030   PMC7794439         10.1038/s41467-020-20447-y   \n",
       "39904   38278841  PMC10817939         10.1038/s41467-024-45141-1   \n",
       "250313  28334889   PMC5449619                 10.1093/nar/gkx168   \n",
       "86351   36073526   PMC9553214                10.7554/eLife.78074   \n",
       "247816  31677552   PMC6920132        10.1016/j.redox.2019.101359   \n",
       "181385  37172728  PMC10267520          10.1016/j.jbc.2023.104805   \n",
       "113274  35140216   PMC8828724         10.1038/s41467-022-28454-x   \n",
       "38338   39041647  PMC11605269                  10.1002/art.42958   \n",
       "156674  33804598   PMC8003772               10.3390/ijms22063164   \n",
       "\n",
       "                                citing_publications_links  \n",
       "112075  https://dx.doi.org/10.1161/CIRCULATIONAHA.121....  \n",
       "171466  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7...  \n",
       "39904   https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...  \n",
       "250313  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5...  \n",
       "86351              https://dx.doi.org/10.7554/eLife.78074  \n",
       "247816  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...  \n",
       "181385  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...  \n",
       "113274      https://dx.doi.org/10.1038/s41467-022-28454-x  \n",
       "38338   https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...  \n",
       "156674            https://dx.doi.org/10.3390/ijms22063164  "
      ],
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>pubmedids</th>\n",
       "      <th>PMCID</th>\n",
       "      <th>DOI</th>\n",
       "      <th>citing_publications_links</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>112075</th>\n",
       "      <td>35184570</td>\n",
       "      <td>PMC8923958</td>\n",
       "      <td>10.1161/CIRCULATIONAHA.121.057613</td>\n",
       "      <td>https://dx.doi.org/10.1161/CIRCULATIONAHA.121....</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>171466</th>\n",
       "      <td>33420030</td>\n",
       "      <td>PMC7794439</td>\n",
       "      <td>10.1038/s41467-020-20447-y</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39904</th>\n",
       "      <td>38278841</td>\n",
       "      <td>PMC10817939</td>\n",
       "      <td>10.1038/s41467-024-45141-1</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>250313</th>\n",
       "      <td>28334889</td>\n",
       "      <td>PMC5449619</td>\n",
       "      <td>10.1093/nar/gkx168</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>86351</th>\n",
       "      <td>36073526</td>\n",
       "      <td>PMC9553214</td>\n",
       "      <td>10.7554/eLife.78074</td>\n",
       "      <td>https://dx.doi.org/10.7554/eLife.78074</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>247816</th>\n",
       "      <td>31677552</td>\n",
       "      <td>PMC6920132</td>\n",
       "      <td>10.1016/j.redox.2019.101359</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>181385</th>\n",
       "      <td>37172728</td>\n",
       "      <td>PMC10267520</td>\n",
       "      <td>10.1016/j.jbc.2023.104805</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>113274</th>\n",
       "      <td>35140216</td>\n",
       "      <td>PMC8828724</td>\n",
       "      <td>10.1038/s41467-022-28454-x</td>\n",
       "      <td>https://dx.doi.org/10.1038/s41467-022-28454-x</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38338</th>\n",
       "      <td>39041647</td>\n",
       "      <td>PMC11605269</td>\n",
       "      <td>10.1002/art.42958</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>156674</th>\n",
       "      <td>33804598</td>\n",
       "      <td>PMC8003772</td>\n",
       "      <td>10.3390/ijms22063164</td>\n",
       "      <td>https://dx.doi.org/10.3390/ijms22063164</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "execution_count": 17
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-15T01:18:03.264288Z",
     "start_time": "2025-03-15T01:18:02.990955Z"
    }
   },
   "cell_type": "code",
   "source": [
    "# drop NaN citing_publications_links\n",
    "GEO_df = GEO_df.dropna(subset=['citing_publications_links'])  # Remove NaNs\n",
    "print(len(GEO_df))"
   ],
   "id": "f31bbcfc2f3f9a0f",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "352982\n"
     ]
    }
   ],
   "execution_count": 18
  },
  {
   "metadata": {},
   "cell_type": "markdown",
   "source": "Union of the two dataframes",
   "id": "19b8d0be5dab663"
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-15T01:18:03.268109Z",
     "start_time": "2025-03-15T01:18:03.265462Z"
    }
   },
   "cell_type": "code",
   "source": [
    "print(f\"len filtered_df: {len(filtered_df)}\")\n",
    "print(f\"len GEO_df: {len(GEO_df)}\")"
   ],
   "id": "ee5f8c6573cd2ba7",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "len filtered_df: 38688\n",
      "len GEO_df: 352982\n"
     ]
    }
   ],
   "execution_count": 19
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-15T01:18:03.279816Z",
     "start_time": "2025-03-15T01:18:03.277446Z"
    }
   },
   "cell_type": "code",
   "source": [
    "column_mappings = { # Manually map columns\n",
    "    'accession': 'identifier',\n",
    "    'taxon': 'species',\n",
    "    'repository': 'entrytype',\n",
    "    'publication_date': 'pdat',\n",
    "    'pubmedids': 'PMID',\n",
    "}"
   ],
   "id": "391db62b2926b62c",
   "outputs": [],
   "execution_count": 20
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-15T01:18:03.458009Z",
     "start_time": "2025-03-15T01:18:03.283372Z"
    }
   },
   "cell_type": "code",
   "source": [
    "GEO_df = GEO_df.rename(columns=column_mappings)  # Rename columns\n",
    "GEO_df['repository'] = 'GEO'  # Add repository column\n",
    "filtered_df = pd.concat([filtered_df, GEO_df], axis=0, ignore_index=True)  # Concatenate dataframes"
   ],
   "id": "5f8032395e9a2dde",
   "outputs": [],
   "execution_count": 21
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-15T01:18:03.470525Z",
     "start_time": "2025-03-15T01:18:03.458978Z"
    }
   },
   "cell_type": "code",
   "source": "filtered_df.iloc[::10000].head(10)  # Show every 10000th row",
   "id": "625641be5d04ca60",
   "outputs": [
    {
     "data": {
      "text/plain": [
       "      identifier                                              title  \\\n",
       "0      PXD059466  Endothelial SHANK3 Regulates Tight Junctions i...   \n",
       "10000  PXD049417  Multiomics approach to decipher pathophysiolog...   \n",
       "20000  PXD031142  Marine Synechococcus sp. strain WH7803 shows s...   \n",
       "30000  PXD014667  Listeria monocytogenes exploits the MICOS comp...   \n",
       "40000  GSE268565  BMPR1A-mediated BMP signalling ensures correct...   \n",
       "50000  GSE240634  Identification of IKK Inhibitor by utilizing ...   \n",
       "60000  GSE245526  Replication-competent HIV-1 gRNA Constructs fo...   \n",
       "70000  GSE255988  Polygenic Risk for Alcohol Use Disorder Affect...   \n",
       "80000  GSE236263  MicroRNA-1 protects the endothelium in acute l...   \n",
       "90000  GSE230619  Gene expression data from human adipose-derive...   \n",
       "\n",
       "      repository                              species             instrument  \\\n",
       "0          PRIDE                         Mus musculus  Orbitrap Fusion Lumos   \n",
       "10000      PRIDE                         Mus musculus        Orbitrap Fusion   \n",
       "20000      PRIDE  Bacteria, Synechococcus sp. WH 7803        Orbitrap Fusion   \n",
       "30000      PRIDE                         Homo sapiens             Q Exactive   \n",
       "40000        GEO                         Mus musculus                    NaN   \n",
       "50000        GEO                         Homo sapiens                    NaN   \n",
       "60000        GEO       Human immunodeficiency virus 1                    NaN   \n",
       "70000        GEO                         Homo sapiens                    NaN   \n",
       "80000        GEO                         Homo sapiens                    NaN   \n",
       "90000        GEO                         Homo sapiens                    NaN   \n",
       "\n",
       "                           lab_head announce_date  \\\n",
       "0                       Il Hwan Kim    2025-02-07   \n",
       "10000                   Jiri Petrak    2024-07-03   \n",
       "20000  Jos Manuel Garca Fernndez    2022-07-11   \n",
       "30000               Pascale Cossart    2020-02-17   \n",
       "40000                           NaN           NaN   \n",
       "50000                           NaN           NaN   \n",
       "60000                           NaN           NaN   \n",
       "70000                           NaN           NaN   \n",
       "80000                           NaN           NaN   \n",
       "90000                           NaN           NaN   \n",
       "\n",
       "                                                keywords  \\\n",
       "0                           BioID2, SHANK3, bEnd3. Cells   \n",
       "10000  Cystathionine beta-synthase, homocystinuria, m...   \n",
       "20000  Orbitrap Fusion, Synechococcus WH7803, nitroge...   \n",
       "30000  Listeria monocytogenes, MICOS complex, Mic10, ...   \n",
       "40000                                                NaN   \n",
       "50000                                                NaN   \n",
       "60000                                                NaN   \n",
       "70000                                                NaN   \n",
       "80000                                                NaN   \n",
       "90000                                                NaN   \n",
       "\n",
       "                               citing_publications_links      PMID  ...  \\\n",
       "0          https://dx.doi.org/10.1038/S41467-025-56720-1      None  ...   \n",
       "10000  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...  38843767  ...   \n",
       "20000               https://dx.doi.org/10.6019/PXD031142      None  ...   \n",
       "30000  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7...  32019800  ...   \n",
       "40000  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...  39817676  ...   \n",
       "50000    https://dx.doi.org/10.1681/ASN.0000000000000460  39133556  ...   \n",
       "60000      https://dx.doi.org/10.1038/s41467-024-48228-x  38714682  ...   \n",
       "70000          https://dx.doi.org/10.1126/sciadv.ado5820  39514655  ...   \n",
       "80000      https://dx.doi.org/10.1172/jci.insight.164816  37737266  ...   \n",
       "90000  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...  37445596  ...   \n",
       "\n",
       "             pdat        suppfile  \\\n",
       "0             NaN             NaN   \n",
       "10000         NaN             NaN   \n",
       "20000         NaN             NaN   \n",
       "30000         NaN             NaN   \n",
       "40000  2025/01/16             TXT   \n",
       "50000  2024/08/15             TXT   \n",
       "60000  2024/04/15       TAB, XLSX   \n",
       "70000  2024/02/19             TSV   \n",
       "80000  2023/10/25             TXT   \n",
       "90000  2023/07/19  CEL, CHP, XLSX   \n",
       "\n",
       "                                                 samples  relations  \\\n",
       "0                                                    NaN        NaN   \n",
       "10000                                                NaN        NaN   \n",
       "20000                                                NaN        NaN   \n",
       "30000                                                NaN        NaN   \n",
       "40000  [{'accession': 'GSM8294327', 'title': 'E14.5 f...         []   \n",
       "50000  [{'accession': 'GSM7706552', 'title': 'Sample ...         []   \n",
       "60000  [{'accession': 'GSM7844529', 'title': 'NL4.3 C...         []   \n",
       "70000  [{'accession': 'GSM8083849', 'title': 'Microgl...         []   \n",
       "80000  [{'accession': 'GSM7524125', 'title': 'HPMEC c...         []   \n",
       "90000  [{'accession': 'GSM7230019', 'title': 'hASCs, ...         []   \n",
       "\n",
       "      extrelations n_samples projects  \\\n",
       "0              NaN       NaN      NaN   \n",
       "10000          NaN       NaN      NaN   \n",
       "20000          NaN       NaN      NaN   \n",
       "30000          NaN       NaN      NaN   \n",
       "40000           []       8.0       []   \n",
       "50000           []      18.0       []   \n",
       "60000           []      49.0       []   \n",
       "70000           []      18.0       []   \n",
       "80000           []       3.0       []   \n",
       "90000           []       9.0       []   \n",
       "\n",
       "                                                 ftplink geo2r    bioproject  \n",
       "0                                                    NaN   NaN           NaN  \n",
       "10000                                                NaN   NaN           NaN  \n",
       "20000                                                NaN   NaN           NaN  \n",
       "30000                                                NaN   NaN           NaN  \n",
       "40000  ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE268nn...    no  PRJNA1117811  \n",
       "50000  ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE240nn...   yes  PRJNA1004406  \n",
       "60000  ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE245nn...    no  PRJNA1028668  \n",
       "70000  ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE255nn...   yes  PRJNA1077355  \n",
       "80000  ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE236nn...   yes   PRJNA989591  \n",
       "90000  ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE230nn...   yes   PRJNA962011  \n",
       "\n",
       "[10 rows x 30 columns]"
      ],
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>identifier</th>\n",
       "      <th>title</th>\n",
       "      <th>repository</th>\n",
       "      <th>species</th>\n",
       "      <th>instrument</th>\n",
       "      <th>lab_head</th>\n",
       "      <th>announce_date</th>\n",
       "      <th>keywords</th>\n",
       "      <th>citing_publications_links</th>\n",
       "      <th>PMID</th>\n",
       "      <th>...</th>\n",
       "      <th>pdat</th>\n",
       "      <th>suppfile</th>\n",
       "      <th>samples</th>\n",
       "      <th>relations</th>\n",
       "      <th>extrelations</th>\n",
       "      <th>n_samples</th>\n",
       "      <th>projects</th>\n",
       "      <th>ftplink</th>\n",
       "      <th>geo2r</th>\n",
       "      <th>bioproject</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>PXD059466</td>\n",
       "      <td>Endothelial SHANK3 Regulates Tight Junctions i...</td>\n",
       "      <td>PRIDE</td>\n",
       "      <td>Mus musculus</td>\n",
       "      <td>Orbitrap Fusion Lumos</td>\n",
       "      <td>Il Hwan Kim</td>\n",
       "      <td>2025-02-07</td>\n",
       "      <td>BioID2, SHANK3, bEnd3. Cells</td>\n",
       "      <td>https://dx.doi.org/10.1038/S41467-025-56720-1</td>\n",
       "      <td>None</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10000</th>\n",
       "      <td>PXD049417</td>\n",
       "      <td>Multiomics approach to decipher pathophysiolog...</td>\n",
       "      <td>PRIDE</td>\n",
       "      <td>Mus musculus</td>\n",
       "      <td>Orbitrap Fusion</td>\n",
       "      <td>Jiri Petrak</td>\n",
       "      <td>2024-07-03</td>\n",
       "      <td>Cystathionine beta-synthase, homocystinuria, m...</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...</td>\n",
       "      <td>38843767</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20000</th>\n",
       "      <td>PXD031142</td>\n",
       "      <td>Marine Synechococcus sp. strain WH7803 shows s...</td>\n",
       "      <td>PRIDE</td>\n",
       "      <td>Bacteria, Synechococcus sp. WH 7803</td>\n",
       "      <td>Orbitrap Fusion</td>\n",
       "      <td>Jos Manuel Garca Fernndez</td>\n",
       "      <td>2022-07-11</td>\n",
       "      <td>Orbitrap Fusion, Synechococcus WH7803, nitroge...</td>\n",
       "      <td>https://dx.doi.org/10.6019/PXD031142</td>\n",
       "      <td>None</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30000</th>\n",
       "      <td>PXD014667</td>\n",
       "      <td>Listeria monocytogenes exploits the MICOS comp...</td>\n",
       "      <td>PRIDE</td>\n",
       "      <td>Homo sapiens</td>\n",
       "      <td>Q Exactive</td>\n",
       "      <td>Pascale Cossart</td>\n",
       "      <td>2020-02-17</td>\n",
       "      <td>Listeria monocytogenes, MICOS complex, Mic10, ...</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7...</td>\n",
       "      <td>32019800</td>\n",
       "      <td>...</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>40000</th>\n",
       "      <td>GSE268565</td>\n",
       "      <td>BMPR1A-mediated BMP signalling ensures correct...</td>\n",
       "      <td>GEO</td>\n",
       "      <td>Mus musculus</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...</td>\n",
       "      <td>39817676</td>\n",
       "      <td>...</td>\n",
       "      <td>2025/01/16</td>\n",
       "      <td>TXT</td>\n",
       "      <td>[{'accession': 'GSM8294327', 'title': 'E14.5 f...</td>\n",
       "      <td>[]</td>\n",
       "      <td>[]</td>\n",
       "      <td>8.0</td>\n",
       "      <td>[]</td>\n",
       "      <td>ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE268nn...</td>\n",
       "      <td>no</td>\n",
       "      <td>PRJNA1117811</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>50000</th>\n",
       "      <td>GSE240634</td>\n",
       "      <td>Identification of IKK Inhibitor by utilizing ...</td>\n",
       "      <td>GEO</td>\n",
       "      <td>Homo sapiens</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://dx.doi.org/10.1681/ASN.0000000000000460</td>\n",
       "      <td>39133556</td>\n",
       "      <td>...</td>\n",
       "      <td>2024/08/15</td>\n",
       "      <td>TXT</td>\n",
       "      <td>[{'accession': 'GSM7706552', 'title': 'Sample ...</td>\n",
       "      <td>[]</td>\n",
       "      <td>[]</td>\n",
       "      <td>18.0</td>\n",
       "      <td>[]</td>\n",
       "      <td>ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE240nn...</td>\n",
       "      <td>yes</td>\n",
       "      <td>PRJNA1004406</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>60000</th>\n",
       "      <td>GSE245526</td>\n",
       "      <td>Replication-competent HIV-1 gRNA Constructs fo...</td>\n",
       "      <td>GEO</td>\n",
       "      <td>Human immunodeficiency virus 1</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://dx.doi.org/10.1038/s41467-024-48228-x</td>\n",
       "      <td>38714682</td>\n",
       "      <td>...</td>\n",
       "      <td>2024/04/15</td>\n",
       "      <td>TAB, XLSX</td>\n",
       "      <td>[{'accession': 'GSM7844529', 'title': 'NL4.3 C...</td>\n",
       "      <td>[]</td>\n",
       "      <td>[]</td>\n",
       "      <td>49.0</td>\n",
       "      <td>[]</td>\n",
       "      <td>ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE245nn...</td>\n",
       "      <td>no</td>\n",
       "      <td>PRJNA1028668</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>70000</th>\n",
       "      <td>GSE255988</td>\n",
       "      <td>Polygenic Risk for Alcohol Use Disorder Affect...</td>\n",
       "      <td>GEO</td>\n",
       "      <td>Homo sapiens</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://dx.doi.org/10.1126/sciadv.ado5820</td>\n",
       "      <td>39514655</td>\n",
       "      <td>...</td>\n",
       "      <td>2024/02/19</td>\n",
       "      <td>TSV</td>\n",
       "      <td>[{'accession': 'GSM8083849', 'title': 'Microgl...</td>\n",
       "      <td>[]</td>\n",
       "      <td>[]</td>\n",
       "      <td>18.0</td>\n",
       "      <td>[]</td>\n",
       "      <td>ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE255nn...</td>\n",
       "      <td>yes</td>\n",
       "      <td>PRJNA1077355</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>80000</th>\n",
       "      <td>GSE236263</td>\n",
       "      <td>MicroRNA-1 protects the endothelium in acute l...</td>\n",
       "      <td>GEO</td>\n",
       "      <td>Homo sapiens</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://dx.doi.org/10.1172/jci.insight.164816</td>\n",
       "      <td>37737266</td>\n",
       "      <td>...</td>\n",
       "      <td>2023/10/25</td>\n",
       "      <td>TXT</td>\n",
       "      <td>[{'accession': 'GSM7524125', 'title': 'HPMEC c...</td>\n",
       "      <td>[]</td>\n",
       "      <td>[]</td>\n",
       "      <td>3.0</td>\n",
       "      <td>[]</td>\n",
       "      <td>ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE236nn...</td>\n",
       "      <td>yes</td>\n",
       "      <td>PRJNA989591</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>90000</th>\n",
       "      <td>GSE230619</td>\n",
       "      <td>Gene expression data from human adipose-derive...</td>\n",
       "      <td>GEO</td>\n",
       "      <td>Homo sapiens</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...</td>\n",
       "      <td>37445596</td>\n",
       "      <td>...</td>\n",
       "      <td>2023/07/19</td>\n",
       "      <td>CEL, CHP, XLSX</td>\n",
       "      <td>[{'accession': 'GSM7230019', 'title': 'hASCs, ...</td>\n",
       "      <td>[]</td>\n",
       "      <td>[]</td>\n",
       "      <td>9.0</td>\n",
       "      <td>[]</td>\n",
       "      <td>ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE230nn...</td>\n",
       "      <td>yes</td>\n",
       "      <td>PRJNA962011</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>10 rows  30 columns</p>\n",
       "</div>"
      ]
     },
     "execution_count": 22,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "execution_count": 22
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-15T01:18:08.330198Z",
     "start_time": "2025-03-15T01:18:03.471788Z"
    }
   },
   "cell_type": "code",
   "source": "filtered_df.to_parquet('exp_input/Table_datasets.parquet', index=False)",
   "id": "b003fb755240d6fd",
   "outputs": [],
   "execution_count": 23
  },
  {
   "metadata": {},
   "cell_type": "markdown",
   "source": "## Creating Publications Data Table",
   "id": "2f67a6fe415b390e"
  },
  {
   "metadata": {},
   "cell_type": "markdown",
   "source": "We start from ProteomeCentral and GEO table we created before",
   "id": "de60a7609166a466"
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-15T01:28:24.956123Z",
     "start_time": "2025-03-15T01:28:20.286399Z"
    }
   },
   "cell_type": "code",
   "source": "filtered_df = pd.read_parquet('exp_input/Table_datasets.parquet')",
   "id": "1ccae6d2f11d9bc6",
   "outputs": [],
   "execution_count": 34
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-15T01:28:24.974618Z",
     "start_time": "2025-03-15T01:28:24.959001Z"
    }
   },
   "cell_type": "code",
   "source": [
    "print(len(filtered_df))\n",
    "filtered_df.head(3)"
   ],
   "id": "8e35612b9cff33aa",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "391670\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "  identifier                                              title repository  \\\n",
       "0  PXD059466  Endothelial SHANK3 Regulates Tight Junctions i...      PRIDE   \n",
       "1  PXD051312  Systemic changes in early pregnancy in the mar...      PRIDE   \n",
       "2  PXD051312  Systemic changes in early pregnancy in the mar...      PRIDE   \n",
       "\n",
       "          species                            instrument     lab_head  \\\n",
       "0    Mus musculus                 Orbitrap Fusion Lumos  Il Hwan Kim   \n",
       "1  Equus caballus  Exactive Plus, Orbitrap Exploris 480    Dr Aleona   \n",
       "2  Equus caballus  Exactive Plus, Orbitrap Exploris 480    Dr Aleona   \n",
       "\n",
       "  announce_date                                           keywords  \\\n",
       "0    2025-02-07                       BioID2, SHANK3, bEnd3. Cells   \n",
       "1    2025-02-07  MRP, TTR, biomarker, early pregnancy loss, sec...   \n",
       "2    2025-02-07  MRP, TTR, biomarker, early pregnancy loss, sec...   \n",
       "\n",
       "                       citing_publications_links      PMID  ...  pdat  \\\n",
       "0  https://dx.doi.org/10.1038/S41467-025-56720-1      None  ...  None   \n",
       "1           https://dx.doi.org/10.6019/PXD051312      None  ...  None   \n",
       "2      https://dx.doi.org/10.1002/prca.202400095  39912552  ...  None   \n",
       "\n",
       "  suppfile  samples  relations extrelations n_samples projects  ftplink geo2r  \\\n",
       "0     None     None       None         None       NaN     None     None  None   \n",
       "1     None     None       None         None       NaN     None     None  None   \n",
       "2     None     None       None         None       NaN     None     None  None   \n",
       "\n",
       "  bioproject  \n",
       "0       None  \n",
       "1       None  \n",
       "2       None  \n",
       "\n",
       "[3 rows x 30 columns]"
      ],
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>identifier</th>\n",
       "      <th>title</th>\n",
       "      <th>repository</th>\n",
       "      <th>species</th>\n",
       "      <th>instrument</th>\n",
       "      <th>lab_head</th>\n",
       "      <th>announce_date</th>\n",
       "      <th>keywords</th>\n",
       "      <th>citing_publications_links</th>\n",
       "      <th>PMID</th>\n",
       "      <th>...</th>\n",
       "      <th>pdat</th>\n",
       "      <th>suppfile</th>\n",
       "      <th>samples</th>\n",
       "      <th>relations</th>\n",
       "      <th>extrelations</th>\n",
       "      <th>n_samples</th>\n",
       "      <th>projects</th>\n",
       "      <th>ftplink</th>\n",
       "      <th>geo2r</th>\n",
       "      <th>bioproject</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>PXD059466</td>\n",
       "      <td>Endothelial SHANK3 Regulates Tight Junctions i...</td>\n",
       "      <td>PRIDE</td>\n",
       "      <td>Mus musculus</td>\n",
       "      <td>Orbitrap Fusion Lumos</td>\n",
       "      <td>Il Hwan Kim</td>\n",
       "      <td>2025-02-07</td>\n",
       "      <td>BioID2, SHANK3, bEnd3. Cells</td>\n",
       "      <td>https://dx.doi.org/10.1038/S41467-025-56720-1</td>\n",
       "      <td>None</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>PXD051312</td>\n",
       "      <td>Systemic changes in early pregnancy in the mar...</td>\n",
       "      <td>PRIDE</td>\n",
       "      <td>Equus caballus</td>\n",
       "      <td>Exactive Plus, Orbitrap Exploris 480</td>\n",
       "      <td>Dr Aleona</td>\n",
       "      <td>2025-02-07</td>\n",
       "      <td>MRP, TTR, biomarker, early pregnancy loss, sec...</td>\n",
       "      <td>https://dx.doi.org/10.6019/PXD051312</td>\n",
       "      <td>None</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>PXD051312</td>\n",
       "      <td>Systemic changes in early pregnancy in the mar...</td>\n",
       "      <td>PRIDE</td>\n",
       "      <td>Equus caballus</td>\n",
       "      <td>Exactive Plus, Orbitrap Exploris 480</td>\n",
       "      <td>Dr Aleona</td>\n",
       "      <td>2025-02-07</td>\n",
       "      <td>MRP, TTR, biomarker, early pregnancy loss, sec...</td>\n",
       "      <td>https://dx.doi.org/10.1002/prca.202400095</td>\n",
       "      <td>39912552</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>3 rows  30 columns</p>\n",
       "</div>"
      ]
     },
     "execution_count": 35,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "execution_count": 35
  },
  {
   "metadata": {},
   "cell_type": "markdown",
   "source": "Some statistics on our publications data",
   "id": "c42c73bed9bf7b9e"
  },
  {
   "metadata": {},
   "cell_type": "markdown",
   "source": "look at the publications one by one",
   "id": "8211ccde7d9ad33d"
  },
  {
   "metadata": {},
   "cell_type": "code",
   "outputs": [],
   "execution_count": 37,
   "source": "filtered_df = filtered_df.explode('citing_publications_links')  # Split lists into rows",
   "id": "180211f0c5f0153f"
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-15T01:30:08.936967Z",
     "start_time": "2025-03-15T01:30:08.664063Z"
    }
   },
   "cell_type": "code",
   "source": [
    "df_publications = filtered_df[['citing_publications_links','DOI','PMID','PMCID']]\n",
    "\n",
    "df_publications = df_publications.sort_values(by='citing_publications_links', key=lambda x: x.str.contains('doi', na=False), ascending=False)\n",
    "\n",
    "df_publications = df_publications.drop_duplicates(subset=['PMCID', 'DOI'], keep='first')\n",
    "\n",
    "len(df_publications)"
   ],
   "id": "a087455a0b721b20",
   "outputs": [
    {
     "data": {
      "text/plain": [
       "126461"
      ]
     },
     "execution_count": 41,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "execution_count": 41
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-15T01:30:11.042466Z",
     "start_time": "2025-03-15T01:30:11.026929Z"
    }
   },
   "cell_type": "code",
   "source": [
    "print(len(df_publications))\n",
    "print(df_publications.dtypes)\n",
    "df_publications.head(3)"
   ],
   "id": "bf596436c1a933b9",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "126461\n",
      "citing_publications_links    object\n",
      "DOI                          object\n",
      "PMID                         object\n",
      "PMCID                        object\n",
      "dtype: object\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "                            citing_publications_links  \\\n",
       "0       https://dx.doi.org/10.1038/S41467-025-56720-1   \n",
       "313946           https://dx.doi.org/10.1038/srep10438   \n",
       "163224         https://dx.doi.org/10.1038/nature11247   \n",
       "\n",
       "                               DOI      PMID       PMCID  \n",
       "0       10.1038/S41467-025-56720-1      None        None  \n",
       "313946           10.1038/srep10438  26039282  PMC4453922  \n",
       "163224         10.1038/nature11247  22955616  PMC3439153  "
      ],
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>citing_publications_links</th>\n",
       "      <th>DOI</th>\n",
       "      <th>PMID</th>\n",
       "      <th>PMCID</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>https://dx.doi.org/10.1038/S41467-025-56720-1</td>\n",
       "      <td>10.1038/S41467-025-56720-1</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>313946</th>\n",
       "      <td>https://dx.doi.org/10.1038/srep10438</td>\n",
       "      <td>10.1038/srep10438</td>\n",
       "      <td>26039282</td>\n",
       "      <td>PMC4453922</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>163224</th>\n",
       "      <td>https://dx.doi.org/10.1038/nature11247</td>\n",
       "      <td>10.1038/nature11247</td>\n",
       "      <td>22955616</td>\n",
       "      <td>PMC3439153</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ]
     },
     "execution_count": 42,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "execution_count": 42
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-15T01:32:04.650259Z",
     "start_time": "2025-03-15T01:32:04.492626Z"
    }
   },
   "cell_type": "code",
   "source": "df_publications.astype(str).to_parquet('exp_input/Table_publications.parquet', index=False)",
   "id": "1ae9d7b082f4ce4",
   "outputs": [],
   "execution_count": 44
  },
  {
   "metadata": {},
   "cell_type": "markdown",
   "source": [
    "## Create the ground truth table\n",
    "\n",
    "each row is a record of a dataset citation in a doi"
   ],
   "id": "2237e2ae0e37158c"
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-15T01:32:12.716845Z",
     "start_time": "2025-03-15T01:32:08.904303Z"
    }
   },
   "cell_type": "code",
   "source": [
    "Publications_table = pd.read_parquet('exp_input/Table_publications.parquet')\n",
    "Datasets_table = pd.read_parquet('exp_input/Table_datasets.parquet')"
   ],
   "id": "373aac28c765113e",
   "outputs": [],
   "execution_count": 45
  },
  {
   "metadata": {},
   "cell_type": "markdown",
   "source": "We iterate over all the datasets and use their publication urls to find the corresponding dois. Since not all papers had dois, we will create a Publication Identifier table",
   "id": "be8e1be86188741c"
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-15T01:34:55.952761Z",
     "start_time": "2025-03-15T01:34:55.890476Z"
    }
   },
   "cell_type": "code",
   "source": "Publication_Dataset_Citations_Ground_Truth = Datasets_table[['identifier','repository','citing_publications_links']].copy()",
   "id": "b156bf9643883482",
   "outputs": [],
   "execution_count": 47
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-15T01:37:38.284603Z",
     "start_time": "2025-03-15T01:37:38.089535Z"
    }
   },
   "cell_type": "code",
   "source": "Publication_Dataset_Citations_Ground_Truth.to_parquet('exp_input/dataset_citation_records_Table.parquet', index=False)",
   "id": "16b6bb5274c87852",
   "outputs": [],
   "execution_count": 49
  },
  {
   "metadata": {},
   "cell_type": "code",
   "source": "",
   "id": "397bc5d96372c11",
   "outputs": [],
   "execution_count": null
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 2
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython2",
   "version": "2.7.6"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
