{
 "cells": [
  {
   "cell_type": "code",
   "id": "initial_id",
   "metadata": {
    "collapsed": true,
    "ExecuteTime": {
     "end_time": "2025-03-12T16:55:14.325711Z",
     "start_time": "2025-03-12T16:55:04.330995Z"
    }
   },
   "source": [
    "import pandas as pd\n",
    "import re\n",
    "import json\n",
    "import requests\n",
    "from xml.etree import ElementTree\n",
    "import time\n",
    "from dotenv import load_dotenv\n",
    "import ast  # To safely evaluate string representation of a list\n",
    "import bdikit as bdi\n",
    "import requests\n",
    "import os"
   ],
   "outputs": [
    {
     "data": {
      "text/html": [
       "<script type=\"esms-options\">{\"shimMode\": true}</script><style>*[data-root-id],\n",
       "*[data-root-id] > * {\n",
       "  box-sizing: border-box;\n",
       "  font-family: var(--jp-ui-font-family);\n",
       "  font-size: var(--jp-ui-font-size1);\n",
       "  color: var(--vscode-editor-foreground, var(--jp-ui-font-color1));\n",
       "}\n",
       "\n",
       "/* Override VSCode background color */\n",
       ".cell-output-ipywidget-background:has(\n",
       "    > .cell-output-ipywidget-background > .lm-Widget > *[data-root-id]\n",
       "  ),\n",
       ".cell-output-ipywidget-background:has(> .lm-Widget > *[data-root-id]) {\n",
       "  background-color: transparent !important;\n",
       "}\n",
       "</style>"
      ]
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "application/javascript": "(function(root) {\n  function now() {\n    return new Date();\n  }\n\n  const force = true;\n  const py_version = '3.6.3'.replace('rc', '-rc.').replace('.dev', '-dev.');\n  const reloading = false;\n  const Bokeh = root.Bokeh;\n\n  // Set a timeout for this load but only if we are not already initializing\n  if (typeof (root._bokeh_timeout) === \"undefined\" || (force || !root._bokeh_is_initializing)) {\n    root._bokeh_timeout = Date.now() + 5000;\n    root._bokeh_failed_load = false;\n  }\n\n  function run_callbacks() {\n    try {\n      root._bokeh_onload_callbacks.forEach(function(callback) {\n        if (callback != null)\n          callback();\n      });\n    } finally {\n      delete root._bokeh_onload_callbacks;\n    }\n    console.debug(\"Bokeh: all callbacks have finished\");\n  }\n\n  function load_libs(css_urls, js_urls, js_modules, js_exports, callback) {\n    if (css_urls == null) css_urls = [];\n    if (js_urls == null) js_urls = [];\n    if (js_modules == null) js_modules = [];\n    if (js_exports == null) js_exports = {};\n\n    root._bokeh_onload_callbacks.push(callback);\n\n    if (root._bokeh_is_loading > 0) {\n      // Don't load bokeh if it is still initializing\n      console.debug(\"Bokeh: BokehJS is being loaded, scheduling callback at\", now());\n      return null;\n    } else if (js_urls.length === 0 && js_modules.length === 0 && Object.keys(js_exports).length === 0) {\n      // There is nothing to load\n      run_callbacks();\n      return null;\n    }\n\n    function on_load() {\n      root._bokeh_is_loading--;\n      if (root._bokeh_is_loading === 0) {\n        console.debug(\"Bokeh: all BokehJS libraries/stylesheets loaded\");\n        run_callbacks()\n      }\n    }\n    window._bokeh_on_load = on_load\n\n    function on_error(e) {\n      const src_el = e.srcElement\n      console.error(\"failed to load \" + (src_el.href || src_el.src));\n    }\n\n    const skip = [];\n    if (window.requirejs) {\n      window.requirejs.config({'packages': {}, 'paths': {'tabulator': 'https://cdn.jsdelivr.net/npm/tabulator-tables@6.3.1/dist/js/tabulator.min', 'moment': 'https://cdn.jsdelivr.net/npm/luxon/build/global/luxon.min'}, 'shim': {}});\n      require([\"tabulator\"], function(Tabulator) {\n        window.Tabulator = Tabulator\n        on_load()\n      })\n      require([\"moment\"], function(moment) {\n        window.moment = moment\n        on_load()\n      })\n      root._bokeh_is_loading = css_urls.length + 2;\n    } else {\n      root._bokeh_is_loading = css_urls.length + js_urls.length + js_modules.length + Object.keys(js_exports).length;\n    }\n\n    const existing_stylesheets = []\n    const links = document.getElementsByTagName('link')\n    for (let i = 0; i < links.length; i++) {\n      const link = links[i]\n      if (link.href != null) {\n        existing_stylesheets.push(link.href)\n      }\n    }\n    for (let i = 0; i < css_urls.length; i++) {\n      const url = css_urls[i];\n      const escaped = encodeURI(url)\n      if (existing_stylesheets.indexOf(escaped) !== -1) {\n        on_load()\n        continue;\n      }\n      const element = document.createElement(\"link\");\n      element.onload = on_load;\n      element.onerror = on_error;\n      element.rel = \"stylesheet\";\n      element.type = \"text/css\";\n      element.href = url;\n      console.debug(\"Bokeh: injecting link tag for BokehJS stylesheet: \", url);\n      document.body.appendChild(element);\n    }    if (((window.Tabulator !== undefined) && (!(window.Tabulator instanceof HTMLElement))) || window.requirejs) {\n      var urls = ['https://cdn.holoviz.org/panel/1.6.1/dist/bundled/datatabulator/tabulator-tables@6.3.1/dist/js/tabulator.min.js'];\n      for (var i = 0; i < urls.length; i++) {\n        skip.push(encodeURI(urls[i]))\n      }\n    }    if (((window.moment !== undefined) && (!(window.moment instanceof HTMLElement))) || window.requirejs) {\n      var urls = ['https://cdn.holoviz.org/panel/1.6.1/dist/bundled/datatabulator/luxon/build/global/luxon.min.js'];\n      for (var i = 0; i < urls.length; i++) {\n        skip.push(encodeURI(urls[i]))\n      }\n    }    var existing_scripts = []\n    const scripts = document.getElementsByTagName('script')\n    for (let i = 0; i < scripts.length; i++) {\n      var script = scripts[i]\n      if (script.src != null) {\n        existing_scripts.push(script.src)\n      }\n    }\n    for (let i = 0; i < js_urls.length; i++) {\n      const url = js_urls[i];\n      const escaped = encodeURI(url)\n      if (skip.indexOf(escaped) !== -1 || existing_scripts.indexOf(escaped) !== -1) {\n        if (!window.requirejs) {\n          on_load();\n        }\n        continue;\n      }\n      const element = document.createElement('script');\n      element.onload = on_load;\n      element.onerror = on_error;\n      element.async = false;\n      element.src = url;\n      console.debug(\"Bokeh: injecting script tag for BokehJS library: \", url);\n      document.head.appendChild(element);\n    }\n    for (let i = 0; i < js_modules.length; i++) {\n      const url = js_modules[i];\n      const escaped = encodeURI(url)\n      if (skip.indexOf(escaped) !== -1 || existing_scripts.indexOf(escaped) !== -1) {\n        if (!window.requirejs) {\n          on_load();\n        }\n        continue;\n      }\n      var element = document.createElement('script');\n      element.onload = on_load;\n      element.onerror = on_error;\n      element.async = false;\n      element.src = url;\n      element.type = \"module\";\n      console.debug(\"Bokeh: injecting script tag for BokehJS library: \", url);\n      document.head.appendChild(element);\n    }\n    for (const name in js_exports) {\n      const url = js_exports[name];\n      const escaped = encodeURI(url)\n      if (skip.indexOf(escaped) >= 0 || root[name] != null) {\n        if (!window.requirejs) {\n          on_load();\n        }\n        continue;\n      }\n      var element = document.createElement('script');\n      element.onerror = on_error;\n      element.async = false;\n      element.type = \"module\";\n      console.debug(\"Bokeh: injecting script tag for BokehJS library: \", url);\n      element.textContent = `\n      import ${name} from \"${url}\"\n      window.${name} = ${name}\n      window._bokeh_on_load()\n      `\n      document.head.appendChild(element);\n    }\n    if (!js_urls.length && !js_modules.length) {\n      on_load()\n    }\n  };\n\n  function inject_raw_css(css) {\n    const element = document.createElement(\"style\");\n    element.appendChild(document.createTextNode(css));\n    document.body.appendChild(element);\n  }\n\n  const js_urls = [\"https://cdn.holoviz.org/panel/1.6.1/dist/bundled/reactiveesm/es-module-shims@^1.10.0/dist/es-module-shims.min.js\", \"https://cdn.holoviz.org/panel/1.6.1/dist/bundled/datatabulator/tabulator-tables@6.3.1/dist/js/tabulator.min.js\", \"https://cdn.holoviz.org/panel/1.6.1/dist/bundled/datatabulator/luxon/build/global/luxon.min.js\", \"https://cdn.bokeh.org/bokeh/release/bokeh-3.6.3.min.js\", \"https://cdn.bokeh.org/bokeh/release/bokeh-gl-3.6.3.min.js\", \"https://cdn.bokeh.org/bokeh/release/bokeh-widgets-3.6.3.min.js\", \"https://cdn.bokeh.org/bokeh/release/bokeh-tables-3.6.3.min.js\", \"https://cdn.holoviz.org/panel/1.6.1/dist/panel.min.js\"];\n  const js_modules = [];\n  const js_exports = {};\n  const css_urls = [\"https://cdn.holoviz.org/panel/1.6.1/dist/bundled/datatabulator/tabulator-tables@6.3.1/dist/css/tabulator_simple.min.css?v=1.6.1\"];\n  const inline_js = [    function(Bokeh) {\n      Bokeh.set_log_level(\"info\");\n    },\nfunction(Bokeh) {} // ensure no trailing comma for IE\n  ];\n\n  function run_inline_js() {\n    if ((root.Bokeh !== undefined) || (force === true)) {\n      for (let i = 0; i < inline_js.length; i++) {\n        try {\n          inline_js[i].call(root, root.Bokeh);\n        } catch(e) {\n          if (!reloading) {\n            throw e;\n          }\n        }\n      }\n      // Cache old bokeh versions\n      if (Bokeh != undefined && !reloading) {\n        var NewBokeh = root.Bokeh;\n        if (Bokeh.versions === undefined) {\n          Bokeh.versions = new Map();\n        }\n        if (NewBokeh.version !== Bokeh.version) {\n          Bokeh.versions.set(NewBokeh.version, NewBokeh)\n        }\n        root.Bokeh = Bokeh;\n      }\n    } else if (Date.now() < root._bokeh_timeout) {\n      setTimeout(run_inline_js, 100);\n    } else if (!root._bokeh_failed_load) {\n      console.log(\"Bokeh: BokehJS failed to load within specified timeout.\");\n      root._bokeh_failed_load = true;\n    }\n    root._bokeh_is_initializing = false\n  }\n\n  function load_or_wait() {\n    // Implement a backoff loop that tries to ensure we do not load multiple\n    // versions of Bokeh and its dependencies at the same time.\n    // In recent versions we use the root._bokeh_is_initializing flag\n    // to determine whether there is an ongoing attempt to initialize\n    // bokeh, however for backward compatibility we also try to ensure\n    // that we do not start loading a newer (Panel>=1.0 and Bokeh>3) version\n    // before older versions are fully initialized.\n    if (root._bokeh_is_initializing && Date.now() > root._bokeh_timeout) {\n      // If the timeout and bokeh was not successfully loaded we reset\n      // everything and try loading again\n      root._bokeh_timeout = Date.now() + 5000;\n      root._bokeh_is_initializing = false;\n      root._bokeh_onload_callbacks = undefined;\n      root._bokeh_is_loading = 0\n      console.log(\"Bokeh: BokehJS was loaded multiple times but one version failed to initialize.\");\n      load_or_wait();\n    } else if (root._bokeh_is_initializing || (typeof root._bokeh_is_initializing === \"undefined\" && root._bokeh_onload_callbacks !== undefined)) {\n      setTimeout(load_or_wait, 100);\n    } else {\n      root._bokeh_is_initializing = true\n      root._bokeh_onload_callbacks = []\n      const bokeh_loaded = root.Bokeh != null && (root.Bokeh.version === py_version || (root.Bokeh.versions !== undefined && root.Bokeh.versions.has(py_version)));\n      if (!reloading && !bokeh_loaded) {\n        if (root.Bokeh) {\n          root.Bokeh = undefined;\n        }\n        console.debug(\"Bokeh: BokehJS not loaded, scheduling load and callback at\", now());\n      }\n      load_libs(css_urls, js_urls, js_modules, js_exports, function() {\n        console.debug(\"Bokeh: BokehJS plotting callback run at\", now());\n        run_inline_js();\n      });\n    }\n  }\n  // Give older versions of the autoload script a head-start to ensure\n  // they initialize before we start loading newer version.\n  setTimeout(load_or_wait, 100)\n}(window));",
      "application/vnd.holoviews_load.v0+json": "(function(root) {\n  function now() {\n    return new Date();\n  }\n\n  const force = true;\n  const py_version = '3.6.3'.replace('rc', '-rc.').replace('.dev', '-dev.');\n  const reloading = false;\n  const Bokeh = root.Bokeh;\n\n  // Set a timeout for this load but only if we are not already initializing\n  if (typeof (root._bokeh_timeout) === \"undefined\" || (force || !root._bokeh_is_initializing)) {\n    root._bokeh_timeout = Date.now() + 5000;\n    root._bokeh_failed_load = false;\n  }\n\n  function run_callbacks() {\n    try {\n      root._bokeh_onload_callbacks.forEach(function(callback) {\n        if (callback != null)\n          callback();\n      });\n    } finally {\n      delete root._bokeh_onload_callbacks;\n    }\n    console.debug(\"Bokeh: all callbacks have finished\");\n  }\n\n  function load_libs(css_urls, js_urls, js_modules, js_exports, callback) {\n    if (css_urls == null) css_urls = [];\n    if (js_urls == null) js_urls = [];\n    if (js_modules == null) js_modules = [];\n    if (js_exports == null) js_exports = {};\n\n    root._bokeh_onload_callbacks.push(callback);\n\n    if (root._bokeh_is_loading > 0) {\n      // Don't load bokeh if it is still initializing\n      console.debug(\"Bokeh: BokehJS is being loaded, scheduling callback at\", now());\n      return null;\n    } else if (js_urls.length === 0 && js_modules.length === 0 && Object.keys(js_exports).length === 0) {\n      // There is nothing to load\n      run_callbacks();\n      return null;\n    }\n\n    function on_load() {\n      root._bokeh_is_loading--;\n      if (root._bokeh_is_loading === 0) {\n        console.debug(\"Bokeh: all BokehJS libraries/stylesheets loaded\");\n        run_callbacks()\n      }\n    }\n    window._bokeh_on_load = on_load\n\n    function on_error(e) {\n      const src_el = e.srcElement\n      console.error(\"failed to load \" + (src_el.href || src_el.src));\n    }\n\n    const skip = [];\n    if (window.requirejs) {\n      window.requirejs.config({'packages': {}, 'paths': {'tabulator': 'https://cdn.jsdelivr.net/npm/tabulator-tables@6.3.1/dist/js/tabulator.min', 'moment': 'https://cdn.jsdelivr.net/npm/luxon/build/global/luxon.min'}, 'shim': {}});\n      require([\"tabulator\"], function(Tabulator) {\n        window.Tabulator = Tabulator\n        on_load()\n      })\n      require([\"moment\"], function(moment) {\n        window.moment = moment\n        on_load()\n      })\n      root._bokeh_is_loading = css_urls.length + 2;\n    } else {\n      root._bokeh_is_loading = css_urls.length + js_urls.length + js_modules.length + Object.keys(js_exports).length;\n    }\n\n    const existing_stylesheets = []\n    const links = document.getElementsByTagName('link')\n    for (let i = 0; i < links.length; i++) {\n      const link = links[i]\n      if (link.href != null) {\n        existing_stylesheets.push(link.href)\n      }\n    }\n    for (let i = 0; i < css_urls.length; i++) {\n      const url = css_urls[i];\n      const escaped = encodeURI(url)\n      if (existing_stylesheets.indexOf(escaped) !== -1) {\n        on_load()\n        continue;\n      }\n      const element = document.createElement(\"link\");\n      element.onload = on_load;\n      element.onerror = on_error;\n      element.rel = \"stylesheet\";\n      element.type = \"text/css\";\n      element.href = url;\n      console.debug(\"Bokeh: injecting link tag for BokehJS stylesheet: \", url);\n      document.body.appendChild(element);\n    }    if (((window.Tabulator !== undefined) && (!(window.Tabulator instanceof HTMLElement))) || window.requirejs) {\n      var urls = ['https://cdn.holoviz.org/panel/1.6.1/dist/bundled/datatabulator/tabulator-tables@6.3.1/dist/js/tabulator.min.js'];\n      for (var i = 0; i < urls.length; i++) {\n        skip.push(encodeURI(urls[i]))\n      }\n    }    if (((window.moment !== undefined) && (!(window.moment instanceof HTMLElement))) || window.requirejs) {\n      var urls = ['https://cdn.holoviz.org/panel/1.6.1/dist/bundled/datatabulator/luxon/build/global/luxon.min.js'];\n      for (var i = 0; i < urls.length; i++) {\n        skip.push(encodeURI(urls[i]))\n      }\n    }    var existing_scripts = []\n    const scripts = document.getElementsByTagName('script')\n    for (let i = 0; i < scripts.length; i++) {\n      var script = scripts[i]\n      if (script.src != null) {\n        existing_scripts.push(script.src)\n      }\n    }\n    for (let i = 0; i < js_urls.length; i++) {\n      const url = js_urls[i];\n      const escaped = encodeURI(url)\n      if (skip.indexOf(escaped) !== -1 || existing_scripts.indexOf(escaped) !== -1) {\n        if (!window.requirejs) {\n          on_load();\n        }\n        continue;\n      }\n      const element = document.createElement('script');\n      element.onload = on_load;\n      element.onerror = on_error;\n      element.async = false;\n      element.src = url;\n      console.debug(\"Bokeh: injecting script tag for BokehJS library: \", url);\n      document.head.appendChild(element);\n    }\n    for (let i = 0; i < js_modules.length; i++) {\n      const url = js_modules[i];\n      const escaped = encodeURI(url)\n      if (skip.indexOf(escaped) !== -1 || existing_scripts.indexOf(escaped) !== -1) {\n        if (!window.requirejs) {\n          on_load();\n        }\n        continue;\n      }\n      var element = document.createElement('script');\n      element.onload = on_load;\n      element.onerror = on_error;\n      element.async = false;\n      element.src = url;\n      element.type = \"module\";\n      console.debug(\"Bokeh: injecting script tag for BokehJS library: \", url);\n      document.head.appendChild(element);\n    }\n    for (const name in js_exports) {\n      const url = js_exports[name];\n      const escaped = encodeURI(url)\n      if (skip.indexOf(escaped) >= 0 || root[name] != null) {\n        if (!window.requirejs) {\n          on_load();\n        }\n        continue;\n      }\n      var element = document.createElement('script');\n      element.onerror = on_error;\n      element.async = false;\n      element.type = \"module\";\n      console.debug(\"Bokeh: injecting script tag for BokehJS library: \", url);\n      element.textContent = `\n      import ${name} from \"${url}\"\n      window.${name} = ${name}\n      window._bokeh_on_load()\n      `\n      document.head.appendChild(element);\n    }\n    if (!js_urls.length && !js_modules.length) {\n      on_load()\n    }\n  };\n\n  function inject_raw_css(css) {\n    const element = document.createElement(\"style\");\n    element.appendChild(document.createTextNode(css));\n    document.body.appendChild(element);\n  }\n\n  const js_urls = [\"https://cdn.holoviz.org/panel/1.6.1/dist/bundled/reactiveesm/es-module-shims@^1.10.0/dist/es-module-shims.min.js\", \"https://cdn.holoviz.org/panel/1.6.1/dist/bundled/datatabulator/tabulator-tables@6.3.1/dist/js/tabulator.min.js\", \"https://cdn.holoviz.org/panel/1.6.1/dist/bundled/datatabulator/luxon/build/global/luxon.min.js\", \"https://cdn.bokeh.org/bokeh/release/bokeh-3.6.3.min.js\", \"https://cdn.bokeh.org/bokeh/release/bokeh-gl-3.6.3.min.js\", \"https://cdn.bokeh.org/bokeh/release/bokeh-widgets-3.6.3.min.js\", \"https://cdn.bokeh.org/bokeh/release/bokeh-tables-3.6.3.min.js\", \"https://cdn.holoviz.org/panel/1.6.1/dist/panel.min.js\"];\n  const js_modules = [];\n  const js_exports = {};\n  const css_urls = [\"https://cdn.holoviz.org/panel/1.6.1/dist/bundled/datatabulator/tabulator-tables@6.3.1/dist/css/tabulator_simple.min.css?v=1.6.1\"];\n  const inline_js = [    function(Bokeh) {\n      Bokeh.set_log_level(\"info\");\n    },\nfunction(Bokeh) {} // ensure no trailing comma for IE\n  ];\n\n  function run_inline_js() {\n    if ((root.Bokeh !== undefined) || (force === true)) {\n      for (let i = 0; i < inline_js.length; i++) {\n        try {\n          inline_js[i].call(root, root.Bokeh);\n        } catch(e) {\n          if (!reloading) {\n            throw e;\n          }\n        }\n      }\n      // Cache old bokeh versions\n      if (Bokeh != undefined && !reloading) {\n        var NewBokeh = root.Bokeh;\n        if (Bokeh.versions === undefined) {\n          Bokeh.versions = new Map();\n        }\n        if (NewBokeh.version !== Bokeh.version) {\n          Bokeh.versions.set(NewBokeh.version, NewBokeh)\n        }\n        root.Bokeh = Bokeh;\n      }\n    } else if (Date.now() < root._bokeh_timeout) {\n      setTimeout(run_inline_js, 100);\n    } else if (!root._bokeh_failed_load) {\n      console.log(\"Bokeh: BokehJS failed to load within specified timeout.\");\n      root._bokeh_failed_load = true;\n    }\n    root._bokeh_is_initializing = false\n  }\n\n  function load_or_wait() {\n    // Implement a backoff loop that tries to ensure we do not load multiple\n    // versions of Bokeh and its dependencies at the same time.\n    // In recent versions we use the root._bokeh_is_initializing flag\n    // to determine whether there is an ongoing attempt to initialize\n    // bokeh, however for backward compatibility we also try to ensure\n    // that we do not start loading a newer (Panel>=1.0 and Bokeh>3) version\n    // before older versions are fully initialized.\n    if (root._bokeh_is_initializing && Date.now() > root._bokeh_timeout) {\n      // If the timeout and bokeh was not successfully loaded we reset\n      // everything and try loading again\n      root._bokeh_timeout = Date.now() + 5000;\n      root._bokeh_is_initializing = false;\n      root._bokeh_onload_callbacks = undefined;\n      root._bokeh_is_loading = 0\n      console.log(\"Bokeh: BokehJS was loaded multiple times but one version failed to initialize.\");\n      load_or_wait();\n    } else if (root._bokeh_is_initializing || (typeof root._bokeh_is_initializing === \"undefined\" && root._bokeh_onload_callbacks !== undefined)) {\n      setTimeout(load_or_wait, 100);\n    } else {\n      root._bokeh_is_initializing = true\n      root._bokeh_onload_callbacks = []\n      const bokeh_loaded = root.Bokeh != null && (root.Bokeh.version === py_version || (root.Bokeh.versions !== undefined && root.Bokeh.versions.has(py_version)));\n      if (!reloading && !bokeh_loaded) {\n        if (root.Bokeh) {\n          root.Bokeh = undefined;\n        }\n        console.debug(\"Bokeh: BokehJS not loaded, scheduling load and callback at\", now());\n      }\n      load_libs(css_urls, js_urls, js_modules, js_exports, function() {\n        console.debug(\"Bokeh: BokehJS plotting callback run at\", now());\n        run_inline_js();\n      });\n    }\n  }\n  // Give older versions of the autoload script a head-start to ensure\n  // they initialize before we start loading newer version.\n  setTimeout(load_or_wait, 100)\n}(window));"
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "application/vnd.holoviews_load.v0+json": "\nif ((window.PyViz === undefined) || (window.PyViz instanceof HTMLElement)) {\n  window.PyViz = {comms: {}, comm_status:{}, kernels:{}, receivers: {}, plot_index: []}\n}\n\n\n    function JupyterCommManager() {\n    }\n\n    JupyterCommManager.prototype.register_target = function(plot_id, comm_id, msg_handler) {\n      if (window.comm_manager || ((window.Jupyter !== undefined) && (Jupyter.notebook.kernel != null))) {\n        var comm_manager = window.comm_manager || Jupyter.notebook.kernel.comm_manager;\n        comm_manager.register_target(comm_id, function(comm) {\n          comm.on_msg(msg_handler);\n        });\n      } else if ((plot_id in window.PyViz.kernels) && (window.PyViz.kernels[plot_id])) {\n        window.PyViz.kernels[plot_id].registerCommTarget(comm_id, function(comm) {\n          comm.onMsg = msg_handler;\n        });\n      } else if (typeof google != 'undefined' && google.colab.kernel != null) {\n        google.colab.kernel.comms.registerTarget(comm_id, (comm) => {\n          var messages = comm.messages[Symbol.asyncIterator]();\n          function processIteratorResult(result) {\n            var message = result.value;\n            console.log(message)\n            var content = {data: message.data, comm_id};\n            var buffers = []\n            for (var buffer of message.buffers || []) {\n              buffers.push(new DataView(buffer))\n            }\n            var metadata = message.metadata || {};\n            var msg = {content, buffers, metadata}\n            msg_handler(msg);\n            return messages.next().then(processIteratorResult);\n          }\n          return messages.next().then(processIteratorResult);\n        })\n      }\n    }\n\n    JupyterCommManager.prototype.get_client_comm = function(plot_id, comm_id, msg_handler) {\n      if (comm_id in window.PyViz.comms) {\n        return window.PyViz.comms[comm_id];\n      } else if (window.comm_manager || ((window.Jupyter !== undefined) && (Jupyter.notebook.kernel != null))) {\n        var comm_manager = window.comm_manager || Jupyter.notebook.kernel.comm_manager;\n        var comm = comm_manager.new_comm(comm_id, {}, {}, {}, comm_id);\n        if (msg_handler) {\n          comm.on_msg(msg_handler);\n        }\n      } else if ((plot_id in window.PyViz.kernels) && (window.PyViz.kernels[plot_id])) {\n        var comm = window.PyViz.kernels[plot_id].connectToComm(comm_id);\n        comm.open();\n        if (msg_handler) {\n          comm.onMsg = msg_handler;\n        }\n      } else if (typeof google != 'undefined' && google.colab.kernel != null) {\n        var comm_promise = google.colab.kernel.comms.open(comm_id)\n        comm_promise.then((comm) => {\n          window.PyViz.comms[comm_id] = comm;\n          if (msg_handler) {\n            var messages = comm.messages[Symbol.asyncIterator]();\n            function processIteratorResult(result) {\n              var message = result.value;\n              var content = {data: message.data};\n              var metadata = message.metadata || {comm_id};\n              var msg = {content, metadata}\n              msg_handler(msg);\n              return messages.next().then(processIteratorResult);\n            }\n            return messages.next().then(processIteratorResult);\n          }\n        })\n        var sendClosure = (data, metadata, buffers, disposeOnDone) => {\n          return comm_promise.then((comm) => {\n            comm.send(data, metadata, buffers, disposeOnDone);\n          });\n        };\n        var comm = {\n          send: sendClosure\n        };\n      }\n      window.PyViz.comms[comm_id] = comm;\n      return comm;\n    }\n    window.PyViz.comm_manager = new JupyterCommManager();\n    \n\n\nvar JS_MIME_TYPE = 'application/javascript';\nvar HTML_MIME_TYPE = 'text/html';\nvar EXEC_MIME_TYPE = 'application/vnd.holoviews_exec.v0+json';\nvar CLASS_NAME = 'output';\n\n/**\n * Render data to the DOM node\n */\nfunction render(props, node) {\n  var div = document.createElement(\"div\");\n  var script = document.createElement(\"script\");\n  node.appendChild(div);\n  node.appendChild(script);\n}\n\n/**\n * Handle when a new output is added\n */\nfunction handle_add_output(event, handle) {\n  var output_area = handle.output_area;\n  var output = handle.output;\n  if ((output.data == undefined) || (!output.data.hasOwnProperty(EXEC_MIME_TYPE))) {\n    return\n  }\n  var id = output.metadata[EXEC_MIME_TYPE][\"id\"];\n  var toinsert = output_area.element.find(\".\" + CLASS_NAME.split(' ')[0]);\n  if (id !== undefined) {\n    var nchildren = toinsert.length;\n    var html_node = toinsert[nchildren-1].children[0];\n    html_node.innerHTML = output.data[HTML_MIME_TYPE];\n    var scripts = [];\n    var nodelist = html_node.querySelectorAll(\"script\");\n    for (var i in nodelist) {\n      if (nodelist.hasOwnProperty(i)) {\n        scripts.push(nodelist[i])\n      }\n    }\n\n    scripts.forEach( function (oldScript) {\n      var newScript = document.createElement(\"script\");\n      var attrs = [];\n      var nodemap = oldScript.attributes;\n      for (var j in nodemap) {\n        if (nodemap.hasOwnProperty(j)) {\n          attrs.push(nodemap[j])\n        }\n      }\n      attrs.forEach(function(attr) { newScript.setAttribute(attr.name, attr.value) });\n      newScript.appendChild(document.createTextNode(oldScript.innerHTML));\n      oldScript.parentNode.replaceChild(newScript, oldScript);\n    });\n    if (JS_MIME_TYPE in output.data) {\n      toinsert[nchildren-1].children[1].textContent = output.data[JS_MIME_TYPE];\n    }\n    output_area._hv_plot_id = id;\n    if ((window.Bokeh !== undefined) && (id in Bokeh.index)) {\n      window.PyViz.plot_index[id] = Bokeh.index[id];\n    } else {\n      window.PyViz.plot_index[id] = null;\n    }\n  } else if (output.metadata[EXEC_MIME_TYPE][\"server_id\"] !== undefined) {\n    var bk_div = document.createElement(\"div\");\n    bk_div.innerHTML = output.data[HTML_MIME_TYPE];\n    var script_attrs = bk_div.children[0].attributes;\n    for (var i = 0; i < script_attrs.length; i++) {\n      toinsert[toinsert.length - 1].childNodes[1].setAttribute(script_attrs[i].name, script_attrs[i].value);\n    }\n    // store reference to server id on output_area\n    output_area._bokeh_server_id = output.metadata[EXEC_MIME_TYPE][\"server_id\"];\n  }\n}\n\n/**\n * Handle when an output is cleared or removed\n */\nfunction handle_clear_output(event, handle) {\n  var id = handle.cell.output_area._hv_plot_id;\n  var server_id = handle.cell.output_area._bokeh_server_id;\n  if (((id === undefined) || !(id in PyViz.plot_index)) && (server_id !== undefined)) { return; }\n  var comm = window.PyViz.comm_manager.get_client_comm(\"hv-extension-comm\", \"hv-extension-comm\", function () {});\n  if (server_id !== null) {\n    comm.send({event_type: 'server_delete', 'id': server_id});\n    return;\n  } else if (comm !== null) {\n    comm.send({event_type: 'delete', 'id': id});\n  }\n  delete PyViz.plot_index[id];\n  if ((window.Bokeh !== undefined) & (id in window.Bokeh.index)) {\n    var doc = window.Bokeh.index[id].model.document\n    doc.clear();\n    const i = window.Bokeh.documents.indexOf(doc);\n    if (i > -1) {\n      window.Bokeh.documents.splice(i, 1);\n    }\n  }\n}\n\n/**\n * Handle kernel restart event\n */\nfunction handle_kernel_cleanup(event, handle) {\n  delete PyViz.comms[\"hv-extension-comm\"];\n  window.PyViz.plot_index = {}\n}\n\n/**\n * Handle update_display_data messages\n */\nfunction handle_update_output(event, handle) {\n  handle_clear_output(event, {cell: {output_area: handle.output_area}})\n  handle_add_output(event, handle)\n}\n\nfunction register_renderer(events, OutputArea) {\n  function append_mime(data, metadata, element) {\n    // create a DOM node to render to\n    var toinsert = this.create_output_subarea(\n    metadata,\n    CLASS_NAME,\n    EXEC_MIME_TYPE\n    );\n    this.keyboard_manager.register_events(toinsert);\n    // Render to node\n    var props = {data: data, metadata: metadata[EXEC_MIME_TYPE]};\n    render(props, toinsert[0]);\n    element.append(toinsert);\n    return toinsert\n  }\n\n  events.on('output_added.OutputArea', handle_add_output);\n  events.on('output_updated.OutputArea', handle_update_output);\n  events.on('clear_output.CodeCell', handle_clear_output);\n  events.on('delete.Cell', handle_clear_output);\n  events.on('kernel_ready.Kernel', handle_kernel_cleanup);\n\n  OutputArea.prototype.register_mime_type(EXEC_MIME_TYPE, append_mime, {\n    safe: true,\n    index: 0\n  });\n}\n\nif (window.Jupyter !== undefined) {\n  try {\n    var events = require('base/js/events');\n    var OutputArea = require('notebook/js/outputarea').OutputArea;\n    if (OutputArea.prototype.mime_types().indexOf(EXEC_MIME_TYPE) == -1) {\n      register_renderer(events, OutputArea);\n    }\n  } catch(err) {\n  }\n}\n",
      "application/javascript": "\nif ((window.PyViz === undefined) || (window.PyViz instanceof HTMLElement)) {\n  window.PyViz = {comms: {}, comm_status:{}, kernels:{}, receivers: {}, plot_index: []}\n}\n\n\n    function JupyterCommManager() {\n    }\n\n    JupyterCommManager.prototype.register_target = function(plot_id, comm_id, msg_handler) {\n      if (window.comm_manager || ((window.Jupyter !== undefined) && (Jupyter.notebook.kernel != null))) {\n        var comm_manager = window.comm_manager || Jupyter.notebook.kernel.comm_manager;\n        comm_manager.register_target(comm_id, function(comm) {\n          comm.on_msg(msg_handler);\n        });\n      } else if ((plot_id in window.PyViz.kernels) && (window.PyViz.kernels[plot_id])) {\n        window.PyViz.kernels[plot_id].registerCommTarget(comm_id, function(comm) {\n          comm.onMsg = msg_handler;\n        });\n      } else if (typeof google != 'undefined' && google.colab.kernel != null) {\n        google.colab.kernel.comms.registerTarget(comm_id, (comm) => {\n          var messages = comm.messages[Symbol.asyncIterator]();\n          function processIteratorResult(result) {\n            var message = result.value;\n            console.log(message)\n            var content = {data: message.data, comm_id};\n            var buffers = []\n            for (var buffer of message.buffers || []) {\n              buffers.push(new DataView(buffer))\n            }\n            var metadata = message.metadata || {};\n            var msg = {content, buffers, metadata}\n            msg_handler(msg);\n            return messages.next().then(processIteratorResult);\n          }\n          return messages.next().then(processIteratorResult);\n        })\n      }\n    }\n\n    JupyterCommManager.prototype.get_client_comm = function(plot_id, comm_id, msg_handler) {\n      if (comm_id in window.PyViz.comms) {\n        return window.PyViz.comms[comm_id];\n      } else if (window.comm_manager || ((window.Jupyter !== undefined) && (Jupyter.notebook.kernel != null))) {\n        var comm_manager = window.comm_manager || Jupyter.notebook.kernel.comm_manager;\n        var comm = comm_manager.new_comm(comm_id, {}, {}, {}, comm_id);\n        if (msg_handler) {\n          comm.on_msg(msg_handler);\n        }\n      } else if ((plot_id in window.PyViz.kernels) && (window.PyViz.kernels[plot_id])) {\n        var comm = window.PyViz.kernels[plot_id].connectToComm(comm_id);\n        comm.open();\n        if (msg_handler) {\n          comm.onMsg = msg_handler;\n        }\n      } else if (typeof google != 'undefined' && google.colab.kernel != null) {\n        var comm_promise = google.colab.kernel.comms.open(comm_id)\n        comm_promise.then((comm) => {\n          window.PyViz.comms[comm_id] = comm;\n          if (msg_handler) {\n            var messages = comm.messages[Symbol.asyncIterator]();\n            function processIteratorResult(result) {\n              var message = result.value;\n              var content = {data: message.data};\n              var metadata = message.metadata || {comm_id};\n              var msg = {content, metadata}\n              msg_handler(msg);\n              return messages.next().then(processIteratorResult);\n            }\n            return messages.next().then(processIteratorResult);\n          }\n        })\n        var sendClosure = (data, metadata, buffers, disposeOnDone) => {\n          return comm_promise.then((comm) => {\n            comm.send(data, metadata, buffers, disposeOnDone);\n          });\n        };\n        var comm = {\n          send: sendClosure\n        };\n      }\n      window.PyViz.comms[comm_id] = comm;\n      return comm;\n    }\n    window.PyViz.comm_manager = new JupyterCommManager();\n    \n\n\nvar JS_MIME_TYPE = 'application/javascript';\nvar HTML_MIME_TYPE = 'text/html';\nvar EXEC_MIME_TYPE = 'application/vnd.holoviews_exec.v0+json';\nvar CLASS_NAME = 'output';\n\n/**\n * Render data to the DOM node\n */\nfunction render(props, node) {\n  var div = document.createElement(\"div\");\n  var script = document.createElement(\"script\");\n  node.appendChild(div);\n  node.appendChild(script);\n}\n\n/**\n * Handle when a new output is added\n */\nfunction handle_add_output(event, handle) {\n  var output_area = handle.output_area;\n  var output = handle.output;\n  if ((output.data == undefined) || (!output.data.hasOwnProperty(EXEC_MIME_TYPE))) {\n    return\n  }\n  var id = output.metadata[EXEC_MIME_TYPE][\"id\"];\n  var toinsert = output_area.element.find(\".\" + CLASS_NAME.split(' ')[0]);\n  if (id !== undefined) {\n    var nchildren = toinsert.length;\n    var html_node = toinsert[nchildren-1].children[0];\n    html_node.innerHTML = output.data[HTML_MIME_TYPE];\n    var scripts = [];\n    var nodelist = html_node.querySelectorAll(\"script\");\n    for (var i in nodelist) {\n      if (nodelist.hasOwnProperty(i)) {\n        scripts.push(nodelist[i])\n      }\n    }\n\n    scripts.forEach( function (oldScript) {\n      var newScript = document.createElement(\"script\");\n      var attrs = [];\n      var nodemap = oldScript.attributes;\n      for (var j in nodemap) {\n        if (nodemap.hasOwnProperty(j)) {\n          attrs.push(nodemap[j])\n        }\n      }\n      attrs.forEach(function(attr) { newScript.setAttribute(attr.name, attr.value) });\n      newScript.appendChild(document.createTextNode(oldScript.innerHTML));\n      oldScript.parentNode.replaceChild(newScript, oldScript);\n    });\n    if (JS_MIME_TYPE in output.data) {\n      toinsert[nchildren-1].children[1].textContent = output.data[JS_MIME_TYPE];\n    }\n    output_area._hv_plot_id = id;\n    if ((window.Bokeh !== undefined) && (id in Bokeh.index)) {\n      window.PyViz.plot_index[id] = Bokeh.index[id];\n    } else {\n      window.PyViz.plot_index[id] = null;\n    }\n  } else if (output.metadata[EXEC_MIME_TYPE][\"server_id\"] !== undefined) {\n    var bk_div = document.createElement(\"div\");\n    bk_div.innerHTML = output.data[HTML_MIME_TYPE];\n    var script_attrs = bk_div.children[0].attributes;\n    for (var i = 0; i < script_attrs.length; i++) {\n      toinsert[toinsert.length - 1].childNodes[1].setAttribute(script_attrs[i].name, script_attrs[i].value);\n    }\n    // store reference to server id on output_area\n    output_area._bokeh_server_id = output.metadata[EXEC_MIME_TYPE][\"server_id\"];\n  }\n}\n\n/**\n * Handle when an output is cleared or removed\n */\nfunction handle_clear_output(event, handle) {\n  var id = handle.cell.output_area._hv_plot_id;\n  var server_id = handle.cell.output_area._bokeh_server_id;\n  if (((id === undefined) || !(id in PyViz.plot_index)) && (server_id !== undefined)) { return; }\n  var comm = window.PyViz.comm_manager.get_client_comm(\"hv-extension-comm\", \"hv-extension-comm\", function () {});\n  if (server_id !== null) {\n    comm.send({event_type: 'server_delete', 'id': server_id});\n    return;\n  } else if (comm !== null) {\n    comm.send({event_type: 'delete', 'id': id});\n  }\n  delete PyViz.plot_index[id];\n  if ((window.Bokeh !== undefined) & (id in window.Bokeh.index)) {\n    var doc = window.Bokeh.index[id].model.document\n    doc.clear();\n    const i = window.Bokeh.documents.indexOf(doc);\n    if (i > -1) {\n      window.Bokeh.documents.splice(i, 1);\n    }\n  }\n}\n\n/**\n * Handle kernel restart event\n */\nfunction handle_kernel_cleanup(event, handle) {\n  delete PyViz.comms[\"hv-extension-comm\"];\n  window.PyViz.plot_index = {}\n}\n\n/**\n * Handle update_display_data messages\n */\nfunction handle_update_output(event, handle) {\n  handle_clear_output(event, {cell: {output_area: handle.output_area}})\n  handle_add_output(event, handle)\n}\n\nfunction register_renderer(events, OutputArea) {\n  function append_mime(data, metadata, element) {\n    // create a DOM node to render to\n    var toinsert = this.create_output_subarea(\n    metadata,\n    CLASS_NAME,\n    EXEC_MIME_TYPE\n    );\n    this.keyboard_manager.register_events(toinsert);\n    // Render to node\n    var props = {data: data, metadata: metadata[EXEC_MIME_TYPE]};\n    render(props, toinsert[0]);\n    element.append(toinsert);\n    return toinsert\n  }\n\n  events.on('output_added.OutputArea', handle_add_output);\n  events.on('output_updated.OutputArea', handle_update_output);\n  events.on('clear_output.CodeCell', handle_clear_output);\n  events.on('delete.Cell', handle_clear_output);\n  events.on('kernel_ready.Kernel', handle_kernel_cleanup);\n\n  OutputArea.prototype.register_mime_type(EXEC_MIME_TYPE, append_mime, {\n    safe: true,\n    index: 0\n  });\n}\n\nif (window.Jupyter !== undefined) {\n  try {\n    var events = require('base/js/events');\n    var OutputArea = require('notebook/js/outputarea').OutputArea;\n    if (OutputArea.prototype.mime_types().indexOf(EXEC_MIME_TYPE) == -1) {\n      register_renderer(events, OutputArea);\n    }\n  } catch(err) {\n  }\n}\n"
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "data": {
      "text/html": [
       "<div id='0b1a8ebb-8376-444d-adf5-3a57a8316264'>\n",
       "  <div id=\"eae0260a-daf7-4327-aaed-ce8f4e48723d\" data-root-id=\"0b1a8ebb-8376-444d-adf5-3a57a8316264\" style=\"display: contents;\"></div>\n",
       "</div>\n",
       "<script type=\"application/javascript\">(function(root) {\n",
       "  var docs_json = {\"5501c1ce-3ca0-4d7d-8723-bf3f909cbc7f\":{\"version\":\"3.6.3\",\"title\":\"Bokeh Application\",\"roots\":[{\"type\":\"object\",\"name\":\"panel.models.browser.BrowserInfo\",\"id\":\"0b1a8ebb-8376-444d-adf5-3a57a8316264\"},{\"type\":\"object\",\"name\":\"panel.models.comm_manager.CommManager\",\"id\":\"9b21ad87-6332-4c0b-9a4f-186c0999a7bb\",\"attributes\":{\"plot_id\":\"0b1a8ebb-8376-444d-adf5-3a57a8316264\",\"comm_id\":\"8de29739d9954407b751d447ceed8ba8\",\"client_comm_id\":\"464d6b51fffd43d78a42566d2270d428\"}}],\"defs\":[{\"type\":\"model\",\"name\":\"ReactiveHTML1\"},{\"type\":\"model\",\"name\":\"FlexBox1\",\"properties\":[{\"name\":\"align_content\",\"kind\":\"Any\",\"default\":\"flex-start\"},{\"name\":\"align_items\",\"kind\":\"Any\",\"default\":\"flex-start\"},{\"name\":\"flex_direction\",\"kind\":\"Any\",\"default\":\"row\"},{\"name\":\"flex_wrap\",\"kind\":\"Any\",\"default\":\"wrap\"},{\"name\":\"gap\",\"kind\":\"Any\",\"default\":\"\"},{\"name\":\"justify_content\",\"kind\":\"Any\",\"default\":\"flex-start\"}]},{\"type\":\"model\",\"name\":\"FloatPanel1\",\"properties\":[{\"name\":\"config\",\"kind\":\"Any\",\"default\":{\"type\":\"map\"}},{\"name\":\"contained\",\"kind\":\"Any\",\"default\":true},{\"name\":\"position\",\"kind\":\"Any\",\"default\":\"right-top\"},{\"name\":\"offsetx\",\"kind\":\"Any\",\"default\":null},{\"name\":\"offsety\",\"kind\":\"Any\",\"default\":null},{\"name\":\"theme\",\"kind\":\"Any\",\"default\":\"primary\"},{\"name\":\"status\",\"kind\":\"Any\",\"default\":\"normalized\"}]},{\"type\":\"model\",\"name\":\"GridStack1\",\"properties\":[{\"name\":\"mode\",\"kind\":\"Any\",\"default\":\"warn\"},{\"name\":\"ncols\",\"kind\":\"Any\",\"default\":null},{\"name\":\"nrows\",\"kind\":\"Any\",\"default\":null},{\"name\":\"allow_resize\",\"kind\":\"Any\",\"default\":true},{\"name\":\"allow_drag\",\"kind\":\"Any\",\"default\":true},{\"name\":\"state\",\"kind\":\"Any\",\"default\":[]}]},{\"type\":\"model\",\"name\":\"drag1\",\"properties\":[{\"name\":\"slider_width\",\"kind\":\"Any\",\"default\":5},{\"name\":\"slider_color\",\"kind\":\"Any\",\"default\":\"black\"},{\"name\":\"value\",\"kind\":\"Any\",\"default\":50}]},{\"type\":\"model\",\"name\":\"click1\",\"properties\":[{\"name\":\"terminal_output\",\"kind\":\"Any\",\"default\":\"\"},{\"name\":\"debug_name\",\"kind\":\"Any\",\"default\":\"\"},{\"name\":\"clears\",\"kind\":\"Any\",\"default\":0}]},{\"type\":\"model\",\"name\":\"FastWrapper1\",\"properties\":[{\"name\":\"object\",\"kind\":\"Any\",\"default\":null},{\"name\":\"style\",\"kind\":\"Any\",\"default\":null}]},{\"type\":\"model\",\"name\":\"NotificationAreaBase1\",\"properties\":[{\"name\":\"js_events\",\"kind\":\"Any\",\"default\":{\"type\":\"map\"}},{\"name\":\"position\",\"kind\":\"Any\",\"default\":\"bottom-right\"},{\"name\":\"_clear\",\"kind\":\"Any\",\"default\":0}]},{\"type\":\"model\",\"name\":\"NotificationArea1\",\"properties\":[{\"name\":\"js_events\",\"kind\":\"Any\",\"default\":{\"type\":\"map\"}},{\"name\":\"notifications\",\"kind\":\"Any\",\"default\":[]},{\"name\":\"position\",\"kind\":\"Any\",\"default\":\"bottom-right\"},{\"name\":\"_clear\",\"kind\":\"Any\",\"default\":0},{\"name\":\"types\",\"kind\":\"Any\",\"default\":[{\"type\":\"map\",\"entries\":[[\"type\",\"warning\"],[\"background\",\"#ffc107\"],[\"icon\",{\"type\":\"map\",\"entries\":[[\"className\",\"fas fa-exclamation-triangle\"],[\"tagName\",\"i\"],[\"color\",\"white\"]]}]]},{\"type\":\"map\",\"entries\":[[\"type\",\"info\"],[\"background\",\"#007bff\"],[\"icon\",{\"type\":\"map\",\"entries\":[[\"className\",\"fas fa-info-circle\"],[\"tagName\",\"i\"],[\"color\",\"white\"]]}]]}]}]},{\"type\":\"model\",\"name\":\"Notification\",\"properties\":[{\"name\":\"background\",\"kind\":\"Any\",\"default\":null},{\"name\":\"duration\",\"kind\":\"Any\",\"default\":3000},{\"name\":\"icon\",\"kind\":\"Any\",\"default\":null},{\"name\":\"message\",\"kind\":\"Any\",\"default\":\"\"},{\"name\":\"notification_type\",\"kind\":\"Any\",\"default\":null},{\"name\":\"_rendered\",\"kind\":\"Any\",\"default\":false},{\"name\":\"_destroyed\",\"kind\":\"Any\",\"default\":false}]},{\"type\":\"model\",\"name\":\"TemplateActions1\",\"properties\":[{\"name\":\"open_modal\",\"kind\":\"Any\",\"default\":0},{\"name\":\"close_modal\",\"kind\":\"Any\",\"default\":0}]},{\"type\":\"model\",\"name\":\"BootstrapTemplateActions1\",\"properties\":[{\"name\":\"open_modal\",\"kind\":\"Any\",\"default\":0},{\"name\":\"close_modal\",\"kind\":\"Any\",\"default\":0}]},{\"type\":\"model\",\"name\":\"TemplateEditor1\",\"properties\":[{\"name\":\"layout\",\"kind\":\"Any\",\"default\":[]}]},{\"type\":\"model\",\"name\":\"MaterialTemplateActions1\",\"properties\":[{\"name\":\"open_modal\",\"kind\":\"Any\",\"default\":0},{\"name\":\"close_modal\",\"kind\":\"Any\",\"default\":0}]},{\"type\":\"model\",\"name\":\"ReactiveESM1\",\"properties\":[{\"name\":\"esm_constants\",\"kind\":\"Any\",\"default\":{\"type\":\"map\"}}]},{\"type\":\"model\",\"name\":\"JSComponent1\",\"properties\":[{\"name\":\"esm_constants\",\"kind\":\"Any\",\"default\":{\"type\":\"map\"}}]},{\"type\":\"model\",\"name\":\"ReactComponent1\",\"properties\":[{\"name\":\"esm_constants\",\"kind\":\"Any\",\"default\":{\"type\":\"map\"}}]},{\"type\":\"model\",\"name\":\"AnyWidgetComponent1\",\"properties\":[{\"name\":\"esm_constants\",\"kind\":\"Any\",\"default\":{\"type\":\"map\"}}]},{\"type\":\"model\",\"name\":\"request_value1\",\"properties\":[{\"name\":\"fill\",\"kind\":\"Any\",\"default\":\"none\"},{\"name\":\"_synced\",\"kind\":\"Any\",\"default\":null},{\"name\":\"_request_sync\",\"kind\":\"Any\",\"default\":0}]}]}};\n",
       "  var render_items = [{\"docid\":\"5501c1ce-3ca0-4d7d-8723-bf3f909cbc7f\",\"roots\":{\"0b1a8ebb-8376-444d-adf5-3a57a8316264\":\"eae0260a-daf7-4327-aaed-ce8f4e48723d\"},\"root_ids\":[\"0b1a8ebb-8376-444d-adf5-3a57a8316264\"]}];\n",
       "  var docs = Object.values(docs_json)\n",
       "  if (!docs) {\n",
       "    return\n",
       "  }\n",
       "  const py_version = docs[0].version.replace('rc', '-rc.').replace('.dev', '-dev.')\n",
       "  async function embed_document(root) {\n",
       "    var Bokeh = get_bokeh(root)\n",
       "    await Bokeh.embed.embed_items_notebook(docs_json, render_items);\n",
       "    for (const render_item of render_items) {\n",
       "      for (const root_id of render_item.root_ids) {\n",
       "\tconst id_el = document.getElementById(root_id)\n",
       "\tif (id_el.children.length && id_el.children[0].hasAttribute('data-root-id')) {\n",
       "\t  const root_el = id_el.children[0]\n",
       "\t  root_el.id = root_el.id + '-rendered'\n",
       "\t  for (const child of root_el.children) {\n",
       "            // Ensure JupyterLab does not capture keyboard shortcuts\n",
       "            // see: https://jupyterlab.readthedocs.io/en/4.1.x/extension/notebook.html#keyboard-interaction-model\n",
       "\t    child.setAttribute('data-lm-suppress-shortcuts', 'true')\n",
       "\t  }\n",
       "\t}\n",
       "      }\n",
       "    }\n",
       "  }\n",
       "  function get_bokeh(root) {\n",
       "    if (root.Bokeh === undefined) {\n",
       "      return null\n",
       "    } else if (root.Bokeh.version !== py_version) {\n",
       "      if (root.Bokeh.versions === undefined || !root.Bokeh.versions.has(py_version)) {\n",
       "\treturn null\n",
       "      }\n",
       "      return root.Bokeh.versions.get(py_version);\n",
       "    } else if (root.Bokeh.version === py_version) {\n",
       "      return root.Bokeh\n",
       "    }\n",
       "    return null\n",
       "  }\n",
       "  function is_loaded(root) {\n",
       "    var Bokeh = get_bokeh(root)\n",
       "    return (Bokeh != null && Bokeh.Panel !== undefined && ( root.Tabulator !== undefined) && ( root.Tabulator !== undefined))\n",
       "  }\n",
       "  if (is_loaded(root)) {\n",
       "    embed_document(root);\n",
       "  } else {\n",
       "    var attempts = 0;\n",
       "    var timer = setInterval(function(root) {\n",
       "      if (is_loaded(root)) {\n",
       "        clearInterval(timer);\n",
       "        embed_document(root);\n",
       "      } else if (document.readyState == \"complete\") {\n",
       "        attempts++;\n",
       "        if (attempts > 200) {\n",
       "          clearInterval(timer);\n",
       "\t  var Bokeh = get_bokeh(root)\n",
       "\t  if (Bokeh == null || Bokeh.Panel == null) {\n",
       "            console.warn(\"Panel: ERROR: Unable to run Panel code because Bokeh or Panel library is missing\");\n",
       "\t  } else {\n",
       "\t    console.warn(\"Panel: WARNING: Attempting to render but not all required libraries could be resolved.\")\n",
       "\t    embed_document(root)\n",
       "\t  }\n",
       "        }\n",
       "      }\n",
       "    }, 25, root)\n",
       "  }\n",
       "})(window);</script>"
      ],
      "application/vnd.holoviews_exec.v0+json": ""
     },
     "metadata": {
      "application/vnd.holoviews_exec.v0+json": {
       "id": "0b1a8ebb-8376-444d-adf5-3a57a8316264"
      }
     },
     "output_type": "display_data"
    }
   ],
   "execution_count": 1
  },
  {
   "metadata": {},
   "cell_type": "markdown",
   "source": [
    "# Extract Ground Truth\n",
    "\n",
    "This Notebook takes as input the Export File from ProteomeCentral: https://proteomecentral.proteomexchange.org/: proteomexchange_search.tsv\n",
    "\n",
    "\n",
    "The final output will be a json file with ground truth public information on datasets"
   ],
   "id": "e88265f76ea59f2d"
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-12T16:55:14.682533Z",
     "start_time": "2025-03-12T16:55:14.328261Z"
    }
   },
   "cell_type": "code",
   "source": [
    "export_df = pd.read_csv('exp_input/proteomexchange_search.tsv', sep='\\t')\n",
    "print(f\"number of dataset ids in export dataframe from ProteomeCentral: {len(export_df)}\")\n",
    "datasets_df = export_df[['publication','identifier','repository','title','keywords']]"
   ],
   "id": "b43764462e1e7210",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "number of dataset ids in export dataframe from ProteomeCentral: 40137\n"
     ]
    }
   ],
   "execution_count": 2
  },
  {
   "metadata": {},
   "cell_type": "markdown",
   "source": [
    "## publications values frequencies\n",
    "\n",
    "to get an idea of the most frequent values in df publication, for all the dataset ids from the source file"
   ],
   "id": "216c36ad08a8ccc8"
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-12T16:55:14.706037Z",
     "start_time": "2025-03-12T16:55:14.683828Z"
    }
   },
   "cell_type": "code",
   "source": "export_df['publication'].value_counts()",
   "id": "3ea4110aabcad351",
   "outputs": [
    {
     "data": {
      "text/plain": [
       "publication\n",
       "Dataset with its publication pending                                                                                                                                                                      11719\n",
       "no publication                                                                                                                                                                                             2550\n",
       "<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/35084980\" target=\"_blank\">Melani et al. (2022)</a>                                                                                                               56\n",
       "<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/28267743\" target=\"_blank\">Matsumoto et al. (2017)</a>                                                                                                            28\n",
       "<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/28071820\" target=\"_blank\">Kreutz et al. (2017)</a>                                                                                                               18\n",
       "                                                                                                                                                                                                          ...  \n",
       "<a href=\"https://dx.doi.org/10.3389/FPLS.2023.1285847\" target=\"_blank\">10.3389/FPLS.2023.1285847</a>                                                                                                          1\n",
       "<a href=\"https://dx.doi.org/10.1083/JCB.201902050\" target=\"_blank\">10.1083/JCB.201902050</a>                                                                                                                  1\n",
       "<a href=\"https://dx.doi.org/10.1038/S41598-023-47767-5\" target=\"_blank\">10.1038/S41598-023-47767-5</a>                                                                                                        1\n",
       "<a href=\"https://dx.doi.org/10.15252/embr.202357741\" target=\"_blank\">10.15252/embr.202357741</a>; <a href=\"https://www.ncbi.nlm.nih.gov/pubmed/38009565\" target=\"_blank\">Due Tankmar et al. (2023)</a>        1\n",
       "<a href=\"http://www.ncbi.nlm.nih.gov/pubmed/22874012\">Vaudel et al. (2012)</a>                                                                                                                                1\n",
       "Name: count, Length: 23462, dtype: int64"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "execution_count": 3
  },
  {
   "metadata": {},
   "cell_type": "markdown",
   "source": "Clean the `publication` column by filtering out unwanted values like `Dataset with its publication pending`, `no publication`. Remove rows with unwanted values",
   "id": "6c9abdec1b840be7"
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-12T16:55:14.716583Z",
     "start_time": "2025-03-12T16:55:14.707457Z"
    }
   },
   "cell_type": "code",
   "source": [
    "filtered_df = export_df[~export_df['publication'].isin([\"Dataset with its publication pending\", \"no publication\"])]\n",
    "print(len(filtered_df),'\\n',filtered_df.columns)"
   ],
   "id": "d2d0fb59a01d4418",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "25868 \n",
      " Index(['identifier', 'title', 'repository', 'species', 'instrument',\n",
      "       'publication', 'lab_head', 'announce_date', 'keywords'],\n",
      "      dtype='object')\n"
     ]
    }
   ],
   "execution_count": 4
  },
  {
   "metadata": {},
   "cell_type": "markdown",
   "source": "## PMID and DOI Mappings extraction functions for future use",
   "id": "c6acc3b0fccd10f6"
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-12T16:55:14.731174Z",
     "start_time": "2025-03-12T16:55:14.719909Z"
    }
   },
   "cell_type": "code",
   "source": [
    "def PMID_to_doi(pmid):\n",
    "    if pmid in pmid_doi_mapping:\n",
    "        return pmid_doi_mapping[pmid]\n",
    "    \n",
    "    else:\n",
    "        print(f\"PMID {pmid} not found in mapping file. Querying API...\")\n",
    "    \n",
    "        base = \"https://www.ncbi.nlm.nih.gov/pmc/utils/idconv/v1.0/\"\n",
    "        params = {\"tool\": \"mytool\", \"email\": \"myemail@example.com\", \"ids\": pmid, \"format\": \"json\"}    \n",
    "        \n",
    "        response = requests.get(base, params=params)\n",
    "        \n",
    "        if response.status_code == 200:\n",
    "            data = response.json()\n",
    "            records = data.get(\"records\", [])\n",
    "            if records and \"doi\" in records[0]:\n",
    "                print(f\"PMID: {pmid}, DOI: {records[0]['doi']}\")\n",
    "                pmid_doi_mapping[pmid] = records[0][\"doi\"]  # Store in mapping\n",
    "                return records[0][\"doi\"]\n",
    "            else:\n",
    "                print(f\"PMID: {pmid}, DOI: xxxx\")  # No doi found\n",
    "                return None  # No doi found\n",
    "        else:\n",
    "            print(f\"API request failed for PMID {pmid}\")\n",
    "            return None  # Request failed\n",
    "\n",
    "def url_to_doi(url):\n",
    "    if \"dx.doi.org\" in url:\n",
    "        match = re.search(r'dx.doi.org/([a-zA-Z0-9./-]+)', url)\n",
    "        if match:\n",
    "            return match.group(1)\n",
    "        else:\n",
    "            print(f\"DOI not found in {url}\")\n",
    "            return None\n",
    "    elif \"pubmed\" in url:\n",
    "        match = re.search(r'pubmed/([0-9]+)', url)\n",
    "        if match:\n",
    "            return PMID_to_doi(match.group(1))\n",
    "        else:\n",
    "            print(f\"PMID not found in {url}\")\n",
    "            return None        \n",
    "\n",
    "def PMID_to_url(pmid):\n",
    "    base_url = \"https://www.ncbi.nlm.nih.gov/pubmed/\"\n",
    "    return base_url + str(pmid)\n",
    "\n",
    "def url_to_pmid(url):\n",
    "    if \"pubmed\" in url:\n",
    "        match = re.search(r'pubmed/([0-9]+)', url)\n",
    "        if match:\n",
    "            return match.group(1)\n",
    "        else:\n",
    "            print(f\"PMID not found in {url}\")\n",
    "            return None\n",
    "    else:\n",
    "        print(f\"URL does not contain a PMID: {url}\") if 'doi' not in url else None\n",
    "        return None\n",
    "    \n",
    "def batch_PMID_to_doi(pmids, batch_size=100):\n",
    "    base_url = \"https://www.ncbi.nlm.nih.gov/pmc/utils/idconv/v1.0/\"\n",
    "    results = {}\n",
    "\n",
    "    for i in range(0, len(pmids), batch_size):\n",
    "        progress = i / len(pmids) * 100\n",
    "        print(f\"Processing batch {i}-{i+batch_size} ({progress:.2f}%)\")\n",
    "        batch = pmids[i:i+batch_size]  # Get a batch of PMIDs\n",
    "        params = {\"tool\": \"mytool\", \"email\": \"myemail@example.com\", \"ids\": \",\".join(batch), \"format\": \"json\"}\n",
    "        \n",
    "        response = requests.get(base_url, params=params)\n",
    "        \n",
    "        if response.status_code == 200:\n",
    "            data = response.json()\n",
    "            records = data.get(\"records\", [])\n",
    "\n",
    "            for record in records:\n",
    "                pmid = record.get(\"pmid\")\n",
    "                doi = record.get(\"doi\", None)  # Get DOI if available\n",
    "                \n",
    "                if pmid and pmid not in results:\n",
    "                    results[pmid] = doi  # Store in dictionary\n",
    "        \n",
    "        else:\n",
    "            print(f\"API request failed for batch {i}-{i+batch_size}: {response.status_code}\")\n",
    "        \n",
    "        time.sleep(0.1)  # Prevent hitting API rate limits (adjust as needed)\n",
    "\n",
    "    return results"
   ],
   "id": "b0d5471843a0a156",
   "outputs": [],
   "execution_count": 5
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-12T16:55:14.796272Z",
     "start_time": "2025-03-12T16:55:14.733078Z"
    }
   },
   "cell_type": "code",
   "source": [
    "# Load Mappings\n",
    "pmid_doi_mapping_file = \"exp_output/pmid_doi_mapping.json\"\n",
    "if os.path.exists(\"exp_output/pmid_doi_mapping.json\"):\n",
    "    \n",
    "    with open(pmid_doi_mapping_file, \"r\") as f:\n",
    "        pmid_doi_mapping = json.load(f)  # read json file\n",
    "        print(f\"Loaded {len(pmid_doi_mapping)} mappings from file.\")\n",
    "else:\n",
    "    pmid_doi_mapping = {}"
   ],
   "id": "39c87d20aba6530f",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Loaded 144993 mappings from file.\n"
     ]
    }
   ],
   "execution_count": 6
  },
  {
   "metadata": {},
   "cell_type": "markdown",
   "source": [
    "## extract the citing Publications for each dataset\n",
    "\n",
    "Create a new column for the links of the citing publications for each dataset"
   ],
   "id": "2fae2af5ee2eb2cd"
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-12T16:55:16.239018Z",
     "start_time": "2025-03-12T16:55:14.797422Z"
    }
   },
   "cell_type": "code",
   "source": [
    "filtered_df = filtered_df.copy()\n",
    "\n",
    "filtered_df.loc[:, 'citing_publications_links'] = None\n",
    "\n",
    "for i, row in filtered_df.iterrows():\n",
    "    pub = str(row['publication'])  # Ensure string type\n",
    "    if \"href\" in pub:\n",
    "        match = re.findall(r'href=[\\'\"]([^\\'\"]+)[\\'\"]', pub)  # Extract href links\n",
    "        filtered_df.at[i, 'citing_publications_links'] = match if match else None\n",
    "    else:\n",
    "        filtered_df.at[i, 'citing_publications_links'] = None\n",
    "\n",
    "# Drop rows with missing links safely\n",
    "filtered_df = filtered_df.dropna(subset=['citing_publications_links']).reset_index(drop=True)\n",
    "\n",
    "print(f\"# n datasets with publications: {len(filtered_df)}\") \n",
    "print(filtered_df.columns)"
   ],
   "id": "8499a95c989f6a04",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "# n datasets with publications: 25696\n",
      "Index(['identifier', 'title', 'repository', 'species', 'instrument',\n",
      "       'publication', 'lab_head', 'announce_date', 'keywords',\n",
      "       'citing_publications_links'],\n",
      "      dtype='object')\n"
     ]
    }
   ],
   "execution_count": 7
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-12T16:55:17.236360Z",
     "start_time": "2025-03-12T16:55:16.240350Z"
    }
   },
   "cell_type": "code",
   "source": [
    "m = 0\n",
    "for i,row in filtered_df.iterrows():\n",
    "    id = row['identifier']\n",
    "    m+=len(row['citing_publications_links'])\n",
    "print(f\"Total number of publications: {m}\")\n",
    "print(f\"Average number of publications per dataset: {m/len(filtered_df)}\")"
   ],
   "id": "72894b5e6cb3a72a",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Total number of publications: 39347\n",
      "Average number of publications per dataset: 1.53125\n"
     ]
    }
   ],
   "execution_count": 8
  },
  {
   "metadata": {},
   "cell_type": "markdown",
   "source": "## Update Mapping",
   "id": "317d23b4ad9407dc"
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-12T16:56:00.482613Z",
     "start_time": "2025-03-12T16:55:17.237596Z"
    }
   },
   "cell_type": "code",
   "source": [
    "urls = filtered_df['citing_publications_links']\n",
    "print(len(urls))\n",
    "urls = urls.explode()\n",
    "print(len(urls))\n",
    "urls = urls.dropna()\n",
    "print(len(urls))\n",
    "pmids = urls.apply(url_to_pmid).dropna()\n",
    "print(len(set(pmids)))\n",
    "results = batch_PMID_to_doi(list(set(pmids)),150)\n",
    "print(len(results))"
   ],
   "id": "1ae9d7b082f4ce4",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "25696\n",
      "39347\n",
      "39347\n",
      "PMID not found in https://www.ncbi.nlm.nih.gov/pubmed/PMC10340108\n",
      "PMID not found in http://www.ncbi.nlm.nih.gov/pubmed/IPX0003899000\n",
      "PMID not found in http://www.ncbi.nlm.nih.gov/pubmed/IPX0003216000\n",
      "PMID not found in http://www.ncbi.nlm.nih.gov/pubmed/TBD\n",
      "PMID not found in http://www.ncbi.nlm.nih.gov/pubmed/\n",
      "PMID not found in http://www.ncbi.nlm.nih.gov/pubmed/\n",
      "PMID not found in http://www.ncbi.nlm.nih.gov/pubmed/\n",
      "19652\n",
      "Processing batch 0-150 (0.00%)\n",
      "Processing batch 150-300 (0.76%)\n",
      "Processing batch 300-450 (1.53%)\n",
      "Processing batch 450-600 (2.29%)\n",
      "Processing batch 600-750 (3.05%)\n",
      "Processing batch 750-900 (3.82%)\n",
      "Processing batch 900-1050 (4.58%)\n",
      "Processing batch 1050-1200 (5.34%)\n",
      "Processing batch 1200-1350 (6.11%)\n",
      "Processing batch 1350-1500 (6.87%)\n",
      "Processing batch 1500-1650 (7.63%)\n",
      "Processing batch 1650-1800 (8.40%)\n",
      "Processing batch 1800-1950 (9.16%)\n",
      "Processing batch 1950-2100 (9.92%)\n",
      "Processing batch 2100-2250 (10.69%)\n",
      "Processing batch 2250-2400 (11.45%)\n",
      "Processing batch 2400-2550 (12.21%)\n",
      "Processing batch 2550-2700 (12.98%)\n",
      "Processing batch 2700-2850 (13.74%)\n",
      "Processing batch 2850-3000 (14.50%)\n",
      "Processing batch 3000-3150 (15.27%)\n",
      "Processing batch 3150-3300 (16.03%)\n",
      "Processing batch 3300-3450 (16.79%)\n",
      "Processing batch 3450-3600 (17.56%)\n",
      "Processing batch 3600-3750 (18.32%)\n",
      "Processing batch 3750-3900 (19.08%)\n",
      "Processing batch 3900-4050 (19.85%)\n",
      "Processing batch 4050-4200 (20.61%)\n",
      "Processing batch 4200-4350 (21.37%)\n",
      "Processing batch 4350-4500 (22.14%)\n",
      "Processing batch 4500-4650 (22.90%)\n",
      "Processing batch 4650-4800 (23.66%)\n",
      "Processing batch 4800-4950 (24.42%)\n",
      "Processing batch 4950-5100 (25.19%)\n",
      "Processing batch 5100-5250 (25.95%)\n",
      "Processing batch 5250-5400 (26.71%)\n",
      "Processing batch 5400-5550 (27.48%)\n",
      "Processing batch 5550-5700 (28.24%)\n",
      "Processing batch 5700-5850 (29.00%)\n",
      "Processing batch 5850-6000 (29.77%)\n",
      "Processing batch 6000-6150 (30.53%)\n",
      "Processing batch 6150-6300 (31.29%)\n",
      "Processing batch 6300-6450 (32.06%)\n",
      "Processing batch 6450-6600 (32.82%)\n",
      "Processing batch 6600-6750 (33.58%)\n",
      "Processing batch 6750-6900 (34.35%)\n",
      "Processing batch 6900-7050 (35.11%)\n",
      "Processing batch 7050-7200 (35.87%)\n",
      "Processing batch 7200-7350 (36.64%)\n",
      "Processing batch 7350-7500 (37.40%)\n",
      "Processing batch 7500-7650 (38.16%)\n",
      "Processing batch 7650-7800 (38.93%)\n",
      "Processing batch 7800-7950 (39.69%)\n",
      "Processing batch 7950-8100 (40.45%)\n",
      "Processing batch 8100-8250 (41.22%)\n",
      "Processing batch 8250-8400 (41.98%)\n",
      "Processing batch 8400-8550 (42.74%)\n",
      "Processing batch 8550-8700 (43.51%)\n",
      "Processing batch 8700-8850 (44.27%)\n",
      "Processing batch 8850-9000 (45.03%)\n",
      "Processing batch 9000-9150 (45.80%)\n",
      "Processing batch 9150-9300 (46.56%)\n",
      "Processing batch 9300-9450 (47.32%)\n",
      "Processing batch 9450-9600 (48.09%)\n",
      "Processing batch 9600-9750 (48.85%)\n",
      "Processing batch 9750-9900 (49.61%)\n",
      "Processing batch 9900-10050 (50.38%)\n",
      "Processing batch 10050-10200 (51.14%)\n",
      "Processing batch 10200-10350 (51.90%)\n",
      "Processing batch 10350-10500 (52.67%)\n",
      "Processing batch 10500-10650 (53.43%)\n",
      "Processing batch 10650-10800 (54.19%)\n",
      "Processing batch 10800-10950 (54.96%)\n",
      "Processing batch 10950-11100 (55.72%)\n",
      "Processing batch 11100-11250 (56.48%)\n",
      "Processing batch 11250-11400 (57.25%)\n",
      "Processing batch 11400-11550 (58.01%)\n",
      "Processing batch 11550-11700 (58.77%)\n",
      "Processing batch 11700-11850 (59.54%)\n",
      "Processing batch 11850-12000 (60.30%)\n",
      "Processing batch 12000-12150 (61.06%)\n",
      "Processing batch 12150-12300 (61.83%)\n",
      "Processing batch 12300-12450 (62.59%)\n",
      "Processing batch 12450-12600 (63.35%)\n",
      "Processing batch 12600-12750 (64.12%)\n",
      "Processing batch 12750-12900 (64.88%)\n",
      "Processing batch 12900-13050 (65.64%)\n",
      "Processing batch 13050-13200 (66.41%)\n",
      "Processing batch 13200-13350 (67.17%)\n",
      "Processing batch 13350-13500 (67.93%)\n",
      "Processing batch 13500-13650 (68.70%)\n",
      "Processing batch 13650-13800 (69.46%)\n",
      "Processing batch 13800-13950 (70.22%)\n",
      "Processing batch 13950-14100 (70.99%)\n",
      "Processing batch 14100-14250 (71.75%)\n",
      "Processing batch 14250-14400 (72.51%)\n",
      "Processing batch 14400-14550 (73.27%)\n",
      "Processing batch 14550-14700 (74.04%)\n",
      "Processing batch 14700-14850 (74.80%)\n",
      "Processing batch 14850-15000 (75.56%)\n",
      "Processing batch 15000-15150 (76.33%)\n",
      "Processing batch 15150-15300 (77.09%)\n",
      "Processing batch 15300-15450 (77.85%)\n",
      "Processing batch 15450-15600 (78.62%)\n",
      "Processing batch 15600-15750 (79.38%)\n",
      "Processing batch 15750-15900 (80.14%)\n",
      "Processing batch 15900-16050 (80.91%)\n",
      "Processing batch 16050-16200 (81.67%)\n",
      "Processing batch 16200-16350 (82.43%)\n",
      "Processing batch 16350-16500 (83.20%)\n",
      "Processing batch 16500-16650 (83.96%)\n",
      "Processing batch 16650-16800 (84.72%)\n",
      "Processing batch 16800-16950 (85.49%)\n",
      "Processing batch 16950-17100 (86.25%)\n",
      "Processing batch 17100-17250 (87.01%)\n",
      "Processing batch 17250-17400 (87.78%)\n",
      "Processing batch 17400-17550 (88.54%)\n",
      "Processing batch 17550-17700 (89.30%)\n",
      "Processing batch 17700-17850 (90.07%)\n",
      "Processing batch 17850-18000 (90.83%)\n",
      "Processing batch 18000-18150 (91.59%)\n",
      "Processing batch 18150-18300 (92.36%)\n",
      "Processing batch 18300-18450 (93.12%)\n",
      "Processing batch 18450-18600 (93.88%)\n",
      "Processing batch 18600-18750 (94.65%)\n",
      "Processing batch 18750-18900 (95.41%)\n",
      "Processing batch 18900-19050 (96.17%)\n",
      "Processing batch 19050-19200 (96.94%)\n",
      "Processing batch 19200-19350 (97.70%)\n",
      "Processing batch 19350-19500 (98.46%)\n",
      "Processing batch 19500-19650 (99.23%)\n",
      "Processing batch 19650-19800 (99.99%)\n",
      "19651\n"
     ]
    }
   ],
   "execution_count": 9
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-12T16:56:00.501417Z",
     "start_time": "2025-03-12T16:56:00.484647Z"
    }
   },
   "cell_type": "code",
   "source": [
    "for key, value in results.items():\n",
    "    if key not in pmid_doi_mapping:\n",
    "        pmid_doi_mapping[key] = value\n",
    "\n",
    "print(len(pmid_doi_mapping))"
   ],
   "id": "373aac28c765113e",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "144993\n"
     ]
    }
   ],
   "execution_count": 10
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-12T16:56:00.656022Z",
     "start_time": "2025-03-12T16:56:00.503746Z"
    }
   },
   "cell_type": "code",
   "source": [
    "# save to file\n",
    "with open(pmid_doi_mapping_file, \"w\") as f:\n",
    "    json.dump(pmid_doi_mapping, f, indent=4) # Save to JSON\n",
    "print(f\"Saved {len(pmid_doi_mapping)} mappings to JSON file.\")"
   ],
   "id": "b5a58d778438d5a8",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Saved 144993 mappings to JSON file.\n"
     ]
    }
   ],
   "execution_count": 11
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-12T16:56:09.963151Z",
     "start_time": "2025-03-12T16:56:00.657328Z"
    }
   },
   "cell_type": "code",
   "source": [
    "filtered_df = filtered_df.explode('citing_publications_links')  # Split lists into rows\n",
    "\n",
    "df_grouped = filtered_df.groupby('citing_publications_links').apply(lambda x: {\n",
    "    \"citing_publication_link\": x.name,  # Grouped key\n",
    "    \"citing_pmid\": url_to_pmid(x.name), \n",
    "    \"citing_doi\": url_to_doi(x.name),  # Extract DOI from URL\n",
    "    \"cited_datasets_count\": len(x),\n",
    "    \"cited_datasets\": x.apply(lambda row: {\n",
    "        \"dataset_id\": row[\"identifier\"] if pd.notna(row[\"identifier\"]) else \"n/a\",\n",
    "        \"dataset_repo\": row[\"repository\"] if pd.notna(row[\"repository\"]) else \"n/a\",\n",
    "        \"dataset_keywords\": row[\"keywords\"] if pd.notna(row[\"keywords\"]) else \"n/a\",\n",
    "        \"dataset_title\": row[\"title\"] if pd.notna(row[\"title\"]) else \"n/a\",\n",
    "    }, axis=1).tolist()\n",
    "}).tolist()\n",
    "\n",
    "json_output = json.dumps(df_grouped, indent=4)\n",
    "\n",
    "with open(\"exp_ground_truth/publications_ground_truth.json\", \"w\") as json_file:\n",
    "    json_file.write(json_output)\n",
    "\n",
    "with open(pmid_doi_mapping_file, \"w\") as f:\n",
    "    json.dump(pmid_doi_mapping, f, indent=4) # Save to JSON\n",
    "    \n",
    "print(f\"Saved {len(df_grouped)} entries to JSON file.\")\n",
    "print(f\"Saved {len(pmid_doi_mapping)} mappings to JSON file.\")"
   ],
   "id": "14f705d80beaad52",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PMID not found in http://www.ncbi.nlm.nih.gov/pubmed/\n",
      "PMID not found in http://www.ncbi.nlm.nih.gov/pubmed/\n",
      "PMID 0 not found in mapping file. Querying API...\n",
      "PMID: 0, DOI: xxxx\n",
      "PMID not found in http://www.ncbi.nlm.nih.gov/pubmed/IPX0003216000\n",
      "PMID not found in http://www.ncbi.nlm.nih.gov/pubmed/IPX0003216000\n",
      "PMID not found in http://www.ncbi.nlm.nih.gov/pubmed/IPX0003899000\n",
      "PMID not found in http://www.ncbi.nlm.nih.gov/pubmed/IPX0003899000\n",
      "PMID not found in http://www.ncbi.nlm.nih.gov/pubmed/TBD\n",
      "PMID not found in http://www.ncbi.nlm.nih.gov/pubmed/TBD\n",
      "PMID 0 not found in mapping file. Querying API...\n",
      "PMID: 0, DOI: xxxx\n",
      "PMID not found in https://www.ncbi.nlm.nih.gov/pubmed/PMC10340108\n",
      "PMID not found in https://www.ncbi.nlm.nih.gov/pubmed/PMC10340108\n",
      "Saved 34879 entries to JSON file.\n",
      "Saved 144993 mappings to JSON file.\n"
     ]
    }
   ],
   "execution_count": 12
  },
  {
   "metadata": {},
   "cell_type": "markdown",
   "source": [
    "## Integration of dataset references from Gene Expression Omnibus   \n",
    "\n",
    "We want to integrate the GEO datasets"
   ],
   "id": "266a64c809ba5677"
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-12T16:56:09.967263Z",
     "start_time": "2025-03-12T16:56:09.964426Z"
    }
   },
   "cell_type": "code",
   "source": [
    "input_ground_truth_file = \"exp_ground_truth/publications_ground_truth.json\"\n",
    "output_ground_truth_file = \"exp_ground_truth/publications_ground_truth_PXD_GSE.json\"\n",
    "GEO_extract = \"exp_input/GEO_data.csv\""
   ],
   "id": "29274a02b1e2eb22",
   "outputs": [],
   "execution_count": 13
  },
  {
   "metadata": {},
   "cell_type": "markdown",
   "source": [
    "### Extracting all the datasets with eutils api\n",
    "\n",
    "We get all the uids of GEO series datasets"
   ],
   "id": "9b2cc02db9878414"
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-12T16:56:11.102774Z",
     "start_time": "2025-03-12T16:56:09.972046Z"
    }
   },
   "cell_type": "code",
   "source": [
    "esearch_url = \"https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi\"\n",
    "params = {\n",
    "    \"db\": \"gds\",  # Search in GEO database\n",
    "    \"term\": '\"gse\"[Entry Type]',  # Only fetch GEO Series (GSE)\n",
    "    \"retmax\": \"1000000\",  # Maximum records to retrieve (adjust as needed)\n",
    "    \"retmode\": \"xml\"\n",
    "}\n",
    "\n",
    "root = ElementTree.fromstring(requests.get(esearch_url, params=params).content)\n",
    "\n",
    "gse_ids = [id_elem.text for id_elem in root.findall(\".//Id\")]\n",
    "\n",
    "print(f\"Total datasets found: {len(gse_ids)}\")\n",
    "print(\"Sample GSE IDs:\", gse_ids[:10])  # Show first 10 results"
   ],
   "id": "e25fa0ce74b27e2f",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Total datasets found: 248808\n",
      "Sample GSE IDs: ['200291629', '200291614', '200291440', '200291300', '200291298', '200291297', '200291295', '200291292', '200291265', '200291226']\n"
     ]
    }
   ],
   "execution_count": 14
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-12T16:56:11.107318Z",
     "start_time": "2025-03-12T16:56:11.103829Z"
    }
   },
   "cell_type": "code",
   "source": [
    "def fetch_GEO_data(IDs,request_url,start,stop):\n",
    "    params = {\n",
    "        \"db\": \"gds\",\n",
    "        \"id\": \",\".join(IDs[start:stop]),  # Query window\n",
    "        \"retmode\": \"json\"\n",
    "    }\n",
    "    response = requests.get(request_url, params=params)\n",
    "    \n",
    "    try:\n",
    "        data = response.json()\n",
    "    except:\n",
    "        raise ValueError(\"Failed to parse JSON response! Please check the response content.\")\n",
    "    \n",
    "    return data"
   ],
   "id": "6f05282134c1e1b8",
   "outputs": [],
   "execution_count": 15
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-12T16:56:11.400669Z",
     "start_time": "2025-03-12T16:56:11.109141Z"
    }
   },
   "cell_type": "code",
   "source": "fetch_GEO_data(gse_ids,\"https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esummary.fcgi\",0,10)",
   "id": "370f3e0fe9744e5e",
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'header': {'type': 'esummary', 'version': '0.3'},\n",
       " 'result': {'uids': ['200291629',\n",
       "   '200291614',\n",
       "   '200291440',\n",
       "   '200291300',\n",
       "   '200291298',\n",
       "   '200291297',\n",
       "   '200291295',\n",
       "   '200291292',\n",
       "   '200291265',\n",
       "   '200291226'],\n",
       "  '200291629': {'uid': '200291629',\n",
       "   'accession': 'GSE291629',\n",
       "   'gds': '',\n",
       "   'title': 'The Role of BAZ2-dependent Chromatin Remodeling in Suppressing G4 DNA Structures and Associated Genomic Instability [INDUCE-seq]',\n",
       "   'summary': 'DNA G-quadruplexes (G4s) are secondary structures with significant roles in regulating genome function and stability. Dysregulation of the dynamic formation of G4s is linked to genomic instability and disease, but the underlying mechanisms are not fully understood. In this study, we conducted a screen of chromatin-modifying enzymes and identified nine potential inhibitors of G4 formation, including seven that were not previously characterized. Among these, we highlight the role of BAZ2 chromatin remodelers as key suppressors of G4 DNA and G4-related genome instability. Depletion of BAZ2 subunits led to increased G4 formation, especially at transcriptional regulatory elements. BAZ2B was found to associate with G4 loci, suggesting that it plays a direct role in suppressing G4s. While BAZ2-deficient cells exhibited modest genomic instability, treatment with the G4-stabilizing ligand BRACO19 exacerbated double-strand breaks (DSBs), highlighting its utility as a tool to study G4-dependent genome instability. DSB profiling using INDUCE-seq uncovered distinct breakage patterns around G4s, further underscoring the impact of G4s on genome integrity. Notably, we found that within G4s, G repeats were more susceptible to DSBs than loops. These results establish BAZ2 chromatin remodeling complexes as direct regulators of G4 dynamics and provide new insights into G4-dependent genome instability.',\n",
       "   'gpl': '21697',\n",
       "   'gse': '291629',\n",
       "   'taxon': 'Homo sapiens',\n",
       "   'entrytype': 'GSE',\n",
       "   'gdstype': 'Other',\n",
       "   'ptechtype': '',\n",
       "   'valtype': '',\n",
       "   'ssinfo': '',\n",
       "   'subsetinfo': '',\n",
       "   'pdat': '2025/03/11',\n",
       "   'suppfile': 'BED',\n",
       "   'samples': [{'accession': 'GSM8838981',\n",
       "     'title': 'siBAZ2B Vehicle Replicate 3'},\n",
       "    {'accession': 'GSM8838984', 'title': 'siBAZ2B BRACO19 Replicate 3'},\n",
       "    {'accession': 'GSM8838973', 'title': 'siBAZ2A Vehicle Replicate 1'},\n",
       "    {'accession': 'GSM8838967', 'title': 'siControl Vehicle Replicate 1'},\n",
       "    {'accession': 'GSM8838970', 'title': 'siControl BRACO19 Replicate 1'},\n",
       "    {'accession': 'GSM8838982', 'title': 'siBAZ2B BRACO19 Replicate 1'},\n",
       "    {'accession': 'GSM8838976', 'title': 'siBAZ2A BRACO19 Replicate 1'},\n",
       "    {'accession': 'GSM8838979', 'title': 'siBAZ2B Vehicle Replicate 1'},\n",
       "    {'accession': 'GSM8838968', 'title': 'siControl Vehicle Replicate 2'},\n",
       "    {'accession': 'GSM8838974', 'title': 'siBAZ2A Vehicle Replicate 2'},\n",
       "    {'accession': 'GSM8838977', 'title': 'siBAZ2A BRACO19 Replicate 2'},\n",
       "    {'accession': 'GSM8838971', 'title': 'siControl BRACO19 Replicate 2'},\n",
       "    {'accession': 'GSM8838983', 'title': 'siBAZ2B BRACO19 Replicate 2'},\n",
       "    {'accession': 'GSM8838980', 'title': 'siBAZ2B Vehicle Replicate 2'},\n",
       "    {'accession': 'GSM8838969', 'title': 'siControl Vehicle Replicate 3'},\n",
       "    {'accession': 'GSM8838972', 'title': 'siControl BRACO19 Replicate 3'},\n",
       "    {'accession': 'GSM8838975', 'title': 'siBAZ2A Vehicle Replicate 3'},\n",
       "    {'accession': 'GSM8838978', 'title': 'siBAZ2A BRACO19 Replicate 3'}],\n",
       "   'relations': [],\n",
       "   'extrelations': [],\n",
       "   'n_samples': 18,\n",
       "   'seriestitle': '',\n",
       "   'platformtitle': '',\n",
       "   'platformtaxa': '',\n",
       "   'samplestaxa': '',\n",
       "   'pubmedids': [],\n",
       "   'projects': [],\n",
       "   'ftplink': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE291nnn/GSE291629/',\n",
       "   'geo2r': 'no',\n",
       "   'bioproject': 'PRJNA1234547'},\n",
       "  '200291614': {'uid': '200291614',\n",
       "   'accession': 'GSE291614',\n",
       "   'gds': '',\n",
       "   'title': 'Prenatally-Derived Macrophages Support Choroidal Health and Decline in Age-Related Macular Degeneration',\n",
       "   'summary': 'Hallmark findings in age-related macular degeneration (AMD) include the accumulation of extracellular lipid and vasodegeneration of the choriocapillaris. Choroidal inflammation has long been linked to AMD, but little is known about the immune cells of the human choroid. Using 3D multiplex immunofluorescence, single-cell RNA-sequencing, and flow cytometry, we unravel the cellular composition and spatial organization of the human choroid and the immune cells within it. We identify two populations of choroidal macrophages with distinct FOLR2 expression that account for the majority of myeloid cells. FOLR2+ macrophages predominate in the nondiseased eye, express lipid-handling machinery, uptake lipoprotein particles, and contain high amounts of lipid. In AMD, FOLR2+ macrophages are decreased and functionally attenuated with respect to lipoprotein metabolism. In mice, FOLR2+ macrophages are negative for the postnatal fate-reporter Ms4a3, and their depletion causes an accelerated AMD-like phenotype. Our results show that prenatally-derived resident macrophages decline in AMD and are implicated in multiple hallmark functions known to be compromised in the disease.  Citation: Fortmann, Seth D. et al. 2025. Prenatally-Derived Macrophages Support Choroidal Health and Decline in Age-Related Macular Degeneration. Journal of Experimental Medicine. In Press.',\n",
       "   'gpl': '',\n",
       "   'gse': '291614',\n",
       "   'taxon': '',\n",
       "   'entrytype': 'GSE',\n",
       "   'gdstype': 'Expression profiling by high throughput sequencing; Third-party reanalysis',\n",
       "   'ptechtype': '',\n",
       "   'valtype': '',\n",
       "   'ssinfo': '',\n",
       "   'subsetinfo': '',\n",
       "   'pdat': '2025/03/11',\n",
       "   'suppfile': 'H5AD, RDS, TXT',\n",
       "   'samples': [],\n",
       "   'relations': [],\n",
       "   'extrelations': [],\n",
       "   'n_samples': '',\n",
       "   'seriestitle': '',\n",
       "   'platformtitle': '',\n",
       "   'platformtaxa': '',\n",
       "   'samplestaxa': '',\n",
       "   'pubmedids': [],\n",
       "   'projects': [],\n",
       "   'ftplink': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE291nnn/GSE291614/',\n",
       "   'geo2r': 'no',\n",
       "   'bioproject': ''},\n",
       "  '200291440': {'uid': '200291440',\n",
       "   'accession': 'GSE291440',\n",
       "   'gds': '',\n",
       "   'title': 'GA-Camk2a mouse RNAseq data for the manuscript by Rezaei et al: Correction of dysregulated lipid metabolism normalizes gene expression in oligodendrocytes and prolongs lifespan in female poly-GA C9orf72 mouse model',\n",
       "   'summary': 'These sequences support the following paper abstract: \"Clinical and genetic research links altered cholesterol metabolism with ALS development and progression, yet pinpointing specific pathomechanisms remain challenging. We investigated how cholesterol dysmetabolism interacts with protein aggregation, demyelination, and neuronal loss in ALS. Bulk RNAseq transcriptomics showed decreased cholesterol biosynthesis and increased cholesterol export in ALS mouse models (GA-Nes, GA-CFP, rNLS8) and patient samples (spinal cord), suggesting an adaptive response to cholesterol overload. Consequently, we assessed the efficacy of the cholesterol-binding drug 2-hydroxypropyl--cyclodextrin (CD) in a fast-progressing C9orf72 ALS mouse model with extensive poly-GA expression and myelination deficits. CD treatment normalized cholesteryl ester levels, lowered neurofilament light chain levels, and prolonged lifespan in female but not male GA-Nes mice, without impacting poly-GA aggregates. Single nucleus transcriptomics indicated that CD primarily affected oligodendrocytes, significantly restored myelin gene expression, increased density of myelinated axons, inhibited the disease-associated oligodendrocyte response, and downregulated the lipid-associated genes Plin4 and ApoD. These results suggest that reducing excess free cholesterol in the CNS could be a viable ALS treatment strategy.\"',\n",
       "   'gpl': '18480',\n",
       "   'gse': '291440',\n",
       "   'taxon': 'Mus musculus',\n",
       "   'entrytype': 'GSE',\n",
       "   'gdstype': 'Expression profiling by high throughput sequencing',\n",
       "   'ptechtype': '',\n",
       "   'valtype': '',\n",
       "   'ssinfo': '',\n",
       "   'subsetinfo': '',\n",
       "   'pdat': '2025/03/11',\n",
       "   'suppfile': 'CSV',\n",
       "   'samples': [{'accession': 'GSM8835722', 'title': 'GC20_hp_ctrl'},\n",
       "    {'accession': 'GSM8835719', 'title': 'GC16_cx_GACamk'},\n",
       "    {'accession': 'GSM8835728', 'title': 'GC31_hp_ctrl'},\n",
       "    {'accession': 'GSM8835731', 'title': 'GC35_cx_GACamk'},\n",
       "    {'accession': 'GSM8835725', 'title': 'GC30_cx_GACamk'},\n",
       "    {'accession': 'GSM8835734', 'title': 'GC37_hp_ctrl'},\n",
       "    {'accession': 'GSM8835737', 'title': 'GC41_hp_GACamk'},\n",
       "    {'accession': 'GSM8835717', 'title': 'GC11_cx_ctrl'},\n",
       "    {'accession': 'GSM8835723', 'title': 'GC23_cx_GACamk'},\n",
       "    {'accession': 'GSM8835720', 'title': 'GC16_hp_GACamk'},\n",
       "    {'accession': 'GSM8835726', 'title': 'GC30_hp_GACamk'},\n",
       "    {'accession': 'GSM8835732', 'title': 'GC35_hp_GACamk'},\n",
       "    {'accession': 'GSM8835729', 'title': 'GC34_cx_GACamk'},\n",
       "    {'accession': 'GSM8835721', 'title': 'GC20_cx_ctrl'},\n",
       "    {'accession': 'GSM8835735', 'title': 'GC38_cx_ctrl'},\n",
       "    {'accession': 'GSM8835718', 'title': 'GC11_hp_ctrl'},\n",
       "    {'accession': 'GSM8835727', 'title': 'GC31_cx_ctrl'},\n",
       "    {'accession': 'GSM8835724', 'title': 'GC23_hp_GACamk'},\n",
       "    {'accession': 'GSM8835733', 'title': 'GC37_cx_ctrl'},\n",
       "    {'accession': 'GSM8835730', 'title': 'GC34_hp_GACamk'},\n",
       "    {'accession': 'GSM8835736', 'title': 'GC41_cx_GACamk'}],\n",
       "   'relations': [],\n",
       "   'extrelations': [],\n",
       "   'n_samples': 21,\n",
       "   'seriestitle': '',\n",
       "   'platformtitle': '',\n",
       "   'platformtaxa': '',\n",
       "   'samplestaxa': '',\n",
       "   'pubmedids': [],\n",
       "   'projects': [],\n",
       "   'ftplink': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE291nnn/GSE291440/',\n",
       "   'geo2r': 'no',\n",
       "   'bioproject': 'PRJNA1233599'},\n",
       "  '200291300': {'uid': '200291300',\n",
       "   'accession': 'GSE291300',\n",
       "   'gds': '',\n",
       "   'title': 'Targeting eRNA-producing super-enhancers regulates TNF expression and mitigates chronic inflammation in mice and patient-derived immune cells [RNA-KO]',\n",
       "   'summary': 'Chronic inflammatory diseases are driven by immune cell dysregulation and overproduction of pro-inflammatory molecules, such as tumor necrosis factor alpha (TNF). Super-enhancers (SEs) and their enhancer RNAs (eRNAs) are critical gene expression regulators and offer therapeutic potential beyond protein-targeting approaches. We hypothesized that targeting eRNAs could reduce chronic inflammation by modulating TNF expression. We generated TNF-9 knockout (KO) mice by deleting a Tnf-regulating enhancer region. These mice exhibited significantly reduced Tnf levels, improved disease outcomes, and diminished immune cell activation in models of rheumatoid arthritis (RA), psoriasis, and lipopolysaccharide (LPS)-induced sepsis. Integrative epigenomic and transcriptomic analysis identified additional LPS-responsive, eRNA-producing enhancers as therapeutic targets. Antisense oligonucleotide (ASO)-mediated knockdown of TNF-9 eRNA in mouse macrophages demonstrated decreased Tnf expression and alleviated RA symptoms. Furthermore, ASO-mediated inhibition of the eRNA of the human homolog of TNF-9 similarly reduced TNF levels. These findings support eRNA-targeted interventions as potential treatment for chronic inflammatory diseases.',\n",
       "   'gpl': '21493',\n",
       "   'gse': '291300',\n",
       "   'taxon': 'Mus musculus',\n",
       "   'entrytype': 'GSE',\n",
       "   'gdstype': 'Expression profiling by high throughput sequencing',\n",
       "   'ptechtype': '',\n",
       "   'valtype': '',\n",
       "   'ssinfo': '',\n",
       "   'subsetinfo': '',\n",
       "   'pdat': '2025/03/11',\n",
       "   'suppfile': 'TXT',\n",
       "   'samples': [{'accession': 'GSM8833123',\n",
       "     'title': 'HSS9 KO mouse, LPS 6hr treated BMDM, RNA-seq, replicate 3'},\n",
       "    {'accession': 'GSM8833126',\n",
       "     'title': 'HSS9 KO mouse, PBS 6hr treated BMDM, RNA-seq, replicate 3'},\n",
       "    {'accession': 'GSM8833129',\n",
       "     'title': 'littermate control mouse, LPS 6hr treated BMDM, RNA-seq, replicate 3'},\n",
       "    {'accession': 'GSM8833132',\n",
       "     'title': 'littermate control mouse, PBS 6hr treated BMDM, RNA-seq, replicate 3'},\n",
       "    {'accession': 'GSM8833127',\n",
       "     'title': 'littermate control mouse, LPS 6hr treated BMDM, RNA-seq, replicate 1'},\n",
       "    {'accession': 'GSM8833121',\n",
       "     'title': 'HSS9 KO mouse, LPS 6hr treated BMDM, RNA-seq, replicate 1'},\n",
       "    {'accession': 'GSM8833124',\n",
       "     'title': 'HSS9 KO mouse, PBS 6hr treated BMDM, RNA-seq, replicate 1'},\n",
       "    {'accession': 'GSM8833130',\n",
       "     'title': 'littermate control mouse, PBS 6hr treated BMDM, RNA-seq, replicate 1'},\n",
       "    {'accession': 'GSM8833122',\n",
       "     'title': 'HSS9 KO mouse, LPS 6hr treated BMDM, RNA-seq, replicate 2'},\n",
       "    {'accession': 'GSM8833128',\n",
       "     'title': 'littermate control mouse, LPS 6hr treated BMDM, RNA-seq, replicate 2'},\n",
       "    {'accession': 'GSM8833125',\n",
       "     'title': 'HSS9 KO mouse, PBS 6hr treated BMDM, RNA-seq, replicate 2'},\n",
       "    {'accession': 'GSM8833131',\n",
       "     'title': 'littermate control mouse, PBS 6hr treated BMDM, RNA-seq, replicate 2'}],\n",
       "   'relations': [],\n",
       "   'extrelations': [],\n",
       "   'n_samples': 12,\n",
       "   'seriestitle': '',\n",
       "   'platformtitle': '',\n",
       "   'platformtaxa': '',\n",
       "   'samplestaxa': '',\n",
       "   'pubmedids': [],\n",
       "   'projects': [],\n",
       "   'ftplink': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE291nnn/GSE291300/',\n",
       "   'geo2r': 'no',\n",
       "   'bioproject': 'PRJNA1232792'},\n",
       "  '200291298': {'uid': '200291298',\n",
       "   'accession': 'GSE291298',\n",
       "   'gds': '',\n",
       "   'title': 'Targeting eRNA-producing super-enhancers regulates TNF expression and mitigates chronic inflammation in mice and patient-derived immune cells [CUT&RUN]',\n",
       "   'summary': 'Chronic inflammatory diseases are driven by immune cell dysregulation and overproduction of pro-inflammatory molecules, such as tumor necrosis factor alpha (TNF). Super-enhancers (SEs) and their enhancer RNAs (eRNAs) are critical gene expression regulators and offer therapeutic potential beyond protein-targeting approaches. We hypothesized that targeting eRNAs could reduce chronic inflammation by modulating TNF expression. We generated TNF-9 knockout (KO) mice by deleting a Tnf-regulating enhancer region. These mice exhibited significantly reduced Tnf levels, improved disease outcomes, and diminished immune cell activation in models of rheumatoid arthritis (RA), psoriasis, and lipopolysaccharide (LPS)-induced sepsis. Integrative epigenomic and transcriptomic analysis identified additional LPS-responsive, eRNA-producing enhancers as therapeutic targets. Antisense oligonucleotide (ASO)-mediated knockdown of TNF-9 eRNA in mouse macrophages demonstrated decreased Tnf expression and alleviated RA symptoms. Furthermore, ASO-mediated inhibition of the eRNA of the human homolog of TNF-9 similarly reduced TNF levels. These findings support eRNA-targeted interventions as potential treatment for chronic inflammatory diseases.',\n",
       "   'gpl': '24247',\n",
       "   'gse': '291298',\n",
       "   'taxon': 'Mus musculus',\n",
       "   'entrytype': 'GSE',\n",
       "   'gdstype': 'Genome binding/occupancy profiling by high throughput sequencing',\n",
       "   'ptechtype': '',\n",
       "   'valtype': '',\n",
       "   'ssinfo': '',\n",
       "   'subsetinfo': '',\n",
       "   'pdat': '2025/03/11',\n",
       "   'suppfile': 'BIGWIG, NARROWPEAK',\n",
       "   'samples': [{'accession': 'GSM8833109',\n",
       "     'title': 'LPS 1hr treated BMDM, IgG CUT&Run (Input)'},\n",
       "    {'accession': 'GSM8833112',\n",
       "     'title': 'PBS 1hr treated BMDM, IgG CUT&Run (Input)'},\n",
       "    {'accession': 'GSM8833107',\n",
       "     'title': 'LPS 1hr treated BMDM, H3K27Ac CUT&Run (ChIP)'},\n",
       "    {'accession': 'GSM8833110',\n",
       "     'title': 'PBS 1hr treated BMDM, H3K27Ac CUT&Run (ChIP)'},\n",
       "    {'accession': 'GSM8833108',\n",
       "     'title': 'LPS 1hr treated BMDM, H3K4me1 CUT&Run (ChIP)'},\n",
       "    {'accession': 'GSM8833111',\n",
       "     'title': 'PBS 1hr treated BMDM, H3K4me1 CUT&Run (ChIP)'}],\n",
       "   'relations': [],\n",
       "   'extrelations': [],\n",
       "   'n_samples': 6,\n",
       "   'seriestitle': '',\n",
       "   'platformtitle': '',\n",
       "   'platformtaxa': '',\n",
       "   'samplestaxa': '',\n",
       "   'pubmedids': [],\n",
       "   'projects': [],\n",
       "   'ftplink': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE291nnn/GSE291298/',\n",
       "   'geo2r': 'no',\n",
       "   'bioproject': 'PRJNA1232783'},\n",
       "  '200291297': {'uid': '200291297',\n",
       "   'accession': 'GSE291297',\n",
       "   'gds': '',\n",
       "   'title': 'Targeting eRNA-producing super-enhancers regulates TNF expression and mitigates chronic inflammation in mice and patient-derived immune cells [RNA B6]',\n",
       "   'summary': 'Chronic inflammatory diseases are driven by immune cell dysregulation and overproduction of pro-inflammatory molecules, such as tumor necrosis factor alpha (TNF). Super-enhancers (SEs) and their enhancer RNAs (eRNAs) are critical gene expression regulators and offer therapeutic potential beyond protein-targeting approaches. We hypothesized that targeting eRNAs could reduce chronic inflammation by modulating TNF expression. We generated TNF-9 knockout (KO) mice by deleting a Tnf-regulating enhancer region. These mice exhibited significantly reduced Tnf levels, improved disease outcomes, and diminished immune cell activation in models of rheumatoid arthritis (RA), psoriasis, and lipopolysaccharide (LPS)-induced sepsis. Integrative epigenomic and transcriptomic analysis identified additional LPS-responsive, eRNA-producing enhancers as therapeutic targets. Antisense oligonucleotide (ASO)-mediated knockdown of TNF-9 eRNA in mouse macrophages demonstrated decreased Tnf expression and alleviated RA symptoms. Furthermore, ASO-mediated inhibition of the eRNA of the human homolog of TNF-9 similarly reduced TNF levels. These findings support eRNA-targeted interventions as potential treatment for chronic inflammatory diseases.',\n",
       "   'gpl': '21493',\n",
       "   'gse': '291297',\n",
       "   'taxon': 'Mus musculus',\n",
       "   'entrytype': 'GSE',\n",
       "   'gdstype': 'Expression profiling by high throughput sequencing',\n",
       "   'ptechtype': '',\n",
       "   'valtype': '',\n",
       "   'ssinfo': '',\n",
       "   'subsetinfo': '',\n",
       "   'pdat': '2025/03/11',\n",
       "   'suppfile': 'TXT',\n",
       "   'samples': [{'accession': 'GSM8833102',\n",
       "     'title': 'PBS 1hr treated BMDM, RNA-seq, replicate 2'},\n",
       "    {'accession': 'GSM8833105',\n",
       "     'title': 'LPS 1hr treated BMDM, RNA-seq, replicate 2'},\n",
       "    {'accession': 'GSM8833103',\n",
       "     'title': 'PBS 1hr treated BMDM, RNA-seq, replicate 3'},\n",
       "    {'accession': 'GSM8833106',\n",
       "     'title': 'LPS 1hr treated BMDM, RNA-seq, replicate 3'},\n",
       "    {'accession': 'GSM8833104',\n",
       "     'title': 'LPS 1hr treated BMDM, RNA-seq, replicate 1'},\n",
       "    {'accession': 'GSM8833101',\n",
       "     'title': 'PBS 1hr treated BMDM, RNA-seq, replicate 1'}],\n",
       "   'relations': [],\n",
       "   'extrelations': [],\n",
       "   'n_samples': 6,\n",
       "   'seriestitle': '',\n",
       "   'platformtitle': '',\n",
       "   'platformtaxa': '',\n",
       "   'samplestaxa': '',\n",
       "   'pubmedids': [],\n",
       "   'projects': [],\n",
       "   'ftplink': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE291nnn/GSE291297/',\n",
       "   'geo2r': 'no',\n",
       "   'bioproject': 'PRJNA1232782'},\n",
       "  '200291295': {'uid': '200291295',\n",
       "   'accession': 'GSE291295',\n",
       "   'gds': '',\n",
       "   'title': 'Single-Cell Resolution of Individual Variation in Hypothalamic Neurons Allows Targeted Manipulation Affecting Social Motivation',\n",
       "   'summary': 'One great challenge in neuroscience is connecting molecular and cellular phenotypes to behavioral consequences. Using social motivation as a test case, we propose that individual behavioral trait variation can be leveraged to identify novel components of social reward circuits in the brain. Specifically, we hypothesized that either proportion or molecular features of distinct neuron classes in the hypothalamus can predict individual differences in social motivation. To test this, we generated high-precision single-nucleus RNA-sequencing profiles of >120,000 neurons from the hypothalamus and adjacent thalamus from 36 mice assessed for social motivation, balancing across sex and an autism-associated mutation genotype. Analysis of IEG patterns revealed that PVN Agtr1a+ neurons negatively predict social behavior. Consistent with this, FDA-approved AGTR1A antagonists increase social orienting. Next, analyzing neuronal subtype proportions as predictors of social behavior, we identified multiple neuronal populations whose relative abundance correlated with individual differences in social reward-seeking, particularly the Nxph4+ neurons of the posterior and lateral hypothalamus. Subsequent chemogenetic inhibition of these suppressed multiple aspects of social motivation. This work establishes a proof-of-principle for a new approach using single-cellsingle cell genomics to identify neural substrates for behavior and identifies cellular determinants of social motivation, which suggest therapeutic avenues for disorders with social deficits.',\n",
       "   'gpl': '34290',\n",
       "   'gse': '291295',\n",
       "   'taxon': 'Mus musculus',\n",
       "   'entrytype': 'GSE',\n",
       "   'gdstype': 'Expression profiling by high throughput sequencing',\n",
       "   'ptechtype': '',\n",
       "   'valtype': '',\n",
       "   'ssinfo': '',\n",
       "   'subsetinfo': '',\n",
       "   'pdat': '2025/03/11',\n",
       "   'suppfile': 'MTX, RDS, TSV',\n",
       "   'samples': [{'accession': 'GSM8833071',\n",
       "     'title': 'Myt1l heterozygous male mouse 6'},\n",
       "    {'accession': 'GSM8833091', 'title': 'Wildtype male mouse 6'},\n",
       "    {'accession': 'GSM8833074', 'title': 'Myt1l heterozygous male mouse 9'},\n",
       "    {'accession': 'GSM8833094', 'title': 'Wildtype male mouse 9'},\n",
       "    {'accession': 'GSM8833077', 'title': 'Wildtype female mouse 2'},\n",
       "    {'accession': 'GSM8833060', 'title': 'Myt1l heterozygous female mouse 2'},\n",
       "    {'accession': 'GSM8833080', 'title': 'Wildtype female mouse 5'},\n",
       "    {'accession': 'GSM8833063', 'title': 'Myt1l heterozygous female mouse 5'},\n",
       "    {'accession': 'GSM8833083', 'title': 'Wildtype female mouse 8'},\n",
       "    {'accession': 'GSM8833086', 'title': 'Wildtype male mouse 10'},\n",
       "    {'accession': 'GSM8833069', 'title': 'Myt1l heterozygous male mouse 4'},\n",
       "    {'accession': 'GSM8833089', 'title': 'Wildtype male mouse 4'},\n",
       "    {'accession': 'GSM8833092', 'title': 'Wildtype male mouse 7'},\n",
       "    {'accession': 'GSM8833066', 'title': 'Myt1l heterozygous male mouse 10'},\n",
       "    {'accession': 'GSM8833078', 'title': 'Wildtype female mouse 3'},\n",
       "    {'accession': 'GSM8833072', 'title': 'Myt1l heterozygous male mouse 7'},\n",
       "    {'accession': 'GSM8833075', 'title': 'Wildtype female mouse 1'},\n",
       "    {'accession': 'GSM8833081', 'title': 'Wildtype female mouse 6'},\n",
       "    {'accession': 'GSM8833061', 'title': 'Myt1l heterozygous female mouse 3'},\n",
       "    {'accession': 'GSM8833064', 'title': 'Myt1l heterozygous female mouse 6'},\n",
       "    {'accession': 'GSM8833084', 'title': 'Wildtype female mouse 9'},\n",
       "    {'accession': 'GSM8833067', 'title': 'Myt1l heterozygous male mouse 2'},\n",
       "    {'accession': 'GSM8833087', 'title': 'Wildtype male mouse 2'},\n",
       "    {'accession': 'GSM8833070', 'title': 'Myt1l heterozygous male mouse 5'},\n",
       "    {'accession': 'GSM8833090', 'title': 'Wildtype male mouse 5'},\n",
       "    {'accession': 'GSM8833073', 'title': 'Myt1l heterozygous male mouse 8'},\n",
       "    {'accession': 'GSM8833059', 'title': 'Myt1l heterozygous female mouse 1'},\n",
       "    {'accession': 'GSM8833093', 'title': 'Wildtype male mouse 8'},\n",
       "    {'accession': 'GSM8833079', 'title': 'Wildtype female mouse 4'},\n",
       "    {'accession': 'GSM8833076', 'title': 'Wildtype female mouse 10'},\n",
       "    {'accession': 'GSM8833068', 'title': 'Myt1l heterozygous male mouse 3'},\n",
       "    {'accession': 'GSM8833088', 'title': 'Wildtype male mouse 3'},\n",
       "    {'accession': 'GSM8833062', 'title': 'Myt1l heterozygous female mouse 4'},\n",
       "    {'accession': 'GSM8833065', 'title': 'Myt1l heterozygous male mouse 1'},\n",
       "    {'accession': 'GSM8833082', 'title': 'Wildtype female mouse 7'},\n",
       "    {'accession': 'GSM8833085', 'title': 'Wildtype male mouse 1'}],\n",
       "   'relations': [],\n",
       "   'extrelations': [],\n",
       "   'n_samples': 36,\n",
       "   'seriestitle': '',\n",
       "   'platformtitle': '',\n",
       "   'platformtaxa': '',\n",
       "   'samplestaxa': '',\n",
       "   'pubmedids': [],\n",
       "   'projects': [],\n",
       "   'ftplink': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE291nnn/GSE291295/',\n",
       "   'geo2r': 'no',\n",
       "   'bioproject': 'PRJNA1232789'},\n",
       "  '200291292': {'uid': '200291292',\n",
       "   'accession': 'GSE291292',\n",
       "   'gds': '',\n",
       "   'title': 'Navarixin alleviates cardiac remodeling after myocardial infarction by decreasing neutrophil infiltration and the inflammatory response',\n",
       "   'summary': 'Introduction: Coronary atherosclerotic heart disease is an important, worldwide burden on human health. Central muscle infarction is the most dangerous condition, has the highest mortality and disability rates, and is gradually becoming more common among young people. After myocardial infarction, neutrophils recruited to the infarcted area promote the myocardial inflammatory response by releasing proinflammatory factors and chemokines and release matrix metalloproteinases and myeloperoxidases that degrade the extracellular matrix and produce reactive oxygen species, resulting in irreversible myocardial damage and thereby promoting ventricular remodeling. Methods: In this study, we constructed a mouse model of myocardial infarction and utilized the CXCR2 receptor inhibitor navarixin (Nav) to reduce neutrophil recruitment after MI. Results: We observed that Nav improved cardiac function, reduced myocardial damage, reduced neutrophil infiltration, reduced inflammatory factor expression and improved cardiac fibrosis in mice. Through transcriptomic analysis, we found that Nav affects signaling pathways such as the innate immune response and the chemokine signaling pathway, thereby decreasing the inflammatory response by reducing neutrophil chemotaxis. Discussion: This study provides new insights for the use of CXCR2 inhibitors as new therapeutic options for myocardial infarction in the future.',\n",
       "   'gpl': '24247',\n",
       "   'gse': '291292',\n",
       "   'taxon': 'Mus musculus',\n",
       "   'entrytype': 'GSE',\n",
       "   'gdstype': 'Expression profiling by high throughput sequencing',\n",
       "   'ptechtype': '',\n",
       "   'valtype': '',\n",
       "   'ssinfo': '',\n",
       "   'subsetinfo': '',\n",
       "   'pdat': '2025/03/11',\n",
       "   'suppfile': 'XLSX',\n",
       "   'samples': [{'accession': 'GSM8833036', 'title': 'MI_2'},\n",
       "    {'accession': 'GSM8833039', 'title': 'MI_5'},\n",
       "    {'accession': 'GSM8833042', 'title': 'MI_NAV3'},\n",
       "    {'accession': 'GSM8833037', 'title': 'MI_3'},\n",
       "    {'accession': 'GSM8833040', 'title': 'MI_NAV1'},\n",
       "    {'accession': 'GSM8833043', 'title': 'MI_NAV4'},\n",
       "    {'accession': 'GSM8833035', 'title': 'MI_1'},\n",
       "    {'accession': 'GSM8833044', 'title': 'MI_NAV5'},\n",
       "    {'accession': 'GSM8833041', 'title': 'MI_NAV2'},\n",
       "    {'accession': 'GSM8833038', 'title': 'MI_4'}],\n",
       "   'relations': [],\n",
       "   'extrelations': [],\n",
       "   'n_samples': 10,\n",
       "   'seriestitle': '',\n",
       "   'platformtitle': '',\n",
       "   'platformtaxa': '',\n",
       "   'samplestaxa': '',\n",
       "   'pubmedids': [],\n",
       "   'projects': [],\n",
       "   'ftplink': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE291nnn/GSE291292/',\n",
       "   'geo2r': 'no',\n",
       "   'bioproject': 'PRJNA1232730'},\n",
       "  '200291265': {'uid': '200291265',\n",
       "   'accession': 'GSE291265',\n",
       "   'gds': '',\n",
       "   'title': 'From the Gut to the Brain: Transcriptomic Insights into Neonatal Meningitis Escherichia coli Across Diverse Host Niches',\n",
       "   'summary': 'Neonatal meningitis caused by Escherichia coli (NMEC) is a leading cause of morbidity and mortality in newborns, and its pathogenesis relies on the ability of the bacterium to adapt and survive in diverse host environments. Despite advances in neonatal care, significant gaps remain in our understanding of how NMEC reprogram their transcriptome to survive in physiologically relevant niches. This study investigated the transcriptomic profiles of  E. coli strain RS218 (O18:H7:K1) in four under host-relevant environment colonic fluid (CF), serum (S), human brain endothelial cells (HBECs) and cerebrospinal fluid (CSF)to mimic the infection landscape of neonatal meningitis. High-throughput RNA sequencing (RNA-seq) was performed to profile NMECs transcriptomic responses in each niche, and differential gene expression analyses were conducted to identify enriched pathways.',\n",
       "   'gpl': '34685',\n",
       "   'gse': '291265',\n",
       "   'taxon': 'Escherichia coli',\n",
       "   'entrytype': 'GSE',\n",
       "   'gdstype': 'Expression profiling by high throughput sequencing',\n",
       "   'ptechtype': '',\n",
       "   'valtype': '',\n",
       "   'ssinfo': '',\n",
       "   'subsetinfo': '',\n",
       "   'pdat': '2025/03/11',\n",
       "   'suppfile': 'TXT',\n",
       "   'samples': [{'accession': 'GSM8832305',\n",
       "     'title': 'E. coli RS218 in Serum-replicate 1'},\n",
       "    {'accession': 'GSM8832302',\n",
       "     'title': 'E. coli RS218 in LB broth-replicate 1'},\n",
       "    {'accession': 'GSM8832308',\n",
       "     'title': 'E. coli RS218 in Colonic Fluid-replicate 1'},\n",
       "    {'accession': 'GSM8832314',\n",
       "     'title': 'Human brain endothelial cells infected with E. coli RS218- replicate 1'},\n",
       "    {'accession': 'GSM8832311',\n",
       "     'title': 'E. coli RS218 in Cerebrospinal Fluid-replicate 1'},\n",
       "    {'accession': 'GSM8832303',\n",
       "     'title': 'E. coli RS218 in LB broth-replicate 2'},\n",
       "    {'accession': 'GSM8832306', 'title': 'E. coli RS218 in Serum-replicate 2'},\n",
       "    {'accession': 'GSM8832309',\n",
       "     'title': 'E. coli RS218 in Colonic Fluid-replicate 2'},\n",
       "    {'accession': 'GSM8832315',\n",
       "     'title': 'Human brain endothelial cells infected with E. coli RS218- replicate 2'},\n",
       "    {'accession': 'GSM8832312',\n",
       "     'title': 'E. coli RS218 in Cerebrospinal Fluid-replicate 2'},\n",
       "    {'accession': 'GSM8832304',\n",
       "     'title': 'E. coli RS218 in LB broth-replicate 3'},\n",
       "    {'accession': 'GSM8832313',\n",
       "     'title': 'E. coli RS218 in Cerebrospinal Fluid-replicate 3'},\n",
       "    {'accession': 'GSM8832310',\n",
       "     'title': 'E. coli RS218 in Colonic Fluid-replicate 3'},\n",
       "    {'accession': 'GSM8832307', 'title': 'E. coli RS218 in Serum-replicate 3'},\n",
       "    {'accession': 'GSM8832316',\n",
       "     'title': 'Human brain endothelial cells infected with E. coli RS218- replicate 3'}],\n",
       "   'relations': [],\n",
       "   'extrelations': [],\n",
       "   'n_samples': 15,\n",
       "   'seriestitle': '',\n",
       "   'platformtitle': '',\n",
       "   'platformtaxa': '',\n",
       "   'samplestaxa': '',\n",
       "   'pubmedids': [],\n",
       "   'projects': [],\n",
       "   'ftplink': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE291nnn/GSE291265/',\n",
       "   'geo2r': 'no',\n",
       "   'bioproject': 'PRJNA1232686'},\n",
       "  '200291226': {'uid': '200291226',\n",
       "   'accession': 'GSE291226',\n",
       "   'gds': '',\n",
       "   'title': 'Transition metal-triggered immunity via an Arabidopsis NLR pair',\n",
       "   'summary': 'Plants growing in soil are challenged by multiple biotic (e.g. pathogens) and abiotic (e.g. heavy metals) stressors. Stress resistance is a key determinant for plants to thrive in the environment. Resistance to pathogens requires innate immunity , whereas tolerance to metal ions is accomplished by mechanisms such as complexation and compartmentation. Some transition metals can enhance plants defense against pathogens4,5, but the mechanism remains unclear. Here, we show that an Arabidopsis head-to-head gene pair of intracellular nucleotide-binding leucine-rich repeat (NLR) receptors antagonistically control transition metal-triggered immunity. One NLR, STM2 directly perceives transition metal ions, such as Cd2+, Cu2+ and Zn2+, as a ligand to activate its NAD+ hydrolytic activity and immune responses, triggering enhanced resistance to the soil-borne bacterial wilt pathogen Ralstonia solanacearum. The other NLR, STM1 suppresses STM2 to protect plants from transition metal-triggered immunity and growth inhibition in the presence of excess metals. STM1 also dampens resistance to the pathogen. Our study defines an NLR activated by transition metals and reveals a trade-off between resistance to pathogens and tolerance to transition metals that are pervasive in soil.',\n",
       "   'gpl': '26208',\n",
       "   'gse': '291226',\n",
       "   'taxon': 'Arabidopsis thaliana',\n",
       "   'entrytype': 'GSE',\n",
       "   'gdstype': 'Expression profiling by high throughput sequencing',\n",
       "   'ptechtype': '',\n",
       "   'valtype': '',\n",
       "   'ssinfo': '',\n",
       "   'subsetinfo': '',\n",
       "   'pdat': '2025/03/11',\n",
       "   'suppfile': 'TXT',\n",
       "   'samples': [{'accession': 'GSM8831173', 'title': 'R8stm1_1Con'},\n",
       "    {'accession': 'GSM8831193', 'title': 'S28Col_0Cd'},\n",
       "    {'accession': 'GSM8831176', 'title': 'R11stm1_1Cd'},\n",
       "    {'accession': 'GSM8831213', 'title': 'S48stm2 stm1Cd'},\n",
       "    {'accession': 'GSM8831196', 'title': 'S31stm1_1Con'},\n",
       "    {'accession': 'GSM8831199', 'title': 'S34stm1_1Cd'},\n",
       "    {'accession': 'GSM8831182', 'title': 'R17stm2_2Cd'},\n",
       "    {'accession': 'GSM8831202', 'title': 'S37stm2_2Con'},\n",
       "    {'accession': 'GSM8831185', 'title': 'R20stm2 stm1Con'},\n",
       "    {'accession': 'GSM8831205', 'title': 'S40stm2_2Cd'},\n",
       "    {'accession': 'GSM8831179', 'title': 'R14stm2_2Con'},\n",
       "    {'accession': 'GSM8831211', 'title': 'S46stm2 stm1Cd'},\n",
       "    {'accession': 'GSM8831191', 'title': 'S26Col_0Con'},\n",
       "    {'accession': 'GSM8831194', 'title': 'S29Col_0Cd'},\n",
       "    {'accession': 'GSM8831188', 'title': 'R23stm2 stm1Cd'},\n",
       "    {'accession': 'GSM8831168', 'title': 'R3Col_0Con'},\n",
       "    {'accession': 'GSM8831171', 'title': 'R6Col_0Cd'},\n",
       "    {'accession': 'GSM8831208', 'title': 'S43stm2 stm1Con'},\n",
       "    {'accession': 'GSM8831200', 'title': 'S35stm1_1Cd'},\n",
       "    {'accession': 'GSM8831174', 'title': 'R9stm1_1Con'},\n",
       "    {'accession': 'GSM8831177', 'title': 'R12stm1_1Cd'},\n",
       "    {'accession': 'GSM8831197', 'title': 'S32stm1_1Con'},\n",
       "    {'accession': 'GSM8831180', 'title': 'R15stm2_2Con'},\n",
       "    {'accession': 'GSM8831183', 'title': 'R18stm2_2Cd'},\n",
       "    {'accession': 'GSM8831166', 'title': 'R1Col_0Con'},\n",
       "    {'accession': 'GSM8831186', 'title': 'R21stm2 stm1Con'},\n",
       "    {'accession': 'GSM8831203', 'title': 'S38stm2_2Con'},\n",
       "    {'accession': 'GSM8831189', 'title': 'R24stm2 stm1Cd'},\n",
       "    {'accession': 'GSM8831206', 'title': 'S41stm2_2Cd'},\n",
       "    {'accession': 'GSM8831192', 'title': 'S27Col_0Con'},\n",
       "    {'accession': 'GSM8831209', 'title': 'S44stm2 stm1Con'},\n",
       "    {'accession': 'GSM8831175', 'title': 'R10stm1_1Cd'},\n",
       "    {'accession': 'GSM8831195', 'title': 'S30Col_0Cd'},\n",
       "    {'accession': 'GSM8831212', 'title': 'S47stm2 stm1Cd'},\n",
       "    {'accession': 'GSM8831169', 'title': 'R4Col_0Cd'},\n",
       "    {'accession': 'GSM8831172', 'title': 'R7stm1_1Con'},\n",
       "    {'accession': 'GSM8831201', 'title': 'S36stm1_1Cd'},\n",
       "    {'accession': 'GSM8831204', 'title': 'S39stm2_2Con'},\n",
       "    {'accession': 'GSM8831184', 'title': 'R19stm2 stm1Con'},\n",
       "    {'accession': 'GSM8831198', 'title': 'S33stm1_1Con'},\n",
       "    {'accession': 'GSM8831178', 'title': 'R13stm2_2Con'},\n",
       "    {'accession': 'GSM8831181', 'title': 'R16stm2_2Cd'},\n",
       "    {'accession': 'GSM8831210', 'title': 'S45stm2 stm1Con'},\n",
       "    {'accession': 'GSM8831170', 'title': 'R5Col_0Cd'},\n",
       "    {'accession': 'GSM8831167', 'title': 'R2Col_0Con'},\n",
       "    {'accession': 'GSM8831187', 'title': 'R22stm2 stm1Cd'},\n",
       "    {'accession': 'GSM8831207', 'title': 'S42stm2_2Cd'},\n",
       "    {'accession': 'GSM8831190', 'title': 'S25Col_0Con'}],\n",
       "   'relations': [],\n",
       "   'extrelations': [],\n",
       "   'n_samples': 48,\n",
       "   'seriestitle': '',\n",
       "   'platformtitle': '',\n",
       "   'platformtaxa': '',\n",
       "   'samplestaxa': '',\n",
       "   'pubmedids': [],\n",
       "   'projects': [],\n",
       "   'ftplink': 'ftp://ftp.ncbi.nlm.nih.gov/geo/series/GSE291nnn/GSE291226/',\n",
       "   'geo2r': 'no',\n",
       "   'bioproject': 'PRJNA1232609'}}}"
      ]
     },
     "execution_count": 16,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "execution_count": 16
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-12T17:21:37.114331Z",
     "start_time": "2025-03-12T16:56:11.402155Z"
    }
   },
   "cell_type": "code",
   "source": [
    "GEO_series_data = {}\n",
    "i = 0\n",
    "mxm = 300\n",
    "while True:\n",
    "    print(f\"Progress: {i / len(gse_ids) * 100} %\")\n",
    "    if i > len(gse_ids):\n",
    "        break\n",
    "    try:\n",
    "        new_data = fetch_GEO_data(gse_ids,\"https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esummary.fcgi\",i,mxm)\n",
    "        #print(f\"Data: {new_data}\")\n",
    "        for uid, details in new_data[\"result\"].items():\n",
    "            if uid == \"uids\":  # Ignore metadata key\n",
    "                continue\n",
    "            GEO_series_data[uid] = details\n",
    "    except:\n",
    "        print(f\"Error at {i}, {mxm}\")\n",
    "            \n",
    "    i += 300\n",
    "    mxm += 300\n",
    "    time.sleep(0.005) "
   ],
   "id": "c996628cbd0fba70",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Progress: 0.0 %\n",
      "Progress: 0.12057490112858107 %\n",
      "Progress: 0.24114980225716215 %\n",
      "Progress: 0.3617247033857432 %\n",
      "Progress: 0.4822996045143243 %\n",
      "Progress: 0.6028745056429053 %\n",
      "Progress: 0.7234494067714864 %\n",
      "Progress: 0.8440243079000675 %\n",
      "Progress: 0.9645992090286486 %\n",
      "Progress: 1.0851741101572296 %\n",
      "Progress: 1.2057490112858107 %\n",
      "Progress: 1.326323912414392 %\n",
      "Progress: 1.4468988135429728 %\n",
      "Progress: 1.567473714671554 %\n",
      "Progress: 1.688048615800135 %\n",
      "Progress: 1.808623516928716 %\n",
      "Progress: 1.9291984180572972 %\n",
      "Progress: 2.0497733191858782 %\n",
      "Progress: 2.1703482203144593 %\n",
      "Progress: 2.2909231214430403 %\n",
      "Progress: 2.4114980225716214 %\n",
      "Progress: 2.5320729237002024 %\n",
      "Progress: 2.652647824828784 %\n",
      "Progress: 2.7732227259573645 %\n",
      "Progress: 2.8937976270859456 %\n",
      "Progress: 3.014372528214527 %\n",
      "Progress: 3.134947429343108 %\n",
      "Progress: 3.255522330471689 %\n",
      "Progress: 3.37609723160027 %\n",
      "Progress: 3.4966721327288517 %\n",
      "Progress: 3.617247033857432 %\n",
      "Progress: 3.7378219349860133 %\n",
      "Progress: 3.8583968361145944 %\n",
      "Progress: 3.9789717372431754 %\n",
      "Progress: 4.0995466383717565 %\n",
      "Progress: 4.2201215395003375 %\n",
      "Progress: 4.340696440628919 %\n",
      "Progress: 4.4612713417575005 %\n",
      "Progress: 4.581846242886081 %\n",
      "Progress: 4.702421144014662 %\n",
      "Progress: 4.822996045143243 %\n",
      "Progress: 4.943570946271824 %\n",
      "Progress: 5.064145847400405 %\n",
      "Progress: 5.184720748528986 %\n",
      "Progress: 5.305295649657568 %\n",
      "Progress: 5.425870550786149 %\n",
      "Progress: 5.546445451914729 %\n",
      "Progress: 5.66702035304331 %\n",
      "Progress: 5.787595254171891 %\n",
      "Progress: 5.908170155300472 %\n",
      "Progress: 6.028745056429054 %\n",
      "Progress: 6.149319957557635 %\n",
      "Progress: 6.269894858686216 %\n",
      "Progress: 6.390469759814797 %\n",
      "Progress: 6.511044660943378 %\n",
      "Progress: 6.631619562071959 %\n",
      "Progress: 6.75219446320054 %\n",
      "Progress: 6.8727693643291214 %\n",
      "Progress: 6.993344265457703 %\n",
      "Progress: 7.113919166586283 %\n",
      "Progress: 7.234494067714864 %\n",
      "Progress: 7.355068968843445 %\n",
      "Progress: 7.475643869972027 %\n",
      "Progress: 7.596218771100608 %\n",
      "Progress: 7.716793672229189 %\n",
      "Progress: 7.83736857335777 %\n",
      "Progress: 7.957943474486351 %\n",
      "Progress: 8.078518375614932 %\n",
      "Progress: 8.199093276743513 %\n",
      "Progress: 8.319668177872094 %\n",
      "Progress: 8.440243079000675 %\n",
      "Progress: 8.560817980129256 %\n",
      "Progress: 8.681392881257837 %\n",
      "Progress: 8.801967782386418 %\n",
      "Progress: 8.922542683515001 %\n",
      "Progress: 9.043117584643582 %\n",
      "Progress: 9.163692485772161 %\n",
      "Progress: 9.284267386900742 %\n",
      "Progress: 9.404842288029323 %\n",
      "Progress: 9.525417189157904 %\n",
      "Progress: 9.645992090286486 %\n",
      "Progress: 9.766566991415067 %\n",
      "Progress: 9.887141892543648 %\n",
      "Progress: 10.007716793672229 %\n",
      "Progress: 10.12829169480081 %\n",
      "Progress: 10.24886659592939 %\n",
      "Progress: 10.369441497057972 %\n",
      "Progress: 10.490016398186555 %\n",
      "Progress: 10.610591299315136 %\n",
      "Progress: 10.731166200443717 %\n",
      "Progress: 10.851741101572298 %\n",
      "Progress: 10.972316002700879 %\n",
      "Progress: 11.092890903829458 %\n",
      "Progress: 11.21346580495804 %\n",
      "Progress: 11.33404070608662 %\n",
      "Progress: 11.454615607215201 %\n",
      "Progress: 11.575190508343782 %\n",
      "Progress: 11.695765409472363 %\n",
      "Progress: 11.816340310600944 %\n",
      "Progress: 11.936915211729527 %\n",
      "Progress: 12.057490112858108 %\n",
      "Progress: 12.17806501398669 %\n",
      "Progress: 12.29863991511527 %\n",
      "Progress: 12.419214816243851 %\n",
      "Progress: 12.539789717372432 %\n",
      "Progress: 12.660364618501013 %\n",
      "Progress: 12.780939519629595 %\n",
      "Progress: 12.901514420758176 %\n",
      "Progress: 13.022089321886757 %\n",
      "Progress: 13.142664223015338 %\n",
      "Progress: 13.263239124143919 %\n",
      "Progress: 13.3838140252725 %\n",
      "Progress: 13.50438892640108 %\n",
      "Progress: 13.624963827529662 %\n",
      "Progress: 13.745538728658243 %\n",
      "Progress: 13.866113629786824 %\n",
      "Progress: 13.986688530915407 %\n",
      "Progress: 14.107263432043984 %\n",
      "Progress: 14.227838333172565 %\n",
      "Progress: 14.348413234301146 %\n",
      "Progress: 14.468988135429727 %\n",
      "Progress: 14.589563036558308 %\n",
      "Progress: 14.71013793768689 %\n",
      "Progress: 14.83071283881547 %\n",
      "Progress: 14.951287739944053 %\n",
      "Progress: 15.071862641072634 %\n",
      "Progress: 15.192437542201215 %\n",
      "Progress: 15.313012443329796 %\n",
      "Progress: 15.433587344458378 %\n",
      "Progress: 15.554162245586959 %\n",
      "Progress: 15.67473714671554 %\n",
      "Progress: 15.79531204784412 %\n",
      "Progress: 15.915886948972702 %\n",
      "Progress: 16.036461850101283 %\n",
      "Progress: 16.157036751229864 %\n",
      "Progress: 16.277611652358445 %\n",
      "Progress: 16.398186553487026 %\n",
      "Progress: 16.518761454615607 %\n",
      "Progress: 16.639336355744188 %\n",
      "Progress: 16.75991125687277 %\n",
      "Progress: 16.88048615800135 %\n",
      "Progress: 17.00106105912993 %\n",
      "Progress: 17.121635960258512 %\n",
      "Progress: 17.242210861387093 %\n",
      "Progress: 17.362785762515674 %\n",
      "Progress: 17.483360663644255 %\n",
      "Progress: 17.603935564772836 %\n",
      "Progress: 17.724510465901417 %\n",
      "Progress: 17.845085367030002 %\n",
      "Progress: 17.965660268158583 %\n",
      "Progress: 18.086235169287164 %\n",
      "Progress: 18.20681007041574 %\n",
      "Error at 45300, 45600\n",
      "Progress: 18.327384971544323 %\n",
      "Progress: 18.447959872672904 %\n",
      "Progress: 18.568534773801485 %\n",
      "Progress: 18.689109674930066 %\n",
      "Progress: 18.809684576058647 %\n",
      "Progress: 18.930259477187228 %\n",
      "Progress: 19.05083437831581 %\n",
      "Progress: 19.17140927944439 %\n",
      "Progress: 19.29198418057297 %\n",
      "Progress: 19.412559081701552 %\n",
      "Progress: 19.533133982830133 %\n",
      "Progress: 19.653708883958714 %\n",
      "Progress: 19.774283785087295 %\n",
      "Progress: 19.894858686215876 %\n",
      "Progress: 20.015433587344457 %\n",
      "Progress: 20.13600848847304 %\n",
      "Progress: 20.25658338960162 %\n",
      "Progress: 20.3771582907302 %\n",
      "Progress: 20.49773319185878 %\n",
      "Progress: 20.618308092987363 %\n",
      "Progress: 20.738882994115944 %\n",
      "Progress: 20.859457895244525 %\n",
      "Progress: 20.98003279637311 %\n",
      "Progress: 21.10060769750169 %\n",
      "Progress: 21.22118259863027 %\n",
      "Progress: 21.341757499758852 %\n",
      "Progress: 21.462332400887433 %\n",
      "Progress: 21.582907302016014 %\n",
      "Progress: 21.703482203144596 %\n",
      "Progress: 21.824057104273177 %\n",
      "Progress: 21.944632005401758 %\n",
      "Progress: 22.06520690653034 %\n",
      "Progress: 22.185781807658916 %\n",
      "Progress: 22.306356708787497 %\n",
      "Progress: 22.42693160991608 %\n",
      "Progress: 22.54750651104466 %\n",
      "Progress: 22.66808141217324 %\n",
      "Progress: 22.78865631330182 %\n",
      "Progress: 22.909231214430402 %\n",
      "Progress: 23.029806115558983 %\n",
      "Progress: 23.150381016687565 %\n",
      "Progress: 23.270955917816146 %\n",
      "Progress: 23.391530818944727 %\n",
      "Progress: 23.512105720073308 %\n",
      "Progress: 23.63268062120189 %\n",
      "Progress: 23.75325552233047 %\n",
      "Progress: 23.873830423459054 %\n",
      "Progress: 23.994405324587635 %\n",
      "Progress: 24.114980225716216 %\n",
      "Progress: 24.235555126844798 %\n",
      "Progress: 24.35613002797338 %\n",
      "Progress: 24.47670492910196 %\n",
      "Progress: 24.59727983023054 %\n",
      "Progress: 24.71785473135912 %\n",
      "Progress: 24.838429632487703 %\n",
      "Progress: 24.959004533616284 %\n",
      "Progress: 25.079579434744865 %\n",
      "Progress: 25.200154335873442 %\n",
      "Progress: 25.320729237002027 %\n",
      "Progress: 25.441304138130604 %\n",
      "Progress: 25.56187903925919 %\n",
      "Progress: 25.682453940387767 %\n",
      "Progress: 25.80302884151635 %\n",
      "Progress: 25.92360374264493 %\n",
      "Progress: 26.044178643773513 %\n",
      "Progress: 26.16475354490209 %\n",
      "Progress: 26.285328446030675 %\n",
      "Progress: 26.405903347159253 %\n",
      "Progress: 26.526478248287837 %\n",
      "Progress: 26.647053149416415 %\n",
      "Progress: 26.767628050545 %\n",
      "Progress: 26.888202951673577 %\n",
      "Progress: 27.00877785280216 %\n",
      "Progress: 27.129352753930743 %\n",
      "Progress: 27.249927655059324 %\n",
      "Progress: 27.370502556187905 %\n",
      "Progress: 27.491077457316486 %\n",
      "Progress: 27.611652358445067 %\n",
      "Progress: 27.732227259573648 %\n",
      "Progress: 27.85280216070223 %\n",
      "Progress: 27.973377061830814 %\n",
      "Progress: 28.09395196295939 %\n",
      "Progress: 28.21452686408797 %\n",
      "Progress: 28.335101765216553 %\n",
      "Progress: 28.45567666634513 %\n",
      "Progress: 28.576251567473715 %\n",
      "Progress: 28.696826468602293 %\n",
      "Progress: 28.817401369730877 %\n",
      "Progress: 28.937976270859455 %\n",
      "Progress: 29.05855117198804 %\n",
      "Progress: 29.179126073116617 %\n",
      "Progress: 29.2997009742452 %\n",
      "Progress: 29.42027587537378 %\n",
      "Progress: 29.540850776502364 %\n",
      "Progress: 29.66142567763094 %\n",
      "Progress: 29.782000578759526 %\n",
      "Progress: 29.902575479888107 %\n",
      "Progress: 30.023150381016688 %\n",
      "Progress: 30.14372528214527 %\n",
      "Progress: 30.26430018327385 %\n",
      "Progress: 30.38487508440243 %\n",
      "Progress: 30.505449985531012 %\n",
      "Progress: 30.626024886659593 %\n",
      "Progress: 30.746599787788178 %\n",
      "Progress: 30.867174688916755 %\n",
      "Progress: 30.98774959004534 %\n",
      "Progress: 31.108324491173917 %\n",
      "Progress: 31.228899392302502 %\n",
      "Progress: 31.34947429343108 %\n",
      "Progress: 31.470049194559664 %\n",
      "Progress: 31.59062409568824 %\n",
      "Progress: 31.711198996816826 %\n",
      "Progress: 31.831773897945403 %\n",
      "Progress: 31.952348799073988 %\n",
      "Progress: 32.072923700202566 %\n",
      "Progress: 32.19349860133114 %\n",
      "Progress: 32.31407350245973 %\n",
      "Progress: 32.434648403588305 %\n",
      "Progress: 32.55522330471689 %\n",
      "Progress: 32.67579820584547 %\n",
      "Progress: 32.79637310697405 %\n",
      "Progress: 32.91694800810263 %\n",
      "Progress: 33.037522909231214 %\n",
      "Progress: 33.15809781035979 %\n",
      "Progress: 33.278672711488376 %\n",
      "Progress: 33.39924761261695 %\n",
      "Progress: 33.51982251374554 %\n",
      "Progress: 33.640397414874116 %\n",
      "Progress: 33.7609723160027 %\n",
      "Progress: 33.88154721713128 %\n",
      "Progress: 34.00212211825986 %\n",
      "Progress: 34.12269701938844 %\n",
      "Progress: 34.243271920517024 %\n",
      "Progress: 34.3638468216456 %\n",
      "Progress: 34.48442172277419 %\n",
      "Progress: 34.604996623902764 %\n",
      "Progress: 34.72557152503135 %\n",
      "Progress: 34.84614642615993 %\n",
      "Progress: 34.96672132728851 %\n",
      "Progress: 35.087296228417095 %\n",
      "Progress: 35.20787112954567 %\n",
      "Progress: 35.32844603067426 %\n",
      "Progress: 35.449020931802835 %\n",
      "Progress: 35.56959583293142 %\n",
      "Progress: 35.690170734060004 %\n",
      "Progress: 35.81074563518858 %\n",
      "Progress: 35.931320536317166 %\n",
      "Progress: 36.051895437445744 %\n",
      "Progress: 36.17247033857433 %\n",
      "Progress: 36.293045239702906 %\n",
      "Progress: 36.41362014083148 %\n",
      "Progress: 36.53419504196007 %\n",
      "Progress: 36.654769943088645 %\n",
      "Progress: 36.77534484421723 %\n",
      "Progress: 36.89591974534581 %\n",
      "Progress: 37.01649464647439 %\n",
      "Progress: 37.13706954760297 %\n",
      "Progress: 37.257644448731554 %\n",
      "Progress: 37.37821934986013 %\n",
      "Progress: 37.498794250988716 %\n",
      "Progress: 37.619369152117294 %\n",
      "Progress: 37.73994405324588 %\n",
      "Progress: 37.860518954374456 %\n",
      "Progress: 37.98109385550304 %\n",
      "Progress: 38.10166875663162 %\n",
      "Progress: 38.2222436577602 %\n",
      "Progress: 38.34281855888878 %\n",
      "Progress: 38.463393460017365 %\n",
      "Progress: 38.58396836114594 %\n",
      "Progress: 38.70454326227453 %\n",
      "Progress: 38.825118163403104 %\n",
      "Progress: 38.94569306453169 %\n",
      "Progress: 39.066267965660266 %\n",
      "Progress: 39.18684286678885 %\n",
      "Progress: 39.30741776791743 %\n",
      "Progress: 39.42799266904601 %\n",
      "Progress: 39.54856757017459 %\n",
      "Progress: 39.669142471303175 %\n",
      "Progress: 39.78971737243175 %\n",
      "Progress: 39.91029227356034 %\n",
      "Progress: 40.030867174688915 %\n",
      "Progress: 40.1514420758175 %\n",
      "Progress: 40.27201697694608 %\n",
      "Progress: 40.392591878074654 %\n",
      "Progress: 40.51316677920324 %\n",
      "Progress: 40.633741680331816 %\n",
      "Progress: 40.7543165814604 %\n",
      "Progress: 40.87489148258898 %\n",
      "Progress: 40.99546638371756 %\n",
      "Progress: 41.11604128484615 %\n",
      "Progress: 41.236616185974725 %\n",
      "Progress: 41.35719108710331 %\n",
      "Progress: 41.47776598823189 %\n",
      "Progress: 41.59834088936047 %\n",
      "Progress: 41.71891579048905 %\n",
      "Progress: 41.839490691617634 %\n",
      "Progress: 41.96006559274622 %\n",
      "Progress: 42.080640493874796 %\n",
      "Progress: 42.20121539500338 %\n",
      "Progress: 42.32179029613196 %\n",
      "Progress: 42.44236519726054 %\n",
      "Progress: 42.56294009838912 %\n",
      "Progress: 42.683514999517705 %\n",
      "Progress: 42.80408990064628 %\n",
      "Progress: 42.92466480177487 %\n",
      "Progress: 43.045239702903444 %\n",
      "Progress: 43.16581460403203 %\n",
      "Progress: 43.28638950516061 %\n",
      "Progress: 43.40696440628919 %\n",
      "Progress: 43.52753930741777 %\n",
      "Progress: 43.64811420854635 %\n",
      "Progress: 43.76868910967493 %\n",
      "Progress: 43.889264010803515 %\n",
      "Progress: 44.00983891193209 %\n",
      "Progress: 44.13041381306068 %\n",
      "Progress: 44.250988714189255 %\n",
      "Progress: 44.37156361531783 %\n",
      "Progress: 44.49213851644642 %\n",
      "Progress: 44.612713417574994 %\n",
      "Progress: 44.73328831870358 %\n",
      "Progress: 44.85386321983216 %\n",
      "Progress: 44.97443812096074 %\n",
      "Progress: 45.09501302208932 %\n",
      "Progress: 45.2155879232179 %\n",
      "Progress: 45.33616282434648 %\n",
      "Progress: 45.456737725475065 %\n",
      "Progress: 45.57731262660364 %\n",
      "Progress: 45.69788752773223 %\n",
      "Progress: 45.818462428860805 %\n",
      "Progress: 45.93903732998939 %\n",
      "Progress: 46.05961223111797 %\n",
      "Progress: 46.18018713224655 %\n",
      "Progress: 46.30076203337513 %\n",
      "Progress: 46.421336934503714 %\n",
      "Progress: 46.54191183563229 %\n",
      "Progress: 46.662486736760876 %\n",
      "Progress: 46.78306163788945 %\n",
      "Progress: 46.90363653901804 %\n",
      "Progress: 47.024211440146615 %\n",
      "Progress: 47.1447863412752 %\n",
      "Progress: 47.26536124240378 %\n",
      "Progress: 47.38593614353236 %\n",
      "Progress: 47.50651104466094 %\n",
      "Progress: 47.627085945789524 %\n",
      "Progress: 47.74766084691811 %\n",
      "Progress: 47.868235748046686 %\n",
      "Progress: 47.98881064917527 %\n",
      "Progress: 48.10938555030385 %\n",
      "Progress: 48.22996045143243 %\n",
      "Progress: 48.35053535256101 %\n",
      "Progress: 48.471110253689595 %\n",
      "Progress: 48.59168515481817 %\n",
      "Progress: 48.71226005594676 %\n",
      "Progress: 48.832834957075335 %\n",
      "Progress: 48.95340985820392 %\n",
      "Progress: 49.0739847593325 %\n",
      "Progress: 49.19455966046108 %\n",
      "Progress: 49.31513456158966 %\n",
      "Progress: 49.43570946271824 %\n",
      "Progress: 49.55628436384682 %\n",
      "Progress: 49.676859264975405 %\n",
      "Progress: 49.79743416610398 %\n",
      "Progress: 49.91800906723257 %\n",
      "Progress: 50.03858396836115 %\n",
      "Progress: 50.15915886948973 %\n",
      "Progress: 50.27973377061831 %\n",
      "Progress: 50.400308671746885 %\n",
      "Progress: 50.520883572875476 %\n",
      "Progress: 50.641458474004054 %\n",
      "Progress: 50.76203337513263 %\n",
      "Progress: 50.88260827626121 %\n",
      "Progress: 51.0031831773898 %\n",
      "Progress: 51.12375807851838 %\n",
      "Progress: 51.244332979646956 %\n",
      "Progress: 51.36490788077553 %\n",
      "Progress: 51.485482781904125 %\n",
      "Progress: 51.6060576830327 %\n",
      "Progress: 51.72663258416128 %\n",
      "Progress: 51.84720748528986 %\n",
      "Progress: 51.96778238641845 %\n",
      "Progress: 52.088357287547026 %\n",
      "Progress: 52.208932188675604 %\n",
      "Progress: 52.32950708980418 %\n",
      "Progress: 52.45008199093276 %\n",
      "Progress: 52.57065689206135 %\n",
      "Progress: 52.69123179318993 %\n",
      "Progress: 52.811806694318506 %\n",
      "Progress: 52.93238159544708 %\n",
      "Progress: 53.052956496575675 %\n",
      "Progress: 53.17353139770425 %\n",
      "Progress: 53.29410629883283 %\n",
      "Progress: 53.414681199961414 %\n",
      "Progress: 53.53525610109 %\n",
      "Progress: 53.65583100221858 %\n",
      "Progress: 53.776405903347154 %\n",
      "Progress: 53.89698080447574 %\n",
      "Progress: 54.01755570560432 %\n",
      "Progress: 54.1381306067329 %\n",
      "Progress: 54.258705507861485 %\n",
      "Progress: 54.37928040899006 %\n",
      "Progress: 54.49985531011865 %\n",
      "Progress: 54.620430211247225 %\n",
      "Progress: 54.74100511237581 %\n",
      "Progress: 54.86158001350439 %\n",
      "Progress: 54.98215491463297 %\n",
      "Progress: 55.102729815761556 %\n",
      "Progress: 55.223304716890134 %\n",
      "Progress: 55.34387961801871 %\n",
      "Progress: 55.464454519147296 %\n",
      "Progress: 55.58502942027588 %\n",
      "Progress: 55.70560432140446 %\n",
      "Progress: 55.826179222533035 %\n",
      "Progress: 55.94675412366163 %\n",
      "Progress: 56.067329024790205 %\n",
      "Progress: 56.18790392591878 %\n",
      "Progress: 56.30847882704736 %\n",
      "Progress: 56.42905372817594 %\n",
      "Progress: 56.54962862930453 %\n",
      "Progress: 56.670203530433106 %\n",
      "Progress: 56.790778431561684 %\n",
      "Progress: 56.91135333269026 %\n",
      "Progress: 57.03192823381885 %\n",
      "Progress: 57.15250313494743 %\n",
      "Progress: 57.27307803607601 %\n",
      "Progress: 57.393652937204585 %\n",
      "Progress: 57.51422783833318 %\n",
      "Progress: 57.634802739461755 %\n",
      "Progress: 57.75537764059033 %\n",
      "Progress: 57.87595254171891 %\n",
      "Progress: 57.9965274428475 %\n",
      "Progress: 58.11710234397608 %\n",
      "Progress: 58.237677245104656 %\n",
      "Progress: 58.358252146233234 %\n",
      "Progress: 58.478827047361825 %\n",
      "Progress: 58.5994019484904 %\n",
      "Progress: 58.71997684961898 %\n",
      "Progress: 58.84055175074756 %\n",
      "Progress: 58.96112665187615 %\n",
      "Progress: 59.08170155300473 %\n",
      "Progress: 59.202276454133305 %\n",
      "Progress: 59.32285135526188 %\n",
      "Progress: 59.443426256390474 %\n",
      "Progress: 59.56400115751905 %\n",
      "Progress: 59.68457605864763 %\n",
      "Progress: 59.80515095977621 %\n",
      "Progress: 59.9257258609048 %\n",
      "Progress: 60.046300762033376 %\n",
      "Progress: 60.16687566316195 %\n",
      "Progress: 60.28745056429054 %\n",
      "Progress: 60.408025465419115 %\n",
      "Progress: 60.5286003665477 %\n",
      "Progress: 60.649175267676284 %\n",
      "Progress: 60.76975016880486 %\n",
      "Progress: 60.89032506993344 %\n",
      "Progress: 61.010899971062024 %\n",
      "Progress: 61.13147487219061 %\n",
      "Progress: 61.252049773319186 %\n",
      "Progress: 61.37262467444776 %\n",
      "Progress: 61.493199575576355 %\n",
      "Progress: 61.61377447670493 %\n",
      "Progress: 61.73434937783351 %\n",
      "Progress: 61.85492427896209 %\n",
      "Progress: 61.97549918009068 %\n",
      "Progress: 62.09607408121926 %\n",
      "Progress: 62.216648982347834 %\n",
      "Progress: 62.33722388347641 %\n",
      "Progress: 62.457798784605004 %\n",
      "Progress: 62.57837368573358 %\n",
      "Progress: 62.69894858686216 %\n",
      "Progress: 62.819523487990736 %\n",
      "Progress: 62.94009838911933 %\n",
      "Progress: 63.060673290247905 %\n",
      "Progress: 63.18124819137648 %\n",
      "Progress: 63.30182309250506 %\n",
      "Progress: 63.42239799363365 %\n",
      "Progress: 63.54297289476223 %\n",
      "Progress: 63.66354779589081 %\n",
      "Progress: 63.784122697019384 %\n",
      "Progress: 63.904697598147976 %\n",
      "Progress: 64.02527249927655 %\n",
      "Progress: 64.14584740040513 %\n",
      "Progress: 64.26642230153371 %\n",
      "Progress: 64.38699720266229 %\n",
      "Progress: 64.50757210379088 %\n",
      "Progress: 64.62814700491946 %\n",
      "Progress: 64.74872190604803 %\n",
      "Progress: 64.86929680717661 %\n",
      "Progress: 64.9898717083052 %\n",
      "Progress: 65.11044660943378 %\n",
      "Progress: 65.23102151056236 %\n",
      "Progress: 65.35159641169093 %\n",
      "Progress: 65.47217131281953 %\n",
      "Progress: 65.5927462139481 %\n",
      "Progress: 65.71332111507668 %\n",
      "Progress: 65.83389601620526 %\n",
      "Progress: 65.95447091733385 %\n",
      "Progress: 66.07504581846243 %\n",
      "Progress: 66.195620719591 %\n",
      "Progress: 66.31619562071958 %\n",
      "Progress: 66.43677052184817 %\n",
      "Progress: 66.55734542297675 %\n",
      "Progress: 66.67792032410533 %\n",
      "Progress: 66.7984952252339 %\n",
      "Progress: 66.9190701263625 %\n",
      "Progress: 67.03964502749108 %\n",
      "Progress: 67.16021992861965 %\n",
      "Progress: 67.28079482974823 %\n",
      "Progress: 67.40136973087682 %\n",
      "Progress: 67.5219446320054 %\n",
      "Progress: 67.64251953313398 %\n",
      "Progress: 67.76309443426256 %\n",
      "Progress: 67.88366933539115 %\n",
      "Progress: 68.00424423651972 %\n",
      "Progress: 68.1248191376483 %\n",
      "Progress: 68.24539403877688 %\n",
      "Progress: 68.36596893990546 %\n",
      "Progress: 68.48654384103405 %\n",
      "Progress: 68.60711874216263 %\n",
      "Progress: 68.7276936432912 %\n",
      "Progress: 68.8482685444198 %\n",
      "Progress: 68.96884344554837 %\n",
      "Progress: 69.08941834667695 %\n",
      "Progress: 69.20999324780553 %\n",
      "Progress: 69.33056814893412 %\n",
      "Progress: 69.4511430500627 %\n",
      "Progress: 69.57171795119127 %\n",
      "Progress: 69.69229285231987 %\n",
      "Progress: 69.81286775344844 %\n",
      "Progress: 69.93344265457702 %\n",
      "Progress: 70.0540175557056 %\n",
      "Progress: 70.17459245683419 %\n",
      "Progress: 70.29516735796277 %\n",
      "Progress: 70.41574225909135 %\n",
      "Progress: 70.53631716021994 %\n",
      "Progress: 70.65689206134851 %\n",
      "Progress: 70.77746696247709 %\n",
      "Progress: 70.89804186360567 %\n",
      "Progress: 71.01861676473426 %\n",
      "Progress: 71.13919166586284 %\n",
      "Progress: 71.25976656699142 %\n",
      "Progress: 71.38034146812001 %\n",
      "Progress: 71.50091636924859 %\n",
      "Progress: 71.62149127037716 %\n",
      "Progress: 71.74206617150574 %\n",
      "Progress: 71.86264107263433 %\n",
      "Progress: 71.98321597376291 %\n",
      "Progress: 72.10379087489149 %\n",
      "Progress: 72.22436577602006 %\n",
      "Progress: 72.34494067714866 %\n",
      "Progress: 72.46551557827723 %\n",
      "Progress: 72.58609047940581 %\n",
      "Progress: 72.70666538053439 %\n",
      "Progress: 72.82724028166297 %\n",
      "Progress: 72.94781518279156 %\n",
      "Progress: 73.06839008392014 %\n",
      "Progress: 73.18896498504871 %\n",
      "Progress: 73.30953988617729 %\n",
      "Progress: 73.43011478730588 %\n",
      "Progress: 73.55068968843446 %\n",
      "Progress: 73.67126458956304 %\n",
      "Progress: 73.79183949069161 %\n",
      "Progress: 73.9124143918202 %\n",
      "Progress: 74.03298929294878 %\n",
      "Progress: 74.15356419407736 %\n",
      "Progress: 74.27413909520594 %\n",
      "Progress: 74.39471399633453 %\n",
      "Progress: 74.51528889746311 %\n",
      "Progress: 74.63586379859169 %\n",
      "Progress: 74.75643869972026 %\n",
      "Progress: 74.87701360084885 %\n",
      "Progress: 74.99758850197743 %\n",
      "Progress: 75.11816340310601 %\n",
      "Progress: 75.23873830423459 %\n",
      "Progress: 75.35931320536318 %\n",
      "Progress: 75.47988810649176 %\n",
      "Progress: 75.60046300762033 %\n",
      "Progress: 75.72103790874891 %\n",
      "Progress: 75.8416128098775 %\n",
      "Progress: 75.96218771100608 %\n",
      "Progress: 76.08276261213466 %\n",
      "Progress: 76.20333751326324 %\n",
      "Progress: 76.32391241439183 %\n",
      "Progress: 76.4444873155204 %\n",
      "Progress: 76.56506221664898 %\n",
      "Progress: 76.68563711777756 %\n",
      "Progress: 76.80621201890614 %\n",
      "Progress: 76.92678692003473 %\n",
      "Progress: 77.0473618211633 %\n",
      "Progress: 77.16793672229188 %\n",
      "Progress: 77.28851162342046 %\n",
      "Progress: 77.40908652454905 %\n",
      "Progress: 77.52966142567763 %\n",
      "Progress: 77.65023632680621 %\n",
      "Progress: 77.77081122793479 %\n",
      "Progress: 77.89138612906338 %\n",
      "Progress: 78.01196103019196 %\n",
      "Progress: 78.13253593132053 %\n",
      "Progress: 78.25311083244911 %\n",
      "Progress: 78.3736857335777 %\n",
      "Progress: 78.49426063470628 %\n",
      "Progress: 78.61483553583486 %\n",
      "Progress: 78.73541043696343 %\n",
      "Progress: 78.85598533809203 %\n",
      "Progress: 78.9765602392206 %\n",
      "Progress: 79.09713514034918 %\n",
      "Progress: 79.21771004147776 %\n",
      "Progress: 79.33828494260635 %\n",
      "Progress: 79.45885984373493 %\n",
      "Progress: 79.5794347448635 %\n",
      "Progress: 79.70000964599208 %\n",
      "Progress: 79.82058454712067 %\n",
      "Progress: 79.94115944824925 %\n",
      "Progress: 80.06173434937783 %\n",
      "Progress: 80.1823092505064 %\n",
      "Progress: 80.302884151635 %\n",
      "Progress: 80.42345905276358 %\n",
      "Progress: 80.54403395389215 %\n",
      "Progress: 80.66460885502073 %\n",
      "Progress: 80.78518375614931 %\n",
      "Progress: 80.9057586572779 %\n",
      "Progress: 81.02633355840648 %\n",
      "Progress: 81.14690845953506 %\n",
      "Progress: 81.26748336066363 %\n",
      "Progress: 81.38805826179222 %\n",
      "Progress: 81.5086331629208 %\n",
      "Progress: 81.62920806404938 %\n",
      "Progress: 81.74978296517796 %\n",
      "Progress: 81.87035786630655 %\n",
      "Progress: 81.99093276743513 %\n",
      "Progress: 82.1115076685637 %\n",
      "Progress: 82.2320825696923 %\n",
      "Progress: 82.35265747082087 %\n",
      "Progress: 82.47323237194945 %\n",
      "Progress: 82.59380727307803 %\n",
      "Progress: 82.71438217420662 %\n",
      "Progress: 82.8349570753352 %\n",
      "Progress: 82.95553197646377 %\n",
      "Progress: 83.07610687759237 %\n",
      "Progress: 83.19668177872094 %\n",
      "Progress: 83.31725667984952 %\n",
      "Progress: 83.4378315809781 %\n",
      "Progress: 83.55840648210669 %\n",
      "Progress: 83.67898138323527 %\n",
      "Progress: 83.79955628436385 %\n",
      "Progress: 83.92013118549244 %\n",
      "Progress: 84.04070608662101 %\n",
      "Progress: 84.16128098774959 %\n",
      "Progress: 84.28185588887817 %\n",
      "Progress: 84.40243079000676 %\n",
      "Progress: 84.52300569113534 %\n",
      "Progress: 84.64358059226392 %\n",
      "Progress: 84.7641554933925 %\n",
      "Progress: 84.88473039452109 %\n",
      "Progress: 85.00530529564966 %\n",
      "Progress: 85.12588019677824 %\n",
      "Progress: 85.24645509790682 %\n",
      "Progress: 85.36702999903541 %\n",
      "Progress: 85.48760490016399 %\n",
      "Progress: 85.60817980129256 %\n",
      "Progress: 85.72875470242114 %\n",
      "Progress: 85.84932960354973 %\n",
      "Progress: 85.96990450467831 %\n",
      "Progress: 86.09047940580689 %\n",
      "Progress: 86.21105430693547 %\n",
      "Progress: 86.33162920806406 %\n",
      "Progress: 86.45220410919264 %\n",
      "Progress: 86.57277901032121 %\n",
      "Progress: 86.69335391144979 %\n",
      "Progress: 86.81392881257838 %\n",
      "Progress: 86.93450371370696 %\n",
      "Progress: 87.05507861483554 %\n",
      "Progress: 87.17565351596411 %\n",
      "Progress: 87.2962284170927 %\n",
      "Progress: 87.41680331822128 %\n",
      "Progress: 87.53737821934986 %\n",
      "Progress: 87.65795312047844 %\n",
      "Progress: 87.77852802160703 %\n",
      "Progress: 87.89910292273561 %\n",
      "Progress: 88.01967782386419 %\n",
      "Progress: 88.14025272499276 %\n",
      "Progress: 88.26082762612135 %\n",
      "Progress: 88.38140252724993 %\n",
      "Progress: 88.50197742837851 %\n",
      "Progress: 88.62255232950709 %\n",
      "Progress: 88.74312723063566 %\n",
      "Progress: 88.86370213176426 %\n",
      "Progress: 88.98427703289283 %\n",
      "Progress: 89.10485193402141 %\n",
      "Progress: 89.22542683514999 %\n",
      "Progress: 89.34600173627858 %\n",
      "Progress: 89.46657663740716 %\n",
      "Progress: 89.58715153853574 %\n",
      "Progress: 89.70772643966431 %\n",
      "Progress: 89.8283013407929 %\n",
      "Progress: 89.94887624192148 %\n",
      "Progress: 90.06945114305006 %\n",
      "Progress: 90.19002604417864 %\n",
      "Progress: 90.31060094530723 %\n",
      "Progress: 90.4311758464358 %\n",
      "Progress: 90.55175074756438 %\n",
      "Progress: 90.67232564869296 %\n",
      "Progress: 90.79290054982155 %\n",
      "Progress: 90.91347545095013 %\n",
      "Progress: 91.03405035207871 %\n",
      "Progress: 91.15462525320729 %\n",
      "Progress: 91.27520015433588 %\n",
      "Progress: 91.39577505546445 %\n",
      "Progress: 91.51634995659303 %\n",
      "Progress: 91.63692485772161 %\n",
      "Progress: 91.7574997588502 %\n",
      "Progress: 91.87807465997878 %\n",
      "Progress: 91.99864956110736 %\n",
      "Progress: 92.11922446223593 %\n",
      "Progress: 92.23979936336453 %\n",
      "Progress: 92.3603742644931 %\n",
      "Progress: 92.48094916562168 %\n",
      "Progress: 92.60152406675026 %\n",
      "Progress: 92.72209896787884 %\n",
      "Progress: 92.84267386900743 %\n",
      "Progress: 92.963248770136 %\n",
      "Progress: 93.08382367126458 %\n",
      "Progress: 93.20439857239316 %\n",
      "Progress: 93.32497347352175 %\n",
      "Progress: 93.44554837465033 %\n",
      "Progress: 93.5661232757789 %\n",
      "Progress: 93.68669817690748 %\n",
      "Progress: 93.80727307803608 %\n",
      "Progress: 93.92784797916465 %\n",
      "Progress: 94.04842288029323 %\n",
      "Progress: 94.16899778142181 %\n",
      "Progress: 94.2895726825504 %\n",
      "Progress: 94.41014758367898 %\n",
      "Progress: 94.53072248480755 %\n",
      "Progress: 94.65129738593615 %\n",
      "Progress: 94.77187228706472 %\n",
      "Progress: 94.8924471881933 %\n",
      "Progress: 95.01302208932188 %\n",
      "Progress: 95.13359699045047 %\n",
      "Progress: 95.25417189157905 %\n",
      "Progress: 95.37474679270763 %\n",
      "Progress: 95.49532169383622 %\n",
      "Progress: 95.6158965949648 %\n",
      "Progress: 95.73647149609337 %\n",
      "Progress: 95.85704639722195 %\n",
      "Progress: 95.97762129835054 %\n",
      "Progress: 96.09819619947912 %\n",
      "Progress: 96.2187711006077 %\n",
      "Progress: 96.33934600173629 %\n",
      "Progress: 96.45992090286487 %\n",
      "Progress: 96.58049580399344 %\n",
      "Progress: 96.70107070512202 %\n",
      "Progress: 96.82164560625061 %\n",
      "Progress: 96.94222050737919 %\n",
      "Progress: 97.06279540850777 %\n",
      "Progress: 97.18337030963635 %\n",
      "Progress: 97.30394521076494 %\n",
      "Progress: 97.42452011189351 %\n",
      "Progress: 97.54509501302209 %\n",
      "Progress: 97.66566991415067 %\n",
      "Progress: 97.78624481527926 %\n",
      "Progress: 97.90681971640784 %\n",
      "Progress: 98.02739461753642 %\n",
      "Progress: 98.147969518665 %\n",
      "Progress: 98.26854441979359 %\n",
      "Progress: 98.38911932092216 %\n",
      "Progress: 98.50969422205074 %\n",
      "Progress: 98.63026912317932 %\n",
      "Progress: 98.75084402430791 %\n",
      "Progress: 98.87141892543649 %\n",
      "Progress: 98.99199382656506 %\n",
      "Progress: 99.11256872769364 %\n",
      "Progress: 99.23314362882223 %\n",
      "Progress: 99.35371852995081 %\n",
      "Progress: 99.47429343107939 %\n",
      "Progress: 99.59486833220797 %\n",
      "Progress: 99.71544323333656 %\n",
      "Progress: 99.83601813446514 %\n",
      "Progress: 99.95659303559371 %\n",
      "Progress: 100.0771679367223 %\n"
     ]
    }
   ],
   "execution_count": 17
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-12T17:21:38.402806Z",
     "start_time": "2025-03-12T17:21:37.118188Z"
    }
   },
   "cell_type": "code",
   "source": [
    "pmids_to_gse_mapping = {} # mapping of pubmed ids to accession codes of GEO datasets\n",
    "not_matched_GSEs = []\n",
    "no_pmid = 0\n",
    "pmids_found = []\n",
    "cnt=0\n",
    "\n",
    "for entry in GEO_series_data: # iterate dataset entries\n",
    "    if type(GEO_series_data[entry]['pubmedids']) == list and len(GEO_series_data[entry]['pubmedids']) == 0:\n",
    "        not_matched_GSEs.append(GEO_series_data[entry]['accession'])\n",
    "        \n",
    "    elif type(GEO_series_data[entry]['pubmedids']) == list:\n",
    "        pmids_found.extend(GEO_series_data[entry]['pubmedids'])\n",
    "        cnt += 1\n",
    "        for pmid in GEO_series_data[entry]['pubmedids']:\n",
    "            if pmid not in pmids_to_gse_mapping:\n",
    "                pmids_to_gse_mapping[pmid] = [GEO_series_data[entry]['accession']]\n",
    "            else:\n",
    "                pmids_to_gse_mapping[pmid].append(GEO_series_data[entry]['accession'])  # Append separately\n",
    "\n",
    "    else:\n",
    "        print(f\"Check why pmid not extracted at: {entry}!\")\n",
    "\n",
    "print(f\"Unique PMIDs found: {len(pmids_to_gse_mapping)}\")\n",
    "print(f\"# of Datasets with a pmid citation: {cnt}\")\n",
    "print(f\"# of Datasets without a pmid citation: {len(not_matched_GSEs)}\")"
   ],
   "id": "7324b3ebc9242e5e",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Unique PMIDs found: 127307\n",
      "# of Datasets with a pmid citation: 194205\n",
      "# of Datasets without a pmid citation: 54303\n"
     ]
    }
   ],
   "execution_count": 18
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-12T17:31:22.584254Z",
     "start_time": "2025-03-12T17:31:17.519833Z"
    }
   },
   "cell_type": "code",
   "source": [
    "pmid_doi_mapping_file = \"exp_output/pmid_doi_mapping.json\"\n",
    "if os.path.exists(\"exp_output/pmid_doi_mapping.json\"):\n",
    "    \n",
    "    with open(pmid_doi_mapping_file, \"r\") as f:\n",
    "        pmid_doi_mapping = json.load(f)  # read json file\n",
    "else:\n",
    "    pmid_doi_mapping = batch_PMID_to_doi(list(set(pmids_found)))\n",
    "    \n",
    "    os.makedirs(os.path.dirname(pmid_doi_mapping_file), exist_ok=True)\n",
    "    with open(pmid_doi_mapping_file, \"w\") as f:\n",
    "        json.dump(pmid_doi_mapping, f, indent=4) # Save to JSON\n",
    "        \n",
    "# reverse the mapping\n",
    "doi_to_pmid_mapping = {}\n",
    "\n",
    "for pmid, doi in pmid_doi_mapping.items():\n",
    "    if isinstance(doi, list):  # If DOI is a list (multiple DOIs per PMID)\n",
    "        for d in doi:\n",
    "            doi_to_pmid_mapping.setdefault(d, []).append(pmid)  # Store multiple PMIDs in a list\n",
    "    else:\n",
    "        doi_to_pmid_mapping.setdefault(doi, []).append(pmid)  # Always store as list\n",
    "\n",
    "# If needed, convert single-item lists to just the value\n",
    "doi_to_pmid_mapping = {k: v[0] if len(v) == 1 else v for k, v in doi_to_pmid_mapping.items()}"
   ],
   "id": "530d5002bb9bcf5d",
   "outputs": [],
   "execution_count": 20
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-12T17:31:28.014315Z",
     "start_time": "2025-03-12T17:31:27.845591Z"
    }
   },
   "cell_type": "code",
   "source": [
    "# count how many times dx.doi.org is in df_groud_truth['publication']\n",
    "with open(input_ground_truth_file, \"r\") as f:\n",
    "    input_ground_truth = json.load(f)\n",
    "\n",
    "count, pmcnt = 0,0\n",
    "\n",
    "for entry in input_ground_truth: # Iterate through the JSON data\n",
    "    publication = entry.get(\"citing_publication_link\", \"\")  # Safely get citing_publication_link field\n",
    "\n",
    "    if isinstance(publication, str):  # Ensure publication is a string\n",
    "        if \"dx.doi.org\" in publication:\n",
    "            count += 1\n",
    "        elif \"pubmed\" in publication:\n",
    "            pmcnt += 1\n",
    "\n",
    "print(f\"Total dx.doi.org found: {count}\")\n",
    "print(f\"Total pubmed found: {pmcnt}\")\n",
    "print(f\"Total publications: {len(input_ground_truth)}\")\n",
    "print(f\"Total matches: {count + pmcnt}\")"
   ],
   "id": "7567e273d1fd565e",
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Total dx.doi.org found: 14887\n",
      "Total pubmed found: 19992\n",
      "Total publications: 34879\n",
      "Total matches: 34879\n"
     ]
    }
   ],
   "execution_count": 21
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-12T17:31:37.429497Z",
     "start_time": "2025-03-12T17:31:36.275799Z"
    }
   },
   "cell_type": "code",
   "source": [
    "# for each entry in ground truth, we will create a column with GSE ids. We can use the publication_col to find the DOI or the PM id\n",
    "# then we can use the mapping {pmid -> doi} to find the PM id for doi\n",
    "# then we can use the mapping {pmid -> gse} to find the GSE id for the PM id\n",
    "import re\n",
    "\n",
    "# Convert `data` into a lookup dictionary using `accession` as the key\n",
    "gse_lookup = {v[\"accession\"]: v for v in GEO_series_data.values()}\n",
    "\n",
    "count, pmcnt = 0, 0\n",
    "cnt_tot_pm, cnt_tot_doi = 0, 0\n",
    "\n",
    "skip_PXD_dataset_doi = 0\n",
    "\n",
    "for entry in input_ground_truth:\n",
    "\n",
    "    publication = entry.get(\"citing_publication_link\", \"\")\n",
    "    \n",
    "    # skip dois of PXD datasets\n",
    "    if publication and \"//dx.doi.org/10.6019/PXD\" in publication:\n",
    "        skip_PXD_dataset_doi += 1\n",
    "        continue\n",
    "\n",
    "    if publication and \"dx.doi.org\" in publication:\n",
    "        match = re.search(r'dx.doi.org/([a-zA-Z0-9./-]+)', publication)\n",
    "        if match:\n",
    "            doi = match.group(1)\n",
    "            count += 1\n",
    "            pmid = doi_to_pmid_mapping.get(doi)\n",
    "            if pmid:\n",
    "                gse_ids = pmids_to_gse_mapping.get(pmid, [])  # Ensure it's a list\n",
    "\n",
    "                for gse_id in gse_ids:  # Iterate over multiple GSEs\n",
    "                    cnt_tot_doi += 1\n",
    "                    dataset_info = gse_lookup.get(gse_id, {})  # Get dataset info\n",
    "\n",
    "                    entry.setdefault(\"cited_datasets\", []).append({\n",
    "                        \"dataset_id\": gse_id,\n",
    "                        \"dataset_repo\": \"GEO\",\n",
    "                        \"dataset_summary\": dataset_info.get(\"summary\", \"N/A\"),\n",
    "                        \"dataset_title\": dataset_info.get(\"title\", \"N/A\")\n",
    "                    })\n",
    "\n",
    "    elif publication and \"pubmed\" in publication:\n",
    "        match = re.search(r'pubmed/([0-9]+)', publication)\n",
    "        if match:\n",
    "            pmid = match.group(1)\n",
    "            pmcnt += 1\n",
    "            gse_ids = pmids_to_gse_mapping.get(pmid, [])  # Ensure it's a list\n",
    "\n",
    "            for gse_id in gse_ids:  # Iterate over multiple GSEs\n",
    "                cnt_tot_pm += 1\n",
    "                dataset_info = gse_lookup.get(gse_id, {})\n",
    "\n",
    "                entry.setdefault(\"cited_datasets\", []).append({\n",
    "                    \"dataset_id\": gse_id,\n",
    "                    \"dataset_repo\": \"GEO\",\n",
    "                    \"dataset_summary\": dataset_info.get(\"summary\", \"N/A\"),\n",
    "                    \"dataset_title\": dataset_info.get(\"title\", \"N/A\")\n",
    "                })\n",
    "\n",
    "# Print results\n",
    "print(f\"Total dx.doi.org found: {count}\")\n",
    "print(f\"Total pubmed found: {pmcnt}\")\n",
    "print(f\"Total matches of gse accession codes: in DOI: {cnt_tot_doi}, in PMID: {cnt_tot_pm}\")\n",
    "print(f\"Skip PXD dataset doi: {skip_PXD_dataset_doi}\")"
   ],
   "id": "bb7f37dceed2342a",
   "execution_count": 22,
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Total dx.doi.org found: 12047\n",
      "Total pubmed found: 19987\n",
      "Total matches of gse accession codes: in DOI: 1498, in PMID: 3410\n",
      "Skip PXD dataset doi: 2840\n"
     ]
    }
   ]
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-12T17:31:53.839634Z",
     "start_time": "2025-03-12T17:31:52.768137Z"
    }
   },
   "cell_type": "code",
   "source": [
    "with open(output_ground_truth_file, 'w') as f:\n",
    "    json.dump(input_ground_truth, f, indent=4) # Save to JSON"
   ],
   "id": "4736f7ddd9a74dc2",
   "outputs": [],
   "execution_count": 23
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-03-12T17:31:55.888801Z",
     "start_time": "2025-03-12T17:31:55.883497Z"
    }
   },
   "cell_type": "code",
   "source": "len(input_ground_truth)",
   "id": "e4644aabcc8ff162",
   "outputs": [
    {
     "data": {
      "text/plain": [
       "34879"
      ]
     },
     "execution_count": 24,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "execution_count": 24
  },
  {
   "metadata": {},
   "cell_type": "code",
   "outputs": [],
   "execution_count": null,
   "source": "",
   "id": "94826b3e49c9f098"
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 2
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython2",
   "version": "2.7.6"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
