{
 "cells": [
  {
   "metadata": {},
   "cell_type": "markdown",
   "source": "Import orchestrator, and load environment variables from \".env\" file",
   "id": "b8f07b9099035d2c"
  },
  {
   "cell_type": "code",
   "id": "initial_id",
   "metadata": {
    "collapsed": true,
    "ExecuteTime": {
     "end_time": "2025-07-07T15:30:15.633889Z",
     "start_time": "2025-07-07T15:30:04.037770Z"
    }
   },
   "source": [
    "from data_gatherer.orchestrator import Orchestrator\n",
    "import time"
   ],
   "outputs": [
    {
     "data": {
      "text/plain": [
       "True"
      ]
     },
     "execution_count": 1,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "execution_count": 1
  },
  {
   "metadata": {},
   "cell_type": "markdown",
   "source": "Instantiate the orchestrator as data gatherer",
   "id": "a300c482cd96117"
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-07-07T15:30:15.645010Z",
     "start_time": "2025-07-07T15:30:15.635783Z"
    }
   },
   "cell_type": "code",
   "source": "data_gatherer = Orchestrator(llm_name=\"gemini-2.0-flash\", log_level=\"DEBUG\")",
   "id": "34f3046b7ea80e86",
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "orchestrator.py - line 85 - INFO - Data_Gatherer Orchestrator initialized. Extraction Model: gemini-2.0-flash\n"
     ]
    }
   ],
   "execution_count": 2
  },
  {
   "metadata": {},
   "cell_type": "markdown",
   "source": "## Step I: Fetch data",
   "id": "c009e195a3bc6fdb"
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-07-07T15:30:15.649156Z",
     "start_time": "2025-07-07T15:30:15.646258Z"
    }
   },
   "cell_type": "code",
   "source": "fetched_data = {}",
   "id": "74efe3d9619c6c3e",
   "outputs": [],
   "execution_count": 3
  },
  {
   "metadata": {},
   "cell_type": "markdown",
   "source": [
    "The Fetcher is responsible for fetching raw data about scientific publications on the internet. The parent class is DataFetcher, and the child classes are the following:\n",
    "\n",
    "\n",
    "\n",
    "- **WebScraper**, given a URL, it scrapes the page extracting the HTML content.\n",
    "\n",
    "\n",
    "- **DatabaseFetcher**, given a key, it fetches raw data (HTML or XML) from a local DataFrame.\n",
    "\n",
    "\n",
    "- **APIClient**, given a URI, it fetches raw data (XML) from an API."
   ],
   "id": "247052f9f7ee22fa"
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-07-07T15:30:15.658850Z",
     "start_time": "2025-07-07T15:30:15.651637Z"
    }
   },
   "cell_type": "code",
   "source": "data_gatherer.setup_data_fetcher()",
   "id": "99a91efa8baec42",
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "orchestrator.py - line 218 - INFO - Setting up data fetcher...\n",
      "data_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\n",
      "data_fetcher.py - line 26 - DEBUG - DataFetcher initialized.\n",
      "orchestrator.py - line 248 - INFO - Data fetcher setup completed.\n"
     ]
    }
   ],
   "execution_count": 4
  },
  {
   "metadata": {},
   "cell_type": "markdown",
   "source": [
    "### Fetch from API\n",
    "\n",
    "When the API is supported (for now only PubMedCentral), the fetcher will use the APIClient to fetch the data in XML format."
   ],
   "id": "c4364fb25bb6ac6b"
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-07-07T15:30:15.662636Z",
     "start_time": "2025-07-07T15:30:15.660033Z"
    }
   },
   "cell_type": "code",
   "source": "API_supported_input = \"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466\"",
   "id": "b50f0289f8a85d1d",
   "outputs": [],
   "execution_count": 5
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-07-07T15:30:15.878102Z",
     "start_time": "2025-07-07T15:30:15.664198Z"
    }
   },
   "cell_type": "code",
   "source": [
    "raw_data_API = data_gatherer.fetch_data(API_supported_input)\n",
    "fetched_data.update(raw_data_API)"
   ],
   "id": "7c532d5582f6d1a7",
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "orchestrator.py - line 218 - INFO - Setting up data fetcher...\n",
      "orchestrator.py - line 226 - WARNING - Failed to quit previous driver: 'NoneType' object has no attribute 'quit'\n",
      "orchestrator.py - line 248 - INFO - Data fetcher setup completed.\n",
      "orchestrator.py - line 135 - INFO - Fetching data from URL: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466\n",
      "data_fetcher.py - line 132 - DEBUG - update_DataFetcher_settings for current URL\n",
      "data_fetcher.py - line 185 - DEBUG - Checking PMC with pattern https://www.ncbi.nlm.nih.gov/pmc/articles/\n",
      "data_fetcher.py - line 188 - DEBUG - URL detected as PMC.\n",
      "data_fetcher.py - line 145 - INFO - Initializing EntrezFetcher(('requests', 'PMC_API', 'self.config'))\n",
      "data_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\n",
      "data_fetcher.py - line 26 - DEBUG - DataFetcher initialized.\n",
      "data_fetcher.py - line 482 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC6141466&retmode=xml\n",
      "data_fetcher.py - line 491 - DEBUG - Successfully fetched data for PMC6141466\n"
     ]
    }
   ],
   "execution_count": 6
  },
  {
   "metadata": {},
   "cell_type": "markdown",
   "source": "### Fetch from Local Data",
   "id": "5e3c60bbdd017ba3"
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-07-07T15:30:15.882285Z",
     "start_time": "2025-07-07T15:30:15.879500Z"
    }
   },
   "cell_type": "code",
   "source": "local_fetch_supported_input = \"https://www.ncbi.nlm.nih.gov/pmc/articles/pmc11425778\"",
   "id": "6193a95867778f36",
   "outputs": [],
   "execution_count": 7
  },
  {
   "metadata": {},
   "cell_type": "markdown",
   "source": "Show the structure of the local DataFrame",
   "id": "ad27aa312de453fd"
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-07-07T15:30:18.871334Z",
     "start_time": "2025-07-07T15:30:15.883590Z"
    }
   },
   "cell_type": "code",
   "source": [
    "raw_data_local = data_gatherer.fetch_data(local_fetch_supported_input, local_fetch_file=\"../scripts/exp_input/Local_fetched_data.parquet\")\n",
    "fetched_data.update(raw_data_local)"
   ],
   "id": "d30d1232bc5c0841",
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "orchestrator.py - line 218 - INFO - Setting up data fetcher...\n",
      "data_fetcher.py - line 26 - DEBUG - DataFetcher initialized.\n",
      "orchestrator.py - line 248 - INFO - Data fetcher setup completed.\n",
      "orchestrator.py - line 135 - INFO - Fetching data from URL: https://www.ncbi.nlm.nih.gov/pmc/articles/pmc11425778\n",
      "data_fetcher.py - line 132 - DEBUG - update_DataFetcher_settings for current URL\n",
      "data_fetcher.py - line 185 - DEBUG - Checking PMC with pattern https://www.ncbi.nlm.nih.gov/pmc/articles/\n",
      "data_fetcher.py - line 188 - DEBUG - URL detected as PMC.\n",
      "data_fetcher.py - line 140 - INFO - URL https://www.ncbi.nlm.nih.gov/pmc/articles/pmc11425778 found in DataFrame. Using DatabaseFetcher.\n",
      "data_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\n",
      "data_fetcher.py - line 26 - DEBUG - DataFetcher initialized.\n",
      "data_fetcher.py - line 421 - INFO - Fetching data for pmc11425778\n",
      "data_fetcher.py - line 422 - DEBUG - Data file: Index(['file_name', 'raw_cont', 'format', 'length', 'path', 'publication'], dtype='object')\n",
      "data_fetcher.py - line 423 - DEBUG - Data file:                                              file_name  \\\n",
      "9     Proteomics Analysis of Interactions between D...   \n",
      "262  Proteomics Analysis of Interactions between Dr...   \n",
      "910  Proteomics Analysis of Interactions between Dr...   \n",
      "\n",
      "                                              raw_cont format  length  \\\n",
      "9    <html lang=\"en\"><head>\\n\\n        <meta charse...   html  205661   \n",
      "262  <?xml version='1.0' encoding='UTF-8'?>\\n<!DOCT...    xml    6870   \n",
      "910  <html lang=\"en\" class=\"\"><head>\\n\\n        <me...   html  208395   \n",
      "\n",
      "                                                  path  publication  \n",
      "9    ../html_xml_samples/PMC/ Proteomics Analysis o...  pmc11425778  \n",
      "262  ../html_xml_samples/PMC/Proteomics Analysis of...  pmc11425778  \n",
      "910  ../html_xml_samples/PMC/Proteomics Analysis of...  pmc11425778  \n",
      "data_fetcher.py - line 424 - INFO - Fetching data from ../scripts/exp_input/Local_fetched_data.parquet\n"
     ]
    }
   ],
   "execution_count": 8
  },
  {
   "metadata": {},
   "cell_type": "markdown",
   "source": "### Fetch Raw HTML from a web page",
   "id": "3d995cafd36f74fe"
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-07-07T15:30:18.875386Z",
     "start_time": "2025-07-07T15:30:18.872548Z"
    }
   },
   "cell_type": "code",
   "source": "API_unsupported_input = \"https://www.nature.com/articles/s41467-024-51831-7\"",
   "id": "96928dc7cf83e5b1",
   "outputs": [],
   "execution_count": 9
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-07-07T15:30:30.415295Z",
     "start_time": "2025-07-07T15:30:18.879413Z"
    }
   },
   "cell_type": "code",
   "source": [
    "raw_html = data_gatherer.fetch_data(API_unsupported_input, browser='Firefox', headless=True)\n",
    "fetched_data.update(raw_html)"
   ],
   "id": "79bb5a9d7a80e20a",
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "orchestrator.py - line 218 - INFO - Setting up data fetcher...\n",
      "data_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\n",
      "data_fetcher.py - line 26 - DEBUG - DataFetcher initialized.\n",
      "orchestrator.py - line 248 - INFO - Data fetcher setup completed.\n",
      "orchestrator.py - line 135 - INFO - Fetching data from URL: https://www.nature.com/articles/s41467-024-51831-7\n",
      "data_fetcher.py - line 132 - DEBUG - update_DataFetcher_settings for current URL\n",
      "data_fetcher.py - line 185 - DEBUG - Checking PMC with pattern https://www.ncbi.nlm.nih.gov/pmc/articles/\n",
      "data_fetcher.py - line 185 - DEBUG - Checking PMC with pattern https://pmc.ncbi.nlm.nih.gov/\n",
      "data_fetcher.py - line 153 - INFO - WebScraper instance: True\n",
      "data_fetcher.py - line 154 - INFO - EntrezFetcher instance: False\n",
      "data_fetcher.py - line 155 - INFO - scraper_tool attribute: True\n",
      "data_fetcher.py - line 157 - INFO - Initializing new selenium driver.\n",
      "selenium_setup.py - line 11 - INFO - Creating WebDriver for browser: Firefox, with driver located at driver_path: None\n",
      "selenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\n",
      "selenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1a72c83d0>.\n",
      "data_fetcher.py - line 24 - WARNING - DataFetcher raw_HTML_data_filepath set no None\n",
      "data_fetcher.py - line 26 - DEBUG - DataFetcher initialized.\n",
      "data_fetcher.py - line 320 - WARNING - Publication name not found in the page title.\n"
     ]
    }
   ],
   "execution_count": 10
  },
  {
   "metadata": {},
   "cell_type": "markdown",
   "source": [
    "## Step II: Parse data\n",
    "\n",
    "The Parser is responsible for extracting the relevant information from the raw documents fetched by the Fetcher. It has two main discovery methods:\n",
    "\n",
    "\n",
    "- **Retrieve-Then-Read**, the parser passes to the LLMs only the relevant sections of the raw document, and the LLMs extract the relevant information.\n",
    "\n",
    "\n",
    "- **Full-Document-Read**, Large-context LLMs read the entire raw document and extract the relevant information."
   ],
   "id": "605ec31956889a57"
  },
  {
   "metadata": {},
   "cell_type": "markdown",
   "source": [
    "### Retrieve-Then-Read \n",
    "version 0.1.0 supports this parse method only for PMC XML and HTML documents"
   ],
   "id": "ece85d2274ea0039"
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-07-07T15:30:32.459346Z",
     "start_time": "2025-07-07T15:30:30.416858Z"
    }
   },
   "cell_type": "code",
   "source": [
    "results_RTR = data_gatherer.parse_data(API_supported_input, raw_data_API, publisher=\"PMC\", use_portkey_for_gemini=True, prompt_name=\"retrieve_datasets_simple_JSON\")\n",
    "results_RTR"
   ],
   "id": "b9d52f23ac6a3c73",
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "orchestrator.py - line 185 - INFO - Parsing data from URL: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466 with publisher: PMC\n",
      "base_parser.py - line 181 - INFO - LLMParser initialized.\n",
      "base_parser.py - line 913 - DEBUG - Repo mapping: {'bigd': 'bigd.big.ac.cn', 'european genome-phenome archive': 'ega-archive.org', 'massive': 'massive.ucsd.edu', 'pride': 'www.ebi.ac.uk', 'iprox': 'www.iprox.cn', 'proteomecentral': 'proteomecentral.proteomexchange.org', 'proteomexchange': 'www.proteomexchange.org', 'proteomic data commons': 'pdc.cancer.gov', 'genomic data commons': 'gdc.cancer.gov', 'the cancer imaging archive': 'cancerimagingarchive.net', 'imaging data commons': 'imaging.datacommons.cancer.gov', 'jpost': 'repository.jpostdb.org', 'doi.org': 'doi.org', 'geo': 'geo', 'dbgap': 'dbgap', 'sra': 'sra', 'arrayexpress': 'ArrayExpress'}\n",
      "xml_parser.py - line 23 - INFO - Initializing xmlRetriever\n",
      "xml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466, additional_data, XML)\n",
      "xml_parser.py - line 440 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\n",
      "base_parser.py - line 284 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\n",
      "base_parser.py - line 285 - INFO - Consider migrating this function to the BaseRetriever class.\n",
      "xml_parser.py - line 448 - DEBUG - Searching for supplementary material sections using XPath: .//supplementary-material\n",
      "xml_parser.py - line 451 - INFO - Found 15 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1a7a4d480>, <Element supplementary-material at 0x1aa724900>, <Element supplementary-material at 0x1aa724740>, <Element supplementary-material at 0x1aa7248c0>, <Element supplementary-material at 0x1aa724d80>, <Element supplementary-material at 0x1aa724980>, <Element supplementary-material at 0x1aa7249c0>, <Element supplementary-material at 0x1aa7244c0>, <Element supplementary-material at 0x1aa724400>, <Element supplementary-material at 0x1aa724480>, <Element supplementary-material at 0x13c0f7d80>, <Element supplementary-material at 0x1aa7243c0>, <Element supplementary-material at 0x1aa724380>, <Element supplementary-material at 0x1aa724340>, <Element supplementary-material at 0x1aa724300>]\n",
      "xml_parser.py - line 454 - DEBUG - Found 1 supplementary-material sections.\n",
      "xml_parser.py - line 459 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1a7a4d480>, <Element supplementary-material at 0x1aa724900>, <Element supplementary-material at 0x1aa724740>, <Element supplementary-material at 0x1aa7248c0>, <Element supplementary-material at 0x1aa724d80>, <Element supplementary-material at 0x1aa724980>, <Element supplementary-material at 0x1aa7249c0>, <Element supplementary-material at 0x1aa7244c0>, <Element supplementary-material at 0x1aa724400>, <Element supplementary-material at 0x1aa724480>, <Element supplementary-material at 0x13c0f7d80>, <Element supplementary-material at 0x1aa7243c0>, <Element supplementary-material at 0x1aa724380>, <Element supplementary-material at 0x1aa724340>, <Element supplementary-material at 0x1aa724300>]}\n",
      "base_parser.py - line 302 - DEBUG - Function_call: reconstruct_download_link(41467_2018_6162_MOESM1_ESM.pdf, local-data, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466)\n",
      "base_parser.py - line 305 - DEBUG - Inputs to reconstruct_download_link: 41467_2018_6162_MOESM1_ESM.pdf, local-data, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466, 6141466\n",
      "xml_parser.py - line 497 - DEBUG - Extracted media item with href: 41467_2018_6162_MOESM1_ESM.pdf\n",
      "xml_parser.py - line 498 - DEBUG - Source url: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466\n",
      "xml_parser.py - line 499 - DEBUG - Supplementary material title: No Title\n",
      "xml_parser.py - line 500 - DEBUG - Content type: local-data, ID: MOESM1\n",
      "xml_parser.py - line 501 - DEBUG - Surrounding text for media: Supplementary  Information\n",
      "xml_parser.py - line 502 - DEBUG - Description: Supplementary Information\n",
      "xml_parser.py - line 503 - DEBUG - Download_link: https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM1_ESM.pdf\n",
      "xml_parser.py - line 518 - DEBUG - Extracted item: {\n",
      "    \"link\": \"41467_2018_6162_MOESM1_ESM.pdf\",\n",
      "    \"source_url\": \"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466\",\n",
      "    \"download_link\": \"https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM1_ESM.pdf\",\n",
      "    \"title\": \"No Title\",\n",
      "    \"content_type\": \"local-data\",\n",
      "    \"id\": \"MOESM1\",\n",
      "    \"surrounding_text\": \"Supplementary  Information\",\n",
      "    \"description\": \"Supplementary Information\",\n",
      "    \"source_section\": \"supplementary material\",\n",
      "    \"retrieval_pattern\": \".//supplementary-material\"\n",
      "}\n",
      "xml_parser.py - line 522 - DEBUG - Found 0 inline-supplementary-material elements.\n",
      "xml_parser.py - line 538 - DEBUG - Extracted supplementary material links:\n",
      "[{'link': '41467_2018_6162_MOESM1_ESM.pdf', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM1_ESM.pdf', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM1', 'surrounding_text': 'Supplementary  Information', 'description': 'Supplementary Information', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}]\n",
      "base_parser.py - line 302 - DEBUG - Function_call: reconstruct_download_link(41467_2018_6162_MOESM2_ESM.pdf, local-data, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466)\n",
      "base_parser.py - line 305 - DEBUG - Inputs to reconstruct_download_link: 41467_2018_6162_MOESM2_ESM.pdf, local-data, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466, 6141466\n",
      "xml_parser.py - line 497 - DEBUG - Extracted media item with href: 41467_2018_6162_MOESM2_ESM.pdf\n",
      "xml_parser.py - line 498 - DEBUG - Source url: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466\n",
      "xml_parser.py - line 499 - DEBUG - Supplementary material title: No Title\n",
      "xml_parser.py - line 500 - DEBUG - Content type: local-data, ID: MOESM2\n",
      "xml_parser.py - line 501 - DEBUG - Surrounding text for media: Peer  Review  File\n",
      "xml_parser.py - line 502 - DEBUG - Description: Peer Review File\n",
      "xml_parser.py - line 503 - DEBUG - Download_link: https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM2_ESM.pdf\n",
      "xml_parser.py - line 518 - DEBUG - Extracted item: {\n",
      "    \"link\": \"41467_2018_6162_MOESM2_ESM.pdf\",\n",
      "    \"source_url\": \"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466\",\n",
      "    \"download_link\": \"https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM2_ESM.pdf\",\n",
      "    \"title\": \"No Title\",\n",
      "    \"content_type\": \"local-data\",\n",
      "    \"id\": \"MOESM2\",\n",
      "    \"surrounding_text\": \"Peer  Review  File\",\n",
      "    \"description\": \"Peer Review File\",\n",
      "    \"source_section\": \"supplementary material\",\n",
      "    \"retrieval_pattern\": \".//supplementary-material\"\n",
      "}\n",
      "xml_parser.py - line 522 - DEBUG - Found 0 inline-supplementary-material elements.\n",
      "xml_parser.py - line 538 - DEBUG - Extracted supplementary material links:\n",
      "[{'link': '41467_2018_6162_MOESM1_ESM.pdf', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM1_ESM.pdf', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM1', 'surrounding_text': 'Supplementary  Information', 'description': 'Supplementary Information', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM2_ESM.pdf', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM2_ESM.pdf', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM2', 'surrounding_text': 'Peer  Review  File', 'description': 'Peer Review File', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}]\n",
      "base_parser.py - line 302 - DEBUG - Function_call: reconstruct_download_link(41467_2018_6162_MOESM3_ESM.pdf, local-data, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466)\n",
      "base_parser.py - line 305 - DEBUG - Inputs to reconstruct_download_link: 41467_2018_6162_MOESM3_ESM.pdf, local-data, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466, 6141466\n",
      "xml_parser.py - line 497 - DEBUG - Extracted media item with href: 41467_2018_6162_MOESM3_ESM.pdf\n",
      "xml_parser.py - line 498 - DEBUG - Source url: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466\n",
      "xml_parser.py - line 499 - DEBUG - Supplementary material title: No Title\n",
      "xml_parser.py - line 500 - DEBUG - Content type: local-data, ID: MOESM3\n",
      "xml_parser.py - line 501 - DEBUG - Surrounding text for media: Description  of  Additional  Supplementary  Files\n",
      "xml_parser.py - line 502 - DEBUG - Description: Description of Additional Supplementary Files\n",
      "xml_parser.py - line 503 - DEBUG - Download_link: https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM3_ESM.pdf\n",
      "xml_parser.py - line 518 - DEBUG - Extracted item: {\n",
      "    \"link\": \"41467_2018_6162_MOESM3_ESM.pdf\",\n",
      "    \"source_url\": \"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466\",\n",
      "    \"download_link\": \"https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM3_ESM.pdf\",\n",
      "    \"title\": \"No Title\",\n",
      "    \"content_type\": \"local-data\",\n",
      "    \"id\": \"MOESM3\",\n",
      "    \"surrounding_text\": \"Description  of  Additional  Supplementary  Files\",\n",
      "    \"description\": \"Description of Additional Supplementary Files\",\n",
      "    \"source_section\": \"supplementary material\",\n",
      "    \"retrieval_pattern\": \".//supplementary-material\"\n",
      "}\n",
      "xml_parser.py - line 522 - DEBUG - Found 0 inline-supplementary-material elements.\n",
      "xml_parser.py - line 538 - DEBUG - Extracted supplementary material links:\n",
      "[{'link': '41467_2018_6162_MOESM1_ESM.pdf', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM1_ESM.pdf', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM1', 'surrounding_text': 'Supplementary  Information', 'description': 'Supplementary Information', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM2_ESM.pdf', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM2_ESM.pdf', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM2', 'surrounding_text': 'Peer  Review  File', 'description': 'Peer Review File', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM3_ESM.pdf', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM3_ESM.pdf', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM3', 'surrounding_text': 'Description  of  Additional  Supplementary  Files', 'description': 'Description of Additional Supplementary Files', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}]\n",
      "base_parser.py - line 302 - DEBUG - Function_call: reconstruct_download_link(41467_2018_6162_MOESM4_ESM.xlsx, local-data, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466)\n",
      "base_parser.py - line 305 - DEBUG - Inputs to reconstruct_download_link: 41467_2018_6162_MOESM4_ESM.xlsx, local-data, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466, 6141466\n",
      "xml_parser.py - line 497 - DEBUG - Extracted media item with href: 41467_2018_6162_MOESM4_ESM.xlsx\n",
      "xml_parser.py - line 498 - DEBUG - Source url: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466\n",
      "xml_parser.py - line 499 - DEBUG - Supplementary material title: No Title\n",
      "xml_parser.py - line 500 - DEBUG - Content type: local-data, ID: MOESM4\n",
      "xml_parser.py - line 501 - DEBUG - Surrounding text for media: Supplementary  Data  1\n",
      "xml_parser.py - line 502 - DEBUG - Description: Supplementary Data 1\n",
      "xml_parser.py - line 503 - DEBUG - Download_link: https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM4_ESM.xlsx\n",
      "xml_parser.py - line 518 - DEBUG - Extracted item: {\n",
      "    \"link\": \"41467_2018_6162_MOESM4_ESM.xlsx\",\n",
      "    \"source_url\": \"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466\",\n",
      "    \"download_link\": \"https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM4_ESM.xlsx\",\n",
      "    \"title\": \"No Title\",\n",
      "    \"content_type\": \"local-data\",\n",
      "    \"id\": \"MOESM4\",\n",
      "    \"surrounding_text\": \"Supplementary  Data  1\",\n",
      "    \"description\": \"Supplementary Data 1\",\n",
      "    \"source_section\": \"supplementary material\",\n",
      "    \"retrieval_pattern\": \".//supplementary-material\"\n",
      "}\n",
      "xml_parser.py - line 522 - DEBUG - Found 0 inline-supplementary-material elements.\n",
      "xml_parser.py - line 538 - DEBUG - Extracted supplementary material links:\n",
      "[{'link': '41467_2018_6162_MOESM1_ESM.pdf', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM1_ESM.pdf', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM1', 'surrounding_text': 'Supplementary  Information', 'description': 'Supplementary Information', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM2_ESM.pdf', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM2_ESM.pdf', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM2', 'surrounding_text': 'Peer  Review  File', 'description': 'Peer Review File', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM3_ESM.pdf', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM3_ESM.pdf', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM3', 'surrounding_text': 'Description  of  Additional  Supplementary  Files', 'description': 'Description of Additional Supplementary Files', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM4_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM4_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM4', 'surrounding_text': 'Supplementary  Data  1', 'description': 'Supplementary Data 1', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}]\n",
      "base_parser.py - line 302 - DEBUG - Function_call: reconstruct_download_link(41467_2018_6162_MOESM5_ESM.xlsx, local-data, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466)\n",
      "base_parser.py - line 305 - DEBUG - Inputs to reconstruct_download_link: 41467_2018_6162_MOESM5_ESM.xlsx, local-data, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466, 6141466\n",
      "xml_parser.py - line 497 - DEBUG - Extracted media item with href: 41467_2018_6162_MOESM5_ESM.xlsx\n",
      "xml_parser.py - line 498 - DEBUG - Source url: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466\n",
      "xml_parser.py - line 499 - DEBUG - Supplementary material title: No Title\n",
      "xml_parser.py - line 500 - DEBUG - Content type: local-data, ID: MOESM5\n",
      "xml_parser.py - line 501 - DEBUG - Surrounding text for media: Supplementary  Data  2\n",
      "xml_parser.py - line 502 - DEBUG - Description: Supplementary Data 2\n",
      "xml_parser.py - line 503 - DEBUG - Download_link: https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM5_ESM.xlsx\n",
      "xml_parser.py - line 518 - DEBUG - Extracted item: {\n",
      "    \"link\": \"41467_2018_6162_MOESM5_ESM.xlsx\",\n",
      "    \"source_url\": \"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466\",\n",
      "    \"download_link\": \"https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM5_ESM.xlsx\",\n",
      "    \"title\": \"No Title\",\n",
      "    \"content_type\": \"local-data\",\n",
      "    \"id\": \"MOESM5\",\n",
      "    \"surrounding_text\": \"Supplementary  Data  2\",\n",
      "    \"description\": \"Supplementary Data 2\",\n",
      "    \"source_section\": \"supplementary material\",\n",
      "    \"retrieval_pattern\": \".//supplementary-material\"\n",
      "}\n",
      "xml_parser.py - line 522 - DEBUG - Found 0 inline-supplementary-material elements.\n",
      "xml_parser.py - line 538 - DEBUG - Extracted supplementary material links:\n",
      "[{'link': '41467_2018_6162_MOESM1_ESM.pdf', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM1_ESM.pdf', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM1', 'surrounding_text': 'Supplementary  Information', 'description': 'Supplementary Information', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM2_ESM.pdf', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM2_ESM.pdf', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM2', 'surrounding_text': 'Peer  Review  File', 'description': 'Peer Review File', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM3_ESM.pdf', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM3_ESM.pdf', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM3', 'surrounding_text': 'Description  of  Additional  Supplementary  Files', 'description': 'Description of Additional Supplementary Files', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM4_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM4_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM4', 'surrounding_text': 'Supplementary  Data  1', 'description': 'Supplementary Data 1', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM5_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM5_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM5', 'surrounding_text': 'Supplementary  Data  2', 'description': 'Supplementary Data 2', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}]\n",
      "base_parser.py - line 302 - DEBUG - Function_call: reconstruct_download_link(41467_2018_6162_MOESM6_ESM.xlsx, local-data, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466)\n",
      "base_parser.py - line 305 - DEBUG - Inputs to reconstruct_download_link: 41467_2018_6162_MOESM6_ESM.xlsx, local-data, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466, 6141466\n",
      "xml_parser.py - line 497 - DEBUG - Extracted media item with href: 41467_2018_6162_MOESM6_ESM.xlsx\n",
      "xml_parser.py - line 498 - DEBUG - Source url: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466\n",
      "xml_parser.py - line 499 - DEBUG - Supplementary material title: No Title\n",
      "xml_parser.py - line 500 - DEBUG - Content type: local-data, ID: MOESM6\n",
      "xml_parser.py - line 501 - DEBUG - Surrounding text for media: Supplementary  Data  3\n",
      "xml_parser.py - line 502 - DEBUG - Description: Supplementary Data 3\n",
      "xml_parser.py - line 503 - DEBUG - Download_link: https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM6_ESM.xlsx\n",
      "xml_parser.py - line 518 - DEBUG - Extracted item: {\n",
      "    \"link\": \"41467_2018_6162_MOESM6_ESM.xlsx\",\n",
      "    \"source_url\": \"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466\",\n",
      "    \"download_link\": \"https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM6_ESM.xlsx\",\n",
      "    \"title\": \"No Title\",\n",
      "    \"content_type\": \"local-data\",\n",
      "    \"id\": \"MOESM6\",\n",
      "    \"surrounding_text\": \"Supplementary  Data  3\",\n",
      "    \"description\": \"Supplementary Data 3\",\n",
      "    \"source_section\": \"supplementary material\",\n",
      "    \"retrieval_pattern\": \".//supplementary-material\"\n",
      "}\n",
      "xml_parser.py - line 522 - DEBUG - Found 0 inline-supplementary-material elements.\n",
      "xml_parser.py - line 538 - DEBUG - Extracted supplementary material links:\n",
      "[{'link': '41467_2018_6162_MOESM1_ESM.pdf', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM1_ESM.pdf', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM1', 'surrounding_text': 'Supplementary  Information', 'description': 'Supplementary Information', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM2_ESM.pdf', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM2_ESM.pdf', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM2', 'surrounding_text': 'Peer  Review  File', 'description': 'Peer Review File', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM3_ESM.pdf', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM3_ESM.pdf', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM3', 'surrounding_text': 'Description  of  Additional  Supplementary  Files', 'description': 'Description of Additional Supplementary Files', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM4_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM4_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM4', 'surrounding_text': 'Supplementary  Data  1', 'description': 'Supplementary Data 1', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM5_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM5_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM5', 'surrounding_text': 'Supplementary  Data  2', 'description': 'Supplementary Data 2', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM6_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM6_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM6', 'surrounding_text': 'Supplementary  Data  3', 'description': 'Supplementary Data 3', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}]\n",
      "base_parser.py - line 302 - DEBUG - Function_call: reconstruct_download_link(41467_2018_6162_MOESM7_ESM.xlsx, local-data, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466)\n",
      "base_parser.py - line 305 - DEBUG - Inputs to reconstruct_download_link: 41467_2018_6162_MOESM7_ESM.xlsx, local-data, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466, 6141466\n",
      "xml_parser.py - line 497 - DEBUG - Extracted media item with href: 41467_2018_6162_MOESM7_ESM.xlsx\n",
      "xml_parser.py - line 498 - DEBUG - Source url: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466\n",
      "xml_parser.py - line 499 - DEBUG - Supplementary material title: No Title\n",
      "xml_parser.py - line 500 - DEBUG - Content type: local-data, ID: MOESM7\n",
      "xml_parser.py - line 501 - DEBUG - Surrounding text for media: Supplementary  Data  4\n",
      "xml_parser.py - line 502 - DEBUG - Description: Supplementary Data 4\n",
      "xml_parser.py - line 503 - DEBUG - Download_link: https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM7_ESM.xlsx\n",
      "xml_parser.py - line 518 - DEBUG - Extracted item: {\n",
      "    \"link\": \"41467_2018_6162_MOESM7_ESM.xlsx\",\n",
      "    \"source_url\": \"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466\",\n",
      "    \"download_link\": \"https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM7_ESM.xlsx\",\n",
      "    \"title\": \"No Title\",\n",
      "    \"content_type\": \"local-data\",\n",
      "    \"id\": \"MOESM7\",\n",
      "    \"surrounding_text\": \"Supplementary  Data  4\",\n",
      "    \"description\": \"Supplementary Data 4\",\n",
      "    \"source_section\": \"supplementary material\",\n",
      "    \"retrieval_pattern\": \".//supplementary-material\"\n",
      "}\n",
      "xml_parser.py - line 522 - DEBUG - Found 0 inline-supplementary-material elements.\n",
      "xml_parser.py - line 538 - DEBUG - Extracted supplementary material links:\n",
      "[{'link': '41467_2018_6162_MOESM1_ESM.pdf', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM1_ESM.pdf', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM1', 'surrounding_text': 'Supplementary  Information', 'description': 'Supplementary Information', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM2_ESM.pdf', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM2_ESM.pdf', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM2', 'surrounding_text': 'Peer  Review  File', 'description': 'Peer Review File', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM3_ESM.pdf', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM3_ESM.pdf', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM3', 'surrounding_text': 'Description  of  Additional  Supplementary  Files', 'description': 'Description of Additional Supplementary Files', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM4_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM4_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM4', 'surrounding_text': 'Supplementary  Data  1', 'description': 'Supplementary Data 1', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM5_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM5_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM5', 'surrounding_text': 'Supplementary  Data  2', 'description': 'Supplementary Data 2', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM6_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM6_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM6', 'surrounding_text': 'Supplementary  Data  3', 'description': 'Supplementary Data 3', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM7_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM7_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM7', 'surrounding_text': 'Supplementary  Data  4', 'description': 'Supplementary Data 4', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}]\n",
      "base_parser.py - line 302 - DEBUG - Function_call: reconstruct_download_link(41467_2018_6162_MOESM8_ESM.xlsx, local-data, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466)\n",
      "base_parser.py - line 305 - DEBUG - Inputs to reconstruct_download_link: 41467_2018_6162_MOESM8_ESM.xlsx, local-data, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466, 6141466\n",
      "xml_parser.py - line 497 - DEBUG - Extracted media item with href: 41467_2018_6162_MOESM8_ESM.xlsx\n",
      "xml_parser.py - line 498 - DEBUG - Source url: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466\n",
      "xml_parser.py - line 499 - DEBUG - Supplementary material title: No Title\n",
      "xml_parser.py - line 500 - DEBUG - Content type: local-data, ID: MOESM8\n",
      "xml_parser.py - line 501 - DEBUG - Surrounding text for media: Supplementary  Data  5\n",
      "xml_parser.py - line 502 - DEBUG - Description: Supplementary Data 5\n",
      "xml_parser.py - line 503 - DEBUG - Download_link: https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM8_ESM.xlsx\n",
      "xml_parser.py - line 518 - DEBUG - Extracted item: {\n",
      "    \"link\": \"41467_2018_6162_MOESM8_ESM.xlsx\",\n",
      "    \"source_url\": \"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466\",\n",
      "    \"download_link\": \"https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM8_ESM.xlsx\",\n",
      "    \"title\": \"No Title\",\n",
      "    \"content_type\": \"local-data\",\n",
      "    \"id\": \"MOESM8\",\n",
      "    \"surrounding_text\": \"Supplementary  Data  5\",\n",
      "    \"description\": \"Supplementary Data 5\",\n",
      "    \"source_section\": \"supplementary material\",\n",
      "    \"retrieval_pattern\": \".//supplementary-material\"\n",
      "}\n",
      "xml_parser.py - line 522 - DEBUG - Found 0 inline-supplementary-material elements.\n",
      "xml_parser.py - line 538 - DEBUG - Extracted supplementary material links:\n",
      "[{'link': '41467_2018_6162_MOESM1_ESM.pdf', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM1_ESM.pdf', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM1', 'surrounding_text': 'Supplementary  Information', 'description': 'Supplementary Information', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM2_ESM.pdf', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM2_ESM.pdf', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM2', 'surrounding_text': 'Peer  Review  File', 'description': 'Peer Review File', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM3_ESM.pdf', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM3_ESM.pdf', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM3', 'surrounding_text': 'Description  of  Additional  Supplementary  Files', 'description': 'Description of Additional Supplementary Files', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM4_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM4_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM4', 'surrounding_text': 'Supplementary  Data  1', 'description': 'Supplementary Data 1', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM5_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM5_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM5', 'surrounding_text': 'Supplementary  Data  2', 'description': 'Supplementary Data 2', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM6_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM6_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM6', 'surrounding_text': 'Supplementary  Data  3', 'description': 'Supplementary Data 3', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM7_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM7_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM7', 'surrounding_text': 'Supplementary  Data  4', 'description': 'Supplementary Data 4', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM8_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM8_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM8', 'surrounding_text': 'Supplementary  Data  5', 'description': 'Supplementary Data 5', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}]\n",
      "base_parser.py - line 302 - DEBUG - Function_call: reconstruct_download_link(41467_2018_6162_MOESM9_ESM.xlsx, local-data, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466)\n",
      "base_parser.py - line 305 - DEBUG - Inputs to reconstruct_download_link: 41467_2018_6162_MOESM9_ESM.xlsx, local-data, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466, 6141466\n",
      "xml_parser.py - line 497 - DEBUG - Extracted media item with href: 41467_2018_6162_MOESM9_ESM.xlsx\n",
      "xml_parser.py - line 498 - DEBUG - Source url: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466\n",
      "xml_parser.py - line 499 - DEBUG - Supplementary material title: No Title\n",
      "xml_parser.py - line 500 - DEBUG - Content type: local-data, ID: MOESM9\n",
      "xml_parser.py - line 501 - DEBUG - Surrounding text for media: Supplementary  Data  6\n",
      "xml_parser.py - line 502 - DEBUG - Description: Supplementary Data 6\n",
      "xml_parser.py - line 503 - DEBUG - Download_link: https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM9_ESM.xlsx\n",
      "xml_parser.py - line 518 - DEBUG - Extracted item: {\n",
      "    \"link\": \"41467_2018_6162_MOESM9_ESM.xlsx\",\n",
      "    \"source_url\": \"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466\",\n",
      "    \"download_link\": \"https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM9_ESM.xlsx\",\n",
      "    \"title\": \"No Title\",\n",
      "    \"content_type\": \"local-data\",\n",
      "    \"id\": \"MOESM9\",\n",
      "    \"surrounding_text\": \"Supplementary  Data  6\",\n",
      "    \"description\": \"Supplementary Data 6\",\n",
      "    \"source_section\": \"supplementary material\",\n",
      "    \"retrieval_pattern\": \".//supplementary-material\"\n",
      "}\n",
      "xml_parser.py - line 522 - DEBUG - Found 0 inline-supplementary-material elements.\n",
      "xml_parser.py - line 538 - DEBUG - Extracted supplementary material links:\n",
      "[{'link': '41467_2018_6162_MOESM1_ESM.pdf', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM1_ESM.pdf', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM1', 'surrounding_text': 'Supplementary  Information', 'description': 'Supplementary Information', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM2_ESM.pdf', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM2_ESM.pdf', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM2', 'surrounding_text': 'Peer  Review  File', 'description': 'Peer Review File', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM3_ESM.pdf', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM3_ESM.pdf', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM3', 'surrounding_text': 'Description  of  Additional  Supplementary  Files', 'description': 'Description of Additional Supplementary Files', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM4_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM4_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM4', 'surrounding_text': 'Supplementary  Data  1', 'description': 'Supplementary Data 1', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM5_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM5_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM5', 'surrounding_text': 'Supplementary  Data  2', 'description': 'Supplementary Data 2', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM6_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM6_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM6', 'surrounding_text': 'Supplementary  Data  3', 'description': 'Supplementary Data 3', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM7_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM7_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM7', 'surrounding_text': 'Supplementary  Data  4', 'description': 'Supplementary Data 4', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM8_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM8_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM8', 'surrounding_text': 'Supplementary  Data  5', 'description': 'Supplementary Data 5', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM9_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM9_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM9', 'surrounding_text': 'Supplementary  Data  6', 'description': 'Supplementary Data 6', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}]\n",
      "base_parser.py - line 302 - DEBUG - Function_call: reconstruct_download_link(41467_2018_6162_MOESM10_ESM.xlsx, local-data, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466)\n",
      "base_parser.py - line 305 - DEBUG - Inputs to reconstruct_download_link: 41467_2018_6162_MOESM10_ESM.xlsx, local-data, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466, 6141466\n",
      "xml_parser.py - line 497 - DEBUG - Extracted media item with href: 41467_2018_6162_MOESM10_ESM.xlsx\n",
      "xml_parser.py - line 498 - DEBUG - Source url: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466\n",
      "xml_parser.py - line 499 - DEBUG - Supplementary material title: No Title\n",
      "xml_parser.py - line 500 - DEBUG - Content type: local-data, ID: MOESM10\n",
      "xml_parser.py - line 501 - DEBUG - Surrounding text for media: Supplementary  Data  7\n",
      "xml_parser.py - line 502 - DEBUG - Description: Supplementary Data 7\n",
      "xml_parser.py - line 503 - DEBUG - Download_link: https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM10_ESM.xlsx\n",
      "xml_parser.py - line 518 - DEBUG - Extracted item: {\n",
      "    \"link\": \"41467_2018_6162_MOESM10_ESM.xlsx\",\n",
      "    \"source_url\": \"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466\",\n",
      "    \"download_link\": \"https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM10_ESM.xlsx\",\n",
      "    \"title\": \"No Title\",\n",
      "    \"content_type\": \"local-data\",\n",
      "    \"id\": \"MOESM10\",\n",
      "    \"surrounding_text\": \"Supplementary  Data  7\",\n",
      "    \"description\": \"Supplementary Data 7\",\n",
      "    \"source_section\": \"supplementary material\",\n",
      "    \"retrieval_pattern\": \".//supplementary-material\"\n",
      "}\n",
      "xml_parser.py - line 522 - DEBUG - Found 0 inline-supplementary-material elements.\n",
      "xml_parser.py - line 538 - DEBUG - Extracted supplementary material links:\n",
      "[{'link': '41467_2018_6162_MOESM1_ESM.pdf', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM1_ESM.pdf', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM1', 'surrounding_text': 'Supplementary  Information', 'description': 'Supplementary Information', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM2_ESM.pdf', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM2_ESM.pdf', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM2', 'surrounding_text': 'Peer  Review  File', 'description': 'Peer Review File', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM3_ESM.pdf', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM3_ESM.pdf', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM3', 'surrounding_text': 'Description  of  Additional  Supplementary  Files', 'description': 'Description of Additional Supplementary Files', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM4_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM4_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM4', 'surrounding_text': 'Supplementary  Data  1', 'description': 'Supplementary Data 1', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM5_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM5_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM5', 'surrounding_text': 'Supplementary  Data  2', 'description': 'Supplementary Data 2', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM6_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM6_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM6', 'surrounding_text': 'Supplementary  Data  3', 'description': 'Supplementary Data 3', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM7_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM7_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM7', 'surrounding_text': 'Supplementary  Data  4', 'description': 'Supplementary Data 4', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM8_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM8_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM8', 'surrounding_text': 'Supplementary  Data  5', 'description': 'Supplementary Data 5', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM9_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM9_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM9', 'surrounding_text': 'Supplementary  Data  6', 'description': 'Supplementary Data 6', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM10_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM10_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM10', 'surrounding_text': 'Supplementary  Data  7', 'description': 'Supplementary Data 7', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}]\n",
      "base_parser.py - line 302 - DEBUG - Function_call: reconstruct_download_link(41467_2018_6162_MOESM11_ESM.xlsx, local-data, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466)\n",
      "base_parser.py - line 305 - DEBUG - Inputs to reconstruct_download_link: 41467_2018_6162_MOESM11_ESM.xlsx, local-data, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466, 6141466\n",
      "xml_parser.py - line 497 - DEBUG - Extracted media item with href: 41467_2018_6162_MOESM11_ESM.xlsx\n",
      "xml_parser.py - line 498 - DEBUG - Source url: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466\n",
      "xml_parser.py - line 499 - DEBUG - Supplementary material title: No Title\n",
      "xml_parser.py - line 500 - DEBUG - Content type: local-data, ID: MOESM11\n",
      "xml_parser.py - line 501 - DEBUG - Surrounding text for media: Supplementary  Data  8\n",
      "xml_parser.py - line 502 - DEBUG - Description: Supplementary Data 8\n",
      "xml_parser.py - line 503 - DEBUG - Download_link: https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM11_ESM.xlsx\n",
      "xml_parser.py - line 518 - DEBUG - Extracted item: {\n",
      "    \"link\": \"41467_2018_6162_MOESM11_ESM.xlsx\",\n",
      "    \"source_url\": \"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466\",\n",
      "    \"download_link\": \"https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM11_ESM.xlsx\",\n",
      "    \"title\": \"No Title\",\n",
      "    \"content_type\": \"local-data\",\n",
      "    \"id\": \"MOESM11\",\n",
      "    \"surrounding_text\": \"Supplementary  Data  8\",\n",
      "    \"description\": \"Supplementary Data 8\",\n",
      "    \"source_section\": \"supplementary material\",\n",
      "    \"retrieval_pattern\": \".//supplementary-material\"\n",
      "}\n",
      "xml_parser.py - line 522 - DEBUG - Found 0 inline-supplementary-material elements.\n",
      "xml_parser.py - line 538 - DEBUG - Extracted supplementary material links:\n",
      "[{'link': '41467_2018_6162_MOESM1_ESM.pdf', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM1_ESM.pdf', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM1', 'surrounding_text': 'Supplementary  Information', 'description': 'Supplementary Information', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM2_ESM.pdf', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM2_ESM.pdf', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM2', 'surrounding_text': 'Peer  Review  File', 'description': 'Peer Review File', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM3_ESM.pdf', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM3_ESM.pdf', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM3', 'surrounding_text': 'Description  of  Additional  Supplementary  Files', 'description': 'Description of Additional Supplementary Files', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM4_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM4_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM4', 'surrounding_text': 'Supplementary  Data  1', 'description': 'Supplementary Data 1', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM5_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM5_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM5', 'surrounding_text': 'Supplementary  Data  2', 'description': 'Supplementary Data 2', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM6_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM6_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM6', 'surrounding_text': 'Supplementary  Data  3', 'description': 'Supplementary Data 3', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM7_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM7_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM7', 'surrounding_text': 'Supplementary  Data  4', 'description': 'Supplementary Data 4', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM8_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM8_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM8', 'surrounding_text': 'Supplementary  Data  5', 'description': 'Supplementary Data 5', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM9_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM9_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM9', 'surrounding_text': 'Supplementary  Data  6', 'description': 'Supplementary Data 6', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM10_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM10_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM10', 'surrounding_text': 'Supplementary  Data  7', 'description': 'Supplementary Data 7', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM11_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM11_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM11', 'surrounding_text': 'Supplementary  Data  8', 'description': 'Supplementary Data 8', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}]\n",
      "base_parser.py - line 302 - DEBUG - Function_call: reconstruct_download_link(41467_2018_6162_MOESM12_ESM.xlsx, local-data, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466)\n",
      "base_parser.py - line 305 - DEBUG - Inputs to reconstruct_download_link: 41467_2018_6162_MOESM12_ESM.xlsx, local-data, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466, 6141466\n",
      "xml_parser.py - line 497 - DEBUG - Extracted media item with href: 41467_2018_6162_MOESM12_ESM.xlsx\n",
      "xml_parser.py - line 498 - DEBUG - Source url: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466\n",
      "xml_parser.py - line 499 - DEBUG - Supplementary material title: No Title\n",
      "xml_parser.py - line 500 - DEBUG - Content type: local-data, ID: MOESM12\n",
      "xml_parser.py - line 501 - DEBUG - Surrounding text for media: Supplementary  Data  9\n",
      "xml_parser.py - line 502 - DEBUG - Description: Supplementary Data 9\n",
      "xml_parser.py - line 503 - DEBUG - Download_link: https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM12_ESM.xlsx\n",
      "xml_parser.py - line 518 - DEBUG - Extracted item: {\n",
      "    \"link\": \"41467_2018_6162_MOESM12_ESM.xlsx\",\n",
      "    \"source_url\": \"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466\",\n",
      "    \"download_link\": \"https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM12_ESM.xlsx\",\n",
      "    \"title\": \"No Title\",\n",
      "    \"content_type\": \"local-data\",\n",
      "    \"id\": \"MOESM12\",\n",
      "    \"surrounding_text\": \"Supplementary  Data  9\",\n",
      "    \"description\": \"Supplementary Data 9\",\n",
      "    \"source_section\": \"supplementary material\",\n",
      "    \"retrieval_pattern\": \".//supplementary-material\"\n",
      "}\n",
      "xml_parser.py - line 522 - DEBUG - Found 0 inline-supplementary-material elements.\n",
      "xml_parser.py - line 538 - DEBUG - Extracted supplementary material links:\n",
      "[{'link': '41467_2018_6162_MOESM1_ESM.pdf', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM1_ESM.pdf', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM1', 'surrounding_text': 'Supplementary  Information', 'description': 'Supplementary Information', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM2_ESM.pdf', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM2_ESM.pdf', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM2', 'surrounding_text': 'Peer  Review  File', 'description': 'Peer Review File', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM3_ESM.pdf', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM3_ESM.pdf', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM3', 'surrounding_text': 'Description  of  Additional  Supplementary  Files', 'description': 'Description of Additional Supplementary Files', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM4_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM4_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM4', 'surrounding_text': 'Supplementary  Data  1', 'description': 'Supplementary Data 1', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM5_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM5_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM5', 'surrounding_text': 'Supplementary  Data  2', 'description': 'Supplementary Data 2', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM6_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM6_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM6', 'surrounding_text': 'Supplementary  Data  3', 'description': 'Supplementary Data 3', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM7_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM7_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM7', 'surrounding_text': 'Supplementary  Data  4', 'description': 'Supplementary Data 4', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM8_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM8_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM8', 'surrounding_text': 'Supplementary  Data  5', 'description': 'Supplementary Data 5', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM9_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM9_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM9', 'surrounding_text': 'Supplementary  Data  6', 'description': 'Supplementary Data 6', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM10_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM10_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM10', 'surrounding_text': 'Supplementary  Data  7', 'description': 'Supplementary Data 7', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM11_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM11_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM11', 'surrounding_text': 'Supplementary  Data  8', 'description': 'Supplementary Data 8', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM12_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM12_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM12', 'surrounding_text': 'Supplementary  Data  9', 'description': 'Supplementary Data 9', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}]\n",
      "base_parser.py - line 302 - DEBUG - Function_call: reconstruct_download_link(41467_2018_6162_MOESM13_ESM.xlsx, local-data, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466)\n",
      "base_parser.py - line 305 - DEBUG - Inputs to reconstruct_download_link: 41467_2018_6162_MOESM13_ESM.xlsx, local-data, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466, 6141466\n",
      "xml_parser.py - line 497 - DEBUG - Extracted media item with href: 41467_2018_6162_MOESM13_ESM.xlsx\n",
      "xml_parser.py - line 498 - DEBUG - Source url: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466\n",
      "xml_parser.py - line 499 - DEBUG - Supplementary material title: No Title\n",
      "xml_parser.py - line 500 - DEBUG - Content type: local-data, ID: MOESM13\n",
      "xml_parser.py - line 501 - DEBUG - Surrounding text for media: Supplementary  Data  10\n",
      "xml_parser.py - line 502 - DEBUG - Description: Supplementary Data 10\n",
      "xml_parser.py - line 503 - DEBUG - Download_link: https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM13_ESM.xlsx\n",
      "xml_parser.py - line 518 - DEBUG - Extracted item: {\n",
      "    \"link\": \"41467_2018_6162_MOESM13_ESM.xlsx\",\n",
      "    \"source_url\": \"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466\",\n",
      "    \"download_link\": \"https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM13_ESM.xlsx\",\n",
      "    \"title\": \"No Title\",\n",
      "    \"content_type\": \"local-data\",\n",
      "    \"id\": \"MOESM13\",\n",
      "    \"surrounding_text\": \"Supplementary  Data  10\",\n",
      "    \"description\": \"Supplementary Data 10\",\n",
      "    \"source_section\": \"supplementary material\",\n",
      "    \"retrieval_pattern\": \".//supplementary-material\"\n",
      "}\n",
      "xml_parser.py - line 522 - DEBUG - Found 0 inline-supplementary-material elements.\n",
      "xml_parser.py - line 538 - DEBUG - Extracted supplementary material links:\n",
      "[{'link': '41467_2018_6162_MOESM1_ESM.pdf', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM1_ESM.pdf', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM1', 'surrounding_text': 'Supplementary  Information', 'description': 'Supplementary Information', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM2_ESM.pdf', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM2_ESM.pdf', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM2', 'surrounding_text': 'Peer  Review  File', 'description': 'Peer Review File', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM3_ESM.pdf', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM3_ESM.pdf', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM3', 'surrounding_text': 'Description  of  Additional  Supplementary  Files', 'description': 'Description of Additional Supplementary Files', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM4_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM4_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM4', 'surrounding_text': 'Supplementary  Data  1', 'description': 'Supplementary Data 1', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM5_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM5_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM5', 'surrounding_text': 'Supplementary  Data  2', 'description': 'Supplementary Data 2', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM6_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM6_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM6', 'surrounding_text': 'Supplementary  Data  3', 'description': 'Supplementary Data 3', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM7_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM7_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM7', 'surrounding_text': 'Supplementary  Data  4', 'description': 'Supplementary Data 4', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM8_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM8_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM8', 'surrounding_text': 'Supplementary  Data  5', 'description': 'Supplementary Data 5', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM9_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM9_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM9', 'surrounding_text': 'Supplementary  Data  6', 'description': 'Supplementary Data 6', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM10_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM10_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM10', 'surrounding_text': 'Supplementary  Data  7', 'description': 'Supplementary Data 7', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM11_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM11_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM11', 'surrounding_text': 'Supplementary  Data  8', 'description': 'Supplementary Data 8', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM12_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM12_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM12', 'surrounding_text': 'Supplementary  Data  9', 'description': 'Supplementary Data 9', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM13_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM13_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM13', 'surrounding_text': 'Supplementary  Data  10', 'description': 'Supplementary Data 10', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}]\n",
      "base_parser.py - line 302 - DEBUG - Function_call: reconstruct_download_link(41467_2018_6162_MOESM14_ESM.xlsx, local-data, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466)\n",
      "base_parser.py - line 305 - DEBUG - Inputs to reconstruct_download_link: 41467_2018_6162_MOESM14_ESM.xlsx, local-data, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466, 6141466\n",
      "xml_parser.py - line 497 - DEBUG - Extracted media item with href: 41467_2018_6162_MOESM14_ESM.xlsx\n",
      "xml_parser.py - line 498 - DEBUG - Source url: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466\n",
      "xml_parser.py - line 499 - DEBUG - Supplementary material title: No Title\n",
      "xml_parser.py - line 500 - DEBUG - Content type: local-data, ID: MOESM14\n",
      "xml_parser.py - line 501 - DEBUG - Surrounding text for media: Supplementary  Data  11\n",
      "xml_parser.py - line 502 - DEBUG - Description: Supplementary Data 11\n",
      "xml_parser.py - line 503 - DEBUG - Download_link: https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM14_ESM.xlsx\n",
      "xml_parser.py - line 518 - DEBUG - Extracted item: {\n",
      "    \"link\": \"41467_2018_6162_MOESM14_ESM.xlsx\",\n",
      "    \"source_url\": \"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466\",\n",
      "    \"download_link\": \"https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM14_ESM.xlsx\",\n",
      "    \"title\": \"No Title\",\n",
      "    \"content_type\": \"local-data\",\n",
      "    \"id\": \"MOESM14\",\n",
      "    \"surrounding_text\": \"Supplementary  Data  11\",\n",
      "    \"description\": \"Supplementary Data 11\",\n",
      "    \"source_section\": \"supplementary material\",\n",
      "    \"retrieval_pattern\": \".//supplementary-material\"\n",
      "}\n",
      "xml_parser.py - line 522 - DEBUG - Found 0 inline-supplementary-material elements.\n",
      "xml_parser.py - line 538 - DEBUG - Extracted supplementary material links:\n",
      "[{'link': '41467_2018_6162_MOESM1_ESM.pdf', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM1_ESM.pdf', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM1', 'surrounding_text': 'Supplementary  Information', 'description': 'Supplementary Information', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM2_ESM.pdf', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM2_ESM.pdf', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM2', 'surrounding_text': 'Peer  Review  File', 'description': 'Peer Review File', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM3_ESM.pdf', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM3_ESM.pdf', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM3', 'surrounding_text': 'Description  of  Additional  Supplementary  Files', 'description': 'Description of Additional Supplementary Files', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM4_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM4_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM4', 'surrounding_text': 'Supplementary  Data  1', 'description': 'Supplementary Data 1', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM5_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM5_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM5', 'surrounding_text': 'Supplementary  Data  2', 'description': 'Supplementary Data 2', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM6_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM6_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM6', 'surrounding_text': 'Supplementary  Data  3', 'description': 'Supplementary Data 3', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM7_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM7_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM7', 'surrounding_text': 'Supplementary  Data  4', 'description': 'Supplementary Data 4', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM8_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM8_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM8', 'surrounding_text': 'Supplementary  Data  5', 'description': 'Supplementary Data 5', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM9_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM9_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM9', 'surrounding_text': 'Supplementary  Data  6', 'description': 'Supplementary Data 6', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM10_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM10_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM10', 'surrounding_text': 'Supplementary  Data  7', 'description': 'Supplementary Data 7', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM11_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM11_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM11', 'surrounding_text': 'Supplementary  Data  8', 'description': 'Supplementary Data 8', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM12_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM12_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM12', 'surrounding_text': 'Supplementary  Data  9', 'description': 'Supplementary Data 9', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM13_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM13_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM13', 'surrounding_text': 'Supplementary  Data  10', 'description': 'Supplementary Data 10', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM14_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM14_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM14', 'surrounding_text': 'Supplementary  Data  11', 'description': 'Supplementary Data 11', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}]\n",
      "base_parser.py - line 302 - DEBUG - Function_call: reconstruct_download_link(41467_2018_6162_MOESM15_ESM.xlsx, local-data, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466)\n",
      "base_parser.py - line 305 - DEBUG - Inputs to reconstruct_download_link: 41467_2018_6162_MOESM15_ESM.xlsx, local-data, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466, 6141466\n",
      "xml_parser.py - line 497 - DEBUG - Extracted media item with href: 41467_2018_6162_MOESM15_ESM.xlsx\n",
      "xml_parser.py - line 498 - DEBUG - Source url: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466\n",
      "xml_parser.py - line 499 - DEBUG - Supplementary material title: No Title\n",
      "xml_parser.py - line 500 - DEBUG - Content type: local-data, ID: MOESM15\n",
      "xml_parser.py - line 501 - DEBUG - Surrounding text for media: Supplementary  Data  12\n",
      "xml_parser.py - line 502 - DEBUG - Description: Supplementary Data 12\n",
      "xml_parser.py - line 503 - DEBUG - Download_link: https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM15_ESM.xlsx\n",
      "xml_parser.py - line 518 - DEBUG - Extracted item: {\n",
      "    \"link\": \"41467_2018_6162_MOESM15_ESM.xlsx\",\n",
      "    \"source_url\": \"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466\",\n",
      "    \"download_link\": \"https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM15_ESM.xlsx\",\n",
      "    \"title\": \"No Title\",\n",
      "    \"content_type\": \"local-data\",\n",
      "    \"id\": \"MOESM15\",\n",
      "    \"surrounding_text\": \"Supplementary  Data  12\",\n",
      "    \"description\": \"Supplementary Data 12\",\n",
      "    \"source_section\": \"supplementary material\",\n",
      "    \"retrieval_pattern\": \".//supplementary-material\"\n",
      "}\n",
      "xml_parser.py - line 522 - DEBUG - Found 0 inline-supplementary-material elements.\n",
      "xml_parser.py - line 538 - DEBUG - Extracted supplementary material links:\n",
      "[{'link': '41467_2018_6162_MOESM1_ESM.pdf', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM1_ESM.pdf', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM1', 'surrounding_text': 'Supplementary  Information', 'description': 'Supplementary Information', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM2_ESM.pdf', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM2_ESM.pdf', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM2', 'surrounding_text': 'Peer  Review  File', 'description': 'Peer Review File', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM3_ESM.pdf', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM3_ESM.pdf', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM3', 'surrounding_text': 'Description  of  Additional  Supplementary  Files', 'description': 'Description of Additional Supplementary Files', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM4_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM4_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM4', 'surrounding_text': 'Supplementary  Data  1', 'description': 'Supplementary Data 1', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM5_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM5_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM5', 'surrounding_text': 'Supplementary  Data  2', 'description': 'Supplementary Data 2', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM6_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM6_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM6', 'surrounding_text': 'Supplementary  Data  3', 'description': 'Supplementary Data 3', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM7_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM7_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM7', 'surrounding_text': 'Supplementary  Data  4', 'description': 'Supplementary Data 4', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM8_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM8_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM8', 'surrounding_text': 'Supplementary  Data  5', 'description': 'Supplementary Data 5', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM9_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM9_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM9', 'surrounding_text': 'Supplementary  Data  6', 'description': 'Supplementary Data 6', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM10_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM10_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM10', 'surrounding_text': 'Supplementary  Data  7', 'description': 'Supplementary Data 7', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM11_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM11_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM11', 'surrounding_text': 'Supplementary  Data  8', 'description': 'Supplementary Data 8', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM12_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM12_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM12', 'surrounding_text': 'Supplementary  Data  9', 'description': 'Supplementary Data 9', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM13_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM13_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM13', 'surrounding_text': 'Supplementary  Data  10', 'description': 'Supplementary Data 10', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM14_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM14_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM14', 'surrounding_text': 'Supplementary  Data  11', 'description': 'Supplementary Data 11', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM15_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM15_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM15', 'surrounding_text': 'Supplementary  Data  12', 'description': 'Supplementary Data 12', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}]\n",
      "xml_parser.py - line 168 - DEBUG - supplementary_material_links: [{'link': '41467_2018_6162_MOESM1_ESM.pdf', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM1_ESM.pdf', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM1', 'surrounding_text': 'Supplementary  Information', 'description': 'Supplementary Information', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM2_ESM.pdf', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM2_ESM.pdf', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM2', 'surrounding_text': 'Peer  Review  File', 'description': 'Peer Review File', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM3_ESM.pdf', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM3_ESM.pdf', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM3', 'surrounding_text': 'Description  of  Additional  Supplementary  Files', 'description': 'Description of Additional Supplementary Files', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM4_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM4_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM4', 'surrounding_text': 'Supplementary  Data  1', 'description': 'Supplementary Data 1', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM5_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM5_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM5', 'surrounding_text': 'Supplementary  Data  2', 'description': 'Supplementary Data 2', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM6_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM6_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM6', 'surrounding_text': 'Supplementary  Data  3', 'description': 'Supplementary Data 3', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM7_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM7_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM7', 'surrounding_text': 'Supplementary  Data  4', 'description': 'Supplementary Data 4', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM8_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM8_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM8', 'surrounding_text': 'Supplementary  Data  5', 'description': 'Supplementary Data 5', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM9_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM9_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM9', 'surrounding_text': 'Supplementary  Data  6', 'description': 'Supplementary Data 6', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM10_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM10_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM10', 'surrounding_text': 'Supplementary  Data  7', 'description': 'Supplementary Data 7', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM11_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM11_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM11', 'surrounding_text': 'Supplementary  Data  8', 'description': 'Supplementary Data 8', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM12_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM12_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM12', 'surrounding_text': 'Supplementary  Data  9', 'description': 'Supplementary Data 9', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM13_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM13_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM13', 'surrounding_text': 'Supplementary  Data  10', 'description': 'Supplementary Data 10', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM14_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM14_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM14', 'surrounding_text': 'Supplementary  Data  11', 'description': 'Supplementary Data 11', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}, {'link': '41467_2018_6162_MOESM15_ESM.xlsx', 'source_url': 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6141466', 'download_link': 'https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM15_ESM.xlsx', 'title': 'No Title', 'content_type': 'local-data', 'id': 'MOESM15', 'surrounding_text': 'Supplementary  Data  12', 'description': 'Supplementary Data 12', 'source_section': 'supplementary material', 'retrieval_pattern': './/supplementary-material'}]\n",
      "xml_retriever.py - line 123 - INFO - Loading patterns for section 'data_availability_sections' for publisher 'PMC'.\n",
      "xml_retriever.py - line 131 - INFO - Section pattern keys for publisher PMC: dict_keys(['data_availability_sections', 'supplementary_material_sections', 'supplementary_data_sections', 'key_resources_table', 'methods_sections'])\n",
      "xml_retriever.py - line 133 - INFO - Section name: data_availability_sections\n",
      "xml_retriever.py - line 135 - INFO - Found section 'data_availability_sections' in patterns for publisher 'PMC'.\n",
      "xml_retriever.py - line 137 - INFO - Loaded patterns for section 'data_availability_sections': [\".//sec[@sec-type='data-availability']\", \".//notes[@notes-type='data-availability']\", \".//notes[@notes-type='data-availability']\"]\n",
      "base_parser.py - line 284 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_data_sections)\n",
      "base_parser.py - line 285 - INFO - Consider migrating this function to the BaseRetriever class.\n",
      "xml_parser.py - line 653 - INFO - Found 16 supplementary data sections\n",
      "xml_parser.py - line 658 - DEBUG - Section with no text: <Element sec at 0x1aa6de540>\n",
      "xml_parser.py - line 658 - DEBUG - Section with no text: <Element supplementary-material at 0x1aa6de4c0>\n",
      "xml_parser.py - line 658 - DEBUG - Section with no text: <Element supplementary-material at 0x1aa6de380>\n",
      "xml_parser.py - line 658 - DEBUG - Section with no text: <Element supplementary-material at 0x1aa6de7c0>\n",
      "xml_parser.py - line 658 - DEBUG - Section with no text: <Element supplementary-material at 0x1aa6de400>\n",
      "xml_parser.py - line 658 - DEBUG - Section with no text: <Element supplementary-material at 0x1aa6de880>\n",
      "xml_parser.py - line 658 - DEBUG - Section with no text: <Element supplementary-material at 0x1aa6de940>\n",
      "xml_parser.py - line 658 - DEBUG - Section with no text: <Element supplementary-material at 0x1aa6de8c0>\n",
      "xml_parser.py - line 658 - DEBUG - Section with no text: <Element supplementary-material at 0x1aa6de980>\n",
      "xml_parser.py - line 658 - DEBUG - Section with no text: <Element supplementary-material at 0x1aa6de9c0>\n",
      "xml_parser.py - line 658 - DEBUG - Section with no text: <Element supplementary-material at 0x1aa6dea00>\n",
      "xml_parser.py - line 658 - DEBUG - Section with no text: <Element supplementary-material at 0x13c864b00>\n",
      "xml_parser.py - line 658 - DEBUG - Section with no text: <Element supplementary-material at 0x1aa6dea40>\n",
      "xml_parser.py - line 658 - DEBUG - Section with no text: <Element supplementary-material at 0x1aa6deac0>\n",
      "xml_parser.py - line 658 - DEBUG - Section with no text: <Element supplementary-material at 0x1aa6deb00>\n",
      "xml_parser.py - line 658 - DEBUG - Section with no text: <Element supplementary-material at 0x1aa6deb40>\n",
      "base_parser.py - line 284 - INFO - Function_call: load_patterns_for_tgt_section(key_resources_table)\n",
      "base_parser.py - line 285 - INFO - Consider migrating this function to the BaseRetriever class.\n",
      "xml_parser.py - line 677 - INFO - Found data availability content: [' Data availability  Data has been deposited in dbGAP with accession #PHS001049 (  https://www.ncbi.nlm.nih.gov/gap/?term=PHS001049  ), and SRA with project accession PRJNA306801 (  https://www.ncbi.nlm.nih.gov/gap/?term=PHS001049  https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA306801)  .  https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA306801) ']\n",
      "base_parser.py - line 381 - INFO - Processing DAS_content: [' Data availability  Data has been deposited in dbGAP with accession #PHS001049 (  https://www.ncbi.nlm.nih.gov/gap/?term=PHS001049  ), and SRA with project accession PRJNA306801 (  https://www.ncbi.nlm.nih.gov/gap/?term=PHS001049  https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA306801)  .  https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA306801) ']\n",
      "base_parser.py - line 426 - INFO - Loading prompt: retrieve_datasets_simple_JSON for model gemini-2.0-flash\n",
      "prompt_manager.py - line 32 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \n",
      "base_parser.py - line 1108 - INFO - Expected string but got list. Converting list to string.\n",
      "base_parser.py - line 1115 - DEBUG - Counting tokens for model: gemini-2.0-flash, prompt length: 1382 char\n",
      "base_parser.py - line 1124 - DEBUG - Rough estimate of token count for Gemini model 'gemini-2.0-flash': 345\n",
      "base_parser.py - line 433 - INFO - Content length: 353\n",
      "base_parser.py - line 435 - DEBUG - static_prompt: [{'role': 'system', 'content': \"You are a specialized assistant that extracts dataset references from the content of scientific papers. You must output a JSON array of objects, where each object has the following keys: 'dataset_identifier', 'data_repository', and 'dataset_webpage'. Follow the structure of the provided examples exactly. We will not wrap the json codes in JSON markers.\"}, {'role': 'user', 'content': 'Extract dataset references based on the examples below:\\n\\nExample 1:\\nContent: \\'The study used dataset EGAS00893672193, which is available at the European Genome Archive.\\'\\nResponse:\\n[\\n    [\\n        \"dataset_identifier\": \"EGAS00893672193\",\\n        \"data_repository\": \"European Genome Archive\",\\n        \"dataset_webpage\": \"https://ega-archive.org/studies/EGAS00893672193\"\\n    ]\\n]\\n\\nExample 2:\\nContent: \\'Proteomics data was obtained from PRIDE, accession PXD029921.\\'\\nResponse:\\n[\\n    [\\n        \"dataset_identifier\": \"PXD029921\",\\n        \"data_repository\": \"PRIDE\",\\n        \"dataset_webpage\": \"https://www.ebi.ac.uk/pride/archive/projects/PXD029921\"\\n    ]\\n]\\n\\nExample 3:\\nContent: \\'The repository dbGaP hosts the dataset phs011366.v1.p1 at this location.\\'\\nResponse:\\n[\\n    [\\n        \"dataset_identifier\": \"phs011366.v1.p1\",\\n        \"data_repository\": \"dbGaP\",\\n        \"dataset_webpage\": \"https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001366.v1.p1\"\\n    ]\\n]\\n\\nNow process the following content:\\n\\nContent: {content}.'}]\n",
      "prompt_manager.py - line 56 - DEBUG - Rendering prompt with dynamic parts(<class 'dict'>): {'content': ' Data availability  Data has been deposited in dbGAP with accession #PHS001049 (  https://www.ncbi.nlm.nih.gov/gap/?term=PHS001049  ), and SRA with project accession PRJNA306801 (  https://www.ncbi.nlm.nih.gov/gap/?term=PHS001049  https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA306801)  .  https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA306801) ', 'repos': 'BIGD, European Genome-Phenome Archive, MassIVE, PRIDE, iProX, ProteomeCentral, ProteomeXchange, Proteomic Data Commons, Genomic Data Commons, The Cancer Imaging Archive, Imaging Data Commons, JPOST, doi.org, GEO, dbGaP, sra, ArrayExpress'}, and items: [{'role': 'system', 'content': \"You are a specialized assistant that extracts dataset references from the content of scientific papers. You must output a JSON array of objects, where each object has the following keys: 'dataset_identifier', 'data_repository', and 'dataset_webpage'. Follow the structure of the provided examples exactly. We will not wrap the json codes in JSON markers.\"}, {'role': 'user', 'content': 'Extract dataset references based on the examples below:\\n\\nExample 1:\\nContent: \\'The study used dataset EGAS00893672193, which is available at the European Genome Archive.\\'\\nResponse:\\n[\\n    [\\n        \"dataset_identifier\": \"EGAS00893672193\",\\n        \"data_repository\": \"European Genome Archive\",\\n        \"dataset_webpage\": \"https://ega-archive.org/studies/EGAS00893672193\"\\n    ]\\n]\\n\\nExample 2:\\nContent: \\'Proteomics data was obtained from PRIDE, accession PXD029921.\\'\\nResponse:\\n[\\n    [\\n        \"dataset_identifier\": \"PXD029921\",\\n        \"data_repository\": \"PRIDE\",\\n        \"dataset_webpage\": \"https://www.ebi.ac.uk/pride/archive/projects/PXD029921\"\\n    ]\\n]\\n\\nExample 3:\\nContent: \\'The repository dbGaP hosts the dataset phs011366.v1.p1 at this location.\\'\\nResponse:\\n[\\n    [\\n        \"dataset_identifier\": \"phs011366.v1.p1\",\\n        \"data_repository\": \"dbGaP\",\\n        \"dataset_webpage\": \"https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001366.v1.p1\"\\n    ]\\n]\\n\\nNow process the following content:\\n\\nContent: {content}.'}]\n",
      "base_parser.py - line 1108 - INFO - Expected string but got list. Converting list to string.\n",
      "base_parser.py - line 1115 - DEBUG - Counting tokens for model: gemini-2.0-flash, prompt length: 1726 char\n",
      "base_parser.py - line 1124 - DEBUG - Rough estimate of token count for Gemini model 'gemini-2.0-flash': 431\n",
      "base_parser.py - line 444 - INFO - Prompt messages total length: 431 tokens\n",
      "base_parser.py - line 445 - DEBUG - Prompt messages: [{'role': 'system', 'content': \"You are a specialized assistant that extracts dataset references from the content of scientific papers. You must output a JSON array of objects, where each object has the following keys: 'dataset_identifier', 'data_repository', and 'dataset_webpage'. Follow the structure of the provided examples exactly. We will not wrap the json codes in JSON markers.\"}, {'role': 'user', 'content': 'Extract dataset references based on the examples below:\\n\\nExample 1:\\nContent: \\'The study used dataset EGAS00893672193, which is available at the European Genome Archive.\\'\\nResponse:\\n[\\n    [\\n        \"dataset_identifier\": \"EGAS00893672193\",\\n        \"data_repository\": \"European Genome Archive\",\\n        \"dataset_webpage\": \"https://ega-archive.org/studies/EGAS00893672193\"\\n    ]\\n]\\n\\nExample 2:\\nContent: \\'Proteomics data was obtained from PRIDE, accession PXD029921.\\'\\nResponse:\\n[\\n    [\\n        \"dataset_identifier\": \"PXD029921\",\\n        \"data_repository\": \"PRIDE\",\\n        \"dataset_webpage\": \"https://www.ebi.ac.uk/pride/archive/projects/PXD029921\"\\n    ]\\n]\\n\\nExample 3:\\nContent: \\'The repository dbGaP hosts the dataset phs011366.v1.p1 at this location.\\'\\nResponse:\\n[\\n    [\\n        \"dataset_identifier\": \"phs011366.v1.p1\",\\n        \"data_repository\": \"dbGaP\",\\n        \"dataset_webpage\": \"https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001366.v1.p1\"\\n    ]\\n]\\n\\nNow process the following content:\\n\\nContent:  Data availability  Data has been deposited in dbGAP with accession #PHS001049 (  https://www.ncbi.nlm.nih.gov/gap/?term=PHS001049  ), and SRA with project accession PRJNA306801 (  https://www.ncbi.nlm.nih.gov/gap/?term=PHS001049  https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA306801)  .  https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA306801) .'}]\n",
      "prompt_manager.py - line 96 - DEBUG - Calculating checksum for content: prompt\n",
      "base_parser.py - line 450 - INFO - Prompt ID: gemini-2.0-flash-0-ce6f552859cf2eead778e2fea3d07d4b5eb6e24b70262116329e848e53fbc2a2\n",
      "base_parser.py - line 1108 - INFO - Expected string but got list. Converting list to string.\n",
      "base_parser.py - line 1115 - DEBUG - Counting tokens for model: gemini-2.0-flash, prompt length: 1726 char\n",
      "base_parser.py - line 1124 - DEBUG - Rough estimate of token count for Gemini model 'gemini-2.0-flash': 431\n",
      "base_parser.py - line 469 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages: 431 tokens\n",
      "base_parser.py - line 540 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-20d394cd-8419-44e5-b7de-1630d8170b61\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"PHS001049\\\",\\n        \\\"data_repository\\\": \\\"dbGAP\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=PHS001049\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"PRJNA306801\\\",\\n        \\\"data_repository\\\": \\\"SRA\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA306801\\\"\\n    }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"PHS001049\\\",\\n        \\\"data_repository\\\": \\\"dbGAP\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=PHS001049\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"PRJNA306801\\\",\\n        \\\"data_repository\\\": \\\"SRA\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA306801\\\"\\n    }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1751902232,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 588,\n",
      "        \"completion_tokens\": 157,\n",
      "        \"total_tokens\": 745,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\n",
      "base_parser.py - line 394 - INFO - datasets (<class 'list'>): [{'dataset_identifier': 'PHS001049', 'data_repository': 'dbGAP', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=PHS001049'}, {'dataset_identifier': 'PRJNA306801', 'data_repository': 'SRA', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA306801'}]\n",
      "base_parser.py - line 396 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'PHS001049', 'data_repository': 'dbGAP', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=PHS001049'}\n",
      "base_parser.py - line 398 - WARNING - Adding retrieval pattern 'data availability' to dataset\n",
      "base_parser.py - line 396 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'PRJNA306801', 'data_repository': 'SRA', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA306801'}\n",
      "base_parser.py - line 398 - WARNING - Adding retrieval pattern 'data availability' to dataset\n",
      "base_parser.py - line 402 - INFO - Final ret additional data: 2 items\n",
      "base_parser.py - line 403 - DEBUG - Final ret additional data: [{'dataset_identifier': 'PHS001049', 'data_repository': 'dbGAP', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=PHS001049', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'PRJNA306801', 'data_repository': 'SRA', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA306801', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}]\n",
      "xml_parser.py - line 214 - DEBUG - Content of augmented_dataset_links: [{'dataset_identifier': 'PHS001049', 'data_repository': 'dbGAP', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=PHS001049', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}, {'dataset_identifier': 'PRJNA306801', 'data_repository': 'SRA', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA306801', 'source_section': 'data_availability', 'retrieval_pattern': 'data availability'}]\n",
      "base_parser.py - line 991 - INFO - Fetching metadata for 2 datasets\n",
      "base_parser.py - line 999 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\n",
      "base_parser.py - line 1010 - INFO - Raw accession ID: PHS001049\n",
      "base_parser.py - line 939 - INFO - Resolving data repository for: dbGAP\n",
      "base_parser.py - line 952 - DEBUG - Checking if dbGAP == bigd.big.ac.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if dbGAP == ega-archive.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if dbGAP == massive.ucsd.edu\n",
      "base_parser.py - line 952 - DEBUG - Checking if dbGAP == www.ebi.ac.uk\n",
      "base_parser.py - line 952 - DEBUG - Checking if dbGAP == www.iprox.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if dbGAP == proteomecentral.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if dbGAP == www.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if dbGAP == pdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if dbGAP == gdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if dbGAP == cancerimagingarchive.net\n",
      "base_parser.py - line 952 - DEBUG - Checking if dbGAP == imaging.datacommons.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if dbGAP == repository.jpostdb.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if dbGAP == doi.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if dbGAP == geo\n",
      "base_parser.py - line 952 - DEBUG - Checking if dbGAP == dbgap\n",
      "base_parser.py - line 963 - INFO - Found repo_name match for dbGAP\n",
      "base_parser.py - line 921 - INFO - Resolving accession ID for PHS001049 in dbgap\n",
      "base_parser.py - line 926 - WARNING - Identifier PHS001049 does not match pattern for dbgap\n",
      "base_parser.py - line 1024 - INFO - Processing dataset 1 with repo: dbgap and accession_id: phs001049.v1.p1\n",
      "base_parser.py - line 1025 - DEBUG - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\n",
      "base_parser.py - line 1050 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001049.v1.p1\n",
      "base_parser.py - line 1057 - INFO - Adding access mode for dataset 1: Application to access\n",
      "base_parser.py - line 999 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\n",
      "base_parser.py - line 1010 - INFO - Raw accession ID: PRJNA306801\n",
      "base_parser.py - line 939 - INFO - Resolving data repository for: SRA\n",
      "base_parser.py - line 952 - DEBUG - Checking if SRA == bigd.big.ac.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if SRA == ega-archive.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if SRA == massive.ucsd.edu\n",
      "base_parser.py - line 952 - DEBUG - Checking if SRA == www.ebi.ac.uk\n",
      "base_parser.py - line 952 - DEBUG - Checking if SRA == www.iprox.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if SRA == proteomecentral.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if SRA == www.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if SRA == pdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if SRA == gdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if SRA == cancerimagingarchive.net\n",
      "base_parser.py - line 952 - DEBUG - Checking if SRA == imaging.datacommons.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if SRA == repository.jpostdb.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if SRA == doi.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if SRA == geo\n",
      "base_parser.py - line 952 - DEBUG - Checking if SRA == dbgap\n",
      "base_parser.py - line 952 - DEBUG - Checking if SRA == sra\n",
      "base_parser.py - line 963 - INFO - Found repo_name match for SRA\n",
      "base_parser.py - line 921 - INFO - Resolving accession ID for PRJNA306801 in sra\n",
      "base_parser.py - line 1024 - INFO - Processing dataset 2 with repo: sra and accession_id: PRJNA306801\n",
      "base_parser.py - line 1025 - DEBUG - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\n",
      "base_parser.py - line 1050 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA306801\n",
      "base_parser.py - line 1057 - INFO - Adding access mode for dataset 2: Complex download\n",
      "base_parser.py - line 1067 - INFO - Updated datasets len: 2\n",
      "xml_parser.py - line 222 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 17, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'source_section', 'retrieval_pattern', 'access_mode', 'link',\n",
      "       'source_url', 'download_link', 'title', 'content_type', 'id',\n",
      "       'surrounding_text', 'description'],\n",
      "      dtype='object')\n",
      "xml_parser.py - line 223 - DEBUG - Datasets:    dataset_identifier data_repository  \\\n",
      "0           PHS001049           dbGAP   \n",
      "1         PRJNA306801             SRA   \n",
      "0                 NaN             NaN   \n",
      "1                 NaN             NaN   \n",
      "2                 NaN             NaN   \n",
      "3                 NaN             NaN   \n",
      "4                 NaN             NaN   \n",
      "5                 NaN             NaN   \n",
      "6                 NaN             NaN   \n",
      "7                 NaN             NaN   \n",
      "8                 NaN             NaN   \n",
      "9                 NaN             NaN   \n",
      "10                NaN             NaN   \n",
      "11                NaN             NaN   \n",
      "12                NaN             NaN   \n",
      "13                NaN             NaN   \n",
      "14                NaN             NaN   \n",
      "\n",
      "                                      dataset_webpage          source_section  \\\n",
      "0   https://www.ncbi.nlm.nih.gov/projects/gap/cgi-...       data_availability   \n",
      "1   https://www.ncbi.nlm.nih.gov/bioproject/?term=...       data_availability   \n",
      "0                                                 NaN  supplementary material   \n",
      "1                                                 NaN  supplementary material   \n",
      "2                                                 NaN  supplementary material   \n",
      "3                                                 NaN  supplementary material   \n",
      "4                                                 NaN  supplementary material   \n",
      "5                                                 NaN  supplementary material   \n",
      "6                                                 NaN  supplementary material   \n",
      "7                                                 NaN  supplementary material   \n",
      "8                                                 NaN  supplementary material   \n",
      "9                                                 NaN  supplementary material   \n",
      "10                                                NaN  supplementary material   \n",
      "11                                                NaN  supplementary material   \n",
      "12                                                NaN  supplementary material   \n",
      "13                                                NaN  supplementary material   \n",
      "14                                                NaN  supplementary material   \n",
      "\n",
      "            retrieval_pattern            access_mode  \\\n",
      "0           data availability  Application to access   \n",
      "1           data availability       Complex download   \n",
      "0   .//supplementary-material                    NaN   \n",
      "1   .//supplementary-material                    NaN   \n",
      "2   .//supplementary-material                    NaN   \n",
      "3   .//supplementary-material                    NaN   \n",
      "4   .//supplementary-material                    NaN   \n",
      "5   .//supplementary-material                    NaN   \n",
      "6   .//supplementary-material                    NaN   \n",
      "7   .//supplementary-material                    NaN   \n",
      "8   .//supplementary-material                    NaN   \n",
      "9   .//supplementary-material                    NaN   \n",
      "10  .//supplementary-material                    NaN   \n",
      "11  .//supplementary-material                    NaN   \n",
      "12  .//supplementary-material                    NaN   \n",
      "13  .//supplementary-material                    NaN   \n",
      "14  .//supplementary-material                    NaN   \n",
      "\n",
      "                                link  \\\n",
      "0                                NaN   \n",
      "1                                NaN   \n",
      "0     41467_2018_6162_MOESM1_ESM.pdf   \n",
      "1     41467_2018_6162_MOESM2_ESM.pdf   \n",
      "2     41467_2018_6162_MOESM3_ESM.pdf   \n",
      "3    41467_2018_6162_MOESM4_ESM.xlsx   \n",
      "4    41467_2018_6162_MOESM5_ESM.xlsx   \n",
      "5    41467_2018_6162_MOESM6_ESM.xlsx   \n",
      "6    41467_2018_6162_MOESM7_ESM.xlsx   \n",
      "7    41467_2018_6162_MOESM8_ESM.xlsx   \n",
      "8    41467_2018_6162_MOESM9_ESM.xlsx   \n",
      "9   41467_2018_6162_MOESM10_ESM.xlsx   \n",
      "10  41467_2018_6162_MOESM11_ESM.xlsx   \n",
      "11  41467_2018_6162_MOESM12_ESM.xlsx   \n",
      "12  41467_2018_6162_MOESM13_ESM.xlsx   \n",
      "13  41467_2018_6162_MOESM14_ESM.xlsx   \n",
      "14  41467_2018_6162_MOESM15_ESM.xlsx   \n",
      "\n",
      "                                           source_url  \\\n",
      "0                                                 NaN   \n",
      "1                                                 NaN   \n",
      "0   https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...   \n",
      "1   https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...   \n",
      "2   https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...   \n",
      "3   https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...   \n",
      "4   https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...   \n",
      "5   https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...   \n",
      "6   https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...   \n",
      "7   https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...   \n",
      "8   https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...   \n",
      "9   https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...   \n",
      "10  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...   \n",
      "11  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...   \n",
      "12  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...   \n",
      "13  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...   \n",
      "14  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...   \n",
      "\n",
      "                                        download_link     title content_type  \\\n",
      "0                                                 NaN       NaN          NaN   \n",
      "1                                                 NaN       NaN          NaN   \n",
      "0   https://pmc.ncbi.nlm.nih.gov/articles/instance...  No Title   local-data   \n",
      "1   https://pmc.ncbi.nlm.nih.gov/articles/instance...  No Title   local-data   \n",
      "2   https://pmc.ncbi.nlm.nih.gov/articles/instance...  No Title   local-data   \n",
      "3   https://pmc.ncbi.nlm.nih.gov/articles/instance...  No Title   local-data   \n",
      "4   https://pmc.ncbi.nlm.nih.gov/articles/instance...  No Title   local-data   \n",
      "5   https://pmc.ncbi.nlm.nih.gov/articles/instance...  No Title   local-data   \n",
      "6   https://pmc.ncbi.nlm.nih.gov/articles/instance...  No Title   local-data   \n",
      "7   https://pmc.ncbi.nlm.nih.gov/articles/instance...  No Title   local-data   \n",
      "8   https://pmc.ncbi.nlm.nih.gov/articles/instance...  No Title   local-data   \n",
      "9   https://pmc.ncbi.nlm.nih.gov/articles/instance...  No Title   local-data   \n",
      "10  https://pmc.ncbi.nlm.nih.gov/articles/instance...  No Title   local-data   \n",
      "11  https://pmc.ncbi.nlm.nih.gov/articles/instance...  No Title   local-data   \n",
      "12  https://pmc.ncbi.nlm.nih.gov/articles/instance...  No Title   local-data   \n",
      "13  https://pmc.ncbi.nlm.nih.gov/articles/instance...  No Title   local-data   \n",
      "14  https://pmc.ncbi.nlm.nih.gov/articles/instance...  No Title   local-data   \n",
      "\n",
      "         id                                   surrounding_text  \\\n",
      "0       NaN                                                NaN   \n",
      "1       NaN                                                NaN   \n",
      "0    MOESM1                         Supplementary  Information   \n",
      "1    MOESM2                                 Peer  Review  File   \n",
      "2    MOESM3  Description  of  Additional  Supplementary  Files   \n",
      "3    MOESM4                             Supplementary  Data  1   \n",
      "4    MOESM5                             Supplementary  Data  2   \n",
      "5    MOESM6                             Supplementary  Data  3   \n",
      "6    MOESM7                             Supplementary  Data  4   \n",
      "7    MOESM8                             Supplementary  Data  5   \n",
      "8    MOESM9                             Supplementary  Data  6   \n",
      "9   MOESM10                             Supplementary  Data  7   \n",
      "10  MOESM11                             Supplementary  Data  8   \n",
      "11  MOESM12                             Supplementary  Data  9   \n",
      "12  MOESM13                            Supplementary  Data  10   \n",
      "13  MOESM14                            Supplementary  Data  11   \n",
      "14  MOESM15                            Supplementary  Data  12   \n",
      "\n",
      "                                      description  \n",
      "0                                             NaN  \n",
      "1                                             NaN  \n",
      "0                       Supplementary Information  \n",
      "1                                Peer Review File  \n",
      "2   Description of Additional Supplementary Files  \n",
      "3                            Supplementary Data 1  \n",
      "4                            Supplementary Data 2  \n",
      "5                            Supplementary Data 3  \n",
      "6                            Supplementary Data 4  \n",
      "7                            Supplementary Data 5  \n",
      "8                            Supplementary Data 6  \n",
      "9                            Supplementary Data 7  \n",
      "10                           Supplementary Data 8  \n",
      "11                           Supplementary Data 9  \n",
      "12                          Supplementary Data 10  \n",
      "13                          Supplementary Data 11  \n",
      "14                          Supplementary Data 12  \n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(nan)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(nan)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM1_ESM.pdf)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM2_ESM.pdf)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM3_ESM.pdf)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM4_ESM.xlsx)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM5_ESM.xlsx)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM6_ESM.xlsx)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM7_ESM.xlsx)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM8_ESM.xlsx)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM9_ESM.xlsx)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM10_ESM.xlsx)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM11_ESM.xlsx)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM12_ESM.xlsx)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM13_ESM.xlsx)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM14_ESM.xlsx)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(https://pmc.ncbi.nlm.nih.gov/articles/instance/6141466/bin/41467_2018_6162_MOESM15_ESM.xlsx)\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "   dataset_identifier data_repository  \\\n",
       "0           PHS001049           dbGAP   \n",
       "1         PRJNA306801             SRA   \n",
       "0                 NaN             NaN   \n",
       "1                 NaN             NaN   \n",
       "2                 NaN             NaN   \n",
       "3                 NaN             NaN   \n",
       "4                 NaN             NaN   \n",
       "5                 NaN             NaN   \n",
       "6                 NaN             NaN   \n",
       "7                 NaN             NaN   \n",
       "8                 NaN             NaN   \n",
       "9                 NaN             NaN   \n",
       "10                NaN             NaN   \n",
       "11                NaN             NaN   \n",
       "12                NaN             NaN   \n",
       "13                NaN             NaN   \n",
       "14                NaN             NaN   \n",
       "\n",
       "                                      dataset_webpage          source_section  \\\n",
       "0   https://www.ncbi.nlm.nih.gov/projects/gap/cgi-...       data_availability   \n",
       "1   https://www.ncbi.nlm.nih.gov/bioproject/?term=...       data_availability   \n",
       "0                                                 NaN  supplementary material   \n",
       "1                                                 NaN  supplementary material   \n",
       "2                                                 NaN  supplementary material   \n",
       "3                                                 NaN  supplementary material   \n",
       "4                                                 NaN  supplementary material   \n",
       "5                                                 NaN  supplementary material   \n",
       "6                                                 NaN  supplementary material   \n",
       "7                                                 NaN  supplementary material   \n",
       "8                                                 NaN  supplementary material   \n",
       "9                                                 NaN  supplementary material   \n",
       "10                                                NaN  supplementary material   \n",
       "11                                                NaN  supplementary material   \n",
       "12                                                NaN  supplementary material   \n",
       "13                                                NaN  supplementary material   \n",
       "14                                                NaN  supplementary material   \n",
       "\n",
       "            retrieval_pattern            access_mode  \\\n",
       "0           data availability  Application to access   \n",
       "1           data availability       Complex download   \n",
       "0   .//supplementary-material                    NaN   \n",
       "1   .//supplementary-material                    NaN   \n",
       "2   .//supplementary-material                    NaN   \n",
       "3   .//supplementary-material                    NaN   \n",
       "4   .//supplementary-material                    NaN   \n",
       "5   .//supplementary-material                    NaN   \n",
       "6   .//supplementary-material                    NaN   \n",
       "7   .//supplementary-material                    NaN   \n",
       "8   .//supplementary-material                    NaN   \n",
       "9   .//supplementary-material                    NaN   \n",
       "10  .//supplementary-material                    NaN   \n",
       "11  .//supplementary-material                    NaN   \n",
       "12  .//supplementary-material                    NaN   \n",
       "13  .//supplementary-material                    NaN   \n",
       "14  .//supplementary-material                    NaN   \n",
       "\n",
       "                                link  \\\n",
       "0                                NaN   \n",
       "1                                NaN   \n",
       "0     41467_2018_6162_MOESM1_ESM.pdf   \n",
       "1     41467_2018_6162_MOESM2_ESM.pdf   \n",
       "2     41467_2018_6162_MOESM3_ESM.pdf   \n",
       "3    41467_2018_6162_MOESM4_ESM.xlsx   \n",
       "4    41467_2018_6162_MOESM5_ESM.xlsx   \n",
       "5    41467_2018_6162_MOESM6_ESM.xlsx   \n",
       "6    41467_2018_6162_MOESM7_ESM.xlsx   \n",
       "7    41467_2018_6162_MOESM8_ESM.xlsx   \n",
       "8    41467_2018_6162_MOESM9_ESM.xlsx   \n",
       "9   41467_2018_6162_MOESM10_ESM.xlsx   \n",
       "10  41467_2018_6162_MOESM11_ESM.xlsx   \n",
       "11  41467_2018_6162_MOESM12_ESM.xlsx   \n",
       "12  41467_2018_6162_MOESM13_ESM.xlsx   \n",
       "13  41467_2018_6162_MOESM14_ESM.xlsx   \n",
       "14  41467_2018_6162_MOESM15_ESM.xlsx   \n",
       "\n",
       "                                           source_url  \\\n",
       "0                                                 NaN   \n",
       "1                                                 NaN   \n",
       "0   https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...   \n",
       "1   https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...   \n",
       "2   https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...   \n",
       "3   https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...   \n",
       "4   https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...   \n",
       "5   https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...   \n",
       "6   https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...   \n",
       "7   https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...   \n",
       "8   https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...   \n",
       "9   https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...   \n",
       "10  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...   \n",
       "11  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...   \n",
       "12  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...   \n",
       "13  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...   \n",
       "14  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...   \n",
       "\n",
       "                                        download_link     title content_type  \\\n",
       "0                                                 NaN       NaN          NaN   \n",
       "1                                                 NaN       NaN          NaN   \n",
       "0   https://pmc.ncbi.nlm.nih.gov/articles/instance...  No Title   local-data   \n",
       "1   https://pmc.ncbi.nlm.nih.gov/articles/instance...  No Title   local-data   \n",
       "2   https://pmc.ncbi.nlm.nih.gov/articles/instance...  No Title   local-data   \n",
       "3   https://pmc.ncbi.nlm.nih.gov/articles/instance...  No Title   local-data   \n",
       "4   https://pmc.ncbi.nlm.nih.gov/articles/instance...  No Title   local-data   \n",
       "5   https://pmc.ncbi.nlm.nih.gov/articles/instance...  No Title   local-data   \n",
       "6   https://pmc.ncbi.nlm.nih.gov/articles/instance...  No Title   local-data   \n",
       "7   https://pmc.ncbi.nlm.nih.gov/articles/instance...  No Title   local-data   \n",
       "8   https://pmc.ncbi.nlm.nih.gov/articles/instance...  No Title   local-data   \n",
       "9   https://pmc.ncbi.nlm.nih.gov/articles/instance...  No Title   local-data   \n",
       "10  https://pmc.ncbi.nlm.nih.gov/articles/instance...  No Title   local-data   \n",
       "11  https://pmc.ncbi.nlm.nih.gov/articles/instance...  No Title   local-data   \n",
       "12  https://pmc.ncbi.nlm.nih.gov/articles/instance...  No Title   local-data   \n",
       "13  https://pmc.ncbi.nlm.nih.gov/articles/instance...  No Title   local-data   \n",
       "14  https://pmc.ncbi.nlm.nih.gov/articles/instance...  No Title   local-data   \n",
       "\n",
       "         id                                   surrounding_text  \\\n",
       "0       NaN                                                NaN   \n",
       "1       NaN                                                NaN   \n",
       "0    MOESM1                         Supplementary  Information   \n",
       "1    MOESM2                                 Peer  Review  File   \n",
       "2    MOESM3  Description  of  Additional  Supplementary  Files   \n",
       "3    MOESM4                             Supplementary  Data  1   \n",
       "4    MOESM5                             Supplementary  Data  2   \n",
       "5    MOESM6                             Supplementary  Data  3   \n",
       "6    MOESM7                             Supplementary  Data  4   \n",
       "7    MOESM8                             Supplementary  Data  5   \n",
       "8    MOESM9                             Supplementary  Data  6   \n",
       "9   MOESM10                             Supplementary  Data  7   \n",
       "10  MOESM11                             Supplementary  Data  8   \n",
       "11  MOESM12                             Supplementary  Data  9   \n",
       "12  MOESM13                            Supplementary  Data  10   \n",
       "13  MOESM14                            Supplementary  Data  11   \n",
       "14  MOESM15                            Supplementary  Data  12   \n",
       "\n",
       "                                      description file_extension  \\\n",
       "0                                             NaN           None   \n",
       "1                                             NaN           None   \n",
       "0                       Supplementary Information            pdf   \n",
       "1                                Peer Review File            pdf   \n",
       "2   Description of Additional Supplementary Files            pdf   \n",
       "3                            Supplementary Data 1           xlsx   \n",
       "4                            Supplementary Data 2           xlsx   \n",
       "5                            Supplementary Data 3           xlsx   \n",
       "6                            Supplementary Data 4           xlsx   \n",
       "7                            Supplementary Data 5           xlsx   \n",
       "8                            Supplementary Data 6           xlsx   \n",
       "9                            Supplementary Data 7           xlsx   \n",
       "10                           Supplementary Data 8           xlsx   \n",
       "11                           Supplementary Data 9           xlsx   \n",
       "12                          Supplementary Data 10           xlsx   \n",
       "13                          Supplementary Data 11           xlsx   \n",
       "14                          Supplementary Data 12           xlsx   \n",
       "\n",
       "                                        article_title  \n",
       "0   Recurrent WNT pathway alterations are frequent...  \n",
       "1   Recurrent WNT pathway alterations are frequent...  \n",
       "0   Recurrent WNT pathway alterations are frequent...  \n",
       "1   Recurrent WNT pathway alterations are frequent...  \n",
       "2   Recurrent WNT pathway alterations are frequent...  \n",
       "3   Recurrent WNT pathway alterations are frequent...  \n",
       "4   Recurrent WNT pathway alterations are frequent...  \n",
       "5   Recurrent WNT pathway alterations are frequent...  \n",
       "6   Recurrent WNT pathway alterations are frequent...  \n",
       "7   Recurrent WNT pathway alterations are frequent...  \n",
       "8   Recurrent WNT pathway alterations are frequent...  \n",
       "9   Recurrent WNT pathway alterations are frequent...  \n",
       "10  Recurrent WNT pathway alterations are frequent...  \n",
       "11  Recurrent WNT pathway alterations are frequent...  \n",
       "12  Recurrent WNT pathway alterations are frequent...  \n",
       "13  Recurrent WNT pathway alterations are frequent...  \n",
       "14  Recurrent WNT pathway alterations are frequent...  "
      ],
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>dataset_identifier</th>\n",
       "      <th>data_repository</th>\n",
       "      <th>dataset_webpage</th>\n",
       "      <th>source_section</th>\n",
       "      <th>retrieval_pattern</th>\n",
       "      <th>access_mode</th>\n",
       "      <th>link</th>\n",
       "      <th>source_url</th>\n",
       "      <th>download_link</th>\n",
       "      <th>title</th>\n",
       "      <th>content_type</th>\n",
       "      <th>id</th>\n",
       "      <th>surrounding_text</th>\n",
       "      <th>description</th>\n",
       "      <th>file_extension</th>\n",
       "      <th>article_title</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>PHS001049</td>\n",
       "      <td>dbGAP</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/projects/gap/cgi-...</td>\n",
       "      <td>data_availability</td>\n",
       "      <td>data availability</td>\n",
       "      <td>Application to access</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "      <td>Recurrent WNT pathway alterations are frequent...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>PRJNA306801</td>\n",
       "      <td>SRA</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/bioproject/?term=...</td>\n",
       "      <td>data_availability</td>\n",
       "      <td>data availability</td>\n",
       "      <td>Complex download</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "      <td>Recurrent WNT pathway alterations are frequent...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>supplementary material</td>\n",
       "      <td>.//supplementary-material</td>\n",
       "      <td>NaN</td>\n",
       "      <td>41467_2018_6162_MOESM1_ESM.pdf</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/instance...</td>\n",
       "      <td>No Title</td>\n",
       "      <td>local-data</td>\n",
       "      <td>MOESM1</td>\n",
       "      <td>Supplementary  Information</td>\n",
       "      <td>Supplementary Information</td>\n",
       "      <td>pdf</td>\n",
       "      <td>Recurrent WNT pathway alterations are frequent...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>supplementary material</td>\n",
       "      <td>.//supplementary-material</td>\n",
       "      <td>NaN</td>\n",
       "      <td>41467_2018_6162_MOESM2_ESM.pdf</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/instance...</td>\n",
       "      <td>No Title</td>\n",
       "      <td>local-data</td>\n",
       "      <td>MOESM2</td>\n",
       "      <td>Peer  Review  File</td>\n",
       "      <td>Peer Review File</td>\n",
       "      <td>pdf</td>\n",
       "      <td>Recurrent WNT pathway alterations are frequent...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>supplementary material</td>\n",
       "      <td>.//supplementary-material</td>\n",
       "      <td>NaN</td>\n",
       "      <td>41467_2018_6162_MOESM3_ESM.pdf</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/instance...</td>\n",
       "      <td>No Title</td>\n",
       "      <td>local-data</td>\n",
       "      <td>MOESM3</td>\n",
       "      <td>Description  of  Additional  Supplementary  Files</td>\n",
       "      <td>Description of Additional Supplementary Files</td>\n",
       "      <td>pdf</td>\n",
       "      <td>Recurrent WNT pathway alterations are frequent...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>supplementary material</td>\n",
       "      <td>.//supplementary-material</td>\n",
       "      <td>NaN</td>\n",
       "      <td>41467_2018_6162_MOESM4_ESM.xlsx</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/instance...</td>\n",
       "      <td>No Title</td>\n",
       "      <td>local-data</td>\n",
       "      <td>MOESM4</td>\n",
       "      <td>Supplementary  Data  1</td>\n",
       "      <td>Supplementary Data 1</td>\n",
       "      <td>xlsx</td>\n",
       "      <td>Recurrent WNT pathway alterations are frequent...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>supplementary material</td>\n",
       "      <td>.//supplementary-material</td>\n",
       "      <td>NaN</td>\n",
       "      <td>41467_2018_6162_MOESM5_ESM.xlsx</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/instance...</td>\n",
       "      <td>No Title</td>\n",
       "      <td>local-data</td>\n",
       "      <td>MOESM5</td>\n",
       "      <td>Supplementary  Data  2</td>\n",
       "      <td>Supplementary Data 2</td>\n",
       "      <td>xlsx</td>\n",
       "      <td>Recurrent WNT pathway alterations are frequent...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>supplementary material</td>\n",
       "      <td>.//supplementary-material</td>\n",
       "      <td>NaN</td>\n",
       "      <td>41467_2018_6162_MOESM6_ESM.xlsx</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/instance...</td>\n",
       "      <td>No Title</td>\n",
       "      <td>local-data</td>\n",
       "      <td>MOESM6</td>\n",
       "      <td>Supplementary  Data  3</td>\n",
       "      <td>Supplementary Data 3</td>\n",
       "      <td>xlsx</td>\n",
       "      <td>Recurrent WNT pathway alterations are frequent...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>supplementary material</td>\n",
       "      <td>.//supplementary-material</td>\n",
       "      <td>NaN</td>\n",
       "      <td>41467_2018_6162_MOESM7_ESM.xlsx</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/instance...</td>\n",
       "      <td>No Title</td>\n",
       "      <td>local-data</td>\n",
       "      <td>MOESM7</td>\n",
       "      <td>Supplementary  Data  4</td>\n",
       "      <td>Supplementary Data 4</td>\n",
       "      <td>xlsx</td>\n",
       "      <td>Recurrent WNT pathway alterations are frequent...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>supplementary material</td>\n",
       "      <td>.//supplementary-material</td>\n",
       "      <td>NaN</td>\n",
       "      <td>41467_2018_6162_MOESM8_ESM.xlsx</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/instance...</td>\n",
       "      <td>No Title</td>\n",
       "      <td>local-data</td>\n",
       "      <td>MOESM8</td>\n",
       "      <td>Supplementary  Data  5</td>\n",
       "      <td>Supplementary Data 5</td>\n",
       "      <td>xlsx</td>\n",
       "      <td>Recurrent WNT pathway alterations are frequent...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>supplementary material</td>\n",
       "      <td>.//supplementary-material</td>\n",
       "      <td>NaN</td>\n",
       "      <td>41467_2018_6162_MOESM9_ESM.xlsx</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/instance...</td>\n",
       "      <td>No Title</td>\n",
       "      <td>local-data</td>\n",
       "      <td>MOESM9</td>\n",
       "      <td>Supplementary  Data  6</td>\n",
       "      <td>Supplementary Data 6</td>\n",
       "      <td>xlsx</td>\n",
       "      <td>Recurrent WNT pathway alterations are frequent...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>supplementary material</td>\n",
       "      <td>.//supplementary-material</td>\n",
       "      <td>NaN</td>\n",
       "      <td>41467_2018_6162_MOESM10_ESM.xlsx</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/instance...</td>\n",
       "      <td>No Title</td>\n",
       "      <td>local-data</td>\n",
       "      <td>MOESM10</td>\n",
       "      <td>Supplementary  Data  7</td>\n",
       "      <td>Supplementary Data 7</td>\n",
       "      <td>xlsx</td>\n",
       "      <td>Recurrent WNT pathway alterations are frequent...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>supplementary material</td>\n",
       "      <td>.//supplementary-material</td>\n",
       "      <td>NaN</td>\n",
       "      <td>41467_2018_6162_MOESM11_ESM.xlsx</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/instance...</td>\n",
       "      <td>No Title</td>\n",
       "      <td>local-data</td>\n",
       "      <td>MOESM11</td>\n",
       "      <td>Supplementary  Data  8</td>\n",
       "      <td>Supplementary Data 8</td>\n",
       "      <td>xlsx</td>\n",
       "      <td>Recurrent WNT pathway alterations are frequent...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>supplementary material</td>\n",
       "      <td>.//supplementary-material</td>\n",
       "      <td>NaN</td>\n",
       "      <td>41467_2018_6162_MOESM12_ESM.xlsx</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/instance...</td>\n",
       "      <td>No Title</td>\n",
       "      <td>local-data</td>\n",
       "      <td>MOESM12</td>\n",
       "      <td>Supplementary  Data  9</td>\n",
       "      <td>Supplementary Data 9</td>\n",
       "      <td>xlsx</td>\n",
       "      <td>Recurrent WNT pathway alterations are frequent...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>supplementary material</td>\n",
       "      <td>.//supplementary-material</td>\n",
       "      <td>NaN</td>\n",
       "      <td>41467_2018_6162_MOESM13_ESM.xlsx</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/instance...</td>\n",
       "      <td>No Title</td>\n",
       "      <td>local-data</td>\n",
       "      <td>MOESM13</td>\n",
       "      <td>Supplementary  Data  10</td>\n",
       "      <td>Supplementary Data 10</td>\n",
       "      <td>xlsx</td>\n",
       "      <td>Recurrent WNT pathway alterations are frequent...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>supplementary material</td>\n",
       "      <td>.//supplementary-material</td>\n",
       "      <td>NaN</td>\n",
       "      <td>41467_2018_6162_MOESM14_ESM.xlsx</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/instance...</td>\n",
       "      <td>No Title</td>\n",
       "      <td>local-data</td>\n",
       "      <td>MOESM14</td>\n",
       "      <td>Supplementary  Data  11</td>\n",
       "      <td>Supplementary Data 11</td>\n",
       "      <td>xlsx</td>\n",
       "      <td>Recurrent WNT pathway alterations are frequent...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>supplementary material</td>\n",
       "      <td>.//supplementary-material</td>\n",
       "      <td>NaN</td>\n",
       "      <td>41467_2018_6162_MOESM15_ESM.xlsx</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6...</td>\n",
       "      <td>https://pmc.ncbi.nlm.nih.gov/articles/instance...</td>\n",
       "      <td>No Title</td>\n",
       "      <td>local-data</td>\n",
       "      <td>MOESM15</td>\n",
       "      <td>Supplementary  Data  12</td>\n",
       "      <td>Supplementary Data 12</td>\n",
       "      <td>xlsx</td>\n",
       "      <td>Recurrent WNT pathway alterations are frequent...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "execution_count": 11
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-07-07T15:30:34.363317Z",
     "start_time": "2025-07-07T15:30:32.461267Z"
    }
   },
   "cell_type": "code",
   "source": [
    "results_RTR = data_gatherer.parse_data(local_fetch_supported_input, raw_data_local, publisher=\"PMC\", use_portkey_for_gemini=True, prompt_name=\"retrieve_datasets_simple_JSON\", raw_data_format=\"HTML\")\n",
    "results_RTR"
   ],
   "id": "7749b302b27a2c37",
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "orchestrator.py - line 185 - INFO - Parsing data from URL: https://www.ncbi.nlm.nih.gov/pmc/articles/pmc11425778 with publisher: PMC\n",
      "base_parser.py - line 181 - INFO - LLMParser initialized.\n",
      "base_parser.py - line 913 - DEBUG - Repo mapping: {'bigd': 'bigd.big.ac.cn', 'european genome-phenome archive': 'ega-archive.org', 'massive': 'massive.ucsd.edu', 'pride': 'www.ebi.ac.uk', 'iprox': 'www.iprox.cn', 'proteomecentral': 'proteomecentral.proteomexchange.org', 'proteomexchange': 'www.proteomexchange.org', 'proteomic data commons': 'pdc.cancer.gov', 'genomic data commons': 'gdc.cancer.gov', 'the cancer imaging archive': 'cancerimagingarchive.net', 'imaging data commons': 'imaging.datacommons.cancer.gov', 'jpost': 'repository.jpostdb.org', 'doi.org': 'doi.org', 'geo': 'geo', 'dbgap': 'dbgap', 'sra': 'sra', 'arrayexpress': 'ArrayExpress'}\n",
      "html_parser.py - line 79 - INFO - Initializing htmlRetriever\n",
      "html_parser.py - line 342 - INFO - Function call: parse_data(api_data(<class 'str'>), PMC, https://www.ncbi.nlm.nih.gov/pmc/articles/pmc11425778, additional_data, HTML)\n",
      "html_parser.py - line 370 - INFO - Chunking the HTML content for the parsing step.\n",
      "html_parser.py - line 427 - INFO - Function_call: extract_href_from_html_supplementary_material(tree, https://www.ncbi.nlm.nih.gov/pmc/articles/pmc11425778)\n",
      "html_parser.py - line 435 - DEBUG - Found 121 anchors with data-ga-action='click_feat_suppl'.\n",
      "base_parser.py - line 302 - DEBUG - Function_call: reconstruct_download_link(https://doi.org/10.1021/acs.jproteome.4c00338, pmc-layout__citation font-secondary font-xs, https://www.ncbi.nlm.nih.gov/pmc/articles/pmc11425778)\n",
      "base_parser.py - line 305 - DEBUG - Inputs to reconstruct_download_link: https://doi.org/10.1021/acs.jproteome.4c00338, pmc-layout__citation font-secondary font-xs, https://www.ncbi.nlm.nih.gov/pmc/articles/pmc11425778, 11425778\n",
      "base_parser.py - line 302 - DEBUG - Function_call: reconstruct_download_link(https://doi.org/10.1021/acs.jproteome.4c00338, ra xbox p, https://www.ncbi.nlm.nih.gov/pmc/articles/pmc11425778)\n",
      "base_parser.py - line 305 - DEBUG - Inputs to reconstruct_download_link: https://doi.org/10.1021/acs.jproteome.4c00338, ra xbox p, https://www.ncbi.nlm.nih.gov/pmc/articles/pmc11425778, 11425778\n",
      "base_parser.py - line 302 - DEBUG - Function_call: reconstruct_download_link(https://www.bioinformatics.com.cn/en, n/a, https://www.ncbi.nlm.nih.gov/pmc/articles/pmc11425778)\n",
      "base_parser.py - line 305 - DEBUG - Inputs to reconstruct_download_link: https://www.bioinformatics.com.cn/en, n/a, https://www.ncbi.nlm.nih.gov/pmc/articles/pmc11425778, 11425778\n",
      "base_parser.py - line 302 - DEBUG - Function_call: reconstruct_download_link(https://string-db.org/, n/a, https://www.ncbi.nlm.nih.gov/pmc/articles/pmc11425778)\n",
      "base_parser.py - line 305 - DEBUG - Inputs to reconstruct_download_link: https://string-db.org/, n/a, https://www.ncbi.nlm.nih.gov/pmc/articles/pmc11425778, 11425778\n",
      "base_parser.py - line 302 - DEBUG - Function_call: reconstruct_download_link(https://www.genome.jp/kegg/pathway.html, n/a, https://www.ncbi.nlm.nih.gov/pmc/articles/pmc11425778)\n",
      "base_parser.py - line 305 - DEBUG - Inputs to reconstruct_download_link: https://www.genome.jp/kegg/pathway.html, n/a, https://www.ncbi.nlm.nih.gov/pmc/articles/pmc11425778, 11425778\n",
      "base_parser.py - line 302 - DEBUG - Function_call: reconstruct_download_link(https://molbiotools.com/listcompare.php, n/a, https://www.ncbi.nlm.nih.gov/pmc/articles/pmc11425778)\n",
      "base_parser.py - line 305 - DEBUG - Inputs to reconstruct_download_link: https://molbiotools.com/listcompare.php, n/a, https://www.ncbi.nlm.nih.gov/pmc/articles/pmc11425778, 11425778\n",
      "base_parser.py - line 302 - DEBUG - Function_call: reconstruct_download_link(https://www.biorender.com/, n/a, https://www.ncbi.nlm.nih.gov/pmc/articles/pmc11425778)\n",
      "base_parser.py - line 305 - DEBUG - Inputs to reconstruct_download_link: https://www.biorender.com/, n/a, https://www.ncbi.nlm.nih.gov/pmc/articles/pmc11425778, 11425778\n",
      "base_parser.py - line 302 - DEBUG - Function_call: reconstruct_download_link(https://www.graphpad.com/, n/a, https://www.ncbi.nlm.nih.gov/pmc/articles/pmc11425778)\n",
      "base_parser.py - line 305 - DEBUG - Inputs to reconstruct_download_link: https://www.graphpad.com/, n/a, https://www.ncbi.nlm.nih.gov/pmc/articles/pmc11425778, 11425778\n",
      "base_parser.py - line 302 - DEBUG - Function_call: reconstruct_download_link(/articles/instance/11425778/bin/NIHMS2020672-supplement-SI.pdf, media p, https://www.ncbi.nlm.nih.gov/pmc/articles/pmc11425778)\n",
      "base_parser.py - line 305 - DEBUG - Inputs to reconstruct_download_link: /articles/instance/11425778/bin/NIHMS2020672-supplement-SI.pdf, media p, https://www.ncbi.nlm.nih.gov/pmc/articles/pmc11425778, 11425778\n",
      "base_parser.py - line 311 - DEBUG - Extracted file name: NIHMS2020672-supplement-SI.pdf from href: /articles/instance/11425778/bin/NIHMS2020672-supplement-SI.pdf\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(https://www.ncbi.nlm.nih.gov/pmc/articles/instance/11425778/bin/NIHMS2020672-supplement-SI.pdf)\n",
      "base_parser.py - line 302 - DEBUG - Function_call: reconstruct_download_link(https://pubs.acs.org/doi/10.1021/acs.jproteome.4c00338?goto=supporting-info, fn p, https://www.ncbi.nlm.nih.gov/pmc/articles/pmc11425778)\n",
      "base_parser.py - line 305 - DEBUG - Inputs to reconstruct_download_link: https://pubs.acs.org/doi/10.1021/acs.jproteome.4c00338?goto=supporting-info, fn p, https://www.ncbi.nlm.nih.gov/pmc/articles/pmc11425778, 11425778\n",
      "base_parser.py - line 302 - DEBUG - Function_call: reconstruct_download_link(https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_001.pdf, fn p, https://www.ncbi.nlm.nih.gov/pmc/articles/pmc11425778)\n",
      "base_parser.py - line 305 - DEBUG - Inputs to reconstruct_download_link: https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_001.pdf, fn p, https://www.ncbi.nlm.nih.gov/pmc/articles/pmc11425778, 11425778\n",
      "base_parser.py - line 302 - DEBUG - Function_call: reconstruct_download_link(https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_002.xlsx, fn p, https://www.ncbi.nlm.nih.gov/pmc/articles/pmc11425778)\n",
      "base_parser.py - line 305 - DEBUG - Inputs to reconstruct_download_link: https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_002.xlsx, fn p, https://www.ncbi.nlm.nih.gov/pmc/articles/pmc11425778, 11425778\n",
      "base_parser.py - line 302 - DEBUG - Function_call: reconstruct_download_link(https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_003.xlsx, fn p, https://www.ncbi.nlm.nih.gov/pmc/articles/pmc11425778)\n",
      "base_parser.py - line 305 - DEBUG - Inputs to reconstruct_download_link: https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_003.xlsx, fn p, https://www.ncbi.nlm.nih.gov/pmc/articles/pmc11425778, 11425778\n",
      "base_parser.py - line 302 - DEBUG - Function_call: reconstruct_download_link(https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_004.xlsx, fn p, https://www.ncbi.nlm.nih.gov/pmc/articles/pmc11425778)\n",
      "base_parser.py - line 305 - DEBUG - Inputs to reconstruct_download_link: https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_004.xlsx, fn p, https://www.ncbi.nlm.nih.gov/pmc/articles/pmc11425778, 11425778\n",
      "base_parser.py - line 302 - DEBUG - Function_call: reconstruct_download_link(https://pubs.acs.org/doi/10.1021/acs.jproteome.4c00338?ref=pdf, fn p, https://www.ncbi.nlm.nih.gov/pmc/articles/pmc11425778)\n",
      "base_parser.py - line 305 - DEBUG - Inputs to reconstruct_download_link: https://pubs.acs.org/doi/10.1021/acs.jproteome.4c00338?ref=pdf, fn p, https://www.ncbi.nlm.nih.gov/pmc/articles/pmc11425778, 11425778\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "html_parser.py - line 468 - DEBUG - Skipping link with section_class 'ref-list font-sm', likely to be a reference list.\n",
      "base_parser.py - line 302 - DEBUG - Function_call: reconstruct_download_link(/articles/instance/11425778/bin/NIHMS2020672-supplement-SI.pdf, media p, https://www.ncbi.nlm.nih.gov/pmc/articles/pmc11425778)\n",
      "base_parser.py - line 305 - DEBUG - Inputs to reconstruct_download_link: /articles/instance/11425778/bin/NIHMS2020672-supplement-SI.pdf, media p, https://www.ncbi.nlm.nih.gov/pmc/articles/pmc11425778, 11425778\n",
      "base_parser.py - line 311 - DEBUG - Extracted file name: NIHMS2020672-supplement-SI.pdf from href: /articles/instance/11425778/bin/NIHMS2020672-supplement-SI.pdf\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(https://www.ncbi.nlm.nih.gov/pmc/articles/instance/11425778/bin/NIHMS2020672-supplement-SI.pdf)\n",
      "html_parser.py - line 488 - INFO - Extracted 14 unique supplementary material links from HTML.\n",
      "html_retriever.py - line 171 - INFO - Extracting data availability elements from HTML\n",
      "html_retriever.py - line 182 - INFO - Using selector: section.data-availability-statement\n",
      "html_retriever.py - line 218 - DEBUG - Extracted data availability element: {'retrieval_pattern': 'section.data-availability-statement', 'text': 'Proteomics data can be found at the MassIVE database via MSV000092944. \\n', 'html': '<p> Proteomics data can be found at the MassIVE database via MSV000092944. </p> \\n'}\n",
      "html_retriever.py - line 182 - INFO - Using selector: section.associated-data\n",
      "html_retriever.py - line 218 - DEBUG - Extracted data availability element: {'retrieval_pattern': 'section.associated-data', 'text': 'This section collects any data citations, data availability statements, or supplementary materials included in this article. \\nSI \\nNIHMS2020672-supplement-SI.pdf(1.4MB, pdf) \\nNIHMS2020672-supplement-SI.pdf(1.4MB, pdf) \\nNIHMS2020672-supplement-SI.pdf \\nProteomics data can be found at the MassIVE database via MSV000092944. \\n', 'html': '<p class=\"font-secondary\"> <em> This section collects any data citations, data availability statements, or supplementary materials included in this article. </em> </p> \\n<div class=\"caption p\"> <span> SI </span> </div> \\n<div class=\"media p\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/11425778/bin/NIHMS2020672-supplement-SI.pdf\"> NIHMS2020672-supplement-SI.pdf </a> <sup> (1.4MB, pdf) </sup> </div> </div> \\n<div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/11425778/bin/NIHMS2020672-supplement-SI.pdf\"> NIHMS2020672-supplement-SI.pdf </a> <sup> (1.4MB, pdf) </sup> </div> \\n<a class=\"usa-link\" href=\"/articles/instance/11425778/bin/NIHMS2020672-supplement-SI.pdf\"> NIHMS2020672-supplement-SI.pdf </a> \\n<p> Proteomics data can be found at the MassIVE database via MSV000092944. </p> \\n'}\n",
      "html_retriever.py - line 182 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Data Availability')\n",
      "html_retriever.py - line 218 - DEBUG - Extracted data availability element: {'retrieval_pattern': \"h2.pmc_sec_title:-soup-contains('Data Availability')\", 'text': 'Proteomics data can be found at the MassIVE database via MSV000092944. \\n', 'html': '<p id=\"P37\"> Proteomics data can be found at the MassIVE database via MSV000092944. </p> \\n'}\n",
      "html_retriever.py - line 182 - INFO - Using selector: h3.pmc_sec_title:-soup-contains('Data Availability')\n",
      "html_retriever.py - line 218 - DEBUG - Extracted data availability element: {'retrieval_pattern': \"h3.pmc_sec_title:-soup-contains('Data Availability')\", 'text': 'Proteomics data can be found at the MassIVE database via MSV000092944. \\n', 'html': '<p> Proteomics data can be found at the MassIVE database via MSV000092944. </p> \\n'}\n",
      "html_retriever.py - line 182 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Associated Data')\n",
      "html_retriever.py - line 218 - DEBUG - Extracted data availability element: {'retrieval_pattern': \"h2.pmc_sec_title:-soup-contains('Associated Data')\", 'text': 'This section collects any data citations, data availability statements, or supplementary materials included in this article. \\nSI \\nNIHMS2020672-supplement-SI.pdf(1.4MB, pdf) \\nNIHMS2020672-supplement-SI.pdf(1.4MB, pdf) \\nNIHMS2020672-supplement-SI.pdf \\nProteomics data can be found at the MassIVE database via MSV000092944. \\n', 'html': '<p class=\"font-secondary\"> <em> This section collects any data citations, data availability statements, or supplementary materials included in this article. </em> </p> \\n<div class=\"caption p\"> <span> SI </span> </div> \\n<div class=\"media p\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/11425778/bin/NIHMS2020672-supplement-SI.pdf\"> NIHMS2020672-supplement-SI.pdf </a> <sup> (1.4MB, pdf) </sup> </div> </div> \\n<div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/11425778/bin/NIHMS2020672-supplement-SI.pdf\"> NIHMS2020672-supplement-SI.pdf </a> <sup> (1.4MB, pdf) </sup> </div> \\n<a class=\"usa-link\" href=\"/articles/instance/11425778/bin/NIHMS2020672-supplement-SI.pdf\"> NIHMS2020672-supplement-SI.pdf </a> \\n<p> Proteomics data can be found at the MassIVE database via MSV000092944. </p> \\n'}\n",
      "html_retriever.py - line 182 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Footnotes')\n",
      "html_retriever.py - line 218 - DEBUG - Extracted data availability element: {'retrieval_pattern': \"h2.pmc_sec_title:-soup-contains('Footnotes')\", 'text': 'Supporting InformationThe Supporting Information is available free of charge athttps://pubs.acs.org/doi/10.1021/acs.jproteome.4c00338.Figure S1, quality control of label-free quantitative proteomics;Figure S2, comparison between the monoculture drug-resistant and drug-sensitive cells;Figure S3, P53 signaling pathway and the highlighted proteins upregulated in drug-resistant cells;Figure S4, workflow of the direct coculture and indirect coculture; andFigure S5, upregulated HIF-1 signaling pathways and TCA cycle in indirect coculture cells (PDF)All identified proteins and upregulated proteins in P53 pathways in monoculture and coculture cells (XLSX)Volcano plot of proteins in monoculture and coculture cells (XLSX)Enriched upregulated KEGG pathways in monoculture and coculture systems (XLSX)The authors declare no competing financial interest.Complete contact information is available at:https://pubs.acs.org/10.1021/acs.jproteome.4c00338 \\nSupporting InformationThe Supporting Information is available free of charge athttps://pubs.acs.org/doi/10.1021/acs.jproteome.4c00338.Figure S1, quality control of label-free quantitative proteomics;Figure S2, comparison between the monoculture drug-resistant and drug-sensitive cells;Figure S3, P53 signaling pathway and the highlighted proteins upregulated in drug-resistant cells;Figure S4, workflow of the direct coculture and indirect coculture; andFigure S5, upregulated HIF-1 signaling pathways and TCA cycle in indirect coculture cells (PDF)All identified proteins and upregulated proteins in P53 pathways in monoculture and coculture cells (XLSX)Volcano plot of proteins in monoculture and coculture cells (XLSX)Enriched upregulated KEGG pathways in monoculture and coculture systems (XLSX) \\nSupporting Information \\nThe Supporting Information is available free of charge athttps://pubs.acs.org/doi/10.1021/acs.jproteome.4c00338. \\nhttps://pubs.acs.org/doi/10.1021/acs.jproteome.4c00338 \\nFigure S1, quality control of label-free quantitative proteomics;Figure S2, comparison between the monoculture drug-resistant and drug-sensitive cells;Figure S3, P53 signaling pathway and the highlighted proteins upregulated in drug-resistant cells;Figure S4, workflow of the direct coculture and indirect coculture; andFigure S5, upregulated HIF-1 signaling pathways and TCA cycle in indirect coculture cells (PDF) \\nFigure S1 \\nFigure S2 \\nFigure S3 \\nFigure S4 \\nFigure S5 \\nPDF \\nAll identified proteins and upregulated proteins in P53 pathways in monoculture and coculture cells (XLSX) \\nXLSX \\nVolcano plot of proteins in monoculture and coculture cells (XLSX) \\nXLSX \\nEnriched upregulated KEGG pathways in monoculture and coculture systems (XLSX) \\nXLSX \\nThe authors declare no competing financial interest. \\nThe authors declare no competing financial interest. \\nComplete contact information is available at:https://pubs.acs.org/10.1021/acs.jproteome.4c00338 \\nComplete contact information is available at:https://pubs.acs.org/10.1021/acs.jproteome.4c00338 \\nhttps://pubs.acs.org/10.1021/acs.jproteome.4c00338 \\n', 'html': '<div class=\"fn-group p font-secondary-light font-sm\"> <div class=\"fn p\" id=\"FN1\"> <p id=\"P39\"> Supporting Information </p> <p id=\"P40\"> The Supporting Information is available free of charge at <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/10.1021/acs.jproteome.4c00338?goto=supporting-info\" rel=\"noopener noreferrer\" target=\"_blank\"> https://pubs.acs.org/doi/10.1021/acs.jproteome.4c00338 </a> . </p> <p id=\"P41\"> <a class=\"usa-link\" href=\"#SD1\"> Figure S1 </a> , quality control of label-free quantitative proteomics; <a class=\"usa-link\" href=\"#SD1\"> Figure S2 </a> , comparison between the monoculture drug-resistant and drug-sensitive cells; <a class=\"usa-link\" href=\"#SD1\"> Figure S3 </a> , P53 signaling pathway and the highlighted proteins upregulated in drug-resistant cells; <a class=\"usa-link\" href=\"#SD1\"> Figure S4 </a> , workflow of the direct coculture and indirect coculture; and <a class=\"usa-link\" href=\"#SD1\"> Figure S5 </a> , upregulated HIF-1 signaling pathways and TCA cycle in indirect coculture cells ( <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_001.pdf\" rel=\"noopener noreferrer\" target=\"_blank\"> PDF </a> ) </p> <p id=\"P42\"> All identified proteins and upregulated proteins in P53 pathways in monoculture and coculture cells ( <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_002.xlsx\" rel=\"noopener noreferrer\" target=\"_blank\"> XLSX </a> ) </p> <p id=\"P420\"> Volcano plot of proteins in monoculture and coculture cells ( <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_003.xlsx\" rel=\"noopener noreferrer\" target=\"_blank\"> XLSX </a> ) </p> <p id=\"P43\"> Enriched upregulated KEGG pathways in monoculture and coculture systems ( <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_004.xlsx\" rel=\"noopener noreferrer\" target=\"_blank\"> XLSX </a> ) </p> </div> <div class=\"fn p\" id=\"FN2\"> <p id=\"P44\"> The authors declare no competing financial interest. </p> </div> <div class=\"fn p\" id=\"FN3\"> <p id=\"P45\"> Complete contact information is available at: <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/10.1021/acs.jproteome.4c00338?ref=pdf\" rel=\"noopener noreferrer\" target=\"_blank\"> https://pubs.acs.org/10.1021/acs.jproteome.4c00338 </a> </p> </div> </div> \\n<div class=\"fn p\" id=\"FN1\"> <p id=\"P39\"> Supporting Information </p> <p id=\"P40\"> The Supporting Information is available free of charge at <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/10.1021/acs.jproteome.4c00338?goto=supporting-info\" rel=\"noopener noreferrer\" target=\"_blank\"> https://pubs.acs.org/doi/10.1021/acs.jproteome.4c00338 </a> . </p> <p id=\"P41\"> <a class=\"usa-link\" href=\"#SD1\"> Figure S1 </a> , quality control of label-free quantitative proteomics; <a class=\"usa-link\" href=\"#SD1\"> Figure S2 </a> , comparison between the monoculture drug-resistant and drug-sensitive cells; <a class=\"usa-link\" href=\"#SD1\"> Figure S3 </a> , P53 signaling pathway and the highlighted proteins upregulated in drug-resistant cells; <a class=\"usa-link\" href=\"#SD1\"> Figure S4 </a> , workflow of the direct coculture and indirect coculture; and <a class=\"usa-link\" href=\"#SD1\"> Figure S5 </a> , upregulated HIF-1 signaling pathways and TCA cycle in indirect coculture cells ( <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_001.pdf\" rel=\"noopener noreferrer\" target=\"_blank\"> PDF </a> ) </p> <p id=\"P42\"> All identified proteins and upregulated proteins in P53 pathways in monoculture and coculture cells ( <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_002.xlsx\" rel=\"noopener noreferrer\" target=\"_blank\"> XLSX </a> ) </p> <p id=\"P420\"> Volcano plot of proteins in monoculture and coculture cells ( <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_003.xlsx\" rel=\"noopener noreferrer\" target=\"_blank\"> XLSX </a> ) </p> <p id=\"P43\"> Enriched upregulated KEGG pathways in monoculture and coculture systems ( <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_004.xlsx\" rel=\"noopener noreferrer\" target=\"_blank\"> XLSX </a> ) </p> </div> \\n<p id=\"P39\"> Supporting Information </p> \\n<p id=\"P40\"> The Supporting Information is available free of charge at <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/10.1021/acs.jproteome.4c00338?goto=supporting-info\" rel=\"noopener noreferrer\" target=\"_blank\"> https://pubs.acs.org/doi/10.1021/acs.jproteome.4c00338 </a> . </p> \\n<a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/10.1021/acs.jproteome.4c00338?goto=supporting-info\" rel=\"noopener noreferrer\" target=\"_blank\"> https://pubs.acs.org/doi/10.1021/acs.jproteome.4c00338 </a> \\n<p id=\"P41\"> <a class=\"usa-link\" href=\"#SD1\"> Figure S1 </a> , quality control of label-free quantitative proteomics; <a class=\"usa-link\" href=\"#SD1\"> Figure S2 </a> , comparison between the monoculture drug-resistant and drug-sensitive cells; <a class=\"usa-link\" href=\"#SD1\"> Figure S3 </a> , P53 signaling pathway and the highlighted proteins upregulated in drug-resistant cells; <a class=\"usa-link\" href=\"#SD1\"> Figure S4 </a> , workflow of the direct coculture and indirect coculture; and <a class=\"usa-link\" href=\"#SD1\"> Figure S5 </a> , upregulated HIF-1 signaling pathways and TCA cycle in indirect coculture cells ( <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_001.pdf\" rel=\"noopener noreferrer\" target=\"_blank\"> PDF </a> ) </p> \\n<a class=\"usa-link\" href=\"#SD1\"> Figure S1 </a> \\n<a class=\"usa-link\" href=\"#SD1\"> Figure S2 </a> \\n<a class=\"usa-link\" href=\"#SD1\"> Figure S3 </a> \\n<a class=\"usa-link\" href=\"#SD1\"> Figure S4 </a> \\n<a class=\"usa-link\" href=\"#SD1\"> Figure S5 </a> \\n<a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_001.pdf\" rel=\"noopener noreferrer\" target=\"_blank\"> PDF </a> \\n<p id=\"P42\"> All identified proteins and upregulated proteins in P53 pathways in monoculture and coculture cells ( <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_002.xlsx\" rel=\"noopener noreferrer\" target=\"_blank\"> XLSX </a> ) </p> \\n<a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_002.xlsx\" rel=\"noopener noreferrer\" target=\"_blank\"> XLSX </a> \\n<p id=\"P420\"> Volcano plot of proteins in monoculture and coculture cells ( <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_003.xlsx\" rel=\"noopener noreferrer\" target=\"_blank\"> XLSX </a> ) </p> \\n<a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_003.xlsx\" rel=\"noopener noreferrer\" target=\"_blank\"> XLSX </a> \\n<p id=\"P43\"> Enriched upregulated KEGG pathways in monoculture and coculture systems ( <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_004.xlsx\" rel=\"noopener noreferrer\" target=\"_blank\"> XLSX </a> ) </p> \\n<a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_004.xlsx\" rel=\"noopener noreferrer\" target=\"_blank\"> XLSX </a> \\n<div class=\"fn p\" id=\"FN2\"> <p id=\"P44\"> The authors declare no competing financial interest. </p> </div> \\n<p id=\"P44\"> The authors declare no competing financial interest. </p> \\n<div class=\"fn p\" id=\"FN3\"> <p id=\"P45\"> Complete contact information is available at: <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/10.1021/acs.jproteome.4c00338?ref=pdf\" rel=\"noopener noreferrer\" target=\"_blank\"> https://pubs.acs.org/10.1021/acs.jproteome.4c00338 </a> </p> </div> \\n<p id=\"P45\"> Complete contact information is available at: <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/10.1021/acs.jproteome.4c00338?ref=pdf\" rel=\"noopener noreferrer\" target=\"_blank\"> https://pubs.acs.org/10.1021/acs.jproteome.4c00338 </a> </p> \\n<a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/10.1021/acs.jproteome.4c00338?ref=pdf\" rel=\"noopener noreferrer\" target=\"_blank\"> https://pubs.acs.org/10.1021/acs.jproteome.4c00338 </a> \\n'}\n",
      "html_retriever.py - line 220 - INFO - Found 6 data availability elements from HTML.\n",
      "base_parser.py - line 426 - INFO - Loading prompt: retrieve_datasets_simple_JSON for model gemini-2.0-flash\n",
      "prompt_manager.py - line 32 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \n",
      "base_parser.py - line 1108 - INFO - Expected string but got list. Converting list to string.\n",
      "base_parser.py - line 1115 - DEBUG - Counting tokens for model: gemini-2.0-flash, prompt length: 1382 char\n",
      "base_parser.py - line 1124 - DEBUG - Rough estimate of token count for Gemini model 'gemini-2.0-flash': 345\n",
      "base_parser.py - line 433 - INFO - Content length: 10807\n",
      "base_parser.py - line 435 - DEBUG - static_prompt: [{'role': 'system', 'content': \"You are a specialized assistant that extracts dataset references from the content of scientific papers. You must output a JSON array of objects, where each object has the following keys: 'dataset_identifier', 'data_repository', and 'dataset_webpage'. Follow the structure of the provided examples exactly. We will not wrap the json codes in JSON markers.\"}, {'role': 'user', 'content': 'Extract dataset references based on the examples below:\\n\\nExample 1:\\nContent: \\'The study used dataset EGAS00893672193, which is available at the European Genome Archive.\\'\\nResponse:\\n[\\n    [\\n        \"dataset_identifier\": \"EGAS00893672193\",\\n        \"data_repository\": \"European Genome Archive\",\\n        \"dataset_webpage\": \"https://ega-archive.org/studies/EGAS00893672193\"\\n    ]\\n]\\n\\nExample 2:\\nContent: \\'Proteomics data was obtained from PRIDE, accession PXD029921.\\'\\nResponse:\\n[\\n    [\\n        \"dataset_identifier\": \"PXD029921\",\\n        \"data_repository\": \"PRIDE\",\\n        \"dataset_webpage\": \"https://www.ebi.ac.uk/pride/archive/projects/PXD029921\"\\n    ]\\n]\\n\\nExample 3:\\nContent: \\'The repository dbGaP hosts the dataset phs011366.v1.p1 at this location.\\'\\nResponse:\\n[\\n    [\\n        \"dataset_identifier\": \"phs011366.v1.p1\",\\n        \"data_repository\": \"dbGaP\",\\n        \"dataset_webpage\": \"https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001366.v1.p1\"\\n    ]\\n]\\n\\nNow process the following content:\\n\\nContent: {content}.'}]\n",
      "prompt_manager.py - line 56 - DEBUG - Rendering prompt with dynamic parts(<class 'dict'>): {'content': '<p> Proteomics data can be found at the MassIVE database via MSV000092944. </p> \\n\\n\\n<p class=\"font-secondary\"> <em> This section collects any data citations, data availability statements, or supplementary materials included in this article. </em> </p> \\n<div class=\"caption p\"> <span> SI </span> </div> \\n<div class=\"media p\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/11425778/bin/NIHMS2020672-supplement-SI.pdf\"> NIHMS2020672-supplement-SI.pdf </a> <sup> (1.4MB, pdf) </sup> </div> </div> \\n<div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/11425778/bin/NIHMS2020672-supplement-SI.pdf\"> NIHMS2020672-supplement-SI.pdf </a> <sup> (1.4MB, pdf) </sup> </div> \\n<a class=\"usa-link\" href=\"/articles/instance/11425778/bin/NIHMS2020672-supplement-SI.pdf\"> NIHMS2020672-supplement-SI.pdf </a> \\n<p> Proteomics data can be found at the MassIVE database via MSV000092944. </p> \\n\\n\\n<p id=\"P37\"> Proteomics data can be found at the MassIVE database via MSV000092944. </p> \\n\\n\\n<p> Proteomics data can be found at the MassIVE database via MSV000092944. </p> \\n\\n\\n<p class=\"font-secondary\"> <em> This section collects any data citations, data availability statements, or supplementary materials included in this article. </em> </p> \\n<div class=\"caption p\"> <span> SI </span> </div> \\n<div class=\"media p\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/11425778/bin/NIHMS2020672-supplement-SI.pdf\"> NIHMS2020672-supplement-SI.pdf </a> <sup> (1.4MB, pdf) </sup> </div> </div> \\n<div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/11425778/bin/NIHMS2020672-supplement-SI.pdf\"> NIHMS2020672-supplement-SI.pdf </a> <sup> (1.4MB, pdf) </sup> </div> \\n<a class=\"usa-link\" href=\"/articles/instance/11425778/bin/NIHMS2020672-supplement-SI.pdf\"> NIHMS2020672-supplement-SI.pdf </a> \\n<p> Proteomics data can be found at the MassIVE database via MSV000092944. </p> \\n\\n\\n<div class=\"fn-group p font-secondary-light font-sm\"> <div class=\"fn p\" id=\"FN1\"> <p id=\"P39\"> Supporting Information </p> <p id=\"P40\"> The Supporting Information is available free of charge at <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/10.1021/acs.jproteome.4c00338?goto=supporting-info\" rel=\"noopener noreferrer\" target=\"_blank\"> https://pubs.acs.org/doi/10.1021/acs.jproteome.4c00338 </a> . </p> <p id=\"P41\"> <a class=\"usa-link\" href=\"#SD1\"> Figure S1 </a> , quality control of label-free quantitative proteomics; <a class=\"usa-link\" href=\"#SD1\"> Figure S2 </a> , comparison between the monoculture drug-resistant and drug-sensitive cells; <a class=\"usa-link\" href=\"#SD1\"> Figure S3 </a> , P53 signaling pathway and the highlighted proteins upregulated in drug-resistant cells; <a class=\"usa-link\" href=\"#SD1\"> Figure S4 </a> , workflow of the direct coculture and indirect coculture; and <a class=\"usa-link\" href=\"#SD1\"> Figure S5 </a> , upregulated HIF-1 signaling pathways and TCA cycle in indirect coculture cells ( <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_001.pdf\" rel=\"noopener noreferrer\" target=\"_blank\"> PDF </a> ) </p> <p id=\"P42\"> All identified proteins and upregulated proteins in P53 pathways in monoculture and coculture cells ( <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_002.xlsx\" rel=\"noopener noreferrer\" target=\"_blank\"> XLSX </a> ) </p> <p id=\"P420\"> Volcano plot of proteins in monoculture and coculture cells ( <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_003.xlsx\" rel=\"noopener noreferrer\" target=\"_blank\"> XLSX </a> ) </p> <p id=\"P43\"> Enriched upregulated KEGG pathways in monoculture and coculture systems ( <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_004.xlsx\" rel=\"noopener noreferrer\" target=\"_blank\"> XLSX </a> ) </p> </div> <div class=\"fn p\" id=\"FN2\"> <p id=\"P44\"> The authors declare no competing financial interest. </p> </div> <div class=\"fn p\" id=\"FN3\"> <p id=\"P45\"> Complete contact information is available at: <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/10.1021/acs.jproteome.4c00338?ref=pdf\" rel=\"noopener noreferrer\" target=\"_blank\"> https://pubs.acs.org/10.1021/acs.jproteome.4c00338 </a> </p> </div> </div> \\n<div class=\"fn p\" id=\"FN1\"> <p id=\"P39\"> Supporting Information </p> <p id=\"P40\"> The Supporting Information is available free of charge at <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/10.1021/acs.jproteome.4c00338?goto=supporting-info\" rel=\"noopener noreferrer\" target=\"_blank\"> https://pubs.acs.org/doi/10.1021/acs.jproteome.4c00338 </a> . </p> <p id=\"P41\"> <a class=\"usa-link\" href=\"#SD1\"> Figure S1 </a> , quality control of label-free quantitative proteomics; <a class=\"usa-link\" href=\"#SD1\"> Figure S2 </a> , comparison between the monoculture drug-resistant and drug-sensitive cells; <a class=\"usa-link\" href=\"#SD1\"> Figure S3 </a> , P53 signaling pathway and the highlighted proteins upregulated in drug-resistant cells; <a class=\"usa-link\" href=\"#SD1\"> Figure S4 </a> , workflow of the direct coculture and indirect coculture; and <a class=\"usa-link\" href=\"#SD1\"> Figure S5 </a> , upregulated HIF-1 signaling pathways and TCA cycle in indirect coculture cells ( <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_001.pdf\" rel=\"noopener noreferrer\" target=\"_blank\"> PDF </a> ) </p> <p id=\"P42\"> All identified proteins and upregulated proteins in P53 pathways in monoculture and coculture cells ( <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_002.xlsx\" rel=\"noopener noreferrer\" target=\"_blank\"> XLSX </a> ) </p> <p id=\"P420\"> Volcano plot of proteins in monoculture and coculture cells ( <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_003.xlsx\" rel=\"noopener noreferrer\" target=\"_blank\"> XLSX </a> ) </p> <p id=\"P43\"> Enriched upregulated KEGG pathways in monoculture and coculture systems ( <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_004.xlsx\" rel=\"noopener noreferrer\" target=\"_blank\"> XLSX </a> ) </p> </div> \\n<p id=\"P39\"> Supporting Information </p> \\n<p id=\"P40\"> The Supporting Information is available free of charge at <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/10.1021/acs.jproteome.4c00338?goto=supporting-info\" rel=\"noopener noreferrer\" target=\"_blank\"> https://pubs.acs.org/doi/10.1021/acs.jproteome.4c00338 </a> . </p> \\n<a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/10.1021/acs.jproteome.4c00338?goto=supporting-info\" rel=\"noopener noreferrer\" target=\"_blank\"> https://pubs.acs.org/doi/10.1021/acs.jproteome.4c00338 </a> \\n<p id=\"P41\"> <a class=\"usa-link\" href=\"#SD1\"> Figure S1 </a> , quality control of label-free quantitative proteomics; <a class=\"usa-link\" href=\"#SD1\"> Figure S2 </a> , comparison between the monoculture drug-resistant and drug-sensitive cells; <a class=\"usa-link\" href=\"#SD1\"> Figure S3 </a> , P53 signaling pathway and the highlighted proteins upregulated in drug-resistant cells; <a class=\"usa-link\" href=\"#SD1\"> Figure S4 </a> , workflow of the direct coculture and indirect coculture; and <a class=\"usa-link\" href=\"#SD1\"> Figure S5 </a> , upregulated HIF-1 signaling pathways and TCA cycle in indirect coculture cells ( <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_001.pdf\" rel=\"noopener noreferrer\" target=\"_blank\"> PDF </a> ) </p> \\n<a class=\"usa-link\" href=\"#SD1\"> Figure S1 </a> \\n<a class=\"usa-link\" href=\"#SD1\"> Figure S2 </a> \\n<a class=\"usa-link\" href=\"#SD1\"> Figure S3 </a> \\n<a class=\"usa-link\" href=\"#SD1\"> Figure S4 </a> \\n<a class=\"usa-link\" href=\"#SD1\"> Figure S5 </a> \\n<a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_001.pdf\" rel=\"noopener noreferrer\" target=\"_blank\"> PDF </a> \\n<p id=\"P42\"> All identified proteins and upregulated proteins in P53 pathways in monoculture and coculture cells ( <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_002.xlsx\" rel=\"noopener noreferrer\" target=\"_blank\"> XLSX </a> ) </p> \\n<a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_002.xlsx\" rel=\"noopener noreferrer\" target=\"_blank\"> XLSX </a> \\n<p id=\"P420\"> Volcano plot of proteins in monoculture and coculture cells ( <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_003.xlsx\" rel=\"noopener noreferrer\" target=\"_blank\"> XLSX </a> ) </p> \\n<a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_003.xlsx\" rel=\"noopener noreferrer\" target=\"_blank\"> XLSX </a> \\n<p id=\"P43\"> Enriched upregulated KEGG pathways in monoculture and coculture systems ( <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_004.xlsx\" rel=\"noopener noreferrer\" target=\"_blank\"> XLSX </a> ) </p> \\n<a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_004.xlsx\" rel=\"noopener noreferrer\" target=\"_blank\"> XLSX </a> \\n<div class=\"fn p\" id=\"FN2\"> <p id=\"P44\"> The authors declare no competing financial interest. </p> </div> \\n<p id=\"P44\"> The authors declare no competing financial interest. </p> \\n<div class=\"fn p\" id=\"FN3\"> <p id=\"P45\"> Complete contact information is available at: <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/10.1021/acs.jproteome.4c00338?ref=pdf\" rel=\"noopener noreferrer\" target=\"_blank\"> https://pubs.acs.org/10.1021/acs.jproteome.4c00338 </a> </p> </div> \\n<p id=\"P45\"> Complete contact information is available at: <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/10.1021/acs.jproteome.4c00338?ref=pdf\" rel=\"noopener noreferrer\" target=\"_blank\"> https://pubs.acs.org/10.1021/acs.jproteome.4c00338 </a> </p> \\n<a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/10.1021/acs.jproteome.4c00338?ref=pdf\" rel=\"noopener noreferrer\" target=\"_blank\"> https://pubs.acs.org/10.1021/acs.jproteome.4c00338 </a> \\n\\n', 'repos': 'bigd.big.ac.cn, ega-archive.org, massive.ucsd.edu, www.ebi.ac.uk, www.iprox.cn, proteomecentral.proteomexchange.org, www.proteomexchange.org, pdc.cancer.gov, gdc.cancer.gov, cancerimagingarchive.net, imaging.datacommons.cancer.gov, repository.jpostdb.org, doi.org, geo, dbgap, sra, ArrayExpress'}, and items: [{'role': 'system', 'content': \"You are a specialized assistant that extracts dataset references from the content of scientific papers. You must output a JSON array of objects, where each object has the following keys: 'dataset_identifier', 'data_repository', and 'dataset_webpage'. Follow the structure of the provided examples exactly. We will not wrap the json codes in JSON markers.\"}, {'role': 'user', 'content': 'Extract dataset references based on the examples below:\\n\\nExample 1:\\nContent: \\'The study used dataset EGAS00893672193, which is available at the European Genome Archive.\\'\\nResponse:\\n[\\n    [\\n        \"dataset_identifier\": \"EGAS00893672193\",\\n        \"data_repository\": \"European Genome Archive\",\\n        \"dataset_webpage\": \"https://ega-archive.org/studies/EGAS00893672193\"\\n    ]\\n]\\n\\nExample 2:\\nContent: \\'Proteomics data was obtained from PRIDE, accession PXD029921.\\'\\nResponse:\\n[\\n    [\\n        \"dataset_identifier\": \"PXD029921\",\\n        \"data_repository\": \"PRIDE\",\\n        \"dataset_webpage\": \"https://www.ebi.ac.uk/pride/archive/projects/PXD029921\"\\n    ]\\n]\\n\\nExample 3:\\nContent: \\'The repository dbGaP hosts the dataset phs011366.v1.p1 at this location.\\'\\nResponse:\\n[\\n    [\\n        \"dataset_identifier\": \"phs011366.v1.p1\",\\n        \"data_repository\": \"dbGaP\",\\n        \"dataset_webpage\": \"https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001366.v1.p1\"\\n    ]\\n]\\n\\nNow process the following content:\\n\\nContent: {content}.'}]\n",
      "base_parser.py - line 1108 - INFO - Expected string but got list. Converting list to string.\n",
      "base_parser.py - line 1115 - DEBUG - Counting tokens for model: gemini-2.0-flash, prompt length: 12180 char\n",
      "base_parser.py - line 1124 - DEBUG - Rough estimate of token count for Gemini model 'gemini-2.0-flash': 3045\n",
      "base_parser.py - line 444 - INFO - Prompt messages total length: 3045 tokens\n",
      "base_parser.py - line 445 - DEBUG - Prompt messages: [{'role': 'system', 'content': \"You are a specialized assistant that extracts dataset references from the content of scientific papers. You must output a JSON array of objects, where each object has the following keys: 'dataset_identifier', 'data_repository', and 'dataset_webpage'. Follow the structure of the provided examples exactly. We will not wrap the json codes in JSON markers.\"}, {'role': 'user', 'content': 'Extract dataset references based on the examples below:\\n\\nExample 1:\\nContent: \\'The study used dataset EGAS00893672193, which is available at the European Genome Archive.\\'\\nResponse:\\n[\\n    [\\n        \"dataset_identifier\": \"EGAS00893672193\",\\n        \"data_repository\": \"European Genome Archive\",\\n        \"dataset_webpage\": \"https://ega-archive.org/studies/EGAS00893672193\"\\n    ]\\n]\\n\\nExample 2:\\nContent: \\'Proteomics data was obtained from PRIDE, accession PXD029921.\\'\\nResponse:\\n[\\n    [\\n        \"dataset_identifier\": \"PXD029921\",\\n        \"data_repository\": \"PRIDE\",\\n        \"dataset_webpage\": \"https://www.ebi.ac.uk/pride/archive/projects/PXD029921\"\\n    ]\\n]\\n\\nExample 3:\\nContent: \\'The repository dbGaP hosts the dataset phs011366.v1.p1 at this location.\\'\\nResponse:\\n[\\n    [\\n        \"dataset_identifier\": \"phs011366.v1.p1\",\\n        \"data_repository\": \"dbGaP\",\\n        \"dataset_webpage\": \"https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001366.v1.p1\"\\n    ]\\n]\\n\\nNow process the following content:\\n\\nContent: <p> Proteomics data can be found at the MassIVE database via MSV000092944. </p> \\n\\n\\n<p class=\"font-secondary\"> <em> This section collects any data citations, data availability statements, or supplementary materials included in this article. </em> </p> \\n<div class=\"caption p\"> <span> SI </span> </div> \\n<div class=\"media p\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/11425778/bin/NIHMS2020672-supplement-SI.pdf\"> NIHMS2020672-supplement-SI.pdf </a> <sup> (1.4MB, pdf) </sup> </div> </div> \\n<div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/11425778/bin/NIHMS2020672-supplement-SI.pdf\"> NIHMS2020672-supplement-SI.pdf </a> <sup> (1.4MB, pdf) </sup> </div> \\n<a class=\"usa-link\" href=\"/articles/instance/11425778/bin/NIHMS2020672-supplement-SI.pdf\"> NIHMS2020672-supplement-SI.pdf </a> \\n<p> Proteomics data can be found at the MassIVE database via MSV000092944. </p> \\n\\n\\n<p id=\"P37\"> Proteomics data can be found at the MassIVE database via MSV000092944. </p> \\n\\n\\n<p> Proteomics data can be found at the MassIVE database via MSV000092944. </p> \\n\\n\\n<p class=\"font-secondary\"> <em> This section collects any data citations, data availability statements, or supplementary materials included in this article. </em> </p> \\n<div class=\"caption p\"> <span> SI </span> </div> \\n<div class=\"media p\"> <div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/11425778/bin/NIHMS2020672-supplement-SI.pdf\"> NIHMS2020672-supplement-SI.pdf </a> <sup> (1.4MB, pdf) </sup> </div> </div> \\n<div class=\"caption\"> <a class=\"usa-link\" href=\"/articles/instance/11425778/bin/NIHMS2020672-supplement-SI.pdf\"> NIHMS2020672-supplement-SI.pdf </a> <sup> (1.4MB, pdf) </sup> </div> \\n<a class=\"usa-link\" href=\"/articles/instance/11425778/bin/NIHMS2020672-supplement-SI.pdf\"> NIHMS2020672-supplement-SI.pdf </a> \\n<p> Proteomics data can be found at the MassIVE database via MSV000092944. </p> \\n\\n\\n<div class=\"fn-group p font-secondary-light font-sm\"> <div class=\"fn p\" id=\"FN1\"> <p id=\"P39\"> Supporting Information </p> <p id=\"P40\"> The Supporting Information is available free of charge at <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/10.1021/acs.jproteome.4c00338?goto=supporting-info\" rel=\"noopener noreferrer\" target=\"_blank\"> https://pubs.acs.org/doi/10.1021/acs.jproteome.4c00338 </a> . </p> <p id=\"P41\"> <a class=\"usa-link\" href=\"#SD1\"> Figure S1 </a> , quality control of label-free quantitative proteomics; <a class=\"usa-link\" href=\"#SD1\"> Figure S2 </a> , comparison between the monoculture drug-resistant and drug-sensitive cells; <a class=\"usa-link\" href=\"#SD1\"> Figure S3 </a> , P53 signaling pathway and the highlighted proteins upregulated in drug-resistant cells; <a class=\"usa-link\" href=\"#SD1\"> Figure S4 </a> , workflow of the direct coculture and indirect coculture; and <a class=\"usa-link\" href=\"#SD1\"> Figure S5 </a> , upregulated HIF-1 signaling pathways and TCA cycle in indirect coculture cells ( <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_001.pdf\" rel=\"noopener noreferrer\" target=\"_blank\"> PDF </a> ) </p> <p id=\"P42\"> All identified proteins and upregulated proteins in P53 pathways in monoculture and coculture cells ( <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_002.xlsx\" rel=\"noopener noreferrer\" target=\"_blank\"> XLSX </a> ) </p> <p id=\"P420\"> Volcano plot of proteins in monoculture and coculture cells ( <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_003.xlsx\" rel=\"noopener noreferrer\" target=\"_blank\"> XLSX </a> ) </p> <p id=\"P43\"> Enriched upregulated KEGG pathways in monoculture and coculture systems ( <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_004.xlsx\" rel=\"noopener noreferrer\" target=\"_blank\"> XLSX </a> ) </p> </div> <div class=\"fn p\" id=\"FN2\"> <p id=\"P44\"> The authors declare no competing financial interest. </p> </div> <div class=\"fn p\" id=\"FN3\"> <p id=\"P45\"> Complete contact information is available at: <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/10.1021/acs.jproteome.4c00338?ref=pdf\" rel=\"noopener noreferrer\" target=\"_blank\"> https://pubs.acs.org/10.1021/acs.jproteome.4c00338 </a> </p> </div> </div> \\n<div class=\"fn p\" id=\"FN1\"> <p id=\"P39\"> Supporting Information </p> <p id=\"P40\"> The Supporting Information is available free of charge at <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/10.1021/acs.jproteome.4c00338?goto=supporting-info\" rel=\"noopener noreferrer\" target=\"_blank\"> https://pubs.acs.org/doi/10.1021/acs.jproteome.4c00338 </a> . </p> <p id=\"P41\"> <a class=\"usa-link\" href=\"#SD1\"> Figure S1 </a> , quality control of label-free quantitative proteomics; <a class=\"usa-link\" href=\"#SD1\"> Figure S2 </a> , comparison between the monoculture drug-resistant and drug-sensitive cells; <a class=\"usa-link\" href=\"#SD1\"> Figure S3 </a> , P53 signaling pathway and the highlighted proteins upregulated in drug-resistant cells; <a class=\"usa-link\" href=\"#SD1\"> Figure S4 </a> , workflow of the direct coculture and indirect coculture; and <a class=\"usa-link\" href=\"#SD1\"> Figure S5 </a> , upregulated HIF-1 signaling pathways and TCA cycle in indirect coculture cells ( <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_001.pdf\" rel=\"noopener noreferrer\" target=\"_blank\"> PDF </a> ) </p> <p id=\"P42\"> All identified proteins and upregulated proteins in P53 pathways in monoculture and coculture cells ( <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_002.xlsx\" rel=\"noopener noreferrer\" target=\"_blank\"> XLSX </a> ) </p> <p id=\"P420\"> Volcano plot of proteins in monoculture and coculture cells ( <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_003.xlsx\" rel=\"noopener noreferrer\" target=\"_blank\"> XLSX </a> ) </p> <p id=\"P43\"> Enriched upregulated KEGG pathways in monoculture and coculture systems ( <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_004.xlsx\" rel=\"noopener noreferrer\" target=\"_blank\"> XLSX </a> ) </p> </div> \\n<p id=\"P39\"> Supporting Information </p> \\n<p id=\"P40\"> The Supporting Information is available free of charge at <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/10.1021/acs.jproteome.4c00338?goto=supporting-info\" rel=\"noopener noreferrer\" target=\"_blank\"> https://pubs.acs.org/doi/10.1021/acs.jproteome.4c00338 </a> . </p> \\n<a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/10.1021/acs.jproteome.4c00338?goto=supporting-info\" rel=\"noopener noreferrer\" target=\"_blank\"> https://pubs.acs.org/doi/10.1021/acs.jproteome.4c00338 </a> \\n<p id=\"P41\"> <a class=\"usa-link\" href=\"#SD1\"> Figure S1 </a> , quality control of label-free quantitative proteomics; <a class=\"usa-link\" href=\"#SD1\"> Figure S2 </a> , comparison between the monoculture drug-resistant and drug-sensitive cells; <a class=\"usa-link\" href=\"#SD1\"> Figure S3 </a> , P53 signaling pathway and the highlighted proteins upregulated in drug-resistant cells; <a class=\"usa-link\" href=\"#SD1\"> Figure S4 </a> , workflow of the direct coculture and indirect coculture; and <a class=\"usa-link\" href=\"#SD1\"> Figure S5 </a> , upregulated HIF-1 signaling pathways and TCA cycle in indirect coculture cells ( <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_001.pdf\" rel=\"noopener noreferrer\" target=\"_blank\"> PDF </a> ) </p> \\n<a class=\"usa-link\" href=\"#SD1\"> Figure S1 </a> \\n<a class=\"usa-link\" href=\"#SD1\"> Figure S2 </a> \\n<a class=\"usa-link\" href=\"#SD1\"> Figure S3 </a> \\n<a class=\"usa-link\" href=\"#SD1\"> Figure S4 </a> \\n<a class=\"usa-link\" href=\"#SD1\"> Figure S5 </a> \\n<a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_001.pdf\" rel=\"noopener noreferrer\" target=\"_blank\"> PDF </a> \\n<p id=\"P42\"> All identified proteins and upregulated proteins in P53 pathways in monoculture and coculture cells ( <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_002.xlsx\" rel=\"noopener noreferrer\" target=\"_blank\"> XLSX </a> ) </p> \\n<a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_002.xlsx\" rel=\"noopener noreferrer\" target=\"_blank\"> XLSX </a> \\n<p id=\"P420\"> Volcano plot of proteins in monoculture and coculture cells ( <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_003.xlsx\" rel=\"noopener noreferrer\" target=\"_blank\"> XLSX </a> ) </p> \\n<a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_003.xlsx\" rel=\"noopener noreferrer\" target=\"_blank\"> XLSX </a> \\n<p id=\"P43\"> Enriched upregulated KEGG pathways in monoculture and coculture systems ( <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_004.xlsx\" rel=\"noopener noreferrer\" target=\"_blank\"> XLSX </a> ) </p> \\n<a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/suppl/10.1021/acs.jproteome.4c00338/suppl_file/pr4c00338_si_004.xlsx\" rel=\"noopener noreferrer\" target=\"_blank\"> XLSX </a> \\n<div class=\"fn p\" id=\"FN2\"> <p id=\"P44\"> The authors declare no competing financial interest. </p> </div> \\n<p id=\"P44\"> The authors declare no competing financial interest. </p> \\n<div class=\"fn p\" id=\"FN3\"> <p id=\"P45\"> Complete contact information is available at: <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/10.1021/acs.jproteome.4c00338?ref=pdf\" rel=\"noopener noreferrer\" target=\"_blank\"> https://pubs.acs.org/10.1021/acs.jproteome.4c00338 </a> </p> </div> \\n<p id=\"P45\"> Complete contact information is available at: <a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/10.1021/acs.jproteome.4c00338?ref=pdf\" rel=\"noopener noreferrer\" target=\"_blank\"> https://pubs.acs.org/10.1021/acs.jproteome.4c00338 </a> </p> \\n<a class=\"usa-link usa-link--external\" href=\"https://pubs.acs.org/doi/10.1021/acs.jproteome.4c00338?ref=pdf\" rel=\"noopener noreferrer\" target=\"_blank\"> https://pubs.acs.org/10.1021/acs.jproteome.4c00338 </a> \\n\\n.'}]\n",
      "prompt_manager.py - line 96 - DEBUG - Calculating checksum for content: prompt\n",
      "base_parser.py - line 450 - INFO - Prompt ID: gemini-2.0-flash-0-eaad1d96501f94fabc9e89c80def198354689d81df2860883669ae3747b5afb7\n",
      "base_parser.py - line 1108 - INFO - Expected string but got list. Converting list to string.\n",
      "base_parser.py - line 1115 - DEBUG - Counting tokens for model: gemini-2.0-flash, prompt length: 12180 char\n",
      "base_parser.py - line 1124 - DEBUG - Rough estimate of token count for Gemini model 'gemini-2.0-flash': 3045\n",
      "base_parser.py - line 469 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages: 3045 tokens\n",
      "base_parser.py - line 540 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-5a8ae020-2fe8-4d3a-9857-181e3569f875\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"MSV000092944\\\",\\n        \\\"data_repository\\\": \\\"MassIVE\\\",\\n        \\\"dataset_webpage\\\": \\\"https://massive.ucsd.edu/ProteoSAFe/QueryDataset?id=MSV000092944\\\"\\n    ]\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"MSV000092944\\\",\\n        \\\"data_repository\\\": \\\"MassIVE\\\",\\n        \\\"dataset_webpage\\\": \\\"https://massive.ucsd.edu/ProteoSAFe/QueryDataset?id=MSV000092944\\\"\\n    ]\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1751902234,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 4595,\n",
      "        \"completion_tokens\": 85,\n",
      "        \"total_tokens\": 4680,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\n",
      "base_parser.py - line 991 - INFO - Fetching metadata for 1 datasets\n",
      "base_parser.py - line 999 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\n",
      "base_parser.py - line 1010 - INFO - Raw accession ID: MSV000092944\n",
      "base_parser.py - line 939 - INFO - Resolving data repository for: MassIVE\n",
      "base_parser.py - line 952 - DEBUG - Checking if MassIVE == bigd.big.ac.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if MassIVE == ega-archive.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if MassIVE == massive.ucsd.edu\n",
      "base_parser.py - line 963 - INFO - Found repo_name match for MassIVE\n",
      "base_parser.py - line 921 - INFO - Resolving accession ID for MSV000092944 in massive.ucsd.edu\n",
      "base_parser.py - line 1024 - INFO - Processing dataset 1 with repo: massive.ucsd.edu and accession_id: MSV000092944\n",
      "base_parser.py - line 1025 - DEBUG - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\n",
      "base_parser.py - line 1050 - INFO - Dataset page: https://massive.ucsd.edu/ProteoSAFe/dataset.jsp?accession=MSV000092944\n",
      "base_parser.py - line 1057 - INFO - Adding access mode for dataset 1: Complex download\n",
      "base_parser.py - line 1067 - INFO - Updated datasets len: 1\n",
      "html_parser.py - line 397 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 15, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel'],\n",
      "      dtype='object')\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(nan)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(None)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(None)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(None)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(None)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(None)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(None)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(None)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(https://www.ncbi.nlm.nih.gov/pmc/articles/instance/11425778/bin/NIHMS2020672-supplement-SI.pdf)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(None)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(None)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(None)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(None)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(None)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(None)\n",
      "html_retriever.py - line 229 - INFO - Extracting publication title from HTML\n",
      "html_retriever.py - line 239 - WARNING - No publication title found in the HTML data.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "  dataset_identifier data_repository  \\\n",
       "0       MSV000092944         MassIVE   \n",
       "0                NaN             NaN   \n",
       "8                NaN             NaN   \n",
       "\n",
       "                                     dataset_webpage       access_mode  \\\n",
       "0  https://massive.ucsd.edu/ProteoSAFe/dataset.js...  Complex download   \n",
       "0                                                NaN               NaN   \n",
       "8                                                NaN               NaN   \n",
       "\n",
       "                                                link  \\\n",
       "0                                                NaN   \n",
       "0      https://doi.org/10.1021/acs.jproteome.4c00338   \n",
       "8  /articles/instance/11425778/bin/NIHMS2020672-s...   \n",
       "\n",
       "                            title     file_info description source_section  \\\n",
       "0                             NaN           NaN         NaN            NaN   \n",
       "0   10.1021/acs.jproteome.4c00338           n/a         n/a            n/a   \n",
       "8  NIHMS2020672-supplement-SI.pdf  (1.4MB, pdf)         n/a        d67e776   \n",
       "\n",
       "                                 section_class  \\\n",
       "0                                          NaN   \n",
       "0  pmc-layout__citation font-secondary font-xs   \n",
       "8                                      media p   \n",
       "\n",
       "                                       download_link file_extension  \\\n",
       "0                                                NaN           None   \n",
       "0                                               None           None   \n",
       "8  https://www.ncbi.nlm.nih.gov/pmc/articles/inst...            pdf   \n",
       "\n",
       "                                         a_attr_href  \\\n",
       "0                                                NaN   \n",
       "0      https://doi.org/10.1021/acs.jproteome.4c00338   \n",
       "8  /articles/instance/11425778/bin/NIHMS2020672-s...   \n",
       "\n",
       "                  a_attr_class a_attr_data-ga-action a_attr_target  \\\n",
       "0                          NaN                   NaN           NaN   \n",
       "0  usa-link usa-link--external      click_feat_suppl        _blank   \n",
       "8                     usa-link      click_feat_suppl           NaN   \n",
       "\n",
       "            a_attr_rel                                         source_url  \\\n",
       "0                  NaN  https://www.ncbi.nlm.nih.gov/pmc/articles/pmc1...   \n",
       "0  noopener noreferrer  https://www.ncbi.nlm.nih.gov/pmc/articles/pmc1...   \n",
       "8                  NaN  https://www.ncbi.nlm.nih.gov/pmc/articles/pmc1...   \n",
       "\n",
       "    article_title  \n",
       "0  No title found  \n",
       "0  No title found  \n",
       "8  No title found  "
      ],
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>dataset_identifier</th>\n",
       "      <th>data_repository</th>\n",
       "      <th>dataset_webpage</th>\n",
       "      <th>access_mode</th>\n",
       "      <th>link</th>\n",
       "      <th>title</th>\n",
       "      <th>file_info</th>\n",
       "      <th>description</th>\n",
       "      <th>source_section</th>\n",
       "      <th>section_class</th>\n",
       "      <th>download_link</th>\n",
       "      <th>file_extension</th>\n",
       "      <th>a_attr_href</th>\n",
       "      <th>a_attr_class</th>\n",
       "      <th>a_attr_data-ga-action</th>\n",
       "      <th>a_attr_target</th>\n",
       "      <th>a_attr_rel</th>\n",
       "      <th>source_url</th>\n",
       "      <th>article_title</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>MSV000092944</td>\n",
       "      <td>MassIVE</td>\n",
       "      <td>https://massive.ucsd.edu/ProteoSAFe/dataset.js...</td>\n",
       "      <td>Complex download</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/pmc1...</td>\n",
       "      <td>No title found</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://doi.org/10.1021/acs.jproteome.4c00338</td>\n",
       "      <td>10.1021/acs.jproteome.4c00338</td>\n",
       "      <td>n/a</td>\n",
       "      <td>n/a</td>\n",
       "      <td>n/a</td>\n",
       "      <td>pmc-layout__citation font-secondary font-xs</td>\n",
       "      <td>None</td>\n",
       "      <td>None</td>\n",
       "      <td>https://doi.org/10.1021/acs.jproteome.4c00338</td>\n",
       "      <td>usa-link usa-link--external</td>\n",
       "      <td>click_feat_suppl</td>\n",
       "      <td>_blank</td>\n",
       "      <td>noopener noreferrer</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/pmc1...</td>\n",
       "      <td>No title found</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>/articles/instance/11425778/bin/NIHMS2020672-s...</td>\n",
       "      <td>NIHMS2020672-supplement-SI.pdf</td>\n",
       "      <td>(1.4MB, pdf)</td>\n",
       "      <td>n/a</td>\n",
       "      <td>d67e776</td>\n",
       "      <td>media p</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/inst...</td>\n",
       "      <td>pdf</td>\n",
       "      <td>/articles/instance/11425778/bin/NIHMS2020672-s...</td>\n",
       "      <td>usa-link</td>\n",
       "      <td>click_feat_suppl</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/pmc/articles/pmc1...</td>\n",
       "      <td>No title found</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "execution_count": 12
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-07-07T15:30:41.738667Z",
     "start_time": "2025-07-07T15:30:34.364550Z"
    }
   },
   "cell_type": "code",
   "source": [
    "results_RTR = data_gatherer.parse_data(API_unsupported_input, raw_html, publisher=\"Nature\", use_portkey_for_gemini=True, prompt_name=\"retrieve_datasets_simple_JSON\", raw_data_format =\"HTML\", semantic_retrieval=True)\n",
    "results_RTR"
   ],
   "id": "6c1e55136a9f206d",
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "orchestrator.py - line 185 - INFO - Parsing data from URL: https://www.nature.com/articles/s41467-024-51831-7 with publisher: Nature\n",
      "base_parser.py - line 181 - INFO - LLMParser initialized.\n",
      "base_parser.py - line 913 - DEBUG - Repo mapping: {'bigd': 'bigd.big.ac.cn', 'european genome-phenome archive': 'ega-archive.org', 'massive': 'massive.ucsd.edu', 'pride': 'www.ebi.ac.uk', 'iprox': 'www.iprox.cn', 'proteomecentral': 'proteomecentral.proteomexchange.org', 'proteomexchange': 'www.proteomexchange.org', 'proteomic data commons': 'pdc.cancer.gov', 'genomic data commons': 'gdc.cancer.gov', 'the cancer imaging archive': 'cancerimagingarchive.net', 'imaging data commons': 'imaging.datacommons.cancer.gov', 'jpost': 'repository.jpostdb.org', 'doi.org': 'doi.org', 'geo': 'geo', 'dbgap': 'dbgap', 'sra': 'sra', 'arrayexpress': 'ArrayExpress'}\n",
      "html_parser.py - line 79 - INFO - Initializing htmlRetriever\n",
      "html_parser.py - line 342 - INFO - Function call: parse_data(api_data(<class 'str'>), Nature, https://www.nature.com/articles/s41467-024-51831-7, additional_data, HTML)\n",
      "html_parser.py - line 370 - INFO - Chunking the HTML content for the parsing step.\n",
      "html_parser.py - line 427 - INFO - Function_call: extract_href_from_html_supplementary_material(tree, https://www.nature.com/articles/s41467-024-51831-7)\n",
      "html_parser.py - line 435 - DEBUG - Found 4 anchors with data-ga-action='click_feat_suppl'.\n",
      "base_parser.py - line 302 - DEBUG - Function_call: reconstruct_download_link(https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-024-51831-7/MediaObjects/41467_2024_51831_MOESM1_ESM.pdf, c-article-supplementary__item, https://www.nature.com/articles/s41467-024-51831-7)\n",
      "base_parser.py - line 302 - DEBUG - Function_call: reconstruct_download_link(https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-024-51831-7/MediaObjects/41467_2024_51831_MOESM2_ESM.pdf, c-article-supplementary__item, https://www.nature.com/articles/s41467-024-51831-7)\n",
      "base_parser.py - line 302 - DEBUG - Function_call: reconstruct_download_link(https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-024-51831-7/MediaObjects/41467_2024_51831_MOESM3_ESM.pdf, c-article-supplementary__item, https://www.nature.com/articles/s41467-024-51831-7)\n",
      "base_parser.py - line 302 - DEBUG - Function_call: reconstruct_download_link(https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-024-51831-7/MediaObjects/41467_2024_51831_MOESM4_ESM.xlsx, c-article-supplementary__item, https://www.nature.com/articles/s41467-024-51831-7)\n",
      "html_parser.py - line 488 - INFO - Extracted 4 unique supplementary material links from HTML.\n",
      "html_retriever.py - line 171 - INFO - Extracting data availability elements from HTML\n",
      "html_retriever.py - line 182 - INFO - Using selector: section.data-availability-statement\n",
      "html_retriever.py - line 182 - INFO - Using selector: section.associated-data\n",
      "html_retriever.py - line 182 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Data Availability')\n",
      "html_retriever.py - line 182 - INFO - Using selector: h3.pmc_sec_title:-soup-contains('Data Availability')\n",
      "html_retriever.py - line 182 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Associated Data')\n",
      "html_retriever.py - line 182 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Footnotes')\n",
      "html_retriever.py - line 220 - INFO - Found 0 data availability elements from HTML.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "Batches:   0%|          | 0/1 [00:00<?, ?it/s]"
      ],
      "application/vnd.jupyter.widget-view+json": {
       "version_major": 2,
       "version_minor": 0,
       "model_id": "c7de1bddaeeb4c5db0ea3271026a0363"
      }
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "base_parser.py - line 426 - INFO - Loading prompt: retrieve_datasets_simple_JSON for model gemini-2.0-flash\n",
      "prompt_manager.py - line 32 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \n",
      "base_parser.py - line 1108 - INFO - Expected string but got list. Converting list to string.\n",
      "base_parser.py - line 1115 - DEBUG - Counting tokens for model: gemini-2.0-flash, prompt length: 1382 char\n",
      "base_parser.py - line 1124 - DEBUG - Rough estimate of token count for Gemini model 'gemini-2.0-flash': 345\n",
      "base_parser.py - line 433 - INFO - Content length: 19082\n",
      "base_parser.py - line 435 - DEBUG - static_prompt: [{'role': 'system', 'content': \"You are a specialized assistant that extracts dataset references from the content of scientific papers. You must output a JSON array of objects, where each object has the following keys: 'dataset_identifier', 'data_repository', and 'dataset_webpage'. Follow the structure of the provided examples exactly. We will not wrap the json codes in JSON markers.\"}, {'role': 'user', 'content': 'Extract dataset references based on the examples below:\\n\\nExample 1:\\nContent: \\'The study used dataset EGAS00893672193, which is available at the European Genome Archive.\\'\\nResponse:\\n[\\n    [\\n        \"dataset_identifier\": \"EGAS00893672193\",\\n        \"data_repository\": \"European Genome Archive\",\\n        \"dataset_webpage\": \"https://ega-archive.org/studies/EGAS00893672193\"\\n    ]\\n]\\n\\nExample 2:\\nContent: \\'Proteomics data was obtained from PRIDE, accession PXD029921.\\'\\nResponse:\\n[\\n    [\\n        \"dataset_identifier\": \"PXD029921\",\\n        \"data_repository\": \"PRIDE\",\\n        \"dataset_webpage\": \"https://www.ebi.ac.uk/pride/archive/projects/PXD029921\"\\n    ]\\n]\\n\\nExample 3:\\nContent: \\'The repository dbGaP hosts the dataset phs011366.v1.p1 at this location.\\'\\nResponse:\\n[\\n    [\\n        \"dataset_identifier\": \"phs011366.v1.p1\",\\n        \"data_repository\": \"dbGaP\",\\n        \"dataset_webpage\": \"https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001366.v1.p1\"\\n    ]\\n]\\n\\nNow process the following content:\\n\\nContent: {content}.'}]\n",
      "prompt_manager.py - line 56 - DEBUG - Rendering prompt with dynamic parts(<class 'dict'>): {'content': 'Source data\\nSource data\\nSupplementary information\\nSupplementary Information\\nPeer Review File\\nReporting Summary\\nData availability\\nThe RNA microarray datasets generated in this study were deposited into the National Center for Biotechnology Information (NCBI) Gene Expression Omnibus (GEO) database under accession number GSE269782 and are available at the following URL:\\nhttps://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE269782\\n. The RNAseq and microarray datasets of human lung adenocarcinoma demonstrated in Figs.\\n1e\\n,\\n6h\\n, and Supplementary Fig.\\n21\\nwere downloaded from NCBI’s GEO:\\nGSE31210\\n;\\nGSE106765\\n;\\nGSE60189\\n;\\nGSE59239\\n;\\nGSE122005\\n;\\nGSE38121\\n;\\nGSE71587\\n;\\nGSE37699\\n. Datasets of human lung adenocarcinoma demonstrated in Fig.\\n8d\\nand Supplementary Fig.\\n26\\nwere from the supplementary information attached to the article of Roper N. et al.\\n59\\n. For glycosylation analysis, protein mass spectrometry raw data and analytical tool data have updated in a repository via ProteomeXchange (\\nhttps://www.proteomexchange.org\\n) with project accession: PXD051771.\\nSource data\\nare provided with this paper.\\nAuthor information\\nAuthors and Affiliations\\nDepartment of Oncology, National Cheng Kung University Hospital, College of Medicine, National Cheng Kung University, Tainan, Taiwan\\nChun-Hua Hung, Shang-Yin Wu & Wu-Chou Su\\nCenter of Applied Nanomedicine, National Cheng Kung University, Tainan, Taiwan\\nChun-Hua Hung, Hsuan-Heng Yeh & Wu-Chou Su\\nInstitute of Basic Medical Sciences, College of Medicine, National Cheng Kung University, Tainan, Taiwan\\nChun-Hua Hung & Wu-Chou Su\\nDepartment of Biomedical Engineering, College of Engineering, National Cheng Kung University, Tainan, Taiwan\\nChun-Hua Hung\\nInstitute of Clinical Medicine, College of Medicine, National Cheng Kung University, Tainan, Taiwan\\nShang-Yin Wu, Chien-Chung Lin & Wu-Chou Su\\nInstitute of Biological Chemistry, Academia Sinica, Taipei, Taiwan\\nCheng-I Daniel Yao & Chun-Hung Lin\\nDepartment of Internal Medicine, National Cheng Kung University Hospital, College of Medicine, National Cheng Kung University, Tainan, Taiwan\\nChien-Chung Lin\\nDepartment of Biochemistry and Molecular Biology, College of Medicine, National Cheng Kung University, Tainan, Taiwan\\nChien-Chung Lin\\nDepartment of Pathology, National Cheng Kung University Hospital, College of Medicine, National Cheng Kung University, Tainan, Taiwan\\nChang-Yao Chu\\nInstitute of Molecular Medicine, College of Medicine, National Cheng Kung University, Tainan, Taiwan\\nTzu-Yu Huang & Wu-Chou Su\\nDepartment of Pharmacology, College of Medicine, National Cheng Kung University, Tainan, Taiwan\\nMeng-Ru Shen\\nDepartment of Obstetrics and Gynecology, National Cheng Kung University Hospital, College of Medicine, National Cheng Kung University, Tainan, Taiwan\\nMeng-Ru Shen\\nDepartment of Chemistry and Institute of Biochemical Sciences, National Taiwan University, Taipei, Taiwan\\nChun-Hung Lin\\nAuthors\\nChun-Hua Hung\\nView author publications\\nSearch author on:\\nPubMed\\nGoogle Scholar\\nShang-Yin Wu\\nView author publications\\nSearch author on:\\nPubMed\\nGoogle Scholar\\nCheng-I Daniel Yao\\nView author publications\\nSearch author on:\\nPubMed\\nGoogle Scholar\\nHsuan-Heng Yeh\\nView author publications\\nSearch author on:\\nPubMed\\nGoogle Scholar\\nChien-Chung Lin\\nView author publications\\nSearch author on:\\nPubMed\\nGoogle Scholar\\nChang-Yao Chu\\nView author publications\\nSearch author on:\\nPubMed\\nGoogle Scholar\\nTzu-Yu Huang\\nView author publications\\nSearch author on:\\nPubMed\\nGoogle Scholar\\nMeng-Ru Shen\\nView author publications\\nSearch author on:\\nPubMed\\nGoogle Scholar\\nChun-Hung Lin\\nView author publications\\nSearch author on:\\nPubMed\\nGoogle Scholar\\nWu-Chou Su\\nView author publications\\nSearch author on:\\nPubMed\\nGoogle Scholar\\nContributions\\nChun-Hua Hung: Conceptualization, data curation, software, formal analysis, validation, investigation, visualization, methodology, writing-original draft. Shang-Yin Wu: Software, formal analysis, investigation. Cheng-I Daniel Yao: Formal analysis, methodology. Hsuan-Heng Yeh: Formal analysis, methodology. Chien-Chung Lin: Resources, formal analysis. Chang-Yao Chu: Formal analysis, methodology. Tzu-Yu Huang: Formal analysis, methodology. Meng-Ru Shen: Formal analysis, methodology, funding acquisition. Chun-Hung Lin: Formal analysis, methodology, funding acquisition. Wu-Chou Su*: Conceptualization, resources, supervision, funding acquisition, writing-original draft, project administration.\\nCorresponding author\\nCorrespondence to\\nWu-Chou Su\\n.\\nReferences\\nYu, H., Lee, H., Herrmann, A., Buettner, R. & Jove, R. Revisiting STAT3 signalling in cancer: new and unexpected biological functions.\\nNat. Rev. Cancer\\n14\\n, 736–746 (2014).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nJones, S. A. & Jenkins, B. J. Recent insights into targeting the IL-6 cytokine family in inflammatory diseases and cancer.\\nNat. Rev. Immunol.\\n18\\n, 773–789 (2018).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nRose-John, S. The soluble interleukin 6 receptor: advanced therapeutic options in inflammation.\\nClin. Pharmacol. Therapeut.\\n102\\n, 591–598 (2017).\\nArticle\\nCAS\\nGoogle Scholar\\nTrikha, M., Corringham, R., Klein, B. & Rossi, J. F. Targeted anti-interleukin-6 monoclonal antibody therapy for cancer: a review of the rationale and clinical evidence.\\nClin. Cancer Res.\\n9\\n, 4653–4665 (2003).\\nCAS\\nPubMed\\nPubMed Central\\nGoogle Scholar\\nHunter, C. A. & Jones, S. A. IL-6 as a keystone cytokine in health and disease.\\nNat. Immunol\\n.\\n16\\n, 448–457 (2015).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nGarbers, C., Heink, S., Korn, T. & Rose-John, S. Interleukin-6: designing specific therapeutics for a complex cytokine.\\nNat. Rev. Drug Discov.\\n17\\n, 395–412 (2018).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nMurakami, M., Kamimura, D. & Hirano, T. Pleiotropy and specificity: insights from the interleukin 6 family of cytokines.\\nImmunity\\n50\\n, 812–831 (2019).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nDuan, Z. et al. Signal transducers and activators of transcription 3 pathway activation in drug-resistant ovarian cancer.\\nClin. Cancer Res.\\n12\\n, 5055–5063 (2006).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nHuang, W. L. et al. Signal transducer and activator of transcription 3 activation up-regulates interleukin-6 autocrine production: a biochemical and genetic study of established cancer cell lines and clinical isolated human cancer cells.\\nMol. Cancer\\n9\\n, 309 (2010).\\nArticle\\nCAS\\nPubMed\\nPubMed Central\\nGoogle Scholar\\nLee, H. J. et al. Drug resistance via feedback activation of Stat3 in oncogene-addicted cancer cells.\\nCancer Cell\\n26\\n, 207–221 (2014).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nRokavec, M. et al. IL-6R/STAT3/miR-34a feedback loop promotes EMT-mediated colorectal cancer invasion and metastasis.\\nJ. Clin. Invest.\\n124\\n, 1853–1867 (2014).\\nArticle\\nCAS\\nPubMed\\nPubMed Central\\nGoogle Scholar\\nTaniguchi, K. et al. A gp130-Src-YAP module links inflammation to epithelial regeneration.\\nNature\\n519\\n, 57–62 (2015).\\nArticle\\nADS\\nCAS\\nPubMed\\nPubMed Central\\nGoogle Scholar\\nQuintanal-Villalonga, A. et al. Lineage plasticity in cancer: a shared pathway of therapeutic resistance.\\nNat. Rev. Clin. Oncol.\\n17\\n, 360–371 (2020).\\nArticle\\nPubMed\\nPubMed Central\\nGoogle Scholar\\nPaez, J. G. et al. EGFR mutations in lung cancer: correlation with clinical response to gefitinib therapy.\\nScience\\n304\\n, 1497–1500 (2004).\\nArticle\\nADS\\nCAS\\nPubMed\\nGoogle Scholar\\nRamalingam, S. S. et al. Overall survival with osimertinib in untreated, EGFR-mutated advanced NSCLC.\\nN. Engl. J. Med.\\n382\\n, 41–50 (2020).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nOxnard, G. R. et al. Assessment of resistance mechanisms and clinical implications in patients with EGFR T790M-positive lung cancer and acquired resistance to osimertinib.\\nJAMA Oncol.\\n4\\n, 1527–1534 (2018).\\nArticle\\nPubMed\\nGoogle Scholar\\nYang, Z. et al. Investigating novel resistance mechanisms to third-generation EGFR tyrosine kinase inhibitor osimertinib in non-small cell lung cancer patients.\\nClin. Cancer Res.\\n24\\n, 3097–3107 (2018).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nSequist, L. V. et al. Genotypic and histological evolution of lung cancers acquiring resistance to EGFR inhibitors.\\nSci. Transl. Med.\\n3\\n, 75ra26 (2011).\\nArticle\\nPubMed\\nPubMed Central\\nGoogle Scholar\\nLambert, A. W. & Weinberg, R. A. Linking EMT programmes to normal and neoplastic epithelial stem cells.\\nNat. Rev. Cancer\\n21\\n, 325–338 (2021).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nWilliams, E. D., Gao, D., Redfern, A. & Thompson, E. W. Controversies around epithelial-mesenchymal plasticity in cancer metastasis.\\nNat. Rev. Cancer\\n19\\n, 716–732 (2019).\\nArticle\\nCAS\\nPubMed\\nPubMed Central\\nGoogle Scholar\\nYao, Z. et al. TGF-beta IL-6 axis mediates selective and adaptive mechanisms of resistance to molecular targeted therapy in lung cancer.\\nProc. Natl Acad. Sci. USA\\n107\\n, 15535–15540 (2010).\\nArticle\\nADS\\nCAS\\nPubMed\\nPubMed Central\\nGoogle Scholar\\nKim, S. M. et al. Activation of IL-6R/JAK1/STAT signaling induces de novo resistance to irreversible EGFR inhibitors in non-small cell lung cancer with T790M resistance mutation.\\nMol. Cancer Ther.\\n11\\n, 2254–2264 (2012).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nKorkaya, H., Liu, S. & Wicha, M. S. Regulation of cancer stem cells by cytokine networks: attacking cancer’s inflammatory roots.\\nClin. Cancer Res.\\n17\\n, 6125–6129 (2011).\\nArticle\\nCAS\\nPubMed\\nPubMed Central\\nGoogle Scholar\\nSenescence-Associated Pro-inflammatory Cytokines, VernotJ. P. and tumor cell plasticity.\\nFront. Mol. Biosci.\\n7\\n, 63 (2020).\\nADS\\nGoogle Scholar\\nKurppa, K. J. et al. Treatment-induced tumor dormancy through YAP-mediated transcriptional reprogramming of the apoptotic pathway.\\nCancer Cell\\n37\\n, 104–122.e112 (2020).\\nArticle\\nCAS\\nPubMed\\nPubMed Central\\nGoogle Scholar\\nFerone, G. et al. SOX2 is the determining oncogenic switch in promoting lung squamous cell carcinoma from different cells of origin.\\nCancer Cell\\n30\\n, 519–532 (2016).\\nArticle\\nCAS\\nPubMed\\nPubMed Central\\nGoogle Scholar\\nBoumahdi, S. & de Sauvage, F. J. The great escape: tumour cell plasticity in resistance to targeted therapy.\\nNat. Rev. Drug Discov.\\n19\\n, 39–56 (2020).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nMosteiro, L. et al. Tissue damage and senescence provide critical signals for cellular reprogramming in vivo.\\nScience\\n354\\n, 6315 (2016).\\nArticle\\nGoogle Scholar\\nBora-Singhal, N. et al. YAP1 regulates OCT4 activity and SOX2 expression to facilitate self-renewal and vascular mimicry of stem-like cells.\\nStem cells\\n33\\n, 1705–1718 (2015).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nPoupart, P. et al. B cell growth modulating and differentiating activity of recombinant human 26-kd protein (BSF-2, HuIFN-beta 2, HPGF).\\nEMBO J\\n6\\n, 1219–1224 (1987).\\nArticle\\nCAS\\nPubMed\\nPubMed Central\\nGoogle Scholar\\nMay, L. T. et al. Phosphorylation of secreted forms of human beta 2-interferon/hepatocyte stimulating factor/interleukin-6.\\nBiochem. Biophys. Res. Commun.\\n152\\n, 1144–1150 (1988).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nHirano, T. et al. Purification to homogeneity and characterization of human B-cell differentiation factor (BCDF or BSFp-2).\\nProc. Natl Acad. Sci. USA\\n82\\n, 5490–5494 (1985).\\nArticle\\nADS\\nCAS\\nPubMed\\nPubMed Central\\nGoogle Scholar\\nMay, L. T., Santhanam, U. & Sehgal, P. B. On the multimeric nature of natural human interleukin-6.\\nJ. Biol. Chem.\\n266\\n, 9950–9955 (1991).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nMereiter, S., Balmana, M., Campos, D., Gomes, J. & Reis, C. A. Glycosylation in the era of cancer-targeted therapy: where are we heading?\\nCancer Cell\\n36\\n, 6–16 (2019).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nParekh, R. B., Dwek, R. A., Rademacher, T. W., Opdenakker, G. & Van Damme, J. Glycosylation of interleukin-6 purified from normal human blood mononuclear cells.\\nEur. J. Biochem.\\n203\\n, 135–141 (1992).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nDela Cruz, C. S. et al. N-linked glycosylation is required for optimal function of Kaposi’s sarcoma herpesvirus-encoded, but not cellular, interleukin 6.\\nJ. Exp. Med.\\n199\\n, 503–514 (2004).\\nArticle\\nGoogle Scholar\\nThomas, D., Rathinavel, A. K. & Radhakrishnan, P. Altered glycosylation in cancer: a promising target for biomarkers and therapeutics.\\nBiochimica. et biophysica. Acta Rev. Cancer\\n1875\\n, 188464 (2021).\\nArticle\\nCAS\\nGoogle Scholar\\nPinho, S. S. & Reis, C. A. Glycosylation in cancer: mechanisms and clinical implications.\\nNat. Rev. Cancer\\n15\\n, 540–555 (2015).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nYeh, H. H., Lai, W. W., Chen, H. H., Liu, H. S. & Su, W. C. Autocrine IL-6-induced Stat3 activation contributes to the pathogenesis of lung adenocarcinoma and malignant pleural effusion.\\nOncogene\\n25\\n, 4300–4309 (2006).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nRoth, J. Protein N-glycosylation along the secretory pathway: relationship to organelle topography and function, protein quality control, and cell interactions.\\nChem. Rev.\\n102\\n, 285–303 (2002).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nBard, F. & Chia, J. Cracking the glycome encoder: signaling, trafficking, and glycosylation.\\nTrends Cell Biol.\\n26\\n, 379–388 (2016).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nSchjoldager, K. T., Narimatsu, Y., Joshi, H. J. & Clausen, H. Global view of human protein glycosylation pathways and functions.\\nNat. Rev. Mol. Cell Biol.\\n21\\n, 729–749 (2020).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nSubramanian, A. et al. Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles.\\nProc. Natl Acad. Sci. USA\\n102\\n, 15545–15550 (2005).\\nArticle\\nADS\\nCAS\\nPubMed\\nPubMed Central\\nGoogle Scholar\\nMootha, V. K. et al. PGC-1alpha-responsive genes involved in oxidative phosphorylation are coordinately downregulated in human diabetes.\\nNat. Genet.\\n34\\n, 267–273 (2003).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nOkayama, H. et al. Identification of genes upregulated in ALK-positive and EGFR/KRAS/ALK-negative lung adenocarcinomas.\\nCancer Res.\\n72\\n, 100–111 (2012).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nLopez-Sambrooks, C. et al. Oligosaccharyltransferase inhibition induces senescence in RTK-driven tumor cells.\\nNat. Chem. Biol.\\n12\\n, 1023–1030 (2016).\\nArticle\\nCAS\\nPubMed\\nPubMed Central\\nGoogle Scholar\\nBlom, N., Sicheritz-Ponten, T., Gupta, R., Gammeltoft, S. & Brunak, S. Prediction of post-translational glycosylation and phosphorylation of proteins from the amino acid sequence.\\nProteomics\\n4\\n, 1633–1649 (2004).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nYamaguchi, K. et al. Effects of site-directed removal of N-glycosylation sites in human erythropoietin on its production and biological properties.\\nJ. Biol. Chem\\n.\\n266\\n, 20434–20439 (1991).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nBobby, R. et al. Functional implications of large backbone amplitude motions of the glycoprotein 130-binding epitope of interleukin-6.\\nFEBS J.\\n281\\n, 2471–2483 (2014).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nXiong, H. et al. Roles of STAT3 and ZEB1 proteins in E-cadherin down-regulation and human colorectal cancer epithelial-mesenchymal transition.\\nJ. Biol. Chem.\\n287\\n, 5819–5832 (2012).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nWei, W. et al. STAT3 signaling is activated preferentially in tumor-initiating cells in claudin-low models of human breast cancer.\\nStem cells\\n32\\n, 2571–2582 (2014).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nYao, C. et al. IGF/STAT3/NANOG/Slug signaling axis simultaneously controls epithelial-mesenchymal transition and stemness maintenance in colorectal cancer.\\nStem Cells\\n34\\n, 820–831 (2016).\\nArticle\\nPubMed\\nGoogle Scholar\\nNa, Y. R., Lee, J. S., Lee, S. J. & Seok, S. H. Interleukin-6-induced twist and N-cadherin enhance melanoma cell metastasis.\\nMelanoma Res.\\n23\\n, 434–443 (2013).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nGallucci, R. M., Lee, E. G. & Tomasek, J. J. IL-6 modulates alpha-smooth muscle actin expression in dermal fibroblasts from IL-6-deficient mice.\\nJ. Invest. Dermatol\\n.\\n126\\n, 561–568 (2006).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nRomera-Hernandez, M. et al. Yap1-driven intestinal repair is controlled by group 3 innate lymphoid cells.\\nCell Rep.\\n30\\n, 37–45.e33 (2020).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nTsuji, T. et al. YAP1 mediates survival of ALK-rearranged lung cancer cells treated with alectinib via pro-apoptotic protein regulation.\\nNat. Commun.\\n11\\n, 74 (2020).\\nArticle\\nADS\\nCAS\\nPubMed\\nPubMed Central\\nGoogle Scholar\\nTakahashi, A. et al. Ankyrin repeat domain 1 overexpression is associated with common resistance to afatinib and osimertinib in EGFR-mutant lung cancer.\\nSci. Rep.\\n8\\n, 14896 (2018).\\nArticle\\nADS\\nPubMed\\nPubMed Central\\nGoogle Scholar\\nZheng, S. et al. SynergyFinder plus: toward better interpretation and annotation of drug combination screening datasets.\\nGenom. Proteom. Bioinform.\\n20\\n, 587–596 (2022).\\nArticle\\nGoogle Scholar\\nRoper, N. et al. Clonal evolution and heterogeneity of osimertinib acquired resistance mechanisms in EGFR mutant lung cancer.\\nCell Rep. Med.\\n1\\n, 10007 (2020).\\nGoogle Scholar\\nBarbie, D. A. et al. Systematic RNA interference reveals that oncogenic KRAS-driven cancers require TBK1.\\nNature\\n462\\n, 108–112 (2009).\\nArticle\\nADS\\nCAS\\nPubMed\\nPubMed Central\\nGoogle Scholar\\nHeinz, M. C. et al. Liver colonization by colorectal cancer metastases requires YAP-controlled plasticity at the micrometastatic stage.\\nCancer Res.\\n82\\n, 1953–1968 (2022).\\nArticle\\nCAS\\nPubMed\\nPubMed Central\\nGoogle Scholar\\nHamanaka, N. et al. YES1 is a targetable oncogene in cancers harboring YES1 gene amplification.\\nCancer Res.\\n79\\n, 5734–5745 (2019).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nLopez Sambrooks, C. et al. Oligosaccharyltransferase inhibition overcomes therapeutic resistance to EGFR tyrosine kinase inhibitors.\\nCancer Res.\\n78\\n, 5094–5106 (2018).\\nArticle\\nPubMed\\nGoogle Scholar\\nChan, L. C. et al. IL-6/JAK1 pathway drives PD-L1 Y112 phosphorylation to promote cancer immune evasion.\\nJ. Clin. Invest.\\n129\\n, 3324–3338 (2019).\\nArticle\\nPubMed\\nPubMed Central\\nGoogle Scholar\\nMiller, M. R. et al. Downregulation of dystroglycan glycosyltransferases LARGE2 and ISPD associate with increased mortality in clear cell renal cell carcinoma.\\nMol. Cancer\\n14\\n, 141 (2015).\\nArticle\\nPubMed\\nPubMed Central\\nGoogle Scholar\\nChugh, S., Meza, J., Sheinin, Y. M., Ponnusamy, M. P. & Batra, S. K. Loss of N-acetylgalactosaminyltransferase 3 in poorly differentiated pancreatic cancer: augmented aggressiveness and aberrant ErbB family glycosylation.\\nBr. J. Cancer\\n114\\n, 1376–1386 (2016).\\nArticle\\nCAS\\nPubMed\\nPubMed Central\\nGoogle Scholar\\nPeng, C. et al. Regulation of the Hippo-YAP pathway by glucose sensor O-GlcNAcylation.\\nMol. Cell\\n68\\n, 591–604.e595 (2017).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nHung, T. L. et al. Clinical evaluation of HER-2/neu protein in malignant pleural effusion-associated lung adenocarcinoma and as a tumor marker in pleural effusion diagnosis.\\nClin. Cancer Res.\\n9\\n, 2605–2612 (2003).\\nCAS\\nPubMed\\nGoogle Scholar\\nChu, Y. W. et al. Selection of invasive and metastatic subpopulations from a human lung adenocarcinoma cell line.\\nAm. J. Respir. Cell Mol. Biol.\\n17\\n, 353–360 (1997).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nWisniewski, J. R., Zougman, A., Nagaraj, N. & Mann, M. Universal sample preparation method for proteome analysis.\\nNat. Methods\\n6\\n, 359–362 (2009).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nBern, M., Kil, Y. J. & Becker, C. Byonic: advanced peptide and protein identification software.\\nCurr. Protocol. Bioinform.\\n13\\n, 13 20 11–13 20 14 (2012).\\nGoogle Scholar\\nDownload references\\n', 'repos': 'bigd.big.ac.cn, ega-archive.org, massive.ucsd.edu, www.ebi.ac.uk, www.iprox.cn, proteomecentral.proteomexchange.org, www.proteomexchange.org, pdc.cancer.gov, gdc.cancer.gov, cancerimagingarchive.net, imaging.datacommons.cancer.gov, repository.jpostdb.org, doi.org, geo, dbgap, sra, ArrayExpress'}, and items: [{'role': 'system', 'content': \"You are a specialized assistant that extracts dataset references from the content of scientific papers. You must output a JSON array of objects, where each object has the following keys: 'dataset_identifier', 'data_repository', and 'dataset_webpage'. Follow the structure of the provided examples exactly. We will not wrap the json codes in JSON markers.\"}, {'role': 'user', 'content': 'Extract dataset references based on the examples below:\\n\\nExample 1:\\nContent: \\'The study used dataset EGAS00893672193, which is available at the European Genome Archive.\\'\\nResponse:\\n[\\n    [\\n        \"dataset_identifier\": \"EGAS00893672193\",\\n        \"data_repository\": \"European Genome Archive\",\\n        \"dataset_webpage\": \"https://ega-archive.org/studies/EGAS00893672193\"\\n    ]\\n]\\n\\nExample 2:\\nContent: \\'Proteomics data was obtained from PRIDE, accession PXD029921.\\'\\nResponse:\\n[\\n    [\\n        \"dataset_identifier\": \"PXD029921\",\\n        \"data_repository\": \"PRIDE\",\\n        \"dataset_webpage\": \"https://www.ebi.ac.uk/pride/archive/projects/PXD029921\"\\n    ]\\n]\\n\\nExample 3:\\nContent: \\'The repository dbGaP hosts the dataset phs011366.v1.p1 at this location.\\'\\nResponse:\\n[\\n    [\\n        \"dataset_identifier\": \"phs011366.v1.p1\",\\n        \"data_repository\": \"dbGaP\",\\n        \"dataset_webpage\": \"https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001366.v1.p1\"\\n    ]\\n]\\n\\nNow process the following content:\\n\\nContent: {content}.'}]\n",
      "base_parser.py - line 1108 - INFO - Expected string but got list. Converting list to string.\n",
      "base_parser.py - line 1115 - DEBUG - Counting tokens for model: gemini-2.0-flash, prompt length: 20455 char\n",
      "base_parser.py - line 1124 - DEBUG - Rough estimate of token count for Gemini model 'gemini-2.0-flash': 5113\n",
      "base_parser.py - line 444 - INFO - Prompt messages total length: 5113 tokens\n",
      "base_parser.py - line 445 - DEBUG - Prompt messages: [{'role': 'system', 'content': \"You are a specialized assistant that extracts dataset references from the content of scientific papers. You must output a JSON array of objects, where each object has the following keys: 'dataset_identifier', 'data_repository', and 'dataset_webpage'. Follow the structure of the provided examples exactly. We will not wrap the json codes in JSON markers.\"}, {'role': 'user', 'content': 'Extract dataset references based on the examples below:\\n\\nExample 1:\\nContent: \\'The study used dataset EGAS00893672193, which is available at the European Genome Archive.\\'\\nResponse:\\n[\\n    [\\n        \"dataset_identifier\": \"EGAS00893672193\",\\n        \"data_repository\": \"European Genome Archive\",\\n        \"dataset_webpage\": \"https://ega-archive.org/studies/EGAS00893672193\"\\n    ]\\n]\\n\\nExample 2:\\nContent: \\'Proteomics data was obtained from PRIDE, accession PXD029921.\\'\\nResponse:\\n[\\n    [\\n        \"dataset_identifier\": \"PXD029921\",\\n        \"data_repository\": \"PRIDE\",\\n        \"dataset_webpage\": \"https://www.ebi.ac.uk/pride/archive/projects/PXD029921\"\\n    ]\\n]\\n\\nExample 3:\\nContent: \\'The repository dbGaP hosts the dataset phs011366.v1.p1 at this location.\\'\\nResponse:\\n[\\n    [\\n        \"dataset_identifier\": \"phs011366.v1.p1\",\\n        \"data_repository\": \"dbGaP\",\\n        \"dataset_webpage\": \"https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001366.v1.p1\"\\n    ]\\n]\\n\\nNow process the following content:\\n\\nContent: Source data\\nSource data\\nSupplementary information\\nSupplementary Information\\nPeer Review File\\nReporting Summary\\nData availability\\nThe RNA microarray datasets generated in this study were deposited into the National Center for Biotechnology Information (NCBI) Gene Expression Omnibus (GEO) database under accession number GSE269782 and are available at the following URL:\\nhttps://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE269782\\n. The RNAseq and microarray datasets of human lung adenocarcinoma demonstrated in Figs.\\n1e\\n,\\n6h\\n, and Supplementary Fig.\\n21\\nwere downloaded from NCBI’s GEO:\\nGSE31210\\n;\\nGSE106765\\n;\\nGSE60189\\n;\\nGSE59239\\n;\\nGSE122005\\n;\\nGSE38121\\n;\\nGSE71587\\n;\\nGSE37699\\n. Datasets of human lung adenocarcinoma demonstrated in Fig.\\n8d\\nand Supplementary Fig.\\n26\\nwere from the supplementary information attached to the article of Roper N. et al.\\n59\\n. For glycosylation analysis, protein mass spectrometry raw data and analytical tool data have updated in a repository via ProteomeXchange (\\nhttps://www.proteomexchange.org\\n) with project accession: PXD051771.\\nSource data\\nare provided with this paper.\\nAuthor information\\nAuthors and Affiliations\\nDepartment of Oncology, National Cheng Kung University Hospital, College of Medicine, National Cheng Kung University, Tainan, Taiwan\\nChun-Hua Hung, Shang-Yin Wu & Wu-Chou Su\\nCenter of Applied Nanomedicine, National Cheng Kung University, Tainan, Taiwan\\nChun-Hua Hung, Hsuan-Heng Yeh & Wu-Chou Su\\nInstitute of Basic Medical Sciences, College of Medicine, National Cheng Kung University, Tainan, Taiwan\\nChun-Hua Hung & Wu-Chou Su\\nDepartment of Biomedical Engineering, College of Engineering, National Cheng Kung University, Tainan, Taiwan\\nChun-Hua Hung\\nInstitute of Clinical Medicine, College of Medicine, National Cheng Kung University, Tainan, Taiwan\\nShang-Yin Wu, Chien-Chung Lin & Wu-Chou Su\\nInstitute of Biological Chemistry, Academia Sinica, Taipei, Taiwan\\nCheng-I Daniel Yao & Chun-Hung Lin\\nDepartment of Internal Medicine, National Cheng Kung University Hospital, College of Medicine, National Cheng Kung University, Tainan, Taiwan\\nChien-Chung Lin\\nDepartment of Biochemistry and Molecular Biology, College of Medicine, National Cheng Kung University, Tainan, Taiwan\\nChien-Chung Lin\\nDepartment of Pathology, National Cheng Kung University Hospital, College of Medicine, National Cheng Kung University, Tainan, Taiwan\\nChang-Yao Chu\\nInstitute of Molecular Medicine, College of Medicine, National Cheng Kung University, Tainan, Taiwan\\nTzu-Yu Huang & Wu-Chou Su\\nDepartment of Pharmacology, College of Medicine, National Cheng Kung University, Tainan, Taiwan\\nMeng-Ru Shen\\nDepartment of Obstetrics and Gynecology, National Cheng Kung University Hospital, College of Medicine, National Cheng Kung University, Tainan, Taiwan\\nMeng-Ru Shen\\nDepartment of Chemistry and Institute of Biochemical Sciences, National Taiwan University, Taipei, Taiwan\\nChun-Hung Lin\\nAuthors\\nChun-Hua Hung\\nView author publications\\nSearch author on:\\nPubMed\\nGoogle Scholar\\nShang-Yin Wu\\nView author publications\\nSearch author on:\\nPubMed\\nGoogle Scholar\\nCheng-I Daniel Yao\\nView author publications\\nSearch author on:\\nPubMed\\nGoogle Scholar\\nHsuan-Heng Yeh\\nView author publications\\nSearch author on:\\nPubMed\\nGoogle Scholar\\nChien-Chung Lin\\nView author publications\\nSearch author on:\\nPubMed\\nGoogle Scholar\\nChang-Yao Chu\\nView author publications\\nSearch author on:\\nPubMed\\nGoogle Scholar\\nTzu-Yu Huang\\nView author publications\\nSearch author on:\\nPubMed\\nGoogle Scholar\\nMeng-Ru Shen\\nView author publications\\nSearch author on:\\nPubMed\\nGoogle Scholar\\nChun-Hung Lin\\nView author publications\\nSearch author on:\\nPubMed\\nGoogle Scholar\\nWu-Chou Su\\nView author publications\\nSearch author on:\\nPubMed\\nGoogle Scholar\\nContributions\\nChun-Hua Hung: Conceptualization, data curation, software, formal analysis, validation, investigation, visualization, methodology, writing-original draft. Shang-Yin Wu: Software, formal analysis, investigation. Cheng-I Daniel Yao: Formal analysis, methodology. Hsuan-Heng Yeh: Formal analysis, methodology. Chien-Chung Lin: Resources, formal analysis. Chang-Yao Chu: Formal analysis, methodology. Tzu-Yu Huang: Formal analysis, methodology. Meng-Ru Shen: Formal analysis, methodology, funding acquisition. Chun-Hung Lin: Formal analysis, methodology, funding acquisition. Wu-Chou Su*: Conceptualization, resources, supervision, funding acquisition, writing-original draft, project administration.\\nCorresponding author\\nCorrespondence to\\nWu-Chou Su\\n.\\nReferences\\nYu, H., Lee, H., Herrmann, A., Buettner, R. & Jove, R. Revisiting STAT3 signalling in cancer: new and unexpected biological functions.\\nNat. Rev. Cancer\\n14\\n, 736–746 (2014).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nJones, S. A. & Jenkins, B. J. Recent insights into targeting the IL-6 cytokine family in inflammatory diseases and cancer.\\nNat. Rev. Immunol.\\n18\\n, 773–789 (2018).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nRose-John, S. The soluble interleukin 6 receptor: advanced therapeutic options in inflammation.\\nClin. Pharmacol. Therapeut.\\n102\\n, 591–598 (2017).\\nArticle\\nCAS\\nGoogle Scholar\\nTrikha, M., Corringham, R., Klein, B. & Rossi, J. F. Targeted anti-interleukin-6 monoclonal antibody therapy for cancer: a review of the rationale and clinical evidence.\\nClin. Cancer Res.\\n9\\n, 4653–4665 (2003).\\nCAS\\nPubMed\\nPubMed Central\\nGoogle Scholar\\nHunter, C. A. & Jones, S. A. IL-6 as a keystone cytokine in health and disease.\\nNat. Immunol\\n.\\n16\\n, 448–457 (2015).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nGarbers, C., Heink, S., Korn, T. & Rose-John, S. Interleukin-6: designing specific therapeutics for a complex cytokine.\\nNat. Rev. Drug Discov.\\n17\\n, 395–412 (2018).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nMurakami, M., Kamimura, D. & Hirano, T. Pleiotropy and specificity: insights from the interleukin 6 family of cytokines.\\nImmunity\\n50\\n, 812–831 (2019).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nDuan, Z. et al. Signal transducers and activators of transcription 3 pathway activation in drug-resistant ovarian cancer.\\nClin. Cancer Res.\\n12\\n, 5055–5063 (2006).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nHuang, W. L. et al. Signal transducer and activator of transcription 3 activation up-regulates interleukin-6 autocrine production: a biochemical and genetic study of established cancer cell lines and clinical isolated human cancer cells.\\nMol. Cancer\\n9\\n, 309 (2010).\\nArticle\\nCAS\\nPubMed\\nPubMed Central\\nGoogle Scholar\\nLee, H. J. et al. Drug resistance via feedback activation of Stat3 in oncogene-addicted cancer cells.\\nCancer Cell\\n26\\n, 207–221 (2014).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nRokavec, M. et al. IL-6R/STAT3/miR-34a feedback loop promotes EMT-mediated colorectal cancer invasion and metastasis.\\nJ. Clin. Invest.\\n124\\n, 1853–1867 (2014).\\nArticle\\nCAS\\nPubMed\\nPubMed Central\\nGoogle Scholar\\nTaniguchi, K. et al. A gp130-Src-YAP module links inflammation to epithelial regeneration.\\nNature\\n519\\n, 57–62 (2015).\\nArticle\\nADS\\nCAS\\nPubMed\\nPubMed Central\\nGoogle Scholar\\nQuintanal-Villalonga, A. et al. Lineage plasticity in cancer: a shared pathway of therapeutic resistance.\\nNat. Rev. Clin. Oncol.\\n17\\n, 360–371 (2020).\\nArticle\\nPubMed\\nPubMed Central\\nGoogle Scholar\\nPaez, J. G. et al. EGFR mutations in lung cancer: correlation with clinical response to gefitinib therapy.\\nScience\\n304\\n, 1497–1500 (2004).\\nArticle\\nADS\\nCAS\\nPubMed\\nGoogle Scholar\\nRamalingam, S. S. et al. Overall survival with osimertinib in untreated, EGFR-mutated advanced NSCLC.\\nN. Engl. J. Med.\\n382\\n, 41–50 (2020).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nOxnard, G. R. et al. Assessment of resistance mechanisms and clinical implications in patients with EGFR T790M-positive lung cancer and acquired resistance to osimertinib.\\nJAMA Oncol.\\n4\\n, 1527–1534 (2018).\\nArticle\\nPubMed\\nGoogle Scholar\\nYang, Z. et al. Investigating novel resistance mechanisms to third-generation EGFR tyrosine kinase inhibitor osimertinib in non-small cell lung cancer patients.\\nClin. Cancer Res.\\n24\\n, 3097–3107 (2018).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nSequist, L. V. et al. Genotypic and histological evolution of lung cancers acquiring resistance to EGFR inhibitors.\\nSci. Transl. Med.\\n3\\n, 75ra26 (2011).\\nArticle\\nPubMed\\nPubMed Central\\nGoogle Scholar\\nLambert, A. W. & Weinberg, R. A. Linking EMT programmes to normal and neoplastic epithelial stem cells.\\nNat. Rev. Cancer\\n21\\n, 325–338 (2021).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nWilliams, E. D., Gao, D., Redfern, A. & Thompson, E. W. Controversies around epithelial-mesenchymal plasticity in cancer metastasis.\\nNat. Rev. Cancer\\n19\\n, 716–732 (2019).\\nArticle\\nCAS\\nPubMed\\nPubMed Central\\nGoogle Scholar\\nYao, Z. et al. TGF-beta IL-6 axis mediates selective and adaptive mechanisms of resistance to molecular targeted therapy in lung cancer.\\nProc. Natl Acad. Sci. USA\\n107\\n, 15535–15540 (2010).\\nArticle\\nADS\\nCAS\\nPubMed\\nPubMed Central\\nGoogle Scholar\\nKim, S. M. et al. Activation of IL-6R/JAK1/STAT signaling induces de novo resistance to irreversible EGFR inhibitors in non-small cell lung cancer with T790M resistance mutation.\\nMol. Cancer Ther.\\n11\\n, 2254–2264 (2012).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nKorkaya, H., Liu, S. & Wicha, M. S. Regulation of cancer stem cells by cytokine networks: attacking cancer’s inflammatory roots.\\nClin. Cancer Res.\\n17\\n, 6125–6129 (2011).\\nArticle\\nCAS\\nPubMed\\nPubMed Central\\nGoogle Scholar\\nSenescence-Associated Pro-inflammatory Cytokines, VernotJ. P. and tumor cell plasticity.\\nFront. Mol. Biosci.\\n7\\n, 63 (2020).\\nADS\\nGoogle Scholar\\nKurppa, K. J. et al. Treatment-induced tumor dormancy through YAP-mediated transcriptional reprogramming of the apoptotic pathway.\\nCancer Cell\\n37\\n, 104–122.e112 (2020).\\nArticle\\nCAS\\nPubMed\\nPubMed Central\\nGoogle Scholar\\nFerone, G. et al. SOX2 is the determining oncogenic switch in promoting lung squamous cell carcinoma from different cells of origin.\\nCancer Cell\\n30\\n, 519–532 (2016).\\nArticle\\nCAS\\nPubMed\\nPubMed Central\\nGoogle Scholar\\nBoumahdi, S. & de Sauvage, F. J. The great escape: tumour cell plasticity in resistance to targeted therapy.\\nNat. Rev. Drug Discov.\\n19\\n, 39–56 (2020).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nMosteiro, L. et al. Tissue damage and senescence provide critical signals for cellular reprogramming in vivo.\\nScience\\n354\\n, 6315 (2016).\\nArticle\\nGoogle Scholar\\nBora-Singhal, N. et al. YAP1 regulates OCT4 activity and SOX2 expression to facilitate self-renewal and vascular mimicry of stem-like cells.\\nStem cells\\n33\\n, 1705–1718 (2015).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nPoupart, P. et al. B cell growth modulating and differentiating activity of recombinant human 26-kd protein (BSF-2, HuIFN-beta 2, HPGF).\\nEMBO J\\n6\\n, 1219–1224 (1987).\\nArticle\\nCAS\\nPubMed\\nPubMed Central\\nGoogle Scholar\\nMay, L. T. et al. Phosphorylation of secreted forms of human beta 2-interferon/hepatocyte stimulating factor/interleukin-6.\\nBiochem. Biophys. Res. Commun.\\n152\\n, 1144–1150 (1988).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nHirano, T. et al. Purification to homogeneity and characterization of human B-cell differentiation factor (BCDF or BSFp-2).\\nProc. Natl Acad. Sci. USA\\n82\\n, 5490–5494 (1985).\\nArticle\\nADS\\nCAS\\nPubMed\\nPubMed Central\\nGoogle Scholar\\nMay, L. T., Santhanam, U. & Sehgal, P. B. On the multimeric nature of natural human interleukin-6.\\nJ. Biol. Chem.\\n266\\n, 9950–9955 (1991).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nMereiter, S., Balmana, M., Campos, D., Gomes, J. & Reis, C. A. Glycosylation in the era of cancer-targeted therapy: where are we heading?\\nCancer Cell\\n36\\n, 6–16 (2019).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nParekh, R. B., Dwek, R. A., Rademacher, T. W., Opdenakker, G. & Van Damme, J. Glycosylation of interleukin-6 purified from normal human blood mononuclear cells.\\nEur. J. Biochem.\\n203\\n, 135–141 (1992).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nDela Cruz, C. S. et al. N-linked glycosylation is required for optimal function of Kaposi’s sarcoma herpesvirus-encoded, but not cellular, interleukin 6.\\nJ. Exp. Med.\\n199\\n, 503–514 (2004).\\nArticle\\nGoogle Scholar\\nThomas, D., Rathinavel, A. K. & Radhakrishnan, P. Altered glycosylation in cancer: a promising target for biomarkers and therapeutics.\\nBiochimica. et biophysica. Acta Rev. Cancer\\n1875\\n, 188464 (2021).\\nArticle\\nCAS\\nGoogle Scholar\\nPinho, S. S. & Reis, C. A. Glycosylation in cancer: mechanisms and clinical implications.\\nNat. Rev. Cancer\\n15\\n, 540–555 (2015).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nYeh, H. H., Lai, W. W., Chen, H. H., Liu, H. S. & Su, W. C. Autocrine IL-6-induced Stat3 activation contributes to the pathogenesis of lung adenocarcinoma and malignant pleural effusion.\\nOncogene\\n25\\n, 4300–4309 (2006).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nRoth, J. Protein N-glycosylation along the secretory pathway: relationship to organelle topography and function, protein quality control, and cell interactions.\\nChem. Rev.\\n102\\n, 285–303 (2002).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nBard, F. & Chia, J. Cracking the glycome encoder: signaling, trafficking, and glycosylation.\\nTrends Cell Biol.\\n26\\n, 379–388 (2016).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nSchjoldager, K. T., Narimatsu, Y., Joshi, H. J. & Clausen, H. Global view of human protein glycosylation pathways and functions.\\nNat. Rev. Mol. Cell Biol.\\n21\\n, 729–749 (2020).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nSubramanian, A. et al. Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles.\\nProc. Natl Acad. Sci. USA\\n102\\n, 15545–15550 (2005).\\nArticle\\nADS\\nCAS\\nPubMed\\nPubMed Central\\nGoogle Scholar\\nMootha, V. K. et al. PGC-1alpha-responsive genes involved in oxidative phosphorylation are coordinately downregulated in human diabetes.\\nNat. Genet.\\n34\\n, 267–273 (2003).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nOkayama, H. et al. Identification of genes upregulated in ALK-positive and EGFR/KRAS/ALK-negative lung adenocarcinomas.\\nCancer Res.\\n72\\n, 100–111 (2012).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nLopez-Sambrooks, C. et al. Oligosaccharyltransferase inhibition induces senescence in RTK-driven tumor cells.\\nNat. Chem. Biol.\\n12\\n, 1023–1030 (2016).\\nArticle\\nCAS\\nPubMed\\nPubMed Central\\nGoogle Scholar\\nBlom, N., Sicheritz-Ponten, T., Gupta, R., Gammeltoft, S. & Brunak, S. Prediction of post-translational glycosylation and phosphorylation of proteins from the amino acid sequence.\\nProteomics\\n4\\n, 1633–1649 (2004).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nYamaguchi, K. et al. Effects of site-directed removal of N-glycosylation sites in human erythropoietin on its production and biological properties.\\nJ. Biol. Chem\\n.\\n266\\n, 20434–20439 (1991).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nBobby, R. et al. Functional implications of large backbone amplitude motions of the glycoprotein 130-binding epitope of interleukin-6.\\nFEBS J.\\n281\\n, 2471–2483 (2014).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nXiong, H. et al. Roles of STAT3 and ZEB1 proteins in E-cadherin down-regulation and human colorectal cancer epithelial-mesenchymal transition.\\nJ. Biol. Chem.\\n287\\n, 5819–5832 (2012).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nWei, W. et al. STAT3 signaling is activated preferentially in tumor-initiating cells in claudin-low models of human breast cancer.\\nStem cells\\n32\\n, 2571–2582 (2014).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nYao, C. et al. IGF/STAT3/NANOG/Slug signaling axis simultaneously controls epithelial-mesenchymal transition and stemness maintenance in colorectal cancer.\\nStem Cells\\n34\\n, 820–831 (2016).\\nArticle\\nPubMed\\nGoogle Scholar\\nNa, Y. R., Lee, J. S., Lee, S. J. & Seok, S. H. Interleukin-6-induced twist and N-cadherin enhance melanoma cell metastasis.\\nMelanoma Res.\\n23\\n, 434–443 (2013).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nGallucci, R. M., Lee, E. G. & Tomasek, J. J. IL-6 modulates alpha-smooth muscle actin expression in dermal fibroblasts from IL-6-deficient mice.\\nJ. Invest. Dermatol\\n.\\n126\\n, 561–568 (2006).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nRomera-Hernandez, M. et al. Yap1-driven intestinal repair is controlled by group 3 innate lymphoid cells.\\nCell Rep.\\n30\\n, 37–45.e33 (2020).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nTsuji, T. et al. YAP1 mediates survival of ALK-rearranged lung cancer cells treated with alectinib via pro-apoptotic protein regulation.\\nNat. Commun.\\n11\\n, 74 (2020).\\nArticle\\nADS\\nCAS\\nPubMed\\nPubMed Central\\nGoogle Scholar\\nTakahashi, A. et al. Ankyrin repeat domain 1 overexpression is associated with common resistance to afatinib and osimertinib in EGFR-mutant lung cancer.\\nSci. Rep.\\n8\\n, 14896 (2018).\\nArticle\\nADS\\nPubMed\\nPubMed Central\\nGoogle Scholar\\nZheng, S. et al. SynergyFinder plus: toward better interpretation and annotation of drug combination screening datasets.\\nGenom. Proteom. Bioinform.\\n20\\n, 587–596 (2022).\\nArticle\\nGoogle Scholar\\nRoper, N. et al. Clonal evolution and heterogeneity of osimertinib acquired resistance mechanisms in EGFR mutant lung cancer.\\nCell Rep. Med.\\n1\\n, 10007 (2020).\\nGoogle Scholar\\nBarbie, D. A. et al. Systematic RNA interference reveals that oncogenic KRAS-driven cancers require TBK1.\\nNature\\n462\\n, 108–112 (2009).\\nArticle\\nADS\\nCAS\\nPubMed\\nPubMed Central\\nGoogle Scholar\\nHeinz, M. C. et al. Liver colonization by colorectal cancer metastases requires YAP-controlled plasticity at the micrometastatic stage.\\nCancer Res.\\n82\\n, 1953–1968 (2022).\\nArticle\\nCAS\\nPubMed\\nPubMed Central\\nGoogle Scholar\\nHamanaka, N. et al. YES1 is a targetable oncogene in cancers harboring YES1 gene amplification.\\nCancer Res.\\n79\\n, 5734–5745 (2019).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nLopez Sambrooks, C. et al. Oligosaccharyltransferase inhibition overcomes therapeutic resistance to EGFR tyrosine kinase inhibitors.\\nCancer Res.\\n78\\n, 5094–5106 (2018).\\nArticle\\nPubMed\\nGoogle Scholar\\nChan, L. C. et al. IL-6/JAK1 pathway drives PD-L1 Y112 phosphorylation to promote cancer immune evasion.\\nJ. Clin. Invest.\\n129\\n, 3324–3338 (2019).\\nArticle\\nPubMed\\nPubMed Central\\nGoogle Scholar\\nMiller, M. R. et al. Downregulation of dystroglycan glycosyltransferases LARGE2 and ISPD associate with increased mortality in clear cell renal cell carcinoma.\\nMol. Cancer\\n14\\n, 141 (2015).\\nArticle\\nPubMed\\nPubMed Central\\nGoogle Scholar\\nChugh, S., Meza, J., Sheinin, Y. M., Ponnusamy, M. P. & Batra, S. K. Loss of N-acetylgalactosaminyltransferase 3 in poorly differentiated pancreatic cancer: augmented aggressiveness and aberrant ErbB family glycosylation.\\nBr. J. Cancer\\n114\\n, 1376–1386 (2016).\\nArticle\\nCAS\\nPubMed\\nPubMed Central\\nGoogle Scholar\\nPeng, C. et al. Regulation of the Hippo-YAP pathway by glucose sensor O-GlcNAcylation.\\nMol. Cell\\n68\\n, 591–604.e595 (2017).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nHung, T. L. et al. Clinical evaluation of HER-2/neu protein in malignant pleural effusion-associated lung adenocarcinoma and as a tumor marker in pleural effusion diagnosis.\\nClin. Cancer Res.\\n9\\n, 2605–2612 (2003).\\nCAS\\nPubMed\\nGoogle Scholar\\nChu, Y. W. et al. Selection of invasive and metastatic subpopulations from a human lung adenocarcinoma cell line.\\nAm. J. Respir. Cell Mol. Biol.\\n17\\n, 353–360 (1997).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nWisniewski, J. R., Zougman, A., Nagaraj, N. & Mann, M. Universal sample preparation method for proteome analysis.\\nNat. Methods\\n6\\n, 359–362 (2009).\\nArticle\\nCAS\\nPubMed\\nGoogle Scholar\\nBern, M., Kil, Y. J. & Becker, C. Byonic: advanced peptide and protein identification software.\\nCurr. Protocol. Bioinform.\\n13\\n, 13 20 11–13 20 14 (2012).\\nGoogle Scholar\\nDownload references\\n.'}]\n",
      "prompt_manager.py - line 96 - DEBUG - Calculating checksum for content: prompt\n",
      "base_parser.py - line 450 - INFO - Prompt ID: gemini-2.0-flash-0-ff14d72780bfa11968c20480e7528540c5731a2d1b12e3406dae2c8b79987faa\n",
      "base_parser.py - line 1108 - INFO - Expected string but got list. Converting list to string.\n",
      "base_parser.py - line 1115 - DEBUG - Counting tokens for model: gemini-2.0-flash, prompt length: 20455 char\n",
      "base_parser.py - line 1124 - DEBUG - Rough estimate of token count for Gemini model 'gemini-2.0-flash': 5113\n",
      "base_parser.py - line 469 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages: 5113 tokens\n",
      "base_parser.py - line 540 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-51352a0d-7256-490f-b7e9-f715ad1884d4\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE269782\\\",\\n        \\\"data_repository\\\": \\\"NCBI Gene Expression Omnibus (GEO)\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE269782\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE31210\\\",\\n        \\\"data_repository\\\": \\\"NCBI GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE31210\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE106765\\\",\\n        \\\"data_repository\\\": \\\"NCBI GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE106765\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE60189\\\",\\n        \\\"data_repository\\\": \\\"NCBI GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60189\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE59239\\\",\\n        \\\"data_repository\\\": \\\"NCBI GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59239\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE122005\\\",\\n        \\\"data_repository\\\": \\\"NCBI GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122005\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE38121\\\",\\n        \\\"data_repository\\\": \\\"NCBI GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38121\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE71587\\\",\\n        \\\"data_repository\\\": \\\"NCBI GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE71587\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE37699\\\",\\n        \\\"data_repository\\\": \\\"NCBI GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE37699\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"PXD051771\\\",\\n        \\\"data_repository\\\": \\\"ProteomeXchange\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.proteomexchange.org/dataset/PXD051771\\\"\\n    }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE269782\\\",\\n        \\\"data_repository\\\": \\\"NCBI Gene Expression Omnibus (GEO)\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE269782\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE31210\\\",\\n        \\\"data_repository\\\": \\\"NCBI GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE31210\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE106765\\\",\\n        \\\"data_repository\\\": \\\"NCBI GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE106765\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE60189\\\",\\n        \\\"data_repository\\\": \\\"NCBI GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60189\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE59239\\\",\\n        \\\"data_repository\\\": \\\"NCBI GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59239\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE122005\\\",\\n        \\\"data_repository\\\": \\\"NCBI GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122005\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE38121\\\",\\n        \\\"data_repository\\\": \\\"NCBI GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38121\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE71587\\\",\\n        \\\"data_repository\\\": \\\"NCBI GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE71587\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"GSE37699\\\",\\n        \\\"data_repository\\\": \\\"NCBI GEO\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE37699\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"PXD051771\\\",\\n        \\\"data_repository\\\": \\\"ProteomeXchange\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.proteomexchange.org/dataset/PXD051771\\\"\\n    }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1751902241,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 6388,\n",
      "        \"completion_tokens\": 732,\n",
      "        \"total_tokens\": 7120,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\n",
      "base_parser.py - line 991 - INFO - Fetching metadata for 10 datasets\n",
      "base_parser.py - line 999 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\n",
      "base_parser.py - line 1010 - INFO - Raw accession ID: GSE269782\n",
      "base_parser.py - line 939 - INFO - Resolving data repository for: NCBI Gene Expression Omnibus (GEO)\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI Gene Expression Omnibus (GEO) == bigd.big.ac.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI Gene Expression Omnibus  GEO  == ega-archive.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI Gene Expression Omnibus  GEO  == massive.ucsd.edu\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI Gene Expression Omnibus  GEO  == www.ebi.ac.uk\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI Gene Expression Omnibus  GEO  == www.iprox.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI Gene Expression Omnibus  GEO  == proteomecentral.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI Gene Expression Omnibus  GEO  == www.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI Gene Expression Omnibus  GEO  == pdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI Gene Expression Omnibus  GEO  == gdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI Gene Expression Omnibus  GEO  == cancerimagingarchive.net\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI Gene Expression Omnibus  GEO  == imaging.datacommons.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI Gene Expression Omnibus  GEO  == repository.jpostdb.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI Gene Expression Omnibus  GEO  == doi.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI Gene Expression Omnibus  GEO  == geo\n",
      "base_parser.py - line 969 - INFO - Found partial match for NCBI Gene Expression Omnibus  GEO  in GEO\n",
      "base_parser.py - line 921 - INFO - Resolving accession ID for GSE269782 in geo\n",
      "base_parser.py - line 1024 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE269782\n",
      "base_parser.py - line 1025 - DEBUG - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\n",
      "base_parser.py - line 1050 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE269782\n",
      "base_parser.py - line 1057 - INFO - Adding access mode for dataset 1: Complex download\n",
      "base_parser.py - line 999 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\n",
      "base_parser.py - line 1010 - INFO - Raw accession ID: GSE31210\n",
      "base_parser.py - line 939 - INFO - Resolving data repository for: NCBI GEO\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == bigd.big.ac.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == ega-archive.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == massive.ucsd.edu\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == www.ebi.ac.uk\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == www.iprox.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == proteomecentral.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == www.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == pdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == gdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == cancerimagingarchive.net\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == imaging.datacommons.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == repository.jpostdb.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == doi.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == geo\n",
      "base_parser.py - line 969 - INFO - Found partial match for NCBI GEO in GEO\n",
      "base_parser.py - line 921 - INFO - Resolving accession ID for GSE31210 in geo\n",
      "base_parser.py - line 1024 - INFO - Processing dataset 2 with repo: geo and accession_id: GSE31210\n",
      "base_parser.py - line 1025 - DEBUG - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\n",
      "base_parser.py - line 1050 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE31210\n",
      "base_parser.py - line 1057 - INFO - Adding access mode for dataset 2: Complex download\n",
      "base_parser.py - line 999 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\n",
      "base_parser.py - line 1010 - INFO - Raw accession ID: GSE106765\n",
      "base_parser.py - line 939 - INFO - Resolving data repository for: NCBI GEO\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == bigd.big.ac.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == ega-archive.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == massive.ucsd.edu\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == www.ebi.ac.uk\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == www.iprox.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == proteomecentral.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == www.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == pdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == gdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == cancerimagingarchive.net\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == imaging.datacommons.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == repository.jpostdb.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == doi.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == geo\n",
      "base_parser.py - line 969 - INFO - Found partial match for NCBI GEO in GEO\n",
      "base_parser.py - line 921 - INFO - Resolving accession ID for GSE106765 in geo\n",
      "base_parser.py - line 1024 - INFO - Processing dataset 3 with repo: geo and accession_id: GSE106765\n",
      "base_parser.py - line 1025 - DEBUG - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\n",
      "base_parser.py - line 1050 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE106765\n",
      "base_parser.py - line 1057 - INFO - Adding access mode for dataset 3: Complex download\n",
      "base_parser.py - line 999 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\n",
      "base_parser.py - line 1010 - INFO - Raw accession ID: GSE60189\n",
      "base_parser.py - line 939 - INFO - Resolving data repository for: NCBI GEO\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == bigd.big.ac.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == ega-archive.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == massive.ucsd.edu\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == www.ebi.ac.uk\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == www.iprox.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == proteomecentral.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == www.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == pdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == gdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == cancerimagingarchive.net\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == imaging.datacommons.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == repository.jpostdb.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == doi.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == geo\n",
      "base_parser.py - line 969 - INFO - Found partial match for NCBI GEO in GEO\n",
      "base_parser.py - line 921 - INFO - Resolving accession ID for GSE60189 in geo\n",
      "base_parser.py - line 1024 - INFO - Processing dataset 4 with repo: geo and accession_id: GSE60189\n",
      "base_parser.py - line 1025 - DEBUG - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\n",
      "base_parser.py - line 1050 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60189\n",
      "base_parser.py - line 1057 - INFO - Adding access mode for dataset 4: Complex download\n",
      "base_parser.py - line 999 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\n",
      "base_parser.py - line 1010 - INFO - Raw accession ID: GSE59239\n",
      "base_parser.py - line 939 - INFO - Resolving data repository for: NCBI GEO\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == bigd.big.ac.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == ega-archive.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == massive.ucsd.edu\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == www.ebi.ac.uk\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == www.iprox.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == proteomecentral.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == www.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == pdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == gdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == cancerimagingarchive.net\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == imaging.datacommons.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == repository.jpostdb.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == doi.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == geo\n",
      "base_parser.py - line 969 - INFO - Found partial match for NCBI GEO in GEO\n",
      "base_parser.py - line 921 - INFO - Resolving accession ID for GSE59239 in geo\n",
      "base_parser.py - line 1024 - INFO - Processing dataset 5 with repo: geo and accession_id: GSE59239\n",
      "base_parser.py - line 1025 - DEBUG - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\n",
      "base_parser.py - line 1050 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59239\n",
      "base_parser.py - line 1057 - INFO - Adding access mode for dataset 5: Complex download\n",
      "base_parser.py - line 999 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\n",
      "base_parser.py - line 1010 - INFO - Raw accession ID: GSE122005\n",
      "base_parser.py - line 939 - INFO - Resolving data repository for: NCBI GEO\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == bigd.big.ac.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == ega-archive.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == massive.ucsd.edu\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == www.ebi.ac.uk\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == www.iprox.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == proteomecentral.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == www.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == pdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == gdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == cancerimagingarchive.net\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == imaging.datacommons.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == repository.jpostdb.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == doi.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == geo\n",
      "base_parser.py - line 969 - INFO - Found partial match for NCBI GEO in GEO\n",
      "base_parser.py - line 921 - INFO - Resolving accession ID for GSE122005 in geo\n",
      "base_parser.py - line 1024 - INFO - Processing dataset 6 with repo: geo and accession_id: GSE122005\n",
      "base_parser.py - line 1025 - DEBUG - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\n",
      "base_parser.py - line 1050 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122005\n",
      "base_parser.py - line 1057 - INFO - Adding access mode for dataset 6: Complex download\n",
      "base_parser.py - line 999 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\n",
      "base_parser.py - line 1010 - INFO - Raw accession ID: GSE38121\n",
      "base_parser.py - line 939 - INFO - Resolving data repository for: NCBI GEO\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == bigd.big.ac.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == ega-archive.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == massive.ucsd.edu\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == www.ebi.ac.uk\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == www.iprox.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == proteomecentral.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == www.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == pdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == gdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == cancerimagingarchive.net\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == imaging.datacommons.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == repository.jpostdb.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == doi.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == geo\n",
      "base_parser.py - line 969 - INFO - Found partial match for NCBI GEO in GEO\n",
      "base_parser.py - line 921 - INFO - Resolving accession ID for GSE38121 in geo\n",
      "base_parser.py - line 1024 - INFO - Processing dataset 7 with repo: geo and accession_id: GSE38121\n",
      "base_parser.py - line 1025 - DEBUG - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\n",
      "base_parser.py - line 1050 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38121\n",
      "base_parser.py - line 1057 - INFO - Adding access mode for dataset 7: Complex download\n",
      "base_parser.py - line 999 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\n",
      "base_parser.py - line 1010 - INFO - Raw accession ID: GSE71587\n",
      "base_parser.py - line 939 - INFO - Resolving data repository for: NCBI GEO\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == bigd.big.ac.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == ega-archive.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == massive.ucsd.edu\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == www.ebi.ac.uk\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == www.iprox.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == proteomecentral.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == www.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == pdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == gdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == cancerimagingarchive.net\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == imaging.datacommons.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == repository.jpostdb.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == doi.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == geo\n",
      "base_parser.py - line 969 - INFO - Found partial match for NCBI GEO in GEO\n",
      "base_parser.py - line 921 - INFO - Resolving accession ID for GSE71587 in geo\n",
      "base_parser.py - line 1024 - INFO - Processing dataset 8 with repo: geo and accession_id: GSE71587\n",
      "base_parser.py - line 1025 - DEBUG - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\n",
      "base_parser.py - line 1050 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE71587\n",
      "base_parser.py - line 1057 - INFO - Adding access mode for dataset 8: Complex download\n",
      "base_parser.py - line 999 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\n",
      "base_parser.py - line 1010 - INFO - Raw accession ID: GSE37699\n",
      "base_parser.py - line 939 - INFO - Resolving data repository for: NCBI GEO\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == bigd.big.ac.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == ega-archive.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == massive.ucsd.edu\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == www.ebi.ac.uk\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == www.iprox.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == proteomecentral.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == www.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == pdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == gdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == cancerimagingarchive.net\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == imaging.datacommons.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == repository.jpostdb.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == doi.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if NCBI GEO == geo\n",
      "base_parser.py - line 969 - INFO - Found partial match for NCBI GEO in GEO\n",
      "base_parser.py - line 921 - INFO - Resolving accession ID for GSE37699 in geo\n",
      "base_parser.py - line 1024 - INFO - Processing dataset 9 with repo: geo and accession_id: GSE37699\n",
      "base_parser.py - line 1025 - DEBUG - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\n",
      "base_parser.py - line 1050 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE37699\n",
      "base_parser.py - line 1057 - INFO - Adding access mode for dataset 9: Complex download\n",
      "base_parser.py - line 999 - INFO - Processing dataset 10 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\n",
      "base_parser.py - line 1010 - INFO - Raw accession ID: PXD051771\n",
      "base_parser.py - line 939 - INFO - Resolving data repository for: ProteomeXchange\n",
      "base_parser.py - line 952 - DEBUG - Checking if ProteomeXchange == bigd.big.ac.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if ProteomeXchange == ega-archive.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if ProteomeXchange == massive.ucsd.edu\n",
      "base_parser.py - line 952 - DEBUG - Checking if ProteomeXchange == www.ebi.ac.uk\n",
      "base_parser.py - line 952 - DEBUG - Checking if ProteomeXchange == www.iprox.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if ProteomeXchange == proteomecentral.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if ProteomeXchange == www.proteomexchange.org\n",
      "base_parser.py - line 963 - INFO - Found repo_name match for ProteomeXchange\n",
      "base_parser.py - line 921 - INFO - Resolving accession ID for PXD051771 in www.proteomexchange.org\n",
      "base_parser.py - line 1024 - INFO - Processing dataset 10 with repo: www.proteomexchange.org and accession_id: PXD051771\n",
      "base_parser.py - line 1025 - DEBUG - Processing dataset 10 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\n",
      "base_parser.py - line 1050 - INFO - Dataset page: https://www.proteomexchange.org/cgi/GetDataset?ID=PXD051771\n",
      "base_parser.py - line 1057 - INFO - Adding access mode for dataset 10: Complex download\n",
      "base_parser.py - line 1067 - INFO - Updated datasets len: 10\n",
      "html_parser.py - line 397 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 14, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_class', 'a_attr_data-track', 'a_attr_data-track-action',\n",
      "       'a_attr_data-test', 'a_attr_data-track-label', 'a_attr_href',\n",
      "       'a_attr_data-supp-info-image'],\n",
      "      dtype='object')\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(nan)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(nan)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(nan)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(nan)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(nan)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(nan)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(nan)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(nan)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(nan)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(nan)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(None)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(None)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(None)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(None)\n",
      "html_retriever.py - line 229 - INFO - Extracting publication title from HTML\n",
      "html_retriever.py - line 239 - WARNING - No publication title found in the HTML data.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "  dataset_identifier                     data_repository  \\\n",
       "0          GSE269782  NCBI Gene Expression Omnibus (GEO)   \n",
       "1           GSE31210                            NCBI GEO   \n",
       "2          GSE106765                            NCBI GEO   \n",
       "3           GSE60189                            NCBI GEO   \n",
       "4           GSE59239                            NCBI GEO   \n",
       "5          GSE122005                            NCBI GEO   \n",
       "6           GSE38121                            NCBI GEO   \n",
       "7           GSE71587                            NCBI GEO   \n",
       "8           GSE37699                            NCBI GEO   \n",
       "9          PXD051771                     ProteomeXchange   \n",
       "0                NaN                                 NaN   \n",
       "\n",
       "                                     dataset_webpage       access_mode  \\\n",
       "0  https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...  Complex download   \n",
       "1  https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...  Complex download   \n",
       "2  https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...  Complex download   \n",
       "3  https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...  Complex download   \n",
       "4  https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...  Complex download   \n",
       "5  https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...  Complex download   \n",
       "6  https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...  Complex download   \n",
       "7  https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...  Complex download   \n",
       "8  https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...  Complex download   \n",
       "9  https://www.proteomexchange.org/cgi/GetDataset...  Complex download   \n",
       "0                                                NaN               NaN   \n",
       "\n",
       "                                                link  \\\n",
       "0                                                NaN   \n",
       "1                                                NaN   \n",
       "2                                                NaN   \n",
       "3                                                NaN   \n",
       "4                                                NaN   \n",
       "5                                                NaN   \n",
       "6                                                NaN   \n",
       "7                                                NaN   \n",
       "8                                                NaN   \n",
       "9                                                NaN   \n",
       "0  https://static-content.springer.com/esm/art%3A...   \n",
       "\n",
       "                       title file_info description source_section  \\\n",
       "0                        NaN       NaN         NaN            NaN   \n",
       "1                        NaN       NaN         NaN            NaN   \n",
       "2                        NaN       NaN         NaN            NaN   \n",
       "3                        NaN       NaN         NaN            NaN   \n",
       "4                        NaN       NaN         NaN            NaN   \n",
       "5                        NaN       NaN         NaN            NaN   \n",
       "6                        NaN       NaN         NaN            NaN   \n",
       "7                        NaN       NaN         NaN            NaN   \n",
       "8                        NaN       NaN         NaN            NaN   \n",
       "9                        NaN       NaN         NaN            NaN   \n",
       "0  Supplementary Information       n/a         n/a         MOESM1   \n",
       "\n",
       "                   section_class  ... file_extension a_attr_class  \\\n",
       "0                            NaN  ...           None          NaN   \n",
       "1                            NaN  ...           None          NaN   \n",
       "2                            NaN  ...           None          NaN   \n",
       "3                            NaN  ...           None          NaN   \n",
       "4                            NaN  ...           None          NaN   \n",
       "5                            NaN  ...           None          NaN   \n",
       "6                            NaN  ...           None          NaN   \n",
       "7                            NaN  ...           None          NaN   \n",
       "8                            NaN  ...           None          NaN   \n",
       "9                            NaN  ...           None          NaN   \n",
       "0  c-article-supplementary__item  ...           None   print-link   \n",
       "\n",
       "  a_attr_data-track a_attr_data-track-action a_attr_data-test  \\\n",
       "0               NaN                      NaN              NaN   \n",
       "1               NaN                      NaN              NaN   \n",
       "2               NaN                      NaN              NaN   \n",
       "3               NaN                      NaN              NaN   \n",
       "4               NaN                      NaN              NaN   \n",
       "5               NaN                      NaN              NaN   \n",
       "6               NaN                      NaN              NaN   \n",
       "7               NaN                      NaN              NaN   \n",
       "8               NaN                      NaN              NaN   \n",
       "9               NaN                      NaN              NaN   \n",
       "0             click  view supplementary info   supp-info-link   \n",
       "\n",
       "     a_attr_data-track-label  \\\n",
       "0                        NaN   \n",
       "1                        NaN   \n",
       "2                        NaN   \n",
       "3                        NaN   \n",
       "4                        NaN   \n",
       "5                        NaN   \n",
       "6                        NaN   \n",
       "7                        NaN   \n",
       "8                        NaN   \n",
       "9                        NaN   \n",
       "0  supplementary information   \n",
       "\n",
       "                                         a_attr_href  \\\n",
       "0                                                NaN   \n",
       "1                                                NaN   \n",
       "2                                                NaN   \n",
       "3                                                NaN   \n",
       "4                                                NaN   \n",
       "5                                                NaN   \n",
       "6                                                NaN   \n",
       "7                                                NaN   \n",
       "8                                                NaN   \n",
       "9                                                NaN   \n",
       "0  https://static-content.springer.com/esm/art%3A...   \n",
       "\n",
       "  a_attr_data-supp-info-image  \\\n",
       "0                         NaN   \n",
       "1                         NaN   \n",
       "2                         NaN   \n",
       "3                         NaN   \n",
       "4                         NaN   \n",
       "5                         NaN   \n",
       "6                         NaN   \n",
       "7                         NaN   \n",
       "8                         NaN   \n",
       "9                         NaN   \n",
       "0                               \n",
       "\n",
       "                                          source_url   article_title  \n",
       "0  https://www.nature.com/articles/s41467-024-518...  No title found  \n",
       "1  https://www.nature.com/articles/s41467-024-518...  No title found  \n",
       "2  https://www.nature.com/articles/s41467-024-518...  No title found  \n",
       "3  https://www.nature.com/articles/s41467-024-518...  No title found  \n",
       "4  https://www.nature.com/articles/s41467-024-518...  No title found  \n",
       "5  https://www.nature.com/articles/s41467-024-518...  No title found  \n",
       "6  https://www.nature.com/articles/s41467-024-518...  No title found  \n",
       "7  https://www.nature.com/articles/s41467-024-518...  No title found  \n",
       "8  https://www.nature.com/articles/s41467-024-518...  No title found  \n",
       "9  https://www.nature.com/articles/s41467-024-518...  No title found  \n",
       "0  https://www.nature.com/articles/s41467-024-518...  No title found  \n",
       "\n",
       "[11 rows x 21 columns]"
      ],
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>dataset_identifier</th>\n",
       "      <th>data_repository</th>\n",
       "      <th>dataset_webpage</th>\n",
       "      <th>access_mode</th>\n",
       "      <th>link</th>\n",
       "      <th>title</th>\n",
       "      <th>file_info</th>\n",
       "      <th>description</th>\n",
       "      <th>source_section</th>\n",
       "      <th>section_class</th>\n",
       "      <th>...</th>\n",
       "      <th>file_extension</th>\n",
       "      <th>a_attr_class</th>\n",
       "      <th>a_attr_data-track</th>\n",
       "      <th>a_attr_data-track-action</th>\n",
       "      <th>a_attr_data-test</th>\n",
       "      <th>a_attr_data-track-label</th>\n",
       "      <th>a_attr_href</th>\n",
       "      <th>a_attr_data-supp-info-image</th>\n",
       "      <th>source_url</th>\n",
       "      <th>article_title</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>GSE269782</td>\n",
       "      <td>NCBI Gene Expression Omnibus (GEO)</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...</td>\n",
       "      <td>Complex download</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.nature.com/articles/s41467-024-518...</td>\n",
       "      <td>No title found</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>GSE31210</td>\n",
       "      <td>NCBI GEO</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...</td>\n",
       "      <td>Complex download</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.nature.com/articles/s41467-024-518...</td>\n",
       "      <td>No title found</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>GSE106765</td>\n",
       "      <td>NCBI GEO</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...</td>\n",
       "      <td>Complex download</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.nature.com/articles/s41467-024-518...</td>\n",
       "      <td>No title found</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>GSE60189</td>\n",
       "      <td>NCBI GEO</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...</td>\n",
       "      <td>Complex download</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.nature.com/articles/s41467-024-518...</td>\n",
       "      <td>No title found</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>GSE59239</td>\n",
       "      <td>NCBI GEO</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...</td>\n",
       "      <td>Complex download</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.nature.com/articles/s41467-024-518...</td>\n",
       "      <td>No title found</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>GSE122005</td>\n",
       "      <td>NCBI GEO</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...</td>\n",
       "      <td>Complex download</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.nature.com/articles/s41467-024-518...</td>\n",
       "      <td>No title found</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>GSE38121</td>\n",
       "      <td>NCBI GEO</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...</td>\n",
       "      <td>Complex download</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.nature.com/articles/s41467-024-518...</td>\n",
       "      <td>No title found</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>GSE71587</td>\n",
       "      <td>NCBI GEO</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...</td>\n",
       "      <td>Complex download</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.nature.com/articles/s41467-024-518...</td>\n",
       "      <td>No title found</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>GSE37699</td>\n",
       "      <td>NCBI GEO</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...</td>\n",
       "      <td>Complex download</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.nature.com/articles/s41467-024-518...</td>\n",
       "      <td>No title found</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>PXD051771</td>\n",
       "      <td>ProteomeXchange</td>\n",
       "      <td>https://www.proteomexchange.org/cgi/GetDataset...</td>\n",
       "      <td>Complex download</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.nature.com/articles/s41467-024-518...</td>\n",
       "      <td>No title found</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://static-content.springer.com/esm/art%3A...</td>\n",
       "      <td>Supplementary Information</td>\n",
       "      <td>n/a</td>\n",
       "      <td>n/a</td>\n",
       "      <td>MOESM1</td>\n",
       "      <td>c-article-supplementary__item</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>print-link</td>\n",
       "      <td>click</td>\n",
       "      <td>view supplementary info</td>\n",
       "      <td>supp-info-link</td>\n",
       "      <td>supplementary information</td>\n",
       "      <td>https://static-content.springer.com/esm/art%3A...</td>\n",
       "      <td></td>\n",
       "      <td>https://www.nature.com/articles/s41467-024-518...</td>\n",
       "      <td>No title found</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>11 rows × 21 columns</p>\n",
       "</div>"
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "execution_count": 13
  },
  {
   "metadata": {},
   "cell_type": "markdown",
   "source": "### Full-Document-Read",
   "id": "2e5f5716d545a191"
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-07-07T15:30:47.684198Z",
     "start_time": "2025-07-07T15:30:41.739973Z"
    }
   },
   "cell_type": "code",
   "source": "results_FDR = data_gatherer.parse_data(API_unsupported_input, fetched_data[API_unsupported_input], publisher=\"Nature\", prompt_name=\"GPT_from_full_input_Examples\", full_document_read=True, raw_data_format=\"HTML\")",
   "id": "85277bc7cf47d4b1",
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "orchestrator.py - line 185 - INFO - Parsing data from URL: https://www.nature.com/articles/s41467-024-51831-7 with publisher: Nature\n",
      "base_parser.py - line 181 - INFO - LLMParser initialized.\n",
      "base_parser.py - line 913 - DEBUG - Repo mapping: {'bigd': 'bigd.big.ac.cn', 'european genome-phenome archive': 'ega-archive.org', 'massive': 'massive.ucsd.edu', 'pride': 'www.ebi.ac.uk', 'iprox': 'www.iprox.cn', 'proteomecentral': 'proteomecentral.proteomexchange.org', 'proteomexchange': 'www.proteomexchange.org', 'proteomic data commons': 'pdc.cancer.gov', 'genomic data commons': 'gdc.cancer.gov', 'the cancer imaging archive': 'cancerimagingarchive.net', 'imaging data commons': 'imaging.datacommons.cancer.gov', 'jpost': 'repository.jpostdb.org', 'doi.org': 'doi.org', 'geo': 'geo', 'dbgap': 'dbgap', 'sra': 'sra', 'arrayexpress': 'ArrayExpress'}\n",
      "html_parser.py - line 79 - INFO - Initializing htmlRetriever\n",
      "html_parser.py - line 342 - INFO - Function call: parse_data(api_data(<class 'str'>), Nature, https://www.nature.com/articles/s41467-024-51831-7, additional_data, HTML)\n",
      "html_parser.py - line 347 - INFO - Extracting links from full HTML content.\n",
      "html_parser.py - line 427 - INFO - Function_call: extract_href_from_html_supplementary_material(tree, https://www.nature.com/articles/s41467-024-51831-7)\n",
      "html_parser.py - line 435 - DEBUG - Found 4 anchors with data-ga-action='click_feat_suppl'.\n",
      "base_parser.py - line 302 - DEBUG - Function_call: reconstruct_download_link(https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-024-51831-7/MediaObjects/41467_2024_51831_MOESM1_ESM.pdf, c-article-supplementary__item, https://www.nature.com/articles/s41467-024-51831-7)\n",
      "base_parser.py - line 302 - DEBUG - Function_call: reconstruct_download_link(https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-024-51831-7/MediaObjects/41467_2024_51831_MOESM2_ESM.pdf, c-article-supplementary__item, https://www.nature.com/articles/s41467-024-51831-7)\n",
      "base_parser.py - line 302 - DEBUG - Function_call: reconstruct_download_link(https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-024-51831-7/MediaObjects/41467_2024_51831_MOESM3_ESM.pdf, c-article-supplementary__item, https://www.nature.com/articles/s41467-024-51831-7)\n",
      "base_parser.py - line 302 - DEBUG - Function_call: reconstruct_download_link(https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-024-51831-7/MediaObjects/41467_2024_51831_MOESM4_ESM.xlsx, c-article-supplementary__item, https://www.nature.com/articles/s41467-024-51831-7)\n",
      "html_parser.py - line 488 - INFO - Extracted 4 unique supplementary material links from HTML.\n",
      "html_parser.py - line 353 - DEBUG - Preprocessed data: <html class=\"js\" lang=\"en\"> <head> <link as=\"font\" crossorigin=\"\" href=\"/static/fonts/HardingText-Regular-Web-cecd90984f.woff2\" rel=\"preload\" type=\"font/woff2\"/> <title> Defective N-glycosylation of IL6 induces metastasis and tyrosine kinase inhibitor resistance in lung cancer | Nature Communications </title> <link href=\"https://www.nature.com/ncomms.rss\" rel=\"alternate\" type=\"application/rss+xml\"/> <link crossorigin=\"\" href=\"https://cmp.nature.com\" rel=\"preconnect\"/> <link href=\"/static/css/enhanced-article-nature-branded-3ad80c7544.css\" media=\"only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)\" onload=\"this.media='only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)';this.onload=null\" rel=\"stylesheet\"/> <link href=\"/static/css/article-print-c9121fecab.css\" media=\"print\" rel=\"stylesheet\" type=\"text/css\"/> <link href=\"/static/images/favicons/nature/apple-touch-icon-f39cb19454.png\" rel=\"apple-touch-icon\" sizes=\"180x180\"/> <link href=\"/static/images/favicons/nature/favicon-48x48-b52890008c.png\" rel=\"icon\" sizes=\"48x48\" type=\"image/png\"/> <link href=\"/static/images/favicons/nature/favicon-32x32-3fe59ece92.png\" rel=\"icon\" sizes=\"32x32\" type=\"image/png\"/> <link href=\"/static/images/favicons/nature/favicon-16x16-951651ab72.png\" rel=\"icon\" sizes=\"16x16\" type=\"image/png\"/> <link crossorigin=\"use-credentials\" href=\"/static/manifest.json\" rel=\"manifest\"/> <link color=\"#000000\" href=\"/static/images/favicons/nature/safari-pinned-tab-69bff48fe6.svg\" rel=\"mask-icon\"/> <link href=\"/static/images/favicons/nature/favicon.ico\" rel=\"shortcut icon\"/> <link href=\"https://www.nature.com/search\" rel=\"search\"/> <link href=\"https://www.nature.com/opensearch/opensearch.xml\" rel=\"search\" title=\"nature.com\" type=\"application/opensearchdescription+xml\"/> <link href=\"https://www.nature.com/opensearch/request\" rel=\"search\" title=\"nature.com\" type=\"application/sru+xml\"/> <link href=\"https://www.nature.com/articles/s41467-024-51831-7\" rel=\"canonical\"/> </head> <body class=\"article-page\"> <dialog aria-labelledby=\"cc-banner-label\" class=\"cc-banner\" open=\"\"> <div autofocus=\"\" class=\"cc-banner__content\" tabindex=\"-1\"> <div class=\"cc-banner__container\"> <div class=\"cc-banner__header\"> <h2 class=\"cc-banner__title\" id=\"cc-banner-label\"> Your privacy, your choice </h2> </div> <div class=\"cc-banner__body\"> <div class=\"cc-banner__policy\"> <p> We use essential cookies to make sure the site can function. We also use optional cookies for advertising, personalisation of content, usage analysis, and social media. </p> <p> By accepting optional cookies, you consent to the processing of your personal data - including transfers to third parties. Some third parties are outside of the European Economic Area, with varying standards of data protection. </p> <p> See our <a href=\"https://www.nature.com/info/privacy\"> privacy policy </a> for more information on the use of your personal data. </p> <p> for further information and to change your choices. </p> </div> </div> <div class=\"cc-banner__footer\"> </div> </div> </div> </dialog> <div class=\"position-relative cleared z-index-50 background-white\"> <a class=\"c-skip-link\" href=\"#content\"> Skip to main content </a> <div class=\"c-grade-c-banner u-hide\"> <div class=\"c-grade-c-banner__container\"> <p> Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. </p> </div> </div> <div class=\"u-hide u-show-following-ad\"> </div> <aside class=\"c-ad c-ad--728x90\"> <div class=\"c-ad__inner\"> <p class=\"c-ad__label\"> Advertisement </p> <div class=\"div-gpt-ad advert leaderboard js-ad text-center hide-print grade-c-hide\" id=\"div-gpt-ad-top-1\"> <div id=\"google_ads_iframe_/270604982/nature/ncomms/articles_0__container__\"> </div> </div> </div> </aside> <header class=\"c-header\" id=\"header\"> <div class=\"c-header__row\"> <div class=\"c-header__container\"> <div class=\"c-header__split\"> <div class=\"c-header__logo-container\"> <a href=\"/ncomms\"> <picture class=\"c-header__logo\"> <source media=\"(min-width: 875px)\" srcset=\"https://media.springernature.com/full/nature-cms/uploads/product/ncomms/header-03d2e325c0a02f6df509e5730e9be304.svg\"/> </picture> </a> </div> <ul class=\"c-header__menu c-header__menu--global\"> <li class=\"c-header__item c-header__item--padding c-header__item--hide-md-max\"> <a class=\"c-header__link\" href=\"https://www.nature.com/siteindex\"> <span> View all journals </span> </a> </li> <li class=\"c-header__item c-header__item--padding c-header__item--pipe\"> <a aria-expanded=\"false\" aria-haspopup=\"true\" class=\"c-header__link c-header__link--search\" href=\"javascript:;\" role=\"button\"> <span> Search </span> </a> <div class=\"c-header__dropdown c-header__dropdown--full-width has-tethered u-js-hide\" hidden=\"\" id=\"search-menu\"> <div class=\"c-header__container\"> <h2 class=\"c-header__visually-hidden\"> Search </h2> <div class=\"c-header__flush\"> <a class=\"c-header__link\" href=\"/search/advanced\"> Advanced search </a> </div> <h3 class=\"c-header__heading c-header__heading--keyline\"> Quick links </h3> <ul class=\"c-header__list\"> <li> <a class=\"c-header__link\" href=\"/subjects\"> Explore articles by subject </a> </li> <li> <a class=\"c-header__link\" href=\"/naturecareers\"> Find a job </a> </li> <li> <a class=\"c-header__link\" href=\"/authors/index.html\"> Guide to authors </a> </li> <li> <a class=\"c-header__link\" href=\"/authors/editorial_policies/\"> Editorial policies </a> </li> </ul> </div> </div> </li> <li class=\"c-header__item c-header__item--padding c-header__item--snid-account-widget c-header__item--pipe\"> <a class=\"c-header__link eds-c-header__link\" href=\"https://idp.nature.com/auth/personal/springernature?redirect_uri=https://www.nature.com/articles/s41467-024-51831-7\" id=\"identity-account-widget\"> <span class=\"eds-c-header__widget-fragment-title\"> Log in </span> </a> </li> </ul> </div> </div> </div> <div class=\"c-header__row\"> <div class=\"c-header__container\"> <div class=\"c-header__split\"> <ul class=\"c-header__menu c-header__menu--journal\"> <li class=\"c-header__item c-header__item--dropdown-menu\"> <a aria-expanded=\"false\" aria-haspopup=\"true\" class=\"c-header__link\" href=\"javascript:;\" role=\"button\"> <span> <span class=\"c-header__show-text\"> Explore </span> content </span> </a> <nav aria-labelledby=\"Explore-content\" class=\"c-header__dropdown has-tethered u-js-hide\" hidden=\"\" id=\"explore\"> <div class=\"c-header__container\"> <h2 class=\"c-header__heading c-header__heading--js-hide\" id=\"Explore-content\"> Explore content </h2> <ul class=\"c-header__list c-header__list--js-stack\"> <li class=\"c-header__item\"> <a class=\"c-header__link\" href=\"/ncomms/research-articles\"> Research articles </a> </li> <li class=\"c-header__item\"> <a class=\"c-header__link\" href=\"/ncomms/reviews-and-analysis\"> Reviews &amp; Analysis </a> </li> <li class=\"c-header__item\"> <a class=\"c-header__link\" href=\"/ncomms/news-and-comment\"> News &amp; Comment </a> </li> <li class=\"c-header__item\"> <a class=\"c-header__link\" href=\"/ncomms/video\"> Videos </a> </li> <li class=\"c-header__item\"> <a class=\"c-header__link\" href=\"/ncomms/collections\"> Collections </a> </li> <li class=\"c-header__item\"> <a class=\"c-header__link\" href=\"/ncomms/browse-subjects\"> Subjects </a> </li> </ul> <ul class=\"c-header__list c-header__list--js-stack\"> <li class=\"c-header__item\"> <a class=\"c-header__link\" href=\"https://www.facebook.com/NatureCommunications\"> Follow us on Facebook </a> </li> <li class=\"c-header__item\"> <a class=\"c-header__link\" href=\"https://twitter.com/NatureComms\"> Follow us on Twitter </a> </li> <li class=\"c-header__item c-header__item--hide-lg\"> <a class=\"c-header__link\" href=\"https://www.nature.com/my-account/alerts/subscribe-journal?list-id=264\" rel=\"nofollow\"> Sign up for alerts </a> </li> <li class=\"c-header__item c-header__item--hide-lg\"> <a class=\"c-header__link\" href=\"https://www.nature.com/ncomms.rss\"> <span> RSS feed </span> </a> </li> </ul> </div> </nav> </li> <li class=\"c-header__item c-header__item--dropdown-menu\"> <a aria-expanded=\"false\" aria-haspopup=\"true\" class=\"c-header__link\" href=\"javascript:;\" role=\"button\"> <span> About <span class=\"c-header__show-text\"> the journal </span> </span> </a> <nav aria-labelledby=\"About-the-journal\" class=\"c-header__dropdown has-tethered u-js-hide\" hidden=\"\" id=\"about-the-journal\"> <div class=\"c-header__container\"> <h2 class=\"c-header__heading c-header__heading--js-hide\" id=\"About-the-journal\"> About the journal </h2> <ul class=\"c-header__list c-header__list--js-stack\"> <li class=\"c-header__item\"> <a class=\"c-header__link\" href=\"/ncomms/aims\"> Aims &amp; Scope </a> </li> <li class=\"c-header__item\"> <a class=\"c-header__link\" href=\"/ncomms/editors\"> Editors </a> </li> <li class=\"c-header__item\"> <a class=\"c-header__link\" href=\"/ncomms/journal-information\"> Journal Information </a> </li> <li class=\"c-header__item\"> <a class=\"c-header__link\" href=\"/ncomms/open-access\"> Open Access Fees and Funding </a> </li> <li class=\"c-header__item\"> <a class=\"c-header__link\" href=\"/ncomms/calls-for-papers\"> Calls for Papers </a> </li> <li class=\"c-header__item\"> <a class=\"c-header__link\" href=\"/ncomms/editorial-values-statement\"> Editorial Values Statement </a> </li> <li class=\"c-header__item\"> <a class=\"c-header__link\" href=\"/ncomms/journal-impact\"> Journal Metrics </a> </li> <li class=\"c-header__item\"> <a class=\"c-header__link\" href=\"/ncomms/editorshighlights\"> Editors' Highlights </a> </li> <li class=\"c-header__item\"> <a class=\"c-header__link\" href=\"/ncomms/contact\"> Contact </a> </li> <li class=\"c-header__item\"> <a class=\"c-header__link\" href=\"/ncomms/editorial-policies\"> Editorial policies </a> </li> <li class=\"c-header__item\"> <a class=\"c-header__link\" href=\"/ncomms/top-articles\"> Top Articles </a> </li> </ul> </div> </nav> </li> <li class=\"c-header__item c-header__item--dropdown-menu\"> <a aria-expanded=\"false\" aria-haspopup=\"true\" class=\"c-header__link c-header__link--dropdown-menu\" href=\"javascript:;\" role=\"button\"> <span> Publish <span class=\"c-header__show-text\"> with us </span> </span> </a> <nav aria-labelledby=\"Publish-with-us-label\" class=\"c-header__dropdown has-tethered u-js-hide\" hidden=\"\" id=\"publish-with-us\"> <div class=\"c-header__container\"> <h2 class=\"c-header__heading c-header__heading--js-hide\" id=\"Publish-with-us-label\"> Publish with us </h2> <ul class=\"c-header__list c-header__list--js-stack\"> <li class=\"c-header__item\"> <a class=\"c-header__link\" href=\"/ncomms/submit\"> For authors </a> </li> <li class=\"c-header__item\"> <a class=\"c-header__link\" href=\"/ncomms/for-reviewers\"> For Reviewers </a> </li> <li class=\"c-header__item\"> <a class=\"c-header__link\" href=\"https://authorservices.springernature.com/go/sn/?utm_source=For+Authors&amp;utm_medium=Website_Nature&amp;utm_campaign=Platform+Experimentation+2022&amp;utm_id=PE2022\"> Language editing services </a> </li> <li class=\"c-header__item\"> <a class=\"c-header__link\" href=\"/ncomms/open-access-funding\"> Open access funding </a> </li> <li class=\"c-header__item c-header__item--keyline\"> <a class=\"c-header__link\" href=\"https://mts-ncomms.nature.com/\"> Submit manuscript </a> </li> </ul> </div> </nav> </li> </ul> <ul class=\"c-header__menu c-header__menu--hide-lg-max\"> <li class=\"c-header__item\"> <a class=\"c-header__link\" href=\"https://journal-alerts.springernature.com/subscribe?journal_id=41467\" rel=\"nofollow\"> <span> Sign up for alerts </span> </a> </li> <li class=\"c-header__item c-header__item--pipe\"> <a class=\"c-header__link\" href=\"https://www.nature.com/ncomms.rss\"> <span> RSS feed </span> </a> </li> </ul> </div> </div> </div> </header> <nav aria-label=\"breadcrumbs\" class=\"u-mb-16\"> <div class=\"u-container\"> <ol class=\"c-breadcrumbs\" itemscope=\"\" itemtype=\"https://schema.org/BreadcrumbList\"> <li class=\"c-breadcrumbs__item\" id=\"breadcrumb0\" itemprop=\"itemListElement\" itemscope=\"\" itemtype=\"https://schema.org/ListItem\"> <a class=\"c-breadcrumbs__link\" href=\"/\" itemprop=\"item\"> <span itemprop=\"name\"> nature </span> </a> </li> <li class=\"c-breadcrumbs__item\" id=\"breadcrumb1\" itemprop=\"itemListElement\" itemscope=\"\" itemtype=\"https://schema.org/ListItem\"> <a class=\"c-breadcrumbs__link\" href=\"/ncomms\" itemprop=\"item\"> <span itemprop=\"name\"> nature communications </span> </a> </li> <li class=\"c-breadcrumbs__item\" id=\"breadcrumb2\" itemprop=\"itemListElement\" itemscope=\"\" itemtype=\"https://schema.org/ListItem\"> <a class=\"c-breadcrumbs__link\" href=\"/ncomms/articles?type=article\" itemprop=\"item\"> <span itemprop=\"name\"> articles </span> </a> </li> <li class=\"c-breadcrumbs__item\" id=\"breadcrumb3\" itemprop=\"itemListElement\" itemscope=\"\" itemtype=\"https://schema.org/ListItem\"> <span itemprop=\"name\"> article </span> </li> </ol> </div> </nav> </div> <div class=\"u-container u-mt-32 u-mb-32 u-clearfix\" id=\"content\"> <main class=\"c-article-main-column u-float-left js-main-column\"> <div aria-hidden=\"false\" class=\"c-context-bar c-context-bar--sticky\" id=\"js-enable-context-bar\"> <div class=\"c-context-bar__container u-container\"> <div class=\"c-context-bar__title\"> Defective N-glycosylation of IL6 induces metastasis and tyrosine kinase inhibitor resistance in lung cancer </div> <div class=\"c-pdf-download u-clear-both js-pdf-download\"> <a class=\"u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link\" download=\"\" href=\"/articles/s41467-024-51831-7.pdf\" tabindex=\"-1\"> <span class=\"c-pdf-download__text\"> Download PDF </span> </a> </div> </div> </div> <article lang=\"en\"> <div class=\"c-pdf-button__container u-mb-16 u-hide-at-lg js-context-bar-sticky-point-mobile\"> <div class=\"c-pdf-container\"> <div class=\"c-pdf-download u-clear-both js-pdf-download\"> <a class=\"u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link\" download=\"\" href=\"/articles/s41467-024-51831-7.pdf\"> <span class=\"c-pdf-download__text\"> Download PDF </span> </a> </div> </div> </div> <div class=\"c-article-header\"> <header> <ul class=\"c-article-identifiers\"> <li class=\"c-article-identifiers__item\"> Article </li> <li class=\"c-article-identifiers__item\"> <a class=\"u-color-open-access\" href=\"https://www.springernature.com/gp/open-research/about/the-fundamentals-of-open-access-and-open-research\"> Open access </a> </li> <li class=\"c-article-identifiers__item\"> Published: <time datetime=\"2024-09-09\"> 09 September 2024 </time> </li> </ul> <h1 class=\"c-article-title\"> Defective N-glycosylation of IL6 induces metastasis and tyrosine kinase inhibitor resistance in lung cancer </h1> <ul class=\"c-article-author-list c-article-author-list--short js-no-scroll\"> <li class=\"c-article-author-list__item\"> <a href=\"#auth-Chun_Hua-Hung-Aff1-Aff2-Aff3-Aff4\"> Chun-Hua Hung </a> <span class=\"u-js-hide\"> <a class=\"js-orcid\" href=\"https://orcid.org/0000-0002-8580-8759\"> <span class=\"u-visually-hidden\"> ORCID: </span> orcid.org/0000-0002-8580-8759 </a> </span> <sup class=\"u-js-hide\"> <a href=\"#Aff1\" tabindex=\"-1\"> 1 </a> , <a href=\"#Aff2\" tabindex=\"-1\"> 2 </a> , <a href=\"#Aff3\" tabindex=\"-1\"> 3 </a> , <a href=\"#Aff4\" tabindex=\"-1\"> 4 </a> </sup> , </li> <li class=\"c-article-author-list__item\"> <a href=\"#auth-Shang_Yin-Wu-Aff1-Aff5\"> Shang-Yin Wu </a> <span class=\"u-js-hide\"> <a class=\"js-orcid\" href=\"https://orcid.org/0000-0003-2914-510X\"> <span class=\"u-visually-hidden\"> ORCID: </span> orcid.org/0000-0003-2914-510X </a> </span> <sup class=\"u-js-hide\"> <a href=\"#Aff1\" tabindex=\"-1\"> 1 </a> , <a href=\"#Aff5\" tabindex=\"-1\"> 5 </a> </sup> , </li> <li class=\"c-article-author-list__item c-article-author-list__item--hide-small-screen\"> <a href=\"#auth-Cheng_I_Daniel-Yao-Aff6\"> Cheng-I Daniel Yao </a> <span class=\"u-js-hide\"> <a class=\"js-orcid\" href=\"https://orcid.org/0000-0002-0902-045X\"> <span class=\"u-visually-hidden\"> ORCID: </span> orcid.org/0000-0002-0902-045X </a> </span> <sup class=\"u-js-hide\"> <a href=\"#Aff6\" tabindex=\"-1\"> 6 </a> </sup> , </li> <li class=\"c-article-author-list__item c-article-author-list__item--hide-small-screen\"> <a href=\"#auth-Hsuan_Heng-Yeh-Aff2\"> Hsuan-Heng Yeh </a> <sup class=\"u-js-hide\"> <a href=\"#Aff2\" tabindex=\"-1\"> 2 </a> </sup> , </li> <li class=\"c-article-author-list__item c-article-author-list__item--hide-small-screen\"> <a href=\"#auth-Chien_Chung-Lin-Aff5-Aff7-Aff8\"> Chien-Chung Lin </a> <sup class=\"u-js-hide\"> <a href=\"#Aff5\" tabindex=\"-1\"> 5 </a> , <a href=\"#Aff7\" tabindex=\"-1\"> 7 </a> , <a href=\"#Aff8\" tabindex=\"-1\"> 8 </a> </sup> , </li> <li class=\"c-article-author-list__item c-article-author-list__item--hide-small-screen\"> <a href=\"#auth-Chang_Yao-Chu-Aff9\"> Chang-Yao Chu </a> <sup class=\"u-js-hide\"> <a href=\"#Aff9\" tabindex=\"-1\"> 9 </a> </sup> , </li> <li class=\"c-article-author-list__item c-article-author-list__item--hide-small-screen\"> <a href=\"#auth-Tzu_Yu-Huang-Aff10\"> Tzu-Yu Huang </a> <sup class=\"u-js-hide\"> <a href=\"#Aff10\" tabindex=\"-1\"> 10 </a> </sup> , </li> <li class=\"c-article-author-list__item c-article-author-list__item--hide-small-screen\"> <a href=\"#auth-Meng_Ru-Shen-Aff11-Aff12\"> Meng-Ru Shen </a> <span class=\"u-js-hide\"> <a class=\"js-orcid\" href=\"https://orcid.org/0000-0003-0506-3505\"> <span class=\"u-visually-hidden\"> ORCID: </span> orcid.org/0000-0003-0506-3505 </a> </span> <sup class=\"u-js-hide\"> <a href=\"#Aff11\" tabindex=\"-1\"> 11 </a> , <a href=\"#Aff12\" tabindex=\"-1\"> 12 </a> </sup> , </li> <li class=\"c-article-author-list__item c-article-author-list__item--hide-small-screen\"> <a href=\"#auth-Chun_Hung-Lin-Aff6-Aff13\"> Chun-Hung Lin </a> <span class=\"u-js-hide\"> <a class=\"js-orcid\" href=\"https://orcid.org/0000-0002-6795-8825\"> <span class=\"u-visually-hidden\"> ORCID: </span> orcid.org/0000-0002-6795-8825 </a> </span> <sup class=\"u-js-hide\"> <a href=\"#Aff6\" tabindex=\"-1\"> 6 </a> , <a href=\"#Aff13\" tabindex=\"-1\"> 13 </a> </sup> &amp; </li> <li aria-label=\"Show all 10 authors for this article\" class=\"c-article-author-list__show-more\" title=\"Show all 10 authors for this article\"> … </li> <li class=\"c-article-author-list__item\"> <a href=\"#auth-Wu_Chou-Su-Aff1-Aff2-Aff3-Aff5-Aff10\"> Wu-Chou Su </a> <span class=\"u-js-hide\"> <a class=\"js-orcid\" href=\"https://orcid.org/0000-0003-2953-4105\"> <span class=\"u-visually-hidden\"> ORCID: </span> orcid.org/0000-0003-2953-4105 </a> </span> <sup class=\"u-js-hide\"> <a href=\"#Aff1\" tabindex=\"-1\"> 1 </a> , <a href=\"#Aff2\" tabindex=\"-1\"> 2 </a> , <a href=\"#Aff3\" tabindex=\"-1\"> 3 </a> , <a href=\"#Aff5\" tabindex=\"-1\"> 5 </a> , <a href=\"#Aff10\" tabindex=\"-1\"> 10 </a> </sup> </li> </ul> <p class=\"c-article-info-details\"> <a href=\"/ncomms\"> <i> Nature Communications </i> </a> <b> <span class=\"u-visually-hidden\"> volume </span> 15 </b> , Article number: <span> 7885 </span> ( <span> 2024 </span> ) <a class=\"c-article-info-details__cite-as u-hide-print\" href=\"#citeas\"> Cite this article </a> </p> <div class=\"c-article-metrics-bar__wrapper u-clear-both\"> <ul class=\"c-article-metrics-bar u-list-reset\"> <li class=\"c-article-metrics-bar__item\"> <p class=\"c-article-metrics-bar__count\"> 9119 <span class=\"c-article-metrics-bar__label\"> Accesses </span> </p> </li> <li class=\"c-article-metrics-bar__item\"> <p class=\"c-article-metrics-bar__count\"> 13 <span class=\"c-article-metrics-bar__label\"> Citations </span> </p> </li> <li class=\"c-article-metrics-bar__item\"> <p class=\"c-article-metrics-bar__count\"> 30 <span class=\"c-article-metrics-bar__label\"> Altmetric </span> </p> </li> <li class=\"c-article-metrics-bar__item\"> <p class=\"c-article-metrics-bar__details\"> <a href=\"/articles/s41467-024-51831-7/metrics\" rel=\"nofollow\"> Metrics <span class=\"u-visually-hidden\"> details </span> </a> </p> </li> </ul> </div> </header> </div> <div class=\"c-article-body\"> <section aria-labelledby=\"Abs1\" lang=\"en\"> <div class=\"c-article-section\" id=\"Abs1-section\"> <h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"Abs1\"> Abstract </h2> <div class=\"c-article-section__content\" id=\"Abs1-content\"> <p> The IL6-GP130-STAT3 pathway facilitates lung cancer progression and resistance to tyrosine kinase inhibitors. Although glycosylation alters the stability of GP130, its effect on the ligand IL6 remains unclear. We herein find that N-glycosylated IL6, especially at Asn73, primarily stimulates JAK-STAT3 signaling and prolongs STAT3 phosphorylation, whereas N-glycosylation-defective IL6 (deNG-IL6) induces shortened STAT3 activation and alters the downstream signaling preference for the SRC-YAP-SOX2 axis. This signaling shift induces <u> e </u> pithelial- <u> m </u> esenchymal <u> t </u> ransition (EMT) and migration in vitro and metastasis in vivo, which are suppressed by targeted inhibitors and shRNAs against SRC, YAP, and SOX2. Osimertinib-resistant lung cancer cells secrete a large amount of deNG-IL6 through reduced N-glycosyltransferase gene expression, leading to clear SRC-YAP activation. deNG-IL6 contributes to drug resistance, as confirmed by in silico analysis of cellular and clinical transcriptomes and signal expression in patient specimens. Therefore, the N-glycosylation status of IL6 not only affects cell behaviors but also shows promise in monitoring the dynamics of lung cancer evolution. </p> </div> </div> </section> <section aria-labelledby=\"inline-recommendations\" class=\"c-article-recommendations\"> <h3 class=\"c-article-recommendations-title\" id=\"inline-recommendations\"> Similar content being viewed by others </h3> <div class=\"c-article-recommendations-list\"> <div class=\"c-article-recommendations-list__item\"> <article class=\"c-article-recommendations-card\" itemscope=\"\" itemtype=\"http://schema.org/ScholarlyArticle\"> <div class=\"c-article-recommendations-card__img\"> </div> <div class=\"c-article-recommendations-card__main\"> <h3 class=\"c-article-recommendations-card__heading\" itemprop=\"name headline\"> <a class=\"c-article-recommendations-card__link\" href=\"https://www.nature.com/articles/s41388-025-03461-6?fromPaywallRec=false\" itemprop=\"url\"> Drug tolerance and persistence to EGFR inhibitor treatment are mediated by an ILK-SFK-YAP signaling axis in lung adenocarcinoma </a> </h3> <div class=\"c-article-meta-recommendations\"> <span class=\"c-article-meta-recommendations__item-type\"> Article </span> <span class=\"c-article-meta-recommendations__access-type\"> Open access </span> <span class=\"c-article-meta-recommendations__date\"> 31 May 2025 </span> </div> </div> </article> </div> <div class=\"c-article-recommendations-list__item\"> <article class=\"c-article-recommendations-card\" itemscope=\"\" itemtype=\"http://schema.org/ScholarlyArticle\"> <div class=\"c-article-recommendations-card__img\"> </div> <div class=\"c-article-recommendations-card__main\"> <h3 class=\"c-article-recommendations-card__heading\" itemprop=\"name headline\"> <a class=\"c-article-recommendations-card__link\" href=\"https://www.nature.com/articles/s41598-022-21419-6?fromPaywallRec=false\" itemprop=\"url\"> NT157 exerts antineoplastic activity by targeting JNK and AXL signaling in lung cancer cells </a> </h3> <div class=\"c-article-meta-recommendations\"> <span class=\"c-article-meta-recommendations__item-type\"> Article </span> <span class=\"c-article-meta-recommendations__access-type\"> Open access </span> <span class=\"c-article-meta-recommendations__date\"> 12 October 2022 </span> </div> </div> </article> </div> <div class=\"c-article-recommendations-list__item\"> <article class=\"c-article-recommendations-card\" itemscope=\"\" itemtype=\"http://schema.org/ScholarlyArticle\"> <div class=\"c-article-recommendations-card__img\"> </div> <div class=\"c-article-recommendations-card__main\"> <h3 class=\"c-article-recommendations-card__heading\" itemprop=\"name headline\"> <a class=\"c-article-recommendations-card__link\" href=\"https://www.nature.com/articles/s41420-022-01104-3?fromPaywallRec=false\" itemprop=\"url\"> Gankyrin modulated non-small cell lung cancer progression via glycolysis metabolism in a YAP1-dependent manner </a> </h3> <div class=\"c-article-meta-recommendations\"> <span class=\"c-article-meta-recommendations__item-type\"> Article </span> <span class=\"c-article-meta-recommendations__access-type\"> Open access </span> <span class=\"c-article-meta-recommendations__date\"> 09 July 2022 </span> </div> </div> </article> </div> </div> </section> <div class=\"main-content\"> <section> <div class=\"c-article-section\" id=\"Sec1-section\"> <h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"Sec1\"> Introduction </h2> <div class=\"c-article-section__content\" id=\"Sec1-content\"> <p> Interleukin-6 (IL6) is produced by both cancerous and noncancerous human cells and binds to IL6-receptor alpha (IL6Rα, also known as GP80). The IL6-IL6Rα complex then associates with coreceptor GP130, primarily triggering the GP130-JAK-STAT3 signaling <sup> <a href=\"#ref-CR1\" id=\"ref-link-section-d6776021e851\" title=\"Yu, H., Lee, H., Herrmann, A., Buettner, R. &amp; Jove, R. Revisiting STAT3 signalling in cancer: new and unexpected biological functions. Nat. Rev. Cancer 14, 736–746 (2014).\"> 1 </a> , <a href=\"#ref-CR2\" id=\"ref-link-section-d6776021e851_1\" title=\"Jones, S. A. &amp; Jenkins, B. J. Recent insights into targeting the IL-6 cytokine family in inflammatory diseases and cancer. Nat. Rev. Immunol. 18, 773–789 (2018).\"> 2 </a> , <a aria-label=\"Reference 3\" href=\"/articles/s41467-024-51831-7#ref-CR3\" id=\"ref-link-section-d6776021e854\" title=\"Rose-John, S. The soluble interleukin 6 receptor: advanced therapeutic options in inflammation. Clin. Pharmacol. Therapeut. 102, 591–598 (2017).\"> 3 </a> </sup> axis to regulate immune responses and cancer pathogenesis <sup> <a href=\"#ref-CR4\" id=\"ref-link-section-d6776021e858\" title=\"Trikha, M., Corringham, R., Klein, B. &amp; Rossi, J. F. Targeted anti-interleukin-6 monoclonal antibody therapy for cancer: a review of the rationale and clinical evidence. Clin. Cancer Res. 9, 4653–4665 (2003).\"> 4 </a> , <a href=\"#ref-CR5\" id=\"ref-link-section-d6776021e858_1\" title=\"Hunter, C. A. &amp; Jones, S. A. IL-6 as a keystone cytokine in health and disease. Nat. Immunol. 16, 448–457 (2015).\"> 5 </a> , <a href=\"#ref-CR6\" id=\"ref-link-section-d6776021e858_2\" title=\"Garbers, C., Heink, S., Korn, T. &amp; Rose-John, S. Interleukin-6: designing specific therapeutics for a complex cytokine. Nat. Rev. Drug Discov. 17, 395–412 (2018).\"> 6 </a> , <a aria-label=\"Reference 7\" href=\"/articles/s41467-024-51831-7#ref-CR7\" id=\"ref-link-section-d6776021e861\" title=\"Murakami, M., Kamimura, D. &amp; Hirano, T. Pleiotropy and specificity: insights from the interleukin 6 family of cytokines. Immunity 50, 812–831 (2019).\"> 7 </a> </sup> . In non-small cell lung cancer (NSCLC), an increase in circulating IL6 is associated with poor survival of patients, and autocrine IL6-mediated GP130-JAK-STAT3 signaling is positively correlated with tumor progression, apoptotic resistance, and drug resistance <sup> <a href=\"#ref-CR8\" id=\"ref-link-section-d6776021e865\" title=\"Duan, Z. et al. Signal transducers and activators of transcription 3 pathway activation in drug-resistant ovarian cancer. Clin. Cancer Res. 12, 5055–5063 (2006).\"> 8 </a> , <a href=\"#ref-CR9\" id=\"ref-link-section-d6776021e865_1\" title=\"Huang, W. L. et al. Signal transducer and activator of transcription 3 activation up-regulates interleukin-6 autocrine production: a biochemical and genetic study of established cancer cell lines and clinical isolated human cancer cells. Mol. Cancer 9, 309 (2010).\"> 9 </a> , <a aria-label=\"Reference 10\" href=\"/articles/s41467-024-51831-7#ref-CR10\" id=\"ref-link-section-d6776021e868\" title=\"Lee, H. J. et al. Drug resistance via feedback activation of Stat3 in oncogene-addicted cancer cells. Cancer Cell 26, 207–221 (2014).\"> 10 </a> </sup> . In addition to JAK-STAT3, IL6 activates the ERK, AKT, and SRC-YAP pathways via GP130 to regulate diverse biological processes <sup> <a aria-label=\"Reference 1\" href=\"/articles/s41467-024-51831-7#ref-CR1\" id=\"ref-link-section-d6776021e872\" title=\"Yu, H., Lee, H., Herrmann, A., Buettner, R. &amp; Jove, R. Revisiting STAT3 signalling in cancer: new and unexpected biological functions. Nat. Rev. Cancer 14, 736–746 (2014).\"> 1 </a> , <a aria-label=\"Reference 11\" href=\"/articles/s41467-024-51831-7#ref-CR11\" id=\"ref-link-section-d6776021e875\" title=\"Rokavec, M. et al. IL-6R/STAT3/miR-34a feedback loop promotes EMT-mediated colorectal cancer invasion and metastasis. J. Clin. Invest. 124, 1853–1867 (2014).\"> 11 </a> , <a aria-label=\"Reference 12\" href=\"/articles/s41467-024-51831-7#ref-CR12\" id=\"ref-link-section-d6776021e878\" title=\"Taniguchi, K. et al. A gp130-Src-YAP module links inflammation to epithelial regeneration. Nature 519, 57–62 (2015).\"> 12 </a> </sup> . </p> <p> Targeted therapy using epidermal growth factor receptor-tyrosine kinase inhibitors (EGFR-TKIs) is the standard care for patients with lung adenocarcinoma harboring sensitive EGFR mutations <sup> <a aria-label=\"Reference 13\" href=\"/articles/s41467-024-51831-7#ref-CR13\" id=\"ref-link-section-d6776021e885\" title=\"Quintanal-Villalonga, A. et al. Lineage plasticity in cancer: a shared pathway of therapeutic resistance. Nat. Rev. Clin. Oncol. 17, 360–371 (2020).\"> 13 </a> , <a aria-label=\"Reference 14\" href=\"/articles/s41467-024-51831-7#ref-CR14\" id=\"ref-link-section-d6776021e888\" title=\"Paez, J. G. et al. EGFR mutations in lung cancer: correlation with clinical response to gefitinib therapy. Science 304, 1497–1500 (2004).\"> 14 </a> </sup> . The third-generation EGFR-TKI osimertinib is currently the preferred first-line therapeutic <sup> <a aria-label=\"Reference 15\" href=\"/articles/s41467-024-51831-7#ref-CR15\" id=\"ref-link-section-d6776021e892\" title=\"Ramalingam, S. S. et al. Overall survival with osimertinib in untreated, EGFR-mutated advanced NSCLC. N. Engl. J. Med. 382, 41–50 (2020).\"> 15 </a> </sup> . Although EGFR-TKIs are highly effective in treating this group of patients, acquired resistance inevitably develops in most subjects <sup> <a aria-label=\"Reference 16\" href=\"/articles/s41467-024-51831-7#ref-CR16\" id=\"ref-link-section-d6776021e896\" title=\"Oxnard, G. R. et al. Assessment of resistance mechanisms and clinical implications in patients with EGFR T790M-positive lung cancer and acquired resistance to osimertinib. JAMA Oncol. 4, 1527–1534 (2018).\"> 16 </a> </sup> . Common mechanisms of acquired resistance to EGFR-TKIs include on-target secondary mutations (e.g., T790M and T797S), genetic alterations that result in activation of multiple bypass pathways such as MET, HER2, KRAS, and PIK3CA <sup> <a aria-label=\"Reference 17\" href=\"/articles/s41467-024-51831-7#ref-CR17\" id=\"ref-link-section-d6776021e900\" title=\"Yang, Z. et al. Investigating novel resistance mechanisms to third-generation EGFR tyrosine kinase inhibitor osimertinib in non-small cell lung cancer patients. Clin. Cancer Res. 24, 3097–3107 (2018).\"> 17 </a> </sup> , and histological transformation from adenocarcinomas to aggressive neuroendocrine derivatives <sup> <a aria-label=\"Reference 18\" href=\"/articles/s41467-024-51831-7#ref-CR18\" id=\"ref-link-section-d6776021e904\" title=\"Sequist, L. V. et al. Genotypic and histological evolution of lung cancers acquiring resistance to EGFR inhibitors. Sci. Transl. Med. 3, 75ra26 (2011).\"> 18 </a> </sup> where cancer cells gain cell plasticity and exhibit an <u> e </u> pithelial- <u> m </u> esenchymal <u> t </u> ransition (EMT) phenotype <sup> <a aria-label=\"Reference 19\" href=\"/articles/s41467-024-51831-7#ref-CR19\" id=\"ref-link-section-d6776021e918\" title=\"Lambert, A. W. &amp; Weinberg, R. A. Linking EMT programmes to normal and neoplastic epithelial stem cells. Nat. Rev. Cancer 21, 325–338 (2021).\"> 19 </a> </sup> . </p> <p> Lineage plasticity is proposed as a shared pathway for resistance to anticancer agents, which encompasses phenotypic changes in cancer cells, such as EMT and reprogramming of cancer stem cells <sup> <a aria-label=\"Reference 19\" href=\"/articles/s41467-024-51831-7#ref-CR19\" id=\"ref-link-section-d6776021e925\" title=\"Lambert, A. W. &amp; Weinberg, R. A. Linking EMT programmes to normal and neoplastic epithelial stem cells. Nat. Rev. Cancer 21, 325–338 (2021).\"> 19 </a> , <a aria-label=\"Reference 20\" href=\"/articles/s41467-024-51831-7#ref-CR20\" id=\"ref-link-section-d6776021e928\" title=\"Williams, E. D., Gao, D., Redfern, A. &amp; Thompson, E. W. Controversies around epithelial-mesenchymal plasticity in cancer metastasis. Nat. Rev. Cancer 19, 716–732 (2019).\"> 20 </a> </sup> . EGFR-TKI-induced high IL6 secretion has been observed in lung cancer cell lines <sup> <a aria-label=\"Reference 21\" href=\"/articles/s41467-024-51831-7#ref-CR21\" id=\"ref-link-section-d6776021e932\" title=\"Yao, Z. et al. TGF-beta IL-6 axis mediates selective and adaptive mechanisms of resistance to molecular targeted therapy in lung cancer. Proc. Natl Acad. Sci. USA 107, 15535–15540 (2010).\"> 21 </a> </sup> and clinical samples <sup> <a aria-label=\"Reference 17\" href=\"/articles/s41467-024-51831-7#ref-CR17\" id=\"ref-link-section-d6776021e936\" title=\"Yang, Z. et al. Investigating novel resistance mechanisms to third-generation EGFR tyrosine kinase inhibitor osimertinib in non-small cell lung cancer patients. Clin. Cancer Res. 24, 3097–3107 (2018).\"> 17 </a> </sup> , which facilitates TKI resistance by promoting EMT in a TGF-β-dependent manner <sup> <a aria-label=\"Reference 21\" href=\"/articles/s41467-024-51831-7#ref-CR21\" id=\"ref-link-section-d6776021e940\" title=\"Yao, Z. et al. TGF-beta IL-6 axis mediates selective and adaptive mechanisms of resistance to molecular targeted therapy in lung cancer. Proc. Natl Acad. Sci. USA 107, 15535–15540 (2010).\"> 21 </a> </sup> or inhibiting apoptosis by activating the JAK1-STAT3 pathway <sup> <a aria-label=\"Reference 22\" href=\"/articles/s41467-024-51831-7#ref-CR22\" id=\"ref-link-section-d6776021e944\" title=\"Kim, S. M. et al. Activation of IL-6R/JAK1/STAT signaling induces de novo resistance to irreversible EGFR inhibitors in non-small cell lung cancer with T790M resistance mutation. Mol. Cancer Ther. 11, 2254–2264 (2012).\"> 22 </a> </sup> . IL6 also induces self-renewal of cancer stem cells (CSCs) <sup> <a aria-label=\"Reference 23\" href=\"/articles/s41467-024-51831-7#ref-CR23\" id=\"ref-link-section-d6776021e949\" title=\"Korkaya, H., Liu, S. &amp; Wicha, M. S. Regulation of cancer stem cells by cytokine networks: attacking cancer’s inflammatory roots. Clin. Cancer Res. 17, 6125–6129 (2011).\"> 23 </a> </sup> and dedifferentiation of tumor cells <sup> <a aria-label=\"Reference 24\" href=\"/articles/s41467-024-51831-7#ref-CR24\" id=\"ref-link-section-d6776021e953\" title=\"Senescence-Associated Pro-inflammatory Cytokines, VernotJ. P. and tumor cell plasticity. Front. Mol. Biosci. 7, 63 (2020).\"> 24 </a> </sup> via activation of the GP130-SRC-YAP pathway <sup> <a aria-label=\"Reference 12\" href=\"/articles/s41467-024-51831-7#ref-CR12\" id=\"ref-link-section-d6776021e957\" title=\"Taniguchi, K. et al. A gp130-Src-YAP module links inflammation to epithelial regeneration. Nature 519, 57–62 (2015).\"> 12 </a> </sup> . In EGFR-mutant lung adenocarcinoma, the osimertinib-activated YAP-TEAD complex has been implicated in cellular dormancy and resistance <sup> <a aria-label=\"Reference 25\" href=\"/articles/s41467-024-51831-7#ref-CR25\" id=\"ref-link-section-d6776021e961\" title=\"Kurppa, K. J. et al. Treatment-induced tumor dormancy through YAP-mediated transcriptional reprogramming of the apoptotic pathway. Cancer Cell 37, 104–122.e112 (2020).\"> 25 </a> </sup> . Studies of transcriptional profiling following EGFR-TKI treatment and animal models revealed that SOX2 is a key transcription factor (TF) in promoting neuroendocrine transformation <sup> <a aria-label=\"Reference 26\" href=\"/articles/s41467-024-51831-7#ref-CR26\" id=\"ref-link-section-d6776021e965\" title=\"Ferone, G. et al. SOX2 is the determining oncogenic switch in promoting lung squamous cell carcinoma from different cells of origin. Cancer Cell 30, 519–532 (2016).\"> 26 </a> </sup> and plasticity-reprogramming processes in lung cancer <sup> <a aria-label=\"Reference 27\" href=\"/articles/s41467-024-51831-7#ref-CR27\" id=\"ref-link-section-d6776021e969\" title=\"Boumahdi, S. &amp; de Sauvage, F. J. The great escape: tumour cell plasticity in resistance to targeted therapy. Nat. Rev. Drug Discov. 19, 39–56 (2020).\"> 27 </a> </sup> , in which IL6 <sup> <a aria-label=\"Reference 28\" href=\"/articles/s41467-024-51831-7#ref-CR28\" id=\"ref-link-section-d6776021e974\" title=\"Mosteiro, L. et al. Tissue damage and senescence provide critical signals for cellular reprogramming in vivo. Science 354, 6315 (2016).\"> 28 </a> </sup> or YAP <sup> <a aria-label=\"Reference 29\" href=\"/articles/s41467-024-51831-7#ref-CR29\" id=\"ref-link-section-d6776021e978\" title=\"Bora-Singhal, N. et al. YAP1 regulates OCT4 activity and SOX2 expression to facilitate self-renewal and vascular mimicry of stem-like cells. Stem cells 33, 1705–1718 (2015).\"> 29 </a> </sup> could be inducers. This evidence suggests that the IL6-YAP-SOX2 pathway may be an unrevealed axis that facilitates the development of lineage plasticity and EGFR TKI resistance in lung cancer. However, how IL6 differentially activates the JAK-STAT3 or SRC-YAP-SOX2 axis is not yet clear. </p> <p> Multiple forms of IL6 ranging from 20 to 30 kDa in molecular weight have been observed <sup> <a href=\"#ref-CR30\" id=\"ref-link-section-d6776021e985\" title=\"Poupart, P. et al. B cell growth modulating and differentiating activity of recombinant human 26-kd protein (BSF-2, HuIFN-beta 2, HPGF). EMBO J 6, 1219–1224 (1987).\"> 30 </a> , <a href=\"#ref-CR31\" id=\"ref-link-section-d6776021e985_1\" title=\"May, L. T. et al. Phosphorylation of secreted forms of human beta 2-interferon/hepatocyte stimulating factor/interleukin-6. Biochem. Biophys. Res. Commun. 152, 1144–1150 (1988).\"> 31 </a> , <a href=\"#ref-CR32\" id=\"ref-link-section-d6776021e985_2\" title=\"Hirano, T. et al. Purification to homogeneity and characterization of human B-cell differentiation factor (BCDF or BSFp-2). Proc. Natl Acad. Sci. USA 82, 5490–5494 (1985).\"> 32 </a> , <a aria-label=\"Reference 33\" href=\"/articles/s41467-024-51831-7#ref-CR33\" id=\"ref-link-section-d6776021e988\" title=\"May, L. T., Santhanam, U. &amp; Sehgal, P. B. On the multimeric nature of natural human interleukin-6. J. Biol. Chem. 266, 9950–9955 (1991).\"> 33 </a> </sup> , but no alternatively spliced isoforms have been reported, suggesting that IL6 undergoes cell type-specific post-translational modification (PTM). As the most common PTM, glycosylation <sup> <a aria-label=\"Reference 34\" href=\"/articles/s41467-024-51831-7#ref-CR34\" id=\"ref-link-section-d6776021e992\" title=\"Mereiter, S., Balmana, M., Campos, D., Gomes, J. &amp; Reis, C. A. Glycosylation in the era of cancer-targeted therapy: where are we heading? Cancer Cell 36, 6–16 (2019).\"> 34 </a> </sup> has been observed in human macrophage-derived IL6 <sup> <a aria-label=\"Reference 35\" href=\"/articles/s41467-024-51831-7#ref-CR35\" id=\"ref-link-section-d6776021e996\" title=\"Parekh, R. B., Dwek, R. A., Rademacher, T. W., Opdenakker, G. &amp; Van Damme, J. Glycosylation of interleukin-6 purified from normal human blood mononuclear cells. Eur. J. Biochem. 203, 135–141 (1992).\"> 35 </a> </sup> and Kaposi’s sarcoma-associated herpesvirus-encoded viral IL6 homolog <sup> <a aria-label=\"Reference 36\" href=\"/articles/s41467-024-51831-7#ref-CR36\" id=\"ref-link-section-d6776021e1000\" title=\"Dela Cruz, C. S. et al. N-linked glycosylation is required for optimal function of Kaposi’s sarcoma herpesvirus-encoded, but not cellular, interleukin 6. J. Exp. Med. 199, 503–514 (2004).\"> 36 </a> </sup> . In a Kaposi’s sarcoma model, N-glycosylation is necessary for viral IL6 but not human IL6 to activate JAK-STAT3 signaling <sup> <a aria-label=\"Reference 36\" href=\"/articles/s41467-024-51831-7#ref-CR36\" id=\"ref-link-section-d6776021e1004\" title=\"Dela Cruz, C. S. et al. N-linked glycosylation is required for optimal function of Kaposi’s sarcoma herpesvirus-encoded, but not cellular, interleukin 6. J. Exp. Med. 199, 503–514 (2004).\"> 36 </a> </sup> . Although aberrant protein glycosylation has been observed in various cancer types and shown to enhance or repress cellular signaling <sup> <a aria-label=\"Reference 37\" href=\"/articles/s41467-024-51831-7#ref-CR37\" id=\"ref-link-section-d6776021e1009\" title=\"Thomas, D., Rathinavel, A. K. &amp; Radhakrishnan, P. Altered glycosylation in cancer: a promising target for biomarkers and therapeutics. Biochimica. et biophysica. Acta Rev. Cancer 1875, 188464 (2021).\"> 37 </a> , <a aria-label=\"Reference 38\" href=\"/articles/s41467-024-51831-7#ref-CR38\" id=\"ref-link-section-d6776021e1012\" title=\"Pinho, S. S. &amp; Reis, C. A. Glycosylation in cancer: mechanisms and clinical implications. Nat. Rev. Cancer 15, 540–555 (2015).\"> 38 </a> </sup> , whether the glycosylation status of IL6 in human lung cancer influences its biological functions remains unclear. Likewise, whether EGFR-TKI treatment affects the glycosylation status of proteins in lung cancer cells is unresolved. </p> <p> Because IL6 activates several signaling pathways through interaction with GP130 and glycosylation of IL6 may change its biological functions, we hypothesize that the glycosylation patterns of human IL6 (hIL6) may contribute to its preference for specific downstream signaling pathways. Here we demonstrate that IL6 secreted from human lung cancer cells is modified by N-glycosylation, and that N-glycosylation of IL6 occurs primarily at asparagine (N) 73 and activates the canonical JAK-STAT3 signaling pathway. In contrast, <u> N </u> - <u> g </u> lycosylation- <u> de </u> fective IL6 (deNG-IL6) switches the signaling preference to the SRC-YAP pathway, which facilitates SOX2 expression and enhances EMT, migration, and metastasis. Additionally, the N-glycosylation process is suppressed in osimertinib-treated lung cancer cells, resulting in an increased proportion of deNG-IL6 and the acquisition of cell plasticity and subsequent TKI resistance. </p> </div> </div> </section> <section> <div class=\"c-article-section\" id=\"Sec2-section\"> <h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"Sec2\"> Results </h2> <div class=\"c-article-section__content\" id=\"Sec2-content\"> <h3 class=\"c-article__sub-heading\" id=\"Sec3\"> IL6 Is Modified by N-Glycosylation in NSCLC </h3> <p> We sought to investigate whether IL6 orchestrates multiple signaling pathways via differential glycosylation. Large quantities of IL6 in malignant pleural effusions (MPEs) have been reported in lung cancer patients <sup> <a aria-label=\"Reference 39\" href=\"/articles/s41467-024-51831-7#ref-CR39\" id=\"ref-link-section-d6776021e1041\" title=\"Yeh, H. H., Lai, W. W., Chen, H. H., Liu, H. S. &amp; Su, W. C. Autocrine IL-6-induced Stat3 activation contributes to the pathogenesis of lung adenocarcinoma and malignant pleural effusion. Oncogene 25, 4300–4309 (2006).\"> 39 </a> </sup> ; thus, we isolated lung adenocarcinoma cells from the MPEs (termed “MPECs”) of 15 NSCLC patients to study the glycosylation pattern of lung cancer cell-derived IL6. Short-term cultured MPECs were verified by double-positive expression of the NSCLC markers pancytokeratin (PanCK) and thyroid transcription factor-1 (TTF1) (Fig. <a href=\"/articles/s41467-024-51831-7#Fig1\"> 1a </a> ). IL6 secreted by each MPEC line detected by Western blotting (WB) showed two IL6 bands [upper, <u> h </u> eavy form (H-IL6); lower, <u> l </u> ight form (L-IL6)] (Fig. <a href=\"/articles/s41467-024-51831-7#Fig1\"> 1b </a> ), and the H/L intensity ratios differed among the MPECs. We next measured the amount of IL6 secreted by multiple NSCLC cell lines, including PC14PE6/AS2 (AS2), H1299, A549, H226, HCC827, H1975, H1650, and H460 (Fig. <a href=\"/articles/s41467-024-51831-7#Fig1\"> 1c </a> ), and examined the IL6 pattern in the cell lines with low baseline secretion (H1299) and high baseline secretion (H226, H1975, and H460). Similar to the results for the MPECs, both H-IL6 and L-IL6 were detected with different H/L IL6 ratios (Fig. <a href=\"/articles/s41467-024-51831-7#Fig1\"> 1d </a> ). Since no alternatively spliced IL6 isoforms have been reported, we assumed that the proteins represented by the two bands resulted from the PTM of IL6. </p> <div class=\"c-article-section__figure js-c-reading-companion-figures-item\" id=\"figure-1\"> <figure> <figcaption> <b class=\"c-article-section__figure-caption\" id=\"Fig1\"> Fig. 1: IL6 Is Modified by N-Glycosylation in NSCLC. </b> </figcaption> <div class=\"c-article-section__figure-content\"> <div class=\"c-article-section__figure-item\"> <a class=\"c-article-section__figure-link\" href=\"/articles/s41467-024-51831-7/figures/1\" rel=\"nofollow\"> <picture> <source srcset=\"//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-024-51831-7/MediaObjects/41467_2024_51831_Fig1_HTML.png?as=webp\" type=\"image/webp\"/> </picture> </a> </div> <div class=\"c-article-section__figure-description\" id=\"figure-1-desc\"> <p> <b> a </b> Process of lung cancer cell isolation from malignant pleural effusions (MPEs) from lung cancer patients (generating MPE cells, MPECs). PanCK and TTF1 in MPECs were stained by immunofluorescence (IF). Images are representative of three technical replicates. Scale bar, 20 μm. <b> b </b> Pattern of immunoprecipitated (IP) secreted IL6 in the CM of MPECs (MPEC Nos. 1-12), as determined by Western blotting (WB). <i> n </i> = 1. The same process was used to analyze secreted IL6 throughout the study. <b> c </b> Quantity of total secreted IL6 in the NSCLC cell lines as measured by ELISAs. <i> n </i> = 3 technical replicates, mean ± SEM. <b> d </b> Patterns of secreted IL6 of NSCLC cell lines. <i> n </i> = 3 independent experiments, mean ± SEM. <b> e </b> Gene set enrichment analysis (GSEA) enrichment plot of N-linked glycosylation in cancerous lung tissue vs. normal lung tissue in the dataset GSE31210. NES, Nominal Enrichment Score. Nominal <i> p- </i> value was one-tailed test on the appropriate side of the null distribution. <b> f </b> Pattern of secreted IL6 in H460 cells ( <i> n </i> = 3 independent experiments, mean ± SEM) and MPECs (MPEC No. 13, <i> n </i> = 1) treated with or without NGI-1 (10 μM) for 48 h. <b> g </b> Pattern of secreted IL6 in MPECs (MPEC Nos. 14 and 15) with or without PNGase F digestion ( <i> n </i> = 1). <b> h </b> <i> IL6 </i> mRNA levels in the AS2 and HCC827 cells stably expressing full-length WT-hIL6 (namely, AS2-IL6-WT and HCC827-IL6-WT cells), as measured by RT-PCR. <i> GAPDH </i> was the internal control. <b> i </b> Quantity of total IL6 in the CM of AS2-IL6-WT and HCC827-IL6-WT cells, as measured by ELISAs ( <i> n </i> = 3 technical replicates, mean ± SEM). <b> j </b> Pattern of secreted IL6 in AS2-IL6-WT and HCC827-IL6-WT cells. The percentage proportions of H-IL6 and L-IL6 were quantified in the segmented histogram. <i> n </i> = 3 independent experiments (mean ± SEM). <b> k </b> Effect of NGI-1 (10 μM) on the pattern of secreted IL6 in the CM of AS2-IL6-WT and HCC827-IL6-WT cells. <i> n </i> = 3 independent experiments (mean ± SEM). <b> l </b> Quantitative characterization of N-glycans on purified secreted WT-IL6 from CM of AS2-IL6-WT cells by lectin microarray, as assessed with GlycoStation <sup> TM </sup> Reader 1200 (GlycoTechnica). The raw data was calculated and output automatically by GlycoStation Tools Pro Suite 1.5 software. <i> n </i> = 3 technical replicates, mean ± S.D. See also Supplementary Table <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 1 </a> . <b> m </b> Quantitative characterization of N-glycans on purified secreted WT-IL6 from CM of AS2-IL6-WT cells treated with NGI-1 (10 μM) by lectin microarray, as assessed with GlycoStation <sup> TM </sup> Reader 1200 (GlycoTechnica). The raw data was calculated and output automatically by GlycoStation Tools Pro Suite 1.5 software. <i> n </i> = 3 technical replicates, mean ± S.D. See also Supplementary Table <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 1 </a> . Source data are provided as a Source Data file. </p> </div> </div> <div class=\"u-text-right u-hide-print\"> <a aria-label=\"Full size image figure 1\" class=\"c-article__pill-button\" href=\"/articles/s41467-024-51831-7/figures/1\" rel=\"nofollow\"> <span> Full size image </span> </a> </div> </figure> </div> <p> N-glycosylation is the most common PTM of secreted proteins <sup> <a aria-label=\"Reference 40\" href=\"/articles/s41467-024-51831-7#ref-CR40\" id=\"ref-link-section-d6776021e1182\" title=\"Roth, J. Protein N-glycosylation along the secretory pathway: relationship to organelle topography and function, protein quality control, and cell interactions. Chem. Rev. 102, 285–303 (2002).\"> 40 </a> , <a aria-label=\"Reference 41\" href=\"/articles/s41467-024-51831-7#ref-CR41\" id=\"ref-link-section-d6776021e1185\" title=\"Bard, F. &amp; Chia, J. Cracking the glycome encoder: signaling, trafficking, and glycosylation. Trends Cell Biol. 26, 379–388 (2016).\"> 41 </a> </sup> , and elevated protein N-glycosylation has been reported in various cancers <sup> <a aria-label=\"Reference 42\" href=\"/articles/s41467-024-51831-7#ref-CR42\" id=\"ref-link-section-d6776021e1189\" title=\"Schjoldager, K. T., Narimatsu, Y., Joshi, H. J. &amp; Clausen, H. Global view of human protein glycosylation pathways and functions. Nat. Rev. Mol. Cell Biol. 21, 729–749 (2020).\"> 42 </a> </sup> . We then analyzed the expression trend of N-linked glycosylation process-related genes by <u> g </u> ene <u> s </u> et <u> e </u> nrichment <u> a </u> nalysis (GSEA) <sup> <a aria-label=\"Reference 43\" href=\"/articles/s41467-024-51831-7#ref-CR43\" id=\"ref-link-section-d6776021e1206\" title=\"Subramanian, A. et al. Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. Proc. Natl Acad. Sci. USA 102, 15545–15550 (2005).\"> 43 </a> , <a aria-label=\"Reference 44\" href=\"/articles/s41467-024-51831-7#ref-CR44\" id=\"ref-link-section-d6776021e1209\" title=\"Mootha, V. K. et al. PGC-1alpha-responsive genes involved in oxidative phosphorylation are coordinately downregulated in human diabetes. Nat. Genet. 34, 267–273 (2003).\"> 44 </a> </sup> of the Gene Expression Omnibus (GEO) dataset GSE31210 <sup> <a aria-label=\"Reference 45\" href=\"/articles/s41467-024-51831-7#ref-CR45\" id=\"ref-link-section-d6776021e1213\" title=\"Okayama, H. et al. Identification of genes upregulated in ALK-positive and EGFR/KRAS/ALK-negative lung adenocarcinomas. Cancer Res. 72, 100–111 (2012).\"> 45 </a> </sup> (Fig. <a href=\"/articles/s41467-024-51831-7#Fig1\"> 1e </a> ). Genes involved in the N-linked glycosylation process were enriched in cancer tissues; thus, we hypothesized that H-IL6 arises from N-linked glycosylation. Notably, treatment with the N-glycosylation inhibitor NGI-1 <sup> <a aria-label=\"Reference 46\" href=\"/articles/s41467-024-51831-7#ref-CR46\" id=\"ref-link-section-d6776021e1220\" title=\"Lopez-Sambrooks, C. et al. Oligosaccharyltransferase inhibition induces senescence in RTK-driven tumor cells. Nat. Chem. Biol. 12, 1023–1030 (2016).\"> 46 </a> </sup> diminished the amount of secreted H-IL6 from H460 cells and MPECs (Fig. <a href=\"/articles/s41467-024-51831-7#Fig1\"> 1f </a> ). When total MPEC-derived IL6 was treated with peptide-N-glycosidase F (PNGase F), which cleaves entire N-glycan chains from corresponding asparagine residues, the amount of H-IL6 was decreased (Fig. <a href=\"/articles/s41467-024-51831-7#Fig1\"> 1g </a> ). In addition, two N-glycosites consisting of the Asn-Xaa-Ser/Thr sequence were predicted in the hIL6 protein sequence by the NetNGlyc 1.0 server ( <a href=\"http://www.cbs.dtu.dk/services/NetNGlyc/\"> http://www.cbs.dtu.dk/services/NetNGlyc/ </a> ) <sup> <a aria-label=\"Reference 47\" href=\"/articles/s41467-024-51831-7#ref-CR47\" id=\"ref-link-section-d6776021e1238\" title=\"Blom, N., Sicheritz-Ponten, T., Gupta, R., Gammeltoft, S. &amp; Brunak, S. Prediction of post-translational glycosylation and phosphorylation of proteins from the amino acid sequence. Proteomics 4, 1633–1649 (2004).\"> 47 </a> </sup> (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 1 </a> ): asparagine (N) 73 and N172 (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 1 </a> , top). The score for only N73 exceeded the threshold score of 0.5 (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 1 </a> , bottom), indicating its potential as an N-glycosite. In addition, four O-glycosites (T166, T170, T171, and S174) were also predicted in IL6 (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 2 </a> and Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2a </a> ); however, the MS data (Fig. <a href=\"/articles/s41467-024-51831-7#Fig6\"> 6l </a> and Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 22 </a> ) provided evidence that, actually, O-glycan was detected only on T170, which existed in over 99% of secreted IL6 molecules. </p> <div class=\"c-article-section__figure js-c-reading-companion-figures-item\" id=\"figure-2\"> <figure> <figcaption> <b class=\"c-article-section__figure-caption\" id=\"Fig2\"> Fig. 2: N-glycosylated IL6 promotes the tyrosine phosphorylation and nuclear retention of STAT3. </b> </figcaption> <div class=\"c-article-section__figure-content\"> <div class=\"c-article-section__figure-item\"> <a class=\"c-article-section__figure-link\" href=\"/articles/s41467-024-51831-7/figures/2\" rel=\"nofollow\"> <picture> <source srcset=\"//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-024-51831-7/MediaObjects/41467_2024_51831_Fig2_HTML.png?as=webp\" type=\"image/webp\"/> </picture> </a> </div> <div class=\"c-article-section__figure-description\" id=\"figure-2-desc\"> <p> <b> a </b> Graphical depiction of N- and O- glycosites on the full-length human IL6 (hIL6) peptide. <b> b </b> Patterns of IL6 with various mutations in N-glycosites. Secreted WT-IL6, N73Q-IL6, N172Q-IL6, and N73Q + N172Q-IL6 were quantified and immunoprecipitated, and the pattern of IL6 was analyzed on the same WB with rhIL6 expressed in <i> E. coli </i> . NG-IL6: <u> N </u> - <u> g </u> lycosylated IL6; deNG-IL6: <u> N </u> - <u> g </u> lycosylation- <u> de </u> fective IL6; deG-IL6: <u> g </u> lycosylation- <u> de </u> ficient IL6. The stick diagram connecting to each form of IL6 band labeled by dotted line showed the locations of glycan chains on N-glycosites and O-glycosites. <i> n </i> = 3 independent experiments, mean ± SEM, two-tailed unpaired <i> t </i> -test, *** <i> p </i> &lt; 0.0001, * <i> p </i> = 0.0182, *** <i> p </i> &lt; 0.0001. <b> c </b> Patterns of secreted IL6 in AS2-IL6-WT, AS2-IL6-N73Q, AS2-IL6-N172Q, and AS2-IL6-N73Q + N172Q cells with or without NGI-1 (10 μM) pretreatment for 48 h. <i> n </i> = 3 independent experiments, mean ± SEM, two-tailed unpaired <i> t </i> -test, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001, n.s. <i> p </i> = 0.3423, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001. <b> d </b> Quantitative characterization of N-glycans on purified secreted WT-IL6 and N73Q-IL6 by lectin microarray, as assessed with GlycoStation <sup> TM </sup> Reader 1200 (GlycoTechnica). The raw data was calculated and output automatically by GlycoStation Tools Pro Suite 1.5 software. <i> n </i> = 3 technical replicates, mean ± S.D. See also Supplementary Table <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 1 </a> . <b> e </b> STAT3 and STAT3-pY705 levels induced by WT-IL6-CM or N73Q-IL6-CM (both IL6 = 1 ng/mL) over time in AS2-Vec, Beas-2B, and NL-20 cells. <i> n </i> = 3 independent experiments, mean ± SEM, two-tailed unpaired <i> t </i> -test, AS2-Vec: * <i> p </i> = 0.0139, ** <i> p </i> = 0.0282, n.s. 0.3663, ***0.0001; NL20: **0.0029, ***0.0004, **0.0014, **0.0010; Beas-2B: **0.0054, n.s. 0.9142, ***0.0006, ***&lt;0.0001. The samples derived from the same experiment but different gels for STAT3 and STAT3-pY705 were processed in parallel. <b> f </b> STAT3 intracellular localization under treatment with WT-IL6-CM or N73Q-IL6-CM (both IL6 = 1 ng/mL) over time in AS2-Vec cells, as determined by IF. The magnified insets show STAT3 localization at 24 h. Scale bars, 20 μm. <i> n </i> = 3 independent experiments, mean ± SEM, two-tailed unpaired <i> t </i> -test, ** <i> p </i> = 0.0012, *** <i> p </i> = 0.0002, n.s. <i> p </i> = 0.0939, *** <i> p </i> = 0.0001. <b> g </b> Transcriptome of AS2-Vec cells treated with WT-IL6-CM or N73Q-IL6-CM over time (see also Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 10 </a> ). Hierarchical clustering showed differentially expressed genes at 3 h. Source data are provided as a Source Data file. </p> </div> </div> <div class=\"u-text-right u-hide-print\"> <a aria-label=\"Full size image figure 2\" class=\"c-article__pill-button\" href=\"/articles/s41467-024-51831-7/figures/2\" rel=\"nofollow\"> <span> Full size image </span> </a> </div> </figure> </div> <p> Moreover, we transfected AS2 and HCC827 cells (which produce relatively low amounts of IL6) with a plasmid harboring human wild-type IL6 (WT-IL6), establishing AS2-IL6-WT and HCC827-IL6-WT cells. In these cells, IL6 increased at both the mRNA (Fig. <a href=\"/articles/s41467-024-51831-7#Fig1\"> 1h </a> ) and protein levels (Fig. <a href=\"/articles/s41467-024-51831-7#Fig1\"> 1i </a> ), and both H-IL6 and L-IL6 were detected (Fig. <a href=\"/articles/s41467-024-51831-7#Fig1\"> 1j </a> ), which was then verified by MALDI-TOF (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 3 </a> ). NGI-1 treatment reduced the levels of H-IL6 in both cell lines (Fig. <a href=\"/articles/s41467-024-51831-7#Fig1\"> 1k </a> ), confirming the existence of N-glycosylation. IL6 isolated from AS2-IL6-WT cells (termed “WT-IL6”) was subjected to lectin microarray analysis. Significant signals were detected for N-glycan-binding lectins, including PSA, LCA, ECA, ConA, etc., suggesting that WT-IL6 carries N-glycan structures (Fig. <a href=\"/articles/s41467-024-51831-7#Fig1\"> 1l </a> and Supplementary Table <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 1 </a> ). These signals were greatly diminished for WT-IL6 isolated from NGI-1-pretreated AS2-IL6-WT cells (Fig. <a href=\"/articles/s41467-024-51831-7#Fig1\"> 1m </a> ), confirming the existence of N-glycosylation in lung cancer cell-derived IL6. </p> <h3 class=\"c-article__sub-heading\" id=\"Sec4\"> N-glycosylated IL6 promotes the tyrosine phosphorylation and nuclear retention of STAT3 </h3> <p> Since IL6 is modified by N-glycosylation in lung cancer, we sought to explore its possible functional roles. To determine the dominant N-glycosite (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 1 </a> , Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 2 </a> , and Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2a </a> ), we generated three mutant IL6 plasmids by site-directed mutagenesis, replacing asparagine with glutamine <sup> <a aria-label=\"Reference 48\" href=\"/articles/s41467-024-51831-7#ref-CR48\" id=\"ref-link-section-d6776021e1463\" title=\"Yamaguchi, K. et al. Effects of site-directed removal of N-glycosylation sites in human erythropoietin on its production and biological properties. J. Biol. Chem. 266, 20434–20439 (1991).\"> 48 </a> </sup> , corresponding to the N-glycosylation-defective IL6s, namely, N73Q-IL6, N172Q-IL6 and N73Q + N172Q-IL6 (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 4a </a> ). Concurrently, AS2-IL6-N73Q cells, AS2-IL6-N172Q cells, and AS2-IL6-N73Q + N172Q cells were established. Secreted WT-IL6, N73Q-IL6, N172Q-IL6, and N73Q + N172Q-IL6 were quantified (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 4b, c </a> ), immunoprecipitated, and examined by WB, along with <u> r </u> ecombinant <u> h </u> uman IL6 (rhIL6) expressed in <i> E. coli </i> (Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2b </a> ). The Western blot for rhIL6 showed a single band representing the non-glycosylated IL6, designated as <b> deG-IL6 </b> ( <u> g </u> lycosylation- <u> de </u> ficient IL6). The blot for WT-IL6 form contained two major bands, one strong L-IL6 band and one weak H-IL6 band (Figs. <a href=\"/articles/s41467-024-51831-7#Fig1\"> 1j </a> and <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2b </a> ), in addition to a faint deG-IL6 band. Compared with the Western blot for WT-IL6, the blot for N73Q-IL6 form lost the H-IL6 band by over 97% (Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2b </a> ), but the results for N172Q-IL6 form exhibited only a slight reduction in the abundance of the H-IL6 band (Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2b </a> ). N172Q-IL6 form exhibited four bands on the Western blot: H-IL6, L-IL6, deG-IL6, and an additional faint band (labeled with an asterisk) between the H-IL6 and L-IL6 bands (Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2b </a> ). The pattern of N73Q + N172Q-IL6 form was similar to that of N73Q-IL6 form. These results showed that the glycoform of H-IL6 band carried only one N-glycan primarily located at N73, and such N-glycosylation is absent in L-IL6 band. Besides, there was no obvious IL6 band detected at a higher molecular weight (MW) than that of the H-IL6 band (Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2b </a> and Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 4d </a> ), suggesting that double N-glycosylated form at N73 and N172 may not exist in our IL6 samples. The stick diagram connected to each form of the IL6 band illustrated the actual locations of glycan chains on N-glycosites and O-glycosites (Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2b </a> ), as both H-IL6 and L-IL6 contained an O-glycan at T170, in accordance with the MS analysis (Fig. <a href=\"/articles/s41467-024-51831-7#Fig6\"> 6l </a> and Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 22 </a> ). </p> <p> In addition, treatment with NGI-1 (10 μM) significantly reduced the abundance of the H-IL6 band (Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2c </a> and Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 4d </a> ). The patterns of IL6 isolated from AS2-IL6-WT cells pretreated with NGI-1 or treated with PNGase F (Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2c </a> and Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 4d </a> ) in vitro were similar to those of IL6 isolated from untreated AS2-N73Q-IL6 cells. In comparison, the pattern of IL6 isolated from AS2-IL6-N73Q cells was not changed by NGI-1 or PNGase F treatment. Furthermore, the N-glycan intensity of N73Q-IL6 in the lectin microarray analysis was obviously reduced by ~50%, as compared to that of WT-IL6 (Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2d </a> ). </p> <p> Henceforth, we propose a nomenclature for each IL6 band based on its molecular weight position as depicted on the Western blot (from top to bottom), as follows: The H-IL6 band in Fig. <a href=\"/articles/s41467-024-51831-7#Fig1\"> 1 </a> , redefined in Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2 </a> , possesses the highest molecular weight and is designated as <b> NG-IL6 </b> for <u> N </u> - <u> g </u> lycosylated IL6. The majority of NG-IL6 glycoform carries glycans on N73 + T170, while a minority carries glycans on N172 + T170, albeit rarely detected. The L-IL6 band in Fig. <a href=\"/articles/s41467-024-51831-7#Fig1\"> 1 </a> , redefined in Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2 </a> , is termed <b> deNG-IL6 </b> for <u> N </u> - <u> g </u> lycosylation- <u> de </u> fective IL6, as the glycan chain resided solely on T170. Additionally, the IL6 band with the smallest molecular weight (Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2b, c </a> ) is named <b> deG-IL6 </b> ( <u> g </u> lycosylation- <u> de </u> ficient IL6), indicating the absence of both N- and O-glycans. This form was observed in various glycosylation-defective IL6 variants (N73Q-IL6, N172Q-IL6, N73Q + N172Q-IL6) and rhIL6 from <i> E. coli </i> . </p> <p> N-glycosylation modulates the functions of glycoproteins <sup> <a aria-label=\"Reference 42\" href=\"/articles/s41467-024-51831-7#ref-CR42\" id=\"ref-link-section-d6776021e1601\" title=\"Schjoldager, K. T., Narimatsu, Y., Joshi, H. J. &amp; Clausen, H. Global view of human protein glycosylation pathways and functions. Nat. Rev. Mol. Cell Biol. 21, 729–749 (2020).\"> 42 </a> </sup> . Based on protein sequence alignment (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 5a </a> ) and the 3-D structure of hIL6 (1ALU, <a href=\"https://www.rcsb.org/structure/1ALU\"> https://www.rcsb.org/structure/1ALU </a> ) and the IL6-GP80-GP130 hexameric complex (1P9M, <a href=\"https://www.rcsb.org/structure/1P9M\"> https://www.rcsb.org/structure/1P9M </a> ) in the Protein Data Bank <sup> <a aria-label=\"Reference 49\" href=\"/articles/s41467-024-51831-7#ref-CR49\" id=\"ref-link-section-d6776021e1622\" title=\"Bobby, R. et al. Functional implications of large backbone amplitude motions of the glycoprotein 130-binding epitope of interleukin-6. FEBS J. 281, 2471–2483 (2014).\"> 49 </a> </sup> , we found that the N73 residue was in space near both K47 (K74 in full-length IL6) in the AB loop and K151 (K178 in full-length IL6) in the E-helix within site III of hIL6 <sup> <a aria-label=\"Reference 49\" href=\"/articles/s41467-024-51831-7#ref-CR49\" id=\"ref-link-section-d6776021e1627\" title=\"Bobby, R. et al. Functional implications of large backbone amplitude motions of the glycoprotein 130-binding epitope of interleukin-6. FEBS J. 281, 2471–2483 (2014).\"> 49 </a> </sup> , which is located in the IL6-GP130 interface and thus essential for ligand-receptor binding (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 5b </a> ). Hence, we hypothesized that the N-glycosylation of IL6 at N73 affects the interaction of IL6 with GP130. </p> <p> Because we could not obtain sufficient amounts of either bioactive NG-IL6 or deNG-IL6, we used <u> c </u> onditioned <u> m </u> edium (CM) from AS2-IL6-WT cells (WT-IL6-CM) or AS2-IL6-N73Q cells (N73Q-IL6-CM) to treat AS2-Vec cells (transfected with empty vector), normal human bronchial NL-20 cells and Beas-2B cells for different durations. Before treatment, the IL6 content in each CM was measured by an ELISA (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 6a </a> ) and adjusted to 1 ng/mL (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 6b </a> ). In addition, the cytokine array results demonstrated that the constituents and quantity of cytokines contained in both undiluted and diluted WT-IL6-CM and N73Q-IL6-CM were nearly the same and the differences were negligible (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 6c, d </a> ); thus, the observed biological changes are likely induced primarily by different N-glycosylations of IL6. </p> <p> WT-IL6-CM-induced STAT3-pY705 was clearly retained for 24 h but not that of N73Q-IL6-CM, in both cancerous and normal cells (Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2e </a> ). Time-dependent IF imaging revealed that treatment with either WT-IL6-CM or N73Q-IL6-CM stimulated the nuclear import of STAT3 at 0.5 h (Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2f </a> ). Notably, WT-IL6-CM induced significantly longer nuclear retention of STAT3 than N73Q-IL6-CM up to 24 h (Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2g </a> , magnified inset). The difference in the STAT3 nuclear retention time was concordant with the persistence of the active STAT3 signal observed by WB (Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2e </a> ), showing different dynamics of STAT3 activation induced by the two forms of IL6. The amount of active STAT3 at 0.5 h was reduced by neutralizing antibodies against GP130 (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 7 </a> ), GP80 (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 8 </a> ), and IL6 (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 9 </a> ), indicating that the IL6-GP80-GP130 axis mediates WT-IL6-CM- and N73Q-IL6-CM-triggered STAT3 signaling. </p> <p> To assess the effects of differential STAT3 activation on transcriptional outcomes, we assessed changes in the transcriptome of the AS2-Vec cells induced by WT-IL6-CM or N73Q-IL6-CM from 0.5 to 24 h by microarray. Genes with a <u> f </u> old <u> c </u> hange (FC) in expression &gt;1.5 after normalization to the control medium treatment value were clustered with GeneSpring software (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 10 </a> and Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2g </a> ). The heatmap showed that N73Q-IL6-CM triggered markedly different gene expression profiles with enhanced intensity, particularly at 3 h, compared to WT-IL6-CM (Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2g </a> ). These results are not fully explained by dynamic changes in STAT3 activation, suggesting that deNG-IL6 induces pathways other than canonical JAK-STAT3 signaling to change the transcriptomic profiles. </p> <h3 class=\"c-article__sub-heading\" id=\"Sec5\"> deNG-IL6 induces gene signatures of EMT </h3> <p> To explore the functional effects of deNG-IL6 on cell behaviors, we analyzed genes with N73Q-IL6-CM-induced differential expression with <u> I </u> ngenuity <u> P </u> athway <u> A </u> nalysis (IPA) software. IPA-built-in “Disease or Functional Annotation” (Fig. <a href=\"/articles/s41467-024-51831-7#Fig3\"> 3a </a> ) predicted that the N73Q-IL6-CM-induced genes increased cell movement, migration, invasion, and EMT. Hierarchical clustering with GeneSpring showed that the expression of the top-ranked involved molecules was markedly increased by N73Q-IL6-CM, especially at 3 h (Fig. <a href=\"/articles/s41467-024-51831-7#Fig3\"> 3b </a> ), revealing that these genes may primarily contribute to metastasis. To further elucidate the functions of NG-IL6 and deNG-IL6, we analyzed the WT-IL6-CM- and N73Q-IL6-CM-induced transcriptomes by GSEA (Fig. <a href=\"/articles/s41467-024-51831-7#Fig3\"> 3c </a> ). N73Q-IL6-CM induced an enriched expression of hallmark EMT-related genes compared to WT-IL6-CM. </p> <div class=\"c-article-section__figure js-c-reading-companion-figures-item\" id=\"figure-3\"> <figure> <figcaption> <b class=\"c-article-section__figure-caption\" id=\"Fig3\"> Fig. 3: deNG-IL6 Induces EMT, Migration, and Metastasis in NSCLC. </b> </figcaption> <div class=\"c-article-section__figure-content\"> <div class=\"c-article-section__figure-item\"> <a class=\"c-article-section__figure-link\" href=\"/articles/s41467-024-51831-7/figures/3\" rel=\"nofollow\"> <picture> <source srcset=\"//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-024-51831-7/MediaObjects/41467_2024_51831_Fig3_HTML.png?as=webp\" type=\"image/webp\"/> </picture> </a> </div> <div class=\"c-article-section__figure-description\" id=\"figure-3-desc\"> <p> <b> a </b> IPA-identified “Disease and Functional Annotation” in the N73Q-IL6-CM-treated cells. A significant correlation was defined as <i> p </i> &lt; 0.001, evaluated by a right-tailed Fisher’s Exact Test. <b> b </b> Hierarchical clustering of the IPA-enriched EMT/metastasis-related genes. <b> c </b> GSEA enrichment plot of EMT process-related genes in the AS2-Vec cells under WT-IL6-CM and N73Q-IL6-CM treatment. Nominal <i> p </i> -value was one-tailed test on the appropriate side of the null distribution. <b> d </b> The morphology of (bright field, left panel, scale bars, 200 μm) and F-ACTIN stress fibers (IF, right panel, scale bars, 20 μm) in AS2-Vec, Beas-2B, and NL-20 cells under WT-IL6-CM or N73Q-IL6-CM treatment (24 h) are shown. F-ACTIN was stained with phalloidin-TRITC (gray) and detected with an FV3000 microscope (60x). <b> e </b> Expression of EMT-associated proteins in AS2-Vec cells treated with WT-IL6-CM or N73Q-IL6-CM for 24 h. <i> n </i> = 3 independent experiments, mean ± SEM, two-tailed unpaired <i> t </i> -test, E-CADHERIN: *** <i> p </i> = 0.0005; CLAUDIN1: *** <i> p </i> &lt; 0.0001; SLUG: *** <i> p </i> &lt; 0.0001; N-CADHERIN: *** <i> p </i> &lt; 0.0001; α-SMA: *** <i> p </i> &lt; 0.0001; β-ACTIN: n.s. <i> p </i> = 0.9997. The samples derived from the same experiment but different gels for E-CADHERIN, CLAUDIN1, SLUG, N-CADHERIN, α-SMA, and β-ACTIN were processed in parallel. <b> f </b> Stress fibers in the AS2-Vec cells under N73Q-IL6-CM treatment (24 h) with or without α-GP130 neutralizing Ab pretreatment (2 h). Images are representative of three independent experiments. Scale bars, 20 μm. See also Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 12 </a> . <b> g </b> Expression of mesenchymal proteins in AS2-Vec cells under N73Q-IL6-CM treatment (24 h) with α-GP80, α-IL6, and α-GP130 neutralizing Ab pretreatment for 2 h. <i> n </i> = 3 independent experiments, mean ± SEM, two-tailed unpaired <i> t </i> -test, SLUG: *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001; N-CADHERIN: *** <i> p </i> &lt; 0.0001, *** <i> p </i> = 0.0001, *** <i> p </i> &lt; 0.0001; α-SMA: *** <i> p </i> = 0.0001, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001; β-ACTIN: n.s. <i> p </i> = 0.9926, n.s. <i> p </i> = 0.6517, n.s. <i> p </i> = 0.8847. The samples derived from the same experiment but different gels for SLUG, N-CADHERIN, α-SMA, and β-ACTIN were processed in parallel. <b> h </b> Migration of AS2-Vec, AS2-IL6-WT, and AS2-IL6- N73Q cells through Transwell membranes <i> n </i> = 3 independent experiments, mean ± SEM, two-tailed unpaired <i> t </i> -test, *** <i> p </i> = 0.0001. <b> i </b> Migration of CL1-0 cells in a Transwell coculture system cocultured with AS2-Vec, AS2-IL6-WT, and AS2-IL6-N73Q cells. <i> n </i> = 3 independent experiments, the percentage of migrated CL1-0 cells is presented as the mean ± SEM, two-tailed unpaired <i> t </i> -test, *** <i> p </i> &lt; 0.0001. <b> j </b> In vivo metastasis of AS2-Vec, AS2-IL6-WT, and AS2-IL6-N73Q cells injected intravenously into nude mice ( <i> n </i> = 3 per cell type, 3 repeated independent experiments; see also Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 15 </a> ) evaluated by gross anatomical examination (left) and TissueFAXS-imaged H&amp;E staining (right). Each replicate experiment was terminated upon the death of the first animal; the necrotic lungs in these mice are indicated by dashed circles and were not used in further experiments. <b> k </b> The tumor area over the total lung area from ( <b> J </b> ), as quantified by HistoQuest. <i> n </i> = 3 independent animals for each glycoform in each experiment, 3 independent experiments, mean ± SEM, two-way ANOVA. <b> l </b> In vivo metastasis of AS2-IL6-WT-pLuc and AS2-IL6-N73Q-pLuc cells in nude mice after i.v. injection ( <i> n </i> = 3 per cell type) detected with IVIS. Source data are provided as a Source Data file. </p> </div> </div> <div class=\"u-text-right u-hide-print\"> <a aria-label=\"Full size image figure 3\" class=\"c-article__pill-button\" href=\"/articles/s41467-024-51831-7/figures/3\" rel=\"nofollow\"> <span> Full size image </span> </a> </div> </figure> </div> <p> We also characterized differentially expressed proteins in the AS2-IL6-WT and AS2-IL6-N73Q cells by two-dimensional fluorescence difference gel electrophoresis (2D-DIGE) (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 11a </a> ). Four spots indicating proteins differentially overexpressed in the AS2-IL6-N73Q cells were identified. Spots 1 and 2 were mainly composed of cytoskeleton-related proteins (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 11b </a> ), while a few mitochondrial proteins were characterized in spots 3 and 4. </p> <h3 class=\"c-article__sub-heading\" id=\"Sec6\"> deNG-IL6 promotes EMT, migration and metastasis in NSCLC </h3> <p> To evaluate how the N-glycosylation status of IL6 influences the EMT behavior of stimulated cells, we treated AS2-Vec, NL-20, and Beas-2B cells with WT-IL6-CM or N73Q-IL6-CM for 24 h, and phalloidin-stained ACTIN filaments (F-ACTIN) were observed by confocal microscopy. N73Q-IL6-CM induced a spindle-like morphology in all three cell types (Fig. <a href=\"/articles/s41467-024-51831-7#Fig3\"> 3d </a> ). In the WT-IL6-CM-treated cells, the majority of ACTIN filaments (F-ACTIN) delineated intercellular tight junctions, and few F-ACTIN stress fibers were found within the cytosol. In contrast, an increased number of stress fibers across the entire cytosolic compartment was observed in the N73Q-IL6-CM-treated cells (Fig. <a href=\"/articles/s41467-024-51831-7#Fig3\"> 3d </a> , arrowheads), indicating that N-glycosylation-defective N73Q-IL6 possesses a strong EMT-promoting capacity. </p> <p> We surveyed the expression of IL6-regulated EMT-related proteins in the AS2-Vec cells treated with WT-IL6-CM or N73Q-IL6-CM. Marked decreases in the levels of the epithelial proteins E-CADHERIN <sup> <a aria-label=\"Reference 50\" href=\"/articles/s41467-024-51831-7#ref-CR50\" id=\"ref-link-section-d6776021e1919\" title=\"Xiong, H. et al. Roles of STAT3 and ZEB1 proteins in E-cadherin down-regulation and human colorectal cancer epithelial-mesenchymal transition. J. Biol. Chem. 287, 5819–5832 (2012).\"> 50 </a> </sup> and CLAUDIN1 <sup> <a aria-label=\"Reference 51\" href=\"/articles/s41467-024-51831-7#ref-CR51\" id=\"ref-link-section-d6776021e1923\" title=\"Wei, W. et al. STAT3 signaling is activated preferentially in tumor-initiating cells in claudin-low models of human breast cancer. Stem cells 32, 2571–2582 (2014).\"> 51 </a> </sup> and increases in the levels of the mesenchymal proteins SLUG <sup> <a aria-label=\"Reference 52\" href=\"/articles/s41467-024-51831-7#ref-CR52\" id=\"ref-link-section-d6776021e1927\" title=\"Yao, C. et al. IGF/STAT3/NANOG/Slug signaling axis simultaneously controls epithelial-mesenchymal transition and stemness maintenance in colorectal cancer. Stem Cells 34, 820–831 (2016).\"> 52 </a> </sup> , N-CADHERIN <sup> <a aria-label=\"Reference 53\" href=\"/articles/s41467-024-51831-7#ref-CR53\" id=\"ref-link-section-d6776021e1931\" title=\"Na, Y. R., Lee, J. S., Lee, S. J. &amp; Seok, S. H. Interleukin-6-induced twist and N-cadherin enhance melanoma cell metastasis. Melanoma Res. 23, 434–443 (2013).\"> 53 </a> </sup> , and α-SMA <sup> <a aria-label=\"Reference 54\" href=\"/articles/s41467-024-51831-7#ref-CR54\" id=\"ref-link-section-d6776021e1935\" title=\"Gallucci, R. M., Lee, E. G. &amp; Tomasek, J. J. IL-6 modulates alpha-smooth muscle actin expression in dermal fibroblasts from IL-6-deficient mice. J. Invest. Dermatol. 126, 561–568 (2006).\"> 54 </a> </sup> were observed in the N73Q-IL6-CM-treated cells (Fig. <a href=\"/articles/s41467-024-51831-7#Fig3\"> 3e </a> ). Furthermore, blockade of GP130 with a neutralizing Ab reduced N73Q-IL6-CM-induced stress fiber formation (Fig. <a href=\"/articles/s41467-024-51831-7#Fig3\"> 3f </a> and Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 12 </a> , arrowheads). Blockade of GP130 and GP80, as well as IL6 neutralization, effectively reduced the expression of mesenchymal proteins, including SLUG, N-CADHERIN, and α-SMA (Fig. <a href=\"/articles/s41467-024-51831-7#Fig3\"> 3g </a> and Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 13 </a> ). </p> <p> We next examined the migration of AS2-Vec, AS2-IL6-WT, and AS2-IL6-N73Q cells by Transwell assays. The AS2-IL6-N73Q cells demonstrated the strongest migratory capacity, followed by the AS2-IL6-WT cells and the AS2-Vec cells (Fig. <a href=\"/articles/s41467-024-51831-7#Fig3\"> 3h </a> ), indicating that IL6 and, to a greater extent, deNG-IL6 enhance migration in an autocrine manner. To determine whether IL6 can promote the migration of neighboring cells in a paracrine manner, we cocultured CL1-0 cells (lung adenocarcinoma cells with low migratory potential) with AS2-Vec, AS2-IL6-WT, or AS2-IL6-N73Q cells (Fig. <a href=\"/articles/s41467-024-51831-7#Fig3\"> 3i </a> ). Similar to the results above, CL1-0 cells cocultured with AS2-IL6-N73Q cells demonstrated the strongest migratory ability, followed by those cocultured with AS2-IL6-WT cells and those cocultured with AS2-Vec cells (Fig. <a href=\"/articles/s41467-024-51831-7#Fig3\"> 3i </a> ). In addition, the migration capacity of AS2-IL6-N172Q cells was not significantly stronger than that of AS2-IL6-WT cells (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 14 </a> ). In contrast, the migration capacity of AS2-IL6- N73Q + N172Q cells was comparable to that of AS2-IL6-N73Q cells. These data provide good evidence that the deNG-IL6-mediated promotion of cell migration was attributed mainly to the loss of N-glycosylation on N73 and that the contribution of N-glycosylation on N172 was minimal. </p> <p> We then assessed the in vivo metastatic potential of the AS2-Vec, AS2-IL6-WT, and AS2-IL6-N73Q cells ( <i> n </i> = 3 per group, triplicate). Cells were injected intravenously (i.v.) into nude mice, which were then observed daily. At the death of the first animal, all of the mice were euthanized, terminating the experiment. In each of three experiments, the first mouse that died was from the AS2-IL6-N73Q group. Upon gross anatomical examination (Fig. <a href=\"/articles/s41467-024-51831-7#Fig3\"> 3j </a> , left panel) and hematoxylin and eosin (H&amp;E) staining (Fig. <a href=\"/articles/s41467-024-51831-7#Fig3\"> 3j </a> , right panel), a marked increase in lung metastases was observed in the AS2-IL6-N73Q group. Because necropsy revealed that the lungs of the earliest succumbing mouse were necrotic (as indicated by the dashed circles; these lungs were not used for further analysis), only two lung sections from this group were analyzed (Fig. <a href=\"/articles/s41467-024-51831-7#Fig3\"> 3j </a> and Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 15 </a> ). Ratio of the <u> t </u> umor area to the total <u> l </u> ung area (T/L) in the mice was quantified with the TissueFAXS system and used to represent the lung metastatic capacity (Fig. <a href=\"/articles/s41467-024-51831-7#Fig3\"> 3k </a> ). The AS2-IL6-N73Q group presented a remarkable average T/L value of 53.11%, consistent with the in vitro metastatic propensity (Fig. <a href=\"/articles/s41467-024-51831-7#Fig3\"> 3h </a> ), while the AS2-Vec and AS2-IL6-WT groups presented mean T/L values of 3.2% and 0.65%, respectively. We also established AS2-IL6-WT-pLuc and AS2-IL6-N73Q-pLuc cells and evaluated in vivo metastasis by the IVIS system (Fig. <a href=\"/articles/s41467-024-51831-7#Fig3\"> 3l </a> ). Similarly, the AS2-IL6-N73Q-pLuc cells showed a strong lung metastatic capacity. Collectively, our in vivo data suggest that deNG-IL6 potentiates lung cancer cell metastasis. </p> <h3 class=\"c-article__sub-heading\" id=\"Sec7\"> deNG-IL6 switches the GP130 signaling preference from the JAK-STAT3 axis to the SRC-YAP Axis </h3> <p> Next, we sought to clarify the signaling downstream of deNG-IL6-GP130 that potentiates metastasis. Constitutively aberrant activation of STAT3 contributes to the migration and metastasis of cancer cells <sup> <a aria-label=\"Reference 1\" href=\"/articles/s41467-024-51831-7#ref-CR1\" id=\"ref-link-section-d6776021e2013\" title=\"Yu, H., Lee, H., Herrmann, A., Buettner, R. &amp; Jove, R. Revisiting STAT3 signalling in cancer: new and unexpected biological functions. Nat. Rev. Cancer 14, 736–746 (2014).\"> 1 </a> </sup> . In addition to STAT3, IL6 activates ERK, AKT, and SRC via GP130 <sup> <a aria-label=\"Reference 5\" href=\"/articles/s41467-024-51831-7#ref-CR5\" id=\"ref-link-section-d6776021e2017\" title=\"Hunter, C. A. &amp; Jones, S. A. IL-6 as a keystone cytokine in health and disease. Nat. Immunol. 16, 448–457 (2015).\"> 5 </a> </sup> ; thus, we examined the expression of the total and active forms of these effectors in both the AS2-IL6-WT cells and the AS2-IL6-N73Q cells (Fig. <a href=\"/articles/s41467-024-51831-7#Fig4\"> 4a </a> ). Notably, higher levels of SRCpY416 were detected in the AS2-IL6-N73Q cells than in the AS2-IL6-WT cells, whereas ERK1/2, pERK, AKT, AKTpY308, and AKTpY473 showed comparable levels, suggesting the involvement of SRC in the deNG-IL6-induced alternative signaling pathway. </p> <div class=\"c-article-section__figure js-c-reading-companion-figures-item\" id=\"figure-4\"> <figure> <figcaption> <b class=\"c-article-section__figure-caption\" id=\"Fig4\"> Fig. 4: deNG-IL6 Switches the GP130 Signaling Preference from the JAK-STAT3 Axis to the SRC-YAP Axis. </b> </figcaption> <div class=\"c-article-section__figure-content\"> <div class=\"c-article-section__figure-item\"> <a class=\"c-article-section__figure-link\" href=\"/articles/s41467-024-51831-7/figures/4\" rel=\"nofollow\"> <picture> <source srcset=\"//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-024-51831-7/MediaObjects/41467_2024_51831_Fig4_HTML.png?as=webp\" type=\"image/webp\"/> </picture> </a> </div> <div class=\"c-article-section__figure-description\" id=\"figure-4-desc\"> <p> <b> a </b> Expression of proteins indicating the activation of ERK, AKT, and SRC in AS2-Vec, AS2-IL6-WT, and AS2-IL6-N73Q cells. <i> n </i> = 3 independent experiments, mean ± SEM, two-tailed unpaired <i> t </i> -test, pERK: n.s. <i> p </i> = 0.5646; ERK1/2: n.s. <i> p </i> = 0.9914; AKTpY308: n.s. <i> p </i> = 0.6477; AKTpS473: n.s. <i> p </i> = 0.0967; AKT: n.s. <i> p </i> = 0.7452; SRCpY416: *** <i> p </i> &lt; 0.0001; SRC: n.s. 0.2598; β-ACTIN: n.s. <i> p </i> = 1.000. FC: fold change relative to AS2-IL6-WT. The samples derived from the same experiment but different gels for pERK, ERK1/2, AKTpY308, AKTpY473, AKT, SRCpY416, SRC, and β-ACTIN were processed in parallel. <b> b </b> Hierarchical clustering of N73Q-IL6-induced genes downstream of SRC identified by IPA. <b> c </b> Recruitment of STAT3 and SRC to WT-IL6- and N73Q-IL6-activated GP130 determined based on IF images acquired with an FV3000 microscope (60x). Images are representative of three technical replicates. Scale bars, 20 μm. <b> d </b> Expression of proteins indicating the activation of STAT3 and SRC in the cytosolic and nuclear fractions of the AS2-IL6-WT and AS2-IL6-N73Q cells. <i> n </i> = 3 independent experiments, mean ± SEM, two-tailed unpaired <i> t </i> -test, STAT3pY705: *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001; STAT3: *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001; SRCpY416: *** <i> p </i> &lt; 0.0001, ** <i> p </i> = 0.0065; SRC: *** <i> p </i> &lt; 0.0001, ** <i> p </i> = 0.0075; LAMIN A/C: n.s. <i> p </i> = 0.1567, n.s. <i> p </i> = 1.0000; β-ACTIN: n.s. <i> p </i> = 1.0000, n.s. <i> p </i> = 1.0000. FC: fold change relative to AS2-IL6-WT. The samples derived from the same experiment but different gels for STAT3pY705, STAT3, SRCpY416, SRC, LAMIN A/C, and β-ACTIN were processed in parallel. <b> e </b> Effect of IL6 antibody (5 μg/mL), JAK-STAT3 inhibitors (S3I-201 = 10 μM, ruxolitinib = 2.5 μM), and SRC inhibitors (PP1 = 10 μM, PP2 = 10 μM, tirbanibulin = 10 μM) on the migration of AS2-IL6-WT and AS2-IL6-N73Q cells through the space made by culture inserts. Images were obtained at 0 and 24 h. Scale bars, 200 μm. Quantitative results are next to the images. <i> n </i> = 3 independent experiments, mean ± SEM, two-tailed unpaired <i> t </i> -test, AS2-IL6-N73Q cells: *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001; AS2-IL6-WT cells: *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001, ** <i> p </i> = 0.0026, ** <i> p </i> = 0.0082, *** <i> p </i> &lt; 0.0001. <b> f </b> Expression of SRC in the AS2-IL6-N73Q cells transduced with shSRC-bearing VSV-G particles. <i> n </i> = 3 independent experiments, mean ± SEM, one-tailed unpaired <i> t </i> -test, SRCpY416: ** <i> p </i> = 0.0022, ** <i> p </i> = 0.0011; SRC: *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001; β-ACTIN: n.s. <i> p </i> = 0.4500, n.s. <i> p </i> = 0.4267. FC: fold change relative to shpLKO.1. The samples derived from the same experiment but different gels for STAT3pY705, STAT3, SRCpY416, SRC, and β-ACTIN were processed in parallel. <b> g </b> Migration of the <i> SRC </i> -silenced AS2-IL6-N73Q cells. Scale bars, 200 μm. <i> n </i> = 3 independent experiments, mean ± SEM, one-tailed unpaired <i> t </i> -test, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001. <b> h </b> In vivo metastasis of SRC-silenced AS2-IL6-N73Q cells injected intravenously into nude mice ( <i> n </i> = 3 per cell type). TissueFAXS-acquired images of H&amp;E-stained lungs are shown (left). The tumor area over the total lung area was quantified by HistoQuest. <i> n </i> = 3 animals in each group, mean ± SEM, one-tailed unpaired <i> t </i> -test, ** <i> p </i> = 0.0062, ** <i> p </i> = 0.0062. Source data are provided as a Source Data file. </p> </div> </div> <div class=\"u-text-right u-hide-print\"> <a aria-label=\"Full size image figure 4\" class=\"c-article__pill-button\" href=\"/articles/s41467-024-51831-7/figures/4\" rel=\"nofollow\"> <span> Full size image </span> </a> </div> </figure> </div> <p> Further IPA prediction showed that the SRC-regulated genes were notably elevated in the N73Q-IL6-CM-treated AS2-Vec cells (Fig. <a href=\"/articles/s41467-024-51831-7#Fig4\"> 4b </a> ), suggesting that defective N-glycosylation of IL6 alters the preference of GP130 to activate its downstream signals. By IF imaging, we observed that WT-IL6-CM led GP130 to preferentially recruit STAT3 to the receptor complex (Fig. <a href=\"/articles/s41467-024-51831-7#Fig4\"> 4c </a> , left panel, merged image, light orange), while N73Q-IL6-CM triggered GP130 to alternatively recruit SRCs (Fig. <a href=\"/articles/s41467-024-51831-7#Fig4\"> 4c </a> , right panel, merged image, yellow), indicating that deNG-IL6 switches the effector preference of GP130 from STAT3 to SRC. In addition, an enriched amount of active STAT3 was observed in the AS2-IL6-WT cells, particularly in the nuclear lysates (Fig. <a href=\"/articles/s41467-024-51831-7#Fig4\"> 4d </a> ). In comparison, the amount of active SRC was enriched in the cytosolic lysates of the AS2-IL6-N73Q cells. These observations were consistent with the signals of GP130-recruited intracellular STAT3 and SRC (Fig. <a href=\"/articles/s41467-024-51831-7#Fig4\"> 4c </a> ), demonstrating that the pattern of N-glycosylation of IL6 influences the preference of downstream signals of GP130. </p> <p> IL6 neutralization effectively inhibited the migration of AS2-IL6-N73Q (Fig. <a href=\"/articles/s41467-024-51831-7#Fig4\"> 4e </a> ), AS2-Vec (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 16 </a> ), and AS2-IL6-WT cells (Fig. <a href=\"/articles/s41467-024-51831-7#Fig4\"> 4e </a> ). To confirm the role of active SRC in N73Q-IL6-induced migration, we treated the AS2-Vec (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 16 </a> ), AS2-IL6-WT, and AS2-IL6-N73Q cells with the SRC inhibitors PP1, PP2, and tirbanibulin and assessed their migratory capacity (Fig. <a href=\"/articles/s41467-024-51831-7#Fig4\"> 4e </a> , upper panel). Compared to the mock control, the SRC inhibitors efficiently inhibited the migration of the AS2-IL6-N73Q cells, while the STAT3 inhibitor S3I-201 and the JAK2 inhibitor ruxolitinib showed modest inhibitory effects. In contrast, S3I-201 and ruxolitinib inhibited the migration of AS2-vec and AS2-IL6-WT cells more effectively than SRC inhibitors (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 16 </a> ), suggesting that STAT3 shows a lower contribution to migration than SRC does in AS2-IL6-N73Q cells. Silencing of SRC (Fig. <a href=\"/articles/s41467-024-51831-7#Fig4\"> 4f </a> ) impeded the migration of the AS2-IL6-N73Q cells (Fig. <a href=\"/articles/s41467-024-51831-7#Fig4\"> 4g </a> ). In addition, the in vivo metastases assessed by i.v. inoculation in nude mice showed that the AS2-IL6-N73Q cells transduced with empty vector (shpLKO.1 cells) demonstrated a strong metastatic capacity (Fig. <a href=\"/articles/s41467-024-51831-7#Fig4\"> 4h </a> ), whereas both the SRC-sh1 and SRC-sh2 cells showed a significantly decreased number of lung metastatic foci and T/L ratio (68.95%, 0.04%, and 0.23%, respectively, <i> p </i> &lt; 0.001, Fig. <a href=\"/articles/s41467-024-51831-7#Fig4\"> 4h </a> ). These data showed that the switch of the GP130 downstream effector to SRC is essential for deNG-IL6-induced metastasis. </p> <p> To determine whether the glycan attached to N73 or N172 is accountable for the signaling switch, we evaluated the signaling expression in whole-cell lysates, as well as nuclear and cytosolic fractions, extracted from AS2-Vec cells, AS2-IL6-WT cells, AS2-IL6-N73Q cells, and AS2-IL6-N172Q cells. Compared to AS2-IL6-WT cells, a signaling shift from the JAK-STAT3 pathway to the alternatively activated SRC-YAP-SOX axis was exclusively observed in AS2-IL6-N73Q cells (but not in AS2-IL6-N172Q cells) in both whole-cell lysates (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 17a </a> ) and nuclear (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 17b </a> ) and cytosolic (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 17c </a> ) fractions. These findings suggest that N-glycosylation on N73, rather than N172, is accountable for the switch in IL6 downstream signaling from the STAT3 axis to the SRC axis. </p> <p> In addition, to confirm that the N-glycan but not the amino acid substitution changed the properties of IL6, we also established AS2 cells that stably expressed N73A-IL6 and N73V-IL6 (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 18a–c </a> ). Compared to WT-IL6, both N73A-IL6 and N73V-IL6 demonstrated a decrease in NG-IL6 (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 18c </a> ), similar to N73Q-IL6. Both N73A-IL6-CM and N73V-IL6-CM induced similar patterns of STAT3 activation (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 18d </a> ) and localization of STAT3 and SRC (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 18e </a> ) as N73Q-IL6-CM. Furthermore, similar to AS2-IL6-N73Q cells, both AS2-IL6-N73A and AS2-IL6-N73V cells presented strong in vitro migratory (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 18f </a> ) and in vivo metastatic capacities (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 18g, h </a> ). The above findings indicated that the loss of N-glycosylation but not the change in amino acids on IL6 induces the signaling shift. </p> <h3 class=\"c-article__sub-heading\" id=\"Sec8\"> SOX2 contributes to deNG-IL6-induced SRC-YAP axis-promoted migration and metastasis </h3> <p> Because YAP is a transcriptional regulator downstream of GP130-SRC <sup> <a aria-label=\"Reference 12\" href=\"/articles/s41467-024-51831-7#ref-CR12\" id=\"ref-link-section-d6776021e2340\" title=\"Taniguchi, K. et al. A gp130-Src-YAP module links inflammation to epithelial regeneration. Nature 519, 57–62 (2015).\"> 12 </a> , <a aria-label=\"Reference 55\" href=\"/articles/s41467-024-51831-7#ref-CR55\" id=\"ref-link-section-d6776021e2343\" title=\"Romera-Hernandez, M. et al. Yap1-driven intestinal repair is controlled by group 3 innate lymphoid cells. Cell Rep. 30, 37–45.e33 (2020).\"> 55 </a> </sup> , we analyzed our microarray data (Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2h </a> ) by GSEA, characterized the enriched YAP conserved signature in the N73Q-IL6CM-treated AS2-Vec cells (Fig. <a href=\"/articles/s41467-024-51831-7#Fig5\"> 5a </a> ) and found that these cells exhibited elevated expression of YAP-regulated genes (Fig. <a href=\"/articles/s41467-024-51831-7#Fig5\"> 5b </a> ). In addition, proteins of the YAP pathway, including YAP, LATS1, TAZ and MST1, were found to be activated and enriched in the nuclear fraction of the AS2-IL6-N73Q cells (Fig. <a href=\"/articles/s41467-024-51831-7#Fig5\"> 5c </a> ), concordant with the elevated cytosolic active SRC (Fig. <a href=\"/articles/s41467-024-51831-7#Fig4\"> 4d </a> ). </p> <div class=\"c-article-section__figure js-c-reading-companion-figures-item\" id=\"figure-5\"> <figure> <figcaption> <b class=\"c-article-section__figure-caption\" id=\"Fig5\"> Fig. 5: SOX2 Contributes to deNG-IL6-Induced SRC-YAP Axis-Promoted Migration and Metastasis. </b> </figcaption> <div class=\"c-article-section__figure-content\"> <div class=\"c-article-section__figure-item\"> <a class=\"c-article-section__figure-link\" href=\"/articles/s41467-024-51831-7/figures/5\" rel=\"nofollow\"> <picture> <source srcset=\"//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-024-51831-7/MediaObjects/41467_2024_51831_Fig5_HTML.png?as=webp\" type=\"image/webp\"/> </picture> </a> </div> <div class=\"c-article-section__figure-description\" id=\"figure-5-desc\"> <p> <b> a </b> GSEA enrichment plot for the conserved YAP signature in the AS2-Vec cells under WT-IL6 and N73Q-IL6 treatment. Nominal <i> p- </i> value was one-tailed test on the appropriate side of the null distribution. <b> b </b> Hierarchical clustering of the N73Q-IL6-induced YAP downstream genes identified by IPA. <b> c </b> Expression of YAP pathway-associated proteins in the nuclear fraction of the AS2-IL6-WT and AS2- IL6-N73Q cells. <i> n </i> = 3 independent experiments, mean ± SEM, two-tailed unpaired <i> t </i> -test, YAPpY357: *** <i> p </i> &lt; 0.0001; YAP: *** <i> p </i> &lt; 0.0001; pLATS: *** <i> p </i> = 0.0001; LATS1: *** <i> p </i> &lt; 0.0001; pTAZ: *** <i> p </i> = 0.0001; TAZ: *** <i> p </i> &lt; 0.0001; MST1: *** <i> p </i> &lt; 0.0001; LAMIN A/C: n.s. <i> p </i> = 0.0779. FC: fold change relative to AS2-IL6-WT. The samples derived from the same experiment but different gels for YAPpY357, YAP, pLATS, LATS1, pTAZ, TAZ, MST1, and LAMIN A/C were processed in parallel. <b> d </b> Expression of stemness-associated proteins in the AS2-IL6-WT and AS2-IL6-N73Q cells. <i> n </i> = 3 independent experiments, mean ± SEM, two-tailed unpaired <i> t </i> -test, SOX2: *** <i> p </i> &lt; 0.0001; NANOG: *** <i> p </i> &lt; 0.0001; OCT3/4: *** <i> p </i> = 0.0010; β-ACTIN: n.s. <i> p </i> = 0.9814. FC: fold change relative to AS2-IL6-WT. The samples derived from the same experiment but different gels for SOX2, NANOG, OCT3/4, and β-ACTIN were processed in parallel. <b> e </b> Expression of stemness-associated proteins in the AS2-IL6-N73Q cells treated with YAPi VP at increasing doses. <i> n </i> = 3 independent experiments, percentage expression relative to none, mean ± SEM, two-tailed unpaired <i> t </i> -test, YAP: *** <i> p </i> &lt; 0.0001, ** <i> p </i> = 0.0013, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001; TAZ: *** <i> p </i> = 0.0001, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001; SOX2: *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001; β-ACTIN: n.s. <i> p </i> = 0.5000, n.s. <i> p </i> = 0.5000, n.s. <i> p </i> = 0.5000, n.s. <i> p </i> = 0.5000, n.s. <i> p </i> = 0.5000, n.s. <i> p </i> = 0.5000. The samples derived from the same experiment but different gels for YAP, TAZ, SOX2, and β-ACTIN were processed in parallel. <b> f </b> Migration of the AS2-IL6-N73Q cells under VP treatment. <i> n </i> = 3 independent experiments, mean ± SEM, one-tailed unpaired <i> t </i> -test, *** <i> p </i> &lt; 0.0001. <b> g </b> Expression of the <i> YAP </i> gene ( <b> g </b> ) and YAP protein ( <b> h </b> ) in the AS2-IL6-N73Q cells transduced with shSRC-bearing VSV-G particles. <i> n </i> = 3 independent experiments, percentage expression relative to shpLKO.1, mean ± SEM, one-tailed unpaired <i> t </i> -test, YAP: *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001; β-ACTIN: n.s. <i> p </i> = 0.5000, n.s. <i> p </i> = 0.5000. The cDNA or protein samples derived from the same experiment but different gels for <i> YAP </i> , <i> GAPDH </i> , YAP, and β-ACTIN were processed in parallel. <b> i </b> Migration of the <i> YAP </i> -silenced AS2-IL6-N73Q cells. Scale bars, 200 μm. <i> n </i> = 3 independent experiments, mean ± SEM, one-tailed unpaired <i> t </i> -test, *** <i> p </i> = 0.0001, *** <i> p </i> &lt; 0.0001. <b> j </b> In vivo metastasis of the <i> YAP </i> -silenced AS2-IL6-N73Q cells injected intravenously into nude mice ( <i> n </i> = 3 per cell type). TissueFAXS-acquired images of H&amp;E-stained lungs are shown (left). The tumor area over the total lung area was quantified by HistoQuest. <i> n </i> = 3 animals in each group, mean ± SEM, one-tailed unpaired <i> t </i> -test, ** <i> p </i> = 0.0015, ** <i> p </i> = 0.0015. <b> k </b> Expression of <i> SOX2 </i> mRNA ( <b> k </b> ) and protein ( <b> l </b> ) in the <i> SOX2 </i> -silenced AS2-IL6-N73Q cells. <i> n </i> = 3 independent experiments, percentage expression relative to shpLKO.1, mean ± SEM, one-tailed unpaired <i> t </i> -test, SOX2: *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001; β-ACTIN: n.s. <i> p </i> = 0.5000, n.s. <i> p </i> = 0.5000. The cDNA or protein samples derived from the same experiment but different gels for <i> SOX2 </i> , <i> GAPDH </i> , SOX2, and β-ACTIN were processed in parallel. <b> m </b> Migration of the <i> SOX2 </i> -silenced AS2-IL6-N73Q cells. Scale bars, 200 μm. <i> n </i> = 3 independent experiments, mean ± SEM, one-tailed unpaired <i> t </i> -test, *** <i> p </i> &lt; 0.0001, *** <i> p </i> = 0.0001. <b> n </b> In vivo metastasis of the <i> SOX2 </i> -silenced AS2-IL6-N73Q cells intravenously injected into nude mice ( <i> n </i> = 3 per cell type). TissueFAXS-acquired images of H&amp;E-stained lungs are shown (left). The tumor area over the total lung area was quantified by HistoQuest. <i> n </i> = 3 animals in each group, mean ± SEM, one-tailed unpaired <i> t </i> -test, ** <i> p </i> = 0.0023, * <i> p </i> = 0.0023. Source data are provided as a Source Data file. </p> </div> </div> <div class=\"u-text-right u-hide-print\"> <a aria-label=\"Full size image figure 5\" class=\"c-article__pill-button\" href=\"/articles/s41467-024-51831-7/figures/5\" rel=\"nofollow\"> <span> Full size image </span> </a> </div> </figure> </div> <p> Activated YAP promotes regeneration of the intestinal epithelium and enhances the expression and activity of stemness-related proteins, including SOX2, NANOG, and OCT3/4 <sup> <a aria-label=\"Reference 12\" href=\"/articles/s41467-024-51831-7#ref-CR12\" id=\"ref-link-section-d6776021e2711\" title=\"Taniguchi, K. et al. A gp130-Src-YAP module links inflammation to epithelial regeneration. Nature 519, 57–62 (2015).\"> 12 </a> , <a aria-label=\"Reference 29\" href=\"/articles/s41467-024-51831-7#ref-CR29\" id=\"ref-link-section-d6776021e2714\" title=\"Bora-Singhal, N. et al. YAP1 regulates OCT4 activity and SOX2 expression to facilitate self-renewal and vascular mimicry of stem-like cells. Stem cells 33, 1705–1718 (2015).\"> 29 </a> </sup> . SOX2 was markedly enriched in the AS2-IL6-N73Q cells (Fig. <a href=\"/articles/s41467-024-51831-7#Fig5\"> 5d </a> ). To verify the role of YAP in deNG-IL6-promoted migration and metastasis, we treated the AS2-IL6-N73Q cells with the YAP inhibitor (YAPi) verteporfin (VP) <sup> <a aria-label=\"Reference 56\" href=\"/articles/s41467-024-51831-7#ref-CR56\" id=\"ref-link-section-d6776021e2721\" title=\"Tsuji, T. et al. YAP1 mediates survival of ALK-rearranged lung cancer cells treated with alectinib via pro-apoptotic protein regulation. Nat. Commun. 11, 74 (2020).\"> 56 </a> </sup> . VP simultaneously reduced the levels of YAP, TAZ, and downstream SOX2 in a dose-dependent manner (Fig. <a href=\"/articles/s41467-024-51831-7#Fig5\"> 5e </a> ) and impeded the migration of the AS2-IL6-N73Q cells (Fig. <a href=\"/articles/s41467-024-51831-7#Fig5\"> 5f </a> ). <i> YAP </i> silencing in the AS2-IL6-N73Q cells with VSV-Gs with two independent shRNAs led to a successful reduction in <i> YAP </i> at both the RNA (Fig. <a href=\"/articles/s41467-024-51831-7#Fig5\"> 5g </a> ) and protein levels (Fig. <a href=\"/articles/s41467-024-51831-7#Fig5\"> 5h </a> ). Compared to the shpLKO.1 cells, both the YAP-sh1 and YAP-sh2 cells exhibited impeded in vitro migration (28.85% and 50.48%, respectively, <i> p </i> &lt; 0.001, Fig. <a href=\"/articles/s41467-024-51831-7#Fig5\"> 5i </a> ) and in vivo metastasis (0.004% and 0.008, respectively, <i> p </i> &lt; 0.001, Fig. <a href=\"/articles/s41467-024-51831-7#Fig5\"> 5j </a> ). </p> <p> We used the same strategy to further address the role of SOX2 in deNG-IL6-promoted migration and metastasis mediated through SRC and YAP. Similarly, SOX2 was successfully reduced at both the RNA and protein levels (Fig. <a href=\"/articles/s41467-024-51831-7#Fig5\"> 5k, l </a> ). Both in vitro migration (Fig. <a href=\"/articles/s41467-024-51831-7#Fig5\"> 5m </a> ) and in vivo metastasis (Fig. <a href=\"/articles/s41467-024-51831-7#Fig5\"> 5n </a> ) were reduced in the <i> SOX2 </i> -silenced SOX2-sh1 and SOX2-sh2 cells compared to the shpLKO.1 cells. By pharmaceutical and genetic inhibitory approaches, we demonstrated that deNG-IL6 promotes the migration and metastasis of lung cancer cells via a switch of GP130 downstream signaling from STAT3 to the SRC-YAP-SOX2 axis. </p> <h3 class=\"c-article__sub-heading\" id=\"Sec9\"> Osimertinib induces expression of deNG-IL6 and subsequent activation of the SRC-YAP-SOX2 axis </h3> <p> Increased IL6 expression was reported in osimertinib-resistant NSCLC cells and patients <sup> <a aria-label=\"Reference 22\" href=\"/articles/s41467-024-51831-7#ref-CR22\" id=\"ref-link-section-d6776021e2780\" title=\"Kim, S. M. et al. Activation of IL-6R/JAK1/STAT signaling induces de novo resistance to irreversible EGFR inhibitors in non-small cell lung cancer with T790M resistance mutation. Mol. Cancer Ther. 11, 2254–2264 (2012).\"> 22 </a> </sup> , but glycosylation pattern and the biological properties of IL6 are unclear. To study this issue, we established osimertinib-resistant (OR) PC9 and HCC827 lung adenocarcinoma cell lines (PC9-OR and HCC827-OR cells, respectively) harboring mutant EGFR, and the resistance was maintained by constant culture in medium containing osimertinib (5 μM). Compared to the corresponding parental cells (PC9-P and HCC827-P), PC9-OR and HCC827-OR cells exhibited increased resistance to osimertinib (Fig. <a href=\"/articles/s41467-024-51831-7#Fig6\"> 6a </a> ) and elevated IL6 secretion (Fig. <a href=\"/articles/s41467-024-51831-7#Fig6\"> 6b </a> and Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 19 </a> ). </p> <div class=\"c-article-section__figure js-c-reading-companion-figures-item\" id=\"figure-6\"> <figure> <figcaption> <b class=\"c-article-section__figure-caption\" id=\"Fig6\"> Fig. 6: Inhibition of osimertinib-induced deNG-IL6 restores TKI sensitivity in TKI-resistant NSCLC. </b> </figcaption> <div class=\"c-article-section__figure-content\"> <div class=\"c-article-section__figure-item\"> <a class=\"c-article-section__figure-link\" href=\"/articles/s41467-024-51831-7/figures/6\" rel=\"nofollow\"> <picture> <source srcset=\"//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-024-51831-7/MediaObjects/41467_2024_51831_Fig6_HTML.png?as=webp\" type=\"image/webp\"/> </picture> </a> </div> <div class=\"c-article-section__figure-description\" id=\"figure-6-desc\"> <p> <b> a </b> Survival of parental and OR PC9 and HCC827 cells with osimertinib treatment (72 h). <i> n </i> = 6 independent experiments per data point, mean ± SEM. <b> b </b> Levels of IL6 secreted by the parental and OR PC9 and HCC827 cells after 72 h of attachment. <i> n </i> = 3 technical replicates, mean ± SEM, two-tailed unpaired <i> t </i> -test, PC9-OR: *** <i> p </i> &lt; 0.0001; HCC827-OR: *** <i> p </i> &lt; 0.0001. <b> c </b> mRNA expression of <i> IL6 </i> in OR cells transduced with sh <i> IL6 </i> -expressing VSV-G particles. The samples were derived from the same experiment for <i> IL6 </i> and <i> GAPDH </i> were seperated in the same gel. <b> d </b> Quantity of total secreted IL6 in OR-sh <i> IL6 </i> cells as measured by ELISAs. <i> n </i> = 3 technical replicates, mean ± SEM, one-tailed unpaired <i> t </i> -test, PC9-OR: *** <i> p </i> &lt; 0.0001; HCC827-OR: *** <i> p </i> &lt; 0.0001. <b> e </b> Morphology of OR and OR-sh <i> IL6 </i> cells treated with osimertinib (5 μM) for 72 h. Scale bar, 100 μm. <b> f </b> Survival of PC9-OR-sh <i> IL6 </i> and HCC827-OR-sh <i> IL6 </i> cells treated with osimertinib at 5 μM (72 h). <i> n </i> = 3 independent experiments, mean ± SEM, two-tailed unpaired <i> t </i> -test, PC9-OR: *** <i> p </i> &lt; 0.0001; HCC827-OR: *** <i> p </i> &lt; 0.0001. <b> g </b> Pattern of secreted IL6 in the parental and OR PC9 and HCC827 cells. Because the PC9-P and HCC827-P cells secreted little IL6, which prevented analysis of its glycosylation pattern, we established PC9-IL6-WT and HCC827-IL6-WT cells as control cell lines. The proportions of heavy IL6 in these cells were quantified and are shown in the right panel. <i> n </i> = 3 independent experiments, mean ± SEM, two-tailed unpaired <i> t </i> -test, PC9-OR: *** <i> p </i> &lt; 0.0001; HCC827-OR: *** <i> p </i> &lt; 0.0001. <b> h </b> GSEA of N-linked glycosylation pathways and EMT, SRC and YAP activation statuses in the transcriptome dataset GSE106765 from the HCC827-OR-Taka cells. Nominal <i> p </i> -value was one-tailed test on the appropriate side of the null distribution. <b> i </b> Expression of N-OST genes in the parental and OR PC9 and HCC827 cells, as detected by qPCR. <i> n </i> = 3 independent experiments, mean ± SEM, one-tailed unpaired <i> t </i> -test, PC9-OR: ** <i> p </i> = 0.0065, * <i> p </i> = 0.0193, n.s. <i> p </i> = 0.1579, *** <i> p </i> &lt; 0.0001, ** <i> p </i> = 0.0099, ** <i> p </i> = 0.0047; HCC827-OR: 827-OR: *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001, * <i> p </i> = 0.0160, *** <i> p </i> = 0.004, ** <i> p </i> = 0.0025, ** <i> p </i> = 0.0014. <b> j </b> N-glycosylated IL6 detected by lectin blotting with <u> w </u> heat <u> g </u> erm <u> a </u> gglutinin (WGA). The proportions of WGA-detected NG-IL6 in WT-IL6 or OR-IL6 were quantified and are shown in the ri <u> g </u> ht p <u> a </u> nel. <i> n </i> = 3 independent experiments, mean ± SEM, one-tailed unpaired <i> t </i> -test, PC9-OR: *** <i> p </i> &lt; 0.0001; HCC827-OR: *** <i> p </i> &lt; 0.0001. <b> k </b> Quantitative characterization of N-glycans on purified secre <i> t </i> ed WT-IL6 or OR-IL6 from CM of PC9 or HCC827 cell derivatives by lectin microarray, as assessed with GlycoStation <sup> TM </sup> Reader 1200 (GlycoTechnica). The raw data was calculated and output automatically by GlycoStation Tools Pro Suite 1.5 software. <i> n </i> = 3 technical replicates, mean ± S.D. See also Supplementary Table <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 1 </a> . <b> l </b> Characterization of glycosylation in IL6 by MS. See also Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 22 </a> . <b> m </b> Pattern of secreted WT-IL6 or OR-IL6 with or without PNGase F digestion. The percentage proportions of H-IL6 and L-IL6 were quantified in the segmented histogram. <i> n </i> = 3 independent experiments, mean ± SEM. Source data are provided as a Source Data file. </p> </div> </div> <div class=\"u-text-right u-hide-print\"> <a aria-label=\"Full size image figure 6\" class=\"c-article__pill-button\" href=\"/articles/s41467-024-51831-7/figures/6\" rel=\"nofollow\"> <span> Full size image </span> </a> </div> </figure> </div> <p> For determination of the function of osimertinib-induced IL6 in resistance, IL6 was inhibited by either an IL6-neutralizing antibody or shRNA against <i> IL6 </i> in PC9-OR and HCC827-OR cells. The IL6 antibody reduced the survival of PC9-OR and HCC827-OR cells by ~15% at 20 μg/mL in osimertinib-containing medium (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 20 </a> ). Silencing <i> IL6 </i> successfully reduced the mRNA levels (Fig. <a href=\"/articles/s41467-024-51831-7#Fig6\"> 6c </a> ) and secretion levels of IL6 (Fig. <a href=\"/articles/s41467-024-51831-7#Fig6\"> 6d </a> ) and restored the sensitivity to osimertinib at tolerant concentrations of OR cells (Fig. <a href=\"/articles/s41467-024-51831-7#Fig6\"> 6e </a> ). The survival was reduced to 57.2% and 16.5% in PC9-OR and HCC827-OR cells, respectively (Fig. <a href=\"/articles/s41467-024-51831-7#Fig6\"> 6f </a> ). In summary, elevated IL6 secretion contributes to cellular resistance to osimertinib. </p> <p> We next examined the pattern of IL6 glycoforms secreted by PC9-OR and HCC827-OR cells and the corresponding parental cell lines. Because the PC9-P and HCC827-P cells secreted little IL6, which prevented analysis of its glycosylation pattern, we established PC9-IL6-WT and HCC827-IL6-WT cells as control cell lines (Figs. <a href=\"/articles/s41467-024-51831-7#Fig6\"> 6g </a> , <a href=\"/articles/s41467-024-51831-7#Fig1\"> 1j </a> ). Both H-IL6 and L-IL6 were detected in the CM of the PC9-IL6-WT and HCC827-IL6-WT cells (Fig. <a href=\"/articles/s41467-024-51831-7#Fig6\"> 6g </a> , upper left and lower left). Unlike the distribution of H-IL6 and L-IL6 in the parental cells, almost all the IL6 secreted by the PC9-OR and HCC827-OR cells was L-IL6 (Fig. <a href=\"/articles/s41467-024-51831-7#Fig6\"> 6g </a> , upper right and lower right), similar to the distribution of deNG-IL6 (Fig. <a href=\"/articles/s41467-024-51831-7#Fig1\"> 1k </a> ). Therefore, we hypothesized that long-term EGFR-TKI exposure may influence the expression of oligosaccharyltransferases for N-glycosylation (N-OSTs) and then interfere with the N-glycosylation process on IL6. </p> <p> We then analyzed the transcriptome profile of HCC827 cells with or without acquired osimertinib resistance established by Takahashi (renamed P-Taka or OR-Taka, GSE106765) <sup> <a aria-label=\"Reference 57\" href=\"/articles/s41467-024-51831-7#ref-CR57\" id=\"ref-link-section-d6776021e3072\" title=\"Takahashi, A. et al. Ankyrin repeat domain 1 overexpression is associated with common resistance to afatinib and osimertinib in EGFR-mutant lung cancer. Sci. Rep. 8, 14896 (2018).\"> 57 </a> </sup> . The expression of N-glycosylation-related genes was enriched in the osimertinib-sensitive P-Taka cells but downregulated in the OR-Taka cells (Fig. <a href=\"/articles/s41467-024-51831-7#Fig6\"> 6h </a> ), where genes involved in signatures of EMT and activation of SRC and YAP were enriched in the OR-Taka cells, indicating that osimertinib concomitantly reduced the expression of N-OSTs and enhanced SRC-YAP-regulated cellular plasticity to mediate metastasis and resistance. We also analyzed datasets from lung cancer cells treated with other EGFR TKIs (gefitinib, afatinib, erlotinib, and WZ4002) published by other groups (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 21 </a> ). Decreased N-glycosylation accompanied by increased EMT and SRC-YAP activation gene sets was also observed in the lung cancer cells treated with EGFR-TKIs, indicating that TKI treatment stress may repress the N-glycosylation process in lung cancer cells. </p> <p> The most common N-glycosylation-related <u> N </u> - <u> o </u> ligo <u> s </u> accharide <u> t </u> ransferase (N-OST) genes include the enzymatically active subunits <i> STT3A </i> and <i> STT3B </i> and the cooperative scaffold members <i> RPN1 </i> , <i> RPN2 </i> , <i> DAD1 </i> , and <i> DDOST </i> <sup> <a aria-label=\"Reference 46\" href=\"/articles/s41467-024-51831-7#ref-CR46\" id=\"ref-link-section-d6776021e3117\" title=\"Lopez-Sambrooks, C. et al. Oligosaccharyltransferase inhibition induces senescence in RTK-driven tumor cells. Nat. Chem. Biol. 12, 1023–1030 (2016).\"> 46 </a> </sup> . We compared their mRNA levels in our PC9-OR and HCC827-OR cells versus the parental cells (Fig. <a href=\"/articles/s41467-024-51831-7#Fig6\"> 6i </a> ). All of the N-OST genes showed significantly downregulated expression in the OR cells, explaining their generation of L-IL6. To determine whether the main component of OR-IL6 was deNG-IL6, we analyzed IL6 from PC9-IL6-WT, PC9-OR, HCC827-IL6-WT, and HCC827-OR cells by lectin blotting probed with wheat-germ-agglutinin (WGA), which recognizes N-acetylglutamate on the target protein (Fig. <a href=\"/articles/s41467-024-51831-7#Fig6\"> 6j </a> ). WT-IL6 from PC9 and HCC827 cells showed positive bands, suggesting that they harbor N-glycosylation. In comparison, the band density suggested that NG-IL6 was reduced by ~80% in both OR cell lines. Furthermore, we analyzed the N-glycans on WT-IL6 and OR-IL6 with a lectin microarray. Because PC9-OR cells secreted a much lower level of IL6 than HCC827-OR cells (Fig. <a href=\"/articles/s41467-024-51831-7#Fig6\"> 6b </a> ), we only analyzed the N-glycans on WT-IL6 and OR-IL6 from the HCC827 cell series (Fig. <a href=\"/articles/s41467-024-51831-7#Fig6\"> 6k </a> ). We further employed MS to examine the glycosylation status of WT-IL6 and OR-IL6. Purified WT-IL6 and OR-IL6 were subjected to digestion with the protease Glu-C or trypsin to generate IL6 peptides containing N73 or N172, respectively. N-glycans were subsequently removed by treatment with PNGase F. Digested IL6 samples were analyzed by LC-ESI-MS (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 22 </a> ). Both WT-IL6 and OR-IL6 were found to contain three glycosylation sites: N73, N172 and T170 (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 22 </a> and Fig. <a href=\"/articles/s41467-024-51831-7#Fig6\"> 6l </a> ). However, these three sites are not homogeneously glycosylated: 28% and 20% of WT-IL6 and OR-IL6 proteins were glycosylated at N73, respectively, i.e., OR-IL6 displayed a 29% reduction in N73 glycosylation compared to WT-IL6. In contrast, there were no significant differences in the percentages of proteins glycosylated at either N172 or T170. The glycosylation percentage was minimal at N172 (4% and 3% of WT-IL6 and OR-IL6, respectively), whereas glycosylation was prevalent at T170 (&gt;99% and 96% of WT-IL6 and OR-IL6, respectively). </p> <p> Based on the MS data, there are presumably four distinct glycoforms of WT-IL6. Twenty-eight percent of WT-IL6 molecules exhibit diglycosylation at N73 + T170, corresponding to H-IL6 and NG-IL6. Sixty-eight percent of WT-IL6 molecules show monoglycosylation at T170, represented by L-IL6 and deNG-IL6. The remaining 4% of WT-IL6 molecules may contain either two (at N172 + T170) or three (at N73 + N172 + T170) glycan chains. However, owing to the limited recognition sites of proteases, the precise percentages of IL6 with single or double N-glycosylation could not be determined simultaneously with the peptide products from a single digestion. The conclusion appears to be consistent with the results of the WB shown in Fig. <a href=\"/articles/s41467-024-51831-7#Fig6\"> 6g </a> . In the Western blots, two prominent bands corresponding to WT-IL6 were observed: a dense L-IL6 band indicative of deNG-IL6 and the H-IL6 band representing NG-IL6. Notably, no discernible band was detected at a higher molecular weight than the H-IL6 band (Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2b, c </a> ), suggesting that the quantity of heavily glycosylated IL6 (at N73 + N172 + T170) must be below the detection limit and was too low to be detected. When treated with PNGase F (Fig. <a href=\"/articles/s41467-024-51831-7#Fig6\"> 6m </a> ), the heavy forms of WT-IL6 were reduced in both the PC9 and HCC827 cell series, whereas little influence was observed in that of OR-IL6. These data provide evidence that OR-IL6 is poorly N-glycosylated. </p> <p> Cytokine array analysis demonstrated that IL6 is predominantly differentially expressed in the CM from OR cells compared to parental cells (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 19 </a> ). To assess the effect of OR-IL6 on STAT3 signaling, we treated PC9-P and HCC827-P cells with WT-IL6-CM (adjusted to 1 ng/mL) from PC9-IL6-WT and HCC827-IL6-WT cells, respectively, or OR-IL6-containing CM (adjusted to 1 ng/mL) from PC9-OR and HCC827-OR cells, respectively. WT-IL6-CM induced a stronger intensity of the STAT3-pY705 band at 24 h than that induced by OR-CM (Fig. <a href=\"/articles/s41467-024-51831-7#Fig7\"> 7a </a> ) in both cell lines. Time-dependent IF imaging revealed that treatment with either WT-IL6-CM or OR-CM stimulated the nuclear import of STAT3 (magenta) at 0.5 h (Fig. <a href=\"/articles/s41467-024-51831-7#Fig7\"> 7b </a> ). Notably, WT-IL6-CM induced significantly longer nuclear retention of STAT3 than OR-CM up to 24 h (Fig. <a href=\"/articles/s41467-024-51831-7#Fig7\"> 7b </a> , magnified inset), whereas OR-CM stimulated stronger cell surface recruitment of SRC (green) than WT-IL6-CM throughout the time course, indicating that OR-IL6, with lower N-glycosylation, tends to activate SRC rather than STAT3 signals. This signaling alteration was also observed in the OR cells compared to their corresponding parental cells (Fig. <a href=\"/articles/s41467-024-51831-7#Fig7\"> 7c </a> ). </p> <div class=\"c-article-section__figure js-c-reading-companion-figures-item\" id=\"figure-7\"> <figure> <figcaption> <b class=\"c-article-section__figure-caption\" id=\"Fig7\"> Fig. 7: Osimertinib-induced deNG-IL6 activates the SRC-YAP-SOX2 axis and migration in TKI-resistant NSCLC. </b> </figcaption> <div class=\"c-article-section__figure-content\"> <div class=\"c-article-section__figure-item\"> <a class=\"c-article-section__figure-link\" href=\"/articles/s41467-024-51831-7/figures/7\" rel=\"nofollow\"> <picture> <source srcset=\"//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-024-51831-7/MediaObjects/41467_2024_51831_Fig7_HTML.png?as=webp\" type=\"image/webp\"/> </picture> </a> </div> <div class=\"c-article-section__figure-description\" id=\"figure-7-desc\"> <p> <b> a </b> STAT3 and STAT3-pY705 levels induced by PC9-IL6-CM, PC9-OR-CM, HCC827-IL6-CM, or HCC827-OR-CM (IL6 = 1 ng/mL) over time in PC9-P or HCC827-P cells. <i> n </i> = 3 independent experiments, mean ± SEM, two-tailed unpaired <i> t </i> -test, PC9: *** <i> p </i> &lt; 0.0001, ** <i> p </i> = 0.0044, *** <i> p </i> &lt; 0.0001, *** <i> p </i> = 0.0001; HCC-827: * <i> p </i> = 0.0324, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001. The samples derived from the same experiment but different gels for STAT3 and STAT3-pY705 were processed in parallel. <b> b </b> Intracellular localization of STAT3 and SRC in PC9-P or HCC827-P cells treated with PC9-IL6-CM, PC9-OR-CM, HCC827-IL6-CM, or HCC827-OR-CM (IL6 = 1 ng/mL) over time, as determined by IF. The magnified insets show STAT3 or SRC localization at 24 h. Scale bars, 20 μm. <i> n </i> = 3 independent experiments, mean ± SEM, two-tailed unpaired <i> t </i> -test, PC9-STAT3: ** <i> p </i> = 0.0085, *** <i> p </i> = 0.002, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001; PC9-SRC: *** <i> p </i> = 0.0002, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001, *** <i> p </i> = 0.0002; HCC827-STAT3: n.s. <i> p </i> = 0.1262, ** <i> p </i> = 0.0018, ** <i> p </i> = 0.0025, *** <i> p </i> = 0.0001; HCC827-SRC: n.s. <i> p </i> = 0.0803, *** <i> p </i> = 0.0007, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001. <b> c </b> Intracellular localization of STAT3 and SRC in PC9-OR and HCC827-OR cells as determined based on IF images acquired with an FV3000 microscope (60x). Images are representative of <i> n </i> = 3 independent experiments. Scale bars, 20 μm. <b> d </b> Expression of proteins indicating the activation of SRC and YAP in the cytosolic and nuclear fractions of the parental and OR PC9 and HCC827 cells. <i> n </i> = 3 independent experiments, expression fold change (FC) relative to corresponding parental cells, mean ± SEM, two-tailed unpaired <i> t </i> -test, PC9-SRCpY416: *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001; PC9-SRC: *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001; PC9-YAPpY357: *** <i> p </i> &lt; 0.0001, ***p &lt; 0.0001; PC9-YAP: *** <i> p </i> &lt; 0.0001, *** <i> p </i> = 0.0004; HCC827-SRCpY416: *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001; HCC827-SRC: n.s. <i> p </i> = 0.8569, *** <i> p </i> &lt; 0.0001; HCC827-YAP-pY357: *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001; HCC827-YAP: *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001. The samples derived from the same experiment but different gels for SRCpY416, SRC, YAPpY357, YAP, β-ACTIN, and LAMIN A/C were processed in parallel. <b> e </b> Time-dependent survival of the PC9-OR and HCC827-OR cells under treatment with osimertinib (5 μM) in combination with VP (5 μM). <i> n </i> = 4 independent experiments, mean ± SEM, two-tailed unpaired <i> t </i> -test, PC9-OR: n.s. <i> p </i> = 0.7764, *** <i> p </i> = 0.0006, *** <i> p </i> = 0.0001; HCC827-OR: n.s. <i> p </i> = 0.9049, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001. <b> f </b> Analysis of the synergistic effect in PC9-OR and HCC827-OR cells treated with osimertinib in combination with DMSO or VP at various concentration combinations by Synergy Finder Plus under the two-tailed Zero interaction potency (ZIP) model. <b> g </b> Migration of PC9-IL6, PC9-OR, HCC827-IL6, and HCC827-OR cells through Transwell membranes. <i> n </i> = 4 independent experiments, mean ± SEM, two-tailed unpaired <i> t </i> -test, PC9: *** <i> p </i> = 0.0001, HCC827: *** <i> p </i> &lt; 0.0001. Source data are provided as a Source Data file. </p> </div> </div> <div class=\"u-text-right u-hide-print\"> <a aria-label=\"Full size image figure 7\" class=\"c-article__pill-button\" href=\"/articles/s41467-024-51831-7/figures/7\" rel=\"nofollow\"> <span> Full size image </span> </a> </div> </figure> </div> <p> We further examined the expression of proteins involved in SRC-YAP signaling in the OR cells and the corresponding parental cells (Fig. <a href=\"/articles/s41467-024-51831-7#Fig7\"> 7d </a> ). Compared to the PC9-P cells, the PC9-OR cells had an increased amount of SRCpY416 in the cytosol. In addition, in the PC9-OR cells, the levels of YAPpY357 were increased in both the cytosolic and nuclear fractions. Similarly, the levels of cytosolic SRCpY416 and YAPpY357 and nuclear YAP and YAPpY357 were elevated in the HCC827-OR cells compared to the HCC827-P cells. Next, to evaluate the requirement of YAP in osimertinib resistance, we treated the PC9-OR and HCC827-OR cells with fixed-dose osimertinib (5 μM) in combination with the YAP inhibitor (YAPi) verteporfin (VP) at 5 μM for different durations (Fig. <a href=\"/articles/s41467-024-51831-7#Fig7\"> 7e </a> ). VP significantly increased sensitivity to osimertinib in PC9-OR and HCC827-OR cells in a time-dependent manner (Fig. <a href=\"/articles/s41467-024-51831-7#Fig7\"> 7e </a> ). When the OR cells were simultaneously treated with osimertinib and the YAPi at various concentration combinations, a synergistic cytotoxic effect (as determined by Synergy Finder Plus: <a href=\"http://www.synergyfinderplus.org/\"> http://www.synergyfinderplus.org/ </a> ) <sup> <a aria-label=\"Reference 58\" href=\"/articles/s41467-024-51831-7#ref-CR58\" id=\"ref-link-section-d6776021e3418\" title=\"Zheng, S. et al. SynergyFinder plus: toward better interpretation and annotation of drug combination screening datasets. Genom. Proteom. Bioinform. 20, 587–596 (2022).\"> 58 </a> </sup> was observed (Fig. <a href=\"/articles/s41467-024-51831-7#Fig7\"> 7f </a> ), indicating that YAP is essential to the maintenance of osimertinib resistance. With the evidence that deNG-IL6-mediated SRC-YAP activation promotes cell migration (Figs. <a href=\"/articles/s41467-024-51831-7#Fig3\"> 3 </a> – <a href=\"/articles/s41467-024-51831-7#Fig5\"> 5 </a> ), we assessed the effects of OR-IL6 on cell migration (Fig. <a href=\"/articles/s41467-024-51831-7#Fig7\"> 7g </a> and Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 23 </a> ). The PC9-OR and HCC827-OR cells showed stronger migratory capacities than the PC9-IL6-WT and HCC827-IL6-WT cells, respectively. These data suggest that deNG-IL6 secreted by OR cells not only sustains cell survival (Fig. <a href=\"/articles/s41467-024-51831-7#Fig6\"> 6e, f </a> ) but also favors cell migration (Fig. <a href=\"/articles/s41467-024-51831-7#Fig7\"> 7g </a> and Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 23 </a> ). </p> <p> To demonstrate the presence of deNG-IL6 in NSCLC patients, we isolated MPECs from MPEs of patients who were naïve (before treatment) or resistant (after development of resistance) to EGFR TKIs and performed short-term culture (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 24 </a> ). Secreted IL6 in the CM was collected by IP and analyzed by WB. TKI-naïve MPECs secreted more H-IL6 than L-IL6, whereas the majority of IL6 from TKI-resistant MPECs was L-IL6 (Fig. <a href=\"/articles/s41467-024-51831-7#Fig8\"> 8a </a> ). Because the quantity of MPEC-derived IL6 was not enough for lectin array analysis, we analyzed them by WGA blotting. Only H-IL6 from TKI-naïve MPECs was detected by WGA blotting (Fig. <a href=\"/articles/s41467-024-51831-7#Fig8\"> 8b </a> ). Quantification results showed a reduction in the proportion of NG-IL6 and an increase in deNG-IL6 in the CM of EGFR TKI-resistant MPECs (Fig. <a href=\"/articles/s41467-024-51831-7#Fig8\"> 8c </a> ). </p> <div class=\"c-article-section__figure js-c-reading-companion-figures-item\" id=\"figure-8\"> <figure> <figcaption> <b class=\"c-article-section__figure-caption\" id=\"Fig8\"> Fig. 8: EGFR TKI-resistant NSCLC patients show high deNG-IL6 expression and a shift signaling preference from JAK-STAT3 to the SRC-YAP axis. </b> </figcaption> <div class=\"c-article-section__figure-content\"> <div class=\"c-article-section__figure-item\"> <a class=\"c-article-section__figure-link\" href=\"/articles/s41467-024-51831-7/figures/8\" rel=\"nofollow\"> <picture> <source srcset=\"//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-024-51831-7/MediaObjects/41467_2024_51831_Fig8_HTML.png?as=webp\" type=\"image/webp\"/> </picture> </a> </div> <div class=\"c-article-section__figure-description\" id=\"figure-8-desc\"> <p> <b> a </b> Pattern of secreted IL6 in MPECs derived from MPE of EGFR-TKI-naïve (TN01-TN04) and EGFR-TKI-resistant (TR01-TR05) patients as detected by WB ( <i> n </i> = 1). <b> b </b> N-glycosylated IL6 detected by lectin blotting with WGA (n = 1). <b> c </b> Percentage proportions of NG-IL6 and deNG-IL6 in the CM of EGFR-TKI-naïve and EGFR-TKI-resistant MPECs ( <i> n </i> = 1). <b> d </b> GSEA enrichment plot showing genes related to the N-glycosylation process derived from transcriptome data for patient tumors before and after the development of osimertinib resistance <sup> <a aria-label=\"Reference 58\" href=\"/articles/s41467-024-51831-7#ref-CR58\" id=\"ref-link-section-d6776021e3493\" title=\"Zheng, S. et al. SynergyFinder plus: toward better interpretation and annotation of drug combination screening datasets. Genom. Proteom. Bioinform. 20, 587–596 (2022).\"> 58 </a> </sup> named OSTu and ORTu, respectively. Sequential process by which 15 patients were selected according to the indicated criteria (left panel). The right panel shows the GSEA enrichment plots for N-glycosylation signature genes (right panel, top) and heatmaps for the enrichment scores (right panel, bottom) of selected patients (determined by Morpheus). Nominal <i> p </i> -value was one-tailed test on the appropriate side of the null distribution. <b> e </b> Expression and intracellular location of STAT3 and SRC in paired lung cancer tissues from six NSCLC patients (TKI_01-TKI_06) before (TKI-naïve) and after development of resistance (TKI resistant) to EGFR TKI as determined by TissueFAXS-imaged IF (see also Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 28 </a> for H&amp;E staining). The paired tissues from the same patient were attached to the same slide to perform IF staining. Images are selected magnified region from the full tissue. Scale bars, 20 μm. <b> f </b> Expression of nuclear STAT3 and membranous SRC as quantified by TissueQuest. Source data are provided as a Source Data file. </p> </div> </div> <div class=\"u-text-right u-hide-print\"> <a aria-label=\"Full size image figure 8\" class=\"c-article__pill-button\" href=\"/articles/s41467-024-51831-7/figures/8\" rel=\"nofollow\"> <span> Full size image </span> </a> </div> </figure> </div> <p> To elucidate the clinical correlation between the reduced expression of N-OSTs and osimertinib resistance, we analyzed online-available RNA sequencing (RNA-seq) data obtained from the lung tumors of fifteen patients who received osimertinib treatment <sup> <a aria-label=\"Reference 59\" href=\"/articles/s41467-024-51831-7#ref-CR59\" id=\"ref-link-section-d6776021e3520\" title=\"Roper, N. et al. Clonal evolution and heterogeneity of osimertinib acquired resistance mechanisms in EGFR mutant lung cancer. Cell Rep. Med. 1, 10007 (2020).\"> 59 </a> </sup> and developed resistance. Of the fifteen RNA-seq datasets, only ten provided paired data of tumors collected before and after the acquisition of resistance (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 25 </a> ), corresponding to <u> o </u> simertinib- <u> s </u> ensitive (OS) or OR tumors, namely, OSTu and ORTu, respectively. Notably, GSEA showed that the N-glycosylation process was enhanced in OSTu and decreased in ORTu (Fig. <a href=\"/articles/s41467-024-51831-7#Fig8\"> 8d </a> , upper right). </p> <p> Resistance mechanisms of the ORTu were characterized by <u> w </u> hole- <u> e </u> xome <u> s </u> equencing (WES) <sup> <a aria-label=\"Reference 59\" href=\"/articles/s41467-024-51831-7#ref-CR59\" id=\"ref-link-section-d6776021e3549\" title=\"Roper, N. et al. Clonal evolution and heterogeneity of osimertinib acquired resistance mechanisms in EGFR mutant lung cancer. Cell Rep. Med. 1, 10007 (2020).\"> 59 </a> </sup> . In most cases, resistance was acquired via known genomic alterations, such as <i> EGFR </i> C797S and the amplification of <i> KRAS </i> and <i> MET </i> (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 25 </a> ), whereas some developed through an undefined mechanism. We excluded ORTus that harbored known genomic mutations and obtained a group of three datasets from three patients: LAT010 (mixed NSCLC and SCLC transformation), LAT011 ( <u> n </u> euro <u> e </u> ndocrine differentiation, NE), and LAT017 ( <i> EGFR </i> and <i> YES1 </i> amplification). LAT010 and LAT011 were categorized as EMT-related cases based on transformation to the aggressive NE subtype <sup> <a aria-label=\"Reference 13\" href=\"/articles/s41467-024-51831-7#ref-CR13\" id=\"ref-link-section-d6776021e3579\" title=\"Quintanal-Villalonga, A. et al. Lineage plasticity in cancer: a shared pathway of therapeutic resistance. Nat. Rev. Clin. Oncol. 17, 360–371 (2020).\"> 13 </a> , <a aria-label=\"Reference 18\" href=\"/articles/s41467-024-51831-7#ref-CR18\" id=\"ref-link-section-d6776021e3582\" title=\"Sequist, L. V. et al. Genotypic and histological evolution of lung cancers acquiring resistance to EGFR inhibitors. Sci. Transl. Med. 3, 75ra26 (2011).\"> 18 </a> , <a aria-label=\"Reference 20\" href=\"/articles/s41467-024-51831-7#ref-CR20\" id=\"ref-link-section-d6776021e3585\" title=\"Williams, E. D., Gao, D., Redfern, A. &amp; Thompson, E. W. Controversies around epithelial-mesenchymal plasticity in cancer metastasis. Nat. Rev. Cancer 19, 716–732 (2019).\"> 20 </a> </sup> . </p> <p> We then analyzed the enrichment in six gene sets of interest related to the N-glycosylation process, EMT, and SRC-YAP-related signaling by single-sample GSEA (ssGSEA) <sup> <a aria-label=\"Reference 60\" href=\"/articles/s41467-024-51831-7#ref-CR60\" id=\"ref-link-section-d6776021e3592\" title=\"Barbie, D. A. et al. Systematic RNA interference reveals that oncogenic KRAS-driven cancers require TBK1. Nature 462, 108–112 (2009).\"> 60 </a> </sup> . The ssGSEA scores for these processes for each patient were compiled into a heatmap. Compared to the OSTu, the ORTu in the three patients exhibited a decreased N-glycosylation process and enhanced EMT traits and SRC-YAP signaling (Fig. <a href=\"/articles/s41467-024-51831-7#Fig8\"> 8d </a> , lower right). This difference was more pronounced after subtracting the ssGSEA scores from the OSTu. Among the three patients, the most significant difference between OSTu and ORTu was observed in patient LAT017, who showed <i> YES1 </i> amplification (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 25 </a> ). </p> <p> YES1 is an SRC family kinase that phosphorylates YAP at Y357, promoting its nuclear translocation and transcriptional activity and thus facilitating the progression of cancer <sup> <a aria-label=\"Reference 61\" href=\"/articles/s41467-024-51831-7#ref-CR61\" id=\"ref-link-section-d6776021e3608\" title=\"Heinz, M. C. et al. Liver colonization by colorectal cancer metastases requires YAP-controlled plasticity at the micrometastatic stage. Cancer Res. 82, 1953–1968 (2022).\"> 61 </a> , <a aria-label=\"Reference 62\" href=\"/articles/s41467-024-51831-7#ref-CR62\" id=\"ref-link-section-d6776021e3611\" title=\"Hamanaka, N. et al. YES1 is a targetable oncogene in cancers harboring YES1 gene amplification. Cancer Res. 79, 5734–5745 (2019).\"> 62 </a> </sup> . Based on this information and because we demonstrated that deNG-IL6 triggers EMT and metastasis via alternative activation of the SRC-YAP-SOX2 axis in previous experiments, we further analyzed the RNA-seq data from patient LAT017 (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 26a </a> ). Consistent with our in vitro and in vivo data, the expression of N-glycosylation-related genes was reduced after acquisition of osimertinib resistance; moreover, signatures of EMT and activation of the SRC and YAP pathways were simultaneously enriched (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 26a </a> ). We also analyzed the expression of transcription factors (TFs) within the 1 K bases upstream of six N-OST genes in patient LAT017 (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 26b, c </a> ) and revealed concordant downregulation of TF expression in the ORTu (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 26d </a> ). </p> <p> We further used paired lung cancer tissues obtained at EGFR-TKI-naïve and EGFR-TKI-resistant statuses from six patients (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 27 </a> , Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 28 </a> , and Fig. <a href=\"/articles/s41467-024-51831-7#Fig8\"> 8e </a> ). In the EGFR-TKI-naïve tissues, we observed strong nuclear STAT3 (Fig. <a href=\"/articles/s41467-024-51831-7#Fig8\"> 8e </a> , upper panel) but weak membranous SRC expression. In contrast, in the EGFR-TKI-resistant group, the expression of membranous SRC was prominent, and nuclear STAT3 staining was barely observed. (Fig. <a href=\"/articles/s41467-024-51831-7#Fig8\"> 8e </a> , lower panel). The difference was highlighted by TissueQuest quantification (Fig. <a href=\"/articles/s41467-024-51831-7#Fig8\"> 8f </a> ). </p> <p> In summary, we demonstrated that in NSCLC, IL6 undergoes N-glycosylation primarily at the N73 residue and O-glycosylation at the T170 residue. The loss of glycan on N73 leads to a switch of downstream signal preference from the JAK-STAT3 axis to the SRC-YAP-SOX2 axis, thereby triggering cell plasticity. Additionally, we have uncovered a bypass mechanism that drives osimertinib resistance by reducing the N-glycosylation of IL6, consequently shifting the survival-dependent pathway from the JAK-STAT3 axis to the SRC-YAP-SOX2 axis (Fig. <a href=\"/articles/s41467-024-51831-7#Fig9\"> 9 </a> ). </p> <div class=\"c-article-section__figure js-c-reading-companion-figures-item\" id=\"figure-9\"> <figure> <figcaption> <b class=\"c-article-section__figure-caption\" id=\"Fig9\"> Fig. 9: Long-Term EGFR TKI Treatment Induces deNG-IL6 to Drive Resistance. </b> </figcaption> <div class=\"c-article-section__figure-content\"> <div class=\"c-article-section__figure-item\"> <a class=\"c-article-section__figure-link\" href=\"/articles/s41467-024-51831-7/figures/9\" rel=\"nofollow\"> <picture> <source srcset=\"//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-024-51831-7/MediaObjects/41467_2024_51831_Fig9_HTML.png?as=webp\" type=\"image/webp\"/> </picture> </a> </div> <div class=\"c-article-section__figure-description\" id=\"figure-9-desc\"> <p> In non-small cell lung cancer (NSCLC) cells, autocrine IL6 (NG-IL6) predominantly exhibits N-glycosylation at residue N73 and O-glycosylation at residue T170 in an EGFR TKI-naïve environment. NG-IL6 predominantly activates the canonical JAK-STAT3 axis, promoting cell proliferation. However, long-term treatment with EGFR TKIs leads to increased secretion of IL6 while concurrently suppressing the expression of N-glycosyltransferase complex components, primarily STT3A or STT3B, RPN1/2, DAD1, and DDOST. This suppression results in diminished N-glycosylation at N73 of NG-IL6, which subsequently reduces N-glycosylation at N73 of NG-IL6. Consequently, the proportion of NG-IL6 decreases, while the proportion of deNG-IL6 increases upon the acquisition of resistance. deNG-IL6, in turn, shifts signaling preference from the JAK-STAT3 axis to the SRC-YAP-SOX2 axis. The alternatively activated SRC-YAP-SOX2 signaling facilitates cellular plasticity and contributes to the development of drug resistance. </p> </div> </div> <div class=\"u-text-right u-hide-print\"> <a aria-label=\"Full size image figure 9\" class=\"c-article__pill-button\" href=\"/articles/s41467-024-51831-7/figures/9\" rel=\"nofollow\"> <span> Full size image </span> </a> </div> </figure> </div> </div> </div> </section> <section> <div class=\"c-article-section\" id=\"Sec10-section\"> <h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"Sec10\"> Discussion </h2> <div class=\"c-article-section__content\" id=\"Sec10-content\"> <p> To date, how IL6 differentially regulates its multiple downstream signals and whether the variation of induced downstream signals has clinical implications remain unclear. Here, we showed that IL6 from both lung cancer cell lines and clinical specimens is modified by N-glycosylation. deNG-IL6 induces a switch in GP130 downstream signals from canonical JAK-STAT3 to the SRC-YAP-SOX2 axis. The changes resulted in different gene expression profiles and enhanced in vitro migration and in vivo lung metastasis. Moreover, we discovered that long-term treatment with the EGFR-TKI osimertinib induces a reduction in N-OST expression and an elevation of deNG-IL6, which results in EGFR-TKI resistance in lung cancer cells. </p> <p> In this study, two potential N-glycosites (N73 and N172) and four possible O-glycosites (T166, T171, T170, and S174) were predicted to exist in hIL6. MS results indicated that the glycan chains reside at N73, N172, and T170. Utilizing site-directed mutagenesis (Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2b </a> ) and NGI-1 treatment (Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2c </a> ), we identified four distinct IL6 bands via Western blot analysis, corresponding to four different glycosylation forms as follows (from top to bottom): (1) NG-IL6 carrying an N-glycan (N73 or N172) and an O-glycan (T170), (2) the band marked with an asterisk (*) likely representing N-glycosylation (N73 or N172) alone, (3) deNG-IL6 containing an O-glycan (T170) exclusively, and (4) deG-IL6 lacking any glycan. Notably, the NG-IL6 band was nearly absent in the analysis of N73Q-IL6, suggesting that the predominant attachment sites for IL6 glycan chains are N73 + T170 rather than N172 + T170. Moreover, no discernible bands higher than NG-IL6 were observed, indicating that tri-glycosylated IL6 (N73 + N172 + T170) may be either non-existent or negligible. This observation is further supported by MS data obtained from HCC827 WT-IL6 (Fig. <a href=\"/articles/s41467-024-51831-7#Fig6\"> 6l </a> and Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 22 </a> ), which revealed that the N-glycans of NG-IL6 originate mainly from N73 (&gt;85%) rather than N172 (&lt;15%). </p> <p> N73 emerges as the predominant N-glycosylation site, a conclusion reinforced by the MS-based glycan analysis of WT-IL6 (Fig. <a href=\"/articles/s41467-024-51831-7#Fig6\"> 6l </a> ). However, lectin microarray analysis revealed only an ~50% reduction in the signal intensity of N-glycan in N73Q-IL6 (Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2d </a> ), presenting a somewhat contradictory outcome. This discrepancy is likely due to the utilization of a polyclonal anti-IL6 antibody for the IP-based isolation process. Consequently, the resultant mixture possibly encompassed not only the desired N73Q-IL6 but also other components, such as antibody fragments and/or nonspecific proteins co-precipitated from the conditioned medium. </p> <p> Additionally, AS2-IL6-N172Q cells exhibited no significant difference in migration capacity compared to AS2-IL6-WT cells (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 14 </a> ), suggesting that the impact of N172 glycosylation on cell migration is minimal. Our findings substantiate the notion that N73 is the principle N-glycosite in IL6, with the abundance of attached N-glycans reduced by approximately one-third when the cells acquired resistance to osimertinib. </p> <p> Intriguingly, N172Q-IL6 exhibited four distinct bands on the Western blot: the NG-IL6, deNG-IL6, and deG-IL6 bands and a faint asterisk-labeled band between the NG-IL6 and deNG-IL6 bands (Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2b </a> ). Notably, the distance between the asterisk-labeled band and the deG-IL6 band mirrors that between the NG-IL6 and deNG-IL6 bands (Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2b </a> ). Furthermore, treatment with NGI-1 resulted in a reduction in the intensity of the faint asterisk-labeled band (Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2c </a> ), suggesting that this faint band possibly represents IL6 N-glycosylation solely at N73. Regarding N172Q-IL6, the appearance of the asterisk-labeled band and the increase in deG-IL6 band intensity imply that the change in the amino acid at residue 172 may interfere with glycosylation at T170. </p> <p> In the CM of established lung cancer cell lines, it appears that deNG-IL6 (L-IL6) is the predominant species. However, in the CM of cancer cells from patient MPECs, most cases showed equal expression of the two species (Fig. <a href=\"/articles/s41467-024-51831-7#Fig1\"> 1b </a> ). In our studies, blockade of the IL6 receptors GP130 (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 7 </a> ) and GP80 (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 8 </a> ) reduced the STAT3 activity induced by both WT-IL6 and N73Q-IL6 in various cell lines. Notably, the inhibition was more effective in the N73Q-IL6-treated cells than in the WT-IL6-treated cells, suggesting that WT-IL6 may have a stronger binding affinity to its receptors and be more resistant to antibody blockade. We assume that NG-IL6, with stronger receptor binding ability, even with only 30% occupancy in CM, may outperform deNG-IL6 in conducting signal transduction. However, when the occupancy of NG-IL6 in CM is reduced, deNG-IL6 becomes dominant in signaling. </p> <p> Targeting N-glycosylation by NGI-1 combined with EGFR-TKIs has been shown to overcome EGFR-TKI resistance in NSCLC <sup> <a aria-label=\"Reference 63\" href=\"/articles/s41467-024-51831-7#ref-CR63\" id=\"ref-link-section-d6776021e3741\" title=\"Lopez Sambrooks, C. et al. Oligosaccharyltransferase inhibition overcomes therapeutic resistance to EGFR tyrosine kinase inhibitors. Cancer Res. 78, 5094–5106 (2018).\"> 63 </a> </sup> . The study concluded that the tumor growth delay was due to N-glycosylation-defective EGFR reducing the activation of receptors and dissociating EGFR signaling from other coexpressed receptors. In hepatocellular carcinoma, IL6-induced JAK-1 phosphorylates PD-L1, enhancing the recruitment of STT3A and thus increasing PD-L1 N-glycosylation, which favors the stability of PD-L1 and promotes immune evasion <sup> <a aria-label=\"Reference 64\" href=\"/articles/s41467-024-51831-7#ref-CR64\" id=\"ref-link-section-d6776021e3745\" title=\"Chan, L. C. et al. IL-6/JAK1 pathway drives PD-L1 Y112 phosphorylation to promote cancer immune evasion. J. Clin. Invest. 129, 3324–3338 (2019).\"> 64 </a> </sup> . These studies indicated that elevated N-glycosylation and glycosyltransferase expression resulted in unfavorable disease progression and poor survival in cancer patients. However, some reports have shown that reduced glycosylation could promote the progression of malignancies. For example, downregulation of the levels of the dystroglycan glycosyltransferases LARGE2 and ISPD was found to be associated with poor survival in patients with clear cell renal cell carcinoma <sup> <a aria-label=\"Reference 65\" href=\"/articles/s41467-024-51831-7#ref-CR65\" id=\"ref-link-section-d6776021e3749\" title=\"Miller, M. R. et al. Downregulation of dystroglycan glycosyltransferases LARGE2 and ISPD associate with increased mortality in clear cell renal cell carcinoma. Mol. Cancer 14, 141 (2015).\"> 65 </a> </sup> , and loss of the polypeptide N-acetylgalactosaminyltransferase-3 (GALNT3) in pancreatic adenocarcinoma was associated with increased aggressiveness of cancer cells <sup> <a aria-label=\"Reference 66\" href=\"/articles/s41467-024-51831-7#ref-CR66\" id=\"ref-link-section-d6776021e3753\" title=\"Chugh, S., Meza, J., Sheinin, Y. M., Ponnusamy, M. P. &amp; Batra, S. K. Loss of N-acetylgalactosaminyltransferase 3 in poorly differentiated pancreatic cancer: augmented aggressiveness and aberrant ErbB family glycosylation. Br. J. Cancer 114, 1376–1386 (2016).\"> 66 </a> </sup> . Accordingly, we found that reduced N-glycosylation on IL6 led to a high metastatic potential and resistance to EGFR-TKIs in NSCLC cells. </p> <p> Peng and colleagues showed that high environmental glucose acts as a stressor that induces the O-GlcNAcylation of intracellular YAP-1 and promotes tumorigenicity <sup> <a aria-label=\"Reference 67\" href=\"/articles/s41467-024-51831-7#ref-CR67\" id=\"ref-link-section-d6776021e3760\" title=\"Peng, C. et al. Regulation of the Hippo-YAP pathway by glucose sensor O-GlcNAcylation. Mol. Cell 68, 591–604.e595 (2017).\"> 67 </a> </sup> , suggesting that stress from the tumor microenvironment can shape the glycosylation pattern of key proteins in cancer cells. Our in silico data indicated that a group of specific TFs may act as TKI stress sensors and repress the expression of N-OSTs. Reduced N-OST expression may result in globally defective N-linked glycosylation of the cellular proteome and subsequently change cell behaviors. Interestingly, we found that defective N-linked glycosylation of a key regulator, such as IL6, is sufficient to induce prominent biological changes such as EMT, migration, metastasis, and drug resistance in NSCLC cells. </p> <p> Collectively, our data revealed that IL6 is modified with N-glycosylation mainly on N73 residue and O-glycosylation on T170 residue in NSCLC. The loss of glycan on N73 induces a switch of downstream signal preference from the JAK-STAT3 axis to the SRC-YAP-SOX2 axis, triggers subsequent cell plasticity. We also demonstrated that an increase in deNG-IL6 production can be induced by environmental stressors, such as EGFR-TKI treatment. This change in the glycosylation status of IL6 can switch the downstream signaling preference of the GP130 receptor from JAK-STAT3 to the SRC-YAP-SOX2 axis. Lung cancer cells with high levels of deNG-IL6 tend to possess EMT-related properties, metastasize to other organs, and exhibit EGFR-TKI resistance. These findings raise the possibility of controlling cancer cell behavior by simply changing the glycosylation pattern on a key regulator and predicting the appearance of EGFR-TKI resistance by monitoring circulating IL6 glycoforms in NSCLC patients. </p> </div> </div> </section> <section> <div class=\"c-article-section\" id=\"Sec11-section\"> <h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"Sec11\"> Methods </h2> <div class=\"c-article-section__content\" id=\"Sec11-content\"> <h3 class=\"c-article__sub-heading\" id=\"Sec12\"> Human <u> m </u> alignant <u> p </u> leural <u> e </u> ffusion (MPE) </h3> <p> The MPE samples from patients who underwent thoracentesis or thoracotomy at National Cheng Kung University Hospital were collected as previously described <sup> <a aria-label=\"Reference 68\" href=\"/articles/s41467-024-51831-7#ref-CR68\" id=\"ref-link-section-d6776021e3789\" title=\"Hung, T. L. et al. Clinical evaluation of HER-2/neu protein in malignant pleural effusion-associated lung adenocarcinoma and as a tumor marker in pleural effusion diagnosis. Clin. Cancer Res. 9, 2605–2612 (2003).\"> 68 </a> </sup> . The TKI-naïve MPEs were collected from patients with histologically proven NSCLC and symptomatic malignant pleural effusion at the time of initial diagnosis, <i> n </i> = 19, female = 13, male = 6, mean age = 64.8 (49, 88). TKI-resistant MPEs were collected from patients who received EGFR TKI treatment and developed resistance, <i> n </i> = 5, female = 4, male = 1, mean age = 67.4 (54, 92). Paired TKI-naïve and TKI-resistant NSCLC lung tissues were obtained from the Human Biobank of National Cheng Kung University Hospital, <i> n </i> = 6, female = 4, male = 2, TKI-naïve mean age = 59.67 (45, 71); TKI-resistant mean age = 61.67 (48, 73). Source data are provided as a Source Data file. No sex or gender analysis was carried out. All subjects gave informed consent to publish those data before contributing samples to the Human Biobank of National Cheng Kung University Hospital. </p> <h3 class=\"c-article__sub-heading\" id=\"Sec13\"> Cell lines </h3> <p> The following human lung cancer cell lines were purchased from the American Type Culture Collection: adenocarcinoma (A549, RRID:CVCL_0023; H1299, RRID:CVCL_0060; HCC827, RRID:CVCL_2063; H1650, RRID:CVCL_1483; and H1975, RRID:CVCL_1511), large cell carcinoma (H460, RRID:CVCL_0459), squamous cell carcinoma (H226, RRID:CVCL_1544), and normal human bronchial epithelial cell lines (NL-20, RRID:CVCL_3756; Beas-2B, RRID:CVCL_0168). The human lung adenocarcinoma cell line PC14PE6 was provided by Dr. Fidler (MD Anderson Cancer Center, Houston, TX, USA). PC14PE6/AS2 (AS2) <sup> <a aria-label=\"Reference 39\" href=\"/articles/s41467-024-51831-7#ref-CR39\" id=\"ref-link-section-d6776021e3810\" title=\"Yeh, H. H., Lai, W. W., Chen, H. H., Liu, H. S. &amp; Su, W. C. Autocrine IL-6-induced Stat3 activation contributes to the pathogenesis of lung adenocarcinoma and malignant pleural effusion. Oncogene 25, 4300–4309 (2006).\"> 39 </a> </sup> was established from ascites in SCID mice generated by inoculation of PC14PE6 cells into the peritoneal cavity. Human lung adenocarcinoma cell lines PC9 (RRID:CVCL_B260) and CL1-0 (RRID:CVCL_3871) <sup> <a aria-label=\"Reference 69\" href=\"/articles/s41467-024-51831-7#ref-CR69\" id=\"ref-link-section-d6776021e3814\" title=\"Chu, Y. W. et al. Selection of invasive and metastatic subpopulations from a human lung adenocarcinoma cell line. Am. J. Respir. Cell Mol. Biol. 17, 353–360 (1997).\"> 69 </a> </sup> were kindly provided by Dr. Chih-Hsin Yang (National Taiwan University, Taipei, Taiwan) and Dr. Pan-Chyr Yang (National Taiwan University), respectively. All cell lines were authenticated by short tandem repeat-polymerase chain reaction (STR-PCR) at the Biosource Collection and Research Center (BCRC) and the Center for Genomic Medicine at National Cheng-Kung University, Taiwan. </p> <h3 class=\"c-article__sub-heading\" id=\"Sec14\"> Mouse models </h3> <p> Six- to eight-week-old male and female nude mice (BALB/cAnN.Cg-Foxn1nu/CrlNarl: <a href=\"https://www.nlac.narl.org.tw/p3_animal2_detail.asp?cid=1&amp;nid=9&amp;ppage=0&amp;type=2\"> https://www.nlac.narl.org.tw/p3_animal2_detail.asp?cid=1&amp;nid=9&amp;ppage=0&amp;type=2 </a> ) were purchased from the National Laboratory Animal Center (Taipei, Taiwan). </p> <h3 class=\"c-article__sub-heading\" id=\"Sec15\"> Cell culture </h3> <p> PC9, H1650, H1299, HCC827, H1975, H226, H460 and CL1-0 cells were maintained in RPMI 1640 medium (Gibco) supplemented with 10% <u> f </u> etal <u> b </u> ovine <u> s </u> erum (FBS; Gibco) and 1% antibiotic-antimycotic (anti-anti, Gibco), and A549 cells were maintained in F12K medium (Gibco) supplemented with 10% FBS and 1% anti-anti. Beas-2B cells were maintained in BEBM supplemented with the components of a BEGM Kit (Lonza). The AS2 human lung adenocarcinoma cell line was maintained in MEM-α medium (Gibco) supplemented with 10% FBS, 1% anti-anti, 1% sodium pyruvate (Gibco), and 1× MEM vitamin solution (Gibco). Cells stably expressing empty vector, wild-type <i> IL6 </i> ( <i> IL6-WT </i> ), or N-glycosylation mutant <i> IL6 </i> ( <i> IL6-N73Q </i> , <i> IL6-N73A </i> , <i> IL6-N73V </i> , <i> IL6-N172Q </i> , and <i> IL6-N73Q </i> + <i> N172Q </i> ) were established by transfection with TurboFect Transfection Reagent (Thermo Fisher), followed by selection of individual clones with 500 μg/mL geneticin (G418) (Calbiochem). AS2-IL6-WT-pLuc and AS2-IL6-N73Q-pLuc cells were generated by transfection of AS2-IL6-WT and AS2-IL6-N73Q cells with the pGL4.50 (Luc2/CMV/Hygro) plasmid (Promega) through TurboFect (Thermo Fisher) and selection with 500 μg/mL hygromycin B (H3274, Sigma-Aldrich), respectively. Gene silencing was performed by transducing target cells with VSV-G particles harboring pLKO.1 empty vector or shRNA against human <i> SRC </i> , <i> YAP </i> , <i> SOX2 </i> , or <i> IL6 </i> mRNA (purchased from RNA Technology Platform and Gene Manipulation Core, Academia Sinica, Taipei, Taiwan) and selected using 2 μg/mL puromycin (Sigma-Aldrich). The <u> o </u> simertinib- <u> r </u> esistant (OR) cells were established by constant culture in medium containing 5 μM osimertinib. All cell lines were maintained in an incubator at 37 °C supplemented with 5% CO <sub> 2 </sub> . </p> <h3 class=\"c-article__sub-heading\" id=\"Sec16\"> Preparation of <u> c </u> onditioned <u> m </u> edium (CM) </h3> <p> A total of 2 × 10 <sup> 6 </sup> cells were seeded in a 10 cm dish with 11 mL of culture medium with or without specific treatment. The <u> c </u> onditioned <u> m </u> edium (CM) was collected at designated times and then centrifuged at 750 × <i> g </i> . The supernatant was collected and stored at −80 °C until further application. Intact or adjusted CM was applied according to the requirements of different experiments. For collection of the OR-CM, the OR cells were seeded in osimertinib-free culture medium. <i> For cell treatment and immunoprecipitation </i> , the IL6 concentration of the CM was measured by ELISAs and adjusted to 1 ng/mL with supplemented culture medium prior to use. <i> For the cytokine array </i> , intact or diluted CM was applied. For diluted CM, the IL6 concentration was adjusted to 1 ng/mL with supplemented culture medium prior to assay. </p> <h3 class=\"c-article__sub-heading\" id=\"Sec17\"> IL6 ELISA </h3> <p> One milliliter of the CM supernatant was used for measurement of IL6 concentration. An IL6 ELISA was performed using a human IL6 CytoSet ELISA kit (Life Technology, CHC1263) following the manufacturer’s protocol. </p> <h3 class=\"c-article__sub-heading\" id=\"Sec18\"> Prediction of glycosylation sites </h3> <p> The full-length IL6 protein sequence obtained from NCBI Protein (GenBank: AAC41704.1) was analyzed with the NetNGlyc 1.0 server (RRID:SCR_001570, <a href=\"https://services.healthtech.dtu.dk/service.php?NetNGlyc-1.0\"> https://services.healthtech.dtu.dk/service.php?NetNGlyc-1.0 </a> ) to identify potential N-linked glycosylation sites and with the NetOGlyc 4.0 server (RRID:SCR_009026, <a href=\"https://services.healthtech.dtu.dk/service.php?NetOGlyc-4.0\"> https://services.healthtech.dtu.dk/service.php?NetOGlyc-4.0 </a> ) to identify potential O-linked glycosylation sites <sup> <a aria-label=\"Reference 47\" href=\"/articles/s41467-024-51831-7#ref-CR47\" id=\"ref-link-section-d6776021e3964\" title=\"Blom, N., Sicheritz-Ponten, T., Gupta, R., Gammeltoft, S. &amp; Brunak, S. Prediction of post-translational glycosylation and phosphorylation of proteins from the amino acid sequence. Proteomics 4, 1633–1649 (2004).\"> 47 </a> </sup> . A default threshold score was applied, and the result was confirmed by nine of nine calculation networks showing jury agreement by their internal calculation assessment. Residues with higher prediction scores showed higher potential as N-glycosites or O-glycosites. </p> <h3 class=\"c-article__sub-heading\" id=\"Sec19\"> Protein extraction </h3> <h4 class=\"c-article__sub-heading c-article__sub-heading--small\" id=\"Sec20\"> Total cellular protein lysates </h4> <p> Cells were washed twice with ice-cold 1x phosphate-buffered saline (PBS) and lysed with 200 μL of radioimmunoprecipitation assay (RIPA) lysis buffer [50 mM Tris (pH 7.5), 1% (v/v) NP-40, 0.5% (v/v) sodium deoxycholate, 150 mM NaCl, 0.1% (v/v) sodium dodecyl sulfate (SDS), 10 mM Na <sub> 3 </sub> VO <sub> 4 </sub> , and protease inhibitor cocktail from Roche Applied Sciences]. Whole-cell lysates were cleared by centrifugation at 20,000 × <i> g </i> and 4 °C for 10 min, and protein concentrations were determined by the Bradford assay (Bio–Rad) according to the manufacturer’s protocol. </p> <h4 class=\"c-article__sub-heading c-article__sub-heading--small\" id=\"Sec21\"> Nuclear and cytosolic fractionation </h4> <p> Cells were collected by trypsinization and washed with 1x PBS three times. The cells were permeabilized with low salt buffer (10 mM HEPES, pH = 7.4, 10 mM KCl, 50 μg/mL digitonin) and then centrifuged at 750 × <i> g </i> and 4 °C for 10 min. The supernatant was the cytosolic fraction of cellular proteins. The pellets were then extracted with RIPA buffer followed by centrifugation at 20,000 × <i> g </i> and 4 °C for 10 min. The resulting supernatants were the nuclear fraction of cellular proteins. </p> <h3 class=\"c-article__sub-heading\" id=\"Sec22\"> Western blotting (WB) </h3> <p> For detection of changes in protein levels, whole-cell lysates containing 15 μg of protein were separated by 8–12% sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE). The transferred membranes were blocked with 5% skim milk and hybridized with primary Ab at 4 °C overnight. After hybridization with secondary Ab, the protein bands were visualized by enhanced chemiluminescence (ECL) and imaged by exposure to X-ray film (Fujifilm) or in an iBright CL1500 Imaging System (Thermo Fisher). The images were analyzed using Gel-Pro Analyzer 4.0 (Media Cybernetics, Inc.) and Prism (GraphPad, Inc., RRID:SCR_005375) software. All immunoblot analyses were performed in triplicate ( <i> n </i> = 3) except for those performed with patient specimens ( <i> n </i> = 1). The following Ab were used for WB: IL6 (Santa Cruz, sc-32296; RRID:AB_627804); STAT3 (BD, 610189; RRID:AB_397588); STAT3-pY705 (CST, 9131;RRID:AB_331757); SRC (CST, 2108; RRID_331137); SRC-pY416 (CST, 2101; RRID:AB_331697); YAP (CST, 4912; RRID:AB_2218911); YAP-pY357 (Abcam, ab62751; RRID:AB_956486); β-ACTIN (Millipore, MAB1501; RRID:AB_2223041); SLUG (Abcam, ab-27568; RRID:AB_777968); CLAUDIN1 (Life Technology, 374900; RRID:AB_2533323); E-CADHERIN (BD, 610181; RRID:AB_397581); N-CADHERIN (BD, 610921; RRID:AB_398236); α-SMA (Abcam, ab-5694; RRID:AB_2223021); AKT (CST, 4691; RRID:AB_915783); AKT-pY308 (CST, 4056; RRID:AB_331163); AKT-pY473 (CST, 9271; RRID:AB_329825); SOX2 (Santa Cruz, sc-365823; RRID:AB_10842165); ERK1/2 (Santa Cruz, sc-514302; RRID:AB_2571739); pERK (Santa Cruz, sc-7383; RRID:AB_627545); LAMIN A/C (Merck Millipore, 05-714; RRID:AB_309928); NANOG (GeneTex, GTX-627421; RRID:AB_11169335); OCT3/4 (Santa Cruz, sc-5279; RRID:AB_628051); LATS1 (CST, 3477; RRID:AB_2133513); pLATS (CST, 8654; RRID:AB_10971635); TAZ (Abcam, ab224239; RRID:AB_2889852); pTAZ (CST, 59971; RRID:AB_2799578); MST1 (CST, 3682; RRID:AB_2144632). </p> <h3 class=\"c-article__sub-heading\" id=\"Sec23\"> Immunoprecipitation (IP) </h3> <p> <i> For WB </i> , ten milliliters of adjusted CM supernatant was used for each IP reaction. The concentration of IL6 in the CM supernatant was measured by ELISAs and adjusted to 1 ng/mL with culture media containing supplements. In addition, 10 ng of <u> r </u> ecombinant <u> h </u> uman IL6 protein (rhIL6, Sino Biological, 10395-HNAE) was added to 10 mL culture media containing supplements as a positive control sample. The adjusted CM was then precleaned with nProtein A Sepharose beads (GE Healthcare, GE17-5280-01) to remove nonspecific binding. IL6 antibody (2 μg, GeneTex, GTX110527; RRID:AB_10721789) was used in each IP reaction, rotated at low speed overnight at 4 °C. The Ag-Ab complex was pulled down with 50 μL of nProtein A at 4 °C for 2 h and released in 26 μL of RIPA lysis buffer plus 6x sample buffer by boiling. The IL6 products were then subjected to WB for pattern analysis. <i> For the lectin microarray </i> , CM supernatant from a hollow fiber bioreactor in which the IL6 concentration was adjusted to 1 μg/mL was applied. A total amount of 100 μg IL6 was pulled down following the above procedures. </p> <h3 class=\"c-article__sub-heading\" id=\"Sec24\"> Cell treatment conditions </h3> <p> <i> For NGI-1 </i> (Sigma-Aldrich, SML1620) <i> treatment </i> , 2 × 10 <sup> 6 </sup> cells were seeded in a 10 cm dish and allowed to attach overnight. The medium was replaced with fresh culture medium containing NGI-1 at 10 μM. After 48 h of treatment, the CM was collected for analysis of the IL6 patterns. <i> For treatment of various CM </i> , cells were seeded and allowed to attach overnight in serum-free medium for 12 h (8 × 10 <sup> 5 </sup> cells/6-cm dish for WB, 1 × 10 <sup> 5 </sup> cells/chamber slide for IF imaging). The serum-free medium was subsequently replaced with CM containing different IL6 glycoforms adjusted to 1 ng/mL and treated for the designated duration. <i> For treatments with neutralizing antibodies </i> αGP130 (R&amp;D, MAB228; RRID:AB_2233737) and αGP80 (R&amp;D, MAB227; RRID:AB_2127908), AS2-Vec cells were pretreated with these antibodies for 2 h followed by treatment with WT-IL6 or N73Q-IL6 CM. For αIL6 (R&amp;D, MAB206; RRID:AB_2127617)-neutralizing antibody treatment, WT-IL6 or N73Q-IL6 CM was incubated with the antibody at 37 °C for 2 h and then added to AS2-Vec cells. <i> For cytotoxicity assays </i> , 4 × 10 <sup> 3 </sup> cells/well 96-well in plates were let attached overnight. To assess the time-dependent response, cells were treated with osimertinib at a tolerant concentration of 5 μM alone or in combination with the YAP inhibitor verteporfin (VP) at 5 μM for the indicated durations. To assess synergistic responses, cells were treated with VP alone at 0 – 10 μM or in combination with osimertinib at 0 – 5 μM. Survival of cells was evaluated with MTT (3-[4,5-dimethylthiazol-2-yl]-2,5 diphenyl tetrazolium bromide) assay. </p> <h3 class=\"c-article__sub-heading\" id=\"Sec25\"> Digestion of N-glycans on IL6 by PNGase F </h3> <p> IL6 in the CM previously described was immunoprecipitated. After elution, the SDS in the product was neutralized by the addition of 15% Triton X-100 (Sigma-Aldrich) to a final concentration of 0.7% before subsequent treatment. PNGase F (Sigma-Aldrich, P7367) was then added to the precipitated product (final concentration = 1 U/μg protein) with 10x buffer and ddH <sub> 2 </sub> O. The digestion reaction was performed by incubation at 37 °C for 72 h. The digested samples were subjected to WB for pattern analysis. </p> <h3 class=\"c-article__sub-heading\" id=\"Sec26\"> Isolation of tumor cells from MPEs </h3> <p> The purity of the attached tumor cells in the target fractions was determined by assessment of double-positive expression of TTF1 (Santa Cruz, sc-8762; RRID:AB_793530) and PanCK (Santa Cruz, sc-8018; RRID:AB_627396) under FV3000 confocal microscopy (Olympus). MPE-derived tumor cells were resuspended, cultured in RPMI 1640 medium supplemented with 10% FBS and maintained in an incubator at 37 °C with 5% CO <sub> 2 </sub> . Precipitated cells were cultured in 10 -cm dishes and allowed to attach for 24 h, after which the medium was replaced with fresh RPMI 1640 medium supplemented with 10% FBS and 1% Anti-Anti. </p> <h3 class=\"c-article__sub-heading\" id=\"Sec27\"> Plasmid cloning </h3> <p> Full-length human IL6 cDNA derived from A549 cells was cloned into the pRcCMV vector (Invitrogen, V750-20) using BamHI (NEB, R0136) and EcoRI (NEB, R0101) restriction sites, and the resulting vector was named pRcCMV-WT-IL6. Site-directed mutagenesis of IL6 by substitution of the N73 residue with Gln, Ala, or Val, was performed using the pRcCMV-WT-IL6 plasmid with a QuikChange Lightning Site-Directed Mutagenesis Kit (Agilent Technologies, 210518) according to the manufacturer’s instructions. The mutant forms were named pRcCMV-N73Q-IL6, pRcCMV-N73A-IL6, pRcCMV-N73V-IL6, pRcCMV-N172Q-IL6, and pRcCMV-N73Q + N172Q-IL6, respectively. The primers used for site-directed mutagenesis are listed in Supplementary Table <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 2 </a> . </p> <h3 class=\"c-article__sub-heading\" id=\"Sec28\"> <u> H </u> ollow <u> f </u> iber <u> b </u> ioreactor culture (HFB) </h3> <p> For collection of IL6 for lectin array analysis, a hollow fiber bioreactor was applied to improve productivity. In total, 1 × 10 <sup> 8 </sup> cells were implanted into a hollow fiber cartridge (FiberCell Systems, Inc., C-2008) after 5 days of preculture according to the manufacturer’s protocol. CM in the cartridge was collected every 5 days and centrifuged at 750 × g and 4 °C for 5 min. The supernatant was collected and stored at −80 °C until further use. </p> <h3 class=\"c-article__sub-heading\" id=\"Sec29\"> MS analysis and protein <u> id </u> entification (ID) </h3> <h4 class=\"c-article__sub-heading c-article__sub-heading--small\" id=\"Sec30\"> Protein identification </h4> <p> IL6 immunoprecipitated from HFB-derived lung cancer cell CM was resolved by 12% SDS-PAGE and visualized by Coomassie blue staining (Brilliant Blue G-Colloidal Concentrate, Sigma-Aldrich, 2025-1EA). Stained bands with an MW of ~20 ~ 30 kDa were excised and in-gel digested with trypsin (1 U/µg protein) (Sigma-Aldrich). The digested peptides were subsequently subjected to fractionation with a nano-HPLC system (LC Packings) coupled to an ion trap mass spectrometer (LCQ DECA XP Plus, Thermo Finnigan) equipped with an electrospray ion source. The resulting MS/MS spectra containing monoisotopic peptide masses were assigned and used for a search of the Swiss-Prot database with the Mascot search engine (Matrix Science, RRID: SCR_014322). </p> <h4 class=\"c-article__sub-heading c-article__sub-heading--small\" id=\"Sec31\"> Glycosylation analysis </h4> <p> Preparation of the IL6 protein was performed by the filter-aided sample preparation (FASP) method <sup> <a aria-label=\"Reference 70\" href=\"/articles/s41467-024-51831-7#ref-CR70\" id=\"ref-link-section-d6776021e4142\" title=\"Wisniewski, J. R., Zougman, A., Nagaraj, N. &amp; Mann, M. Universal sample preparation method for proteome analysis. Nat. Methods 6, 359–362 (2009).\"> 70 </a> </sup> . In brief, IL6 (10 μg samples) was reduced with 25 mM dithiothreitol and alkylated with 50 mM iodoacetamide in 0.1 M Tris-HCl (pH 8.5) with 8 M urea. Afterward, the buffer was replaced with 50 mM tetraethylammonium bromide (TEAB) for protein digestion. A total of 2 µg of glutamyl endopeptidase C (Glu-C) (Promega, V5111) or 1 μg of trypsin (Promega, V1651) was utilized for the analysis of IL6 peptides containing Asn73 (N73) or Asn 172 (N172), respectively. Then, N-glycans were removed by treatment with PNGase F. Digested IL6 samples were analyzed by LC–ESI-MS on an Orbitrap Fusion mass spectrometer (Thermo Fisher Scientific) connected to an EASY-nLC 1200 system (Thermo Fisher Scientific) and EASY-Spray™ Source (Thermo Fisher Scientific). The digestion solution was injected (5 μL) at a 1 μL/min flow rate onto an EASY-Column (C18, 0.075 mm × 150 mm, ID 3 μm; Thermo Fisher Scientific). Chromatographic separation was performed using 0.1% formic acid in water as mobile phase A and 0.1% formic acid in 80% acetonitrile as mobile phase B at a 300 nL/min flow rate. Briefly, the gradient employed was 5% buffer B at 2 min increasing to 60% buffer B at 55 min. Full-scan MS conditions: mass range, m/z 375–1800 (AGC target, 5E5) with lock mass; resolution, 60,000 at m/z 200; and maximum injection time, 50 ms. MS/MS scanning was performed in top speed mode with 3 s cycles with CID and HCD, while the dynamic exclusion duration was set to 60 s with a 10 ppm tolerance around the selected precursor and its isotopes. The electrospray voltage was maintained at 1.8 kV, and the capillary temperature was set at 275 °C. For glycopeptide identification, Byonic software 4.2.10 (Protein Metrics; RRID:SCR_016735) was used for the identification of summary formulas of glycans associated with glycopeptides (protein database, target protein; glycan database, 132 entries for N-glycans and 9 entries for O-glycans; precursor mass tolerance, 10 ppm; fragment mass tolerance, 0.5 Da; cleavage sites: K and R by trypsin and E and D by Glu-C; maximum missed cleavages, 5; and cysteine carbamidomethylation and methionine oxidation; FDR 0.01) <sup> <a aria-label=\"Reference 71\" href=\"/articles/s41467-024-51831-7#ref-CR71\" id=\"ref-link-section-d6776021e4146\" title=\"Bern, M., Kil, Y. J. &amp; Becker, C. Byonic: advanced peptide and protein identification software. Curr. Protocol. Bioinform. 13, 13 20 11–13 20 14 (2012).\"> 71 </a> </sup> . The percentage of glycosylation at N73 was determined from the areas in the LC spectra from the peptide sequence TCN <sup> 73 </sup> KSNMCE with/without deamidation at N73, and the percentages of glycosylation at N172 and O170 were calculated from the areas by Xcalibur in the LC spectra from the peptide sequence NLDAITTPDPT <sup> 170 </sup> TN <sup> 172 </sup> ASLLTK with/without deamidation at N172. The identified peptides are shown in Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 22 </a> . </p> <h3 class=\"c-article__sub-heading\" id=\"Sec32\"> RNA extraction, RT-PCR, and qPCR </h3> <p> Total RNA was extracted from the examined cells and tissues with TRIzol™ Reagent (Invitrogen, 15596026) following the manufacturer’s protocol. cDNA samples were synthesized from 1 μg of total RNA using the AMV Reverse Transcription System Kit (Promega, A3500) under the following thermal cycling conditions: 42 °C for 60 min, 99 °C for 5 min, and 4 °C for 30 min. Fifty nanograms of cDNA was subjected to RT-PCR, followed by agarose gel electrophoresis with SYBR Green. The expression of target genes was visualized with a UVP BioSpectrum® Imaging System™. Quantification of the transcript expression of human glycosyltransferases by qPCR was performed with 50 ng of cDNA in Power SYBR® Green Master Mix (Applied Bioscience, 4367659) using a Step One Real-Time PCR Detection System (Applied Bioscience). The relative expression of target genes was calculated by percentage changes of 2 <sup> −ΔΔCT </sup> values. The primer sequences used for qPCR analysis of glycosyltransferase expression were kindly provided by Prof. Chuan-Fa Chang. The primer sequences used for qPCR and RT-PCR are listed in Supplementary Table <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 2 </a> . </p> <h3 class=\"c-article__sub-heading\" id=\"Sec33\"> Cytokine array </h3> <p> Cytokine array assays were performed using the C-Series Human Cytokine Antibody Array (Ray-Bio, AAH-CYT-5-8) with intact or diluted CM following the manufacturer’s protocol. </p> <h3 class=\"c-article__sub-heading\" id=\"Sec34\"> Characterization of N-linked glycosylation by lectin microarray </h3> <p> Immunoprecipitated IL6 from CM of AS2-IL6-WT (with or without pretreatment with NGI-1), AS2-IL6-N73Q, HCC827-IL6, and HCC827-OR cells was separated by 12% SDS-PAGE to remove excess IgG and stained with InstantBlue® Coomassie Protein Stain (Abcam, ab119211). IL6 bands were excised and electroeluted with a D-tube of 3 MWCO. The eluted samples were then desalted by D-tube (Merck Millipore, 71506) dialysis against 1x PBS for 5 days (fresh 1x PBS was replaced every 12 h) and concentrated in 3 kDa Amicon Ultra-0.5 concentration tubes (Millipore, UFC500396). Lectin microarray assays were performed as described in GlycoStation Standard Experimental Protocol Ver.1.0. Briefly, the concentrations of the different IL6 glycoforms were determined by BCA Protein Assay reagent and adjusted to 50 µg/mL in 1x PBS. One microgram of each glycoform was labeled with Cy3 Mono-Reactive Dye (GE Healthcare, PA23001) and applied to a lectin microarray chip (LecChip Ver. 1.0 and LecChip Probing Solution, GlycoTechnica) for hybridization overnight at 4 °C. The resulting lectin chip was scanned with an evanescent-field fluorescence scanner (GlycoStation™ Reader 1200; GlycoTechnica) system ( <a href=\"https://www.glycotechnica.com/en/tools/\"> https://www.glycotechnica.com/en/tools/ </a> ). The net intensity of lectin signals was averaged from triplicate spots and automatically calculated with GlycoStation Tools Pro Suite 1.5 software. </p> <h3 class=\"c-article__sub-heading\" id=\"Sec35\"> Migration assays </h3> <h4 class=\"c-article__sub-heading c-article__sub-heading--small\" id=\"Sec36\"> Transwell </h4> <h3 class=\"c-article__sub-heading\" id=\"FPar1\"> CL1-0 cell coculture </h3> <p> AS2-Vec, AS2-IL6-WT, and AS2-IL6-N73Q cells (2.5 × 10 <sup> 5 </sup> ) were plated in a 24-well plate. After 24 h, the medium was replaced with fresh culture medium. In total, 1 × 10 <sup> 4 </sup> CL1-0 cells were plated in a Transwell MilliCell (Millipore, MCEP24H48), which was set on top of each AS2 cell derivative and incubated for 24 h. The cells that migrated through the membrane with an 8 μm pore size in the Transwell chamber were stained with Liu’s staining solution (ASK®). Images were acquired by an Olympus BX50 microscope (Olympus) under a 10x objective with VImage 2014 software, and the area of migrated cells was analyzed using ImageJ software (National Institutes of Health, RRID:SCR_003070). </p> <h3 class=\"c-article__sub-heading\" id=\"FPar2\"> Autonomous migration </h3> <p> AS2-Vec, AS2-IL6-WT, AS2-IL6-N73Q, AS2-IL6-N172Q, and AS2-IL6-N73Q + N172Q cells (1 × 10 <sup> 4 </sup> ) were plated in a Transwell MilliCell (Millipore), which was set on top of each AS2 cell derivative and incubated for 24 h. The cells that migrated through the membrane in the chamber were stained with Liu’s staining solution (ASK®). Images were acquired by an Olympus BX50 microscope (Olympus) under a 10x objective with VImage 2014 software, and the area of migrated cells was analyzed using ImageJ software (National Institutes of Health, RRID:SCR_003070). </p> <h4 class=\"c-article__sub-heading c-article__sub-heading--small\" id=\"Sec37\"> Culture inserts </h4> <p> Appropriate cells were seeded (1 × 10 <sup> 4 </sup> ) into both sides of a culture insert (ibidi, 81176) with mitomycin C (5 μg/mL, Sigma-Aldrich, M4287) and then allowed to attach overnight. The inserts were carefully removed, and the medium was replaced with fresh culture medium or, for inhibitor-treated cells, with medium containing the IL6-neutralizing antibody (5 μg/mL, R&amp;D, MAB206), STAT3 inhibitor S3I-201 (10 μM, Sigma-Aldrich, 573102), JAK inhibitor ruxolitinib (2.5 μM, Selleck Chemicals, S1378), and SRC inhibitors PP1 (10 μM, Sigma-Aldrich, P0040), PP2 (10 μM, Selleck Chemicals, S7008), and tirbanibulin (10 μM, MedChemExpress, HY-10340). Images were acquired by a Nikon ECLIPSE Ti microscope (Nikon) under a 20x objective at 0 h and 24 h after insert removal, and the area of migrated cells was analyzed using ImageJ software (National Institutes of Health, RRID: SCR_003070). </p> <h3 class=\"c-article__sub-heading\" id=\"Sec38\"> Animal experiments </h3> <p> All animal experiments were conducted under the Institutional Animal Care and Use Committee of NCKU (NCKU IACUC)-approved animal protocol No. 103091, No.104104, and No.107174, following the guidance of the of the Laboratory Animal Center of NCKU and the Association for Assessment and Accreditation of Laboratory Animal Care (AAALAC). For analysis of the influence of IL6 glycoforms on lung metastasis, 1 × 10 <sup> 6 </sup> AS2-Vec, AS2-IL6-WT, AS2-IL6-N73Q, AS2-IL6-N73A, and AS2-IL6-N73V cells were transplanted into female nude mice (randomized into three groups for comparison of AS2-Vec, AS2-IL6-WT, and AS2-IL6-N73Q cells, and two groups for comparison of AS2-IL6-N73A and AS2-IL6-N73V cells) by i.v. injection. Mice were observed every other day. For IVIS observation, 1 × 10 <sup> 6 </sup> AS2-IL6-WT-pLuc and AS2-IL6-N73Q-pLuc cells were transplanted into male nude mice (randomized into two groups) by i.v. injection and observed with IVIS Spectrum Noninvasive Quantitative Molecular Imaging System (Xenogen) twice a week for 4 weeks. Prior to IVIS measurement each time, the mice were administered 200 μL of D-luciferin (dissolved in normal saline at 3.7 mg/mL, Biosynth, L-8220) through intraperitoneal (i.p.) injection. Due to individual variation, if the maximum metastasis reached 60% of total lung area or the animal was dead before the set end point, the experiment would be humanely terminated by euthanasia with CO <sub> 2 </sub> upon that time. Lungs of these nude mice were harvested, fixed with formalin and embedded in paraffin (generating formalin-fixed, paraffin-embedded (FFPE) samples). We used both female and male nude mice for in vivo study due to the shortage of mice under the SARS-COVID-19 pandemic. </p> <h3 class=\"c-article__sub-heading\" id=\"Sec39\"> Histology </h3> <p> For histological analysis, 4 μm FFPE sections from the longitudinal central plane of the lungs harvested in the i.v. injection experiments, or paired TKI-naïve and TKI-resistant NSCLC lung tissues were stained with H&amp;E following a standard protocol and scanned with the TissueFAXS System (TissueGnostics GmbH). Both tumor area and total lung area were quantified using HistoQuest software (TissueGnostics GmbH, RRID:SCR_014823) and visually confirmed by a pathologist in a blinded manner. The metastatic capacity of lung tumor cells secreting various IL6 glycoforms was evaluated by calculating the ratio of the tumor area to the total lung area. </p> <h3 class=\"c-article__sub-heading\" id=\"Sec40\"> Microarray analysis </h3> <p> AS2-Vec cells were treated with WT-IL6-containing and N73Q-IL6-containing CM for 0.5, 3, 6, or 24 h. Total RNA was extracted and subjected to microarray analysis. The integrity of all the RNA samples was evaluated using a Bioanalyzer 2100 (Agilent Technologies) according to the manufacturer’s protocol. Analysis of gene expression in each sample was performed utilizing an Agilent SurePrint G3 Human GE 8 × 60 K Microarray (Agilent Technologies), and scanning was performed using an Agilent microarray scanner (Agilent Technologies). The acquired images were analyzed using Feature Extraction 10.5.1.1 software (Agilent Technologies). The experiments were performed in triplicate. The WT-IL6-CM-induced and N73Q-IL6-CM-induced gene expression at 0.5, 3, 6, and 24 h was normalized to that at 0 h. Differentially expressed genes (fold change &gt; 1.5) were characterized and clustered using GeneSpring software (RRID:SCR_009196, Agilent Technologies, assisted by Welgene, Inc., Taiwan). The data were subsequently analyzed with Ingenuity Pathway Analysis (IPA) software (Qiagen, RRID:SCR_008653) and GSEA <sup> <a aria-label=\"Reference 43\" href=\"/articles/s41467-024-51831-7#ref-CR43\" id=\"ref-link-section-d6776021e4271\" title=\"Subramanian, A. et al. Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. Proc. Natl Acad. Sci. USA 102, 15545–15550 (2005).\"> 43 </a> </sup> . </p> <h3 class=\"c-article__sub-heading\" id=\"Sec41\"> In silico transcriptome analysis </h3> <p> The datasets of EGFR-TKI-treated human lung cancer cells GSE31210, GSE60189, GSE59239, GSE106765, GSE122005, GSE38121, and GSE71587 were obtained from the NCBI GEO database, and patient RNA-seq results were obtained from the supplemental information from Roper et al. <sup> <a aria-label=\"Reference 59\" href=\"/articles/s41467-024-51831-7#ref-CR59\" id=\"ref-link-section-d6776021e4283\" title=\"Roper, N. et al. Clonal evolution and heterogeneity of osimertinib acquired resistance mechanisms in EGFR mutant lung cancer. Cell Rep. Med. 1, 10007 (2020).\"> 59 </a> </sup> . Data were converted to “.gct” files prior to GSEA. The.gct files contained the <u> r </u> obust <u> m </u> ulti <u> a </u> rray (RMA) normalized data matrix obtained from GEO with a probe set identifier. Enrichment analyses were performed by GSEA using the standard default settings with well-documented gene sets encompassing signatures of the N-glycosylation process, EMT, and SRC-YAP signaling obtained from the Molecular Signatures Database (MSigDB) ( <a href=\"https://www.gsea-msigdb.org/gsea/msigdb\"> https://www.gsea-msigdb.org/gsea/msigdb </a> ). Heatmaps showing ssGSEA scores and ΔssGSEA scores were generated with Morpheus open-source software (RRID:SCR_014975, <a href=\"https://software.broadinstitute.org/morpheus/\"> https://software.broadinstitute.org/morpheus/ </a> ). </p> <h3 class=\"c-article__sub-heading\" id=\"Sec42\"> Immunofluorescence staining and confocal microscopy </h3> <p> Cells in a 4-well chamber slide (Millipore, PEZGS0416) were fixed with 4% paraformaldehyde (PFA) for 10 min and then washed three times with 1x PBS for 10 min each time. The cells were then permeabilized with 0.1% Triton X-100 (Sigma-Aldrich) for 10 min and washed three times with 1x PBS for 10 min each time. The samples were then blocked with antibody diluent (Dako, S3022) for 1 h at room temperature. For analysis of the expression of TTF1 (Santa Cruz, sc-8762; RRID:AB_793530) and PanCK (Santa Cruz, sc-8018; RRID:AB_627396) or the localization of STAT3 (CST, 9193; RRID:AB_331757), SRC (CST, 2108; RRID:AB_331137), and GP130 (Santa Cruz, sc-376280; RRID:AB_10986409), the samples were reacted with the appropriate primary Ab diluted in antibody diluent overnight at 4 °C and then washed three times. The samples were then reacted with appropriate secondary Ab diluted in antibody diluent at room temperature for 1 h and washed three times. Nuclear staining was performed with Hoechst 33342 dye at 20 μg/mL (Sigma-Aldrich, 14533) for 1 min, and the samples were then washed three times with 1x PBS. The samples were subsequently mounted with Fluoromount-G (Electron Microscopy Sciences, 50-259-73) and sealed with cover glasses. Immunofluorescence images were acquired with an FV3000 confocal microscope (Olympus) and analyzed using FV10-ASW (Olympus) or TissueQuest (TissueGnostics GmbH) software. For analysis of the formation of F-ACTIN, cells and blocked samples were reacted with phalloidin-TRITC (Life Technology, R415) for 1 h at room temperature; the subsequent experimental steps were carried out as described above. For analysis of the expression and localization of STAT3 and SRC in paired TKI-naïve and TKI-resistant NSCLC tissues, 4 μm FFPE sections were applied, and the paired tissues were attached to the same slide to perform the IF process at the same time. Immunofluorescence images were acquired with a TissueFAXS System (TissueGnostics GmbH) and analyzed using TissueQuest (TissueGnostics GmbH) software. </p> <h3 class=\"c-article__sub-heading\" id=\"Sec43\"> Study approval </h3> <p> Research associated with the collection of human specimens was conducted in accordance with the Declaration of Helsinki (2008) of the World Medical Association. This study was approved by the institutional review boards (IRBs) of National Cheng Kung University Hospital (NCKU Hospital, Tainan, Taiwan). Written informed consent was obtained from each study subject (B-ER-1010170, BR-100-034, A-BR-102-098, A-ER-105-451). </p> <h3 class=\"c-article__sub-heading\" id=\"Sec44\"> Statistics and reproducibility </h3> <p> Statistical analyses were performed using Prism software (GraphPad, Inc., RRID:SCR_005375). The results of WB, qPCR, confocal microscopy, and migration and soft agar assays were statistically analyzed by the Student’s <i> t- </i> test. In vivo lung metastasis assay results were analyzed by two-way ANOVA. For Ingenuity Pathway Analysis (IPA) - identified “Disease and Functional Annotation”, a significant correlation was defined as <i> p </i> &lt; 0.001, evaluated by a right-tailed Fisher’s Exact Test. The synergistic effect of combination treatment with the YAPi and osimertinib was evaluated by Synergy Finder Plus ( <a href=\"http://www.synergyfinderplus.org/\"> http://www.synergyfinderplus.org/ </a> ) under the two-tailed Zero interaction potency (ZIP) model <sup> <a aria-label=\"Reference 58\" href=\"/articles/s41467-024-51831-7#ref-CR58\" id=\"ref-link-section-d6776021e4348\" title=\"Zheng, S. et al. SynergyFinder plus: toward better interpretation and annotation of drug combination screening datasets. Genom. Proteom. Bioinform. 20, 587–596 (2022).\"> 58 </a> </sup> . A <i> p </i> -value of &lt;0.05 was considered significant. The data are presented as the mean ± SEM for quantitative results. No statistical method was used to predetermine sample size. </p> <h3 class=\"c-article__sub-heading\" id=\"Sec45\"> Reporting summary </h3> <p> Further information on research design is available in the <a href=\"/articles/s41467-024-51831-7#MOESM3\"> Nature Portfolio Reporting Summary </a> linked to this article. </p> </div> </div> </section> </div> <div class=\"u-mt-32\"> <section> <div class=\"c-article-section\" id=\"data-availability-section\"> <h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"data-availability\"> Data availability </h2> <div class=\"c-article-section__content\" id=\"data-availability-content\"> <p> The RNA microarray datasets generated in this study were deposited into the National Center for Biotechnology Information (NCBI) Gene Expression Omnibus (GEO) database under accession number GSE269782 and are available at the following URL: <a href=\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE269782\"> https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE269782 </a> . The RNAseq and microarray datasets of human lung adenocarcinoma demonstrated in Figs. <a href=\"/articles/s41467-024-51831-7#Fig1\"> 1e </a> , <a href=\"/articles/s41467-024-51831-7#Fig6\"> 6h </a> , and Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 21 </a> were downloaded from NCBI’s GEO: <a href=\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE31210\"> GSE31210 </a> ; <a href=\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE269782\"> GSE106765 </a> ; <a href=\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60189\"> GSE60189 </a> ; <a href=\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59293\"> GSE59239 </a> ; <a href=\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=gse122005\"> GSE122005 </a> ; <a href=\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38121\"> GSE38121 </a> ; <a href=\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE71587\"> GSE71587 </a> ; <a href=\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE37699\"> GSE37699 </a> . Datasets of human lung adenocarcinoma demonstrated in Fig. <a href=\"/articles/s41467-024-51831-7#Fig8\"> 8d </a> and Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 26 </a> were from the supplementary information attached to the article of Roper N. et al. <sup> <a aria-label=\"Reference 59\" href=\"/articles/s41467-024-51831-7#ref-CR59\" id=\"ref-link-section-d6776021e4513\" title=\"Roper, N. et al. Clonal evolution and heterogeneity of osimertinib acquired resistance mechanisms in EGFR mutant lung cancer. Cell Rep. Med. 1, 10007 (2020).\"> 59 </a> </sup> . For glycosylation analysis, protein mass spectrometry raw data and analytical tool data have updated in a repository via ProteomeXchange ( <a href=\"https://www.proteomexchange.org\"> https://www.proteomexchange.org </a> ) with project accession: PXD051771. <a href=\"/articles/s41467-024-51831-7#Sec47\"> Source data </a> are provided with this paper. </p> </div> </div> </section> <div id=\"MagazineFulltextArticleBodySuffix\"> <section aria-labelledby=\"Bib1\"> <div class=\"c-article-section\" id=\"Bib1-section\"> <h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"Bib1\"> References </h2> <div class=\"c-article-section__content\" id=\"Bib1-content\"> <div> <ol class=\"c-article-references\"> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR1\"> Yu, H., Lee, H., Herrmann, A., Buettner, R. &amp; Jove, R. Revisiting STAT3 signalling in cancer: new and unexpected biological functions. <i> Nat. Rev. Cancer </i> <b> 14 </b> , 736–746 (2014). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 1\" href=\"https://doi.org/10.1038%2Fnrc3818\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 1\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC2cXhvVSmtLnF\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 1\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25342631\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 1\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Revisiting%20STAT3%20signalling%20in%20cancer%3A%20new%20and%20unexpected%20biological%20functions&amp;journal=Nat.%20Rev.%20Cancer&amp;doi=10.1038%2Fnrc3818&amp;volume=14&amp;pages=736-746&amp;publication_year=2014&amp;author=Yu%2CH&amp;author=Lee%2CH&amp;author=Herrmann%2CA&amp;author=Buettner%2CR&amp;author=Jove%2CR\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR2\"> Jones, S. A. &amp; Jenkins, B. J. Recent insights into targeting the IL-6 cytokine family in inflammatory diseases and cancer. <i> Nat. Rev. Immunol. </i> <b> 18 </b> , 773–789 (2018). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 2\" href=\"https://doi.org/10.1038%2Fs41577-018-0066-7\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 2\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC1cXhslymur7N\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 2\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30254251\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 2\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Recent%20insights%20into%20targeting%20the%20IL-6%20cytokine%20family%20in%20inflammatory%20diseases%20and%20cancer&amp;journal=Nat.%20Rev.%20Immunol.&amp;doi=10.1038%2Fs41577-018-0066-7&amp;volume=18&amp;pages=773-789&amp;publication_year=2018&amp;author=Jones%2CSA&amp;author=Jenkins%2CBJ\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR3\"> Rose-John, S. The soluble interleukin 6 receptor: advanced therapeutic options in inflammation. <i> Clin. Pharmacol. Therapeut. </i> <b> 102 </b> , 591–598 (2017). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 3\" href=\"https://doi.org/10.1002%2Fcpt.782\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 3\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC2sXhsFWjt77I\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"Google Scholar reference 3\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=The%20soluble%20interleukin%206%20receptor%3A%20advanced%20therapeutic%20options%20in%20inflammation&amp;journal=Clin.%20Pharmacol.%20Therapeut.&amp;doi=10.1002%2Fcpt.782&amp;volume=102&amp;pages=591-598&amp;publication_year=2017&amp;author=Rose-John%2CS\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR4\"> Trikha, M., Corringham, R., Klein, B. &amp; Rossi, J. F. Targeted anti-interleukin-6 monoclonal antibody therapy for cancer: a review of the rationale and clinical evidence. <i> Clin. Cancer Res. </i> <b> 9 </b> , 4653–4665 (2003). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"CAS reference 4\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BD3sXosVGjsrw%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 4\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=14581334\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"PubMed Central reference 4\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2929399\" rel=\"nofollow noopener\"> PubMed Central </a> <a aria-label=\"Google Scholar reference 4\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Targeted%20anti-interleukin-6%20monoclonal%20antibody%20therapy%20for%20cancer%3A%20a%20review%20of%20the%20rationale%20and%20clinical%20evidence&amp;journal=Clin.%20Cancer%20Res.&amp;volume=9&amp;pages=4653-4665&amp;publication_year=2003&amp;author=Trikha%2CM&amp;author=Corringham%2CR&amp;author=Klein%2CB&amp;author=Rossi%2CJF\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR5\"> Hunter, C. A. &amp; Jones, S. A. IL-6 as a keystone cytokine in health and disease. <i> Nat. Immunol </i> . <b> 16 </b> , 448–457 (2015). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 5\" href=\"https://doi.org/10.1038%2Fni.3153\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 5\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC2MXhtF2murbF\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 5\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25898198\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 5\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=IL-6%20as%20a%20keystone%20cytokine%20in%20health%20and%20disease&amp;journal=Nat.%20Immunol&amp;doi=10.1038%2Fni.3153&amp;volume=16&amp;pages=448-457&amp;publication_year=2015&amp;author=Hunter%2CCA&amp;author=Jones%2CSA\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR6\"> Garbers, C., Heink, S., Korn, T. &amp; Rose-John, S. Interleukin-6: designing specific therapeutics for a complex cytokine. <i> Nat. Rev. Drug Discov. </i> <b> 17 </b> , 395–412 (2018). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 6\" href=\"https://doi.org/10.1038%2Fnrd.2018.45\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 6\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC1cXovVagtbw%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 6\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29725131\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 6\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Interleukin-6%3A%20designing%20specific%20therapeutics%20for%20a%20complex%20cytokine&amp;journal=Nat.%20Rev.%20Drug%20Discov.&amp;doi=10.1038%2Fnrd.2018.45&amp;volume=17&amp;pages=395-412&amp;publication_year=2018&amp;author=Garbers%2CC&amp;author=Heink%2CS&amp;author=Korn%2CT&amp;author=Rose-John%2CS\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR7\"> Murakami, M., Kamimura, D. &amp; Hirano, T. Pleiotropy and specificity: insights from the interleukin 6 family of cytokines. <i> Immunity </i> <b> 50 </b> , 812–831 (2019). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 7\" href=\"https://doi.org/10.1016%2Fj.immuni.2019.03.027\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 7\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC1MXnsFGksLY%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 7\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30995501\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 7\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Pleiotropy%20and%20specificity%3A%20insights%20from%20the%20interleukin%206%20family%20of%20cytokines&amp;journal=Immunity&amp;doi=10.1016%2Fj.immuni.2019.03.027&amp;volume=50&amp;pages=812-831&amp;publication_year=2019&amp;author=Murakami%2CM&amp;author=Kamimura%2CD&amp;author=Hirano%2CT\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR8\"> Duan, Z. et al. Signal transducers and activators of transcription 3 pathway activation in drug-resistant ovarian cancer. <i> Clin. Cancer Res. </i> <b> 12 </b> , 5055–5063 (2006). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 8\" href=\"https://doi.org/10.1158%2F1078-0432.CCR-06-0861\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 8\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BD28XovF2hu7s%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 8\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16951221\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 8\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Signal%20transducers%20and%20activators%20of%20transcription%203%20pathway%20activation%20in%20drug-resistant%20ovarian%20cancer&amp;journal=Clin.%20Cancer%20Res.&amp;doi=10.1158%2F1078-0432.CCR-06-0861&amp;volume=12&amp;pages=5055-5063&amp;publication_year=2006&amp;author=Duan%2CZ\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR9\"> Huang, W. L. et al. Signal transducer and activator of transcription 3 activation up-regulates interleukin-6 autocrine production: a biochemical and genetic study of established cancer cell lines and clinical isolated human cancer cells. <i> Mol. Cancer </i> <b> 9 </b> , 309 (2010). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 9\" href=\"https://link.springer.com/doi/10.1186/1476-4598-9-309\" rel=\"noopener\"> Article </a> <a aria-label=\"CAS reference 9\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC3cXhsF2hs7rK\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 9\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21122157\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"PubMed Central reference 9\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3027602\" rel=\"nofollow noopener\"> PubMed Central </a> <a aria-label=\"Google Scholar reference 9\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Signal%20transducer%20and%20activator%20of%20transcription%203%20activation%20up-regulates%20interleukin-6%20autocrine%20production%3A%20a%20biochemical%20and%20genetic%20study%20of%20established%20cancer%20cell%20lines%20and%20clinical%20isolated%20human%20cancer%20cells&amp;journal=Mol.%20Cancer&amp;doi=10.1186%2F1476-4598-9-309&amp;volume=9&amp;publication_year=2010&amp;author=Huang%2CWL\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR10\"> Lee, H. J. et al. Drug resistance via feedback activation of Stat3 in oncogene-addicted cancer cells. <i> Cancer Cell </i> <b> 26 </b> , 207–221 (2014). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 10\" href=\"https://doi.org/10.1016%2Fj.ccr.2014.05.019\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 10\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC2cXht1agsL7I\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 10\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25065853\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 10\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Drug%20resistance%20via%20feedback%20activation%20of%20Stat3%20in%20oncogene-addicted%20cancer%20cells&amp;journal=Cancer%20Cell&amp;doi=10.1016%2Fj.ccr.2014.05.019&amp;volume=26&amp;pages=207-221&amp;publication_year=2014&amp;author=Lee%2CHJ\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR11\"> Rokavec, M. et al. IL-6R/STAT3/miR-34a feedback loop promotes EMT-mediated colorectal cancer invasion and metastasis. <i> J. Clin. Invest. </i> <b> 124 </b> , 1853–1867 (2014). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 11\" href=\"https://doi.org/10.1172%2FJCI73531\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 11\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC2cXmtVGgur4%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 11\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24642471\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"PubMed Central reference 11\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3973098\" rel=\"nofollow noopener\"> PubMed Central </a> <a aria-label=\"Google Scholar reference 11\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=IL-6R%2FSTAT3%2FmiR-34a%20feedback%20loop%20promotes%20EMT-mediated%20colorectal%20cancer%20invasion%20and%20metastasis&amp;journal=J.%20Clin.%20Invest.&amp;doi=10.1172%2FJCI73531&amp;volume=124&amp;pages=1853-1867&amp;publication_year=2014&amp;author=Rokavec%2CM\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR12\"> Taniguchi, K. et al. A gp130-Src-YAP module links inflammation to epithelial regeneration. <i> Nature </i> <b> 519 </b> , 57–62 (2015). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 12\" href=\"https://doi.org/10.1038%2Fnature14228\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"ADS reference 12\" href=\"http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2015Natur.519...57T\" rel=\"nofollow noopener\"> ADS </a> <a aria-label=\"CAS reference 12\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC2MXjvVGjsrk%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 12\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25731159\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"PubMed Central reference 12\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4447318\" rel=\"nofollow noopener\"> PubMed Central </a> <a aria-label=\"Google Scholar reference 12\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=A%20gp130-Src-YAP%20module%20links%20inflammation%20to%20epithelial%20regeneration&amp;journal=Nature&amp;doi=10.1038%2Fnature14228&amp;volume=519&amp;pages=57-62&amp;publication_year=2015&amp;author=Taniguchi%2CK\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR13\"> Quintanal-Villalonga, A. et al. Lineage plasticity in cancer: a shared pathway of therapeutic resistance. <i> Nat. Rev. Clin. Oncol. </i> <b> 17 </b> , 360–371 (2020). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 13\" href=\"https://doi.org/10.1038%2Fs41571-020-0340-z\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"PubMed reference 13\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32152485\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"PubMed Central reference 13\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7397755\" rel=\"nofollow noopener\"> PubMed Central </a> <a aria-label=\"Google Scholar reference 13\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Lineage%20plasticity%20in%20cancer%3A%20a%20shared%20pathway%20of%20therapeutic%20resistance&amp;journal=Nat.%20Rev.%20Clin.%20Oncol.&amp;doi=10.1038%2Fs41571-020-0340-z&amp;volume=17&amp;pages=360-371&amp;publication_year=2020&amp;author=Quintanal-Villalonga%2CA\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR14\"> Paez, J. G. et al. EGFR mutations in lung cancer: correlation with clinical response to gefitinib therapy. <i> Science </i> <b> 304 </b> , 1497–1500 (2004). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 14\" href=\"https://doi.org/10.1126%2Fscience.1099314\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"ADS reference 14\" href=\"http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2004Sci...304.1497P\" rel=\"nofollow noopener\"> ADS </a> <a aria-label=\"CAS reference 14\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BD2cXksVGmsbs%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 14\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15118125\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 14\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=EGFR%20mutations%20in%20lung%20cancer%3A%20correlation%20with%20clinical%20response%20to%20gefitinib%20therapy&amp;journal=Science&amp;doi=10.1126%2Fscience.1099314&amp;volume=304&amp;pages=1497-1500&amp;publication_year=2004&amp;author=Paez%2CJG\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR15\"> Ramalingam, S. S. et al. Overall survival with osimertinib in untreated, EGFR-mutated advanced NSCLC. <i> N. Engl. J. Med. </i> <b> 382 </b> , 41–50 (2020). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 15\" href=\"https://doi.org/10.1056%2FNEJMoa1913662\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 15\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BB3cXnsFGgtL4%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 15\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31751012\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 15\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Overall%20survival%20with%20osimertinib%20in%20untreated%2C%20EGFR-mutated%20advanced%20NSCLC&amp;journal=N.%20Engl.%20J.%20Med.&amp;doi=10.1056%2FNEJMoa1913662&amp;volume=382&amp;pages=41-50&amp;publication_year=2020&amp;author=Ramalingam%2CSS\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR16\"> Oxnard, G. R. et al. Assessment of resistance mechanisms and clinical implications in patients with EGFR T790M-positive lung cancer and acquired resistance to osimertinib. <i> JAMA Oncol. </i> <b> 4 </b> , 1527–1534 (2018). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 16\" href=\"https://doi.org/10.1001%2Fjamaoncol.2018.2969\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"PubMed reference 16\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30073261\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 16\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Assessment%20of%20resistance%20mechanisms%20and%20clinical%20implications%20in%20patients%20with%20EGFR%20T790M-positive%20lung%20cancer%20and%20acquired%20resistance%20to%20osimertinib&amp;journal=JAMA%20Oncol.&amp;doi=10.1001%2Fjamaoncol.2018.2969&amp;volume=4&amp;pages=1527-1534&amp;publication_year=2018&amp;author=Oxnard%2CGR\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR17\"> Yang, Z. et al. Investigating novel resistance mechanisms to third-generation EGFR tyrosine kinase inhibitor osimertinib in non-small cell lung cancer patients. <i> Clin. Cancer Res. </i> <b> 24 </b> , 3097–3107 (2018). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 17\" href=\"https://doi.org/10.1158%2F1078-0432.CCR-17-2310\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 17\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC1cXht1alurvK\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 17\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29506987\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 17\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Investigating%20novel%20resistance%20mechanisms%20to%20third-generation%20EGFR%20tyrosine%20kinase%20inhibitor%20osimertinib%20in%20non-small%20cell%20lung%20cancer%20patients&amp;journal=Clin.%20Cancer%20Res.&amp;doi=10.1158%2F1078-0432.CCR-17-2310&amp;volume=24&amp;pages=3097-3107&amp;publication_year=2018&amp;author=Yang%2CZ\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR18\"> Sequist, L. V. et al. Genotypic and histological evolution of lung cancers acquiring resistance to EGFR inhibitors. <i> Sci. Transl. Med. </i> <b> 3 </b> , 75ra26 (2011). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 18\" href=\"https://doi.org/10.1126%2Fscitranslmed.3002003\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"PubMed reference 18\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21430269\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"PubMed Central reference 18\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3132801\" rel=\"nofollow noopener\"> PubMed Central </a> <a aria-label=\"Google Scholar reference 18\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Genotypic%20and%20histological%20evolution%20of%20lung%20cancers%20acquiring%20resistance%20to%20EGFR%20inhibitors&amp;journal=Sci.%20Transl.%20Med.&amp;doi=10.1126%2Fscitranslmed.3002003&amp;volume=3&amp;publication_year=2011&amp;author=Sequist%2CLV\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR19\"> Lambert, A. W. &amp; Weinberg, R. A. Linking EMT programmes to normal and neoplastic epithelial stem cells. <i> Nat. Rev. Cancer </i> <b> 21 </b> , 325–338 (2021). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 19\" href=\"https://doi.org/10.1038%2Fs41568-021-00332-6\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 19\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BB3MXjsVCquro%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 19\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33547455\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 19\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Linking%20EMT%20programmes%20to%20normal%20and%20neoplastic%20epithelial%20stem%20cells&amp;journal=Nat.%20Rev.%20Cancer&amp;doi=10.1038%2Fs41568-021-00332-6&amp;volume=21&amp;pages=325-338&amp;publication_year=2021&amp;author=Lambert%2CAW&amp;author=Weinberg%2CRA\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR20\"> Williams, E. D., Gao, D., Redfern, A. &amp; Thompson, E. W. Controversies around epithelial-mesenchymal plasticity in cancer metastasis. <i> Nat. Rev. Cancer </i> <b> 19 </b> , 716–732 (2019). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 20\" href=\"https://doi.org/10.1038%2Fs41568-019-0213-x\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 20\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC1MXitV2hsb7J\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 20\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31666716\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"PubMed Central reference 20\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7055151\" rel=\"nofollow noopener\"> PubMed Central </a> <a aria-label=\"Google Scholar reference 20\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Controversies%20around%20epithelial-mesenchymal%20plasticity%20in%20cancer%20metastasis&amp;journal=Nat.%20Rev.%20Cancer&amp;doi=10.1038%2Fs41568-019-0213-x&amp;volume=19&amp;pages=716-732&amp;publication_year=2019&amp;author=Williams%2CED&amp;author=Gao%2CD&amp;author=Redfern%2CA&amp;author=Thompson%2CEW\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR21\"> Yao, Z. et al. TGF-beta IL-6 axis mediates selective and adaptive mechanisms of resistance to molecular targeted therapy in lung cancer. <i> Proc. Natl Acad. Sci. USA </i> <b> 107 </b> , 15535–15540 (2010). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 21\" href=\"https://doi.org/10.1073%2Fpnas.1009472107\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"ADS reference 21\" href=\"http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2010PNAS..10715535Y\" rel=\"nofollow noopener\"> ADS </a> <a aria-label=\"CAS reference 21\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtFaqu7%2FN\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 21\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20713723\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"PubMed Central reference 21\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2932568\" rel=\"nofollow noopener\"> PubMed Central </a> <a aria-label=\"Google Scholar reference 21\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=TGF-beta%20IL-6%20axis%20mediates%20selective%20and%20adaptive%20mechanisms%20of%20resistance%20to%20molecular%20targeted%20therapy%20in%20lung%20cancer&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.1009472107&amp;volume=107&amp;pages=15535-15540&amp;publication_year=2010&amp;author=Yao%2CZ\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR22\"> Kim, S. M. et al. Activation of IL-6R/JAK1/STAT signaling induces de novo resistance to irreversible EGFR inhibitors in non-small cell lung cancer with T790M resistance mutation. <i> Mol. Cancer Ther. </i> <b> 11 </b> , 2254–2264 (2012). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 22\" href=\"https://doi.org/10.1158%2F1535-7163.MCT-12-0311\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 22\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC38XhsVyksrvI\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 22\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22891040\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 22\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Activation%20of%20IL-6R%2FJAK1%2FSTAT%20signaling%20induces%20de%20novo%20resistance%20to%20irreversible%20EGFR%20inhibitors%20in%20non-small%20cell%20lung%20cancer%20with%20T790M%20resistance%20mutation&amp;journal=Mol.%20Cancer%20Ther.&amp;doi=10.1158%2F1535-7163.MCT-12-0311&amp;volume=11&amp;pages=2254-2264&amp;publication_year=2012&amp;author=Kim%2CSM\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR23\"> Korkaya, H., Liu, S. &amp; Wicha, M. S. Regulation of cancer stem cells by cytokine networks: attacking cancer’s inflammatory roots. <i> Clin. Cancer Res. </i> <b> 17 </b> , 6125–6129 (2011). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 23\" href=\"https://doi.org/10.1158%2F1078-0432.CCR-10-2743\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 23\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC3MXht1Ggs7jP\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 23\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21685479\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"PubMed Central reference 23\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3312242\" rel=\"nofollow noopener\"> PubMed Central </a> <a aria-label=\"Google Scholar reference 23\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Regulation%20of%20cancer%20stem%20cells%20by%20cytokine%20networks%3A%20attacking%20cancer%E2%80%99s%20inflammatory%20roots&amp;journal=Clin.%20Cancer%20Res.&amp;doi=10.1158%2F1078-0432.CCR-10-2743&amp;volume=17&amp;pages=6125-6129&amp;publication_year=2011&amp;author=Korkaya%2CH&amp;author=Liu%2CS&amp;author=Wicha%2CMS\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR24\"> Senescence-Associated Pro-inflammatory Cytokines, VernotJ. P. and tumor cell plasticity. <i> Front. Mol. Biosci. </i> <b> 7 </b> , 63 (2020). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"ADS reference 24\" href=\"http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2020FrMat...7...63S\" rel=\"nofollow noopener\"> ADS </a> <a aria-label=\"Google Scholar reference 24\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=and%20tumor%20cell%20plasticity&amp;journal=Front.%20Mol.%20Biosci.&amp;volume=7&amp;publication_year=2020&amp;author=Senescence-Associated%20Pro-inflammatory%20Cytokines%2CVernotJP\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR25\"> Kurppa, K. J. et al. Treatment-induced tumor dormancy through YAP-mediated transcriptional reprogramming of the apoptotic pathway. <i> Cancer Cell </i> <b> 37 </b> , 104–122.e112 (2020). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 25\" href=\"https://doi.org/10.1016%2Fj.ccell.2019.12.006\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 25\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BB3cXotVent78%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 25\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31935369\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"PubMed Central reference 25\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7146079\" rel=\"nofollow noopener\"> PubMed Central </a> <a aria-label=\"Google Scholar reference 25\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Treatment-induced%20tumor%20dormancy%20through%20YAP-mediated%20transcriptional%20reprogramming%20of%20the%20apoptotic%20pathway&amp;journal=Cancer%20Cell&amp;doi=10.1016%2Fj.ccell.2019.12.006&amp;volume=37&amp;pages=104-122.e112&amp;publication_year=2020&amp;author=Kurppa%2CKJ\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR26\"> Ferone, G. et al. SOX2 is the determining oncogenic switch in promoting lung squamous cell carcinoma from different cells of origin. <i> Cancer Cell </i> <b> 30 </b> , 519–532 (2016). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 26\" href=\"https://doi.org/10.1016%2Fj.ccell.2016.09.001\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 26\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC28Xhs1GqtLbP\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 26\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27728803\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"PubMed Central reference 26\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5065004\" rel=\"nofollow noopener\"> PubMed Central </a> <a aria-label=\"Google Scholar reference 26\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=SOX2%20is%20the%20determining%20oncogenic%20switch%20in%20promoting%20lung%20squamous%20cell%20carcinoma%20from%20different%20cells%20of%20origin&amp;journal=Cancer%20Cell&amp;doi=10.1016%2Fj.ccell.2016.09.001&amp;volume=30&amp;pages=519-532&amp;publication_year=2016&amp;author=Ferone%2CG\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR27\"> Boumahdi, S. &amp; de Sauvage, F. J. The great escape: tumour cell plasticity in resistance to targeted therapy. <i> Nat. Rev. Drug Discov. </i> <b> 19 </b> , 39–56 (2020). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 27\" href=\"https://doi.org/10.1038%2Fs41573-019-0044-1\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 27\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC1MXhvFCktL%2FM\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 27\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31601994\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 27\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=The%20great%20escape%3A%20tumour%20cell%20plasticity%20in%20resistance%20to%20targeted%20therapy&amp;journal=Nat.%20Rev.%20Drug%20Discov.&amp;doi=10.1038%2Fs41573-019-0044-1&amp;volume=19&amp;pages=39-56&amp;publication_year=2020&amp;author=Boumahdi%2CS&amp;author=Sauvage%2CFJ\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR28\"> Mosteiro, L. et al. Tissue damage and senescence provide critical signals for cellular reprogramming in vivo. <i> Science </i> <b> 354 </b> , 6315 (2016). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 28\" href=\"https://doi.org/10.1126%2Fscience.aaf4445\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"Google Scholar reference 28\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Tissue%20damage%20and%20senescence%20provide%20critical%20signals%20for%20cellular%20reprogramming%20in%20vivo&amp;journal=Science&amp;doi=10.1126%2Fscience.aaf4445&amp;volume=354&amp;publication_year=2016&amp;author=Mosteiro%2CL\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR29\"> Bora-Singhal, N. et al. YAP1 regulates OCT4 activity and SOX2 expression to facilitate self-renewal and vascular mimicry of stem-like cells. <i> Stem cells </i> <b> 33 </b> , 1705–1718 (2015). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 29\" href=\"https://doi.org/10.1002%2Fstem.1993\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 29\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC2MXhtleqsLvK\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 29\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25754111\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 29\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=YAP1%20regulates%20OCT4%20activity%20and%20SOX2%20expression%20to%20facilitate%20self-renewal%20and%20vascular%20mimicry%20of%20stem-like%20cells&amp;journal=Stem%20cells&amp;doi=10.1002%2Fstem.1993&amp;volume=33&amp;pages=1705-1718&amp;publication_year=2015&amp;author=Bora-Singhal%2CN\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR30\"> Poupart, P. et al. B cell growth modulating and differentiating activity of recombinant human 26-kd protein (BSF-2, HuIFN-beta 2, HPGF). <i> EMBO J </i> <b> 6 </b> , 1219–1224 (1987). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 30\" href=\"https://doi.org/10.1002%2Fj.1460-2075.1987.tb02357.x\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 30\" href=\"/articles/cas-redirect/1:CAS:528:DyaL2sXkt1Wktr0%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 30\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=3497029\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"PubMed Central reference 30\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC553922\" rel=\"nofollow noopener\"> PubMed Central </a> <a aria-label=\"Google Scholar reference 30\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=B%20cell%20growth%20modulating%20and%20differentiating%20activity%20of%20recombinant%20human%2026-kd%20protein%20%28BSF-2%2C%20HuIFN-beta%202%2C%20HPGF%29&amp;journal=EMBO%20J&amp;doi=10.1002%2Fj.1460-2075.1987.tb02357.x&amp;volume=6&amp;pages=1219-1224&amp;publication_year=1987&amp;author=Poupart%2CP\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR31\"> May, L. T. et al. Phosphorylation of secreted forms of human beta 2-interferon/hepatocyte stimulating factor/interleukin-6. <i> Biochem. Biophys. Res. Commun. </i> <b> 152 </b> , 1144–1150 (1988). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 31\" href=\"https://doi.org/10.1016%2FS0006-291X%2888%2980404-2\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 31\" href=\"/articles/cas-redirect/1:CAS:528:DyaL1cXktVGjt70%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 31\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=3259872\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 31\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Phosphorylation%20of%20secreted%20forms%20of%20human%20beta%202-interferon%2Fhepatocyte%20stimulating%20factor%2Finterleukin-6&amp;journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;doi=10.1016%2FS0006-291X%2888%2980404-2&amp;volume=152&amp;pages=1144-1150&amp;publication_year=1988&amp;author=May%2CLT\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR32\"> Hirano, T. et al. Purification to homogeneity and characterization of human B-cell differentiation factor (BCDF or BSFp-2). <i> Proc. Natl Acad. Sci. USA </i> <b> 82 </b> , 5490–5494 (1985). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 32\" href=\"https://doi.org/10.1073%2Fpnas.82.16.5490\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"ADS reference 32\" href=\"http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=1985PNAS...82.5490H\" rel=\"nofollow noopener\"> ADS </a> <a aria-label=\"CAS reference 32\" href=\"/articles/cas-redirect/1:CAS:528:DyaL2MXlt1Wnt7s%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 32\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=2410927\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"PubMed Central reference 32\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC391148\" rel=\"nofollow noopener\"> PubMed Central </a> <a aria-label=\"Google Scholar reference 32\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Purification%20to%20homogeneity%20and%20characterization%20of%20human%20B-cell%20differentiation%20factor%20%28BCDF%20or%20BSFp-2%29&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.82.16.5490&amp;volume=82&amp;pages=5490-5494&amp;publication_year=1985&amp;author=Hirano%2CT\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR33\"> May, L. T., Santhanam, U. &amp; Sehgal, P. B. On the multimeric nature of natural human interleukin-6. <i> J. Biol. Chem. </i> <b> 266 </b> , 9950–9955 (1991). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 33\" href=\"https://doi.org/10.1016%2FS0021-9258%2818%2992911-X\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 33\" href=\"/articles/cas-redirect/1:CAS:528:DyaK3MXkt1Onsbw%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 33\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=2033081\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 33\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=On%20the%20multimeric%20nature%20of%20natural%20human%20interleukin-6&amp;journal=J.%20Biol.%20Chem.&amp;doi=10.1016%2FS0021-9258%2818%2992911-X&amp;volume=266&amp;pages=9950-9955&amp;publication_year=1991&amp;author=May%2CLT&amp;author=Santhanam%2CU&amp;author=Sehgal%2CPB\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR34\"> Mereiter, S., Balmana, M., Campos, D., Gomes, J. &amp; Reis, C. A. Glycosylation in the era of cancer-targeted therapy: where are we heading? <i> Cancer Cell </i> <b> 36 </b> , 6–16 (2019). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 34\" href=\"https://doi.org/10.1016%2Fj.ccell.2019.06.006\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 34\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtlersb%2FJ\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 34\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31287993\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 34\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Glycosylation%20in%20the%20era%20of%20cancer-targeted%20therapy%3A%20where%20are%20we%20heading%3F&amp;journal=Cancer%20Cell&amp;doi=10.1016%2Fj.ccell.2019.06.006&amp;volume=36&amp;pages=6-16&amp;publication_year=2019&amp;author=Mereiter%2CS&amp;author=Balmana%2CM&amp;author=Campos%2CD&amp;author=Gomes%2CJ&amp;author=Reis%2CCA\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR35\"> Parekh, R. B., Dwek, R. A., Rademacher, T. W., Opdenakker, G. &amp; Van Damme, J. Glycosylation of interleukin-6 purified from normal human blood mononuclear cells. <i> Eur. J. Biochem. </i> <b> 203 </b> , 135–141 (1992). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 35\" href=\"https://doi.org/10.1111%2Fj.1432-1033.1992.tb19838.x\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 35\" href=\"/articles/cas-redirect/1:CAS:528:DyaK38Xht1KmtL8%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 35\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=1730219\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 35\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Glycosylation%20of%20interleukin-6%20purified%20from%20normal%20human%20blood%20mononuclear%20cells&amp;journal=Eur.%20J.%20Biochem.&amp;doi=10.1111%2Fj.1432-1033.1992.tb19838.x&amp;volume=203&amp;pages=135-141&amp;publication_year=1992&amp;author=Parekh%2CRB&amp;author=Dwek%2CRA&amp;author=Rademacher%2CTW&amp;author=Opdenakker%2CG&amp;author=Damme%2CJ\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR36\"> Dela Cruz, C. S. et al. N-linked glycosylation is required for optimal function of Kaposi’s sarcoma herpesvirus-encoded, but not cellular, interleukin 6. <i> J. Exp. Med. </i> <b> 199 </b> , 503–514 (2004). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 36\" href=\"https://doi.org/10.1084%2Fjem.20031205\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"Google Scholar reference 36\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=N-linked%20glycosylation%20is%20required%20for%20optimal%20function%20of%20Kaposi%E2%80%99s%20sarcoma%20herpesvirus-encoded%2C%20but%20not%20cellular%2C%20interleukin%206&amp;journal=J.%20Exp.%20Med.&amp;doi=10.1084%2Fjem.20031205&amp;volume=199&amp;pages=503-514&amp;publication_year=2004&amp;author=Cruz%2CCS\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR37\"> Thomas, D., Rathinavel, A. K. &amp; Radhakrishnan, P. Altered glycosylation in cancer: a promising target for biomarkers and therapeutics. <i> Biochimica. et biophysica. Acta Rev. Cancer </i> <b> 1875 </b> , 188464 (2021). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 37\" href=\"https://doi.org/10.1016%2Fj.bbcan.2020.188464\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 37\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BB3cXitl2msr%2FP\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"Google Scholar reference 37\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Altered%20glycosylation%20in%20cancer%3A%20a%20promising%20target%20for%20biomarkers%20and%20therapeutics&amp;journal=Biochimica.%20et%20biophysica.%20Acta%20Rev.%20Cancer&amp;doi=10.1016%2Fj.bbcan.2020.188464&amp;volume=1875&amp;publication_year=2021&amp;author=Thomas%2CD&amp;author=Rathinavel%2CAK&amp;author=Radhakrishnan%2CP\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR38\"> Pinho, S. S. &amp; Reis, C. A. Glycosylation in cancer: mechanisms and clinical implications. <i> Nat. Rev. Cancer </i> <b> 15 </b> , 540–555 (2015). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 38\" href=\"https://doi.org/10.1038%2Fnrc3982\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 38\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC2MXhtlKkt7fK\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 38\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26289314\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 38\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Glycosylation%20in%20cancer%3A%20mechanisms%20and%20clinical%20implications&amp;journal=Nat.%20Rev.%20Cancer&amp;doi=10.1038%2Fnrc3982&amp;volume=15&amp;pages=540-555&amp;publication_year=2015&amp;author=Pinho%2CSS&amp;author=Reis%2CCA\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR39\"> Yeh, H. H., Lai, W. W., Chen, H. H., Liu, H. S. &amp; Su, W. C. Autocrine IL-6-induced Stat3 activation contributes to the pathogenesis of lung adenocarcinoma and malignant pleural effusion. <i> Oncogene </i> <b> 25 </b> , 4300–4309 (2006). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 39\" href=\"https://doi.org/10.1038%2Fsj.onc.1209464\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 39\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BD28XntVWks7o%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 39\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16518408\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 39\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Autocrine%20IL-6-induced%20Stat3%20activation%20contributes%20to%20the%20pathogenesis%20of%20lung%20adenocarcinoma%20and%20malignant%20pleural%20effusion&amp;journal=Oncogene&amp;doi=10.1038%2Fsj.onc.1209464&amp;volume=25&amp;pages=4300-4309&amp;publication_year=2006&amp;author=Yeh%2CHH&amp;author=Lai%2CWW&amp;author=Chen%2CHH&amp;author=Liu%2CHS&amp;author=Su%2CWC\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR40\"> Roth, J. Protein N-glycosylation along the secretory pathway: relationship to organelle topography and function, protein quality control, and cell interactions. <i> Chem. Rev. </i> <b> 102 </b> , 285–303 (2002). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 40\" href=\"https://doi.org/10.1021%2Fcr000423j\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 40\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BD38XntlGrug%3D%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 40\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11841244\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 40\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Protein%20N-glycosylation%20along%20the%20secretory%20pathway%3A%20relationship%20to%20organelle%20topography%20and%20function%2C%20protein%20quality%20control%2C%20and%20cell%20interactions&amp;journal=Chem.%20Rev.&amp;doi=10.1021%2Fcr000423j&amp;volume=102&amp;pages=285-303&amp;publication_year=2002&amp;author=Roth%2CJ\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR41\"> Bard, F. &amp; Chia, J. Cracking the glycome encoder: signaling, trafficking, and glycosylation. <i> Trends Cell Biol. </i> <b> 26 </b> , 379–388 (2016). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 41\" href=\"https://doi.org/10.1016%2Fj.tcb.2015.12.004\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 41\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC28XjslKrtA%3D%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 41\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26832820\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 41\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Cracking%20the%20glycome%20encoder%3A%20signaling%2C%20trafficking%2C%20and%20glycosylation&amp;journal=Trends%20Cell%20Biol.&amp;doi=10.1016%2Fj.tcb.2015.12.004&amp;volume=26&amp;pages=379-388&amp;publication_year=2016&amp;author=Bard%2CF&amp;author=Chia%2CJ\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR42\"> Schjoldager, K. T., Narimatsu, Y., Joshi, H. J. &amp; Clausen, H. Global view of human protein glycosylation pathways and functions. <i> Nat. Rev. Mol. Cell Biol. </i> <b> 21 </b> , 729–749 (2020). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 42\" href=\"https://doi.org/10.1038%2Fs41580-020-00294-x\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 42\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BB3cXitFGmu73N\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 42\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33087899\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 42\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Global%20view%20of%20human%20protein%20glycosylation%20pathways%20and%20functions&amp;journal=Nat.%20Rev.%20Mol.%20Cell%20Biol.&amp;doi=10.1038%2Fs41580-020-00294-x&amp;volume=21&amp;pages=729-749&amp;publication_year=2020&amp;author=Schjoldager%2CKT&amp;author=Narimatsu%2CY&amp;author=Joshi%2CHJ&amp;author=Clausen%2CH\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR43\"> Subramanian, A. et al. Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. <i> Proc. Natl Acad. Sci. USA </i> <b> 102 </b> , 15545–15550 (2005). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 43\" href=\"https://doi.org/10.1073%2Fpnas.0506580102\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"ADS reference 43\" href=\"http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2005PNAS..10215545S\" rel=\"nofollow noopener\"> ADS </a> <a aria-label=\"CAS reference 43\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BD2MXht1ShtrnO\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 43\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16199517\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"PubMed Central reference 43\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1239896\" rel=\"nofollow noopener\"> PubMed Central </a> <a aria-label=\"Google Scholar reference 43\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Gene%20set%20enrichment%20analysis%3A%20a%20knowledge-based%20approach%20for%20interpreting%20genome-wide%20expression%20profiles&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.0506580102&amp;volume=102&amp;pages=15545-15550&amp;publication_year=2005&amp;author=Subramanian%2CA\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR44\"> Mootha, V. K. et al. PGC-1alpha-responsive genes involved in oxidative phosphorylation are coordinately downregulated in human diabetes. <i> Nat. Genet. </i> <b> 34 </b> , 267–273 (2003). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 44\" href=\"https://doi.org/10.1038%2Fng1180\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 44\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BD3sXkvFSrs7o%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 44\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12808457\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 44\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=PGC-1alpha-responsive%20genes%20involved%20in%20oxidative%20phosphorylation%20are%20coordinately%20downregulated%20in%20human%20diabetes&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fng1180&amp;volume=34&amp;pages=267-273&amp;publication_year=2003&amp;author=Mootha%2CVK\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR45\"> Okayama, H. et al. Identification of genes upregulated in ALK-positive and EGFR/KRAS/ALK-negative lung adenocarcinomas. <i> Cancer Res. </i> <b> 72 </b> , 100–111 (2012). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 45\" href=\"https://doi.org/10.1158%2F0008-5472.CAN-11-1403\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 45\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC38XhvVCmtg%3D%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 45\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22080568\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 45\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Identification%20of%20genes%20upregulated%20in%20ALK-positive%20and%20EGFR%2FKRAS%2FALK-negative%20lung%20adenocarcinomas&amp;journal=Cancer%20Res.&amp;doi=10.1158%2F0008-5472.CAN-11-1403&amp;volume=72&amp;pages=100-111&amp;publication_year=2012&amp;author=Okayama%2CH\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR46\"> Lopez-Sambrooks, C. et al. Oligosaccharyltransferase inhibition induces senescence in RTK-driven tumor cells. <i> Nat. Chem. Biol. </i> <b> 12 </b> , 1023–1030 (2016). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 46\" href=\"https://doi.org/10.1038%2Fnchembio.2194\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 46\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC28XhsFOmsLvE\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 46\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27694802\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"PubMed Central reference 46\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5393272\" rel=\"nofollow noopener\"> PubMed Central </a> <a aria-label=\"Google Scholar reference 46\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Oligosaccharyltransferase%20inhibition%20induces%20senescence%20in%20RTK-driven%20tumor%20cells&amp;journal=Nat.%20Chem.%20Biol.&amp;doi=10.1038%2Fnchembio.2194&amp;volume=12&amp;pages=1023-1030&amp;publication_year=2016&amp;author=Lopez-Sambrooks%2CC\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR47\"> Blom, N., Sicheritz-Ponten, T., Gupta, R., Gammeltoft, S. &amp; Brunak, S. Prediction of post-translational glycosylation and phosphorylation of proteins from the amino acid sequence. <i> Proteomics </i> <b> 4 </b> , 1633–1649 (2004). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 47\" href=\"https://doi.org/10.1002%2Fpmic.200300771\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 47\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BD2cXkvFGitLg%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 47\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15174133\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 47\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Prediction%20of%20post-translational%20glycosylation%20and%20phosphorylation%20of%20proteins%20from%20the%20amino%20acid%20sequence&amp;journal=Proteomics&amp;doi=10.1002%2Fpmic.200300771&amp;volume=4&amp;pages=1633-1649&amp;publication_year=2004&amp;author=Blom%2CN&amp;author=Sicheritz-Ponten%2CT&amp;author=Gupta%2CR&amp;author=Gammeltoft%2CS&amp;author=Brunak%2CS\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR48\"> Yamaguchi, K. et al. Effects of site-directed removal of N-glycosylation sites in human erythropoietin on its production and biological properties. <i> J. Biol. Chem </i> . <b> 266 </b> , 20434–20439 (1991). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 48\" href=\"https://doi.org/10.1016%2FS0021-9258%2818%2954942-5\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 48\" href=\"/articles/cas-redirect/1:CAS:528:DyaK3MXlvFykt70%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 48\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=1657925\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 48\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Effects%20of%20site-directed%20removal%20of%20N-glycosylation%20sites%20in%20human%20erythropoietin%20on%20its%20production%20and%20biological%20properties&amp;journal=J.%20Biol.%20Chem&amp;doi=10.1016%2FS0021-9258%2818%2954942-5&amp;volume=266&amp;pages=20434-20439&amp;publication_year=1991&amp;author=Yamaguchi%2CK\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR49\"> Bobby, R. et al. Functional implications of large backbone amplitude motions of the glycoprotein 130-binding epitope of interleukin-6. <i> FEBS J. </i> <b> 281 </b> , 2471–2483 (2014). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 49\" href=\"https://doi.org/10.1111%2Ffebs.12800\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 49\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC2cXotFemtL0%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 49\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24712547\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 49\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Functional%20implications%20of%20large%20backbone%20amplitude%20motions%20of%20the%20glycoprotein%20130-binding%20epitope%20of%20interleukin-6&amp;journal=FEBS%20J.&amp;doi=10.1111%2Ffebs.12800&amp;volume=281&amp;pages=2471-2483&amp;publication_year=2014&amp;author=Bobby%2CR\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR50\"> Xiong, H. et al. Roles of STAT3 and ZEB1 proteins in E-cadherin down-regulation and human colorectal cancer epithelial-mesenchymal transition. <i> J. Biol. Chem. </i> <b> 287 </b> , 5819–5832 (2012). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 50\" href=\"https://doi.org/10.1074%2Fjbc.M111.295964\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 50\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC38XisVyluro%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 50\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22205702\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 50\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Roles%20of%20STAT3%20and%20ZEB1%20proteins%20in%20E-cadherin%20down-regulation%20and%20human%20colorectal%20cancer%20epithelial-mesenchymal%20transition&amp;journal=J.%20Biol.%20Chem.&amp;doi=10.1074%2Fjbc.M111.295964&amp;volume=287&amp;pages=5819-5832&amp;publication_year=2012&amp;author=Xiong%2CH\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR51\"> Wei, W. et al. STAT3 signaling is activated preferentially in tumor-initiating cells in claudin-low models of human breast cancer. <i> Stem cells </i> <b> 32 </b> , 2571–2582 (2014). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 51\" href=\"https://doi.org/10.1002%2Fstem.1752\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 51\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC2cXhvFGjtLzE\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 51\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24891218\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 51\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=STAT3%20signaling%20is%20activated%20preferentially%20in%20tumor-initiating%20cells%20in%20claudin-low%20models%20of%20human%20breast%20cancer&amp;journal=Stem%20cells&amp;doi=10.1002%2Fstem.1752&amp;volume=32&amp;pages=2571-2582&amp;publication_year=2014&amp;author=Wei%2CW\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR52\"> Yao, C. et al. IGF/STAT3/NANOG/Slug signaling axis simultaneously controls epithelial-mesenchymal transition and stemness maintenance in colorectal cancer. <i> Stem Cells </i> <b> 34 </b> , 820–831 (2016). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 52\" href=\"https://doi.org/10.1002%2Fstem.2320\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"PubMed reference 52\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26840943\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 52\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=IGF%2FSTAT3%2FNANOG%2FSlug%20signaling%20axis%20simultaneously%20controls%20epithelial-mesenchymal%20transition%20and%20stemness%20maintenance%20in%20colorectal%20cancer&amp;journal=Stem%20Cells&amp;doi=10.1002%2Fstem.2320&amp;volume=34&amp;pages=820-831&amp;publication_year=2016&amp;author=Yao%2CC\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR53\"> Na, Y. R., Lee, J. S., Lee, S. J. &amp; Seok, S. H. Interleukin-6-induced twist and N-cadherin enhance melanoma cell metastasis. <i> Melanoma Res. </i> <b> 23 </b> , 434–443 (2013). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 53\" href=\"https://doi.org/10.1097%2FCMR.0000000000000021\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 53\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC3sXhs12rtbbK\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 53\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24051540\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 53\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Interleukin-6-induced%20twist%20and%20N-cadherin%20enhance%20melanoma%20cell%20metastasis&amp;journal=Melanoma%20Res.&amp;doi=10.1097%2FCMR.0000000000000021&amp;volume=23&amp;pages=434-443&amp;publication_year=2013&amp;author=Na%2CYR&amp;author=Lee%2CJS&amp;author=Lee%2CSJ&amp;author=Seok%2CSH\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR54\"> Gallucci, R. M., Lee, E. G. &amp; Tomasek, J. J. IL-6 modulates alpha-smooth muscle actin expression in dermal fibroblasts from IL-6-deficient mice. <i> J. Invest. Dermatol </i> . <b> 126 </b> , 561–568 (2006). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 54\" href=\"https://doi.org/10.1038%2Fsj.jid.5700109\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 54\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BD28XhsVSrsLc%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 54\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16397521\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 54\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=IL-6%20modulates%20alpha-smooth%20muscle%20actin%20expression%20in%20dermal%20fibroblasts%20from%20IL-6-deficient%20mice&amp;journal=J.%20Invest.%20Dermatol&amp;doi=10.1038%2Fsj.jid.5700109&amp;volume=126&amp;pages=561-568&amp;publication_year=2006&amp;author=Gallucci%2CRM&amp;author=Lee%2CEG&amp;author=Tomasek%2CJJ\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR55\"> Romera-Hernandez, M. et al. Yap1-driven intestinal repair is controlled by group 3 innate lymphoid cells. <i> Cell Rep. </i> <b> 30 </b> , 37–45.e33 (2020). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 55\" href=\"https://doi.org/10.1016%2Fj.celrep.2019.11.115\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 55\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BB3cXotFert70%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 55\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31914395\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 55\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Yap1-driven%20intestinal%20repair%20is%20controlled%20by%20group%203%20innate%20lymphoid%20cells&amp;journal=Cell%20Rep.&amp;doi=10.1016%2Fj.celrep.2019.11.115&amp;volume=30&amp;pages=37-45.e33&amp;publication_year=2020&amp;author=Romera-Hernandez%2CM\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR56\"> Tsuji, T. et al. YAP1 mediates survival of ALK-rearranged lung cancer cells treated with alectinib via pro-apoptotic protein regulation. <i> Nat. Commun. </i> <b> 11 </b> , 74 (2020). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 56\" href=\"https://doi.org/10.1038%2Fs41467-019-13771-5\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"ADS reference 56\" href=\"http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2020NatCo..11...74T\" rel=\"nofollow noopener\"> ADS </a> <a aria-label=\"CAS reference 56\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BB3cXlslWisg%3D%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 56\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31900393\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"PubMed Central reference 56\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6941996\" rel=\"nofollow noopener\"> PubMed Central </a> <a aria-label=\"Google Scholar reference 56\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=YAP1%20mediates%20survival%20of%20ALK-rearranged%20lung%20cancer%20cells%20treated%20with%20alectinib%20via%20pro-apoptotic%20protein%20regulation&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-019-13771-5&amp;volume=11&amp;publication_year=2020&amp;author=Tsuji%2CT\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR57\"> Takahashi, A. et al. Ankyrin repeat domain 1 overexpression is associated with common resistance to afatinib and osimertinib in EGFR-mutant lung cancer. <i> Sci. Rep. </i> <b> 8 </b> , 14896 (2018). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 57\" href=\"https://doi.org/10.1038%2Fs41598-018-33190-8\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"ADS reference 57\" href=\"http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2018NatSR...814896T\" rel=\"nofollow noopener\"> ADS </a> <a aria-label=\"PubMed reference 57\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30291293\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"PubMed Central reference 57\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6173712\" rel=\"nofollow noopener\"> PubMed Central </a> <a aria-label=\"Google Scholar reference 57\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Ankyrin%20repeat%20domain%201%20overexpression%20is%20associated%20with%20common%20resistance%20to%20afatinib%20and%20osimertinib%20in%20EGFR-mutant%20lung%20cancer&amp;journal=Sci.%20Rep.&amp;doi=10.1038%2Fs41598-018-33190-8&amp;volume=8&amp;publication_year=2018&amp;author=Takahashi%2CA\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR58\"> Zheng, S. et al. SynergyFinder plus: toward better interpretation and annotation of drug combination screening datasets. <i> Genom. Proteom. Bioinform. </i> <b> 20 </b> , 587–596 (2022). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 58\" href=\"https://doi.org/10.1016%2Fj.gpb.2022.01.004\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"Google Scholar reference 58\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=SynergyFinder%20plus%3A%20toward%20better%20interpretation%20and%20annotation%20of%20drug%20combination%20screening%20datasets&amp;journal=Genom.%20Proteom.%20Bioinform.&amp;doi=10.1016%2Fj.gpb.2022.01.004&amp;volume=20&amp;pages=587-596&amp;publication_year=2022&amp;author=Zheng%2CS\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR59\"> Roper, N. et al. Clonal evolution and heterogeneity of osimertinib acquired resistance mechanisms in EGFR mutant lung cancer. <i> Cell Rep. Med. </i> <b> 1 </b> , 10007 (2020). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Google Scholar reference 59\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Clonal%20evolution%20and%20heterogeneity%20of%20osimertinib%20acquired%20resistance%20mechanisms%20in%20EGFR%20mutant%20lung%20cancer&amp;journal=Cell%20Rep.%20Med.&amp;volume=1&amp;publication_year=2020&amp;author=Roper%2CN\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR60\"> Barbie, D. A. et al. Systematic RNA interference reveals that oncogenic KRAS-driven cancers require TBK1. <i> Nature </i> <b> 462 </b> , 108–112 (2009). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 60\" href=\"https://doi.org/10.1038%2Fnature08460\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"ADS reference 60\" href=\"http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2009Natur.462..108B\" rel=\"nofollow noopener\"> ADS </a> <a aria-label=\"CAS reference 60\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BD1MXht12gtrvL\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 60\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19847166\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"PubMed Central reference 60\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2783335\" rel=\"nofollow noopener\"> PubMed Central </a> <a aria-label=\"Google Scholar reference 60\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Systematic%20RNA%20interference%20reveals%20that%20oncogenic%20KRAS-driven%20cancers%20require%20TBK1&amp;journal=Nature&amp;doi=10.1038%2Fnature08460&amp;volume=462&amp;pages=108-112&amp;publication_year=2009&amp;author=Barbie%2CDA\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR61\"> Heinz, M. C. et al. Liver colonization by colorectal cancer metastases requires YAP-controlled plasticity at the micrometastatic stage. <i> Cancer Res. </i> <b> 82 </b> , 1953–1968 (2022). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 61\" href=\"https://doi.org/10.1158%2F0008-5472.CAN-21-0933\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 61\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BB38XhsFehsbrE\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 61\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35570706\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"PubMed Central reference 61\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9381095\" rel=\"nofollow noopener\"> PubMed Central </a> <a aria-label=\"Google Scholar reference 61\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Liver%20colonization%20by%20colorectal%20cancer%20metastases%20requires%20YAP-controlled%20plasticity%20at%20the%20micrometastatic%20stage&amp;journal=Cancer%20Res.&amp;doi=10.1158%2F0008-5472.CAN-21-0933&amp;volume=82&amp;pages=1953-1968&amp;publication_year=2022&amp;author=Heinz%2CMC\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR62\"> Hamanaka, N. et al. YES1 is a targetable oncogene in cancers harboring YES1 gene amplification. <i> Cancer Res. </i> <b> 79 </b> , 5734–5745 (2019). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 62\" href=\"https://doi.org/10.1158%2F0008-5472.CAN-18-3376\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 62\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BB3cXjslGjs70%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 62\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31391186\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 62\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=YES1%20is%20a%20targetable%20oncogene%20in%20cancers%20harboring%20YES1%20gene%20amplification&amp;journal=Cancer%20Res.&amp;doi=10.1158%2F0008-5472.CAN-18-3376&amp;volume=79&amp;pages=5734-5745&amp;publication_year=2019&amp;author=Hamanaka%2CN\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR63\"> Lopez Sambrooks, C. et al. Oligosaccharyltransferase inhibition overcomes therapeutic resistance to EGFR tyrosine kinase inhibitors. <i> Cancer Res. </i> <b> 78 </b> , 5094–5106 (2018). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 63\" href=\"https://doi.org/10.1158%2F0008-5472.CAN-18-0505\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"PubMed reference 63\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30026325\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 63\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Oligosaccharyltransferase%20inhibition%20overcomes%20therapeutic%20resistance%20to%20EGFR%20tyrosine%20kinase%20inhibitors&amp;journal=Cancer%20Res.&amp;doi=10.1158%2F0008-5472.CAN-18-0505&amp;volume=78&amp;pages=5094-5106&amp;publication_year=2018&amp;author=Lopez%20Sambrooks%2CC\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR64\"> Chan, L. C. et al. IL-6/JAK1 pathway drives PD-L1 Y112 phosphorylation to promote cancer immune evasion. <i> J. Clin. Invest. </i> <b> 129 </b> , 3324–3338 (2019). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 64\" href=\"https://doi.org/10.1172%2FJCI126022\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"PubMed reference 64\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31305264\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"PubMed Central reference 64\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6668668\" rel=\"nofollow noopener\"> PubMed Central </a> <a aria-label=\"Google Scholar reference 64\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=IL-6%2FJAK1%20pathway%20drives%20PD-L1%20Y112%20phosphorylation%20to%20promote%20cancer%20immune%20evasion&amp;journal=J.%20Clin.%20Invest.&amp;doi=10.1172%2FJCI126022&amp;volume=129&amp;pages=3324-3338&amp;publication_year=2019&amp;author=Chan%2CLC\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR65\"> Miller, M. R. et al. Downregulation of dystroglycan glycosyltransferases LARGE2 and ISPD associate with increased mortality in clear cell renal cell carcinoma. <i> Mol. Cancer </i> <b> 14 </b> , 141 (2015). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 65\" href=\"https://link.springer.com/doi/10.1186/s12943-015-0416-z\" rel=\"noopener\"> Article </a> <a aria-label=\"PubMed reference 65\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26220087\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"PubMed Central reference 65\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4518861\" rel=\"nofollow noopener\"> PubMed Central </a> <a aria-label=\"Google Scholar reference 65\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Downregulation%20of%20dystroglycan%20glycosyltransferases%20LARGE2%20and%20ISPD%20associate%20with%20increased%20mortality%20in%20clear%20cell%20renal%20cell%20carcinoma&amp;journal=Mol.%20Cancer&amp;doi=10.1186%2Fs12943-015-0416-z&amp;volume=14&amp;publication_year=2015&amp;author=Miller%2CMR\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR66\"> Chugh, S., Meza, J., Sheinin, Y. M., Ponnusamy, M. P. &amp; Batra, S. K. Loss of N-acetylgalactosaminyltransferase 3 in poorly differentiated pancreatic cancer: augmented aggressiveness and aberrant ErbB family glycosylation. <i> Br. J. Cancer </i> <b> 114 </b> , 1376–1386 (2016). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 66\" href=\"https://doi.org/10.1038%2Fbjc.2016.116\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 66\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC28XotFOhurw%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 66\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27187683\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"PubMed Central reference 66\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4984453\" rel=\"nofollow noopener\"> PubMed Central </a> <a aria-label=\"Google Scholar reference 66\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Loss%20of%20N-acetylgalactosaminyltransferase%203%20in%20poorly%20differentiated%20pancreatic%20cancer%3A%20augmented%20aggressiveness%20and%20aberrant%20ErbB%20family%20glycosylation&amp;journal=Br.%20J.%20Cancer&amp;doi=10.1038%2Fbjc.2016.116&amp;volume=114&amp;pages=1376-1386&amp;publication_year=2016&amp;author=Chugh%2CS&amp;author=Meza%2CJ&amp;author=Sheinin%2CYM&amp;author=Ponnusamy%2CMP&amp;author=Batra%2CSK\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR67\"> Peng, C. et al. Regulation of the Hippo-YAP pathway by glucose sensor O-GlcNAcylation. <i> Mol. Cell </i> <b> 68 </b> , 591–604.e595 (2017). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 67\" href=\"https://doi.org/10.1016%2Fj.molcel.2017.10.010\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 67\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC2sXhslOju7rJ\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 67\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29100056\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 67\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Regulation%20of%20the%20Hippo-YAP%20pathway%20by%20glucose%20sensor%20O-GlcNAcylation&amp;journal=Mol.%20Cell&amp;doi=10.1016%2Fj.molcel.2017.10.010&amp;volume=68&amp;pages=591-604.e595&amp;publication_year=2017&amp;author=Peng%2CC\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR68\"> Hung, T. L. et al. Clinical evaluation of HER-2/neu protein in malignant pleural effusion-associated lung adenocarcinoma and as a tumor marker in pleural effusion diagnosis. <i> Clin. Cancer Res. </i> <b> 9 </b> , 2605–2612 (2003). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"CAS reference 68\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BD3sXlt1yrsrg%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 68\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12855637\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 68\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Clinical%20evaluation%20of%20HER-2%2Fneu%20protein%20in%20malignant%20pleural%20effusion-associated%20lung%20adenocarcinoma%20and%20as%20a%20tumor%20marker%20in%20pleural%20effusion%20diagnosis&amp;journal=Clin.%20Cancer%20Res.&amp;volume=9&amp;pages=2605-2612&amp;publication_year=2003&amp;author=Hung%2CTL\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR69\"> Chu, Y. W. et al. Selection of invasive and metastatic subpopulations from a human lung adenocarcinoma cell line. <i> Am. J. Respir. Cell Mol. Biol. </i> <b> 17 </b> , 353–360 (1997). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 69\" href=\"https://doi.org/10.1165%2Fajrcmb.17.3.2837\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 69\" href=\"/articles/cas-redirect/1:CAS:528:DyaK2sXmtVOrs7c%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 69\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9308922\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 69\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Selection%20of%20invasive%20and%20metastatic%20subpopulations%20from%20a%20human%20lung%20adenocarcinoma%20cell%20line&amp;journal=Am.%20J.%20Respir.%20Cell%20Mol.%20Biol.&amp;doi=10.1165%2Fajrcmb.17.3.2837&amp;volume=17&amp;pages=353-360&amp;publication_year=1997&amp;author=Chu%2CYW\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR70\"> Wisniewski, J. R., Zougman, A., Nagaraj, N. &amp; Mann, M. Universal sample preparation method for proteome analysis. <i> Nat. Methods </i> <b> 6 </b> , 359–362 (2009). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 70\" href=\"https://doi.org/10.1038%2Fnmeth.1322\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 70\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BD1MXks12ksb0%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 70\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19377485\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 70\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Universal%20sample%20preparation%20method%20for%20proteome%20analysis&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fnmeth.1322&amp;volume=6&amp;pages=359-362&amp;publication_year=2009&amp;author=Wisniewski%2CJR&amp;author=Zougman%2CA&amp;author=Nagaraj%2CN&amp;author=Mann%2CM\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR71\"> Bern, M., Kil, Y. J. &amp; Becker, C. Byonic: advanced peptide and protein identification software. <i> Curr. Protocol. Bioinform. </i> <b> 13 </b> , 13 20 11–13 20 14 (2012). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Google Scholar reference 71\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Byonic%3A%20advanced%20peptide%20and%20protein%20identification%20software&amp;journal=Curr.%20Protocol.%20Bioinform.&amp;volume=13&amp;pages=13%2020%2011-13%2020%2014&amp;publication_year=2012&amp;author=Bern%2CM&amp;author=Kil%2CYJ&amp;author=Becker%2CC\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> </ol> <p class=\"c-article-references__download u-hide-print\"> <a href=\"https://citation-needed.springer.com/v2/references/10.1038/s41467-024-51831-7?format=refman&amp;flavour=references\" rel=\"nofollow\"> Download references </a> </p> </div> </div> </div> </section> </div> <section> <div class=\"c-article-section\" id=\"Ack1-section\"> <h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"Ack1\"> Acknowledgements </h2> <div class=\"c-article-section__content\" id=\"Ack1-content\"> <p> We acknowledge the Human BioBank (Research Center of Clinical Medicine), National Cheng Kung University Hospital (NCKUH), the Bioinformatics Center (National Cheng Kung University, NCKU), and the National Center for High Performance Computing for bioinformatics consulting, the Proteomics Core Laboratory of NCKU and the Mass Spectrometry Facility of National Yang Ming Chiao Tung University for protein identification, the Bioimaging Core Facility of the National Core Facility for Biopharmaceuticals (Ministry of Science and Technology, Taiwan) for assistance in analysis of confocal microscopic images and FACS-like tissue cytometric images, the Laboratory Animal Center, College of Medicine at National Cheng Kung University (NCKU), Taiwan, accredited by AAALAC International and Taiwan Animal Consortium for the technical support in Xenogen IVISR Spectrum Noninvasive Quantitative Molecular Imaging System, and the Biostatistics Consulting Center (NCKUH) for statistical consulting. We thank Dr. Chun-Hui Lee (Department of Oncology, NCKUH) for assistance in collecting MPE specimens, Dr. Wei-Pang Chung (Department of Oncology, NCKUH) for assistance in analyzing synergy of cytotoxicity, Prof. Chuan-Fa Chang (Department of Medical Laboratory Science and Biotechnology, NCKU) for providing primer sequences against N-OSTs, and Mr. Chein-Hung Chen (Academia Sinica Glycoscience Core Facility and the GRC Mass Core Facility) for glycopeptide analysis. This work was supported by grants from the Ministry of Science and Technology, Taiwan (MOST 104-2314-B-006-046-MY3, MOST 106-2314-B-006-040-MY2, MOST 107-2314-B-006-059-MY3), the Center of Applied Nanomedicine of National Cheng Kung University from Featured Areas Research Center Program within the framework of the Higher Education Sprout Project by the Ministry of Education (MOE), Taiwan, the Yushan Fellow Program, Ministry of Education and NCKU, Taiwan, the Ministry of Health and Welfare (MOHW), Taiwan (MOHW112-TDU-B-221-124005), and Academia Sinica, Taiwan (AS-GC-110-04, AS-GC-110-MD-04). </p> </div> </div> </section> <section aria-labelledby=\"author-information\"> <div class=\"c-article-section\" id=\"author-information-section\"> <h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"author-information\"> Author information </h2> <div class=\"c-article-section__content\" id=\"author-information-content\"> <h3 class=\"c-article__sub-heading\" id=\"affiliations\"> Authors and Affiliations </h3> <ol class=\"c-article-author-affiliation__list\"> <li id=\"Aff1\"> <p class=\"c-article-author-affiliation__address\"> Department of Oncology, National Cheng Kung University Hospital, College of Medicine, National Cheng Kung University, Tainan, Taiwan </p> <p class=\"c-article-author-affiliation__authors-list\"> Chun-Hua Hung, Shang-Yin Wu &amp; Wu-Chou Su </p> </li> <li id=\"Aff2\"> <p class=\"c-article-author-affiliation__address\"> Center of Applied Nanomedicine, National Cheng Kung University, Tainan, Taiwan </p> <p class=\"c-article-author-affiliation__authors-list\"> Chun-Hua Hung, Hsuan-Heng Yeh &amp; Wu-Chou Su </p> </li> <li id=\"Aff3\"> <p class=\"c-article-author-affiliation__address\"> Institute of Basic Medical Sciences, College of Medicine, National Cheng Kung University, Tainan, Taiwan </p> <p class=\"c-article-author-affiliation__authors-list\"> Chun-Hua Hung &amp; Wu-Chou Su </p> </li> <li id=\"Aff4\"> <p class=\"c-article-author-affiliation__address\"> Department of Biomedical Engineering, College of Engineering, National Cheng Kung University, Tainan, Taiwan </p> <p class=\"c-article-author-affiliation__authors-list\"> Chun-Hua Hung </p> </li> <li id=\"Aff5\"> <p class=\"c-article-author-affiliation__address\"> Institute of Clinical Medicine, College of Medicine, National Cheng Kung University, Tainan, Taiwan </p> <p class=\"c-article-author-affiliation__authors-list\"> Shang-Yin Wu, Chien-Chung Lin &amp; Wu-Chou Su </p> </li> <li id=\"Aff6\"> <p class=\"c-article-author-affiliation__address\"> Institute of Biological Chemistry, Academia Sinica, Taipei, Taiwan </p> <p class=\"c-article-author-affiliation__authors-list\"> Cheng-I Daniel Yao &amp; Chun-Hung Lin </p> </li> <li id=\"Aff7\"> <p class=\"c-article-author-affiliation__address\"> Department of Internal Medicine, National Cheng Kung University Hospital, College of Medicine, National Cheng Kung University, Tainan, Taiwan </p> <p class=\"c-article-author-affiliation__authors-list\"> Chien-Chung Lin </p> </li> <li id=\"Aff8\"> <p class=\"c-article-author-affiliation__address\"> Department of Biochemistry and Molecular Biology, College of Medicine, National Cheng Kung University, Tainan, Taiwan </p> <p class=\"c-article-author-affiliation__authors-list\"> Chien-Chung Lin </p> </li> <li id=\"Aff9\"> <p class=\"c-article-author-affiliation__address\"> Department of Pathology, National Cheng Kung University Hospital, College of Medicine, National Cheng Kung University, Tainan, Taiwan </p> <p class=\"c-article-author-affiliation__authors-list\"> Chang-Yao Chu </p> </li> <li id=\"Aff10\"> <p class=\"c-article-author-affiliation__address\"> Institute of Molecular Medicine, College of Medicine, National Cheng Kung University, Tainan, Taiwan </p> <p class=\"c-article-author-affiliation__authors-list\"> Tzu-Yu Huang &amp; Wu-Chou Su </p> </li> <li id=\"Aff11\"> <p class=\"c-article-author-affiliation__address\"> Department of Pharmacology, College of Medicine, National Cheng Kung University, Tainan, Taiwan </p> <p class=\"c-article-author-affiliation__authors-list\"> Meng-Ru Shen </p> </li> <li id=\"Aff12\"> <p class=\"c-article-author-affiliation__address\"> Department of Obstetrics and Gynecology, National Cheng Kung University Hospital, College of Medicine, National Cheng Kung University, Tainan, Taiwan </p> <p class=\"c-article-author-affiliation__authors-list\"> Meng-Ru Shen </p> </li> <li id=\"Aff13\"> <p class=\"c-article-author-affiliation__address\"> Department of Chemistry and Institute of Biochemical Sciences, National Taiwan University, Taipei, Taiwan </p> <p class=\"c-article-author-affiliation__authors-list\"> Chun-Hung Lin </p> </li> </ol> <div class=\"u-js-hide u-hide-print\"> <span class=\"c-article__sub-heading\"> Authors </span> <ol class=\"c-article-authors-search u-list-reset\"> <li id=\"auth-Chun_Hua-Hung-Aff1-Aff2-Aff3-Aff4\"> <span class=\"c-article-authors-search__title u-h3 js-search-name\"> Chun-Hua Hung </span> <div class=\"c-article-authors-search__list\"> <div class=\"c-article-authors-search__item c-article-authors-search__list-item--left\"> <a class=\"c-article-button\" href=\"/search?author=Chun-Hua%20Hung\" rel=\"nofollow\"> View author publications </a> </div> <div class=\"c-article-authors-search__item c-article-authors-search__list-item--right\"> <p class=\"search-in-title-js c-article-authors-search__text\"> <span class=\"c-article-authors-search__links-text\"> Search author on: </span> <span class=\"c-article-identifiers\"> <a class=\"c-article-identifiers__item\" href=\"https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Chun-Hua%20Hung\" rel=\"nofollow\"> PubMed </a> <span class=\"u-hide\"> </span> <a class=\"c-article-identifiers__item\" href=\"https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Chun-Hua%20Hung%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en\" rel=\"nofollow\"> Google Scholar </a> </span> </p> </div> </div> </li> <li id=\"auth-Shang_Yin-Wu-Aff1-Aff5\"> <span class=\"c-article-authors-search__title u-h3 js-search-name\"> Shang-Yin Wu </span> <div class=\"c-article-authors-search__list\"> <div class=\"c-article-authors-search__item c-article-authors-search__list-item--left\"> <a class=\"c-article-button\" href=\"/search?author=Shang-Yin%20Wu\" rel=\"nofollow\"> View author publications </a> </div> <div class=\"c-article-authors-search__item c-article-authors-search__list-item--right\"> <p class=\"search-in-title-js c-article-authors-search__text\"> <span class=\"c-article-authors-search__links-text\"> Search author on: </span> <span class=\"c-article-identifiers\"> <a class=\"c-article-identifiers__item\" href=\"https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Shang-Yin%20Wu\" rel=\"nofollow\"> PubMed </a> <span class=\"u-hide\"> </span> <a class=\"c-article-identifiers__item\" href=\"https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Shang-Yin%20Wu%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en\" rel=\"nofollow\"> Google Scholar </a> </span> </p> </div> </div> </li> <li id=\"auth-Cheng_I_Daniel-Yao-Aff6\"> <span class=\"c-article-authors-search__title u-h3 js-search-name\"> Cheng-I Daniel Yao </span> <div class=\"c-article-authors-search__list\"> <div class=\"c-article-authors-search__item c-article-authors-search__list-item--left\"> <a class=\"c-article-button\" href=\"/search?author=Cheng-I%20Daniel%20Yao\" rel=\"nofollow\"> View author publications </a> </div> <div class=\"c-article-authors-search__item c-article-authors-search__list-item--right\"> <p class=\"search-in-title-js c-article-authors-search__text\"> <span class=\"c-article-authors-search__links-text\"> Search author on: </span> <span class=\"c-article-identifiers\"> <a class=\"c-article-identifiers__item\" href=\"https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Cheng-I%20Daniel%20Yao\" rel=\"nofollow\"> PubMed </a> <span class=\"u-hide\"> </span> <a class=\"c-article-identifiers__item\" href=\"https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Cheng-I%20Daniel%20Yao%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en\" rel=\"nofollow\"> Google Scholar </a> </span> </p> </div> </div> </li> <li id=\"auth-Hsuan_Heng-Yeh-Aff2\"> <span class=\"c-article-authors-search__title u-h3 js-search-name\"> Hsuan-Heng Yeh </span> <div class=\"c-article-authors-search__list\"> <div class=\"c-article-authors-search__item c-article-authors-search__list-item--left\"> <a class=\"c-article-button\" href=\"/search?author=Hsuan-Heng%20Yeh\" rel=\"nofollow\"> View author publications </a> </div> <div class=\"c-article-authors-search__item c-article-authors-search__list-item--right\"> <p class=\"search-in-title-js c-article-authors-search__text\"> <span class=\"c-article-authors-search__links-text\"> Search author on: </span> <span class=\"c-article-identifiers\"> <a class=\"c-article-identifiers__item\" href=\"https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Hsuan-Heng%20Yeh\" rel=\"nofollow\"> PubMed </a> <span class=\"u-hide\"> </span> <a class=\"c-article-identifiers__item\" href=\"https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Hsuan-Heng%20Yeh%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en\" rel=\"nofollow\"> Google Scholar </a> </span> </p> </div> </div> </li> <li id=\"auth-Chien_Chung-Lin-Aff5-Aff7-Aff8\"> <span class=\"c-article-authors-search__title u-h3 js-search-name\"> Chien-Chung Lin </span> <div class=\"c-article-authors-search__list\"> <div class=\"c-article-authors-search__item c-article-authors-search__list-item--left\"> <a class=\"c-article-button\" href=\"/search?author=Chien-Chung%20Lin\" rel=\"nofollow\"> View author publications </a> </div> <div class=\"c-article-authors-search__item c-article-authors-search__list-item--right\"> <p class=\"search-in-title-js c-article-authors-search__text\"> <span class=\"c-article-authors-search__links-text\"> Search author on: </span> <span class=\"c-article-identifiers\"> <a class=\"c-article-identifiers__item\" href=\"https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Chien-Chung%20Lin\" rel=\"nofollow\"> PubMed </a> <span class=\"u-hide\"> </span> <a class=\"c-article-identifiers__item\" href=\"https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Chien-Chung%20Lin%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en\" rel=\"nofollow\"> Google Scholar </a> </span> </p> </div> </div> </li> <li id=\"auth-Chang_Yao-Chu-Aff9\"> <span class=\"c-article-authors-search__title u-h3 js-search-name\"> Chang-Yao Chu </span> <div class=\"c-article-authors-search__list\"> <div class=\"c-article-authors-search__item c-article-authors-search__list-item--left\"> <a class=\"c-article-button\" href=\"/search?author=Chang-Yao%20Chu\" rel=\"nofollow\"> View author publications </a> </div> <div class=\"c-article-authors-search__item c-article-authors-search__list-item--right\"> <p class=\"search-in-title-js c-article-authors-search__text\"> <span class=\"c-article-authors-search__links-text\"> Search author on: </span> <span class=\"c-article-identifiers\"> <a class=\"c-article-identifiers__item\" href=\"https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Chang-Yao%20Chu\" rel=\"nofollow\"> PubMed </a> <span class=\"u-hide\"> </span> <a class=\"c-article-identifiers__item\" href=\"https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Chang-Yao%20Chu%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en\" rel=\"nofollow\"> Google Scholar </a> </span> </p> </div> </div> </li> <li id=\"auth-Tzu_Yu-Huang-Aff10\"> <span class=\"c-article-authors-search__title u-h3 js-search-name\"> Tzu-Yu Huang </span> <div class=\"c-article-authors-search__list\"> <div class=\"c-article-authors-search__item c-article-authors-search__list-item--left\"> <a class=\"c-article-button\" href=\"/search?author=Tzu-Yu%20Huang\" rel=\"nofollow\"> View author publications </a> </div> <div class=\"c-article-authors-search__item c-article-authors-search__list-item--right\"> <p class=\"search-in-title-js c-article-authors-search__text\"> <span class=\"c-article-authors-search__links-text\"> Search author on: </span> <span class=\"c-article-identifiers\"> <a class=\"c-article-identifiers__item\" href=\"https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Tzu-Yu%20Huang\" rel=\"nofollow\"> PubMed </a> <span class=\"u-hide\"> </span> <a class=\"c-article-identifiers__item\" href=\"https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Tzu-Yu%20Huang%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en\" rel=\"nofollow\"> Google Scholar </a> </span> </p> </div> </div> </li> <li id=\"auth-Meng_Ru-Shen-Aff11-Aff12\"> <span class=\"c-article-authors-search__title u-h3 js-search-name\"> Meng-Ru Shen </span> <div class=\"c-article-authors-search__list\"> <div class=\"c-article-authors-search__item c-article-authors-search__list-item--left\"> <a class=\"c-article-button\" href=\"/search?author=Meng-Ru%20Shen\" rel=\"nofollow\"> View author publications </a> </div> <div class=\"c-article-authors-search__item c-article-authors-search__list-item--right\"> <p class=\"search-in-title-js c-article-authors-search__text\"> <span class=\"c-article-authors-search__links-text\"> Search author on: </span> <span class=\"c-article-identifiers\"> <a class=\"c-article-identifiers__item\" href=\"https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Meng-Ru%20Shen\" rel=\"nofollow\"> PubMed </a> <span class=\"u-hide\"> </span> <a class=\"c-article-identifiers__item\" href=\"https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Meng-Ru%20Shen%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en\" rel=\"nofollow\"> Google Scholar </a> </span> </p> </div> </div> </li> <li id=\"auth-Chun_Hung-Lin-Aff6-Aff13\"> <span class=\"c-article-authors-search__title u-h3 js-search-name\"> Chun-Hung Lin </span> <div class=\"c-article-authors-search__list\"> <div class=\"c-article-authors-search__item c-article-authors-search__list-item--left\"> <a class=\"c-article-button\" href=\"/search?author=Chun-Hung%20Lin\" rel=\"nofollow\"> View author publications </a> </div> <div class=\"c-article-authors-search__item c-article-authors-search__list-item--right\"> <p class=\"search-in-title-js c-article-authors-search__text\"> <span class=\"c-article-authors-search__links-text\"> Search author on: </span> <span class=\"c-article-identifiers\"> <a class=\"c-article-identifiers__item\" href=\"https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Chun-Hung%20Lin\" rel=\"nofollow\"> PubMed </a> <span class=\"u-hide\"> </span> <a class=\"c-article-identifiers__item\" href=\"https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Chun-Hung%20Lin%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en\" rel=\"nofollow\"> Google Scholar </a> </span> </p> </div> </div> </li> <li id=\"auth-Wu_Chou-Su-Aff1-Aff2-Aff3-Aff5-Aff10\"> <span class=\"c-article-authors-search__title u-h3 js-search-name\"> Wu-Chou Su </span> <div class=\"c-article-authors-search__list\"> <div class=\"c-article-authors-search__item c-article-authors-search__list-item--left\"> <a class=\"c-article-button\" href=\"/search?author=Wu-Chou%20Su\" rel=\"nofollow\"> View author publications </a> </div> <div class=\"c-article-authors-search__item c-article-authors-search__list-item--right\"> <p class=\"search-in-title-js c-article-authors-search__text\"> <span class=\"c-article-authors-search__links-text\"> Search author on: </span> <span class=\"c-article-identifiers\"> <a class=\"c-article-identifiers__item\" href=\"https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Wu-Chou%20Su\" rel=\"nofollow\"> PubMed </a> <span class=\"u-hide\"> </span> <a class=\"c-article-identifiers__item\" href=\"https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Wu-Chou%20Su%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en\" rel=\"nofollow\"> Google Scholar </a> </span> </p> </div> </div> </li> </ol> </div> <h3 class=\"c-article__sub-heading\" id=\"contributions\"> Contributions </h3> <p> Chun-Hua Hung: Conceptualization, data curation, software, formal analysis, validation, investigation, visualization, methodology, writing-original draft. Shang-Yin Wu: Software, formal analysis, investigation. Cheng-I Daniel Yao: Formal analysis, methodology. Hsuan-Heng Yeh: Formal analysis, methodology. Chien-Chung Lin: Resources, formal analysis. Chang-Yao Chu: Formal analysis, methodology. Tzu-Yu Huang: Formal analysis, methodology. Meng-Ru Shen: Formal analysis, methodology, funding acquisition. Chun-Hung Lin: Formal analysis, methodology, funding acquisition. Wu-Chou Su*: Conceptualization, resources, supervision, funding acquisition, writing-original draft, project administration. </p> <h3 class=\"c-article__sub-heading\" id=\"corresponding-author\"> Corresponding author </h3> <p id=\"corresponding-author-list\"> Correspondence to <a href=\"mailto:sunnysu@mail.ncku.edu.tw\" id=\"corresp-c1\"> Wu-Chou Su </a> . </p> </div> </div> </section> <section> <div class=\"c-article-section\" id=\"ethics-section\"> <h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"ethics\"> Ethics declarations </h2> <div class=\"c-article-section__content\" id=\"ethics-content\"> <h3 class=\"c-article__sub-heading\" id=\"FPar4\"> Competing interests </h3> <p> The authors declare no competing interests. </p> </div> </div> </section> <section> <div class=\"c-article-section\" id=\"peer-review-section\"> <h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"peer-review\"> Peer review </h2> <div class=\"c-article-section__content\" id=\"peer-review-content\"> <h3 class=\"c-article__sub-heading\" id=\"FPar3\"> Peer review information </h3> <p> <i> Nature Communications </i> thanks Seiji Yano, and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. A peer review file is available. </p> </div> </div> </section> <section> <div class=\"c-article-section\" id=\"additional-information-section\"> <h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"additional-information\"> Additional information </h2> <div class=\"c-article-section__content\" id=\"additional-information-content\"> <p> <b> Publisher’s note </b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. </p> </div> </div> </section> <section> <div class=\"c-article-section\" id=\"Sec46-section\"> <h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"Sec46\"> Supplementary information </h2> <div class=\"c-article-section__content\" id=\"Sec46-content\"> <div> <div class=\"c-article-figshare-container\" id=\"figshareContainer\"> </div> <div class=\"c-article-supplementary__item\" id=\"MOESM1\"> <h3 class=\"c-article-supplementary__title u-h3\"> <a class=\"print-link\" href=\"https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-024-51831-7/MediaObjects/41467_2024_51831_MOESM1_ESM.pdf\"> Supplementary Information </a> </h3> </div> <div class=\"c-article-supplementary__item\" id=\"MOESM2\"> <h3 class=\"c-article-supplementary__title u-h3\"> <a class=\"print-link\" href=\"https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-024-51831-7/MediaObjects/41467_2024_51831_MOESM2_ESM.pdf\"> Peer Review File </a> </h3> </div> <div class=\"c-article-supplementary__item\" id=\"MOESM3\"> <h3 class=\"c-article-supplementary__title u-h3\"> <a class=\"print-link\" href=\"https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-024-51831-7/MediaObjects/41467_2024_51831_MOESM3_ESM.pdf\"> Reporting Summary </a> </h3> </div> </div> </div> </div> </section> <section> <div class=\"c-article-section\" id=\"Sec47-section\"> <h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"Sec47\"> Source data </h2> <div class=\"c-article-section__content\" id=\"Sec47-content\"> <div> <div class=\"c-article-supplementary__item\" id=\"MOESM4\"> <h3 class=\"c-article-supplementary__title u-h3\"> <a class=\"print-link\" href=\"https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-024-51831-7/MediaObjects/41467_2024_51831_MOESM4_ESM.xlsx\"> Source data </a> </h3> </div> </div> </div> </div> </section> <section> <div class=\"c-article-section\" id=\"rightslink-section\"> <h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"rightslink\"> Rights and permissions </h2> <div class=\"c-article-section__content\" id=\"rightslink-content\"> <p> <b> Open Access </b> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href=\"http://creativecommons.org/licenses/by-nc-nd/4.0/\" rel=\"license\"> http://creativecommons.org/licenses/by-nc-nd/4.0/ </a> . </p> <p class=\"c-article-rights\"> <a href=\"https://s100.copyright.com/AppDispatchServlet?title=Defective%20N-glycosylation%20of%20IL6%20induces%20metastasis%20and%20tyrosine%20kinase%20inhibitor%20resistance%20in%20lung%20cancer&amp;author=Chun-Hua%20Hung%20et%20al&amp;contentID=10.1038%2Fs41467-024-51831-7&amp;copyright=The%20Author%28s%29&amp;publication=2041-1723&amp;publicationDate=2024-09-09&amp;publisherName=SpringerNature&amp;orderBeanReset=true&amp;oa=CC%20BY-NC-ND\"> Reprints and permissions </a> </p> </div> </div> </section> <section aria-labelledby=\"article-info\"> <div class=\"c-article-section\" id=\"article-info-section\"> <h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"article-info\"> About this article </h2> <div class=\"c-article-section__content\" id=\"article-info-content\"> <div class=\"c-bibliographic-information\"> <div class=\"u-hide-print c-bibliographic-information__column c-bibliographic-information__column--border\"> <a href=\"https://crossmark.crossref.org/dialog/?doi=10.1038/s41467-024-51831-7\" rel=\"noopener\" target=\"_blank\"> </a> </div> <div class=\"c-bibliographic-information__column\"> <h3 class=\"c-article__sub-heading\" id=\"citeas\"> Cite this article </h3> <p class=\"c-bibliographic-information__citation\"> Hung, CH., Wu, SY., Yao, CI.D. <i> et al. </i> Defective N-glycosylation of IL6 induces metastasis and tyrosine kinase inhibitor resistance in lung cancer. <i> Nat Commun </i> <b> 15 </b> , 7885 (2024). https://doi.org/10.1038/s41467-024-51831-7 </p> <p class=\"c-bibliographic-information__download-citation u-hide-print\"> <a href=\"https://citation-needed.springer.com/v2/references/10.1038/s41467-024-51831-7?format=refman&amp;flavour=citation\" rel=\"nofollow\"> Download citation </a> </p> <ul class=\"c-bibliographic-information__list\"> <li class=\"c-bibliographic-information__list-item\"> <p> Received <span class=\"u-hide\"> : </span> <span class=\"c-bibliographic-information__value\"> <time datetime=\"2021-10-25\"> 25 October 2021 </time> </span> </p> </li> <li class=\"c-bibliographic-information__list-item\"> <p> Accepted <span class=\"u-hide\"> : </span> <span class=\"c-bibliographic-information__value\"> <time datetime=\"2024-08-19\"> 19 August 2024 </time> </span> </p> </li> <li class=\"c-bibliographic-information__list-item\"> <p> Published <span class=\"u-hide\"> : </span> <span class=\"c-bibliographic-information__value\"> <time datetime=\"2024-09-09\"> 09 September 2024 </time> </span> </p> </li> <li class=\"c-bibliographic-information__list-item c-bibliographic-information__list-item--full-width\"> <p> <abbr title=\"Digital Object Identifier\"> DOI </abbr> <span class=\"u-hide\"> : </span> <span class=\"c-bibliographic-information__value\"> https://doi.org/10.1038/s41467-024-51831-7 </span> </p> </li> </ul> <div> <div class=\"c-article-share-box u-display-block\"> <h3 class=\"c-article__sub-heading\"> Share this article </h3> <p class=\"c-article-share-box__description\"> Anyone you share the following link with will be able to read this content: </p> <div class=\"js-no-share-url-container u-display-none\" hidden=\"\"> <p class=\"js-c-article-share-box__no-sharelink-info c-article-share-box__no-sharelink-info\"> Sorry, a shareable link is not currently available for this article. </p> </div> <div class=\"js-share-url-container u-display-none\" hidden=\"\"> <p class=\"js-share-url c-article-share-box__only-read-input\" id=\"share-url\"> </p> </div> <p class=\"js-c-article-share-box__additional-info c-article-share-box__additional-info\"> Provided by the Springer Nature SharedIt content-sharing initiative </p> </div> </div> <div> <h3 class=\"c-article__sub-heading\"> Subjects </h3> <ul class=\"c-article-subject-list\"> <li class=\"c-article-subject-list__subject\"> <a href=\"/subjects/cancer-therapeutic-resistance\"> Cancer therapeutic resistance </a> </li> <li class=\"c-article-subject-list__subject\"> <a href=\"/subjects/glycobiology\"> Glycobiology </a> </li> <li class=\"c-article-subject-list__subject\"> <a href=\"/subjects/non-small-cell-lung-cancer\"> Non-small-cell lung cancer </a> </li> </ul> </div> </div> </div> </div> </div> </section> </div> <section> <div class=\"c-article-section js-article-section\" id=\"further-reading-section\"> <h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"further-reading\"> This article is cited by </h2> <div class=\"c-article-section__content js-collapsible-section\" id=\"further-reading-content\"> <ul class=\"c-article-further-reading__list\" id=\"further-reading-list\"> <li class=\"c-article-further-reading__item js-ref-item\"> <h3 class=\"c-article-further-reading__title\"> <a class=\"print-link\" href=\"https://doi.org/10.1186/s12967-025-06571-7\"> Nordihydroguaiaretic acid inhibits bladder cancer metastasis through suppression of α1,3-mannosyltransferase expression and LRFN4 N-glycosylation </a> </h3> <ul class=\"app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto\"> <li> Mulin Liu </li> <li> Jingyang Zhang </li> <li> Shijun Li </li> </ul> <p class=\"c-article-further-reading__journal-title\"> <i> Journal of Translational Medicine </i> (2025) </p> </li> <li class=\"c-article-further-reading__item js-ref-item\"> <h3 class=\"c-article-further-reading__title\"> <a class=\"print-link\" href=\"https://doi.org/10.1007/s00280-025-04772-x\"> Influence of interleukin-6 on the pharmacokinetics and pharmacodynamics of osimertinib in patients with non-small cell lung cancer </a> </h3> <ul class=\"app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto\"> <li> Hayato Yokota </li> <li> Kazuhiro Sato </li> <li> Masatomo Miura </li> </ul> <p class=\"c-article-further-reading__journal-title\"> <i> Cancer Chemotherapy and Pharmacology </i> (2025) </p> </li> <li class=\"c-article-further-reading__item js-ref-item\"> <h3 class=\"c-article-further-reading__title\"> <a class=\"print-link\" href=\"https://doi.org/10.1007/s12672-025-02830-7\"> Hotspots and frontiers of extracellular vesicles and lung cancer: a bibliometric and visualization analysis from 2002 to 2024 </a> </h3> <ul class=\"app-author-list app-author-list--compact u-sans-serif u-mb-4 u-mt-auto\"> <li> Zelin Yang </li> <li> Jie Tian </li> </ul> <p class=\"c-article-further-reading__journal-title\"> <i> Discover Oncology </i> (2025) </p> </li> <li class=\"c-article-further-reading__item js-ref-item\"> <h3 class=\"c-article-further-reading__title\"> <a class=\"print-link\" href=\"https://doi.org/10.1007/s12672-025-01782-2\"> Global research trends and emerging hotspots in nano-drug delivery systems for lung cancer: a comprehensive bibliometric analysis (1998–2024) </a> </h3> <ul class=\"app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto\"> <li> Cao Yu </li> <li> Chong-Qi Fan </li> <li> Huan-Wen Chen </li> </ul> <p class=\"c-article-further-reading__journal-title\"> <i> Discover Oncology </i> (2025) </p> </li> </ul> </div> </div> </section> </div> </article> </main> <aside aria-label=\"Article navigation\" class=\"c-article-extras u-hide-print\"> <div class=\"js-context-bar-sticky-point-desktop\"> <div class=\"c-pdf-download u-clear-both js-pdf-download\"> <a class=\"u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link\" download=\"\" href=\"/articles/s41467-024-51831-7.pdf\"> <span class=\"c-pdf-download__text\"> Download PDF </span> </a> </div> </div> <div class=\"c-reading-companion\"> <div class=\"c-reading-companion__sticky c-reading-companion__sticky--stuck\"> <ul class=\"c-reading-companion__tabs\" role=\"tablist\"> <li role=\"presentation\"> </li> <li role=\"presentation\"> </li> <li role=\"presentation\"> </li> </ul> <div aria-labelledby=\"tab-sections\" class=\"c-reading-companion__panel c-reading-companion__sections c-reading-companion__panel--active\" id=\"tabpanel-sections\" role=\"tabpanel\"> <div class=\"c-reading-companion__scroll-pane\"> <ul class=\"c-reading-companion__sections-list\"> <li class=\"c-reading-companion__section-item\" id=\"rc-sec-Abs1\"> <a href=\"#Abs1\"> Abstract </a> </li> <li class=\"c-reading-companion__section-item\" id=\"rc-sec-Sec1\"> <a href=\"#Sec1\"> Introduction </a> </li> <li class=\"c-reading-companion__section-item\" id=\"rc-sec-Sec2\"> <a href=\"#Sec2\"> Results </a> </li> <li class=\"c-reading-companion__section-item\" id=\"rc-sec-Sec10\"> <a href=\"#Sec10\"> Discussion </a> </li> <li class=\"c-reading-companion__section-item\" id=\"rc-sec-Sec11\"> <a href=\"#Sec11\"> Methods </a> </li> <li class=\"c-reading-companion__section-item\" id=\"rc-sec-data-availability\"> <a href=\"#data-availability\"> Data availability </a> </li> <li class=\"c-reading-companion__section-item\" id=\"rc-sec-Bib1\"> <a href=\"#Bib1\"> References </a> </li> <li class=\"c-reading-companion__section-item\" id=\"rc-sec-Ack1\"> <a href=\"#Ack1\"> Acknowledgements </a> </li> <li class=\"c-reading-companion__section-item\" id=\"rc-sec-author-information\"> <a href=\"#author-information\"> Author information </a> </li> <li class=\"c-reading-companion__section-item\" id=\"rc-sec-ethics\"> <a href=\"#ethics\"> Ethics declarations </a> </li> <li class=\"c-reading-companion__section-item\" id=\"rc-sec-peer-review\"> <a href=\"#peer-review\"> Peer review </a> </li> <li class=\"c-reading-companion__section-item\" id=\"rc-sec-additional-information\"> <a href=\"#additional-information\"> Additional information </a> </li> <li class=\"c-reading-companion__section-item\" id=\"rc-sec-Sec46\"> <a href=\"#Sec46\"> Supplementary information </a> </li> <li class=\"c-reading-companion__section-item\" id=\"rc-sec-Sec47\"> <a href=\"#Sec47\"> Source data </a> </li> <li class=\"c-reading-companion__section-item\" id=\"rc-sec-rightslink\"> <a href=\"#rightslink\"> Rights and permissions </a> </li> <li class=\"c-reading-companion__section-item\" id=\"rc-sec-article-info\"> <a href=\"#article-info\"> About this article </a> </li> <li class=\"c-reading-companion__section-item c-reading-companion__section-item--active\" id=\"rc-sec-further-reading\"> <a href=\"#further-reading\"> This article is cited by </a> </li> </ul> </div> <div class=\"u-lazy-ad-wrapper u-mt-16 u-show\"> <div class=\"c-ad c-ad--300x250\"> <div class=\"c-ad__inner\"> <p class=\"c-ad__label\"> Advertisement </p> <div class=\"div-gpt-ad advert medium-rectangle js-ad text-center hide-print grade-c-hide\" id=\"div-gpt-ad-right-2\"> <div id=\"google_ads_iframe_/270604982/nature/ncomms/articles_1__container__\"> </div> </div> </div> </div> </div> </div> <div aria-labelledby=\"tab-figures\" class=\"c-reading-companion__panel c-reading-companion__figures c-reading-companion__panel--full-width\" id=\"tabpanel-figures\" role=\"tabpanel\"> <div class=\"c-reading-companion__scroll-pane\"> <ul class=\"c-reading-companion__figures-list\"> <li class=\"c-reading-companion__figure-item\"> <figure> <figcaption> <b class=\"c-reading-companion__figure-title u-h4\" id=\"rc-Fig1\"> Fig. 1: IL6 Is Modified by N-Glycosylation in NSCLC. </b> </figcaption> <picture> <source/> </picture> <p class=\"c-reading-companion__figure-links\"> <a href=\"#Fig1\"> View in article </a> <a class=\"c-reading-companion__figure-full-link\" href=\"https://www.nature.com/articles/s41467-024-51831-7/figures/1\" rel=\"nofollow\"> Full size image </a> </p> </figure> </li> <li class=\"c-reading-companion__figure-item\"> <figure> <figcaption> <b class=\"c-reading-companion__figure-title u-h4\" id=\"rc-Fig2\"> Fig. 2: N-glycosylated IL6 promotes the tyrosine phosphorylation and nuclear retention of STAT3. </b> </figcaption> <picture> <source/> </picture> <p class=\"c-reading-companion__figure-links\"> <a href=\"#Fig2\"> View in article </a> <a class=\"c-reading-companion__figure-full-link\" href=\"https://www.nature.com/articles/s41467-024-51831-7/figures/2\" rel=\"nofollow\"> Full size image </a> </p> </figure> </li> <li class=\"c-reading-companion__figure-item\"> <figure> <figcaption> <b class=\"c-reading-companion__figure-title u-h4\" id=\"rc-Fig3\"> Fig. 3: deNG-IL6 Induces EMT, Migration, and Metastasis in NSCLC. </b> </figcaption> <picture> <source/> </picture> <p class=\"c-reading-companion__figure-links\"> <a href=\"#Fig3\"> View in article </a> <a class=\"c-reading-companion__figure-full-link\" href=\"https://www.nature.com/articles/s41467-024-51831-7/figures/3\" rel=\"nofollow\"> Full size image </a> </p> </figure> </li> <li class=\"c-reading-companion__figure-item\"> <figure> <figcaption> <b class=\"c-reading-companion__figure-title u-h4\" id=\"rc-Fig4\"> Fig. 4: deNG-IL6 Switches the GP130 Signaling Preference from the JAK-STAT3 Axis to the SRC-YAP Axis. </b> </figcaption> <picture> <source/> </picture> <p class=\"c-reading-companion__figure-links\"> <a href=\"#Fig4\"> View in article </a> <a class=\"c-reading-companion__figure-full-link\" href=\"https://www.nature.com/articles/s41467-024-51831-7/figures/4\" rel=\"nofollow\"> Full size image </a> </p> </figure> </li> <li class=\"c-reading-companion__figure-item\"> <figure> <figcaption> <b class=\"c-reading-companion__figure-title u-h4\" id=\"rc-Fig5\"> Fig. 5: SOX2 Contributes to deNG-IL6-Induced SRC-YAP Axis-Promoted Migration and Metastasis. </b> </figcaption> <picture> <source/> </picture> <p class=\"c-reading-companion__figure-links\"> <a href=\"#Fig5\"> View in article </a> <a class=\"c-reading-companion__figure-full-link\" href=\"https://www.nature.com/articles/s41467-024-51831-7/figures/5\" rel=\"nofollow\"> Full size image </a> </p> </figure> </li> <li class=\"c-reading-companion__figure-item\"> <figure> <figcaption> <b class=\"c-reading-companion__figure-title u-h4\" id=\"rc-Fig6\"> Fig. 6: Inhibition of osimertinib-induced deNG-IL6 restores TKI sensitivity in TKI-resistant NSCLC. </b> </figcaption> <picture> <source/> </picture> <p class=\"c-reading-companion__figure-links\"> <a href=\"#Fig6\"> View in article </a> <a class=\"c-reading-companion__figure-full-link\" href=\"https://www.nature.com/articles/s41467-024-51831-7/figures/6\" rel=\"nofollow\"> Full size image </a> </p> </figure> </li> <li class=\"c-reading-companion__figure-item\"> <figure> <figcaption> <b class=\"c-reading-companion__figure-title u-h4\" id=\"rc-Fig7\"> Fig. 7: Osimertinib-induced deNG-IL6 activates the SRC-YAP-SOX2 axis and migration in TKI-resistant NSCLC. </b> </figcaption> <picture> <source/> </picture> <p class=\"c-reading-companion__figure-links\"> <a href=\"#Fig7\"> View in article </a> <a class=\"c-reading-companion__figure-full-link\" href=\"https://www.nature.com/articles/s41467-024-51831-7/figures/7\" rel=\"nofollow\"> Full size image </a> </p> </figure> </li> <li class=\"c-reading-companion__figure-item\"> <figure> <figcaption> <b class=\"c-reading-companion__figure-title u-h4\" id=\"rc-Fig8\"> Fig. 8: EGFR TKI-resistant NSCLC patients show high deNG-IL6 expression and a shift signaling preference from JAK-STAT3 to the SRC-YAP axis. </b> </figcaption> <picture> <source/> </picture> <p class=\"c-reading-companion__figure-links\"> <a href=\"#Fig8\"> View in article </a> <a class=\"c-reading-companion__figure-full-link\" href=\"https://www.nature.com/articles/s41467-024-51831-7/figures/8\" rel=\"nofollow\"> Full size image </a> </p> </figure> </li> <li class=\"c-reading-companion__figure-item\"> <figure> <figcaption> <b class=\"c-reading-companion__figure-title u-h4\" id=\"rc-Fig9\"> Fig. 9: Long-Term EGFR TKI Treatment Induces deNG-IL6 to Drive Resistance. </b> </figcaption> <picture> <source/> </picture> <p class=\"c-reading-companion__figure-links\"> <a href=\"#Fig9\"> View in article </a> <a class=\"c-reading-companion__figure-full-link\" href=\"https://www.nature.com/articles/s41467-024-51831-7/figures/9\" rel=\"nofollow\"> Full size image </a> </p> </figure> </li> </ul> </div> </div> <div aria-labelledby=\"tab-references\" class=\"c-reading-companion__panel c-reading-companion__references c-reading-companion__panel--full-width\" id=\"tabpanel-references\" role=\"tabpanel\"> <div class=\"c-reading-companion__scroll-pane\"> <ol class=\"c-reading-companion__references-list c-reading-companion__references-list--numeric\"> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR1\"> Yu, H., Lee, H., Herrmann, A., Buettner, R. &amp; Jove, R. Revisiting STAT3 signalling in cancer: new and unexpected biological functions. <i> Nat. Rev. Cancer </i> <b> 14 </b> , 736–746 (2014). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1038%2Fnrc3818\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhvVSmtLnF\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25342631\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Revisiting%20STAT3%20signalling%20in%20cancer%3A%20new%20and%20unexpected%20biological%20functions&amp;journal=Nat.%20Rev.%20Cancer&amp;doi=10.1038%2Fnrc3818&amp;volume=14&amp;pages=736-746&amp;publication_year=2014&amp;author=Yu%2CH&amp;author=Lee%2CH&amp;author=Herrmann%2CA&amp;author=Buettner%2CR&amp;author=Jove%2CR\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR2\"> Jones, S. A. &amp; Jenkins, B. J. Recent insights into targeting the IL-6 cytokine family in inflammatory diseases and cancer. <i> Nat. Rev. Immunol. </i> <b> 18 </b> , 773–789 (2018). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1038%2Fs41577-018-0066-7\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhslymur7N\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30254251\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Recent%20insights%20into%20targeting%20the%20IL-6%20cytokine%20family%20in%20inflammatory%20diseases%20and%20cancer&amp;journal=Nat.%20Rev.%20Immunol.&amp;doi=10.1038%2Fs41577-018-0066-7&amp;volume=18&amp;pages=773-789&amp;publication_year=2018&amp;author=Jones%2CSA&amp;author=Jenkins%2CBJ\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR3\"> Rose-John, S. The soluble interleukin 6 receptor: advanced therapeutic options in inflammation. <i> Clin. Pharmacol. Therapeut. </i> <b> 102 </b> , 591–598 (2017). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1002%2Fcpt.782\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhsFWjt77I\"> CAS </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=The%20soluble%20interleukin%206%20receptor%3A%20advanced%20therapeutic%20options%20in%20inflammation&amp;journal=Clin.%20Pharmacol.%20Therapeut.&amp;doi=10.1002%2Fcpt.782&amp;volume=102&amp;pages=591-598&amp;publication_year=2017&amp;author=Rose-John%2CS\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR4\"> Trikha, M., Corringham, R., Klein, B. &amp; Rossi, J. F. Targeted anti-interleukin-6 monoclonal antibody therapy for cancer: a review of the rationale and clinical evidence. <i> Clin. Cancer Res. </i> <b> 9 </b> , 4653–4665 (2003). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3sXosVGjsrw%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=14581334\"> PubMed </a> <a href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2929399\"> PubMed Central </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Targeted%20anti-interleukin-6%20monoclonal%20antibody%20therapy%20for%20cancer%3A%20a%20review%20of%20the%20rationale%20and%20clinical%20evidence&amp;journal=Clin.%20Cancer%20Res.&amp;volume=9&amp;pages=4653-4665&amp;publication_year=2003&amp;author=Trikha%2CM&amp;author=Corringham%2CR&amp;author=Klein%2CB&amp;author=Rossi%2CJF\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR5\"> Hunter, C. A. &amp; Jones, S. A. IL-6 as a keystone cytokine in health and disease. <i> Nat. Immunol </i> . <b> 16 </b> , 448–457 (2015). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1038%2Fni.3153\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhtF2murbF\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25898198\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=IL-6%20as%20a%20keystone%20cytokine%20in%20health%20and%20disease&amp;journal=Nat.%20Immunol&amp;doi=10.1038%2Fni.3153&amp;volume=16&amp;pages=448-457&amp;publication_year=2015&amp;author=Hunter%2CCA&amp;author=Jones%2CSA\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR6\"> Garbers, C., Heink, S., Korn, T. &amp; Rose-John, S. Interleukin-6: designing specific therapeutics for a complex cytokine. <i> Nat. Rev. Drug Discov. </i> <b> 17 </b> , 395–412 (2018). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1038%2Fnrd.2018.45\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXovVagtbw%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29725131\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Interleukin-6%3A%20designing%20specific%20therapeutics%20for%20a%20complex%20cytokine&amp;journal=Nat.%20Rev.%20Drug%20Discov.&amp;doi=10.1038%2Fnrd.2018.45&amp;volume=17&amp;pages=395-412&amp;publication_year=2018&amp;author=Garbers%2CC&amp;author=Heink%2CS&amp;author=Korn%2CT&amp;author=Rose-John%2CS\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR7\"> Murakami, M., Kamimura, D. &amp; Hirano, T. Pleiotropy and specificity: insights from the interleukin 6 family of cytokines. <i> Immunity </i> <b> 50 </b> , 812–831 (2019). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1016%2Fj.immuni.2019.03.027\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXnsFGksLY%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30995501\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Pleiotropy%20and%20specificity%3A%20insights%20from%20the%20interleukin%206%20family%20of%20cytokines&amp;journal=Immunity&amp;doi=10.1016%2Fj.immuni.2019.03.027&amp;volume=50&amp;pages=812-831&amp;publication_year=2019&amp;author=Murakami%2CM&amp;author=Kamimura%2CD&amp;author=Hirano%2CT\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR8\"> Duan, Z. et al. Signal transducers and activators of transcription 3 pathway activation in drug-resistant ovarian cancer. <i> Clin. Cancer Res. </i> <b> 12 </b> , 5055–5063 (2006). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1158%2F1078-0432.CCR-06-0861\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28XovF2hu7s%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16951221\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Signal%20transducers%20and%20activators%20of%20transcription%203%20pathway%20activation%20in%20drug-resistant%20ovarian%20cancer&amp;journal=Clin.%20Cancer%20Res.&amp;doi=10.1158%2F1078-0432.CCR-06-0861&amp;volume=12&amp;pages=5055-5063&amp;publication_year=2006&amp;author=Duan%2CZ\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR9\"> Huang, W. L. et al. Signal transducer and activator of transcription 3 activation up-regulates interleukin-6 autocrine production: a biochemical and genetic study of established cancer cell lines and clinical isolated human cancer cells. <i> Mol. Cancer </i> <b> 9 </b> , 309 (2010). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://link.springer.com/doi/10.1186/1476-4598-9-309\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhsF2hs7rK\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21122157\"> PubMed </a> <a href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3027602\"> PubMed Central </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Signal%20transducer%20and%20activator%20of%20transcription%203%20activation%20up-regulates%20interleukin-6%20autocrine%20production%3A%20a%20biochemical%20and%20genetic%20study%20of%20established%20cancer%20cell%20lines%20and%20clinical%20isolated%20human%20cancer%20cells&amp;journal=Mol.%20Cancer&amp;doi=10.1186%2F1476-4598-9-309&amp;volume=9&amp;publication_year=2010&amp;author=Huang%2CWL\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR10\"> Lee, H. J. et al. Drug resistance via feedback activation of Stat3 in oncogene-addicted cancer cells. <i> Cancer Cell </i> <b> 26 </b> , 207–221 (2014). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1016%2Fj.ccr.2014.05.019\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXht1agsL7I\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25065853\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Drug%20resistance%20via%20feedback%20activation%20of%20Stat3%20in%20oncogene-addicted%20cancer%20cells&amp;journal=Cancer%20Cell&amp;doi=10.1016%2Fj.ccr.2014.05.019&amp;volume=26&amp;pages=207-221&amp;publication_year=2014&amp;author=Lee%2CHJ\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR11\"> Rokavec, M. et al. IL-6R/STAT3/miR-34a feedback loop promotes EMT-mediated colorectal cancer invasion and metastasis. <i> J. Clin. Invest. </i> <b> 124 </b> , 1853–1867 (2014). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1172%2FJCI73531\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXmtVGgur4%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24642471\"> PubMed </a> <a href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3973098\"> PubMed Central </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=IL-6R%2FSTAT3%2FmiR-34a%20feedback%20loop%20promotes%20EMT-mediated%20colorectal%20cancer%20invasion%20and%20metastasis&amp;journal=J.%20Clin.%20Invest.&amp;doi=10.1172%2FJCI73531&amp;volume=124&amp;pages=1853-1867&amp;publication_year=2014&amp;author=Rokavec%2CM\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR12\"> Taniguchi, K. et al. A gp130-Src-YAP module links inflammation to epithelial regeneration. <i> Nature </i> <b> 519 </b> , 57–62 (2015). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1038%2Fnature14228\"> Article </a> <a href=\"http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2015Natur.519...57T\"> ADS </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXjvVGjsrk%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25731159\"> PubMed </a> <a href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4447318\"> PubMed Central </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=A%20gp130-Src-YAP%20module%20links%20inflammation%20to%20epithelial%20regeneration&amp;journal=Nature&amp;doi=10.1038%2Fnature14228&amp;volume=519&amp;pages=57-62&amp;publication_year=2015&amp;author=Taniguchi%2CK\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR13\"> Quintanal-Villalonga, A. et al. Lineage plasticity in cancer: a shared pathway of therapeutic resistance. <i> Nat. Rev. Clin. Oncol. </i> <b> 17 </b> , 360–371 (2020). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1038%2Fs41571-020-0340-z\"> Article </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32152485\"> PubMed </a> <a href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7397755\"> PubMed Central </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Lineage%20plasticity%20in%20cancer%3A%20a%20shared%20pathway%20of%20therapeutic%20resistance&amp;journal=Nat.%20Rev.%20Clin.%20Oncol.&amp;doi=10.1038%2Fs41571-020-0340-z&amp;volume=17&amp;pages=360-371&amp;publication_year=2020&amp;author=Quintanal-Villalonga%2CA\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR14\"> Paez, J. G. et al. EGFR mutations in lung cancer: correlation with clinical response to gefitinib therapy. <i> Science </i> <b> 304 </b> , 1497–1500 (2004). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1126%2Fscience.1099314\"> Article </a> <a href=\"http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2004Sci...304.1497P\"> ADS </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXksVGmsbs%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15118125\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=EGFR%20mutations%20in%20lung%20cancer%3A%20correlation%20with%20clinical%20response%20to%20gefitinib%20therapy&amp;journal=Science&amp;doi=10.1126%2Fscience.1099314&amp;volume=304&amp;pages=1497-1500&amp;publication_year=2004&amp;author=Paez%2CJG\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR15\"> Ramalingam, S. S. et al. Overall survival with osimertinib in untreated, EGFR-mutated advanced NSCLC. <i> N. Engl. J. Med. </i> <b> 382 </b> , 41–50 (2020). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1056%2FNEJMoa1913662\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXnsFGgtL4%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31751012\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Overall%20survival%20with%20osimertinib%20in%20untreated%2C%20EGFR-mutated%20advanced%20NSCLC&amp;journal=N.%20Engl.%20J.%20Med.&amp;doi=10.1056%2FNEJMoa1913662&amp;volume=382&amp;pages=41-50&amp;publication_year=2020&amp;author=Ramalingam%2CSS\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR16\"> Oxnard, G. R. et al. Assessment of resistance mechanisms and clinical implications in patients with EGFR T790M-positive lung cancer and acquired resistance to osimertinib. <i> JAMA Oncol. </i> <b> 4 </b> , 1527–1534 (2018). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1001%2Fjamaoncol.2018.2969\"> Article </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30073261\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Assessment%20of%20resistance%20mechanisms%20and%20clinical%20implications%20in%20patients%20with%20EGFR%20T790M-positive%20lung%20cancer%20and%20acquired%20resistance%20to%20osimertinib&amp;journal=JAMA%20Oncol.&amp;doi=10.1001%2Fjamaoncol.2018.2969&amp;volume=4&amp;pages=1527-1534&amp;publication_year=2018&amp;author=Oxnard%2CGR\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR17\"> Yang, Z. et al. Investigating novel resistance mechanisms to third-generation EGFR tyrosine kinase inhibitor osimertinib in non-small cell lung cancer patients. <i> Clin. Cancer Res. </i> <b> 24 </b> , 3097–3107 (2018). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1158%2F1078-0432.CCR-17-2310\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXht1alurvK\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29506987\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Investigating%20novel%20resistance%20mechanisms%20to%20third-generation%20EGFR%20tyrosine%20kinase%20inhibitor%20osimertinib%20in%20non-small%20cell%20lung%20cancer%20patients&amp;journal=Clin.%20Cancer%20Res.&amp;doi=10.1158%2F1078-0432.CCR-17-2310&amp;volume=24&amp;pages=3097-3107&amp;publication_year=2018&amp;author=Yang%2CZ\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR18\"> Sequist, L. V. et al. Genotypic and histological evolution of lung cancers acquiring resistance to EGFR inhibitors. <i> Sci. Transl. Med. </i> <b> 3 </b> , 75ra26 (2011). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1126%2Fscitranslmed.3002003\"> Article </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21430269\"> PubMed </a> <a href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3132801\"> PubMed Central </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Genotypic%20and%20histological%20evolution%20of%20lung%20cancers%20acquiring%20resistance%20to%20EGFR%20inhibitors&amp;journal=Sci.%20Transl.%20Med.&amp;doi=10.1126%2Fscitranslmed.3002003&amp;volume=3&amp;publication_year=2011&amp;author=Sequist%2CLV\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR19\"> Lambert, A. W. &amp; Weinberg, R. A. Linking EMT programmes to normal and neoplastic epithelial stem cells. <i> Nat. Rev. Cancer </i> <b> 21 </b> , 325–338 (2021). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1038%2Fs41568-021-00332-6\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXjsVCquro%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33547455\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Linking%20EMT%20programmes%20to%20normal%20and%20neoplastic%20epithelial%20stem%20cells&amp;journal=Nat.%20Rev.%20Cancer&amp;doi=10.1038%2Fs41568-021-00332-6&amp;volume=21&amp;pages=325-338&amp;publication_year=2021&amp;author=Lambert%2CAW&amp;author=Weinberg%2CRA\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR20\"> Williams, E. D., Gao, D., Redfern, A. &amp; Thompson, E. W. Controversies around epithelial-mesenchymal plasticity in cancer metastasis. <i> Nat. Rev. Cancer </i> <b> 19 </b> , 716–732 (2019). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1038%2Fs41568-019-0213-x\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXitV2hsb7J\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31666716\"> PubMed </a> <a href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7055151\"> PubMed Central </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Controversies%20around%20epithelial-mesenchymal%20plasticity%20in%20cancer%20metastasis&amp;journal=Nat.%20Rev.%20Cancer&amp;doi=10.1038%2Fs41568-019-0213-x&amp;volume=19&amp;pages=716-732&amp;publication_year=2019&amp;author=Williams%2CED&amp;author=Gao%2CD&amp;author=Redfern%2CA&amp;author=Thompson%2CEW\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR21\"> Yao, Z. et al. TGF-beta IL-6 axis mediates selective and adaptive mechanisms of resistance to molecular targeted therapy in lung cancer. <i> Proc. Natl Acad. Sci. USA </i> <b> 107 </b> , 15535–15540 (2010). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1073%2Fpnas.1009472107\"> Article </a> <a href=\"http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2010PNAS..10715535Y\"> ADS </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtFaqu7%2FN\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20713723\"> PubMed </a> <a href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2932568\"> PubMed Central </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=TGF-beta%20IL-6%20axis%20mediates%20selective%20and%20adaptive%20mechanisms%20of%20resistance%20to%20molecular%20targeted%20therapy%20in%20lung%20cancer&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.1009472107&amp;volume=107&amp;pages=15535-15540&amp;publication_year=2010&amp;author=Yao%2CZ\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR22\"> Kim, S. M. et al. Activation of IL-6R/JAK1/STAT signaling induces de novo resistance to irreversible EGFR inhibitors in non-small cell lung cancer with T790M resistance mutation. <i> Mol. Cancer Ther. </i> <b> 11 </b> , 2254–2264 (2012). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1158%2F1535-7163.MCT-12-0311\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhsVyksrvI\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22891040\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Activation%20of%20IL-6R%2FJAK1%2FSTAT%20signaling%20induces%20de%20novo%20resistance%20to%20irreversible%20EGFR%20inhibitors%20in%20non-small%20cell%20lung%20cancer%20with%20T790M%20resistance%20mutation&amp;journal=Mol.%20Cancer%20Ther.&amp;doi=10.1158%2F1535-7163.MCT-12-0311&amp;volume=11&amp;pages=2254-2264&amp;publication_year=2012&amp;author=Kim%2CSM\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR23\"> Korkaya, H., Liu, S. &amp; Wicha, M. S. Regulation of cancer stem cells by cytokine networks: attacking cancer’s inflammatory roots. <i> Clin. Cancer Res. </i> <b> 17 </b> , 6125–6129 (2011). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1158%2F1078-0432.CCR-10-2743\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXht1Ggs7jP\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21685479\"> PubMed </a> <a href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3312242\"> PubMed Central </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Regulation%20of%20cancer%20stem%20cells%20by%20cytokine%20networks%3A%20attacking%20cancer%E2%80%99s%20inflammatory%20roots&amp;journal=Clin.%20Cancer%20Res.&amp;doi=10.1158%2F1078-0432.CCR-10-2743&amp;volume=17&amp;pages=6125-6129&amp;publication_year=2011&amp;author=Korkaya%2CH&amp;author=Liu%2CS&amp;author=Wicha%2CMS\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR24\"> Senescence-Associated Pro-inflammatory Cytokines, VernotJ. P. and tumor cell plasticity. <i> Front. Mol. Biosci. </i> <b> 7 </b> , 63 (2020). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2020FrMat...7...63S\"> ADS </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=and%20tumor%20cell%20plasticity&amp;journal=Front.%20Mol.%20Biosci.&amp;volume=7&amp;publication_year=2020&amp;author=Senescence-Associated%20Pro-inflammatory%20Cytokines%2CVernotJP\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR25\"> Kurppa, K. J. et al. Treatment-induced tumor dormancy through YAP-mediated transcriptional reprogramming of the apoptotic pathway. <i> Cancer Cell </i> <b> 37 </b> , 104–122.e112 (2020). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1016%2Fj.ccell.2019.12.006\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXotVent78%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31935369\"> PubMed </a> <a href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7146079\"> PubMed Central </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Treatment-induced%20tumor%20dormancy%20through%20YAP-mediated%20transcriptional%20reprogramming%20of%20the%20apoptotic%20pathway&amp;journal=Cancer%20Cell&amp;doi=10.1016%2Fj.ccell.2019.12.006&amp;volume=37&amp;pages=104-122.e112&amp;publication_year=2020&amp;author=Kurppa%2CKJ\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR26\"> Ferone, G. et al. SOX2 is the determining oncogenic switch in promoting lung squamous cell carcinoma from different cells of origin. <i> Cancer Cell </i> <b> 30 </b> , 519–532 (2016). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1016%2Fj.ccell.2016.09.001\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xhs1GqtLbP\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27728803\"> PubMed </a> <a href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5065004\"> PubMed Central </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=SOX2%20is%20the%20determining%20oncogenic%20switch%20in%20promoting%20lung%20squamous%20cell%20carcinoma%20from%20different%20cells%20of%20origin&amp;journal=Cancer%20Cell&amp;doi=10.1016%2Fj.ccell.2016.09.001&amp;volume=30&amp;pages=519-532&amp;publication_year=2016&amp;author=Ferone%2CG\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR27\"> Boumahdi, S. &amp; de Sauvage, F. J. The great escape: tumour cell plasticity in resistance to targeted therapy. <i> Nat. Rev. Drug Discov. </i> <b> 19 </b> , 39–56 (2020). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1038%2Fs41573-019-0044-1\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhvFCktL%2FM\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31601994\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=The%20great%20escape%3A%20tumour%20cell%20plasticity%20in%20resistance%20to%20targeted%20therapy&amp;journal=Nat.%20Rev.%20Drug%20Discov.&amp;doi=10.1038%2Fs41573-019-0044-1&amp;volume=19&amp;pages=39-56&amp;publication_year=2020&amp;author=Boumahdi%2CS&amp;author=Sauvage%2CFJ\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR28\"> Mosteiro, L. et al. Tissue damage and senescence provide critical signals for cellular reprogramming in vivo. <i> Science </i> <b> 354 </b> , 6315 (2016). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1126%2Fscience.aaf4445\"> Article </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Tissue%20damage%20and%20senescence%20provide%20critical%20signals%20for%20cellular%20reprogramming%20in%20vivo&amp;journal=Science&amp;doi=10.1126%2Fscience.aaf4445&amp;volume=354&amp;publication_year=2016&amp;author=Mosteiro%2CL\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR29\"> Bora-Singhal, N. et al. YAP1 regulates OCT4 activity and SOX2 expression to facilitate self-renewal and vascular mimicry of stem-like cells. <i> Stem cells </i> <b> 33 </b> , 1705–1718 (2015). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1002%2Fstem.1993\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhtleqsLvK\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25754111\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=YAP1%20regulates%20OCT4%20activity%20and%20SOX2%20expression%20to%20facilitate%20self-renewal%20and%20vascular%20mimicry%20of%20stem-like%20cells&amp;journal=Stem%20cells&amp;doi=10.1002%2Fstem.1993&amp;volume=33&amp;pages=1705-1718&amp;publication_year=2015&amp;author=Bora-Singhal%2CN\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR30\"> Poupart, P. et al. B cell growth modulating and differentiating activity of recombinant human 26-kd protein (BSF-2, HuIFN-beta 2, HPGF). <i> EMBO J </i> <b> 6 </b> , 1219–1224 (1987). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1002%2Fj.1460-2075.1987.tb02357.x\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaL2sXkt1Wktr0%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=3497029\"> PubMed </a> <a href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC553922\"> PubMed Central </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=B%20cell%20growth%20modulating%20and%20differentiating%20activity%20of%20recombinant%20human%2026-kd%20protein%20%28BSF-2%2C%20HuIFN-beta%202%2C%20HPGF%29&amp;journal=EMBO%20J&amp;doi=10.1002%2Fj.1460-2075.1987.tb02357.x&amp;volume=6&amp;pages=1219-1224&amp;publication_year=1987&amp;author=Poupart%2CP\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR31\"> May, L. T. et al. Phosphorylation of secreted forms of human beta 2-interferon/hepatocyte stimulating factor/interleukin-6. <i> Biochem. Biophys. Res. Commun. </i> <b> 152 </b> , 1144–1150 (1988). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1016%2FS0006-291X%2888%2980404-2\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaL1cXktVGjt70%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=3259872\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Phosphorylation%20of%20secreted%20forms%20of%20human%20beta%202-interferon%2Fhepatocyte%20stimulating%20factor%2Finterleukin-6&amp;journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;doi=10.1016%2FS0006-291X%2888%2980404-2&amp;volume=152&amp;pages=1144-1150&amp;publication_year=1988&amp;author=May%2CLT\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR32\"> Hirano, T. et al. Purification to homogeneity and characterization of human B-cell differentiation factor (BCDF or BSFp-2). <i> Proc. Natl Acad. Sci. USA </i> <b> 82 </b> , 5490–5494 (1985). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1073%2Fpnas.82.16.5490\"> Article </a> <a href=\"http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=1985PNAS...82.5490H\"> ADS </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaL2MXlt1Wnt7s%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=2410927\"> PubMed </a> <a href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC391148\"> PubMed Central </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Purification%20to%20homogeneity%20and%20characterization%20of%20human%20B-cell%20differentiation%20factor%20%28BCDF%20or%20BSFp-2%29&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.82.16.5490&amp;volume=82&amp;pages=5490-5494&amp;publication_year=1985&amp;author=Hirano%2CT\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR33\"> May, L. T., Santhanam, U. &amp; Sehgal, P. B. On the multimeric nature of natural human interleukin-6. <i> J. Biol. Chem. </i> <b> 266 </b> , 9950–9955 (1991). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1016%2FS0021-9258%2818%2992911-X\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK3MXkt1Onsbw%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=2033081\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=On%20the%20multimeric%20nature%20of%20natural%20human%20interleukin-6&amp;journal=J.%20Biol.%20Chem.&amp;doi=10.1016%2FS0021-9258%2818%2992911-X&amp;volume=266&amp;pages=9950-9955&amp;publication_year=1991&amp;author=May%2CLT&amp;author=Santhanam%2CU&amp;author=Sehgal%2CPB\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR34\"> Mereiter, S., Balmana, M., Campos, D., Gomes, J. &amp; Reis, C. A. Glycosylation in the era of cancer-targeted therapy: where are we heading? <i> Cancer Cell </i> <b> 36 </b> , 6–16 (2019). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1016%2Fj.ccell.2019.06.006\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtlersb%2FJ\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31287993\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Glycosylation%20in%20the%20era%20of%20cancer-targeted%20therapy%3A%20where%20are%20we%20heading%3F&amp;journal=Cancer%20Cell&amp;doi=10.1016%2Fj.ccell.2019.06.006&amp;volume=36&amp;pages=6-16&amp;publication_year=2019&amp;author=Mereiter%2CS&amp;author=Balmana%2CM&amp;author=Campos%2CD&amp;author=Gomes%2CJ&amp;author=Reis%2CCA\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR35\"> Parekh, R. B., Dwek, R. A., Rademacher, T. W., Opdenakker, G. &amp; Van Damme, J. Glycosylation of interleukin-6 purified from normal human blood mononuclear cells. <i> Eur. J. Biochem. </i> <b> 203 </b> , 135–141 (1992). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1111%2Fj.1432-1033.1992.tb19838.x\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK38Xht1KmtL8%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=1730219\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Glycosylation%20of%20interleukin-6%20purified%20from%20normal%20human%20blood%20mononuclear%20cells&amp;journal=Eur.%20J.%20Biochem.&amp;doi=10.1111%2Fj.1432-1033.1992.tb19838.x&amp;volume=203&amp;pages=135-141&amp;publication_year=1992&amp;author=Parekh%2CRB&amp;author=Dwek%2CRA&amp;author=Rademacher%2CTW&amp;author=Opdenakker%2CG&amp;author=Damme%2CJ\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR36\"> Dela Cruz, C. S. et al. N-linked glycosylation is required for optimal function of Kaposi’s sarcoma herpesvirus-encoded, but not cellular, interleukin 6. <i> J. Exp. Med. </i> <b> 199 </b> , 503–514 (2004). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1084%2Fjem.20031205\"> Article </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=N-linked%20glycosylation%20is%20required%20for%20optimal%20function%20of%20Kaposi%E2%80%99s%20sarcoma%20herpesvirus-encoded%2C%20but%20not%20cellular%2C%20interleukin%206&amp;journal=J.%20Exp.%20Med.&amp;doi=10.1084%2Fjem.20031205&amp;volume=199&amp;pages=503-514&amp;publication_year=2004&amp;author=Cruz%2CCS\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR37\"> Thomas, D., Rathinavel, A. K. &amp; Radhakrishnan, P. Altered glycosylation in cancer: a promising target for biomarkers and therapeutics. <i> Biochimica. et biophysica. Acta Rev. Cancer </i> <b> 1875 </b> , 188464 (2021). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1016%2Fj.bbcan.2020.188464\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitl2msr%2FP\"> CAS </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Altered%20glycosylation%20in%20cancer%3A%20a%20promising%20target%20for%20biomarkers%20and%20therapeutics&amp;journal=Biochimica.%20et%20biophysica.%20Acta%20Rev.%20Cancer&amp;doi=10.1016%2Fj.bbcan.2020.188464&amp;volume=1875&amp;publication_year=2021&amp;author=Thomas%2CD&amp;author=Rathinavel%2CAK&amp;author=Radhakrishnan%2CP\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR38\"> Pinho, S. S. &amp; Reis, C. A. Glycosylation in cancer: mechanisms and clinical implications. <i> Nat. Rev. Cancer </i> <b> 15 </b> , 540–555 (2015). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1038%2Fnrc3982\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhtlKkt7fK\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26289314\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Glycosylation%20in%20cancer%3A%20mechanisms%20and%20clinical%20implications&amp;journal=Nat.%20Rev.%20Cancer&amp;doi=10.1038%2Fnrc3982&amp;volume=15&amp;pages=540-555&amp;publication_year=2015&amp;author=Pinho%2CSS&amp;author=Reis%2CCA\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR39\"> Yeh, H. H., Lai, W. W., Chen, H. H., Liu, H. S. &amp; Su, W. C. Autocrine IL-6-induced Stat3 activation contributes to the pathogenesis of lung adenocarcinoma and malignant pleural effusion. <i> Oncogene </i> <b> 25 </b> , 4300–4309 (2006). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1038%2Fsj.onc.1209464\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28XntVWks7o%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16518408\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Autocrine%20IL-6-induced%20Stat3%20activation%20contributes%20to%20the%20pathogenesis%20of%20lung%20adenocarcinoma%20and%20malignant%20pleural%20effusion&amp;journal=Oncogene&amp;doi=10.1038%2Fsj.onc.1209464&amp;volume=25&amp;pages=4300-4309&amp;publication_year=2006&amp;author=Yeh%2CHH&amp;author=Lai%2CWW&amp;author=Chen%2CHH&amp;author=Liu%2CHS&amp;author=Su%2CWC\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR40\"> Roth, J. Protein N-glycosylation along the secretory pathway: relationship to organelle topography and function, protein quality control, and cell interactions. <i> Chem. Rev. </i> <b> 102 </b> , 285–303 (2002). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1021%2Fcr000423j\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD38XntlGrug%3D%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11841244\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Protein%20N-glycosylation%20along%20the%20secretory%20pathway%3A%20relationship%20to%20organelle%20topography%20and%20function%2C%20protein%20quality%20control%2C%20and%20cell%20interactions&amp;journal=Chem.%20Rev.&amp;doi=10.1021%2Fcr000423j&amp;volume=102&amp;pages=285-303&amp;publication_year=2002&amp;author=Roth%2CJ\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR41\"> Bard, F. &amp; Chia, J. Cracking the glycome encoder: signaling, trafficking, and glycosylation. <i> Trends Cell Biol. </i> <b> 26 </b> , 379–388 (2016). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1016%2Fj.tcb.2015.12.004\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XjslKrtA%3D%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26832820\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Cracking%20the%20glycome%20encoder%3A%20signaling%2C%20trafficking%2C%20and%20glycosylation&amp;journal=Trends%20Cell%20Biol.&amp;doi=10.1016%2Fj.tcb.2015.12.004&amp;volume=26&amp;pages=379-388&amp;publication_year=2016&amp;author=Bard%2CF&amp;author=Chia%2CJ\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR42\"> Schjoldager, K. T., Narimatsu, Y., Joshi, H. J. &amp; Clausen, H. Global view of human protein glycosylation pathways and functions. <i> Nat. Rev. Mol. Cell Biol. </i> <b> 21 </b> , 729–749 (2020). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1038%2Fs41580-020-00294-x\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitFGmu73N\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33087899\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Global%20view%20of%20human%20protein%20glycosylation%20pathways%20and%20functions&amp;journal=Nat.%20Rev.%20Mol.%20Cell%20Biol.&amp;doi=10.1038%2Fs41580-020-00294-x&amp;volume=21&amp;pages=729-749&amp;publication_year=2020&amp;author=Schjoldager%2CKT&amp;author=Narimatsu%2CY&amp;author=Joshi%2CHJ&amp;author=Clausen%2CH\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR43\"> Subramanian, A. et al. Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. <i> Proc. Natl Acad. Sci. USA </i> <b> 102 </b> , 15545–15550 (2005). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1073%2Fpnas.0506580102\"> Article </a> <a href=\"http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2005PNAS..10215545S\"> ADS </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXht1ShtrnO\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16199517\"> PubMed </a> <a href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1239896\"> PubMed Central </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Gene%20set%20enrichment%20analysis%3A%20a%20knowledge-based%20approach%20for%20interpreting%20genome-wide%20expression%20profiles&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.0506580102&amp;volume=102&amp;pages=15545-15550&amp;publication_year=2005&amp;author=Subramanian%2CA\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR44\"> Mootha, V. K. et al. PGC-1alpha-responsive genes involved in oxidative phosphorylation are coordinately downregulated in human diabetes. <i> Nat. Genet. </i> <b> 34 </b> , 267–273 (2003). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1038%2Fng1180\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3sXkvFSrs7o%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12808457\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=PGC-1alpha-responsive%20genes%20involved%20in%20oxidative%20phosphorylation%20are%20coordinately%20downregulated%20in%20human%20diabetes&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fng1180&amp;volume=34&amp;pages=267-273&amp;publication_year=2003&amp;author=Mootha%2CVK\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR45\"> Okayama, H. et al. Identification of genes upregulated in ALK-positive and EGFR/KRAS/ALK-negative lung adenocarcinomas. <i> Cancer Res. </i> <b> 72 </b> , 100–111 (2012). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1158%2F0008-5472.CAN-11-1403\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhvVCmtg%3D%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22080568\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Identification%20of%20genes%20upregulated%20in%20ALK-positive%20and%20EGFR%2FKRAS%2FALK-negative%20lung%20adenocarcinomas&amp;journal=Cancer%20Res.&amp;doi=10.1158%2F0008-5472.CAN-11-1403&amp;volume=72&amp;pages=100-111&amp;publication_year=2012&amp;author=Okayama%2CH\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR46\"> Lopez-Sambrooks, C. et al. Oligosaccharyltransferase inhibition induces senescence in RTK-driven tumor cells. <i> Nat. Chem. Biol. </i> <b> 12 </b> , 1023–1030 (2016). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1038%2Fnchembio.2194\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhsFOmsLvE\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27694802\"> PubMed </a> <a href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5393272\"> PubMed Central </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Oligosaccharyltransferase%20inhibition%20induces%20senescence%20in%20RTK-driven%20tumor%20cells&amp;journal=Nat.%20Chem.%20Biol.&amp;doi=10.1038%2Fnchembio.2194&amp;volume=12&amp;pages=1023-1030&amp;publication_year=2016&amp;author=Lopez-Sambrooks%2CC\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR47\"> Blom, N., Sicheritz-Ponten, T., Gupta, R., Gammeltoft, S. &amp; Brunak, S. Prediction of post-translational glycosylation and phosphorylation of proteins from the amino acid sequence. <i> Proteomics </i> <b> 4 </b> , 1633–1649 (2004). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1002%2Fpmic.200300771\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXkvFGitLg%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15174133\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Prediction%20of%20post-translational%20glycosylation%20and%20phosphorylation%20of%20proteins%20from%20the%20amino%20acid%20sequence&amp;journal=Proteomics&amp;doi=10.1002%2Fpmic.200300771&amp;volume=4&amp;pages=1633-1649&amp;publication_year=2004&amp;author=Blom%2CN&amp;author=Sicheritz-Ponten%2CT&amp;author=Gupta%2CR&amp;author=Gammeltoft%2CS&amp;author=Brunak%2CS\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR48\"> Yamaguchi, K. et al. Effects of site-directed removal of N-glycosylation sites in human erythropoietin on its production and biological properties. <i> J. Biol. Chem </i> . <b> 266 </b> , 20434–20439 (1991). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1016%2FS0021-9258%2818%2954942-5\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK3MXlvFykt70%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=1657925\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Effects%20of%20site-directed%20removal%20of%20N-glycosylation%20sites%20in%20human%20erythropoietin%20on%20its%20production%20and%20biological%20properties&amp;journal=J.%20Biol.%20Chem&amp;doi=10.1016%2FS0021-9258%2818%2954942-5&amp;volume=266&amp;pages=20434-20439&amp;publication_year=1991&amp;author=Yamaguchi%2CK\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR49\"> Bobby, R. et al. Functional implications of large backbone amplitude motions of the glycoprotein 130-binding epitope of interleukin-6. <i> FEBS J. </i> <b> 281 </b> , 2471–2483 (2014). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1111%2Ffebs.12800\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXotFemtL0%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24712547\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Functional%20implications%20of%20large%20backbone%20amplitude%20motions%20of%20the%20glycoprotein%20130-binding%20epitope%20of%20interleukin-6&amp;journal=FEBS%20J.&amp;doi=10.1111%2Ffebs.12800&amp;volume=281&amp;pages=2471-2483&amp;publication_year=2014&amp;author=Bobby%2CR\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR50\"> Xiong, H. et al. Roles of STAT3 and ZEB1 proteins in E-cadherin down-regulation and human colorectal cancer epithelial-mesenchymal transition. <i> J. Biol. Chem. </i> <b> 287 </b> , 5819–5832 (2012). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1074%2Fjbc.M111.295964\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XisVyluro%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22205702\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Roles%20of%20STAT3%20and%20ZEB1%20proteins%20in%20E-cadherin%20down-regulation%20and%20human%20colorectal%20cancer%20epithelial-mesenchymal%20transition&amp;journal=J.%20Biol.%20Chem.&amp;doi=10.1074%2Fjbc.M111.295964&amp;volume=287&amp;pages=5819-5832&amp;publication_year=2012&amp;author=Xiong%2CH\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR51\"> Wei, W. et al. STAT3 signaling is activated preferentially in tumor-initiating cells in claudin-low models of human breast cancer. <i> Stem cells </i> <b> 32 </b> , 2571–2582 (2014). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1002%2Fstem.1752\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhvFGjtLzE\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24891218\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=STAT3%20signaling%20is%20activated%20preferentially%20in%20tumor-initiating%20cells%20in%20claudin-low%20models%20of%20human%20breast%20cancer&amp;journal=Stem%20cells&amp;doi=10.1002%2Fstem.1752&amp;volume=32&amp;pages=2571-2582&amp;publication_year=2014&amp;author=Wei%2CW\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR52\"> Yao, C. et al. IGF/STAT3/NANOG/Slug signaling axis simultaneously controls epithelial-mesenchymal transition and stemness maintenance in colorectal cancer. <i> Stem Cells </i> <b> 34 </b> , 820–831 (2016). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1002%2Fstem.2320\"> Article </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26840943\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=IGF%2FSTAT3%2FNANOG%2FSlug%20signaling%20axis%20simultaneously%20controls%20epithelial-mesenchymal%20transition%20and%20stemness%20maintenance%20in%20colorectal%20cancer&amp;journal=Stem%20Cells&amp;doi=10.1002%2Fstem.2320&amp;volume=34&amp;pages=820-831&amp;publication_year=2016&amp;author=Yao%2CC\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR53\"> Na, Y. R., Lee, J. S., Lee, S. J. &amp; Seok, S. H. Interleukin-6-induced twist and N-cadherin enhance melanoma cell metastasis. <i> Melanoma Res. </i> <b> 23 </b> , 434–443 (2013). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1097%2FCMR.0000000000000021\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhs12rtbbK\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24051540\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Interleukin-6-induced%20twist%20and%20N-cadherin%20enhance%20melanoma%20cell%20metastasis&amp;journal=Melanoma%20Res.&amp;doi=10.1097%2FCMR.0000000000000021&amp;volume=23&amp;pages=434-443&amp;publication_year=2013&amp;author=Na%2CYR&amp;author=Lee%2CJS&amp;author=Lee%2CSJ&amp;author=Seok%2CSH\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR54\"> Gallucci, R. M., Lee, E. G. &amp; Tomasek, J. J. IL-6 modulates alpha-smooth muscle actin expression in dermal fibroblasts from IL-6-deficient mice. <i> J. Invest. Dermatol </i> . <b> 126 </b> , 561–568 (2006). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1038%2Fsj.jid.5700109\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28XhsVSrsLc%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16397521\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=IL-6%20modulates%20alpha-smooth%20muscle%20actin%20expression%20in%20dermal%20fibroblasts%20from%20IL-6-deficient%20mice&amp;journal=J.%20Invest.%20Dermatol&amp;doi=10.1038%2Fsj.jid.5700109&amp;volume=126&amp;pages=561-568&amp;publication_year=2006&amp;author=Gallucci%2CRM&amp;author=Lee%2CEG&amp;author=Tomasek%2CJJ\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR55\"> Romera-Hernandez, M. et al. Yap1-driven intestinal repair is controlled by group 3 innate lymphoid cells. <i> Cell Rep. </i> <b> 30 </b> , 37–45.e33 (2020). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1016%2Fj.celrep.2019.11.115\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXotFert70%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31914395\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Yap1-driven%20intestinal%20repair%20is%20controlled%20by%20group%203%20innate%20lymphoid%20cells&amp;journal=Cell%20Rep.&amp;doi=10.1016%2Fj.celrep.2019.11.115&amp;volume=30&amp;pages=37-45.e33&amp;publication_year=2020&amp;author=Romera-Hernandez%2CM\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR56\"> Tsuji, T. et al. YAP1 mediates survival of ALK-rearranged lung cancer cells treated with alectinib via pro-apoptotic protein regulation. <i> Nat. Commun. </i> <b> 11 </b> , 74 (2020). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1038%2Fs41467-019-13771-5\"> Article </a> <a href=\"http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2020NatCo..11...74T\"> ADS </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXlslWisg%3D%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31900393\"> PubMed </a> <a href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6941996\"> PubMed Central </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=YAP1%20mediates%20survival%20of%20ALK-rearranged%20lung%20cancer%20cells%20treated%20with%20alectinib%20via%20pro-apoptotic%20protein%20regulation&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-019-13771-5&amp;volume=11&amp;publication_year=2020&amp;author=Tsuji%2CT\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR57\"> Takahashi, A. et al. Ankyrin repeat domain 1 overexpression is associated with common resistance to afatinib and osimertinib in EGFR-mutant lung cancer. <i> Sci. Rep. </i> <b> 8 </b> , 14896 (2018). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1038%2Fs41598-018-33190-8\"> Article </a> <a href=\"http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2018NatSR...814896T\"> ADS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30291293\"> PubMed </a> <a href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6173712\"> PubMed Central </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Ankyrin%20repeat%20domain%201%20overexpression%20is%20associated%20with%20common%20resistance%20to%20afatinib%20and%20osimertinib%20in%20EGFR-mutant%20lung%20cancer&amp;journal=Sci.%20Rep.&amp;doi=10.1038%2Fs41598-018-33190-8&amp;volume=8&amp;publication_year=2018&amp;author=Takahashi%2CA\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR58\"> Zheng, S. et al. SynergyFinder plus: toward better interpretation and annotation of drug combination screening datasets. <i> Genom. Proteom. Bioinform. </i> <b> 20 </b> , 587–596 (2022). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1016%2Fj.gpb.2022.01.004\"> Article </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=SynergyFinder%20plus%3A%20toward%20better%20interpretation%20and%20annotation%20of%20drug%20combination%20screening%20datasets&amp;journal=Genom.%20Proteom.%20Bioinform.&amp;doi=10.1016%2Fj.gpb.2022.01.004&amp;volume=20&amp;pages=587-596&amp;publication_year=2022&amp;author=Zheng%2CS\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR59\"> Roper, N. et al. Clonal evolution and heterogeneity of osimertinib acquired resistance mechanisms in EGFR mutant lung cancer. <i> Cell Rep. Med. </i> <b> 1 </b> , 10007 (2020). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Clonal%20evolution%20and%20heterogeneity%20of%20osimertinib%20acquired%20resistance%20mechanisms%20in%20EGFR%20mutant%20lung%20cancer&amp;journal=Cell%20Rep.%20Med.&amp;volume=1&amp;publication_year=2020&amp;author=Roper%2CN\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR60\"> Barbie, D. A. et al. Systematic RNA interference reveals that oncogenic KRAS-driven cancers require TBK1. <i> Nature </i> <b> 462 </b> , 108–112 (2009). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1038%2Fnature08460\"> Article </a> <a href=\"http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2009Natur.462..108B\"> ADS </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXht12gtrvL\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19847166\"> PubMed </a> <a href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2783335\"> PubMed Central </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Systematic%20RNA%20interference%20reveals%20that%20oncogenic%20KRAS-driven%20cancers%20require%20TBK1&amp;journal=Nature&amp;doi=10.1038%2Fnature08460&amp;volume=462&amp;pages=108-112&amp;publication_year=2009&amp;author=Barbie%2CDA\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR61\"> Heinz, M. C. et al. Liver colonization by colorectal cancer metastases requires YAP-controlled plasticity at the micrometastatic stage. <i> Cancer Res. </i> <b> 82 </b> , 1953–1968 (2022). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1158%2F0008-5472.CAN-21-0933\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhsFehsbrE\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35570706\"> PubMed </a> <a href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9381095\"> PubMed Central </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Liver%20colonization%20by%20colorectal%20cancer%20metastases%20requires%20YAP-controlled%20plasticity%20at%20the%20micrometastatic%20stage&amp;journal=Cancer%20Res.&amp;doi=10.1158%2F0008-5472.CAN-21-0933&amp;volume=82&amp;pages=1953-1968&amp;publication_year=2022&amp;author=Heinz%2CMC\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR62\"> Hamanaka, N. et al. YES1 is a targetable oncogene in cancers harboring YES1 gene amplification. <i> Cancer Res. </i> <b> 79 </b> , 5734–5745 (2019). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1158%2F0008-5472.CAN-18-3376\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXjslGjs70%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31391186\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=YES1%20is%20a%20targetable%20oncogene%20in%20cancers%20harboring%20YES1%20gene%20amplification&amp;journal=Cancer%20Res.&amp;doi=10.1158%2F0008-5472.CAN-18-3376&amp;volume=79&amp;pages=5734-5745&amp;publication_year=2019&amp;author=Hamanaka%2CN\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR63\"> Lopez Sambrooks, C. et al. Oligosaccharyltransferase inhibition overcomes therapeutic resistance to EGFR tyrosine kinase inhibitors. <i> Cancer Res. </i> <b> 78 </b> , 5094–5106 (2018). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1158%2F0008-5472.CAN-18-0505\"> Article </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30026325\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Oligosaccharyltransferase%20inhibition%20overcomes%20therapeutic%20resistance%20to%20EGFR%20tyrosine%20kinase%20inhibitors&amp;journal=Cancer%20Res.&amp;doi=10.1158%2F0008-5472.CAN-18-0505&amp;volume=78&amp;pages=5094-5106&amp;publication_year=2018&amp;author=Lopez%20Sambrooks%2CC\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR64\"> Chan, L. C. et al. IL-6/JAK1 pathway drives PD-L1 Y112 phosphorylation to promote cancer immune evasion. <i> J. Clin. Invest. </i> <b> 129 </b> , 3324–3338 (2019). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1172%2FJCI126022\"> Article </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31305264\"> PubMed </a> <a href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6668668\"> PubMed Central </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=IL-6%2FJAK1%20pathway%20drives%20PD-L1%20Y112%20phosphorylation%20to%20promote%20cancer%20immune%20evasion&amp;journal=J.%20Clin.%20Invest.&amp;doi=10.1172%2FJCI126022&amp;volume=129&amp;pages=3324-3338&amp;publication_year=2019&amp;author=Chan%2CLC\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR65\"> Miller, M. R. et al. Downregulation of dystroglycan glycosyltransferases LARGE2 and ISPD associate with increased mortality in clear cell renal cell carcinoma. <i> Mol. Cancer </i> <b> 14 </b> , 141 (2015). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://link.springer.com/doi/10.1186/s12943-015-0416-z\"> Article </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26220087\"> PubMed </a> <a href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4518861\"> PubMed Central </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Downregulation%20of%20dystroglycan%20glycosyltransferases%20LARGE2%20and%20ISPD%20associate%20with%20increased%20mortality%20in%20clear%20cell%20renal%20cell%20carcinoma&amp;journal=Mol.%20Cancer&amp;doi=10.1186%2Fs12943-015-0416-z&amp;volume=14&amp;publication_year=2015&amp;author=Miller%2CMR\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR66\"> Chugh, S., Meza, J., Sheinin, Y. M., Ponnusamy, M. P. &amp; Batra, S. K. Loss of N-acetylgalactosaminyltransferase 3 in poorly differentiated pancreatic cancer: augmented aggressiveness and aberrant ErbB family glycosylation. <i> Br. J. Cancer </i> <b> 114 </b> , 1376–1386 (2016). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1038%2Fbjc.2016.116\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XotFOhurw%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27187683\"> PubMed </a> <a href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4984453\"> PubMed Central </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Loss%20of%20N-acetylgalactosaminyltransferase%203%20in%20poorly%20differentiated%20pancreatic%20cancer%3A%20augmented%20aggressiveness%20and%20aberrant%20ErbB%20family%20glycosylation&amp;journal=Br.%20J.%20Cancer&amp;doi=10.1038%2Fbjc.2016.116&amp;volume=114&amp;pages=1376-1386&amp;publication_year=2016&amp;author=Chugh%2CS&amp;author=Meza%2CJ&amp;author=Sheinin%2CYM&amp;author=Ponnusamy%2CMP&amp;author=Batra%2CSK\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR67\"> Peng, C. et al. Regulation of the Hippo-YAP pathway by glucose sensor O-GlcNAcylation. <i> Mol. Cell </i> <b> 68 </b> , 591–604.e595 (2017). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1016%2Fj.molcel.2017.10.010\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhslOju7rJ\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29100056\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Regulation%20of%20the%20Hippo-YAP%20pathway%20by%20glucose%20sensor%20O-GlcNAcylation&amp;journal=Mol.%20Cell&amp;doi=10.1016%2Fj.molcel.2017.10.010&amp;volume=68&amp;pages=591-604.e595&amp;publication_year=2017&amp;author=Peng%2CC\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR68\"> Hung, T. L. et al. Clinical evaluation of HER-2/neu protein in malignant pleural effusion-associated lung adenocarcinoma and as a tumor marker in pleural effusion diagnosis. <i> Clin. Cancer Res. </i> <b> 9 </b> , 2605–2612 (2003). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3sXlt1yrsrg%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12855637\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Clinical%20evaluation%20of%20HER-2%2Fneu%20protein%20in%20malignant%20pleural%20effusion-associated%20lung%20adenocarcinoma%20and%20as%20a%20tumor%20marker%20in%20pleural%20effusion%20diagnosis&amp;journal=Clin.%20Cancer%20Res.&amp;volume=9&amp;pages=2605-2612&amp;publication_year=2003&amp;author=Hung%2CTL\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR69\"> Chu, Y. W. et al. Selection of invasive and metastatic subpopulations from a human lung adenocarcinoma cell line. <i> Am. J. Respir. Cell Mol. Biol. </i> <b> 17 </b> , 353–360 (1997). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1165%2Fajrcmb.17.3.2837\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2sXmtVOrs7c%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9308922\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Selection%20of%20invasive%20and%20metastatic%20subpopulations%20from%20a%20human%20lung%20adenocarcinoma%20cell%20line&amp;journal=Am.%20J.%20Respir.%20Cell%20Mol.%20Biol.&amp;doi=10.1165%2Fajrcmb.17.3.2837&amp;volume=17&amp;pages=353-360&amp;publication_year=1997&amp;author=Chu%2CYW\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR70\"> Wisniewski, J. R., Zougman, A., Nagaraj, N. &amp; Mann, M. Universal sample preparation method for proteome analysis. <i> Nat. Methods </i> <b> 6 </b> , 359–362 (2009). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1038%2Fnmeth.1322\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXks12ksb0%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19377485\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Universal%20sample%20preparation%20method%20for%20proteome%20analysis&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fnmeth.1322&amp;volume=6&amp;pages=359-362&amp;publication_year=2009&amp;author=Wisniewski%2CJR&amp;author=Zougman%2CA&amp;author=Nagaraj%2CN&amp;author=Mann%2CM\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR71\"> Bern, M., Kil, Y. J. &amp; Becker, C. Byonic: advanced peptide and protein identification software. <i> Curr. Protocol. Bioinform. </i> <b> 13 </b> , 13 20 11–13 20 14 (2012). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Byonic%3A%20advanced%20peptide%20and%20protein%20identification%20software&amp;journal=Curr.%20Protocol.%20Bioinform.&amp;volume=13&amp;pages=13%2020%2011-13%2020%2014&amp;publication_year=2012&amp;author=Bern%2CM&amp;author=Kil%2CYJ&amp;author=Becker%2CC\"> Google Scholar </a> </p> </li> </ol> </div> </div> </div> </div> </aside> </div> <footer class=\"composite-layer\" itemscope=\"\" itemtype=\"http://schema.org/Periodical\"> <div class=\"u-mt-16 u-mb-16\"> <div class=\"u-container\"> <div class=\"u-display-flex u-flex-wrap u-justify-content-space-between\"> <div class=\"c-meta u-ma-0 u-flex-shrink\"> <p class=\"c-meta__item c-meta__type u-mt-0\"> <span itemprop=\"name\"> Nature Communications </span> ( <i itemprop=\"alternateName\"> Nat Commun </i> ) </p> <p class=\"c-meta__item u-mt-0\"> <abbr title=\"International Standard Serial Number\"> ISSN </abbr> <span itemprop=\"issn\"> 2041-1723 </span> (online) </p> </div> </div> </div> </div> <div class=\"c-footer\"> <div class=\"u-hide-print\"> <h2 class=\"u-visually-hidden\"> nature.com sitemap </h2> <div class=\"c-footer__container\"> <div class=\"c-footer__grid c-footer__group--separator\"> <div class=\"c-footer__group\"> <h3 class=\"c-footer__heading u-mt-0\"> About Nature Portfolio </h3> <ul class=\"c-footer__list\"> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.nature.com/npg_/company_info/index.html\"> About us </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.nature.com/npg_/press_room/press_releases.html\"> Press releases </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://press.nature.com/\"> Press office </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://support.nature.com/support/home\"> Contact us </a> </li> </ul> </div> <div class=\"c-footer__group\"> <h3 class=\"c-footer__heading u-mt-0\"> Discover content </h3> <ul class=\"c-footer__list\"> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.nature.com/siteindex\"> Journals A-Z </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.nature.com/subjects\"> Articles by subject </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.protocols.io/\"> protocols.io </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.natureindex.com/\"> Nature Index </a> </li> </ul> </div> <div class=\"c-footer__group\"> <h3 class=\"c-footer__heading u-mt-0\"> Publishing policies </h3> <ul class=\"c-footer__list\"> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.nature.com/authors/editorial_policies\"> Nature portfolio policies </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.nature.com/nature-research/open-access\"> Open access </a> </li> </ul> </div> <div class=\"c-footer__group\"> <h3 class=\"c-footer__heading u-mt-0\"> Author &amp; Researcher services </h3> <ul class=\"c-footer__list\"> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.nature.com/reprints\"> Reprints &amp; permissions </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.springernature.com/gp/authors/research-data\"> Research data </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://authorservices.springernature.com/language-editing/\"> Language editing </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://authorservices.springernature.com/scientific-editing/\"> Scientific editing </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://masterclasses.nature.com/\"> Nature Masterclasses </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://solutions.springernature.com/\"> Research Solutions </a> </li> </ul> </div> <div class=\"c-footer__group\"> <h3 class=\"c-footer__heading u-mt-0\"> Libraries &amp; institutions </h3> <ul class=\"c-footer__list\"> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.springernature.com/gp/librarians/tools-services\"> Librarian service &amp; tools </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.springernature.com/gp/librarians/manage-your-account/librarianportal\"> Librarian portal </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.nature.com/openresearch/about-open-access/information-for-institutions\"> Open research </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.springernature.com/gp/librarians/recommend-to-your-library\"> Recommend to library </a> </li> </ul> </div> <div class=\"c-footer__group\"> <h3 class=\"c-footer__heading u-mt-0\"> Advertising &amp; partnerships </h3> <ul class=\"c-footer__list\"> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://partnerships.nature.com/product/digital-advertising/\"> Advertising </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://partnerships.nature.com/\"> Partnerships &amp; Services </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://partnerships.nature.com/media-kits/\"> Media kits </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://partnerships.nature.com/product/branded-content-native-advertising/\"> Branded content </a> </li> </ul> </div> <div class=\"c-footer__group\"> <h3 class=\"c-footer__heading u-mt-0\"> Professional development </h3> <ul class=\"c-footer__list\"> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.nature.com/naturecareers/\"> Nature Careers </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://conferences.nature.com\"> Nature <span class=\"u-visually-hidden\"> </span> Conferences </a> </li> </ul> </div> <div class=\"c-footer__group\"> <h3 class=\"c-footer__heading u-mt-0\"> Regional websites </h3> <ul class=\"c-footer__list\"> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.nature.com/natafrica\"> Nature Africa </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"http://www.naturechina.com\"> Nature China </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.nature.com/nindia\"> Nature India </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.nature.com/natitaly\"> Nature Italy </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.natureasia.com/ja-jp\"> Nature Japan </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.nature.com/nmiddleeast\"> Nature Middle East </a> </li> </ul> </div> </div> </div> <div class=\"c-footer__container\"> <ul class=\"c-footer__links\"> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.nature.com/info/privacy\"> Privacy Policy </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.nature.com/info/cookies\"> Use of cookies </a> </li> <li class=\"c-footer__item\"> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.nature.com/info/legal-notice\"> Legal notice </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.nature.com/info/accessibility-statement\"> Accessibility statement </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.nature.com/info/terms-and-conditions\"> Terms &amp; Conditions </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.springernature.com/ccpa\"> Your US state privacy rights </a> </li> </ul> </div> </div> <div class=\"c-footer__container\"> <a class=\"c-footer__link\" href=\"https://www.springernature.com/\"> </a> <p class=\"c-footer__legal\"> © 2025 Springer Nature Limited </p> </div> </div> <div aria-hidden=\"true\" class=\"u-visually-hidden\"> </div> </footer> <div class=\"c-site-messages message u-hide-print c-site-messages--nature-briefing c-site-messages--nature-briefing-email-variant c-site-messages--nature-briefing-redesign-2020 sans-serif\"> <div class=\"c-site-messages__banner-large\"> <div class=\"c-site-messages__close-container\"> </div> <div class=\"c-site-messages__form-container\"> <div class=\"grid grid-12 last\"> <div class=\"grid grid-4\"> <p class=\"c-site-messages--nature-briefing__strapline extra-tight-line-height\"> Sign up for the <em> Nature Briefing </em> newsletter — what matters in science, free to your inbox daily. </p> </div> <div class=\"grid grid-8 last\"> </div> </div> </div> </div> <div class=\"c-site-messages__banner-small\"> <div class=\"c-site-messages__close-container\"> </div> <div class=\"c-site-messages__content text14\"> <span class=\"c-site-messages--nature-briefing__strapline strong\"> Get the most important science stories of the day, free in your inbox. </span> <a class=\"nature-briefing__link text14 sans-serif\" href=\"https://www.nature.com/briefing/signup/?brieferEntryPoint=MainBriefingBanner\" rel=\"noreferrer noopener\" target=\"_blank\"> Sign up for Nature Briefing </a> </div> </div> </div> <div> </div> </body> </html>\n",
      "base_parser.py - line 426 - INFO - Loading prompt: GPT_from_full_input_Examples for model gemini-2.0-flash\n",
      "prompt_manager.py - line 32 - INFO - Loading prompt: GPT_from_full_input_Examples from user_prompt_dir: None, subdir: \n",
      "base_parser.py - line 1108 - INFO - Expected string but got list. Converting list to string.\n",
      "base_parser.py - line 1115 - DEBUG - Counting tokens for model: gemini-2.0-flash, prompt length: 2493 char\n",
      "base_parser.py - line 1124 - DEBUG - Rough estimate of token count for Gemini model 'gemini-2.0-flash': 623\n",
      "base_parser.py - line 433 - INFO - Content length: 408685\n",
      "base_parser.py - line 435 - DEBUG - static_prompt: [{'role': 'system', 'content': 'I am a large language model trained to be informative and comprehensive. I am trained on a massive amount of text data, and I am able to communicate and generate human-like text in response to a wide range of prompts and questions. For this task, I will act as a specialized assistant that can identify datasets mentioned in a publication and create a summary suitable for non-specialists.\\n\\nThe output should be a JSON array of objects, where each object has the following keys:\\n- \"dataset_identifier\": This is any alphanumeric string (maybe including punctuation marks) that uniquely identifies or provides access to a dataset, most frequently an accession code.\\n- \"repository_reference\": This is the URL or reference to the data repository where the dataset can be found.\\n\\nHere are some examples for reference:\\n\\n[\\'dataset_identifier\\'=> \\'EGAS00001000925\\', \\'repository_reference\\'=> \\'https=>//ega-archive.org/datasets/EGAS00001000925\\',\\n \\'dataset_identifier\\'=> \\'GSE69091\\', \\'repository_reference\\'=> \\'https=>//www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE69091\\',\\n \\'dataset_identifier\\'=> \\'PRJNA306801\\', \\'repository_reference\\'=> \\'https=>//www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA306801\\',\\n \\'dataset_identifier\\'=> \\'phs003416.v1.p1\\', \\'repository_reference\\'=> \\'dbGaP\\',\\n \\'dataset_identifier\\'=> \\'PXD049309\\', \\'repository_reference\\'=> \\'https=>//www.ebi.ac.uk/pride/archive/projects/PXD049309\\',\\n \\'dataset_identifier\\'=> \\'IPX0004230000\\', \\'repository_reference\\'=> \\'http=>//www.iprox.org\\',\\n \\'dataset_identifier\\'=> \\'MSV000092944\\', \\'repository_reference\\'=> \\'https=>//massive.ucsd.edu/\\',\\n \\'dataset_identifier\\'=> \\'n/a\\', \\'repository_reference\\'=> \\'https://data.broadinstitute.org/ccle_legacy_data/mRNA_expression/\\'\\n]'}, {'role': 'user', 'content': 'Given the information that I am going to share:\\n 1) the webpage in HTML format that you have to extract datasets information from.\\n 2) a sample of already known data repositories.\\n\\nPlease return a JSON array of objects where each object has the following structure:\\n- `dataset_identifier`: The dataset identifier (a code). If not found, set it to \"n/a\".\\n- `repository_reference`: The URL or reference to the data repository. If not found, set it to \"n/a\".\\n\\nPlease follow these strict instructions:\\n- The output must be a valid JSON array of objects.\\n- Each object must contain the keys `dataset_identifier` and `repository_reference`.\\n- Any other output format will be considered invalid.\\x08\\x08Below is the input data that you will use to generate the output:\\n1) html => {content}\\n2) repos => {repos}.'}]\n",
      "base_parser.py - line 1108 - INFO - Expected string but got list. Converting list to string.\n",
      "base_parser.py - line 1115 - DEBUG - Counting tokens for model: gemini-2.0-flash, prompt length: 411456 char\n",
      "base_parser.py - line 1124 - DEBUG - Rough estimate of token count for Gemini model 'gemini-2.0-flash': 102864\n",
      "base_parser.py - line 444 - INFO - Prompt messages total length: 102864 tokens\n",
      "base_parser.py - line 445 - DEBUG - Prompt messages: [{'role': 'system', 'content': 'I am a large language model trained to be informative and comprehensive. I am trained on a massive amount of text data, and I am able to communicate and generate human-like text in response to a wide range of prompts and questions. For this task, I will act as a specialized assistant that can identify datasets mentioned in a publication and create a summary suitable for non-specialists.\\n\\nThe output should be a JSON array of objects, where each object has the following keys:\\n- \"dataset_identifier\": This is any alphanumeric string (maybe including punctuation marks) that uniquely identifies or provides access to a dataset, most frequently an accession code.\\n- \"repository_reference\": This is the URL or reference to the data repository where the dataset can be found.\\n\\nHere are some examples for reference:\\n\\n[\\'dataset_identifier\\'=> \\'EGAS00001000925\\', \\'repository_reference\\'=> \\'https=>//ega-archive.org/datasets/EGAS00001000925\\',\\n \\'dataset_identifier\\'=> \\'GSE69091\\', \\'repository_reference\\'=> \\'https=>//www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE69091\\',\\n \\'dataset_identifier\\'=> \\'PRJNA306801\\', \\'repository_reference\\'=> \\'https=>//www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA306801\\',\\n \\'dataset_identifier\\'=> \\'phs003416.v1.p1\\', \\'repository_reference\\'=> \\'dbGaP\\',\\n \\'dataset_identifier\\'=> \\'PXD049309\\', \\'repository_reference\\'=> \\'https=>//www.ebi.ac.uk/pride/archive/projects/PXD049309\\',\\n \\'dataset_identifier\\'=> \\'IPX0004230000\\', \\'repository_reference\\'=> \\'http=>//www.iprox.org\\',\\n \\'dataset_identifier\\'=> \\'MSV000092944\\', \\'repository_reference\\'=> \\'https=>//massive.ucsd.edu/\\',\\n \\'dataset_identifier\\'=> \\'n/a\\', \\'repository_reference\\'=> \\'https://data.broadinstitute.org/ccle_legacy_data/mRNA_expression/\\'\\n]'}, {'role': 'user', 'content': 'Given the information that I am going to share:\\n 1) the webpage in HTML format that you have to extract datasets information from.\\n 2) a sample of already known data repositories.\\n\\nPlease return a JSON array of objects where each object has the following structure:\\n- `dataset_identifier`: The dataset identifier (a code). If not found, set it to \"n/a\".\\n- `repository_reference`: The URL or reference to the data repository. If not found, set it to \"n/a\".\\n\\nPlease follow these strict instructions:\\n- The output must be a valid JSON array of objects.\\n- Each object must contain the keys `dataset_identifier` and `repository_reference`.\\n- Any other output format will be considered invalid.\\x08\\x08Below is the input data that you will use to generate the output:\\n1) html => <html class=\"js\" lang=\"en\"> <head> <link as=\"font\" crossorigin=\"\" href=\"/static/fonts/HardingText-Regular-Web-cecd90984f.woff2\" rel=\"preload\" type=\"font/woff2\"/> <title> Defective N-glycosylation of IL6 induces metastasis and tyrosine kinase inhibitor resistance in lung cancer | Nature Communications </title> <link href=\"https://www.nature.com/ncomms.rss\" rel=\"alternate\" type=\"application/rss+xml\"/> <link crossorigin=\"\" href=\"https://cmp.nature.com\" rel=\"preconnect\"/> <link href=\"/static/css/enhanced-article-nature-branded-3ad80c7544.css\" media=\"only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)\" onload=\"this.media=\\'only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)\\';this.onload=null\" rel=\"stylesheet\"/> <link href=\"/static/css/article-print-c9121fecab.css\" media=\"print\" rel=\"stylesheet\" type=\"text/css\"/> <link href=\"/static/images/favicons/nature/apple-touch-icon-f39cb19454.png\" rel=\"apple-touch-icon\" sizes=\"180x180\"/> <link href=\"/static/images/favicons/nature/favicon-48x48-b52890008c.png\" rel=\"icon\" sizes=\"48x48\" type=\"image/png\"/> <link href=\"/static/images/favicons/nature/favicon-32x32-3fe59ece92.png\" rel=\"icon\" sizes=\"32x32\" type=\"image/png\"/> <link href=\"/static/images/favicons/nature/favicon-16x16-951651ab72.png\" rel=\"icon\" sizes=\"16x16\" type=\"image/png\"/> <link crossorigin=\"use-credentials\" href=\"/static/manifest.json\" rel=\"manifest\"/> <link color=\"#000000\" href=\"/static/images/favicons/nature/safari-pinned-tab-69bff48fe6.svg\" rel=\"mask-icon\"/> <link href=\"/static/images/favicons/nature/favicon.ico\" rel=\"shortcut icon\"/> <link href=\"https://www.nature.com/search\" rel=\"search\"/> <link href=\"https://www.nature.com/opensearch/opensearch.xml\" rel=\"search\" title=\"nature.com\" type=\"application/opensearchdescription+xml\"/> <link href=\"https://www.nature.com/opensearch/request\" rel=\"search\" title=\"nature.com\" type=\"application/sru+xml\"/> <link href=\"https://www.nature.com/articles/s41467-024-51831-7\" rel=\"canonical\"/> </head> <body class=\"article-page\"> <dialog aria-labelledby=\"cc-banner-label\" class=\"cc-banner\" open=\"\"> <div autofocus=\"\" class=\"cc-banner__content\" tabindex=\"-1\"> <div class=\"cc-banner__container\"> <div class=\"cc-banner__header\"> <h2 class=\"cc-banner__title\" id=\"cc-banner-label\"> Your privacy, your choice </h2> </div> <div class=\"cc-banner__body\"> <div class=\"cc-banner__policy\"> <p> We use essential cookies to make sure the site can function. We also use optional cookies for advertising, personalisation of content, usage analysis, and social media. </p> <p> By accepting optional cookies, you consent to the processing of your personal data - including transfers to third parties. Some third parties are outside of the European Economic Area, with varying standards of data protection. </p> <p> See our <a href=\"https://www.nature.com/info/privacy\"> privacy policy </a> for more information on the use of your personal data. </p> <p> for further information and to change your choices. </p> </div> </div> <div class=\"cc-banner__footer\"> </div> </div> </div> </dialog> <div class=\"position-relative cleared z-index-50 background-white\"> <a class=\"c-skip-link\" href=\"#content\"> Skip to main content </a> <div class=\"c-grade-c-banner u-hide\"> <div class=\"c-grade-c-banner__container\"> <p> Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles and JavaScript. </p> </div> </div> <div class=\"u-hide u-show-following-ad\"> </div> <aside class=\"c-ad c-ad--728x90\"> <div class=\"c-ad__inner\"> <p class=\"c-ad__label\"> Advertisement </p> <div class=\"div-gpt-ad advert leaderboard js-ad text-center hide-print grade-c-hide\" id=\"div-gpt-ad-top-1\"> <div id=\"google_ads_iframe_/270604982/nature/ncomms/articles_0__container__\"> </div> </div> </div> </aside> <header class=\"c-header\" id=\"header\"> <div class=\"c-header__row\"> <div class=\"c-header__container\"> <div class=\"c-header__split\"> <div class=\"c-header__logo-container\"> <a href=\"/ncomms\"> <picture class=\"c-header__logo\"> <source media=\"(min-width: 875px)\" srcset=\"https://media.springernature.com/full/nature-cms/uploads/product/ncomms/header-03d2e325c0a02f6df509e5730e9be304.svg\"/> </picture> </a> </div> <ul class=\"c-header__menu c-header__menu--global\"> <li class=\"c-header__item c-header__item--padding c-header__item--hide-md-max\"> <a class=\"c-header__link\" href=\"https://www.nature.com/siteindex\"> <span> View all journals </span> </a> </li> <li class=\"c-header__item c-header__item--padding c-header__item--pipe\"> <a aria-expanded=\"false\" aria-haspopup=\"true\" class=\"c-header__link c-header__link--search\" href=\"javascript:;\" role=\"button\"> <span> Search </span> </a> <div class=\"c-header__dropdown c-header__dropdown--full-width has-tethered u-js-hide\" hidden=\"\" id=\"search-menu\"> <div class=\"c-header__container\"> <h2 class=\"c-header__visually-hidden\"> Search </h2> <div class=\"c-header__flush\"> <a class=\"c-header__link\" href=\"/search/advanced\"> Advanced search </a> </div> <h3 class=\"c-header__heading c-header__heading--keyline\"> Quick links </h3> <ul class=\"c-header__list\"> <li> <a class=\"c-header__link\" href=\"/subjects\"> Explore articles by subject </a> </li> <li> <a class=\"c-header__link\" href=\"/naturecareers\"> Find a job </a> </li> <li> <a class=\"c-header__link\" href=\"/authors/index.html\"> Guide to authors </a> </li> <li> <a class=\"c-header__link\" href=\"/authors/editorial_policies/\"> Editorial policies </a> </li> </ul> </div> </div> </li> <li class=\"c-header__item c-header__item--padding c-header__item--snid-account-widget c-header__item--pipe\"> <a class=\"c-header__link eds-c-header__link\" href=\"https://idp.nature.com/auth/personal/springernature?redirect_uri=https://www.nature.com/articles/s41467-024-51831-7\" id=\"identity-account-widget\"> <span class=\"eds-c-header__widget-fragment-title\"> Log in </span> </a> </li> </ul> </div> </div> </div> <div class=\"c-header__row\"> <div class=\"c-header__container\"> <div class=\"c-header__split\"> <ul class=\"c-header__menu c-header__menu--journal\"> <li class=\"c-header__item c-header__item--dropdown-menu\"> <a aria-expanded=\"false\" aria-haspopup=\"true\" class=\"c-header__link\" href=\"javascript:;\" role=\"button\"> <span> <span class=\"c-header__show-text\"> Explore </span> content </span> </a> <nav aria-labelledby=\"Explore-content\" class=\"c-header__dropdown has-tethered u-js-hide\" hidden=\"\" id=\"explore\"> <div class=\"c-header__container\"> <h2 class=\"c-header__heading c-header__heading--js-hide\" id=\"Explore-content\"> Explore content </h2> <ul class=\"c-header__list c-header__list--js-stack\"> <li class=\"c-header__item\"> <a class=\"c-header__link\" href=\"/ncomms/research-articles\"> Research articles </a> </li> <li class=\"c-header__item\"> <a class=\"c-header__link\" href=\"/ncomms/reviews-and-analysis\"> Reviews &amp; Analysis </a> </li> <li class=\"c-header__item\"> <a class=\"c-header__link\" href=\"/ncomms/news-and-comment\"> News &amp; Comment </a> </li> <li class=\"c-header__item\"> <a class=\"c-header__link\" href=\"/ncomms/video\"> Videos </a> </li> <li class=\"c-header__item\"> <a class=\"c-header__link\" href=\"/ncomms/collections\"> Collections </a> </li> <li class=\"c-header__item\"> <a class=\"c-header__link\" href=\"/ncomms/browse-subjects\"> Subjects </a> </li> </ul> <ul class=\"c-header__list c-header__list--js-stack\"> <li class=\"c-header__item\"> <a class=\"c-header__link\" href=\"https://www.facebook.com/NatureCommunications\"> Follow us on Facebook </a> </li> <li class=\"c-header__item\"> <a class=\"c-header__link\" href=\"https://twitter.com/NatureComms\"> Follow us on Twitter </a> </li> <li class=\"c-header__item c-header__item--hide-lg\"> <a class=\"c-header__link\" href=\"https://www.nature.com/my-account/alerts/subscribe-journal?list-id=264\" rel=\"nofollow\"> Sign up for alerts </a> </li> <li class=\"c-header__item c-header__item--hide-lg\"> <a class=\"c-header__link\" href=\"https://www.nature.com/ncomms.rss\"> <span> RSS feed </span> </a> </li> </ul> </div> </nav> </li> <li class=\"c-header__item c-header__item--dropdown-menu\"> <a aria-expanded=\"false\" aria-haspopup=\"true\" class=\"c-header__link\" href=\"javascript:;\" role=\"button\"> <span> About <span class=\"c-header__show-text\"> the journal </span> </span> </a> <nav aria-labelledby=\"About-the-journal\" class=\"c-header__dropdown has-tethered u-js-hide\" hidden=\"\" id=\"about-the-journal\"> <div class=\"c-header__container\"> <h2 class=\"c-header__heading c-header__heading--js-hide\" id=\"About-the-journal\"> About the journal </h2> <ul class=\"c-header__list c-header__list--js-stack\"> <li class=\"c-header__item\"> <a class=\"c-header__link\" href=\"/ncomms/aims\"> Aims &amp; Scope </a> </li> <li class=\"c-header__item\"> <a class=\"c-header__link\" href=\"/ncomms/editors\"> Editors </a> </li> <li class=\"c-header__item\"> <a class=\"c-header__link\" href=\"/ncomms/journal-information\"> Journal Information </a> </li> <li class=\"c-header__item\"> <a class=\"c-header__link\" href=\"/ncomms/open-access\"> Open Access Fees and Funding </a> </li> <li class=\"c-header__item\"> <a class=\"c-header__link\" href=\"/ncomms/calls-for-papers\"> Calls for Papers </a> </li> <li class=\"c-header__item\"> <a class=\"c-header__link\" href=\"/ncomms/editorial-values-statement\"> Editorial Values Statement </a> </li> <li class=\"c-header__item\"> <a class=\"c-header__link\" href=\"/ncomms/journal-impact\"> Journal Metrics </a> </li> <li class=\"c-header__item\"> <a class=\"c-header__link\" href=\"/ncomms/editorshighlights\"> Editors\\' Highlights </a> </li> <li class=\"c-header__item\"> <a class=\"c-header__link\" href=\"/ncomms/contact\"> Contact </a> </li> <li class=\"c-header__item\"> <a class=\"c-header__link\" href=\"/ncomms/editorial-policies\"> Editorial policies </a> </li> <li class=\"c-header__item\"> <a class=\"c-header__link\" href=\"/ncomms/top-articles\"> Top Articles </a> </li> </ul> </div> </nav> </li> <li class=\"c-header__item c-header__item--dropdown-menu\"> <a aria-expanded=\"false\" aria-haspopup=\"true\" class=\"c-header__link c-header__link--dropdown-menu\" href=\"javascript:;\" role=\"button\"> <span> Publish <span class=\"c-header__show-text\"> with us </span> </span> </a> <nav aria-labelledby=\"Publish-with-us-label\" class=\"c-header__dropdown has-tethered u-js-hide\" hidden=\"\" id=\"publish-with-us\"> <div class=\"c-header__container\"> <h2 class=\"c-header__heading c-header__heading--js-hide\" id=\"Publish-with-us-label\"> Publish with us </h2> <ul class=\"c-header__list c-header__list--js-stack\"> <li class=\"c-header__item\"> <a class=\"c-header__link\" href=\"/ncomms/submit\"> For authors </a> </li> <li class=\"c-header__item\"> <a class=\"c-header__link\" href=\"/ncomms/for-reviewers\"> For Reviewers </a> </li> <li class=\"c-header__item\"> <a class=\"c-header__link\" href=\"https://authorservices.springernature.com/go/sn/?utm_source=For+Authors&amp;utm_medium=Website_Nature&amp;utm_campaign=Platform+Experimentation+2022&amp;utm_id=PE2022\"> Language editing services </a> </li> <li class=\"c-header__item\"> <a class=\"c-header__link\" href=\"/ncomms/open-access-funding\"> Open access funding </a> </li> <li class=\"c-header__item c-header__item--keyline\"> <a class=\"c-header__link\" href=\"https://mts-ncomms.nature.com/\"> Submit manuscript </a> </li> </ul> </div> </nav> </li> </ul> <ul class=\"c-header__menu c-header__menu--hide-lg-max\"> <li class=\"c-header__item\"> <a class=\"c-header__link\" href=\"https://journal-alerts.springernature.com/subscribe?journal_id=41467\" rel=\"nofollow\"> <span> Sign up for alerts </span> </a> </li> <li class=\"c-header__item c-header__item--pipe\"> <a class=\"c-header__link\" href=\"https://www.nature.com/ncomms.rss\"> <span> RSS feed </span> </a> </li> </ul> </div> </div> </div> </header> <nav aria-label=\"breadcrumbs\" class=\"u-mb-16\"> <div class=\"u-container\"> <ol class=\"c-breadcrumbs\" itemscope=\"\" itemtype=\"https://schema.org/BreadcrumbList\"> <li class=\"c-breadcrumbs__item\" id=\"breadcrumb0\" itemprop=\"itemListElement\" itemscope=\"\" itemtype=\"https://schema.org/ListItem\"> <a class=\"c-breadcrumbs__link\" href=\"/\" itemprop=\"item\"> <span itemprop=\"name\"> nature </span> </a> </li> <li class=\"c-breadcrumbs__item\" id=\"breadcrumb1\" itemprop=\"itemListElement\" itemscope=\"\" itemtype=\"https://schema.org/ListItem\"> <a class=\"c-breadcrumbs__link\" href=\"/ncomms\" itemprop=\"item\"> <span itemprop=\"name\"> nature communications </span> </a> </li> <li class=\"c-breadcrumbs__item\" id=\"breadcrumb2\" itemprop=\"itemListElement\" itemscope=\"\" itemtype=\"https://schema.org/ListItem\"> <a class=\"c-breadcrumbs__link\" href=\"/ncomms/articles?type=article\" itemprop=\"item\"> <span itemprop=\"name\"> articles </span> </a> </li> <li class=\"c-breadcrumbs__item\" id=\"breadcrumb3\" itemprop=\"itemListElement\" itemscope=\"\" itemtype=\"https://schema.org/ListItem\"> <span itemprop=\"name\"> article </span> </li> </ol> </div> </nav> </div> <div class=\"u-container u-mt-32 u-mb-32 u-clearfix\" id=\"content\"> <main class=\"c-article-main-column u-float-left js-main-column\"> <div aria-hidden=\"false\" class=\"c-context-bar c-context-bar--sticky\" id=\"js-enable-context-bar\"> <div class=\"c-context-bar__container u-container\"> <div class=\"c-context-bar__title\"> Defective N-glycosylation of IL6 induces metastasis and tyrosine kinase inhibitor resistance in lung cancer </div> <div class=\"c-pdf-download u-clear-both js-pdf-download\"> <a class=\"u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link\" download=\"\" href=\"/articles/s41467-024-51831-7.pdf\" tabindex=\"-1\"> <span class=\"c-pdf-download__text\"> Download PDF </span> </a> </div> </div> </div> <article lang=\"en\"> <div class=\"c-pdf-button__container u-mb-16 u-hide-at-lg js-context-bar-sticky-point-mobile\"> <div class=\"c-pdf-container\"> <div class=\"c-pdf-download u-clear-both js-pdf-download\"> <a class=\"u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link\" download=\"\" href=\"/articles/s41467-024-51831-7.pdf\"> <span class=\"c-pdf-download__text\"> Download PDF </span> </a> </div> </div> </div> <div class=\"c-article-header\"> <header> <ul class=\"c-article-identifiers\"> <li class=\"c-article-identifiers__item\"> Article </li> <li class=\"c-article-identifiers__item\"> <a class=\"u-color-open-access\" href=\"https://www.springernature.com/gp/open-research/about/the-fundamentals-of-open-access-and-open-research\"> Open access </a> </li> <li class=\"c-article-identifiers__item\"> Published: <time datetime=\"2024-09-09\"> 09 September 2024 </time> </li> </ul> <h1 class=\"c-article-title\"> Defective N-glycosylation of IL6 induces metastasis and tyrosine kinase inhibitor resistance in lung cancer </h1> <ul class=\"c-article-author-list c-article-author-list--short js-no-scroll\"> <li class=\"c-article-author-list__item\"> <a href=\"#auth-Chun_Hua-Hung-Aff1-Aff2-Aff3-Aff4\"> Chun-Hua Hung </a> <span class=\"u-js-hide\"> <a class=\"js-orcid\" href=\"https://orcid.org/0000-0002-8580-8759\"> <span class=\"u-visually-hidden\"> ORCID: </span> orcid.org/0000-0002-8580-8759 </a> </span> <sup class=\"u-js-hide\"> <a href=\"#Aff1\" tabindex=\"-1\"> 1 </a> , <a href=\"#Aff2\" tabindex=\"-1\"> 2 </a> , <a href=\"#Aff3\" tabindex=\"-1\"> 3 </a> , <a href=\"#Aff4\" tabindex=\"-1\"> 4 </a> </sup> , </li> <li class=\"c-article-author-list__item\"> <a href=\"#auth-Shang_Yin-Wu-Aff1-Aff5\"> Shang-Yin Wu </a> <span class=\"u-js-hide\"> <a class=\"js-orcid\" href=\"https://orcid.org/0000-0003-2914-510X\"> <span class=\"u-visually-hidden\"> ORCID: </span> orcid.org/0000-0003-2914-510X </a> </span> <sup class=\"u-js-hide\"> <a href=\"#Aff1\" tabindex=\"-1\"> 1 </a> , <a href=\"#Aff5\" tabindex=\"-1\"> 5 </a> </sup> , </li> <li class=\"c-article-author-list__item c-article-author-list__item--hide-small-screen\"> <a href=\"#auth-Cheng_I_Daniel-Yao-Aff6\"> Cheng-I Daniel Yao </a> <span class=\"u-js-hide\"> <a class=\"js-orcid\" href=\"https://orcid.org/0000-0002-0902-045X\"> <span class=\"u-visually-hidden\"> ORCID: </span> orcid.org/0000-0002-0902-045X </a> </span> <sup class=\"u-js-hide\"> <a href=\"#Aff6\" tabindex=\"-1\"> 6 </a> </sup> , </li> <li class=\"c-article-author-list__item c-article-author-list__item--hide-small-screen\"> <a href=\"#auth-Hsuan_Heng-Yeh-Aff2\"> Hsuan-Heng Yeh </a> <sup class=\"u-js-hide\"> <a href=\"#Aff2\" tabindex=\"-1\"> 2 </a> </sup> , </li> <li class=\"c-article-author-list__item c-article-author-list__item--hide-small-screen\"> <a href=\"#auth-Chien_Chung-Lin-Aff5-Aff7-Aff8\"> Chien-Chung Lin </a> <sup class=\"u-js-hide\"> <a href=\"#Aff5\" tabindex=\"-1\"> 5 </a> , <a href=\"#Aff7\" tabindex=\"-1\"> 7 </a> , <a href=\"#Aff8\" tabindex=\"-1\"> 8 </a> </sup> , </li> <li class=\"c-article-author-list__item c-article-author-list__item--hide-small-screen\"> <a href=\"#auth-Chang_Yao-Chu-Aff9\"> Chang-Yao Chu </a> <sup class=\"u-js-hide\"> <a href=\"#Aff9\" tabindex=\"-1\"> 9 </a> </sup> , </li> <li class=\"c-article-author-list__item c-article-author-list__item--hide-small-screen\"> <a href=\"#auth-Tzu_Yu-Huang-Aff10\"> Tzu-Yu Huang </a> <sup class=\"u-js-hide\"> <a href=\"#Aff10\" tabindex=\"-1\"> 10 </a> </sup> , </li> <li class=\"c-article-author-list__item c-article-author-list__item--hide-small-screen\"> <a href=\"#auth-Meng_Ru-Shen-Aff11-Aff12\"> Meng-Ru Shen </a> <span class=\"u-js-hide\"> <a class=\"js-orcid\" href=\"https://orcid.org/0000-0003-0506-3505\"> <span class=\"u-visually-hidden\"> ORCID: </span> orcid.org/0000-0003-0506-3505 </a> </span> <sup class=\"u-js-hide\"> <a href=\"#Aff11\" tabindex=\"-1\"> 11 </a> , <a href=\"#Aff12\" tabindex=\"-1\"> 12 </a> </sup> , </li> <li class=\"c-article-author-list__item c-article-author-list__item--hide-small-screen\"> <a href=\"#auth-Chun_Hung-Lin-Aff6-Aff13\"> Chun-Hung Lin </a> <span class=\"u-js-hide\"> <a class=\"js-orcid\" href=\"https://orcid.org/0000-0002-6795-8825\"> <span class=\"u-visually-hidden\"> ORCID: </span> orcid.org/0000-0002-6795-8825 </a> </span> <sup class=\"u-js-hide\"> <a href=\"#Aff6\" tabindex=\"-1\"> 6 </a> , <a href=\"#Aff13\" tabindex=\"-1\"> 13 </a> </sup> &amp; </li> <li aria-label=\"Show all 10 authors for this article\" class=\"c-article-author-list__show-more\" title=\"Show all 10 authors for this article\"> … </li> <li class=\"c-article-author-list__item\"> <a href=\"#auth-Wu_Chou-Su-Aff1-Aff2-Aff3-Aff5-Aff10\"> Wu-Chou Su </a> <span class=\"u-js-hide\"> <a class=\"js-orcid\" href=\"https://orcid.org/0000-0003-2953-4105\"> <span class=\"u-visually-hidden\"> ORCID: </span> orcid.org/0000-0003-2953-4105 </a> </span> <sup class=\"u-js-hide\"> <a href=\"#Aff1\" tabindex=\"-1\"> 1 </a> , <a href=\"#Aff2\" tabindex=\"-1\"> 2 </a> , <a href=\"#Aff3\" tabindex=\"-1\"> 3 </a> , <a href=\"#Aff5\" tabindex=\"-1\"> 5 </a> , <a href=\"#Aff10\" tabindex=\"-1\"> 10 </a> </sup> </li> </ul> <p class=\"c-article-info-details\"> <a href=\"/ncomms\"> <i> Nature Communications </i> </a> <b> <span class=\"u-visually-hidden\"> volume </span> 15 </b> , Article number: <span> 7885 </span> ( <span> 2024 </span> ) <a class=\"c-article-info-details__cite-as u-hide-print\" href=\"#citeas\"> Cite this article </a> </p> <div class=\"c-article-metrics-bar__wrapper u-clear-both\"> <ul class=\"c-article-metrics-bar u-list-reset\"> <li class=\"c-article-metrics-bar__item\"> <p class=\"c-article-metrics-bar__count\"> 9119 <span class=\"c-article-metrics-bar__label\"> Accesses </span> </p> </li> <li class=\"c-article-metrics-bar__item\"> <p class=\"c-article-metrics-bar__count\"> 13 <span class=\"c-article-metrics-bar__label\"> Citations </span> </p> </li> <li class=\"c-article-metrics-bar__item\"> <p class=\"c-article-metrics-bar__count\"> 30 <span class=\"c-article-metrics-bar__label\"> Altmetric </span> </p> </li> <li class=\"c-article-metrics-bar__item\"> <p class=\"c-article-metrics-bar__details\"> <a href=\"/articles/s41467-024-51831-7/metrics\" rel=\"nofollow\"> Metrics <span class=\"u-visually-hidden\"> details </span> </a> </p> </li> </ul> </div> </header> </div> <div class=\"c-article-body\"> <section aria-labelledby=\"Abs1\" lang=\"en\"> <div class=\"c-article-section\" id=\"Abs1-section\"> <h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"Abs1\"> Abstract </h2> <div class=\"c-article-section__content\" id=\"Abs1-content\"> <p> The IL6-GP130-STAT3 pathway facilitates lung cancer progression and resistance to tyrosine kinase inhibitors. Although glycosylation alters the stability of GP130, its effect on the ligand IL6 remains unclear. We herein find that N-glycosylated IL6, especially at Asn73, primarily stimulates JAK-STAT3 signaling and prolongs STAT3 phosphorylation, whereas N-glycosylation-defective IL6 (deNG-IL6) induces shortened STAT3 activation and alters the downstream signaling preference for the SRC-YAP-SOX2 axis. This signaling shift induces <u> e </u> pithelial- <u> m </u> esenchymal <u> t </u> ransition (EMT) and migration in vitro and metastasis in vivo, which are suppressed by targeted inhibitors and shRNAs against SRC, YAP, and SOX2. Osimertinib-resistant lung cancer cells secrete a large amount of deNG-IL6 through reduced N-glycosyltransferase gene expression, leading to clear SRC-YAP activation. deNG-IL6 contributes to drug resistance, as confirmed by in silico analysis of cellular and clinical transcriptomes and signal expression in patient specimens. Therefore, the N-glycosylation status of IL6 not only affects cell behaviors but also shows promise in monitoring the dynamics of lung cancer evolution. </p> </div> </div> </section> <section aria-labelledby=\"inline-recommendations\" class=\"c-article-recommendations\"> <h3 class=\"c-article-recommendations-title\" id=\"inline-recommendations\"> Similar content being viewed by others </h3> <div class=\"c-article-recommendations-list\"> <div class=\"c-article-recommendations-list__item\"> <article class=\"c-article-recommendations-card\" itemscope=\"\" itemtype=\"http://schema.org/ScholarlyArticle\"> <div class=\"c-article-recommendations-card__img\"> </div> <div class=\"c-article-recommendations-card__main\"> <h3 class=\"c-article-recommendations-card__heading\" itemprop=\"name headline\"> <a class=\"c-article-recommendations-card__link\" href=\"https://www.nature.com/articles/s41388-025-03461-6?fromPaywallRec=false\" itemprop=\"url\"> Drug tolerance and persistence to EGFR inhibitor treatment are mediated by an ILK-SFK-YAP signaling axis in lung adenocarcinoma </a> </h3> <div class=\"c-article-meta-recommendations\"> <span class=\"c-article-meta-recommendations__item-type\"> Article </span> <span class=\"c-article-meta-recommendations__access-type\"> Open access </span> <span class=\"c-article-meta-recommendations__date\"> 31 May 2025 </span> </div> </div> </article> </div> <div class=\"c-article-recommendations-list__item\"> <article class=\"c-article-recommendations-card\" itemscope=\"\" itemtype=\"http://schema.org/ScholarlyArticle\"> <div class=\"c-article-recommendations-card__img\"> </div> <div class=\"c-article-recommendations-card__main\"> <h3 class=\"c-article-recommendations-card__heading\" itemprop=\"name headline\"> <a class=\"c-article-recommendations-card__link\" href=\"https://www.nature.com/articles/s41598-022-21419-6?fromPaywallRec=false\" itemprop=\"url\"> NT157 exerts antineoplastic activity by targeting JNK and AXL signaling in lung cancer cells </a> </h3> <div class=\"c-article-meta-recommendations\"> <span class=\"c-article-meta-recommendations__item-type\"> Article </span> <span class=\"c-article-meta-recommendations__access-type\"> Open access </span> <span class=\"c-article-meta-recommendations__date\"> 12 October 2022 </span> </div> </div> </article> </div> <div class=\"c-article-recommendations-list__item\"> <article class=\"c-article-recommendations-card\" itemscope=\"\" itemtype=\"http://schema.org/ScholarlyArticle\"> <div class=\"c-article-recommendations-card__img\"> </div> <div class=\"c-article-recommendations-card__main\"> <h3 class=\"c-article-recommendations-card__heading\" itemprop=\"name headline\"> <a class=\"c-article-recommendations-card__link\" href=\"https://www.nature.com/articles/s41420-022-01104-3?fromPaywallRec=false\" itemprop=\"url\"> Gankyrin modulated non-small cell lung cancer progression via glycolysis metabolism in a YAP1-dependent manner </a> </h3> <div class=\"c-article-meta-recommendations\"> <span class=\"c-article-meta-recommendations__item-type\"> Article </span> <span class=\"c-article-meta-recommendations__access-type\"> Open access </span> <span class=\"c-article-meta-recommendations__date\"> 09 July 2022 </span> </div> </div> </article> </div> </div> </section> <div class=\"main-content\"> <section> <div class=\"c-article-section\" id=\"Sec1-section\"> <h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"Sec1\"> Introduction </h2> <div class=\"c-article-section__content\" id=\"Sec1-content\"> <p> Interleukin-6 (IL6) is produced by both cancerous and noncancerous human cells and binds to IL6-receptor alpha (IL6Rα, also known as GP80). The IL6-IL6Rα complex then associates with coreceptor GP130, primarily triggering the GP130-JAK-STAT3 signaling <sup> <a href=\"#ref-CR1\" id=\"ref-link-section-d6776021e851\" title=\"Yu, H., Lee, H., Herrmann, A., Buettner, R. &amp; Jove, R. Revisiting STAT3 signalling in cancer: new and unexpected biological functions. Nat. Rev. Cancer 14, 736–746 (2014).\"> 1 </a> , <a href=\"#ref-CR2\" id=\"ref-link-section-d6776021e851_1\" title=\"Jones, S. A. &amp; Jenkins, B. J. Recent insights into targeting the IL-6 cytokine family in inflammatory diseases and cancer. Nat. Rev. Immunol. 18, 773–789 (2018).\"> 2 </a> , <a aria-label=\"Reference 3\" href=\"/articles/s41467-024-51831-7#ref-CR3\" id=\"ref-link-section-d6776021e854\" title=\"Rose-John, S. The soluble interleukin 6 receptor: advanced therapeutic options in inflammation. Clin. Pharmacol. Therapeut. 102, 591–598 (2017).\"> 3 </a> </sup> axis to regulate immune responses and cancer pathogenesis <sup> <a href=\"#ref-CR4\" id=\"ref-link-section-d6776021e858\" title=\"Trikha, M., Corringham, R., Klein, B. &amp; Rossi, J. F. Targeted anti-interleukin-6 monoclonal antibody therapy for cancer: a review of the rationale and clinical evidence. Clin. Cancer Res. 9, 4653–4665 (2003).\"> 4 </a> , <a href=\"#ref-CR5\" id=\"ref-link-section-d6776021e858_1\" title=\"Hunter, C. A. &amp; Jones, S. A. IL-6 as a keystone cytokine in health and disease. Nat. Immunol. 16, 448–457 (2015).\"> 5 </a> , <a href=\"#ref-CR6\" id=\"ref-link-section-d6776021e858_2\" title=\"Garbers, C., Heink, S., Korn, T. &amp; Rose-John, S. Interleukin-6: designing specific therapeutics for a complex cytokine. Nat. Rev. Drug Discov. 17, 395–412 (2018).\"> 6 </a> , <a aria-label=\"Reference 7\" href=\"/articles/s41467-024-51831-7#ref-CR7\" id=\"ref-link-section-d6776021e861\" title=\"Murakami, M., Kamimura, D. &amp; Hirano, T. Pleiotropy and specificity: insights from the interleukin 6 family of cytokines. Immunity 50, 812–831 (2019).\"> 7 </a> </sup> . In non-small cell lung cancer (NSCLC), an increase in circulating IL6 is associated with poor survival of patients, and autocrine IL6-mediated GP130-JAK-STAT3 signaling is positively correlated with tumor progression, apoptotic resistance, and drug resistance <sup> <a href=\"#ref-CR8\" id=\"ref-link-section-d6776021e865\" title=\"Duan, Z. et al. Signal transducers and activators of transcription 3 pathway activation in drug-resistant ovarian cancer. Clin. Cancer Res. 12, 5055–5063 (2006).\"> 8 </a> , <a href=\"#ref-CR9\" id=\"ref-link-section-d6776021e865_1\" title=\"Huang, W. L. et al. Signal transducer and activator of transcription 3 activation up-regulates interleukin-6 autocrine production: a biochemical and genetic study of established cancer cell lines and clinical isolated human cancer cells. Mol. Cancer 9, 309 (2010).\"> 9 </a> , <a aria-label=\"Reference 10\" href=\"/articles/s41467-024-51831-7#ref-CR10\" id=\"ref-link-section-d6776021e868\" title=\"Lee, H. J. et al. Drug resistance via feedback activation of Stat3 in oncogene-addicted cancer cells. Cancer Cell 26, 207–221 (2014).\"> 10 </a> </sup> . In addition to JAK-STAT3, IL6 activates the ERK, AKT, and SRC-YAP pathways via GP130 to regulate diverse biological processes <sup> <a aria-label=\"Reference 1\" href=\"/articles/s41467-024-51831-7#ref-CR1\" id=\"ref-link-section-d6776021e872\" title=\"Yu, H., Lee, H., Herrmann, A., Buettner, R. &amp; Jove, R. Revisiting STAT3 signalling in cancer: new and unexpected biological functions. Nat. Rev. Cancer 14, 736–746 (2014).\"> 1 </a> , <a aria-label=\"Reference 11\" href=\"/articles/s41467-024-51831-7#ref-CR11\" id=\"ref-link-section-d6776021e875\" title=\"Rokavec, M. et al. IL-6R/STAT3/miR-34a feedback loop promotes EMT-mediated colorectal cancer invasion and metastasis. J. Clin. Invest. 124, 1853–1867 (2014).\"> 11 </a> , <a aria-label=\"Reference 12\" href=\"/articles/s41467-024-51831-7#ref-CR12\" id=\"ref-link-section-d6776021e878\" title=\"Taniguchi, K. et al. A gp130-Src-YAP module links inflammation to epithelial regeneration. Nature 519, 57–62 (2015).\"> 12 </a> </sup> . </p> <p> Targeted therapy using epidermal growth factor receptor-tyrosine kinase inhibitors (EGFR-TKIs) is the standard care for patients with lung adenocarcinoma harboring sensitive EGFR mutations <sup> <a aria-label=\"Reference 13\" href=\"/articles/s41467-024-51831-7#ref-CR13\" id=\"ref-link-section-d6776021e885\" title=\"Quintanal-Villalonga, A. et al. Lineage plasticity in cancer: a shared pathway of therapeutic resistance. Nat. Rev. Clin. Oncol. 17, 360–371 (2020).\"> 13 </a> , <a aria-label=\"Reference 14\" href=\"/articles/s41467-024-51831-7#ref-CR14\" id=\"ref-link-section-d6776021e888\" title=\"Paez, J. G. et al. EGFR mutations in lung cancer: correlation with clinical response to gefitinib therapy. Science 304, 1497–1500 (2004).\"> 14 </a> </sup> . The third-generation EGFR-TKI osimertinib is currently the preferred first-line therapeutic <sup> <a aria-label=\"Reference 15\" href=\"/articles/s41467-024-51831-7#ref-CR15\" id=\"ref-link-section-d6776021e892\" title=\"Ramalingam, S. S. et al. Overall survival with osimertinib in untreated, EGFR-mutated advanced NSCLC. N. Engl. J. Med. 382, 41–50 (2020).\"> 15 </a> </sup> . Although EGFR-TKIs are highly effective in treating this group of patients, acquired resistance inevitably develops in most subjects <sup> <a aria-label=\"Reference 16\" href=\"/articles/s41467-024-51831-7#ref-CR16\" id=\"ref-link-section-d6776021e896\" title=\"Oxnard, G. R. et al. Assessment of resistance mechanisms and clinical implications in patients with EGFR T790M-positive lung cancer and acquired resistance to osimertinib. JAMA Oncol. 4, 1527–1534 (2018).\"> 16 </a> </sup> . Common mechanisms of acquired resistance to EGFR-TKIs include on-target secondary mutations (e.g., T790M and T797S), genetic alterations that result in activation of multiple bypass pathways such as MET, HER2, KRAS, and PIK3CA <sup> <a aria-label=\"Reference 17\" href=\"/articles/s41467-024-51831-7#ref-CR17\" id=\"ref-link-section-d6776021e900\" title=\"Yang, Z. et al. Investigating novel resistance mechanisms to third-generation EGFR tyrosine kinase inhibitor osimertinib in non-small cell lung cancer patients. Clin. Cancer Res. 24, 3097–3107 (2018).\"> 17 </a> </sup> , and histological transformation from adenocarcinomas to aggressive neuroendocrine derivatives <sup> <a aria-label=\"Reference 18\" href=\"/articles/s41467-024-51831-7#ref-CR18\" id=\"ref-link-section-d6776021e904\" title=\"Sequist, L. V. et al. Genotypic and histological evolution of lung cancers acquiring resistance to EGFR inhibitors. Sci. Transl. Med. 3, 75ra26 (2011).\"> 18 </a> </sup> where cancer cells gain cell plasticity and exhibit an <u> e </u> pithelial- <u> m </u> esenchymal <u> t </u> ransition (EMT) phenotype <sup> <a aria-label=\"Reference 19\" href=\"/articles/s41467-024-51831-7#ref-CR19\" id=\"ref-link-section-d6776021e918\" title=\"Lambert, A. W. &amp; Weinberg, R. A. Linking EMT programmes to normal and neoplastic epithelial stem cells. Nat. Rev. Cancer 21, 325–338 (2021).\"> 19 </a> </sup> . </p> <p> Lineage plasticity is proposed as a shared pathway for resistance to anticancer agents, which encompasses phenotypic changes in cancer cells, such as EMT and reprogramming of cancer stem cells <sup> <a aria-label=\"Reference 19\" href=\"/articles/s41467-024-51831-7#ref-CR19\" id=\"ref-link-section-d6776021e925\" title=\"Lambert, A. W. &amp; Weinberg, R. A. Linking EMT programmes to normal and neoplastic epithelial stem cells. Nat. Rev. Cancer 21, 325–338 (2021).\"> 19 </a> , <a aria-label=\"Reference 20\" href=\"/articles/s41467-024-51831-7#ref-CR20\" id=\"ref-link-section-d6776021e928\" title=\"Williams, E. D., Gao, D., Redfern, A. &amp; Thompson, E. W. Controversies around epithelial-mesenchymal plasticity in cancer metastasis. Nat. Rev. Cancer 19, 716–732 (2019).\"> 20 </a> </sup> . EGFR-TKI-induced high IL6 secretion has been observed in lung cancer cell lines <sup> <a aria-label=\"Reference 21\" href=\"/articles/s41467-024-51831-7#ref-CR21\" id=\"ref-link-section-d6776021e932\" title=\"Yao, Z. et al. TGF-beta IL-6 axis mediates selective and adaptive mechanisms of resistance to molecular targeted therapy in lung cancer. Proc. Natl Acad. Sci. USA 107, 15535–15540 (2010).\"> 21 </a> </sup> and clinical samples <sup> <a aria-label=\"Reference 17\" href=\"/articles/s41467-024-51831-7#ref-CR17\" id=\"ref-link-section-d6776021e936\" title=\"Yang, Z. et al. Investigating novel resistance mechanisms to third-generation EGFR tyrosine kinase inhibitor osimertinib in non-small cell lung cancer patients. Clin. Cancer Res. 24, 3097–3107 (2018).\"> 17 </a> </sup> , which facilitates TKI resistance by promoting EMT in a TGF-β-dependent manner <sup> <a aria-label=\"Reference 21\" href=\"/articles/s41467-024-51831-7#ref-CR21\" id=\"ref-link-section-d6776021e940\" title=\"Yao, Z. et al. TGF-beta IL-6 axis mediates selective and adaptive mechanisms of resistance to molecular targeted therapy in lung cancer. Proc. Natl Acad. Sci. USA 107, 15535–15540 (2010).\"> 21 </a> </sup> or inhibiting apoptosis by activating the JAK1-STAT3 pathway <sup> <a aria-label=\"Reference 22\" href=\"/articles/s41467-024-51831-7#ref-CR22\" id=\"ref-link-section-d6776021e944\" title=\"Kim, S. M. et al. Activation of IL-6R/JAK1/STAT signaling induces de novo resistance to irreversible EGFR inhibitors in non-small cell lung cancer with T790M resistance mutation. Mol. Cancer Ther. 11, 2254–2264 (2012).\"> 22 </a> </sup> . IL6 also induces self-renewal of cancer stem cells (CSCs) <sup> <a aria-label=\"Reference 23\" href=\"/articles/s41467-024-51831-7#ref-CR23\" id=\"ref-link-section-d6776021e949\" title=\"Korkaya, H., Liu, S. &amp; Wicha, M. S. Regulation of cancer stem cells by cytokine networks: attacking cancer’s inflammatory roots. Clin. Cancer Res. 17, 6125–6129 (2011).\"> 23 </a> </sup> and dedifferentiation of tumor cells <sup> <a aria-label=\"Reference 24\" href=\"/articles/s41467-024-51831-7#ref-CR24\" id=\"ref-link-section-d6776021e953\" title=\"Senescence-Associated Pro-inflammatory Cytokines, VernotJ. P. and tumor cell plasticity. Front. Mol. Biosci. 7, 63 (2020).\"> 24 </a> </sup> via activation of the GP130-SRC-YAP pathway <sup> <a aria-label=\"Reference 12\" href=\"/articles/s41467-024-51831-7#ref-CR12\" id=\"ref-link-section-d6776021e957\" title=\"Taniguchi, K. et al. A gp130-Src-YAP module links inflammation to epithelial regeneration. Nature 519, 57–62 (2015).\"> 12 </a> </sup> . In EGFR-mutant lung adenocarcinoma, the osimertinib-activated YAP-TEAD complex has been implicated in cellular dormancy and resistance <sup> <a aria-label=\"Reference 25\" href=\"/articles/s41467-024-51831-7#ref-CR25\" id=\"ref-link-section-d6776021e961\" title=\"Kurppa, K. J. et al. Treatment-induced tumor dormancy through YAP-mediated transcriptional reprogramming of the apoptotic pathway. Cancer Cell 37, 104–122.e112 (2020).\"> 25 </a> </sup> . Studies of transcriptional profiling following EGFR-TKI treatment and animal models revealed that SOX2 is a key transcription factor (TF) in promoting neuroendocrine transformation <sup> <a aria-label=\"Reference 26\" href=\"/articles/s41467-024-51831-7#ref-CR26\" id=\"ref-link-section-d6776021e965\" title=\"Ferone, G. et al. SOX2 is the determining oncogenic switch in promoting lung squamous cell carcinoma from different cells of origin. Cancer Cell 30, 519–532 (2016).\"> 26 </a> </sup> and plasticity-reprogramming processes in lung cancer <sup> <a aria-label=\"Reference 27\" href=\"/articles/s41467-024-51831-7#ref-CR27\" id=\"ref-link-section-d6776021e969\" title=\"Boumahdi, S. &amp; de Sauvage, F. J. The great escape: tumour cell plasticity in resistance to targeted therapy. Nat. Rev. Drug Discov. 19, 39–56 (2020).\"> 27 </a> </sup> , in which IL6 <sup> <a aria-label=\"Reference 28\" href=\"/articles/s41467-024-51831-7#ref-CR28\" id=\"ref-link-section-d6776021e974\" title=\"Mosteiro, L. et al. Tissue damage and senescence provide critical signals for cellular reprogramming in vivo. Science 354, 6315 (2016).\"> 28 </a> </sup> or YAP <sup> <a aria-label=\"Reference 29\" href=\"/articles/s41467-024-51831-7#ref-CR29\" id=\"ref-link-section-d6776021e978\" title=\"Bora-Singhal, N. et al. YAP1 regulates OCT4 activity and SOX2 expression to facilitate self-renewal and vascular mimicry of stem-like cells. Stem cells 33, 1705–1718 (2015).\"> 29 </a> </sup> could be inducers. This evidence suggests that the IL6-YAP-SOX2 pathway may be an unrevealed axis that facilitates the development of lineage plasticity and EGFR TKI resistance in lung cancer. However, how IL6 differentially activates the JAK-STAT3 or SRC-YAP-SOX2 axis is not yet clear. </p> <p> Multiple forms of IL6 ranging from 20 to 30 kDa in molecular weight have been observed <sup> <a href=\"#ref-CR30\" id=\"ref-link-section-d6776021e985\" title=\"Poupart, P. et al. B cell growth modulating and differentiating activity of recombinant human 26-kd protein (BSF-2, HuIFN-beta 2, HPGF). EMBO J 6, 1219–1224 (1987).\"> 30 </a> , <a href=\"#ref-CR31\" id=\"ref-link-section-d6776021e985_1\" title=\"May, L. T. et al. Phosphorylation of secreted forms of human beta 2-interferon/hepatocyte stimulating factor/interleukin-6. Biochem. Biophys. Res. Commun. 152, 1144–1150 (1988).\"> 31 </a> , <a href=\"#ref-CR32\" id=\"ref-link-section-d6776021e985_2\" title=\"Hirano, T. et al. Purification to homogeneity and characterization of human B-cell differentiation factor (BCDF or BSFp-2). Proc. Natl Acad. Sci. USA 82, 5490–5494 (1985).\"> 32 </a> , <a aria-label=\"Reference 33\" href=\"/articles/s41467-024-51831-7#ref-CR33\" id=\"ref-link-section-d6776021e988\" title=\"May, L. T., Santhanam, U. &amp; Sehgal, P. B. On the multimeric nature of natural human interleukin-6. J. Biol. Chem. 266, 9950–9955 (1991).\"> 33 </a> </sup> , but no alternatively spliced isoforms have been reported, suggesting that IL6 undergoes cell type-specific post-translational modification (PTM). As the most common PTM, glycosylation <sup> <a aria-label=\"Reference 34\" href=\"/articles/s41467-024-51831-7#ref-CR34\" id=\"ref-link-section-d6776021e992\" title=\"Mereiter, S., Balmana, M., Campos, D., Gomes, J. &amp; Reis, C. A. Glycosylation in the era of cancer-targeted therapy: where are we heading? Cancer Cell 36, 6–16 (2019).\"> 34 </a> </sup> has been observed in human macrophage-derived IL6 <sup> <a aria-label=\"Reference 35\" href=\"/articles/s41467-024-51831-7#ref-CR35\" id=\"ref-link-section-d6776021e996\" title=\"Parekh, R. B., Dwek, R. A., Rademacher, T. W., Opdenakker, G. &amp; Van Damme, J. Glycosylation of interleukin-6 purified from normal human blood mononuclear cells. Eur. J. Biochem. 203, 135–141 (1992).\"> 35 </a> </sup> and Kaposi’s sarcoma-associated herpesvirus-encoded viral IL6 homolog <sup> <a aria-label=\"Reference 36\" href=\"/articles/s41467-024-51831-7#ref-CR36\" id=\"ref-link-section-d6776021e1000\" title=\"Dela Cruz, C. S. et al. N-linked glycosylation is required for optimal function of Kaposi’s sarcoma herpesvirus-encoded, but not cellular, interleukin 6. J. Exp. Med. 199, 503–514 (2004).\"> 36 </a> </sup> . In a Kaposi’s sarcoma model, N-glycosylation is necessary for viral IL6 but not human IL6 to activate JAK-STAT3 signaling <sup> <a aria-label=\"Reference 36\" href=\"/articles/s41467-024-51831-7#ref-CR36\" id=\"ref-link-section-d6776021e1004\" title=\"Dela Cruz, C. S. et al. N-linked glycosylation is required for optimal function of Kaposi’s sarcoma herpesvirus-encoded, but not cellular, interleukin 6. J. Exp. Med. 199, 503–514 (2004).\"> 36 </a> </sup> . Although aberrant protein glycosylation has been observed in various cancer types and shown to enhance or repress cellular signaling <sup> <a aria-label=\"Reference 37\" href=\"/articles/s41467-024-51831-7#ref-CR37\" id=\"ref-link-section-d6776021e1009\" title=\"Thomas, D., Rathinavel, A. K. &amp; Radhakrishnan, P. Altered glycosylation in cancer: a promising target for biomarkers and therapeutics. Biochimica. et biophysica. Acta Rev. Cancer 1875, 188464 (2021).\"> 37 </a> , <a aria-label=\"Reference 38\" href=\"/articles/s41467-024-51831-7#ref-CR38\" id=\"ref-link-section-d6776021e1012\" title=\"Pinho, S. S. &amp; Reis, C. A. Glycosylation in cancer: mechanisms and clinical implications. Nat. Rev. Cancer 15, 540–555 (2015).\"> 38 </a> </sup> , whether the glycosylation status of IL6 in human lung cancer influences its biological functions remains unclear. Likewise, whether EGFR-TKI treatment affects the glycosylation status of proteins in lung cancer cells is unresolved. </p> <p> Because IL6 activates several signaling pathways through interaction with GP130 and glycosylation of IL6 may change its biological functions, we hypothesize that the glycosylation patterns of human IL6 (hIL6) may contribute to its preference for specific downstream signaling pathways. Here we demonstrate that IL6 secreted from human lung cancer cells is modified by N-glycosylation, and that N-glycosylation of IL6 occurs primarily at asparagine (N) 73 and activates the canonical JAK-STAT3 signaling pathway. In contrast, <u> N </u> - <u> g </u> lycosylation- <u> de </u> fective IL6 (deNG-IL6) switches the signaling preference to the SRC-YAP pathway, which facilitates SOX2 expression and enhances EMT, migration, and metastasis. Additionally, the N-glycosylation process is suppressed in osimertinib-treated lung cancer cells, resulting in an increased proportion of deNG-IL6 and the acquisition of cell plasticity and subsequent TKI resistance. </p> </div> </div> </section> <section> <div class=\"c-article-section\" id=\"Sec2-section\"> <h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"Sec2\"> Results </h2> <div class=\"c-article-section__content\" id=\"Sec2-content\"> <h3 class=\"c-article__sub-heading\" id=\"Sec3\"> IL6 Is Modified by N-Glycosylation in NSCLC </h3> <p> We sought to investigate whether IL6 orchestrates multiple signaling pathways via differential glycosylation. Large quantities of IL6 in malignant pleural effusions (MPEs) have been reported in lung cancer patients <sup> <a aria-label=\"Reference 39\" href=\"/articles/s41467-024-51831-7#ref-CR39\" id=\"ref-link-section-d6776021e1041\" title=\"Yeh, H. H., Lai, W. W., Chen, H. H., Liu, H. S. &amp; Su, W. C. Autocrine IL-6-induced Stat3 activation contributes to the pathogenesis of lung adenocarcinoma and malignant pleural effusion. Oncogene 25, 4300–4309 (2006).\"> 39 </a> </sup> ; thus, we isolated lung adenocarcinoma cells from the MPEs (termed “MPECs”) of 15 NSCLC patients to study the glycosylation pattern of lung cancer cell-derived IL6. Short-term cultured MPECs were verified by double-positive expression of the NSCLC markers pancytokeratin (PanCK) and thyroid transcription factor-1 (TTF1) (Fig. <a href=\"/articles/s41467-024-51831-7#Fig1\"> 1a </a> ). IL6 secreted by each MPEC line detected by Western blotting (WB) showed two IL6 bands [upper, <u> h </u> eavy form (H-IL6); lower, <u> l </u> ight form (L-IL6)] (Fig. <a href=\"/articles/s41467-024-51831-7#Fig1\"> 1b </a> ), and the H/L intensity ratios differed among the MPECs. We next measured the amount of IL6 secreted by multiple NSCLC cell lines, including PC14PE6/AS2 (AS2), H1299, A549, H226, HCC827, H1975, H1650, and H460 (Fig. <a href=\"/articles/s41467-024-51831-7#Fig1\"> 1c </a> ), and examined the IL6 pattern in the cell lines with low baseline secretion (H1299) and high baseline secretion (H226, H1975, and H460). Similar to the results for the MPECs, both H-IL6 and L-IL6 were detected with different H/L IL6 ratios (Fig. <a href=\"/articles/s41467-024-51831-7#Fig1\"> 1d </a> ). Since no alternatively spliced IL6 isoforms have been reported, we assumed that the proteins represented by the two bands resulted from the PTM of IL6. </p> <div class=\"c-article-section__figure js-c-reading-companion-figures-item\" id=\"figure-1\"> <figure> <figcaption> <b class=\"c-article-section__figure-caption\" id=\"Fig1\"> Fig. 1: IL6 Is Modified by N-Glycosylation in NSCLC. </b> </figcaption> <div class=\"c-article-section__figure-content\"> <div class=\"c-article-section__figure-item\"> <a class=\"c-article-section__figure-link\" href=\"/articles/s41467-024-51831-7/figures/1\" rel=\"nofollow\"> <picture> <source srcset=\"//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-024-51831-7/MediaObjects/41467_2024_51831_Fig1_HTML.png?as=webp\" type=\"image/webp\"/> </picture> </a> </div> <div class=\"c-article-section__figure-description\" id=\"figure-1-desc\"> <p> <b> a </b> Process of lung cancer cell isolation from malignant pleural effusions (MPEs) from lung cancer patients (generating MPE cells, MPECs). PanCK and TTF1 in MPECs were stained by immunofluorescence (IF). Images are representative of three technical replicates. Scale bar, 20 μm. <b> b </b> Pattern of immunoprecipitated (IP) secreted IL6 in the CM of MPECs (MPEC Nos. 1-12), as determined by Western blotting (WB). <i> n </i> = 1. The same process was used to analyze secreted IL6 throughout the study. <b> c </b> Quantity of total secreted IL6 in the NSCLC cell lines as measured by ELISAs. <i> n </i> = 3 technical replicates, mean ± SEM. <b> d </b> Patterns of secreted IL6 of NSCLC cell lines. <i> n </i> = 3 independent experiments, mean ± SEM. <b> e </b> Gene set enrichment analysis (GSEA) enrichment plot of N-linked glycosylation in cancerous lung tissue vs. normal lung tissue in the dataset GSE31210. NES, Nominal Enrichment Score. Nominal <i> p- </i> value was one-tailed test on the appropriate side of the null distribution. <b> f </b> Pattern of secreted IL6 in H460 cells ( <i> n </i> = 3 independent experiments, mean ± SEM) and MPECs (MPEC No. 13, <i> n </i> = 1) treated with or without NGI-1 (10 μM) for 48 h. <b> g </b> Pattern of secreted IL6 in MPECs (MPEC Nos. 14 and 15) with or without PNGase F digestion ( <i> n </i> = 1). <b> h </b> <i> IL6 </i> mRNA levels in the AS2 and HCC827 cells stably expressing full-length WT-hIL6 (namely, AS2-IL6-WT and HCC827-IL6-WT cells), as measured by RT-PCR. <i> GAPDH </i> was the internal control. <b> i </b> Quantity of total IL6 in the CM of AS2-IL6-WT and HCC827-IL6-WT cells, as measured by ELISAs ( <i> n </i> = 3 technical replicates, mean ± SEM). <b> j </b> Pattern of secreted IL6 in AS2-IL6-WT and HCC827-IL6-WT cells. The percentage proportions of H-IL6 and L-IL6 were quantified in the segmented histogram. <i> n </i> = 3 independent experiments (mean ± SEM). <b> k </b> Effect of NGI-1 (10 μM) on the pattern of secreted IL6 in the CM of AS2-IL6-WT and HCC827-IL6-WT cells. <i> n </i> = 3 independent experiments (mean ± SEM). <b> l </b> Quantitative characterization of N-glycans on purified secreted WT-IL6 from CM of AS2-IL6-WT cells by lectin microarray, as assessed with GlycoStation <sup> TM </sup> Reader 1200 (GlycoTechnica). The raw data was calculated and output automatically by GlycoStation Tools Pro Suite 1.5 software. <i> n </i> = 3 technical replicates, mean ± S.D. See also Supplementary Table <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 1 </a> . <b> m </b> Quantitative characterization of N-glycans on purified secreted WT-IL6 from CM of AS2-IL6-WT cells treated with NGI-1 (10 μM) by lectin microarray, as assessed with GlycoStation <sup> TM </sup> Reader 1200 (GlycoTechnica). The raw data was calculated and output automatically by GlycoStation Tools Pro Suite 1.5 software. <i> n </i> = 3 technical replicates, mean ± S.D. See also Supplementary Table <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 1 </a> . Source data are provided as a Source Data file. </p> </div> </div> <div class=\"u-text-right u-hide-print\"> <a aria-label=\"Full size image figure 1\" class=\"c-article__pill-button\" href=\"/articles/s41467-024-51831-7/figures/1\" rel=\"nofollow\"> <span> Full size image </span> </a> </div> </figure> </div> <p> N-glycosylation is the most common PTM of secreted proteins <sup> <a aria-label=\"Reference 40\" href=\"/articles/s41467-024-51831-7#ref-CR40\" id=\"ref-link-section-d6776021e1182\" title=\"Roth, J. Protein N-glycosylation along the secretory pathway: relationship to organelle topography and function, protein quality control, and cell interactions. Chem. Rev. 102, 285–303 (2002).\"> 40 </a> , <a aria-label=\"Reference 41\" href=\"/articles/s41467-024-51831-7#ref-CR41\" id=\"ref-link-section-d6776021e1185\" title=\"Bard, F. &amp; Chia, J. Cracking the glycome encoder: signaling, trafficking, and glycosylation. Trends Cell Biol. 26, 379–388 (2016).\"> 41 </a> </sup> , and elevated protein N-glycosylation has been reported in various cancers <sup> <a aria-label=\"Reference 42\" href=\"/articles/s41467-024-51831-7#ref-CR42\" id=\"ref-link-section-d6776021e1189\" title=\"Schjoldager, K. T., Narimatsu, Y., Joshi, H. J. &amp; Clausen, H. Global view of human protein glycosylation pathways and functions. Nat. Rev. Mol. Cell Biol. 21, 729–749 (2020).\"> 42 </a> </sup> . We then analyzed the expression trend of N-linked glycosylation process-related genes by <u> g </u> ene <u> s </u> et <u> e </u> nrichment <u> a </u> nalysis (GSEA) <sup> <a aria-label=\"Reference 43\" href=\"/articles/s41467-024-51831-7#ref-CR43\" id=\"ref-link-section-d6776021e1206\" title=\"Subramanian, A. et al. Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. Proc. Natl Acad. Sci. USA 102, 15545–15550 (2005).\"> 43 </a> , <a aria-label=\"Reference 44\" href=\"/articles/s41467-024-51831-7#ref-CR44\" id=\"ref-link-section-d6776021e1209\" title=\"Mootha, V. K. et al. PGC-1alpha-responsive genes involved in oxidative phosphorylation are coordinately downregulated in human diabetes. Nat. Genet. 34, 267–273 (2003).\"> 44 </a> </sup> of the Gene Expression Omnibus (GEO) dataset GSE31210 <sup> <a aria-label=\"Reference 45\" href=\"/articles/s41467-024-51831-7#ref-CR45\" id=\"ref-link-section-d6776021e1213\" title=\"Okayama, H. et al. Identification of genes upregulated in ALK-positive and EGFR/KRAS/ALK-negative lung adenocarcinomas. Cancer Res. 72, 100–111 (2012).\"> 45 </a> </sup> (Fig. <a href=\"/articles/s41467-024-51831-7#Fig1\"> 1e </a> ). Genes involved in the N-linked glycosylation process were enriched in cancer tissues; thus, we hypothesized that H-IL6 arises from N-linked glycosylation. Notably, treatment with the N-glycosylation inhibitor NGI-1 <sup> <a aria-label=\"Reference 46\" href=\"/articles/s41467-024-51831-7#ref-CR46\" id=\"ref-link-section-d6776021e1220\" title=\"Lopez-Sambrooks, C. et al. Oligosaccharyltransferase inhibition induces senescence in RTK-driven tumor cells. Nat. Chem. Biol. 12, 1023–1030 (2016).\"> 46 </a> </sup> diminished the amount of secreted H-IL6 from H460 cells and MPECs (Fig. <a href=\"/articles/s41467-024-51831-7#Fig1\"> 1f </a> ). When total MPEC-derived IL6 was treated with peptide-N-glycosidase F (PNGase F), which cleaves entire N-glycan chains from corresponding asparagine residues, the amount of H-IL6 was decreased (Fig. <a href=\"/articles/s41467-024-51831-7#Fig1\"> 1g </a> ). In addition, two N-glycosites consisting of the Asn-Xaa-Ser/Thr sequence were predicted in the hIL6 protein sequence by the NetNGlyc 1.0 server ( <a href=\"http://www.cbs.dtu.dk/services/NetNGlyc/\"> http://www.cbs.dtu.dk/services/NetNGlyc/ </a> ) <sup> <a aria-label=\"Reference 47\" href=\"/articles/s41467-024-51831-7#ref-CR47\" id=\"ref-link-section-d6776021e1238\" title=\"Blom, N., Sicheritz-Ponten, T., Gupta, R., Gammeltoft, S. &amp; Brunak, S. Prediction of post-translational glycosylation and phosphorylation of proteins from the amino acid sequence. Proteomics 4, 1633–1649 (2004).\"> 47 </a> </sup> (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 1 </a> ): asparagine (N) 73 and N172 (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 1 </a> , top). The score for only N73 exceeded the threshold score of 0.5 (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 1 </a> , bottom), indicating its potential as an N-glycosite. In addition, four O-glycosites (T166, T170, T171, and S174) were also predicted in IL6 (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 2 </a> and Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2a </a> ); however, the MS data (Fig. <a href=\"/articles/s41467-024-51831-7#Fig6\"> 6l </a> and Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 22 </a> ) provided evidence that, actually, O-glycan was detected only on T170, which existed in over 99% of secreted IL6 molecules. </p> <div class=\"c-article-section__figure js-c-reading-companion-figures-item\" id=\"figure-2\"> <figure> <figcaption> <b class=\"c-article-section__figure-caption\" id=\"Fig2\"> Fig. 2: N-glycosylated IL6 promotes the tyrosine phosphorylation and nuclear retention of STAT3. </b> </figcaption> <div class=\"c-article-section__figure-content\"> <div class=\"c-article-section__figure-item\"> <a class=\"c-article-section__figure-link\" href=\"/articles/s41467-024-51831-7/figures/2\" rel=\"nofollow\"> <picture> <source srcset=\"//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-024-51831-7/MediaObjects/41467_2024_51831_Fig2_HTML.png?as=webp\" type=\"image/webp\"/> </picture> </a> </div> <div class=\"c-article-section__figure-description\" id=\"figure-2-desc\"> <p> <b> a </b> Graphical depiction of N- and O- glycosites on the full-length human IL6 (hIL6) peptide. <b> b </b> Patterns of IL6 with various mutations in N-glycosites. Secreted WT-IL6, N73Q-IL6, N172Q-IL6, and N73Q + N172Q-IL6 were quantified and immunoprecipitated, and the pattern of IL6 was analyzed on the same WB with rhIL6 expressed in <i> E. coli </i> . NG-IL6: <u> N </u> - <u> g </u> lycosylated IL6; deNG-IL6: <u> N </u> - <u> g </u> lycosylation- <u> de </u> fective IL6; deG-IL6: <u> g </u> lycosylation- <u> de </u> ficient IL6. The stick diagram connecting to each form of IL6 band labeled by dotted line showed the locations of glycan chains on N-glycosites and O-glycosites. <i> n </i> = 3 independent experiments, mean ± SEM, two-tailed unpaired <i> t </i> -test, *** <i> p </i> &lt; 0.0001, * <i> p </i> = 0.0182, *** <i> p </i> &lt; 0.0001. <b> c </b> Patterns of secreted IL6 in AS2-IL6-WT, AS2-IL6-N73Q, AS2-IL6-N172Q, and AS2-IL6-N73Q + N172Q cells with or without NGI-1 (10 μM) pretreatment for 48 h. <i> n </i> = 3 independent experiments, mean ± SEM, two-tailed unpaired <i> t </i> -test, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001, n.s. <i> p </i> = 0.3423, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001. <b> d </b> Quantitative characterization of N-glycans on purified secreted WT-IL6 and N73Q-IL6 by lectin microarray, as assessed with GlycoStation <sup> TM </sup> Reader 1200 (GlycoTechnica). The raw data was calculated and output automatically by GlycoStation Tools Pro Suite 1.5 software. <i> n </i> = 3 technical replicates, mean ± S.D. See also Supplementary Table <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 1 </a> . <b> e </b> STAT3 and STAT3-pY705 levels induced by WT-IL6-CM or N73Q-IL6-CM (both IL6 = 1 ng/mL) over time in AS2-Vec, Beas-2B, and NL-20 cells. <i> n </i> = 3 independent experiments, mean ± SEM, two-tailed unpaired <i> t </i> -test, AS2-Vec: * <i> p </i> = 0.0139, ** <i> p </i> = 0.0282, n.s. 0.3663, ***0.0001; NL20: **0.0029, ***0.0004, **0.0014, **0.0010; Beas-2B: **0.0054, n.s. 0.9142, ***0.0006, ***&lt;0.0001. The samples derived from the same experiment but different gels for STAT3 and STAT3-pY705 were processed in parallel. <b> f </b> STAT3 intracellular localization under treatment with WT-IL6-CM or N73Q-IL6-CM (both IL6 = 1 ng/mL) over time in AS2-Vec cells, as determined by IF. The magnified insets show STAT3 localization at 24 h. Scale bars, 20 μm. <i> n </i> = 3 independent experiments, mean ± SEM, two-tailed unpaired <i> t </i> -test, ** <i> p </i> = 0.0012, *** <i> p </i> = 0.0002, n.s. <i> p </i> = 0.0939, *** <i> p </i> = 0.0001. <b> g </b> Transcriptome of AS2-Vec cells treated with WT-IL6-CM or N73Q-IL6-CM over time (see also Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 10 </a> ). Hierarchical clustering showed differentially expressed genes at 3 h. Source data are provided as a Source Data file. </p> </div> </div> <div class=\"u-text-right u-hide-print\"> <a aria-label=\"Full size image figure 2\" class=\"c-article__pill-button\" href=\"/articles/s41467-024-51831-7/figures/2\" rel=\"nofollow\"> <span> Full size image </span> </a> </div> </figure> </div> <p> Moreover, we transfected AS2 and HCC827 cells (which produce relatively low amounts of IL6) with a plasmid harboring human wild-type IL6 (WT-IL6), establishing AS2-IL6-WT and HCC827-IL6-WT cells. In these cells, IL6 increased at both the mRNA (Fig. <a href=\"/articles/s41467-024-51831-7#Fig1\"> 1h </a> ) and protein levels (Fig. <a href=\"/articles/s41467-024-51831-7#Fig1\"> 1i </a> ), and both H-IL6 and L-IL6 were detected (Fig. <a href=\"/articles/s41467-024-51831-7#Fig1\"> 1j </a> ), which was then verified by MALDI-TOF (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 3 </a> ). NGI-1 treatment reduced the levels of H-IL6 in both cell lines (Fig. <a href=\"/articles/s41467-024-51831-7#Fig1\"> 1k </a> ), confirming the existence of N-glycosylation. IL6 isolated from AS2-IL6-WT cells (termed “WT-IL6”) was subjected to lectin microarray analysis. Significant signals were detected for N-glycan-binding lectins, including PSA, LCA, ECA, ConA, etc., suggesting that WT-IL6 carries N-glycan structures (Fig. <a href=\"/articles/s41467-024-51831-7#Fig1\"> 1l </a> and Supplementary Table <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 1 </a> ). These signals were greatly diminished for WT-IL6 isolated from NGI-1-pretreated AS2-IL6-WT cells (Fig. <a href=\"/articles/s41467-024-51831-7#Fig1\"> 1m </a> ), confirming the existence of N-glycosylation in lung cancer cell-derived IL6. </p> <h3 class=\"c-article__sub-heading\" id=\"Sec4\"> N-glycosylated IL6 promotes the tyrosine phosphorylation and nuclear retention of STAT3 </h3> <p> Since IL6 is modified by N-glycosylation in lung cancer, we sought to explore its possible functional roles. To determine the dominant N-glycosite (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 1 </a> , Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 2 </a> , and Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2a </a> ), we generated three mutant IL6 plasmids by site-directed mutagenesis, replacing asparagine with glutamine <sup> <a aria-label=\"Reference 48\" href=\"/articles/s41467-024-51831-7#ref-CR48\" id=\"ref-link-section-d6776021e1463\" title=\"Yamaguchi, K. et al. Effects of site-directed removal of N-glycosylation sites in human erythropoietin on its production and biological properties. J. Biol. Chem. 266, 20434–20439 (1991).\"> 48 </a> </sup> , corresponding to the N-glycosylation-defective IL6s, namely, N73Q-IL6, N172Q-IL6 and N73Q + N172Q-IL6 (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 4a </a> ). Concurrently, AS2-IL6-N73Q cells, AS2-IL6-N172Q cells, and AS2-IL6-N73Q + N172Q cells were established. Secreted WT-IL6, N73Q-IL6, N172Q-IL6, and N73Q + N172Q-IL6 were quantified (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 4b, c </a> ), immunoprecipitated, and examined by WB, along with <u> r </u> ecombinant <u> h </u> uman IL6 (rhIL6) expressed in <i> E. coli </i> (Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2b </a> ). The Western blot for rhIL6 showed a single band representing the non-glycosylated IL6, designated as <b> deG-IL6 </b> ( <u> g </u> lycosylation- <u> de </u> ficient IL6). The blot for WT-IL6 form contained two major bands, one strong L-IL6 band and one weak H-IL6 band (Figs. <a href=\"/articles/s41467-024-51831-7#Fig1\"> 1j </a> and <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2b </a> ), in addition to a faint deG-IL6 band. Compared with the Western blot for WT-IL6, the blot for N73Q-IL6 form lost the H-IL6 band by over 97% (Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2b </a> ), but the results for N172Q-IL6 form exhibited only a slight reduction in the abundance of the H-IL6 band (Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2b </a> ). N172Q-IL6 form exhibited four bands on the Western blot: H-IL6, L-IL6, deG-IL6, and an additional faint band (labeled with an asterisk) between the H-IL6 and L-IL6 bands (Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2b </a> ). The pattern of N73Q + N172Q-IL6 form was similar to that of N73Q-IL6 form. These results showed that the glycoform of H-IL6 band carried only one N-glycan primarily located at N73, and such N-glycosylation is absent in L-IL6 band. Besides, there was no obvious IL6 band detected at a higher molecular weight (MW) than that of the H-IL6 band (Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2b </a> and Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 4d </a> ), suggesting that double N-glycosylated form at N73 and N172 may not exist in our IL6 samples. The stick diagram connected to each form of the IL6 band illustrated the actual locations of glycan chains on N-glycosites and O-glycosites (Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2b </a> ), as both H-IL6 and L-IL6 contained an O-glycan at T170, in accordance with the MS analysis (Fig. <a href=\"/articles/s41467-024-51831-7#Fig6\"> 6l </a> and Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 22 </a> ). </p> <p> In addition, treatment with NGI-1 (10 μM) significantly reduced the abundance of the H-IL6 band (Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2c </a> and Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 4d </a> ). The patterns of IL6 isolated from AS2-IL6-WT cells pretreated with NGI-1 or treated with PNGase F (Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2c </a> and Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 4d </a> ) in vitro were similar to those of IL6 isolated from untreated AS2-N73Q-IL6 cells. In comparison, the pattern of IL6 isolated from AS2-IL6-N73Q cells was not changed by NGI-1 or PNGase F treatment. Furthermore, the N-glycan intensity of N73Q-IL6 in the lectin microarray analysis was obviously reduced by ~50%, as compared to that of WT-IL6 (Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2d </a> ). </p> <p> Henceforth, we propose a nomenclature for each IL6 band based on its molecular weight position as depicted on the Western blot (from top to bottom), as follows: The H-IL6 band in Fig. <a href=\"/articles/s41467-024-51831-7#Fig1\"> 1 </a> , redefined in Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2 </a> , possesses the highest molecular weight and is designated as <b> NG-IL6 </b> for <u> N </u> - <u> g </u> lycosylated IL6. The majority of NG-IL6 glycoform carries glycans on N73 + T170, while a minority carries glycans on N172 + T170, albeit rarely detected. The L-IL6 band in Fig. <a href=\"/articles/s41467-024-51831-7#Fig1\"> 1 </a> , redefined in Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2 </a> , is termed <b> deNG-IL6 </b> for <u> N </u> - <u> g </u> lycosylation- <u> de </u> fective IL6, as the glycan chain resided solely on T170. Additionally, the IL6 band with the smallest molecular weight (Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2b, c </a> ) is named <b> deG-IL6 </b> ( <u> g </u> lycosylation- <u> de </u> ficient IL6), indicating the absence of both N- and O-glycans. This form was observed in various glycosylation-defective IL6 variants (N73Q-IL6, N172Q-IL6, N73Q + N172Q-IL6) and rhIL6 from <i> E. coli </i> . </p> <p> N-glycosylation modulates the functions of glycoproteins <sup> <a aria-label=\"Reference 42\" href=\"/articles/s41467-024-51831-7#ref-CR42\" id=\"ref-link-section-d6776021e1601\" title=\"Schjoldager, K. T., Narimatsu, Y., Joshi, H. J. &amp; Clausen, H. Global view of human protein glycosylation pathways and functions. Nat. Rev. Mol. Cell Biol. 21, 729–749 (2020).\"> 42 </a> </sup> . Based on protein sequence alignment (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 5a </a> ) and the 3-D structure of hIL6 (1ALU, <a href=\"https://www.rcsb.org/structure/1ALU\"> https://www.rcsb.org/structure/1ALU </a> ) and the IL6-GP80-GP130 hexameric complex (1P9M, <a href=\"https://www.rcsb.org/structure/1P9M\"> https://www.rcsb.org/structure/1P9M </a> ) in the Protein Data Bank <sup> <a aria-label=\"Reference 49\" href=\"/articles/s41467-024-51831-7#ref-CR49\" id=\"ref-link-section-d6776021e1622\" title=\"Bobby, R. et al. Functional implications of large backbone amplitude motions of the glycoprotein 130-binding epitope of interleukin-6. FEBS J. 281, 2471–2483 (2014).\"> 49 </a> </sup> , we found that the N73 residue was in space near both K47 (K74 in full-length IL6) in the AB loop and K151 (K178 in full-length IL6) in the E-helix within site III of hIL6 <sup> <a aria-label=\"Reference 49\" href=\"/articles/s41467-024-51831-7#ref-CR49\" id=\"ref-link-section-d6776021e1627\" title=\"Bobby, R. et al. Functional implications of large backbone amplitude motions of the glycoprotein 130-binding epitope of interleukin-6. FEBS J. 281, 2471–2483 (2014).\"> 49 </a> </sup> , which is located in the IL6-GP130 interface and thus essential for ligand-receptor binding (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 5b </a> ). Hence, we hypothesized that the N-glycosylation of IL6 at N73 affects the interaction of IL6 with GP130. </p> <p> Because we could not obtain sufficient amounts of either bioactive NG-IL6 or deNG-IL6, we used <u> c </u> onditioned <u> m </u> edium (CM) from AS2-IL6-WT cells (WT-IL6-CM) or AS2-IL6-N73Q cells (N73Q-IL6-CM) to treat AS2-Vec cells (transfected with empty vector), normal human bronchial NL-20 cells and Beas-2B cells for different durations. Before treatment, the IL6 content in each CM was measured by an ELISA (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 6a </a> ) and adjusted to 1 ng/mL (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 6b </a> ). In addition, the cytokine array results demonstrated that the constituents and quantity of cytokines contained in both undiluted and diluted WT-IL6-CM and N73Q-IL6-CM were nearly the same and the differences were negligible (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 6c, d </a> ); thus, the observed biological changes are likely induced primarily by different N-glycosylations of IL6. </p> <p> WT-IL6-CM-induced STAT3-pY705 was clearly retained for 24 h but not that of N73Q-IL6-CM, in both cancerous and normal cells (Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2e </a> ). Time-dependent IF imaging revealed that treatment with either WT-IL6-CM or N73Q-IL6-CM stimulated the nuclear import of STAT3 at 0.5 h (Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2f </a> ). Notably, WT-IL6-CM induced significantly longer nuclear retention of STAT3 than N73Q-IL6-CM up to 24 h (Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2g </a> , magnified inset). The difference in the STAT3 nuclear retention time was concordant with the persistence of the active STAT3 signal observed by WB (Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2e </a> ), showing different dynamics of STAT3 activation induced by the two forms of IL6. The amount of active STAT3 at 0.5 h was reduced by neutralizing antibodies against GP130 (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 7 </a> ), GP80 (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 8 </a> ), and IL6 (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 9 </a> ), indicating that the IL6-GP80-GP130 axis mediates WT-IL6-CM- and N73Q-IL6-CM-triggered STAT3 signaling. </p> <p> To assess the effects of differential STAT3 activation on transcriptional outcomes, we assessed changes in the transcriptome of the AS2-Vec cells induced by WT-IL6-CM or N73Q-IL6-CM from 0.5 to 24 h by microarray. Genes with a <u> f </u> old <u> c </u> hange (FC) in expression &gt;1.5 after normalization to the control medium treatment value were clustered with GeneSpring software (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 10 </a> and Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2g </a> ). The heatmap showed that N73Q-IL6-CM triggered markedly different gene expression profiles with enhanced intensity, particularly at 3 h, compared to WT-IL6-CM (Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2g </a> ). These results are not fully explained by dynamic changes in STAT3 activation, suggesting that deNG-IL6 induces pathways other than canonical JAK-STAT3 signaling to change the transcriptomic profiles. </p> <h3 class=\"c-article__sub-heading\" id=\"Sec5\"> deNG-IL6 induces gene signatures of EMT </h3> <p> To explore the functional effects of deNG-IL6 on cell behaviors, we analyzed genes with N73Q-IL6-CM-induced differential expression with <u> I </u> ngenuity <u> P </u> athway <u> A </u> nalysis (IPA) software. IPA-built-in “Disease or Functional Annotation” (Fig. <a href=\"/articles/s41467-024-51831-7#Fig3\"> 3a </a> ) predicted that the N73Q-IL6-CM-induced genes increased cell movement, migration, invasion, and EMT. Hierarchical clustering with GeneSpring showed that the expression of the top-ranked involved molecules was markedly increased by N73Q-IL6-CM, especially at 3 h (Fig. <a href=\"/articles/s41467-024-51831-7#Fig3\"> 3b </a> ), revealing that these genes may primarily contribute to metastasis. To further elucidate the functions of NG-IL6 and deNG-IL6, we analyzed the WT-IL6-CM- and N73Q-IL6-CM-induced transcriptomes by GSEA (Fig. <a href=\"/articles/s41467-024-51831-7#Fig3\"> 3c </a> ). N73Q-IL6-CM induced an enriched expression of hallmark EMT-related genes compared to WT-IL6-CM. </p> <div class=\"c-article-section__figure js-c-reading-companion-figures-item\" id=\"figure-3\"> <figure> <figcaption> <b class=\"c-article-section__figure-caption\" id=\"Fig3\"> Fig. 3: deNG-IL6 Induces EMT, Migration, and Metastasis in NSCLC. </b> </figcaption> <div class=\"c-article-section__figure-content\"> <div class=\"c-article-section__figure-item\"> <a class=\"c-article-section__figure-link\" href=\"/articles/s41467-024-51831-7/figures/3\" rel=\"nofollow\"> <picture> <source srcset=\"//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-024-51831-7/MediaObjects/41467_2024_51831_Fig3_HTML.png?as=webp\" type=\"image/webp\"/> </picture> </a> </div> <div class=\"c-article-section__figure-description\" id=\"figure-3-desc\"> <p> <b> a </b> IPA-identified “Disease and Functional Annotation” in the N73Q-IL6-CM-treated cells. A significant correlation was defined as <i> p </i> &lt; 0.001, evaluated by a right-tailed Fisher’s Exact Test. <b> b </b> Hierarchical clustering of the IPA-enriched EMT/metastasis-related genes. <b> c </b> GSEA enrichment plot of EMT process-related genes in the AS2-Vec cells under WT-IL6-CM and N73Q-IL6-CM treatment. Nominal <i> p </i> -value was one-tailed test on the appropriate side of the null distribution. <b> d </b> The morphology of (bright field, left panel, scale bars, 200 μm) and F-ACTIN stress fibers (IF, right panel, scale bars, 20 μm) in AS2-Vec, Beas-2B, and NL-20 cells under WT-IL6-CM or N73Q-IL6-CM treatment (24 h) are shown. F-ACTIN was stained with phalloidin-TRITC (gray) and detected with an FV3000 microscope (60x). <b> e </b> Expression of EMT-associated proteins in AS2-Vec cells treated with WT-IL6-CM or N73Q-IL6-CM for 24 h. <i> n </i> = 3 independent experiments, mean ± SEM, two-tailed unpaired <i> t </i> -test, E-CADHERIN: *** <i> p </i> = 0.0005; CLAUDIN1: *** <i> p </i> &lt; 0.0001; SLUG: *** <i> p </i> &lt; 0.0001; N-CADHERIN: *** <i> p </i> &lt; 0.0001; α-SMA: *** <i> p </i> &lt; 0.0001; β-ACTIN: n.s. <i> p </i> = 0.9997. The samples derived from the same experiment but different gels for E-CADHERIN, CLAUDIN1, SLUG, N-CADHERIN, α-SMA, and β-ACTIN were processed in parallel. <b> f </b> Stress fibers in the AS2-Vec cells under N73Q-IL6-CM treatment (24 h) with or without α-GP130 neutralizing Ab pretreatment (2 h). Images are representative of three independent experiments. Scale bars, 20 μm. See also Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 12 </a> . <b> g </b> Expression of mesenchymal proteins in AS2-Vec cells under N73Q-IL6-CM treatment (24 h) with α-GP80, α-IL6, and α-GP130 neutralizing Ab pretreatment for 2 h. <i> n </i> = 3 independent experiments, mean ± SEM, two-tailed unpaired <i> t </i> -test, SLUG: *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001; N-CADHERIN: *** <i> p </i> &lt; 0.0001, *** <i> p </i> = 0.0001, *** <i> p </i> &lt; 0.0001; α-SMA: *** <i> p </i> = 0.0001, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001; β-ACTIN: n.s. <i> p </i> = 0.9926, n.s. <i> p </i> = 0.6517, n.s. <i> p </i> = 0.8847. The samples derived from the same experiment but different gels for SLUG, N-CADHERIN, α-SMA, and β-ACTIN were processed in parallel. <b> h </b> Migration of AS2-Vec, AS2-IL6-WT, and AS2-IL6- N73Q cells through Transwell membranes <i> n </i> = 3 independent experiments, mean ± SEM, two-tailed unpaired <i> t </i> -test, *** <i> p </i> = 0.0001. <b> i </b> Migration of CL1-0 cells in a Transwell coculture system cocultured with AS2-Vec, AS2-IL6-WT, and AS2-IL6-N73Q cells. <i> n </i> = 3 independent experiments, the percentage of migrated CL1-0 cells is presented as the mean ± SEM, two-tailed unpaired <i> t </i> -test, *** <i> p </i> &lt; 0.0001. <b> j </b> In vivo metastasis of AS2-Vec, AS2-IL6-WT, and AS2-IL6-N73Q cells injected intravenously into nude mice ( <i> n </i> = 3 per cell type, 3 repeated independent experiments; see also Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 15 </a> ) evaluated by gross anatomical examination (left) and TissueFAXS-imaged H&amp;E staining (right). Each replicate experiment was terminated upon the death of the first animal; the necrotic lungs in these mice are indicated by dashed circles and were not used in further experiments. <b> k </b> The tumor area over the total lung area from ( <b> J </b> ), as quantified by HistoQuest. <i> n </i> = 3 independent animals for each glycoform in each experiment, 3 independent experiments, mean ± SEM, two-way ANOVA. <b> l </b> In vivo metastasis of AS2-IL6-WT-pLuc and AS2-IL6-N73Q-pLuc cells in nude mice after i.v. injection ( <i> n </i> = 3 per cell type) detected with IVIS. Source data are provided as a Source Data file. </p> </div> </div> <div class=\"u-text-right u-hide-print\"> <a aria-label=\"Full size image figure 3\" class=\"c-article__pill-button\" href=\"/articles/s41467-024-51831-7/figures/3\" rel=\"nofollow\"> <span> Full size image </span> </a> </div> </figure> </div> <p> We also characterized differentially expressed proteins in the AS2-IL6-WT and AS2-IL6-N73Q cells by two-dimensional fluorescence difference gel electrophoresis (2D-DIGE) (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 11a </a> ). Four spots indicating proteins differentially overexpressed in the AS2-IL6-N73Q cells were identified. Spots 1 and 2 were mainly composed of cytoskeleton-related proteins (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 11b </a> ), while a few mitochondrial proteins were characterized in spots 3 and 4. </p> <h3 class=\"c-article__sub-heading\" id=\"Sec6\"> deNG-IL6 promotes EMT, migration and metastasis in NSCLC </h3> <p> To evaluate how the N-glycosylation status of IL6 influences the EMT behavior of stimulated cells, we treated AS2-Vec, NL-20, and Beas-2B cells with WT-IL6-CM or N73Q-IL6-CM for 24 h, and phalloidin-stained ACTIN filaments (F-ACTIN) were observed by confocal microscopy. N73Q-IL6-CM induced a spindle-like morphology in all three cell types (Fig. <a href=\"/articles/s41467-024-51831-7#Fig3\"> 3d </a> ). In the WT-IL6-CM-treated cells, the majority of ACTIN filaments (F-ACTIN) delineated intercellular tight junctions, and few F-ACTIN stress fibers were found within the cytosol. In contrast, an increased number of stress fibers across the entire cytosolic compartment was observed in the N73Q-IL6-CM-treated cells (Fig. <a href=\"/articles/s41467-024-51831-7#Fig3\"> 3d </a> , arrowheads), indicating that N-glycosylation-defective N73Q-IL6 possesses a strong EMT-promoting capacity. </p> <p> We surveyed the expression of IL6-regulated EMT-related proteins in the AS2-Vec cells treated with WT-IL6-CM or N73Q-IL6-CM. Marked decreases in the levels of the epithelial proteins E-CADHERIN <sup> <a aria-label=\"Reference 50\" href=\"/articles/s41467-024-51831-7#ref-CR50\" id=\"ref-link-section-d6776021e1919\" title=\"Xiong, H. et al. Roles of STAT3 and ZEB1 proteins in E-cadherin down-regulation and human colorectal cancer epithelial-mesenchymal transition. J. Biol. Chem. 287, 5819–5832 (2012).\"> 50 </a> </sup> and CLAUDIN1 <sup> <a aria-label=\"Reference 51\" href=\"/articles/s41467-024-51831-7#ref-CR51\" id=\"ref-link-section-d6776021e1923\" title=\"Wei, W. et al. STAT3 signaling is activated preferentially in tumor-initiating cells in claudin-low models of human breast cancer. Stem cells 32, 2571–2582 (2014).\"> 51 </a> </sup> and increases in the levels of the mesenchymal proteins SLUG <sup> <a aria-label=\"Reference 52\" href=\"/articles/s41467-024-51831-7#ref-CR52\" id=\"ref-link-section-d6776021e1927\" title=\"Yao, C. et al. IGF/STAT3/NANOG/Slug signaling axis simultaneously controls epithelial-mesenchymal transition and stemness maintenance in colorectal cancer. Stem Cells 34, 820–831 (2016).\"> 52 </a> </sup> , N-CADHERIN <sup> <a aria-label=\"Reference 53\" href=\"/articles/s41467-024-51831-7#ref-CR53\" id=\"ref-link-section-d6776021e1931\" title=\"Na, Y. R., Lee, J. S., Lee, S. J. &amp; Seok, S. H. Interleukin-6-induced twist and N-cadherin enhance melanoma cell metastasis. Melanoma Res. 23, 434–443 (2013).\"> 53 </a> </sup> , and α-SMA <sup> <a aria-label=\"Reference 54\" href=\"/articles/s41467-024-51831-7#ref-CR54\" id=\"ref-link-section-d6776021e1935\" title=\"Gallucci, R. M., Lee, E. G. &amp; Tomasek, J. J. IL-6 modulates alpha-smooth muscle actin expression in dermal fibroblasts from IL-6-deficient mice. J. Invest. Dermatol. 126, 561–568 (2006).\"> 54 </a> </sup> were observed in the N73Q-IL6-CM-treated cells (Fig. <a href=\"/articles/s41467-024-51831-7#Fig3\"> 3e </a> ). Furthermore, blockade of GP130 with a neutralizing Ab reduced N73Q-IL6-CM-induced stress fiber formation (Fig. <a href=\"/articles/s41467-024-51831-7#Fig3\"> 3f </a> and Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 12 </a> , arrowheads). Blockade of GP130 and GP80, as well as IL6 neutralization, effectively reduced the expression of mesenchymal proteins, including SLUG, N-CADHERIN, and α-SMA (Fig. <a href=\"/articles/s41467-024-51831-7#Fig3\"> 3g </a> and Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 13 </a> ). </p> <p> We next examined the migration of AS2-Vec, AS2-IL6-WT, and AS2-IL6-N73Q cells by Transwell assays. The AS2-IL6-N73Q cells demonstrated the strongest migratory capacity, followed by the AS2-IL6-WT cells and the AS2-Vec cells (Fig. <a href=\"/articles/s41467-024-51831-7#Fig3\"> 3h </a> ), indicating that IL6 and, to a greater extent, deNG-IL6 enhance migration in an autocrine manner. To determine whether IL6 can promote the migration of neighboring cells in a paracrine manner, we cocultured CL1-0 cells (lung adenocarcinoma cells with low migratory potential) with AS2-Vec, AS2-IL6-WT, or AS2-IL6-N73Q cells (Fig. <a href=\"/articles/s41467-024-51831-7#Fig3\"> 3i </a> ). Similar to the results above, CL1-0 cells cocultured with AS2-IL6-N73Q cells demonstrated the strongest migratory ability, followed by those cocultured with AS2-IL6-WT cells and those cocultured with AS2-Vec cells (Fig. <a href=\"/articles/s41467-024-51831-7#Fig3\"> 3i </a> ). In addition, the migration capacity of AS2-IL6-N172Q cells was not significantly stronger than that of AS2-IL6-WT cells (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 14 </a> ). In contrast, the migration capacity of AS2-IL6- N73Q + N172Q cells was comparable to that of AS2-IL6-N73Q cells. These data provide good evidence that the deNG-IL6-mediated promotion of cell migration was attributed mainly to the loss of N-glycosylation on N73 and that the contribution of N-glycosylation on N172 was minimal. </p> <p> We then assessed the in vivo metastatic potential of the AS2-Vec, AS2-IL6-WT, and AS2-IL6-N73Q cells ( <i> n </i> = 3 per group, triplicate). Cells were injected intravenously (i.v.) into nude mice, which were then observed daily. At the death of the first animal, all of the mice were euthanized, terminating the experiment. In each of three experiments, the first mouse that died was from the AS2-IL6-N73Q group. Upon gross anatomical examination (Fig. <a href=\"/articles/s41467-024-51831-7#Fig3\"> 3j </a> , left panel) and hematoxylin and eosin (H&amp;E) staining (Fig. <a href=\"/articles/s41467-024-51831-7#Fig3\"> 3j </a> , right panel), a marked increase in lung metastases was observed in the AS2-IL6-N73Q group. Because necropsy revealed that the lungs of the earliest succumbing mouse were necrotic (as indicated by the dashed circles; these lungs were not used for further analysis), only two lung sections from this group were analyzed (Fig. <a href=\"/articles/s41467-024-51831-7#Fig3\"> 3j </a> and Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 15 </a> ). Ratio of the <u> t </u> umor area to the total <u> l </u> ung area (T/L) in the mice was quantified with the TissueFAXS system and used to represent the lung metastatic capacity (Fig. <a href=\"/articles/s41467-024-51831-7#Fig3\"> 3k </a> ). The AS2-IL6-N73Q group presented a remarkable average T/L value of 53.11%, consistent with the in vitro metastatic propensity (Fig. <a href=\"/articles/s41467-024-51831-7#Fig3\"> 3h </a> ), while the AS2-Vec and AS2-IL6-WT groups presented mean T/L values of 3.2% and 0.65%, respectively. We also established AS2-IL6-WT-pLuc and AS2-IL6-N73Q-pLuc cells and evaluated in vivo metastasis by the IVIS system (Fig. <a href=\"/articles/s41467-024-51831-7#Fig3\"> 3l </a> ). Similarly, the AS2-IL6-N73Q-pLuc cells showed a strong lung metastatic capacity. Collectively, our in vivo data suggest that deNG-IL6 potentiates lung cancer cell metastasis. </p> <h3 class=\"c-article__sub-heading\" id=\"Sec7\"> deNG-IL6 switches the GP130 signaling preference from the JAK-STAT3 axis to the SRC-YAP Axis </h3> <p> Next, we sought to clarify the signaling downstream of deNG-IL6-GP130 that potentiates metastasis. Constitutively aberrant activation of STAT3 contributes to the migration and metastasis of cancer cells <sup> <a aria-label=\"Reference 1\" href=\"/articles/s41467-024-51831-7#ref-CR1\" id=\"ref-link-section-d6776021e2013\" title=\"Yu, H., Lee, H., Herrmann, A., Buettner, R. &amp; Jove, R. Revisiting STAT3 signalling in cancer: new and unexpected biological functions. Nat. Rev. Cancer 14, 736–746 (2014).\"> 1 </a> </sup> . In addition to STAT3, IL6 activates ERK, AKT, and SRC via GP130 <sup> <a aria-label=\"Reference 5\" href=\"/articles/s41467-024-51831-7#ref-CR5\" id=\"ref-link-section-d6776021e2017\" title=\"Hunter, C. A. &amp; Jones, S. A. IL-6 as a keystone cytokine in health and disease. Nat. Immunol. 16, 448–457 (2015).\"> 5 </a> </sup> ; thus, we examined the expression of the total and active forms of these effectors in both the AS2-IL6-WT cells and the AS2-IL6-N73Q cells (Fig. <a href=\"/articles/s41467-024-51831-7#Fig4\"> 4a </a> ). Notably, higher levels of SRCpY416 were detected in the AS2-IL6-N73Q cells than in the AS2-IL6-WT cells, whereas ERK1/2, pERK, AKT, AKTpY308, and AKTpY473 showed comparable levels, suggesting the involvement of SRC in the deNG-IL6-induced alternative signaling pathway. </p> <div class=\"c-article-section__figure js-c-reading-companion-figures-item\" id=\"figure-4\"> <figure> <figcaption> <b class=\"c-article-section__figure-caption\" id=\"Fig4\"> Fig. 4: deNG-IL6 Switches the GP130 Signaling Preference from the JAK-STAT3 Axis to the SRC-YAP Axis. </b> </figcaption> <div class=\"c-article-section__figure-content\"> <div class=\"c-article-section__figure-item\"> <a class=\"c-article-section__figure-link\" href=\"/articles/s41467-024-51831-7/figures/4\" rel=\"nofollow\"> <picture> <source srcset=\"//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-024-51831-7/MediaObjects/41467_2024_51831_Fig4_HTML.png?as=webp\" type=\"image/webp\"/> </picture> </a> </div> <div class=\"c-article-section__figure-description\" id=\"figure-4-desc\"> <p> <b> a </b> Expression of proteins indicating the activation of ERK, AKT, and SRC in AS2-Vec, AS2-IL6-WT, and AS2-IL6-N73Q cells. <i> n </i> = 3 independent experiments, mean ± SEM, two-tailed unpaired <i> t </i> -test, pERK: n.s. <i> p </i> = 0.5646; ERK1/2: n.s. <i> p </i> = 0.9914; AKTpY308: n.s. <i> p </i> = 0.6477; AKTpS473: n.s. <i> p </i> = 0.0967; AKT: n.s. <i> p </i> = 0.7452; SRCpY416: *** <i> p </i> &lt; 0.0001; SRC: n.s. 0.2598; β-ACTIN: n.s. <i> p </i> = 1.000. FC: fold change relative to AS2-IL6-WT. The samples derived from the same experiment but different gels for pERK, ERK1/2, AKTpY308, AKTpY473, AKT, SRCpY416, SRC, and β-ACTIN were processed in parallel. <b> b </b> Hierarchical clustering of N73Q-IL6-induced genes downstream of SRC identified by IPA. <b> c </b> Recruitment of STAT3 and SRC to WT-IL6- and N73Q-IL6-activated GP130 determined based on IF images acquired with an FV3000 microscope (60x). Images are representative of three technical replicates. Scale bars, 20 μm. <b> d </b> Expression of proteins indicating the activation of STAT3 and SRC in the cytosolic and nuclear fractions of the AS2-IL6-WT and AS2-IL6-N73Q cells. <i> n </i> = 3 independent experiments, mean ± SEM, two-tailed unpaired <i> t </i> -test, STAT3pY705: *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001; STAT3: *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001; SRCpY416: *** <i> p </i> &lt; 0.0001, ** <i> p </i> = 0.0065; SRC: *** <i> p </i> &lt; 0.0001, ** <i> p </i> = 0.0075; LAMIN A/C: n.s. <i> p </i> = 0.1567, n.s. <i> p </i> = 1.0000; β-ACTIN: n.s. <i> p </i> = 1.0000, n.s. <i> p </i> = 1.0000. FC: fold change relative to AS2-IL6-WT. The samples derived from the same experiment but different gels for STAT3pY705, STAT3, SRCpY416, SRC, LAMIN A/C, and β-ACTIN were processed in parallel. <b> e </b> Effect of IL6 antibody (5 μg/mL), JAK-STAT3 inhibitors (S3I-201 = 10 μM, ruxolitinib = 2.5 μM), and SRC inhibitors (PP1 = 10 μM, PP2 = 10 μM, tirbanibulin = 10 μM) on the migration of AS2-IL6-WT and AS2-IL6-N73Q cells through the space made by culture inserts. Images were obtained at 0 and 24 h. Scale bars, 200 μm. Quantitative results are next to the images. <i> n </i> = 3 independent experiments, mean ± SEM, two-tailed unpaired <i> t </i> -test, AS2-IL6-N73Q cells: *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001; AS2-IL6-WT cells: *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001, ** <i> p </i> = 0.0026, ** <i> p </i> = 0.0082, *** <i> p </i> &lt; 0.0001. <b> f </b> Expression of SRC in the AS2-IL6-N73Q cells transduced with shSRC-bearing VSV-G particles. <i> n </i> = 3 independent experiments, mean ± SEM, one-tailed unpaired <i> t </i> -test, SRCpY416: ** <i> p </i> = 0.0022, ** <i> p </i> = 0.0011; SRC: *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001; β-ACTIN: n.s. <i> p </i> = 0.4500, n.s. <i> p </i> = 0.4267. FC: fold change relative to shpLKO.1. The samples derived from the same experiment but different gels for STAT3pY705, STAT3, SRCpY416, SRC, and β-ACTIN were processed in parallel. <b> g </b> Migration of the <i> SRC </i> -silenced AS2-IL6-N73Q cells. Scale bars, 200 μm. <i> n </i> = 3 independent experiments, mean ± SEM, one-tailed unpaired <i> t </i> -test, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001. <b> h </b> In vivo metastasis of SRC-silenced AS2-IL6-N73Q cells injected intravenously into nude mice ( <i> n </i> = 3 per cell type). TissueFAXS-acquired images of H&amp;E-stained lungs are shown (left). The tumor area over the total lung area was quantified by HistoQuest. <i> n </i> = 3 animals in each group, mean ± SEM, one-tailed unpaired <i> t </i> -test, ** <i> p </i> = 0.0062, ** <i> p </i> = 0.0062. Source data are provided as a Source Data file. </p> </div> </div> <div class=\"u-text-right u-hide-print\"> <a aria-label=\"Full size image figure 4\" class=\"c-article__pill-button\" href=\"/articles/s41467-024-51831-7/figures/4\" rel=\"nofollow\"> <span> Full size image </span> </a> </div> </figure> </div> <p> Further IPA prediction showed that the SRC-regulated genes were notably elevated in the N73Q-IL6-CM-treated AS2-Vec cells (Fig. <a href=\"/articles/s41467-024-51831-7#Fig4\"> 4b </a> ), suggesting that defective N-glycosylation of IL6 alters the preference of GP130 to activate its downstream signals. By IF imaging, we observed that WT-IL6-CM led GP130 to preferentially recruit STAT3 to the receptor complex (Fig. <a href=\"/articles/s41467-024-51831-7#Fig4\"> 4c </a> , left panel, merged image, light orange), while N73Q-IL6-CM triggered GP130 to alternatively recruit SRCs (Fig. <a href=\"/articles/s41467-024-51831-7#Fig4\"> 4c </a> , right panel, merged image, yellow), indicating that deNG-IL6 switches the effector preference of GP130 from STAT3 to SRC. In addition, an enriched amount of active STAT3 was observed in the AS2-IL6-WT cells, particularly in the nuclear lysates (Fig. <a href=\"/articles/s41467-024-51831-7#Fig4\"> 4d </a> ). In comparison, the amount of active SRC was enriched in the cytosolic lysates of the AS2-IL6-N73Q cells. These observations were consistent with the signals of GP130-recruited intracellular STAT3 and SRC (Fig. <a href=\"/articles/s41467-024-51831-7#Fig4\"> 4c </a> ), demonstrating that the pattern of N-glycosylation of IL6 influences the preference of downstream signals of GP130. </p> <p> IL6 neutralization effectively inhibited the migration of AS2-IL6-N73Q (Fig. <a href=\"/articles/s41467-024-51831-7#Fig4\"> 4e </a> ), AS2-Vec (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 16 </a> ), and AS2-IL6-WT cells (Fig. <a href=\"/articles/s41467-024-51831-7#Fig4\"> 4e </a> ). To confirm the role of active SRC in N73Q-IL6-induced migration, we treated the AS2-Vec (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 16 </a> ), AS2-IL6-WT, and AS2-IL6-N73Q cells with the SRC inhibitors PP1, PP2, and tirbanibulin and assessed their migratory capacity (Fig. <a href=\"/articles/s41467-024-51831-7#Fig4\"> 4e </a> , upper panel). Compared to the mock control, the SRC inhibitors efficiently inhibited the migration of the AS2-IL6-N73Q cells, while the STAT3 inhibitor S3I-201 and the JAK2 inhibitor ruxolitinib showed modest inhibitory effects. In contrast, S3I-201 and ruxolitinib inhibited the migration of AS2-vec and AS2-IL6-WT cells more effectively than SRC inhibitors (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 16 </a> ), suggesting that STAT3 shows a lower contribution to migration than SRC does in AS2-IL6-N73Q cells. Silencing of SRC (Fig. <a href=\"/articles/s41467-024-51831-7#Fig4\"> 4f </a> ) impeded the migration of the AS2-IL6-N73Q cells (Fig. <a href=\"/articles/s41467-024-51831-7#Fig4\"> 4g </a> ). In addition, the in vivo metastases assessed by i.v. inoculation in nude mice showed that the AS2-IL6-N73Q cells transduced with empty vector (shpLKO.1 cells) demonstrated a strong metastatic capacity (Fig. <a href=\"/articles/s41467-024-51831-7#Fig4\"> 4h </a> ), whereas both the SRC-sh1 and SRC-sh2 cells showed a significantly decreased number of lung metastatic foci and T/L ratio (68.95%, 0.04%, and 0.23%, respectively, <i> p </i> &lt; 0.001, Fig. <a href=\"/articles/s41467-024-51831-7#Fig4\"> 4h </a> ). These data showed that the switch of the GP130 downstream effector to SRC is essential for deNG-IL6-induced metastasis. </p> <p> To determine whether the glycan attached to N73 or N172 is accountable for the signaling switch, we evaluated the signaling expression in whole-cell lysates, as well as nuclear and cytosolic fractions, extracted from AS2-Vec cells, AS2-IL6-WT cells, AS2-IL6-N73Q cells, and AS2-IL6-N172Q cells. Compared to AS2-IL6-WT cells, a signaling shift from the JAK-STAT3 pathway to the alternatively activated SRC-YAP-SOX axis was exclusively observed in AS2-IL6-N73Q cells (but not in AS2-IL6-N172Q cells) in both whole-cell lysates (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 17a </a> ) and nuclear (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 17b </a> ) and cytosolic (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 17c </a> ) fractions. These findings suggest that N-glycosylation on N73, rather than N172, is accountable for the switch in IL6 downstream signaling from the STAT3 axis to the SRC axis. </p> <p> In addition, to confirm that the N-glycan but not the amino acid substitution changed the properties of IL6, we also established AS2 cells that stably expressed N73A-IL6 and N73V-IL6 (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 18a–c </a> ). Compared to WT-IL6, both N73A-IL6 and N73V-IL6 demonstrated a decrease in NG-IL6 (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 18c </a> ), similar to N73Q-IL6. Both N73A-IL6-CM and N73V-IL6-CM induced similar patterns of STAT3 activation (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 18d </a> ) and localization of STAT3 and SRC (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 18e </a> ) as N73Q-IL6-CM. Furthermore, similar to AS2-IL6-N73Q cells, both AS2-IL6-N73A and AS2-IL6-N73V cells presented strong in vitro migratory (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 18f </a> ) and in vivo metastatic capacities (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 18g, h </a> ). The above findings indicated that the loss of N-glycosylation but not the change in amino acids on IL6 induces the signaling shift. </p> <h3 class=\"c-article__sub-heading\" id=\"Sec8\"> SOX2 contributes to deNG-IL6-induced SRC-YAP axis-promoted migration and metastasis </h3> <p> Because YAP is a transcriptional regulator downstream of GP130-SRC <sup> <a aria-label=\"Reference 12\" href=\"/articles/s41467-024-51831-7#ref-CR12\" id=\"ref-link-section-d6776021e2340\" title=\"Taniguchi, K. et al. A gp130-Src-YAP module links inflammation to epithelial regeneration. Nature 519, 57–62 (2015).\"> 12 </a> , <a aria-label=\"Reference 55\" href=\"/articles/s41467-024-51831-7#ref-CR55\" id=\"ref-link-section-d6776021e2343\" title=\"Romera-Hernandez, M. et al. Yap1-driven intestinal repair is controlled by group 3 innate lymphoid cells. Cell Rep. 30, 37–45.e33 (2020).\"> 55 </a> </sup> , we analyzed our microarray data (Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2h </a> ) by GSEA, characterized the enriched YAP conserved signature in the N73Q-IL6CM-treated AS2-Vec cells (Fig. <a href=\"/articles/s41467-024-51831-7#Fig5\"> 5a </a> ) and found that these cells exhibited elevated expression of YAP-regulated genes (Fig. <a href=\"/articles/s41467-024-51831-7#Fig5\"> 5b </a> ). In addition, proteins of the YAP pathway, including YAP, LATS1, TAZ and MST1, were found to be activated and enriched in the nuclear fraction of the AS2-IL6-N73Q cells (Fig. <a href=\"/articles/s41467-024-51831-7#Fig5\"> 5c </a> ), concordant with the elevated cytosolic active SRC (Fig. <a href=\"/articles/s41467-024-51831-7#Fig4\"> 4d </a> ). </p> <div class=\"c-article-section__figure js-c-reading-companion-figures-item\" id=\"figure-5\"> <figure> <figcaption> <b class=\"c-article-section__figure-caption\" id=\"Fig5\"> Fig. 5: SOX2 Contributes to deNG-IL6-Induced SRC-YAP Axis-Promoted Migration and Metastasis. </b> </figcaption> <div class=\"c-article-section__figure-content\"> <div class=\"c-article-section__figure-item\"> <a class=\"c-article-section__figure-link\" href=\"/articles/s41467-024-51831-7/figures/5\" rel=\"nofollow\"> <picture> <source srcset=\"//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-024-51831-7/MediaObjects/41467_2024_51831_Fig5_HTML.png?as=webp\" type=\"image/webp\"/> </picture> </a> </div> <div class=\"c-article-section__figure-description\" id=\"figure-5-desc\"> <p> <b> a </b> GSEA enrichment plot for the conserved YAP signature in the AS2-Vec cells under WT-IL6 and N73Q-IL6 treatment. Nominal <i> p- </i> value was one-tailed test on the appropriate side of the null distribution. <b> b </b> Hierarchical clustering of the N73Q-IL6-induced YAP downstream genes identified by IPA. <b> c </b> Expression of YAP pathway-associated proteins in the nuclear fraction of the AS2-IL6-WT and AS2- IL6-N73Q cells. <i> n </i> = 3 independent experiments, mean ± SEM, two-tailed unpaired <i> t </i> -test, YAPpY357: *** <i> p </i> &lt; 0.0001; YAP: *** <i> p </i> &lt; 0.0001; pLATS: *** <i> p </i> = 0.0001; LATS1: *** <i> p </i> &lt; 0.0001; pTAZ: *** <i> p </i> = 0.0001; TAZ: *** <i> p </i> &lt; 0.0001; MST1: *** <i> p </i> &lt; 0.0001; LAMIN A/C: n.s. <i> p </i> = 0.0779. FC: fold change relative to AS2-IL6-WT. The samples derived from the same experiment but different gels for YAPpY357, YAP, pLATS, LATS1, pTAZ, TAZ, MST1, and LAMIN A/C were processed in parallel. <b> d </b> Expression of stemness-associated proteins in the AS2-IL6-WT and AS2-IL6-N73Q cells. <i> n </i> = 3 independent experiments, mean ± SEM, two-tailed unpaired <i> t </i> -test, SOX2: *** <i> p </i> &lt; 0.0001; NANOG: *** <i> p </i> &lt; 0.0001; OCT3/4: *** <i> p </i> = 0.0010; β-ACTIN: n.s. <i> p </i> = 0.9814. FC: fold change relative to AS2-IL6-WT. The samples derived from the same experiment but different gels for SOX2, NANOG, OCT3/4, and β-ACTIN were processed in parallel. <b> e </b> Expression of stemness-associated proteins in the AS2-IL6-N73Q cells treated with YAPi VP at increasing doses. <i> n </i> = 3 independent experiments, percentage expression relative to none, mean ± SEM, two-tailed unpaired <i> t </i> -test, YAP: *** <i> p </i> &lt; 0.0001, ** <i> p </i> = 0.0013, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001; TAZ: *** <i> p </i> = 0.0001, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001; SOX2: *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001; β-ACTIN: n.s. <i> p </i> = 0.5000, n.s. <i> p </i> = 0.5000, n.s. <i> p </i> = 0.5000, n.s. <i> p </i> = 0.5000, n.s. <i> p </i> = 0.5000, n.s. <i> p </i> = 0.5000. The samples derived from the same experiment but different gels for YAP, TAZ, SOX2, and β-ACTIN were processed in parallel. <b> f </b> Migration of the AS2-IL6-N73Q cells under VP treatment. <i> n </i> = 3 independent experiments, mean ± SEM, one-tailed unpaired <i> t </i> -test, *** <i> p </i> &lt; 0.0001. <b> g </b> Expression of the <i> YAP </i> gene ( <b> g </b> ) and YAP protein ( <b> h </b> ) in the AS2-IL6-N73Q cells transduced with shSRC-bearing VSV-G particles. <i> n </i> = 3 independent experiments, percentage expression relative to shpLKO.1, mean ± SEM, one-tailed unpaired <i> t </i> -test, YAP: *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001; β-ACTIN: n.s. <i> p </i> = 0.5000, n.s. <i> p </i> = 0.5000. The cDNA or protein samples derived from the same experiment but different gels for <i> YAP </i> , <i> GAPDH </i> , YAP, and β-ACTIN were processed in parallel. <b> i </b> Migration of the <i> YAP </i> -silenced AS2-IL6-N73Q cells. Scale bars, 200 μm. <i> n </i> = 3 independent experiments, mean ± SEM, one-tailed unpaired <i> t </i> -test, *** <i> p </i> = 0.0001, *** <i> p </i> &lt; 0.0001. <b> j </b> In vivo metastasis of the <i> YAP </i> -silenced AS2-IL6-N73Q cells injected intravenously into nude mice ( <i> n </i> = 3 per cell type). TissueFAXS-acquired images of H&amp;E-stained lungs are shown (left). The tumor area over the total lung area was quantified by HistoQuest. <i> n </i> = 3 animals in each group, mean ± SEM, one-tailed unpaired <i> t </i> -test, ** <i> p </i> = 0.0015, ** <i> p </i> = 0.0015. <b> k </b> Expression of <i> SOX2 </i> mRNA ( <b> k </b> ) and protein ( <b> l </b> ) in the <i> SOX2 </i> -silenced AS2-IL6-N73Q cells. <i> n </i> = 3 independent experiments, percentage expression relative to shpLKO.1, mean ± SEM, one-tailed unpaired <i> t </i> -test, SOX2: *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001; β-ACTIN: n.s. <i> p </i> = 0.5000, n.s. <i> p </i> = 0.5000. The cDNA or protein samples derived from the same experiment but different gels for <i> SOX2 </i> , <i> GAPDH </i> , SOX2, and β-ACTIN were processed in parallel. <b> m </b> Migration of the <i> SOX2 </i> -silenced AS2-IL6-N73Q cells. Scale bars, 200 μm. <i> n </i> = 3 independent experiments, mean ± SEM, one-tailed unpaired <i> t </i> -test, *** <i> p </i> &lt; 0.0001, *** <i> p </i> = 0.0001. <b> n </b> In vivo metastasis of the <i> SOX2 </i> -silenced AS2-IL6-N73Q cells intravenously injected into nude mice ( <i> n </i> = 3 per cell type). TissueFAXS-acquired images of H&amp;E-stained lungs are shown (left). The tumor area over the total lung area was quantified by HistoQuest. <i> n </i> = 3 animals in each group, mean ± SEM, one-tailed unpaired <i> t </i> -test, ** <i> p </i> = 0.0023, * <i> p </i> = 0.0023. Source data are provided as a Source Data file. </p> </div> </div> <div class=\"u-text-right u-hide-print\"> <a aria-label=\"Full size image figure 5\" class=\"c-article__pill-button\" href=\"/articles/s41467-024-51831-7/figures/5\" rel=\"nofollow\"> <span> Full size image </span> </a> </div> </figure> </div> <p> Activated YAP promotes regeneration of the intestinal epithelium and enhances the expression and activity of stemness-related proteins, including SOX2, NANOG, and OCT3/4 <sup> <a aria-label=\"Reference 12\" href=\"/articles/s41467-024-51831-7#ref-CR12\" id=\"ref-link-section-d6776021e2711\" title=\"Taniguchi, K. et al. A gp130-Src-YAP module links inflammation to epithelial regeneration. Nature 519, 57–62 (2015).\"> 12 </a> , <a aria-label=\"Reference 29\" href=\"/articles/s41467-024-51831-7#ref-CR29\" id=\"ref-link-section-d6776021e2714\" title=\"Bora-Singhal, N. et al. YAP1 regulates OCT4 activity and SOX2 expression to facilitate self-renewal and vascular mimicry of stem-like cells. Stem cells 33, 1705–1718 (2015).\"> 29 </a> </sup> . SOX2 was markedly enriched in the AS2-IL6-N73Q cells (Fig. <a href=\"/articles/s41467-024-51831-7#Fig5\"> 5d </a> ). To verify the role of YAP in deNG-IL6-promoted migration and metastasis, we treated the AS2-IL6-N73Q cells with the YAP inhibitor (YAPi) verteporfin (VP) <sup> <a aria-label=\"Reference 56\" href=\"/articles/s41467-024-51831-7#ref-CR56\" id=\"ref-link-section-d6776021e2721\" title=\"Tsuji, T. et al. YAP1 mediates survival of ALK-rearranged lung cancer cells treated with alectinib via pro-apoptotic protein regulation. Nat. Commun. 11, 74 (2020).\"> 56 </a> </sup> . VP simultaneously reduced the levels of YAP, TAZ, and downstream SOX2 in a dose-dependent manner (Fig. <a href=\"/articles/s41467-024-51831-7#Fig5\"> 5e </a> ) and impeded the migration of the AS2-IL6-N73Q cells (Fig. <a href=\"/articles/s41467-024-51831-7#Fig5\"> 5f </a> ). <i> YAP </i> silencing in the AS2-IL6-N73Q cells with VSV-Gs with two independent shRNAs led to a successful reduction in <i> YAP </i> at both the RNA (Fig. <a href=\"/articles/s41467-024-51831-7#Fig5\"> 5g </a> ) and protein levels (Fig. <a href=\"/articles/s41467-024-51831-7#Fig5\"> 5h </a> ). Compared to the shpLKO.1 cells, both the YAP-sh1 and YAP-sh2 cells exhibited impeded in vitro migration (28.85% and 50.48%, respectively, <i> p </i> &lt; 0.001, Fig. <a href=\"/articles/s41467-024-51831-7#Fig5\"> 5i </a> ) and in vivo metastasis (0.004% and 0.008, respectively, <i> p </i> &lt; 0.001, Fig. <a href=\"/articles/s41467-024-51831-7#Fig5\"> 5j </a> ). </p> <p> We used the same strategy to further address the role of SOX2 in deNG-IL6-promoted migration and metastasis mediated through SRC and YAP. Similarly, SOX2 was successfully reduced at both the RNA and protein levels (Fig. <a href=\"/articles/s41467-024-51831-7#Fig5\"> 5k, l </a> ). Both in vitro migration (Fig. <a href=\"/articles/s41467-024-51831-7#Fig5\"> 5m </a> ) and in vivo metastasis (Fig. <a href=\"/articles/s41467-024-51831-7#Fig5\"> 5n </a> ) were reduced in the <i> SOX2 </i> -silenced SOX2-sh1 and SOX2-sh2 cells compared to the shpLKO.1 cells. By pharmaceutical and genetic inhibitory approaches, we demonstrated that deNG-IL6 promotes the migration and metastasis of lung cancer cells via a switch of GP130 downstream signaling from STAT3 to the SRC-YAP-SOX2 axis. </p> <h3 class=\"c-article__sub-heading\" id=\"Sec9\"> Osimertinib induces expression of deNG-IL6 and subsequent activation of the SRC-YAP-SOX2 axis </h3> <p> Increased IL6 expression was reported in osimertinib-resistant NSCLC cells and patients <sup> <a aria-label=\"Reference 22\" href=\"/articles/s41467-024-51831-7#ref-CR22\" id=\"ref-link-section-d6776021e2780\" title=\"Kim, S. M. et al. Activation of IL-6R/JAK1/STAT signaling induces de novo resistance to irreversible EGFR inhibitors in non-small cell lung cancer with T790M resistance mutation. Mol. Cancer Ther. 11, 2254–2264 (2012).\"> 22 </a> </sup> , but glycosylation pattern and the biological properties of IL6 are unclear. To study this issue, we established osimertinib-resistant (OR) PC9 and HCC827 lung adenocarcinoma cell lines (PC9-OR and HCC827-OR cells, respectively) harboring mutant EGFR, and the resistance was maintained by constant culture in medium containing osimertinib (5 μM). Compared to the corresponding parental cells (PC9-P and HCC827-P), PC9-OR and HCC827-OR cells exhibited increased resistance to osimertinib (Fig. <a href=\"/articles/s41467-024-51831-7#Fig6\"> 6a </a> ) and elevated IL6 secretion (Fig. <a href=\"/articles/s41467-024-51831-7#Fig6\"> 6b </a> and Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 19 </a> ). </p> <div class=\"c-article-section__figure js-c-reading-companion-figures-item\" id=\"figure-6\"> <figure> <figcaption> <b class=\"c-article-section__figure-caption\" id=\"Fig6\"> Fig. 6: Inhibition of osimertinib-induced deNG-IL6 restores TKI sensitivity in TKI-resistant NSCLC. </b> </figcaption> <div class=\"c-article-section__figure-content\"> <div class=\"c-article-section__figure-item\"> <a class=\"c-article-section__figure-link\" href=\"/articles/s41467-024-51831-7/figures/6\" rel=\"nofollow\"> <picture> <source srcset=\"//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-024-51831-7/MediaObjects/41467_2024_51831_Fig6_HTML.png?as=webp\" type=\"image/webp\"/> </picture> </a> </div> <div class=\"c-article-section__figure-description\" id=\"figure-6-desc\"> <p> <b> a </b> Survival of parental and OR PC9 and HCC827 cells with osimertinib treatment (72 h). <i> n </i> = 6 independent experiments per data point, mean ± SEM. <b> b </b> Levels of IL6 secreted by the parental and OR PC9 and HCC827 cells after 72 h of attachment. <i> n </i> = 3 technical replicates, mean ± SEM, two-tailed unpaired <i> t </i> -test, PC9-OR: *** <i> p </i> &lt; 0.0001; HCC827-OR: *** <i> p </i> &lt; 0.0001. <b> c </b> mRNA expression of <i> IL6 </i> in OR cells transduced with sh <i> IL6 </i> -expressing VSV-G particles. The samples were derived from the same experiment for <i> IL6 </i> and <i> GAPDH </i> were seperated in the same gel. <b> d </b> Quantity of total secreted IL6 in OR-sh <i> IL6 </i> cells as measured by ELISAs. <i> n </i> = 3 technical replicates, mean ± SEM, one-tailed unpaired <i> t </i> -test, PC9-OR: *** <i> p </i> &lt; 0.0001; HCC827-OR: *** <i> p </i> &lt; 0.0001. <b> e </b> Morphology of OR and OR-sh <i> IL6 </i> cells treated with osimertinib (5 μM) for 72 h. Scale bar, 100 μm. <b> f </b> Survival of PC9-OR-sh <i> IL6 </i> and HCC827-OR-sh <i> IL6 </i> cells treated with osimertinib at 5 μM (72 h). <i> n </i> = 3 independent experiments, mean ± SEM, two-tailed unpaired <i> t </i> -test, PC9-OR: *** <i> p </i> &lt; 0.0001; HCC827-OR: *** <i> p </i> &lt; 0.0001. <b> g </b> Pattern of secreted IL6 in the parental and OR PC9 and HCC827 cells. Because the PC9-P and HCC827-P cells secreted little IL6, which prevented analysis of its glycosylation pattern, we established PC9-IL6-WT and HCC827-IL6-WT cells as control cell lines. The proportions of heavy IL6 in these cells were quantified and are shown in the right panel. <i> n </i> = 3 independent experiments, mean ± SEM, two-tailed unpaired <i> t </i> -test, PC9-OR: *** <i> p </i> &lt; 0.0001; HCC827-OR: *** <i> p </i> &lt; 0.0001. <b> h </b> GSEA of N-linked glycosylation pathways and EMT, SRC and YAP activation statuses in the transcriptome dataset GSE106765 from the HCC827-OR-Taka cells. Nominal <i> p </i> -value was one-tailed test on the appropriate side of the null distribution. <b> i </b> Expression of N-OST genes in the parental and OR PC9 and HCC827 cells, as detected by qPCR. <i> n </i> = 3 independent experiments, mean ± SEM, one-tailed unpaired <i> t </i> -test, PC9-OR: ** <i> p </i> = 0.0065, * <i> p </i> = 0.0193, n.s. <i> p </i> = 0.1579, *** <i> p </i> &lt; 0.0001, ** <i> p </i> = 0.0099, ** <i> p </i> = 0.0047; HCC827-OR: 827-OR: *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001, * <i> p </i> = 0.0160, *** <i> p </i> = 0.004, ** <i> p </i> = 0.0025, ** <i> p </i> = 0.0014. <b> j </b> N-glycosylated IL6 detected by lectin blotting with <u> w </u> heat <u> g </u> erm <u> a </u> gglutinin (WGA). The proportions of WGA-detected NG-IL6 in WT-IL6 or OR-IL6 were quantified and are shown in the ri <u> g </u> ht p <u> a </u> nel. <i> n </i> = 3 independent experiments, mean ± SEM, one-tailed unpaired <i> t </i> -test, PC9-OR: *** <i> p </i> &lt; 0.0001; HCC827-OR: *** <i> p </i> &lt; 0.0001. <b> k </b> Quantitative characterization of N-glycans on purified secre <i> t </i> ed WT-IL6 or OR-IL6 from CM of PC9 or HCC827 cell derivatives by lectin microarray, as assessed with GlycoStation <sup> TM </sup> Reader 1200 (GlycoTechnica). The raw data was calculated and output automatically by GlycoStation Tools Pro Suite 1.5 software. <i> n </i> = 3 technical replicates, mean ± S.D. See also Supplementary Table <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 1 </a> . <b> l </b> Characterization of glycosylation in IL6 by MS. See also Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 22 </a> . <b> m </b> Pattern of secreted WT-IL6 or OR-IL6 with or without PNGase F digestion. The percentage proportions of H-IL6 and L-IL6 were quantified in the segmented histogram. <i> n </i> = 3 independent experiments, mean ± SEM. Source data are provided as a Source Data file. </p> </div> </div> <div class=\"u-text-right u-hide-print\"> <a aria-label=\"Full size image figure 6\" class=\"c-article__pill-button\" href=\"/articles/s41467-024-51831-7/figures/6\" rel=\"nofollow\"> <span> Full size image </span> </a> </div> </figure> </div> <p> For determination of the function of osimertinib-induced IL6 in resistance, IL6 was inhibited by either an IL6-neutralizing antibody or shRNA against <i> IL6 </i> in PC9-OR and HCC827-OR cells. The IL6 antibody reduced the survival of PC9-OR and HCC827-OR cells by ~15% at 20 μg/mL in osimertinib-containing medium (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 20 </a> ). Silencing <i> IL6 </i> successfully reduced the mRNA levels (Fig. <a href=\"/articles/s41467-024-51831-7#Fig6\"> 6c </a> ) and secretion levels of IL6 (Fig. <a href=\"/articles/s41467-024-51831-7#Fig6\"> 6d </a> ) and restored the sensitivity to osimertinib at tolerant concentrations of OR cells (Fig. <a href=\"/articles/s41467-024-51831-7#Fig6\"> 6e </a> ). The survival was reduced to 57.2% and 16.5% in PC9-OR and HCC827-OR cells, respectively (Fig. <a href=\"/articles/s41467-024-51831-7#Fig6\"> 6f </a> ). In summary, elevated IL6 secretion contributes to cellular resistance to osimertinib. </p> <p> We next examined the pattern of IL6 glycoforms secreted by PC9-OR and HCC827-OR cells and the corresponding parental cell lines. Because the PC9-P and HCC827-P cells secreted little IL6, which prevented analysis of its glycosylation pattern, we established PC9-IL6-WT and HCC827-IL6-WT cells as control cell lines (Figs. <a href=\"/articles/s41467-024-51831-7#Fig6\"> 6g </a> , <a href=\"/articles/s41467-024-51831-7#Fig1\"> 1j </a> ). Both H-IL6 and L-IL6 were detected in the CM of the PC9-IL6-WT and HCC827-IL6-WT cells (Fig. <a href=\"/articles/s41467-024-51831-7#Fig6\"> 6g </a> , upper left and lower left). Unlike the distribution of H-IL6 and L-IL6 in the parental cells, almost all the IL6 secreted by the PC9-OR and HCC827-OR cells was L-IL6 (Fig. <a href=\"/articles/s41467-024-51831-7#Fig6\"> 6g </a> , upper right and lower right), similar to the distribution of deNG-IL6 (Fig. <a href=\"/articles/s41467-024-51831-7#Fig1\"> 1k </a> ). Therefore, we hypothesized that long-term EGFR-TKI exposure may influence the expression of oligosaccharyltransferases for N-glycosylation (N-OSTs) and then interfere with the N-glycosylation process on IL6. </p> <p> We then analyzed the transcriptome profile of HCC827 cells with or without acquired osimertinib resistance established by Takahashi (renamed P-Taka or OR-Taka, GSE106765) <sup> <a aria-label=\"Reference 57\" href=\"/articles/s41467-024-51831-7#ref-CR57\" id=\"ref-link-section-d6776021e3072\" title=\"Takahashi, A. et al. Ankyrin repeat domain 1 overexpression is associated with common resistance to afatinib and osimertinib in EGFR-mutant lung cancer. Sci. Rep. 8, 14896 (2018).\"> 57 </a> </sup> . The expression of N-glycosylation-related genes was enriched in the osimertinib-sensitive P-Taka cells but downregulated in the OR-Taka cells (Fig. <a href=\"/articles/s41467-024-51831-7#Fig6\"> 6h </a> ), where genes involved in signatures of EMT and activation of SRC and YAP were enriched in the OR-Taka cells, indicating that osimertinib concomitantly reduced the expression of N-OSTs and enhanced SRC-YAP-regulated cellular plasticity to mediate metastasis and resistance. We also analyzed datasets from lung cancer cells treated with other EGFR TKIs (gefitinib, afatinib, erlotinib, and WZ4002) published by other groups (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 21 </a> ). Decreased N-glycosylation accompanied by increased EMT and SRC-YAP activation gene sets was also observed in the lung cancer cells treated with EGFR-TKIs, indicating that TKI treatment stress may repress the N-glycosylation process in lung cancer cells. </p> <p> The most common N-glycosylation-related <u> N </u> - <u> o </u> ligo <u> s </u> accharide <u> t </u> ransferase (N-OST) genes include the enzymatically active subunits <i> STT3A </i> and <i> STT3B </i> and the cooperative scaffold members <i> RPN1 </i> , <i> RPN2 </i> , <i> DAD1 </i> , and <i> DDOST </i> <sup> <a aria-label=\"Reference 46\" href=\"/articles/s41467-024-51831-7#ref-CR46\" id=\"ref-link-section-d6776021e3117\" title=\"Lopez-Sambrooks, C. et al. Oligosaccharyltransferase inhibition induces senescence in RTK-driven tumor cells. Nat. Chem. Biol. 12, 1023–1030 (2016).\"> 46 </a> </sup> . We compared their mRNA levels in our PC9-OR and HCC827-OR cells versus the parental cells (Fig. <a href=\"/articles/s41467-024-51831-7#Fig6\"> 6i </a> ). All of the N-OST genes showed significantly downregulated expression in the OR cells, explaining their generation of L-IL6. To determine whether the main component of OR-IL6 was deNG-IL6, we analyzed IL6 from PC9-IL6-WT, PC9-OR, HCC827-IL6-WT, and HCC827-OR cells by lectin blotting probed with wheat-germ-agglutinin (WGA), which recognizes N-acetylglutamate on the target protein (Fig. <a href=\"/articles/s41467-024-51831-7#Fig6\"> 6j </a> ). WT-IL6 from PC9 and HCC827 cells showed positive bands, suggesting that they harbor N-glycosylation. In comparison, the band density suggested that NG-IL6 was reduced by ~80% in both OR cell lines. Furthermore, we analyzed the N-glycans on WT-IL6 and OR-IL6 with a lectin microarray. Because PC9-OR cells secreted a much lower level of IL6 than HCC827-OR cells (Fig. <a href=\"/articles/s41467-024-51831-7#Fig6\"> 6b </a> ), we only analyzed the N-glycans on WT-IL6 and OR-IL6 from the HCC827 cell series (Fig. <a href=\"/articles/s41467-024-51831-7#Fig6\"> 6k </a> ). We further employed MS to examine the glycosylation status of WT-IL6 and OR-IL6. Purified WT-IL6 and OR-IL6 were subjected to digestion with the protease Glu-C or trypsin to generate IL6 peptides containing N73 or N172, respectively. N-glycans were subsequently removed by treatment with PNGase F. Digested IL6 samples were analyzed by LC-ESI-MS (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 22 </a> ). Both WT-IL6 and OR-IL6 were found to contain three glycosylation sites: N73, N172 and T170 (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 22 </a> and Fig. <a href=\"/articles/s41467-024-51831-7#Fig6\"> 6l </a> ). However, these three sites are not homogeneously glycosylated: 28% and 20% of WT-IL6 and OR-IL6 proteins were glycosylated at N73, respectively, i.e., OR-IL6 displayed a 29% reduction in N73 glycosylation compared to WT-IL6. In contrast, there were no significant differences in the percentages of proteins glycosylated at either N172 or T170. The glycosylation percentage was minimal at N172 (4% and 3% of WT-IL6 and OR-IL6, respectively), whereas glycosylation was prevalent at T170 (&gt;99% and 96% of WT-IL6 and OR-IL6, respectively). </p> <p> Based on the MS data, there are presumably four distinct glycoforms of WT-IL6. Twenty-eight percent of WT-IL6 molecules exhibit diglycosylation at N73 + T170, corresponding to H-IL6 and NG-IL6. Sixty-eight percent of WT-IL6 molecules show monoglycosylation at T170, represented by L-IL6 and deNG-IL6. The remaining 4% of WT-IL6 molecules may contain either two (at N172 + T170) or three (at N73 + N172 + T170) glycan chains. However, owing to the limited recognition sites of proteases, the precise percentages of IL6 with single or double N-glycosylation could not be determined simultaneously with the peptide products from a single digestion. The conclusion appears to be consistent with the results of the WB shown in Fig. <a href=\"/articles/s41467-024-51831-7#Fig6\"> 6g </a> . In the Western blots, two prominent bands corresponding to WT-IL6 were observed: a dense L-IL6 band indicative of deNG-IL6 and the H-IL6 band representing NG-IL6. Notably, no discernible band was detected at a higher molecular weight than the H-IL6 band (Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2b, c </a> ), suggesting that the quantity of heavily glycosylated IL6 (at N73 + N172 + T170) must be below the detection limit and was too low to be detected. When treated with PNGase F (Fig. <a href=\"/articles/s41467-024-51831-7#Fig6\"> 6m </a> ), the heavy forms of WT-IL6 were reduced in both the PC9 and HCC827 cell series, whereas little influence was observed in that of OR-IL6. These data provide evidence that OR-IL6 is poorly N-glycosylated. </p> <p> Cytokine array analysis demonstrated that IL6 is predominantly differentially expressed in the CM from OR cells compared to parental cells (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 19 </a> ). To assess the effect of OR-IL6 on STAT3 signaling, we treated PC9-P and HCC827-P cells with WT-IL6-CM (adjusted to 1 ng/mL) from PC9-IL6-WT and HCC827-IL6-WT cells, respectively, or OR-IL6-containing CM (adjusted to 1 ng/mL) from PC9-OR and HCC827-OR cells, respectively. WT-IL6-CM induced a stronger intensity of the STAT3-pY705 band at 24 h than that induced by OR-CM (Fig. <a href=\"/articles/s41467-024-51831-7#Fig7\"> 7a </a> ) in both cell lines. Time-dependent IF imaging revealed that treatment with either WT-IL6-CM or OR-CM stimulated the nuclear import of STAT3 (magenta) at 0.5 h (Fig. <a href=\"/articles/s41467-024-51831-7#Fig7\"> 7b </a> ). Notably, WT-IL6-CM induced significantly longer nuclear retention of STAT3 than OR-CM up to 24 h (Fig. <a href=\"/articles/s41467-024-51831-7#Fig7\"> 7b </a> , magnified inset), whereas OR-CM stimulated stronger cell surface recruitment of SRC (green) than WT-IL6-CM throughout the time course, indicating that OR-IL6, with lower N-glycosylation, tends to activate SRC rather than STAT3 signals. This signaling alteration was also observed in the OR cells compared to their corresponding parental cells (Fig. <a href=\"/articles/s41467-024-51831-7#Fig7\"> 7c </a> ). </p> <div class=\"c-article-section__figure js-c-reading-companion-figures-item\" id=\"figure-7\"> <figure> <figcaption> <b class=\"c-article-section__figure-caption\" id=\"Fig7\"> Fig. 7: Osimertinib-induced deNG-IL6 activates the SRC-YAP-SOX2 axis and migration in TKI-resistant NSCLC. </b> </figcaption> <div class=\"c-article-section__figure-content\"> <div class=\"c-article-section__figure-item\"> <a class=\"c-article-section__figure-link\" href=\"/articles/s41467-024-51831-7/figures/7\" rel=\"nofollow\"> <picture> <source srcset=\"//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-024-51831-7/MediaObjects/41467_2024_51831_Fig7_HTML.png?as=webp\" type=\"image/webp\"/> </picture> </a> </div> <div class=\"c-article-section__figure-description\" id=\"figure-7-desc\"> <p> <b> a </b> STAT3 and STAT3-pY705 levels induced by PC9-IL6-CM, PC9-OR-CM, HCC827-IL6-CM, or HCC827-OR-CM (IL6 = 1 ng/mL) over time in PC9-P or HCC827-P cells. <i> n </i> = 3 independent experiments, mean ± SEM, two-tailed unpaired <i> t </i> -test, PC9: *** <i> p </i> &lt; 0.0001, ** <i> p </i> = 0.0044, *** <i> p </i> &lt; 0.0001, *** <i> p </i> = 0.0001; HCC-827: * <i> p </i> = 0.0324, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001. The samples derived from the same experiment but different gels for STAT3 and STAT3-pY705 were processed in parallel. <b> b </b> Intracellular localization of STAT3 and SRC in PC9-P or HCC827-P cells treated with PC9-IL6-CM, PC9-OR-CM, HCC827-IL6-CM, or HCC827-OR-CM (IL6 = 1 ng/mL) over time, as determined by IF. The magnified insets show STAT3 or SRC localization at 24 h. Scale bars, 20 μm. <i> n </i> = 3 independent experiments, mean ± SEM, two-tailed unpaired <i> t </i> -test, PC9-STAT3: ** <i> p </i> = 0.0085, *** <i> p </i> = 0.002, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001; PC9-SRC: *** <i> p </i> = 0.0002, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001, *** <i> p </i> = 0.0002; HCC827-STAT3: n.s. <i> p </i> = 0.1262, ** <i> p </i> = 0.0018, ** <i> p </i> = 0.0025, *** <i> p </i> = 0.0001; HCC827-SRC: n.s. <i> p </i> = 0.0803, *** <i> p </i> = 0.0007, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001. <b> c </b> Intracellular localization of STAT3 and SRC in PC9-OR and HCC827-OR cells as determined based on IF images acquired with an FV3000 microscope (60x). Images are representative of <i> n </i> = 3 independent experiments. Scale bars, 20 μm. <b> d </b> Expression of proteins indicating the activation of SRC and YAP in the cytosolic and nuclear fractions of the parental and OR PC9 and HCC827 cells. <i> n </i> = 3 independent experiments, expression fold change (FC) relative to corresponding parental cells, mean ± SEM, two-tailed unpaired <i> t </i> -test, PC9-SRCpY416: *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001; PC9-SRC: *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001; PC9-YAPpY357: *** <i> p </i> &lt; 0.0001, ***p &lt; 0.0001; PC9-YAP: *** <i> p </i> &lt; 0.0001, *** <i> p </i> = 0.0004; HCC827-SRCpY416: *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001; HCC827-SRC: n.s. <i> p </i> = 0.8569, *** <i> p </i> &lt; 0.0001; HCC827-YAP-pY357: *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001; HCC827-YAP: *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001. The samples derived from the same experiment but different gels for SRCpY416, SRC, YAPpY357, YAP, β-ACTIN, and LAMIN A/C were processed in parallel. <b> e </b> Time-dependent survival of the PC9-OR and HCC827-OR cells under treatment with osimertinib (5 μM) in combination with VP (5 μM). <i> n </i> = 4 independent experiments, mean ± SEM, two-tailed unpaired <i> t </i> -test, PC9-OR: n.s. <i> p </i> = 0.7764, *** <i> p </i> = 0.0006, *** <i> p </i> = 0.0001; HCC827-OR: n.s. <i> p </i> = 0.9049, *** <i> p </i> &lt; 0.0001, *** <i> p </i> &lt; 0.0001. <b> f </b> Analysis of the synergistic effect in PC9-OR and HCC827-OR cells treated with osimertinib in combination with DMSO or VP at various concentration combinations by Synergy Finder Plus under the two-tailed Zero interaction potency (ZIP) model. <b> g </b> Migration of PC9-IL6, PC9-OR, HCC827-IL6, and HCC827-OR cells through Transwell membranes. <i> n </i> = 4 independent experiments, mean ± SEM, two-tailed unpaired <i> t </i> -test, PC9: *** <i> p </i> = 0.0001, HCC827: *** <i> p </i> &lt; 0.0001. Source data are provided as a Source Data file. </p> </div> </div> <div class=\"u-text-right u-hide-print\"> <a aria-label=\"Full size image figure 7\" class=\"c-article__pill-button\" href=\"/articles/s41467-024-51831-7/figures/7\" rel=\"nofollow\"> <span> Full size image </span> </a> </div> </figure> </div> <p> We further examined the expression of proteins involved in SRC-YAP signaling in the OR cells and the corresponding parental cells (Fig. <a href=\"/articles/s41467-024-51831-7#Fig7\"> 7d </a> ). Compared to the PC9-P cells, the PC9-OR cells had an increased amount of SRCpY416 in the cytosol. In addition, in the PC9-OR cells, the levels of YAPpY357 were increased in both the cytosolic and nuclear fractions. Similarly, the levels of cytosolic SRCpY416 and YAPpY357 and nuclear YAP and YAPpY357 were elevated in the HCC827-OR cells compared to the HCC827-P cells. Next, to evaluate the requirement of YAP in osimertinib resistance, we treated the PC9-OR and HCC827-OR cells with fixed-dose osimertinib (5 μM) in combination with the YAP inhibitor (YAPi) verteporfin (VP) at 5 μM for different durations (Fig. <a href=\"/articles/s41467-024-51831-7#Fig7\"> 7e </a> ). VP significantly increased sensitivity to osimertinib in PC9-OR and HCC827-OR cells in a time-dependent manner (Fig. <a href=\"/articles/s41467-024-51831-7#Fig7\"> 7e </a> ). When the OR cells were simultaneously treated with osimertinib and the YAPi at various concentration combinations, a synergistic cytotoxic effect (as determined by Synergy Finder Plus: <a href=\"http://www.synergyfinderplus.org/\"> http://www.synergyfinderplus.org/ </a> ) <sup> <a aria-label=\"Reference 58\" href=\"/articles/s41467-024-51831-7#ref-CR58\" id=\"ref-link-section-d6776021e3418\" title=\"Zheng, S. et al. SynergyFinder plus: toward better interpretation and annotation of drug combination screening datasets. Genom. Proteom. Bioinform. 20, 587–596 (2022).\"> 58 </a> </sup> was observed (Fig. <a href=\"/articles/s41467-024-51831-7#Fig7\"> 7f </a> ), indicating that YAP is essential to the maintenance of osimertinib resistance. With the evidence that deNG-IL6-mediated SRC-YAP activation promotes cell migration (Figs. <a href=\"/articles/s41467-024-51831-7#Fig3\"> 3 </a> – <a href=\"/articles/s41467-024-51831-7#Fig5\"> 5 </a> ), we assessed the effects of OR-IL6 on cell migration (Fig. <a href=\"/articles/s41467-024-51831-7#Fig7\"> 7g </a> and Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 23 </a> ). The PC9-OR and HCC827-OR cells showed stronger migratory capacities than the PC9-IL6-WT and HCC827-IL6-WT cells, respectively. These data suggest that deNG-IL6 secreted by OR cells not only sustains cell survival (Fig. <a href=\"/articles/s41467-024-51831-7#Fig6\"> 6e, f </a> ) but also favors cell migration (Fig. <a href=\"/articles/s41467-024-51831-7#Fig7\"> 7g </a> and Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 23 </a> ). </p> <p> To demonstrate the presence of deNG-IL6 in NSCLC patients, we isolated MPECs from MPEs of patients who were naïve (before treatment) or resistant (after development of resistance) to EGFR TKIs and performed short-term culture (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 24 </a> ). Secreted IL6 in the CM was collected by IP and analyzed by WB. TKI-naïve MPECs secreted more H-IL6 than L-IL6, whereas the majority of IL6 from TKI-resistant MPECs was L-IL6 (Fig. <a href=\"/articles/s41467-024-51831-7#Fig8\"> 8a </a> ). Because the quantity of MPEC-derived IL6 was not enough for lectin array analysis, we analyzed them by WGA blotting. Only H-IL6 from TKI-naïve MPECs was detected by WGA blotting (Fig. <a href=\"/articles/s41467-024-51831-7#Fig8\"> 8b </a> ). Quantification results showed a reduction in the proportion of NG-IL6 and an increase in deNG-IL6 in the CM of EGFR TKI-resistant MPECs (Fig. <a href=\"/articles/s41467-024-51831-7#Fig8\"> 8c </a> ). </p> <div class=\"c-article-section__figure js-c-reading-companion-figures-item\" id=\"figure-8\"> <figure> <figcaption> <b class=\"c-article-section__figure-caption\" id=\"Fig8\"> Fig. 8: EGFR TKI-resistant NSCLC patients show high deNG-IL6 expression and a shift signaling preference from JAK-STAT3 to the SRC-YAP axis. </b> </figcaption> <div class=\"c-article-section__figure-content\"> <div class=\"c-article-section__figure-item\"> <a class=\"c-article-section__figure-link\" href=\"/articles/s41467-024-51831-7/figures/8\" rel=\"nofollow\"> <picture> <source srcset=\"//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-024-51831-7/MediaObjects/41467_2024_51831_Fig8_HTML.png?as=webp\" type=\"image/webp\"/> </picture> </a> </div> <div class=\"c-article-section__figure-description\" id=\"figure-8-desc\"> <p> <b> a </b> Pattern of secreted IL6 in MPECs derived from MPE of EGFR-TKI-naïve (TN01-TN04) and EGFR-TKI-resistant (TR01-TR05) patients as detected by WB ( <i> n </i> = 1). <b> b </b> N-glycosylated IL6 detected by lectin blotting with WGA (n = 1). <b> c </b> Percentage proportions of NG-IL6 and deNG-IL6 in the CM of EGFR-TKI-naïve and EGFR-TKI-resistant MPECs ( <i> n </i> = 1). <b> d </b> GSEA enrichment plot showing genes related to the N-glycosylation process derived from transcriptome data for patient tumors before and after the development of osimertinib resistance <sup> <a aria-label=\"Reference 58\" href=\"/articles/s41467-024-51831-7#ref-CR58\" id=\"ref-link-section-d6776021e3493\" title=\"Zheng, S. et al. SynergyFinder plus: toward better interpretation and annotation of drug combination screening datasets. Genom. Proteom. Bioinform. 20, 587–596 (2022).\"> 58 </a> </sup> named OSTu and ORTu, respectively. Sequential process by which 15 patients were selected according to the indicated criteria (left panel). The right panel shows the GSEA enrichment plots for N-glycosylation signature genes (right panel, top) and heatmaps for the enrichment scores (right panel, bottom) of selected patients (determined by Morpheus). Nominal <i> p </i> -value was one-tailed test on the appropriate side of the null distribution. <b> e </b> Expression and intracellular location of STAT3 and SRC in paired lung cancer tissues from six NSCLC patients (TKI_01-TKI_06) before (TKI-naïve) and after development of resistance (TKI resistant) to EGFR TKI as determined by TissueFAXS-imaged IF (see also Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 28 </a> for H&amp;E staining). The paired tissues from the same patient were attached to the same slide to perform IF staining. Images are selected magnified region from the full tissue. Scale bars, 20 μm. <b> f </b> Expression of nuclear STAT3 and membranous SRC as quantified by TissueQuest. Source data are provided as a Source Data file. </p> </div> </div> <div class=\"u-text-right u-hide-print\"> <a aria-label=\"Full size image figure 8\" class=\"c-article__pill-button\" href=\"/articles/s41467-024-51831-7/figures/8\" rel=\"nofollow\"> <span> Full size image </span> </a> </div> </figure> </div> <p> To elucidate the clinical correlation between the reduced expression of N-OSTs and osimertinib resistance, we analyzed online-available RNA sequencing (RNA-seq) data obtained from the lung tumors of fifteen patients who received osimertinib treatment <sup> <a aria-label=\"Reference 59\" href=\"/articles/s41467-024-51831-7#ref-CR59\" id=\"ref-link-section-d6776021e3520\" title=\"Roper, N. et al. Clonal evolution and heterogeneity of osimertinib acquired resistance mechanisms in EGFR mutant lung cancer. Cell Rep. Med. 1, 10007 (2020).\"> 59 </a> </sup> and developed resistance. Of the fifteen RNA-seq datasets, only ten provided paired data of tumors collected before and after the acquisition of resistance (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 25 </a> ), corresponding to <u> o </u> simertinib- <u> s </u> ensitive (OS) or OR tumors, namely, OSTu and ORTu, respectively. Notably, GSEA showed that the N-glycosylation process was enhanced in OSTu and decreased in ORTu (Fig. <a href=\"/articles/s41467-024-51831-7#Fig8\"> 8d </a> , upper right). </p> <p> Resistance mechanisms of the ORTu were characterized by <u> w </u> hole- <u> e </u> xome <u> s </u> equencing (WES) <sup> <a aria-label=\"Reference 59\" href=\"/articles/s41467-024-51831-7#ref-CR59\" id=\"ref-link-section-d6776021e3549\" title=\"Roper, N. et al. Clonal evolution and heterogeneity of osimertinib acquired resistance mechanisms in EGFR mutant lung cancer. Cell Rep. Med. 1, 10007 (2020).\"> 59 </a> </sup> . In most cases, resistance was acquired via known genomic alterations, such as <i> EGFR </i> C797S and the amplification of <i> KRAS </i> and <i> MET </i> (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 25 </a> ), whereas some developed through an undefined mechanism. We excluded ORTus that harbored known genomic mutations and obtained a group of three datasets from three patients: LAT010 (mixed NSCLC and SCLC transformation), LAT011 ( <u> n </u> euro <u> e </u> ndocrine differentiation, NE), and LAT017 ( <i> EGFR </i> and <i> YES1 </i> amplification). LAT010 and LAT011 were categorized as EMT-related cases based on transformation to the aggressive NE subtype <sup> <a aria-label=\"Reference 13\" href=\"/articles/s41467-024-51831-7#ref-CR13\" id=\"ref-link-section-d6776021e3579\" title=\"Quintanal-Villalonga, A. et al. Lineage plasticity in cancer: a shared pathway of therapeutic resistance. Nat. Rev. Clin. Oncol. 17, 360–371 (2020).\"> 13 </a> , <a aria-label=\"Reference 18\" href=\"/articles/s41467-024-51831-7#ref-CR18\" id=\"ref-link-section-d6776021e3582\" title=\"Sequist, L. V. et al. Genotypic and histological evolution of lung cancers acquiring resistance to EGFR inhibitors. Sci. Transl. Med. 3, 75ra26 (2011).\"> 18 </a> , <a aria-label=\"Reference 20\" href=\"/articles/s41467-024-51831-7#ref-CR20\" id=\"ref-link-section-d6776021e3585\" title=\"Williams, E. D., Gao, D., Redfern, A. &amp; Thompson, E. W. Controversies around epithelial-mesenchymal plasticity in cancer metastasis. Nat. Rev. Cancer 19, 716–732 (2019).\"> 20 </a> </sup> . </p> <p> We then analyzed the enrichment in six gene sets of interest related to the N-glycosylation process, EMT, and SRC-YAP-related signaling by single-sample GSEA (ssGSEA) <sup> <a aria-label=\"Reference 60\" href=\"/articles/s41467-024-51831-7#ref-CR60\" id=\"ref-link-section-d6776021e3592\" title=\"Barbie, D. A. et al. Systematic RNA interference reveals that oncogenic KRAS-driven cancers require TBK1. Nature 462, 108–112 (2009).\"> 60 </a> </sup> . The ssGSEA scores for these processes for each patient were compiled into a heatmap. Compared to the OSTu, the ORTu in the three patients exhibited a decreased N-glycosylation process and enhanced EMT traits and SRC-YAP signaling (Fig. <a href=\"/articles/s41467-024-51831-7#Fig8\"> 8d </a> , lower right). This difference was more pronounced after subtracting the ssGSEA scores from the OSTu. Among the three patients, the most significant difference between OSTu and ORTu was observed in patient LAT017, who showed <i> YES1 </i> amplification (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 25 </a> ). </p> <p> YES1 is an SRC family kinase that phosphorylates YAP at Y357, promoting its nuclear translocation and transcriptional activity and thus facilitating the progression of cancer <sup> <a aria-label=\"Reference 61\" href=\"/articles/s41467-024-51831-7#ref-CR61\" id=\"ref-link-section-d6776021e3608\" title=\"Heinz, M. C. et al. Liver colonization by colorectal cancer metastases requires YAP-controlled plasticity at the micrometastatic stage. Cancer Res. 82, 1953–1968 (2022).\"> 61 </a> , <a aria-label=\"Reference 62\" href=\"/articles/s41467-024-51831-7#ref-CR62\" id=\"ref-link-section-d6776021e3611\" title=\"Hamanaka, N. et al. YES1 is a targetable oncogene in cancers harboring YES1 gene amplification. Cancer Res. 79, 5734–5745 (2019).\"> 62 </a> </sup> . Based on this information and because we demonstrated that deNG-IL6 triggers EMT and metastasis via alternative activation of the SRC-YAP-SOX2 axis in previous experiments, we further analyzed the RNA-seq data from patient LAT017 (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 26a </a> ). Consistent with our in vitro and in vivo data, the expression of N-glycosylation-related genes was reduced after acquisition of osimertinib resistance; moreover, signatures of EMT and activation of the SRC and YAP pathways were simultaneously enriched (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 26a </a> ). We also analyzed the expression of transcription factors (TFs) within the 1 K bases upstream of six N-OST genes in patient LAT017 (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 26b, c </a> ) and revealed concordant downregulation of TF expression in the ORTu (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 26d </a> ). </p> <p> We further used paired lung cancer tissues obtained at EGFR-TKI-naïve and EGFR-TKI-resistant statuses from six patients (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 27 </a> , Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 28 </a> , and Fig. <a href=\"/articles/s41467-024-51831-7#Fig8\"> 8e </a> ). In the EGFR-TKI-naïve tissues, we observed strong nuclear STAT3 (Fig. <a href=\"/articles/s41467-024-51831-7#Fig8\"> 8e </a> , upper panel) but weak membranous SRC expression. In contrast, in the EGFR-TKI-resistant group, the expression of membranous SRC was prominent, and nuclear STAT3 staining was barely observed. (Fig. <a href=\"/articles/s41467-024-51831-7#Fig8\"> 8e </a> , lower panel). The difference was highlighted by TissueQuest quantification (Fig. <a href=\"/articles/s41467-024-51831-7#Fig8\"> 8f </a> ). </p> <p> In summary, we demonstrated that in NSCLC, IL6 undergoes N-glycosylation primarily at the N73 residue and O-glycosylation at the T170 residue. The loss of glycan on N73 leads to a switch of downstream signal preference from the JAK-STAT3 axis to the SRC-YAP-SOX2 axis, thereby triggering cell plasticity. Additionally, we have uncovered a bypass mechanism that drives osimertinib resistance by reducing the N-glycosylation of IL6, consequently shifting the survival-dependent pathway from the JAK-STAT3 axis to the SRC-YAP-SOX2 axis (Fig. <a href=\"/articles/s41467-024-51831-7#Fig9\"> 9 </a> ). </p> <div class=\"c-article-section__figure js-c-reading-companion-figures-item\" id=\"figure-9\"> <figure> <figcaption> <b class=\"c-article-section__figure-caption\" id=\"Fig9\"> Fig. 9: Long-Term EGFR TKI Treatment Induces deNG-IL6 to Drive Resistance. </b> </figcaption> <div class=\"c-article-section__figure-content\"> <div class=\"c-article-section__figure-item\"> <a class=\"c-article-section__figure-link\" href=\"/articles/s41467-024-51831-7/figures/9\" rel=\"nofollow\"> <picture> <source srcset=\"//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41467-024-51831-7/MediaObjects/41467_2024_51831_Fig9_HTML.png?as=webp\" type=\"image/webp\"/> </picture> </a> </div> <div class=\"c-article-section__figure-description\" id=\"figure-9-desc\"> <p> In non-small cell lung cancer (NSCLC) cells, autocrine IL6 (NG-IL6) predominantly exhibits N-glycosylation at residue N73 and O-glycosylation at residue T170 in an EGFR TKI-naïve environment. NG-IL6 predominantly activates the canonical JAK-STAT3 axis, promoting cell proliferation. However, long-term treatment with EGFR TKIs leads to increased secretion of IL6 while concurrently suppressing the expression of N-glycosyltransferase complex components, primarily STT3A or STT3B, RPN1/2, DAD1, and DDOST. This suppression results in diminished N-glycosylation at N73 of NG-IL6, which subsequently reduces N-glycosylation at N73 of NG-IL6. Consequently, the proportion of NG-IL6 decreases, while the proportion of deNG-IL6 increases upon the acquisition of resistance. deNG-IL6, in turn, shifts signaling preference from the JAK-STAT3 axis to the SRC-YAP-SOX2 axis. The alternatively activated SRC-YAP-SOX2 signaling facilitates cellular plasticity and contributes to the development of drug resistance. </p> </div> </div> <div class=\"u-text-right u-hide-print\"> <a aria-label=\"Full size image figure 9\" class=\"c-article__pill-button\" href=\"/articles/s41467-024-51831-7/figures/9\" rel=\"nofollow\"> <span> Full size image </span> </a> </div> </figure> </div> </div> </div> </section> <section> <div class=\"c-article-section\" id=\"Sec10-section\"> <h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"Sec10\"> Discussion </h2> <div class=\"c-article-section__content\" id=\"Sec10-content\"> <p> To date, how IL6 differentially regulates its multiple downstream signals and whether the variation of induced downstream signals has clinical implications remain unclear. Here, we showed that IL6 from both lung cancer cell lines and clinical specimens is modified by N-glycosylation. deNG-IL6 induces a switch in GP130 downstream signals from canonical JAK-STAT3 to the SRC-YAP-SOX2 axis. The changes resulted in different gene expression profiles and enhanced in vitro migration and in vivo lung metastasis. Moreover, we discovered that long-term treatment with the EGFR-TKI osimertinib induces a reduction in N-OST expression and an elevation of deNG-IL6, which results in EGFR-TKI resistance in lung cancer cells. </p> <p> In this study, two potential N-glycosites (N73 and N172) and four possible O-glycosites (T166, T171, T170, and S174) were predicted to exist in hIL6. MS results indicated that the glycan chains reside at N73, N172, and T170. Utilizing site-directed mutagenesis (Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2b </a> ) and NGI-1 treatment (Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2c </a> ), we identified four distinct IL6 bands via Western blot analysis, corresponding to four different glycosylation forms as follows (from top to bottom): (1) NG-IL6 carrying an N-glycan (N73 or N172) and an O-glycan (T170), (2) the band marked with an asterisk (*) likely representing N-glycosylation (N73 or N172) alone, (3) deNG-IL6 containing an O-glycan (T170) exclusively, and (4) deG-IL6 lacking any glycan. Notably, the NG-IL6 band was nearly absent in the analysis of N73Q-IL6, suggesting that the predominant attachment sites for IL6 glycan chains are N73 + T170 rather than N172 + T170. Moreover, no discernible bands higher than NG-IL6 were observed, indicating that tri-glycosylated IL6 (N73 + N172 + T170) may be either non-existent or negligible. This observation is further supported by MS data obtained from HCC827 WT-IL6 (Fig. <a href=\"/articles/s41467-024-51831-7#Fig6\"> 6l </a> and Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 22 </a> ), which revealed that the N-glycans of NG-IL6 originate mainly from N73 (&gt;85%) rather than N172 (&lt;15%). </p> <p> N73 emerges as the predominant N-glycosylation site, a conclusion reinforced by the MS-based glycan analysis of WT-IL6 (Fig. <a href=\"/articles/s41467-024-51831-7#Fig6\"> 6l </a> ). However, lectin microarray analysis revealed only an ~50% reduction in the signal intensity of N-glycan in N73Q-IL6 (Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2d </a> ), presenting a somewhat contradictory outcome. This discrepancy is likely due to the utilization of a polyclonal anti-IL6 antibody for the IP-based isolation process. Consequently, the resultant mixture possibly encompassed not only the desired N73Q-IL6 but also other components, such as antibody fragments and/or nonspecific proteins co-precipitated from the conditioned medium. </p> <p> Additionally, AS2-IL6-N172Q cells exhibited no significant difference in migration capacity compared to AS2-IL6-WT cells (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 14 </a> ), suggesting that the impact of N172 glycosylation on cell migration is minimal. Our findings substantiate the notion that N73 is the principle N-glycosite in IL6, with the abundance of attached N-glycans reduced by approximately one-third when the cells acquired resistance to osimertinib. </p> <p> Intriguingly, N172Q-IL6 exhibited four distinct bands on the Western blot: the NG-IL6, deNG-IL6, and deG-IL6 bands and a faint asterisk-labeled band between the NG-IL6 and deNG-IL6 bands (Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2b </a> ). Notably, the distance between the asterisk-labeled band and the deG-IL6 band mirrors that between the NG-IL6 and deNG-IL6 bands (Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2b </a> ). Furthermore, treatment with NGI-1 resulted in a reduction in the intensity of the faint asterisk-labeled band (Fig. <a href=\"/articles/s41467-024-51831-7#Fig2\"> 2c </a> ), suggesting that this faint band possibly represents IL6 N-glycosylation solely at N73. Regarding N172Q-IL6, the appearance of the asterisk-labeled band and the increase in deG-IL6 band intensity imply that the change in the amino acid at residue 172 may interfere with glycosylation at T170. </p> <p> In the CM of established lung cancer cell lines, it appears that deNG-IL6 (L-IL6) is the predominant species. However, in the CM of cancer cells from patient MPECs, most cases showed equal expression of the two species (Fig. <a href=\"/articles/s41467-024-51831-7#Fig1\"> 1b </a> ). In our studies, blockade of the IL6 receptors GP130 (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 7 </a> ) and GP80 (Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 8 </a> ) reduced the STAT3 activity induced by both WT-IL6 and N73Q-IL6 in various cell lines. Notably, the inhibition was more effective in the N73Q-IL6-treated cells than in the WT-IL6-treated cells, suggesting that WT-IL6 may have a stronger binding affinity to its receptors and be more resistant to antibody blockade. We assume that NG-IL6, with stronger receptor binding ability, even with only 30% occupancy in CM, may outperform deNG-IL6 in conducting signal transduction. However, when the occupancy of NG-IL6 in CM is reduced, deNG-IL6 becomes dominant in signaling. </p> <p> Targeting N-glycosylation by NGI-1 combined with EGFR-TKIs has been shown to overcome EGFR-TKI resistance in NSCLC <sup> <a aria-label=\"Reference 63\" href=\"/articles/s41467-024-51831-7#ref-CR63\" id=\"ref-link-section-d6776021e3741\" title=\"Lopez Sambrooks, C. et al. Oligosaccharyltransferase inhibition overcomes therapeutic resistance to EGFR tyrosine kinase inhibitors. Cancer Res. 78, 5094–5106 (2018).\"> 63 </a> </sup> . The study concluded that the tumor growth delay was due to N-glycosylation-defective EGFR reducing the activation of receptors and dissociating EGFR signaling from other coexpressed receptors. In hepatocellular carcinoma, IL6-induced JAK-1 phosphorylates PD-L1, enhancing the recruitment of STT3A and thus increasing PD-L1 N-glycosylation, which favors the stability of PD-L1 and promotes immune evasion <sup> <a aria-label=\"Reference 64\" href=\"/articles/s41467-024-51831-7#ref-CR64\" id=\"ref-link-section-d6776021e3745\" title=\"Chan, L. C. et al. IL-6/JAK1 pathway drives PD-L1 Y112 phosphorylation to promote cancer immune evasion. J. Clin. Invest. 129, 3324–3338 (2019).\"> 64 </a> </sup> . These studies indicated that elevated N-glycosylation and glycosyltransferase expression resulted in unfavorable disease progression and poor survival in cancer patients. However, some reports have shown that reduced glycosylation could promote the progression of malignancies. For example, downregulation of the levels of the dystroglycan glycosyltransferases LARGE2 and ISPD was found to be associated with poor survival in patients with clear cell renal cell carcinoma <sup> <a aria-label=\"Reference 65\" href=\"/articles/s41467-024-51831-7#ref-CR65\" id=\"ref-link-section-d6776021e3749\" title=\"Miller, M. R. et al. Downregulation of dystroglycan glycosyltransferases LARGE2 and ISPD associate with increased mortality in clear cell renal cell carcinoma. Mol. Cancer 14, 141 (2015).\"> 65 </a> </sup> , and loss of the polypeptide N-acetylgalactosaminyltransferase-3 (GALNT3) in pancreatic adenocarcinoma was associated with increased aggressiveness of cancer cells <sup> <a aria-label=\"Reference 66\" href=\"/articles/s41467-024-51831-7#ref-CR66\" id=\"ref-link-section-d6776021e3753\" title=\"Chugh, S., Meza, J., Sheinin, Y. M., Ponnusamy, M. P. &amp; Batra, S. K. Loss of N-acetylgalactosaminyltransferase 3 in poorly differentiated pancreatic cancer: augmented aggressiveness and aberrant ErbB family glycosylation. Br. J. Cancer 114, 1376–1386 (2016).\"> 66 </a> </sup> . Accordingly, we found that reduced N-glycosylation on IL6 led to a high metastatic potential and resistance to EGFR-TKIs in NSCLC cells. </p> <p> Peng and colleagues showed that high environmental glucose acts as a stressor that induces the O-GlcNAcylation of intracellular YAP-1 and promotes tumorigenicity <sup> <a aria-label=\"Reference 67\" href=\"/articles/s41467-024-51831-7#ref-CR67\" id=\"ref-link-section-d6776021e3760\" title=\"Peng, C. et al. Regulation of the Hippo-YAP pathway by glucose sensor O-GlcNAcylation. Mol. Cell 68, 591–604.e595 (2017).\"> 67 </a> </sup> , suggesting that stress from the tumor microenvironment can shape the glycosylation pattern of key proteins in cancer cells. Our in silico data indicated that a group of specific TFs may act as TKI stress sensors and repress the expression of N-OSTs. Reduced N-OST expression may result in globally defective N-linked glycosylation of the cellular proteome and subsequently change cell behaviors. Interestingly, we found that defective N-linked glycosylation of a key regulator, such as IL6, is sufficient to induce prominent biological changes such as EMT, migration, metastasis, and drug resistance in NSCLC cells. </p> <p> Collectively, our data revealed that IL6 is modified with N-glycosylation mainly on N73 residue and O-glycosylation on T170 residue in NSCLC. The loss of glycan on N73 induces a switch of downstream signal preference from the JAK-STAT3 axis to the SRC-YAP-SOX2 axis, triggers subsequent cell plasticity. We also demonstrated that an increase in deNG-IL6 production can be induced by environmental stressors, such as EGFR-TKI treatment. This change in the glycosylation status of IL6 can switch the downstream signaling preference of the GP130 receptor from JAK-STAT3 to the SRC-YAP-SOX2 axis. Lung cancer cells with high levels of deNG-IL6 tend to possess EMT-related properties, metastasize to other organs, and exhibit EGFR-TKI resistance. These findings raise the possibility of controlling cancer cell behavior by simply changing the glycosylation pattern on a key regulator and predicting the appearance of EGFR-TKI resistance by monitoring circulating IL6 glycoforms in NSCLC patients. </p> </div> </div> </section> <section> <div class=\"c-article-section\" id=\"Sec11-section\"> <h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"Sec11\"> Methods </h2> <div class=\"c-article-section__content\" id=\"Sec11-content\"> <h3 class=\"c-article__sub-heading\" id=\"Sec12\"> Human <u> m </u> alignant <u> p </u> leural <u> e </u> ffusion (MPE) </h3> <p> The MPE samples from patients who underwent thoracentesis or thoracotomy at National Cheng Kung University Hospital were collected as previously described <sup> <a aria-label=\"Reference 68\" href=\"/articles/s41467-024-51831-7#ref-CR68\" id=\"ref-link-section-d6776021e3789\" title=\"Hung, T. L. et al. Clinical evaluation of HER-2/neu protein in malignant pleural effusion-associated lung adenocarcinoma and as a tumor marker in pleural effusion diagnosis. Clin. Cancer Res. 9, 2605–2612 (2003).\"> 68 </a> </sup> . The TKI-naïve MPEs were collected from patients with histologically proven NSCLC and symptomatic malignant pleural effusion at the time of initial diagnosis, <i> n </i> = 19, female = 13, male = 6, mean age = 64.8 (49, 88). TKI-resistant MPEs were collected from patients who received EGFR TKI treatment and developed resistance, <i> n </i> = 5, female = 4, male = 1, mean age = 67.4 (54, 92). Paired TKI-naïve and TKI-resistant NSCLC lung tissues were obtained from the Human Biobank of National Cheng Kung University Hospital, <i> n </i> = 6, female = 4, male = 2, TKI-naïve mean age = 59.67 (45, 71); TKI-resistant mean age = 61.67 (48, 73). Source data are provided as a Source Data file. No sex or gender analysis was carried out. All subjects gave informed consent to publish those data before contributing samples to the Human Biobank of National Cheng Kung University Hospital. </p> <h3 class=\"c-article__sub-heading\" id=\"Sec13\"> Cell lines </h3> <p> The following human lung cancer cell lines were purchased from the American Type Culture Collection: adenocarcinoma (A549, RRID:CVCL_0023; H1299, RRID:CVCL_0060; HCC827, RRID:CVCL_2063; H1650, RRID:CVCL_1483; and H1975, RRID:CVCL_1511), large cell carcinoma (H460, RRID:CVCL_0459), squamous cell carcinoma (H226, RRID:CVCL_1544), and normal human bronchial epithelial cell lines (NL-20, RRID:CVCL_3756; Beas-2B, RRID:CVCL_0168). The human lung adenocarcinoma cell line PC14PE6 was provided by Dr. Fidler (MD Anderson Cancer Center, Houston, TX, USA). PC14PE6/AS2 (AS2) <sup> <a aria-label=\"Reference 39\" href=\"/articles/s41467-024-51831-7#ref-CR39\" id=\"ref-link-section-d6776021e3810\" title=\"Yeh, H. H., Lai, W. W., Chen, H. H., Liu, H. S. &amp; Su, W. C. Autocrine IL-6-induced Stat3 activation contributes to the pathogenesis of lung adenocarcinoma and malignant pleural effusion. Oncogene 25, 4300–4309 (2006).\"> 39 </a> </sup> was established from ascites in SCID mice generated by inoculation of PC14PE6 cells into the peritoneal cavity. Human lung adenocarcinoma cell lines PC9 (RRID:CVCL_B260) and CL1-0 (RRID:CVCL_3871) <sup> <a aria-label=\"Reference 69\" href=\"/articles/s41467-024-51831-7#ref-CR69\" id=\"ref-link-section-d6776021e3814\" title=\"Chu, Y. W. et al. Selection of invasive and metastatic subpopulations from a human lung adenocarcinoma cell line. Am. J. Respir. Cell Mol. Biol. 17, 353–360 (1997).\"> 69 </a> </sup> were kindly provided by Dr. Chih-Hsin Yang (National Taiwan University, Taipei, Taiwan) and Dr. Pan-Chyr Yang (National Taiwan University), respectively. All cell lines were authenticated by short tandem repeat-polymerase chain reaction (STR-PCR) at the Biosource Collection and Research Center (BCRC) and the Center for Genomic Medicine at National Cheng-Kung University, Taiwan. </p> <h3 class=\"c-article__sub-heading\" id=\"Sec14\"> Mouse models </h3> <p> Six- to eight-week-old male and female nude mice (BALB/cAnN.Cg-Foxn1nu/CrlNarl: <a href=\"https://www.nlac.narl.org.tw/p3_animal2_detail.asp?cid=1&amp;nid=9&amp;ppage=0&amp;type=2\"> https://www.nlac.narl.org.tw/p3_animal2_detail.asp?cid=1&amp;nid=9&amp;ppage=0&amp;type=2 </a> ) were purchased from the National Laboratory Animal Center (Taipei, Taiwan). </p> <h3 class=\"c-article__sub-heading\" id=\"Sec15\"> Cell culture </h3> <p> PC9, H1650, H1299, HCC827, H1975, H226, H460 and CL1-0 cells were maintained in RPMI 1640 medium (Gibco) supplemented with 10% <u> f </u> etal <u> b </u> ovine <u> s </u> erum (FBS; Gibco) and 1% antibiotic-antimycotic (anti-anti, Gibco), and A549 cells were maintained in F12K medium (Gibco) supplemented with 10% FBS and 1% anti-anti. Beas-2B cells were maintained in BEBM supplemented with the components of a BEGM Kit (Lonza). The AS2 human lung adenocarcinoma cell line was maintained in MEM-α medium (Gibco) supplemented with 10% FBS, 1% anti-anti, 1% sodium pyruvate (Gibco), and 1× MEM vitamin solution (Gibco). Cells stably expressing empty vector, wild-type <i> IL6 </i> ( <i> IL6-WT </i> ), or N-glycosylation mutant <i> IL6 </i> ( <i> IL6-N73Q </i> , <i> IL6-N73A </i> , <i> IL6-N73V </i> , <i> IL6-N172Q </i> , and <i> IL6-N73Q </i> + <i> N172Q </i> ) were established by transfection with TurboFect Transfection Reagent (Thermo Fisher), followed by selection of individual clones with 500 μg/mL geneticin (G418) (Calbiochem). AS2-IL6-WT-pLuc and AS2-IL6-N73Q-pLuc cells were generated by transfection of AS2-IL6-WT and AS2-IL6-N73Q cells with the pGL4.50 (Luc2/CMV/Hygro) plasmid (Promega) through TurboFect (Thermo Fisher) and selection with 500 μg/mL hygromycin B (H3274, Sigma-Aldrich), respectively. Gene silencing was performed by transducing target cells with VSV-G particles harboring pLKO.1 empty vector or shRNA against human <i> SRC </i> , <i> YAP </i> , <i> SOX2 </i> , or <i> IL6 </i> mRNA (purchased from RNA Technology Platform and Gene Manipulation Core, Academia Sinica, Taipei, Taiwan) and selected using 2 μg/mL puromycin (Sigma-Aldrich). The <u> o </u> simertinib- <u> r </u> esistant (OR) cells were established by constant culture in medium containing 5 μM osimertinib. All cell lines were maintained in an incubator at 37 °C supplemented with 5% CO <sub> 2 </sub> . </p> <h3 class=\"c-article__sub-heading\" id=\"Sec16\"> Preparation of <u> c </u> onditioned <u> m </u> edium (CM) </h3> <p> A total of 2 × 10 <sup> 6 </sup> cells were seeded in a 10 cm dish with 11 mL of culture medium with or without specific treatment. The <u> c </u> onditioned <u> m </u> edium (CM) was collected at designated times and then centrifuged at 750 × <i> g </i> . The supernatant was collected and stored at −80 °C until further application. Intact or adjusted CM was applied according to the requirements of different experiments. For collection of the OR-CM, the OR cells were seeded in osimertinib-free culture medium. <i> For cell treatment and immunoprecipitation </i> , the IL6 concentration of the CM was measured by ELISAs and adjusted to 1 ng/mL with supplemented culture medium prior to use. <i> For the cytokine array </i> , intact or diluted CM was applied. For diluted CM, the IL6 concentration was adjusted to 1 ng/mL with supplemented culture medium prior to assay. </p> <h3 class=\"c-article__sub-heading\" id=\"Sec17\"> IL6 ELISA </h3> <p> One milliliter of the CM supernatant was used for measurement of IL6 concentration. An IL6 ELISA was performed using a human IL6 CytoSet ELISA kit (Life Technology, CHC1263) following the manufacturer’s protocol. </p> <h3 class=\"c-article__sub-heading\" id=\"Sec18\"> Prediction of glycosylation sites </h3> <p> The full-length IL6 protein sequence obtained from NCBI Protein (GenBank: AAC41704.1) was analyzed with the NetNGlyc 1.0 server (RRID:SCR_001570, <a href=\"https://services.healthtech.dtu.dk/service.php?NetNGlyc-1.0\"> https://services.healthtech.dtu.dk/service.php?NetNGlyc-1.0 </a> ) to identify potential N-linked glycosylation sites and with the NetOGlyc 4.0 server (RRID:SCR_009026, <a href=\"https://services.healthtech.dtu.dk/service.php?NetOGlyc-4.0\"> https://services.healthtech.dtu.dk/service.php?NetOGlyc-4.0 </a> ) to identify potential O-linked glycosylation sites <sup> <a aria-label=\"Reference 47\" href=\"/articles/s41467-024-51831-7#ref-CR47\" id=\"ref-link-section-d6776021e3964\" title=\"Blom, N., Sicheritz-Ponten, T., Gupta, R., Gammeltoft, S. &amp; Brunak, S. Prediction of post-translational glycosylation and phosphorylation of proteins from the amino acid sequence. Proteomics 4, 1633–1649 (2004).\"> 47 </a> </sup> . A default threshold score was applied, and the result was confirmed by nine of nine calculation networks showing jury agreement by their internal calculation assessment. Residues with higher prediction scores showed higher potential as N-glycosites or O-glycosites. </p> <h3 class=\"c-article__sub-heading\" id=\"Sec19\"> Protein extraction </h3> <h4 class=\"c-article__sub-heading c-article__sub-heading--small\" id=\"Sec20\"> Total cellular protein lysates </h4> <p> Cells were washed twice with ice-cold 1x phosphate-buffered saline (PBS) and lysed with 200 μL of radioimmunoprecipitation assay (RIPA) lysis buffer [50 mM Tris (pH 7.5), 1% (v/v) NP-40, 0.5% (v/v) sodium deoxycholate, 150 mM NaCl, 0.1% (v/v) sodium dodecyl sulfate (SDS), 10 mM Na <sub> 3 </sub> VO <sub> 4 </sub> , and protease inhibitor cocktail from Roche Applied Sciences]. Whole-cell lysates were cleared by centrifugation at 20,000 × <i> g </i> and 4 °C for 10 min, and protein concentrations were determined by the Bradford assay (Bio–Rad) according to the manufacturer’s protocol. </p> <h4 class=\"c-article__sub-heading c-article__sub-heading--small\" id=\"Sec21\"> Nuclear and cytosolic fractionation </h4> <p> Cells were collected by trypsinization and washed with 1x PBS three times. The cells were permeabilized with low salt buffer (10 mM HEPES, pH = 7.4, 10 mM KCl, 50 μg/mL digitonin) and then centrifuged at 750 × <i> g </i> and 4 °C for 10 min. The supernatant was the cytosolic fraction of cellular proteins. The pellets were then extracted with RIPA buffer followed by centrifugation at 20,000 × <i> g </i> and 4 °C for 10 min. The resulting supernatants were the nuclear fraction of cellular proteins. </p> <h3 class=\"c-article__sub-heading\" id=\"Sec22\"> Western blotting (WB) </h3> <p> For detection of changes in protein levels, whole-cell lysates containing 15 μg of protein were separated by 8–12% sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE). The transferred membranes were blocked with 5% skim milk and hybridized with primary Ab at 4 °C overnight. After hybridization with secondary Ab, the protein bands were visualized by enhanced chemiluminescence (ECL) and imaged by exposure to X-ray film (Fujifilm) or in an iBright CL1500 Imaging System (Thermo Fisher). The images were analyzed using Gel-Pro Analyzer 4.0 (Media Cybernetics, Inc.) and Prism (GraphPad, Inc., RRID:SCR_005375) software. All immunoblot analyses were performed in triplicate ( <i> n </i> = 3) except for those performed with patient specimens ( <i> n </i> = 1). The following Ab were used for WB: IL6 (Santa Cruz, sc-32296; RRID:AB_627804); STAT3 (BD, 610189; RRID:AB_397588); STAT3-pY705 (CST, 9131;RRID:AB_331757); SRC (CST, 2108; RRID_331137); SRC-pY416 (CST, 2101; RRID:AB_331697); YAP (CST, 4912; RRID:AB_2218911); YAP-pY357 (Abcam, ab62751; RRID:AB_956486); β-ACTIN (Millipore, MAB1501; RRID:AB_2223041); SLUG (Abcam, ab-27568; RRID:AB_777968); CLAUDIN1 (Life Technology, 374900; RRID:AB_2533323); E-CADHERIN (BD, 610181; RRID:AB_397581); N-CADHERIN (BD, 610921; RRID:AB_398236); α-SMA (Abcam, ab-5694; RRID:AB_2223021); AKT (CST, 4691; RRID:AB_915783); AKT-pY308 (CST, 4056; RRID:AB_331163); AKT-pY473 (CST, 9271; RRID:AB_329825); SOX2 (Santa Cruz, sc-365823; RRID:AB_10842165); ERK1/2 (Santa Cruz, sc-514302; RRID:AB_2571739); pERK (Santa Cruz, sc-7383; RRID:AB_627545); LAMIN A/C (Merck Millipore, 05-714; RRID:AB_309928); NANOG (GeneTex, GTX-627421; RRID:AB_11169335); OCT3/4 (Santa Cruz, sc-5279; RRID:AB_628051); LATS1 (CST, 3477; RRID:AB_2133513); pLATS (CST, 8654; RRID:AB_10971635); TAZ (Abcam, ab224239; RRID:AB_2889852); pTAZ (CST, 59971; RRID:AB_2799578); MST1 (CST, 3682; RRID:AB_2144632). </p> <h3 class=\"c-article__sub-heading\" id=\"Sec23\"> Immunoprecipitation (IP) </h3> <p> <i> For WB </i> , ten milliliters of adjusted CM supernatant was used for each IP reaction. The concentration of IL6 in the CM supernatant was measured by ELISAs and adjusted to 1 ng/mL with culture media containing supplements. In addition, 10 ng of <u> r </u> ecombinant <u> h </u> uman IL6 protein (rhIL6, Sino Biological, 10395-HNAE) was added to 10 mL culture media containing supplements as a positive control sample. The adjusted CM was then precleaned with nProtein A Sepharose beads (GE Healthcare, GE17-5280-01) to remove nonspecific binding. IL6 antibody (2 μg, GeneTex, GTX110527; RRID:AB_10721789) was used in each IP reaction, rotated at low speed overnight at 4 °C. The Ag-Ab complex was pulled down with 50 μL of nProtein A at 4 °C for 2 h and released in 26 μL of RIPA lysis buffer plus 6x sample buffer by boiling. The IL6 products were then subjected to WB for pattern analysis. <i> For the lectin microarray </i> , CM supernatant from a hollow fiber bioreactor in which the IL6 concentration was adjusted to 1 μg/mL was applied. A total amount of 100 μg IL6 was pulled down following the above procedures. </p> <h3 class=\"c-article__sub-heading\" id=\"Sec24\"> Cell treatment conditions </h3> <p> <i> For NGI-1 </i> (Sigma-Aldrich, SML1620) <i> treatment </i> , 2 × 10 <sup> 6 </sup> cells were seeded in a 10 cm dish and allowed to attach overnight. The medium was replaced with fresh culture medium containing NGI-1 at 10 μM. After 48 h of treatment, the CM was collected for analysis of the IL6 patterns. <i> For treatment of various CM </i> , cells were seeded and allowed to attach overnight in serum-free medium for 12 h (8 × 10 <sup> 5 </sup> cells/6-cm dish for WB, 1 × 10 <sup> 5 </sup> cells/chamber slide for IF imaging). The serum-free medium was subsequently replaced with CM containing different IL6 glycoforms adjusted to 1 ng/mL and treated for the designated duration. <i> For treatments with neutralizing antibodies </i> αGP130 (R&amp;D, MAB228; RRID:AB_2233737) and αGP80 (R&amp;D, MAB227; RRID:AB_2127908), AS2-Vec cells were pretreated with these antibodies for 2 h followed by treatment with WT-IL6 or N73Q-IL6 CM. For αIL6 (R&amp;D, MAB206; RRID:AB_2127617)-neutralizing antibody treatment, WT-IL6 or N73Q-IL6 CM was incubated with the antibody at 37 °C for 2 h and then added to AS2-Vec cells. <i> For cytotoxicity assays </i> , 4 × 10 <sup> 3 </sup> cells/well 96-well in plates were let attached overnight. To assess the time-dependent response, cells were treated with osimertinib at a tolerant concentration of 5 μM alone or in combination with the YAP inhibitor verteporfin (VP) at 5 μM for the indicated durations. To assess synergistic responses, cells were treated with VP alone at 0 – 10 μM or in combination with osimertinib at 0 – 5 μM. Survival of cells was evaluated with MTT (3-[4,5-dimethylthiazol-2-yl]-2,5 diphenyl tetrazolium bromide) assay. </p> <h3 class=\"c-article__sub-heading\" id=\"Sec25\"> Digestion of N-glycans on IL6 by PNGase F </h3> <p> IL6 in the CM previously described was immunoprecipitated. After elution, the SDS in the product was neutralized by the addition of 15% Triton X-100 (Sigma-Aldrich) to a final concentration of 0.7% before subsequent treatment. PNGase F (Sigma-Aldrich, P7367) was then added to the precipitated product (final concentration = 1 U/μg protein) with 10x buffer and ddH <sub> 2 </sub> O. The digestion reaction was performed by incubation at 37 °C for 72 h. The digested samples were subjected to WB for pattern analysis. </p> <h3 class=\"c-article__sub-heading\" id=\"Sec26\"> Isolation of tumor cells from MPEs </h3> <p> The purity of the attached tumor cells in the target fractions was determined by assessment of double-positive expression of TTF1 (Santa Cruz, sc-8762; RRID:AB_793530) and PanCK (Santa Cruz, sc-8018; RRID:AB_627396) under FV3000 confocal microscopy (Olympus). MPE-derived tumor cells were resuspended, cultured in RPMI 1640 medium supplemented with 10% FBS and maintained in an incubator at 37 °C with 5% CO <sub> 2 </sub> . Precipitated cells were cultured in 10 -cm dishes and allowed to attach for 24 h, after which the medium was replaced with fresh RPMI 1640 medium supplemented with 10% FBS and 1% Anti-Anti. </p> <h3 class=\"c-article__sub-heading\" id=\"Sec27\"> Plasmid cloning </h3> <p> Full-length human IL6 cDNA derived from A549 cells was cloned into the pRcCMV vector (Invitrogen, V750-20) using BamHI (NEB, R0136) and EcoRI (NEB, R0101) restriction sites, and the resulting vector was named pRcCMV-WT-IL6. Site-directed mutagenesis of IL6 by substitution of the N73 residue with Gln, Ala, or Val, was performed using the pRcCMV-WT-IL6 plasmid with a QuikChange Lightning Site-Directed Mutagenesis Kit (Agilent Technologies, 210518) according to the manufacturer’s instructions. The mutant forms were named pRcCMV-N73Q-IL6, pRcCMV-N73A-IL6, pRcCMV-N73V-IL6, pRcCMV-N172Q-IL6, and pRcCMV-N73Q + N172Q-IL6, respectively. The primers used for site-directed mutagenesis are listed in Supplementary Table <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 2 </a> . </p> <h3 class=\"c-article__sub-heading\" id=\"Sec28\"> <u> H </u> ollow <u> f </u> iber <u> b </u> ioreactor culture (HFB) </h3> <p> For collection of IL6 for lectin array analysis, a hollow fiber bioreactor was applied to improve productivity. In total, 1 × 10 <sup> 8 </sup> cells were implanted into a hollow fiber cartridge (FiberCell Systems, Inc., C-2008) after 5 days of preculture according to the manufacturer’s protocol. CM in the cartridge was collected every 5 days and centrifuged at 750 × g and 4 °C for 5 min. The supernatant was collected and stored at −80 °C until further use. </p> <h3 class=\"c-article__sub-heading\" id=\"Sec29\"> MS analysis and protein <u> id </u> entification (ID) </h3> <h4 class=\"c-article__sub-heading c-article__sub-heading--small\" id=\"Sec30\"> Protein identification </h4> <p> IL6 immunoprecipitated from HFB-derived lung cancer cell CM was resolved by 12% SDS-PAGE and visualized by Coomassie blue staining (Brilliant Blue G-Colloidal Concentrate, Sigma-Aldrich, 2025-1EA). Stained bands with an MW of ~20 ~ 30 kDa were excised and in-gel digested with trypsin (1 U/µg protein) (Sigma-Aldrich). The digested peptides were subsequently subjected to fractionation with a nano-HPLC system (LC Packings) coupled to an ion trap mass spectrometer (LCQ DECA XP Plus, Thermo Finnigan) equipped with an electrospray ion source. The resulting MS/MS spectra containing monoisotopic peptide masses were assigned and used for a search of the Swiss-Prot database with the Mascot search engine (Matrix Science, RRID: SCR_014322). </p> <h4 class=\"c-article__sub-heading c-article__sub-heading--small\" id=\"Sec31\"> Glycosylation analysis </h4> <p> Preparation of the IL6 protein was performed by the filter-aided sample preparation (FASP) method <sup> <a aria-label=\"Reference 70\" href=\"/articles/s41467-024-51831-7#ref-CR70\" id=\"ref-link-section-d6776021e4142\" title=\"Wisniewski, J. R., Zougman, A., Nagaraj, N. &amp; Mann, M. Universal sample preparation method for proteome analysis. Nat. Methods 6, 359–362 (2009).\"> 70 </a> </sup> . In brief, IL6 (10 μg samples) was reduced with 25 mM dithiothreitol and alkylated with 50 mM iodoacetamide in 0.1 M Tris-HCl (pH 8.5) with 8 M urea. Afterward, the buffer was replaced with 50 mM tetraethylammonium bromide (TEAB) for protein digestion. A total of 2 µg of glutamyl endopeptidase C (Glu-C) (Promega, V5111) or 1 μg of trypsin (Promega, V1651) was utilized for the analysis of IL6 peptides containing Asn73 (N73) or Asn 172 (N172), respectively. Then, N-glycans were removed by treatment with PNGase F. Digested IL6 samples were analyzed by LC–ESI-MS on an Orbitrap Fusion mass spectrometer (Thermo Fisher Scientific) connected to an EASY-nLC 1200 system (Thermo Fisher Scientific) and EASY-Spray™ Source (Thermo Fisher Scientific). The digestion solution was injected (5 μL) at a 1 μL/min flow rate onto an EASY-Column (C18, 0.075 mm × 150 mm, ID 3 μm; Thermo Fisher Scientific). Chromatographic separation was performed using 0.1% formic acid in water as mobile phase A and 0.1% formic acid in 80% acetonitrile as mobile phase B at a 300 nL/min flow rate. Briefly, the gradient employed was 5% buffer B at 2 min increasing to 60% buffer B at 55 min. Full-scan MS conditions: mass range, m/z 375–1800 (AGC target, 5E5) with lock mass; resolution, 60,000 at m/z 200; and maximum injection time, 50 ms. MS/MS scanning was performed in top speed mode with 3 s cycles with CID and HCD, while the dynamic exclusion duration was set to 60 s with a 10 ppm tolerance around the selected precursor and its isotopes. The electrospray voltage was maintained at 1.8 kV, and the capillary temperature was set at 275 °C. For glycopeptide identification, Byonic software 4.2.10 (Protein Metrics; RRID:SCR_016735) was used for the identification of summary formulas of glycans associated with glycopeptides (protein database, target protein; glycan database, 132 entries for N-glycans and 9 entries for O-glycans; precursor mass tolerance, 10 ppm; fragment mass tolerance, 0.5 Da; cleavage sites: K and R by trypsin and E and D by Glu-C; maximum missed cleavages, 5; and cysteine carbamidomethylation and methionine oxidation; FDR 0.01) <sup> <a aria-label=\"Reference 71\" href=\"/articles/s41467-024-51831-7#ref-CR71\" id=\"ref-link-section-d6776021e4146\" title=\"Bern, M., Kil, Y. J. &amp; Becker, C. Byonic: advanced peptide and protein identification software. Curr. Protocol. Bioinform. 13, 13 20 11–13 20 14 (2012).\"> 71 </a> </sup> . The percentage of glycosylation at N73 was determined from the areas in the LC spectra from the peptide sequence TCN <sup> 73 </sup> KSNMCE with/without deamidation at N73, and the percentages of glycosylation at N172 and O170 were calculated from the areas by Xcalibur in the LC spectra from the peptide sequence NLDAITTPDPT <sup> 170 </sup> TN <sup> 172 </sup> ASLLTK with/without deamidation at N172. The identified peptides are shown in Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 22 </a> . </p> <h3 class=\"c-article__sub-heading\" id=\"Sec32\"> RNA extraction, RT-PCR, and qPCR </h3> <p> Total RNA was extracted from the examined cells and tissues with TRIzol™ Reagent (Invitrogen, 15596026) following the manufacturer’s protocol. cDNA samples were synthesized from 1 μg of total RNA using the AMV Reverse Transcription System Kit (Promega, A3500) under the following thermal cycling conditions: 42 °C for 60 min, 99 °C for 5 min, and 4 °C for 30 min. Fifty nanograms of cDNA was subjected to RT-PCR, followed by agarose gel electrophoresis with SYBR Green. The expression of target genes was visualized with a UVP BioSpectrum® Imaging System™. Quantification of the transcript expression of human glycosyltransferases by qPCR was performed with 50 ng of cDNA in Power SYBR® Green Master Mix (Applied Bioscience, 4367659) using a Step One Real-Time PCR Detection System (Applied Bioscience). The relative expression of target genes was calculated by percentage changes of 2 <sup> −ΔΔCT </sup> values. The primer sequences used for qPCR analysis of glycosyltransferase expression were kindly provided by Prof. Chuan-Fa Chang. The primer sequences used for qPCR and RT-PCR are listed in Supplementary Table <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 2 </a> . </p> <h3 class=\"c-article__sub-heading\" id=\"Sec33\"> Cytokine array </h3> <p> Cytokine array assays were performed using the C-Series Human Cytokine Antibody Array (Ray-Bio, AAH-CYT-5-8) with intact or diluted CM following the manufacturer’s protocol. </p> <h3 class=\"c-article__sub-heading\" id=\"Sec34\"> Characterization of N-linked glycosylation by lectin microarray </h3> <p> Immunoprecipitated IL6 from CM of AS2-IL6-WT (with or without pretreatment with NGI-1), AS2-IL6-N73Q, HCC827-IL6, and HCC827-OR cells was separated by 12% SDS-PAGE to remove excess IgG and stained with InstantBlue® Coomassie Protein Stain (Abcam, ab119211). IL6 bands were excised and electroeluted with a D-tube of 3 MWCO. The eluted samples were then desalted by D-tube (Merck Millipore, 71506) dialysis against 1x PBS for 5 days (fresh 1x PBS was replaced every 12 h) and concentrated in 3 kDa Amicon Ultra-0.5 concentration tubes (Millipore, UFC500396). Lectin microarray assays were performed as described in GlycoStation Standard Experimental Protocol Ver.1.0. Briefly, the concentrations of the different IL6 glycoforms were determined by BCA Protein Assay reagent and adjusted to 50 µg/mL in 1x PBS. One microgram of each glycoform was labeled with Cy3 Mono-Reactive Dye (GE Healthcare, PA23001) and applied to a lectin microarray chip (LecChip Ver. 1.0 and LecChip Probing Solution, GlycoTechnica) for hybridization overnight at 4 °C. The resulting lectin chip was scanned with an evanescent-field fluorescence scanner (GlycoStation™ Reader 1200; GlycoTechnica) system ( <a href=\"https://www.glycotechnica.com/en/tools/\"> https://www.glycotechnica.com/en/tools/ </a> ). The net intensity of lectin signals was averaged from triplicate spots and automatically calculated with GlycoStation Tools Pro Suite 1.5 software. </p> <h3 class=\"c-article__sub-heading\" id=\"Sec35\"> Migration assays </h3> <h4 class=\"c-article__sub-heading c-article__sub-heading--small\" id=\"Sec36\"> Transwell </h4> <h3 class=\"c-article__sub-heading\" id=\"FPar1\"> CL1-0 cell coculture </h3> <p> AS2-Vec, AS2-IL6-WT, and AS2-IL6-N73Q cells (2.5 × 10 <sup> 5 </sup> ) were plated in a 24-well plate. After 24 h, the medium was replaced with fresh culture medium. In total, 1 × 10 <sup> 4 </sup> CL1-0 cells were plated in a Transwell MilliCell (Millipore, MCEP24H48), which was set on top of each AS2 cell derivative and incubated for 24 h. The cells that migrated through the membrane with an 8 μm pore size in the Transwell chamber were stained with Liu’s staining solution (ASK®). Images were acquired by an Olympus BX50 microscope (Olympus) under a 10x objective with VImage 2014 software, and the area of migrated cells was analyzed using ImageJ software (National Institutes of Health, RRID:SCR_003070). </p> <h3 class=\"c-article__sub-heading\" id=\"FPar2\"> Autonomous migration </h3> <p> AS2-Vec, AS2-IL6-WT, AS2-IL6-N73Q, AS2-IL6-N172Q, and AS2-IL6-N73Q + N172Q cells (1 × 10 <sup> 4 </sup> ) were plated in a Transwell MilliCell (Millipore), which was set on top of each AS2 cell derivative and incubated for 24 h. The cells that migrated through the membrane in the chamber were stained with Liu’s staining solution (ASK®). Images were acquired by an Olympus BX50 microscope (Olympus) under a 10x objective with VImage 2014 software, and the area of migrated cells was analyzed using ImageJ software (National Institutes of Health, RRID:SCR_003070). </p> <h4 class=\"c-article__sub-heading c-article__sub-heading--small\" id=\"Sec37\"> Culture inserts </h4> <p> Appropriate cells were seeded (1 × 10 <sup> 4 </sup> ) into both sides of a culture insert (ibidi, 81176) with mitomycin C (5 μg/mL, Sigma-Aldrich, M4287) and then allowed to attach overnight. The inserts were carefully removed, and the medium was replaced with fresh culture medium or, for inhibitor-treated cells, with medium containing the IL6-neutralizing antibody (5 μg/mL, R&amp;D, MAB206), STAT3 inhibitor S3I-201 (10 μM, Sigma-Aldrich, 573102), JAK inhibitor ruxolitinib (2.5 μM, Selleck Chemicals, S1378), and SRC inhibitors PP1 (10 μM, Sigma-Aldrich, P0040), PP2 (10 μM, Selleck Chemicals, S7008), and tirbanibulin (10 μM, MedChemExpress, HY-10340). Images were acquired by a Nikon ECLIPSE Ti microscope (Nikon) under a 20x objective at 0 h and 24 h after insert removal, and the area of migrated cells was analyzed using ImageJ software (National Institutes of Health, RRID: SCR_003070). </p> <h3 class=\"c-article__sub-heading\" id=\"Sec38\"> Animal experiments </h3> <p> All animal experiments were conducted under the Institutional Animal Care and Use Committee of NCKU (NCKU IACUC)-approved animal protocol No. 103091, No.104104, and No.107174, following the guidance of the of the Laboratory Animal Center of NCKU and the Association for Assessment and Accreditation of Laboratory Animal Care (AAALAC). For analysis of the influence of IL6 glycoforms on lung metastasis, 1 × 10 <sup> 6 </sup> AS2-Vec, AS2-IL6-WT, AS2-IL6-N73Q, AS2-IL6-N73A, and AS2-IL6-N73V cells were transplanted into female nude mice (randomized into three groups for comparison of AS2-Vec, AS2-IL6-WT, and AS2-IL6-N73Q cells, and two groups for comparison of AS2-IL6-N73A and AS2-IL6-N73V cells) by i.v. injection. Mice were observed every other day. For IVIS observation, 1 × 10 <sup> 6 </sup> AS2-IL6-WT-pLuc and AS2-IL6-N73Q-pLuc cells were transplanted into male nude mice (randomized into two groups) by i.v. injection and observed with IVIS Spectrum Noninvasive Quantitative Molecular Imaging System (Xenogen) twice a week for 4 weeks. Prior to IVIS measurement each time, the mice were administered 200 μL of D-luciferin (dissolved in normal saline at 3.7 mg/mL, Biosynth, L-8220) through intraperitoneal (i.p.) injection. Due to individual variation, if the maximum metastasis reached 60% of total lung area or the animal was dead before the set end point, the experiment would be humanely terminated by euthanasia with CO <sub> 2 </sub> upon that time. Lungs of these nude mice were harvested, fixed with formalin and embedded in paraffin (generating formalin-fixed, paraffin-embedded (FFPE) samples). We used both female and male nude mice for in vivo study due to the shortage of mice under the SARS-COVID-19 pandemic. </p> <h3 class=\"c-article__sub-heading\" id=\"Sec39\"> Histology </h3> <p> For histological analysis, 4 μm FFPE sections from the longitudinal central plane of the lungs harvested in the i.v. injection experiments, or paired TKI-naïve and TKI-resistant NSCLC lung tissues were stained with H&amp;E following a standard protocol and scanned with the TissueFAXS System (TissueGnostics GmbH). Both tumor area and total lung area were quantified using HistoQuest software (TissueGnostics GmbH, RRID:SCR_014823) and visually confirmed by a pathologist in a blinded manner. The metastatic capacity of lung tumor cells secreting various IL6 glycoforms was evaluated by calculating the ratio of the tumor area to the total lung area. </p> <h3 class=\"c-article__sub-heading\" id=\"Sec40\"> Microarray analysis </h3> <p> AS2-Vec cells were treated with WT-IL6-containing and N73Q-IL6-containing CM for 0.5, 3, 6, or 24 h. Total RNA was extracted and subjected to microarray analysis. The integrity of all the RNA samples was evaluated using a Bioanalyzer 2100 (Agilent Technologies) according to the manufacturer’s protocol. Analysis of gene expression in each sample was performed utilizing an Agilent SurePrint G3 Human GE 8 × 60 K Microarray (Agilent Technologies), and scanning was performed using an Agilent microarray scanner (Agilent Technologies). The acquired images were analyzed using Feature Extraction 10.5.1.1 software (Agilent Technologies). The experiments were performed in triplicate. The WT-IL6-CM-induced and N73Q-IL6-CM-induced gene expression at 0.5, 3, 6, and 24 h was normalized to that at 0 h. Differentially expressed genes (fold change &gt; 1.5) were characterized and clustered using GeneSpring software (RRID:SCR_009196, Agilent Technologies, assisted by Welgene, Inc., Taiwan). The data were subsequently analyzed with Ingenuity Pathway Analysis (IPA) software (Qiagen, RRID:SCR_008653) and GSEA <sup> <a aria-label=\"Reference 43\" href=\"/articles/s41467-024-51831-7#ref-CR43\" id=\"ref-link-section-d6776021e4271\" title=\"Subramanian, A. et al. Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. Proc. Natl Acad. Sci. USA 102, 15545–15550 (2005).\"> 43 </a> </sup> . </p> <h3 class=\"c-article__sub-heading\" id=\"Sec41\"> In silico transcriptome analysis </h3> <p> The datasets of EGFR-TKI-treated human lung cancer cells GSE31210, GSE60189, GSE59239, GSE106765, GSE122005, GSE38121, and GSE71587 were obtained from the NCBI GEO database, and patient RNA-seq results were obtained from the supplemental information from Roper et al. <sup> <a aria-label=\"Reference 59\" href=\"/articles/s41467-024-51831-7#ref-CR59\" id=\"ref-link-section-d6776021e4283\" title=\"Roper, N. et al. Clonal evolution and heterogeneity of osimertinib acquired resistance mechanisms in EGFR mutant lung cancer. Cell Rep. Med. 1, 10007 (2020).\"> 59 </a> </sup> . Data were converted to “.gct” files prior to GSEA. The.gct files contained the <u> r </u> obust <u> m </u> ulti <u> a </u> rray (RMA) normalized data matrix obtained from GEO with a probe set identifier. Enrichment analyses were performed by GSEA using the standard default settings with well-documented gene sets encompassing signatures of the N-glycosylation process, EMT, and SRC-YAP signaling obtained from the Molecular Signatures Database (MSigDB) ( <a href=\"https://www.gsea-msigdb.org/gsea/msigdb\"> https://www.gsea-msigdb.org/gsea/msigdb </a> ). Heatmaps showing ssGSEA scores and ΔssGSEA scores were generated with Morpheus open-source software (RRID:SCR_014975, <a href=\"https://software.broadinstitute.org/morpheus/\"> https://software.broadinstitute.org/morpheus/ </a> ). </p> <h3 class=\"c-article__sub-heading\" id=\"Sec42\"> Immunofluorescence staining and confocal microscopy </h3> <p> Cells in a 4-well chamber slide (Millipore, PEZGS0416) were fixed with 4% paraformaldehyde (PFA) for 10 min and then washed three times with 1x PBS for 10 min each time. The cells were then permeabilized with 0.1% Triton X-100 (Sigma-Aldrich) for 10 min and washed three times with 1x PBS for 10 min each time. The samples were then blocked with antibody diluent (Dako, S3022) for 1 h at room temperature. For analysis of the expression of TTF1 (Santa Cruz, sc-8762; RRID:AB_793530) and PanCK (Santa Cruz, sc-8018; RRID:AB_627396) or the localization of STAT3 (CST, 9193; RRID:AB_331757), SRC (CST, 2108; RRID:AB_331137), and GP130 (Santa Cruz, sc-376280; RRID:AB_10986409), the samples were reacted with the appropriate primary Ab diluted in antibody diluent overnight at 4 °C and then washed three times. The samples were then reacted with appropriate secondary Ab diluted in antibody diluent at room temperature for 1 h and washed three times. Nuclear staining was performed with Hoechst 33342 dye at 20 μg/mL (Sigma-Aldrich, 14533) for 1 min, and the samples were then washed three times with 1x PBS. The samples were subsequently mounted with Fluoromount-G (Electron Microscopy Sciences, 50-259-73) and sealed with cover glasses. Immunofluorescence images were acquired with an FV3000 confocal microscope (Olympus) and analyzed using FV10-ASW (Olympus) or TissueQuest (TissueGnostics GmbH) software. For analysis of the formation of F-ACTIN, cells and blocked samples were reacted with phalloidin-TRITC (Life Technology, R415) for 1 h at room temperature; the subsequent experimental steps were carried out as described above. For analysis of the expression and localization of STAT3 and SRC in paired TKI-naïve and TKI-resistant NSCLC tissues, 4 μm FFPE sections were applied, and the paired tissues were attached to the same slide to perform the IF process at the same time. Immunofluorescence images were acquired with a TissueFAXS System (TissueGnostics GmbH) and analyzed using TissueQuest (TissueGnostics GmbH) software. </p> <h3 class=\"c-article__sub-heading\" id=\"Sec43\"> Study approval </h3> <p> Research associated with the collection of human specimens was conducted in accordance with the Declaration of Helsinki (2008) of the World Medical Association. This study was approved by the institutional review boards (IRBs) of National Cheng Kung University Hospital (NCKU Hospital, Tainan, Taiwan). Written informed consent was obtained from each study subject (B-ER-1010170, BR-100-034, A-BR-102-098, A-ER-105-451). </p> <h3 class=\"c-article__sub-heading\" id=\"Sec44\"> Statistics and reproducibility </h3> <p> Statistical analyses were performed using Prism software (GraphPad, Inc., RRID:SCR_005375). The results of WB, qPCR, confocal microscopy, and migration and soft agar assays were statistically analyzed by the Student’s <i> t- </i> test. In vivo lung metastasis assay results were analyzed by two-way ANOVA. For Ingenuity Pathway Analysis (IPA) - identified “Disease and Functional Annotation”, a significant correlation was defined as <i> p </i> &lt; 0.001, evaluated by a right-tailed Fisher’s Exact Test. The synergistic effect of combination treatment with the YAPi and osimertinib was evaluated by Synergy Finder Plus ( <a href=\"http://www.synergyfinderplus.org/\"> http://www.synergyfinderplus.org/ </a> ) under the two-tailed Zero interaction potency (ZIP) model <sup> <a aria-label=\"Reference 58\" href=\"/articles/s41467-024-51831-7#ref-CR58\" id=\"ref-link-section-d6776021e4348\" title=\"Zheng, S. et al. SynergyFinder plus: toward better interpretation and annotation of drug combination screening datasets. Genom. Proteom. Bioinform. 20, 587–596 (2022).\"> 58 </a> </sup> . A <i> p </i> -value of &lt;0.05 was considered significant. The data are presented as the mean ± SEM for quantitative results. No statistical method was used to predetermine sample size. </p> <h3 class=\"c-article__sub-heading\" id=\"Sec45\"> Reporting summary </h3> <p> Further information on research design is available in the <a href=\"/articles/s41467-024-51831-7#MOESM3\"> Nature Portfolio Reporting Summary </a> linked to this article. </p> </div> </div> </section> </div> <div class=\"u-mt-32\"> <section> <div class=\"c-article-section\" id=\"data-availability-section\"> <h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"data-availability\"> Data availability </h2> <div class=\"c-article-section__content\" id=\"data-availability-content\"> <p> The RNA microarray datasets generated in this study were deposited into the National Center for Biotechnology Information (NCBI) Gene Expression Omnibus (GEO) database under accession number GSE269782 and are available at the following URL: <a href=\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE269782\"> https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE269782 </a> . The RNAseq and microarray datasets of human lung adenocarcinoma demonstrated in Figs. <a href=\"/articles/s41467-024-51831-7#Fig1\"> 1e </a> , <a href=\"/articles/s41467-024-51831-7#Fig6\"> 6h </a> , and Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 21 </a> were downloaded from NCBI’s GEO: <a href=\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE31210\"> GSE31210 </a> ; <a href=\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE269782\"> GSE106765 </a> ; <a href=\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60189\"> GSE60189 </a> ; <a href=\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59293\"> GSE59239 </a> ; <a href=\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=gse122005\"> GSE122005 </a> ; <a href=\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38121\"> GSE38121 </a> ; <a href=\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE71587\"> GSE71587 </a> ; <a href=\"https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE37699\"> GSE37699 </a> . Datasets of human lung adenocarcinoma demonstrated in Fig. <a href=\"/articles/s41467-024-51831-7#Fig8\"> 8d </a> and Supplementary Fig. <a href=\"/articles/s41467-024-51831-7#MOESM1\"> 26 </a> were from the supplementary information attached to the article of Roper N. et al. <sup> <a aria-label=\"Reference 59\" href=\"/articles/s41467-024-51831-7#ref-CR59\" id=\"ref-link-section-d6776021e4513\" title=\"Roper, N. et al. Clonal evolution and heterogeneity of osimertinib acquired resistance mechanisms in EGFR mutant lung cancer. Cell Rep. Med. 1, 10007 (2020).\"> 59 </a> </sup> . For glycosylation analysis, protein mass spectrometry raw data and analytical tool data have updated in a repository via ProteomeXchange ( <a href=\"https://www.proteomexchange.org\"> https://www.proteomexchange.org </a> ) with project accession: PXD051771. <a href=\"/articles/s41467-024-51831-7#Sec47\"> Source data </a> are provided with this paper. </p> </div> </div> </section> <div id=\"MagazineFulltextArticleBodySuffix\"> <section aria-labelledby=\"Bib1\"> <div class=\"c-article-section\" id=\"Bib1-section\"> <h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"Bib1\"> References </h2> <div class=\"c-article-section__content\" id=\"Bib1-content\"> <div> <ol class=\"c-article-references\"> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR1\"> Yu, H., Lee, H., Herrmann, A., Buettner, R. &amp; Jove, R. Revisiting STAT3 signalling in cancer: new and unexpected biological functions. <i> Nat. Rev. Cancer </i> <b> 14 </b> , 736–746 (2014). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 1\" href=\"https://doi.org/10.1038%2Fnrc3818\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 1\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC2cXhvVSmtLnF\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 1\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25342631\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 1\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Revisiting%20STAT3%20signalling%20in%20cancer%3A%20new%20and%20unexpected%20biological%20functions&amp;journal=Nat.%20Rev.%20Cancer&amp;doi=10.1038%2Fnrc3818&amp;volume=14&amp;pages=736-746&amp;publication_year=2014&amp;author=Yu%2CH&amp;author=Lee%2CH&amp;author=Herrmann%2CA&amp;author=Buettner%2CR&amp;author=Jove%2CR\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR2\"> Jones, S. A. &amp; Jenkins, B. J. Recent insights into targeting the IL-6 cytokine family in inflammatory diseases and cancer. <i> Nat. Rev. Immunol. </i> <b> 18 </b> , 773–789 (2018). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 2\" href=\"https://doi.org/10.1038%2Fs41577-018-0066-7\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 2\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC1cXhslymur7N\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 2\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30254251\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 2\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Recent%20insights%20into%20targeting%20the%20IL-6%20cytokine%20family%20in%20inflammatory%20diseases%20and%20cancer&amp;journal=Nat.%20Rev.%20Immunol.&amp;doi=10.1038%2Fs41577-018-0066-7&amp;volume=18&amp;pages=773-789&amp;publication_year=2018&amp;author=Jones%2CSA&amp;author=Jenkins%2CBJ\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR3\"> Rose-John, S. The soluble interleukin 6 receptor: advanced therapeutic options in inflammation. <i> Clin. Pharmacol. Therapeut. </i> <b> 102 </b> , 591–598 (2017). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 3\" href=\"https://doi.org/10.1002%2Fcpt.782\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 3\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC2sXhsFWjt77I\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"Google Scholar reference 3\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=The%20soluble%20interleukin%206%20receptor%3A%20advanced%20therapeutic%20options%20in%20inflammation&amp;journal=Clin.%20Pharmacol.%20Therapeut.&amp;doi=10.1002%2Fcpt.782&amp;volume=102&amp;pages=591-598&amp;publication_year=2017&amp;author=Rose-John%2CS\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR4\"> Trikha, M., Corringham, R., Klein, B. &amp; Rossi, J. F. Targeted anti-interleukin-6 monoclonal antibody therapy for cancer: a review of the rationale and clinical evidence. <i> Clin. Cancer Res. </i> <b> 9 </b> , 4653–4665 (2003). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"CAS reference 4\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BD3sXosVGjsrw%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 4\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=14581334\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"PubMed Central reference 4\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2929399\" rel=\"nofollow noopener\"> PubMed Central </a> <a aria-label=\"Google Scholar reference 4\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Targeted%20anti-interleukin-6%20monoclonal%20antibody%20therapy%20for%20cancer%3A%20a%20review%20of%20the%20rationale%20and%20clinical%20evidence&amp;journal=Clin.%20Cancer%20Res.&amp;volume=9&amp;pages=4653-4665&amp;publication_year=2003&amp;author=Trikha%2CM&amp;author=Corringham%2CR&amp;author=Klein%2CB&amp;author=Rossi%2CJF\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR5\"> Hunter, C. A. &amp; Jones, S. A. IL-6 as a keystone cytokine in health and disease. <i> Nat. Immunol </i> . <b> 16 </b> , 448–457 (2015). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 5\" href=\"https://doi.org/10.1038%2Fni.3153\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 5\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC2MXhtF2murbF\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 5\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25898198\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 5\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=IL-6%20as%20a%20keystone%20cytokine%20in%20health%20and%20disease&amp;journal=Nat.%20Immunol&amp;doi=10.1038%2Fni.3153&amp;volume=16&amp;pages=448-457&amp;publication_year=2015&amp;author=Hunter%2CCA&amp;author=Jones%2CSA\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR6\"> Garbers, C., Heink, S., Korn, T. &amp; Rose-John, S. Interleukin-6: designing specific therapeutics for a complex cytokine. <i> Nat. Rev. Drug Discov. </i> <b> 17 </b> , 395–412 (2018). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 6\" href=\"https://doi.org/10.1038%2Fnrd.2018.45\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 6\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC1cXovVagtbw%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 6\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29725131\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 6\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Interleukin-6%3A%20designing%20specific%20therapeutics%20for%20a%20complex%20cytokine&amp;journal=Nat.%20Rev.%20Drug%20Discov.&amp;doi=10.1038%2Fnrd.2018.45&amp;volume=17&amp;pages=395-412&amp;publication_year=2018&amp;author=Garbers%2CC&amp;author=Heink%2CS&amp;author=Korn%2CT&amp;author=Rose-John%2CS\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR7\"> Murakami, M., Kamimura, D. &amp; Hirano, T. Pleiotropy and specificity: insights from the interleukin 6 family of cytokines. <i> Immunity </i> <b> 50 </b> , 812–831 (2019). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 7\" href=\"https://doi.org/10.1016%2Fj.immuni.2019.03.027\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 7\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC1MXnsFGksLY%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 7\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30995501\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 7\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Pleiotropy%20and%20specificity%3A%20insights%20from%20the%20interleukin%206%20family%20of%20cytokines&amp;journal=Immunity&amp;doi=10.1016%2Fj.immuni.2019.03.027&amp;volume=50&amp;pages=812-831&amp;publication_year=2019&amp;author=Murakami%2CM&amp;author=Kamimura%2CD&amp;author=Hirano%2CT\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR8\"> Duan, Z. et al. Signal transducers and activators of transcription 3 pathway activation in drug-resistant ovarian cancer. <i> Clin. Cancer Res. </i> <b> 12 </b> , 5055–5063 (2006). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 8\" href=\"https://doi.org/10.1158%2F1078-0432.CCR-06-0861\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 8\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BD28XovF2hu7s%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 8\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16951221\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 8\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Signal%20transducers%20and%20activators%20of%20transcription%203%20pathway%20activation%20in%20drug-resistant%20ovarian%20cancer&amp;journal=Clin.%20Cancer%20Res.&amp;doi=10.1158%2F1078-0432.CCR-06-0861&amp;volume=12&amp;pages=5055-5063&amp;publication_year=2006&amp;author=Duan%2CZ\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR9\"> Huang, W. L. et al. Signal transducer and activator of transcription 3 activation up-regulates interleukin-6 autocrine production: a biochemical and genetic study of established cancer cell lines and clinical isolated human cancer cells. <i> Mol. Cancer </i> <b> 9 </b> , 309 (2010). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 9\" href=\"https://link.springer.com/doi/10.1186/1476-4598-9-309\" rel=\"noopener\"> Article </a> <a aria-label=\"CAS reference 9\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC3cXhsF2hs7rK\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 9\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21122157\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"PubMed Central reference 9\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3027602\" rel=\"nofollow noopener\"> PubMed Central </a> <a aria-label=\"Google Scholar reference 9\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Signal%20transducer%20and%20activator%20of%20transcription%203%20activation%20up-regulates%20interleukin-6%20autocrine%20production%3A%20a%20biochemical%20and%20genetic%20study%20of%20established%20cancer%20cell%20lines%20and%20clinical%20isolated%20human%20cancer%20cells&amp;journal=Mol.%20Cancer&amp;doi=10.1186%2F1476-4598-9-309&amp;volume=9&amp;publication_year=2010&amp;author=Huang%2CWL\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR10\"> Lee, H. J. et al. Drug resistance via feedback activation of Stat3 in oncogene-addicted cancer cells. <i> Cancer Cell </i> <b> 26 </b> , 207–221 (2014). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 10\" href=\"https://doi.org/10.1016%2Fj.ccr.2014.05.019\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 10\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC2cXht1agsL7I\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 10\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25065853\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 10\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Drug%20resistance%20via%20feedback%20activation%20of%20Stat3%20in%20oncogene-addicted%20cancer%20cells&amp;journal=Cancer%20Cell&amp;doi=10.1016%2Fj.ccr.2014.05.019&amp;volume=26&amp;pages=207-221&amp;publication_year=2014&amp;author=Lee%2CHJ\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR11\"> Rokavec, M. et al. IL-6R/STAT3/miR-34a feedback loop promotes EMT-mediated colorectal cancer invasion and metastasis. <i> J. Clin. Invest. </i> <b> 124 </b> , 1853–1867 (2014). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 11\" href=\"https://doi.org/10.1172%2FJCI73531\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 11\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC2cXmtVGgur4%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 11\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24642471\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"PubMed Central reference 11\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3973098\" rel=\"nofollow noopener\"> PubMed Central </a> <a aria-label=\"Google Scholar reference 11\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=IL-6R%2FSTAT3%2FmiR-34a%20feedback%20loop%20promotes%20EMT-mediated%20colorectal%20cancer%20invasion%20and%20metastasis&amp;journal=J.%20Clin.%20Invest.&amp;doi=10.1172%2FJCI73531&amp;volume=124&amp;pages=1853-1867&amp;publication_year=2014&amp;author=Rokavec%2CM\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR12\"> Taniguchi, K. et al. A gp130-Src-YAP module links inflammation to epithelial regeneration. <i> Nature </i> <b> 519 </b> , 57–62 (2015). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 12\" href=\"https://doi.org/10.1038%2Fnature14228\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"ADS reference 12\" href=\"http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2015Natur.519...57T\" rel=\"nofollow noopener\"> ADS </a> <a aria-label=\"CAS reference 12\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC2MXjvVGjsrk%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 12\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25731159\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"PubMed Central reference 12\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4447318\" rel=\"nofollow noopener\"> PubMed Central </a> <a aria-label=\"Google Scholar reference 12\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=A%20gp130-Src-YAP%20module%20links%20inflammation%20to%20epithelial%20regeneration&amp;journal=Nature&amp;doi=10.1038%2Fnature14228&amp;volume=519&amp;pages=57-62&amp;publication_year=2015&amp;author=Taniguchi%2CK\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR13\"> Quintanal-Villalonga, A. et al. Lineage plasticity in cancer: a shared pathway of therapeutic resistance. <i> Nat. Rev. Clin. Oncol. </i> <b> 17 </b> , 360–371 (2020). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 13\" href=\"https://doi.org/10.1038%2Fs41571-020-0340-z\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"PubMed reference 13\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32152485\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"PubMed Central reference 13\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7397755\" rel=\"nofollow noopener\"> PubMed Central </a> <a aria-label=\"Google Scholar reference 13\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Lineage%20plasticity%20in%20cancer%3A%20a%20shared%20pathway%20of%20therapeutic%20resistance&amp;journal=Nat.%20Rev.%20Clin.%20Oncol.&amp;doi=10.1038%2Fs41571-020-0340-z&amp;volume=17&amp;pages=360-371&amp;publication_year=2020&amp;author=Quintanal-Villalonga%2CA\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR14\"> Paez, J. G. et al. EGFR mutations in lung cancer: correlation with clinical response to gefitinib therapy. <i> Science </i> <b> 304 </b> , 1497–1500 (2004). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 14\" href=\"https://doi.org/10.1126%2Fscience.1099314\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"ADS reference 14\" href=\"http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2004Sci...304.1497P\" rel=\"nofollow noopener\"> ADS </a> <a aria-label=\"CAS reference 14\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BD2cXksVGmsbs%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 14\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15118125\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 14\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=EGFR%20mutations%20in%20lung%20cancer%3A%20correlation%20with%20clinical%20response%20to%20gefitinib%20therapy&amp;journal=Science&amp;doi=10.1126%2Fscience.1099314&amp;volume=304&amp;pages=1497-1500&amp;publication_year=2004&amp;author=Paez%2CJG\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR15\"> Ramalingam, S. S. et al. Overall survival with osimertinib in untreated, EGFR-mutated advanced NSCLC. <i> N. Engl. J. Med. </i> <b> 382 </b> , 41–50 (2020). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 15\" href=\"https://doi.org/10.1056%2FNEJMoa1913662\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 15\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BB3cXnsFGgtL4%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 15\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31751012\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 15\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Overall%20survival%20with%20osimertinib%20in%20untreated%2C%20EGFR-mutated%20advanced%20NSCLC&amp;journal=N.%20Engl.%20J.%20Med.&amp;doi=10.1056%2FNEJMoa1913662&amp;volume=382&amp;pages=41-50&amp;publication_year=2020&amp;author=Ramalingam%2CSS\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR16\"> Oxnard, G. R. et al. Assessment of resistance mechanisms and clinical implications in patients with EGFR T790M-positive lung cancer and acquired resistance to osimertinib. <i> JAMA Oncol. </i> <b> 4 </b> , 1527–1534 (2018). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 16\" href=\"https://doi.org/10.1001%2Fjamaoncol.2018.2969\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"PubMed reference 16\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30073261\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 16\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Assessment%20of%20resistance%20mechanisms%20and%20clinical%20implications%20in%20patients%20with%20EGFR%20T790M-positive%20lung%20cancer%20and%20acquired%20resistance%20to%20osimertinib&amp;journal=JAMA%20Oncol.&amp;doi=10.1001%2Fjamaoncol.2018.2969&amp;volume=4&amp;pages=1527-1534&amp;publication_year=2018&amp;author=Oxnard%2CGR\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR17\"> Yang, Z. et al. Investigating novel resistance mechanisms to third-generation EGFR tyrosine kinase inhibitor osimertinib in non-small cell lung cancer patients. <i> Clin. Cancer Res. </i> <b> 24 </b> , 3097–3107 (2018). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 17\" href=\"https://doi.org/10.1158%2F1078-0432.CCR-17-2310\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 17\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC1cXht1alurvK\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 17\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29506987\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 17\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Investigating%20novel%20resistance%20mechanisms%20to%20third-generation%20EGFR%20tyrosine%20kinase%20inhibitor%20osimertinib%20in%20non-small%20cell%20lung%20cancer%20patients&amp;journal=Clin.%20Cancer%20Res.&amp;doi=10.1158%2F1078-0432.CCR-17-2310&amp;volume=24&amp;pages=3097-3107&amp;publication_year=2018&amp;author=Yang%2CZ\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR18\"> Sequist, L. V. et al. Genotypic and histological evolution of lung cancers acquiring resistance to EGFR inhibitors. <i> Sci. Transl. Med. </i> <b> 3 </b> , 75ra26 (2011). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 18\" href=\"https://doi.org/10.1126%2Fscitranslmed.3002003\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"PubMed reference 18\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21430269\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"PubMed Central reference 18\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3132801\" rel=\"nofollow noopener\"> PubMed Central </a> <a aria-label=\"Google Scholar reference 18\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Genotypic%20and%20histological%20evolution%20of%20lung%20cancers%20acquiring%20resistance%20to%20EGFR%20inhibitors&amp;journal=Sci.%20Transl.%20Med.&amp;doi=10.1126%2Fscitranslmed.3002003&amp;volume=3&amp;publication_year=2011&amp;author=Sequist%2CLV\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR19\"> Lambert, A. W. &amp; Weinberg, R. A. Linking EMT programmes to normal and neoplastic epithelial stem cells. <i> Nat. Rev. Cancer </i> <b> 21 </b> , 325–338 (2021). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 19\" href=\"https://doi.org/10.1038%2Fs41568-021-00332-6\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 19\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BB3MXjsVCquro%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 19\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33547455\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 19\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Linking%20EMT%20programmes%20to%20normal%20and%20neoplastic%20epithelial%20stem%20cells&amp;journal=Nat.%20Rev.%20Cancer&amp;doi=10.1038%2Fs41568-021-00332-6&amp;volume=21&amp;pages=325-338&amp;publication_year=2021&amp;author=Lambert%2CAW&amp;author=Weinberg%2CRA\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR20\"> Williams, E. D., Gao, D., Redfern, A. &amp; Thompson, E. W. Controversies around epithelial-mesenchymal plasticity in cancer metastasis. <i> Nat. Rev. Cancer </i> <b> 19 </b> , 716–732 (2019). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 20\" href=\"https://doi.org/10.1038%2Fs41568-019-0213-x\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 20\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC1MXitV2hsb7J\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 20\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31666716\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"PubMed Central reference 20\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7055151\" rel=\"nofollow noopener\"> PubMed Central </a> <a aria-label=\"Google Scholar reference 20\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Controversies%20around%20epithelial-mesenchymal%20plasticity%20in%20cancer%20metastasis&amp;journal=Nat.%20Rev.%20Cancer&amp;doi=10.1038%2Fs41568-019-0213-x&amp;volume=19&amp;pages=716-732&amp;publication_year=2019&amp;author=Williams%2CED&amp;author=Gao%2CD&amp;author=Redfern%2CA&amp;author=Thompson%2CEW\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR21\"> Yao, Z. et al. TGF-beta IL-6 axis mediates selective and adaptive mechanisms of resistance to molecular targeted therapy in lung cancer. <i> Proc. Natl Acad. Sci. USA </i> <b> 107 </b> , 15535–15540 (2010). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 21\" href=\"https://doi.org/10.1073%2Fpnas.1009472107\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"ADS reference 21\" href=\"http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2010PNAS..10715535Y\" rel=\"nofollow noopener\"> ADS </a> <a aria-label=\"CAS reference 21\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtFaqu7%2FN\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 21\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20713723\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"PubMed Central reference 21\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2932568\" rel=\"nofollow noopener\"> PubMed Central </a> <a aria-label=\"Google Scholar reference 21\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=TGF-beta%20IL-6%20axis%20mediates%20selective%20and%20adaptive%20mechanisms%20of%20resistance%20to%20molecular%20targeted%20therapy%20in%20lung%20cancer&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.1009472107&amp;volume=107&amp;pages=15535-15540&amp;publication_year=2010&amp;author=Yao%2CZ\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR22\"> Kim, S. M. et al. Activation of IL-6R/JAK1/STAT signaling induces de novo resistance to irreversible EGFR inhibitors in non-small cell lung cancer with T790M resistance mutation. <i> Mol. Cancer Ther. </i> <b> 11 </b> , 2254–2264 (2012). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 22\" href=\"https://doi.org/10.1158%2F1535-7163.MCT-12-0311\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 22\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC38XhsVyksrvI\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 22\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22891040\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 22\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Activation%20of%20IL-6R%2FJAK1%2FSTAT%20signaling%20induces%20de%20novo%20resistance%20to%20irreversible%20EGFR%20inhibitors%20in%20non-small%20cell%20lung%20cancer%20with%20T790M%20resistance%20mutation&amp;journal=Mol.%20Cancer%20Ther.&amp;doi=10.1158%2F1535-7163.MCT-12-0311&amp;volume=11&amp;pages=2254-2264&amp;publication_year=2012&amp;author=Kim%2CSM\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR23\"> Korkaya, H., Liu, S. &amp; Wicha, M. S. Regulation of cancer stem cells by cytokine networks: attacking cancer’s inflammatory roots. <i> Clin. Cancer Res. </i> <b> 17 </b> , 6125–6129 (2011). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 23\" href=\"https://doi.org/10.1158%2F1078-0432.CCR-10-2743\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 23\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC3MXht1Ggs7jP\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 23\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21685479\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"PubMed Central reference 23\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3312242\" rel=\"nofollow noopener\"> PubMed Central </a> <a aria-label=\"Google Scholar reference 23\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Regulation%20of%20cancer%20stem%20cells%20by%20cytokine%20networks%3A%20attacking%20cancer%E2%80%99s%20inflammatory%20roots&amp;journal=Clin.%20Cancer%20Res.&amp;doi=10.1158%2F1078-0432.CCR-10-2743&amp;volume=17&amp;pages=6125-6129&amp;publication_year=2011&amp;author=Korkaya%2CH&amp;author=Liu%2CS&amp;author=Wicha%2CMS\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR24\"> Senescence-Associated Pro-inflammatory Cytokines, VernotJ. P. and tumor cell plasticity. <i> Front. Mol. Biosci. </i> <b> 7 </b> , 63 (2020). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"ADS reference 24\" href=\"http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2020FrMat...7...63S\" rel=\"nofollow noopener\"> ADS </a> <a aria-label=\"Google Scholar reference 24\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=and%20tumor%20cell%20plasticity&amp;journal=Front.%20Mol.%20Biosci.&amp;volume=7&amp;publication_year=2020&amp;author=Senescence-Associated%20Pro-inflammatory%20Cytokines%2CVernotJP\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR25\"> Kurppa, K. J. et al. Treatment-induced tumor dormancy through YAP-mediated transcriptional reprogramming of the apoptotic pathway. <i> Cancer Cell </i> <b> 37 </b> , 104–122.e112 (2020). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 25\" href=\"https://doi.org/10.1016%2Fj.ccell.2019.12.006\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 25\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BB3cXotVent78%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 25\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31935369\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"PubMed Central reference 25\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7146079\" rel=\"nofollow noopener\"> PubMed Central </a> <a aria-label=\"Google Scholar reference 25\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Treatment-induced%20tumor%20dormancy%20through%20YAP-mediated%20transcriptional%20reprogramming%20of%20the%20apoptotic%20pathway&amp;journal=Cancer%20Cell&amp;doi=10.1016%2Fj.ccell.2019.12.006&amp;volume=37&amp;pages=104-122.e112&amp;publication_year=2020&amp;author=Kurppa%2CKJ\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR26\"> Ferone, G. et al. SOX2 is the determining oncogenic switch in promoting lung squamous cell carcinoma from different cells of origin. <i> Cancer Cell </i> <b> 30 </b> , 519–532 (2016). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 26\" href=\"https://doi.org/10.1016%2Fj.ccell.2016.09.001\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 26\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC28Xhs1GqtLbP\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 26\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27728803\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"PubMed Central reference 26\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5065004\" rel=\"nofollow noopener\"> PubMed Central </a> <a aria-label=\"Google Scholar reference 26\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=SOX2%20is%20the%20determining%20oncogenic%20switch%20in%20promoting%20lung%20squamous%20cell%20carcinoma%20from%20different%20cells%20of%20origin&amp;journal=Cancer%20Cell&amp;doi=10.1016%2Fj.ccell.2016.09.001&amp;volume=30&amp;pages=519-532&amp;publication_year=2016&amp;author=Ferone%2CG\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR27\"> Boumahdi, S. &amp; de Sauvage, F. J. The great escape: tumour cell plasticity in resistance to targeted therapy. <i> Nat. Rev. Drug Discov. </i> <b> 19 </b> , 39–56 (2020). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 27\" href=\"https://doi.org/10.1038%2Fs41573-019-0044-1\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 27\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC1MXhvFCktL%2FM\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 27\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31601994\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 27\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=The%20great%20escape%3A%20tumour%20cell%20plasticity%20in%20resistance%20to%20targeted%20therapy&amp;journal=Nat.%20Rev.%20Drug%20Discov.&amp;doi=10.1038%2Fs41573-019-0044-1&amp;volume=19&amp;pages=39-56&amp;publication_year=2020&amp;author=Boumahdi%2CS&amp;author=Sauvage%2CFJ\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR28\"> Mosteiro, L. et al. Tissue damage and senescence provide critical signals for cellular reprogramming in vivo. <i> Science </i> <b> 354 </b> , 6315 (2016). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 28\" href=\"https://doi.org/10.1126%2Fscience.aaf4445\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"Google Scholar reference 28\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Tissue%20damage%20and%20senescence%20provide%20critical%20signals%20for%20cellular%20reprogramming%20in%20vivo&amp;journal=Science&amp;doi=10.1126%2Fscience.aaf4445&amp;volume=354&amp;publication_year=2016&amp;author=Mosteiro%2CL\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR29\"> Bora-Singhal, N. et al. YAP1 regulates OCT4 activity and SOX2 expression to facilitate self-renewal and vascular mimicry of stem-like cells. <i> Stem cells </i> <b> 33 </b> , 1705–1718 (2015). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 29\" href=\"https://doi.org/10.1002%2Fstem.1993\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 29\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC2MXhtleqsLvK\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 29\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25754111\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 29\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=YAP1%20regulates%20OCT4%20activity%20and%20SOX2%20expression%20to%20facilitate%20self-renewal%20and%20vascular%20mimicry%20of%20stem-like%20cells&amp;journal=Stem%20cells&amp;doi=10.1002%2Fstem.1993&amp;volume=33&amp;pages=1705-1718&amp;publication_year=2015&amp;author=Bora-Singhal%2CN\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR30\"> Poupart, P. et al. B cell growth modulating and differentiating activity of recombinant human 26-kd protein (BSF-2, HuIFN-beta 2, HPGF). <i> EMBO J </i> <b> 6 </b> , 1219–1224 (1987). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 30\" href=\"https://doi.org/10.1002%2Fj.1460-2075.1987.tb02357.x\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 30\" href=\"/articles/cas-redirect/1:CAS:528:DyaL2sXkt1Wktr0%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 30\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=3497029\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"PubMed Central reference 30\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC553922\" rel=\"nofollow noopener\"> PubMed Central </a> <a aria-label=\"Google Scholar reference 30\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=B%20cell%20growth%20modulating%20and%20differentiating%20activity%20of%20recombinant%20human%2026-kd%20protein%20%28BSF-2%2C%20HuIFN-beta%202%2C%20HPGF%29&amp;journal=EMBO%20J&amp;doi=10.1002%2Fj.1460-2075.1987.tb02357.x&amp;volume=6&amp;pages=1219-1224&amp;publication_year=1987&amp;author=Poupart%2CP\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR31\"> May, L. T. et al. Phosphorylation of secreted forms of human beta 2-interferon/hepatocyte stimulating factor/interleukin-6. <i> Biochem. Biophys. Res. Commun. </i> <b> 152 </b> , 1144–1150 (1988). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 31\" href=\"https://doi.org/10.1016%2FS0006-291X%2888%2980404-2\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 31\" href=\"/articles/cas-redirect/1:CAS:528:DyaL1cXktVGjt70%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 31\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=3259872\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 31\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Phosphorylation%20of%20secreted%20forms%20of%20human%20beta%202-interferon%2Fhepatocyte%20stimulating%20factor%2Finterleukin-6&amp;journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;doi=10.1016%2FS0006-291X%2888%2980404-2&amp;volume=152&amp;pages=1144-1150&amp;publication_year=1988&amp;author=May%2CLT\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR32\"> Hirano, T. et al. Purification to homogeneity and characterization of human B-cell differentiation factor (BCDF or BSFp-2). <i> Proc. Natl Acad. Sci. USA </i> <b> 82 </b> , 5490–5494 (1985). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 32\" href=\"https://doi.org/10.1073%2Fpnas.82.16.5490\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"ADS reference 32\" href=\"http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=1985PNAS...82.5490H\" rel=\"nofollow noopener\"> ADS </a> <a aria-label=\"CAS reference 32\" href=\"/articles/cas-redirect/1:CAS:528:DyaL2MXlt1Wnt7s%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 32\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=2410927\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"PubMed Central reference 32\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC391148\" rel=\"nofollow noopener\"> PubMed Central </a> <a aria-label=\"Google Scholar reference 32\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Purification%20to%20homogeneity%20and%20characterization%20of%20human%20B-cell%20differentiation%20factor%20%28BCDF%20or%20BSFp-2%29&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.82.16.5490&amp;volume=82&amp;pages=5490-5494&amp;publication_year=1985&amp;author=Hirano%2CT\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR33\"> May, L. T., Santhanam, U. &amp; Sehgal, P. B. On the multimeric nature of natural human interleukin-6. <i> J. Biol. Chem. </i> <b> 266 </b> , 9950–9955 (1991). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 33\" href=\"https://doi.org/10.1016%2FS0021-9258%2818%2992911-X\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 33\" href=\"/articles/cas-redirect/1:CAS:528:DyaK3MXkt1Onsbw%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 33\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=2033081\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 33\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=On%20the%20multimeric%20nature%20of%20natural%20human%20interleukin-6&amp;journal=J.%20Biol.%20Chem.&amp;doi=10.1016%2FS0021-9258%2818%2992911-X&amp;volume=266&amp;pages=9950-9955&amp;publication_year=1991&amp;author=May%2CLT&amp;author=Santhanam%2CU&amp;author=Sehgal%2CPB\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR34\"> Mereiter, S., Balmana, M., Campos, D., Gomes, J. &amp; Reis, C. A. Glycosylation in the era of cancer-targeted therapy: where are we heading? <i> Cancer Cell </i> <b> 36 </b> , 6–16 (2019). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 34\" href=\"https://doi.org/10.1016%2Fj.ccell.2019.06.006\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 34\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtlersb%2FJ\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 34\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31287993\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 34\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Glycosylation%20in%20the%20era%20of%20cancer-targeted%20therapy%3A%20where%20are%20we%20heading%3F&amp;journal=Cancer%20Cell&amp;doi=10.1016%2Fj.ccell.2019.06.006&amp;volume=36&amp;pages=6-16&amp;publication_year=2019&amp;author=Mereiter%2CS&amp;author=Balmana%2CM&amp;author=Campos%2CD&amp;author=Gomes%2CJ&amp;author=Reis%2CCA\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR35\"> Parekh, R. B., Dwek, R. A., Rademacher, T. W., Opdenakker, G. &amp; Van Damme, J. Glycosylation of interleukin-6 purified from normal human blood mononuclear cells. <i> Eur. J. Biochem. </i> <b> 203 </b> , 135–141 (1992). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 35\" href=\"https://doi.org/10.1111%2Fj.1432-1033.1992.tb19838.x\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 35\" href=\"/articles/cas-redirect/1:CAS:528:DyaK38Xht1KmtL8%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 35\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=1730219\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 35\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Glycosylation%20of%20interleukin-6%20purified%20from%20normal%20human%20blood%20mononuclear%20cells&amp;journal=Eur.%20J.%20Biochem.&amp;doi=10.1111%2Fj.1432-1033.1992.tb19838.x&amp;volume=203&amp;pages=135-141&amp;publication_year=1992&amp;author=Parekh%2CRB&amp;author=Dwek%2CRA&amp;author=Rademacher%2CTW&amp;author=Opdenakker%2CG&amp;author=Damme%2CJ\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR36\"> Dela Cruz, C. S. et al. N-linked glycosylation is required for optimal function of Kaposi’s sarcoma herpesvirus-encoded, but not cellular, interleukin 6. <i> J. Exp. Med. </i> <b> 199 </b> , 503–514 (2004). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 36\" href=\"https://doi.org/10.1084%2Fjem.20031205\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"Google Scholar reference 36\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=N-linked%20glycosylation%20is%20required%20for%20optimal%20function%20of%20Kaposi%E2%80%99s%20sarcoma%20herpesvirus-encoded%2C%20but%20not%20cellular%2C%20interleukin%206&amp;journal=J.%20Exp.%20Med.&amp;doi=10.1084%2Fjem.20031205&amp;volume=199&amp;pages=503-514&amp;publication_year=2004&amp;author=Cruz%2CCS\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR37\"> Thomas, D., Rathinavel, A. K. &amp; Radhakrishnan, P. Altered glycosylation in cancer: a promising target for biomarkers and therapeutics. <i> Biochimica. et biophysica. Acta Rev. Cancer </i> <b> 1875 </b> , 188464 (2021). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 37\" href=\"https://doi.org/10.1016%2Fj.bbcan.2020.188464\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 37\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BB3cXitl2msr%2FP\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"Google Scholar reference 37\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Altered%20glycosylation%20in%20cancer%3A%20a%20promising%20target%20for%20biomarkers%20and%20therapeutics&amp;journal=Biochimica.%20et%20biophysica.%20Acta%20Rev.%20Cancer&amp;doi=10.1016%2Fj.bbcan.2020.188464&amp;volume=1875&amp;publication_year=2021&amp;author=Thomas%2CD&amp;author=Rathinavel%2CAK&amp;author=Radhakrishnan%2CP\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR38\"> Pinho, S. S. &amp; Reis, C. A. Glycosylation in cancer: mechanisms and clinical implications. <i> Nat. Rev. Cancer </i> <b> 15 </b> , 540–555 (2015). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 38\" href=\"https://doi.org/10.1038%2Fnrc3982\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 38\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC2MXhtlKkt7fK\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 38\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26289314\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 38\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Glycosylation%20in%20cancer%3A%20mechanisms%20and%20clinical%20implications&amp;journal=Nat.%20Rev.%20Cancer&amp;doi=10.1038%2Fnrc3982&amp;volume=15&amp;pages=540-555&amp;publication_year=2015&amp;author=Pinho%2CSS&amp;author=Reis%2CCA\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR39\"> Yeh, H. H., Lai, W. W., Chen, H. H., Liu, H. S. &amp; Su, W. C. Autocrine IL-6-induced Stat3 activation contributes to the pathogenesis of lung adenocarcinoma and malignant pleural effusion. <i> Oncogene </i> <b> 25 </b> , 4300–4309 (2006). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 39\" href=\"https://doi.org/10.1038%2Fsj.onc.1209464\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 39\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BD28XntVWks7o%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 39\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16518408\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 39\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Autocrine%20IL-6-induced%20Stat3%20activation%20contributes%20to%20the%20pathogenesis%20of%20lung%20adenocarcinoma%20and%20malignant%20pleural%20effusion&amp;journal=Oncogene&amp;doi=10.1038%2Fsj.onc.1209464&amp;volume=25&amp;pages=4300-4309&amp;publication_year=2006&amp;author=Yeh%2CHH&amp;author=Lai%2CWW&amp;author=Chen%2CHH&amp;author=Liu%2CHS&amp;author=Su%2CWC\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR40\"> Roth, J. Protein N-glycosylation along the secretory pathway: relationship to organelle topography and function, protein quality control, and cell interactions. <i> Chem. Rev. </i> <b> 102 </b> , 285–303 (2002). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 40\" href=\"https://doi.org/10.1021%2Fcr000423j\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 40\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BD38XntlGrug%3D%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 40\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11841244\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 40\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Protein%20N-glycosylation%20along%20the%20secretory%20pathway%3A%20relationship%20to%20organelle%20topography%20and%20function%2C%20protein%20quality%20control%2C%20and%20cell%20interactions&amp;journal=Chem.%20Rev.&amp;doi=10.1021%2Fcr000423j&amp;volume=102&amp;pages=285-303&amp;publication_year=2002&amp;author=Roth%2CJ\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR41\"> Bard, F. &amp; Chia, J. Cracking the glycome encoder: signaling, trafficking, and glycosylation. <i> Trends Cell Biol. </i> <b> 26 </b> , 379–388 (2016). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 41\" href=\"https://doi.org/10.1016%2Fj.tcb.2015.12.004\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 41\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC28XjslKrtA%3D%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 41\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26832820\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 41\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Cracking%20the%20glycome%20encoder%3A%20signaling%2C%20trafficking%2C%20and%20glycosylation&amp;journal=Trends%20Cell%20Biol.&amp;doi=10.1016%2Fj.tcb.2015.12.004&amp;volume=26&amp;pages=379-388&amp;publication_year=2016&amp;author=Bard%2CF&amp;author=Chia%2CJ\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR42\"> Schjoldager, K. T., Narimatsu, Y., Joshi, H. J. &amp; Clausen, H. Global view of human protein glycosylation pathways and functions. <i> Nat. Rev. Mol. Cell Biol. </i> <b> 21 </b> , 729–749 (2020). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 42\" href=\"https://doi.org/10.1038%2Fs41580-020-00294-x\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 42\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BB3cXitFGmu73N\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 42\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33087899\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 42\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Global%20view%20of%20human%20protein%20glycosylation%20pathways%20and%20functions&amp;journal=Nat.%20Rev.%20Mol.%20Cell%20Biol.&amp;doi=10.1038%2Fs41580-020-00294-x&amp;volume=21&amp;pages=729-749&amp;publication_year=2020&amp;author=Schjoldager%2CKT&amp;author=Narimatsu%2CY&amp;author=Joshi%2CHJ&amp;author=Clausen%2CH\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR43\"> Subramanian, A. et al. Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. <i> Proc. Natl Acad. Sci. USA </i> <b> 102 </b> , 15545–15550 (2005). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 43\" href=\"https://doi.org/10.1073%2Fpnas.0506580102\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"ADS reference 43\" href=\"http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2005PNAS..10215545S\" rel=\"nofollow noopener\"> ADS </a> <a aria-label=\"CAS reference 43\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BD2MXht1ShtrnO\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 43\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16199517\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"PubMed Central reference 43\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1239896\" rel=\"nofollow noopener\"> PubMed Central </a> <a aria-label=\"Google Scholar reference 43\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Gene%20set%20enrichment%20analysis%3A%20a%20knowledge-based%20approach%20for%20interpreting%20genome-wide%20expression%20profiles&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.0506580102&amp;volume=102&amp;pages=15545-15550&amp;publication_year=2005&amp;author=Subramanian%2CA\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR44\"> Mootha, V. K. et al. PGC-1alpha-responsive genes involved in oxidative phosphorylation are coordinately downregulated in human diabetes. <i> Nat. Genet. </i> <b> 34 </b> , 267–273 (2003). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 44\" href=\"https://doi.org/10.1038%2Fng1180\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 44\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BD3sXkvFSrs7o%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 44\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12808457\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 44\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=PGC-1alpha-responsive%20genes%20involved%20in%20oxidative%20phosphorylation%20are%20coordinately%20downregulated%20in%20human%20diabetes&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fng1180&amp;volume=34&amp;pages=267-273&amp;publication_year=2003&amp;author=Mootha%2CVK\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR45\"> Okayama, H. et al. Identification of genes upregulated in ALK-positive and EGFR/KRAS/ALK-negative lung adenocarcinomas. <i> Cancer Res. </i> <b> 72 </b> , 100–111 (2012). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 45\" href=\"https://doi.org/10.1158%2F0008-5472.CAN-11-1403\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 45\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC38XhvVCmtg%3D%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 45\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22080568\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 45\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Identification%20of%20genes%20upregulated%20in%20ALK-positive%20and%20EGFR%2FKRAS%2FALK-negative%20lung%20adenocarcinomas&amp;journal=Cancer%20Res.&amp;doi=10.1158%2F0008-5472.CAN-11-1403&amp;volume=72&amp;pages=100-111&amp;publication_year=2012&amp;author=Okayama%2CH\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR46\"> Lopez-Sambrooks, C. et al. Oligosaccharyltransferase inhibition induces senescence in RTK-driven tumor cells. <i> Nat. Chem. Biol. </i> <b> 12 </b> , 1023–1030 (2016). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 46\" href=\"https://doi.org/10.1038%2Fnchembio.2194\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 46\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC28XhsFOmsLvE\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 46\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27694802\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"PubMed Central reference 46\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5393272\" rel=\"nofollow noopener\"> PubMed Central </a> <a aria-label=\"Google Scholar reference 46\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Oligosaccharyltransferase%20inhibition%20induces%20senescence%20in%20RTK-driven%20tumor%20cells&amp;journal=Nat.%20Chem.%20Biol.&amp;doi=10.1038%2Fnchembio.2194&amp;volume=12&amp;pages=1023-1030&amp;publication_year=2016&amp;author=Lopez-Sambrooks%2CC\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR47\"> Blom, N., Sicheritz-Ponten, T., Gupta, R., Gammeltoft, S. &amp; Brunak, S. Prediction of post-translational glycosylation and phosphorylation of proteins from the amino acid sequence. <i> Proteomics </i> <b> 4 </b> , 1633–1649 (2004). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 47\" href=\"https://doi.org/10.1002%2Fpmic.200300771\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 47\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BD2cXkvFGitLg%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 47\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15174133\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 47\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Prediction%20of%20post-translational%20glycosylation%20and%20phosphorylation%20of%20proteins%20from%20the%20amino%20acid%20sequence&amp;journal=Proteomics&amp;doi=10.1002%2Fpmic.200300771&amp;volume=4&amp;pages=1633-1649&amp;publication_year=2004&amp;author=Blom%2CN&amp;author=Sicheritz-Ponten%2CT&amp;author=Gupta%2CR&amp;author=Gammeltoft%2CS&amp;author=Brunak%2CS\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR48\"> Yamaguchi, K. et al. Effects of site-directed removal of N-glycosylation sites in human erythropoietin on its production and biological properties. <i> J. Biol. Chem </i> . <b> 266 </b> , 20434–20439 (1991). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 48\" href=\"https://doi.org/10.1016%2FS0021-9258%2818%2954942-5\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 48\" href=\"/articles/cas-redirect/1:CAS:528:DyaK3MXlvFykt70%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 48\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=1657925\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 48\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Effects%20of%20site-directed%20removal%20of%20N-glycosylation%20sites%20in%20human%20erythropoietin%20on%20its%20production%20and%20biological%20properties&amp;journal=J.%20Biol.%20Chem&amp;doi=10.1016%2FS0021-9258%2818%2954942-5&amp;volume=266&amp;pages=20434-20439&amp;publication_year=1991&amp;author=Yamaguchi%2CK\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR49\"> Bobby, R. et al. Functional implications of large backbone amplitude motions of the glycoprotein 130-binding epitope of interleukin-6. <i> FEBS J. </i> <b> 281 </b> , 2471–2483 (2014). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 49\" href=\"https://doi.org/10.1111%2Ffebs.12800\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 49\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC2cXotFemtL0%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 49\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24712547\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 49\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Functional%20implications%20of%20large%20backbone%20amplitude%20motions%20of%20the%20glycoprotein%20130-binding%20epitope%20of%20interleukin-6&amp;journal=FEBS%20J.&amp;doi=10.1111%2Ffebs.12800&amp;volume=281&amp;pages=2471-2483&amp;publication_year=2014&amp;author=Bobby%2CR\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR50\"> Xiong, H. et al. Roles of STAT3 and ZEB1 proteins in E-cadherin down-regulation and human colorectal cancer epithelial-mesenchymal transition. <i> J. Biol. Chem. </i> <b> 287 </b> , 5819–5832 (2012). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 50\" href=\"https://doi.org/10.1074%2Fjbc.M111.295964\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 50\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC38XisVyluro%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 50\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22205702\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 50\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Roles%20of%20STAT3%20and%20ZEB1%20proteins%20in%20E-cadherin%20down-regulation%20and%20human%20colorectal%20cancer%20epithelial-mesenchymal%20transition&amp;journal=J.%20Biol.%20Chem.&amp;doi=10.1074%2Fjbc.M111.295964&amp;volume=287&amp;pages=5819-5832&amp;publication_year=2012&amp;author=Xiong%2CH\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR51\"> Wei, W. et al. STAT3 signaling is activated preferentially in tumor-initiating cells in claudin-low models of human breast cancer. <i> Stem cells </i> <b> 32 </b> , 2571–2582 (2014). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 51\" href=\"https://doi.org/10.1002%2Fstem.1752\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 51\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC2cXhvFGjtLzE\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 51\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24891218\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 51\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=STAT3%20signaling%20is%20activated%20preferentially%20in%20tumor-initiating%20cells%20in%20claudin-low%20models%20of%20human%20breast%20cancer&amp;journal=Stem%20cells&amp;doi=10.1002%2Fstem.1752&amp;volume=32&amp;pages=2571-2582&amp;publication_year=2014&amp;author=Wei%2CW\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR52\"> Yao, C. et al. IGF/STAT3/NANOG/Slug signaling axis simultaneously controls epithelial-mesenchymal transition and stemness maintenance in colorectal cancer. <i> Stem Cells </i> <b> 34 </b> , 820–831 (2016). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 52\" href=\"https://doi.org/10.1002%2Fstem.2320\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"PubMed reference 52\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26840943\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 52\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=IGF%2FSTAT3%2FNANOG%2FSlug%20signaling%20axis%20simultaneously%20controls%20epithelial-mesenchymal%20transition%20and%20stemness%20maintenance%20in%20colorectal%20cancer&amp;journal=Stem%20Cells&amp;doi=10.1002%2Fstem.2320&amp;volume=34&amp;pages=820-831&amp;publication_year=2016&amp;author=Yao%2CC\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR53\"> Na, Y. R., Lee, J. S., Lee, S. J. &amp; Seok, S. H. Interleukin-6-induced twist and N-cadherin enhance melanoma cell metastasis. <i> Melanoma Res. </i> <b> 23 </b> , 434–443 (2013). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 53\" href=\"https://doi.org/10.1097%2FCMR.0000000000000021\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 53\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC3sXhs12rtbbK\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 53\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24051540\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 53\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Interleukin-6-induced%20twist%20and%20N-cadherin%20enhance%20melanoma%20cell%20metastasis&amp;journal=Melanoma%20Res.&amp;doi=10.1097%2FCMR.0000000000000021&amp;volume=23&amp;pages=434-443&amp;publication_year=2013&amp;author=Na%2CYR&amp;author=Lee%2CJS&amp;author=Lee%2CSJ&amp;author=Seok%2CSH\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR54\"> Gallucci, R. M., Lee, E. G. &amp; Tomasek, J. J. IL-6 modulates alpha-smooth muscle actin expression in dermal fibroblasts from IL-6-deficient mice. <i> J. Invest. Dermatol </i> . <b> 126 </b> , 561–568 (2006). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 54\" href=\"https://doi.org/10.1038%2Fsj.jid.5700109\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 54\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BD28XhsVSrsLc%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 54\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16397521\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 54\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=IL-6%20modulates%20alpha-smooth%20muscle%20actin%20expression%20in%20dermal%20fibroblasts%20from%20IL-6-deficient%20mice&amp;journal=J.%20Invest.%20Dermatol&amp;doi=10.1038%2Fsj.jid.5700109&amp;volume=126&amp;pages=561-568&amp;publication_year=2006&amp;author=Gallucci%2CRM&amp;author=Lee%2CEG&amp;author=Tomasek%2CJJ\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR55\"> Romera-Hernandez, M. et al. Yap1-driven intestinal repair is controlled by group 3 innate lymphoid cells. <i> Cell Rep. </i> <b> 30 </b> , 37–45.e33 (2020). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 55\" href=\"https://doi.org/10.1016%2Fj.celrep.2019.11.115\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 55\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BB3cXotFert70%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 55\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31914395\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 55\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Yap1-driven%20intestinal%20repair%20is%20controlled%20by%20group%203%20innate%20lymphoid%20cells&amp;journal=Cell%20Rep.&amp;doi=10.1016%2Fj.celrep.2019.11.115&amp;volume=30&amp;pages=37-45.e33&amp;publication_year=2020&amp;author=Romera-Hernandez%2CM\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR56\"> Tsuji, T. et al. YAP1 mediates survival of ALK-rearranged lung cancer cells treated with alectinib via pro-apoptotic protein regulation. <i> Nat. Commun. </i> <b> 11 </b> , 74 (2020). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 56\" href=\"https://doi.org/10.1038%2Fs41467-019-13771-5\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"ADS reference 56\" href=\"http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2020NatCo..11...74T\" rel=\"nofollow noopener\"> ADS </a> <a aria-label=\"CAS reference 56\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BB3cXlslWisg%3D%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 56\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31900393\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"PubMed Central reference 56\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6941996\" rel=\"nofollow noopener\"> PubMed Central </a> <a aria-label=\"Google Scholar reference 56\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=YAP1%20mediates%20survival%20of%20ALK-rearranged%20lung%20cancer%20cells%20treated%20with%20alectinib%20via%20pro-apoptotic%20protein%20regulation&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-019-13771-5&amp;volume=11&amp;publication_year=2020&amp;author=Tsuji%2CT\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR57\"> Takahashi, A. et al. Ankyrin repeat domain 1 overexpression is associated with common resistance to afatinib and osimertinib in EGFR-mutant lung cancer. <i> Sci. Rep. </i> <b> 8 </b> , 14896 (2018). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 57\" href=\"https://doi.org/10.1038%2Fs41598-018-33190-8\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"ADS reference 57\" href=\"http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2018NatSR...814896T\" rel=\"nofollow noopener\"> ADS </a> <a aria-label=\"PubMed reference 57\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30291293\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"PubMed Central reference 57\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6173712\" rel=\"nofollow noopener\"> PubMed Central </a> <a aria-label=\"Google Scholar reference 57\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Ankyrin%20repeat%20domain%201%20overexpression%20is%20associated%20with%20common%20resistance%20to%20afatinib%20and%20osimertinib%20in%20EGFR-mutant%20lung%20cancer&amp;journal=Sci.%20Rep.&amp;doi=10.1038%2Fs41598-018-33190-8&amp;volume=8&amp;publication_year=2018&amp;author=Takahashi%2CA\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR58\"> Zheng, S. et al. SynergyFinder plus: toward better interpretation and annotation of drug combination screening datasets. <i> Genom. Proteom. Bioinform. </i> <b> 20 </b> , 587–596 (2022). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 58\" href=\"https://doi.org/10.1016%2Fj.gpb.2022.01.004\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"Google Scholar reference 58\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=SynergyFinder%20plus%3A%20toward%20better%20interpretation%20and%20annotation%20of%20drug%20combination%20screening%20datasets&amp;journal=Genom.%20Proteom.%20Bioinform.&amp;doi=10.1016%2Fj.gpb.2022.01.004&amp;volume=20&amp;pages=587-596&amp;publication_year=2022&amp;author=Zheng%2CS\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR59\"> Roper, N. et al. Clonal evolution and heterogeneity of osimertinib acquired resistance mechanisms in EGFR mutant lung cancer. <i> Cell Rep. Med. </i> <b> 1 </b> , 10007 (2020). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Google Scholar reference 59\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Clonal%20evolution%20and%20heterogeneity%20of%20osimertinib%20acquired%20resistance%20mechanisms%20in%20EGFR%20mutant%20lung%20cancer&amp;journal=Cell%20Rep.%20Med.&amp;volume=1&amp;publication_year=2020&amp;author=Roper%2CN\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR60\"> Barbie, D. A. et al. Systematic RNA interference reveals that oncogenic KRAS-driven cancers require TBK1. <i> Nature </i> <b> 462 </b> , 108–112 (2009). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 60\" href=\"https://doi.org/10.1038%2Fnature08460\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"ADS reference 60\" href=\"http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2009Natur.462..108B\" rel=\"nofollow noopener\"> ADS </a> <a aria-label=\"CAS reference 60\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BD1MXht12gtrvL\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 60\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19847166\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"PubMed Central reference 60\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2783335\" rel=\"nofollow noopener\"> PubMed Central </a> <a aria-label=\"Google Scholar reference 60\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Systematic%20RNA%20interference%20reveals%20that%20oncogenic%20KRAS-driven%20cancers%20require%20TBK1&amp;journal=Nature&amp;doi=10.1038%2Fnature08460&amp;volume=462&amp;pages=108-112&amp;publication_year=2009&amp;author=Barbie%2CDA\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR61\"> Heinz, M. C. et al. Liver colonization by colorectal cancer metastases requires YAP-controlled plasticity at the micrometastatic stage. <i> Cancer Res. </i> <b> 82 </b> , 1953–1968 (2022). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 61\" href=\"https://doi.org/10.1158%2F0008-5472.CAN-21-0933\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 61\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BB38XhsFehsbrE\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 61\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35570706\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"PubMed Central reference 61\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9381095\" rel=\"nofollow noopener\"> PubMed Central </a> <a aria-label=\"Google Scholar reference 61\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Liver%20colonization%20by%20colorectal%20cancer%20metastases%20requires%20YAP-controlled%20plasticity%20at%20the%20micrometastatic%20stage&amp;journal=Cancer%20Res.&amp;doi=10.1158%2F0008-5472.CAN-21-0933&amp;volume=82&amp;pages=1953-1968&amp;publication_year=2022&amp;author=Heinz%2CMC\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR62\"> Hamanaka, N. et al. YES1 is a targetable oncogene in cancers harboring YES1 gene amplification. <i> Cancer Res. </i> <b> 79 </b> , 5734–5745 (2019). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 62\" href=\"https://doi.org/10.1158%2F0008-5472.CAN-18-3376\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 62\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BB3cXjslGjs70%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 62\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31391186\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 62\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=YES1%20is%20a%20targetable%20oncogene%20in%20cancers%20harboring%20YES1%20gene%20amplification&amp;journal=Cancer%20Res.&amp;doi=10.1158%2F0008-5472.CAN-18-3376&amp;volume=79&amp;pages=5734-5745&amp;publication_year=2019&amp;author=Hamanaka%2CN\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR63\"> Lopez Sambrooks, C. et al. Oligosaccharyltransferase inhibition overcomes therapeutic resistance to EGFR tyrosine kinase inhibitors. <i> Cancer Res. </i> <b> 78 </b> , 5094–5106 (2018). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 63\" href=\"https://doi.org/10.1158%2F0008-5472.CAN-18-0505\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"PubMed reference 63\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30026325\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 63\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Oligosaccharyltransferase%20inhibition%20overcomes%20therapeutic%20resistance%20to%20EGFR%20tyrosine%20kinase%20inhibitors&amp;journal=Cancer%20Res.&amp;doi=10.1158%2F0008-5472.CAN-18-0505&amp;volume=78&amp;pages=5094-5106&amp;publication_year=2018&amp;author=Lopez%20Sambrooks%2CC\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR64\"> Chan, L. C. et al. IL-6/JAK1 pathway drives PD-L1 Y112 phosphorylation to promote cancer immune evasion. <i> J. Clin. Invest. </i> <b> 129 </b> , 3324–3338 (2019). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 64\" href=\"https://doi.org/10.1172%2FJCI126022\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"PubMed reference 64\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31305264\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"PubMed Central reference 64\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6668668\" rel=\"nofollow noopener\"> PubMed Central </a> <a aria-label=\"Google Scholar reference 64\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=IL-6%2FJAK1%20pathway%20drives%20PD-L1%20Y112%20phosphorylation%20to%20promote%20cancer%20immune%20evasion&amp;journal=J.%20Clin.%20Invest.&amp;doi=10.1172%2FJCI126022&amp;volume=129&amp;pages=3324-3338&amp;publication_year=2019&amp;author=Chan%2CLC\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR65\"> Miller, M. R. et al. Downregulation of dystroglycan glycosyltransferases LARGE2 and ISPD associate with increased mortality in clear cell renal cell carcinoma. <i> Mol. Cancer </i> <b> 14 </b> , 141 (2015). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 65\" href=\"https://link.springer.com/doi/10.1186/s12943-015-0416-z\" rel=\"noopener\"> Article </a> <a aria-label=\"PubMed reference 65\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26220087\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"PubMed Central reference 65\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4518861\" rel=\"nofollow noopener\"> PubMed Central </a> <a aria-label=\"Google Scholar reference 65\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Downregulation%20of%20dystroglycan%20glycosyltransferases%20LARGE2%20and%20ISPD%20associate%20with%20increased%20mortality%20in%20clear%20cell%20renal%20cell%20carcinoma&amp;journal=Mol.%20Cancer&amp;doi=10.1186%2Fs12943-015-0416-z&amp;volume=14&amp;publication_year=2015&amp;author=Miller%2CMR\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR66\"> Chugh, S., Meza, J., Sheinin, Y. M., Ponnusamy, M. P. &amp; Batra, S. K. Loss of N-acetylgalactosaminyltransferase 3 in poorly differentiated pancreatic cancer: augmented aggressiveness and aberrant ErbB family glycosylation. <i> Br. J. Cancer </i> <b> 114 </b> , 1376–1386 (2016). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 66\" href=\"https://doi.org/10.1038%2Fbjc.2016.116\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 66\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC28XotFOhurw%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 66\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27187683\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"PubMed Central reference 66\" href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4984453\" rel=\"nofollow noopener\"> PubMed Central </a> <a aria-label=\"Google Scholar reference 66\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Loss%20of%20N-acetylgalactosaminyltransferase%203%20in%20poorly%20differentiated%20pancreatic%20cancer%3A%20augmented%20aggressiveness%20and%20aberrant%20ErbB%20family%20glycosylation&amp;journal=Br.%20J.%20Cancer&amp;doi=10.1038%2Fbjc.2016.116&amp;volume=114&amp;pages=1376-1386&amp;publication_year=2016&amp;author=Chugh%2CS&amp;author=Meza%2CJ&amp;author=Sheinin%2CYM&amp;author=Ponnusamy%2CMP&amp;author=Batra%2CSK\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR67\"> Peng, C. et al. Regulation of the Hippo-YAP pathway by glucose sensor O-GlcNAcylation. <i> Mol. Cell </i> <b> 68 </b> , 591–604.e595 (2017). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 67\" href=\"https://doi.org/10.1016%2Fj.molcel.2017.10.010\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 67\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BC2sXhslOju7rJ\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 67\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29100056\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 67\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Regulation%20of%20the%20Hippo-YAP%20pathway%20by%20glucose%20sensor%20O-GlcNAcylation&amp;journal=Mol.%20Cell&amp;doi=10.1016%2Fj.molcel.2017.10.010&amp;volume=68&amp;pages=591-604.e595&amp;publication_year=2017&amp;author=Peng%2CC\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR68\"> Hung, T. L. et al. Clinical evaluation of HER-2/neu protein in malignant pleural effusion-associated lung adenocarcinoma and as a tumor marker in pleural effusion diagnosis. <i> Clin. Cancer Res. </i> <b> 9 </b> , 2605–2612 (2003). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"CAS reference 68\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BD3sXlt1yrsrg%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 68\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12855637\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 68\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Clinical%20evaluation%20of%20HER-2%2Fneu%20protein%20in%20malignant%20pleural%20effusion-associated%20lung%20adenocarcinoma%20and%20as%20a%20tumor%20marker%20in%20pleural%20effusion%20diagnosis&amp;journal=Clin.%20Cancer%20Res.&amp;volume=9&amp;pages=2605-2612&amp;publication_year=2003&amp;author=Hung%2CTL\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR69\"> Chu, Y. W. et al. Selection of invasive and metastatic subpopulations from a human lung adenocarcinoma cell line. <i> Am. J. Respir. Cell Mol. Biol. </i> <b> 17 </b> , 353–360 (1997). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 69\" href=\"https://doi.org/10.1165%2Fajrcmb.17.3.2837\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 69\" href=\"/articles/cas-redirect/1:CAS:528:DyaK2sXmtVOrs7c%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 69\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9308922\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 69\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Selection%20of%20invasive%20and%20metastatic%20subpopulations%20from%20a%20human%20lung%20adenocarcinoma%20cell%20line&amp;journal=Am.%20J.%20Respir.%20Cell%20Mol.%20Biol.&amp;doi=10.1165%2Fajrcmb.17.3.2837&amp;volume=17&amp;pages=353-360&amp;publication_year=1997&amp;author=Chu%2CYW\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR70\"> Wisniewski, J. R., Zougman, A., Nagaraj, N. &amp; Mann, M. Universal sample preparation method for proteome analysis. <i> Nat. Methods </i> <b> 6 </b> , 359–362 (2009). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Article reference 70\" href=\"https://doi.org/10.1038%2Fnmeth.1322\" rel=\"nofollow noopener\"> Article </a> <a aria-label=\"CAS reference 70\" href=\"/articles/cas-redirect/1:CAS:528:DC%2BD1MXks12ksb0%3D\" rel=\"nofollow noopener\"> CAS </a> <a aria-label=\"PubMed reference 70\" href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19377485\" rel=\"nofollow noopener\"> PubMed </a> <a aria-label=\"Google Scholar reference 70\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Universal%20sample%20preparation%20method%20for%20proteome%20analysis&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fnmeth.1322&amp;volume=6&amp;pages=359-362&amp;publication_year=2009&amp;author=Wisniewski%2CJR&amp;author=Zougman%2CA&amp;author=Nagaraj%2CN&amp;author=Mann%2CM\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> <li class=\"c-article-references__item js-c-reading-companion-references-item\"> <p class=\"c-article-references__text\" id=\"ref-CR71\"> Bern, M., Kil, Y. J. &amp; Becker, C. Byonic: advanced peptide and protein identification software. <i> Curr. Protocol. Bioinform. </i> <b> 13 </b> , 13 20 11–13 20 14 (2012). </p> <p class=\"c-article-references__links u-hide-print\"> <a aria-label=\"Google Scholar reference 71\" href=\"http://scholar.google.com/scholar_lookup?&amp;title=Byonic%3A%20advanced%20peptide%20and%20protein%20identification%20software&amp;journal=Curr.%20Protocol.%20Bioinform.&amp;volume=13&amp;pages=13%2020%2011-13%2020%2014&amp;publication_year=2012&amp;author=Bern%2CM&amp;author=Kil%2CYJ&amp;author=Becker%2CC\" rel=\"nofollow noopener\"> Google Scholar </a> </p> </li> </ol> <p class=\"c-article-references__download u-hide-print\"> <a href=\"https://citation-needed.springer.com/v2/references/10.1038/s41467-024-51831-7?format=refman&amp;flavour=references\" rel=\"nofollow\"> Download references </a> </p> </div> </div> </div> </section> </div> <section> <div class=\"c-article-section\" id=\"Ack1-section\"> <h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"Ack1\"> Acknowledgements </h2> <div class=\"c-article-section__content\" id=\"Ack1-content\"> <p> We acknowledge the Human BioBank (Research Center of Clinical Medicine), National Cheng Kung University Hospital (NCKUH), the Bioinformatics Center (National Cheng Kung University, NCKU), and the National Center for High Performance Computing for bioinformatics consulting, the Proteomics Core Laboratory of NCKU and the Mass Spectrometry Facility of National Yang Ming Chiao Tung University for protein identification, the Bioimaging Core Facility of the National Core Facility for Biopharmaceuticals (Ministry of Science and Technology, Taiwan) for assistance in analysis of confocal microscopic images and FACS-like tissue cytometric images, the Laboratory Animal Center, College of Medicine at National Cheng Kung University (NCKU), Taiwan, accredited by AAALAC International and Taiwan Animal Consortium for the technical support in Xenogen IVISR Spectrum Noninvasive Quantitative Molecular Imaging System, and the Biostatistics Consulting Center (NCKUH) for statistical consulting. We thank Dr. Chun-Hui Lee (Department of Oncology, NCKUH) for assistance in collecting MPE specimens, Dr. Wei-Pang Chung (Department of Oncology, NCKUH) for assistance in analyzing synergy of cytotoxicity, Prof. Chuan-Fa Chang (Department of Medical Laboratory Science and Biotechnology, NCKU) for providing primer sequences against N-OSTs, and Mr. Chein-Hung Chen (Academia Sinica Glycoscience Core Facility and the GRC Mass Core Facility) for glycopeptide analysis. This work was supported by grants from the Ministry of Science and Technology, Taiwan (MOST 104-2314-B-006-046-MY3, MOST 106-2314-B-006-040-MY2, MOST 107-2314-B-006-059-MY3), the Center of Applied Nanomedicine of National Cheng Kung University from Featured Areas Research Center Program within the framework of the Higher Education Sprout Project by the Ministry of Education (MOE), Taiwan, the Yushan Fellow Program, Ministry of Education and NCKU, Taiwan, the Ministry of Health and Welfare (MOHW), Taiwan (MOHW112-TDU-B-221-124005), and Academia Sinica, Taiwan (AS-GC-110-04, AS-GC-110-MD-04). </p> </div> </div> </section> <section aria-labelledby=\"author-information\"> <div class=\"c-article-section\" id=\"author-information-section\"> <h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"author-information\"> Author information </h2> <div class=\"c-article-section__content\" id=\"author-information-content\"> <h3 class=\"c-article__sub-heading\" id=\"affiliations\"> Authors and Affiliations </h3> <ol class=\"c-article-author-affiliation__list\"> <li id=\"Aff1\"> <p class=\"c-article-author-affiliation__address\"> Department of Oncology, National Cheng Kung University Hospital, College of Medicine, National Cheng Kung University, Tainan, Taiwan </p> <p class=\"c-article-author-affiliation__authors-list\"> Chun-Hua Hung, Shang-Yin Wu &amp; Wu-Chou Su </p> </li> <li id=\"Aff2\"> <p class=\"c-article-author-affiliation__address\"> Center of Applied Nanomedicine, National Cheng Kung University, Tainan, Taiwan </p> <p class=\"c-article-author-affiliation__authors-list\"> Chun-Hua Hung, Hsuan-Heng Yeh &amp; Wu-Chou Su </p> </li> <li id=\"Aff3\"> <p class=\"c-article-author-affiliation__address\"> Institute of Basic Medical Sciences, College of Medicine, National Cheng Kung University, Tainan, Taiwan </p> <p class=\"c-article-author-affiliation__authors-list\"> Chun-Hua Hung &amp; Wu-Chou Su </p> </li> <li id=\"Aff4\"> <p class=\"c-article-author-affiliation__address\"> Department of Biomedical Engineering, College of Engineering, National Cheng Kung University, Tainan, Taiwan </p> <p class=\"c-article-author-affiliation__authors-list\"> Chun-Hua Hung </p> </li> <li id=\"Aff5\"> <p class=\"c-article-author-affiliation__address\"> Institute of Clinical Medicine, College of Medicine, National Cheng Kung University, Tainan, Taiwan </p> <p class=\"c-article-author-affiliation__authors-list\"> Shang-Yin Wu, Chien-Chung Lin &amp; Wu-Chou Su </p> </li> <li id=\"Aff6\"> <p class=\"c-article-author-affiliation__address\"> Institute of Biological Chemistry, Academia Sinica, Taipei, Taiwan </p> <p class=\"c-article-author-affiliation__authors-list\"> Cheng-I Daniel Yao &amp; Chun-Hung Lin </p> </li> <li id=\"Aff7\"> <p class=\"c-article-author-affiliation__address\"> Department of Internal Medicine, National Cheng Kung University Hospital, College of Medicine, National Cheng Kung University, Tainan, Taiwan </p> <p class=\"c-article-author-affiliation__authors-list\"> Chien-Chung Lin </p> </li> <li id=\"Aff8\"> <p class=\"c-article-author-affiliation__address\"> Department of Biochemistry and Molecular Biology, College of Medicine, National Cheng Kung University, Tainan, Taiwan </p> <p class=\"c-article-author-affiliation__authors-list\"> Chien-Chung Lin </p> </li> <li id=\"Aff9\"> <p class=\"c-article-author-affiliation__address\"> Department of Pathology, National Cheng Kung University Hospital, College of Medicine, National Cheng Kung University, Tainan, Taiwan </p> <p class=\"c-article-author-affiliation__authors-list\"> Chang-Yao Chu </p> </li> <li id=\"Aff10\"> <p class=\"c-article-author-affiliation__address\"> Institute of Molecular Medicine, College of Medicine, National Cheng Kung University, Tainan, Taiwan </p> <p class=\"c-article-author-affiliation__authors-list\"> Tzu-Yu Huang &amp; Wu-Chou Su </p> </li> <li id=\"Aff11\"> <p class=\"c-article-author-affiliation__address\"> Department of Pharmacology, College of Medicine, National Cheng Kung University, Tainan, Taiwan </p> <p class=\"c-article-author-affiliation__authors-list\"> Meng-Ru Shen </p> </li> <li id=\"Aff12\"> <p class=\"c-article-author-affiliation__address\"> Department of Obstetrics and Gynecology, National Cheng Kung University Hospital, College of Medicine, National Cheng Kung University, Tainan, Taiwan </p> <p class=\"c-article-author-affiliation__authors-list\"> Meng-Ru Shen </p> </li> <li id=\"Aff13\"> <p class=\"c-article-author-affiliation__address\"> Department of Chemistry and Institute of Biochemical Sciences, National Taiwan University, Taipei, Taiwan </p> <p class=\"c-article-author-affiliation__authors-list\"> Chun-Hung Lin </p> </li> </ol> <div class=\"u-js-hide u-hide-print\"> <span class=\"c-article__sub-heading\"> Authors </span> <ol class=\"c-article-authors-search u-list-reset\"> <li id=\"auth-Chun_Hua-Hung-Aff1-Aff2-Aff3-Aff4\"> <span class=\"c-article-authors-search__title u-h3 js-search-name\"> Chun-Hua Hung </span> <div class=\"c-article-authors-search__list\"> <div class=\"c-article-authors-search__item c-article-authors-search__list-item--left\"> <a class=\"c-article-button\" href=\"/search?author=Chun-Hua%20Hung\" rel=\"nofollow\"> View author publications </a> </div> <div class=\"c-article-authors-search__item c-article-authors-search__list-item--right\"> <p class=\"search-in-title-js c-article-authors-search__text\"> <span class=\"c-article-authors-search__links-text\"> Search author on: </span> <span class=\"c-article-identifiers\"> <a class=\"c-article-identifiers__item\" href=\"https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Chun-Hua%20Hung\" rel=\"nofollow\"> PubMed </a> <span class=\"u-hide\"> </span> <a class=\"c-article-identifiers__item\" href=\"https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Chun-Hua%20Hung%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en\" rel=\"nofollow\"> Google Scholar </a> </span> </p> </div> </div> </li> <li id=\"auth-Shang_Yin-Wu-Aff1-Aff5\"> <span class=\"c-article-authors-search__title u-h3 js-search-name\"> Shang-Yin Wu </span> <div class=\"c-article-authors-search__list\"> <div class=\"c-article-authors-search__item c-article-authors-search__list-item--left\"> <a class=\"c-article-button\" href=\"/search?author=Shang-Yin%20Wu\" rel=\"nofollow\"> View author publications </a> </div> <div class=\"c-article-authors-search__item c-article-authors-search__list-item--right\"> <p class=\"search-in-title-js c-article-authors-search__text\"> <span class=\"c-article-authors-search__links-text\"> Search author on: </span> <span class=\"c-article-identifiers\"> <a class=\"c-article-identifiers__item\" href=\"https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Shang-Yin%20Wu\" rel=\"nofollow\"> PubMed </a> <span class=\"u-hide\"> </span> <a class=\"c-article-identifiers__item\" href=\"https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Shang-Yin%20Wu%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en\" rel=\"nofollow\"> Google Scholar </a> </span> </p> </div> </div> </li> <li id=\"auth-Cheng_I_Daniel-Yao-Aff6\"> <span class=\"c-article-authors-search__title u-h3 js-search-name\"> Cheng-I Daniel Yao </span> <div class=\"c-article-authors-search__list\"> <div class=\"c-article-authors-search__item c-article-authors-search__list-item--left\"> <a class=\"c-article-button\" href=\"/search?author=Cheng-I%20Daniel%20Yao\" rel=\"nofollow\"> View author publications </a> </div> <div class=\"c-article-authors-search__item c-article-authors-search__list-item--right\"> <p class=\"search-in-title-js c-article-authors-search__text\"> <span class=\"c-article-authors-search__links-text\"> Search author on: </span> <span class=\"c-article-identifiers\"> <a class=\"c-article-identifiers__item\" href=\"https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Cheng-I%20Daniel%20Yao\" rel=\"nofollow\"> PubMed </a> <span class=\"u-hide\"> </span> <a class=\"c-article-identifiers__item\" href=\"https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Cheng-I%20Daniel%20Yao%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en\" rel=\"nofollow\"> Google Scholar </a> </span> </p> </div> </div> </li> <li id=\"auth-Hsuan_Heng-Yeh-Aff2\"> <span class=\"c-article-authors-search__title u-h3 js-search-name\"> Hsuan-Heng Yeh </span> <div class=\"c-article-authors-search__list\"> <div class=\"c-article-authors-search__item c-article-authors-search__list-item--left\"> <a class=\"c-article-button\" href=\"/search?author=Hsuan-Heng%20Yeh\" rel=\"nofollow\"> View author publications </a> </div> <div class=\"c-article-authors-search__item c-article-authors-search__list-item--right\"> <p class=\"search-in-title-js c-article-authors-search__text\"> <span class=\"c-article-authors-search__links-text\"> Search author on: </span> <span class=\"c-article-identifiers\"> <a class=\"c-article-identifiers__item\" href=\"https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Hsuan-Heng%20Yeh\" rel=\"nofollow\"> PubMed </a> <span class=\"u-hide\"> </span> <a class=\"c-article-identifiers__item\" href=\"https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Hsuan-Heng%20Yeh%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en\" rel=\"nofollow\"> Google Scholar </a> </span> </p> </div> </div> </li> <li id=\"auth-Chien_Chung-Lin-Aff5-Aff7-Aff8\"> <span class=\"c-article-authors-search__title u-h3 js-search-name\"> Chien-Chung Lin </span> <div class=\"c-article-authors-search__list\"> <div class=\"c-article-authors-search__item c-article-authors-search__list-item--left\"> <a class=\"c-article-button\" href=\"/search?author=Chien-Chung%20Lin\" rel=\"nofollow\"> View author publications </a> </div> <div class=\"c-article-authors-search__item c-article-authors-search__list-item--right\"> <p class=\"search-in-title-js c-article-authors-search__text\"> <span class=\"c-article-authors-search__links-text\"> Search author on: </span> <span class=\"c-article-identifiers\"> <a class=\"c-article-identifiers__item\" href=\"https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Chien-Chung%20Lin\" rel=\"nofollow\"> PubMed </a> <span class=\"u-hide\"> </span> <a class=\"c-article-identifiers__item\" href=\"https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Chien-Chung%20Lin%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en\" rel=\"nofollow\"> Google Scholar </a> </span> </p> </div> </div> </li> <li id=\"auth-Chang_Yao-Chu-Aff9\"> <span class=\"c-article-authors-search__title u-h3 js-search-name\"> Chang-Yao Chu </span> <div class=\"c-article-authors-search__list\"> <div class=\"c-article-authors-search__item c-article-authors-search__list-item--left\"> <a class=\"c-article-button\" href=\"/search?author=Chang-Yao%20Chu\" rel=\"nofollow\"> View author publications </a> </div> <div class=\"c-article-authors-search__item c-article-authors-search__list-item--right\"> <p class=\"search-in-title-js c-article-authors-search__text\"> <span class=\"c-article-authors-search__links-text\"> Search author on: </span> <span class=\"c-article-identifiers\"> <a class=\"c-article-identifiers__item\" href=\"https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Chang-Yao%20Chu\" rel=\"nofollow\"> PubMed </a> <span class=\"u-hide\"> </span> <a class=\"c-article-identifiers__item\" href=\"https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Chang-Yao%20Chu%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en\" rel=\"nofollow\"> Google Scholar </a> </span> </p> </div> </div> </li> <li id=\"auth-Tzu_Yu-Huang-Aff10\"> <span class=\"c-article-authors-search__title u-h3 js-search-name\"> Tzu-Yu Huang </span> <div class=\"c-article-authors-search__list\"> <div class=\"c-article-authors-search__item c-article-authors-search__list-item--left\"> <a class=\"c-article-button\" href=\"/search?author=Tzu-Yu%20Huang\" rel=\"nofollow\"> View author publications </a> </div> <div class=\"c-article-authors-search__item c-article-authors-search__list-item--right\"> <p class=\"search-in-title-js c-article-authors-search__text\"> <span class=\"c-article-authors-search__links-text\"> Search author on: </span> <span class=\"c-article-identifiers\"> <a class=\"c-article-identifiers__item\" href=\"https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Tzu-Yu%20Huang\" rel=\"nofollow\"> PubMed </a> <span class=\"u-hide\"> </span> <a class=\"c-article-identifiers__item\" href=\"https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Tzu-Yu%20Huang%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en\" rel=\"nofollow\"> Google Scholar </a> </span> </p> </div> </div> </li> <li id=\"auth-Meng_Ru-Shen-Aff11-Aff12\"> <span class=\"c-article-authors-search__title u-h3 js-search-name\"> Meng-Ru Shen </span> <div class=\"c-article-authors-search__list\"> <div class=\"c-article-authors-search__item c-article-authors-search__list-item--left\"> <a class=\"c-article-button\" href=\"/search?author=Meng-Ru%20Shen\" rel=\"nofollow\"> View author publications </a> </div> <div class=\"c-article-authors-search__item c-article-authors-search__list-item--right\"> <p class=\"search-in-title-js c-article-authors-search__text\"> <span class=\"c-article-authors-search__links-text\"> Search author on: </span> <span class=\"c-article-identifiers\"> <a class=\"c-article-identifiers__item\" href=\"https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Meng-Ru%20Shen\" rel=\"nofollow\"> PubMed </a> <span class=\"u-hide\"> </span> <a class=\"c-article-identifiers__item\" href=\"https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Meng-Ru%20Shen%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en\" rel=\"nofollow\"> Google Scholar </a> </span> </p> </div> </div> </li> <li id=\"auth-Chun_Hung-Lin-Aff6-Aff13\"> <span class=\"c-article-authors-search__title u-h3 js-search-name\"> Chun-Hung Lin </span> <div class=\"c-article-authors-search__list\"> <div class=\"c-article-authors-search__item c-article-authors-search__list-item--left\"> <a class=\"c-article-button\" href=\"/search?author=Chun-Hung%20Lin\" rel=\"nofollow\"> View author publications </a> </div> <div class=\"c-article-authors-search__item c-article-authors-search__list-item--right\"> <p class=\"search-in-title-js c-article-authors-search__text\"> <span class=\"c-article-authors-search__links-text\"> Search author on: </span> <span class=\"c-article-identifiers\"> <a class=\"c-article-identifiers__item\" href=\"https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Chun-Hung%20Lin\" rel=\"nofollow\"> PubMed </a> <span class=\"u-hide\"> </span> <a class=\"c-article-identifiers__item\" href=\"https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Chun-Hung%20Lin%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en\" rel=\"nofollow\"> Google Scholar </a> </span> </p> </div> </div> </li> <li id=\"auth-Wu_Chou-Su-Aff1-Aff2-Aff3-Aff5-Aff10\"> <span class=\"c-article-authors-search__title u-h3 js-search-name\"> Wu-Chou Su </span> <div class=\"c-article-authors-search__list\"> <div class=\"c-article-authors-search__item c-article-authors-search__list-item--left\"> <a class=\"c-article-button\" href=\"/search?author=Wu-Chou%20Su\" rel=\"nofollow\"> View author publications </a> </div> <div class=\"c-article-authors-search__item c-article-authors-search__list-item--right\"> <p class=\"search-in-title-js c-article-authors-search__text\"> <span class=\"c-article-authors-search__links-text\"> Search author on: </span> <span class=\"c-article-identifiers\"> <a class=\"c-article-identifiers__item\" href=\"https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Wu-Chou%20Su\" rel=\"nofollow\"> PubMed </a> <span class=\"u-hide\"> </span> <a class=\"c-article-identifiers__item\" href=\"https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Wu-Chou%20Su%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en\" rel=\"nofollow\"> Google Scholar </a> </span> </p> </div> </div> </li> </ol> </div> <h3 class=\"c-article__sub-heading\" id=\"contributions\"> Contributions </h3> <p> Chun-Hua Hung: Conceptualization, data curation, software, formal analysis, validation, investigation, visualization, methodology, writing-original draft. Shang-Yin Wu: Software, formal analysis, investigation. Cheng-I Daniel Yao: Formal analysis, methodology. Hsuan-Heng Yeh: Formal analysis, methodology. Chien-Chung Lin: Resources, formal analysis. Chang-Yao Chu: Formal analysis, methodology. Tzu-Yu Huang: Formal analysis, methodology. Meng-Ru Shen: Formal analysis, methodology, funding acquisition. Chun-Hung Lin: Formal analysis, methodology, funding acquisition. Wu-Chou Su*: Conceptualization, resources, supervision, funding acquisition, writing-original draft, project administration. </p> <h3 class=\"c-article__sub-heading\" id=\"corresponding-author\"> Corresponding author </h3> <p id=\"corresponding-author-list\"> Correspondence to <a href=\"mailto:sunnysu@mail.ncku.edu.tw\" id=\"corresp-c1\"> Wu-Chou Su </a> . </p> </div> </div> </section> <section> <div class=\"c-article-section\" id=\"ethics-section\"> <h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"ethics\"> Ethics declarations </h2> <div class=\"c-article-section__content\" id=\"ethics-content\"> <h3 class=\"c-article__sub-heading\" id=\"FPar4\"> Competing interests </h3> <p> The authors declare no competing interests. </p> </div> </div> </section> <section> <div class=\"c-article-section\" id=\"peer-review-section\"> <h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"peer-review\"> Peer review </h2> <div class=\"c-article-section__content\" id=\"peer-review-content\"> <h3 class=\"c-article__sub-heading\" id=\"FPar3\"> Peer review information </h3> <p> <i> Nature Communications </i> thanks Seiji Yano, and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. A peer review file is available. </p> </div> </div> </section> <section> <div class=\"c-article-section\" id=\"additional-information-section\"> <h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"additional-information\"> Additional information </h2> <div class=\"c-article-section__content\" id=\"additional-information-content\"> <p> <b> Publisher’s note </b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations. </p> </div> </div> </section> <section> <div class=\"c-article-section\" id=\"Sec46-section\"> <h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"Sec46\"> Supplementary information </h2> <div class=\"c-article-section__content\" id=\"Sec46-content\"> <div> <div class=\"c-article-figshare-container\" id=\"figshareContainer\"> </div> <div class=\"c-article-supplementary__item\" id=\"MOESM1\"> <h3 class=\"c-article-supplementary__title u-h3\"> <a class=\"print-link\" href=\"https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-024-51831-7/MediaObjects/41467_2024_51831_MOESM1_ESM.pdf\"> Supplementary Information </a> </h3> </div> <div class=\"c-article-supplementary__item\" id=\"MOESM2\"> <h3 class=\"c-article-supplementary__title u-h3\"> <a class=\"print-link\" href=\"https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-024-51831-7/MediaObjects/41467_2024_51831_MOESM2_ESM.pdf\"> Peer Review File </a> </h3> </div> <div class=\"c-article-supplementary__item\" id=\"MOESM3\"> <h3 class=\"c-article-supplementary__title u-h3\"> <a class=\"print-link\" href=\"https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-024-51831-7/MediaObjects/41467_2024_51831_MOESM3_ESM.pdf\"> Reporting Summary </a> </h3> </div> </div> </div> </div> </section> <section> <div class=\"c-article-section\" id=\"Sec47-section\"> <h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"Sec47\"> Source data </h2> <div class=\"c-article-section__content\" id=\"Sec47-content\"> <div> <div class=\"c-article-supplementary__item\" id=\"MOESM4\"> <h3 class=\"c-article-supplementary__title u-h3\"> <a class=\"print-link\" href=\"https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-024-51831-7/MediaObjects/41467_2024_51831_MOESM4_ESM.xlsx\"> Source data </a> </h3> </div> </div> </div> </div> </section> <section> <div class=\"c-article-section\" id=\"rightslink-section\"> <h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"rightslink\"> Rights and permissions </h2> <div class=\"c-article-section__content\" id=\"rightslink-content\"> <p> <b> Open Access </b> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href=\"http://creativecommons.org/licenses/by-nc-nd/4.0/\" rel=\"license\"> http://creativecommons.org/licenses/by-nc-nd/4.0/ </a> . </p> <p class=\"c-article-rights\"> <a href=\"https://s100.copyright.com/AppDispatchServlet?title=Defective%20N-glycosylation%20of%20IL6%20induces%20metastasis%20and%20tyrosine%20kinase%20inhibitor%20resistance%20in%20lung%20cancer&amp;author=Chun-Hua%20Hung%20et%20al&amp;contentID=10.1038%2Fs41467-024-51831-7&amp;copyright=The%20Author%28s%29&amp;publication=2041-1723&amp;publicationDate=2024-09-09&amp;publisherName=SpringerNature&amp;orderBeanReset=true&amp;oa=CC%20BY-NC-ND\"> Reprints and permissions </a> </p> </div> </div> </section> <section aria-labelledby=\"article-info\"> <div class=\"c-article-section\" id=\"article-info-section\"> <h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"article-info\"> About this article </h2> <div class=\"c-article-section__content\" id=\"article-info-content\"> <div class=\"c-bibliographic-information\"> <div class=\"u-hide-print c-bibliographic-information__column c-bibliographic-information__column--border\"> <a href=\"https://crossmark.crossref.org/dialog/?doi=10.1038/s41467-024-51831-7\" rel=\"noopener\" target=\"_blank\"> </a> </div> <div class=\"c-bibliographic-information__column\"> <h3 class=\"c-article__sub-heading\" id=\"citeas\"> Cite this article </h3> <p class=\"c-bibliographic-information__citation\"> Hung, CH., Wu, SY., Yao, CI.D. <i> et al. </i> Defective N-glycosylation of IL6 induces metastasis and tyrosine kinase inhibitor resistance in lung cancer. <i> Nat Commun </i> <b> 15 </b> , 7885 (2024). https://doi.org/10.1038/s41467-024-51831-7 </p> <p class=\"c-bibliographic-information__download-citation u-hide-print\"> <a href=\"https://citation-needed.springer.com/v2/references/10.1038/s41467-024-51831-7?format=refman&amp;flavour=citation\" rel=\"nofollow\"> Download citation </a> </p> <ul class=\"c-bibliographic-information__list\"> <li class=\"c-bibliographic-information__list-item\"> <p> Received <span class=\"u-hide\"> : </span> <span class=\"c-bibliographic-information__value\"> <time datetime=\"2021-10-25\"> 25 October 2021 </time> </span> </p> </li> <li class=\"c-bibliographic-information__list-item\"> <p> Accepted <span class=\"u-hide\"> : </span> <span class=\"c-bibliographic-information__value\"> <time datetime=\"2024-08-19\"> 19 August 2024 </time> </span> </p> </li> <li class=\"c-bibliographic-information__list-item\"> <p> Published <span class=\"u-hide\"> : </span> <span class=\"c-bibliographic-information__value\"> <time datetime=\"2024-09-09\"> 09 September 2024 </time> </span> </p> </li> <li class=\"c-bibliographic-information__list-item c-bibliographic-information__list-item--full-width\"> <p> <abbr title=\"Digital Object Identifier\"> DOI </abbr> <span class=\"u-hide\"> : </span> <span class=\"c-bibliographic-information__value\"> https://doi.org/10.1038/s41467-024-51831-7 </span> </p> </li> </ul> <div> <div class=\"c-article-share-box u-display-block\"> <h3 class=\"c-article__sub-heading\"> Share this article </h3> <p class=\"c-article-share-box__description\"> Anyone you share the following link with will be able to read this content: </p> <div class=\"js-no-share-url-container u-display-none\" hidden=\"\"> <p class=\"js-c-article-share-box__no-sharelink-info c-article-share-box__no-sharelink-info\"> Sorry, a shareable link is not currently available for this article. </p> </div> <div class=\"js-share-url-container u-display-none\" hidden=\"\"> <p class=\"js-share-url c-article-share-box__only-read-input\" id=\"share-url\"> </p> </div> <p class=\"js-c-article-share-box__additional-info c-article-share-box__additional-info\"> Provided by the Springer Nature SharedIt content-sharing initiative </p> </div> </div> <div> <h3 class=\"c-article__sub-heading\"> Subjects </h3> <ul class=\"c-article-subject-list\"> <li class=\"c-article-subject-list__subject\"> <a href=\"/subjects/cancer-therapeutic-resistance\"> Cancer therapeutic resistance </a> </li> <li class=\"c-article-subject-list__subject\"> <a href=\"/subjects/glycobiology\"> Glycobiology </a> </li> <li class=\"c-article-subject-list__subject\"> <a href=\"/subjects/non-small-cell-lung-cancer\"> Non-small-cell lung cancer </a> </li> </ul> </div> </div> </div> </div> </div> </section> </div> <section> <div class=\"c-article-section js-article-section\" id=\"further-reading-section\"> <h2 class=\"c-article-section__title js-section-title js-c-reading-companion-sections-item\" id=\"further-reading\"> This article is cited by </h2> <div class=\"c-article-section__content js-collapsible-section\" id=\"further-reading-content\"> <ul class=\"c-article-further-reading__list\" id=\"further-reading-list\"> <li class=\"c-article-further-reading__item js-ref-item\"> <h3 class=\"c-article-further-reading__title\"> <a class=\"print-link\" href=\"https://doi.org/10.1186/s12967-025-06571-7\"> Nordihydroguaiaretic acid inhibits bladder cancer metastasis through suppression of α1,3-mannosyltransferase expression and LRFN4 N-glycosylation </a> </h3> <ul class=\"app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto\"> <li> Mulin Liu </li> <li> Jingyang Zhang </li> <li> Shijun Li </li> </ul> <p class=\"c-article-further-reading__journal-title\"> <i> Journal of Translational Medicine </i> (2025) </p> </li> <li class=\"c-article-further-reading__item js-ref-item\"> <h3 class=\"c-article-further-reading__title\"> <a class=\"print-link\" href=\"https://doi.org/10.1007/s00280-025-04772-x\"> Influence of interleukin-6 on the pharmacokinetics and pharmacodynamics of osimertinib in patients with non-small cell lung cancer </a> </h3> <ul class=\"app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto\"> <li> Hayato Yokota </li> <li> Kazuhiro Sato </li> <li> Masatomo Miura </li> </ul> <p class=\"c-article-further-reading__journal-title\"> <i> Cancer Chemotherapy and Pharmacology </i> (2025) </p> </li> <li class=\"c-article-further-reading__item js-ref-item\"> <h3 class=\"c-article-further-reading__title\"> <a class=\"print-link\" href=\"https://doi.org/10.1007/s12672-025-02830-7\"> Hotspots and frontiers of extracellular vesicles and lung cancer: a bibliometric and visualization analysis from 2002 to 2024 </a> </h3> <ul class=\"app-author-list app-author-list--compact u-sans-serif u-mb-4 u-mt-auto\"> <li> Zelin Yang </li> <li> Jie Tian </li> </ul> <p class=\"c-article-further-reading__journal-title\"> <i> Discover Oncology </i> (2025) </p> </li> <li class=\"c-article-further-reading__item js-ref-item\"> <h3 class=\"c-article-further-reading__title\"> <a class=\"print-link\" href=\"https://doi.org/10.1007/s12672-025-01782-2\"> Global research trends and emerging hotspots in nano-drug delivery systems for lung cancer: a comprehensive bibliometric analysis (1998–2024) </a> </h3> <ul class=\"app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto\"> <li> Cao Yu </li> <li> Chong-Qi Fan </li> <li> Huan-Wen Chen </li> </ul> <p class=\"c-article-further-reading__journal-title\"> <i> Discover Oncology </i> (2025) </p> </li> </ul> </div> </div> </section> </div> </article> </main> <aside aria-label=\"Article navigation\" class=\"c-article-extras u-hide-print\"> <div class=\"js-context-bar-sticky-point-desktop\"> <div class=\"c-pdf-download u-clear-both js-pdf-download\"> <a class=\"u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link\" download=\"\" href=\"/articles/s41467-024-51831-7.pdf\"> <span class=\"c-pdf-download__text\"> Download PDF </span> </a> </div> </div> <div class=\"c-reading-companion\"> <div class=\"c-reading-companion__sticky c-reading-companion__sticky--stuck\"> <ul class=\"c-reading-companion__tabs\" role=\"tablist\"> <li role=\"presentation\"> </li> <li role=\"presentation\"> </li> <li role=\"presentation\"> </li> </ul> <div aria-labelledby=\"tab-sections\" class=\"c-reading-companion__panel c-reading-companion__sections c-reading-companion__panel--active\" id=\"tabpanel-sections\" role=\"tabpanel\"> <div class=\"c-reading-companion__scroll-pane\"> <ul class=\"c-reading-companion__sections-list\"> <li class=\"c-reading-companion__section-item\" id=\"rc-sec-Abs1\"> <a href=\"#Abs1\"> Abstract </a> </li> <li class=\"c-reading-companion__section-item\" id=\"rc-sec-Sec1\"> <a href=\"#Sec1\"> Introduction </a> </li> <li class=\"c-reading-companion__section-item\" id=\"rc-sec-Sec2\"> <a href=\"#Sec2\"> Results </a> </li> <li class=\"c-reading-companion__section-item\" id=\"rc-sec-Sec10\"> <a href=\"#Sec10\"> Discussion </a> </li> <li class=\"c-reading-companion__section-item\" id=\"rc-sec-Sec11\"> <a href=\"#Sec11\"> Methods </a> </li> <li class=\"c-reading-companion__section-item\" id=\"rc-sec-data-availability\"> <a href=\"#data-availability\"> Data availability </a> </li> <li class=\"c-reading-companion__section-item\" id=\"rc-sec-Bib1\"> <a href=\"#Bib1\"> References </a> </li> <li class=\"c-reading-companion__section-item\" id=\"rc-sec-Ack1\"> <a href=\"#Ack1\"> Acknowledgements </a> </li> <li class=\"c-reading-companion__section-item\" id=\"rc-sec-author-information\"> <a href=\"#author-information\"> Author information </a> </li> <li class=\"c-reading-companion__section-item\" id=\"rc-sec-ethics\"> <a href=\"#ethics\"> Ethics declarations </a> </li> <li class=\"c-reading-companion__section-item\" id=\"rc-sec-peer-review\"> <a href=\"#peer-review\"> Peer review </a> </li> <li class=\"c-reading-companion__section-item\" id=\"rc-sec-additional-information\"> <a href=\"#additional-information\"> Additional information </a> </li> <li class=\"c-reading-companion__section-item\" id=\"rc-sec-Sec46\"> <a href=\"#Sec46\"> Supplementary information </a> </li> <li class=\"c-reading-companion__section-item\" id=\"rc-sec-Sec47\"> <a href=\"#Sec47\"> Source data </a> </li> <li class=\"c-reading-companion__section-item\" id=\"rc-sec-rightslink\"> <a href=\"#rightslink\"> Rights and permissions </a> </li> <li class=\"c-reading-companion__section-item\" id=\"rc-sec-article-info\"> <a href=\"#article-info\"> About this article </a> </li> <li class=\"c-reading-companion__section-item c-reading-companion__section-item--active\" id=\"rc-sec-further-reading\"> <a href=\"#further-reading\"> This article is cited by </a> </li> </ul> </div> <div class=\"u-lazy-ad-wrapper u-mt-16 u-show\"> <div class=\"c-ad c-ad--300x250\"> <div class=\"c-ad__inner\"> <p class=\"c-ad__label\"> Advertisement </p> <div class=\"div-gpt-ad advert medium-rectangle js-ad text-center hide-print grade-c-hide\" id=\"div-gpt-ad-right-2\"> <div id=\"google_ads_iframe_/270604982/nature/ncomms/articles_1__container__\"> </div> </div> </div> </div> </div> </div> <div aria-labelledby=\"tab-figures\" class=\"c-reading-companion__panel c-reading-companion__figures c-reading-companion__panel--full-width\" id=\"tabpanel-figures\" role=\"tabpanel\"> <div class=\"c-reading-companion__scroll-pane\"> <ul class=\"c-reading-companion__figures-list\"> <li class=\"c-reading-companion__figure-item\"> <figure> <figcaption> <b class=\"c-reading-companion__figure-title u-h4\" id=\"rc-Fig1\"> Fig. 1: IL6 Is Modified by N-Glycosylation in NSCLC. </b> </figcaption> <picture> <source/> </picture> <p class=\"c-reading-companion__figure-links\"> <a href=\"#Fig1\"> View in article </a> <a class=\"c-reading-companion__figure-full-link\" href=\"https://www.nature.com/articles/s41467-024-51831-7/figures/1\" rel=\"nofollow\"> Full size image </a> </p> </figure> </li> <li class=\"c-reading-companion__figure-item\"> <figure> <figcaption> <b class=\"c-reading-companion__figure-title u-h4\" id=\"rc-Fig2\"> Fig. 2: N-glycosylated IL6 promotes the tyrosine phosphorylation and nuclear retention of STAT3. </b> </figcaption> <picture> <source/> </picture> <p class=\"c-reading-companion__figure-links\"> <a href=\"#Fig2\"> View in article </a> <a class=\"c-reading-companion__figure-full-link\" href=\"https://www.nature.com/articles/s41467-024-51831-7/figures/2\" rel=\"nofollow\"> Full size image </a> </p> </figure> </li> <li class=\"c-reading-companion__figure-item\"> <figure> <figcaption> <b class=\"c-reading-companion__figure-title u-h4\" id=\"rc-Fig3\"> Fig. 3: deNG-IL6 Induces EMT, Migration, and Metastasis in NSCLC. </b> </figcaption> <picture> <source/> </picture> <p class=\"c-reading-companion__figure-links\"> <a href=\"#Fig3\"> View in article </a> <a class=\"c-reading-companion__figure-full-link\" href=\"https://www.nature.com/articles/s41467-024-51831-7/figures/3\" rel=\"nofollow\"> Full size image </a> </p> </figure> </li> <li class=\"c-reading-companion__figure-item\"> <figure> <figcaption> <b class=\"c-reading-companion__figure-title u-h4\" id=\"rc-Fig4\"> Fig. 4: deNG-IL6 Switches the GP130 Signaling Preference from the JAK-STAT3 Axis to the SRC-YAP Axis. </b> </figcaption> <picture> <source/> </picture> <p class=\"c-reading-companion__figure-links\"> <a href=\"#Fig4\"> View in article </a> <a class=\"c-reading-companion__figure-full-link\" href=\"https://www.nature.com/articles/s41467-024-51831-7/figures/4\" rel=\"nofollow\"> Full size image </a> </p> </figure> </li> <li class=\"c-reading-companion__figure-item\"> <figure> <figcaption> <b class=\"c-reading-companion__figure-title u-h4\" id=\"rc-Fig5\"> Fig. 5: SOX2 Contributes to deNG-IL6-Induced SRC-YAP Axis-Promoted Migration and Metastasis. </b> </figcaption> <picture> <source/> </picture> <p class=\"c-reading-companion__figure-links\"> <a href=\"#Fig5\"> View in article </a> <a class=\"c-reading-companion__figure-full-link\" href=\"https://www.nature.com/articles/s41467-024-51831-7/figures/5\" rel=\"nofollow\"> Full size image </a> </p> </figure> </li> <li class=\"c-reading-companion__figure-item\"> <figure> <figcaption> <b class=\"c-reading-companion__figure-title u-h4\" id=\"rc-Fig6\"> Fig. 6: Inhibition of osimertinib-induced deNG-IL6 restores TKI sensitivity in TKI-resistant NSCLC. </b> </figcaption> <picture> <source/> </picture> <p class=\"c-reading-companion__figure-links\"> <a href=\"#Fig6\"> View in article </a> <a class=\"c-reading-companion__figure-full-link\" href=\"https://www.nature.com/articles/s41467-024-51831-7/figures/6\" rel=\"nofollow\"> Full size image </a> </p> </figure> </li> <li class=\"c-reading-companion__figure-item\"> <figure> <figcaption> <b class=\"c-reading-companion__figure-title u-h4\" id=\"rc-Fig7\"> Fig. 7: Osimertinib-induced deNG-IL6 activates the SRC-YAP-SOX2 axis and migration in TKI-resistant NSCLC. </b> </figcaption> <picture> <source/> </picture> <p class=\"c-reading-companion__figure-links\"> <a href=\"#Fig7\"> View in article </a> <a class=\"c-reading-companion__figure-full-link\" href=\"https://www.nature.com/articles/s41467-024-51831-7/figures/7\" rel=\"nofollow\"> Full size image </a> </p> </figure> </li> <li class=\"c-reading-companion__figure-item\"> <figure> <figcaption> <b class=\"c-reading-companion__figure-title u-h4\" id=\"rc-Fig8\"> Fig. 8: EGFR TKI-resistant NSCLC patients show high deNG-IL6 expression and a shift signaling preference from JAK-STAT3 to the SRC-YAP axis. </b> </figcaption> <picture> <source/> </picture> <p class=\"c-reading-companion__figure-links\"> <a href=\"#Fig8\"> View in article </a> <a class=\"c-reading-companion__figure-full-link\" href=\"https://www.nature.com/articles/s41467-024-51831-7/figures/8\" rel=\"nofollow\"> Full size image </a> </p> </figure> </li> <li class=\"c-reading-companion__figure-item\"> <figure> <figcaption> <b class=\"c-reading-companion__figure-title u-h4\" id=\"rc-Fig9\"> Fig. 9: Long-Term EGFR TKI Treatment Induces deNG-IL6 to Drive Resistance. </b> </figcaption> <picture> <source/> </picture> <p class=\"c-reading-companion__figure-links\"> <a href=\"#Fig9\"> View in article </a> <a class=\"c-reading-companion__figure-full-link\" href=\"https://www.nature.com/articles/s41467-024-51831-7/figures/9\" rel=\"nofollow\"> Full size image </a> </p> </figure> </li> </ul> </div> </div> <div aria-labelledby=\"tab-references\" class=\"c-reading-companion__panel c-reading-companion__references c-reading-companion__panel--full-width\" id=\"tabpanel-references\" role=\"tabpanel\"> <div class=\"c-reading-companion__scroll-pane\"> <ol class=\"c-reading-companion__references-list c-reading-companion__references-list--numeric\"> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR1\"> Yu, H., Lee, H., Herrmann, A., Buettner, R. &amp; Jove, R. Revisiting STAT3 signalling in cancer: new and unexpected biological functions. <i> Nat. Rev. Cancer </i> <b> 14 </b> , 736–746 (2014). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1038%2Fnrc3818\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhvVSmtLnF\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25342631\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Revisiting%20STAT3%20signalling%20in%20cancer%3A%20new%20and%20unexpected%20biological%20functions&amp;journal=Nat.%20Rev.%20Cancer&amp;doi=10.1038%2Fnrc3818&amp;volume=14&amp;pages=736-746&amp;publication_year=2014&amp;author=Yu%2CH&amp;author=Lee%2CH&amp;author=Herrmann%2CA&amp;author=Buettner%2CR&amp;author=Jove%2CR\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR2\"> Jones, S. A. &amp; Jenkins, B. J. Recent insights into targeting the IL-6 cytokine family in inflammatory diseases and cancer. <i> Nat. Rev. Immunol. </i> <b> 18 </b> , 773–789 (2018). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1038%2Fs41577-018-0066-7\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhslymur7N\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30254251\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Recent%20insights%20into%20targeting%20the%20IL-6%20cytokine%20family%20in%20inflammatory%20diseases%20and%20cancer&amp;journal=Nat.%20Rev.%20Immunol.&amp;doi=10.1038%2Fs41577-018-0066-7&amp;volume=18&amp;pages=773-789&amp;publication_year=2018&amp;author=Jones%2CSA&amp;author=Jenkins%2CBJ\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR3\"> Rose-John, S. The soluble interleukin 6 receptor: advanced therapeutic options in inflammation. <i> Clin. Pharmacol. Therapeut. </i> <b> 102 </b> , 591–598 (2017). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1002%2Fcpt.782\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhsFWjt77I\"> CAS </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=The%20soluble%20interleukin%206%20receptor%3A%20advanced%20therapeutic%20options%20in%20inflammation&amp;journal=Clin.%20Pharmacol.%20Therapeut.&amp;doi=10.1002%2Fcpt.782&amp;volume=102&amp;pages=591-598&amp;publication_year=2017&amp;author=Rose-John%2CS\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR4\"> Trikha, M., Corringham, R., Klein, B. &amp; Rossi, J. F. Targeted anti-interleukin-6 monoclonal antibody therapy for cancer: a review of the rationale and clinical evidence. <i> Clin. Cancer Res. </i> <b> 9 </b> , 4653–4665 (2003). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3sXosVGjsrw%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=14581334\"> PubMed </a> <a href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2929399\"> PubMed Central </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Targeted%20anti-interleukin-6%20monoclonal%20antibody%20therapy%20for%20cancer%3A%20a%20review%20of%20the%20rationale%20and%20clinical%20evidence&amp;journal=Clin.%20Cancer%20Res.&amp;volume=9&amp;pages=4653-4665&amp;publication_year=2003&amp;author=Trikha%2CM&amp;author=Corringham%2CR&amp;author=Klein%2CB&amp;author=Rossi%2CJF\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR5\"> Hunter, C. A. &amp; Jones, S. A. IL-6 as a keystone cytokine in health and disease. <i> Nat. Immunol </i> . <b> 16 </b> , 448–457 (2015). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1038%2Fni.3153\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhtF2murbF\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25898198\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=IL-6%20as%20a%20keystone%20cytokine%20in%20health%20and%20disease&amp;journal=Nat.%20Immunol&amp;doi=10.1038%2Fni.3153&amp;volume=16&amp;pages=448-457&amp;publication_year=2015&amp;author=Hunter%2CCA&amp;author=Jones%2CSA\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR6\"> Garbers, C., Heink, S., Korn, T. &amp; Rose-John, S. Interleukin-6: designing specific therapeutics for a complex cytokine. <i> Nat. Rev. Drug Discov. </i> <b> 17 </b> , 395–412 (2018). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1038%2Fnrd.2018.45\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXovVagtbw%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29725131\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Interleukin-6%3A%20designing%20specific%20therapeutics%20for%20a%20complex%20cytokine&amp;journal=Nat.%20Rev.%20Drug%20Discov.&amp;doi=10.1038%2Fnrd.2018.45&amp;volume=17&amp;pages=395-412&amp;publication_year=2018&amp;author=Garbers%2CC&amp;author=Heink%2CS&amp;author=Korn%2CT&amp;author=Rose-John%2CS\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR7\"> Murakami, M., Kamimura, D. &amp; Hirano, T. Pleiotropy and specificity: insights from the interleukin 6 family of cytokines. <i> Immunity </i> <b> 50 </b> , 812–831 (2019). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1016%2Fj.immuni.2019.03.027\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXnsFGksLY%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30995501\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Pleiotropy%20and%20specificity%3A%20insights%20from%20the%20interleukin%206%20family%20of%20cytokines&amp;journal=Immunity&amp;doi=10.1016%2Fj.immuni.2019.03.027&amp;volume=50&amp;pages=812-831&amp;publication_year=2019&amp;author=Murakami%2CM&amp;author=Kamimura%2CD&amp;author=Hirano%2CT\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR8\"> Duan, Z. et al. Signal transducers and activators of transcription 3 pathway activation in drug-resistant ovarian cancer. <i> Clin. Cancer Res. </i> <b> 12 </b> , 5055–5063 (2006). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1158%2F1078-0432.CCR-06-0861\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28XovF2hu7s%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16951221\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Signal%20transducers%20and%20activators%20of%20transcription%203%20pathway%20activation%20in%20drug-resistant%20ovarian%20cancer&amp;journal=Clin.%20Cancer%20Res.&amp;doi=10.1158%2F1078-0432.CCR-06-0861&amp;volume=12&amp;pages=5055-5063&amp;publication_year=2006&amp;author=Duan%2CZ\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR9\"> Huang, W. L. et al. Signal transducer and activator of transcription 3 activation up-regulates interleukin-6 autocrine production: a biochemical and genetic study of established cancer cell lines and clinical isolated human cancer cells. <i> Mol. Cancer </i> <b> 9 </b> , 309 (2010). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://link.springer.com/doi/10.1186/1476-4598-9-309\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhsF2hs7rK\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21122157\"> PubMed </a> <a href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3027602\"> PubMed Central </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Signal%20transducer%20and%20activator%20of%20transcription%203%20activation%20up-regulates%20interleukin-6%20autocrine%20production%3A%20a%20biochemical%20and%20genetic%20study%20of%20established%20cancer%20cell%20lines%20and%20clinical%20isolated%20human%20cancer%20cells&amp;journal=Mol.%20Cancer&amp;doi=10.1186%2F1476-4598-9-309&amp;volume=9&amp;publication_year=2010&amp;author=Huang%2CWL\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR10\"> Lee, H. J. et al. Drug resistance via feedback activation of Stat3 in oncogene-addicted cancer cells. <i> Cancer Cell </i> <b> 26 </b> , 207–221 (2014). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1016%2Fj.ccr.2014.05.019\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXht1agsL7I\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25065853\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Drug%20resistance%20via%20feedback%20activation%20of%20Stat3%20in%20oncogene-addicted%20cancer%20cells&amp;journal=Cancer%20Cell&amp;doi=10.1016%2Fj.ccr.2014.05.019&amp;volume=26&amp;pages=207-221&amp;publication_year=2014&amp;author=Lee%2CHJ\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR11\"> Rokavec, M. et al. IL-6R/STAT3/miR-34a feedback loop promotes EMT-mediated colorectal cancer invasion and metastasis. <i> J. Clin. Invest. </i> <b> 124 </b> , 1853–1867 (2014). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1172%2FJCI73531\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXmtVGgur4%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24642471\"> PubMed </a> <a href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3973098\"> PubMed Central </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=IL-6R%2FSTAT3%2FmiR-34a%20feedback%20loop%20promotes%20EMT-mediated%20colorectal%20cancer%20invasion%20and%20metastasis&amp;journal=J.%20Clin.%20Invest.&amp;doi=10.1172%2FJCI73531&amp;volume=124&amp;pages=1853-1867&amp;publication_year=2014&amp;author=Rokavec%2CM\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR12\"> Taniguchi, K. et al. A gp130-Src-YAP module links inflammation to epithelial regeneration. <i> Nature </i> <b> 519 </b> , 57–62 (2015). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1038%2Fnature14228\"> Article </a> <a href=\"http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2015Natur.519...57T\"> ADS </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXjvVGjsrk%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25731159\"> PubMed </a> <a href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4447318\"> PubMed Central </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=A%20gp130-Src-YAP%20module%20links%20inflammation%20to%20epithelial%20regeneration&amp;journal=Nature&amp;doi=10.1038%2Fnature14228&amp;volume=519&amp;pages=57-62&amp;publication_year=2015&amp;author=Taniguchi%2CK\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR13\"> Quintanal-Villalonga, A. et al. Lineage plasticity in cancer: a shared pathway of therapeutic resistance. <i> Nat. Rev. Clin. Oncol. </i> <b> 17 </b> , 360–371 (2020). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1038%2Fs41571-020-0340-z\"> Article </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32152485\"> PubMed </a> <a href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7397755\"> PubMed Central </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Lineage%20plasticity%20in%20cancer%3A%20a%20shared%20pathway%20of%20therapeutic%20resistance&amp;journal=Nat.%20Rev.%20Clin.%20Oncol.&amp;doi=10.1038%2Fs41571-020-0340-z&amp;volume=17&amp;pages=360-371&amp;publication_year=2020&amp;author=Quintanal-Villalonga%2CA\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR14\"> Paez, J. G. et al. EGFR mutations in lung cancer: correlation with clinical response to gefitinib therapy. <i> Science </i> <b> 304 </b> , 1497–1500 (2004). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1126%2Fscience.1099314\"> Article </a> <a href=\"http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2004Sci...304.1497P\"> ADS </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXksVGmsbs%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15118125\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=EGFR%20mutations%20in%20lung%20cancer%3A%20correlation%20with%20clinical%20response%20to%20gefitinib%20therapy&amp;journal=Science&amp;doi=10.1126%2Fscience.1099314&amp;volume=304&amp;pages=1497-1500&amp;publication_year=2004&amp;author=Paez%2CJG\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR15\"> Ramalingam, S. S. et al. Overall survival with osimertinib in untreated, EGFR-mutated advanced NSCLC. <i> N. Engl. J. Med. </i> <b> 382 </b> , 41–50 (2020). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1056%2FNEJMoa1913662\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXnsFGgtL4%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31751012\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Overall%20survival%20with%20osimertinib%20in%20untreated%2C%20EGFR-mutated%20advanced%20NSCLC&amp;journal=N.%20Engl.%20J.%20Med.&amp;doi=10.1056%2FNEJMoa1913662&amp;volume=382&amp;pages=41-50&amp;publication_year=2020&amp;author=Ramalingam%2CSS\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR16\"> Oxnard, G. R. et al. Assessment of resistance mechanisms and clinical implications in patients with EGFR T790M-positive lung cancer and acquired resistance to osimertinib. <i> JAMA Oncol. </i> <b> 4 </b> , 1527–1534 (2018). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1001%2Fjamaoncol.2018.2969\"> Article </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30073261\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Assessment%20of%20resistance%20mechanisms%20and%20clinical%20implications%20in%20patients%20with%20EGFR%20T790M-positive%20lung%20cancer%20and%20acquired%20resistance%20to%20osimertinib&amp;journal=JAMA%20Oncol.&amp;doi=10.1001%2Fjamaoncol.2018.2969&amp;volume=4&amp;pages=1527-1534&amp;publication_year=2018&amp;author=Oxnard%2CGR\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR17\"> Yang, Z. et al. Investigating novel resistance mechanisms to third-generation EGFR tyrosine kinase inhibitor osimertinib in non-small cell lung cancer patients. <i> Clin. Cancer Res. </i> <b> 24 </b> , 3097–3107 (2018). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1158%2F1078-0432.CCR-17-2310\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXht1alurvK\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29506987\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Investigating%20novel%20resistance%20mechanisms%20to%20third-generation%20EGFR%20tyrosine%20kinase%20inhibitor%20osimertinib%20in%20non-small%20cell%20lung%20cancer%20patients&amp;journal=Clin.%20Cancer%20Res.&amp;doi=10.1158%2F1078-0432.CCR-17-2310&amp;volume=24&amp;pages=3097-3107&amp;publication_year=2018&amp;author=Yang%2CZ\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR18\"> Sequist, L. V. et al. Genotypic and histological evolution of lung cancers acquiring resistance to EGFR inhibitors. <i> Sci. Transl. Med. </i> <b> 3 </b> , 75ra26 (2011). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1126%2Fscitranslmed.3002003\"> Article </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21430269\"> PubMed </a> <a href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3132801\"> PubMed Central </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Genotypic%20and%20histological%20evolution%20of%20lung%20cancers%20acquiring%20resistance%20to%20EGFR%20inhibitors&amp;journal=Sci.%20Transl.%20Med.&amp;doi=10.1126%2Fscitranslmed.3002003&amp;volume=3&amp;publication_year=2011&amp;author=Sequist%2CLV\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR19\"> Lambert, A. W. &amp; Weinberg, R. A. Linking EMT programmes to normal and neoplastic epithelial stem cells. <i> Nat. Rev. Cancer </i> <b> 21 </b> , 325–338 (2021). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1038%2Fs41568-021-00332-6\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXjsVCquro%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33547455\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Linking%20EMT%20programmes%20to%20normal%20and%20neoplastic%20epithelial%20stem%20cells&amp;journal=Nat.%20Rev.%20Cancer&amp;doi=10.1038%2Fs41568-021-00332-6&amp;volume=21&amp;pages=325-338&amp;publication_year=2021&amp;author=Lambert%2CAW&amp;author=Weinberg%2CRA\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR20\"> Williams, E. D., Gao, D., Redfern, A. &amp; Thompson, E. W. Controversies around epithelial-mesenchymal plasticity in cancer metastasis. <i> Nat. Rev. Cancer </i> <b> 19 </b> , 716–732 (2019). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1038%2Fs41568-019-0213-x\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXitV2hsb7J\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31666716\"> PubMed </a> <a href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7055151\"> PubMed Central </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Controversies%20around%20epithelial-mesenchymal%20plasticity%20in%20cancer%20metastasis&amp;journal=Nat.%20Rev.%20Cancer&amp;doi=10.1038%2Fs41568-019-0213-x&amp;volume=19&amp;pages=716-732&amp;publication_year=2019&amp;author=Williams%2CED&amp;author=Gao%2CD&amp;author=Redfern%2CA&amp;author=Thompson%2CEW\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR21\"> Yao, Z. et al. TGF-beta IL-6 axis mediates selective and adaptive mechanisms of resistance to molecular targeted therapy in lung cancer. <i> Proc. Natl Acad. Sci. USA </i> <b> 107 </b> , 15535–15540 (2010). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1073%2Fpnas.1009472107\"> Article </a> <a href=\"http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2010PNAS..10715535Y\"> ADS </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtFaqu7%2FN\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20713723\"> PubMed </a> <a href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2932568\"> PubMed Central </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=TGF-beta%20IL-6%20axis%20mediates%20selective%20and%20adaptive%20mechanisms%20of%20resistance%20to%20molecular%20targeted%20therapy%20in%20lung%20cancer&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.1009472107&amp;volume=107&amp;pages=15535-15540&amp;publication_year=2010&amp;author=Yao%2CZ\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR22\"> Kim, S. M. et al. Activation of IL-6R/JAK1/STAT signaling induces de novo resistance to irreversible EGFR inhibitors in non-small cell lung cancer with T790M resistance mutation. <i> Mol. Cancer Ther. </i> <b> 11 </b> , 2254–2264 (2012). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1158%2F1535-7163.MCT-12-0311\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhsVyksrvI\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22891040\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Activation%20of%20IL-6R%2FJAK1%2FSTAT%20signaling%20induces%20de%20novo%20resistance%20to%20irreversible%20EGFR%20inhibitors%20in%20non-small%20cell%20lung%20cancer%20with%20T790M%20resistance%20mutation&amp;journal=Mol.%20Cancer%20Ther.&amp;doi=10.1158%2F1535-7163.MCT-12-0311&amp;volume=11&amp;pages=2254-2264&amp;publication_year=2012&amp;author=Kim%2CSM\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR23\"> Korkaya, H., Liu, S. &amp; Wicha, M. S. Regulation of cancer stem cells by cytokine networks: attacking cancer’s inflammatory roots. <i> Clin. Cancer Res. </i> <b> 17 </b> , 6125–6129 (2011). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1158%2F1078-0432.CCR-10-2743\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXht1Ggs7jP\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21685479\"> PubMed </a> <a href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3312242\"> PubMed Central </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Regulation%20of%20cancer%20stem%20cells%20by%20cytokine%20networks%3A%20attacking%20cancer%E2%80%99s%20inflammatory%20roots&amp;journal=Clin.%20Cancer%20Res.&amp;doi=10.1158%2F1078-0432.CCR-10-2743&amp;volume=17&amp;pages=6125-6129&amp;publication_year=2011&amp;author=Korkaya%2CH&amp;author=Liu%2CS&amp;author=Wicha%2CMS\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR24\"> Senescence-Associated Pro-inflammatory Cytokines, VernotJ. P. and tumor cell plasticity. <i> Front. Mol. Biosci. </i> <b> 7 </b> , 63 (2020). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2020FrMat...7...63S\"> ADS </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=and%20tumor%20cell%20plasticity&amp;journal=Front.%20Mol.%20Biosci.&amp;volume=7&amp;publication_year=2020&amp;author=Senescence-Associated%20Pro-inflammatory%20Cytokines%2CVernotJP\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR25\"> Kurppa, K. J. et al. Treatment-induced tumor dormancy through YAP-mediated transcriptional reprogramming of the apoptotic pathway. <i> Cancer Cell </i> <b> 37 </b> , 104–122.e112 (2020). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1016%2Fj.ccell.2019.12.006\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXotVent78%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31935369\"> PubMed </a> <a href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7146079\"> PubMed Central </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Treatment-induced%20tumor%20dormancy%20through%20YAP-mediated%20transcriptional%20reprogramming%20of%20the%20apoptotic%20pathway&amp;journal=Cancer%20Cell&amp;doi=10.1016%2Fj.ccell.2019.12.006&amp;volume=37&amp;pages=104-122.e112&amp;publication_year=2020&amp;author=Kurppa%2CKJ\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR26\"> Ferone, G. et al. SOX2 is the determining oncogenic switch in promoting lung squamous cell carcinoma from different cells of origin. <i> Cancer Cell </i> <b> 30 </b> , 519–532 (2016). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1016%2Fj.ccell.2016.09.001\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xhs1GqtLbP\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27728803\"> PubMed </a> <a href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5065004\"> PubMed Central </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=SOX2%20is%20the%20determining%20oncogenic%20switch%20in%20promoting%20lung%20squamous%20cell%20carcinoma%20from%20different%20cells%20of%20origin&amp;journal=Cancer%20Cell&amp;doi=10.1016%2Fj.ccell.2016.09.001&amp;volume=30&amp;pages=519-532&amp;publication_year=2016&amp;author=Ferone%2CG\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR27\"> Boumahdi, S. &amp; de Sauvage, F. J. The great escape: tumour cell plasticity in resistance to targeted therapy. <i> Nat. Rev. Drug Discov. </i> <b> 19 </b> , 39–56 (2020). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1038%2Fs41573-019-0044-1\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhvFCktL%2FM\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31601994\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=The%20great%20escape%3A%20tumour%20cell%20plasticity%20in%20resistance%20to%20targeted%20therapy&amp;journal=Nat.%20Rev.%20Drug%20Discov.&amp;doi=10.1038%2Fs41573-019-0044-1&amp;volume=19&amp;pages=39-56&amp;publication_year=2020&amp;author=Boumahdi%2CS&amp;author=Sauvage%2CFJ\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR28\"> Mosteiro, L. et al. Tissue damage and senescence provide critical signals for cellular reprogramming in vivo. <i> Science </i> <b> 354 </b> , 6315 (2016). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1126%2Fscience.aaf4445\"> Article </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Tissue%20damage%20and%20senescence%20provide%20critical%20signals%20for%20cellular%20reprogramming%20in%20vivo&amp;journal=Science&amp;doi=10.1126%2Fscience.aaf4445&amp;volume=354&amp;publication_year=2016&amp;author=Mosteiro%2CL\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR29\"> Bora-Singhal, N. et al. YAP1 regulates OCT4 activity and SOX2 expression to facilitate self-renewal and vascular mimicry of stem-like cells. <i> Stem cells </i> <b> 33 </b> , 1705–1718 (2015). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1002%2Fstem.1993\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhtleqsLvK\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25754111\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=YAP1%20regulates%20OCT4%20activity%20and%20SOX2%20expression%20to%20facilitate%20self-renewal%20and%20vascular%20mimicry%20of%20stem-like%20cells&amp;journal=Stem%20cells&amp;doi=10.1002%2Fstem.1993&amp;volume=33&amp;pages=1705-1718&amp;publication_year=2015&amp;author=Bora-Singhal%2CN\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR30\"> Poupart, P. et al. B cell growth modulating and differentiating activity of recombinant human 26-kd protein (BSF-2, HuIFN-beta 2, HPGF). <i> EMBO J </i> <b> 6 </b> , 1219–1224 (1987). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1002%2Fj.1460-2075.1987.tb02357.x\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaL2sXkt1Wktr0%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=3497029\"> PubMed </a> <a href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC553922\"> PubMed Central </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=B%20cell%20growth%20modulating%20and%20differentiating%20activity%20of%20recombinant%20human%2026-kd%20protein%20%28BSF-2%2C%20HuIFN-beta%202%2C%20HPGF%29&amp;journal=EMBO%20J&amp;doi=10.1002%2Fj.1460-2075.1987.tb02357.x&amp;volume=6&amp;pages=1219-1224&amp;publication_year=1987&amp;author=Poupart%2CP\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR31\"> May, L. T. et al. Phosphorylation of secreted forms of human beta 2-interferon/hepatocyte stimulating factor/interleukin-6. <i> Biochem. Biophys. Res. Commun. </i> <b> 152 </b> , 1144–1150 (1988). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1016%2FS0006-291X%2888%2980404-2\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaL1cXktVGjt70%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=3259872\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Phosphorylation%20of%20secreted%20forms%20of%20human%20beta%202-interferon%2Fhepatocyte%20stimulating%20factor%2Finterleukin-6&amp;journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;doi=10.1016%2FS0006-291X%2888%2980404-2&amp;volume=152&amp;pages=1144-1150&amp;publication_year=1988&amp;author=May%2CLT\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR32\"> Hirano, T. et al. Purification to homogeneity and characterization of human B-cell differentiation factor (BCDF or BSFp-2). <i> Proc. Natl Acad. Sci. USA </i> <b> 82 </b> , 5490–5494 (1985). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1073%2Fpnas.82.16.5490\"> Article </a> <a href=\"http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=1985PNAS...82.5490H\"> ADS </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaL2MXlt1Wnt7s%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=2410927\"> PubMed </a> <a href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC391148\"> PubMed Central </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Purification%20to%20homogeneity%20and%20characterization%20of%20human%20B-cell%20differentiation%20factor%20%28BCDF%20or%20BSFp-2%29&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.82.16.5490&amp;volume=82&amp;pages=5490-5494&amp;publication_year=1985&amp;author=Hirano%2CT\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR33\"> May, L. T., Santhanam, U. &amp; Sehgal, P. B. On the multimeric nature of natural human interleukin-6. <i> J. Biol. Chem. </i> <b> 266 </b> , 9950–9955 (1991). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1016%2FS0021-9258%2818%2992911-X\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK3MXkt1Onsbw%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=2033081\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=On%20the%20multimeric%20nature%20of%20natural%20human%20interleukin-6&amp;journal=J.%20Biol.%20Chem.&amp;doi=10.1016%2FS0021-9258%2818%2992911-X&amp;volume=266&amp;pages=9950-9955&amp;publication_year=1991&amp;author=May%2CLT&amp;author=Santhanam%2CU&amp;author=Sehgal%2CPB\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR34\"> Mereiter, S., Balmana, M., Campos, D., Gomes, J. &amp; Reis, C. A. Glycosylation in the era of cancer-targeted therapy: where are we heading? <i> Cancer Cell </i> <b> 36 </b> , 6–16 (2019). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1016%2Fj.ccell.2019.06.006\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtlersb%2FJ\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31287993\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Glycosylation%20in%20the%20era%20of%20cancer-targeted%20therapy%3A%20where%20are%20we%20heading%3F&amp;journal=Cancer%20Cell&amp;doi=10.1016%2Fj.ccell.2019.06.006&amp;volume=36&amp;pages=6-16&amp;publication_year=2019&amp;author=Mereiter%2CS&amp;author=Balmana%2CM&amp;author=Campos%2CD&amp;author=Gomes%2CJ&amp;author=Reis%2CCA\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR35\"> Parekh, R. B., Dwek, R. A., Rademacher, T. W., Opdenakker, G. &amp; Van Damme, J. Glycosylation of interleukin-6 purified from normal human blood mononuclear cells. <i> Eur. J. Biochem. </i> <b> 203 </b> , 135–141 (1992). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1111%2Fj.1432-1033.1992.tb19838.x\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK38Xht1KmtL8%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=1730219\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Glycosylation%20of%20interleukin-6%20purified%20from%20normal%20human%20blood%20mononuclear%20cells&amp;journal=Eur.%20J.%20Biochem.&amp;doi=10.1111%2Fj.1432-1033.1992.tb19838.x&amp;volume=203&amp;pages=135-141&amp;publication_year=1992&amp;author=Parekh%2CRB&amp;author=Dwek%2CRA&amp;author=Rademacher%2CTW&amp;author=Opdenakker%2CG&amp;author=Damme%2CJ\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR36\"> Dela Cruz, C. S. et al. N-linked glycosylation is required for optimal function of Kaposi’s sarcoma herpesvirus-encoded, but not cellular, interleukin 6. <i> J. Exp. Med. </i> <b> 199 </b> , 503–514 (2004). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1084%2Fjem.20031205\"> Article </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=N-linked%20glycosylation%20is%20required%20for%20optimal%20function%20of%20Kaposi%E2%80%99s%20sarcoma%20herpesvirus-encoded%2C%20but%20not%20cellular%2C%20interleukin%206&amp;journal=J.%20Exp.%20Med.&amp;doi=10.1084%2Fjem.20031205&amp;volume=199&amp;pages=503-514&amp;publication_year=2004&amp;author=Cruz%2CCS\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR37\"> Thomas, D., Rathinavel, A. K. &amp; Radhakrishnan, P. Altered glycosylation in cancer: a promising target for biomarkers and therapeutics. <i> Biochimica. et biophysica. Acta Rev. Cancer </i> <b> 1875 </b> , 188464 (2021). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1016%2Fj.bbcan.2020.188464\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitl2msr%2FP\"> CAS </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Altered%20glycosylation%20in%20cancer%3A%20a%20promising%20target%20for%20biomarkers%20and%20therapeutics&amp;journal=Biochimica.%20et%20biophysica.%20Acta%20Rev.%20Cancer&amp;doi=10.1016%2Fj.bbcan.2020.188464&amp;volume=1875&amp;publication_year=2021&amp;author=Thomas%2CD&amp;author=Rathinavel%2CAK&amp;author=Radhakrishnan%2CP\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR38\"> Pinho, S. S. &amp; Reis, C. A. Glycosylation in cancer: mechanisms and clinical implications. <i> Nat. Rev. Cancer </i> <b> 15 </b> , 540–555 (2015). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1038%2Fnrc3982\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhtlKkt7fK\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26289314\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Glycosylation%20in%20cancer%3A%20mechanisms%20and%20clinical%20implications&amp;journal=Nat.%20Rev.%20Cancer&amp;doi=10.1038%2Fnrc3982&amp;volume=15&amp;pages=540-555&amp;publication_year=2015&amp;author=Pinho%2CSS&amp;author=Reis%2CCA\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR39\"> Yeh, H. H., Lai, W. W., Chen, H. H., Liu, H. S. &amp; Su, W. C. Autocrine IL-6-induced Stat3 activation contributes to the pathogenesis of lung adenocarcinoma and malignant pleural effusion. <i> Oncogene </i> <b> 25 </b> , 4300–4309 (2006). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1038%2Fsj.onc.1209464\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28XntVWks7o%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16518408\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Autocrine%20IL-6-induced%20Stat3%20activation%20contributes%20to%20the%20pathogenesis%20of%20lung%20adenocarcinoma%20and%20malignant%20pleural%20effusion&amp;journal=Oncogene&amp;doi=10.1038%2Fsj.onc.1209464&amp;volume=25&amp;pages=4300-4309&amp;publication_year=2006&amp;author=Yeh%2CHH&amp;author=Lai%2CWW&amp;author=Chen%2CHH&amp;author=Liu%2CHS&amp;author=Su%2CWC\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR40\"> Roth, J. Protein N-glycosylation along the secretory pathway: relationship to organelle topography and function, protein quality control, and cell interactions. <i> Chem. Rev. </i> <b> 102 </b> , 285–303 (2002). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1021%2Fcr000423j\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD38XntlGrug%3D%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11841244\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Protein%20N-glycosylation%20along%20the%20secretory%20pathway%3A%20relationship%20to%20organelle%20topography%20and%20function%2C%20protein%20quality%20control%2C%20and%20cell%20interactions&amp;journal=Chem.%20Rev.&amp;doi=10.1021%2Fcr000423j&amp;volume=102&amp;pages=285-303&amp;publication_year=2002&amp;author=Roth%2CJ\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR41\"> Bard, F. &amp; Chia, J. Cracking the glycome encoder: signaling, trafficking, and glycosylation. <i> Trends Cell Biol. </i> <b> 26 </b> , 379–388 (2016). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1016%2Fj.tcb.2015.12.004\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XjslKrtA%3D%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26832820\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Cracking%20the%20glycome%20encoder%3A%20signaling%2C%20trafficking%2C%20and%20glycosylation&amp;journal=Trends%20Cell%20Biol.&amp;doi=10.1016%2Fj.tcb.2015.12.004&amp;volume=26&amp;pages=379-388&amp;publication_year=2016&amp;author=Bard%2CF&amp;author=Chia%2CJ\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR42\"> Schjoldager, K. T., Narimatsu, Y., Joshi, H. J. &amp; Clausen, H. Global view of human protein glycosylation pathways and functions. <i> Nat. Rev. Mol. Cell Biol. </i> <b> 21 </b> , 729–749 (2020). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1038%2Fs41580-020-00294-x\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitFGmu73N\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33087899\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Global%20view%20of%20human%20protein%20glycosylation%20pathways%20and%20functions&amp;journal=Nat.%20Rev.%20Mol.%20Cell%20Biol.&amp;doi=10.1038%2Fs41580-020-00294-x&amp;volume=21&amp;pages=729-749&amp;publication_year=2020&amp;author=Schjoldager%2CKT&amp;author=Narimatsu%2CY&amp;author=Joshi%2CHJ&amp;author=Clausen%2CH\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR43\"> Subramanian, A. et al. Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. <i> Proc. Natl Acad. Sci. USA </i> <b> 102 </b> , 15545–15550 (2005). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1073%2Fpnas.0506580102\"> Article </a> <a href=\"http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2005PNAS..10215545S\"> ADS </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXht1ShtrnO\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16199517\"> PubMed </a> <a href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1239896\"> PubMed Central </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Gene%20set%20enrichment%20analysis%3A%20a%20knowledge-based%20approach%20for%20interpreting%20genome-wide%20expression%20profiles&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.0506580102&amp;volume=102&amp;pages=15545-15550&amp;publication_year=2005&amp;author=Subramanian%2CA\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR44\"> Mootha, V. K. et al. PGC-1alpha-responsive genes involved in oxidative phosphorylation are coordinately downregulated in human diabetes. <i> Nat. Genet. </i> <b> 34 </b> , 267–273 (2003). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1038%2Fng1180\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3sXkvFSrs7o%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12808457\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=PGC-1alpha-responsive%20genes%20involved%20in%20oxidative%20phosphorylation%20are%20coordinately%20downregulated%20in%20human%20diabetes&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fng1180&amp;volume=34&amp;pages=267-273&amp;publication_year=2003&amp;author=Mootha%2CVK\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR45\"> Okayama, H. et al. Identification of genes upregulated in ALK-positive and EGFR/KRAS/ALK-negative lung adenocarcinomas. <i> Cancer Res. </i> <b> 72 </b> , 100–111 (2012). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1158%2F0008-5472.CAN-11-1403\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhvVCmtg%3D%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22080568\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Identification%20of%20genes%20upregulated%20in%20ALK-positive%20and%20EGFR%2FKRAS%2FALK-negative%20lung%20adenocarcinomas&amp;journal=Cancer%20Res.&amp;doi=10.1158%2F0008-5472.CAN-11-1403&amp;volume=72&amp;pages=100-111&amp;publication_year=2012&amp;author=Okayama%2CH\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR46\"> Lopez-Sambrooks, C. et al. Oligosaccharyltransferase inhibition induces senescence in RTK-driven tumor cells. <i> Nat. Chem. Biol. </i> <b> 12 </b> , 1023–1030 (2016). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1038%2Fnchembio.2194\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhsFOmsLvE\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27694802\"> PubMed </a> <a href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5393272\"> PubMed Central </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Oligosaccharyltransferase%20inhibition%20induces%20senescence%20in%20RTK-driven%20tumor%20cells&amp;journal=Nat.%20Chem.%20Biol.&amp;doi=10.1038%2Fnchembio.2194&amp;volume=12&amp;pages=1023-1030&amp;publication_year=2016&amp;author=Lopez-Sambrooks%2CC\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR47\"> Blom, N., Sicheritz-Ponten, T., Gupta, R., Gammeltoft, S. &amp; Brunak, S. Prediction of post-translational glycosylation and phosphorylation of proteins from the amino acid sequence. <i> Proteomics </i> <b> 4 </b> , 1633–1649 (2004). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1002%2Fpmic.200300771\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXkvFGitLg%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15174133\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Prediction%20of%20post-translational%20glycosylation%20and%20phosphorylation%20of%20proteins%20from%20the%20amino%20acid%20sequence&amp;journal=Proteomics&amp;doi=10.1002%2Fpmic.200300771&amp;volume=4&amp;pages=1633-1649&amp;publication_year=2004&amp;author=Blom%2CN&amp;author=Sicheritz-Ponten%2CT&amp;author=Gupta%2CR&amp;author=Gammeltoft%2CS&amp;author=Brunak%2CS\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR48\"> Yamaguchi, K. et al. Effects of site-directed removal of N-glycosylation sites in human erythropoietin on its production and biological properties. <i> J. Biol. Chem </i> . <b> 266 </b> , 20434–20439 (1991). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1016%2FS0021-9258%2818%2954942-5\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK3MXlvFykt70%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=1657925\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Effects%20of%20site-directed%20removal%20of%20N-glycosylation%20sites%20in%20human%20erythropoietin%20on%20its%20production%20and%20biological%20properties&amp;journal=J.%20Biol.%20Chem&amp;doi=10.1016%2FS0021-9258%2818%2954942-5&amp;volume=266&amp;pages=20434-20439&amp;publication_year=1991&amp;author=Yamaguchi%2CK\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR49\"> Bobby, R. et al. Functional implications of large backbone amplitude motions of the glycoprotein 130-binding epitope of interleukin-6. <i> FEBS J. </i> <b> 281 </b> , 2471–2483 (2014). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1111%2Ffebs.12800\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXotFemtL0%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24712547\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Functional%20implications%20of%20large%20backbone%20amplitude%20motions%20of%20the%20glycoprotein%20130-binding%20epitope%20of%20interleukin-6&amp;journal=FEBS%20J.&amp;doi=10.1111%2Ffebs.12800&amp;volume=281&amp;pages=2471-2483&amp;publication_year=2014&amp;author=Bobby%2CR\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR50\"> Xiong, H. et al. Roles of STAT3 and ZEB1 proteins in E-cadherin down-regulation and human colorectal cancer epithelial-mesenchymal transition. <i> J. Biol. Chem. </i> <b> 287 </b> , 5819–5832 (2012). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1074%2Fjbc.M111.295964\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XisVyluro%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22205702\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Roles%20of%20STAT3%20and%20ZEB1%20proteins%20in%20E-cadherin%20down-regulation%20and%20human%20colorectal%20cancer%20epithelial-mesenchymal%20transition&amp;journal=J.%20Biol.%20Chem.&amp;doi=10.1074%2Fjbc.M111.295964&amp;volume=287&amp;pages=5819-5832&amp;publication_year=2012&amp;author=Xiong%2CH\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR51\"> Wei, W. et al. STAT3 signaling is activated preferentially in tumor-initiating cells in claudin-low models of human breast cancer. <i> Stem cells </i> <b> 32 </b> , 2571–2582 (2014). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1002%2Fstem.1752\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhvFGjtLzE\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24891218\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=STAT3%20signaling%20is%20activated%20preferentially%20in%20tumor-initiating%20cells%20in%20claudin-low%20models%20of%20human%20breast%20cancer&amp;journal=Stem%20cells&amp;doi=10.1002%2Fstem.1752&amp;volume=32&amp;pages=2571-2582&amp;publication_year=2014&amp;author=Wei%2CW\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR52\"> Yao, C. et al. IGF/STAT3/NANOG/Slug signaling axis simultaneously controls epithelial-mesenchymal transition and stemness maintenance in colorectal cancer. <i> Stem Cells </i> <b> 34 </b> , 820–831 (2016). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1002%2Fstem.2320\"> Article </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26840943\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=IGF%2FSTAT3%2FNANOG%2FSlug%20signaling%20axis%20simultaneously%20controls%20epithelial-mesenchymal%20transition%20and%20stemness%20maintenance%20in%20colorectal%20cancer&amp;journal=Stem%20Cells&amp;doi=10.1002%2Fstem.2320&amp;volume=34&amp;pages=820-831&amp;publication_year=2016&amp;author=Yao%2CC\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR53\"> Na, Y. R., Lee, J. S., Lee, S. J. &amp; Seok, S. H. Interleukin-6-induced twist and N-cadherin enhance melanoma cell metastasis. <i> Melanoma Res. </i> <b> 23 </b> , 434–443 (2013). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1097%2FCMR.0000000000000021\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhs12rtbbK\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24051540\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Interleukin-6-induced%20twist%20and%20N-cadherin%20enhance%20melanoma%20cell%20metastasis&amp;journal=Melanoma%20Res.&amp;doi=10.1097%2FCMR.0000000000000021&amp;volume=23&amp;pages=434-443&amp;publication_year=2013&amp;author=Na%2CYR&amp;author=Lee%2CJS&amp;author=Lee%2CSJ&amp;author=Seok%2CSH\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR54\"> Gallucci, R. M., Lee, E. G. &amp; Tomasek, J. J. IL-6 modulates alpha-smooth muscle actin expression in dermal fibroblasts from IL-6-deficient mice. <i> J. Invest. Dermatol </i> . <b> 126 </b> , 561–568 (2006). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1038%2Fsj.jid.5700109\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28XhsVSrsLc%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16397521\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=IL-6%20modulates%20alpha-smooth%20muscle%20actin%20expression%20in%20dermal%20fibroblasts%20from%20IL-6-deficient%20mice&amp;journal=J.%20Invest.%20Dermatol&amp;doi=10.1038%2Fsj.jid.5700109&amp;volume=126&amp;pages=561-568&amp;publication_year=2006&amp;author=Gallucci%2CRM&amp;author=Lee%2CEG&amp;author=Tomasek%2CJJ\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR55\"> Romera-Hernandez, M. et al. Yap1-driven intestinal repair is controlled by group 3 innate lymphoid cells. <i> Cell Rep. </i> <b> 30 </b> , 37–45.e33 (2020). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1016%2Fj.celrep.2019.11.115\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXotFert70%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31914395\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Yap1-driven%20intestinal%20repair%20is%20controlled%20by%20group%203%20innate%20lymphoid%20cells&amp;journal=Cell%20Rep.&amp;doi=10.1016%2Fj.celrep.2019.11.115&amp;volume=30&amp;pages=37-45.e33&amp;publication_year=2020&amp;author=Romera-Hernandez%2CM\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR56\"> Tsuji, T. et al. YAP1 mediates survival of ALK-rearranged lung cancer cells treated with alectinib via pro-apoptotic protein regulation. <i> Nat. Commun. </i> <b> 11 </b> , 74 (2020). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1038%2Fs41467-019-13771-5\"> Article </a> <a href=\"http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2020NatCo..11...74T\"> ADS </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXlslWisg%3D%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31900393\"> PubMed </a> <a href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6941996\"> PubMed Central </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=YAP1%20mediates%20survival%20of%20ALK-rearranged%20lung%20cancer%20cells%20treated%20with%20alectinib%20via%20pro-apoptotic%20protein%20regulation&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-019-13771-5&amp;volume=11&amp;publication_year=2020&amp;author=Tsuji%2CT\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR57\"> Takahashi, A. et al. Ankyrin repeat domain 1 overexpression is associated with common resistance to afatinib and osimertinib in EGFR-mutant lung cancer. <i> Sci. Rep. </i> <b> 8 </b> , 14896 (2018). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1038%2Fs41598-018-33190-8\"> Article </a> <a href=\"http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2018NatSR...814896T\"> ADS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30291293\"> PubMed </a> <a href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6173712\"> PubMed Central </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Ankyrin%20repeat%20domain%201%20overexpression%20is%20associated%20with%20common%20resistance%20to%20afatinib%20and%20osimertinib%20in%20EGFR-mutant%20lung%20cancer&amp;journal=Sci.%20Rep.&amp;doi=10.1038%2Fs41598-018-33190-8&amp;volume=8&amp;publication_year=2018&amp;author=Takahashi%2CA\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR58\"> Zheng, S. et al. SynergyFinder plus: toward better interpretation and annotation of drug combination screening datasets. <i> Genom. Proteom. Bioinform. </i> <b> 20 </b> , 587–596 (2022). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1016%2Fj.gpb.2022.01.004\"> Article </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=SynergyFinder%20plus%3A%20toward%20better%20interpretation%20and%20annotation%20of%20drug%20combination%20screening%20datasets&amp;journal=Genom.%20Proteom.%20Bioinform.&amp;doi=10.1016%2Fj.gpb.2022.01.004&amp;volume=20&amp;pages=587-596&amp;publication_year=2022&amp;author=Zheng%2CS\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR59\"> Roper, N. et al. Clonal evolution and heterogeneity of osimertinib acquired resistance mechanisms in EGFR mutant lung cancer. <i> Cell Rep. Med. </i> <b> 1 </b> , 10007 (2020). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Clonal%20evolution%20and%20heterogeneity%20of%20osimertinib%20acquired%20resistance%20mechanisms%20in%20EGFR%20mutant%20lung%20cancer&amp;journal=Cell%20Rep.%20Med.&amp;volume=1&amp;publication_year=2020&amp;author=Roper%2CN\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR60\"> Barbie, D. A. et al. Systematic RNA interference reveals that oncogenic KRAS-driven cancers require TBK1. <i> Nature </i> <b> 462 </b> , 108–112 (2009). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1038%2Fnature08460\"> Article </a> <a href=\"http://adsabs.harvard.edu/cgi-bin/nph-data_query?link_type=ABSTRACT&amp;bibcode=2009Natur.462..108B\"> ADS </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXht12gtrvL\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19847166\"> PubMed </a> <a href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2783335\"> PubMed Central </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Systematic%20RNA%20interference%20reveals%20that%20oncogenic%20KRAS-driven%20cancers%20require%20TBK1&amp;journal=Nature&amp;doi=10.1038%2Fnature08460&amp;volume=462&amp;pages=108-112&amp;publication_year=2009&amp;author=Barbie%2CDA\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR61\"> Heinz, M. C. et al. Liver colonization by colorectal cancer metastases requires YAP-controlled plasticity at the micrometastatic stage. <i> Cancer Res. </i> <b> 82 </b> , 1953–1968 (2022). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1158%2F0008-5472.CAN-21-0933\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhsFehsbrE\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35570706\"> PubMed </a> <a href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9381095\"> PubMed Central </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Liver%20colonization%20by%20colorectal%20cancer%20metastases%20requires%20YAP-controlled%20plasticity%20at%20the%20micrometastatic%20stage&amp;journal=Cancer%20Res.&amp;doi=10.1158%2F0008-5472.CAN-21-0933&amp;volume=82&amp;pages=1953-1968&amp;publication_year=2022&amp;author=Heinz%2CMC\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR62\"> Hamanaka, N. et al. YES1 is a targetable oncogene in cancers harboring YES1 gene amplification. <i> Cancer Res. </i> <b> 79 </b> , 5734–5745 (2019). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1158%2F0008-5472.CAN-18-3376\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXjslGjs70%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31391186\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=YES1%20is%20a%20targetable%20oncogene%20in%20cancers%20harboring%20YES1%20gene%20amplification&amp;journal=Cancer%20Res.&amp;doi=10.1158%2F0008-5472.CAN-18-3376&amp;volume=79&amp;pages=5734-5745&amp;publication_year=2019&amp;author=Hamanaka%2CN\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR63\"> Lopez Sambrooks, C. et al. Oligosaccharyltransferase inhibition overcomes therapeutic resistance to EGFR tyrosine kinase inhibitors. <i> Cancer Res. </i> <b> 78 </b> , 5094–5106 (2018). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1158%2F0008-5472.CAN-18-0505\"> Article </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30026325\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Oligosaccharyltransferase%20inhibition%20overcomes%20therapeutic%20resistance%20to%20EGFR%20tyrosine%20kinase%20inhibitors&amp;journal=Cancer%20Res.&amp;doi=10.1158%2F0008-5472.CAN-18-0505&amp;volume=78&amp;pages=5094-5106&amp;publication_year=2018&amp;author=Lopez%20Sambrooks%2CC\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR64\"> Chan, L. C. et al. IL-6/JAK1 pathway drives PD-L1 Y112 phosphorylation to promote cancer immune evasion. <i> J. Clin. Invest. </i> <b> 129 </b> , 3324–3338 (2019). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1172%2FJCI126022\"> Article </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31305264\"> PubMed </a> <a href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6668668\"> PubMed Central </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=IL-6%2FJAK1%20pathway%20drives%20PD-L1%20Y112%20phosphorylation%20to%20promote%20cancer%20immune%20evasion&amp;journal=J.%20Clin.%20Invest.&amp;doi=10.1172%2FJCI126022&amp;volume=129&amp;pages=3324-3338&amp;publication_year=2019&amp;author=Chan%2CLC\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR65\"> Miller, M. R. et al. Downregulation of dystroglycan glycosyltransferases LARGE2 and ISPD associate with increased mortality in clear cell renal cell carcinoma. <i> Mol. Cancer </i> <b> 14 </b> , 141 (2015). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://link.springer.com/doi/10.1186/s12943-015-0416-z\"> Article </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26220087\"> PubMed </a> <a href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4518861\"> PubMed Central </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Downregulation%20of%20dystroglycan%20glycosyltransferases%20LARGE2%20and%20ISPD%20associate%20with%20increased%20mortality%20in%20clear%20cell%20renal%20cell%20carcinoma&amp;journal=Mol.%20Cancer&amp;doi=10.1186%2Fs12943-015-0416-z&amp;volume=14&amp;publication_year=2015&amp;author=Miller%2CMR\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR66\"> Chugh, S., Meza, J., Sheinin, Y. M., Ponnusamy, M. P. &amp; Batra, S. K. Loss of N-acetylgalactosaminyltransferase 3 in poorly differentiated pancreatic cancer: augmented aggressiveness and aberrant ErbB family glycosylation. <i> Br. J. Cancer </i> <b> 114 </b> , 1376–1386 (2016). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1038%2Fbjc.2016.116\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XotFOhurw%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27187683\"> PubMed </a> <a href=\"http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4984453\"> PubMed Central </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Loss%20of%20N-acetylgalactosaminyltransferase%203%20in%20poorly%20differentiated%20pancreatic%20cancer%3A%20augmented%20aggressiveness%20and%20aberrant%20ErbB%20family%20glycosylation&amp;journal=Br.%20J.%20Cancer&amp;doi=10.1038%2Fbjc.2016.116&amp;volume=114&amp;pages=1376-1386&amp;publication_year=2016&amp;author=Chugh%2CS&amp;author=Meza%2CJ&amp;author=Sheinin%2CYM&amp;author=Ponnusamy%2CMP&amp;author=Batra%2CSK\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR67\"> Peng, C. et al. Regulation of the Hippo-YAP pathway by glucose sensor O-GlcNAcylation. <i> Mol. Cell </i> <b> 68 </b> , 591–604.e595 (2017). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1016%2Fj.molcel.2017.10.010\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhslOju7rJ\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29100056\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Regulation%20of%20the%20Hippo-YAP%20pathway%20by%20glucose%20sensor%20O-GlcNAcylation&amp;journal=Mol.%20Cell&amp;doi=10.1016%2Fj.molcel.2017.10.010&amp;volume=68&amp;pages=591-604.e595&amp;publication_year=2017&amp;author=Peng%2CC\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR68\"> Hung, T. L. et al. Clinical evaluation of HER-2/neu protein in malignant pleural effusion-associated lung adenocarcinoma and as a tumor marker in pleural effusion diagnosis. <i> Clin. Cancer Res. </i> <b> 9 </b> , 2605–2612 (2003). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3sXlt1yrsrg%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12855637\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Clinical%20evaluation%20of%20HER-2%2Fneu%20protein%20in%20malignant%20pleural%20effusion-associated%20lung%20adenocarcinoma%20and%20as%20a%20tumor%20marker%20in%20pleural%20effusion%20diagnosis&amp;journal=Clin.%20Cancer%20Res.&amp;volume=9&amp;pages=2605-2612&amp;publication_year=2003&amp;author=Hung%2CTL\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR69\"> Chu, Y. W. et al. Selection of invasive and metastatic subpopulations from a human lung adenocarcinoma cell line. <i> Am. J. Respir. Cell Mol. Biol. </i> <b> 17 </b> , 353–360 (1997). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1165%2Fajrcmb.17.3.2837\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK2sXmtVOrs7c%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9308922\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Selection%20of%20invasive%20and%20metastatic%20subpopulations%20from%20a%20human%20lung%20adenocarcinoma%20cell%20line&amp;journal=Am.%20J.%20Respir.%20Cell%20Mol.%20Biol.&amp;doi=10.1165%2Fajrcmb.17.3.2837&amp;volume=17&amp;pages=353-360&amp;publication_year=1997&amp;author=Chu%2CYW\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR70\"> Wisniewski, J. R., Zougman, A., Nagaraj, N. &amp; Mann, M. Universal sample preparation method for proteome analysis. <i> Nat. Methods </i> <b> 6 </b> , 359–362 (2009). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"https://doi.org/10.1038%2Fnmeth.1322\"> Article </a> <a href=\"https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXks12ksb0%3D\"> CAS </a> <a href=\"http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19377485\"> PubMed </a> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Universal%20sample%20preparation%20method%20for%20proteome%20analysis&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fnmeth.1322&amp;volume=6&amp;pages=359-362&amp;publication_year=2009&amp;author=Wisniewski%2CJR&amp;author=Zougman%2CA&amp;author=Nagaraj%2CN&amp;author=Mann%2CM\"> Google Scholar </a> </p> </li> <li class=\"c-reading-companion__reference-item\"> <p class=\"c-reading-companion__reference-citation u-font-family-serif\" id=\"rc-ref-CR71\"> Bern, M., Kil, Y. J. &amp; Becker, C. Byonic: advanced peptide and protein identification software. <i> Curr. Protocol. Bioinform. </i> <b> 13 </b> , 13 20 11–13 20 14 (2012). </p> <p class=\"c-reading-companion__reference-links\"> <a href=\"http://scholar.google.com/scholar_lookup?&amp;title=Byonic%3A%20advanced%20peptide%20and%20protein%20identification%20software&amp;journal=Curr.%20Protocol.%20Bioinform.&amp;volume=13&amp;pages=13%2020%2011-13%2020%2014&amp;publication_year=2012&amp;author=Bern%2CM&amp;author=Kil%2CYJ&amp;author=Becker%2CC\"> Google Scholar </a> </p> </li> </ol> </div> </div> </div> </div> </aside> </div> <footer class=\"composite-layer\" itemscope=\"\" itemtype=\"http://schema.org/Periodical\"> <div class=\"u-mt-16 u-mb-16\"> <div class=\"u-container\"> <div class=\"u-display-flex u-flex-wrap u-justify-content-space-between\"> <div class=\"c-meta u-ma-0 u-flex-shrink\"> <p class=\"c-meta__item c-meta__type u-mt-0\"> <span itemprop=\"name\"> Nature Communications </span> ( <i itemprop=\"alternateName\"> Nat Commun </i> ) </p> <p class=\"c-meta__item u-mt-0\"> <abbr title=\"International Standard Serial Number\"> ISSN </abbr> <span itemprop=\"issn\"> 2041-1723 </span> (online) </p> </div> </div> </div> </div> <div class=\"c-footer\"> <div class=\"u-hide-print\"> <h2 class=\"u-visually-hidden\"> nature.com sitemap </h2> <div class=\"c-footer__container\"> <div class=\"c-footer__grid c-footer__group--separator\"> <div class=\"c-footer__group\"> <h3 class=\"c-footer__heading u-mt-0\"> About Nature Portfolio </h3> <ul class=\"c-footer__list\"> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.nature.com/npg_/company_info/index.html\"> About us </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.nature.com/npg_/press_room/press_releases.html\"> Press releases </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://press.nature.com/\"> Press office </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://support.nature.com/support/home\"> Contact us </a> </li> </ul> </div> <div class=\"c-footer__group\"> <h3 class=\"c-footer__heading u-mt-0\"> Discover content </h3> <ul class=\"c-footer__list\"> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.nature.com/siteindex\"> Journals A-Z </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.nature.com/subjects\"> Articles by subject </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.protocols.io/\"> protocols.io </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.natureindex.com/\"> Nature Index </a> </li> </ul> </div> <div class=\"c-footer__group\"> <h3 class=\"c-footer__heading u-mt-0\"> Publishing policies </h3> <ul class=\"c-footer__list\"> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.nature.com/authors/editorial_policies\"> Nature portfolio policies </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.nature.com/nature-research/open-access\"> Open access </a> </li> </ul> </div> <div class=\"c-footer__group\"> <h3 class=\"c-footer__heading u-mt-0\"> Author &amp; Researcher services </h3> <ul class=\"c-footer__list\"> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.nature.com/reprints\"> Reprints &amp; permissions </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.springernature.com/gp/authors/research-data\"> Research data </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://authorservices.springernature.com/language-editing/\"> Language editing </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://authorservices.springernature.com/scientific-editing/\"> Scientific editing </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://masterclasses.nature.com/\"> Nature Masterclasses </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://solutions.springernature.com/\"> Research Solutions </a> </li> </ul> </div> <div class=\"c-footer__group\"> <h3 class=\"c-footer__heading u-mt-0\"> Libraries &amp; institutions </h3> <ul class=\"c-footer__list\"> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.springernature.com/gp/librarians/tools-services\"> Librarian service &amp; tools </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.springernature.com/gp/librarians/manage-your-account/librarianportal\"> Librarian portal </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.nature.com/openresearch/about-open-access/information-for-institutions\"> Open research </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.springernature.com/gp/librarians/recommend-to-your-library\"> Recommend to library </a> </li> </ul> </div> <div class=\"c-footer__group\"> <h3 class=\"c-footer__heading u-mt-0\"> Advertising &amp; partnerships </h3> <ul class=\"c-footer__list\"> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://partnerships.nature.com/product/digital-advertising/\"> Advertising </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://partnerships.nature.com/\"> Partnerships &amp; Services </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://partnerships.nature.com/media-kits/\"> Media kits </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://partnerships.nature.com/product/branded-content-native-advertising/\"> Branded content </a> </li> </ul> </div> <div class=\"c-footer__group\"> <h3 class=\"c-footer__heading u-mt-0\"> Professional development </h3> <ul class=\"c-footer__list\"> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.nature.com/naturecareers/\"> Nature Careers </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://conferences.nature.com\"> Nature <span class=\"u-visually-hidden\"> </span> Conferences </a> </li> </ul> </div> <div class=\"c-footer__group\"> <h3 class=\"c-footer__heading u-mt-0\"> Regional websites </h3> <ul class=\"c-footer__list\"> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.nature.com/natafrica\"> Nature Africa </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"http://www.naturechina.com\"> Nature China </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.nature.com/nindia\"> Nature India </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.nature.com/natitaly\"> Nature Italy </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.natureasia.com/ja-jp\"> Nature Japan </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.nature.com/nmiddleeast\"> Nature Middle East </a> </li> </ul> </div> </div> </div> <div class=\"c-footer__container\"> <ul class=\"c-footer__links\"> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.nature.com/info/privacy\"> Privacy Policy </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.nature.com/info/cookies\"> Use of cookies </a> </li> <li class=\"c-footer__item\"> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.nature.com/info/legal-notice\"> Legal notice </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.nature.com/info/accessibility-statement\"> Accessibility statement </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.nature.com/info/terms-and-conditions\"> Terms &amp; Conditions </a> </li> <li class=\"c-footer__item\"> <a class=\"c-footer__link\" href=\"https://www.springernature.com/ccpa\"> Your US state privacy rights </a> </li> </ul> </div> </div> <div class=\"c-footer__container\"> <a class=\"c-footer__link\" href=\"https://www.springernature.com/\"> </a> <p class=\"c-footer__legal\"> © 2025 Springer Nature Limited </p> </div> </div> <div aria-hidden=\"true\" class=\"u-visually-hidden\"> </div> </footer> <div class=\"c-site-messages message u-hide-print c-site-messages--nature-briefing c-site-messages--nature-briefing-email-variant c-site-messages--nature-briefing-redesign-2020 sans-serif\"> <div class=\"c-site-messages__banner-large\"> <div class=\"c-site-messages__close-container\"> </div> <div class=\"c-site-messages__form-container\"> <div class=\"grid grid-12 last\"> <div class=\"grid grid-4\"> <p class=\"c-site-messages--nature-briefing__strapline extra-tight-line-height\"> Sign up for the <em> Nature Briefing </em> newsletter — what matters in science, free to your inbox daily. </p> </div> <div class=\"grid grid-8 last\"> </div> </div> </div> </div> <div class=\"c-site-messages__banner-small\"> <div class=\"c-site-messages__close-container\"> </div> <div class=\"c-site-messages__content text14\"> <span class=\"c-site-messages--nature-briefing__strapline strong\"> Get the most important science stories of the day, free in your inbox. </span> <a class=\"nature-briefing__link text14 sans-serif\" href=\"https://www.nature.com/briefing/signup/?brieferEntryPoint=MainBriefingBanner\" rel=\"noreferrer noopener\" target=\"_blank\"> Sign up for Nature Briefing </a> </div> </div> </div> <div> </div> </body> </html>\\n2) repos => bigd.big.ac.cn, ega-archive.org, massive.ucsd.edu, www.ebi.ac.uk, www.iprox.cn, proteomecentral.proteomexchange.org, www.proteomexchange.org, pdc.cancer.gov, gdc.cancer.gov, cancerimagingarchive.net, imaging.datacommons.cancer.gov, repository.jpostdb.org, doi.org, geo, dbgap, sra, ArrayExpress.'}]\n",
      "prompt_manager.py - line 96 - DEBUG - Calculating checksum for content: prompt\n",
      "base_parser.py - line 450 - INFO - Prompt ID: gemini-2.0-flash-0-bd1d3ee58ecfcd1eb2f8141131038c766a7b7f76f85a555b7cd94040b39b651c\n",
      "base_parser.py - line 1108 - INFO - Expected string but got list. Converting list to string.\n",
      "base_parser.py - line 1115 - DEBUG - Counting tokens for model: gemini-2.0-flash, prompt length: 411456 char\n",
      "base_parser.py - line 1124 - DEBUG - Rough estimate of token count for Gemini model 'gemini-2.0-flash': 102864\n",
      "base_parser.py - line 469 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages: 102864 tokens\n",
      "base_parser.py - line 600 - INFO - Processing dataset: {'dataset_identifier': 'GSE31210', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE31210'}\n",
      "base_parser.py - line 615 - INFO - Dataset is a dictionary\n",
      "base_parser.py - line 939 - INFO - Resolving data repository for: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE31210\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE31210 == bigd.big.ac.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE31210 == ega-archive.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE31210 == massive.ucsd.edu\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE31210 == www.ebi.ac.uk\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE31210 == www.iprox.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE31210 == proteomecentral.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE31210 == www.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE31210 == pdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE31210 == gdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE31210 == cancerimagingarchive.net\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE31210 == imaging.datacommons.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE31210 == repository.jpostdb.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE31210 == doi.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE31210 == geo\n",
      "base_parser.py - line 969 - INFO - Found partial match for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE31210 in GEO\n",
      "base_parser.py - line 641 - INFO - Extracted dataset: {'dataset_identifier': 'GSE31210', 'data_repository': 'geo'}\n",
      "base_parser.py - line 600 - INFO - Processing dataset: {'dataset_identifier': 'GSE269782', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE269782'}\n",
      "base_parser.py - line 615 - INFO - Dataset is a dictionary\n",
      "base_parser.py - line 939 - INFO - Resolving data repository for: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE269782\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE269782 == bigd.big.ac.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE269782 == ega-archive.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE269782 == massive.ucsd.edu\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE269782 == www.ebi.ac.uk\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE269782 == www.iprox.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE269782 == proteomecentral.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE269782 == www.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE269782 == pdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE269782 == gdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE269782 == cancerimagingarchive.net\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE269782 == imaging.datacommons.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE269782 == repository.jpostdb.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE269782 == doi.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE269782 == geo\n",
      "base_parser.py - line 969 - INFO - Found partial match for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE269782 in GEO\n",
      "base_parser.py - line 641 - INFO - Extracted dataset: {'dataset_identifier': 'GSE269782', 'data_repository': 'geo'}\n",
      "base_parser.py - line 600 - INFO - Processing dataset: {'dataset_identifier': 'GSE60189', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60189'}\n",
      "base_parser.py - line 615 - INFO - Dataset is a dictionary\n",
      "base_parser.py - line 939 - INFO - Resolving data repository for: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60189\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60189 == bigd.big.ac.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60189 == ega-archive.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60189 == massive.ucsd.edu\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60189 == www.ebi.ac.uk\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60189 == www.iprox.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60189 == proteomecentral.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60189 == www.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60189 == pdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60189 == gdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60189 == cancerimagingarchive.net\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60189 == imaging.datacommons.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60189 == repository.jpostdb.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60189 == doi.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60189 == geo\n",
      "base_parser.py - line 969 - INFO - Found partial match for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60189 in GEO\n",
      "base_parser.py - line 641 - INFO - Extracted dataset: {'dataset_identifier': 'GSE60189', 'data_repository': 'geo'}\n",
      "base_parser.py - line 600 - INFO - Processing dataset: {'dataset_identifier': 'GSE59239', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59239'}\n",
      "base_parser.py - line 615 - INFO - Dataset is a dictionary\n",
      "base_parser.py - line 939 - INFO - Resolving data repository for: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59239\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59239 == bigd.big.ac.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59239 == ega-archive.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59239 == massive.ucsd.edu\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59239 == www.ebi.ac.uk\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59239 == www.iprox.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59239 == proteomecentral.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59239 == www.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59239 == pdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59239 == gdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59239 == cancerimagingarchive.net\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59239 == imaging.datacommons.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59239 == repository.jpostdb.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59239 == doi.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59239 == geo\n",
      "base_parser.py - line 969 - INFO - Found partial match for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59239 in GEO\n",
      "base_parser.py - line 641 - INFO - Extracted dataset: {'dataset_identifier': 'GSE59239', 'data_repository': 'geo'}\n",
      "base_parser.py - line 600 - INFO - Processing dataset: {'dataset_identifier': 'GSE122005', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=gse122005'}\n",
      "base_parser.py - line 615 - INFO - Dataset is a dictionary\n",
      "base_parser.py - line 939 - INFO - Resolving data repository for: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=gse122005\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=gse122005 == bigd.big.ac.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=gse122005 == ega-archive.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=gse122005 == massive.ucsd.edu\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=gse122005 == www.ebi.ac.uk\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=gse122005 == www.iprox.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=gse122005 == proteomecentral.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=gse122005 == www.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=gse122005 == pdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=gse122005 == gdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=gse122005 == cancerimagingarchive.net\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=gse122005 == imaging.datacommons.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=gse122005 == repository.jpostdb.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=gse122005 == doi.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=gse122005 == geo\n",
      "base_parser.py - line 969 - INFO - Found partial match for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=gse122005 in GEO\n",
      "base_parser.py - line 641 - INFO - Extracted dataset: {'dataset_identifier': 'GSE122005', 'data_repository': 'geo'}\n",
      "base_parser.py - line 600 - INFO - Processing dataset: {'dataset_identifier': 'GSE38121', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38121'}\n",
      "base_parser.py - line 615 - INFO - Dataset is a dictionary\n",
      "base_parser.py - line 939 - INFO - Resolving data repository for: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38121\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38121 == bigd.big.ac.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38121 == ega-archive.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38121 == massive.ucsd.edu\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38121 == www.ebi.ac.uk\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38121 == www.iprox.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38121 == proteomecentral.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38121 == www.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38121 == pdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38121 == gdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38121 == cancerimagingarchive.net\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38121 == imaging.datacommons.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38121 == repository.jpostdb.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38121 == doi.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38121 == geo\n",
      "base_parser.py - line 969 - INFO - Found partial match for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38121 in GEO\n",
      "base_parser.py - line 641 - INFO - Extracted dataset: {'dataset_identifier': 'GSE38121', 'data_repository': 'geo'}\n",
      "base_parser.py - line 600 - INFO - Processing dataset: {'dataset_identifier': 'GSE71587', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE71587'}\n",
      "base_parser.py - line 615 - INFO - Dataset is a dictionary\n",
      "base_parser.py - line 939 - INFO - Resolving data repository for: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE71587\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE71587 == bigd.big.ac.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE71587 == ega-archive.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE71587 == massive.ucsd.edu\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE71587 == www.ebi.ac.uk\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE71587 == www.iprox.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE71587 == proteomecentral.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE71587 == www.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE71587 == pdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE71587 == gdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE71587 == cancerimagingarchive.net\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE71587 == imaging.datacommons.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE71587 == repository.jpostdb.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE71587 == doi.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE71587 == geo\n",
      "base_parser.py - line 969 - INFO - Found partial match for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE71587 in GEO\n",
      "base_parser.py - line 641 - INFO - Extracted dataset: {'dataset_identifier': 'GSE71587', 'data_repository': 'geo'}\n",
      "base_parser.py - line 600 - INFO - Processing dataset: {'dataset_identifier': 'GSE37699', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE37699'}\n",
      "base_parser.py - line 615 - INFO - Dataset is a dictionary\n",
      "base_parser.py - line 939 - INFO - Resolving data repository for: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE37699\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE37699 == bigd.big.ac.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE37699 == ega-archive.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE37699 == massive.ucsd.edu\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE37699 == www.ebi.ac.uk\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE37699 == www.iprox.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE37699 == proteomecentral.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE37699 == www.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE37699 == pdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE37699 == gdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE37699 == cancerimagingarchive.net\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE37699 == imaging.datacommons.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE37699 == repository.jpostdb.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE37699 == doi.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE37699 == geo\n",
      "base_parser.py - line 969 - INFO - Found partial match for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE37699 in GEO\n",
      "base_parser.py - line 641 - INFO - Extracted dataset: {'dataset_identifier': 'GSE37699', 'data_repository': 'geo'}\n",
      "base_parser.py - line 600 - INFO - Processing dataset: {'dataset_identifier': 'GSE106765', 'repository_reference': 'https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE106765'}\n",
      "base_parser.py - line 615 - INFO - Dataset is a dictionary\n",
      "base_parser.py - line 939 - INFO - Resolving data repository for: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE106765\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE106765 == bigd.big.ac.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE106765 == ega-archive.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE106765 == massive.ucsd.edu\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE106765 == www.ebi.ac.uk\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE106765 == www.iprox.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE106765 == proteomecentral.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE106765 == www.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE106765 == pdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE106765 == gdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE106765 == cancerimagingarchive.net\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE106765 == imaging.datacommons.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE106765 == repository.jpostdb.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE106765 == doi.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE106765 == geo\n",
      "base_parser.py - line 969 - INFO - Found partial match for https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE106765 in GEO\n",
      "base_parser.py - line 641 - INFO - Extracted dataset: {'dataset_identifier': 'GSE106765', 'data_repository': 'geo'}\n",
      "base_parser.py - line 600 - INFO - Processing dataset: {'dataset_identifier': 'PXD051771', 'repository_reference': 'https://www.proteomexchange.org'}\n",
      "base_parser.py - line 615 - INFO - Dataset is a dictionary\n",
      "base_parser.py - line 939 - INFO - Resolving data repository for: https://www.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.proteomexchange.org == bigd.big.ac.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.proteomexchange.org == ega-archive.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.proteomexchange.org == massive.ucsd.edu\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.proteomexchange.org == www.ebi.ac.uk\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.proteomexchange.org == www.iprox.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.proteomexchange.org == proteomecentral.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.proteomexchange.org == www.proteomexchange.org\n",
      "base_parser.py - line 969 - INFO - Found partial match for https://www.proteomexchange.org in ProteomeXchange\n",
      "base_parser.py - line 641 - INFO - Extracted dataset: {'dataset_identifier': 'PXD051771', 'data_repository': 'www.proteomexchange.org'}\n",
      "base_parser.py - line 600 - INFO - Processing dataset: {'dataset_identifier': '1ALU', 'repository_reference': 'https://www.rcsb.org/structure/1ALU'}\n",
      "base_parser.py - line 615 - INFO - Dataset is a dictionary\n",
      "base_parser.py - line 939 - INFO - Resolving data repository for: https://www.rcsb.org/structure/1ALU\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.rcsb.org/structure/1ALU == bigd.big.ac.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.rcsb.org/structure/1ALU == ega-archive.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.rcsb.org/structure/1ALU == massive.ucsd.edu\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.rcsb.org/structure/1ALU == www.ebi.ac.uk\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.rcsb.org/structure/1ALU == www.iprox.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.rcsb.org/structure/1ALU == proteomecentral.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.rcsb.org/structure/1ALU == www.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.rcsb.org/structure/1ALU == pdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.rcsb.org/structure/1ALU == gdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.rcsb.org/structure/1ALU == cancerimagingarchive.net\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.rcsb.org/structure/1ALU == imaging.datacommons.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.rcsb.org/structure/1ALU == repository.jpostdb.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.rcsb.org/structure/1ALU == doi.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.rcsb.org/structure/1ALU == geo\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.rcsb.org/structure/1ALU == dbgap\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.rcsb.org/structure/1ALU == sra\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.rcsb.org/structure/1ALU == ArrayExpress\n",
      "base_parser.py - line 878 - INFO - Extracting repo domain from URL: https://www.rcsb.org/structure/1ALU\n",
      "base_parser.py - line 882 - DEBUG - Repo Domain: www.rcsb.org\n",
      "base_parser.py - line 641 - INFO - Extracted dataset: {'dataset_identifier': '1ALU', 'data_repository': 'www.rcsb.org'}\n",
      "base_parser.py - line 600 - INFO - Processing dataset: {'dataset_identifier': '1P9M', 'repository_reference': 'https://www.rcsb.org/structure/1P9M'}\n",
      "base_parser.py - line 615 - INFO - Dataset is a dictionary\n",
      "base_parser.py - line 939 - INFO - Resolving data repository for: https://www.rcsb.org/structure/1P9M\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.rcsb.org/structure/1P9M == bigd.big.ac.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.rcsb.org/structure/1P9M == ega-archive.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.rcsb.org/structure/1P9M == massive.ucsd.edu\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.rcsb.org/structure/1P9M == www.ebi.ac.uk\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.rcsb.org/structure/1P9M == www.iprox.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.rcsb.org/structure/1P9M == proteomecentral.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.rcsb.org/structure/1P9M == www.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.rcsb.org/structure/1P9M == pdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.rcsb.org/structure/1P9M == gdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.rcsb.org/structure/1P9M == cancerimagingarchive.net\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.rcsb.org/structure/1P9M == imaging.datacommons.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.rcsb.org/structure/1P9M == repository.jpostdb.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.rcsb.org/structure/1P9M == doi.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.rcsb.org/structure/1P9M == geo\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.rcsb.org/structure/1P9M == dbgap\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.rcsb.org/structure/1P9M == sra\n",
      "base_parser.py - line 952 - DEBUG - Checking if https://www.rcsb.org/structure/1P9M == ArrayExpress\n",
      "base_parser.py - line 878 - INFO - Extracting repo domain from URL: https://www.rcsb.org/structure/1P9M\n",
      "base_parser.py - line 882 - DEBUG - Repo Domain: www.rcsb.org\n",
      "base_parser.py - line 641 - INFO - Extracted dataset: {'dataset_identifier': '1P9M', 'data_repository': 'www.rcsb.org'}\n",
      "html_parser.py - line 362 - INFO - Augmented dataset links: [{'dataset_identifier': 'GSE31210', 'data_repository': 'geo'}, {'dataset_identifier': 'GSE269782', 'data_repository': 'geo'}, {'dataset_identifier': 'GSE60189', 'data_repository': 'geo'}, {'dataset_identifier': 'GSE59239', 'data_repository': 'geo'}, {'dataset_identifier': 'GSE122005', 'data_repository': 'geo'}, {'dataset_identifier': 'GSE38121', 'data_repository': 'geo'}, {'dataset_identifier': 'GSE71587', 'data_repository': 'geo'}, {'dataset_identifier': 'GSE37699', 'data_repository': 'geo'}, {'dataset_identifier': 'GSE106765', 'data_repository': 'geo'}, {'dataset_identifier': 'PXD051771', 'data_repository': 'www.proteomexchange.org'}, {'dataset_identifier': '1ALU', 'data_repository': 'www.rcsb.org'}, {'dataset_identifier': '1P9M', 'data_repository': 'www.rcsb.org'}]\n",
      "base_parser.py - line 991 - INFO - Fetching metadata for 12 datasets\n",
      "base_parser.py - line 999 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository'])\n",
      "base_parser.py - line 1010 - INFO - Raw accession ID: GSE31210\n",
      "base_parser.py - line 939 - INFO - Resolving data repository for: geo\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == bigd.big.ac.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == ega-archive.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == massive.ucsd.edu\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == www.ebi.ac.uk\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == www.iprox.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == proteomecentral.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == www.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == pdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == gdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == cancerimagingarchive.net\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == imaging.datacommons.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == repository.jpostdb.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == doi.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == geo\n",
      "base_parser.py - line 957 - INFO - Exact match found for repo: geo\n",
      "base_parser.py - line 921 - INFO - Resolving accession ID for GSE31210 in geo\n",
      "base_parser.py - line 1024 - INFO - Processing dataset 1 with repo: geo and accession_id: GSE31210\n",
      "base_parser.py - line 1025 - DEBUG - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository'])\n",
      "base_parser.py - line 1050 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE31210\n",
      "base_parser.py - line 1057 - INFO - Adding access mode for dataset 1: Complex download\n",
      "base_parser.py - line 999 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository'])\n",
      "base_parser.py - line 1010 - INFO - Raw accession ID: GSE269782\n",
      "base_parser.py - line 939 - INFO - Resolving data repository for: geo\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == bigd.big.ac.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == ega-archive.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == massive.ucsd.edu\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == www.ebi.ac.uk\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == www.iprox.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == proteomecentral.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == www.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == pdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == gdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == cancerimagingarchive.net\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == imaging.datacommons.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == repository.jpostdb.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == doi.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == geo\n",
      "base_parser.py - line 957 - INFO - Exact match found for repo: geo\n",
      "base_parser.py - line 921 - INFO - Resolving accession ID for GSE269782 in geo\n",
      "base_parser.py - line 1024 - INFO - Processing dataset 2 with repo: geo and accession_id: GSE269782\n",
      "base_parser.py - line 1025 - DEBUG - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository'])\n",
      "base_parser.py - line 1050 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE269782\n",
      "base_parser.py - line 1057 - INFO - Adding access mode for dataset 2: Complex download\n",
      "base_parser.py - line 999 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository'])\n",
      "base_parser.py - line 1010 - INFO - Raw accession ID: GSE60189\n",
      "base_parser.py - line 939 - INFO - Resolving data repository for: geo\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == bigd.big.ac.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == ega-archive.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == massive.ucsd.edu\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == www.ebi.ac.uk\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == www.iprox.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == proteomecentral.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == www.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == pdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == gdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == cancerimagingarchive.net\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == imaging.datacommons.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == repository.jpostdb.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == doi.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == geo\n",
      "base_parser.py - line 957 - INFO - Exact match found for repo: geo\n",
      "base_parser.py - line 921 - INFO - Resolving accession ID for GSE60189 in geo\n",
      "base_parser.py - line 1024 - INFO - Processing dataset 3 with repo: geo and accession_id: GSE60189\n",
      "base_parser.py - line 1025 - DEBUG - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository'])\n",
      "base_parser.py - line 1050 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE60189\n",
      "base_parser.py - line 1057 - INFO - Adding access mode for dataset 3: Complex download\n",
      "base_parser.py - line 999 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository'])\n",
      "base_parser.py - line 1010 - INFO - Raw accession ID: GSE59239\n",
      "base_parser.py - line 939 - INFO - Resolving data repository for: geo\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == bigd.big.ac.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == ega-archive.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == massive.ucsd.edu\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == www.ebi.ac.uk\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == www.iprox.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == proteomecentral.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == www.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == pdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == gdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == cancerimagingarchive.net\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == imaging.datacommons.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == repository.jpostdb.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == doi.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == geo\n",
      "base_parser.py - line 957 - INFO - Exact match found for repo: geo\n",
      "base_parser.py - line 921 - INFO - Resolving accession ID for GSE59239 in geo\n",
      "base_parser.py - line 1024 - INFO - Processing dataset 4 with repo: geo and accession_id: GSE59239\n",
      "base_parser.py - line 1025 - DEBUG - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository'])\n",
      "base_parser.py - line 1050 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE59239\n",
      "base_parser.py - line 1057 - INFO - Adding access mode for dataset 4: Complex download\n",
      "base_parser.py - line 999 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository'])\n",
      "base_parser.py - line 1010 - INFO - Raw accession ID: GSE122005\n",
      "base_parser.py - line 939 - INFO - Resolving data repository for: geo\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == bigd.big.ac.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == ega-archive.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == massive.ucsd.edu\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == www.ebi.ac.uk\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == www.iprox.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == proteomecentral.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == www.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == pdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == gdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == cancerimagingarchive.net\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == imaging.datacommons.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == repository.jpostdb.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == doi.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == geo\n",
      "base_parser.py - line 957 - INFO - Exact match found for repo: geo\n",
      "base_parser.py - line 921 - INFO - Resolving accession ID for GSE122005 in geo\n",
      "base_parser.py - line 1024 - INFO - Processing dataset 5 with repo: geo and accession_id: GSE122005\n",
      "base_parser.py - line 1025 - DEBUG - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository'])\n",
      "base_parser.py - line 1050 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE122005\n",
      "base_parser.py - line 1057 - INFO - Adding access mode for dataset 5: Complex download\n",
      "base_parser.py - line 999 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository'])\n",
      "base_parser.py - line 1010 - INFO - Raw accession ID: GSE38121\n",
      "base_parser.py - line 939 - INFO - Resolving data repository for: geo\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == bigd.big.ac.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == ega-archive.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == massive.ucsd.edu\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == www.ebi.ac.uk\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == www.iprox.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == proteomecentral.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == www.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == pdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == gdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == cancerimagingarchive.net\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == imaging.datacommons.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == repository.jpostdb.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == doi.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == geo\n",
      "base_parser.py - line 957 - INFO - Exact match found for repo: geo\n",
      "base_parser.py - line 921 - INFO - Resolving accession ID for GSE38121 in geo\n",
      "base_parser.py - line 1024 - INFO - Processing dataset 6 with repo: geo and accession_id: GSE38121\n",
      "base_parser.py - line 1025 - DEBUG - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository'])\n",
      "base_parser.py - line 1050 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE38121\n",
      "base_parser.py - line 1057 - INFO - Adding access mode for dataset 6: Complex download\n",
      "base_parser.py - line 999 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository'])\n",
      "base_parser.py - line 1010 - INFO - Raw accession ID: GSE71587\n",
      "base_parser.py - line 939 - INFO - Resolving data repository for: geo\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == bigd.big.ac.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == ega-archive.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == massive.ucsd.edu\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == www.ebi.ac.uk\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == www.iprox.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == proteomecentral.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == www.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == pdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == gdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == cancerimagingarchive.net\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == imaging.datacommons.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == repository.jpostdb.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == doi.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == geo\n",
      "base_parser.py - line 957 - INFO - Exact match found for repo: geo\n",
      "base_parser.py - line 921 - INFO - Resolving accession ID for GSE71587 in geo\n",
      "base_parser.py - line 1024 - INFO - Processing dataset 7 with repo: geo and accession_id: GSE71587\n",
      "base_parser.py - line 1025 - DEBUG - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository'])\n",
      "base_parser.py - line 1050 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE71587\n",
      "base_parser.py - line 1057 - INFO - Adding access mode for dataset 7: Complex download\n",
      "base_parser.py - line 999 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository'])\n",
      "base_parser.py - line 1010 - INFO - Raw accession ID: GSE37699\n",
      "base_parser.py - line 939 - INFO - Resolving data repository for: geo\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == bigd.big.ac.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == ega-archive.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == massive.ucsd.edu\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == www.ebi.ac.uk\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == www.iprox.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == proteomecentral.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == www.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == pdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == gdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == cancerimagingarchive.net\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == imaging.datacommons.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == repository.jpostdb.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == doi.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == geo\n",
      "base_parser.py - line 957 - INFO - Exact match found for repo: geo\n",
      "base_parser.py - line 921 - INFO - Resolving accession ID for GSE37699 in geo\n",
      "base_parser.py - line 1024 - INFO - Processing dataset 8 with repo: geo and accession_id: GSE37699\n",
      "base_parser.py - line 1025 - DEBUG - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository'])\n",
      "base_parser.py - line 1050 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE37699\n",
      "base_parser.py - line 1057 - INFO - Adding access mode for dataset 8: Complex download\n",
      "base_parser.py - line 999 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository'])\n",
      "base_parser.py - line 1010 - INFO - Raw accession ID: GSE106765\n",
      "base_parser.py - line 939 - INFO - Resolving data repository for: geo\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == bigd.big.ac.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == ega-archive.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == massive.ucsd.edu\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == www.ebi.ac.uk\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == www.iprox.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == proteomecentral.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == www.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == pdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == gdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == cancerimagingarchive.net\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == imaging.datacommons.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == repository.jpostdb.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == doi.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if geo == geo\n",
      "base_parser.py - line 957 - INFO - Exact match found for repo: geo\n",
      "base_parser.py - line 921 - INFO - Resolving accession ID for GSE106765 in geo\n",
      "base_parser.py - line 1024 - INFO - Processing dataset 9 with repo: geo and accession_id: GSE106765\n",
      "base_parser.py - line 1025 - DEBUG - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository'])\n",
      "base_parser.py - line 1050 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE106765\n",
      "base_parser.py - line 1057 - INFO - Adding access mode for dataset 9: Complex download\n",
      "base_parser.py - line 999 - INFO - Processing dataset 10 with keys: dict_keys(['dataset_identifier', 'data_repository'])\n",
      "base_parser.py - line 1010 - INFO - Raw accession ID: PXD051771\n",
      "base_parser.py - line 939 - INFO - Resolving data repository for: www.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if www.proteomexchange.org == bigd.big.ac.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if www.proteomexchange.org == ega-archive.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if www.proteomexchange.org == massive.ucsd.edu\n",
      "base_parser.py - line 952 - DEBUG - Checking if www.proteomexchange.org == www.ebi.ac.uk\n",
      "base_parser.py - line 952 - DEBUG - Checking if www.proteomexchange.org == www.iprox.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if www.proteomexchange.org == proteomecentral.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if www.proteomexchange.org == www.proteomexchange.org\n",
      "base_parser.py - line 957 - INFO - Exact match found for repo: www.proteomexchange.org\n",
      "base_parser.py - line 921 - INFO - Resolving accession ID for PXD051771 in www.proteomexchange.org\n",
      "base_parser.py - line 1024 - INFO - Processing dataset 10 with repo: www.proteomexchange.org and accession_id: PXD051771\n",
      "base_parser.py - line 1025 - DEBUG - Processing dataset 10 with keys: dict_keys(['dataset_identifier', 'data_repository'])\n",
      "base_parser.py - line 1050 - INFO - Dataset page: https://www.proteomexchange.org/cgi/GetDataset?ID=PXD051771\n",
      "base_parser.py - line 1057 - INFO - Adding access mode for dataset 10: Complex download\n",
      "base_parser.py - line 999 - INFO - Processing dataset 11 with keys: dict_keys(['dataset_identifier', 'data_repository'])\n",
      "base_parser.py - line 1010 - INFO - Raw accession ID: 1ALU\n",
      "base_parser.py - line 939 - INFO - Resolving data repository for: www.rcsb.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if www.rcsb.org == bigd.big.ac.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if www.rcsb.org == ega-archive.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if www.rcsb.org == massive.ucsd.edu\n",
      "base_parser.py - line 952 - DEBUG - Checking if www.rcsb.org == www.ebi.ac.uk\n",
      "base_parser.py - line 952 - DEBUG - Checking if www.rcsb.org == www.iprox.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if www.rcsb.org == proteomecentral.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if www.rcsb.org == www.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if www.rcsb.org == pdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if www.rcsb.org == gdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if www.rcsb.org == cancerimagingarchive.net\n",
      "base_parser.py - line 952 - DEBUG - Checking if www.rcsb.org == imaging.datacommons.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if www.rcsb.org == repository.jpostdb.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if www.rcsb.org == doi.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if www.rcsb.org == geo\n",
      "base_parser.py - line 952 - DEBUG - Checking if www.rcsb.org == dbgap\n",
      "base_parser.py - line 952 - DEBUG - Checking if www.rcsb.org == sra\n",
      "base_parser.py - line 952 - DEBUG - Checking if www.rcsb.org == ArrayExpress\n",
      "base_parser.py - line 878 - INFO - Extracting repo domain from URL: www.rcsb.org\n",
      "base_parser.py - line 890 - ERROR - Error extracting domain from URL: www.rcsb.org\n",
      "base_parser.py - line 921 - INFO - Resolving accession ID for 1ALU in unknown_publisher\n",
      "base_parser.py - line 1024 - INFO - Processing dataset 11 with repo: unknown_publisher and accession_id: 1ALU\n",
      "base_parser.py - line 1025 - DEBUG - Processing dataset 11 with keys: dict_keys(['dataset_identifier', 'data_repository'])\n",
      "base_parser.py - line 1064 - WARNING - Repository unknown_publisher unknown in Ontology. Skipping dataset page 11.\n",
      "base_parser.py - line 999 - INFO - Processing dataset 12 with keys: dict_keys(['dataset_identifier', 'data_repository'])\n",
      "base_parser.py - line 1010 - INFO - Raw accession ID: 1P9M\n",
      "base_parser.py - line 939 - INFO - Resolving data repository for: www.rcsb.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if www.rcsb.org == bigd.big.ac.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if www.rcsb.org == ega-archive.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if www.rcsb.org == massive.ucsd.edu\n",
      "base_parser.py - line 952 - DEBUG - Checking if www.rcsb.org == www.ebi.ac.uk\n",
      "base_parser.py - line 952 - DEBUG - Checking if www.rcsb.org == www.iprox.cn\n",
      "base_parser.py - line 952 - DEBUG - Checking if www.rcsb.org == proteomecentral.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if www.rcsb.org == www.proteomexchange.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if www.rcsb.org == pdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if www.rcsb.org == gdc.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if www.rcsb.org == cancerimagingarchive.net\n",
      "base_parser.py - line 952 - DEBUG - Checking if www.rcsb.org == imaging.datacommons.cancer.gov\n",
      "base_parser.py - line 952 - DEBUG - Checking if www.rcsb.org == repository.jpostdb.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if www.rcsb.org == doi.org\n",
      "base_parser.py - line 952 - DEBUG - Checking if www.rcsb.org == geo\n",
      "base_parser.py - line 952 - DEBUG - Checking if www.rcsb.org == dbgap\n",
      "base_parser.py - line 952 - DEBUG - Checking if www.rcsb.org == sra\n",
      "base_parser.py - line 952 - DEBUG - Checking if www.rcsb.org == ArrayExpress\n",
      "base_parser.py - line 878 - INFO - Extracting repo domain from URL: www.rcsb.org\n",
      "base_parser.py - line 890 - ERROR - Error extracting domain from URL: www.rcsb.org\n",
      "base_parser.py - line 921 - INFO - Resolving accession ID for 1P9M in unknown_publisher\n",
      "base_parser.py - line 1024 - INFO - Processing dataset 12 with repo: unknown_publisher and accession_id: 1P9M\n",
      "base_parser.py - line 1025 - DEBUG - Processing dataset 12 with keys: dict_keys(['dataset_identifier', 'data_repository'])\n",
      "base_parser.py - line 1064 - WARNING - Repository unknown_publisher unknown in Ontology. Skipping dataset page 12.\n",
      "base_parser.py - line 1067 - INFO - Updated datasets len: 12\n",
      "html_parser.py - line 397 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 16, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_class', 'a_attr_data-track', 'a_attr_data-track-action',\n",
      "       'a_attr_data-test', 'a_attr_data-track-label', 'a_attr_href',\n",
      "       'a_attr_data-supp-info-image'],\n",
      "      dtype='object')\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(nan)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(nan)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(nan)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(nan)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(nan)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(nan)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(nan)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(nan)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(nan)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(nan)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(nan)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(nan)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(None)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(None)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(None)\n",
      "base_parser.py - line 266 - DEBUG - Function_call: extract_file_extension(None)\n",
      "html_retriever.py - line 229 - INFO - Extracting publication title from HTML\n",
      "html_retriever.py - line 239 - WARNING - No publication title found in the HTML data.\n"
     ]
    }
   ],
   "execution_count": 14
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-07-07T15:30:47.712906Z",
     "start_time": "2025-07-07T15:30:47.686153Z"
    }
   },
   "cell_type": "code",
   "source": "results_FDR",
   "id": "e77187b1bdda3ecc",
   "outputs": [
    {
     "data": {
      "text/plain": [
       "   dataset_identifier          data_repository  \\\n",
       "0            GSE31210                      geo   \n",
       "1           GSE269782                      geo   \n",
       "2            GSE60189                      geo   \n",
       "3            GSE59239                      geo   \n",
       "4           GSE122005                      geo   \n",
       "5            GSE38121                      geo   \n",
       "6            GSE71587                      geo   \n",
       "7            GSE37699                      geo   \n",
       "8           GSE106765                      geo   \n",
       "9           PXD051771  www.proteomexchange.org   \n",
       "10               1ALU             www.rcsb.org   \n",
       "11               1P9M             www.rcsb.org   \n",
       "0                 NaN                      NaN   \n",
       "\n",
       "                                      dataset_webpage       access_mode  \\\n",
       "0   https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...  Complex download   \n",
       "1   https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...  Complex download   \n",
       "2   https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...  Complex download   \n",
       "3   https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...  Complex download   \n",
       "4   https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...  Complex download   \n",
       "5   https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...  Complex download   \n",
       "6   https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...  Complex download   \n",
       "7   https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...  Complex download   \n",
       "8   https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...  Complex download   \n",
       "9   https://www.proteomexchange.org/cgi/GetDataset...  Complex download   \n",
       "10                                                NaN               NaN   \n",
       "11                                                NaN               NaN   \n",
       "0                                                 NaN               NaN   \n",
       "\n",
       "                                                 link  \\\n",
       "0                                                 NaN   \n",
       "1                                                 NaN   \n",
       "2                                                 NaN   \n",
       "3                                                 NaN   \n",
       "4                                                 NaN   \n",
       "5                                                 NaN   \n",
       "6                                                 NaN   \n",
       "7                                                 NaN   \n",
       "8                                                 NaN   \n",
       "9                                                 NaN   \n",
       "10                                                NaN   \n",
       "11                                                NaN   \n",
       "0   https://static-content.springer.com/esm/art%3A...   \n",
       "\n",
       "                        title file_info description source_section  \\\n",
       "0                         NaN       NaN         NaN            NaN   \n",
       "1                         NaN       NaN         NaN            NaN   \n",
       "2                         NaN       NaN         NaN            NaN   \n",
       "3                         NaN       NaN         NaN            NaN   \n",
       "4                         NaN       NaN         NaN            NaN   \n",
       "5                         NaN       NaN         NaN            NaN   \n",
       "6                         NaN       NaN         NaN            NaN   \n",
       "7                         NaN       NaN         NaN            NaN   \n",
       "8                         NaN       NaN         NaN            NaN   \n",
       "9                         NaN       NaN         NaN            NaN   \n",
       "10                        NaN       NaN         NaN            NaN   \n",
       "11                        NaN       NaN         NaN            NaN   \n",
       "0   Supplementary Information       n/a         n/a         MOESM1   \n",
       "\n",
       "                    section_class  ... file_extension a_attr_class  \\\n",
       "0                             NaN  ...           None          NaN   \n",
       "1                             NaN  ...           None          NaN   \n",
       "2                             NaN  ...           None          NaN   \n",
       "3                             NaN  ...           None          NaN   \n",
       "4                             NaN  ...           None          NaN   \n",
       "5                             NaN  ...           None          NaN   \n",
       "6                             NaN  ...           None          NaN   \n",
       "7                             NaN  ...           None          NaN   \n",
       "8                             NaN  ...           None          NaN   \n",
       "9                             NaN  ...           None          NaN   \n",
       "10                            NaN  ...           None          NaN   \n",
       "11                            NaN  ...           None          NaN   \n",
       "0   c-article-supplementary__item  ...           None   print-link   \n",
       "\n",
       "   a_attr_data-track a_attr_data-track-action a_attr_data-test  \\\n",
       "0                NaN                      NaN              NaN   \n",
       "1                NaN                      NaN              NaN   \n",
       "2                NaN                      NaN              NaN   \n",
       "3                NaN                      NaN              NaN   \n",
       "4                NaN                      NaN              NaN   \n",
       "5                NaN                      NaN              NaN   \n",
       "6                NaN                      NaN              NaN   \n",
       "7                NaN                      NaN              NaN   \n",
       "8                NaN                      NaN              NaN   \n",
       "9                NaN                      NaN              NaN   \n",
       "10               NaN                      NaN              NaN   \n",
       "11               NaN                      NaN              NaN   \n",
       "0              click  view supplementary info   supp-info-link   \n",
       "\n",
       "      a_attr_data-track-label  \\\n",
       "0                         NaN   \n",
       "1                         NaN   \n",
       "2                         NaN   \n",
       "3                         NaN   \n",
       "4                         NaN   \n",
       "5                         NaN   \n",
       "6                         NaN   \n",
       "7                         NaN   \n",
       "8                         NaN   \n",
       "9                         NaN   \n",
       "10                        NaN   \n",
       "11                        NaN   \n",
       "0   supplementary information   \n",
       "\n",
       "                                          a_attr_href  \\\n",
       "0                                                 NaN   \n",
       "1                                                 NaN   \n",
       "2                                                 NaN   \n",
       "3                                                 NaN   \n",
       "4                                                 NaN   \n",
       "5                                                 NaN   \n",
       "6                                                 NaN   \n",
       "7                                                 NaN   \n",
       "8                                                 NaN   \n",
       "9                                                 NaN   \n",
       "10                                                NaN   \n",
       "11                                                NaN   \n",
       "0   https://static-content.springer.com/esm/art%3A...   \n",
       "\n",
       "   a_attr_data-supp-info-image  \\\n",
       "0                          NaN   \n",
       "1                          NaN   \n",
       "2                          NaN   \n",
       "3                          NaN   \n",
       "4                          NaN   \n",
       "5                          NaN   \n",
       "6                          NaN   \n",
       "7                          NaN   \n",
       "8                          NaN   \n",
       "9                          NaN   \n",
       "10                         NaN   \n",
       "11                         NaN   \n",
       "0                                \n",
       "\n",
       "                                           source_url   article_title  \n",
       "0   https://www.nature.com/articles/s41467-024-518...  No title found  \n",
       "1   https://www.nature.com/articles/s41467-024-518...  No title found  \n",
       "2   https://www.nature.com/articles/s41467-024-518...  No title found  \n",
       "3   https://www.nature.com/articles/s41467-024-518...  No title found  \n",
       "4   https://www.nature.com/articles/s41467-024-518...  No title found  \n",
       "5   https://www.nature.com/articles/s41467-024-518...  No title found  \n",
       "6   https://www.nature.com/articles/s41467-024-518...  No title found  \n",
       "7   https://www.nature.com/articles/s41467-024-518...  No title found  \n",
       "8   https://www.nature.com/articles/s41467-024-518...  No title found  \n",
       "9   https://www.nature.com/articles/s41467-024-518...  No title found  \n",
       "10  https://www.nature.com/articles/s41467-024-518...  No title found  \n",
       "11  https://www.nature.com/articles/s41467-024-518...  No title found  \n",
       "0   https://www.nature.com/articles/s41467-024-518...  No title found  \n",
       "\n",
       "[13 rows x 21 columns]"
      ],
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>dataset_identifier</th>\n",
       "      <th>data_repository</th>\n",
       "      <th>dataset_webpage</th>\n",
       "      <th>access_mode</th>\n",
       "      <th>link</th>\n",
       "      <th>title</th>\n",
       "      <th>file_info</th>\n",
       "      <th>description</th>\n",
       "      <th>source_section</th>\n",
       "      <th>section_class</th>\n",
       "      <th>...</th>\n",
       "      <th>file_extension</th>\n",
       "      <th>a_attr_class</th>\n",
       "      <th>a_attr_data-track</th>\n",
       "      <th>a_attr_data-track-action</th>\n",
       "      <th>a_attr_data-test</th>\n",
       "      <th>a_attr_data-track-label</th>\n",
       "      <th>a_attr_href</th>\n",
       "      <th>a_attr_data-supp-info-image</th>\n",
       "      <th>source_url</th>\n",
       "      <th>article_title</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>GSE31210</td>\n",
       "      <td>geo</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...</td>\n",
       "      <td>Complex download</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.nature.com/articles/s41467-024-518...</td>\n",
       "      <td>No title found</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>GSE269782</td>\n",
       "      <td>geo</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...</td>\n",
       "      <td>Complex download</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.nature.com/articles/s41467-024-518...</td>\n",
       "      <td>No title found</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>GSE60189</td>\n",
       "      <td>geo</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...</td>\n",
       "      <td>Complex download</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.nature.com/articles/s41467-024-518...</td>\n",
       "      <td>No title found</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>GSE59239</td>\n",
       "      <td>geo</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...</td>\n",
       "      <td>Complex download</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.nature.com/articles/s41467-024-518...</td>\n",
       "      <td>No title found</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>GSE122005</td>\n",
       "      <td>geo</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...</td>\n",
       "      <td>Complex download</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.nature.com/articles/s41467-024-518...</td>\n",
       "      <td>No title found</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>GSE38121</td>\n",
       "      <td>geo</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...</td>\n",
       "      <td>Complex download</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.nature.com/articles/s41467-024-518...</td>\n",
       "      <td>No title found</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>GSE71587</td>\n",
       "      <td>geo</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...</td>\n",
       "      <td>Complex download</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.nature.com/articles/s41467-024-518...</td>\n",
       "      <td>No title found</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>GSE37699</td>\n",
       "      <td>geo</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...</td>\n",
       "      <td>Complex download</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.nature.com/articles/s41467-024-518...</td>\n",
       "      <td>No title found</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>GSE106765</td>\n",
       "      <td>geo</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...</td>\n",
       "      <td>Complex download</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.nature.com/articles/s41467-024-518...</td>\n",
       "      <td>No title found</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>PXD051771</td>\n",
       "      <td>www.proteomexchange.org</td>\n",
       "      <td>https://www.proteomexchange.org/cgi/GetDataset...</td>\n",
       "      <td>Complex download</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.nature.com/articles/s41467-024-518...</td>\n",
       "      <td>No title found</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>1ALU</td>\n",
       "      <td>www.rcsb.org</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.nature.com/articles/s41467-024-518...</td>\n",
       "      <td>No title found</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>1P9M</td>\n",
       "      <td>www.rcsb.org</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://www.nature.com/articles/s41467-024-518...</td>\n",
       "      <td>No title found</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>NaN</td>\n",
       "      <td>https://static-content.springer.com/esm/art%3A...</td>\n",
       "      <td>Supplementary Information</td>\n",
       "      <td>n/a</td>\n",
       "      <td>n/a</td>\n",
       "      <td>MOESM1</td>\n",
       "      <td>c-article-supplementary__item</td>\n",
       "      <td>...</td>\n",
       "      <td>None</td>\n",
       "      <td>print-link</td>\n",
       "      <td>click</td>\n",
       "      <td>view supplementary info</td>\n",
       "      <td>supp-info-link</td>\n",
       "      <td>supplementary information</td>\n",
       "      <td>https://static-content.springer.com/esm/art%3A...</td>\n",
       "      <td></td>\n",
       "      <td>https://www.nature.com/articles/s41467-024-518...</td>\n",
       "      <td>No title found</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>13 rows × 21 columns</p>\n",
       "</div>"
      ]
     },
     "execution_count": 15,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "execution_count": 15
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-07-07T15:30:47.719105Z",
     "start_time": "2025-07-07T15:30:47.714810Z"
    }
   },
   "cell_type": "code",
   "source": "parsed_data = {\"results_RTR\" : results_RTR, \"results_FDR\": results_FDR}",
   "id": "5e47d903a8f2eb07",
   "outputs": [],
   "execution_count": 16
  },
  {
   "metadata": {},
   "cell_type": "markdown",
   "source": [
    "## Step III: Classify data\n",
    "\n",
    "For now, we are only considering raw data files, i.e. those objects that can be accessed from a repository with an accession code. However, the classifier can be extended to include other types of objects, such as supplementary materials."
   ],
   "id": "686d613afae5e90c"
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-07-07T15:30:47.737216Z",
     "start_time": "2025-07-07T15:30:47.721757Z"
    }
   },
   "cell_type": "code",
   "source": [
    "print(f\"Fetched from {len(parsed_data)} publications.\")\n",
    "raw_data_files = data_gatherer.classifier.get_raw_data_files(parsed_data)\n",
    "print(f\"Fetched {len(raw_data_files)} raw data files:\")"
   ],
   "id": "d8eb143abe00f7db",
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "classifier.py - line 98 - DEBUG - Input type: <class 'dict'>\n",
      "classifier.py - line 106 - INFO - Processing DataFrame for URL: results_RTR\n",
      "classifier.py - line 106 - INFO - Processing DataFrame for URL: results_FDR\n",
      "classifier.py - line 112 - DEBUG - Resources type: <class 'pandas.core.frame.DataFrame'>\n",
      "classifier.py - line 113 - DEBUG - Resources columns: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_class', 'a_attr_data-track', 'a_attr_data-track-action',\n",
      "       'a_attr_data-test', 'a_attr_data-track-label', 'a_attr_href',\n",
      "       'a_attr_data-supp-info-image', 'source_url', 'article_title',\n",
      "       'publication_url'],\n",
      "      dtype='object')\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Fetched from 2 publications.\n",
      "Fetched 22 raw data files:\n"
     ]
    }
   ],
   "execution_count": 17
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-07-07T15:30:47.749861Z",
     "start_time": "2025-07-07T15:30:47.738942Z"
    }
   },
   "cell_type": "code",
   "source": "raw_data_files",
   "id": "e18c2df3980ef877",
   "outputs": [
    {
     "data": {
      "text/plain": [
       "   publication_url dataset_identifier                     data_repository  \\\n",
       "0      results_RTR          GSE269782  NCBI Gene Expression Omnibus (GEO)   \n",
       "1      results_RTR           GSE31210                            NCBI GEO   \n",
       "2      results_RTR          GSE106765                            NCBI GEO   \n",
       "3      results_RTR           GSE60189                            NCBI GEO   \n",
       "4      results_RTR           GSE59239                            NCBI GEO   \n",
       "5      results_RTR          GSE122005                            NCBI GEO   \n",
       "6      results_RTR           GSE38121                            NCBI GEO   \n",
       "7      results_RTR           GSE71587                            NCBI GEO   \n",
       "8      results_RTR           GSE37699                            NCBI GEO   \n",
       "9      results_RTR          PXD051771                     ProteomeXchange   \n",
       "11     results_FDR           GSE31210                                 geo   \n",
       "12     results_FDR          GSE269782                                 geo   \n",
       "13     results_FDR           GSE60189                                 geo   \n",
       "14     results_FDR           GSE59239                                 geo   \n",
       "15     results_FDR          GSE122005                                 geo   \n",
       "16     results_FDR           GSE38121                                 geo   \n",
       "17     results_FDR           GSE71587                                 geo   \n",
       "18     results_FDR           GSE37699                                 geo   \n",
       "19     results_FDR          GSE106765                                 geo   \n",
       "20     results_FDR          PXD051771             www.proteomexchange.org   \n",
       "21     results_FDR               1ALU                        www.rcsb.org   \n",
       "22     results_FDR               1P9M                        www.rcsb.org   \n",
       "\n",
       "                                      dataset_webpage  \n",
       "0   https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...  \n",
       "1   https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...  \n",
       "2   https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...  \n",
       "3   https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...  \n",
       "4   https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...  \n",
       "5   https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...  \n",
       "6   https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...  \n",
       "7   https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...  \n",
       "8   https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...  \n",
       "9   https://www.proteomexchange.org/cgi/GetDataset...  \n",
       "11  https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...  \n",
       "12  https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...  \n",
       "13  https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...  \n",
       "14  https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...  \n",
       "15  https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...  \n",
       "16  https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...  \n",
       "17  https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...  \n",
       "18  https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...  \n",
       "19  https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...  \n",
       "20  https://www.proteomexchange.org/cgi/GetDataset...  \n",
       "21                                                NaN  \n",
       "22                                                NaN  "
      ],
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>publication_url</th>\n",
       "      <th>dataset_identifier</th>\n",
       "      <th>data_repository</th>\n",
       "      <th>dataset_webpage</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>results_RTR</td>\n",
       "      <td>GSE269782</td>\n",
       "      <td>NCBI Gene Expression Omnibus (GEO)</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>results_RTR</td>\n",
       "      <td>GSE31210</td>\n",
       "      <td>NCBI GEO</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>results_RTR</td>\n",
       "      <td>GSE106765</td>\n",
       "      <td>NCBI GEO</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>results_RTR</td>\n",
       "      <td>GSE60189</td>\n",
       "      <td>NCBI GEO</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>results_RTR</td>\n",
       "      <td>GSE59239</td>\n",
       "      <td>NCBI GEO</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>results_RTR</td>\n",
       "      <td>GSE122005</td>\n",
       "      <td>NCBI GEO</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>results_RTR</td>\n",
       "      <td>GSE38121</td>\n",
       "      <td>NCBI GEO</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>results_RTR</td>\n",
       "      <td>GSE71587</td>\n",
       "      <td>NCBI GEO</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>results_RTR</td>\n",
       "      <td>GSE37699</td>\n",
       "      <td>NCBI GEO</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>results_RTR</td>\n",
       "      <td>PXD051771</td>\n",
       "      <td>ProteomeXchange</td>\n",
       "      <td>https://www.proteomexchange.org/cgi/GetDataset...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>results_FDR</td>\n",
       "      <td>GSE31210</td>\n",
       "      <td>geo</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>results_FDR</td>\n",
       "      <td>GSE269782</td>\n",
       "      <td>geo</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>results_FDR</td>\n",
       "      <td>GSE60189</td>\n",
       "      <td>geo</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>results_FDR</td>\n",
       "      <td>GSE59239</td>\n",
       "      <td>geo</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>results_FDR</td>\n",
       "      <td>GSE122005</td>\n",
       "      <td>geo</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>results_FDR</td>\n",
       "      <td>GSE38121</td>\n",
       "      <td>geo</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>results_FDR</td>\n",
       "      <td>GSE71587</td>\n",
       "      <td>geo</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>results_FDR</td>\n",
       "      <td>GSE37699</td>\n",
       "      <td>geo</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>results_FDR</td>\n",
       "      <td>GSE106765</td>\n",
       "      <td>geo</td>\n",
       "      <td>https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>results_FDR</td>\n",
       "      <td>PXD051771</td>\n",
       "      <td>www.proteomexchange.org</td>\n",
       "      <td>https://www.proteomexchange.org/cgi/GetDataset...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>results_FDR</td>\n",
       "      <td>1ALU</td>\n",
       "      <td>www.rcsb.org</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>results_FDR</td>\n",
       "      <td>1P9M</td>\n",
       "      <td>www.rcsb.org</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ]
     },
     "execution_count": 18,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "execution_count": 18
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-07-07T15:30:47.754040Z",
     "start_time": "2025-07-07T15:30:47.751282Z"
    }
   },
   "cell_type": "code",
   "source": "",
   "id": "8d449d0536dc734b",
   "outputs": [],
   "execution_count": 18
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 2
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython2",
   "version": "2.7.6"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
