{
 "cells": [
  {
   "cell_type": "code",
   "id": "initial_id",
   "metadata": {
    "collapsed": true,
    "ExecuteTime": {
     "end_time": "2025-07-10T15:28:05.996589Z",
     "start_time": "2025-07-10T15:27:54.400133Z"
    }
   },
   "source": "from data_gatherer.orchestrator import Orchestrator",
   "outputs": [],
   "execution_count": 1
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-07-10T15:28:06.007725Z",
     "start_time": "2025-07-10T15:28:05.999131Z"
    }
   },
   "cell_type": "code",
   "source": "data_gatherer = Orchestrator(llm_name=\"gemini-2.0-flash\", log_level=\"INFO\")",
   "id": "34f3046b7ea80e86",
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001B[97morchestrator.py - line 82 - INFO - Data_Gatherer Orchestrator initialized. Extraction Model: gemini-2.0-flash\u001B[0m\n"
     ]
    }
   ],
   "execution_count": 2
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-07-10T15:28:06.014645Z",
     "start_time": "2025-07-10T15:28:06.009345Z"
    }
   },
   "cell_type": "code",
   "source": "data_gatherer.setup_data_fetcher()",
   "id": "99a91efa8baec42",
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001B[97morchestrator.py - line 219 - INFO - Setting up data fetcher...\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 23 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97morchestrator.py - line 249 - INFO - Data fetcher setup completed.\u001B[0m\n"
     ]
    }
   ],
   "execution_count": 3
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-07-10T15:28:06.019116Z",
     "start_time": "2025-07-10T15:28:06.016064Z"
    }
   },
   "cell_type": "code",
   "source": "input_articles = [\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8628860\",\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11129317\",\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11425778\",\"https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5313262\"]",
   "id": "b50f0289f8a85d1d",
   "outputs": [],
   "execution_count": 4
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-07-10T15:29:05.687763Z",
     "start_time": "2025-07-10T15:28:08.544402Z"
    }
   },
   "cell_type": "code",
   "source": "parsed_data = data_gatherer.process_articles(input_articles, semantic_retrieval=True)",
   "id": "7c532d5582f6d1a7",
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001B[97morchestrator.py - line 477 - INFO - 0th function call: self.process_url(https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8628860)\u001B[0m\n",
      "\u001B[97morchestrator.py - line 307 - INFO - Processing URL: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8628860\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 172 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 23 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97morchestrator.py - line 314 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 498 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC8628860&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97morchestrator.py - line 335 - INFO - Fallback to HTML data fetcher for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8628860.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 180 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 181 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 182 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 184 - INFO - Initializing new selenium driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1a5d4e950>.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 23 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 353 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 288 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97morchestrator.py - line 351 - INFO - Raw HTML data fetched from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8628860 is ready for parsing.\u001B[0m\n",
      "\u001B[97morchestrator.py - line 365 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97morchestrator.py - line 389 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 181 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 82 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 345 - INFO - Function call: parse_data(api_data(html_str, PMC, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8628860, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 426 - INFO - Function_call: extract_href_from_html_supplementary_material(tree, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8628860)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 490 - INFO - Extracted 58 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 372 - INFO - Chunking the HTML content for the parsing step.\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 171 - INFO - Extracting data availability elements from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 182 - INFO - Using selector: section.data-availability-statement\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 182 - INFO - Using selector: section.associated-data\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 182 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Data Availability')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 182 - INFO - Using selector: h3.pmc_sec_title:-soup-contains('Data Availability')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 182 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Associated Data')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 182 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Footnotes')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 220 - INFO - Found 5 data availability elements from HTML.\u001B[0m\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "Batches:   0%|          | 0/5 [00:00<?, ?it/s]"
      ],
      "application/vnd.jupyter.widget-view+json": {
       "version_major": 2,
       "version_minor": 0,
       "model_id": "e4a1f18377b542f8a12504a1251e914d"
      }
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001B[97membeddings_retriever.py - line 51 - INFO - Searching for top-5 passages similar to the query by embeddings.\u001B[0m\n",
      "\u001B[97membeddings_retriever.py - line 33 - INFO - Computing L2 distances using numpy.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 432 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 32 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1193 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 440 - INFO - Content length: 82018\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1193 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 451 - INFO - Prompt messages total length: 20847 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 457 - INFO - Prompt ID: gemini-2.0-flash-0-567cf3d310ff974474d13cbc855a4da99479e24fec3fe96753ede8c704421ea2\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1193 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 476 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages: 20847 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 547 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-e177046a-47f9-4bcf-9f72-6eebf9032dde\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"scRNA-seq and MIBI data\\\",\\n        \\\"data_repository\\\": \\\"HTAN Data Portal\\\",\\n        \\\"dataset_webpage\\\": \\\"https://data.humantumoratlas.org/\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"MIBI-TOF data\\\",\\n        \\\"data_repository\\\": \\\"mskcc.ionpath.com\\\",\\n        \\\"dataset_webpage\\\": \\\"https://mskcc.ionpath.com/tracker\\\"\\n    }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"scRNA-seq and MIBI data\\\",\\n        \\\"data_repository\\\": \\\"HTAN Data Portal\\\",\\n        \\\"dataset_webpage\\\": \\\"https://data.humantumoratlas.org/\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"MIBI-TOF data\\\",\\n        \\\"data_repository\\\": \\\"mskcc.ionpath.com\\\",\\n        \\\"dataset_webpage\\\": \\\"https://mskcc.ionpath.com/tracker\\\"\\n    }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1752012592,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 31456,\n",
      "        \"completion_tokens\": 121,\n",
      "        \"total_tokens\": 31577,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 607 - INFO - Processing dataset: {'dataset_identifier': 'scRNA-seq and MIBI data', 'data_repository': 'HTAN Data Portal', 'dataset_webpage': 'https://data.humantumoratlas.org/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 622 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 954 - INFO - Validating accession ID: scRNA-seq and MIBI data\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Accession ID scRNA-seq and MIBI data is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Resolving data repository for candidate: HTAN Data Portal, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 910 - INFO - Extracting repo domain from URL: https://data.humantumoratlas.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1059 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 972 - INFO - Validating Dataset Page: https://data.humantumoratlas.org/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 987 - INFO - Repository data.humantumoratlas.org not found in ontology\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 668 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'data.humantumoratlas.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 607 - INFO - Processing dataset: {'dataset_identifier': 'MIBI-TOF data', 'data_repository': 'mskcc.ionpath.com', 'dataset_webpage': 'https://mskcc.ionpath.com/tracker'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 622 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 954 - INFO - Validating accession ID: MIBI-TOF data\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Accession ID MIBI-TOF data is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Resolving data repository for candidate: mskcc.ionpath.com, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 910 - INFO - Extracting repo domain from URL: https://mskcc.ionpath.com/tracker\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1059 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 972 - INFO - Validating Dataset Page: https://mskcc.ionpath.com/tracker\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 987 - INFO - Repository mskcc.ionpath.com not found in ontology\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 668 - INFO - Extracted dataset: {'dataset_identifier': 'n/a', 'data_repository': 'mskcc.ionpath.com', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1075 - INFO - Fetching metadata for 2 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1083 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1091 - INFO - Skipping dataset 1: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1083 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1091 - INFO - Skipping dataset 2: no dataset_identifier for item\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1152 - INFO - Updated datasets len: 2\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 396 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 60, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage', 'link',\n",
      "       'title', 'file_info', 'description', 'source_section', 'section_class',\n",
      "       'download_link', 'file_extension', 'a_attr_href', 'a_attr_class',\n",
      "       'a_attr_data-ga-action', 'a_attr_target', 'a_attr_rel'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 229 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 239 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 500 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97morchestrator.py - line 399 - INFO - Parsed data extraction completed. Elements collected: 29\u001B[0m\n",
      "\u001B[97morchestrator.py - line 405 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97morchestrator.py - line 415 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97morchestrator.py - line 487 - INFO - \n",
      "Progress: 1/4 (25.00%) | Elapsed: 00:00:20 | ETA: 00:01:00\n",
      "\u001B[0m\n",
      "\u001B[97morchestrator.py - line 477 - INFO - 1th function call: self.process_url(https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11129317)\u001B[0m\n",
      "\u001B[97morchestrator.py - line 307 - INFO - Processing URL: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11129317\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 172 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 23 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97morchestrator.py - line 314 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 498 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC11129317&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\" sec-type=\"data-availability\" id=\"sec4.2.3\"><title>Data and code availability</title><p id=\"p0195\">\n",
      "<list list-type=\"simple\" id=\"ulist0015\"><list-item id=\"u0030\"><label>•</label><p id=\"p0200\">In this study, a comprehensive dataset was assembled, comprising in total of 911 tumor tissues and 558 adjacent normal tissues across multiple cancer types. The Dataset was assembled from PDC commons (<ext-link xmlns:xlink=\"http://www.w3.org/1999/xlink\" ext-link-type=\"uri\" xlink:href=\"https://pdc.cancer.gov/pdc/\" id=\"intref0120\">https://pdc.cancer.gov/pdc/</ext-link>), and the PDC IDs are reported in the <xref rid=\"sec4.1\" ref-type=\"sec\">key resources table</xref> (above). The Mass spectrometry proteomics data have been deposited at Pride, and original western blot images have been deposited in Mendeley, all will be publicly available as of the date of publication. The DOI is listed in the <xref rid=\"sec4.1\" ref-type=\"sec\">key resources table</xref>.</p></list-item><list-item id=\"u0035\"><label>•</label><p id=\"p0205\">All original codes have been deposited at Mendeley Data and are publicly available as of the date of publication. DOIs are listed in the <xref rid=\"sec4.1\" ref-type=\"sec\">key resources table</xref>.</p></list-item><list-item id=\"u0040\"><label>•</label><p id=\"p0210\">Any additional information required to reanalyze the data reported in the paper is available from the <xref rid=\"sec4.2.1\" ref-type=\"sec\">lead contact</xref> upon request.</p></list-item></list>\n",
      "</p></sec>\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for supplementary_data_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 457 - INFO - ----Found section: <sec xmlns:mml=\"http://www.w3.org/1998/Math/MathML\" xmlns:ali=\"http://www.niso.org/schemas/ali/1.0/\" id=\"app2\" sec-type=\"supplementary-material\"><title>Supplemental information</title><p id=\"p0425\">\n",
      "<supplementary-material content-type=\"local-data\" id=\"mmc1\" position=\"float\" orientation=\"portrait\"><caption><title>Document S1. Figures S1–S6</title></caption><media xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"mmc1.pdf\" position=\"float\" orientation=\"portrait\"/></supplementary-material>\n",
      "<supplementary-material content-type=\"local-data\" id=\"mmc2\" position=\"float\" orientation=\"portrait\"><caption><title>File S1. Detailed Philosopher parameters for substitutant library scanning</title></caption><media xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"mmc2.zip\" position=\"float\" orientation=\"portrait\"/></supplementary-material>\n",
      "<supplementary-material content-type=\"local-data\" id=\"mmc3\" position=\"float\" orientation=\"portrait\"><caption><title>Table S1. Identified arginine to all other amino acid substitution peptides in different cancers, related to Figure 1</title></caption><media xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"mmc3.xlsx\" position=\"float\" orientation=\"portrait\"/></supplementary-material>\n",
      "<supplementary-material content-type=\"local-data\" id=\"mmc4\" position=\"float\" orientation=\"portrait\"><caption><title>Table S2. Arginine-to-cysteine substitutants with codon-specific analysis in LSCC, related to Figure 3</title></caption><media xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"mmc4.xlsx\" position=\"float\" orientation=\"portrait\"/></supplementary-material>\n",
      "<supplementary-material content-type=\"local-data\" id=\"mmc5\" position=\"float\" orientation=\"portrait\"><caption><title>Table S3. Arginine substitutants codon-specific analysis in different cancers, related to Figure 1</title></caption><media xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"mmc5.xlsx\" position=\"float\" orientation=\"portrait\"/></supplementary-material>\n",
      "<supplementary-material content-type=\"local-data\" id=\"mmc6\" position=\"float\" orientation=\"portrait\"><caption><title>Table S4. Proteomic analysis of MDA-MB-231 cells treated with arginine depletion, related to Figure 4</title></caption><media xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"mmc6.xlsx\" position=\"float\" orientation=\"portrait\"/></supplementary-material>\n",
      "<supplementary-material content-type=\"local-data\" id=\"mmc7\" position=\"float\" orientation=\"portrait\"><caption><title>Table S5. Ontology-enrichment analysis of arginine-to-cysteine substitutants, related to Figure 5</title></caption><media xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"mmc7.xlsx\" position=\"float\" orientation=\"portrait\"/></supplementary-material>\n",
      "<supplementary-material content-type=\"local-data\" id=\"mmc8\" position=\"float\" orientation=\"portrait\"><caption><title>Table S6. Proteomics analysis of A549 cells treated with arginine depletion, related to Figure 5</title></caption><media xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"mmc8.xlsx\" position=\"float\" orientation=\"portrait\"/></supplementary-material>\n",
      "<supplementary-material content-type=\"local-data\" id=\"mmc9\" position=\"float\" orientation=\"portrait\"><caption><title>Table S7. DNA fragments of the plasmids used in this study, related to Figures 3, 4, and 5</title></caption><media xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"mmc9.xlsx\" position=\"float\" orientation=\"portrait\"/></supplementary-material>\n",
      "<supplementary-material content-type=\"local-data\" id=\"mmc10\" position=\"float\" orientation=\"portrait\"><caption><title>Document S2. Article plus supplemental information</title></caption><media xmlns:xlink=\"http://www.w3.org/1999/xlink\" xlink:href=\"mmc10.pdf\" position=\"float\" orientation=\"portrait\"/></supplementary-material>\n",
      "</p></sec>\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 62 - INFO - XML data contains all required sections.\u001B[0m\n",
      "\u001B[97morchestrator.py - line 347 - INFO - XML data is complete for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11129317.\u001B[0m\n",
      "\u001B[97morchestrator.py - line 351 - INFO - Raw XML data fetched from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11129317 is ready for parsing.\u001B[0m\n",
      "\u001B[97morchestrator.py - line 365 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97morchestrator.py - line 369 - INFO - Using XMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 181 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 23 - INFO - Initializing xmlRetriever\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 141 - INFO - Function call: parse_data(api_data(<class 'lxml.etree._Element'>), PMC, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11129317, additional_data, XML)\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 440 - INFO - Function_call: extract_href_from_supplementary_material(api_xml, current_url_address)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 286 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_material_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 287 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 451 - INFO - Found 10 supplementary material sections .//supplementary-material. cont: [<Element supplementary-material at 0x1a713d940>, <Element supplementary-material at 0x1aee62b80>, <Element supplementary-material at 0x1ad473780>, <Element supplementary-material at 0x1ad4734c0>, <Element supplementary-material at 0x1ad473300>, <Element supplementary-material at 0x1ad473200>, <Element supplementary-material at 0x1ad473000>, <Element supplementary-material at 0x1ad473480>, <Element supplementary-material at 0x1ad472e40>, <Element supplementary-material at 0x1ad472c40>]\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 459 - INFO - Processing section: {'ptr': './/supplementary-material', 'cont': [<Element supplementary-material at 0x1a713d940>, <Element supplementary-material at 0x1aee62b80>, <Element supplementary-material at 0x1ad473780>, <Element supplementary-material at 0x1ad4734c0>, <Element supplementary-material at 0x1ad473300>, <Element supplementary-material at 0x1ad473200>, <Element supplementary-material at 0x1ad473000>, <Element supplementary-material at 0x1ad473480>, <Element supplementary-material at 0x1ad472e40>, <Element supplementary-material at 0x1ad472c40>]}\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 123 - INFO - Loading patterns for section 'data_availability_sections' for publisher 'PMC'.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 131 - INFO - Section pattern keys for publisher PMC: dict_keys(['data_availability_sections', 'supplementary_material_sections', 'supplementary_data_sections', 'key_resources_table', 'methods_sections'])\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 133 - INFO - Section name: data_availability_sections\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 135 - INFO - Found section 'data_availability_sections' in patterns for publisher 'PMC'.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 137 - INFO - Loaded patterns for section 'data_availability_sections': [\".//sec[@sec-type='data-availability']\", \".//notes[@notes-type='data-availability']\", \".//notes[@notes-type='data-availability']\"]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 286 - INFO - Function_call: load_patterns_for_tgt_section(supplementary_data_sections)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 287 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 653 - INFO - Found 11 supplementary data sections\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 286 - INFO - Function_call: load_patterns_for_tgt_section(key_resources_table)\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 287 - INFO - Consider migrating this function to the BaseRetriever class.\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 672 - INFO - Found key resources table: <Element table-wrap at 0x1aeb0f300>.\u001B[0m\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "Batches:   0%|          | 0/2 [00:00<?, ?it/s]"
      ],
      "application/vnd.jupyter.widget-view+json": {
       "version_major": 2,
       "version_minor": 0,
       "model_id": "7acf1a3479864662ae4153b634e8b544"
      }
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001B[97membeddings_retriever.py - line 51 - INFO - Searching for top-2 passages similar to the query by embeddings.\u001B[0m\n",
      "\u001B[97membeddings_retriever.py - line 33 - INFO - Computing L2 distances using numpy.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 388 - INFO - Processing DAS_content: 4 elements\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 432 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 32 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1193 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 440 - INFO - Content length: 997\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1193 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 451 - INFO - Prompt messages total length: 592 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 457 - INFO - Prompt ID: gemini-2.0-flash-0-0cb7d4109d68dfc3135db7bab7d7f20166d6d8fd3ab1e577750f4773d09f3d99\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1193 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 476 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages: 592 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 547 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-7b4d40a7-00ae-4bc2-9fbb-bcc5f6c84535\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"PDC commons\\\",\\n        \\\"data_repository\\\": \\\"PDC\\\",\\n        \\\"dataset_webpage\\\": \\\"https://pdc.cancer.gov/pdc/\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"Pride\\\",\\n        \\\"data_repository\\\": \\\"Pride\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/pride/archive/\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"Mendeley\\\",\\n        \\\"data_repository\\\": \\\"Mendeley\\\",\\n        \\\"dataset_webpage\\\": \\\"https://data.mendeley.com/\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"Mendeley Data\\\",\\n        \\\"data_repository\\\": \\\"Mendeley\\\",\\n        \\\"dataset_webpage\\\": \\\"https://data.mendeley.com/\\\"\\n    }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"PDC commons\\\",\\n        \\\"data_repository\\\": \\\"PDC\\\",\\n        \\\"dataset_webpage\\\": \\\"https://pdc.cancer.gov/pdc/\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"Pride\\\",\\n        \\\"data_repository\\\": \\\"Pride\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/pride/archive/\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"Mendeley\\\",\\n        \\\"data_repository\\\": \\\"Mendeley\\\",\\n        \\\"dataset_webpage\\\": \\\"https://data.mendeley.com/\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"Mendeley Data\\\",\\n        \\\"data_repository\\\": \\\"Mendeley\\\",\\n        \\\"dataset_webpage\\\": \\\"https://data.mendeley.com/\\\"\\n    }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1752012031,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 666,\n",
      "        \"completion_tokens\": 202,\n",
      "        \"total_tokens\": 868,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 607 - INFO - Processing dataset: {'dataset_identifier': 'PDC commons', 'data_repository': 'PDC', 'dataset_webpage': 'https://pdc.cancer.gov/pdc/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 622 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 954 - INFO - Validating accession ID: PDC commons\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Accession ID PDC commons is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Resolving data repository for candidate: PDC, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 910 - INFO - Extracting repo domain from URL: https://pdc.cancer.gov/pdc/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1059 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 972 - INFO - Validating Dataset Page: https://pdc.cancer.gov/pdc/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 982 - INFO - Link does not match the pattern https://proteomic.datacommons.cancer.gov/pdc/study/__ID__\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Dataset ID is 'n/a' and repository name from prompt\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 607 - INFO - Processing dataset: {'dataset_identifier': 'Pride', 'data_repository': 'Pride', 'dataset_webpage': 'https://www.ebi.ac.uk/pride/archive/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 622 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 954 - INFO - Validating accession ID: Pride\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 956 - INFO - Accession ID Pride is a known repository --> invalid, returning 'n/a'\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Resolving data repository for candidate: Pride, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1038 - INFO - Found repo_name match for Pride\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 972 - INFO - Validating Dataset Page: https://www.ebi.ac.uk/pride/archive/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 985 - INFO - No dataset_webpage_url_ptr found for www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Dataset ID is 'n/a' and repository name from prompt\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 607 - INFO - Processing dataset: {'dataset_identifier': 'Mendeley', 'data_repository': 'Mendeley', 'dataset_webpage': 'https://data.mendeley.com/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 622 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 954 - INFO - Validating accession ID: Mendeley\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 956 - INFO - Accession ID Mendeley is a known repository --> invalid, returning 'n/a'\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Resolving data repository for candidate: Mendeley, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1038 - INFO - Found repo_name match for Mendeley\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 972 - INFO - Validating Dataset Page: https://data.mendeley.com/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 985 - INFO - No dataset_webpage_url_ptr found for data.mendeley.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Dataset ID is 'n/a' and repository name from prompt\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 607 - INFO - Processing dataset: {'dataset_identifier': 'Mendeley Data', 'data_repository': 'Mendeley', 'dataset_webpage': 'https://data.mendeley.com/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 622 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 954 - INFO - Validating accession ID: Mendeley Data\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Accession ID Mendeley Data is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Resolving data repository for candidate: Mendeley, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1038 - INFO - Found repo_name match for Mendeley\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 972 - INFO - Validating Dataset Page: https://data.mendeley.com/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 985 - INFO - No dataset_webpage_url_ptr found for data.mendeley.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Dataset ID is 'n/a' and repository name from prompt\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 432 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 32 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1193 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 440 - INFO - Content length: 4478\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1193 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 451 - INFO - Prompt messages total length: 1462 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 457 - INFO - Prompt ID: gemini-2.0-flash-0-94aab37a96b7649f10a13893a0d80c0bc3a0ed9bad4986e281d2b1d1fbf1c18f\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1193 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 476 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages: 1462 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 547 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-8f7049e6-8f20-46ef-9c39-df83f043dde6\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"PXD043612\\\",\\n        \\\"data_repository\\\": \\\"Pride\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/pride/archive/projects/PXD043612\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"https://doi.org/10.17632/3wfxrz66w2.1\\\",\\n        \\\"data_repository\\\": \\\"This study\\\",\\n        \\\"dataset_webpage\\\": \\\"https://doi.org/10.17632/3wfxrz66w2.1\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"https://doi.org/10.17632/bvdn865y9c.1\\\",\\n        \\\"data_repository\\\": \\\"This study\\\",\\n        \\\"dataset_webpage\\\": \\\"https://doi.org/10.17632/bvdn865y9c.1\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"PDC000234\\\",\\n        \\\"data_repository\\\": \\\"CPTAC\\\",\\n        \\\"dataset_webpage\\\": \\\"https://pdc.cancer.gov/pdc/study/PDC000234\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"PDC000127\\\",\\n        \\\"data_repository\\\": \\\"CPTAC\\\",\\n        \\\"dataset_webpage\\\": \\\"https://pdc.cancer.gov/pdc/study/PDC000127\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"PDC000204\\\",\\n        \\\"data_repository\\\": \\\"CPTAC\\\",\\n        \\\"dataset_webpage\\\": \\\"https://pdc.cancer.gov/pdc/study/PDC000204\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"PDC000221\\\",\\n        \\\"data_repository\\\": \\\"CPTAC\\\",\\n        \\\"dataset_webpage\\\": \\\"https://pdc.cancer.gov/pdc/study/PDC000221\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"PDC000198\\\",\\n        \\\"data_repository\\\": \\\"CPTAC\\\",\\n        \\\"dataset_webpage\\\": \\\"https://pdc.cancer.gov/pdc/study/PDC000198\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"PDC000110\\\",\\n        \\\"data_repository\\\": \\\"CPTAC\\\",\\n        \\\"dataset_webpage\\\": \\\"https://pdc.cancer.gov/pdc/study/PDC000110\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"PDC000270\\\",\\n        \\\"data_repository\\\": \\\"CPTAC\\\",\\n        \\\"dataset_webpage\\\": \\\"https://pdc.cancer.gov/pdc/study/PDC000270\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"PDC000125\\\",\\n        \\\"data_repository\\\": \\\"CPTAC\\\",\\n        \\\"dataset_webpage\\\": \\\"https://pdc.cancer.gov/pdc/study/PDC000125\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"PDC000153\\\",\\n        \\\"data_repository\\\": \\\"CPTAC\\\",\\n        \\\"dataset_webpage\\\": \\\"https://pdc.cancer.gov/pdc/study/PDC000153\\\"\\n    }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"PXD043612\\\",\\n        \\\"data_repository\\\": \\\"Pride\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/pride/archive/projects/PXD043612\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"https://doi.org/10.17632/3wfxrz66w2.1\\\",\\n        \\\"data_repository\\\": \\\"This study\\\",\\n        \\\"dataset_webpage\\\": \\\"https://doi.org/10.17632/3wfxrz66w2.1\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"https://doi.org/10.17632/bvdn865y9c.1\\\",\\n        \\\"data_repository\\\": \\\"This study\\\",\\n        \\\"dataset_webpage\\\": \\\"https://doi.org/10.17632/bvdn865y9c.1\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"PDC000234\\\",\\n        \\\"data_repository\\\": \\\"CPTAC\\\",\\n        \\\"dataset_webpage\\\": \\\"https://pdc.cancer.gov/pdc/study/PDC000234\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"PDC000127\\\",\\n        \\\"data_repository\\\": \\\"CPTAC\\\",\\n        \\\"dataset_webpage\\\": \\\"https://pdc.cancer.gov/pdc/study/PDC000127\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"PDC000204\\\",\\n        \\\"data_repository\\\": \\\"CPTAC\\\",\\n        \\\"dataset_webpage\\\": \\\"https://pdc.cancer.gov/pdc/study/PDC000204\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"PDC000221\\\",\\n        \\\"data_repository\\\": \\\"CPTAC\\\",\\n        \\\"dataset_webpage\\\": \\\"https://pdc.cancer.gov/pdc/study/PDC000221\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"PDC000198\\\",\\n        \\\"data_repository\\\": \\\"CPTAC\\\",\\n        \\\"dataset_webpage\\\": \\\"https://pdc.cancer.gov/pdc/study/PDC000198\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"PDC000110\\\",\\n        \\\"data_repository\\\": \\\"CPTAC\\\",\\n        \\\"dataset_webpage\\\": \\\"https://pdc.cancer.gov/pdc/study/PDC000110\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"PDC000270\\\",\\n        \\\"data_repository\\\": \\\"CPTAC\\\",\\n        \\\"dataset_webpage\\\": \\\"https://pdc.cancer.gov/pdc/study/PDC000270\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"PDC000125\\\",\\n        \\\"data_repository\\\": \\\"CPTAC\\\",\\n        \\\"dataset_webpage\\\": \\\"https://pdc.cancer.gov/pdc/study/PDC000125\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"PDC000153\\\",\\n        \\\"data_repository\\\": \\\"CPTAC\\\",\\n        \\\"dataset_webpage\\\": \\\"https://pdc.cancer.gov/pdc/study/PDC000153\\\"\\n    }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1752012036,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 2306,\n",
      "        \"completion_tokens\": 825,\n",
      "        \"total_tokens\": 3131,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 607 - INFO - Processing dataset: {'dataset_identifier': 'PXD043612', 'data_repository': 'Pride', 'dataset_webpage': 'https://www.ebi.ac.uk/pride/archive/projects/PXD043612'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 622 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 954 - INFO - Validating accession ID: PXD043612\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 965 - INFO - Accession ID PXD043612 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Resolving data repository for candidate: Pride, identifier: PXD043612\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1038 - INFO - Found repo_name match for Pride\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 972 - INFO - Validating Dataset Page: https://www.ebi.ac.uk/pride/archive/projects/PXD043612\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 985 - INFO - No dataset_webpage_url_ptr found for www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 668 - INFO - Extracted dataset: {'dataset_identifier': 'PXD043612', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'https://www.ebi.ac.uk/pride/archive/projects/PXD043612'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 607 - INFO - Processing dataset: {'dataset_identifier': 'https://doi.org/10.17632/3wfxrz66w2.1', 'data_repository': 'This study', 'dataset_webpage': 'https://doi.org/10.17632/3wfxrz66w2.1'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 622 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 954 - INFO - Validating accession ID: https://doi.org/10.17632/3wfxrz66w2.1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Resolving data repository for candidate: This study, identifier: 10.17632/3wfxrz66w2.1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 910 - INFO - Extracting repo domain from URL: https://doi.org/10.17632/3wfxrz66w2.1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1059 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 972 - INFO - Validating Dataset Page: https://doi.org/10.17632/3wfxrz66w2.1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 982 - INFO - Link does not match the pattern https://doi.org/__ID__\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 668 - INFO - Extracted dataset: {'dataset_identifier': '10.17632/3wfxrz66w2.1', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 607 - INFO - Processing dataset: {'dataset_identifier': 'https://doi.org/10.17632/bvdn865y9c.1', 'data_repository': 'This study', 'dataset_webpage': 'https://doi.org/10.17632/bvdn865y9c.1'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 622 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 954 - INFO - Validating accession ID: https://doi.org/10.17632/bvdn865y9c.1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Resolving data repository for candidate: This study, identifier: 10.17632/bvdn865y9c.1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 910 - INFO - Extracting repo domain from URL: https://doi.org/10.17632/bvdn865y9c.1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1059 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 972 - INFO - Validating Dataset Page: https://doi.org/10.17632/bvdn865y9c.1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 982 - INFO - Link does not match the pattern https://doi.org/__ID__\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 668 - INFO - Extracted dataset: {'dataset_identifier': '10.17632/bvdn865y9c.1', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 607 - INFO - Processing dataset: {'dataset_identifier': 'PDC000234', 'data_repository': 'CPTAC', 'dataset_webpage': 'https://pdc.cancer.gov/pdc/study/PDC000234'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 622 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 954 - INFO - Validating accession ID: PDC000234\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 965 - INFO - Accession ID PDC000234 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Resolving data repository for candidate: CPTAC, identifier: PDC000234\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 910 - INFO - Extracting repo domain from URL: https://pdc.cancer.gov/pdc/study/PDC000234\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1059 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 972 - INFO - Validating Dataset Page: https://pdc.cancer.gov/pdc/study/PDC000234\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 982 - INFO - Link does not match the pattern https://proteomic.datacommons.cancer.gov/pdc/study/__ID__\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 668 - INFO - Extracted dataset: {'dataset_identifier': 'PDC000234', 'data_repository': 'pdc.cancer.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 607 - INFO - Processing dataset: {'dataset_identifier': 'PDC000127', 'data_repository': 'CPTAC', 'dataset_webpage': 'https://pdc.cancer.gov/pdc/study/PDC000127'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 622 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 954 - INFO - Validating accession ID: PDC000127\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 965 - INFO - Accession ID PDC000127 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Resolving data repository for candidate: CPTAC, identifier: PDC000127\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 910 - INFO - Extracting repo domain from URL: https://pdc.cancer.gov/pdc/study/PDC000127\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1059 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 972 - INFO - Validating Dataset Page: https://pdc.cancer.gov/pdc/study/PDC000127\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 982 - INFO - Link does not match the pattern https://proteomic.datacommons.cancer.gov/pdc/study/__ID__\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 668 - INFO - Extracted dataset: {'dataset_identifier': 'PDC000127', 'data_repository': 'pdc.cancer.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 607 - INFO - Processing dataset: {'dataset_identifier': 'PDC000204', 'data_repository': 'CPTAC', 'dataset_webpage': 'https://pdc.cancer.gov/pdc/study/PDC000204'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 622 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 954 - INFO - Validating accession ID: PDC000204\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 965 - INFO - Accession ID PDC000204 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Resolving data repository for candidate: CPTAC, identifier: PDC000204\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 910 - INFO - Extracting repo domain from URL: https://pdc.cancer.gov/pdc/study/PDC000204\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1059 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 972 - INFO - Validating Dataset Page: https://pdc.cancer.gov/pdc/study/PDC000204\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 982 - INFO - Link does not match the pattern https://proteomic.datacommons.cancer.gov/pdc/study/__ID__\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 668 - INFO - Extracted dataset: {'dataset_identifier': 'PDC000204', 'data_repository': 'pdc.cancer.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 607 - INFO - Processing dataset: {'dataset_identifier': 'PDC000221', 'data_repository': 'CPTAC', 'dataset_webpage': 'https://pdc.cancer.gov/pdc/study/PDC000221'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 622 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 954 - INFO - Validating accession ID: PDC000221\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 965 - INFO - Accession ID PDC000221 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Resolving data repository for candidate: CPTAC, identifier: PDC000221\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 910 - INFO - Extracting repo domain from URL: https://pdc.cancer.gov/pdc/study/PDC000221\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1059 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 972 - INFO - Validating Dataset Page: https://pdc.cancer.gov/pdc/study/PDC000221\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 982 - INFO - Link does not match the pattern https://proteomic.datacommons.cancer.gov/pdc/study/__ID__\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 668 - INFO - Extracted dataset: {'dataset_identifier': 'PDC000221', 'data_repository': 'pdc.cancer.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 607 - INFO - Processing dataset: {'dataset_identifier': 'PDC000198', 'data_repository': 'CPTAC', 'dataset_webpage': 'https://pdc.cancer.gov/pdc/study/PDC000198'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 622 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 954 - INFO - Validating accession ID: PDC000198\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 965 - INFO - Accession ID PDC000198 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Resolving data repository for candidate: CPTAC, identifier: PDC000198\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 910 - INFO - Extracting repo domain from URL: https://pdc.cancer.gov/pdc/study/PDC000198\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1059 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 972 - INFO - Validating Dataset Page: https://pdc.cancer.gov/pdc/study/PDC000198\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 982 - INFO - Link does not match the pattern https://proteomic.datacommons.cancer.gov/pdc/study/__ID__\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 668 - INFO - Extracted dataset: {'dataset_identifier': 'PDC000198', 'data_repository': 'pdc.cancer.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 607 - INFO - Processing dataset: {'dataset_identifier': 'PDC000110', 'data_repository': 'CPTAC', 'dataset_webpage': 'https://pdc.cancer.gov/pdc/study/PDC000110'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 622 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 954 - INFO - Validating accession ID: PDC000110\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 965 - INFO - Accession ID PDC000110 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Resolving data repository for candidate: CPTAC, identifier: PDC000110\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 910 - INFO - Extracting repo domain from URL: https://pdc.cancer.gov/pdc/study/PDC000110\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1059 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 972 - INFO - Validating Dataset Page: https://pdc.cancer.gov/pdc/study/PDC000110\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 982 - INFO - Link does not match the pattern https://proteomic.datacommons.cancer.gov/pdc/study/__ID__\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 668 - INFO - Extracted dataset: {'dataset_identifier': 'PDC000110', 'data_repository': 'pdc.cancer.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 607 - INFO - Processing dataset: {'dataset_identifier': 'PDC000270', 'data_repository': 'CPTAC', 'dataset_webpage': 'https://pdc.cancer.gov/pdc/study/PDC000270'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 622 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 954 - INFO - Validating accession ID: PDC000270\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 965 - INFO - Accession ID PDC000270 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Resolving data repository for candidate: CPTAC, identifier: PDC000270\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 910 - INFO - Extracting repo domain from URL: https://pdc.cancer.gov/pdc/study/PDC000270\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1059 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 972 - INFO - Validating Dataset Page: https://pdc.cancer.gov/pdc/study/PDC000270\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 982 - INFO - Link does not match the pattern https://proteomic.datacommons.cancer.gov/pdc/study/__ID__\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 668 - INFO - Extracted dataset: {'dataset_identifier': 'PDC000270', 'data_repository': 'pdc.cancer.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 607 - INFO - Processing dataset: {'dataset_identifier': 'PDC000125', 'data_repository': 'CPTAC', 'dataset_webpage': 'https://pdc.cancer.gov/pdc/study/PDC000125'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 622 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 954 - INFO - Validating accession ID: PDC000125\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 965 - INFO - Accession ID PDC000125 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Resolving data repository for candidate: CPTAC, identifier: PDC000125\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 910 - INFO - Extracting repo domain from URL: https://pdc.cancer.gov/pdc/study/PDC000125\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1059 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 972 - INFO - Validating Dataset Page: https://pdc.cancer.gov/pdc/study/PDC000125\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 982 - INFO - Link does not match the pattern https://proteomic.datacommons.cancer.gov/pdc/study/__ID__\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 668 - INFO - Extracted dataset: {'dataset_identifier': 'PDC000125', 'data_repository': 'pdc.cancer.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 607 - INFO - Processing dataset: {'dataset_identifier': 'PDC000153', 'data_repository': 'CPTAC', 'dataset_webpage': 'https://pdc.cancer.gov/pdc/study/PDC000153'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 622 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 954 - INFO - Validating accession ID: PDC000153\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 965 - INFO - Accession ID PDC000153 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Resolving data repository for candidate: CPTAC, identifier: PDC000153\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 910 - INFO - Extracting repo domain from URL: https://pdc.cancer.gov/pdc/study/PDC000153\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1059 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 972 - INFO - Validating Dataset Page: https://pdc.cancer.gov/pdc/study/PDC000153\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 982 - INFO - Link does not match the pattern https://proteomic.datacommons.cancer.gov/pdc/study/__ID__\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 668 - INFO - Extracted dataset: {'dataset_identifier': 'PDC000153', 'data_repository': 'pdc.cancer.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 432 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 32 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1193 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 440 - INFO - Content length: 2975\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1193 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 451 - INFO - Prompt messages total length: 1087 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 457 - INFO - Prompt ID: gemini-2.0-flash-0-03cf82093044666e9d818017e75c687e2a57c8358268e03aa549a8dd608513d8\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1193 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 476 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages: 1087 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 547 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-cbabac91-f324-4a5e-9509-1cd76b339197\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"PDC IDs\\\",\\n        \\\"data_repository\\\": \\\"PDC commons\\\",\\n        \\\"dataset_webpage\\\": \\\"https://pdc.cancer.gov/pdc/\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"Mass spectrometry proteomics data\\\",\\n        \\\"data_repository\\\": \\\"Pride\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/pride/\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"original western blot images\\\",\\n        \\\"data_repository\\\": \\\"Mendeley\\\",\\n        \\\"dataset_webpage\\\": \\\"https://data.mendeley.com/\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"All original codes\\\",\\n        \\\"data_repository\\\": \\\"Mendeley Data\\\",\\n        \\\"dataset_webpage\\\": \\\"https://data.mendeley.com/\\\"\\n    }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"PDC IDs\\\",\\n        \\\"data_repository\\\": \\\"PDC commons\\\",\\n        \\\"dataset_webpage\\\": \\\"https://pdc.cancer.gov/pdc/\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"Mass spectrometry proteomics data\\\",\\n        \\\"data_repository\\\": \\\"Pride\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/pride/\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"original western blot images\\\",\\n        \\\"data_repository\\\": \\\"Mendeley\\\",\\n        \\\"dataset_webpage\\\": \\\"https://data.mendeley.com/\\\"\\n    },\\n    {\\n        \\\"dataset_identifier\\\": \\\"All original codes\\\",\\n        \\\"data_repository\\\": \\\"Mendeley Data\\\",\\n        \\\"dataset_webpage\\\": \\\"https://data.mendeley.com/\\\"\\n    }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1752012952,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 1387,\n",
      "        \"completion_tokens\": 206,\n",
      "        \"total_tokens\": 1593,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 607 - INFO - Processing dataset: {'dataset_identifier': 'PDC IDs', 'data_repository': 'PDC commons', 'dataset_webpage': 'https://pdc.cancer.gov/pdc/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 622 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 954 - INFO - Validating accession ID: PDC IDs\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Accession ID PDC IDs is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Resolving data repository for candidate: PDC commons, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 910 - INFO - Extracting repo domain from URL: https://pdc.cancer.gov/pdc/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1059 - INFO - data_repository not resolved, using domain\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 972 - INFO - Validating Dataset Page: https://pdc.cancer.gov/pdc/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 982 - INFO - Link does not match the pattern https://proteomic.datacommons.cancer.gov/pdc/study/__ID__\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Dataset ID is 'n/a' and repository name from prompt\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 607 - INFO - Processing dataset: {'dataset_identifier': 'Mass spectrometry proteomics data', 'data_repository': 'Pride', 'dataset_webpage': 'https://www.ebi.ac.uk/pride/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 622 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 954 - INFO - Validating accession ID: Mass spectrometry proteomics data\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Accession ID Mass spectrometry proteomics data is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Resolving data repository for candidate: Pride, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1038 - INFO - Found repo_name match for Pride\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 972 - INFO - Validating Dataset Page: https://www.ebi.ac.uk/pride/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 985 - INFO - No dataset_webpage_url_ptr found for www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Dataset ID is 'n/a' and repository name from prompt\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 607 - INFO - Processing dataset: {'dataset_identifier': 'original western blot images', 'data_repository': 'Mendeley', 'dataset_webpage': 'https://data.mendeley.com/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 622 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 954 - INFO - Validating accession ID: original western blot images\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Accession ID original western blot images is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Resolving data repository for candidate: Mendeley, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1038 - INFO - Found repo_name match for Mendeley\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 972 - INFO - Validating Dataset Page: https://data.mendeley.com/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 985 - INFO - No dataset_webpage_url_ptr found for data.mendeley.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Dataset ID is 'n/a' and repository name from prompt\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 607 - INFO - Processing dataset: {'dataset_identifier': 'All original codes', 'data_repository': 'Mendeley Data', 'dataset_webpage': 'https://data.mendeley.com/'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 622 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 954 - INFO - Validating accession ID: All original codes\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 959 - INFO - Accession ID All original codes is invalid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Resolving data repository for candidate: Mendeley Data, identifier: n/a\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1044 - INFO - Found partial match for Mendeley Data in Mendeley\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 972 - INFO - Validating Dataset Page: https://data.mendeley.com/\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 985 - INFO - No dataset_webpage_url_ptr found for data.mendeley.com\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 649 - INFO - Dataset ID is 'n/a' and repository name from prompt\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 432 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 32 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1193 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 440 - INFO - Content length: 1284\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1193 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 451 - INFO - Prompt messages total length: 664 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 457 - INFO - Prompt ID: gemini-2.0-flash-0-96eb34fe8a74bbdf6f8760c39e2158e4e6675b0abcf9f3654d3052e695b8a3cd\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1193 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 476 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages: 664 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 547 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-234e1265-289e-4c05-9e30-99c9e402db0e\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"PXD043612\\\",\\n        \\\"data_repository\\\": \\\"PRIDE\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/pride/archive/projects/PXD043612\\\"\\n    ]\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    [\\n        \\\"dataset_identifier\\\": \\\"PXD043612\\\",\\n        \\\"data_repository\\\": \\\"PRIDE\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ebi.ac.uk/pride/archive/projects/PXD043612\\\"\\n    ]\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1752012953,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 797,\n",
      "        \"completion_tokens\": 76,\n",
      "        \"total_tokens\": 873,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 607 - INFO - Processing dataset: {'dataset_identifier': 'PXD043612', 'data_repository': 'PRIDE', 'dataset_webpage': 'https://www.ebi.ac.uk/pride/archive/projects/PXD043612'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 622 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 954 - INFO - Validating accession ID: PXD043612\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 965 - INFO - Accession ID PXD043612 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Resolving data repository for candidate: PRIDE, identifier: PXD043612\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1038 - INFO - Found repo_name match for PRIDE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 972 - INFO - Validating Dataset Page: https://www.ebi.ac.uk/pride/archive/projects/PXD043612\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 985 - INFO - No dataset_webpage_url_ptr found for www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 668 - INFO - Extracted dataset: {'dataset_identifier': 'PXD043612', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'https://www.ebi.ac.uk/pride/archive/projects/PXD043612'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 401 - INFO - datasets (<class 'list'>): [{'dataset_identifier': 'PXD043612', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'https://www.ebi.ac.uk/pride/archive/projects/PXD043612'}, {'dataset_identifier': '10.17632/3wfxrz66w2.1', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': '10.17632/bvdn865y9c.1', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'PDC000234', 'data_repository': 'pdc.cancer.gov', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'PDC000127', 'data_repository': 'pdc.cancer.gov', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'PDC000204', 'data_repository': 'pdc.cancer.gov', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'PDC000221', 'data_repository': 'pdc.cancer.gov', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'PDC000198', 'data_repository': 'pdc.cancer.gov', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'PDC000110', 'data_repository': 'pdc.cancer.gov', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'PDC000270', 'data_repository': 'pdc.cancer.gov', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'PDC000125', 'data_repository': 'pdc.cancer.gov', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'PDC000153', 'data_repository': 'pdc.cancer.gov', 'dataset_webpage': 'n/a'}, {'dataset_identifier': 'PXD043612', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'https://www.ebi.ac.uk/pride/archive/projects/PXD043612'}]\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 403 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'PXD043612', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'https://www.ebi.ac.uk/pride/archive/projects/PXD043612'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 405 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 403 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': '10.17632/3wfxrz66w2.1', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 405 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 403 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': '10.17632/bvdn865y9c.1', 'data_repository': 'doi.org', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 405 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 403 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'PDC000234', 'data_repository': 'pdc.cancer.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 405 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 403 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'PDC000127', 'data_repository': 'pdc.cancer.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 405 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 403 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'PDC000204', 'data_repository': 'pdc.cancer.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 405 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 403 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'PDC000221', 'data_repository': 'pdc.cancer.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 405 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 403 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'PDC000198', 'data_repository': 'pdc.cancer.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 405 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 403 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'PDC000110', 'data_repository': 'pdc.cancer.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 405 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 403 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'PDC000270', 'data_repository': 'pdc.cancer.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 405 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 403 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'PDC000125', 'data_repository': 'pdc.cancer.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 405 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 403 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'PDC000153', 'data_repository': 'pdc.cancer.gov', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 405 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 403 - INFO - iter dataset (<class 'dict'>): {'dataset_identifier': 'PXD043612', 'data_repository': 'www.ebi.ac.uk', 'dataset_webpage': 'https://www.ebi.ac.uk/pride/archive/projects/PXD043612'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 405 - WARNING - Adding retrieval pattern 'data availability' to dataset\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 409 - INFO - Final ret additional data: 13 items\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1075 - INFO - Fetching metadata for 13 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1083 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1094 - INFO - Raw accession ID: PXD043612\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Resolving data repository for candidate: www.ebi.ac.uk, identifier: PXD043612\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1032 - INFO - Exact match found for repo: www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 996 - INFO - Resolving accession ID for PXD043612 in www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1108 - INFO - Processing dataset 1 with repo: www.ebi.ac.uk and accession_id: PXD043612\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1134 - INFO - Dataset page: https://www.ebi.ac.uk/pride/archive/projects/PXD043612\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1141 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1083 - INFO - Processing dataset 2 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1094 - INFO - Raw accession ID: 10.17632/3wfxrz66w2.1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Resolving data repository for candidate: doi.org, identifier: 10.17632/3wfxrz66w2.1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1032 - INFO - Exact match found for repo: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 996 - INFO - Resolving accession ID for 10.17632/3wfxrz66w2.1 in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1108 - INFO - Processing dataset 2 with repo: doi.org and accession_id: 10.17632/3wfxrz66w2.1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1134 - INFO - Dataset page: https://doi.org/10.17632/3wfxrz66w2.1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1083 - INFO - Processing dataset 3 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1094 - INFO - Raw accession ID: 10.17632/bvdn865y9c.1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Resolving data repository for candidate: doi.org, identifier: 10.17632/bvdn865y9c.1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1032 - INFO - Exact match found for repo: doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 996 - INFO - Resolving accession ID for 10.17632/bvdn865y9c.1 in doi.org\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1108 - INFO - Processing dataset 3 with repo: doi.org and accession_id: 10.17632/bvdn865y9c.1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1134 - INFO - Dataset page: https://doi.org/10.17632/bvdn865y9c.1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1083 - INFO - Processing dataset 4 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1094 - INFO - Raw accession ID: PDC000234\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Resolving data repository for candidate: pdc.cancer.gov, identifier: PDC000234\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1032 - INFO - Exact match found for repo: pdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 996 - INFO - Resolving accession ID for PDC000234 in pdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1108 - INFO - Processing dataset 4 with repo: pdc.cancer.gov and accession_id: PDC000234\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1134 - INFO - Dataset page: https://proteomic.datacommons.cancer.gov/pdc/study/PDC000234\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1141 - INFO - Adding access mode for dataset 4: Complex download or Application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1083 - INFO - Processing dataset 5 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1094 - INFO - Raw accession ID: PDC000127\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Resolving data repository for candidate: pdc.cancer.gov, identifier: PDC000127\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1032 - INFO - Exact match found for repo: pdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 996 - INFO - Resolving accession ID for PDC000127 in pdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1108 - INFO - Processing dataset 5 with repo: pdc.cancer.gov and accession_id: PDC000127\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1134 - INFO - Dataset page: https://proteomic.datacommons.cancer.gov/pdc/study/PDC000127\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1141 - INFO - Adding access mode for dataset 5: Complex download or Application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1083 - INFO - Processing dataset 6 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1094 - INFO - Raw accession ID: PDC000204\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Resolving data repository for candidate: pdc.cancer.gov, identifier: PDC000204\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1032 - INFO - Exact match found for repo: pdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 996 - INFO - Resolving accession ID for PDC000204 in pdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1108 - INFO - Processing dataset 6 with repo: pdc.cancer.gov and accession_id: PDC000204\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1134 - INFO - Dataset page: https://proteomic.datacommons.cancer.gov/pdc/study/PDC000204\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1141 - INFO - Adding access mode for dataset 6: Complex download or Application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1083 - INFO - Processing dataset 7 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1094 - INFO - Raw accession ID: PDC000221\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Resolving data repository for candidate: pdc.cancer.gov, identifier: PDC000221\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1032 - INFO - Exact match found for repo: pdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 996 - INFO - Resolving accession ID for PDC000221 in pdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1108 - INFO - Processing dataset 7 with repo: pdc.cancer.gov and accession_id: PDC000221\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1134 - INFO - Dataset page: https://proteomic.datacommons.cancer.gov/pdc/study/PDC000221\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1141 - INFO - Adding access mode for dataset 7: Complex download or Application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1083 - INFO - Processing dataset 8 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1094 - INFO - Raw accession ID: PDC000198\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Resolving data repository for candidate: pdc.cancer.gov, identifier: PDC000198\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1032 - INFO - Exact match found for repo: pdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 996 - INFO - Resolving accession ID for PDC000198 in pdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1108 - INFO - Processing dataset 8 with repo: pdc.cancer.gov and accession_id: PDC000198\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1134 - INFO - Dataset page: https://proteomic.datacommons.cancer.gov/pdc/study/PDC000198\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1141 - INFO - Adding access mode for dataset 8: Complex download or Application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1083 - INFO - Processing dataset 9 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1094 - INFO - Raw accession ID: PDC000110\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Resolving data repository for candidate: pdc.cancer.gov, identifier: PDC000110\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1032 - INFO - Exact match found for repo: pdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 996 - INFO - Resolving accession ID for PDC000110 in pdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1108 - INFO - Processing dataset 9 with repo: pdc.cancer.gov and accession_id: PDC000110\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1134 - INFO - Dataset page: https://proteomic.datacommons.cancer.gov/pdc/study/PDC000110\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1141 - INFO - Adding access mode for dataset 9: Complex download or Application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1083 - INFO - Processing dataset 10 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1094 - INFO - Raw accession ID: PDC000270\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Resolving data repository for candidate: pdc.cancer.gov, identifier: PDC000270\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1032 - INFO - Exact match found for repo: pdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 996 - INFO - Resolving accession ID for PDC000270 in pdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1108 - INFO - Processing dataset 10 with repo: pdc.cancer.gov and accession_id: PDC000270\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1134 - INFO - Dataset page: https://proteomic.datacommons.cancer.gov/pdc/study/PDC000270\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1141 - INFO - Adding access mode for dataset 10: Complex download or Application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1083 - INFO - Processing dataset 11 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1094 - INFO - Raw accession ID: PDC000125\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Resolving data repository for candidate: pdc.cancer.gov, identifier: PDC000125\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1032 - INFO - Exact match found for repo: pdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 996 - INFO - Resolving accession ID for PDC000125 in pdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1108 - INFO - Processing dataset 11 with repo: pdc.cancer.gov and accession_id: PDC000125\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1134 - INFO - Dataset page: https://proteomic.datacommons.cancer.gov/pdc/study/PDC000125\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1141 - INFO - Adding access mode for dataset 11: Complex download or Application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1083 - INFO - Processing dataset 12 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1094 - INFO - Raw accession ID: PDC000153\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Resolving data repository for candidate: pdc.cancer.gov, identifier: PDC000153\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1032 - INFO - Exact match found for repo: pdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 996 - INFO - Resolving accession ID for PDC000153 in pdc.cancer.gov\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1108 - INFO - Processing dataset 12 with repo: pdc.cancer.gov and accession_id: PDC000153\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1134 - INFO - Dataset page: https://proteomic.datacommons.cancer.gov/pdc/study/PDC000153\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1141 - INFO - Adding access mode for dataset 12: Complex download or Application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1083 - INFO - Processing dataset 13 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage', 'source_section', 'retrieval_pattern'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1094 - INFO - Raw accession ID: PXD043612\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Resolving data repository for candidate: www.ebi.ac.uk, identifier: PXD043612\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1032 - INFO - Exact match found for repo: www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 996 - INFO - Resolving accession ID for PXD043612 in www.ebi.ac.uk\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1108 - INFO - Processing dataset 13 with repo: www.ebi.ac.uk and accession_id: PXD043612\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1134 - INFO - Dataset page: https://www.ebi.ac.uk/pride/archive/projects/PXD043612\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1141 - INFO - Adding access mode for dataset 13: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1152 - INFO - Updated datasets len: 13\u001B[0m\n",
      "\u001B[97mxml_parser.py - line 222 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 23, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'source_section', 'retrieval_pattern', 'access_mode', 'link',\n",
      "       'source_url', 'download_link', 'title', 'content_type', 'id',\n",
      "       'surrounding_text', 'description'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97morchestrator.py - line 405 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97morchestrator.py - line 411 - INFO - XML element classification not needed. Using parsed_data.\u001B[0m\n",
      "\u001B[97morchestrator.py - line 477 - INFO - 2th function call: self.process_url(https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11425778)\u001B[0m\n",
      "\u001B[97morchestrator.py - line 307 - INFO - Processing URL: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11425778\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 172 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 23 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97morchestrator.py - line 314 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 498 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC11425778&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97morchestrator.py - line 335 - INFO - Fallback to HTML data fetcher for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11425778.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 180 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 181 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 182 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 184 - INFO - Initializing new selenium driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...\n",
      "To disable this warning, you can either:\n",
      "\t- Avoid using `tokenizers` before the fork if possible\n",
      "\t- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)\n",
      "huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...\n",
      "To disable this warning, you can either:\n",
      "\t- Avoid using `tokenizers` before the fork if possible\n",
      "\t- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)\n",
      "huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...\n",
      "To disable this warning, you can either:\n",
      "\t- Avoid using `tokenizers` before the fork if possible\n",
      "\t- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)\n",
      "huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...\n",
      "To disable this warning, you can either:\n",
      "\t- Avoid using `tokenizers` before the fork if possible\n",
      "\t- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)\n",
      "huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...\n",
      "To disable this warning, you can either:\n",
      "\t- Avoid using `tokenizers` before the fork if possible\n",
      "\t- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)\n",
      "huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...\n",
      "To disable this warning, you can either:\n",
      "\t- Avoid using `tokenizers` before the fork if possible\n",
      "\t- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)\n",
      "huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...\n",
      "To disable this warning, you can either:\n",
      "\t- Avoid using `tokenizers` before the fork if possible\n",
      "\t- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)\n",
      "huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...\n",
      "To disable this warning, you can either:\n",
      "\t- Avoid using `tokenizers` before the fork if possible\n",
      "\t- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)\n",
      "huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...\n",
      "To disable this warning, you can either:\n",
      "\t- Avoid using `tokenizers` before the fork if possible\n",
      "\t- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)\n",
      "huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...\n",
      "To disable this warning, you can either:\n",
      "\t- Avoid using `tokenizers` before the fork if possible\n",
      "\t- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)\n",
      "huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...\n",
      "To disable this warning, you can either:\n",
      "\t- Avoid using `tokenizers` before the fork if possible\n",
      "\t- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)\n",
      "huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...\n",
      "To disable this warning, you can either:\n",
      "\t- Avoid using `tokenizers` before the fork if possible\n",
      "\t- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1ae627f90>.\u001B[0m\n",
      "huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...\n",
      "To disable this warning, you can either:\n",
      "\t- Avoid using `tokenizers` before the fork if possible\n",
      "\t- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)\n",
      "\u001B[97mdata_fetcher.py - line 23 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 353 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 288 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97morchestrator.py - line 351 - INFO - Raw HTML data fetched from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11425778 is ready for parsing.\u001B[0m\n",
      "\u001B[97morchestrator.py - line 365 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97morchestrator.py - line 389 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 181 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 82 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 345 - INFO - Function call: parse_data(api_data(html_str, PMC, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11425778, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 426 - INFO - Function_call: extract_href_from_html_supplementary_material(tree, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11425778)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 490 - INFO - Extracted 14 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 372 - INFO - Chunking the HTML content for the parsing step.\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 171 - INFO - Extracting data availability elements from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 182 - INFO - Using selector: section.data-availability-statement\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 182 - INFO - Using selector: section.associated-data\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 182 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Data Availability')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 182 - INFO - Using selector: h3.pmc_sec_title:-soup-contains('Data Availability')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 182 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Associated Data')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 182 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Footnotes')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 220 - INFO - Found 6 data availability elements from HTML.\u001B[0m\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "Batches:   0%|          | 0/2 [00:00<?, ?it/s]"
      ],
      "application/vnd.jupyter.widget-view+json": {
       "version_major": 2,
       "version_minor": 0,
       "model_id": "264435b44ebd4989b6c87c96fc652c29"
      }
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001B[97membeddings_retriever.py - line 51 - INFO - Searching for top-5 passages similar to the query by embeddings.\u001B[0m\n",
      "\u001B[97membeddings_retriever.py - line 33 - INFO - Computing L2 distances using numpy.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 432 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 32 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1193 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 440 - INFO - Content length: 14177\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1193 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 451 - INFO - Prompt messages total length: 3887 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 457 - INFO - Prompt ID: gemini-2.0-flash-0-78e11020bf4baeabead2cec314a21f4d275513c9f11c90bdceab9ad16b7b0147\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1193 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 476 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages: 3887 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 547 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-15d08896-5fe1-4bba-93c3-c36caf7d6dbb\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"MSV000092944\\\",\\n        \\\"data_repository\\\": \\\"MassIVE\\\",\\n        \\\"dataset_webpage\\\": \\\"https://massive.ucsd.edu/ProteoSAFe/QueryDatasets?id=MSV000092944\\\"\\n    }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"MSV000092944\\\",\\n        \\\"data_repository\\\": \\\"MassIVE\\\",\\n        \\\"dataset_webpage\\\": \\\"https://massive.ucsd.edu/ProteoSAFe/QueryDatasets?id=MSV000092944\\\"\\n    }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1751998871,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 5468,\n",
      "        \"completion_tokens\": 85,\n",
      "        \"total_tokens\": 5553,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 607 - INFO - Processing dataset: {'dataset_identifier': 'MSV000092944', 'data_repository': 'MassIVE', 'dataset_webpage': 'https://massive.ucsd.edu/ProteoSAFe/QueryDatasets?id=MSV000092944'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 622 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 954 - INFO - Validating accession ID: MSV000092944\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 965 - INFO - Accession ID MSV000092944 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Resolving data repository for candidate: MassIVE, identifier: MSV000092944\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1038 - INFO - Found repo_name match for MassIVE\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 972 - INFO - Validating Dataset Page: https://massive.ucsd.edu/ProteoSAFe/QueryDatasets?id=MSV000092944\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 985 - INFO - No dataset_webpage_url_ptr found for massive.ucsd.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 668 - INFO - Extracted dataset: {'dataset_identifier': 'MSV000092944', 'data_repository': 'massive.ucsd.edu', 'dataset_webpage': 'https://massive.ucsd.edu/ProteoSAFe/QueryDatasets?id=MSV000092944'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1075 - INFO - Fetching metadata for 1 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1083 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1094 - INFO - Raw accession ID: MSV000092944\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Resolving data repository for candidate: massive.ucsd.edu, identifier: MSV000092944\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1032 - INFO - Exact match found for repo: massive.ucsd.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 996 - INFO - Resolving accession ID for MSV000092944 in massive.ucsd.edu\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1108 - INFO - Processing dataset 1 with repo: massive.ucsd.edu and accession_id: MSV000092944\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1134 - INFO - Dataset page: https://massive.ucsd.edu/ProteoSAFe/dataset.jsp?accession=MSV000092944\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1141 - INFO - Adding access mode for dataset 1: Complex download\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1152 - INFO - Updated datasets len: 1\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 396 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 15, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 229 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 239 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 500 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97morchestrator.py - line 399 - INFO - Parsed data extraction completed. Elements collected: 3\u001B[0m\n",
      "\u001B[97morchestrator.py - line 405 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97morchestrator.py - line 415 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n",
      "\u001B[97morchestrator.py - line 477 - INFO - 3th function call: self.process_url(https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5313262)\u001B[0m\n",
      "\u001B[97morchestrator.py - line 307 - INFO - Processing URL: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5313262\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 172 - INFO - Initializing EntrezFetcher(('requests', 'self.config'))\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 23 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97morchestrator.py - line 314 - INFO - Type of data_fetcher EntrezFetcher\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 498 - INFO - Fetching data from request: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pmc&id=PMC5313262&retmode=xml\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 446 - INFO - ----Checking for data_availability_sections section in raw data.\u001B[0m\n",
      "\u001B[97mxml_retriever.py - line 59 - INFO - Missing section in XML: data_availability_sections\u001B[0m\n",
      "\u001B[97morchestrator.py - line 335 - INFO - Fallback to HTML data fetcher for https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5313262.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 180 - INFO - WebScraper instance: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 181 - INFO - EntrezFetcher instance: True\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 182 - INFO - scraper_tool attribute: False\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 184 - INFO - Initializing new selenium driver.\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 11 - INFO - Creating WebDriver for browser: firefox, with driver located at driver_path: None\u001B[0m\n",
      "\u001B[97mselenium_setup.py - line 41 - INFO - No driver path provided, using GeckoDriverManager to auto-install Firefox driver.\u001B[0m\n",
      "huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...\n",
      "To disable this warning, you can either:\n",
      "\t- Avoid using `tokenizers` before the fork if possible\n",
      "\t- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)\n",
      "huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...\n",
      "To disable this warning, you can either:\n",
      "\t- Avoid using `tokenizers` before the fork if possible\n",
      "\t- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)\n",
      "huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...\n",
      "To disable this warning, you can either:\n",
      "\t- Avoid using `tokenizers` before the fork if possible\n",
      "\t- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)\n",
      "huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...\n",
      "To disable this warning, you can either:\n",
      "\t- Avoid using `tokenizers` before the fork if possible\n",
      "\t- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)\n",
      "huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...\n",
      "To disable this warning, you can either:\n",
      "\t- Avoid using `tokenizers` before the fork if possible\n",
      "\t- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)\n",
      "huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...\n",
      "To disable this warning, you can either:\n",
      "\t- Avoid using `tokenizers` before the fork if possible\n",
      "\t- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)\n",
      "huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...\n",
      "To disable this warning, you can either:\n",
      "\t- Avoid using `tokenizers` before the fork if possible\n",
      "\t- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)\n",
      "huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...\n",
      "To disable this warning, you can either:\n",
      "\t- Avoid using `tokenizers` before the fork if possible\n",
      "\t- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)\n",
      "huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...\n",
      "To disable this warning, you can either:\n",
      "\t- Avoid using `tokenizers` before the fork if possible\n",
      "\t- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)\n",
      "huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...\n",
      "To disable this warning, you can either:\n",
      "\t- Avoid using `tokenizers` before the fork if possible\n",
      "\t- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)\n",
      "huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...\n",
      "To disable this warning, you can either:\n",
      "\t- Avoid using `tokenizers` before the fork if possible\n",
      "\t- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)\n",
      "huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...\n",
      "To disable this warning, you can either:\n",
      "\t- Avoid using `tokenizers` before the fork if possible\n",
      "\t- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)\n",
      "\u001B[97mselenium_setup.py - line 43 - INFO - Using GeckoDriverManager to auto-install Firefox driver <selenium.webdriver.firefox.service.Service object at 0x1ae627f90>.\u001B[0m\n",
      "huggingface/tokenizers: The current process just got forked, after parallelism has already been used. Disabling parallelism to avoid deadlocks...\n",
      "To disable this warning, you can either:\n",
      "\t- Avoid using `tokenizers` before the fork if possible\n",
      "\t- Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false)\n",
      "\u001B[97mdata_fetcher.py - line 23 - WARNING - DataFetcher raw_HTML_data_filepath set no None\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 353 - WARNING - Publication name not found in the page title.\u001B[0m\n",
      "\u001B[97mdata_fetcher.py - line 288 - INFO - No cookie pattern 1 found in HTML\u001B[0m\n",
      "\u001B[97morchestrator.py - line 351 - INFO - Raw HTML data fetched from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5313262 is ready for parsing.\u001B[0m\n",
      "\u001B[97morchestrator.py - line 365 - INFO - Skipping raw HTML/XML saving.\u001B[0m\n",
      "\u001B[97morchestrator.py - line 389 - INFO - Using HTMLParser to parse data.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 181 - INFO - LLMParser initialized.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 82 - INFO - Initializing htmlRetriever\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 345 - INFO - Function call: parse_data(api_data(html_str, PMC, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5313262, additional_data, HTML)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 426 - INFO - Function_call: extract_href_from_html_supplementary_material(tree, https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5313262)\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 490 - INFO - Extracted 4 unique supplementary material links from HTML.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 372 - INFO - Chunking the HTML content for the parsing step.\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 171 - INFO - Extracting data availability elements from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 182 - INFO - Using selector: section.data-availability-statement\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 182 - INFO - Using selector: section.associated-data\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 182 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Data Availability')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 182 - INFO - Using selector: h3.pmc_sec_title:-soup-contains('Data Availability')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 182 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Associated Data')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 182 - INFO - Using selector: h2.pmc_sec_title:-soup-contains('Footnotes')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 220 - INFO - Found 3 data availability elements from HTML.\u001B[0m\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "Batches:   0%|          | 0/1 [00:00<?, ?it/s]"
      ],
      "application/vnd.jupyter.widget-view+json": {
       "version_major": 2,
       "version_minor": 0,
       "model_id": "307d09a2ca084b1099937d61df4a91d9"
      }
     },
     "metadata": {},
     "output_type": "display_data"
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\u001B[97membeddings_retriever.py - line 51 - INFO - Searching for top-5 passages similar to the query by embeddings.\u001B[0m\n",
      "\u001B[97membeddings_retriever.py - line 33 - INFO - Computing L2 distances using numpy.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 432 - INFO - Function_call: extract_datasets_info_from_content(...)\u001B[0m\n",
      "\u001B[97mprompt_manager.py - line 32 - INFO - Loading prompt: retrieve_datasets_simple_JSON from user_prompt_dir: None, subdir: \u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1193 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 440 - INFO - Content length: 9445\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1193 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 451 - INFO - Prompt messages total length: 2704 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 457 - INFO - Prompt ID: gemini-2.0-flash-0-1d1254812faffbb2b822c78d65a8fee6d5d390abd440b6774957b175be52d8f5\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1193 - INFO - Expected string but got list. Converting list to string.\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 476 - INFO - Requesting datasets from content using model: gemini-2.0-flash, temperature: 0, messages: 2704 tokens\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 547 - INFO - Portkey Gemini response: {\n",
      "    \"id\": \"portkey-fed7c35d-4652-4626-a237-24e75fcb6e87\",\n",
      "    \"choices\": [\n",
      "        {\n",
      "            \"finish_reason\": \"STOP\",\n",
      "            \"index\": 0,\n",
      "            \"logprobs\": null,\n",
      "            \"message\": {\n",
      "                \"content\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"phs001249.v1.p1\\\",\\n        \\\"data_repository\\\": \\\"dbGaP\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001249.v1.p1\\\"\\n    }\\n]\\n```\",\n",
      "                \"role\": \"assistant\",\n",
      "                \"function_call\": null,\n",
      "                \"tool_calls\": null,\n",
      "                \"refusal\": null,\n",
      "                \"audio\": null,\n",
      "                \"content_blocks\": [\n",
      "                    {\n",
      "                        \"type\": \"text\",\n",
      "                        \"text\": \"```json\\n[\\n    {\\n        \\\"dataset_identifier\\\": \\\"phs001249.v1.p1\\\",\\n        \\\"data_repository\\\": \\\"dbGaP\\\",\\n        \\\"dataset_webpage\\\": \\\"https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001249.v1.p1\\\"\\n    }\\n]\\n```\"\n",
      "                    }\n",
      "                ]\n",
      "            }\n",
      "        }\n",
      "    ],\n",
      "    \"created\": 1752012982,\n",
      "    \"model\": \"gemini-2.0-flash\",\n",
      "    \"object\": \"chat.completion\",\n",
      "    \"system_fingerprint\": null,\n",
      "    \"usage\": {\n",
      "        \"prompt_tokens\": 3319,\n",
      "        \"completion_tokens\": 99,\n",
      "        \"total_tokens\": 3418,\n",
      "        \"completion_tokens_details\": {\n",
      "            \"accepted_prediction_tokens\": null,\n",
      "            \"audio_tokens\": null,\n",
      "            \"reasoning_tokens\": 0,\n",
      "            \"rejected_prediction_tokens\": null\n",
      "        },\n",
      "        \"prompt_tokens_details\": null\n",
      "    },\n",
      "    \"service_tier\": null,\n",
      "    \"provider\": \"vertex-ai\"\n",
      "}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 607 - INFO - Processing dataset: {'dataset_identifier': 'phs001249.v1.p1', 'data_repository': 'dbGaP', 'dataset_webpage': 'https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001249.v1.p1'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 622 - INFO - Dataset is a dictionary\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 954 - INFO - Validating accession ID: phs001249.v1.p1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 965 - INFO - Accession ID phs001249.v1.p1 is valid\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Resolving data repository for candidate: dbGaP, identifier: phs001249.v1.p1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1038 - INFO - Found repo_name match for dbGaP\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 972 - INFO - Validating Dataset Page: https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001249.v1.p1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 982 - INFO - Link does not match the pattern https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=__ID__\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 668 - INFO - Extracted dataset: {'dataset_identifier': 'phs001249.v1.p1', 'data_repository': 'dbgap', 'dataset_webpage': 'n/a'}\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1075 - INFO - Fetching metadata for 1 datasets\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1083 - INFO - Processing dataset 1 with keys: dict_keys(['dataset_identifier', 'data_repository', 'dataset_webpage'])\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1094 - INFO - Raw accession ID: phs001249.v1.p1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1014 - INFO - Resolving data repository for candidate: dbgap, identifier: phs001249.v1.p1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1032 - INFO - Exact match found for repo: dbgap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 996 - INFO - Resolving accession ID for phs001249.v1.p1 in dbgap\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1108 - INFO - Processing dataset 1 with repo: dbgap and accession_id: phs001249.v1.p1.v1.p1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1134 - INFO - Dataset page: https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001249.v1.p1.v1.p1\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1141 - INFO - Adding access mode for dataset 1: Application to access\u001B[0m\n",
      "\u001B[97mbase_parser.py - line 1152 - INFO - Updated datasets len: 1\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 396 - INFO - Dataset Links type: <class 'pandas.core.frame.DataFrame'> of len 5, with cols: Index(['dataset_identifier', 'data_repository', 'dataset_webpage',\n",
      "       'access_mode', 'link', 'title', 'file_info', 'description',\n",
      "       'source_section', 'section_class', 'download_link', 'file_extension',\n",
      "       'a_attr_href', 'a_attr_class', 'a_attr_data-ga-action', 'a_attr_target',\n",
      "       'a_attr_rel'],\n",
      "      dtype='object')\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 229 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97mhtml_retriever.py - line 239 - WARNING - No publication title found in the HTML data.\u001B[0m\n",
      "\u001B[97mhtml_parser.py - line 500 - INFO - Extracting publication title from HTML\u001B[0m\n",
      "\u001B[97morchestrator.py - line 399 - INFO - Parsed data extraction completed. Elements collected: 4\u001B[0m\n",
      "\u001B[97morchestrator.py - line 405 - INFO - Raw Data parsing completed.\u001B[0m\n",
      "\u001B[97morchestrator.py - line 415 - INFO - HTML element classification not supported. Using parsed_data.\u001B[0m\n"
     ]
    }
   ],
   "execution_count": 5
  },
  {
   "metadata": {
    "ExecuteTime": {
     "end_time": "2025-07-10T15:29:05.735978Z",
     "start_time": "2025-07-10T15:29:05.697763Z"
    }
   },
   "cell_type": "code",
   "source": "parsed_data",
   "id": "4e3896522649a12a",
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8628860':    dataset_identifier           data_repository dataset_webpage  \\\n",
       " 0                 n/a  data.humantumoratlas.org             n/a   \n",
       " 0                 NaN                       NaN             NaN   \n",
       " 14                NaN                       NaN             NaN   \n",
       " 15                NaN                       NaN             NaN   \n",
       " 16                NaN                       NaN             NaN   \n",
       " 17                NaN                       NaN             NaN   \n",
       " 18                NaN                       NaN             NaN   \n",
       " 19                NaN                       NaN             NaN   \n",
       " 20                NaN                       NaN             NaN   \n",
       " 21                NaN                       NaN             NaN   \n",
       " 22                NaN                       NaN             NaN   \n",
       " 23                NaN                       NaN             NaN   \n",
       " 24                NaN                       NaN             NaN   \n",
       " 25                NaN                       NaN             NaN   \n",
       " 26                NaN                       NaN             NaN   \n",
       " 27                NaN                       NaN             NaN   \n",
       " 28                NaN                       NaN             NaN   \n",
       " 29                NaN                       NaN             NaN   \n",
       " 30                NaN                       NaN             NaN   \n",
       " 31                NaN                       NaN             NaN   \n",
       " 32                NaN                       NaN             NaN   \n",
       " 33                NaN                       NaN             NaN   \n",
       " 34                NaN                       NaN             NaN   \n",
       " 35                NaN                       NaN             NaN   \n",
       " 36                NaN                       NaN             NaN   \n",
       " 37                NaN                       NaN             NaN   \n",
       " 38                NaN                       NaN             NaN   \n",
       " 39                NaN                       NaN             NaN   \n",
       " 40                NaN                       NaN             NaN   \n",
       " \n",
       "                                                  link  \\\n",
       " 0                                                 NaN   \n",
       " 0         https://doi.org/10.1016/j.ccell.2021.09.008   \n",
       " 14  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 15  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 16  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 17  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 18  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 19  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 20  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 21  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 22  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 23  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 24  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 25  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 26  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 27  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 28  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 29  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 30  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 31  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 32  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 33  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 34  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 35  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 36  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 37  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 38  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 39  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 40  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " \n",
       "                               title        file_info description  \\\n",
       " 0                               NaN              NaN         NaN   \n",
       " 0       10.1016/j.ccell.2021.09.008              n/a         n/a   \n",
       " 14    NIHMS1754019-supplement-1.pdf     (9.9MB, pdf)         n/a   \n",
       " 15   NIHMS1754019-supplement-2.xlsx   (16.4KB, xlsx)         n/a   \n",
       " 16   NIHMS1754019-supplement-3.xlsx   (10.1KB, xlsx)         n/a   \n",
       " 17   NIHMS1754019-supplement-4.xlsx   (20.7KB, xlsx)         n/a   \n",
       " 18   NIHMS1754019-supplement-5.xlsx   (15.8KB, xlsx)         n/a   \n",
       " 19   NIHMS1754019-supplement-6.xlsx   (16.5KB, xlsx)         n/a   \n",
       " 20   NIHMS1754019-supplement-7.xlsx   (37.4KB, xlsx)         n/a   \n",
       " 21   NIHMS1754019-supplement-8.xlsx   (50.6KB, xlsx)         n/a   \n",
       " 22   NIHMS1754019-supplement-9.xlsx   (56.3KB, xlsx)         n/a   \n",
       " 23  NIHMS1754019-supplement-10.xlsx   (17.5KB, xlsx)         n/a   \n",
       " 24  NIHMS1754019-supplement-11.xlsx   (22.3KB, xlsx)         n/a   \n",
       " 25  NIHMS1754019-supplement-12.xlsx  (183.6KB, xlsx)         n/a   \n",
       " 26  NIHMS1754019-supplement-13.xlsx   (10.1KB, xlsx)         n/a   \n",
       " 27  NIHMS1754019-supplement-14.xlsx   (11.5KB, xlsx)         n/a   \n",
       " 28  NIHMS1754019-supplement-15.xlsx   (14.1KB, xlsx)         n/a   \n",
       " 29  NIHMS1754019-supplement-16.xlsx   (11.5KB, xlsx)         n/a   \n",
       " 30  NIHMS1754019-supplement-17.xlsx   (10.8KB, xlsx)         n/a   \n",
       " 31  NIHMS1754019-supplement-18.xlsx     (11KB, xlsx)         n/a   \n",
       " 32  NIHMS1754019-supplement-19.xlsx     (14KB, xlsx)         n/a   \n",
       " 33  NIHMS1754019-supplement-20.xlsx    (1.5MB, xlsx)         n/a   \n",
       " 34  NIHMS1754019-supplement-21.xlsx  (147.8KB, xlsx)         n/a   \n",
       " 35  NIHMS1754019-supplement-22.xlsx  (126.8KB, xlsx)         n/a   \n",
       " 36  NIHMS1754019-supplement-23.xlsx  (142.4KB, xlsx)         n/a   \n",
       " 37  NIHMS1754019-supplement-24.xlsx    (126KB, xlsx)         n/a   \n",
       " 38  NIHMS1754019-supplement-25.xlsx   (53.4KB, xlsx)         n/a   \n",
       " 39   NIHMS1754019-supplement-26.gmt     (1.1MB, gmt)         n/a   \n",
       " 40   NIHMS1754019-supplement-27.gmt    (19.8KB, gmt)         n/a   \n",
       " \n",
       "    source_section                                section_class  \\\n",
       " 0             NaN                                          NaN   \n",
       " 0             n/a  pmc-layout__citation font-secondary font-xs   \n",
       " 14       d40e3618                                      media p   \n",
       " 15       d40e3630                                      media p   \n",
       " 16       d40e3639                                      media p   \n",
       " 17       d40e3651                                      media p   \n",
       " 18       d40e3663                                      media p   \n",
       " 19       d40e3675                                      media p   \n",
       " 20       d40e3687                                      media p   \n",
       " 21       d40e3699                                      media p   \n",
       " 22       d40e3711                                      media p   \n",
       " 23       d40e3726                                      media p   \n",
       " 24       d40e3739                                      media p   \n",
       " 25       d40e3754                                      media p   \n",
       " 26       d40e3769                                      media p   \n",
       " 27       d40e3781                                      media p   \n",
       " 28       d40e3802                                      media p   \n",
       " 29       d40e3823                                      media p   \n",
       " 30       d40e3838                                      media p   \n",
       " 31       d40e3847                                      media p   \n",
       " 32       d40e3856                                      media p   \n",
       " 33       d40e3865                                      media p   \n",
       " 34       d40e3880                                      media p   \n",
       " 35       d40e3896                                      media p   \n",
       " 36       d40e3911                                      media p   \n",
       " 37       d40e3926                                      media p   \n",
       " 38       d40e3935                                      media p   \n",
       " 39       d40e3944                                      media p   \n",
       " 40       d40e3953                                      media p   \n",
       " \n",
       "                                         download_link file_extension  \\\n",
       " 0                                                 NaN           None   \n",
       " 0                                                None           None   \n",
       " 14  https://www.ncbi.nlm.nih.gov/pmc/articles/inst...            pdf   \n",
       " 15  https://www.ncbi.nlm.nih.gov/pmc/articles/inst...           xlsx   \n",
       " 16  https://www.ncbi.nlm.nih.gov/pmc/articles/inst...           xlsx   \n",
       " 17  https://www.ncbi.nlm.nih.gov/pmc/articles/inst...           xlsx   \n",
       " 18  https://www.ncbi.nlm.nih.gov/pmc/articles/inst...           xlsx   \n",
       " 19  https://www.ncbi.nlm.nih.gov/pmc/articles/inst...           xlsx   \n",
       " 20  https://www.ncbi.nlm.nih.gov/pmc/articles/inst...           xlsx   \n",
       " 21  https://www.ncbi.nlm.nih.gov/pmc/articles/inst...           xlsx   \n",
       " 22  https://www.ncbi.nlm.nih.gov/pmc/articles/inst...           xlsx   \n",
       " 23  https://www.ncbi.nlm.nih.gov/pmc/articles/inst...           xlsx   \n",
       " 24  https://www.ncbi.nlm.nih.gov/pmc/articles/inst...           xlsx   \n",
       " 25  https://www.ncbi.nlm.nih.gov/pmc/articles/inst...           xlsx   \n",
       " 26  https://www.ncbi.nlm.nih.gov/pmc/articles/inst...           xlsx   \n",
       " 27  https://www.ncbi.nlm.nih.gov/pmc/articles/inst...           xlsx   \n",
       " 28  https://www.ncbi.nlm.nih.gov/pmc/articles/inst...           xlsx   \n",
       " 29  https://www.ncbi.nlm.nih.gov/pmc/articles/inst...           xlsx   \n",
       " 30  https://www.ncbi.nlm.nih.gov/pmc/articles/inst...           xlsx   \n",
       " 31  https://www.ncbi.nlm.nih.gov/pmc/articles/inst...           xlsx   \n",
       " 32  https://www.ncbi.nlm.nih.gov/pmc/articles/inst...           xlsx   \n",
       " 33  https://www.ncbi.nlm.nih.gov/pmc/articles/inst...           xlsx   \n",
       " 34  https://www.ncbi.nlm.nih.gov/pmc/articles/inst...           xlsx   \n",
       " 35  https://www.ncbi.nlm.nih.gov/pmc/articles/inst...           xlsx   \n",
       " 36  https://www.ncbi.nlm.nih.gov/pmc/articles/inst...           xlsx   \n",
       " 37  https://www.ncbi.nlm.nih.gov/pmc/articles/inst...           xlsx   \n",
       " 38  https://www.ncbi.nlm.nih.gov/pmc/articles/inst...           xlsx   \n",
       " 39  https://www.ncbi.nlm.nih.gov/pmc/articles/inst...            gmt   \n",
       " 40  https://www.ncbi.nlm.nih.gov/pmc/articles/inst...            gmt   \n",
       " \n",
       "                                           a_attr_href  \\\n",
       " 0                                                 NaN   \n",
       " 0         https://doi.org/10.1016/j.ccell.2021.09.008   \n",
       " 14  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 15  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 16  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 17  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 18  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 19  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 20  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 21  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 22  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 23  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 24  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 25  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 26  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 27  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 28  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 29  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 30  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 31  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 32  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 33  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 34  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 35  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 36  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 37  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 38  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 39  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " 40  /articles/instance/8628860/bin/NIHMS1754019-su...   \n",
       " \n",
       "                    a_attr_class a_attr_data-ga-action a_attr_target  \\\n",
       " 0                           NaN                   NaN           NaN   \n",
       " 0   usa-link usa-link--external      click_feat_suppl        _blank   \n",
       " 14                     usa-link      click_feat_suppl           NaN   \n",
       " 15                     usa-link      click_feat_suppl           NaN   \n",
       " 16                     usa-link      click_feat_suppl           NaN   \n",
       " 17                     usa-link      click_feat_suppl           NaN   \n",
       " 18                     usa-link      click_feat_suppl           NaN   \n",
       " 19                     usa-link      click_feat_suppl           NaN   \n",
       " 20                     usa-link      click_feat_suppl           NaN   \n",
       " 21                     usa-link      click_feat_suppl           NaN   \n",
       " 22                     usa-link      click_feat_suppl           NaN   \n",
       " 23                     usa-link      click_feat_suppl           NaN   \n",
       " 24                     usa-link      click_feat_suppl           NaN   \n",
       " 25                     usa-link      click_feat_suppl           NaN   \n",
       " 26                     usa-link      click_feat_suppl           NaN   \n",
       " 27                     usa-link      click_feat_suppl           NaN   \n",
       " 28                     usa-link      click_feat_suppl           NaN   \n",
       " 29                     usa-link      click_feat_suppl           NaN   \n",
       " 30                     usa-link      click_feat_suppl           NaN   \n",
       " 31                     usa-link      click_feat_suppl           NaN   \n",
       " 32                     usa-link      click_feat_suppl           NaN   \n",
       " 33                     usa-link      click_feat_suppl           NaN   \n",
       " 34                     usa-link      click_feat_suppl           NaN   \n",
       " 35                     usa-link      click_feat_suppl           NaN   \n",
       " 36                     usa-link      click_feat_suppl           NaN   \n",
       " 37                     usa-link      click_feat_suppl           NaN   \n",
       " 38                     usa-link      click_feat_suppl           NaN   \n",
       " 39                     usa-link      click_feat_suppl           NaN   \n",
       " 40                     usa-link      click_feat_suppl           NaN   \n",
       " \n",
       "              a_attr_rel                                         source_url  \\\n",
       " 0                   NaN  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8...   \n",
       " 0   noopener noreferrer  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8...   \n",
       " 14                  NaN  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8...   \n",
       " 15                  NaN  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8...   \n",
       " 16                  NaN  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8...   \n",
       " 17                  NaN  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8...   \n",
       " 18                  NaN  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8...   \n",
       " 19                  NaN  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8...   \n",
       " 20                  NaN  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8...   \n",
       " 21                  NaN  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8...   \n",
       " 22                  NaN  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8...   \n",
       " 23                  NaN  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8...   \n",
       " 24                  NaN  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8...   \n",
       " 25                  NaN  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8...   \n",
       " 26                  NaN  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8...   \n",
       " 27                  NaN  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8...   \n",
       " 28                  NaN  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8...   \n",
       " 29                  NaN  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8...   \n",
       " 30                  NaN  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8...   \n",
       " 31                  NaN  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8...   \n",
       " 32                  NaN  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8...   \n",
       " 33                  NaN  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8...   \n",
       " 34                  NaN  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8...   \n",
       " 35                  NaN  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8...   \n",
       " 36                  NaN  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8...   \n",
       " 37                  NaN  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8...   \n",
       " 38                  NaN  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8...   \n",
       " 39                  NaN  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8...   \n",
       " 40                  NaN  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8...   \n",
       " \n",
       "                                             pub_title  \n",
       " 0   Signatures of plasticity, metastasis, and immu...  \n",
       " 0   Signatures of plasticity, metastasis, and immu...  \n",
       " 14  Signatures of plasticity, metastasis, and immu...  \n",
       " 15  Signatures of plasticity, metastasis, and immu...  \n",
       " 16  Signatures of plasticity, metastasis, and immu...  \n",
       " 17  Signatures of plasticity, metastasis, and immu...  \n",
       " 18  Signatures of plasticity, metastasis, and immu...  \n",
       " 19  Signatures of plasticity, metastasis, and immu...  \n",
       " 20  Signatures of plasticity, metastasis, and immu...  \n",
       " 21  Signatures of plasticity, metastasis, and immu...  \n",
       " 22  Signatures of plasticity, metastasis, and immu...  \n",
       " 23  Signatures of plasticity, metastasis, and immu...  \n",
       " 24  Signatures of plasticity, metastasis, and immu...  \n",
       " 25  Signatures of plasticity, metastasis, and immu...  \n",
       " 26  Signatures of plasticity, metastasis, and immu...  \n",
       " 27  Signatures of plasticity, metastasis, and immu...  \n",
       " 28  Signatures of plasticity, metastasis, and immu...  \n",
       " 29  Signatures of plasticity, metastasis, and immu...  \n",
       " 30  Signatures of plasticity, metastasis, and immu...  \n",
       " 31  Signatures of plasticity, metastasis, and immu...  \n",
       " 32  Signatures of plasticity, metastasis, and immu...  \n",
       " 33  Signatures of plasticity, metastasis, and immu...  \n",
       " 34  Signatures of plasticity, metastasis, and immu...  \n",
       " 35  Signatures of plasticity, metastasis, and immu...  \n",
       " 36  Signatures of plasticity, metastasis, and immu...  \n",
       " 37  Signatures of plasticity, metastasis, and immu...  \n",
       " 38  Signatures of plasticity, metastasis, and immu...  \n",
       " 39  Signatures of plasticity, metastasis, and immu...  \n",
       " 40  Signatures of plasticity, metastasis, and immu...  ,\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11129317':        dataset_identifier data_repository  \\\n",
       " 0               PXD043612   www.ebi.ac.uk   \n",
       " 1   10.17632/3wfxrz66w2.1         doi.org   \n",
       " 2   10.17632/bvdn865y9c.1         doi.org   \n",
       " 3               PDC000234  pdc.cancer.gov   \n",
       " 4               PDC000127  pdc.cancer.gov   \n",
       " 5               PDC000204  pdc.cancer.gov   \n",
       " 6               PDC000221  pdc.cancer.gov   \n",
       " 7               PDC000198  pdc.cancer.gov   \n",
       " 8               PDC000110  pdc.cancer.gov   \n",
       " 9               PDC000270  pdc.cancer.gov   \n",
       " 10              PDC000125  pdc.cancer.gov   \n",
       " 11              PDC000153  pdc.cancer.gov   \n",
       " 0                     NaN             NaN   \n",
       " 1                     NaN             NaN   \n",
       " 2                     NaN             NaN   \n",
       " 3                     NaN             NaN   \n",
       " 4                     NaN             NaN   \n",
       " 5                     NaN             NaN   \n",
       " 6                     NaN             NaN   \n",
       " 7                     NaN             NaN   \n",
       " 8                     NaN             NaN   \n",
       " 9                     NaN             NaN   \n",
       " \n",
       "                                       dataset_webpage          source_section  \\\n",
       " 0   https://www.ebi.ac.uk/pride/archive/projects/P...       data_availability   \n",
       " 1               https://doi.org/10.17632/3wfxrz66w2.1       data_availability   \n",
       " 2               https://doi.org/10.17632/bvdn865y9c.1       data_availability   \n",
       " 3   https://proteomic.datacommons.cancer.gov/pdc/s...       data_availability   \n",
       " 4   https://proteomic.datacommons.cancer.gov/pdc/s...       data_availability   \n",
       " 5   https://proteomic.datacommons.cancer.gov/pdc/s...       data_availability   \n",
       " 6   https://proteomic.datacommons.cancer.gov/pdc/s...       data_availability   \n",
       " 7   https://proteomic.datacommons.cancer.gov/pdc/s...       data_availability   \n",
       " 8   https://proteomic.datacommons.cancer.gov/pdc/s...       data_availability   \n",
       " 9   https://proteomic.datacommons.cancer.gov/pdc/s...       data_availability   \n",
       " 10  https://proteomic.datacommons.cancer.gov/pdc/s...       data_availability   \n",
       " 11  https://proteomic.datacommons.cancer.gov/pdc/s...       data_availability   \n",
       " 0                                                 NaN  supplementary material   \n",
       " 1                                                 NaN  supplementary material   \n",
       " 2                                                 NaN  supplementary material   \n",
       " 3                                                 NaN  supplementary material   \n",
       " 4                                                 NaN  supplementary material   \n",
       " 5                                                 NaN  supplementary material   \n",
       " 6                                                 NaN  supplementary material   \n",
       " 7                                                 NaN  supplementary material   \n",
       " 8                                                 NaN  supplementary material   \n",
       " 9                                                 NaN  supplementary material   \n",
       " \n",
       "             retrieval_pattern                                access_mode  \\\n",
       " 0           data availability                           Complex download   \n",
       " 1           data availability                                        NaN   \n",
       " 2           data availability                                        NaN   \n",
       " 3           data availability  Complex download or Application to access   \n",
       " 4           data availability  Complex download or Application to access   \n",
       " 5           data availability  Complex download or Application to access   \n",
       " 6           data availability  Complex download or Application to access   \n",
       " 7           data availability  Complex download or Application to access   \n",
       " 8           data availability  Complex download or Application to access   \n",
       " 9           data availability  Complex download or Application to access   \n",
       " 10          data availability  Complex download or Application to access   \n",
       " 11          data availability  Complex download or Application to access   \n",
       " 0   .//supplementary-material                                        NaN   \n",
       " 1   .//supplementary-material                                        NaN   \n",
       " 2   .//supplementary-material                                        NaN   \n",
       " 3   .//supplementary-material                                        NaN   \n",
       " 4   .//supplementary-material                                        NaN   \n",
       " 5   .//supplementary-material                                        NaN   \n",
       " 6   .//supplementary-material                                        NaN   \n",
       " 7   .//supplementary-material                                        NaN   \n",
       " 8   .//supplementary-material                                        NaN   \n",
       " 9   .//supplementary-material                                        NaN   \n",
       " \n",
       "          link                                         source_url  \\\n",
       " 0         NaN                                                NaN   \n",
       " 1         NaN                                                NaN   \n",
       " 2         NaN                                                NaN   \n",
       " 3         NaN                                                NaN   \n",
       " 4         NaN                                                NaN   \n",
       " 5         NaN                                                NaN   \n",
       " 6         NaN                                                NaN   \n",
       " 7         NaN                                                NaN   \n",
       " 8         NaN                                                NaN   \n",
       " 9         NaN                                                NaN   \n",
       " 10        NaN                                                NaN   \n",
       " 11        NaN                                                NaN   \n",
       " 0    mmc1.pdf  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...   \n",
       " 1    mmc2.zip  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...   \n",
       " 2   mmc3.xlsx  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...   \n",
       " 3   mmc4.xlsx  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...   \n",
       " 4   mmc5.xlsx  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...   \n",
       " 5   mmc6.xlsx  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...   \n",
       " 6   mmc7.xlsx  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...   \n",
       " 7   mmc8.xlsx  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...   \n",
       " 8   mmc9.xlsx  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...   \n",
       " 9   mmc10.pdf  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...   \n",
       " \n",
       "                                         download_link  \\\n",
       " 0                                                 NaN   \n",
       " 1                                                 NaN   \n",
       " 2                                                 NaN   \n",
       " 3                                                 NaN   \n",
       " 4                                                 NaN   \n",
       " 5                                                 NaN   \n",
       " 6                                                 NaN   \n",
       " 7                                                 NaN   \n",
       " 8                                                 NaN   \n",
       " 9                                                 NaN   \n",
       " 10                                                NaN   \n",
       " 11                                                NaN   \n",
       " 0   https://pmc.ncbi.nlm.nih.gov/articles/instance...   \n",
       " 1   https://pmc.ncbi.nlm.nih.gov/articles/instance...   \n",
       " 2   https://pmc.ncbi.nlm.nih.gov/articles/instance...   \n",
       " 3   https://pmc.ncbi.nlm.nih.gov/articles/instance...   \n",
       " 4   https://pmc.ncbi.nlm.nih.gov/articles/instance...   \n",
       " 5   https://pmc.ncbi.nlm.nih.gov/articles/instance...   \n",
       " 6   https://pmc.ncbi.nlm.nih.gov/articles/instance...   \n",
       " 7   https://pmc.ncbi.nlm.nih.gov/articles/instance...   \n",
       " 8   https://pmc.ncbi.nlm.nih.gov/articles/instance...   \n",
       " 9   https://pmc.ncbi.nlm.nih.gov/articles/instance...   \n",
       " \n",
       "                                                 title content_type     id  \\\n",
       " 0                                                 NaN          NaN    NaN   \n",
       " 1                                                 NaN          NaN    NaN   \n",
       " 2                                                 NaN          NaN    NaN   \n",
       " 3                                                 NaN          NaN    NaN   \n",
       " 4                                                 NaN          NaN    NaN   \n",
       " 5                                                 NaN          NaN    NaN   \n",
       " 6                                                 NaN          NaN    NaN   \n",
       " 7                                                 NaN          NaN    NaN   \n",
       " 8                                                 NaN          NaN    NaN   \n",
       " 9                                                 NaN          NaN    NaN   \n",
       " 10                                                NaN          NaN    NaN   \n",
       " 11                                                NaN          NaN    NaN   \n",
       " 0                          Document S1. Figures S1–S6   local-data   mmc1   \n",
       " 1   File S1. Detailed Philosopher parameters for s...   local-data   mmc2   \n",
       " 2   Table S1. Identified arginine to all other ami...   local-data   mmc3   \n",
       " 3   Table S2. Arginine-to-cysteine substitutants w...   local-data   mmc4   \n",
       " 4   Table S3. Arginine substitutants codon-specifi...   local-data   mmc5   \n",
       " 5   Table S4. Proteomic analysis of MDA-MB-231 cel...   local-data   mmc6   \n",
       " 6   Table S5. Ontology-enrichment analysis of argi...   local-data   mmc7   \n",
       " 7   Table S6. Proteomics analysis of A549 cells tr...   local-data   mmc8   \n",
       " 8   Table S7. DNA fragments of the plasmids used i...   local-data   mmc9   \n",
       " 9   Document S2. Article plus supplemental informa...   local-data  mmc10   \n",
       " \n",
       "                                      surrounding_text     description  \\\n",
       " 0                                                 NaN             NaN   \n",
       " 1                                                 NaN             NaN   \n",
       " 2                                                 NaN             NaN   \n",
       " 3                                                 NaN             NaN   \n",
       " 4                                                 NaN             NaN   \n",
       " 5                                                 NaN             NaN   \n",
       " 6                                                 NaN             NaN   \n",
       " 7                                                 NaN             NaN   \n",
       " 8                                                 NaN             NaN   \n",
       " 9                                                 NaN             NaN   \n",
       " 10                                                NaN             NaN   \n",
       " 11                                                NaN             NaN   \n",
       " 0                       Document  S1.  Figures  S1–S6  No description   \n",
       " 1   File  S1.  Detailed  Philosopher  parameters  ...  No description   \n",
       " 2   Table  S1.  Identified  arginine  to  all  oth...  No description   \n",
       " 3   Table  S2.  Arginine-to-cysteine  substitutant...  No description   \n",
       " 4   Table  S3.  Arginine  substitutants  codon-spe...  No description   \n",
       " 5   Table  S4.  Proteomic  analysis  of  MDA-MB-23...  No description   \n",
       " 6   Table  S5.  Ontology-enrichment  analysis  of ...  No description   \n",
       " 7   Table  S6.  Proteomics  analysis  of  A549  ce...  No description   \n",
       " 8   Table  S7.  DNA  fragments  of  the  plasmids ...  No description   \n",
       " 9   Document  S2.  Article  plus  supplemental  in...  No description   \n",
       " \n",
       "    file_extension                                          pub_title  \n",
       " 0            None  Arginine deprivation enriches lung cancer prot...  \n",
       " 1            None  Arginine deprivation enriches lung cancer prot...  \n",
       " 2            None  Arginine deprivation enriches lung cancer prot...  \n",
       " 3            None  Arginine deprivation enriches lung cancer prot...  \n",
       " 4            None  Arginine deprivation enriches lung cancer prot...  \n",
       " 5            None  Arginine deprivation enriches lung cancer prot...  \n",
       " 6            None  Arginine deprivation enriches lung cancer prot...  \n",
       " 7            None  Arginine deprivation enriches lung cancer prot...  \n",
       " 8            None  Arginine deprivation enriches lung cancer prot...  \n",
       " 9            None  Arginine deprivation enriches lung cancer prot...  \n",
       " 10           None  Arginine deprivation enriches lung cancer prot...  \n",
       " 11           None  Arginine deprivation enriches lung cancer prot...  \n",
       " 0             pdf  Arginine deprivation enriches lung cancer prot...  \n",
       " 1             zip  Arginine deprivation enriches lung cancer prot...  \n",
       " 2            xlsx  Arginine deprivation enriches lung cancer prot...  \n",
       " 3            xlsx  Arginine deprivation enriches lung cancer prot...  \n",
       " 4            xlsx  Arginine deprivation enriches lung cancer prot...  \n",
       " 5            xlsx  Arginine deprivation enriches lung cancer prot...  \n",
       " 6            xlsx  Arginine deprivation enriches lung cancer prot...  \n",
       " 7            xlsx  Arginine deprivation enriches lung cancer prot...  \n",
       " 8            xlsx  Arginine deprivation enriches lung cancer prot...  \n",
       " 9             pdf  Arginine deprivation enriches lung cancer prot...  ,\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11425778':   dataset_identifier   data_repository  \\\n",
       " 0       MSV000092944  massive.ucsd.edu   \n",
       " 0                NaN               NaN   \n",
       " 8                NaN               NaN   \n",
       " \n",
       "                                      dataset_webpage       access_mode  \\\n",
       " 0  https://massive.ucsd.edu/ProteoSAFe/dataset.js...  Complex download   \n",
       " 0                                                NaN               NaN   \n",
       " 8                                                NaN               NaN   \n",
       " \n",
       "                                                 link  \\\n",
       " 0                                                NaN   \n",
       " 0      https://doi.org/10.1021/acs.jproteome.4c00338   \n",
       " 8  /articles/instance/11425778/bin/NIHMS2020672-s...   \n",
       " \n",
       "                             title     file_info description source_section  \\\n",
       " 0                             NaN           NaN         NaN            NaN   \n",
       " 0   10.1021/acs.jproteome.4c00338           n/a         n/a            n/a   \n",
       " 8  NIHMS2020672-supplement-SI.pdf  (1.4MB, pdf)         n/a        d67e776   \n",
       " \n",
       "                                  section_class  \\\n",
       " 0                                          NaN   \n",
       " 0  pmc-layout__citation font-secondary font-xs   \n",
       " 8                                      media p   \n",
       " \n",
       "                                        download_link file_extension  \\\n",
       " 0                                                NaN           None   \n",
       " 0                                               None           None   \n",
       " 8  https://www.ncbi.nlm.nih.gov/pmc/articles/inst...            pdf   \n",
       " \n",
       "                                          a_attr_href  \\\n",
       " 0                                                NaN   \n",
       " 0      https://doi.org/10.1021/acs.jproteome.4c00338   \n",
       " 8  /articles/instance/11425778/bin/NIHMS2020672-s...   \n",
       " \n",
       "                   a_attr_class a_attr_data-ga-action a_attr_target  \\\n",
       " 0                          NaN                   NaN           NaN   \n",
       " 0  usa-link usa-link--external      click_feat_suppl        _blank   \n",
       " 8                     usa-link      click_feat_suppl           NaN   \n",
       " \n",
       "             a_attr_rel                                         source_url  \\\n",
       " 0                  NaN  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...   \n",
       " 0  noopener noreferrer  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...   \n",
       " 8                  NaN  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1...   \n",
       " \n",
       "                                            pub_title  \n",
       " 0  Proteomics Analysis of Interactions between Dr...  \n",
       " 0  Proteomics Analysis of Interactions between Dr...  \n",
       " 8  Proteomics Analysis of Interactions between Dr...  ,\n",
       " 'https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5313262':   dataset_identifier data_repository  \\\n",
       " 0    phs001249.v1.p1           dbgap   \n",
       " 0                NaN             NaN   \n",
       " 3                NaN             NaN   \n",
       " 4                NaN             NaN   \n",
       " \n",
       "                                      dataset_webpage            access_mode  \\\n",
       " 0  https://www.ncbi.nlm.nih.gov/projects/gap/cgi-...  Application to access   \n",
       " 0                                                NaN                    NaN   \n",
       " 3                                                NaN                    NaN   \n",
       " 4                                                NaN                    NaN   \n",
       " \n",
       "                                                 link  \\\n",
       " 0                                                NaN   \n",
       " 0        https://doi.org/10.1016/j.ccell.2017.01.006   \n",
       " 3  /articles/instance/5313262/bin/NIHMS845798-sup...   \n",
       " 4  /articles/instance/5313262/bin/NIHMS845798-sup...   \n",
       " \n",
       "                            title       file_info description source_section  \\\n",
       " 0                            NaN             NaN         NaN            NaN   \n",
       " 0    10.1016/j.ccell.2017.01.006             n/a         n/a            n/a   \n",
       " 3   NIHMS845798-supplement-1.pdf    (1.3MB, pdf)         n/a       d36e1100   \n",
       " 4  NIHMS845798-supplement-2.xlsx  (78.5KB, xlsx)         n/a       d36e1104   \n",
       " \n",
       "                                  section_class  \\\n",
       " 0                                          NaN   \n",
       " 0  pmc-layout__citation font-secondary font-xs   \n",
       " 3                                      media p   \n",
       " 4                                      media p   \n",
       " \n",
       "                                        download_link file_extension  \\\n",
       " 0                                                NaN           None   \n",
       " 0                                               None           None   \n",
       " 3  https://www.ncbi.nlm.nih.gov/pmc/articles/inst...            pdf   \n",
       " 4  https://www.ncbi.nlm.nih.gov/pmc/articles/inst...           xlsx   \n",
       " \n",
       "                                          a_attr_href  \\\n",
       " 0                                                NaN   \n",
       " 0        https://doi.org/10.1016/j.ccell.2017.01.006   \n",
       " 3  /articles/instance/5313262/bin/NIHMS845798-sup...   \n",
       " 4  /articles/instance/5313262/bin/NIHMS845798-sup...   \n",
       " \n",
       "                   a_attr_class a_attr_data-ga-action a_attr_target  \\\n",
       " 0                          NaN                   NaN           NaN   \n",
       " 0  usa-link usa-link--external      click_feat_suppl        _blank   \n",
       " 3                     usa-link      click_feat_suppl           NaN   \n",
       " 4                     usa-link      click_feat_suppl           NaN   \n",
       " \n",
       "             a_attr_rel                                         source_url  \\\n",
       " 0                  NaN  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5...   \n",
       " 0  noopener noreferrer  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5...   \n",
       " 3                  NaN  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5...   \n",
       " 4                  NaN  https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5...   \n",
       " \n",
       "                                            pub_title  \n",
       " 0  Chemosensitive relapse in small cell lung canc...  \n",
       " 0  Chemosensitive relapse in small cell lung canc...  \n",
       " 3  Chemosensitive relapse in small cell lung canc...  \n",
       " 4  Chemosensitive relapse in small cell lung canc...  }"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "execution_count": 6
  },
  {
   "metadata": {},
   "cell_type": "code",
   "outputs": [],
   "execution_count": null,
   "source": "",
   "id": "f3988ee99b5573e9"
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 2
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython2",
   "version": "2.7.6"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
