<html lang="en"><head>

        <meta charset="UTF-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="HandheldFriendly" content="True">
        <meta name="MobileOptimized" content="320">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-70b9163a.css">
<script type="text/javascript" async="" src="https://www.googletagmanager.com/gtag/js?id=G-DP2X732JSX&amp;l=pingerDataLayer&amp;cx=c&amp;gtm=45He4bk0v847342615za200"></script><script async="" src="https://www.googletagmanager.com/gtm.js?id=GTM-PC9B6M3&amp;l=pingerDataLayer" id="pingerInjectedGTM"></script><script type="module" crossorigin="" src="/static/assets/base_style-ec2bc71e.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css">
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css">
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon" sizes="180x180" href="/static/img/favicons/apple-touch-icon.png">
            <link rel="icon" type="image/png" sizes="48x48" href="/static/img/favicons/favicon-48x48.png">
            <link rel="icon" type="image/png" sizes="32x32" href="/static/img/favicons/favicon-32x32.png">
            <link rel="icon" type="image/png" sizes="16x16" href="/static/img/favicons/favicon-16x16.png">
            <link rel="manifest" href="/static/img/favicons/site.webmanifest">
            <link rel="mask-icon" href="/static/img/favicons/safari-pinned-tab.svg" color="#0071bc">
            <meta name="msapplication-config" content="/static/img/favicons/browserconfig.xml">
            <meta name="theme-color" content="#ffffff">
        

        <title>
            Evolutionary trajectories of small cell lung cancer under therapy - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer">
<meta name="ncbi_db" content="pmc">
<meta name="ncbi_phid" content="6E4B3F2474791673063F24001D64E7C7.m_1">
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article">
  
    <link rel="preconnect" href="https://www.google-analytics.com">

    
        <link rel="dns-prefetch" href="https://cdn.ncbi.nlm.nih.gov">
    

    <link rel="preconnect" href="https://code.jquery.com">
    <meta name="ncbi_domain" content="npgopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10972747/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature">
<meta name="citation_title" content="Evolutionary trajectories of small cell lung cancer under therapy">
<meta name="citation_author" content="Julie George">
<meta name="citation_author_institution" content="Department of Translational Genomics, Faculty of Medicine and University Hospital Cologne, University of Cologne, Cologne, Germany">
<meta name="citation_author_institution" content="Department of Otorhinolaryngology, Head and Neck Surgery, Faculty of Medicine and University Hospital Cologne, University Hospital of Cologne, Cologne, Germany">
<meta name="citation_author" content="Lukas Maas">
<meta name="citation_author_institution" content="Department of Translational Genomics, Faculty of Medicine and University Hospital Cologne, University of Cologne, Cologne, Germany">
<meta name="citation_author" content="Nima Abedpour">
<meta name="citation_author_institution" content="Department of Translational Genomics, Faculty of Medicine and University Hospital Cologne, University of Cologne, Cologne, Germany">
<meta name="citation_author_institution" content="Department I of Internal Medicine, Centre for Integrated Oncology Aachen Bonn Cologne Duesseldorf, University Hospital Cologne, Cologne, Germany">
<meta name="citation_author_institution" content="Cancer Research Centre Cologne Essen, Faculty of Medicine and University Hospital Cologne, University of Cologne, Cologne, Germany">
<meta name="citation_author" content="Maria Cartolano">
<meta name="citation_author_institution" content="Department of Translational Genomics, Faculty of Medicine and University Hospital Cologne, University of Cologne, Cologne, Germany">
<meta name="citation_author_institution" content="Centre for Molecular Medicine, University of Cologne, Cologne, Germany">
<meta name="citation_author" content="Laura Kaiser">
<meta name="citation_author_institution" content="Department of Translational Genomics, Faculty of Medicine and University Hospital Cologne, University of Cologne, Cologne, Germany">
<meta name="citation_author" content="Rieke N Fischer">
<meta name="citation_author_institution" content="Department I of Internal Medicine, Lung Cancer Group Cologne, University Hospital Cologne, Cologne, Germany">
<meta name="citation_author" content="Andreas H Scheel">
<meta name="citation_author_institution" content="Institute of Pathology, Medical Faculty, University Hospital Cologne, University of Cologne, Cologne, Germany">
<meta name="citation_author" content="Jan-Philipp Weber">
<meta name="citation_author_institution" content="Department I of Internal Medicine, Lung Cancer Group Cologne, University Hospital Cologne, Cologne, Germany">
<meta name="citation_author" content="Martin Hellmich">
<meta name="citation_author_institution" content="Institute of Medical Statistics, and Computational Biology, Faculty of Medicine and University Hospital Cologne, University of Cologne, Cologne, Germany">
<meta name="citation_author" content="Graziella Bosco">
<meta name="citation_author_institution" content="Department of Translational Genomics, Faculty of Medicine and University Hospital Cologne, University of Cologne, Cologne, Germany">
<meta name="citation_author" content="Caroline Volz">
<meta name="citation_author_institution" content="Department I of Internal Medicine, Centre for Integrated Oncology Aachen Bonn Cologne Duesseldorf, University Hospital Cologne, Cologne, Germany">
<meta name="citation_author_institution" content="Centre for Molecular Medicine, University of Cologne, Cologne, Germany">
<meta name="citation_author" content="Christian Mueller">
<meta name="citation_author_institution" content="Department of Translational Genomics, Faculty of Medicine and University Hospital Cologne, University of Cologne, Cologne, Germany">
<meta name="citation_author_institution" content="Department of Otorhinolaryngology, Head and Neck Surgery, Faculty of Medicine and University Hospital Cologne, University Hospital of Cologne, Cologne, Germany">
<meta name="citation_author" content="Ilona Dahmen">
<meta name="citation_author_institution" content="Department of Translational Genomics, Faculty of Medicine and University Hospital Cologne, University of Cologne, Cologne, Germany">
<meta name="citation_author" content="Felix John">
<meta name="citation_author_institution" content="Department I of Internal Medicine, Lung Cancer Group Cologne, University Hospital Cologne, Cologne, Germany">
<meta name="citation_author" content="Cleidson Padua Alves">
<meta name="citation_author_institution" content="Department of Translational Genomics, Faculty of Medicine and University Hospital Cologne, University of Cologne, Cologne, Germany">
<meta name="citation_author" content="Lisa Werr">
<meta name="citation_author_institution" content="Department of Translational Genomics, Faculty of Medicine and University Hospital Cologne, University of Cologne, Cologne, Germany">
<meta name="citation_author" content="Jens Peter Panse">
<meta name="citation_author_institution" content="Department of Haematology, Oncology, Haemostaseology and Stem Cell Transplantation, University Hospital RWTH Aachen, Aachen, Germany">
<meta name="citation_author_institution" content="Centre for Integrated Oncology, Aachen Bonn Cologne Düsseldorf, Aachen, Germany">
<meta name="citation_author" content="Martin Kirschner">
<meta name="citation_author_institution" content="Department of Haematology, Oncology, Haemostaseology and Stem Cell Transplantation, University Hospital RWTH Aachen, Aachen, Germany">
<meta name="citation_author_institution" content="Centre for Integrated Oncology, Aachen Bonn Cologne Düsseldorf, Aachen, Germany">
<meta name="citation_author" content="Walburga Engel-Riedel">
<meta name="citation_author_institution" content="Department of Pneumology, City of Cologne Municipal Hospitals, Lung Hospital Cologne Merheim, Cologne, Germany">
<meta name="citation_author" content="Jessica Jürgens">
<meta name="citation_author_institution" content="Department of Pneumology, City of Cologne Municipal Hospitals, Lung Hospital Cologne Merheim, Cologne, Germany">
<meta name="citation_author" content="Erich Stoelben">
<meta name="citation_author_institution" content="Thoraxclinic Cologne, Thoracic Surgery, St. Hildegardis-Krankenhaus, Cologne, Germany">
<meta name="citation_author" content="Michael Brockmann">
<meta name="citation_author_institution" content="Department of Pathology, City of Cologne Municipal Hospitals, Witten/Herdecke University, Cologne, Germany">
<meta name="citation_author" content="Stefan Grau">
<meta name="citation_author_institution" content="Department of General Neurosurgery, Centre of Neurosurgery, University Hospital Cologne, Cologne, Germany">
<meta name="citation_author_institution" content="University Medicine Marburg – Campus Fulda, Department of Neurosurgery, Fulda, Germany">
<meta name="citation_author" content="Martin Sebastian">
<meta name="citation_author_institution" content="Department of Medicine II, Haematology/Oncology, University Hospital Frankfurt, Goethe University, Frankfurt, Germany">
<meta name="citation_author_institution" content="Frankfurt Cancer Institute, Goethe University Frankfurt, Frankfurt, Germany">
<meta name="citation_author_institution" content="DKFZ, German Cancer Research Centre, German Cancer Consortium, Heidelberg, Germany">
<meta name="citation_author" content="Jan A Stratmann">
<meta name="citation_author_institution" content="Department of Medicine II, Haematology/Oncology, University Hospital Frankfurt, Goethe University, Frankfurt, Germany">
<meta name="citation_author_institution" content="Frankfurt Cancer Institute, Goethe University Frankfurt, Frankfurt, Germany">
<meta name="citation_author" content="Jens Kern">
<meta name="citation_author_institution" content="Klinikum Würzburg Mitte – Missioklinik site, Pneumology and Respiratory Medicine, Würzburg, Germany">
<meta name="citation_author" content="Horst-Dieter Hummel">
<meta name="citation_author_institution" content="Translational Oncology/Early Clinical Trial Unit, Comprehensive Cancer Centre Mainfranken, University Hospital Wuerzburg, Wuerzburg, Germany">
<meta name="citation_author" content="Balazs Hegedüs">
<meta name="citation_author_institution" content="Department of Thoracic Surgery, University Medicine Essen – Ruhrlandklinik, University Duisburg-Essen, Essen, Germany">
<meta name="citation_author" content="Martin Schuler">
<meta name="citation_author_institution" content="DKFZ, German Cancer Research Centre, German Cancer Consortium, Heidelberg, Germany">
<meta name="citation_author_institution" content="Department of Medical Oncology, West German Cancer Centre Essen, University Duisburg-Essen, Essen, Germany">
<meta name="citation_author" content="Till Plönes">
<meta name="citation_author_institution" content="Department of Medical Oncology, West German Cancer Centre Essen, University Duisburg-Essen, Essen, Germany">
<meta name="citation_author_institution" content="Division of Thoracic Surgery, Department of General, Thoracic and Vascular Surgery, University Hospital Carl Gustav Carus, Dresden, Germany">
<meta name="citation_author" content="Clemens Aigner">
<meta name="citation_author_institution" content="Department of Thoracic Surgery, University Medicine Essen – Ruhrlandklinik, University Duisburg-Essen, Essen, Germany">
<meta name="citation_author_institution" content="Department of Thoracic Surgery, Medical University of Vienna, Vienna General Hospital, Vienna, Austria">
<meta name="citation_author" content="Thomas Elter">
<meta name="citation_author_institution" content="Department I of Internal Medicine, Centre for Integrated Oncology Aachen Bonn Cologne Duesseldorf, University Hospital Cologne, Cologne, Germany">
<meta name="citation_author" content="Karin Toepelt">
<meta name="citation_author_institution" content="Department I of Internal Medicine, Centre for Integrated Oncology Aachen Bonn Cologne Duesseldorf, University Hospital Cologne, Cologne, Germany">
<meta name="citation_author" content="Yon-Dschun Ko">
<meta name="citation_author_institution" content="Comprehensive Cancer Centre CIO Bonn, Bonn, Germany">
<meta name="citation_author" content="Sylke Kurz">
<meta name="citation_author_institution" content="Department of Respiratory Diseases, Evangelische Lungenklinik, Berlin, Germany">
<meta name="citation_author" content="Christian Grohé">
<meta name="citation_author_institution" content="Department of Respiratory Diseases, Evangelische Lungenklinik, Berlin, Germany">
<meta name="citation_author" content="Monika Serke">
<meta name="citation_author_institution" content="DGD Lungenklinik Hemer, Internal Medicine, Pneumology and Oncology, Hemer, Germany">
<meta name="citation_author" content="Katja Höpker">
<meta name="citation_author_institution" content="Clinic III for Internal Medicine, Faculty of Medicine and University Hospital Cologne, University of Cologne, Cologne, Germany">
<meta name="citation_author" content="Lars Hagmeyer">
<meta name="citation_author_institution" content="Clinic of Pneumology and Allergology, Centre for Sleep Medicine and Respiratory Care, Bethanien Hospital Solingen, Solingen, Germany">
<meta name="citation_author" content="Fabian Doerr">
<meta name="citation_author_institution" content="Department of Thoracic Surgery, University Medicine Essen – Ruhrlandklinik, University Duisburg-Essen, Essen, Germany">
<meta name="citation_author_institution" content="Department of Cardiothoracic Surgery, University Hospital of Cologne, Cologne, Germany">
<meta name="citation_author" content="Khosro Hekmath">
<meta name="citation_author_institution" content="Department of Cardiothoracic Surgery, University Hospital of Cologne, Cologne, Germany">
<meta name="citation_author" content="Judith Strapatsas">
<meta name="citation_author_institution" content="Department of Haematology, Oncology and Clinical Immunology, University Hospital of Duesseldorf, Düsseldorf, Germany">
<meta name="citation_author" content="Karl-Otto Kambartel">
<meta name="citation_author_institution" content="Krankenhaus Bethanien Moers, Moers, Germany">
<meta name="citation_author" content="Geothy Chakupurakal">
<meta name="citation_author_institution" content="Praxis für Hämatologie und Onkologie, Koblenz, Germany">
<meta name="citation_author" content="Annette Busch">
<meta name="citation_author_institution" content="Medical Clinic III for Oncology, Haematology, Immune-Oncology and Rheumatology, Centre for Integrative Medicine, University Hospital Bonn, Bonn, Germany">
<meta name="citation_author" content="Franz-Georg Bauernfeind">
<meta name="citation_author_institution" content="Medical Clinic III for Oncology, Haematology, Immune-Oncology and Rheumatology, Centre for Integrative Medicine, University Hospital Bonn, Bonn, Germany">
<meta name="citation_author" content="Frank Griesinger">
<meta name="citation_author_institution" content="Pius-Hospital Oldenburg, Department of Haematology and Oncology, University Department Internal Medicine-Oncology, University Medicine Oldenburg, Oldenburg, Germany">
<meta name="citation_author" content="Anne Luers">
<meta name="citation_author_institution" content="Pius-Hospital Oldenburg, Department of Haematology and Oncology, University Department Internal Medicine-Oncology, University Medicine Oldenburg, Oldenburg, Germany">
<meta name="citation_author" content="Wiebke Dirks">
<meta name="citation_author_institution" content="Pius-Hospital Oldenburg, Department of Haematology and Oncology, University Department Internal Medicine-Oncology, University Medicine Oldenburg, Oldenburg, Germany">
<meta name="citation_author" content="Rainer Wiewrodt">
<meta name="citation_author_institution" content="Pulmonary Division, Department of Medicine A, Münster University Hospital, Münster, Germany">
<meta name="citation_author" content="Andrea Luecke">
<meta name="citation_author_institution" content="Pulmonary Division, Department of Medicine A, Münster University Hospital, Münster, Germany">
<meta name="citation_author" content="Ernst Rodermann">
<meta name="citation_author_institution" content="Onkologie Rheinsieg, Praxisnetzwerk Hämatologie und Internistische Onkologie, Troisdorf, Germany">
<meta name="citation_author" content="Andreas Diel">
<meta name="citation_author_institution" content="Onkologie Rheinsieg, Praxisnetzwerk Hämatologie und Internistische Onkologie, Troisdorf, Germany">
<meta name="citation_author" content="Volker Hagen">
<meta name="citation_author_institution" content="Clinic II for Internal Medicine, St.-Johannes-Hospital Dortmund, Dortmund, Germany">
<meta name="citation_author" content="Kai Severin">
<meta name="citation_author_institution" content="Haematologie und Onkologie Köln MV-Zentrum, Cologne, Germany">
<meta name="citation_author" content="Roland T Ullrich">
<meta name="citation_author_institution" content="Department I of Internal Medicine, Centre for Integrated Oncology Aachen Bonn Cologne Duesseldorf, University Hospital Cologne, Cologne, Germany">
<meta name="citation_author_institution" content="Centre for Molecular Medicine, University of Cologne, Cologne, Germany">
<meta name="citation_author" content="Hans Christian Reinhardt">
<meta name="citation_author_institution" content="Department of Haematology and Stem Cell Transplantation, University Hospital Essen, Essen, Germany">
<meta name="citation_author_institution" content="West German Cancer Centre, University Hospital Essen, Essen, Germany">
<meta name="citation_author" content="Alexander Quaas">
<meta name="citation_author_institution" content="Institute of Pathology, Medical Faculty, University Hospital Cologne, University of Cologne, Cologne, Germany">
<meta name="citation_author" content="Magdalena Bogus">
<meta name="citation_author_institution" content="Institute of Legal Medicine, University of Cologne, Cologne, Germany">
<meta name="citation_author" content="Cornelius Courts">
<meta name="citation_author_institution" content="Institute of Legal Medicine, University of Cologne, Cologne, Germany">
<meta name="citation_author" content="Peter Nürnberg">
<meta name="citation_author_institution" content="Cologne Centre for Genomics, West German Genome Centre, University of Cologne, Cologne, Germany">
<meta name="citation_author" content="Kerstin Becker">
<meta name="citation_author_institution" content="Cologne Centre for Genomics, West German Genome Centre, University of Cologne, Cologne, Germany">
<meta name="citation_author" content="Viktor Achter">
<meta name="citation_author_institution" content="Computing Centre, University of Cologne, Cologne, Germany">
<meta name="citation_author" content="Reinhard Büttner">
<meta name="citation_author_institution" content="Institute of Pathology, Medical Faculty, University Hospital Cologne, University of Cologne, Cologne, Germany">
<meta name="citation_author" content="Jürgen Wolf">
<meta name="citation_author_institution" content="Department I of Internal Medicine, Lung Cancer Group Cologne, University Hospital Cologne, Cologne, Germany">
<meta name="citation_author" content="Martin Peifer">
<meta name="citation_author_institution" content="Department of Translational Genomics, Faculty of Medicine and University Hospital Cologne, University of Cologne, Cologne, Germany">
<meta name="citation_author_institution" content="Centre for Molecular Medicine, University of Cologne, Cologne, Germany">
<meta name="citation_author" content="Roman K Thomas">
<meta name="citation_author_institution" content="Department of Translational Genomics, Faculty of Medicine and University Hospital Cologne, University of Cologne, Cologne, Germany">
<meta name="citation_author_institution" content="Institute of Pathology, Medical Faculty, University Hospital Cologne, University of Cologne, Cologne, Germany">
<meta name="citation_author_institution" content="DKFZ, German Cancer Research Centre, German Cancer Consortium, Heidelberg, Germany">
<meta name="citation_publication_date" content="2024 Mar 13">
<meta name="citation_volume" content="627">
<meta name="citation_issue" content="8005">
<meta name="citation_firstpage" content="880">
<meta name="citation_doi" content="10.1038/s41586-024-07177-7">
<meta name="citation_pmid" content="38480884">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC10972747/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC10972747/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC10972747/pdf/41586_2024_Article_7177.pdf">
<meta name="description" content="The evolutionary processes that underlie the marked sensitivity of small cell lung cancer (SCLC) to chemotherapy and rapid relapse are unknown1–3. Here&nbsp;we determined tumour phylogenies at diagnosis and throughout chemotherapy and immunotherapy by ...">
<meta name="og:title" content="Evolutionary trajectories of small cell lung cancer under therapy">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The evolutionary processes that underlie the marked sensitivity of small cell lung cancer (SCLC) to chemotherapy and rapid relapse are unknown1–3. Here&nbsp;we determined tumour phylogenies at diagnosis and throughout chemotherapy and immunotherapy by ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC10972747/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    <script type="text/javascript" src="https://www.googletagmanager.com/gtag/js?id=G-CSLL4ZEK4L"></script><script charset="utf-8" src="https://siteintercept.qualtrics.com/dxjsmodule/10.07268bfc859327bf20d5.chunk.js?Q_CLIENTVERSION=2.20.0&amp;Q_CLIENTTYPE=web&amp;Q_BRANDID=pmc.ncbi.nlm.nih.gov"></script><script charset="utf-8" src="https://siteintercept.qualtrics.com/dxjsmodule/5.d83df5c454102e31d5df.chunk.js?Q_CLIENTVERSION=2.20.0&amp;Q_CLIENTTYPE=web&amp;Q_BRANDID=nlmenterprise"></script><script charset="utf-8" src="https://siteintercept.qualtrics.com/dxjsmodule/1.4a290fea10f6e9b6f375.chunk.js?Q_CLIENTVERSION=2.20.0&amp;Q_CLIENTTYPE=web&amp;Q_BRANDID=nlmenterprise"></script></head>
    <body>
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government">
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true" class="usa-banner__header-flag" src="/static/img/us_flag.svg" alt="">
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button type="button" class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           " aria-expanded="false" aria-controls="gov-banner-default" data-testid="storybook-django-banner">
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content" id="gov-banner-default" hidden="">
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img" src="/static/img/icon-dot-gov.svg" alt="" aria-hidden="true">
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br>
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img" src="/static/img/icon-https.svg" alt="" aria-hidden="true">

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br>
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg" width="52" height="64" viewBox="0 0 52 64" class="usa-banner__lock-image" role="graphics-symbol" aria-labelledby="banner-lock-description" focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000" fill-rule="evenodd" d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z"></path>
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header="">
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="/">
                    <img alt="
                                  PMC home page
                              " class="ncbi-header__logo-image" src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg">
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button type="button" class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           " aria-label="Show menu" data-testid="navMenuButton">
    
        

        
                <svg aria-hidden="true" class="ncbi-hamburger-icon" fill="none" focusable="false" height="21" viewBox="0 0 31 21" width="31" xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd" d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z" fill="#F1F1F1" fill-rule="evenodd"></path>
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button type="button" class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           " aria-expanded="false" aria-controls="search-field-desktop-navigation" aria-label="Show search overlay" data-testid="toggleSearchPanelButton" data-toggle-search-panel-button="" data-ga-category="header" data-ga-action="NCBI" data-ga-label="header_search_open">
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true">
        
        <use xlink:href="/static/img/sprite.svg#search"></use>
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button type="button" class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           " aria-expanded="false" aria-controls="login-dropdown-menu" aria-label="Show login menu" data-testid="toggleLoginMenuDropdown" data-desktop-login-button="">
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true">
        
        <use xlink:href="/static/img/sprite.svg#person"></use>
    </svg>



                                <span data-login-dropdown-text="">Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow="">
        
        <use xlink:href="/static/img/sprite.svg#expand_less"></use>
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow="">
        
        <use xlink:href="/static/img/sprite.svg#expand_more"></use>
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu" id="login-dropdown-menu" data-desktop-login-menu-dropdown="" hidden="">
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  ">
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  ">
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  ">
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button type="button" class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           " data-testid="desktopLogoutButton" data-desktop-logout-button="">
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop" data-testid="searchPanel" data-header-search-panel="" hidden="">
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/" aria-describedby="search-field-desktop-navigation-help-text" autocomplete="off" class="usa-search usa-search--big ncbi-search-panel__form" data-testid="form" method="GET" role="search">
                    <label class="usa-sr-only" data-testid="label" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input" data-testid="textInput" id="search-field-desktop-navigation" name="term" placeholder="Search NCBI" type="search" value="">
                    



















    
        <button type="submit" class="usa-button
           

           
               
               
               
               
            

           
           
           
           " data-testid="button" data-ga-category="header" data-ga-action="NCBI" data-ga-label="header_search_button">
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button type="button" class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           " aria-label="Close navigation menu" data-testid="navCloseButton">
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close">
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6" role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input" id="search-field-mobile-navigation" type="search" placeholder="Search NCBI" name="search">

                



















    
        <button type="submit" class="usa-button
           

           
               
               
               
               
            

           
           
           
           " data-ga-category="header" data-ga-action="NCBI" data-ga-label="header_search_button">
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg==" class="usa-search__submit-icon" alt="Search">
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop" data-mobile-login-menu="" hidden="">
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email" data-mobile-login-email-text=""></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  ">
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  ">
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  ">
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button type="button" class="usa-button ncbi-nav__mobile-login-button ncbi--hide-at-desktop" data-testid="mobileLoginButton" data-mobile-login-button="">Log in</button>
        


    </nav>
</header>

    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button type="button" class="usa-button usa-button--unstyled pmc-header__search__button" aria-label="Open search" data-ga-category="search" data-ga-action="PMC" data-ga-label="pmc_search_panel_mobile">
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column pmc-header__search__form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search" type="search" name="term" data-autocomplete-url="/search/autocomplete/" aria-controls="autoComplete_list_1" aria-autocomplete="both" role="combobox" aria-owns="autoComplete_list_1" aria-haspopup="true" aria-expanded="false"><ul id="autoComplete_list_1" role="listbox" hidden="" aria-label="Suggestions"></ul>
</span>
<button class="usa-button" type="submit" formaction="https://www.ncbi.nlm.nih.gov/pmc/" data-ga-category="search" data-ga-action="PMC" data-ga-label="PMC_search_button">
<span class="usa-search__submit-text">Search in PMC</span>
<img src="/static/img/usa-icons-bg/search--white.svg" class="usa-search__submit-icon" alt="Search">
</button>
</form>
            <ul class="pmc-header__search__menu">
                <li>
                    <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                        Advanced Search
                    </a>
                </li>
                <li>
                    
                        <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                            Journal List
                        </a>
                    
                </li>
                <li>
                    
                        <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link" data-ga-label="user guide">
                            User Guide
                        </a>
                    
                </li>
            </ul>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="10972747">

    

   



<div class="grid-container pmc-actions-bar is-not-intersecting" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button type="button" class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex" aria-label="Open resources" data-extra-class="is-visible-resources" data-ga-category="resources_accordion" data-ga-action="click" data-ga-label="mobile_icon">
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a href="https://doi.org/10.1038/s41586-024-07177-7" class="usa-link display-flex" role="button" target="_blank" rel="noreferrer noopener" aria-label="View on publisher site" data-ga-category="actions" data-ga-action="click" data-ga-label="publisher_link_mobile">
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a href="pdf/41586_2024_Article_7177.pdf" class="usa-link display-flex" role="button" aria-label="Download PDF" data-ga-category="actions" data-ga-action="click" data-ga-label="pdf_download_mobile">
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button class="usa-button usa-button--unstyled collections-dialog-trigger collections-button display-flex collections-button-empty" aria-label="Save article in MyNCBI collections." data-ga-category="actions" data-ga-action="click" data-ga-label="collections_button_mobile" data-collections-open-dialog-enabled="false" data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC10972747%2F%23open-collections-dialog" data-in-collections="false">
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden="">
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden="">
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled citation-dialog-trigger display-flex" aria-label="Open dialog with citation text in different styles" data-ga-category="actions" data-ga-action="open" data-ga-label="cite_mobile" data-all-citations-url="/resources/citations/10972747/" data-citation-style="nlm" data-download-format-link="/resources/citations/10972747/export/">
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden="">
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex">
                         <button type="button" class="usa-button usa-button--unstyled display-flex" aria-label="Show article permalink" aria-expanded="false" aria-haspopup="true" data-ga-category="actions" data-ga-action="open" data-ga-label="permalink_mobile">
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden="">
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC10972747/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link" aria-expanded="false">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button type="button" class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex" aria-label="Open article navigation" data-extra-class="is-visible-in-page" data-ga-category="actions" data-ga-action="open" data-ga-label="article_nav_mobile">
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main id="main-content" class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0">

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-npgopen.png" alt="Nature Portfolio logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Portfolio" title="Link to Nature Portfolio" shape="default" href="https://doi.org/10.1038/s41586-024-07177-7" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu" aria-expanded="false">Nature</button></div>. 2024 Mar 13;627(8005):880–889. doi: <a href="https://doi.org/10.1038/s41586-024-07177-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41586-024-07177-7</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nature%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nature%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nature%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nature%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Evolutionary trajectories of small cell lung cancer under therapy</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22George%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id1" aria-expanded="false"><span class="name western">Julie George</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Julie George</span></h3>
<div class="p">
<sup>1</sup>Department of Translational Genomics, Faculty of Medicine and University Hospital Cologne, University of Cologne, Cologne, Germany </div>
<div class="p">
<sup>2</sup>Department of Otorhinolaryngology, Head and Neck Surgery, Faculty of Medicine and University Hospital Cologne, University Hospital of Cologne, Cologne, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22George%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Julie George</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Maas%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id2" aria-expanded="false"><span class="name western">Lukas Maas</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Lukas Maas</span></h3>
<div class="p">
<sup>1</sup>Department of Translational Genomics, Faculty of Medicine and University Hospital Cologne, University of Cologne, Cologne, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Maas%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lukas Maas</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Abedpour%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id3" aria-expanded="false"><span class="name western">Nima Abedpour</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Nima Abedpour</span></h3>
<div class="p">
<sup>1</sup>Department of Translational Genomics, Faculty of Medicine and University Hospital Cologne, University of Cologne, Cologne, Germany </div>
<div class="p">
<sup>3</sup>Department I of Internal Medicine, Centre for Integrated Oncology Aachen Bonn Cologne Duesseldorf, University Hospital Cologne, Cologne, Germany </div>
<div class="p">
<sup>4</sup>Cancer Research Centre Cologne Essen, Faculty of Medicine and University Hospital Cologne, University of Cologne, Cologne, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Abedpour%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Nima Abedpour</span></a>
</div>
</div>
<sup>1,</sup><sup>3,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cartolano%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id4" aria-expanded="false"><span class="name western">Maria Cartolano</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Maria Cartolano</span></h3>
<div class="p">
<sup>1</sup>Department of Translational Genomics, Faculty of Medicine and University Hospital Cologne, University of Cologne, Cologne, Germany </div>
<div class="p">
<sup>5</sup>Centre for Molecular Medicine, University of Cologne, Cologne, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cartolano%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Maria Cartolano</span></a>
</div>
</div>
<sup>1,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kaiser%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id5" aria-expanded="false"><span class="name western">Laura Kaiser</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Laura Kaiser</span></h3>
<div class="p">
<sup>1</sup>Department of Translational Genomics, Faculty of Medicine and University Hospital Cologne, University of Cologne, Cologne, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kaiser%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Laura Kaiser</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fischer%20RN%22%5BAuthor%5D" class="usa-link" aria-describedby="id6" aria-expanded="false"><span class="name western">Rieke N Fischer</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Rieke N Fischer</span></h3>
<div class="p">
<sup>6</sup>Department I of Internal Medicine, Lung Cancer Group Cologne, University Hospital Cologne, Cologne, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fischer%20RN%22%5BAuthor%5D" class="usa-link"><span class="name western">Rieke N Fischer</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Scheel%20AH%22%5BAuthor%5D" class="usa-link" aria-describedby="id7" aria-expanded="false"><span class="name western">Andreas H Scheel</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Andreas H Scheel</span></h3>
<div class="p">
<sup>7</sup>Institute of Pathology, Medical Faculty, University Hospital Cologne, University of Cologne, Cologne, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Scheel%20AH%22%5BAuthor%5D" class="usa-link"><span class="name western">Andreas H Scheel</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Weber%20JP%22%5BAuthor%5D" class="usa-link" aria-describedby="id8" aria-expanded="false"><span class="name western">Jan-Philipp Weber</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Jan-Philipp Weber</span></h3>
<div class="p">
<sup>6</sup>Department I of Internal Medicine, Lung Cancer Group Cologne, University Hospital Cologne, Cologne, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Weber%20JP%22%5BAuthor%5D" class="usa-link"><span class="name western">Jan-Philipp Weber</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hellmich%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id9" aria-expanded="false"><span class="name western">Martin Hellmich</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Martin Hellmich</span></h3>
<div class="p">
<sup>8</sup>Institute of Medical Statistics, and Computational Biology, Faculty of Medicine and University Hospital Cologne, University of Cologne, Cologne, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hellmich%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Martin Hellmich</span></a>
</div>
</div>
<sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bosco%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id10" aria-expanded="false"><span class="name western">Graziella Bosco</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Graziella Bosco</span></h3>
<div class="p">
<sup>1</sup>Department of Translational Genomics, Faculty of Medicine and University Hospital Cologne, University of Cologne, Cologne, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bosco%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Graziella Bosco</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Volz%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id11" aria-expanded="false"><span class="name western">Caroline Volz</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Caroline Volz</span></h3>
<div class="p">
<sup>3</sup>Department I of Internal Medicine, Centre for Integrated Oncology Aachen Bonn Cologne Duesseldorf, University Hospital Cologne, Cologne, Germany </div>
<div class="p">
<sup>5</sup>Centre for Molecular Medicine, University of Cologne, Cologne, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Volz%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Caroline Volz</span></a>
</div>
</div>
<sup>3,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mueller%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id12" aria-expanded="false"><span class="name western">Christian Mueller</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Christian Mueller</span></h3>
<div class="p">
<sup>1</sup>Department of Translational Genomics, Faculty of Medicine and University Hospital Cologne, University of Cologne, Cologne, Germany </div>
<div class="p">
<sup>2</sup>Department of Otorhinolaryngology, Head and Neck Surgery, Faculty of Medicine and University Hospital Cologne, University Hospital of Cologne, Cologne, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mueller%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Christian Mueller</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dahmen%20I%22%5BAuthor%5D" class="usa-link" aria-describedby="id13" aria-expanded="false"><span class="name western">Ilona Dahmen</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Ilona Dahmen</span></h3>
<div class="p">
<sup>1</sup>Department of Translational Genomics, Faculty of Medicine and University Hospital Cologne, University of Cologne, Cologne, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dahmen%20I%22%5BAuthor%5D" class="usa-link"><span class="name western">Ilona Dahmen</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22John%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id14" aria-expanded="false"><span class="name western">Felix John</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Felix John</span></h3>
<div class="p">
<sup>6</sup>Department I of Internal Medicine, Lung Cancer Group Cologne, University Hospital Cologne, Cologne, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22John%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Felix John</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Alves%20CP%22%5BAuthor%5D" class="usa-link" aria-describedby="id15" aria-expanded="false"><span class="name western">Cleidson Padua Alves</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">Cleidson Padua Alves</span></h3>
<div class="p">
<sup>1</sup>Department of Translational Genomics, Faculty of Medicine and University Hospital Cologne, University of Cologne, Cologne, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Alves%20CP%22%5BAuthor%5D" class="usa-link"><span class="name western">Cleidson Padua Alves</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Werr%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id16" aria-expanded="false"><span class="name western">Lisa Werr</span></a><div hidden="hidden" id="id16">
<h3><span class="name western">Lisa Werr</span></h3>
<div class="p">
<sup>1</sup>Department of Translational Genomics, Faculty of Medicine and University Hospital Cologne, University of Cologne, Cologne, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Werr%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lisa Werr</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Panse%20JP%22%5BAuthor%5D" class="usa-link" aria-describedby="id17" aria-expanded="false"><span class="name western">Jens Peter Panse</span></a><div hidden="hidden" id="id17">
<h3><span class="name western">Jens Peter Panse</span></h3>
<div class="p">
<sup>9</sup>Department of Haematology, Oncology, Haemostaseology and Stem Cell Transplantation, University Hospital RWTH Aachen, Aachen, Germany </div>
<div class="p">
<sup>10</sup>Centre for Integrated Oncology, Aachen Bonn Cologne Düsseldorf, Aachen, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Panse%20JP%22%5BAuthor%5D" class="usa-link"><span class="name western">Jens Peter Panse</span></a>
</div>
</div>
<sup>9,</sup><sup>10</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kirschner%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id18" aria-expanded="false"><span class="name western">Martin Kirschner</span></a><div hidden="hidden" id="id18">
<h3><span class="name western">Martin Kirschner</span></h3>
<div class="p">
<sup>9</sup>Department of Haematology, Oncology, Haemostaseology and Stem Cell Transplantation, University Hospital RWTH Aachen, Aachen, Germany </div>
<div class="p">
<sup>10</sup>Centre for Integrated Oncology, Aachen Bonn Cologne Düsseldorf, Aachen, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kirschner%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Martin Kirschner</span></a>
</div>
</div>
<sup>9,</sup><sup>10</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Engel-Riedel%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id19" aria-expanded="false"><span class="name western">Walburga Engel-Riedel</span></a><div hidden="hidden" id="id19">
<h3><span class="name western">Walburga Engel-Riedel</span></h3>
<div class="p">
<sup>11</sup>Department of Pneumology, City of Cologne Municipal Hospitals, Lung Hospital Cologne Merheim, Cologne, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Engel-Riedel%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Walburga Engel-Riedel</span></a>
</div>
</div>
<sup>11</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22J%C3%BCrgens%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id20" aria-expanded="false"><span class="name western">Jessica Jürgens</span></a><div hidden="hidden" id="id20">
<h3><span class="name western">Jessica Jürgens</span></h3>
<div class="p">
<sup>11</sup>Department of Pneumology, City of Cologne Municipal Hospitals, Lung Hospital Cologne Merheim, Cologne, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22J%C3%BCrgens%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jessica Jürgens</span></a>
</div>
</div>
<sup>11</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Stoelben%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id21" aria-expanded="false"><span class="name western">Erich Stoelben</span></a><div hidden="hidden" id="id21">
<h3><span class="name western">Erich Stoelben</span></h3>
<div class="p">
<sup>12</sup>Thoraxclinic Cologne, Thoracic Surgery, St. Hildegardis-Krankenhaus, Cologne, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Stoelben%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Erich Stoelben</span></a>
</div>
</div>
<sup>12</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Brockmann%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id22" aria-expanded="false"><span class="name western">Michael Brockmann</span></a><div hidden="hidden" id="id22">
<h3><span class="name western">Michael Brockmann</span></h3>
<div class="p">
<sup>13</sup>Department of Pathology, City of Cologne Municipal Hospitals, Witten/Herdecke University, Cologne, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Brockmann%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Michael Brockmann</span></a>
</div>
</div>
<sup>13</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Grau%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id23" aria-expanded="false"><span class="name western">Stefan Grau</span></a><div hidden="hidden" id="id23">
<h3><span class="name western">Stefan Grau</span></h3>
<div class="p">
<sup>14</sup>Department of General Neurosurgery, Centre of Neurosurgery, University Hospital Cologne, Cologne, Germany </div>
<div class="p">
<sup>15</sup>University Medicine Marburg – Campus Fulda, Department of Neurosurgery, Fulda, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Grau%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Stefan Grau</span></a>
</div>
</div>
<sup>14,</sup><sup>15</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sebastian%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id24" aria-expanded="false"><span class="name western">Martin Sebastian</span></a><div hidden="hidden" id="id24">
<h3><span class="name western">Martin Sebastian</span></h3>
<div class="p">
<sup>16</sup>Department of Medicine II, Haematology/Oncology, University Hospital Frankfurt, Goethe University, Frankfurt, Germany </div>
<div class="p">
<sup>17</sup>Frankfurt Cancer Institute, Goethe University Frankfurt, Frankfurt, Germany </div>
<div class="p">
<sup>18</sup>DKFZ, German Cancer Research Centre, German Cancer Consortium, Heidelberg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sebastian%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Martin Sebastian</span></a>
</div>
</div>
<sup>16,</sup><sup>17,</sup><sup>18</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Stratmann%20JA%22%5BAuthor%5D" class="usa-link" aria-describedby="id25" aria-expanded="false"><span class="name western">Jan A Stratmann</span></a><div hidden="hidden" id="id25">
<h3><span class="name western">Jan A Stratmann</span></h3>
<div class="p">
<sup>16</sup>Department of Medicine II, Haematology/Oncology, University Hospital Frankfurt, Goethe University, Frankfurt, Germany </div>
<div class="p">
<sup>17</sup>Frankfurt Cancer Institute, Goethe University Frankfurt, Frankfurt, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Stratmann%20JA%22%5BAuthor%5D" class="usa-link"><span class="name western">Jan A Stratmann</span></a>
</div>
</div>
<sup>16,</sup><sup>17</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kern%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id26" aria-expanded="false"><span class="name western">Jens Kern</span></a><div hidden="hidden" id="id26">
<h3><span class="name western">Jens Kern</span></h3>
<div class="p">
<sup>19</sup>Klinikum Würzburg Mitte – Missioklinik site, Pneumology and Respiratory Medicine, Würzburg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kern%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jens Kern</span></a>
</div>
</div>
<sup>19</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hummel%20HD%22%5BAuthor%5D" class="usa-link" aria-describedby="id27" aria-expanded="false"><span class="name western">Horst-Dieter Hummel</span></a><div hidden="hidden" id="id27">
<h3><span class="name western">Horst-Dieter Hummel</span></h3>
<div class="p">
<sup>20</sup>Translational Oncology/Early Clinical Trial Unit, Comprehensive Cancer Centre Mainfranken, University Hospital Wuerzburg, Wuerzburg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hummel%20HD%22%5BAuthor%5D" class="usa-link"><span class="name western">Horst-Dieter Hummel</span></a>
</div>
</div>
<sup>20</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Heged%C3%BCs%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id28" aria-expanded="false"><span class="name western">Balazs Hegedüs</span></a><div hidden="hidden" id="id28">
<h3><span class="name western">Balazs Hegedüs</span></h3>
<div class="p">
<sup>21</sup>Department of Thoracic Surgery, University Medicine Essen – Ruhrlandklinik, University Duisburg-Essen, Essen, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Heged%C3%BCs%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Balazs Hegedüs</span></a>
</div>
</div>
<sup>21</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schuler%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id29" aria-expanded="false"><span class="name western">Martin Schuler</span></a><div hidden="hidden" id="id29">
<h3><span class="name western">Martin Schuler</span></h3>
<div class="p">
<sup>18</sup>DKFZ, German Cancer Research Centre, German Cancer Consortium, Heidelberg, Germany </div>
<div class="p">
<sup>22</sup>Department of Medical Oncology, West German Cancer Centre Essen, University Duisburg-Essen, Essen, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schuler%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Martin Schuler</span></a>
</div>
</div>
<sup>18,</sup><sup>22</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pl%C3%B6nes%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id30" aria-expanded="false"><span class="name western">Till Plönes</span></a><div hidden="hidden" id="id30">
<h3><span class="name western">Till Plönes</span></h3>
<div class="p">
<sup>22</sup>Department of Medical Oncology, West German Cancer Centre Essen, University Duisburg-Essen, Essen, Germany </div>
<div class="p">
<sup>23</sup>Division of Thoracic Surgery, Department of General, Thoracic and Vascular Surgery, University Hospital Carl Gustav Carus, Dresden, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pl%C3%B6nes%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Till Plönes</span></a>
</div>
</div>
<sup>22,</sup><sup>23</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Aigner%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id31" aria-expanded="false"><span class="name western">Clemens Aigner</span></a><div hidden="hidden" id="id31">
<h3><span class="name western">Clemens Aigner</span></h3>
<div class="p">
<sup>21</sup>Department of Thoracic Surgery, University Medicine Essen – Ruhrlandklinik, University Duisburg-Essen, Essen, Germany </div>
<div class="p">
<sup>24</sup>Department of Thoracic Surgery, Medical University of Vienna, Vienna General Hospital, Vienna, Austria </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Aigner%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Clemens Aigner</span></a>
</div>
</div>
<sup>21,</sup><sup>24</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Elter%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id32" aria-expanded="false"><span class="name western">Thomas Elter</span></a><div hidden="hidden" id="id32">
<h3><span class="name western">Thomas Elter</span></h3>
<div class="p">
<sup>3</sup>Department I of Internal Medicine, Centre for Integrated Oncology Aachen Bonn Cologne Duesseldorf, University Hospital Cologne, Cologne, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Elter%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Thomas Elter</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Toepelt%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id33" aria-expanded="false"><span class="name western">Karin Toepelt</span></a><div hidden="hidden" id="id33">
<h3><span class="name western">Karin Toepelt</span></h3>
<div class="p">
<sup>3</sup>Department I of Internal Medicine, Centre for Integrated Oncology Aachen Bonn Cologne Duesseldorf, University Hospital Cologne, Cologne, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Toepelt%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Karin Toepelt</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ko%20YD%22%5BAuthor%5D" class="usa-link" aria-describedby="id34" aria-expanded="false"><span class="name western">Yon-Dschun Ko</span></a><div hidden="hidden" id="id34">
<h3><span class="name western">Yon-Dschun Ko</span></h3>
<div class="p">
<sup>25</sup>Comprehensive Cancer Centre CIO Bonn, Bonn, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ko%20YD%22%5BAuthor%5D" class="usa-link"><span class="name western">Yon-Dschun Ko</span></a>
</div>
</div>
<sup>25</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kurz%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id35" aria-expanded="false"><span class="name western">Sylke Kurz</span></a><div hidden="hidden" id="id35">
<h3><span class="name western">Sylke Kurz</span></h3>
<div class="p">
<sup>26</sup>Department of Respiratory Diseases, Evangelische Lungenklinik, Berlin, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kurz%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sylke Kurz</span></a>
</div>
</div>
<sup>26</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Groh%C3%A9%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id36" aria-expanded="false"><span class="name western">Christian Grohé</span></a><div hidden="hidden" id="id36">
<h3><span class="name western">Christian Grohé</span></h3>
<div class="p">
<sup>26</sup>Department of Respiratory Diseases, Evangelische Lungenklinik, Berlin, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Groh%C3%A9%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Christian Grohé</span></a>
</div>
</div>
<sup>26</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Serke%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id37" aria-expanded="false"><span class="name western">Monika Serke</span></a><div hidden="hidden" id="id37">
<h3><span class="name western">Monika Serke</span></h3>
<div class="p">
<sup>27</sup>DGD Lungenklinik Hemer, Internal Medicine, Pneumology and Oncology, Hemer, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Serke%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Monika Serke</span></a>
</div>
</div>
<sup>27</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22H%C3%B6pker%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id38" aria-expanded="false"><span class="name western">Katja Höpker</span></a><div hidden="hidden" id="id38">
<h3><span class="name western">Katja Höpker</span></h3>
<div class="p">
<sup>28</sup>Clinic III for Internal Medicine, Faculty of Medicine and University Hospital Cologne, University of Cologne, Cologne, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22H%C3%B6pker%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Katja Höpker</span></a>
</div>
</div>
<sup>28</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hagmeyer%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id39" aria-expanded="false"><span class="name western">Lars Hagmeyer</span></a><div hidden="hidden" id="id39">
<h3><span class="name western">Lars Hagmeyer</span></h3>
<div class="p">
<sup>29</sup>Clinic of Pneumology and Allergology, Centre for Sleep Medicine and Respiratory Care, Bethanien Hospital Solingen, Solingen, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hagmeyer%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lars Hagmeyer</span></a>
</div>
</div>
<sup>29</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Doerr%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id40" aria-expanded="false"><span class="name western">Fabian Doerr</span></a><div hidden="hidden" id="id40">
<h3><span class="name western">Fabian Doerr</span></h3>
<div class="p">
<sup>21</sup>Department of Thoracic Surgery, University Medicine Essen – Ruhrlandklinik, University Duisburg-Essen, Essen, Germany </div>
<div class="p">
<sup>30</sup>Department of Cardiothoracic Surgery, University Hospital of Cologne, Cologne, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Doerr%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Fabian Doerr</span></a>
</div>
</div>
<sup>21,</sup><sup>30</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hekmath%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id41" aria-expanded="false"><span class="name western">Khosro Hekmath</span></a><div hidden="hidden" id="id41">
<h3><span class="name western">Khosro Hekmath</span></h3>
<div class="p">
<sup>30</sup>Department of Cardiothoracic Surgery, University Hospital of Cologne, Cologne, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hekmath%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Khosro Hekmath</span></a>
</div>
</div>
<sup>30</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Strapatsas%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id42" aria-expanded="false"><span class="name western">Judith Strapatsas</span></a><div hidden="hidden" id="id42">
<h3><span class="name western">Judith Strapatsas</span></h3>
<div class="p">
<sup>31</sup>Department of Haematology, Oncology and Clinical Immunology, University Hospital of Duesseldorf, Düsseldorf, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Strapatsas%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Judith Strapatsas</span></a>
</div>
</div>
<sup>31</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kambartel%20KO%22%5BAuthor%5D" class="usa-link" aria-describedby="id43" aria-expanded="false"><span class="name western">Karl-Otto Kambartel</span></a><div hidden="hidden" id="id43">
<h3><span class="name western">Karl-Otto Kambartel</span></h3>
<div class="p">
<sup>32</sup>Krankenhaus Bethanien Moers, Moers, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kambartel%20KO%22%5BAuthor%5D" class="usa-link"><span class="name western">Karl-Otto Kambartel</span></a>
</div>
</div>
<sup>32</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chakupurakal%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id44" aria-expanded="false"><span class="name western">Geothy Chakupurakal</span></a><div hidden="hidden" id="id44">
<h3><span class="name western">Geothy Chakupurakal</span></h3>
<div class="p">
<sup>33</sup>Praxis für Hämatologie und Onkologie, Koblenz, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chakupurakal%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Geothy Chakupurakal</span></a>
</div>
</div>
<sup>33</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Busch%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id45" aria-expanded="false"><span class="name western">Annette Busch</span></a><div hidden="hidden" id="id45">
<h3><span class="name western">Annette Busch</span></h3>
<div class="p">
<sup>34</sup>Medical Clinic III for Oncology, Haematology, Immune-Oncology and Rheumatology, Centre for Integrative Medicine, University Hospital Bonn, Bonn, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Busch%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Annette Busch</span></a>
</div>
</div>
<sup>34</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bauernfeind%20FG%22%5BAuthor%5D" class="usa-link" aria-describedby="id46" aria-expanded="false"><span class="name western">Franz-Georg Bauernfeind</span></a><div hidden="hidden" id="id46">
<h3><span class="name western">Franz-Georg Bauernfeind</span></h3>
<div class="p">
<sup>34</sup>Medical Clinic III for Oncology, Haematology, Immune-Oncology and Rheumatology, Centre for Integrative Medicine, University Hospital Bonn, Bonn, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bauernfeind%20FG%22%5BAuthor%5D" class="usa-link"><span class="name western">Franz-Georg Bauernfeind</span></a>
</div>
</div>
<sup>34</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Griesinger%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id47" aria-expanded="false"><span class="name western">Frank Griesinger</span></a><div hidden="hidden" id="id47">
<h3><span class="name western">Frank Griesinger</span></h3>
<div class="p">
<sup>35</sup>Pius-Hospital Oldenburg, Department of Haematology and Oncology, University Department Internal Medicine-Oncology, University Medicine Oldenburg, Oldenburg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Griesinger%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Frank Griesinger</span></a>
</div>
</div>
<sup>35</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Luers%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id48" aria-expanded="false"><span class="name western">Anne Luers</span></a><div hidden="hidden" id="id48">
<h3><span class="name western">Anne Luers</span></h3>
<div class="p">
<sup>35</sup>Pius-Hospital Oldenburg, Department of Haematology and Oncology, University Department Internal Medicine-Oncology, University Medicine Oldenburg, Oldenburg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Luers%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Anne Luers</span></a>
</div>
</div>
<sup>35</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dirks%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id49" aria-expanded="false"><span class="name western">Wiebke Dirks</span></a><div hidden="hidden" id="id49">
<h3><span class="name western">Wiebke Dirks</span></h3>
<div class="p">
<sup>35</sup>Pius-Hospital Oldenburg, Department of Haematology and Oncology, University Department Internal Medicine-Oncology, University Medicine Oldenburg, Oldenburg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dirks%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Wiebke Dirks</span></a>
</div>
</div>
<sup>35</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wiewrodt%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id50" aria-expanded="false"><span class="name western">Rainer Wiewrodt</span></a><div hidden="hidden" id="id50">
<h3><span class="name western">Rainer Wiewrodt</span></h3>
<div class="p">
<sup>36</sup>Pulmonary Division, Department of Medicine A, Münster University Hospital, Münster, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wiewrodt%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Rainer Wiewrodt</span></a>
</div>
</div>
<sup>36</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Luecke%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id51" aria-expanded="false"><span class="name western">Andrea Luecke</span></a><div hidden="hidden" id="id51">
<h3><span class="name western">Andrea Luecke</span></h3>
<div class="p">
<sup>36</sup>Pulmonary Division, Department of Medicine A, Münster University Hospital, Münster, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Luecke%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Andrea Luecke</span></a>
</div>
</div>
<sup>36</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rodermann%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id52" aria-expanded="false"><span class="name western">Ernst Rodermann</span></a><div hidden="hidden" id="id52">
<h3><span class="name western">Ernst Rodermann</span></h3>
<div class="p">
<sup>37</sup>Onkologie Rheinsieg, Praxisnetzwerk Hämatologie und Internistische Onkologie, Troisdorf, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rodermann%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Ernst Rodermann</span></a>
</div>
</div>
<sup>37</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Diel%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id53" aria-expanded="false"><span class="name western">Andreas Diel</span></a><div hidden="hidden" id="id53">
<h3><span class="name western">Andreas Diel</span></h3>
<div class="p">
<sup>37</sup>Onkologie Rheinsieg, Praxisnetzwerk Hämatologie und Internistische Onkologie, Troisdorf, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Diel%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Andreas Diel</span></a>
</div>
</div>
<sup>37</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hagen%20V%22%5BAuthor%5D" class="usa-link" aria-describedby="id54" aria-expanded="false"><span class="name western">Volker Hagen</span></a><div hidden="hidden" id="id54">
<h3><span class="name western">Volker Hagen</span></h3>
<div class="p">
<sup>38</sup>Clinic II for Internal Medicine, St.-Johannes-Hospital Dortmund, Dortmund, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hagen%20V%22%5BAuthor%5D" class="usa-link"><span class="name western">Volker Hagen</span></a>
</div>
</div>
<sup>38</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Severin%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id55" aria-expanded="false"><span class="name western">Kai Severin</span></a><div hidden="hidden" id="id55">
<h3><span class="name western">Kai Severin</span></h3>
<div class="p">
<sup>39</sup>Haematologie und Onkologie Köln MV-Zentrum, Cologne, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Severin%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kai Severin</span></a>
</div>
</div>
<sup>39</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ullrich%20RT%22%5BAuthor%5D" class="usa-link" aria-describedby="id56" aria-expanded="false"><span class="name western">Roland T Ullrich</span></a><div hidden="hidden" id="id56">
<h3><span class="name western">Roland T Ullrich</span></h3>
<div class="p">
<sup>3</sup>Department I of Internal Medicine, Centre for Integrated Oncology Aachen Bonn Cologne Duesseldorf, University Hospital Cologne, Cologne, Germany </div>
<div class="p">
<sup>5</sup>Centre for Molecular Medicine, University of Cologne, Cologne, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ullrich%20RT%22%5BAuthor%5D" class="usa-link"><span class="name western">Roland T Ullrich</span></a>
</div>
</div>
<sup>3,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Reinhardt%20HC%22%5BAuthor%5D" class="usa-link" aria-describedby="id57" aria-expanded="false"><span class="name western">Hans Christian Reinhardt</span></a><div hidden="hidden" id="id57">
<h3><span class="name western">Hans Christian Reinhardt</span></h3>
<div class="p">
<sup>40</sup>Department of Haematology and Stem Cell Transplantation, University Hospital Essen, Essen, Germany </div>
<div class="p">
<sup>41</sup>West German Cancer Centre, University Hospital Essen, Essen, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Reinhardt%20HC%22%5BAuthor%5D" class="usa-link"><span class="name western">Hans Christian Reinhardt</span></a>
</div>
</div>
<sup>40,</sup><sup>41</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Quaas%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id58" aria-expanded="false"><span class="name western">Alexander Quaas</span></a><div hidden="hidden" id="id58">
<h3><span class="name western">Alexander Quaas</span></h3>
<div class="p">
<sup>7</sup>Institute of Pathology, Medical Faculty, University Hospital Cologne, University of Cologne, Cologne, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Quaas%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Alexander Quaas</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bogus%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id59" aria-expanded="false"><span class="name western">Magdalena Bogus</span></a><div hidden="hidden" id="id59">
<h3><span class="name western">Magdalena Bogus</span></h3>
<div class="p">
<sup>42</sup>Institute of Legal Medicine, University of Cologne, Cologne, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bogus%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Magdalena Bogus</span></a>
</div>
</div>
<sup>42</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Courts%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id60" aria-expanded="false"><span class="name western">Cornelius Courts</span></a><div hidden="hidden" id="id60">
<h3><span class="name western">Cornelius Courts</span></h3>
<div class="p">
<sup>42</sup>Institute of Legal Medicine, University of Cologne, Cologne, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Courts%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Cornelius Courts</span></a>
</div>
</div>
<sup>42</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22N%C3%BCrnberg%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id61" aria-expanded="false"><span class="name western">Peter Nürnberg</span></a><div hidden="hidden" id="id61">
<h3><span class="name western">Peter Nürnberg</span></h3>
<div class="p">
<sup>43</sup>Cologne Centre for Genomics, West German Genome Centre, University of Cologne, Cologne, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22N%C3%BCrnberg%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Peter Nürnberg</span></a>
</div>
</div>
<sup>43</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Becker%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id62" aria-expanded="false"><span class="name western">Kerstin Becker</span></a><div hidden="hidden" id="id62">
<h3><span class="name western">Kerstin Becker</span></h3>
<div class="p">
<sup>43</sup>Cologne Centre for Genomics, West German Genome Centre, University of Cologne, Cologne, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Becker%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kerstin Becker</span></a>
</div>
</div>
<sup>43</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Achter%20V%22%5BAuthor%5D" class="usa-link" aria-describedby="id63" aria-expanded="false"><span class="name western">Viktor Achter</span></a><div hidden="hidden" id="id63">
<h3><span class="name western">Viktor Achter</span></h3>
<div class="p">
<sup>44</sup>Computing Centre, University of Cologne, Cologne, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Achter%20V%22%5BAuthor%5D" class="usa-link"><span class="name western">Viktor Achter</span></a>
</div>
</div>
<sup>44</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22B%C3%BCttner%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id64" aria-expanded="false"><span class="name western">Reinhard Büttner</span></a><div hidden="hidden" id="id64">
<h3><span class="name western">Reinhard Büttner</span></h3>
<div class="p">
<sup>7</sup>Institute of Pathology, Medical Faculty, University Hospital Cologne, University of Cologne, Cologne, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22B%C3%BCttner%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Reinhard Büttner</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wolf%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id65" aria-expanded="false"><span class="name western">Jürgen Wolf</span></a><div hidden="hidden" id="id65">
<h3><span class="name western">Jürgen Wolf</span></h3>
<div class="p">
<sup>6</sup>Department I of Internal Medicine, Lung Cancer Group Cologne, University Hospital Cologne, Cologne, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wolf%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jürgen Wolf</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Peifer%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id66" aria-expanded="false"><span class="name western">Martin Peifer</span></a><div hidden="hidden" id="id66">
<h3><span class="name western">Martin Peifer</span></h3>
<div class="p">
<sup>1</sup>Department of Translational Genomics, Faculty of Medicine and University Hospital Cologne, University of Cologne, Cologne, Germany </div>
<div class="p">
<sup>5</sup>Centre for Molecular Medicine, University of Cologne, Cologne, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Peifer%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Martin Peifer</span></a>
</div>
</div>
<sup>1,</sup><sup>5,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Thomas%20RK%22%5BAuthor%5D" class="usa-link" aria-describedby="id67" aria-expanded="false"><span class="name western">Roman K Thomas</span></a><div hidden="hidden" id="id67">
<h3><span class="name western">Roman K Thomas</span></h3>
<div class="p">
<sup>1</sup>Department of Translational Genomics, Faculty of Medicine and University Hospital Cologne, University of Cologne, Cologne, Germany </div>
<div class="p">
<sup>7</sup>Institute of Pathology, Medical Faculty, University Hospital Cologne, University of Cologne, Cologne, Germany </div>
<div class="p">
<sup>18</sup>DKFZ, German Cancer Research Centre, German Cancer Consortium, Heidelberg, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Thomas%20RK%22%5BAuthor%5D" class="usa-link"><span class="name western">Roman K Thomas</span></a>
</div>
</div>
<sup>1,</sup><sup>7,</sup><sup>18,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Translational Genomics, Faculty of Medicine and University Hospital Cologne, University of Cologne, Cologne, Germany </div>
<div id="Aff2">
<sup>2</sup>Department of Otorhinolaryngology, Head and Neck Surgery, Faculty of Medicine and University Hospital Cologne, University Hospital of Cologne, Cologne, Germany </div>
<div id="Aff3">
<sup>3</sup>Department I of Internal Medicine, Centre for Integrated Oncology Aachen Bonn Cologne Duesseldorf, University Hospital Cologne, Cologne, Germany </div>
<div id="Aff4">
<sup>4</sup>Cancer Research Centre Cologne Essen, Faculty of Medicine and University Hospital Cologne, University of Cologne, Cologne, Germany </div>
<div id="Aff5">
<sup>5</sup>Centre for Molecular Medicine, University of Cologne, Cologne, Germany </div>
<div id="Aff6">
<sup>6</sup>Department I of Internal Medicine, Lung Cancer Group Cologne, University Hospital Cologne, Cologne, Germany </div>
<div id="Aff7">
<sup>7</sup>Institute of Pathology, Medical Faculty, University Hospital Cologne, University of Cologne, Cologne, Germany </div>
<div id="Aff8">
<sup>8</sup>Institute of Medical Statistics, and Computational Biology, Faculty of Medicine and University Hospital Cologne, University of Cologne, Cologne, Germany </div>
<div id="Aff9">
<sup>9</sup>Department of Haematology, Oncology, Haemostaseology and Stem Cell Transplantation, University Hospital RWTH Aachen, Aachen, Germany </div>
<div id="Aff10">
<sup>10</sup>Centre for Integrated Oncology, Aachen Bonn Cologne Düsseldorf, Aachen, Germany </div>
<div id="Aff11">
<sup>11</sup>Department of Pneumology, City of Cologne Municipal Hospitals, Lung Hospital Cologne Merheim, Cologne, Germany </div>
<div id="Aff12">
<sup>12</sup>Thoraxclinic Cologne, Thoracic Surgery, St. Hildegardis-Krankenhaus, Cologne, Germany </div>
<div id="Aff13">
<sup>13</sup>Department of Pathology, City of Cologne Municipal Hospitals, Witten/Herdecke University, Cologne, Germany </div>
<div id="Aff14">
<sup>14</sup>Department of General Neurosurgery, Centre of Neurosurgery, University Hospital Cologne, Cologne, Germany </div>
<div id="Aff15">
<sup>15</sup>University Medicine Marburg – Campus Fulda, Department of Neurosurgery, Fulda, Germany </div>
<div id="Aff16">
<sup>16</sup>Department of Medicine II, Haematology/Oncology, University Hospital Frankfurt, Goethe University, Frankfurt, Germany </div>
<div id="Aff17">
<sup>17</sup>Frankfurt Cancer Institute, Goethe University Frankfurt, Frankfurt, Germany </div>
<div id="Aff18">
<sup>18</sup>DKFZ, German Cancer Research Centre, German Cancer Consortium, Heidelberg, Germany </div>
<div id="Aff19">
<sup>19</sup>Klinikum Würzburg Mitte – Missioklinik site, Pneumology and Respiratory Medicine, Würzburg, Germany </div>
<div id="Aff20">
<sup>20</sup>Translational Oncology/Early Clinical Trial Unit, Comprehensive Cancer Centre Mainfranken, University Hospital Wuerzburg, Wuerzburg, Germany </div>
<div id="Aff21">
<sup>21</sup>Department of Thoracic Surgery, University Medicine Essen – Ruhrlandklinik, University Duisburg-Essen, Essen, Germany </div>
<div id="Aff22">
<sup>22</sup>Department of Medical Oncology, West German Cancer Centre Essen, University Duisburg-Essen, Essen, Germany </div>
<div id="Aff23">
<sup>23</sup>Division of Thoracic Surgery, Department of General, Thoracic and Vascular Surgery, University Hospital Carl Gustav Carus, Dresden, Germany </div>
<div id="Aff24">
<sup>24</sup>Department of Thoracic Surgery, Medical University of Vienna, Vienna General Hospital, Vienna, Austria </div>
<div id="Aff25">
<sup>25</sup>Comprehensive Cancer Centre CIO Bonn, Bonn, Germany </div>
<div id="Aff26">
<sup>26</sup>Department of Respiratory Diseases, Evangelische Lungenklinik, Berlin, Germany </div>
<div id="Aff27">
<sup>27</sup>DGD Lungenklinik Hemer, Internal Medicine, Pneumology and Oncology, Hemer, Germany </div>
<div id="Aff28">
<sup>28</sup>Clinic III for Internal Medicine, Faculty of Medicine and University Hospital Cologne, University of Cologne, Cologne, Germany </div>
<div id="Aff29">
<sup>29</sup>Clinic of Pneumology and Allergology, Centre for Sleep Medicine and Respiratory Care, Bethanien Hospital Solingen, Solingen, Germany </div>
<div id="Aff30">
<sup>30</sup>Department of Cardiothoracic Surgery, University Hospital of Cologne, Cologne, Germany </div>
<div id="Aff31">
<sup>31</sup>Department of Haematology, Oncology and Clinical Immunology, University Hospital of Duesseldorf, Düsseldorf, Germany </div>
<div id="Aff32">
<sup>32</sup>Krankenhaus Bethanien Moers, Moers, Germany </div>
<div id="Aff33">
<sup>33</sup>Praxis für Hämatologie und Onkologie, Koblenz, Germany </div>
<div id="Aff34">
<sup>34</sup>Medical Clinic III for Oncology, Haematology, Immune-Oncology and Rheumatology, Centre for Integrative Medicine, University Hospital Bonn, Bonn, Germany </div>
<div id="Aff35">
<sup>35</sup>Pius-Hospital Oldenburg, Department of Haematology and Oncology, University Department Internal Medicine-Oncology, University Medicine Oldenburg, Oldenburg, Germany </div>
<div id="Aff36">
<sup>36</sup>Pulmonary Division, Department of Medicine A, Münster University Hospital, Münster, Germany </div>
<div id="Aff37">
<sup>37</sup>Onkologie Rheinsieg, Praxisnetzwerk Hämatologie und Internistische Onkologie, Troisdorf, Germany </div>
<div id="Aff38">
<sup>38</sup>Clinic II for Internal Medicine, St.-Johannes-Hospital Dortmund, Dortmund, Germany </div>
<div id="Aff39">
<sup>39</sup>Haematologie und Onkologie Köln MV-Zentrum, Cologne, Germany </div>
<div id="Aff40">
<sup>40</sup>Department of Haematology and Stem Cell Transplantation, University Hospital Essen, Essen, Germany </div>
<div id="Aff41">
<sup>41</sup>West German Cancer Centre, University Hospital Essen, Essen, Germany </div>
<div id="Aff42">
<sup>42</sup>Institute of Legal Medicine, University of Cologne, Cologne, Germany </div>
<div id="Aff43">
<sup>43</sup>Cologne Centre for Genomics, West German Genome Centre, University of Cologne, Cologne, Germany </div>
<div id="Aff44">
<sup>44</sup>Computing Centre, University of Cologne, Cologne, Germany </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2023 Jan 25; Accepted 2024 Feb 7; Issue date 2024.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2024</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC10972747&nbsp;&nbsp;PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/38480884/" class="usa-link">38480884</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2 data-anchor-id="Abs1"><a id="Abs1-anchor" data-anchor-id="Abs1" class="usa-anchor"></a>Abstract</h2>
<p id="Par1">The evolutionary processes that underlie the marked sensitivity of small cell lung cancer (SCLC) to chemotherapy and rapid relapse are unknown<sup><a href="#CR1" class="usa-link" aria-describedby="CR1" aria-expanded="false">1</a>–<a href="#CR3" class="usa-link" aria-describedby="CR3" aria-expanded="false">3</a></sup>. Here&nbsp;we determined tumour phylogenies at diagnosis and throughout chemotherapy and immunotherapy by multiregion sequencing of 160 tumours from 65 patients. Treatment-naive SCLC exhibited clonal homogeneity at distinct tumour sites, whereas first-line platinum-based chemotherapy led to a burst in genomic intratumour heterogeneity and spatial clonal diversity. We observed branched evolution and a shift to ancestral clones underlying tumour relapse. Effective radio- or immunotherapy induced a re-expansion of founder clones with acquired genomic damage from first-line chemotherapy. Whereas <em>TP53</em> and <em>RB1</em> alterations were exclusively part of the common ancestor, <em>MYC</em> family amplifications were frequently not constituents of the founder clone. At relapse, emerging subclonal mutations affected key genes associated with SCLC biology, and tumours harbouring clonal <em>CREBBP</em>/<em>EP300</em> alterations underwent genome duplications. Gene-damaging <em>TP53</em> alterations and co-alterations of <em>TP53</em> missense mutations with <em>TP73</em>, <em>CREBBP</em>/<em>EP300</em> or <em>FMN2</em> were significantly associated with shorter disease relapse following chemotherapy. In summary, we uncover key processes of the genomic evolution of SCLC under therapy, identify the common ancestor as the source of clonal diversity at relapse and show central genomic patterns associated with sensitivity and resistance to chemotherapy.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Tumour heterogeneity, Small-cell lung cancer, Cancer genomics, Chemotherapy, Cancer therapeutic resistance</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">We uncover key processes of the genomic evolution of small cell lung cancer under therapy, identify the common ancestor as the source of clonal diversity at relapse and show central genomic patterns associated with&nbsp;drug response.</p></section><section id="Sec1"><h2 class="pmc_sec_title" data-anchor-id="Sec1"><a id="Sec1-anchor" data-anchor-id="Sec1" class="usa-anchor"></a>Main</h2>
<p id="Par3">Small cell lung cancer (SCLC) is one of the deadliest human cancers, with a 5 year survival rate of less than 7%<sup><a href="#CR1" class="usa-link" aria-describedby="CR1" aria-expanded="false">1</a>–<a href="#CR4" class="usa-link" aria-describedby="CR4" aria-expanded="false">4</a></sup>. The standard of care for extensive-stage SCLC consists of systemic treatment with platinum and etoposide, recently combined with programmed death-ligand 1 (PD-L1) immune checkpoint inhibitors (ICIs)<sup><a href="#CR2" class="usa-link" aria-describedby="CR2" aria-expanded="false">2</a></sup>. One peculiarity of SCLC is its typically high sensitivity to platinum-based chemotherapy followed by rapid recurrence, which distinguishes it from most other human cancers. Unfortunately, second-line treatment with other chemotherapeutics or immunotherapy is only marginally effective and patients ultimately succumb to their disease<sup><a href="#CR1" class="usa-link" aria-describedby="CR1" aria-expanded="false">1</a>,<a href="#CR2" class="usa-link" aria-describedby="CR2" aria-expanded="false">2</a>,<a href="#CR4" class="usa-link" aria-describedby="CR4" aria-expanded="false">4</a></sup>.</p>
<p id="Par4">We and others have previously performed large-scale genome sequencing to comprehensively characterize cancer genome alterations in SCLC, which showed universal biallelic losses of the tumour suppressors <em>TP53</em> and <em>RB1</em>, additional alterations to histone-modifying enzymes&nbsp;and cell cycle regulators, and <em>MYC</em> transcription factor amplifications<sup><a href="#CR5" class="usa-link" aria-describedby="CR5" aria-expanded="false">5</a>–<a href="#CR7" class="usa-link" aria-describedby="CR7" aria-expanded="false">7</a></sup>. Furthermore, SCLC subgroups were defined on the basis of the&nbsp;expression of neuroendocrine lineage transcription factors, which impact tumour biology and treatment outcome<sup><a href="#CR4" class="usa-link" aria-describedby="CR4" aria-expanded="false">4</a>,<a href="#CR8" class="usa-link" aria-describedby="CR8" aria-expanded="false">8</a>,<a href="#CR9" class="usa-link" aria-describedby="CR9" aria-expanded="false">9</a></sup>. Finally, preliminary studies have provided initial clues in regard to molecular pathways associated with resistance to chemotherapy<sup><a href="#CR10" class="usa-link" aria-describedby="CR10" aria-expanded="false">10</a>,<a href="#CR11" class="usa-link" aria-describedby="CR11" aria-expanded="false">11</a></sup>.</p>
<p id="Par5">Despite progress in characterization of the molecular basis of SCLC, the underlying patterns of clonal evolution and the mechanisms causing drug resistance have remained unclear. We suggest that cancer genome alterations not only drive malignant transformation in SCLC but also influence the clinical phenotypes of chemotherapy sensitivity, tumour progression and relapse. We therefore performed comprehensive multiregional and longitudinal studies of tumours obtained from 65 patients to decipher the evolutionary and genomic principles that govern response and resistance to therapy in SCLC.</p></section><section id="Sec2"><h2 class="pmc_sec_title" data-anchor-id="Sec2"><a id="Sec2-anchor" data-anchor-id="Sec2" class="usa-anchor"></a>Tumour specimens and clinical data</h2>
<p id="Par6">We collected 160 tumour specimens from 65 patients with SCLC under institutional review board approval and performed whole-exome, genome and transcriptome sequencing of samples with an average tumour purity of 85% (Fig. <a href="#Fig1" class="usa-link">1a–c</a> and Supplementary Tables <a href="#MOESM4" class="usa-link">1</a>–<a href="#MOESM4" class="usa-link">3</a>). We most frequently sampled the primary lung tumour, pulmonary lymph nodes, liver, pleura and brain metastases. Furthermore, patient-derived xenotransplants were established from fine-needle biopsies or circulating tumour cells (CTCs),&nbsp;which have been previously shown to recapitulate the genomic profiles of patients’ tumours<sup><a href="#CR12" class="usa-link" aria-describedby="CR12" aria-expanded="false">12</a>,<a href="#CR13" class="usa-link" aria-describedby="CR13" aria-expanded="false">13</a></sup> (Fig. <a href="#Fig1" class="usa-link">1a</a> and <a href="#Sec9" class="usa-link">Methods</a>). The histology of SCLC was confirmed in all cases; additional components of adenocarcinoma or large-cell neuroendocrine carcinoma (LCNEC) were identified in three patients (Supplementary Table <a href="#MOESM4" class="usa-link">1</a>). The clinical history was typical of SCLC and the majority of patients had received first-line treatment with platinum-based chemotherapy, achieving a median relapse-free interval of 88 days (Fig. <a href="#Fig1" class="usa-link">1b</a>, Extended Data Table <a href="#Tab1" class="usa-link">1</a> and Supplementary Table <a href="#MOESM4" class="usa-link">1</a>). In line with clinical guidelines<sup><a href="#CR14" class="usa-link" aria-describedby="CR14" aria-expanded="false">14</a></sup> we grouped patients according to their duration of response to first-line chemotherapy, referring to the chemotherapy-free interval (CTFI) of 45, 90 and 180 days (Fig. <a href="#Fig1" class="usa-link">1b</a>). At relapse, 80% of these patients (<em>n</em> = 44 of 55) received additional lines of therapy, which included other chemotherapeutics or treatment with anti-PD-1 and/or anti-CTLA-4 ICIs (Supplementary Table <a href="#MOESM4" class="usa-link">1</a>).</p>
<figure class="fig xbox font-sm" id="Fig1"><h3 class="obj_head">Fig. 1. Tumour samples and clinical history of 65 patients with SCLC.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10972747_41586_2024_7177_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fee5/10972747/55e1cca0e7bc/41586_2024_7177_Fig1_HTML.jpg" loading="lazy" id="d33e1112" height="674" width="707" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong>, Tumour sites sampled from 65 patients with SCLC. Frequently sampled sites are highlighted in bold. Tumours were acquired either at the time of first diagnosis (treatment-naive) or following initiation of treatment (post-treatment). Tumour samples analysed as patient-derived xenotransplant (PDX) models are indicated. <strong>b</strong>, Schematic overview of the clinical course of 65 patients with SCLC. Patients were ordered according to their duration of response to first-line platinum-based chemotherapy, referring to a CTFI of 45, 90 and 180 days (National Comprehensive Cancer Network (NCCN) guidelines). Patients who, following initiation of first-line treatment, were either lost to follow-up or underwent surgical resection of the primary tumour were sorted to separate panels. The treatment administered to each patient is annotated and the clinical response is described as either complete response (CR), partial response (PR), stable disease (SD), progressive disease (PD) or mixed response (PR/PD). A detailed description of all clinical characteristics is provided in Supplementary Table <a href="#MOESM4" class="usa-link">1</a> and <a href="#Sec9" class="usa-link">Methods</a>. <strong>c</strong>, Schematic overview showing the analysis of paired, patient-matched tumour sites: paired studies of spatially distinct tumours at the time of first diagnosis (treatment-naive, <em>n</em> = 16); paired studies of tumour sites pretreatment and during treatment (<em>n</em> = 5) or at clinical relapse following completion of first-line platinum-based chemotherapy (<em>n</em> = 42); paired analyses of spatially distinct tumour sites at relapse (<em>n</em> = 14); and analyses of tumours acquired before and after subsequent lines of treatment with ICIs (<em>n</em> = 7). The scheme shows tumour sites in the lung, referring to primary and metastatic sites (larger and smaller red circles, respectively). LN, lymph node.</p></figcaption></figure><section class="tw xbox font-sm" id="Tab1"><h3 class="obj_head">Extended Data Table 1.</h3>
<div class="caption p"><p>Information on clinical characteristics and tumour samples acquired from 65 patients with SCLC</p></div>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10972747_41586_2024_7177_Tab1_ESM.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fee5/10972747/03964d125741/41586_2024_7177_Tab1_ESM.jpg" loading="lazy" id="d33e4026" height="996" width="759" alt="graphic file with name 41586_2024_7177_Tab1_ESM.jpg"></a></p>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p114">
<p>Upper panel, overview on the clinical information for 65 patients with SCLC. Additional information is provided in Supplementary Table <a href="#MOESM1" class="usa-link">1</a>.</p>
<p>Lower panel, overview on multi-regional tumour sites acquired from&nbsp;65 patients with SCLC. The number of&nbsp;tumour samples acquired at distinct timepoints throughout the clinical course of treatment is indicated. Paired tumour analyses were performed focusing on distinct clinical scenarios to thus permit interpatient comparisons (compare Fig. <a href="#Fig1" class="usa-link">1b</a>, Supplementary Table <a href="#MOESM4" class="usa-link">2</a>).</p>
</div></div></section><p id="Par7">We analysed at least two tumour samples per patient, obtained at either single or multiple time points throughout the course of treatment. For interpatient comparisons we focused on paired studies of tumours acquired under distinct scenarios throughout the clinical course of the patients: (1) spatially distinct tumour samples in the treatment-naive setting at the time of first diagnosis (<em>n</em> = 16); (2) temporally distinct tumours acquired at first diagnosis before initiation of therapy and either during first-line platinum-based chemotherapy (<em>n</em> = 5) or following completion of chemotherapy (<em>n</em> = 42); (3) spatially, but not temporally, separate tumours analysed solely at the time of relapse (<em>n</em> = 14); and (4) tumours obtained before and after subsequent lines of treatment with immunotherapy (<em>n</em> = 7) (Fig. <a href="#Fig1" class="usa-link">1c</a>, Extended Data Table <a href="#Tab1" class="usa-link">1</a> and <a href="#Sec9" class="usa-link">Methods</a>).</p></section><section id="Sec3"><h2 class="pmc_sec_title" data-anchor-id="Sec3"><a id="Sec3-anchor" data-anchor-id="Sec3" class="usa-anchor"></a>Tumour phylogenies of metastatic SCLC</h2>
<p id="Par8">Aiming to shed light on the dynamics of genome evolution in metastatic SCLC, we performed genome sequencing of all tumour specimens to identify genomic alterations. Whole-exome sequencing data at an average coverage of 127-fold were used to compute cancer cell fraction (CCF) for somatic mutations, a metric of relative abundance of mutant alleles corrected for purity, ploidy and absolute copy number, which affords the assignment of mutations to individual tumour clones and enables tracking of single clones in spatially and temporally distinct tumours<sup><a href="#CR15" class="usa-link" aria-describedby="CR15" aria-expanded="false">15</a></sup> (<a href="#Sec9" class="usa-link">Methods</a>). We assigned mutations to the most recent common ancestor (C0) if mutations were shared and clonal with CCFs of 100% across all samples analysed, and to subclones (C1, C2 and C3) if clusters of mutations were either private to specific tumour sites or found at lower CCF. We thus reconstructed the clonal lineage and determined tumour phylogenies for all 65 patients (<a href="#Sec30" class="usa-link">Supplementary Appendix</a>, Supplementary Tables <a href="#MOESM4" class="usa-link">2</a> and <a href="#MOESM4" class="usa-link">4</a> and <a href="#Sec9" class="usa-link">Methods</a>).</p>
<p id="Par9">Previous genomic studies conducted for single&nbsp;tumour sites obtained from treatment-naive patients indicated low levels of genomic intratumour heterogeneity in SCLC compared with lung adenocarcinoma<sup><a href="#CR5" class="usa-link" aria-describedby="CR5" aria-expanded="false">5</a></sup>. Through analysis of spatially and temporally distinct tumours, we now observed a wide range in the absolute number of subclonal mutations and subclones previously observed in other cancers as well<sup><a href="#CR16" class="usa-link" aria-describedby="CR16" aria-expanded="false">16</a></sup> (Extended Data Fig. <a href="#Fig6" class="usa-link">1a,b</a>). Tumour phylogenies across all patients exhibited patterns of linear and branched evolution, in some cases indicating a sequential acquisition of genome alterations and thereby giving rise to a dominant clone. In other patients, emerging subclones branched from ancestral clones thus creating multiple lineages<sup><a href="#CR16" class="usa-link" aria-describedby="CR16" aria-expanded="false">16</a></sup>. For systematic study of the evolutionary patterns we assigned tumour phylogenies to distinct classes (Fig. <a href="#Fig2" class="usa-link">2b,c</a>): class A, if no subclones were identified, which was frequently observed when comparing more than one anatomic site at a single time point (Fisher’s exact test, **<em>P</em> &lt; 0.01; Fig. <a href="#Fig2" class="usa-link">2c</a>); classes B and C, with one or at least two subclones, respectively, compatible with linear phylogenies; classes D and E, phylogenies with one branching event in which tumour clones descend from either C1 subclones (class D) or the common ancestral clone C0 (class E); and class F, phylogenies with at least two branching events exclusively identified in patients, with higher numbers of specimens referring to at least three spatially or temporally distinct tumours (Fisher’s exact test, <em>P</em> &lt; 0.001; Fig. <a href="#Fig2" class="usa-link">2c</a> and Extended Data Fig. <a href="#Fig6" class="usa-link">1c</a>), thus providing further information on phylogenetic complexity. To permit interpatient comparisons we therefore sought to perform paired analyses, considering a maximum of two samples per patient (Fig. <a href="#Fig2" class="usa-link">2d</a> and Extended Data Table <a href="#Tab1" class="usa-link">1</a>), which did not show any significant change in the absolute number of subclonal mutations but led to reduced phylogenetic complexities assigned to classes A–E (<a href="#Sec9" class="usa-link">Methods</a> and Extended Data Fig. <a href="#Fig6" class="usa-link">1a,b</a>). We thus observed a significantly lower clonal diversity in treatment-naive patients across different tumour sites than in temporally and spatially distinct tumours from patients undergoing treatment (**<em>P</em> &lt; 0.01; Fig. <a href="#Fig2" class="usa-link">2d,e</a>). Consequently the genomic heterogeneity of the tumour—although limited at diagnosis—increased markedly as a result of therapeutic intervention.</p>
<figure class="fig xbox font-sm" id="Fig6"><h3 class="obj_head">Extended Data Fig. 1. Clonality analysis on tumours from 65 patients with SCLC.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10972747_41586_2024_7177_Fig6_ESM.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fee5/10972747/e203e02894e1/41586_2024_7177_Fig6_ESM.jpg" loading="lazy" id="d33e3515" height="1260" width="765" alt="Extended Data Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong>, Number of subclonal mutations (left panel, box plot displaying median with interquartile range) and number of subclones (right panel, displaying the median); whiskers indicate the range of minimum and maximum value. Mann-Whitney <em>U</em>-test, two-sided, at P &lt; 0.05. NS,&nbsp;not significant. <strong>b</strong>, the assignment of phylogeny classes (<strong>b</strong>) for 65 patients with SCLC and for the analyses of n = 84 paired tumour analyses to permit interpatient comparisons for distinct the clinical scenarios described in Fig. <a href="#Fig1" class="usa-link">1b</a> (Methods, Supplementary Table <a href="#MOESM4" class="usa-link">4</a>). <strong>c</strong>, Patient cases assigned to distinct classes of tumour phylogenies plotting the number of distinct timepoints and samples considered for the assignment. The median is indicated by grey lines. <strong>d</strong>, Level of subclonal&nbsp;mutations determined from multi-regional samples at relapse, focusing on paired analyses of distinct samples acquired from a given tumour site (n = 9 patients, dark blue) and for spatially distinct inter-metastatic sites (n = 5 patients, red). For comparisons, subclonal mutations determined from spatially distinct sites in treatment-naïve patients (n = 16) are plotted (grey). Data is presented as median with whiskers indicating the interquartile range. Mann-Whitney <em>U</em>-test, two-sided, P** &lt; 0.01. <strong>e</strong>, Clonal dynamics in patient <a href="https://www.ncbi.nlm.nih.gov/nuccore/S02783" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S02783</a> with phylogeny class D, tracking tumour clones at the site of the primary after neo-adjuvant chemotherapy (sample S1) and at relapse (samples S2 and S3). The clonal dynamics and the clonal composition are provided for all three samples. <strong>f</strong>, Level of subclonal diversity determined from the same tumour site (primary tumour, LN or liver metastases) sampled pre- and post-first-line chemotherapy referring to the number of subclonal mutations (left panel), the number of clones (middle panel), and to the assignment of phylogeny classes (right panel). Relapsing tumours revealing ancestral C0 and C1 tumour clones are plotted (figure at the right). Data is presented as median with whiskers indicating the interquartile range. <strong>g</strong>, Level of&nbsp;subclonality referring to the number of&nbsp;subclonal mutations assigned for distinct clinical settings. The data presented is a subset of the data presented in Fig. <a href="#Fig2" class="usa-link">2e</a>, and refers only to cases for which the paired include samples from PDX models. Mann-Whitney <em>U</em>-test, two-sided, P** &lt; 0.01;.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig2"><h3 class="obj_head">Fig. 2. Tumour phylogenies and clonal dynamics in 65 patients with SCLC.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10972747_41586_2024_7177_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fee5/10972747/0cf369d69f58/41586_2024_7177_Fig2_HTML.jpg" loading="lazy" id="d33e1286" height="699" width="708" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong>, Schematic of clone phylogeny depicting the most recent common ancestral clone, C0, descending C1, C2 and C3 and subsequent subclones numbered accordingly. <strong>b</strong>, Phylogeny classes: class A, no subclones; linear phylogenies with one subclone (class B) or at least two subclones (class C); phylogenies with one branching event from C1 subclones (class D) or the common ancestral clone C0 (class E), or at least two branching events (class F). <strong>c</strong>, Number of samples and distinct time points associated with phylogeny class for each patient (Fisher’s exact test, two-sided, **<em>P</em> &lt; 0.01). <strong>d</strong>, Tumour phylogenies at distinct clinical scenorios determined for each patient from paired analyses of WES data (samples S1 and S2; Fig. <a href="#Fig1" class="usa-link">1c</a> and <a href="#Sec9" class="usa-link">Methods</a>), sorted according to the number of clones and subclonal mutations (top), showing site-specific CCF of identified clones (intratumour heterogeneity, middle) and phylogeny class (bottom). Cases pre- and post-platinum-based chemotherapy are sorted according to clinical response and exposure of tumour sites to radiation (Rx, green line). Double-headed arrows represent comparisons of distinct samples from the primary tumour and either intrapulmonary metastases (dark blue) or extrapulmonary metastases (light blue), or within intermetastatic sites (red). Asterisks mark samples from PDX models. <strong>e</strong>–<strong>g</strong>, Subclonal mutations, tumour clones and phylogeny class (median with whiskers representing minimum and maximum values) under distinct clinical scenarios. <strong>e</strong>, Branching evolution (classes D and E). Fisher’s exact test, two-sided, *<em>P</em> &lt; 0.05. <strong>f</strong>,<strong>g</strong>, Spatially distinct sites from treatment-naive setting (<strong>f</strong>) and pre-/post-first-line platinum-based chemotherapy (<strong>g</strong>). **<em>P</em> &lt; 0.01; Mann–Whitney <em>U</em>-test, two-sided, not significant. <strong>h</strong>, Clonal dynamics of patient S02706 for tumours acquired before (S1), after neo-adjuvant chemotherapy (S2) and at relapse (S3). <strong>i</strong>, Proportion of ancestral C0 or C1 clones in relapsing tumours.</p></figcaption></figure><p id="Par10">We sought to determine the subclonal composition at the time of first diagnosis to study the evolutionary dynamics of tumour progression in a highly metastatic disease. Our analysis in these treatment-naive patients included spatially distinct intra- and extrapulmonary sites exhibiting either no evidence of subclones (class A) or limited mutational changes with patterns of linear evolution (Fig. <a href="#Fig2" class="usa-link">2d</a>, left). Clonal diversity was lower when comparing metastatic sites with one another, which frequently included CTC-derived tumours and confirmed earlier observations<sup><a href="#CR12" class="usa-link" aria-describedby="CR12" aria-expanded="false">12</a>,<a href="#CR13" class="usa-link" aria-describedby="CR13" aria-expanded="false">13</a></sup>. However, tumour regions simultaneously obtained from the primary site and intrapulmonary metastases exhibited increased subclonal mutations (**<em>P</em> &lt; 0.01) and branched evolutionary processes (classes D and E; Fig. <a href="#Fig2" class="usa-link">2e</a>). Thus, following the successful establishment of metastases,&nbsp;the subclonal composition appeared largely unchanged. Additionally, increased clonal heterogeneity and ongoing evolution appeared to occur during the first steps of metastatic seeding in the physical proximity of the original founder clone, driving the outgrowth of one rapidly expanding tumour.</p>
<p id="Par11">We next analysed the impact of chemotherapy on the dynamics of tumour evolution and compared tumours before therapy with tumour sites acquired during treatment and at the time of relapse. Most tumours exhibited clonal branching from ancestral clones C1 or C0 (67%, <em>n</em> = 31 of 46, <em>P</em> &lt; 0.05) under therapy, causing increased site-specific intratumour heterogeneity (sample 2; Fig. <a href="#Fig2" class="usa-link">2d,e</a>) and spatial clonal diversity when comparing specimens sampled simultaneously from different sites at relapse (**<em>P</em> &lt; 0.01; Extended Data Fig. <a href="#Fig6" class="usa-link">1d</a>). In two patients we tracked the evolutionary processes at the site of the primary tumour before and during therapy, following neo-adjuvant chemotherapy and at the time of subsequent relapse (Fig. <a href="#Fig2" class="usa-link">2h</a> and Extended Data Fig. <a href="#Fig6" class="usa-link">1e</a>). In both cases we found phylogenies of class D showing several distinct clones at the site of the primary tumour and repression of the initial dominating clone by chemotherapy, followed by the emergence and expansion of subclones descending from ancestral clone C1 that had caused relapse. Class D phylogenies were frequently identified in comparison with primary lung specimens (Fig. <a href="#Fig2" class="usa-link">2g</a>; <em>n</em> = 6 of seven cases), again emphasizing that the site of the primary tumour serves as a source for ancestral clones that cause metastatic seeding and tumour recurrence. At relapse, both tumour sites exposed to treatment and newly formed metastatic lesions harboured a substantial fraction of pre-existing ancestral clones, most frequently the common ancestor C0 (<em>n</em> = 16 of 42, 38%), confirming its critical role in relapse (Fig. <a href="#Fig2" class="usa-link">2i</a>). Because these branching events were frequently detected in comparisons from different sites, we next analysed repeated biopsies from the same site over time (<em>n</em> = 9) and found branching events and the presence of ancestral clones at relapse in these as well (Extended Data Fig. <a href="#Fig6" class="usa-link">1f</a>). Furthermore, focusing the analysis on samples derived from xenotransplant models similarly showed a significant increase in subclonal mutations following treatment (Extended Data Fig. <a href="#Fig6" class="usa-link">1g</a>), suggesting no bias with regard to sample type.</p>
<p id="Par12">Our data thus suggest that neither is&nbsp;the observed level of evolutionary heterogeneity driven by different anatomic sites nor does first-line chemotherapy primarily drive linear evolution of tumour clones to the state of relapse. By contrast, our data support the view that one highly proliferating clone dominates the tumour at the time of first diagnosis, representing pseudo-clonality<sup><a href="#CR16" class="usa-link" aria-describedby="CR16" aria-expanded="false">16</a></sup> which is then suppressed and eliminated by therapy. At clinically overt recurrence, a multitude of subclones has emerged that are driven by the most recent common ancestor, which markedly increases spatial and intratumour heterogeneity.</p></section><section id="Sec4"><h2 class="pmc_sec_title" data-anchor-id="Sec4"><a id="Sec4-anchor" data-anchor-id="Sec4" class="usa-anchor"></a>Mutation signatures of clonal diversity</h2>
<p id="Par13">To pinpoint the underlying processes that cause the observed treatment-dependent increase in clonal diversity, we determined signatures for mutations defining the common ancestor and subclones<sup><a href="#CR17" class="usa-link" aria-describedby="CR17" aria-expanded="false">17</a></sup>. Confirming previous studies in lung cancer<sup><a href="#CR18" class="usa-link" aria-describedby="CR18" aria-expanded="false">18</a>–<a href="#CR21" class="usa-link" aria-describedby="CR21" aria-expanded="false">21</a></sup>, age-like and tobacco-associated processes dominated within the mutations of the common ancestor, which correlated with the level of smoking in these patients (Spearman correlation = 0.39, **<em>P</em> &lt; 0.01; Fig. <a href="#Fig3" class="usa-link">3a</a> and Extended Data Fig. <a href="#Fig7" class="usa-link">2a,b</a>). Furthermore, clonal mutational processes in some patients were related to apolipoprotein B messenger RNA-editing enzyme, catalytic polypeptide (APOBEC), defective DNA repair and aflatoxin, the latter previously associated with lung cancer<sup><a href="#CR22" class="usa-link" aria-describedby="CR22" aria-expanded="false">22</a></sup>. Mutational processes assigned to subclones were less frequently associated with tobacco exposure, and we observed a predominance of clock-like signatures shaping subclonal mutations in both treatment-naive and recurring tumours (Fig. <a href="#Fig3" class="usa-link">3a–c</a> and Extended Data Fig. <a href="#Fig7" class="usa-link">2c</a>), implying that branching from ancestral clones involved acquisition of mutations at a steady rate, which may have happened earlier throughout the patient’s lifetime<sup><a href="#CR21" class="usa-link" aria-describedby="CR21" aria-expanded="false">21</a></sup>. We furthermore identified, in a subset of patients,&nbsp;mutational patterns associated with platinum-based chemotherapy (single-base substitutions SBS31 and SBS35), which were presumably acquired during first-line chemotherapy<sup><a href="#CR23" class="usa-link" aria-describedby="CR23" aria-expanded="false">23</a>,<a href="#CR24" class="usa-link" aria-describedby="CR24" aria-expanded="false">24</a></sup> (Fig. <a href="#Fig3" class="usa-link">3a</a> and Extended Data Fig. <a href="#Fig7" class="usa-link">2d</a>).</p>
<figure class="fig xbox font-sm" id="Fig3"><h3 class="obj_head">Fig. 3. Mutation signatures of subclones.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10972747_41586_2024_7177_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fee5/10972747/578bb2238699/41586_2024_7177_Fig3_HTML.jpg" loading="lazy" id="d33e1467" height="496" width="706" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong>, Mutational signatures of SBS assigned to clonal (ancestral clone C0) and subclonal mutations in treatment-naive and post-treatment tumours. <strong>b</strong>, Subclonal mutations determined for multiregional samples from treatment-naive patients (grey, left) and for tumours pre-/post-first-line systemic platinum-based chemotherapy (middle and right). Patients are grouped according to clinical response (middle) and exposure of relapsing tumours to previous radiation. Median and interquartile range, minimum and maximum values. Mann–Whitney <em>U</em>-test, two-sided, *<em>P</em> &lt; 0.05, **<em>P</em> &lt; 0.01. <strong>c</strong>, Mutational signatures for paired pre-/post-treatment samples from patients receiving chemotherapy. Relative contributions assigned to clonal (grey) and subclonal mutations of pre-therapy (blue) and post-therapy tumours exposed to platinum-based chemotherapy (blue, <em>n</em> = 21) and to additional site-specific radiation (pink, <em>n</em> = 25). Median and interquartile range and whiskers (minimum and maximum values). Paired two-sided Wilcoxon test, **<em>P</em> &lt; 0.01. <strong>d</strong>, Seven patients were receiving second- or third-line treatment with ICI, and the scheme for their clinical course is shown in Fig. <a href="#Fig1" class="usa-link">1b</a>. Waterfall plot showing tumour site-specific response to ICI (lower right). Numbers of private subclonal mutations pre- and post-ICI, grouped according to clinical response (lower right, median with maximum and minimum values). <strong>e</strong>,<strong>f</strong>, Clonal dynamics at first diagnosis (treatment-naive, grey box), at relapse following first-line chemotherapy (post-chemotherapy, orange arrows and dashed box) and following treatment with ICI (post-ICI, blue arrows and dashed box). Arrows assigned to branches of clone trees indicate the relative contribution of mutational signatures in ancestral clone C0 and subclones. Site-specific CCFs of tumour clones are plotted. Clinical response to the respective treatment is indicated, distinguishing patients with progressive disease (<strong>e</strong>) and partial response (<strong>f</strong>) under ICI. <strong>g</strong>, Relative contribution of mutational signatures in patients receiving ICI assigned to clonal and subclonal mutations of tumours post-chemotherapy and post-ICI. NS, not significant.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig7"><h3 class="obj_head">Extended Data Fig. 2. Mutational signatures in 65 patients with SCLC.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10972747_41586_2024_7177_Fig7_ESM.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fee5/10972747/68eaf99bdfee/41586_2024_7177_Fig7_ESM.jpg" loading="lazy" id="d33e3573" height="1008" width="731" alt="Extended Data Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong>, Relative contribution of mutational processes in treatment-naïve tumours, referring to clonal mutations as part of the common ancestral clone (left panel, determined for n = 58 treatment-naïve patient tumours) and to subclonal mutations (right panel, determined for n = 20/58 patient tumours). Mutational signatures refer to single base substitutions (SBS) defined in COSMIC (<em>Alexandrov et al, Nature 2020</em>; Supplementary Table <a href="#MOESM4" class="usa-link">5</a>). <strong>b</strong>, Correlation of the amount of smoking determined as packyears (PY) with the relative contribution of mutational processes defined for clonal mutations. Correlations with tobacco exposure were performed for n = 48 cases for which the amount of smoking was documented. Spearman’s rho correlation coefficient determined, significance at P* &lt; 0.05. <strong>c</strong>, Activity of mutational signatures assigned to clonal (grey, n = 58) and subclonal proportion of tumours determined for n = 20 treatment-naïve (blue) and n = 35 post-treatment tumours (orange). Data is presented as box plot displaying median with interquartile range and whiskers indicating maximum and minimum values. Mann-Whitney <em>U</em>-tests, two-sided, at P* &lt; 0.05 and P*** &lt; 0.001. Significant differences for all groups are highlighted by dashed boxes. NS, not significant. <strong>d</strong>, Activity of platinum-based mutational signatures (SBS31, SBS35) in patient-matched paired analysis of the clonal proportion, and the subclonal proportion&nbsp;of treatment-naïve tumours and of tumours after first-line platinum-based therapy. Significance determined for n = 24 patients with pre-/post-therapy tumours, Wilcoxon test, two-sided, P*** &lt; 0.001. Data is presented as box plot showing median with interquartile range and whiskers indicating maximum and minimum values. <strong>e</strong>, Activity of mutational signatures assigned to the&nbsp;subclonal proportion of post-treatment tumours following first-line platinum-based therapy. Patient samples are grouped according to clinical remissions or stable disease (blue, PR, SD) or progressive disease (red, PD and mixed responses). Mann Whitney <em>U</em>-test, two-sided, significance determined at P* &lt; 0.05,&nbsp;NS not signficant. Data is presented as box plot showing median with interquartile range and whiskers indicating maximum and minimum values. <strong>f</strong>, Rates of insertions and deletions (indels) versus single base substitutions (SBS), and rates for deletions versus insertions in tumours acquired from treatment-naïve patients and post-treatment following platinum-based chemotherapy without and with additional site-specific exposure to radiation (“no Rx” and “with Rx”). Data presented as box plot with median and interquartile range and whiskers for maximum and minimum value. Ratios were determined for whole genome sequencing data (genome-wide, top plots, Mann Whitney <em>U</em>-test, two-sided,) and whole exome sequencing data (exome-wide, bottom plots, Wilcoxon matched pair test). NS,&nbsp;not significant&nbsp;at P &lt; 0.05. <strong>g</strong>, Relative contribution of mutational processes assigned to clonal and&nbsp;subclonal mutations in paired studies of pre-treatment (treatment-naïve) and post-treatment tumours following platinum-based chemotherapy (cases, left panel) and additional site-specific exposure to radiation (Rx) (cases, right panel). <strong>h</strong>, Relative contribution of mutational signatures assigned to platinum chemotherapy (SBS31) for paired studies of tumours from n = 25 patients with subclonal mutations identified in the treatment-naïve setting and after treatment with chemotherapy (post-chemotherapy) and additional site-specific exposure to radiation (Rx). The analysis was performed with CaMuS (<em>Cartolano et al., Scientific Reports, 2020</em>; right plot; Supplementary Table <a href="#MOESM4" class="usa-link">5</a>, Methods).</p></figcaption></figure><p id="Par14">We proposed that the extent of subclonal diversity and associated mutational signatures at relapse relate directly to the type and efficacy of previous treatment. Patients with clinical response to systemic treatment with first-line platinum-based chemotherapy exhibited a significant increase in subclonal mutations when analysing tumours before treatment and at relapse (**<em>P</em> &lt; 0.01); by contrast, the number of subclonal mutations in specimens before and after chemotherapy from patients with refractory SCLC did not differ significantly compared with the level of subclonality determined for multiregional samples in treatment-naive patients (Fig. <a href="#Fig3" class="usa-link">3b</a>). These observations support the notion that treatment fails to suppress the original dominating clone in chemorefractory patients whereas successful chemotherapy eliminates the most abundant clone, which is followed by the observed expansion of a multitude of subclones.</p>
<p id="Par15">The level of subclonal mutations differed substantially across samples (Fig. <a href="#Fig3" class="usa-link">3b</a>), and we could not identify specific mutational processes that related to the efficacy of chemotherapy in these patients (Extended Data Fig. <a href="#Fig7" class="usa-link">2e</a>). By contrast, independent of the overall clinical response, we found a significant increase in subclonal mutations when analysing those tumour sites at relapse that had also been exposed to radiotherapy (Fig. <a href="#Fig3" class="usa-link">3b</a> and <a href="#Sec9" class="usa-link">Methods</a>). Ionizing radiation does not typically induce signatures marked by single-base substitutions, and we could not identify other signs of radiation-induced DNA damage in tumours at relapse<sup><a href="#CR25" class="usa-link" aria-describedby="CR25" aria-expanded="false">25</a></sup> (Extended Data Fig. <a href="#Fig7" class="usa-link">2f</a>). To our surprise, however, paired studies of pre- and post-therapy tumours frequently showed platinum-associated genomic scars in those sites previously exposed to radiotherapy (Fig. <a href="#Fig3" class="usa-link">3c</a> and Extended Data Fig. <a href="#Fig7" class="usa-link">2g–i</a>). The mutational patterns that underlie platinum damage have previously been identified both analytically and experimentally<sup><a href="#CR23" class="usa-link" aria-describedby="CR23" aria-expanded="false">23</a>,<a href="#CR24" class="usa-link" aria-describedby="CR24" aria-expanded="false">24</a></sup>, and our own confidence in the respective assignments is based on both the large number of specimens (26%, <em>n</em> = 12 of 46) and significant increase in platinum damage in tumours at relapse (**<em>P</em> &lt; 0.01; Fig. <a href="#Fig3" class="usa-link">3c</a>, Extended Data Fig. <a href="#Fig7" class="usa-link">2d</a> and Supplementary Table <a href="#MOESM4" class="usa-link">5</a>). Although we have no formal explanation for this observation, our data are compatible with the view that marked tumour growth suppression by radiotherapy permits the outgrowth of diverse subclones, including tumour clones that had acquired genomic scars from previous lines of chemotherapy<sup><a href="#CR23" class="usa-link" aria-describedby="CR23" aria-expanded="false">23</a></sup>.</p></section><section id="Sec5"><h2 class="pmc_sec_title" data-anchor-id="Sec5"><a id="Sec5-anchor" data-anchor-id="Sec5" class="usa-anchor"></a>Tumour evolution under immunotherapy</h2>
<p id="Par16">We reasoned that the burst in clonal diversity induced by chemotherapy might impact the efficacy of any subsequent treatments such as ICI. We therefore analysed the evolutionary dynamics in seven patients who had received, as second- or third-line treatment, the PD-1 inhibitor nivolumab, alone or in combination with the CTLA-4 inhibitor ipilimumab (clinical trial no. <a href="https://clinicaltrials.gov/study/NCT03083691" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NCT03083691</a>). We sampled tumour biopsies before and after treatment with ICI, and in five patients we also performed comparisons with the treatment-naive tumour acquired at the time of first diagnosis (Figs. <a href="#Fig1" class="usa-link">1b,c</a>,&nbsp;<a href="#Fig2" class="usa-link">2d</a> and&nbsp;<a href="#Fig3" class="usa-link">3d</a> and Extended Data Fig. <a href="#Fig8" class="usa-link">3a</a>). Two patients experienced disease stabilization throughout treatment with ICI and, in agreement with radiological disease assessment, subclonal tumour cell populations before and throughout immunotherapy were conserved (Fig. <a href="#Fig3" class="usa-link">3d</a> and Extended Data Fig. <a href="#Fig8" class="usa-link">3b,c</a>). Two patients who progressed under immunotherapy exhibited a limited but detectable change in subclonal mutations, and assignment of tumour clones showed shifts to ancestral clones already existing before the initiation of ICI (Fig. <a href="#Fig3" class="usa-link">3d,e</a> and Extended Data Fig. <a href="#Fig8" class="usa-link">3b</a>). Thus, tumour progression under immunotherapy led to the expansion of subclones already extant at the time of relapse. This was similarly observed in one patient who experienced an initial clinical response to ICI (<a href="https://www.ncbi.nlm.nih.gov/protein/S02775" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S02775</a>; Extended Data Fig. <a href="#Fig8" class="usa-link">3d</a>). By contrast, two patients who experienced tumour shrinkage under ICI showed an increase in subclonal mutations at the time of relapse (<a href="https://www.ncbi.nlm.nih.gov/protein/S02764" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S02764</a> and <a href="https://www.ncbi.nlm.nih.gov/protein/S03325" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S03325</a>; Fig. <a href="#Fig3" class="usa-link">3d</a> and Extended Data Fig. <a href="#Fig8" class="usa-link">3b</a>). In comparison with corresponding treatment-naive tumours, we found that these subclones originated from ancestral clones that were dominant at the time of first diagnosis in these patients (Fig. <a href="#Fig3" class="usa-link">3f</a>). Thus, tumour clones that initially dominated tumour sites at the time of first diagnosis—and that had effectively been suppressed by first-line chemotherapy and not identified at the time of relapse—reappeared and provided the seed for tumours causing relapse following subsequent lines of immunotherapy. Furthermore, similar to our observation in irradiated tumours, recurring tumour clones dominating at relapse following effective immunotherapy exhibited imprints of platinum-based DNA damage (<em>n</em> = 4 of five patients with stable disease and partial response; Fig. <a href="#Fig3" class="usa-link">3f,g</a>, Extended Data Fig. <a href="#Fig7" class="usa-link">2c</a> and Supplementary Table <a href="#MOESM4" class="usa-link">5</a>). The emerging subclone with signs of platinum-based DNA damage was not detectable at the time of relapse from first-line chemotherapy in two patients—before initiation of successful treatment with ICI (Fig. <a href="#Fig3" class="usa-link">3g</a>). Of note, the tumour obtained before immunotherapy from patient <a href="https://www.ncbi.nlm.nih.gov/protein/S03325" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S03325</a> contained a subclone with a signature of platinum-based DNA damage, which was different from that detected at the time of relapse post immunotherapy. Furthermore, patient <a href="https://www.ncbi.nlm.nih.gov/protein/S02764" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S02764</a> was refractory to chemotherapy, with a limited subclonal drift following first-line chemotherapy (Fig. <a href="#Fig3" class="usa-link">3g</a> and <a href="#Sec30" class="usa-link">Supplementary Appendix</a>). However, in both patients, at relapse from initially effective second-line immunotherapy, ancestral clones emerged with acquired platinum-related DNA damage, presumably acquired throughout ineffective first-line treatment with chemotherapy.</p>
<figure class="fig xbox font-sm" id="Fig8"><h3 class="obj_head">Extended Data Fig. 3. Clinical response and clonal dynamics in patients receiving second- or third-line treatment with immunotherapy.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10972747_41586_2024_7177_Fig8_ESM.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fee5/10972747/fa40357dc598/41586_2024_7177_Fig8_ESM.jpg" loading="lazy" id="d33e3611" height="1008" width="747" alt="Extended Data Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong>, Scheme for clinical course of seven patients receiving 2<sup>nd</sup> or 3<sup>rd</sup> line treatment with ICI. Annotation as in Fig. <a href="#Fig1" class="usa-link">1b</a>. Wedges indicate sample acquisitons at first diagnosis (blue) and pre- and post-ICI (red). <strong>b</strong>, Clinical response in seven patients receiving 2<sup>nd</sup> or 3<sup>rd</sup> line treatment with immune checkpoint inhibitors (ICI). Patients revealed either a stable disease (n = 2), progressive disease (n = 2) or a partial response (n = 3). Timepoints for sample acquisitions are indicated (grey wedges). Site-specific tumour responses are plotted. Genomic analysis of tumours pre- and post-ICI is described as 2-dimensional contour plots in which cancer cell fractions (CCF) for each mutation are plotted and assigned to clusters of mutations as ancestral clones C0 (grey) and subclones (colored dashed circles, compare “Supplementary&nbsp;Appendix-Patient Cases”). <strong>c,d</strong>, Tumour phylogenies for patients with stable disease (SD, <strong>c</strong>) or for one patient with partial response (PR, <strong>d</strong>) following ICI treatment. Samples were acquired at first diagnosis (treatment-naïve, grey box), at tumour recurrence after first-line platinum-based chemotherapy (pre-ICI, red arrows, red dashed box) and after treatment with ICI (post-ICI, blue arrows, blue dashed box). The response to the respective treatment is indicated. Dashed arrows assigned to the branches of the phylogenetic trees refer to the relative contribution of mutational signatures assigned to the common ancestral clone C0 and to subclones. Temporal distinct tumours are numbered S1, S2 and S3, and the site-specific clonal composition is plotted for all samples.</p></figcaption></figure><p id="Par17">Taken together, our data show that derivatives of earlier ancestral clones persisted, despite the disappearance of the original dominating clone following first-line therapy, and then reappeared under subsequent lines of therapy thus causing clinical relapse. We could not identify any specific mutational processes or genomic patterns that resulted only from treatment with ICI and that might be indicative for effective ICI therapy. However, our data emphasize that, regardless of the efficacy of first-line treatment, ancestral clones appear to acquire platinum-induced DNA damage throughout first-line chemotherapy. Radiation, or other effective second- or third-line line therapies, can permit the subsequent expansion of these clones, even in the evolutionary short time interval of clinical care.</p></section><section id="Sec6"><h2 class="pmc_sec_title" data-anchor-id="Sec6"><a id="Sec6-anchor" data-anchor-id="Sec6" class="usa-anchor"></a>Clonality of central genome alterations</h2>
<p id="Par18">We next sought to identify those genomic alterations that segregate with treatment-associated clonal diversity in SCLC. We confirmed a key role of <em>TP53</em> and <em>RB1</em>, which were altered as part of the common ancestral clone in all patients (Fig. <a href="#Fig4" class="usa-link">4a</a>, Extended Data Figs. <a href="#Fig9" class="usa-link">4a</a> and&nbsp;<a href="#Fig10" class="usa-link">5a–c</a> and <a href="#Sec9" class="usa-link">Methods</a>). In agreement with previous studies<sup><a href="#CR26" class="usa-link" aria-describedby="CR26" aria-expanded="false">26</a></sup>, tumours with a combined histology at the time of first diagnosis (S02500, S02814 and S02917) also harboured <em>TP53</em> and <em>RB1</em> alterations as part of the common ancestor (<a href="#Sec30" class="usa-link">Supplementary Appendix</a>) whereas oncogenic mutations, such as in <em>KRAS</em>, were no longer apparent in relapsing tumours with SCLC histology<sup><a href="#CR19" class="usa-link" aria-describedby="CR19" aria-expanded="false">19</a>,<a href="#CR27" class="usa-link" aria-describedby="CR27" aria-expanded="false">27</a></sup> (Fig. <a href="#Fig4" class="usa-link">4b</a>).</p>
<figure class="fig xbox font-sm" id="Fig4"><h3 class="obj_head">Fig. 4. Clonal occurence of key gene alterations.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10972747_41586_2024_7177_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fee5/10972747/adbdc9085d61/41586_2024_7177_Fig4_HTML.jpg" loading="lazy" id="d33e1705" height="462" width="705" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong>, Gene alterations referring to significant mutations (*), hotspots (#) and damaging mutations (§), and copy number alterations. NOTCH genes include all alterations affecting NOTCH family members (<a href="#Sec9" class="usa-link">Methods</a> and Extended Data Fig. <a href="#Fig9" class="usa-link">4</a>). Corrected <em>Q</em> &lt;0.05. <strong>b</strong>, Tumour phylogeny of patient S02814 with mixed SCLC/LCNEC histology harbouring <em>KRAS</em> p.G13D at first diagnosis, and SCLC histology and acquired <em>EP300</em> p.Q160E at relapse. Additional mutations annotated as&nbsp;ms (missense),&nbsp;fs (frameshift) or&nbsp;ns (nonsense).&nbsp;<strong>c</strong>, Change in CCF of key gene mutations across distinct tumour samples in a patient (S1, S2, S3) acquired either at first diagnosis (treatment-naive), post-treatment or at relapse. Mutations are shown as either clonal (part of the common most recent ancestor, grey), subclonal with lower CCFs (yellow) or higher CCFs identified in distinct samples (blue). For amplifications, changes in integral copy number (iCN) are plotted for distinct patient-matched samples, indicating either no amplification (white) or focal amplifications (red) exceeding iCN &gt; 5 (red dashed line). <strong>d</strong>, Scheme for patients with subclonal occurrence of focal <em>MYCL</em> amplifications annotated for sampled tumour sites (dark grey wedges). <strong>e</strong>, Genome ploidy observed in paired tumours from patients at first diagnosis (treatment-naive) and following chemotherapy (post-chemo., <em>n</em> = 42). Tumours with acquired genome doubling are highlighted (pink, right), and cases with <em>CREBBP</em>/<em>EP300</em> alterations are indicated (blue). Fisher’s exact test, two-sided, *<em>P</em> &lt; 0.05.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig9"><h3 class="obj_head">Extended Data Fig. 4. Genomic alterations in 65 patients with SCLC.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10972747_41586_2024_7177_Fig9_ESM.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fee5/10972747/4d5be859de4b/41586_2024_7177_Fig9_ESM.jpg" loading="lazy" id="d33e3678" height="1007" width="718" alt="Extended Data Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig9/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong>, Significant genome alterations identified in 65 patients with SCLC. Patients are arranged according to the chemotherapy-free interval (CTFI). The response to first-line chemotherapy is annotated as circles referring to complete response (CR, dark blue), partial response (PR, blue), stable disease (SD, light blue), progressive disease (PD, red) and mixed response (PR/PD, orange). Patient cases at the far left received lung resections and at the far right were alive or lost-to-follow up and the first-line response to chemotherapy was not assessed (N/A), Significant mutations and copy number alterations are annotated according to Supplementary Tables <a href="#MOESM4" class="usa-link">8</a> and <a href="#MOESM4" class="usa-link">9</a>. For <em>TP53</em> and <em>RB1</em>, alterations for alleles A and B are provided. Additional annotation for significant alterations previously published (<em>George et al, Nature, 2015</em>) is provided. Somatic alteration frequencies are provided in the panel on the right side. <strong>b</strong>, Frequency of genome alterations in <em>TP73</em> and <em>CREBBP/EP300</em> identified in a previous cohort enriched for early-stage SCLC (n = 110, <em>George et al, Nature, 2015</em>) and in the present cohort of n = 65 patients with mostly advanced stage SCLC. <strong>c</strong>, Focal copy number alterations identified in treatment-naïve tumours (left panel) and in post-treatment (right panel). Amplifications are plotted in red, and deletions in blue. Dashed blue lines refer to the significance threshold at corrected <em>Q</em>-values &lt; 0.05. Genes which are part of chromosomal segments with significant focal copy number changes are highlighted. <strong>d</strong>, Focal copy number loss of the chromosomal segment encompassing <em>TP73</em>. The copy number state is displayed as a heatmap for treatment-naïve and post-treatment tumours. <strong>e</strong>, Comparisons of somatic mutation frequencies identified in this cohort (blue) with our previous WGS sequencing dataset (dark blue, <em>George et al, Nature, 2015)</em> and with mutation frequencies determined by targeted sequencing panels (<em>Rudin et al., Nat. Rev. Dis. Primers, 2021</em>). Genes affected in more than 5 patients within this current cohort are arranged to the left. Gene alterations which were found to not show expression are displayed on the right. Significant gene alterations are highlighted in bold (marked with *, Supplementary Table <a href="#MOESM4" class="usa-link">8</a>).</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig10"><h3 class="obj_head">Extended Data Fig. 5. Key genome alterations as part of clonal and subclonal proportions in patient tumours.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10972747_41586_2024_7177_Fig10_ESM.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fee5/10972747/1e019373cd8f/41586_2024_7177_Fig10_ESM.jpg" loading="lazy" id="d33e3758" height="1008" width="731" alt="Extended Data Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig10/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong>, Clonal occurrence of significantly mutated genes distinguishing alterations as part of the common ancestor or of subclones with mutations identified at high CCFs (blue) or low CCFs (yellow). The respective CCFs were determined for somatic single nucleotide variants occurring affecting key genes (top panel). Data is presented as median with interquartile range and whiskers to minimum and maximum value. <strong>b</strong>, CCFs determined for significantly mutated genes in distinct tumour samples (S1, S2). Alterations in these genes were assigned to the common ancestor (grey) or identified as subclonal in only one patient case either at higher CCFs (blue) or lower CCFs (yellow). Connected lines describe the change of the CCF of a given mutation across different sites in a patient in the treatment-naïve setting, pre-/post-treatment or at relapse. <strong>c</strong>, CCFs determined for genomic rearrangements identified in key genes. CCFs were determined with SVclone (Methods) referring to whole genome sequencing data (Supplementary Table <a href="#MOESM4" class="usa-link">7</a>). All gene alterations were assigned to the clonal proportion of the respective sample. <strong>d</strong>, Copy number states for <em>TP73</em> plotting copy number losses of one allele (referred to as the minor allele B with loss of heterozygosity, LOH). Copy numbers were determined as integral copy numbers (iCN) for patient-matched samples (S1, S2, S3). <strong>e,f</strong>, Copy number states of <em>MYC</em> transcription factors. Copy numbers were determined as integral copy numbers (iCN) and plotted for all samples (S1, S2, S3) analysed&nbsp;in patients with private focal and high-level copy number amplifications. The respective chromosomal position for <em>MYCL</em> (chromosome 1), <em>MYCN</em> (chromosome 2) and <em>MYC</em> (chromosome 8) is indicated and the gene locus is highlighted (pink) (<strong>e</strong>). A schematic overview of the affected sites in patients with&nbsp;subclonal occurrence of focal <em>MYCL</em>, <em>MYCN</em> and <em>MYC</em> transcription factor amplifications is provided and the sampled tumour site is indicated (dark grey wedges) (<strong>f</strong>). <strong>g</strong>, Level of subclonal mutations (left panel) and distribution of phylogeny classes (right panel) in the paired analysis of tumours pre- and post-chemotherapy (n = 42), further distinguishing ploidy states of cancer genomes (lower ploidy with ploidy &lt;2.8, and higher ploidy with ploidy 2.8 or above) and cases which acquired genome doubling. Data is presented as median with interquartile range and whiskers to minimum and maximum value. Two-sided Mann-Whitney <em>U</em>-test,&nbsp;NS, not significant at P* &lt; 0.05. <strong>h</strong>, Whole genome doubling (WGD) determined according to genome ploidy (y-axis) and the fraction of the genome with LOH (x-axis). The dotted blue line distinguishes tumours with (pink) and without (white) genome doubling, following previously described approaches (<em>Dentro et al., Cell, 2021; Bielski et al. Nature Genetics, 2018</em>). Connected lines refer to paired tumours pre- and post-treatment. Sample IDs are provided highlighting in blue patients with <em>CREBBP</em>/<em>EP300</em> alterations.</p></figcaption></figure><p id="Par19">Our genome data confirmed a significant role of key genes previously identified in cohorts enriched for early-stage tumours<sup><a href="#CR5" class="usa-link" aria-describedby="CR5" aria-expanded="false">5</a>,<a href="#CR6" class="usa-link" aria-describedby="CR6" aria-expanded="false">6</a></sup>. We also applied different approaches to identify significantly mutated genes with various levels of stringency and found that the core set of mutated genes was shared between other models and ours (<a href="#Sec9" class="usa-link">Methods</a>, Supplementary Tables <a href="#MOESM4" class="usa-link">6</a>–<a href="#MOESM4" class="usa-link">9</a> and Extended Data Figs. <a href="#Fig9" class="usa-link">4</a> and <a href="#Fig11" class="usa-link">6</a>). Whereas the functional relevance of <em>CREBBP</em>/<em>EP300</em> and <em>TP73</em> was identified previously when analysing locally clustered hotspot and damaging mutations<sup><a href="#CR5" class="usa-link" aria-describedby="CR5" aria-expanded="false">5</a>,<a href="#CR6" class="usa-link" aria-describedby="CR6" aria-expanded="false">6</a></sup>, our present cohort enriched for metastatic SCLC showed higher mutation frequencies of these genes (<em>Q</em> &lt; 0.01; <a href="#Sec9" class="usa-link">Methods</a> and Extended Data Fig. <a href="#Fig9" class="usa-link">4b</a>). We also identified significant focal chromosomal losses of <em>TP73</em> and recurrent mutations of position R273 and other conserved residues in <em>TP73</em>, which are homologous to known hotspot mutations of <em>TP53</em> (ref. <sup><a href="#CR28" class="usa-link" aria-describedby="CR28" aria-expanded="false">28</a></sup>) (Extended Data Figs. <a href="#Fig9" class="usa-link">4c,d</a> and <a href="#Fig11" class="usa-link">6a,b</a>). Our data thus further emphasize the functional relevance of <em>TP73</em> and <em>CREBBP</em>/<em>EP300</em> in advanced-stage SCLC.</p>
<figure class="fig xbox font-sm" id="Fig11"><h3 class="obj_head">Extended Data Fig. 6. Genome alterations in <em>TP73</em>, <em>CREBBP</em>/<em>EP300</em> and novel key genes.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10972747_41586_2024_7177_Fig11_ESM.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fee5/10972747/b7fa8cc0b109/41586_2024_7177_Fig11_ESM.jpg" loading="lazy" id="d33e3835" height="1008" width="729" alt="Extended Data Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig11/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong>, Protein sequence alignment of p53 family members referring to the DNA binding domain of p53 (<a href="https://www.ncbi.nlm.nih.gov/protein/NP_000537" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NP_000537</a>), p63 (<a href="https://www.ncbi.nlm.nih.gov/protein/NP_003713" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NP_003713</a>) and p73 (<a href="https://www.ncbi.nlm.nih.gov/protein/NP_005418" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NP_005418</a>). Conserved residues are highlighted in dark blue. Orange boxes indicate hotspot residues in p53 (<em>Stiewe et al., Drug Resistance Updates, 2018</em>), for which equivalent positions in p73 were mutated in our cohort. <strong>b</strong>, Overview <em>TP73</em> alterations. Patient samples identified with genome alterations are indicated. Recurrent changes included <em>TP73</em>-deltaEx2/3 (<em>George et al., Nature 2015</em>), and mutations affecting residue R293. <strong>c</strong>, Schematic representation of protein domains in Crebbp and Ep300 with information on mutated regions identified in <em>CREBBP</em> (annotated above) and in <em>EP300</em> (annotated below). Damaging alterations are highlighted in red. Hotspot and damaging alterations are presented in bold. Mutations found in tumours with acquired whole genome doubling during treatment are highlighted in blue. <strong>d</strong>, Expression levels of key gene alterations in <em>EPHB1</em> and <em>CNTNAP2</em> identified in the combined analysis of this cohort and earlier studies (Supplementary Table <a href="#MOESM4" class="usa-link">8</a>). Gene expression levels were determined based on the transcriptome data of SCLC tumours (n = 81, <em>George et al., Nature 2015</em>), and displayed as median with interquartile range and whiskers to minimum and maximum value. <strong>e</strong>, Schematic representation of mutated residues affecting protein domains in <em>EPHB1</em> (left panel) and <em>CNTNAP2</em> (right panel). Patient samples identified with genome alterations are indicated. Samples denoted with § were studied as part of earlier cohorts (<em>George et al., Nature 2015)</em>.</p></figcaption></figure><p id="Par20">We performed a combined analysis of this cohort and previously published datasets<sup><a href="#CR4" class="usa-link" aria-describedby="CR4" aria-expanded="false">4</a>–<a href="#CR6" class="usa-link" aria-describedby="CR6" aria-expanded="false">6</a></sup> (<a href="#Sec9" class="usa-link">Methods</a>), which showed significant mutations in ephrin-type B receptor 1 (<em>EPHB1</em>) and neuronal cell-adhesion gene <em>CNTNAP2</em> (Supplementary Table <a href="#MOESM4" class="usa-link">8</a> and Extended Data Figs. <a href="#Fig9" class="usa-link">4a,e</a> and <a href="#Fig11" class="usa-link">6d,e</a>). Although the majority of these significantly mutated genes were frequently part of the common ancestor (Fig. <a href="#Fig4" class="usa-link">4a</a>), some exhibited signs of ongoing subclonal evolution including protein-damaging alterations, hotspot mutations and focal losses affecting <em>CREBBP</em>/<em>EP300</em>, <em>TP73</em>, <em>KMT2D</em> and NOTCH&nbsp;genes (Fig. <a href="#Fig4" class="usa-link">4c</a> and Extended Data Fig. <a href="#Fig10" class="usa-link">5a–d</a>). Several of these alterations were enriched in the outgrowing tumour at relapse, thus further indicating a role in conferring acquired resistance to chemotherapy. To our surprise, significant high-level focal amplifications of all three <em>MYC</em> family genes (<em>MYC</em>, <em>MYCL1</em> and <em>MYCN</em>) were frequently identified as subclonal events private to one tumour site sampled (56%, <em>n</em> = 9 of 16 cases), occurring either before (<em>n</em> = 3) or after therapy (<em>n</em> = 6), whereas patient-matched spatially or temporally distinct tumours lacked the amplification event (Fig. <a href="#Fig4" class="usa-link">4c,d</a> and Extended Data Figs. <a href="#Fig9" class="usa-link">4c</a> and <a href="#Fig10" class="usa-link">5e,f</a>). Thus, despite their undoubted role in SCLC<sup><a href="#CR29" class="usa-link" aria-describedby="CR29" aria-expanded="false">29</a>–<a href="#CR32" class="usa-link" aria-describedby="CR32" aria-expanded="false">32</a></sup>, <em>MYC</em> gene amplifications are often not part of the most recent common ancestor.</p>
<p id="Par21">SCLC genomes are frequently polyploid, which is typically associated with inferior clinical outcome in cancer<sup><a href="#CR33" class="usa-link" aria-describedby="CR33" aria-expanded="false">33</a>,<a href="#CR34" class="usa-link" aria-describedby="CR34" aria-expanded="false">34</a></sup>. In our cohort, 36% of untreated tumours (<em>n</em> = 15 of 42) exhibited with higher ploidy, which had no impact on clinical response to first-line therapy and clonal diversity throughout treatment (Extended Data Fig. <a href="#Fig10" class="usa-link">5g</a>). However, in these 42 pairs of tumours obtained before and after chemotherapy, tumours in eight patients exhibited events of acquired genome duplication at the time of recurrence. The majority of these tumours harboured either functionally relevant HAT domain mutations<sup><a href="#CR6" class="usa-link" aria-describedby="CR6" aria-expanded="false">6</a></sup> or damaging alterations in <em>CREBBP</em>/<em>EP300</em>, all of which were part of the common ancestor (<em>n</em> = 6 of 8, *<em>P</em> &lt; 0.05; Fig. <a href="#Fig4" class="usa-link">4e</a> and Extended Data Figs. <a href="#Fig10" class="usa-link">5g</a> and <a href="#Fig11" class="usa-link">6c</a>). Thus, acquired resistance in tumours bearing clonal <em>CREBBP</em>/<em>EP300</em> alterations may be driven by genome duplication, which could potentiate the oncogenic functions of <em>CREBBP</em>/<em>EP300</em> already present in the founder clone<sup><a href="#CR33" class="usa-link" aria-describedby="CR33" aria-expanded="false">33</a>,<a href="#CR34" class="usa-link" aria-describedby="CR34" aria-expanded="false">34</a></sup>.</p>
<p id="Par22">We could not identify significant mutations that occurred exclusively in subclonal fractions across all patients, or those that may be related to specific mutational processes. Thus, overall, our observations provide further support for a central role of the founder clone, universally defined by mutations of <em>TP53</em> and <em>RB1</em>, in driving relapse. Furthermore, in several instances specific somatic alterations in genes implied in the biology of SCLC are enriched—but not exclusively—in recurring tumours and are therefore also likely to play a mechanistic role in the processes of drug sensitivity.</p></section><section id="Sec7"><h2 class="pmc_sec_title" data-anchor-id="Sec7"><a id="Sec7-anchor" data-anchor-id="Sec7" class="usa-anchor"></a>Impact of mutations on drug sensitivity</h2>
<p id="Par23">We next sought to study how molecular features in SCLC determine the response of patients to first-line platinum-based chemotherapy. Recent studies have proposed a major role for the expression of lineage transcription factors in treatment response in SCLC<sup><a href="#CR8" class="usa-link" aria-describedby="CR8" aria-expanded="false">8</a>,<a href="#CR9" class="usa-link" aria-describedby="CR9" aria-expanded="false">9</a>,<a href="#CR32" class="usa-link" aria-describedby="CR32" aria-expanded="false">32</a>,<a href="#CR35" class="usa-link" aria-describedby="CR35" aria-expanded="false">35</a></sup>. In the present study, too, cases with predominant expression of <em>POU2F3</em> or <em>NEUROD1</em> showed a trend towards inferior relapse-free survival; however, sample size was small (<em>n</em> = 3) and correlations did not remain significant following correction for clinical parameters (Extended Data Fig. <a href="#Fig12" class="usa-link">7a–d</a> and Supplementary Table <a href="#MOESM4" class="usa-link">10</a>). Furthermore, although studies in mice have suggested a plasticity in the expression of lineage transcription factors due to tumour progression and chemotherapeutic intervention<sup><a href="#CR32" class="usa-link" aria-describedby="CR32" aria-expanded="false">32</a>,<a href="#CR35" class="usa-link" aria-describedby="CR35" aria-expanded="false">35</a></sup>, spatially and temporally distinct tumours from patients with SCLC in our cohort did not show changes in the expression of these key transcription factors<sup><a href="#CR32" class="usa-link" aria-describedby="CR32" aria-expanded="false">32</a>,<a href="#CR35" class="usa-link" aria-describedby="CR35" aria-expanded="false">35</a></sup> (Extended Data Fig. <a href="#Fig12" class="usa-link">7e</a>). Finally, we could not observe a correlation of <em>MYC</em> family gene amplification with the expression of key transcription factors or subtype conversion in our cohort (Extended Data Fig. <a href="#Fig12" class="usa-link">7f</a>).</p>
<figure class="fig xbox font-sm" id="Fig12"><h3 class="obj_head">Extended Data Fig. 7. Gene expression of key lineage transcription factors in SCLC.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10972747_41586_2024_7177_Fig12_ESM.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fee5/10972747/5301acc0d121/41586_2024_7177_Fig12_ESM.jpg" loading="lazy" id="d33e3915" height="1008" width="756" alt="Extended Data Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig12/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a,b</strong> Expression of the four key lineage transcription factors <em>ASCL1</em>, <em>NEUROD1</em>, <em>POU2F3</em> and <em>YAP1</em> in our cohort (n = 52 patients), categorized as: <em>ASCL1</em>+, <em>ASCL1</em>+/<em>NEUROD1</em>+ double-positive, mainly <em>NEUROD1</em>+ and mainly <em>POU2F3</em>+ positive tumours (heatmap representation, <strong>a</strong>). The distribution of patient samples with the expression of key transcription factors to an independent larger cohort of n = 81 samples (<em>George et al. Nature 2015</em>, <strong>b</strong>). <strong>c,d</strong> Kaplan-Meier curve and Cox regression model for relapse-free survival in patients stratified according to the expression of key transcription factors. <strong>c</strong>, Log-rank Mantel-Cox test at P*** = 0.0009, <strong>d</strong>,Significant associations are highlighted in bold. Hazard ratios (HR) were determined at 95% confidence interval and significance was determined by Log-rank tests at P** &lt; 0.01. <strong>e</strong>, Expression of <em>ASCL1</em>, <em>NEUROD1</em>, <em>POU2F3</em> and <em>YAP1</em> in patient-matched spatially or temporally distinct tumours. Samples were analysed across distinct sites in treatment-naïve patients, or pre- and post-treatment with platinum-based chemotherapy or immune checkpoint inhibition (ICI). Sampled tumour sites and the response to the respective treatment is indicated. <strong>f</strong>, Expression of MYC transcription factor family members and lineage transcription factors in cases found with focal and high-level amplification of MYC genes in at least one of the multi-regional tumour sites. Sample names and the MYC gene amplification status is provided in the top panel further indicating if the tumour was samples in the treatment-naïve setting or at relapse after treatment with platinum-based chemotherapy. The expression levels are provided as a heatmap.</p></figcaption></figure><p id="Par24">We therefore proposed that the overall genomic make-up of the common ancestral clone is the main driver of the sensitivity of patients to first-line chemotherapy. <em>TP53</em> and <em>RB1</em> alterations were universally part of the common ancestral clone, and we sought to further classify alterations in both genes according to their impact on the functionality of the encoded protein. We distinguished between missense mutations creating a full-length protein and other somatic alterations as probably ‘gene damaging’ due to either out-of-frame transcription, early termination or larger insertions or deletions impacting protein expression (Fig. <a href="#Fig5" class="usa-link">5a</a>, Supplementary Table <a href="#MOESM4" class="usa-link">11</a> and <a href="#Sec9" class="usa-link">Methods</a>). When assessing clinical outcome as a function of the qualitative nature of all significant gene alterations, we thus identified a higher risk of relapse in patients with these ‘other gene-damaging’ alterations in <em>TP53</em> (**<em>P</em> &lt; 0.01; Fig. <a href="#Fig5" class="usa-link">5b</a> and Extended Data Fig. <a href="#Fig13" class="usa-link">8a,b</a>), which had similarly been observed in other lung cancers<sup><a href="#CR36" class="usa-link" aria-describedby="CR36" aria-expanded="false">36</a></sup>. Although patients frequently harboured point mutations in the DNA-binding domain of <em>TP53</em> affecting well-known hotspot sites<sup><a href="#CR28" class="usa-link" aria-describedby="CR28" aria-expanded="false">28</a></sup>, gene-damaging alterations occurred in 40% of patients and we confirmed either truncated or absent protein products in tumours of these patients (Fig. <a href="#Fig5" class="usa-link">5a</a> and Extended Data Fig. <a href="#Fig14" class="usa-link">9a</a>). By contrast, damaging alterations constituted the vast majority of all <em>RB1</em> lesions (95%; Supplementary Table <a href="#MOESM4" class="usa-link">11</a>) and no difference in response could be identified. Although frequently part of subclones, <em>MYC</em> gene amplifications were also not found to correlate with chemotherapeutic response (Extended Data Fig. <a href="#Fig13" class="usa-link">8</a>). <em>TP53</em> gene-damaging alterations associated with marginal or no response to chemotherapy (*<em>P</em> &lt; 0.05; Fig. <a href="#Fig5" class="usa-link">5c</a>) resulted in a median time to disease recurrence of 63 days and almost all patients relapsed within 6 months (<em>n</em> = 22 of 23; Fig. <a href="#Fig5" class="usa-link">5d</a>). This observation remained significant in Cox regression models considering all genomic patterns after adjusting for age, sex and tumour stage (hazard ratio 2.12 and 95% confidence interval 1.06–4.23; Extended Data Fig. <a href="#Fig14" class="usa-link">9b,c</a>). On the basis of&nbsp;these findings, we analysed an independent cohort of 63 patients with SCLC who were treated with first-line platinum-based chemotherapy, to validate the clinical relevance of destructive <em>TP53</em> mutations. In this cohort, too, damaging alterations of <em>TP53</em> segregated with a short duration of relapse-free interval (Fig. <a href="#Fig5" class="usa-link">5b</a> and Supplementary Table <a href="#MOESM4" class="usa-link">12</a>).</p>
<figure class="fig xbox font-sm" id="Fig5"><h3 class="obj_head">Fig. 5. <em>TP53</em> and significant co-alterations impacting chemotherapeutic response.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10972747_41586_2024_7177_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fee5/10972747/51bc32ec6588/41586_2024_7177_Fig5_HTML.jpg" loading="lazy" id="d33e2232" height="407" width="704" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong>, Somatic alterations in <em>TP53</em>. Point mutations mapped to the protein structure (DNA-binding domain, PDB-ID: <a href="https://doi.org/10.2210/pdb2AHI/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">2AHI</a>, top). Hotspots (pink, residues annotated), other point mutations (blue) and interaction with DNA (teal) are shown. Damaging gene alterations creating deletions, insertions and destructive transcripts are described (bottom; transactivation and tetramerization domains (TAD, TD, respectively); transcript ID: <a href="https://www.ncbi.nlm.nih.gov/nuccore/NM_000546" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NM_000546</a>). <strong>b</strong>, Kaplan–Meier curve of patients grouped for p53 point mutations (blue) and other gene-damaging <em>TP53</em> alterations (red). Relapse-free survival refers to CTFI and is plotted for patients in this cohort who received only first-line systemic platinum-based chemotherapy (top, <em>n</em> = 55 of 65 patients; grey points, <em>n</em> = 2 censored subjects); and for an independent cohort (bottom, <em>n</em> = 64 patients). Log-rank test, **<em>P</em> &lt; 0.01. <strong>c</strong>,<strong>d</strong>, Clinical response (defined as complete response/partial response) to first-line systemic chemotherapy for <em>n</em> = 54 of 65 patients grouped for p53 point mutations and other gene-damaging <em>TP53</em> alterations. Fisher’s exact test, two-sided, *<em>P</em> = 0.022. Patients with information available for relapse-free survival (<em>n</em> = 53) were grouped for <em>TP53</em> gene-damaging (<em>n</em> = 22) or p53 point mutations (<em>n</em> = 31) (<strong>c</strong>) and further stratified for co-alterations in <em>CREBBP</em>/<em>EP300</em>, <em>TP73</em> or <em>FMN2</em> (<em>n</em> = 20) or none (<em>n</em> = 11) (<strong>d</strong>). CTFI range was 45, 90 and 180 days (red, yellow and light blue background, respectively). Boxplot, median and interquartile range, minimum and maximum values. <strong>e</strong>,<strong>f</strong>, Relapse-free survival in patients of this cohort receiving only first-line systemic platinum-based chemotherapy (<em>n</em> = 55 of 65, <em>n</em> = 2 patients censored). <strong>e</strong>, Patients are grouped according to clonal other gene-damaging alterations in <em>TP53</em> and p53 point mutations that were further stratified for significant co-alterations of <em>CREBBP</em>/<em>EP300</em>, <em>TP73</em> and <em>FMN2</em>. Cox regression model adjusting for age, sex and tumour stage. HR showing the median and 95% confidence interval (CI). <strong>f</strong>, Kaplan–Meier curve (<em>n</em> = 55 of 65 patients; grey points, censored subjects, <em>n</em> = 2); log-rank Mantel–Cox test, ***<em>P</em> = 0.0003.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig13"><h3 class="obj_head">Extended Data Fig. 8. Genomic alterations associating with duration of response to first-line platinum-based chemotherapy.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10972747_41586_2024_7177_Fig13_ESM.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fee5/10972747/4573735a6d61/41586_2024_7177_Fig13_ESM.jpg" loading="lazy" id="d33e3938" height="1008" width="727" alt="Extended Data Fig. 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig13/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong>, Kaplan-Meier curves of relapse-free survival for patients with SCLC receiving first-line systemic treatment with platinum-based chemotherapy. Patient were grouped based on clinical variables and significant genome alterations identified in Extended Data Fig. <a href="#Fig10" class="usa-link">5</a>. P-values and hazard ratios determined by Log-rank tests and at 95% confidence interval univariable Cox regression models. <strong>b</strong>, Backward Wald tests within Cox models (retention threshold P &lt; 0.05, two-sided) testing all genome alterations described in (<strong>a</strong>).</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig14"><h3 class="obj_head">Extended Data Fig. 9. Associations of <em>TP53</em> and other key patterns with duration of response to first-line platinum-based chemotherapy.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10972747_41586_2024_7177_Fig14_ESM.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/fee5/10972747/ae1b072d1cdd/41586_2024_7177_Fig14_ESM.jpg" loading="lazy" id="d33e4016" height="1008" width="750" alt="Extended Data Fig. 9"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig14/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong>, p53 protein expression in tumour cell lysates of this cohort. Western blot analysis probing with anti-p53 and anti-HSP90 (loading control). <em>TP53</em> alterations resulting in gene damaging (red) or other point mutations (blue) are indicated. Lysates of the NSCLC cell line A549 served as a control for wild-type p53 (right lane). Samples were analyzed on two blots, quantitave comparisons are performed on the same blot. Gel source data, see Supplementary Fig. <a href="#MOESM3" class="usa-link">S1</a>. <strong>b,c</strong> Cox regression model for relapse-free survival in patients (n = 53) with SCLC stratifying for gene alterations in <em>TP53</em> and adjusting for age, sex and stage (<strong>b</strong>), and analyzing those patients without gene damaging alterations in <em>TP53</em> (n = 30) <strong>(c)</strong> with respect to significant genome alterations and clinical variables. Significant associations are highlighted in bold. Hazard ratio (HR) displaying median, 5% and 95% confidence intervals. Log-rank Mantel-Cox test at P* &lt; 0.05, P** &lt; 0.01 and P*** &lt; 0.001. <strong>d–e</strong>, Genome alterations significantly associating with an increased hazard of disease recurrence for patients receiving first-line systemic therapy with platinum-based chemotherapy referring to the current study cohort (n =  n = 55/65 patients, Kaplan-Meier curve: grey points for censored subjects n = 2). Log-rank tests at P*** &lt; 0.001 (<strong>d</strong>), and for the independent cohort of patients with SCLC (<strong>e</strong>). <strong>f</strong>, <em>TP53</em>, <em>FMN2</em>, <em>TP73</em> and <em>CREBBP</em>/<em>EP300</em> alteration status in patients receiving first-line systemic treatment with chemotherapy (n = 55). Cases are arranged from left to right, and the chemotherapy-free interval (CTFI) is provided for each patient (*n = 2 patients censored). The bar graph at the bottom plots the CTFI for patients with <em>TP53</em> point mutations grouped according to the indicated co-alterations. Data is presented as median with interquartile range and whiskers to minimum and maximum value, P* = 0.036; Mann-Whitney U-test, two-sided. <strong>g</strong>, Associations of male and female patients with the smoking status documented as “former” or “current” smoker (Fisher’s exact test, P* &lt; 0.05; left panel). The smoking quantity as measured by the number of packyears is plotted for male and female patients with median and interquartile range, whiskers to minimum and maximum value (Mann-Whitney <em>U</em>-test at P* &lt; 0.05, two-sided; right panel). n.s. = not significant.</p></figcaption></figure><p id="Par25">Because some key mutations were acquired throughout the course of treatment, we next proposed that co-alterations of relevant genes might also impact patient survival. We therefore performed regression models and found that co-alterations of <em>CREBBP</em>/<em>EP300</em>, <em>TP73</em> or <em>FMN2</em> increased the relative risk of disease recurrence in patients without <em>TP53-</em>damaging alterations, which remained significant when adjusting for clinical parameters (HR 2.74, 95% confidence interval 1.01–7.44, *<em>P</em> &lt; 0.05); a similar trend was observed in the independent patient cohort (Fig. <a href="#Fig5" class="usa-link">5e,f</a> and Extended Data Fig. <a href="#Fig14" class="usa-link">9c–e</a>). Furthermore, co-alterations of <em>CREBBP</em>/<em>EP300</em>, <em>TP73</em> or <em>FMN2</em> suggested epistasis (Extended Data Fig. <a href="#Fig14" class="usa-link">9f</a>). Of note, in addition to stage, our data showed longer relapse-free survival in women not related to smoking behaviour in these patients, and may point to a sex bias (Fig. <a href="#Fig5" class="usa-link">5e</a> and Extended Data Fig. <a href="#Fig14" class="usa-link">9g</a>). Taken together, our genome analyses show that <em>TP53-</em>damaging alterations associate with resistance to chemotherapy and that coexisting alterations of <em>TP73</em>, <em>CREBBP</em>/<em>EP300</em> or <em>FMN2</em> compromise the clinical efficacy of chemotherapy in patients with SCLC—even in the absence of gene-damaging <em>TP53</em> alterations.</p></section><section id="Sec8"><h2 class="pmc_sec_title" data-anchor-id="Sec8"><a id="Sec8-anchor" data-anchor-id="Sec8" class="usa-anchor"></a>Discussion</h2>
<p id="Par26">Our findings provide a mechanistic explanation for the clinical phenomenon of the initial high sensitivity of SCLC to first-line platinum-based chemotherapy followed by rapid relapse. We show that effective chemotherapy leads to elimination of a rapidly growing, pseudo-clonal population of cancer cells that dominates the tumour at diagnosis, followed by expansion of a large number of subclones derived from the common ancestor. We identify the primary tumour as a site with ongoing evolutionary adaption: following treatment-induced evolutionary pressure, ancestral clones already present in the primary tumour emerge from the common ancestor and give rise to subclones shaping clinical relapse. Our study thus establishes a critical role for the genomic context of the common ancestor in drug resistance, and we uncover its genomic portrait that is largely confined to biallelic losses of <em>TP53</em> and <em>RB1</em>. Gene-damaging alterations in <em>TP53</em> associate with a particularly chemotherapy-resistant state in patients with SCLC, which is in line with studies establishing a role of functionally distinct <em>TP53</em> alterations impacting the response to chemotherapy and clinical outcome in cancer<sup><a href="#CR36" class="usa-link" aria-describedby="CR36" aria-expanded="false">36</a>–<a href="#CR38" class="usa-link" aria-describedby="CR38" aria-expanded="false">38</a></sup>. However, patients with <em>TP53</em> missense mutations can suffer a similarly poor response if co-occurring alterations of <em>TP73</em>, <em>CREBBP/EP300</em> or <em>FMN2</em> complement the dysfunction of <em>TP53</em>. Of note, the high frequency of mutations in these genes in the advanced-stage population of this study corroborates our previous reports of an important role of <em>TP73</em>, <em>CREBBP/EP300</em> or <em>FMN2</em> in SCLC<sup><a href="#CR5" class="usa-link" aria-describedby="CR5" aria-expanded="false">5</a>,<a href="#CR6" class="usa-link" aria-describedby="CR6" aria-expanded="false">6</a></sup>. Furthermore, adding to our previous discovery of somatic rearrangements of <em>TP73</em> (ref. <sup><a href="#CR5" class="usa-link" aria-describedby="CR5" aria-expanded="false">5</a></sup>), we now report recurrent <em>TP73</em> hotspot mutations at highly conserved residues. Although it is known that&nbsp;genome ploidy contributes to tumour malignancy and inferior survival<sup><a href="#CR33" class="usa-link" aria-describedby="CR33" aria-expanded="false">33</a>,<a href="#CR34" class="usa-link" aria-describedby="CR34" aria-expanded="false">34</a></sup>, we&nbsp;found that clonal mutations of <em>CREBBP</em>/<em>EP300</em> associated with acquired genome duplication cause relapse and thereby provide a clear genetic mechanism of drug resistance<sup><a href="#CR39" class="usa-link" aria-describedby="CR39" aria-expanded="false">39</a></sup>. Finally we demonstrate that although <em>MYC</em> family genes play an important role in SCLC biology, amplification events were often not part of the founder clone, and, furthermore, no associations with selection pressure and drug resistance&nbsp;were identified. Thus, our data provide a core set of recurrently altered genes that have a particular impact on drug sensitivity and resistance in SCLC.</p>
<p id="Par27">Recent studies have established an impact of the expression of lineage transcription factors on drug response<sup><a href="#CR8" class="usa-link" aria-describedby="CR8" aria-expanded="false">8</a>,<a href="#CR9" class="usa-link" aria-describedby="CR9" aria-expanded="false">9</a>,<a href="#CR32" class="usa-link" aria-describedby="CR32" aria-expanded="false">32</a>,<a href="#CR35" class="usa-link" aria-describedby="CR35" aria-expanded="false">35</a></sup>. In this study we could not identify notable transcriptional subtype conversion or correlation of major subtypes with treatment response, but found a strong relationship of certain genome alterations with clinical outcome. Future studies focused on combining genome evolutionary processes with single-cell transcriptome data are therefore warranted to elucidate the interplay of genomic and transcriptional heterogeneity in SCLC.</p>
<p id="Par28">New drugs are typically tested in the second or third line of treatment, almost always with limited efficacy. We speculate that this phenomenon may be due, at least partially, to the massive increase in clonal heterogeneity following first-line chemotherapy described herein. Subclonal diversity following treatment was largely attributable to clock-like mutational processes, thus indicating that subclones at clinical relapse had existed before therapy. Independent of the sensitivity to first-line platinum-based chemotherapy, we found that ancestral clones acquire platinum-induced DNA damage throughout first-line therapy and emerge at relapse, which is more pronounced after effective radiation. We similarly observed platinum-based genomic damage in patients relapsing following effective ICI. Although we could not determine genomic or molecular patterns associated with response to ICI in these patients, our data demonstrate that genomic damage from first-line chemotherapy can complicate the efficacy and duration of response to other treatments initiated in subsequent lines. Despite the overall short time window of clinical care, effective treatment—including radiation or successful immunotherapy—can accelerate the emergence of ancestral clones with platinum-induced genomic scars that subsequently cause relapse. Although we could not identify specific&nbsp;gene mutations associated with these mutational processes, our data warrant studies focused on the consequences of platinum-induced changes on genome integrity and maintenance.</p>
<p id="Par29">Overall, our findings related to the most recent common ancestor also have clinical implications. First, it may be an attractive concept for the discovery of new therapies to focus on alterations specifically present in the common ancestor. These may also serve as markers to monitor response and resistance to treatment. Second, we consider effective first-line treatment to be critical also for subsequent lines of therapy, because of the sheer increase in clonal diversity that drives drug resistance. Third, drugs in development may still be efficacious when tested early in the first-line setting, without being affected by a clonally diverse relapse that complicates subsequent lines of treatment.</p>
<p id="Par30">In summary, we uncover genomic alterations underlying poor response and rapid relapse, which put the most recent common ancestral clone at the centre of cancer genome evolution. Our study therefore emphasizes the need for future therapeutic strategies to be tailored to target the detrimental cellular component of the founder clone to improve the outcome of patients with SCLC.</p></section><section id="Sec9"><h2 class="pmc_sec_title" data-anchor-id="Sec9"><a id="Sec9-anchor" data-anchor-id="Sec9" class="usa-anchor"></a>Methods</h2>
<section id="Sec10"><h3 class="pmc_sec_title">Human lung tumour specimens</h3>
<p id="Par31">This study was approved by the institutional review board of the University of Cologne. We analysed 160 tumours and patient-matched blood samples from 65 patients with SCLC (Fig. <a href="#Fig1" class="usa-link">1a</a>). The samples were collected from multiple collaborating hospitals and clinical facilities under institutional review board-approved protocols, and all patients provided written informed consent. For some patients the material was collected as part of an ongoing clinical trial (BIOLUMA, study no. <a href="https://clinicaltrials.gov/ct2/show/NCT03083691" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NCT03083691</a>), and those patients received as second- or third-line treatment anti-PD-1 either alone or in combination with anti-CTLA-4 immune checkpoint inhibitors. The course of treatment for all patients and information on all samples are detailed below and summarized in Extended Data Table <a href="#Tab1" class="usa-link">1</a> and Supplementary Tables <a href="#MOESM4" class="usa-link">1</a> and <a href="#MOESM4" class="usa-link">2</a>.</p>
<p id="Par32">All tumour samples were pathologically reviewed by at least two independent expert pathologists who inspected the histomorphology based on haematoxylin and eosin and immunohistochemical staining. All tumours were confirmed with SCLC histology; tumours from three patients were diagnosed with additional morphological components of LCNEC or adenocarcinoma (Extended Data Table <a href="#Tab1" class="usa-link">1</a> and Supplementary Table <a href="#MOESM4" class="usa-link">1</a>). All patient-matched, multiregional tumour and normal blood samples were confirmed as belonging to the same patient by short tandem repeat (STR) analysis conducted at the Institute of Legal Medicine at the University of Cologne, Germany, and further confirmed by genome sequencing data.</p>
<p id="Par33">In the majority of cases we analysed at least two tumour samples per patient, which were acquired at either single or multiple timepoints throughout the clinical course of treatment (Supplementary Table <a href="#MOESM4" class="usa-link">2</a>). More than two tumour samples were acquired for 37% of patients (<em>n</em> = 24 of 65). For five patients we analysed tumour samples at three distinct time points (<em>n</em> = 5 of 65, 8%; Extended Data Table <a href="#Tab1" class="usa-link">1</a> and Supplementary Table <a href="#MOESM4" class="usa-link">2</a>). Samples were acquired as biopsies and lung resections, and we additionally engrafted tumour tissue from fine-needle biopsies (<em>n</em> = 2, one pleural and one lymph node metastasis) and CTCs (<em>n</em> = 29 of 160, 18%) onto immune-compromised mice (NSG mice) to establish PDX (in total <em>n</em> = 31 of 160, 19%; Fig. <a href="#Fig1" class="usa-link">1a</a>); this approach allowed for enrichment of limited tumour material for in-depth genomic studies. Samples analysed as PDX are listed in Supplementary Tables <a href="#MOESM4" class="usa-link">2</a> and <a href="#MOESM4" class="usa-link">3</a> and are highlighted in Fig. <a href="#Fig2" class="usa-link">2d</a>. As previously described<sup><a href="#CR12" class="usa-link" aria-describedby="CR12" aria-expanded="false">12</a>,<a href="#CR13" class="usa-link" aria-describedby="CR13" aria-expanded="false">13</a></sup>, sampling a patient’s blood for CTCs provides a minimally invasive approach towards analysis of tumour cells under therapy, and xenotransplant models have been shown to recapitulate the genomic profiles of the patient’s tumour. Xenotransplant models were established following an approach previously described<sup><a href="#CR12" class="usa-link" aria-describedby="CR12" aria-expanded="false">12</a></sup>; tumour cells were engrafted subcutaneously into the flanks of 7–14-week-old NSG mice (male and female, NOD.Cg-Prkdcscid Il2rgtm1Wjl/SzJ; Jackson Laboratories), and tumours were harvested at a maximum volume of 1.5 mm<sup>3</sup>. Tumour histology was confirmed by pathological review, and STR profiling with patient-matched normal and tumour samples confirmed the identity of the engrafted patient-derived material. All animals were housed in a specific-pathogen-free facility under ambient temperature and humidity while maintaining a 12/12 h light/dark cycle. Animal experiments were approved by, and conducted in accordance with, the regulations of the local animal welfare authorities (State Agency for Nature, Environment and Consumer Protection of the State of North Rhine-Westphalia, nos. AZ: 84-02.04.2012.A281, 84-02.04.2015.A172 and 84-02.04.2018.A002).</p>
<p id="Par34">Samples were categorized by location: we referred to the primary lung tumour and grouped metastatic sites as intrapulmonary metastases, including pulmonary and lung and mediastinal lymph node metastases; tumour sites grouped as extrapulmonary metastases include intrathoracic distant metastases of the pleura and extrathoracic distant metastases affecting abdominal sites, the brain or other less common metastatic sites (breast, skin, sternum), as well as CTCs propagated as CTC-derived xenotransplant models, which represent cells that spread to the bloodstream with the potential to seed distant metastases. In patients with highly metastatic disease we furthermore assessed whether, based on radiological images, tumour sites sampled throughout therapy were pre-existing at the time of first diagnosis or before treatment, and whether these sites were exposed to any given therapy (chemotherapy, radiation or immune checkpoint blockade). Furthermore, we assessed whether any samples were taken from a newly formed metastatic site which, according to radiological imaging, was not pre-existing at the time of first diagnosis or before any other treatment exposure. For CTC-derived models, because we had no information regarding whether the tumour site may have shed cells to the bloodstream, we classified any CTC-derived sample as tumour cells that may have been exposed to any given treatment. A schematic overview of the acquired samples and affected organ sites is depicted for each patient in the <a href="#Sec30" class="usa-link">Supplementary Appendix</a>.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Clinical characteristics</h3>
<p id="Par35">The clinical characteristics of the patients in our cohort are in line with those typically found in SCLC (Fig. <a href="#Fig1" class="usa-link">1b</a>, Extended Data Table <a href="#Tab1" class="usa-link">1</a> and Supplementary Table <a href="#MOESM4" class="usa-link">1</a>). Median age at the time of first diagnosis was 64 years, and patients were predominantly male (<em>n</em> = 43 of 65, 66%) with a history of heavy smoking and a median number of 40 pack years (smoking history was known for 89% of patients, <em>n</em> = 58 of 65; the number of pack years was determined for 85% of patients, <em>n</em> = 55 of 65). For clinical correlations the following categories were defined: age groups of 65 years or more and under 65. Smoking status was classified as ‘current smoker’, ‘former smoker’ or ‘never smoker’.</p>
<p id="Par36">The majority of the patients presented with a highly metastatic tumour classified as stages III and IV (<em>n</em> = 57 of 65, 88%; additional information on tumour, node and metastasis staging is provided in Supplementary Table <a href="#MOESM4" class="usa-link">1</a>). Seven patients were diagnosed with limited-stage disease or with tumour stage I, II or IIIA, and were therefore amenable to surgical lung resection.</p>
<p id="Par37">Although one patient declined further therapy, all other patients in our cohort received systemic treatment with platinum-based chemotherapy. The majority of patients were treated with a combination of cisplatin/carboplatin and etoposide (<em>n</em> = 61 of 65; 94%); with regard to recent changes in the treatment of SCLC<sup><a href="#CR2" class="usa-link" aria-describedby="CR2" aria-expanded="false">2</a></sup>, additional PD-L1 inhibition was administered to five of these patients. Due to the initial diagnosis with histological components of non-SCLC (adenocarcinoma or LCNEC), two patients were treated with cisplatin/carboplatin combined with vinorelbine (patients S02814 and S02917). Furthermore, one patient received only monotherapy with carboplatin. Throughout the course of treatment 72% of patients (<em>n</em> = 47 of 65) received additional radiation, mainly of the chest/lung/mediastinum (<em>n</em> = 35 or 47) or brain (<em>n</em> = 38 of 47); four patients underwent stereotactic surgery of brain metastases.</p>
<p id="Par38">The clinical response to treatment was assessed by radiological imaging and classified as either complete response&nbsp;(CR), partial response&nbsp;(PR), stable disease&nbsp;(SD), progressive disease&nbsp;(PD) or mixed response&nbsp;(PR/PD). The clinical response to systemic first-line platinum-based chemotherapy was analysed for <em>n</em> = 55 patients; these patients receiving treatment with only systemic&nbsp;chemotherapy&nbsp;and were therefore considered for subsequent correlations of genomic and molecular phenotypes with clinical response. Genomic and molecular correlations with clinical response to chemotherapy were not considered for <em>n</em> = 10 patients in our cohort, because these patients were either lost to follow-up (<em>n</em> = 2), declined further treatment (<em>n</em> = 1) or received a lung resection resulting in differences in the dynamics of disease progression (<em>n</em> = 7).</p>
<p id="Par39">Of the 55 patients who received only first-line systemic platinum-based chemotherapy, 60% (<em>n</em> = 33 of 55) responded to treatment with PR (<em>n</em> = 32) or CR (<em>n</em> = 1), 9% had stable disease (<em>n</em> = 5 of 55), 11% showed mixed response (<em>n</em> = 6 of 55) and 20% (<em>n</em> = 11 of 55) experienced a&nbsp;progressive disease, of which three succumbed to the disease during first-line treatment. Following treatment, two patients experienced fatal sepsis (patient S02608 while receiving treatment and experiencing disease progression; and patient S02658 following completion of chemotherapy; Supplementary Table <a href="#MOESM4" class="usa-link">1</a>); both patients were consequently censored when performing correlations with relapse-free survival, and the therapy response of patient S02608 was not evaluated. Median progression-free survival was 6.3 months. In addition we determined CTFI as an independent measure of sensitivity and duration of response to first-line platinum-based chemotherapy; median CTFI was 88 days. Fifty-three per cent of patients (<em>n</em> = 28 of 53) either did not respond, relapsed or succumbed to the tumour disease within 90 days following completion of first-line chemotherapy (following the guidelines of NCCN)<sup><a href="#CR14" class="usa-link" aria-describedby="CR14" aria-expanded="false">14</a></sup>, and these patients were thus clinically classified as either chemorefractory or -resistant). Of the remaining patients who, based on NCCN guidelines, were considered as ‘platinum-sensitive’, 30% (<em>n</em> = 16 of 53) relapsed within 6 months following completion of chemotherapy and 17% were relapse-free for more than 6 months (<em>n</em> = 9 of 53). At relapse, 83% of patients (<em>n</em> = 44 of 53) received second-line systemic therapies that included treatment with anti-PD-1 and/or anti-CTLA-4 immune checkpoint inhibitors (<em>n</em> = 27) or other chemotherapeutics, including topotecan (<em>n</em> = 8), rechallenge with carboplatin and etoposide (<em>n</em> = 2) or combinations of adriamycin, cyclophosphamide and vincristine (<em>n</em> = 7) (Fig. <a href="#Fig1" class="usa-link">1</a> and Supplementary Table <a href="#MOESM4" class="usa-link">1</a>). Following tumour progression, ten patients were amenable to additional lines of therapy including immune checkpoint inhibitors (<em>n</em> = 6) or chemotherapeutics (<em>n</em> = 4).</p>
<p id="Par40">The analysis of multiregional and longitudinal tumour sites from 65 patients with SCLC focused on distinct clinical scenarios. For interpatient comparisons we focused on studies of tumour pairs (‘Analysis of clonal architecture from multiregional and longitudinal tumour samples’; Fig. <a href="#Fig1" class="usa-link">1c</a>). We focused on distinct clinical scenarios: (1) analysis of tumour samples from spatially distinct sites obtained from treatment-naive patients at the time of first diagnosis (<em>n</em> = 16); (2) analysis of temporally distinct tumour sites referring to samples acquired before treatment and during therapy, including those from patients undergoing neo-adjuvant treatment (<em>n</em> = 5); and (3) samples acquired before treatment and at relapse following completion of first-line platinum-based chemotherapy (that is, either following an initial response or disease progression despite treatment, <em>n</em> = 42). The analysis further focused on (4) spatially, but not temporally, separate tumours analysed solely at the time of relapse (<em>n</em> = 14), and (5) tumour sites acquired at the time of relapse from platinum-based therapy and following subsequent lines of treatment with immune checkpoint inhibitors (pre- and post-treatment with ICI, <em>n</em> = 7). We thus performed in total <em>n</em> = 84 paired analyses of tumour sites in 65 patients with SCLC (Supplementary Table <a href="#MOESM4" class="usa-link">4</a>).</p>
<p id="Par41">In addition we performed&nbsp;clinical correlations in an independent cohort of patients with SCLC,&nbsp;who all received first-line systemic treatment with platinum-based chemotherapy; we performed whole-exome sequencing of the tumour samples and identified&nbsp;key genome alterations (<em>n</em> = 64 patients; Supplementary Table <a href="#MOESM4" class="usa-link">12</a>).&nbsp;This cohort was analysed to validate findings described in Fig. <a href="#Fig5" class="usa-link">5b</a>; at least 56 samples&nbsp;are required to validate the findings at a significance level of 5% and a power of 80%; thus, we validated our findings at a power of greater than 80%.</p></section><section id="Sec12"><h3 class="pmc_sec_title">DNA and RNA extraction</h3>
<p id="Par42">Nucleic acids were extracted from fresh-frozen blood or tissue or from formalin-fixed, paraffin-embedded (FFPE) tissue specimens (Supplementary Table <a href="#MOESM4" class="usa-link">3</a>). Tumour tissues were analysed by haematoxylin and eosin staining and nucleic acids were extracted from regions with a tumour content of at least 70%. All tumour samples derived from murine xenotransplant models showed a tumour content of at least 95% with no discernible innervation of murine cells, which was similarly observed in previous studies<sup><a href="#CR12" class="usa-link" aria-describedby="CR12" aria-expanded="false">12</a>,<a href="#CR13" class="usa-link" aria-describedby="CR13" aria-expanded="false">13</a></sup>. Fresh-frozen samples were processed by preparation of tissue sections, each of 20 μm thickness, on a cryostat (Leica) while maintaining a temperature of −20 °C. In the case of FFPE samples, sections of 20 μm thickness were prepared on slides on a microtome. DNA was extracted from both fresh-frozen tissues and EDTA blood with the Gentra Puregene DNA extraction kit (Qiagen) according to the protocol of the manufacturer.</p>
<p id="Par43">To allow for high-quality sequencing data of FFPE material we applied ultrasonic acoustic energy, using the adaptive focused acoustics technology from Covaris and following the protocol of the manufacturer. DNA isolation was then performed with a bead-based approach (AMPure XP Beads, Beckman) and any fractions containing paraffin material were excluded from subsequent DNA isolation steps.</p>
<p id="Par44">For samples with limited tumour material we further adjusted protocols, which included repeated rounds of protein and nucleic acid precipitation, to increase the DNA yield for subsequent sequencing studies. All DNA isolates were hydrated in TE buffer and molecular weight was assessed using the Agilent TapeStation system (Genomic DNA ScreenTape no. 5067-5365, Agilent Technologies). DNA isolates from fresh-frozen samples were confirmed as being of high molecular weight (above 10 kb), and samples with evident signs of degradation were excluded from further sequencing studies.</p>
<p id="Par45">For RNA extraction, tissue sections were first lysed and homogenized with the Tissue Lyzer (Qiagen). Subsequent RNA extraction was performed with the Qiagen RNAeasy Mini Kit according to the instructions of the manufacturer. Alternatively we used the RNAeasy Micro Kit to extract RNA from small tissue biopsies. RNA quality was assessed with RNA Screen Tape (no. 5067-5576, Agilent Technologies) at the TapeStation. Samples with RNA integrity number above 7 were further analysed by RNA sequencing (RNA-seq).</p></section><section id="Sec13"><h3 class="pmc_sec_title">Next-generation sequencing</h3>
<p id="Par46">All sequencing reactions were performed on either the Illumina HiSeq or NovaSeq sequencing platform. Details on genome sequencing data and quality metrics are provided in Supplementary Table <a href="#MOESM4" class="usa-link">3</a>. Sequencing data are deposited in the European Genome-Phenome Archive (accession no. <a href="https://ega-archive.org/studies/EGAS50000000169" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EGAS50000000169</a>).</p>
<section id="Sec14"><h4 class="pmc_sec_title">Whole-exome sequencing</h4>
<p id="Par47">We performed whole-exome sequencing for all patient samples with the SureSelect Human All Exon V6 Kit (Agilent) following the protocol of the manufacturer. Exon-enriched libraries were subjected to paired-end sequencing on either the Illumina NovaSeq or Illumina HiSeq platform. For the former, libraries were prepared to reach a mean insert size of 200 base pairs (bp) for sequencing with a read length of 2× 100 bp. For the latter, DNA was prepared with a mean insert size of 160 bp for 2× 75 bp paired-end sequencing. Both tumour and normal DNA material were sequenced aiming for a coverage of at least 150× which, following filtering of PCR-duplicated reads and alignment to the annotated human genome (hg19), resulted in an average coverage of 127×. Tumour samples showed a median purity of 88% (interquartile range 78–96%), thus minimizing problems in the assessment of tumour-specific mutations. This allowed for sufficient sequencing depth for reliable analysis for allelic fractions and clonality, as described below. Median genome ploidy was determined at 2.5 (interquartile range 1.9–3.2; Supplementary Table <a href="#MOESM4" class="usa-link">3</a>).</p></section><section id="Sec15"><h4 class="pmc_sec_title">WGS</h4>
<p id="Par48">Whole-genome sequencing (WGS) was performed for samples with sufficient DNA material and quality, additionally providing information on genomic rearrangements not identified by WES. Short-insert DNA libraries from fresh-frozen samples were prepared with the TruSeq DNA Nano PCRfree sample preparation kit (Illumina), and FFPE samples were prepared with the Aceel-NGS 2S Plus DNA library Kit. Paired-end sequencing at a minimum read length of 2× 150 bp was performed, and human DNA libraries were sequenced to an average coverage of 31× for both tumour and matched normal tissue&nbsp;(Supplementary Table <a href="#MOESM4" class="usa-link">3</a>).</p></section><section id="Sec16"><h4 class="pmc_sec_title">RNA-seq</h4>
<p id="Par49">Whole-transcriptome sequencing was performed to determine expression profiles for SCLC tumours in this cohort. RNA-seq was performed with RNA extracted from fresh-frozen human tumour tissue samples. Complementary DNA libraries were prepared from poly-A-selected RNA, applying the Illumina TruSeq protocol for messenger RNA. Libraries were then sequenced with a 2× 100 bp paired-end protocol, generating 50 Mio reads and thus accounting for a minimum mean coverage of 30× of the annotated transcriptome. Samples analysed by transcriptome sequencing are shown in Supplementary Table <a href="#MOESM4" class="usa-link">2</a>.</p></section><section id="Sec17"><h4 class="pmc_sec_title">Dideoxynucleotide sequencing for validation of somatic alterations</h4>
<p id="Par50">If available, transcriptome or additional genome sequencing data were used to validate somatic mutations determined by genome sequencing. In cases without additional sequencing data, dideoxynucleotide chain termination sequencing (Sanger sequencing) was performed to validate key mutations, genomic rearrangements and chimeric fusion transcripts. Specifically, shared clonal mutations of key mutated genome alterations were confirmed by Sanger sequencing as being present in all tumour samples from a patient. For genomic rearrangements determined by WGS in a subset of samples per patient (Supplementary Tables <a href="#MOESM4" class="usa-link">2</a> and <a href="#MOESM4" class="usa-link">3</a>), PCR reactions were performed and the genomic breakpoint was probed and analysed in that subset of samples. Complex genome alterations affecting <em>TP53</em>, <em>RB1</em> and <em>TP73</em> were thus confirmed in all samples of the respective patient (annotation provided in Extended Data Fig. <a href="#Fig9" class="usa-link">4</a>). Clonal assessment of genomic rearrangement affecting key genes was determined with SVclone<sup><a href="#CR40" class="usa-link" aria-describedby="CR40" aria-expanded="false">40</a></sup> (see below). For subclonal and private mutations of key gene alterations, Sanger sequencing was performed to confirm both the mutation call and absence of these alterations in matching tumour samples. Primer pairs were designed to amplify the target region encompassing the somatic alteration. PCR reactions were performed with either genomic DNA, whole-genome-amplified DNA or cDNA. Amplified products were subjected to Sanger sequencing and the respective electropherogram was analysed with Geneious v.8 (<a href="http://www.geneious.com" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">www.geneious.com</a>).</p></section></section><section id="Sec18"><h3 class="pmc_sec_title">Data processing of transcriptome sequencing data</h3>
<p id="Par51">As previously described<sup><a href="#CR5" class="usa-link" aria-describedby="CR5" aria-expanded="false">5</a>,<a href="#CR41" class="usa-link" aria-describedby="CR41" aria-expanded="false">41</a></sup>, transcriptome sequencing data were processed with TRUP&nbsp;(tumour-specimen suited RNA-seq unified pipeline). Paired-end reads were mapped to the human reference genome (GRCh37/hg19). Samples obtained from patient-derived xenotransplant models were mapped to a combined human and murine reference genome (GRCh37/hg19 and GRCm38/mm10). Expression levels were determined for uniquely mapped paired-end reads using Cufflinks&nbsp;referring to the human reference genome, and expression levels were quantified as fragments per kilobase exon per million mapped reads (Supplementary Table <a href="#MOESM4" class="usa-link">10</a>).</p></section><section id="Sec19"><h3 class="pmc_sec_title">Data processing of genome sequencing data</h3>
<p id="Par52">Raw sequencing reads were processed as previously described<sup><a href="#CR5" class="usa-link" aria-describedby="CR5" aria-expanded="false">5</a>,<a href="#CR6" class="usa-link" aria-describedby="CR6" aria-expanded="false">6</a>,<a href="#CR15" class="usa-link" aria-describedby="CR15" aria-expanded="false">15</a></sup>. Reads were aligned to the human reference genome (GRCh37/hg19). Our cohort additionally included patient tumours expanded in immune-compromised mice (<em>n</em> = 32 samples; Fig. <a href="#Fig1" class="usa-link">1a</a> and Supplementary Table <a href="#MOESM4" class="usa-link">2</a>). In these cases, sequencing reads of all samples from a given patient (including the normal reference sample and tumour samples obtained directly from the patient and derived from murine xenotransplant models) were aligned to a combined human and murine reference genome (GRCh37/hg19 and GRCm38/mm10), to exclude sequencing reads from murine cells and to allow for uniform processing of all samples from a given patient. Concordant read-pairs were identified as potential PCR duplicates and were subsequently masked in the alignment file and annotated as the number of masked reads. The quality of the sequencing data is summarized in Supplementary Table <a href="#MOESM4" class="usa-link">3</a>.</p>
<p id="Par53">Human sequencing reads (mapped to the human reference genome) were analysed for tumour purity, tumour ploidy, somatic mutations and copy number alterations<sup><a href="#CR15" class="usa-link" aria-describedby="CR15" aria-expanded="false">15</a></sup>. In addition, WGS data were analysed for genomic rearrangements with the previously described analysis pipeline<sup><a href="#CR5" class="usa-link" aria-describedby="CR5" aria-expanded="false">5</a>,<a href="#CR6" class="usa-link" aria-describedby="CR6" aria-expanded="false">6</a>,<a href="#CR15" class="usa-link" aria-describedby="CR15" aria-expanded="false">15</a>,<a href="#CR42" class="usa-link" aria-describedby="CR42" aria-expanded="false">42</a></sup>. Mutation calling was performed as previously described<sup><a href="#CR5" class="usa-link" aria-describedby="CR5" aria-expanded="false">5</a>,<a href="#CR6" class="usa-link" aria-describedby="CR6" aria-expanded="false">6</a>,<a href="#CR43" class="usa-link" aria-describedby="CR43" aria-expanded="false">43</a></sup>. In brief, variant counts were assessed for tumour and matching normal samples, corrected for sequencing noise and compared with a database of 300 whole-exome and genome sequenced normal samples to filter and determine somatic mutation calls. Variants at low allelic fractions are often prone to result from sequencing artefacts, which occur as a consequence of sequencing noise arising from high-coverage WES due to either fragmented DNA as part of FFPE material or low-level contamination with murine reads in tumours derived from murine xenograft models. We therefore implemented strict filtering criteria for mutations occurring at allelic fractions of less than 0.2. Mutations were then filtered out if (1) the forward–reverse score was below 0.2 (forward–reverse score is 1.0 if 50% of variant reads are found on the forward or reverse read,&nbsp;and 0 if all variant reads are on one orientation); and (2) the allelic fraction of the variant <em>v</em> in consideration of minimal coverage <em>C</em> of the normal or matching tumour sample at position <em>i</em> (<em>C</em><sub><em>i</em></sub><sup>min(tumour/normal)</sup>) did not exceed the read count (rc) threshold with a default value of 10. This was calculated as <em>C</em><sub><em>i</em></sub><sup>min(tumour/normal)</sup> × <em>v</em><sub><em>i</em></sub> &lt; rc. We thus introduced a decision boundary that filters out mutations at relatively low allelic fractions and low sequencing coverage; mutations with low allelic fractions but high coverage were retained for further analyses. In addition we adjusted the stringency of this cut-off for individual samples. Although this stringent cut-off limits the identification of subclonal mutations, we have thus controlled for potential sequencing noise and false-positive mutation calls. As described below, multiregional studies may suggest mutations at very low allele fractions in one tumour that might be more abundant at another tumour site. In this instance, truly subclonal mutations at low allelic fractions that were filtered out in one sample at this step of the analysis were reintroduced as somatic mutation calls if the same mutation passed all stringent filtering criteria in another matched tumour sample.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Analysis of clonal architecture from multiregional and longitudinal tumour samples</h3>
<p id="Par54">We have developed a computational approach to identify individual clones from tumour sequencing data by applying a model that assigns an expected allelic fraction to each mutation under the assumption of clonality (that is, all tumour cells carry this mutation). The expected allelic fraction is corrected for tumour purity, average tumour ploidy and copy number state at the respective genomic coordinates of the said mutation. Relating the observed to the expected allelic fraction results in an estimated CCF that is a specific metric pertinent to each mutation<sup><a href="#CR15" class="usa-link" aria-describedby="CR15" aria-expanded="false">15</a>,<a href="#CR44" class="usa-link" aria-describedby="CR44" aria-expanded="false">44</a>,<a href="#CR45" class="usa-link" aria-describedby="CR45" aria-expanded="false">45</a></sup>. Subsequent clustering of CCFs enabled identification of cell clones represented by subsets of individual mutations. The CCFs and associated clones present in a given tumour thus define the overall clonal composition at the time point of sampling. Through a one-dimensional approach to CCF clustering, we determined for each single tumour its clonal composition (one-dimensional mutation clustering<sup><a href="#CR15" class="usa-link" aria-describedby="CR15" aria-expanded="false">15</a></sup>), a method benchmarked in pan-cancer studies for tumour heterogeneity<sup><a href="#CR44" class="usa-link" aria-describedby="CR44" aria-expanded="false">44</a>,<a href="#CR45" class="usa-link" aria-describedby="CR45" aria-expanded="false">45</a></sup>.</p>
<p id="Par55">To study tumour evolution from multiregional or longitudinal tumour samples from a given patient, we further developed a two-dimensional approach to analysing pairs of samples from the same patient (two-dimensional clustering) and thus to the reconstruction of clonal dynamics<sup><a href="#CR15" class="usa-link" aria-describedby="CR15" aria-expanded="false">15</a>,<a href="#CR43" class="usa-link" aria-describedby="CR43" aria-expanded="false">43</a></sup> (manuscript in preparation). Information on tumour phylogenies, subclonal mutations, subclones and clonal composition of sites is summarized in Fig. <a href="#Fig2" class="usa-link">2</a> and detailed information is provided in Supplementary Table <a href="#MOESM4" class="usa-link">4</a>. In addition, tumour phylogenies determined for each patient are&nbsp;provided in the <a href="#Sec30" class="usa-link">Supplementary Appendix</a>.</p>
<p id="Par56">The sequencing data of tumour samples in our cohort showed an average purity of 85% (Supplementary Table <a href="#MOESM4" class="usa-link">3</a>). Thus WES at an average coverage of 127× provided the required sequencing depth to determine subclones in our data. The analysis of tumour subclones focused on mutation calls as determined by exome sequencing in each sample to track individual tumour clones.</p>
<p id="Par57">The computational method for tumour phylogeny reconstruction starts by executing an extensive set of comparisons and quality controls of copy number states, and a set of mutations and their respective CCFs for each sample. Rather than working with mutation calls and copy number states assessed individually for each tumour sample, we first performed comparisons and adjustments across all samples of a given patient. This included generating a unified copy number segmentation for all samples, which is critical for assigning within each chromosomal segment allele-specific mutation calls, and subsequently to compute CCFs for each mutation. We furthermore created for each patient a unified list of all somatic single-nucleotide mutations (SNMs) determined from each sample, and in all samples we reprobed the presence of somatic mutations of the unified list with relaxed filter criteria for calling somatic mutations at low allelic fractions from sequencing data. This approach allowed us to confirm whether high-confidence mutation calls from one sample were either private events or also present in other patient-matched samples but occurring at lower allelic fractions. Following the refined assessment of copy number states and somatic mutation calls, we determined for each mutation both the observed and expected allele frequency under the assumption of clonality (that is, a cancer cell fraction of 1), so that the CCF of the mutation can be calculated as the ratio between observed and expected allele frequency<sup><a href="#CR15" class="usa-link" aria-describedby="CR15" aria-expanded="false">15</a></sup>. We applied additional filter criteria to mark somatic mutations calls occurring near telomers (that is, located in the tails determined by 1.5% of chromosome length) or centromeric regions on the chromosome, where copy number estimations are frequently error prone and therefore lead to a potentially incorrect calculation of CCF.</p>
<p id="Par58">Somatic insertions and deletions (indel calls) can lead to additional false-positive calls of SNMs as a consequence of improper mapping of reads with inserted or deleted bases. To reduce the number of false-positive SNM calls resulting from indels, we filtered out all SNMs in close proximity (less than 10 bp away) to any mutation call for insertions and deletions.</p>
<p id="Par59">We applied filtering criteria for mutation calls present on chromosomal areas and which, in multiregional analyses, were found to undergo loss of heterozygosity (LOH) in at least one, but not all, of the samples of a given patient<sup><a href="#CR45" class="usa-link" aria-describedby="CR45" aria-expanded="false">45</a>–<a href="#CR47" class="usa-link" aria-describedby="CR47" aria-expanded="false">47</a></sup>. Samples with LOH may not harbour certain mutational calls due to the LOH event, whereas patient-matched samples without LOH may show those mutations. Consequently observed private, or almost private (CCF &lt; 0.2), mutations in one sample lacking LOH events (whereas other patient-matched samples show the LOH event) may indicate a shared clone that undergoes copy number losses, and argue against the subclonal private acquisition of these mutations in this chromosomal area. A clear phylogenetic reconstruction in these cases is not straightforward: due to the inherent uncertainty if the mutations were not present in the other sample (that is, truly private) or lost via the LOH event, these mutations were excluded from phylogenetic tumour clone reconstructions. Following the same criteria, mutations in areas with subclonal copy number events in which one of the copy number clones was hit by an LOH event were also filtered out to avoid further uncertainty in the reconstruction of tumour phylogenies. As previously described<sup><a href="#CR15" class="usa-link" aria-describedby="CR15" aria-expanded="false">15</a></sup>, our method also considered subclonal copy number changes in single-tumour samples. In consideration of copy number status and the observed allele frequency, the number of mutated copies was estimated and the CCF of the mutation determined. Somatic mutations that were found as clonal and that were the subject of subclonal copy number changes within single samples were filtered out.</p>
<p id="Par60">In addition, we used the mapping qualities of the aligner (bwa mem, v.0.7.13-r1126) to filter out mutations in regions where more than 10% of uniquely mapped reads had a mapping quality below 10 (that is, less than 90% probability of having identified the correct mapping position).</p>
<p id="Par61">With regard to potentially shared mutations, we also performed a power analysis to compare the CCFs of a given mutation between two samples with regard to their sequencing depth: we calculated a score per sample to consider the contribution of a single mutated read to the CCF. Per sample, the distribution of these scores could be estimated by a log-normal distribution whose 2.5% tails (<em>z</em>-score = 1.96) were cut off to filter out subsets of over- and underpowered mutations.</p>
<p id="Par62">Last, to check further whether mutations observed as being private to one of the samples were truly private or simply not detected in the other sample (for example, due to insufficient coverage), we applied this statistical test: under the null hypothesis, the mutation is shared with an allelic fraction at least as high as that observed in one of the samples, and the probability (<em>P</em> value) of not detecting it within the given number of sequencing reads can be estimated using a binomial model. If the null hypothesis is rejected, the mutation is considered as being truly private, or otherwise is being filtered out. To determine those mutations that are rejected we apply the false discovery rate control at 5% by Benjamini–Hochberg correction.</p>
<p id="Par63">Subsequent two-dimensional cluster analyses were performed with the set of mutations that passed all filters. This set was binned into a two-dimensional histogram of CCFs representing the observed data, which were modelled as a surface using two-dimensional smoothing splines with a common smoothing parameter. Based on an error estimate of the samples’ CCFs, this method deconvolutes part of the sequencing noise from the data. Subsequently the peaks of the surface were identified and interpreted as cluster centres (marked as red triangles in the cluster images for each patient; <a href="#Sec30" class="usa-link">Supplementary Appendix</a>), and all mutations were assigned to their nearest cluster centre by Euclidean distance. During the assignment procedure we require that shared mutations are assigned only to shared clusters whereas private mutations (that is, those exclusively called in one of the two samples) are assigned only to private clusters. Moreover, we set a minimum threshold of four mutations per cluster and disregarded identified surface peaks otherwise. Considering the cluster centre’s CCF as being representative of the corresponding cell clone, we applied the infinite sites hypothesis assuming that mutations appear once in the evolutionary history, and then determined the CCF sum rule<sup><a href="#CR46" class="usa-link" aria-describedby="CR46" aria-expanded="false">46</a>,<a href="#CR47" class="usa-link" aria-describedby="CR47" aria-expanded="false">47</a></sup> to infer the most probable phylogenetic tree and, in particular, clonal composition per sample at the time point when sampling was derived. In the rare event that tumour phylogenetic rules allow for multiple solutions of tumour phylogeny, we assume maximum parsimony and prefer linear evolution over branched evolution&nbsp;within one sample.</p>
<p id="Par64">In the case of CCF clusters that conflicted with phylogenetic rules we reanalysed somatic mutation calls initially computed with expected allele frequencies under the assumption of clonality. However, chromosomal segments with polyploidy allow for multiple values of absolute numbers of mutated copies (the so-called mutation multiplicity of each mutation call<sup><a href="#CR15" class="usa-link" aria-describedby="CR15" aria-expanded="false">15</a>,<a href="#CR48" class="usa-link" aria-describedby="CR48" aria-expanded="false">48</a></sup>). We therefore accounted for all potential solutions for mutation multiplicity of a given somatic mutation call and computed CCFs that rejected the assumption of clonality (null hypothesis) within the sample and which, in subsequent paired two-dimensional cluster analyses, resolved conflicts in phylogenetic tumour clone reconstruction.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Analysis of tumour phylogenies</h3>
<p id="Par65">Our approach thus enabled us to assign tumour phylogenies for all 65 patients, and to track individual clones from multiregional and longitudinal data. We assigned mutations to the most recent common ancestor (C0) if they were shared and found to be clonal across all tumour sites sampled (that is, having CCFs of approximately 1.0). Alterations with lower CCFs, or those found to be private to single-tumour sites, were determined as subclones. Clusters of at least <em>n</em> = 5 subclonal mutations were defined and labelled as subclone C1, C2 or C3, and derivates of these subclones were assigned accordingly (Fig. <a href="#Fig2" class="usa-link">2a</a>). The resulting tumour phylogenies for all 65 patients are provided in the <a href="#Sec30" class="usa-link">Supplementary Appendix</a>, detailing all spatially and temporally distinct sites analysed and depicting the clinical treatment history for each patient. Additional information is provided in Supplementary Table <a href="#MOESM4" class="usa-link">4</a>.</p>
<p id="Par66">To study patterns of tumour evolution we assigned tumour phylogenetic trees to the following classes (Fig. <a href="#Fig2" class="usa-link">2a</a>): class A if no subclones were identified; class B if one subclone was identified, allowing only for linear evolvement of this subclone; class B if at least two subclones were found with linear phylogenies; class D, phylogenies with one branching event from C1 subclones; class E, phylogenies with one branching event from the most recent common ancestor clone C0; and class F, tumour phylogenies showing two or more branching events.</p>
<p id="Par67">In this regard, increasing the number of tumour samples per patient will enhance the ability to determine subclonal mutations and subclones<sup><a href="#CR16" class="usa-link" aria-describedby="CR16" aria-expanded="false">16</a></sup>. Because we analysed various numbers of samples for each patient (in 37% of cases, more than two samples per patient) we additionally downscaled our analyses to only two samples per patient to permit interpatient comparisons (Fig. <a href="#Fig2" class="usa-link">2d</a>); we thus performed a total of <em>n</em> = 84 paired analyses (Fig. <a href="#Fig1" class="usa-link">1c</a> and Supplementary Table <a href="#MOESM4" class="usa-link">4</a>). In the paired analysis for each patient we chose as representative the analysis showing the highest level of subclonal complexity, defined by the number of subclones and subclonal mutations identified. Downscaling the number of tumour samples per patient did not show any significant change in the absolute number of subclonal mutations but led to reduced numbers of assigned subclones with phylogenetic complexity of classes A–E only (Extended Data Fig. <a href="#Fig6" class="usa-link">1a,b</a>). Downscaling the analysis to two samples per patient for interpatient comparison enabled the study of distinct scenarios throughout the clinical course of the patients (Fig. <a href="#Fig1" class="usa-link">1b</a>). To study the full complexity of a patient’s tumour, all available samples were taken into consideration (<a href="#Sec30" class="usa-link">Supplementary Appendix</a>).</p></section><section id="Sec22"><h3 class="pmc_sec_title">Analysis of cancer cell fractions for structural rearrangements</h3>
<p id="Par68">The analysis of the clonal architecture from multiregional and longitudinal tumour samples focused on the study of CCFs assigned to SNMs. In addition, to assess the clonality of structural rearrangement we applied SVclone (with default settings) to the whole-genome sequencing data of cases harbouring genomic rearrangements in key genes including <em>RB1</em>, <em>TP53</em>, <em>TP73</em> and <em>CREBBP</em>/<em>EP300</em>. We first performed local remapping to the human genome for genomic rearrangements identified by our in-house pipeline<sup><a href="#CR42" class="usa-link" aria-describedby="CR42" aria-expanded="false">42</a></sup> and assigned CCFs for both chromosomal pairs of a given rearrangement with SVclone<sup><a href="#CR40" class="usa-link" aria-describedby="CR40" aria-expanded="false">40</a></sup> (Supplementary Table <a href="#MOESM4" class="usa-link">7</a>). The data are presented in Extended Data Fig. <a href="#Fig10" class="usa-link">5c</a>; the gene alterations identified were found to be part of the clonal proportion of the respective sample.</p></section><section id="Sec23"><h3 class="pmc_sec_title">Analysis of mutational signatures</h3>
<p id="Par69">We analysed our data for the activity of mutational signatures available in COSMIC, referring to SBS (COSMIC_v3.3_SBS_GRChr37_exome<sup><a href="#CR17" class="usa-link" aria-describedby="CR17" aria-expanded="false">17</a></sup>).</p>
<p id="Par70">Mutational signatures were analysed for the following categories: (1) the clonal proportion of all treatment-naive tumours, (2) the subclonal proportion of all treatment-naive tumours and (3) the subclonal proportion of all post-treatment tumours acquired following first-line platinum-based chemotherapy (Fig. <a href="#Fig3" class="usa-link">3a</a>). The analysis of treatment-naive tumours refers to all naive samples available in this cohort (<em>n</em> = 58); signatures assigned to post-treatment tumours included all patients who received first-line platinum-based chemotherapy (<em>n</em> = 45), and we further distinguished whether tumour sites were exposed to chemotherapy alone (<em>n</em> = 20) or were potentially exposed to additional ionizing radiation (<em>n</em> = 25; Supplementary Table <a href="#MOESM4" class="usa-link">2</a>). Due to the high tumour mutational burden, signature assignments to clonal mutations were performed in cases with a median of over 300 mutations. To avoid overfitting and noise, assignments for subclonal mutations were performed only for cases with at least <em>n</em> &gt; 20 mutations.</p>
<p id="Par71">To fit mutational signatures to our samples we applied SigProfilerAssignment (that is, Analyze.cosmic_fit function<sup><a href="#CR17" class="usa-link" aria-describedby="CR17" aria-expanded="false">17</a>,<a href="#CR49" class="usa-link" aria-describedby="CR49" aria-expanded="false">49</a></sup>) to identify a representative subset of signatures. We initially fitted SBS mutational signatures to the mutation catalogue of each sample assigned to the categories. Selecting mutational signatures found in at least <em>n</em> = 5 cases, we thus identified the most prevalent subset of signatures in the clonal and subclonal proportions of treatment-naive and post-treatment tumours (SBS1, SBS2, SBS3, SBS4, SBS5, SBS13, SBS15, SBS16, SBS24, SBS29, SBS39, SBS40 and SBS92), to which all mutations were then fitted. Post-treatment samples additionally showed platinum-based signatures (SBS31 and SBS35), which were therefore included for the assignment of signatures for the subclonal proportion of post-treatment tumours. In addition we applied the in-house-developed computational tool CaMuS<sup><a href="#CR50" class="usa-link" aria-describedby="CR50" aria-expanded="false">50</a></sup> to confirm signature assignments. With CaMuS we first linearly fitted the COSMIC signatures to all mutations for each sample (including clonal and subclonal mutations) using a backward selection procedure. We next selected only those signatures that markedly reduced the cost of the model calculated over the whole dataset. Both tools generated similar results. The results of SigProfilerAssignment are provided in Fig. <a href="#Fig3" class="usa-link">3</a> and Extended Data Figs. <a href="#Fig7" class="usa-link">2</a> and <a href="#Fig8" class="usa-link">3</a>. Comparisons with CaMuS are provided in Extended Data Fig. <a href="#Fig7" class="usa-link">2h</a> and the data are summarized in Supplementary Table <a href="#MOESM4" class="usa-link">5</a>.</p>
<p id="Par72">To track the dynamic activity of mutational signatures in patient-matched tumour samples over the course of the disease, we specifically assigned the subset of signatures identified with SigProfilerAssignment to patient-matched clonal and subclonal mutations pre- and post-treatment, including SBS31 and SBS35 (both related to platinum chemotherapy treatment) for all assignments of signatures. We thus confirmed the presence of platinum-based signatures only in post-treatment subclonal mutations of tumour samples but not in the patient-matched treatment-naive clonal or subclonal proportion of the tumour. In addition we analysed tumour samples from a cohort of patients undergoing subsequent second- or third-line treatment with immune checkpoint inhibition (<em>n</em> = 7). Tumour samples acquired before treatment with ICI were analysed in the categories above (corresponding to samples acquired at the time of relapse following first-line platinum-based chemotherapy). Samples pre- and post-treatment with ICI were analysed with the subset described above (Supplementary Table <a href="#MOESM4" class="usa-link">5</a>).</p>
<p id="Par73">We furthermore tested our whole-genome and whole-exome sequencing data for mutational processes related to ionizing radiation. Following previous studies in this field<sup><a href="#CR25" class="usa-link" aria-describedby="CR25" aria-expanded="false">25</a></sup>, we determined the ratio of insertions to deletions (indels) versus substitution burden and the ratio of deletions versus insertions based on exome- and genome-wide data (Extended Data Fig. <a href="#Fig7" class="usa-link">2f</a>).</p></section><section id="Sec24"><h3 class="pmc_sec_title">Analysis of significant mutations, copy number alterations and genome ploidy</h3>
<p id="Par74">To assess the relevance of key gene alterations in our cohort we referred to our previous study of significant gene alterations determined for 110 human SCLC samples<sup><a href="#CR5" class="usa-link" aria-describedby="CR5" aria-expanded="false">5</a></sup> (Supplementary Table <a href="#MOESM4" class="usa-link">8</a>). In addition we expanded this analysis to our present cohort of 65 patients. We determined the mutational landscape for each patient by creating the union of all mutations identified in multiple samples—this refers to the sum of mutually inclusive and private events (Supplementary Table <a href="#MOESM4" class="usa-link">6</a>). We combined the data from our current cohort of 65 patients with mutational data for 110 human SCLC samples<sup><a href="#CR5" class="usa-link" aria-describedby="CR5" aria-expanded="false">5</a></sup> (<em>n</em> = 175 patients) and determined significant gene alterations at a significance threshold of <em>Q</em> &lt; 0.05 following our previously described method<sup><a href="#CR5" class="usa-link" aria-describedby="CR5" aria-expanded="false">5</a></sup>. In brief, our approach estimates the background mutation rate for each gene and corrects for both synonymous mutations and the expression in human SCLC, referring to the transcriptional data of human SCLC<sup><a href="#CR5" class="usa-link" aria-describedby="CR5" aria-expanded="false">5</a></sup>. The analysis included genes with fragments per kilobase exon per million mapped reads values of over 1 in at least 50 samples. Furthermore we analysed the data for significant mutational hotspots and significant enrichment of gene-damaging mutations. Mutations that significantly cluster within a gene were determined at <em>Q</em> &lt; 0.05 (mutational hotspots). The analysis of gene-damaging mutations refers to (1) nonsense mutations resulting in early stop codons, (2) splice site mutations resulting in aberrant splicing, intron retention or in-frame losses of larger regions within the protein product and (3) frameshift mutations leading to early stop codons and thus resulting in greater changes in the gene and encoded transcript, presumably leading to either no protein product, to proteins with larger deletions within the protein structure or to truncated proteins. The enrichment of gene-damaging alterations was determined at <em>Q</em> &lt; 0.05. We focused our studies on genes recurrently mutated in at least 8% of cases (affecting at least <em>n</em> = 14 patients in the combined analysis of this cohort and the previous cohort<sup><a href="#CR5" class="usa-link" aria-describedby="CR5" aria-expanded="false">5</a></sup>); this allowed us to perform interpatient comparisons and to study a sufficient number of cases in our present cohort of <em>n</em> = 65 patients. To complement our analytical approach we also used other computational tools to study significant gene alterations, including MutSig2CV<sup><a href="#CR51" class="usa-link" aria-describedby="CR51" aria-expanded="false">51</a></sup>, dNdSCV<sup><a href="#CR52" class="usa-link" aria-describedby="CR52" aria-expanded="false">52</a></sup> and OncodriveFML<sup><a href="#CR53" class="usa-link" aria-describedby="CR53" aria-expanded="false">53</a></sup>. In brief, MutSig2CV and dNdSCV were run using their default configuration; for OncodriveFML we used the ‘complement’ method for the signature and ‘amean’ as statistics. Taking into account different levels of stringency, all computational models showed a high degree of overlap. All relevant and significant gene alterations are listed in Supplementary Table <a href="#MOESM4" class="usa-link">8</a>. In addition we studied gene alterations previously reported for targeted sequencing data from larger cohorts of patients with SCLC<sup><a href="#CR4" class="usa-link" aria-describedby="CR4" aria-expanded="false">4</a></sup>; we scored the frequency and significance level of reported alterations for the samples in our cohort. Comparison of these data is provided in Extended Data Fig. <a href="#Fig9" class="usa-link">4e</a> and Supplementary Table <a href="#MOESM4" class="usa-link">8</a>.</p>
<p id="Par75">With regard to frequent alterations affecting <em>TP53</em>, <em>RB1</em> and <em>TP73</em> (Supplementary Table <a href="#MOESM4" class="usa-link">8</a>), which also included larger genomic rearrangements of these genes (Supplementary Table <a href="#MOESM4" class="usa-link">7</a>), we further analysed the gene-damaging effect of alterations. The impact of any genome alterations was evaluated in combination with the transcriptome sequencing data of these tumours, thus further informing on the presumed damage to the gene transcript and resulting protein product (Supplementary Table <a href="#MOESM4" class="usa-link">11</a>).</p>
<p id="Par76">Significant copy number alterations were determined from uncorrected unsegmented copy number signals obtained from whole-exome sequencing data by applying the method CGARS<sup><a href="#CR54" class="usa-link" aria-describedby="CR54" aria-expanded="false">54</a></sup>. We determined the analysis separately for pre- and post-treatment tumour samples, referring to one sample per patient case in both scenarios. Significant amplifications were determined with the upper quantiles 0.30, 0.10 and 0.05; deletions were computed in reference to lower quartiles 0.30, 0.15 and 0.05. Significance threshold was set at <em>Q</em> = 0.05. Significant copy number alterations are listed in Supplementary Table <a href="#MOESM4" class="usa-link">9</a>.</p>
<p id="Par77">Overall genome ploidy was assigned for all patient tumours (Supplementary Table <a href="#MOESM4" class="usa-link">3</a>), with a threshold of 2.8 or above set to define those with higher genome ploidy<sup><a href="#CR33" class="usa-link" aria-describedby="CR33" aria-expanded="false">33</a></sup>. Higher ploidy in cancer genomes can result either from multiple successive and independent copy number gains or through events of whole-genome doubling. To further determine events of genome duplication (or whole-genome doubling), tumours found to undergo ploidy changes were further analysed for the fraction of the genome with LOH to assign an event of genome doubling<sup><a href="#CR45" class="usa-link" aria-describedby="CR45" aria-expanded="false">45</a></sup> (Extended Data Fig. <a href="#Fig10" class="usa-link">5g,h</a>).</p></section><section id="Sec25"><h3 class="pmc_sec_title">Clinical correlations with chemotherapy relapse-free survival</h3>
<p id="Par78">We studied correlations of genomic subsets with relapse-free survival in patients receiving first-line systemic treatment with platinum-based chemotherapy. The analysis focused on the study of <em>n</em> = 55 patients for whom the clinical response to first-line platinum-based chemotherapy was determined. Ten patients from our cohort were not considered for this analysis because of either loss to follow-up (<em>n</em> = 2), declined further treatment and no longer in clinical care (<em>n</em> = 1) or received a lung resection resulting in longer disease-free survival and differences in the dynamics of disease progression (<em>n</em> = 7). We determined relapse-free survival by referring to CTFI, defined as the time between the end of chemotherapy and tumour recurrence, including for patients with disease progression resulting in death. Two patients in our cohort were reported with sepsis-related mortality and were censored in the analysis for recurrence-free survival, leaving a final total of <em>n</em> = 53 patients. All survival analyses were performed with SPSS. Survival distributions were plotted as Kaplan–Meier curves, with <em>P</em> values determined by log-rank test (Extended Data Fig. <a href="#Fig13" class="usa-link">8a</a>). Hazard ratios with a 95% confidence interval and <em>P</em> values were further derived from Cox proportional hazard models. We performed correlations with key genomic parameters referring to significant gene mutations identified in Extended Data Fig. <a href="#Fig9" class="usa-link">4</a> and, in addition, we stratified patients according to genome ploidy (information available for <em>n</em> = 53 patients). We included in our analysis as clinical characteristics information on sex, age and tumour stage. We performed additional analyses on both smoking status and pack years of patients (available for <em>n</em> = 50 and <em>n</em> = 47 patients, respectively). Furthermore we included in our analyses the gene expression of key lineage transcription factors <em>ASCL1</em>, <em>NEUROD1</em> and <em>POU2F3</em> (available for <em>n</em> = 45 patients).</p>
<p id="Par79">We checked that the assumption of proportional hazards was provided by log-minus-log survival plots and by the addition of time-dependent covariates to models. We performed multicollinearity assessment of predictors. We identified relevant gene alterations by performing regressions with backward elimination of insignificant predictors (backwards Wald, at a retention threshold of <em>P</em> &lt; 0.05). The results of the Cox proportional hazard model are shown as forest plots.</p>
<p id="Par80">Clinical correlations of genomic alterations with relapse-free survival were additionally analysed in an independent cohort of patients with SCLC (<em>n</em> = 64) who all received first-line systemic treatment with platinum-based chemotherapy. Note that we used WES and WGS to determine the full spectrum of alterations in key genes in our discovery cohort. By contrast, data for the independent cohort refer to WES data, which limits the detection of complex gene rearrangements that frequently affect <em>CREBBP</em>, <em>EP300</em>, <em>TP73</em> and, to some extent, <em>TP53</em> (ref. <sup><a href="#CR5" class="usa-link" aria-describedby="CR5" aria-expanded="false">5</a></sup>) (Extended Data Fig. <a href="#Fig9" class="usa-link">4a</a>). The somatic alteration status for <em>TP53</em>, <em>TP73</em>, <em>CREBBP</em>, <em>EP300</em> and <em>FMN2</em> as determined by WES is provided in Supplementary Table <a href="#MOESM4" class="usa-link">12</a>.</p></section><section id="Sec26"><h3 class="pmc_sec_title">Immunoblot analysis</h3>
<p id="Par81">Immunoblots were performed to probe tumour cell lysates for the expression of p53 (Extended Data Fig. <a href="#Fig14" class="usa-link">9a</a>). Tissue samples from this cohort containing sufficient material were processed to 5 μm sections on a cryostat maintained at −20 °C. The non-SCLC cell line A549 served as control for the expression of wild-type p53 (ref. <sup><a href="#CR55" class="usa-link" aria-describedby="CR55" aria-expanded="false">55</a></sup>); we confirmed the identity of this cell line by STR profiling and performed tests to ensure no contamination with mycoplasma. Between 40 and 50 tissue sections per sample were sonicated for 3× 10 min and incubated for an additional 30 min in RIPA buffer supplemented with protease inhibitors (cOmplete Mini Protease Inhibitor Cocktail, Roche) and nuclease (benzonase, Millipore) at 4 °C. A549 cells were incubated in RIPA for 30 min at 4 °C. Supernatants were collected following centrifugation at 4 °C for 10 min at 20,000<em>g</em> and protein concentrations determined by bicinchoninic acid assay (Pierce). Either 15 μg (tissue samples) or 90 μg (A549) of protein in 3× Laemmli buffer was separated on 4–12% Tris-glycine SDS–polyacrylamide gel electrophoresis gels (Thermo Fisher Scientific) and transferred to polyvinylidene difluoride membranes (Millipore). PageRuler 10–180 kDa (Thermo Scientific) served as the protein ladder for size determination. Membranes were blocked with Tris buffered saline with 5% milk powder for 1 h at room temperature and incubated overnight with&nbsp;a 1:1,000 dilution of anti-p53 (clone D07, mouse monoclonal antibody, abcam, no. ab80644) and anti-HSP90 (clone C45G5, rabbit monoclonal antibody, Cell Signaling, no. 4877) at 4 °C, washed in Tris buffered saline with Tween 20 and incubated for 1 h with a 1:10,000 dilution of fluorescence-labelled secondary anti-mouse (IRDye 800CW goat anti-mouse, LI-COR, no. 926-32210) and anti-rabbit (IRDye 800CW goat anti-rabbit, LI-COR, no. 926-32211) antibodies. Blots were analysed with the&nbsp;Odyssey CLx imaging system (LI-COR).</p></section><section id="Sec27"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par82">Further information on research design is available in the&nbsp;<a href="#MOESM2" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec28"><h2 class="pmc_sec_title" data-anchor-id="Sec28"><a id="Sec28-anchor" data-anchor-id="Sec28" class="usa-anchor"></a>Online content</h2>
<p id="Par83">Any methods, additional references, Nature Portfolio reporting summaries, source data, extended data, supplementary information, acknowledgements, peer review information; details of author contributions and competing interests; and statements of data and code availability are available at 10.1038/s41586-024-07177-7.</p></section><section id="Sec30"><h2 class="pmc_sec_title" data-anchor-id="Sec30"><a id="Sec30-anchor" data-anchor-id="Sec30" class="usa-anchor"></a>Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/10972747/bin/41586_2024_7177_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link"><span>Supplementary Information</span></a><sup> (16.2MB, pdf) </sup><p>Clinical course and tumour phylogeny determined for 65 patients with SCLC.</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/10972747/bin/41586_2024_7177_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (2.4MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/10972747/bin/41586_2024_7177_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link"><span>Supplementary Fig. 1</span></a><sup> (1.7MB, pdf) </sup><p>Uncropped immunoblots from Extended Data Fig. 10a.</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/10972747/bin/41586_2024_7177_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link"><span>Supplementary Tables</span></a><sup> (24.2MB, xlsx) </sup><p>Supplementary Tables 1–12.</p>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title" data-anchor-id="ack1"><a id="ack1-anchor" data-anchor-id="ack1" class="usa-anchor"></a>Acknowledgements</h2>
<p>We thank all patients who contributed to this study. This work was supported by the German Research Foundation (DFGl project no. 413326622 – SFB1399 to J.G., R.K.T., M.P., R.B., H.C.R., M.C., K.&nbsp;Höpker, A.Q. and J.W.). J.G., R.K.T., H.C.R. and M.P. are also supported by the German Federal Ministry of Education and Research (e:Med consortium InCa, grant nos. 01ZX1901A and 01ZX2201A). R.K.T. and R.B. are also supported by the German state of North Rhine-Westphalia as part of the EFRE initiative (no. EFRE-0800397) and by the Ministry of Culture and Science of the German state of North Rhine-Westphalia as part of the support programme Cancer Center Cologne Essen to N.A. Additional funding was received from the programme Netzwerke 2021, an initiative of the Ministry of Culture and Science of the German state of North Rhine-Westphalia for the CANTAR project, to J.G., R.K.T., H.C.R. and M.P. Support was also received by the DFG (project nos. 497777992 to J.G., N.A., M.P. and A.Q. and 418074181 to M.P. and as part of SFB 1530 (project no. 455784452 to M.P., C.R., R.B., R.T.U. and N.A.), by the German Cancer Aid (Mildred-Scheel professorship to M.P.) and by the Bruno-Helene-Joester foundation (J.G. and M.P.). We thank the computing centre of the University of Cologne&nbsp;(RRZK) for providing computer processing time on the DFG-funded supercomputer&nbsp;CHEOPS, as well as for their support. This work was further supported by the DFG Research Infrastructure West German Genome Centre (project no. 407493903) as part of the Next Generation Sequencing Competence Network (project no. 423957469). Parts of the next-generation sequencing analyses were carried out at the production site in Cologne (Cologne Centre for Genomics). We thank T. Hohenauer for graphical support (life[science]graphics).</p></section><section id="App1" class="app"><section id="Sec29"><h3 class="pmc_sec_title">Extended data figures and tables</h3></section></section><section id="notes1"><h2 class="pmc_sec_title" data-anchor-id="notes1"><a id="notes1-anchor" data-anchor-id="notes1" class="usa-anchor"></a>Author contributions</h2>
<p>J.G. and R.K.T. conceived the project, analysed and interpreted the data and wrote the manuscript. M.P. conceived computational analysis pipelines for the study of genome data to perform tumour evolutionary studies. J.G., L.M., M.C., N.A. and M.P. designed data analyses and implemented methodological approaches for the analysis of tumour evolution and genome data. J.G., L.M. and M.C. performed data analyses. L.M., M.C. and M.P. contributed to writing the manuscript. M.H. and J.G. performed statistical analyses on clinical data. J.G., C.M. and L.K. designed experiments. J.G., L.K., C.M., I.D., M.&nbsp;Bogus and C.C. performed experiments. C.V., C.P.A., L.W. and R.T.U. performed in vivo experiments in mice. V.A., G.B., H.C.R., L.M., K.B. and P.N. provided logistical support for data analysis and genome sequencing. A.H.S., M.&nbsp;Brockmann, A.Q. and R.B. performed pathology review and conducted immunohistochemistry studies. J.-P.W., R.N.F., J.-P.P., M.K., W.E.-R., J.J., E.S., M.&nbsp;Brockmann, S.G., M.&nbsp;Sebastian, J.A.S., J.K., H.-D.H., B.H., M. Schuler, T.P., C.A., T.E., K.T., Y.-D.K., S.K., C.G., M.&nbsp;Serke, F.D., K.&nbsp;Hekmath, J.S., K.-O.K., A.B., F.-G.B., F.G., A.&nbsp;Luers, W.D., R.W., A.&nbsp;Luecke, E.R., A.D., H.C.R., K.&nbsp;Höpker, F.J., G.C., L.H., V.H., K.S. and J.W. provided patient samples and logistical support in the collection of clinical data, and performed clinical data analysis.</p></section><section id="notes2"><h2 class="pmc_sec_title" data-anchor-id="notes2"><a id="notes2-anchor" data-anchor-id="notes2" class="usa-anchor"></a>Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par84"><em>Nature</em> thanks Lauren Byers, Maria Teresa Landi, Nuria Lopez-Bigas and the other, anonymous, reviewer(s) for their contribution to the peer review of this work.</p></section></section><section id="notes3"><h2 class="pmc_sec_title" data-anchor-id="notes3"><a id="notes3-anchor" data-anchor-id="notes3" class="usa-anchor"></a>Funding</h2>
<p>Open access funding provided by Universität zu Köln.</p></section><section id="notes4"><h2 class="pmc_sec_title" data-anchor-id="notes4"><a id="notes4-anchor" data-anchor-id="notes4" class="usa-anchor"></a>Data availability</h2>
<p>The raw sequencing data are deposited in the European Genome-Phenome Archive under accession no. <a href="https://ega-archive.org/studies/EGAS50000000169" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EGAS50000000169</a>. Supporting data are provided as Supplementary Tables.</p></section><section id="notes5"><h2 class="pmc_sec_title" data-anchor-id="notes5"><a id="notes5-anchor" data-anchor-id="notes5" class="usa-anchor"></a>Code availability</h2>
<p>Computational approaches applied in this manuscript are described in <a href="#Sec9" class="usa-link">Methods</a> and were previously published<sup><a href="#CR5" class="usa-link" aria-describedby="CR5" aria-expanded="false">5</a>,<a href="#CR6" class="usa-link" aria-describedby="CR6" aria-expanded="false">6</a>,<a href="#CR15" class="usa-link" aria-describedby="CR15" aria-expanded="false">15</a>,<a href="#CR42" class="usa-link" aria-describedby="CR42" aria-expanded="false">42</a>,<a href="#CR43" class="usa-link" aria-describedby="CR43" aria-expanded="false">43</a></sup>. Our genome data-processing workflow ‘peiflyne’, which we use to perform duplicate masking, mutation calling, mutation filtering and annotation, as well as the algorithm used to identify significant mutational drivers, are available at <a href="http://www.uni-koeln.de/med-fak/peiflyne/peiflyne.tgz" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.uni-koeln.de/med-fak/peiflyne/peiflyne.tgz</a>.</p></section><section id="FPar2"><h2 class="pmc_sec_title" data-anchor-id="FPar2"><a id="FPar2-anchor" data-anchor-id="FPar2" class="usa-anchor"></a>Competing interests</h2>
<p id="Par85">R.K.T. is a founder of, and consultant to, PearlRiver Bio, acquired by Centessa, and a shareholder of Centessa; a founder and shareholder of, and consultant to, Epiphanes; a founder, shareholder and CEO of DISCO Pharmaceuticals. M.P. is consultant to DISCO Pharmaceuticals. J.G. is consultant to DISCO Pharmaceuticals and received honoraria from MSD. J.W. participated in advisory boards and received lecture fees from Abion, Amgen, AstraZeneca, Bayer, Blueprint, BMS, Boehringer Ingelheim, Chugai, Daiichi Sankyo, Janssen, Lilly, Loxo, Merck, Mirati, MSD, Novartis, Nuvalent, Pfizer, Pierre-Fabre, Roche, Seattle Genetics, Takeda and Turning Point; and additional research support from BMS, Janssen Pharmaceutica, Novartis and Pfizer. M. Schuler is consultant to Amgen, AstraZeneca, Blueprint Medicines, Boehringer Ingelheim, Bristol Myers Squibb, GlaxoSmithKline, Janssen, Merck Serono, Novartis, Roche, Sanofi, Takeda, received honoraria from Amgen, Boehringer Ingelheim, Bristol Myers Squibb, Janssen, Novartis, Roche and Sanofi and received research funding from AstraZeneca and Bristol Myers Squibb. R.B. received honoraria for lectures and advisory boards for AbbVie, Amgen, AstraZeneca, Bayer, BMS, Boehringer Ingelheim, Illumina, Janssen, Lilly, Merck Serono, MSD, Novartis, Qiagen, Pfizer, Roche and Targos MP/ Discovery Life Sciences. R.B. serves as a member of the board of directors and is a shareholder of Gnothis, Inc. (SE). A.Q. is consultant to Astellas, BMS, MSD, AstraZeneca, Servier and Oncowissen App/TV. R.N.F. received funding from BMS and MSD. J.K. received honoraria from Roche, Sanofi, Boehringer Ingelheim, MSD, BMS/Celgene, Lilly, Takeda, Novartis, AstraZeneca and Pfizer; is consultant to Roche, Boehringer Ingelheim, MSD, BMS/Celgene, Lilly, Takeda, Novartis, AstraZeneca, Pfizer and Janssen; and received travel and congress support from Roche, Sanofi, Boehringer Ingelheim, MSD, BMS/Celgene, Lilly, Takeda, Novartis, AstraZeneca, Pfizer and Janssen. H.-D.H. has participated in advisory board meetings of Amgen and Boehringer Ingelheim, is a steering committee member for Amgen and coordinating or local PI for Amgen, BMS, Revolution Medicines, Boehringer Ingelheim, Merck, Norvatis, AstraZeneca, Dracen, Daiichi Sankyo and AIO-Studien-gGmbH and reports personal fees from Amgen, BMS and Johnson &amp; Johnson. J.P.P. reports personal fees from Apellis/Sobi, Alexion/AstraZeneca, Amgen, Blueprint Medicines, BMS, Boehringer Ingelheim, Gilead, MSD, Novartis, Pfizer, Roche, Sanofi and SwixxBiopharma outside the submitted work. M.K. is consultant to Boehringer Ingelheim, Bayer, BMS, Chugai and Roche and received honoraria from Novartis and Boehringer Ingelheim. L.H. received honoraria and personal fees from Boehringer Ingelheim, BMS, Pfizer, Roche and AstraZeneca. F.J. received scientific support from Merck and AstraZeneca. V.H. is a shareholder of BMS and Johnson &amp; Johnson and received honoraria from Roche, Amgen, Pfizer, Celgene, BMS, Boehringer, Novartis, AstraZeneca and Lilly. F.G. received funding from ASTRA, Boehringer Ingelheim, BMS, Celgene, Lilly, MSD, Novartis, Pfizer, Roche, Takeda, Siemens and GSK; has been appointed speaker for ASTRA, Boehringer, BMS, Celgene, Lilly, MSD, Novartis, Pfizer, Roche, Takeda, Ariad, Abbvie, Siemens, Tesaro/GSK, Amgen and Daiichi Sankyo; and participates in administrative boards for ASTRA, Boehringer, BMS, Celgene, Lilly, MSD, Novartis, Pfizer, Roche, Takeda, Ariad, Abbvie, Tesaro/GSK, Siemens, Tesaro, Amgen and Daiichi Sankyo. C.G. received honoraria from Roche, Sanofi, Boehringer Ingelheim, MSD, BMS/Celgene, Lilly, Takeda, Novartis, AstraZeneca and Pfizer; is consultant to Roche, Boehringer Ingelheim, MSD, BMS/Celgene, Lilly, Takeda, Novartis, AstraZeneca, Pfizer and Janssen; and has received travel and congress support from Boehringer Ingelheim, MSD, Lilly, Takeda, Novartis, AstraZeneca, Pfizer and Janssen. R.T.U. received honoraria from Roche, Boehringer Ingelheim, GSK, PharmaMar and Bayer. C.A. participated on advisory boards and received honoraria from AstraZeneca, Biotest, Bristol Myers Squibb, MSD and Roche; and is a consultant to Ewimed and received research support from Bristol Myers Squibb and PharmaCep—all outside of the submitted work. M. Sebastian reports grants from AstraZeneca; consulting fees from AstraZeneca, Bristol Myers Squibb, Merck Sharp &amp; Dohme, Novartis, Lilly, Roche, Boehringer lngelheim, Amgen, Takeda, Johnson, Merck Serono and GSK; honoraria for lectures from AstraZeneca, Bristol Myers Squibb, Merck Sharp &amp; Dohme, Novartis, Lilly, Roche, Boehringer lngelheim, Amgen, Takeda, Johnson, CureVac, BioNTech, Merck Serono, GSK, Daiichi and Pfizer; travel support from Takeda and Pfizer; and membership of the advisory board of CureVac and BioNTech. J.A.S. reports personal fees from Boehringer Ingelheim, AstraZeneca, Roche, BMS, Amgen, LEO pharma, Novartis and Takeda—all outside of the submitted work. All other authors report no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title" data-anchor-id="fn-group1"><a id="fn-group1-anchor" data-anchor-id="fn-group1" class="usa-anchor"></a>Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title" data-anchor-id="_ci93_"><a id="_ci93_-anchor" data-anchor-id="_ci93_" class="usa-anchor"></a>Contributor Information</h2>
<p>Julie George, Email: jgeorge@uni-koeln.de.</p>
<p>Martin Peifer, Email: mpeifer@uni-koeln.de.</p>
<p>Roman K. Thomas, Email: roman.thomas@uni-koeln.de</p></section><section id="sec32"><h2 class="pmc_sec_title" data-anchor-id="sec32"><a id="sec32-anchor" data-anchor-id="sec32" class="usa-anchor"></a>Extended data</h2>
<p>is available for this paper at 10.1038/s41586-024-07177-7.</p></section><section id="sec33"><h2 class="pmc_sec_title" data-anchor-id="sec33"><a id="sec33-anchor" data-anchor-id="sec33" class="usa-anchor"></a>Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41586-024-07177-7.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title" data-anchor-id="Bib1"><a id="Bib1-anchor" data-anchor-id="Bib1" class="usa-anchor"></a>References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Gazdar AF, Bunn PA, Minna JD. Small-cell lung cancer: what we know, what we need to know and the path forward. Nat. Rev. Cancer. 2017;17:725–737. doi: 10.1038/nrc.2017.87.</cite> [<a href="https://doi.org/10.1038/nrc.2017.87" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29077690/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Cancer&amp;title=Small-cell%20lung%20cancer:%20what%20we%20know,%20what%20we%20need%20to%20know%20and%20the%20path%20forward&amp;author=AF%20Gazdar&amp;author=PA%20Bunn&amp;author=JD%20Minna&amp;volume=17&amp;publication_year=2017&amp;pages=725-737&amp;pmid=29077690&amp;doi=10.1038/nrc.2017.87&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Zugazagoitia, J. &amp; Paz-ares, L. Extensive-stage small-cell lung cancer: first-line and second-line treatment options. <em>J. Clin. Oncol</em>. <strong>40</strong>, 671–681 (2022).</cite> [<a href="https://doi.org/10.1200/JCO.21.01881" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34985925/" class="usa-link">PubMed</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Howlader N, et al.  The effect of advances in lung-cancer treatment on population mortality. New Engl. J. Med. 2020;383:640–649. doi: 10.1056/NEJMoa1916623.</cite> [<a href="https://doi.org/10.1056/NEJMoa1916623" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8577315/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32786189/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=New%20Engl.%20J.%20Med.&amp;title=The%20effect%20of%20advances%20in%20lung-cancer%20treatment%20on%20population%20mortality&amp;author=N%20Howlader&amp;volume=383&amp;publication_year=2020&amp;pages=640-649&amp;pmid=32786189&amp;doi=10.1056/NEJMoa1916623&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Rudin, C. M., Brambilla, E., Faivre-Finn, C. &amp; Sage, J. Small-cell lung cancer. <em>Nat. Rev. Dis. Primers</em><strong>7</strong>, 3 (2021).</cite> [<a href="https://doi.org/10.1038/s41572-020-00235-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8177722/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33446664/" class="usa-link">PubMed</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>George J, et al.  Comprehensive genomic profiles of small cell lung cancer. Nature. 2015;524:47–53. doi: 10.1038/nature14664.</cite> [<a href="https://doi.org/10.1038/nature14664" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4861069/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26168399/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Comprehensive%20genomic%20profiles%20of%20small%20cell%20lung%20cancer&amp;author=J%20George&amp;volume=524&amp;publication_year=2015&amp;pages=47-53&amp;pmid=26168399&amp;doi=10.1038/nature14664&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Peifer M, et al.  Integrative genome analyses identify key somatic driver mutations of small-cell lung cancer. Nat. Genet. 2012;44:1104–1110. doi: 10.1038/ng.2396.</cite> [<a href="https://doi.org/10.1038/ng.2396" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4915822/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22941188/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&amp;title=Integrative%20genome%20analyses%20identify%20key%20somatic%20driver%20mutations%20of%20small-cell%20lung%20cancer&amp;author=M%20Peifer&amp;volume=44&amp;publication_year=2012&amp;pages=1104-1110&amp;pmid=22941188&amp;doi=10.1038/ng.2396&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Rudin CM, et al.  Comprehensive genomic analysis identifies SOX2 as a frequently amplified gene in small-cell lung cancer. Nat. Genet. 2012;44:1111–1116. doi: 10.1038/ng.2405.</cite> [<a href="https://doi.org/10.1038/ng.2405" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3557461/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22941189/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&amp;title=Comprehensive%20genomic%20analysis%20identifies%20SOX2%20as%20a%20frequently%20amplified%20gene%20in%20small-cell%20lung%20cancer&amp;author=CM%20Rudin&amp;volume=44&amp;publication_year=2012&amp;pages=1111-1116&amp;pmid=22941189&amp;doi=10.1038/ng.2405&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Rudin CM, et al.  Molecular subtypes of small cell lung cancer: a synthesis of human and mouse model data. Nat. Rev. Cancer. 2019;19:289–297. doi: 10.1038/s41568-019-0133-9.</cite> [<a href="https://doi.org/10.1038/s41568-019-0133-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6538259/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30926931/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Cancer&amp;title=Molecular%20subtypes%20of%20small%20cell%20lung%20cancer:%20a%20synthesis%20of%20human%20and%20mouse%20model%20data&amp;author=CM%20Rudin&amp;volume=19&amp;publication_year=2019&amp;pages=289-297&amp;pmid=30926931&amp;doi=10.1038/s41568-019-0133-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Gay CM, et al.  Patterns of transcription factor programs and immune pathway activation define four major subtypes of SCLC with distinct therapeutic vulnerabilities. Cancer Cell. 2021;39:346–360. doi: 10.1016/j.ccell.2020.12.014.</cite> [<a href="https://doi.org/10.1016/j.ccell.2020.12.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8143037/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33482121/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&amp;title=Patterns%20of%20transcription%20factor%20programs%20and%20immune%20pathway%20activation%20define%20four%20major%20subtypes%20of%20SCLC%20with%20distinct%20therapeutic%20vulnerabilities&amp;author=CM%20Gay&amp;volume=39&amp;publication_year=2021&amp;pages=346-360&amp;pmid=33482121&amp;doi=10.1016/j.ccell.2020.12.014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Gardner EE, et al.  Chemosensitive relapse in small cell lung cancer proceeds through an EZH2-SLFN11 axis. Cancer Cell. 2017;31:286–299. doi: 10.1016/j.ccell.2017.01.006.</cite> [<a href="https://doi.org/10.1016/j.ccell.2017.01.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5313262/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28196596/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&amp;title=Chemosensitive%20relapse%20in%20small%20cell%20lung%20cancer%20proceeds%20through%20an%20EZH2-SLFN11%20axis&amp;author=EE%20Gardner&amp;volume=31&amp;publication_year=2017&amp;pages=286-299&amp;pmid=28196596&amp;doi=10.1016/j.ccell.2017.01.006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Wagner AH, et al.  Recurrent WNT pathway alterations are frequent in relapsed small cell lung cancer. Nat. Commun. 2018;9:3787. doi: 10.1038/s41467-018-06162-9.</cite> [<a href="https://doi.org/10.1038/s41467-018-06162-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6141466/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30224629/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=Recurrent%20WNT%20pathway%20alterations%20are%20frequent%20in%20relapsed%20small%20cell%20lung%20cancer&amp;author=AH%20Wagner&amp;volume=9&amp;publication_year=2018&amp;pages=3787&amp;pmid=30224629&amp;doi=10.1038/s41467-018-06162-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Hodgkinson CL, et al.  Tumorigenicity and genetic profiling of circulating tumor cells in small-cell lung cancer. Nat. Med. 2014;20:897–903. doi: 10.1038/nm.3600.</cite> [<a href="https://doi.org/10.1038/nm.3600" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24880617/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Med.&amp;title=Tumorigenicity%20and%20genetic%20profiling%20of%20circulating%20tumor%20cells%20in%20small-cell%20lung%20cancer&amp;author=CL%20Hodgkinson&amp;volume=20&amp;publication_year=2014&amp;pages=897-903&amp;pmid=24880617&amp;doi=10.1038/nm.3600&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Drapkin, B. J. et al. Genomic and functional fidelity of small cell lung cancer patient-derived xenografts. <em>Cancer Discov</em>. 10.1158/2159-8290.CD-17-0935 (2018).</cite> [<a href="https://doi.org/10.1158/2159-8290.CD-17-0935" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6369413/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29483136/" class="usa-link">PubMed</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>NCCN. <em>Clinical Practice Guidelines in Oncology: Small Cell Lung Cancer</em><a href="https://www.nccn.org/guidelines/guidelines-detail?category=1&amp;id=1462" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.nccn.org/guidelines/guidelines-detail?category=1&amp;id=1462</a> (2013).</cite>
</li>
<li id="CR15">
<span class="label">15.</span><cite>Cun Y, Yang TP, Achter V, Lang U, Peifer M. Copy-number analysis and inference of subclonal populations in cancer genomes using Sclust. Nat. Protoc. 2018;13:1488–1501. doi: 10.1038/nprot.2018.033.</cite> [<a href="https://doi.org/10.1038/nprot.2018.033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29844525/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Protoc.&amp;title=Copy-number%20analysis%20and%20inference%20of%20subclonal%20populations%20in%20cancer%20genomes%20using%20Sclust&amp;author=Y%20Cun&amp;author=TP%20Yang&amp;author=V%20Achter&amp;author=U%20Lang&amp;author=M%20Peifer&amp;volume=13&amp;publication_year=2018&amp;pages=1488-1501&amp;pmid=29844525&amp;doi=10.1038/nprot.2018.033&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Mcgranahan N, Swanton C. lonal heterogeneity and tumor evolution: past, present, and the future. Cell. 2017;168:613–628. doi: 10.1016/j.cell.2017.01.018.</cite> [<a href="https://doi.org/10.1016/j.cell.2017.01.018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28187284/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=lonal%20heterogeneity%20and%20tumor%20evolution:%20past,%20present,%20and%20the%20future&amp;author=N%20Mcgranahan&amp;author=C%20Swanton&amp;volume=168&amp;publication_year=2017&amp;pages=613-628&amp;pmid=28187284&amp;doi=10.1016/j.cell.2017.01.018&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Alexandrov LB, et al.  The repertoire of mutational signatures in human cancer. Nature. 2020;578:94–101. doi: 10.1038/s41586-020-1943-3.</cite> [<a href="https://doi.org/10.1038/s41586-020-1943-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7054213/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32025018/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The%20repertoire%20of%20mutational%20signatures%20in%20human%20cancer&amp;author=LB%20Alexandrov&amp;volume=578&amp;publication_year=2020&amp;pages=94-101&amp;pmid=32025018&amp;doi=10.1038/s41586-020-1943-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Alexandrov LB, et al.  Mutational signatures associated with tobacco smoking in human cancer. Science. 2016;354:618–622. doi: 10.1126/science.aag0299.</cite> [<a href="https://doi.org/10.1126/science.aag0299" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6141049/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27811275/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Mutational%20signatures%20associated%20with%20tobacco%20smoking%20in%20human%20cancer&amp;author=LB%20Alexandrov&amp;volume=354&amp;publication_year=2016&amp;pages=618-622&amp;pmid=27811275&amp;doi=10.1126/science.aag0299&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>McGranahan N, et al.  Clonal status of actionable driver events and the timing of mutational processes in cancer evolution. Sci. Transl. Med. 2015;7:283ra54. doi: 10.1126/scitranslmed.aaa1408.</cite> [<a href="https://doi.org/10.1126/scitranslmed.aaa1408" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4636056/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25877892/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Transl.%20Med.&amp;title=Clonal%20status%20of%20actionable%20driver%20events%20and%20the%20timing%20of%20mutational%20processes%20in%20cancer%20evolution&amp;author=N%20McGranahan&amp;volume=7&amp;publication_year=2015&amp;pages=283ra54&amp;pmid=25877892&amp;doi=10.1126/scitranslmed.aaa1408&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Jamal-Hanjani, M. et al. Tracking the evolution of non-small-cell lung cancer. <em>N. Engl. J. Med</em>. <strong>376</strong>, 2109–2121 (2017).</cite> [<a href="https://doi.org/10.1056/NEJMoa1616288" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28445112/" class="usa-link">PubMed</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Alexandrov LB, Jones PH, Wedge DC, Sale JE, Peter J. Clock-like mutational processes in human somatic cells. Nat. Genet. 2015;47:1402–1407. doi: 10.1038/ng.3441.</cite> [<a href="https://doi.org/10.1038/ng.3441" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4783858/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26551669/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&amp;title=Clock-like%20mutational%20processes%20in%20human%20somatic%20cells&amp;author=LB%20Alexandrov&amp;author=PH%20Jones&amp;author=DC%20Wedge&amp;author=JE%20Sale&amp;author=J%20Peter&amp;volume=47&amp;publication_year=2015&amp;pages=1402-1407&amp;pmid=26551669&amp;doi=10.1038/ng.3441&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Marchese S, et al.  Aflatoxin B1 and M1: biological properties and their involvement in cancer development. Toxins (Basel) 2018;10:214. doi: 10.3390/toxins10060214.</cite> [<a href="https://doi.org/10.3390/toxins10060214" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6024316/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29794965/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Toxins%20(Basel)&amp;title=Aflatoxin%20B1%20and%20M1:%20biological%20properties%20and%20their%20involvement%20in%20cancer%20development&amp;author=S%20Marchese&amp;volume=10&amp;publication_year=2018&amp;pages=214&amp;pmid=29794965&amp;doi=10.3390/toxins10060214&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Pich O, et al.  The mutational footprints of cancer therapies. Nat. Genet. 2019;51:1732–1740. doi: 10.1038/s41588-019-0525-5.</cite> [<a href="https://doi.org/10.1038/s41588-019-0525-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6887544/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31740835/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&amp;title=The%20mutational%20footprints%20of%20cancer%20therapies&amp;author=O%20Pich&amp;volume=51&amp;publication_year=2019&amp;pages=1732-1740&amp;pmid=31740835&amp;doi=10.1038/s41588-019-0525-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Kucab JE, et al.  A compendium of mutational signatures of environmental agents. Cell. 2019;177:821–836. doi: 10.1016/j.cell.2019.03.001.</cite> [<a href="https://doi.org/10.1016/j.cell.2019.03.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6506336/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30982602/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=A%20compendium%20of%20mutational%20signatures%20of%20environmental%20agents&amp;author=JE%20Kucab&amp;volume=177&amp;publication_year=2019&amp;pages=821-836&amp;pmid=30982602&amp;doi=10.1016/j.cell.2019.03.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Behjati S, et al.  Mutational signatures of ionizing radiation in second malignancies. Nat. Commun. 2016;7:12605. doi: 10.1038/ncomms12605.</cite> [<a href="https://doi.org/10.1038/ncomms12605" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5027243/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27615322/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=Mutational%20signatures%20of%20ionizing%20radiation%20in%20second%20malignancies&amp;author=S%20Behjati&amp;volume=7&amp;publication_year=2016&amp;pages=12605&amp;pmid=27615322&amp;doi=10.1038/ncomms12605&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Lee JK, et al.  Clonal history and genetic predictors of transformation into small-cell carcinomas from lung adenocarcinomas. J. Clin. Oncol. 2017;35:3065–3074. doi: 10.1200/JCO.2016.71.9096.</cite> [<a href="https://doi.org/10.1200/JCO.2016.71.9096" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28498782/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol.&amp;title=Clonal%20history%20and%20genetic%20predictors%20of%20transformation%20into%20small-cell%20carcinomas%20from%20lung%20adenocarcinomas&amp;author=JK%20Lee&amp;volume=35&amp;publication_year=2017&amp;pages=3065-3074&amp;pmid=28498782&amp;doi=10.1200/JCO.2016.71.9096&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Brastianos PK, et al.  Genomic characterization of brain metastases reveals branched evolution and potential therapeutic targets. Cancer Discov. 2015;5:1164–1177. doi: 10.1158/2159-8290.CD-15-0369.</cite> [<a href="https://doi.org/10.1158/2159-8290.CD-15-0369" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4916970/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26410082/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Discov.&amp;title=Genomic%20characterization%20of%20brain%20metastases%20reveals%20branched%20evolution%20and%20potential%20therapeutic%20targets&amp;author=PK%20Brastianos&amp;volume=5&amp;publication_year=2015&amp;pages=1164-1177&amp;pmid=26410082&amp;doi=10.1158/2159-8290.CD-15-0369&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Stiewe T, Haran TE. How mutations shape p53 interactions with the genome to promote tumorigenesis and drug resistance. Drug Resist. Updat. 2018;38:27–43. doi: 10.1016/j.drup.2018.05.001.</cite> [<a href="https://doi.org/10.1016/j.drup.2018.05.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29857816/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Drug%20Resist.%20Updat.&amp;title=How%20mutations%20shape%20p53%20interactions%20with%20the%20genome%20to%20promote%20tumorigenesis%20and%20drug%20resistance&amp;author=T%20Stiewe&amp;author=TE%20Haran&amp;volume=38&amp;publication_year=2018&amp;pages=27-43&amp;pmid=29857816&amp;doi=10.1016/j.drup.2018.05.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Dammert MA, et al.  MYC paralog-dependent apoptotic priming orchestrates a spectrum of vulnerabilities in small cell lung cancer. Nat. Commun. 2019;10:3485. doi: 10.1038/s41467-019-11371-x.</cite> [<a href="https://doi.org/10.1038/s41467-019-11371-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6677768/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31375684/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=MYC%20paralog-dependent%20apoptotic%20priming%20orchestrates%20a%20spectrum%20of%20vulnerabilities%20in%20small%20cell%20lung%20cancer&amp;author=MA%20Dammert&amp;volume=10&amp;publication_year=2019&amp;pages=3485&amp;pmid=31375684&amp;doi=10.1038/s41467-019-11371-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Mollaoglu G, et al.  MYC drives progression of small cell lung cancer to a variant neuroendocrine subtype with vulnerability to aurora kinase inhibition. Cancer Cell. 2017;31:270–285. doi: 10.1016/j.ccell.2016.12.005.</cite> [<a href="https://doi.org/10.1016/j.ccell.2016.12.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5310991/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28089889/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&amp;title=MYC%20drives%20progression%20of%20small%20cell%20lung%20cancer%20to%20a%20variant%20neuroendocrine%20subtype%20with%20vulnerability%20to%20aurora%20kinase%20inhibition&amp;author=G%20Mollaoglu&amp;volume=31&amp;publication_year=2017&amp;pages=270-285&amp;pmid=28089889&amp;doi=10.1016/j.ccell.2016.12.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Sos ML, et al.  A framework for identification of actionable cancer genome dependencies in small cell lung cancer. Proc. Natl Acad. Sci. USA. 2012;109:17034–17039. doi: 10.1073/pnas.1207310109.</cite> [<a href="https://doi.org/10.1073/pnas.1207310109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3479457/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23035247/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;title=A%20framework%20for%20identification%20of%20actionable%20cancer%20genome%20dependencies%20in%20small%20cell%20lung%20cancer&amp;author=ML%20Sos&amp;volume=109&amp;publication_year=2012&amp;pages=17034-17039&amp;pmid=23035247&amp;doi=10.1073/pnas.1207310109&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Ireland AS, et al.  MYC drives temporal evolution of small cell lung cancer subtypes by reprogramming neuroendocrine fate. Cancer Cell. 2020;38:60–78. doi: 10.1016/j.ccell.2020.05.001.</cite> [<a href="https://doi.org/10.1016/j.ccell.2020.05.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7393942/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32473656/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&amp;title=MYC%20drives%20temporal%20evolution%20of%20small%20cell%20lung%20cancer%20subtypes%20by%20reprogramming%20neuroendocrine%20fate&amp;author=AS%20Ireland&amp;volume=38&amp;publication_year=2020&amp;pages=60-78&amp;pmid=32473656&amp;doi=10.1016/j.ccell.2020.05.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Bielski CM, et al.  Genome doubling shapes the evolution and prognosis of advanced cancers. Nat. Genet. 2018;50:1189–1195. doi: 10.1038/s41588-018-0165-1.</cite> [<a href="https://doi.org/10.1038/s41588-018-0165-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6072608/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30013179/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&amp;title=Genome%20doubling%20shapes%20the%20evolution%20and%20prognosis%20of%20advanced%20cancers&amp;author=CM%20Bielski&amp;volume=50&amp;publication_year=2018&amp;pages=1189-1195&amp;pmid=30013179&amp;doi=10.1038/s41588-018-0165-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Dewhurst SM, et al.  Tolerance of whole-genome doubling propagates chromosomal instability and accelerates cancer genome evolution. Cancer Discov. 2014;4:175–185. doi: 10.1158/2159-8290.CD-13-0285.</cite> [<a href="https://doi.org/10.1158/2159-8290.CD-13-0285" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4293454/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24436049/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Discov.&amp;title=Tolerance%20of%20whole-genome%20doubling%20propagates%20chromosomal%20instability%20and%20accelerates%20cancer%20genome%20evolution&amp;author=SM%20Dewhurst&amp;volume=4&amp;publication_year=2014&amp;pages=175-185&amp;pmid=24436049&amp;doi=10.1158/2159-8290.CD-13-0285&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Stewart CA, et al.  Single-cell analyses reveal increased intratumoral heterogeneity after the onset of therapy resistance in small-cell lung cancer. Nat. Cancer. 2020;1:423–436. doi: 10.1038/s43018-019-0020-z.</cite> [<a href="https://doi.org/10.1038/s43018-019-0020-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7842382/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33521652/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Cancer&amp;title=Single-cell%20analyses%20reveal%20increased%20intratumoral%20heterogeneity%20after%20the%20onset%20of%20therapy%20resistance%20in%20small-cell%20lung%20cancer&amp;author=CA%20Stewart&amp;volume=1&amp;publication_year=2020&amp;pages=423-436&amp;pmid=33521652&amp;doi=10.1038/s43018-019-0020-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Saleh MM, et al.  Comprehensive analysis of TP53 and KEAP1 mutations and their impact on survival in localized- and advanced-stage NSCLC. J. Thorac. Oncol. 2022;17:76–88. doi: 10.1016/j.jtho.2021.08.764.</cite> [<a href="https://doi.org/10.1016/j.jtho.2021.08.764" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34601169/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Thorac.%20Oncol.&amp;title=Comprehensive%20analysis%20of%20TP53%20and%20KEAP1%20mutations%20and%20their%20impact%20on%20survival%20in%20localized-%20and%20advanced-stage%20NSCLC&amp;author=MM%20Saleh&amp;volume=17&amp;publication_year=2022&amp;pages=76-88&amp;pmid=34601169&amp;doi=10.1016/j.jtho.2021.08.764&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Bernard E, et al.  Implications of TP53 allelic state for genome stability, clinical presentation and outcomes in myelodysplastic syndromes. Nat. Med. 2020;26:1549–1556. doi: 10.1038/s41591-020-1008-z.</cite> [<a href="https://doi.org/10.1038/s41591-020-1008-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8381722/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32747829/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Med.&amp;title=Implications%20of%20TP53%20allelic%20state%20for%20genome%20stability,%20clinical%20presentation%20and%20outcomes%20in%20myelodysplastic%20syndromes&amp;author=E%20Bernard&amp;volume=26&amp;publication_year=2020&amp;pages=1549-1556&amp;pmid=32747829&amp;doi=10.1038/s41591-020-1008-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Klimovich B, et al.  P53 partial loss-of-function mutations sensitize to chemotherapy. Oncogene. 2022;41:1011–1023. doi: 10.1038/s41388-021-02141-5.</cite> [<a href="https://doi.org/10.1038/s41388-021-02141-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8837531/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34907344/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=P53%20partial%20loss-of-function%20mutations%20sensitize%20to%20chemotherapy&amp;author=B%20Klimovich&amp;volume=41&amp;publication_year=2022&amp;pages=1011-1023&amp;pmid=34907344&amp;doi=10.1038/s41388-021-02141-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Malinowska-Ozdowy K, et al.  KRAS and CREBBP mutations: a relapse-linked malicious liaison in childhood high hyperdiploid acute lymphoblastic leukemia. Leukemia. 2015;29:1656–1667. doi: 10.1038/leu.2015.107.</cite> [<a href="https://doi.org/10.1038/leu.2015.107" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4530204/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25917266/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Leukemia&amp;title=KRAS%20and%20CREBBP%20mutations:%20a%20relapse-linked%20malicious%20liaison%20in%20childhood%20high%20hyperdiploid%20acute%20lymphoblastic%20leukemia&amp;author=K%20Malinowska-Ozdowy&amp;volume=29&amp;publication_year=2015&amp;pages=1656-1667&amp;pmid=25917266&amp;doi=10.1038/leu.2015.107&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Cmero M, et al.  Inferring structural variant cancer cell fraction. Nat. Commun. 2020;11:730. doi: 10.1038/s41467-020-14351-8.</cite> [<a href="https://doi.org/10.1038/s41467-020-14351-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7002525/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32024845/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=Inferring%20structural%20variant%20cancer%20cell%20fraction&amp;author=M%20Cmero&amp;volume=11&amp;publication_year=2020&amp;pages=730&amp;pmid=32024845&amp;doi=10.1038/s41467-020-14351-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Fernandez-Cuesta L, et al.  Identification of novel fusion genes in lung cancer using breakpoint assembly of transcriptome sequencing data. Genome Biol. 2015;16:7. doi: 10.1186/s13059-014-0558-0.</cite> [<a href="https://doi.org/10.1186/s13059-014-0558-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4300615/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25650807/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol.&amp;title=Identification%20of%20novel%20fusion%20genes%20in%20lung%20cancer%20using%20breakpoint%20assembly%20of%20transcriptome%20sequencing%20data&amp;author=L%20Fernandez-Cuesta&amp;volume=16&amp;publication_year=2015&amp;pages=7&amp;pmid=25650807&amp;doi=10.1186/s13059-014-0558-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Rosswog C, et al.  Chromothripsis followed by circular recombination drives oncogene amplification in human cancer. Nat. Genet. 2021;53:1673–1685. doi: 10.1038/s41588-021-00951-7.</cite> [<a href="https://doi.org/10.1038/s41588-021-00951-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34782764/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&amp;title=Chromothripsis%20followed%20by%20circular%20recombination%20drives%20oncogene%20amplification%20in%20human%20cancer&amp;author=C%20Rosswog&amp;volume=53&amp;publication_year=2021&amp;pages=1673-1685&amp;pmid=34782764&amp;doi=10.1038/s41588-021-00951-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Herling CD, et al.  Clonal dynamics towards the development of venetoclax resistance in chronic lymphocytic leukemia. Nat. Commun. 2018;9:727. doi: 10.1038/s41467-018-03170-7.</cite> [<a href="https://doi.org/10.1038/s41467-018-03170-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5820258/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29463802/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=Clonal%20dynamics%20towards%20the%20development%20of%20venetoclax%20resistance%20in%20chronic%20lymphocytic%20leukemia&amp;author=CD%20Herling&amp;volume=9&amp;publication_year=2018&amp;pages=727&amp;pmid=29463802&amp;doi=10.1038/s41467-018-03170-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Gerstung M, et al.  The evolutionary history of 2,658 cancers. Nature. 2020;578:122–128. doi: 10.1038/s41586-019-1907-7.</cite> [<a href="https://doi.org/10.1038/s41586-019-1907-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7054212/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32025013/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The%20evolutionary%20history%20of%202,658%20cancers&amp;author=M%20Gerstung&amp;volume=578&amp;publication_year=2020&amp;pages=122-128&amp;pmid=32025013&amp;doi=10.1038/s41586-019-1907-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Dentro SC, et al.  Characterizing genetic intra-tumor heterogeneity across 2,658 human cancer genomes. Cell. 2021;184:2239–2254. doi: 10.1016/j.cell.2021.03.009.</cite> [<a href="https://doi.org/10.1016/j.cell.2021.03.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8054914/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33831375/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Characterizing%20genetic%20intra-tumor%20heterogeneity%20across%202,658%20human%20cancer%20genomes&amp;author=SC%20Dentro&amp;volume=184&amp;publication_year=2021&amp;pages=2239-2254&amp;pmid=33831375&amp;doi=10.1016/j.cell.2021.03.009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Schwartz R, Schäffer AA. The evolution of tumour phylogenetics: principles and practice. Nat. Rev. Genet. 2017;18:213–229. doi: 10.1038/nrg.2016.170.</cite> [<a href="https://doi.org/10.1038/nrg.2016.170" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5886015/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28190876/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Genet.&amp;title=The%20evolution%20of%20tumour%20phylogenetics:%20principles%20and%20practice&amp;author=R%20Schwartz&amp;author=AA%20Sch%C3%A4ffer&amp;volume=18&amp;publication_year=2017&amp;pages=213-229&amp;pmid=28190876&amp;doi=10.1038/nrg.2016.170&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Tarabichi M, et al.  A practical guide to cancer subclonal reconstruction from DNA sequencing. Nat. Methods. 2021;18:144–155. doi: 10.1038/s41592-020-01013-2.</cite> [<a href="https://doi.org/10.1038/s41592-020-01013-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7867630/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33398189/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Methods&amp;title=A%20practical%20guide%20to%20cancer%20subclonal%20reconstruction%20from%20DNA%20sequencing&amp;author=M%20Tarabichi&amp;volume=18&amp;publication_year=2021&amp;pages=144-155&amp;pmid=33398189&amp;doi=10.1038/s41592-020-01013-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Carter SL, et al.  Absolute quantification of somatic DNA alterations in human cancer. Nat. Biotechnol. 2012;30:413–421. doi: 10.1038/nbt.2203.</cite> [<a href="https://doi.org/10.1038/nbt.2203" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4383288/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22544022/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Biotechnol.&amp;title=Absolute%20quantification%20of%20somatic%20DNA%20alterations%20in%20human%20cancer&amp;author=SL%20Carter&amp;volume=30&amp;publication_year=2012&amp;pages=413-421&amp;pmid=22544022&amp;doi=10.1038/nbt.2203&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Islam SMA, et al.  Uncovering novel mutational signatures by de novo extraction with SigProfilerExtractor. Cell Genomics. 2022;2:100179. doi: 10.1016/j.xgen.2022.100179.</cite> [<a href="https://doi.org/10.1016/j.xgen.2022.100179" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9646490/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36388765/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Genomics&amp;title=Uncovering%20novel%20mutational%20signatures%20by%20de%20novo%20extraction%20with%20SigProfilerExtractor&amp;author=SMA%20Islam&amp;volume=2&amp;publication_year=2022&amp;pages=100179&amp;pmid=36388765&amp;doi=10.1016/j.xgen.2022.100179&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Cartolano M, et al.  CaMuS: simultaneous fitting and de novo imputation of cancer mutational signature. Sci. Rep. 2020;10:19316. doi: 10.1038/s41598-020-75753-8.</cite> [<a href="https://doi.org/10.1038/s41598-020-75753-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7653908/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33168834/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=CaMuS:%20simultaneous%20fitting%20and%20de%20novo%20imputation%20of%20cancer%20mutational%20signature&amp;author=M%20Cartolano&amp;volume=10&amp;publication_year=2020&amp;pages=19316&amp;pmid=33168834&amp;doi=10.1038/s41598-020-75753-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Lawrence MS, et al.  Mutational heterogeneity in cancer and the search for new cancer-associated genes. Nature. 2013;499:214–218. doi: 10.1038/nature12213.</cite> [<a href="https://doi.org/10.1038/nature12213" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3919509/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23770567/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Mutational%20heterogeneity%20in%20cancer%20and%20the%20search%20for%20new%20cancer-associated%20genes&amp;author=MS%20Lawrence&amp;volume=499&amp;publication_year=2013&amp;pages=214-218&amp;pmid=23770567&amp;doi=10.1038/nature12213&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Martincorena I, et al.  Universal patterns of selection in cancer and somatic tissues. Cell. 2017;171:1029–1041. doi: 10.1016/j.cell.2017.09.042.</cite> [<a href="https://doi.org/10.1016/j.cell.2017.09.042" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5720395/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29056346/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Universal%20patterns%20of%20selection%20in%20cancer%20and%20somatic%20tissues&amp;author=I%20Martincorena&amp;volume=171&amp;publication_year=2017&amp;pages=1029-1041&amp;pmid=29056346&amp;doi=10.1016/j.cell.2017.09.042&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Mularoni L, Sabarinathan R, Deu-Pons J, Gonzalez-Perez A, López-Bigas N. OncodriveFML: a general framework to identify coding and non-coding regions with cancer driver mutations. Genome Biol. 2016;17:128. doi: 10.1186/s13059-016-0994-0.</cite> [<a href="https://doi.org/10.1186/s13059-016-0994-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4910259/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27311963/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol.&amp;title=OncodriveFML:%20a%20general%20framework%20to%20identify%20coding%20and%20non-coding%20regions%20with%20cancer%20driver%20mutations&amp;author=L%20Mularoni&amp;author=R%20Sabarinathan&amp;author=J%20Deu-Pons&amp;author=A%20Gonzalez-Perez&amp;author=N%20L%C3%B3pez-Bigas&amp;volume=17&amp;publication_year=2016&amp;pages=128&amp;pmid=27311963&amp;doi=10.1186/s13059-016-0994-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Lu X, Thomas RK, Peifer M. CGARS: cancer genome analysis by rank sums. Bioinformatics. 2014;30:1295–1296. doi: 10.1093/bioinformatics/btu011.</cite> [<a href="https://doi.org/10.1093/bioinformatics/btu011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24413525/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=CGARS:%20cancer%20genome%20analysis%20by%20rank%20sums&amp;author=X%20Lu&amp;author=RK%20Thomas&amp;author=M%20Peifer&amp;volume=30&amp;publication_year=2014&amp;pages=1295-1296&amp;pmid=24413525&amp;doi=10.1093/bioinformatics/btu011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Leroy B, et al.  Analysis of TP53 mutation status in human cancer cell lines: a reassessment. Hum. Mutat. 2014;35:756–765. doi: 10.1002/humu.22556.</cite> [<a href="https://doi.org/10.1002/humu.22556" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4451114/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24700732/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Hum.%20Mutat.&amp;title=Analysis%20of%20TP53%20mutation%20status%20in%20human%20cancer%20cell%20lines:%20a%20reassessment&amp;author=B%20Leroy&amp;volume=35&amp;publication_year=2014&amp;pages=756-765&amp;pmid=24700732&amp;doi=10.1002/humu.22556&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title" data-anchor-id="_ad93_"><a id="_ad93_-anchor" data-anchor-id="_ad93_" class="usa-anchor"></a>Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/10972747/bin/41586_2024_7177_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link"><span>Supplementary Information</span></a><sup> (16.2MB, pdf) </sup><p>Clinical course and tumour phylogeny determined for 65 patients with SCLC.</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/10972747/bin/41586_2024_7177_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (2.4MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/10972747/bin/41586_2024_7177_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link"><span>Supplementary Fig. 1</span></a><sup> (1.7MB, pdf) </sup><p>Uncropped immunoblots from Extended Data Fig. 10a.</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/10972747/bin/41586_2024_7177_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link"><span>Supplementary Tables</span></a><sup> (24.2MB, xlsx) </sup><p>Supplementary Tables 1–12.</p>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The raw sequencing data are deposited in the European Genome-Phenome Archive under accession no. <a href="https://ega-archive.org/studies/EGAS50000000169" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EGAS50000000169</a>. Supporting data are provided as Supplementary Tables.</p>
<p>Computational approaches applied in this manuscript are described in <a href="#Sec9" class="usa-link">Methods</a> and were previously published<sup><a href="#CR5" class="usa-link" aria-describedby="CR5" aria-expanded="false">5</a>,<a href="#CR6" class="usa-link" aria-describedby="CR6" aria-expanded="false">6</a>,<a href="#CR15" class="usa-link" aria-describedby="CR15" aria-expanded="false">15</a>,<a href="#CR42" class="usa-link" aria-describedby="CR42" aria-expanded="false">42</a>,<a href="#CR43" class="usa-link" aria-describedby="CR43" aria-expanded="false">43</a></sup>. Our genome data-processing workflow ‘peiflyne’, which we use to perform duplicate masking, mutation calling, mutation filtering and annotation, as well as the algorithm used to identify significant mutational drivers, are available at <a href="http://www.uni-koeln.de/med-fak/peiflyne/peiflyne.tgz" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.uni-koeln.de/med-fak/peiflyne/peiflyne.tgz</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close">
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a href="https://doi.org/10.1038/s41586-024-07177-7" class="usa-button usa-button--outline width-24 font-xs usa-link--external padding-left-0 padding-right-0" target="_blank" rel="noreferrer noopener" data-ga-category="actions" data-ga-action="click" data-ga-label="publisher_link_desktop">
                         <span class="height-3 display-inline-flex flex-align-center">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a href="pdf/41586_2024_Article_7177.pdf" class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1" data-ga-category="actions" data-ga-action="click" data-ga-label="pdf_download_desktop">
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden="">
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (12.0&nbsp;MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1" aria-label="Open dialog with citation text in different styles" data-ga-category="actions" data-ga-action="open" data-ga-label="cite_desktop" data-all-citations-url="/resources/citations/10972747/" data-citation-style="nlm" data-download-format-link="/resources/citations/10972747/export/">
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden="">
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty" aria-label="Save article in MyNCBI collections." data-ga-category="actions" data-ga-action="click" data-ga-label="collections_button_desktop" data-collections-open-dialog-enabled="false" data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC10972747%2F%23open-collections-dialog" data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden="">
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden="">
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button type="button" class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none" aria-label="Show article permalink" aria-expanded="false" aria-haspopup="true" data-ga-category="actions" data-ga-action="open" data-ga-label="permalink_desktop">
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden="">
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden="">
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC10972747/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link" aria-expanded="false">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC10972747/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple="">
                    <h3 class="usa-accordion__heading">
                        <button type="button" class="usa-accordion__button" aria-expanded="false" aria-controls="resources-similar-articles" data-ga-category="resources_accordion" data-ga-action="open_similar_articles" data-ga-label="/articles/PMC10972747/" data-action-open="open_similar_articles" data-action-close="close_similar_articles">
                            Similar articles
                        </button>
                    </h3>
                    <div id="resources-similar-articles" class="usa-accordion__content usa-prose" data-source-url="/resources/similar-article-links/38480884/" hidden="">
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button type="button" class="usa-accordion__button" aria-expanded="false" aria-controls="resources-cited-by-other-articles" data-ga-category="resources_accordion" data-ga-action="open_cited_by" data-ga-label="/articles/PMC10972747/" data-action-open="open_cited_by" data-action-close="close_cited_by">
                             Cited by other articles
                        </button>
                    </h3>
                    <div id="resources-cited-by-other-articles" class="usa-accordion__content usa-prose" data-source-url="/resources/cited-by-links/38480884/" hidden="">
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button type="button" class="usa-accordion__button" aria-expanded="false" aria-controls="resources-links-to-ncbi-databases" data-ga-category="resources_accordion" data-ga-action="open_NCBI_links" data-ga-label="/articles/PMC10972747/" data-action-open="open_NCBI_links" data-action-close="close_NCBI_link">
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div id="resources-links-to-ncbi-databases" class="usa-accordion__content usa-prose" data-source-url="/resources/db-links/10972747/" hidden="">
                        </div>
                    
                    
                </div>
            
        </section>


        <section class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4" data-title-text="On this page" data-title-heading-level="h2" data-scroll-offset="0" data-root-margin="-10% 0px -80% 0px" data-main-content-selector="main" data-threshold="1"><nav aria-label="On this page" class="usa-in-page-nav__nav"><h2 class="usa-in-page-nav__heading" tabindex="0">On this page</h2><ul class="usa-in-page-nav__list"><li class="usa-in-page-nav__item"><a data-anchor-id="Abs1" href="#Abs1" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Abstract">Abstract</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="Sec1" href="#Sec1" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Main">Main</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="Sec2" href="#Sec2" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Tumour specimens and clinical data">Tumour specimens and clinical data</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="Sec3" href="#Sec3" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Tumour phylogenies of metastatic SCLC">Tumour phylogenies of metastatic SCLC</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="Sec4" href="#Sec4" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Mutation signatures of clonal diversity">Mutation signatures of clonal diversity</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="Sec5" href="#Sec5" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Tumour evolution under immunotherapy">Tumour evolution under immunotherapy</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="Sec6" href="#Sec6" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Clonality of central genome alterations">Clonality of central genome alterations</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="Sec7" href="#Sec7" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Impact of mutations on drug sensitivity">Impact of mutations on drug sensitivity</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="Sec8" href="#Sec8" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Discussion">Discussion</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="Sec9" href="#Sec9" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Methods">Methods</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="Sec28" href="#Sec28" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Online content">Online content</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="Sec30" href="#Sec30" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Supplementary information">Supplementary information</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="ack1" href="#ack1" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Acknowledgements">Acknowledgements</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="notes1" href="#notes1" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Author contributions">Author contributions</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="notes2" href="#notes2" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Peer review">Peer review</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="notes3" href="#notes3" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Funding">Funding</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="notes4" href="#notes4" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Data availability">Data availability</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="notes5" href="#notes5" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Code availability">Code availability</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="FPar2" href="#FPar2" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Competing interests">Competing interests</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="fn-group1" href="#fn-group1" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Footnotes">Footnotes</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="_ci93_" href="#_ci93_" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Contributor Information">Contributor Information</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="sec32" href="#sec32" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Extended data">Extended data</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="sec33" href="#sec33" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Supplementary information">Supplementary information</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="Bib1" href="#Bib1" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="References">References</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="_ad93_" href="#_ad93_" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Associated Data">Associated Data</a></li></ul></nav></section>
    </section>
</div>


        

<div class="overlay citation-dialog-overlay" role="dialog" aria-label="Citation Dialog">
    <div class="dialog citation-dialog" aria-hidden="true" role="document">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto" tabindex="1" data-pinger-ignore="true">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus" data-ga-category="save_share" data-ga-action="cite" data-ga-label="copy" tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a href="#" role="button" class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button" data-ga-category="save_share" data-ga-action="cite" data-ga-label="download" title="Download a file for external citation management software" tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4">
    
      <option data-style-url-name="ama" value="AMA">
        AMA
      </option>
    
      <option data-style-url-name="apa" value="APA">
        APA
      </option>
    
      <option data-style-url-name="mla" value="MLA">
        MLA
      </option>
    
      <option data-style-url-name="nlm" value="NLM" selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
<div class="dots-loading-indicator citation-loading-indicator">
        <div class="dot dot-1"></div>
        <div class="dot dot-2"></div>
        <div class="dot dot-3"></div>
      </div></div>
    </div>
</div>

        <div class="overlay collections-dialog-overlay" role="dialog">
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true" role="document">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form" class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors" data-existing-collections-url="/list-existing-collections/" data-add-to-existing-collection-url="/add-to-existing-collection/" data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/" data-myncbi-max-collection-name-length="100" data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="OpV1Q48vi4yEmryD4IkO7sj8zBUqgf5MX19qtWoQlcMJl7Fpy5N07nSLIzyrLrst">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio" id="collections-action-dialog-new" class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0" name="collections" value="new" data-ga-category="collections_button" data-ga-action="click" data-ga-label="collections_radio_new">
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio" id="collections-action-dialog-existing" class="usa-radio__input usa-radio__input--tile collections-existing" name="collections" value="existing" checked="true" data-ga-category="collections_button" data-ga-action="click" data-ga-label="collections_radio_existing">
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input type="text" name="add-to-new-collection" id="collections-action-dialog-add-to-new" class="usa-input collections-action-add-to-new" pattern="[^&quot;&amp;=<>/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=<>/" maxlength="" data-ga-category="collections_button" data-ga-action="create_collection" data-ga-label="non_favorties_collection" required="">
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing" class="usa-select collections-action-add-to-existing" data-ga-category="collections_button" data-ga-action="select_collection" data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit" type="submit" data-loading-label="Adding..." data-pinger-ignore="" data-ga-category="collections_button" data-ga-action="click" data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel" aria-label="Close 'Add to Collections' panel" ref="linksrc=close_collections_panel" data-ga-category="collections_button" data-ga-action="click" data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    

<footer class="ncbi-footer ncbi-dark-background ">
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi" class="ncbi-footer__social-icon ncbi-footer__social-icon--gray" target="_blank" rel="noreferrer noopener">
                        <svg width="40" height="40" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm" class="ncbi-footer__social-icon ncbi-footer__social-icon--gray" target="_blank" rel="noreferrer noopener">
                        <svg width="16" height="29" focusable="false" aria-hidden="true" viewBox="0 0 16 29" fill="none" xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm" class="ncbi-footer__social-icon ncbi-footer__social-icon--gray" target="_blank" rel="noreferrer noopener">
                        <svg width="25" height="23" viewBox="0 0 26 24" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi" class="ncbi-footer__social-icon ncbi-footer__social-icon--gray" target="_blank" rel="noreferrer noopener">
                        <svg width="28" height="27" viewBox="0 0 28 28" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/" class="ncbi-footer__social-icon ncbi-footer__social-icon--gray" target="_blank" rel="noreferrer noopener">
                        <svg width="26" height="26" viewBox="0 0 27 27" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer" class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container" data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href="https://twitter.com/nlm_nih" class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32" height="32" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href="https://www.facebook.com/nationallibraryofmedicine" class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13" height="24" viewBox="0 0 13 24" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href="https://www.youtube.com/user/NLMNIH" class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21" height="15" viewBox="0 0 21 15" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z" fill="white"></path>
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external" href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323" rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br> 8600 Rockville Pike<br> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link">
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link">
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target="_blank">
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/?pagename=cloudpmc-viewer%3Apmc%3Aarticle%3Ajournal" class="usa-link  usa-link--alt ncbi-footer__link" data-pinger-pagename-param="true">
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link">
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link">
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link">
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link">
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target="_blank">
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target="_blank">
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>



        
        
    
  <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script><div id="ZN_dikYWqsjiUWN0Q5"></div>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
     <script src="https://code.jquery.com/jquery-3.5.0.min.js" integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ=" crossorigin="anonymous">
    </script>
    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-9bea7450.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-722d91a2.js"></script>
    
    

    

<div class="fake-body-scroll"></div><div class="fake-body-scroll"></div><script id="_fed_an_ua_tag" text="" charset="" type="text/javascript" src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js?agency=HHS&amp;subagency=NCBI%20-%20ncbi.nlm.nih.gov&amp;sitetopic=NCBI%20Pinger%200.39.3&amp;siteplatform=NCBI%20Pinger%200.39.3"></script><script type="text/javascript" src="https://zndikYWqsjiUWN0Q5-nlmenterprise.siteintercept.qualtrics.com/SIE/?Q_ZID=ZN_dikYWqsjiUWN0Q5"></script><script src="https://siteintercept.qualtrics.com/dxjsmodule/CoreModule.js?Q_CLIENTVERSION=2.20.0&amp;Q_CLIENTTYPE=web&amp;Q_BRANDID=nlmenterprise" defer=""></script><script src="https://siteintercept.qualtrics.com/dxjsmodule/FeedbackButtonModule.js?Q_CLIENTVERSION=2.20.0&amp;Q_CLIENTTYPE=web&amp;Q_BRANDID=nlmenterprise" defer=""></script><style type="text/css">.QSIFeedbackButton div,.QSIFeedbackButton dl,.QSIFeedbackButton dt,.QSIFeedbackButton dd,.QSIFeedbackButton ul,.QSIFeedbackButton ol,.QSIFeedbackButton li,.QSIFeedbackButton h1,.QSIFeedbackButton h2,.QSIFeedbackButton h3,.QSIFeedbackButton h4,.QSIFeedbackButton h5,.QSIFeedbackButton h6,.QSIFeedbackButton span,.QSIFeedbackButton pre,.QSIFeedbackButton form,.QSIFeedbackButton fieldset,.QSIFeedbackButton textarea,.QSIFeedbackButton p,.QSIFeedbackButton blockquote,.QSIFeedbackButton tr,.QSIFeedbackButton th,.QSIFeedbackButton td{ margin: 0; padding: 0;background-color: transparent; border: 0; font-size: 12px; line-height: normal; vertical-align:baseline; box-shadow: none; }.QSIFeedbackButton img{ height: auto; width: auto; margin: 0; padding: 0 }.QSIFeedbackButton ul,.QSIFeedbackButton ol{ margin: 12px 0; padding-left: 40px; }.QSIFeedbackButton ul li{ list-style-type: disc; }.QSIFeedbackButton ol li{ list-style-type: decimal; }.QSIFeedbackButton .scrollable{ -webkit-overflow-scrolling: touch; }.QSIFeedbackButton table{ border-collapse: collapse; border-spacing: 0; }.QSIFeedbackButton table td{ padding: 2px; }.QSIFeedbackButton iframe{ max-height: none; }.QSIFeedbackButton *{ box-sizing: content-box; }</style><div class="QSIFeedbackButton" style="position: fixed; visibility: hidden; inset: 0px; display: flex; flex-direction: column; justify-content: flex-end; align-items: flex-start; margin: 0px; padding: 0px; z-index: 2000000000;"><button role="button" style="position: fixed; visibility: visible; cursor: pointer; border: medium; background-color: transparent; padding: 0px; margin: 0px; bottom: 97px; right: -1px; width: 52px;" aria-label="Tell us what you think!" id="QSIFeedbackButton-btn"><img src="https://iad1.qualtrics.com/WRQualtricsSiteIntercept/Graphic.php?IM=IM_3xWqwNkIHc7cA0S" alt="Tell us what you think!" style="max-width: 110px; max-height: 110px; display: block;"></button></div></body></html>