<html lang="en"><head>

        <meta charset="UTF-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="HandheldFriendly" content="True">
        <meta name="MobileOptimized" content="320">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-70b9163a.css">
<script type="text/javascript" async="" src="https://www.googletagmanager.com/gtag/js?id=G-DP2X732JSX&amp;l=pingerDataLayer&amp;cx=c&amp;gtm=45He4bk0v847342615za200"></script><script async="" src="https://www.googletagmanager.com/gtm.js?id=GTM-PC9B6M3&amp;l=pingerDataLayer" id="pingerInjectedGTM"></script><script type="module" crossorigin="" src="/static/assets/base_style-ec2bc71e.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css">
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css">
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon" sizes="180x180" href="/static/img/favicons/apple-touch-icon.png">
            <link rel="icon" type="image/png" sizes="48x48" href="/static/img/favicons/favicon-48x48.png">
            <link rel="icon" type="image/png" sizes="32x32" href="/static/img/favicons/favicon-32x32.png">
            <link rel="icon" type="image/png" sizes="16x16" href="/static/img/favicons/favicon-16x16.png">
            <link rel="manifest" href="/static/img/favicons/site.webmanifest">
            <link rel="mask-icon" href="/static/img/favicons/safari-pinned-tab.svg" color="#0071bc">
            <meta name="msapplication-config" content="/static/img/favicons/browserconfig.xml">
            <meta name="theme-color" content="#ffffff">
        

        <title>
            In‐depth proteomic analysis reveals unique subtype‐specific signatures in human small‐cell lung cancer - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer">
<meta name="ncbi_db" content="pmc">
<meta name="ncbi_phid" content="76C1A9D27478E91306A9D20043AEAB9C.m_1">
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article">
  
    <link rel="preconnect" href="https://www.google-analytics.com">

    
        <link rel="dns-prefetch" href="https://cdn.ncbi.nlm.nih.gov">
    

    <link rel="preconnect" href="https://code.jquery.com">
    <meta name="ncbi_domain" content="clintransmed">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC9506422/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Clinical and Translational Medicine">
<meta name="citation_title" content="In‐depth proteomic analysis reveals unique subtype‐specific signatures in human small‐cell lung cancer">
<meta name="citation_author" content="Beáta Szeitz">
<meta name="citation_author_institution" content="Division of Oncology, Department of Internal Medicine and Oncology, Semmelweis University, Budapest, Hungary">
<meta name="citation_author" content="Zsolt Megyesfalvi">
<meta name="citation_author_institution" content="National Korányi Institute of Pulmonology, Budapest, Hungary">
<meta name="citation_author_institution" content="Department of Thoracic Surgery, Medical University of Vienna, Vienna, Austria">
<meta name="citation_author_institution" content="Department of Thoracic Surgery, National Institute of Oncology, Semmelweis University, Budapest, Hungary">
<meta name="citation_author" content="Nicole Woldmar">
<meta name="citation_author_institution" content="Division of Clinical Protein Science, &amp; Imaging, Department of Clinical Sciences (Lund) and Department of Biomedical Engineering, Lund University, Lund, Sweden">
<meta name="citation_author_institution" content="Laboratory of Molecular Biology and Proteomics of Blood/LADETEC, Institute of Chemistry, Federal University of Rio de Janeiro, Rio de Janeiro, Brazil">
<meta name="citation_author" content="Zsuzsanna Valkó">
<meta name="citation_author_institution" content="National Korányi Institute of Pulmonology, Budapest, Hungary">
<meta name="citation_author_institution" content="Department of Thoracic Surgery, Medical University of Vienna, Vienna, Austria">
<meta name="citation_author" content="Anna Schwendenwein">
<meta name="citation_author_institution" content="Department of Thoracic Surgery, Medical University of Vienna, Vienna, Austria">
<meta name="citation_author" content="Nándor Bárány">
<meta name="citation_author_institution" content="National Korányi Institute of Pulmonology, Budapest, Hungary">
<meta name="citation_author_institution" content="Department of Thoracic Surgery, Medical University of Vienna, Vienna, Austria">
<meta name="citation_author_institution" content="First Department of Pathology and Experimental Cancer Research, Semmelweis University, Budapest, Hungary">
<meta name="citation_author" content="Sándor Paku">
<meta name="citation_author_institution" content="First Department of Pathology and Experimental Cancer Research, Semmelweis University, Budapest, Hungary">
<meta name="citation_author" content="Viktória László">
<meta name="citation_author_institution" content="National Korányi Institute of Pulmonology, Budapest, Hungary">
<meta name="citation_author_institution" content="Department of Thoracic Surgery, Medical University of Vienna, Vienna, Austria">
<meta name="citation_author" content="Helga Kiss">
<meta name="citation_author_institution" content="Department of Thoracic Surgery, National Institute of Oncology, Semmelweis University, Budapest, Hungary">
<meta name="citation_author_institution" content="University of Pécs, Pécs, Hungary">
<meta name="citation_author" content="Edina Bugyik">
<meta name="citation_author_institution" content="National Korányi Institute of Pulmonology, Budapest, Hungary">
<meta name="citation_author_institution" content="First Department of Pathology and Experimental Cancer Research, Semmelweis University, Budapest, Hungary">
<meta name="citation_author" content="Christian Lang">
<meta name="citation_author_institution" content="Department of Thoracic Surgery, Medical University of Vienna, Vienna, Austria">
<meta name="citation_author" content="Attila Marcell Szász">
<meta name="citation_author_institution" content="National Korányi Institute of Pulmonology, Budapest, Hungary">
<meta name="citation_author_institution" content="Department of Bioinformatics, Semmelweis University, Budapest, Hungary">
<meta name="citation_author" content="Luciana Pizzatti">
<meta name="citation_author_institution" content="Laboratory of Molecular Biology and Proteomics of Blood/LADETEC, Institute of Chemistry, Federal University of Rio de Janeiro, Rio de Janeiro, Brazil">
<meta name="citation_author" content="Krisztina Bogos">
<meta name="citation_author_institution" content="National Korányi Institute of Pulmonology, Budapest, Hungary">
<meta name="citation_author" content="Mir Alireza Hoda">
<meta name="citation_author_institution" content="Department of Thoracic Surgery, Medical University of Vienna, Vienna, Austria">
<meta name="citation_author" content="Konrad Hoetzenecker">
<meta name="citation_author_institution" content="Department of Thoracic Surgery, Medical University of Vienna, Vienna, Austria">
<meta name="citation_author" content="György Marko‐Varga">
<meta name="citation_author_institution" content="Division of Clinical Protein Science, &amp; Imaging, Department of Clinical Sciences (Lund) and Department of Biomedical Engineering, Lund University, Lund, Sweden">
<meta name="citation_author" content="Peter Horvatovich">
<meta name="citation_author_institution" content="Department of Analytical Biochemistry, Groningen Research Institute of Pharmacy, University of Groningen, Groningen, The Netherlands">
<meta name="citation_author" content="Balázs Döme">
<meta name="citation_author_institution" content="National Korányi Institute of Pulmonology, Budapest, Hungary">
<meta name="citation_author_institution" content="Department of Thoracic Surgery, Medical University of Vienna, Vienna, Austria">
<meta name="citation_author_institution" content="Department of Thoracic Surgery, National Institute of Oncology, Semmelweis University, Budapest, Hungary">
<meta name="citation_author_institution" content="Department of Translational Medicine, Lund University, Lund, Sweden">
<meta name="citation_author" content="Karin Schelch">
<meta name="citation_author_institution" content="Center for Cancer Research, Medical University of Vienna, Vienna, Austria">
<meta name="citation_author" content="Melinda Rezeli">
<meta name="citation_author_institution" content="Division of Clinical Protein Science, &amp; Imaging, Department of Clinical Sciences (Lund) and Department of Biomedical Engineering, Lund University, Lund, Sweden">
<meta name="citation_publication_date" content="2022 Sep 23">
<meta name="citation_volume" content="12">
<meta name="citation_issue" content="9">
<meta name="citation_firstpage" content="e1060">
<meta name="citation_doi" content="10.1002/ctm2.1060">
<meta name="citation_pmid" content="36149789">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC9506422/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC9506422/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC9506422/pdf/CTM2-12-e1060.pdf">
<meta name="description" content="Small‐cell lung cancer (SCLC) molecular subtypes have been primarily characterized based on the expression pattern of the following key transcription regulators: ASCL1 (SCLC‐A), NEUROD1 (SCLC‐N), POU2F3 (SCLC‐P) and YAP1 (SCLC‐Y). Here, we ...">
<meta name="og:title" content="In‐depth proteomic analysis reveals unique subtype‐specific signatures in human small‐cell lung cancer">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Small‐cell lung cancer (SCLC) molecular subtypes have been primarily characterized based on the expression pattern of the following key transcription regulators: ASCL1 (SCLC‐A), NEUROD1 (SCLC‐N), POU2F3 (SCLC‐P) and YAP1 (SCLC‐Y). Here, we ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC9506422/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    <script type="text/javascript" src="https://www.googletagmanager.com/gtag/js?id=G-CSLL4ZEK4L"></script><script charset="utf-8" src="https://siteintercept.qualtrics.com/dxjsmodule/10.07268bfc859327bf20d5.chunk.js?Q_CLIENTVERSION=2.20.0&amp;Q_CLIENTTYPE=web&amp;Q_BRANDID=pmc.ncbi.nlm.nih.gov"></script><script charset="utf-8" src="https://siteintercept.qualtrics.com/dxjsmodule/5.d83df5c454102e31d5df.chunk.js?Q_CLIENTVERSION=2.20.0&amp;Q_CLIENTTYPE=web&amp;Q_BRANDID=nlmenterprise"></script><script charset="utf-8" src="https://siteintercept.qualtrics.com/dxjsmodule/1.4a290fea10f6e9b6f375.chunk.js?Q_CLIENTVERSION=2.20.0&amp;Q_CLIENTTYPE=web&amp;Q_BRANDID=nlmenterprise"></script></head>
    <body>
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government">
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true" class="usa-banner__header-flag" src="/static/img/us_flag.svg" alt="">
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button type="button" class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           " aria-expanded="false" aria-controls="gov-banner-default" data-testid="storybook-django-banner">
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content" id="gov-banner-default" hidden="">
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img" src="/static/img/icon-dot-gov.svg" alt="" aria-hidden="true">
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br>
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img" src="/static/img/icon-https.svg" alt="" aria-hidden="true">

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br>
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg" width="52" height="64" viewBox="0 0 52 64" class="usa-banner__lock-image" role="graphics-symbol" aria-labelledby="banner-lock-description" focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000" fill-rule="evenodd" d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z"></path>
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header="">
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="/">
                    <img alt="
                                  PMC home page
                              " class="ncbi-header__logo-image" src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg">
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button type="button" class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           " aria-label="Show menu" data-testid="navMenuButton">
    
        

        
                <svg aria-hidden="true" class="ncbi-hamburger-icon" fill="none" focusable="false" height="21" viewBox="0 0 31 21" width="31" xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd" d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z" fill="#F1F1F1" fill-rule="evenodd"></path>
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button type="button" class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           " aria-expanded="false" aria-controls="search-field-desktop-navigation" aria-label="Show search overlay" data-testid="toggleSearchPanelButton" data-toggle-search-panel-button="" data-ga-category="header" data-ga-action="NCBI" data-ga-label="header_search_open">
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true">
        
        <use xlink:href="/static/img/sprite.svg#search"></use>
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button type="button" class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           " aria-expanded="false" aria-controls="login-dropdown-menu" aria-label="Show login menu" data-testid="toggleLoginMenuDropdown" data-desktop-login-button="">
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true">
        
        <use xlink:href="/static/img/sprite.svg#person"></use>
    </svg>



                                <span data-login-dropdown-text="">Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow="">
        
        <use xlink:href="/static/img/sprite.svg#expand_less"></use>
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow="">
        
        <use xlink:href="/static/img/sprite.svg#expand_more"></use>
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu" id="login-dropdown-menu" data-desktop-login-menu-dropdown="" hidden="">
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  ">
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  ">
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  ">
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button type="button" class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           " data-testid="desktopLogoutButton" data-desktop-logout-button="">
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop" data-testid="searchPanel" data-header-search-panel="" hidden="">
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/" aria-describedby="search-field-desktop-navigation-help-text" autocomplete="off" class="usa-search usa-search--big ncbi-search-panel__form" data-testid="form" method="GET" role="search">
                    <label class="usa-sr-only" data-testid="label" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input" data-testid="textInput" id="search-field-desktop-navigation" name="term" placeholder="Search NCBI" type="search" value="">
                    



















    
        <button type="submit" class="usa-button
           

           
               
               
               
               
            

           
           
           
           " data-testid="button" data-ga-category="header" data-ga-action="NCBI" data-ga-label="header_search_button">
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button type="button" class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           " aria-label="Close navigation menu" data-testid="navCloseButton">
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close">
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6" role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input" id="search-field-mobile-navigation" type="search" placeholder="Search NCBI" name="search">

                



















    
        <button type="submit" class="usa-button
           

           
               
               
               
               
            

           
           
           
           " data-ga-category="header" data-ga-action="NCBI" data-ga-label="header_search_button">
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg==" class="usa-search__submit-icon" alt="Search">
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop" data-mobile-login-menu="" hidden="">
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email" data-mobile-login-email-text=""></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  ">
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  ">
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  ">
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button type="button" class="usa-button ncbi-nav__mobile-login-button ncbi--hide-at-desktop" data-testid="mobileLoginButton" data-mobile-login-button="">Log in</button>
        


    </nav>
</header>

    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button type="button" class="usa-button usa-button--unstyled pmc-header__search__button" aria-label="Open search" data-ga-category="search" data-ga-action="PMC" data-ga-label="pmc_search_panel_mobile">
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column pmc-header__search__form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search" type="search" name="term" data-autocomplete-url="/search/autocomplete/" aria-controls="autoComplete_list_1" aria-autocomplete="both" role="combobox" aria-owns="autoComplete_list_1" aria-haspopup="true" aria-expanded="false"><ul id="autoComplete_list_1" role="listbox" hidden="" aria-label="Suggestions"></ul>
</span>
<button class="usa-button" type="submit" formaction="https://www.ncbi.nlm.nih.gov/pmc/" data-ga-category="search" data-ga-action="PMC" data-ga-label="PMC_search_button">
<span class="usa-search__submit-text">Search in PMC</span>
<img src="/static/img/usa-icons-bg/search--white.svg" class="usa-search__submit-icon" alt="Search">
</button>
</form>
            <ul class="pmc-header__search__menu">
                <li>
                    <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                        Advanced Search
                    </a>
                </li>
                <li>
                    
                        <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                            Journal List
                        </a>
                    
                </li>
                <li>
                    
                        <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link" data-ga-label="user guide">
                            User Guide
                        </a>
                    
                </li>
            </ul>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="9506422">

    

   



<div class="grid-container pmc-actions-bar is-not-intersecting" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button type="button" class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex" aria-label="Open resources" data-extra-class="is-visible-resources" data-ga-category="resources_accordion" data-ga-action="click" data-ga-label="mobile_icon">
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a href="https://doi.org/10.1002/ctm2.1060" class="usa-link display-flex" role="button" target="_blank" rel="noreferrer noopener" aria-label="View on publisher site" data-ga-category="actions" data-ga-action="click" data-ga-label="publisher_link_mobile">
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a href="pdf/CTM2-12-e1060.pdf" class="usa-link display-flex" role="button" aria-label="Download PDF" data-ga-category="actions" data-ga-action="click" data-ga-label="pdf_download_mobile">
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button class="usa-button usa-button--unstyled collections-dialog-trigger collections-button display-flex collections-button-empty" aria-label="Save article in MyNCBI collections." data-ga-category="actions" data-ga-action="click" data-ga-label="collections_button_mobile" data-collections-open-dialog-enabled="false" data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC9506422%2F%23open-collections-dialog" data-in-collections="false">
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden="">
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden="">
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled citation-dialog-trigger display-flex" aria-label="Open dialog with citation text in different styles" data-ga-category="actions" data-ga-action="open" data-ga-label="cite_mobile" data-all-citations-url="/resources/citations/9506422/" data-citation-style="nlm" data-download-format-link="/resources/citations/9506422/export/">
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden="">
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex">
                         <button type="button" class="usa-button usa-button--unstyled display-flex" aria-label="Show article permalink" aria-expanded="false" aria-haspopup="true" data-ga-category="actions" data-ga-action="open" data-ga-label="permalink_mobile">
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden="">
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC9506422/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link" aria-expanded="false">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button type="button" class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex" aria-label="Open article navigation" data-extra-class="is-visible-in-page" data-ga-category="actions" data-ga-action="open" data-ga-label="article_nav_mobile">
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main id="main-content" class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0">

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-clintransmed.jpg" alt="Clinical and Translational Medicine logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Clinical and Translational Medicine" title="Link to Clinical and Translational Medicine" shape="default" href="https://onlinelibrary.wiley.com/journal/20011326" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu" aria-expanded="false">Clin Transl Med</button></div>. 2022 Sep 23;12(9):e1060. doi: <a href="https://doi.org/10.1002/ctm2.1060" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/ctm2.1060</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Clin%20Transl%20Med%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Clin%20Transl%20Med%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Clin%20Transl%20Med%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Clin%20Transl%20Med%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>In‐depth proteomic analysis reveals unique subtype‐specific signatures in human small‐cell lung cancer</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Szeitz%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id1" aria-expanded="false"><span class="name western">Beáta Szeitz</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Beáta Szeitz</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Division of Oncology, Department of Internal Medicine and Oncology, Semmelweis University, Budapest, Hungary</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Szeitz%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Beáta Szeitz</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Megyesfalvi%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id2" aria-expanded="false"><span class="name western">Zsolt Megyesfalvi</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Zsolt Megyesfalvi</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>National Korányi Institute of Pulmonology, Budapest, Hungary</div>
<div class="p">
<sup>
<sup>3</sup>
</sup>Department of Thoracic Surgery, Medical University of Vienna, Vienna, Austria</div>
<div class="p">
<sup>
<sup>4</sup>
</sup>Department of Thoracic Surgery, National Institute of Oncology, Semmelweis University, Budapest, Hungary</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Megyesfalvi%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zsolt Megyesfalvi</span></a>
</div>
</div>
<sup>2,</sup><sup>3,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Woldmar%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id3" aria-expanded="false"><span class="name western">Nicole Woldmar</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Nicole Woldmar</span></h3>
<div class="p">
<sup>
<sup>5</sup>
</sup>Division of Clinical Protein Science, &amp; Imaging, Department of Clinical Sciences (Lund) and Department of Biomedical Engineering, Lund University, Lund, Sweden</div>
<div class="p">
<sup>
<sup>6</sup>
</sup>Laboratory of Molecular Biology and Proteomics of Blood/LADETEC, Institute of Chemistry, Federal University of Rio de Janeiro, Rio de Janeiro, Brazil</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Woldmar%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Nicole Woldmar</span></a>
</div>
</div>
<sup>5,</sup><sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Valk%C3%B3%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id4" aria-expanded="false"><span class="name western">Zsuzsanna Valkó</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Zsuzsanna Valkó</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>National Korányi Institute of Pulmonology, Budapest, Hungary</div>
<div class="p">
<sup>
<sup>3</sup>
</sup>Department of Thoracic Surgery, Medical University of Vienna, Vienna, Austria</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Valk%C3%B3%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zsuzsanna Valkó</span></a>
</div>
</div>
<sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schwendenwein%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id5" aria-expanded="false"><span class="name western">Anna Schwendenwein</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Anna Schwendenwein</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>Department of Thoracic Surgery, Medical University of Vienna, Vienna, Austria</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schwendenwein%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Anna Schwendenwein</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22B%C3%A1r%C3%A1ny%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id6" aria-expanded="false"><span class="name western">Nándor Bárány</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Nándor Bárány</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>National Korányi Institute of Pulmonology, Budapest, Hungary</div>
<div class="p">
<sup>
<sup>3</sup>
</sup>Department of Thoracic Surgery, Medical University of Vienna, Vienna, Austria</div>
<div class="p">
<sup>
<sup>7</sup>
</sup>First Department of Pathology and Experimental Cancer Research, Semmelweis University, Budapest, Hungary</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22B%C3%A1r%C3%A1ny%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Nándor Bárány</span></a>
</div>
</div>
<sup>2,</sup><sup>3,</sup><sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Paku%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id7" aria-expanded="false"><span class="name western">Sándor Paku</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Sándor Paku</span></h3>
<div class="p">
<sup>
<sup>7</sup>
</sup>First Department of Pathology and Experimental Cancer Research, Semmelweis University, Budapest, Hungary</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Paku%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sándor Paku</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22L%C3%A1szl%C3%B3%20V%22%5BAuthor%5D" class="usa-link" aria-describedby="id8" aria-expanded="false"><span class="name western">Viktória László</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Viktória László</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>National Korányi Institute of Pulmonology, Budapest, Hungary</div>
<div class="p">
<sup>
<sup>3</sup>
</sup>Department of Thoracic Surgery, Medical University of Vienna, Vienna, Austria</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22L%C3%A1szl%C3%B3%20V%22%5BAuthor%5D" class="usa-link"><span class="name western">Viktória László</span></a>
</div>
</div>
<sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kiss%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id9" aria-expanded="false"><span class="name western">Helga Kiss</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Helga Kiss</span></h3>
<div class="p">
<sup>
<sup>4</sup>
</sup>Department of Thoracic Surgery, National Institute of Oncology, Semmelweis University, Budapest, Hungary</div>
<div class="p">
<sup>
<sup>8</sup>
</sup>University of Pécs, Pécs, Hungary</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kiss%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Helga Kiss</span></a>
</div>
</div>
<sup>4,</sup><sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bugyik%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id10" aria-expanded="false"><span class="name western">Edina Bugyik</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Edina Bugyik</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>National Korányi Institute of Pulmonology, Budapest, Hungary</div>
<div class="p">
<sup>
<sup>7</sup>
</sup>First Department of Pathology and Experimental Cancer Research, Semmelweis University, Budapest, Hungary</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bugyik%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Edina Bugyik</span></a>
</div>
</div>
<sup>2,</sup><sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lang%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id11" aria-expanded="false"><span class="name western">Christian Lang</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Christian Lang</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>Department of Thoracic Surgery, Medical University of Vienna, Vienna, Austria</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lang%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Christian Lang</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sz%C3%A1sz%20AM%22%5BAuthor%5D" class="usa-link" aria-describedby="id12" aria-expanded="false"><span class="name western">Attila Marcell Szász</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Attila Marcell Szász</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>National Korányi Institute of Pulmonology, Budapest, Hungary</div>
<div class="p">
<sup>
<sup>9</sup>
</sup>Department of Bioinformatics, Semmelweis University, Budapest, Hungary</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sz%C3%A1sz%20AM%22%5BAuthor%5D" class="usa-link"><span class="name western">Attila Marcell Szász</span></a>
</div>
</div>
<sup>2,</sup><sup>9</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pizzatti%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id13" aria-expanded="false"><span class="name western">Luciana Pizzatti</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Luciana Pizzatti</span></h3>
<div class="p">
<sup>
<sup>6</sup>
</sup>Laboratory of Molecular Biology and Proteomics of Blood/LADETEC, Institute of Chemistry, Federal University of Rio de Janeiro, Rio de Janeiro, Brazil</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pizzatti%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Luciana Pizzatti</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bogos%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id14" aria-expanded="false"><span class="name western">Krisztina Bogos</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Krisztina Bogos</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>National Korányi Institute of Pulmonology, Budapest, Hungary</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bogos%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Krisztina Bogos</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hoda%20MA%22%5BAuthor%5D" class="usa-link" aria-describedby="id15" aria-expanded="false"><span class="name western">Mir Alireza Hoda</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">Mir Alireza Hoda</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>Department of Thoracic Surgery, Medical University of Vienna, Vienna, Austria</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hoda%20MA%22%5BAuthor%5D" class="usa-link"><span class="name western">Mir Alireza Hoda</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hoetzenecker%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id16" aria-expanded="false"><span class="name western">Konrad Hoetzenecker</span></a><div hidden="hidden" id="id16">
<h3><span class="name western">Konrad Hoetzenecker</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>Department of Thoracic Surgery, Medical University of Vienna, Vienna, Austria</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hoetzenecker%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Konrad Hoetzenecker</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Marko%E2%80%90Varga%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id17" aria-expanded="false"><span class="name western">György Marko‐Varga</span></a><div hidden="hidden" id="id17">
<h3><span class="name western">György Marko‐Varga</span></h3>
<div class="p">
<sup>
<sup>5</sup>
</sup>Division of Clinical Protein Science, &amp; Imaging, Department of Clinical Sciences (Lund) and Department of Biomedical Engineering, Lund University, Lund, Sweden</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Marko%E2%80%90Varga%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">György Marko‐Varga</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Horvatovich%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id18" aria-expanded="false"><span class="name western">Peter Horvatovich</span></a><div hidden="hidden" id="id18">
<h3><span class="name western">Peter Horvatovich</span></h3>
<div class="p">
<sup>
<sup>10</sup>
</sup>Department of Analytical Biochemistry, Groningen Research Institute of Pharmacy, University of Groningen, Groningen, The Netherlands</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Horvatovich%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Peter Horvatovich</span></a>
</div>
</div>
<sup>10</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22D%C3%B6me%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id19" aria-expanded="false"><span class="name western">Balázs Döme</span></a><div hidden="hidden" id="id19">
<h3><span class="name western">Balázs Döme</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>National Korányi Institute of Pulmonology, Budapest, Hungary</div>
<div class="p">
<sup>
<sup>3</sup>
</sup>Department of Thoracic Surgery, Medical University of Vienna, Vienna, Austria</div>
<div class="p">
<sup>
<sup>4</sup>
</sup>Department of Thoracic Surgery, National Institute of Oncology, Semmelweis University, Budapest, Hungary</div>
<div class="p">
<sup>
<sup>11</sup>
</sup>Department of Translational Medicine, Lund University, Lund, Sweden</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22D%C3%B6me%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Balázs Döme</span></a>
</div>
</div>
<sup>2,</sup><sup>3,</sup><sup>4,</sup><sup>11,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schelch%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id20" aria-expanded="false"><span class="name western">Karin Schelch</span></a><div hidden="hidden" id="id20">
<h3><span class="name western">Karin Schelch</span></h3>
<div class="p">
<sup>
<sup>12</sup>
</sup>Center for Cancer Research, Medical University of Vienna, Vienna, Austria</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schelch%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Karin Schelch</span></a>
</div>
</div>
<sup>12</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rezeli%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id21" aria-expanded="false"><span class="name western">Melinda Rezeli</span></a><div hidden="hidden" id="id21">
<h3><span class="name western">Melinda Rezeli</span></h3>
<div class="p">
<sup>
<sup>5</sup>
</sup>Division of Clinical Protein Science, &amp; Imaging, Department of Clinical Sciences (Lund) and Department of Biomedical Engineering, Lund University, Lund, Sweden</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rezeli%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Melinda Rezeli</span></a>
</div>
</div>
<sup>5,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="ctm21060-aff-0001">
<sup>
<sup>1</sup>
</sup>Division of Oncology, Department of Internal Medicine and Oncology, Semmelweis University, Budapest, Hungary</div>
<div id="ctm21060-aff-0002">
<sup>
<sup>2</sup>
</sup>National Korányi Institute of Pulmonology, Budapest, Hungary</div>
<div id="ctm21060-aff-0003">
<sup>
<sup>3</sup>
</sup>Department of Thoracic Surgery, Medical University of Vienna, Vienna, Austria</div>
<div id="ctm21060-aff-0004">
<sup>
<sup>4</sup>
</sup>Department of Thoracic Surgery, National Institute of Oncology, Semmelweis University, Budapest, Hungary</div>
<div id="ctm21060-aff-0005">
<sup>
<sup>5</sup>
</sup>Division of Clinical Protein Science, &amp; Imaging, Department of Clinical Sciences (Lund) and Department of Biomedical Engineering, Lund University, Lund, Sweden</div>
<div id="ctm21060-aff-0006">
<sup>
<sup>6</sup>
</sup>Laboratory of Molecular Biology and Proteomics of Blood/LADETEC, Institute of Chemistry, Federal University of Rio de Janeiro, Rio de Janeiro, Brazil</div>
<div id="ctm21060-aff-0007">
<sup>
<sup>7</sup>
</sup>First Department of Pathology and Experimental Cancer Research, Semmelweis University, Budapest, Hungary</div>
<div id="ctm21060-aff-0008">
<sup>
<sup>8</sup>
</sup>University of Pécs, Pécs, Hungary</div>
<div id="ctm21060-aff-0009">
<sup>
<sup>9</sup>
</sup>Department of Bioinformatics, Semmelweis University, Budapest, Hungary</div>
<div id="ctm21060-aff-0010">
<sup>
<sup>10</sup>
</sup>Department of Analytical Biochemistry, Groningen Research Institute of Pharmacy, University of Groningen, Groningen, The Netherlands</div>
<div id="ctm21060-aff-0011">
<sup>
<sup>11</sup>
</sup>Department of Translational Medicine, Lund University, Lund, Sweden</div>
<div id="ctm21060-aff-0012">
<sup>
<sup>12</sup>
</sup>Center for Cancer Research, Medical University of Vienna, Vienna, Austria</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence</strong>
, Balázs Döme, Department of Thoracic Surgery, Medical University of Vienna, Waehringer Guertel 18‐20, 1090 Vienna, Austria. Email: <span>balazs.dome@meduniwien.ac.at</span>
, Melinda Rezeli, Division of Clinical Protein Science &amp; Imaging, Department of Clinical Sciences (Lund) and Department of Biomedical Engineering, Lund University, Lund, Sweden. Email: <span>melinda.rezeli@bme.lth.se</span>

</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2022 Aug 31; Received 2022 May 26; Accepted 2022 Sep 6; Collection date 2022 Sep.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2022 The Authors. <em>Clinical and Translational Medicine</em> published by John Wiley &amp; Sons Australia, Ltd on behalf of Shanghai Institute of Clinical Bioinformatics.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC9506422&nbsp;&nbsp;PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/36149789/" class="usa-link">36149789</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2 data-anchor-id="abstract1"><a id="abstract1-anchor" data-anchor-id="abstract1" class="usa-anchor"></a>Abstract</h2>
<section id="ctm21060-sec-0010"><h3 class="pmc_sec_title">Background</h3>
<p>Small‐cell lung cancer (SCLC) molecular subtypes have been primarily characterized based on the expression pattern of the following key transcription regulators: <em>ASCL1</em> (SCLC‐A), <em>NEUROD1</em> (SCLC‐N), <em>POU2F3</em> (SCLC‐P) and <em>YAP1</em> (SCLC‐Y). Here, we investigated the proteomic landscape of these molecular subsets with the aim to identify novel subtype‐specific proteins of diagnostic and therapeutic relevance.</p></section><section id="ctm21060-sec-0020"><h3 class="pmc_sec_title">Methods</h3>
<p>Pellets and cell media of 26 human SCLC cell lines were subjected to label‐free shotgun proteomics for large‐scale protein identification and quantitation, followed by in‐depth bioinformatic analyses. Proteomic data were correlated with the cell lines’ phenotypic characteristics and with public transcriptomic data of SCLC cell lines and tissues.</p></section><section id="ctm21060-sec-0030"><h3 class="pmc_sec_title">Results</h3>
<p>Our quantitative proteomic data highlighted that four molecular subtypes are clearly distinguishable at the protein level. The cell lines exhibited diverse neuroendocrine and epithelial–mesenchymal characteristics that varied by subtype. A total of 367 proteins were identified in the cell pellet and 34 in the culture media that showed significant up‐ or downregulation in one subtype, including known druggable proteins and potential blood‐based markers. Pathway enrichment analysis and parallel investigation of transcriptomics from SCLC cell lines outlined unique signatures for each subtype, such as upregulated oxidative phosphorylation in SCLC‐A, DNA replication in SCLC‐N, neurotrophin signalling in SCLC‐P and epithelial–mesenchymal transition in SCLC‐Y. Importantly, we identified the <em>YAP1</em>‐driven subtype as the most distinct SCLC subgroup. Using sparse partial least squares discriminant analysis, we identified proteins that clearly distinguish four SCLC subtypes based on their expression pattern, including potential diagnostic markers for SCLC‐Y (e.g. GPX8, PKD2 and UFO).</p></section><section id="ctm21060-sec-0040"><h3 class="pmc_sec_title">Conclusions</h3>
<p>We report for the first time, the protein expression differences among SCLC subtypes. By shedding light on potential subtype‐specific therapeutic vulnerabilities and diagnostic biomarkers, our results may contribute to a better understanding of SCLC biology and the development of novel therapies.</p></section><section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> diagnostic biomarkers, molecular targets, proteomics, secretome, small‐cell lung cancer, subtype, transcriptomics</p></section></section><section class="abstract" id="ctm21060-abs-0001"><hr class="headless">
<ol id="ctm21060-list-0001" class="list" style="list-style-type:decimal">
<li><p>The first mass spectrometry–based proteomic analysis on small‐cell lung cancer (SCLC) cell lines, reporting unique proteomic profiles of its proposed molecular subtypes.</p></li>
<li><p>SCLC subgroups detected by proteomics are consistent with mRNA‐based subtypes.</p></li>
<li><p>Unique proteomic profiles of SCLC subtypes highlight potential subtype‐specific therapeutic vulnerabilities and diagnostic biomarkers.</p></li>
</ol>
<section class="bt xbox font-sm" id="boxed-text1"><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5517/9506422/bdbd73c11f71/CTM2-12-e1060-g006.jpg" loading="lazy" id="jats-graphic-1" height="92" width="180" alt="graphic file with name CTM2-12-e1060-g006.jpg"></p></section></section><section id="ctm21060-sec-0050"><h2 class="pmc_sec_title" data-anchor-id="ctm21060-sec-0050"><a id="ctm21060-sec-0050-anchor" data-anchor-id="ctm21060-sec-0050" class="usa-anchor"></a>1. INTRODUCTION</h2>
<p>Small‐cell lung cancer (SCLC) represents about 13%–15% of all lung cancers and with a 5‐year survival rate of less than 7%, it remains one of the most lethal forms of malignant diseases.<a href="#ctm21060-bib-0001" class="usa-link" aria-describedby="ctm21060-bib-0001" aria-expanded="false">
<sup>1</sup>
</a>, <a href="#ctm21060-bib-0002" class="usa-link" aria-describedby="ctm21060-bib-0002" aria-expanded="false">
<sup>2</sup>
</a>, <a href="#ctm21060-bib-0003" class="usa-link" aria-describedby="ctm21060-bib-0003" aria-expanded="false">
<sup>3</sup>
</a> It has a very aggressive course and is characterized by extensive chromosomal rearrangements, high mutation burden and almost universal inactivation of the tumour suppressor genes <em>TP53</em> and <em>RB1</em>.<a href="#ctm21060-bib-0001" class="usa-link" aria-describedby="ctm21060-bib-0001" aria-expanded="false">
<sup>1</sup>
</a>, <a href="#ctm21060-bib-0004" class="usa-link" aria-describedby="ctm21060-bib-0004" aria-expanded="false">
<sup>4</sup>
</a> Therefore, the vast majority of SCLC patients are diagnosed with extensive‐stage disease when surgery is not feasible, and the treatment options are mostly limited to cytotoxic chemotherapy and radiation.<a href="#ctm21060-bib-0004" class="usa-link" aria-describedby="ctm21060-bib-0004" aria-expanded="false">
<sup>4</sup>
</a> Importantly, targeted therapies for these patients have so far failed, and the success of immunotherapy in non‐SCLC has not been reflected in SCLC.<a href="#ctm21060-bib-0003" class="usa-link" aria-describedby="ctm21060-bib-0003" aria-expanded="false">
<sup>3</sup>
</a>, <a href="#ctm21060-bib-0005" class="usa-link" aria-describedby="ctm21060-bib-0005" aria-expanded="false">
<sup>5</sup>
</a>
</p>
<p>Although SCLC has been formerly considered a homogeneous disease with a single morphological type, recent advances in SCLC research have led to the development of subtype‐specific classifications primarily based on neuroendocrine (NE) features and unique molecular profiles.<a href="#ctm21060-bib-0003" class="usa-link" aria-describedby="ctm21060-bib-0003" aria-expanded="false">
<sup>3</sup>
</a>, <a href="#ctm21060-bib-0006" class="usa-link" aria-describedby="ctm21060-bib-0006" aria-expanded="false">
<sup>6</sup>
</a>, <a href="#ctm21060-bib-0007" class="usa-link" aria-describedby="ctm21060-bib-0007" aria-expanded="false">
<sup>7</sup>
</a> Accordingly, SCLC can be classified into NE‐high and NE‐low subtypes based on the expression pattern of key NE markers (i.e. <em>SYP</em>, <em>CHGA</em>, <em>NCAM1/CD56</em> and <em>GRP</em>).<a href="#ctm21060-bib-0001" class="usa-link" aria-describedby="ctm21060-bib-0001" aria-expanded="false">
<sup>1</sup>
</a>, <a href="#ctm21060-bib-0008" class="usa-link" aria-describedby="ctm21060-bib-0008" aria-expanded="false">
<sup>8</sup>
</a> To further elucidate the clinicopathological relevance of NE subtypes, Zhang et&nbsp;al. developed a numeric score to evaluate the degree of NE differentiation in SCLC.<a href="#ctm21060-bib-0007" class="usa-link" aria-describedby="ctm21060-bib-0007" aria-expanded="false">
<sup>7</sup>
</a> By using the top 50 genes most strongly associated with NE differentiation, they successfully separated NE high and low subsets of SCLC tumours and cell lines and showed that these subtypes had widely different growth properties and morphology.<a href="#ctm21060-bib-0007" class="usa-link" aria-describedby="ctm21060-bib-0007" aria-expanded="false">
<sup>7</sup>
</a> Of note, some SCLCs lack NE differentiation and are termed non‐NE tumours.<a href="#ctm21060-bib-0006" class="usa-link" aria-describedby="ctm21060-bib-0006" aria-expanded="false">
<sup>6</sup>
</a> Importantly, the NE differentiation also serves as a framework for the recently emerged molecular subtypes.<a href="#ctm21060-bib-0006" class="usa-link" aria-describedby="ctm21060-bib-0006" aria-expanded="false">
<sup>6</sup>
</a> These newly proposed molecular subgroups (SCLC‐A, SCLC‐N, SCLC‐P and SCLC‐Y) have been defined based on the expression pattern of transcription factors <em>ASCL1</em>, <em>NEUROD1</em>, <em>POU2F3</em> and <em>YAP1</em>, respectively.<a href="#ctm21060-bib-0004" class="usa-link" aria-describedby="ctm21060-bib-0004" aria-expanded="false">
<sup>4</sup>
</a>, <a href="#ctm21060-bib-0006" class="usa-link" aria-describedby="ctm21060-bib-0006" aria-expanded="false">
<sup>6</sup>
</a> SCLC‐P and SCLC‐Y show a non‐NE phenotype, whereas SCLC‐A and SCLC‐N are classified as NE tumours (NE high and low, respectively).<a href="#ctm21060-bib-0004" class="usa-link" aria-describedby="ctm21060-bib-0004" aria-expanded="false">
<sup>4</sup>
</a>, <a href="#ctm21060-bib-0006" class="usa-link" aria-describedby="ctm21060-bib-0006" aria-expanded="false">
<sup>6</sup>
</a> Moreover, our group also found recently that high POU2F3 expression is associated with improved survival outcomes, whereas elevated ASCL1 expression is an independent negative prognosticator in surgically treated SCLC patients,<a href="#ctm21060-bib-0009" class="usa-link" aria-describedby="ctm21060-bib-0009" aria-expanded="false">
<sup>9</sup>
</a> highlighting the prognostic relevance of SCLC subgroups. So far, the vast majority of SCLC profiling studies have been conducted by analysing transcriptomic data and gene expression profiles of clinical samples and preclinical models, such as patient‐derived and circulating tumour cell‐derived SCLC xenografts.<a href="#ctm21060-bib-0006" class="usa-link" aria-describedby="ctm21060-bib-0006" aria-expanded="false">
<sup>6</sup>
</a>, <a href="#ctm21060-bib-0008" class="usa-link" aria-describedby="ctm21060-bib-0008" aria-expanded="false">
<sup>8</sup>
</a>, <a href="#ctm21060-bib-0010" class="usa-link" aria-describedby="ctm21060-bib-0010" aria-expanded="false">
<sup>10</sup>
</a>, <a href="#ctm21060-bib-0011" class="usa-link" aria-describedby="ctm21060-bib-0011" aria-expanded="false">
<sup>11</sup>
</a> Although recent immunohistochemistry (IHC) analyses confirmed that subtype‐specific markers are indeed detectable in human tumour tissue samples, these studies failed to distinguish a unique <em>YAP1</em>‐driven subtype.<a href="#ctm21060-bib-0012" class="usa-link" aria-describedby="ctm21060-bib-0012" aria-expanded="false">
<sup>12</sup>
</a> Of note, based on tumour expression data analysis by non‐negative matrix factorization, Gay et&nbsp;al. proposed a unique inflamed (SCLC‐I) subtype characterized by an inflamed gene signature as an alternative to the YAP1‐defined subtype.<a href="#ctm21060-bib-0013" class="usa-link" aria-describedby="ctm21060-bib-0013" aria-expanded="false">
<sup>13</sup>
</a> Additionally, not all previously expected RNA‐based correlation patterns between subtype markers and NE features could have been validated by IHC either.<a href="#ctm21060-bib-0012" class="usa-link" aria-describedby="ctm21060-bib-0012" aria-expanded="false">
<sup>12</sup>
</a> Therefore, subtype classification based on transcriptional profiling might not be exclusively representative concerning protein expression.</p>
<p>Mass spectrometry (MS)‐based proteomics enables a large‐scale analysis of complex biological systems, such as cells, tissues or blood plasma. Parallel detection and quantitation of thousands of proteins, including those with lower abundance, is feasible with modern high‐resolution mass spectrometers and advanced sample preparation workflows, thus achieving a better understanding of molecular interactions and signalling pathways in cancer.<a href="#ctm21060-bib-0014" class="usa-link" aria-describedby="ctm21060-bib-0014" aria-expanded="false">
<sup>14</sup>
</a>, <a href="#ctm21060-bib-0015" class="usa-link" aria-describedby="ctm21060-bib-0015" aria-expanded="false">
<sup>15</sup>
</a> The scarcity of available material has so far hindered a comprehensive proteomic examination of SCLC tissues, whereas the proteome of non‐SCLC tissues were thoroughly mapped previously.<a href="#ctm21060-bib-0016" class="usa-link" aria-describedby="ctm21060-bib-0016" aria-expanded="false">
<sup>16</sup>
</a>, <a href="#ctm21060-bib-0017" class="usa-link" aria-describedby="ctm21060-bib-0017" aria-expanded="false">
<sup>17</sup>
</a> To date, proteomic studies considered SCLC as a single entity and compared SCLC samples with normal bronchial epithelial tissues,<a href="#ctm21060-bib-0018" class="usa-link" aria-describedby="ctm21060-bib-0018" aria-expanded="false">
<sup>18</sup>
</a> non‐SCLC cell lines<a href="#ctm21060-bib-0019" class="usa-link" aria-describedby="ctm21060-bib-0019" aria-expanded="false">
<sup>19</sup>
</a> or carcinoid tumour tissues.<a href="#ctm21060-bib-0020" class="usa-link" aria-describedby="ctm21060-bib-0020" aria-expanded="false">
<sup>20</sup>
</a> Of note, these studies reported relatively low number of identifiable proteins, ranging from 193 to 1991. A proteomic study of 949 cancer cell lines from 28 tissue types, including 57 SCLC cell lines, was published only recently, in which a total of 8498 proteins were quantified.<a href="#ctm21060-bib-0021" class="usa-link" aria-describedby="ctm21060-bib-0021" aria-expanded="false">
<sup>21</sup>
</a>
</p>
<p>In the current work, we conducted a comparative proteomic analysis of SCLC subtypes using cell lines categorized based on the relative expression of four key transcription factors. Our label‐free shotgun proteomic approach assessing both the cell pellet (CP) and culture media (CM) resulted in the confident quantification of nearly 9000 proteins. This comprehensive proteomic evaluation of SCLC cell lines integrated with knowledge from existing transcriptomic datasets can give a clearer definition of SCLC subtypes and might provide insights into their specific features that govern therapy response.</p></section><section id="ctm21060-sec-0060"><h2 class="pmc_sec_title" data-anchor-id="ctm21060-sec-0060"><a id="ctm21060-sec-0060-anchor" data-anchor-id="ctm21060-sec-0060" class="usa-anchor"></a>2. MATERIALS AND METHODS</h2>
<section id="ctm21060-sec-0070"><h3 class="pmc_sec_title">2.1. Cell culture</h3>
<p>Human SCLC cell lines were maintained in RPMI‐1640 with 10% fetal calf serum (Sigma Chemical Co.), 100 U/ml penicillin and 10 mg/ml streptomycin (Sigma Chemical Co.) at 37°C in a humidified incubator with 5% CO<sub>2</sub>. SCLC cell lines were either purchased from ATCC or kindly provided by our collaborators. All cell lines were regularly checked for mycoplasma contamination using the luminescence‐based MycoAlert mycoplasma detection kit (Lonza) with supernatant from cells cultured for &gt;3 days and used within 10 passages after authentication. The cell lines and their key characteristics are listed in Table&nbsp;<a href="#ctm21060-tbl-0001" class="usa-link">1</a>.</p>
<section class="tw xbox font-sm" id="ctm21060-tbl-0001"><h4 class="obj_head">TABLE 1.</h4>
<div class="caption p"><p>Cell lines included in the study, and their general characteristics</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup><col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
</colgroup><thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Cell line ID</th>
<th align="left" rowspan="1" colspan="1">Other cell line ID</th>
<th align="left" rowspan="1" colspan="1">Subtype</th>
<th align="left" rowspan="1" colspan="1">Cell line origin</th>
<th align="left" rowspan="1" colspan="1">Chemotherapy</th>
<th align="left" rowspan="1" colspan="1">Culture type</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">DMS153</td>
<td align="left" rowspan="1" colspan="1">CRL‐2064</td>
<td align="left" rowspan="1" colspan="1">SCLC‐A</td>
<td align="left" rowspan="1" colspan="1">Metastatic</td>
<td align="left" rowspan="1" colspan="1">Post‐chemo</td>
<td align="left" rowspan="1" colspan="1">Semi‐adherent</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">DMS53</td>
<td align="left" rowspan="1" colspan="1">CRL‐2062</td>
<td align="left" rowspan="1" colspan="1">SCLC‐A</td>
<td align="left" rowspan="1" colspan="1">Lung</td>
<td align="left" rowspan="1" colspan="1">Chemo‐naïve</td>
<td align="left" rowspan="1" colspan="1">Adherent</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">H146</td>
<td align="left" rowspan="1" colspan="1">HTB‐173</td>
<td align="left" rowspan="1" colspan="1">SCLC‐A</td>
<td align="left" rowspan="1" colspan="1">Metastatic</td>
<td align="left" rowspan="1" colspan="1">Chemo‐naïve</td>
<td align="left" rowspan="1" colspan="1">Suspension</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">H1688</td>
<td align="left" rowspan="1" colspan="1">CCL‐257</td>
<td align="left" rowspan="1" colspan="1">SCLC‐A</td>
<td align="left" rowspan="1" colspan="1">Metastatic</td>
<td align="left" rowspan="1" colspan="1">Chemo‐naïve</td>
<td align="left" rowspan="1" colspan="1">Adherent</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">H1882</td>
<td align="left" rowspan="1" colspan="1">CRL‐5903</td>
<td align="left" rowspan="1" colspan="1">SCLC‐A</td>
<td align="left" rowspan="1" colspan="1">Metastatic</td>
<td align="left" rowspan="1" colspan="1">N/A</td>
<td align="left" rowspan="1" colspan="1">Adherent</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">H209</td>
<td align="left" rowspan="1" colspan="1">HTB‐172</td>
<td align="left" rowspan="1" colspan="1">SCLC‐A</td>
<td align="left" rowspan="1" colspan="1">Metastatic</td>
<td align="left" rowspan="1" colspan="1">Chemo‐naïve</td>
<td align="left" rowspan="1" colspan="1">Suspension</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">H378</td>
<td align="left" rowspan="1" colspan="1">CRL‐5808</td>
<td align="left" rowspan="1" colspan="1">SCLC‐A</td>
<td align="left" rowspan="1" colspan="1">Lung</td>
<td align="left" rowspan="1" colspan="1">Post‐chemo</td>
<td align="left" rowspan="1" colspan="1">Suspension</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SHP77</td>
<td align="left" rowspan="1" colspan="1">CRL‐2195</td>
<td align="left" rowspan="1" colspan="1">SCLC‐A</td>
<td align="left" rowspan="1" colspan="1">Lung</td>
<td align="left" rowspan="1" colspan="1">N/A</td>
<td align="left" rowspan="1" colspan="1">Adherent</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">GLC4</td>
<td align="left" rowspan="1" colspan="1">N/A</td>
<td align="left" rowspan="1" colspan="1">SCLC‐N</td>
<td align="left" rowspan="1" colspan="1">Pleural eff.</td>
<td align="left" rowspan="1" colspan="1">Chemo‐naïve</td>
<td align="left" rowspan="1" colspan="1">Suspension</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">H1694</td>
<td align="left" rowspan="1" colspan="1">CRL‐5888</td>
<td align="left" rowspan="1" colspan="1">SCLC‐N</td>
<td align="left" rowspan="1" colspan="1">Lung</td>
<td align="left" rowspan="1" colspan="1">N/A</td>
<td align="left" rowspan="1" colspan="1">Semi‐adherent</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">H2171</td>
<td align="left" rowspan="1" colspan="1">CRL‐5929</td>
<td align="left" rowspan="1" colspan="1">SCLC‐N</td>
<td align="left" rowspan="1" colspan="1">Pleural eff.</td>
<td align="left" rowspan="1" colspan="1">Post‐chemo</td>
<td align="left" rowspan="1" colspan="1">Suspension</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">H446</td>
<td align="left" rowspan="1" colspan="1">HTB‐171</td>
<td align="left" rowspan="1" colspan="1">SCLC‐N</td>
<td align="left" rowspan="1" colspan="1">Pleural eff.</td>
<td align="left" rowspan="1" colspan="1">N/A</td>
<td align="left" rowspan="1" colspan="1">Adherent</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">H524</td>
<td align="left" rowspan="1" colspan="1">CRL‐5831</td>
<td align="left" rowspan="1" colspan="1">SCLC‐N</td>
<td align="left" rowspan="1" colspan="1">Metastatic</td>
<td align="left" rowspan="1" colspan="1">Post‐chemo</td>
<td align="left" rowspan="1" colspan="1">Suspension</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">H82</td>
<td align="left" rowspan="1" colspan="1">HTB‐175</td>
<td align="left" rowspan="1" colspan="1">SCLC‐N</td>
<td align="left" rowspan="1" colspan="1">Metastatic</td>
<td align="left" rowspan="1" colspan="1">N/A</td>
<td align="left" rowspan="1" colspan="1">Semi‐adherent</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">N417</td>
<td align="left" rowspan="1" colspan="1">CRL‐5809</td>
<td align="left" rowspan="1" colspan="1">SCLC‐N</td>
<td align="left" rowspan="1" colspan="1">Lung</td>
<td align="left" rowspan="1" colspan="1">N/A</td>
<td align="left" rowspan="1" colspan="1">Suspension</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">COR‐L311</td>
<td align="left" rowspan="1" colspan="1">N/A</td>
<td align="left" rowspan="1" colspan="1">SCLC‐P</td>
<td align="left" rowspan="1" colspan="1">Lung</td>
<td align="left" rowspan="1" colspan="1">Post‐chemo</td>
<td align="left" rowspan="1" colspan="1">Suspension</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">H1048</td>
<td align="left" rowspan="1" colspan="1">CRL‐5853</td>
<td align="left" rowspan="1" colspan="1">SCLC‐P</td>
<td align="left" rowspan="1" colspan="1">Pleural eff.</td>
<td align="left" rowspan="1" colspan="1">N/A</td>
<td align="left" rowspan="1" colspan="1">Adherent</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">H211</td>
<td align="left" rowspan="1" colspan="1">CRL‐5824</td>
<td align="left" rowspan="1" colspan="1">SCLC‐P</td>
<td align="left" rowspan="1" colspan="1">Lung</td>
<td align="left" rowspan="1" colspan="1">Post‐chemo</td>
<td align="left" rowspan="1" colspan="1">Suspension</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">H526</td>
<td align="left" rowspan="1" colspan="1">CRL‐5811</td>
<td align="left" rowspan="1" colspan="1">SCLC‐P</td>
<td align="left" rowspan="1" colspan="1">Metastatic</td>
<td align="left" rowspan="1" colspan="1">Chemo‐naïve</td>
<td align="left" rowspan="1" colspan="1">Suspension</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">CRL‐2066</td>
<td align="left" rowspan="1" colspan="1">DMS 114</td>
<td align="left" rowspan="1" colspan="1">SCLC‐Y</td>
<td align="left" rowspan="1" colspan="1">Lung</td>
<td align="left" rowspan="1" colspan="1">Chemo‐naïve</td>
<td align="left" rowspan="1" colspan="1">Adherent</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">CRL‐2177</td>
<td align="left" rowspan="1" colspan="1">SW1271</td>
<td align="left" rowspan="1" colspan="1">SCLC‐Y</td>
<td align="left" rowspan="1" colspan="1">Lung</td>
<td align="left" rowspan="1" colspan="1">N/A</td>
<td align="left" rowspan="1" colspan="1">Adherent</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">H1341</td>
<td align="left" rowspan="1" colspan="1">CRL‐5864</td>
<td align="left" rowspan="1" colspan="1">SCLC‐Y</td>
<td align="left" rowspan="1" colspan="1">Metastatic</td>
<td align="left" rowspan="1" colspan="1">N/A</td>
<td align="left" rowspan="1" colspan="1">Adherent</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">H196</td>
<td align="left" rowspan="1" colspan="1">CRL‐5823</td>
<td align="left" rowspan="1" colspan="1">SCLC‐Y</td>
<td align="left" rowspan="1" colspan="1">Pleural eff.</td>
<td align="left" rowspan="1" colspan="1">Post‐chemo</td>
<td align="left" rowspan="1" colspan="1">Adherent</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">H372</td>
<td align="left" rowspan="1" colspan="1">N/A</td>
<td align="left" rowspan="1" colspan="1">SCLC‐Y</td>
<td align="left" rowspan="1" colspan="1">Metastatic</td>
<td align="left" rowspan="1" colspan="1">N/A</td>
<td align="left" rowspan="1" colspan="1">Adherent</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">H841</td>
<td align="left" rowspan="1" colspan="1">CRL‐5845</td>
<td align="left" rowspan="1" colspan="1">SCLC‐Y</td>
<td align="left" rowspan="1" colspan="1">Lung</td>
<td align="left" rowspan="1" colspan="1">Post‐chemo</td>
<td align="left" rowspan="1" colspan="1">Adherent</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">HLHE</td>
<td align="left" rowspan="1" colspan="1">N/A</td>
<td align="left" rowspan="1" colspan="1">SCLC‐Y</td>
<td align="left" rowspan="1" colspan="1">Metastatic</td>
<td align="left" rowspan="1" colspan="1">N/A</td>
<td align="left" rowspan="1" colspan="1">Adherent</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/ctm21060-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="ctm21060-tbl1-note-0001"><p>
<em>Note</em>: Pleural eff., pleural effusion.</p></div>
<div class="fn" id="ctm21060-tbl1-note-0002"><p>Abbreviations: N/A, not available; SCLC, small‐cell lung cancer.</p></div>
</div></section></section><section id="ctm21060-sec-0080"><h3 class="pmc_sec_title">2.2. RNA isolation and qPCR</h3>
<p>Cells were incubated in T25 flasks until 70% confluency. Total RNA was isolated with TRIzol reagent and reverse transcribed with MMLV reverse transcriptase (Thermo Fisher Scientific) according to the manufacturer's protocol. Gene expression was analysed by quantitative polymerase chain reaction (qPCR) using TaqMan gene expression assays on the Applied Biosystems 7500 Fast Real‐Time PCR System (Assay IDs: <em>ASCL1</em>: Hs00269932_m1, <em>NEUROD1</em>: Hs00159598_m1, <em>POU2F3</em>: Hs00205009_m1, <em>YAP1</em>: Hs00371735_m1, <em>GAPDH</em>: Hs02786624_g1; Thermo Fisher Scientific). Gene expression was calculated as 2<sup>−Δ</sup>
<em>
<sup>Ct</sup>
</em> using glyceraldehyde 3‐phosphate dehydrogenase (<em>GAPDH</em>) as reference gene.<a href="#ctm21060-bib-0022" class="usa-link" aria-describedby="ctm21060-bib-0022" aria-expanded="false">
<sup>22</sup>
</a>
</p></section><section id="ctm21060-sec-0090"><h3 class="pmc_sec_title">2.3. Sample processing for proteomics</h3>
<p>CPs and CM from 26 cell lines were processed and subjected to an MS‐based proteomic analysis. In brief, the CPs were solubilized with protein extraction buffer (25 mM dithiothreitol [DTT], 10% sodium dodecyl sulphate (SDS), 100 mM triethylammonium bicarbonate [TEAB], pH 8), incubating for 5&nbsp;min at 95°C with 500 rpm shaking. The volume of the buffer was adjusted to the number of cells in each sample, that is 250&nbsp;μl of protein extraction buffer was added to samples containing 5 M cells. Proteins were extracted via a 20 min sonication at 4°C (Bioruptor Plus, Diagenode) with 40 cycles (15 s on/15 s off), followed by a brief centrifugation at 20000 × <em>g</em> at 18°C, discarding the cell debris. Protein determination was performed using a Pierce 660 nm Protein Assay kit (Thermo Scientific), following the manufacturer's instructions.</p>
<p>Additionally, the filtered CM samples were concentrated using spin concentrators (5K 4&nbsp;ml, Agilent Technologies) to ∼100&nbsp;μl. Protein determination was performed with NanoDrop (DeNovix DS‐11 FX +). Prior to digestion, SDS was added to a final concentration of 3%, and 100 mM TEAB was added to increase the pH. Reduction was performed with 10 mM DTT and a 1&nbsp;h incubation at 37°C.</p>
<p>Protein digestion for both the CP and media (100&nbsp;μg of proteins per sample) was accomplished using the S‐Trap technology (ProtiFi) with few modifications, as previously described by our group.<a href="#ctm21060-bib-0023" class="usa-link" aria-describedby="ctm21060-bib-0023" aria-expanded="false">
<sup>23</sup>
</a> In brief, the samples were alkylated with 50 mM iodoacetamide and acidified with 1.2% phosphoric acid (final concentration). Then, S‐Trap binding buffer was added (90% methanol, 100 mM TEAB) to 7× the final sample volume, and the samples were transferred to the S‐Trap 96‐well digestion plate. Captured proteins were washed four times with 200 μl S‐Trap binding buffer and brief centrifugations (2&nbsp;min at 1000 × <em>g</em>). Digestion buffer (50 mM TEAB) containing LysC at 1:50 enzyme‐to‐protein ratio was added on top of the filters, incubating for 2&nbsp;h at 37°C. Next, digestion buffer containing trypsin at 1:50 enzyme‐to‐protein ratio was added to the samples, incubating overnight at 37°C. On the following day, the peptides were eluted in three steps, first with 80&nbsp;μl of digestion buffer, then with 80&nbsp;μl of 0.2% formic acid, and finally with 80&nbsp;μl of 50% acetonitrile (ACN) containing 0.2% formic acid. All peptides were dried down in a vacuum concentrator. Peptide determination was performed for all samples using Pierce Quantitative Colorimetric Peptide Assay kit (Thermo Scientific), following the manufacturer's instructions.</p></section><section id="ctm21060-sec-0100"><h3 class="pmc_sec_title">2.4. Nano LC–MS/MS analysis</h3>
<p>The nLC–MS/MS analysis was performed on a Q Exactive HF‐X mass spectrometer coupled to a Dionex UltiMate 3000 RSLCnano UPLC system (Thermo Scientific), with an EASY‐Spray ion source. Peptides from CPs and media were injected in triplicates (1.5 and 1&nbsp;μg, respectively), utilizing two MS methods detailed later. All samples were loaded onto an Acclaim PepMap 100 C18 (75&nbsp;μm × 2&nbsp;cm, 3&nbsp;μm, 100&nbsp;Å, nanoViper) trap column and separated on an Acclaim PepMap RSLC C18 column (75&nbsp;μm × 50&nbsp;cm, 2&nbsp;μm, 100&nbsp;Å) (Thermo Scientific) using a flow rate of 300&nbsp;nl/min, a column temperature of 60°C. A 145‐min gradient was applied for separation, using solvents A (0.1% formic acid) and B (0.1% formic acid in 80% ACN), increasing solvent B from 2% to 25% in 115&nbsp;min, then to 32% in the next 10&nbsp;min, and to 45% in 7&nbsp;min. Finally, the gradient increased to 90% solvent B in 8&nbsp;min, continuing for another 5&nbsp;min.</p>
<p>Regarding the MS approach, peptides from the CP were analysed with one data‐dependent acquisition (DDA) and two data‐independent acquisitions (DIA), whereas peptides from the CM were analysed with two DDA and one DIA runs.</p>
<p>The top 20 DDA method was applied with full MS1 scans at <em>m</em>/<em>z</em> 375–1500, resolution of 120000 (at 200 <em>m</em>/<em>z</em>), target AGC value of 3&nbsp;×&nbsp;10<sup>6</sup> and maximum injection time of 100&nbsp;ms. Fragmentation was done with an NCE of 28, and the isolation window was set to 1.2 <em>m</em>/<em>z</em>. MS2 scans were acquired with a resolution of 15000 (at 200 <em>m</em>/<em>z</em>), target AGC value of 1&nbsp;×&nbsp;10<sup>5</sup>, maximum injection time of 50&nbsp;ms, ion selection threshold of 8&nbsp;×&nbsp;10<sup>3</sup> and dynamic exclusion of 40&nbsp;s.</p>
<p>For the DIA analysis, a complete acquisition cycle consisted of 3 MS1 full scans, each followed by 18 MS2 DIA scans with variable isolation windows. MS1 full scans were acquired at <em>m</em>/<em>z</em> 375–1455, with a resolution of 120000 (at 200 <em>m</em>/<em>z</em>), target AGC value of 3&nbsp;×&nbsp;10<sup>6</sup> and maximum injection time of 50&nbsp;ms. The MS2 scans were acquired with a resolution of 30000 (at 200 <em>m</em>/<em>z</em>), fragmentation with NCE of 28, target AGC value of 1&nbsp;×&nbsp;10<sup>6</sup>, automatic maximum injection time, fixed first mass of 200&nbsp;<em>m</em>/<em>z</em> and the variable isolation windows were 13.0, 16.0, 26.0 and 61.0 <em>m</em>/<em>z</em> (with 27, 13, 8 and 6 loop counts, respectively).</p>
<p>See Table <a href="#ctm21060-supitem-0002" class="usa-link">S1</a> for details on biological and technical repeats. Samples were run in a randomized order, in two distinct batches.</p></section><section id="ctm21060-sec-0110"><h3 class="pmc_sec_title">2.5. Proteomic data processing</h3>
<section id="ctm21060-sec-0120"><h4 class="pmc_sec_title">2.5.1. Database search</h4>
<p>Database search was done on Proteome Discoverer v2.4 using SEQUEST HT search engine combined with spectral library search, using the UniProtKB human database (accessed on 15 January 2019) and Proteome Tools spectral libraries. Dynamic modifications included oxidation of methionine and N‐terminal acetylation, whereas carbamidomethylation of cysteine was implemented as a static modification. Precursor tolerance was set to 10&nbsp;ppm and fragment mass tolerance was set to 0.02&nbsp;Da. Additionally, a maximum of two missed cleavages were allowed, and a false discovery rate (FDR) of 1% was set both on peptide and protein levels. For protein quantitation, the top three average methods were used (i.e. it was calculated based on the average of top three peptide abundances unique to the protein group).</p></section><section id="ctm21060-sec-0130"><h4 class="pmc_sec_title">2.5.2. Post‐processing</h4>
<p>The data‐processing steps were performed separately for the CP and CM samples. Both the processing and the statistical tests were performed in R v.4.2.0<a href="#ctm21060-bib-0024" class="usa-link" aria-describedby="ctm21060-bib-0024" aria-expanded="false">
<sup>24</sup>
</a> unless specified otherwise. See <a href="https://github.com/bszeitz/SCLC_proteomics" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/bszeitz/SCLC_proteomics</a> for details on all used packages.</p>
<p>The raw protein intensities were log<sub>2</sub>‐transformed, and individual measurements were median normalized by centring their intensities around the global median. Measurements from the same batch were then checked for outliers. After removal of low‐quality measurements in the CM data (see Table <a href="#ctm21060-supitem-0002" class="usa-link">S1</a> for details on removed samples), the median intensity of proteins in repeated measurements from the same MS vial and in the same batch was used for further analysis. Proteins with no quantitative values in any samples were removed, leading to the 9570 and 6425 proteins in the CP and CM, respectively. A strong batch effect that was independent from other factors was observed in the data (Figure <a href="#ctm21060-supitem-0001" class="usa-link">S1a</a>); therefore, differences in protein expression caused by batch effect were removed using univariate linear regression. Proteins that had no quantitative values in one of the batches were excluded from the batch effect correction. Consequently, the variance explained by the subtype increased (Figure <a href="#ctm21060-supitem-0001" class="usa-link">S1b</a>), and the technical and biological replicates from the same cell line showed high similarity according to unsupervised hierarchical clustering results (Figure <a href="#ctm21060-supitem-0001" class="usa-link">S1c</a>). All replicates from the same cell line were, therefore, averaged by taking their arithmetic mean.</p>
<p>For statistical analyses, the expression table was filtered for proteins with min. 80% valid values across samples, resulting in 8405 and 5408 proteins in the CP and CM, respectively. Missing value imputation was performed in Perseus v.1.6<a href="#ctm21060-bib-0025" class="usa-link" aria-describedby="ctm21060-bib-0025" aria-expanded="false">
<sup>25</sup>
</a> using imputation based on normal distribution (width&nbsp;=&nbsp;0.3, down shift&nbsp;=&nbsp;1.8). The protein intensity histogram per sample, also highlighting the frequency distribution of imputed values, is shown on Figure <a href="#ctm21060-supitem-0001" class="usa-link">S1d</a> for the samples with the highest missing value content. Additionally, potential ‘on/off proteins’ were also sought out during the filtering step. The ‘on/off proteins’ were defined as proteins that were present in min. 85% of the samples in one subtype and simultaneously, in max. 15% of the samples in the other subtypes (‘on’), or in max. 15% of the samples in one subtype and simultaneously, in min. 85% of the samples in the other subtypes (‘off’). These proteins, together with the filtered and imputed expression table elements, were also examined as part of the analysis of subtype‐specific proteins.</p></section><section id="ctm21060-sec-0140"><h4 class="pmc_sec_title">2.5.3. Annotation of secreted, cell‐surface, plasma and ‘druggable’ proteins</h4>
<p>To map secreted proteins from the list of identified proteins, three secretome databases were utilized: Human Protein Atlas (The Human Protein Atlas v20.1 and Ensembl v92.38, retrieved on 11 April 2021),<a href="#ctm21060-bib-0026" class="usa-link" aria-describedby="ctm21060-bib-0026" aria-expanded="false">
<sup>26</sup>
</a> SPRomeDB (accessed on 7 April 2021)<a href="#ctm21060-bib-0027" class="usa-link" aria-describedby="ctm21060-bib-0027" aria-expanded="false">
<sup>27</sup>
</a> and MetazSecKB (accessed on 7 April 2021).<a href="#ctm21060-bib-0028" class="usa-link" aria-describedby="ctm21060-bib-0028" aria-expanded="false">
<sup>28</sup>
</a> These databases were created using curated experimental evidence and a number of prediction tools that do not completely overlap. All proteins mentioned in at least two databases were considered secreted proteins. In sum, 422 and 636 secreted proteins were detected in the CP and CM, respectively, out of which 295 and 514 were also quantified in min. 80% of the samples.</p>
<p>Additionally, information from the Human Protein Atlas (The Human Protein Atlas v21.1,
<a href="https://www.proteinatlas.org/humanproteome/blood%2Bprotein" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.proteinatlas.org/humanproteome/blood+protein</a>, accessed on 23 July 2022) was used to annotate proteins detectable in human blood plasma by immunoassay, MS or proximity extension assays. This was supplemented with a list of proteins actively secreted into the blood (<a href="https://www.proteinatlas.org/search/sa_location%3ASecreted%2Bto%2Bblood" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.proteinatlas.org/search/sa_location%3ASecreted+to+blood</a>). In total, 3076 and 3126 plasma proteins were identified in the CP and CM, respectively, of which 2763 and 2823 were quantified in at least 80% of the samples. Among the proteins actively secreted in the blood, 229 and 334 were found in the CP and CM, respectively, and 172 and 278 proteins were quantified in min. 80% of the samples.</p>
<p>Two databases were downloaded to retrieve a list of cell‐surface proteins, namely The Cancer Surfaceome Atlas<a href="#ctm21060-bib-0029" class="usa-link" aria-describedby="ctm21060-bib-0029" aria-expanded="false">
<sup>29</sup>
</a> (<a href="http://fcgportal.org/TCSA/Download.php" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://fcgportal.org/TCSA/Download.php</a>, accessed on 28 July 2022) and the in silico human surfaceome by Bausch‐Fluck et&nbsp;al.<a href="#ctm21060-bib-0030" class="usa-link" aria-describedby="ctm21060-bib-0030" aria-expanded="false">
<sup>30</sup>
</a> (<a href="http://wlab.ethz.ch/surfaceome/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://wlab.ethz.ch/surfaceome/</a>, accessed on 28 July 2022). In the CP and CM, we found a total of 682 and 549 cell‐surface proteins that are included in both databases, of which 477 and 416 were quantified in at least 80% of the samples.</p>
<p>To annotate ‘druggable’ proteins with subtype specific profiles, we used The druggable proteome database available at the Human Protein Atlas website (<a href="https://www.proteinatlas.org/humanproteome/tissue/druggable" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.proteinatlas.org/humanproteome/tissue/druggable</a>, accessed on 22 October 2021), which contains the protein targets of the current Food and Drug Administration (FDA)‐approved drugs, including enzymes, transporters, ion channels, and receptors. Functional annotations were collected from the UniProt<a href="#ctm21060-bib-0031" class="usa-link" aria-describedby="ctm21060-bib-0031" aria-expanded="false">
<sup>31</sup>
</a> website (<a href="https://www.uniprot.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.uniprot.org/</a>, Release 2022_03) for selected proteins, and the list of FDA‐approved drugs that directly interact with those proteins as part of their mechanism of action was gathered from the DrugBank database (<a href="http://www.drugbank.ca" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">www.drugbank.ca</a>).<a href="#ctm21060-bib-0032" class="usa-link" aria-describedby="ctm21060-bib-0032" aria-expanded="false">
<sup>32</sup>
</a>
</p></section></section><section id="ctm21060-sec-0150"><h3 class="pmc_sec_title">2.6. External data retrieval and processing</h3>
<section id="ctm21060-sec-0160"><h4 class="pmc_sec_title">2.6.1. Transcriptomic data of SCLC tissue samples</h4>
<p>The mRNA data of SCLC tissue samples was accessed from cBioPortal<a href="#ctm21060-bib-0033" class="usa-link" aria-describedby="ctm21060-bib-0033" aria-expanded="false">
<sup>33</sup>
</a>, <a href="#ctm21060-bib-0034" class="usa-link" aria-describedby="ctm21060-bib-0034" aria-expanded="false">
<sup>34</sup>
</a> on 24 May 2021, downloading George et&nbsp;al.’s work.<a href="#ctm21060-bib-0008" class="usa-link" aria-describedby="ctm21060-bib-0008" aria-expanded="false">
<sup>8</sup>
</a> Only samples that were also categorized by Rudin et&nbsp;al.<a href="#ctm21060-bib-0006" class="usa-link" aria-describedby="ctm21060-bib-0006" aria-expanded="false">
<sup>6</sup>
</a> were further analysed (52 samples out of 81). This resulted in 38 SCLC‐A, 5 SCLC‐N, 7 SCLC‐P and 2 SCLC‐Y samples. These 52 samples’ Fragments per kilobase million (FPKM) values were utilized to perform the single‐sample gene set enrichment analysis (ssGSEA), as well as the log‐transformed mRNA <em>Z</em>‐scores (where <em>Z</em>‐score was calculated compared to the expression distribution of all samples), were used to assess the expression profile differences between selected transcripts across subtypes (SCLC‐A/N/P/Y).</p></section><section id="ctm21060-sec-0170"><h4 class="pmc_sec_title">2.6.2. Genomic and transcriptomic data of SCLC cell lines</h4>
<p>Mutation and RNA‐Seq data (gene count data, normalized using RNA‐Seq by Expectation‐Maximization, i.e. RSEM method) of Cancer Cell Line Encyclopedia (CCLE) cancer cell lines were accessed on 28 June 2022 and 23 August 2021, respectively, from the Cancer Dependency Map website, <a href="https://depmap.org/portal/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://depmap.org/portal/</a>.<sup>35</sup> Compared to our set of cell lines, only five cell lines are not included in the mutation table (GLC4, HLHE, H1882, N417, H372), and two additional cell lines are not present in the RNA sequencing table (H378 and H1688). The RNA‐Seq data contains the measurements from 50 SCLC cell lines, which were also categorized into subtypes by Rudin et&nbsp;al.<a href="#ctm21060-bib-0006" class="usa-link" aria-describedby="ctm21060-bib-0006" aria-expanded="false">
<sup>6</sup>
</a> (26 SCLC‐A, 12 SCLC‐N, 4 SCLC‐P and 8 SCLC‐Y). The transcriptomic data was further processed via limma R package v.3.46.0,<a href="#ctm21060-bib-0036" class="usa-link" aria-describedby="ctm21060-bib-0036" aria-expanded="false">
<sup>36</sup>
</a> following the standard RNA‐Seq data‐processing steps. This included normalization factor calculation with default settings, filter of low‐expressed genes (only genes with minimum one expression value equal to or higher than 50 should be kept; fulfilled by 9237 genes) and voom transformation with default settings where model matrix included the subtype assignment. This was followed by differential expression analysis described in detail in Section <a href="#ctm21060-sec-0190" class="usa-link">2.7</a>. Log<sub>2</sub> CPM counts were calculated using the ‘cpm’ function from edgeR v.3.32.1<a href="#ctm21060-bib-0037" class="usa-link" aria-describedby="ctm21060-bib-0037" aria-expanded="false">
<sup>37</sup>
</a>, <a href="#ctm21060-bib-0038" class="usa-link" aria-describedby="ctm21060-bib-0038" aria-expanded="false">
<sup>38</sup>
</a>, <a href="#ctm21060-bib-0039" class="usa-link" aria-describedby="ctm21060-bib-0039" aria-expanded="false">
<sup>39</sup>
</a> in which prior count was set to 3 (i.e. the average count to be added to each observation to avoid zeros in the dataset during log<sub>2</sub>‐transformation). This expression matrix was used to display the differential expression of selected transcripts across subtypes.</p></section><section id="ctm21060-sec-0180"><h4 class="pmc_sec_title">2.6.3. Drug sensitivity data of SCLC cell lines</h4>
<p>CancerRxGene drug sensitivity data<a href="#ctm21060-bib-0040" class="usa-link" aria-describedby="ctm21060-bib-0040" aria-expanded="false">
<sup>40</sup>
</a> (Release 8.3, available from June 2020) was downloaded from the FTP Server of the Wellcome Sanger Institute (<a href="https://ftp.sanger.ac.uk/pub/project/cancerrxgene/releases/current_release/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">ftp://ftp.sanger.ac.uk/pub/project/cancerrxgene/releases/current_release/</a>) on 18 July 2022. Both the Genomics of Drug Sensitivity in Cancer 1 (GDSC1) and GDSC2 datasets were investigated. In total, the subtype classification of 38 SCLC cell lines is known,<a href="#ctm21060-bib-0006" class="usa-link" aria-describedby="ctm21060-bib-0006" aria-expanded="false">
<sup>6</sup>
</a> of which 20 were also measured by proteomics in our study (the missing cell lines: DMS153, GLC4, H1882, H372, HLHE, N417). The logarithmic IC50 values of these cell lines for selected drugs were then compared across the subtypes or were correlated with protein abundance.</p></section></section><section id="ctm21060-sec-0190"><h3 class="pmc_sec_title">2.7. Statistics and bioinformatics</h3>
<section id="ctm21060-sec-0200"><h4 class="pmc_sec_title">2.7.1. Differential expression analyses</h4>
<p>Differential expression analyses for the proteomic data were performed via ANOVA, followed by multiple testing correction (Benjamini–Hochberg, BH) of ANOVA <em>p</em>‐values and Tukey's honestly significant difference (HSD) post hoc tests. Comparisons were done between culture types (adherent, suspension) and subtypes (SCLC‐A, ‐N, ‐P, ‐Y). A protein was considered significant for a comparison if both the ANOVA FDR and the corresponding pairwise Tukey's HSD test <em>p</em>‐value were less than 0.05. Subtype‐specific proteins were selected based on the following criteria: ANOVA FDR&nbsp;&lt;&nbsp;0.05, as well as Tukey's HSD test <em>p</em>&nbsp;&lt;&nbsp;0.05 for all comparisons of the subtype of interest against the other subtypes. Differential expression analysis for the CCLE RNA‐Seq data was performed via limma by fitting a linear model for the six subtype comparisons, followed by Empirical Bayes smoothing of standard errors. Multiple testing correction with the BH method was applied and FDR&nbsp;&lt;&nbsp;0.05 was considered significant. For the transcriptomic dataset by George et&nbsp;al., pairwise Wilcoxon tests were performed on the <em>Z</em>‐scored values to check gene expression differences between the subtypes. Significance level was set to nominal <em>p</em>&nbsp;&lt;&nbsp;0.05.</p>
<p>Overrepresentation analyses (ORAs) were conducted using clusterProfiler v.3.18.1<a href="#ctm21060-bib-0041" class="usa-link" aria-describedby="ctm21060-bib-0041" aria-expanded="false">
<sup>41</sup>
</a>, <a href="#ctm21060-bib-0042" class="usa-link" aria-describedby="ctm21060-bib-0042" aria-expanded="false">
<sup>42</sup>
</a> and ReactomePA v.1.34.0.<a href="#ctm21060-bib-0043" class="usa-link" aria-describedby="ctm21060-bib-0043" aria-expanded="false">
<sup>43</sup>
</a> The default list of human genes was used as background.</p></section><section id="ctm21060-sec-0210"><h4 class="pmc_sec_title">2.7.2. Neuroendocrine and epithelial–mesenchymal transition scores</h4>
<p>Two previously published gene sets were used to introduce a scoring system for the NE and epithelial–mesenchymal transition (EMT) characteristics of cell lines. For NE characteristics, a 50‐gene NE signature built for SCLC cell lines described by Zhang et&nbsp;al.<a href="#ctm21060-bib-0007" class="usa-link" aria-describedby="ctm21060-bib-0007" aria-expanded="false">
<sup>7</sup>
</a> was examined. The number of NE (i.e. markers involved in NE differentiation) and non‐NE markers present in our CP data was 19 and 17, respectively (Figure <a href="#ctm21060-supitem-0001" class="usa-link">S3a</a>). NE score was calculated the following way for each cell line:</p>
<p>NE score&nbsp;=&nbsp;(Mean <em>Z</em>‐score of NE markers) − (Mean <em>Z</em>‐score of non‐NE markers).</p>
<p>Similarly, a gene signature with 22 epithelial and 15 mesenchymal genes published by Kohn et&nbsp;al.<a href="#ctm21060-bib-0044" class="usa-link" aria-describedby="ctm21060-bib-0044" aria-expanded="false">
<sup>44</sup>
</a> was examined, from which 12 and 10 corresponding proteins were quantified in our CP data (Figure <a href="#ctm21060-supitem-0001" class="usa-link">S3b</a>). EMT score was calculated the following way for each cell line:</p>
<p>EMT score&nbsp;=&nbsp;(Mean <em>Z</em>‐score of mesenchymal markers) − (Mean <em>Z</em>‐score of epithelial markers).</p></section><section id="ctm21060-sec-0220"><h4 class="pmc_sec_title">2.7.3. Consensus clustering</h4>
<p>Samples were grouped in an unsupervised manner using the consensus clustering algorithm, implemented in the ConsensusClusterPlus R package v.1.54.0.<a href="#ctm21060-bib-0045" class="usa-link" aria-describedby="ctm21060-bib-0045" aria-expanded="false">
<sup>45</sup>
</a> The basis of clustering was the samples’ global protein expression profile (filtered and imputed) in the CP dataset, which was further restricted to only those proteins that have an SD above 1.25. This list of proteins included protein products of <em>POU2F3</em> and <em>YAP1</em>, both showing large SD (SD&nbsp;=&nbsp;1.40 and 2.05, respectively), whereas <em>NEUROD1</em> did not show sufficiently large variance (SD&nbsp;=&nbsp;0.50), and <em>ASCL1</em> was not quantified in enough samples to be included in the list. The resulting proteome data (including the expression of 1157 proteins without <em>Z</em>‐score normalization) were resampled 1000 times via the bootstrap method with a probability of 0.8 for selecting any item (i.e. sample) and any feature (i.e. protein). The bootstrap sample datasets were clustered using the partitioning around medoids method with the Pearson distance and complete linkage, exploring the range of two to seven clusters. The visual inspection of consensus matrices, as well as the silhouette plots for <em>K</em>&nbsp;=&nbsp;4 and <em>K</em>&nbsp;=&nbsp;5 revealed that a cleaner separation of the clusters can be achieved with <em>K</em>&nbsp;=&nbsp;4; moreover, the average silhouette width was higher for <em>K</em>&nbsp;=&nbsp;4. Based on these observations, and considering the relatively low number of samples, consensus clustering with <em>K</em>&nbsp;=&nbsp;4 was used as sample grouping. Graphical outputs from the ConsensusClusterPlus R package and silhouette plots for <em>K</em>&nbsp;=&nbsp;4 and <em>K</em>&nbsp;=&nbsp;5 are shown in Figure <a href="#ctm21060-supitem-0001" class="usa-link">S4a–c</a>. Silhouette information was computed using the ‘silhouette’ function from R package ‘cluster’ v.2.1.1.</p></section><section id="ctm21060-sec-0230"><h4 class="pmc_sec_title">2.7.4. Gene set enrichment analysis</h4>
<p>Pre‐ranked GSEA (pGSEA) was performed via the ‘GSEA’ function of the clusterProfiler R package for all six subtype comparisons. As input, the hallmark,<a href="#ctm21060-bib-0046" class="usa-link" aria-describedby="ctm21060-bib-0046" aria-expanded="false">
<sup>46</sup>
</a> KEGG,<a href="#ctm21060-bib-0047" class="usa-link" aria-describedby="ctm21060-bib-0047" aria-expanded="false">
<sup>47</sup>
</a> Reactome,<a href="#ctm21060-bib-0048" class="usa-link" aria-describedby="ctm21060-bib-0048" aria-expanded="false">
<sup>48</sup>
</a> Gene Ontology biological process<a href="#ctm21060-bib-0049" class="usa-link" aria-describedby="ctm21060-bib-0049" aria-expanded="false">
<sup>49</sup>
</a>, <a href="#ctm21060-bib-0050" class="usa-link" aria-describedby="ctm21060-bib-0050" aria-expanded="false">
<sup>50</sup>
</a> and oncogenic curated gene sets were used, downloaded from the Molecular Signatures Database (MSigDB) v.7.4,<a href="#ctm21060-bib-0051" class="usa-link" aria-describedby="ctm21060-bib-0051" aria-expanded="false">
<sup>51</sup>
</a>, <a href="#ctm21060-bib-0052" class="usa-link" aria-describedby="ctm21060-bib-0052" aria-expanded="false">
<sup>52</sup>
</a> as well as a ranked list of genes/proteins (based on the multiplication of the FC and −log<sub>10</sub> (<em>p</em>‐value of pairwise test), calculated for all proteins/genes used in the statistical analyses). The pairwise test was the Tukey's HSD post hoc test for proteomics, whereas for CCLE RNA‐Seq, the <em>p</em>‐values were derived from the moderated <em>t</em>‐test statistic. The BH method was used for <em>p</em>‐value adjustment in pGSEA. To obtain the list of subtype‐characteristic gene sets, the following filter was applied on the pGSEA results: <em>p</em>&nbsp;&lt;&nbsp;0.01 in all comparisons of the subtype of interest against the other subtypes with an unequivocal sign (either positive or negative) normalized enrichment score (NES). Gene sets with a <em>p</em>&nbsp;&lt;&nbsp;0.01 for at least one comparison are listed in Table <a href="#ctm21060-supitem-0005" class="usa-link">S4</a>. The relationship between proteomic and transcriptomic results was examined on these subtype‐specific gene sets as follows: if a gene set in one dataset (either proteomic or transcriptomic) was found to be characteristic for a subtype, and the <em>p</em>‐value of the same gene set was less than 0.1 in relevant comparisons of the other dataset, as well as its NES values showed the same direction, the gene set was declared significant in both proteomics and transcriptomics. In contrast, if the gene set in the other dataset did not meet the previous criteria, the gene set was declared significant in only one dataset. A visual explanation for the filtering steps can be seen in Figure <a href="#ctm21060-supitem-0001" class="usa-link">S5</a> and a summary of the obtained results in Table <a href="#ctm21060-supitem-0005" class="usa-link">S4</a>.</p>
<p>ssGSEA<a href="#ctm21060-bib-0053" class="usa-link" aria-describedby="ctm21060-bib-0053" aria-expanded="false">
<sup>53</sup>
</a> was performed on the transcriptomic dataset by George et&nbsp;al., using only transcripts with sum FPKM higher than 50, and only one transcript for each gene was retained (the transcript with the highest sum FPKM value). This resulted in an expression matrix with 13542 transcripts. It should be noted that different transcripts from the same gene may have different biological functions; however, we cannot preserve such depth of information in this analysis. The NES was calculated for subtype‐specific gene sets as defined earlier (the list can be accessed in Table <a href="#ctm21060-supitem-0005" class="usa-link">S4</a>), with the script available on <a href="https://github.com/broadinstitute/ssGSEA2.0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/broadinstitute/ssGSEA2.0</a> (accessed on 05 February 2022). The parameters were set as follows: sample.norm.type&nbsp;=&nbsp;‘rank’, weight&nbsp;=&nbsp;0.75, statistic&nbsp;=&nbsp;‘area.under.RES’, output.score.type&nbsp;=&nbsp;‘NES’, nperm&nbsp;=&nbsp;1000, min.overlap&nbsp;=&nbsp;5, and correl.type&nbsp;=&nbsp;‘z.score’. The gene sets were then filtered for the most representative gene sets (Table <a href="#ctm21060-supitem-0005" class="usa-link">S4</a>), and ssGSEA results for these are depicted in Figure <a href="#ctm21060-supitem-0001" class="usa-link">S6</a>. For visualization purposes, NES values of each gene set were min.–max. scaled across the samples.</p></section><section id="ctm21060-sec-0240"><h4 class="pmc_sec_title">2.7.5. Sparse partial least squares discriminant analysis</h4>
<p>The sparse partial least squares discriminant analysis (sPLS‐DA) was performed via the mixOmics R package v.6.14.0,<a href="#ctm21060-bib-0054" class="usa-link" aria-describedby="ctm21060-bib-0054" aria-expanded="false">
<sup>54</sup>
</a> following the guidelines provided on its website (<a href="http://mixomics.org/case-studies/splsda-srbct/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://mixomics.org/case‐studies/splsda‐srbct/</a>, accessed on 27 July 2021). The full CP and CM data (filtered and imputed) were analysed separately. First, a PLS‐DA model was fitted with 10 components to evaluate the performance and to select the optimal number of components for the final model. The performance plot showed that five and four components with centroid distance measure are sufficient for good performance (0.169 and 0.476 balanced error rate) in the CP and CM data, respectively (Figure <a href="#ctm21060-supitem-0001" class="usa-link">S7a</a>). Then sPLS‐DA model was tuned, estimating the classification performance with respect to the number of selected variables in the model. Maximum number of components was set to five and four in CP and CM, respectively, threefold cross‐validation was used which was repeated 50 times, and centroid distance measure was selected. As a result, the settings of the final model were the following in CP: (1) optimal number of components is 3, (2) the number of proteins to select on each component are 45, 30 and 7. In the case of CM, the settings of the final model: (1) optimal number of components is 2; (2) the number of proteins to select on each component are 8 and 15. The sample plots of the first three components in CP (Figure <a href="#ctm21060-supitem-0001" class="usa-link">S7b</a>) showed that SCLC‐Y is well separated on the first component, SCLC‐A and ‐N are distinguishable on the second component and adding the third component further discriminates SCLC‐P from the rest. Similarly, the first two components were visualized for CM, displaying that SCLC‐Y is best separated on the first component, whereas SCLC‐A/P can be distinguished from SCLC‐N on the second component. The classification performance of these final sPLS‐DA models were 0.113 and 0.484 with centroid distance for CP and CM, respectively (Figure <a href="#ctm21060-supitem-0001" class="usa-link">S7c</a>). Additionally, selected proteins in each component with their loading weight and stability (i.e. ratio of how many times it was selected across the cross‐validation runs) were extracted. The selected proteins are shown in Supplementary Figure <a href="#ctm21060-supitem-0001" class="usa-link">S8</a>. To select the transcripts showing the best match with proteomic results, transcripts were ranked based on the multiplication of the fold change (FC) and −log<sub>10</sub> (<em>p</em>‐value of pairwise test).</p></section><section id="ctm21060-sec-0250"><h4 class="pmc_sec_title">2.7.6. Visualizations</h4>
<p>Individual clustering settings for heat map visualizations were hierarchical clustering, Euclidean distance and complete linkage, unless specified otherwise in the figure legends. Heat maps were plotted using ComplexHeatmap R package v.2.6.2.<a href="#ctm21060-bib-0055" class="usa-link" aria-describedby="ctm21060-bib-0055" aria-expanded="false">
<sup>55</sup>
</a> Principal component analyses (PCAs) and principal variance component analyses (PVCAs) were done on the <em>Z</em>‐scored protein expression tables. PCA biplots were drawn using the ggbiplot R package v.0.55 (<a href="https://github.com/vqv/ggbiplot" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/vqv/ggbiplot</a>). Further visualizations were done using R v.4.2.0 and GraphPad Prism v.8 for Windows, GraphPad Software, San Diego, California USA, <a href="http://www.graphpad.com" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">www.graphpad.com</a>.</p></section></section></section><section id="ctm21060-sec-0260"><h2 class="pmc_sec_title" data-anchor-id="ctm21060-sec-0260"><a id="ctm21060-sec-0260-anchor" data-anchor-id="ctm21060-sec-0260" class="usa-anchor"></a>3. RESULTS</h2>
<section id="ctm21060-sec-0270"><h3 class="pmc_sec_title">3.1. Molecular heterogeneity of SCLC cell lines detected via proteomics</h3>
<p>We characterized 26 cell lines derived from primary or metastatic human SCLC lesions (Table&nbsp;<a href="#ctm21060-tbl-0001" class="usa-link">1</a>) using label‐free proteomic analysis. A total of 10161 proteins were identified and quantified (9570 and 6425 proteins in CP and CM, respectively), and the majority of these proteins were quantified in minimum 80% of the samples (8405 and 5408 proteins in CP and CM, respectively). Altogether, we annotated 699 secreted proteins, 800 cell‐surface proteins, 3440 proteins detectable in human blood plasma, of which 367 are actively secreted into the blood, and 289 ‘druggable’ proteins in our cell lines. The CM consisted of relatively more secreted and plasma proteins, highlighting its added value in the search for potential blood‐based biomarkers.</p>
<p>First, according to their <em>ASCL1</em>, <em>NEUROD1</em>, <em>POU2F3</em> and <em>YAP1</em> mRNA expression patterns – we grouped the cell lines into one of the four respective subgroups: SCLC‐A, SCLC‐N, SCLC‐P and SCLC‐Y (eight, seven, four and seven cell lines, respectively) (Figure&nbsp;<a href="#ctm21060-fig-0001" class="usa-link">1A</a>, upper panel). These transcription factors also showed increased protein levels in their respective subtype (Figure&nbsp;<a href="#ctm21060-fig-0001" class="usa-link">1A</a>, lower panel).</p>
<figure class="fig xbox font-sm" id="ctm21060-fig-0001"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5517/9506422/db0f8dab78ed/CTM2-12-e1060-g007.jpg" loading="lazy" id="jats-graphic-3" height="574" width="535" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/ctm21060-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Proteomic analysis of small‐cell lung cancer (SCLC) cell lines highlights molecular heterogeneity: (A) The mRNA expression of key genes <em>ASCL1</em>, <em>NEUROD1</em>, <em>POU2F3</em> and <em>YAP1</em> to determine the molecular subtypes (top). Data is shown as mean&nbsp;±&nbsp;SD of the 2<sup>−Δ</sup>
<em>
<sup>Ct</sup>
</em>&nbsp;×&nbsp;1000 value, normalized to glyceraldehyde 3‐phosphate dehydrogenase (<em>GAPDH</em>). Each dot represents one cell line and is the mean of two biological replicates performed in triplicates. The significance of Mann–Whitney <em>U</em> tests is indicated above the boxplots (*<em>p</em>&nbsp;&lt;&nbsp;0.05; **<em>p</em>&nbsp;&lt;&nbsp;0.01; ***<em>p</em>&nbsp;&lt;&nbsp;0.001). Bottom panel shows label‐free quantitation (LFQ) values derived from the proteomic analysis, shown as mean&nbsp;±&nbsp;SD of each cell line, for each defined subtype. Missing values are indicated by an <em>x</em>. The significance of independent <em>t</em>‐tests is indicated above the boxplots (*<em>p</em>&nbsp;&lt;&nbsp;0.05; **<em>p</em>&nbsp;&lt;&nbsp;0.01; ***<em>p</em>&nbsp;&lt;&nbsp;0.001); (B) protein expression profile of four well‐known subtype markers (from left to right: <em>CHGA</em>, <em>ANTXR1</em>, <em>AVIL</em> and <em>ITGA5</em>, markers for SCLC‐A, ‐N, ‐P and ‐Y, respectively). The significance of independent <em>t</em>‐tests is indicated above the boxplots (*<em>p</em>&nbsp;&lt;&nbsp;0.05; **<em>p</em>&nbsp;&lt;&nbsp;0.01; ***<em>p</em>&nbsp;&lt;&nbsp;0.001); (C) mean <em>Z</em>‐score values for neuroendocrine (NE) and non‐NE markers in each cell line (left), and mean <em>Z</em>‐score values for epithelial and mesenchymal markers (right)</p></figcaption></figure><p>In general, we identified several members of the Myc proto‐oncogene (MYC) family universally expressed across our samples, such as MYC and protein L‐Myc (MYCL),<a href="#ctm21060-bib-0006" class="usa-link" aria-describedby="ctm21060-bib-0006" aria-expanded="false">
<sup>6</sup>
</a> or the transcription activator nuclear factor 1 B‐type (Figure <a href="#ctm21060-supitem-0001" class="usa-link">S2a</a>).<a href="#ctm21060-bib-0056" class="usa-link" aria-describedby="ctm21060-bib-0056" aria-expanded="false">
<sup>56</sup>
</a> The protein products of <em>TP53</em> and <em>RB1</em> genes, which are well known to be genetically altered in SCLC,<a href="#ctm21060-bib-0057" class="usa-link" aria-describedby="ctm21060-bib-0057" aria-expanded="false">
<sup>57</sup>
</a>, <a href="#ctm21060-bib-0058" class="usa-link" aria-describedby="ctm21060-bib-0058" aria-expanded="false">
<sup>58</sup>
</a>, <a href="#ctm21060-bib-0059" class="usa-link" aria-describedby="ctm21060-bib-0059" aria-expanded="false">
<sup>59</sup>
</a> were quantified in 100.0% and 88.5% of the cell lines, respectively, independent of the previously described <em>TP53</em> and <em>RB1</em> mutational status of the cell lines<a href="#ctm21060-bib-0035" class="usa-link" aria-describedby="ctm21060-bib-0035" aria-expanded="false">
<sup>35</sup>
</a> (Figure <a href="#ctm21060-supitem-0001" class="usa-link">S2b</a>). Several well‐accepted subtype markers<a href="#ctm21060-bib-0003" class="usa-link" aria-describedby="ctm21060-bib-0003" aria-expanded="false">
<sup>3</sup>
</a> showed the expected protein expression profile across the subtypes (Figure <a href="#ctm21060-supitem-0001" class="usa-link">S2c</a>), such as chromogranin‐A (SCLC‐A marker), anthrax toxin receptor 1 (SCLC‐N marker), advillin (SCLC‐P marker) and multiple integrins (SCLC‐Y markers) (Figure&nbsp;<a href="#ctm21060-fig-0001" class="usa-link">1B</a>).</p>
<p>The cell lines were further characterized by their NE and EMT features. NE scores built from 19 NE and 17 non‐NE markers,<a href="#ctm21060-bib-0007" class="usa-link" aria-describedby="ctm21060-bib-0007" aria-expanded="false">
<sup>7</sup>
</a> and EMT scores based on 12 epithelial and 10 mesenchymal markers,<a href="#ctm21060-bib-0044" class="usa-link" aria-describedby="ctm21060-bib-0044" aria-expanded="false">
<sup>44</sup>
</a> were calculated for each cell line (Figure <a href="#ctm21060-supitem-0001" class="usa-link">S3a,b</a>). The mean protein abundances of NE, non‐NE, epithelial and mesenchymal markers in each sample are shown in Figure&nbsp;<a href="#ctm21060-fig-0001" class="usa-link">1D</a>, whereas the mean NE and EMT scores across the subtypes are depicted in Figure <a href="#ctm21060-supitem-0001" class="usa-link">S3c</a>. As expected, most SCLC‐A cell lines expressed NE and epithelial markers more strongly than non‐NE and mesenchymal markers (<em>M</em>
<sub>NE score</sub>&nbsp;=&nbsp;0.71, <em>M</em>
<sub>EMT score</sub>&nbsp;=&nbsp;−0.70). SCLC‐N was found to be a rather NE subtype with mixed epithelial–mesenchymal characteristics (<em>M</em>
<sub>NE score</sub>&nbsp;=&nbsp;0.56 and <em>M</em>
<sub>EMT score</sub>&nbsp;=&nbsp;0.39). SCLC‐P exhibited moderate non‐NE characteristics in our dataset (i.e. lower than SCLC‐A and ‐N, but higher than SCLC‐Y); however, a high expression of epithelial markers was detected (M<sub>NE score</sub>&nbsp;=&nbsp;−0.05 and M<sub>EMT score</sub>&nbsp;=&nbsp;−0.61) in this subtype. In contrast, SCLC‐Y cell lines exhibited prominent non‐NE and mesenchymal traits (M<sub>NE score</sub>&nbsp;=&nbsp;−1.35 and M<sub>EMT score</sub>&nbsp;=&nbsp;0.75). In line with these findings, delta‐like protein 3 (DLL3) protein, an inhibitory Notch pathway ligand,<a href="#ctm21060-bib-0060" class="usa-link" aria-describedby="ctm21060-bib-0060" aria-expanded="false">
<sup>60</sup>
</a> was expressed in decreasing amounts from SCLC‐A to SCLC‐Y, indicating the Notch pathway's gradual activation (Figure <a href="#ctm21060-supitem-0001" class="usa-link">S2c</a>).</p>
<p>Comparing NE and EMT scores between cell lines with different properties, such as culture type, cell line origin and treatment with chemotherapy, we found that adherent cell lines harbour significantly lower NE scores than the non‐adherent ones (Figure <a href="#ctm21060-supitem-0001" class="usa-link">S3d–f</a>). In addition, a significant negative correlation was observed between NE and EMT scores (Figure <a href="#ctm21060-supitem-0001" class="usa-link">S3g</a>).</p></section><section id="ctm21060-sec-0280"><h3 class="pmc_sec_title">3.2. Manifestation of heterogeneous in vitro growth characteristics in the proteome</h3>
<p>Although maintained in the same in vitro conditions, the cell lines showed significantly different growth characteristics. Specifically, out of the 26 cell lines, 10 (38.5%) grew in suspension, 3 (11.5%) in a semi‐adherent form and the other 13 (50.0%) grew adherent on the plastic (Figure&nbsp;<a href="#ctm21060-fig-0002" class="usa-link">2A</a>). Adherent and non‐adherent cell lines showed clearly distinct protein expression profiles (Figure&nbsp;<a href="#ctm21060-fig-0002" class="usa-link">2B,C</a>). In total, 270 and 148 proteins were significantly (BH adjusted <em>p</em>&nbsp;&lt;&nbsp;0.05) downregulated in suspension cell lines compared to adherent cell lines in the CP and CM, respectively, whereas 244 and 244 proteins were significantly upregulated in suspension cell lines in the CP and CM, respectively (Figure&nbsp;<a href="#ctm21060-fig-0002" class="usa-link">2B,C</a>, left). An ORA of differentially expressed proteins, separately for CP and CM but combining up‐ and downregulated proteins, showed that KEGG pathways such as protein processing in endoplasmic reticulum, lysosome and glycosaminoglycan degradation were significantly (<em>p</em>&nbsp;&lt;&nbsp;0.05) enriched both in CP and CM, as well as other pathways such as endocytosis in CP and gap junction in CM were overrepresented, thereby supporting the phenotypic cell line differences on protein level (Figure&nbsp;<a href="#ctm21060-fig-0002" class="usa-link">2B,C</a>, right and Table <a href="#ctm21060-supitem-0003" class="usa-link">S2</a>).</p>
<figure class="fig xbox font-sm" id="ctm21060-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5517/9506422/45bfedd452e3/CTM2-12-e1060-g001.jpg" loading="lazy" id="jats-graphic-5" height="1005" width="792" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/ctm21060-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>In vitro growth characteristics mirrored in the proteome: (A) Pie chart and representative images (taken on a Zeiss Axiovert 40 C microscope) of the different culture types of small‐cell lung cancer (SCLC) cell lines (<em>n</em> =&nbsp;26). Scale bar&nbsp;=&nbsp;100&nbsp;μm; (B) volcano plot depicting the results of differential expression analysis results between suspension and adherent cell lines (left) with the corresponding enrichment map of the overrepresented KEGG pathways (right) in the cell pellet (CP) data; (C) volcano plot depicting the results of differential expression analysis between suspension and adherent cell lines (left) with corresponding enrichment map of the overrepresented KEGG pathways (right) in the culture media (CM) data</p></figcaption></figure></section><section id="ctm21060-sec-0290"><h3 class="pmc_sec_title">3.3. Proteome‐based SCLC subgroups match with mRNA‐based subtyping</h3>
<p>To investigate whether proteomic subtyping correlates with the mRNA‐based classification, unsupervised consensus clustering of the CP samples was performed based on the most variable proteins (Figure <a href="#ctm21060-supitem-0001" class="usa-link">S4a–c</a>). The analysis revealed four clusters in the proteomic data, which agrees with the mRNA‐based subtyping (Figure&nbsp;<a href="#ctm21060-fig-0003" class="usa-link">3A</a>). A discrepancy was detected in only one cell line (H1882), which was classified in the SCLC‐A subgroup according to the qPCR data and in the SCLC‐P subset based on the proteomic results. Of note, this cell line also displayed a higher <em>POU2F3</em> mRNA expression than other SCLC‐A cell lines (Figure&nbsp;<a href="#ctm21060-fig-0003" class="usa-link">3A</a>). Additionally, two adherent SCLC‐A samples (H1688, DMS53) were rather separated from their group members (Figure <a href="#ctm21060-supitem-0001" class="usa-link">S4c</a>). Notably, SCLC‐Y samples exhibited the most distinct protein expression profile. The previous observations are also well reflected on a PCA plot of the CP samples (Figure&nbsp;<a href="#ctm21060-fig-0003" class="usa-link">3B</a>). The CM samples, on the other hand, exhibit a rather heterogeneous expression profile based on their most variable proteins, and the PCA plot showed no apparent separation according to the mRNA‐based subtype classification (Figure&nbsp;<a href="#ctm21060-fig-0003" class="usa-link">3C</a>). Correspondingly, PVCA revealed that the molecular subtypes were less prominent contributors to protein expression variability in CM compared to CP (Figure&nbsp;<a href="#ctm21060-fig-0003" class="usa-link">3D</a>). Furthermore, culture type was identified as an important contributor to the proteomic profiles of SCLC cell lines, which was more pronounced in CM.</p>
<figure class="fig xbox font-sm" id="ctm21060-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5517/9506422/290f5b709216/CTM2-12-e1060-g005.jpg" loading="lazy" id="jats-graphic-7" height="1125" width="788" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/ctm21060-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The mRNA‐based classification of small‐cell lung cancer (SCLC) subtypes correlates with proteomic data: (A) Heat map of consensus clustering results using proteins from the cell pellet (CP) data showing high variation (&gt;1.25 SD). Samples are sorted according to their cluster assignments and their representative protein expression profiles are shown; (B) principal component analyses (PCA) plot of the CP data using the most variable (&gt;1.25 SD) proteins. Cell lines are coloured according to the subtype and shape corresponds to culture type. Normal data ellipses are also drawn for each subtype (probability&nbsp;=&nbsp;68%); (C) PCA plot of the culture media (CM) data using only the most variable (&gt;1.25 SD) proteins. Cell lines are coloured according to the subtype and shape corresponds to culture type; (D) principal variance component analysis (PVCA) reveals the contribution of subtype and culture type to the proteomic profile differences in CP and CM. The residual variation (noted as ‘residual’) represents the remaining biological and technical variance in the dataset which could not be attributed to the abovementioned factors.</p></figcaption></figure></section><section id="ctm21060-sec-0300"><h3 class="pmc_sec_title">3.4. Multi‐omic portraits of SCLC subtypes outline potential subtype‐specific vulnerabilities</h3>
<p>According to the high concordance between mRNA‐ and proteome‐based subtypes, we used the mRNA‐based classification system in differential expression analyses between subtypes. We found 367 and 34 subtype‐specific proteins (ANOVA BH adjusted <em>p</em>&nbsp;&lt;&nbsp;0.05, Tukey's HSD post hoc <em>p</em>&nbsp;&lt;&nbsp;0.05), the levels of which differ in a given subtype compared to all the three other subtypes in the CP and CM data, respectively (Table <a href="#ctm21060-supitem-0004" class="usa-link">S3</a>). This also included four proteins with on/off characteristics in the CP data, namely achaete‐scute homolog 1 (ASCL1; ‘on’ in SCLC‐A), regulator of G‐protein signalling 22 (RGS22; ‘on’ in SCLC‐P), neurexophilin‐4 and puratrophin‐1 (NXPH4 and PKHG4; ‘off’ in SCLC‐Y). All subtype‐specific proteins (including those from CP and CM) were then subjected to the pathway analysis (Table <a href="#ctm21060-supitem-0004" class="usa-link">S3</a>).</p>
<p>In order to more systematically investigate the underlying biology of SCLC subtypes, in addition to identifying subtype‐specific proteins, we also performed a pathway‐based comparison of subgroups using pGSEAs. We looked for significant pathways that were concordantly activated or suppressed in a certain subtype compared to the other three subtypes. To do this, all pairwise subtype comparisons were performed, using the full list of quantified proteins (<em>n</em>&nbsp;=&nbsp;8405) from the CP data. Furthermore, we performed the same analysis on the RNA‐Seq data of 50 SCLC cell lines from CCLE<a href="#ctm21060-bib-0035" class="usa-link" aria-describedby="ctm21060-bib-0035" aria-expanded="false">
<sup>35</sup>
</a> (<em>n</em>&nbsp;=&nbsp;9237 genes) and finally assessed the relationship between proteomic and transcriptomic results (Table <a href="#ctm21060-supitem-0005" class="usa-link">S4</a>). These steps are described in detail in Section 2 and Figure <a href="#ctm21060-supitem-0001" class="usa-link">S5</a>.</p>
<p>Significantly overrepresented (<em>p</em>&nbsp;&lt;&nbsp;0.05) KEGG processes in SCLC‐A according to subtype‐specific proteins (<em>n</em>&nbsp;=&nbsp;33) include oxidative phosphorylation (OXPHOS), as well as phenylalanine metabolism and leukocyte transendothelial migration (Figure&nbsp;<a href="#ctm21060-fig-0004" class="usa-link">4A</a>). Consistent with this, pGSEA results also showed upregulation of OXPHOS and respiratory chain elements based on proteomic data. Positive regulation of neural precursor cell proliferation was supported by both datasets, whereas transcriptomics showed the activation of the subpallium development gene set in SCLC‐A (Figure&nbsp;<a href="#ctm21060-fig-0004" class="usa-link">4B</a>).</p>
<figure class="fig xbox font-sm" id="ctm21060-fig-0004"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5517/9506422/b6ca5dc689df/CTM2-12-e1060-g002.jpg" loading="lazy" id="jats-graphic-9" height="969" width="603" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/ctm21060-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Subtype‐specific biological processes in small‐cell lung cancer (SCLC)‐A/N/P. Significantly overrepresented KEGG pathways (<em>p</em>&nbsp;&lt;&nbsp;0.05) derived from the list of subtype‐specific proteins, as well as the members of these pathways (red and blue colours mean up‐ and downregulated in the given subtype compared to the other subtypes, respectively) are shown on the left side of the panel. The characteristic gene sets for each subtype, as determined by pre‐ranked gene set enrichment analysis (pGSEA), are shown on the right side of the panel. The <em>x</em> axis indicates the average normalized enrichment score (av. NES) in proteomics for comparisons SCLC‐A versus ‐N/P/Y, SCLC‐N versus ‐A/P/Y, or SCLC‐P versus ‐A/N/Y, whereas <em>y</em> axis indicates the av. NES in transcriptomics. Dots refer to individual gene sets, which are summarized by keywords. Gene set activation or suppression supported by both omic data is shown in purple, whereas green and blue means gene sets supported only by proteomics or transcriptomics, respectively. (A and B) SCLC‐A, (C and D) SCLC‐N and (E and F) SCLC‐P</p></figcaption></figure><p>Subtype‐specific proteins in SCLC‐N (<em>n</em>&nbsp;=&nbsp;54) contributed to the significant overrepresentation of KEGG pathways, such as cell cycle, phagosome, riboflavin metabolism and peroxisome proliferator‐activated receptor signalling pathway (Figure&nbsp;<a href="#ctm21060-fig-0004" class="usa-link">4C</a>). SCLC‐N could be further characterized by the suppression of epidermis development processes based on pGSEA. Proteomic data also outlined the downregulation of immune response, cytokine signalling, cell adhesion and cytoskeleton organization, as well as upregulation of transcription and DNA replication (Figure&nbsp;<a href="#ctm21060-fig-0004" class="usa-link">4D</a>).</p>
<p>Considering SCLC‐P specific proteins (<em>n</em>&nbsp;=&nbsp;32), three significantly enriched KEGG pathways were detected, namely phospholipase D signalling, lysosome as well as other glycan degradation (Figure&nbsp;<a href="#ctm21060-fig-0004" class="usa-link">4E</a>). Furthermore, pGSEA showed activation of the neurotrophin signalling pathway and the lamellipodium organization gene set in SCLC‐P (Figure&nbsp;<a href="#ctm21060-fig-0004" class="usa-link">4F</a>).</p>
<p>Regarding SCLC‐Y, multiple KEGG pathways were significantly overrepresented in the context of subtype‐specific proteins (<em>n</em>&nbsp;=&nbsp;271), such as extracellular matrix (ECM)‐receptor interaction, focal adhesion, spliceosome, peroxisome or <em>O</em>‐glycan biosynthesis (Figure&nbsp;<a href="#ctm21060-fig-0005" class="usa-link">5A</a>). Similarly, a bulky list of processes showed upregulation in SCLC‐Y compared to other subtypes according to pGSEA, such as ECM organization, cytokine‐mediated signalling, interleukin signalling, inflammatory response, EMT, response to growth factors, cell‐substrate adhesion and mitogen‐activated protein kinase (MAPK) cascade. Transcriptomic data showed an activation of apoptotic pathways and the Janus kinase‐signal transducer and activator of transcription signalling, whereas proteomics revealed the upregulation of signalling by Rho‐GTPases, as well as activation of transmembrane transporter disorder‐related processes. Moreover, DNA repair, protein acetylation and chromatin modification were found downregulated in this subtype, as detected by the proteomic data (Figure&nbsp;<a href="#ctm21060-fig-0005" class="usa-link">5B</a>).</p>
<figure class="fig xbox font-sm" id="ctm21060-fig-0005"><h4 class="obj_head">FIGURE 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5517/9506422/52c5020d943f/CTM2-12-e1060-g003.jpg" loading="lazy" id="jats-graphic-11" height="587" width="534" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/ctm21060-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Subtype‐specific biological processes in small‐cell lung cancer (SCLC)‐Y and processes verified using tissue transcriptomics: (A) Significantly overrepresented KEGG pathways (<em>p</em>&nbsp;&lt;&nbsp;0.05) derived from the list SCLC‐Y specific proteins, as well as the members of these pathways (red and blue means up‐ and downregulated in SCLC‐Y, respectively); (B) the characteristic gene sets for SCLC‐Y determined by pre‐ranked gene set enrichment analysis (pGSEA). The <em>x</em> axis indicates the average normalized enrichment score (NES) in proteomics for comparisons SCLC‐Y versus ‐A/N/P, whereas the <em>y</em> axis indicates the average NES in transcriptomics. Dots show the individual gene sets, which are then summarized by keywords. Colouring is based on whether the gene set activation or suppression was detected by both proteomics and transcriptomics (purple), or only by proteomics (green) or transcriptomics (blue). (C) The average value of NESs per subtype for SCLC tissue samples, derived from single‐sample gene set enrichment analysis (ssGSEA) for some representative gene sets where the subtype specificity was also supported by the tissue data.</p></figcaption></figure><p>In order to verify the subtype‐specificity observed in our cell lines in SCLC tissues as well, we investigated the behaviour of representative gene sets (<em>n</em>&nbsp;=&nbsp;33) of subtype‐specific processes outlined earlier in the SCLC tissue transcriptomic dataset published by George et&nbsp;al.<a href="#ctm21060-bib-0008" class="usa-link" aria-describedby="ctm21060-bib-0008" aria-expanded="false">
<sup>8</sup>
</a> ssGSEA identified 22 gene sets for which subtype‐specificity was confirmed to some extent in the tissue data (Figures&nbsp;<a href="#ctm21060-fig-0005" class="usa-link">5C</a> and <a href="#ctm21060-supitem-0001" class="usa-link">S6</a>). Of these, OXPHOS activation in SCLC‐A, upregulated DNA replication and downregulated immune response in SCLC‐N, as well as more active EMT and suppressed DNA repair in SCLC‐Y could be highlighted. Neither of the subtype‐specific processes in SCLC‐P could be verified by tissue transcriptomics.</p></section><section id="ctm21060-sec-0310"><h3 class="pmc_sec_title">3.5. Proteomic analysis identifies potential diagnostic markers and druggable targets for SCLC subtypes</h3>
<p>In addition to differential expression analysis, sPLS‐DA (Figure <a href="#ctm21060-supitem-0001" class="usa-link">S7</a>) was performed separately for CP and CM to identify the proteins that are most suitable for subtype classifications based on their expression patterns (i.e. potential IHC‐ or blood‐based markers). The analysis resulted in 104 proteins (82 and 23 in the CP and CM datasets, respectively; one protein was detected in both datasets) showing clearly distinct profiles between at least two subtypes (Figure <a href="#ctm21060-supitem-0001" class="usa-link">S8</a>). The proteins selected by sPLS‐DA could be separated into the following expression pattern categories: upregulated in SCLC‐A versus ‐N, upregulated in SCLC‐N versus SCLC‐A, upregulated in SCLC‐P (vs. other subtypes), upregulated in SCLC‐Y and downregulated in SCLC‐Y (Figure&nbsp;<a href="#ctm21060-fig-0006" class="usa-link">6A</a>). Of note, 35 and 17 out of the 104 proteins showed either clear up‐ or downregulation in the SCLC‐Y subtype, respectively. For several markers, the expression pattern was matching with transcriptomic data from SCLC tissues (top three is shown in Figure&nbsp;<a href="#ctm21060-fig-0006" class="usa-link">6B</a>). Notably, eight of such proteins were found to be overexpressed in SCLC‐Y, including probable glutathione peroxidase 8 (<em>GPX8</em>), pyruvate dehydrogenase kinase isoform 2 (<em>PDK2</em>) and tyrosine–protein kinase receptor UFO (<em>AXL)</em> (Figures&nbsp;<a href="#ctm21060-fig-0006" class="usa-link">6B</a> and <a href="#ctm21060-supitem-0001" class="usa-link">S9a</a>).</p>
<figure class="fig xbox font-sm" id="ctm21060-fig-0006"><h4 class="obj_head">FIGURE 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5517/9506422/dd56407bb15e/CTM2-12-e1060-g004.jpg" loading="lazy" id="jats-graphic-13" height="1094" width="800" alt="FIGURE 6"></p>
<div class="p text-right font-secondary"><a href="figure/ctm21060-fig-0006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Proteins with diagnostic and therapeutic relevance in small‐cell lung cancer (SCLC) subtypes: (A) The proteins selected by partial least squares discriminant analysis (sPLS‐DA), which are suitable for separating the subtypes based on their expression profile. Results from cell pellet (CP) and culture media (CM) are displayed on the top and bottom heat map, respectively. Proteins with the best matching expression profiles between cell line proteomics and tissue transcriptomics are highlighted; (B) gene expression differences of the best matching transcripts showing subtype‐specific expression profile. The significance of Wilcoxon tests is indicated above the boxplots (*<em>p</em>&nbsp;&lt;&nbsp;0.05; **<em>p</em>&nbsp;&lt;&nbsp;0.01; ***<em>p</em>&nbsp;&lt;&nbsp;0.001); (C) the ln(IC50) values of the cell lines for drugs selected from the Genomics of Drug Sensitivity in Cancer 1 (GDSC1) database (from left to right: dasatinib, pazopanib, and vorinostat), as a function of the measured protein expression. The results of the Pearson correlation analysis are indicated above the scatter plots. Dots are coloured according to the cell line's subtype assignment.</p></figcaption></figure><p>Among the proteins confirmed on tissue transcriptome level, six are also detectable in human blood plasma by MS, namely tyrosine–protein kinase receptor UFO (<em>AXL</em>, upregulated in SCLC‐Y), basal cell adhesion molecule (<em>BCAM</em>, upregulated in SCLC‐A vs. ‐N), GTP‐binding protein 1 (<em>GTPBP1</em>, upregulated in SCLC‐N vs. ‐A), keratin‐18 (<em>KRT18</em>, upregulated in SCLC‐A vs. ‐N), noelin (<em>OLFM1</em>, upregulated in SCLC‐N vs. ‐A) and phospholipase C‐gamma‐2 (<em>PLCG2</em>, upregulated in SCLC‐P). The most promising blood‐based biomarker from this list is the protein UFO, which has been previously detected via immunoassay as well.</p>
<p>Finally, we investigated whether among the proteins detected by differential expression analysis or sPLS‐DA (a total of 418 unique proteins), we could find ‘druggable’ proteins<a href="#ctm21060-bib-0061" class="usa-link" aria-describedby="ctm21060-bib-0061" aria-expanded="false">
<sup>61</sup>
</a> (i.e. drug targets approved by FDA). We identified six of such proteins, listed in Table&nbsp;<a href="#ctm21060-tbl-0002" class="usa-link">2</a>, as potential targets for subtype‐specific therapies: aromatic‐L‐amino‐acid decarboxylase (<em>DDC</em>, overexpressed in SCLC‐A), ephrin type‐A receptor 2 (<em>EPHA2</em>), integrin alpha‐V and beta‐1 (<em>ITGAV</em>, <em>ITGB1</em>, overexpressed in SCLC‐Y), histone deacetylase 1 (overexpressed in SCLC‐A/N/P vs. SCLC‐Y), and mast/stem cell growth factor receptor Kit (<em>KIT</em>, overexpressed in SCLC‐P). Multiple drugs that directly interact with the aforementioned proteins as part of their mechanism of action could be delineated (Table&nbsp;<a href="#ctm21060-tbl-0002" class="usa-link">2</a>). Moreover, SCLC cell lines have already been tested against seven of these drugs (CancerRxGene database<a href="#ctm21060-bib-0040" class="usa-link" aria-describedby="ctm21060-bib-0040" aria-expanded="false">
<sup>40</sup>
</a>), namely dasatinib (targeting EPHA2), vorinostat (targeting HDAC1), imatinib, pazopanib, sorafenib, sunitinib and tivozanib (all targeting KIT). We therefore investigated whether the subtypes showed differences in sensitivity to these drugs, either in GDSC1 or in GDSC2 datasets (Figure <a href="#ctm21060-supitem-0001" class="usa-link">S9b</a>). Generally, we found that lower EPHA2, KIT and HDAC1 protein abundance in our cell lines were indicative of increased resistance to the drugs targeting these proteins (Figure&nbsp;<a href="#ctm21060-fig-0006" class="usa-link">6C</a>) according to the GDSC1 dataset. This trend was not verified by the GDSC2 dataset for dasatinib; however, median IC50 values of some KIT‐targeting drugs (pazopanib, sunitinib, tivozanib) were lowest in SCLC‐P cell lines, as well as SCLC‐Y subtype's higher resistance to vorinostat could be validated (Figure <a href="#ctm21060-supitem-0001" class="usa-link">S9b</a>).</p>
<section class="tw xbox font-sm" id="ctm21060-tbl-0002"><h4 class="obj_head">TABLE 2.</h4>
<div class="caption p"><p>Potentially targetable subtype‐specific proteins</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup><col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
</colgroup><thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Protein name (gene name)</th>
<th align="left" rowspan="1" colspan="1">Dataset</th>
<th align="left" rowspan="1" colspan="1">Specificity</th>
<th align="left" rowspan="1" colspan="1">Annotation</th>
<th align="left" rowspan="1" colspan="1">FDA‐approved drugs with pharmacological action</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">Aromatic‐<span class="font-variant-small-caps">l</span>‐amino‐acid decarboxylase (<em>DDC</em>)</td>
<td align="left" rowspan="1" colspan="1">CP</td>
<td align="left" rowspan="1" colspan="1">↑ in A</td>
<td align="left" rowspan="1" colspan="1">Enzyme that catalyses dopamine and serotonin synthesis</td>
<td align="left" rowspan="1" colspan="1">Benserazide, carbidopa, methyldopa</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Ephrin type‐A receptor 2 (<em>EPHA2</em>)</td>
<td align="left" rowspan="1" colspan="1">CP</td>
<td align="left" rowspan="1" colspan="1">↑ in Y</td>
<td align="left" rowspan="1" colspan="1">Receptor tyrosine kinase involved in contact‐dependent bidirectional signalling with neighbouring cells</td>
<td align="left" rowspan="1" colspan="1">Dasatinib, regorafenib</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Histone deacetylase 1 (<em>HDAC1</em>)</td>
<td align="left" rowspan="1" colspan="1">CP</td>
<td align="left" rowspan="1" colspan="1">↑ in A/N/P</td>
<td align="left" rowspan="1" colspan="1">Histone deacetylase with regulatory function in transcriptional processes</td>
<td align="left" rowspan="1" colspan="1">Romidepsin, vorinostat</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Integrin alpha‐V (<em>ITGAV</em>)</td>
<td align="left" rowspan="1" colspan="1">CP</td>
<td align="left" rowspan="1" colspan="1">↑ in Y</td>
<td align="left" rowspan="1" colspan="1">Integrin, receptor for a wide array of proteins. CD marker</td>
<td align="left" rowspan="1" colspan="1">Antithymocyte immunoglobulin, levothyroxine</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Integrin beta‐1 (<em>ITGB1</em>)</td>
<td align="left" rowspan="1" colspan="1">CP</td>
<td align="left" rowspan="1" colspan="1">↑ in Y</td>
<td align="left" rowspan="1" colspan="1">Integrin, receptor for a wide array of proteins. CD marker</td>
<td align="left" rowspan="1" colspan="1">Antithymocyte immunoglobulin</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Mast/stem cell growth factor receptor kit (<em>KIT</em>)</td>
<td align="left" rowspan="1" colspan="1">CP and CM</td>
<td align="left" rowspan="1" colspan="1">↑ in P</td>
<td align="left" rowspan="1" colspan="1">Receptor tyrosine kinase, acts as cell‐surface receptor for the cytokine KITLG/SCF. CD marker</td>
<td align="left" rowspan="1" colspan="1">Ancestim, imatinib, lenvatinib, pazopanib, regorafenib, ripretinib, sorafenib, sunitinib, tivozanib</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/ctm21060-tbl-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="ctm21060-tbl2-note-0001"><p>Abbreviations: ↑, higher expression; A, SCLC‐A; CD, cluster of differentiation; CM, culture media; CP, cell pellet; FDA: Food and Drug Administration; N, SCLC‐N; P, SCLC‐P; SCLC, small cell lung cancer; Y, SCLC‐Y.</p></div></div></section></section></section><section id="ctm21060-sec-0320"><h2 class="pmc_sec_title" data-anchor-id="ctm21060-sec-0320"><a id="ctm21060-sec-0320-anchor" data-anchor-id="ctm21060-sec-0320" class="usa-anchor"></a>4. DISCUSSION</h2>
<p>Although clinically SCLC is still regarded as a homogeneous tumour type with NE characteristics, it was described decades ago that besides the ‘classical’ form of SCLC with suspension growth type and NE phenotype, a ‘variant’ form (NE‐low type) of SCLC also exists, mainly forming adherent cell cultures.<a href="#ctm21060-bib-0062" class="usa-link" aria-describedby="ctm21060-bib-0062" aria-expanded="false">
<sup>62</sup>
</a>, <a href="#ctm21060-bib-0063" class="usa-link" aria-describedby="ctm21060-bib-0063" aria-expanded="false">
<sup>63</sup>
</a> In addition, tumours lacking NE differentiation (non‐NE) have been also described recently.<a href="#ctm21060-bib-0006" class="usa-link" aria-describedby="ctm21060-bib-0006" aria-expanded="false">
<sup>6</sup>
</a> In our SCLC cell line panel, 50.0% of the samples grew adherently, whereas the other half semi‐adherently or in suspension. A significant positive relationship between the adherent phenotype and the non‐NE features was observed. Importantly, the proteomic landscape of the examined cell lines also mirrored these phenotypic differences, and proteins associated with several pathways involved in cell adhesion were identified as differentially expressed between adherently or in‐suspension‐growing cell lines, including glycosaminoglycan degradation and endocytosis or gap junction.</p>
<p>Recent profiling studies highlighted that the NE and non‐NE classes of SCLC can be further divided into additional subtypes, namely SCLC‐A and ‐N (NE‐high and NE‐low, respectively) and SCLC‐P and ‐Y (non‐NE types).<a href="#ctm21060-bib-0006" class="usa-link" aria-describedby="ctm21060-bib-0006" aria-expanded="false">
<sup>6</sup>
</a> Our cell lines were also categorized accordingly using qPCR, resulting in eight, seven, four and seven <em>ASCL1</em>+, <em>NEUROD1</em>+, <em>POU2F3</em>+ and <em>YAP1</em>+ cell lines, respectively. In the global proteomic data, these transcription factors showed higher abundances in their respective subtype, with only the elevated level of neurogenic differentiation factor 1 (<em>NEUROD1</em>) in SCLC‐N cell lines being not significantly different in all three comparisons.</p>
<p>According to the protein expression of NE and non‐NE markers described by Zhang et&nbsp;al.,<a href="#ctm21060-bib-0007" class="usa-link" aria-describedby="ctm21060-bib-0007" aria-expanded="false">
<sup>7</sup>
</a> as expected, SCLC‐A and ‐N cell lines showed NE features, whereas those in the SCLC‐P and ‐Y groups were associated with rather non‐NE protein profiles. This is in line with the results of a recent IHC‐based study on human tissue specimens, in which the authors also found that the likelihood of POU2F3 expression in SCLC is quantitatively linked with the level of NE marker expression and SCLC‐P tumours are characterized by the near‐complete absence of NE differentiation.<a href="#ctm21060-bib-0064" class="usa-link" aria-describedby="ctm21060-bib-0064" aria-expanded="false">
<sup>64</sup>
</a> Therefore, including POU2F3 as a potential additional diagnostic marker might represent an appealing approach for the diagnosis of SCLC tumours that lack or exhibit minimal level of standard NE markers.<a href="#ctm21060-bib-0064" class="usa-link" aria-describedby="ctm21060-bib-0064" aria-expanded="false">
<sup>64</sup>
</a> Meanwhile, in another study, the same group also showed that the expression of conventional markers linked with NE differentiation is substantially higher in ASCL1‐ and NEUROD1‐defined tumours (vs. ASCL1/NEUROD1‐double‐negative tumours), which as well corresponds with our findings.<a href="#ctm21060-bib-0012" class="usa-link" aria-describedby="ctm21060-bib-0012" aria-expanded="false">
<sup>12</sup>
</a> Regarding epithelial and mesenchymal traits, a study on SCLC cell lines<a href="#ctm21060-bib-0065" class="usa-link" aria-describedby="ctm21060-bib-0065" aria-expanded="false">
<sup>65</sup>
</a> outlined that SCLC‐P and ‐Y carry epithelial and mesenchymal attributes, respectively, whereas SCLC‐A and ‐N subtypes have mixed characteristics. Our proteomic study, based on the epithelial and mesenchymal markers established by Kohn et&nbsp;al.,<a href="#ctm21060-bib-0044" class="usa-link" aria-describedby="ctm21060-bib-0044" aria-expanded="false">
<sup>44</sup>
</a> confirmed these observations. Interestingly, the EMT score was not influenced by the cell lines’ site of origin. Our data also showed that strong NE or EMT characteristics tend to be mutually exclusive.</p>
<p>Importantly, we could categorize our SCLC cell lines into four subgroups based on their comprehensive cellular proteomic profiles, which agreed with the pre‐defined mRNA‐based subtypes (only one cell line was misclassified in proteomics). Of note, the overall high variability of cell media proteins compared to cellular proteins probably interfered with the detection of subtype‐specific signatures in this dataset (M<sub>SD in CP</sub>&nbsp;=&nbsp;0.85, M<sub>SD in CM</sub>&nbsp;=&nbsp;1.20). Another driver of the proteomic differences was the culture type, affecting both the CP and CM.</p>
<p>To describe the unique traits of SCLC subtypes, we examined the differentially expressed proteins between subtypes and performed a multi‐omic pathway‐level analysis using the expression differences of all quantified proteins (from our dataset) and transcripts (from CCLE transcriptomic data of SCLC cell lines) across the subtypes. Accordingly, we uncovered a list of potential subtype‐specific therapeutic vulnerabilities.</p>
<p>Due to the well‐known regulatory role of <em>ASCL1</em> in neural differentiation,<a href="#ctm21060-bib-0066" class="usa-link" aria-describedby="ctm21060-bib-0066" aria-expanded="false">
<sup>66</sup>
</a> neural precursor cell proliferation and subpallium development–related proteins showed a concordant upregulation in SCLC‐A subtype. Interestingly, proteomic data clearly indicated that the activation of OXPHOS and respiratory chain elements are strongly specific for SCLC‐A. It has been recently described that cell lines not expressing MYC (characteristic for SCLC‐A<a href="#ctm21060-bib-0006" class="usa-link" aria-describedby="ctm21060-bib-0006" aria-expanded="false">
<sup>6</sup>
</a>, <a href="#ctm21060-bib-0008" class="usa-link" aria-describedby="ctm21060-bib-0008" aria-expanded="false">
<sup>8</sup>
</a>) relies more on oxidative metabolism<a href="#ctm21060-bib-0067" class="usa-link" aria-describedby="ctm21060-bib-0067" aria-expanded="false">
<sup>67</sup>
</a> suggesting that SCLC‐A tumours might be susceptible to OXPHOS inhibitors.<a href="#ctm21060-bib-0068" class="usa-link" aria-describedby="ctm21060-bib-0068" aria-expanded="false">
<sup>68</sup>
</a>
</p>
<p>SCLC‐N cell lines, which predominantly formed suspension cultures in our study, showed a concordant downregulation of cell adhesion pathways. Harvesting the protein‐level data also highlighted the increased activity of DNA replication and transcription and the depletion of cytokine‐mediated signalling in this subtype. Interestingly, a similar trend was described in a study where gene expression differences between SCLC and normal tissue were addressed.<a href="#ctm21060-bib-0069" class="usa-link" aria-describedby="ctm21060-bib-0069" aria-expanded="false">
<sup>69</sup>
</a>
</p>
<p>In the epithelial‐like SCLC‐P cell lines, upregulation of the lamellipodium organization pathway was observed, which constitutes a crucial step in EMT leading to increased cell motility and invasive capacities.<a href="#ctm21060-bib-0070" class="usa-link" aria-describedby="ctm21060-bib-0070" aria-expanded="false">
<sup>70</sup>
</a> In addition, upregulation of neurotrophin signalling foreshadows that poly(adenosine diphosphate‐ribose) polymerase (PARP) inhibitors can be effective therapeutic agents for SCLC‐P as previously proposed,<a href="#ctm21060-bib-0013" class="usa-link" aria-describedby="ctm21060-bib-0013" aria-expanded="false">
<sup>13</sup>
</a> as prolonged PARP activation was found to contribute to neurotrophin‐induced neuronal death.<a href="#ctm21060-bib-0071" class="usa-link" aria-describedby="ctm21060-bib-0071" aria-expanded="false">
<sup>71</sup>
</a> Moreover, our data suggests that direct targeting of neurotrophin signalling might be an appropriate treatment option in <em>POU2F3</em>‐driven SCLC.<a href="#ctm21060-bib-0072" class="usa-link" aria-describedby="ctm21060-bib-0072" aria-expanded="false">
<sup>72</sup>
</a>
</p>
<p>SCLC‐Y cell lines formed a clearly defined subgroup in our SCLC samples. A unique finding in the proteomic data was the downregulation of protein acetylation, chromatin modification (driven by the decreased levels of several histone acetyltransferases) and DNA double‐strand break repair pathways in this subtype. Moreover, we also detected overexpression of the MAPK‐cascade and Rho‐GTPase signalling members in this subtype. Rho is crucial for YAP/TAZ activity<a href="#ctm21060-bib-0073" class="usa-link" aria-describedby="ctm21060-bib-0073" aria-expanded="false">
<sup>73</sup>
</a>; therefore, the upregulation of the latter pathways can be expected in <em>YAP1+</em> cell lines. Recently, Caeser et&nbsp;al. reported that in SCLC‐A, the activation of the MAPK pathway rather has inhibitory functions, compared to SCLC‐N/P cell lines. However, no cell lines from the SCLC‐Y subtype were tested by these authors.<a href="#ctm21060-bib-0074" class="usa-link" aria-describedby="ctm21060-bib-0074" aria-expanded="false">
<sup>74</sup>
</a> In line with the adherent nature of the <em>YAP1</em>‐driven cell lines, we also observed that focal adhesion, ECM organization and cell–substrate protein pathways were uniformly upregulated in the SCLC‐Y subgroup, just as peroxisome and endocytosis‐related proteins. Tlemsani et&nbsp;al. described that SCLC‐Y cell lines demonstrate high presenting and native immune predisposition.<a href="#ctm21060-bib-0065" class="usa-link" aria-describedby="ctm21060-bib-0065" aria-expanded="false">
<sup>65</sup>
</a> Additionally, these cells also have the highest antigen‐presenting machinery scores, thus anticipating that the SCLC‐Y subtype might be sensitive to immune‐checkpoint inhibitors.<a href="#ctm21060-bib-0065" class="usa-link" aria-describedby="ctm21060-bib-0065" aria-expanded="false">
<sup>65</sup>
</a> In line with this, we also identified a distinctive upregulation of cytokine‐mediated signalling and inflammatory response in this subtype.</p>
<p>Of note, controversies around SCLC‐Y still exists, because comprehensive immunohistochemical and histopathologic analyses of SCLC subtypes in patient samples failed to identify a distinct <em>YAP1</em>‐driven subtype.<a href="#ctm21060-bib-0012" class="usa-link" aria-describedby="ctm21060-bib-0012" aria-expanded="false">
<sup>12</sup>
</a> In contrast, our preclinical proteomic study clearly identifies a unique SCLC‐Y subtype among the examined cell lines. Accordingly, a significant proportion of the 104 proteins with diagnostic relevance was linked to SCLC‐Y. The most promising biomarkers for SCLC‐Y include GPX8, PKD2 and UFO, from which UFO is also potentially detectable in the human blood plasma. This is of clinical importance because the lack of appropriate tissue samples highlights the diagnostic relevance of blood‐based biomarkers. Indeed, so far only a single study attempted to define a panel of subtype‐specific blood‐based biomarkers.<a href="#ctm21060-bib-0075" class="usa-link" aria-describedby="ctm21060-bib-0075" aria-expanded="false">
<sup>75</sup>
</a> In their study, however, the authors examined the diagnostic relevance of circulating cell‐free DNA, not proteins.<a href="#ctm21060-bib-0075" class="usa-link" aria-describedby="ctm21060-bib-0075" aria-expanded="false">
<sup>75</sup>
</a> Regarding the potential SCLC‐Y markers identified in our study, gene expression of <em>GPX8</em> was previously correlated with YAP1 expression in SCLC cell lines.<a href="#ctm21060-bib-0076" class="usa-link" aria-describedby="ctm21060-bib-0076" aria-expanded="false">
<sup>76</sup>
</a> Little is known about the role of protein kinase D (PKD) in SCLC, but a previous study has shown that PKD is activated in cell lines H69, H345 and H510 via a protein kinase C‐dependent pathway activation.<a href="#ctm21060-bib-0077" class="usa-link" aria-describedby="ctm21060-bib-0077" aria-expanded="false">
<sup>77</sup>
</a> The <em>AXL</em> gene (encoding the UFO protein) is associated with the mesenchymal phenotype and a potential target for overcoming resistance to epidermal growth factor (EGF) receptor inhibitors.<a href="#ctm21060-bib-0078" class="usa-link" aria-describedby="ctm21060-bib-0078" aria-expanded="false">
<sup>78</sup>
</a>
</p>
<p>Our data outlined six proteins that show remarkable differences in abundance across the subtypes and are also FDA‐approved drug targets, namely DDC (overexpressed in SCLC‐A), EPHA2, ITAV and ITB1 (corresponding to genes <em>EPHA2</em>, <em>ITGAV</em> and <em>ITGB1</em>, overexpressed in SCLC‐Y), HDAC1 (downregulated in SCLC‐Y) and KIT (upregulated in SCLC‐P). Cells with high levels of ASCL1 also showed stronger expression of DDC,<a href="#ctm21060-bib-0079" class="usa-link" aria-describedby="ctm21060-bib-0079" aria-expanded="false">
<sup>79</sup>
</a> whereas EPHA2, a non‐NE marker,<a href="#ctm21060-bib-0007" class="usa-link" aria-describedby="ctm21060-bib-0007" aria-expanded="false">
<sup>7</sup>
</a> was previously described as upregulated in SCLC‐Y.<a href="#ctm21060-bib-0065" class="usa-link" aria-describedby="ctm21060-bib-0065" aria-expanded="false">
<sup>65</sup>
</a> Overexpression of integrins in SCLC‐Y contributes to chemotherapy resistance through the suppression of chemotherapy‐induced apoptosis.<a href="#ctm21060-bib-0080" class="usa-link" aria-describedby="ctm21060-bib-0080" aria-expanded="false">
<sup>80</sup>
</a> In line with this, our group previously reported a positive correlation between YAP1 protein abundance and resistance to chemotherapeutic agents in SCLC cell lines.<a href="#ctm21060-bib-0009" class="usa-link" aria-describedby="ctm21060-bib-0009" aria-expanded="false">
<sup>9</sup>
</a> An HDAC inhibitor resistance in SCLC‐Y, foreshadowed by HDAC1 downregulation in our study, was also reported recently.<a href="#ctm21060-bib-0081" class="usa-link" aria-describedby="ctm21060-bib-0081" aria-expanded="false">
<sup>81</sup>
</a> KIT protein is a known SCLC‐P marker.<a href="#ctm21060-bib-0082" class="usa-link" aria-describedby="ctm21060-bib-0082" aria-expanded="false">
<sup>82</sup>
</a> Importantly, our hypothesis that subtype‐specific overexpression of proteins indicates sensitivity to certain drugs was supported by data from CancerRxGene.<a href="#ctm21060-bib-0040" class="usa-link" aria-describedby="ctm21060-bib-0040" aria-expanded="false">
<sup>40</sup>
</a> In particular, we confirmed that KIT‐targeting drugs, such as pazopanib, are potentially adequate to target <em>POU2F3</em>‐driven SCLCs, whereas <em>YAP1</em>‐driven tumours are more resistant to vorinostat (targeting HDAC1).</p>
<p>Among others, the main advantage of cell line–based studies is that one can examine pure populations of homogeneous tumour cells without admixed stromal or inflammatory cells. This is of scientific importance in drug sensitivity assays and subtyping studies such as the current one.<a href="#ctm21060-bib-0083" class="usa-link" aria-describedby="ctm21060-bib-0083" aria-expanded="false">
<sup>83</sup>
</a>, <a href="#ctm21060-bib-0084" class="usa-link" aria-describedby="ctm21060-bib-0084" aria-expanded="false">
<sup>84</sup>
</a> Importantly, our proteomic data from cell lines showed a notable overlap with transcriptomic data from SCLC tissue samples in terms of subtype‐specific pathways, suggesting that in vitro investigations of SCLC cell lines are useful in characterizing SCLC tissue subtypes. A comprehensive multi‐omic investigation of SCLC tissue and cell line data by Gay et&nbsp;al. demonstrated the same, namely that SCLCs can be subcategorized both in the presence and absence of tumour microenvironment.<a href="#ctm21060-bib-0013" class="usa-link" aria-describedby="ctm21060-bib-0013" aria-expanded="false">
<sup>13</sup>
</a>
</p>
<p>Cell lines, however undoubtedly not ideal models for profiling studies, offer many advantages that complement the use of tumour tissues and animal models for the examination of SCLC.<a href="#ctm21060-bib-0083" class="usa-link" aria-describedby="ctm21060-bib-0083" aria-expanded="false">
<sup>83</sup>
</a> Nevertheless, some study limitations should be underscored. Although we confirmed several previously described subtype markers, and our results are largely consistent with previous transcriptomic data for subtype‐specific pathways, the total number of cell lines included was relatively small. Additionally, given that all <em>YAP1</em>‐driven cell lines had adherent culture type, no subsequent analyses were feasible to investigate subtype and culture type separately. Therefore, the influence of culture type on the distinct characteristics of SCLC‐Y is still unclear. It is also important to note that unique up‐ or downregulation of pathways in a subtype might not translate to dependency or independency from those pathways, as interconnected biological processes and numerous regulatory factors including feedback and feed‐forward loops interfere with such processes. The lack of appropriate treatment‐related data on some cell lines represents another potential study limitation. This is partly because the majority of the examined cell lines were established in the past century, and therefore, the clinicopathological data concerning the tumour they were derived from could not be retrieved even after an in‐depth literature search. Notably, however, neither the NE/mesenchymal characteristics nor the protein expression profile differed significantly according to the presence or absence of chemotherapy (data not shown).</p>
<p>Taken together, our results should primary be considered hypothesis‐generating for future studies, and all finding should be examined in light of the abovementioned limitations. Accordingly, proteomic analyses of larger SCLC cohorts, preferably including fresh tissue samples of patients with homogeneous treatment histories, are needed to validate our findings.</p></section><section id="ctm21060-sec-0330"><h2 class="pmc_sec_title" data-anchor-id="ctm21060-sec-0330"><a id="ctm21060-sec-0330-anchor" data-anchor-id="ctm21060-sec-0330" class="usa-anchor"></a>5. CONCLUSIONS</h2>
<p>To conclude, this is the first large‐scale proteomic study in human SCLC cell lines concerning the newly defined molecular subtypes. We report that SCLC cell lines can be divided into four distinct subtypes by MS‐based proteomics, which accords well with the qPCR‐based classification. Importantly, a distinct <em>YAP1</em>‐driven subtype with specific proteomic features could be also distinguished. Comprehensive proteomic profiling of these subtypes has uncovered a list of candidate subtype‐specific therapeutic vulnerabilities for this once enigmatic cancer. Additionally, we also identified several potential IHC‐ and blood‐based biomarkers that might facilitate subtype diagnosis in the future. Altogether, our results might pave the way for future subtype‐specific personalized therapeutic approaches in SCLC.</p></section><section id="ctm21060-sec-0340"><h2 class="pmc_sec_title" data-anchor-id="ctm21060-sec-0340"><a id="ctm21060-sec-0340-anchor" data-anchor-id="ctm21060-sec-0340" class="usa-anchor"></a>CONFLICT OF INTEREST</h2>
<p>The authors have no conflicts of interest to declare.</p></section><section id="ctm21060-sec-0350"><h2 class="pmc_sec_title" data-anchor-id="ctm21060-sec-0350"><a id="ctm21060-sec-0350-anchor" data-anchor-id="ctm21060-sec-0350" class="usa-anchor"></a>FUNDING INFORMATION</h2>
<p>BS is a recipient of the Semmelweis 250+ Excellence PhD Scholarship (EFOP‐3.6.3‐VEKOP‐16‐2017‐00009). BD, AMS and ZM acknowledge funding from the Hungarian National Research, Development and Innovation Office (KH130356 and KKP126790 to BD; KNN121510 to AMS, 2020‐1.1.6‐JÖVŐ and TKP2021‐EGA‐33 to BD and ZM). BD was also supported by the Austrian Science Fund (FWF I3522, FWF I3977 and I4677). ZM was supported by the UNKP‐20‐3 and UNKP‐21‐3 New National Excellence Program of the Ministry for Innovation and Technology of Hungary, and by the Hungarian Respiratory Society (MPA #2020), PH was supported by the Netherlands X‐omics Initiative (NWO, project 184.034.019). VL is a recipient of the Bolyai Research Scholarship of the Hungarian Academy of Sciences and the UNKP‐19‐4 New National Excellence Program of the Ministry for Innovation and Technology. KS was supported by the Austrian Science Fund (FWF No. T 1062‐B33) and the City of Vienna Fund for Innovative Interdisciplinary Cancer Research. MR acknowledges funding from the Mrs. Berta Kamprad´s Cancer Foundation (FBKS‐2020‐22‐(291)).</p></section><section id="sec36"><h2 class="pmc_sec_title" data-anchor-id="sec36"><a id="sec36-anchor" data-anchor-id="sec36" class="usa-anchor"></a>Supporting information</h2>
<section class="sm xbox font-sm" id="ctm21060-supitem-0001"><div class="caption p"><p>Supporting Information</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/9506422/bin/CTM2-12-e1060-s002.docx" data-ga-action="click_feat_suppl" class="usa-link">Click here for additional data file.</a><sup> (4.1MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="ctm21060-supitem-0002"><div class="caption p"><p>Supporting Information</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/9506422/bin/CTM2-12-e1060-s003.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Click here for additional data file.</a><sup> (30.9KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="ctm21060-supitem-0003"><div class="caption p"><p>Supporting Information</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/9506422/bin/CTM2-12-e1060-s004.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Click here for additional data file.</a><sup> (317.4KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="ctm21060-supitem-0004"><div class="caption p"><p>Supporting Information</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/9506422/bin/CTM2-12-e1060-s001.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Click here for additional data file.</a><sup> (295.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="ctm21060-supitem-0005"><div class="caption p"><p>Supporting Information</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/9506422/bin/CTM2-12-e1060-s005.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Click here for additional data file.</a><sup> (258.1KB, xlsx) </sup>
</div></div></section></section><section id="notes1"><p>

Szeitz B, Megyesfalvi Z, Woldmar N, et&nbsp;al. In‐depth proteomic analysis reveals unique subtype‐specific signatures in human small‐cell lung cancer. Clin Transl Med. 2022;12:e1060. 10.1002/ctm2.1060

</p>
<section id="fn-group1" class="fn-group"><div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="ctm21060-note-0001"><p>Beáta Szeitz and Zsolt Megyesfalvi contributed equally.</p></div>
<div class="fn p" id="ctm21060-note-0002"><p>Balázs Döme, Karin Schelch, and Melinda Rezeli share the last authorship.</p></div>
</div></section></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title" data-anchor-id="_ci93_"><a id="_ci93_-anchor" data-anchor-id="_ci93_" class="usa-anchor"></a>Contributor Information</h2>
<p>Balázs Döme, Email: balazs.dome@meduniwien.ac.at.</p>
<p>Melinda Rezeli, Email: melinda.rezeli@bme.lth.se.</p></section><section id="ctm21060-sec-0370"><h2 class="pmc_sec_title" data-anchor-id="ctm21060-sec-0370"><a id="ctm21060-sec-0370-anchor" data-anchor-id="ctm21060-sec-0370" class="usa-anchor"></a>DATA AVAILABILITY STATEMENT</h2>
<p>The mass spectrometry proteomics data has been deposited to the ProteomeXchange Consortium via the PRIDE<a href="#ctm21060-bib-0085" class="usa-link" aria-describedby="ctm21060-bib-0085" aria-expanded="false">
<sup>85</sup>
</a> partner repository with the dataset identifiers PXD029805 and 10.6019/PXD029805 (cell pellet data), PXD029821 and 10.6019/PXD029821 (culture media data). The scripts for the proteomic data analyses can be obtained at: <a href="https://github.com/bszeitz/SCLC_proteomics" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/bszeitz/SCLC_proteomics</a>.</p></section><section id="ctm21060-bibl-0001" class="ref-list"><h2 class="pmc_sec_title" data-anchor-id="ctm21060-bibl-0001"><a id="ctm21060-bibl-0001-anchor" data-anchor-id="ctm21060-bibl-0001" class="usa-anchor"></a>REFERENCES</h2>
<section id="ctm21060-bibl-0001_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="ctm21060-bib-0001">
<span class="label">1.</span><cite>
Gazdar AF, Bunn PA, Minna JD. Small‐cell lung cancer: what we know, what we need to know and the path forward. Nat Rev Cancer. 2017;17(12):725‐737. doi: 10.1038/nrc.2017.87
</cite> [<a href="https://doi.org/10.1038/nrc.2017.87" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29077690/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Cancer&amp;title=Small%E2%80%90cell%20lung%20cancer:%20what%20we%20know,%20what%20we%20need%20to%20know%20and%20the%20path%20forward&amp;author=AF%20Gazdar&amp;author=PA%20Bunn&amp;author=JD%20Minna&amp;volume=17&amp;issue=12&amp;publication_year=2017&amp;pages=725-737&amp;pmid=29077690&amp;doi=10.1038/nrc.2017.87&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0002">
<span class="label">2.</span><cite>
Kahnert K, Kauffmann‐Guerrero D, Huber RM. SCLC‐state of the art and what does the future have in store?
Clin Lung Cancer. 2016;17(5):325‐333. doi: 10.1016/j.cllc.2016.05.014
</cite> [<a href="https://doi.org/10.1016/j.cllc.2016.05.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27397481/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin%20Lung%20Cancer&amp;title=SCLC%E2%80%90state%20of%20the%20art%20and%20what%20does%20the%20future%20have%20in%20store?&amp;author=K%20Kahnert&amp;author=D%20Kauffmann%E2%80%90Guerrero&amp;author=RM%20Huber&amp;volume=17&amp;issue=5&amp;publication_year=2016&amp;pages=325-333&amp;pmid=27397481&amp;doi=10.1016/j.cllc.2016.05.014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0003">
<span class="label">3.</span><cite>
Schwendenwein A, Megyesfalvi Z, Barany N, et&nbsp;al. Molecular profiles of small cell lung cancer subtypes: therapeutic implications. Mol Ther – Oncolytics. 2021;20:470‐483. doi: 10.1016/j.omto.2021.02.004
</cite> [<a href="https://doi.org/10.1016/j.omto.2021.02.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7917449/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33718595/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Ther%20%E2%80%93%20Oncolytics&amp;title=Molecular%20profiles%20of%20small%20cell%20lung%20cancer%20subtypes:%20therapeutic%20implications&amp;author=A%20Schwendenwein&amp;author=Z%20Megyesfalvi&amp;author=N%20Barany&amp;volume=20&amp;publication_year=2021&amp;pages=470-483&amp;pmid=33718595&amp;doi=10.1016/j.omto.2021.02.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0004">
<span class="label">4.</span><cite>
Rudin CM, Brambilla E, Faivre‐Finn C, Sage J. Small‐cell lung cancer. Nat Rev Dis Primers. 2021;7(1):3. doi: 10.1038/s41572-020-00235-0
</cite> [<a href="https://doi.org/10.1038/s41572-020-00235-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8177722/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33446664/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Dis%20Primers&amp;title=Small%E2%80%90cell%20lung%20cancer&amp;author=CM%20Rudin&amp;author=E%20Brambilla&amp;author=C%20Faivre%E2%80%90Finn&amp;author=J%20Sage&amp;volume=7&amp;issue=1&amp;publication_year=2021&amp;pages=3&amp;pmid=33446664&amp;doi=10.1038/s41572-020-00235-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0005">
<span class="label">5.</span><cite>
Taniguchi H, Sen T, Rudin CM. Targeted therapies and biomarkers in small cell lung cancer. Front Oncol. 2020;10:741. doi: 10.3389/fonc.2020.00741
</cite> [<a href="https://doi.org/10.3389/fonc.2020.00741" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7251180/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32509576/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Oncol&amp;title=Targeted%20therapies%20and%20biomarkers%20in%20small%20cell%20lung%20cancer&amp;author=H%20Taniguchi&amp;author=T%20Sen&amp;author=CM%20Rudin&amp;volume=10&amp;publication_year=2020&amp;pages=741&amp;pmid=32509576&amp;doi=10.3389/fonc.2020.00741&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0006">
<span class="label">6.</span><cite>
Rudin CM, Poirier JT, Byers LA, et&nbsp;al. Molecular subtypes of small cell lung cancer: a synthesis of human and mouse model data. Nat Rev Cancer. 2019;19(5):289‐297. doi: 10.1038/s41568-019-0133-9
</cite> [<a href="https://doi.org/10.1038/s41568-019-0133-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6538259/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30926931/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Cancer&amp;title=Molecular%20subtypes%20of%20small%20cell%20lung%20cancer:%20a%20synthesis%20of%20human%20and%20mouse%20model%20data&amp;author=CM%20Rudin&amp;author=JT%20Poirier&amp;author=LA%20Byers&amp;volume=19&amp;issue=5&amp;publication_year=2019&amp;pages=289-297&amp;pmid=30926931&amp;doi=10.1038/s41568-019-0133-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0007">
<span class="label">7.</span><cite>
Zhang W, Girard L, Zhang Y‐A, et&nbsp;al. Small cell lung cancer tumors and preclinical models display heterogeneity of neuroendocrine phenotypes. Transl Lung Cancer Res. 2018;7:32‐49. doi: 10.21037/tlcr.2018.02.02
</cite> [<a href="https://doi.org/10.21037/tlcr.2018.02.02" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5835590/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29535911/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Transl%20Lung%20Cancer%20Res&amp;title=Small%20cell%20lung%20cancer%20tumors%20and%20preclinical%20models%20display%20heterogeneity%20of%20neuroendocrine%20phenotypes&amp;author=W%20Zhang&amp;author=L%20Girard&amp;author=Y%E2%80%90A%20Zhang&amp;volume=7&amp;publication_year=2018&amp;pages=32-49&amp;pmid=29535911&amp;doi=10.21037/tlcr.2018.02.02&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0008">
<span class="label">8.</span><cite>
George J, Lim JS, Jang SJ, et&nbsp;al. Comprehensive genomic profiles of small cell lung cancer. Nature. 2015;524(7563):47‐53. doi: 10.1038/nature14664
</cite> [<a href="https://doi.org/10.1038/nature14664" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4861069/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26168399/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Comprehensive%20genomic%20profiles%20of%20small%20cell%20lung%20cancer&amp;author=J%20George&amp;author=JS%20Lim&amp;author=SJ%20Jang&amp;volume=524&amp;issue=7563&amp;publication_year=2015&amp;pages=47-53&amp;pmid=26168399&amp;doi=10.1038/nature14664&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0009">
<span class="label">9.</span><cite>
Megyesfalvi Z, Barany N, Lantos A, et&nbsp;al. Expression patterns and prognostic relevance of subtype‐specific transcription factors in surgically resected small‐cell lung cancer: an international multicenter study. J Pathol. 2022;257(5):674‐686. doi: 10.1002/path.5922
</cite> [<a href="https://doi.org/10.1002/path.5922" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9541929/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35489038/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Pathol&amp;title=Expression%20patterns%20and%20prognostic%20relevance%20of%20subtype%E2%80%90specific%20transcription%20factors%20in%20surgically%20resected%20small%E2%80%90cell%20lung%20cancer:%20an%20international%20multicenter%20study&amp;author=Z%20Megyesfalvi&amp;author=N%20Barany&amp;author=A%20Lantos&amp;volume=257&amp;issue=5&amp;publication_year=2022&amp;pages=674-686&amp;pmid=35489038&amp;doi=10.1002/path.5922&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0010">
<span class="label">10.</span><cite>
Drapkin BJ, Rudin CM. Advances in small‐cell lung cancer (SCLC) translational research. Cold Spring Harb Perspect Med. 2021;11(4):a038240. doi: 10.1101/cshperspect.a038240
</cite> [<a href="https://doi.org/10.1101/cshperspect.a038240" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8015694/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32513672/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cold%20Spring%20Harb%20Perspect%20Med&amp;title=Advances%20in%20small%E2%80%90cell%20lung%20cancer%20(SCLC)%20translational%20research&amp;author=BJ%20Drapkin&amp;author=CM%20Rudin&amp;volume=11&amp;issue=4&amp;publication_year=2021&amp;pages=a038240&amp;pmid=32513672&amp;doi=10.1101/cshperspect.a038240&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0011">
<span class="label">11.</span><cite>
Caeser R, Egger JV, Chavan S, et&nbsp;al. Genomic and transcriptomic analysis of a library of small cell lung cancer patient‐derived xenografts. Nat Commun. 2022;13(1):2144. doi: 10.1038/s41467-022-29794-4
</cite> [<a href="https://doi.org/10.1038/s41467-022-29794-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9018685/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35440124/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=Genomic%20and%20transcriptomic%20analysis%20of%20a%20library%20of%20small%20cell%20lung%20cancer%20patient%E2%80%90derived%20xenografts&amp;author=R%20Caeser&amp;author=JV%20Egger&amp;author=S%20Chavan&amp;volume=13&amp;issue=1&amp;publication_year=2022&amp;pages=2144&amp;pmid=35440124&amp;doi=10.1038/s41467-022-29794-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0012">
<span class="label">12.</span><cite>
Baine MK, Hsieh M‐S, Lai WV, et&nbsp;al. SCLC Subtypes Defined by ASCL1, NEUROD1, POU2F3, and YAP1: a comprehensive immunohistochemical and histopathologic characterization. J Thorac Oncol. 2020;15(12):1823‐1835. doi: 10.1016/j.jtho.2020.09.009
</cite> [<a href="https://doi.org/10.1016/j.jtho.2020.09.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8362797/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33011388/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Thorac%20Oncol&amp;title=SCLC%20Subtypes%20Defined%20by%20ASCL1,%20NEUROD1,%20POU2F3,%20and%20YAP1:%20a%20comprehensive%20immunohistochemical%20and%20histopathologic%20characterization&amp;author=MK%20Baine&amp;author=M%E2%80%90S%20Hsieh&amp;author=WV%20Lai&amp;volume=15&amp;issue=12&amp;publication_year=2020&amp;pages=1823-1835&amp;pmid=33011388&amp;doi=10.1016/j.jtho.2020.09.009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0013">
<span class="label">13.</span><cite>
Gay CM, Stewart CA, Park EM, et&nbsp;al. Patterns of transcription factor programs and immune pathway activation define four major subtypes of SCLC with distinct therapeutic vulnerabilities. Cancer Cell. 2021;39(3):346‐360. doi: 10.1016/j.ccell.2020.12.014
</cite> [<a href="https://doi.org/10.1016/j.ccell.2020.12.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8143037/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33482121/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&amp;title=Patterns%20of%20transcription%20factor%20programs%20and%20immune%20pathway%20activation%20define%20four%20major%20subtypes%20of%20SCLC%20with%20distinct%20therapeutic%20vulnerabilities&amp;author=CM%20Gay&amp;author=CA%20Stewart&amp;author=EM%20Park&amp;volume=39&amp;issue=3&amp;publication_year=2021&amp;pages=346-360&amp;pmid=33482121&amp;doi=10.1016/j.ccell.2020.12.014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0014">
<span class="label">14.</span><cite>
Macklin A, Khan S, Kislinger T. Recent advances in mass spectrometry based clinical proteomics: applications to cancer research. Clin Proteomics. 2020;17(1):17. doi: 10.1186/s12014-020-09283-w
</cite> [<a href="https://doi.org/10.1186/s12014-020-09283-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7247207/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32489335/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin%20Proteomics&amp;title=Recent%20advances%20in%20mass%20spectrometry%20based%20clinical%20proteomics:%20applications%20to%20cancer%20research&amp;author=A%20Macklin&amp;author=S%20Khan&amp;author=T%20Kislinger&amp;volume=17&amp;issue=1&amp;publication_year=2020&amp;pages=17&amp;pmid=32489335&amp;doi=10.1186/s12014-020-09283-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0015">
<span class="label">15.</span><cite>
Kelemen O, Pla I, Sanchez A, et&nbsp;al. Proteomic analysis enables distinction of early‐ versus advanced‐stage lung adenocarcinomas. Clin Transl Med. 2020;10(2):e106. doi: 10.1002/ctm2.106
</cite> [<a href="https://doi.org/10.1002/ctm2.106" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7403673/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32536039/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin%20Transl%20Med&amp;title=Proteomic%20analysis%20enables%20distinction%20of%20early%E2%80%90%20versus%20advanced%E2%80%90stage%20lung%20adenocarcinomas&amp;author=O%20Kelemen&amp;author=I%20Pla&amp;author=A%20Sanchez&amp;volume=10&amp;issue=2&amp;publication_year=2020&amp;pages=e106&amp;pmid=32536039&amp;doi=10.1002/ctm2.106&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0016">
<span class="label">16.</span><cite>
Gillette MA, Satpathy S, Cao S, et&nbsp;al. Proteogenomic characterization reveals therapeutic vulnerabilities in lung adenocarcinoma. Cell. 2020;182(1):200‐225. doi: 10.1016/j.cell.2020.06.013
</cite> [<a href="https://doi.org/10.1016/j.cell.2020.06.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7373300/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32649874/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Proteogenomic%20characterization%20reveals%20therapeutic%20vulnerabilities%20in%20lung%20adenocarcinoma&amp;author=MA%20Gillette&amp;author=S%20Satpathy&amp;author=S%20Cao&amp;volume=182&amp;issue=1&amp;publication_year=2020&amp;pages=200-225&amp;pmid=32649874&amp;doi=10.1016/j.cell.2020.06.013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0017">
<span class="label">17.</span><cite>
Satpathy S, Krug K, Jean Beltran PM, et&nbsp;al. A proteogenomic portrait of lung squamous cell carcinoma. Cell. 2021;184(16):4348‐4371.e40. doi: 10.1016/j.cell.2021.07.016
</cite> [<a href="https://doi.org/10.1016/j.cell.2021.07.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8475722/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34358469/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=A%20proteogenomic%20portrait%20of%20lung%20squamous%20cell%20carcinoma&amp;author=S%20Satpathy&amp;author=K%20Krug&amp;author=PM%20Jean%20Beltran&amp;volume=184&amp;issue=16&amp;publication_year=2021&amp;pages=4348-4371.e40&amp;pmid=34358469&amp;doi=10.1016/j.cell.2021.07.016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0018">
<span class="label">18.</span><cite>
Jeong H‐C, Kim G‐I, Cho S‐H, et&nbsp;al. Proteomic analysis of human small cell lung cancer tissues: up‐regulation of coactosin‐like protein‐1. J Proteome Res. 2011;10(1):269‐276. doi: 10.1021/pr100714b
</cite> [<a href="https://doi.org/10.1021/pr100714b" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21047053/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Proteome%20Res&amp;title=Proteomic%20analysis%20of%20human%20small%20cell%20lung%20cancer%20tissues:%20up%E2%80%90regulation%20of%20coactosin%E2%80%90like%20protein%E2%80%901&amp;author=H%E2%80%90C%20Jeong&amp;author=G%E2%80%90I%20Kim&amp;author=S%E2%80%90H%20Cho&amp;volume=10&amp;issue=1&amp;publication_year=2011&amp;pages=269-276&amp;pmid=21047053&amp;doi=10.1021/pr100714b&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0019">
<span class="label">19.</span><cite>
Byers LA, Wang J, Nilsson MB, et&nbsp;al. Proteomic profiling identifies dysregulated pathways in small cell lung cancer and novel therapeutic targets including PARP1. Cancer Discov. 2012;2(9):798‐811. doi: 10.1158/2159-8290.CD-12-0112
</cite> [<a href="https://doi.org/10.1158/2159-8290.CD-12-0112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3567922/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22961666/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Discov&amp;title=Proteomic%20profiling%20identifies%20dysregulated%20pathways%20in%20small%20cell%20lung%20cancer%20and%20novel%20therapeutic%20targets%20including%20PARP1&amp;author=LA%20Byers&amp;author=J%20Wang&amp;author=MB%20Nilsson&amp;volume=2&amp;issue=9&amp;publication_year=2012&amp;pages=798-811&amp;pmid=22961666&amp;doi=10.1158/2159-8290.CD-12-0112&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0020">
<span class="label">20.</span><cite>
Fujii K, Miyata Y, Takahashi I, et&nbsp;al. Differential proteomic analysis between small cell lung carcinoma (SCLC) and pulmonary carcinoid tumors reveals molecular signatures for malignancy in lung cancer. Proteomics Clin Appl. 2018;12(6):e1800015. doi: 10.1002/prca.201800015
</cite> [<a href="https://doi.org/10.1002/prca.201800015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29888431/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proteomics%20Clin%20Appl&amp;title=Differential%20proteomic%20analysis%20between%20small%20cell%20lung%20carcinoma%20(SCLC)%20and%20pulmonary%20carcinoid%20tumors%20reveals%20molecular%20signatures%20for%20malignancy%20in%20lung%20cancer&amp;author=K%20Fujii&amp;author=Y%20Miyata&amp;author=I%20Takahashi&amp;volume=12&amp;issue=6&amp;publication_year=2018&amp;pages=e1800015&amp;pmid=29888431&amp;doi=10.1002/prca.201800015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0021">
<span class="label">21.</span><cite>
Gonçalves E, Poulos RC, Cai Z, et&nbsp;al. Pan‐cancer proteomic map of 949 human cell lines. Cancer Cell. 2022;40(8):835‐849. doi: 10.1016/j.ccell.2022.06.010
</cite> [<a href="https://doi.org/10.1016/j.ccell.2022.06.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9387775/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35839778/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&amp;title=Pan%E2%80%90cancer%20proteomic%20map%20of%20949%20human%20cell%20lines&amp;author=E%20Gon%C3%A7alves&amp;author=RC%20Poulos&amp;author=Z%20Cai&amp;volume=40&amp;issue=8&amp;publication_year=2022&amp;pages=835-849&amp;pmid=35839778&amp;doi=10.1016/j.ccell.2022.06.010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0022">
<span class="label">22.</span><cite>
Chou T‐C, Talalay P. Quantitative analysis of dose‐effect relationships: the combined effects of multiple drugs or enzyme inhibitors. Adv Enzyme Regul. 1984;22:27‐55. doi: 10.1016/0065-2571(84)90007-4
</cite> [<a href="https://doi.org/10.1016/0065-2571(84)90007-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/6382953/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Adv%20Enzyme%20Regul&amp;title=Quantitative%20analysis%20of%20dose%E2%80%90effect%20relationships:%20the%20combined%20effects%20of%20multiple%20drugs%20or%20enzyme%20inhibitors&amp;author=T%E2%80%90C%20Chou&amp;author=P%20Talalay&amp;volume=22&amp;publication_year=1984&amp;pages=27-55&amp;pmid=6382953&amp;doi=10.1016/0065-2571(84)90007-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0023">
<span class="label">23.</span><cite>
Kuras M, Woldmar N, Kim Y, et&nbsp;al. Proteomic workflows for high‐quality quantitative proteome and post‐translational modification analysis of clinically relevant samples from formalin‐fixed paraffin‐embedded archives. J Proteome Res. 2021;20(1):1027‐1039. doi: 10.1021/acs.jproteome.0c00850
</cite> [<a href="https://doi.org/10.1021/acs.jproteome.0c00850" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33301673/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Proteome%20Res&amp;title=Proteomic%20workflows%20for%20high%E2%80%90quality%20quantitative%20proteome%20and%20post%E2%80%90translational%20modification%20analysis%20of%20clinically%20relevant%20samples%20from%20formalin%E2%80%90fixed%20paraffin%E2%80%90embedded%20archives&amp;author=M%20Kuras&amp;author=N%20Woldmar&amp;author=Y%20Kim&amp;volume=20&amp;issue=1&amp;publication_year=2021&amp;pages=1027-1039&amp;pmid=33301673&amp;doi=10.1021/acs.jproteome.0c00850&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0024">
<span class="label">24.</span><cite>
R Core Team
. R: A Language and Environment for Statistical Computing. R Foundation for Statistical Computing; 2022. <a href="https://www.R-project.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.R‐project.org/</a> (accessed on 4 May 2022)</cite> [<a href="https://scholar.google.com/scholar_lookup?title=R:%20A%20Language%20and%20Environment%20for%20Statistical%20Computing&amp;publication_year=2022&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0025">
<span class="label">25.</span><cite>
Tyanova S, Temu T, Sinitcyn P, et&nbsp;al. The Perseus computational platform for comprehensive analysis of (prote)omics data. Nat Methods. 2016;13:731‐740. doi: 10.1038/nmeth.3901
</cite> [<a href="https://doi.org/10.1038/nmeth.3901" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27348712/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&amp;title=The%20Perseus%20computational%20platform%20for%20comprehensive%20analysis%20of%20(prote)omics%20data&amp;author=S%20Tyanova&amp;author=T%20Temu&amp;author=P%20Sinitcyn&amp;volume=13&amp;publication_year=2016&amp;pages=731-740&amp;pmid=27348712&amp;doi=10.1038/nmeth.3901&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0026">
<span class="label">26.</span><cite>
Uhlén M, Karlsson MJ, Hober A, et&nbsp;al. The human secretome. Sci Signal. 2019;12(609):eaaz0274. doi: 10.1126/scisignal.aaz0274
</cite> [<a href="https://doi.org/10.1126/scisignal.aaz0274" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31772123/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Signal&amp;title=The%20human%20secretome&amp;author=M%20Uhl%C3%A9n&amp;author=MJ%20Karlsson&amp;author=A%20Hober&amp;volume=12&amp;issue=609&amp;publication_year=2019&amp;pages=eaaz0274&amp;pmid=31772123&amp;doi=10.1126/scisignal.aaz0274&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0027">
<span class="label">27.</span><cite>
Chen G, Chen J, Liu H, et&nbsp;al. Comprehensive identification and characterization of human secretome based on integrative proteomic and transcriptomic data. Front Cell Dev Biol. 2019;7:299. doi: 10.3389/fcell.2019.00299
</cite> [<a href="https://doi.org/10.3389/fcell.2019.00299" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6881247/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31824949/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Cell%20Dev%20Biol&amp;title=Comprehensive%20identification%20and%20characterization%20of%20human%20secretome%20based%20on%20integrative%20proteomic%20and%20transcriptomic%20data&amp;author=G%20Chen&amp;author=J%20Chen&amp;author=H%20Liu&amp;volume=7&amp;publication_year=2019&amp;pages=299&amp;pmid=31824949&amp;doi=10.3389/fcell.2019.00299&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0028">
<span class="label">28.</span><cite>
Meinken J, Walker G, Cooper CR, Min XJ. MetazSecKB: the human and animal secretome and subcellular proteome knowledgebase. Database (Oxford). 2015;2015:bav077. doi: 10.1093/database/bav077
</cite> [<a href="https://doi.org/10.1093/database/bav077" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4529745/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26255309/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Database%20(Oxford)&amp;title=MetazSecKB:%20the%20human%20and%20animal%20secretome%20and%20subcellular%20proteome%20knowledgebase&amp;author=J%20Meinken&amp;author=G%20Walker&amp;author=CR%20Cooper&amp;author=XJ%20Min&amp;volume=2015&amp;publication_year=2015&amp;pages=bav077&amp;pmid=26255309&amp;doi=10.1093/database/bav077&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0029">
<span class="label">29.</span><cite>
Hu Z, Yuan J, Long M, et&nbsp;al. The Cancer Surfaceome Atlas integrates genomic, functional and drug response data to identify actionable targets. Nat Cancer. 2021;2(12):1406‐1422. doi: 10.1038/s43018-021-00282-w
</cite> [<a href="https://doi.org/10.1038/s43018-021-00282-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9940627/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35121907/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Cancer&amp;title=The%20Cancer%20Surfaceome%20Atlas%20integrates%20genomic,%20functional%20and%20drug%20response%20data%20to%20identify%20actionable%20targets&amp;author=Z%20Hu&amp;author=J%20Yuan&amp;author=M%20Long&amp;volume=2&amp;issue=12&amp;publication_year=2021&amp;pages=1406-1422&amp;pmid=35121907&amp;doi=10.1038/s43018-021-00282-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0030">
<span class="label">30.</span><cite>
Bausch‐Fluck D, Goldmann U, Müller S, et&nbsp;al. The in silico human surfaceome. Proc Natl Acad Sci USA. 2018;115(46):E10988‐E10997. doi: 10.1073/pnas.1808790115
</cite> [<a href="https://doi.org/10.1073/pnas.1808790115" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6243280/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30373828/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&amp;title=The%20in%20silico%20human%20surfaceome&amp;author=D%20Bausch%E2%80%90Fluck&amp;author=U%20Goldmann&amp;author=S%20M%C3%BCller&amp;volume=115&amp;issue=46&amp;publication_year=2018&amp;pages=E10988-E10997&amp;pmid=30373828&amp;doi=10.1073/pnas.1808790115&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0031">
<span class="label">31.</span><cite>
The UniProt Consortium
. UniProt: the universal protein knowledgebase in 2021. Nucleic Acids Res. 2021;49(D1):D480‐D489. doi: 10.1093/nar/gkaa1100
</cite> [<a href="https://doi.org/10.1093/nar/gkaa1100" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7778908/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33237286/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=UniProt:%20the%20universal%20protein%20knowledgebase%20in%202021&amp;volume=49&amp;issue=D1&amp;publication_year=2021&amp;pages=D480-D489&amp;pmid=33237286&amp;doi=10.1093/nar/gkaa1100&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0032">
<span class="label">32.</span><cite>
Wishart DS, Feunang YD, Guo AC, et&nbsp;al. DrugBank 5.0: a major update to the DrugBank database for 2018. Nucleic Acids Res. 2018;46(D1):D1074‐D1082. doi: 10.1093/nar/gkx1037
</cite> [<a href="https://doi.org/10.1093/nar/gkx1037" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5753335/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29126136/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=DrugBank%205.0:%20a%20major%20update%20to%20the%20DrugBank%20database%20for%202018&amp;author=DS%20Wishart&amp;author=YD%20Feunang&amp;author=AC%20Guo&amp;volume=46&amp;issue=D1&amp;publication_year=2018&amp;pages=D1074-D1082&amp;pmid=29126136&amp;doi=10.1093/nar/gkx1037&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0033">
<span class="label">33.</span><cite>
Cerami E, Gao J, Dogrusoz U, et&nbsp;al. The cBio cancer genomics portal: an open platform for exploring multidimensional cancer genomics data. Cancer Discov. 2012;2(5):401‐404. doi: 10.1158/2159-8290.CD-12-0095
</cite> [<a href="https://doi.org/10.1158/2159-8290.CD-12-0095" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3956037/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22588877/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Discov&amp;title=The%20cBio%20cancer%20genomics%20portal:%20an%20open%20platform%20for%20exploring%20multidimensional%20cancer%20genomics%20data&amp;author=E%20Cerami&amp;author=J%20Gao&amp;author=U%20Dogrusoz&amp;volume=2&amp;issue=5&amp;publication_year=2012&amp;pages=401-404&amp;pmid=22588877&amp;doi=10.1158/2159-8290.CD-12-0095&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0034">
<span class="label">34.</span><cite>
Gao J, Aksoy BA, Dogrusoz U, et&nbsp;al. Integrative analysis of complex cancer genomics and clinical profiles using the cBioPortal. Sci Signal. 2013;6(269):pl1. doi: 10.1126/scisignal.2004088
</cite> [<a href="https://doi.org/10.1126/scisignal.2004088" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4160307/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23550210/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Signal&amp;title=Integrative%20analysis%20of%20complex%20cancer%20genomics%20and%20clinical%20profiles%20using%20the%20cBioPortal&amp;author=J%20Gao&amp;author=BA%20Aksoy&amp;author=U%20Dogrusoz&amp;volume=6&amp;issue=269&amp;publication_year=2013&amp;pages=pl1&amp;pmid=23550210&amp;doi=10.1126/scisignal.2004088&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0035">
<span class="label">35.</span><cite>
Ghandi M, Huang FW, Jané‐Valbuena J, et&nbsp;al. Next‐generation characterization of the Cancer Cell Line Encyclopedia. Nature. 2019;569(7757):503‐508. doi: 10.1038/s41586-019-1186-3
</cite> [<a href="https://doi.org/10.1038/s41586-019-1186-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6697103/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31068700/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Next%E2%80%90generation%20characterization%20of%20the%20Cancer%20Cell%20Line%20Encyclopedia&amp;author=M%20Ghandi&amp;author=FW%20Huang&amp;author=J%20Jan%C3%A9%E2%80%90Valbuena&amp;volume=569&amp;issue=7757&amp;publication_year=2019&amp;pages=503-508&amp;pmid=31068700&amp;doi=10.1038/s41586-019-1186-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0036">
<span class="label">36.</span><cite>
Ritchie ME, Phipson B, Wu D, et&nbsp;al. limma powers differential expression analyses for RNA‐sequencing and microarray studies. Nucleic Acids Res. 2015;43(7):e47‐e47. doi: 10.1093/nar/gkv007
</cite> [<a href="https://doi.org/10.1093/nar/gkv007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4402510/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25605792/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=limma%20powers%20differential%20expression%20analyses%20for%20RNA%E2%80%90sequencing%20and%20microarray%20studies&amp;author=ME%20Ritchie&amp;author=B%20Phipson&amp;author=D%20Wu&amp;volume=43&amp;issue=7&amp;publication_year=2015&amp;pages=e47-e47&amp;pmid=25605792&amp;doi=10.1093/nar/gkv007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0037">
<span class="label">37.</span><cite>
Robinson MD, McCarthy DJ, Smyth GK. edgeR: a bioconductor package for differential expression analysis of digital gene expression data. Bioinformatics. 2010;26(1):139‐140. doi: 10.1093/bioinformatics/btp616
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp616" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2796818/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19910308/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=edgeR:%20a%20bioconductor%20package%20for%20differential%20expression%20analysis%20of%20digital%20gene%20expression%20data&amp;author=MD%20Robinson&amp;author=DJ%20McCarthy&amp;author=GK%20Smyth&amp;volume=26&amp;issue=1&amp;publication_year=2010&amp;pages=139-140&amp;pmid=19910308&amp;doi=10.1093/bioinformatics/btp616&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0038">
<span class="label">38.</span><cite>
McCarthy DJ, Chen Y, Smyth GK. Differential expression analysis of multifactor RNA‐Seq experiments with respect to biological variation. Nucleic Acids Res. 2012;40(10):4288‐4297. doi: 10.1093/nar/gks042
</cite> [<a href="https://doi.org/10.1093/nar/gks042" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3378882/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22287627/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=Differential%20expression%20analysis%20of%20multifactor%20RNA%E2%80%90Seq%20experiments%20with%20respect%20to%20biological%20variation&amp;author=DJ%20McCarthy&amp;author=Y%20Chen&amp;author=GK%20Smyth&amp;volume=40&amp;issue=10&amp;publication_year=2012&amp;pages=4288-4297&amp;pmid=22287627&amp;doi=10.1093/nar/gks042&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0039">
<span class="label">39.</span><cite>
Chen Y, Lun ATL, Smyth GK. From reads to genes to pathways: differential expression analysis of RNA‐Seq experiments using Rsubread and the edgeR quasi‐likelihood pipeline [version 2; peer review: 5 approved]. F1000Research. 2016;5:1438. doi: 10.12688/F1000RESEARCH.8987.2
</cite> [<a href="https://doi.org/10.12688/F1000RESEARCH.8987.2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4934518/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27508061/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=F1000Research&amp;title=From%20reads%20to%20genes%20to%20pathways:%20differential%20expression%20analysis%20of%20RNA%E2%80%90Seq%20experiments%20using%20Rsubread%20and%20the%20edgeR%20quasi%E2%80%90likelihood%20pipeline%20%5Bversion%202;%20peer%20review:%205%20approved%5D&amp;author=Y%20Chen&amp;author=ATL%20Lun&amp;author=GK%20Smyth&amp;volume=5&amp;publication_year=2016&amp;pages=1438&amp;pmid=27508061&amp;doi=10.12688/F1000RESEARCH.8987.2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0040">
<span class="label">40.</span><cite>
Yang W, Soares J, Greninger P, et&nbsp;al. Genomics of Drug Sensitivity in Cancer (GDSC): a resource for therapeutic biomarker discovery in cancer cells. Nucleic Acids Res. 2013;41(D1):D955‐D961. doi: 10.1093/nar/gks1111
</cite> [<a href="https://doi.org/10.1093/nar/gks1111" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3531057/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23180760/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=Genomics%20of%20Drug%20Sensitivity%20in%20Cancer%20(GDSC):%20a%20resource%20for%20therapeutic%20biomarker%20discovery%20in%20cancer%20cells&amp;author=W%20Yang&amp;author=J%20Soares&amp;author=P%20Greninger&amp;volume=41&amp;issue=D1&amp;publication_year=2013&amp;pages=D955-D961&amp;pmid=23180760&amp;doi=10.1093/nar/gks1111&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0041">
<span class="label">41.</span><cite>
Yu G, Wang L‐G, Han Y, He Q‐Y. clusterProfiler: an R package for comparing biological themes among gene clusters. OMICS: J Integr Biol. 2012;16(5):284‐287. doi: 10.1089/omi.2011.0118
</cite> [<a href="https://doi.org/10.1089/omi.2011.0118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3339379/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22455463/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=OMICS:%20J%20Integr%20Biol&amp;title=clusterProfiler:%20an%20R%20package%20for%20comparing%20biological%20themes%20among%20gene%20clusters&amp;author=G%20Yu&amp;author=L%E2%80%90G%20Wang&amp;author=Y%20Han&amp;author=Q%E2%80%90Y%20He&amp;volume=16&amp;issue=5&amp;publication_year=2012&amp;pages=284-287&amp;pmid=22455463&amp;doi=10.1089/omi.2011.0118&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0042">
<span class="label">42.</span><cite>
Wu T, Hu E, Xu S, et&nbsp;al. clusterProfiler 4.0: a universal enrichment tool for interpreting omics data. Innovation. 2021;2(3):100141. doi: 10.1016/j.xinn.2021.100141
</cite> [<a href="https://doi.org/10.1016/j.xinn.2021.100141" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8454663/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34557778/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Innovation&amp;title=clusterProfiler%204.0:%20a%20universal%20enrichment%20tool%20for%20interpreting%20omics%20data&amp;author=T%20Wu&amp;author=E%20Hu&amp;author=S%20Xu&amp;volume=2&amp;issue=3&amp;publication_year=2021&amp;pages=100141&amp;pmid=34557778&amp;doi=10.1016/j.xinn.2021.100141&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0043">
<span class="label">43.</span><cite>
Yu G, He Q‐Y. ReactomePA: an R/Bioconductor package for reactome pathway analysis and visualization. Mol Biosyst. 2016;12(2):477‐479. doi: 10.1039/C5MB00663E
</cite> [<a href="https://doi.org/10.1039/C5MB00663E" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26661513/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Biosyst&amp;title=ReactomePA:%20an%20R/Bioconductor%20package%20for%20reactome%20pathway%20analysis%20and%20visualization&amp;author=G%20Yu&amp;author=Q%E2%80%90Y%20He&amp;volume=12&amp;issue=2&amp;publication_year=2016&amp;pages=477-479&amp;pmid=26661513&amp;doi=10.1039/C5MB00663E&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0044">
<span class="label">44.</span><cite>
Kohn KW, Zeeberg BM, Reinhold WC, Pommier Y. Gene expression correlations in human cancer cell lines define molecular interaction networks for epithelial phenotype. PLoS One. 2014;9(6):e99269. doi: 10.1371/journal.pone.0099269
</cite> [<a href="https://doi.org/10.1371/journal.pone.0099269" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4062414/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24940735/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Gene%20expression%20correlations%20in%20human%20cancer%20cell%20lines%20define%20molecular%20interaction%20networks%20for%20epithelial%20phenotype&amp;author=KW%20Kohn&amp;author=BM%20Zeeberg&amp;author=WC%20Reinhold&amp;author=Y%20Pommier&amp;volume=9&amp;issue=6&amp;publication_year=2014&amp;pages=e99269&amp;pmid=24940735&amp;doi=10.1371/journal.pone.0099269&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0045">
<span class="label">45.</span><cite>
Wilkerson MD, Hayes DN. ConsensusClusterPlus: a class discovery tool with confidence assessments and item tracking. Bioinformatics. 2010;26(12):1572‐1573. doi: 10.1093/bioinformatics/btq170
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btq170" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2881355/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20427518/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=ConsensusClusterPlus:%20a%20class%20discovery%20tool%20with%20confidence%20assessments%20and%20item%20tracking&amp;author=MD%20Wilkerson&amp;author=DN%20Hayes&amp;volume=26&amp;issue=12&amp;publication_year=2010&amp;pages=1572-1573&amp;pmid=20427518&amp;doi=10.1093/bioinformatics/btq170&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0046">
<span class="label">46.</span><cite>
Liberzon A, Birger C, Thorvaldsdóttir H, Ghandi M, Mesirov JP, Tamayo P. The Molecular Signatures Database (MSigDB) hallmark gene set collection. Cell Syst. 2015;1(6):417‐425. doi: 10.1016/j.cels.2015.12.004
</cite> [<a href="https://doi.org/10.1016/j.cels.2015.12.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4707969/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26771021/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Syst&amp;title=The%20Molecular%20Signatures%20Database%20(MSigDB)%20hallmark%20gene%20set%20collection&amp;author=A%20Liberzon&amp;author=C%20Birger&amp;author=H%20Thorvaldsd%C3%B3ttir&amp;author=M%20Ghandi&amp;author=JP%20Mesirov&amp;volume=1&amp;issue=6&amp;publication_year=2015&amp;pages=417-425&amp;pmid=26771021&amp;doi=10.1016/j.cels.2015.12.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0047">
<span class="label">47.</span><cite>
Kanehisa M, Goto SKEGG. Kyoto encyclopedia of genes and genomes. Nucleic Acids Res. 2000;28(1):27‐30. doi: 10.1093/nar/28.1.27
</cite> [<a href="https://doi.org/10.1093/nar/28.1.27" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC102409/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10592173/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=Kyoto%20encyclopedia%20of%20genes%20and%20genomes&amp;author=M%20Kanehisa&amp;author=SKEGG%20Goto&amp;volume=28&amp;issue=1&amp;publication_year=2000&amp;pages=27-30&amp;pmid=10592173&amp;doi=10.1093/nar/28.1.27&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0048">
<span class="label">48.</span><cite>
Jassal B, Matthews L, Viteri G, et&nbsp;al. The reactome pathway knowledgebase. Nucleic Acids Res. 2020;48(D1):D498‐D503. doi: 10.1093/nar/gkz1031
</cite> [<a href="https://doi.org/10.1093/nar/gkz1031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7145712/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31691815/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=The%20reactome%20pathway%20knowledgebase&amp;author=B%20Jassal&amp;author=L%20Matthews&amp;author=G%20Viteri&amp;volume=48&amp;issue=D1&amp;publication_year=2020&amp;pages=D498-D503&amp;pmid=31691815&amp;doi=10.1093/nar/gkz1031&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0049">
<span class="label">49.</span><cite>
Ashburner M, Ball CA, Blake JA, et&nbsp;al. Gene ontology: tool for the unification of biology. The Gene Ontology Consortium. Nat Genet. 2000;25(1):25‐29. doi: 10.1038/75556
</cite> [<a href="https://doi.org/10.1038/75556" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3037419/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10802651/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&amp;title=Gene%20ontology:%20tool%20for%20the%20unification%20of%20biology.%20The%20Gene%20Ontology%20Consortium&amp;author=M%20Ashburner&amp;author=CA%20Ball&amp;author=JA%20Blake&amp;volume=25&amp;issue=1&amp;publication_year=2000&amp;pages=25-29&amp;pmid=10802651&amp;doi=10.1038/75556&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0050">
<span class="label">50.</span><cite>
Gene Ontology Consortium
. The gene ontology resource: enriching a GOld mine. Nucleic Acids Res. 2021;49(D1):D325‐D334. doi: 10.1093/nar/gkaa1113
</cite> [<a href="https://doi.org/10.1093/nar/gkaa1113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7779012/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33290552/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=The%20gene%20ontology%20resource:%20enriching%20a%20GOld%20mine&amp;volume=49&amp;issue=D1&amp;publication_year=2021&amp;pages=D325-D334&amp;pmid=33290552&amp;doi=10.1093/nar/gkaa1113&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0051">
<span class="label">51.</span><cite>
Subramanian A, Tamayo P, Mootha VK, et&nbsp;al. Gene set enrichment analysis: a knowledge‐based approach for interpreting genome‐wide expression profiles. Proc Natl Acad Sci USA. 2005;102(43):15545‐15550. doi: 10.1073/pnas.0506580102
</cite> [<a href="https://doi.org/10.1073/pnas.0506580102" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1239896/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16199517/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&amp;title=Gene%20set%20enrichment%20analysis:%20a%20knowledge%E2%80%90based%20approach%20for%20interpreting%20genome%E2%80%90wide%20expression%20profiles&amp;author=A%20Subramanian&amp;author=P%20Tamayo&amp;author=VK%20Mootha&amp;volume=102&amp;issue=43&amp;publication_year=2005&amp;pages=15545-15550&amp;pmid=16199517&amp;doi=10.1073/pnas.0506580102&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0052">
<span class="label">52.</span><cite>
Liberzon A, Subramanian A, Pinchback R, Thorvaldsdóttir H, Tamayo P, Mesirov JP. Molecular signatures database (MSigDB) 3.0. Bioinformatics. 2011;27(12):1739‐1740. doi: 10.1093/bioinformatics/btr260
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btr260" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3106198/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21546393/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Molecular%20signatures%20database%20(MSigDB)%203.0&amp;author=A%20Liberzon&amp;author=A%20Subramanian&amp;author=R%20Pinchback&amp;author=H%20Thorvaldsd%C3%B3ttir&amp;author=P%20Tamayo&amp;volume=27&amp;issue=12&amp;publication_year=2011&amp;pages=1739-1740&amp;pmid=21546393&amp;doi=10.1093/bioinformatics/btr260&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0053">
<span class="label">53.</span><cite>
Barbie DA, Tamayo P, Boehm JS, et&nbsp;al. Systematic RNA interference reveals that oncogenic KRAS‐driven cancers require TBK1. Nature. 2009;462(7269):108‐112. doi: 10.1038/nature08460
</cite> [<a href="https://doi.org/10.1038/nature08460" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2783335/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19847166/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Systematic%20RNA%20interference%20reveals%20that%20oncogenic%20KRAS%E2%80%90driven%20cancers%20require%20TBK1&amp;author=DA%20Barbie&amp;author=P%20Tamayo&amp;author=JS%20Boehm&amp;volume=462&amp;issue=7269&amp;publication_year=2009&amp;pages=108-112&amp;pmid=19847166&amp;doi=10.1038/nature08460&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0054">
<span class="label">54.</span><cite>
Rohart F, Gautier B, Singh A. Lê Cao K‐A. mixOmics: an R package for ‘omics feature selection and multiple data integration. PLoS Comput Biol. 2017;13(11):e1005752. doi: 10.1371/journal.pcbi.1005752
</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1005752" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5687754/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29099853/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Comput%20Biol&amp;title=L%C3%AA%20Cao%20K%E2%80%90A.%20mixOmics:%20an%20R%20package%20for%20%E2%80%98omics%20feature%20selection%20and%20multiple%20data%20integration&amp;author=F%20Rohart&amp;author=B%20Gautier&amp;author=A%20Singh&amp;volume=13&amp;issue=11&amp;publication_year=2017&amp;pages=e1005752&amp;pmid=29099853&amp;doi=10.1371/journal.pcbi.1005752&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0055">
<span class="label">55.</span><cite>
Gu Z, Eils R, Schlesner M. Complex heatmaps reveal patterns and correlations in multidimensional genomic data. Bioinformatics. 2016;32(18):2847‐2849. doi: 10.1093/bioinformatics/btw313
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btw313" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27207943/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Complex%20heatmaps%20reveal%20patterns%20and%20correlations%20in%20multidimensional%20genomic%20data&amp;author=Z%20Gu&amp;author=R%20Eils&amp;author=M%20Schlesner&amp;volume=32&amp;issue=18&amp;publication_year=2016&amp;pages=2847-2849&amp;pmid=27207943&amp;doi=10.1093/bioinformatics/btw313&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0056">
<span class="label">56.</span><cite>
Denny SK, Yang D, Chuang C‐H, et&nbsp;al. Nfib promotes metastasis through a widespread increase in chromatin accessibility. Cell. 2016;166(2):328‐342. doi: 10.1016/j.cell.2016.05.052
</cite> [<a href="https://doi.org/10.1016/j.cell.2016.05.052" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5004630/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27374332/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Nfib%20promotes%20metastasis%20through%20a%20widespread%20increase%20in%20chromatin%20accessibility&amp;author=SK%20Denny&amp;author=D%20Yang&amp;author=C%E2%80%90H%20Chuang&amp;volume=166&amp;issue=2&amp;publication_year=2016&amp;pages=328-342&amp;pmid=27374332&amp;doi=10.1016/j.cell.2016.05.052&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0057">
<span class="label">57.</span><cite>
Mori N, Yokota J, Akiyama T, et&nbsp;al. Variable mutations of the RB gene in small‐cell lung carcinoma. Oncogene. 1990;5(11):1713‐1717.</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/2176283/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=Variable%20mutations%20of%20the%20RB%20gene%20in%20small%E2%80%90cell%20lung%20carcinoma&amp;author=N%20Mori&amp;author=J%20Yokota&amp;author=T%20Akiyama&amp;volume=5&amp;issue=11&amp;publication_year=1990&amp;pages=1713-1717&amp;pmid=2176283&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0058">
<span class="label">58.</span><cite>
Wistuba II, Gazdar AF, Minna JD. Molecular genetics of small cell lung carcinoma. Semin Oncol. 2001;28(2)(suppl 4):3‐13.</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/11479891/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Semin%20Oncol&amp;title=Molecular%20genetics%20of%20small%20cell%20lung%20carcinoma&amp;author=II%20Wistuba&amp;author=AF%20Gazdar&amp;author=JD%20Minna&amp;volume=28&amp;issue=2&amp;publication_year=2001&amp;pages=3-13&amp;pmid=11479891&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0059">
<span class="label">59.</span><cite>
Arriola E, Cañadas I, Arumí M, Rojo F, Rovira A, Albanell J. Genetic changes in small cell lung carcinoma. Clin Transl Oncol. 2008;10(4):189‐197. doi: 10.1007/s12094-008-0181-1
</cite> [<a href="https://doi.org/10.1007/s12094-008-0181-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18411191/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin%20Transl%20Oncol&amp;title=Genetic%20changes%20in%20small%20cell%20lung%20carcinoma&amp;author=E%20Arriola&amp;author=I%20Ca%C3%B1adas&amp;author=M%20Arum%C3%AD&amp;author=F%20Rojo&amp;author=A%20Rovira&amp;volume=10&amp;issue=4&amp;publication_year=2008&amp;pages=189-197&amp;pmid=18411191&amp;doi=10.1007/s12094-008-0181-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0060">
<span class="label">60.</span><cite>
Sabari JK, Lok BH, Laird JH, Poirier JT, Rudin CM. Unravelling the biology of SCLC: implications for therapy. Nat Rev Clin Oncol. 2017;14(9):549‐561. doi: 10.1038/nrclinonc.2017.71
</cite> [<a href="https://doi.org/10.1038/nrclinonc.2017.71" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5843484/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28534531/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Clin%20Oncol&amp;title=Unravelling%20the%20biology%20of%20SCLC:%20implications%20for%20therapy&amp;author=JK%20Sabari&amp;author=BH%20Lok&amp;author=JH%20Laird&amp;author=JT%20Poirier&amp;author=CM%20Rudin&amp;volume=14&amp;issue=9&amp;publication_year=2017&amp;pages=549-561&amp;pmid=28534531&amp;doi=10.1038/nrclinonc.2017.71&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0061">
<span class="label">61.</span><cite>
Uhlén M, Fagerberg L, Hallström BM, et&nbsp;al. Tissue‐based map of the human proteome. Science (80–). 2015;347(6220):1260419. doi: 10.1126/science.1260419
</cite> [<a href="https://doi.org/10.1126/science.1260419" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25613900/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science%20(80%E2%80%93)&amp;title=Tissue%E2%80%90based%20map%20of%20the%20human%20proteome&amp;author=M%20Uhl%C3%A9n&amp;author=L%20Fagerberg&amp;author=BM%20Hallstr%C3%B6m&amp;volume=347&amp;issue=6220&amp;publication_year=2015&amp;pages=1260419&amp;pmid=25613900&amp;doi=10.1126/science.1260419&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0062">
<span class="label">62.</span><cite>
Carney DN, Gazdar AF, Bepler G, et&nbsp;al. Establishment and identification of small cell lung cancer cell lines having classic and variant features. Cancer Res. 1985;45(6):2913‐2923.</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/2985257/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&amp;title=Establishment%20and%20identification%20of%20small%20cell%20lung%20cancer%20cell%20lines%20having%20classic%20and%20variant%20features&amp;author=DN%20Carney&amp;author=AF%20Gazdar&amp;author=G%20Bepler&amp;volume=45&amp;issue=6&amp;publication_year=1985&amp;pages=2913-2923&amp;pmid=2985257&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0063">
<span class="label">63.</span><cite>
Gazdar AF, Carney DN, Nau MM, Minna JD. Characterization of variant subclasses of cell lines derived from small cell lung cancer having distinctive biochemical, morphological, and growth properties. Cancer Res. 1985;45(6):2924‐2930.</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/2985258/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&amp;title=Characterization%20of%20variant%20subclasses%20of%20cell%20lines%20derived%20from%20small%20cell%20lung%20cancer%20having%20distinctive%20biochemical,%20morphological,%20and%20growth%20properties&amp;author=AF%20Gazdar&amp;author=DN%20Carney&amp;author=MM%20Nau&amp;author=JD%20Minna&amp;volume=45&amp;issue=6&amp;publication_year=1985&amp;pages=2924-2930&amp;pmid=2985258&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0064">
<span class="label">64.</span><cite>
Baine MK, Febres‐Aldana CA, Chang JC, et&nbsp;al. POU2F3 in SCLC: clinicopathologic and genomic analysis with a focus on its diagnostic utility in neuroendocrine‐low SCLC. J Thorac Oncol. 2022;17:1109‐1121. doi: 10.1016/j.jtho.2022.06.004
</cite> [<a href="https://doi.org/10.1016/j.jtho.2022.06.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9427708/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35760287/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Thorac%20Oncol&amp;title=POU2F3%20in%20SCLC:%20clinicopathologic%20and%20genomic%20analysis%20with%20a%20focus%20on%20its%20diagnostic%20utility%20in%20neuroendocrine%E2%80%90low%20SCLC&amp;author=MK%20Baine&amp;author=CA%20Febres%E2%80%90Aldana&amp;author=JC%20Chang&amp;volume=17&amp;publication_year=2022&amp;pages=1109-1121&amp;pmid=35760287&amp;doi=10.1016/j.jtho.2022.06.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0065">
<span class="label">65.</span><cite>
Tlemsani C, Pongor L, Elloumi F, et&nbsp;al. SCLC‐CellMiner: a resource for small cell lung cancer cell line genomics and pharmacology based on genomic signatures. Cell Rep. 2020;33(3):108296. doi: 10.1016/j.celrep.2020.108296
</cite> [<a href="https://doi.org/10.1016/j.celrep.2020.108296" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7643325/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33086069/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Rep&amp;title=SCLC%E2%80%90CellMiner:%20a%20resource%20for%20small%20cell%20lung%20cancer%20cell%20line%20genomics%20and%20pharmacology%20based%20on%20genomic%20signatures&amp;author=C%20Tlemsani&amp;author=L%20Pongor&amp;author=F%20Elloumi&amp;volume=33&amp;issue=3&amp;publication_year=2020&amp;pages=108296&amp;pmid=33086069&amp;doi=10.1016/j.celrep.2020.108296&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0066">
<span class="label">66.</span><cite>
Guillemot F, Hassan BA. Beyond proneural: emerging functions and regulations of proneural proteins. Curr Opin Neurobiol. 2017;42:93‐101. doi: 10.1016/j.conb.2016.11.011
</cite> [<a href="https://doi.org/10.1016/j.conb.2016.11.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28025176/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Neurobiol&amp;title=Beyond%20proneural:%20emerging%20functions%20and%20regulations%20of%20proneural%20proteins&amp;author=F%20Guillemot&amp;author=BA%20Hassan&amp;volume=42&amp;publication_year=2017&amp;pages=93-101&amp;pmid=28025176&amp;doi=10.1016/j.conb.2016.11.011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0067">
<span class="label">67.</span><cite>
Cargill KR, Stewart CA, Park EM, et&nbsp;al. Targeting MYC‐enhanced glycolysis for the treatment of small cell lung cancer. Cancer Metab. 2021;9(1):33. doi: 10.1186/s40170-021-00270-9
</cite> [<a href="https://doi.org/10.1186/s40170-021-00270-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8461854/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34556188/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Metab&amp;title=Targeting%20MYC%E2%80%90enhanced%20glycolysis%20for%20the%20treatment%20of%20small%20cell%20lung%20cancer&amp;author=KR%20Cargill&amp;author=CA%20Stewart&amp;author=EM%20Park&amp;volume=9&amp;issue=1&amp;publication_year=2021&amp;pages=33&amp;pmid=34556188&amp;doi=10.1186/s40170-021-00270-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0068">
<span class="label">68.</span><cite>
Sica V, Bravo‐San Pedro JM, Stoll G, Kroemer G. Oxidative phosphorylation as a potential therapeutic target for cancer therapy. Int J Cancer. 2020;146(1):10‐17. doi: 10.1002/ijc.32616
</cite> [<a href="https://doi.org/10.1002/ijc.32616" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31396957/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Cancer&amp;title=Oxidative%20phosphorylation%20as%20a%20potential%20therapeutic%20target%20for%20cancer%20therapy&amp;author=V%20Sica&amp;author=JM%20Bravo%E2%80%90San%20Pedro&amp;author=G%20Stoll&amp;author=G%20Kroemer&amp;volume=146&amp;issue=1&amp;publication_year=2020&amp;pages=10-17&amp;pmid=31396957&amp;doi=10.1002/ijc.32616&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0069">
<span class="label">69.</span><cite>
Liao Y, Yin G, Wang X, Zhong P, Fan X, Huang C. Identification of candidate genes associated with the pathogenesis of small cell lung cancer via integrated bioinformatics analysis. Oncol Lett. 2019;18(4):3723‐3733. doi: 10.3892/ol.2019.10685
</cite> [<a href="https://doi.org/10.3892/ol.2019.10685" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6732946/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31516585/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncol%20Lett&amp;title=Identification%20of%20candidate%20genes%20associated%20with%20the%20pathogenesis%20of%20small%20cell%20lung%20cancer%20via%20integrated%20bioinformatics%20analysis&amp;author=Y%20Liao&amp;author=G%20Yin&amp;author=X%20Wang&amp;author=P%20Zhong&amp;author=X%20Fan&amp;volume=18&amp;issue=4&amp;publication_year=2019&amp;pages=3723-3733&amp;pmid=31516585&amp;doi=10.3892/ol.2019.10685&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0070">
<span class="label">70.</span><cite>
Lamouille S, Xu J, Derynck R. Molecular mechanisms of epithelial–mesenchymal transition. Nat Rev Mol Cell Biol. 2014;15(3):178‐196. doi: 10.1038/nrm3758
</cite> [<a href="https://doi.org/10.1038/nrm3758" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4240281/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24556840/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Mol%20Cell%20Biol&amp;title=Molecular%20mechanisms%20of%20epithelial%E2%80%93mesenchymal%20transition&amp;author=S%20Lamouille&amp;author=J%20Xu&amp;author=R%20Derynck&amp;volume=15&amp;issue=3&amp;publication_year=2014&amp;pages=178-196&amp;pmid=24556840&amp;doi=10.1038/nrm3758&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0071">
<span class="label">71.</span><cite>
Hwang J‐J, Choi S‐Y, Koh J‐Y. The role of NADPH oxidase, neuronal nitric oxide synthase and poly(ADP ribose) polymerase in oxidative neuronal death induced in cortical cultures by brain‐derived neurotrophic factor and neurotrophin‐4/5. J Neurochem. 2002;82(4):894‐902. doi: 10.1046/j.1471-4159.2002.01040.x
</cite> [<a href="https://doi.org/10.1046/j.1471-4159.2002.01040.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12358795/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Neurochem&amp;title=The%20role%20of%20NADPH%20oxidase,%20neuronal%20nitric%20oxide%20synthase%20and%20poly(ADP%20ribose)%20polymerase%20in%20oxidative%20neuronal%20death%20induced%20in%20cortical%20cultures%20by%20brain%E2%80%90derived%20neurotrophic%20factor%20and%20neurotrophin%E2%80%904/5&amp;author=J%E2%80%90J%20Hwang&amp;author=S%E2%80%90Y%20Choi&amp;author=J%E2%80%90Y%20Koh&amp;volume=82&amp;issue=4&amp;publication_year=2002&amp;pages=894-902&amp;pmid=12358795&amp;doi=10.1046/j.1471-4159.2002.01040.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0072">
<span class="label">72.</span><cite>
Griffin N, Faulkner S, Jobling P, Hondermarck H. Targeting neurotrophin signaling in cancer: the renaissance. Pharmacol Res. 2018;135:12‐17. doi: 10.1016/j.phrs.2018.07.019
</cite> [<a href="https://doi.org/10.1016/j.phrs.2018.07.019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30031169/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacol%20Res&amp;title=Targeting%20neurotrophin%20signaling%20in%20cancer:%20the%20renaissance&amp;author=N%20Griffin&amp;author=S%20Faulkner&amp;author=P%20Jobling&amp;author=H%20Hondermarck&amp;volume=135&amp;publication_year=2018&amp;pages=12-17&amp;pmid=30031169&amp;doi=10.1016/j.phrs.2018.07.019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0073">
<span class="label">73.</span><cite>
Dupont S, Morsut L, Aragona M, et&nbsp;al. Role of YAP/TAZ in mechanotransduction. Nature. 2011;474(7350):179‐183. doi: 10.1038/nature10137
</cite> [<a href="https://doi.org/10.1038/nature10137" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21654799/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Role%20of%20YAP/TAZ%20in%20mechanotransduction&amp;author=S%20Dupont&amp;author=L%20Morsut&amp;author=M%20Aragona&amp;volume=474&amp;issue=7350&amp;publication_year=2011&amp;pages=179-183&amp;pmid=21654799&amp;doi=10.1038/nature10137&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0074">
<span class="label">74.</span><cite>
Caeser R, Hulton C, Costa E, et&nbsp;al. MAPK pathway activation selectively inhibits ASCL1‐driven small cell lung cancer. iScience. 2021;24(11):103224. doi: 10.1016/j.isci.2021.103224
</cite> [<a href="https://doi.org/10.1016/j.isci.2021.103224" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8528729/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34712921/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=iScience&amp;title=MAPK%20pathway%20activation%20selectively%20inhibits%20ASCL1%E2%80%90driven%20small%20cell%20lung%20cancer&amp;author=R%20Caeser&amp;author=C%20Hulton&amp;author=E%20Costa&amp;volume=24&amp;issue=11&amp;publication_year=2021&amp;pages=103224&amp;pmid=34712921&amp;doi=10.1016/j.isci.2021.103224&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0075">
<span class="label">75.</span><cite>
Chemi F, Pearce SP, Clipson A, et&nbsp;al. cfDNA methylome profiling for detection and subtyping of small cell lung cancer. Nat Cancer. 2022. doi: 10.1038/s43018-022-00415-9
</cite> [<a href="https://doi.org/10.1038/s43018-022-00415-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9586870/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35941262/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Cancer&amp;title=cfDNA%20methylome%20profiling%20for%20detection%20and%20subtyping%20of%20small%20cell%20lung%20cancer&amp;author=F%20Chemi&amp;author=SP%20Pearce&amp;author=A%20Clipson&amp;publication_year=2022&amp;pmid=35941262&amp;doi=10.1038/s43018-022-00415-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0076">
<span class="label">76.</span><cite>
Horie M, Saito A, Ohshima M, Suzuki HI, Nagase T. YAP and TAZ modulate cell phenotype in a subset of small cell lung cancer. Cancer Sci. 2016;107(12):1755‐1766. doi: 10.1111/cas.13078
</cite> [<a href="https://doi.org/10.1111/cas.13078" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5198951/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27627196/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Sci&amp;title=YAP%20and%20TAZ%20modulate%20cell%20phenotype%20in%20a%20subset%20of%20small%20cell%20lung%20cancer&amp;author=M%20Horie&amp;author=A%20Saito&amp;author=M%20Ohshima&amp;author=HI%20Suzuki&amp;author=T%20Nagase&amp;volume=107&amp;issue=12&amp;publication_year=2016&amp;pages=1755-1766&amp;pmid=27627196&amp;doi=10.1111/cas.13078&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0077">
<span class="label">77.</span><cite>
Paolucci L, Rozengurt E. Protein kinase D in small cell lung cancer cells: rapid activation through protein kinase C. Cancer Res. 1999;59(3):572‐577.</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/9973202/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&amp;title=Protein%20kinase%20D%20in%20small%20cell%20lung%20cancer%20cells:%20rapid%20activation%20through%20protein%20kinase%20C&amp;author=L%20Paolucci&amp;author=E%20Rozengurt&amp;volume=59&amp;issue=3&amp;publication_year=1999&amp;pages=572-577&amp;pmid=9973202&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0078">
<span class="label">78.</span><cite>
Byers LA, Diao L, Wang J, et&nbsp;al. An epithelial–mesenchymal transition gene signature predicts resistance to EGFR and PI3K inhibitors and identifies Axl as a therapeutic target for overcoming EGFR inhibitor resistance. Clin Cancer Res. 2013;19(1):279‐290. doi: 10.1158/1078-0432.CCR-12-1558
</cite> [<a href="https://doi.org/10.1158/1078-0432.CCR-12-1558" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3567921/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23091115/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&amp;title=An%20epithelial%E2%80%93mesenchymal%20transition%20gene%20signature%20predicts%20resistance%20to%20EGFR%20and%20PI3K%20inhibitors%20and%20identifies%20Axl%20as%20a%20therapeutic%20target%20for%20overcoming%20EGFR%20inhibitor%20resistance&amp;author=LA%20Byers&amp;author=L%20Diao&amp;author=J%20Wang&amp;volume=19&amp;issue=1&amp;publication_year=2013&amp;pages=279-290&amp;pmid=23091115&amp;doi=10.1158/1078-0432.CCR-12-1558&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0079">
<span class="label">79.</span><cite>
Borromeo MD, Savage TK, Kollipara RK, et&nbsp;al. ASCL1 and NEUROD1 reveal heterogeneity in pulmonary neuroendocrine tumors and regulate distinct genetic programs. Cell Rep. 2016;16(5):1259‐1272. doi: 10.1016/j.celrep.2016.06.081
</cite> [<a href="https://doi.org/10.1016/j.celrep.2016.06.081" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4972690/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27452466/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Rep&amp;title=ASCL1%20and%20NEUROD1%20reveal%20heterogeneity%20in%20pulmonary%20neuroendocrine%20tumors%20and%20regulate%20distinct%20genetic%20programs&amp;author=MD%20Borromeo&amp;author=TK%20Savage&amp;author=RK%20Kollipara&amp;volume=16&amp;issue=5&amp;publication_year=2016&amp;pages=1259-1272&amp;pmid=27452466&amp;doi=10.1016/j.celrep.2016.06.081&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0080">
<span class="label">80.</span><cite>
Sethi T, Rintoul RC, Moore SM, et&nbsp;al. Extracellular matrix proteins protect small cell lung cancer cells against apoptosis: a mechanism for small cell lung cancer growth and drug resistance in vivo. Nat Med. 1999;5(6):662‐668. doi: 10.1038/9511
</cite> [<a href="https://doi.org/10.1038/9511" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10371505/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Med&amp;title=Extracellular%20matrix%20proteins%20protect%20small%20cell%20lung%20cancer%20cells%20against%20apoptosis:%20a%20mechanism%20for%20small%20cell%20lung%20cancer%20growth%20and%20drug%20resistance%20in%20vivo&amp;author=T%20Sethi&amp;author=RC%20Rintoul&amp;author=SM%20Moore&amp;volume=5&amp;issue=6&amp;publication_year=1999&amp;pages=662-668&amp;pmid=10371505&amp;doi=10.1038/9511&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0081">
<span class="label">81.</span><cite>
Zhao P, Sun X, Li H, et&nbsp;al. c‐Myc targets HDAC3 to suppress NKG2DL expression and innate immune response in N‐Type SCLC through histone deacetylation. Cancers (Basel). 2022;14(3). doi: 10.3390/cancers14030457
</cite> [<a href="https://doi.org/10.3390/cancers14030457" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8833590/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35158730/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancers%20(Basel)&amp;title=c%E2%80%90Myc%20targets%20HDAC3%20to%20suppress%20NKG2DL%20expression%20and%20innate%20immune%20response%20in%20N%E2%80%90Type%20SCLC%20through%20histone%20deacetylation&amp;author=P%20Zhao&amp;author=X%20Sun&amp;author=H%20Li&amp;volume=14&amp;issue=3&amp;publication_year=2022&amp;pmid=35158730&amp;doi=10.3390/cancers14030457&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0082">
<span class="label">82.</span><cite>
Chan JM, Quintanal‐Villalonga Á, Gao VR, et&nbsp;al. Signatures of plasticity, metastasis, and immunosuppression in an atlas of human small cell lung cancer. Cancer Cell. 2021;39(11):1479‐1496. doi: 10.1016/j.ccell.2021.09.008
</cite> [<a href="https://doi.org/10.1016/j.ccell.2021.09.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8628860/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34653364/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&amp;title=Signatures%20of%20plasticity,%20metastasis,%20and%20immunosuppression%20in%20an%20atlas%20of%20human%20small%20cell%20lung%20cancer&amp;author=JM%20Chan&amp;author=%C3%81%20Quintanal%E2%80%90Villalonga&amp;author=VR%20Gao&amp;volume=39&amp;issue=11&amp;publication_year=2021&amp;pages=1479-1496&amp;pmid=34653364&amp;doi=10.1016/j.ccell.2021.09.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0083">
<span class="label">83.</span><cite>
Gazdar AF, Gao B, Minna JD. Lung cancer cell lines: useless artifacts or invaluable tools for medical science?
Lung Cancer. 2010;68(3):309‐318. doi: 10.1016/j.lungcan.2009.12.005
</cite> [<a href="https://doi.org/10.1016/j.lungcan.2009.12.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3110769/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20079948/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Lung%20Cancer&amp;title=Lung%20cancer%20cell%20lines:%20useless%20artifacts%20or%20invaluable%20tools%20for%20medical%20science?&amp;author=AF%20Gazdar&amp;author=B%20Gao&amp;author=JD%20Minna&amp;volume=68&amp;issue=3&amp;publication_year=2010&amp;pages=309-318&amp;pmid=20079948&amp;doi=10.1016/j.lungcan.2009.12.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0084">
<span class="label">84.</span><cite>
van Staveren WCG, Solís DYW, Hébrant A, Detours V, Dumont JE, Maenhaut C. Human cancer cell lines: experimental models for cancer cells in situ? For cancer stem cells?
Biochim Biophys Acta. 2009;1795(2):92‐103. doi: 10.1016/j.bbcan.2008.12.004
</cite> [<a href="https://doi.org/10.1016/j.bbcan.2008.12.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19167460/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochim%20Biophys%20Acta&amp;title=Human%20cancer%20cell%20lines:%20experimental%20models%20for%20cancer%20cells%20in%20situ?%20For%20cancer%20stem%20cells?&amp;author=WCG%20van%20Staveren&amp;author=DYW%20Sol%C3%ADs&amp;author=A%20H%C3%A9brant&amp;author=V%20Detours&amp;author=JE%20Dumont&amp;volume=1795&amp;issue=2&amp;publication_year=2009&amp;pages=92-103&amp;pmid=19167460&amp;doi=10.1016/j.bbcan.2008.12.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ctm21060-bib-0085">
<span class="label">85.</span><cite>
Perez‐Riverol Y, Csordas A, Bai J, et&nbsp;al. The PRIDE database and related tools and resources in 2019: improving support for quantification data. Nucleic Acids Res. 2019;47(D1):D442‐D450. doi: 10.1093/nar/gky1106
</cite> [<a href="https://doi.org/10.1093/nar/gky1106" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6323896/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30395289/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=The%20PRIDE%20database%20and%20related%20tools%20and%20resources%20in%202019:%20improving%20support%20for%20quantification%20data&amp;author=Y%20Perez%E2%80%90Riverol&amp;author=A%20Csordas&amp;author=J%20Bai&amp;volume=47&amp;issue=D1&amp;publication_year=2019&amp;pages=D442-D450&amp;pmid=30395289&amp;doi=10.1093/nar/gky1106&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title" data-anchor-id="_ad93_"><a id="_ad93_-anchor" data-anchor-id="_ad93_" class="usa-anchor"></a>Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p"><p>Supporting Information</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/9506422/bin/CTM2-12-e1060-s002.docx" data-ga-action="click_feat_suppl" class="usa-link">Click here for additional data file.</a><sup> (4.1MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p"><p>Supporting Information</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/9506422/bin/CTM2-12-e1060-s003.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Click here for additional data file.</a><sup> (30.9KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p"><p>Supporting Information</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/9506422/bin/CTM2-12-e1060-s004.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Click here for additional data file.</a><sup> (317.4KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="caption p"><p>Supporting Information</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/9506422/bin/CTM2-12-e1060-s001.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Click here for additional data file.</a><sup> (295.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="caption p"><p>Supporting Information</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/9506422/bin/CTM2-12-e1060-s005.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Click here for additional data file.</a><sup> (258.1KB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The mass spectrometry proteomics data has been deposited to the ProteomeXchange Consortium via the PRIDE<a href="#ctm21060-bib-0085" class="usa-link" aria-describedby="ctm21060-bib-0085" aria-expanded="false">
<sup>85</sup>
</a> partner repository with the dataset identifiers PXD029805 and 10.6019/PXD029805 (cell pellet data), PXD029821 and 10.6019/PXD029821 (culture media data). The scripts for the proteomic data analyses can be obtained at: <a href="https://github.com/bszeitz/SCLC_proteomics" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/bszeitz/SCLC_proteomics</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Clinical and Translational Medicine are provided here courtesy of <strong>John Wiley &amp; Sons Australia, Ltd on behalf of Shanghai Institute of Clinical Bioinformatics</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close">
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a href="https://doi.org/10.1002/ctm2.1060" class="usa-button usa-button--outline width-24 font-xs usa-link--external padding-left-0 padding-right-0" target="_blank" rel="noreferrer noopener" data-ga-category="actions" data-ga-action="click" data-ga-label="publisher_link_desktop">
                         <span class="height-3 display-inline-flex flex-align-center">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a href="pdf/CTM2-12-e1060.pdf" class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1" data-ga-category="actions" data-ga-action="click" data-ga-label="pdf_download_desktop">
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden="">
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.8&nbsp;MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1" aria-label="Open dialog with citation text in different styles" data-ga-category="actions" data-ga-action="open" data-ga-label="cite_desktop" data-all-citations-url="/resources/citations/9506422/" data-citation-style="nlm" data-download-format-link="/resources/citations/9506422/export/">
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden="">
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty" aria-label="Save article in MyNCBI collections." data-ga-category="actions" data-ga-action="click" data-ga-label="collections_button_desktop" data-collections-open-dialog-enabled="false" data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC9506422%2F%23open-collections-dialog" data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden="">
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden="">
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button type="button" class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none" aria-label="Show article permalink" aria-expanded="false" aria-haspopup="true" data-ga-category="actions" data-ga-action="open" data-ga-label="permalink_desktop">
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden="">
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden="">
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC9506422/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link" aria-expanded="false">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC9506422/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple="">
                    <h3 class="usa-accordion__heading">
                        <button type="button" class="usa-accordion__button" aria-expanded="false" aria-controls="resources-similar-articles" data-ga-category="resources_accordion" data-ga-action="open_similar_articles" data-ga-label="/articles/PMC9506422/" data-action-open="open_similar_articles" data-action-close="close_similar_articles">
                            Similar articles
                        </button>
                    </h3>
                    <div id="resources-similar-articles" class="usa-accordion__content usa-prose" data-source-url="/resources/similar-article-links/36149789/" hidden="">
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button type="button" class="usa-accordion__button" aria-expanded="false" aria-controls="resources-cited-by-other-articles" data-ga-category="resources_accordion" data-ga-action="open_cited_by" data-ga-label="/articles/PMC9506422/" data-action-open="open_cited_by" data-action-close="close_cited_by">
                             Cited by other articles
                        </button>
                    </h3>
                    <div id="resources-cited-by-other-articles" class="usa-accordion__content usa-prose" data-source-url="/resources/cited-by-links/36149789/" hidden="">
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button type="button" class="usa-accordion__button" aria-expanded="false" aria-controls="resources-links-to-ncbi-databases" data-ga-category="resources_accordion" data-ga-action="open_NCBI_links" data-ga-label="/articles/PMC9506422/" data-action-open="open_NCBI_links" data-action-close="close_NCBI_link">
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div id="resources-links-to-ncbi-databases" class="usa-accordion__content usa-prose" data-source-url="/resources/db-links/9506422/" hidden="">
                        </div>
                    
                    
                </div>
            
        </section>


        <section class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4" data-title-text="On this page" data-title-heading-level="h2" data-scroll-offset="0" data-root-margin="-10% 0px -80% 0px" data-main-content-selector="main" data-threshold="1"><nav aria-label="On this page" class="usa-in-page-nav__nav"><h2 class="usa-in-page-nav__heading" tabindex="0">On this page</h2><ul class="usa-in-page-nav__list"><li class="usa-in-page-nav__item"><a data-anchor-id="abstract1" href="#abstract1" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Abstract">Abstract</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="ctm21060-sec-0050" href="#ctm21060-sec-0050" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="1. INTRODUCTION">1. INTRODUCTION</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="ctm21060-sec-0060" href="#ctm21060-sec-0060" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="2. MATERIALS AND METHODS">2. MATERIALS AND METHODS</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="ctm21060-sec-0260" href="#ctm21060-sec-0260" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="3. RESULTS">3. RESULTS</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="ctm21060-sec-0320" href="#ctm21060-sec-0320" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="4. DISCUSSION">4. DISCUSSION</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="ctm21060-sec-0330" href="#ctm21060-sec-0330" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="5. CONCLUSIONS">5. CONCLUSIONS</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="ctm21060-sec-0340" href="#ctm21060-sec-0340" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="CONFLICT OF INTEREST">CONFLICT OF INTEREST</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="ctm21060-sec-0350" href="#ctm21060-sec-0350" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="FUNDING INFORMATION">FUNDING INFORMATION</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="sec36" href="#sec36" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Supporting information">Supporting information</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="_ci93_" href="#_ci93_" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Contributor Information">Contributor Information</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="ctm21060-sec-0370" href="#ctm21060-sec-0370" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="DATA AVAILABILITY STATEMENT">DATA AVAILABILITY STATEMENT</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="ctm21060-bibl-0001" href="#ctm21060-bibl-0001" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="REFERENCES">REFERENCES</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="_ad93_" href="#_ad93_" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Associated Data">Associated Data</a></li></ul></nav></section>
    </section>
</div>


        

<div class="overlay citation-dialog-overlay" role="dialog" aria-label="Citation Dialog">
    <div class="dialog citation-dialog" aria-hidden="true" role="document">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto" tabindex="1" data-pinger-ignore="true">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus" data-ga-category="save_share" data-ga-action="cite" data-ga-label="copy" tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a href="#" role="button" class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button" data-ga-category="save_share" data-ga-action="cite" data-ga-label="download" title="Download a file for external citation management software" tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4">
    
      <option data-style-url-name="ama" value="AMA">
        AMA
      </option>
    
      <option data-style-url-name="apa" value="APA">
        APA
      </option>
    
      <option data-style-url-name="mla" value="MLA">
        MLA
      </option>
    
      <option data-style-url-name="nlm" value="NLM" selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
<div class="dots-loading-indicator citation-loading-indicator">
        <div class="dot dot-1"></div>
        <div class="dot dot-2"></div>
        <div class="dot dot-3"></div>
      </div></div>
    </div>
</div>

        <div class="overlay collections-dialog-overlay" role="dialog">
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true" role="document">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form" class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors" data-existing-collections-url="/list-existing-collections/" data-add-to-existing-collection-url="/add-to-existing-collection/" data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/" data-myncbi-max-collection-name-length="100" data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="qCBxgl9DyOivmJUOVlabbVqrryNxyWk8zePWTdpYBWwAlp1ApIDnbQZ4Awry38HP">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio" id="collections-action-dialog-new" class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0" name="collections" value="new" data-ga-category="collections_button" data-ga-action="click" data-ga-label="collections_radio_new">
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio" id="collections-action-dialog-existing" class="usa-radio__input usa-radio__input--tile collections-existing" name="collections" value="existing" checked="true" data-ga-category="collections_button" data-ga-action="click" data-ga-label="collections_radio_existing">
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input type="text" name="add-to-new-collection" id="collections-action-dialog-add-to-new" class="usa-input collections-action-add-to-new" pattern="[^&quot;&amp;=<>/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=<>/" maxlength="" data-ga-category="collections_button" data-ga-action="create_collection" data-ga-label="non_favorties_collection" required="">
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing" class="usa-select collections-action-add-to-existing" data-ga-category="collections_button" data-ga-action="select_collection" data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit" type="submit" data-loading-label="Adding..." data-pinger-ignore="" data-ga-category="collections_button" data-ga-action="click" data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel" aria-label="Close 'Add to Collections' panel" ref="linksrc=close_collections_panel" data-ga-category="collections_button" data-ga-action="click" data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    

<footer class="ncbi-footer ncbi-dark-background ">
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi" class="ncbi-footer__social-icon ncbi-footer__social-icon--gray" target="_blank" rel="noreferrer noopener">
                        <svg width="40" height="40" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm" class="ncbi-footer__social-icon ncbi-footer__social-icon--gray" target="_blank" rel="noreferrer noopener">
                        <svg width="16" height="29" focusable="false" aria-hidden="true" viewBox="0 0 16 29" fill="none" xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm" class="ncbi-footer__social-icon ncbi-footer__social-icon--gray" target="_blank" rel="noreferrer noopener">
                        <svg width="25" height="23" viewBox="0 0 26 24" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi" class="ncbi-footer__social-icon ncbi-footer__social-icon--gray" target="_blank" rel="noreferrer noopener">
                        <svg width="28" height="27" viewBox="0 0 28 28" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/" class="ncbi-footer__social-icon ncbi-footer__social-icon--gray" target="_blank" rel="noreferrer noopener">
                        <svg width="26" height="26" viewBox="0 0 27 27" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer" class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container" data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href="https://twitter.com/nlm_nih" class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32" height="32" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href="https://www.facebook.com/nationallibraryofmedicine" class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13" height="24" viewBox="0 0 13 24" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href="https://www.youtube.com/user/NLMNIH" class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21" height="15" viewBox="0 0 21 15" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z" fill="white"></path>
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external" href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323" rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br> 8600 Rockville Pike<br> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link">
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link">
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target="_blank">
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/?pagename=cloudpmc-viewer%3Apmc%3Aarticle%3Ajournal" class="usa-link  usa-link--alt ncbi-footer__link" data-pinger-pagename-param="true">
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link">
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link">
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link">
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link">
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target="_blank">
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target="_blank">
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>



        
        
    
  <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script><div id="ZN_dikYWqsjiUWN0Q5"></div>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
     <script src="https://code.jquery.com/jquery-3.5.0.min.js" integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ=" crossorigin="anonymous">
    </script>
    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-9bea7450.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-722d91a2.js"></script>
    
    

    

<div class="fake-body-scroll"></div><div class="fake-body-scroll"></div><script id="_fed_an_ua_tag" text="" charset="" type="text/javascript" src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js?agency=HHS&amp;subagency=NCBI%20-%20ncbi.nlm.nih.gov&amp;sitetopic=NCBI%20Pinger%200.39.3&amp;siteplatform=NCBI%20Pinger%200.39.3"></script><script type="text/javascript" src="https://zndikYWqsjiUWN0Q5-nlmenterprise.siteintercept.qualtrics.com/SIE/?Q_ZID=ZN_dikYWqsjiUWN0Q5"></script><script src="https://siteintercept.qualtrics.com/dxjsmodule/CoreModule.js?Q_CLIENTVERSION=2.20.0&amp;Q_CLIENTTYPE=web&amp;Q_BRANDID=nlmenterprise" defer=""></script><script src="https://siteintercept.qualtrics.com/dxjsmodule/FeedbackButtonModule.js?Q_CLIENTVERSION=2.20.0&amp;Q_CLIENTTYPE=web&amp;Q_BRANDID=nlmenterprise" defer=""></script><style type="text/css">.QSIFeedbackButton div,.QSIFeedbackButton dl,.QSIFeedbackButton dt,.QSIFeedbackButton dd,.QSIFeedbackButton ul,.QSIFeedbackButton ol,.QSIFeedbackButton li,.QSIFeedbackButton h1,.QSIFeedbackButton h2,.QSIFeedbackButton h3,.QSIFeedbackButton h4,.QSIFeedbackButton h5,.QSIFeedbackButton h6,.QSIFeedbackButton span,.QSIFeedbackButton pre,.QSIFeedbackButton form,.QSIFeedbackButton fieldset,.QSIFeedbackButton textarea,.QSIFeedbackButton p,.QSIFeedbackButton blockquote,.QSIFeedbackButton tr,.QSIFeedbackButton th,.QSIFeedbackButton td{ margin: 0; padding: 0;background-color: transparent; border: 0; font-size: 12px; line-height: normal; vertical-align:baseline; box-shadow: none; }.QSIFeedbackButton img{ height: auto; width: auto; margin: 0; padding: 0 }.QSIFeedbackButton ul,.QSIFeedbackButton ol{ margin: 12px 0; padding-left: 40px; }.QSIFeedbackButton ul li{ list-style-type: disc; }.QSIFeedbackButton ol li{ list-style-type: decimal; }.QSIFeedbackButton .scrollable{ -webkit-overflow-scrolling: touch; }.QSIFeedbackButton table{ border-collapse: collapse; border-spacing: 0; }.QSIFeedbackButton table td{ padding: 2px; }.QSIFeedbackButton iframe{ max-height: none; }.QSIFeedbackButton *{ box-sizing: content-box; }</style><div class="QSIFeedbackButton" style="position: fixed; visibility: hidden; inset: 0px; display: flex; flex-direction: column; justify-content: flex-end; align-items: flex-start; margin: 0px; padding: 0px; z-index: 2000000000;"><button role="button" style="position: fixed; visibility: visible; cursor: pointer; border: medium; background-color: transparent; padding: 0px; margin: 0px; bottom: 97px; right: -1px; width: 52px;" aria-label="Tell us what you think!" id="QSIFeedbackButton-btn"><img src="https://iad1.qualtrics.com/WRQualtricsSiteIntercept/Graphic.php?IM=IM_3xWqwNkIHc7cA0S" alt="Tell us what you think!" style="max-width: 110px; max-height: 110px; display: block;"></button></div></body></html>