<html lang="en"><head>

        <meta charset="UTF-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="HandheldFriendly" content="True">
        <meta name="MobileOptimized" content="320">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-70b9163a.css">
<script type="text/javascript" async="" src="https://www.googletagmanager.com/gtag/js?id=G-DP2X732JSX&amp;l=pingerDataLayer&amp;cx=c&amp;gtm=45He4bk0v847342615za200"></script><script async="" src="https://www.googletagmanager.com/gtm.js?id=GTM-PC9B6M3&amp;l=pingerDataLayer" id="pingerInjectedGTM"></script><script type="module" crossorigin="" src="/static/assets/base_style-ec2bc71e.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css">
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css">
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon" sizes="180x180" href="/static/img/favicons/apple-touch-icon.png">
            <link rel="icon" type="image/png" sizes="48x48" href="/static/img/favicons/favicon-48x48.png">
            <link rel="icon" type="image/png" sizes="32x32" href="/static/img/favicons/favicon-32x32.png">
            <link rel="icon" type="image/png" sizes="16x16" href="/static/img/favicons/favicon-16x16.png">
            <link rel="manifest" href="/static/img/favicons/site.webmanifest">
            <link rel="mask-icon" href="/static/img/favicons/safari-pinned-tab.svg" color="#0071bc">
            <meta name="msapplication-config" content="/static/img/favicons/browserconfig.xml">
            <meta name="theme-color" content="#ffffff">
        

        <title>
            Molecular and Pathologic Characterization of YAP1-Expressing Small Cell Lung Cancer Cell Lines Leads to Reclassification as SMARCA4-Deficient Malignancies - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer">
<meta name="ncbi_db" content="pmc">
<meta name="ncbi_phid" content="76C1A9367478E30306A936003F23B3D6.m_1">
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article">
  
    <link rel="preconnect" href="https://www.google-analytics.com">

    
        <link rel="dns-prefetch" href="https://cdn.ncbi.nlm.nih.gov">
    

    <link rel="preconnect" href="https://code.jquery.com">
    <meta name="ncbi_domain" content="aacrsd">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11061608/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Clinical Cancer Research">
<meta name="citation_title" content="Molecular and Pathologic Characterization of YAP1-Expressing Small Cell Lung Cancer Cell Lines Leads to Reclassification as SMARCA4-Deficient Malignancies">
<meta name="citation_author" content="Jin Ng">
<meta name="citation_author_institution" content="ACRF Cancer Biology and Stem Cells Division, The Walter and Eliza Hall Institute of Medical Research, Parkville, Victoria, Australia.">
<meta name="citation_author_institution" content="Department of Medical Biology, The University of Melbourne, Parkville, Victoria, Australia.">
<meta name="citation_author" content="Ling Cai">
<meta name="citation_author_institution" content="Quantitative Biomedical Research Center, Peter O'Donnell Jr. School of Public Health, UT Southwestern Medical Center, Dallas, Texas.">
<meta name="citation_author_institution" content="Children's Research Institute, UT Southwestern Medical Center, Dallas, Texas.">
<meta name="citation_author_institution" content="Simmons Comprehensive Cancer Center, UT Southwestern Medical Center, Dallas, Texas.">
<meta name="citation_author" content="Luc Girard">
<meta name="citation_author_institution" content="Simmons Comprehensive Cancer Center, UT Southwestern Medical Center, Dallas, Texas.">
<meta name="citation_author_institution" content="Hamon Center for Therapeutic Oncology Research, UT Southwestern Medical Center, Dallas, Texas.">
<meta name="citation_author" content="Owen WJ Prall">
<meta name="citation_author_institution" content="Department of Anatomical Pathology, Peter MacCallum Cancer Centre, Melbourne, Victoria, Australia.">
<meta name="citation_author" content="Neeha Rajan">
<meta name="citation_author_institution" content="Department of Anatomical Pathology, The Royal Melbourne Hospital, Parkville, Victoria, Australia.">
<meta name="citation_author" content="Christine Khoo">
<meta name="citation_author_institution" content="Department of Anatomical Pathology, Peter MacCallum Cancer Centre, Melbourne, Victoria, Australia.">
<meta name="citation_author" content="Ahida Batrouney">
<meta name="citation_author_institution" content="Department of Anatomical Pathology, The Royal Melbourne Hospital, Parkville, Victoria, Australia.">
<meta name="citation_author" content="David J Byrne">
<meta name="citation_author_institution" content="Department of Anatomical Pathology, Peter MacCallum Cancer Centre, Melbourne, Victoria, Australia.">
<meta name="citation_author" content="Danielle K Boyd">
<meta name="citation_author_institution" content="ACRF Cancer Biology and Stem Cells Division, The Walter and Eliza Hall Institute of Medical Research, Parkville, Victoria, Australia.">
<meta name="citation_author" content="Ariena J Kersbergen">
<meta name="citation_author_institution" content="ACRF Cancer Biology and Stem Cells Division, The Walter and Eliza Hall Institute of Medical Research, Parkville, Victoria, Australia.">
<meta name="citation_author" content="Michael Christie">
<meta name="citation_author_institution" content="Department of Anatomical Pathology, The Royal Melbourne Hospital, Parkville, Victoria, Australia.">
<meta name="citation_author_institution" content="Personalised Oncology Division, The Walter and Eliza Hall Institute of Medical Research, Parkville, Victoria, Australia.">
<meta name="citation_author" content="John D Minna">
<meta name="citation_author_institution" content="Simmons Comprehensive Cancer Center, UT Southwestern Medical Center, Dallas, Texas.">
<meta name="citation_author_institution" content="Hamon Center for Therapeutic Oncology Research, UT Southwestern Medical Center, Dallas, Texas.">
<meta name="citation_author_institution" content="Department of Internal Medicine, UT Southwestern Medical Center, Dallas, Texas.">
<meta name="citation_author_institution" content="Department of Pharmacology, UT Southwestern Medical Center, Dallas, Texas.">
<meta name="citation_author" content="Marian L Burr">
<meta name="citation_author_institution" content="Division of Genome Science and Cancer, The John Curtin School of Medical Research, The Australian National University, Canberra, Australian Capital Territory, Australia.">
<meta name="citation_author_institution" content="Department of Anatomical Pathology, ACT Pathology, Canberra Health Services, Canberra, Australian Capital Territory, Australia.">
<meta name="citation_author_institution" content="Sir Peter MacCallum Department of Oncology, University of Melbourne, Victoria, Australia.">
<meta name="citation_author" content="Kate D Sutherland">
<meta name="citation_author_institution" content="ACRF Cancer Biology and Stem Cells Division, The Walter and Eliza Hall Institute of Medical Research, Parkville, Victoria, Australia.">
<meta name="citation_author_institution" content="Department of Medical Biology, The University of Melbourne, Parkville, Victoria, Australia.">
<meta name="citation_publication_date" content="2023 Dec 7">
<meta name="citation_volume" content="30">
<meta name="citation_issue" content="9">
<meta name="citation_firstpage" content="1846">
<meta name="citation_doi" content="10.1158/1078-0432.CCR-23-2360">
<meta name="citation_pmid" content="38180245">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11061608/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11061608/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11061608/pdf/1846.pdf">
<meta name="description" content="The classification of small cell lung cancer (SCLC) into distinct molecular subtypes defined by ASCL1, NEUROD1, POU2F3, or YAP1 (SCLC-A, -N, -P, or -Y) expression, paves the way for a personalized treatment approach. However, the existence of a ...">
<meta name="og:title" content="Molecular and Pathologic Characterization of YAP1-Expressing Small Cell Lung Cancer Cell Lines Leads to Reclassification as SMARCA4-Deficient Malignancies">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The classification of small cell lung cancer (SCLC) into distinct molecular subtypes defined by ASCL1, NEUROD1, POU2F3, or YAP1 (SCLC-A, -N, -P, or -Y) expression, paves the way for a personalized treatment approach. However, the existence of a ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11061608/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    <script type="text/javascript" src="https://www.googletagmanager.com/gtag/js?id=G-CSLL4ZEK4L"></script><script charset="utf-8" src="https://siteintercept.qualtrics.com/dxjsmodule/10.07268bfc859327bf20d5.chunk.js?Q_CLIENTVERSION=2.20.0&amp;Q_CLIENTTYPE=web&amp;Q_BRANDID=pmc.ncbi.nlm.nih.gov"></script><script charset="utf-8" src="https://siteintercept.qualtrics.com/dxjsmodule/5.d83df5c454102e31d5df.chunk.js?Q_CLIENTVERSION=2.20.0&amp;Q_CLIENTTYPE=web&amp;Q_BRANDID=nlmenterprise"></script><script charset="utf-8" src="https://siteintercept.qualtrics.com/dxjsmodule/1.4a290fea10f6e9b6f375.chunk.js?Q_CLIENTVERSION=2.20.0&amp;Q_CLIENTTYPE=web&amp;Q_BRANDID=nlmenterprise"></script></head>
    <body>
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government">
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true" class="usa-banner__header-flag" src="/static/img/us_flag.svg" alt="">
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button type="button" class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           " aria-expanded="false" aria-controls="gov-banner-default" data-testid="storybook-django-banner">
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content" id="gov-banner-default" hidden="">
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img" src="/static/img/icon-dot-gov.svg" alt="" aria-hidden="true">
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br>
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img" src="/static/img/icon-https.svg" alt="" aria-hidden="true">

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br>
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg" width="52" height="64" viewBox="0 0 52 64" class="usa-banner__lock-image" role="graphics-symbol" aria-labelledby="banner-lock-description" focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000" fill-rule="evenodd" d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z"></path>
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header="">
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="/">
                    <img alt="
                                  PMC home page
                              " class="ncbi-header__logo-image" src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg">
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button type="button" class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           " aria-label="Show menu" data-testid="navMenuButton">
    
        

        
                <svg aria-hidden="true" class="ncbi-hamburger-icon" fill="none" focusable="false" height="21" viewBox="0 0 31 21" width="31" xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd" d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z" fill="#F1F1F1" fill-rule="evenodd"></path>
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button type="button" class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           " aria-expanded="false" aria-controls="search-field-desktop-navigation" aria-label="Show search overlay" data-testid="toggleSearchPanelButton" data-toggle-search-panel-button="" data-ga-category="header" data-ga-action="NCBI" data-ga-label="header_search_open">
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true">
        
        <use xlink:href="/static/img/sprite.svg#search"></use>
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button type="button" class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           " aria-expanded="false" aria-controls="login-dropdown-menu" aria-label="Show login menu" data-testid="toggleLoginMenuDropdown" data-desktop-login-button="">
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true">
        
        <use xlink:href="/static/img/sprite.svg#person"></use>
    </svg>



                                <span data-login-dropdown-text="">Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow="">
        
        <use xlink:href="/static/img/sprite.svg#expand_less"></use>
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow="">
        
        <use xlink:href="/static/img/sprite.svg#expand_more"></use>
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu" id="login-dropdown-menu" data-desktop-login-menu-dropdown="" hidden="">
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  ">
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  ">
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  ">
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button type="button" class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           " data-testid="desktopLogoutButton" data-desktop-logout-button="">
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop" data-testid="searchPanel" data-header-search-panel="" hidden="">
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/" aria-describedby="search-field-desktop-navigation-help-text" autocomplete="off" class="usa-search usa-search--big ncbi-search-panel__form" data-testid="form" method="GET" role="search">
                    <label class="usa-sr-only" data-testid="label" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input" data-testid="textInput" id="search-field-desktop-navigation" name="term" placeholder="Search NCBI" type="search" value="">
                    



















    
        <button type="submit" class="usa-button
           

           
               
               
               
               
            

           
           
           
           " data-testid="button" data-ga-category="header" data-ga-action="NCBI" data-ga-label="header_search_button">
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button type="button" class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           " aria-label="Close navigation menu" data-testid="navCloseButton">
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close">
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6" role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input" id="search-field-mobile-navigation" type="search" placeholder="Search NCBI" name="search">

                



















    
        <button type="submit" class="usa-button
           

           
               
               
               
               
            

           
           
           
           " data-ga-category="header" data-ga-action="NCBI" data-ga-label="header_search_button">
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg==" class="usa-search__submit-icon" alt="Search">
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop" data-mobile-login-menu="" hidden="">
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email" data-mobile-login-email-text=""></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  ">
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  ">
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  ">
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button type="button" class="usa-button ncbi-nav__mobile-login-button ncbi--hide-at-desktop" data-testid="mobileLoginButton" data-mobile-login-button="">Log in</button>
        


    </nav>
</header>

    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button type="button" class="usa-button usa-button--unstyled pmc-header__search__button" aria-label="Open search" data-ga-category="search" data-ga-action="PMC" data-ga-label="pmc_search_panel_mobile">
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column pmc-header__search__form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search" type="search" name="term" data-autocomplete-url="/search/autocomplete/" aria-controls="autoComplete_list_1" aria-autocomplete="both" role="combobox" aria-owns="autoComplete_list_1" aria-haspopup="true" aria-expanded="false"><ul id="autoComplete_list_1" role="listbox" hidden="" aria-label="Suggestions"></ul>
</span>
<button class="usa-button" type="submit" formaction="https://www.ncbi.nlm.nih.gov/pmc/" data-ga-category="search" data-ga-action="PMC" data-ga-label="PMC_search_button">
<span class="usa-search__submit-text">Search in PMC</span>
<img src="/static/img/usa-icons-bg/search--white.svg" class="usa-search__submit-icon" alt="Search">
</button>
</form>
            <ul class="pmc-header__search__menu">
                <li>
                    <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                        Advanced Search
                    </a>
                </li>
                <li>
                    
                        <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                            Journal List
                        </a>
                    
                </li>
                <li>
                    
                        <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link" data-ga-label="user guide">
                            User Guide
                        </a>
                    
                </li>
            </ul>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="11061608">

    

   



<div class="grid-container pmc-actions-bar is-not-intersecting" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button type="button" class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex" aria-label="Open resources" data-extra-class="is-visible-resources" data-ga-category="resources_accordion" data-ga-action="click" data-ga-label="mobile_icon">
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a href="https://doi.org/10.1158/1078-0432.CCR-23-2360" class="usa-link display-flex" role="button" target="_blank" rel="noreferrer noopener" aria-label="View on publisher site" data-ga-category="actions" data-ga-action="click" data-ga-label="publisher_link_mobile">
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a href="pdf/1846.pdf" class="usa-link display-flex" role="button" aria-label="Download PDF" data-ga-category="actions" data-ga-action="click" data-ga-label="pdf_download_mobile">
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button class="usa-button usa-button--unstyled collections-dialog-trigger collections-button display-flex collections-button-empty" aria-label="Save article in MyNCBI collections." data-ga-category="actions" data-ga-action="click" data-ga-label="collections_button_mobile" data-collections-open-dialog-enabled="false" data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11061608%2F%23open-collections-dialog" data-in-collections="false">
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden="">
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden="">
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled citation-dialog-trigger display-flex" aria-label="Open dialog with citation text in different styles" data-ga-category="actions" data-ga-action="open" data-ga-label="cite_mobile" data-all-citations-url="/resources/citations/11061608/" data-citation-style="nlm" data-download-format-link="/resources/citations/11061608/export/">
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden="">
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex">
                         <button type="button" class="usa-button usa-button--unstyled display-flex" aria-label="Show article permalink" aria-expanded="false" aria-haspopup="true" data-ga-category="actions" data-ga-action="open" data-ga-label="permalink_mobile">
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden="">
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11061608/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link" aria-expanded="false">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button type="button" class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex" aria-label="Open article navigation" data-extra-class="is-visible-in-page" data-ga-category="actions" data-ga-action="open" data-ga-label="article_nav_mobile">
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main id="main-content" class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0">

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-aacrsd.png" alt="AACR Open Access logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to AACR Open Access" title="Link to AACR Open Access" shape="default" href="https://doi.org/10.1158/1078-0432.CCR-23-2360" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu" aria-expanded="false">Clin Cancer Res</button></div>. 2023 Dec 7;30(9):1846–1858. doi: <a href="https://doi.org/10.1158/1078-0432.CCR-23-2360" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1158/1078-0432.CCR-23-2360</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Clin%20Cancer%20Res%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Clin%20Cancer%20Res%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Clin%20Cancer%20Res%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Clin%20Cancer%20Res%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Molecular and Pathologic Characterization of YAP1-Expressing Small Cell Lung Cancer Cell Lines Leads to Reclassification as SMARCA4-Deficient Malignancies</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ng%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id1" aria-expanded="false"><span class="name western">Jin Ng</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Jin Ng</span></h3>
<div class="p">
<sup>1</sup>ACRF Cancer Biology and Stem Cells Division, The Walter and Eliza Hall Institute of Medical Research, Parkville, Victoria, Australia.</div>
<div class="p">
<sup>2</sup>Department of Medical Biology, The University of Melbourne, Parkville, Victoria, Australia.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ng%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jin Ng</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cai%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id2" aria-expanded="false"><span class="name western">Ling Cai</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Ling Cai</span></h3>
<div class="p">
<sup>3</sup>Quantitative Biomedical Research Center, Peter O'Donnell Jr. School of Public Health, UT Southwestern Medical Center, Dallas, Texas.</div>
<div class="p">
<sup>4</sup>Children's Research Institute, UT Southwestern Medical Center, Dallas, Texas.</div>
<div class="p">
<sup>5</sup>Simmons Comprehensive Cancer Center, UT Southwestern Medical Center, Dallas, Texas.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cai%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Ling Cai</span></a>
</div>
</div>
<sup>3,</sup><sup>4,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Girard%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id3" aria-expanded="false"><span class="name western">Luc Girard</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Luc Girard</span></h3>
<div class="p">
<sup>5</sup>Simmons Comprehensive Cancer Center, UT Southwestern Medical Center, Dallas, Texas.</div>
<div class="p">
<sup>6</sup>Hamon Center for Therapeutic Oncology Research, UT Southwestern Medical Center, Dallas, Texas.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Girard%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Luc Girard</span></a>
</div>
</div>
<sup>5,</sup><sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Prall%20OWJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id4" aria-expanded="false"><span class="name western">Owen WJ Prall</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Owen WJ Prall</span></h3>
<div class="p">
<sup>7</sup>Department of Anatomical Pathology, Peter MacCallum Cancer Centre, Melbourne, Victoria, Australia.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Prall%20OWJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Owen WJ Prall</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rajan%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id5" aria-expanded="false"><span class="name western">Neeha Rajan</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Neeha Rajan</span></h3>
<div class="p">
<sup>8</sup>Department of Anatomical Pathology, The Royal Melbourne Hospital, Parkville, Victoria, Australia.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rajan%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Neeha Rajan</span></a>
</div>
</div>
<sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Khoo%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id6" aria-expanded="false"><span class="name western">Christine Khoo</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Christine Khoo</span></h3>
<div class="p">
<sup>7</sup>Department of Anatomical Pathology, Peter MacCallum Cancer Centre, Melbourne, Victoria, Australia.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Khoo%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Christine Khoo</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Batrouney%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id7" aria-expanded="false"><span class="name western">Ahida Batrouney</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Ahida Batrouney</span></h3>
<div class="p">
<sup>8</sup>Department of Anatomical Pathology, The Royal Melbourne Hospital, Parkville, Victoria, Australia.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Batrouney%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Ahida Batrouney</span></a>
</div>
</div>
<sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Byrne%20DJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id8" aria-expanded="false"><span class="name western">David J Byrne</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">David J Byrne</span></h3>
<div class="p">
<sup>7</sup>Department of Anatomical Pathology, Peter MacCallum Cancer Centre, Melbourne, Victoria, Australia.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Byrne%20DJ%22%5BAuthor%5D" class="usa-link"><span class="name western">David J Byrne</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Boyd%20DK%22%5BAuthor%5D" class="usa-link" aria-describedby="id9" aria-expanded="false"><span class="name western">Danielle K Boyd</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Danielle K Boyd</span></h3>
<div class="p">
<sup>1</sup>ACRF Cancer Biology and Stem Cells Division, The Walter and Eliza Hall Institute of Medical Research, Parkville, Victoria, Australia.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Boyd%20DK%22%5BAuthor%5D" class="usa-link"><span class="name western">Danielle K Boyd</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kersbergen%20AJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id10" aria-expanded="false"><span class="name western">Ariena J Kersbergen</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Ariena J Kersbergen</span></h3>
<div class="p">
<sup>1</sup>ACRF Cancer Biology and Stem Cells Division, The Walter and Eliza Hall Institute of Medical Research, Parkville, Victoria, Australia.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kersbergen%20AJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Ariena J Kersbergen</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Christie%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id11" aria-expanded="false"><span class="name western">Michael Christie</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Michael Christie</span></h3>
<div class="p">
<sup>8</sup>Department of Anatomical Pathology, The Royal Melbourne Hospital, Parkville, Victoria, Australia.</div>
<div class="p">
<sup>9</sup>Personalised Oncology Division, The Walter and Eliza Hall Institute of Medical Research, Parkville, Victoria, Australia.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Christie%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Michael Christie</span></a>
</div>
</div>
<sup>8,</sup><sup>9</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Minna%20JD%22%5BAuthor%5D" class="usa-link" aria-describedby="id12" aria-expanded="false"><span class="name western">John D Minna</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">John D Minna</span></h3>
<div class="p">
<sup>5</sup>Simmons Comprehensive Cancer Center, UT Southwestern Medical Center, Dallas, Texas.</div>
<div class="p">
<sup>6</sup>Hamon Center for Therapeutic Oncology Research, UT Southwestern Medical Center, Dallas, Texas.</div>
<div class="p">
<sup>10</sup>Department of Internal Medicine, UT Southwestern Medical Center, Dallas, Texas.</div>
<div class="p">
<sup>11</sup>Department of Pharmacology, UT Southwestern Medical Center, Dallas, Texas.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Minna%20JD%22%5BAuthor%5D" class="usa-link"><span class="name western">John D Minna</span></a>
</div>
</div>
<sup>5,</sup><sup>6,</sup><sup>10,</sup><sup>11</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Burr%20ML%22%5BAuthor%5D" class="usa-link" aria-describedby="id13" aria-expanded="false"><span class="name western">Marian L Burr</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Marian L Burr</span></h3>
<div class="p">
<sup>12</sup>Division of Genome Science and Cancer, The John Curtin School of Medical Research, The Australian National University, Canberra, Australian Capital Territory, Australia.</div>
<div class="p">
<sup>13</sup>Department of Anatomical Pathology, ACT Pathology, Canberra Health Services, Canberra, Australian Capital Territory, Australia.</div>
<div class="p">
<sup>14</sup>Sir Peter MacCallum Department of Oncology, University of Melbourne, Victoria, Australia.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Burr%20ML%22%5BAuthor%5D" class="usa-link"><span class="name western">Marian L Burr</span></a>
</div>
</div>
<sup>12,</sup><sup>13,</sup><sup>14,</sup><sup>#,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sutherland%20KD%22%5BAuthor%5D" class="usa-link" aria-describedby="id14" aria-expanded="false"><span class="name western">Kate D Sutherland</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Kate D Sutherland</span></h3>
<div class="p">
<sup>1</sup>ACRF Cancer Biology and Stem Cells Division, The Walter and Eliza Hall Institute of Medical Research, Parkville, Victoria, Australia.</div>
<div class="p">
<sup>2</sup>Department of Medical Biology, The University of Melbourne, Parkville, Victoria, Australia.</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sutherland%20KD%22%5BAuthor%5D" class="usa-link"><span class="name western">Kate D Sutherland</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>#,</sup><sup>*</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff1">
<sup>1</sup>ACRF Cancer Biology and Stem Cells Division, The Walter and Eliza Hall Institute of Medical Research, Parkville, Victoria, Australia.</div>
<div id="aff2">
<sup>2</sup>Department of Medical Biology, The University of Melbourne, Parkville, Victoria, Australia.</div>
<div id="aff3">
<sup>3</sup>Quantitative Biomedical Research Center, Peter O'Donnell Jr. School of Public Health, UT Southwestern Medical Center, Dallas, Texas.</div>
<div id="aff4">
<sup>4</sup>Children's Research Institute, UT Southwestern Medical Center, Dallas, Texas.</div>
<div id="aff5">
<sup>5</sup>Simmons Comprehensive Cancer Center, UT Southwestern Medical Center, Dallas, Texas.</div>
<div id="aff6">
<sup>6</sup>Hamon Center for Therapeutic Oncology Research, UT Southwestern Medical Center, Dallas, Texas.</div>
<div id="aff7">
<sup>7</sup>Department of Anatomical Pathology, Peter MacCallum Cancer Centre, Melbourne, Victoria, Australia.</div>
<div id="aff8">
<sup>8</sup>Department of Anatomical Pathology, The Royal Melbourne Hospital, Parkville, Victoria, Australia.</div>
<div id="aff9">
<sup>9</sup>Personalised Oncology Division, The Walter and Eliza Hall Institute of Medical Research, Parkville, Victoria, Australia.</div>
<div id="aff10">
<sup>10</sup>Department of Internal Medicine, UT Southwestern Medical Center, Dallas, Texas.</div>
<div id="aff11">
<sup>11</sup>Department of Pharmacology, UT Southwestern Medical Center, Dallas, Texas.</div>
<div id="aff12">
<sup>12</sup>Division of Genome Science and Cancer, The John Curtin School of Medical Research, The Australian National University, Canberra, Australian Capital Territory, Australia.</div>
<div id="aff13">
<sup>13</sup>Department of Anatomical Pathology, ACT Pathology, Canberra Health Services, Canberra, Australian Capital Territory, Australia.</div>
<div id="aff14">
<sup>14</sup>Sir Peter MacCallum Department of Oncology, University of Melbourne, Victoria, Australia.</div>
<div class="author-notes p">
<div class="fn" id="num1">
<sup>#</sup><p class="display-inline">Marian L. Burr and Kate D. Sutherland contributed equally as the co-senior authors of this article.</p>
</div>
<div class="fn" id="cor1">
<sup>*</sup><p class="display-inline">

<strong>Corresponding Authors:</strong> Kate D. Sutherland, ACRF Cancer Biology and Stem Cells Division, Walter and Eliza Hall Institute of Medical Research, 1G Royal Parade, Parkville, VIC 3052, Australia. E-mail: <span>sutherland.k@wehi.edu.au</span>; and Marian L. Burr, Division of Genome Science and Cancer, The John Curtin School of Medical Research, The Australian National University, ACT 2601, Australia. E-mail: <span>marian.burr@anu.edu.au</span></p>
</div>
<div class="fn" id="fn2"><p>Clin Cancer Res 2024;30:1846–58</p></div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2023 Aug 4; Revised 2023 Oct 8; Accepted 2023 Dec 1; Issue date 2024 May 1.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>©2023 The Authors; Published by the American Association for Cancer Research</div>
<p>This open access article is distributed under the Creative Commons Attribution 4.0 International (CC BY 4.0) license.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC11061608&nbsp;&nbsp;PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/38180245/" class="usa-link">38180245</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2 data-anchor-id="abstract1"><a id="abstract1-anchor" data-anchor-id="abstract1" class="usa-anchor"></a>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Purpose:</h3>
<p>The classification of small cell lung cancer (SCLC) into distinct molecular subtypes defined by ASCL1, NEUROD1, POU2F3, or YAP1 (SCLC-A, -N, -P, or -Y) expression, paves the way for a personalized treatment approach. However, the existence of a distinct YAP1-expressing SCLC subtype remains controversial.</p></section><section id="sec2_pmc"><h3 class="pmc_sec_title">Experimental Design:</h3>
<p>To better understand YAP1-expressing SCLC, the mutational landscape of human SCLC cell lines was interrogated to identify pathogenic alterations unique to SCLC-Y. Xenograft tumors, generated from cell lines representing the four SCLC molecular subtypes, were evaluated by a panel of pathologists who routinely diagnose thoracic malignancies. Diagnoses were complemented by transcriptomic analysis of primary tumors and human cell line datasets. Protein expression profiles were validated in patient tumor tissue.</p></section><section id="sec3_pmc"><h3 class="pmc_sec_title">Results:</h3>
<p>Unexpectedly, pathogenic mutations in <em>SMARCA4</em> were identified in six of eight SCLC-Y cell lines and correlated with reduced SMARCA4 mRNA and protein expression. Pathologist evaluations revealed that SMARCA4-deficient SCLC-Y tumors exhibited features consistent with thoracic SMARCA4-deficient undifferentiated tumors (SMARCA4-UT). Similarly, the transcriptional profile <em>SMARCA4</em>-mutant SCLC-Y lines more closely resembled primary SMARCA4-UT, or SMARCA4-deficient non–small cell carcinoma, than SCLC. Furthermore, SMARCA4-UT patient samples were associated with a YAP1 transcriptional signature and exhibited strong YAP1 protein expression. Together, we found little evidence to support a diagnosis of SCLC for any of the YAP1-expressing cell lines originally used to define the SCLC-Y subtype.</p></section><section id="sec4_pmc"><h3 class="pmc_sec_title">Conclusions:</h3>
<p>SMARCA4-mutant SCLC-Y cell lines exhibit characteristics consistent with SMARCA4-deficient malignancies rather than SCLC. Our findings suggest that, unlike ASCL1, NEUROD1, and POU2F3, YAP1 is not a subtype defining transcription factor in SCLC.</p>
<p>
<em>
<a href="https://aacrjournals.org/clincancerres/article/doi/10.1158/1078-0432.CCR-24-0227" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">See related commentary by Rekhtman, p. 1708</a>
</em>
</p></section></section><hr class="headless">
<section class="bt xbox font-sm" id="b1"><h2 class="obj_head">Translational Relevance.</h2>
<p>In this study, we perform an in-depth characterization of the YAP1-expressing human SCLC cell lines, utilized widely for studying subtype-specific vulnerabilities in SCLC. Genomic analyses identified an association between SCLC-Y cell lines and pathogenic alterations in <em>SMARCA4</em>. Morphologic and molecular evaluations performed with <em>SMARCA4</em>-mutant SCLC-Y lines revealed inconsistencies with an SCLC diagnosis. Rather, <em>SMARCA4</em>-mutant SCLC-Y cells represent SMARCA4-deficient undifferentiated tumors (SMARCA4-UT), a recently described lung malignancy that can mimic SCLC. Importantly, we show that primary human SMARCA4-UT also express high levels of YAP1. Our results suggest that YAP1 is not a reliable subtype defining transcription factor for SCLC. Importantly, cell lines commonly used as preclinical models of YAP1-expressing SCLC are in fact SMARCA4-UT, for which to date there has been a lack of representative human disease models. Thus, these preclinical models previously thought to be SCLC now represent a new preclinical model resource for study of SMARCA4-UT.</p></section><section id="sec2"><h2 class="pmc_sec_title" data-anchor-id="sec2"><a id="sec2-anchor" data-anchor-id="sec2" class="usa-anchor"></a>Introduction</h2>
<p>Small cell lung cancer (SCLC) is the deadliest form of lung cancer with limited durable responses to platinum-based chemotherapy and immune checkpoint blockade-based immunotherapy (<a href="#bib1" class="usa-link" aria-describedby="bib1" aria-expanded="false">1</a>). Unlike lung adenocarcinoma, where the development of therapies targeting specific driver oncogenes has enabled genetic alteration-based stratification, recurrent targetable mutations have not been identified in SCLC. The recent classification of SCLC into four molecular subtypes based on transcription factor expression therefore represents a major advance, providing a new paradigm to facilitate better understanding of SCLC biology and a path towards the development of molecularly targeted treatments (<a href="#bib2" class="usa-link" aria-describedby="bib2" aria-expanded="false">2, 3</a>). These four molecular subtypes are defined by the dominant expression of <em>ASCL1</em> (SCLC-A), <em>NEUROD1</em> (SCLC-N), <em>POU2F3</em> (SCLC-P), or <em>YAP1</em> (SCLC-Y). Indeed, recent studies characterizing human SCLC cell lines have identified several subtype-specific therapeutic vulnerabilities (<a href="#bib4" class="usa-link" aria-describedby="bib4" aria-expanded="false">4</a>). For example, Delta-like ligand 3 (DLL3), an inhibitory Notch ligand, is the target of a plethora of clinical therapeutics in SCLC and is reported to exhibit high expression in SCLC-A tumors (<a href="#bib5" class="usa-link" aria-describedby="bib5" aria-expanded="false">5</a>).</p>
<p>The seminal initial classification of SCLC subtypes was based on RNA-sequencing data from SCLC cell line data in the Cancer Cell Line Encyclopedia (CCLE; ref. <a href="#bib2" class="usa-link" aria-describedby="bib2" aria-expanded="false">2</a>) and deposited data from two well-characterized SCLC patient cohorts [Rudin and colleagues (<a href="#bib6" class="usa-link" aria-describedby="bib6" aria-expanded="false">6</a>) and George and colleagues (<a href="#bib7" class="usa-link" aria-describedby="bib7" aria-expanded="false">7</a>)]. Although the SCLC-A, SCLC-N, and SCLC-P groups within this classification are comprised of both primary tumors and cell lines, notably the SCLC-Y group consists mostly of cell lines (<a href="#bib6" class="usa-link" aria-describedby="bib6" aria-expanded="false">6</a>). Furthermore, although a follow-up study performing immunohistochemistry (IHC) in a well-characterized patient cohort validated the presence of NEUROD1, ASCL1, and POU2F3 expressing primary SCLC tumors, a distinct YAP1 expressing SCLC subtype was not identified (<a href="#bib8" class="usa-link" aria-describedby="bib8" aria-expanded="false">8</a>). Nevertheless, YAP1 expressing SCLC cells have been detected in patient-derived xenografts, genetically engineered mouse (GEM) models, and some patient cohorts (<a href="#bib9" class="usa-link" aria-describedby="bib9" aria-expanded="false">9–11</a>). In most cases, YAP1 expression in these settings was focal or heterogenous, although rare YAP1 dominant primary SCLC tumors have been reported (<a href="#bib12" class="usa-link" aria-describedby="bib12" aria-expanded="false">12</a>). Together, these observations have fuelled the controversy around the existence of SCLC-Y as a <em>bona fide</em> SCLC subtype (<a href="#bib8" class="usa-link" aria-describedby="bib8" aria-expanded="false">8, 11, 13</a>).</p>
<p>YAP1 expressing SCLC cells are characterized by low expression of classical neuroendocrine markers synaptophysin, chromogranin A, NCAM1 (CD56), and INSM1. Although substantial expression of classic neuroendocrine markers helps validate a SCLC diagnosis, 5% to 10% of SCLC primary tumors may show low or absent neuroendocrine marker expression (<a href="#bib14" class="usa-link" aria-describedby="bib14" aria-expanded="false">14</a>). Thus, a histologic diagnosis of SCLC can be made in the absence of positive neuroendocrine marker IHC if the tumor morphology is characteristic (<a href="#bib15" class="usa-link" aria-describedby="bib15" aria-expanded="false">15</a>). Nevertheless, neuroendocrine marker low/negative SCLC presents a clinical diagnostic challenge as there are several other malignancies that can pathologically mimic SCLC, such as basaloid squamous cell carcinoma and poorly differentiated adenocarcinoma, which must be excluded (<a href="#bib14" class="usa-link" aria-describedby="bib14" aria-expanded="false">14</a>). Importantly, many of the tumors that can mimic SCLC express YAP1, which is an additional complicating factor to the use of YAP1 as a defining factor for SCLC. Because there are standard treatment regimens applied when a patient receives a clinical/pathologic diagnosis of SCLC, it is of substantial importance to be sure of this diagnosis when lung tumors posing diagnostic problems such as a YAP1 expressing, neuroendocrine marker low expressing tumor are encountered.</p>
<p>Although the role of YAP1 as a subtype defining transcription factor for SCLC remains controversial, the existence of a subset of SCLC tumors that lack expression of NEUROD1, ASCL1, or POU2F3 and characteristically exhibit low expression of neuroendocrine markers is well recognized and has been reproducibly identified in several SCLC patient cohorts (<a href="#bib8" class="usa-link" aria-describedby="bib8" aria-expanded="false">8, 12, 13</a>). This subset of neuroendocrine low and A/N/P negative SCLC tumors has become particularly significant following the recognition that these tumors typically express higher levels of inflammation-associated genes and have increased T-cell infiltration compared with “classical” neuroendocrine high SCLC and thus are potentially the most likely to benefit from immune checkpoint blockade immune therapy (<a href="#bib13" class="usa-link" aria-describedby="bib13" aria-expanded="false">13, 16</a>). In some studies these A/N/P negative tumors have been termed “triple negative,” while other studies have termed them neuroendocrine low SCLC with an “inflamed” gene signature “SCLC-I.” Importantly, retrospective stratification of patients with SCLC in the IMpower133 clinical trial demonstrates that patients with SCLC-I tumors show increased survival following chemoimmunotherapy with atezolizumab (anti-PD-L1) and etoposide, compared with SCLC-A, -N, or -P (<a href="#bib13" class="usa-link" aria-describedby="bib13" aria-expanded="false">13</a>).</p>
<p>Given the apparent increased responsiveness of SCLC-I subtype tumors to immunotherapy, there is a critical need to better understand the origin and biology of these tumors. Notably, overlapping features of low neuroendocrine marker expression and an “inflamed” gene signature in cell lines defined as SCLC-Y has meant that these lines have frequently been used as models of non-neurodocrine, “inflamed” or “triple negative” SCLC (<a href="#bib17" class="usa-link" aria-describedby="bib17" aria-expanded="false">17</a>). However, the relationship between SCLC-I and the initially defined SCLC-Y subtype, and whether these YAP1 positive SCLC-Y cell lines present a reliable model of “neurodocrine low” SCLC, has not been clearly defined. This is of particular significance given that YAP1 expression has been reported to be an unreliable marker of SCLC-I in clinical cohorts (<a href="#bib13" class="usa-link" aria-describedby="bib13" aria-expanded="false">13, 18</a>). To better understand the biology and origins of SCLC-Y, we therefore performed a comprehensive molecular and histologic characterization of the tumor lines used to define this proposed subtype of SCLC.</p></section><section id="sec3"><h2 class="pmc_sec_title" data-anchor-id="sec3"><a id="sec3-anchor" data-anchor-id="sec3" class="usa-anchor"></a>Materials and Methods</h2>
<section id="sec3-1"><h3 class="pmc_sec_title">Cell culture and transplantation studies</h3>
<p>The following human SCLC cell lines were purchased from ATCC: NCI-H69 (RRID:CVCL_1579), NCI-H1092 (RRID:CVCL_1454), NCI-H2227 (RRID:CVCL_1542), NCI-H2171 (RRID:CVCL_1536), NCI-H1694 (RRID:CVCL_1489), NCI-H211 (RRID:CVCL_1529), NCI-H1048 (RRID:CVCL_1453), NCI-H1341 (RRID:CVCL_1463), SW1271 (RRID:CVCL_1716), NCI-H841 (RRID:CVCL_1595), DMS114 (RRID:CVCL_1174), NCI-H196 (RRID:CVCL_1509), NCI-H661 (RRID:CVCL_1577), and NCI-H1581 (RRID:CVCL_1479). Human SCLC cell line SBC5 (RRID:CVCL_1679) was obtained directly from Cell Bank Australia. Human SCLC cell lines NCI-H82 (RRID:CVCL_1591), NCI-H446 (RRID:CVCL_1562), and NCI-H146 (RRID:CVCL_1473) were obtained from Dr. Marian L. Burr (The John Curtin School of Medical Research) and NCI-H526 (RRID:CVCL_1569) was obtained from Dr. David Huang (Walter and Eliza Hall Institute). All cell lines were cultured according to the manufacturer's recommendations with routine PCR-based mycoplasma testing every 3 months. All cell lines were authenticated by short tandem repeat (STR) profiling from the cell banks they were originally purchased from and by the Australian Genome Research Facility (Melbourne). The cell lines in this study were cultured for fewer than 6 months after receipt. Tumor xenografts were generated by subcutaneous injection of 1 × 10<sup>6</sup> cells in 50% growth factor reduced Matrigel (BD Biosciences) into the flanks of CBA.Nude mice and harvested for paraffin embedding once tumors reached a volume of 200 to 1,000 mm<sup>3</sup>. All animal experiments were conducted according to the regulatory standards approved by the Walter and Eliza Hall Institute Animal Ethics Committee.</p></section><section id="sec3-2"><h3 class="pmc_sec_title">Histology and IHC</h3>
<p>Subcutaneous tumors were harvested and fixed in 10% (v/v) neutral-buffered formalin at room temperature or 4% (w/v) paraformaldehyde at 4°C for at least 24 hours. Fixed samples were dehydrated, cleared, and embedded in paraffin. Sections 3 μm thick were stained with haematoxylin and eosin (H&amp;E) and sections 4-μm thick were immunostained. The following antibodies for IHC were purchased from Abcam: anti-SMARCA4 (#ab110641:RRID:AB_10861578), anti-synaptophysin (#ab32127:RRID:AB_2286949), anti-Claudin 4 (#ab53156:RRID:AB_869176), and anti-wide-spectrum cytokeratin (#ab9377:RRID:AB_307222). The following antibodies were purchased from Cell Signaling Technology: anti-RB1 (#9309:RRID:AB_823629), anti-YAP1 (#14074: RRID:AB_2650491), and anti-SMARCA2 (#11966:RRID:AB_2797783). Anti-CD56 was purchased from Millipore Sigma (#AB5032:RRID:AB_2291692), anti-INSM1 was purchased from Santa Cruz Biotechnology (#sc271408:RRID:AB_10607955), anti-TTF1 was purchased from Leica Biosystems (#NCL-TTF-1:RRID:AB_442138), anti-p40 was purchased from Biocare Medical (#ACI3066 A, C: RRID:AB_2858274), and Roche (#790–4950: RRID:AB_2935820). A complete list of antibody dilutions, antigen retrieval conditions, and detection solutions are detailed in Supplementary Table S1.</p>
<p>Primary antibodies (SMARCA4, INSM1, TTF1, p40, and RB1) were stained using the Roche Ventana BenchMark ULTRA automated staining system. While staining with primary antibodies (CD56, synaptophysin, wide-spectrum cytokeratin, YAP1, SMARCA2, and Claudin 4) was performed on either the Dako Omnis or Leica Bond automated staining system, using standard protocols. Stained xenografts were evaluated by M.L.B. for either nuclear, membranous, or cytoplasmic staining and <em>H</em> scores were calculated as described previously (Supplementary Table S1; ref. <a href="#bib8" class="usa-link" aria-describedby="bib8" aria-expanded="false">8</a>).</p></section><section id="sec3-3"><h3 class="pmc_sec_title">Pathology assessment of cell line xenografts</h3>
<p>Five anatomical pathologists who routinely diagnose thoracic malignancies across different hospitals within Australia were recruited. Prior to examination of the cell line xenografts, pathologists were provided with the same clinical history for all slides: 60-year-old male with a lung mass and mediastinal lymphadenopathy.</p>
<p>Pathologists were first asked to assess the H&amp;E xenografts, in a blinded fashion, and provide a provisional diagnosis, as a “yes” or “no,” of whether the H&amp;E appearances of each xenograft were consistent with a pathologic diagnosis of SCLC. Following this provisional diagnosis, the pathologists were provided with images of IHC staining and a table (Supplementary Table S2) summarizing the immunophenotype of each xenograft together with a link to the <em>Thoracic SMARCA4-deficient undifferentiated tumor</em> chapter in the 5th Edition of WHO Classification of Tumors (<a href="#bib15" class="usa-link" aria-describedby="bib15" aria-expanded="false">15</a>). Pathologic assessment of each xenograft was performed independently with each pathologist blinded to the provisional and re-evaluated diagnosis from the other pathologists.</p></section><section id="sec3-4"><h3 class="pmc_sec_title">Ethics and patient samples</h3>
<p>All procedures performed in this study involving patient material were in accordance with the ethical standards of the institutional research committees and with the Declaration of Helsinki. This study was approved by the Walter and Eliza Hall Institute Human Research Ethics Committee (#22/7), Australian Capital Territory (ACT) Health Human Research Ethics Committee (2022.LRE.00216, 2022/<a href="https://www.ncbi.nlm.nih.gov/protein/ETH02563" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">ETH02563</a>) and Peter MacCallum Cancer Centre Human Research Ethics Committee (HREC no.: 03/90). Tumor samples from patients diagnosed with SMARCA4-UT were identified through review of pathology records at Peter MacCallum Cancer Centre and Canberra Hospital. The clinical profiles for all SMARCA4-UT patients such as sex and age at diagnosis are summarized in Supplementary Table S3. Clinical profiles for Case 1, Case 2, and Case 3 were not collected as the ethics under which these samples were collected did not permit for this. In addition, the results of the diagnostic immunohistochemical panel performed for these samples are summarized in Supplementary Table S3. H&amp;E stained slides and IHC were reviewed by pathologists involved in this study to verify the original diagnosis of SMARCA4-UT and sections from stored FFPE samples were additionally stained for YAP1 (Supplementary Table S1). H&amp;E scans of the patient biopsy blocks from which cell lines NCI-H661 and NCI-H1581 were derived, were acquired through UTSW.</p></section><section id="sec3-5"><h3 class="pmc_sec_title">Multi-omics analysis of cell line and patient samples</h3>
<p>SCLC class was determined by CCLE RNA-seq (<a href="#bib19" class="usa-link" aria-describedby="bib19" aria-expanded="false">19, 20</a>) or the separate SCLC molecular dataset for RNA-seq and whole-exome sequencing performed by the Hamon Center (deposited at dbGaP Study Accession: phs001823.v1.p1; <a href="https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001823.v1.p1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001823.v1.p1</a>) and cited in Cai and colleagues (<a href="#bib16" class="usa-link" aria-describedby="bib16" aria-expanded="false">16</a>) data for driver transcription factors; <em>ASCL1</em>, <em>NEUROD1</em>, <em>POU2F3</em>, and <em>YAP</em>. For each cell line, the transcription factor with the highest expression among the four is used to assign the transcription factor class. These annotations were concordant with that described by Rudin and colleagues (<a href="#bib2" class="usa-link" aria-describedby="bib2" aria-expanded="false">2</a>) except for one cell line (H2227), which was annotated SCLC-A using the UTSW dataset (Supplementary Table S4). Mutations present in over 50% of cell lines within each molecular subtype were extracted via cBioportal.org (<a href="#bib21" class="usa-link" aria-describedby="bib21" aria-expanded="false">21, 22</a>). Known pathogenic mutations were annotated using OncoKB (RRID:SCR_014782) and <a href="https://cancerhotspots.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://cancerhotspots.org</a> available through cBioportal.org (RRID:SCR_014555).</p>
<p>We used the following datasets from UT Lung SPORE: microarray, RNA-seq gene expression data and mutation data (dbGAP Study Accession No.: phs001823.v1.p1). We used the following datasets from DepMap (<a href="#bib19" class="usa-link" aria-describedby="bib19" aria-expanded="false">19</a>): RNA-seq(CCLE_depMap_19Q1_TPM.csv), microarray (CCLE_Expression_Entrez_2012–09–29.gct), proteomics (Table_S2_Protein_Quant_Normalized.xlsx; ref. <a href="#bib20" class="usa-link" aria-describedby="bib20" aria-expanded="false">20</a>), mutation (downloaded as CCLE_DepMap_18q3_maf_20180718.txt), and structural variation (CCLE_translocations_SvABA_20181221.xlsx) datasets. For each cell line, we computed the neuroendocrine (NE) score as described previously (<a href="#bib23" class="usa-link" aria-describedby="bib23" aria-expanded="false">23</a>), but with the updated NE signature based on RNA-seq data (<a href="#bib16" class="usa-link" aria-describedby="bib16" aria-expanded="false">16</a>). Le Loarer datasets (<a href="#bib24" class="usa-link" aria-describedby="bib24" aria-expanded="false">24</a>) were downloaded from SRA under accession no. SRP052896 and processed using the RNA-seq pipeline (<a href="https://git.biohpc.swmed.edu/BICF/Astrocyte/rnaseq" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://git.biohpc.swmed.edu/BICF/Astrocyte/rnaseq</a>) from UTSW Bioinformatics Core Facility. The CCLE cell lines annotated as SCLC, NSCLC, or SCCOHT was merged with the Le Loarer RNA-seq data and were quantile normalized. We used a previously published YAP1 SCLC gene signature (<a href="#bib10" class="usa-link" aria-describedby="bib10" aria-expanded="false">10</a>) for unsupervised hierarchal clustering of CCLE cell lines and Le Loarer patient samples.</p>
<p>To analyze mRNA expression of markers associated with SMARCA4 and a neuroendocrine phenotype, CCLE cell lines were grouped into “SMARCA4-UT cluster” (<em>n</em> = 7), “SMARCA4 mutant NSCLC” (<em>n</em> = 31), “SMARCA4 proficient NSCLC” (<em>n</em> = 88), and “SCLC-A, -N, -P” (<em>n</em> = 43). See Supplementary Table S5 for a full annotated list of cell lines. Gene expression (<em>Z</em>-score; log RNAseq RPKM) values were accessed through cBioportal.org (<a href="#bib21" class="usa-link" aria-describedby="bib21" aria-expanded="false">21, 22</a>).</p></section><section id="sec3-6"><h3 class="pmc_sec_title">Statistical analysis</h3>
<p>All statistical analyses were conducted in Graph Pad Prism Version 9 (RRID:SCR_002798) or R studio Version 4.0.2 (RRID:SCR_001905). Data were assessed for normal distribution using a Shapiro–Wilk test. Data were considered normally distributed if the <em>P</em>-value was &gt; 0.05. If the data were not normally distributed, a Kruskal–Wallis test with a Dunn's multiple comparison test (between group) was conducted for multiple groups, or a Mann–Whitney test comparing tanks between two groups. The level of significance was set to <em>P</em> &lt; 0.05.</p></section><section id="sec3-7"><h3 class="pmc_sec_title">Data availability</h3>
<p>The DNA and mRNA expression sequencing data generated by UTSW are available in Supplementary Table S4. The NE score for SCLC cell lines is available in Supplementary Table S6. The data analysed in this study were obtained from Sequence Read Archive (SRA; RRID:SCR_004891) at SRP052896 (<a href="https://www.ncbi.nlm.nih.gov/sra?term=SRP052896" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/sra?term=SRP052896</a>); from database of Genotypes and Phenotypes (dbGaP) (RRID:SCR_002709) at phs001823.v1.p1 (<a href="https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001823.v1.p1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001823.v1.p1</a>); from Dependency Map (DepMap) (RRID:SCR_017655) at 19Q1 (<a href="https://depmap.org/portal/download/all/?releasename=DepMap+Public+19Q1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://depmap.org/portal/download/all/?releasename=DepMap+Public+19Q1</a>), 18Q3 (<a href="https://depmap.org/portal/download/all/?releasename=DepMap+Public+18Q3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://depmap.org/portal/download/all/?releasename=DepMap+Public+18Q3</a>), CCLE2019 (<a href="https://depmap.org/portal/download/all/?releasename=CCLE+2019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://depmap.org/portal/download/all/?releasename=CCLE+2019</a>); from the Broad Institute (RRID:SCR_013836) at CCLE Legacy Data (<a href="https://data.broadinstitute.org/ccle_legacy_data/mRNA_expression/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://data.broadinstitute.org/ccle_legacy_data/mRNA_expression/</a>); and from original articles Nusinow and colleagues (<a href="#bib20" class="usa-link" aria-describedby="bib20" aria-expanded="false">20</a>) at Supplementary Table S2.</p></section></section><section id="sec4"><h2 class="pmc_sec_title" data-anchor-id="sec4"><a id="sec4-anchor" data-anchor-id="sec4" class="usa-anchor"></a>Results</h2>
<section id="sec4-1"><h3 class="pmc_sec_title">Pathogenic <em>SMARCA4</em> mutations are enriched in SCLC-Y</h3>
<p>SCLC almost universally harbors mutations of both <em>TP53</em> and <em>RB1</em>, unusually however, SCLC-Y cell lines frequently lack <em>RB1</em> mutations (<a href="#bib25" class="usa-link" aria-describedby="bib25" aria-expanded="false">25</a>). We therefore interrogated the mutational landscape of 50 SCLC cell lines used to classify the SCLC-Y subtype through the Cancer Cell Line Encyclopaedia (CCLE) and Hamon Center genomic datasets (<a href="#bib19" class="usa-link" aria-describedby="bib19" aria-expanded="false">19</a>) and identified 26 frequently mutated genes unique to SCLC-Y cell lines (<a href="#fig1" class="usa-link">Fig. 1A</a>; Supplementary Table S7). This dataset included all eight YAP1 expressing SCLC lines previously included in the classification of the SCLC-Y lineage (<a href="#bib2" class="usa-link" aria-describedby="bib2" aria-expanded="false">2</a>). Notably, mutations in <em>SMARCA4</em> (also known as <em>BRG1</em>), which encodes an ATPase subunit of SWI/SNF chromatin-remodeling complexes (<a href="#bib26" class="usa-link" aria-describedby="bib26" aria-expanded="false">26</a>), were observed in six out of eight SCLC-Y lines and found mutually exclusive with <em>RB1</em> mutations (<a href="#fig1" class="usa-link">Fig. 1B</a>; Supplementary Fig. S1A). All cancer cell lines harbouring frame shift or nonsense mutations in <em>SMARCA4</em> showed reduced SMARCA4 mRNA and protein abundance (<a href="#fig1" class="usa-link">Fig. 1C</a>; Supplementary Fig. S1B), consistent with the <em>SMARCA4</em> “class 1 alterations” observed in non–small cell lung carcinoma (NSCLC; ref. <a href="#bib27" class="usa-link" aria-describedby="bib27" aria-expanded="false">27</a>). All six <em>SMARCA4</em>-mutant SCLC-Y lines also harboured <em>TP53</em> mutations, but other genomic alterations previously observed to co-occur with <em>SMARCA4</em>-mutant NSCLC, such as <em>STK11</em>, <em>KEAP1</em>, and <em>KRAS</em>, were not identified (Supplementary Table S8). Furthermore, in line with the specific enrichment of <em>SMARCA4</em> mutations in SCLC-Y tumors, <em>SMARCA4</em>-mutant SCLC cell lines exhibited significantly lower expression of a previously defined neuroendocrine (NE) transcriptomic signature (<a href="#bib16" class="usa-link" aria-describedby="bib16" aria-expanded="false">16, 23</a>) compared with <em>SMARCA4</em> wildtype SCLC cell lines (<a href="#fig1" class="usa-link">Fig. 1D</a>; Supplementary Table S6).</p>
<figure class="fig xbox font-sm" id="fig1" title="Figure 1. SMARCA4 mutations are enriched within SCLC-Y cell lines. A, Venn diagram showing co-occurring and subtype-specific mutations associated with each SCLC molecular subtype. Genetic mutations that are present in at least 50% of samples within each SCLC subtype based on Rudin and colleagues (2) were identified through cBioportal. There are 26 mutations exclusive to SCLC-Y cell lines (Supplementary Table S7). B, Of all 26 mutations identified, SMARCA4 mutations were present in six of eight SCLC-Y cell lines, together with TP53 mutations on an RB1 wild-type background. *, H157DM was previously annotated in CCLE as H1339. C, All cell lines in the CCLE (lung and nonlung cancer cell lines) that had complete proteomic and RNA-seq profiles were interrogated for the correlation between SMARCA4 protein and mRNA. There is a positive correlation (Pearson correlation = 0.53; P = 1.5e−27) between low SMARCA4 mRNA and loss of SMARCA4 protein. D, SCLC cell lines with SMARCA4 mutations have a significantly lower NE-score compared with wild-type (WT) SMARCA4 SCLC cell lines (Mann–Whitney test; **, P = 0.0068). SCLC cell lines in the CCLE were binned into SMARCA4 mutant and WT groups. NE scoring was performed using a previously published 50-gene transcriptomic signature (16)."><h4 class="obj_head">Figure 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11061608_1846fig1.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7b43/11061608/042f06e79f69/1846fig1.jpg" loading="lazy" height="571" width="679" alt="Figure 1. SMARCA4 mutations are enriched within SCLC-Y cell lines. A, Venn diagram showing co-occurring and subtype-specific mutations associated with each SCLC molecular subtype. Genetic mutations that are present in at least 50% of samples within each SCLC subtype based on Rudin and colleagues (2) were identified through cBioportal. There are 26 mutations exclusive to SCLC-Y cell lines (Supplementary Table S7). B, Of all 26 mutations identified, SMARCA4 mutations were present in six of eight SCLC-Y cell lines, together with TP53 mutations on an RB1 wild-type background. *, H157DM was previously annotated in CCLE as H1339. C, All cell lines in the CCLE (lung and nonlung cancer cell lines) that had complete proteomic and RNA-seq profiles were interrogated for the correlation between SMARCA4 protein and mRNA. There is a positive correlation (Pearson correlation = 0.53; P = 1.5e−27) between low SMARCA4 mRNA and loss of SMARCA4 protein. D, SCLC cell lines with SMARCA4 mutations have a significantly lower NE-score compared with wild-type (WT) SMARCA4 SCLC cell lines (Mann–Whitney test; **, P = 0.0068). SCLC cell lines in the CCLE were binned into SMARCA4 mutant and WT groups. NE scoring was performed using a previously published 50-gene transcriptomic signature (16)."></a></p>
<div class="p text-right font-secondary"><a href="figure/fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>
<em>SMARCA4</em> mutations are enriched within SCLC-Y cell lines. <strong>A,</strong> Venn diagram showing co-occurring and subtype-specific mutations associated with each SCLC molecular subtype. Genetic mutations that are present in at least 50% of samples within each SCLC subtype based on Rudin and colleagues (<a href="#bib2" class="usa-link" aria-describedby="bib2" aria-expanded="false">2</a>) were identified through cBioportal. There are 26 mutations exclusive to SCLC-Y cell lines (Supplementary Table S7). <strong>B,</strong> Of all 26 mutations identified, <em>SMARCA4</em> mutations were present in six of eight SCLC-Y cell lines, together with <em>TP53</em> mutations on an <em>RB1</em> wild-type background. *, H157DM was previously annotated in CCLE as H1339. <strong>C,</strong> All cell lines in the CCLE (lung and nonlung cancer cell lines) that had complete proteomic and RNA-seq profiles were interrogated for the correlation between SMARCA4 protein and mRNA. There is a positive correlation (Pearson correlation = 0.53; <em>P</em> = 1.5e−27) between low <em>SMARCA4</em> mRNA and loss of SMARCA4 protein. <strong>D,</strong> SCLC cell lines with <em>SMARCA4</em> mutations have a significantly lower NE-score compared with wild-type (WT) <em>SMARCA4</em> SCLC cell lines (Mann–Whitney test; **, <em>P</em> = 0.0068). SCLC cell lines in the CCLE were binned into <em>SMARCA4</em> mutant and WT groups. NE scoring was performed using a previously published 50-gene transcriptomic signature (<a href="#bib16" class="usa-link" aria-describedby="bib16" aria-expanded="false">16</a>).</p></figcaption></figure></section><section id="sec4-2"><h3 class="pmc_sec_title">Histopathologic classification of SMARCA4-deficient SCLC-Y</h3>
<p>Inactivating mutations in <em>SMARCA4</em> are characteristic of small cell carcinoma of the ovary, hypercalcaemic type (SCCOHT) and can also occur in malignant rhabdoid tumors (<a href="#bib24" class="usa-link" aria-describedby="bib24" aria-expanded="false">24</a>). Within the thorax, SMARCA4 loss is seen in two different settings; either in NSCLC harbouring <em>SMARCA4</em> mutations (<a href="#bib27" class="usa-link" aria-describedby="bib27" aria-expanded="false">27</a>), or in thoracic SMARCA4-deficient undifferentiated tumors (previously named SMARCA4-deficient thoracic sarcoma; ref. <a href="#bib24" class="usa-link" aria-describedby="bib24" aria-expanded="false">24</a>). Although SMARCA4-deficient NSCLC (frequently adenocarcinomas) may display histologic features of de-differentiation including loss of TTF1 expression and a solid growth pattern; thoracic SMARCA4-deficient undifferentiated tumors (SMARCA4-UT) show undifferentiated round cell or rhabdoid features and reduced expression of epithelial markers such as cytokeratins and claudin-4. To better define the identity of <em>SMARCA4</em>-mutant SCLC-Y tumors, we established a panel of SCLC xenograft models derived from cell lines representative of all SCLC transcriptional subtypes, including three <em>SMARCA4</em>-mutant SCLC-Y lines. As SCLC diagnosis is primarily based on morphologic assessment of the tumor (<a href="#bib28" class="usa-link" aria-describedby="bib28" aria-expanded="false">28</a>), five pathologists who routinely report thoracic pathology were asked to independently evaluate the de-identified H&amp;E slides and indicate whether the tumor morphology was consistent with SCLC (<a href="#fig2" class="usa-link">Fig. 2A</a>). Although there was some variability in individual pathologist assessments for SCLC-A, -N, and -P tumors, there was near complete consensus that the H&amp;E appearances of four of five SCLC-Y tumors were not consistent with SCLC (<a href="#fig2" class="usa-link">Fig. 2B</a>; Supplementary Figs. S2A and S2B).</p>
<figure class="fig xbox font-sm" id="fig2" title="Figure 2. SMARCA4-deficient SCLC cell lines are morphologically and immunophenotypically similar to SMARCA4-UT. A, Outline of the methodology employed for assessing SMARCA4-deficient SCLC cell lines by a panel of five anatomical pathologists. B, Representative H&amp;E images of cell line xenografts diagnosed as high-grade NE carcinoma (SCLC) and other malignant neoplasms (non-SCLC). Scale bar = 100 μm. C, Heatmap of re-evaluated diagnosis of SCLC cell line xenografts (row) and each pathologists’ classification (column). Abbreviations: LCNEC, large cell neuroendocrine cancer; SMARCA4-UT, SMARCA4-deficient undifferentiated tumor; SCLC/LCNEC, combined SCLC and LCNEC components; Other, undifferentiated tumor or small blue round cell tumor."><h4 class="obj_head">Figure 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11061608_1846fig2.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7b43/11061608/c804f0f47e4b/1846fig2.jpg" loading="lazy" height="449" width="711" alt="Figure 2. SMARCA4-deficient SCLC cell lines are morphologically and immunophenotypically similar to SMARCA4-UT. A, Outline of the methodology employed for assessing SMARCA4-deficient SCLC cell lines by a panel of five anatomical pathologists. B, Representative H&amp;E images of cell line xenografts diagnosed as high-grade NE carcinoma (SCLC) and other malignant neoplasms (non-SCLC). Scale bar = 100 μm. C, Heatmap of re-evaluated diagnosis of SCLC cell line xenografts (row) and each pathologists’ classification (column). Abbreviations: LCNEC, large cell neuroendocrine cancer; SMARCA4-UT, SMARCA4-deficient undifferentiated tumor; SCLC/LCNEC, combined SCLC and LCNEC components; Other, undifferentiated tumor or small blue round cell tumor."></a></p>
<div class="p text-right font-secondary"><a href="figure/fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>SMARCA4-deficient SCLC cell lines are morphologically and immunophenotypically similar to SMARCA4-UT. <strong>A,</strong> Outline of the methodology employed for assessing SMARCA4-deficient SCLC cell lines by a panel of five anatomical pathologists. <strong>B,</strong> Representative H&amp;E images of cell line xenografts diagnosed as high-grade NE carcinoma (SCLC) and other malignant neoplasms (non-SCLC). Scale bar = 100 μm. <strong>C,</strong> Heatmap of re-evaluated diagnosis of SCLC cell line xenografts (row) and each pathologists’ classification (column). Abbreviations: LCNEC, large cell neuroendocrine cancer; SMARCA4-UT, SMARCA4-deficient undifferentiated tumor; SCLC/LCNEC, combined SCLC and LCNEC components; Other, undifferentiated tumor or small blue round cell tumor.</p></figcaption></figure><p>Pathologists were then provided with an IHC panel encompassing standard diagnostic markers of thoracic malignancies (Supplementary Table S2) and asked to provide an updated diagnosis (<a href="#fig2" class="usa-link">Fig. 2C</a>). All three of the SMARCA4-deficient SCLC-Y xenograft tumors (SW1271, H841, and DMS114) were favoured by pathologists to represent either SMARCA4-UT or SMARCA4-deficient NSCLC, rather than SCLC (<a href="#fig2" class="usa-link">Fig. 2C</a>; ref. <a href="#bib15" class="usa-link" aria-describedby="bib15" aria-expanded="false">15</a>). IHC for SMARCA4 confirmed that these three SCLC-Y tumors harboring <em>SMARCA4</em> mutations were SMARCA4-deficient, whereas SMARCA4 protein expression was retained in all other tumors (<a href="#fig3" class="usa-link">Fig. 3</a>; Supplementary Table S2). In contrast to other SCLC subtypes, these SMARCA4-deficient SCLC-Y tumors showed expression of RB1, weak cytokeratin staining and isolated expression of synaptophysin in the absence of other neuroendocrine markers INSM1 and CD56 (<a href="#fig3" class="usa-link">Fig. 3</a>; Supplementary Fig. S3A). This IHC profile would be highly unusual for SCLC but is characteristic of thoracic SMARCA4-UT (<a href="#bib29" class="usa-link" aria-describedby="bib29" aria-expanded="false">29</a>). We observed high concordance between neuroendocrine marker mRNA and protein expression (Supplementary Fig. S3B). Of the two SMARCA4 proficient SCLC-Y lines (H1341 and H196), only H1341 was considered morphologically and immunophenotypically consistent with SCLC (<a href="#fig2" class="usa-link">Fig. 2B</a> and <a href="#fig2" class="usa-link">C</a>; Supplementary Fig. S3C). Conversely, all five pathologists independently designated H196 to be a sarcomatoid (spindle cell) malignancy (<a href="#fig2" class="usa-link">Fig. 2B</a> and <a href="#fig2" class="usa-link">C</a>), with a differential diagnosis including sarcomatoid carcinoma, sarcoma, malignant mesothelioma, and melanoma.</p>
<figure class="fig xbox font-sm" id="fig3" title="Figure 3. Distinct immunophenotype of SMARCA4-mutant compared with SMARCA4-WT SCLC cell lines. Representative IHC images of SMARCA4, synaptophysin (SYN), CD56, TTF1, INSM1, and RB1 in SMARCA4-WT, NE-high SCLC cell line (H1092) compared with SMARCA4-mutant SCLC cell lines. Scale bar = 100 μm."><h4 class="obj_head">Figure 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11061608_1846fig3.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7b43/11061608/73f257c05431/1846fig3.jpg" loading="lazy" height="918" width="790" alt="Figure 3. Distinct immunophenotype of SMARCA4-mutant compared with SMARCA4-WT SCLC cell lines. Representative IHC images of SMARCA4, synaptophysin (SYN), CD56, TTF1, INSM1, and RB1 in SMARCA4-WT, NE-high SCLC cell line (H1092) compared with SMARCA4-mutant SCLC cell lines. Scale bar = 100 μm."></a></p>
<div class="p text-right font-secondary"><a href="figure/fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Distinct immunophenotype of SMARCA4-mutant compared with SMARCA4-WT SCLC cell lines. Representative IHC images of SMARCA4, synaptophysin (SYN), CD56, TTF1, INSM1, and RB1 in <em>SMARCA4</em>-WT, NE-high SCLC cell line (H1092) compared with <em>SMARCA4</em>-mutant SCLC cell lines. Scale bar = 100 μm.</p></figcaption></figure><p>To further validate the identity of the SCLC-Y lines, we reviewed the data in CCLE and compared it to independent DNA-sequencing, RNA-sequencing (RNA-seq) and signature-based histologic analysis of the same tumor lines undertaken at UT Southwestern Medical Center (UTSW; Supplementary Table S4). This comparison demonstrated high concordance between independent sequencing datasets but revealed that two of the eight previously annotated SCLC-Y lines (H1339 and H2286) were derived from tumors with a primary histologic diagnosis of NSCLC tumors rather than SCLC (<a href="#bib30" class="usa-link" aria-describedby="bib30" aria-expanded="false">30</a>). The H157DM (<em>TP53</em><sup>mut</sup>, <em>RB1</em><sup>wt</sup>, <em>SMARCA4</em><sup>truncating</sup>) was previously thought to be the SCLC line H1339 but was more recently identified to be the lung squamous cell carcinoma line H157, an identity that was confirmed on comparison with UTSW data (Supplementary Table S4). In addition, H2286 (<em>TP53</em><sup>mut</sup>, <em>RB1</em><sup>wt</sup>, <em>SMARCA4</em><sup>missense</sup>) was derived from a tumor that showed mixed histology and was classified as an adenocarcinoma by UTSW according to an adenocarcinoma-squamous cell carcinoma RNA-seq signature (<a href="#bib30" class="usa-link" aria-describedby="bib30" aria-expanded="false">30</a>). Thus, in addition to the five SCLC-Y lines directly examined in this study, these findings suggest that an additional two previously designated SCLC-Y lines (H1339/H157DM and H2286) are <em>SMARCA4</em>-mutant NSCLC rather than SCLC. On review of the origin of the final SCLC-Y line H1341, this line was derived from a biopsy of a cervical tumor in 26-year-old female. Importantly, this patient had no lung tumor on CT scan and there was no evidence of disease spread outside the cervix. This tumor was therefore diagnosed as a primary small cell carcinoma of the cervix (<a href="#bib31" class="usa-link" aria-describedby="bib31" aria-expanded="false">31</a>). Consistent with our pathological evaluation of the H1341 xenograft (<a href="#fig2" class="usa-link">Fig. 2B</a>; Supplementary Fig. S2A), the morphologic appearances of small cell neuroendocrine carcinomas (SCNEC) of the cervix can be indistinguishable from SCLC (<a href="#bib32" class="usa-link" aria-describedby="bib32" aria-expanded="false">32</a>). However, cervical SCNEC tumors are typically associated with high-risk human papillomavirus (HPV) infection and have a distinct molecular profile compared with SCLC, with a lower frequency of <em>TP53</em> and <em>RB1</em> mutations (<a href="#bib33" class="usa-link" aria-describedby="bib33" aria-expanded="false">33</a>). In keeping with a diagnosis of cervical SCNEC, H1341 lacks mutations in <em>TP53</em> and <em>RB1</em> and instead possesses mutations frequently seen in cervical SCNEC including a gain-of-function mutations in <em>PIK3CA</em> (E524K) and a <em>PTEN</em> deletion (<a href="#bib22" class="usa-link" aria-describedby="bib22" aria-expanded="false">22, 33–35</a>). Furthermore, H1341 is positive for HPV (<a href="https://www.atcc.org/products/crl-5864" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.atcc.org/products/crl-5864</a>, accessed October 31, 2023). Interestingly, H1341 also possesses a <em>SMARCB1</em> deletion, which has previously been reported in cervical SCNEC (<a href="#bib34" class="usa-link" aria-describedby="bib34" aria-expanded="false">34</a>). Taken together, our findings demonstrate that all cancer cell lines originally used to classify the SCLC-Y subtype show clinical, molecular, morphologic, and/or transcriptomic features that are inconsistent with a diagnosis of SCLC (<a href="#tbl1" class="usa-link">Table 1</a>).</p>
<section class="tw xbox font-sm" id="tbl1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Revised tumor classification for previously designated SCLC-Y lines.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" rowspan="1" colspan="1">Cell line</th>
<th align="left" rowspan="1" colspan="1">DepMap ID</th>
<th align="left" rowspan="1" colspan="1">Revised tumor classification</th>
<th align="left" rowspan="1" colspan="1">Age/sex</th>
<th align="left" rowspan="1" colspan="1">Smoking status</th>
<th align="left" rowspan="1" colspan="1">NE-score (CCLE RNA-seq)</th>
<th align="left" rowspan="1" colspan="1">
<em>TP53</em>
</th>
<th align="left" rowspan="1" colspan="1">
<em>RB1</em>
</th>
<th align="left" rowspan="1" colspan="1">
<em>SMARCA4</em>
</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">H1341</td>
<td align="left" rowspan="1" colspan="1">ACH-000129</td>
<td align="left" rowspan="1" colspan="1">Small cell neuroendocrine carcinoma of the cervix</td>
<td align="left" rowspan="1" colspan="1">26/F</td>
<td align="left" rowspan="1" colspan="1">Smoker</td>
<td align="left" rowspan="1" colspan="1">0.2294</td>
<td align="left" rowspan="1" colspan="1">WT</td>
<td align="left" rowspan="1" colspan="1">WT</td>
<td align="left" rowspan="1" colspan="1">WT</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">DMS114</td>
<td align="left" rowspan="1" colspan="1">ACH-000530</td>
<td align="left" rowspan="1" colspan="1">SMARCA4-UT</td>
<td align="left" rowspan="1" colspan="1">68/M</td>
<td align="left" rowspan="1" colspan="1">Unknown</td>
<td align="left" rowspan="1" colspan="1">0.1365</td>
<td align="left" rowspan="1" colspan="1">
<strong>T</strong>
</td>
<td align="left" rowspan="1" colspan="1">WT</td>
<td align="left" rowspan="1" colspan="1">
<strong>T</strong>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SBC5</td>
<td align="left" rowspan="1" colspan="1">ACH-000670</td>
<td align="left" rowspan="1" colspan="1">SMARCA4-UT</td>
<td align="left" rowspan="1" colspan="1">65/M</td>
<td align="left" rowspan="1" colspan="1">Unknown</td>
<td align="left" rowspan="1" colspan="1">−0.2384</td>
<td align="left" rowspan="1" colspan="1">
<strong>M</strong>
</td>
<td align="left" rowspan="1" colspan="1">WT</td>
<td align="left" rowspan="1" colspan="1">
<strong>T</strong>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">H841</td>
<td align="left" rowspan="1" colspan="1">ACH-000292</td>
<td align="left" rowspan="1" colspan="1">SMARCA4-UT</td>
<td align="left" rowspan="1" colspan="1">51/M</td>
<td align="left" rowspan="1" colspan="1">Smoker</td>
<td align="left" rowspan="1" colspan="1">−0.3397</td>
<td align="left" rowspan="1" colspan="1">
<strong>M</strong>
</td>
<td align="left" rowspan="1" colspan="1">WT</td>
<td align="left" rowspan="1" colspan="1">
<strong>T</strong>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">H2286</td>
<td align="left" rowspan="1" colspan="1">ACH-000912</td>
<td align="left" rowspan="1" colspan="1">Adenocarcinoma</td>
<td align="left" rowspan="1" colspan="1">57/F</td>
<td align="left" rowspan="1" colspan="1">Smoker</td>
<td align="left" rowspan="1" colspan="1">−0.6746</td>
<td align="left" rowspan="1" colspan="1">
<strong>T</strong>
</td>
<td align="left" rowspan="1" colspan="1">WT</td>
<td align="left" rowspan="1" colspan="1">
<strong>M</strong>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">H157DM<a href="#tb1fn1" class="usa-link"><sup>a</sup></a>
</td>
<td align="left" rowspan="1" colspan="1">ACH-000921</td>
<td align="left" rowspan="1" colspan="1">Squamous cell carcinoma</td>
<td align="left" rowspan="1" colspan="1">59/M</td>
<td align="left" rowspan="1" colspan="1">Unknown</td>
<td align="left" rowspan="1" colspan="1">−0.7356</td>
<td align="left" rowspan="1" colspan="1">
<strong>T</strong>
</td>
<td align="left" rowspan="1" colspan="1">WT</td>
<td align="left" rowspan="1" colspan="1">
<strong>T</strong>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">H196</td>
<td align="left" rowspan="1" colspan="1">ACH-000752</td>
<td align="left" rowspan="1" colspan="1">Sarcomatoid malignancy</td>
<td align="left" rowspan="1" colspan="1">68/M</td>
<td align="left" rowspan="1" colspan="1">Nonsmoker</td>
<td align="left" rowspan="1" colspan="1">−0.7438</td>
<td align="left" rowspan="1" colspan="1">
<strong>M</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>T</strong>
</td>
<td align="left" rowspan="1" colspan="1">WT</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SW1271</td>
<td align="left" rowspan="1" colspan="1">ACH-000890</td>
<td align="left" rowspan="1" colspan="1">SMARCA4-deficient malignancy</td>
<td align="left" rowspan="1" colspan="1">69/M</td>
<td align="left" rowspan="1" colspan="1">Unknown</td>
<td align="left" rowspan="1" colspan="1">−0.8004</td>
<td align="left" rowspan="1" colspan="1">
<strong>M</strong>
</td>
<td align="left" rowspan="1" colspan="1">WT</td>
<td align="left" rowspan="1" colspan="1">
<strong>M</strong>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tbl1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="fn3"><p>Abbreviations: F, female; M, male; WT, wild type; T, truncating mutation; M, missense mutation.</p></div>
<div class="fn" id="tb1fn1"><p>
<sup>a</sup>H157DM was previously named H1339 in CCLE.</p></div>
</div></section></section><section id="sec4-3"><h3 class="pmc_sec_title">SMARCA4-deficient SCLC cell lines have a similar transcriptome to SMARCA4-UT</h3>
<p>
<em>SMARCA4</em> and <em>SMARCA2</em> encode mutually exclusive core ATPase subunits of SWI/SNF chromatin remodeling complexes (<a href="#bib26" class="usa-link" aria-describedby="bib26" aria-expanded="false">26, 27</a>). In contrast to SMARCA4-deficient NSCLC in which SMARCA2 is essential for survival (<a href="#bib36" class="usa-link" aria-describedby="bib36" aria-expanded="false">36</a>), SMARCA4-UT and SCCOHT typically exhibit concurrent silencing of <em>SMARCA2</em> expression (<a href="#bib29" class="usa-link" aria-describedby="bib29" aria-expanded="false">29, 37</a>). Disruption of SWI/SNF function in these tumors leads to broad transcriptional dysregulation and remarkably this epigenetic reprogramming appears to be a more dominant driver of phenotype than the tissue of origin. Thus, the transcriptome of thoracic SMARCA4-UT more closely resembles SCCOHT than SMARCA4-deficient NSCLC (<a href="#bib24" class="usa-link" aria-describedby="bib24" aria-expanded="false">24</a>). To investigate whether the transcriptional profile of <em>SMARCA4</em>-mutant SCLC-Y lines are closely related to SMARCA4-UT or to other SCLC subtypes, we integrated RNA sequencing data from all SCLC, NSCLC, and SCCOHT cell lines in CCLE with RNA-seq data from primary tumors, including primary thoracic SMARCA4-UT, SCCOHT, and unclassified thoracic sarcomas (<a href="#bib24" class="usa-link" aria-describedby="bib24" aria-expanded="false">24</a>). Unsupervised hierarchical clustering using a previously defined set of differentially expressed genes in primary SMARCA4-UT compared with SMARCA4-deficient NSCLC (<a href="#bib24" class="usa-link" aria-describedby="bib24" aria-expanded="false">24</a>) revealed clustering of three SMARCA4-deficient SCLC-Y lines (H841, DMS114, SBC5) with primary thoracic SMARCA4-UT and SCCOHT (<a href="#fig4" class="usa-link">Fig. 4A</a>). SBC5 had a similar immunophenotype (Supplementary Fig. S4) to <em>SMARCA4</em>-mutant SCLC-Y cell lines H841, DMS114, and SW1271 (<a href="#fig3" class="usa-link">Fig. 3</a>). Conversely, the other SMARCA4-deficient SCLC-Y lines (SW1271 and H2286) clustered with <em>SMARCA4</em>-mutant NSCLC rather than SCLC (<a href="#fig4" class="usa-link">Fig. 4A</a>). This observation was replicated through principal component (PC) analysis (Supplementary Fig. S5A), with the PC2 dimension found to be influenced by the data source (i.e., CCLE or ref. <a href="#bib24" class="usa-link" aria-describedby="bib24" aria-expanded="false">24</a>; Supplementary Fig. S5B). Using the top 50 genes from PC1, PC3, and PC4 for unsupervised hierarchical clustering using the top 50 genes from PC1, PC3, and PC4 confirmed the clustering of <em>SMARCA4</em>-mutant SCLC-Y cell lines with SMARCA4-UT and SCCOHT samples (Supplementary Fig. S5C).</p>
<figure class="fig xbox font-sm" id="fig4" title="Figure 4. SCLC-Y cell lines with SMARCA4 loss share a similar transcriptome to SMARCA4-UT. A, Unsupervised hierarchical clustering, based on differentially expressed genes in primary SMARCA4-UT compared with primary SMARCA4-mutant NSCLC identified by Le Loarer and colleagues (24; n = 758 genes), was performed on SCCOHT and lung carcinoma samples [cell lines from the CCLE and bulk-RNA sequenced patient samples from Le Loarer and colleagues (24)]. Inset: SMARCA4-UT cluster from A with cell line and sample annotation. B, mRNA expression of SMARCA4, SMARCA2, and CLDN4 (claudin-4) in cell lines of the SMARCA4-UT cluster, SMARCA4 mutant NSCLC, SMARCA4 proficient NSCLC, and SCLC-A, -N, -P (Supplementary Table S5). Kruskal–Wallis test with a Dunn multiple comparison test; *, P < 0.05; **, P < 0.01; ***, P < 0.001; ****, P < 0.0001; ns, P ≥ 0.05. C, IHC of SMARCA2 and claudin-4 in SMARCA4-WT and SMARCA4-mutant cell line xenografts. Scale bar = 100 μm."><h4 class="obj_head">Figure 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11061608_1846fig4.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7b43/11061608/f445d5902b41/1846fig4.jpg" loading="lazy" height="752" width="790" alt="Figure 4. SCLC-Y cell lines with SMARCA4 loss share a similar transcriptome to SMARCA4-UT. A, Unsupervised hierarchical clustering, based on differentially expressed genes in primary SMARCA4-UT compared with primary SMARCA4-mutant NSCLC identified by Le Loarer and colleagues (24; n = 758 genes), was performed on SCCOHT and lung carcinoma samples [cell lines from the CCLE and bulk-RNA sequenced patient samples from Le Loarer and colleagues (24)]. Inset: SMARCA4-UT cluster from A with cell line and sample annotation. B, mRNA expression of SMARCA4, SMARCA2, and CLDN4 (claudin-4) in cell lines of the SMARCA4-UT cluster, SMARCA4 mutant NSCLC, SMARCA4 proficient NSCLC, and SCLC-A, -N, -P (Supplementary Table S5). Kruskal–Wallis test with a Dunn multiple comparison test; *, P < 0.05; **, P < 0.01; ***, P < 0.001; ****, P < 0.0001; ns, P ≥ 0.05. C, IHC of SMARCA2 and claudin-4 in SMARCA4-WT and SMARCA4-mutant cell line xenografts. Scale bar = 100 μm."></a></p>
<div class="p text-right font-secondary"><a href="figure/fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>SCLC-Y cell lines with SMARCA4 loss share a similar transcriptome to SMARCA4-UT. <strong>A,</strong> Unsupervised hierarchical clustering, based on differentially expressed genes in primary SMARCA4-UT compared with primary <em>SMARCA4</em>-mutant NSCLC identified by Le Loarer and colleagues (<a href="#bib24" class="usa-link" aria-describedby="bib24" aria-expanded="false">24</a>; <em>n</em> = 758 genes), was performed on SCCOHT and lung carcinoma samples [cell lines from the CCLE and bulk-RNA sequenced patient samples from Le Loarer and colleagues (<a href="#bib24" class="usa-link" aria-describedby="bib24" aria-expanded="false">24</a>)]. Inset: SMARCA4-UT cluster from <strong>A</strong> with cell line and sample annotation. <strong>B,</strong> mRNA expression of <em>SMARCA4</em>, <em>SMARCA2</em>, and <em>CLDN4</em> (claudin-4) in cell lines of the SMARCA4-UT cluster, SMARCA4 mutant NSCLC, SMARCA4 proficient NSCLC, and SCLC-A, -N, -P (Supplementary Table S5). Kruskal–Wallis test with a Dunn multiple comparison test; *, <em>P</em> &lt; 0.05; **, <em>P</em> &lt; 0.01; ***, <em>P</em> &lt; 0.001; ****, <em>P</em> &lt; 0.0001; ns, <em>P</em> ≥ 0.05. <strong>C,</strong> IHC of SMARCA2 and claudin-4 in <em>SMARCA4</em>-WT and <em>SMARCA4</em>-mutant cell line xenografts. Scale bar = 100 μm.</p></figcaption></figure><p>In addition to the SCLC-Y lines, four <em>SMARCA4</em>-mutant NSCLC lines (H522, H2077, H1581, H661) localized to the SMARCA4-UT/SCCOHT cluster (<a href="#fig4" class="usa-link">Fig. 4A</a>). The primary tumors from which the H1581 and H661 cell lines were derived, lacked morphologic features of differentiated adenocarcinoma or squamous cell carcinoma (Supplementary Fig. S4). Furthermore, all the SMARCA4-deficient lung cancer cell lines localising to this SMARCA4-UT cluster showed a gene expression profile characteristic of SMARCA4-UT, which was distinct from that of SCLC and <em>SMARCA4</em>-mutant NSCLC (<a href="#bib29" class="usa-link" aria-describedby="bib29" aria-expanded="false">29</a>). This included SMARCA4/SMARCA2 co-deficiency, low Claudin-4, variable expression of stem cells markers SALL4, SOX2, and CD34, synaptophysin expression and absent INSM1 and NCAM1 expression (<a href="#fig4" class="usa-link">Fig. 4B</a> and <a href="#fig4" class="usa-link">C</a>; Supplementary Fig. S6A). Taken together, our findings demonstrate that six lines all coming from thoracic malignancies and previously designated as SCLC, harbor <em>SMARCA4</em> mutations and display histologic, and a lack of neuroendocrine features, consistent with either SMARCA4-deficient NSCLC or thoracic SMARCA4-UT rather than SCLC.</p></section><section id="sec4-4"><h3 class="pmc_sec_title">SMARCA4-UT express YAP1</h3>
<p>Although transcriptional expression of YAP1 was initially used to stratify the SCLC-Y subtype (<a href="#bib2" class="usa-link" aria-describedby="bib2" aria-expanded="false">2</a>), IHC staining of SCLC patient samples has largely failed to support YAP1 as a defining molecular marker in SCLC (<a href="#bib8" class="usa-link" aria-describedby="bib8" aria-expanded="false">8</a>). To first correlate YAP1 mRNA and protein expression, we performed YAP1 IHC on <em>SMARCA4</em>-mutant (SW1271, H841, DMS114, H661, and H1581) and <em>SMARCA4</em>-wildtype (H2171, H1092, and H526) cell line xenografts. Consistent with the high YAP1 mRNA expression seen in SCLC-Y cell lines (<a href="#bib2" class="usa-link" aria-describedby="bib2" aria-expanded="false">2</a>), strong YAP1 protein expression was observed in all tumor cell lines within the SMARCA4-UT cluster and NSCLC cluster (<a href="#fig5" class="usa-link">Fig. 5A</a>). Interestingly, the cervical SCNEC line H1341 showed patchy expression of YAP1 and had the strongest neuroendocrine marker expression of the previously designated SCLC-Y lines (Supplementary Fig. S6B; <a href="#tbl1" class="usa-link">Table 1</a>).</p>
<figure class="fig xbox font-sm" id="fig5" title="Figure 5. YAP1 is expressed in SMARCA4-UT primary samples. A, Unsupervised hierarchical clustering, based on a SCLC-specific YAP1 signature (n = 49 genes) identified by Pearsall and colleagues (10), was performed on SCCOHT and lung carcinoma samples [cell lines from the CCLE and bulk-RNA sequenced patient samples from Le Loarer and colleagues (24)]. Cell lines that cluster with the SMARCA4-UT samples are highlighted in the gray box. B, YAP1 IHC on SCLC-A/N/P and SMARCA4-mutant cell line xenografts. Scale bar = 100 μm. C, H&amp;E, SMARCA4, and YAP1 IHC in primary SMARCA4-UT samples (n = 6) together with an SCLC patient sample. Scale bar = 100 μm."><h4 class="obj_head">Figure 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11061608_1846fig5.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7b43/11061608/68220905a62f/1846fig5.jpg" loading="lazy" height="699" width="789" alt="Figure 5. YAP1 is expressed in SMARCA4-UT primary samples. A, Unsupervised hierarchical clustering, based on a SCLC-specific YAP1 signature (n = 49 genes) identified by Pearsall and colleagues (10), was performed on SCCOHT and lung carcinoma samples [cell lines from the CCLE and bulk-RNA sequenced patient samples from Le Loarer and colleagues (24)]. Cell lines that cluster with the SMARCA4-UT samples are highlighted in the gray box. B, YAP1 IHC on SCLC-A/N/P and SMARCA4-mutant cell line xenografts. Scale bar = 100 μm. C, H&amp;E, SMARCA4, and YAP1 IHC in primary SMARCA4-UT samples (n = 6) together with an SCLC patient sample. Scale bar = 100 μm."></a></p>
<div class="p text-right font-secondary"><a href="figure/fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>YAP1 is expressed in SMARCA4-UT primary samples. <strong>A,</strong> Unsupervised hierarchical clustering, based on a SCLC-specific YAP1 signature (<em>n</em> = 49 genes) identified by Pearsall and colleagues (<a href="#bib10" class="usa-link" aria-describedby="bib10" aria-expanded="false">10</a>), was performed on SCCOHT and lung carcinoma samples [cell lines from the CCLE and bulk-RNA sequenced patient samples from Le Loarer and colleagues (<a href="#bib24" class="usa-link" aria-describedby="bib24" aria-expanded="false">24</a>)]. Cell lines that cluster with the SMARCA4-UT samples are highlighted in the gray box. <strong>B,</strong> YAP1 IHC on SCLC-A/N/P and <em>SMARCA4</em>-mutant cell line xenografts. Scale bar = 100 μm. <strong>C,</strong> H&amp;E, SMARCA4, and YAP1 IHC in primary SMARCA4-UT samples (<em>n</em> = 6) together with an SCLC patient sample. Scale bar = 100 μm.</p></figcaption></figure><p>To validate whether our findings in cell line xenografts are representative of primary SMARCA4-UT tumors in patients, we initially made use of a YAP1 signature, previously used to interrogated YAP1 expression in SCLC Circulating Tumor Xenograft (CTX) models (<a href="#bib10" class="usa-link" aria-describedby="bib10" aria-expanded="false">10</a>). As expected, SCLC cell lines representing the -A, -N, and -P subtypes exhibited low YAP1 signature expression and displayed a distinct clustering pattern (<a href="#fig5" class="usa-link">Fig. 5B</a>). Conversely, <em>SMARCA4</em>-mutant SCLC-Y cell lines clustered with primary thoracic SMARCA4-UT and SCCOHT patient samples (<a href="#fig5" class="usa-link">Fig. 5B</a>; Supplementary Table S9; <a href="#fig5" class="usa-link">Fig. 5B</a>). Moreover, <em>YAP1</em> mRNA expression was comparable between SMARCA4-UT and <em>SMARCA4</em>-mutant NSCLC (Supplementary Fig. S6A), consistent with the proposed oncogenic role of YAP1 in NSCLC (<a href="#bib38" class="usa-link" aria-describedby="bib38" aria-expanded="false">38</a>). To validate these findings at the protein level, we performed IHC staining for YAP1 on primary SMARCA4-UT patient samples (<em>n</em> = 6; <a href="#fig5" class="usa-link">Fig. 5C</a>). Indeed, in addition to displaying a characteristic undifferentiated morphology with loss of SMARCA4 expression, SMARCA4-UT patient material showed strong expression of YAP1 (<a href="#fig5" class="usa-link">Fig. 5C</a>). Critically, this was in stark contrast to the lack of YAP1 expression seen in a primary SCLC tumor, where high YAP1 expression is detected specifically in endothelial and stromal cells (<a href="#fig5" class="usa-link">Fig. 5C</a>; ref. <a href="#bib8" class="usa-link" aria-describedby="bib8" aria-expanded="false">8</a>). Our findings demonstrate that, in contrast to SCLC (<a href="#bib8" class="usa-link" aria-describedby="bib8" aria-expanded="false">8</a>), both xenografts of SMARCA4-mutant cell lines previously classified as SCLC-Y and patient-derived primary SMARCA4-UT tumor samples show diffuse, uniformly strong YAP1 protein staining. Thus, altogether our findings indicate that diffuse YAP1 expression is a consistent feature of SMARCA4-UT, explaining how previous erroneous annotation of both SMARCA4-UT cell lines and YAP1-positive NSCLC lines as SCLC, led to the grouping of these lines within a distinct YAP1-positive SCLC subgroup.</p></section></section><section id="sec5"><h2 class="pmc_sec_title" data-anchor-id="sec5"><a id="sec5-anchor" data-anchor-id="sec5" class="usa-anchor"></a>Discussion</h2>
<p>Understanding SCLC heterogeneity and the distinct biology of the proposed SCLC subtypes is critical for the development of targeted therapies for this devastating disease. Here, we shed light on the origins of the proposed controversial “SCLC-Y” subtype, making the surprising observation that the majority of SCLC-Y tumor cell lines, on which the classification of SCLC-Y category was largely based, harbour <em>SMARCA4</em> mutations. Detailed molecular and histopathological characterisation of these tumors revealed that these tumors show features in keeping with SMARCA4-deficient malignancies rather than SCLC.</p>
<p>Several of the SCLC-Y tumor lines showed close transcriptional similarities with primary thoracic SMARCA4-UT. These tumors showed pathologic features consistent with SMARCA4-UT and gene expression profiles consistent with this diagnosis, including loss of both SMARCA4 and SMARCA2 expression and low claudin-4. In contrast to SCLC-A, -N, and -P subtypes, these SMARCA4-deficient SCLC-Y tumors retained RB1 expression and expressed synaptophysin, but lacked both CD56 and INSM1, a profile that is characteristic of SMARCA4-UT. Interestingly, <em>SMARCA4</em> inactivating mutations were recently detected in 1.5% of SCLC patient samples (<a href="#bib39" class="usa-link" aria-describedby="bib39" aria-expanded="false">39</a>). Although these tumors also harbored <em>TP53</em> and <em>RB1</em> genetic alternations, importantly accompanying histopathology was not available, thus the inclusion of SCLC mimics, such as SMARCA4-UT could not be excluded.</p>
<p>The classification of SCLC subtypes was based both on primary SCLC samples as well as SCLC cell lines, however, cell lines were particularly enriched within the SCLC-Y category (<a href="#bib2" class="usa-link" aria-describedby="bib2" aria-expanded="false">2</a>). All of these lines were developed between 1975 and 1991 from patients clinically diagnosed as having SCLC, in nearly every case by pathologists recognized as world experts in SCLC diagnosis working as part of a group with extensive experience in SCLC clinical trials (NCI-Navy Medical Oncology Branch; ref. <a href="#bib40" class="usa-link" aria-describedby="bib40" aria-expanded="false">40</a>). Nevertheless, these diagnoses were made well before our current knowledge of the key genomic features and IHC markers characteristic of SCLC and other lung tumors, such as SMARCA4 deficient malignancies. Thoracic SMARCA4-UT is a recognized mimic of SCLC, and 23% of SMARCA4-UT in a recent series were initially diagnosed as small cell or large cell neuroendocrine carcinoma (<a href="#bib29" class="usa-link" aria-describedby="bib29" aria-expanded="false">29</a>). Like SCLC, SMARCA4-UT typically occurs in middle-aged smokers and can mimic several histological features of SCLC including small cell morphology, high proliferation index, crush artefact, and synaptophysin expression (<a href="#bib29" class="usa-link" aria-describedby="bib29" aria-expanded="false">29</a>). These features may account for the original SCLC diagnosis, however the morphology and IHC profile of the SMARCA4-deficient SCLC-Y xenografts were unanimously considered by a panel of pathologists to be consistent with thoracic SMARCA4-UT or SMARCA4-deficient carcinoma as opposed to SCLC. Although we did not identify <em>SMARCA4</em> mutations in two of the eight SCLC-Y lines, one of these lines was pathologically not consistent with SCLC and the other was derived from a 26-year-old patient diagnosed with a primary small cell carcinoma of the cervix rather than SCLC. Thus, altogether we found little evidence to support a diagnosis of SCLC for any of the SCLC-Y lines tested (<a href="#tbl1" class="usa-link">Table 1</a>).</p>
<p>The expression of YAP1 is inversely correlated with the expression of neuroendocrine markers in SCLC, and thus “classical” neuroendocrine high SCLC lacks expression of YAP1. In contrast, YAP1 is expressed in NSCLC, including adenocarcinoma, squamous cell carcinoma, and a proportion of large cell neuroendocrine carcinomas (<a href="#bib8" class="usa-link" aria-describedby="bib8" aria-expanded="false">8, 41</a>). Importantly, here we also demonstrate that primary SMARCA4-UT universally exhibit diffuse and strong expression of YAP1 protein. The expression of YAP1 in multiple lung malignancies therefore complicates the use of YAP1 expression to define a specific subtype of SCLC, particularly given the occurrence of combined tumors in which YAP1 expressing NSCLC may be admixed with SCLC, and the existence of YAP1-positive tumors that can mimic SCLC histologically, such as basaloid squamous cell carcinoma, poorly differentiated adenocarcinoma, high-grade adenoid cystic carcinoma, and SMARCA4-UT.</p>
<p>Altogether, our findings suggest that, unlike ASCL1, NEUROD1, and POU2F3, YAP1 is not a subtype defining transcription factor in SCLC. This is consistent with a recent study in a patient cohort, which failed to identify a distinct YAP1 expressing SCLC subtype (<a href="#bib8" class="usa-link" aria-describedby="bib8" aria-expanded="false">8</a>). Although focal YAP1 expression has been detected in some primary SCLC samples (<a href="#bib11" class="usa-link" aria-describedby="bib11" aria-expanded="false">11</a>), our findings together with recent studies in SCLC xenograft and GEM models suggests that this is due to intratumoral heterogeneity (<a href="#bib9" class="usa-link" aria-describedby="bib9" aria-expanded="false">9, 10, 13, 42</a>). In this context, classical ASCL1 driven SCLC can transition to a “neuroendocrine low” phenotype, which is associated with expression of YAP1. This phenotypic plasticity is a feature of <em>RB1</em> null SCLC, and the emergence of neuroendocrine low, YAP1 expressing cells has been associated with chemoresistance, activation of Notch signaling, and expression of mesenchymal and inflammation-associated genes (<a href="#bib10" class="usa-link" aria-describedby="bib10" aria-expanded="false">10, 13, 42</a>).</p>
<p>Importantly, our findings demonstrate that the patient-derived cancer cell lines initially used to define the SCLC-Y subtype actually represent SMARCA4-UT, NSCLC, or other SCLC mimics, and therefore are not representative models of “triple negative” or “inflamed” SCLC. We also stress that a subset of human SCLC tumors exhibit a triple A/N/P negative phenotype, low neuroendocrine marker expression and a more “inflamed” gene signature. However, there is little evidence to date supporting the use of YAP1 as a reliable marker for these tumors. Although the majority of CCLE cell lines defined as SCLC-Y appear to be SMARCA4-deficient malignancies, this is likely because these tumors were diagnosed historically, without access to immunohistochemical and molecular testing used today and before SMARCA4-deficient lung malignancies were a recognized disease entity. We therefore do not anticipate that “triple negative” or “SCLC-I” tumors identified in recent, thoroughly histopathologically characterized SCLC cohorts are likely to represent SMARCA4-UT. However, our work highlights the importance of comprehensive histopathologic and molecular characterization of SCLC tumors subtyped in clinical cohorts, with particular focus on excluding potential SCLC mimics for tumors that lack neuroendocrine marker expression or have an unusual molecular background, for example retained expression of wild-type RB.</p>
<p>SMARCA4-UT is a highly aggressive lung malignancy for which effective treatments are needed. Furthermore, these tumors exhibit shared transcriptomic and phenotypic features with aggressive SMARCA4-deficient malignancies occurring at other sites, including SSCOHT and malignant rhabdoid tumor. SMARCA4-UT typically present as a large central thoracic tumor involving the pulmonary hilum and/or mediastinum in young to middle-aged smokers. Histologically the tumor consists of sheets of variably discohesive epithelioid cells, which typically have prominent nucleoli and frequently show focal rhabdoid morphology. SMARCA4-UT lack clear evidence of epithelial differentiation and the characteristic diagnostic IHC profile is weak or absent expression of epithelial markers such as Cytokeratins and Claudin-4, negative or focal TTF-1 or p40 staining, and loss of SMARCA4 expression. In addition, some cases show expression of stem cell markers such as CD34, SOX2, or SALL4 (<a href="#bib15" class="usa-link" aria-describedby="bib15" aria-expanded="false">15</a>). Approximately 70% of SMARCA4-UT show IHC staining for synaptophysin but other neuroendocrine markers such as CD56 and INSM1 are typically negative (<a href="#bib29" class="usa-link" aria-describedby="bib29" aria-expanded="false">29</a>).</p>
<p>Given the broad use of the SCLC-Y cell lines we now identify to be SMARCA4-UT in therapeutic studies, our findings provide new insight into potential therapeutic vulnerabilities in SMARCA4-deficient malignancies. For example, the IL-15 super-agonist, N-808 (<a href="#bib43" class="usa-link" aria-describedby="bib43" aria-expanded="false">43</a>), arginine deprivation (<a href="#bib44" class="usa-link" aria-describedby="bib44" aria-expanded="false">44</a>), and inhibition of Aurora kinase B (<a href="#bib45" class="usa-link" aria-describedby="bib45" aria-expanded="false">45</a>), checkpoint kinase 1 (CHK1; ref. <a href="#bib46" class="usa-link" aria-describedby="bib46" aria-expanded="false">46</a>) have all demonstrated anticancer activity in SMARCA4-deficient SCLC-Y cell lines. Furthermore, findings from preclinical studies employing these SMARCA4-deficient SCLC-Y cell lines have been used as a basis to initiate clinical trials in SCLC, highlighting the clinical importance of resolving the identity of these tumors (<a href="#bib47" class="usa-link" aria-describedby="bib47" aria-expanded="false">47, 48</a>). In contrast to SCLC, emerging evidence suggests that SMARCA4-UT may respond poorly to chemotherapy (<a href="#bib49" class="usa-link" aria-describedby="bib49" aria-expanded="false">49, 50</a>). Conversely, several of the SCLC-Y lines that we now identify to be SMARCA4-UT have previously been shown to be sensitive to CDK4/6 inhibitors (<a href="#bib25" class="usa-link" aria-describedby="bib25" aria-expanded="false">25</a>), and consistent with this SMARCA4-deficiency, has been shown to be a strong predictor of sensitivity to CDK4/6 inhibitors in NSCLC and SCCOHT tumor models (<a href="#bib51" class="usa-link" aria-describedby="bib51" aria-expanded="false">51, 52</a>). The cell lines we have identified as SMARCA4-UT have higher expression of immune and MHC antigen presentation genes than classical neuroendocrine SCLC-A and SCLC-N, raising the possibility that these tumors may respond to immunotherapy. Interestingly, there are several reports of SMARCA4-UT exhibiting responses to immune checkpoint blockade with anti-PD-1 or anti-PD-L1 therapy (<a href="#bib53" class="usa-link" aria-describedby="bib53" aria-expanded="false">53, 54</a>).</p>
<p>Taken together, these SMARCA4-deficient cell lines previously characterized as SCLC-Y may serve as patient derived preclinical models of SMARCA4-UT to accelerate the discovery of new therapeutics for this aggressive malignancy.</p></section><section id="sec20"><h2 class="pmc_sec_title" data-anchor-id="sec20"><a id="sec20-anchor" data-anchor-id="sec20" class="usa-anchor"></a>Supplementary Material</h2>
<section class="sm xbox font-sm" id="supplementary-material1"><div class="caption p">
<span>Supplementary Figure S1</span><p>Characterisation of SMARCA4 mutations in lung cancer cell lines from the CCLE.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11061608/bin/ccr-23-2360_supplementary_figure_s1_suppsf1.pdf" data-ga-action="click_feat_suppl" class="usa-link">ccr-23-2360_supplementary_figure_s1_suppsf1.pdf</a><sup> (1.5MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="supplementary-material2"><div class="caption p">
<span>Supplementary Figure S2</span><p>Initial histopathological evaluation of SCLC xenografts.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11061608/bin/ccr-23-2360_supplementary_figure_s2_suppsf2.pdf" data-ga-action="click_feat_suppl" class="usa-link">ccr-23-2360_supplementary_figure_s2_suppsf2.pdf</a><sup> (2.2MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="supplementary-material3"><div class="caption p">
<span>Supplementary Figure S3</span><p>SCLC-specific markers are weak/lost in SMARCA4-deficient SCLC-Y cell lines.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11061608/bin/ccr-23-2360_supplementary_figure_s3_suppsf3.pdf" data-ga-action="click_feat_suppl" class="usa-link">ccr-23-2360_supplementary_figure_s3_suppsf3.pdf</a><sup> (1.2MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="supplementary-material4"><div class="caption p">
<span>Supplementary Figure S4</span><p>H&amp;E and immunohistochemistry of cell line xenografts H661, H1581 and SBC5.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11061608/bin/ccr-23-2360_supplementary_figure_s4_suppsf4.pdf" data-ga-action="click_feat_suppl" class="usa-link">ccr-23-2360_supplementary_figure_s4_suppsf4.pdf</a><sup> (16.5MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="supplementary-material5"><div class="caption p">
<span>Supplementary Figure S5</span><p>SMARCA4-UT cluster validation.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11061608/bin/ccr-23-2360_supplementary_figure_s5_suppsf5.pdf" data-ga-action="click_feat_suppl" class="usa-link">ccr-23-2360_supplementary_figure_s5_suppsf5.pdf</a><sup> (1.2MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="supplementary-material6"><div class="caption p">
<span>Supplementary Figure S6</span><p>mRNA expression of SMARCA4-UT associated genes and YAP1 IHC.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11061608/bin/ccr-23-2360_supplementary_figure_s6_suppsf6.pdf" data-ga-action="click_feat_suppl" class="usa-link">ccr-23-2360_supplementary_figure_s6_suppsf6.pdf</a><sup> (5.3MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="supplementary-material7"><div class="caption p">
<span>Supplementary Table S1</span><p>Immunohistochemistry antibodies, protocols and evaluation criteria.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11061608/bin/ccr-23-2360_supplementary_table_s1_suppst1.xlsx" data-ga-action="click_feat_suppl" class="usa-link">ccr-23-2360_supplementary_table_s1_suppst1.xlsx</a><sup> (13KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="supplementary-material8"><div class="caption p">
<span>Supplementary Table S2</span><p>Immunohistochemistry scoring table.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11061608/bin/ccr-23-2360_supplementary_table_s2_suppst2.xlsx" data-ga-action="click_feat_suppl" class="usa-link">ccr-23-2360_supplementary_table_s2_suppst2.xlsx</a><sup> (11.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="supplementary-material9"><div class="caption p">
<span>Supplementary Table S3</span><p>Clinical profiles and immunohistochemistry of SMARCA4-UT patients.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11061608/bin/ccr-23-2360_supplementary_table_s3_suppst3.xlsx" data-ga-action="click_feat_suppl" class="usa-link">ccr-23-2360_supplementary_table_s3_suppst3.xlsx</a><sup> (10.1KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="supplementary-material10"><div class="caption p">
<span>Supplementary Table S4</span><p>Comparison of sequencing data between CCLE and UTSW.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11061608/bin/ccr-23-2360_supplementary_table_s4_suppst4.xlsx" data-ga-action="click_feat_suppl" class="usa-link">ccr-23-2360_supplementary_table_s4_suppst4.xlsx</a><sup> (23KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="supplementary-material11"><div class="caption p">
<span>Supplementary Table S5</span><p>CCLE cell lines used in Figure 4 and Supplementary Figure S6A.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11061608/bin/ccr-23-2360_supplementary_table_s5_suppst5.xlsx" data-ga-action="click_feat_suppl" class="usa-link">ccr-23-2360_supplementary_table_s5_suppst5.xlsx</a><sup> (11.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="supplementary-material12"><div class="caption p">
<span>Supplementary Table S6</span><p>List of cell lines from Figure 1D and their NE-score.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11061608/bin/ccr-23-2360_supplementary_table_s6_suppst6.xlsx" data-ga-action="click_feat_suppl" class="usa-link">ccr-23-2360_supplementary_table_s6_suppst6.xlsx</a><sup> (13.9KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="supplementary-material13"><div class="caption p">
<span>Supplementary Table S7</span><p>26 mutated genes unique to SCLC-Y cell lines.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11061608/bin/ccr-23-2360_supplementary_table_s7_suppst7.xlsx" data-ga-action="click_feat_suppl" class="usa-link">ccr-23-2360_supplementary_table_s7_suppst7.xlsx</a><sup> (10KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="supplementary-material14"><div class="caption p">
<span>Supplementary Table S8</span><p>Other mutations and copy number alterations associated with SCLC-Y.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11061608/bin/ccr-23-2360_supplementary_table_s8_suppst8.xlsx" data-ga-action="click_feat_suppl" class="usa-link">ccr-23-2360_supplementary_table_s8_suppst8.xlsx</a><sup> (11.1KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="supplementary-material15"><div class="caption p">
<span>Supplementary Table S9</span><p>SMARCA4-UT cell lines and patient samples based on unsupervised hierarchical clustering using YAP1 signature genes.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11061608/bin/ccr-23-2360_supplementary_table_s9_suppst9.xlsx" data-ga-action="click_feat_suppl" class="usa-link">ccr-23-2360_supplementary_table_s9_suppst9.xlsx</a><sup> (10.5KB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title" data-anchor-id="ack1"><a id="ack1-anchor" data-anchor-id="ack1" class="usa-anchor"></a>Acknowledgments</h2>
<p>We thank E. Tsui and the WEHI Histology Facility for providing IHC expertise. We are grateful to D. Boyd, L. Scott, and R. Monaghan for technical support and animal husbandry, and M. Papari-Zareei and V. Stasny for aid in searching for the original biopsies for each cell line and preservation of SCLC lines. K.D. Sutherland was supported by the Peter and Julie Alston Centenary Fellowship and an Australian National Health and Medical Research Council (NHMRC) Project Grant 1159955. M.L. Burr was supported by a Snow Fellowship from the Snow Medical Research Foundation, NHMRC Investigator Grant 1196598 and Project Grant 1164054. K.D. Sutherland and M.L. Burr are supported by NHMRC Synergy Grant 2010275. J.D. Minna, L. Girard, and L. Cai are supported by P50 CA070907, J.D. Minna by CA213338 and CA213274. The bioinformatic analysis was funded by the Cancer Prevention and Research Institute of Texas (RP150596) and UTSW ACS-IRG (IRG-21–142–16). This work was made possible through the Victorian Government Operational Infrastructure Support and Australian Government.</p>
<p>The publication costs of this article were defrayed in part by the payment of publication fees. Therefore, and solely to indicate this fact, this article is hereby marked “advertisement” in accordance with 18 USC section 1734.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title" data-anchor-id="fn-group1"><a id="fn-group1-anchor" data-anchor-id="fn-group1" class="usa-anchor"></a>Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn5"><p>
<strong>Note:</strong> Supplementary data for this article are available at Clinical Cancer Research Online (http://clincancerres.aacrjournals.org/).</p></div></div></section><section id="sec6"><h2 class="pmc_sec_title" data-anchor-id="sec6"><a id="sec6-anchor" data-anchor-id="sec6" class="usa-anchor"></a>Authors' Disclosures</h2>
<p>J.D. Minna reports grants from NCI and personal fees from NIH and University of Texas Southwestern Medical Center during the conduct of the study. M.L. Burr reports grants from Snow Medical Research Foundation and NHMRC during the conduct of the study. K.D. Sutherland reports grants from National Health and Medical Research Council (NHMRC) and Peter and Julie Alston Centenary Fellowship during the conduct of the study. No disclosures were reported by the other authors.</p></section><section id="sec7"><h2 class="pmc_sec_title" data-anchor-id="sec7"><a id="sec7-anchor" data-anchor-id="sec7" class="usa-anchor"></a>Authors' Contributions</h2>
<p>
<strong>J. Ng:</strong> Conceptualization, data curation, formal analysis, investigation, visualization, methodology, writing–original draft, writing–review and editing. <strong>L. Cai:</strong> Data curation, software, formal analysis, funding acquisition, investigation, visualization, methodology, writing–original draft, writing–review and editing. <strong>L. Girard:</strong> Data curation, software, formal analysis, writing–review and editing. <strong>O.W.J. Prall:</strong> Resources, investigation, writing–review and editing. <strong>N. Rajan:</strong> Investigation, writing–review and editing. <strong>C. Khoo:</strong> Investigation, writing–review and editing. <strong>A. Batrouney:</strong> Investigation, writing–review and editing. <strong>D.J. Byrne:</strong> Investigation, writing–review and editing. <strong>D.K. Boyd:</strong> Investigation, writing–review and editing. <strong>A.J. Kersbergen:</strong> Investigation, writing–review and editing. <strong>M. Christie:</strong> Resources, supervision, investigation, writing–review and editing. <strong>J.D. Minna:</strong> Resources, supervision, funding acquisition, validation, writing–original draft, project administration, writing–review and editing. <strong>M.L. Burr:</strong> Conceptualization, resources, data curation, formal analysis, supervision, funding acquisition, validation, investigation, visualization, methodology, writing–original draft, project administration, writing–review and editing. <strong>K.D. Sutherland:</strong> Conceptualization, resources, formal analysis, supervision, funding acquisition, validation, visualization, methodology, writing–original draft, project administration, writing–review and editing.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title" data-anchor-id="ref-list1"><a id="ref-list1-anchor" data-anchor-id="ref-list1" class="usa-anchor"></a>References</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="bib1">
<span class="label">1.</span><cite>
Rudin CM, Brambilla E, Faivre-Finn C, Sage J. Small-cell lung cancer. 
Nat Rev Dis Primers 2021;7:1–20.
</cite> [<a href="https://doi.org/10.1038/s41572-020-00235-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8177722/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33446664/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Dis%20Primers&amp;title=Small-cell%20lung%20cancer&amp;volume=7&amp;publication_year=2021&amp;pages=1-20&amp;pmid=33446664&amp;doi=10.1038/s41572-020-00235-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib2">
<span class="label">2.</span><cite>
Rudin CM, Poirier JT, Byers LA, Dive C, Dowlati A, George J, 
et al. Molecular subtypes of small cell lung cancer: a synthesis of human and mouse model data. 
Nat Rev Cancer 2019;19:289–97.
</cite> [<a href="https://doi.org/10.1038/s41568-019-0133-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6538259/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30926931/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Cancer&amp;title=Molecular%20subtypes%20of%20small%20cell%20lung%20cancer:%20a%20synthesis%20of%20human%20and%20mouse%20model%20data&amp;volume=19&amp;publication_year=2019&amp;pages=289-97&amp;pmid=30926931&amp;doi=10.1038/s41568-019-0133-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib3">
<span class="label">3.</span><cite>
Poirier JT, George J, Owonikoko TK, Berns A, Brambilla E, Byers LA, 
et al. New approaches to SCLC therapy: from the laboratory to the clinic. 
J Thorac Oncol 2020;15:520–40.
</cite> [<a href="https://doi.org/10.1016/j.jtho.2020.01.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7263769/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32018053/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Thorac%20Oncol&amp;title=New%20approaches%20to%20SCLC%20therapy:%20from%20the%20laboratory%20to%20the%20clinic&amp;volume=15&amp;publication_year=2020&amp;pages=520-40&amp;pmid=32018053&amp;doi=10.1016/j.jtho.2020.01.016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib4">
<span class="label">4.</span><cite>
Megyesfalvi Z, Gay CM, Popper H, Pirker R, Ostoros G, Heeke S, 
et al. Clinical insights into small cell lung cancer: tumor heterogeneity, diagnosis, therapy, and future directions. 
CA Cancer J Clin 2023;73:620–52.
</cite> [<a href="https://doi.org/10.3322/caac.21785" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37329269/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=CA%20Cancer%20J%20Clin&amp;title=Clinical%20insights%20into%20small%20cell%20lung%20cancer:%20tumor%20heterogeneity,%20diagnosis,%20therapy,%20and%20future%20directions&amp;volume=73&amp;publication_year=2023&amp;pages=620-52&amp;pmid=37329269&amp;doi=10.3322/caac.21785&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib5">
<span class="label">5.</span><cite>
Rudin CM, Reck M, Johnson ML, Blackhall F, Hann CL, Yang JC-H, 
et al. Emerging therapies targeting the Delta-like ligand 3 (DLL3) in small cell lung cancer. 
J Hematol Oncol 2023;16:66.
</cite> [<a href="https://doi.org/10.1186/s13045-023-01464-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10290806/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37355629/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Hematol%20Oncol&amp;title=Emerging%20therapies%20targeting%20the%20Delta-like%20ligand%203%20(DLL3)%20in%20small%20cell%20lung%20cancer&amp;volume=16&amp;publication_year=2023&amp;pages=66&amp;pmid=37355629&amp;doi=10.1186/s13045-023-01464-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib6">
<span class="label">6.</span><cite>
Rudin CM, Durinck S, Stawiski EW, Poirier JT, Modrusan Z, Shames DS, 
et al. Comprehensive genomic analysis identifies SOX2 as a frequently amplified gene in small-cell lung cancer. 
Nat Genet 2012;44:1111–6.
</cite> [<a href="https://doi.org/10.1038/ng.2405" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3557461/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22941189/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&amp;title=Comprehensive%20genomic%20analysis%20identifies%20SOX2%20as%20a%20frequently%20amplified%20gene%20in%20small-cell%20lung%20cancer&amp;volume=44&amp;publication_year=2012&amp;pages=1111-6&amp;pmid=22941189&amp;doi=10.1038/ng.2405&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib7">
<span class="label">7.</span><cite>
George J, Lim JS, Jang SJ, Cun Y, Ozretić L, Kong G, 
et al. Comprehensive genomic profiles of small cell lung cancer. 
Nature 2015;524:47–53.
</cite> [<a href="https://doi.org/10.1038/nature14664" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4861069/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26168399/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Comprehensive%20genomic%20profiles%20of%20small%20cell%20lung%20cancer&amp;volume=524&amp;publication_year=2015&amp;pages=47-53&amp;pmid=26168399&amp;doi=10.1038/nature14664&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib8">
<span class="label">8.</span><cite>
Baine MK, Hsieh M-S, Lai WV, Egger JV, Jungbluth AA, Daneshbod Y, 
et al. SCLC subtypes defined by ASCL1, NEUROD1, POU2F3, and YAP1: a comprehensive immunohistochemical and histopathologic characterization. 
J Thorac Oncol 2020;15:1823–35.
</cite> [<a href="https://doi.org/10.1016/j.jtho.2020.09.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8362797/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33011388/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Thorac%20Oncol&amp;title=SCLC%20subtypes%20defined%20by%20ASCL1,%20NEUROD1,%20POU2F3,%20and%20YAP1:%20a%20comprehensive%20immunohistochemical%20and%20histopathologic%20characterization&amp;volume=15&amp;publication_year=2020&amp;pages=1823-35&amp;pmid=33011388&amp;doi=10.1016/j.jtho.2020.09.009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib9">
<span class="label">9.</span><cite>
Ireland AS, Micinski AM, Kastner DW, Guo B, Wait SJ, Spainhower KB, 
et al. MYC drives temporal evolution of small cell lung cancer subtypes by reprogramming neuroendocrine fate. 
Cancer Cell 2020;38:60–78.
</cite> [<a href="https://doi.org/10.1016/j.ccell.2020.05.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7393942/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32473656/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&amp;title=MYC%20drives%20temporal%20evolution%20of%20small%20cell%20lung%20cancer%20subtypes%20by%20reprogramming%20neuroendocrine%20fate&amp;volume=38&amp;publication_year=2020&amp;pages=60-78&amp;pmid=32473656&amp;doi=10.1016/j.ccell.2020.05.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib10">
<span class="label">10.</span><cite>
Pearsall SM, Humphrey S, Revill M, Morgan D, Frese KK, Galvin M, 
et al. The rare YAP1 subtype of SCLC revisited in a biobank of 39 circulating tumor cell patient derived explant models: a brief report. 
J Thorac Oncol 2020;15:1836–43.
</cite> [<a href="https://doi.org/10.1016/j.jtho.2020.07.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7718082/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32721553/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Thorac%20Oncol&amp;title=The%20rare%20YAP1%20subtype%20of%20SCLC%20revisited%20in%20a%20biobank%20of%2039%20circulating%20tumor%20cell%20patient%20derived%20explant%20models:%20a%20brief%20report&amp;volume=15&amp;publication_year=2020&amp;pages=1836-43&amp;pmid=32721553&amp;doi=10.1016/j.jtho.2020.07.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib11">
<span class="label">11.</span><cite>
Owonikoko TK, Dwivedi B, Chen Z, Zhang C, Barwick B, Ernani V, 
et al. YAP1 expression in SCLC defines a distinct subtype with T-cell–inflamed phenotype. 
J Thorac Oncol 2021;16:464–76.
</cite> [<a href="https://doi.org/10.1016/j.jtho.2020.11.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7920957/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33248321/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Thorac%20Oncol&amp;title=YAP1%20expression%20in%20SCLC%20defines%20a%20distinct%20subtype%20with%20T-cell%E2%80%93inflamed%20phenotype&amp;volume=16&amp;publication_year=2021&amp;pages=464-76&amp;pmid=33248321&amp;doi=10.1016/j.jtho.2020.11.006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib12">
<span class="label">12.</span><cite>
Qu S, Fetsch P, Thomas A, Pommier Y, Schrump DS, Miettinen MM, 
et al. Molecular subtypes of primary SCLC tumors and their associations with neuroendocrine and therapeutic markers. 
J Thorac Oncol 2022;17:141–53.
</cite> [<a href="https://doi.org/10.1016/j.jtho.2021.08.763" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8692365/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34534680/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Thorac%20Oncol&amp;title=Molecular%20subtypes%20of%20primary%20SCLC%20tumors%20and%20their%20associations%20with%20neuroendocrine%20and%20therapeutic%20markers&amp;volume=17&amp;publication_year=2022&amp;pages=141-53&amp;pmid=34534680&amp;doi=10.1016/j.jtho.2021.08.763&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib13">
<span class="label">13.</span><cite>
Gay CM, Stewart CA, Park EM, Diao L, Groves SM, Heeke S, 
et al. Patterns of transcription factor programs and immune pathway activation define four major subtypes of SCLC with distinct therapeutic vulnerabilities. 
Cancer Cell 2021;39:346–60.
</cite> [<a href="https://doi.org/10.1016/j.ccell.2020.12.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8143037/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33482121/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&amp;title=Patterns%20of%20transcription%20factor%20programs%20and%20immune%20pathway%20activation%20define%20four%20major%20subtypes%20of%20SCLC%20with%20distinct%20therapeutic%20vulnerabilities&amp;volume=39&amp;publication_year=2021&amp;pages=346-60&amp;pmid=33482121&amp;doi=10.1016/j.ccell.2020.12.014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib14">
<span class="label">14.</span><cite>
Rekhtman N. Lung neuroendocrine neoplasms: recent progress and persistent challenges. 
Mod Pathol 2022;35:36–50.
</cite> [<a href="https://doi.org/10.1038/s41379-021-00943-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8695375/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34663914/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mod%20Pathol&amp;title=Lung%20neuroendocrine%20neoplasms:%20recent%20progress%20and%20persistent%20challenges&amp;volume=35&amp;publication_year=2022&amp;pages=36-50&amp;pmid=34663914&amp;doi=10.1038/s41379-021-00943-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib15">
<span class="label">15.</span><cite>
Nicholson AG, Tsao MS, Beasley MB, Borczuk AC, Brambilla E, Cooper WA, 
et al. The 2021 WHO classification of lung tumors: impact of advances since 2015. 
J Thorac Oncol 2022;17:362–87.
</cite> [<a href="https://doi.org/10.1016/j.jtho.2021.11.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34808341/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Thorac%20Oncol&amp;title=The%202021%20WHO%20classification%20of%20lung%20tumors:%20impact%20of%20advances%20since%202015&amp;volume=17&amp;publication_year=2022&amp;pages=362-87&amp;pmid=34808341&amp;doi=10.1016/j.jtho.2021.11.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib16">
<span class="label">16.</span><cite>
Cai L, Liu H, Huang F, Fujimoto J, Girard L, Chen J, 
et al. Cell-autonomous immune gene expression is repressed in pulmonary neuroendocrine cells and small cell lung cancer. 
Commun Biol 2021;4:1–13.
</cite> [<a href="https://doi.org/10.1038/s42003-021-01842-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7943563/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33750914/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Commun%20Biol&amp;title=Cell-autonomous%20immune%20gene%20expression%20is%20repressed%20in%20pulmonary%20neuroendocrine%20cells%20and%20small%20cell%20lung%20cancer&amp;volume=4&amp;publication_year=2021&amp;pages=1-13&amp;pmid=33750914&amp;doi=10.1038/s42003-021-01842-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib17">
<span class="label">17.</span><cite>
Tlemsani C, Pongor L, Elloumi F, Girard L, Huffman KE, Roper N, 
et al. SCLC-CellMiner: a resource for small cell lung cancer cell line genomics and pharmacology based on genomic signatures. 
Cell Rep 2020;33:108296.
</cite> [<a href="https://doi.org/10.1016/j.celrep.2020.108296" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7643325/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33086069/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Rep&amp;title=SCLC-CellMiner:%20a%20resource%20for%20small%20cell%20lung%20cancer%20cell%20line%20genomics%20and%20pharmacology%20based%20on%20genomic%20signatures&amp;volume=33&amp;publication_year=2020&amp;pages=108296&amp;pmid=33086069&amp;doi=10.1016/j.celrep.2020.108296&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib18">
<span class="label">18.</span><cite>
Tian Y, Li Q, Yang Z, Zhang S, Xu J, Wang Z, 
et al. Single-cell transcriptomic profiling reveals the tumor heterogeneity of small-cell lung cancer. 
Sig Transduct Target Ther 2022;7:1–16.</cite> [<a href="https://doi.org/10.1038/s41392-022-01150-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9532437/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36195615/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sig%20Transduct%20Target%20Ther&amp;title=Single-cell%20transcriptomic%20profiling%20reveals%20the%20tumor%20heterogeneity%20of%20small-cell%20lung%20cancer&amp;volume=7&amp;publication_year=2022&amp;pages=1-16&amp;pmid=36195615&amp;doi=10.1038/s41392-022-01150-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib19">
<span class="label">19.</span><cite>
Ghandi M, Huang FW, Jané-Valbuena J, Kryukov GV, Lo CC, McDonald ER, 
et al. Next-generation characterization of the cancer cell line encyclopedia. 
Nature 2019;569:503–8.
</cite> [<a href="https://doi.org/10.1038/s41586-019-1186-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6697103/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31068700/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Next-generation%20characterization%20of%20the%20cancer%20cell%20line%20encyclopedia&amp;volume=569&amp;publication_year=2019&amp;pages=503-8&amp;pmid=31068700&amp;doi=10.1038/s41586-019-1186-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib20">
<span class="label">20.</span><cite>
Nusinow DP, Szpyt J, Ghandi M, Rose CM, McDonald ER, Kalocsay M, 
et al. Quantitative proteomics of the cancer cell line encyclopedia. 
Cell 2020;180:387–402.
</cite> [<a href="https://doi.org/10.1016/j.cell.2019.12.023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7339254/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31978347/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Quantitative%20proteomics%20of%20the%20cancer%20cell%20line%20encyclopedia&amp;volume=180&amp;publication_year=2020&amp;pages=387-402&amp;pmid=31978347&amp;doi=10.1016/j.cell.2019.12.023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib21">
<span class="label">21.</span><cite>
Cerami E, Gao J, Dogrusoz U, Gross BE, Sumer SO, Aksoy BA, 
et al. The cBio cancer genomics portal: an open platform for exploring multidimensional cancer genomics data. 
Cancer Discov 2012;2:401–4.
</cite> [<a href="https://doi.org/10.1158/2159-8290.CD-12-0095" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3956037/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22588877/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Discov&amp;title=The%20cBio%20cancer%20genomics%20portal:%20an%20open%20platform%20for%20exploring%20multidimensional%20cancer%20genomics%20data&amp;volume=2&amp;publication_year=2012&amp;pages=401-4&amp;pmid=22588877&amp;doi=10.1158/2159-8290.CD-12-0095&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib22">
<span class="label">22.</span><cite>
Gao J, Aksoy BA, Dogrusoz U, Dresdner G, Gross B, Sumer SO, 
et al. Integrative analysis of complex cancer genomics and clinical profiles using the cBioPortal. 
Sci Signal 2013;6:pl1–.
</cite> [<a href="https://doi.org/10.1126/scisignal.2004088" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4160307/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23550210/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Signal&amp;title=Integrative%20analysis%20of%20complex%20cancer%20genomics%20and%20clinical%20profiles%20using%20the%20cBioPortal&amp;volume=6&amp;publication_year=2013&amp;pages=pl1%E2%80%93&amp;pmid=23550210&amp;doi=10.1126/scisignal.2004088&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib23">
<span class="label">23.</span><cite>
Zhang W, Girard L, Zhang Y-A, Haruki T, Papari-Zareei M, Stastny V, 
et al. Small cell lung cancer tumors and preclinical models display heterogeneity of neuroendocrine phenotypes. 
Transl Lung Cancer Res 2018;7:32–49.
</cite> [<a href="https://doi.org/10.21037/tlcr.2018.02.02" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5835590/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29535911/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Transl%20Lung%20Cancer%20Res&amp;title=Small%20cell%20lung%20cancer%20tumors%20and%20preclinical%20models%20display%20heterogeneity%20of%20neuroendocrine%20phenotypes&amp;volume=7&amp;publication_year=2018&amp;pages=32-49&amp;pmid=29535911&amp;doi=10.21037/tlcr.2018.02.02&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib24">
<span class="label">24.</span><cite>
Le Loarer F, Watson S, Pierron G, de Montpreville VT, Ballet S, Firmin N, 
et al. SMARCA4 inactivation defines a group of undifferentiated thoracic malignancies transcriptionally related to BAF-deficient sarcomas. 
Nat Genet 2015;47:1200–5.
</cite> [<a href="https://doi.org/10.1038/ng.3399" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26343384/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&amp;title=SMARCA4%20inactivation%20defines%20a%20group%20of%20undifferentiated%20thoracic%20malignancies%20transcriptionally%20related%20to%20BAF-deficient%20sarcomas&amp;volume=47&amp;publication_year=2015&amp;pages=1200-5&amp;pmid=26343384&amp;doi=10.1038/ng.3399&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib25">
<span class="label">25.</span><cite>
McColl K, Wildey G, Sakre N, Lipka MB, Behtaj M, Kresak A, 
et al. Reciprocal expression of INSM1 and YAP1 defines subgroups in small cell lung cancer. 
Oncotarget 2017;8:73745–56.
</cite> [<a href="https://doi.org/10.18632/oncotarget.20572" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5650296/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29088741/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncotarget&amp;title=Reciprocal%20expression%20of%20INSM1%20and%20YAP1%20defines%20subgroups%20in%20small%20cell%20lung%20cancer&amp;volume=8&amp;publication_year=2017&amp;pages=73745-56&amp;pmid=29088741&amp;doi=10.18632/oncotarget.20572&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib26">
<span class="label">26.</span><cite>
Pan J, McKenzie ZM, D'Avino AR, Mashtalir N, Lareau CA, Pierre RS, 
et al. The ATPase module of mammalian SWI/SNF family complexes mediates subcomplex identity and catalytic activity–independent genomic targeting. 
Nat Genet 2019;51:618–26.
</cite> [<a href="https://doi.org/10.1038/s41588-019-0363-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6755913/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30858614/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&amp;title=The%20ATPase%20module%20of%20mammalian%20SWI/SNF%20family%20complexes%20mediates%20subcomplex%20identity%20and%20catalytic%20activity%E2%80%93independent%20genomic%20targeting&amp;volume=51&amp;publication_year=2019&amp;pages=618-26&amp;pmid=30858614&amp;doi=10.1038/s41588-019-0363-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib27">
<span class="label">27.</span><cite>
Schoenfeld AJ, Bandlamudi C, Lavery JA, Montecalvo J, Namakydoust A, Rizvi H, 
et al. The genomic landscape of SMARCA4 alterations and associations with outcomes in patients with lung cancer. 
Clin Cancer Res 2020;26:5701–8.
</cite> [<a href="https://doi.org/10.1158/1078-0432.CCR-20-1825" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7641983/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32709715/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&amp;title=The%20genomic%20landscape%20of%20SMARCA4%20alterations%20and%20associations%20with%20outcomes%20in%20patients%20with%20lung%20cancer&amp;volume=26&amp;publication_year=2020&amp;pages=5701-8&amp;pmid=32709715&amp;doi=10.1158/1078-0432.CCR-20-1825&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib28">
<span class="label">28.</span><cite>
Nicholson AG, Scagliotti G, Tsao MS, Yatabe Y, Travis WD. 2021 WHO classification of lung cancer: a globally applicable and molecular biomarker-relevant classification. 
J Thorac Oncol 2022;17:e80–3.
</cite> [<a href="https://doi.org/10.1016/j.jtho.2022.07.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36031295/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Thorac%20Oncol&amp;title=2021%20WHO%20classification%20of%20lung%20cancer:%20a%20globally%20applicable%20and%20molecular%20biomarker-relevant%20classification&amp;volume=17&amp;publication_year=2022&amp;pages=e80-3&amp;pmid=36031295&amp;doi=10.1016/j.jtho.2022.07.006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib29">
<span class="label">29.</span><cite>
Rekhtman N, Montecalvo J, Chang JC, Alex D, Ptashkin RN, Ai N, 
et al. SMARCA4-deficient thoracic sarcomatoid tumors represent primarily smoking-related undifferentiated carcinomas rather than primary thoracic sarcomas. 
J Thorac Oncol 2020;15:231–47.
</cite> [<a href="https://doi.org/10.1016/j.jtho.2019.10.023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7556987/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31751681/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Thorac%20Oncol&amp;title=SMARCA4-deficient%20thoracic%20sarcomatoid%20tumors%20represent%20primarily%20smoking-related%20undifferentiated%20carcinomas%20rather%20than%20primary%20thoracic%20sarcomas&amp;volume=15&amp;publication_year=2020&amp;pages=231-47&amp;pmid=31751681&amp;doi=10.1016/j.jtho.2019.10.023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib30">
<span class="label">30.</span><cite>
Girard L, Rodriguez-Canales J, Behrens C, Thompson DM, Botros IW, Tang H, 
et al. An expression signature as an aid to the histologic classification of non–small cell lung cancer. 
Clin Cancer Res 2016;22:4880–9.
</cite> [<a href="https://doi.org/10.1158/1078-0432.CCR-15-2900" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5492382/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27354471/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&amp;title=An%20expression%20signature%20as%20an%20aid%20to%20the%20histologic%20classification%20of%20non%E2%80%93small%20cell%20lung%20cancer&amp;volume=22&amp;publication_year=2016&amp;pages=4880-9&amp;pmid=27354471&amp;doi=10.1158/1078-0432.CCR-15-2900&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib31">
<span class="label">31.</span><cite>
Johnson BE, Whang-Peng J, Naylor SL, Zbar B, Brauch H, Lee E, 
et al. Retention of chromosome 3 in extrapulmonary small cell cancer shown by molecular and cytogenetic studies. 
J Natl Cancer Inst 1989;81:1223–8.
</cite> [<a href="https://doi.org/10.1093/jnci/81.16.1223" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2569043/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Natl%20Cancer%20Inst&amp;title=Retention%20of%20chromosome%203%20in%20extrapulmonary%20small%20cell%20cancer%20shown%20by%20molecular%20and%20cytogenetic%20studies&amp;volume=81&amp;publication_year=1989&amp;pages=1223-8&amp;pmid=2569043&amp;doi=10.1093/jnci/81.16.1223&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib32">
<span class="label">32.</span><cite>
WHO Classification of Tumours Editorial Board. 
Female genital tumours [Internet]. 5th ed.Lyon, France: 
International Agency for Research on Cancer; 
2020. Available from: <a href="https://publications.iarc.fr/592" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://publications.iarc.fr/592</a></cite> [<a href="https://scholar.google.com/scholar_lookup?title=Female%20genital%20tumours&amp;publication_year=2020&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib33">
<span class="label">33.</span><cite>
Pei X, Xiang L, Chen W, Jiang W, Yin L, Shen X, 
et al. The next generation sequencing of cancer-related genes in small cell neuroendocrine carcinoma of the cervix. 
Gynecol Oncol 2021;161:779–86.
</cite> [<a href="https://doi.org/10.1016/j.ygyno.2021.04.019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33888337/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Gynecol%20Oncol&amp;title=The%20next%20generation%20sequencing%20of%20cancer-related%20genes%20in%20small%20cell%20neuroendocrine%20carcinoma%20of%20the%20cervix&amp;volume=161&amp;publication_year=2021&amp;pages=779-86&amp;pmid=33888337&amp;doi=10.1016/j.ygyno.2021.04.019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib34">
<span class="label">34.</span><cite>
Frumovitz M, Burzawa JK, Byers LA, Lyons YA, Ramalingam P, Coleman RC, 
et al. Sequencing of mutational hotspots in cancer-related genes in small cell neuroendocrine cervical cancer. 
Gynecol Oncol 2016;141:588–91.
</cite> [<a href="https://doi.org/10.1016/j.ygyno.2016.04.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4877250/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27079212/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Gynecol%20Oncol&amp;title=Sequencing%20of%20mutational%20hotspots%20in%20cancer-related%20genes%20in%20small%20cell%20neuroendocrine%20cervical%20cancer&amp;volume=141&amp;publication_year=2016&amp;pages=588-91&amp;pmid=27079212&amp;doi=10.1016/j.ygyno.2016.04.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib35">
<span class="label">35.</span><cite>
Xing D, Zheng G, Schoolmeester JK, Li Z, Pallavajjala A, Haley L, 
et al. Next-generation sequencing reveals recurrent somatic mutations in small cell neuroendocrine carcinoma of the uterine cervix. 
Am J Surg Pathol 2018;42:750.
</cite> [<a href="https://doi.org/10.1097/PAS.0000000000001042" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5943084/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29505425/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Am%20J%20Surg%20Pathol&amp;title=Next-generation%20sequencing%20reveals%20recurrent%20somatic%20mutations%20in%20small%20cell%20neuroendocrine%20carcinoma%20of%20the%20uterine%20cervix&amp;volume=42&amp;publication_year=2018&amp;pages=750&amp;pmid=29505425&amp;doi=10.1097/PAS.0000000000001042&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib36">
<span class="label">36.</span><cite>
Hoffman GR, Rahal R, Buxton F, Xiang K, McAllister G, Frias E, 
et al. Functional epigenetics approach identifies BRM/SMARCA2 as a critical synthetic lethal target in BRG1-deficient cancers. 
Proc Natl Acad Sci USA 2014;111:3128–33.
</cite> [<a href="https://doi.org/10.1073/pnas.1316793111" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3939885/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24520176/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&amp;title=Functional%20epigenetics%20approach%20identifies%20BRM/SMARCA2%20as%20a%20critical%20synthetic%20lethal%20target%20in%20BRG1-deficient%20cancers&amp;volume=111&amp;publication_year=2014&amp;pages=3128-33&amp;pmid=24520176&amp;doi=10.1073/pnas.1316793111&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib37">
<span class="label">37.</span><cite>
Karnezis AN, Wang Y, Ramos P, Hendricks WP, Oliva E, D'Angelo E, 
et al. Dual loss of the SWI/SNF complex ATPases SMARCA4/BRG1 and SMARCA2/BRM is highly sensitive and specific for small cell carcinoma of the ovary, hypercalcaemic type. 
J Pathol 2016;238:389–400.
</cite> [<a href="https://doi.org/10.1002/path.4633" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4832362/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26356327/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Pathol&amp;title=Dual%20loss%20of%20the%20SWI/SNF%20complex%20ATPases%20SMARCA4/BRG1%20and%20SMARCA2/BRM%20is%20highly%20sensitive%20and%20specific%20for%20small%20cell%20carcinoma%20of%20the%20ovary,%20hypercalcaemic%20type&amp;volume=238&amp;publication_year=2016&amp;pages=389-400&amp;pmid=26356327&amp;doi=10.1002/path.4633&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib38">
<span class="label">38.</span><cite>
Lo Sardo F, Strano S, Blandino G. YAP and TAZ in lung cancer: oncogenic role and clinical targeting. 
Cancers (Basel) 2018;10:137.
</cite> [<a href="https://doi.org/10.3390/cancers10050137" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5977110/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29734788/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancers%20(Basel)&amp;title=YAP%20and%20TAZ%20in%20lung%20cancer:%20oncogenic%20role%20and%20clinical%20targeting&amp;volume=10&amp;publication_year=2018&amp;pages=137&amp;pmid=29734788&amp;doi=10.3390/cancers10050137&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib39">
<span class="label">39.</span><cite>
Sivakumar S, Moore JA, Montesion M, Sharaf R, Lin DI, Colón CI, 
et al. Integrative analysis of a large real-world cohort of small cell lung cancer identifies distinct genetic subtypes and insights into histologic transformation. 
Cancer Discov 2023;13:1572–91.
</cite> [<a href="https://doi.org/10.1158/2159-8290.CD-22-0620" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10326603/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37062002/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Discov&amp;title=Integrative%20analysis%20of%20a%20large%20real-world%20cohort%20of%20small%20cell%20lung%20cancer%20identifies%20distinct%20genetic%20subtypes%20and%20insights%20into%20histologic%20transformation&amp;volume=13&amp;publication_year=2023&amp;pages=1572-91&amp;pmid=37062002&amp;doi=10.1158/2159-8290.CD-22-0620&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib40">
<span class="label">40.</span><cite>
Carney DN, Gazdar AF, Bepler G, Guccion JG, Marangos PJ, Moody TW, 
et al. Establishment and identification of small cell lung cancer cell lines having classic and variant features. 
Cancer Res 1985;45:2913–23.
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/2985257/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&amp;title=Establishment%20and%20identification%20of%20small%20cell%20lung%20cancer%20cell%20lines%20having%20classic%20and%20variant%20features&amp;volume=45&amp;publication_year=1985&amp;pages=2913-23&amp;pmid=2985257&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib41">
<span class="label">41.</span><cite>
Ito T, Matsubara D, Tanaka I, Makiya K, Tanei Z, Kumagai Y, 
et al. Loss of YAP1 defines neuroendocrine differentiation of lung tumors. 
Cancer Sci 2016;107:1527–38.
</cite> [<a href="https://doi.org/10.1111/cas.13013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5084673/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27418196/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Sci&amp;title=Loss%20of%20YAP1%20defines%20neuroendocrine%20differentiation%20of%20lung%20tumors&amp;volume=107&amp;publication_year=2016&amp;pages=1527-38&amp;pmid=27418196&amp;doi=10.1111/cas.13013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib42">
<span class="label">42.</span><cite>
Shue YT, Drainas AP, Li NY, Pearsall SM, Morgan D, Sinnott-Armstrong N, 
et al. A conserved YAP/Notch/REST network controls the neuroendocrine cell fate in the lungs. 
Nat Commun 2022;13:2690.
</cite> [<a href="https://doi.org/10.1038/s41467-022-30416-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9110333/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35577801/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=A%20conserved%20YAP/Notch/REST%20network%20controls%20the%20neuroendocrine%20cell%20fate%20in%20the%20lungs&amp;volume=13&amp;publication_year=2022&amp;pages=2690&amp;pmid=35577801&amp;doi=10.1038/s41467-022-30416-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib43">
<span class="label">43.</span><cite>
Fousek K, Horn LA, Qin H, Dahut M, Iida M, Yacubovich D, 
et al. An interleukin-15 superagonist enables antitumor efficacy of natural killer cells against all molecular variants of SCLC. 
J Thorac Oncol 2023;18:350–68.
</cite> [<a href="https://doi.org/10.1016/j.jtho.2022.11.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9974560/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36410696/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Thorac%20Oncol&amp;title=An%20interleukin-15%20superagonist%20enables%20antitumor%20efficacy%20of%20natural%20killer%20cells%20against%20all%20molecular%20variants%20of%20SCLC&amp;volume=18&amp;publication_year=2023&amp;pages=350-68&amp;pmid=36410696&amp;doi=10.1016/j.jtho.2022.11.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib44">
<span class="label">44.</span><cite>
Kelly MP, Jungbluth AA, Wu B-W, Bomalaski J, Old LJ, Ritter G. Arginine deiminase PEG20 inhibits growth of small cell lung cancers lacking expression of argininosuccinate synthetase. 
Br J Cancer 2012;106:324–32.
</cite> [<a href="https://doi.org/10.1038/bjc.2011.524" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3261683/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22134507/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&amp;title=Arginine%20deiminase%20PEG20%20inhibits%20growth%20of%20small%20cell%20lung%20cancers%20lacking%20expression%20of%20argininosuccinate%20synthetase&amp;volume=106&amp;publication_year=2012&amp;pages=324-32&amp;pmid=22134507&amp;doi=10.1038/bjc.2011.524&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib45">
<span class="label">45.</span><cite>
Helfrich BA, Kim J, Gao D, Chan DC, Zhang Z, Tan A-C, 
et al. Barasertib (AZD1152), a small molecule Aurora B inhibitor, inhibits the growth of SCLC cell lines in vitro and in vivo. 
Mol Cancer Ther 2016;15:2314–22.
</cite> [<a href="https://doi.org/10.1158/1535-7163.MCT-16-0298" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5050114/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27496133/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Cancer%20Ther&amp;title=Barasertib%20(AZD1152),%20a%20small%20molecule%20Aurora%20B%20inhibitor,%20inhibits%20the%20growth%20of%20SCLC%20cell%20lines%20in%20vitro%20and%20in%20vivo&amp;volume=15&amp;publication_year=2016&amp;pages=2314-22&amp;pmid=27496133&amp;doi=10.1158/1535-7163.MCT-16-0298&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib46">
<span class="label">46.</span><cite>
Hsu W-H, Zhao X, Zhu J, Kim I-K, Rao G, McCutcheon J, 
et al. Checkpoint kinase 1 inhibition enhances cisplatin cytotoxicity and overcomes cisplatin resistance in SCLC by promoting mitotic cell death. 
J Thorac Oncol 2019;14:1032–45.
</cite> [<a href="https://doi.org/10.1016/j.jtho.2019.01.028" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6534433/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30771522/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Thorac%20Oncol&amp;title=Checkpoint%20kinase%201%20inhibition%20enhances%20cisplatin%20cytotoxicity%20and%20overcomes%20cisplatin%20resistance%20in%20SCLC%20by%20promoting%20mitotic%20cell%20death&amp;volume=14&amp;publication_year=2019&amp;pages=1032-45&amp;pmid=30771522&amp;doi=10.1016/j.jtho.2019.01.028&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib47">
<span class="label">47.</span><cite>
Wildey G, Shay AM, McColl KS, Yoon S, Shatat MA, Perwez A, 
et al. Retinoblastoma expression and targeting by CDK4/6 inhibitors in small cell lung cancer. 
Mol Cancer Ther 2023;22:264–73.
</cite> [<a href="https://doi.org/10.1158/1535-7163.MCT-22-0365" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9898162/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36399634/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Cancer%20Ther&amp;title=Retinoblastoma%20expression%20and%20targeting%20by%20CDK4/6%20inhibitors%20in%20small%20cell%20lung%20cancer&amp;volume=22&amp;publication_year=2023&amp;pages=264-73&amp;pmid=36399634&amp;doi=10.1158/1535-7163.MCT-22-0365&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib48">
<span class="label">48.</span><cite>
Guo C, Wan R, He Y, Lin S-H, Cao J, Qiu Y, 
et al. Therapeutic targeting of the mevalonate–geranylgeranyl diphosphate pathway with statins overcomes chemotherapy resistance in small cell lung cancer. 
Nat Cancer 2022;3:614–28.
</cite> [<a href="https://doi.org/10.1038/s43018-022-00358-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35449308/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Cancer&amp;title=Therapeutic%20targeting%20of%20the%20mevalonate%E2%80%93geranylgeranyl%20diphosphate%20pathway%20with%20statins%20overcomes%20chemotherapy%20resistance%20in%20small%20cell%20lung%20cancer&amp;volume=3&amp;publication_year=2022&amp;pages=614-28&amp;pmid=35449308&amp;doi=10.1038/s43018-022-00358-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib49">
<span class="label">49.</span><cite>
Henon C, Blay J-Y, Massard C, Mir O, Bahleda R, Dumont S, 
et al. Long lasting major response to pembrolizumab in a thoracic malignant rhabdoid-like SMARCA4-deficient tumor. 
Ann Oncol 2019;30:1401–3.
</cite> [<a href="https://doi.org/10.1093/annonc/mdz160" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31114851/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ann%20Oncol&amp;title=Long%20lasting%20major%20response%20to%20pembrolizumab%20in%20a%20thoracic%20malignant%20rhabdoid-like%20SMARCA4-deficient%20tumor&amp;volume=30&amp;publication_year=2019&amp;pages=1401-3&amp;pmid=31114851&amp;doi=10.1093/annonc/mdz160&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib50">
<span class="label">50.</span><cite>
Iijima Y, Sakakibara R, Ishizuka M, Honda T, Shirai T, Okamoto T, 
et al. Notable response to nivolumab during the treatment of SMARCA4-deficient thoracic sarcoma: a case report. 
Immunotherapy 2020;12:563–9.
</cite> [<a href="https://doi.org/10.2217/imt-2019-0142" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32363992/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Immunotherapy&amp;title=Notable%20response%20to%20nivolumab%20during%20the%20treatment%20of%20SMARCA4-deficient%20thoracic%20sarcoma:%20a%20case%20report&amp;volume=12&amp;publication_year=2020&amp;pages=563-9&amp;pmid=32363992&amp;doi=10.2217/imt-2019-0142&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib51">
<span class="label">51.</span><cite>
Xue Y, Meehan B, Fu Z, Wang XQD, Fiset PO, Rieker R, 
et al. SMARCA4 loss is synthetic lethal with CDK4/6 inhibition in non-small cell lung cancer. 
Nat Commun 2019;10:1–13.
</cite> [<a href="https://doi.org/10.1038/s41467-019-08380-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6362083/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30718506/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=SMARCA4%20loss%20is%20synthetic%20lethal%20with%20CDK4/6%20inhibition%20in%20non-small%20cell%20lung%20cancer&amp;volume=10&amp;publication_year=2019&amp;pages=1-13&amp;pmid=30718506&amp;doi=10.1038/s41467-019-08380-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib52">
<span class="label">52.</span><cite>
Xue Y, Meehan B, Macdonald E, Venneti S, Wang XQD, Witkowski L, 
et al. CDK4/6 inhibitors target SMARCA4-determined cyclin D1 deficiency in hypercalcemic small cell carcinoma of the ovary. 
Nat Commun 2019;10:558.
</cite> [<a href="https://doi.org/10.1038/s41467-018-06958-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6361890/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30718512/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=CDK4/6%20inhibitors%20target%20SMARCA4-determined%20cyclin%20D1%20deficiency%20in%20hypercalcemic%20small%20cell%20carcinoma%20of%20the%20ovary&amp;volume=10&amp;publication_year=2019&amp;pages=558&amp;pmid=30718512&amp;doi=10.1038/s41467-018-06958-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib53">
<span class="label">53.</span><cite>
Shi L, Lin L, Ding Y, Zeng Y, Chen X. Case report: a rapid response to immunotherapy in a thoracic SMARCA4-deficient undifferentiated tumor with respiratory failure. 
Front Oncol 2022;12:1020875.
</cite> [<a href="https://doi.org/10.3389/fonc.2022.1020875" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9703460/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36452500/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Oncol&amp;title=Case%20report:%20a%20rapid%20response%20to%20immunotherapy%20in%20a%20thoracic%20SMARCA4-deficient%20undifferentiated%20tumor%20with%20respiratory%20failure&amp;volume=12&amp;publication_year=2022&amp;pages=1020875&amp;pmid=36452500&amp;doi=10.3389/fonc.2022.1020875&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib54">
<span class="label">54.</span><cite>
Kunimasa K, Okami J, Takenaka S, Honma K, Kukita Y, Nagata S, 
et al. Conversion surgery for advanced thoracic SMARCA4-deficient undifferentiated tumor with atezolizumab in combination with bevacizumab, paclitaxel, and carboplatin treatment: a case report. 
JTO Clin Res Rep 2021;2:100235.
</cite> [<a href="https://doi.org/10.1016/j.jtocrr.2021.100235" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8551843/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34746887/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=JTO%20Clin%20Res%20Rep&amp;title=Conversion%20surgery%20for%20advanced%20thoracic%20SMARCA4-deficient%20undifferentiated%20tumor%20with%20atezolizumab%20in%20combination%20with%20bevacizumab,%20paclitaxel,%20and%20carboplatin%20treatment:%20a%20case%20report&amp;volume=2&amp;publication_year=2021&amp;pages=100235&amp;pmid=34746887&amp;doi=10.1016/j.jtocrr.2021.100235&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title" data-anchor-id="_ad93_"><a id="_ad93_-anchor" data-anchor-id="_ad93_" class="usa-anchor"></a>Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p">
<span>Supplementary Figure S1</span><p>Characterisation of SMARCA4 mutations in lung cancer cell lines from the CCLE.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11061608/bin/ccr-23-2360_supplementary_figure_s1_suppsf1.pdf" data-ga-action="click_feat_suppl" class="usa-link">ccr-23-2360_supplementary_figure_s1_suppsf1.pdf</a><sup> (1.5MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p">
<span>Supplementary Figure S2</span><p>Initial histopathological evaluation of SCLC xenografts.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11061608/bin/ccr-23-2360_supplementary_figure_s2_suppsf2.pdf" data-ga-action="click_feat_suppl" class="usa-link">ccr-23-2360_supplementary_figure_s2_suppsf2.pdf</a><sup> (2.2MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p">
<span>Supplementary Figure S3</span><p>SCLC-specific markers are weak/lost in SMARCA4-deficient SCLC-Y cell lines.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11061608/bin/ccr-23-2360_supplementary_figure_s3_suppsf3.pdf" data-ga-action="click_feat_suppl" class="usa-link">ccr-23-2360_supplementary_figure_s3_suppsf3.pdf</a><sup> (1.2MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="caption p">
<span>Supplementary Figure S4</span><p>H&amp;E and immunohistochemistry of cell line xenografts H661, H1581 and SBC5.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11061608/bin/ccr-23-2360_supplementary_figure_s4_suppsf4.pdf" data-ga-action="click_feat_suppl" class="usa-link">ccr-23-2360_supplementary_figure_s4_suppsf4.pdf</a><sup> (16.5MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="caption p">
<span>Supplementary Figure S5</span><p>SMARCA4-UT cluster validation.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11061608/bin/ccr-23-2360_supplementary_figure_s5_suppsf5.pdf" data-ga-action="click_feat_suppl" class="usa-link">ccr-23-2360_supplementary_figure_s5_suppsf5.pdf</a><sup> (1.2MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="caption p">
<span>Supplementary Figure S6</span><p>mRNA expression of SMARCA4-UT associated genes and YAP1 IHC.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11061608/bin/ccr-23-2360_supplementary_figure_s6_suppsf6.pdf" data-ga-action="click_feat_suppl" class="usa-link">ccr-23-2360_supplementary_figure_s6_suppsf6.pdf</a><sup> (5.3MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="caption p">
<span>Supplementary Table S1</span><p>Immunohistochemistry antibodies, protocols and evaluation criteria.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11061608/bin/ccr-23-2360_supplementary_table_s1_suppst1.xlsx" data-ga-action="click_feat_suppl" class="usa-link">ccr-23-2360_supplementary_table_s1_suppst1.xlsx</a><sup> (13KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="caption p">
<span>Supplementary Table S2</span><p>Immunohistochemistry scoring table.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11061608/bin/ccr-23-2360_supplementary_table_s2_suppst2.xlsx" data-ga-action="click_feat_suppl" class="usa-link">ccr-23-2360_supplementary_table_s2_suppst2.xlsx</a><sup> (11.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material9_reqid_"><div class="caption p">
<span>Supplementary Table S3</span><p>Clinical profiles and immunohistochemistry of SMARCA4-UT patients.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11061608/bin/ccr-23-2360_supplementary_table_s3_suppst3.xlsx" data-ga-action="click_feat_suppl" class="usa-link">ccr-23-2360_supplementary_table_s3_suppst3.xlsx</a><sup> (10.1KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material10_reqid_"><div class="caption p">
<span>Supplementary Table S4</span><p>Comparison of sequencing data between CCLE and UTSW.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11061608/bin/ccr-23-2360_supplementary_table_s4_suppst4.xlsx" data-ga-action="click_feat_suppl" class="usa-link">ccr-23-2360_supplementary_table_s4_suppst4.xlsx</a><sup> (23KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material11_reqid_"><div class="caption p">
<span>Supplementary Table S5</span><p>CCLE cell lines used in Figure 4 and Supplementary Figure S6A.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11061608/bin/ccr-23-2360_supplementary_table_s5_suppst5.xlsx" data-ga-action="click_feat_suppl" class="usa-link">ccr-23-2360_supplementary_table_s5_suppst5.xlsx</a><sup> (11.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material12_reqid_"><div class="caption p">
<span>Supplementary Table S6</span><p>List of cell lines from Figure 1D and their NE-score.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11061608/bin/ccr-23-2360_supplementary_table_s6_suppst6.xlsx" data-ga-action="click_feat_suppl" class="usa-link">ccr-23-2360_supplementary_table_s6_suppst6.xlsx</a><sup> (13.9KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material13_reqid_"><div class="caption p">
<span>Supplementary Table S7</span><p>26 mutated genes unique to SCLC-Y cell lines.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11061608/bin/ccr-23-2360_supplementary_table_s7_suppst7.xlsx" data-ga-action="click_feat_suppl" class="usa-link">ccr-23-2360_supplementary_table_s7_suppst7.xlsx</a><sup> (10KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material14_reqid_"><div class="caption p">
<span>Supplementary Table S8</span><p>Other mutations and copy number alterations associated with SCLC-Y.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11061608/bin/ccr-23-2360_supplementary_table_s8_suppst8.xlsx" data-ga-action="click_feat_suppl" class="usa-link">ccr-23-2360_supplementary_table_s8_suppst8.xlsx</a><sup> (11.1KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material15_reqid_"><div class="caption p">
<span>Supplementary Table S9</span><p>SMARCA4-UT cell lines and patient samples based on unsupervised hierarchical clustering using YAP1 signature genes.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11061608/bin/ccr-23-2360_supplementary_table_s9_suppst9.xlsx" data-ga-action="click_feat_suppl" class="usa-link">ccr-23-2360_supplementary_table_s9_suppst9.xlsx</a><sup> (10.5KB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The DNA and mRNA expression sequencing data generated by UTSW are available in Supplementary Table S4. The NE score for SCLC cell lines is available in Supplementary Table S6. The data analysed in this study were obtained from Sequence Read Archive (SRA; RRID:SCR_004891) at SRP052896 (<a href="https://www.ncbi.nlm.nih.gov/sra?term=SRP052896" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/sra?term=SRP052896</a>); from database of Genotypes and Phenotypes (dbGaP) (RRID:SCR_002709) at phs001823.v1.p1 (<a href="https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001823.v1.p1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001823.v1.p1</a>); from Dependency Map (DepMap) (RRID:SCR_017655) at 19Q1 (<a href="https://depmap.org/portal/download/all/?releasename=DepMap+Public+19Q1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://depmap.org/portal/download/all/?releasename=DepMap+Public+19Q1</a>), 18Q3 (<a href="https://depmap.org/portal/download/all/?releasename=DepMap+Public+18Q3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://depmap.org/portal/download/all/?releasename=DepMap+Public+18Q3</a>), CCLE2019 (<a href="https://depmap.org/portal/download/all/?releasename=CCLE+2019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://depmap.org/portal/download/all/?releasename=CCLE+2019</a>); from the Broad Institute (RRID:SCR_013836) at CCLE Legacy Data (<a href="https://data.broadinstitute.org/ccle_legacy_data/mRNA_expression/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://data.broadinstitute.org/ccle_legacy_data/mRNA_expression/</a>); and from original articles Nusinow and colleagues (<a href="#bib20" class="usa-link" aria-describedby="bib20" aria-expanded="false">20</a>) at Supplementary Table S2.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Clinical Cancer Research are provided here courtesy of <strong>American Association for Cancer Research</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close">
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a href="https://doi.org/10.1158/1078-0432.CCR-23-2360" class="usa-button usa-button--outline width-24 font-xs usa-link--external padding-left-0 padding-right-0" target="_blank" rel="noreferrer noopener" data-ga-category="actions" data-ga-action="click" data-ga-label="publisher_link_desktop">
                         <span class="height-3 display-inline-flex flex-align-center">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a href="pdf/1846.pdf" class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1" data-ga-category="actions" data-ga-action="click" data-ga-label="pdf_download_desktop">
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden="">
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (34.6&nbsp;MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1" aria-label="Open dialog with citation text in different styles" data-ga-category="actions" data-ga-action="open" data-ga-label="cite_desktop" data-all-citations-url="/resources/citations/11061608/" data-citation-style="nlm" data-download-format-link="/resources/citations/11061608/export/">
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden="">
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty" aria-label="Save article in MyNCBI collections." data-ga-category="actions" data-ga-action="click" data-ga-label="collections_button_desktop" data-collections-open-dialog-enabled="false" data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11061608%2F%23open-collections-dialog" data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden="">
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden="">
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button type="button" class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none" aria-label="Show article permalink" aria-expanded="false" aria-haspopup="true" data-ga-category="actions" data-ga-action="open" data-ga-label="permalink_desktop">
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden="">
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden="">
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11061608/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link" aria-expanded="false">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC11061608/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple="">
                    <h3 class="usa-accordion__heading">
                        <button type="button" class="usa-accordion__button" aria-expanded="false" aria-controls="resources-similar-articles" data-ga-category="resources_accordion" data-ga-action="open_similar_articles" data-ga-label="/articles/PMC11061608/" data-action-open="open_similar_articles" data-action-close="close_similar_articles">
                            Similar articles
                        </button>
                    </h3>
                    <div id="resources-similar-articles" class="usa-accordion__content usa-prose" data-source-url="/resources/similar-article-links/38180245/" hidden="">
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button type="button" class="usa-accordion__button" aria-expanded="false" aria-controls="resources-cited-by-other-articles" data-ga-category="resources_accordion" data-ga-action="open_cited_by" data-ga-label="/articles/PMC11061608/" data-action-open="open_cited_by" data-action-close="close_cited_by">
                             Cited by other articles
                        </button>
                    </h3>
                    <div id="resources-cited-by-other-articles" class="usa-accordion__content usa-prose" data-source-url="/resources/cited-by-links/38180245/" hidden="">
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button type="button" class="usa-accordion__button" aria-expanded="false" aria-controls="resources-links-to-ncbi-databases" data-ga-category="resources_accordion" data-ga-action="open_NCBI_links" data-ga-label="/articles/PMC11061608/" data-action-open="open_NCBI_links" data-action-close="close_NCBI_link">
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div id="resources-links-to-ncbi-databases" class="usa-accordion__content usa-prose" data-source-url="/resources/db-links/11061608/" hidden="">
                        </div>
                    
                    
                </div>
            
        </section>


        <section class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4" data-title-text="On this page" data-title-heading-level="h2" data-scroll-offset="0" data-root-margin="-10% 0px -80% 0px" data-main-content-selector="main" data-threshold="1"><nav aria-label="On this page" class="usa-in-page-nav__nav"><h2 class="usa-in-page-nav__heading" tabindex="0">On this page</h2><ul class="usa-in-page-nav__list"><li class="usa-in-page-nav__item"><a data-anchor-id="abstract1" href="#abstract1" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Abstract">Abstract</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="sec2" href="#sec2" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Introduction">Introduction</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="sec3" href="#sec3" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Materials and Methods">Materials and Methods</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="sec4" href="#sec4" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Results">Results</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="sec5" href="#sec5" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Discussion">Discussion</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="sec20" href="#sec20" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Supplementary Material">Supplementary Material</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="ack1" href="#ack1" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Acknowledgments">Acknowledgments</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="fn-group1" href="#fn-group1" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Footnotes">Footnotes</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="sec6" href="#sec6" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Authors' Disclosures">Authors' Disclosures</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="sec7" href="#sec7" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Authors' Contributions">Authors' Contributions</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="ref-list1" href="#ref-list1" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="References">References</a></li><li class="usa-in-page-nav__item"><a data-anchor-id="_ad93_" href="#_ad93_" class="usa-in-page-nav__link" data-ga-category="article_side_nav" data-ga-action="click" data-ga-label="Associated Data">Associated Data</a></li></ul></nav></section>
    </section>
</div>


        

<div class="overlay citation-dialog-overlay" role="dialog" aria-label="Citation Dialog">
    <div class="dialog citation-dialog" aria-hidden="true" role="document">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto" tabindex="1" data-pinger-ignore="true">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus" data-ga-category="save_share" data-ga-action="cite" data-ga-label="copy" tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a href="#" role="button" class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button" data-ga-category="save_share" data-ga-action="cite" data-ga-label="download" title="Download a file for external citation management software" tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4">
    
      <option data-style-url-name="ama" value="AMA">
        AMA
      </option>
    
      <option data-style-url-name="apa" value="APA">
        APA
      </option>
    
      <option data-style-url-name="mla" value="MLA">
        MLA
      </option>
    
      <option data-style-url-name="nlm" value="NLM" selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
<div class="dots-loading-indicator citation-loading-indicator">
        <div class="dot dot-1"></div>
        <div class="dot dot-2"></div>
        <div class="dot dot-3"></div>
      </div></div>
    </div>
</div>

        <div class="overlay collections-dialog-overlay" role="dialog">
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true" role="document">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form" class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors" data-existing-collections-url="/list-existing-collections/" data-add-to-existing-collection-url="/add-to-existing-collection/" data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/" data-myncbi-max-collection-name-length="100" data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="1K2getTTf9f6vByy6l1bImEFMJ065xkMamgFRl9eihtbuhFkAIunIhdiVHE7AJHt">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio" id="collections-action-dialog-new" class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0" name="collections" value="new" data-ga-category="collections_button" data-ga-action="click" data-ga-label="collections_radio_new">
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio" id="collections-action-dialog-existing" class="usa-radio__input usa-radio__input--tile collections-existing" name="collections" value="existing" checked="true" data-ga-category="collections_button" data-ga-action="click" data-ga-label="collections_radio_existing">
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input type="text" name="add-to-new-collection" id="collections-action-dialog-add-to-new" class="usa-input collections-action-add-to-new" pattern="[^&quot;&amp;=<>/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=<>/" maxlength="" data-ga-category="collections_button" data-ga-action="create_collection" data-ga-label="non_favorties_collection" required="">
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing" class="usa-select collections-action-add-to-existing" data-ga-category="collections_button" data-ga-action="select_collection" data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit" type="submit" data-loading-label="Adding..." data-pinger-ignore="" data-ga-category="collections_button" data-ga-action="click" data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel" aria-label="Close 'Add to Collections' panel" ref="linksrc=close_collections_panel" data-ga-category="collections_button" data-ga-action="click" data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    

<footer class="ncbi-footer ncbi-dark-background ">
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi" class="ncbi-footer__social-icon ncbi-footer__social-icon--gray" target="_blank" rel="noreferrer noopener">
                        <svg width="40" height="40" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm" class="ncbi-footer__social-icon ncbi-footer__social-icon--gray" target="_blank" rel="noreferrer noopener">
                        <svg width="16" height="29" focusable="false" aria-hidden="true" viewBox="0 0 16 29" fill="none" xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm" class="ncbi-footer__social-icon ncbi-footer__social-icon--gray" target="_blank" rel="noreferrer noopener">
                        <svg width="25" height="23" viewBox="0 0 26 24" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi" class="ncbi-footer__social-icon ncbi-footer__social-icon--gray" target="_blank" rel="noreferrer noopener">
                        <svg width="28" height="27" viewBox="0 0 28 28" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/" class="ncbi-footer__social-icon ncbi-footer__social-icon--gray" target="_blank" rel="noreferrer noopener">
                        <svg width="26" height="26" viewBox="0 0 27 27" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer" class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container" data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href="https://twitter.com/nlm_nih" class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32" height="32" viewBox="0 0 40 40" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href="https://www.facebook.com/nationallibraryofmedicine" class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13" height="24" viewBox="0 0 13 24" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href="https://www.youtube.com/user/NLMNIH" class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21" height="15" viewBox="0 0 21 15" fill="none" xmlns="http://www.w3.org/2000/svg" focusable="false" aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z" fill="white"></path>
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external" href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323" rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br> 8600 Rockville Pike<br> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link">
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link">
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target="_blank">
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/?pagename=cloudpmc-viewer%3Apmc%3Aarticle%3Ajournal" class="usa-link  usa-link--alt ncbi-footer__link" data-pinger-pagename-param="true">
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link">
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link">
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link">
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link">
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target="_blank">
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target="_blank">
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>



        
        
    
  <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script><div id="ZN_dikYWqsjiUWN0Q5"></div>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
     <script src="https://code.jquery.com/jquery-3.5.0.min.js" integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ=" crossorigin="anonymous">
    </script>
    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-9bea7450.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-722d91a2.js"></script>
    
    

    

<div class="fake-body-scroll"></div><div class="fake-body-scroll"></div><script id="_fed_an_ua_tag" text="" charset="" type="text/javascript" src="https://dap.digitalgov.gov/Universal-Federated-Analytics-Min.js?agency=HHS&amp;subagency=NCBI%20-%20ncbi.nlm.nih.gov&amp;sitetopic=NCBI%20Pinger%200.39.3&amp;siteplatform=NCBI%20Pinger%200.39.3"></script><script type="text/javascript" src="https://zndikYWqsjiUWN0Q5-nlmenterprise.siteintercept.qualtrics.com/SIE/?Q_ZID=ZN_dikYWqsjiUWN0Q5"></script><script src="https://siteintercept.qualtrics.com/dxjsmodule/CoreModule.js?Q_CLIENTVERSION=2.20.0&amp;Q_CLIENTTYPE=web&amp;Q_BRANDID=nlmenterprise" defer=""></script><script src="https://siteintercept.qualtrics.com/dxjsmodule/FeedbackButtonModule.js?Q_CLIENTVERSION=2.20.0&amp;Q_CLIENTTYPE=web&amp;Q_BRANDID=nlmenterprise" defer=""></script><style type="text/css">.QSIFeedbackButton div,.QSIFeedbackButton dl,.QSIFeedbackButton dt,.QSIFeedbackButton dd,.QSIFeedbackButton ul,.QSIFeedbackButton ol,.QSIFeedbackButton li,.QSIFeedbackButton h1,.QSIFeedbackButton h2,.QSIFeedbackButton h3,.QSIFeedbackButton h4,.QSIFeedbackButton h5,.QSIFeedbackButton h6,.QSIFeedbackButton span,.QSIFeedbackButton pre,.QSIFeedbackButton form,.QSIFeedbackButton fieldset,.QSIFeedbackButton textarea,.QSIFeedbackButton p,.QSIFeedbackButton blockquote,.QSIFeedbackButton tr,.QSIFeedbackButton th,.QSIFeedbackButton td{ margin: 0; padding: 0;background-color: transparent; border: 0; font-size: 12px; line-height: normal; vertical-align:baseline; box-shadow: none; }.QSIFeedbackButton img{ height: auto; width: auto; margin: 0; padding: 0 }.QSIFeedbackButton ul,.QSIFeedbackButton ol{ margin: 12px 0; padding-left: 40px; }.QSIFeedbackButton ul li{ list-style-type: disc; }.QSIFeedbackButton ol li{ list-style-type: decimal; }.QSIFeedbackButton .scrollable{ -webkit-overflow-scrolling: touch; }.QSIFeedbackButton table{ border-collapse: collapse; border-spacing: 0; }.QSIFeedbackButton table td{ padding: 2px; }.QSIFeedbackButton iframe{ max-height: none; }.QSIFeedbackButton *{ box-sizing: content-box; }</style><div class="QSIFeedbackButton" style="position: fixed; visibility: hidden; inset: 0px; display: flex; flex-direction: column; justify-content: flex-end; align-items: flex-start; margin: 0px; padding: 0px; z-index: 2000000000;"><button role="button" style="position: fixed; visibility: visible; cursor: pointer; border: medium; background-color: transparent; padding: 0px; margin: 0px; bottom: 97px; right: -1px; width: 52px;" aria-label="Tell us what you think!" id="QSIFeedbackButton-btn"><img src="https://iad1.qualtrics.com/WRQualtricsSiteIntercept/Graphic.php?IM=IM_3xWqwNkIHc7cA0S" alt="Tell us what you think!" style="max-width: 110px; max-height: 110px; display: block;"></button></div></body></html>